

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Adaptation of HIV-1 Depends on the Host-Cell Environment</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0000271"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0000271" />

    <meta name="citation_title" content="Adaptation of HIV-1 Depends on the Host-Cell Environment"/>
    <meta itemprop="name" content="Adaptation of HIV-1 Depends on the Host-Cell Environment"/>

      <meta name="citation_author" content="Tim van Opijnen"/>
            <meta name="citation_author_institution" content="Department of Human Retrovirology, Academic Medical Center, University of Amsterdam, Amsterdam, The Netherlands"/>
            <meta name="citation_author_institution" content="Section Population Biology, Institute for Biodiversity and Ecosystem Dynamics, University of Amsterdam, Amsterdam, The Netherlands"/>
      <meta name="citation_author" content="Anthony de Ronde"/>
            <meta name="citation_author_institution" content="Department of Human Retrovirology, Academic Medical Center, University of Amsterdam, Amsterdam, The Netherlands"/>
      <meta name="citation_author" content="Maarten C. Boerlijst"/>
            <meta name="citation_author_institution" content="Section Population Biology, Institute for Biodiversity and Ecosystem Dynamics, University of Amsterdam, Amsterdam, The Netherlands"/>
      <meta name="citation_author" content="Ben Berkhout"/>
            <meta name="citation_author_institution" content="Department of Human Retrovirology, Academic Medical Center, University of Amsterdam, Amsterdam, The Netherlands"/>

    <meta name="citation_date" content="2007/3/7"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0000271.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e271"/>
    <meta name="citation_issue" content="3"/>
    <meta name="citation_volume" content="2"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=HIV long-term non-progressors maintain brisk CD8 T cell responses to other viral antigens.; citation_author=H Valdez; citation_author=NL Carlson; citation_author=AB Post; citation_author=R Asaad; citation_author=PS Heeger; citation_journal_title=AIDS; citation_volume=16; citation_number=1; citation_pages=1113-1118; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Severe CD4+T-cell depletion in gut lymphoid tissue during primary human immunodeficiency virus type 1 infection and substantial delay in restoration following highly active antiretroviral therapy.; citation_author=M Guadalupe; citation_author=E Reay; citation_author=S Sankaran; citation_author=T Prindiville; citation_author=J Flamm; citation_journal_title=J Virol; citation_volume=77; citation_number=2; citation_pages=11708-11717; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Role of avidity and breadth of the CD4 T cell response in progression to AIDS.; citation_author=AH Korthals; citation_author=R de Boer; citation_author=M Boerlijst; citation_journal_title=Proc Biol Sci; citation_volume=273; citation_number=3; citation_pages=1697-1704; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=The host environment drives HIV-1 fitness.; citation_author=T Van Opijnen; citation_author=B Berkhout; citation_journal_title=RevMed Virol; citation_volume=15; citation_number=4; citation_pages=219-233; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=HIV-1 dynamics in vivo: virion clearance rate, infected cell life-span, and viral generation time.; citation_author=AS Perelson; citation_author=AU Neumann; citation_author=M Markowitz; citation_author=JM Leonard; citation_author=DD Ho; citation_journal_title=Science; citation_volume=271; citation_number=5; citation_pages=1582-1586; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Coalescent estimates of HIV-1 generation time in vivo.; citation_author=AG Rodrigo; citation_author=EG Shpaer; citation_author=EL Delwart; citation_author=AK Iversen; citation_author=MV Gallo; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=96; citation_number=6; citation_pages=2187-2191; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Rapid turnover of plasma virions and CD4 lymphocytes in HIV-1 infection.; citation_author=DD Ho; citation_author=AU Neumann; citation_author=AS Perelson; citation_author=W Chen; citation_author=JM Leonard; citation_journal_title=Nature; citation_volume=373; citation_number=7; citation_pages=123-126; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Viral dynamics in human immunodeficiency virus type 1 infection.; citation_author=X Wei; citation_author=SK Ghosh; citation_author=ME Taylor; citation_author=VA Johnson; citation_author=EA Emini; citation_journal_title=Nature; citation_volume=373; citation_number=8; citation_pages=117-126; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Lower in vivo mutation rate of human immunodeficiency virus type 1 than predicted from the fidelity of purified reverse transcriptase.; citation_author=LM Mansky; citation_author=HM Temin; citation_journal_title=The Journal of Virology; citation_volume=69; citation_number=9; citation_pages=5087-5094; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Retroviral recombination can lead to linkage of reverse transcriptase mutations that confer increased zidovudine resistance.; citation_author=P Kellam; citation_author=BA Larder; citation_journal_title=J Virol; citation_volume=69; citation_number=10; citation_pages=669-674; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Recombination leads to the rapid emergence of HIV-1 dually resistant mutants under selective drug pressure.; citation_author=L Moutouh; citation_author=J Corbeil; citation_author=DD Richman; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=93; citation_number=11; citation_pages=6106-6111; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Human immunodeficiency virus type 1 subtypes have a distinct long terminal repeat that determines the replication rate in a host-cell-specific manner.; citation_author=T Van Opijnen; citation_author=RE Jeeninga; citation_author=MC Boerlijst; citation_author=GP Pollakis; citation_author=V Zetterberg; citation_journal_title=J Virol; citation_volume=78; citation_number=12; citation_pages=3675-3683; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=The host cell MAP kinase ERK-2 regulates viral assembly and release by phosphorylating the p6gag protein of HIV-1.; citation_author=B Hemonnot; citation_author=C Cartier; citation_author=B Gay; citation_author=S Rebuffat; citation_author=M Bardy; citation_journal_title=J Biol Chem; citation_volume=279; citation_number=13; citation_pages=32426-32434; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Dynamics of HIV-1 recombination in its natural target cells.; citation_author=DN Levy; citation_author=GM Aldrovandi; citation_author=O Kutsch; citation_author=GM Shaw; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=101; citation_number=14; citation_pages=4204-4209; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Mechanisms of retroviral recombination.; citation_author=M Negroni; citation_author=H Buc; citation_journal_title=AnnuRevGenet; citation_volume=35; citation_number=15; citation_pages=275-302; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Viral quasi-species and recombination.; citation_author=MC Boerlijst; citation_author=S Bonhoeffer; citation_author=MA Nowak; citation_journal_title=Proc R SocLond B Biol Sci; citation_volume=263; citation_number=16; citation_pages=1577-1584; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Biological and molecular characterization of subtype D, G, and A/D recombinant HIV-1 transmissions in Sweden.; citation_author=T Leitner; citation_author=D Escanilla; citation_author=S Marquina; citation_author=J Wahlberg; citation_author=C Brostrom; citation_journal_title=Virology; citation_volume=209; citation_number=17; citation_pages=136-146; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Extensive recombination among human immunodeficiency virus type 1 quasispecies makes an important contribution to viral diversity in individual patients.; citation_author=C Charpentier; citation_author=T Nora; citation_author=O Tenaillon; citation_author=F Clavel; citation_author=AJ Hance; citation_journal_title=J Virol; citation_volume=80; citation_number=18; citation_pages=2472-2482; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Comparison of in vivo plasma and peripheral blood mononuclear cell HIV-1 quasispecies to short-term tissue culture isolates: an analysis of tat and C2-V3 env regions.; citation_author=E Sabino; citation_author=LZ Pan; citation_author=C Cheng-Mayer; citation_author=A Mayer; citation_journal_title=AIDS; citation_volume=8; citation_number=19; citation_pages=901-909; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=Evolution of human immunodeficiency virus under selection and weak recombination.; citation_author=IM Rouzine; citation_author=JM Coffin; citation_journal_title=Genetics; citation_volume=170; citation_number=20; citation_pages=7-18; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=A genetic-algorithm approach to simulating human immunodeficiency virus evolution reveals the strong impact of multiply infected cells and recombination.; citation_author=G Bocharov; citation_author=NJ Ford; citation_author=J Edwards; citation_author=T Breinig; citation_author=S Wain-Hobson; citation_journal_title=J Gen Virol; citation_volume=86; citation_number=21; citation_pages=3109-3118; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Stochastic interplay between mutation and recombination during the acquisition of drug resistance mutations in human immunodeficiency virus type 1.; citation_author=CL Althaus; citation_author=S Bonhoeffer; citation_journal_title=J Virol; citation_volume=79; citation_number=22; citation_pages=13572-13578; citation_date=2005; " />
      <meta name="citation_reference" content="citation_author=RA Fisher; citation_number=23; citation_date=1930; citation_publisher=Oxford University Press; " />
      <meta name="citation_reference" content="citation_title=Some genetic aspects of sex.; citation_author=HJ Muller; citation_journal_title=American Naturalist; citation_volume=66; citation_number=24; citation_pages=118-138; citation_date=1932; " />
      <meta name="citation_reference" content="citation_title=Resolving the paradox of sex and recombination.; citation_author=SP Otto; citation_author=T Lenormand; citation_journal_title=NatRev Genet; citation_volume=3; citation_number=25; citation_pages=252-261; citation_date=2002; " />
      <meta name="citation_reference" content="citation_author=J Maynard Smith; citation_number=26; citation_date=1978; citation_publisher=Cambridge University Press; " />
      <meta name="citation_reference" content="citation_title=Experimental tests of the adaptive significance of sexual recombination.; citation_author=WR Rice; citation_journal_title=NatRev Genet; citation_volume=3; citation_number=27; citation_pages=241-251; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Evolutionin sexual and asexual populations.; citation_author=J Maynard Smith; citation_journal_title=American Naturalist; citation_volume=102; citation_number=28; citation_pages=469-473; citation_date=1968; " />
      <meta name="citation_reference" content="citation_title=On the evolutionary effect of recombination.; citation_author=I Eshel; citation_author=MW Feldman; citation_journal_title=TheorPopulBiol; citation_volume=1; citation_number=29; citation_pages=88-100; citation_date=1970; " />
      <meta name="citation_reference" content="citation_title=Changes in growth properties on passage in tissue culture of viruses derived from infectious molecular clones of HIV-1 LAI , HIV-1 MAL , and HIV-1 ELI; citation_author=K Peden; citation_author=M Emerman; citation_author=L Montagnier; citation_journal_title=Virology; citation_volume=185; citation_number=30; citation_pages=661-672; citation_date=1991; " />
      <meta name="citation_reference" content="citation_title=Contribution of NF-kappa B and Sp1 binding motifs to the replicative capacity of human immunodeficiency virus type 1: distinct patterns of viral growth are determined by T-cell types.; citation_author=EK Ross; citation_author=AJ Buckler White; citation_author=AB Rabson; citation_author=G Englund; citation_author=MA Martin; citation_journal_title=The Journal of Virology; citation_volume=65; citation_number=31; citation_pages=4350-4358; citation_date=1991; " />
      <meta name="citation_reference" content="citation_title=Monoclonal antibody and enzymatic profiles of human malignant T-lymphoid cells and derived cell lines.; citation_author=SD Smith; citation_author=M Shatsky; citation_author=PS Cohen; citation_author=R Warnke; citation_author=MP Link; citation_journal_title=Cancer Res; citation_volume=44; citation_number=32; citation_pages=5657-5660; citation_date=1984; " />
      <meta name="citation_reference" content="citation_title=Effects of random mutations in the Human Immunodeficiency Virus type I transcriptional promoter on viral fitness in different host-cell environments.; citation_author=T Van Opijnen; citation_author=MC Boerlijst; citation_author= Berkhout; citation_journal_title=J Virol; citation_number=33; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Experimental evolution of parasites.; citation_author=D Ebert; citation_journal_title=Science; citation_volume=282; citation_number=34; citation_pages=1432-1435; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=New concepts for dealing with the evolution of nucleic acids.; citation_author=M Eigen; citation_journal_title=Cold Spring HarbSympQuantBiol; citation_volume=52; citation_number=35; citation_pages=307-320; citation_date=1987; " />
      <meta name="citation_reference" content="citation_author=SA Kauffman; citation_number=36; citation_date=1993; citation_publisher=Oxford Univ. Press; " />
      <meta name="citation_reference" content="citation_title=Statistical tests of neutrality of mutations.; citation_author=YX Fu; citation_author=WH Li; citation_journal_title=Genetics; citation_volume=133; citation_number=37; citation_pages=693-709; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=Clonal interference and the evolution of RNA viruses.; citation_author=R Miralles; citation_author=PJ Gerrish; citation_author=A Moya; citation_author=SF Elena; citation_journal_title=Science; citation_volume=285; citation_number=38; citation_pages=1745-1747; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Diminishing returns of population size in the rate of RNA virus adaptation.; citation_author=R Miralles; citation_author=A Moya; citation_author=SF Elena; citation_journal_title=J Virol; citation_volume=74; citation_number=39; citation_pages=3566-3571; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=The effect of linkage on limits to artificial selection.; citation_author=WG Hill; citation_author=A Robertson; citation_journal_title=GenetRes; citation_volume=8; citation_number=40; citation_pages=269-294; citation_date=1966; " />
      <meta name="citation_reference" content="citation_title=Linkage and the limits to natural selection.; citation_author=NH Barton; citation_journal_title=Genetics; citation_volume=140; citation_number=41; citation_pages=821-841; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=The fate of competing beneficial mutations in an asexual population.; citation_author=PJ Gerrish; citation_author=RE Lenski; citation_journal_title=Genetica; citation_volume=102–103; citation_number=42; citation_pages=127-144; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=The rate of adaptation in asexuals.; citation_author=HA Orr; citation_journal_title=Genetics; citation_volume=155; citation_number=43; citation_pages=961-968; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Selective sweeps in the presence of interference among partially linked loci.; citation_author=Y Kim; citation_author=W Stephan; citation_journal_title=Genetics; citation_volume=164; citation_number=44; citation_pages=389-398; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Diminishing returns from mutation supply rate in asexual populations.; citation_author=JA de Visser; citation_author=CW Zeyl; citation_author=PJ Gerrish; citation_author=JL Blanchard; citation_author=RE Lenski; citation_journal_title=Science; citation_volume=283; citation_number=45; citation_pages=404-406; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=The evolution of recombination: removing the limits to natural selection.; citation_author=SP Otto; citation_author=NH Barton; citation_journal_title=Genetics; citation_volume=147; citation_number=46; citation_pages=879-906; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Selfish genes, pleiotropy and the origin of recombination.; citation_author=J Hey; citation_journal_title=Genetics; citation_volume=149; citation_number=47; citation_pages=2089-2097; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Muller's ratchet, epistasis and mutation effects.; citation_author=D Butcher; citation_journal_title=Genetics; citation_volume=141; citation_number=48; citation_pages=431-437; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=On the speed of Muller's ratchet.; citation_author=I Gordo; citation_author=B Charlesworth; citation_journal_title=Genetics; citation_volume=156; citation_number=49; citation_pages=2137-2140; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=The constraints of finite size in asexual populations and the rate of the ratchet.; citation_author=DD Gessler; citation_journal_title=GenetRes; citation_volume=66; citation_number=50; citation_pages=241-253; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Size of genetic bottlenecks leading to virus fitness loss is determined by mean initial population fitness.; citation_author=IS Novella; citation_author=SF Elena; citation_author=A Moya; citation_author=E Domingo; citation_author=JJ Holland; citation_journal_title=J Virol; citation_volume=69; citation_number=51; citation_pages=2869-2872; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=The roles of mutation, inbreeding, crossbreeding and selection in evolution.; citation_author=S Wright; citation_journal_title=Proc Sixth Int Congress Genetics; citation_volume=1; citation_number=52; citation_pages=356-366; citation_date=1932; " />
      <meta name="citation_reference" content="citation_title=Reduced replication of 3TC-resistant HIV-1 variants in primary cells due to a processivity defect of the reverse transcriptase enzyme.; citation_author=NKT Back; citation_author=M Nijhuis; citation_author=W Keulen; citation_author=CAB Boucher; citation_author=BB Oude Essink; citation_journal_title=The EMBO Journal; citation_volume=15; citation_number=53; citation_pages=4040-4049; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Ordered appearance of zidovudine resistance mutations during treatment of 18 human immunodeficiency virus-positive subjects.; citation_author=CAB Boucher; citation_author=E O'Sullivan; citation_author=JW Mulder; citation_author=C Ramautarsing; citation_author=P Kellam; citation_journal_title=Journal of Infectious Diseases; citation_volume=165; citation_number=54; citation_pages=105-110; citation_date=1992; " />
      <meta name="citation_reference" content="citation_title=Interactions between drug resistance mutations in human immunodeficiency virus type 1 reverse transcriptase.; citation_author=BA Larder; citation_journal_title=Journal of General Virology; citation_volume=75; citation_number=55; citation_pages=951-957; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=Nucleotide substitution patterns can predict the requirements for drug-resistance of HIV-1 proteins.; citation_author=W Keulen; citation_author=C Boucher; citation_author=B Berkhout; citation_journal_title=Antiviral Research; citation_volume=31; citation_number=56; citation_pages=45-57; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Impaired fitness of human immunodeficiency virus type 1 variants with high-level resistance to protease inhibitors.; citation_author=G Croteau; citation_author=L Doyon; citation_author=D Thibeault; citation_author=G McKercher; citation_author=L Pilote; citation_journal_title=The Journal of Virology; citation_volume=71; citation_number=57; citation_pages=1089-1096; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Novel Gag-Pol frameshift site in Human Immunodeficiency Virus type 1 variants resistant to Protease inhibitors.; citation_author=L Doyon; citation_author=C Payant; citation_author=L Brakier-Gingras; citation_author=D Lamarre; citation_journal_title=The Journal of Virology; citation_volume=72; citation_number=58; citation_pages=6146-6150; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Resistance-associated loss of viral fitness in human immunodeficiency virus type 1: phenotypic analysis of Protease and gag coevolution in protease inhibitor-treated patients.; citation_author=F Mammano; citation_author=C Petit; citation_author=F Clavel; citation_journal_title=The Journal of Virology; citation_volume=72; citation_number=59; citation_pages=7632-7637; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Individual contributions of mutant protease and reverse transcriptase to viral infectivity, replication, and protein maturation of antiretroviral drug-resistant Human Immunodeficiency Virus type 1.; citation_author=G Bleiber; citation_author=M Munoz; citation_author=A Ciuffi; citation_author=P Meylan; citation_author=A Telenti; citation_journal_title=The Journal of Virology; citation_volume=75; citation_number=60; citation_pages=3291-3300; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=HIV-1 evolution under pressure of protease inhibitors, climbing the stairs of viral fitness.; citation_author=B Berkhout; citation_journal_title=Journal Biomedical Science; citation_volume=6; citation_number=61; citation_pages=298-305; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Virologic and immunologic consequences of discontinuing combination antiretroviral-drug therapy in HIV-infected patients with detectable viremia.; citation_author=SG Deeks; citation_author=T Wrin; citation_author=T Liegler; citation_author=R Hoh; citation_author=M Hayden; citation_journal_title=The New England Journal of Medicine; citation_volume=344; citation_number=62; citation_pages=472-480; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Should patients with drug-resistant HIV-1 continue to receive antiretroviral therapy?; citation_author=LM Frenkel; citation_author=JI Mullins; citation_journal_title=The New England Journal of Medicine; citation_volume=344; citation_number=63; citation_pages=520-522; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=The challenge of antiretroviral-drug-resistant HIV: is there any possible clinical advantage?; citation_author=M Zaccarelli; citation_author=V Tozzi; citation_author=CF Perno; citation_author=A Antinori; citation_journal_title=CurrHIV Res; citation_volume=2; citation_number=64; citation_pages=283-292; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Can modulation of viral fitness represent a target for anti-HIV-1 strategies?; citation_author=M Clementi; citation_journal_title=New Microbiol; citation_volume=27; citation_number=65; citation_pages=207-214; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Rapid in vitro selection of human immunodeficiency virus type 1 resistant to 3′-thiacytidine inhibitors due to a mutation in the YMDD region of reverse transcriptase.; citation_author=M Tisdale; citation_author=SD Kemp; citation_author=NR Parry; citation_author=BA Larder; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=90; citation_number=66; citation_pages=5653-5656; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=Increased fitness of drug resistant HIV-1 protease as a result of acquisition of compensatory mutations during suboptimal therapy.; citation_author=M Nijhuis; citation_author=R Schuurman; citation_author=D de Jong; citation_author=J Erickson; citation_author=E Gustchina; citation_journal_title=AIDS; citation_volume=13; citation_number=67; citation_pages=2349-2359; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Deleterious mutations as an evolutionary factor. 1. The advantage of recombination.; citation_author=AS Kondrashov; citation_journal_title=GenetRes; citation_volume=44; citation_number=68; citation_pages=199-217; citation_date=1984; " />
      <meta name="citation_reference" content="citation_title=Requisite mutational load, pathway epistasis and deterministic mutation accumulation in sexual versus asexual populations.; citation_author=WR Rice; citation_journal_title=Genetica; citation_volume=102–103; citation_number=69; citation_pages=71-81; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=The effect of background selection against deleterious mutations on weakly selected, linked variants.; citation_author=B Charlesworth; citation_journal_title=GenetRes; citation_volume=63; citation_number=70; citation_pages=213-227; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=The pattern of neutral molecular variation under the background selection model.; citation_author=D Charlesworth; citation_author=B Charlesworth; citation_author=MT Morgan; citation_journal_title=Genetics; citation_volume=141; citation_number=71; citation_pages=1619-1632; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=A ruby in the rubbish: beneficial mutations, deleterious mutations and the evolution of sex.; citation_author=JR Peck; citation_journal_title=Genetics; citation_volume=137; citation_number=72; citation_pages=597-606; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=Possible involvement of cell fusion and viral recombination in generation of human immunodeficiency virus variants that display dual resistance to AZT and 3TC.; citation_author=Z Gu; citation_author=Q Gao; citation_author=EA Faust; citation_author=MA Wainberg; citation_journal_title=J Gen Virol; citation_volume=76 ( Pt 10); citation_number=73; citation_pages=2601-2605; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Genome-wide screen identifies host genes affecting viral RNA recombination.; citation_author=E Serviene; citation_author=N Shapka; citation_author=CP Cheng; citation_author=T Panavas; citation_author=B Phuangrat; citation_journal_title=Proc Natl Acad Sci USA; citation_number=74; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Functional differences between the LTR transcriptional promoters of HIV-1 subtypes A through G.; citation_author=RE Jeeninga; citation_author=M Hoogenkamp; citation_author=M Armand-Ugon; citation_author=M de Baar; citation_author=K Verhoef; citation_journal_title=The Journal of Virology; citation_volume=74; citation_number=75; citation_pages=3740-3751; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=The k B sites in the Human Immunodeficiency Virus type 1 Long Terminal Repeat enhance virus replication yet are not absolutely required for viral growth.; citation_author=BK Chen; citation_author=MB Feinberg; citation_author=D Baltimore; citation_journal_title=The Journal of Virology; citation_volume=71; citation_number=76; citation_pages=5495-5504; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Differential patterns of human cytomegalovirus gene expression in various T-cell lines carrying human T-cell leukemia-lymphoma virus type I: role of Tax-activated cellular transcription factors.; citation_author=Z Beck; citation_author=A Bacsi; citation_author=X Liu; citation_author=P Ebbesen; citation_author=I Andirko; citation_journal_title=J Med Virol; citation_volume=71; citation_number=77; citation_pages=94-104; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Roscovitine inhibits STAT5 activity and induces apoptosis in the human leukemia virus type 1-transformed cell line MT-2.; citation_author=S Mohapatra; citation_author=B Chu; citation_author=S Wei; citation_author=J Djeu; citation_author=PK Epling-Burnette; citation_journal_title=Cancer Res; citation_volume=63; citation_number=78; citation_pages=8523-8530; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Transcription factor expression in cell lines derived from natural killer-cell and natural killer-like T-cell leukemia-lymphoma.; citation_author=Y Matsuo; citation_author=HG Drexler; citation_author=A Harashima; citation_author=A Okochi; citation_author=N Shimizu; citation_journal_title=Hum Cell; citation_volume=17; citation_number=79; citation_pages=85-92; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Quantitative detection of HIV-1 drug resistance mutations by automated DNA sequencing.; citation_author=BA Larder; citation_author=A Kohli; citation_author=P Kellam; citation_author=SD Kemp; citation_author=M Kronick; citation_journal_title=Nature; citation_volume=365; citation_number=80; citation_pages=671-673; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=Molecular evolutionary genetics analysis (MEGA).; citation_author=S Kumar; citation_author=K Tamura; citation_author=M Nei; citation_journal_title=ComputApplBiosc; citation_volume=10; citation_number=81; citation_pages=189-191; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=DNA polymorphism analyses by the coalescent and other methods.; citation_author=JC Rozas J Fau-Sanchez-DelBarrio; citation_author=X Sanchez-DelBarrio Jc Fau-Messeguer; citation_author=R Messeguer X Fau-Rozas; citation_author=SP Rozas R Dna; citation_number=82; " />
      <meta name="citation_reference" content="citation_title=Long-Term experimental evolution in Escherichia coli. I. Adaptation and divergence during 2,000 generations.; citation_author=RE Lenski; citation_author=MRS Rose; citation_author=SC S.C.Tadler; citation_journal_title=The American Naturalist; citation_volume=138; citation_number=83; citation_pages=1315-1341; citation_date=1991; " />
      <meta name="citation_reference" content="citation_title=Estimating relative fitness in viral competition experiments.; citation_author=AF Maree; citation_author=W Keulen; citation_author=CA Boucher; citation_author=RJ de Boer; citation_journal_title=J Virol; citation_volume=74; citation_number=84; citation_pages=11067-11072; citation_date=2000; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000271" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Adaptation of HIV-1 Depends on the Host-Cell Environment"/>
    <meta name="twitter:description" content="Many viruses have the ability to rapidly develop resistance against antiviral drugs and escape from the host immune system. To which extent the host environment affects this adaptive potential of viruses is largely unknown. Here we show that for HIV-1, the host-cell environment is key to the adaptive potential of the virus. We performed a large-scale selection experiment with two HIV-1 strains in two different T-cell lines (MT4 and C8166). Over 110 days of culture, both virus strains adapted rapidly to the MT4 T-cell line. In contrast, when cultured on the C8166 T-cell line, the same strains did not show any increase in fitness. By sequence analyses and infections with viruses expressing either yellow or cyan fluorescent protein, we were able to show that the absence of adaptation was linked to a lower recombination rate in the C8166 T-cell line. Our findings suggest that if we can manipulate the host-cellular factors that mediate viral evolution, we may be able to significantly retard viral adaptability."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0000271.g002"/>

  <meta property="og:title" content="Adaptation of HIV-1 Depends on the Host-Cell Environment" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000271" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=2544'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=7725'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=7230&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0000271">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000271" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000271&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Adaptation of HIV-1 Depends on the Host-Cell Environment
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Tim van Opijnen
              <span class="corresponding">mail</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:Tim.van_Opijnen@tufts.edu">Tim.van_Opijnen@tufts.edu</a></p>

                <p>Affiliations:
                  Department of Human Retrovirology, Academic Medical Center, University of Amsterdam, Amsterdam, The Netherlands, 
                  Section Population Biology, Institute for Biodiversity and Ecosystem Dynamics, University of Amsterdam, Amsterdam, The Netherlands
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Anthony de Ronde, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Human Retrovirology, Academic Medical Center, University of Amsterdam, Amsterdam, The Netherlands
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Maarten C. Boerlijst, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Section Population Biology, Institute for Biodiversity and Ecosystem Dynamics, University of Amsterdam, Amsterdam, The Netherlands
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Ben Berkhout
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Human Retrovirology, Academic Medical Center, University of Amsterdam, Amsterdam, The Netherlands
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: March 07, 2007</li>
    <li>DOI: 10.1371/journal.pone.0000271</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0000271" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0000271" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000271" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0000271" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0000271-t001" data-doi="info:doi/10.1371/journal.pone.0000271" data-uri="info:doi/10.1371/journal.pone.0000271.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000271.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000271-t002" data-doi="info:doi/10.1371/journal.pone.0000271" data-uri="info:doi/10.1371/journal.pone.0000271.t002" title="Table 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000271.t002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000271-g001" data-doi="info:doi/10.1371/journal.pone.0000271" data-uri="info:doi/10.1371/journal.pone.0000271.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000271.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000271-t003" data-doi="info:doi/10.1371/journal.pone.0000271" data-uri="info:doi/10.1371/journal.pone.0000271.t003" title="Table 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000271.t003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000271-t004" data-doi="info:doi/10.1371/journal.pone.0000271" data-uri="info:doi/10.1371/journal.pone.0000271.t004" title="Table 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000271.t004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000271-t005" data-doi="info:doi/10.1371/journal.pone.0000271" data-uri="info:doi/10.1371/journal.pone.0000271.t005" title="Table 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000271.t005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000271-g002" data-doi="info:doi/10.1371/journal.pone.0000271" data-uri="info:doi/10.1371/journal.pone.0000271.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000271.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000271-t006" data-doi="info:doi/10.1371/journal.pone.0000271" data-uri="info:doi/10.1371/journal.pone.0000271.t006" title="Table 6">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000271.t006&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000271">Reader Comments (0)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0000271" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1"></a><p>Many viruses have the ability to rapidly develop resistance against antiviral drugs and escape from the host immune system. To which extent the host environment affects this adaptive potential of viruses is largely unknown. Here we show that for HIV-1, the host-cell environment is key to the adaptive potential of the virus. We performed a large-scale selection experiment with two HIV-1 strains in two different T-cell lines (MT4 and C8166). Over 110 days of culture, both virus strains adapted rapidly to the MT4 T-cell line. In contrast, when cultured on the C8166 T-cell line, the same strains did not show any increase in fitness. By sequence analyses and infections with viruses expressing either yellow or cyan fluorescent protein, we were able to show that the absence of adaptation was linked to a lower recombination rate in the C8166 T-cell line. Our findings suggest that if we can manipulate the host-cellular factors that mediate viral evolution, we may be able to significantly retard viral adaptability.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>van Opijnen T, de Ronde A, Boerlijst MC, Berkhout B (2007) Adaptation of HIV-1 Depends on the Host-Cell Environment. PLoS ONE 2(3):
          e271.
            doi:10.1371/journal.pone.0000271</p><p><strong>Academic Editor: </strong>Frank Maldarelli, National Cancer Institute at Frederick, United States of America</p><p><strong>Received:</strong> August 7, 2006; <strong>Accepted:</strong> February 14, 2007; <strong>Published:</strong> March 7, 2007</p><p><strong>Copyright:</strong> © 2007 van Opijnen et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>This research was supported by NWO-ALW, project number 811.35.001, as well as by ZonMw (vici-grant) and NWO-CW (TOP-grant).</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>Disease progression in HIV-1 infected patients shows extensive variation; some patients efficiently suppress HIV-1 for more than 15 years while others develop AIDS within several years <a href="#pone.0000271-Valdez1">[1]</a>, <a href="#pone.0000271-Guadalupe1">[2]</a>. Disease progression is related to the viral set-point and viral load, and patient to patient variability can be explained, to some extent, by the efficacy of the host-immune response <a href="#pone.0000271-Korthals1">[3]</a>, which is influenced by <em>e.g.</em> the CD4+repertoire, CD8+repertoire and antibodies <a href="#pone.0000271-VanOpijnen1">[4]</a>. Without intervention, the virus almost always brakes through the host defenses and the patient will eventually develop AIDS. One of the most important and hard to battle characteristics of HIV-1 is its large adaptive potential. It is this potential that makes the virus escape from the host-immune system and resistant to anti-retroviral drugs. Several viral-characteristics that contribute to HIV-1′s adaptive potential are its short generation time <a href="#pone.0000271-Perelson1">[5]</a>, <a href="#pone.0000271-Rodrigo1">[6]</a>, high copy numbers <a href="#pone.0000271-Ho1">[7]</a>, <a href="#pone.0000271-Wei1">[8]</a>, and high mutation rate <a href="#pone.0000271-Mansky1">[9]</a>. In addition, recombination may be important, for example, in combining resistance mutations into a single genome to achieve drug-resistance <a href="#pone.0000271-Kellam1">[10]</a>, <a href="#pone.0000271-Moutouh1">[11]</a>. The host may also affect the adaptive potential of the virus. For instance different target cell environments support differential viral replication rates <a href="#pone.0000271-VanOpijnen2">[12]</a>, generation time <a href="#pone.0000271-Hemonnot1">[13]</a> and recombination rates <a href="#pone.0000271-Levy1">[14]</a>. Thus, the adaptive potential of HIV-1 is both shaped by the virus and the interaction between the virus and the host-environment. To which extent the host affects the adaptability of HIV-1, however, is still largely unexplored.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>One viral trait that contributes to an unknown degree to the adaptive potential of HIV-1 is viral recombination. For effective recombination to occur in HIV-1, a single host-cell needs to be infected with two distinct viral strains. This double infected cell can then produce viral particles with a mixed dimeric RNA genome. Subsequently, cells infected by such heterozygous virions can obtain a hybrid provirus due to template switching during reverse transcription <a href="#pone.0000271-Negroni1">[15]</a>, <a href="#pone.0000271-Boerlijst1">[16]</a>. Recombination may be important, for example, in combining resistance mutations into a single genome to achieve fully drug-resistant strains <a href="#pone.0000271-Kellam1">[10]</a>, <a href="#pone.0000271-Moutouh1">[11]</a>. However, until now the contribution of recombination to HIV-1 evolution has been predominantly shown in an indirect manner, through sequence analyses <a href="#pone.0000271-Leitner1">[17]</a>–<a href="#pone.0000271-Sabino1">[19]</a> and <em>in silico</em> studies <a href="#pone.0000271-Rouzine1">[20]</a>–<a href="#pone.0000271-Althaus1">[22]</a>. It is not indisputably clear whether recombination always leads to a higher rate of adaptation. The Fisher-Muller model predicts that in asexuals (non-recombining individuals) two beneficial mutations have to be fixed sequentially, whereas recombination can combine beneficial mutations that have evolved in parallel. Furthermore, since asexual organisms have genetically linked loci, in theory they are more prone to the accumulation of deleterious mutations. On the other hand recombination may be disadvantageous since the net effect may result in breaking up favorable combinations of mutations more often than combining beneficial mutations <a href="#pone.0000271-Fisher1">[23]</a>–<a href="#pone.0000271-Rice1">[27]</a>. In addition, recombination may not be an essential process, since if the mutation rate is sufficiently high and the population size large, genomes carrying multiple beneficial mutations should appear even in asexual populations <a href="#pone.0000271-MaynardSmith2">[28]</a>, <a href="#pone.0000271-Eshel1">[29]</a>.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>In this study we set out to determine whether the host-cell environment can affect the adaptability of HIV-1 <em>in vitro</em>. Two HIV-1 strains were each cultured in two different T-cell lines (MT4 and C8166) for 110 days (with six replicas for each treatment; making a total of 24 serial passage lines). Relative viral fitness was determined at regular time intervals by letting the evolved viral strains compete against a reference strain. All viruses cultured in the MT4 T-cell line increased rapidly in fitness while viruses cultured in the C8166 T-cell line did not show any increase in fitness. Through sequence analyses and infections with HIV-1 strains expressing fluorescent protein we were able to relate the lack of adaptation in the C8166 T-cell line to a decreased rate of recombination of HIV-1 in this T cell-line.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3>
<h4>Serial passage of two HIV-1 strains on MT4 and C8166 T-cells</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>To assess to what degree the host-cell environment influences the adaptability of HIV-1 we performed a large-scale selection experiment with two distinct HIV-1 strains in two T-cell lines (MT4 and C8166). The ‘wild type’ (wt) virus is the widely used molecular clone pLAI <a href="#pone.0000271-Peden1">[30]</a>, whereas, the mutant (mt) virus lacks one NF-κB DNA-binding site in its Long Terminal Repeat (LTR) transcriptional promoter, and is therefore expected to have a reduced initial fitness <a href="#pone.0000271-Ross1">[31]</a>. MT4 and C8166 are two genetically distinct T-cell lines, which have been immortalized by the human T-cell-leukemia-lymphoma virus type-I <a href="#pone.0000271-Smith1">[32]</a>. We have shown previously that mutations in the transcriptional promoter have different fitness effects when tested in the MT4 and C8116 cell line <a href="#pone.0000271-VanOpijnen3">[33]</a>. This suggests that differences in cellular environment may affect the evolutionary outcome of a replicating virus. Initial relative fitness and replication efficiency were determined for both viruses in the MT4 and C8166 T-cell lines (<a href="#pone-0000271-t001">Table 1</a> and Supplemental <a href="#pone.0000271.s001">Figure S1</a>), which indicates that viral population size and expansion were similar between cellular environments. As expected initial wt viral fitness was significantly higher than mt viral fitness in both cell lines (<em>two-tailed t-test</em>, p&lt;0.0001). Wt and mt viruses were used to infect MT4 and C8166 T-cells to establish 24 serial passage lines (six replicates for each virus and cell line combination). Over a period of 110 days, virus was harvested from each culture at peak infection (typically after 4 to 6 days) and subsequently used to infect a ‘fresh’ batch of cells. Each time after serial passage the same initial viral density and number of host cells were used. Host cell populations typically expanded from 10<sup>6</sup> to 10<sup>7</sup> cells over 3–4 days, and no significant differences in growth between cell lines (non-infected and infected) were found.</p>
<div class="figure" id="pone-0000271-t001"><div class="img"><a name="pone-0000271-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000271.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000271" data-uri="info:doi/10.1371/journal.pone.0000271.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000271.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000271.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000271.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000271.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000271.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000271.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000271.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000271.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000271.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>Initial relative fitness<span class="xref"><sup>a</sup></span> (±s.e.m.) of wt and mt virus</span></strong></p><span>doi:10.1371/journal.pone.0000271.t001</span></div><a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2"></a><p>Since the wt and mt virus have not been specifically adapted to either cell line we expected to observe a fitness increase for all serial passage lines <a href="#pone.0000271-Ebert1">[34]</a>. Indeed, the six wt replicates cultured in the MT4 T-cell line (wt-MT4), increased rapidly in fitness in a seemingly linear fashion and reached a significantly higher fitness level of 1.54±0.14 over the course of the experiment (<a href="#pone-0000271-g001">Figure 1A</a>; <a href="#pone-0000271-t002">Table 2</a>, <em>ANOVA</em>, F<sub>1,9</sub> = 11.276-16.350, p&lt;0.0001). The six mt-MT4 serial passage lines initially also displayed a rapid fitness increase and they too reached a significant higher final fitness around 1.04±0.02 (<a href="#pone-0000271-g001">Figure 1B</a>, <a href="#pone-0000271-t002">Table 2</a>, <em>ANOVA</em>, F<sub>1,9</sub> = 10.713-16.904, p&lt;0.0001). However, fitness increase in the mt lines decelerated over the course of the selection experiment and seemed to reach a plateau, which was confirmed by the superior fit of an exponential model with an upper limit over that of a linear model (<a href="#pone-0000271-g001">Figure 1B</a>, <em>F-test,</em> p&lt;0.05 for all replicates). The final fitness values of the mt replicates are well below that of the wt replicates (<em>ANOVA</em>, p&lt;0.001 with Bonferroni correction). This indicates that the functional defect of the mt strain caused the virus to ‘lock’ on a lower fitness peak in the adaptive landscape <a href="#pone.0000271-Eigen1">[35]</a>, <a href="#pone.0000271-Kauffman1">[36]</a>, at least for the duration of the experiment.</p>
<div class="figure" id="pone-0000271-t002"><div class="img"><a name="pone-0000271-t002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000271.t002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000271" data-uri="info:doi/10.1371/journal.pone.0000271.t002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000271.t002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000271.t002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000271.t002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000271.t002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000271.t002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000271.t002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000271.t002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000271.t002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000271.t002.TIF"></span>)
                </a></li></ul></div><p><strong>Table 2.  <span>Relative fitness<span class="xref"><sup>a</sup></span> (±s.e.m.) of wt and mt virus after 110 days of culture in the MT4 T-cell line</span></strong></p><span>doi:10.1371/journal.pone.0000271.t002</span></div><div class="figure" id="pone-0000271-g001"><div class="img"><a name="pone-0000271-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000271.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000271" data-uri="info:doi/10.1371/journal.pone.0000271.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000271.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000271.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000271.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000271.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000271.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000271.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000271.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000271.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000271.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Change in relative fitness of viruses cultured on two distinct T-cell lines.</span></strong></p><a id="article1.body1.sec2.sec1.fig1.caption1.p1" name="article1.body1.sec2.sec1.fig1.caption1.p1"></a><p>A) The 6 wildtype replicates that are cultured on MT4 T-cells show a linear increase in fitness. The inset shows a single representative example of replicate wt-1. B) The 6 mutant MT4 cultured replicates show an initial fitness increase that seems to reach a maximum at around a relative fitness of 1.04±0.02. All trajectories are best described by an exponential regression model with an upper limit (F-test, p&lt;0.05). The inset shows a single representative example of replicate mt-1 (see <a href="#pone-0000271-t002">Table 2</a> for relative fitness values at day 110). C and D) relative fitness of 6 wt and 6 mt C8166 cultured viruses, respectively. Relative fitness did not significantly deviate from the starting virus over the course of the experiment (ANOVA, p&gt;&gt;0.05; see <a href="#pone-0000271-t003">Table 3</a> for relative fitness values at day 110). Fitness is relative to reference strain v13 (also see <a href="#s4">materials and methods</a> section.</p>
<span>doi:10.1371/journal.pone.0000271.g001</span></div><div class="figure" id="pone-0000271-t003"><div class="img"><a name="pone-0000271-t003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000271.t003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000271" data-uri="info:doi/10.1371/journal.pone.0000271.t003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000271.t003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000271.t003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000271.t003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000271.t003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000271.t003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000271.t003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000271.t003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000271.t003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000271.t003.TIF"></span>)
                </a></li></ul></div><p><strong>Table 3.  <span>Relative fitness<span class="xref"><sup>a</sup></span> (±s.e.m.) of wt and mt virus after 110 days of culture in the C8166 T-cell line</span></strong></p><span>doi:10.1371/journal.pone.0000271.t003</span></div><a id="article1.body1.sec2.sec1.p3" name="article1.body1.sec2.sec1.p3"></a><p>Surprisingly, none of the twelve serial passage lines that were cultured on C8166 T-cells significantly changed their fitness (<a href="#pone-0000271-g001">Figure 1C and, D</a>, <a href="#pone-0000271-t003">Table 3</a>, <em>ANOVA</em>, F<sub>1,9</sub> = 0.062-2.164, p&gt;0.05). There are several potential explanations for this apparent lack of adaptation. First, the basic mutation rate might be low in the C8166 T-cell line as compared to the MT4 T-cell line. This, however, is unlikely since the mutation rate depends mainly on the viral reverse transcriptase enzyme, which was identical in all situations. Besides, we confirmed by sequence analysis that the basic mutation rate was similar in both cell lines (see next section). Second, there may have been fewer adaptive mutations available in the C8166 T-cell line, for instance, because the initial viral strains were already better adapted to the C8166 T-cell line than to the MT4 cell line. We were able to dismiss this possibility as it turned out that the viruses that were cultured on the MT4 line also had a significantly higher fitness on the C8166 cell-line (<a href="#pone-0000271-t002">Table 2</a>, <em>ANOVA</em>, F<sub>1,3</sub> = 6.367-17.874, p&lt;0.01), which illustrates that in principle adaptation to the C8166 cell line was possible. As a cross-check, we also determined fitness of the C8166 serial passage lines on the MT4 T-cell line, which demonstrates that these viruses had not changed their fitness on MT4 cells (<a href="#pone-0000271-t003">Table 3</a>, <em>ANOVA</em>, F<sub>1,3</sub> = 0.117-2.842, p&gt;0.05). These data illustrate that both wt and mt virus had a large adaptive potential. In spite of this potential, none of the C8166 cultured viruses had increased their fitness, neither on C8166 cells nor on MT4 cells.</p>


<h4>High polymorphism and linkage between genetic regions in C8166 cultured viruses</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>To determine whether there was a genetic basis for differences in adaptation we sequenced two domains of the HIV-1 genome; the LTR and viral Envelope (<em>Env</em>). Each of the 24 serial passage lines were analyzed by sequencing ten clones for each region and this was repeated for four time points, whereas for a fifth time point (at 61 days) only <em>Env</em> was sequenced. On average, nucleotide diversity in both cell lines was similar, which underpins our previous assumption that the basic mutation rate in both cell lines was similar (<a href="#pone-0000271-t004">Table 4</a>). There was no significant difference in linkage disequilibrium between MT4 and C8166 cultures, possibly due to the overall relatively low genetic variation. Sequence analysis of the LTR region showed that inactivation of the NF-κB site in the mt virus was stably present up to the end of the experiment in all serial passage lines. This confirms that the LTR ‘handicap’ remained present during the time frame of the experiment and this gives a likely explanation why the mt-MT4 replicates were ‘locked’ on a lower fitness peak compared to wt-MT4.</p>
<div class="figure" id="pone-0000271-t004"><div class="img"><a name="pone-0000271-t004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000271.t004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000271" data-uri="info:doi/10.1371/journal.pone.0000271.t004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000271.t004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000271.t004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000271.t004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000271.t004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000271.t004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000271.t004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000271.t004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000271.t004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000271.t004.TIF"></span>)
                </a></li></ul></div><p><strong>Table 4.  <span>Nucleotide diversity (π±s.e.m.)</span></strong></p><span>doi:10.1371/journal.pone.0000271.t004</span></div><a id="article1.body1.sec2.sec2.p2" name="article1.body1.sec2.sec2.p2"></a><p>There was no obvious bias in the observed mutations; all cultures had a similar number of transitions and transversions and also synonymous and non-synonymous changes in <em>Env</em> were not significantly different between cultures (<a href="#pone-0000271-t005">Table 5</a>). After 110 days of culture four MT4 cultures contained a single point mutation that had gone to fixation. In three cases this mutation was located in the LTR region; two of these mutations were transitions (G 224 A and G 368 A) and the third was a transversion (G 280 C). The fourth fixed mutation, a non-synonymous substitution at the second codon position, caused a threonine-to-lysine change in <em>Env</em> at position+7133. This lack of mutations that have achieved fixation might be partly due to coexistence of beneficial mutants. Also adaptive mutations in parts of the genome that we did not sequence might have contributed to the observed increase in fitness.</p>
<div class="figure" id="pone-0000271-t005"><div class="img"><a name="pone-0000271-t005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000271.t005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000271" data-uri="info:doi/10.1371/journal.pone.0000271.t005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000271.t005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000271.t005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000271.t005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000271.t005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000271.t005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000271.t005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000271.t005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000271.t005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000271.t005.TIF"></span>)
                </a></li></ul></div><p><strong>Table 5.  <span>Transition/transversion ratio (R) in LTR and <em>Env</em> and non-synonymous (dN)/synonymous (dS) ratio in <em>Env</em> (±s.e.m.).</span></strong></p><span>doi:10.1371/journal.pone.0000271.t005</span></div><a id="article1.body1.sec2.sec2.p3" name="article1.body1.sec2.sec2.p3"></a><p>From the sequence analyses two distinct patterns emerge. First, plotting genetic variation for both regions for viruses cultured on the MT4 cell line reveals there is no correlation in genetic variation between the LTR and <em>Env</em> region (n = 44, r<sup>2</sup> = 0.011, p = 0.49; <a href="#pone-0000271-g002">Figure 2a</a>), whereas there is a strong linear correlation in genetic variation between these two regions in C8166 cultured viruses (n = 39, r<sup>2</sup> = 0.745, p&lt;0.0001; <a href="#pone-0000271-g002">Figure 2b</a>). Even if we exclude the two outliers from <a href="#pone-0000271-g002">Figure 2b</a> the strong relationship remains (n = 37, r<sup>2</sup> = 0.475, p&lt;0.0001). In other words, MT4 cultured viruses show no linkage in genetic variation between the LTR and <em>Env</em> region, which is in accordance with strong selection. In contrast, C8166 cultured viruses show a strong correlation between genetic variation in the LTR and <em>Env</em> region, which is indicative of the variation being (nearly) neutral.</p>
<div class="figure" id="pone-0000271-g002"><div class="img"><a name="pone-0000271-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000271.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000271" data-uri="info:doi/10.1371/journal.pone.0000271.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000271.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000271.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000271.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000271.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000271.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000271.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000271.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000271.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000271.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Genetic variation in the Long Terminal Repeat (LTR) and envelope region (<em>Env</em>) of MT4 and C8166 cultured viruses.</span></strong></p><a id="article1.body1.sec2.sec2.fig1.caption1.p1" name="article1.body1.sec2.sec2.fig1.caption1.p1"></a><p>For each culture, ten clones of both the LTR and <em>Env</em> region were sequenced and this was repeated for five points in time. Thus, each data point corresponds to the amount of genetic variation within a single serial passage line at a certain point in time. Genetic variation (Kimura 2 parameter) is plotted separately for the two cell-lines. A) No correlation in genetic variation in MT4 evolved viruses was observed (n = 44, r<sup>2</sup> = 0.011, p = 0.49). B) There is a strong linear correlation in genetic variation between the LTR and <em>Env</em> in C8166 cultured viruses (n = 39, r<sup>2</sup> = 0.745, p&lt;0.0001). Even if the two outliers are taken out, the strong correlation remains (n = 37, r<sup>2</sup> = 0.475, p&lt;0.0001).</p>
<span>doi:10.1371/journal.pone.0000271.g002</span></div><a id="article1.body1.sec2.sec2.p4" name="article1.body1.sec2.sec2.p4"></a><p>A second pattern emerges from the level of viral polymorphism in each culture. We observed that the polymorphism level for the wt-C8166 viruses was twice as high compared to wt-MT4 viruses (0.372±0.044 wt-C8166 vs 0.179±0.026 wt-MT4; <em>two tailed t-test</em> p&lt;0.0001; omitting singletons from the analysis). The same pattern was present for mt viruses; the polymorphism level being twice as high in mt-C8166 viruses compared to mt-MT4 viruses (0.247±0.032 mt-C8166 vs 0.130±0.018 mt-MT4; <em>two tailed t-test</em>, p&lt;0.001). In combination with the similar levels of nucleotide diversity between environments this implies that singletons were at least two times more common in the MT4 cultures (data not shown). These results suggest that: i) selective sweeps in the MT4 cultures have triggered a higher number of singletons in this environment <a href="#pone.0000271-Fu1">[37]</a>, ii) the high polymorphism level in the C8166 cultures could be indicative for ‘clonal interference’ (see below).</p>


<h4>Lack of recombination in the C8166 T-cell line</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>We were interested in how different cellular environments may drive differences in viral adaptation. We had minimized the chance that differences in adaptation would be triggered by differences in: i) population size; both viral population size, cell population size and expansion size were similar between cultures, ii) bottlenecks; the same amount of cells were infected with the same amount of virus, and iii) mutation rate; the mutation rate of HIV-1 is (mainly) driven by the erroneous nature of the reverse transcriptase (RT) enzyme, which is produced by the virus itself. The RT gene was identical in all viruses, and consequently the basic mutation rate should be similar across viruses. We show in <a href="#pone-0000271-t004">Table 4</a> that, indeed, the accumulated nucleotide diversity, which is the average number of nucleotide differences per site, was similar between the two cell types.</p>
<a id="article1.body1.sec2.sec3.p2" name="article1.body1.sec2.sec3.p2"></a><p>Nevertheless, we found a strong effect of cellular environment on adaptation, with strong fitness increase in MT4 and no fitness increase in C8166. We think this difference in adaptive potential is possibly caused by differences in recombination rate between the two cell lines. Based on the genetic analyses, the large number of singletons in MT4 cultures point to strong selection whereas the higher polymorphism level in the ‘non-adapted’ C8166 replicates might be indicative for ‘clonal interference’ between clones with marginal fitness effects competing to achieve fixation <a href="#pone.0000271-Miralles1">[38]</a>–<a href="#pone.0000271-Kim1">[44]</a>. Clonal interference can put a ‘speed limit’ on the rate of evolution. Recombination can alleviate this restriction by combining beneficial mutations into a single genome and thus circumvent competition among strains <a href="#pone.0000271-deVisser1">[45]</a>–<a href="#pone.0000271-Hey1">[47]</a>. A second important process that can slow down adaptation or even result in a decrease in fitness is Muller's ratchet, which can be triggered by repeated bottlenecks <a href="#pone.0000271-Butcher1">[48]</a>–<a href="#pone.0000271-Novella1">[51]</a>. Although bottleneck size was the same for both environments, differences in recombination rate between environments may have caused the C8166 cultured viruses to be more affected by Muller's ratchet and/or by clonal interference between marginally beneficial mutations.</p>
<a id="article1.body1.sec2.sec3.p3" name="article1.body1.sec2.sec3.p3"></a><p>To directly test for the ‘lack of recombination’ hypothesis we assessed differences in viral recombination between MT4 and C8166 T-cells. Here we used modified HIV-1 strains encoding either the gene expressing yellow or cyan fluorescent protein (YFP/CFP) <a href="#pone.0000271-Levy1">[14]</a>. If recombination occurs between specific regions of the YFP and CFP genes these HIV-1 recombinants will express modified Green Fluorescent Protein (GFP*) which can be observed by FACS analysis <a href="#pone.0000271-Levy1">[14]</a>. MT4 and C8166 cells were infected with equal amounts of HIV-YFP and HIV-CFP (2 replicates each). Every four days (well before peak of infection) virus was harvested and a new batch of cells was infected. FACS analysis was performed after two and three rounds of serial passage (after 8 and 12 days, respectively). After eight days, over ten times more MT4 cells than C8166 cells displayed recombinant GFP* (<a href="#pone-0000271-t006">Table 6</a>). After twelve days the differences between cell lines had increased even further, and now almost 50 times more MT4 cells expressed GFP*. The differences in recombination correspond to differences in double infected cells (expressing both YFP and CFP). We observed 36x and 124x more double infected cells in the MT4 cultures compared to the C8166 cultures, after eight and twelve days, respectively. These data confirm that effective recombination occurs less frequently in C8166 cells than in MT4 cells, which is the result of a lower number of double infections in C8166 cells.</p>
<div class="figure" id="pone-0000271-t006"><div class="img"><a name="pone-0000271-t006" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000271.t006&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000271" data-uri="info:doi/10.1371/journal.pone.0000271.t006"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000271.t006&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000271.t006/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000271.t006/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000271.t006/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000271.t006/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000271.t006.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000271.t006/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000271.t006/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000271.t006.TIF"></span>)
                </a></li></ul></div><p><strong>Table 6.  <span>Recombination in MT4 and C8166 cells</span></strong></p><span>doi:10.1371/journal.pone.0000271.t006</span></div>
</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3>
<h4>Viral genotype can affect the evolutionary outcome</h4>
<a id="article1.body1.sec3.sec1.p1" name="article1.body1.sec3.sec1.p1"></a><p>In this paper we set out to determine whether differences in the host-cell environment can drive divergent evolution of HIV-1. We tested two different virus strains, one ‘wildtype’ and a deliberately handicapped mutant, across two distinct T-cell environments. After 110 days of culture we obtained some very interesting results. First, we showed that viruses when cultured on MT4 cells evolved to a higher fitness level. Interestingly, the wild type virus showed a continual linear increase in fitness, whereas the adaptive rate of the mutant virus slowed down and a fitness plateau was reached. As a consequence, the mutant virus obtained a lower fitness level than the wildtype virus. This suggests that viruses with a small initial genetic difference can consistently evolve towards different local fitness peaks in the viral adaptive landscape <a href="#pone.0000271-Eigen1">[35]</a>, <a href="#pone.0000271-Kauffman1">[36]</a>, <a href="#pone.0000271-Wright1">[52]</a>. It also underlines that the viral genotype significantly contributes to the adaptive potential. This is especially interesting in the clinical context of viruses that carry drug resistance mutations, for which the virus pays a fitness cost <a href="#pone.0000271-Back1">[53]</a>. The virus may balance this cost by accumulating compensatory mutations <a href="#pone.0000271-Boucher1">[54]</a>–<a href="#pone.0000271-Berkhout1">[61]</a>. Our results suggest that the adaptive potential of the resistant genotype may be diminished from that of the original wildtype. Therefore, it might be advantageous to continue treatment of an HIV-1 infected patient, even when resistance develops <a href="#pone.0000271-Berkhout1">[61]</a>–<a href="#pone.0000271-Clementi1">[65]</a>. However, this approach needs careful consideration since cross resistance against other drugs may evolve <a href="#pone.0000271-Tisdale1">[66]</a> and there is at least one example of a drug resistant HIV-1 strain that evolved to a higher fitness than that of the initial wildtype strain <a href="#pone.0000271-Nijhuis1">[67]</a>.</p>


<h4>The host-cell environment affects the evolutionary potential of HIV-1</h4>
<a id="article1.body1.sec3.sec2.p1" name="article1.body1.sec3.sec2.p1"></a><p>The most important finding in our study is that none of the twelve serial passage lines that were cultured on C8166 cells resulted in an increase of viral fitness. The subsequent sequence analyses and infections with fluorescent protein expressing viruses demonstrated a lower effective recombination rate in C8166 cells, as compared to MT4 cells. This reduced recombination rate is a likely candidate to have caused the evolutionary stagnation. Recombining populations may have several advantages over non-recombining populations in adapting to a new situation <a href="#pone.0000271-Otto1">[25]</a>–<a href="#pone.0000271-Rice1">[27]</a>. Recombination can avoid clonal interference, that is, competition between mutations of almost equal fitness increase. Furthermore, non-recombining populations may experience Muller's ratchet, which is the stochastic loss of the most fit genotype in the population due to the gradual accumulation of deleterious mutations <a href="#pone.0000271-Butcher1">[48]</a>–<a href="#pone.0000271-Gessler1">[50]</a>. Muller's ratchet is highly sensitive to drift and may for instance be triggered by repeated bottlenecks at serial passage. Moreover, recombination can reduce the mutational load, which is a deterministic process where deleterious mutations accumulate in the genome due to insufficient productivity of the fittest class <a href="#pone.0000271-Rice1">[27]</a>, <a href="#pone.0000271-Kondrashov1">[68]</a>, <a href="#pone.0000271-Rice2">[69]</a> and background selection, where the probability of fixation of a beneficial mutation is lowered if it arises in a genetic background with low fitness <a href="#pone.0000271-Barton1">[41]</a>, <a href="#pone.0000271-Charlesworth1">[70]</a>–<a href="#pone.0000271-Peck1">[72]</a>. All these mechanisms may have contributed to our results, although it is impossible at this stage to pinpoint which effects have been most important.</p>
<a id="article1.body1.sec3.sec2.p2" name="article1.body1.sec3.sec2.p2"></a><p>It is not clear to what extent recombination contributes to the adaptability of HIV-1 <em>in vivo</em>. An <em>in silico</em> study by Althaus and Bonhoeffer <a href="#pone.0000271-Althaus1">[22]</a> indicates that recombination can speed up the adaptive rate of HIV-1 if the effective population size is sufficiently large. Also, <em>in vitro</em> experiments have shown that recombination can be beneficial and can <em>e.g.</em> lead to the rapid emergence of resistance against multiple drugs <a href="#pone.0000271-Kellam1">[10]</a>, <a href="#pone.0000271-Moutouh1">[11]</a>, <a href="#pone.0000271-Gu1">[73]</a>. However, the conditions in which these experiments were performed, infecting cells with equal mixtures of two partially resistant strains, strongly favour the emergence of recombinants, as the beneficial mutations are already present at high frequency. In this report we show that the host-cell environment can greatly influence the adaptive potential of HIV-1. We conclude that differences in co-infection rate and consequently the effective recombination rate provide a mechanistic explanation for these differences in adaptability. Interestingly, two recent publications demonstrate that cellular factors are involved in mediating viral recombination <a href="#pone.0000271-Levy1">[14]</a>, <a href="#pone.0000271-Serviene1">[74]</a>. In our experiments differences in recombination rate seem to be largely due to differences in the level of co-infection of host cells, which could be due to cell-type specific expression levels of the viral (co)-receptor or other cellular factors. These combined results raise the exciting possibility to combat retroviral infections by modulating cellular factors that reduce viral recombination and consequently the virus' adaptive potential.</p>

</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4>Virus constructs</h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p>Two viruses were used in the selection experiment; the molecular clone pLAI (wt) <a href="#pone.0000271-Peden1">[30]</a> and pLAI with an inactivated NF-κB-II site in its Long Terminal Repeat (mt). The NF-κB-II DNA-binding site, which is positioned from-105 to-96 relative to the transcription start site, was inactivated by exchanging the sequence 5′-GGGACTTTCC-3′ with 5′-CCCCCCCCCC-3′ through a PCR strategy with a mutagenic primer on the pBlue3′LTR and pBlue5′LTR intermediate plasmids as described previously <a href="#pone.0000271-Jeeninga1">[75]</a>. The mt molecular clone was obtained by insertion of the <em>XhoI-HindIII</em> fragment of pBlue3′LTR into the 3′LTR of the HIV-1 molecular clone pLAI and the <em>ClaI</em>-<em>XbaI</em> fragment of pBlue5′LTR into the 5′LTR of the HIV-1 molecular clone pLAI.</p>


<h4>Cell lines</h4>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1"></a><p>The human lymphocytic MT4 and C8166 T-cell lines were cultured in RPMI 1640 (Gibco BRL) supplemented with 10%fetal calf serum, penicillin (100 units/ml) and streptomycin (100 units/ml). MT4 and C8166 are T-cell lines that have been immortalized through transformation by the human T-cell leukemia-lymphoma virus type I <a href="#pone.0000271-Smith1">[32]</a>. Although exact differences between the T-cell lines have not been assessed, several differences are known, for instance in the pool of available transcription factors such as ETS1, C/EBP, GATA, NF-kB, STAT5 and SP1 <a href="#pone.0000271-Chen1">[76]</a>–<a href="#pone.0000271-Matsuo1">[79]</a>. The cervix carcinoma cell line C33A (ATCC HTB31) was cultured in Dulbecco's Modified Eagle's Medium (Gibco BRL) with the same supplements. All cell lines were kept at 37°C and 5%CO<sub>2</sub>.</p>


<h4>Virus stocks and replication</h4>
<a id="article1.body1.sec4.sec3.p1" name="article1.body1.sec4.sec3.p1"></a><p>C33A cells were calcium phosphate transfected with 5 µg DNA of the respective molecular clone to produce virus stocks. The virus concentration was determined by the CA-p24 Enzyme-Linked ImmunoSorbent Assay (ELISA) <a href="#pone.0000271-Jeeninga1">[75]</a>. Replication assays were performed by infecting 10<sup>6</sup> MT4 or C8166 T-cells with 1 ng CA-p24 virus stock, and virus replication was followed by measuring CA-p24 production.</p>


<h4>Selection experiment</h4>
<a id="article1.body1.sec4.sec4.p1" name="article1.body1.sec4.sec4.p1"></a><p>Twenty four cultures were founded by infecting 10<sup>6</sup> MT4 or C8166 T-cells and 1 ng CA-p24 of virus (wt or mt; 6 replicates each). Infections were monitored for viral replication by measuring CA-p24 production in the culture supernatant and by monitoring syncytia formation by microscopic inspection, with peak of infection typically reached between 4–6 days of culture. Subsequently, virus was isolated and a fresh batch of cells was infected (MOI = 0.001). The cultures were maintained up to 110 days and approximately every three days virus and cellular DNA was isolated and frozen at −80°C.</p>


<h4>Competition experiments</h4>
<a id="article1.body1.sec4.sec5.p1" name="article1.body1.sec4.sec5.p1"></a><p>Competition experiments were performed against “virus 13” (v13) to determine relative fitness of wt and mt at the start of the culture and at specific time points during the selection experiment. v13 contains a single mutation (C to G change at position-68 relative to the transcription start site) in the Sp1 DNA binding site in the Long Terminal Repeat. The mutation served as a marker that can be tracked by sequencing, and makes the v13 virus slightly less fit compared to wt in MT4 cells, but is indistinguishable in fitness from wt in C8166 cells <a href="#pone.0000271-VanOpijnen3">[33]</a>. For each competition experiment, a total of 10<sup>6</sup> cells were infected with v13 and a sample of the cultured virus of interest (0.5 ng CA-p24 each). Competition experiments were performed as previously described <a href="#pone.0000271-VanOpijnen2">[12]</a>, <a href="#pone.0000271-VanOpijnen3">[33]</a> at an initial multiplicity of infection (MOI) of approximately 0.001. The relatively small number of viral generations within the competition experiments, compared to the large initial number of infected cells, ensures that <em>de novo</em> mutations during the competition experiment will not affect our fitness estimates. Even if such mutations are advantageous, they will not reach substantial copy numbers over the duration of the experiment. Direct evidence for insensitivity of our method for <em>de novo</em> mutations comes from the high reproducibility of fitness estimates between replicates and the fact that we never observed any new mutations in our sequencing <a href="#pone.0000271-VanOpijnen3">[33]</a>. Also recombination is unlikely to affect our fitness estimates because, as explained in the main text, effective recombination for HIV depends on multiple infection of cells, and as indicated the MOI in our experiments is low. Furthermore, we have never observed recombinant genotypes in our sequencing data <a href="#pone.0000271-VanOpijnen3">[33]</a>. Competitions were repeated two to three times and each competition was continued for three passages by isolating virus at peak infection and subsequently infecting a fresh batch of host cells. Total cell DNA was isolated before passage from approximately 0.25x10<sup>6</sup> cells as described previously <a href="#pone.0000271-VanOpijnen2">[12]</a>. Proviral LTR sequences were PCR-amplified and sequenced with the-21M13 Big Dye Primer cycle sequencing kit (ABI) to determine the ratio of both competitors <a href="#pone.0000271-VanOpijnen2">[12]</a>, <a href="#pone.0000271-VanOpijnen3">[33]</a>, <a href="#pone.0000271-Larder2">[80]</a> and subsequently calculate relative fitness.</p>


<h4>Sequencing and genetic analyses</h4>
<a id="article1.body1.sec4.sec6.p1" name="article1.body1.sec4.sec6.p1"></a><p>At specific time points (days 18, 43, 61, 77 and 110) we PCR-amplified the LTR region from position+37 to+668 and the envelope region from+7010 to+7732 (Supplemental <a href="#pone.0000271.s002">Figure S2A and S2B</a>). PCR fragments were subsequently cloned into a pCRII-TOPO vector (Invitrogen) and ten clones were sequenced (forward and reverse) from each region and serial passage line and for each time-point (at time point 61 only <em>Env</em> was sequenced) with the Big Dye Terminator cycle sequencing kit (ABI). In addition, we re-sequenced two time points for eight cultures to determine whether our sequencing method was accurately displaying the viral variation in each culture. Indeed, nucleotide diversity was similar between original and re-sequenced cultures, although some specific observed mutations differed. Nucleotide diversity and the transition/transversion ratio (R) were calculated with MEGAv3.1 <a href="#pone.0000271-Kumar1">[81]</a> and the Kimura 2-parameter model was used to calculate genetic variation in each genetic region for each serial passage line (only time points were included when at least eight clones from each region were successfully sequenced). dN/dS ratio was calculated with DnaSP <a href="#pone.0000271-RozasJFauSanchezDelBarrio1">[82]</a> The degree of polymorphism in a replicate was determined by Σ<em>p<sub>i</sub></em>(1−<em>p<sub>i</sub></em>), where <em>p<sub>i</sub></em> is the frequency of the mutation at a specific site (singletons were excluded).</p>


<h4>Fitness calculation and statistics</h4>
<a id="article1.body1.sec4.sec7.p1" name="article1.body1.sec4.sec7.p1"></a><p>For each competition experiment we compute the relative fitness <em>W<sub>m</sub></em> of the mutant, by comparing the fold expansion of the mutant and wild type <a href="#pone.0000271-VanOpijnen3">[33]</a>, <a href="#pone.0000271-Lenski1">[83]</a>.<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000271.e001&amp;representation=PNG" class="inline-graphic"><span class="note">1</span></span><br>In which, <em>N<sub>m</sub>(0)</em>and <em>N<sub>w</sub>(0)</em> are the initial densities of the mutant and wild type genotype, respectively, and <em>N<sub>m</sub>(T)</em>and <em>N<sub>w</sub>(T)</em> are the corresponding densities at the end of the experiment and <em>d</em> is the dilution factor. The relative fitness <em>W<sub>m</sub></em> of a mutant is a dimensionless factor that scales the growth rate of the mutant compared to the wild type. For simplicity, we assume that mutations affect the Malthusian (<em>i.e.</em> exponential) net replication rate, as we cannot distinguish between effects on replication rate or death rate <a href="#pone.0000271-Maree1">[84]</a>. Actually, our selection regime with expanding host cell populations and serial passage transfer will tend to select for viral clones that are able to expand fast. Statistics were performed with the programs Prism 3.0 and SPSS 12.0.1.</p>


<h4>HIV-1 recombination assay</h4>
<a id="article1.body1.sec4.sec8.p1" name="article1.body1.sec4.sec8.p1"></a><p>The genes encoding cyan or yellow fluorescent protein (CFP/YFP) are derived from pECFP-N1 and pEYFP-N1 (Clontech) and were inserted into the HIV-1 molecular clone pLAI downstream from the Env-gene and upstream of the Nef-gene. By recombination between the ECFP and EYFP genes a functional green fluorescent protein (GFP*) expressing gene is produced <a href="#pone.0000271-Levy1">[14]</a>. The detection of cells expressing fluorescent proteins was performed on a LSR-II FACS (Becton Dickinson) with a 488 nm laser equipped with standard filters or with adapted bandpass filters. 10<sup>6</sup> MT4 or C8166 T-cells were infected with 10 ng CA-p24 virus stock, and virus replication was followed by measuring CA-p24 production. Due to insertion in the HIV-1 genome of the gene coding for fluorescent protein, the viruses are somewhat attenuated when compared to wt pLAI, which explains why a higher viral input was necessary in these experiments. Every four days (approximately four days before the peak of infection was reached) virus was harvested and a new batch of cells were infected. After eight and twelve days of replication 0.25×10<sup>6</sup> to 0.5×10<sup>6</sup> T-cells were analyzed by FACS (the experiments were performed in duplo).</p>

</div>

<div id="section5" class="section"><a id="s5" name="s5" toc="s5" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0000271"></div><a name="pone.0000271.s001" id="pone.0000271.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000271.s001">Figure S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1"></a><p class="preSiDOI">Replication efficiency of wt and mt viruses in MT4 and C8166 T-cells. Virus production over time was followed by measuring CA-p24 in the culture supernatant. The wt virus has a clear advantage over the mt virus in both host-cell environments.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000271.s001</p><a id="article1.body1.sec5.supplementary-material1.caption1.p2" name="article1.body1.sec5.supplementary-material1.caption1.p2"></a><p class="postSiDOI">(0.78 MB EPS)</p>
<a name="pone.0000271.s002" id="pone.0000271.s002"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000271.s002">Figure S2. </a></strong></p><a id="article1.body1.sec5.supplementary-material2.caption1.p1" name="article1.body1.sec5.supplementary-material2.caption1.p1"></a><p class="preSiDOI">A. LTR sequence wt from+37 to+668. Positions are according to HXB2 numbering. B. <em>Env</em> sequence from+7010 to+7732. Positions are according to HXB2 numbering.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000271.s002</p><a id="article1.body1.sec5.supplementary-material2.caption1.p2" name="article1.body1.sec5.supplementary-material2.caption1.p2"></a><p class="postSiDOI">(0.03 MB DOC)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>We acknowledge Sara Magalhaes, Pleuni Pennings, Maus Sabelis, Nadine Vastenhouw and Arjan de Visser for discussions and critical reading of the manuscript. Furthermore we thank Emiel van Loon for advice on statistics.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: Tv Ad. Performed the experiments: Tv Ad. Analyzed the data: Tv MB. Wrote the paper: BB Tv MB.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0000271-Valdez1" id="pone.0000271-Valdez1"></a>Valdez H, Carlson NL, Post AB, Asaad R, Heeger PS, et al.  (2002) HIV long-term non-progressors maintain brisk CD8 T cell responses to other viral antigens. AIDS  16: 1113–1118.  <ul class="find" data-citedArticleID="935600" data-doi="10.1097/00002030-200205240-00004"><li><a href="http://dx.doi.org/10.1097/00002030-200205240-00004" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=HIV+long-term+non-progressors+maintain+brisk+CD8+T+cell+responses+to+other+viral+antigens." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22HIV+long-term+non-progressors+maintain+brisk+CD8+T+cell+responses+to+other+viral+antigens.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0000271-Guadalupe1" id="pone.0000271-Guadalupe1"></a>Guadalupe M, Reay E, Sankaran S, Prindiville T, Flamm J, et al.  (2003) Severe CD4+T-cell depletion in gut lymphoid tissue during primary human immunodeficiency virus type 1 infection and substantial delay in restoration following highly active antiretroviral therapy. J Virol  77: 11708–11717.  <ul class="find" data-citedArticleID="935504" data-doi="10.1128/jvi.77.21.11708-11717.2003"><li><a href="http://dx.doi.org/10.1128/jvi.77.21.11708-11717.2003" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Severe+CD4%2BT-cell+depletion+in+gut+lymphoid+tissue+during+primary+human+immunodeficiency+virus+type+1+infection+and+substantial+delay+in+restoration+following+highly+active+antiretroviral+therapy." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Severe+CD4%2BT-cell+depletion+in+gut+lymphoid+tissue+during+primary+human+immunodeficiency+virus+type+1+infection+and+substantial+delay+in+restoration+following+highly+active+antiretroviral+therapy.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0000271-Korthals1" id="pone.0000271-Korthals1"></a>Korthals AH, de Boer R, Boerlijst M (2006) Role of avidity and breadth of the CD4 T cell response in progression to AIDS. Proc Biol Sci  273: 1697–1704.  <ul class="find" data-citedArticleID="935526" data-doi="10.1098/rspb.2006.3511"><li><a href="http://dx.doi.org/10.1098/rspb.2006.3511" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Role+of+avidity+and+breadth+of+the+CD4+T+cell+response+in+progression+to+AIDS." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Role+of+avidity+and+breadth+of+the+CD4+T+cell+response+in+progression+to+AIDS.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0000271-VanOpijnen1" id="pone.0000271-VanOpijnen1"></a>Van Opijnen T, Berkhout B (2005) The host environment drives HIV-1 fitness. RevMed Virol  15: 219–233.  <ul class="find" data-citedArticleID="935602" data-doi="10.1002/rmv.472"><li><a href="http://dx.doi.org/10.1002/rmv.472" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+host+environment+drives+HIV-1+fitness." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+host+environment+drives+HIV-1+fitness.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0000271-Perelson1" id="pone.0000271-Perelson1"></a>Perelson AS, Neumann AU, Markowitz M, Leonard JM, Ho DD (1996) HIV-1 dynamics in vivo: virion clearance rate, infected cell life-span, and viral generation time. Science  271: 1582–1586.  <ul class="find" data-citedArticleID="935578" data-doi="10.1126/science.271.5255.1582"><li><a href="http://dx.doi.org/10.1126/science.271.5255.1582" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=HIV-1+dynamics+in+vivo%3A+virion+clearance+rate%2C+infected+cell+life-span%2C+and+viral+generation+time." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22HIV-1+dynamics+in+vivo%3A+virion+clearance+rate%2C+infected+cell+life-span%2C+and+viral+generation+time.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0000271-Rodrigo1" id="pone.0000271-Rodrigo1"></a>Rodrigo AG, Shpaer EG, Delwart EL, Iversen AK, Gallo MV, et al.  (1999) Coalescent estimates of HIV-1 generation time in vivo. Proc Natl Acad Sci U S A  96: 2187–2191.  <ul class="find" data-citedArticleID="935584" data-doi="10.1073/pnas.96.5.2187"><li><a href="http://dx.doi.org/10.1073/pnas.96.5.2187" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Coalescent+estimates+of+HIV-1+generation+time+in+vivo." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Coalescent+estimates+of+HIV-1+generation+time+in+vivo.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0000271-Ho1" id="pone.0000271-Ho1"></a>Ho DD, Neumann AU, Perelson AS, Chen W, Leonard JM, et al.  (1995) Rapid turnover of plasma virions and CD4 lymphocytes in HIV-1 infection. Nature  373: 123–126.  <ul class="find" data-citedArticleID="935512" data-doi="10.1038/373123a0"><li><a href="http://dx.doi.org/10.1038/373123a0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Rapid+turnover+of+plasma+virions+and+CD4+lymphocytes+in+HIV-1+infection." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Rapid+turnover+of+plasma+virions+and+CD4+lymphocytes+in+HIV-1+infection.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0000271-Wei1" id="pone.0000271-Wei1"></a>Wei X, Ghosh SK, Taylor ME, Johnson VA, Emini EA, et al.  (1995) Viral dynamics in human immunodeficiency virus type 1 infection. Nature  373: 117–126.  <ul class="find" data-citedArticleID="935608" data-doi="10.1038/373117a0"><li><a href="http://dx.doi.org/10.1038/373117a0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Viral+dynamics+in+human+immunodeficiency+virus+type+1+infection." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Viral+dynamics+in+human+immunodeficiency+virus+type+1+infection.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0000271-Mansky1" id="pone.0000271-Mansky1"></a>Mansky LM, Temin HM (1995) Lower in vivo mutation rate of human immunodeficiency virus type 1 than predicted from the fidelity of purified reverse transcriptase. The Journal of Virology  69: 5087–5094.  <ul class="find" data-citedArticleID="935542"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Lower+in+vivo+mutation+rate+of+human+immunodeficiency+virus+type+1+than+predicted+from+the+fidelity+of+purified+reverse+transcriptase.&amp;auth=&amp;atitle=Lower+in+vivo+mutation+rate+of+human+immunodeficiency+virus+type+1+than+predicted+from+the+fidelity+of+purified+reverse+transcriptase." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Lower+in+vivo+mutation+rate+of+human+immunodeficiency+virus+type+1+than+predicted+from+the+fidelity+of+purified+reverse+transcriptase." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Lower+in+vivo+mutation+rate+of+human+immunodeficiency+virus+type+1+than+predicted+from+the+fidelity+of+purified+reverse+transcriptase.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0000271-Kellam1" id="pone.0000271-Kellam1"></a>Kellam P, Larder BA (1995) Retroviral recombination can lead to linkage of reverse transcriptase mutations that confer increased zidovudine resistance. J Virol  69: 669–674.  <ul class="find" data-citedArticleID="935518"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Retroviral+recombination+can+lead+to+linkage+of+reverse+transcriptase+mutations+that+confer+increased+zidovudine+resistance.&amp;auth=&amp;atitle=Retroviral+recombination+can+lead+to+linkage+of+reverse+transcriptase+mutations+that+confer+increased+zidovudine+resistance." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Retroviral+recombination+can+lead+to+linkage+of+reverse+transcriptase+mutations+that+confer+increased+zidovudine+resistance." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Retroviral+recombination+can+lead+to+linkage+of+reverse+transcriptase+mutations+that+confer+increased+zidovudine+resistance.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0000271-Moutouh1" id="pone.0000271-Moutouh1"></a>Moutouh L, Corbeil J, Richman DD (1996) Recombination leads to the rapid emergence of HIV-1 dually resistant mutants under selective drug pressure. Proc Natl Acad Sci U S A  93: 6106–6111.  <ul class="find" data-citedArticleID="935558" data-doi="10.1073/pnas.93.12.6106"><li><a href="http://dx.doi.org/10.1073/pnas.93.12.6106" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Recombination+leads+to+the+rapid+emergence+of+HIV-1+dually+resistant+mutants+under+selective+drug+pressure." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Recombination+leads+to+the+rapid+emergence+of+HIV-1+dually+resistant+mutants+under+selective+drug+pressure.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0000271-VanOpijnen2" id="pone.0000271-VanOpijnen2"></a>Van Opijnen T, Jeeninga RE, Boerlijst MC, Pollakis GP, Zetterberg V, et al.  (2004) Human immunodeficiency virus type 1 subtypes have a distinct long terminal repeat that determines the replication rate in a host-cell-specific manner. J Virol  78: 3675–3683.  <ul class="find" data-citedArticleID="935604" data-doi="10.1128/jvi.78.7.3675-3683.2004"><li><a href="http://dx.doi.org/10.1128/jvi.78.7.3675-3683.2004" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Human+immunodeficiency+virus+type+1+subtypes+have+a+distinct+long+terminal+repeat+that+determines+the+replication+rate+in+a+host-cell-specific+manner." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Human+immunodeficiency+virus+type+1+subtypes+have+a+distinct+long+terminal+repeat+that+determines+the+replication+rate+in+a+host-cell-specific+manner.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0000271-Hemonnot1" id="pone.0000271-Hemonnot1"></a>Hemonnot B, Cartier C, Gay B, Rebuffat S, Bardy M, et al.  (2004) The host cell MAP kinase ERK-2 regulates viral assembly and release by phosphorylating the p6gag protein of HIV-1. J Biol Chem  279: 32426–32434.  <ul class="find" data-citedArticleID="935506" data-doi="10.1074/jbc.m313137200"><li><a href="http://dx.doi.org/10.1074/jbc.m313137200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+host+cell+MAP+kinase+ERK-2+regulates+viral+assembly+and+release+by+phosphorylating+the+p6gag+protein+of+HIV-1." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+host+cell+MAP+kinase+ERK-2+regulates+viral+assembly+and+release+by+phosphorylating+the+p6gag+protein+of+HIV-1.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0000271-Levy1" id="pone.0000271-Levy1"></a>Levy DN, Aldrovandi GM, Kutsch O, Shaw GM (2004) Dynamics of HIV-1 recombination in its natural target cells. Proc Natl Acad Sci U S A  101: 4204–4209.  <ul class="find" data-citedArticleID="935538" data-doi="10.1073/pnas.0306764101"><li><a href="http://dx.doi.org/10.1073/pnas.0306764101" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Dynamics+of+HIV-1+recombination+in+its+natural+target+cells." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Dynamics+of+HIV-1+recombination+in+its+natural+target+cells.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0000271-Negroni1" id="pone.0000271-Negroni1"></a>Negroni M, Buc H (2001) Mechanisms of retroviral recombination. AnnuRevGenet  35: 275–302.  <ul class="find" data-citedArticleID="935562"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Mechanisms+of+retroviral+recombination.&amp;auth=&amp;atitle=Mechanisms+of+retroviral+recombination." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mechanisms+of+retroviral+recombination." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mechanisms+of+retroviral+recombination.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0000271-Boerlijst1" id="pone.0000271-Boerlijst1"></a>Boerlijst MC, Bonhoeffer S, Nowak MA (1996) Viral quasi-species and recombination. Proc R SocLond B Biol Sci  263: 1577–1584.  <ul class="find" data-citedArticleID="935462" data-doi="10.1098/rspb.1996.0231"><li><a href="http://dx.doi.org/10.1098/rspb.1996.0231" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Viral+quasi-species+and+recombination." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Viral+quasi-species+and+recombination.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0000271-Leitner1" id="pone.0000271-Leitner1"></a>Leitner T, Escanilla D, Marquina S, Wahlberg J, Brostrom C, et al.  (1995) Biological and molecular characterization of subtype D, G, and A/D recombinant HIV-1 transmissions in Sweden. Virology  209: 136–146.  <ul class="find" data-citedArticleID="935534" data-doi="10.1006/viro.1995.1237"><li><a href="http://dx.doi.org/10.1006/viro.1995.1237" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Biological+and+molecular+characterization+of+subtype+D%2C+G%2C+and+A%2FD+recombinant+HIV-1+transmissions+in+Sweden." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Biological+and+molecular+characterization+of+subtype+D%2C+G%2C+and+A%2FD+recombinant+HIV-1+transmissions+in+Sweden.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0000271-Charpentier1" id="pone.0000271-Charpentier1"></a>Charpentier C, Nora T, Tenaillon O, Clavel F, Hance AJ (2006) Extensive recombination among human immunodeficiency virus type 1 quasispecies makes an important contribution to viral diversity in individual patients. J Virol  80: 2472–2482.  <ul class="find" data-citedArticleID="935472" data-doi="10.1128/jvi.80.5.2472-2482.2006"><li><a href="http://dx.doi.org/10.1128/jvi.80.5.2472-2482.2006" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Extensive+recombination+among+human+immunodeficiency+virus+type+1+quasispecies+makes+an+important+contribution+to+viral+diversity+in+individual+patients." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Extensive+recombination+among+human+immunodeficiency+virus+type+1+quasispecies+makes+an+important+contribution+to+viral+diversity+in+individual+patients.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0000271-Sabino1" id="pone.0000271-Sabino1"></a>Sabino E, Pan LZ, Cheng-Mayer C, Mayer A (1994) Comparison of in vivo plasma and peripheral blood mononuclear cell HIV-1 quasispecies to short-term tissue culture isolates: an analysis of tat and C2-V3 env regions. AIDS  8: 901–909.  <ul class="find" data-citedArticleID="935592" data-doi="10.1097/00002030-199407000-00006"><li><a href="http://dx.doi.org/10.1097/00002030-199407000-00006" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Comparison+of+in+vivo+plasma+and+peripheral+blood+mononuclear+cell+HIV-1+quasispecies+to+short-term+tissue+culture+isolates%3A+an+analysis+of+tat+and+C2-V3+env+regions." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Comparison+of+in+vivo+plasma+and+peripheral+blood+mononuclear+cell+HIV-1+quasispecies+to+short-term+tissue+culture+isolates%3A+an+analysis+of+tat+and+C2-V3+env+regions.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0000271-Rouzine1" id="pone.0000271-Rouzine1"></a>Rouzine IM, Coffin JM (2005) Evolution of human immunodeficiency virus under selection and weak recombination. Genetics  170: 7–18.  <ul class="find" data-citedArticleID="935588" data-doi="10.1534/genetics.104.029926"><li><a href="http://dx.doi.org/10.1534/genetics.104.029926" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evolution+of+human+immunodeficiency+virus+under+selection+and+weak+recombination." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evolution+of+human+immunodeficiency+virus+under+selection+and+weak+recombination.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0000271-Bocharov1" id="pone.0000271-Bocharov1"></a>Bocharov G, Ford NJ, Edwards J, Breinig T, Wain-Hobson S, et al.  (2005) A genetic-algorithm approach to simulating human immunodeficiency virus evolution reveals the strong impact of multiply infected cells and recombination. J Gen Virol  86: 3109–3118.  <ul class="find" data-citedArticleID="935460" data-doi="10.1099/vir.0.81138-0"><li><a href="http://dx.doi.org/10.1099/vir.0.81138-0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+genetic-algorithm+approach+to+simulating+human+immunodeficiency+virus+evolution+reveals+the+strong+impact+of+multiply+infected+cells+and+recombination." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+genetic-algorithm+approach+to+simulating+human+immunodeficiency+virus+evolution+reveals+the+strong+impact+of+multiply+infected+cells+and+recombination.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0000271-Althaus1" id="pone.0000271-Althaus1"></a>Althaus CL, Bonhoeffer S (2005) Stochastic interplay between mutation and recombination during the acquisition of drug resistance mutations in human immunodeficiency virus type 1. J Virol  79: 13572–13578.  <ul class="find" data-citedArticleID="935448" data-doi="10.1128/jvi.79.21.13572-13578.2005"><li><a href="http://dx.doi.org/10.1128/jvi.79.21.13572-13578.2005" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Stochastic+interplay+between+mutation+and+recombination+during+the+acquisition+of+drug+resistance+mutations+in+human+immunodeficiency+virus+type+1." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Stochastic+interplay+between+mutation+and+recombination+during+the+acquisition+of+drug+resistance+mutations+in+human+immunodeficiency+virus+type+1.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0000271-Fisher1" id="pone.0000271-Fisher1"></a>Fisher RA (1930) The Genetical Theory of Natural Selection. Oxford, UK: Oxford University Press.   <ul class="find-nolinks"></ul></li><li><span class="label">24.
              </span><a name="pone.0000271-Muller1" id="pone.0000271-Muller1"></a>Muller HJ (1932) Some genetic aspects of sex. American Naturalist  66: 118–138.  <ul class="find" data-citedArticleID="935560" data-doi="10.1086/280418"><li><a href="http://dx.doi.org/10.1086/280418" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Some+genetic+aspects+of+sex." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Some+genetic+aspects+of+sex.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0000271-Otto1" id="pone.0000271-Otto1"></a>Otto SP, Lenormand T (2002) Resolving the paradox of sex and recombination. NatRev Genet  3: 252–261.  <ul class="find" data-citedArticleID="935570" data-doi="10.1038/nrg761"><li><a href="http://dx.doi.org/10.1038/nrg761" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Resolving+the+paradox+of+sex+and+recombination." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Resolving+the+paradox+of+sex+and+recombination.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0000271-MaynardSmith1" id="pone.0000271-MaynardSmith1"></a>Maynard Smith J (1978) The Evolution of Sex. Cambridge, UK: Cambridge University Press.   <ul class="find-nolinks"></ul></li><li><span class="label">27.
              </span><a name="pone.0000271-Rice1" id="pone.0000271-Rice1"></a>Rice WR (2002) Experimental tests of the adaptive significance of sexual recombination. NatRev Genet  3: 241–251.  <ul class="find" data-citedArticleID="935580" data-doi="10.1038/nrg760"><li><a href="http://dx.doi.org/10.1038/nrg760" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Experimental+tests+of+the+adaptive+significance+of+sexual+recombination." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Experimental+tests+of+the+adaptive+significance+of+sexual+recombination.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0000271-MaynardSmith2" id="pone.0000271-MaynardSmith2"></a>Maynard Smith J (1968) Evolutionin sexual and asexual populations. American Naturalist  102: 469–473.  <ul class="find" data-citedArticleID="935550" data-doi="10.1086/282559"><li><a href="http://dx.doi.org/10.1086/282559" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evolutionin+sexual+and+asexual+populations." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evolutionin+sexual+and+asexual+populations.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0000271-Eshel1" id="pone.0000271-Eshel1"></a>Eshel I, Feldman MW (1970) On the evolutionary effect of recombination. TheorPopulBiol  1: 88–100.  <ul class="find" data-citedArticleID="935488" data-doi="10.1016/0040-5809(70)90043-2"><li><a href="http://dx.doi.org/10.1016/0040-5809(70)90043-2" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=On+the+evolutionary+effect+of+recombination." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22On+the+evolutionary+effect+of+recombination.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0000271-Peden1" id="pone.0000271-Peden1"></a>Peden K, Emerman M, Montagnier L (1991) Changes in growth properties on passage in tissue culture of viruses derived from infectious molecular clones of HIV-1 LAI , HIV-1 MAL , and HIV-1 ELI Virology  185: 661–672.  <ul class="find" data-citedArticleID="935576" data-doi="10.1016/0042-6822(91)90537-l"><li><a href="http://dx.doi.org/10.1016/0042-6822(91)90537-l" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Changes+in+growth+properties+on+passage+in+tissue+culture+of+viruses+derived+from+infectious+molecular+clones+of+HIV-1+LAI+%2C+HIV-1+MAL+%2C+and+HIV-1+ELI" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Changes+in+growth+properties+on+passage+in+tissue+culture+of+viruses+derived+from+infectious+molecular+clones+of+HIV-1+LAI+%2C+HIV-1+MAL+%2C+and+HIV-1+ELI%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0000271-Ross1" id="pone.0000271-Ross1"></a>Ross EK, Buckler White AJ, Rabson AB, Englund G, Martin MA (1991) Contribution of NF-kappa B and Sp1 binding motifs to the replicative capacity of human immunodeficiency virus type 1: distinct patterns of viral growth are determined by T-cell types. The Journal of Virology  65: 4350–4358.  <ul class="find" data-citedArticleID="935586"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Contribution+of+NF-kappa+B+and+Sp1+binding+motifs+to+the+replicative+capacity+of+human+immunodeficiency+virus+type+1%3A+distinct+patterns+of+viral+growth+are+determined+by+T-cell+types.&amp;auth=&amp;atitle=Contribution+of+NF-kappa+B+and+Sp1+binding+motifs+to+the+replicative+capacity+of+human+immunodeficiency+virus+type+1%3A+distinct+patterns+of+viral+growth+are+determined+by+T-cell+types." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Contribution+of+NF-kappa+B+and+Sp1+binding+motifs+to+the+replicative+capacity+of+human+immunodeficiency+virus+type+1%3A+distinct+patterns+of+viral+growth+are+determined+by+T-cell+types." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Contribution+of+NF-kappa+B+and+Sp1+binding+motifs+to+the+replicative+capacity+of+human+immunodeficiency+virus+type+1%3A+distinct+patterns+of+viral+growth+are+determined+by+T-cell+types.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0000271-Smith1" id="pone.0000271-Smith1"></a>Smith SD, Shatsky M, Cohen PS, Warnke R, Link MP, et al.  (1984) Monoclonal antibody and enzymatic profiles of human malignant T-lymphoid cells and derived cell lines. Cancer Res  44: 5657–5660.  <ul class="find" data-citedArticleID="935596"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Monoclonal+antibody+and+enzymatic+profiles+of+human+malignant+T-lymphoid+cells+and+derived+cell+lines.&amp;auth=&amp;atitle=Monoclonal+antibody+and+enzymatic+profiles+of+human+malignant+T-lymphoid+cells+and+derived+cell+lines." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Monoclonal+antibody+and+enzymatic+profiles+of+human+malignant+T-lymphoid+cells+and+derived+cell+lines." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Monoclonal+antibody+and+enzymatic+profiles+of+human+malignant+T-lymphoid+cells+and+derived+cell+lines.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0000271-VanOpijnen3" id="pone.0000271-VanOpijnen3"></a>Van Opijnen T, Boerlijst MC, Berkhout  (2006) Effects of random mutations in the Human Immunodeficiency Virus type I transcriptional promoter on viral fitness in different host-cell environments. J Virol.   in press.  <ul class="find" data-citedArticleID="935606" data-doi="10.1128/jvi.02547-05"><li><a href="http://dx.doi.org/10.1128/jvi.02547-05" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Effects+of+random+mutations+in+the+Human+Immunodeficiency+Virus+type+I+transcriptional+promoter+on+viral+fitness+in+different+host-cell+environments." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Effects+of+random+mutations+in+the+Human+Immunodeficiency+Virus+type+I+transcriptional+promoter+on+viral+fitness+in+different+host-cell+environments.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0000271-Ebert1" id="pone.0000271-Ebert1"></a>Ebert D (1998) Experimental evolution of parasites. Science  282: 1432–1435.  <ul class="find" data-citedArticleID="935484" data-doi="10.1126/science.282.5393.1432"><li><a href="http://dx.doi.org/10.1126/science.282.5393.1432" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Experimental+evolution+of+parasites." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Experimental+evolution+of+parasites.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0000271-Eigen1" id="pone.0000271-Eigen1"></a>Eigen M (1987) New concepts for dealing with the evolution of nucleic acids. Cold Spring HarbSympQuantBiol  52: 307–320.  <ul class="find" data-citedArticleID="935486" data-doi="10.1101/sqb.1987.052.01.036"><li><a href="http://dx.doi.org/10.1101/sqb.1987.052.01.036" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=New+concepts+for+dealing+with+the+evolution+of+nucleic+acids." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22New+concepts+for+dealing+with+the+evolution+of+nucleic+acids.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">36.
              </span><a name="pone.0000271-Kauffman1" id="pone.0000271-Kauffman1"></a>Kauffman SA (1993) The Origins of Order. Oxford Univ. Press. New York.  <ul class="find-nolinks"></ul></li><li><span class="label">37.
              </span><a name="pone.0000271-Fu1" id="pone.0000271-Fu1"></a>Fu YX, Li WH (1993) Statistical tests of neutrality of mutations. Genetics  133: 693–709.  <ul class="find" data-citedArticleID="935494"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Statistical+tests+of+neutrality+of+mutations.&amp;auth=&amp;atitle=Statistical+tests+of+neutrality+of+mutations." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Statistical+tests+of+neutrality+of+mutations." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Statistical+tests+of+neutrality+of+mutations.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">38.
              </span><a name="pone.0000271-Miralles1" id="pone.0000271-Miralles1"></a>Miralles R, Gerrish PJ, Moya A, Elena SF (1999) Clonal interference and the evolution of RNA viruses. Science  285: 1745–1747.  <ul class="find" data-citedArticleID="935552" data-doi="10.1126/science.285.5434.1745"><li><a href="http://dx.doi.org/10.1126/science.285.5434.1745" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Clonal+interference+and+the+evolution+of+RNA+viruses." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Clonal+interference+and+the+evolution+of+RNA+viruses.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">39.
              </span><a name="pone.0000271-Miralles2" id="pone.0000271-Miralles2"></a>Miralles R, Moya A, Elena SF (2000) Diminishing returns of population size in the rate of RNA virus adaptation. J Virol  74: 3566–3571.  <ul class="find" data-citedArticleID="935554" data-doi="10.1128/jvi.74.8.3566-3571.2000"><li><a href="http://dx.doi.org/10.1128/jvi.74.8.3566-3571.2000" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Diminishing+returns+of+population+size+in+the+rate+of+RNA+virus+adaptation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Diminishing+returns+of+population+size+in+the+rate+of+RNA+virus+adaptation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">40.
              </span><a name="pone.0000271-Hill1" id="pone.0000271-Hill1"></a>Hill WG, Robertson A (1966) The effect of linkage on limits to artificial selection. GenetRes  8: 269–294.  <ul class="find" data-citedArticleID="935510" data-doi="10.1017/s0016672300010156"><li><a href="http://dx.doi.org/10.1017/s0016672300010156" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+effect+of+linkage+on+limits+to+artificial+selection." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+effect+of+linkage+on+limits+to+artificial+selection.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">41.
              </span><a name="pone.0000271-Barton1" id="pone.0000271-Barton1"></a>Barton NH (1995) Linkage and the limits to natural selection. Genetics  140: 821–841.  <ul class="find" data-citedArticleID="935452"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Linkage+and+the+limits+to+natural+selection.&amp;auth=&amp;atitle=Linkage+and+the+limits+to+natural+selection." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Linkage+and+the+limits+to+natural+selection." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Linkage+and+the+limits+to+natural+selection.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">42.
              </span><a name="pone.0000271-Gerrish1" id="pone.0000271-Gerrish1"></a>Gerrish PJ, Lenski RE (1998) The fate of competing beneficial mutations in an asexual population. Genetica  102–103: 127–144.  <ul class="find" data-citedArticleID="935496" data-doi="10.1007/978-94-011-5210-5_12"><li><a href="http://dx.doi.org/10.1007/978-94-011-5210-5_12" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+fate+of+competing+beneficial+mutations+in+an+asexual+population." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+fate+of+competing+beneficial+mutations+in+an+asexual+population.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">43.
              </span><a name="pone.0000271-Orr1" id="pone.0000271-Orr1"></a>Orr HA (2000) The rate of adaptation in asexuals. Genetics  155: 961–968.  <ul class="find" data-citedArticleID="935568"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=The+rate+of+adaptation+in+asexuals.&amp;auth=&amp;atitle=The+rate+of+adaptation+in+asexuals." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+rate+of+adaptation+in+asexuals." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+rate+of+adaptation+in+asexuals.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">44.
              </span><a name="pone.0000271-Kim1" id="pone.0000271-Kim1"></a>Kim Y, Stephan W (2003) Selective sweeps in the presence of interference among partially linked loci. Genetics  164: 389–398.  <ul class="find" data-citedArticleID="935522"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Selective+sweeps+in+the+presence+of+interference+among+partially+linked+loci.&amp;auth=&amp;atitle=Selective+sweeps+in+the+presence+of+interference+among+partially+linked+loci." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Selective+sweeps+in+the+presence+of+interference+among+partially+linked+loci." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Selective+sweeps+in+the+presence+of+interference+among+partially+linked+loci.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">45.
              </span><a name="pone.0000271-deVisser1" id="pone.0000271-deVisser1"></a>de Visser JA, Zeyl CW, Gerrish PJ, Blanchard JL, Lenski RE (1999) Diminishing returns from mutation supply rate in asexual populations. Science  283: 404–406.  <ul class="find" data-citedArticleID="935614" data-doi="10.1126/science.283.5400.404"><li><a href="http://dx.doi.org/10.1126/science.283.5400.404" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Diminishing+returns+from+mutation+supply+rate+in+asexual+populations." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Diminishing+returns+from+mutation+supply+rate+in+asexual+populations.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">46.
              </span><a name="pone.0000271-Otto2" id="pone.0000271-Otto2"></a>Otto SP, Barton NH (1997) The evolution of recombination: removing the limits to natural selection. Genetics  147: 879–906.  <ul class="find" data-citedArticleID="935572"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=The+evolution+of+recombination%3A+removing+the+limits+to+natural+selection.&amp;auth=&amp;atitle=The+evolution+of+recombination%3A+removing+the+limits+to+natural+selection." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+evolution+of+recombination%3A+removing+the+limits+to+natural+selection." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+evolution+of+recombination%3A+removing+the+limits+to+natural+selection.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">47.
              </span><a name="pone.0000271-Hey1" id="pone.0000271-Hey1"></a>Hey J (1998) Selfish genes, pleiotropy and the origin of recombination. Genetics  149: 2089–2097.  <ul class="find" data-citedArticleID="935508"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Selfish+genes%2C+pleiotropy+and+the+origin+of+recombination.&amp;auth=&amp;atitle=Selfish+genes%2C+pleiotropy+and+the+origin+of+recombination." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Selfish+genes%2C+pleiotropy+and+the+origin+of+recombination." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Selfish+genes%2C+pleiotropy+and+the+origin+of+recombination.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">48.
              </span><a name="pone.0000271-Butcher1" id="pone.0000271-Butcher1"></a>Butcher D (1995) Muller's ratchet, epistasis and mutation effects. Genetics  141: 431–437.  <ul class="find" data-citedArticleID="935466"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Muller%27s+ratchet%2C+epistasis+and+mutation+effects.&amp;auth=&amp;atitle=Muller%27s+ratchet%2C+epistasis+and+mutation+effects." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Muller%27s+ratchet%2C+epistasis+and+mutation+effects." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Muller%27s+ratchet%2C+epistasis+and+mutation+effects.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">49.
              </span><a name="pone.0000271-Gordo1" id="pone.0000271-Gordo1"></a>Gordo I, Charlesworth B (2000) On the speed of Muller's ratchet. Genetics  156: 2137–2140.  <ul class="find" data-citedArticleID="935500" data-doi="10.1017/s0016672301005213"><li><a href="http://dx.doi.org/10.1017/s0016672301005213" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=On+the+speed+of+Muller%27s+ratchet." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22On+the+speed+of+Muller%27s+ratchet.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">50.
              </span><a name="pone.0000271-Gessler1" id="pone.0000271-Gessler1"></a>Gessler DD (1995) The constraints of finite size in asexual populations and the rate of the ratchet. GenetRes  66: 241–253.  <ul class="find" data-citedArticleID="935498" data-doi="10.1017/s0016672300034686"><li><a href="http://dx.doi.org/10.1017/s0016672300034686" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+constraints+of+finite+size+in+asexual+populations+and+the+rate+of+the+ratchet." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+constraints+of+finite+size+in+asexual+populations+and+the+rate+of+the+ratchet.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">51.
              </span><a name="pone.0000271-Novella1" id="pone.0000271-Novella1"></a>Novella IS, Elena SF, Moya A, Domingo E, Holland JJ (1995) Size of genetic bottlenecks leading to virus fitness loss is determined by mean initial population fitness. J Virol  69: 2869–2872.  <ul class="find" data-citedArticleID="935566"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Size+of+genetic+bottlenecks+leading+to+virus+fitness+loss+is+determined+by+mean+initial+population+fitness.&amp;auth=&amp;atitle=Size+of+genetic+bottlenecks+leading+to+virus+fitness+loss+is+determined+by+mean+initial+population+fitness." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Size+of+genetic+bottlenecks+leading+to+virus+fitness+loss+is+determined+by+mean+initial+population+fitness." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Size+of+genetic+bottlenecks+leading+to+virus+fitness+loss+is+determined+by+mean+initial+population+fitness.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">52.
              </span><a name="pone.0000271-Wright1" id="pone.0000271-Wright1"></a>Wright S (1932) The roles of mutation, inbreeding, crossbreeding and selection in evolution. Proc Sixth Int Congress Genetics  1: 356–366.  <ul class="find" data-citedArticleID="935610"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=The+roles+of+mutation%2C+inbreeding%2C+crossbreeding+and+selection+in+evolution.&amp;auth=&amp;atitle=The+roles+of+mutation%2C+inbreeding%2C+crossbreeding+and+selection+in+evolution." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+roles+of+mutation%2C+inbreeding%2C+crossbreeding+and+selection+in+evolution." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+roles+of+mutation%2C+inbreeding%2C+crossbreeding+and+selection+in+evolution.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">53.
              </span><a name="pone.0000271-Back1" id="pone.0000271-Back1"></a>Back NKT, Nijhuis M, Keulen W, Boucher CAB, Oude Essink BB, et al.  (1996) Reduced replication of 3TC-resistant HIV-1 variants in primary cells due to a processivity defect of the reverse transcriptase enzyme. The EMBO Journal  15: 4040–4049.  <ul class="find" data-citedArticleID="935450"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Reduced+replication+of+3TC-resistant+HIV-1+variants+in+primary+cells+due+to+a+processivity+defect+of+the+reverse+transcriptase+enzyme.&amp;auth=&amp;atitle=Reduced+replication+of+3TC-resistant+HIV-1+variants+in+primary+cells+due+to+a+processivity+defect+of+the+reverse+transcriptase+enzyme." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Reduced+replication+of+3TC-resistant+HIV-1+variants+in+primary+cells+due+to+a+processivity+defect+of+the+reverse+transcriptase+enzyme." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Reduced+replication+of+3TC-resistant+HIV-1+variants+in+primary+cells+due+to+a+processivity+defect+of+the+reverse+transcriptase+enzyme.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">54.
              </span><a name="pone.0000271-Boucher1" id="pone.0000271-Boucher1"></a>Boucher CAB, O'Sullivan E, Mulder JW, Ramautarsing C, Kellam P, et al.  (1992) Ordered appearance of zidovudine resistance mutations during treatment of 18 human immunodeficiency virus-positive subjects. Journal of Infectious Diseases  165: 105–110.  <ul class="find" data-citedArticleID="935464" data-doi="10.1093/infdis/165.1.105"><li><a href="http://dx.doi.org/10.1093/infdis/165.1.105" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Ordered+appearance+of+zidovudine+resistance+mutations+during+treatment+of+18+human+immunodeficiency+virus-positive+subjects." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Ordered+appearance+of+zidovudine+resistance+mutations+during+treatment+of+18+human+immunodeficiency+virus-positive+subjects.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">55.
              </span><a name="pone.0000271-Larder1" id="pone.0000271-Larder1"></a>Larder BA (1994) Interactions between drug resistance mutations in human immunodeficiency virus type 1 reverse transcriptase. Journal of General Virology  75: 951–957.  <ul class="find" data-citedArticleID="935530" data-doi="10.1099/0022-1317-75-5-951"><li><a href="http://dx.doi.org/10.1099/0022-1317-75-5-951" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Interactions+between+drug+resistance+mutations+in+human+immunodeficiency+virus+type+1+reverse+transcriptase." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Interactions+between+drug+resistance+mutations+in+human+immunodeficiency+virus+type+1+reverse+transcriptase.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">56.
              </span><a name="pone.0000271-Keulen1" id="pone.0000271-Keulen1"></a>Keulen W, Boucher C, Berkhout B (1996) Nucleotide substitution patterns can predict the requirements for drug-resistance of HIV-1 proteins. Antiviral Research  31: 45–57.  <ul class="find" data-citedArticleID="935520" data-doi="10.1016/0166-3542(96)00944-8"><li><a href="http://dx.doi.org/10.1016/0166-3542(96)00944-8" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Nucleotide+substitution+patterns+can+predict+the+requirements+for+drug-resistance+of+HIV-1+proteins." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Nucleotide+substitution+patterns+can+predict+the+requirements+for+drug-resistance+of+HIV-1+proteins.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">57.
              </span><a name="pone.0000271-Croteau1" id="pone.0000271-Croteau1"></a>Croteau G, Doyon L, Thibeault D, McKercher G, Pilote L, et al.  (1997) Impaired fitness of human immunodeficiency virus type 1 variants with high-level resistance to protease inhibitors. The Journal of Virology  71: 1089–1096.  <ul class="find" data-citedArticleID="935478"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Impaired+fitness+of+human+immunodeficiency+virus+type+1+variants+with+high-level+resistance+to+protease+inhibitors.&amp;auth=&amp;atitle=Impaired+fitness+of+human+immunodeficiency+virus+type+1+variants+with+high-level+resistance+to+protease+inhibitors." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Impaired+fitness+of+human+immunodeficiency+virus+type+1+variants+with+high-level+resistance+to+protease+inhibitors." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Impaired+fitness+of+human+immunodeficiency+virus+type+1+variants+with+high-level+resistance+to+protease+inhibitors.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">58.
              </span><a name="pone.0000271-Doyon1" id="pone.0000271-Doyon1"></a>Doyon L, Payant C, Brakier-Gingras L, Lamarre D (1998) Novel Gag-Pol frameshift site in Human Immunodeficiency Virus type 1 variants resistant to Protease inhibitors. The Journal of Virology  72: 6146–6150.  <ul class="find" data-citedArticleID="935482"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Novel+Gag-Pol+frameshift+site+in+Human+Immunodeficiency+Virus+type+1+variants+resistant+to+Protease+inhibitors.&amp;auth=&amp;atitle=Novel+Gag-Pol+frameshift+site+in+Human+Immunodeficiency+Virus+type+1+variants+resistant+to+Protease+inhibitors." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Novel+Gag-Pol+frameshift+site+in+Human+Immunodeficiency+Virus+type+1+variants+resistant+to+Protease+inhibitors." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Novel+Gag-Pol+frameshift+site+in+Human+Immunodeficiency+Virus+type+1+variants+resistant+to+Protease+inhibitors.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">59.
              </span><a name="pone.0000271-Mammano1" id="pone.0000271-Mammano1"></a>Mammano F, Petit C, Clavel F (1998) Resistance-associated loss of viral fitness in human immunodeficiency virus type 1: phenotypic analysis of Protease and gag coevolution in protease inhibitor-treated patients. The Journal of Virology  72: 7632–7637.  <ul class="find" data-citedArticleID="935540"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Resistance-associated+loss+of+viral+fitness+in+human+immunodeficiency+virus+type+1%3A+phenotypic+analysis+of+Protease+and+gag+coevolution+in+protease+inhibitor-treated+patients.&amp;auth=&amp;atitle=Resistance-associated+loss+of+viral+fitness+in+human+immunodeficiency+virus+type+1%3A+phenotypic+analysis+of+Protease+and+gag+coevolution+in+protease+inhibitor-treated+patients." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Resistance-associated+loss+of+viral+fitness+in+human+immunodeficiency+virus+type+1%3A+phenotypic+analysis+of+Protease+and+gag+coevolution+in+protease+inhibitor-treated+patients." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Resistance-associated+loss+of+viral+fitness+in+human+immunodeficiency+virus+type+1%3A+phenotypic+analysis+of+Protease+and+gag+coevolution+in+protease+inhibitor-treated+patients.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">60.
              </span><a name="pone.0000271-Bleiber1" id="pone.0000271-Bleiber1"></a>Bleiber G, Munoz M, Ciuffi A, Meylan P, Telenti A (2001) Individual contributions of mutant protease and reverse transcriptase to viral infectivity, replication, and protein maturation of antiretroviral drug-resistant Human Immunodeficiency Virus type 1. The Journal of Virology  75: 3291–3300.  <ul class="find" data-citedArticleID="935458" data-doi="10.1128/jvi.75.7.3291-3300.2001"><li><a href="http://dx.doi.org/10.1128/jvi.75.7.3291-3300.2001" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Individual+contributions+of+mutant+protease+and+reverse+transcriptase+to+viral+infectivity%2C+replication%2C+and+protein+maturation+of+antiretroviral+drug-resistant+Human+Immunodeficiency+Virus+type+1." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Individual+contributions+of+mutant+protease+and+reverse+transcriptase+to+viral+infectivity%2C+replication%2C+and+protein+maturation+of+antiretroviral+drug-resistant+Human+Immunodeficiency+Virus+type+1.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">61.
              </span><a name="pone.0000271-Berkhout1" id="pone.0000271-Berkhout1"></a>Berkhout B (1999) HIV-1 evolution under pressure of protease inhibitors, climbing the stairs of viral fitness. Journal Biomedical Science  6: 298–305.  <ul class="find" data-citedArticleID="935456" data-doi="10.1159/000025401"><li><a href="http://dx.doi.org/10.1159/000025401" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=HIV-1+evolution+under+pressure+of+protease+inhibitors%2C+climbing+the+stairs+of+viral+fitness." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22HIV-1+evolution+under+pressure+of+protease+inhibitors%2C+climbing+the+stairs+of+viral+fitness.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">62.
              </span><a name="pone.0000271-Deeks1" id="pone.0000271-Deeks1"></a>Deeks SG, Wrin T, Liegler T, Hoh R, Hayden M, et al.  (2001) Virologic and immunologic consequences of discontinuing combination antiretroviral-drug therapy in HIV-infected patients with detectable viremia. The New England Journal of Medicine  344: 472–480.  <ul class="find" data-citedArticleID="935480" data-doi="10.1056/nejm200102153440702"><li><a href="http://dx.doi.org/10.1056/nejm200102153440702" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Virologic+and+immunologic+consequences+of+discontinuing+combination+antiretroviral-drug+therapy+in+HIV-infected+patients+with+detectable+viremia." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Virologic+and+immunologic+consequences+of+discontinuing+combination+antiretroviral-drug+therapy+in+HIV-infected+patients+with+detectable+viremia.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">63.
              </span><a name="pone.0000271-Frenkel1" id="pone.0000271-Frenkel1"></a>Frenkel LM, Mullins JI (2001) Should patients with drug-resistant HIV-1 continue to receive antiretroviral therapy? The New England Journal of Medicine  344: 520–522.  <ul class="find" data-citedArticleID="935492" data-doi="10.1056/nejm200102153440710"><li><a href="http://dx.doi.org/10.1056/nejm200102153440710" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Should+patients+with+drug-resistant+HIV-1+continue+to+receive+antiretroviral+therapy%3F" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Should+patients+with+drug-resistant+HIV-1+continue+to+receive+antiretroviral+therapy%3F%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">64.
              </span><a name="pone.0000271-Zaccarelli1" id="pone.0000271-Zaccarelli1"></a>Zaccarelli M, Tozzi V, Perno CF, Antinori A (2004) The challenge of antiretroviral-drug-resistant HIV: is there any possible clinical advantage? CurrHIV Res  2: 283–292.  <ul class="find" data-citedArticleID="935612" data-doi="10.2174/1570162043351192"><li><a href="http://dx.doi.org/10.2174/1570162043351192" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+challenge+of+antiretroviral-drug-resistant+HIV%3A+is+there+any+possible+clinical+advantage%3F" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+challenge+of+antiretroviral-drug-resistant+HIV%3A+is+there+any+possible+clinical+advantage%3F%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">65.
              </span><a name="pone.0000271-Clementi1" id="pone.0000271-Clementi1"></a>Clementi M (2004) Can modulation of viral fitness represent a target for anti-HIV-1 strategies? New Microbiol  27: 207–214.  <ul class="find" data-citedArticleID="935476"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Can+modulation+of+viral+fitness+represent+a+target+for+anti-HIV-1+strategies%3F&amp;auth=&amp;atitle=Can+modulation+of+viral+fitness+represent+a+target+for+anti-HIV-1+strategies%3F" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Can+modulation+of+viral+fitness+represent+a+target+for+anti-HIV-1+strategies%3F" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Can+modulation+of+viral+fitness+represent+a+target+for+anti-HIV-1+strategies%3F%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">66.
              </span><a name="pone.0000271-Tisdale1" id="pone.0000271-Tisdale1"></a>Tisdale M, Kemp SD, Parry NR, Larder BA (1993) Rapid in vitro selection of human immunodeficiency virus type 1 resistant to 3′-thiacytidine inhibitors due to a mutation in the YMDD region of reverse transcriptase. Proc Natl Acad Sci U S A  90: 5653–5656.  <ul class="find" data-citedArticleID="935598" data-doi="10.1073/pnas.90.12.5653"><li><a href="http://dx.doi.org/10.1073/pnas.90.12.5653" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Rapid+in+vitro+selection+of+human+immunodeficiency+virus+type+1+resistant+to+3%26%23x2032%3B-thiacytidine+inhibitors+due+to+a+mutation+in+the+YMDD+region+of+reverse+transcriptase." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Rapid+in+vitro+selection+of+human+immunodeficiency+virus+type+1+resistant+to+3%26%23x2032%3B-thiacytidine+inhibitors+due+to+a+mutation+in+the+YMDD+region+of+reverse+transcriptase.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">67.
              </span><a name="pone.0000271-Nijhuis1" id="pone.0000271-Nijhuis1"></a>Nijhuis M, Schuurman R, de Jong D, Erickson J, Gustchina E, et al.  (1999) Increased fitness of drug resistant HIV-1 protease as a result of acquisition of compensatory mutations during suboptimal therapy. AIDS  13: 2349–2359.  <ul class="find" data-citedArticleID="935564" data-doi="10.1097/00002030-199912030-00006"><li><a href="http://dx.doi.org/10.1097/00002030-199912030-00006" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Increased+fitness+of+drug+resistant+HIV-1+protease+as+a+result+of+acquisition+of+compensatory+mutations+during+suboptimal+therapy." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Increased+fitness+of+drug+resistant+HIV-1+protease+as+a+result+of+acquisition+of+compensatory+mutations+during+suboptimal+therapy.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">68.
              </span><a name="pone.0000271-Kondrashov1" id="pone.0000271-Kondrashov1"></a>Kondrashov AS (1984) Deleterious mutations as an evolutionary factor. 1. The advantage of recombination. GenetRes  44: 199–217.  <ul class="find" data-citedArticleID="935524" data-doi="10.1017/s0016672300026392"><li><a href="http://dx.doi.org/10.1017/s0016672300026392" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Deleterious+mutations+as+an+evolutionary+factor.+1.+The+advantage+of+recombination." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Deleterious+mutations+as+an+evolutionary+factor.+1.+The+advantage+of+recombination.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">69.
              </span><a name="pone.0000271-Rice2" id="pone.0000271-Rice2"></a>Rice WR (1998) Requisite mutational load, pathway epistasis and deterministic mutation accumulation in sexual versus asexual populations. Genetica  102–103: 71–81.  <ul class="find" data-citedArticleID="935582" data-doi="10.1007/978-94-011-5210-5_7"><li><a href="http://dx.doi.org/10.1007/978-94-011-5210-5_7" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Requisite+mutational+load%2C+pathway+epistasis+and+deterministic+mutation+accumulation+in+sexual+versus+asexual+populations." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Requisite+mutational+load%2C+pathway+epistasis+and+deterministic+mutation+accumulation+in+sexual+versus+asexual+populations.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">70.
              </span><a name="pone.0000271-Charlesworth1" id="pone.0000271-Charlesworth1"></a>Charlesworth B (1994) The effect of background selection against deleterious mutations on weakly selected, linked variants. GenetRes  63: 213–227.  <ul class="find" data-citedArticleID="935468" data-doi="10.1017/s0016672300032365"><li><a href="http://dx.doi.org/10.1017/s0016672300032365" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+effect+of+background+selection+against+deleterious+mutations+on+weakly+selected%2C+linked+variants." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+effect+of+background+selection+against+deleterious+mutations+on+weakly+selected%2C+linked+variants.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">71.
              </span><a name="pone.0000271-Charlesworth2" id="pone.0000271-Charlesworth2"></a>Charlesworth D, Charlesworth B, Morgan MT (1995) The pattern of neutral molecular variation under the background selection model. Genetics  141: 1619–1632.  <ul class="find" data-citedArticleID="935470"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=The+pattern+of+neutral+molecular+variation+under+the+background+selection+model.&amp;auth=&amp;atitle=The+pattern+of+neutral+molecular+variation+under+the+background+selection+model." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+pattern+of+neutral+molecular+variation+under+the+background+selection+model." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+pattern+of+neutral+molecular+variation+under+the+background+selection+model.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">72.
              </span><a name="pone.0000271-Peck1" id="pone.0000271-Peck1"></a>Peck JR (1994) A ruby in the rubbish: beneficial mutations, deleterious mutations and the evolution of sex. Genetics  137: 597–606.  <ul class="find" data-citedArticleID="935574"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=A+ruby+in+the+rubbish%3A+beneficial+mutations%2C+deleterious+mutations+and+the+evolution+of+sex.&amp;auth=&amp;atitle=A+ruby+in+the+rubbish%3A+beneficial+mutations%2C+deleterious+mutations+and+the+evolution+of+sex." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+ruby+in+the+rubbish%3A+beneficial+mutations%2C+deleterious+mutations+and+the+evolution+of+sex." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+ruby+in+the+rubbish%3A+beneficial+mutations%2C+deleterious+mutations+and+the+evolution+of+sex.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">73.
              </span><a name="pone.0000271-Gu1" id="pone.0000271-Gu1"></a>Gu Z, Gao Q, Faust EA, Wainberg MA (1995) Possible involvement of cell fusion and viral recombination in generation of human immunodeficiency virus variants that display dual resistance to AZT and 3TC. J Gen Virol  76 ( Pt 10): 2601–2605.  <ul class="find" data-citedArticleID="935502" data-doi="10.1099/0022-1317-76-10-2601"><li><a href="http://dx.doi.org/10.1099/0022-1317-76-10-2601" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Possible+involvement+of+cell+fusion+and+viral+recombination+in+generation+of+human+immunodeficiency+virus+variants+that+display+dual+resistance+to+AZT+and+3TC." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Possible+involvement+of+cell+fusion+and+viral+recombination+in+generation+of+human+immunodeficiency+virus+variants+that+display+dual+resistance+to+AZT+and+3TC.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">74.
              </span><a name="pone.0000271-Serviene1" id="pone.0000271-Serviene1"></a>Serviene E, Shapka N, Cheng CP, Panavas T, Phuangrat B, et al.  (2005) Genome-wide screen identifies host genes affecting viral RNA recombination. Proc Natl Acad Sci USA.   <ul class="find" data-citedArticleID="935594" data-doi="10.1073/pnas.0504844102"><li><a href="http://dx.doi.org/10.1073/pnas.0504844102" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genome-wide+screen+identifies+host+genes+affecting+viral+RNA+recombination." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genome-wide+screen+identifies+host+genes+affecting+viral+RNA+recombination.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">75.
              </span><a name="pone.0000271-Jeeninga1" id="pone.0000271-Jeeninga1"></a>Jeeninga RE, Hoogenkamp M, Armand-Ugon M, de Baar M, Verhoef K, et al.  (2000) Functional differences between the LTR transcriptional promoters of HIV-1 subtypes A through G. The Journal of Virology  74: 3740–3751.  <ul class="find" data-citedArticleID="935514" data-doi="10.1128/jvi.74.8.3740-3751.2000"><li><a href="http://dx.doi.org/10.1128/jvi.74.8.3740-3751.2000" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Functional+differences+between+the+LTR+transcriptional+promoters+of+HIV-1+subtypes+A+through+G." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Functional+differences+between+the+LTR+transcriptional+promoters+of+HIV-1+subtypes+A+through+G.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">76.
              </span><a name="pone.0000271-Chen1" id="pone.0000271-Chen1"></a>Chen BK, Feinberg MB, Baltimore D (1997) The k B sites in the Human Immunodeficiency Virus type 1 Long Terminal Repeat enhance virus replication yet are not absolutely required for viral growth. The Journal of Virology  71: 5495–5504.  <ul class="find" data-citedArticleID="935474"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=The+k+B+sites+in+the+Human+Immunodeficiency+Virus+type+1+Long+Terminal+Repeat+enhance+virus+replication+yet+are+not+absolutely+required+for+viral+growth.&amp;auth=&amp;atitle=The+k+B+sites+in+the+Human+Immunodeficiency+Virus+type+1+Long+Terminal+Repeat+enhance+virus+replication+yet+are+not+absolutely+required+for+viral+growth." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+k+B+sites+in+the+Human+Immunodeficiency+Virus+type+1+Long+Terminal+Repeat+enhance+virus+replication+yet+are+not+absolutely+required+for+viral+growth." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+k+B+sites+in+the+Human+Immunodeficiency+Virus+type+1+Long+Terminal+Repeat+enhance+virus+replication+yet+are+not+absolutely+required+for+viral+growth.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">77.
              </span><a name="pone.0000271-Beck1" id="pone.0000271-Beck1"></a>Beck Z, Bacsi A, Liu X, Ebbesen P, Andirko I, et al.  (2003) Differential patterns of human cytomegalovirus gene expression in various T-cell lines carrying human T-cell leukemia-lymphoma virus type I: role of Tax-activated cellular transcription factors. J Med Virol  71: 94–104.  <ul class="find" data-citedArticleID="935454" data-doi="10.1002/jmv.10447"><li><a href="http://dx.doi.org/10.1002/jmv.10447" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Differential+patterns+of+human+cytomegalovirus+gene+expression+in+various+T-cell+lines+carrying+human+T-cell+leukemia-lymphoma+virus+type+I%3A+role+of+Tax-activated+cellular+transcription+factors." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Differential+patterns+of+human+cytomegalovirus+gene+expression+in+various+T-cell+lines+carrying+human+T-cell+leukemia-lymphoma+virus+type+I%3A+role+of+Tax-activated+cellular+transcription+factors.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">78.
              </span><a name="pone.0000271-Mohapatra1" id="pone.0000271-Mohapatra1"></a>Mohapatra S, Chu B, Wei S, Djeu J, Epling-Burnette PK, et al.  (2003) Roscovitine inhibits STAT5 activity and induces apoptosis in the human leukemia virus type 1-transformed cell line MT-2. Cancer Res  63: 8523–8530.  <ul class="find" data-citedArticleID="935556"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Roscovitine+inhibits+STAT5+activity+and+induces+apoptosis+in+the+human+leukemia+virus+type+1-transformed+cell+line+MT-2.&amp;auth=&amp;atitle=Roscovitine+inhibits+STAT5+activity+and+induces+apoptosis+in+the+human+leukemia+virus+type+1-transformed+cell+line+MT-2." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Roscovitine+inhibits+STAT5+activity+and+induces+apoptosis+in+the+human+leukemia+virus+type+1-transformed+cell+line+MT-2." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Roscovitine+inhibits+STAT5+activity+and+induces+apoptosis+in+the+human+leukemia+virus+type+1-transformed+cell+line+MT-2.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">79.
              </span><a name="pone.0000271-Matsuo1" id="pone.0000271-Matsuo1"></a>Matsuo Y, Drexler HG, Harashima A, Okochi A, Shimizu N, et al.  (2004) Transcription factor expression in cell lines derived from natural killer-cell and natural killer-like T-cell leukemia-lymphoma. Hum Cell  17: 85–92.  <ul class="find" data-citedArticleID="935546" data-doi="10.1111/j.1749-0774.2004.tb00079.x"><li><a href="http://dx.doi.org/10.1111/j.1749-0774.2004.tb00079.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Transcription+factor+expression+in+cell+lines+derived+from+natural+killer-cell+and+natural+killer-like+T-cell+leukemia-lymphoma." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Transcription+factor+expression+in+cell+lines+derived+from+natural+killer-cell+and+natural+killer-like+T-cell+leukemia-lymphoma.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">80.
              </span><a name="pone.0000271-Larder2" id="pone.0000271-Larder2"></a>Larder BA, Kohli A, Kellam P, Kemp SD, Kronick M, et al.  (1993) Quantitative detection of HIV-1 drug resistance mutations by automated DNA sequencing. Nature  365: 671–673.  <ul class="find" data-citedArticleID="935532" data-doi="10.1038/365671a0"><li><a href="http://dx.doi.org/10.1038/365671a0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Quantitative+detection+of+HIV-1+drug+resistance+mutations+by+automated+DNA+sequencing." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Quantitative+detection+of+HIV-1+drug+resistance+mutations+by+automated+DNA+sequencing.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">81.
              </span><a name="pone.0000271-Kumar1" id="pone.0000271-Kumar1"></a>Kumar S, Tamura K, Nei M (1994) Molecular evolutionary genetics analysis (MEGA). ComputApplBiosc  10: 189–191.  <ul class="find" data-citedArticleID="935528" data-doi="10.1093/bioinformatics/10.2.189"><li><a href="http://dx.doi.org/10.1093/bioinformatics/10.2.189" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Molecular+evolutionary+genetics+analysis+%28MEGA%29." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Molecular+evolutionary+genetics+analysis+%28MEGA%29.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">82.
              </span><a name="pone.0000271-RozasJFauSanchezDelBarrio1" id="pone.0000271-RozasJFauSanchezDelBarrio1"></a>Rozas J Fau-Sanchez-DelBarrio JC, Sanchez-DelBarrio Jc Fau-Messeguer X, Messeguer X Fau-Rozas R, Rozas R Dna SPDNA polymorphism analyses by the coalescent and other methods.  <ul class="find" data-citedArticleID="935590" data-doi="10.1093/bioinformatics/btg359"><li><a href="http://dx.doi.org/10.1093/bioinformatics/btg359" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=DNA+polymorphism+analyses+by+the+coalescent+and+other+methods." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22DNA+polymorphism+analyses+by+the+coalescent+and+other+methods.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">83.
              </span><a name="pone.0000271-Lenski1" id="pone.0000271-Lenski1"></a>Lenski RE, Rose MRS, S.C.Tadler SC (1991) Long-Term experimental evolution in Escherichia coli. I. Adaptation and divergence during 2,000 generations. The American Naturalist  138: 1315–1341.  <ul class="find" data-citedArticleID="935536" data-doi="10.1086/285289"><li><a href="http://dx.doi.org/10.1086/285289" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Long-Term+experimental+evolution+in+Escherichia+coli.+I.+Adaptation+and+divergence+during+2%2C000+generations." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Long-Term+experimental+evolution+in+Escherichia+coli.+I.+Adaptation+and+divergence+during+2%2C000+generations.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">84.
              </span><a name="pone.0000271-Maree1" id="pone.0000271-Maree1"></a>Maree AF, Keulen W, Boucher CA, de Boer RJ (2000) Estimating relative fitness in viral competition experiments. J Virol  74: 11067–11072.  <ul class="find" data-citedArticleID="935544" data-doi="10.1128/jvi.74.23.11067-11072.2000"><li><a href="http://dx.doi.org/10.1128/jvi.74.23.11067-11072.2000" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Estimating+relative+fitness+in+viral+competition+experiments." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Estimating+relative+fitness+in+viral+competition+experiments.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000271.XML" value="137495"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000271.PDF" value="225086"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000271.t001.PNG_L" value="28420"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000271.t001.PNG_M" value="27075"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000271.t001.PNG_S" value="2127"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000271.t001.TIF" value="174664"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000271.t001.PNG_I" value="12795"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000271.t002.PNG_L" value="155375"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000271.t002.PNG_M" value="128285"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000271.t002.PNG_S" value="7852"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000271.t002.TIF" value="757812"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000271.t002.PNG_I" value="84770"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000271.g001.PNG_L" value="474395"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000271.g001.PNG_M" value="63626"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000271.g001.PNG_S" value="2965"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000271.g001.TIF" value="848756"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000271.g001.PNG_I" value="26823"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000271.t003.PNG_L" value="161140"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000271.t003.PNG_M" value="129373"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000271.t003.PNG_S" value="8028"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000271.t003.TIF" value="754158"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000271.t003.PNG_I" value="84815"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000271.t004.PNG_L" value="35425"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000271.t004.PNG_M" value="35300"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000271.t004.PNG_S" value="2181"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000271.t004.TIF" value="201480"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000271.t004.PNG_I" value="15543"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000271.t005.PNG_L" value="39755"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000271.t005.PNG_M" value="41538"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000271.t005.PNG_S" value="2590"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000271.t005.TIF" value="213308"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000271.t005.PNG_I" value="19867"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000271.g002.PNG_L" value="196963"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000271.g002.PNG_M" value="30274"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000271.g002.PNG_S" value="2316"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000271.g002.TIF" value="391184"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000271.g002.PNG_I" value="13968"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000271.t006.PNG_L" value="67356"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000271.t006.PNG_M" value="24948"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000271.t006.PNG_S" value="1853"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000271.t006.TIF" value="326328"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000271.t006.PNG_I" value="11515"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000271.e001.PNG" value="5686"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000271.e001.TIF" value="28672"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000271.s001.EPS" value="778183"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000271.s002.DOC" value="33280"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000271&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000271" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000271&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0000271&volume=&issue=&title=Adaptation of HIV-1 Depends on the Host-Cell Environment&author_name=Tim%20van%20Opijnen%2C%20Anthony%20de%20Ronde%2C%20Maarten%20C.%20Boerlijst%2C%20Ben%20Berkhout&start_page=1&end_page=9" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000271" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000271" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0000271" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000271&amp;t=Adaptation%20of%20HIV-1%20Depends%20on%20the%20Host-Cell%20Environment" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000271&title=Adaptation%20of%20HIV-1%20Depends%20on%20the%20Host-Cell%20Environment&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000271&amp;title=Adaptation%20of%20HIV-1%20Depends%20on%20the%20Host-Cell%20Environment" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000271" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0000271&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Adaptation of HIV-1 Depends on the Host-Cell Environment';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0000271';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Adaptation%20of%20HIV-1%20Depends%20on%20the%20Host-Cell%20Environment http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000271" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0000271" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">

















          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Genome+evolution%22" title="Search for articles in the subject area:'Genome evolution'"><div class="flagText">Genome evolution</div></a>
              <div data-categoryid="17207" data-articleid="23380"
                   data-categoryname="Genome evolution"
                   class="flagImage" title="Flag 'Genome evolution' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22HIV-1%22" title="Search for articles in the subject area:'HIV-1'"><div class="flagText">HIV-1</div></a>
              <div data-categoryid="42865" data-articleid="23380"
                   data-categoryname="HIV-1"
                   class="flagImage" title="Flag 'HIV-1' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Microbial+genetics%22" title="Search for articles in the subject area:'Microbial genetics'"><div class="flagText">Microbial genetics</div></a>
              <div data-categoryid="20041" data-articleid="23380"
                   data-categoryname="Microbial genetics"
                   class="flagImage" title="Flag 'Microbial genetics' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Microbial+mutation%22" title="Search for articles in the subject area:'Microbial mutation'"><div class="flagText">Microbial mutation</div></a>
              <div data-categoryid="20011" data-articleid="23380"
                   data-categoryname="Microbial mutation"
                   class="flagImage" title="Flag 'Microbial mutation' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Recombinant+proteins%22" title="Search for articles in the subject area:'Recombinant proteins'"><div class="flagText">Recombinant proteins</div></a>
              <div data-categoryid="18471" data-articleid="23380"
                   data-categoryname="Recombinant proteins"
                   class="flagImage" title="Flag 'Recombinant proteins' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Sequence+analysis%22" title="Search for articles in the subject area:'Sequence analysis'"><div class="flagText">Sequence analysis</div></a>
              <div data-categoryid="47475" data-articleid="23380"
                   data-categoryname="Sequence analysis"
                   class="flagImage" title="Flag 'Sequence analysis' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22T+cells%22" title="Search for articles in the subject area:'T cells'"><div class="flagText">T cells</div></a>
              <div data-categoryid="40147" data-articleid="23380"
                   data-categoryname="T cells"
                   class="flagImage" title="Flag 'T cells' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Viral+replication%22" title="Search for articles in the subject area:'Viral replication'"><div class="flagText">Viral replication</div></a>
              <div data-categoryid="20015" data-articleid="23380"
                   data-categoryname="Viral replication"
                   class="flagImage" title="Flag 'Viral replication' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=8804'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=9637'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=1995&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>


</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
