

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Genome Biology of Actinobacillus pleuropneumoniae JL03, an Isolate of Serotype 3 Prevalent in China</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0001450"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0001450" />

    <meta name="citation_title" content="Genome Biology of Actinobacillus pleuropneumoniae JL03, an Isolate of Serotype 3 Prevalent in China"/>
    <meta itemprop="name" content="Genome Biology of Actinobacillus pleuropneumoniae JL03, an Isolate of Serotype 3 Prevalent in China"/>

      <meta name="citation_author" content="Zhuofei Xu"/>
            <meta name="citation_author_institution" content="Division of Animal Infectious Disease, State Key Laboratory of Agricultural Microbiology, College of Veterinary Medicine, Huazhong Agricultural University, Wuhan, China"/>
      <meta name="citation_author" content="Yan Zhou"/>
            <meta name="citation_author_institution" content="Shanghai - MOST Key Laboratory of Health and Disease Genomics, Chinese National Human Genome Center at Shanghai, Shanghai, China"/>
            <meta name="citation_author_institution" content="State Key Laboratory of Genetic Engineering, Department of Microbiology, School of Life Science, Fudan University, Shanghai, China"/>
      <meta name="citation_author" content="Liangjun Li"/>
            <meta name="citation_author_institution" content="Division of Animal Infectious Disease, State Key Laboratory of Agricultural Microbiology, College of Veterinary Medicine, Huazhong Agricultural University, Wuhan, China"/>
      <meta name="citation_author" content="Rui Zhou"/>
            <meta name="citation_author_institution" content="Division of Animal Infectious Disease, State Key Laboratory of Agricultural Microbiology, College of Veterinary Medicine, Huazhong Agricultural University, Wuhan, China"/>
      <meta name="citation_author" content="Shaobo Xiao"/>
            <meta name="citation_author_institution" content="Division of Animal Infectious Disease, State Key Laboratory of Agricultural Microbiology, College of Veterinary Medicine, Huazhong Agricultural University, Wuhan, China"/>
      <meta name="citation_author" content="Yun Wan"/>
            <meta name="citation_author_institution" content="Division of Animal Infectious Disease, State Key Laboratory of Agricultural Microbiology, College of Veterinary Medicine, Huazhong Agricultural University, Wuhan, China"/>
      <meta name="citation_author" content="Sihua Zhang"/>
            <meta name="citation_author_institution" content="Division of Animal Infectious Disease, State Key Laboratory of Agricultural Microbiology, College of Veterinary Medicine, Huazhong Agricultural University, Wuhan, China"/>
      <meta name="citation_author" content="Kai Wang"/>
            <meta name="citation_author_institution" content="Shanghai - MOST Key Laboratory of Health and Disease Genomics, Chinese National Human Genome Center at Shanghai, Shanghai, China"/>
      <meta name="citation_author" content="Wei Li"/>
            <meta name="citation_author_institution" content="Division of Animal Infectious Disease, State Key Laboratory of Agricultural Microbiology, College of Veterinary Medicine, Huazhong Agricultural University, Wuhan, China"/>
      <meta name="citation_author" content="Lu Li"/>
            <meta name="citation_author_institution" content="Division of Animal Infectious Disease, State Key Laboratory of Agricultural Microbiology, College of Veterinary Medicine, Huazhong Agricultural University, Wuhan, China"/>
      <meta name="citation_author" content="Hui Jin"/>
            <meta name="citation_author_institution" content="Division of Animal Infectious Disease, State Key Laboratory of Agricultural Microbiology, College of Veterinary Medicine, Huazhong Agricultural University, Wuhan, China"/>
      <meta name="citation_author" content="Mingsong Kang"/>
            <meta name="citation_author_institution" content="Division of Animal Infectious Disease, State Key Laboratory of Agricultural Microbiology, College of Veterinary Medicine, Huazhong Agricultural University, Wuhan, China"/>
      <meta name="citation_author" content="Baolige Dalai"/>
            <meta name="citation_author_institution" content="Division of Animal Infectious Disease, State Key Laboratory of Agricultural Microbiology, College of Veterinary Medicine, Huazhong Agricultural University, Wuhan, China"/>
      <meta name="citation_author" content="Tingting Li"/>
            <meta name="citation_author_institution" content="Division of Animal Infectious Disease, State Key Laboratory of Agricultural Microbiology, College of Veterinary Medicine, Huazhong Agricultural University, Wuhan, China"/>
      <meta name="citation_author" content="Lei Liu"/>
            <meta name="citation_author_institution" content="Division of Animal Infectious Disease, State Key Laboratory of Agricultural Microbiology, College of Veterinary Medicine, Huazhong Agricultural University, Wuhan, China"/>
      <meta name="citation_author" content="Yangyi Cheng"/>
            <meta name="citation_author_institution" content="Shanghai - MOST Key Laboratory of Health and Disease Genomics, Chinese National Human Genome Center at Shanghai, Shanghai, China"/>
      <meta name="citation_author" content="Lei Zhang"/>
            <meta name="citation_author_institution" content="Shanghai - MOST Key Laboratory of Health and Disease Genomics, Chinese National Human Genome Center at Shanghai, Shanghai, China"/>
      <meta name="citation_author" content="Tao Xu"/>
            <meta name="citation_author_institution" content="Shanghai - MOST Key Laboratory of Health and Disease Genomics, Chinese National Human Genome Center at Shanghai, Shanghai, China"/>
      <meta name="citation_author" content="Huajun Zheng"/>
            <meta name="citation_author_institution" content="Shanghai - MOST Key Laboratory of Health and Disease Genomics, Chinese National Human Genome Center at Shanghai, Shanghai, China"/>
      <meta name="citation_author" content="Shiying Pu"/>
            <meta name="citation_author_institution" content="Shanghai - MOST Key Laboratory of Health and Disease Genomics, Chinese National Human Genome Center at Shanghai, Shanghai, China"/>
      <meta name="citation_author" content="Bofei Wang"/>
            <meta name="citation_author_institution" content="Shanghai - MOST Key Laboratory of Health and Disease Genomics, Chinese National Human Genome Center at Shanghai, Shanghai, China"/>
      <meta name="citation_author" content="Wenyi Gu"/>
            <meta name="citation_author_institution" content="Shanghai - MOST Key Laboratory of Health and Disease Genomics, Chinese National Human Genome Center at Shanghai, Shanghai, China"/>
      <meta name="citation_author" content="Xiang-Lin Zhang"/>
            <meta name="citation_author_institution" content="Shanghai - MOST Key Laboratory of Health and Disease Genomics, Chinese National Human Genome Center at Shanghai, Shanghai, China"/>
      <meta name="citation_author" content="Geng-Feng Zhu"/>
            <meta name="citation_author_institution" content="Shanghai - MOST Key Laboratory of Health and Disease Genomics, Chinese National Human Genome Center at Shanghai, Shanghai, China"/>
      <meta name="citation_author" content="Shengyue Wang"/>
            <meta name="citation_author_institution" content="Shanghai - MOST Key Laboratory of Health and Disease Genomics, Chinese National Human Genome Center at Shanghai, Shanghai, China"/>
            <meta name="citation_author_institution" content="State Key Laboratory of Genetic Engineering, Department of Microbiology, School of Life Science, Fudan University, Shanghai, China"/>
      <meta name="citation_author" content="Guo-Ping Zhao"/>
            <meta name="citation_author_institution" content="Shanghai - MOST Key Laboratory of Health and Disease Genomics, Chinese National Human Genome Center at Shanghai, Shanghai, China"/>
            <meta name="citation_author_institution" content="State Key Laboratory of Genetic Engineering, Department of Microbiology, School of Life Science, Fudan University, Shanghai, China"/>
            <meta name="citation_author_institution" content="National Engineering Center for Biochip Research at Shanghai, Shanghai, China"/>
            <meta name="citation_author_institution" content="Laboratory of Molecular Microbiology, Institute of Plant Physiology and Ecology, Shanghai Institutes for Biological Sciences, Chinese Academy of Sciences, Shanghai, China"/>
      <meta name="citation_author" content="Huanchun Chen"/>
            <meta name="citation_author_institution" content="Division of Animal Infectious Disease, State Key Laboratory of Agricultural Microbiology, College of Veterinary Medicine, Huazhong Agricultural University, Wuhan, China"/>

    <meta name="citation_date" content="2008/1/16"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0001450.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e1450"/>
    <meta name="citation_issue" content="1"/>
    <meta name="citation_volume" content="3"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Actinobacillus pleuropneumoniae: pathobiology and pathogenesis of infection.; citation_author=JT Bosse; citation_author=H Janson; citation_author=BJ Sheehan; citation_author=AJ Beddek; citation_author=AN Rycroft; citation_journal_title=Microbes Infect; citation_volume=4; citation_number=1; citation_pages=225-235; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Detection of Actinobacillus pleuropneumoniae in the porcine upper respiratory tract as a complement to serological tests.; citation_author=M Sidibé; citation_author=S Messier; citation_author=S Larivière; citation_author=M Gottschalk; citation_author=KR Mittal; citation_journal_title=Can J Vet Res; citation_volume=57; citation_number=2; citation_pages=204-208; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=Surface polysaccharides and iron-uptake systems of Actinobacillus pleuropneumoniae.; citation_author=M Jacques; citation_journal_title=Can J Vet Res; citation_volume=68; citation_number=3; citation_pages=81-85; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Enzymes involved in anaerobic respiration appear to play a role in Actinobacillus pleuropneumoniae virulence.; citation_author=I Jacobsen; citation_author=I Hennig-Pauka; citation_author=N Baltes; citation_author=M Trost; citation_author=GF Gerlach; citation_journal_title=Infect Immun; citation_volume=73; citation_number=4; citation_pages=226-234; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Proposal of a new serovar of Actinobacillus pleuropneumoniae: serovar 15.; citation_author=PJ Blackall; citation_author=HI Klaasen; citation_author=H van den Bosch; citation_author=P Kuhnert; citation_author=J Frey; citation_journal_title=Vet Microbiol; citation_volume=84; citation_number=5; citation_pages=47-52; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Study on the inactivated trivalent vaccine against swine infectious pleuropneumoniae: selection of the seed strain, preparation and safety trials of the vaccine.; citation_author=Z Lu; citation_author=P Zhao; citation_author=Y Shao; citation_author=J Liu; citation_author=B Lu; citation_journal_title=Chin J Vet Sci Tech; citation_volume=37; citation_number=6; citation_pages=33-35. (in Chinese); citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Complete genomic sequence of Pasteurella multocida, Pm70.; citation_author=BJ May; citation_author=Q Zhang; citation_author=LL Li; citation_author=ML Paustian; citation_author=TS Whittam; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=98; citation_number=7; citation_pages=3460-3465; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Whole-genome random sequencing and assembly of Haemophilus influenzae Rd.; citation_author=RD Fleischmann; citation_author=MD Adams; citation_author=O White; citation_author=RA Clayton; citation_author=EF Kirkness; citation_journal_title=Science; citation_volume=269; citation_number=8; citation_pages=496-512; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Genomic sequence of an otitis Media isolate of nontypeable Haemophilus influenzae: comparative study with H. influenzae serotype d, strain KW20.; citation_author=A Harrison; citation_author=DW Dyer; citation_author=A Gillaspy; citation_author=WC Ray; citation_author=R Mungur; citation_journal_title=J Bacteriol; citation_volume=187; citation_number=9; citation_pages=4627-4636; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=The genome sequence of the capnophilic rumen bacterium Mannheimia succiniciproducens.; citation_author=SH Hong; citation_author=JS Kim; citation_author=SY Lee; citation_author=YH In; citation_author=SS Choi; citation_journal_title=Nat Biotechnol; citation_volume=22; citation_number=10; citation_pages=1275-1281; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Complete genome sequence of Haemophilus somnus (Histophilus somni) strain 129Pt and comparison to Haemophilus ducreyi 35000HP and Haemophilus influenzae Rd.; citation_author=JF Challacombe; citation_author=AJ Duncan; citation_author=TS Brettin; citation_author=D Bruce; citation_author=O Chertkov; citation_journal_title=J Bacteriol; citation_volume=189; citation_number=11; citation_pages=1890-1898; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Where does bacterial replication start? Rules for predicting the oriC region.; citation_author=P Mackiewicz; citation_author=J Zakrzewska-Czerwinska; citation_author=A Zawilak; citation_author=MR Dudek; citation_author=S Cebrat; citation_journal_title=Nucleic Acids Res; citation_volume=25; citation_number=12; citation_pages=955-964; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Cloning and characterization of the chromosomal replication origin region of Amycolatopsis mediterranei U32.; citation_author=Y Tian; citation_author=P Hao; citation_author=G Zhao; citation_author=Z Qin; citation_journal_title=Biochem Biophys Res Commun; citation_volume=333; citation_number=13; citation_pages=14-20; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Identification of genes that are associated with DNA repeats in prokaryotes.; citation_author=R Jansen; citation_author=JDA Embden; citation_author=W van Gaastra; citation_author=LM Schouls; citation_journal_title=Mol Microbiol; citation_volume=43; citation_number=14; citation_pages=1565-1575; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Clustered regularly interspaced short palindrome repeats (CRISPRs) have spacers of extrachromosomal origin.; citation_author=A Bolotin; citation_author=B Quinquis; citation_author=A Sorokin; citation_author=SD Ehrlich; citation_journal_title=Microbiology; citation_volume=151; citation_number=15; citation_pages=2551-2561; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=CRISPR Provides Acquired Resistance Against Viruses in Prokaryotes.; citation_author=R Barrangou; citation_author=C Fremaux; citation_author=H Deveau; citation_author=M Richards; citation_author=P Boyaval; citation_journal_title=Science; citation_volume=315; citation_number=16; citation_pages=1709-1712; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Phylogeny of 54 representative strains of species in the family Pasteurellaceae as determined by comparison of 16S rRNA sequences.; citation_author=FE Dewhirst; citation_author=BJ Paster; citation_author=I Olsen; citation_author=GJ Fraser; citation_journal_title=J Bacteriol; citation_volume=174; citation_number=17; citation_pages=2002-2013; citation_date=1992; " />
      <meta name="citation_reference" content="citation_title=The genome sequence of Mannheimia haemolytica A1: Insights into Virulence, Natural Competence, and Pasteurellaceae Phylogeny.; citation_author=J Gioia; citation_author=X Qin; citation_author=H Jiang; citation_author=K Clinkenbeard; citation_author= Lo; citation_journal_title=J Bacteriol; citation_volume=188; citation_number=18; citation_pages=7257-7266; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=The maltodextrin system of Escherichia coli: metabolism and transport.; citation_author=R Dippel; citation_author=W Boos; citation_journal_title=J Bacteriol; citation_volume=187; citation_number=19; citation_pages=8322-8331; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Crystal structure of a CAP-DNA complex: the DNA is bent by 90 degrees.; citation_author=SC Schultz; citation_author=GC Shields; citation_author=TA Steitz; citation_journal_title=Science; citation_volume=253; citation_number=20; citation_pages=1001-1007; citation_date=1991; " />
      <meta name="citation_reference" content="citation_title=Effect of microaerophilic cell growth conditions on expression of the aerobic (cyoABCDE and cydAB) and anaerobic (narGHJI, frdABCD, and dmsABC) respiratory pathway genes in Escherichia coli.; citation_author=CP Tseng; citation_author=J Albrecht; citation_author=RP Gunsalus; citation_journal_title=J Bacteriol; citation_volume=178; citation_number=21; citation_pages=1094-1098; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Roles of NapF, NapG and NapH, subunits of the Escherichia coli periplasmic nitrate reductase, in ubiquinol oxidation.; citation_author=TH Brondijk; citation_author=D Fregen; citation_author=DJ Richardson; citation_author=JA Cole; citation_journal_title=Mol Microbiol; citation_volume=44; citation_number=22; citation_pages=245-255; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Identification of Actinobacillus pleuropneumoniae genes important for survival during infection in its natural host.; citation_author=BJ Sheehan; citation_author=JT Bosse; citation_author=AJ Beddek; citation_author=AN Rycroft; citation_author=JS Kroll; citation_journal_title=Infect Immun; citation_volume=71; citation_number=23; citation_pages=4124-4132; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Expression and functional properties of fumarate reductase.; citation_author=JJ Van Hellemond; citation_author=AG Tielens; citation_journal_title=Biochem J; citation_volume=304; citation_number=24; citation_pages=321-331; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=Comparative genomic analysis of regulation of anaerobic respiration in ten genomes from three families of gamma-proteobacteria (Enterobacteriaceae, Pasteurellaceae, Vibrionaceae).; citation_author=DA Ravcheev; citation_author=AV Gerasimova; citation_author=AA Mironov; citation_author=MS Gelfand; citation_journal_title=BMC Genomics; citation_volume=8; citation_number=25; citation_pages=54; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Sulfur metabolism in Escherichia coli and related bacteria: facts and fiction.; citation_author=A Sekowska; citation_author=HF Kung; citation_author=A Danchin; citation_journal_title=J Mol Microbiol Biotechnol; citation_volume=2; citation_number=26; citation_pages=145-177; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Functional genomics and expression analysis of the Corynebacterium glutamicum fpr2-cysIXHDNYZ gene cluster involved in assimilatory sulphate reduction.; citation_author=C Rückert; citation_author=DJ Koch; citation_author=DA Rey; citation_author=A Albersmeier; citation_author=S Mormann; citation_journal_title=BMC Genomics; citation_volume=6; citation_number=27; citation_pages=121; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Comparison of virulence of different Actinobacillus pleuropneumoniae serotypes and biotypes using an aerosol infection model.; citation_author=MJ Jacobsen; citation_author=JP Nielsen; citation_author=R Nielsen; citation_journal_title=Vet Microbiol; citation_volume=49; citation_number=28; citation_pages=159-168; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Virulence in Actinobacillus pleuropneumoniae and RTX toxins.; citation_author=J Frey; citation_journal_title=Trends Microbiol; citation_volume=3; citation_number=29; citation_pages=257-261; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Actinobacillus pleuropneumoniae RTX toxins: uniform designation of haemolysins, cytolisins, pleurotoxin and their genes.; citation_author=J Frey; citation_author=JT Bosse; citation_author=YF Chang; citation_author=JM Cullen; citation_author=B Fenwich; citation_journal_title=J Gen Microbiol; citation_volume=139; citation_number=30; citation_pages=1723-1728; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=Prevalence of serotypes, production of Apx toxins, and antibiotic resistance in strains of Actinobacillus pleuropneumoniae isolated in the Czech Republic.; citation_author=P Satran; citation_author=K Nedbalcova; citation_journal_title=Vet Med; citation_volume=47; citation_number=31; citation_pages=92-98; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Endobronchial inoculation with Apx toxins of Actinobacillus pleuropneumoniae leads to pleuropneumonia in pigs.; citation_author=EM Kamp; citation_author=N Stockhofe-Zurwieden; citation_author=LA van Leengoed; citation_author=MA Smits; citation_journal_title=Infect Immun; citation_volume=65; citation_number=32; citation_pages=4350-4354; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Genetic map of the Actinobacillus pleuropneumoniae RTX-toxin (Apx) operons: characterization of the ApxIII operons.; citation_author=R Jansen; citation_author=J Briaire; citation_author=AB van-Geel; citation_author=EM Kamp; citation_author=AL Gielkens; citation_journal_title=Infect Immun; citation_volume=62; citation_number=33; citation_pages=4411-4418; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=The tad locus: postcards from the widespread colonization island.; citation_author=M Tomich; citation_author=PJ Planet; citation_author=DH Fiqurski; citation_journal_title=Nat Rev Microbiol; citation_volume=5; citation_number=34; citation_pages=363-375; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Haemophilus ducreyi requires the flp gene cluster for microcolony formation in vitro.; citation_author=JR Nika; citation_author=JL Latimer; citation_author=CK Ward; citation_author=RJ Blick; citation_author=NJ Wagner; citation_journal_title=Infect Immun; citation_volume=70; citation_number=35; citation_pages=2965-2975; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Genes involved in the synthesis and degradation of matrix polysaccharide in Actinobacillus actinomycetemcomitans and Actinobacillus pleuropneumoniae Biofilms.; citation_author=JB Kaplan; citation_author=K Velliyagounder; citation_author=C Ragunath; citation_author=H Rohde; citation_author=D Mack; citation_journal_title=J Bacteriol; citation_volume=186; citation_number=36; citation_pages=8213-8220; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Biosynthesis of O-antigens: genes and pathways involved in nucleotide sugar precursor synthesis and O-antigen assembly.; citation_author=G Samuel; citation_author=P Reeves; citation_journal_title=Carbohydr Res; citation_volume=338; citation_number=37; citation_pages=2503-2519; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Association of Actinobacillus pleuropneumoniae capsular polysaccharide with virulence in pigs.; citation_author=AB Bandara; citation_author=ML Lawrence; citation_author=HP Veit; citation_author=TJ Inzana; citation_journal_title=Infect Immun; citation_volume=71; citation_number=38; citation_pages=3320-3328; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Actinobacillus pleuropneumoniae surface polysacchrides: their role in diagnosis and immunogenicity.; citation_author=JD Dubreuil; citation_author=M Jacques; citation_author=KR Mittal; citation_author=M Gottschalk; citation_journal_title=Anim Health Res Rev; citation_volume=1; citation_number=39; citation_pages=73-93; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Mechanisms of bacterial pathogenicity.; citation_author=JW Wilson; citation_author=MJ Schurr; citation_author=CL LeBlanc; citation_author=R Ramamurthy; citation_author=KL Buchanan; citation_journal_title=Postgrad Med J; citation_volume=78; citation_number=40; citation_pages=216-224; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Identification, cloning and characterization of rfaE of Actinobacillus pleuropneumoniae serotype 1, a gene involved in lipopolysaccharide inner-core biosynthesis.; citation_author=M Provost; citation_author=J Harel; citation_author=J Labrie; citation_author=M Sirois; citation_author=M Jacques; citation_journal_title=FEMS Microbiol Lett; citation_volume=223; citation_number=41; citation_pages=7-14; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=A gene cluster for the synthesis of serotype d-specific polysaccharide antigen in Actinobacillus actinomycetemcomitans.; citation_author=Y Nakano; citation_author=Y Yoshida; citation_author=N Suzuki; citation_author=Y Yamashita; citation_author=T Koga; citation_journal_title=Biochim Biophys Acta; citation_volume=1493; citation_number=42; citation_pages=259-263; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Genetic and Biochemical analyses of Actinobacillus pleuropneumoniae urease.; citation_author=JT Bosse; citation_author=JI MacInnes; citation_journal_title=Infect Immun; citation_volume=65; citation_number=43; citation_pages=4389-4394; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Novel genes affecting urease activity in Actinobacillus pleuropneumoniae.; citation_author=JT Bosse; citation_author=HD Glimour; citation_author=JI MacInnes; citation_journal_title=J Bacteriol; citation_volume=183; citation_number=44; citation_pages=1242-1247; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Actinobacillus pleuropneumoniae metalloprotease: cloning and in vivo expression.; citation_author=O García González; citation_author=RM García; citation_author=M de la Garza; citation_author=S Vaca; citation_author=GL Paniagua; citation_journal_title=FEMS Microbiol Lett; citation_volume=234; citation_number=45; citation_pages=81-86; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Two tonB systems in Actinobacillus pleuropneumoniae: their roles in iron acquisition and virulence.; citation_author=AJ Beddek; citation_author=BJ Sheehan; citation_author=JT Bosse; citation_author=AN Rycroft; citation_author=JS Kroll; citation_journal_title=Infect Immun; citation_volume=74; citation_number=46; citation_pages=4124-4132; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=A second tonB gene in Pseudomonas aeruginosa is linked to the exbB and exbD genes.; citation_author=Q Zhao; citation_author=K Poole; citation_journal_title=FEMS microbiol Lett; citation_volume=184; citation_number=47; citation_pages=127-132; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Molecular cloning and characterization of the ferric hydroxamate uptake (fhu) operon in Actinobacillus pleuropneumoniae.; citation_author=LG Mikael; citation_author=PD Pawelek; citation_author=J Labrie; citation_author=M Sirois; citation_author=JW Coulton; citation_journal_title=Microbiology; citation_volume=148; citation_number=48; citation_pages=2869-2882; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=The histidine of the c-type cytochrome CXXCH haem-binding motif is essential for haem attachment by the Escherichia coli cytochrome c maturation (Ccm) apparatus.; citation_author=JW Allen; citation_author=N Leach; citation_author=SJ Ferguson; citation_journal_title=Biochem J; citation_volume=389; citation_number=49; citation_pages=587-592; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Molecular cloning of haemoglobin-binding protein HgbA in the outer membrane of Actinobacillus pleuropneumoniae.; citation_author=R Srikumar; citation_author=LG Mikael; citation_author=PD Pawelek; citation_author=A Khamessan; citation_author=BF Gibbs; citation_journal_title=Microbiology; citation_volume=150; citation_number=50; citation_pages=1723-1734; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Deletion of the ferric uptake regulator Fur impairs the in vitro growth and virulence of Actinobacillus pleuropneumoniae.; citation_author=I Jacobsen; citation_author=J Gerstenberger; citation_author=AD Gruber; citation_author=JT Bossé; citation_author=PR Langford; citation_journal_title=Infect Immun; citation_volume=73; citation_number=51; citation_pages=3740-3744; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Isolation, identification and medicine sensitivity test of Actinobacillus pleuropneumoniae.; citation_author=QG He; citation_author=GP Wang; citation_author=JF Liu; citation_author=HC Chen; citation_author=HH Zhou; citation_journal_title=Chin J Prevent Vet Med; citation_volume=25; citation_number=52; citation_pages=360-363. (in Chinese); citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Development of genotyping system of Actinobacillus pleuropneumoniae and its clinical application.; citation_author=F Chen; citation_author=ML Chen; citation_author=HC Chen; citation_author=QG He; citation_author=P Blackall; citation_journal_title=Acta Microbiologica Sinica; citation_volume=44; citation_number=53; citation_pages=679-682. (in Chinese); citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=A serotype 3 specific PCR for Actinobacillus pleuropneumoniae.; citation_author=L Zhou; citation_author=SCP Jones; citation_author=O Angen; citation_author=JT Bosse; citation_author=J Nash; citation_journal_title=Vet Record; citation_volume=161; citation_number=54; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Base-calling of automated sequencer traces using phred.; citation_author=B Ewing; citation_author=P Green; citation_journal_title=Genome Res; citation_volume=8; citation_number=55; citation_pages=175-185; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=The Staden sequence analysis package.; citation_author=R Staden; citation_journal_title=Mol Biotechnol; citation_volume=5; citation_number=56; citation_pages=233-241; citation_date=1996; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001450" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Genome Biology of Actinobacillus pleuropneumoniae JL03, an Isolate of Serotype 3 Prevalent in China"/>
    <meta name="twitter:description" content="Actinobacillus pleuropneumoniae is the etiologic agent of porcine contagious pleuropneumonia, a cause of considerable world wide economic losses in the swine industry. We sequenced the complete genome of A. pleuropneumoniae, JL03, an isolate of serotype 3 prevalent in China. Its genome is a single chromosome of 2,242,062 base pairs containing 2,097 predicted protein-coding sequences, six ribosomal rRNA operons, and 63 tRNA genes. Preliminary analysis of the genomic sequence and the functions of the encoded proteins not only confirmed the present physiological and pathological knowledge but also offered new insights into the metabolic and virulence characteristics of this important pathogen. We identified a full spectrum of genes related to its characteristic chemoheterotrophic catabolism of fermentation and respiration with an incomplete TCA system for anabolism. In addition to confirming the lack of ApxI toxin, identification of a nonsense mutation in apxIVA and a 5&prime;-proximal truncation of the flp operon deleting both its promoter and the flp1flp2tadV genes have provided convincing scenarios for the low virulence property of JL03. Comparative genomic analysis using the available sequences of other serotypes, probable strain (serotype)-specific genomic islands related to capsular polysaccharides and lipopolysaccharide O-antigen biosyntheses were identified in JL03, which provides a foundation for future research into the mechanisms of serotypic diversity of A. pleuropneumoniae."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0001450.g005"/>

  <meta property="og:title" content="Genome Biology of Actinobacillus pleuropneumoniae JL03, an Isolate of Serotype 3 Prevalent in China" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001450" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=4043'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=2796'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=1653&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0001450">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001450" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001450&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Genome Biology of <i>Actinobacillus pleuropneumoniae</i> JL03, an Isolate of Serotype 3 Prevalent in China
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Zhuofei Xu
              <span class="equal-contrib"
                    title="These authors contributed equally to this work">equal contributor</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">

                <p><span class="equal-contrib" title="These authors contributed equally to this work">equal contributor</span>
                Contributed equally to this work with: Zhuofei Xu, Yan Zhou</p>

              
              

                <p>Affiliation:
                  Division of Animal Infectious Disease, State Key Laboratory of Agricultural Microbiology, College of Veterinary Medicine, Huazhong Agricultural University, Wuhan, China
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Yan Zhou
              <span class="equal-contrib"
                    title="These authors contributed equally to this work">equal contributor</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">

                <p><span class="equal-contrib" title="These authors contributed equally to this work">equal contributor</span>
                Contributed equally to this work with: Zhuofei Xu, Yan Zhou</p>

              
              

                <p>Affiliations:
                  Shanghai - MOST Key Laboratory of Health and Disease Genomics, Chinese National Human Genome Center at Shanghai, Shanghai, China, 
                  State Key Laboratory of Genetic Engineering, Department of Microbiology, School of Life Science, Fudan University, Shanghai, China
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Liangjun Li, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Division of Animal Infectious Disease, State Key Laboratory of Agricultural Microbiology, College of Veterinary Medicine, Huazhong Agricultural University, Wuhan, China
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Rui Zhou
              <span class="corresponding">mail</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:rzhou@mail.hzau.edu.cn">rzhou@mail.hzau.edu.cn</a> (RZ); <a href="mailto:gpzhao@sibs.ac.cn">gpzhao@sibs.ac.cn</a> (GZ); <a href="mailto:chenhch@mail.hzau.edu.cn">chenhch@mail.hzau.edu.cn</a> (HC)</p>

                <p>Affiliation:
                  Division of Animal Infectious Disease, State Key Laboratory of Agricultural Microbiology, College of Veterinary Medicine, Huazhong Agricultural University, Wuhan, China
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Shaobo Xiao, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Division of Animal Infectious Disease, State Key Laboratory of Agricultural Microbiology, College of Veterinary Medicine, Huazhong Agricultural University, Wuhan, China
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Yun Wan, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Division of Animal Infectious Disease, State Key Laboratory of Agricultural Microbiology, College of Veterinary Medicine, Huazhong Agricultural University, Wuhan, China
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Sihua Zhang, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Division of Animal Infectious Disease, State Key Laboratory of Agricultural Microbiology, College of Veterinary Medicine, Huazhong Agricultural University, Wuhan, China
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Kai Wang, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Shanghai - MOST Key Laboratory of Health and Disease Genomics, Chinese National Human Genome Center at Shanghai, Shanghai, China
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Wei Li, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Division of Animal Infectious Disease, State Key Laboratory of Agricultural Microbiology, College of Veterinary Medicine, Huazhong Agricultural University, Wuhan, China
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Lu Li, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Division of Animal Infectious Disease, State Key Laboratory of Agricultural Microbiology, College of Veterinary Medicine, Huazhong Agricultural University, Wuhan, China
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Hui Jin, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Division of Animal Infectious Disease, State Key Laboratory of Agricultural Microbiology, College of Veterinary Medicine, Huazhong Agricultural University, Wuhan, China
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Mingsong Kang, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Division of Animal Infectious Disease, State Key Laboratory of Agricultural Microbiology, College of Veterinary Medicine, Huazhong Agricultural University, Wuhan, China
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Baolige Dalai, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Division of Animal Infectious Disease, State Key Laboratory of Agricultural Microbiology, College of Veterinary Medicine, Huazhong Agricultural University, Wuhan, China
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Tingting Li, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Division of Animal Infectious Disease, State Key Laboratory of Agricultural Microbiology, College of Veterinary Medicine, Huazhong Agricultural University, Wuhan, China
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Lei Liu, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Division of Animal Infectious Disease, State Key Laboratory of Agricultural Microbiology, College of Veterinary Medicine, Huazhong Agricultural University, Wuhan, China
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Yangyi Cheng, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Shanghai - MOST Key Laboratory of Health and Disease Genomics, Chinese National Human Genome Center at Shanghai, Shanghai, China
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Lei Zhang, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Shanghai - MOST Key Laboratory of Health and Disease Genomics, Chinese National Human Genome Center at Shanghai, Shanghai, China
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Tao Xu, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Shanghai - MOST Key Laboratory of Health and Disease Genomics, Chinese National Human Genome Center at Shanghai, Shanghai, China
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Huajun Zheng, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Shanghai - MOST Key Laboratory of Health and Disease Genomics, Chinese National Human Genome Center at Shanghai, Shanghai, China
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Shiying Pu, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Shanghai - MOST Key Laboratory of Health and Disease Genomics, Chinese National Human Genome Center at Shanghai, Shanghai, China
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Bofei Wang, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Shanghai - MOST Key Laboratory of Health and Disease Genomics, Chinese National Human Genome Center at Shanghai, Shanghai, China
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Wenyi Gu, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Shanghai - MOST Key Laboratory of Health and Disease Genomics, Chinese National Human Genome Center at Shanghai, Shanghai, China
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Xiang-Lin Zhang, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Shanghai - MOST Key Laboratory of Health and Disease Genomics, Chinese National Human Genome Center at Shanghai, Shanghai, China
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Geng-Feng Zhu, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Shanghai - MOST Key Laboratory of Health and Disease Genomics, Chinese National Human Genome Center at Shanghai, Shanghai, China
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Shengyue Wang, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliations:
                  Shanghai - MOST Key Laboratory of Health and Disease Genomics, Chinese National Human Genome Center at Shanghai, Shanghai, China, 
                  State Key Laboratory of Genetic Engineering, Department of Microbiology, School of Life Science, Fudan University, Shanghai, China
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Guo-Ping Zhao
              <span class="corresponding">mail</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:rzhou@mail.hzau.edu.cn">rzhou@mail.hzau.edu.cn</a> (RZ); <a href="mailto:gpzhao@sibs.ac.cn">gpzhao@sibs.ac.cn</a> (GZ); <a href="mailto:chenhch@mail.hzau.edu.cn">chenhch@mail.hzau.edu.cn</a> (HC)</p>

                <p>Affiliations:
                  Shanghai - MOST Key Laboratory of Health and Disease Genomics, Chinese National Human Genome Center at Shanghai, Shanghai, China, 
                  State Key Laboratory of Genetic Engineering, Department of Microbiology, School of Life Science, Fudan University, Shanghai, China, 
                  National Engineering Center for Biochip Research at Shanghai, Shanghai, China, 
                  Laboratory of Molecular Microbiology, Institute of Plant Physiology and Ecology, Shanghai Institutes for Biological Sciences, Chinese Academy of Sciences, Shanghai, China
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Huanchun Chen
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:rzhou@mail.hzau.edu.cn">rzhou@mail.hzau.edu.cn</a> (RZ); <a href="mailto:gpzhao@sibs.ac.cn">gpzhao@sibs.ac.cn</a> (GZ); <a href="mailto:chenhch@mail.hzau.edu.cn">chenhch@mail.hzau.edu.cn</a> (HC)</p>

                <p>Affiliation:
                  Division of Animal Infectious Disease, State Key Laboratory of Agricultural Microbiology, College of Veterinary Medicine, Huazhong Agricultural University, Wuhan, China
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: January 16, 2008</li>
    <li>DOI: 10.1371/journal.pone.0001450</li>
      <li>Featured in <a href="http://www.ploscollections.org">PLOS Collections</a></li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0001450" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0001450" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001450" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0001450" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0001450-g001" data-doi="info:doi/10.1371/journal.pone.0001450" data-uri="info:doi/10.1371/journal.pone.0001450.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001450.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001450-t001" data-doi="info:doi/10.1371/journal.pone.0001450" data-uri="info:doi/10.1371/journal.pone.0001450.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001450.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001450-g002" data-doi="info:doi/10.1371/journal.pone.0001450" data-uri="info:doi/10.1371/journal.pone.0001450.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001450.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001450-t002" data-doi="info:doi/10.1371/journal.pone.0001450" data-uri="info:doi/10.1371/journal.pone.0001450.t002" title="Table 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001450.t002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001450-t003" data-doi="info:doi/10.1371/journal.pone.0001450" data-uri="info:doi/10.1371/journal.pone.0001450.t003" title="Table 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001450.t003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001450-t004" data-doi="info:doi/10.1371/journal.pone.0001450" data-uri="info:doi/10.1371/journal.pone.0001450.t004" title="Table 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001450.t004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001450-g003" data-doi="info:doi/10.1371/journal.pone.0001450" data-uri="info:doi/10.1371/journal.pone.0001450.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001450.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001450-t005" data-doi="info:doi/10.1371/journal.pone.0001450" data-uri="info:doi/10.1371/journal.pone.0001450.t005" title="Table 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001450.t005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001450-t006" data-doi="info:doi/10.1371/journal.pone.0001450" data-uri="info:doi/10.1371/journal.pone.0001450.t006" title="Table 6">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001450.t006&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001450-t007" data-doi="info:doi/10.1371/journal.pone.0001450" data-uri="info:doi/10.1371/journal.pone.0001450.t007" title="Table 7">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001450.t007&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001450-g004" data-doi="info:doi/10.1371/journal.pone.0001450" data-uri="info:doi/10.1371/journal.pone.0001450.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001450.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001450-g005" data-doi="info:doi/10.1371/journal.pone.0001450" data-uri="info:doi/10.1371/journal.pone.0001450.g005" title="Figure 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001450.g005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001450">Reader Comments (4)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0001450" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1"></a><p><em>Actinobacillus pleuropneumoniae</em> is the etiologic agent of porcine contagious pleuropneumonia, a cause of considerable world wide economic losses in the swine industry. We sequenced the complete genome of <em>A. pleuropneumoniae</em>, JL03, an isolate of serotype 3 prevalent in China. Its genome is a single chromosome of 2,242,062 base pairs containing 2,097 predicted protein-coding sequences, six ribosomal rRNA operons, and 63 tRNA genes. Preliminary analysis of the genomic sequence and the functions of the encoded proteins not only confirmed the present physiological and pathological knowledge but also offered new insights into the metabolic and virulence characteristics of this important pathogen. We identified a full spectrum of genes related to its characteristic chemoheterotrophic catabolism of fermentation and respiration with an incomplete TCA system for anabolism. In addition to confirming the lack of ApxI toxin, identification of a nonsense mutation in <em>apxIVA</em> and a 5′-proximal truncation of the <em>flp</em> operon deleting both its promoter and the <em>flp1flp2tadV</em> genes have provided convincing scenarios for the low virulence property of JL03. Comparative genomic analysis using the available sequences of other serotypes, probable strain (serotype)-specific genomic islands related to capsular polysaccharides and lipopolysaccharide O-antigen biosyntheses were identified in JL03, which provides a foundation for future research into the mechanisms of serotypic diversity of <em>A. pleuropneumoniae</em>.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>Xu Z, Zhou Y, Li L, Zhou R, Xiao S, et al.  (2008) Genome Biology of <em>Actinobacillus pleuropneumoniae</em> JL03, an Isolate of Serotype 3 Prevalent in China. PLoS ONE 3(1):
          e1450.
            doi:10.1371/journal.pone.0001450</p><p><strong>Academic Editor: </strong>Niyaz Ahmed, Centre for DNA Fingerprinting and Diagnostics, India</p><p><strong>Received:</strong> October 1, 2007; <strong>Accepted:</strong> December 19, 2007; <strong>Published:</strong> January 16, 2008</p><p><strong>Copyright:</strong> © 2008 Xu et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>This work was supported by the National Natural Science Foundation of China (30530590; 30771599), 973 Program (2006CB504402), 863 Program (2007AA02Z193; 2006AA02Z335), National Scientific and Technical Supporting Programs (2006BAD06A01; 2006BAD06A04), Cultivation Fund of the Key Scientific and Technical Innovation Project, Ministry of Education of China (706042) and Shanghai Commission for Science and Technology (05DZ22201).</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p><em>Actinobacillus pleuropneumoniae</em> is a Gram-negative, facultatively anaerobic, non-motile, rod-shaped bacillus in the family of <em>Pasteurellaceae</em>, which is chemoheterotrophic possessing both metabolic patterns of fermentation and respiration. Members of the <em>Pasteurellaceae</em> are obligate parasites, primarily of mammals and birds, while <em>A. pleuropneumoniae</em> is the etiologic agent of porcine contagious pleuropneumonia, an infectious respiratory disease of swine, which causes important world wide economic losses in the pig industry. The pathogen invades the porcine tonsil and upper respiratory tract, and can be isolated from nasal cavities, tonsils, the middle ear cavity and the lungs of infected animals <a href="#pone.0001450-Bosse1">[1]</a>, <a href="#pone.0001450-Sidib1">[2]</a>. Depending on the number of bacteria reaching the lung, the particular serotype of the infection and the immunological status of the host, the course of the disease can be divided into peracute, acute and chronic forms <a href="#pone.0001450-Bosse1">[1]</a>. Peracute and acute cases usually show high mortality with pulmonary lesions characterized by severe oedema, inflammation, haemorrhage and necrosis, whereas the chronic form of disease is characterized by haemorrhagic, fibrinous and necrotic pleuritis, pericarditis and pneumonia <a href="#pone.0001450-Bosse1">[1]</a>.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>The virulence of <em>A. pleuropneumoniae</em> is known to be associated with several factors, such as exotoxins, capsular polysaccharide (CPS), lipopolysaccharide (LPS), outer membrane proteins (OMPs), and iron uptake proteins <a href="#pone.0001450-Jacques1">[3]</a>. In addition, some enzymes involved in anaerobic respiration also appear to play an important role in the virulence of <em>A. pleuropneumoniae</em> <a href="#pone.0001450-Jacobsen1">[4]</a>.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p><em>A. pleuropneumoniae</em> has been classified into two nutritional biotypes: the biovar 1 is β-NAD-dependent while the less common biovar 2 is β-NAD-independent <a href="#pone.0001450-Jacques1">[3]</a>. On the basis of their capsular and lipopolysaccharide antigens, 15 serotypes of <em>A. pleuropneumoniae</em> have been recognized, with variations in their virulence and regional distributions <a href="#pone.0001450-Blackall1">[5]</a>. Serotypes 1, 5, and 7 are most commonly found in North America, whereas serotype 2 predominates in many European countries <a href="#pone.0001450-Jacques1">[3]</a>. In China, the prevalent serotypes are 1, 3, 4, 5 and 7 <a href="#pone.0001450-Lu1">[6]</a>.</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p>To date, 8 complete genomic sequences are available within the family of <em>Pasteurellaceae.</em> Seven of them, <em>i.e.</em>, <em>A. pleuropneumoniae</em> L20 (accession no. CP000569), <em>Pasteurella multocida</em> Pm70 (AE004439) <a href="#pone.0001450-May1">[7]</a>, <em>Haemophilus influenzae</em> Rd KW20 (L42023) <a href="#pone.0001450-Fleischmann1">[8]</a> and 86-028NP (CP000057) <a href="#pone.0001450-Harrison1">[9]</a>, <em>H. ducreyi</em> 35000HP (AE017143), <em>Mannheimia succiniciproducens</em> MBEL55E (AE016827) <a href="#pone.0001450-Hong1">[10]</a> and <em>H. somnus</em> 129PT (CP000436) <a href="#pone.0001450-Challacombe1">[11]</a>, are in the GenBank. While the complete sequence of <em>A. actinomycetemcomitans</em> is available from the web site of University of Oklahoma's Advanced Center for Genome Technology (<a href="http://www.genome.ou.edu">http://www.genome.ou.edu</a>). Among them, <em>A. pleuropneumoniae</em> L20 (serotype 5b) genomic sequence is the only one available in the <em>Actinobacillus</em> genus. In this study, we sequenced and analyzed the genome of <em>A. pleuropneumoniae</em> strain JL03, a Chinese field isolate of serotype 3. Together with the genomic sequence of L20, this information provides a firm foundation for future research into the genetic basis of metabolism, pathogenesis, virulence and serotype/biotype determination in <em>A. pleuropneumoniae</em>.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results and Discussion"></a><h3>Results and Discussion</h3>
<h4>General features of the genome</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>The genome of <em>A. pleuropneumoniae</em> strain JL03 is composed of 2,242,062 base pairs (bps) with a single circular chromosome (<a href="#pone-0001450-g001">Figure 1A</a>). Referring to genomic coordinates of strain L20, the <em>dnaA</em> gene, designated APJL0001, was selected as the first gene of the JL03 genome. The putative replication origin (<em>oriC</em>) of JL03 chromosome was identified between two genes, <em>gidA</em> (APJL1688) and <em>cof</em> (APJL1689), based on GC skew and the presence of DnaA protein recognition sequences (DnaA-boxes) <a href="#pone.0001450-Mackiewicz1">[12]</a>, <a href="#pone.0001450-Tian1">[13]</a> with typical gamma proteobacterium <em>oriC</em> features as what found in other genera of the <em>Pasteurellaceae</em> family (<a href="#pone-0001450-g001">Figure 1B</a> and <a href="#pone-0001450-t001">Table 1</a>).</p>
<div class="figure" id="pone-0001450-g001"><div class="img"><a name="pone-0001450-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001450.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001450" data-uri="info:doi/10.1371/journal.pone.0001450.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001450.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001450.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001450.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001450.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001450.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001450.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001450.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001450.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001450.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>The characterizations of <em>A. pleuropneumoniae</em> JL03's genome and the <em>oriC</em> region.</span></strong></p><a id="article1.body1.sec2.sec1.fig1.caption1.p1" name="article1.body1.sec2.sec1.fig1.caption1.p1"></a><p>(A) Circular genome representation of JL03. Circles are numbered from 1 (outer circle) to 10 (inner circle). The circles 1/2 shows predicted CDSs on the plus and minus strand in JL03 color-coded by COG categories. All genes are colored according to biological functions: gold for translation, ribosomal structure and biogenesis; orange for RNA processing and modification; light orange for transcription; dark orange for DNA replication, recombination and repair; antique white for cell division and chromosome partitioning; pink for defense mechanisms; tomato for signal transduction mechanisms; peach for cell envelope biogenesis and outer membrane; deep pink for intracellular trafficking, secretion and vesicular transport; pale green for posttranslational modification, protein turnover and chaperones; royal blue energy production and conversion; blue for carbohydrate transport and metabolism; dodger blue for amino acid transport and metabolism; sky blue for nucleotide transport and metabolism; light blue for coenzyme metabolism; cyan for lipid metabolism; medium purple for inorganic ion transport and metabolism; aquamarine for secondary metabolites biosynthesis, transport and catabolism; gray for function unknown. Circle 3/4, the putative horizontal transferred genes in deep pink identified by SIGI-HMM on the forward and reverse strand. Circle 5, repetitive elements in yellow, above 200nt and cutoff value 1e-10. Circle 6, transposases in green and potential prophage genes in dark orange. Circle 7, mean centered GC content of JL03 genes (red: above mean, blue-below mean). Circle 8, tRNA genes in orange. Circle 9, rRNA genes in red. Circle 10, GC Skew plot (windowsize: 1000, windowoverlap: 500). (B) Genetic organization of the <em>oriC</em> regions in three representative organisms within the family of <em>Pasteurellaceae</em>: JL03, <em>A. pleuropneumoniae</em>; 35000HP, <em>H. ducreyi</em>; and Pm70, <em>P. multocida</em>.</p>
<span>doi:10.1371/journal.pone.0001450.g001</span></div><div class="figure" id="pone-0001450-t001"><div class="img"><a name="pone-0001450-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001450.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001450" data-uri="info:doi/10.1371/journal.pone.0001450.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001450.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001450.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001450.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001450.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001450.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001450.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001450.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001450.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001450.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>Comparison of the nucleotides conservation in DnaA-boxes of <em>oriC</em> among closely related organisms within the family <em>Pasteurellaceae</em></span></strong></p><span>doi:10.1371/journal.pone.0001450.t001</span></div><a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2"></a><p>The JL03 genome is approximately 1.4% smaller than that of strain L20 (2,274,482 bps). The genomic comparison of the linear organization at the nucleotide level between strains JL03 and L20 is presented in <a href="#pone-0001450-g002">Figure 2A</a>. Notably, strain L20 possesses a strain-specific genomic island of 37.7 kb encoding a number of phage-related proteins, which is absent in strain JL03.</p>
<div class="figure" id="pone-0001450-g002"><div class="img"><a name="pone-0001450-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001450.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001450" data-uri="info:doi/10.1371/journal.pone.0001450.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001450.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001450.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001450.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001450.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001450.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001450.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001450.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001450.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001450.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>The schematic comparison of genetic organizations among three isolates of <em>A. pleuropneumoniae</em>. A co-linearity comparison diagram of the genomic organization at the nucleotide level between <em>A. pleuropneumoniae</em> strain JL03 and strain L20 (A).</span></strong></p><a id="article1.body1.sec2.sec1.fig2.caption1.p1" name="article1.body1.sec2.sec1.fig2.caption1.p1"></a><p> Color code stands for maximal length of those regions with highly homologous sequences between genomes: red, &gt;10 kb; blue, 5–10 kb; cyan, 1–5 kb. The boxes in green represent phage-associated CDSs of L20. Besides the strain L20-specific prophage region illustrated below the linear genomic diagram as an enlarged drawing, four special genomic regions highlighted (B, C, D, E) were magnified in the corresponding panels. The genetic organizations of the <em>cys</em> operons (B), the CPS biosynthesis and export gene clusters (D), and the LPS O-antigen biosynthesis gene clusters (E) were compared among three isolates of <em>A. pleuropneumoniae</em>: JL03 (serotype 3), L20 (serotype 5b) and 4074 (serotype 1). Comparative genetic organization of the <em>flp</em> operons between JL03 and L20 is illustrated in panel C. Regions presented in gray represent highly homologous sequences. Blue arrows represent putative CDSs with either forward or reverse transcription directions.</p>
<span>doi:10.1371/journal.pone.0001450.g002</span></div><a id="article1.body1.sec2.sec1.p3" name="article1.body1.sec2.sec1.p3"></a><p>There were eleven repetitive elements in the JL03 genome (designated JLRP1 to 11, hereafter) divided into several categories according to their coding sequences, <em>i.e.</em>, transposase, adhesin, elongation factor Tu and unknown proteins (<a href="#pone-0001450-t002">Table 2</a>). Among them, JLRP2, with its characteristic 25 bp inverted repeats in both ends, was presumed to be a novel insertion sequence element (IS) of the IS3 family. Submitted to the IS database (<a href="http://www-is.biotoul.fr">http://www-is.biotoul.fr</a>), this sequence was designated ISAp12. In addition, a noncoding 2071 bp <strong><span class="underline">c</span></strong>lustered <strong><span class="underline">r</span></strong>egularly <strong><span class="underline">i</span></strong>nterspaced <strong><span class="underline">s</span></strong>hort <strong><span class="underline">p</span></strong>alindromic <strong><span class="underline">r</span></strong>epeats region (CRISPR) was identified in the vicinity of the <em>cas1</em> gene (APJL0215) that has been found adjacent to CRISPR loci in different bacteria <a href="#pone.0001450-Jansen1">[14]</a>. This CRISPR is composed of an array of 28 bp direct repeats (DR) individually separated by 34 unique spacers of 32 bp or 33 bp. On the other hand, nine spacers in JL03's CRISPR all bear high sequence similarities with the corresponding sequences of plasmids from related bacteria (<em>A. actinomycetemcomitans</em>, <em>H. influenzae</em> and <em>H. ducreyi</em>). The inheritable feature of CRISPR spacers has been interpreted as evolutionary remnants derived from other extrachromosomal elements <a href="#pone.0001450-Bolotin1">[15]</a>, and the CRISPR loci were successfully applied to studies in evolution, typing, and comparative genomics <a href="#pone.0001450-Barrangou1">[16]</a>.</p>
<div class="figure" id="pone-0001450-t002"><div class="img"><a name="pone-0001450-t002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001450.t002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001450" data-uri="info:doi/10.1371/journal.pone.0001450.t002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001450.t002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001450.t002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001450.t002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001450.t002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001450.t002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001450.t002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001450.t002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001450.t002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001450.t002.TIF"></span>)
                </a></li></ul></div><p><strong>Table 2.  <span>List of repetitive elements in <em>A. pleuropneumoniae</em> JL03 genome</span></strong></p><span>doi:10.1371/journal.pone.0001450.t002</span></div><a id="article1.body1.sec2.sec1.p4" name="article1.body1.sec2.sec1.p4"></a><p>Annotation of the JL03 genome is summarized in <a href="#pone-0001450-t003">Table 3</a> and compared to those of strains L20 (<em>A. pleuropneumoniae</em>), Pm70 (<em>P. multocida</em>) and 35000HP (<em>H. ducreyi</em>). The entire JL03 genome has six ribosomal operons (16S-23S-5S rRNA) and an additional 5S rRNA. Sixty-three tRNA genes corresponding to the 20 common amino acids were identified in the JL03 genome. Four copies of tRNA-Ile and tRNA-Ala genes were located in the spacer regions between the 16S and 23S rRNA genes. A distinct selenocysteine tRNA gene containing the UCA anticodon was also identified. This tRNA gene is located adjacent to two genes (APJL1590, 1589) encoding L-seryl-tRNA selenium transferase (SelA) and selenocysteine-specific elongation factor (SelB), respectively. This kind of organization is the same as that found in <em>H. influenzae</em> strain 86-028NP <a href="#pone.0001450-Harrison1">[9]</a>.</p>
<div class="figure" id="pone-0001450-t003"><div class="img"><a name="pone-0001450-t003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001450.t003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001450" data-uri="info:doi/10.1371/journal.pone.0001450.t003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001450.t003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001450.t003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001450.t003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001450.t003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001450.t003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001450.t003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001450.t003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001450.t003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001450.t003.TIF"></span>)
                </a></li></ul></div><p><strong>Table 3.  <span>General features of the <em>A. pleuropneumoniae</em> JL03, L20, <em>P. multocida</em> Pm70 and <em>H. ducreyi</em> 35000HP genomes</span></strong></p><span>doi:10.1371/journal.pone.0001450.t003</span></div><a id="article1.body1.sec2.sec1.p5" name="article1.body1.sec2.sec1.p5"></a><p>The JL03 genome contained 2,097 potential CDSs with an average size of 941 bps, which in sum account for 88.1% of the whole chromosome. A graphical representation of CDSs by category and genetic characteristics of the JL03 genome are shown in <a href="#pone-0001450-g001">Figure 1A</a>. As shown in other completed microbial genomes, 18.3% of the CDSs were found to be similar to hypothetical proteins of unknown functions. The ortholog relationship between <em>A. pleuropneumoniae</em> and other species within the family <em>Pasteurellaceae</em> was consistent with their phylogenetic relationship based on the sequence analyses of 16S rRNA <a href="#pone.0001450-Dewhirst1">[17]</a> and 50 highly conserved housekeeping genes <a href="#pone.0001450-Gioia1">[18]</a>. Furthermore, protein homology comparisons demonstrated that <em>A. pleuropneumoniae</em> was closely related to <em>H. ducreyi</em> (1011 orthologous CDSs) but only distantly related to <em>H. somnus</em> (762 orthologous CDSs) (<a href="#pone-0001450-t004">Table 4</a>).</p>
<div class="figure" id="pone-0001450-t004"><div class="img"><a name="pone-0001450-t004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001450.t004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001450" data-uri="info:doi/10.1371/journal.pone.0001450.t004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001450.t004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001450.t004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001450.t004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001450.t004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001450.t004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001450.t004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001450.t004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001450.t004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001450.t004.TIF"></span>)
                </a></li></ul></div><p><strong>Table 4.  <span>Orthologs of predicted CDS of <em>A. pleuropneumoniae</em> JL03 compared with retrieved genomes</span></strong></p><span>doi:10.1371/journal.pone.0001450.t004</span></div>

<h4>Analysis of metabolism</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>A predicted set of genes encoding phosphotransferase systems (PTS) were identified in the genome of <em>A. pleuropneumoniae</em> JL03 supporting its utilization of various sugars, including mannose (<em>man</em>, APJL1410-1414), mannitol (<em>mtlADR</em>, APJL1663-1661), glucose (<em>ptsHI-crr</em>, APJL1336-1338), fructose (<em>ptsN-fruKA</em>, APJL0361-0359) and sucrose (<em>ptsB</em>, APJL1333) to generate energy <em>via</em> both fermentation and respiration (<a href="#pone-0001450-g003">Figure 3</a>). On the other hand, MalEFGK (APJL1249-1251, APJL1248) consist of an ABC (ATP-binding cassette) transport complex involved in maltose-specific transport system <a href="#pone.0001450-Dippel1">[19]</a>. Concordantly, the CAP (<em>crp</em>, APJL2012)-cAMP (<em>cyaA</em>, APJL1072) system was annotated, which generally regulates the transcriptional rate of sugar utilization operons in multiple sugar utilization bacteria <a href="#pone.0001450-Schultz1">[20]</a>.</p>
<div class="figure" id="pone-0001450-g003"><div class="img"><a name="pone-0001450-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001450.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001450" data-uri="info:doi/10.1371/journal.pone.0001450.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001450.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001450.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001450.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001450.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001450.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001450.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001450.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001450.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001450.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Overview of metabolic pathways in <em>A. pleuropneumoniae</em> JL03.</span></strong></p><a id="article1.body1.sec2.sec2.fig1.caption1.p1" name="article1.body1.sec2.sec2.fig1.caption1.p1"></a><p>Diagrammatic representation of carbon flow, electron flow, and biosynthesis of major metabolic intermediates and fatty acid are showed. Midpoint potentials (<em>E<sup>'</sup><sub>0</sub></em>) of some electron donors and acceptors of respiration chains are marked. Genes encoding crucial enzymes and functional proteins involved in the metabolic pathways are illustrated while the corresponding CDSs designated with APJL numbers are listed in <a href="#pone.0001450.s002">Table S2</a>.</p>
<span>doi:10.1371/journal.pone.0001450.g003</span></div><a id="article1.body1.sec2.sec2.p2" name="article1.body1.sec2.sec2.p2"></a><p>Besides fermentation, <em>A. pleuropneumoniae</em> performs both aerobic and anaerobic respirations and the latter is an important factor for pathogenesis (see <a href="#pone.0001450.s001">Table S1</a>). The electron transport chains in <em>A. pleuropneumoniae</em> might be branched and modular depending on its growth conditions (<a href="#pone-0001450-g003">Figure 3</a>). Cytochrome D ubiquinol oxidase encoded by <em>cydAB</em> (APJL0308, 0309) should be responsible for reducing the terminal electron acceptor oxygen to water in aerobic environments <a href="#pone.0001450-Tseng1">[21]</a>. While, genes coding for various kinds of reductases specific for terminal electron acceptors of anaerobic respiration were also identified (<a href="#pone-0001450-g003">Figure 3</a>). Besides the arsenate reductase encoded by <em>arsC</em> (APJL1105), the <em>napFDAGHBC</em> (APJL1463-1457) operon encodes a periplasmic nitrate reductase system (NAP) highly homologous to that in <em>H. ducreyi</em> <a href="#pone.0001450-Brondijk1">[22]</a>, which, as the sole nitrate reductase in <em>A. pleuropneumoniae</em>, should be essential to support anaerobic growth in the presence of nitrate <a href="#pone.0001450-Sheehan1">[23]</a>. Furthermore, albeit less favorable than nitrate, identification of <em>frdABCD</em> (APJL1556-1553) encoding a fumarate reductase and <em>dmsABC</em> (APJL1705-1707) encoding an anaerobic DMSO reductase in.JL03 inferred that this strain may be able to utilizing fumarate or dimethyl sulfoxide (DMSO) as electron acceptors as well (<a href="#pone-0001450-g003">Figure 3</a>) <a href="#pone.0001450-Jacobsen1">[4]</a>, <a href="#pone.0001450-VanHellemond1">[24]</a>.</p>
<a id="article1.body1.sec2.sec2.p3" name="article1.body1.sec2.sec2.p3"></a><p>Three global transcription regulators Hlyx (APJL0646), ArcA (APJL0049) and NarP (APJL0059) are encoded in all known genomes of <em>Pasteurellaceae,</em> including <em>A. pleuropneumoniae</em>. Under anaerobic conditions, these transcription factors may activate genes for anaerobic respiration while repress genes for aerobic respiration and fermentation <a href="#pone.0001450-Ravcheev1">[25]</a>.</p>
<a id="article1.body1.sec2.sec2.p4" name="article1.body1.sec2.sec2.p4"></a><p>Complete sets of genes coding for enzymes of glycolysis and gluconeogenesis, as well as non-oxidative pentose phosphate pathways were confirmed in strain JL03 (<a href="#pone-0001450-g003">Figure 3</a>). However, the tricarboxylic acid (TCA) cycle pathway in <em>A. pleuropneumoniae</em> was incomplete. Genes encoding three key enzymes of TCA cycle, <em>i.e.</em>, citrate synthase, aconitase and isocitrate dehydrogenase were not found in the genome. This pattern of metabolism was the same as species of genus of <em>Haemophilus</em>, e.g., <em>H. influenzae</em>, <em>H. ducreyi</em>, and <em>H. somnus</em> <a href="#pone.0001450-Challacombe1">[11]</a>. In addition, genes encoding malate synthase and isocitrate lyase, essential for glyoxylate pathway were also missing in JL03. Nevertheless, in JL03, the provision of C4 metabolites is unaffected and C5 metabolic intermediates should be offered by the non-oxidative synthesis process of the pentose phosphate pathway (<a href="#pone-0001450-g003">Figure 3</a>). In contrast to the bacteria in the genera <em>Actinobacillus</em> and <em>Haemophilus</em>, <em>P. multocida</em> and <em>M. succiniciproducens</em>, species of the genera <em>Pasteurella</em> and <em>Mannheimia</em> respectively, may perform their catabolism <em>via</em> an intact TCA cycle pathway <a href="#pone.0001450-May1">[7]</a>.</p>
<a id="article1.body1.sec2.sec2.p5" name="article1.body1.sec2.sec2.p5"></a><p>The JL03 genome encodes almost all the enzymes involved in fatty acid metabolism, biosynthesis of glycerophospholipid, terpenoid, amino acid and purine/pyrimidine nucleotides (<a href="#pone-0001450-g003">Figure 3</a>). Interestingly, a specific operon <em>cysGHDNJI</em> (APJL1886-1881) encoding several proteins involved in assimilatory sulfate reduction was identified in JL03, and this operon is incomplete in the genome of <em>A. pleuropneumoniae</em> strain L20 (<a href="#pone-0001450-g002">Figure 2B</a>), and can not be found in the genomes of other members of the family <em>Pasteurellaceae</em> except for <em>M. succiniciproducens</em> MBEL55E. The assimilatory sulfate reduction has been extensively studied in <em>Escherichia coli</em> as a model for Gram-negative bacteria <a href="#pone.0001450-Sekowska1">[26]</a>, <a href="#pone.0001450-Rckert1">[27]</a>, and the related genes, organized in a single operon <em>cysGHDNJI</em> in JL03, are dispersed into three operons in the genome of <em>E. coli</em> K12 MG1655 (U00096) <a href="#pone.0001450-Sekowska1">[26]</a>. The biological significance of the different genotypes and genomic organization deserves to further study.</p>
<a id="article1.body1.sec2.sec2.p6" name="article1.body1.sec2.sec2.p6"></a><p>JL03 needs nicotinamide adenine dinucleotide (NAD) for <em>in vitro</em> growth, but NAD is not required by <em>H. ducreyi</em> or <em>H. somnus</em> <a href="#pone.0001450-Challacombe1">[11]</a>. Concordantly, <em>H. ducreyi</em> 35000HP genome contains a duplication of the intact gene <em>nadV</em> (HD1447, 1455, 495aa) while <em>H. somnus</em> 129PT has one (HS0002, 465aa), which encodes the nicotinamide phosphoribosyltransferase (NAmPRTase) <a href="#pone.0001450-Challacombe1">[11]</a>. However, JL03 only bears a mutated <em>nadV</em>' CDS (APJL0638, 203aa) encoding merely a truncated domain. This is, for the first time, that genetic evidence was presented to support the previous notion that <em>A. pleuropneumoniae</em> serotype 3 belongs to the NAD-dependent biotype I category <a href="#pone.0001450-Jacobsen2">[28]</a>. In addition, pathways involved in ubiquinone biosynthesis as well as riboflavin and vitamin B6 metabolism were also complete in JL03, as they are in <em>H. ducreyi</em> and <em>P. multocida</em>.</p>


<h4>Analysis of pathogenesis/virulence factors</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>Pathogenesis/virulence of <em>A. pleuropneumoniae</em> has been known to be related to many specific factors in addition to its metabolic features well adapted to <em>in vivo</em> growth and <em>in vitro</em> survive described above. The genomic characteristics of the specific pathogenesis/virulence factors are described in detail below:</p>
<h5>APX exotoxins.</h5><a id="article1.body1.sec2.sec3.sec1.p1" name="article1.body1.sec2.sec3.sec1.p1"></a><p>Although the virulence of <em>A. pleuropneumoniae</em> is multifactorial, the major factor primarily responsible for the characteristic hemorrhagic lesions of the porcine contagious pleuropneumoniae are the pore-forming exotoxins belonging to the <strong><span class="underline">r</span></strong>epeat in <strong><span class="underline">t</span></strong>o<strong><span class="underline">x</span></strong>in (RTX) family <a href="#pone.0001450-Frey1">[29]</a>, <a href="#pone.0001450-Frey2">[30]</a>. Widely distributed among Gram-negative bacteria, RTX toxins share structural and functional properties, including a characteristic nonapepetide glycine-rich repeat motif, a particular mode of secretion with a signal sequence at the C-terminus, post-translational activation, and cell toxicity via pore-forming mechanism <a href="#pone.0001450-Satran1">[31]</a>. RTX toxins in <em>A. pleuropneumoniae</em> are called Apx toxins (for <em>A. pleuropneumoniae</em> RTX toxins): the strongly hemolytic and cytotoxic ApxI, the weakly hemolytic and moderately cytotoxic ApxII, the nonhemolytic but strongly cytotoxic ApxIII, and the weakly hemolytic and cytotoxic ApxIV <a href="#pone.0001450-Bosse1">[1]</a>, <a href="#pone.0001450-Frey1">[29]</a>. Different serotypes secrete different sets of Apx toxins, causing variations in both of their hemolytic and cytotoxic activities <a href="#pone.0001450-Kamp1">[32]</a>. Apx toxins are encoded by <em>apx</em> operons that usually consist of four contiguous genes arranged in the order of <em>apxCABD</em>. The <em>apxC</em> encodes a product that directs the cytoplasmic conversion by an acylation reaction of the structural toxin encoded by <em>apxA</em> to the active form, exported by a transporter encoded by <em>apxBD</em> <a href="#pone.0001450-Satran1">[31]</a>. The high degree of conservation of the RTX-B and RTX-D secretion proteins is reflected by the functional exchangeability of these proteins <a href="#pone.0001450-Jansen2">[33]</a>.</p>
<a id="article1.body1.sec2.sec3.sec1.p2" name="article1.body1.sec2.sec3.sec1.p2"></a><p>The absence of <em>apxICABD</em> operon in JL03 genome confirmed that it bore the moderate toxicity property of serotype 3, in contrast to serotypes 1, 5, 9, 10, and 11, all of which secrete ApxI <a href="#pone.0001450-Jansen2">[33]</a>. Two tightly linked gene clusters, <em>apxIICAB</em>' (APJL0968-0966) and <em>apxIIICABD</em> (APJL1347-1344) <a href="#pone.0001450-Frey2">[30]</a>, <a href="#pone.0001450-Tomich1">[34]</a>, were identified in the JL03 genome. The <em>apxII</em> operon is truncated in JL03, consisting of <em>apxIICA</em> but a partial <em>apxIIB</em>' without <em>apxIID</em>. It was evident that the secretion of ApxII may use the exporter encoded by <em>apxIBD</em> which are present in all serotypes except serotype 3, and that ApxII effect in serotype 3 is mainly cytoplasmic but barely hemolytic <a href="#pone.0001450-Frey1">[29]</a>. ApxIII has been known to be expressed and secreted by serotypes 2, 3, 4, 6, and 8 <a href="#pone.0001450-Frey1">[29]</a>.</p>
<a id="article1.body1.sec2.sec3.sec1.p3" name="article1.body1.sec2.sec3.sec1.p3"></a><p>Remarkably, ApxIVA might be impaired in JL03 due to a Trp to nonsense mutation (tgG→tgA) in the coding region of this gene (APJL1015-1016). We further sequenced four independent isolates of serotype 3 and found that besides JL03, both S1421 and HB12 had the same TGA mutation. On the other hand, neither strain GDSB nor strain HV114 is mutated, bearing the prototype Trp codon (tgG). Concerning all of the genetic determinants of Apx encoded by JL03, it is worth mentioning that serotype 3 has very low virulence and secretes little ApxII, but normal amounts of ApxIII <a href="#pone.0001450-Frey1">[29]</a>. The absence of the most important operon <em>apxI</em> is likely to be an important factor leading to a decreased virulence of JL03.</p>

<h5>Adherence.</h5><a id="article1.body1.sec2.sec3.sec2.p1" name="article1.body1.sec2.sec3.sec2.p1"></a><p>As previously reported, a 14-gene <em>flp</em> (fimbrial low-molecular-weight protein) operon (<em>flp1-flp2-tadV-rcpCAB-tadZABCDEFG</em>) has been found in the genera of <em>Haemophilus</em>, <em>Pasteurella</em>, <em>Pseudomonas, Yersinia, Caulobacter</em> and others, which is essential for Flp-pilus production, rough colony morphology, autoaggregation, and biofilm formation <a href="#pone.0001450-Tomich1">[34]</a>. However, although JL03 possesses a series of genes encoding proteins responsible for bacterial adherence to host cells and biofilm formation (<a href="#pone-0001450-t005">Table 5</a>), the <em>flp</em> operon of it is truncated, composed of only 11 genes (APJL0549-0539), where the 5′-proximal <em>flp1</em>, <em>flp2</em> and <em>tadV</em> genes found in strain L20 were absent in JL03 (<a href="#pone-0001450-g002">Figure 2C</a>). In addition, the JL03 <em>rcpC</em> (APJL0549) is truncated with only a quarter of the C-terminal CDS maintained comparing to that of L20. The remaining JL03 <em>flp</em> operon genes were highly homologous to those from <em>H. ducreyi</em> 35000HP. For instance, the TadZABCDEFG CDSs have 66%, 88%, 71%, 75%, 70%, 54%, 52%, and 49% amino acid identity comparing to those of 35000HP, respectively <a href="#pone.0001450-Nika1">[35]</a>. However, RT-PCR experiments indicated that the <em>tadZABCDEFG</em> genes were not expressed in JL03 (data not shown) and it is likely due to the truncation of the promoter region of the JL03 <em>flp</em> operon, which was identified in strain L20. Because the Flp1 protein is the major structural component of Flp pili required for adherence-related phenotypes <a href="#pone.0001450-Tomich1">[34]</a> and TadZABCDEFG is also known for tight adhesion <a href="#pone.0001450-Nika1">[35]</a>, a truncated non-expression <em>flp</em> operon of strain JL03 should lead to failure of adherence.</p>
<div class="figure" id="pone-0001450-t005"><div class="img"><a name="pone-0001450-t005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001450.t005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001450" data-uri="info:doi/10.1371/journal.pone.0001450.t005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001450.t005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001450.t005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001450.t005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001450.t005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001450.t005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001450.t005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001450.t005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001450.t005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001450.t005.TIF"></span>)
                </a></li></ul></div><p><strong>Table 5.  <span>Genes encoding proteins with a role in adherence and secretion of strain JL03</span></strong></p><span>doi:10.1371/journal.pone.0001450.t005</span></div><a id="article1.body1.sec2.sec3.sec2.p2" name="article1.body1.sec2.sec3.sec2.p2"></a><p>Polyglycolic acid (PGA), a linear polymer of N-acetyl-D-glucosamine residues in β-1,6 linkage, has been suggested to play a role in the intercellular adhesion and cellular detachment and dispersal in <em>A. actinomycetemcomitans</em> biofilm <a href="#pone.0001450-Kaplan1">[36]</a>. An operon consisted of <em>pgaABCD</em> genes (APJL1968-1971) encoding hexosamine-containing extracellular polysaccharide adhesin biosynthesis enzymes and another gene <em>dspB</em> (APJL1110) encoding N-acetyl-β-hexosaminidase were identified in the JL03 genome. The presence of these genes is consistent with the hypothesis that biofilm formation may be relevant to the colonization, pathogenesis and transmission of <em>A. pleuropneumoniae</em> <a href="#pone.0001450-Kaplan1">[36]</a>.</p>

<h5>Capsular polysaccharides (CPS).</h5><a id="article1.body1.sec2.sec3.sec3.p1" name="article1.body1.sec2.sec3.sec3.p1"></a><p>Bacterial polysaccharides are extremely diverse and occur in multiple forms, with substantial variations within a species. They include CPS, exopolysaccharides (EPS) and O-antigens <a href="#pone.0001450-Samuel1">[37]</a>. CPS is required for virulence of bacteria and variation in CPS content may contribute to the differences in virulence among <em>A. pleuropneumoniae</em> isolates <a href="#pone.0001450-Bandara1">[38]</a>. A specific genomic island-like fragment, approximately 8.6 kb, encoding genes involved in CPS biosynthesis and export was identified in JL03 (<a href="#pone-0001450-t006">Table 6</a>). BLASTn searches revealed that the CPS biosynthetic enzymes encoded by the <em>cps3ABCD</em> (APJL1614-1611) operon were serotype 3-specific in strain JL03 (<a href="#pone-0001450-g002">Figure 2D</a>). The putative proteins encoded by <em>cps3A</em> and <em>cps3D</em> both have CDP-glycerol glycerophosphotransferase motifs. Cps3B contained a cytidylyltransferase motif. This is a key regulatory enzyme for phosphatidylcholine biosynthesis. The putative Cps3D was 57% similar to the TagF protein of <em>Campylobacter jejuni</em> involved in teichoic acid biosynthesis. The proteins encoded by the genes of the <em>cps</em> operon showed low homology with those encoded by genes of different <em>A. pleuropneumoniae</em> serotypes <a href="#pone.0001450-Dubreuil1">[39]</a>.</p>
<div class="figure" id="pone-0001450-t006"><div class="img"><a name="pone-0001450-t006" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001450.t006&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001450" data-uri="info:doi/10.1371/journal.pone.0001450.t006"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001450.t006&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001450.t006/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001450.t006/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001450.t006/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001450.t006/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001450.t006.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001450.t006/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001450.t006/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001450.t006.TIF"></span>)
                </a></li></ul></div><p><strong>Table 6.  <span>Genes encoding proteins with a role in capsular polysaccharide biosynthesis of strain JL03</span></strong></p><span>doi:10.1371/journal.pone.0001450.t006</span></div><a id="article1.body1.sec2.sec3.sec3.p2" name="article1.body1.sec2.sec3.sec3.p2"></a><p>Upstream of <em>cps3A</em>, transcribed in the opposite orientation, there is another operon of four genes <em>cpxDCBA</em> (APJL1615-1618) encoding proteins involved in the export of CPS. These genes showed a high degree of homology to the group II capsule export genes <em>hexDCBA</em> in <em>P. multocida</em> strain Pm70 and <em>bexDCBA</em> in <em>H. influenzae</em> type b strain <a href="#pone.0001450-Sheehan1">[23]</a>, <a href="#pone.0001450-Dubreuil1">[39]</a>. Surprisingly, nucleotide sequences of the <em>cps</em> genes had no sequence similarity or seemingly no DNA rearrangement between JL03 of serotype 3 and L20 of serotype 5b. On the other hand, the <em>cps</em> genes seemed to be identical between L20 of serotype 5b and 4074 of serotype 1. These data suggested that the serotype diversity of <em>A. pleuropneumoniae</em> might be associated with CPS biosynthesis, but not with CPS export.</p>

<h5>Lipopolysaccharide (LPS) biosynthesis.</h5><a id="article1.body1.sec2.sec3.sec4.p1" name="article1.body1.sec2.sec3.sec4.p1"></a><p>LPSs of Gram-negative bacteria are essential structural components of the cell membrane and are considered to be virulence determinants <a href="#pone.0001450-Wilson1">[40]</a>. These substances are complex molecules composed of three well-defined biochemical moieties: the lipid A; the core oligosaccharide, which contains 3-deoxy-D-manno-oct-2-ulosonic acid (KDO) and is essential for optimal adhesion of <em>A. pleuropneumoniae</em> to the host <a href="#pone.0001450-Jacques1">[3]</a>; and the O-antigen, a polysaccharide consisting of repeating sugar units <a href="#pone.0001450-Provost1">[41]</a>.</p>
<a id="article1.body1.sec2.sec3.sec4.p2" name="article1.body1.sec2.sec3.sec4.p2"></a><p>We identified all the genes encoding enzymes for LPS biosynthesis in the JL03 genome (<a href="#pone-0001450-t007">Table 7</a>). Majority of the genes encoding enzymes for lipid A and KDO core biosynthesis were highly conserved among different species within the family <em>Pasteurellaceae</em> <a href="#pone.0001450-Challacombe1">[11]</a>, <a href="#pone.0001450-Gioia1">[18]</a>. These CDSs are scattered throughout the JL03 chromosome, like most Gram-negative prokaryotes, such as <em>E. coli</em>, <em>Neisseria meningitidis</em>, <em>Yersinia pestis</em>, <em>P. aeruginosa</em> and <em>Fusobacterium nucleatum</em>.</p>
<div class="figure" id="pone-0001450-t007"><div class="img"><a name="pone-0001450-t007" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001450.t007&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001450" data-uri="info:doi/10.1371/journal.pone.0001450.t007"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001450.t007&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001450.t007/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001450.t007/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001450.t007/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001450.t007/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001450.t007.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001450.t007/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001450.t007/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001450.t007.TIF"></span>)
                </a></li></ul></div><p><strong>Table 7.  <span>Genes encoding enzymes with a role in lipopolysaccharide metabolism of strain JL03 and orthologs present in genomes of <em>A. pleuropneumoniae</em> L20 and 4074</span></strong></p><span>doi:10.1371/journal.pone.0001450.t007</span></div><a id="article1.body1.sec2.sec3.sec4.p3" name="article1.body1.sec2.sec3.sec4.p3"></a><p>A cluster of genes coding for enzymes that catalyze the biosynthesis of O-antigen was identified in JL03 ranging from <em>wzz</em> (APJL1485) to <em>rmlB</em> (APJL1497), of which, only the dTDP-glucose 4,6-dehydratase RmlB was conserved across a wide range of species. These CDSs could be divided into three groups <a href="#pone.0001450-Samuel1">[37]</a>: nucleotide sugar biosynthesis related (APJL1496 and 1497); glycosyltransferases (APJL1486-1489 and 1493) involved in sugar transfer; and oligosaccharide repeat unit processing related, <em>wzz</em>, <em>wzy</em> and <em>wzx</em> (APJL1490 and 1491).</p>
<a id="article1.body1.sec2.sec3.sec4.p4" name="article1.body1.sec2.sec3.sec4.p4"></a><p>A bacterial sugar transferase (436aa) encoded by APJL1493 shares 55% identity with Orf9 (400aa) found in <em>A. actinomycetemcomitans</em> <a href="#pone.0001450-Nakano1">[42]</a>, but only 34% identity with a sugar transferase (472aa) encoded by APL1471 of strain L20. Two proteins (encoded by APJL1487 and 1488) among the four closely linked glycosyl transferases contain a <em>Glycos_transf_1</em> domain (PF00534) in their C-termini and a <em>Glycos_transf_2</em> domain (PF00535) in their N-termini, respectively, both unique among <em>Pasteurellaceae</em> species. Gene <em>wzz</em> encodes a protein (370aa) bearing 45% identity with the O-antigen chain length determining protein (MHA1853, 375aa) found in <em>M. haemolytica</em> <a href="#pone.0001450-Gioia1">[18]</a>. Although there are much sequence variabilities among the O-antigen-processing enzymes in different Gram-negative bacteria, structural conservation and stability of membrane spanning regions still indicate that they should perform similar function predicted by the numbers and loci of relevant transmembrane helices (TMHs) (<a href="#pone-0001450-g004">Figure 4</a>).</p>
<div class="figure" id="pone-0001450-g004"><div class="img"><a name="pone-0001450-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001450.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001450" data-uri="info:doi/10.1371/journal.pone.0001450.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001450.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001450.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001450.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001450.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001450.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001450.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001450.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001450.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001450.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>Schematic illustration of functional assignment to genes coding for enzymes for O-antigen biosynthesis based on predicting the topology of transmembrane proteins.</span></strong></p><a id="article1.body1.sec2.sec3.sec4.fig1.caption1.p1" name="article1.body1.sec2.sec3.sec4.fig1.caption1.p1"></a><p>All the amino acid sequences are from the genomes of <em>A. pleuropneumoniae</em> JL03 and <em>E. coli</em> K12 MG1655. CDSs designated with numbers and corresponding annotations are listed below: A. <em>wzy</em> (APJL1490) (<em>wbbH</em>), encoding oligosaccharide repeat unit polymerase; B. <em>wzx</em> (APJL1491) (<em>rfbX</em>), encoding O-antigen flippase; C. <em>wcaJ</em> (APJL1493), encoding glycosyltransferase; D. <em>wzz</em> (APJL1485) (<em>cld</em>), encoding an O-antigen chain length determining protein.</p>
<span>doi:10.1371/journal.pone.0001450.g004</span></div><a id="article1.body1.sec2.sec3.sec4.p5" name="article1.body1.sec2.sec3.sec4.p5"></a><p>The G+C content of the gene cluster coding for enzymes responsible for O-antigen chain biosynthesis was much lower (31%) than that of the JL03 chromosome (41%). On the other hand, genomic comparison with strains L20 or 4074 revealed that these CDSs were more variable than those for the synthesis of LPS lipid A and core oligosaccharide between serotypes (<a href="#pone-0001450-t007">Table 7</a>).</p>
<a id="article1.body1.sec2.sec3.sec4.p6" name="article1.body1.sec2.sec3.sec4.p6"></a><p>As defined by the sequences of these serotype-specific CDSs, serotype 3 should be able to be distinguished from serotype 5b and 1 as another group (<a href="#pone-0001450-g002">Figure 2E</a>). Therefore, comparison of the O-antigen regions, which were analogous with that of the <em>cps</em> operons, could also be used as one of the markers in classifying serotypes of <em>A. pleuropneumoniae</em>.</p>

<h5>Virulence related enzymes.</h5><a id="article1.body1.sec2.sec3.sec5.p1" name="article1.body1.sec2.sec3.sec5.p1"></a><p>Various enzymes, such as urease and proteases, are known to play important roles in the disease process of <em>A. pleuropneumoniae</em>. Quite a few respiratory tract pathogens produce urease, which catalyzes the hydrolysis of urea to produce ammonia and carbon dioxide <a href="#pone.0001450-Bosse2">[43]</a>. As previously reported <a href="#pone.0001450-Bosse3">[44]</a>, the gene cluster (<em>ureABC</em>, APJL1651-1649) identified in the genome of JL03 encodes the structural subunits of urease while the closely linked genes <em>ureEFGH</em> (APJL1647-1644) encode the accessory subunits. All of these genes are orthologues of those in the <em>ure</em> operon of <em>H. influenzae</em>. Furthermore, a 6-gene cluster, upstream of the <em>ure</em> operon and transcribed in the same direction, was also identified. Five of the aforementioned six genes formed a <em>cbi</em> operon (APJL1657-1652) encoding a putative nickel and cobalt periplasmic permease system, which may affect the total urease activity in <em>A. pleuropneumoniae</em> <a href="#pone.0001450-Bosse3">[44]</a>.</p>
<a id="article1.body1.sec2.sec3.sec5.p2" name="article1.body1.sec2.sec3.sec5.p2"></a><p>The <em>pepN</em> (APJL1358) encoding aminopeptidase N was identified in JL03. It was characterized on the basis of a zinc binding motif (aa 294-303) found in metalloproteases. Expression of this protease was observed in lung tissue of pigs that had died from porcine pleuropneumonia <a href="#pone.0001450-GarcaGonzlez1">[45]</a>.</p>
<a id="article1.body1.sec2.sec3.sec5.p3" name="article1.body1.sec2.sec3.sec5.p3"></a><p>Genes encoding the enzymes required for anaerobic respiration, <em>i.e.</em>, periplasmic nitrate reductase and DMSO reductase were described above. These enzymes are probably accessory virulence factors in <em>A. pleuropneumoniae</em> pathogenesis <a href="#pone.0001450-Jacobsen1">[4]</a>.</p>

<h5>Iron acquisition and utilization.</h5><a id="article1.body1.sec2.sec3.sec6.p1" name="article1.body1.sec2.sec3.sec6.p1"></a><p>Iron is essential for bacterial growth and acts as an environmental signal that regulates the expression of many virulence factors <a href="#pone.0001450-Jacques1">[3]</a>. Mammals have evolved a mechanism to reduce the availability of iron to potential bacterial pathogens by using of very-high-affinity iron-chelating molecules, while host-adapted pathogens have accordingly evolved means to use these iron-bearing molecules as an iron source <a href="#pone.0001450-Beddek1">[46]</a>. It is known that <em>A. pleuropneumoniae</em> can use porcine transferrin, hemoglobin and ferrichrome <a href="#pone.0001450-Jacques1">[3]</a>, <a href="#pone.0001450-Beddek1">[46]</a>. Approximately 2.6% (55 genes) of the JL03 genome are involved in iron uptake with additional 5 related pseudogenes likely impaired by mutations. Comparing with the genomes of other <em>Pasteurellaceae</em> members, large proportion of genes involved in iron metabolism seems common at least within the family (<a href="#pone.0001450.s003">Table S3</a>). These iron metabolism related proteins are highly conserved between JL03 and L20 except for TbpB1 (APJL1598) and FhuA (APJL2066). An analogous cell model of some iron-related protein complexes and other virulence factors is illustrated in <a href="#pone-0001450-g005">Figure 5</a>.</p>
<div class="figure" id="pone-0001450-g005"><div class="img"><a name="pone-0001450-g005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001450.g005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001450" data-uri="info:doi/10.1371/journal.pone.0001450.g005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001450.g005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001450.g005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001450.g005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001450.g005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001450.g005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001450.g005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001450.g005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001450.g005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001450.g005.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 5.  <span>A schematic diagram of virulence factors located in virtual cell environment of <em>A. pleuropneumoniae</em> JL03.</span></strong></p><a id="article1.body1.sec2.sec3.sec6.fig1.caption1.p1" name="article1.body1.sec2.sec3.sec6.fig1.caption1.p1"></a><p>CDSs corresponding to the illustrated proteins with designated APJL numbers are listed in <a href="#pone-0001450-t005">Table 5</a> (adherence and secretion relevant genes), <a href="#pone-0001450-t006">Table 6</a> (capsule polysaccharide relevant genes), <a href="#pone-0001450-t007">Table 7</a> (lipopolysaccharide relevant genes), <a href="#pone.0001450.s003">Table S3</a> (iron relevant genes) and <a href="#pone.0001450.s004">S4</a> (the rest portion relevant genes). Proteins involved in iron uptake, transport and regulation are colored in yellow.</p>
<span>doi:10.1371/journal.pone.0001450.g005</span></div><a id="article1.body1.sec2.sec3.sec6.p2" name="article1.body1.sec2.sec3.sec6.p2"></a><p>The TonB system plays a key role in iron acquisition by many Gram-negative bacteria. Two functional TonB systems were reported in detail for <em>P. aeruginosa</em> <a href="#pone.0001450-Zhao1">[47]</a> and <em>A. pleuropneumoniae</em> <a href="#pone.0001450-Beddek1">[46]</a>. Two sets of closely linked genes encoding the TonB1 and TonB2 systems, namely <em>tonB1</em>(246aa)-<em>exbB1</em>-<em>exbD1</em> (APJL1601-1599) and <em>exbB2</em>-<em>exbD2</em>-<em>tonB2</em> (244aa) (APJL0078-0076), were identified in the JL03 genome. The identity between <em>tonb1/tonb2</em> was only 15%, suggesting two structurally independent TonB systems for iron uptake. Transferrin-binding protein (Tbp), a kind of iron receptors, has been found in many species of the families <em>Pasteurellaceae</em> and <em>Neisseriaceae</em> <a href="#pone.0001450-Jacques1">[3]</a>. Two pairs of <em>tbp</em> genes were found in the JL03 genome, <em>tbpB1</em>-<em>tbpA1</em> (APJL1598-1597) and <em>tbpB2</em>-<em>tbpA2</em>′ (partial)-<em>tbpA2</em>′ (partial) (APJL0250-0252). Moreover, the nucleotide sequence identity between the two sets of genes was 54.4%, indicating that they were likely to be duplicated copies. The <em>tbpB1</em>-<em>tbpA1</em> operon was located immediately downstream of the <em>tonB1</em>. It is unclear whether these two sets of Tbps are functional. If they are both functional, it would also be interesting to learn the corresponding TonB system relevant to each set of Tbps.</p>
<a id="article1.body1.sec2.sec3.sec6.p3" name="article1.body1.sec2.sec3.sec6.p3"></a><p>The <em>fhu</em> operon encodes CDSs homologous to proteins involved in the uptake of hydroxamate siderophore across the outer membrane of several bacteria <a href="#pone.0001450-Jacques1">[3]</a>. The <em>fhuCDBA</em> (APJL2063-2066) genes encode four proteins with 28.5 kDa, 35.8 kDa, 69.4 kDa and 77.1 kDa in sequential order, in agreement with previous studies <a href="#pone.0001450-Mikael1">[48]</a>. Orthologs of <em>ccmABCDEF</em> (APJL1390-1385) were found in <em>H. ducreyi</em> and are involved in post-translational attachment of heme and catalyze the reduction of disulfide bonds in the cytochrome c apoprotein <a href="#pone.0001450-Brondijk1">[22]</a>, <a href="#pone.0001450-Allen1">[49]</a>. Both <em>ccmE</em> and <em>ccmF</em> have multiple TMHs and probable signal peptide in their N-termini. A number of genes that encode putative iron-binding receptors were found in JL03 in addition to the <em>tbp</em> genes, such as two <em>hbpA</em> genes (APJL0866 and 2060) encoding the heme-binding protein A and an outer membrane iron-receptor protein (99 kDa, APJL1922). JL03 also has the <em>hgbA</em> gene (APJL1065) encoding a haemoglobin-binding protein located at the outer membrane, which is regulated by a highly conserved gene <em>fur</em> (APJL1231) encoding ferric uptake regulator Fur <a href="#pone.0001450-Srikumar1">[50]</a>, <a href="#pone.0001450-Jacobsen3">[51]</a>. Upstream of <em>hgbA</em>, there is a CDS encoding a potential haemin-binding protein homologous to the HugZ in <em>Plesiomonas shigelloides</em> <a href="#pone.0001450-Srikumar1">[50]</a>.</p>
<a id="article1.body1.sec2.sec3.sec6.p4" name="article1.body1.sec2.sec3.sec6.p4"></a><p>Thus, a remarkably large number of genes encoding putative iron uptake proteins were found in the genome of JL03. <em>A. pleuropneumoniae</em> appears well-equipped to overcome iron shortages during infection.</p>
<a id="article1.body1.sec2.sec3.sec6.p5" name="article1.body1.sec2.sec3.sec6.p5"></a><p>In summary, we have sequenced the complete genome of <em>A. pleuropneumoniae</em> strain JL03, a Chinese field isolate of serotype 3 and annotated the genome in comparison against other members of the <em>Pasteurellaceae</em> family. A complicated metabolic network with various kinds of oxidation-reduction enzymes for catabolism and anabolism was comprehensively illustrated at the genomics level for this genus, and, for the first time, genomic discoveries were made to account for assimilatory sulfate reduction (intact operon <em>cysGHDNJI</em>) and NAD-dependent biotype I character (truncated <em>nadV</em>) of the strain (<a href="#pone-0001450-g003">Figure 3</a>). Meanwhile, we identified a series of genes encoding proteins of Apx toxins, adhesins, iron-uptake systems as well as enzymes for the biosynthesis of CPS and LPS, which underlined the genetic basis related to the pathogenesis/virulence of <em>A. pleuropneumoniae</em>. Furthermore, comparing to the genomes of strains L20 (serotype 5b) and 4074 (serotype 1) of <em>A. pleuropneumoniae</em>, probable strain (serotype)-specific genomic islands and genome reductions were identified in JL03. These data should provide a foundation for future research into the mechanisms of virulence and serotype diversity of <em>A. pleuropneumoniae</em>.</p>


</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4>Bacterial strain</h4>
<a id="article1.body1.sec3.sec1.p1" name="article1.body1.sec3.sec1.p1"></a><p>The <em>A. pleuropneumoniae</em> strain JL03 used for genomic sequencing was isolated from the lung of a pig from a Chinese commercial pig farm in 2003 <a href="#pone.0001450-He1">[52]</a> This isolate was identified as serotype 3 <a href="#pone.0001450-Chen1">[53]</a>, <a href="#pone.0001450-Zhou1">[54]</a> and was deposited in China Center for Type Culture Collection (CCTCC, Wuhan) available upon request. It grows well at 37°C on Tryptic Soy Agar or in Tryptic Soy Broth, supplemented with 10 mg/ml nicotinamide adenine dinucleotide (NAD) and 5% bovine serum.</p>


<h4>Genomic sequencing, assembly and analysis</h4>
<a id="article1.body1.sec3.sec2.p1" name="article1.body1.sec3.sec2.p1"></a><p>A whole genome shotgun strategy was adopted. Two genomic DNA libraries of JL03 with 1.5–4 kb or 6–8 kb insertion fragments were constructed in pUC18 or pSmart-LC respectively. Total of 13,440 clones (10,080 from the 1.5–4 kb pUC18 clones and 3,360 from the 6–8 kb pSmart-LC clones) were sequenced from both ends by ABI 3700 DNA analyzer, and altogether, 25,650 sequencing reads (Phred value&gt;Q20, of which 760 bp was the confirmed mean length of reads) gave an 8.6-fold coverage of the genome. Employing Phred <a href="#pone.0001450-Ewing1">[55]</a> and the Staden software package <a href="#pone.0001450-Staden1">[56]</a>, 170 contigs were assembled. Sequence and physical gaps of the unfinished genome were filled by primer walking with 226 effective PCRs. The final closure was confirmed by sequencing the PCR amplified corresponding contig-connecting fragments using the JL03 genomic DNA as the template. The finished complete genomic sequence was analyzed by conventional genomic annotation methods, which was described in detail in the <a href="#pone.0001450.s005">Text S1</a> as <em>Materials and Methods</em>.</p>

</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0001450"></div><a name="pone.0001450.s001" id="pone.0001450.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001450.s001">Table S1. </a></strong></p><a id="article1.body1.sec4.supplementary-material1.caption1.p1" name="article1.body1.sec4.supplementary-material1.caption1.p1"></a><p class="preSiDOI">Orthologs comparison of genes involved in respiration, central metabolism and corresponding regulation</p>
<p class="siDoi">doi:10.1371/journal.pone.0001450.s001</p><a id="article1.body1.sec4.supplementary-material1.caption1.p2" name="article1.body1.sec4.supplementary-material1.caption1.p2"></a><p class="postSiDOI">(0.32 MB DOC)</p>
<a name="pone.0001450.s002" id="pone.0001450.s002"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001450.s002">Table S2. </a></strong></p><a id="article1.body1.sec4.supplementary-material2.caption1.p1" name="article1.body1.sec4.supplementary-material2.caption1.p1"></a><p class="preSiDOI">Crucial enzymes and functional proteins involved in metabolic pathways of <em>A. pleuropneumoniae</em> strain JL03</p>
<p class="siDoi">doi:10.1371/journal.pone.0001450.s002</p><a id="article1.body1.sec4.supplementary-material2.caption1.p2" name="article1.body1.sec4.supplementary-material2.caption1.p2"></a><p class="postSiDOI">(0.13 MB DOC)</p>
<a name="pone.0001450.s003" id="pone.0001450.s003"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001450.s003">Table S3. </a></strong></p><a id="article1.body1.sec4.supplementary-material3.caption1.p1" name="article1.body1.sec4.supplementary-material3.caption1.p1"></a><p class="preSiDOI">Genes encoding proteins involved in iron metabolism of <em>A. pleuropneumoniae</em> JL03 compared with the homologous proteins from three representative genomes within <em>Pasteurellaceae</em></p>
<p class="siDoi">doi:10.1371/journal.pone.0001450.s003</p><a id="article1.body1.sec4.supplementary-material3.caption1.p2" name="article1.body1.sec4.supplementary-material3.caption1.p2"></a><p class="postSiDOI">(0.19 MB DOC)</p>
<a name="pone.0001450.s004" id="pone.0001450.s004"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001450.s004">Table S4. </a></strong></p><a id="article1.body1.sec4.supplementary-material4.caption1.p1" name="article1.body1.sec4.supplementary-material4.caption1.p1"></a><p class="preSiDOI">Genes encoding proteins associated with virulence factors (apx toxins, proteases, and urease) in <em>A. pleuropneumoniae</em> JL03</p>
<p class="siDoi">doi:10.1371/journal.pone.0001450.s004</p><a id="article1.body1.sec4.supplementary-material4.caption1.p2" name="article1.body1.sec4.supplementary-material4.caption1.p2"></a><p class="postSiDOI">(0.06 MB DOC)</p>
<a name="pone.0001450.s005" id="pone.0001450.s005"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001450.s005">Text S1. </a></strong></p><a id="article1.body1.sec4.supplementary-material5.caption1.p1" name="article1.body1.sec4.supplementary-material5.caption1.p1"></a><p class="preSiDOI">Supplementary Materials and Methods</p>
<p class="siDoi">doi:10.1371/journal.pone.0001450.s005</p><a id="article1.body1.sec4.supplementary-material5.caption1.p2" name="article1.body1.sec4.supplementary-material5.caption1.p2"></a><p class="postSiDOI">(0.04 MB DOC)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>We thank Dr. R. L. Somerville and Dr. P. Li for language revision.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: GZ HC RZ. Performed the experiments: XZ ZX YZ LL SX SZ LL HJ MK BD LL LZ HZ SP WG GZ WL TL. Analyzed the data: SW BW RZ ZX YZ YW KW YC TX. Wrote the paper: GZ RZ ZX.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0001450-Bosse1" id="pone.0001450-Bosse1"></a>Bosse JT, Janson H, Sheehan BJ, Beddek AJ, Rycroft AN, et al.  (2002) <em>Actinobacillus pleuropneumoniae</em>: pathobiology and pathogenesis of infection. Microbes Infect  4: 225–235.  <ul class="find" data-citedArticleID="1043028" data-doi="10.1016/s1286-4579(01)01534-9"><li><a href="http://dx.doi.org/10.1016/s1286-4579(01)01534-9" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Actinobacillus+pleuropneumoniae%3A+pathobiology+and+pathogenesis+of+infection." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Actinobacillus+pleuropneumoniae%3A+pathobiology+and+pathogenesis+of+infection.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0001450-Sidib1" id="pone.0001450-Sidib1"></a>Sidibé M, Messier S, Larivière S, Gottschalk M, Mittal KR (1993) Detection of <em>Actinobacillus pleuropneumoniae</em> in the porcine upper respiratory tract as a complement to serological tests. Can J Vet Res  57: 204–208.  <ul class="find" data-citedArticleID="1043108"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Detection+of+Actinobacillus+pleuropneumoniae+in+the+porcine+upper+respiratory+tract+as+a+complement+to+serological+tests.&amp;auth=&amp;atitle=Detection+of+Actinobacillus+pleuropneumoniae+in+the+porcine+upper+respiratory+tract+as+a+complement+to+serological+tests." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Detection+of+Actinobacillus+pleuropneumoniae+in+the+porcine+upper+respiratory+tract+as+a+complement+to+serological+tests." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Detection+of+Actinobacillus+pleuropneumoniae+in+the+porcine+upper+respiratory+tract+as+a+complement+to+serological+tests.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0001450-Jacques1" id="pone.0001450-Jacques1"></a>Jacques M (2004) Surface polysaccharides and iron-uptake systems of <em>Actinobacillus pleuropneumoniae</em>. Can J Vet Res  68: 81–85.  <ul class="find" data-citedArticleID="1043070"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Surface+polysaccharides+and+iron-uptake+systems+of+Actinobacillus+pleuropneumoniae.&amp;auth=&amp;atitle=Surface+polysaccharides+and+iron-uptake+systems+of+Actinobacillus+pleuropneumoniae." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Surface+polysaccharides+and+iron-uptake+systems+of+Actinobacillus+pleuropneumoniae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Surface+polysaccharides+and+iron-uptake+systems+of+Actinobacillus+pleuropneumoniae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0001450-Jacobsen1" id="pone.0001450-Jacobsen1"></a>Jacobsen I, Hennig-Pauka I, Baltes N, Trost M, Gerlach GF (2005) Enzymes involved in anaerobic respiration appear to play a role in <em>Actinobacillus pleuropneumoniae</em> virulence. Infect Immun  73: 226–234.  <ul class="find" data-citedArticleID="1043064" data-doi="10.1128/iai.73.1.226-234.2005"><li><a href="http://dx.doi.org/10.1128/iai.73.1.226-234.2005" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Enzymes+involved+in+anaerobic+respiration+appear+to+play+a+role+in+Actinobacillus+pleuropneumoniae+virulence." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Enzymes+involved+in+anaerobic+respiration+appear+to+play+a+role+in+Actinobacillus+pleuropneumoniae+virulence.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0001450-Blackall1" id="pone.0001450-Blackall1"></a>Blackall PJ, Klaasen HI, van den Bosch H, Kuhnert P, Frey J (2002) Proposal of a new serovar of <em>Actinobacillus pleuropneumoniae</em>: serovar 15. Vet Microbiol  84: 47–52.  <ul class="find" data-citedArticleID="1043024" data-doi="10.1016/s0378-1135(01)00428-x"><li><a href="http://dx.doi.org/10.1016/s0378-1135(01)00428-x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Proposal+of+a+new+serovar+of+Actinobacillus+pleuropneumoniae%3A+serovar+15." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Proposal+of+a+new+serovar+of+Actinobacillus+pleuropneumoniae%3A+serovar+15.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0001450-Lu1" id="pone.0001450-Lu1"></a>Lu Z, Zhao P, Shao Y, Liu J, Lu B, et al.  (2002) Study on the inactivated trivalent vaccine against swine infectious pleuropneumoniae: selection of the seed strain, preparation and safety trials of the vaccine. Chin J Vet Sci Tech  37: 33–35. (in Chinese).  <ul class="find" data-citedArticleID="1043080"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Study+on+the+inactivated+trivalent+vaccine+against+swine+infectious+pleuropneumoniae%3A+selection+of+the+seed+strain%2C+preparation+and+safety+trials+of+the+vaccine.&amp;auth=&amp;atitle=Study+on+the+inactivated+trivalent+vaccine+against+swine+infectious+pleuropneumoniae%3A+selection+of+the+seed+strain%2C+preparation+and+safety+trials+of+the+vaccine." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Study+on+the+inactivated+trivalent+vaccine+against+swine+infectious+pleuropneumoniae%3A+selection+of+the+seed+strain%2C+preparation+and+safety+trials+of+the+vaccine." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Study+on+the+inactivated+trivalent+vaccine+against+swine+infectious+pleuropneumoniae%3A+selection+of+the+seed+strain%2C+preparation+and+safety+trials+of+the+vaccine.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0001450-May1" id="pone.0001450-May1"></a>May BJ, Zhang Q, Li LL, Paustian ML, Whittam TS, et al.  (2001) Complete genomic sequence of <em>Pasteurella multocida</em>, Pm70. Proc Natl Acad Sci U S A  98: 3460–3465.  <ul class="find" data-citedArticleID="1043084" data-doi="10.1073/pnas.051634598"><li><a href="http://dx.doi.org/10.1073/pnas.051634598" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Complete+genomic+sequence+of+Pasteurella+multocida%2C+Pm70." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Complete+genomic+sequence+of+Pasteurella+multocida%2C+Pm70.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0001450-Fleischmann1" id="pone.0001450-Fleischmann1"></a>Fleischmann RD, Adams MD, White O, Clayton RA, Kirkness EF, et al.  (1995) Whole-genome random sequencing and assembly of Haemophilus influenzae Rd. Science  269: 496–512.  <ul class="find" data-citedArticleID="1043048" data-doi="10.1126/science.7542800"><li><a href="http://dx.doi.org/10.1126/science.7542800" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Whole-genome+random+sequencing+and+assembly+of+Haemophilus+influenzae+Rd." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Whole-genome+random+sequencing+and+assembly+of+Haemophilus+influenzae+Rd.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0001450-Harrison1" id="pone.0001450-Harrison1"></a>Harrison A, Dyer DW, Gillaspy A, Ray WC, Mungur R, et al.  (2005) Genomic sequence of an otitis Media isolate of nontypeable <em>Haemophilus influenzae</em>: comparative study with <em>H. influenzae</em> serotype d, strain KW20. J Bacteriol  187: 4627–4636.  <ul class="find" data-citedArticleID="1043058" data-doi="10.1128/jb.187.13.4627-4636.2005"><li><a href="http://dx.doi.org/10.1128/jb.187.13.4627-4636.2005" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genomic+sequence+of+an+otitis+Media+isolate+of+nontypeable+Haemophilus+influenzae%3A+comparative+study+with+H.+influenzae+serotype+d%2C+strain+KW20." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genomic+sequence+of+an+otitis+Media+isolate+of+nontypeable+Haemophilus+influenzae%3A+comparative+study+with+H.+influenzae+serotype+d%2C+strain+KW20.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0001450-Hong1" id="pone.0001450-Hong1"></a>Hong SH, Kim JS, Lee SY, In YH, Choi SS, et al.  (2004) The genome sequence of the capnophilic rumen bacterium <em>Mannheimia succiniciproducens</em>. Nat Biotechnol  22: 1275–1281.  <ul class="find" data-citedArticleID="1043062" data-doi="10.1038/nbt1010"><li><a href="http://dx.doi.org/10.1038/nbt1010" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+genome+sequence+of+the+capnophilic+rumen+bacterium+Mannheimia+succiniciproducens." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+genome+sequence+of+the+capnophilic+rumen+bacterium+Mannheimia+succiniciproducens.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0001450-Challacombe1" id="pone.0001450-Challacombe1"></a>Challacombe JF, Duncan AJ, Brettin TS, Bruce D, Chertkov O, et al.  (2007) Complete genome sequence of Haemophilus somnus (Histophilus somni) strain 129Pt and comparison to Haemophilus ducreyi 35000HP and Haemophilus influenzae Rd. J Bacteriol  189: 1890–1898.  <ul class="find" data-citedArticleID="1043036" data-doi="10.1128/jb.01422-06"><li><a href="http://dx.doi.org/10.1128/jb.01422-06" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Complete+genome+sequence+of+Haemophilus+somnus+%28Histophilus+somni%29+strain+129Pt+and+comparison+to+Haemophilus+ducreyi+35000HP+and+Haemophilus+influenzae+Rd." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Complete+genome+sequence+of+Haemophilus+somnus+%28Histophilus+somni%29+strain+129Pt+and+comparison+to+Haemophilus+ducreyi+35000HP+and+Haemophilus+influenzae+Rd.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0001450-Mackiewicz1" id="pone.0001450-Mackiewicz1"></a>Mackiewicz P, Zakrzewska-Czerwinska J, Zawilak A, Dudek MR, Cebrat S (2004) Where does bacterial replication start? Rules for predicting the <em>oriC</em> region. Nucleic Acids Res  25: 955–964.  <ul class="find" data-citedArticleID="1043082" data-doi="10.1093/nar/gkh699"><li><a href="http://dx.doi.org/10.1093/nar/gkh699" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Where+does+bacterial+replication+start%3F+Rules+for+predicting+the+oriC+region." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Where+does+bacterial+replication+start%3F+Rules+for+predicting+the+oriC+region.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0001450-Tian1" id="pone.0001450-Tian1"></a>Tian Y, Hao P, Zhao G, Qin Z (2005) Cloning and characterization of the chromosomal replication origin region of <em>Amycolatopsis mediterranei</em> U32. Biochem Biophys Res Commun  333: 14–20.  <ul class="find" data-citedArticleID="1043114" data-doi="10.1016/j.bbrc.2005.05.070"><li><a href="http://dx.doi.org/10.1016/j.bbrc.2005.05.070" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Cloning+and+characterization+of+the+chromosomal+replication+origin+region+of+Amycolatopsis+mediterranei+U32." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Cloning+and+characterization+of+the+chromosomal+replication+origin+region+of+Amycolatopsis+mediterranei+U32.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0001450-Jansen1" id="pone.0001450-Jansen1"></a>Jansen R, Embden JDA, van Gaastra W, Schouls LM (2002) Identification of genes that are associated with DNA repeats in prokaryotes. Mol Microbiol  43: 1565–1575.  <ul class="find" data-citedArticleID="1043072" data-doi="10.1046/j.1365-2958.2002.02839.x"><li><a href="http://dx.doi.org/10.1046/j.1365-2958.2002.02839.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Identification+of+genes+that+are+associated+with+DNA+repeats+in+prokaryotes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Identification+of+genes+that+are+associated+with+DNA+repeats+in+prokaryotes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0001450-Bolotin1" id="pone.0001450-Bolotin1"></a>Bolotin A, Quinquis B, Sorokin A, Ehrlich SD (2005) Clustered regularly interspaced short palindrome repeats (CRISPRs) have spacers of extrachromosomal origin. Microbiology  151: 2551–2561.  <ul class="find" data-citedArticleID="1043026" data-doi="10.1099/mic.0.28048-0"><li><a href="http://dx.doi.org/10.1099/mic.0.28048-0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Clustered+regularly+interspaced+short+palindrome+repeats+%28CRISPRs%29+have+spacers+of+extrachromosomal+origin." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Clustered+regularly+interspaced+short+palindrome+repeats+%28CRISPRs%29+have+spacers+of+extrachromosomal+origin.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0001450-Barrangou1" id="pone.0001450-Barrangou1"></a>Barrangou R, Fremaux C, Deveau H, Richards M, Boyaval P, et al.  (2007) CRISPR Provides Acquired Resistance Against Viruses in Prokaryotes. Science  315: 1709–1712.  <ul class="find" data-citedArticleID="1043020" data-doi="10.1126/science.1138140"><li><a href="http://dx.doi.org/10.1126/science.1138140" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=CRISPR+Provides+Acquired+Resistance+Against+Viruses+in+Prokaryotes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22CRISPR+Provides+Acquired+Resistance+Against+Viruses+in+Prokaryotes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0001450-Dewhirst1" id="pone.0001450-Dewhirst1"></a>Dewhirst FE, Paster BJ, Olsen I, Fraser GJ (1992) Phylogeny of 54 representative strains of species in the family <em>Pasteurellaceae</em> as determined by comparison of 16S rRNA sequences. J Bacteriol  174: 2002–2013.  <ul class="find" data-citedArticleID="1043040"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Phylogeny+of+54+representative+strains+of+species+in+the+family+Pasteurellaceae+as+determined+by+comparison+of+16S+rRNA+sequences.&amp;auth=&amp;atitle=Phylogeny+of+54+representative+strains+of+species+in+the+family+Pasteurellaceae+as+determined+by+comparison+of+16S+rRNA+sequences." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Phylogeny+of+54+representative+strains+of+species+in+the+family+Pasteurellaceae+as+determined+by+comparison+of+16S+rRNA+sequences." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Phylogeny+of+54+representative+strains+of+species+in+the+family+Pasteurellaceae+as+determined+by+comparison+of+16S+rRNA+sequences.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0001450-Gioia1" id="pone.0001450-Gioia1"></a>Gioia J, Qin X, Jiang H, Clinkenbeard K, Lo , et al.  (2006) The genome sequence of <em>Mannheimia haemolytica</em> A1: Insights into Virulence, Natural Competence, and <em>Pasteurellaceae</em> Phylogeny. J Bacteriol  188: 7257–7266.  <ul class="find" data-citedArticleID="1043056" data-doi="10.1128/jb.00675-06"><li><a href="http://dx.doi.org/10.1128/jb.00675-06" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+genome+sequence+of+Mannheimia+haemolytica+A1%3A+Insights+into+Virulence%2C+Natural+Competence%2C+and+Pasteurellaceae+Phylogeny." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+genome+sequence+of+Mannheimia+haemolytica+A1%3A+Insights+into+Virulence%2C+Natural+Competence%2C+and+Pasteurellaceae+Phylogeny.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0001450-Dippel1" id="pone.0001450-Dippel1"></a>Dippel R, Boos W (2005) The maltodextrin system of <em>Escherichia coli</em>: metabolism and transport. J Bacteriol  187: 8322–8331.  <ul class="find" data-citedArticleID="1043042" data-doi="10.1128/jb.187.24.8322-8331.2005"><li><a href="http://dx.doi.org/10.1128/jb.187.24.8322-8331.2005" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+maltodextrin+system+of+Escherichia+coli%3A+metabolism+and+transport." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+maltodextrin+system+of+Escherichia+coli%3A+metabolism+and+transport.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0001450-Schultz1" id="pone.0001450-Schultz1"></a>Schultz SC, Shields GC, Steitz TA (1991) Crystal structure of a CAP-DNA complex: the DNA is bent by 90 degrees. Science  253: 1001–1007.  <ul class="find" data-citedArticleID="1043102" data-doi="10.1126/science.1653449"><li><a href="http://dx.doi.org/10.1126/science.1653449" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Crystal+structure+of+a+CAP-DNA+complex%3A+the+DNA+is+bent+by+90+degrees." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Crystal+structure+of+a+CAP-DNA+complex%3A+the+DNA+is+bent+by+90+degrees.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0001450-Tseng1" id="pone.0001450-Tseng1"></a>Tseng CP, Albrecht J, Gunsalus RP (1996) Effect of microaerophilic cell growth conditions on expression of the aerobic (cyoABCDE and cydAB) and anaerobic (narGHJI, frdABCD, and dmsABC) respiratory pathway genes in <em>Escherichia coli</em>. J Bacteriol  178: 1094–1098.  <ul class="find" data-citedArticleID="1043118"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Effect+of+microaerophilic+cell+growth+conditions+on+expression+of+the+aerobic+%28cyoABCDE+and+cydAB%29+and+anaerobic+%28narGHJI%2C+frdABCD%2C+and+dmsABC%29+respiratory+pathway+genes+in+Escherichia+coli.&amp;auth=&amp;atitle=Effect+of+microaerophilic+cell+growth+conditions+on+expression+of+the+aerobic+%28cyoABCDE+and+cydAB%29+and+anaerobic+%28narGHJI%2C+frdABCD%2C+and+dmsABC%29+respiratory+pathway+genes+in+Escherichia+coli." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Effect+of+microaerophilic+cell+growth+conditions+on+expression+of+the+aerobic+%28cyoABCDE+and+cydAB%29+and+anaerobic+%28narGHJI%2C+frdABCD%2C+and+dmsABC%29+respiratory+pathway+genes+in+Escherichia+coli." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Effect+of+microaerophilic+cell+growth+conditions+on+expression+of+the+aerobic+%28cyoABCDE+and+cydAB%29+and+anaerobic+%28narGHJI%2C+frdABCD%2C+and+dmsABC%29+respiratory+pathway+genes+in+Escherichia+coli.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0001450-Brondijk1" id="pone.0001450-Brondijk1"></a>Brondijk TH, Fregen D, Richardson DJ, Cole JA (2002) Roles of NapF, NapG and NapH, subunits of the <em>Escherichia coli</em> periplasmic nitrate reductase, in ubiquinol oxidation. Mol Microbiol  44: 245–255.  <ul class="find" data-citedArticleID="1043034" data-doi="10.1046/j.1365-2958.2002.02875.x"><li><a href="http://dx.doi.org/10.1046/j.1365-2958.2002.02875.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Roles+of+NapF%2C+NapG+and+NapH%2C+subunits+of+the+Escherichia+coli+periplasmic+nitrate+reductase%2C+in+ubiquinol+oxidation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Roles+of+NapF%2C+NapG+and+NapH%2C+subunits+of+the+Escherichia+coli+periplasmic+nitrate+reductase%2C+in+ubiquinol+oxidation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0001450-Sheehan1" id="pone.0001450-Sheehan1"></a>Sheehan BJ, Bosse JT, Beddek AJ, Rycroft AN, Kroll JS, et al.  (2003) Identification of <em>Actinobacillus pleuropneumoniae</em> genes important for survival during infection in its natural host. Infect Immun  71: 4124–4132.  <ul class="find" data-citedArticleID="1043106" data-doi="10.1128/iai.71.7.3960-3970.2003"><li><a href="http://dx.doi.org/10.1128/iai.71.7.3960-3970.2003" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Identification+of+Actinobacillus+pleuropneumoniae+genes+important+for+survival+during+infection+in+its+natural+host." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Identification+of+Actinobacillus+pleuropneumoniae+genes+important+for+survival+during+infection+in+its+natural+host.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0001450-VanHellemond1" id="pone.0001450-VanHellemond1"></a>Van Hellemond JJ, Tielens AG (1994) Expression and functional properties of fumarate reductase. Biochem J  304: 321–331.  <ul class="find" data-citedArticleID="1043120"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Expression+and+functional+properties+of+fumarate+reductase.&amp;auth=&amp;atitle=Expression+and+functional+properties+of+fumarate+reductase." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Expression+and+functional+properties+of+fumarate+reductase." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Expression+and+functional+properties+of+fumarate+reductase.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0001450-Ravcheev1" id="pone.0001450-Ravcheev1"></a>Ravcheev DA, Gerasimova AV, Mironov AA, Gelfand MS (2007) Comparative genomic analysis of regulation of anaerobic respiration in ten genomes from three families of gamma-proteobacteria (<em>Enterobacteriaceae</em>, <em>Pasteurellaceae</em>, <em>Vibrionaceae</em>). BMC Genomics  8: 54.  <ul class="find" data-citedArticleID="1043094"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Comparative+genomic+analysis+of+regulation+of+anaerobic+respiration+in+ten+genomes+from+three+families+of+gamma-proteobacteria+%28Enterobacteriaceae%2C+Pasteurellaceae%2C+Vibrionaceae%29.&amp;auth=&amp;atitle=Comparative+genomic+analysis+of+regulation+of+anaerobic+respiration+in+ten+genomes+from+three+families+of+gamma-proteobacteria+%28Enterobacteriaceae%2C+Pasteurellaceae%2C+Vibrionaceae%29." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Comparative+genomic+analysis+of+regulation+of+anaerobic+respiration+in+ten+genomes+from+three+families+of+gamma-proteobacteria+%28Enterobacteriaceae%2C+Pasteurellaceae%2C+Vibrionaceae%29." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Comparative+genomic+analysis+of+regulation+of+anaerobic+respiration+in+ten+genomes+from+three+families+of+gamma-proteobacteria+%28Enterobacteriaceae%2C+Pasteurellaceae%2C+Vibrionaceae%29.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0001450-Sekowska1" id="pone.0001450-Sekowska1"></a>Sekowska A, Kung HF, Danchin A (2000) Sulfur metabolism in <em>Escherichia coli</em> and related bacteria: facts and fiction. J Mol Microbiol Biotechnol  2: 145–177.  <ul class="find" data-citedArticleID="1043104"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Sulfur+metabolism+in+Escherichia+coli+and+related+bacteria%3A+facts+and+fiction.&amp;auth=&amp;atitle=Sulfur+metabolism+in+Escherichia+coli+and+related+bacteria%3A+facts+and+fiction." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Sulfur+metabolism+in+Escherichia+coli+and+related+bacteria%3A+facts+and+fiction." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Sulfur+metabolism+in+Escherichia+coli+and+related+bacteria%3A+facts+and+fiction.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0001450-Rckert1" id="pone.0001450-Rckert1"></a>Rückert C, Koch DJ, Rey DA, Albersmeier A, Mormann S, et al.  (2005) Functional genomics and expression analysis of the <em>Corynebacterium glutamicum fpr2-cysIXHDNYZ</em> gene cluster involved in assimilatory sulphate reduction. BMC Genomics  6: 121.  <ul class="find" data-citedArticleID="1043096"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Functional+genomics+and+expression+analysis+of+the+Corynebacterium+glutamicum+fpr2-cysIXHDNYZ+gene+cluster+involved+in+assimilatory+sulphate+reduction.&amp;auth=&amp;atitle=Functional+genomics+and+expression+analysis+of+the+Corynebacterium+glutamicum+fpr2-cysIXHDNYZ+gene+cluster+involved+in+assimilatory+sulphate+reduction." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Functional+genomics+and+expression+analysis+of+the+Corynebacterium+glutamicum+fpr2-cysIXHDNYZ+gene+cluster+involved+in+assimilatory+sulphate+reduction." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Functional+genomics+and+expression+analysis+of+the+Corynebacterium+glutamicum+fpr2-cysIXHDNYZ+gene+cluster+involved+in+assimilatory+sulphate+reduction.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0001450-Jacobsen2" id="pone.0001450-Jacobsen2"></a>Jacobsen MJ, Nielsen JP, Nielsen R (1996) Comparison of virulence of different <em>Actinobacillus pleuropneumoniae</em> serotypes and biotypes using an aerosol infection model. Vet Microbiol  49: 159–168.  <ul class="find" data-citedArticleID="1043066" data-doi="10.1016/0378-1135(95)00184-0"><li><a href="http://dx.doi.org/10.1016/0378-1135(95)00184-0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Comparison+of+virulence+of+different+Actinobacillus+pleuropneumoniae+serotypes+and+biotypes+using+an+aerosol+infection+model." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Comparison+of+virulence+of+different+Actinobacillus+pleuropneumoniae+serotypes+and+biotypes+using+an+aerosol+infection+model.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0001450-Frey1" id="pone.0001450-Frey1"></a>Frey J (1995) Virulence in <em>Actinobacillus pleuropneumoniae</em> and RTX toxins. Trends Microbiol  3: 257–261.  <ul class="find" data-citedArticleID="1043050" data-doi="10.1016/s0966-842x(00)88939-8"><li><a href="http://dx.doi.org/10.1016/s0966-842x(00)88939-8" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Virulence+in+Actinobacillus+pleuropneumoniae+and+RTX+toxins." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Virulence+in+Actinobacillus+pleuropneumoniae+and+RTX+toxins.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0001450-Frey2" id="pone.0001450-Frey2"></a>Frey J, Bosse JT, Chang YF, Cullen JM, Fenwich B, et al.  (1993) <em>Actinobacillus pleuropneumoniae</em> RTX toxins: uniform designation of haemolysins, cytolisins, pleurotoxin and their genes. J Gen Microbiol  139: 1723–1728.  <ul class="find" data-citedArticleID="1043052" data-doi="10.1099/00221287-139-8-1723"><li><a href="http://dx.doi.org/10.1099/00221287-139-8-1723" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Actinobacillus+pleuropneumoniae+RTX+toxins%3A+uniform+designation+of+haemolysins%2C+cytolisins%2C+pleurotoxin+and+their+genes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Actinobacillus+pleuropneumoniae+RTX+toxins%3A+uniform+designation+of+haemolysins%2C+cytolisins%2C+pleurotoxin+and+their+genes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0001450-Satran1" id="pone.0001450-Satran1"></a>Satran P, Nedbalcova K (2002) Prevalence of serotypes, production of Apx toxins, and antibiotic resistance in strains of <em>Actinobacillus pleuropneumoniae</em> isolated in the Czech Republic. Vet Med  47: 92–98.  <ul class="find" data-citedArticleID="1043100"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Prevalence+of+serotypes%2C+production+of+Apx+toxins%2C+and+antibiotic+resistance+in+strains+of+Actinobacillus+pleuropneumoniae+isolated+in+the+Czech+Republic.&amp;auth=&amp;atitle=Prevalence+of+serotypes%2C+production+of+Apx+toxins%2C+and+antibiotic+resistance+in+strains+of+Actinobacillus+pleuropneumoniae+isolated+in+the+Czech+Republic." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Prevalence+of+serotypes%2C+production+of+Apx+toxins%2C+and+antibiotic+resistance+in+strains+of+Actinobacillus+pleuropneumoniae+isolated+in+the+Czech+Republic." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Prevalence+of+serotypes%2C+production+of+Apx+toxins%2C+and+antibiotic+resistance+in+strains+of+Actinobacillus+pleuropneumoniae+isolated+in+the+Czech+Republic.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0001450-Kamp1" id="pone.0001450-Kamp1"></a>Kamp EM, Stockhofe-Zurwieden N, van Leengoed LA, Smits MA (1997) Endobronchial inoculation with Apx toxins of <em>Actinobacillus pleuropneumoniae</em> leads to pleuropneumonia in pigs. Infect Immun  65: 4350–4354.  <ul class="find" data-citedArticleID="1043076"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Endobronchial+inoculation+with+Apx+toxins+of+Actinobacillus+pleuropneumoniae+leads+to+pleuropneumonia+in+pigs.&amp;auth=&amp;atitle=Endobronchial+inoculation+with+Apx+toxins+of+Actinobacillus+pleuropneumoniae+leads+to+pleuropneumonia+in+pigs." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Endobronchial+inoculation+with+Apx+toxins+of+Actinobacillus+pleuropneumoniae+leads+to+pleuropneumonia+in+pigs." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Endobronchial+inoculation+with+Apx+toxins+of+Actinobacillus+pleuropneumoniae+leads+to+pleuropneumonia+in+pigs.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0001450-Jansen2" id="pone.0001450-Jansen2"></a>Jansen R, Briaire J, van-Geel AB, Kamp EM, Gielkens AL, et al.  (1994) Genetic map of the <em>Actinobacillus pleuropneumoniae</em> RTX-toxin (Apx) operons: characterization of the ApxIII operons. Infect Immun  62: 4411–4418.  <ul class="find" data-citedArticleID="1043074"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Genetic+map+of+the+Actinobacillus+pleuropneumoniae+RTX-toxin+%28Apx%29+operons%3A+characterization+of+the+ApxIII+operons.&amp;auth=&amp;atitle=Genetic+map+of+the+Actinobacillus+pleuropneumoniae+RTX-toxin+%28Apx%29+operons%3A+characterization+of+the+ApxIII+operons." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genetic+map+of+the+Actinobacillus+pleuropneumoniae+RTX-toxin+%28Apx%29+operons%3A+characterization+of+the+ApxIII+operons." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genetic+map+of+the+Actinobacillus+pleuropneumoniae+RTX-toxin+%28Apx%29+operons%3A+characterization+of+the+ApxIII+operons.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0001450-Tomich1" id="pone.0001450-Tomich1"></a>Tomich M, Planet PJ, Fiqurski DH (2007) The tad locus: postcards from the widespread colonization island. Nat Rev Microbiol  5: 363–375.  <ul class="find" data-citedArticleID="1043116" data-doi="10.1038/nrmicro1636"><li><a href="http://dx.doi.org/10.1038/nrmicro1636" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+tad+locus%3A+postcards+from+the+widespread+colonization+island." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+tad+locus%3A+postcards+from+the+widespread+colonization+island.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0001450-Nika1" id="pone.0001450-Nika1"></a>Nika JR, Latimer JL, Ward CK, Blick RJ, Wagner NJ, et al.  (2002) <em>Haemophilus ducreyi</em> requires the flp gene cluster for microcolony formation in vitro. Infect Immun  70: 2965–2975.  <ul class="find" data-citedArticleID="1043090" data-doi="10.1128/iai.70.6.2965-2975.2002"><li><a href="http://dx.doi.org/10.1128/iai.70.6.2965-2975.2002" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Haemophilus+ducreyi+requires+the+flp+gene+cluster+for+microcolony+formation+in+vitro." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Haemophilus+ducreyi+requires+the+flp+gene+cluster+for+microcolony+formation+in+vitro.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">36.
              </span><a name="pone.0001450-Kaplan1" id="pone.0001450-Kaplan1"></a>Kaplan JB, Velliyagounder K, Ragunath C, Rohde H, Mack D, et al.  (2004) Genes involved in the synthesis and degradation of matrix polysaccharide in <em>Actinobacillus actinomycetemcomitans</em> and <em>Actinobacillus pleuropneumoniae</em> Biofilms. J Bacteriol  186: 8213–8220.  <ul class="find" data-citedArticleID="1043078" data-doi="10.1128/jb.186.24.8213-8220.2004"><li><a href="http://dx.doi.org/10.1128/jb.186.24.8213-8220.2004" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genes+involved+in+the+synthesis+and+degradation+of+matrix+polysaccharide+in+Actinobacillus+actinomycetemcomitans+and+Actinobacillus+pleuropneumoniae+Biofilms." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genes+involved+in+the+synthesis+and+degradation+of+matrix+polysaccharide+in+Actinobacillus+actinomycetemcomitans+and+Actinobacillus+pleuropneumoniae+Biofilms.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">37.
              </span><a name="pone.0001450-Samuel1" id="pone.0001450-Samuel1"></a>Samuel G, Reeves P (2003) Biosynthesis of O-antigens: genes and pathways involved in nucleotide sugar precursor synthesis and O-antigen assembly. Carbohydr Res  338: 2503–2519.  <ul class="find" data-citedArticleID="1043098" data-doi="10.1016/j.carres.2003.07.009"><li><a href="http://dx.doi.org/10.1016/j.carres.2003.07.009" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Biosynthesis+of+O-antigens%3A+genes+and+pathways+involved+in+nucleotide+sugar+precursor+synthesis+and+O-antigen+assembly." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Biosynthesis+of+O-antigens%3A+genes+and+pathways+involved+in+nucleotide+sugar+precursor+synthesis+and+O-antigen+assembly.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">38.
              </span><a name="pone.0001450-Bandara1" id="pone.0001450-Bandara1"></a>Bandara AB, Lawrence ML, Veit HP, Inzana TJ (2003) Association of <em>Actinobacillus pleuropneumoniae</em> capsular polysaccharide with virulence in pigs. Infect Immun  71: 3320–3328.  <ul class="find" data-citedArticleID="1043018" data-doi="10.1128/iai.71.6.3320-3328.2003"><li><a href="http://dx.doi.org/10.1128/iai.71.6.3320-3328.2003" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Association+of+Actinobacillus+pleuropneumoniae+capsular+polysaccharide+with+virulence+in+pigs." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Association+of+Actinobacillus+pleuropneumoniae+capsular+polysaccharide+with+virulence+in+pigs.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">39.
              </span><a name="pone.0001450-Dubreuil1" id="pone.0001450-Dubreuil1"></a>Dubreuil JD, Jacques M, Mittal KR, Gottschalk M (2000) <em>Actinobacillus pleuropneumoniae</em> surface polysacchrides: their role in diagnosis and immunogenicity. Anim Health Res Rev  1: 73–93.  <ul class="find" data-citedArticleID="1043044" data-doi="10.1017/s1466252300000074"><li><a href="http://dx.doi.org/10.1017/s1466252300000074" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Actinobacillus+pleuropneumoniae+surface+polysacchrides%3A+their+role+in+diagnosis+and+immunogenicity." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Actinobacillus+pleuropneumoniae+surface+polysacchrides%3A+their+role+in+diagnosis+and+immunogenicity.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">40.
              </span><a name="pone.0001450-Wilson1" id="pone.0001450-Wilson1"></a>Wilson JW, Schurr MJ, LeBlanc CL, Ramamurthy R, Buchanan KL, et al.  (2002) Mechanisms of bacterial pathogenicity. Postgrad Med J  78: 216–224.  <ul class="find" data-citedArticleID="1043122"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Mechanisms+of+bacterial+pathogenicity.&amp;auth=&amp;atitle=Mechanisms+of+bacterial+pathogenicity." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mechanisms+of+bacterial+pathogenicity." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mechanisms+of+bacterial+pathogenicity.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">41.
              </span><a name="pone.0001450-Provost1" id="pone.0001450-Provost1"></a>Provost M, Harel J, Labrie J, Sirois M, Jacques M (2003) Identification, cloning and characterization of rfaE of <em>Actinobacillus pleuropneumoniae</em> serotype 1, a gene involved in lipopolysaccharide inner-core biosynthesis. FEMS Microbiol Lett  223: 7–14.  <ul class="find" data-citedArticleID="1043092" data-doi="10.1016/s0378-1097(03)00247-7"><li><a href="http://dx.doi.org/10.1016/s0378-1097(03)00247-7" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Identification%2C+cloning+and+characterization+of+rfaE+of+Actinobacillus+pleuropneumoniae+serotype+1%2C+a+gene+involved+in+lipopolysaccharide+inner-core+biosynthesis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Identification%2C+cloning+and+characterization+of+rfaE+of+Actinobacillus+pleuropneumoniae+serotype+1%2C+a+gene+involved+in+lipopolysaccharide+inner-core+biosynthesis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">42.
              </span><a name="pone.0001450-Nakano1" id="pone.0001450-Nakano1"></a>Nakano Y, Yoshida Y, Suzuki N, Yamashita Y, Koga T (2000) A gene cluster for the synthesis of serotype d-specific polysaccharide antigen in <em>Actinobacillus actinomycetemcomitans</em>. Biochim Biophys Acta  1493: 259–263.  <ul class="find" data-citedArticleID="1043088" data-doi="10.1016/s0167-4781(00)00179-2"><li><a href="http://dx.doi.org/10.1016/s0167-4781(00)00179-2" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+gene+cluster+for+the+synthesis+of+serotype+d-specific+polysaccharide+antigen+in+Actinobacillus+actinomycetemcomitans." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+gene+cluster+for+the+synthesis+of+serotype+d-specific+polysaccharide+antigen+in+Actinobacillus+actinomycetemcomitans.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">43.
              </span><a name="pone.0001450-Bosse2" id="pone.0001450-Bosse2"></a>Bosse JT, MacInnes JI (1997) Genetic and Biochemical analyses of <em>Actinobacillus pleuropneumoniae</em> urease. Infect Immun  65: 4389–4394.  <ul class="find" data-citedArticleID="1043030"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Genetic+and+Biochemical+analyses+of+Actinobacillus+pleuropneumoniae+urease.&amp;auth=&amp;atitle=Genetic+and+Biochemical+analyses+of+Actinobacillus+pleuropneumoniae+urease." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genetic+and+Biochemical+analyses+of+Actinobacillus+pleuropneumoniae+urease." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genetic+and+Biochemical+analyses+of+Actinobacillus+pleuropneumoniae+urease.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">44.
              </span><a name="pone.0001450-Bosse3" id="pone.0001450-Bosse3"></a>Bosse JT, Glimour HD, MacInnes JI (2001) Novel genes affecting urease activity in <em>Actinobacillus pleuropneumoniae</em>. J Bacteriol  183: 1242–1247.  <ul class="find" data-citedArticleID="1043032" data-doi="10.1128/jb.183.4.1242-1247.2001"><li><a href="http://dx.doi.org/10.1128/jb.183.4.1242-1247.2001" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Novel+genes+affecting+urease+activity+in+Actinobacillus+pleuropneumoniae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Novel+genes+affecting+urease+activity+in+Actinobacillus+pleuropneumoniae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">45.
              </span><a name="pone.0001450-GarcaGonzlez1" id="pone.0001450-GarcaGonzlez1"></a>García González O, García RM, de la Garza M, Vaca S, Paniagua GL, et al.  (2004) <em>Actinobacillus pleuropneumoniae</em> metalloprotease: cloning and in vivo expression. FEMS Microbiol Lett  234: 81–86.  <ul class="find" data-citedArticleID="1043054" data-doi="10.1111/j.1574-6968.2004.tb09516.x"><li><a href="http://dx.doi.org/10.1111/j.1574-6968.2004.tb09516.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Actinobacillus+pleuropneumoniae+metalloprotease%3A+cloning+and+in+vivo+expression." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Actinobacillus+pleuropneumoniae+metalloprotease%3A+cloning+and+in+vivo+expression.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">46.
              </span><a name="pone.0001450-Beddek1" id="pone.0001450-Beddek1"></a>Beddek AJ, Sheehan BJ, Bosse JT, Rycroft AN, Kroll JS, et al.  (2004) Two tonB systems in <em>Actinobacillus pleuropneumoniae</em>: their roles in iron acquisition and virulence. Infect Immun  74: 4124–4132.  <ul class="find" data-citedArticleID="1043022" data-doi="10.1128/iai.72.2.701-708.2004"><li><a href="http://dx.doi.org/10.1128/iai.72.2.701-708.2004" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Two+tonB+systems+in+Actinobacillus+pleuropneumoniae%3A+their+roles+in+iron+acquisition+and+virulence." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Two+tonB+systems+in+Actinobacillus+pleuropneumoniae%3A+their+roles+in+iron+acquisition+and+virulence.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">47.
              </span><a name="pone.0001450-Zhao1" id="pone.0001450-Zhao1"></a>Zhao Q, Poole K (2000) A second tonB gene in <em>Pseudomonas aeruginosa</em> is linked to the exbB and exbD genes. FEMS microbiol Lett  184: 127–132.  <ul class="find" data-citedArticleID="1043124" data-doi="10.1111/j.1574-6968.2000.tb09002.x"><li><a href="http://dx.doi.org/10.1111/j.1574-6968.2000.tb09002.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+second+tonB+gene+in+Pseudomonas+aeruginosa+is+linked+to+the+exbB+and+exbD+genes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+second+tonB+gene+in+Pseudomonas+aeruginosa+is+linked+to+the+exbB+and+exbD+genes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">48.
              </span><a name="pone.0001450-Mikael1" id="pone.0001450-Mikael1"></a>Mikael LG, Pawelek PD, Labrie J, Sirois M, Coulton JW, et al.  (2002) Molecular cloning and characterization of the ferric hydroxamate uptake (fhu) operon in <em>Actinobacillus pleuropneumoniae</em>. Microbiology  148: 2869–2882.  <ul class="find" data-citedArticleID="1043086"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Molecular+cloning+and+characterization+of+the+ferric+hydroxamate+uptake+%28fhu%29+operon+in+Actinobacillus+pleuropneumoniae.&amp;auth=&amp;atitle=Molecular+cloning+and+characterization+of+the+ferric+hydroxamate+uptake+%28fhu%29+operon+in+Actinobacillus+pleuropneumoniae." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Molecular+cloning+and+characterization+of+the+ferric+hydroxamate+uptake+%28fhu%29+operon+in+Actinobacillus+pleuropneumoniae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Molecular+cloning+and+characterization+of+the+ferric+hydroxamate+uptake+%28fhu%29+operon+in+Actinobacillus+pleuropneumoniae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">49.
              </span><a name="pone.0001450-Allen1" id="pone.0001450-Allen1"></a>Allen JW, Leach N, Ferguson SJ (2005) The histidine of the c-type cytochrome CXXCH haem-binding motif is essential for haem attachment by the <em>Escherichia coli</em> cytochrome c maturation (Ccm) apparatus. Biochem J  389: 587–592.  <ul class="find" data-citedArticleID="1043016" data-doi="10.1042/bj20041894"><li><a href="http://dx.doi.org/10.1042/bj20041894" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+histidine+of+the+c-type+cytochrome+CXXCH+haem-binding+motif+is+essential+for+haem+attachment+by+the+Escherichia+coli+cytochrome+c+maturation+%28Ccm%29+apparatus." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+histidine+of+the+c-type+cytochrome+CXXCH+haem-binding+motif+is+essential+for+haem+attachment+by+the+Escherichia+coli+cytochrome+c+maturation+%28Ccm%29+apparatus.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">50.
              </span><a name="pone.0001450-Srikumar1" id="pone.0001450-Srikumar1"></a>Srikumar R, Mikael LG, Pawelek PD, Khamessan A, Gibbs BF, et al.  (2004) Molecular cloning of haemoglobin-binding protein HgbA in the outer membrane of <em>Actinobacillus pleuropneumoniae</em>. Microbiology  150: 1723–1734.  <ul class="find" data-citedArticleID="1043110" data-doi="10.1099/mic.0.27046-0"><li><a href="http://dx.doi.org/10.1099/mic.0.27046-0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Molecular+cloning+of+haemoglobin-binding+protein+HgbA+in+the+outer+membrane+of+Actinobacillus+pleuropneumoniae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Molecular+cloning+of+haemoglobin-binding+protein+HgbA+in+the+outer+membrane+of+Actinobacillus+pleuropneumoniae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">51.
              </span><a name="pone.0001450-Jacobsen3" id="pone.0001450-Jacobsen3"></a>Jacobsen I, Gerstenberger J, Gruber AD, Bossé JT, Langford PR, et al.  (2005) Deletion of the ferric uptake regulator Fur impairs the in vitro growth and virulence of <em>Actinobacillus pleuropneumoniae</em>. Infect Immun  73: 3740–3744.  <ul class="find" data-citedArticleID="1043068" data-doi="10.1128/iai.73.6.3740-3744.2005"><li><a href="http://dx.doi.org/10.1128/iai.73.6.3740-3744.2005" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Deletion+of+the+ferric+uptake+regulator+Fur+impairs+the+in+vitro+growth+and+virulence+of+Actinobacillus+pleuropneumoniae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Deletion+of+the+ferric+uptake+regulator+Fur+impairs+the+in+vitro+growth+and+virulence+of+Actinobacillus+pleuropneumoniae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">52.
              </span><a name="pone.0001450-He1" id="pone.0001450-He1"></a>He QG, Wang GP, Liu JF, Chen HC, Zhou HH, et al.  (2003) Isolation, identification and medicine sensitivity test of <em>Actinobacillus pleuropneumonia</em>e. Chin J Prevent Vet Med  25: 360–363. (in Chinese).  <ul class="find" data-citedArticleID="1043060"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Isolation%2C+identification+and+medicine+sensitivity+test+of+Actinobacillus+pleuropneumoniae.&amp;auth=&amp;atitle=Isolation%2C+identification+and+medicine+sensitivity+test+of+Actinobacillus+pleuropneumoniae." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Isolation%2C+identification+and+medicine+sensitivity+test+of+Actinobacillus+pleuropneumoniae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Isolation%2C+identification+and+medicine+sensitivity+test+of+Actinobacillus+pleuropneumoniae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">53.
              </span><a name="pone.0001450-Chen1" id="pone.0001450-Chen1"></a>Chen F, Chen ML, Chen HC, He QG, Blackall P (2004) Development of genotyping system of <em>Actinobacillus pleuropneumoniae</em> and its clinical application. Acta Microbiologica Sinica  44: 679–682. (in Chinese).  <ul class="find" data-citedArticleID="1043038"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Development+of+genotyping+system+of+Actinobacillus+pleuropneumoniae+and+its+clinical+application.&amp;auth=&amp;atitle=Development+of+genotyping+system+of+Actinobacillus+pleuropneumoniae+and+its+clinical+application." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Development+of+genotyping+system+of+Actinobacillus+pleuropneumoniae+and+its+clinical+application." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Development+of+genotyping+system+of+Actinobacillus+pleuropneumoniae+and+its+clinical+application.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">54.
              </span><a name="pone.0001450-Zhou1" id="pone.0001450-Zhou1"></a>Zhou L, Jones SCP, Angen O, Bosse JT, Nash J, et al.  (2007) A serotype 3 specific PCR for <em>Actinobacillus pleuropneumoniae</em>. Vet Record  161:   in press. <ul class="find" data-citedArticleID="1043126" data-doi="10.1136/vr.162.20.648"><li><a href="http://dx.doi.org/10.1136/vr.162.20.648" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+serotype+3+specific+PCR+for+Actinobacillus+pleuropneumoniae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+serotype+3+specific+PCR+for+Actinobacillus+pleuropneumoniae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">55.
              </span><a name="pone.0001450-Ewing1" id="pone.0001450-Ewing1"></a>Ewing B, Green P (1998) Base-calling of automated sequencer traces using phred. Genome Res  8: 175–185.  <ul class="find" data-citedArticleID="1043046" data-doi="10.1101/gr.8.3.175"><li><a href="http://dx.doi.org/10.1101/gr.8.3.175" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Base-calling+of+automated+sequencer+traces+using+phred." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Base-calling+of+automated+sequencer+traces+using+phred.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">56.
              </span><a name="pone.0001450-Staden1" id="pone.0001450-Staden1"></a>Staden R (1996) The Staden sequence analysis package. Mol Biotechnol  5: 233–241.  <ul class="find" data-citedArticleID="1043112" data-doi="10.1007/bf02900361"><li><a href="http://dx.doi.org/10.1007/bf02900361" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+Staden+sequence+analysis+package." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+Staden+sequence+analysis+package.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001450.XML" value="156790"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001450.PDF" value="601939"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001450.g001.PNG_L" value="2481909"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001450.g001.PNG_M" value="231075"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001450.g001.PNG_S" value="15213"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001450.g001.TIF" value="3475544"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001450.g001.PNG_I" value="107635"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001450.t001.PNG_L" value="68810"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001450.t001.PNG_M" value="40953"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001450.t001.PNG_S" value="8631"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001450.t001.TIF" value="317046"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001450.t001.PNG_I" value="12538"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001450.g002.PNG_L" value="5557391"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001450.g002.PNG_M" value="198360"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001450.g002.PNG_S" value="15400"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001450.g002.TIF" value="7109764"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001450.g002.PNG_I" value="109094"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001450.t002.PNG_L" value="110766"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001450.t002.PNG_M" value="63500"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001450.t002.PNG_S" value="10031"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001450.t002.TIF" value="542992"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001450.t002.PNG_I" value="19087"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001450.t003.PNG_L" value="106669"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001450.t003.PNG_M" value="146200"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001450.t003.PNG_S" value="19468"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001450.t003.TIF" value="489484"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001450.t003.PNG_I" value="49466"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001450.t004.PNG_L" value="64666"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001450.t004.PNG_M" value="84037"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001450.t004.PNG_S" value="12647"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001450.t004.TIF" value="261490"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001450.t004.PNG_I" value="28262"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001450.g003.PNG_L" value="2573066"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001450.g003.PNG_M" value="181224"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001450.g003.PNG_S" value="10066"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001450.g003.TIF" value="3731272"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001450.g003.PNG_I" value="59754"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001450.t005.PNG_L" value="266133"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001450.t005.PNG_M" value="152385"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001450.t005.PNG_S" value="22540"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001450.t005.TIF" value="1185106"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001450.t005.PNG_I" value="175534"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001450.t006.PNG_L" value="103631"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001450.t006.PNG_M" value="146783"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001450.t006.PNG_S" value="18034"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001450.t006.TIF" value="442566"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001450.t006.PNG_I" value="47891"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001450.t007.PNG_L" value="347008"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001450.t007.PNG_M" value="143429"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001450.t007.PNG_S" value="26338"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001450.t007.TIF" value="1575068"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001450.t007.PNG_I" value="240949"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001450.g004.PNG_L" value="550658"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001450.g004.PNG_M" value="113097"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001450.g004.PNG_S" value="12223"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001450.g004.TIF" value="3138338"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001450.g004.PNG_I" value="45947"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001450.g005.PNG_L" value="1463693"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001450.g005.PNG_M" value="157752"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001450.g005.PNG_S" value="14435"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001450.g005.TIF" value="2822040"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001450.g005.PNG_I" value="53242"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001450.s001.DOC" value="322048"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001450.s002.DOC" value="125952"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001450.s003.DOC" value="187904"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001450.s004.DOC" value="59904"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001450.s005.DOC" value="35328"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001450&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001450" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001450&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0001450&volume=&issue=&title=Genome Biology of Actinobacillus pleuropneumoniae JL03, an Isolate of Serotype 3 Prevalent in China&author_name=Zhuofei%20Xu%2C%20Yan%20Zhou%2C%20Liangjun%20Li%2C%20Rui%20Zhou%2C%20Shaobo%20Xiao%2C%20Yun%20Wan%2C%20Sihua%20Zhang%2C%20Kai%20Wang%2C%20Wei%20Li%2C%20Lu%20Li%2C%20Hui%20Jin%2C%20Mingsong%20Kang%2C%20Baolige%20Dalai%2C%20Tingting%20Li%2C%20Lei%20Liu%2C%20Yangyi%20Cheng%2C%20Lei%20Zhang%2C%20Tao%20Xu%2C%20Huajun%20Zheng%2C%20Shiying%20Pu%2C%20Bofei%20Wang%2C%20Wenyi%20Gu%2C%20Xiang-Lin%20Zhang%2C%20Geng-Feng%20Zhu%2C%20Shengyue%20Wang%2C%20Guo-Ping%20Zhao%2C%20Huanchun%20Chen&start_page=1&end_page=12" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001450" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001450" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0001450" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001450&amp;t=Genome%20Biology%20of%20Actinobacillus%20pleuropneumoniae%20JL03%2C%20an%20Isolate%20of%20Serotype%203%20Prevalent%20in%20China" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001450&title=Genome%20Biology%20of%20Actinobacillus%20pleuropneumoniae%20JL03%2C%20an%20Isolate%20of%20Serotype%203%20Prevalent%20in%20China&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001450&amp;title=Genome%20Biology%20of%20Actinobacillus%20pleuropneumoniae%20JL03%2C%20an%20Isolate%20of%20Serotype%203%20Prevalent%20in%20China" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001450" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0001450&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Genome Biology of Actinobacillus pleuropneumoniae JL03, an Isolate of Serotype 3 Prevalent in China';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0001450';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Genome%20Biology%20of%20Actinobacillus%20pleuropneumoniae%20JL03%2C%20an%20Isolate%20of%20Serotype%203%20Prevalent%20in%20China http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001450" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0001450" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->

<div class="block">
    <div class="header">
        <h3>Included in the Following Collection</h3>
    </div>
    <p><dd><a href="http://www.ploscollections.org/article/browse/issue/info%3Adoi%2F10.1371%2Fissue.pcol.v02.i03" title="Browse the Open-Access Collection"><em>PLoS ONE</em>: Prokaryotic Genome Collection</a></dd></p>
</div>

<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">












          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Bacterial+genomics%22" title="Search for articles in the subject area:'Bacterial genomics'"><div class="flagText">Bacterial genomics</div></a>
              <div data-categoryid="32801" data-articleid="25728"
                   data-categoryname="Bacterial genomics"
                   class="flagImage" title="Flag 'Bacterial genomics' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Biosynthesis%22" title="Search for articles in the subject area:'Biosynthesis'"><div class="flagText">Biosynthesis</div></a>
              <div data-categoryid="40369" data-articleid="25728"
                   data-categoryname="Biosynthesis"
                   class="flagImage" title="Flag 'Biosynthesis' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Comparative+genomics%22" title="Search for articles in the subject area:'Comparative genomics'"><div class="flagText">Comparative genomics</div></a>
              <div data-categoryid="32937" data-articleid="25728"
                   data-categoryname="Comparative genomics"
                   class="flagImage" title="Flag 'Comparative genomics' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Genomics%22" title="Search for articles in the subject area:'Genomics'"><div class="flagText">Genomics</div></a>
              <div data-categoryid="18221" data-articleid="25728"
                   data-categoryname="Genomics"
                   class="flagImage" title="Flag 'Genomics' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Membrane+proteins%22" title="Search for articles in the subject area:'Membrane proteins'"><div class="flagText">Membrane proteins</div></a>
              <div data-categoryid="17569" data-articleid="25728"
                   data-categoryname="Membrane proteins"
                   class="flagImage" title="Flag 'Membrane proteins' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Operons%22" title="Search for articles in the subject area:'Operons'"><div class="flagText">Operons</div></a>
              <div data-categoryid="33133" data-articleid="25728"
                   data-categoryname="Operons"
                   class="flagImage" title="Flag 'Operons' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Outer+membrane+proteins%22" title="Search for articles in the subject area:'Outer membrane proteins'"><div class="flagText">Outer membrane proteins</div></a>
              <div data-categoryid="20669" data-articleid="25728"
                   data-categoryname="Outer membrane proteins"
                   class="flagImage" title="Flag 'Outer membrane proteins' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Virulence+factors%22" title="Search for articles in the subject area:'Virulence factors'"><div class="flagText">Virulence factors</div></a>
              <div data-categoryid="19615" data-articleid="25728"
                   data-categoryname="Virulence factors"
                   class="flagImage" title="Flag 'Virulence factors' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=76'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=8953'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=4468&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>

<div class="block sidebar-comments">
    <div class="header">
        <h3>Comments</h3>
    </div>
      <p><a href="/annotation/listThread.action?root=41939">See evaluation on F1000 Biol</a><br>Posted by F1000Medicine</p>
      <p><a href="/annotation/listThread.action?root=21007">Evaluation of this article on Faculty of 1000 Biology</a><br>Posted by NiyazAhmed</p>
      <p><a href="/annotation/listThread.action?root=12429">Referee Comments: Referee 3</a><br>Posted by PLoS_ONE_Group</p>
</div>

</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
