

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Evidence-Based Annotation of the Malaria Parasite's Genome Using Comparative Expression Profiling</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0001570"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0001570" />

    <meta name="citation_title" content="Evidence-Based Annotation of the Malaria Parasite's Genome Using Comparative Expression Profiling"/>
    <meta itemprop="name" content="Evidence-Based Annotation of the Malaria Parasite's Genome Using Comparative Expression Profiling"/>

      <meta name="citation_author" content="Yingyao Zhou"/>
            <meta name="citation_author_institution" content="Genomics Institute of the Novartis Research Foundation, San Diego, California, United States of America"/>
      <meta name="citation_author" content="Vandana Ramachandran"/>
            <meta name="citation_author_institution" content="Department of Cell Biology ICND202, The Scripps Research Institute, La Jolla, California, United States of America"/>
      <meta name="citation_author" content="Kota Arun Kumar"/>
            <meta name="citation_author_institution" content="Department of Pathology, New York University, New York, New York, United States of America"/>
      <meta name="citation_author" content="Scott Westenberger"/>
            <meta name="citation_author_institution" content="Department of Cell Biology ICND202, The Scripps Research Institute, La Jolla, California, United States of America"/>
      <meta name="citation_author" content="Phillippe Refour"/>
            <meta name="citation_author_institution" content="Department of Infectious Diseases and Immunology, Harvard School of Public Health, Boston, Massachusetts, United States of America"/>
      <meta name="citation_author" content="Bin Zhou"/>
            <meta name="citation_author_institution" content="Genomics Institute of the Novartis Research Foundation, San Diego, California, United States of America"/>
      <meta name="citation_author" content="Fengwu Li"/>
            <meta name="citation_author_institution" content="Division of Infectious Diseases, Department of Medicine, University of California San Diego, San Diego, California, United States of America"/>
      <meta name="citation_author" content="Jason A. Young"/>
            <meta name="citation_author_institution" content="Department of Cell Biology ICND202, The Scripps Research Institute, La Jolla, California, United States of America"/>
      <meta name="citation_author" content="Kaisheng Chen"/>
            <meta name="citation_author_institution" content="Genomics Institute of the Novartis Research Foundation, San Diego, California, United States of America"/>
      <meta name="citation_author" content="David Plouffe"/>
            <meta name="citation_author_institution" content="Genomics Institute of the Novartis Research Foundation, San Diego, California, United States of America"/>
      <meta name="citation_author" content="Kerstin Henson"/>
            <meta name="citation_author_institution" content="Genomics Institute of the Novartis Research Foundation, San Diego, California, United States of America"/>
            <meta name="citation_author_institution" content="Department of Cell Biology ICND202, The Scripps Research Institute, La Jolla, California, United States of America"/>
      <meta name="citation_author" content="Victor Nussenzweig"/>
            <meta name="citation_author_institution" content="Department of Pathology, New York University, New York, New York, United States of America"/>
      <meta name="citation_author" content="Jane Carlton"/>
            <meta name="citation_author_institution" content="Department of Pathology, New York University, New York, New York, United States of America"/>
      <meta name="citation_author" content="Joseph M. Vinetz"/>
            <meta name="citation_author_institution" content="Division of Infectious Diseases, Department of Medicine, University of California San Diego, San Diego, California, United States of America"/>
      <meta name="citation_author" content="Manoj T. Duraisingh"/>
            <meta name="citation_author_institution" content="Department of Infectious Diseases and Immunology, Harvard School of Public Health, Boston, Massachusetts, United States of America"/>
      <meta name="citation_author" content="Elizabeth A. Winzeler"/>
            <meta name="citation_author_institution" content="Genomics Institute of the Novartis Research Foundation, San Diego, California, United States of America"/>
            <meta name="citation_author_institution" content="Department of Cell Biology ICND202, The Scripps Research Institute, La Jolla, California, United States of America"/>

    <meta name="citation_date" content="2008/2/13"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0001570.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e1570"/>
    <meta name="citation_issue" content="2"/>
    <meta name="citation_volume" content="3"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Genome sequence of the human malaria parasite Plasmodium falciparum.; citation_author=MJ Gardner; citation_author=N Hall; citation_author=E Fung; citation_author=O White; citation_author=M Berriman; citation_journal_title=Nature; citation_volume=419; citation_number=1; citation_pages=498-511; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Laser capture microdissection and molecular analysis of Plasmodium yoelii liver-stage parasites.; citation_author=JB Sacci; citation_author=JC Aguiar; citation_author=AO Lau; citation_author=SL Hoffman; citation_journal_title=Mol Biochem Parasitol; citation_volume=119; citation_number=2; citation_pages=285-289; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=The Transcriptome of the Intraerythrocytic Developmental Cycle of Plasmodium falciparum.; citation_author=Z Bozdech; citation_author=M Llinas; citation_author=BL Pulliam; citation_author=ED Wong; citation_author=J Zhu; citation_journal_title=PLoS Biol; citation_volume=1; citation_number=3; citation_pages=E5; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Analysis of the Plasmodium falciparum proteome by high-accuracy mass spectrometry.; citation_author=E Lasonder; citation_author=Y Ishihama; citation_author=JS Andersen; citation_author=AM Vermunt; citation_author=A Pain; citation_journal_title=Nature; citation_volume=419; citation_number=4; citation_pages=537-542; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Genome sequence and comparative analysis of the model rodent malaria parasite Plasmodium yoelii yoelii.; citation_author=JM Carlton; citation_author=SV Angiuoli; citation_author=BB Suh; citation_author=TW Kooij; citation_author=M Pertea; citation_journal_title=Nature; citation_volume=419; citation_number=5; citation_pages=512-519; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=A proteomic view of the Plasmodium falciparum life cycle.; citation_author=L Florens; citation_author=MP Washburn; citation_author=JD Raine; citation_author=RM Anthony; citation_author=M Grainger; citation_journal_title=Nature; citation_volume=419; citation_number=6; citation_pages=520-526; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=A comprehensive survey of the Plasmodium life cycle by genomic, transcriptomic, and proteomic analyses.; citation_author=N Hall; citation_author=M Karras; citation_author=JD Raine; citation_author=JM Carlton; citation_author=TW Kooij; citation_journal_title=Science; citation_volume=307; citation_number=7; citation_pages=82-86; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=In silico gene function prediction using ontology-based pattern identification.; citation_author=Y Zhou; citation_author=JA Young; citation_author=A Santrosyan; citation_author=K Chen; citation_author=FS Yan; citation_journal_title=Bioinformatics; citation_number=8; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Discovery of gene function by expression profiling of the malaria parasite life cycle.; citation_author=KG Le Roch; citation_author=Y Zhou; citation_author=PL Blair; citation_author=M Grainger; citation_author=JK Moch; citation_journal_title=Science; citation_volume=301; citation_number=9; citation_pages=1503-1508; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Gene ontology: tool for the unification of biology. The Gene Ontology Consortium.; citation_author=M Ashburner; citation_author=CA Ball; citation_author=JA Blake; citation_author=D Botstein; citation_author=H Butler; citation_journal_title=Nat Genet; citation_volume=25; citation_number=10; citation_pages=25-29; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Exposure of Plasmodium sporozoites to the intracellular concentration of potassium enhances infectivity and reduces cell passage activity.; citation_author=KA Kumar; citation_author=CR Garcia; citation_author=VR Chandran; citation_author=N Van Rooijen; citation_author=Y Zhou; citation_journal_title=Mol Biochem Parasitol; citation_volume=156; citation_number=11; citation_pages=32-40; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Exploring the transcriptome of the malaria sporozoite stage.; citation_author=SH Kappe; citation_author=MJ Gardner; citation_author=SM Brown; citation_author=J Ross; citation_author=K Matuschewski; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=98; citation_number=12; citation_pages=9895-9900; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Infectivity-associated changes in the transcriptional repertoire of the malaria parasite sporozoite stage.; citation_author=K Matuschewski; citation_author=J Ross; citation_author=SM Brown; citation_author=K Kaiser; citation_author=V Nussenzweig; citation_journal_title=J Biol Chem; citation_volume=277; citation_number=13; citation_pages=41948-41953; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Genetically modified Plasmodium parasites as a protective experimental malaria vaccine.; citation_author=AK Mueller; citation_author=M Labaied; citation_author=SH Kappe; citation_author=K Matuschewski; citation_journal_title=Nature; citation_volume=433; citation_number=14; citation_pages=164-167; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=etramps, a New Plasmodium falciparum Gene Family Coding for Developmentally Regulated and Highly Charged Membrane Proteins Located at the Parasite-Host Cell Interface.; citation_author=T Spielmann; citation_author=DJ Fergusen; citation_author=HP Beck; citation_journal_title=Mol Biol Cell; citation_volume=14; citation_number=15; citation_pages=1529-1544; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Plasmodium liver stage developmental arrest by depletion of a protein at the parasite-host interface.; citation_author=AK Mueller; citation_author=N Camargo; citation_author=K Kaiser; citation_author=C Andorfer; citation_author=U Frevert; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=102; citation_number=16; citation_pages=3022-3027; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Regulation of sexual development of Plasmodium by translational repression.; citation_author=GR Mair; citation_author=JA Braks; citation_author=LS Garver; citation_author=JC Wiegant; citation_author=N Hall; citation_journal_title=Science; citation_volume=313; citation_number=17; citation_pages=667-669; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Plasmodium falciparum ookinetes require mosquito midgut chondroitin sulfate proteoglycans for cell invasion.; citation_author=RR Dinglasan; citation_author=A Alaganan; citation_author=AK Ghosh; citation_author=A Saito; citation_author=TH van Kuppevelt; citation_journal_title=Proc Natl Acad Sci U S A; citation_number=18; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Global analysis of transcript and protein levels across the Plasmodium falciparum life cycle.; citation_author=KG Le Roch; citation_author=JR Johnson; citation_author=L Florens; citation_author=Y Zhou; citation_author=A Santrosyan; citation_journal_title=Genome Res; citation_volume=14; citation_number=19; citation_pages=2308-2318; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Chitinases of the avian malaria parasite Plasmodium gallinaceum, a class of enzymes necessary for parasite invasion of the mosquito midgut.; citation_author=JM Vinetz; citation_author=JG Valenzuela; citation_author=CA Specht; citation_author=L Aravind; citation_author=RC Langer; citation_journal_title=J Biol Chem; citation_volume=275; citation_number=20; citation_pages=10331-10341; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Molecular cloning of a gene from Plasmodium falciparum that codes for a protein sharing motifs found in adhesive molecules from mammals and plasmodia.; citation_author=F Trottein; citation_author=T Triglia; citation_author=AF Cowman; citation_journal_title=Mol Biochem Parasitol; citation_volume=74; citation_number=21; citation_pages=129-141; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Plasmodium circumsporozoite protein promotes the development of the liver stages of the parasite.; citation_author=AP Singh; citation_author=CA Buscaglia; citation_author=Q Wang; citation_author=A Levay; citation_author=DR Nussenzweig; citation_journal_title=Cell; citation_volume=131; citation_number=22; citation_pages=492-504; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Tropical infectious diseases: metabolic maps and functions of the Plasmodium falciparum apicoplast.; citation_author=SA Ralph; citation_author=GG van Dooren; citation_author=RF Waller; citation_author=MJ Crawford; citation_author=MJ Fraunholz; citation_journal_title=Nat Rev Microbiol; citation_volume=2; citation_number=23; citation_pages=203-216; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Hemoglobin degradation.; citation_author=DE Goldberg; citation_journal_title=Curr Top Microbiol Immunol; citation_volume=295; citation_number=24; citation_pages=275-291; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=A gene atlas of the mouse and human protein-encoding transcriptomes.; citation_author=AI Su; citation_author=T Wiltshire; citation_author=S Batalov; citation_author=H Lapp; citation_author=KA Ching; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=101; citation_number=25; citation_pages=6062-6067; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Yeast Asc1p and mammalian RACK1 are functionally orthologous core 40S ribosomal proteins that repress gene expression.; citation_author=VR Gerbasi; citation_author=CM Weaver; citation_author=S Hill; citation_author=DB Friedman; citation_author=AJ Link; citation_journal_title=Mol Cell Biol; citation_volume=24; citation_number=26; citation_pages=8276-8287; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Proteome analysis of separated male and female gametocytes reveals novel sex-specific Plasmodium biology.; citation_author=SM Khan; citation_author=B Franke-Fayard; citation_author=GR Mair; citation_author=E Lasonder; citation_author=CJ Janse; citation_journal_title=Cell; citation_volume=121; citation_number=27; citation_pages=675-687; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Invasion of red blood cells by malaria parasites.; citation_author=AF Cowman; citation_author=BS Crabb; citation_journal_title=Cell; citation_volume=124; citation_number=28; citation_pages=755-766; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Identification of the moving junction complex of Toxoplasma gondii: a collaboration between distinct secretory organelles.; citation_author=DL Alexander; citation_author=J Mital; citation_author=GE Ward; citation_author=P Bradley; citation_author=JC Boothroyd; citation_journal_title=PLoS Pathog; citation_volume=1; citation_number=29; citation_pages=e17; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Excess polymorphisms in genes for membrane proteins in Plasmodium falciparum.; citation_author=SK Volkman; citation_author=DL Hartl; citation_author=DF Wirth; citation_author=KM Nielsen; citation_author=M Choi; citation_journal_title=Science; citation_volume=298; citation_number=30; citation_pages=216-218; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=A genome-wide map of diversity in Plasmodium falciparum.; citation_author=SK Volkman; citation_author=PC Sabeti; citation_author=D DeCaprio; citation_author=DE Neafsey; citation_author=SF Schaffner; citation_journal_title=Nat Genet; citation_volume=39; citation_number=31; citation_pages=113-119; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=A systematic map of genetic variation in Plasmodium falciparum.; citation_author=C Kidgell; citation_author=SK Volkman; citation_author=J Daily; citation_author=JO Borevitz; citation_author=D Plouffe; citation_journal_title=PLoS Pathog; citation_volume=2; citation_number=32; citation_pages=e57; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Genome variation and evolution of the malaria parasite Plasmodium falciparum.; citation_author=DC Jeffares; citation_author=A Pain; citation_author=A Berry; citation_author=AV Cox; citation_author=J Stalker; citation_journal_title=Nat Genet; citation_volume=39; citation_number=33; citation_pages=120-125; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=A protein interaction network of the malaria parasite Plasmodium falciparum.; citation_author=DJ LaCount; citation_author=M Vignali; citation_author=R Chettier; citation_author=A Phansalkar; citation_author=R Bell; citation_journal_title=Nature; citation_volume=438; citation_number=34; citation_pages=103-107; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Toward a comprehensive atlas of the physical interactome of Saccharomyces cerevisiae.; citation_author=SR Collins; citation_author=P Kemmeren; citation_author=XC Zhao; citation_author=JF Greenblatt; citation_author=F Spencer; citation_journal_title=Mol Cell Proteomics; citation_volume=6; citation_number=35; citation_pages=439-450; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=‘FAS’t inhibition of malaria.; citation_author=A Surolia; citation_author=TN Ramya; citation_author=V Ramya; citation_author=N Surolia; citation_journal_title=Biochem J; citation_volume=383; citation_number=36; citation_pages=401-412; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=inhA, a gene encoding a target for isoniazid and ethionamide in Mycobacterium tuberculosis.; citation_author=A Banerjee; citation_author=E Dubnau; citation_author=A Quemard; citation_author=V Balasubramanian; citation_author=KS Um; citation_journal_title=Science; citation_volume=263; citation_number=37; citation_pages=227-230; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=Triclosan offers protection against blood stages of malaria by inhibiting enoyl-ACP reductase of Plasmodium falciparum.; citation_author=N Surolia; citation_author=A Surolia; citation_journal_title=Nat Med; citation_volume=7; citation_number=38; citation_pages=167-173; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=The Plasmodium protein network diverges from those of other eukaryotes.; citation_author=S Suthram; citation_author=T Sittler; citation_author=T Ideker; citation_journal_title=Nature; citation_volume=438; citation_number=39; citation_pages=108-112; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Plasmodium sporozoite molecular cell biology.; citation_author=SH Kappe; citation_author=CA Buscaglia; citation_author=V Nussenzweig; citation_journal_title=Annu Rev Cell Dev Biol; citation_volume=20; citation_number=40; citation_pages=29-59; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Differential transcriptome profiling identifies Plasmodium genes encoding pre-erythrocytic stage-specific proteins.; citation_author=K Kaiser; citation_author=K Matuschewski; citation_author=N Camargo; citation_author=J Ross; citation_author=SH Kappe; citation_journal_title=Mol Microbiol; citation_volume=51; citation_number=41; citation_pages=1221-1232; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Differential gene expression in the ookinete stage of the malaria parasite Plasmodium berghei.; citation_author=A Raibaud; citation_author=K Brahimi; citation_author=CW Roth; citation_author=PT Brey; citation_author=DM Faust; citation_journal_title=Mol Biochem Parasitol; citation_volume=150; citation_number=42; citation_pages=107-113; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Cross-species comparison of genome-wide expression patterns.; citation_author=XJ Zhou; citation_author=G Gibson; citation_journal_title=Genome Biol; citation_volume=5; citation_number=43; citation_pages=232; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Similarities and differences in genome-wide expression data of six organisms.; citation_author=S Bergmann; citation_author=J Ihmels; citation_author=N Barkai; citation_journal_title=PLoS Biol; citation_volume=2; citation_number=44; citation_pages=E9; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=A gene-coexpression network for global discovery of conserved genetic modules.; citation_author=JM Stuart; citation_author=E Segal; citation_author=D Koller; citation_author=SK Kim; citation_journal_title=Science; citation_volume=302; citation_number=45; citation_pages=249-255; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=An oncogenic KRAS2 expression signature identified by cross-species gene-expression analysis.; citation_author=A Sweet-Cordero; citation_author=S Mukherjee; citation_author=A Subramanian; citation_author=H You; citation_author=JJ Roix; citation_journal_title=Nat Genet; citation_volume=37; citation_number=46; citation_pages=48-55; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Comparing genomic expression patterns across species identifies shared transcriptional profile in aging.; citation_author=SA McCarroll; citation_author=CT Murphy; citation_author=S Zou; citation_author=SD Pletcher; citation_author=CS Chin; citation_journal_title=Nat Genet; citation_volume=36; citation_number=47; citation_pages=197-204; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Global protein function prediction from protein-protein interaction networks.; citation_author=A Vazquez; citation_author=A Flammini; citation_author=A Maritan; citation_author=A Vespignani; citation_journal_title=Nat Biotechnol; citation_volume=21; citation_number=48; citation_pages=697-700; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=The art of gene function prediction.; citation_author=TM Murali; citation_author=CJ Wu; citation_author=S Kasif; citation_journal_title=Nat Biotechnol; citation_volume=24; citation_number=49; citation_pages=1474-1475; author reply 1475–1476; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Genetically attenuated P36p-deficient Plasmodium berghei sporozoites confer long-lasting and partial cross-species protection.; citation_author=B Douradinha; citation_author=MR van Dijk; citation_author=R Ataide; citation_author=GJ van Gemert; citation_author=J Thompson; citation_journal_title=Int J Parasitol; citation_volume=37; citation_number=50; citation_pages=1511-1519; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Plasmodium berghei ANKA: purification of large numbers of infectious gametocytes.; citation_author=AL Beetsma; citation_author=TJ van de Wiel; citation_author=RW Sauerwein; citation_author=WM Eling; citation_journal_title=Exp Parasitol; citation_volume=88; citation_number=51; citation_pages=69-72; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Improved transfection and new selectable markers for the rodent malaria parasite Plasmodium yoelii.; citation_author=AM Jongco; citation_author=LM Ting; citation_author=V Thathy; citation_author=MM Mota; citation_author=K Kim; citation_journal_title=Mol Biochem Parasitol; citation_volume=146; citation_number=52; citation_pages=242-250; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Column separation of Plasmodium berghei sporozoites.; citation_author=SR Mack; citation_author=JP Vanderberg; citation_author=R Nawrot; citation_journal_title=J Parasitol; citation_volume=64; citation_number=53; citation_pages=166-168; citation_date=1978; " />
      <meta name="citation_reference" content="citation_title=Binding of factor VIIa to tissue factor on keratinocytes induces gene expression.; citation_author=E Camerer; citation_author=E Gjernes; citation_author=M Wiiger; citation_author=S Pringle; citation_author=H Prydz; citation_journal_title=J Biol Chem; citation_volume=275; citation_number=54; citation_pages=6580-6585; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Complete in vitro maturation of Plasmodium falciparum gametocytes.; citation_author=T Ifediba; citation_author=JP Vanderberg; citation_journal_title=Nature; citation_volume=294; citation_number=55; citation_pages=364-366; citation_date=1981; " />
      <meta name="citation_reference" content="citation_title=Plasmodiun falciparum gametocyte culture, purification, and gametogenesis.; citation_author=MKBaN Kumar; citation_number=56; citation_pages=93-95; citation_date=2004; citation_publisher=MR4/ATCC:; " />
      <meta name="citation_reference" content="citation_title=Complete development of mosquito phases of the malaria parasite in vitro.; citation_author=EM Al-Olayan; citation_author=AL Beetsma; citation_author=GA Butcher; citation_author=RE Sinden; citation_author=H Hurd; citation_journal_title=Science; citation_volume=295; citation_number=57; citation_pages=677-679; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Plasmodium falciparum: purification of the various gametocyte developmental stages from in vitro-cultivated parasites.; citation_author=MM Kariuki; citation_author=JK Kiaira; citation_author=FK Mulaa; citation_author=JK Mwangi; citation_author=MK Wasunna; citation_journal_title=Am J Trop Med Hyg; citation_volume=59; citation_number=58; citation_pages=505-508; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Synchronization of Plasmodium falciparum erythrocytic stages in culture.; citation_author=C Lambros; citation_author=JP Vanderberg; citation_journal_title=J Parasitol; citation_volume=65; citation_number=59; citation_pages=418-420; citation_date=1979; " />
      <meta name="citation_reference" content="citation_title=The Plasmodium falciparum sexual development transcriptome: A microarray analysis using ontology-based pattern identification.; citation_author=JA Young; citation_author=QL Fivelman; citation_author=PL Blair; citation_author=P de la Vega; citation_author=KG Le Roch; citation_journal_title=Mol Biochem Parasitol; citation_volume=143; citation_number=60; citation_pages=67-79; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=A comparison of normalization methods for high density oligonucleotide array data based on variance and bias.; citation_author=BM Bolstad; citation_author=RA Irizarry; citation_author=M Astrand; citation_author=TP Speed; citation_journal_title=Bioinformatics; citation_volume=19; citation_number=61; citation_pages=185-193; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Parallel identification of new genes in Saccharomyces cerevisiae.; citation_author=G Oshiro; citation_author=LM Wodicka; citation_author=MP Washburn; citation_author=JR Yates; citation_author=DJ Lockhart; citation_journal_title=Genome Res; citation_volume=12; citation_number=62; citation_pages=1210-1220; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Protein pathway and complex clustering of correlated mRNA and protein expression analyses in Saccharomyces cerevisiae.; citation_author=MP Washburn; citation_author=A Koller; citation_author=G Oshiro; citation_author=RR Ulaszek; citation_author=D Plouffe; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=100; citation_number=63; citation_pages=3107-3112; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=OrthoMCL-DB: querying a comprehensive multi-species collection of ortholog groups.; citation_author=F Chen; citation_author=AJ Mackey; citation_author=CJ Stoeckert; citation_author=DS Roos; citation_journal_title=Nucleic Acids Res; citation_volume=34; citation_number=64; citation_pages=D363-368; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Gene set enrichment analysis: a knowledge-based approach for interpreting genome-wide expression profiles.; citation_author=A Subramanian; citation_author=P Tamayo; citation_author=VK Mootha; citation_author=S Mukherjee; citation_author=BL Ebert; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=102; citation_number=65; citation_pages=15545-15550; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Sequestrin, a CD36 recognition protein on Plasmodium falciparum malaria-infected erythrocytes identified by anti-idiotype antibodies.; citation_author=CF Ockenhouse; citation_author=FW Klotz; citation_author=NN Tandon; citation_author=GA Jamieson; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=88; citation_number=66; citation_pages=3175-3179; citation_date=1991; " />
      <meta name="citation_reference" content="citation_title=gar2 is a nucleolar protein from Schizosaccharomyces pombe required for 18S rRNA and 40S ribosomal subunit accumulation.; citation_author=MP Gulli; citation_author=JP Girard; citation_author=D Zabetakis; citation_author=B Lapeyre; citation_author=T Melese; citation_journal_title=Nucleic Acids Res; citation_volume=23; citation_number=67; citation_pages=1912-1918; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Targeting malaria virulence and remodeling proteins to the host erythrocyte.; citation_author=M Marti; citation_author=RT Good; citation_author=M Rug; citation_author=E Knuepfer; citation_author=AF Cowman; citation_journal_title=Science; citation_volume=306; citation_number=68; citation_pages=1930-1933; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=The Plasmodium falciparum clag9 gene encodes a rhoptry protein that is transferred to the host erythrocyte upon invasion.; citation_author=IT Ling; citation_author=L Florens; citation_author=AR Dluzewski; citation_author=O Kaneko; citation_author=M Grainger; citation_journal_title=Mol Microbiol; citation_volume=52; citation_number=69; citation_pages=107-118; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Identification of Plasmodium falciparum antigens by antigenic analysis of genomic and proteomic data.; citation_author=DL Doolan; citation_author=S Southwood; citation_author=DA Freilich; citation_author=J Sidney; citation_author=NL Graber; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=100; citation_number=70; citation_pages=9952-9957; citation_date=2003; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001570" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Evidence-Based Annotation of the Malaria Parasite's Genome Using Comparative Expression Profiling"/>
    <meta name="twitter:description" content="A fundamental problem in systems biology and whole genome sequence analysis is how to infer functions for the many uncharacterized proteins that are identified, whether they are conserved across organisms of different phyla or are phylum-specific. This problem is especially acute in pathogens, such as malaria parasites, where genetic and biochemical investigations are likely to be more difficult. Here we perform comparative expression analysis on Plasmodium parasite life cycle data derived from P. falciparum blood, sporozoite, zygote and ookinete stages, and P. yoelii mosquito oocyst and salivary gland sporozoites, blood and liver stages and show that type II fatty acid biosynthesis genes are upregulated in liver and insect stages relative to asexual blood stages. We also show that some universally uncharacterized genes with orthologs in Plasmodium species, Saccharomyces cerevisiae and humans show coordinated transcription patterns in large collections of human and yeast expression data and that the function of the uncharacterized genes can sometimes be predicted based on the expression patterns across these diverse organisms. We also use a comprehensive and unbiased literature mining method to predict which uncharacterized parasite-specific genes are likely to have roles in processes such as gliding motility, host-cell interactions, sporozoite stage, or rhoptry function. These analyses, together with protein-protein interaction data, provide probabilistic models that predict the function of 926 uncharacterized malaria genes and also suggest that malaria parasites may provide a simple model system for the study of some human processes. These data also provide a foundation for further studies of transcriptional regulation in malaria parasites."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0001570.g005"/>

  <meta property="og:title" content="Evidence-Based Annotation of the Malaria Parasite's Genome Using Comparative Expression Profiling" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001570" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=4930'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=2754'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=2559&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0001570">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001570" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001570&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Evidence-Based Annotation of the Malaria Parasite's Genome Using Comparative Expression Profiling
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Yingyao Zhou
              <span class="equal-contrib"
                    title="These authors contributed equally to this work">equal contributor</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">

                <p><span class="equal-contrib" title="These authors contributed equally to this work">equal contributor</span>
                Contributed equally to this work with: Yingyao Zhou, Vandana Ramachandran, Kota Arun Kumar</p>

              
              

                <p>Affiliation:
                  Genomics Institute of the Novartis Research Foundation, San Diego, California, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Vandana Ramachandran
              <span class="equal-contrib"
                    title="These authors contributed equally to this work">equal contributor</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">

                <p><span class="equal-contrib" title="These authors contributed equally to this work">equal contributor</span>
                Contributed equally to this work with: Yingyao Zhou, Vandana Ramachandran, Kota Arun Kumar</p>

              
              

                <p>Affiliation:
                  Department of Cell Biology ICND202, The Scripps Research Institute, La Jolla, California, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Kota Arun Kumar
              <span class="equal-contrib"
                    title="These authors contributed equally to this work">equal contributor</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">

                <p><span class="equal-contrib" title="These authors contributed equally to this work">equal contributor</span>
                Contributed equally to this work with: Yingyao Zhou, Vandana Ramachandran, Kota Arun Kumar</p>

              
              

                <p>Affiliation:
                  Department of Pathology, New York University, New York, New York, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Scott Westenberger, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Cell Biology ICND202, The Scripps Research Institute, La Jolla, California, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Phillippe Refour, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Infectious Diseases and Immunology, Harvard School of Public Health, Boston, Massachusetts, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Bin Zhou, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Genomics Institute of the Novartis Research Foundation, San Diego, California, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Fengwu Li, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Division of Infectious Diseases, Department of Medicine, University of California San Diego, San Diego, California, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Jason A. Young, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Cell Biology ICND202, The Scripps Research Institute, La Jolla, California, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Kaisheng Chen, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Genomics Institute of the Novartis Research Foundation, San Diego, California, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            David Plouffe, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Genomics Institute of the Novartis Research Foundation, San Diego, California, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Kerstin Henson, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliations:
                  Genomics Institute of the Novartis Research Foundation, San Diego, California, United States of America, 
                  Department of Cell Biology ICND202, The Scripps Research Institute, La Jolla, California, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Victor Nussenzweig, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Pathology, New York University, New York, New York, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Jane Carlton, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Pathology, New York University, New York, New York, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Joseph M. Vinetz, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Division of Infectious Diseases, Department of Medicine, University of California San Diego, San Diego, California, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Manoj T. Duraisingh, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Infectious Diseases and Immunology, Harvard School of Public Health, Boston, Massachusetts, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Elizabeth A. Winzeler
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p>*E-mail: <a href="mailto:winzeler@scripps.edu">winzeler@scripps.edu</a></p>

                <p>Affiliations:
                  Genomics Institute of the Novartis Research Foundation, San Diego, California, United States of America, 
                  Department of Cell Biology ICND202, The Scripps Research Institute, La Jolla, California, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: February 13, 2008</li>
    <li>DOI: 10.1371/journal.pone.0001570</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0001570" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0001570" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001570" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0001570" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0001570-t001" data-doi="info:doi/10.1371/journal.pone.0001570" data-uri="info:doi/10.1371/journal.pone.0001570.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001570.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001570-g001" data-doi="info:doi/10.1371/journal.pone.0001570" data-uri="info:doi/10.1371/journal.pone.0001570.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001570.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001570-t002" data-doi="info:doi/10.1371/journal.pone.0001570" data-uri="info:doi/10.1371/journal.pone.0001570.t002" title="Table 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001570.t002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001570-g002" data-doi="info:doi/10.1371/journal.pone.0001570" data-uri="info:doi/10.1371/journal.pone.0001570.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001570.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001570-g003" data-doi="info:doi/10.1371/journal.pone.0001570" data-uri="info:doi/10.1371/journal.pone.0001570.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001570.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001570-g004" data-doi="info:doi/10.1371/journal.pone.0001570" data-uri="info:doi/10.1371/journal.pone.0001570.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001570.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001570-g005" data-doi="info:doi/10.1371/journal.pone.0001570" data-uri="info:doi/10.1371/journal.pone.0001570.g005" title="Figure 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001570.g005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001570">Reader Comments (0)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0001570" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1"></a><p>A fundamental problem in systems biology and whole genome sequence analysis is how to infer functions for the many uncharacterized proteins that are identified, whether they are conserved across organisms of different phyla or are phylum-specific. This problem is especially acute in pathogens, such as malaria parasites, where genetic and biochemical investigations are likely to be more difficult. Here we perform comparative expression analysis on <em>Plasmodium</em> parasite life cycle data derived from <em>P. falciparum</em> blood, sporozoite, zygote and ookinete stages, and <em>P. yoelii</em> mosquito oocyst and salivary gland sporozoites, blood and liver stages and show that type II fatty acid biosynthesis genes are upregulated in liver and insect stages relative to asexual blood stages. We also show that some universally uncharacterized genes with orthologs in <em>Plasmodium</em> species, <em>Saccharomyces cerevisiae</em> and humans show coordinated transcription patterns in large collections of human and yeast expression data and that the function of the uncharacterized genes can sometimes be predicted based on the expression patterns across these diverse organisms. We also use a comprehensive and unbiased literature mining method to predict which uncharacterized parasite-specific genes are likely to have roles in processes such as gliding motility, host-cell interactions, sporozoite stage, or rhoptry function. These analyses, together with protein-protein interaction data, provide probabilistic models that predict the function of 926 uncharacterized malaria genes and also suggest that malaria parasites may provide a simple model system for the study of some human processes. These data also provide a foundation for further studies of transcriptional regulation in malaria parasites.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>Zhou Y, Ramachandran V, Kumar KA, Westenberger S, Refour P, et al.  (2008) Evidence-Based Annotation of the Malaria Parasite's Genome Using Comparative Expression Profiling. PLoS ONE 3(2):
          e1570.
            doi:10.1371/journal.pone.0001570</p><p><strong>Academic Editor: </strong>Oliver Hofmann, Harvard School of Public Health, United States of America</p><p><strong>Received:</strong> November 13, 2007; <strong>Accepted:</strong> January 9, 2008; <strong>Published:</strong> February 13, 2008</p><p><strong>Copyright:</strong> © 2008 Zhou et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>This work was partially supported by grants to EAW from the Keck Foundation, the Ellison Foundation, NIH, the Wellcome Trust and the Medicines for Malaria Venture, and in part by grants to JMV from the NIH. JAY was supported by a pre-doctoral fellowship from the National Science Foundation.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>Drug targets for many pathogenic microbes such as <em>Mycobacterium tuberculosis</em> or <em>Plasmodium falciparum</em> are often selected from proteins identified through genome sequencing efforts, for which enzymatic function is already known allowing for biochemical assay development in drug screening. This expedient approach means that many genes encoding proteins with uncharacterized functions may be disregarded as potential therapeutic or disease-preventive targets. These assumptions have several consequences. First, the diversity in many chemical libraries is not utilized. Second, focusing on the pathogen enzymes most likely to be essential to all life increases the likelihood that a compound will also have activity against the host.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>While using well-characterized targets (e.g. dihydrofolate reductase, dihydropteroate synthase, ubiquinol-cytochrome c reductase) studied previously in model systems such as <em>Saccharomyces cerevisiae</em> or <em>Escherichia coli</em> is perhaps inevitable for practical reasons, the genome sequence of the malaria parasite, <em>P. falciparum</em>, does encode a plethora of potentially novel drug and vaccine targets as indicated in the genome sequencing project <a href="#pone.0001570-Gardner1">[1]</a>. It has also been noted that the proportion of uncharacterized genes <em>in P. falciparum</em> is much higher than in many other sequenced genomes <a href="#pone.0001570-Gardner1">[1]</a>. While some of these uncharacterized proteins may be very specific to <em>P. falciparum</em>, which causes the most severe form of human malaria, others have orthologs in many other parasites and even in plants, but their roles are not understood in any species. Apicomplexan parasites, including <em>Babesia</em>, <em>Theileria</em>, <em>Toxoplasma</em>, and <em>Cryptosporidium spp.</em> generally are not easy to manipulate experimentally as many have complex life cycles within multiple hosts and it may be difficult to obtain sufficient quantities of parasites for biochemical assays. Given the impact of malaria on human society, the problem of discovering what these conserved, but uncharacterized, genes are doing for the cell is particularly pressing and new approaches that take advantage of empirical data derived from parasites instead of model systems are needed. Here we use detailed cross-species expression data to create maps that associate uncharacterized genes with different cellular processes with an emphasis on those that are not represented in model systems.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>While numerous microarray or cDNA sequencing experiments have been performed on malaria parasites, the use of two-channel comparative formats, different array designs and, in particular, the limited coverage (e.g. 2,045 independent clones obtained from a <em>P. berghei</em> mung bean nuclease DNA libraries) sometimes limits the usefulness of these data. To illustrate, a cDNA library was constructed from liver stage schizonts that had been obtained by laser capture microdissection and 623 transcripts were identified <a href="#pone.0001570-Sacci1">[2]</a>. Of these, 25% were not detected by cDNA sequencing of blood or sporozoite stages. However, inspection shows some of the presumed liver-specific transcripts are likely the result of sequencing an insufficient number of cDNAs in the different stages. Replication factor A is reported as being expressed exclusively in liver stages although there is only one copy of this gene in the genome and DNA replication is also happening in blood and oocyst stages. Likewise, GMP synthetase is reported as expressed exclusively in liver stages while knowledge of metabolism suggests that its product should be required throughout the lifecycle. Indeed, previous transcription and proteomic studies of <em>P. falciparum</em> indicate high levels of GMP synthetase in trophozoite stages. PY01200, a transcript shown to be liver specific by RT-PCR also shows high expression in <em>P. falciparum</em> merozoites and protein has been detected in <em>P. falciparum</em> sporozoites. A very comprehensive expression analysis of <em>P. falciparum</em> blood stages <a href="#pone.0001570-Bozdech1">[3]</a> was also performed, but again, without considering insect stage data, it is hard to determine the significance of genes showing low amplitude changes in the erythrocytic cycle. These observations do not indicate that the data from these previous studies are of poor quality, but rather that coverage is insufficient to assume presence and absence of transcripts in the various parasite lifecycle stages. Proteomic expression data is also available for many parasite species <a href="#pone.0001570-Lasonder1">[4]</a>, <a href="#pone.0001570-Carlton1">[5]</a>, <a href="#pone.0001570-Florens1">[6]</a>, <a href="#pone.0001570-Hall1">[7]</a> but coverage, especially when only high quality spectra are used, is generally much sparser than for microarray studies, thus limiting their usefulness. For example, one would expect stochiometric representation of proteins comprising different protein complexes (e.g. proteosomes, ribosomes, t-complex) across parasite lifecycle stages in proteomic data from <em>P. berghei</em> but this is not observed due to random sampling error at low coverage and chance. Likewise, histones, a component of chromatin that should be found in all lifecycle stages, are detected in only some stages—H3 is not found in ookinetes while H2A is not found in sporozoites <a href="#pone.0001570-Hall1">[7]</a>. Thus, more studies are necessary. Here we aim to provide the first comprehensive genome-wide expression analysis of malaria parasite transcription including all life cycle stages.</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p>Even if comprehensive data were already available for malaria parasites, a robust method for analyzing all of the data would also be needed. Researchers generally use gene expression data to find new genes involved in processes using a “guilt-by-association” approach and to understand transcriptional regulation. In both cases genes first need to be clustered by their expression similarity. However, cluster boundary determinations often are subjective and non-optimal for the purpose of function prediction. This challenge is addressed by an algorithm termed Ontology-based Pattern Identification (OPI) <a href="#pone.0001570-Zhou1">[8]</a>, which has been shown to identify gene clusters of better quality than unsupervised clustering algorithms such as the robust <em>k</em>-means clustering we used previously <a href="#pone.0001570-LeRoch1">[9]</a>. The OPI analysis begins with a piece of knowledge, such as a group of genes believed to share a set of characteristics—e.g. all genes known to be involved in gametocytogenesis in malaria parasites. From this list of “seed genes” an “average” expression profile that best represents gametocytogenesis can be constructed, and all of the malaria genes in the genome can be relatively ranked according to their similarity to the average profile as measured by Pearson correlation. Genes ranked near the top are more likely to be involved with the gametocytogenesis process. OPI iteratively descends the rank list and identifies a cutoff, where the largest number of seed genes are included within the smallest cluster size as computed by the minimization of a hypergeometric probability score. As the <em>p</em>-value represents the odds of non-seed genes in the resultant cluster sharing a similar expression profile to the seed genes by chance, OPI clusters automatically balance between false discovery rate and true positive rate for the purpose of functional prediction, without relying on subjective parameters. This process is repeated for all possible pieces of knowledge and results in gene clusters containing both known genes and unassigned genes for each functional group. The resultant clusters are then subjected to statistical permutation tests. It is apparent that the accuracy of such predictions relies on both the completeness of the underlying data set and the quality of the seeding knowledge. Although we had previously carried out OPI studies, it was based on a limited set of <em>P. falciparum</em> data, as well as knowledge obtained from the Gene Ontology (GO) consortium <a href="#pone.0001570-Ashburner1">[10]</a>, which may not represent parasite-specific processes well. The accuracy of our predictions in this study is significantly improved as we not only combine gene expression data from both <em>P. yoelii</em> and <em>P. falciparum</em> and cover virtually all lifecycle stages, but also establish a systematic literature mining pipeline to explore many new groups of functionally-related genes based on co-citation in 1,278 malaria-related publications. To validate the predictions we apply the same routine to human and yeast expression data, as well as protein-protein interaction data, and illustrate that the method can accurately predict many functions. The results are a searchable database that can be used by researchers seeking information about the possible functions of uncharacterized malaria proteins as well as a comprehensive set of unprocessed data that may be of use to those interested in systems biology.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3>
<h4><em>P. yoelii</em> sporozoite transcription</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>To infer function of malaria proteins, we had previously collected detailed expression data from <em>P. falciparum</em> <a href="#pone.0001570-LeRoch1">[9]</a>. Here we add <em>P. falciparum in vitro</em> zygote and ookinete gene expression data, as well as data from a variety of developmental stages from the rodent malaria parasite, <em>P. yoelii</em>. For this project, rodent parasite data were collected from ~150,000 25mer <em>P. yoelii</em> probes contained on a previously described custom-designed oligonucleotide array <a href="#pone.0001570-LeRoch1">[9]</a>. These new <em>P. yoelii</em> data included oocyst-stage sporozoites, blood-stage parasites, late-stage gametocytes and 36 and 40 hour liver stage parasites obtained by a cDNA subtraction method. Previously described <em>P. yoelii</em> salivary gland sporozoite data were also used <a href="#pone.0001570-Kumar1">[11]</a>. Expression data were collected for 5,521 different <em>P. yoelii</em> genes. The new expression data for <em>P. yoelii</em> sporozoite stages agree with results obtained previously from sequencing cDNA libraries <a href="#pone.0001570-Kappe1">[12]</a>. As expected, <em>UIS3</em> (Up In Sporozoites) shows substantial (5-6 fold) induction in salivary gland sporozoites relative to oocyst stage sporozoites <a href="#pone.0001570-Matuschewski1">[13]</a>. Interestingly, although <em>UIS3</em> is transcribed in sporozoites, it has been shown to be essential for liver stage development, <a href="#pone.0001570-Mueller1">[14]</a> suggesting that transcriptional silencing may also play a role in the biology of exoerythrocytic forms (EEFs) of malaria parasites. Likewise, <em>UIS4</em> (PY00204) is upregulated. This gene has a <em>P. falciparum</em> homolog (PF10_0164, by syntenic analysis) that is abundantly expressed in sporozoites and encodes a member of the early transmembrane protein family important for establishing the parasitophorous vacuoles in blood stage parasites <a href="#pone.0001570-Spielmann1">[15]</a>. The rodent version is also critical for early liver stage development rather than sporozoites function <a href="#pone.0001570-Mueller2">[16]</a>. Other proteins with similar expression patterns (i.e., induction in salivary gland sporozoites relative to oocyst stages) are likely to be essential for early liver stage development. Genes highly expressed in midgut sporozoites relative to salivary gland sporozoites are mostly uncharacterized but include MAEBL (PY05977) and a calcium dependent kinase (PY06394), both of which were upregulated by about 20 fold.</p>


<h4><em>P. falciparum</em> ookinete and zygote transcriptional patterns</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>New <em>P. falciparum</em> ookinete and zygote data indicate unexpected dynamics of gene expression that likely reflect novel mechanisms of gene expression regulation including post-transcriptional repression <a href="#pone.0001570-Mair1">[17]</a>. The zygote/ookinete stages of <em>P. falciparum</em> are diploid and <em>P. falciparum</em> ookinetes have been difficult to isolate in large numbers. Here, we were able to obtain <em>P. falciparum</em> ookinetes using a modified approach <a href="#pone.0001570-Dinglasan1">[18]</a> in which an estimated 5–10% morphological transformation of zygotes to ookinetes was seen. The genes showing the largest expression values in ookinetes included many glycolysis genes such as glyceraldehyde-3-phosphate dehydrogenase and fructose-bisphosphate aldolase, which are not highly transcribed during gametocytogenesis, reflecting a metabolic shift needed for development within the mosquito midgut. The most highly expressed genes in the zygote (with 6 or more probes) included the heat shock 70 protein, PF08_0054 (ranked 1st); the <em>P. falciparum</em> gamete antigen, PF13_0011 (ranked 2nd); a sexual stage specific protein, PFD0310w (ranked 24th); and circumsporozoite-trap related protein (CTRP), PFC0640w (ranked 20th). Many of the most highly transcribed genes with rodent homologs had been identified in analysis of the rodent ookinete proteome at levels that would not be expected by chance. Among genes with expression values of greater than 1000 (≥6 probes), and with rodent orthologs (233 genes), 96 were detected with more than 4 spectra in the ookinete proteome analysis. In contrast, only 15 genes with more than four spectra were detected amongst the 270 with transcript levels of less than 25. This difference is not expected by chance (<em>p</em> = 10<sup>−21</sup> by χ<sup>2</sup>-test). Several of the most abundant genes in the proteome analysis were also the most abundant as measured on the array including a glyceraldehyde-3-phosphate dehydrogenase (ranked 3rd by proteomics, 2nd by gene expression) and heat shock protein 70 (ranked 5th by proteomics, 3rd by gene expression, out of 3,137 genes with 6 or more probes). The 15 proteins that were detected by MudPIT but were not considered “transcribed” included a helicase, and a translation factor. It is difficult to determine whether these represent species-specific differences, annotation problems or the effects of translational repression <a href="#pone.0001570-Hall1">[7]</a>, <a href="#pone.0001570-Mair1">[17]</a>, <a href="#pone.0001570-LeRoch2">[19]</a>, because in most cases transcripts were detected in earlier gametocyte stages. Genes considered abundant by transcript analysis, but which were not detected in the proteomic analysis included many membrane proteins, which can be difficult to detect by mass-spectrometry. Several highly expressed genes do not have rodent orthologs and thus their abundant expression has not been previously reported. Although <em>P. falciparum</em> ookinetes and zygotes are morphologically similar, comparison of gene expression patterns showed that a clear up- and down-regulation of some genes was occurring, despite an overall similarity (<em>r</em> = 0.92). Differences between the zygote and the ookinete included a 5-fold downregulation of the transcript for chitinase, a gene required for penetration of the mosquito midgut epithelium, which is likely transcribed in zygotes and used in ookinetes <a href="#pone.0001570-Vinetz1">[20]</a>, and a 5-fold downregulation of PFC0640w, the CTRP <a href="#pone.0001570-Trottein1">[21]</a>, a gene required for gliding motility in ookinetes. The downregulation of important ookinete proteins mostly likely reflects the kinetic differences between transcription and translation <a href="#pone.0001570-LeRoch2">[19]</a>, <a href="#pone.0001570-Vinetz1">[20]</a>.</p>


<h4><em>P. yoelii</em> liver stage transcription</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>We also examined expression data from <em>P. yoelii</em> liver stage parasites. From an immunological perspective this is the most interesting stage but also the most difficult to study. We performed a cDNA subtraction using RNA from uninfected livers because of the high potential for host RNA contamination. We anticipated that this would be necessary to detect parasite transcript signal because the parasite genome is 1,000 times smaller than the mouse genome and because only 10% of the cells were infected. Overall, these data were of a lower intensity than for other samples, but the expression differences relative to blood stages and the enrichment of genes involved in various pathways were consistently observed between the two liver samples. The characterized genes (<a href="#pone-0001570-t001">Table 1</a>) that showed the largest rank ratio changes in liver against mixed blood stages in <em>P. yoelii</em> were PY01852, a putative phosphotidylethanolamine binding protein; PY03168, the circumsporozoite protein; PY00446, a polyprenyl synthetase; PY00573 lipoamide dehydrogenase; and PY01586, beta-hydroxyacyl-acp dehydratase precursor. With the exception of the circumsporozoite protein, which is translocated to the host hepatocyte cytoplasm where it exerts a profound effect on host transcription <a href="#pone.0001570-Singh1">[22]</a>, the rest are localized to the apicoplast <a href="#pone.0001570-Ralph1">[23]</a>. The apicoplast, like the chloroplast, is a thought to be a remnant of an endosymbiotic bacterium and is the site of type II fatty acid biosynthesis. Several other proteins with non-specific roles in signaling were also upregulated including PY01269, a phosphotyrosyl phosphatase activator; PY01857, a cAMP-specific 3′, 5′-cyclic phosphodiesterase; and PY07390, a calcium-dependent protein kinase. Genes showing the greatest upregulation in blood stages relative to liver included many ribosomal proteins, plasmepsins, heat shock proteins, glycolysis enzymes (e.g. lactate dehydrogenase) and a falcipain. The plasmepsins and falcipains are proteases involved in hemoglobin degradation <a href="#pone.0001570-Goldberg1">[24]</a> and thus their absence in the liver stages is to be expected.</p>
<div class="figure" id="pone-0001570-t001"><div class="img"><a name="pone-0001570-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001570.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001570" data-uri="info:doi/10.1371/journal.pone.0001570.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001570.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001570.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001570.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001570.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001570.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001570.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001570.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001570.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001570.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>Genes upregulated 36 and 40 hpi in liver stages relative to blood stages.</span></strong></p><span>doi:10.1371/journal.pone.0001570.t001</span></div>

<h4>Grouping genes by expression pattern and statistically testing associations</h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>To create models for the function of various malaria genes and to further validate the data, <em>yoelii-falciparum</em> ortholog pairs were identified and the combined expression data vectors were then hierarchically arranged. The OPI clustering algorithm was then applied, which relies on knowledge such as Gene Ontology annotations to optimize cluster boundaries, normalization and weighting methods, so that the highest proportion of genes with particular classifications is contained in the smallest sized cluster of co-expressed genes. The approach allows us to assess the quality of the expression data and is more rigorous than spot-checking a random collection of genes. For example, it is now accepted that genes encoding proteins that are part of multi-protein complexes (e. g. ribosomes) are likely to be co-transcribed. Therefore, if expression data are universally high-quality, one would expect that almost all members of a complex to show highly correlated expression patterns across conditions and that the correlation would be much higher than if genes were randomly assorted. While it is relatively trivial to choose a few genes and then to show that they are upregulated by quantitative RT-PCR, or that their proteins are developmentally regulated by a western blot, such an approach just validates the behavior of the genes that were chosen (and whose selection might not have been random) and does not reveal much about the quality of the measurements for the other 5000 genes in the dataset. However, there are also problems with relying solely on statistical measurements because of inherent correlations between genes and ontology terms. Therefore, permutation testing was performed.</p>
<a id="article1.body1.sec2.sec4.p2" name="article1.body1.sec2.sec4.p2"></a><p>Applying OPI resulted in 98 non-redundant clusters derived from gene ontologies highly enriched for processes such as glycolysis or protein synthesis, or for cellular components such as the proteosome core complex (<a href="#pone-0001570-g001">Figure 1</a>, <a href="#pone.0001570.s001">Table S1</a>). While we had previously performed a similar analysis on a limited set of sexual development and erythrocytic stage <em>P. falciparum</em> data, the addition of the new data from oocyst and salivary gland sporozoite stages and the creation of combined expression vectors with data from both human and rodent parasites substantially improved the quality of the predictions and allowed the separation of genes which had previously been grouped together. In a previous analysis of sexual development and asexual cycles we identified 246 genes associated with gametocytogenesis, which included the genes involved in type II fatty acid biosynthesis such as PF11_0256, the pyruvate dehydrogenase E1 component. Here we can show that while type II fatty acid biosynthesis genes are upregulated during sexual development they are also upregulated in liver stage development, while others are not. For this study the <em>p</em>-values for functional enrichment calculated with the accumulated hypergeometric distribution ranged from 10<sup>−69.0</sup> to 10<sup>−8.1</sup>. An example of one of the clusters is shown in <a href="#pone-0001570-t002">Table 2</a> (others can be downloaded as additional data files, <a href="http://carrier.gnf.org/publications/Py">http://carrier.gnf.org/publications/Py</a>, the companion website), which shows the <em>P. falciparum</em> genes in a group enriched for the cellular component, nucleolus (GO:0005730). This group contains eight of the fourteen annotated <em>P. falciparum</em> or <em>P. yoelii</em> nucleolus genes in a group of 28. Given that 6,592 genes were considered in this analysis the probability of enrichment by chance is very low (<em>p</em> = 10<sup>−15.3</sup>). Not only is the <em>p</em>-value low, but it is likely much higher than it should be: Evidence compiled independently indicates that almost every “hypothetical” gene in the cluster has a yeast ortholog, most with likely roles in RNA polymerase I processing and transcription (<a href="#pone-0001570-t002">Table 2</a>). There are numerous other similar examples of the quality of the expression data as evidenced by the functional enrichments. For example, of the twelve genes in GO group GO:0005663 (DNA replication factor C complex), twelve are found in a cluster of 27 (<em>p</em> = 10<sup>−28.9</sup>), with most of the other genes having a role in DNA replication. Of the 12 components of the chaperonin-containing T complex (GO:0005832), ten are contained in a group of 12 genes (<em>p</em> = 10<sup>−27.2</sup>). The gluconeogenesis cluster contains 9 of the 13 annotated genes in a group of 14 (<em>p</em> = 10<sup>−21.9</sup>), with lactate dehydrogenase considered a miss. The patterns of gene regulation for the different functionally-enriched categories can be seen by following the “OPI Web Portal” link on the companion website and the representative profile for each cluster is shown in <a href="#pone-0001570-g001">Figure 1</a>.</p>
<div class="figure" id="pone-0001570-g001"><div class="img"><a name="pone-0001570-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001570.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001570" data-uri="info:doi/10.1371/journal.pone.0001570.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001570.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001570.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001570.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001570.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001570.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001570.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001570.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001570.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001570.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Temporal expression patterns were constructed from 54 <em>P. falciparum</em> and <em>P. yoelii</em> life cycle samples.</span></strong></p><a id="article1.body1.sec2.sec4.fig1.caption1.p1" name="article1.body1.sec2.sec4.fig1.caption1.p1"></a><p>A total of 156 statistically enriched gene clusters identified by OPI analysis illustrates the transcription regulation characteristics of all key biological processes in <em>Plasmodium</em> species. Their yeast and human orthologs contents are represented by the white-blue heatmap, indicating parasite-specific processes generally found fewer orthologs in model organisms. The percentage of proteins that form statistically significant within-cluster networks are also white-blue color coded; most networks occur in blood stage processes. Altogether 33 manuscripts were identified with significant overlap to the clusters, nine of which <a href="#pone.0001570-Matuschewski1">[13]</a>, <a href="#pone.0001570-Khan1">[27]</a>, <a href="#pone.0001570-Cowman1">[28]</a>, <a href="#pone.0001570-Surolia1">[36]</a>, <a href="#pone.0001570-Raibaud1">[42]</a>, <a href="#pone.0001570-Young1">[60]</a>, <a href="#pone.0001570-Marti1">[68]</a>, <a href="#pone.0001570-Ling1">[69]</a>, <a href="#pone.0001570-Doolan1">[70]</a> are referenced in the figure. Two clusters were enriched for proteins predicted to have a parasite export signal <a href="#pone.0001570-Marti1">[68]</a> and were labeled as “Exported proteins (sporozoite)” and “Exported proteins (trophozoite)”—one of which peaks in the trophozoite stage and a second which peaks in sporozoite stages (see GO:PM15591202_Trp and GO:PM15591202_Spo in <a href="#pone.0001570.s001">Table S1</a>-<a href="#pone.0001570.s002">S2</a>). S &amp; T indicate that the <em>P. falciparum</em> parasites were synchronized within the asexual cycle by the thermocycling or sorbitol method <a href="#pone.0001570-LeRoch1">[9]</a>. The figure does not comprehensively describe all gene expression patterns contained within the data as there are ~1,026 genes which are not found in any of the groups depicted here because they do not share expression patterns with a sufficient number of previously characterized genes.</p>
<span>doi:10.1371/journal.pone.0001570.g001</span></div><div class="figure" id="pone-0001570-t002"><div class="img"><a name="pone-0001570-t002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001570.t002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001570" data-uri="info:doi/10.1371/journal.pone.0001570.t002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001570.t002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001570.t002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001570.t002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001570.t002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001570.t002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001570.t002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001570.t002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001570.t002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001570.t002.TIF"></span>)
                </a></li></ul></div><p><strong>Table 2.  <span>Example of a gene expression cluster enriched for genes with roles in RNA polymerase nucleolus I activity (GO:0005730).</span></strong></p><span>doi:10.1371/journal.pone.0001570.t002</span></div><a id="article1.body1.sec2.sec4.p3" name="article1.body1.sec2.sec4.p3"></a><p>One drawback of expression-based function assignment is that in some cases different processes may share similar expression patterns due to possible incomplete coverage of lifecycle stages, inherent noise in the measurements, or the possibility that transcripts for proteins with relatively different cellular roles may be controlled by the same transcription factors. For example, genes involved in both ribosomal protein function and glycolysis are both transcriptionally induced during the trophozoite, or feeding stage of parasite growth. We thus also calculated a false discovery rate (FDR) for each group of genes. Smaller clusters often have FDR below 50%, indicating its function assignments are of better quality compared to larger clusters of higher FDR, therefore only clusters of size ≤500 were reported (<a href="#pone-0001570-g001">Figure 1</a>, <a href="#pone.0001570.s001">Table S1</a>, <a href="#pone.0001570.s002">S2</a>). The FDR assigned here tend to be conservative, because false discovery (FD) genes in some clusters are likely due to incorrect or incomplete annotation. For example, a group 16 genes enriched for proteins with roles in DNA replication initiation (GO:0006270) contains ten of the 15 annotated genes (<em>p</em> = 10<sup>−23.4</sup>). Examination of the list, however, shows that four of the six “FD genes” are DNA replication licensing factors, proteins needed for DNA replication initiation.</p>


<h4>Validating the data with <em>S. cerevisiae</em> expression patterns</h4>
<a id="article1.body1.sec2.sec5.p1" name="article1.body1.sec2.sec5.p1"></a><p>Clusters with a low FDR contain mostly known proteins, and therefore one can postulate a role for many conserved FD proteins using annotations and the literature for these clusters. However, to identify correct predictions for more difficult clusters, we examined a collection of <em>S. cerevisiae</em> gene expression data that had been collected on the S98 array and human expression data derived from 79 tissues collected on the U133A array <a href="#pone.0001570-Su1">[25]</a>. These data were processed in a manner similar to the <em>Plasmodium</em> gene expression data resulting in 1,046 and 684 statistically significant groups, respectively. For each <em>Plasmodium</em> gene in an OPI cluster, we looked for the occurrence of its yeast and human orthologs in the related GO clusters in model organisms as additional evidence of correct functional assignment. To be conservative, only evidence from the same or child cross-species GO clusters were automatically compiled, which resulted in 2,807 gene-GO assignments for 696 malaria genes (<a href="#pone.0001570.s002">Table S2</a>). These assignments included all co-expression evidence, i.e., assignments supported by either yeast or human co-expression analysis, as well as those derived from clusters identified based on literature mining approach as discussed later. There are numerous examples of cross-species support for uncharacterized proteins, in fact, as many as 66% (1,839) of our functional assignments have not been captured by existing manual curation. The OPI cluster generated using the replisome GO group (GO:0030894) contains 22 of 30 members of the replisome in a group of 109 genes. This cluster contains PFE0090w and PFF0785w, uncharacterized proteins whose orthologs are associated with the numerous DNA replication processes in humans and yeast (e.g. yeast replisome, <em>p</em> = 10<sup>−9.38</sup>, human alpha DNA polymerase primase complex, <em>p</em> = 10<sup>−5.31</sup>). Most of the proteins in this group have expression profiles that peak while DNA replication is occurring in blood, liver and sexual stages and appear to have roles in cell cycle processes. Another gene that has strong support for its involvement in translation is PF08_0019 (yeast ortholog, YMR116C). This gene is found in a cluster that is strongly enriched for components of the eukaryotic cytosolic ribosome (GO:0005830) in <em>Plasmodium</em> species (<em>p</em> = 10<sup>−37.2</sup>) and <em>S. cerevisiae</em> (p&lt;10<sup>−163.0</sup>). Furthermore, while not annotated as being a component of the ribosome, its disruption in <em>S. cerevisae</em> leads to defects in translation <a href="#pone.0001570-Gerbasi1">[26]</a>. Of course, these data are dependent on having accurate gene models and protein sequence alignments. If the protein sequence is not correctly predicted, spurious matches between unrelated genes may occur. Indeed, based on our work expressing recombinant proteins we estimate that there may be minor problems with the gene models for 20% of <em>P. falciparum</em> proteins. These expression data may assist in correcting and evaluating gene models.</p>


<h4>Confirmation in human expression patterns</h4>
<a id="article1.body1.sec2.sec6.p1" name="article1.body1.sec2.sec6.p1"></a><p>Better support for the functions of proteins was sometimes found in the human gene expression atlas data compared to the yeast data. One striking category was the group of genes induced during gametocytogenesis in <em>Plasmodium spp</em>. (GO:GNF0004). Malaria parasites form male and female gametes within the mosquito and <em>Plasmodium spp.</em> male gametes, like their human counterparts, bear flagella. A number of the genes in this cluster of 480 genes have human orthologs, but as one might expect, no yeast orthologs. Of these genes 26 are found in a spermatogenesis-specific human cluster (483 transcripts, <em>p</em> = 10<sup>−28.2</sup>). The gene PF13_0269 encodes a putative glycerol kinase and is induced during gametocytogenesis. Its human ortholog, GK2, is strongly associated with male sexual development with a large induction of expression specifically in human testis-derived tissues. While the co-expression of characterized genes forming the flagellar axoneme is expected based on previous work <a href="#pone.0001570-Khan1">[27]</a>, the presence of uncharacterized <em>P. falciparum</em> proteins, PFL0325w, PF14_0493, PFL1295w, and PF13_0060, all of which map to uncharacterized genes in the human spermatogenesis cluster, was not expected. We also found that some genes in the sexual development cluster have human counterparts and that they are expressed specifically during early erythroid development. These included PF14_0774 and PFE0930w, both of which likely play roles in heme biosynthesis.</p>


<h4>Using automated literature mining to create new biologically relevant groups of genes</h4>
<a id="article1.body1.sec2.sec7.p1" name="article1.body1.sec2.sec7.p1"></a><p>Since malaria parasites have not traditionally served as model experimental systems, genome curation has mostly relied on transferring GO annotations from other model organisms via ortholog mapping. Unfortunately, this evidence-based annotation scheme is not applicable to the many parasite-specific processes that do not exist in humans or yeast. However, over the past few decades, the malaria community has investigated many <em>Plasmodium</em>-specific biological processes. In other organisms, high quality functional annotations have been assembled through automated and manual literature mining (e.g. the Saccharomyces Genome Database, <a href="http://www.yeastgenome.org">http://www.yeastgenome.org</a>). Such an approach has been adopted for model organisms, but not for <em>Plasmodium spp.</em> largely because of the low/non-profit nature of malaria research fails to justify the prohibitive cost. Therefore, we developed an automated literature-mining tool to identify groups of functionally-related malaria proteins based on their co-citation in the same manuscript or related group of publications (<a href="#pone-0001570-g002">Figure 2</a>). First, the World Wide Web was searched for occurrences of <em>P. falciparum</em> or <em>P. yoelii</em> locus names. An informatics pipeline was then used to process co-cited genes (<a href="#pone.0001570.s003">Table S3</a>, <a href="#pone.0001570.s004">S4</a>). Groups of co-cited genes in one manuscript or several closely related papers comprised over 1,023 virtual GO categories. A comparison in which gene expression correlation between randomly-associated genes, genes co-cited in a manuscript, or genes found in an ontology group indicated that the best correlation was amongst genes that were co-cited, no doubt because of the inclusion of expert knowledge (<a href="#pone-0001570-g003">Figure 3</a>). The figure shows that gene pairs within literature groups are 1,506 times more likely to have a correlation coefficient above 0.9 compared to a random gene pair. The enrichment factor for ontology groups are also as high as 62. The clear differences in the three distributions indicate genes mentioned in the same publication and genes sharing the same ontology terms are more likely to be co-regulated than by chance.</p>
<div class="figure" id="pone-0001570-g002"><div class="img"><a name="pone-0001570-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001570.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001570" data-uri="info:doi/10.1371/journal.pone.0001570.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001570.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001570.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001570.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001570.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001570.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001570.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001570.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001570.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001570.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>An automatic pipeline for malaria literature mining.</span></strong></p><a id="article1.body1.sec2.sec7.fig1.caption1.p1" name="article1.body1.sec2.sec7.fig1.caption1.p1"></a><p>Approach A, full text search by literature search engines: A1) All <em>P. falciparum</em> and <em>P. yoelii</em> locus names were downloaded from PlasmoDB and searched against Google Scholar and SCIRUS one at a time; A2) URL hits were then mapped to PubMed entries. Approach B, NCBI database mining: B1) Mapping between GenBank sequence entries and PubMed entries were systematically retrieved from NCBI for four <em>Plasmodium</em> species; B2) Sequences were mapped to malaria locus names by BLAST alignment. The pipeline resulted in 6,428 functional associations between 3,262 malaria proteins and 1,278 PubMed papers.</p>
<span>doi:10.1371/journal.pone.0001570.g002</span></div><div class="figure" id="pone-0001570-g003"><div class="img"><a name="pone-0001570-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001570.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001570" data-uri="info:doi/10.1371/journal.pone.0001570.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001570.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001570.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001570.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001570.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001570.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001570.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001570.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001570.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001570.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Probability density distributions of gene expression correlation coefficients.</span></strong></p><a id="article1.body1.sec2.sec7.fig2.caption1.p1" name="article1.body1.sec2.sec7.fig2.caption1.p1"></a><p>Gene pairs that either share the same ontology term (red) or are co-cited in a paper (green) are more likely to be co-expressed than randomly paired genes (blue).</p>
<span>doi:10.1371/journal.pone.0001570.g003</span></div><a id="article1.body1.sec2.sec7.p2" name="article1.body1.sec2.sec7.p2"></a><p>The OPI analysis was applied iteratively using this electronic knowledgebase, leading to another 58 non-redundant, statistically significant clusters (making 156 OPI clusters total when including the previous 98 clusters). Thus, groups of genes specific to parasite-specific processes such as invasion, or sporozoite development were identified (<a href="#pone-0001570-g001">Figure 1</a>). In many cases this approach produced better functional enrichments (<em>p</em>-value ranges from 10<sup>−91.0</sup> to 10<sup>−8.3</sup>) than with the ontology-based method. For example, there was a high degree of co-expression among genes mentioned in a review of the process by which the parasite invades new red cells <a href="#pone.0001570-Cowman1">[28]</a>. When these genes were used as seeds, the resulting OPI cluster contained 62 of the 81 <em>P. yoelii</em> or <em>P. falciparum</em> invasion genes mentioned in this review (GO:PM16497586, 134 genes, <em>p</em> = 10<sup>−91.0</sup>). Thus, there is a high probability that the remaining 72 uncharacterized genes in the cluster are also involved in invasion. In fact many, of these literature-driven predictions can be verified by independent protein network analysis as is discussed later.</p>
<a id="article1.body1.sec2.sec7.p3" name="article1.body1.sec2.sec7.p3"></a><p>The malaria literature database also revealed connections between proteins studied in the related apicomplexan parasite, <em>Toxoplasma gondii.</em> During invasion, apicomplexan parasites enter into a host cell by attaching to and then creating an invagination in the host cell plasma membrane. Interactions between parasite and host plasma membranes occur in the form of a ring-shaped moving junction that begins at the anterior end of the parasite and then migrates to the posterior end. The proteins are initially localized to the rhoptry neck, a subcellular organelle located at the apical end of the parasite. There were seven proteins, four from <em>P. yoelii</em> and three from <em>P. falciparum</em>, discovered in <em>T. gondii</em> that are believed to be involved in this process. All seven were found in a group of seven genes (GO:PM16244709), most of which are unannotated in <em>P. falciparum</em> <a href="#pone.0001570-Alexander1">[29]</a>.</p>


<h4>Confirmation in genetic diversity data</h4>
<a id="article1.body1.sec2.sec8.p1" name="article1.body1.sec2.sec8.p1"></a><p>Previous work has shown that genes with known roles in immune evasion and host parasite interactions or which are surface-exposed have high rates of genetic variability while genes with known housekeeping function across are more conserved within and across <em>Plasmodium</em> species <a href="#pone.0001570-Hall1">[7]</a>, <a href="#pone.0001570-Volkman1">[30]</a>, <a href="#pone.0001570-Volkman2">[31]</a>, <a href="#pone.0001570-Kidgell1">[32]</a>, <a href="#pone.0001570-Jeffares1">[33]</a>. We also found this to be true for uncharacterized genes that are co-expressed with such characterized genes. The invasion group mentioned above (GO:PM16497586) contains some of the most polymorphic genes in the <em>P. falciparum</em> genome <a href="#pone.0001570-Kidgell1">[32]</a> including the merozoite surface protein 1 (PFI1475w) in which hundreds of nonsynonymous SNPs have been identified <a href="#pone.0001570-Volkman2">[31]</a>. Inspection (see <a href="http://www.plasmodb.org">http://www.plasmodb.org</a> and <a href="#pone.0001570-Volkman2">[31]</a>, <a href="#pone.0001570-Jeffares1">[33]</a>) reveals that most characterized genes (with a few exceptions) in the cluster are very polymorphic as are the uncharacterized ones. Of the uncharacterized proteins PFL2505c has at least 13 different nonsynonymous mutations across 12 <em>P. falciparum</em> strains and <em>P. reichenowi</em> relative to the sequenced strain, 3D7, PF14_0495 has 35, PF08_0035 has 16 and PFF0675c, a myosin-like protein has at least 24. In contrast, the uncharacterized proteins that are associated with genes having a known role RNA polymerase nucleolus I activity (GO:0005730, <a href="#pone-0001570-t002">Table 2</a>) are overwhelmingly very conserved: PF11_0471 has only one nonsynonymous mutation, PF14_0734 has one, PF08_0055 has one, PF07_0121 has one, PF11_0305 has one, PF11_0090 has four and PF10_0200 has five. This is also true for many of the other groups containing large numbers of housekeeping genes such as DNA replication clusters. Thus the genetic diversity data provides additional evidence for evaluation of possible functions.</p>


<h4>Independent testing using <em>P. knowlesi</em></h4>
<a id="article1.body1.sec2.sec9.p1" name="article1.body1.sec2.sec9.p1"></a><p>To empirically test our predictions we identified <em>P. knowlesi</em> orthologs of genes in the literature-derived invasion cluster. <em>P. knowlesi</em> is a simian model for human malaria and is closely related to <em>P. vivax</em>. <em>P. knowlesi</em> blood stage parasites were synchronized within the erythrocytic cycle using two independent methods and RNA was collected and hybridized to a <em>P. knowlesi</em> microarray. We expected that if the predictions from the <em>P. yoelii</em> and <em>P. falciparum</em> data were valid, we would also see upregulation of uncharacterized genes in the schizonts in <em>P. knowlesi</em>. 67 predictions were mapped to 41 unique <em>P. knowlesi</em> orthologs available on the array, almost all showed substantial upregulation (&gt;5×) in the late schizont stage (<a href="#pone-0001570-g004">Figure 4</a>). The <em>P. knowlesi</em> ortholog of PFD1130w showed a 100-fold upregulation in the late schizont relative to the trophozoite stage. These data show that our predictions are likely applicable to other <em>Plasmodium</em> species as well as to other apicomplexan parasites.</p>
<div class="figure" id="pone-0001570-g004"><div class="img"><a name="pone-0001570-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001570.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001570" data-uri="info:doi/10.1371/journal.pone.0001570.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001570.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001570.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001570.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001570.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001570.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001570.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001570.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001570.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001570.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>Late schizont upregulation of <em>P. knowlesi</em> genes that are orthologs of uncharacterized genes in <em>P. falciparum</em> predicted to be involved in invasion.</span></strong></p><span>doi:10.1371/journal.pone.0001570.g004</span></div>

<h4>Deriving support from two-hybrid interaction data</h4>
<a id="article1.body1.sec2.sec10.p1" name="article1.body1.sec2.sec10.p1"></a><p>Further support for expression-based function predictions were obtained from two-hybrid data from a large scale study of interactions between erythrocytic stage proteins <a href="#pone.0001570-LaCount1">[34]</a>. It had previously been noted that there was enrichment for two-hybrid interactions amongst genes expressed late in the erythrocytic cycle <a href="#pone.0001570-LaCount1">[34]</a>. Our new data and analysis method provided additional support for the likely significance of the two hybrid interactions in 57 clusters (<em>p</em>≤0.01) (<a href="#pone-0001570-g001">Figure 1</a>, <a href="#pone.0001570.s001">Table S1</a>). As expected, there was an abundance of interactions in the invasion cluster mentioned previously. Two hybrid interaction data were available for 35 of the 71 <em>P. falciparum</em> genes in the invasion cluster; 31 of these form 83 direct or indirect interactions pairs among themselves (<a href="#pone-0001570-g005">Figure 5A</a>). The probability of observing this enrichment by chance is &lt;10<sup>−6</sup>. A group containing many of the genes in the cellular carbohydrate catabolism pathway (GO:0044275) also had more two-hybrid interactions than could be expected by chance alone (45 <em>P. f.</em> genes by expression, 29 proteins with two hybrid data, ten of which interact with other members of the group (<em>p</em> = 0.003)). This group included MAL8P1.17, a protein disulfide isomerase that interacts with enolase (PF10_0155), glyceraldehyde-3-phosphate dehydrogenase (PF14_0598) and phosphoglycerate kinase (PFI1105w) (<a href="#pone-0001570-g005">Figure 5B</a>). The cluster of 99 <em>P. falciparum</em> genes enriched for structural constituents of the ribosome had 54 genes with protein-protein interaction data within the group and 35 of these have interactions with other members of the group (<em>p</em> = 0.003) (<a href="#pone-0001570-g005">Figure 5C</a>). In a fourth example, we considered an expression cluster of 100 genes, several with RNA methyltransferase activity (<a href="#pone-0001570-g005">Figure 5D</a>). The combined two-hybrid and expression data indicated that most of the proteins in this group were very likely involved in RNA processing. For example, the uncharacterized gene PF07_0121 had a yeast ortholog factor required for a late assembly step of the 60S subunit; PF11_0305, another uncharacterized protein, had a yeast ortholog which is a constituent of the 66S pre-ribosome particle; and PF10_0200 has a yeast ortholog, YNL132w (See <a href="#pone-0001570-t002">Table 2</a>), which also interacts with many nucleolus ribosome assembly proteins (<a href="http://www.yeastgenome.org/">http://www.yeastgenome.org/</a>, <a href="#pone.0001570-Collins1">[35]</a>). MAL13P1.14 encodes an ATP-dependent DEAD box helicase and interacts with four other members of the expression set, including PFL0815w (a DNA-binding chaperone), and two uncharacterized proteins, PF07_0106 and PFA0410w. The yeast orthologs of both MAL13P1.14 and PFL0815w are co-expressed along with other RNA modification enzymes and the yeast ortholog for MAL13P1.14 is the U3 snoRNP protein, a nucleolar protein involved in mRNA processing. Although many of the proteins were uncharacterized, this is probably because many of the genes were also uncharacterized in yeast or humans. These genes would be less likely to be good drug targets due to their cross-species conservation. The two-hybrid data includes few proteins that are expressed in the sexual or insect stages because the library was derived from blood stage material. Thus, little two-hybrid support was found for genes that have roles in sexual development, sporozoite function or other processes occurring in the non-erythrocytic stages (<a href="#pone-0001570-g001">Figure 1</a>). In all, protein-protein interaction analysis provided support for 970 OPI function assignments, 78% of which belong to 229 <em>P. falciparum</em> proteins not yet captured by existing manual annotations. Together with the previous cross-species co-expression evidence, we identified a total of 5,471 supported function predictions, including new functional characterizations for 926 malaria genes (<a href="#pone.0001570.s002">Table S2</a>).</p>
<div class="figure" id="pone-0001570-g005"><div class="img"><a name="pone-0001570-g005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001570.g005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001570" data-uri="info:doi/10.1371/journal.pone.0001570.g005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001570.g005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001570.g005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001570.g005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001570.g005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001570.g005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001570.g005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001570.g005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001570.g005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001570.g005.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 5.  <span>Two-hybrid interaction networks among genes that are co-expressed.</span></strong></p><a id="article1.body1.sec2.sec10.fig1.caption1.p1" name="article1.body1.sec2.sec10.fig1.caption1.p1"></a><p>A. Invasion; B. Cellular carbohydrate catabolism; C. Cytosolic ribosome; D. RNA methyltransferase activity.</p>
<span>doi:10.1371/journal.pone.0001570.g005</span></div>

<h4>Upregulation of type II fatty acid biosynthesis in liver stages</h4>
<a id="article1.body1.sec2.sec11.p1" name="article1.body1.sec2.sec11.p1"></a><p>One other cluster of particular interest to those interested in drug development contained a number of the genes, which were previously mentioned as upregulated in liver stage parasites relative to blood stages some of which were mentioned in a manuscript <a href="#pone.0001570-Surolia1">[36]</a> describing type II fatty acid biosynthesis (GO:PM15315475, 19 genes, <em>p</em> = 10<sup>−17.7</sup>). Most of these genes in these clusters are expressed at very low levels in <em>P. falciparum</em> and <em>P. yoelii</em> blood stages but increase substantially in ookinetes, gametocytes and in particular in liver stages (<a href="#pone-0001570-g001">Figure 1</a>). The group includes PF11_0256, the pyruvate dehydrogenase alpha subunit 1; PF14_0441, the pyruvate dehydrogenase E1 beta subunit; PF08_0066, dihydrolipoamide dehydrogenase; PF10_0407, dihydrolipoamide acetyltransferase and PFF0730c, enoyl-acyl carrier reductase (ENR, the presumed target of the <em>Mycobacterium</em> tuberculosis drug, isoniazid <a href="#pone.0001570-Banerjee1">[37]</a> and the herbicide, triclosan) as well as their <em>P. yoelii</em> orthologs. All of these proteins have an apicoplast targeting signal and most are involved in fatty acid biosynthesis. In contrast to replication and schizogony in blood stages where a single parasite gives rise to 18 progeny, in the liver a single parasite replicates within the hepatocyte to form thousands of progeny. It is likely that there are additional needs for fatty acids that can be used to produce the membranes that will eventually surround each parasite for the thousands of new nuclei that are formed. These data suggest that fatty acid biosynthesis inhibitors may be useful against liver stage parasites. The relatively low levels of PfENR transcript in blood stage parasites relative to liver and sexual stage parasites is somewhat surprising given that inhibitors of the PfENR, such as triclosan, can cure blood stage infections in rodent parasites <a href="#pone.0001570-Surolia2">[38]</a>. However, triclosan is slightly more active against parasites than against the enzyme suggesting possible off-targets effects. This may indicate that triclosan could be a more potent inhibitor of liver stage development. Given the effectiveness of the irradiated sporozoite vaccine in inducing immunity, it seems likely that drugs that cure early liver stage infections could also result in increased immunity. Furthermore, with the exception of primaquine, there are few compounds that can clear liver stage malaria infections.</p>
<a id="article1.body1.sec2.sec11.p2" name="article1.body1.sec2.sec11.p2"></a><p>It has been proposed that the protein network structure in <em>P. falciparum</em> differs from those found in other organisms <a href="#pone.0001570-Suthram1">[39]</a> based on analysis of protein-protein interactions <a href="#pone.0001570-LaCount1">[34]</a>. Although only 1,682 <em>P. falciparum</em> proteins have orthologs in <em>S. cerevisiae</em> or humans, proteins with orthologs are not unexpectedly concentrated in functional clusters such as mitochondrial matrix, translation, glucose metabolism, where nearly all the proteins have <em>S. cerevisiae</em> or human orthologs (<a href="#pone-0001570-g001">Figure 1</a>). In the group enriched for ribonucleoprotein complex (GO:0030529), 53 of the 57 genes have yeast or human orthologs. In contrast to those having to do with metabolism, genes that generally lack any type of ortholog populate clusters that contain genes involved in parasitic function such invasion or gliding motility <a href="#pone.0001570-Kappe2">[40]</a>. One such cluster was enriched for genes localized to the rhoptry (combined literature, GO:PM15049814). This group contained all 11 rhoptry genes in a group of 20 (<em>p</em> = 10<sup>−28.6</sup>). Of these 20, none had a yeast or human orthologs. These data suggest that any small molecule targeting proteins in this cluster would be unlikely to have off-target effects. In another interesting example, a parasite-specific cluster of 65 genes contains 21 of the 30 genes that were described as sporozoite-specific (<em>p</em> = 10<sup>−34.8</sup>) <a href="#pone.0001570-Kaiser1">[41]</a>. Of the twelve proteins in the group with homologs in yeast or in humans, only succinate dehydrogenase and NAD(P)H-dependent glutamate synthase have clear cellular roles. Using annotation from a previously published analysis of expression in <em>P. berghei</em> <a href="#pone.0001570-Raibaud1">[42]</a>, a cluster (GO:PM16908078) that contained 19 of 31 genes mentioned in the previous work was identified (61 genes, <em>p</em> = 10<sup>−30.8</sup>). Only 11 genes had yeast orthologs and in yeast the roles were relatively uncharacterized as well. While one possibility is that these uncharacterized genes are involved in immune evasion, or speciation, most of the genes in this study are conserved across malaria parasite species and most also have homologs in related apicomplexan parasites.</p>

</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p>Existing studies have provided examples where gene functions can be correctly predicted using a “guilt-by-association” co-expression analysis, or through protein-protein interaction network analysis. Compared to co-expression analysis within a single species, identification of evolutionarily conserved cross-species co-expression patterns provides reliable functional information complementary to sequence information. Multiple species data is also much less likely to be affected by statistical randomness in the dataset or by the complexity of transcription programs <a href="#pone.0001570-Zhou2">[43]</a>, <a href="#pone.0001570-Bergmann1">[44]</a>, <a href="#pone.0001570-Stuart1">[45]</a>. Although cross-species expression analysis can also be applied to a few microarray experiments or sample groups <a href="#pone.0001570-SweetCordero1">[46]</a>, <a href="#pone.0001570-McCarroll1">[47]</a>, a large panel of diversified biological conditions such as those given here provides much higher resolution and offers a level of detail that cannot be observed in smaller datasets. Since orthologous gene mapping is more likely to be a many-to-many relationship instead of one-to-one mapping, we purposefully avoided using “metagenes” <a href="#pone.0001570-Stuart1">[45]</a>. Instead both <em>P. falciparum</em> and <em>P. yoelii</em> genes were retained in our expression matrix and their orthologous relationships were evaluated through unbiased statistical permutation tests. New methods have also been proposed to exploit protein-protein interaction data for functional predictions <a href="#pone.0001570-Vazquez1">[48]</a>. Here we use a two-hybrid data set to evaluate the statistical significance of the co-expression clusters, based on <em>p</em>-values estimated from simple node and edge counting during sampling simulations. This straightforward approach is unlikely to be controversial compared to some existing methods <a href="#pone.0001570-Murali1">[49]</a>.</p>
<a id="article1.body1.sec3.p2" name="article1.body1.sec3.p2"></a><p>The comprehensive data collection presented here shows that the function of many uncharacterized proteins encoded by malaria parasites can likely be predicted based on expression patterns. To illustrate the validity of the approach we have discussed many well-characterized processes involving genes with known roles in human, parasites and yeast, as well as others where the function can be easily inferred. However, based on the accuracy of these predictions for these well-studied cellular processes such as ribosome biogenesis, it is clear that the same analysis can predict which uncharacterized proteins that are likely to have roles in less understood processes such as sporozoite function, gliding motility, or ookinete function. For example, it is very likely that the RNA binding protein PFD0825c is involved in early liver stage function given that its expression pattern mimics <em>pbs36</em>, a <em>P. berghei</em> protein critical for early liver stage development <a href="#pone.0001570-Douradinha1">[50]</a>. Similarly, we predict that the protein encoded by the hypothetical gene, PFI0210c, is likely involved in late stage liver development and possibly in type II fatty acid biosynthesis. The comprehensiveness of the data in this study enhances the specificity of the predictions over previous studies. While these remain only predictions, a significant portion has additional co-expression support from their yeast and human counterparts, or from independent protein network studies. It is our hope that they will serve as a basis for evaluating other types of systematic data that are produced in large-scale experiments, such as mass-spectrometric analysis of protein complexes and provide a complement to orthology mapping from model organisms. Implicit in these data is the assumption that genes that are conserved across species will be playing similar cellular roles in different malaria parasite species. Our data supports this assumption many times over, thus, if a gene is upregulated in <em>P. yoelii</em> liver stages, it seems likely that its <em>P. falciparum</em> ortholog will also be upregulated in liver stages. These data also show 549 of the uncharacterized proteins in the malaria genome participate in conserved processes that are found in other eukaryotes. Knowing whether an uncharacterized protein is likely a member of the small nucleolar complex or plays a role in pathogenesis will aid researchers in making better informed decisions. These data will also be useful for predicting the functions of genes in many related apicomplexan parasites, such as <em>Toxoplasma</em>, <em>Babesia</em> or <em>Cryptosporidium</em>. Finally, it may be that these cross-species data will be useful for understanding the function of uncharacterized genes in <em>S. cerevisiae</em> and even in humans.</p>
</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4>Isolation of <em>P. yoelii</em> gametocytes</h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p>A protocol similar to that previously described <a href="#pone.0001570-Beetsma1">[51]</a> was used to isolate gametocytes. Three mice (strain: BALB/cByJ) were injected intraperitoneally with 0.1 ml of a 25 mg/ml stock solution of phenylhydrazine (phz) to increase reticulocytosis in the mice and increase parasitemia. The phz-treated mice were then injected with approximately 10<sup>7</sup> <em>P. yoelii</em> infected erythrocytes (obtained from MR4 : <em>P. yoelii</em> 17XNL (1.1)). When the parasitemia of the mice reached ~ 40% (after about 4 days), they were treated with sulfadiazine (15 mg/L) in their drinking water. This treatment kills asexual parasites and leaves gametocytes unaffected. The treatment was carried out for two days and mice were heart bled for the collection of the gametocytes. The blood from the mice was diluted in PBS and the leucocytes were removed by passing the blood suspension through a Plasmodipur filter (Euro-diagnostica). The gametocytes were purified from 49% Nycodenz/PBS-solution (v/v) gradient interfaces. The gametocytes found at the interface were collected and pelleted before RNA was extracted.</p>


<h4>Isolation of <em>P. yoelii</em> blood stages</h4>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1"></a><p>Blood stages were isolated as previously described <a href="#pone.0001570-Jongco1">[52]</a>. For schizonts blood from three infected mice (50% parasitemia) was collected and placed in complete culture medium (RPMI 1640 medium containing heat-inactivated fetal calf serum (20%), neomycin (50 IU/ml) and HEPES (25mM) containing 40 IU of heparin). The blood was incubated for 12 hours at 37°C and the schizonts were collected from the 60% Nycodenz/PBS gradient interface. These schizonts were pelleted for RNA extraction.</p>


<h4>Purification and preparation of total RNA from midgut and salivary gland sporozoites</h4>
<a id="article1.body1.sec4.sec3.p1" name="article1.body1.sec4.sec3.p1"></a><p>The midgut and salivary gland sporozoites were obtained from infected female <em>Anopheles</em> mosquitoes on days 9 and 14 respectively post infection. The midgut and salivary gland tissues were disrupted by hand grinding and the sporozoites were isolated and counted in a hemocytometer. Sporozoite populations were purified as described previously <a href="#pone.0001570-Mack1">[53]</a>. In brief, 3 g of Diethylaminoethyl Cellulose (DE 52, Whatman) was incubated in an equilibration buffer containing 8.8 g of Tris-HCl, 5 g of NaH<sub>2</sub>PO<sub>4</sub>H<sub>2</sub>O, 5 g of NaCl, 18 g of glucose per liter, adjusted to pH 8.2. The preswollen DE-52 was transferred into 10 ml plastic syringe plugged with glass wool at the bottom. Ten million sporozoites were mixed with equal volume of 35% BSA and incubated on ice for 30 min. Following incubation, the sporozoites were passed through the DE-52 column. The sporozoites were eluted in 10 ml of equilibration buffer and collected as 500 µl fractions. The fractions having highest recovery of sporozoites were pooled and counted. The purification yielded 2.5–3.0 million highly purified sporozoites from each population. The sporozoites were washed twice with ice-cold PBS prior to RNA isolation. Total RNA was isolated from sporozoites using micro to midi total RNA isolation kit (Invitrogen) following manufacturer's instructions. The yields of RNA varied between 20–30 ng for each sample.</p>


<h4>Isolation of total RNA from <em>P. yoelii</em> infected liver</h4>
<a id="article1.body1.sec4.sec4.p1" name="article1.body1.sec4.sec4.p1"></a><p>Balb/c mice were infected with 6 million <em>P. yoelii</em> sporozoites by an intravenous route. At the end of 36 hours and 40 hours post infection the mice were anesthetized and the livers were dissected out. The livers were washed two times in ice-cold PBS buffer before proceeding with the RNA extraction.</p>
<a id="article1.body1.sec4.sec4.p2" name="article1.body1.sec4.sec4.p2"></a><p>All reagents used for RNA isolation were procured from Sigma. Each liver sample was homogenized in 4 ml of denaturation buffer containing 4 M guanidium thiocyanate, 25 mM sodium citrate pH 7, 0.5% N-lauroyl sarcosine and 0.1M β-mercaptoethanol. Total RNA was extracted from six hundred microliter aliquots of each liver homogenate using the phenol chloroform method. In brief, 60 µl of 2 M sodium acetate pH 4 was added to 600 µl of liver homogenate followed by addition of 600 µl of buffer saturated phenol and 150 µl of chloroform-isoamyl alcohol. The samples were incubated on ice for 15 min and centrifuged at 10,000×g for 20 min at 4°C. Following centrifugation, 400 µl of the upper aqueous phase was transferred into a clean microfuge tube and RNA was precipitated by adding equal volumes of ice cold isopropanol. The samples were incubated at −20°C for one hour and centrifuged at 10,000×g for 15 min. The RNA pellet was washed once with cold 75% ethanol, air-dried and solubilized in 150 µl of RNAse free water. The concentration and purity of RNA were estimated spectrophotometrically.</p>


<h4>cDNA subtraction of liver stages</h4>
<a id="article1.body1.sec4.sec5.p1" name="article1.body1.sec4.sec5.p1"></a><p>Subtraction of cDNA was performed as previously described <a href="#pone.0001570-Camerer1">[54]</a>. Briefly the subtractor (uninfected mouse liver) mRNA was isolated from the total RNA using magnetic Dynabeads Oligo (dT)<sub>25</sub> (Invitrogen) and directly converted to the complementary first strand cDNA, leading to immobilized cDNA on Dynabeads (DB) after melting away the mRNA. The target (mouse liver infected with <em>P. yoelii</em>) mRNA was isolated from total RNA using DB. The target mRNA was hybridized to the immobilized subtractor cDNA at 65–68°C for 20–24 hours and the common transcripts were removed by collecting the beads with cDNA/mRNA hybrids. Bead-coupled cDNA was regenerated and two more rounds of hybridization were performed using the same subtractor cDNA. After the final hybridization step, the supernatant of DB was enriched for target-specific mRNAs (<em>P. yoelii</em> liver stages). These transcripts were then captured with fresh DB and double amplified for microarray analysis.</p>


<h4>Isolation of <em>P. falciparum</em> zygotes</h4>
<a id="article1.body1.sec4.sec6.p1" name="article1.body1.sec4.sec6.p1"></a><p>Mature gametocytes of <em>P. falciparum</em> NF54 isolate cultured <em>in vitro</em> <a href="#pone.0001570-Ifediba1">[55]</a>. Gametogenesis was induced by incubating mature gametocytes in exflagellation-inducing medium <a href="#pone.0001570-Kumar2">[56]</a> at RT for 30 min. Zygote formation was achieved by incubation of emerged gametes in the medium for 4 hours at RT. Zygotes and gametes were enriched by centrifugation over a discontinuous 6/11/16% Nycodenz gradient and harvested from the 6–11% interface. Zygotes and gametes were washed 3 times with incomplete medium and counted in a hemocytometer. Smears were prepared, Giemsa-stained, and examined under light microscopy.</p>


<h4>Isolation of <em>P. falciparum</em> ookinetes</h4>
<a id="article1.body1.sec4.sec7.p1" name="article1.body1.sec4.sec7.p1"></a><p>Mature gametocytes from an 18 day old gametocyte culture were diluted 1:10 with complete ookinete medium <a href="#pone.0001570-AlOlayan1">[57]</a> supplemented with 15% FBS and incubated for 24 hours at RT with rocking. Ookinetes were purified by treatment with Lympholyte (Cedarlane Lab, Ontario, Canada), followed by centrifugation over a discontinuous 35/50/65/80% Percoll gradient <a href="#pone.0001570-Kariuki1">[58]</a>. Ookinetes were collected from the 35–50% interface and washed 3 times with incomplete medium, then counted in a hemocytometer. Smears were prepared, Giemsa-stained, and examined under light microscopy.</p>


<h4>Collection of <em>P. knowlesi</em> erythocytic expression data</h4>
<a id="article1.body1.sec4.sec8.p1" name="article1.body1.sec4.sec8.p1"></a><p><em>P. knowlesi</em> strain H1 was synchronized by two different methods, Percoll 60% <a href="#pone.0001570-Trottein1">[21]</a> and Sorbitol synchronization <a href="#pone.0001570-Lambros1">[59]</a>. Seven time points were collected every 4 hours for 28 hours throughout the <em>P. knowlesi</em> intra-erythrocytic cell cycle. For each time point, RNA was prepared from one 25ml culture flask with an average of 2% parasitemia and 4% hematocrit. To purify tightly synchronous schizont stages, cells were synchronized at 8 hour intervals by Percoll gradient separation for two consecutive cycles. The first time point was collected after 4 hours of incubation following the second synchronization cycle. For sorbitol synchronization, the cells were pelleted and resuspended in 5% sorbitol for 10 min, then washed and resuspended in complete RPMI media. Two successive cycles of synchronization resulted in tightly synchronous parasites. The first time point was collected after 4 hours of incubation following the second sorbitol treatment. For each time point, the cells were pelleted and resuspended in 5 volumes of Trizol. RNA was purified by Phenol extraction and purified using Qiagen RNeasy kit. RNA was amplified using the Affymetrix Genechip IVT labeling kit. 20 µg of amplified cRNA was hybridized to the Pftiling microarray for 14 hours. The gene chips were washed and scanned using standard Affymetrix protocols.</p>


<h4>RNA extraction and preparation of cRNA hybridizations on array</h4>
<a id="article1.body1.sec4.sec9.p1" name="article1.body1.sec4.sec9.p1"></a><p>RNA extraction was performed as previously described <a href="#pone.0001570-LeRoch1">[9]</a>. Eight micrograms of total RNA was used for cDNA synthesis. An oligo dT primer containing a phage T7 promoter at its 5′ end was used to prime the cDNA synthesis reaction. A second strand of cDNA was then synthesized and used as a template for <em>in vitro</em> transcription in the presence of biotinylated ribonucleotide (Enzo). The labeled cRNA was then fragmented, hybridized to the array, and stained with a streptavidin phycoerythrin conjugate. Hybridizations were carried out with 15 µg of fragmented cRNA at 45°C for 16 hours, then the hybridization solution was removed and the arrays were stained and washed following Affymetrix protocols. Arrays were scanned with an emission wavelength of 560 nm at 3 µm resolution using a confocal scanner (Affymetrix), and the signal intensity for each sequence feature on the array was determined using the 70th percentile method in Microarray Suite 5 (Affymetrix).</p>


<h4>Double amplification methods</h4>
<a id="article1.body1.sec4.sec10.p1" name="article1.body1.sec4.sec10.p1"></a><p><em>P. yoelii</em> sporozoite RNA (initial concentration of 100ng) was subjected to a double amplification step, using a modified Eberwine protocol, to obtain eight micrograms of RNA. After the first cDNA synthesis, using an oligo dT primer containing a phage T7 promoter at its 5′ end which was utilized to prime the cDNA synthesis reaction, we used T7-in vitro transcription (Ambion MEGAscript Kit) following the manufacturer's protocol. The labeled cRNA was then fragmented, hybridized to the array, and stained with a streptavidin phycoerythrin conjugate. Hybridizations were carried out with 15 µg of fragmented cRNA, at 45°C for 16 hours, then the hybridization solution was removed and the arrays were stained and washed following the Affymetrix protocols. Arrays were scanned with an emission wavelength of 560nm at 3 µm resolution using a confocal scanner (Affymetrix), and the signal intensity for each sequence feature on the array was determined using the 70th percentile.</p>


<h4>Malaria gene expression datasets</h4>
<a id="article1.body1.sec4.sec11.p1" name="article1.body1.sec4.sec11.p1"></a><p>The TSRI Malaria custom array (scrMalaria) contains probes for 5,159 <em>P. falciparum</em> genes and 5,521 <em>P. yoelii</em> genes. The <em>P. yoelii</em> probes were designed to hybridize to predicted genes from <em>P. yoelii</em> 17XNL line clone 1.1 genome sequence <a href="#pone.0001570-Carlton1">[5]</a>. The array was used previously to systematically profile the <em>P. falciparum</em> life cycle. Previous studies included the hybridization of 38 <em>P. falciparum</em> samples, including erythrocyte stages, detailed gametocyte time courses, and one sporozoite stage <a href="#pone.0001570-LeRoch1">[9]</a>, <a href="#pone.0001570-Young1">[60]</a>. This study adds 2 <em>P. falciparum</em> samples, including zygote and ookinete stages, and 14 <em>P. yoelii</em> samples, including three blood stages (one schizont and two mixed stages), four gametocyte samples, five sporozoite and two liver stage (36 and 40 hours post infection). As with the <em>P. falciparum</em> design, there are a variable number of probes per gene and thus higher confidence data is obtained for longer genes. All array measurements were processed using match-only integral distribution (MOID) algorithm as described previously <a href="#pone.0001570-Young1">[60]</a>. A quantile normalization method <a href="#pone.0001570-Bolstad1">[61]</a> was then applied to the <em>P. yoelii</em> and <em>P. falciparum</em> data independently, in order to minimize the variation that resulted from different hybridizations (collected over a five year period) having different dynamic ranges. Genes with six or more probes, maximum expression level of 50 units in one sample, a fold change greater than two within a dataset were considered non-trivial for correlation calculations. This includes 3,690 <em>P. falciparum</em> genes and 2,902 <em>P. yoelii</em> genes. All malaria expression data are available through our web site (<a href="http://carrier.gnf.org/publications/Py">http://carrier.gnf.org/publications/Py</a>) and PlasmoDB (<a href="http://www.plasmoDB.org">http://www.plasmoDB.org</a>).</p>


<h4>Yeast gene expression datasets</h4>
<a id="article1.body1.sec4.sec12.p1" name="article1.body1.sec4.sec12.p1"></a><p>The yeast expression dataset consists of 74 samples accumulated in GNF's in-house yeast gene chip database, many of which have not been published before. The set includes data from 62 yeast cell cultures that had been treated with small molecules (latrunculin, itraconazole, ketoconazole, etc.), exposed to different growth conditions (UV treatment, heat shock, cold shock, low salt, high salt media) <a href="#pone.0001570-Oshiro1">[62]</a>, <a href="#pone.0001570-Washburn1">[63]</a>, or synchronized within the cell cycle (sampled every 10 min). All samples were profiled using the Affymetrix S98 array that consists of 6,135 yeast genes. All array measurements were processed and normalized using Affymetrix MAS4 algorithm, 4,515 unique gene entries with expression fold change greater than 1.5 and standard deviation greater than 40 were retained. This dataset, as well as detailed sample descripftions, is available through our web site (<a href="http://carrier.gnf.org/publications/Py">http://carrier.gnf.org/publications/Py</a>).</p>


<h4>Human gene expression datasets</h4>
<a id="article1.body1.sec4.sec13.p1" name="article1.body1.sec4.sec13.p1"></a><p>The human tissue expression dataset is distributed by GNF's SymAtlas web site (<a href="http://symatlas.gnf.org">http://symatlas.gnf.org</a>) <a href="#pone.0001570-Su1">[25]</a>. This dataset consists of 79 human tissues in duplicates, measured using Affymetrix U133A array that consists of 22,215 probe sets. All array measurements were processed and normalized using Affymetrix MAS5 algorithm; 13,499 unique gene entries with expression level both above noise and a fold change greater than 1.5 across all tissues were retained.</p>


<h4>Ortholog pairing and malaria expression extrapolation</h4>
<a id="article1.body1.sec4.sec14.p1" name="article1.body1.sec4.sec14.p1"></a><p>Orthologous groups of proteins defined by OrthoMCL database (version 2.0) were used in this study <a href="#pone.0001570-Chen1">[64]</a>. OrthoMCL maps proteins across multiple species, including <em>P. falciparum</em>, <em>P. yoelii</em>, <em>P. berghei</em>, <em>P. chabaudi</em>, <em>S. cerevisiae</em>, <em>H. sapiens</em>, etc. Within each ortholog group, all <em>P. falciparum</em> protein sequences and <em>P. yoelii</em> protein sequences pairs are first pair-wise aligned using BLAST. Then for a given <em>P. falciparum</em> protein, the <em>P. yoelii</em> ortholog that can be found in the malaria array (≥6 probes) and provides the best alignment score is selected as the best match. The same matching process was applied to identify the best <em>P. falciparum</em> match for each <em>P. yoelii</em> protein. We also took into account 3,969 best reciprocal BLAST match pairs between the two species identified using E-value≤10<sup>−2</sup>, as well as additional sequence alignment pairs of E-value≤10<sup>−6</sup>. Best orthologs were assigned to 4,469 <em>P. falciparum</em> genes and 5,258 <em>P. yoelii</em> genes.</p>
<a id="article1.body1.sec4.sec14.p2" name="article1.body1.sec4.sec14.p2"></a><p>We aimed to construct a mostly complete malaria expression matrix, i.e., the rows comprise all the differentially expressed <em>P. falciparum</em> and <em>P. yoelii</em> genes, and the columns are the combination of all parasite life stages profiled in either species (54 samples in total). For each <em>P. falciparum</em> gene, its expression values in the 14 <em>P. yoelii</em> samples were extrapolated by copying the expression vector of its <em>P. yoelii</em> best ortholog match. For each <em>P. yoelii</em> gene, its expression values in the 40 <em>P. falciparum</em> samples were copied from the expression vector of its <em>P. falciparum</em> best ortholog match. The expression extrapolation process resulted in a final matrix of 6,592 malaria genes across all 54 parasite samples. Notice that a reciprocal best match protein pairs will lead to two identical expression vectors, therefore such gene expression dependency will need to be carefully modeled in the OPI permutation testing described later.</p>


<h4>Derivation of P. yoelii Gene Ontology database</h4>
<a id="article1.body1.sec4.sec15.p1" name="article1.body1.sec4.sec15.p1"></a><p>Gene annotations for <em>P. falciparum</em> were downloaded from Gene Ontology consortium (<a href="http://www.geneontology.org">http://www.geneontology.org</a>). Additional annotations were downloaded from PlasmoDB (<a href="http://www.plasmodb.org">http://www.plasmodb.org</a>) and combined. GO assignments for <em>P. yoelii</em> genes were derived by inheriting all GO terms assigned to its <em>P. falciparum</em> orthologs, which were defined according to OrthoMCL database. As the result 3,763 <em>P. yoelii</em> genes have assignments in 2,889 GO categories.</p>


<h4>Protein network analysis</h4>
<a id="article1.body1.sec4.sec16.p1" name="article1.body1.sec4.sec16.p1"></a><p>Systematic protein-protein interaction dataset for <em>P. falciparum</em> was made available by a previous study <a href="#pone.0001570-LaCount1">[34]</a>. Protein network analysis described below was applied to the <em>P. falciparum</em> proteins within each of the 156 OPI clusters. For any given cluster of <em>N</em><sub>T</sub> <em>P. f.</em> proteins, the subset of entries that form either direct or indirect protein-protein interaction pairs among themselves was identified. An indirect interaction was defined as two proteins that interact and only interact via at least a third protein, similar to a previous network study <a href="#pone.0001570-Suthram1">[39]</a>. We focused on two characteristics of each resultant network, <em>i.e</em>., its node count <em>N</em><sub>N</sub> and edge count <em>N</em><sub>E</sub>, where each node uniquely represents a protein and each edge represents a unique interaction pair. We then randomly sampled <em>N</em><sub>T</sub> number of proteins from the total collection of 3,691 <em>P. f</em>. proteins comprising the expression matrix described above, and counted the nodes and edges in the corresponding resultant network. Such simulations were repeated at least 1000 times, and the percentage of runs that led to either a higher node count than <em>N</em><sub>N</sub> or a higher edge count than <em>N</em><sub>E</sub> gave an unbiased estimation of the <em>p</em>-value. Protein networks with a <em>p</em>-value≤0.01 are considered to show significant connectivity (<a href="#pone-0001570-g001">Figure 1</a>, <a href="#pone.0001570.s001">Table S1</a>). For such cases, protein network data provide an additional piece of evidence for the correct functional assignment based on expression clustering (<a href="#pone.0001570.s002">Table S2</a>). All statistically significant protein networks were plotted graphically and made available on our web site, together with the human and yeast orthologous evidences.</p>


<h4>An automatic pipeline for malaria literature mining</h4>
<a id="article1.body1.sec4.sec17.p1" name="article1.body1.sec4.sec17.p1"></a><p>To collect malaria functional annotations from existing literatures, we took two complementary approaches. In approach A, <em>P. falciparum</em> and <em>P. yoelii</em> locus names downloaded from PlasmoDB were used as search terms one at a time through Google Scholar beta (<a href="http://scholar.google.com">http://scholar.google.com</a>) and SCIRUS (<a href="http://www.scirus.com">http://www.scirus.com</a>) search engines. Gene/protein names or description keywords were not used in our searches, because of their high number of false positives. Not limited by the title and abstract stored in the NCBI PubMed database, both Google Scholar and SCIRUS index the full literature text, and therefore are expected to provide complementary annotations to what already exists in NCBI. Google Scholar database mostly consists of manuscripts accessible publicly via World Wide Web, while SCIRUS collects additional non-publicly accessible literature; including some essential Elsevier journals, which frequently publish research results from malaria scientific community. Each search returned a list of URLs (we kept a maximum 500) and some association summary data such as paper title, journal name, volume, and start page, etc. As multiple URLs could refer to the same publication, we mapped every URL hit by an automatic query to the PubMed database and translated the URL into a PubMed identifier whenever possible. In this way, proteins co-cited in the same paper were discovered even though they were obtained from independent searches. Developing efficient search robots in this approach was difficult, mainly due to the lack of programming interface from both search engines. We therefore had to limit our searches to <em>P. falciparum</em> and <em>P. yoelii</em> only. Searches with <em>P. falciparum</em> systematic names resulted in 337 papers and searches with the <em>P. yoelii</em> names only identified an additional 15 new papers, therefore, we anticipate relatively few papers exist that cite <em>P. berghei</em> and <em>P. chabaudi</em> genes by their locus names.</p>
<a id="article1.body1.sec4.sec17.p2" name="article1.body1.sec4.sec17.p2"></a><p>When publications are related to genome sequencing projects, reports of new technology development for malaria systems biology, or other studies irrelevant to specific parasite functional characterization, they tend to contain large number of locus identifiers but are not interesting to our study. We manually filtered out 26 such papers. In total, we identified 2,605 associations between 1,509 malaria proteins and 326 PubMed papers through approach A.</p>
<a id="article1.body1.sec4.sec17.p3" name="article1.body1.sec4.sec17.p3"></a><p>In Approach B, we first downloaded all GenBank sequences for <em>P. falciparum</em>, <em>P. yoelii</em>, <em>P. berghei</em> and <em>P. chabaudi</em> that have associations with PubMed entries. The association can be either a direct link or sometimes an indirect link derived through its link first to an NCBI Gene database entry then to PubMed database entries. A unique GenBank gi number identifies each sequence. Genomic sequence entries such as chromosome fragments or genome shotgun sequences were filtered out. Extremely long or short sequences were also discarded, so that only nucleotide sequences of length between nine and 60,000 bases and proteins sequences with lengths between three and 20,000 amino acids were retained. All the data were retrieved through NCBI eUtils programming interface. We then downloaded the complete transcriptome and proteome sequences of all four Plasmodium species from PlasmoDB, where sequences are identified by their locus names. Next, a BLAST search was conducted for each gi sequence against the corresponding transcriptome or proteome within the same species with an E-value threshold of 10<sup>−6</sup>. Given either a protein sequence alignment of at least 90% identity over 50 amino acids or a nucleotide sequence alignment of minimal 95% identity over 100 nucleotide bases, a gi number was mapped to a locus name whenever possible. Once matched, all PubMed entries associated with the gi number were automatically transferred to the locus entry. After filtering out 18 less relevant publications, we identified 3,901 associations between 2,350 malaria proteins and 1,006 PubMed papers through this approach.</p>
<a id="article1.body1.sec4.sec17.p4" name="article1.body1.sec4.sec17.p4"></a><p>Combining both approaches into an automatic literature-mining pipeline resulted in a total of 6,428 associations between 3,262 proteins/genes and 1,278 papers. The complete literature mining results are available in a searchable database on our website <a href="http://carrier.gnf.org/publications/Py">http://carrier.gnf.org/publications/Py</a>. (follow the “literature database” link, also <a href="#pone.0001570.s003">Table S3</a>, <a href="#pone.0001570.s004">S4</a>).</p>


<h4>Derivation of literature-based malaria annotations</h4>
<a id="article1.body1.sec4.sec18.p1" name="article1.body1.sec4.sec18.p1"></a><p>Our literature-mining pipeline produced custom annotations in the form of malaria protein and PubMed paper associations. For a given paper, we expand the malaria protein list by including all orthologs across four Plasmodium species as defined by OrthoMCL database. Each paper was treated as its own virtual GO category and all the malaria proteins associated were considered as members of that category. Gene Ontology database consists of three branches, i.e., biological process, molecular function, and cellular component. These literature-derived virtual GO categories did not undergo the same curation process as real GO categories and therefore best kept under a separate “literature” branch to avoid misinterpretation. Otherwise, these virtual GO categories were treated the same by the later OPI clustering algorithm. We also carried out limited manual curation in order to merge proteins of some closely related papers into a newer virtual category. The rationale is such combination typically produces more complete annotation categories and should lead to better sensitivity in the clustering analysis.</p>


<h4>Annotation-driven OPI clustering analyses</h4>
<a id="article1.body1.sec4.sec19.p1" name="article1.body1.sec4.sec19.p1"></a><p>OPI is a supervised clustering algorithm that produces the highest statistical enrichment for the known genes in an optimal co-expression cluster of a given GO category. The details of OPI algorithm, its comparison with other clustering algorithms, as well as its applications to the <em>P. falciparum</em> global gene expression analysis were described previously <a href="#pone.0001570-Zhou1">[8]</a>, <a href="#pone.0001570-Young1">[60]</a>. One may also find many applications of a similar supervised clustering algorithm called Gene Set Enrichment Analysis (GSEA) <a href="#pone.0001570-Subramanian1">[65]</a>, however, OPI is more suitable for function prediction of uncharacterized genes. In this study, clustering analysis was independently applied to the three gene expression datasets mentioned above. First, genes that did not show significant expression variation were filtered out. Then, the program iterates over all gene ontology groups containing a minimum of two known genes in the expression data matrix; this corresponds to 2,708 GO groups (including our virtual literature-derived categories) for the malaria dataset, 2,210 GO groups for the yeast dataset, and 4,428 GO groups for the human tissue dataset.</p>
<a id="article1.body1.sec4.sec19.p2" name="article1.body1.sec4.sec19.p2"></a><p>Expression vectors of genes in the expression matrix are not independent and the various categories in the GO tree are also correlated, therefore the optimal hypergeometric <em>p</em>-value produced by OPI algorithm will need to be double checked by randomly generated dataset so that dependency are modeled correctly. For yeast and human dataset, randomization testing was carried out straightforwardly by shuffling gene labels in the expression matrix as described previously <a href="#pone.0001570-Young1">[60]</a>. Permutation schema for malaria dataset, however, requires extra steps. This is because the expression extrapolation step made use of ortholog mapping information and introduced additional coupling between expression vectors. <em>P. falciparum</em> and <em>P. yoelii</em> genes were first randomly relabeled among themselves, respectively. The whole expression extrapolation process used to produce the original combined malaria dataset was then repeated, so that the ortholog correlation is retained in the resultant randomized datasets as well. For each dataset we repeated OPI analyses on 100 independently randomized expression matrix. An original OPI cluster is considered truly statistical significant only if equal or better <em>p</em>-value can be achieved by no more than 5% of the permutation runs. The percentage of genes not already annotated under the given GO category in an OPI cluster estimates the false discovery rate (FDR) of co-expression-based function prediction. OPI clusters with size greater than 500 are often associated with relatively high false discovery rate and are unsuitable for the sole purpose of gene function prediction; these clusters were filtered out despite the fact that they may still be justified by pure statistical significance. The final results of the OPI analyses included 684 clusters for yeast and 1,047 clusters for human. The list of known genes in different ontology groups can be sometimes identical or very similar, resulting in closely related OPI clusters. To ease the analysis, we only kept a total of 156 clusters for malaria dataset, where such redundancy has been removed. Among them, 58 clusters were derived from literature-based custom GO groups. If a biological process of the parasite is conserved in yeast or human, cross-species comparison of related OPI clusters enables us to find additional evidence to support newly predicted <em>Plasmodium</em> genes functions. All clusters for the three datasets and ortholog evidences are accessible on our web site (<a href="http://carrier.gnf.org/publications/Py">http://carrier.gnf.org/publications/Py</a>).</p>


<h4>Unsupervised clustering of distinctly expressed malaria genes</h4>
<a id="article1.body1.sec4.sec20.p1" name="article1.body1.sec4.sec20.p1"></a><p>For all 6,592 non-trivially expressed <em>P. falciparum</em> and <em>P. yoelii</em> genes used for OPI analysis, 4,712 genes are members of the 156 OPI clusters, another 736 genes fall into clusters that are statistically significant (permutation <em>p</em>≤0.05), but their function predictions are not presented due to the large cluster size, and another 118 genes share similar expression profiles (Pearson correlation coefficient ≥0.8) with at least one statistically significant OPI cluster. The remaining 1026 genes do not share expression pattern with sufficient number of previous characterized genes. The hierarchical clustering algorithm was applied to each subset of genes and the results are accessible on our website (<a href="http://carrier.gnf.org/publications/Py">http://carrier.gnf.org/publications/Py</a>).</p>


<h4>Probability density distributions</h4>
<a id="article1.body1.sec4.sec21.p1" name="article1.body1.sec4.sec21.p1"></a><p>For the 3,690 <em>P. falciparum</em> genes used in the OPI analysis, 1,037 GO groups and 514 literature groups were found to contain 2 to 100 genes. The probability density distribution of two randomly selected genes was calculated by using all the 6,806,205 possible gene pairs. For the literature and GO distributions, Pearson correlation coefficients were first calculated for all gene pairs within a group. The coefficients were then weighted, such that each group contributed equally to avoid bias caused by large groups. In total, 239,577 and 139,811 gene pairs were used to derive the GO distribution and literature distribution, respectively.</p>

</div>

<div id="section5" class="section"><a id="s5" name="s5" toc="s5" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0001570"></div><a name="pone.0001570.s001" id="pone.0001570.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001570.s001">Table S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1"></a><p class="preSiDOI">The list of 156 statistically significant OPI clusters, their ortholog counts in <em>S. cerevisiae</em> and human, and their <em>P. falciparum</em> protein counts that form within-cluster networks.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001570.s001</p><a id="article1.body1.sec5.supplementary-material1.caption1.p2" name="article1.body1.sec5.supplementary-material1.caption1.p2"></a><p class="postSiDOI">(0.08 MB XLS)</p>
<a name="pone.0001570.s002" id="pone.0001570.s002"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001570.s002">Table S2. </a></strong></p><a id="article1.body1.sec5.supplementary-material2.caption1.p1" name="article1.body1.sec5.supplementary-material2.caption1.p1"></a><p class="preSiDOI">Yeast and human orthologs evidence and protein interaction evidence for assigned and predicted gene ontology memberships of malaria genes.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001570.s002</p><a id="article1.body1.sec5.supplementary-material2.caption1.p2" name="article1.body1.sec5.supplementary-material2.caption1.p2"></a><p class="postSiDOI">(3.40 MB XLS)</p>
<a name="pone.0001570.s003" id="pone.0001570.s003"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001570.s003">Table S3. </a></strong></p><a id="article1.body1.sec5.supplementary-material3.caption1.p1" name="article1.body1.sec5.supplementary-material3.caption1.p1"></a><p class="preSiDOI">The complete list of malaria-related PubMed publications.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001570.s003</p><a id="article1.body1.sec5.supplementary-material3.caption1.p2" name="article1.body1.sec5.supplementary-material3.caption1.p2"></a><p class="postSiDOI">(3.20 MB XLS)</p>
<a name="pone.0001570.s004" id="pone.0001570.s004"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001570.s004">Table S4. </a></strong></p><a id="article1.body1.sec5.supplementary-material4.caption1.p1" name="article1.body1.sec5.supplementary-material4.caption1.p1"></a><p class="preSiDOI">The complete list of malaria genes cited in each malaria publication.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001570.s004</p><a id="article1.body1.sec5.supplementary-material4.caption1.p2" name="article1.body1.sec5.supplementary-material4.caption1.p2"></a><p class="postSiDOI">(0.75 MB XLS)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>The authors are grateful to support from the Novartis Research Foundation and to the Malaria Research Resource Repository (MR4) for reagents. We would like to thank Dr. Dan Carucci for providing <em>P. yoelii</em> blood stage samples. We wish to thank Matt Berriman for making <em>P. knowlesi</em> sequences available before publication of the genome sequence. Companion <a href="http://website: http://carrier.gnf.org/publications/Py">website: http://carrier.gnf.org/publications/Py</a></p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: JV EW YZ MD VR KK SW PR JY FL. Performed the experiments: DP VR KK SW PR JY KH FL. Analyzed the data: EW YZ. Contributed reagents/materials/analysis tools: JV YZ VN MD JC BZ KC. Wrote the paper: EW YZ.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0001570-Gardner1" id="pone.0001570-Gardner1"></a>Gardner MJ, Hall N, Fung E, White O, Berriman M, et al.  (2002) Genome sequence of the human malaria parasite Plasmodium falciparum. Nature  419: 498–511.  <ul class="find" data-citedArticleID="1053206" data-doi="10.1038/nature01097"><li><a href="http://dx.doi.org/10.1038/nature01097" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genome+sequence+of+the+human+malaria+parasite+Plasmodium+falciparum." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genome+sequence+of+the+human+malaria+parasite+Plasmodium+falciparum.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0001570-Sacci1" id="pone.0001570-Sacci1"></a>Sacci JB Jr, Aguiar JC, Lau AO, Hoffman SL (2002) Laser capture microdissection and molecular analysis of Plasmodium yoelii liver-stage parasites. Mol Biochem Parasitol  119: 285–289.  <ul class="find" data-citedArticleID="1053274" data-doi="10.1016/s0166-6851(01)00411-x"><li><a href="http://dx.doi.org/10.1016/s0166-6851(01)00411-x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Laser+capture+microdissection+and+molecular+analysis+of+Plasmodium+yoelii+liver-stage+parasites." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Laser+capture+microdissection+and+molecular+analysis+of+Plasmodium+yoelii+liver-stage+parasites.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0001570-Bozdech1" id="pone.0001570-Bozdech1"></a>Bozdech Z, Llinas M, Pulliam BL, Wong ED, Zhu J, et al.  (2003) The Transcriptome of the Intraerythrocytic Developmental Cycle of Plasmodium falciparum. PLoS Biol  1: E5.  <ul class="find" data-citedArticleID="1053186" data-doi="10.1371/journal.pbio.0000005"><li><a href="http://dx.doi.org/10.1371/journal.pbio.0000005" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+Transcriptome+of+the+Intraerythrocytic+Developmental+Cycle+of+Plasmodium+falciparum." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+Transcriptome+of+the+Intraerythrocytic+Developmental+Cycle+of+Plasmodium+falciparum.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0001570-Lasonder1" id="pone.0001570-Lasonder1"></a>Lasonder E, Ishihama Y, Andersen JS, Vermunt AM, Pain A, et al.  (2002) Analysis of the Plasmodium falciparum proteome by high-accuracy mass spectrometry. Nature  419: 537–542.  <ul class="find" data-citedArticleID="1053242" data-doi="10.1038/nature01111"><li><a href="http://dx.doi.org/10.1038/nature01111" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Analysis+of+the+Plasmodium+falciparum+proteome+by+high-accuracy+mass+spectrometry." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Analysis+of+the+Plasmodium+falciparum+proteome+by+high-accuracy+mass+spectrometry.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0001570-Carlton1" id="pone.0001570-Carlton1"></a>Carlton JM, Angiuoli SV, Suh BB, Kooij TW, Pertea M, et al.  (2002) Genome sequence and comparative analysis of the model rodent malaria parasite Plasmodium yoelii yoelii. Nature  419: 512–519.  <ul class="find" data-citedArticleID="1053190" data-doi="10.1038/nature01099"><li><a href="http://dx.doi.org/10.1038/nature01099" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genome+sequence+and+comparative+analysis+of+the+model+rodent+malaria+parasite+Plasmodium+yoelii+yoelii." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genome+sequence+and+comparative+analysis+of+the+model+rodent+malaria+parasite+Plasmodium+yoelii+yoelii.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0001570-Florens1" id="pone.0001570-Florens1"></a>Florens L, Washburn MP, Raine JD, Anthony RM, Grainger M, et al.  (2002) A proteomic view of the Plasmodium falciparum life cycle. Nature  419: 520–526.  <ul class="find" data-citedArticleID="1053204" data-doi="10.1038/nature01107"><li><a href="http://dx.doi.org/10.1038/nature01107" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+proteomic+view+of+the+Plasmodium+falciparum+life+cycle." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+proteomic+view+of+the+Plasmodium+falciparum+life+cycle.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0001570-Hall1" id="pone.0001570-Hall1"></a>Hall N, Karras M, Raine JD, Carlton JM, Kooij TW, et al.  (2005) A comprehensive survey of the Plasmodium life cycle by genomic, transcriptomic, and proteomic analyses. Science  307: 82–86.  <ul class="find" data-citedArticleID="1053214" data-doi="10.1126/science.1103717"><li><a href="http://dx.doi.org/10.1126/science.1103717" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+comprehensive+survey+of+the+Plasmodium+life+cycle+by+genomic%2C+transcriptomic%2C+and+proteomic+analyses." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+comprehensive+survey+of+the+Plasmodium+life+cycle+by+genomic%2C+transcriptomic%2C+and+proteomic+analyses.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0001570-Zhou1" id="pone.0001570-Zhou1"></a>Zhou Y, Young JA, Santrosyan A, Chen K, Yan FS, et al.  (2004) In silico gene function prediction using ontology-based pattern identification. Bioinformatics.   <ul class="find" data-citedArticleID="1053308" data-doi="10.1093/bioinformatics/bti111"><li><a href="http://dx.doi.org/10.1093/bioinformatics/bti111" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=In+silico+gene+function+prediction+using+ontology-based+pattern+identification." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22In+silico+gene+function+prediction+using+ontology-based+pattern+identification.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0001570-LeRoch1" id="pone.0001570-LeRoch1"></a>Le Roch KG, Zhou Y, Blair PL, Grainger M, Moch JK, et al.  (2003) Discovery of gene function by expression profiling of the malaria parasite life cycle. Science  301: 1503–1508.  <ul class="find" data-citedArticleID="1053244" data-doi="10.1126/science.1087025"><li><a href="http://dx.doi.org/10.1126/science.1087025" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Discovery+of+gene+function+by+expression+profiling+of+the+malaria+parasite+life+cycle." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Discovery+of+gene+function+by+expression+profiling+of+the+malaria+parasite+life+cycle.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0001570-Ashburner1" id="pone.0001570-Ashburner1"></a>Ashburner M, Ball CA, Blake JA, Botstein D, Butler H, et al.  (2000) Gene ontology: tool for the unification of biology. The Gene Ontology Consortium. Nat Genet  25: 25–29.  <ul class="find" data-citedArticleID="1053176"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Gene+ontology%3A+tool+for+the+unification+of+biology.+The+Gene+Ontology+Consortium.&amp;auth=&amp;atitle=Gene+ontology%3A+tool+for+the+unification+of+biology.+The+Gene+Ontology+Consortium." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Gene+ontology%3A+tool+for+the+unification+of+biology.+The+Gene+Ontology+Consortium." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Gene+ontology%3A+tool+for+the+unification+of+biology.+The+Gene+Ontology+Consortium.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0001570-Kumar1" id="pone.0001570-Kumar1"></a>Kumar KA, Garcia CR, Chandran VR, Van Rooijen N, Zhou Y, et al.  (2007) Exposure of Plasmodium sporozoites to the intracellular concentration of potassium enhances infectivity and reduces cell passage activity. Mol Biochem Parasitol  156: 32–40.  <ul class="find" data-citedArticleID="1053234" data-doi="10.1016/j.molbiopara.2007.07.004"><li><a href="http://dx.doi.org/10.1016/j.molbiopara.2007.07.004" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Exposure+of+Plasmodium+sporozoites+to+the+intracellular+concentration+of+potassium+enhances+infectivity+and+reduces+cell+passage+activity." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Exposure+of+Plasmodium+sporozoites+to+the+intracellular+concentration+of+potassium+enhances+infectivity+and+reduces+cell+passage+activity.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0001570-Kappe1" id="pone.0001570-Kappe1"></a>Kappe SH, Gardner MJ, Brown SM, Ross J, Matuschewski K, et al.  (2001) Exploring the transcriptome of the malaria sporozoite stage. Proc Natl Acad Sci U S A  98: 9895–9900.  <ul class="find" data-citedArticleID="1053224" data-doi="10.1073/pnas.171185198"><li><a href="http://dx.doi.org/10.1073/pnas.171185198" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Exploring+the+transcriptome+of+the+malaria+sporozoite+stage." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Exploring+the+transcriptome+of+the+malaria+sporozoite+stage.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0001570-Matuschewski1" id="pone.0001570-Matuschewski1"></a>Matuschewski K, Ross J, Brown SM, Kaiser K, Nussenzweig V, et al.  (2002) Infectivity-associated changes in the transcriptional repertoire of the malaria parasite sporozoite stage. J Biol Chem  277: 41948–41953.  <ul class="find" data-citedArticleID="1053256" data-doi="10.1074/jbc.m207315200"><li><a href="http://dx.doi.org/10.1074/jbc.m207315200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Infectivity-associated+changes+in+the+transcriptional+repertoire+of+the+malaria+parasite+sporozoite+stage." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Infectivity-associated+changes+in+the+transcriptional+repertoire+of+the+malaria+parasite+sporozoite+stage.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0001570-Mueller1" id="pone.0001570-Mueller1"></a>Mueller AK, Labaied M, Kappe SH, Matuschewski K (2005) Genetically modified Plasmodium parasites as a protective experimental malaria vaccine. Nature  433: 164–167.  <ul class="find" data-citedArticleID="1053260" data-doi="10.1038/nature03188"><li><a href="http://dx.doi.org/10.1038/nature03188" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genetically+modified+Plasmodium+parasites+as+a+protective+experimental+malaria+vaccine." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genetically+modified+Plasmodium+parasites+as+a+protective+experimental+malaria+vaccine.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0001570-Spielmann1" id="pone.0001570-Spielmann1"></a>Spielmann T, Fergusen DJ, Beck HP (2003) etramps, a New Plasmodium falciparum Gene Family Coding for Developmentally Regulated and Highly Charged Membrane Proteins Located at the Parasite-Host Cell Interface. Mol Biol Cell  14: 1529–1544.  <ul class="find" data-citedArticleID="1053278" data-doi="10.1091/mbc.e02-04-0240"><li><a href="http://dx.doi.org/10.1091/mbc.e02-04-0240" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=etramps%2C+a+New+Plasmodium+falciparum+Gene+Family+Coding+for+Developmentally+Regulated+and+Highly+Charged+Membrane+Proteins+Located+at+the+Parasite-Host+Cell+Interface." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22etramps%2C+a+New+Plasmodium+falciparum+Gene+Family+Coding+for+Developmentally+Regulated+and+Highly+Charged+Membrane+Proteins+Located+at+the+Parasite-Host+Cell+Interface.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0001570-Mueller2" id="pone.0001570-Mueller2"></a>Mueller AK, Camargo N, Kaiser K, Andorfer C, Frevert U, et al.  (2005) Plasmodium liver stage developmental arrest by depletion of a protein at the parasite-host interface. Proc Natl Acad Sci U S A  102: 3022–3027.  <ul class="find" data-citedArticleID="1053262" data-doi="10.1073/pnas.0408442102"><li><a href="http://dx.doi.org/10.1073/pnas.0408442102" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Plasmodium+liver+stage+developmental+arrest+by+depletion+of+a+protein+at+the+parasite-host+interface." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Plasmodium+liver+stage+developmental+arrest+by+depletion+of+a+protein+at+the+parasite-host+interface.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0001570-Mair1" id="pone.0001570-Mair1"></a>Mair GR, Braks JA, Garver LS, Wiegant JC, Hall N, et al.  (2006) Regulation of sexual development of Plasmodium by translational repression. Science  313: 667–669.  <ul class="find" data-citedArticleID="1053252" data-doi="10.1126/science.1125129"><li><a href="http://dx.doi.org/10.1126/science.1125129" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Regulation+of+sexual+development+of+Plasmodium+by+translational+repression." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Regulation+of+sexual+development+of+Plasmodium+by+translational+repression.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0001570-Dinglasan1" id="pone.0001570-Dinglasan1"></a>Dinglasan RR, Alaganan A, Ghosh AK, Saito A, van Kuppevelt TH, et al.  (2007) Plasmodium falciparum ookinetes require mosquito midgut chondroitin sulfate proteoglycans for cell invasion. Proc Natl Acad Sci U S A.   <ul class="find" data-citedArticleID="1053198" data-doi="10.1073/pnas.0706340104"><li><a href="http://dx.doi.org/10.1073/pnas.0706340104" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Plasmodium+falciparum+ookinetes+require+mosquito+midgut+chondroitin+sulfate+proteoglycans+for+cell+invasion." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Plasmodium+falciparum+ookinetes+require+mosquito+midgut+chondroitin+sulfate+proteoglycans+for+cell+invasion.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0001570-LeRoch2" id="pone.0001570-LeRoch2"></a>Le Roch KG, Johnson JR, Florens L, Zhou Y, Santrosyan A, et al.  (2004) Global analysis of transcript and protein levels across the Plasmodium falciparum life cycle. Genome Res  14: 2308–2318.  <ul class="find" data-citedArticleID="1053246" data-doi="10.1101/gr.2523904"><li><a href="http://dx.doi.org/10.1101/gr.2523904" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Global+analysis+of+transcript+and+protein+levels+across+the+Plasmodium+falciparum+life+cycle." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Global+analysis+of+transcript+and+protein+levels+across+the+Plasmodium+falciparum+life+cycle.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0001570-Vinetz1" id="pone.0001570-Vinetz1"></a>Vinetz JM, Valenzuela JG, Specht CA, Aravind L, Langer RC, et al.  (2000) Chitinases of the avian malaria parasite Plasmodium gallinaceum, a class of enzymes necessary for parasite invasion of the mosquito midgut. J Biol Chem  275: 10331–10341.  <ul class="find" data-citedArticleID="1053298" data-doi="10.1074/jbc.275.14.10331"><li><a href="http://dx.doi.org/10.1074/jbc.275.14.10331" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Chitinases+of+the+avian+malaria+parasite+Plasmodium+gallinaceum%2C+a+class+of+enzymes+necessary+for+parasite+invasion+of+the+mosquito+midgut." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Chitinases+of+the+avian+malaria+parasite+Plasmodium+gallinaceum%2C+a+class+of+enzymes+necessary+for+parasite+invasion+of+the+mosquito+midgut.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0001570-Trottein1" id="pone.0001570-Trottein1"></a>Trottein F, Triglia T, Cowman AF (1995) Molecular cloning of a gene from Plasmodium falciparum that codes for a protein sharing motifs found in adhesive molecules from mammals and plasmodia. Mol Biochem Parasitol  74: 129–141.  <ul class="find" data-citedArticleID="1053294" data-doi="10.1016/0166-6851(95)02489-1"><li><a href="http://dx.doi.org/10.1016/0166-6851(95)02489-1" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Molecular+cloning+of+a+gene+from+Plasmodium+falciparum+that+codes+for+a+protein+sharing+motifs+found+in+adhesive+molecules+from+mammals+and+plasmodia." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Molecular+cloning+of+a+gene+from+Plasmodium+falciparum+that+codes+for+a+protein+sharing+motifs+found+in+adhesive+molecules+from+mammals+and+plasmodia.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0001570-Singh1" id="pone.0001570-Singh1"></a>Singh AP, Buscaglia CA, Wang Q, Levay A, Nussenzweig DR, et al.  (2007) Plasmodium circumsporozoite protein promotes the development of the liver stages of the parasite. Cell  131: 492–504.  <ul class="find" data-citedArticleID="1053276" data-doi="10.1016/j.cell.2007.09.013"><li><a href="http://dx.doi.org/10.1016/j.cell.2007.09.013" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Plasmodium+circumsporozoite+protein+promotes+the+development+of+the+liver+stages+of+the+parasite." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Plasmodium+circumsporozoite+protein+promotes+the+development+of+the+liver+stages+of+the+parasite.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0001570-Ralph1" id="pone.0001570-Ralph1"></a>Ralph SA, van Dooren GG, Waller RF, Crawford MJ, Fraunholz MJ, et al.  (2004) Tropical infectious diseases: metabolic maps and functions of the Plasmodium falciparum apicoplast. Nat Rev Microbiol  2: 203–216.  <ul class="find" data-citedArticleID="1053272" data-doi="10.1038/nrmicro843"><li><a href="http://dx.doi.org/10.1038/nrmicro843" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Tropical+infectious+diseases%3A+metabolic+maps+and+functions+of+the+Plasmodium+falciparum+apicoplast." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Tropical+infectious+diseases%3A+metabolic+maps+and+functions+of+the+Plasmodium+falciparum+apicoplast.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0001570-Goldberg1" id="pone.0001570-Goldberg1"></a>Goldberg DE (2005) Hemoglobin degradation. Curr Top Microbiol Immunol  295: 275–291.  <ul class="find" data-citedArticleID="1053210"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Hemoglobin+degradation.&amp;auth=&amp;atitle=Hemoglobin+degradation." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Hemoglobin+degradation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Hemoglobin+degradation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0001570-Su1" id="pone.0001570-Su1"></a>Su AI, Wiltshire T, Batalov S, Lapp H, Ching KA, et al.  (2004) A gene atlas of the mouse and human protein-encoding transcriptomes. Proc Natl Acad Sci U S A  101: 6062–6067.  <ul class="find" data-citedArticleID="1053282" data-doi="10.1073/pnas.0400782101"><li><a href="http://dx.doi.org/10.1073/pnas.0400782101" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+gene+atlas+of+the+mouse+and+human+protein-encoding+transcriptomes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+gene+atlas+of+the+mouse+and+human+protein-encoding+transcriptomes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0001570-Gerbasi1" id="pone.0001570-Gerbasi1"></a>Gerbasi VR, Weaver CM, Hill S, Friedman DB, Link AJ (2004) Yeast Asc1p and mammalian RACK1 are functionally orthologous core 40S ribosomal proteins that repress gene expression. Mol Cell Biol  24: 8276–8287.  <ul class="find" data-citedArticleID="1053208" data-doi="10.1128/mcb.24.18.8276-8287.2004"><li><a href="http://dx.doi.org/10.1128/mcb.24.18.8276-8287.2004" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Yeast+Asc1p+and+mammalian+RACK1+are+functionally+orthologous+core+40S+ribosomal+proteins+that+repress+gene+expression." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Yeast+Asc1p+and+mammalian+RACK1+are+functionally+orthologous+core+40S+ribosomal+proteins+that+repress+gene+expression.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0001570-Khan1" id="pone.0001570-Khan1"></a>Khan SM, Franke-Fayard B, Mair GR, Lasonder E, Janse CJ, et al.  (2005) Proteome analysis of separated male and female gametocytes reveals novel sex-specific Plasmodium biology. Cell  121: 675–687.  <ul class="find" data-citedArticleID="1053230" data-doi="10.1016/j.cell.2005.03.027"><li><a href="http://dx.doi.org/10.1016/j.cell.2005.03.027" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Proteome+analysis+of+separated+male+and+female+gametocytes+reveals+novel+sex-specific+Plasmodium+biology." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Proteome+analysis+of+separated+male+and+female+gametocytes+reveals+novel+sex-specific+Plasmodium+biology.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0001570-Cowman1" id="pone.0001570-Cowman1"></a>Cowman AF, Crabb BS (2006) Invasion of red blood cells by malaria parasites. Cell  124: 755–766.  <ul class="find" data-citedArticleID="1053196" data-doi="10.1016/j.cell.2006.02.006"><li><a href="http://dx.doi.org/10.1016/j.cell.2006.02.006" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Invasion+of+red+blood+cells+by+malaria+parasites." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Invasion+of+red+blood+cells+by+malaria+parasites.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0001570-Alexander1" id="pone.0001570-Alexander1"></a>Alexander DL, Mital J, Ward GE, Bradley P, Boothroyd JC (2005) Identification of the moving junction complex of Toxoplasma gondii: a collaboration between distinct secretory organelles. PLoS Pathog  1: e17.  <ul class="find" data-citedArticleID="1053174" data-doi="10.1371/journal.ppat.0010017"><li><a href="http://dx.doi.org/10.1371/journal.ppat.0010017" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Identification+of+the+moving+junction+complex+of+Toxoplasma+gondii%3A+a+collaboration+between+distinct+secretory+organelles." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Identification+of+the+moving+junction+complex+of+Toxoplasma+gondii%3A+a+collaboration+between+distinct+secretory+organelles.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0001570-Volkman1" id="pone.0001570-Volkman1"></a>Volkman SK, Hartl DL, Wirth DF, Nielsen KM, Choi M, et al.  (2002) Excess polymorphisms in genes for membrane proteins in Plasmodium falciparum. Science  298: 216–218.  <ul class="find" data-citedArticleID="1053300" data-doi="10.1126/science.1075642"><li><a href="http://dx.doi.org/10.1126/science.1075642" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Excess+polymorphisms+in+genes+for+membrane+proteins+in+Plasmodium+falciparum." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Excess+polymorphisms+in+genes+for+membrane+proteins+in+Plasmodium+falciparum.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0001570-Volkman2" id="pone.0001570-Volkman2"></a>Volkman SK, Sabeti PC, DeCaprio D, Neafsey DE, Schaffner SF, et al.  (2007) A genome-wide map of diversity in Plasmodium falciparum. Nat Genet  39: 113–119.  <ul class="find" data-citedArticleID="1053302" data-doi="10.1038/ng1930"><li><a href="http://dx.doi.org/10.1038/ng1930" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+genome-wide+map+of+diversity+in+Plasmodium+falciparum." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+genome-wide+map+of+diversity+in+Plasmodium+falciparum.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0001570-Kidgell1" id="pone.0001570-Kidgell1"></a>Kidgell C, Volkman SK, Daily J, Borevitz JO, Plouffe D, et al.  (2006) A systematic map of genetic variation in Plasmodium falciparum. PLoS Pathog  2: e57.  <ul class="find" data-citedArticleID="1053232" data-doi="10.1371/journal.ppat.0020057"><li><a href="http://dx.doi.org/10.1371/journal.ppat.0020057" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+systematic+map+of+genetic+variation+in+Plasmodium+falciparum." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+systematic+map+of+genetic+variation+in+Plasmodium+falciparum.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0001570-Jeffares1" id="pone.0001570-Jeffares1"></a>Jeffares DC, Pain A, Berry A, Cox AV, Stalker J, et al.  (2007) Genome variation and evolution of the malaria parasite Plasmodium falciparum. Nat Genet  39: 120–125.  <ul class="find" data-citedArticleID="1053218" data-doi="10.1038/ng1931"><li><a href="http://dx.doi.org/10.1038/ng1931" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genome+variation+and+evolution+of+the+malaria+parasite+Plasmodium+falciparum." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genome+variation+and+evolution+of+the+malaria+parasite+Plasmodium+falciparum.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0001570-LaCount1" id="pone.0001570-LaCount1"></a>LaCount DJ, Vignali M, Chettier R, Phansalkar A, Bell R, et al.  (2005) A protein interaction network of the malaria parasite Plasmodium falciparum. Nature  438: 103–107.  <ul class="find" data-citedArticleID="1053238" data-doi="10.1038/nature04104"><li><a href="http://dx.doi.org/10.1038/nature04104" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+protein+interaction+network+of+the+malaria+parasite+Plasmodium+falciparum." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+protein+interaction+network+of+the+malaria+parasite+Plasmodium+falciparum.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0001570-Collins1" id="pone.0001570-Collins1"></a>Collins SR, Kemmeren P, Zhao XC, Greenblatt JF, Spencer F, et al.  (2007) Toward a comprehensive atlas of the physical interactome of Saccharomyces cerevisiae. Mol Cell Proteomics  6: 439–450.  <ul class="find" data-citedArticleID="1053194" data-doi="10.1074/mcp.m600381-mcp200"><li><a href="http://dx.doi.org/10.1074/mcp.m600381-mcp200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Toward+a+comprehensive+atlas+of+the+physical+interactome+of+Saccharomyces+cerevisiae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Toward+a+comprehensive+atlas+of+the+physical+interactome+of+Saccharomyces+cerevisiae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">36.
              </span><a name="pone.0001570-Surolia1" id="pone.0001570-Surolia1"></a>Surolia A, Ramya TN, Ramya V, Surolia N (2004) ‘FAS’t inhibition of malaria. Biochem J  383: 401–412.  <ul class="find" data-citedArticleID="1053286" data-doi="10.1042/bj20041051"><li><a href="http://dx.doi.org/10.1042/bj20041051" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=%E2%80%98FAS%E2%80%99t+inhibition+of+malaria." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22%E2%80%98FAS%E2%80%99t+inhibition+of+malaria.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">37.
              </span><a name="pone.0001570-Banerjee1" id="pone.0001570-Banerjee1"></a>Banerjee A, Dubnau E, Quemard A, Balasubramanian V, Um KS, et al.  (1994) inhA, a gene encoding a target for isoniazid and ethionamide in Mycobacterium tuberculosis. Science  263: 227–230.  <ul class="find" data-citedArticleID="1053178" data-doi="10.1126/science.8284673"><li><a href="http://dx.doi.org/10.1126/science.8284673" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=inhA%2C+a+gene+encoding+a+target+for+isoniazid+and+ethionamide+in+Mycobacterium+tuberculosis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22inhA%2C+a+gene+encoding+a+target+for+isoniazid+and+ethionamide+in+Mycobacterium+tuberculosis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">38.
              </span><a name="pone.0001570-Surolia2" id="pone.0001570-Surolia2"></a>Surolia N, Surolia A (2001) Triclosan offers protection against blood stages of malaria by inhibiting enoyl-ACP reductase of Plasmodium falciparum. Nat Med  7: 167–173.  <ul class="find" data-citedArticleID="1053288" data-doi="10.1053/tmrv.2002.31458"><li><a href="http://dx.doi.org/10.1053/tmrv.2002.31458" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Triclosan+offers+protection+against+blood+stages+of+malaria+by+inhibiting+enoyl-ACP+reductase+of+Plasmodium+falciparum." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Triclosan+offers+protection+against+blood+stages+of+malaria+by+inhibiting+enoyl-ACP+reductase+of+Plasmodium+falciparum.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">39.
              </span><a name="pone.0001570-Suthram1" id="pone.0001570-Suthram1"></a>Suthram S, Sittler T, Ideker T (2005) The Plasmodium protein network diverges from those of other eukaryotes. Nature  438: 108–112.  <ul class="find" data-citedArticleID="1053290" data-doi="10.1038/nature04135"><li><a href="http://dx.doi.org/10.1038/nature04135" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+Plasmodium+protein+network+diverges+from+those+of+other+eukaryotes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+Plasmodium+protein+network+diverges+from+those+of+other+eukaryotes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">40.
              </span><a name="pone.0001570-Kappe2" id="pone.0001570-Kappe2"></a>Kappe SH, Buscaglia CA, Nussenzweig V (2004) Plasmodium sporozoite molecular cell biology. Annu Rev Cell Dev Biol  20: 29–59.  <ul class="find" data-citedArticleID="1053226" data-doi="10.1146/annurev.cellbio.20.011603.150935"><li><a href="http://dx.doi.org/10.1146/annurev.cellbio.20.011603.150935" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Plasmodium+sporozoite+molecular+cell+biology." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Plasmodium+sporozoite+molecular+cell+biology.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">41.
              </span><a name="pone.0001570-Kaiser1" id="pone.0001570-Kaiser1"></a>Kaiser K, Matuschewski K, Camargo N, Ross J, Kappe SH (2004) Differential transcriptome profiling identifies Plasmodium genes encoding pre-erythrocytic stage-specific proteins. Mol Microbiol  51: 1221–1232.  <ul class="find" data-citedArticleID="1053222" data-doi="10.1046/j.1365-2958.2003.03909.x"><li><a href="http://dx.doi.org/10.1046/j.1365-2958.2003.03909.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Differential+transcriptome+profiling+identifies+Plasmodium+genes+encoding+pre-erythrocytic+stage-specific+proteins." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Differential+transcriptome+profiling+identifies+Plasmodium+genes+encoding+pre-erythrocytic+stage-specific+proteins.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">42.
              </span><a name="pone.0001570-Raibaud1" id="pone.0001570-Raibaud1"></a>Raibaud A, Brahimi K, Roth CW, Brey PT, Faust DM (2006) Differential gene expression in the ookinete stage of the malaria parasite Plasmodium berghei. Mol Biochem Parasitol  150: 107–113.  <ul class="find" data-citedArticleID="1053270" data-doi="10.1016/j.molbiopara.2006.07.001"><li><a href="http://dx.doi.org/10.1016/j.molbiopara.2006.07.001" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Differential+gene+expression+in+the+ookinete+stage+of+the+malaria+parasite+Plasmodium+berghei." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Differential+gene+expression+in+the+ookinete+stage+of+the+malaria+parasite+Plasmodium+berghei.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">43.
              </span><a name="pone.0001570-Zhou2" id="pone.0001570-Zhou2"></a>Zhou XJ, Gibson G (2004) Cross-species comparison of genome-wide expression patterns. Genome Biol  5: 232.  <ul class="find" data-citedArticleID="1053310" data-doi="10.1186/gb-2004-5-7-232"><li><a href="http://dx.doi.org/10.1186/gb-2004-5-7-232" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Cross-species+comparison+of+genome-wide+expression+patterns." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Cross-species+comparison+of+genome-wide+expression+patterns.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">44.
              </span><a name="pone.0001570-Bergmann1" id="pone.0001570-Bergmann1"></a>Bergmann S, Ihmels J, Barkai N (2004) Similarities and differences in genome-wide expression data of six organisms. PLoS Biol  2: E9.  <ul class="find" data-citedArticleID="1053182" data-doi="10.1371/journal.pbio.0020009"><li><a href="http://dx.doi.org/10.1371/journal.pbio.0020009" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Similarities+and+differences+in+genome-wide+expression+data+of+six+organisms." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Similarities+and+differences+in+genome-wide+expression+data+of+six+organisms.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">45.
              </span><a name="pone.0001570-Stuart1" id="pone.0001570-Stuart1"></a>Stuart JM, Segal E, Koller D, Kim SK (2003) A gene-coexpression network for global discovery of conserved genetic modules. Science  302: 249–255.  <ul class="find" data-citedArticleID="1053280" data-doi="10.1126/science.1087447"><li><a href="http://dx.doi.org/10.1126/science.1087447" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+gene-coexpression+network+for+global+discovery+of+conserved+genetic+modules." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+gene-coexpression+network+for+global+discovery+of+conserved+genetic+modules.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">46.
              </span><a name="pone.0001570-SweetCordero1" id="pone.0001570-SweetCordero1"></a>Sweet-Cordero A, Mukherjee S, Subramanian A, You H, Roix JJ, et al.  (2005) An oncogenic KRAS2 expression signature identified by cross-species gene-expression analysis. Nat Genet  37: 48–55.  <ul class="find" data-citedArticleID="1053292" data-doi="10.1038/ng1490"><li><a href="http://dx.doi.org/10.1038/ng1490" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=An+oncogenic+KRAS2+expression+signature+identified+by+cross-species+gene-expression+analysis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22An+oncogenic+KRAS2+expression+signature+identified+by+cross-species+gene-expression+analysis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">47.
              </span><a name="pone.0001570-McCarroll1" id="pone.0001570-McCarroll1"></a>McCarroll SA, Murphy CT, Zou S, Pletcher SD, Chin CS, et al.  (2004) Comparing genomic expression patterns across species identifies shared transcriptional profile in aging. Nat Genet  36: 197–204.  <ul class="find" data-citedArticleID="1053258" data-doi="10.1038/ng1291"><li><a href="http://dx.doi.org/10.1038/ng1291" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Comparing+genomic+expression+patterns+across+species+identifies+shared+transcriptional+profile+in+aging." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Comparing+genomic+expression+patterns+across+species+identifies+shared+transcriptional+profile+in+aging.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">48.
              </span><a name="pone.0001570-Vazquez1" id="pone.0001570-Vazquez1"></a>Vazquez A, Flammini A, Maritan A, Vespignani A (2003) Global protein function prediction from protein-protein interaction networks. Nat Biotechnol  21: 697–700.  <ul class="find" data-citedArticleID="1053296" data-doi="10.1038/nbt825"><li><a href="http://dx.doi.org/10.1038/nbt825" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Global+protein+function+prediction+from+protein-protein+interaction+networks." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Global+protein+function+prediction+from+protein-protein+interaction+networks.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">49.
              </span><a name="pone.0001570-Murali1" id="pone.0001570-Murali1"></a>Murali TM, Wu CJ, Kasif S (2006) The art of gene function prediction. Nat Biotechnol  24: 1474–1475; author reply 1475–1476.  <ul class="find" data-citedArticleID="1053264" data-doi="10.1038/nbt1206-1474"><li><a href="http://dx.doi.org/10.1038/nbt1206-1474" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+art+of+gene+function+prediction." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+art+of+gene+function+prediction.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">50.
              </span><a name="pone.0001570-Douradinha1" id="pone.0001570-Douradinha1"></a>Douradinha B, van Dijk MR, Ataide R, van Gemert GJ, Thompson J, et al.  (2007) Genetically attenuated P36p-deficient Plasmodium berghei sporozoites confer long-lasting and partial cross-species protection. Int J Parasitol  37: 1511–1519.  <ul class="find" data-citedArticleID="1053202" data-doi="10.1016/j.ijpara.2007.05.005"><li><a href="http://dx.doi.org/10.1016/j.ijpara.2007.05.005" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genetically+attenuated+P36p-deficient+Plasmodium+berghei+sporozoites+confer+long-lasting+and+partial+cross-species+protection." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genetically+attenuated+P36p-deficient+Plasmodium+berghei+sporozoites+confer+long-lasting+and+partial+cross-species+protection.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">51.
              </span><a name="pone.0001570-Beetsma1" id="pone.0001570-Beetsma1"></a>Beetsma AL, van de Wiel TJ, Sauerwein RW, Eling WM (1998) Plasmodium berghei ANKA: purification of large numbers of infectious gametocytes. Exp Parasitol  88: 69–72.  <ul class="find" data-citedArticleID="1053180" data-doi="10.1006/expr.1998.4203"><li><a href="http://dx.doi.org/10.1006/expr.1998.4203" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Plasmodium+berghei+ANKA%3A+purification+of+large+numbers+of+infectious+gametocytes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Plasmodium+berghei+ANKA%3A+purification+of+large+numbers+of+infectious+gametocytes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">52.
              </span><a name="pone.0001570-Jongco1" id="pone.0001570-Jongco1"></a>Jongco AM, Ting LM, Thathy V, Mota MM, Kim K (2006) Improved transfection and new selectable markers for the rodent malaria parasite Plasmodium yoelii. Mol Biochem Parasitol  146: 242–250.  <ul class="find" data-citedArticleID="1053220" data-doi="10.1016/j.molbiopara.2006.01.001"><li><a href="http://dx.doi.org/10.1016/j.molbiopara.2006.01.001" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Improved+transfection+and+new+selectable+markers+for+the+rodent+malaria+parasite+Plasmodium+yoelii." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Improved+transfection+and+new+selectable+markers+for+the+rodent+malaria+parasite+Plasmodium+yoelii.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">53.
              </span><a name="pone.0001570-Mack1" id="pone.0001570-Mack1"></a>Mack SR, Vanderberg JP, Nawrot R (1978) Column separation of Plasmodium berghei sporozoites. J Parasitol  64: 166–168.  <ul class="find" data-citedArticleID="1053250" data-doi="10.2307/3279638"><li><a href="http://dx.doi.org/10.2307/3279638" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Column+separation+of+Plasmodium+berghei+sporozoites." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Column+separation+of+Plasmodium+berghei+sporozoites.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">54.
              </span><a name="pone.0001570-Camerer1" id="pone.0001570-Camerer1"></a>Camerer E, Gjernes E, Wiiger M, Pringle S, Prydz H (2000) Binding of factor VIIa to tissue factor on keratinocytes induces gene expression. J Biol Chem  275: 6580–6585.  <ul class="find" data-citedArticleID="1053188" data-doi="10.1074/jbc.275.9.6580"><li><a href="http://dx.doi.org/10.1074/jbc.275.9.6580" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Binding+of+factor+VIIa+to+tissue+factor+on+keratinocytes+induces+gene+expression." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Binding+of+factor+VIIa+to+tissue+factor+on+keratinocytes+induces+gene+expression.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">55.
              </span><a name="pone.0001570-Ifediba1" id="pone.0001570-Ifediba1"></a>Ifediba T, Vanderberg JP (1981) Complete in vitro maturation of Plasmodium falciparum gametocytes. Nature  294: 364–366.  <ul class="find" data-citedArticleID="1053216" data-doi="10.1038/294364a0"><li><a href="http://dx.doi.org/10.1038/294364a0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Complete+in+vitro+maturation+of+Plasmodium+falciparum+gametocytes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Complete+in+vitro+maturation+of+Plasmodium+falciparum+gametocytes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">56.
              </span><a name="pone.0001570-Kumar2" id="pone.0001570-Kumar2"></a>Kumar MKBaN (2004)  Plasmodiun falciparum gametocyte culture, purification, and gametogenesis. In: Ljungstrom HPI, Schlichtherle M, Scherf A, Wahlgren M, editors. Methods in malaria research. Manassas: MR4/ATCC:. pp. 93–95.  <ul class="find-nolinks"></ul></li><li><span class="label">57.
              </span><a name="pone.0001570-AlOlayan1" id="pone.0001570-AlOlayan1"></a>Al-Olayan EM, Beetsma AL, Butcher GA, Sinden RE, Hurd H (2002) Complete development of mosquito phases of the malaria parasite in vitro. Science  295: 677–679.  <ul class="find" data-citedArticleID="1053172" data-doi="10.1126/science.1067159"><li><a href="http://dx.doi.org/10.1126/science.1067159" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Complete+development+of+mosquito+phases+of+the+malaria+parasite+in+vitro." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Complete+development+of+mosquito+phases+of+the+malaria+parasite+in+vitro.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">58.
              </span><a name="pone.0001570-Kariuki1" id="pone.0001570-Kariuki1"></a>Kariuki MM, Kiaira JK, Mulaa FK, Mwangi JK, Wasunna MK, et al.  (1998) Plasmodium falciparum: purification of the various gametocyte developmental stages from in vitro-cultivated parasites. Am J Trop Med Hyg  59: 505–508.  <ul class="find" data-citedArticleID="1053228"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Plasmodium+falciparum%3A+purification+of+the+various+gametocyte+developmental+stages+from+in+vitro-cultivated+parasites.&amp;auth=&amp;atitle=Plasmodium+falciparum%3A+purification+of+the+various+gametocyte+developmental+stages+from+in+vitro-cultivated+parasites." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Plasmodium+falciparum%3A+purification+of+the+various+gametocyte+developmental+stages+from+in+vitro-cultivated+parasites." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Plasmodium+falciparum%3A+purification+of+the+various+gametocyte+developmental+stages+from+in+vitro-cultivated+parasites.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">59.
              </span><a name="pone.0001570-Lambros1" id="pone.0001570-Lambros1"></a>Lambros C, Vanderberg JP (1979) Synchronization of Plasmodium falciparum erythrocytic stages in culture. J Parasitol  65: 418–420.  <ul class="find" data-citedArticleID="1053240" data-doi="10.2307/3280287"><li><a href="http://dx.doi.org/10.2307/3280287" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Synchronization+of+Plasmodium+falciparum+erythrocytic+stages+in+culture." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Synchronization+of+Plasmodium+falciparum+erythrocytic+stages+in+culture.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">60.
              </span><a name="pone.0001570-Young1" id="pone.0001570-Young1"></a>Young JA, Fivelman QL, Blair PL, de la Vega P, Le Roch KG, et al.  (2005) The Plasmodium falciparum sexual development transcriptome: A microarray analysis using ontology-based pattern identification. Mol Biochem Parasitol  143: 67–79.  <ul class="find" data-citedArticleID="1053306" data-doi="10.1016/j.molbiopara.2005.05.007"><li><a href="http://dx.doi.org/10.1016/j.molbiopara.2005.05.007" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+Plasmodium+falciparum+sexual+development+transcriptome%3A+A+microarray+analysis+using+ontology-based+pattern+identification." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+Plasmodium+falciparum+sexual+development+transcriptome%3A+A+microarray+analysis+using+ontology-based+pattern+identification.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">61.
              </span><a name="pone.0001570-Bolstad1" id="pone.0001570-Bolstad1"></a>Bolstad BM, Irizarry RA, Astrand M, Speed TP (2003) A comparison of normalization methods for high density oligonucleotide array data based on variance and bias. Bioinformatics  19: 185–193.  <ul class="find" data-citedArticleID="1053184" data-doi="10.1093/bioinformatics/19.2.185"><li><a href="http://dx.doi.org/10.1093/bioinformatics/19.2.185" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+comparison+of+normalization+methods+for+high+density+oligonucleotide+array+data+based+on+variance+and+bias." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+comparison+of+normalization+methods+for+high+density+oligonucleotide+array+data+based+on+variance+and+bias.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">62.
              </span><a name="pone.0001570-Oshiro1" id="pone.0001570-Oshiro1"></a>Oshiro G, Wodicka LM, Washburn MP, Yates JR 3rd, Lockhart DJ, et al.  (2002) Parallel identification of new genes in Saccharomyces cerevisiae. Genome Res  12: 1210–1220.  <ul class="find" data-citedArticleID="1053268" data-doi="10.1101/gr.226802"><li><a href="http://dx.doi.org/10.1101/gr.226802" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Parallel+identification+of+new+genes+in+Saccharomyces+cerevisiae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Parallel+identification+of+new+genes+in+Saccharomyces+cerevisiae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">63.
              </span><a name="pone.0001570-Washburn1" id="pone.0001570-Washburn1"></a>Washburn MP, Koller A, Oshiro G, Ulaszek RR, Plouffe D, et al.  (2003) Protein pathway and complex clustering of correlated mRNA and protein expression analyses in Saccharomyces cerevisiae. Proc Natl Acad Sci U S A  100: 3107–3112.  <ul class="find" data-citedArticleID="1053304" data-doi="10.1073/pnas.0634629100"><li><a href="http://dx.doi.org/10.1073/pnas.0634629100" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Protein+pathway+and+complex+clustering+of+correlated+mRNA+and+protein+expression+analyses+in+Saccharomyces+cerevisiae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Protein+pathway+and+complex+clustering+of+correlated+mRNA+and+protein+expression+analyses+in+Saccharomyces+cerevisiae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">64.
              </span><a name="pone.0001570-Chen1" id="pone.0001570-Chen1"></a>Chen F, Mackey AJ, Stoeckert CJ Jr, Roos DS (2006) OrthoMCL-DB: querying a comprehensive multi-species collection of ortholog groups. Nucleic Acids Res  34: D363–368.  <ul class="find" data-citedArticleID="1053192" data-doi="10.1093/nar/gkj123"><li><a href="http://dx.doi.org/10.1093/nar/gkj123" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=OrthoMCL-DB%3A+querying+a+comprehensive+multi-species+collection+of+ortholog+groups." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22OrthoMCL-DB%3A+querying+a+comprehensive+multi-species+collection+of+ortholog+groups.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">65.
              </span><a name="pone.0001570-Subramanian1" id="pone.0001570-Subramanian1"></a>Subramanian A, Tamayo P, Mootha VK, Mukherjee S, Ebert BL, et al.  (2005) Gene set enrichment analysis: a knowledge-based approach for interpreting genome-wide expression profiles. Proc Natl Acad Sci U S A  102: 15545–15550.  <ul class="find" data-citedArticleID="1053284" data-doi="10.1073/pnas.0506580102"><li><a href="http://dx.doi.org/10.1073/pnas.0506580102" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Gene+set+enrichment+analysis%3A+a+knowledge-based+approach+for+interpreting+genome-wide+expression+profiles." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Gene+set+enrichment+analysis%3A+a+knowledge-based+approach+for+interpreting+genome-wide+expression+profiles.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">66.
              </span><a name="pone.0001570-Ockenhouse1" id="pone.0001570-Ockenhouse1"></a>Ockenhouse CF, Klotz FW, Tandon NN, Jamieson GA (1991) Sequestrin, a CD36 recognition protein on Plasmodium falciparum malaria-infected erythrocytes identified by anti-idiotype antibodies. Proc Natl Acad Sci U S A  88: 3175–3179.  <ul class="find" data-citedArticleID="1053266" data-doi="10.1073/pnas.88.8.3175"><li><a href="http://dx.doi.org/10.1073/pnas.88.8.3175" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Sequestrin%2C+a+CD36+recognition+protein+on+Plasmodium+falciparum+malaria-infected+erythrocytes+identified+by+anti-idiotype+antibodies." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Sequestrin%2C+a+CD36+recognition+protein+on+Plasmodium+falciparum+malaria-infected+erythrocytes+identified+by+anti-idiotype+antibodies.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">67.
              </span><a name="pone.0001570-Gulli1" id="pone.0001570-Gulli1"></a>Gulli MP, Girard JP, Zabetakis D, Lapeyre B, Melese T, et al.  (1995) gar2 is a nucleolar protein from Schizosaccharomyces pombe required for 18S rRNA and 40S ribosomal subunit accumulation. Nucleic Acids Res  23: 1912–1918.  <ul class="find" data-citedArticleID="1053212" data-doi="10.1093/nar/23.11.1912"><li><a href="http://dx.doi.org/10.1093/nar/23.11.1912" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=gar2+is+a+nucleolar+protein+from+Schizosaccharomyces+pombe+required+for+18S+rRNA+and+40S+ribosomal+subunit+accumulation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22gar2+is+a+nucleolar+protein+from+Schizosaccharomyces+pombe+required+for+18S+rRNA+and+40S+ribosomal+subunit+accumulation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">68.
              </span><a name="pone.0001570-Marti1" id="pone.0001570-Marti1"></a>Marti M, Good RT, Rug M, Knuepfer E, Cowman AF (2004) Targeting malaria virulence and remodeling proteins to the host erythrocyte. Science  306: 1930–1933.  <ul class="find" data-citedArticleID="1053254" data-doi="10.1126/science.1102452"><li><a href="http://dx.doi.org/10.1126/science.1102452" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Targeting+malaria+virulence+and+remodeling+proteins+to+the+host+erythrocyte." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Targeting+malaria+virulence+and+remodeling+proteins+to+the+host+erythrocyte.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">69.
              </span><a name="pone.0001570-Ling1" id="pone.0001570-Ling1"></a>Ling IT, Florens L, Dluzewski AR, Kaneko O, Grainger M, et al.  (2004) The Plasmodium falciparum clag9 gene encodes a rhoptry protein that is transferred to the host erythrocyte upon invasion. Mol Microbiol  52: 107–118.  <ul class="find" data-citedArticleID="1053248" data-doi="10.1111/j.1365-2958.2003.03969.x"><li><a href="http://dx.doi.org/10.1111/j.1365-2958.2003.03969.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+Plasmodium+falciparum+clag9+gene+encodes+a+rhoptry+protein+that+is+transferred+to+the+host+erythrocyte+upon+invasion." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+Plasmodium+falciparum+clag9+gene+encodes+a+rhoptry+protein+that+is+transferred+to+the+host+erythrocyte+upon+invasion.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">70.
              </span><a name="pone.0001570-Doolan1" id="pone.0001570-Doolan1"></a>Doolan DL, Southwood S, Freilich DA, Sidney J, Graber NL, et al.  (2003) Identification of Plasmodium falciparum antigens by antigenic analysis of genomic and proteomic data. Proc Natl Acad Sci U S A  100: 9952–9957.  <ul class="find" data-citedArticleID="1053200" data-doi="10.1073/pnas.1633254100"><li><a href="http://dx.doi.org/10.1073/pnas.1633254100" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Identification+of+Plasmodium+falciparum+antigens+by+antigenic+analysis+of+genomic+and+proteomic+data." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Identification+of+Plasmodium+falciparum+antigens+by+antigenic+analysis+of+genomic+and+proteomic+data.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001570.XML" value="188040"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001570.PDF" value="592708"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001570.t001.PNG_L" value="375015"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001570.t001.PNG_M" value="215046"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001570.t001.PNG_S" value="17568"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001570.t001.TIF" value="1560504"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001570.t001.PNG_I" value="65033"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001570.g001.PNG_L" value="1015389"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001570.g001.PNG_M" value="243188"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001570.g001.PNG_S" value="17522"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001570.g001.TIF" value="2837180"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001570.g001.PNG_I" value="89454"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001570.t002.PNG_L" value="345354"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001570.t002.PNG_M" value="199765"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001570.t002.PNG_S" value="16914"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001570.t002.TIF" value="1465558"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001570.t002.PNG_I" value="60530"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001570.g002.PNG_L" value="840244"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001570.g002.PNG_M" value="219672"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001570.g002.PNG_S" value="18101"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001570.g002.TIF" value="1622266"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001570.g002.PNG_I" value="74789"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001570.g003.PNG_L" value="200477"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001570.g003.PNG_M" value="63261"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001570.g003.PNG_S" value="12136"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001570.g003.TIF" value="778878"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001570.g003.PNG_I" value="22863"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001570.g004.PNG_L" value="200375"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001570.g004.PNG_M" value="92010"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001570.g004.PNG_S" value="18563"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001570.g004.TIF" value="577926"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001570.g004.PNG_I" value="38502"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001570.g005.PNG_L" value="1733393"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001570.g005.PNG_M" value="253689"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001570.g005.PNG_S" value="20146"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001570.g005.TIF" value="2549944"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001570.g005.PNG_I" value="116125"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001570.s001.XLS" value="76800"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001570.s002.XLS" value="3398656"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001570.s003.XLS" value="3198976"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001570.s004.XLS" value="753152"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001570&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001570" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001570&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0001570&volume=&issue=&title=Evidence-Based Annotation of the Malaria Parasite's Genome Using Comparative Expression Profiling&author_name=Yingyao%20Zhou%2C%20Vandana%20Ramachandran%2C%20Kota%20Arun%20Kumar%2C%20Scott%20Westenberger%2C%20Phillippe%20Refour%2C%20Bin%20Zhou%2C%20Fengwu%20Li%2C%20Jason%20A.%20Young%2C%20Kaisheng%20Chen%2C%20David%20Plouffe%2C%20Kerstin%20Henson%2C%20Victor%20Nussenzweig%2C%20Jane%20Carlton%2C%20Joseph%20M.%20Vinetz%2C%20Manoj%20T.%20Duraisingh%2C%20Elizabeth%20A.%20Winzeler&start_page=1&end_page=16" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001570" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001570" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0001570" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001570&amp;t=Evidence-Based%20Annotation%20of%20the%20Malaria%20Parasite's%20Genome%20Using%20Comparative%20Expression%20Profiling" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001570&title=Evidence-Based%20Annotation%20of%20the%20Malaria%20Parasite's%20Genome%20Using%20Comparative%20Expression%20Profiling&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001570&amp;title=Evidence-Based%20Annotation%20of%20the%20Malaria%20Parasite's%20Genome%20Using%20Comparative%20Expression%20Profiling" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001570" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0001570&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Evidence-Based Annotation of the Malaria Parasite\'s Genome Using Comparative Expression Profiling';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0001570';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Evidence-Based%20Annotation%20of%20the%20Malaria%20Parasite's%20Genome%20Using%20Comparative%20Expression%20Profiling http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001570" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0001570" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">














          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Blood%22" title="Search for articles in the subject area:'Blood'"><div class="flagText">Blood</div></a>
              <div data-categoryid="47189" data-articleid="25966"
                   data-categoryname="Blood"
                   class="flagImage" title="Flag 'Blood' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Gene+expression%22" title="Search for articles in the subject area:'Gene expression'"><div class="flagText">Gene expression</div></a>
              <div data-categoryid="17515" data-articleid="25966"
                   data-categoryname="Gene expression"
                   class="flagImage" title="Flag 'Gene expression' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Malaria%22" title="Search for articles in the subject area:'Malaria'"><div class="flagText">Malaria</div></a>
              <div data-categoryid="47357" data-articleid="25966"
                   data-categoryname="Malaria"
                   class="flagImage" title="Flag 'Malaria' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Malarial+parasites%22" title="Search for articles in the subject area:'Malarial parasites'"><div class="flagText">Malarial parasites</div></a>
              <div data-categoryid="42885" data-articleid="25966"
                   data-categoryname="Malarial parasites"
                   class="flagImage" title="Flag 'Malarial parasites' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Parasitic+life+cycles%22" title="Search for articles in the subject area:'Parasitic life cycles'"><div class="flagText">Parasitic life cycles</div></a>
              <div data-categoryid="34861" data-articleid="25966"
                   data-categoryname="Parasitic life cycles"
                   class="flagImage" title="Flag 'Parasitic life cycles' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Plasmodium%22" title="Search for articles in the subject area:'Plasmodium'"><div class="flagText">Plasmodium</div></a>
              <div data-categoryid="18205" data-articleid="25966"
                   data-categoryname="Plasmodium"
                   class="flagImage" title="Flag 'Plasmodium' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Plasmodium+yoelii%22" title="Search for articles in the subject area:'Plasmodium yoelii'"><div class="flagText">Plasmodium yoelii</div></a>
              <div data-categoryid="43387" data-articleid="25966"
                   data-categoryname="Plasmodium yoelii"
                   class="flagImage" title="Flag 'Plasmodium yoelii' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Sporozoites%22" title="Search for articles in the subject area:'Sporozoites'"><div class="flagText">Sporozoites</div></a>
              <div data-categoryid="17283" data-articleid="25966"
                   data-categoryname="Sporozoites"
                   class="flagImage" title="Flag 'Sporozoites' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=6010'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=5229'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=379&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>


</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
