

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Genetic Evidence for a Mitochondriate Ancestry in the ‘Amitochondriate’ Flagellate Trimastix pyriformis</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0001383"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0001383" />

    <meta name="citation_title" content="Genetic Evidence for a Mitochondriate Ancestry in the ‘Amitochondriate’ Flagellate Trimastix pyriformis"/>
    <meta itemprop="name" content="Genetic Evidence for a Mitochondriate Ancestry in the ‘Amitochondriate’ Flagellate Trimastix pyriformis"/>

      <meta name="citation_author" content="Vladimir Hampl"/>
            <meta name="citation_author_institution" content="Department of Biochemistry and Molecular Biology, Dalhousie University, Halifax, Nova Scotia, Canada"/>
      <meta name="citation_author" content="Jeffrey D. Silberman"/>
            <meta name="citation_author_institution" content="Department of Biochemistry and Molecular Biology, Dalhousie University, Halifax, Nova Scotia, Canada"/>
            <meta name="citation_author_institution" content="Department of Biological Sciences, University of Arkansas, Fayetteville, Arkansas, United States of America"/>
      <meta name="citation_author" content="Alexandra Stechmann"/>
            <meta name="citation_author_institution" content="Department of Biochemistry and Molecular Biology, Dalhousie University, Halifax, Nova Scotia, Canada"/>
      <meta name="citation_author" content="Sara Diaz-Triviño"/>
            <meta name="citation_author_institution" content="Department of Biochemistry and Molecular Biology, Dalhousie University, Halifax, Nova Scotia, Canada"/>
      <meta name="citation_author" content="Patricia J. Johnson"/>
            <meta name="citation_author_institution" content="Department of Microbiology, Immunology, and Molecular Genetics, University of California at Los Angeles, Los Angeles, California, United States of America"/>
      <meta name="citation_author" content="Andrew J. Roger"/>
            <meta name="citation_author_institution" content="Department of Biochemistry and Molecular Biology, Dalhousie University, Halifax, Nova Scotia, Canada"/>

    <meta name="citation_date" content="2008/1/2"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0001383.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e1383"/>
    <meta name="citation_issue" content="1"/>
    <meta name="citation_volume" content="3"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=The phagotrophic origin of eukaryotes and phylogenetic classification of protozoa.; citation_author=T Cavalier-Smith; citation_journal_title=Int J Syst Evol Microbiol; citation_volume=52; citation_number=1; citation_pages=297-354; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=The chimeric eukaryote: origin of the nucleus from the karyomastigont in amitochondriate protists.; citation_author=L Margulis; citation_author=MF Dolan; citation_author=R Guerrero; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=97; citation_number=2; citation_pages=6954-6959; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Symbiosis between methanogenic archaea and delta-proteobacteria as the origin of eukaryotes: The syntrophic hypothesis.; citation_author=D Moreira; citation_author=P Lopez-Garcia; citation_journal_title=J Mol Evol; citation_volume=47; citation_number=3; citation_pages=517-530; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=The hydrogen hypothesis for the first eukaryote.; citation_author=W Martin; citation_author=M Müller; citation_journal_title=Nature; citation_volume=392; citation_number=4; citation_pages=37-41; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=A chimeric prokaryotic ancestry of mitochondria and primitive eukaryotes.; citation_author=S Karlin; citation_author=L Brocchieri; citation_author=J Mrazek; citation_author=AM Campbell; citation_author=AM Spormann; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=96; citation_number=5; citation_pages=9190-9195; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Endosymbiotic origin of the mitochondrial envelope.; citation_author=T Cavalier-Smith; citation_number=6; citation_pages=265-279; citation_date=1983; citation_publisher=de Gruyter; " />
      <meta name="citation_reference" content="citation_title=Direct evidence for secondary loss of mitochondria in Entamoeba histolytica.; citation_author=CG Clark; citation_author=AJ Roger; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=92; citation_number=7; citation_pages=6518-6521; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Molecular data suggest an early acquisition of the mitochondrion endosymbiont.; citation_author=DS Horner; citation_author=RP Hirt; citation_author=S Kilvington; citation_author=D Lloyd; citation_author=TM Embley; citation_journal_title=P Roy Soc Lond B Bio; citation_volume=263; citation_number=8; citation_pages=1053-1059; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=A common evolutionary origin for mitochondria and hydrogenosomes.; citation_author=ET Bui; citation_author=PJ Bradley; citation_author=PJ Johnson; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=93; citation_number=9; citation_pages=9651-9656; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Presence of a mitochondrial-type 70-kDa heat shock protein in Trichomonas vaginalis suggests a very early mitochondrial endosymbiosis in eukaryotes.; citation_author=A Germot; citation_author=H Philippe; citation_author=H Le Guyader; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=93; citation_number=10; citation_pages=14614-14617; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=A possible mitochondrial gene in the early-branching amitochondriate protist Trichomonas vaginalis.; citation_author=AJ Roger; citation_author=CG Clark; citation_author=WF Doolittle; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=93; citation_number=11; citation_pages=14618-14622; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Immunological detection and subcellular localization of HSP70 and HSP60 homologs in Trichomonas vaginalis.; citation_author=P Bozner; citation_journal_title=J Parasitol; citation_volume=83; citation_number=12; citation_pages=224-229; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=A mitochondrial Hsp70 orthologue in Vairimorpha necatrix: Molecular evidence that microsporidia once contained mitochondria.; citation_author=RP Hirt; citation_author=B Healy; citation_author=CR Vossbrinck; citation_author=EU Canning; citation_author=TM Embley; citation_journal_title=Curr Biol; citation_volume=7; citation_number=13; citation_pages=995-998; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=A mitochondrial-like chaperonin 60 gene in Giardia lamblia: evidence that diplomonads once harbored an endosymbiont related to the progenitor of mitochondria.; citation_author=AJ Roger; citation_author=SG Svard; citation_author=J Tovar; citation_author=CG Clark; citation_author=MW Smith; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=95; citation_number=14; citation_pages=229-234; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=The mitosome, a novel organelle related to mitochondria in the amitochondrial parasite Entamoeba histolytica.; citation_author=J Tovar; citation_author=A Fischer; citation_author=CG Clark; citation_journal_title=Mol Microbiol; citation_volume=32; citation_number=15; citation_pages=1013-1021; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Mitochondrial type iron-sulfur cluster assembly in the amitochondriate eukaryotes Trichomonas vaginalis and Giardia intestinalis, as indicated by the phylogeny of IscS.; citation_author=J Tachezy; citation_author=LB Sánchez; citation_author=M Müller; citation_journal_title=Mol Biol Evol; citation_volume=18; citation_number=16; citation_pages=1919-1928; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Chaperonin 60 phylogeny provides further evidence for secondary loss of mitochondria among putative early-branching eukaryotes.; citation_author=DS Horner; citation_author=TM Embley; citation_journal_title=Mol Biol Evol; citation_volume=18; citation_number=17; citation_pages=1970-1975; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=A mitochondrial remnant in the microsporidian Trachipleistophora hominis.; citation_author=BAP Williams; citation_author=RP Hirt; citation_author=JM Lucocq; citation_author=TM Embley; citation_journal_title=Nature; citation_volume=418; citation_number=18; citation_pages=865-869; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Mitochondrial-type iron-sulfur cluster biosynthesis genes (IscS and IscU) in the apicomplexan Cryptosporidium parvum.; citation_author=MJ LaGier; citation_author=J Tachezy; citation_author=F Stejskal; citation_author=K Kutišová; citation_author=JS Keithly; citation_journal_title=Microbiology; citation_volume=149; citation_number=19; citation_pages=3519-3530; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Fungal hydrogenosomes contain mitochondrial heat-shock proteins.; citation_author=M van der Giezen; citation_author=GM Birdsey; citation_author=DS Horner; citation_author=J Lucocq; citation_author=PL Dyal; citation_journal_title=Mol Biol Evol; citation_volume=20; citation_number=20; citation_pages=1051-1061; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Trichomonas hydrogenosomes contain the NADH dehydrogenase module of mitochondrial complex I.; citation_author=I Hrdý; citation_author=RP Hirt; citation_author=P Doležal; citation_author=L Bardoňová; citation_author=PG Foster; citation_journal_title=Nature; citation_volume=432; citation_number=21; citation_pages=618-622; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Non-mitochondrial complex I proteins in a hydrogenosomal oxidoreductase complex.; citation_author=SD Dyall; citation_author=WH Yan; citation_author=MG Delgadillo-Correa; citation_author=A Lunceford; citation_author=JA Loo; citation_journal_title=Nature; citation_volume=431; citation_number=22; citation_pages=1103-1107; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Mitochondrial remnant organelles of Giardia function in iron-sulphur protein maturation.; citation_author=J Tovar; citation_author=G Leon-Avila; citation_author=LB Sánchez; citation_author=R Šuťák; citation_author=J Tachezy; citation_journal_title=Nature; citation_volume=426; citation_number=23; citation_pages=172-176; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Cryptosporidium parvum Cpn60 targets a relict organelle.; citation_author=CE Riordan; citation_author=JG Ault; citation_author=SG Langreth; citation_author=JS Keithly; citation_journal_title=Curr Genet; citation_volume=44; citation_number=24; citation_pages=138-147; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Multiple secondary origins of the anaerobic lifestyle in eukaryotes.; citation_author=TM Embley; citation_journal_title=Phil Trans R Soc B; citation_volume=361; citation_number=25; citation_pages=1055-1067; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Oxymonads are closely related to the excavate taxon Trimastix.; citation_author=JB Dacks; citation_author=JD Silberman; citation_author=AGB Simpson; citation_author=S Moriya; citation_author=T Kudo; citation_journal_title=Mol Biol Evol; citation_volume=18; citation_number=26; citation_pages=1034-1044; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Cytoskeletal organization, phylogenetic affinities and systematics in the contentious taxon Excavata (Eukaryota).; citation_author=AGB Simpson; citation_journal_title=Int J Syst Evol Microbiol; citation_volume=53; citation_number=27; citation_pages=1759-1777; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Inference of the phylogenetic position of oxymonads based on nine genes: Support for Metamonada and Excavata.; citation_author=V Hampl; citation_author=DS Horner; citation_author=P Dyal; citation_author=J Kulda; citation_author=J Flegr; citation_journal_title=Mol Biol Evol; citation_volume=22; citation_number=28; citation_pages=2508-2518; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Comprehensive multigene phylogenies of excavate protists reveal the evolutionary positions of “primitive” eukaryotes.; citation_author=AGB Simpson; citation_author=Y Inagaki; citation_author=AJ Roger; citation_journal_title=Mol Biol Evol; citation_volume=23; citation_number=29; citation_pages=615-625; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Ultrastructure of Trimastix convexa Hollande, an amitochondriate anaerobic flagellate with a previously undescribed organization.; citation_author=G Brugerolle; citation_author=D Patterson; citation_journal_title=Europ J Protistol; citation_volume=33; citation_number=30; citation_pages=121-130; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Ultrastructure of Trimastix pyriformis (Klebs) Bernard et al.: similarities of Trimastix species with retortamonad and jakobid flagellates.; citation_author=CJ O'Kelly; citation_author=MA Farmer; citation_author=TA Nerad; citation_journal_title=Protist; citation_volume=150; citation_number=31; citation_pages=149-162; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=The ultrastructure of Trimastix marina Kent, 1880 (eukaryota), an excavate flagellate.; citation_author=AGB Simpson; citation_author=C Bernard; citation_author=DJ Patterson; citation_journal_title=Europ J Protistol; citation_volume=36; citation_number=32; citation_pages=229-251; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Flagellates of the human intestine and of intestines of other species.; citation_author=J Kulda; citation_author=E Nohýnková; citation_number=33; citation_pages=1-138; citation_date=1978; citation_publisher=Academic Press; " />
      <meta name="citation_reference" content="citation_title=How oxymonads lost their groove: An ultrastructural comparison of Monocercomonoides and excavate taxa.; citation_author=AGB Simpson; citation_author=R Radek; citation_author=JB Dacks; citation_author=CJ O'Kelly; citation_journal_title=J Euk Microbiol; citation_volume=49; citation_number=34; citation_pages=239-248; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Symbiotic innovation in the oxymonad Streblomastix strix.; citation_author=BS Leander; citation_author=PJ Keeling; citation_journal_title=J Euk Microbiol; citation_volume=51; citation_number=35; citation_pages=291-300; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=The Ultrastructure of Pyrsonympha and Its Associated Microorganisms.; citation_author=RA Bloodgood; citation_author=KR Miller; citation_author=TP Fitzharris; citation_author=JR McIntosh; citation_journal_title=J Morph; citation_volume=143; citation_number=36; citation_pages=77-105; citation_date=1974; " />
      <meta name="citation_reference" content="citation_title=Surface Morphology of Saccinobaculus (Oxymonadida): Implications for Character Evolution and Function in Oxymonads.; citation_author=KJ Carpenter; citation_author=RF Waller; citation_author=PJ Keeling; citation_journal_title=Protist.; citation_number=37; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=An energy-conserving pyruvate-to-acetate pathway in Entamoeba histolytica. Pyruvate synthase and a new acetate thiokinase.; citation_author=RE Reeves; citation_author=LG Warren; citation_author=B Susskind; citation_author=HS Loi; citation_journal_title=J Biol Chem; citation_volume=252; citation_number=38; citation_pages=726-731; citation_date=1977; " />
      <meta name="citation_reference" content="citation_title=Characterisation and purification of pyruvate:ferredoxin oxidoreductase from Giardia duodenalis.; citation_author=SM Townson; citation_author=JA Upcroft; citation_author=P Upcroft; citation_journal_title=Mol Biochem Parasitol; citation_volume=79; citation_number=39; citation_pages=183-193; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=The Hydrogenosome.; citation_author=M Müller; citation_journal_title=J Gen Microbiol; citation_volume=139; citation_number=40; citation_pages=2879-2889; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=Classification and phylogeny of hydrogenases.; citation_author=PM Vignais; citation_author=B Billoud; citation_author=J Meyer; citation_journal_title=Fems Microbiol Rev; citation_volume=25; citation_number=41; citation_pages=455-501; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=[FeFe] hydrogenases and their evolution: a genomic perspective.; citation_author=J Meyer; citation_journal_title=Cell Mol Life Sci; citation_volume=64; citation_number=42; citation_pages=1063-1084; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Discovery of two novel radical S-adenosylmethionine proteins required for the assembly of an active [Fe] hydrogenase.; citation_author=MC Posewitz; citation_author=PW King; citation_author=SL Smolinski; citation_author=L Zhang; citation_author=M Seibert; citation_author=ML Ghirardi; citation_journal_title=J Biol Chem; citation_volume=279; citation_number=43; citation_pages=25711-25720; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Fe-hydrogenase maturases in the hydrogenosomes of Trichomonas vaginalis.; citation_author=S Putz; citation_author=P Doležal; citation_author=G Gelius-Dietrich; citation_author=L Boháčová; citation_author=J Tachezy; citation_author=K Henze; citation_journal_title=Eukaryot Cell; citation_volume=5; citation_number=44; citation_pages=579-586; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=The glycine decarboxylase system: a fascinating complex.; citation_author=R Douce; citation_author=J Bourguignon; citation_author=M Neuburger; citation_author=F Rebeille; citation_journal_title=Trends Plant Sci; citation_volume=6; citation_number=45; citation_pages=167-176; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Dihydrolipoamide dehydrogenase: functional similarities and divergent evolution of the pyridine nucleotide-disulfide oxidoreductases.; citation_author=DJ Carothers; citation_author=G Pons; citation_author=MS Patel; citation_journal_title=Arch Biochem Biophys; citation_volume=268; citation_number=46; citation_pages=409-425; citation_date=1989; " />
      <meta name="citation_reference" content="citation_title=Proteins of the glycine decarboxylase complex in the hydrogenosome of Trichomonas vaginalis.; citation_author=M Mukherjee; citation_author=MT Brown; citation_author=AG McArthur; citation_author=PJ Johnson; citation_journal_title=Eukaryot Cell; citation_volume=5; citation_number=47; citation_pages=2062-2071; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Biochemical and molecular characterization of the Clostridium magnum acetoin dehydrogenase enzyme system.; citation_author=N Kruger; citation_author=FB Oppermann; citation_author=H Lorenzl; citation_author=A Steinbuchel; citation_journal_title=J Bacteriol; citation_volume=176; citation_number=48; citation_pages=3614-3630; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=A new metabolic link. The acyl carrier protein of lipid synthesis donates lipoic acid to the pyruvate dehydrogenase complex in Escherichia coli and mitochondria.; citation_author=SW Jordan; citation_author=JE Cronan; citation_journal_title=J Biol Chem; citation_volume=272; citation_number=49; citation_pages=17903-17906; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Molecular biology of nicotinamide nucleotide transhydrogenase–a unique proton pump.; citation_author=T Olausson; citation_author=O Fjellstrom; citation_author=J Meuller; citation_author=J Rydstrom; citation_journal_title=Biochim Biophys Acta; citation_volume=1231; citation_number=50; citation_pages=1-19; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Evolution of the molecular machines for protein import into mitochondria.; citation_author=P Doležal; citation_author=V Likic; citation_author=J Tachezy; citation_author=T Lithgow; citation_journal_title=Science; citation_volume=313; citation_number=51; citation_pages=314-318; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Tom22', an 8-kDa trans-site receptor in plants and protozoans, is a conserved feature of the TOM complex that appeared early in the evolution of eukaryotes.; citation_author=D Macasev; citation_author=J Whelan; citation_author=E Newbigin; citation_author=MC Silva-Filho; citation_author=TD Mulhern; citation_journal_title=Mol Biol Evol; citation_volume=21; citation_number=52; citation_pages=1557-1564; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Functional cooperation of the mitochondrial processing peptidase subunits.; citation_author=P Luciano; citation_author=S Geoffroy; citation_author=A Brandt; citation_author=JF Hernandez; citation_author=V Geli; citation_journal_title=J Mol Biol; citation_volume=272; citation_number=53; citation_pages=213-225; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Protein import, replication, and inheritance of a vestigial mitochondrion.; citation_author=A Regoes; citation_author=D Zourmpanou; citation_author=G Leon-Avila; citation_author=M van der Giezen; citation_author=J Tovar; citation_journal_title=J Biol Chem; citation_volume=280; citation_number=54; citation_pages=30557-30563; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Giardia mitosomes and trichomonad hydrogenosomes share a common mode of protein targeting.; citation_author=P Doležal; citation_author=O Šmíd; citation_author=P Rada; citation_author=Z Zubáčová; citation_author=D Bursac; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=102; citation_number=55; citation_pages=10924-10929; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=A functionally divergent hydrogenosomal peptidase with protomitochondrial ancestry.; citation_author=MT Brown; citation_author=HM Goldstone; citation_author=F Bastida-Corcuera; citation_author=MG gadillo-Correa; citation_author=AG McArthur; citation_journal_title=Mol Microbiol; citation_volume=64; citation_number=56; citation_pages=1154-1163; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=The mitochondrial processing peptidase.; citation_author=HP Braun; citation_author=UK Schmitz; citation_journal_title=Int J Biochem Cell Biol; citation_volume=29; citation_number=57; citation_pages=1043-1045; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Glycine-rich region of mitochondrial processing peptidase alpha-subunit is essential for binding and cleavage of the precursor proteins.; citation_author=Y Nagao; citation_author=S Kitada; citation_author=K Kojima; citation_author=H Toh; citation_author=S Kuhara; citation_journal_title=J Biol Chem; citation_volume=275; citation_number=58; citation_pages=34552-34556; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Identification and functional reconstitution of the yeast peroxisomal adenine nucleotide transporter.; citation_author=L Palmieri; citation_author=H Rottensteiner; citation_author=W Girzalsky; citation_author=P Scarcia; citation_author=F Palmieri; citation_journal_title=EMBO J; citation_volume=20; citation_number=59; citation_pages=5049-5059; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Folate metabolism in plants: an Arabidopsis homolog of the mammalian mitochondrial folate transporter mediates folate import into chloroplasts.; citation_author=M Bedhomme; citation_author=M Hoffmann; citation_author=EA McCarthy; citation_author=B Gambonnet; citation_author=RG Moran; citation_journal_title=J Biol Chem; citation_volume=280; citation_number=60; citation_pages=34823-34831; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Mitochondrial carrier family: Repertoire and peculiarities of the cellular slime mould Dictyostelium discoideum.; citation_author=M Satre; citation_author=S Mattei; citation_author=L Aubry; citation_author=P Gaudet; citation_author=L Pelosi; citation_journal_title=Biochimie; citation_volume=89; citation_number=61; citation_pages=1058-1069; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Presence of a member of the mitochondrial carrier family in hydrogenosomes: Conservation of membrane-targeting pathways between hydrogenosomes and mitochondria.; citation_author=SD Dyall; citation_author=CM Koehler; citation_author=MG Delgadillo-Correa; citation_author=PJ Bradley; citation_author=E Plumper; citation_journal_title=Mol Cell Biol; citation_volume=20; citation_number=62; citation_pages=2488-2497; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Conserved properties of hydrogenosomal and mitochondrial ADP/ATP carriers: a common origin for both organelles.; citation_author=M van der Giezen; citation_author=DJ Slotboom; citation_author=DS Horner; citation_author=PL Dyal; citation_author=M Harding; citation_journal_title=EMBO J; citation_volume=21; citation_number=63; citation_pages=572-579; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=A novel ADP/ATP transporter in the mitosome of the microaerophilic human parasite Entamoeba histolytica.; citation_author=KW Chan; citation_author=DJ Slotboom; citation_author=S Cox; citation_author=TM Embley; citation_author=O Fabre; citation_journal_title=Curr Biol; citation_volume=15; citation_number=64; citation_pages=737-742; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=The conserved substrate binding site of mitochondrial carriers.; citation_author=ER Kunji; citation_author=AJ Robinson; citation_journal_title=Biochim Biophys Acta; citation_volume=1757; citation_number=65; citation_pages=1237-1248; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Characterization of the human heart mitochondrial proteome.; citation_author=SW Taylor; citation_author=E Fahy; citation_author=B Zhang; citation_author=GM Glenn; citation_author=DE Warnock; citation_journal_title=Nat Biotechnol; citation_volume=21; citation_number=66; citation_pages=281-286; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=MitoP2, an integrated database on mitochondrial proteins in yeast and man.; citation_author=C Andreoli; citation_author=H Prokisch; citation_author=K Hortnagel; citation_author=JC Mueller; citation_author=M Munsterkotter; citation_journal_title=Nucleic Acids Res; citation_volume=32; citation_number=67; citation_pages=D459-D462; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Gapped BLAST and PSI-BLAST: a new generation of protein database search programs.; citation_author=SF Altschul; citation_author=TL Madden; citation_author=AA Schaffer; citation_author=J Zhang; citation_author=Z Zhang; citation_journal_title=Nucleic Acids Res; citation_volume=25; citation_number=68; citation_pages=3389-3402; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=BioEdit: a user-friendly biological sequence alignment editor and analysis program for Windows 95/98/NT.; citation_author=TA Hall; citation_journal_title=Nucl Acids Symp Ser; citation_number=69; citation_pages=95-98; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=ProbCons: Probabilistic consistency-based multiple sequence alignment.; citation_author=CB Do; citation_author=MS Mahabhashyam; citation_author=M Brudno; citation_author=S Batzoglou; citation_journal_title=Genome Res; citation_volume=15; citation_number=70; citation_pages=330-340; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=RAxML-VI-HPC: maximum likelihood-based phylogenetic analyses with thousands of taxa and mixed models.; citation_author=A Stamatakis; citation_journal_title=Bioinformatics; citation_volume=22; citation_number=71; citation_pages=2688-2690; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=MrBayes 3: Bayesian phylogenetic inference under mixed models.; citation_author=F Ronquist; citation_author=JP Huelsenbeck; citation_journal_title=Bioinformatics; citation_volume=19; citation_number=72; citation_pages=1572-1574; citation_date=2003; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001383" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Genetic Evidence for a Mitochondriate Ancestry in the ‘Amitochondriate’ Flagellate Trimastix pyriformis"/>
    <meta name="twitter:description" content="Most modern eukaryotes diverged from a common ancestor that contained the &alpha;-proteobacterial endosymbiont that gave rise to mitochondria. The &lsquo;amitochondriate&rsquo; anaerobic protist parasites that have been studied to date, such as Giardia and Trichomonas harbor mitochondrion-related organelles, such as mitosomes or hydrogenosomes. Yet there is one remaining group of mitochondrion-lacking flagellates known as the Preaxostyla that could represent a primitive &lsquo;pre-mitochondrial&rsquo; lineage of eukaryotes. To test this hypothesis, we conducted an expressed sequence tag (EST) survey on the preaxostylid flagellate Trimastix pyriformis, a poorly-studied free-living anaerobe. Among the ESTs we detected 19 proteins that, in other eukaryotes, typically function in mitochondria, hydrogenosomes or mitosomes, 12 of which are found exclusively within these organelles. Interestingly, one of the proteins, aconitase, functions in the tricarboxylic acid cycle typical of aerobic mitochondria, whereas others, such as pyruvate:ferredoxin oxidoreductase and [FeFe] hydrogenase, are characteristic of anaerobic hydrogenosomes. Since Trimastix retains genetic evidence of a mitochondriate ancestry, we can now say definitively that all known living eukaryote lineages descend from a common ancestor that had mitochondria."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0001383.g002"/>

  <meta property="og:title" content="Genetic Evidence for a Mitochondriate Ancestry in the ‘Amitochondriate’ Flagellate Trimastix pyriformis" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001383" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=9829'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=784'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=4727&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0001383">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001383" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001383&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Genetic Evidence for a Mitochondriate Ancestry in the ‘Amitochondriate’ Flagellate <i>Trimastix pyriformis</i>
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Vladimir Hampl, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Biochemistry and Molecular Biology, Dalhousie University, Halifax, Nova Scotia, Canada
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Jeffrey D. Silberman, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliations:
                  Department of Biochemistry and Molecular Biology, Dalhousie University, Halifax, Nova Scotia, Canada, 
                  Department of Biological Sciences, University of Arkansas, Fayetteville, Arkansas, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Alexandra Stechmann, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Biochemistry and Molecular Biology, Dalhousie University, Halifax, Nova Scotia, Canada
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Sara Diaz-Triviño, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Biochemistry and Molecular Biology, Dalhousie University, Halifax, Nova Scotia, Canada
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Patricia J. Johnson, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Microbiology, Immunology, and Molecular Genetics, University of California at Los Angeles, Los Angeles, California, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Andrew J. Roger
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:Andrew.Roger@Dal.Ca">Andrew.Roger@Dal.Ca</a></p>

                <p>Affiliation:
                  Department of Biochemistry and Molecular Biology, Dalhousie University, Halifax, Nova Scotia, Canada
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: January 02, 2008</li>
    <li>DOI: 10.1371/journal.pone.0001383</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0001383" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0001383" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001383" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0001383" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0001383-t001" data-doi="info:doi/10.1371/journal.pone.0001383" data-uri="info:doi/10.1371/journal.pone.0001383.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001383.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001383-g001" data-doi="info:doi/10.1371/journal.pone.0001383" data-uri="info:doi/10.1371/journal.pone.0001383.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001383.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001383-g002" data-doi="info:doi/10.1371/journal.pone.0001383" data-uri="info:doi/10.1371/journal.pone.0001383.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001383.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001383">Reader Comments (0)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0001383" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1"></a><p>Most modern eukaryotes diverged from a common ancestor that contained the α-proteobacterial endosymbiont that gave rise to mitochondria. The ‘amitochondriate’ anaerobic protist parasites that have been studied to date, such as <em>Giardia</em> and <em>Trichomonas</em> harbor mitochondrion-related organelles, such as mitosomes or hydrogenosomes. Yet there is one remaining group of mitochondrion-lacking flagellates known as the Preaxostyla that could represent a primitive ‘pre-mitochondrial’ lineage of eukaryotes. To test this hypothesis, we conducted an expressed sequence tag (EST) survey on the preaxostylid flagellate <em>Trimastix pyriformis</em>, a poorly-studied free-living anaerobe. Among the ESTs we detected 19 proteins that, in other eukaryotes, typically function in mitochondria, hydrogenosomes or mitosomes, 12 of which are found exclusively within these organelles. Interestingly, one of the proteins, aconitase, functions in the tricarboxylic acid cycle typical of aerobic mitochondria, whereas others, such as pyruvate:ferredoxin oxidoreductase and [FeFe] hydrogenase, are characteristic of anaerobic hydrogenosomes. Since <em>Trimastix</em> retains genetic evidence of a mitochondriate ancestry, we can now say definitively that all known living eukaryote lineages descend from a common ancestor that had mitochondria.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>Hampl V, Silberman JD, Stechmann A, Diaz-Triviño S, Johnson PJ, et al.  (2008) Genetic Evidence for a Mitochondriate Ancestry in the ‘Amitochondriate’ Flagellate <em>Trimastix pyriformis</em>. PLoS ONE 3(1):
          e1383.
            doi:10.1371/journal.pone.0001383</p><p><strong>Academic Editor: </strong>Rosemary Redfield, University of British Columbia, Canada</p><p><strong>Received:</strong> September 4, 2007; <strong>Accepted:</strong> December 7, 2007; <strong>Published:</strong> January 2, 2008</p><p><strong>Copyright:</strong> © 2008 Hampl et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>Trimastix pyriformis EST data were generated as part of AJRs involvement in the Genome Atlantic/Genome Canada sponsored Protist EST Program Project. This work and partial salary support for VH, JDS and SDT came from an operating grant MOP-62809 from the Canadian Institutes for Health Research (CIHR) awarded to AJR. VH was also supported by a fellowship from the Canadian Institute for Advanced Research (CIFAR). AS was supported by a postdoctoral fellowship from the Canadian Institutes of Health Research (CIHR). AJR is supported by the E.W.R. Steacie Memorial Fellowship from the Natural Sciences and Engineering Research Council of Canada and the CIFAR Program in Integrated Microbial Diversity.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>The origin of the eukaryotic cell and mitochondria were major transitions in the evolution of life. However, the mechanisms and the temporal ordering of events underlying these transitions remain poorly understood. There are two main kinds of hypotheses regarding the sequence of events for these transitions. The first kind invokes the origin of the nucleus, cytoskeleton and endomembrane system to yield an amitochondriate eukaryote, followed later by the acquisition of the mitochondrion through endosymbiosis <a href="#pone.0001383-CavalierSmith1">[1]</a>–<a href="#pone.0001383-Moreira1">[3]</a>. The second kind proposes that the mitochondrial endosymbiosis is the key innovation in eukaryogenesis, occurring simultaneously with the formation of the nucleus, or even beforehand <a href="#pone.0001383-Martin1">[4]</a>, <a href="#pone.0001383-Karlin1">[5]</a>. An important difference between these scenarios is that the first predicts that primitively amitochondriate eukaryotes (Archezoa) exist, or once existed but are now extinct, whereas, according to the latter hypotheses, no such organisms ever existed. Among unicellular eukaryotes there are several taxa that lack classical mitochondria (e.g. diplomonads, trichomonads, <em>Entamoeba,</em> pelobionts, <em>Cryptosporidium</em>, chytrid fungi, microsporidia, some ciliates and heterolobosea) and some of these groups were thought to actually be representatives of primitively amitochondriate Archezoa <a href="#pone.0001383-CavalierSmith2">[6]</a>. However, genes of mitochondrial origin have been identified in all of these groups indicating that they contain (or once contained) an organelle homologous to mitochondria <a href="#pone.0001383-Clark1">[7]</a>–<a href="#pone.0001383-Dyall1">[22]</a>. The nuclear location of most genes encoding mitochondrial proteins is the result of genetic transfer from the endosymbiotic α-proteobacterial ancestor of the organelle. Immunological detection of proteins that function within the mitochondrion, such as those involved in iron-sulphur cluster biogenesis (IscS, IscU) or in protein import and refolding (mt-hsp70, cpn60) enabled the visualization of double-membrane bounded organelles in these “amitochondriates” <a href="#pone.0001383-Bozner1">[12]</a>, <a href="#pone.0001383-Tovar1">[15]</a>, <a href="#pone.0001383-Williams1">[18]</a>, <a href="#pone.0001383-vanderGiezen1">[20]</a>, <a href="#pone.0001383-Tovar2">[23]</a>, <a href="#pone.0001383-Riordan1">[24]</a>. These organelles likely share a common evolutionary history with mitochondria implying that these organisms and presumably all modern eukaryotes diverged from an organism that contained a mitochondrion or its homolog <a href="#pone.0001383-Embley1">[25]</a>.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>However, there is still one key lineage of amitochondriate protists that has not been investigated: the Preaxostyla. This group is comprised of oxymonads, gut symbionts in animals, and the free-living flagellates of the genus <em>Trimastix</em> <a href="#pone.0001383-Dacks1">[26]</a>, <a href="#pone.0001383-Simpson1">[27]</a>. The phylogenetic position of Preaxostyla has not yet been firmly established (compare <a href="#pone.0001383-Hampl1">[28]</a> and <a href="#pone.0001383-Simpson2">[29]</a>), but they are regarded as members of a eukaryotic “supergroup” Excavata <a href="#pone.0001383-CavalierSmith1">[1]</a>, <a href="#pone.0001383-Simpson1">[27]</a>. To date, the monophyly of the Excavata has not been proven, and thus it remains possible (although improbable) that Preaxostyla emerge at the base of eukaryotes, a position that would be consistent with a primitively amitochondriate status for this lineage. Double membrane bounded organelles presumed to be related to hydrogenosomes and mitochondria can be found in the cytoplasm of <em>Trimastix</em> <a href="#pone.0001383-Brugerolle1">[30]</a>–<a href="#pone.0001383-Simpson3">[32]</a>. Typically, no such organelles are observed in oxymonads <a href="#pone.0001383-Kulda1">[33]</a>–<a href="#pone.0001383-Leander1">[35]</a>. Bloodgood et al. <a href="#pone.0001383-Bloodgood1">[36]</a> reported large dense cytoplasmic bodies in the oxymonad <em>Pyrsonympha</em>, however, these authors consider them to be neither hydrogenosomes nor other derivatives of mitochondria and they may represent endosymbiotic or engulfed bacteria. Carpenter et al. <a href="#pone.0001383-Carpenter1">[37]</a> reported membrane-bounded, rounded, electron-dense bodies present in <em>Saccinobaculus doroaxostylus</em>, but absent in other investigated species of the genus. Some of their micrographs suggest that this body may be bounded by two membranes. We have chosen <em>Trimastix pyriformis</em>, as the representative of Preaxostyla, for an expressed sequence tag (EST) survey to search for genes of mitochondrial origin. The survey revealed 19 genes typical for mitochondria or hydrogenosomes that potentially function in a mitochondrion-related organelle in <em>Trimastix</em>.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results and Discussion"></a><h3>Results and Discussion</h3><a id="article1.body1.sec2.p1" name="article1.body1.sec2.p1"></a><p>We constructed a cDNA library from <em>Trimastix pyriformis</em> and sequenced 9615 expressed sequence tags (ESTs) that grouped into 2686 clusters. Using our bioinformatic tool Blastcompare, we found genes that unambiguously code for proteins functioning in mitochondria and related organelles in other organisms. In addition to these genes, we also found genes coding for the typical hydrogenosomal enzymes pyruvate:ferredoxin oxidoreductase (PFO), [FeFe] hydrogenase and two [FeFe] hydrogenase maturases–hydE and hydG. The complete list of putative organellar genes is given in <a href="#pone-0001383-t001">Table 1</a>.</p>
<div class="figure" id="pone-0001383-t001"><div class="img"><a name="pone-0001383-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001383.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001383" data-uri="info:doi/10.1371/journal.pone.0001383.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001383.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001383.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001383.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001383.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001383.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001383.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001383.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001383.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001383.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>Information on the putative organellar genes in <em>Trimastix</em>.</span></strong></p><span>doi:10.1371/journal.pone.0001383.t001</span></div><a id="article1.body1.sec2.p2" name="article1.body1.sec2.p2"></a><p>The codon usages of the genes are similar to the codon usages of other <em>Trimastix</em> genes. Preferred codons have relatively high GC content and most often contain C in the 3<sup>rd</sup> position. This observation increased our confidence that the genes originated from <em>Trimastix</em> transcripts and do not represent the contamination from bacteria that are present in the culture.</p>
<a id="article1.body1.sec2.p3" name="article1.body1.sec2.p3"></a><p>We used rapid amplification of cDNA ends (RACE) to characterize the full-length sequences of some of these genes. For nine genes, the N-terminus of the protein sequence was identified. Four of them contained N-terminal extensions compared to bacteria. They were not recognized as mitochondrial pre-sequences by prediction software TargetP and Mitoprot, but possessed relevant hallmarks of targeting signals–rich in small, hydroxylated, hydrophobic and positively charged amino acids (<a href="#pone-0001383-t001">Table 1</a>, <a href="#pone-0001383-g001">Figure 1</a>).</p>
<div class="figure" id="pone-0001383-g001"><div class="img"><a name="pone-0001383-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001383.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001383" data-uri="info:doi/10.1371/journal.pone.0001383.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001383.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001383.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001383.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001383.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001383.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001383.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001383.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001383.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001383.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>The N-terminal portion of alignments showing the extensions on <em>Trimastix</em> sequences.</span></strong></p><a id="article1.body1.sec2.fig1.caption1.p1" name="article1.body1.sec2.fig1.caption1.p1"></a><p>(A) H-, (B) P1-, (C) T-proteins of GCS and (D) cpn60.</p>
<span>doi:10.1371/journal.pone.0001383.g001</span></div><a id="article1.body1.sec2.p4" name="article1.body1.sec2.p4"></a><p>The cellular localization of the gene products was not proven experimentally; however, we could tentatively infer their localization based on their phylogenetic relationship to other mitochondrial or hydrogenosomal homologs and, if present, on the putative N-terminal targeting peptides. The putative organellar proteins fell into five functional classes that are reviewed below.</p>

<h4>Energy metabolism</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>We identified key enzymes of anaerobic energy metabolism, PFO and [FeFe] hydrogenase and one protein involved in the energy metabolism of typical aerobic mitochondria, aconitase.</p>
<a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2"></a><p>In the majority of aerobic eukaryotes, decarboxylation of pyruvate to form acetyl-CoA is performed by the pyruvate dehydrogenase complex (PDH), which is located in mitochondria. By contrast, in anaerobes, the non-homologous enzyme PFO typically catalyzes this reaction and, in the process, electrons are transferred to a ferredoxin. In the hydrogenosomes of chytrid fungi, pyruvate is degraded by yet another type of enzyme, pyruvate:formate lyase, and no electrons are released. PFO can be located in the cytosol (i.e. <em>Giardia, Entamoeba</em>) <a href="#pone.0001383-Reeves1">[38]</a>, <a href="#pone.0001383-Townson1">[39]</a> or in hydrogenosomes (i.e. <em>Trichomonas</em>) <a href="#pone.0001383-Mller1">[40]</a>. In our phylogenetic analysis, eukaryotic PFO sequences formed a single, poorly-supported clade (<a href="#pone.0001383.s001">Figure S1</a>), in which the <em>Trimastix</em> sequence emerged as a sister lineage to two cytosolic <em>Entamoeba</em> sequences (1.00 PP/67% BP).</p>
<a id="article1.body1.sec2.sec1.p3" name="article1.body1.sec2.sec1.p3"></a><p>Hydrogenases are widely distributed among eukaryotes and prokaryotes and can be divided into three different classes: [FeFe], [NiFe] and metal-free hydrogenases <a href="#pone.0001383-Vignais1">[41]</a>, <a href="#pone.0001383-Meyer1">[42]</a>. They can be cytosolic, membrane bound, periplasmic or organellar and catalyze the coupling of electrons with protons to form hydrogen gas or the reverse reaction. We found three distinct sequences of [FeFe] hydrogenase among the ESTs (<a href="#pone-0001383-t001">Table 1</a>). The sequence EU086507 was completed on the 5′ end and sequences EU086508 and EU086509 on both ends using RACE. The obtained coding sequences differed in both length (292–445 aa) and sequence (56–70 aa differences) but formed a robust clade in the tree (not shown). EU086507 corresponded to the most abundant cluster (22 ESTs) and was used for phylogenetic reconstruction (<a href="#pone-0001383-g002">Figure 2A</a>). The sequence branched robustly outside of most other eukaryotes in a strongly supported (1.00 PP/100% BP) group that also contained three bacterial and one <em>Entamoeba</em> sequence that weakly formed its sister branch. This gene phylogeny strongly indicated that <em>Trimastix</em> acquired its hydrogenase independently from the majority of eukaryotes. The specific relationship of both key enzymes of anaerobic metabolism, PFO and [FeFe] hydrogenase, to homologs from <em>Entamoeba</em> indicates a possible lateral gene transfer of these enzymes between <em>Trimastix</em> and <em>Entamoeba.</em> Two out of three genes required for maturation of [FeFe] hydrogenases were found among the ESTs–hydE and hydG. Although these proteins are regarded as mandatory for the production of the active [FeFe] hydrogenase enzyme in bacteria <a href="#pone.0001383-Meyer1">[42]</a>, they have been reported only from two eukaryotes so far, <em>Chlamydomonas</em> <a href="#pone.0001383-Posewitz1">[43]</a> and <em>Trichomonas</em> <a href="#pone.0001383-Putz1">[44]</a>, and are absent from the draft genome sequences of <em>Giardia</em> and <em>Entamoeba</em>.</p>
<div class="figure" id="pone-0001383-g002"><div class="img"><a name="pone-0001383-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001383.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001383" data-uri="info:doi/10.1371/journal.pone.0001383.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001383.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001383.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001383.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001383.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001383.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001383.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001383.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001383.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001383.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Phylogenetic trees of (A) [FeFe] hydrogenase, (B) TOM40 and (C) cpn60.</span></strong></p><a id="article1.body1.sec2.sec1.fig1.caption1.p1" name="article1.body1.sec2.sec1.fig1.caption1.p1"></a><p>Trees were constructed using MrBayes. Numbers on the branches represent statistical support expressed in Bayesian posterior probabilities/maximum likelihood bootstrap support computed with RAxML. Asterisks (*) indicate a bootstrap value of &lt;50%. In (B) brief names of the proteins annotated according to GenBank nomenclature follow the taxon names. The statistical support for non-relevant nodes was not included for simplicity.</p>
<span>doi:10.1371/journal.pone.0001383.g002</span></div><a id="article1.body1.sec2.sec1.p4" name="article1.body1.sec2.sec1.p4"></a><p>Aconitase performs stereo-specific isomerization of citrate to isocitrate and is present both in mitochondria (a part of the TCA cycle) and the cytosol, but the two types are unrelated and form distinct clades in the phylogenetic tree (<a href="#pone.0001383.s002">Figure S2</a>). In <em>Trimastix</em>, both types were detected but the mitochondrial one appears to be more highly expressed because it was found in 11 ESTs, in contrast to the cytosolic type, the partial sequence of which was present in only a single EST (not included in the tree).</p>


<h4>Amino acid metabolism</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>Five proteins comprising a system that decarboxylates a single amino acid were detected among the <em>Trimastix</em> ESTs. They assemble to a complete glycine cleavage system complex (GCS) that is usually located in mitochondria and performs rapid breakdown of glycine molecules to produce methyl-tetrahydrofolate and NADH. The H-protein plays a pivotal role in the GCS complex; its lipoyl group interacts with three GCS proteins: P-protein (glycine dehydrogenase [decarboxylating]), L-protein (dihydrolipoyl dehydrogenase), and T-protein (aminomethyltransferase) <a href="#pone.0001383-Douce1">[45]</a>. In bacterial systems, the P-protein is composed of two subunits that are encoded by two distinct genes. This is in contrast to eukaryotes where the P-protein is encoded as a single polypeptide. In <em>Trimastix</em>, the transcripts for the two subunits were found in two different clusters, each containing a poly A tail, indicating that they may be transcribed separately. The L-protein (dihydrolipoyl dehydrogenase) is an enzyme shared by four pathways, all of which are mitochondrial. In addition to being part of the GCS, it can function as the E3 subunit of the PDH complex, the oxoglutarate dehydrogenase complex and the branched-chain alpha-keto acid dehydrogenase complex <a href="#pone.0001383-Carothers1">[46]</a>. Because the other subunits of the latter three complexes were missing among the ESTs and PFO appears to have taken the role of PDH, it is probable that the dihydrolipoyl dehydrogenase in <em>Trimastix</em> is involved exclusively in the GCS complex. Curiously, our phylogenetic analyses indicated that the GCS has a mixed evolutionary origin in <em>Trimastix</em>. While both T- and L-protein sequences were robustly embedded within the eukaryotic mitochondrial clade (<a href="#pone.0001383.s003">Figures S3</a> and <a href="#pone.0001383.s004">S4</a>), the two P-protein subunits were robustly related to α-proteobacteria (<a href="#pone.0001383.s005">Figures S5</a> and <a href="#pone.0001383.s006">S6</a>) and the H-protein branched weakly with bacteria, although the overall tree topology was only poorly supported (<a href="#pone.0001383.s007">Figure S7</a>). The N-terminus of the H-, P1- and T-proteins included extensions compared to bacterial sequences (<a href="#pone-0001383-g001">Figure 1</a>). It was recently reported that the <em>Trichomonas vaginalis</em> hydrogenosome also harbors some GCS subunits, however it seems that the GCS is incomplete, consisting only of H- and L-proteins, the latter being of apparent prokaryotic origin <a href="#pone.0001383-Mukherjee1">[47]</a>.</p>


<h4>Cofactor metabolism</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>Three proteins involved in metabolism of cofactors were detected–lipoyltransferase and both subunits of pyridine nucleotide transhydrogenase (PNT).</p>
<a id="article1.body1.sec2.sec3.p2" name="article1.body1.sec2.sec3.p2"></a><p>Lipoyltransferase typically performs the first step in lipoylation (covalent binding of lipoic acid) of several enzymes functioning in complexes involved in oxidative and amino acid metabolism <a href="#pone.0001383-Kruger1">[48]</a>, <a href="#pone.0001383-Jordan1">[49]</a>. All of these enzyme complexes are present in bacteria, and those in eukaryotes are located in mitochondria and plastids. In the phylogeny of lipoyltransferase (<a href="#pone.0001383.s008">Figure S8</a>), the <em>Trimastix</em> homolog grouped weakly with <em>Dictyostelium</em> (0.58 PP/36% BP) as a deep branch of a moderately supported clade (1.00 PP/79% BP) that consisted mostly of eukaryotes, but included one branch of archaeal and bacterial lipoyltransferases.</p>
<a id="article1.body1.sec2.sec3.p3" name="article1.body1.sec2.sec3.p3"></a><p>PNT is an enzyme exclusively located in the inner membrane of mitochondria or the cytoplasmic membrane of bacteria and transfers hydride ion equivalents between NAD(H) and NADP(H) and, in the process, translocates protons across the inner mitochondrial/cellular membrane <a href="#pone.0001383-Olausson1">[50]</a>. The enzyme is a homodimer and one monomer consists of two domains (α and β) that are expressed as two proteins in <em>Escherichia coli</em> but as a single protein in eukaryotes, e. g. <em>Bos</em>, <em>Eimeria</em> and <em>Entamoeba</em> <a href="#pone.0001383-Olausson1">[50]</a>. The two domains of the <em>Trimastix</em> PNT were found in different EST clusters, each containing a poly A tail, indicating that they are transcribed separately. The sequence available for the α-subunit was too short for a reliable phylogenetic reconstruction (&lt;&lt;100 amino acids). In the phylogenetic tree made from β subunit sequences (<a href="#pone.0001383.s009">Figure S9</a>), <em>Trimastix</em> formed a branch with <em>Entamoeba</em> (1.00 PP/81% BP) at the base of an exclusively eukaryotic clade, however with relatively low support (0.99 PP/51% BP).</p>


<h4>Protein import and maturation</h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>Three proteins involved in protein import and maturation were detected suggesting that the <em>Trimastix</em> organelle actively retains the ability to translocate (i.e. import) nuclear-encoded/cytoplasmically translated proteins. The proteins are: mitochondrial translocase of the outer membrane 40 (TOM40), chaperonin 60 (cpn60) and the α-subunit of the mitochondrial processing peptidase (α-MPP). The TOM complex is specific to eukaryotes and evolved early after endosymbiosis <a href="#pone.0001383-Doleal1">[51]</a>. TOM40 is a β-barrel protein that forms the translocation channel in the membrane. It is an essential part of the complex and it seems to be universally distributed among eukaryotes <a href="#pone.0001383-Doleal1">[51]</a>, <a href="#pone.0001383-Macasev1">[52]</a>. Our phylogenetic analyses strongly suggest that the <em>Trimastix</em> EST sequence is a TOM40 homolog as it is robustly (1.00 PP/100% BP) embedded in a clade of plant, animal and fungal sequences that, with a few exceptions, have been annotated as TOM40 (<a href="#pone-0001383-g002">Figure 2B</a>). The presence of TOM40 in the anaerobic excavate <em>Trimastix</em> further strengthens the hypothesis that this protein represents an early eukaryotic invention.</p>
<a id="article1.body1.sec2.sec4.p2" name="article1.body1.sec2.sec4.p2"></a><p>MPP is responsible for the processing of N-terminal pre-sequences after proteins have been imported into the mitochondrial matrix. MPP is active usually as a heterodimer of the paralogous alpha and beta subunits <a href="#pone.0001383-Luciano1">[53]</a>, the exceptions are <em>Trichomonas</em> and <em>Giardia</em>, in which only beta subunits were found <a href="#pone.0001383-Regoes1">[54]</a>–<a href="#pone.0001383-Brown1">[56]</a> functioning as a homodimer <a href="#pone.0001383-Brown1">[56]</a>. The alpha-subunit participates in substrate binding and possibly product release while the catalytic activity responsible for transit peptide cleavage resides in the beta-subunit <a href="#pone.0001383-Luciano1">[53]</a>, <a href="#pone.0001383-Braun1">[57]</a>. All sites required for enzymatic activity <a href="#pone.0001383-Nagao1">[58]</a> are present in the putative α-MPP protein we recovered from <em>Trimastix</em>. Phylogenetic analyses of MPP protein sequences robustly supported clades of the alpha and beta subunits (<a href="#pone.0001383.s010">Figure S10</a>) and the <em>Trimastix</em> sequence was embedded within the α-MPP clade with the <em>Trypanosoma</em> homolog as a sister branch (1.00 PP/79% BP). Eukaryote and α-proteobacterial peptidases shared a most recent common ancestor consistent with the hypothesis that MPP came into eukaryotes with the ancestor of the mitochondrion <a href="#pone.0001383-Braun1">[57]</a>. The gene duplication leading to the closely related MPP paralogs occurred very early in eukaryote history, probably before the divergence of extant eukaryotes.</p>
<a id="article1.body1.sec2.sec4.p3" name="article1.body1.sec2.sec4.p3"></a><p>Cpn60 is the mitochondrial homolog of GroEL and is involved with the refolding of proteins imported into mitochondria. This molecule is often used as a ‘mitochondrial marker’ because it unambiguously traces its ancestry to α-proteobacteria and is localized not only in mitochondria but also in the hydrogenosome of <em>Trichomonas</em> and the mitosomes of <em>Entamoeba</em> and <em>Giardia</em> <a href="#pone.0001383-Roger1">[11]</a>, <a href="#pone.0001383-Tovar1">[15]</a>, <a href="#pone.0001383-Regoes1">[54]</a>. Cpn60 from <em>Trimastix</em> branched within the eukaryote mitochondrial clade with very high statistical support (1.00 PP/93% BP), but without strong affiliation to any particular organism or group (<a href="#pone-0001383-g002">Figure 2C</a>). The various bioinformatic tools we employed did not recognize a mitochondrial targeting signal on the <em>Trimastix</em> cpn60 protein, however it does possess an N-terminal extension relative to bacterial homologs (<a href="#pone-0001383-g001">Fig. 1</a>).</p>


<h4>Transport of other molecules</h4>
<a id="article1.body1.sec2.sec5.p1" name="article1.body1.sec2.sec5.p1"></a><p>Three members of the mitochondrial carrier family were identified among the <em>Trimastix</em> ESTs (<a href="#pone-0001383-t001">Table 1</a>). This diverse family of proteins facilitates the bidirectional transport of metabolites, nucleotides, amino acids, co-factors, carboxylic acids and inorganic anions, across the inner membrane of the mitochondrion. A few members were also found in the membranes of peroxisomes and plastids <a href="#pone.0001383-Palmieri1">[59]</a>–<a href="#pone.0001383-Satre1">[61]</a>. One member of this family, the ATP/ADP translocator, was detected in the hydrogenosomal inner membrane of <em>Trichomonas</em>, <em>Neocallimastix</em> and in the mitosomal inner membrane of <em>Entamoeba</em> <a href="#pone.0001383-Dyall2">[62]</a>–<a href="#pone.0001383-Chan1">[64]</a>. The sequences of the three <em>Trimastix</em> homologs were clearly different from each other. The sequence of carrier 3 was the most divergent; moreover, its C-terminus contained a MQGP-rich repetition and showed no sequence similarity to any other eukaryote homolog. The other two carriers were included in the phylogenetic analysis of this protein family (<a href="#pone.0001383.s011">Figure S11</a>). Although support for the backbone tree topology was generally low, it roughly corresponded to substrate specificities. <em>Trimastix</em> carrier 1 showed a weak phylogenetic affiliation to adenine nucleotide (e.g. ATP, NAD) transporters and, in agreement with this, the sequence contained motifs characteristic for this category of carriers representing the binding site of adenine nucleotides <a href="#pone.0001383-Kunji1">[65]</a>–GQ at positions 182/183 and R at position 83 (numbering according to <a href="#pone.0001383-Kunji1">[65]</a>). Carrier 2 did not contain any specific binding site motifs and clustered with pyruvate and folate transporters in the tree.</p>


<h4>Conclusions</h4>
<a id="article1.body1.sec2.sec6.p1" name="article1.body1.sec2.sec6.p1"></a><p>Among the ESTs we identified 12 proteins that are unique to mitochondrial- or plastid-derived organelles and have never been observed in other cellular compartments of eukaryotes (<a href="#pone-0001383-t001">Table 1</a>). Although we do not have direct evidence of organelle targeting, four of these proteins show clear N-terminal extensions. The proteins known to be specific to the outer (TOM40) or inner (PNTα, PNTβ) membrane of mitochondria are also very unlikely to function in other ‘non-mitochondrial’ membranes in <em>Trimastix</em> and the traditional function assigned to MPP makes sense only if localized in the organellar matrix.</p>
<a id="article1.body1.sec2.sec6.p2" name="article1.body1.sec2.sec6.p2"></a><p>Considering these data, there is little doubt that <em>Trimastix</em> had a mitochondriate past. Furthermore, given the presence of N-terminal extensions on mitochondrion ‘hallmark’ proteins as well as our finding of a component of the protein import system it seems likely that <em>Trimastix</em> contains anaerobic organelles homologous to mitochondria. These in turn likely correspond to the densely-staining double membrane bounded structures described in electron micrographs of these organisms <a href="#pone.0001383-Brugerolle1">[30]</a>–<a href="#pone.0001383-Simpson3">[32]</a>. As the Preaxostyla (<em>Trimastix</em> and oxymonads) were the last major candidate primitive ‘pre-mitochondrial’ eukaryote group, we can now say definitively that all known extant eukaryote lineages diverged after the mitochondrial symbiosis.</p>
<a id="article1.body1.sec2.sec6.p3" name="article1.body1.sec2.sec6.p3"></a><p>Enzymes typical for anaerobic metabolism and the characteristic enzymes of hydrogenosomes (PFO and [FeFe] hydrogenase) are relatively highly-expressed in <em>Trimastix</em> (each comprised &gt;0.2% of the ESTs). Future localization studies are needed to establish if these enzymes are indeed active in the mitochondrion-derived organelles as in hydrogenosomes or in the cytosol, like in mitosome-containing eukaryotes. The potential presence of a TCA cycle enzyme in the organelle indicates that <em>Trimastix</em> harbors another unique version of anaerobic mitochondrion-like organelles with a unique spectrum of metabolic properties.</p>

</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4>Cultures and Molecular biology</h4>
<a id="article1.body1.sec3.sec1.p1" name="article1.body1.sec3.sec1.p1"></a><p><em>Trimastix pyriformis</em> (ATCC 50935) was grown at room temperature under anaerobic/microaerophilic conditions in 1 litre tissue culture flasks (tightly sealed) in Sonneborn's <em>Paramecium</em> medium (ATCC 802 medium) pre-inoculated with <em>Stenotrophomonas maltophilia</em> as the sole food source. Cells in exponential growth were harvested by centrifugation at 1200×g for 10 minutes at 4°C. Total RNA was isolated via Tri-reagent (Sigma-Aldrich). Approximately 3mg of total RNA was sent to a commercial vendor (Agencourt Bioscience, Beverly MA, USA), to construct the cDNA library used for EST sequencing.</p>
<a id="article1.body1.sec3.sec1.p2" name="article1.body1.sec3.sec1.p2"></a><p>Both 5′ and 3′ RACE were performed from oligo-capped 1<sup>st</sup> strand cDNA. PolyA+ RNA was enriched from approximately 1 mg of total RNA with the Poly(A) Purist kit (Ambion, Austin TX) and full-length 1<sup>st</sup> strand cDNA was prepared with the GeneRacer Kit using SuperScript III RT (Invitrogen, Carlsbad CA). This served as template for both 5′ and 3′ RACE utilizing Taq DNA polymerase (Sigma) with either the GeneRacer 5′ or 3′ primer (GR5′/GR3′ primer) plus a gene specific primer. Often nested reactions were necessary to obtain the desired RACE products: NestedGR5′/NestedGR3′ primer plus a nested gene specific primer, plus 0.5–1.0 µl (of 50 µl) of the primary RACE PCR reaction as template. Products were cloned into TOPO T/A pCR2.1 vector (Invitrogen) and sequenced. For genomic gene sequencing, gDNA was obtained either via Tri-reagent (as a “by product” of RNA isolation) or using the PureGene DNA isolation kit (Gentra Systems, Minneapolis MN). Gene specific primers were used for PCR and products were cloned into TOPO T/A pCR2.1 vector (Invitrogen) and sequenced.</p>
<a id="article1.body1.sec3.sec1.p3" name="article1.body1.sec3.sec1.p3"></a><p>Selected clones from the cDNA library were completely sequenced using vector and gene specific primers.</p>


<h4>Comparative BLAST searching of the <em>Trimastix</em> clusters</h4>
<a id="article1.body1.sec3.sec2.p1" name="article1.body1.sec3.sec2.p1"></a><p>Four databases were used in ‘subtractive’ BLAST searching to identify putative mitochondrial proteins in the <em>Trimastix</em> ESTs. These included mitochondrial proteome data bases from the human mitoproteome and the yeast Mitop2 database <a href="#pone.0001383-Taylor1">[66]</a>, <a href="#pone.0001383-Andreoli1">[67]</a> and ‘subtractive’ databases, created by removing proteins matching the mitochondrial proteomes from the whole predicted proteomes of these organisms. The <em>Trimastix</em> clusters were then compared to all four databases, human/yeast non-mitochondrial subtractive and human/yeast mitoproteome, using BLAST <a href="#pone.0001383-Altschul1">[68]</a>. The top scoring hits for each of the subtractive and mitoproteome databases were then compared and the corresponding queries were then sorted into one of four categories: 1) Both human and yeast top hits from the mitoproteome, 2) both human and yeast top hits from the subtractive proteome, 3) human and yeast top hits from different databases, 4) or there were no significant hits in any of the databases. 128 clusters that fell into category 1 or 3 were manually inspected by comparison to the GenBank non-redundant (nr) database using BLAST and ψ-BLAST. For the most ambiguous cases preliminary trees were constructed for the cluster sequence and a selection of its homologs from the nr database. This approach narrowed the selection to 18 clusters coding for 17 unique genes.</p>


<h4>Codon usage analysis and removal of probable contaminants of ESTs</h4>
<a id="article1.body1.sec3.sec3.p1" name="article1.body1.sec3.sec3.p1"></a><p>Using the INCA2 software (<a href="http://www.bioinfo-hr.org/en/research/inca/">http://www.bioinfo-hr.org/en/research/in​ca/</a>), the frequencies of codons (codon usages) of the putative mitochondrial/hydrogenosomal genes were compared with the frequency of codons found in 29 other <em>bona fide Trimastix</em> genes that were downloaded from the GenBank nucleotide nr database or assembled from <em>Trimastix</em> EST project.</p>
<a id="article1.body1.sec3.sec3.p2" name="article1.body1.sec3.sec3.p2"></a><p>One transcript for a putative mitochondrial carrier (EC837420) showed considerable differences in codon usage. Since the transcript was present in a single EST clone and the clone did not contain a poly-A tail, we regarded it to be a possible contaminant of the ESTs and removed it from the list of putative organellar proteins.</p>
<a id="article1.body1.sec3.sec3.p3" name="article1.body1.sec3.sec3.p3"></a><p>A transcript encoding the B14 subunit of the mitochondrial electron transport chain complex I (EU086486) also differed in codon usage from other <em>Trimastix</em> genes. This transcript was also present in a single EST, and so we regarded its occurrence in <em>Trimastix</em> as questionable. We carried out several checks to confirm its presence in <em>Trimastix</em>: PCR and nested PCR using exact match gene specific primers on gDNA and cDNA as template, 3′ and 5′ RACE using GeneRacer (Invitrogen, Carlsbad CA) and gene specific primers, and finally hybridization of a DIG-labeled probe prepared using the PCR digoxygenin (DIG) Synthesis Kit (Roche Diagnostics Corp.) to a Southern blot of restriction enzyme digested <em>Trimastix</em> gDNA. As none of these experiments showed positive results, we regard this transcript as a probable rare contaminant of the cDNA library and we removed it from the list of putative organellar proteins.</p>


<h4>Construction of phylogenetic trees</h4>
<a id="article1.body1.sec3.sec4.p1" name="article1.body1.sec3.sec4.p1"></a><p>Orthologs of the <em>Trimastix</em> genes were downloaded from GenBank, and from the <em>Trichomonas vaginalis</em> (<a href="http://www.tigr.org/tdb/e2k1/tvg/">http://www.tigr.org/tdb/e2k1/tvg/</a>) and <em>Entamoeba histolytica</em> (<a href="http://www.tigr.org/tdb/e2k1/eha1/">http://www.tigr.org/tdb/e2k1/eha1/</a>) genome projects. Sequences were aligned by ClustalW implemented in BioEdit 7.0.5.3 <a href="#pone.0001383-Hall1">[69]</a> or using the ProbCons server (<a href="http://probcons.stanford.edu/">http://probcons.stanford.edu/</a>) <a href="#pone.0001383-Do1">[70]</a>. Alignments were manually refined in BioEdit 7.0.5.3 and unambiguously aligned positions were subjected to phylogenetic analyses using RAxML <a href="#pone.0001383-Stamatakis1">[71]</a> and MrBayes 3.1.2. <a href="#pone.0001383-Ronquist1">[72]</a>. The PROTMIXWAG model was used in RAxML and the branching support was assessed by 100 bootstrap replicates. Two parallel runs of four chains (temp = 0.5) were run in MrBayes 3.1.2. using the JTT+Γ model with 8 discrete-rate categories. The run was considered as converged after the average standard deviation of split frequencies dropped below 0.01. The profile of tree likelihoods was inspected and the first 25% of the trees were removed from the consensus as the burn-in.</p>

</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0001383"></div><a name="pone.0001383.s001" id="pone.0001383.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001383.s001">Figure S1. </a></strong></p><a id="article1.body1.sec4.supplementary-material1.caption1.p1" name="article1.body1.sec4.supplementary-material1.caption1.p1"></a><p class="preSiDOI">Phylogenetic tree of PFO. Tree was constructed by Bayesian method. Numbers at the nodes represent statistical support expressed in Bayesian posterior probabilities/maximum likelihood bootstraps computed in RaxML. * Indicates bootstrap value below 50%.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001383.s001</p><a id="article1.body1.sec4.supplementary-material1.caption1.p2" name="article1.body1.sec4.supplementary-material1.caption1.p2"></a><p class="postSiDOI">(0.58 MB TIF)</p>
<a name="pone.0001383.s002" id="pone.0001383.s002"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001383.s002">Figure S2. </a></strong></p><a id="article1.body1.sec4.supplementary-material2.caption1.p1" name="article1.body1.sec4.supplementary-material2.caption1.p1"></a><p class="preSiDOI">Phylogenetic tree of aconitase. Tree was constructed by Bayesian method. Numbers at the nodes represent statistical support expressed in Bayesian posterior probabilities/maximum likelihood bootstraps computed in RaxML. * Indicates bootstrap value below 50%.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001383.s002</p><a id="article1.body1.sec4.supplementary-material2.caption1.p2" name="article1.body1.sec4.supplementary-material2.caption1.p2"></a><p class="postSiDOI">(0.68 MB TIF)</p>
<a name="pone.0001383.s003" id="pone.0001383.s003"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001383.s003">Figure S3. </a></strong></p><a id="article1.body1.sec4.supplementary-material3.caption1.p1" name="article1.body1.sec4.supplementary-material3.caption1.p1"></a><p class="preSiDOI">Phylogenetic tree of T-protein of GCS. Tree was constructed by Bayesian method. Numbers at the nodes represent statistical support expressed in Bayesian posterior probabilities/maximum likelihood bootstraps computed in RaxML. * Indicates bootstrap value below 50%.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001383.s003</p><a id="article1.body1.sec4.supplementary-material3.caption1.p2" name="article1.body1.sec4.supplementary-material3.caption1.p2"></a><p class="postSiDOI">(0.69 MB TIF)</p>
<a name="pone.0001383.s004" id="pone.0001383.s004"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001383.s004">Figure S4. </a></strong></p><a id="article1.body1.sec4.supplementary-material4.caption1.p1" name="article1.body1.sec4.supplementary-material4.caption1.p1"></a><p class="preSiDOI">Phylogenetic tree of L-protein of GCS. Tree was constructed by Bayesian method. Numbers at the nodes represent statistical support expressed in Bayesian posterior probabilities/maximum likelihood bootstraps computed in RaxML. * Indicates bootstrap value below 50%.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001383.s004</p><a id="article1.body1.sec4.supplementary-material4.caption1.p2" name="article1.body1.sec4.supplementary-material4.caption1.p2"></a><p class="postSiDOI">(0.52 MB TIF)</p>
<a name="pone.0001383.s005" id="pone.0001383.s005"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001383.s005">Figure S5. </a></strong></p><a id="article1.body1.sec4.supplementary-material5.caption1.p1" name="article1.body1.sec4.supplementary-material5.caption1.p1"></a><p class="preSiDOI">Phylogenetic tree of P1-protein of GCS. Tree was constructed by Bayesian method. Numbers at the nodes represent statistical support expressed in Bayesian posterior probabilities/maximum likelihood bootstraps computed in RaxML. * Indicates bootstrap value below 50%.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001383.s005</p><a id="article1.body1.sec4.supplementary-material5.caption1.p2" name="article1.body1.sec4.supplementary-material5.caption1.p2"></a><p class="postSiDOI">(0.63 MB TIF)</p>
<a name="pone.0001383.s006" id="pone.0001383.s006"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001383.s006">Figure S6. </a></strong></p><a id="article1.body1.sec4.supplementary-material6.caption1.p1" name="article1.body1.sec4.supplementary-material6.caption1.p1"></a><p class="preSiDOI">Phylogenetic tree of P2-protein of GCS. Tree was constructed by Bayesian method. Numbers at the nodes represent statistical support expressed in Bayesian posterior probabilities/maximum likelihood bootstraps computed in RaxML. * Indicates bootstrap value below 50%.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001383.s006</p><a id="article1.body1.sec4.supplementary-material6.caption1.p2" name="article1.body1.sec4.supplementary-material6.caption1.p2"></a><p class="postSiDOI">(0.71 MB TIF)</p>
<a name="pone.0001383.s007" id="pone.0001383.s007"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001383.s007">Figure S7. </a></strong></p><a id="article1.body1.sec4.supplementary-material7.caption1.p1" name="article1.body1.sec4.supplementary-material7.caption1.p1"></a><p class="preSiDOI">Phylogenetic tree of H-protein of GCS. Tree was constructed by Bayesian method. Numbers at the nodes represent statistical support expressed in Bayesian posterior probabilities/maximum likelihood bootstraps computed in RaxML. * Indicates bootstrap value below 50%.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001383.s007</p><a id="article1.body1.sec4.supplementary-material7.caption1.p2" name="article1.body1.sec4.supplementary-material7.caption1.p2"></a><p class="postSiDOI">(0.56 MB TIF)</p>
<a name="pone.0001383.s008" id="pone.0001383.s008"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001383.s008">Figure S8. </a></strong></p><a id="article1.body1.sec4.supplementary-material8.caption1.p1" name="article1.body1.sec4.supplementary-material8.caption1.p1"></a><p class="preSiDOI">Phylogenetic tree of lipoyltransferase. Tree was constructed by Bayesian method. Numbers at the nodes represent statistical support expressed in Bayesian posterior probabilities/maximum likelihood bootstraps computed in RaxML. * Indicates bootstrap value below 50%.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001383.s008</p><a id="article1.body1.sec4.supplementary-material8.caption1.p2" name="article1.body1.sec4.supplementary-material8.caption1.p2"></a><p class="postSiDOI">(0.62 MB TIF)</p>
<a name="pone.0001383.s009" id="pone.0001383.s009"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001383.s009">Figure S9. </a></strong></p><a id="article1.body1.sec4.supplementary-material9.caption1.p1" name="article1.body1.sec4.supplementary-material9.caption1.p1"></a><p class="preSiDOI">Phylogenetic tree of β subunit of pyridine nucleotide transhydrogenase. Tree was constructed by Bayesian method. Numbers at the nodes represent statistical support expressed in Bayesian posterior probabilities/maximum likelihood bootstraps computed in RaxML. * Indicates bootstrap value below 50%.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001383.s009</p><a id="article1.body1.sec4.supplementary-material9.caption1.p2" name="article1.body1.sec4.supplementary-material9.caption1.p2"></a><p class="postSiDOI">(0.54 MB TIF)</p>
<a name="pone.0001383.s010" id="pone.0001383.s010"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001383.s010">Figure S10. </a></strong></p><a id="article1.body1.sec4.supplementary-material10.caption1.p1" name="article1.body1.sec4.supplementary-material10.caption1.p1"></a><p class="preSiDOI">Phylogenetic tree of mitochondrial processing peptidase. Tree was constructed by Bayesian method. Numbers at the nodes represent statistical support expressed in Bayesian posterior probabilities/maximum likelihood bootstraps computed in RaxML. * Indicates bootstrap value below 50%.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001383.s010</p><a id="article1.body1.sec4.supplementary-material10.caption1.p2" name="article1.body1.sec4.supplementary-material10.caption1.p2"></a><p class="postSiDOI">(0.48 MB TIF)</p>
<a name="pone.0001383.s011" id="pone.0001383.s011"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001383.s011">Figure S11. </a></strong></p><a id="article1.body1.sec4.supplementary-material11.caption1.p1" name="article1.body1.sec4.supplementary-material11.caption1.p1"></a><p class="preSiDOI">Phylogenetic tree of mitochondrial carrier protein family. Tree was constructed by Bayesian method. Numbers at the nodes represent statistical support expressed in Bayesian posterior probabilities/maximum likelihood bootstraps computed in RaxML. * Indicates bootstrap value below 50%.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001383.s011</p><a id="article1.body1.sec4.supplementary-material11.caption1.p2" name="article1.body1.sec4.supplementary-material11.caption1.p2"></a><p class="postSiDOI">(0.48 MB TIF)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>The authors would like to thank Dr. Miklós Müller and an anonymous reviewer for helpful comments on the manuscript and Jacqueline de Mestral for technical assistance.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: AR. Performed the experiments: AS VH JS SD. Analyzed the data: VH. Contributed reagents/materials/analysis tools: AR PJ. Wrote the paper: VH.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0001383-CavalierSmith1" id="pone.0001383-CavalierSmith1"></a>Cavalier-Smith T (2002) The phagotrophic origin of eukaryotes and phylogenetic classification of protozoa. Int J Syst Evol Microbiol  52: 297–354.  <ul class="find" data-citedArticleID="1036668"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=The+phagotrophic+origin+of+eukaryotes+and+phylogenetic+classification+of+protozoa.&amp;auth=&amp;atitle=The+phagotrophic+origin+of+eukaryotes+and+phylogenetic+classification+of+protozoa." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+phagotrophic+origin+of+eukaryotes+and+phylogenetic+classification+of+protozoa." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+phagotrophic+origin+of+eukaryotes+and+phylogenetic+classification+of+protozoa.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0001383-Margulis1" id="pone.0001383-Margulis1"></a>Margulis L, Dolan MF, Guerrero R (2000) The chimeric eukaryote: origin of the nucleus from the karyomastigont in amitochondriate protists. Proc Natl Acad Sci U S A  97: 6954–6959.  <ul class="find" data-citedArticleID="1036724" data-doi="10.1073/pnas.97.13.6954"><li><a href="http://dx.doi.org/10.1073/pnas.97.13.6954" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+chimeric+eukaryote%3A+origin+of+the+nucleus+from+the+karyomastigont+in+amitochondriate+protists." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+chimeric+eukaryote%3A+origin+of+the+nucleus+from+the+karyomastigont+in+amitochondriate+protists.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0001383-Moreira1" id="pone.0001383-Moreira1"></a>Moreira D, Lopez-Garcia P (1998) Symbiosis between methanogenic archaea and delta-proteobacteria as the origin of eukaryotes: The syntrophic hypothesis. J Mol Evol  47: 517–530.  <ul class="find" data-citedArticleID="1036732" data-doi="10.1007/pl00006408"><li><a href="http://dx.doi.org/10.1007/pl00006408" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Symbiosis+between+methanogenic+archaea+and+delta-proteobacteria+as+the+origin+of+eukaryotes%3A+The+syntrophic+hypothesis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Symbiosis+between+methanogenic+archaea+and+delta-proteobacteria+as+the+origin+of+eukaryotes%3A+The+syntrophic+hypothesis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0001383-Martin1" id="pone.0001383-Martin1"></a>Martin W, Müller M (1998) The hydrogen hypothesis for the first eukaryote. Nature  392: 37–41.  <ul class="find" data-citedArticleID="1036726"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=The+hydrogen+hypothesis+for+the+first+eukaryote.&amp;auth=&amp;atitle=The+hydrogen+hypothesis+for+the+first+eukaryote." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+hydrogen+hypothesis+for+the+first+eukaryote." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+hydrogen+hypothesis+for+the+first+eukaryote.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0001383-Karlin1" id="pone.0001383-Karlin1"></a>Karlin S, Brocchieri L, Mrazek J, Campbell AM, Spormann AM (1999) A chimeric prokaryotic ancestry of mitochondria and primitive eukaryotes. Proc Natl Acad Sci U S A  96: 9190–9195.  <ul class="find" data-citedArticleID="1036708" data-doi="10.1073/pnas.96.16.9190"><li><a href="http://dx.doi.org/10.1073/pnas.96.16.9190" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+chimeric+prokaryotic+ancestry+of+mitochondria+and+primitive+eukaryotes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+chimeric+prokaryotic+ancestry+of+mitochondria+and+primitive+eukaryotes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0001383-CavalierSmith2" id="pone.0001383-CavalierSmith2"></a>Cavalier-Smith T (1983)  Endosymbiotic origin of the mitochondrial envelope. In: Schwemmler W, Schenk HEA, editors. Endocytobiology II. Berlin: de Gruyter. pp. 265–279.  <ul class="find-nolinks"></ul></li><li><span class="label">7.
              </span><a name="pone.0001383-Clark1" id="pone.0001383-Clark1"></a>Clark CG, Roger AJ (1995) Direct evidence for secondary loss of mitochondria in <em>Entamoeba histolytica</em>. Proc Natl Acad Sci U S A  92: 6518–6521.  <ul class="find" data-citedArticleID="1036674" data-doi="10.1073/pnas.92.14.6518"><li><a href="http://dx.doi.org/10.1073/pnas.92.14.6518" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Direct+evidence+for+secondary+loss+of+mitochondria+in+Entamoeba+histolytica." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Direct+evidence+for+secondary+loss+of+mitochondria+in+Entamoeba+histolytica.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0001383-Horner1" id="pone.0001383-Horner1"></a>Horner DS, Hirt RP, Kilvington S, Lloyd D, Embley TM (1996) Molecular data suggest an early acquisition of the mitochondrion endosymbiont. P Roy Soc Lond B Bio  263: 1053–1059.  <ul class="find" data-citedArticleID="1036700" data-doi="10.1098/rspb.1996.0155"><li><a href="http://dx.doi.org/10.1098/rspb.1996.0155" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Molecular+data+suggest+an+early+acquisition+of+the+mitochondrion+endosymbiont." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Molecular+data+suggest+an+early+acquisition+of+the+mitochondrion+endosymbiont.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0001383-Bui1" id="pone.0001383-Bui1"></a>Bui ET, Bradley PJ, Johnson PJ (1996) A common evolutionary origin for mitochondria and hydrogenosomes. Proc Natl Acad Sci U S A  93: 9651–9656.  <ul class="find" data-citedArticleID="1036662" data-doi="10.1073/pnas.93.18.9651"><li><a href="http://dx.doi.org/10.1073/pnas.93.18.9651" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+common+evolutionary+origin+for+mitochondria+and+hydrogenosomes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+common+evolutionary+origin+for+mitochondria+and+hydrogenosomes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0001383-Germot1" id="pone.0001383-Germot1"></a>Germot A, Philippe H, Le Guyader H (1996) Presence of a mitochondrial-type 70-kDa heat shock protein in <em>Trichomonas vaginalis</em> suggests a very early mitochondrial endosymbiosis in eukaryotes. Proc Natl Acad Sci U S A  93: 14614–14617.  <ul class="find" data-citedArticleID="1036692" data-doi="10.1073/pnas.93.25.14614"><li><a href="http://dx.doi.org/10.1073/pnas.93.25.14614" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Presence+of+a+mitochondrial-type+70-kDa+heat+shock+protein+in+Trichomonas+vaginalis+suggests+a+very+early+mitochondrial+endosymbiosis+in+eukaryotes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Presence+of+a+mitochondrial-type+70-kDa+heat+shock+protein+in+Trichomonas+vaginalis+suggests+a+very+early+mitochondrial+endosymbiosis+in+eukaryotes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0001383-Roger1" id="pone.0001383-Roger1"></a>Roger AJ, Clark CG, Doolittle WF (1996) A possible mitochondrial gene in the early-branching amitochondriate protist <em>Trichomonas vaginalis</em>. Proc Natl Acad Sci U S A  93: 14618–14622.  <ul class="find" data-citedArticleID="1036754" data-doi="10.1073/pnas.93.25.14618"><li><a href="http://dx.doi.org/10.1073/pnas.93.25.14618" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+possible+mitochondrial+gene+in+the+early-branching+amitochondriate+protist+Trichomonas+vaginalis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+possible+mitochondrial+gene+in+the+early-branching+amitochondriate+protist+Trichomonas+vaginalis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0001383-Bozner1" id="pone.0001383-Bozner1"></a>Bozner P (1997) Immunological detection and subcellular localization of HSP70 and HSP60 homologs in <em>Trichomonas vaginalis</em>. J Parasitol  83: 224–229.  <ul class="find" data-citedArticleID="1036654" data-doi="10.2307/3284444"><li><a href="http://dx.doi.org/10.2307/3284444" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Immunological+detection+and+subcellular+localization+of+HSP70+and+HSP60+homologs+in+Trichomonas+vaginalis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Immunological+detection+and+subcellular+localization+of+HSP70+and+HSP60+homologs+in+Trichomonas+vaginalis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0001383-Hirt1" id="pone.0001383-Hirt1"></a>Hirt RP, Healy B, Vossbrinck CR, Canning EU, Embley TM (1997) A mitochondrial Hsp70 orthologue in <em>Vairimorpha necatrix</em>: Molecular evidence that microsporidia once contained mitochondria. Curr Biol  7: 995–998.  <ul class="find" data-citedArticleID="1036698" data-doi="10.1016/s0960-9822(06)00420-9"><li><a href="http://dx.doi.org/10.1016/s0960-9822(06)00420-9" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+mitochondrial+Hsp70+orthologue+in+Vairimorpha+necatrix%3A+Molecular+evidence+that+microsporidia+once+contained+mitochondria." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+mitochondrial+Hsp70+orthologue+in+Vairimorpha+necatrix%3A+Molecular+evidence+that+microsporidia+once+contained+mitochondria.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0001383-Roger2" id="pone.0001383-Roger2"></a>Roger AJ, Svard SG, Tovar J, Clark CG, Smith MW, et al.  (1998) A mitochondrial-like chaperonin 60 gene in <em>Giardia lamblia</em>: evidence that diplomonads once harbored an endosymbiont related to the progenitor of mitochondria. Proc Natl Acad Sci U S A  95: 229–234.  <ul class="find" data-citedArticleID="1036756" data-doi="10.1073/pnas.95.1.229"><li><a href="http://dx.doi.org/10.1073/pnas.95.1.229" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+mitochondrial-like+chaperonin+60+gene+in+Giardia+lamblia%3A+evidence+that+diplomonads+once+harbored+an+endosymbiont+related+to+the+progenitor+of+mitochondria." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+mitochondrial-like+chaperonin+60+gene+in+Giardia+lamblia%3A+evidence+that+diplomonads+once+harbored+an+endosymbiont+related+to+the+progenitor+of+mitochondria.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0001383-Tovar1" id="pone.0001383-Tovar1"></a>Tovar J, Fischer A, Clark CG (1999) The mitosome, a novel organelle related to mitochondria in the amitochondrial parasite <em>Entamoeba histolytica</em>. Mol Microbiol  32: 1013–1021.  <ul class="find" data-citedArticleID="1036776" data-doi="10.1046/j.1365-2958.1999.01414.x"><li><a href="http://dx.doi.org/10.1046/j.1365-2958.1999.01414.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+mitosome%2C+a+novel+organelle+related+to+mitochondria+in+the+amitochondrial+parasite+Entamoeba+histolytica." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+mitosome%2C+a+novel+organelle+related+to+mitochondria+in+the+amitochondrial+parasite+Entamoeba+histolytica.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0001383-Tachezy1" id="pone.0001383-Tachezy1"></a>Tachezy J, Sánchez LB, Müller M (2001) Mitochondrial type iron-sulfur cluster assembly in the amitochondriate eukaryotes <em>Trichomonas vaginalis</em> and <em>Giardia intestinalis</em>, as indicated by the phylogeny of IscS. Mol Biol Evol  18: 1919–1928.  <ul class="find" data-citedArticleID="1036772" data-doi="10.1093/oxfordjournals.molbev.a003732"><li><a href="http://dx.doi.org/10.1093/oxfordjournals.molbev.a003732" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mitochondrial+type+iron-sulfur+cluster+assembly+in+the+amitochondriate+eukaryotes+Trichomonas+vaginalis+and+Giardia+intestinalis%2C+as+indicated+by+the+phylogeny+of+IscS." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mitochondrial+type+iron-sulfur+cluster+assembly+in+the+amitochondriate+eukaryotes+Trichomonas+vaginalis+and+Giardia+intestinalis%2C+as+indicated+by+the+phylogeny+of+IscS.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0001383-Horner2" id="pone.0001383-Horner2"></a>Horner DS, Embley TM (2001) Chaperonin 60 phylogeny provides further evidence for secondary loss of mitochondria among putative early-branching eukaryotes. Mol Biol Evol  18: 1970–1975.  <ul class="find" data-citedArticleID="1036702" data-doi="10.1093/oxfordjournals.molbev.a003737"><li><a href="http://dx.doi.org/10.1093/oxfordjournals.molbev.a003737" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Chaperonin+60+phylogeny+provides+further+evidence+for+secondary+loss+of+mitochondria+among+putative+early-branching+eukaryotes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Chaperonin+60+phylogeny+provides+further+evidence+for+secondary+loss+of+mitochondria+among+putative+early-branching+eukaryotes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0001383-Williams1" id="pone.0001383-Williams1"></a>Williams BAP, Hirt RP, Lucocq JM, Embley TM (2002) A mitochondrial remnant in the microsporidian <em>Trachipleistophora hominis</em>. Nature  418: 865–869.  <ul class="find" data-citedArticleID="1036784" data-doi="10.1038/nature00949"><li><a href="http://dx.doi.org/10.1038/nature00949" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+mitochondrial+remnant+in+the+microsporidian+Trachipleistophora+hominis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+mitochondrial+remnant+in+the+microsporidian+Trachipleistophora+hominis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0001383-LaGier1" id="pone.0001383-LaGier1"></a>LaGier MJ, Tachezy J, Stejskal F, Kutišová K, Keithly JS (2003) Mitochondrial-type iron-sulfur cluster biosynthesis genes (IscS and IscU) in the apicomplexan <em>Cryptosporidium parvum</em>. Microbiology  149: 3519–3530.  <ul class="find" data-citedArticleID="1036716" data-doi="10.1099/mic.0.26365-0"><li><a href="http://dx.doi.org/10.1099/mic.0.26365-0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mitochondrial-type+iron-sulfur+cluster+biosynthesis+genes+%28IscS+and+IscU%29+in+the+apicomplexan+Cryptosporidium+parvum." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mitochondrial-type+iron-sulfur+cluster+biosynthesis+genes+%28IscS+and+IscU%29+in+the+apicomplexan+Cryptosporidium+parvum.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0001383-vanderGiezen1" id="pone.0001383-vanderGiezen1"></a>van der Giezen M, Birdsey GM, Horner DS, Lucocq J, Dyal PL, et al.  (2003) Fungal hydrogenosomes contain mitochondrial heat-shock proteins. Mol Biol Evol  20: 1051–1061.  <ul class="find" data-citedArticleID="1036786" data-doi="10.1093/molbev/msg103"><li><a href="http://dx.doi.org/10.1093/molbev/msg103" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Fungal+hydrogenosomes+contain+mitochondrial+heat-shock+proteins." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Fungal+hydrogenosomes+contain+mitochondrial+heat-shock+proteins.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0001383-Hrd1" id="pone.0001383-Hrd1"></a>Hrdý I, Hirt RP, Doležal P, Bardoňová L, Foster PG, et al.  (2004) <em>Trichomonas</em> hydrogenosomes contain the NADH dehydrogenase module of mitochondrial complex I. Nature  432: 618–622.  <ul class="find" data-citedArticleID="1036704" data-doi="10.1038/nature03149"><li><a href="http://dx.doi.org/10.1038/nature03149" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Trichomonas+hydrogenosomes+contain+the+NADH+dehydrogenase+module+of+mitochondrial+complex+I." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Trichomonas+hydrogenosomes+contain+the+NADH+dehydrogenase+module+of+mitochondrial+complex+I.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0001383-Dyall1" id="pone.0001383-Dyall1"></a>Dyall SD, Yan WH, Delgadillo-Correa MG, Lunceford A, Loo JA, et al.  (2004) Non-mitochondrial complex I proteins in a hydrogenosomal oxidoreductase complex. Nature  431: 1103–1107.  <ul class="find" data-citedArticleID="1036686" data-doi="10.1038/nature02990"><li><a href="http://dx.doi.org/10.1038/nature02990" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Non-mitochondrial+complex+I+proteins+in+a+hydrogenosomal+oxidoreductase+complex." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Non-mitochondrial+complex+I+proteins+in+a+hydrogenosomal+oxidoreductase+complex.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0001383-Tovar2" id="pone.0001383-Tovar2"></a>Tovar J, Leon-Avila G, Sánchez LB, Šuťák R, Tachezy J, et al.  (2003) Mitochondrial remnant organelles of <em>Giardia</em> function in iron-sulphur protein maturation. Nature  426: 172–176.  <ul class="find" data-citedArticleID="1036778" data-doi="10.1038/nature01945"><li><a href="http://dx.doi.org/10.1038/nature01945" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mitochondrial+remnant+organelles+of+Giardia+function+in+iron-sulphur+protein+maturation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mitochondrial+remnant+organelles+of+Giardia+function+in+iron-sulphur+protein+maturation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0001383-Riordan1" id="pone.0001383-Riordan1"></a>Riordan CE, Ault JG, Langreth SG, Keithly JS (2003) <em>Cryptosporidium parvum</em> Cpn60 targets a relict organelle. Curr Genet  44: 138–147.  <ul class="find" data-citedArticleID="1036752" data-doi="10.1007/s00294-003-0432-1"><li><a href="http://dx.doi.org/10.1007/s00294-003-0432-1" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Cryptosporidium+parvum+Cpn60+targets+a+relict+organelle." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Cryptosporidium+parvum+Cpn60+targets+a+relict+organelle.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0001383-Embley1" id="pone.0001383-Embley1"></a>Embley TM (2006) Multiple secondary origins of the anaerobic lifestyle in eukaryotes. Phil Trans R Soc B  361: 1055–1067.  <ul class="find" data-citedArticleID="1036690" data-doi="10.1098/rstb.2006.1844"><li><a href="http://dx.doi.org/10.1098/rstb.2006.1844" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Multiple+secondary+origins+of+the+anaerobic+lifestyle+in+eukaryotes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Multiple+secondary+origins+of+the+anaerobic+lifestyle+in+eukaryotes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0001383-Dacks1" id="pone.0001383-Dacks1"></a>Dacks JB, Silberman JD, Simpson AGB, Moriya S, Kudo T, et al.  (2001) Oxymonads are closely related to the excavate taxon <em>Trimastix</em>. Mol Biol Evol  18: 1034–1044.  <ul class="find" data-citedArticleID="1036676" data-doi="10.1093/oxfordjournals.molbev.a003875"><li><a href="http://dx.doi.org/10.1093/oxfordjournals.molbev.a003875" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Oxymonads+are+closely+related+to+the+excavate+taxon+Trimastix." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Oxymonads+are+closely+related+to+the+excavate+taxon+Trimastix.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0001383-Simpson1" id="pone.0001383-Simpson1"></a>Simpson AGB (2003) Cytoskeletal organization, phylogenetic affinities and systematics in the contentious taxon Excavata (Eukaryota). Int J Syst Evol Microbiol  53: 1759–1777.  <ul class="find" data-citedArticleID="1036762" data-doi="10.1099/ijs.0.02578-0"><li><a href="http://dx.doi.org/10.1099/ijs.0.02578-0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Cytoskeletal+organization%2C+phylogenetic+affinities+and+systematics+in+the+contentious+taxon+Excavata+%28Eukaryota%29." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Cytoskeletal+organization%2C+phylogenetic+affinities+and+systematics+in+the+contentious+taxon+Excavata+%28Eukaryota%29.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0001383-Hampl1" id="pone.0001383-Hampl1"></a>Hampl V, Horner DS, Dyal P, Kulda J, Flegr J, et al.  (2005) Inference of the phylogenetic position of oxymonads based on nine genes: Support for Metamonada and Excavata. Mol Biol Evol  22: 2508–2518.  <ul class="find" data-citedArticleID="1036696" data-doi="10.1093/molbev/msi245"><li><a href="http://dx.doi.org/10.1093/molbev/msi245" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Inference+of+the+phylogenetic+position+of+oxymonads+based+on+nine+genes%3A+Support+for+Metamonada+and+Excavata." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Inference+of+the+phylogenetic+position+of+oxymonads+based+on+nine+genes%3A+Support+for+Metamonada+and+Excavata.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0001383-Simpson2" id="pone.0001383-Simpson2"></a>Simpson AGB, Inagaki Y, Roger AJ (2006) Comprehensive multigene phylogenies of excavate protists reveal the evolutionary positions of “primitive” eukaryotes. Mol Biol Evol  23: 615–625.  <ul class="find" data-citedArticleID="1036764" data-doi="10.1093/molbev/msj068"><li><a href="http://dx.doi.org/10.1093/molbev/msj068" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Comprehensive+multigene+phylogenies+of+excavate+protists+reveal+the+evolutionary+positions+of+%E2%80%9Cprimitive%E2%80%9D+eukaryotes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Comprehensive+multigene+phylogenies+of+excavate+protists+reveal+the+evolutionary+positions+of+%E2%80%9Cprimitive%E2%80%9D+eukaryotes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0001383-Brugerolle1" id="pone.0001383-Brugerolle1"></a>Brugerolle G, Patterson D (1997) Ultrastructure of <em>Trimastix</em> convexa Hollande, an amitochondriate anaerobic flagellate with a previously undescribed organization. Europ J Protistol  33: 121–130.  <ul class="find" data-citedArticleID="1036660" data-doi="10.1016/s0932-4739(97)80029-6"><li><a href="http://dx.doi.org/10.1016/s0932-4739(97)80029-6" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Ultrastructure+of+Trimastix+convexa+Hollande%2C+an+amitochondriate+anaerobic+flagellate+with+a+previously+undescribed+organization." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Ultrastructure+of+Trimastix+convexa+Hollande%2C+an+amitochondriate+anaerobic+flagellate+with+a+previously+undescribed+organization.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0001383-OKelly1" id="pone.0001383-OKelly1"></a>O'Kelly CJ, Farmer MA, Nerad TA (1999) Ultrastructure of <em>Trimastix pyriformis</em> (Klebs) Bernard et al.: similarities of <em>Trimastix</em> species with retortamonad and jakobid flagellates. Protist  150: 149–162.  <ul class="find" data-citedArticleID="1036738" data-doi="10.1016/s1434-4610(99)70018-0"><li><a href="http://dx.doi.org/10.1016/s1434-4610(99)70018-0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Ultrastructure+of+Trimastix+pyriformis+%28Klebs%29+Bernard+et+al.%3A+similarities+of+Trimastix+species+with+retortamonad+and+jakobid+flagellates." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Ultrastructure+of+Trimastix+pyriformis+%28Klebs%29+Bernard+et+al.%3A+similarities+of+Trimastix+species+with+retortamonad+and+jakobid+flagellates.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0001383-Simpson3" id="pone.0001383-Simpson3"></a>Simpson AGB, Bernard C, Patterson DJ (2000) The ultrastructure of <em>Trimastix marina</em> Kent, 1880 (eukaryota), an excavate flagellate. Europ J Protistol  36: 229–251.  <ul class="find" data-citedArticleID="1036766" data-doi="10.1016/s0932-4739(00)80001-2"><li><a href="http://dx.doi.org/10.1016/s0932-4739(00)80001-2" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+ultrastructure+of+Trimastix+marina+Kent%2C+1880+%28eukaryota%29%2C+an+excavate+flagellate." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+ultrastructure+of+Trimastix+marina+Kent%2C+1880+%28eukaryota%29%2C+an+excavate+flagellate.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0001383-Kulda1" id="pone.0001383-Kulda1"></a>Kulda J, Nohýnková E (1978)  Flagellates of the human intestine and of intestines of other species. In: Kreier JP, editor. Parasitic protozoa. New York: Academic Press. pp. 1–138.  <ul class="find-nolinks"></ul></li><li><span class="label">34.
              </span><a name="pone.0001383-Simpson4" id="pone.0001383-Simpson4"></a>Simpson AGB, Radek R, Dacks JB, O'Kelly CJ (2002) How oxymonads lost their groove: An ultrastructural comparison of <em>Monocercomonoides</em> and excavate taxa. J Euk Microbiol  49: 239–248.  <ul class="find" data-citedArticleID="1036768" data-doi="10.1111/j.1550-7408.2002.tb00529.x"><li><a href="http://dx.doi.org/10.1111/j.1550-7408.2002.tb00529.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=How+oxymonads+lost+their+groove%3A+An+ultrastructural+comparison+of+Monocercomonoides+and+excavate+taxa." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22How+oxymonads+lost+their+groove%3A+An+ultrastructural+comparison+of+Monocercomonoides+and+excavate+taxa.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0001383-Leander1" id="pone.0001383-Leander1"></a>Leander BS, Keeling PJ (2004) Symbiotic innovation in the oxymonad <em>Streblomastix strix</em>. J Euk Microbiol  51: 291–300.  <ul class="find" data-citedArticleID="1036718" data-doi="10.1111/j.1550-7408.2004.tb00569.x"><li><a href="http://dx.doi.org/10.1111/j.1550-7408.2004.tb00569.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Symbiotic+innovation+in+the+oxymonad+Streblomastix+strix." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Symbiotic+innovation+in+the+oxymonad+Streblomastix+strix.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">36.
              </span><a name="pone.0001383-Bloodgood1" id="pone.0001383-Bloodgood1"></a>Bloodgood RA, Miller KR, Fitzharris TP, McIntosh JR (1974) The Ultrastructure of <em>Pyrsonympha</em> and Its Associated Microorganisms. J Morph  143: 77–105.  <ul class="find" data-citedArticleID="1036652" data-doi="10.1002/jmor.1051430104"><li><a href="http://dx.doi.org/10.1002/jmor.1051430104" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+Ultrastructure+of+Pyrsonympha+and+Its+Associated+Microorganisms." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+Ultrastructure+of+Pyrsonympha+and+Its+Associated+Microorganisms.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">37.
              </span><a name="pone.0001383-Carpenter1" id="pone.0001383-Carpenter1"></a>Carpenter KJ, Waller RF, Keeling PJ (2007) Surface Morphology of <em>Saccinobaculus</em> (Oxymonadida): Implications for Character Evolution and Function in Oxymonads. Protist..   In press: doi:10.1016/j.protis.2007.09.002.  <ul class="find" data-citedArticleID="1036666" data-doi="10.1016/j.protis.2007.09.002"><li><a href="http://dx.doi.org/10.1016/j.protis.2007.09.002" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Surface+Morphology+of+Saccinobaculus+%28Oxymonadida%29%3A+Implications+for+Character+Evolution+and+Function+in+Oxymonads." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Surface+Morphology+of+Saccinobaculus+%28Oxymonadida%29%3A+Implications+for+Character+Evolution+and+Function+in+Oxymonads.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">38.
              </span><a name="pone.0001383-Reeves1" id="pone.0001383-Reeves1"></a>Reeves RE, Warren LG, Susskind B, Loi HS (1977) An energy-conserving pyruvate-to-acetate pathway in <em>Entamoeba histolytica</em>. Pyruvate synthase and a new acetate thiokinase. J Biol Chem  252: 726–731.  <ul class="find" data-citedArticleID="1036748"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=An+energy-conserving+pyruvate-to-acetate+pathway+in+Entamoeba+histolytica.+Pyruvate+synthase+and+a+new+acetate+thiokinase.&amp;auth=&amp;atitle=An+energy-conserving+pyruvate-to-acetate+pathway+in+Entamoeba+histolytica.+Pyruvate+synthase+and+a+new+acetate+thiokinase." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=An+energy-conserving+pyruvate-to-acetate+pathway+in+Entamoeba+histolytica.+Pyruvate+synthase+and+a+new+acetate+thiokinase." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22An+energy-conserving+pyruvate-to-acetate+pathway+in+Entamoeba+histolytica.+Pyruvate+synthase+and+a+new+acetate+thiokinase.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">39.
              </span><a name="pone.0001383-Townson1" id="pone.0001383-Townson1"></a>Townson SM, Upcroft JA, Upcroft P (1996) Characterisation and purification of pyruvate:ferredoxin oxidoreductase from <em>Giardia duodenalis</em>. Mol Biochem Parasitol  79: 183–193.  <ul class="find" data-citedArticleID="1036780" data-doi="10.1016/0166-6851(96)02661-8"><li><a href="http://dx.doi.org/10.1016/0166-6851(96)02661-8" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Characterisation+and+purification+of+pyruvate%3Aferredoxin+oxidoreductase+from+Giardia+duodenalis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Characterisation+and+purification+of+pyruvate%3Aferredoxin+oxidoreductase+from+Giardia+duodenalis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">40.
              </span><a name="pone.0001383-Mller1" id="pone.0001383-Mller1"></a>Müller M (1993) The Hydrogenosome. J Gen Microbiol  139: 2879–2889.  <ul class="find" data-citedArticleID="1036730" data-doi="10.1099/00221287-139-12-2879"><li><a href="http://dx.doi.org/10.1099/00221287-139-12-2879" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+Hydrogenosome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+Hydrogenosome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">41.
              </span><a name="pone.0001383-Vignais1" id="pone.0001383-Vignais1"></a>Vignais PM, Billoud B, Meyer J (2001) Classification and phylogeny of hydrogenases. Fems Microbiol Rev  25: 455–501.  <ul class="find" data-citedArticleID="1036782" data-doi="10.1111/j.1574-6976.2001.tb00587.x"><li><a href="http://dx.doi.org/10.1111/j.1574-6976.2001.tb00587.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Classification+and+phylogeny+of+hydrogenases." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Classification+and+phylogeny+of+hydrogenases.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">42.
              </span><a name="pone.0001383-Meyer1" id="pone.0001383-Meyer1"></a>Meyer J (2007) [FeFe] hydrogenases and their evolution: a genomic perspective. Cell Mol Life Sci  64: 1063–1084.  <ul class="find" data-citedArticleID="1036728" data-doi="10.1007/s00018-007-6477-4"><li><a href="http://dx.doi.org/10.1007/s00018-007-6477-4" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=%5BFeFe%5D+hydrogenases+and+their+evolution%3A+a+genomic+perspective." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22%5BFeFe%5D+hydrogenases+and+their+evolution%3A+a+genomic+perspective.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">43.
              </span><a name="pone.0001383-Posewitz1" id="pone.0001383-Posewitz1"></a>Posewitz MC, King PW, Smolinski SL, Zhang L, Seibert M, Ghirardi ML (2004) Discovery of two novel radical S-adenosylmethionine proteins required for the assembly of an active [Fe] hydrogenase. J Biol Chem  279: 25711–25720.  <ul class="find" data-citedArticleID="1036744" data-doi="10.1074/jbc.m403206200"><li><a href="http://dx.doi.org/10.1074/jbc.m403206200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Discovery+of+two+novel+radical+S-adenosylmethionine+proteins+required+for+the+assembly+of+an+active+%5BFe%5D+hydrogenase." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Discovery+of+two+novel+radical+S-adenosylmethionine+proteins+required+for+the+assembly+of+an+active+%5BFe%5D+hydrogenase.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">44.
              </span><a name="pone.0001383-Putz1" id="pone.0001383-Putz1"></a>Putz S, Doležal P, Gelius-Dietrich G, Boháčová L, Tachezy J, Henze K (2006) Fe-hydrogenase maturases in the hydrogenosomes of <em>Trichomonas vaginalis</em>. Eukaryot Cell  5: 579–586.  <ul class="find" data-citedArticleID="1036746" data-doi="10.1128/ec.5.3.579-586.2006"><li><a href="http://dx.doi.org/10.1128/ec.5.3.579-586.2006" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Fe-hydrogenase+maturases+in+the+hydrogenosomes+of+Trichomonas+vaginalis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Fe-hydrogenase+maturases+in+the+hydrogenosomes+of+Trichomonas+vaginalis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">45.
              </span><a name="pone.0001383-Douce1" id="pone.0001383-Douce1"></a>Douce R, Bourguignon J, Neuburger M, Rebeille F (2001) The glycine decarboxylase system: a fascinating complex. Trends Plant Sci  6: 167–176.  <ul class="find" data-citedArticleID="1036684" data-doi="10.1016/s1360-1385(01)01892-1"><li><a href="http://dx.doi.org/10.1016/s1360-1385(01)01892-1" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+glycine+decarboxylase+system%3A+a+fascinating+complex." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+glycine+decarboxylase+system%3A+a+fascinating+complex.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">46.
              </span><a name="pone.0001383-Carothers1" id="pone.0001383-Carothers1"></a>Carothers DJ, Pons G, Patel MS (1989) Dihydrolipoamide dehydrogenase: functional similarities and divergent evolution of the pyridine nucleotide-disulfide oxidoreductases. Arch Biochem Biophys  268: 409–425.  <ul class="find" data-citedArticleID="1036664" data-doi="10.1016/0003-9861(89)90309-3"><li><a href="http://dx.doi.org/10.1016/0003-9861(89)90309-3" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Dihydrolipoamide+dehydrogenase%3A+functional+similarities+and+divergent+evolution+of+the+pyridine+nucleotide-disulfide+oxidoreductases." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Dihydrolipoamide+dehydrogenase%3A+functional+similarities+and+divergent+evolution+of+the+pyridine+nucleotide-disulfide+oxidoreductases.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">47.
              </span><a name="pone.0001383-Mukherjee1" id="pone.0001383-Mukherjee1"></a>Mukherjee M, Brown MT, McArthur AG, Johnson PJ (2006) Proteins of the glycine decarboxylase complex in the hydrogenosome of <em>Trichomonas vaginalis</em>. Eukaryot Cell  5: 2062–2071.  <ul class="find" data-citedArticleID="1036734" data-doi="10.1128/ec.00205-06"><li><a href="http://dx.doi.org/10.1128/ec.00205-06" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Proteins+of+the+glycine+decarboxylase+complex+in+the+hydrogenosome+of+Trichomonas+vaginalis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Proteins+of+the+glycine+decarboxylase+complex+in+the+hydrogenosome+of+Trichomonas+vaginalis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">48.
              </span><a name="pone.0001383-Kruger1" id="pone.0001383-Kruger1"></a>Kruger N, Oppermann FB, Lorenzl H, Steinbuchel A (1994) Biochemical and molecular characterization of the <em>Clostridium magnum</em> acetoin dehydrogenase enzyme system. J Bacteriol  176: 3614–3630.  <ul class="find" data-citedArticleID="1036710"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Biochemical+and+molecular+characterization+of+the+Clostridium+magnum+acetoin+dehydrogenase+enzyme+system.&amp;auth=&amp;atitle=Biochemical+and+molecular+characterization+of+the+Clostridium+magnum+acetoin+dehydrogenase+enzyme+system." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Biochemical+and+molecular+characterization+of+the+Clostridium+magnum+acetoin+dehydrogenase+enzyme+system." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Biochemical+and+molecular+characterization+of+the+Clostridium+magnum+acetoin+dehydrogenase+enzyme+system.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">49.
              </span><a name="pone.0001383-Jordan1" id="pone.0001383-Jordan1"></a>Jordan SW, Cronan JE Jr (1997) A new metabolic link. The acyl carrier protein of lipid synthesis donates lipoic acid to the pyruvate dehydrogenase complex in <em>Escherichia coli</em> and mitochondria. J Biol Chem  272: 17903–17906.  <ul class="find" data-citedArticleID="1036706" data-doi="10.1074/jbc.272.29.17903"><li><a href="http://dx.doi.org/10.1074/jbc.272.29.17903" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+new+metabolic+link.+The+acyl+carrier+protein+of+lipid+synthesis+donates+lipoic+acid+to+the+pyruvate+dehydrogenase+complex+in+Escherichia+coli+and+mitochondria." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+new+metabolic+link.+The+acyl+carrier+protein+of+lipid+synthesis+donates+lipoic+acid+to+the+pyruvate+dehydrogenase+complex+in+Escherichia+coli+and+mitochondria.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">50.
              </span><a name="pone.0001383-Olausson1" id="pone.0001383-Olausson1"></a>Olausson T, Fjellstrom O, Meuller J, Rydstrom J (1995) Molecular biology of nicotinamide nucleotide transhydrogenase–a unique proton pump. Biochim Biophys Acta  1231: 1–19.  <ul class="find" data-citedArticleID="1036740" data-doi="10.1016/0005-2728(95)00058-q"><li><a href="http://dx.doi.org/10.1016/0005-2728(95)00058-q" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Molecular+biology+of+nicotinamide+nucleotide+transhydrogenase%E2%80%93a+unique+proton+pump." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Molecular+biology+of+nicotinamide+nucleotide+transhydrogenase%E2%80%93a+unique+proton+pump.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">51.
              </span><a name="pone.0001383-Doleal1" id="pone.0001383-Doleal1"></a>Doležal P, Likic V, Tachezy J, Lithgow T (2006) Evolution of the molecular machines for protein import into mitochondria. Science  313: 314–318.  <ul class="find" data-citedArticleID="1036680" data-doi="10.1126/science.1127895"><li><a href="http://dx.doi.org/10.1126/science.1127895" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evolution+of+the+molecular+machines+for+protein+import+into+mitochondria." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evolution+of+the+molecular+machines+for+protein+import+into+mitochondria.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">52.
              </span><a name="pone.0001383-Macasev1" id="pone.0001383-Macasev1"></a>Macasev D, Whelan J, Newbigin E, Silva-Filho MC, Mulhern TD, et al.  (2004) Tom22', an 8-kDa trans-site receptor in plants and protozoans, is a conserved feature of the TOM complex that appeared early in the evolution of eukaryotes. Mol Biol Evol  21: 1557–1564.  <ul class="find" data-citedArticleID="1036722" data-doi="10.1093/molbev/msh166"><li><a href="http://dx.doi.org/10.1093/molbev/msh166" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Tom22%27%2C+an+8-kDa+trans-site+receptor+in+plants+and+protozoans%2C+is+a+conserved+feature+of+the+TOM+complex+that+appeared+early+in+the+evolution+of+eukaryotes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Tom22%27%2C+an+8-kDa+trans-site+receptor+in+plants+and+protozoans%2C+is+a+conserved+feature+of+the+TOM+complex+that+appeared+early+in+the+evolution+of+eukaryotes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">53.
              </span><a name="pone.0001383-Luciano1" id="pone.0001383-Luciano1"></a>Luciano P, Geoffroy S, Brandt A, Hernandez JF, Geli V (1997) Functional cooperation of the mitochondrial processing peptidase subunits. J Mol Biol  272: 213–225.  <ul class="find" data-citedArticleID="1036720" data-doi="10.1006/jmbi.1997.1231"><li><a href="http://dx.doi.org/10.1006/jmbi.1997.1231" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Functional+cooperation+of+the+mitochondrial+processing+peptidase+subunits." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Functional+cooperation+of+the+mitochondrial+processing+peptidase+subunits.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">54.
              </span><a name="pone.0001383-Regoes1" id="pone.0001383-Regoes1"></a>Regoes A, Zourmpanou D, Leon-Avila G, van der Giezen M, Tovar J, et al.  (2005) Protein import, replication, and inheritance of a vestigial mitochondrion. J Biol Chem  280: 30557–30563.  <ul class="find" data-citedArticleID="1036750" data-doi="10.1074/jbc.m500787200"><li><a href="http://dx.doi.org/10.1074/jbc.m500787200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Protein+import%2C+replication%2C+and+inheritance+of+a+vestigial+mitochondrion." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Protein+import%2C+replication%2C+and+inheritance+of+a+vestigial+mitochondrion.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">55.
              </span><a name="pone.0001383-Doleal2" id="pone.0001383-Doleal2"></a>Doležal P, Šmíd O, Rada P, Zubáčová Z, Bursac D, et al.  (2005) <em>Giardia</em> mitosomes and trichomonad hydrogenosomes share a common mode of protein targeting. Proc Natl Acad Sci U S A  102: 10924–10929.  <ul class="find" data-citedArticleID="1036682" data-doi="10.1073/pnas.0500349102"><li><a href="http://dx.doi.org/10.1073/pnas.0500349102" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Giardia+mitosomes+and+trichomonad+hydrogenosomes+share+a+common+mode+of+protein+targeting." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Giardia+mitosomes+and+trichomonad+hydrogenosomes+share+a+common+mode+of+protein+targeting.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">56.
              </span><a name="pone.0001383-Brown1" id="pone.0001383-Brown1"></a>Brown MT, Goldstone HM, Bastida-Corcuera F, gadillo-Correa MG, McArthur AG, et al.  (2007) A functionally divergent hydrogenosomal peptidase with protomitochondrial ancestry. Mol Microbiol  64: 1154–1163.  <ul class="find" data-citedArticleID="1036658" data-doi="10.1111/j.1365-2958.2007.05719.x"><li><a href="http://dx.doi.org/10.1111/j.1365-2958.2007.05719.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+functionally+divergent+hydrogenosomal+peptidase+with+protomitochondrial+ancestry." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+functionally+divergent+hydrogenosomal+peptidase+with+protomitochondrial+ancestry.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">57.
              </span><a name="pone.0001383-Braun1" id="pone.0001383-Braun1"></a>Braun HP, Schmitz UK (1997) The mitochondrial processing peptidase. Int J Biochem Cell Biol  29: 1043–1045.  <ul class="find" data-citedArticleID="1036656" data-doi="10.1016/s1357-2725(97)00032-0"><li><a href="http://dx.doi.org/10.1016/s1357-2725(97)00032-0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+mitochondrial+processing+peptidase." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+mitochondrial+processing+peptidase.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">58.
              </span><a name="pone.0001383-Nagao1" id="pone.0001383-Nagao1"></a>Nagao Y, Kitada S, Kojima K, Toh H, Kuhara S, et al.  (2000) Glycine-rich region of mitochondrial processing peptidase alpha-subunit is essential for binding and cleavage of the precursor proteins. J Biol Chem  275: 34552–34556.  <ul class="find" data-citedArticleID="1036736" data-doi="10.1074/jbc.m003110200"><li><a href="http://dx.doi.org/10.1074/jbc.m003110200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Glycine-rich+region+of+mitochondrial+processing+peptidase+alpha-subunit+is+essential+for+binding+and+cleavage+of+the+precursor+proteins." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Glycine-rich+region+of+mitochondrial+processing+peptidase+alpha-subunit+is+essential+for+binding+and+cleavage+of+the+precursor+proteins.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">59.
              </span><a name="pone.0001383-Palmieri1" id="pone.0001383-Palmieri1"></a>Palmieri L, Rottensteiner H, Girzalsky W, Scarcia P, Palmieri F, et al.  (2001) Identification and functional reconstitution of the yeast peroxisomal adenine nucleotide transporter. EMBO J  20: 5049–5059.  <ul class="find" data-citedArticleID="1036742" data-doi="10.1093/emboj/20.18.5049"><li><a href="http://dx.doi.org/10.1093/emboj/20.18.5049" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Identification+and+functional+reconstitution+of+the+yeast+peroxisomal+adenine+nucleotide+transporter." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Identification+and+functional+reconstitution+of+the+yeast+peroxisomal+adenine+nucleotide+transporter.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">60.
              </span><a name="pone.0001383-Bedhomme1" id="pone.0001383-Bedhomme1"></a>Bedhomme M, Hoffmann M, McCarthy EA, Gambonnet B, Moran RG, et al.  (2005) Folate metabolism in plants: an <em>Arabidopsis</em> homolog of the mammalian mitochondrial folate transporter mediates folate import into chloroplasts. J Biol Chem  280: 34823–34831.  <ul class="find" data-citedArticleID="1036650" data-doi="10.1074/jbc.m506045200"><li><a href="http://dx.doi.org/10.1074/jbc.m506045200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Folate+metabolism+in+plants%3A+an+Arabidopsis+homolog+of+the+mammalian+mitochondrial+folate+transporter+mediates+folate+import+into+chloroplasts." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Folate+metabolism+in+plants%3A+an+Arabidopsis+homolog+of+the+mammalian+mitochondrial+folate+transporter+mediates+folate+import+into+chloroplasts.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">61.
              </span><a name="pone.0001383-Satre1" id="pone.0001383-Satre1"></a>Satre M, Mattei S, Aubry L, Gaudet P, Pelosi L, et al.  (2007) Mitochondrial carrier family: Repertoire and peculiarities of the cellular slime mould <em>Dictyostelium discoideum</em>. Biochimie  89: 1058–1069.  <ul class="find" data-citedArticleID="1036760" data-doi="10.1016/j.biochi.2007.03.004"><li><a href="http://dx.doi.org/10.1016/j.biochi.2007.03.004" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mitochondrial+carrier+family%3A+Repertoire+and+peculiarities+of+the+cellular+slime+mould+Dictyostelium+discoideum." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mitochondrial+carrier+family%3A+Repertoire+and+peculiarities+of+the+cellular+slime+mould+Dictyostelium+discoideum.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">62.
              </span><a name="pone.0001383-Dyall2" id="pone.0001383-Dyall2"></a>Dyall SD, Koehler CM, Delgadillo-Correa MG, Bradley PJ, Plumper E, et al.  (2000) Presence of a member of the mitochondrial carrier family in hydrogenosomes: Conservation of membrane-targeting pathways between hydrogenosomes and mitochondria. Mol Cell Biol  20: 2488–2497.  <ul class="find" data-citedArticleID="1036688" data-doi="10.1128/mcb.20.7.2488-2497.2000"><li><a href="http://dx.doi.org/10.1128/mcb.20.7.2488-2497.2000" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Presence+of+a+member+of+the+mitochondrial+carrier+family+in+hydrogenosomes%3A+Conservation+of+membrane-targeting+pathways+between+hydrogenosomes+and+mitochondria." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Presence+of+a+member+of+the+mitochondrial+carrier+family+in+hydrogenosomes%3A+Conservation+of+membrane-targeting+pathways+between+hydrogenosomes+and+mitochondria.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">63.
              </span><a name="pone.0001383-vanderGiezen2" id="pone.0001383-vanderGiezen2"></a>van der Giezen M, Slotboom DJ, Horner DS, Dyal PL, Harding M, et al.  (2002) Conserved properties of hydrogenosomal and mitochondrial ADP/ATP carriers: a common origin for both organelles. EMBO J  21: 572–579.  <ul class="find" data-citedArticleID="1036788" data-doi="10.1093/emboj/21.4.572"><li><a href="http://dx.doi.org/10.1093/emboj/21.4.572" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Conserved+properties+of+hydrogenosomal+and+mitochondrial+ADP%2FATP+carriers%3A+a+common+origin+for+both+organelles." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Conserved+properties+of+hydrogenosomal+and+mitochondrial+ADP%2FATP+carriers%3A+a+common+origin+for+both+organelles.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">64.
              </span><a name="pone.0001383-Chan1" id="pone.0001383-Chan1"></a>Chan KW, Slotboom DJ, Cox S, Embley TM, Fabre O, et al.  (2005) A novel ADP/ATP transporter in the mitosome of the microaerophilic human parasite <em>Entamoeba histolytica</em>. Curr Biol  15: 737–742.  <ul class="find" data-citedArticleID="1036672" data-doi="10.1016/j.cub.2005.02.068"><li><a href="http://dx.doi.org/10.1016/j.cub.2005.02.068" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+novel+ADP%2FATP+transporter+in+the+mitosome+of+the+microaerophilic+human+parasite+Entamoeba+histolytica." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+novel+ADP%2FATP+transporter+in+the+mitosome+of+the+microaerophilic+human+parasite+Entamoeba+histolytica.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">65.
              </span><a name="pone.0001383-Kunji1" id="pone.0001383-Kunji1"></a>Kunji ER, Robinson AJ (2006) The conserved substrate binding site of mitochondrial carriers. Biochim Biophys Acta  1757: 1237–1248.  <ul class="find" data-citedArticleID="1036714" data-doi="10.1016/j.bbabio.2006.03.021"><li><a href="http://dx.doi.org/10.1016/j.bbabio.2006.03.021" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+conserved+substrate+binding+site+of+mitochondrial+carriers." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+conserved+substrate+binding+site+of+mitochondrial+carriers.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">66.
              </span><a name="pone.0001383-Taylor1" id="pone.0001383-Taylor1"></a>Taylor SW, Fahy E, Zhang B, Glenn GM, Warnock DE, et al.  (2003) Characterization of the human heart mitochondrial proteome. Nat Biotechnol  21: 281–286.  <ul class="find" data-citedArticleID="1036774" data-doi="10.1038/nbt793"><li><a href="http://dx.doi.org/10.1038/nbt793" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Characterization+of+the+human+heart+mitochondrial+proteome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Characterization+of+the+human+heart+mitochondrial+proteome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">67.
              </span><a name="pone.0001383-Andreoli1" id="pone.0001383-Andreoli1"></a>Andreoli C, Prokisch H, Hortnagel K, Mueller JC, Munsterkotter M, et al.  (2004) MitoP2, an integrated database on mitochondrial proteins in yeast and man. Nucleic Acids Res  32: D459–D462.  <ul class="find" data-citedArticleID="1036648" data-doi="10.1093/nar/gkh137"><li><a href="http://dx.doi.org/10.1093/nar/gkh137" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=MitoP2%2C+an+integrated+database+on+mitochondrial+proteins+in+yeast+and+man." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22MitoP2%2C+an+integrated+database+on+mitochondrial+proteins+in+yeast+and+man.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">68.
              </span><a name="pone.0001383-Altschul1" id="pone.0001383-Altschul1"></a>Altschul SF, Madden TL, Schaffer AA, Zhang J, Zhang Z, et al.  (1997) Gapped BLAST and PSI-BLAST: a new generation of protein database search programs. Nucleic Acids Res  25: 3389–3402.  <ul class="find" data-citedArticleID="1036646" data-doi="10.1093/nar/25.17.3389"><li><a href="http://dx.doi.org/10.1093/nar/25.17.3389" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Gapped+BLAST+and+PSI-BLAST%3A+a+new+generation+of+protein+database+search+programs." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Gapped+BLAST+and+PSI-BLAST%3A+a+new+generation+of+protein+database+search+programs.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">69.
              </span><a name="pone.0001383-Hall1" id="pone.0001383-Hall1"></a>Hall TA (1999) BioEdit: a user-friendly biological sequence alignment editor and analysis program for Windows 95/98/NT. Nucl Acids Symp Ser 95–98.  <ul class="find" data-citedArticleID="1036694"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=BioEdit%3A+a+user-friendly+biological+sequence+alignment+editor+and+analysis+program+for+Windows+95%2F98%2FNT.&amp;auth=&amp;atitle=BioEdit%3A+a+user-friendly+biological+sequence+alignment+editor+and+analysis+program+for+Windows+95%2F98%2FNT." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=BioEdit%3A+a+user-friendly+biological+sequence+alignment+editor+and+analysis+program+for+Windows+95%2F98%2FNT." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22BioEdit%3A+a+user-friendly+biological+sequence+alignment+editor+and+analysis+program+for+Windows+95%2F98%2FNT.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">70.
              </span><a name="pone.0001383-Do1" id="pone.0001383-Do1"></a>Do CB, Mahabhashyam MS, Brudno M, Batzoglou S (2005) ProbCons: Probabilistic consistency-based multiple sequence alignment. Genome Res  15: 330–340.  <ul class="find" data-citedArticleID="1036678" data-doi="10.1101/gr.2821705"><li><a href="http://dx.doi.org/10.1101/gr.2821705" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=ProbCons%3A+Probabilistic+consistency-based+multiple+sequence+alignment." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22ProbCons%3A+Probabilistic+consistency-based+multiple+sequence+alignment.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">71.
              </span><a name="pone.0001383-Stamatakis1" id="pone.0001383-Stamatakis1"></a>Stamatakis A (2006) RAxML-VI-HPC: maximum likelihood-based phylogenetic analyses with thousands of taxa and mixed models. Bioinformatics  22: 2688–2690.  <ul class="find" data-citedArticleID="1036770" data-doi="10.1093/bioinformatics/btl446"><li><a href="http://dx.doi.org/10.1093/bioinformatics/btl446" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=RAxML-VI-HPC%3A+maximum+likelihood-based+phylogenetic+analyses+with+thousands+of+taxa+and+mixed+models." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22RAxML-VI-HPC%3A+maximum+likelihood-based+phylogenetic+analyses+with+thousands+of+taxa+and+mixed+models.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">72.
              </span><a name="pone.0001383-Ronquist1" id="pone.0001383-Ronquist1"></a>Ronquist F, Huelsenbeck JP (2003) MrBayes 3: Bayesian phylogenetic inference under mixed models. Bioinformatics  19: 1572–1574.  <ul class="find" data-citedArticleID="1036758" data-doi="10.1093/bioinformatics/btg180"><li><a href="http://dx.doi.org/10.1093/bioinformatics/btg180" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=MrBayes+3%3A+Bayesian+phylogenetic+inference+under+mixed+models." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22MrBayes+3%3A+Bayesian+phylogenetic+inference+under+mixed+models.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001383.XML" value="119183"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001383.PDF" value="516419"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001383.t001.PNG_L" value="305986"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001383.t001.PNG_M" value="170874"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001383.t001.PNG_S" value="15374"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001383.t001.TIF" value="1265508"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001383.t001.PNG_I" value="51406"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001383.g001.PNG_L" value="263330"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001383.g001.PNG_M" value="332303"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001383.g001.PNG_S" value="7542"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001383.g001.TIF" value="1521204"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001383.g001.PNG_I" value="112610"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001383.g002.PNG_L" value="311406"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001383.g002.PNG_M" value="162817"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001383.g002.PNG_S" value="10963"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001383.g002.TIF" value="2055026"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001383.g002.PNG_I" value="100340"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001383.s001.TIF" value="847317"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001383.s002.TIF" value="986577"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001383.s003.TIF" value="961112"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001383.s004.TIF" value="773054"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001383.s005.TIF" value="925327"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001383.s006.TIF" value="997711"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001383.s007.TIF" value="808102"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001383.s008.TIF" value="924447"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001383.s009.TIF" value="822980"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001383.s010.TIF" value="707127"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001383.s011.TIF" value="593453"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001383&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001383" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001383&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0001383&volume=&issue=&title=Genetic Evidence for a Mitochondriate Ancestry in the ‘Amitochondriate’ Flagellate Trimastix pyriformis&author_name=Vladimir%20Hampl%2C%20Jeffrey%20D.%20Silberman%2C%20Alexandra%20Stechmann%2C%20Sara%20Diaz-Trivi%C3%B1o%2C%20Patricia%20J.%20Johnson%2C%20Andrew%20J.%20Roger&start_page=1&end_page=9" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001383" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001383" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0001383" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001383&amp;t=Genetic%20Evidence%20for%20a%20Mitochondriate%20Ancestry%20in%20the%20%E2%80%98Amitochondriate%E2%80%99%20Flagellate%20Trimastix%20pyriformis" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001383&title=Genetic%20Evidence%20for%20a%20Mitochondriate%20Ancestry%20in%20the%20%E2%80%98Amitochondriate%E2%80%99%20Flagellate%20Trimastix%20pyriformis&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001383&amp;title=Genetic%20Evidence%20for%20a%20Mitochondriate%20Ancestry%20in%20the%20%E2%80%98Amitochondriate%E2%80%99%20Flagellate%20Trimastix%20pyriformis" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001383" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0001383&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Genetic Evidence for a Mitochondriate Ancestry in the ‘Amitochondriate’ Flagellate Trimastix pyriformis';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0001383';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Genetic%20Evidence%20for%20a%20Mitochondriate%20Ancestry%20in%20the%20%E2%80%98Amitochondriate%E2%80%99%20Flagellate%20Trimastix%20pyriformis http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001383" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0001383" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">














          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Bayes+theorem%22" title="Search for articles in the subject area:'Bayes theorem'"><div class="flagText">Bayes theorem</div></a>
              <div data-categoryid="32251" data-articleid="25596"
                   data-categoryname="Bayes theorem"
                   class="flagImage" title="Flag 'Bayes theorem' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Bayesian+method%22" title="Search for articles in the subject area:'Bayesian method'"><div class="flagText">Bayesian method</div></a>
              <div data-categoryid="23807" data-articleid="25596"
                   data-categoryname="Bayesian method"
                   class="flagImage" title="Flag 'Bayesian method' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Cellular+structures+and+organelles%22" title="Search for articles in the subject area:'Cellular structures and organelles'"><div class="flagText">Cellular structures and organelles</div></a>
              <div data-categoryid="17573" data-articleid="25596"
                   data-categoryname="Cellular structures and organelles"
                   class="flagImage" title="Flag 'Cellular structures and organelles' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Enzymes%22" title="Search for articles in the subject area:'Enzymes'"><div class="flagText">Enzymes</div></a>
              <div data-categoryid="17863" data-articleid="25596"
                   data-categoryname="Enzymes"
                   class="flagImage" title="Flag 'Enzymes' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Expressed+sequence+tags%22" title="Search for articles in the subject area:'Expressed sequence tags'"><div class="flagText">Expressed sequence tags</div></a>
              <div data-categoryid="34427" data-articleid="25596"
                   data-categoryname="Expressed sequence tags"
                   class="flagImage" title="Flag 'Expressed sequence tags' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Mitochondria%22" title="Search for articles in the subject area:'Mitochondria'"><div class="flagText">Mitochondria</div></a>
              <div data-categoryid="32453" data-articleid="25596"
                   data-categoryname="Mitochondria"
                   class="flagImage" title="Flag 'Mitochondria' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Phylogenetic+analysis%22" title="Search for articles in the subject area:'Phylogenetic analysis'"><div class="flagText">Phylogenetic analysis</div></a>
              <div data-categoryid="17851" data-articleid="25596"
                   data-categoryname="Phylogenetic analysis"
                   class="flagImage" title="Flag 'Phylogenetic analysis' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Sequence+databases%22" title="Search for articles in the subject area:'Sequence databases'"><div class="flagText">Sequence databases</div></a>
              <div data-categoryid="42787" data-articleid="25596"
                   data-categoryname="Sequence databases"
                   class="flagImage" title="Flag 'Sequence databases' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=9027'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=5023'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=1126&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>


</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
