

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Clinical Implementation of Chromosomal Microarray Analysis: Summary of 2513 Postnatal Cases</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0000327"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0000327" />

    <meta name="citation_title" content="Clinical Implementation of Chromosomal Microarray Analysis: Summary of 2513 Postnatal Cases"/>
    <meta itemprop="name" content="Clinical Implementation of Chromosomal Microarray Analysis: Summary of 2513 Postnatal Cases"/>

      <meta name="citation_author" content="Xinyan Lu"/>
            <meta name="citation_author_institution" content="Department of Molecular and Human Genetics, Baylor College of Medicine, Houston, Texas, United States of America"/>
      <meta name="citation_author" content="Chad A. Shaw"/>
            <meta name="citation_author_institution" content="Department of Molecular and Human Genetics, Baylor College of Medicine, Houston, Texas, United States of America"/>
      <meta name="citation_author" content="Ankita Patel"/>
            <meta name="citation_author_institution" content="Department of Molecular and Human Genetics, Baylor College of Medicine, Houston, Texas, United States of America"/>
      <meta name="citation_author" content="Jiangzhen Li"/>
            <meta name="citation_author_institution" content="Department of Molecular and Human Genetics, Baylor College of Medicine, Houston, Texas, United States of America"/>
      <meta name="citation_author" content="M. Lance Cooper"/>
            <meta name="citation_author_institution" content="Department of Molecular and Human Genetics, Baylor College of Medicine, Houston, Texas, United States of America"/>
      <meta name="citation_author" content="William R. Wells"/>
            <meta name="citation_author_institution" content="Department of Molecular and Human Genetics, Baylor College of Medicine, Houston, Texas, United States of America"/>
      <meta name="citation_author" content="Cathy M. Sullivan"/>
            <meta name="citation_author_institution" content="Department of Molecular and Human Genetics, Baylor College of Medicine, Houston, Texas, United States of America"/>
      <meta name="citation_author" content="Trilochan Sahoo"/>
            <meta name="citation_author_institution" content="Department of Molecular and Human Genetics, Baylor College of Medicine, Houston, Texas, United States of America"/>
      <meta name="citation_author" content="Svetlana A. Yatsenko"/>
            <meta name="citation_author_institution" content="Department of Molecular and Human Genetics, Baylor College of Medicine, Houston, Texas, United States of America"/>
      <meta name="citation_author" content="Carlos A. Bacino"/>
            <meta name="citation_author_institution" content="Department of Molecular and Human Genetics, Baylor College of Medicine, Houston, Texas, United States of America"/>
      <meta name="citation_author" content="Pawel Stankiewicz"/>
            <meta name="citation_author_institution" content="Department of Molecular and Human Genetics, Baylor College of Medicine, Houston, Texas, United States of America"/>
      <meta name="citation_author" content="Zhishu Ou"/>
            <meta name="citation_author_institution" content="Department of Molecular and Human Genetics, Baylor College of Medicine, Houston, Texas, United States of America"/>
      <meta name="citation_author" content="A. Craig Chinault"/>
            <meta name="citation_author_institution" content="Department of Molecular and Human Genetics, Baylor College of Medicine, Houston, Texas, United States of America"/>
      <meta name="citation_author" content="Arthur L. Beaudet"/>
            <meta name="citation_author_institution" content="Department of Molecular and Human Genetics, Baylor College of Medicine, Houston, Texas, United States of America"/>
      <meta name="citation_author" content="James R. Lupski"/>
            <meta name="citation_author_institution" content="Department of Molecular and Human Genetics, Baylor College of Medicine, Houston, Texas, United States of America"/>
      <meta name="citation_author" content="Sau W. Cheung"/>
            <meta name="citation_author_institution" content="Department of Molecular and Human Genetics, Baylor College of Medicine, Houston, Texas, United States of America"/>
      <meta name="citation_author" content="Patricia A. Ward"/>
            <meta name="citation_author_institution" content="Department of Molecular and Human Genetics, Baylor College of Medicine, Houston, Texas, United States of America"/>

    <meta name="citation_date" content="2007/3/28"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0000327.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e327"/>
    <meta name="citation_issue" content="3"/>
    <meta name="citation_volume" content="2"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Molecular mechanisms for constitutional chromosomal rearrangements in humans.; citation_author=LG Shaffer; citation_author=JR Lupski; citation_journal_title=Annu Rev Genet; citation_volume=34; citation_number=1; citation_pages=297-329; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Fluorescence in situ hybridization: applications in cytogenetics and gene mapping.; citation_author=BJ Trask; citation_journal_title=Trends Genet; citation_volume=7; citation_number=2; citation_pages=149-154; citation_date=1991; " />
      <meta name="citation_reference" content="citation_title=Chromosomal and gene amplification in diffuse large B-cell lymphoma.; citation_author=PH Rao; citation_author=J Houldsworth; citation_author=K Dyomina; citation_author=NZ Parsa; citation_author=JC Cigudosa; citation_journal_title=Blood; citation_volume=92; citation_number=3; citation_pages=234-240; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Karyotyping human chromosomes by combinatorial multi-fluor FISH.; citation_author=MR Speicher; citation_author=BS Gwyn; citation_author=DC Ward; citation_journal_title=Nat Genet; citation_volume=12; citation_number=4; citation_pages=368-375; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=The utility of spectral karyotyping in the cytogenetic analysis of newly diagnosed pediatric acute lymphoblastic leukemia.; citation_author=XY Lu; citation_author=CP Harris; citation_author=L Cooley; citation_author=J Margolin; citation_author=PC Steuber; citation_journal_title=Leukemia; citation_volume=16; citation_number=5; citation_pages=2222-2227; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Comprehensive molecular cytogenetic characterization of cervical cancer cell lines.; citation_author=CP Harris; citation_author=XY Lu; citation_author=G Narayan; citation_author=B Singh; citation_author=VV Murty; citation_journal_title=Genes Chromosomes Cancer; citation_volume=36; citation_number=6; citation_pages=233-241; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Frequent amplification and rearrangement of chromosomal bands 6p12â€“p21 and 17p11.2 in osteosarcoma.; citation_author=CC Lau; citation_author=CP Harris; citation_author=XY Lu; citation_author=L Perlaky; citation_author=S Gogineni; citation_journal_title=Genes Chromosomes Cancer; citation_volume=39; citation_number=7; citation_pages=11-21; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Subtelomere FISH analysis of 11 688 cases: an evaluation of the frequency and pattern of subtelomere rearrangements in individuals with developmental disabilities.; citation_author=JB Ravnan; citation_author=JH Tepperberg; citation_author=P Papenhausen; citation_author=AN Lamb; citation_author=J Hedrick; citation_journal_title=J Med Genet; citation_volume=43; citation_number=8; citation_pages=478-489; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Comparative genomic hybridization.; citation_author=D Pinkel; citation_author=DG Albertson; citation_journal_title=Annu Rev Genomics Hum Genet; citation_volume=6; citation_number=9; citation_pages=331-354; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Molecular karyotyping: array CGH quality criteria for constitutional genetic diagnosis.; citation_author=JR Vermeesch; citation_author=C Melotte; citation_author=G Froyen; citation_author=VS Van; citation_author=B Dutta; citation_journal_title=J Histochem Cytochem; citation_volume=53; citation_number=10; citation_pages=413-422; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=The new cytogenetics: blurring the boundaries with molecular biology.; citation_author=MR Speicher; citation_author=NP Carter; citation_journal_title=Nat Rev Genet; citation_volume=6; citation_number=11; citation_pages=782-792; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Array-based comparative genomic hybridization for the genomewide detection of submicroscopic chromosomal abnormalities.; citation_author=LE Vissers; citation_author=BB de Vries; citation_author=K Osoegawa; citation_author=IM Janssen; citation_author=T Feuth; citation_journal_title=Am J Hum Genet; citation_volume=73; citation_number=12; citation_pages=1261-1270; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Microarray based comparative genomic hybridisation (array-CGH) detects submicroscopic chromosomal deletions and duplications in patients with learning disability/mental retardation and dysmorphic features.; citation_author=C Shaw-Smith; citation_author=R Redon; citation_author=L Rickman; citation_author=M Rio; citation_author=L Willatt; citation_journal_title=J Med Genet; citation_volume=41; citation_number=13; citation_pages=241-248; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Emerging patterns of cryptic chromosomal imbalance in patients with idiopathic mental retardation and multiple congenital anomalies: a new series of 140 patients and review of published reports.; citation_author=B Menten; citation_author=N Maas; citation_author=B Thienpont; citation_author=K Buysse; citation_author=J Vandesompele; citation_journal_title=J Med Genet; citation_volume=43; citation_number=14; citation_pages=625-633; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=A tiling resolution DNA microarray with complete coverage of the human genome.; citation_author=AS Ishkanian; citation_author=CA Malloff; citation_author=SK Watson; citation_author=RJ DeLeeuw; citation_author=B Chi; citation_journal_title=Nat Genet; citation_volume=36; citation_number=15; citation_pages=299-303; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Development of a comparative genomic hybridization microarray and demonstration of its utility with 25 well-characterized 1p36 deletions.; citation_author=W Yu; citation_author=BC Ballif; citation_author=CD Kashork; citation_author=HA Heilstedt; citation_author=LA Howard; citation_journal_title=Hum Mol Genet; citation_volume=12; citation_number=16; citation_pages=2145-2152; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Comparative genomic hybridisation using a proximal 17p BAC/PAC array detects rearrangements responsible for four genomic disorders.; citation_author=CJ Shaw; citation_author=CA Shaw; citation_author=W Yu; citation_author=P Stankiewicz; citation_author=LD White; citation_journal_title=J Med Genet; citation_volume=41; citation_number=17; citation_pages=113-119; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Molecular characterisation of patients with subtelomeric 22q abnormalities using chromosome specific array-based comparative genomic hybridisation.; citation_author=DA Koolen; citation_author=W Reardon; citation_author=EM Rosser; citation_author=D Lacombe; citation_author=JA Hurst; citation_journal_title=Eur J Hum Genet; citation_volume=13; citation_number=18; citation_pages=1019-1024; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Identification of disease genes by whole genome CGH arrays.; citation_author=LE Vissers; citation_author=JA Veltman; citation_author=AG van Kessel; citation_author=HG Brunner; citation_journal_title=Hum Mol Genet; citation_volume=14 Spec No. 2; citation_number=19; citation_pages=R215-R223; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Exon array CGH: detection of copy-number changes at the resolution of individual exons in the human genome.; citation_author=P Dhami; citation_author=AJ Coffey; citation_author=S Abbs; citation_author=JR Vermeesch; citation_author=JP Dumanski; citation_journal_title=Am J Hum Genet; citation_volume=76; citation_number=20; citation_pages=750-762; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Methylation-associated silencing of the nuclear receptor 1I2 gene in advanced-type neuroblastomas, identified by bacterial artificial chromosome array-based methylated CpG island amplification.; citation_author=A Misawa; citation_author=J Inoue; citation_author=Y Sugino; citation_author=H Hosoi; citation_author=T Sugimoto; citation_journal_title=Cancer Res; citation_volume=65; citation_number=21; citation_pages=10233-10242; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Use of targeted array-based CGH for the clinical diagnosis of chromosomal imbalance: is less more?; citation_author=BA Bejjani; citation_author=R Saleki; citation_author=BC Ballif; citation_author=EA Rorem; citation_author=K Sundin; citation_journal_title=Am J Med Genet A; citation_volume=134; citation_number=22; citation_pages=259-267; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Diagnostic genome profiling in mental retardation.; citation_author=BB de Vries; citation_author=R Pfundt; citation_author=M Leisink; citation_author=DA Koolen; citation_author=LE Vissers; citation_journal_title=Am J Hum Genet; citation_volume=77; citation_number=23; citation_pages=606-616; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Development and validation of a CGH microarray for clinical cytogenetic diagnosis.; citation_author=SW Cheung; citation_author=CA Shaw; citation_author=W Yu; citation_author=J Li; citation_author=Z Ou; citation_journal_title=Genet Med; citation_volume=7; citation_number=24; citation_pages=422-432; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Targeted genomic microarray analysis for identification of chromosome abnormalities in 1500 consecutive clinical cases.; citation_author=LG Shaffer; citation_author=CD Kashork; citation_author=R Saleki; citation_author=E Rorem; citation_author=K Sundin; citation_journal_title=J Pediatr; citation_volume=149; citation_number=25; citation_pages=98-102; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Prenatal diagnosis of PLP1 copy number by array comparative genomic hybridization.; citation_author=JA Lee; citation_author=SW Cheung; citation_author=PA Ward; citation_author=K Inoue; citation_author=JR Lupski; citation_journal_title=Prenat Diagn; citation_volume=25; citation_number=26; citation_pages=1188-1191; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Prenatal diagnosis by array-CGH.; citation_author=L Rickman; citation_author=H Fiegler; citation_author=NP Carter; citation_author=M Bobrow; citation_journal_title=Eur J Med Genet; citation_volume=48; citation_number=27; citation_pages=232-240; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Prenatal detection of unbalanced chromosomal rearrangements by array CGH.; citation_author=L Rickman; citation_author=H Fiegler; citation_author=C Shaw-Smith; citation_author=R Nash; citation_author=V Cirigliano; citation_journal_title=J Med Genet; citation_volume=43; citation_number=28; citation_pages=353-361; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Prenatal diagnosis of de novo interstitial 2q14.2-2q21.3 deletion assisted by array-based comparative genomic hybridization: a case report.; citation_author=HH Peng; citation_author=CJ Wang; citation_author=TH Wang; citation_author=SD Chang; citation_journal_title=J Reprod Med; citation_volume=51; citation_number=29; citation_pages=438-442; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Microarray comparative genomic hybridization (CGH)-based prenatal diagnosis for chromosome abnormalities using cell-free fetal DNA in amniotic fluid.; citation_author=S Miura; citation_author=K Miura; citation_author=H Masuzaki; citation_author=N Miyake; citation_author=K Yoshiura; citation_journal_title=J Hum Genet; citation_volume=51; citation_number=30; citation_pages=412-417; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Prenatal diagnosis of chromosomal abnormalities using array-based comparative genomic hybridization.; citation_author=T Sahoo; citation_author=SW Cheung; citation_author=P Ward; citation_author=S Darilek; citation_author=A Patel; citation_journal_title=Genet Med; citation_volume=8; citation_number=31; citation_pages=719-727; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Genomic Disorders-The Genomic Basis of Disease.; citation_number=32; citation_pages=427 p; citation_date=2006; citation_publisher=Human Press; " />
      <meta name="citation_reference" content="citation_title=Submicroscopic deletions and duplications in individuals with intellectual disability detected by array-CGH.; citation_author=C Tyson; citation_author=C Harvard; citation_author=R Locker; citation_author=JM Friedman; citation_author=S Langlois; citation_journal_title=Am J Med Genet A; citation_volume=139; citation_number=33; citation_pages=173-185; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Microarray-based CGH detects chromosomal mosaicism not revealed by conventional cytogenetics.; citation_author=SW Cheung; citation_author=C Shaw; citation_author=DA Scott; citation_author=A Patel; citation_author=T Sahoo; citation_journal_title=Am J Med Genet (in press); citation_number=34; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Genome structural variation and sporadic disease traits.; citation_author=JR Lupski; citation_journal_title=Nat Genet; citation_volume=38; citation_number=35; citation_pages=974-976; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Increased MECP2 gene copy number as the result of genomic duplication in neurodevelopmentally delayed males.; citation_author=D del Gaudio; citation_author=P Fang; citation_author=F Scaglia; citation_author=PA Ward; citation_author=WJ Craigen; citation_journal_title=Genet Med; citation_volume=8; citation_number=36; citation_pages=784-792; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Global variation in copy number in the human genome.; citation_author=R Redon; citation_author=S Ishikawa; citation_author=KR Fitch; citation_author=L Feuk; citation_author=GH Perry; citation_journal_title=Nature; citation_volume=444; citation_number=37; citation_pages=444-454; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Detection of large-scale variation in the human genome.; citation_author=AJ Iafrate; citation_author=L Feuk; citation_author=MN Rivera; citation_author=ML Listewnik; citation_author=PK Donahoe; citation_journal_title=Nat Genet; citation_volume=36; citation_number=38; citation_pages=949-951; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Deletion of 7q31.1 supports involvement of FOXP2 in language impairment.; citation_author=PA Lennon; citation_author=ML Cooper; citation_author=D Peiffer; citation_author=K Gunderson; citation_author=A Patel; citation_journal_title=Am J Med Gene (in press); citation_number=39; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=The end of the beginning of chromosome ends.; citation_author=LG Biesecker; citation_journal_title=Am J Med Genet; citation_volume=107; citation_number=40; citation_pages=263-266; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Cryptic unbalanced translocation t(17;18)(p13.2;q22.3) identified by subtelomeric FISH and defined by array-based comparative genomic hybridization in a patient with mental retardation and dysmorphic features.; citation_author=KS Hwang; citation_author=MA Pearson; citation_author=P Stankiewicz; citation_author=PA Lennon; citation_author=ML Cooper; citation_journal_title=Am J Med Genet A; citation_volume=137; citation_number=41; citation_pages=88-93; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Subtelomeric 6p deletion: clinical, FISH, and array CGH characterization of two cases.; citation_author=CC Le; citation_author=MP De; citation_author=MC Vincent; citation_author=M Boceno; citation_author=G Bourrouillou; citation_journal_title=Am J Med Genet A; citation_volume=132; citation_number=42; citation_pages=175-180; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=DNA duplication associated with Charcot-Marie-Tooth disease type 1A.; citation_author=JR Lupski; citation_author=RM de Oca-Luna; citation_author=S Slaugenhaupt; citation_author=L Pentao; citation_author=V Guzzetta; citation_journal_title=Cell; citation_volume=66; citation_number=43; citation_pages=219-232; citation_date=1991; " />
      <meta name="citation_reference" content="citation_title=Charcot-Marie-Tooth disease: lessons in genetic mechanisms.; citation_author=JR Lupski; citation_journal_title=Mol Med; citation_volume=4; citation_number=44; citation_pages=3-11; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Genome architecture, rearrangements and genomic disorders.; citation_author=P Stankiewicz; citation_author=JR Lupski; citation_journal_title=Trends Genet; citation_volume=18; citation_number=45; citation_pages=74-82; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Diagnosis of CMT1A duplications and HNPP deletions by interphase FISH: implications for testing in the cytogenetics laboratory.; citation_author=LG Shaffer; citation_author=GM Kennedy; citation_author=AS Spikes; citation_author=JR Lupski; citation_journal_title=Am J Med Genet; citation_volume=69; citation_number=46; citation_pages=325-331; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Characterization of the Potocki-Lupski syndrome [dup(17)(p11.2p11.2)] and delineation of a dosage-sensitive critical interval that can convey an autism phenotype.; citation_author=L Potocki; citation_author=W Bi; citation_author=D Treadwell-Deering; citation_author=cMB Carvalho; citation_author=Anna Eifert; citation_journal_title=Am J Hum Genet (in press); citation_number=47; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Molecular mechanisms for genomic disorders.; citation_author=K Inoue; citation_author=JR Lupski; citation_journal_title=Annu Rev Genomics Hum Genet; citation_volume=3; citation_number=48; citation_pages=199-242; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Reciprocal translocations: a trap for cytogenetists?; citation_author=R Ciccone; citation_author=R Giorda; citation_author=G Gregato; citation_author=R Guerrini; citation_author=S Giglio; citation_journal_title=Hum Genet; citation_volume=117; citation_number=49; citation_pages=571-582; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Array CGH detection of a cryptic deletion in a complex chromosome rearrangement.; citation_author=C Rosenberg; citation_author=J Knijnenburg; citation_author=ML Chauffaille; citation_author=D Brunoni; citation_author=AL Catelani; citation_journal_title=Hum Genet; citation_volume=116; citation_number=50; citation_pages=390-394; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=The complex nature of constitutional de novo apparently balanced translocations in patients presenting with abnormal phenotypes.; citation_author=SM Gribble; citation_author=E Prigmore; citation_author=DC Burford; citation_author=KM Porter; citation_author=BL Ng; citation_journal_title=J Med Genet; citation_volume=42; citation_number=51; citation_pages=8-16; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Applications of combined DNA microarray and chromosome sorting technologies.; citation_author=SM Gribble; citation_author=H Fiegler; citation_author=DC Burford; citation_author=E Prigmore; citation_author=F Yang; citation_journal_title=Chromosome Res; citation_volume=12; citation_number=52; citation_pages=35-43; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Micro-array analyses decipher exceptional complex familial chromosomal rearrangement.; citation_author=C Fauth; citation_author=SM Gribble; citation_author=KM Porter; citation_author=M Codina-Pascual; citation_author=BL Ng; citation_journal_title=Hum Genet; citation_volume=119; citation_number=53; citation_pages=145-153; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Design considerations for array CGH to oligonucleotide arrays.; citation_author=RA Baldocchi; citation_author=RJ Glynne; citation_author=K Chin; citation_author=D Kowbel; citation_author=C Collins; citation_journal_title=Cytometry A; citation_volume=67; citation_number=54; citation_pages=129-136; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Chromosome-wide identification of novel imprinted genes using microarrays and uniparental disomies.; citation_author=R Schulz; citation_author=TR Menheniott; citation_author=K Woodfine; citation_author=AJ Wood; citation_author=JD Choi; citation_journal_title=Nucleic Acids Res; citation_volume=34; citation_number=55; citation_pages=e88; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Large-scale copy number polymorphism in the human genome.; citation_author=J Sebat; citation_author=B Lakshmi; citation_author=J Troge; citation_author=J Alexander; citation_author=J Young; citation_journal_title=Science; citation_volume=305; citation_number=56; citation_pages=525-528; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Oligonucleotide microarray analysis of genomic imbalance in children with mental retardation.; citation_author=JM Friedman; citation_author=A Baross; citation_author=AD Delaney; citation_author=A Ally; citation_author=L Arbour; citation_journal_title=Am J Hum Genet; citation_volume=79; citation_number=57; citation_pages=500-513; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Identification of large-scale human-specific copy number differences by inter-species array comparative genomic hybridization.; citation_author=V Goidts; citation_author=L Armengol; citation_author=W Schempp; citation_author=J Conroy; citation_author=N Nowak; citation_journal_title=Hum Genet; citation_volume=119; citation_number=58; citation_pages=185-198; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Identification by full-coverage array CGH of human DNA copy number increases relative to chimpanzee and gorilla.; citation_author=GM Wilson; citation_author=S Flibotte; citation_author=PI Missirlis; citation_author=MA Marra; citation_author=S Jones; citation_journal_title=Genome Res; citation_volume=16; citation_number=59; citation_pages=173-181; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Genomic disorders: molecular mechanisms for rearrangements and conveyed phenotypes.; citation_author=JR Lupski; citation_author=P Stankiewicz; citation_journal_title=PLoS Genet; citation_volume=1; citation_number=60; citation_pages=e49; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Structural Variation in the Human Genome.; citation_author=JR Lupski; citation_journal_title=N Eng J Med (submitted); citation_number=61; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=High-throughput analysis of subtelomeric chromosome rearrangements by use of array-based comparative genomic hybridization.; citation_author=JA Veltman; citation_author=EF Schoenmakers; citation_author=BH Eussen; citation_author=I Janssen; citation_author=G Merkx; citation_journal_title=Am J Hum Genet; citation_volume=70; citation_number=62; citation_pages=1269-1276; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=In vivo dissection of the chromosome condensation machinery: reversibility of condensation distinguishes contributions of condensin and cohesin.; citation_author=BD Lavoie; citation_author=E Hogan; citation_author=D Koshland; citation_journal_title=J Cell Biol; citation_volume=156; citation_number=63; citation_pages=805-815; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Visualization of early chromosome condensation: a hierarchical folding, axial glue model of chromosome structure.; citation_author=N Kireeva; citation_author=M Lakonishok; citation_author=I Kireev; citation_author=T Hirano; citation_author=AS Belmont; citation_journal_title=J Cell Biol; citation_volume=166; citation_number=64; citation_pages=775-785; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Folding and organization of a contiguous chromosome region according to the gene distribution pattern in primary genomic sequence.; citation_author=LS Shopland; citation_author=CR Lynch; citation_author=KA Peterson; citation_author=K Thornton; citation_author=N Kepper; citation_journal_title=J Cell Biol; citation_volume=174; citation_number=65; citation_pages=27-38; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=High-resolution analysis of DNA copy number using oligonucleotide microarrays.; citation_author=GR Bignell; citation_author=J Huang; citation_author=J Greshock; citation_author=S Watt; citation_author=A Butler; citation_journal_title=Genome Res; citation_volume=14; citation_number=66; citation_pages=287-295; citation_date=2004; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000327" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Clinical Implementation of Chromosomal Microarray Analysis: Summary of 2513 Postnatal Cases"/>
    <meta name="twitter:description" content="BackgroundArray Comparative Genomic Hybridization (a-CGH) is a powerful molecular cytogenetic tool to detect genomic imbalances and study disease mechanism and pathogenesis. We report our experience with the clinical implementation of this high resolution human genome analysis, referred to as Chromosomal Microarray Analysis (CMA).Methods and FindingsCMA was performed clinically on 2513 postnatal samples from patients referred with a variety of clinical phenotypes. The initial 775 samples were studied using CMA array version 4 and the remaining 1738 samples were analyzed with CMA version 5 containing expanded genomic coverage. Overall, CMA identified clinically relevant genomic imbalances in 8.5% of patients: 7.6% using V4 and 8.9% using V5. Among 117 cases referred for additional investigation of a known cytogenetically detectable rearrangement, CMA identified the majority (92.5%) of the genomic imbalances. Importantly, abnormal CMA findings were observed in 5.2% of patients (98/1872) with normal karyotypes/FISH results, and V5, with expanded genomic coverage, enabled a higher detection rate in this category than V4. For cases without cytogenetic results available, 8.0% (42/524) abnormal CMA results were detected; again, V5 demonstrated an increased ability to detect abnormality. Improved diagnostic potential of CMA is illustrated by 90 cases identified with 51 cryptic microdeletions and 39 predicted apparent reciprocal microduplications in 13 specific chromosomal regions associated with 11 known genomic disorders. In addition, CMA identified copy number variations (CNVs) of uncertain significance in 262 probands; however, parental studies usually facilitated clinical interpretation. Of these, 217 were interpreted as familial variants and 11 were determined to be de novo; the remaining 34 await parental studies to resolve the clinical significance.ConclusionsThis large set of clinical results demonstrates the significantly improved sensitivity of CMA for the detection of clinically relevant genomic imbalances and highlights the need for comprehensive genetic counseling to facilitate accurate clinical correlation and interpretation."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0000327.g002"/>

  <meta property="og:title" content="Clinical Implementation of Chromosomal Microarray Analysis: Summary of 2513 Postnatal Cases" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000327" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=1713'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=661'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=6968&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0000327">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000327" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000327&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Clinical Implementation of Chromosomal Microarray Analysis: Summary of 2513 Postnatal Cases
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Xinyan Lu, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Molecular and Human Genetics, Baylor College of Medicine, Houston, Texas, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Chad A. Shaw, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Molecular and Human Genetics, Baylor College of Medicine, Houston, Texas, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Ankita Patel, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Molecular and Human Genetics, Baylor College of Medicine, Houston, Texas, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Jiangzhen Li, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Molecular and Human Genetics, Baylor College of Medicine, Houston, Texas, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            M. Lance Cooper, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Molecular and Human Genetics, Baylor College of Medicine, Houston, Texas, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            William R. Wells, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Molecular and Human Genetics, Baylor College of Medicine, Houston, Texas, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Cathy M. Sullivan, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Molecular and Human Genetics, Baylor College of Medicine, Houston, Texas, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Trilochan Sahoo, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Molecular and Human Genetics, Baylor College of Medicine, Houston, Texas, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Svetlana A. Yatsenko, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Molecular and Human Genetics, Baylor College of Medicine, Houston, Texas, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Carlos A. Bacino, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Molecular and Human Genetics, Baylor College of Medicine, Houston, Texas, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Pawel Stankiewicz, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Molecular and Human Genetics, Baylor College of Medicine, Houston, Texas, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Zhishu Ou, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Molecular and Human Genetics, Baylor College of Medicine, Houston, Texas, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            A. Craig Chinault, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Molecular and Human Genetics, Baylor College of Medicine, Houston, Texas, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Arthur L. Beaudet, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Molecular and Human Genetics, Baylor College of Medicine, Houston, Texas, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            James R. Lupski, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Molecular and Human Genetics, Baylor College of Medicine, Houston, Texas, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Sau W. Cheung
              <span class="corresponding">mail</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:scheung@bcm.tmc.edu">scheung@bcm.tmc.edu</a></p>

                <p>Affiliation:
                  Department of Molecular and Human Genetics, Baylor College of Medicine, Houston, Texas, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Patricia A. Ward
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Molecular and Human Genetics, Baylor College of Medicine, Houston, Texas, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: March 28, 2007</li>
    <li>DOI: 10.1371/journal.pone.0000327</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0000327" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0000327" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000327" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0000327" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0000327-t001" data-doi="info:doi/10.1371/journal.pone.0000327" data-uri="info:doi/10.1371/journal.pone.0000327.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000327.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000327-t002" data-doi="info:doi/10.1371/journal.pone.0000327" data-uri="info:doi/10.1371/journal.pone.0000327.t002" title="Table 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000327.t002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000327-g001" data-doi="info:doi/10.1371/journal.pone.0000327" data-uri="info:doi/10.1371/journal.pone.0000327.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000327.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000327-t003" data-doi="info:doi/10.1371/journal.pone.0000327" data-uri="info:doi/10.1371/journal.pone.0000327.t003" title="Table 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000327.t003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000327-g002" data-doi="info:doi/10.1371/journal.pone.0000327" data-uri="info:doi/10.1371/journal.pone.0000327.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000327.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000327-t004" data-doi="info:doi/10.1371/journal.pone.0000327" data-uri="info:doi/10.1371/journal.pone.0000327.t004" title="Table 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000327.t004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000327-t005" data-doi="info:doi/10.1371/journal.pone.0000327" data-uri="info:doi/10.1371/journal.pone.0000327.t005" title="Table 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000327.t005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000327-t006" data-doi="info:doi/10.1371/journal.pone.0000327" data-uri="info:doi/10.1371/journal.pone.0000327.t006" title="Table 6">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000327.t006&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000327">Reader Comments (1)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0000327" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2>
<h3>Background</h3>
<a id="article1.front1.article-meta1.abstract1.sec1.p1" name="article1.front1.article-meta1.abstract1.sec1.p1"></a><p>Array Comparative Genomic Hybridization (a-CGH) is a powerful molecular cytogenetic tool to detect genomic imbalances and study disease mechanism and pathogenesis. We report our experience with the clinical implementation of this high resolution human genome analysis, referred to as Chromosomal Microarray Analysis (CMA).</p>


<h3>Methods and Findings</h3>
<a id="article1.front1.article-meta1.abstract1.sec2.p1" name="article1.front1.article-meta1.abstract1.sec2.p1"></a><p>CMA was performed clinically on 2513 postnatal samples from patients referred with a variety of clinical phenotypes. The initial 775 samples were studied using CMA array version 4 and the remaining 1738 samples were analyzed with CMA version 5 containing expanded genomic coverage. Overall, CMA identified clinically relevant genomic imbalances in 8.5% of patients: 7.6% using V4 and 8.9% using V5. Among 117 cases referred for additional investigation of a known cytogenetically detectable rearrangement, CMA identified the majority (92.5%) of the genomic imbalances. Importantly, abnormal CMA findings were observed in 5.2% of patients (98/1872) with normal karyotypes/FISH results, and V5, with expanded genomic coverage, enabled a higher detection rate in this category than V4. For cases without cytogenetic results available, 8.0% (42/524) abnormal CMA results were detected; again, V5 demonstrated an increased ability to detect abnormality. Improved diagnostic potential of CMA is illustrated by 90 cases identified with 51 cryptic microdeletions and 39 predicted apparent reciprocal microduplications in 13 specific chromosomal regions associated with 11 known genomic disorders. In addition, CMA identified copy number variations (CNVs) of uncertain significance in 262 probands; however, parental studies usually facilitated clinical interpretation. Of these, 217 were interpreted as familial variants and 11 were determined to be <em>de novo</em>; the remaining 34 await parental studies to resolve the clinical significance.</p>


<h3>Conclusions</h3>
<a id="article1.front1.article-meta1.abstract1.sec3.p1" name="article1.front1.article-meta1.abstract1.sec3.p1"></a><p>This large set of clinical results demonstrates the significantly improved sensitivity of CMA for the detection of clinically relevant genomic imbalances and highlights the need for comprehensive genetic counseling to facilitate accurate clinical correlation and interpretation.</p>

</div>


<div class="articleinfo"><p><strong>Citation: </strong>Lu X, Shaw CA, Patel A, Li J, Cooper ML, et al.  (2007) Clinical Implementation of Chromosomal Microarray Analysis: Summary of 2513 Postnatal Cases. PLoS ONE 2(3):
          e327.
            doi:10.1371/journal.pone.0000327</p><p><strong>Academic Editor: </strong>Aravinda Chakravarti, Johns Hopkins Medical School, United States of America</p><p><strong>Received:</strong> October 25, 2006; <strong>Accepted:</strong> March 5, 2007; <strong>Published:</strong> March 28, 2007</p><p><strong>Copyright:</strong> Â© 2007 Lu et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>There is no funding support for this study.</p><p><strong>Competing interests:</strong> The authors are members of the Department of Molecular and Human Genetics at Baylor College of Medicine, which offers extensive genetic laboratory testing and derives revenue from this activity. Baylor College of Medicine, but not the authors personally, own equity in Spectral Genomics, which sells arrays similar to those used in this study.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>Genomic imbalances are a major cause of congenital and developmental abnormalities including dysmorphic features, mental retardation and developmental delay, and multiple congenital anomalies. Using routine cytogenetics techniques, chromosomal imbalances such as aneuploidies and segmental aneusomies must be larger than 3â€“5 Mb in size to be detected by GTG-banding at 450â€“600 band resolution <a href="#pone.0000327-Shaffer1">[1]</a>. In the past two decades, traditional banding has been combined with targeted molecular technologies to improve the resolution at which one can detect genomic changes. Fluorescence In Situ Hybridization (FISH) <a href="#pone.0000327-Trask1">[2]</a>, Comparative Genomic Hybridization (CGH) <a href="#pone.0000327-Rao1">[3]</a> and Multiplex-FISH <a href="#pone.0000327-Speicher1">[4]</a> or Spectral Karyotyping (SKY) <a href="#pone.0000327-Lu1">[5]</a> have been used to detect additional complex or submicroscopic abnormalities; however, such technologies are not suitable for whole genome scans in routine clinical testing because they either lack the necessary resolution or are too time consuming, labor intensive, and/or costly <a href="#pone.0000327-Harris1">[6]</a>â€“<a href="#pone.0000327-Ravnan1">[8]</a>. Array CGH technology has higher resolution and excellent throughput when compared with conventional and molecular cytogenetics <a href="#pone.0000327-Pinkel1">[9]</a>â€“<a href="#pone.0000327-Speicher2">[11]</a>. In array CGH, genomic DNA from the patient and reference are labeled with different fluorescent dyes and co-hybridized to the array matrix containing clone DNA. The content of an array may include specific targeted regions of the genome or the entire genome arrayed on a single glass slide. Similar to conventional CGH, genome imbalances are quantified by analyzing the ratio of the two fluorescent hybridizing signals. The resolution of array CGH is determined by the size and number of the clones placed on the array to interrogate genome copy number changes. Using 1-Mb average coverage (3500 clones) BAC/PAC array analysis was shown to detect cryptic chromosomal imbalances in patients with idiopathic mental retardation and multiple congenital anomalies <a href="#pone.0000327-Vissers1">[12]</a>â€“<a href="#pone.0000327-Menten1">[14]</a>. A tiling path BAC array (32K clones) has been developed and implemented in the clinical research field to identify novel genes associated with disease <a href="#pone.0000327-Ishkanian1">[15]</a>. Chromosomal specific arrays, such as those for distal chromosomes 1p, proximal 17p and chromosome 22, have been utilized to identify subtle deletions and map specific rearrangement breakpoints <a href="#pone.0000327-Yu1">[16]</a>â€“<a href="#pone.0000327-Koolen1">[18]</a>. Using a combination of a 1 Mb interval array and a specific chromosome 8 array, Visser and colleagues were able to identify the critical genomic region and disease gene responsible for CHARGE syndrome <a href="#pone.0000327-Vissers2">[19]</a>. Exon array CGH has also been reported as a method to measure copy-number changes of individual exons in selected genes <a href="#pone.0000327-Dhami1">[20]</a>, although its implementation to broad genome coverage and the ability to discriminate multi-copy-exon encoding domains at the whole genome level remains to be determined. High-resolution array CGH has also been implemented in cancer research. For example, array-based methylated CpG island amplification (BAMCA) was developed to screen aberrant methylated CpG-rich sequences genome-wide in order to study the silencing of tumor suppressor genes <a href="#pone.0000327-Misawa1">[21]</a>.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>Array CGH has unequivocally established its reliability in detecting copy-number changes, and several groups have now implemented this technology in clinical evaluation of both postnatal <a href="#pone.0000327-Bejjani1">[22]</a>â€“<a href="#pone.0000327-Shaffer2">[25]</a> and prenatal patient samples <a href="#pone.0000327-Lee1">[26]</a>â€“<a href="#pone.0000327-Sahoo1">[31]</a>. In comparison with research arrays, the clinical arrays must be carefully designed to precisely detect genomic imbalances for which clinical interpretations can be rendered. Our group initially developed and validated a BAC/PAC array in our clinical cytogenetics laboratory for Chromosomal Microarray Analysis (CMA V4) <a href="#pone.0000327-Cheung1">[24]</a>. This version contained 366 FISH verified BAC/PAC clones that span genomic regions implicated in over 43 known genomic disorders <a href="#pone.0000327-Lupski1">[32]</a> as well as all 41 subtelomeric regions. Subsequently, an expanded version of the array, CMA V5 that contains 853 FISH verified clones was implemented that detects gains and losses due to genomic rearrangements associated with over 70 disorders. Expansion of genome coverage in clinical arrays is always accompanied by the detection of copy number variation (CNVs) that might represent benign polymorphic changes.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>Here, we present clinical experience with two versions of CMA arrays in the evaluation of 2513 clinical postnatal patient samples studied in our clinical diagnostic laboratory between February 2004 and March 2006 (<a href="#pone-0000327-t001">Table 1</a>). We compare these two versions with respect to detection rates for pathological chromosomal abnormalities and for CNVs, the latter most often representing benign variation. Our substantive dataset enables robust assessment of the implementation of this technology in patient management and as an aid to the establishment of a specific etiologic diagnosis.</p>
<div class="figure" id="pone-0000327-t001"><div class="img"><a name="pone-0000327-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000327.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000327" data-uri="info:doi/10.1371/journal.pone.0000327.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000327.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000327.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000327.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000327.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000327.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000327.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000327.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000327.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000327.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>General comparison of CMA results correlated with routine cytogenetic results</span></strong></p><span>doi:10.1371/journal.pone.0000327.t001</span></div></div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3><a id="article1.body1.sec2.p1" name="article1.body1.sec2.p1"></a><p>All clinically relevant abnormal CMA results including <em>de novo</em> single clone changes <a href="#pone.0000327-Tyson1">[33]</a> described in this section are listed in the Supplementary <a href="#pone.0000327.s001">Tables S1</a> and <a href="#pone.0000327.s002">S2</a>; case ID numbers V4-1~59 were performed on CMA V4 and ID numbers V5-1~154 were conducted on CMA V5.</p>

<h4>CMA Concordance with Abnormal Cytogenetics Findings</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>The molecular based and targeted CMA approach detected the majority (92.5%) of those genomic imbalances identified by cytogenetic studies, including numerical changes, deletions and duplications (N = 43) and two marker chromosomes (case V4-27 and case V5-115) as well as two cases with &gt;50% mosaicism involving a ring chromosome 22 (case V4-50) or monosomy X (case V4-52). CMA further identified the additional material of unknown origin and delineated breakpoints on all derivative chromosomes, thereby often having the added advantage of refining genomic intervals involved in the imbalance (N = 26) such as in case V4-31, case V5-41 and case V5-35 (<a href="#pone.0000327.s001">Table S1</a> and <a href="#pone.0000327.s002">S2</a>). Moreover, on CMA V4, (case V4-33) a microduplication at 17p11.2-p12, that includes both the Smith-Magenis syndrome (SMS) and Charcot-Marie Tooth disease type 1 A (CMT1A) critical regions, was observed in one patient previously reported as having an abnormal 47,XYY karyotype. The additional genomic imbalance detected by CMA may explain the clinical findings including growth retardation and dysmorphic features, which were inconsistent with 47,XYY karyotype in this patient. Likewise, in a patient (case V5-144) with autistic behavior, developmental delay and dysmorphic features for whom cytogenetic studies showed a 47,XXY karyotype, CMA V5 revealed a smaller gain in the pericentromeric region of chromosome 7 including the Williams-Beuren syndrome (WBS) critical region at 7q11.23, which was confirmed by FISH analysis to be caused by a mosaic small ring chromosome 7 in 33% of the cells.</p>
<a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2"></a><p>As anticipated, neither CMA V4 nor V5 could detect the apparently balanced translocations (N = 12), inversions (N = 21), insertions (N = 2), fra(X) expansions (N = 2) or extremely low level (&lt;10%) mosaicism (N = 3). CMA V4 also failed to identify one marker chromosome, but this could be due to the limited array coverage. However, CMA V5 identified gains of the DiGeorge syndrome/Velocardiofacial syndrome (DGS/VCFS) genomic region not identified in two patients undergoing cytogenetic evaluations, which described apparently balanced translocations. One patient (case V5-109) with developmental delay and ambiguous genitalia had a karyotype indicating 46,XX,t(2;9)(p25.3;p22.1) whereas the other patient (case V5-110) manifested developmental delay and dysmorphic features and the identified karyotype of a three-way translocation 46,XX,t(12;18;13)(q14;q21.3;q14.2). Thus, although CMA does not detect balanced translocations or inversions it may uncover other cryptic genomic imbalances that are not detected by conventional cytogenetics in patients with phenotypic abnormalities and apparently balanced rearrangements. Interestingly, V5 also detected an apparent low level mosaic trisomy 14 (case V5-143); although subsequently validated by karyotype and FISH, both of these methods identified only 2% of cells with trisomy 14 in PHA stimulated T-cells <a href="#pone.0000327-Cheung2">[34]</a>.</p>


<h4>Clinically Relevant Genomic Imbalances Identified by CMA in Cases with Normal Cytogenetics Findings</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>CMA identified genomic imbalances not detected by conventional clinical cytogenetic analyses in a total of 5.2% of the patients; this represents 4.1% (20 losses and 5 gains) and 5.9% (39 losses and 38 gains) rates using V4 or V5, respectively (<a href="#pone-0000327-t001">Table 1</a>). Each of these abnormalities was confirmed independently by repeated GTG-banding and/or FISH tests. The confirmatory method used was dependent on the size of the genomic imbalance detected by CMA. These findings indicate that increasing the genomic coverage on the V5 targeted array enabled higher resolution and a 44% higher rate of detection of genomic imbalance compared to V4.</p>
<a id="article1.body1.sec2.sec2.p2" name="article1.body1.sec2.sec2.p2"></a><p>All the patients studied in this category are summarized in 5 groups based on the clinical indications provided at time of referral in <a href="#pone-0000327-t002">Table 2</a>. In group I, which are the patients with DD/MR only, the CMA clinically relevant detection rate was 3.9%; in group II, which are patients whose clinical indication was a combination of DD/MR and DF or MCA, CMA detected clinically relevant imbalances in 5.6%. CMA was most clinically sensitive, with the highest detection rate at 8.4%, in Group III, which are the patients with dysmorphic features (DF), multiple congenital anomalies (MCA), or the combination of both. Our targeted CMA detected fewer (3.4%) genomic imbalances in the Group IV patients with autistic behaviors; however, this group of patients will be further analyzed using a higher density array. CMA sensitivity in group V patients whose clinical indications represent â€œothersâ€ (e.g., seizure disorders, learning disability, failure to thrive, etc.) was 5.3%.</p>
<div class="figure" id="pone-0000327-t002"><div class="img"><a name="pone-0000327-t002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000327.t002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000327" data-uri="info:doi/10.1371/journal.pone.0000327.t002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000327.t002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000327.t002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000327.t002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000327.t002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000327.t002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000327.t002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000327.t002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000327.t002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000327.t002.TIF"></span>)
                </a></li></ul></div><p><strong>Table 2.  <span>CMA results correlated with clinical indication in the patients with normal cytogenetic tests</span></strong></p><span>doi:10.1371/journal.pone.0000327.t002</span></div><a id="article1.body1.sec2.sec2.p3" name="article1.body1.sec2.sec2.p3"></a><p>There were three cases in the category reported to have normal karyotype, normal locus specific FISH test at the DGS/VCFS critical region and normal telomeric FISH test. CMA V4 identified a <em>de novo</em> gain in the Miller-Dieker lissencephaly syndrome (MDLS) critical region in 17p13.3 for one patient (case V4-35); V5 revealed one duplication at the DGS/VCFS located in 22q11.2 (case V5-103), and a <em>de novo</em> gain at 4q22.3 (case V5-31) that was detected by a single clone. CMA V5 also detected five cases with mosaicism for aneuploidies ranging from 10â€“60% (cases V5-121, V5-151~V5-154) that were not detected by routine cytogenetic analysis. The somewhat surprising ability of CMA to identify mosaicism not detected by karyotype analysis may be due to the difference in samples used for each test, as only PHA-stimulated T-cells are analyzed in routine chromosomal analysis whereas CMA uses genomic DNA from whole blood containing multi-lineage nucleated cells <a href="#pone.0000327-Cheung2">[34]</a>.</p>


<h4>Clinically Relevant Genomic Imbalances Identified by CMA in Cases without a Karyotype</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>For this group of patients, the sensitivity for detecting clinically relevant genomic imbalances was 5.9% (3/51) on version 4 and 8.2% (39/473) on version 5, respectively (<a href="#pone-0000327-t001">Table 1</a>). All the genomic imbalances detected by CMA were verified using the most appropriate method, either GTG-banding or FISH. Using CMA V4, there were two cases identified with microdeletions, one at 1p36.3 (case V4-3) and one at the DGS/VCFS locus (case V4-51). Using CMA V5, 19 cases had interstitial microrearrangements (12 losses and 7 gains), and 14 cases were identified with subtelomeric rearrangements. In retrospective high-resolution GTG-banding analyses of these 42 abnormal cases, 24 of these genomic imbalances were not detectable by routine cytogenetic analysis (<a href="#pone.0000327.s001">Table S1</a> and <a href="#pone.0000327.s002">S2</a>).</p>
<a id="article1.body1.sec2.sec3.p2" name="article1.body1.sec2.sec3.p2"></a><p>Notably, there was an increase in the overall detection rate of genomic imbalances with CMA V5 (8.9%) relative to CMA V4 (7.6%) (<a href="#pone-0000327-t001">Table 1</a>), indicating that the increase in coverage of genomic disorder regions in the array enables additional imbalances to be identified in these screening tests even in the absence of case selection on the basis of normal cytogenetic studies.</p>


<h4>Microdeletions and the Reciprocal Microduplications</h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>There were 90 cases in which losses or gains associated with cryptic chromosomal microdeletions (51 cases) or the apparent predicted microduplications (39 cases) were detected (<a href="#pone-0000327-t003">Table 3</a>). Thirteen chromosomal regions associated with 11 known disorders involving microdeletions and microduplications, including 1p36 monosomy, 1q21.1 (novel), 1q44 (novel), Wolf-Hirschhorn syndrome (WHS) (4p16.3), cri-du-chat syndrome (CDCS) (5p15.3), WBS (7q11.23), WAGR (Wilm's tumor, aniridia, genital/or urinary tract abnormalities, mental retardation) (11p13), Prader-Willi/Angelman syndromes (PWS/AS) (15q11.2), MDLS (17p13.3), CMT1A/hereditary neuropathy with pressure palsies (HNPP) (17p12), SMS (17p11.2), neurofibromatosis type 1 (NF1) (17q11.2) and DGS/VCFS (22q11.2). For all 13 regions that were observed to undergo genomic losses, we also found cases with gains potentially representing reciprocal duplications; such events were observed at unequal frequencies (<a href="#pone-0000327-t003">Table 3</a>). The most commonly identified rearranged genomic regions include 1p36 (11 del/5 dup), 7q11.23 (6 del/3 dup), 15q11.2 (3 del/3 dup) and 22q11.2 (17 del/7 dup). Among these 90 cases, a subset of 70 patients (45 deletions and 25 duplications), had undergone previous cytogenetics and subtelomeric and/or locus specific FISH tests. Of these, 35 deletions (78%) and 23 duplications (92%) were not detected by conventional cytogenetic methods. Chromosomal regions at 1q21.1 (2 del/4 dup) and 1q44 (1 del/5 dup) include microrearrangements which have not been extensively reported in the literature. Genotype-phenotype correlation studies are ongoing to determine whether imbalances in these two regions represent novel genomic disorders <a href="#pone.0000327-Lupski2">[35]</a>. <a href="#pone-0000327-g001">Figure 1</a> illustrates examples of three pairs of microdeletions and apparent reciprocal duplication syndromes at 15q11.2q13, 17q11.2 and 22q11.2 detected by CMA V4 and V5.</p>
<div class="figure" id="pone-0000327-g001"><div class="img"><a name="pone-0000327-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000327.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000327" data-uri="info:doi/10.1371/journal.pone.0000327.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000327.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000327.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000327.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000327.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000327.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000327.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000327.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000327.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000327.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1. </strong></p><a id="article1.body1.sec2.sec4.fig1.caption1.p1" name="article1.body1.sec2.sec4.fig1.caption1.p1"></a><p>Example of reciprocal deletion/duplication CMA ratio plots with FISH validation. A Three sets of ratio plots showing examples of reciprocal microdeletion and microduplication at PWS/AS (on V5), NF1 (on V4) and DGS/VSFS (on V5) critical regions on chromosomes 15q11.2, 17q11.2 and 22q11.2, respectively. The vertical lines represent the BAC clones interrogating targeted regions of the genome, aligned according to their chromosomal position with the signal intensity ratios observed from the combined dye-swap experiments. Gains (G) are to the right and losses (L) to the left. Red circles highlight the signal ratios revealing genomic losses whereas the blue circles indicate copy number gains consistent with genomic deletions and reciprocal duplications, respectively. B FISH validation results. Control probes were labeled with a green chromophore and clones detecting deletions (in metaphase) or duplications (in interphase) were labeled in red for PWS/AS del/dup, NF1 del/dup and DG/VCFS del/dup.</p>
<span>doi:10.1371/journal.pone.0000327.g001</span></div><div class="figure" id="pone-0000327-t003"><div class="img"><a name="pone-0000327-t003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000327.t003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000327" data-uri="info:doi/10.1371/journal.pone.0000327.t003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000327.t003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000327.t003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000327.t003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000327.t003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000327.t003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000327.t003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000327.t003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000327.t003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000327.t003.TIF"></span>)
                </a></li></ul></div><p><strong>Table 3.  <span>Microdeletions and the potential reciprocal duplications</span></strong></p><span>doi:10.1371/journal.pone.0000327.t003</span></div><a id="article1.body1.sec2.sec4.p2" name="article1.body1.sec2.sec4.p2"></a><p>Moreover, CMA V5 detected an additional eight cases with Xq28 gains that include the <em>MECP2</em> gene responsible for Rett syndrome <a href="#pone.0000327-delGaudio1">[36]</a>. Both CMA V4 and V5 illustrated improved potential over conventional cytogenetic analysis for identifying genomic imbalances associated with known microdeletions and evolving microduplication syndromes.</p>


<h4>Copy Number Variations (CNVs)</h4>
<a id="article1.body1.sec2.sec5.p1" name="article1.body1.sec2.sec5.p1"></a><p>Two different types of apparent copy number variations were detected (<a href="#pone-0000327-t004">Table 4</a>). Multi-clone CNVs were classified as genomic segmental CNVs involving multiple interrogating clones from a single genetic locus whereas single clone CNVs represent genomic changes detected by only a single clone. Multi-clone CNVs, detected in 50 cases by CMA V4 and 69 cases on V5, were observed multiple times with either gains (in the majority of cases) or losses at five pericentromeric or subtelomeric regions including 2q13 (8/V4, 19/V5), 11q25 (14/V5), 15q11.2 (10/V4 and 14/V5), Xp22.3 (6/V4 and 13/V5), Yq11.2 (<em>AZFc</em>) (26/V4 and 9/V5) (<a href="#pone-0000327-g002">Figure 2</a>). Most of these changes were also found in the phenotypically normal parents, suggesting likely benign alterations; however, the potential clinical relevance of these CNVs remains unknown. We interpreted several multi-clone CNVs as likely representing normal variants by comparison to the Database of Genomic Variants <a href="#pone.0000327-Redon1">[37]</a> (<a href="http://www.sanger.ac.uk/PostGenomics/decipher">http://www.sanger.ac.uk/PostGenomics/decâ€‹ipher</a>).</p>
<div class="figure" id="pone-0000327-g002"><div class="img"><a name="pone-0000327-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000327.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000327" data-uri="info:doi/10.1371/journal.pone.0000327.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000327.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000327.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000327.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000327.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000327.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000327.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000327.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000327.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000327.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2. </strong></p><a id="article1.body1.sec2.sec5.fig1.caption1.p1" name="article1.body1.sec2.sec5.fig1.caption1.p1"></a><p>Ideogram of Copy Number Variations (CNVs). Big arrows indicate CNVs Type 1 (N = 119), except <em>AZFc</em> on the Y chromosome (N = 35). * represents V4 and for V5 for CNVs Type 2 (N = 143). A total of 262 copy number variations (CNVs) were identified by CMA. Of these, 217 were interpreted as familial variants (green) and 11 were determined to be <em>de novo</em> (red); 34 CNVs that await parental studies are depicted with unknown clinical significance (black). Positions showing losses are on the left side of the chromosome whereas gains are on the right side.</p>
<span>doi:10.1371/journal.pone.0000327.g002</span></div><div class="figure" id="pone-0000327-t004"><div class="img"><a name="pone-0000327-t004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000327.t004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000327" data-uri="info:doi/10.1371/journal.pone.0000327.t004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000327.t004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000327.t004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000327.t004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000327.t004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000327.t004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000327.t004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000327.t004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000327.t004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000327.t004.TIF"></span>)
                </a></li></ul></div><p><strong>Table 4.  <span>Copy Number Variations (CNVs)</span></strong></p><span>doi:10.1371/journal.pone.0000327.t004</span></div><a id="article1.body1.sec2.sec5.p2" name="article1.body1.sec2.sec5.p2"></a><p>There were 34 clones on version 4 and 109 clones on version 5 that were noted to reveal single clone gains or losses (<a href="#pone-0000327-g002">Figure 2</a>). The results for each single clone CNV were independently confirmed by FISH. A total of 98 CNVs (27 on V4 and 71 on V5) were inherited from normal parents as determined by either parental FISH or CMA testing and thus reported as familial variants. The remaining 45 changes (seven on V4 and 38 on V5) were queried on the public normal variation database (Toronto database <a href="http://projects.tcag.ca/variation">http://projects.tcag.ca/variation</a>) and the database on array CGH published by Iafrate et al <a href="#pone.0000327-Iafrate1">[38]</a>, and none of these were reported as known normal variants. Of the 45 single clone CNVs, there were 11 (two on V4 and nine on V5) that revealed <em>de novo</em> (<a href="#pone-0000327-g002">Figure 2</a>) rearrangements in comparison to parental samples and thus, these are considered potentially causative for a phenotype <a href="#pone.0000327-Tyson1">[33]</a>. It should be noted that paternity testing was not performed for these cases. For example, a single clone deletion (RP5-888M10) at 1p36.1 on CMA V4 was seen in two patients (cases V4-1~2) whose clinical phenotypic appears distinct from the 1p36.3 deletion syndrome and repeated CMA of one patient on V5, with expanded coverage to 1p36.1 region, showed a more than 10 Mb deletion proximal to the typical 1p36.3 syndrome region. Another single clone deletion (RP11-12L9) on chromosome 7 that encodes the <em>FOXP2</em> gene located at 7q31.1 was detected in case V5-48. Parental <em>FISH</em> studies with this clone further indicated that the second copy of this clone was inserted into one chromosome 10 in the mother, uncovering a complex rearrangement in the child. This rearrangement was interpreted to be the cause of patient's phenotype of speech and language impairment <a href="#pone.0000327-Lennon1">[39]</a>. Finally, 34 clones revealing gains or losses were considered as CNVs with unknown clinical significance and await familial studies for interpretation.</p>

</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3>
<h4>CMA Increases Detection of Genomic Imbalances</h4>
<a id="article1.body1.sec3.sec1.p1" name="article1.body1.sec3.sec1.p1"></a><p>Array CGH offers the potential for high throughput and high-resolution genome analysis and is a less labor intensive approach than other molecular cytogenetic analyses. In this study, we illustrate the value of array CGH in a large collection of clinical data (2513 cases). The targeted array enabled the detection of genomic imbalances, especially in the patients with previous or concurrent normal cytogenetic tests that were referred for DF, MCA or both (<a href="#pone-0000327-t002">Table 2</a>). As the targeted genomic coverage was increased, 17 additional abnormal cases were detected on the V5 array, including six cases associated with the newly assayed genomic regions, nine cases involving newly added pericentromeric regions and two cases with gains/losses of subtelomeric clones (data not shown), thus resulting in an enhanced overall detection rate. This greater genomic coverage on V5 also increased the rate at which single clone CNVs were identified. The heightened overall detection rate using V5 may also be due to factors such as technical improvements, improved experience in the data interpretation, and better selection of the clone coverage. For example, selected reduction of clones revealing known multi-clone CNVs led to a very significant reduction (57.5%) in the identification of these CNVs in patient samples using targeted V5 versus V4 arrays (<a href="#pone-0000327-t004">Table 4</a>).</p>
<a id="article1.body1.sec3.sec1.p2" name="article1.body1.sec3.sec1.p2"></a><p>Submicroscopic or subtelomeric chromosomal rearrangements have been found in about 6% of patients with otherwise unexplained mental retardation or multiple congenital anomalies <a href="#pone.0000327-Ravnan1">[8]</a>, <a href="#pone.0000327-Biesecker1">[40]</a>. Array CGH has the capability of identifying such subtelomeric cryptic imbalances <a href="#pone.0000327-Hwang1">[41]</a>, <a href="#pone.0000327-Le1">[42]</a>. In addition to the previously identified interstitial and subtelomeric imbalances, our CMA detected also all genomic gains and losses in patients with both normal karyotypes and FISH results. Because subtelomeric FISH analyses routinely include only one clone for each chromosome end, smaller and/or interstitial rearrangements can be missed. Similarly, microduplications are often missed by metaphase FISH because the duplicated signals are in close proximity to each other and cannot be distinguished on metaphase chromosomes; interphase FISH is required to resolve the duplicated signals and it is important to differentiate duplication from replication in a G2 phase nucleus <a href="#pone.0000327-Lupski3">[43]</a>. Using CMA we identified four cases with gains at telomeric regions that were missed in previous subtelomeric FISH analyses. One of these patients had dysmorphic features, developmental delay and an autistic-like behavioral phenotype. Normal results were reported for routine G-banding, subtelomeric and probe specific FISH (at DGS/VCFS region) and fragileÃ—testing. CMA identified a three-clone <em>de novo</em> gain due to duplication at the MDLS locus on 17p13.3 and these findings were confirmed by FISH. This latter case further illustrates the power of CMA as a rapid diagnostic technology capable of simultaneously interrogating multiple regions of the genome in a single test.</p>
<a id="article1.body1.sec3.sec1.p3" name="article1.body1.sec3.sec1.p3"></a><p>Array CGH also has an advantage in detecting microdeletions and the reciprocal microduplications at both interstitial and subtelomeric regions. A common mechanism for recurrent interstitial microrearrangements has been determined to result from <span class="underline">n</span>on<span class="underline">a</span>llelic <span class="underline">h</span>omologous <span class="underline">r</span>ecombination (NAHR) (intrachromosomal, interchromsomal or intrachromatid unequal crossing-over) between low-copy repeats (LCRs) <a href="#pone.0000327-Lupski4">[44]</a>, <a href="#pone.0000327-Stankiewicz1">[45]</a>. Such NAHR may produce two reciprocal products; a tandem (or direct) duplication and a deletion. The <em>de novo</em> events are predicted to occur at similar frequencies unless there is rearrangement specific selection or ascertainment bias, and, thus, it might be anticipated that both events would be seen in the population with equal frequency unless prenatal lethality, a milder phenotype with less likelihood to be studied or reduced penetrance occurs for the duplication or the deletion. Several reciprocal recombination disorders were delineated under this hypothesis, including duplications causing CMT1A and deletions causing HNPP <a href="#pone.0000327-Lupski3">[43]</a>, <a href="#pone.0000327-Shaffer3">[46]</a> at 17p12 and SMS deletion and Potocki-Lupski dup(17)(p11.2p11.2) syndrome <a href="#pone.0000327-Potocki1">[47]</a> as well as rearrangements in 7q11.23, 15q11.2 and 22q11.2 <a href="#pone.0000327-Inoue1">[48]</a>. In this study, CMA identified 90 cases that involved 13 different reciprocal microduplication-microdeletion rearrangements (<a href="#pone-0000327-t003">Table 3</a>). The most common cytogenetic regions involved include 1p36.3, 15q11-q13, and 22q11.2. Two of 13 regions were novel and patients with these reciprocal rearrangements are now being analyzed in depth.</p>


<h4>Balanced Translocations and ArrayCGH</h4>
<a id="article1.body1.sec3.sec2.p1" name="article1.body1.sec3.sec2.p1"></a><p>CMA is limited to identifying genomic imbalances. Thus, balanced chromosomal translocations, inversions and insertions are not detected. Balanced translocations, in theory, should be benign unless the breakpoint disrupts a dosage sensitive gene or conveys a position effect. This is particularly important to consider in patients who have an abnormal phenotype, but the karyotype appears to represent a <em>de novo</em> balanced translocation. Several groups have focused on array CGH studies of patients with abnormal phenotypes and apparently balanced (primarily <em>de novo</em>) translocations based on routine cytogenetic analysis. Ciccone <a href="#pone.0000327-Ciccone1">[49]</a> et al reported that three out of four of such patients with apparently balanced translocations were actually unbalanced, and had complex rearrangements. They proposed that the array detected imbalances accounting for the mental retardation and other phenotypic abnormalities observed. Using a 1 Mb genome array, a cryptic 0.15â€“1.5 Mb deletion was detected at one of eight breakpoints of an apparently balanced three-way chromosomal translocation in a patient with behavior problems <a href="#pone.0000327-Rosenberg1">[50]</a>. In another study, involving ten patients with <em>de novo</em> balanced translocations and abnormal phenotypes, a 1 Mb array detected genomic imbalances in six patients <a href="#pone.0000327-Gribble1">[51]</a>. Thus, array CGH has the potential to detect imbalances in a considerable percentage of patients who have apparently balanced translocations based on routine cytogenetics, particularly in patients with abnormal phenotypes. For this application, tiling path rather than focused arrays are more appropriate.</p>
<a id="article1.body1.sec3.sec2.p2" name="article1.body1.sec3.sec2.p2"></a><p>In patients with a previous cytogenetic study, CMA did not detect additional gains or losses in 12 patients with apparently balanced translocations (4 <em>de novo</em>, 2 familial and 6 unknown). There are three possible explanations. First, there were neither genomic regulatory elements nor genes disrupted at breakpoint regions or if they were, the genes were not dosage sensitive. Second, the negative CMA results of balanced translocations could reflect the limitation of our current array with respect to clone coverage at these chromosomal breakpoints. Third, the unbalanced rearrangements at breakpoints were too small to detect using BAC/PAC array technology. Although we did not identify any genomic imbalances at the breakpoint regions in patients who had previously reported balanced translocations, we did discover microduplications at 22q11.2 in two patients that might account for their abnormal phenotypes. A recently developed array painting technology <a href="#pone.0000327-Gribble2">[52]</a>, <a href="#pone.0000327-Fauth1">[53]</a> could potentially aid in detection and further resolution of complex chromosomal translocations that appear to be balanced by providing higher resolution genome and accurate breakpoint analyses. Likewise, extremely high density microarrays, such as oligonucleotide arrays <a href="#pone.0000327-Baldocchi1">[54]</a> could be used to characterize apparently balanced translocations, inversions, genomic imprinting and with the inclusion of oligonucleotides interrogating single nucleotide polymorphisms (SNPs) also detect uniparental disomy (UPD) <a href="#pone.0000327-Schulz1">[55]</a>.</p>


<h4>Increasing Clone Coverage with Targeted Arrays Detected More Genomic Imbalances but also Increased the Detection of Benign Variants</h4>
<a id="article1.body1.sec3.sec3.p1" name="article1.body1.sec3.sec3.p1"></a><p>Several recent studies have emphasized the extent of copy-number variations (CNVs) in the human genome. Iafrate et al. used a human 1 Mb BAC array with ~3000 clones on a panel of 55 unrelated individuals and found more than 200 clones detecting CNVs, 24 of which were present in &gt;10% of the individuals; there was an average of 12.4 CNVs per individual <a href="#pone.0000327-Iafrate1">[38]</a>. Sebat et al. using oligonucleotide arrays to study only 20 individuals described 221 copy number differences representing 76 unique CNVs; they found an average of 11 CNVs per individual and the average length of the CNVs was 465 kb <a href="#pone.0000327-Sebat1">[56]</a>. Both studies found 60â€“70 entire genes within CNVs. One study using oligonucleotide arrays found a median of 30 (range 19â€“34) CNVs per index case with mental retardation <a href="#pone.0000327-Friedman1">[57]</a>. More recently, Goidts and colleagues found copy number differences by inter-species array CGH analysis, and a total of 322 sites of large-scale interspecies differences which were identified by comparing human genomes with those of other primates. They suggested that human-specific duplications (i.e. 2q13) may predispose to chromosomal rearrangement <a href="#pone.0000327-Goidts1">[58]</a>. By using full coverage BAC array CGH, Wilson et al. discovered that human DNA copy-number is often increased relative to chimpanzee and gorilla. Several chromosomal regions, such as 1p36.13, 1q21.1, 2q13, 4p16.3, 7q11.23, 9p12, 15q11.2, were established with higher copy number changes in human <a href="#pone.0000327-Wilson1">[59]</a>. Most of these regions overlap with those involved with known human microdeletions and microduplication syndromes <a href="#pone.0000327-Stankiewicz1">[45]</a>, <a href="#pone.0000327-Lupski5">[60]</a>.</p>
<a id="article1.body1.sec3.sec3.p2" name="article1.body1.sec3.sec3.p2"></a><p>In the current study, the enhanced resolution of CMA V5 improved the overall detection rate of chromosomal abnormalities significantly (<a href="#pone-0000327-t005">Table 5</a>), but also identified more single clone CNVs; however, the multi-clone CNV detection rate was reduced (<a href="#pone-0000327-t004">Table 4</a>) due to improved array design. We identified two major types of CNVs according to likely phenotypic effects. One group involving five chromosomal regions 2q13, 11q25, 15q11.2, Xp22.3 and Yq12 (<em>AZFc</em>) were hypothesized as benign variations because of their presence in a reportedly unaffected parent. This interpretation is consistent with the CNV database (Toronto database <a href="http://projects.tcag.ca/variation">http://projects.tcag.ca/variation</a>). However, a continuing genotype-phenotype correlation study is required to better understand any potential relevance of these CNVs to a particular phenotype, particularly when inherited in combination as distinct alleles from each parent <a href="#pone.0000327-Lupski5">[60]</a>, <a href="#pone.0000327-Lupski6">[61]</a>. CNVs involving at least 34 BAC/PAC clones remain of uncertain clinical significance; the majority of these CNVs involve gain (N = 26) detected by clones located at the telomeric/subtelomeric and pericentromeric regions. It is uncertain at present what proportion of these variations are benign versus pathological, and whether they occur with increased frequency near telomeres <a href="#pone.0000327-Veltman1">[62]</a>. <em>De novo</em> gains or losses are much more likely to be pathological <a href="#pone.0000327-Tyson1">[33]</a> compared to those inherited from a normal parent, and one study using SNP arrays interpreted the clinical significance almost entirely according to whether variation was inherited or <em>de novo</em> <a href="#pone.0000327-Friedman1">[57]</a>. However, variants could occasionally have variable penetrance with the same allele causing a phenotype in the child but not the parent. Studies of variants within extended families may be required to determine if certain CNVs have variable penetrance for phenotypic effects. Additional investigations of family members, computational analysis in combination with published databases and systematic analysis of genomic polymorphisms will provide further insight into the plasticity of the human genome, therefore helping to determine if these are normal variations, disease susceptibility variants, or may cause disease in association with other alleles <a href="#pone.0000327-Lupski5">[60]</a>.</p>
<div class="figure" id="pone-0000327-t005"><div class="img"><a name="pone-0000327-t005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000327.t005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000327" data-uri="info:doi/10.1371/journal.pone.0000327.t005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000327.t005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000327.t005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000327.t005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000327.t005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000327.t005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000327.t005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000327.t005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000327.t005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000327.t005.TIF"></span>)
                </a></li></ul></div><p><strong>Table 5.  <span>Comparison of targeted array CGH clinical implementation studies</span></strong></p><span>doi:10.1371/journal.pone.0000327.t005</span></div>

<h4>Does Genomic Sequence Determine Chromosome Folding?</h4>
<a id="article1.body1.sec3.sec4.p1" name="article1.body1.sec3.sec4.p1"></a><p>There were 14 cases on V4 and 46 cases on V5 reported with a normal karyotype and/or FISH performed either prior to CMA or concurrently with CMA. After CMA revealed the abnormalities of deletions and/or duplications ranging from 0.5 Mb to over than 10 Mb, a high resolution (at average 650 banding) retrospective karyotype was applied (<a href="#pone.0000327.s001">Tables S1</a> and <a href="#pone.0000327.s002">S2</a>). In general, on V4, six cases (&lt;3 Mb) of 14 showed no evidence of cytogenetic abnormalities while the remaining eight cases (&gt;5 Mb) were found to have abnormal chromosomes. On V5, 33 cases showed a negative retrospective karyotype whereas the remaining 13 cases revealed visible chromosomal abnormalities. The estimated smallest rearrangement visible by cytogenetic banding was a 2.7 Mb loss (in case V5-28), and the estimated largest size aberration missed by traditional cytogenetics was a 7.2 Mb duplication (in case V5-59). However, the size of deletion or duplication estimated by CMA might not truly reflect the exact size of the rearrangement due to the limitation of the current targeted array with lack of clone coverage at some of the chromosome regions. Our data from retrospective karyotyping raises an important question: why are some changes of ~3 Mb cytogenetically visible whereas ~7 Mb changes may not be seen cytogenetically? Does this relate to the mechanism of packaging of the genome into chromosomes and genomic sequences required for chromosome folding revealed by G-banding? Potential explanations for discrepancies between genome size and cytogenetic detection include: 1) altered trypsin access to the folded chromosomes, 2) chromosomes failing to stretch during the preparations in the region of genomic imbalances or, 3) the distribution of chromosome condensation and the hierarchical folding <a href="#pone.0000327-Lavoie1">[63]</a>, <a href="#pone.0000327-Kireeva1">[64]</a>. Shopland and colleagues found that the gene distribution pattern in primary sequence contributes to chromosome folding and organization in mice <a href="#pone.0000327-Shopland1">[65]</a>, and thus genomic sequence could be involved in the determination of chromosome folding. Systematic evaluation of genomic sequences from regions discrepant for rearrangement size visualized by arrays versus G-banding may reveal further insight into chromosome folding.</p>


<h4>Future Perspectives</h4>
<a id="article1.body1.sec3.sec5.p1" name="article1.body1.sec3.sec5.p1"></a><p>In conclusion, this study demonstrates that CMA is a valid methodology for clinical testing. CMA is equivalent to performing hundreds of subtelomeric and locus specific FISH tests simultaneously, and it also reliably detects duplications that are not usually seen by metaphase FISH. Thus, CMA offers an efficient and high-throughput alternative for detecting and characterizing genomic imbalances. An informed approach in targeted array CGH design can improve detection of clinically relevant genomic imbalances and reduce detection of common, apparently benign CNVs as shown for our V5.</p>
<a id="article1.body1.sec3.sec5.p2" name="article1.body1.sec3.sec5.p2"></a><p>We have designed a higher density array CMA V6 which will cover more disease specific regions, enhance the coverage of each chromosomal band and further incorporate interrogation of genomic regions with architectural features causing susceptibility to rearrangements <a href="#pone.0000327-Bignell1">[66]</a>. By doing so, we should increase the detection of rearrangements associated with novel genomic disorders, and perhaps also increase the detection rate among patients with apparently <em>de novo</em> balanced translocations but having unexplained phenotypes as we showed for the V5 to V4 transition.</p>
<a id="article1.body1.sec3.sec5.p3" name="article1.body1.sec3.sec5.p3"></a><p>Much higher density arrays, such as the whole genome tiling path arrays <a href="#pone.0000327-Ishkanian1">[15]</a> or oligonucleotide arrays <a href="#pone.0000327-Baldocchi1">[54]</a>, <a href="#pone.0000327-Bignell1">[66]</a> are likely to be implemented in clinical diagnosis in the near future, but clinical interpretation and counseling will be challenging because of CNVs. A targeted oligoarray will eventually have the potential feasibility in design that enables detection of all clinically relevant genomic imbalances and, in combination with CNV databases, interpretation of benign polymorphic CNVs. Such high resolution genome analysis is a complementary alternative to DNA sequence based genetic testing and other biochemical and cytological evaluations for exploring the genetic etiologies of disease.</p>

</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Material and Methods"></a><h3>Material and Methods</h3><a id="article1.body1.sec4.p1" name="article1.body1.sec4.p1"></a><p>CMA was previously validated on DNA samples from human subjects under an IRB approved protocol. It was introduced in February 2004 as a clinical test coupled with FISH confirmation of potential imbalances ascertained after screening by CMA <a href="#pone.0000327-Cheung1">[24]</a> (<a href="http://www.bcm.edu/cma/">http://www.bcm.edu/cma/</a>).</p>

<h4>Patients' Sample Collection</h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p>Peripheral blood samples from a total of 2513 patients were submitted to the BCM Clinical Cytogenetics Laboratory between February 2004 and March 2006 with a request for CMA. Most cases had either previous or concurrent conventional cytogenetic analysis by GTG-banding or FISH (<a href="#pone-0000327-t001">Table 1</a>). In general, patients referred for CMA analysis had variable clinical phenotypes, most often including: developmental delay and/or mental retardation (DD/MR), dysmorphic features (DF), multiple congenital anomalies (MCA), seizure disorders and autistic or other behavioral abnormalities. Clinical indications at the time of referral are summarized in 5 groups (<a href="#pone-0000327-t006">Table 6</a>). We also noted that for the patients in Group III, the majority (~75%) of the patients were under age 2 at the time being tested, therefore, they were not fully evaluated for DD/MR.</p>
<div class="figure" id="pone-0000327-t006"><div class="img"><a name="pone-0000327-t006" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000327.t006&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000327" data-uri="info:doi/10.1371/journal.pone.0000327.t006"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000327.t006&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000327.t006/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000327.t006/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000327.t006/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000327.t006/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000327.t006.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000327.t006/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000327.t006/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000327.t006.TIF"></span>)
                </a></li></ul></div><p><strong>Table 6.  <span>Summary of patient clinical indications at the time of referral</span></strong></p><span>doi:10.1371/journal.pone.0000327.t006</span></div><a id="article1.body1.sec4.sec1.p2" name="article1.body1.sec4.sec1.p2"></a><p>Overall, 775 samples which were analyzed using CMA V4 and 1738 samples were studied using CMA V5 (<a href="#pone-0000327-t001">Table 1</a>).</p>
<a id="article1.body1.sec4.sec1.p3" name="article1.body1.sec4.sec1.p3"></a><p>There were a total of 117 patients (44 on V4 and 73 on V5) for whom cytogenetic studies were abnormal by GTG-banding analysis (N = 113) and FISH tests (N = 4). In 69 cases the cytogenetic studies were done previously whereas in 48 cases the cytogenetic and CMA studies were performed concurrently (<a href="#pone-0000327-t001">Table 1</a>). Unbalanced chromosomal abnormalities (N = 80) detected included: 18 numeric changes of whole chromosomes; 22 chromosomal deletions and five duplications; nine with additional chromosomal material of unknown origin; 17 with derivative chromosomes and three with marker chromosomes. There are six cases with mosaicism that included two ring chromosomes (90% and 6% mosaicism) and four numeric abnormalities (84% mosaic monosomy chromosome X, 8% mosaic trisomy chromosome X, 2% mosaic trisomy chromosome 14 and 1% mosaic trisomy chromosome 9). Apparently balanced chromosomal abnormalities subjected to further analysis (N = 37) were found to include 12 balanced translocations, 21 inversions, two insertions, and two fra(X) expansions (detected by molecular methods).</p>
<a id="article1.body1.sec4.sec1.p4" name="article1.body1.sec4.sec1.p4"></a><p>CMA studies were conducted on 1872 patients with a normal karyotype by GTG-banding, a subset of whom also had normal subtelomeric and/or locus specific FISH. CMA V4, which contained 366 arranged BAC/PAC clones, was used in 680 cases, in which 462 cases had cytogenetic tests prior to CMA and 218 cases were concurrent with CMA; CMA V5 with 853 interrogating clones was implemented in 1192 cases, there were 855 cases that had previous cytogenetic tests, whereas 337 cases whose cytogenetic tests were performed with CMA concurrently (<a href="#pone-0000327-t001">Table 1</a>).</p>
<a id="article1.body1.sec4.sec1.p5" name="article1.body1.sec4.sec1.p5"></a><p>There were 524 patients studied, 51 performed using CMA V4 and 473 using CMA V5, for whom no information was available regarding a previous or concurrent karyotype and/or FISH analysis. A subset of the cases on V5 was ordered for CMA only.</p>


<h4>Array CGH</h4>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1"></a><p>The CMA array V4 consisted of 366 BAC/PAC clones and V5 consisted of 853 clones (<a href="#pone.0000327.s003">Table S3</a>). Both were designed to contain 3 to 10 BAC/PAC clones per genomic disorder specific locus and subtelomeric regions. The version CMA V5 covered over 70 genetic disorders, all the clinically relevant subtelomeric regions (expanded from 5 Mb to 10 Mb per region), pericentric clones for detecting marker chromosomes, and clones that broadly span chromosomes 13, 18, 21, X, and Y for detection of these most common aneuploidies and their mosaicism. Additional â€œbackboneâ€ interrogating clones along each chromosome were added to aid in the detection of chromosome imbalance outside of targeted regions.</p>
<a id="article1.body1.sec4.sec2.p2" name="article1.body1.sec4.sec2.p2"></a><p>Array technology for printing and hybridization for this study was described before <a href="#pone.0000327-Cheung1">[24]</a> except for the additional use of a Tecan automated hybridization unit for some samples for CMA version 5 with slight modification of the manufacturer's recommended protocol (Tecan HS 4800 hybridization station, Tecan Group Ltd, Switzerland).</p>


<h4>Array Scan and Data Analysis</h4>
<a id="article1.body1.sec4.sec3.p1" name="article1.body1.sec4.sec3.p1"></a><p>Arrays were scanned into 16-bit tiff image files using an Axon two-color microarray scanner (Model 4000B) and quantified using GenePix Pro 6.0 (Molecular Devices Corp, Union City, CA) or Bluefuse (BlueGnome, Cambridge, UK).</p>
<a id="article1.body1.sec4.sec3.p2" name="article1.body1.sec4.sec3.p2"></a><p>Data analysis was performed by a web-based software platform utilizing an analysis scheme in the R open source statistical computing language. The system generates results in the form of raw, normalized and integrated data for all clones analyzed from a single patient sample as described previously <a href="#pone.0000327-Cheung1">[24]</a>. In addition to computing the log<sub>2</sub> Test/Reference intensity ratio value, a single clone T-statistic and permutation based <em>p</em>-value is also computed thereby providing further criterion to determine whether a clone deviates significantly from the mean. In general, abnormal results can be classified as gain or loss. A gain is defined when the combined score of a clone is greater than 0.2 for the log Test/Reference ratios and T-statistics permutation based <em>p</em>-value is &lt;0.05 while a loss is defined by the combined score being less than âˆ’0.2 with T-statistics permutation based p-value &lt;0.05. If a gain or loss of a single or multiple clones are indicated, confirmatory FISH analyses are performed. There are several categories of results that were reported: (1) no abnormalities detected by this version of the microarray; (2) clinically significant abnormalities detected, which were known to be associated with a genetic condition and confirmed by either FISH analysis and/or chromosome analysis and/or partial karyotype; (3) copy number gain or loss of uncertain clinical significance. This latter category might involve either single clone or multiple clone copy number changes that were either (3a) confirmed by FISH analysis or (3b) NOT confirmed by FISH analysis. Subsequent parental FISH and/or CMA studies were requested to determine if the copy number changes represent familial variants or a <em>de novo</em> event and potentially a clinically significant finding.</p>


<h4>FISH Validation</h4>
<a id="article1.body1.sec4.sec4.p1" name="article1.body1.sec4.sec4.p1"></a><p>Metaphase preparations from patient peripheral lymphocytes followed a standard protocol <a href="#pone.0000327-Shaffer3">[46]</a>. BAC or PAC clone DNA probes were labeled using Biotin-14-dUTP, Digoxigenin dUTP, Spectrum Orange/Green according to published protocols. FISH analysis was performed using the standard clinical cytogenetics laboratory protocol.</p>

</div>

<div id="section5" class="section"><a id="s5" name="s5" toc="s5" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0000327"></div><a name="pone.0000327.s001" id="pone.0000327.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000327.s001">Table S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1"></a><p class="preSiDOI">Clinically Relevant Abnormal CMA Cases for Version 4</p>
<p class="siDoi">doi:10.1371/journal.pone.0000327.s001</p><a id="article1.body1.sec5.supplementary-material1.caption1.p2" name="article1.body1.sec5.supplementary-material1.caption1.p2"></a><p class="postSiDOI">(0.14 MB DOC)</p>
<a name="pone.0000327.s002" id="pone.0000327.s002"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000327.s002">Table S2. </a></strong></p><a id="article1.body1.sec5.supplementary-material2.caption1.p1" name="article1.body1.sec5.supplementary-material2.caption1.p1"></a><p class="preSiDOI">Clinically Relevant Abnormal CMA Cases for Version 5</p>
<p class="siDoi">doi:10.1371/journal.pone.0000327.s002</p><a id="article1.body1.sec5.supplementary-material2.caption1.p2" name="article1.body1.sec5.supplementary-material2.caption1.p2"></a><p class="postSiDOI">(0.34 MB DOC)</p>
<a name="pone.0000327.s003" id="pone.0000327.s003"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000327.s003">Table S3. </a></strong></p><a id="article1.body1.sec5.supplementary-material3.caption1.p1" name="article1.body1.sec5.supplementary-material3.caption1.p1"></a><p class="preSiDOI">Genomic Disorder loci coverage on CMA Version 4</p>
<p class="siDoi">doi:10.1371/journal.pone.0000327.s003</p><a id="article1.body1.sec5.supplementary-material3.caption1.p2" name="article1.body1.sec5.supplementary-material3.caption1.p2"></a><p class="postSiDOI">(0.32 MB DOC)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>We thank all the referring physicians, patients and their families for their cooperation. We would like to thank Dr. Lisa White for her critical review of the manuscript. In addition, we also thank all our colleagues and coworkers from the BCM CMA lab, Array Production lab, Cytogenetic/FISH lab and DNA Diagnostic lab for their contributions and support.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: SC. Performed the experiments: JL. Analyzed the data: CS SC XL AP PW. Contributed reagents/materials/analysis tools: PS CS JL MC WW CS TS SY AC ZO. Wrote the paper: XL. Other: Revised the paper: AB PS JL AP CB AC PW. Oversaw the operation and accuracy of the entire study: SC.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0000327-Shaffer1" id="pone.0000327-Shaffer1"></a>Shaffer LG, Lupski JR (2000) Molecular mechanisms for constitutional chromosomal rearrangements in humans. Annu Rev Genet  34: 297â€“329.  <ul class="find" data-citedArticleID="940478"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Molecular+mechanisms+for+constitutional+chromosomal+rearrangements+in+humans.&amp;auth=&amp;atitle=Molecular+mechanisms+for+constitutional+chromosomal+rearrangements+in+humans." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Molecular+mechanisms+for+constitutional+chromosomal+rearrangements+in+humans." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Molecular+mechanisms+for+constitutional+chromosomal+rearrangements+in+humans.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0000327-Trask1" id="pone.0000327-Trask1"></a>Trask BJ (1991) Fluorescence in situ hybridization: applications in cytogenetics and gene mapping. Trends Genet  7: 149â€“154.  <ul class="find" data-citedArticleID="940496" data-doi="10.1016/0168-9525(91)90103-w"><li><a href="http://dx.doi.org/10.1016/0168-9525(91)90103-w" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Fluorescence+in+situ+hybridization%3A+applications+in+cytogenetics+and+gene+mapping." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Fluorescence+in+situ+hybridization%3A+applications+in+cytogenetics+and+gene+mapping.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0000327-Rao1" id="pone.0000327-Rao1"></a>Rao PH, Houldsworth J, Dyomina K, Parsa NZ, Cigudosa JC, et al.  (1998) Chromosomal and gene amplification in diffuse large B-cell lymphoma. Blood  92: 234â€“240.  <ul class="find" data-citedArticleID="940460"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Chromosomal+and+gene+amplification+in+diffuse+large+B-cell+lymphoma.&amp;auth=&amp;atitle=Chromosomal+and+gene+amplification+in+diffuse+large+B-cell+lymphoma." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Chromosomal+and+gene+amplification+in+diffuse+large+B-cell+lymphoma." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Chromosomal+and+gene+amplification+in+diffuse+large+B-cell+lymphoma.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0000327-Speicher1" id="pone.0000327-Speicher1"></a>Speicher MR, Gwyn BS, Ward DC (1996) Karyotyping human chromosomes by combinatorial multi-fluor FISH. Nat Genet  12: 368â€“375.  <ul class="find" data-citedArticleID="940490" data-doi="10.1038/ng0496-368"><li><a href="http://dx.doi.org/10.1038/ng0496-368" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Karyotyping+human+chromosomes+by+combinatorial+multi-fluor+FISH." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Karyotyping+human+chromosomes+by+combinatorial+multi-fluor+FISH.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0000327-Lu1" id="pone.0000327-Lu1"></a>Lu XY, Harris CP, Cooley L, Margolin J, Steuber PC, et al.  (2002) The utility of spectral karyotyping in the cytogenetic analysis of newly diagnosed pediatric acute lymphoblastic leukemia. Leukemia  16: 2222â€“2227.  <ul class="find" data-citedArticleID="940434" data-doi="10.1038/sj.leu.2402662"><li><a href="http://dx.doi.org/10.1038/sj.leu.2402662" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+utility+of+spectral+karyotyping+in+the+cytogenetic+analysis+of+newly+diagnosed+pediatric+acute+lymphoblastic+leukemia." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+utility+of+spectral+karyotyping+in+the+cytogenetic+analysis+of+newly+diagnosed+pediatric+acute+lymphoblastic+leukemia.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0000327-Harris1" id="pone.0000327-Harris1"></a>Harris CP, Lu XY, Narayan G, Singh B, Murty VV, et al.  (2003) Comprehensive molecular cytogenetic characterization of cervical cancer cell lines. Genes Chromosomes Cancer  36: 233â€“241.  <ul class="find" data-citedArticleID="940410" data-doi="10.1002/gcc.10158"><li><a href="http://dx.doi.org/10.1002/gcc.10158" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Comprehensive+molecular+cytogenetic+characterization+of+cervical+cancer+cell+lines." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Comprehensive+molecular+cytogenetic+characterization+of+cervical+cancer+cell+lines.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0000327-Lau1" id="pone.0000327-Lau1"></a>Lau CC, Harris CP, Lu XY, Perlaky L, Gogineni S, et al.  (2004) Frequent amplification and rearrangement of chromosomal bands 6p12â€“p21 and 17p11.2 in osteosarcoma. Genes Chromosomes Cancer  39: 11â€“21.  <ul class="find" data-citedArticleID="940424" data-doi="10.1002/gcc.10291"><li><a href="http://dx.doi.org/10.1002/gcc.10291" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Frequent+amplification+and+rearrangement+of+chromosomal+bands+6p12%26%23x2013%3Bp21+and+17p11.2+in+osteosarcoma." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Frequent+amplification+and+rearrangement+of+chromosomal+bands+6p12%26%23x2013%3Bp21+and+17p11.2+in+osteosarcoma.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0000327-Ravnan1" id="pone.0000327-Ravnan1"></a>Ravnan JB, Tepperberg JH, Papenhausen P, Lamb AN, Hedrick J, et al.  (2006) Subtelomere FISH analysis of 11 688 cases: an evaluation of the frequency and pattern of subtelomere rearrangements in individuals with developmental disabilities. J Med Genet  43: 478â€“489.  <ul class="find" data-citedArticleID="940462" data-doi="10.1136/jmg.2005.036350"><li><a href="http://dx.doi.org/10.1136/jmg.2005.036350" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Subtelomere+FISH+analysis+of+11+688+cases%3A+an+evaluation+of+the+frequency+and+pattern+of+subtelomere+rearrangements+in+individuals+with+developmental+disabilities." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Subtelomere+FISH+analysis+of+11+688+cases%3A+an+evaluation+of+the+frequency+and+pattern+of+subtelomere+rearrangements+in+individuals+with+developmental+disabilities.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0000327-Pinkel1" id="pone.0000327-Pinkel1"></a>Pinkel D, Albertson DG (2005) Comparative genomic hybridization. Annu Rev Genomics Hum Genet  6: 331â€“354.  <ul class="find" data-citedArticleID="940456" data-doi="10.1146/annurev.genom.6.080604.162140"><li><a href="http://dx.doi.org/10.1146/annurev.genom.6.080604.162140" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Comparative+genomic+hybridization." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Comparative+genomic+hybridization.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0000327-Vermeesch1" id="pone.0000327-Vermeesch1"></a>Vermeesch JR, Melotte C, Froyen G, Van VS, Dutta B, et al.  (2005) Molecular karyotyping: array CGH quality criteria for constitutional genetic diagnosis. J Histochem Cytochem  53: 413â€“422.  <ul class="find" data-citedArticleID="940502" data-doi="10.1369/jhc.4a6436.2005"><li><a href="http://dx.doi.org/10.1369/jhc.4a6436.2005" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Molecular+karyotyping%3A+array+CGH+quality+criteria+for+constitutional+genetic+diagnosis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Molecular+karyotyping%3A+array+CGH+quality+criteria+for+constitutional+genetic+diagnosis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0000327-Speicher2" id="pone.0000327-Speicher2"></a>Speicher MR, Carter NP (2005) The new cytogenetics: blurring the boundaries with molecular biology. Nat Rev Genet  6: 782â€“792.  <ul class="find" data-citedArticleID="940492" data-doi="10.1038/nrg1692"><li><a href="http://dx.doi.org/10.1038/nrg1692" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+new+cytogenetics%3A+blurring+the+boundaries+with+molecular+biology." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+new+cytogenetics%3A+blurring+the+boundaries+with+molecular+biology.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0000327-Vissers1" id="pone.0000327-Vissers1"></a>Vissers LE, de Vries BB, Osoegawa K, Janssen IM, Feuth T, et al.  (2003) Array-based comparative genomic hybridization for the genomewide detection of submicroscopic chromosomal abnormalities. Am J Hum Genet  73: 1261â€“1270.  <ul class="find" data-citedArticleID="940504" data-doi="10.1086/379977"><li><a href="http://dx.doi.org/10.1086/379977" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Array-based+comparative+genomic+hybridization+for+the+genomewide+detection+of+submicroscopic+chromosomal+abnormalities." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Array-based+comparative+genomic+hybridization+for+the+genomewide+detection+of+submicroscopic+chromosomal+abnormalities.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0000327-ShawSmith1" id="pone.0000327-ShawSmith1"></a>Shaw-Smith C, Redon R, Rickman L, Rio M, Willatt L, et al.  (2004) Microarray based comparative genomic hybridisation (array-CGH) detects submicroscopic chromosomal deletions and duplications in patients with learning disability/mental retardation and dysmorphic features. J Med Genet  41: 241â€“248.  <ul class="find" data-citedArticleID="940486" data-doi="10.1136/jmg.2003.017731"><li><a href="http://dx.doi.org/10.1136/jmg.2003.017731" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Microarray+based+comparative+genomic+hybridisation+%28array-CGH%29+detects+submicroscopic+chromosomal+deletions+and+duplications+in+patients+with+learning+disability%2Fmental+retardation+and+dysmorphic+features." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Microarray+based+comparative+genomic+hybridisation+%28array-CGH%29+detects+submicroscopic+chromosomal+deletions+and+duplications+in+patients+with+learning+disability%2Fmental+retardation+and+dysmorphic+features.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0000327-Menten1" id="pone.0000327-Menten1"></a>Menten B, Maas N, Thienpont B, Buysse K, Vandesompele J, et al.  (2006) Emerging patterns of cryptic chromosomal imbalance in patients with idiopathic mental retardation and multiple congenital anomalies: a new series of 140 patients and review of published reports. J Med Genet  43: 625â€“633.  <ul class="find" data-citedArticleID="940448" data-doi="10.1136/jmg.2005.039453"><li><a href="http://dx.doi.org/10.1136/jmg.2005.039453" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Emerging+patterns+of+cryptic+chromosomal+imbalance+in+patients+with+idiopathic+mental+retardation+and+multiple+congenital+anomalies%3A+a+new+series+of+140+patients+and+review+of+published+reports." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Emerging+patterns+of+cryptic+chromosomal+imbalance+in+patients+with+idiopathic+mental+retardation+and+multiple+congenital+anomalies%3A+a+new+series+of+140+patients+and+review+of+published+reports.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0000327-Ishkanian1" id="pone.0000327-Ishkanian1"></a>Ishkanian AS, Malloff CA, Watson SK, DeLeeuw RJ, Chi B, et al.  (2004) A tiling resolution DNA microarray with complete coverage of the human genome. Nat Genet  36: 299â€“303.  <ul class="find" data-citedArticleID="940418" data-doi="10.1038/ng1307"><li><a href="http://dx.doi.org/10.1038/ng1307" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+tiling+resolution+DNA+microarray+with+complete+coverage+of+the+human+genome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+tiling+resolution+DNA+microarray+with+complete+coverage+of+the+human+genome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0000327-Yu1" id="pone.0000327-Yu1"></a>Yu W, Ballif BC, Kashork CD, Heilstedt HA, Howard LA, et al.  (2003) Development of a comparative genomic hybridization microarray and demonstration of its utility with 25 well-characterized 1p36 deletions. Hum Mol Genet  12: 2145â€“2152.  <ul class="find" data-citedArticleID="940510" data-doi="10.1093/hmg/ddg230"><li><a href="http://dx.doi.org/10.1093/hmg/ddg230" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Development+of+a+comparative+genomic+hybridization+microarray+and+demonstration+of+its+utility+with+25+well-characterized+1p36+deletions." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Development+of+a+comparative+genomic+hybridization+microarray+and+demonstration+of+its+utility+with+25+well-characterized+1p36+deletions.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0000327-Shaw1" id="pone.0000327-Shaw1"></a>Shaw CJ, Shaw CA, Yu W, Stankiewicz P, White LD, et al.  (2004) Comparative genomic hybridisation using a proximal 17p BAC/PAC array detects rearrangements responsible for four genomic disorders. J Med Genet  41: 113â€“119.  <ul class="find" data-citedArticleID="940484" data-doi="10.1136/jmg.2003.012831"><li><a href="http://dx.doi.org/10.1136/jmg.2003.012831" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Comparative+genomic+hybridisation+using+a+proximal+17p+BAC%2FPAC+array+detects+rearrangements+responsible+for+four+genomic+disorders." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Comparative+genomic+hybridisation+using+a+proximal+17p+BAC%2FPAC+array+detects+rearrangements+responsible+for+four+genomic+disorders.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0000327-Koolen1" id="pone.0000327-Koolen1"></a>Koolen DA, Reardon W, Rosser EM, Lacombe D, Hurst JA, et al.  (2005) Molecular characterisation of patients with subtelomeric 22q abnormalities using chromosome specific array-based comparative genomic hybridisation. Eur J Hum Genet  13: 1019â€“1024.  <ul class="find" data-citedArticleID="940422" data-doi="10.1038/sj.ejhg.5201456"><li><a href="http://dx.doi.org/10.1038/sj.ejhg.5201456" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Molecular+characterisation+of+patients+with+subtelomeric+22q+abnormalities+using+chromosome+specific+array-based+comparative+genomic+hybridisation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Molecular+characterisation+of+patients+with+subtelomeric+22q+abnormalities+using+chromosome+specific+array-based+comparative+genomic+hybridisation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0000327-Vissers2" id="pone.0000327-Vissers2"></a>Vissers LE, Veltman JA, van Kessel AG, Brunner HG (2005) Identification of disease genes by whole genome CGH arrays. Hum Mol Genet  14 Spec No. 2: R215â€“R223.  <ul class="find" data-citedArticleID="940506" data-doi="10.1093/hmg/ddi268"><li><a href="http://dx.doi.org/10.1093/hmg/ddi268" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Identification+of+disease+genes+by+whole+genome+CGH+arrays." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Identification+of+disease+genes+by+whole+genome+CGH+arrays.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0000327-Dhami1" id="pone.0000327-Dhami1"></a>Dhami P, Coffey AJ, Abbs S, Vermeesch JR, Dumanski JP, et al.  (2005) Exon array CGH: detection of copy-number changes at the resolution of individual exons in the human genome. Am J Hum Genet  76: 750â€“762.  <ul class="find" data-citedArticleID="940398" data-doi="10.1086/429588"><li><a href="http://dx.doi.org/10.1086/429588" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Exon+array+CGH%3A+detection+of+copy-number+changes+at+the+resolution+of+individual+exons+in+the+human+genome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Exon+array+CGH%3A+detection+of+copy-number+changes+at+the+resolution+of+individual+exons+in+the+human+genome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0000327-Misawa1" id="pone.0000327-Misawa1"></a>Misawa A, Inoue J, Sugino Y, Hosoi H, Sugimoto T, et al.  (2005) Methylation-associated silencing of the nuclear receptor 1I2 gene in advanced-type neuroblastomas, identified by bacterial artificial chromosome array-based methylated CpG island amplification. Cancer Res  65: 10233â€“10242.  <ul class="find" data-citedArticleID="940450" data-doi="10.1158/0008-5472.can-05-1073"><li><a href="http://dx.doi.org/10.1158/0008-5472.can-05-1073" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Methylation-associated+silencing+of+the+nuclear+receptor+1I2+gene+in+advanced-type+neuroblastomas%2C+identified+by+bacterial+artificial+chromosome+array-based+methylated+CpG+island+amplification." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Methylation-associated+silencing+of+the+nuclear+receptor+1I2+gene+in+advanced-type+neuroblastomas%2C+identified+by+bacterial+artificial+chromosome+array-based+methylated+CpG+island+amplification.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0000327-Bejjani1" id="pone.0000327-Bejjani1"></a>Bejjani BA, Saleki R, Ballif BC, Rorem EA, Sundin K, et al.  (2005) Use of targeted array-based CGH for the clinical diagnosis of chromosomal imbalance: is less more? Am J Med Genet A  134: 259â€“267.  <ul class="find" data-citedArticleID="940386" data-doi="10.1002/ajmg.a.30621"><li><a href="http://dx.doi.org/10.1002/ajmg.a.30621" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Use+of+targeted+array-based+CGH+for+the+clinical+diagnosis+of+chromosomal+imbalance%3A+is+less+more%3F" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Use+of+targeted+array-based+CGH+for+the+clinical+diagnosis+of+chromosomal+imbalance%3A+is+less+more%3F%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0000327-deVries1" id="pone.0000327-deVries1"></a>de Vries BB, Pfundt R, Leisink M, Koolen DA, Vissers LE, et al.  (2005) Diagnostic genome profiling in mental retardation. Am J Hum Genet  77: 606â€“616.  <ul class="find" data-citedArticleID="940512" data-doi="10.1086/491719"><li><a href="http://dx.doi.org/10.1086/491719" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Diagnostic+genome+profiling+in+mental+retardation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Diagnostic+genome+profiling+in+mental+retardation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0000327-Cheung1" id="pone.0000327-Cheung1"></a>Cheung SW, Shaw CA, Yu W, Li J, Ou Z, et al.  (2005) Development and validation of a CGH microarray for clinical cytogenetic diagnosis. Genet Med  7: 422â€“432.  <ul class="find" data-citedArticleID="940392" data-doi="10.1097/01.gim.0000170992.63691.32"><li><a href="http://dx.doi.org/10.1097/01.gim.0000170992.63691.32" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Development+and+validation+of+a+CGH+microarray+for+clinical+cytogenetic+diagnosis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Development+and+validation+of+a+CGH+microarray+for+clinical+cytogenetic+diagnosis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0000327-Shaffer2" id="pone.0000327-Shaffer2"></a>Shaffer LG, Kashork CD, Saleki R, Rorem E, Sundin K, et al.  (2006) Targeted genomic microarray analysis for identification of chromosome abnormalities in 1500 consecutive clinical cases. J Pediatr  149: 98â€“102.  <ul class="find" data-citedArticleID="940480" data-doi="10.1016/j.jpeds.2006.02.006"><li><a href="http://dx.doi.org/10.1016/j.jpeds.2006.02.006" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Targeted+genomic+microarray+analysis+for+identification+of+chromosome+abnormalities+in+1500+consecutive+clinical+cases." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Targeted+genomic+microarray+analysis+for+identification+of+chromosome+abnormalities+in+1500+consecutive+clinical+cases.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0000327-Lee1" id="pone.0000327-Lee1"></a>Lee JA, Cheung SW, Ward PA, Inoue K, Lupski JR (2005) Prenatal diagnosis of PLP1 copy number by array comparative genomic hybridization. Prenat Diagn  25: 1188â€“1191.  <ul class="find" data-citedArticleID="940430" data-doi="10.1002/pd.1308"><li><a href="http://dx.doi.org/10.1002/pd.1308" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Prenatal+diagnosis+of+PLP1+copy+number+by+array+comparative+genomic+hybridization." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Prenatal+diagnosis+of+PLP1+copy+number+by+array+comparative+genomic+hybridization.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0000327-Rickman1" id="pone.0000327-Rickman1"></a>Rickman L, Fiegler H, Carter NP, Bobrow M (2005) Prenatal diagnosis by array-CGH. Eur J Med Genet  48: 232â€“240.  <ul class="find" data-citedArticleID="940466" data-doi="10.1016/j.ejmg.2005.03.003"><li><a href="http://dx.doi.org/10.1016/j.ejmg.2005.03.003" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Prenatal+diagnosis+by+array-CGH." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Prenatal+diagnosis+by+array-CGH.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0000327-Rickman2" id="pone.0000327-Rickman2"></a>Rickman L, Fiegler H, Shaw-Smith C, Nash R, Cirigliano V, et al.  (2006) Prenatal detection of unbalanced chromosomal rearrangements by array CGH. J Med Genet  43: 353â€“361.  <ul class="find" data-citedArticleID="940468" data-doi="10.1136/jmg.2005.037648"><li><a href="http://dx.doi.org/10.1136/jmg.2005.037648" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Prenatal+detection+of+unbalanced+chromosomal+rearrangements+by+array+CGH." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Prenatal+detection+of+unbalanced+chromosomal+rearrangements+by+array+CGH.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0000327-Peng1" id="pone.0000327-Peng1"></a>Peng HH, Wang CJ, Wang TH, Chang SD (2006) Prenatal diagnosis of de novo interstitial 2q14.2-2q21.3 deletion assisted by array-based comparative genomic hybridization: a case report. J Reprod Med  51: 438â€“442.  <ul class="find" data-citedArticleID="940454"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Prenatal+diagnosis+of+de+novo+interstitial+2q14.2-2q21.3+deletion+assisted+by+array-based+comparative+genomic+hybridization%3A+a+case+report.&amp;auth=&amp;atitle=Prenatal+diagnosis+of+de+novo+interstitial+2q14.2-2q21.3+deletion+assisted+by+array-based+comparative+genomic+hybridization%3A+a+case+report." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Prenatal+diagnosis+of+de+novo+interstitial+2q14.2-2q21.3+deletion+assisted+by+array-based+comparative+genomic+hybridization%3A+a+case+report." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Prenatal+diagnosis+of+de+novo+interstitial+2q14.2-2q21.3+deletion+assisted+by+array-based+comparative+genomic+hybridization%3A+a+case+report.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0000327-Miura1" id="pone.0000327-Miura1"></a>Miura S, Miura K, Masuzaki H, Miyake N, Yoshiura K, et al.  (2006) Microarray comparative genomic hybridization (CGH)-based prenatal diagnosis for chromosome abnormalities using cell-free fetal DNA in amniotic fluid. J Hum Genet  51: 412â€“417.  <ul class="find" data-citedArticleID="940452" data-doi="10.1007/s10038-006-0376-7"><li><a href="http://dx.doi.org/10.1007/s10038-006-0376-7" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Microarray+comparative+genomic+hybridization+%28CGH%29-based+prenatal+diagnosis+for+chromosome+abnormalities+using+cell-free+fetal+DNA+in+amniotic+fluid." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Microarray+comparative+genomic+hybridization+%28CGH%29-based+prenatal+diagnosis+for+chromosome+abnormalities+using+cell-free+fetal+DNA+in+amniotic+fluid.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0000327-Sahoo1" id="pone.0000327-Sahoo1"></a>Sahoo T, Cheung SW, Ward P, Darilek S, Patel A, et al.  (2006) Prenatal diagnosis of chromosomal abnormalities using array-based comparative genomic hybridization. Genet Med  8: 719â€“727.  <ul class="find" data-citedArticleID="940472" data-doi="10.1097/01.gim.0000245576.47154.63"><li><a href="http://dx.doi.org/10.1097/01.gim.0000245576.47154.63" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Prenatal+diagnosis+of+chromosomal+abnormalities+using+array-based+comparative+genomic+hybridization." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Prenatal+diagnosis+of+chromosomal+abnormalities+using+array-based+comparative+genomic+hybridization.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0000327-Lupski1" id="pone.0000327-Lupski1"></a> Lupski JR, Stankiewicz P, editors.  (2006) Genomic Disorders-The Genomic Basis of Disease. Totowa, NJ: Human Press.   <ul class="find-nolinks"></ul></li><li><span class="label">33.
              </span><a name="pone.0000327-Tyson1" id="pone.0000327-Tyson1"></a>Tyson C, Harvard C, Locker R, Friedman JM, Langlois S, et al.  (2005) Submicroscopic deletions and duplications in individuals with intellectual disability detected by array-CGH. Am J Med Genet A  139: 173â€“185.  <ul class="find" data-citedArticleID="940498" data-doi="10.1002/ajmg.a.31015"><li><a href="http://dx.doi.org/10.1002/ajmg.a.31015" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Submicroscopic+deletions+and+duplications+in+individuals+with+intellectual+disability+detected+by+array-CGH." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Submicroscopic+deletions+and+duplications+in+individuals+with+intellectual+disability+detected+by+array-CGH.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0000327-Cheung2" id="pone.0000327-Cheung2"></a>Cheung SW, Shaw C, Scott DA, Patel A, Sahoo T, et al.  (2007) Microarray-based CGH detects chromosomal mosaicism not revealed by conventional cytogenetics. Am J Med Genet (in press).   <ul class="find" data-citedArticleID="940394" data-doi="10.1002/ajmg.a.31740"><li><a href="http://dx.doi.org/10.1002/ajmg.a.31740" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Microarray-based+CGH+detects+chromosomal+mosaicism+not+revealed+by+conventional+cytogenetics." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Microarray-based+CGH+detects+chromosomal+mosaicism+not+revealed+by+conventional+cytogenetics.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0000327-Lupski2" id="pone.0000327-Lupski2"></a>Lupski JR (2006) Genome structural variation and sporadic disease traits. Nat Genet  38: 974â€“976.  <ul class="find" data-citedArticleID="940438" data-doi="10.1038/ng0906-974"><li><a href="http://dx.doi.org/10.1038/ng0906-974" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genome+structural+variation+and+sporadic+disease+traits." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genome+structural+variation+and+sporadic+disease+traits.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">36.
              </span><a name="pone.0000327-delGaudio1" id="pone.0000327-delGaudio1"></a>del Gaudio D, Fang P, Scaglia F, Ward PA, Craigen WJ, et al.  (2006) Increased MECP2 gene copy number as the result of genomic duplication in neurodevelopmentally delayed males. Genet Med  8: 784â€“792.  <ul class="find" data-citedArticleID="940514" data-doi="10.1097/01.gim.0000250502.28516.3c"><li><a href="http://dx.doi.org/10.1097/01.gim.0000250502.28516.3c" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Increased+MECP2+gene+copy+number+as+the+result+of+genomic+duplication+in+neurodevelopmentally+delayed+males." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Increased+MECP2+gene+copy+number+as+the+result+of+genomic+duplication+in+neurodevelopmentally+delayed+males.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">37.
              </span><a name="pone.0000327-Redon1" id="pone.0000327-Redon1"></a>Redon R, Ishikawa S, Fitch KR, Feuk L, Perry GH, et al.  (2006) Global variation in copy number in the human genome. Nature  444: 444â€“454.  <ul class="find" data-citedArticleID="940464" data-doi="10.1038/nature05329"><li><a href="http://dx.doi.org/10.1038/nature05329" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Global+variation+in+copy+number+in+the+human+genome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Global+variation+in+copy+number+in+the+human+genome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">38.
              </span><a name="pone.0000327-Iafrate1" id="pone.0000327-Iafrate1"></a>Iafrate AJ, Feuk L, Rivera MN, Listewnik ML, Donahoe PK, et al.  (2004) Detection of large-scale variation in the human genome. Nat Genet  36: 949â€“951.  <ul class="find" data-citedArticleID="940414" data-doi="10.1038/ng1416"><li><a href="http://dx.doi.org/10.1038/ng1416" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Detection+of+large-scale+variation+in+the+human+genome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Detection+of+large-scale+variation+in+the+human+genome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">39.
              </span><a name="pone.0000327-Lennon1" id="pone.0000327-Lennon1"></a>Lennon PA, Cooper ML, Peiffer D, Gunderson K, Patel A, et al.  (2007) Deletion of 7q31.1 supports involvement of <em>FOXP2</em> in language impairment. Am J Med Gene (in press).   <ul class="find" data-citedArticleID="940432" data-doi="10.1002/ajmg.a.31632"><li><a href="http://dx.doi.org/10.1002/ajmg.a.31632" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Deletion+of+7q31.1+supports+involvement+of+FOXP2+in+language+impairment." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Deletion+of+7q31.1+supports+involvement+of+FOXP2+in+language+impairment.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">40.
              </span><a name="pone.0000327-Biesecker1" id="pone.0000327-Biesecker1"></a>Biesecker LG (2002) The end of the beginning of chromosome ends. Am J Med Genet  107: 263â€“266.  <ul class="find" data-citedArticleID="940388" data-doi="10.1002/ajmg.10160"><li><a href="http://dx.doi.org/10.1002/ajmg.10160" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+end+of+the+beginning+of+chromosome+ends." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+end+of+the+beginning+of+chromosome+ends.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">41.
              </span><a name="pone.0000327-Hwang1" id="pone.0000327-Hwang1"></a>Hwang KS, Pearson MA, Stankiewicz P, Lennon PA, Cooper ML, et al.  (2005) Cryptic unbalanced translocation t(17;18)(p13.2;q22.3) identified by subtelomeric FISH and defined by array-based comparative genomic hybridization in a patient with mental retardation and dysmorphic features. Am J Med Genet A  137: 88â€“93.  <ul class="find" data-citedArticleID="940412" data-doi="10.1002/ajmg.a.30858"><li><a href="http://dx.doi.org/10.1002/ajmg.a.30858" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Cryptic+unbalanced+translocation+t%2817%3B18%29%28p13.2%3Bq22.3%29+identified+by+subtelomeric+FISH+and+defined+by+array-based+comparative+genomic+hybridization+in+a+patient+with+mental+retardation+and+dysmorphic+features." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Cryptic+unbalanced+translocation+t%2817%3B18%29%28p13.2%3Bq22.3%29+identified+by+subtelomeric+FISH+and+defined+by+array-based+comparative+genomic+hybridization+in+a+patient+with+mental+retardation+and+dysmorphic+features.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">42.
              </span><a name="pone.0000327-Le1" id="pone.0000327-Le1"></a>Le CC, De MP, Vincent MC, Boceno M, Bourrouillou G, et al.  (2005) Subtelomeric 6p deletion: clinical, FISH, and array CGH characterization of two cases. Am J Med Genet A  132: 175â€“180.  <ul class="find" data-citedArticleID="940428" data-doi="10.1002/ajmg.a.30409"><li><a href="http://dx.doi.org/10.1002/ajmg.a.30409" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Subtelomeric+6p+deletion%3A+clinical%2C+FISH%2C+and+array+CGH+characterization+of+two+cases." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Subtelomeric+6p+deletion%3A+clinical%2C+FISH%2C+and+array+CGH+characterization+of+two+cases.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">43.
              </span><a name="pone.0000327-Lupski3" id="pone.0000327-Lupski3"></a>Lupski JR, de Oca-Luna RM, Slaugenhaupt S, Pentao L, Guzzetta V, et al.  (1991) DNA duplication associated with Charcot-Marie-Tooth disease type 1A. Cell  66: 219â€“232.  <ul class="find" data-citedArticleID="940440" data-doi="10.1016/0092-8674(91)90613-4"><li><a href="http://dx.doi.org/10.1016/0092-8674(91)90613-4" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=DNA+duplication+associated+with+Charcot-Marie-Tooth+disease+type+1A." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22DNA+duplication+associated+with+Charcot-Marie-Tooth+disease+type+1A.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">44.
              </span><a name="pone.0000327-Lupski4" id="pone.0000327-Lupski4"></a>Lupski JR (1998) Charcot-Marie-Tooth disease: lessons in genetic mechanisms. Mol Med  4: 3â€“11.  <ul class="find" data-citedArticleID="940442"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Charcot-Marie-Tooth+disease%3A+lessons+in+genetic+mechanisms.&amp;auth=&amp;atitle=Charcot-Marie-Tooth+disease%3A+lessons+in+genetic+mechanisms." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Charcot-Marie-Tooth+disease%3A+lessons+in+genetic+mechanisms." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Charcot-Marie-Tooth+disease%3A+lessons+in+genetic+mechanisms.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">45.
              </span><a name="pone.0000327-Stankiewicz1" id="pone.0000327-Stankiewicz1"></a>Stankiewicz P, Lupski JR (2002) Genome architecture, rearrangements and genomic disorders. Trends Genet  18: 74â€“82.  <ul class="find" data-citedArticleID="940494" data-doi="10.1016/s0168-9525(02)02592-1"><li><a href="http://dx.doi.org/10.1016/s0168-9525(02)02592-1" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genome+architecture%2C+rearrangements+and+genomic+disorders." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genome+architecture%2C+rearrangements+and+genomic+disorders.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">46.
              </span><a name="pone.0000327-Shaffer3" id="pone.0000327-Shaffer3"></a>Shaffer LG, Kennedy GM, Spikes AS, Lupski JR (1997) Diagnosis of CMT1A duplications and HNPP deletions by interphase FISH: implications for testing in the cytogenetics laboratory. Am J Med Genet  69: 325â€“331.  <ul class="find" data-citedArticleID="940482" data-doi="10.1002/(sici)1096-8628(19970331)69:3<325::aid-ajmg20&gt;3.3.co;2-w"><li><a href="http://dx.doi.org/10.1002/(sici)1096-8628(19970331)69:3<325::aid-ajmg20&gt;3.3.co;2-w" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Diagnosis+of+CMT1A+duplications+and+HNPP+deletions+by+interphase+FISH%3A+implications+for+testing+in+the+cytogenetics+laboratory." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Diagnosis+of+CMT1A+duplications+and+HNPP+deletions+by+interphase+FISH%3A+implications+for+testing+in+the+cytogenetics+laboratory.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">47.
              </span><a name="pone.0000327-Potocki1" id="pone.0000327-Potocki1"></a>Potocki L, Bi W, Treadwell-Deering D, Carvalho cMB, Eifert Anna, et al.  (2007) Characterization of the Potocki-Lupski syndrome [dup(17)(p11.2p11.2)] and delineation of a dosage-sensitive critical interval that can convey an autism phenotype. Am J Hum Genet (in press).   <ul class="find" data-citedArticleID="940458" data-doi="10.1086/512864"><li><a href="http://dx.doi.org/10.1086/512864" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Characterization+of+the+Potocki-Lupski+syndrome+%5Bdup%2817%29%28p11.2p11.2%29%5D+and+delineation+of+a+dosage-sensitive+critical+interval+that+can+convey+an+autism+phenotype." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Characterization+of+the+Potocki-Lupski+syndrome+%5Bdup%2817%29%28p11.2p11.2%29%5D+and+delineation+of+a+dosage-sensitive+critical+interval+that+can+convey+an+autism+phenotype.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">48.
              </span><a name="pone.0000327-Inoue1" id="pone.0000327-Inoue1"></a>Inoue K, Lupski JR (2002) Molecular mechanisms for genomic disorders. Annu Rev Genomics Hum Genet  3: 199â€“242.  <ul class="find" data-citedArticleID="940416"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Molecular+mechanisms+for+genomic+disorders.&amp;auth=&amp;atitle=Molecular+mechanisms+for+genomic+disorders." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Molecular+mechanisms+for+genomic+disorders." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Molecular+mechanisms+for+genomic+disorders.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">49.
              </span><a name="pone.0000327-Ciccone1" id="pone.0000327-Ciccone1"></a>Ciccone R, Giorda R, Gregato G, Guerrini R, Giglio S, et al.  (2005) Reciprocal translocations: a trap for cytogenetists? Hum Genet  117: 571â€“582.  <ul class="find" data-citedArticleID="940396" data-doi="10.1007/s00439-005-1324-x"><li><a href="http://dx.doi.org/10.1007/s00439-005-1324-x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Reciprocal+translocations%3A+a+trap+for+cytogenetists%3F" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Reciprocal+translocations%3A+a+trap+for+cytogenetists%3F%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">50.
              </span><a name="pone.0000327-Rosenberg1" id="pone.0000327-Rosenberg1"></a>Rosenberg C, Knijnenburg J, Chauffaille ML, Brunoni D, Catelani AL, et al.  (2005) Array CGH detection of a cryptic deletion in a complex chromosome rearrangement. Hum Genet  116: 390â€“394.  <ul class="find" data-citedArticleID="940470" data-doi="10.1007/s00439-004-1248-x"><li><a href="http://dx.doi.org/10.1007/s00439-004-1248-x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Array+CGH+detection+of+a+cryptic+deletion+in+a+complex+chromosome+rearrangement." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Array+CGH+detection+of+a+cryptic+deletion+in+a+complex+chromosome+rearrangement.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">51.
              </span><a name="pone.0000327-Gribble1" id="pone.0000327-Gribble1"></a>Gribble SM, Prigmore E, Burford DC, Porter KM, Ng BL, et al.  (2005) The complex nature of constitutional de novo apparently balanced translocations in patients presenting with abnormal phenotypes. J Med Genet  42: 8â€“16.  <ul class="find" data-citedArticleID="940406" data-doi="10.1136/jmg.2004.024141"><li><a href="http://dx.doi.org/10.1136/jmg.2004.024141" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+complex+nature+of+constitutional+de+novo+apparently+balanced+translocations+in+patients+presenting+with+abnormal+phenotypes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+complex+nature+of+constitutional+de+novo+apparently+balanced+translocations+in+patients+presenting+with+abnormal+phenotypes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">52.
              </span><a name="pone.0000327-Gribble2" id="pone.0000327-Gribble2"></a>Gribble SM, Fiegler H, Burford DC, Prigmore E, Yang F, et al.  (2004) Applications of combined DNA microarray and chromosome sorting technologies. Chromosome Res  12: 35â€“43.  <ul class="find" data-citedArticleID="940408" data-doi="10.1023/b:chro.0000009325.69828.83"><li><a href="http://dx.doi.org/10.1023/b:chro.0000009325.69828.83" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Applications+of+combined+DNA+microarray+and+chromosome+sorting+technologies." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Applications+of+combined+DNA+microarray+and+chromosome+sorting+technologies.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">53.
              </span><a name="pone.0000327-Fauth1" id="pone.0000327-Fauth1"></a>Fauth C, Gribble SM, Porter KM, Codina-Pascual M, Ng BL, et al.  (2006) Micro-array analyses decipher exceptional complex familial chromosomal rearrangement. Hum Genet  119: 145â€“153.  <ul class="find" data-citedArticleID="940400" data-doi="10.1007/s00439-005-0103-z"><li><a href="http://dx.doi.org/10.1007/s00439-005-0103-z" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Micro-array+analyses+decipher+exceptional+complex+familial+chromosomal+rearrangement." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Micro-array+analyses+decipher+exceptional+complex+familial+chromosomal+rearrangement.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">54.
              </span><a name="pone.0000327-Baldocchi1" id="pone.0000327-Baldocchi1"></a>Baldocchi RA, Glynne RJ, Chin K, Kowbel D, Collins C, et al.  (2005) Design considerations for array CGH to oligonucleotide arrays. Cytometry A  67: 129â€“136.  <ul class="find" data-citedArticleID="940384" data-doi="10.1002/cyto.a.20161"><li><a href="http://dx.doi.org/10.1002/cyto.a.20161" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Design+considerations+for+array+CGH+to+oligonucleotide+arrays." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Design+considerations+for+array+CGH+to+oligonucleotide+arrays.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">55.
              </span><a name="pone.0000327-Schulz1" id="pone.0000327-Schulz1"></a>Schulz R, Menheniott TR, Woodfine K, Wood AJ, Choi JD, et al.  (2006) Chromosome-wide identification of novel imprinted genes using microarrays and uniparental disomies. Nucleic Acids Res  34: e88.  <ul class="find" data-citedArticleID="940474" data-doi="10.1093/nar/gkl461"><li><a href="http://dx.doi.org/10.1093/nar/gkl461" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Chromosome-wide+identification+of+novel+imprinted+genes+using+microarrays+and+uniparental+disomies." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Chromosome-wide+identification+of+novel+imprinted+genes+using+microarrays+and+uniparental+disomies.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">56.
              </span><a name="pone.0000327-Sebat1" id="pone.0000327-Sebat1"></a>Sebat J, Lakshmi B, Troge J, Alexander J, Young J, et al.  (2004) Large-scale copy number polymorphism in the human genome. Science  305: 525â€“528.  <ul class="find" data-citedArticleID="940476" data-doi="10.1126/science.1098918"><li><a href="http://dx.doi.org/10.1126/science.1098918" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Large-scale+copy+number+polymorphism+in+the+human+genome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Large-scale+copy+number+polymorphism+in+the+human+genome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">57.
              </span><a name="pone.0000327-Friedman1" id="pone.0000327-Friedman1"></a>Friedman JM, Baross A, Delaney AD, Ally A, Arbour L, et al.  (2006) Oligonucleotide microarray analysis of genomic imbalance in children with mental retardation. Am J Hum Genet  79: 500â€“513.  <ul class="find" data-citedArticleID="940402" data-doi="10.1086/507471"><li><a href="http://dx.doi.org/10.1086/507471" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Oligonucleotide+microarray+analysis+of+genomic+imbalance+in+children+with+mental+retardation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Oligonucleotide+microarray+analysis+of+genomic+imbalance+in+children+with+mental+retardation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">58.
              </span><a name="pone.0000327-Goidts1" id="pone.0000327-Goidts1"></a>Goidts V, Armengol L, Schempp W, Conroy J, Nowak N, et al.  (2006) Identification of large-scale human-specific copy number differences by inter-species array comparative genomic hybridization. Hum Genet  119: 185â€“198.  <ul class="find" data-citedArticleID="940404" data-doi="10.1007/s00439-005-0130-9"><li><a href="http://dx.doi.org/10.1007/s00439-005-0130-9" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Identification+of+large-scale+human-specific+copy+number+differences+by+inter-species+array+comparative+genomic+hybridization." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Identification+of+large-scale+human-specific+copy+number+differences+by+inter-species+array+comparative+genomic+hybridization.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">59.
              </span><a name="pone.0000327-Wilson1" id="pone.0000327-Wilson1"></a>Wilson GM, Flibotte S, Missirlis PI, Marra MA, Jones S, et al.  (2006) Identification by full-coverage array CGH of human DNA copy number increases relative to chimpanzee and gorilla. Genome Res  16: 173â€“181.  <ul class="find" data-citedArticleID="940508" data-doi="10.1101/gr.4456006"><li><a href="http://dx.doi.org/10.1101/gr.4456006" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Identification+by+full-coverage+array+CGH+of+human+DNA+copy+number+increases+relative+to+chimpanzee+and+gorilla." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Identification+by+full-coverage+array+CGH+of+human+DNA+copy+number+increases+relative+to+chimpanzee+and+gorilla.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">60.
              </span><a name="pone.0000327-Lupski5" id="pone.0000327-Lupski5"></a>Lupski JR, Stankiewicz P (2005) Genomic disorders: molecular mechanisms for rearrangements and conveyed phenotypes. PLoS Genet  1: e49.  <ul class="find" data-citedArticleID="940444" data-doi="10.1371/journal.pgen.0010049"><li><a href="http://dx.doi.org/10.1371/journal.pgen.0010049" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genomic+disorders%3A+molecular+mechanisms+for+rearrangements+and+conveyed+phenotypes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genomic+disorders%3A+molecular+mechanisms+for+rearrangements+and+conveyed+phenotypes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">61.
              </span><a name="pone.0000327-Lupski6" id="pone.0000327-Lupski6"></a>Lupski JR (2007) Structural Variation in the Human Genome. N Eng J Med (submitted).   <ul class="find" data-citedArticleID="940446" data-doi="10.1056/nejmcibr067658"><li><a href="http://dx.doi.org/10.1056/nejmcibr067658" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Structural+Variation+in+the+Human+Genome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Structural+Variation+in+the+Human+Genome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">62.
              </span><a name="pone.0000327-Veltman1" id="pone.0000327-Veltman1"></a>Veltman JA, Schoenmakers EF, Eussen BH, Janssen I, Merkx G, et al.  (2002) High-throughput analysis of subtelomeric chromosome rearrangements by use of array-based comparative genomic hybridization. Am J Hum Genet  70: 1269â€“1276.  <ul class="find" data-citedArticleID="940500" data-doi="10.1086/340426"><li><a href="http://dx.doi.org/10.1086/340426" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=High-throughput+analysis+of+subtelomeric+chromosome+rearrangements+by+use+of+array-based+comparative+genomic+hybridization." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22High-throughput+analysis+of+subtelomeric+chromosome+rearrangements+by+use+of+array-based+comparative+genomic+hybridization.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">63.
              </span><a name="pone.0000327-Lavoie1" id="pone.0000327-Lavoie1"></a>Lavoie BD, Hogan E, Koshland D (2002) In vivo dissection of the chromosome condensation machinery: reversibility of condensation distinguishes contributions of condensin and cohesin. J Cell Biol  156: 805â€“815.  <ul class="find" data-citedArticleID="940426" data-doi="10.1083/jcb.200109056"><li><a href="http://dx.doi.org/10.1083/jcb.200109056" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=In+vivo+dissection+of+the+chromosome+condensation+machinery%3A+reversibility+of+condensation+distinguishes+contributions+of+condensin+and+cohesin." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22In+vivo+dissection+of+the+chromosome+condensation+machinery%3A+reversibility+of+condensation+distinguishes+contributions+of+condensin+and+cohesin.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">64.
              </span><a name="pone.0000327-Kireeva1" id="pone.0000327-Kireeva1"></a>Kireeva N, Lakonishok M, Kireev I, Hirano T, Belmont AS (2004) Visualization of early chromosome condensation: a hierarchical folding, axial glue model of chromosome structure. J Cell Biol  166: 775â€“785.  <ul class="find" data-citedArticleID="940420" data-doi="10.1083/jcb.200406049"><li><a href="http://dx.doi.org/10.1083/jcb.200406049" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Visualization+of+early+chromosome+condensation%3A+a+hierarchical+folding%2C+axial+glue+model+of+chromosome+structure." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Visualization+of+early+chromosome+condensation%3A+a+hierarchical+folding%2C+axial+glue+model+of+chromosome+structure.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">65.
              </span><a name="pone.0000327-Shopland1" id="pone.0000327-Shopland1"></a>Shopland LS, Lynch CR, Peterson KA, Thornton K, Kepper N, et al.  (2006) Folding and organization of a contiguous chromosome region according to the gene distribution pattern in primary genomic sequence. J Cell Biol  174: 27â€“38.  <ul class="find" data-citedArticleID="940488" data-doi="10.1083/jcb.200603083"><li><a href="http://dx.doi.org/10.1083/jcb.200603083" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Folding+and+organization+of+a+contiguous+chromosome+region+according+to+the+gene+distribution+pattern+in+primary+genomic+sequence." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Folding+and+organization+of+a+contiguous+chromosome+region+according+to+the+gene+distribution+pattern+in+primary+genomic+sequence.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">66.
              </span><a name="pone.0000327-Bignell1" id="pone.0000327-Bignell1"></a>Bignell GR, Huang J, Greshock J, Watt S, Butler A, et al.  (2004) High-resolution analysis of DNA copy number using oligonucleotide microarrays. Genome Res  14: 287â€“295.  <ul class="find" data-citedArticleID="940390" data-doi="10.1101/gr.2012304"><li><a href="http://dx.doi.org/10.1101/gr.2012304" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=High-resolution+analysis+of+DNA+copy+number+using+oligonucleotide+microarrays." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22High-resolution+analysis+of+DNA+copy+number+using+oligonucleotide+microarrays.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000327.XML" value="143791"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000327.PDF" value="266994"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000327.t001.PNG_L" value="123569"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000327.t001.PNG_M" value="42591"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000327.t001.PNG_S" value="2719"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000327.t001.TIF" value="593588"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000327.t001.PNG_I" value="18131"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000327.t002.PNG_L" value="74965"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000327.t002.PNG_M" value="73934"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000327.t002.PNG_S" value="3274"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000327.t002.TIF" value="370856"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000327.t002.PNG_I" value="33470"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000327.g001.PNG_L" value="923973"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000327.g001.PNG_M" value="150264"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000327.g001.PNG_S" value="5053"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000327.g001.TIF" value="1696944"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000327.g001.PNG_I" value="55488"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000327.t003.PNG_L" value="126683"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000327.t003.PNG_M" value="46130"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000327.t003.PNG_S" value="2822"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000327.t003.TIF" value="666008"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000327.t003.PNG_I" value="20108"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000327.g002.PNG_L" value="70827"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000327.g002.PNG_M" value="119437"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000327.g002.PNG_S" value="7502"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000327.g002.TIF" value="153438"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000327.g002.PNG_I" value="87844"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000327.t004.PNG_L" value="39009"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000327.t004.PNG_M" value="39398"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000327.t004.PNG_S" value="2406"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000327.t004.TIF" value="207996"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000327.t004.PNG_I" value="17546"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000327.t005.PNG_L" value="72052"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000327.t005.PNG_M" value="72563"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000327.t005.PNG_S" value="3944"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000327.t005.TIF" value="349194"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000327.t005.PNG_I" value="31884"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000327.t006.PNG_L" value="65845"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000327.t006.PNG_M" value="63530"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000327.t006.PNG_S" value="2993"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000327.t006.TIF" value="327654"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000327.t006.PNG_I" value="29675"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000327.s001.DOC" value="138240"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000327.s002.DOC" value="344064"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000327.s003.DOC" value="320000"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000327&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000327" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000327&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0000327&volume=&issue=&title=Clinical Implementation of Chromosomal Microarray Analysis: Summary of 2513 Postnatal Cases&author_name=Xinyan%20Lu%2C%20Chad%20A.%20Shaw%2C%20Ankita%20Patel%2C%20Jiangzhen%20Li%2C%20M.%20Lance%20Cooper%2C%20William%20R.%20Wells%2C%20Cathy%20M.%20Sullivan%2C%20Trilochan%20Sahoo%2C%20Svetlana%20A.%20Yatsenko%2C%20Carlos%20A.%20Bacino%2C%20Pawel%20Stankiewicz%2C%20Zhishu%20Ou%2C%20A.%20Craig%20Chinault%2C%20Arthur%20L.%20Beaudet%2C%20James%20R.%20Lupski%2C%20Sau%20W.%20Cheung%2C%20Patricia%20A.%20Ward&start_page=1&end_page=11" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000327" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000327" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0000327" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000327&amp;t=Clinical%20Implementation%20of%20Chromosomal%20Microarray%20Analysis%3A%20Summary%20of%202513%20Postnatal%20Cases" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000327&title=Clinical%20Implementation%20of%20Chromosomal%20Microarray%20Analysis%3A%20Summary%20of%202513%20Postnatal%20Cases&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000327&amp;title=Clinical%20Implementation%20of%20Chromosomal%20Microarray%20Analysis%3A%20Summary%20of%202513%20Postnatal%20Cases" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000327" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0000327&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Clinical Implementation of Chromosomal Microarray Analysis: Summary of 2513 Postnatal Cases';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0000327';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Clinical%20Implementation%20of%20Chromosomal%20Microarray%20Analysis%3A%20Summary%20of%202513%20Postnatal%20Cases http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000327" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0000327" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">











          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Array+CGH%22" title="Search for articles in the subject area:'Array CGH'"><div class="flagText">Array CGH</div></a>
              <div data-categoryid="41443" data-articleid="23492"
                   data-categoryname="Array CGH"
                   class="flagImage" title="Flag 'Array CGH' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Chromosomes%22" title="Search for articles in the subject area:'Chromosomes'"><div class="flagText">Chromosomes</div></a>
              <div data-categoryid="17251" data-articleid="23492"
                   data-categoryname="Chromosomes"
                   class="flagImage" title="Flag 'Chromosomes' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Cytogenetics%22" title="Search for articles in the subject area:'Cytogenetics'"><div class="flagText">Cytogenetics</div></a>
              <div data-categoryid="18527" data-articleid="23492"
                   data-categoryname="Cytogenetics"
                   class="flagImage" title="Flag 'Cytogenetics' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Genomic+databases%22" title="Search for articles in the subject area:'Genomic databases'"><div class="flagText">Genomic databases</div></a>
              <div data-categoryid="42727" data-articleid="23492"
                   data-categoryname="Genomic databases"
                   class="flagImage" title="Flag 'Genomic databases' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Genomics%22" title="Search for articles in the subject area:'Genomics'"><div class="flagText">Genomics</div></a>
              <div data-categoryid="18221" data-articleid="23492"
                   data-categoryname="Genomics"
                   class="flagImage" title="Flag 'Genomics' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Karyotypes%22" title="Search for articles in the subject area:'Karyotypes'"><div class="flagText">Karyotypes</div></a>
              <div data-categoryid="21591" data-articleid="23492"
                   data-categoryname="Karyotypes"
                   class="flagImage" title="Flag 'Karyotypes' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Mammalian+genomics%22" title="Search for articles in the subject area:'Mammalian genomics'"><div class="flagText">Mammalian genomics</div></a>
              <div data-categoryid="17625" data-articleid="23492"
                   data-categoryname="Mammalian genomics"
                   class="flagImage" title="Flag 'Mammalian genomics' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Translocations%22" title="Search for articles in the subject area:'Translocations'"><div class="flagText">Translocations</div></a>
              <div data-categoryid="43565" data-articleid="23492"
                   data-categoryname="Translocations"
                   class="flagImage" title="Flag 'Translocations' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=4580'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=281'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=5838&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>

<div class="block sidebar-comments">
    <div class="header">
        <h3>Comments</h3>
    </div>
      <p><a href="/annotation/listThread.action?root=8135">Don't use oligo arrays</a><br>Posted by dsingh</p>
</div>

</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
