

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Association of Killer Cell Immunoglobulin-Like Receptor Genes with Hodgkin's Lymphoma in a Familial Study</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0000406"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0000406" />

    <meta name="citation_title" content="Association of Killer Cell Immunoglobulin-Like Receptor Genes with Hodgkin's Lymphoma in a Familial Study"/>
    <meta itemprop="name" content="Association of Killer Cell Immunoglobulin-Like Receptor Genes with Hodgkin's Lymphoma in a Familial Study"/>

      <meta name="citation_author" content="Caroline Besson"/>
            <meta name="citation_author_institution" content="Laboratoire de Génétique Humaine des Maladies Infectieuses, INSERM, U550, Paris, France"/>
            <meta name="citation_author_institution" content="Université Paris Descartes, Faculté de Médecine René Descartes, Paris, France"/>
            <meta name="citation_author_institution" content="Assistance Publique-Hôpitaux de Paris (AP-HP), Service d'Hématologie et Immunologie Biologiques, CHU Bicêtre, Le Kremlin-Bicêtre, France"/>
            <meta name="citation_author_institution" content="Université Paris Sud, Faculté de Médecine, Le Kremlin-Bicêtre, France"/>
      <meta name="citation_author" content="Sophie Roetynck"/>
            <meta name="citation_author_institution" content="Centre d'Immunologie de Marseille-Luminy, Université de la Méditerranée, Marseille, France"/>
            <meta name="citation_author_institution" content="INSERM, U631, Marseille, France"/>
            <meta name="citation_author_institution" content="CNRS, UMR6102, Marseille, France"/>
      <meta name="citation_author" content="Fionnuala Williams"/>
            <meta name="citation_author_institution" content="Northern Ireland Regional Histocompatibility and Immunogenetics Laboratory, City Hospital, Belfast, Northern Ireland"/>
      <meta name="citation_author" content="Laurent Orsi"/>
            <meta name="citation_author_institution" content="INSERM, U754, Université Paris XI, Villejuif, France"/>
      <meta name="citation_author" content="Corinne Amiel"/>
            <meta name="citation_author_institution" content="Laboratoire de Virologie, Hôpital Tenon, Paris, France"/>
      <meta name="citation_author" content="Catherine Lependeven"/>
            <meta name="citation_author_institution" content="Laboratoire de Virologie, Hôpital Tenon, Paris, France"/>
      <meta name="citation_author" content="Guillemette Antoni"/>
            <meta name="citation_author_institution" content="Laboratoire de Génétique Humaine des Maladies Infectieuses, INSERM, U550, Paris, France"/>
            <meta name="citation_author_institution" content="Université Paris Descartes, Faculté de Médecine René Descartes, Paris, France"/>
      <meta name="citation_author" content="Olivier Hermine"/>
            <meta name="citation_author_institution" content="Service d'Hématologie adultes, Hôpital Necker, Paris, France"/>
      <meta name="citation_author" content="Pauline Brice"/>
            <meta name="citation_author_institution" content="Service d'Onco-hématologie, Hôpital Saint-Louis, Paris, France"/>
      <meta name="citation_author" content="Christophe Ferme"/>
            <meta name="citation_author_institution" content="Département d'Hématologie, Institut Gustave Roussy, Villejuif, France"/>
      <meta name="citation_author" content="Patrice Carde"/>
            <meta name="citation_author_institution" content="Département d'Hématologie, Institut Gustave Roussy, Villejuif, France"/>
      <meta name="citation_author" content="Danielle Canioni"/>
            <meta name="citation_author_institution" content="Service d'anatomo-pathologie, Hôpital Necker, Paris, France"/>
      <meta name="citation_author" content="Josette Brière"/>
            <meta name="citation_author_institution" content="Service d'anatomo-pathologie, Hôpital Saint-Louis, Paris, France"/>
      <meta name="citation_author" content="Martine Raphael"/>
            <meta name="citation_author_institution" content="Assistance Publique-Hôpitaux de Paris (AP-HP), Service d'Hématologie et Immunologie Biologiques, CHU Bicêtre, Le Kremlin-Bicêtre, France"/>
            <meta name="citation_author_institution" content="Université Paris Sud, Faculté de Médecine, Le Kremlin-Bicêtre, France"/>
      <meta name="citation_author" content="Jean-Claude Nicolas"/>
            <meta name="citation_author_institution" content="Laboratoire de Virologie, Hôpital Tenon, Paris, France"/>
      <meta name="citation_author" content="Jacqueline Clavel"/>
            <meta name="citation_author_institution" content="Université Paris Sud, Faculté de Médecine, Le Kremlin-Bicêtre, France"/>
            <meta name="citation_author_institution" content="INSERM, U754, Université Paris XI, Villejuif, France"/>
      <meta name="citation_author" content="Derek Middleton"/>
            <meta name="citation_author_institution" content="Northern Ireland Regional Histocompatibility and Immunogenetics Laboratory, City Hospital, Belfast, Northern Ireland"/>
            <meta name="citation_author_institution" content="School of Biomedical Sciences, University of Ulster, Coleraine, Northern Ireland"/>
      <meta name="citation_author" content="Eric Vivier"/>
            <meta name="citation_author_institution" content="Centre d'Immunologie de Marseille-Luminy, Université de la Méditerranée, Marseille, France"/>
            <meta name="citation_author_institution" content="INSERM, U631, Marseille, France"/>
            <meta name="citation_author_institution" content="CNRS, UMR6102, Marseille, France"/>
            <meta name="citation_author_institution" content="Assistance Publique–Hôpitaux de Marseille, Hôpital de la Conception, France"/>
      <meta name="citation_author" content="Laurent Abel"/>
            <meta name="citation_author_institution" content="Laboratoire de Génétique Humaine des Maladies Infectieuses, INSERM, U550, Paris, France"/>
            <meta name="citation_author_institution" content="Université Paris Descartes, Faculté de Médecine René Descartes, Paris, France"/>

    <meta name="citation_date" content="2007/5/2"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0000406.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e406"/>
    <meta name="citation_issue" content="5"/>
    <meta name="citation_volume" content="2"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Molecular pathogenesis of Hodgkin's lymphoma.; citation_author=D Re; citation_author=R Kuppers; citation_author=V Diehl; citation_journal_title=J Clin Oncol; citation_volume=23; citation_number=1; citation_pages=6379-6386; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Epidemiology of Hodgkin's disease: a review.; citation_author=RA Cartwright; citation_author=G Watkins; citation_journal_title=Hematol Oncol; citation_volume=22; citation_number=2; citation_pages=11-26; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Incidence of Hodgkin's disease in Nordic countries.; citation_author=H Hjalgrim; citation_author=J Askling; citation_author=E Pukkala; citation_author=S Hansen; citation_author=L Munksgaard; citation_journal_title=Lancet; citation_volume=358; citation_number=3; citation_pages=297-298; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Hodgkin's disease in siblings.; citation_author=S Grufferman; citation_author=P Cole; citation_author=PG Smith; citation_author=RJ Lukes; citation_journal_title=N Engl J Med; citation_volume=296; citation_number=4; citation_pages=248-250; citation_date=1977; " />
      <meta name="citation_reference" content="citation_title=A genetic study of Hodgkin's lymphoma: an estimate of heritability and anticipation based on the familial cancer database in Sweden.; citation_author=YY Shugart; citation_author=K Hemminki; citation_author=P Vaittinen; citation_author=A Kingman; citation_author=C Dong; citation_journal_title=Hum Genet; citation_volume=106; citation_number=5; citation_pages=553-556; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Familial Hodgkin's disease: a disease of young adulthood?; citation_author=AM Ferraris; citation_author=O Racchi; citation_author=D Rapezzi; citation_author=GF Gaetani; citation_author=P Boffetta; citation_journal_title=Ann Hematol; citation_volume=74; citation_number=6; citation_pages=131-134; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Concordance for Hodgkin's disease in identical twins suggesting genetic susceptibility to the young-adult form of the disease.; citation_author=TM Mack; citation_author=W Cozen; citation_author=DK Shibata; citation_author=LM Weiss; citation_author=BN Nathwani; citation_journal_title=N Engl J Med; citation_volume=332; citation_number=7; citation_pages=413-418; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=HLA and susceptibility to Hodgkin's disease.; citation_author=J Hors; citation_author=J Dausset; citation_journal_title=Immunol Rev; citation_volume=70; citation_number=8; citation_pages=167-192; citation_date=1983; " />
      <meta name="citation_reference" content="citation_title=HLA-DR, HLA-DQ, and TAP genes in familial Hodgkin disease.; citation_author=LC Harty; citation_author=AY Lin; citation_author=AM Goldstein; citation_author=ES Jaffe; citation_author=M Carrington; citation_journal_title=Blood; citation_volume=99; citation_number=9; citation_pages=690-693; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Association with HLA class I in Epstein-Barr-virus-positive and with HLA class III in Epstein-Barr-virus-negative Hodgkin's lymphoma.; citation_author=A Diepstra; citation_author=M Niens; citation_author=E Vellenga; citation_author=GW van Imhoff; citation_author=IM Nolte; citation_journal_title=Lancet; citation_volume=365; citation_number=10; citation_pages=2216-2224; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=A genome screen of families at high risk for Hodgkin lymphoma: evidence for a susceptibility gene on chromosome 4.; citation_author=LR Goldin; citation_author=ML McMaster; citation_author=M Ter-Minassian; citation_author=S Saddlemire; citation_author=B Harmsen; citation_journal_title=J Med Genet; citation_volume=42; citation_number=11; citation_pages=595-601; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Epstein-Barr virus-associated Hodgkin's lymphoma.; citation_author=MK Gandhi; citation_author=JT Tellam; citation_author=R Khanna; citation_journal_title=Br J Haematol; citation_volume=125; citation_number=12; citation_pages=267-281; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Part I: Hodgkin's lymphoma–molecular biology of Hodgkin and Reed-Sternberg cells.; citation_author=RK Thomas; citation_author=D Re; citation_author=J Wolf; citation_author=V Diehl; citation_journal_title=Lancet Oncol; citation_volume=5; citation_number=13; citation_pages=11-18; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Persistence of the Epstein-Barr virus and the origins of associated lymphomas.; citation_author=DA Thorley-Lawson; citation_author=A Gross; citation_journal_title=N Engl J Med; citation_volume=350; citation_number=14; citation_pages=1328-1337; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Characteristics of Hodgkin's lymphoma after infectious mononucleosis.; citation_author=H Hjalgrim; citation_author=J Askling; citation_author=K Rostgaard; citation_author=S Hamilton-Dutoit; citation_author=M Frisch; citation_journal_title=N Engl J Med; citation_volume=349; citation_number=15; citation_pages=1324-1332; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Hodgkin's disease and Epstein-Barr virus. Altered antibody pattern before diagnosis.; citation_author=N Mueller; citation_author=A Evans; citation_author=NL Harris; citation_author=GW Comstock; citation_author=E Jellum; citation_journal_title=N Engl J Med; citation_volume=320; citation_number=16; citation_pages=689-695; citation_date=1989; " />
      <meta name="citation_reference" content="citation_title=Natural killer cells and viral infections.; citation_author=AR French; citation_author=WM Yokoyama; citation_journal_title=Curr Opin Immunol; citation_volume=15; citation_number=17; citation_pages=45-51; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Natural killer cells as an initial defense against pathogens.; citation_author=MB Lodoen; citation_author=LL Lanier; citation_journal_title=Curr Opin Immunol; citation_volume=18; citation_number=18; citation_pages=391-398; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=The immune response to Epstein-Barr virus.; citation_author=MF Callan; citation_journal_title=Microbes Infect; citation_volume=6; citation_number=19; citation_pages=937-945; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=The immune response to primary EBV infection: a role for natural killer cells.; citation_author=H Williams; citation_author=K McAulay; citation_author=KF Macsween; citation_author=NJ Gallacher; citation_author=CD Higgins; citation_journal_title=Br J Haematol; citation_volume=129; citation_number=20; citation_pages=266-274; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=A novel primary immunodeficiency with specific natural-killer cell deficiency maps to the centromeric region of chromosome 8.; citation_author=C Eidenschenk; citation_author=J Dunne; citation_author=E Jouanguy; citation_author=C Fourlinnie; citation_author=L Gineau; citation_journal_title=Am J Hum Genet; citation_volume=78; citation_number=21; citation_pages=721-727; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Susceptibility to mouse cytomegalovirus is associated with deletion of an activating natural killer cell receptor of the C-type lectin superfamily.; citation_author=SH Lee; citation_author=S Girard; citation_author=D Macina; citation_author=M Busa; citation_author=A Zafer; citation_journal_title=Nat Genet; citation_volume=28; citation_number=22; citation_pages=42-45; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Direct recognition of cytomegalovirus by activating and inhibitory NK cell receptors.; citation_author=H Arase; citation_author=ES Mocarski; citation_author=AE Campbell; citation_author=AB Hill; citation_author=LL Lanier; citation_journal_title=Science; citation_volume=296; citation_number=23; citation_pages=1323-1326; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=KIR: diverse, rapidly evolving receptors of innate and adaptive immunity.; citation_author=C Vilches; citation_author=P Parham; citation_journal_title=Annu Rev Immunol; citation_volume=20; citation_number=24; citation_pages=217-251; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=The Killer Immunoglobulin-like Receptor Gene Cluster: Tuning the Genome for Defense.; citation_author=AA Bashirova; citation_author=MP Martin; citation_author=DW McVicar; citation_author=M Carrington; citation_journal_title=Annu Rev Genomics Hum Genet; citation_number=25; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=The killer cell immunoglobulin-like receptor (KIR) genomic region: gene-order, haplotypes and allelic polymorphism.; citation_author=KC Hsu; citation_author=S Chida; citation_author=DE Geraghty; citation_author=B Dupont; citation_journal_title=Immunol Rev; citation_volume=190; citation_number=26; citation_pages=40-52; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Recognition of peptide-MHC class I complexes by activating killer immunoglobulin-like receptors.; citation_author=CA Stewart; citation_author=F Laugier-Anfossi; citation_author=F Vely; citation_author=X Saulquin; citation_author=J Riedmuller; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=102; citation_number=27; citation_pages=13224-13229; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Epistatic interaction between KIR3DS1 and HLA-B delays the progression to AIDS.; citation_author=MP Martin; citation_author=X Gao; citation_author=JH Lee; citation_author=GW Nelson; citation_author=R Detels; citation_journal_title=Nat Genet; citation_volume=31; citation_number=28; citation_pages=429-434; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Protective effect of the HLA-Bw4I80 epitope and the killer cell immunoglobulin-like receptor 3DS1 gene against the development of hepatocellular carcinoma in patients with hepatitis C virus infection.; citation_author=A Lopez-Vazquez; citation_author=L Rodrigo; citation_author=J Martinez-Borra; citation_author=R Perez; citation_author=M Rodriguez; citation_journal_title=J Infect Dis; citation_volume=192; citation_number=29; citation_pages=162-165; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Hierarchy of resistance to cervical neoplasia mediated by combinations of killer immunoglobulin-like receptor and human leukocyte antigen loci.; citation_author=M Carrington; citation_author=S Wang; citation_author=MP Martin; citation_author=X Gao; citation_author=M Schiffman; citation_journal_title=J Exp Med; citation_volume=201; citation_number=30; citation_pages=1069-1075; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Variation of the killer cell immunoglobulin-like receptors and HLA-C genes in nasopharyngeal carcinoma.; citation_author=M Butsch Kovacic; citation_author=M Martin; citation_author=X Gao; citation_author=T Fuksenko; citation_author=CJ Chen; citation_journal_title=Cancer Epidemiol Biomarkers Prev; citation_volume=14; citation_number=31; citation_pages=2673-2677; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Genetic diversity of KIR natural killer cell markers in populations from France, Guadeloupe, Finland, Senegal and Reunion.; citation_author=L Denis; citation_author=J Sivula; citation_author=PA Gourraud; citation_author=N Kerdudou; citation_author=R Chout; citation_journal_title=Tissue Antigens; citation_volume=66; citation_number=32; citation_pages=267-276; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Definition of gene content for nine common group B haplotypes of the Caucasoid population: KIR haplotypes contain between seven and eleven KIR genes.; citation_author=M Uhrberg; citation_author=P Parham; citation_author=P Wernet; citation_journal_title=Immunogenetics; citation_volume=54; citation_number=33; citation_pages=221-229; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Killer Ig-like receptor haplotype analysis by gene content: evidence for genomic diversity with a minimum of six basic framework haplotypes, each with multiple subsets.; citation_author=KC Hsu; citation_author=XR Liu; citation_author=A Selvakumar; citation_author=E Mickelson; citation_author=RJ O'Reilly; citation_journal_title=J Immunol; citation_volume=169; citation_number=34; citation_pages=5118-5129; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=A common KIR2DS4 deletion variant in the human that predicts a soluble KIR molecule analogous to the KIR1D molecule observed in the rhesus monkey.; citation_author=LD Maxwell; citation_author=A Wallace; citation_author=D Middleton; citation_author=MD Curran; citation_journal_title=Tissue Antigens; citation_volume=60; citation_number=35; citation_pages=254-258; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Investigation of killer cell immunoglobulin-like receptor gene diversity: IV. KIR3DL1/S1.; citation_author=IA Halfpenny; citation_author=D Middleton; citation_author=YA Barnett; citation_author=F Williams; citation_journal_title=Hum Immunol; citation_volume=65; citation_number=36; citation_pages=602-612; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Natural killer cell signaling pathways.; citation_author=E Vivier; citation_author=JA Nunes; citation_author=F Vely; citation_journal_title=Science; citation_volume=306; citation_number=37; citation_pages=1517-1519; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=The Switch from Latent to Productive Infection in Epstein-Barr Virus-Infected B Cells Is Associated with Sensitization to NK Cell Killing.; citation_author=I Pappworth; citation_author=E Wang; citation_author=M Rowe; citation_journal_title=J Virol; citation_volume=81; citation_number=38; citation_pages=474-482; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Natural killer cells utilize both perforin and gamma interferon to regulate murine cytomegalovirus infection in the spleen and liver.; citation_author=J Loh; citation_author=DT Chu; citation_author=AK O'Guin; citation_author=WM Yokoyama; citation_author=HWt Virgin; citation_journal_title=J Virol; citation_volume=79; citation_number=39; citation_pages=661-667; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Epistasis between mouse Klra and major histocompatibility complex class I loci is associated with a new mechanism of natural killer cell-mediated innate resistance to cytomegalovirus infection.; citation_author=MP Desrosiers; citation_author=A Kielczewska; citation_author=JC Loredo-Osti; citation_author=SG Adam; citation_author=AP Makrigiannis; citation_journal_title=Nat Genet; citation_volume=37; citation_number=40; citation_pages=593-599; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Analysis of major histocompatibility complex class I expression on Reed-Sternberg cells in relation to the cytotoxic T-cell response in Epstein-Barr virus-positive and -negative Hodgkin's disease.; citation_author=JJ Oudejans; citation_author=NM Jiwa; citation_author=JA Kummer; citation_author=A Horstman; citation_author=W Vos; citation_journal_title=Blood; citation_volume=87; citation_number=41; citation_pages=3844-3851; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Positive correlation between Epstein-Barr virus viral load and anti-viral capsid immunoglobulin G titers determined for Hodgkin's lymphoma patients and their relatives.; citation_author=C Besson; citation_author=C Amiel; citation_author=C Le-Pendeven; citation_author=P Brice; citation_author=C Ferme; citation_journal_title=J Clin Microbiol; citation_volume=44; citation_number=42; citation_pages=47-50; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Development of a PCR-SSOP approach capable of defining the natural killer cell inhibitory receptor (KIR) gene sequence repertoires.; citation_author=KA Crum; citation_author=SE Logue; citation_author=MD Curran; citation_author=D Middleton; citation_journal_title=Tissue Antigens; citation_volume=56; citation_number=43; citation_pages=313-326; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=KIR genes.; citation_author=D Middleton; citation_author=F Williams; citation_author=IA Halfpenny; citation_journal_title=Transpl Immunol; citation_volume=14; citation_number=44; citation_pages=135-142; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Investigation of killer cell immunoglobulinlike receptor gene diversity: I. KIR2DL4.; citation_author=F Williams; citation_author=A Meenagh; citation_author=C Sleator; citation_author=D Middleton; citation_journal_title=Hum Immunol; citation_volume=65; citation_number=45; citation_pages=31-38; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Investigation of killer cell immunoglobulin-like receptor gene diversity: II. KIR2DS4.; citation_author=LD Maxwell; citation_author=F Williams; citation_author=P Gilmore; citation_author=A Meenagh; citation_author=D Middleton; citation_journal_title=Hum Immunol; citation_volume=65; citation_number=46; citation_pages=613-621; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Study of KIR genes in tuberculosis patients.; citation_author=AGH Méndez; citation_author=A Meenagh; citation_author=S Contreras; citation_author=R Zavaleta; citation_author=MF Mendoza; citation_author=L Izquierdo; citation_author=ME Sarmiento; citation_author=A Acosta; citation_author=D Middleton; citation_journal_title=Tissue Antigens In Press; citation_number=47; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Modification of an HLA-B PCR-SSOP typing system leading to improved allele determination.; citation_author=D Middleton; citation_author=F Williams; citation_author=C Cullen; citation_author=E Mallon; citation_journal_title=Tissue Antigens; citation_volume=45; citation_number=48; citation_pages=232-236; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Molecular diversity of the HLA-C gene identified in a caucasian population.; citation_author=F Williams; citation_author=A Meenagh; citation_author=C Patterson; citation_author=D Middleton; citation_journal_title=Hum Immunol; citation_volume=63; citation_number=49; citation_pages=602-613; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Haplotype blocks and linkage disequilibrium in the human genome.; citation_author=JD Wall; citation_author=JK Pritchard; citation_journal_title=Nat Rev Genet; citation_volume=4; citation_number=50; citation_pages=587-597; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Transmission test for linkage disequilibrium: the insulin gene region and insulin-dependent diabetes mellitus (IDDM).; citation_author=RS Spielman; citation_author=RE McGinnis; citation_author=WJ Ewens; citation_journal_title=Am J Hum Genet; citation_volume=52; citation_number=51; citation_pages=506-516; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=The transmission/disequilibrium test and parental-genotype reconstruction: the reconstruction-combined transmission/disequilibrium test.; citation_author=M Knapp; citation_journal_title=Am J Hum Genet; citation_volume=64; citation_number=52; citation_pages=861-870; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=A sibship test for linkage in the presence of association: the sib transmission/disequilibrium test.; citation_author=RS Spielman; citation_author=WJ Ewens; citation_journal_title=Am J Hum Genet; citation_volume=62; citation_number=53; citation_pages=450-458; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=The family based association test method: strategies for studying general genotype–phenotype associations.; citation_author=S Horvath; citation_author=X Xu; citation_author=NM Laird; citation_journal_title=Eur J Hum Genet; citation_volume=9; citation_number=54; citation_pages=301-306; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Family-based tests of association in the presence of linkage.; citation_author=SL Lake; citation_author=D Blacker; citation_author=NM Laird; citation_journal_title=Am J Hum Genet; citation_volume=67; citation_number=55; citation_pages=1515-1525; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Use of parents, sibs, and unrelated controls for detection of associations between genetic markers and disease.; citation_author=DJ Schaid; citation_author=C Rowland; citation_journal_title=Am J Hum Genet; citation_volume=63; citation_number=56; citation_pages=1492-1506; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Alleles of the NRAMP1 gene are risk factors for pediatric tuberculosis disease.; citation_author=S Malik; citation_author=L Abel; citation_author=H Tooker; citation_author=A Poon; citation_author=L Simkin; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=102; citation_number=57; citation_pages=12183-12188; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=New nomenclature for MHC receptors.; citation_author=P Andre; citation_author=R Biassoni; citation_author=M Colonna; citation_author=D Cosman; citation_author=LL Lanier; citation_journal_title=Nat Immunol.; citation_volume=2; citation_number=58; citation_pages=661; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=MHC class I molecules and KIRs in human history, health and survival.; citation_author=P Parham; citation_journal_title=Nat Rev Immunol; citation_volume=5; citation_number=59; citation_pages=201-214; citation_date=2005; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000406" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Association of Killer Cell Immunoglobulin-Like Receptor Genes with Hodgkin's Lymphoma in a Familial Study"/>
    <meta name="twitter:description" content="BackgroundEpstein-Barr virus (EBV) is the major environmental factor associated with Hodgkin's lymphoma (HL), a common lymphoma in young adults. Natural killer (NK) cells are key actors of the innate immune response against viruses. The regulation of NK cell function involves activating and inhibitory Killer cell Immunoglobulin-like receptors (KIRs), which are expressed in variable numbers on NK cells. Various viral and virus-related malignant disorders have been associated with the presence/absence of certain KIR genes in case/control studies. We investigated the role of the KIR cluster in HL in a family-based association study.MethodologyWe included 90 families with 90 HL index cases (age 16&ndash;35 years) and 255 first-degree relatives (parents and siblings). We developed a procedure for reconstructing full genotypic information (number of gene copies) at each KIR locus from the standard KIR gene content. Out of the 90 collected families, 84 were informative and suitable for further analysis. An association study was then carried out with specific family-based analysis methods on these 84 families.Principal FindingsFive KIR genes in strong linkage disequilibrium were found significantly associated with HL. Refined haplotype analysis showed that the association was supported by a dominant protective effect of KIR3DS1 and/or KIR2DS1, both of which are activating receptors. The odds ratios for developing HL in subjects with at least one copy of KIR3DS1 or KIR2DS1 with respect to subjects with neither of these genes were 0.44[95% confidence interval 0.23&ndash;0.85] and 0.42[0.21&ndash;0.85], respectively. No significant association was found in a tentative replication case/control study of 68 HL cases (age 18&ndash;71 years). In the familial study, the protective effect of KIR3DS1/KIR2DS1 tended to be stronger in HL patients with detectable EBV in blood or tumour cells.ConclusionsThis work defines a template for family-based association studies based on full genotypic information for the KIR cluster, and provides the first evidence that activating KIRs can have a protective role in HL."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0000406.g004"/>

  <meta property="og:title" content="Association of Killer Cell Immunoglobulin-Like Receptor Genes with Hodgkin's Lymphoma in a Familial Study" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000406" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=572'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=4529'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=6710&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0000406">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000406" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000406&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Association of Killer Cell Immunoglobulin-Like Receptor Genes with Hodgkin's Lymphoma in a Familial Study
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Caroline Besson
              <span class="equal-contrib"
                    title="These authors contributed equally to this work">equal contributor</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">

                <p><span class="equal-contrib" title="These authors contributed equally to this work">equal contributor</span>
                Contributed equally to this work with: Caroline Besson, Sophie Roetynck</p>

              
              

                <p>Affiliations:
                  Laboratoire de Génétique Humaine des Maladies Infectieuses, INSERM, U550, Paris, France, 
                  Université Paris Descartes, Faculté de Médecine René Descartes, Paris, France, 
                  Assistance Publique-Hôpitaux de Paris (AP-HP), Service d'Hématologie et Immunologie Biologiques, CHU Bicêtre, Le Kremlin-Bicêtre, France, 
                  Université Paris Sud, Faculté de Médecine, Le Kremlin-Bicêtre, France
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Sophie Roetynck
              <span class="equal-contrib"
                    title="These authors contributed equally to this work">equal contributor</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">

                <p><span class="equal-contrib" title="These authors contributed equally to this work">equal contributor</span>
                Contributed equally to this work with: Caroline Besson, Sophie Roetynck</p>

              
              

                <p>Affiliations:
                  Centre d'Immunologie de Marseille-Luminy, Université de la Méditerranée, Marseille, France, 
                  INSERM, U631, Marseille, France, 
                  CNRS, UMR6102, Marseille, France
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Fionnuala Williams, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Northern Ireland Regional Histocompatibility and Immunogenetics Laboratory, City Hospital, Belfast, Northern Ireland
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Laurent Orsi, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  INSERM, U754, Université Paris XI, Villejuif, France
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Corinne Amiel, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Laboratoire de Virologie, Hôpital Tenon, Paris, France
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Catherine Lependeven, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Laboratoire de Virologie, Hôpital Tenon, Paris, France
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Guillemette Antoni, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliations:
                  Laboratoire de Génétique Humaine des Maladies Infectieuses, INSERM, U550, Paris, France, 
                  Université Paris Descartes, Faculté de Médecine René Descartes, Paris, France
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Olivier Hermine, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Service d'Hématologie adultes, Hôpital Necker, Paris, France
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Pauline Brice, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Service d'Onco-hématologie, Hôpital Saint-Louis, Paris, France
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Christophe Ferme, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Département d'Hématologie, Institut Gustave Roussy, Villejuif, France
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Patrice Carde, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Département d'Hématologie, Institut Gustave Roussy, Villejuif, France
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Danielle Canioni, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Service d'anatomo-pathologie, Hôpital Necker, Paris, France
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Josette Brière, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Service d'anatomo-pathologie, Hôpital Saint-Louis, Paris, France
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Martine Raphael, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliations:
                  Assistance Publique-Hôpitaux de Paris (AP-HP), Service d'Hématologie et Immunologie Biologiques, CHU Bicêtre, Le Kremlin-Bicêtre, France, 
                  Université Paris Sud, Faculté de Médecine, Le Kremlin-Bicêtre, France
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Jean-Claude Nicolas, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Laboratoire de Virologie, Hôpital Tenon, Paris, France
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Jacqueline Clavel, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliations:
                  Université Paris Sud, Faculté de Médecine, Le Kremlin-Bicêtre, France, 
                  INSERM, U754, Université Paris XI, Villejuif, France
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Derek Middleton, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliations:
                  Northern Ireland Regional Histocompatibility and Immunogenetics Laboratory, City Hospital, Belfast, Northern Ireland, 
                  School of Biomedical Sciences, University of Ulster, Coleraine, Northern Ireland
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Eric Vivier, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliations:
                  Centre d'Immunologie de Marseille-Luminy, Université de la Méditerranée, Marseille, France, 
                  INSERM, U631, Marseille, France, 
                  CNRS, UMR6102, Marseille, France, 
                  Assistance Publique–Hôpitaux de Marseille, Hôpital de la Conception, France
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Laurent Abel
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:abel@necker.fr">abel@necker.fr</a></p>

                <p>Affiliations:
                  Laboratoire de Génétique Humaine des Maladies Infectieuses, INSERM, U550, Paris, France, 
                  Université Paris Descartes, Faculté de Médecine René Descartes, Paris, France
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: May 02, 2007</li>
    <li>DOI: 10.1371/journal.pone.0000406</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0000406" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0000406" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000406" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0000406" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0000406-g001" data-doi="info:doi/10.1371/journal.pone.0000406" data-uri="info:doi/10.1371/journal.pone.0000406.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000406.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000406-g002" data-doi="info:doi/10.1371/journal.pone.0000406" data-uri="info:doi/10.1371/journal.pone.0000406.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000406.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000406-g003" data-doi="info:doi/10.1371/journal.pone.0000406" data-uri="info:doi/10.1371/journal.pone.0000406.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000406.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000406-t001" data-doi="info:doi/10.1371/journal.pone.0000406" data-uri="info:doi/10.1371/journal.pone.0000406.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000406.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000406-g004" data-doi="info:doi/10.1371/journal.pone.0000406" data-uri="info:doi/10.1371/journal.pone.0000406.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000406.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000406-t002" data-doi="info:doi/10.1371/journal.pone.0000406" data-uri="info:doi/10.1371/journal.pone.0000406.t002" title="Table 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000406.t002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000406-t003" data-doi="info:doi/10.1371/journal.pone.0000406" data-uri="info:doi/10.1371/journal.pone.0000406.t003" title="Table 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000406.t003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000406-t004" data-doi="info:doi/10.1371/journal.pone.0000406" data-uri="info:doi/10.1371/journal.pone.0000406.t004" title="Table 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000406.t004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000406">Reader Comments (1)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0000406" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2>
<h3>Background</h3>
<a id="article1.front1.article-meta1.abstract1.sec1.p1" name="article1.front1.article-meta1.abstract1.sec1.p1"></a><p>Epstein-Barr virus (EBV) is the major environmental factor associated with Hodgkin's lymphoma (HL), a common lymphoma in young adults. Natural killer (NK) cells are key actors of the innate immune response against viruses. The regulation of NK cell function involves activating and inhibitory Killer cell Immunoglobulin-like receptors (KIRs), which are expressed in variable numbers on NK cells. Various viral and virus-related malignant disorders have been associated with the presence/absence of certain <em>KIR</em> genes in case/control studies. We investigated the role of the <em>KIR</em> cluster in HL in a family-based association study.</p>


<h3>Methodology</h3>
<a id="article1.front1.article-meta1.abstract1.sec2.p1" name="article1.front1.article-meta1.abstract1.sec2.p1"></a><p>We included 90 families with 90 HL index cases (age 16–35 years) and 255 first-degree relatives (parents and siblings). We developed a procedure for reconstructing full genotypic information (number of gene copies) at each <em>KIR</em> locus from the standard <em>KIR</em> gene content. Out of the 90 collected families, 84 were informative and suitable for further analysis. An association study was then carried out with specific family-based analysis methods on these 84 families.</p>


<h3>Principal Findings</h3>
<a id="article1.front1.article-meta1.abstract1.sec3.p1" name="article1.front1.article-meta1.abstract1.sec3.p1"></a><p>Five <em>KIR</em> genes in strong linkage disequilibrium were found significantly associated with HL. Refined haplotype analysis showed that the association was supported by a dominant protective effect of <em>KIR3DS1</em> and/or <em>KIR2DS1</em>, both of which are activating receptors. The odds ratios for developing HL in subjects with at least one copy of <em>KIR3DS1</em> or <em>KIR2DS1</em> with respect to subjects with neither of these genes were 0.44[95% confidence interval 0.23–0.85] and 0.42[0.21–0.85], respectively. No significant association was found in a tentative replication case/control study of 68 HL cases (age 18–71 years). In the familial study, the protective effect of <em>KIR3DS1</em>/<em>KIR2DS1</em> tended to be stronger in HL patients with detectable EBV in blood or tumour cells.</p>


<h3>Conclusions</h3>
<a id="article1.front1.article-meta1.abstract1.sec4.p1" name="article1.front1.article-meta1.abstract1.sec4.p1"></a><p>This work defines a template for family-based association studies based on full genotypic information for the <em>KIR</em> cluster, and provides the first evidence that activating KIRs can have a protective role in HL.</p>

</div>


<div class="articleinfo"><p><strong>Citation: </strong>Besson C, Roetynck S, Williams F, Orsi L, Amiel C, et al.  (2007) Association of Killer Cell Immunoglobulin-Like Receptor Genes with Hodgkin's Lymphoma in a Familial Study. PLoS ONE 2(5):
          e406.
            doi:10.1371/journal.pone.0000406</p><p><strong>Academic Editor: </strong>Nina Papavasiliou, Rockefeller University, United States of America</p><p><strong>Received:</strong> February 26, 2007; <strong>Accepted:</strong> March 26, 2007; <strong>Published:</strong> May 2, 2007</p><p><strong>Copyright:</strong> © 2007 Besson et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>This work was supported by Association pour la Recherche contre le Cancer, Ligue Nationale contre le cancer (EV : ‘Equipe labellisée La Ligue’), Fondation de France European Union FP6, LSHB-CT-2004-503319-Allostem, Agence Nationale de la Recherche (Réseau Innovation Biotechnologiesand Microbiologie Immunologie Maladies Emergentes), INSERM, CNRS and Ministère de l'Enseignement Supérieur et de la Recherche. and by donations from the Fondation Schlumberger and Fondation BNP Paribas. CB was supported by Fondation de France. SR was supported in part by Northern Ireland Histocompatibility and Immunogenetics Laboratory Research Trust Fund. The sponsors and funders did not participate with the design, conduct of the study, analysis, interpretation of the data, preparation, review or approval of the manuscript.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>Hodgkin's lymphoma (HL) differs from other lymphomas in terms of both specific pathological and epidemiological features. HL is characterised by the presence of large tumour cells known as Hodgkin and Reed-Sternberg cells, derived from a germinal centre B cell <a href="#pone.0000406-Re1">[1]</a>. The incidence of HL displays an unusual age distribution, with two peaks—between the ages of 15 to 34 years and over the age of 60 years in most Western countries, but in children and in the oldest age groups in developing countries <a href="#pone.0000406-Cartwright1">[2]</a>. HL is one of the most common forms of lymphoma occurring in young adults in developed countries, with an annual incidence of around 3 per 100,000 <a href="#pone.0000406-Cartwright1">[2]</a>, <a href="#pone.0000406-Hjalgrim1">[3]</a>. Both genetic and environmental factors are thought to be involved in the pathogenesis of HL <a href="#pone.0000406-Cartwright1">[2]</a>. There is growing evidence to suggest a genetic predisposition to HL, based on many reports of familial aggregation of the disease <a href="#pone.0000406-Grufferman1">[4]</a>–<a href="#pone.0000406-Ferraris1">[6]</a>, including a twin study <a href="#pone.0000406-Mack1">[7]</a>. Interestingly, a review of these studies found that familial HL lacked the classic bimodal age distribution, with only one peak observed, between the ages of 15 and 34 years <a href="#pone.0000406-Ferraris1">[6]</a>. Several variants of the major histocompatibility complex (MHC) region have been reported to be associated with HL <a href="#pone.0000406-Hors1">[8]</a>–<a href="#pone.0000406-Diepstra1">[10]</a>, but there is no consensus on the role of specific human leukocyte antigen (HLA) alleles or haplotypes in HL. In the only genome-wide scan by linkage analysis conducted to date, suggestive evidence was obtained for a HL susceptibility locus on chromosome 4p16 <a href="#pone.0000406-Goldin1">[11]</a>. The genetic basis of HL thus remains elusive.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>Several findings strongly suggest that Epstein-Barr virus (EBV) is a major environmental factor contributing to oncogenesis in HL <a href="#pone.0000406-Gandhi1">[12]</a>, <a href="#pone.0000406-Thomas1">[13]</a>. EBV clonal DNA is identified in the Reed-Sternberg cells in around 30% of cases of HL, and EBV infection is thought to provide survival signals for these abnormal B cells, leading to their proliferation <a href="#pone.0000406-ThorleyLawson1">[14]</a>. Moreover, epidemiological studies have clearly shown that the risk of developing HL be up to three times higher in subjects with a previous history of infectious mononucleosis—the symptomatic form of primary EBV infection, particularly frequent in adolescence—than in other subjects <a href="#pone.0000406-Hjalgrim2">[15]</a>. HL patients have also been found to have high EBV antibody titres at the time of HL diagnosis, and years before and after diagnosis <a href="#pone.0000406-Mueller1">[16]</a>. All these findings suggest that impairment of the immune response to EBV infection may contribute to the pathogenesis of HL. Natural killer (NK) cells are key actors of the innate immune response to viruses <a href="#pone.0000406-French1">[17]</a> <a href="#pone.0000406-Lodoen1">[18]</a>, including EBV <a href="#pone.0000406-Callan1">[19]</a>, <a href="#pone.0000406-Williams1">[20]</a>. Their role is illustrated by the recent report of a child who developed an EBV-driven lymphoproliferative disorder associated with a novel specific NK cell deficiency <a href="#pone.0000406-Eidenschenk1">[21]</a>. Further support for an involvement of these cells in innate immunity to viruses has been provided by experimental models as susceptibility to murine cytomegalovirus (MCMV), another herpes-virus, is controlled by a single gene, <em>Ly49H</em> (also called <em>Klra8</em>) encoding an NK activating receptor in mice <a href="#pone.0000406-Lee1">[22]</a>, <a href="#pone.0000406-Arase1">[23]</a>.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>The human <em>Killer cell Immunoglobulin-like receptor</em> (<em>KIR</em>) genes correspond functionally to the murine <em>Ly49</em> gene family, providing a potential example of convergent evolution <a href="#pone.0000406-Vilches1">[24]</a>. KIRs are inhibitory or activating transmembrane receptors, present on the surface of NK cells <a href="#pone.0000406-Bashirova1">[25]</a>. The length of their intracytoplasmic domains—long (KIR-L), or short (KIR-S)—determines whether they are inhibitory (KIR-L) or activating (KIR-S). Inhibitory KIRs are essential for the regulation of NK cell function via interactions with MHC class I molecules <a href="#pone.0000406-Hsu1">[26]</a>. The ligands of activating KIRs are less well known, although recent results have suggested that activating and inhibitory receptors recognise the same sets of peptide-MHC class I complexes, but differ in binding affinities <a href="#pone.0000406-Stewart1">[27]</a>. The common Human Genome Organization (HUGO) nomenclature for KIRs will be used throughout this report (<a href="#pone-0000406-g001">Figure 1</a>). The HUGO nomenclature accounts for KIR protein structure and consists of four major subdivisions based on two features: the number of extracellular Ig domains (2D or 3D) and the length of the cytoplasmic tail (L or S). The KIRs are encoded by a cluster of genes (<a href="#pone-0000406-g001">Figure 1</a>) varying considerably between individuals, resulting in the expression of between seven and 12 receptors <a href="#pone.0000406-Hsu1">[26]</a>. <em>KIR</em> gene polymorphism adds further variability, with multiple alleles identified for each <em>KIR</em> locus, but the principal source of diversity in the <em>KIR</em> region remains the presence/absence of <em>KIR</em> genes. Several case/control studies have reported an association between the presence/absence of specific <em>KIRs</em> and disease progression following infection with some oncogenic viruses <a href="#pone.0000406-Bashirova1">[25]</a>. For example, the presence of activating <em>KIR3DS1</em> in combination with its HLA class I ligand (the <em>Bw4I80</em> allele) has been shown to protect against progression to acquired immunodeficiency syndrome in HIV-infected patients <a href="#pone.0000406-Martin1">[28]</a>, and against the development of hepatocellular carcinoma in hepatitis C virus-infected patients <a href="#pone.0000406-LopezVazquez1">[29]</a>. However, the presence of this gene has also been associated with an increase in the risk of human papilloma virus-induced cervical cancer <a href="#pone.0000406-Carrington1">[30]</a>. Consistent with this finding, patients with EBV-associated nasopharyngeal carcinoma (NPC) tend to have more activating <em>KIRs</em> than controls <a href="#pone.0000406-ButschKovacic1">[31]</a>.</p>
<div class="figure" id="pone-0000406-g001"><div class="img"><a name="pone-0000406-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000406.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000406" data-uri="info:doi/10.1371/journal.pone.0000406.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000406.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000406.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000406.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000406.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000406.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000406.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000406.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000406.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000406.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Organisation of the human KIR cluster.</span></strong></p><a id="article1.body1.sec1.fig1.caption1.p1" name="article1.body1.sec1.fig1.caption1.p1"></a><p>The <em>Killer-cell Immunoglobulin-like receptor (KIR)</em> genes are organised in a head-to-tail fashion on human chromosome region 19q13.4. Each gene has nine exons (illustrated for <em>KIR3DL1</em>) and is roughly 10–16 kb in length, with short and equally homologous intergenic sequences of about 2 kb separating each pair of genes. The organisation of the exon-intron structure of the different <em>KIR</em> genes is fairly consistent with the following basic arrangement: the signal sequence is encoded by the first two exons (in grey), each Ig domain (D0, D1, D2) corresponds to a single exon (exons 3–5, respectively, in blue), the linker (in yellow) and transmembrane (in white) regions are each encoded by a single exon (exons 6 and 7, respectively), and the cytoplasmic domain is encoded by two final exons (8 and 9, in red). Two systems have been generated for naming <em>KIR</em> genes. The first follows the CD nomenclature system as CD158a, CD158b, etc., based on an approximate centromerictelomeric order of the genes on chromosome 19 <a href="#pone.0000406-Andre1">[58]</a>. As the CD nomenclature is not used routinely since it does not reflect structure, function, expression or localization, the Human Genome Organization (HUGO) nomenclature will be used throughout this report (<a href="http://www.gene.ucl.ac.uk/nomenclature/genefamily/kir.html">www.gene.ucl.ac.uk/nomenclature/genefami​ly/kir.html</a>). The HUGO nomenclature system, accounts for KIR protein structure and consists of four major subdivisions based on two features: the number of extracellular Ig domains (2D or 3D) and the length of the cytoplasmic tail (L: long or S: short). This latter information determines their functions: inhibitory (L) or activating (S). The boxes in bold indicate the activating <em>KIR</em> genes, and the dotted boxes correspond to the pseudogenes. The two boundary genes of the <em>KIR</em> cluster, <em>KIR3DL3</em> and <em>KIR3DL2,</em> and the centrally situated <em>KIR3DP1</em> pseudogene and <em>KIR2DL4</em> are present in almost all individuals and are therefore considered to be framework (or anchor) genes (shown in white) <a href="#pone.0000406-Parham1">[59]</a>. <em>KIR3DL3</em> and <em>KIR3DP1</em> delimit the centromeric part of the <em>KIR</em> locus, whereas <em>KIR2DL4</em> and <em>KIR3DL2</em> delimit the telomeric part. A 14 kb stretch of unique sequence separating <em>KIR3DP1</em> from <em>KIR2DL4</em> is the preferred site for reciprocal recombination, a mechanism resulting in the formation of new haplotypes by the reassortment of centromeric and telomeric genes. Apart from these framework genes, <em>KIR</em> gene content is highly variable in terms of both the number and type of genes present. Although initially considered to be separate genes, <em>KIR2DL2</em> and <em>KIR2DL3</em> segregate as alleles of the same locus. Similarly, <em>KIR3DS1</em> segregates as an allele of the inhibitory <em>KIR3DL1</em>. Overall, there are nine variable genes—<em>KIR2DS2, KIR2DL2/2DL3, KIR2DL1, KIR3DS1/3DL1, KIR2DL5, KIR2DS3, KIR2DS5, KIR2DS1, KIR2DS4</em>—and one variable pseudogene (<em>KIR2DP1</em>). <em>KIR2DL5</em> <em>KIR2DS3</em> and <em>KIR2DS5</em> may be found in both parts of the locus. We decided to represent these genes in the telomeric part of the cluster in this figure. Two major <em>KIR</em> haplotype groups, A and B, are classically described. The A haplotype is defined as containing the <em>KIR3DL3, KIR2DL3, KIR2DL1, KIR2DL4, KIR3DL1, KIR2DS4</em> and <em>KIR3DL2</em> genes. <em>KIR</em> gene polymorphism adds further diversity to the <em>KIR</em> region, with multiple alleles known for each <em>KIR</em> locus. Allele identification was performed for <em>KIR2DL4, KIR2DS4</em> and <em>KIR3DS1/3DL1</em> (indicated in red). This makes it possible to subdivide haplotype A further, according to the presence of the two common forms of <em>KIR2DS4</em>—the wild type <em>KIR2DS4wt</em> (haplotype A1) or the deletion variant <em>KIR2DS4del,</em> identical to <em>KIR2DS4wt</em> except for a 22 base-pair deletion causing a frame shift in translation (haplotype A2). B haplotypes are more variable and are characterised by the presence of more than one activating <em>KIR</em> gene. Genes that can be present in both group A and group B <em>KIR</em> haplotypes are shown in brown, and genes and/or alleles specific to group B <em>KIR</em> haplotypes are shown in green.</p>
<span>doi:10.1371/journal.pone.0000406.g001</span></div><a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p>Alltogether these data suggest that KIRs may be involved in the pathogenesis of virus-induced malignancies, although the role of activating and inhibitory receptors remains unclear. In the present work, we investigate the role of KIRs in the occurrence of HL. Our main study was based on a familial design, presenting two major advantages. First, we were able to use full genotypic information for <em>KIR</em> loci (number of gene copies), with the corresponding haplotypes, rather than the phenotypic information (presence/absence of the gene) used in previously published case/control designs investigating the association of KIRs with other virus-induced diseases. Second, family-based association studies are not subject to the possible confounding of gene-phenotype associations due to inappropriately chosen controls or population substructures. Our results for this familial sample show that some activating <em>KIRs</em> have a protective effect against HL, although this association was not reproduced in a tentative replication case/control study.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3>
<h4>KIR gene content in HL patients</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>Our sample consisted of 90 nuclear families, including 345 subjects (90 index HL cases, 2 affected parents, 143 unaffected parents, 3 affected sibs and 107 unaffected sibs). The mean age at diagnosis of the 90 index cases of HL was 26.2 years (range 16 to 35 years). The index cases comprised 51 men (57%) and 39 women (43%). The main morphological group was nodular sclerosis (87%), followed by mixed cellularity (10%). <a href="#pone-0000406-g002">Figure 2</a> shows the distribution of the presence of the variable <em>KIRs</em>, as determined by <em>KIR</em> gene content analysis for the HL index cases (n = 90). The proportion of HL cases with a positive KIR phenotype varied from 23% for <em>KIR2DS3</em> to more than 95% for <em>KIR2DL1</em>. Standard <em>KIR</em> gene content analysis provides only binary phenotypic results for each <em>KIR</em> (absent or present), with no information about the number of copies/alleles (1 or 2) when the <em>KIR</em> is present. As our analysis focuses on the genotypes (number of copies) of each of the nine variable <em>KIR</em> genes shown in <a href="#pone-0000406-g001">figure 1</a> (<em>KIR2DS2, KIR2DL2/2DL3, KIR2DL1, KIR3DS1/3DL1, KIR2DL5, KIR2DS3, KIR2DS5, KIR2DS1, KIR2DS4</em>), we first reconstructed the genotypes from the <em>KIR</em> gene content results.</p>
<div class="figure" id="pone-0000406-g002"><div class="img"><a name="pone-0000406-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000406.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000406" data-uri="info:doi/10.1371/journal.pone.0000406.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000406.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000406.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000406.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000406.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000406.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000406.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000406.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000406.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000406.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Phenotypic distribution of KIRs in Hodgkin's lymphoma cases.</span></strong></p><a id="article1.body1.sec2.sec1.fig1.caption1.p1" name="article1.body1.sec2.sec1.fig1.caption1.p1"></a><p>The proportion (percentages) of present <em>KIR</em> genes, as determined by <em>KIR</em> gene content analysis are presented for the index HL cases (n = 90). Eleven <em>KIR</em> genes are presented in the figure. Two pairs of <em>KIR</em> genes are allelic: <em>KIR3DL1/3DS1,</em> and <em>KIR2DL2/2DL3</em>.</p>
<span>doi:10.1371/journal.pone.0000406.g002</span></div>

<h4>Genotype reconstruction in families</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>The procedure used for genotypic reconstruction is described in detail in <a href="#pone-0000406-g003">Figure 3</a>. In the first step, genotypes could be unambiguously determined for all negative KIR phenotypes, and for the two <em>KIR</em> genes with known allelic forms, <em>KIR2DL2/2DL3</em> and <em>KIR3DS1/3DL1</em>. Allele typing for <em>KIR2DS4</em> distinguished between the two forms—<em>KIR2DS4wt</em> and <em>KIR2DS4del—</em>making it possible to reconstruct most of the genotypes for this receptor. Finally, an analysis of familial segregation provided additional genotype determination, resulting in 60% to 100% of the genotypes being known (<a href="#pone-0000406-t001">Table 1</a>), for all <em>KIRs</em> except <em>KIR2DL1</em>. The very small number of negative subjects (10/345) and the absence of additional information for <em>KIR2DL1</em> made it impossible to distinguish between heterozygous <em>KIR2DL1</em> (+,−) and homozygous <em>KIR2DL1</em> (+,+) in most families. Based on both genotypic reconstitution and additional allele typing of the anchor gene <em>KIR2DL4</em>, two families with Mendelian inconsistencies were excluded. Four additional families were also excluded, as individuals did not fit the pattern of familial segregation at two adjacent loci, <em>KIR2DL4</em> and <em>KIR3DS1/3DL1</em>, due either to Mendelian incompatibilities or to possible deletion of these loci in one haplotype. Finally, the sample subsequently used for linkage disequilibrium (LD) and association studies consisted of 84 families, comprising 322 subjects and including a total of 88 HL patients.</p>
<div class="figure" id="pone-0000406-g003"><div class="img"><a name="pone-0000406-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000406.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000406" data-uri="info:doi/10.1371/journal.pone.0000406.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000406.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000406.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000406.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000406.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000406.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000406.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000406.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000406.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000406.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Genotypic reconstruction.</span></strong></p><a id="article1.body1.sec2.sec2.fig1.caption1.p1" name="article1.body1.sec2.sec2.fig1.caption1.p1"></a><p><em>KIR</em> gene content analysis provides only binary phenotyping results for each KIR (absent or present), rather than complete genotypic information. Thus, for genotype analysis (i.e. the number of copies of each <em>KIR</em> gene), genotypic reconstruction in two successive steps was required. The first step combined the results of <em>KIR</em> gene content analysis, knowledge of allelic forms of <em>KIRs</em>, the additional allele typing carried out for three <em>KIR</em> genes, and the analysis of intra-familial segregation. <em>KIR</em> gene content made it possible to determine genotype unambiguously only in individuals with a negative phenotypic result. For example, an individual with a negative <em>KIR2DS2</em> phenotype has no copy of the gene, and its genotype can be deduced and denoted <em>KIR2DS2</em>(−,−). Conversely, an individual with a positive phenotype for <em>KIR2DS2</em> may have either one copy of the gene, denoted <em>KIR2DS2</em>(+,−), or two copies of the gene, denoted <em>KIR2DS2</em>(+,+). Two of the nine variable <em>KIR</em> genes have known allelic forms: <em>KIR2DL2/2DL3</em> and <em>KIR3DS1/3DL1</em>. We defined as (+,+) the <em>KIR3DS1/3DL1</em> genotype of subjects homozygous for the presence of <em>KIR3DS1</em> (phenotype <em>KIR3DS1</em>+, <em>KIR3DL1</em>−) and as (−,−) the genotype of subjects homozygous for the absence of <em>KIR3DS1</em> (phenotype <em>KIR3DS1</em>−, <em>KIR3DL1+</em>). An individual with the <em>KIR3DS1</em>+, <em>KIR3DL1</em>+ phenotype has one copy of each gene and is therefore heterozygous at the locus, with genotype <em>KIR3DS1/3DL1</em>(+,−). The genotypes of <em>KIR2DL2/2DL3</em> followed the same principle. Additional <em>KIR</em> allele typing was especially useful for the analysis of familial gene segregation<em>.</em> In particular, for <em>KIR2DS4,</em> it made it possible to distinguish between the two common forms, <em>2DS4wt</em> and <em>2DS4del</em> (<a href="#pone-0000406-g001">Figure 1</a>). The second step of the reconstruction was based on the <em>KIR</em> haplotype determination of individuals in each family. We first estimated the pairwise linkage disequilibrium (LD) pattern between <em>KIR</em> genes in our families, using the genotypic information obtained from the first step of reconstruction (see statistical analysis). Perfect and complete LD between pairs of <em>KIRs</em> made it possible to infer certain genotypes. For the few remaining unsolved <em>KIR</em> genotypes, haplotypic reconstruction was based on previously described haplotypes <a href="#pone.0000406-Hsu1">[26]</a>, <a href="#pone.0000406-Uhrberg1">[33]</a>.</p>
<span>doi:10.1371/journal.pone.0000406.g003</span></div><div class="figure" id="pone-0000406-t001"><div class="img"><a name="pone-0000406-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000406.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000406" data-uri="info:doi/10.1371/journal.pone.0000406.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000406.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000406.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000406.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000406.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000406.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000406.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000406.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000406.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000406.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>Summary of the genotypic reconstruction results for the nine variable <em>KIR</em> genes in 322 subjects. The procedure of reconstruction is detailed in <a href="#pone-0000406-g003">figure 3</a>.</span></strong></p><span>doi:10.1371/journal.pone.0000406.t001</span></div><a id="article1.body1.sec2.sec2.p2" name="article1.body1.sec2.sec2.p2"></a><p>We then estimated pairwise LD between <em>KIRs</em> based on the genotypes determined in this first reconstruction step (<a href="#pone-0000406-g004">Figure 4</a>). As previously reported, the <em>KIR</em> region was in strong LD, and two pairs of <em>KIRs</em> were found to be in perfect LD (r<sup>2</sup> = 1). <em>KIR2DS1</em> and <em>KIR2DS4</em> were in perfect negative LD—the presence of one gene in a given haplotype excluding the presence of the other, with only two haplotypes, <em>KIR2DS1+/KIR2DS4</em>- and <em>KIR2DS1-/KIR2DS4</em>+, observed. <em>KIR2DL2</em> and <em>KIR2DS2</em> were in perfect positive LD—with only the <em>KIR2DL2+/KIRDS2</em>+ and <em>KIR2DL2-/KIRDS2</em>- haplotypes observed. These findings, which are fully consistent with previously reported haplotypes <a href="#pone.0000406-Hsu1">[26]</a>, made possible the complete genotypic determination of these four <em>KIRs</em>. Three of the four remaining <em>KIRs</em> that were not fully reconstructed, <em>KIR2DL5</em>, <em>KIR2DS3</em> and <em>KIR2DS5</em>, presented complete LD (|D'| = 1) with several other <em>KIRs</em> (<a href="#pone-0000406-g004">Figure 4</a>). Based on these observations, genotypes at these three <em>KIRs</em> could be determined for most subjects. Reconstruction of the few remaining unknown <em>KIR2DL5, KIR2DS3</em> and <em>KIR2DS5</em> genotypes was based on previously reported haplotypes <a href="#pone.0000406-Hsu1">[26]</a>, and led to the genotypic determination of all but two individuals for <em>KIR2DL5</em> and all but one individual for <em>KIR2DS3</em>. By contrast, substantial genotypic reconstruction was not possible for <em>KIR2DL1</em>, which was not in strong LD with other <em>KIRs</em>. As <em>KIR2DL1</em> was not expected to be informative for the association study because of its very high phenotypic frequency (&gt;95%), this receptor was excluded from further analysis.</p>
<div class="figure" id="pone-0000406-g004"><div class="img"><a name="pone-0000406-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000406.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000406" data-uri="info:doi/10.1371/journal.pone.0000406.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000406.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000406.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000406.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000406.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000406.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000406.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000406.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000406.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000406.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>Pairwise linkage disequilibrium between <em>KIR</em> genes in the family sample.</span></strong></p><a id="article1.body1.sec2.sec2.fig2.caption1.p1" name="article1.body1.sec2.sec2.fig2.caption1.p1"></a><p>This analysis was based on the genotypes reconstructed after the first step, without using any information about LD pattern between <em>KIR</em> genes. Each square represents the magnitude of pairwise LD between <em>KIR</em> genes, as measured by D' (panel A) or r<sup>2</sup> (panel B). Red squares represent a |D'| or r<sup>2</sup> value of 1.0, corresponding to complete and perfect LD, respectively. Orange, yellow, and white squares indicate |D'| or R<sup>2</sup> values of [0.75-1[, [0.5-0.75[, and&lt;0.5, respectively.</p>
<span>doi:10.1371/journal.pone.0000406.g004</span></div><a id="article1.body1.sec2.sec2.p3" name="article1.body1.sec2.sec2.p3"></a><p>After these two reconstruction steps, study subjects had known genotypes for eight <em>KIR</em> gene systems—<em>KIR2DS2, KIR2DL2/2DL3, KIR3DS1/3DL1, KIR2DL5, KIR2DS3, KIR2DS5, KIR2DS1</em>, and <em>KIR2DS4</em>—which were used for the family-based association study (<a href="#pone-0000406-t001">Table 1</a>). All these <em>KIR</em> gene systems were diallelic (+,−), except <em>KIR2DS4</em>, which was triallelic (+wt, +del, and−). Allelic frequencies ranged from 0.15 for <em>KIR2DS3+</em> to 0.77 for <em>KIR2DS4</em>+ (0.58 for <em>KIR2DS4del</em> and 0.19 for <em>KIR2DS4wt</em>). The eight <em>KIR</em> gene systems were in Hardy-Weinberg equilibrium. The frequencies of the three main <em>KIR</em> haplotypes, A1, A2 and B, estimated from these family data were 0.46, 0.12, and 0.42, respectively, consistent with the haplotype A and B frequencies reported in other populations of European origin <a href="#pone.0000406-Denis1">[32]</a>, <a href="#pone.0000406-Uhrberg1">[33]</a>.</p>


<h4>Association study of HL with KIR genotypes</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>The results of the family-based study of association between HL and the eight <em>KIR</em> gene systems are presented in <a href="#pone-0000406-t002">Table 2</a>, assuming a dominant mode of inheritance, which gave the best fit in conditional logistic regression analysis. Alleles <em>KIR3DS1</em>+, <em>KIR2DL5</em>+, <em>KIR2DS5</em>+and <em>KIR2DS1</em>+were significantly associated (p&lt;0.05) with a protective effect against HL whereas allele <em>KIR2DS4del</em> was associated with an increase in the risk of HL. The strongest effects were observed for alleles <em>KIR3DS1</em>+(p&lt;0.006), and <em>KIR2DS1</em>+ (p&lt;0.01). Under a model of dominant inheritance, the odds ratio (OR) for developing HL in subjects homozygous <em>KIR3DS1</em>(+,+) or heterozygous <em>KIR3DS1</em>(+,−) with respect to subjects homozygous <em>KIR3DS1</em>(−,−) was 0.44 [95% confidence interval: 0.23–0.85]. Similarly, the OR for <em>KIR2DS1</em>(+,+) or <em>KIR2DS1</em>(+,−) subjects with respect to <em>KIR2DS1</em>(−,−) subjects was 0.42 [0.21–0.85].</p>
<div class="figure" id="pone-0000406-t002"><div class="img"><a name="pone-0000406-t002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000406.t002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000406" data-uri="info:doi/10.1371/journal.pone.0000406.t002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000406.t002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000406.t002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000406.t002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000406.t002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000406.t002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000406.t002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000406.t002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000406.t002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000406.t002.TIF"></span>)
                </a></li></ul></div><p><strong>Table 2.  <span>Results of the family-based association study between <em>KIR</em> genes and Hodgkin's lymphoma using a dominant model</span></strong></p><span>doi:10.1371/journal.pone.0000406.t002</span></div><a id="article1.body1.sec2.sec3.p2" name="article1.body1.sec2.sec3.p2"></a><p>As the five <em>KIR</em> genes significantly associated with HL were in strong LD (<a href="#pone-0000406-g004">Figure 4</a>), the next step was to carry out both multivariate and haplotype analyses. Multivariate conditional logistic regression demonstrated that the association between HL and the <em>KIR</em> region was supported by a single signal. Once one of the five associated <em>KIR</em> genes was entered in the regression model, particularly for the most significant system, <em>KIR3DS1/3DL1,</em> the other four genes ceased to have a significant effect. Haplotype analysis provided further detail. It showed that, due to LD, the five associated <em>KIR</em> genes were organised into only five common haplotypes (frequency&gt;0.02), accounting for 96% of the haplotypes observed in our population (<a href="#pone-0000406-t003">Table 3</a>). The two haplotypes significantly associated with a protective effect against HL (H2 and H5 in <a href="#pone-0000406-t003">Table 3</a>) contained the same alleles for <em>KIR3DS1 (+), KIR2DL5 (+)</em>, <em>KIR2DS1 (+)</em>, and <em>KIR2DS4 (−)</em>, but different alleles for <em>KIR2DS5</em> (+ and −, respectively). We also noted that the two protective haplotypes H2 and H5 did not carry <em>KIR2DS4</em>, whereas the risk-associated haplotype (H1) carried the deleted form, resulting in the production of a truncated protein predicted to have no functional effect <a href="#pone.0000406-Hsu2">[34]</a>, <a href="#pone.0000406-Maxwell1">[35]</a>. These observations made <em>KIR2DS5</em> and <em>KIR2DS4</em> unlikely candidates for the observed effect. Finally, haplotypes H3 and H4 which carried different <em>KIR2DL5</em> alleles, seemed to have opposite effects, although this difference was not significant. Thus, these results suggest that the protective effect against HL is mediated by the presence of <em>KIR3DS1</em> or <em>KIR2DS1</em> or both genes.</p>
<div class="figure" id="pone-0000406-t003"><div class="img"><a name="pone-0000406-t003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000406.t003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000406" data-uri="info:doi/10.1371/journal.pone.0000406.t003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000406.t003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000406.t003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000406.t003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000406.t003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000406.t003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000406.t003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000406.t003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000406.t003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000406.t003.TIF"></span>)
                </a></li></ul></div><p><strong>Table 3.  <span>Association study based on haplotypes constructed from five KIR genes</span></strong></p><span>doi:10.1371/journal.pone.0000406.t003</span></div><a id="article1.body1.sec2.sec3.p3" name="article1.body1.sec2.sec3.p3"></a><p>We then investigated whether the association between <em>KIR3DS1, KIR2DS1</em> and HL was influenced by the HLA class I status of the patients, according to their known ligand specificity. For <em>KIR3DS1,</em> we considered the HLA–B genotype, as determined by the <em>Bw4I80</em> allele and, for <em>KIR2DS1</em>, we considered the HLA–C genotype, as determined by the <em>C2</em> allele. No significant heterogeneity (p&gt;0.79) was found in the association between <em>KIR3DS1</em> and HL, according to the presence or absence of the <em>Bw4I80</em> allele (<a href="#pone-0000406-t004">Table 4</a>). For <em>KIR2DS1</em>, the protective effect seemed to increase with the number of <em>HLAC2</em> alleles, although the heterogeneity test was not significant (p = 0.35) (<a href="#pone-0000406-t004">Table 4</a>). The OR decreased from 0.52 [0.17–1.65] for <em>C1/C1</em> homozygotes to 0.37 [0.15–0.90] for subjects with at least one <em>C2</em> allele. Interestingly, when we considered the presence of both ligands (i.e. patients with at least one <em>Bw4I80</em> allele and one <em>C2</em> allele), the protective effect of <em>KIR2DS1</em> and <em>KIR3DS1</em> tends to be stronger with an OR of 0.21 [0.04–1.07] for both <em>KIR</em> alleles. However, the number of informative families in this latter analysis was small (seven families), precluding to draw definitive conclusion from this result. Finally, we investigated the role of some of the characteristics of EBV-infected patients, focusing on the EBV status of the tumour (positive/negative) or EBV load in peripheral blood mononuclear cells (detectable/undetectable). No significant heterogeneity in the association between HL and <em>KIR3DS1</em> or <em>KIR2DS1</em> was found for any of these characteristics (<a href="#pone-0000406-t004">Table 4</a>). However, the protective effect seemed to be stronger in patients with EBV-positive tumours or detectable viral load. This effect was clearest for <em>KIR3DS1</em>, with an OR of 0.27 [0.08–0.87] for patients with detectable viral load (40% of patients) versus 0.61 [0.27–1.37] for those with no detectable viral load (60%).</p>
<div class="figure" id="pone-0000406-t004"><div class="img"><a name="pone-0000406-t004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000406.t004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000406" data-uri="info:doi/10.1371/journal.pone.0000406.t004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000406.t004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000406.t004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000406.t004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000406.t004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000406.t004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000406.t004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000406.t004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000406.t004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000406.t004.TIF"></span>)
                </a></li></ul></div><p><strong>Table 4.  <span>Effect of <em>KIR3DS1</em> and <em>KIR2DS1</em> stratified on HL patients characteristics</span></strong></p><span>doi:10.1371/journal.pone.0000406.t004</span></div><a id="article1.body1.sec2.sec3.p4" name="article1.body1.sec2.sec3.p4"></a><p>We tried to replicate these results by comparing KIR phenotype frequencies in the independent sample from the ENGELA study consisting of 68 HL patients and 60 controls. In this sample, no significant association was observed between any KIR phenotype and HL. Further analyses stratified for age (e.g. restricted to the 15–35 age class), HLA class I genotype, and the EBV status of tumour tissues identified no HL subgroup associated with <em>KIR3DS1</em> or <em>KIR2DS1</em>. In this sample, the frequencies of positive phenotypes for KIR3DS1 and KIR2DS1 among HL cases were 43% (29/68) and 41% (28/68), respectively. These frequencies were higher than for the 84 index HL cases of the familial study (33% for both KIR phenotypes), although these differences were not significant (p = 0.27 and 0.24 respectively).</p>

</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p>We report here the first family-based association study to show an association between a disease and genes of the <em>KIR</em> cluster. Familial studies have been conducted to determine the structure of the <em>KIR</em> region <a href="#pone.0000406-Hsu1">[26]</a>, <a href="#pone.0000406-Halfpenny1">[36]</a>, but the previously reported associations between the presence/absence of <em>KIRs</em> and various disease phenotypes were identified in case/control studies <a href="#pone.0000406-Bashirova1">[25]</a>. Family-based association studies have important advantages over case/control studies in that they can make use of full genotypic information at <em>KIR</em> loci (number of gene copies) and make it possible to avoid the possible confounding of gene-phenotype associations due to inappropriately chosen controls or population admixture. A first phase of genotypic reconstruction is required, based principally on a combination of information from the familial data. Published data were required for the full determination of only a minority of genotypes (for <em>KIR2DL5, KIR2DS3</em> and <em>KIR2DS5</em>), and <em>KIR2DL1</em> was the only locus that could not be reconstructed, due to its high frequency. We propose a method of <em>KIR</em> genotypic reconstruction in familial samples that could be applied to other family-based studies, as already demonstrated for a sample of African origin (S.R. <em>et al</em>., unpublished data).</p>
<a id="article1.body1.sec3.p2" name="article1.body1.sec3.p2"></a><p>Host genetic factors are known to be important in HL pathogenesis <a href="#pone.0000406-Grufferman1">[4]</a>–<a href="#pone.0000406-Mack1">[7]</a>, <a href="#pone.0000406-Goldin1">[11]</a>, and many association studies have suggested a possible association between HL and HLA genes <a href="#pone.0000406-Hors1">[8]</a>–<a href="#pone.0000406-Diepstra1">[10]</a>. Given the strong link between EBV and HL and the role of HLA in the presentation of viral antigens, EBV specificities in the role of HLA genes in HL have recently been investigated. An association was found between EBV-positive HL and markers located in the HLA class I region, whereas EBV-negative HL was found to be associated with markers of the HLA class III region <a href="#pone.0000406-Diepstra1">[10]</a>. However, no consensus has yet emerged on the role of any specific HLA molecule <a href="#pone.0000406-Hors1">[8]</a>–<a href="#pone.0000406-Diepstra1">[10]</a>. KIRs were considered as candidate genes for HL predisposition in this study since they are alternative receptors for HLA class I molecules.</p>
<a id="article1.body1.sec3.p3" name="article1.body1.sec3.p3"></a><p>Our family-based analysis provides the first evidence for an association between HL and certain <em>KIR</em> alleles in strong linkage disequilibrium. Refined haplotype analysis results were consistent with a dominant protective effect of <em>KIR3DS1</em> and/or <em>KIR2DS1</em> against HL. The validity of our familial analysis is also strengthened by the observation of random transmission of the <em>KIR</em> alleles considered in the 100 unaffected children of our sample (i.e. the healthy siblings of HL patients). We tried to replicate this association, using a case/control sample collected in the context of a French survey on adult lymphoma and including 68 HL patients. No significant association between HL and the presence/absence of any <em>KIRs</em> was found. The HL patients did not differ between the two samples in terms of ethnic origin and clinical features. EBV viral load in peripheral blood mononuclear cells was not determined in the case/control study, whereas we found that the protective effect of <em>KIR3DS1</em> and/or <em>KIR2DS1</em> in familial sample tended to be stronger in patients with detectable viral load. We also restricted the analysis to the 34 patients belonging to the same age class (15–35 years) as our familial patients, corresponding to the first peak of HL incidence in developed countries, but no significant association was observed in this small sample. In any case, a larger sample is certainly required for a more powerful replication study.</p>
<a id="article1.body1.sec3.p4" name="article1.body1.sec3.p4"></a><p>Due to strong linkage disequilibrium, it was almost impossible to distinguish between the effects of <em>KIR2DS1</em> and <em>KIR3DS1</em>. The observed trend for the protective effect of <em>KIR2DS1</em> to increase in the presence of its putative ligand, HLA-C2, may be more consistent with a role for this receptor although the effectives of each strata are small. We cannot exclude the possibility that both these KIRs are responsible for the observed protective effect, as different cognate KIR-HLA class I pairs may have the same biological effects, since, for example, all activating KIR-S share the same KARAP/DAP12-dependent signalling pathways <a href="#pone.0000406-Vivier1">[37]</a>. This hypothesis is supported by the trend toward a stronger protective effect in presence of at least one allele of each HLA class I ligands of <em>KIR3DS1</em> and <em>KIR2DS1</em>, and the even greater trend in presence of both ligands. Our results are consistent with those of a study showing the protective effect of <em>KIR3DS1</em> against HCV-related hepatocarcinoma <a href="#pone.0000406-LopezVazquez1">[29]</a>, and with the general idea that activating KIRs are beneficial for the response to infectious diseases and tumours. However, immune activation may not be beneficial at all stages of these disorders and could, for example, increase the risk of other tumours associated with local inflammation <a href="#pone.0000406-Bashirova1">[25]</a>. This may account for two other reports of activating KIRs being associated with an increase in the risk of virus-related localised cancers, such as NPC <a href="#pone.0000406-ButschKovacic1">[31]</a> and cervical cancer <a href="#pone.0000406-Carrington1">[30]</a>.</p>
<a id="article1.body1.sec3.p5" name="article1.body1.sec3.p5"></a><p>We observed an interesting trend towards a stronger protective effect of <em>KIR2DS1</em> and/or <em>KIR3DS1</em> in the presence of EBV, detected in either HL tumour cells or in peripheral mononuclear blood cells. This suggests that the protective role of activating KIRs may be more pronounced in HL cases associated with EBV. NK cells were recently shown to limit lytic EBV infection through the down-regulation of MHC class I molecules that bind to inhibitory KIRs, and up-regulation of activating ligands such as ULBP-1 and CD112 <a href="#pone.0000406-Pappworth1">[38]</a>. An indirect control of virus replication by cytokines produced by NK cells may also be involved in this protective effect, as shown for mouse cytomegalovirus <a href="#pone.0000406-Loh1">[39]</a>. We favor another model for the potential role of NK cells in the control of HL via activating KIRs. The malignant Hodgkin/Reed-Sternberg (HRS) cells of EBV-associated HL display high-level expression of MHC class I molecules <a href="#pone.0000406-Gandhi1">[12]</a>, <a href="#pone.0000406-Thomas1">[13]</a>, <a href="#pone.0000406-Oudejans1">[41]</a>. The up-regulation of HLA class I-peptide complexes in HRS cells would then trigger NK cell effector functions via activating KIRs, leading to the participation of NK cells in the control of HL. This hypothesis is consistent with previous biological findings as <em>KIR2DS1</em> has been shown to recognize peptide-HLA class I complexes that are up-regulated on EBV human infected B cells <a href="#pone.0000406-Stewart1">[27]</a>. Similarly, the control of mouse cytomegalovirus infection in certain strains of mice involves the recognition of peptide-MHC class I complexes through the activation of Ly49 receptors <a href="#pone.0000406-Desrosiers1">[40]</a>. To further confirm our hypothesis, it will be of particular interest to follow the activation of NK and T cell subsets according to the presence of <em>KIR2DS1</em> and/or <em>KIR3DS1</em> when the detection of these receptors will be possible. In any case, the recognition of HL cells by KIRs warrants further investigation, and we are currently carrying out an immunohistological study, comparing the presence of NK cells in EBV-positive and EBV-negative HL specimens.</p>
</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Methods"></a><h3>Methods</h3>
<h4>Study population</h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p>HL index cases were recruited from three haematology units in the Paris area. Patients were included at diagnosis (n = 21) or whilst in complete remission, at least one year after diagnosis (n = 69). Inclusion criteria were age between 15 and 35 years and negative serological tests for HIV at diagnosis. Tumour biopsies were reviewed by expert haematological pathologists (JB, DC, MR). Intratumoral EBV expression was assessed by mRNA <em>in situ</em> hybridisation for Epstein-Barr virus-encoded small non polyadenylated RNA (EBER1) sequences or by indirect immunofluorescence staining for latent membrane protein 1 (LMP1). First-degree relatives (parents and siblings) of the patients were asked to participate in the study. All participating subjects were interviewed to determine their medical history, and all gave a blood sample for DNA extraction and EBV measurements. EBV viral load was measured in peripheral blood mononuclear cells, as previously described <a href="#pone.0000406-Besson1">[42]</a>. All recruited families (n = 90) included the index HL case and at least two other individuals (either both parents or one parent and one unaffected sibling). Five of the 90 families were multiplex (three families with two affected siblings, and two families with an affected parent and an affected child).</p>
<a id="article1.body1.sec4.sec1.p2" name="article1.body1.sec4.sec1.p2"></a><p>In addition to this family sample, we also studied a case-control sample recruited in the framework of the ENGELA (<em>Environnement et Génétique des Lymphomes de l'Adulte</em>) study on adult lymphoma. This study was conducted in six French cities (Brest, Bordeaux, Caen, Lille, Nantes and Toulouse), and included newly diagnosed HIV-negative HL cases confirmed by histology. Controls and cases were matched for centre, sex, age (±3 years) and ethnic origin, with the controls hospitalised or consulting at the same hospitals as the cases, but for reasons other than cancer, occupational diseases or injuries, and tobacco-or alcohol-related diseases. DNA samples were available for 68 Caucasian cases (52 with nodular sclerosis, 11 with mixed cellularity, 5 with HL of another or an unspecified type), including 40 men and 28 women, and 60 Caucasian controls (36 men and 24 women), mostly from orthopaedic departments. The subjects included were aged 18 to 71 years, and 32 cases and 23 controls were less than 35 years old. The EBV status of tumour tissue was available for 46 cases, 13 (28%) of which were positive.</p>
<a id="article1.body1.sec4.sec1.p3" name="article1.body1.sec4.sec1.p3"></a><p>Both studies were approved by the French Consultative Committee for Protecting Persons in Biomedical Research (CCPPRB) of Paris Necker and Kremlin-Bicêtre, respectively. Human experimentation guidelines were followed in the conduct of the research. Informed consent was obtained from adults or from parents of minors.</p>


<h4>Molecular methods</h4>
<h5>KIR gene content.</h5><a id="article1.body1.sec4.sec2.sec1.p1" name="article1.body1.sec4.sec2.sec1.p1"></a><p><em>KIR</em> gene content was determined with an updated version of the previously reported PCR-SSOP technique <a href="#pone.0000406-Crum1">[43]</a>, which consists in oligonucleotide primers and probes selected from the first <em>KIR</em> gene typing system, as described in <a href="#pone.0000406-Middleton1">[44]</a>. Briefly, we used two separate locus-specific PCR to amplify the D1, D2 and transmembrane/cytoplasmic tail regions. The D1 and D2 regions were amplified together, with primers flanking exons 4 and 5; 5′AGAGAXXGTCATCCTGCA(A/G)TGTTGGTC 3′ (exon 4, nucleotides 56-82) and 5′CTCACCTXTGAC(G/A)GAAACAAGCAGTGG 3′ (intron 5-273 exon 5). The transmembrane/cytoplasmic tail region was amplified as previously described. PCR products were blotted onto nylon membranes and hybridised with 19 digoxigenin-labelled sequence-specific oligonucleotide probes, using hybridisation and chemiluminescence procedures, as previously described. Internal control samples, identified with a unique probe pattern, were also included in each experiment.</p>

<h5>Additional KIR allele typing.</h5><a id="article1.body1.sec4.sec2.sec2.p1" name="article1.body1.sec4.sec2.sec2.p1"></a><p><em>KIR2DL4, KIR2DS4</em> and <em>KIR3DS1/3DL1</em> alleles were identified using previously described SSOP techniques, based on a gene-specific PCR product <a href="#pone.0000406-Halfpenny1">[36]</a>, <a href="#pone.0000406-Williams2">[45]</a>, <a href="#pone.0000406-Maxwell2">[46]</a>. <em>KIR2DL4</em> allele typing consists involves the PCR amplification of a 1440 bp region of <em>KIR2DL4</em> exons 3/5 (D0/D2 domains), followed by hybridisation, using 16 digoxigenin-labelled probes for allele discrimination. The <em>KIR2DS4</em> allele typing system is based on the specific amplification of the 199/221bp region of <em>KIR2DS4</em> exon 5 (D2 domain) and 11 probes were used for allele identification. Finally, the <em>KIR3DS1/3DL1</em> allele typing strategy uses three separate specific amplifications of the extracellular D0 (exons2/3), D1 and D2 domains (exons 4/5) and a section of the cytoplasmic tail (exons 8/9), respectively. The third amplification concerned a single region of <em>KIR3DS1</em> only, and was required for discrimination between certain <em>KIR3DS1</em> alleles. The amplified DNA was subjected to SSOP typing, using a total of 28 oligonucleotide probes, as previously reported. In each system, a set of nine internal control samples with a unique pattern of probe reactivity was included among the samples investigated. Computational analysis was carried out on the data obtained with the various probes, to identify the alleles of each gene present.</p>

<h5>HLA class I KIR ligand typing.</h5><a id="article1.body1.sec4.sec2.sec3.p1" name="article1.body1.sec4.sec2.sec3.p1"></a><p>The study was focused on the identification of HLA-Bw4 and HLA-C1/C2 KIR ligands, that were typed using updated methods, as described in <a href="#pone.0000406-Mndez1">[47]</a>. Briefly, we used four probes (BL20, BL21, BL22, BL23) to define HLA-Bw4, with the same PCR conditions as the previously report typing system for HLA–B <a href="#pone.0000406-Middleton2">[48]</a>.&nbsp;These probes are able to distinguish threonine or isoleucine present at position 80 of the HLA-Bw4 molecules. Similarly, HLA-C1 and C2 groups were defined using only two probes from the full HLA–C allele typing system <a href="#pone.0000406-Williams3">[49]</a>, which differentiated nucleotides at positions 77 and 80. These probes were C293 for C1 group and C291 for C2 group.</p>



<h4>Statistical analysis</h4>
<a id="article1.body1.sec4.sec3.p1" name="article1.body1.sec4.sec3.p1"></a><p>Hardy-Weinberg equilibrium was tested after genotypic reconstruction for each <em>KIR</em> gene, using all the parents of the families. Measures of pairwise linkage disequilibrium (LD) between <em>KIR</em> genes, such as D' and r<sup>2</sup> <a href="#pone.0000406-Wall1">[50]</a>, were estimated using Haploview (<a href="http://www.broad.mit.edu/personl/jcbarret/haplo/">http://www.broad.mit.edu/personl/jcbarre​t/haplo/</a>). |D'| and r<sup>2</sup> vary between 0 and 1. Complete LD in a pair, defined as |D'| = 1, is obtained when one of the four possible haplotypes is not observed. Perfect LD between two KIR genes, defined as r<sup>2</sup> = 1, is reached when two of the four possible haplotypes are not observed, i.e |D'| = 1 and the frequencies of the (+,−) alleles of the two <em>KIR</em> genes are equal.</p>
<a id="article1.body1.sec4.sec3.p2" name="article1.body1.sec4.sec3.p2"></a><p>The main association study was conducted on the family sample, using specific methods developed in this context. Family-based association studies avoid the possible confounding of gene-phenotype associations due to inappropriately chosen controls that can occur in case/control studies. The general principle of these studies is to search for a distortion of the transmission of alleles from parents to affected offspring, by means of the “transmission disequilibrium test” (TDT) <a href="#pone.0000406-Spielman1">[51]</a>. Families with one missing parent (30/84 in our sample) can be analysed either by reconstructing parental genotypes from children (RC-TDT) <a href="#pone.0000406-Knapp1">[52]</a>, or by using unaffected siblings as controls (Sib-TDT) <a href="#pone.0000406-Spielman2">[53]</a>. The family-based association study was carried out mainly according to the method implemented in the FBAT program <a href="#pone.0000406-Horvath1">[54]</a>, which combines these three different methods (TDT, RC-TDT, and Sib-TDT). This method also allows the use of an empirical variance-covariance estimator for the statistic, which is consistent when sibling marker genotypes are correlated (e.g. when the analysis includes multiple affected sibs) <a href="#pone.0000406-Lake1">[55]</a>. Finally, alleles for which evidence of association was obtained were also analysed by conditional logistic regression, as previously described <a href="#pone.0000406-Schaid1">[56]</a>. This analysis generated odds-ratio (OR) estimates and made it possible to compare different models of dominance (recessive, dominant or additive). It also made it possible to test for heterogeneity of the sample according to a qualitative criterion (e.g. HLA class I genotypes) by performing the analysis on the whole sample, and separately on subsamples, as described elsewhere <a href="#pone.0000406-Malik1">[57]</a>. For the association study conducted on the case/control sample, KIR phenotype distributions were compared between HL patients and controls, using classical chi-squared tests. Conditional logistic regression analyses (proc PHREG) and chi-squared tests (proc FREQ) were carried out with SAS software (version 8.2, SAS Institute, Cary, NC).</p>

</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>We would like to thank J.L. Casanova, A. Alcaïs, S. Plancoulaine, and all members of the Laboratory of Human Genetics of Infectious Diseases for helpful discussions. We are grateful to C. Mulot (INSERM U490, Paris, France), P.Rincé (Hôpital Bicêtre, Le Kremlin-Bicêtre, France) and F. Zatla (Hôpital Tenon, Paris, France) for assistance with the management of biological and pathological samples and to all staff of the Northern Ireland Histocompatibility and Immunogenetics Laboratory for precious help in this study. We also thank the patients and their families for their trust and willingness to participate in this study, and R. Delarue and J. Vargaftig (Hopital Necker, Paris, France) for identifying patients. Thanks are also due to J.L. Bresson, A. Mogenet and all members of the <em>Centre d'Investigation Clinique de l'Hôpital Necker</em> (Paris, France) for the cordial reception of patients and their families.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: EV LA DM CB JC. Performed the experiments: EV JB CA JN CL MR SR FW DC. Analyzed the data: LA CB GA LO SR. Contributed reagents/materials/analysis tools: OH EV DM CB PB JB CA JN CL MR JC LO CF PC SR FW DC. Wrote the paper: EV LA DM CB SR.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0000406-Re1" id="pone.0000406-Re1"></a>Re D, Kuppers R, Diehl V (2005) Molecular pathogenesis of Hodgkin's lymphoma. J Clin Oncol  23: 6379–6386.  <ul class="find" data-citedArticleID="947666" data-doi="10.1200/jco.2005.55.013"><li><a href="http://dx.doi.org/10.1200/jco.2005.55.013" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Molecular+pathogenesis+of+Hodgkin%27s+lymphoma." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Molecular+pathogenesis+of+Hodgkin%27s+lymphoma.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0000406-Cartwright1" id="pone.0000406-Cartwright1"></a>Cartwright RA, Watkins G (2004) Epidemiology of Hodgkin's disease: a review. Hematol Oncol  22: 11–26.  <ul class="find" data-citedArticleID="947592" data-doi="10.1002/hon.723"><li><a href="http://dx.doi.org/10.1002/hon.723" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Epidemiology+of+Hodgkin%27s+disease%3A+a+review." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Epidemiology+of+Hodgkin%27s+disease%3A+a+review.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0000406-Hjalgrim1" id="pone.0000406-Hjalgrim1"></a>Hjalgrim H, Askling J, Pukkala E, Hansen S, Munksgaard L, et al.  (2001) Incidence of Hodgkin's disease in Nordic countries. Lancet  358: 297–298.  <ul class="find" data-citedArticleID="947618" data-doi="10.1016/s0140-6736(01)05498-8"><li><a href="http://dx.doi.org/10.1016/s0140-6736(01)05498-8" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Incidence+of+Hodgkin%27s+disease+in+Nordic+countries." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Incidence+of+Hodgkin%27s+disease+in+Nordic+countries.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0000406-Grufferman1" id="pone.0000406-Grufferman1"></a>Grufferman S, Cole P, Smith PG, Lukes RJ (1977) Hodgkin's disease in siblings. N Engl J Med  296: 248–250.  <ul class="find" data-citedArticleID="947612" data-doi="10.1056/nejm197702032960504"><li><a href="http://dx.doi.org/10.1056/nejm197702032960504" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Hodgkin%27s+disease+in+siblings." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Hodgkin%27s+disease+in+siblings.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0000406-Shugart1" id="pone.0000406-Shugart1"></a>Shugart YY, Hemminki K, Vaittinen P, Kingman A, Dong C (2000) A genetic study of Hodgkin's lymphoma: an estimate of heritability and anticipation based on the familial cancer database in Sweden. Hum Genet  106: 553–556.  <ul class="find" data-citedArticleID="947670" data-doi="10.1007/s004390000291"><li><a href="http://dx.doi.org/10.1007/s004390000291" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+genetic+study+of+Hodgkin%27s+lymphoma%3A+an+estimate+of+heritability+and+anticipation+based+on+the+familial+cancer+database+in+Sweden." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+genetic+study+of+Hodgkin%27s+lymphoma%3A+an+estimate+of+heritability+and+anticipation+based+on+the+familial+cancer+database+in+Sweden.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0000406-Ferraris1" id="pone.0000406-Ferraris1"></a>Ferraris AM, Racchi O, Rapezzi D, Gaetani GF, Boffetta P (1997) Familial Hodgkin's disease: a disease of young adulthood? Ann Hematol  74: 131–134.  <ul class="find" data-citedArticleID="947604" data-doi="10.1007/s002770050270"><li><a href="http://dx.doi.org/10.1007/s002770050270" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Familial+Hodgkin%27s+disease%3A+a+disease+of+young+adulthood%3F" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Familial+Hodgkin%27s+disease%3A+a+disease+of+young+adulthood%3F%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0000406-Mack1" id="pone.0000406-Mack1"></a>Mack TM, Cozen W, Shibata DK, Weiss LM, Nathwani BN, et al.  (1995) Concordance for Hodgkin's disease in identical twins suggesting genetic susceptibility to the young-adult form of the disease. N Engl J Med  332: 413–418.  <ul class="find" data-citedArticleID="947642" data-doi="10.1056/nejm199502163320701"><li><a href="http://dx.doi.org/10.1056/nejm199502163320701" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Concordance+for+Hodgkin%27s+disease+in+identical+twins+suggesting+genetic+susceptibility+to+the+young-adult+form+of+the+disease." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Concordance+for+Hodgkin%27s+disease+in+identical+twins+suggesting+genetic+susceptibility+to+the+young-adult+form+of+the+disease.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0000406-Hors1" id="pone.0000406-Hors1"></a>Hors J, Dausset J (1983) HLA and susceptibility to Hodgkin's disease. Immunol Rev  70: 167–192.  <ul class="find" data-citedArticleID="947622" data-doi="10.1111/j.1600-065x.1983.tb00714.x"><li><a href="http://dx.doi.org/10.1111/j.1600-065x.1983.tb00714.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=HLA+and+susceptibility+to+Hodgkin%27s+disease." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22HLA+and+susceptibility+to+Hodgkin%27s+disease.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0000406-Harty1" id="pone.0000406-Harty1"></a>Harty LC, Lin AY, Goldstein AM, Jaffe ES, Carrington M, et al.  (2002) HLA-DR, HLA-DQ, and TAP genes in familial Hodgkin disease. Blood  99: 690–693.  <ul class="find" data-citedArticleID="947616" data-doi="10.1182/blood.v99.2.690"><li><a href="http://dx.doi.org/10.1182/blood.v99.2.690" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=HLA-DR%2C+HLA-DQ%2C+and+TAP+genes+in+familial+Hodgkin+disease." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22HLA-DR%2C+HLA-DQ%2C+and+TAP+genes+in+familial+Hodgkin+disease.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0000406-Diepstra1" id="pone.0000406-Diepstra1"></a>Diepstra A, Niens M, Vellenga E, van Imhoff GW, Nolte IM, et al.  (2005) Association with HLA class I in Epstein-Barr-virus-positive and with HLA class III in Epstein-Barr-virus-negative Hodgkin's lymphoma. Lancet  365: 2216–2224.  <ul class="find" data-citedArticleID="947600" data-doi="10.1016/s0140-6736(05)66780-3"><li><a href="http://dx.doi.org/10.1016/s0140-6736(05)66780-3" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Association+with+HLA+class+I+in+Epstein-Barr-virus-positive+and+with+HLA+class+III+in+Epstein-Barr-virus-negative+Hodgkin%27s+lymphoma." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Association+with+HLA+class+I+in+Epstein-Barr-virus-positive+and+with+HLA+class+III+in+Epstein-Barr-virus-negative+Hodgkin%27s+lymphoma.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0000406-Goldin1" id="pone.0000406-Goldin1"></a>Goldin LR, McMaster ML, Ter-Minassian M, Saddlemire S, Harmsen B, et al.  (2005) A genome screen of families at high risk for Hodgkin lymphoma: evidence for a susceptibility gene on chromosome 4. J Med Genet  42: 595–601.  <ul class="find" data-citedArticleID="947610" data-doi="10.1136/jmg.2004.027433"><li><a href="http://dx.doi.org/10.1136/jmg.2004.027433" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+genome+screen+of+families+at+high+risk+for+Hodgkin+lymphoma%3A+evidence+for+a+susceptibility+gene+on+chromosome+4." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+genome+screen+of+families+at+high+risk+for+Hodgkin+lymphoma%3A+evidence+for+a+susceptibility+gene+on+chromosome+4.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0000406-Gandhi1" id="pone.0000406-Gandhi1"></a>Gandhi MK, Tellam JT, Khanna R (2004) Epstein-Barr virus-associated Hodgkin's lymphoma. Br J Haematol  125: 267–281.  <ul class="find" data-citedArticleID="947608" data-doi="10.1111/j.1365-2141.2004.04902.x"><li><a href="http://dx.doi.org/10.1111/j.1365-2141.2004.04902.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Epstein-Barr+virus-associated+Hodgkin%27s+lymphoma." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Epstein-Barr+virus-associated+Hodgkin%27s+lymphoma.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0000406-Thomas1" id="pone.0000406-Thomas1"></a>Thomas RK, Re D, Wolf J, Diehl V (2004) Part I: Hodgkin's lymphoma–molecular biology of Hodgkin and Reed-Sternberg cells. Lancet Oncol  5: 11–18.  <ul class="find" data-citedArticleID="947678" data-doi="10.1016/s1470-2045(03)01319-6"><li><a href="http://dx.doi.org/10.1016/s1470-2045(03)01319-6" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Part+I%3A+Hodgkin%27s+lymphoma%26%23x2013%3Bmolecular+biology+of+Hodgkin+and+Reed-Sternberg+cells." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Part+I%3A+Hodgkin%27s+lymphoma%26%23x2013%3Bmolecular+biology+of+Hodgkin+and+Reed-Sternberg+cells.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0000406-ThorleyLawson1" id="pone.0000406-ThorleyLawson1"></a>Thorley-Lawson DA, Gross A (2004) Persistence of the Epstein-Barr virus and the origins of associated lymphomas. N Engl J Med  350: 1328–1337.  <ul class="find" data-citedArticleID="947680" data-doi="10.1056/nejmra032015"><li><a href="http://dx.doi.org/10.1056/nejmra032015" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Persistence+of+the+Epstein-Barr+virus+and+the+origins+of+associated+lymphomas." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Persistence+of+the+Epstein-Barr+virus+and+the+origins+of+associated+lymphomas.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0000406-Hjalgrim2" id="pone.0000406-Hjalgrim2"></a>Hjalgrim H, Askling J, Rostgaard K, Hamilton-Dutoit S, Frisch M, et al.  (2003) Characteristics of Hodgkin's lymphoma after infectious mononucleosis. N Engl J Med  349: 1324–1332.  <ul class="find" data-citedArticleID="947620" data-doi="10.1056/nejmoa023141"><li><a href="http://dx.doi.org/10.1056/nejmoa023141" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Characteristics+of+Hodgkin%27s+lymphoma+after+infectious+mononucleosis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Characteristics+of+Hodgkin%27s+lymphoma+after+infectious+mononucleosis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0000406-Mueller1" id="pone.0000406-Mueller1"></a>Mueller N, Evans A, Harris NL, Comstock GW, Jellum E, et al.  (1989) Hodgkin's disease and Epstein-Barr virus. Altered antibody pattern before diagnosis. N Engl J Med  320: 689–695.  <ul class="find" data-citedArticleID="947658" data-doi="10.1056/nejm198903163201103"><li><a href="http://dx.doi.org/10.1056/nejm198903163201103" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Hodgkin%27s+disease+and+Epstein-Barr+virus.+Altered+antibody+pattern+before+diagnosis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Hodgkin%27s+disease+and+Epstein-Barr+virus.+Altered+antibody+pattern+before+diagnosis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0000406-French1" id="pone.0000406-French1"></a>French AR, Yokoyama WM (2003) Natural killer cells and viral infections. Curr Opin Immunol  15: 45–51.  <ul class="find" data-citedArticleID="947606" data-doi="10.1016/s095279150200002x"><li><a href="http://dx.doi.org/10.1016/s095279150200002x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Natural+killer+cells+and+viral+infections." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Natural+killer+cells+and+viral+infections.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0000406-Lodoen1" id="pone.0000406-Lodoen1"></a>Lodoen MB, Lanier LL (2006) Natural killer cells as an initial defense against pathogens. Curr Opin Immunol  18: 391–398.  <ul class="find" data-citedArticleID="947636" data-doi="10.1016/j.coi.2006.05.002"><li><a href="http://dx.doi.org/10.1016/j.coi.2006.05.002" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Natural+killer+cells+as+an+initial+defense+against+pathogens." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Natural+killer+cells+as+an+initial+defense+against+pathogens.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0000406-Callan1" id="pone.0000406-Callan1"></a>Callan MF (2004) The immune response to Epstein-Barr virus. Microbes Infect  6: 937–945.  <ul class="find" data-citedArticleID="947588" data-doi="10.1016/j.micinf.2004.04.014"><li><a href="http://dx.doi.org/10.1016/j.micinf.2004.04.014" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+immune+response+to+Epstein-Barr+virus." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+immune+response+to+Epstein-Barr+virus.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0000406-Williams1" id="pone.0000406-Williams1"></a>Williams H, McAulay K, Macsween KF, Gallacher NJ, Higgins CD, et al.  (2005) The immune response to primary EBV infection: a role for natural killer cells. Br J Haematol  129: 266–274.  <ul class="find" data-citedArticleID="947690" data-doi="10.1111/j.1365-2141.2005.05452.x"><li><a href="http://dx.doi.org/10.1111/j.1365-2141.2005.05452.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+immune+response+to+primary+EBV+infection%3A+a+role+for+natural+killer+cells." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+immune+response+to+primary+EBV+infection%3A+a+role+for+natural+killer+cells.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0000406-Eidenschenk1" id="pone.0000406-Eidenschenk1"></a>Eidenschenk C, Dunne J, Jouanguy E, Fourlinnie C, Gineau L, et al.  (2006) A novel primary immunodeficiency with specific natural-killer cell deficiency maps to the centromeric region of chromosome 8. Am J Hum Genet  78: 721–727.  <ul class="find" data-citedArticleID="947602" data-doi="10.1086/503269"><li><a href="http://dx.doi.org/10.1086/503269" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+novel+primary+immunodeficiency+with+specific+natural-killer+cell+deficiency+maps+to+the+centromeric+region+of+chromosome+8." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+novel+primary+immunodeficiency+with+specific+natural-killer+cell+deficiency+maps+to+the+centromeric+region+of+chromosome+8.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0000406-Lee1" id="pone.0000406-Lee1"></a>Lee SH, Girard S, Macina D, Busa M, Zafer A, et al.  (2001) Susceptibility to mouse cytomegalovirus is associated with deletion of an activating natural killer cell receptor of the C-type lectin superfamily. Nat Genet  28: 42–45.  <ul class="find" data-citedArticleID="947634" data-doi="10.1038/ng0501-42"><li><a href="http://dx.doi.org/10.1038/ng0501-42" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Susceptibility+to+mouse+cytomegalovirus+is+associated+with+deletion+of+an+activating+natural+killer+cell+receptor+of+the+C-type+lectin+superfamily." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Susceptibility+to+mouse+cytomegalovirus+is+associated+with+deletion+of+an+activating+natural+killer+cell+receptor+of+the+C-type+lectin+superfamily.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0000406-Arase1" id="pone.0000406-Arase1"></a>Arase H, Mocarski ES, Campbell AE, Hill AB, Lanier LL (2002) Direct recognition of cytomegalovirus by activating and inhibitory NK cell receptors. Science  296: 1323–1326.  <ul class="find" data-citedArticleID="947580" data-doi="10.1126/science.1070884"><li><a href="http://dx.doi.org/10.1126/science.1070884" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Direct+recognition+of+cytomegalovirus+by+activating+and+inhibitory+NK+cell+receptors." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Direct+recognition+of+cytomegalovirus+by+activating+and+inhibitory+NK+cell+receptors.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0000406-Vilches1" id="pone.0000406-Vilches1"></a>Vilches C, Parham P (2002) KIR: diverse, rapidly evolving receptors of innate and adaptive immunity. Annu Rev Immunol  20: 217–251.  <ul class="find" data-citedArticleID="947684" data-doi="10.1146/annurev.immunol.20.092501.134942"><li><a href="http://dx.doi.org/10.1146/annurev.immunol.20.092501.134942" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=KIR%3A+diverse%2C+rapidly+evolving+receptors+of+innate+and+adaptive+immunity." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22KIR%3A+diverse%2C+rapidly+evolving+receptors+of+innate+and+adaptive+immunity.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0000406-Bashirova1" id="pone.0000406-Bashirova1"></a>Bashirova AA, Martin MP, McVicar DW, Carrington M (2006) The Killer Immunoglobulin-like Receptor Gene Cluster: Tuning the Genome for Defense. Annu Rev Genomics Hum Genet.   <ul class="find" data-citedArticleID="947582" data-doi="10.1146/annurev.genom.7.080505.115726"><li><a href="http://dx.doi.org/10.1146/annurev.genom.7.080505.115726" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+Killer+Immunoglobulin-like+Receptor+Gene+Cluster%3A+Tuning+the+Genome+for+Defense." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+Killer+Immunoglobulin-like+Receptor+Gene+Cluster%3A+Tuning+the+Genome+for+Defense.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0000406-Hsu1" id="pone.0000406-Hsu1"></a>Hsu KC, Chida S, Geraghty DE, Dupont B (2002) The killer cell immunoglobulin-like receptor (KIR) genomic region: gene-order, haplotypes and allelic polymorphism. Immunol Rev  190: 40–52.  <ul class="find" data-citedArticleID="947626" data-doi="10.1034/j.1600-065x.2002.19004.x"><li><a href="http://dx.doi.org/10.1034/j.1600-065x.2002.19004.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+killer+cell+immunoglobulin-like+receptor+%28KIR%29+genomic+region%3A+gene-order%2C+haplotypes+and+allelic+polymorphism." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+killer+cell+immunoglobulin-like+receptor+%28KIR%29+genomic+region%3A+gene-order%2C+haplotypes+and+allelic+polymorphism.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0000406-Stewart1" id="pone.0000406-Stewart1"></a>Stewart CA, Laugier-Anfossi F, Vely F, Saulquin X, Riedmuller J, et al.  (2005) Recognition of peptide-MHC class I complexes by activating killer immunoglobulin-like receptors. Proc Natl Acad Sci U S A  102: 13224–13229.  <ul class="find" data-citedArticleID="947676" data-doi="10.1073/pnas.0503594102"><li><a href="http://dx.doi.org/10.1073/pnas.0503594102" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Recognition+of+peptide-MHC+class+I+complexes+by+activating+killer+immunoglobulin-like+receptors." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Recognition+of+peptide-MHC+class+I+complexes+by+activating+killer+immunoglobulin-like+receptors.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0000406-Martin1" id="pone.0000406-Martin1"></a>Martin MP, Gao X, Lee JH, Nelson GW, Detels R, et al.  (2002) Epistatic interaction between KIR3DS1 and HLA-B delays the progression to AIDS. Nat Genet  31: 429–434.  <ul class="find" data-citedArticleID="947646" data-doi="10.1038/ng934"><li><a href="http://dx.doi.org/10.1038/ng934" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Epistatic+interaction+between+KIR3DS1+and+HLA-B+delays+the+progression+to+AIDS." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Epistatic+interaction+between+KIR3DS1+and+HLA-B+delays+the+progression+to+AIDS.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0000406-LopezVazquez1" id="pone.0000406-LopezVazquez1"></a>Lopez-Vazquez A, Rodrigo L, Martinez-Borra J, Perez R, Rodriguez M, et al.  (2005) Protective effect of the HLA-Bw4I80 epitope and the killer cell immunoglobulin-like receptor 3DS1 gene against the development of hepatocellular carcinoma in patients with hepatitis C virus infection. J Infect Dis  192: 162–165.  <ul class="find" data-citedArticleID="947640" data-doi="10.1086/430351"><li><a href="http://dx.doi.org/10.1086/430351" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Protective+effect+of+the+HLA-Bw4I80+epitope+and+the+killer+cell+immunoglobulin-like+receptor+3DS1+gene+against+the+development+of+hepatocellular+carcinoma+in+patients+with+hepatitis+C+virus+infection." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Protective+effect+of+the+HLA-Bw4I80+epitope+and+the+killer+cell+immunoglobulin-like+receptor+3DS1+gene+against+the+development+of+hepatocellular+carcinoma+in+patients+with+hepatitis+C+virus+infection.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0000406-Carrington1" id="pone.0000406-Carrington1"></a>Carrington M, Wang S, Martin MP, Gao X, Schiffman M, et al.  (2005) Hierarchy of resistance to cervical neoplasia mediated by combinations of killer immunoglobulin-like receptor and human leukocyte antigen loci. J Exp Med  201: 1069–1075.  <ul class="find" data-citedArticleID="947590" data-doi="10.1084/jem.20042158"><li><a href="http://dx.doi.org/10.1084/jem.20042158" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Hierarchy+of+resistance+to+cervical+neoplasia+mediated+by+combinations+of+killer+immunoglobulin-like+receptor+and+human+leukocyte+antigen+loci." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Hierarchy+of+resistance+to+cervical+neoplasia+mediated+by+combinations+of+killer+immunoglobulin-like+receptor+and+human+leukocyte+antigen+loci.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0000406-ButschKovacic1" id="pone.0000406-ButschKovacic1"></a>Butsch Kovacic M, Martin M, Gao X, Fuksenko T, Chen CJ, et al.  (2005) Variation of the killer cell immunoglobulin-like receptors and HLA-C genes in nasopharyngeal carcinoma. Cancer Epidemiol Biomarkers Prev  14: 2673–2677.  <ul class="find" data-citedArticleID="947586" data-doi="10.1158/1055-9965.epi-05-0229"><li><a href="http://dx.doi.org/10.1158/1055-9965.epi-05-0229" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Variation+of+the+killer+cell+immunoglobulin-like+receptors+and+HLA-C+genes+in+nasopharyngeal+carcinoma." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Variation+of+the+killer+cell+immunoglobulin-like+receptors+and+HLA-C+genes+in+nasopharyngeal+carcinoma.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0000406-Denis1" id="pone.0000406-Denis1"></a>Denis L, Sivula J, Gourraud PA, Kerdudou N, Chout R, et al.  (2005) Genetic diversity of KIR natural killer cell markers in populations from France, Guadeloupe, Finland, Senegal and Reunion. Tissue Antigens  66: 267–276.  <ul class="find" data-citedArticleID="947596" data-doi="10.1111/j.1399-0039.2005.00473.x"><li><a href="http://dx.doi.org/10.1111/j.1399-0039.2005.00473.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genetic+diversity+of+KIR+natural+killer+cell+markers+in+populations+from+France%2C+Guadeloupe%2C+Finland%2C+Senegal+and+Reunion." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genetic+diversity+of+KIR+natural+killer+cell+markers+in+populations+from+France%2C+Guadeloupe%2C+Finland%2C+Senegal+and+Reunion.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0000406-Uhrberg1" id="pone.0000406-Uhrberg1"></a>Uhrberg M, Parham P, Wernet P (2002) Definition of gene content for nine common group B haplotypes of the Caucasoid population: KIR haplotypes contain between seven and eleven KIR genes. Immunogenetics  54: 221–229.  <ul class="find" data-citedArticleID="947682" data-doi="10.1007/s00251-002-0463-7"><li><a href="http://dx.doi.org/10.1007/s00251-002-0463-7" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Definition+of+gene+content+for+nine+common+group+B+haplotypes+of+the+Caucasoid+population%3A+KIR+haplotypes+contain+between+seven+and+eleven+KIR+genes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Definition+of+gene+content+for+nine+common+group+B+haplotypes+of+the+Caucasoid+population%3A+KIR+haplotypes+contain+between+seven+and+eleven+KIR+genes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0000406-Hsu2" id="pone.0000406-Hsu2"></a>Hsu KC, Liu XR, Selvakumar A, Mickelson E, O'Reilly RJ, et al.  (2002) Killer Ig-like receptor haplotype analysis by gene content: evidence for genomic diversity with a minimum of six basic framework haplotypes, each with multiple subsets. J Immunol  169: 5118–5129.  <ul class="find" data-citedArticleID="947628"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Killer+Ig-like+receptor+haplotype+analysis+by+gene+content%3A+evidence+for+genomic+diversity+with+a+minimum+of+six+basic+framework+haplotypes%2C+each+with+multiple+subsets.&amp;auth=&amp;atitle=Killer+Ig-like+receptor+haplotype+analysis+by+gene+content%3A+evidence+for+genomic+diversity+with+a+minimum+of+six+basic+framework+haplotypes%2C+each+with+multiple+subsets." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Killer+Ig-like+receptor+haplotype+analysis+by+gene+content%3A+evidence+for+genomic+diversity+with+a+minimum+of+six+basic+framework+haplotypes%2C+each+with+multiple+subsets." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Killer+Ig-like+receptor+haplotype+analysis+by+gene+content%3A+evidence+for+genomic+diversity+with+a+minimum+of+six+basic+framework+haplotypes%2C+each+with+multiple+subsets.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0000406-Maxwell1" id="pone.0000406-Maxwell1"></a>Maxwell LD, Wallace A, Middleton D, Curran MD (2002) A common KIR2DS4 deletion variant in the human that predicts a soluble KIR molecule analogous to the KIR1D molecule observed in the rhesus monkey. Tissue Antigens  60: 254–258.  <ul class="find" data-citedArticleID="947648" data-doi="10.1034/j.1399-0039.2002.600307.x"><li><a href="http://dx.doi.org/10.1034/j.1399-0039.2002.600307.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+common+KIR2DS4+deletion+variant+in+the+human+that+predicts+a+soluble+KIR+molecule+analogous+to+the+KIR1D+molecule+observed+in+the+rhesus+monkey." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+common+KIR2DS4+deletion+variant+in+the+human+that+predicts+a+soluble+KIR+molecule+analogous+to+the+KIR1D+molecule+observed+in+the+rhesus+monkey.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">36.
              </span><a name="pone.0000406-Halfpenny1" id="pone.0000406-Halfpenny1"></a>Halfpenny IA, Middleton D, Barnett YA, Williams F (2004) Investigation of killer cell immunoglobulin-like receptor gene diversity: IV. KIR3DL1/S1. Hum Immunol  65: 602–612.  <ul class="find" data-citedArticleID="947614" data-doi="10.1016/j.humimm.2004.03.003"><li><a href="http://dx.doi.org/10.1016/j.humimm.2004.03.003" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Investigation+of+killer+cell+immunoglobulin-like+receptor+gene+diversity%3A+IV.+KIR3DL1%2FS1." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Investigation+of+killer+cell+immunoglobulin-like+receptor+gene+diversity%3A+IV.+KIR3DL1%2FS1.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">37.
              </span><a name="pone.0000406-Vivier1" id="pone.0000406-Vivier1"></a>Vivier E, Nunes JA, Vely F (2004) Natural killer cell signaling pathways. Science  306: 1517–1519.  <ul class="find" data-citedArticleID="947686" data-doi="10.1126/science.1103478"><li><a href="http://dx.doi.org/10.1126/science.1103478" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Natural+killer+cell+signaling+pathways." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Natural+killer+cell+signaling+pathways.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">38.
              </span><a name="pone.0000406-Pappworth1" id="pone.0000406-Pappworth1"></a>Pappworth I, Wang E, Rowe M (2007) The Switch from Latent to Productive Infection in Epstein-Barr Virus-Infected B Cells Is Associated with Sensitization to NK Cell Killing. J Virol  81: 474–482.  <ul class="find" data-citedArticleID="947662" data-doi="10.1128/jvi.01777-06"><li><a href="http://dx.doi.org/10.1128/jvi.01777-06" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+Switch+from+Latent+to+Productive+Infection+in+Epstein-Barr+Virus-Infected+B+Cells+Is+Associated+with+Sensitization+to+NK+Cell+Killing." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+Switch+from+Latent+to+Productive+Infection+in+Epstein-Barr+Virus-Infected+B+Cells+Is+Associated+with+Sensitization+to+NK+Cell+Killing.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">39.
              </span><a name="pone.0000406-Loh1" id="pone.0000406-Loh1"></a>Loh J, Chu DT, O'Guin AK, Yokoyama WM, Virgin HWt (2005) Natural killer cells utilize both perforin and gamma interferon to regulate murine cytomegalovirus infection in the spleen and liver. J Virol  79: 661–667.  <ul class="find" data-citedArticleID="947638" data-doi="10.1128/jvi.79.1.661-667.2005"><li><a href="http://dx.doi.org/10.1128/jvi.79.1.661-667.2005" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Natural+killer+cells+utilize+both+perforin+and+gamma+interferon+to+regulate+murine+cytomegalovirus+infection+in+the+spleen+and+liver." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Natural+killer+cells+utilize+both+perforin+and+gamma+interferon+to+regulate+murine+cytomegalovirus+infection+in+the+spleen+and+liver.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">40.
              </span><a name="pone.0000406-Desrosiers1" id="pone.0000406-Desrosiers1"></a>Desrosiers MP, Kielczewska A, Loredo-Osti JC, Adam SG, Makrigiannis AP, et al.  (2005) Epistasis between mouse Klra and major histocompatibility complex class I loci is associated with a new mechanism of natural killer cell-mediated innate resistance to cytomegalovirus infection. Nat Genet  37: 593–599.  <ul class="find" data-citedArticleID="947598" data-doi="10.1038/ng1564"><li><a href="http://dx.doi.org/10.1038/ng1564" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Epistasis+between+mouse+Klra+and+major+histocompatibility+complex+class+I+loci+is+associated+with+a+new+mechanism+of+natural+killer+cell-mediated+innate+resistance+to+cytomegalovirus+infection." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Epistasis+between+mouse+Klra+and+major+histocompatibility+complex+class+I+loci+is+associated+with+a+new+mechanism+of+natural+killer+cell-mediated+innate+resistance+to+cytomegalovirus+infection.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">41.
              </span><a name="pone.0000406-Oudejans1" id="pone.0000406-Oudejans1"></a>Oudejans JJ, Jiwa NM, Kummer JA, Horstman A, Vos W, et al.  (1996) Analysis of major histocompatibility complex class I expression on Reed-Sternberg cells in relation to the cytotoxic T-cell response in Epstein-Barr virus-positive and -negative Hodgkin's disease. Blood  87: 3844–3851.  <ul class="find" data-citedArticleID="947660"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Analysis+of+major+histocompatibility+complex+class+I+expression+on+Reed-Sternberg+cells+in+relation+to+the+cytotoxic+T-cell+response+in+Epstein-Barr+virus-positive+and+-negative+Hodgkin%27s+disease.&amp;auth=&amp;atitle=Analysis+of+major+histocompatibility+complex+class+I+expression+on+Reed-Sternberg+cells+in+relation+to+the+cytotoxic+T-cell+response+in+Epstein-Barr+virus-positive+and+-negative+Hodgkin%27s+disease." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Analysis+of+major+histocompatibility+complex+class+I+expression+on+Reed-Sternberg+cells+in+relation+to+the+cytotoxic+T-cell+response+in+Epstein-Barr+virus-positive+and+-negative+Hodgkin%27s+disease." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Analysis+of+major+histocompatibility+complex+class+I+expression+on+Reed-Sternberg+cells+in+relation+to+the+cytotoxic+T-cell+response+in+Epstein-Barr+virus-positive+and+-negative+Hodgkin%27s+disease.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">42.
              </span><a name="pone.0000406-Besson1" id="pone.0000406-Besson1"></a>Besson C, Amiel C, Le-Pendeven C, Brice P, Ferme C, et al.  (2006) Positive correlation between Epstein-Barr virus viral load and anti-viral capsid immunoglobulin G titers determined for Hodgkin's lymphoma patients and their relatives. J Clin Microbiol  44: 47–50.  <ul class="find" data-citedArticleID="947584" data-doi="10.1128/jcm.44.1.47-50.2006"><li><a href="http://dx.doi.org/10.1128/jcm.44.1.47-50.2006" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Positive+correlation+between+Epstein-Barr+virus+viral+load+and+anti-viral+capsid+immunoglobulin+G+titers+determined+for+Hodgkin%27s+lymphoma+patients+and+their+relatives." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Positive+correlation+between+Epstein-Barr+virus+viral+load+and+anti-viral+capsid+immunoglobulin+G+titers+determined+for+Hodgkin%27s+lymphoma+patients+and+their+relatives.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">43.
              </span><a name="pone.0000406-Crum1" id="pone.0000406-Crum1"></a>Crum KA, Logue SE, Curran MD, Middleton D (2000) Development of a PCR-SSOP approach capable of defining the natural killer cell inhibitory receptor (KIR) gene sequence repertoires. Tissue Antigens  56: 313–326.  <ul class="find" data-citedArticleID="947594" data-doi="10.1034/j.1399-0039.2000.560403.x"><li><a href="http://dx.doi.org/10.1034/j.1399-0039.2000.560403.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Development+of+a+PCR-SSOP+approach+capable+of+defining+the+natural+killer+cell+inhibitory+receptor+%28KIR%29+gene+sequence+repertoires." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Development+of+a+PCR-SSOP+approach+capable+of+defining+the+natural+killer+cell+inhibitory+receptor+%28KIR%29+gene+sequence+repertoires.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">44.
              </span><a name="pone.0000406-Middleton1" id="pone.0000406-Middleton1"></a>Middleton D, Williams F, Halfpenny IA (2005) KIR genes. Transpl Immunol  14: 135–142.  <ul class="find" data-citedArticleID="947652" data-doi="10.1016/j.trim.2005.03.002"><li><a href="http://dx.doi.org/10.1016/j.trim.2005.03.002" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=KIR+genes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22KIR+genes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">45.
              </span><a name="pone.0000406-Williams2" id="pone.0000406-Williams2"></a>Williams F, Meenagh A, Sleator C, Middleton D (2004) Investigation of killer cell immunoglobulinlike receptor gene diversity: I. KIR2DL4. Hum Immunol  65: 31–38.  <ul class="find" data-citedArticleID="947692" data-doi="10.1016/j.humimm.2003.10.009"><li><a href="http://dx.doi.org/10.1016/j.humimm.2003.10.009" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Investigation+of+killer+cell+immunoglobulinlike+receptor+gene+diversity%3A+I.+KIR2DL4." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Investigation+of+killer+cell+immunoglobulinlike+receptor+gene+diversity%3A+I.+KIR2DL4.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">46.
              </span><a name="pone.0000406-Maxwell2" id="pone.0000406-Maxwell2"></a>Maxwell LD, Williams F, Gilmore P, Meenagh A, Middleton D (2004) Investigation of killer cell immunoglobulin-like receptor gene diversity: II. KIR2DS4. Hum Immunol  65: 613–621.  <ul class="find" data-citedArticleID="947650" data-doi="10.1016/j.humimm.2004.02.028"><li><a href="http://dx.doi.org/10.1016/j.humimm.2004.02.028" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Investigation+of+killer+cell+immunoglobulin-like+receptor+gene+diversity%3A+II.+KIR2DS4." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Investigation+of+killer+cell+immunoglobulin-like+receptor+gene+diversity%3A+II.+KIR2DS4.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">47.
              </span><a name="pone.0000406-Mndez1" id="pone.0000406-Mndez1"></a>Méndez AGH, Meenagh A, Contreras S, Zavaleta R, Mendoza MF, Izquierdo L, Sarmiento ME, Acosta A, Middleton D (2006) Study of KIR genes in tuberculosis patients. Tissue Antigens In Press.   <ul class="find" data-citedArticleID="947656" data-doi="10.1111/j.1399-0039.2006.00685.x"><li><a href="http://dx.doi.org/10.1111/j.1399-0039.2006.00685.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Study+of+KIR+genes+in+tuberculosis+patients." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Study+of+KIR+genes+in+tuberculosis+patients.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">48.
              </span><a name="pone.0000406-Middleton2" id="pone.0000406-Middleton2"></a>Middleton D, Williams F, Cullen C, Mallon E (1995) Modification of an HLA-B PCR-SSOP typing system leading to improved allele determination. Tissue Antigens  45: 232–236.  <ul class="find" data-citedArticleID="947654" data-doi="10.1111/j.1399-0039.1995.tb02445.x"><li><a href="http://dx.doi.org/10.1111/j.1399-0039.1995.tb02445.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Modification+of+an+HLA-B+PCR-SSOP+typing+system+leading+to+improved+allele+determination." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Modification+of+an+HLA-B+PCR-SSOP+typing+system+leading+to+improved+allele+determination.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">49.
              </span><a name="pone.0000406-Williams3" id="pone.0000406-Williams3"></a>Williams F, Meenagh A, Patterson C, Middleton D (2002) Molecular diversity of the HLA-C gene identified in a caucasian population. Hum Immunol  63: 602–613.  <ul class="find" data-citedArticleID="947694" data-doi="10.1016/s0198-8859(02)00408-1"><li><a href="http://dx.doi.org/10.1016/s0198-8859(02)00408-1" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Molecular+diversity+of+the+HLA-C+gene+identified+in+a+caucasian+population." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Molecular+diversity+of+the+HLA-C+gene+identified+in+a+caucasian+population.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">50.
              </span><a name="pone.0000406-Wall1" id="pone.0000406-Wall1"></a>Wall JD, Pritchard JK (2003) Haplotype blocks and linkage disequilibrium in the human genome. Nat Rev Genet  4: 587–597.  <ul class="find" data-citedArticleID="947688" data-doi="10.1038/nrg1123"><li><a href="http://dx.doi.org/10.1038/nrg1123" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Haplotype+blocks+and+linkage+disequilibrium+in+the+human+genome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Haplotype+blocks+and+linkage+disequilibrium+in+the+human+genome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">51.
              </span><a name="pone.0000406-Spielman1" id="pone.0000406-Spielman1"></a>Spielman RS, McGinnis RE, Ewens WJ (1993) Transmission test for linkage disequilibrium: the insulin gene region and insulin-dependent diabetes mellitus (IDDM). Am J Hum Genet  52: 506–516.  <ul class="find" data-citedArticleID="947672"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Transmission+test+for+linkage+disequilibrium%3A+the+insulin+gene+region+and+insulin-dependent+diabetes+mellitus+%28IDDM%29.&amp;auth=&amp;atitle=Transmission+test+for+linkage+disequilibrium%3A+the+insulin+gene+region+and+insulin-dependent+diabetes+mellitus+%28IDDM%29." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Transmission+test+for+linkage+disequilibrium%3A+the+insulin+gene+region+and+insulin-dependent+diabetes+mellitus+%28IDDM%29." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Transmission+test+for+linkage+disequilibrium%3A+the+insulin+gene+region+and+insulin-dependent+diabetes+mellitus+%28IDDM%29.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">52.
              </span><a name="pone.0000406-Knapp1" id="pone.0000406-Knapp1"></a>Knapp M (1999) The transmission/disequilibrium test and parental-genotype reconstruction: the reconstruction-combined transmission/disequilibrium test. Am J Hum Genet  64: 861–870.  <ul class="find" data-citedArticleID="947630" data-doi="10.1086/302285"><li><a href="http://dx.doi.org/10.1086/302285" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+transmission%2Fdisequilibrium+test+and+parental-genotype+reconstruction%3A+the+reconstruction-combined+transmission%2Fdisequilibrium+test." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+transmission%2Fdisequilibrium+test+and+parental-genotype+reconstruction%3A+the+reconstruction-combined+transmission%2Fdisequilibrium+test.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">53.
              </span><a name="pone.0000406-Spielman2" id="pone.0000406-Spielman2"></a>Spielman RS, Ewens WJ (1998) A sibship test for linkage in the presence of association: the sib transmission/disequilibrium test. Am J Hum Genet  62: 450–458.  <ul class="find" data-citedArticleID="947674" data-doi="10.1086/301714"><li><a href="http://dx.doi.org/10.1086/301714" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+sibship+test+for+linkage+in+the+presence+of+association%3A+the+sib+transmission%2Fdisequilibrium+test." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+sibship+test+for+linkage+in+the+presence+of+association%3A+the+sib+transmission%2Fdisequilibrium+test.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">54.
              </span><a name="pone.0000406-Horvath1" id="pone.0000406-Horvath1"></a>Horvath S, Xu X, Laird NM (2001) The family based association test method: strategies for studying general genotype–phenotype associations. Eur J Hum Genet  9: 301–306.  <ul class="find" data-citedArticleID="947624" data-doi="10.1038/sj.ejhg.5200625"><li><a href="http://dx.doi.org/10.1038/sj.ejhg.5200625" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+family+based+association+test+method%3A+strategies+for+studying+general+genotype%26%23x2013%3Bphenotype+associations." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+family+based+association+test+method%3A+strategies+for+studying+general+genotype%26%23x2013%3Bphenotype+associations.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">55.
              </span><a name="pone.0000406-Lake1" id="pone.0000406-Lake1"></a>Lake SL, Blacker D, Laird NM (2000) Family-based tests of association in the presence of linkage. Am J Hum Genet  67: 1515–1525.  <ul class="find" data-citedArticleID="947632" data-doi="10.1086/316895"><li><a href="http://dx.doi.org/10.1086/316895" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Family-based+tests+of+association+in+the+presence+of+linkage." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Family-based+tests+of+association+in+the+presence+of+linkage.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">56.
              </span><a name="pone.0000406-Schaid1" id="pone.0000406-Schaid1"></a>Schaid DJ, Rowland C (1998) Use of parents, sibs, and unrelated controls for detection of associations between genetic markers and disease. Am J Hum Genet  63: 1492–1506.  <ul class="find" data-citedArticleID="947668" data-doi="10.1086/302094"><li><a href="http://dx.doi.org/10.1086/302094" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Use+of+parents%2C+sibs%2C+and+unrelated+controls+for+detection+of+associations+between+genetic+markers+and+disease." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Use+of+parents%2C+sibs%2C+and+unrelated+controls+for+detection+of+associations+between+genetic+markers+and+disease.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">57.
              </span><a name="pone.0000406-Malik1" id="pone.0000406-Malik1"></a>Malik S, Abel L, Tooker H, Poon A, Simkin L, et al.  (2005) Alleles of the NRAMP1 gene are risk factors for pediatric tuberculosis disease. Proc Natl Acad Sci U S A  102: 12183–12188.  <ul class="find" data-citedArticleID="947644" data-doi="10.1073/pnas.0503368102"><li><a href="http://dx.doi.org/10.1073/pnas.0503368102" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Alleles+of+the+NRAMP1+gene+are+risk+factors+for+pediatric+tuberculosis+disease." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Alleles+of+the+NRAMP1+gene+are+risk+factors+for+pediatric+tuberculosis+disease.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">58.
              </span><a name="pone.0000406-Andre1" id="pone.0000406-Andre1"></a>Andre P, Biassoni R, Colonna M, Cosman D, Lanier LL, et al.  (2001) New nomenclature for MHC receptors. Nat Immunol.  2: 661.  <ul class="find" data-citedArticleID="947578" data-doi="10.1038/90589"><li><a href="http://dx.doi.org/10.1038/90589" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=New+nomenclature+for+MHC+receptors." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22New+nomenclature+for+MHC+receptors.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">59.
              </span><a name="pone.0000406-Parham1" id="pone.0000406-Parham1"></a>Parham P (2005) MHC class I molecules and KIRs in human history, health and survival. Nat Rev Immunol  5: 201–214.  <ul class="find" data-citedArticleID="947664" data-doi="10.1038/nri1570"><li><a href="http://dx.doi.org/10.1038/nri1570" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=MHC+class+I+molecules+and+KIRs+in+human+history%2C+health+and+survival." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22MHC+class+I+molecules+and+KIRs+in+human+history%2C+health+and+survival.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000406.XML" value="138496"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000406.PDF" value="223847"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000406.g001.PNG_L" value="37082"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000406.g001.PNG_M" value="50212"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000406.g001.PNG_S" value="4896"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000406.g001.TIF" value="75810"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000406.g001.PNG_I" value="22126"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000406.g002.PNG_L" value="2886"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000406.g002.PNG_M" value="2886"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000406.g002.PNG_S" value="4018"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000406.g002.TIF" value="26644"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000406.g002.PNG_I" value="36686"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000406.g003.PNG_L" value="10026"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000406.g003.PNG_M" value="27666"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000406.g003.PNG_S" value="3242"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000406.g003.TIF" value="23432"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000406.g003.PNG_I" value="11469"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000406.t001.PNG_L" value="75653"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000406.t001.PNG_M" value="27150"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000406.t001.PNG_S" value="2363"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000406.t001.TIF" value="343338"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000406.t001.PNG_I" value="12779"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000406.g004.PNG_L" value="51601"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000406.g004.PNG_M" value="65977"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000406.g004.PNG_S" value="2889"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000406.g004.TIF" value="60612"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000406.g004.PNG_I" value="27659"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000406.t002.PNG_L" value="125882"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000406.t002.PNG_M" value="44374"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000406.t002.PNG_S" value="2626"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000406.t002.TIF" value="605044"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000406.t002.PNG_I" value="18685"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000406.t003.PNG_L" value="57235"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000406.t003.PNG_M" value="20192"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000406.t003.PNG_S" value="2036"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000406.t003.TIF" value="322252"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000406.t003.PNG_I" value="9094"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000406.t004.PNG_L" value="159494"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000406.t004.PNG_M" value="55475"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000406.t004.PNG_S" value="3032"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000406.t004.TIF" value="875980"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000406.t004.PNG_I" value="24653"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000406&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000406" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000406&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0000406&volume=&issue=&title=Association of Killer Cell Immunoglobulin-Like Receptor Genes with Hodgkin's Lymphoma in a Familial Study&author_name=Caroline%20Besson%2C%20Sophie%20Roetynck%2C%20Fionnuala%20Williams%2C%20Laurent%20Orsi%2C%20Corinne%20Amiel%2C%20Catherine%20Lependeven%2C%20Guillemette%20Antoni%2C%20Olivier%20Hermine%2C%20Pauline%20Brice%2C%20Christophe%20Ferme%2C%20Patrice%20Carde%2C%20Danielle%20Canioni%2C%20Josette%20Bri%C3%A8re%2C%20Martine%20Raphael%2C%20Jean-Claude%20Nicolas%2C%20Jacqueline%20Clavel%2C%20Derek%20Middleton%2C%20Eric%20Vivier%2C%20Laurent%20Abel&start_page=1&end_page=10" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000406" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000406" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0000406" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000406&amp;t=Association%20of%20Killer%20Cell%20Immunoglobulin-Like%20Receptor%20Genes%20with%20Hodgkin's%20Lymphoma%20in%20a%20Familial%20Study" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000406&title=Association%20of%20Killer%20Cell%20Immunoglobulin-Like%20Receptor%20Genes%20with%20Hodgkin's%20Lymphoma%20in%20a%20Familial%20Study&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000406&amp;title=Association%20of%20Killer%20Cell%20Immunoglobulin-Like%20Receptor%20Genes%20with%20Hodgkin's%20Lymphoma%20in%20a%20Familial%20Study" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000406" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0000406&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Association of Killer Cell Immunoglobulin-Like Receptor Genes with Hodgkin\'s Lymphoma in a Familial Study';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0000406';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Association%20of%20Killer%20Cell%20Immunoglobulin-Like%20Receptor%20Genes%20with%20Hodgkin's%20Lymphoma%20in%20a%20Familial%20Study http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000406" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0000406" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">
















          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Family-based+association+studies%22" title="Search for articles in the subject area:'Family-based association studies'"><div class="flagText">Family-based association studies</div></a>
              <div data-categoryid="41455" data-articleid="23650"
                   data-categoryname="Family-based association studies"
                   class="flagImage" title="Flag 'Family-based association studies' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Genetic+loci%22" title="Search for articles in the subject area:'Genetic loci'"><div class="flagText">Genetic loci</div></a>
              <div data-categoryid="17713" data-articleid="23650"
                   data-categoryname="Genetic loci"
                   class="flagImage" title="Flag 'Genetic loci' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Haplotypes%22" title="Search for articles in the subject area:'Haplotypes'"><div class="flagText">Haplotypes</div></a>
              <div data-categoryid="19681" data-articleid="23650"
                   data-categoryname="Haplotypes"
                   class="flagImage" title="Flag 'Haplotypes' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Linkage+disequilibrium%22" title="Search for articles in the subject area:'Linkage disequilibrium'"><div class="flagText">Linkage disequilibrium</div></a>
              <div data-categoryid="42897" data-articleid="23650"
                   data-categoryname="Linkage disequilibrium"
                   class="flagImage" title="Flag 'Linkage disequilibrium' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22NK+cells%22" title="Search for articles in the subject area:'NK cells'"><div class="flagText">NK cells</div></a>
              <div data-categoryid="40367" data-articleid="23650"
                   data-categoryname="NK cells"
                   class="flagImage" title="Flag 'NK cells' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Phenotypes%22" title="Search for articles in the subject area:'Phenotypes'"><div class="flagText">Phenotypes</div></a>
              <div data-categoryid="40235" data-articleid="23650"
                   data-categoryname="Phenotypes"
                   class="flagImage" title="Flag 'Phenotypes' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Polymerase+chain+reaction%22" title="Search for articles in the subject area:'Polymerase chain reaction'"><div class="flagText">Polymerase chain reaction</div></a>
              <div data-categoryid="26251" data-articleid="23650"
                   data-categoryname="Polymerase chain reaction"
                   class="flagImage" title="Flag 'Polymerase chain reaction' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Variant+genotypes%22" title="Search for articles in the subject area:'Variant genotypes'"><div class="flagText">Variant genotypes</div></a>
              <div data-categoryid="40547" data-articleid="23650"
                   data-categoryname="Variant genotypes"
                   class="flagImage" title="Flag 'Variant genotypes' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=1091'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=6441'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=5526&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>

<div class="block sidebar-comments">
    <div class="header">
        <h3>Comments</h3>
    </div>
      <p><a href="/annotation/listThread.action?root=7851">Referee Comments: Referee 1</a><br>Posted by PLoS_ONE_Group</p>
</div>

</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
