

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Calculation of the Free Energy and Cooperativity of Protein Folding</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0000446"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0000446" />

    <meta name="citation_title" content="Calculation of the Free Energy and Cooperativity of Protein Folding"/>
    <meta itemprop="name" content="Calculation of the Free Energy and Cooperativity of Protein Folding"/>

      <meta name="citation_author" content="Alex Kentsis"/>
            <meta name="citation_author_institution" content="Department of Molecular Physiology and Biophysics, Mount Sinai School of Medicine, New York University, New York, New York, United States of America"/>
      <meta name="citation_author" content="Tatyana Gindin"/>
            <meta name="citation_author_institution" content="Department of Molecular Physiology and Biophysics, Mount Sinai School of Medicine, New York University, New York, New York, United States of America"/>
      <meta name="citation_author" content="Mihaly Mezei"/>
            <meta name="citation_author_institution" content="Department of Molecular Physiology and Biophysics, Mount Sinai School of Medicine, New York University, New York, New York, United States of America"/>
      <meta name="citation_author" content="Roman Osman"/>
            <meta name="citation_author_institution" content="Department of Molecular Physiology and Biophysics, Mount Sinai School of Medicine, New York University, New York, New York, United States of America"/>

    <meta name="citation_date" content="2007/5/16"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0000446.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e446"/>
    <meta name="citation_issue" content="5"/>
    <meta name="citation_volume" content="2"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Are there pathways for protein folding?; citation_author=C Levinthal; citation_journal_title=Extrait du Journal de Chimie Physique; citation_volume=65; citation_number=1; citation_pages=44-45; citation_date=1968; " />
      <meta name="citation_reference" content="citation_title=Molecular collapse: the rate-limiting step in two-state cytochrome c folding.; citation_author=TR Sosnick; citation_author=L Mayne; citation_author=SW Englander; citation_journal_title=Proteins; citation_volume=24; citation_number=2; citation_pages=413-426; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Is protein folding hierarchic? II. Folding intermediates and transition states.; citation_author=RL Baldwin; citation_author=GD Rose; citation_journal_title=Trends Biochem Sci; citation_volume=24; citation_number=3; citation_pages=77-83; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Is protein folding hierarchic? I. Local structure and peptide folding.; citation_author=RL Baldwin; citation_author=GD Rose; citation_journal_title=Trends Biochem Sci; citation_volume=24; citation_number=4; citation_pages=26-33; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Is there a unifying mechanism for protein folding?; citation_author=V Daggett; citation_author=AR Fersht; citation_journal_title=Trends Biochem Sci; citation_volume=28; citation_number=5; citation_pages=18-25; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=NMR determination of residual structure in a urea-denatured protein, the 434-repressor.; citation_author=D Neri; citation_author=M Billeter; citation_author=G Wider; citation_author=K Wuthrich; citation_journal_title=Science; citation_volume=257; citation_number=6; citation_pages=1559-1563; citation_date=1992; " />
      <meta name="citation_reference" content="citation_title=Structural characterization of the FK506 binding protein unfolded in urea and guanidine hydrochloride.; citation_author=TM Logan; citation_author=Y Theriault; citation_author=SW Fesik; citation_journal_title=J Mol Biol; citation_volume=236; citation_number=7; citation_pages=637-648; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=Structural and dynamic characterization of an unfolded state of poplar apo-plastocyanin formed under nondenaturing conditions.; citation_author=Y Bai; citation_author=J Chung; citation_author=HJ Dyson; citation_author=PE Wright; citation_journal_title=Protein Sci; citation_volume=10; citation_number=8; citation_pages=1056-1066; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Structural and dynamical properties of a denatured protein. Heteronuclear 3D NMR experiments and theoretical simulations of lysozyme in 8 M urea.; citation_author=H Schwalbe; citation_author=KM Fiebig; citation_author=M Buck; citation_author=JA Jones; citation_author=SB Grimshaw; citation_journal_title=Biochemistry; citation_volume=36; citation_number=9; citation_pages=8977-8991; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Long-range interactions within a nonnative protein.; citation_author=J Klein-Seetharaman; citation_author=M Oikawa; citation_author=SB Grimshaw; citation_author=J Wirmer; citation_author=E Duchardt; citation_journal_title=Science; citation_volume=295; citation_number=10; citation_pages=1719-1722; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Similarities between the spectrin SH3 domain denatured state and its folding transition state.; citation_author=T Kortemme; citation_author=MJ Kelly; citation_author=LE Kay; citation_author=J Forman-Kay; citation_author=L Serrano; citation_journal_title=J Mol Biol; citation_volume=297; citation_number=11; citation_pages=1217-1229; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Elucidation of the protein folding landscape by NMR.; citation_author=HJ Dyson; citation_author=PE Wright; citation_journal_title=Methods Enzymol; citation_volume=394; citation_number=12; citation_pages=299-321; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Semianalytical treatment of solvation for molecular mechanics and dynamics.; citation_author=WC Still; citation_author=A Tempczyk; citation_author=RC Hawley; citation_author=T Hendrickson; citation_journal_title=J Am Chem Soc; citation_volume=112; citation_number=13; citation_pages=6127-6129; citation_date=1990; " />
      <meta name="citation_reference" content="citation_title=Can a continuum solvent model reproduce the free energy landscape of a beta-hairpin folding in water?; citation_author=R Zhou; citation_author=BJ Berne; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=99; citation_number=14; citation_pages=12777-12782; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Is Poisson-Boltzmann theory insufficient for protein folding simulations?; citation_author=TZ Lwin; citation_author=R Zhou; citation_author=R Luo; citation_journal_title=J Chem Phys; citation_volume=124; citation_number=15; citation_pages=034902; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Balancing solvation and intramolecular interactions: toward a consistent generalized born force field.; citation_author=J Chen; citation_author=W Im; citation_author=CL Brooks; citation_journal_title=J Am Chem Soc; citation_volume=128; citation_number=16; citation_pages=3728-3736; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Investigation of Salt Bridge Stability in a Generalized Born Solvent Model.; citation_author=R Geney; citation_author=M Layten; citation_author=R Gomperts; citation_author=V Hornak; citation_author=C Simmerling; citation_journal_title=J Chem Theory Comput; citation_volume=2; citation_number=17; citation_pages=115-127; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Exchange Monte Carlo method and application to spin glass simulations.; citation_author=K Hukushima; citation_author=K Nemoto; citation_journal_title=J Phys Soc Japan; citation_volume=65; citation_number=18; citation_pages=1604-1608; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Parallel-tempering algorithm for conformational studies of biological molecules.; citation_author=UH Hansmann; citation_journal_title=Chem Phys Lett; citation_volume=281; citation_number=19; citation_pages=140-150; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Replica-exchange molecular dynamics method for protein folding.; citation_author=Y Sugita; citation_author=Y Okamoto; citation_journal_title=Chem Phys Lett; citation_volume=314; citation_number=20; citation_pages=141-151; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Unfolded state of polyalanine is a segmented polyproline II helix.; citation_author=A Kentsis; citation_author=M Mezei; citation_author=T Gindin; citation_author=R Osman; citation_journal_title=Proteins; citation_volume=55; citation_number=21; citation_pages=493-501; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Polypeptide Folding Using Monte Carlo Sampling, Concerted Rotation, and Continuum Solvation.; citation_author=JP Ulmschneider; citation_author=WL Jorgensen; citation_journal_title=J Am Chem Soc; citation_volume=126; citation_number=22; citation_pages=1849-1857; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Efficient Monte Carlo sampling for long molecular chains using local moves, tested on a solvated lipid bilayer.; citation_author=M Mezei; citation_journal_title=J Chem Phys; citation_volume=118; citation_number=23; citation_pages=3874-3880; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Replica-exchange extensions of simulated tempering method.; citation_author=A Mitsutake; citation_author=Y Okamoto; citation_journal_title=J Chem Phys; citation_volume=121; citation_number=24; citation_pages=2491-2504; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Atomic simulations of protein folding, using the replica exchange algorithm.; citation_author=H Nymeyer; citation_author=S Gnanakaran; citation_author=AE Garcia; citation_journal_title=Methods Enzymol; citation_volume=383; citation_number=25; citation_pages=119-149; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Modified Replica Exchange Simulation Methods for Local Structure Refinement.; citation_author=X Cheng; citation_author=G Cui; citation_author=V Hornak; citation_author=C Simmerling; citation_journal_title=J Phys Chem B; citation_volume=109; citation_number=26; citation_pages=8220-8230; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Replica exchange with solute tempering: a method for sampling biological systems in explicit water.; citation_author=P Liu; citation_author=B Kim; citation_author=RA Friesner; citation_author=BJ Berne; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=102; citation_number=27; citation_pages=13749-13754; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Designing a 20-residue protein.; citation_author=JW Neidigh; citation_author=RM Fesinmeyer; citation_author=NH Andersen; citation_journal_title=Nat Struct Biol; citation_volume=9; citation_number=28; citation_pages=425-430; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Trp-cage: folding free energy landscape in explicit water.; citation_author=R Zhou; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=100; citation_number=29; citation_pages=13280-13285; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Understanding folding and design: replica-exchange simulations of “Trp-cage” miniproteins.; citation_author=JW Pitera; citation_author=W Swope; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=100; citation_number=30; citation_pages=7587-7592; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=All-atom folding of the trp-cage protein with an adaptive parallel tempering method.; citation_author=A Schug; citation_author=W Wenzel; citation_journal_title=Europhys Lett; citation_volume=67; citation_number=31; citation_pages=307-313; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=A microscopic view of miniprotein folding: enhanced folding efficiency through formation of an intermediate.; citation_author=H Neuweiler; citation_author=S Doose; citation_author=M Sauer; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=102; citation_number=32; citation_pages=16650-16655; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=UV-resonance raman thermal unfolding study of Trp-cage shows that it is not a simple two-state miniprotein.; citation_author=Z Ahmed; citation_author=IA Beta; citation_author=AV Mikhonin; citation_author=SA Asher; citation_journal_title=J Am Chem Soc; citation_volume=127; citation_number=33; citation_pages=10943-10950; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Smaller and faster: the 20-residue Trp-cage protein folds in 4 micros.; citation_author=L Qiu; citation_author=SA Pabit; citation_author=AE Roitberg; citation_author=SJ Hagen; citation_journal_title=J Am Chem Soc; citation_volume=124; citation_number=34; citation_pages=12952-12953; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Possible locally driven folding pathways of TC5b, a 20-residue protein.; citation_author=GV Nikiforovich; citation_author=NH Andersen; citation_author=RM Fesinmeyer; citation_author=C Frieden; citation_journal_title=Proteins; citation_volume=52; citation_number=35; citation_pages=292-302; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Respective roles of short-and long-range interactions in protein folding.; citation_author=N Go; citation_author=H Taketomi; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=75; citation_number=36; citation_pages=559-563; citation_date=1978; " />
      <meta name="citation_reference" content="citation_title=Impact of local and non-local interactions on thermodynamics and kinetics of protein folding.; citation_author=VI Abkevich; citation_author=AM Gutin; citation_author=EI Shakhnovich; citation_journal_title=J Mol Biol; citation_volume=252; citation_number=37; citation_pages=460-471; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=The Go model revisited: Native structure and the geometric coupling between local and long-range contacts.; citation_author=PF Faisca; citation_author=MM Telo da Gama; citation_author=A Nunes; citation_journal_title=Proteins; citation_volume=60; citation_number=38; citation_pages=712-722; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Exploring the Interplay between Topology and Secondary Structural Formation in the Protein Folding Problem.; citation_author=MS Cheung; citation_author=JM Finke; citation_author=B Callahan; citation_author=JN Onuchic; citation_journal_title=J Phys Chem B; citation_volume=107; citation_number=39; citation_pages=11193-11200; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Polyproline II structure in a sequence of seven alanine residues.; citation_author=Z Shi; citation_author=CA Olson; citation_author=GD Rose; citation_author=RL Baldwin; citation_author=NR Kallenbach; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=99; citation_number=40; citation_pages=9190-9195; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=The solvation free energy of the peptide backbone is strongly conformation-dependent.; citation_author=M Mezei; citation_author=PJ Fleming; citation_author=R Srinivasan; citation_author=GD Rose; citation_journal_title=Proteins; citation_volume=55; citation_number=41; citation_pages=502-507; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Structural correlations in protein folding funnels.; citation_author=BA Shoemaker; citation_author=J Wang; citation_author=PG Wolynes; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=94; citation_number=42; citation_pages=777-782; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Discerning the structure and energy of multiple transition states in protein folding using psi-analysis.; citation_author=BA Krantz; citation_author=RS Dothager; citation_author=TR Sosnick; citation_journal_title=J Mol Biol; citation_volume=337; citation_number=43; citation_pages=463-475; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Progress in modeling of protein structures and interactions.; citation_author=O Schueler-Furman; citation_author=C Wang; citation_author=P Bradley; citation_author=K Misura; citation_author=D Baker; citation_journal_title=Science; citation_volume=310; citation_number=44; citation_pages=638-642; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Statistical theory for protein ensembles with designed energy landscapes.; citation_author=P Biswas; citation_author=J Zou; citation_author=JG Saven; citation_journal_title=J Chem Phys; citation_volume=123; citation_number=45; citation_pages=154908; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Origin of the sequence-dependent polyproline II structure in unfolded peptides.; citation_author=A Kentsis; citation_author=M Mezei; citation_author=R Osman; citation_journal_title=Proteins; citation_volume=61; citation_number=46; citation_pages=769-776; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Ultrafast folding of a computationally designed trp-cage mutant: trp(2)-cage.; citation_author=MR Bunagan; citation_author=X Yang; citation_author=JG Saven; citation_author=F Gai; citation_journal_title=J Phys Chem B Condens Matter Mater Surf Interfaces Biophys; citation_volume=110; citation_number=47; citation_pages=3759-3763; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Comparison of simple potential functions for simulating liquid water.; citation_author=W Jorgensen; citation_author=J Chandrasekhar; citation_author=JD Madura; citation_author=RW Impey; citation_author=ML Klein; citation_journal_title=J Chem Phys; citation_volume=79; citation_number=48; citation_pages=926-935; citation_date=1983; " />
      <meta name="citation_reference" content="citation_title=All-atom empirical potential for molecular modeling and dynamics studies of proteins.; citation_author=AD MacKerell; citation_author=D Bashford; citation_author=M Bellott; citation_author=RL Dunbrack; citation_author=JD Evanseck; citation_journal_title=J Phys Chem B; citation_volume=102; citation_number=49; citation_pages=3586-3616; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Extending the treatment of backbone energetics in protein force fields: limitations of gas-phase quantum mechanics in reproducing protein conformational distributions in molecular dynamics simulations.; citation_author=AD Mackerell; citation_author=M Feig; citation_author=CL Brooks; citation_journal_title=J Comput Chem; citation_volume=25; citation_number=50; citation_pages=1400-1415; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=MMTSB Tool Set: enhanced sampling and multiscale modeling methods for applications in structural biology.; citation_author=M Feig; citation_author=J Karanicolas; citation_author=CL Brooks; citation_journal_title=J Mol Graph Model; citation_volume=22; citation_number=51; citation_pages=377-395; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Annealing Markov Chain Monte Carlo With Applications to Ancestral Inference.; citation_author=CJ Geyer; citation_author=EA Thompson; citation_journal_title=J Am Stat Assoc; citation_volume=90; citation_number=52; citation_pages=909; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Extended state-space Monte Carlo methods.; citation_author=SB Opps; citation_author=J Schofield; citation_journal_title=Phys Rev E; citation_volume=63; citation_number=53; citation_pages=056701; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Phase changes in 38-atom Lennard-Jones clusters. I. A parallel tempering study in the canonical ensemble.; citation_author=JP Neirotti; citation_author=F Calvo; citation_author=DL Freeman; citation_author=JD Doll; citation_journal_title=J Chem Phys; citation_volume=112; citation_number=54; citation_pages=10340-10349; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Cool walking: A new Markov chain Monte Carlo sampling method.; citation_author=S Brown; citation_author=T Head-Gordon; citation_journal_title=J Comp Chem; citation_volume=24; citation_number=55; citation_pages=68-76; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Statistical clustering techniques for the analysis of long molecular dynamics trajectories: analysis of 2.2-ns trajectories of YPGDV.; citation_author=ME Karpen; citation_author=DJ Tobias; citation_author=CL Brooks; citation_journal_title=Biochemistry; citation_volume=32; citation_number=56; citation_pages=412-420; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=Spectral methods for dimensionality reduction.; citation_author=LK Saul; citation_author=KQ Weinberger; citation_author=JH Ham; citation_author=F Sha; citation_author=DD Lee; citation_number=57; citation_date=2005; citation_publisher=MIT Press; " />
      <meta name="citation_reference" content="citation_title=Nonlinear dimensionality reduction by locally linear embedding.; citation_author=ST Roweis; citation_author=LK Saul; citation_journal_title=Science; citation_volume=290; citation_number=58; citation_pages=2323-2326; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Fluctuation and cross-correlation analysis of protein motions observed in nanosecond molecular dynamics simulations.; citation_author=PH Hunenberger; citation_author=AE Mark; citation_author=WF van Gunsteren; citation_journal_title=J Mol Biol; citation_volume=252; citation_number=59; citation_pages=492-503; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Dynamic and structural analysis of isotropically distributed molecular ensembles.; citation_author=JJ Prompers; citation_author=R Bruschweiler; citation_journal_title=Proteins; citation_volume=46; citation_number=60; citation_pages=177-189; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Using path sampling to build better Markovian state models: predicting the folding rate and mechanism of a tryptophan zipper beta hairpin.; citation_author=N Singhal; citation_author=CD Snow; citation_author=VS Pande; citation_journal_title=J Chem Phys; citation_volume=121; citation_number=61; citation_pages=415-425; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=The protein folding network.; citation_author=F Rao; citation_author=A Caflisch; citation_journal_title=J Mol Biol; citation_volume=342; citation_number=62; citation_pages=299-306; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Protein folding pathways from replica exchange simulations and a kinetic network model.; citation_author=M Andrec; citation_author=AK Felts; citation_author=E Gallicchio; citation_author=RM Levy; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=102; citation_number=63; citation_pages=6801-6806; citation_date=2005; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000446" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Calculation of the Free Energy and Cooperativity of Protein Folding"/>
    <meta name="twitter:description" content="Calculation of the free energy of protein folding and delineation of its pre-organization are of foremost importance for understanding, predicting and designing biological macromolecules. Here, we introduce an energy smoothing variant of parallel tempering replica exchange Monte Carlo (REMS) that allows for efficient configurational sampling of flexible solutes under the conditions of molecular hydration. Its usage to calculate the thermal stability of a model globular protein, Trp cage TC5b, achieves excellent agreement with experimental measurements. We find that the stability of TC5b is attained through the coupled formation of local and non-local interactions. Remarkably, many of these structures persist at high temperature, concomitant with the origin of native-like configurations and mesostates in an otherwise macroscopically disordered unfolded state. Graph manifold learning reveals that the conversion of these mesostates to the native state is structurally heterogeneous, and that the cooperativity of their formation is encoded largely by the unfolded state ensemble. In all, these studies establish the extent of thermodynamic and structural pre-organization of folding of this model globular protein, and achieve the calculation of macromolecular stability ab initio, as required for ab initio structure prediction, genome annotation, and drug design."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0000446.g007"/>

  <meta property="og:title" content="Calculation of the Free Energy and Cooperativity of Protein Folding" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000446" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=6220'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=9220'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=4174&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0000446">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000446" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000446&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Calculation of the Free Energy and Cooperativity of Protein Folding
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Alex Kentsis
              <span class="corresponding">mail</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:alex.kentsis@childrens.harvard.edu.">alex.kentsis@childrens.harvard.edu.</a></p>

                <p>Affiliation:
                  Department of Molecular Physiology and Biophysics, Mount Sinai School of Medicine, New York University, New York, New York, United States of America
                </p>

                <p>
                    Current address: Department of Medicine, Children's Hospital Boston, Harvard Medical School, Boston, Massachusetts, United States of America
                </p>

              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Tatyana Gindin, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Molecular Physiology and Biophysics, Mount Sinai School of Medicine, New York University, New York, New York, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Mihaly Mezei, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Molecular Physiology and Biophysics, Mount Sinai School of Medicine, New York University, New York, New York, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Roman Osman
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Molecular Physiology and Biophysics, Mount Sinai School of Medicine, New York University, New York, New York, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: May 16, 2007</li>
    <li>DOI: 10.1371/journal.pone.0000446</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0000446" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0000446" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000446" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0000446" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0000446-g001" data-doi="info:doi/10.1371/journal.pone.0000446" data-uri="info:doi/10.1371/journal.pone.0000446.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000446.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000446-g002" data-doi="info:doi/10.1371/journal.pone.0000446" data-uri="info:doi/10.1371/journal.pone.0000446.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000446.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000446-g003" data-doi="info:doi/10.1371/journal.pone.0000446" data-uri="info:doi/10.1371/journal.pone.0000446.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000446.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000446-g004" data-doi="info:doi/10.1371/journal.pone.0000446" data-uri="info:doi/10.1371/journal.pone.0000446.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000446.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000446-g005" data-doi="info:doi/10.1371/journal.pone.0000446" data-uri="info:doi/10.1371/journal.pone.0000446.g005" title="Figure 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000446.g005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000446-g006" data-doi="info:doi/10.1371/journal.pone.0000446" data-uri="info:doi/10.1371/journal.pone.0000446.g006" title="Figure 6">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000446.g006&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000446-t001" data-doi="info:doi/10.1371/journal.pone.0000446" data-uri="info:doi/10.1371/journal.pone.0000446.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000446.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000446-g007" data-doi="info:doi/10.1371/journal.pone.0000446" data-uri="info:doi/10.1371/journal.pone.0000446.g007" title="Figure 7">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000446.g007&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000446">Reader Comments (1)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0000446" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1"></a><p>Calculation of the free energy of protein folding and delineation of its pre-organization are of foremost importance for understanding, predicting and designing biological macromolecules. Here, we introduce an energy smoothing variant of parallel tempering replica exchange Monte Carlo (REMS) that allows for efficient configurational sampling of flexible solutes under the conditions of molecular hydration. Its usage to calculate the thermal stability of a model globular protein, Trp cage TC5b, achieves excellent agreement with experimental measurements. We find that the stability of TC5b is attained through the coupled formation of local and non-local interactions. Remarkably, many of these structures persist at high temperature, concomitant with the origin of native-like configurations and mesostates in an otherwise macroscopically disordered unfolded state. Graph manifold learning reveals that the conversion of these mesostates to the native state is structurally heterogeneous, and that the cooperativity of their formation is encoded largely by the unfolded state ensemble. In all, these studies establish the extent of thermodynamic and structural pre-organization of folding of this model globular protein, and achieve the calculation of macromolecular stability <em>ab initio</em>, as required for <em>ab initio</em> structure prediction, genome annotation, and drug design.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>Kentsis A, Gindin T, Mezei M, Osman R (2007) Calculation of the Free Energy and Cooperativity of Protein Folding. PLoS ONE 2(5):
          e446.
            doi:10.1371/journal.pone.0000446</p><p><strong>Academic Editor: </strong>Hilal Lashuel, Swiss Federal Institute of Technology Lausanne (EPFL), Switzerland</p><p><strong>Received:</strong> January 17, 2007; <strong>Accepted:</strong> April 20, 2007; <strong>Published:</strong> May 16, 2007</p><p><strong>Copyright:</strong> © 2007 Kentsis et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>National Institutes of Health Medical Scientist Training Program, Grant DK 43046.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>The importance of accurately defining the molecular ensembles of proteins was recognized early by Levinthal, who concluded that folding of a random coil by way of a diffusive search of its combinatorially vast conformational space is incompatible with the biological energies and timescales of protein folding <a href="#pone.0000446-Levinthal1">[1]</a>. Consequently, either the conformational space of unfolded proteins deviates from that of a random coil, or the conformational search is not entirely diffusive, being guided by folding pathway(s), leading to proposals of hierarchical (thermodynamic) and framework (kinetic) folding models, respectively <a href="#pone.0000446-Sosnick1">[2]</a>–<a href="#pone.0000446-Daggett1">[5]</a>. The extent to which these two fundamental mechanisms cooperate in the biological attainability of the folding search is unknown.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>Structured unfolded states have been observed in a variety proteins <a href="#pone.0000446-Neri1">[6]</a>–<a href="#pone.0000446-Kortemme1">[11]</a>. However, it is unknown whether these macroscopically observed structures correspond to the conformations of individual residues, or to an average of microscopic configurational states that are composed of groups of residues. The former is consistent with the random, albeit conformationally biased (statistical) coil model of the unfolded state, and means that efficient folding is achieved largely by way of kinetic pathways. The latter is not, and implies that the unfolded state is thermodynamically pre-organized. Establishment of the extent of such pre-organization determines the relative contribution of the hierarchical (thermodynamic) and framework (kinetic) folding mechanisms, and is thus of major importance for understanding, predicting and designing biological macromolecules.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>Study of this question has been made difficult by the spectroscopic limits of resolving microscopic ensemble sub-states that exist under the conditions of physiologic temperature, pressure, and hydration <a href="#pone.0000446-Dyson1">[12]</a>. Such resolution is achievable theoretically by using molecular mechanics calculations, but is practically limited by the computational limits of simulating proteins in water under physiological conditions. These limits stem precisely from the dependence of conformational sampling of flexible solutes on the molecular properties of the solvent. Conventionally, these limitations are overcome through the implicit treatment of solvent effects, as in the approximation of Born <a href="#pone.0000446-Still1">[13]</a>. However, its tested implementations do not appear to reproduce the thermodynamics and structures of natural proteins under physiologic conditions <a href="#pone.0000446-Zhou1">[14]</a>, though recently introduced algorithms appear to be more accurate <a href="#pone.0000446-Lwin1">[15]</a>–<a href="#pone.0000446-Geney1">[17]</a>.</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p>Usage of Monte Carlo (MC) algorithms that utilize simultaneous changes of many conformational variables, such as loop torsion MC and replica exchange MC (REM), has shown promise in efficiently calculating convergent ensembles of proteins in aqueous solution <a href="#pone.0000446-Hukushima1">[18]</a>–<a href="#pone.0000446-Ulmschneider1">[22]</a>. However, application of loop torsion MC to protein folding depends on the analytical solutions of the loop closure problem, currently available for six polypeptide torsions <a href="#pone.0000446-Mezei1">[23]</a>. REM or parallel tempering MC achieves changes of all conformational variables in aqueous solution through the use of global updates such as molecular dynamics (MD), but requires prohibitively large numbers of replicas in order to generate sufficient energy overlaps, as required by the Metropolis criterion <a href="#pone.0000446-Mitsutake1">[24]</a>, <a href="#pone.0000446-Nymeyer1">[25]</a>. This Metropolis limit derives from the statistics of energy fluctuations, whereby the energy overlap between adjoining replicas required for efficient MC exchange scales as <em>n</em><sup>−½</sup>, where <em>n</em> is the number of degrees of freedom, which are mostly of bulk water molecules. Recently, partial REM and REM with solute tempering (REST) have been developed to extend the Metropolis limit of REM for the simulations of protein folding in aqueous solution <a href="#pone.0000446-Cheng1">[26]</a>, <a href="#pone.0000446-Liu1">[27]</a>. Both do so by reducing the effective number of the degrees of freedom that contribute to the Metropolis energy criterion.</p>
<a id="article1.body1.sec1.p5" name="article1.body1.sec1.p5"></a><p>Here, we introduce another such variant, termed replica exchange MC with energy smoothing (REMS), that does so by manipulating the energy expression itself. We show that in spite of deforming the free energy surface to some extent, REMS yields apparently canonical free energy distributions in the energetic regime of biological systems. Consequently, we apply REMS to simulate the thermal folding of a small globular protein, the 20-residue Trp cage TC5b, under the near physiologic conditions of molecular hydration. We show that such an approach can be used for efficient and accurate calculation of protein stability <em>ab initio</em>, at least with respect to the folding of TC5b. And finally, by using self-consistent clustering and machine graph learning, we examine the origin of cooperativity and thermal stability of various structural motifs in this model protein. As a result, we offer a demonstration of the extent of thermodynamic and structural pre-organization of protein folding, important for understanding the mechanics of this phenomenon, with implications for a variety of problems, such as those that require calculations of free energies including structure prediction, genome annotation, and drug design.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results and Discussion"></a><h3>Results and Discussion</h3>
<h4>Canonical molecular ensembles in water using REMS</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>TC5b is a small globular protein, consisting of several natural and redesigned structural motifs (<a href="#pone-0000446-g001">Fig. 1</a>). To generate a set of molecular ensembles of the thermal folding of TC5b, we equilibrated 32 replicas of TC5b in explicit water at 273–363 K, corresponding to the temperature range of experimentally measured thermal stability of TC5b <a href="#pone.0000446-Neidigh1">[28]</a>. This approach differs from earlier replica exchange simulations of TC5b <a href="#pone.0000446-Zhou2">[29]</a>–<a href="#pone.0000446-Schug1">[31]</a>, in particular by using periodic boundary conditions that are large enough (60×60×60 Å<sup>3</sup>) to accommodate a fully extended TC5b in explicit water, a 100 ps MD trajectory phase prior to replica exchange to achieve equilibration (<a href="#pone-0000446-g002">Fig. 2A</a>), and a 2 ps thermalization time during exchange to prevent quenching (<a href="#pone-0000446-g002">Fig. 2B</a>).</p>
<div class="figure" id="pone-0000446-g001"><div class="img"><a name="pone-0000446-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000446.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000446" data-uri="info:doi/10.1371/journal.pone.0000446.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000446.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000446.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000446.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000446.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000446.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000446.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000446.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000446.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000446.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Structure of native TC5b.</span></strong></p><a id="article1.body1.sec2.sec1.fig1.caption1.p1" name="article1.body1.sec2.sec1.fig1.caption1.p1"></a><p>The structure is composed of an N-terminal α-helix with its α-helical/secondary Q<sup>5</sup>:K<sup>8</sup> salt bridge (red), type I β-turn S<sup>13</sup>-S<sup>14</sup>-G<sup>15</sup> with its β-turn/tertiary D<sup>9</sup>:R<sup>16</sup> salt bridge (blue), and a hydrophobic core that includes both α-helical Y<sup>3</sup>:W<sup>6</sup> and tertiary W<sup>6</sup>:P<sup>19</sup> interactions (gold mesh).</p>
<span>doi:10.1371/journal.pone.0000446.g001</span></div><div class="figure" id="pone-0000446-g002"><div class="img"><a name="pone-0000446-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000446.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000446" data-uri="info:doi/10.1371/journal.pone.0000446.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000446.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000446.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000446.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000446.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000446.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000446.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000446.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000446.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000446.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Equilibration and calibration of REMS simulations of TC5b in explicit water.</span></strong></p><a id="article1.body1.sec2.sec1.fig2.caption1.p1" name="article1.body1.sec2.sec1.fig2.caption1.p1"></a><p>A. Instantaneous potential energy (<em>U</em>) as a function of MD time during evolution of the 273 K replica in the canonical <em>NVT</em> ensemble prior to initiating REMS, demonstrating its equilibration, as reflected in the energetic stability during the last 50 ps. B. Instantaneous potential energy (<em>U</em>) as a function of MD time upon replica exchange from 276 to 273 K, demonstrating thermalization in less than 2 ps. C. Average potential energy 〈<em>U</em>〉 of 273 K replica as a function of energy smoothing time (<em>ts</em>). As <em>ts</em> approaches 2000 fs, the standard deviation of &lt;<em>U&gt;</em> approaches the fluctuation of the energy distribution in that time domain. At <em>ts</em>  =  200 fs, energy-smoothed 〈<em>U</em>〉 of REMS is statistically indistinguishable from the instantaneous <em>U</em> used during conventional REM; double-sided <em>p</em> = 0.73. Bars represent ±1σ.</p>
<span>doi:10.1371/journal.pone.0000446.g002</span></div><a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2"></a><p>We used a smoothing time of 200 fs for the calculation of the Metropolis criterion during REMS (<a href="#s3">Methods</a>), since the smoothed energy at this time shows small fluctuations, and most importantly, preserves approximately Boltzmann-weighted sampling. The difference between smoothed and near instant mean energies is less than 2.8 kcal/mol, and is not significantly different from that at shorter and near instant time intervals (Student's <em>t</em>-test <em>p</em> = 0.73; <a href="#pone-0000446-g002">Fig. 2C</a>), less than 10% of the total energy of the system. Usage of such energy smoothing leads to a distribution of and a mean potential energy of water (<a href="#pone-0000446-g003">Fig. 3B</a>) as well as temperature dependent heat capacity of water (<a href="#pone-0000446-g003">Fig. 3C</a>) which are statistically indistinguishable from those of exactly canonical simulations. On the other hand, usage of extremely long smoothing time of 600 fs leads to a gross underestimation of water's heat capacity, consistent with significant deviation from canonical sampling (<a href="#pone-0000446-g003">Fig. 3C</a>). Approximately canonical REMS with smoothing time of 200 fs leads to efficient replica exchange with mean exchange probabilities of about 30% (<a href="#pone-0000446-g004">Fig. 4A</a>); conventional REM of this system in explicit water being limited by Metropolis statistics of less than 1% (data not shown). Evolution of the calculated 32 ensembles for more than 4,000 exchanges with mean transition probabilities of about 30% means that the highest and lowest temperature replicas are exchanged on average more than 40 times, as confirmed by tracking the initial lowest and highest temperature replicas, containing the predominantly native and unfolded states, respectively, as they diffuse in temperature space in the course of the simulation (<a href="#pone-0000446-g004">Fig. 4B</a>). Consequently, the final simulation exceeds the apparent computational time constant of self-diffusion of TC5b by nearly three orders of magnitude (<a href="#pone-0000446-g004">Fig. 4C</a>), consistent with the simulation's convergence <a href="#pone.0000446-Kentsis1">[21]</a>.</p>
<div class="figure" id="pone-0000446-g003"><div class="img"><a name="pone-0000446-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000446.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000446" data-uri="info:doi/10.1371/journal.pone.0000446.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000446.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000446.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000446.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000446.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000446.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000446.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000446.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000446.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000446.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>REMS calculation of approximately canonical ensembles of pure water.</span></strong></p><a id="article1.body1.sec2.sec1.fig3.caption1.p1" name="article1.body1.sec2.sec1.fig3.caption1.p1"></a><p>A. Histograms of potential energies (<em>U</em>) of different temperature replicas, demonstrating energy overlaps between adjoining temperature REMS replicas, as required for efficient MC exchange. B. Comparison of histograms of potential energies (<em>U</em>) of water ensembles at 273 K calculated using canonical MD (<em>NVT</em>) and REMS. Usage of REMS yields statistically indistinguishable mean energies and slightly increased energy fluctuations, as compared to those of canonical MD simulations, as shown by their normal fits (solid curves). C. Heat capacities at constant volume (<em>C<sub>V</sub></em>) of pure water at different temperatures, as obtained experimentally (solid squares), and calculated using canonical MD (dotted diamonds) and REMS (dashed circles). Usage of REMS with extremely long smoothing time of 600 fs (REMS<sup>*</sup>, solid stars) leads to a significant underestimation of the heat capacity of water at low temperature. Sizes of symbols represent ±1σ.</p>
<span>doi:10.1371/journal.pone.0000446.g003</span></div><div class="figure" id="pone-0000446-g004"><div class="img"><a name="pone-0000446-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000446.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000446" data-uri="info:doi/10.1371/journal.pone.0000446.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000446.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000446.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000446.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000446.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000446.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000446.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000446.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000446.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000446.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>Sampling and efficiency of REMS simulations of TC5b in explicit water.</span></strong></p><a id="article1.body1.sec2.sec1.fig4.caption1.p1" name="article1.body1.sec2.sec1.fig4.caption1.p1"></a><p>A. Mean probabilities 〈<em>P</em>〉 of MC exchange between adjoining replicas <em>x<sub>n</sub></em> and <em>x<sub>m</sub></em> as a function of temperature, demonstrating that usage of REMS leads to efficient replica exchange. B. Exchanges of replicas in the temperature space, tracking the initial lowest (red dashed) and highest (blue solid) containing the predominantly native and unfolded states, respectively, as they diffuse in temperature space in the course of the simulation. C. Divergence of the normalized difference (<em>Δ</em>) of fraction of formed hydrophobic core W<sup>6</sup>:P<sup>19</sup> (closed squares), hydrophobic core Y<sup>3</sup>:W<sup>6</sup> (open circles), salt bridge D<sup>9</sup>:R<sup>16</sup> (closed stars), α-helical Y3:L7 (solid circles) and the β-turn D9:S14 (open squares) hydrogen bonds between initial and final structures as a function of replica exchange for the 363 K replica. These measure were chosen because their non-local nature should be most sensitive to initial configuration memory effects. The total length of REMS simulation exceeds the apparent computational time constant of self-diffusion by nearly three orders of magnitude.</p>
<span>doi:10.1371/journal.pone.0000446.g004</span></div>

<h4>Calculation of thermal stability of TC5b</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>In order to examine the origin of thermal stability of TC5b, we calculated the apparent stabilities of various conformational motifs of TC5b as a function of temperature (<a href="#pone-0000446-g005">Fig. 5</a>). Their choice was guided by the naturally occurring secondary and tertiary structural elements, as well as those that were specifically redesigned in TC5b <a href="#pone.0000446-Neidigh1">[28]</a> (<a href="#pone-0000446-g001">Fig. 1</a>). At 273 K, REMS calculated conformational ensemble of TC5b is largely folded, with nearly all molecules forming the N-terminal α-helix, the β-turn, the C-terminal polyprolyl helix (<a href="#pone-0000446-g005">Fig. 5A</a>), and tertiary and secondary hydrophobic cores (<a href="#pone-0000446-g005">Fig. 5B</a>). In contrast to the average NMR structure, the β-turn hydrogen bond and salt bridge, as well as the α-helical hydrogen bonds are largely (~90%) but not persistently formed. On the other hand, the α-helical salt bridge is formed only in half of the ensemble (<a href="#pone-0000446-g005">Fig. 5B</a>), in agreement with the experimental observations of TC5b <a href="#pone.0000446-Neidigh1">[28]</a>, <a href="#pone.0000446-Neuweiler1">[32]</a>–<a href="#pone.0000446-Qiu1">[34]</a>.</p>
<div class="figure" id="pone-0000446-g005"><div class="img"><a name="pone-0000446-g005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000446.g005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000446" data-uri="info:doi/10.1371/journal.pone.0000446.g005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000446.g005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000446.g005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000446.g005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000446.g005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000446.g005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000446.g005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000446.g005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000446.g005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000446.g005.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 5.  <span>Thermal stability of TC5b.</span></strong></p><a id="article1.body1.sec2.sec2.fig1.caption1.p1" name="article1.body1.sec2.sec2.fig1.caption1.p1"></a><p>A. Fraction of formed α-helix L<sup>2</sup>YIQWLK<sup>8</sup> (dashed black), β-turn S<sup>14</sup> (solid green), and polyprolyl helix P<sup>18</sup> (dotted blue), as defined using self-consistent clustering and enumeration of their backbone dihedral angles. Note that P<sup>18</sup> remains unchanged in its backbone conformation due to its definition in CHARMM. Individual α-helical residues have varying thermal stability, with the more N-terminal ones being less stable, consistent with the existence of α-helical fraying. B. Fraction of formed α-helical salt/secondary bridge Q<sup>5</sup>:K<sup>8</sup> (solid red), α-helical hydrogen bond Y<sup>3</sup>:L<sup>7</sup> (dotted red), β-turn/tertiary salt bridge D<sup>9</sup>:R<sup>16</sup> (solid blue), β-turn hydrogen bond D<sup>9</sup>:S<sup>14</sup> (dashed green), tertiary hydrophobic core W<sup>6</sup>:P<sup>19</sup> and Y<sup>3</sup>:P<sup>19</sup> (solid and dashed black), and secondary hydrophobic core Y<sup>3</sup>:W<sup>6</sup> (dashed red), as defined by using self-consistent clustering and enumeration of their distances. Note that the α-helical salt/secondary bridge is only partially formed at low temperature, even though the rest of the structure is nearly fully folded by other measures. Similarly, the secondary hydrophobic core Y<sup>3</sup>:W<sup>6</sup> persists even at high temperature, where the rest of the protein is largely unfolded by other measures. Importantly, substantial amount of residual native structure persists at high temperature. C. Fraction of formed mean α-helical structure (dashed black), mean β-turn structure (solid green), mean tertiary structure (solid black) in the REMS calculated ensembles, and native fraction measured experimentally using chemical shift dispersion (squares), as adapted from the first study of TC5b <a href="#pone.0000446-Neidigh1">[28]</a>.</p>
<span>doi:10.1371/journal.pone.0000446.g005</span></div><a id="article1.body1.sec2.sec2.p2" name="article1.body1.sec2.sec2.p2"></a><p>Stabilities of both local and non-local structural motifs exhibit an apparently sigmoid melting transition (<a href="#pone-0000446-g005">Fig. 5</a>). In particular, the N-terminal α-helix melts with an apparent melting temperature of 300–310 K, depending on the exact residue monitored, consistent with the presence of N-terminal fraying, wherein the helical residues closer to the terminus are less stable (<a href="#pone-0000446-g005">Fig. 5A</a>). Similarly, the β-turn melts with an apparent melting temperature of about 310 K (<a href="#pone-0000446-g005">Fig. 5A</a>), associated with the destabilization of the tertiary core and the β-turn/tertiary salt bridge (<a href="#pone-0000446-g005">Fig. 5B</a>). Calculation of the overall melting temperature of TC5b, by using mean ensemble conformational statistics, where all conformational motifs are equally weighted, yields an apparent value of approximately 310 K (<a href="#pone-0000446-g005">Fig. 5C</a>), in good agreement with the experimentally measured value of 315 K <a href="#pone.0000446-Neidigh1">[28]</a>. This finding differs from those of earlier replica exchange simulations of TC5b, which overestimated the apparent melting temperature by about 100 K, possibly because of continuum Born solvation <a href="#pone.0000446-Pitera1">[30]</a>, or constricted boundary conditions <a href="#pone.0000446-Zhou2">[29]</a>. However, it is important to note that our study is limited by the use of a single (CHARMM) force field and initial (native) conditions, which may bias and limit sampling, respectively.</p>


<h4>Unfolded state ensemble</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>Experimental studies of TC5b indicate a substantial amount of residual structure in the unfolded state ensemble at high temperature <a href="#pone.0000446-Neidigh1">[28]</a>. Our calculated high temperature ensemble also exhibits such structures, in particular possessing up to about 10% β-turn, 20% α-helical content, and 30% tertiary hydrophobic core (<a href="#pone-0000446-g005">Fig. 5</a>). Remarkably, nearly 60% of molecules in the high temperature ensemble contain the secondary hydrophobic core (<a href="#pone-0000446-g005">Fig. 5B</a>), in agreement with the experimental findings of such residual structure, as observed by using both NMR and fluorescence spectroscopies <a href="#pone.0000446-Neidigh1">[28]</a>, <a href="#pone.0000446-Neuweiler1">[32]</a>. This residual structure may arise from the persistence of various native-like conformations (isolated α-helical turn or hydrophobic core) in different molecules of the unfolded state ensemble that is otherwise non-native and heterogeneous. Conversely, this residual structure may be due to configurations of groups of conformations (associated α-helix and hydrophobic core) in an unfolded state ensemble that is relatively homogeneous with respect to these native-like configurations. Though indistinguishable macroscopically, these characteristically composed molecular ensembles diverge in the ways they affect protein folding and stability.</p>


<h4>Graph manifold learning of the unfolded state ensemble</h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>In order to discover the origin of residual structure at high temperature, we applied a graph-based approach designed to learn the natural coordinates of highly dimensioned data. By embedding the molecular ensemble in a graph based on geometric similarity, and projecting the individual structures onto a manifold that preserves nearest-neighbor geometric relations of this graph, we are able to distinguish globally organized configurations, termed mesostates, from groups of structures comprised of unrelated conformations (<a href="#s3">Methods</a>). Indeed, the high temperature manifold is comprised of several such mesostates, including configurations of secondary structures such as the N-terminal α-helix and the β-turn, as well as more complex configurations that contain both the α-helix and the tertiary hydrophobic core, for example (<a href="#pone-0000446-g006">Fig. 6</a>). These configurations are not due to the use of the NMR structure as the starting configuration for REMS, as the latter's memory is lost after about 30 replica exchanges and the final ensemble is evolved for more than 4,000 exchanges (<a href="#pone-0000446-g004">Fig. 4B&amp;C</a>). Instead, these configurations appear to pre-organize the unfolded state ensemble for folding by virtue of arranging individual interactions and conformations in the context of native-like mesostates. This pre-organization is likely inherent to the polypeptide sequence of TC5b, as suggested by energy minimization calculations of fragments of TC5b <a href="#pone.0000446-Nikiforovich1">[35]</a>.</p>
<div class="figure" id="pone-0000446-g006"><div class="img"><a name="pone-0000446-g006" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000446.g006&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000446" data-uri="info:doi/10.1371/journal.pone.0000446.g006"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000446.g006&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000446.g006/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000446.g006/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000446.g006/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000446.g006/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000446.g006.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000446.g006/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000446.g006/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000446.g006.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 6.  <span>Manifold of unfolded mesostates.</span></strong></p><a id="article1.body1.sec2.sec4.fig1.caption1.p1" name="article1.body1.sec2.sec4.fig1.caption1.p1"></a><p>Mapping of the unfolded state ensemble, as calculated using the 363 K replica, onto the two top coordinates of its locally linear embedding space (open black circles), and the two top coordinates of its principal component projection (solid green circles). Principle component analysis fails to discern mesostate structure of the unfolded state ensemble, with the entire ensemble located near the origin of the PCA projection. On the other hand, displacement along the manifold from the origin of the LLE map coincides with the formation of native-like mesostates, containing: 1) α-helical/secondary salt bridge (red), 2) β-turn/tertiary salt bridge (blue), 3) α-helix and α-helical hydrophobic core, and 4) nearly native configurations with both the α-helix and the tertiary hydrophobic core.</p>
<span>doi:10.1371/journal.pone.0000446.g006</span></div>

<h4>Folding cooperativity and pre-organization</h4>
<a id="article1.body1.sec2.sec5.p1" name="article1.body1.sec2.sec5.p1"></a><p>In order to estimate the extent of pre-organization of the thermal folding of TC5b by the residual structure of the high temperature ensemble, we calculated the apparent cooperativities of forming pairs of conformations into configurations, as expressed by the probabilities of forming these configurations conditional on the formation of their constituent conformations (<a href="#pone-0000446-t001">Table 1</a>). The conditional probabilities of forming these four-body interactions are related to the mesostate organization of the thermal folding reaction. Consistent with the presence of configurations containing the N-terminal α-helix in the high temperature ensemble (<a href="#pone-0000446-g006">Fig. 6</a>), the apparent conditional probability of forming both the α-helical hydrogen bond and the α-helical salt bridge exceeds the expected probability of forming this configuration assuming independence of its constituent conformations by more than a factor of 10 (<a href="#pone-0000446-t001">Table 1</a>). Similar effect is observed for the apparent cooperativities of formation of other configurations, involving both the secondary and tertiary hydrophobic cores and β-turn (<a href="#pone-0000446-t001">Table 1</a>). This surprising phenomenon is likely due to the thermodynamic coupling between the formation of individual native-like conformations and their organized configurations, such that native-like conformations are adopted essentially in the context of topologically native configurations.</p>
<div class="figure" id="pone-0000446-t001"><div class="img"><a name="pone-0000446-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000446.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000446" data-uri="info:doi/10.1371/journal.pone.0000446.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000446.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000446.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000446.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000446.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000446.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000446.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000446.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000446.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000446.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>Folding cooperativity in the unfolded state ensemble of TC5b</span></strong></p><span>doi:10.1371/journal.pone.0000446.t001</span></div><a id="article1.body1.sec2.sec5.p2" name="article1.body1.sec2.sec5.p2"></a><p>Apparent coupling between local (conformational) and non-local (configurational) contacts has been noted earlier during the folding of Gō lattice polymers, where its origins were related to the details of the potential energy function defining the native state <a href="#pone.0000446-Go1">[36]</a>–<a href="#pone.0000446-Faisca1">[38]</a>. As the Gō protein model is supplemented with backbone interactions, local backbone conformations can lead to progressive non-local organization <a href="#pone.0000446-Cheung1">[39]</a>. The finding of analogous conformational-configurational coupling during the folding of TC5b (<a href="#pone-0000446-t001">Table 1</a>), where on the other hand the folding process is defined by an atomic polypeptide in the context of a semi-empirical, classical force field (<a href="#s3">Methods</a>), suggests that such coupling is inherent to the properties of the hydrated polypeptide itself.</p>
<a id="article1.body1.sec2.sec5.p3" name="article1.body1.sec2.sec5.p3"></a><p>The apparent cooperativity between forming concomitant α-helical and tertiary hydrophobic cores of TC5b exceeds the expected non-cooperative value by nearly a factor of 3 (<a href="#pone-0000446-t001">Table 1</a>). Because the formation of both α-helical and tertiary hydrophobic cores defines most of the native topology of TC5b (<a href="#pone-0000446-g001">Fig. 1</a>), this suggests that the residual unfolded state structure in the form of native-like configurations and mesostates at high temperature (<a href="#pone-0000446-g006">Fig. 6</a>) is responsible for most of the folding search. This phenomenon is similar to the pre-organization of α-helix formation in hydrated polyalanine <a href="#pone.0000446-Shi1">[40]</a>, for which the microscopically pre-organized unfolded state contributes as much as half to the folding search <a href="#pone.0000446-Kentsis1">[21]</a>, <a href="#pone.0000446-Mezei2">[41]</a>. Altogether, the findings of such extensive pre-organization of both secondary structures as well as globular proteins suggest that the apparent biological efficiency of protein folding is due in large part to the thermodynamic pre-organization, as opposed to kinetic guidance. This pre-organization acts to reduce the conformational space available to the diffusive search of the unfolded state ensembles that are pre-ordered in configurational mesostates.</p>


<h4>Graph manifold learning of folding mesostates</h4>
<a id="article1.body1.sec2.sec6.p1" name="article1.body1.sec2.sec6.p1"></a><p>In order to assess how the thermal folding reaction can proceed by way of configurational mesostates, we examined the folding ensemble at the midpoint of its folding transition as comprised by the 310 K replica, by using graph manifold learning. At the folding midpoint, the unfolded and native state ensembles are equi-populated, and their inter-conversion defines all of the possible folding pathways <a href="#pone.0000446-Shoemaker1">[42]</a>. Projections of individual configurations of the folding ensemble onto its LLE space map a star-shaped manifold, with multiple mesostates radiating from the origin of the projection (<a href="#pone-0000446-g007">Fig. 7</a>). Displacement along the <em>M<sub>1</sub></em> coordinate of the manifold coincides with the transformation between the native and unfolded structures, with configurations near the origin of the LLE map being partially native-like (<a href="#pone-0000446-g007">Fig. 7</a>). Displacements along the <em>M<sub>2</sub></em> and <em>M<sub>3</sub></em> coordinates coincide in part with the transformations of the α-helix and the β-turn, respectively, either in the context of native-like or unfolded topologies, depending on the particular location along the <em>M<sub>1</sub></em> coordinate (<a href="#pone-0000446-g007">Fig. 7</a>). The LLE mapping identifies a wide variety of folding mesostates, including those that possess a near native topology and α-helix but lack a native β-turn, those that lack the tertiary hydrophobic core and the native β-turn but retain the α-helix, as well as those that possess a near native β-turn and hydrophobic cores but lack the α-helix (<a href="#pone-0000446-g007">Fig. 7</a>). The existence of such mesostates explains the observed stabilities of their constituent conformational motifs (<a href="#pone-0000446-g005">Fig. 5</a>), as well as the apparent cooperativities of their configurations (<a href="#pone-0000446-t001">Table 1</a>). Combined with the star-shaped organization of the manifold of TC5b's folding ensemble, the variety of these folding mesostates suggest that the thermal folding reaction of TC5b is structurally heterogeneous. Though the folding of TC5b is pre-organized extensively by the unfolded state ensemble (<a href="#pone-0000446-t001">Table 1</a>&amp;<a href="#pone-0000446-g006">Fig. 6</a>), this heterogeneity implies a relative diversity of available folding pathways, in agreement with experimental studies <a href="#pone.0000446-Krantz1">[43]</a>. Determination of the exact subset(s) of folding mesostates that contribute to the kinetic transition state ensemble and the overall folding mechanism is an important direction of future work.</p>
<div class="figure" id="pone-0000446-g007"><div class="img"><a name="pone-0000446-g007" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000446.g007&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000446" data-uri="info:doi/10.1371/journal.pone.0000446.g007"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000446.g007&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000446.g007/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000446.g007/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000446.g007/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000446.g007/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000446.g007.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000446.g007/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000446.g007/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000446.g007.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 7.  <span>Folding reaction manifold.</span></strong></p><a id="article1.body1.sec2.sec6.fig1.caption1.p1" name="article1.body1.sec2.sec6.fig1.caption1.p1"></a><p>Mapping of TC5b's folding ensemble at the midpoint of its thermal transition, as calculated using the 310 K replica, onto the top three coordinates of its LLE manifold. Displacement along the <em>M<sub>1</sub></em> coordinate of the manifold coincides with the transformation of the 5) nearly native and 6) partially unfolded mesostates that lack the tertiary hydrophobic core and the native β-turn, but retain a frayed α-helix and the tertiary salt bridge. Displacement along the <em>M<sub>2</sub></em> coordinate coincides in part with the transformation of the α-helix from mesostate 7) that possesses a near native β-turn and hydrophobic cores and a non-α-helical but compact N-terminus, and mesostate 8) that lacks the native hydrophobic cores and has a non-native β-turn centered at K<sup>8</sup> that is part of the N-terminal α-helix in the NMR structure. Displacement along the <em>M<sub>3</sub></em> coordinate coincides with the transformation of the β-turn, including mesostates 9) that have a near native β-turn and tertiary salt bridge but have an unfolded α-helix and hydrophobic cores, and 10) possess a near native topology and α-helix but lack a native β-turn.</p>
<span>doi:10.1371/journal.pone.0000446.g007</span></div>

<h4>Conclusions</h4>
<a id="article1.body1.sec2.sec7.p1" name="article1.body1.sec2.sec7.p1"></a><p>Insofar as the free energy of flexible polymers can be described by a configurational partition function, our study shows that molecularly adapted variants of replica exchange, including REMS introduced here, can be used for the calculation of the free energy and cooperativity of protein folding <em>ab initio</em>. In addition, structural configurations and mesostates unknown <em>a priori</em> but adopted by the folding ensemble can be discovered and characterized by using graph manifold learning methods such as LLE. Our findings indicate that the thermal folding of a model globular protein, Trp cage TC5b, involves a structurally heterogeneous set of configurations and mesostates (<a href="#pone-0000446-g007">Fig. 7</a>). Some of these configurations persist in the molecular ensemble at high temperature (<a href="#pone-0000446-g006">Fig. 6</a>), concomitant with the pre-organization of TC5b's folding by such ordered unfolded state ensemble (<a href="#pone-0000446-t001">Table 1</a>). Combined with observations of thermodynamic pre-organization of polypeptide secondary structures <a href="#pone.0000446-Kentsis1">[21]</a>, <a href="#pone.0000446-Mezei2">[41]</a>, these findings suggest that macromolecular modularity, as described by ensemble mesostates, likely plays an essential role in determining the structures and stabilities of biological macromolecules.</p>
<a id="article1.body1.sec2.sec7.p2" name="article1.body1.sec2.sec7.p2"></a><p>Furthermore, the successes and failures of current <em>de novo</em> protein design approaches likely reflect the significance of configurational organization of protein ensembles and the latter's contribution to protein stability, respectively <a href="#pone.0000446-SchuelerFurman1">[44]</a>. Since TC5b's thermal stability and apparent folding cooperativity, two hallmark features of equilibrium folding of all natural proteins, are related to the residual structure of TC5b's unfolded state ensemble (<a href="#pone-0000446-g005">Figs. 5</a>&amp;<a href="#pone-0000446-g006">6</a>), this suggests that the design of naturally stable proteins may be based on the structural preferences of unfolded polypeptides <a href="#pone.0000446-Biswas1">[45]</a>, as obtained computationally for example <a href="#pone.0000446-Kentsis2">[46]</a>. Indeed, TC5b has been re-designed recently by using just such an approach <a href="#pone.0000446-Bunagan1">[47]</a>. Application of advanced methods such as replica exchange Monte Carlo to sample the rugged energy spaces of proteins, and graph manifold learning to analyze the vast structural spaces of the molecular ensembles that constitute them, should prove useful for a variety of <em>ab initio</em> approaches to structure prediction, genome annotation, and drug design.</p>

</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Methods"></a><h3>Methods</h3>
<h4>Molecular systems</h4>
<a id="article1.body1.sec3.sec1.p1" name="article1.body1.sec3.sec1.p1"></a><p>To understand the origin of protein stability and cooperativity, we chose to examine a protein the folding of which is well characterized structurally, thermodynamically, and kinetically. The smallest such protein is the 20-residue Trp cage <a href="#pone.0000446-Neidigh1">[28]</a>, TC5b (NLYIQWLKDGGPSSGRPPPS; <a href="#pone-0000446-g001">Fig. 1</a>), a derivative of the Gila monster extendin-4 that has been truncated and redesigned to include an N-terminal α-helix cap (N<sup>1</sup>), α-helical/secondary salt bridge (Q<sup>5</sup>:K<sup>8</sup>), β-turn/tertiary salt bridge (D<sup>9</sup>:R<sup>16</sup>), and optimized hydrophobic stack (Y<sup>3</sup>:W<sup>6</sup>). In addition, TC5b contains a naturally occurring type I β-turn S<sup>13</sup>-S<sup>14</sup>-G<sup>15</sup>, type II polyproline helix P<sup>17</sup>-P<sup>18</sup>-P<sup>19</sup>, and a hydrophobic core containing both local secondary L<sup>2</sup>-Y<sup>3</sup>-I<sup>4</sup> and non-local tertiary W<sup>6</sup>:P<sup>18</sup> and Y<sup>3</sup>:P<sup>19</sup> interactions.</p>
<a id="article1.body1.sec3.sec1.p2" name="article1.body1.sec3.sec1.p2"></a><p>NMR structure of TC5b (PDB code 1L2Y; model 1) was used as the starting configuration for our studies. The structure was solvated under periodic boundary conditions using a 60×60×60 Å<sup>3</sup> cubic box of equilibrated TIP3 water, and energy minimized using the CHARMM27 potential energy function in the presence of one randomly placed chloride ion to yield electroneutrality <a href="#pone.0000446-Jorgensen1">[48]</a>, <a href="#pone.0000446-MacKerell1">[49]</a>. The resulting system was heated using molecular dynamics with a linear gradient of 20 K/ps and equilibrated in the isothermal-isobaric (<em>NPT</em>) ensemble at 273 K and 1 atm pressure for 100 ps, using the Leapfrog Verlet integrator with velocity rescaling, 2 fs integration time step, energy-conserving Nose-Hoover thermostat, SHAKE to constrain hydrogen atoms, center of mass updates to remove rotational and translational solute motion, and particle mesh Ewald (PME) summation to calculate electrostatic interactions, as implemented in CHARMM. Because these calculations were already in progress, we did not use the subsequently introduced CHARMM backbone dihedral parameter correction <a href="#pone.0000446-Mackerell1">[50]</a>. Upon equilibration, systems were 60.7×60.7×60.7 Å<sup>3</sup> in volume, containing a total of 21,640 atoms and 7,112 water molecules. Such size and equilibration was necessary to thermalize and unfold this protein (see below). This system was used as the initial state for molecular dynamics equilibrations in the canonical (<em>NVT</em>) ensemble for 100 ps at mean temperatures of 273+3<em>n</em> K, where <em>n</em> = (0, 31).</p>


<h4>Replica exchange</h4>
<a id="article1.body1.sec3.sec2.p1" name="article1.body1.sec3.sec2.p1"></a><p>For REM, we utilized the MMTSB Tool Set, a recently developed collection of Perl scripts that interface with CHARMM <a href="#pone.0000446-Feig1">[51]</a>. Thirty two replicas were prepared as described above, thermalized at temperatures that were spaced between 273 and 363 K, a range chosen based on the experimentally observed thermostability of TC5b <a href="#pone.0000446-Neidigh1">[28]</a>. Each replica was simulated independently in the canonical ensemble under periodic boundary conditions using Leapfrog Verlet molecular dynamics with velocity rescaling, 2 fs integration time step, Nose-Hoover thermostat, SHAKE constraint, and PME electrostatics. Every 2 ps, an exchange between replicas <em>n</em> and <em>m</em> neighboring in temperature was attempted using the energy smoothed Metropolis criterion: <em>P</em> (<em>x<sub>n</sub></em> | <em>x<sub>m</sub></em>) = 1 if Δ<em>E</em> ≤ 0 and <em>P</em> (<em>x<sub>n</sub></em> | <em>x<sub>m</sub></em>) = exp(–Δ<em>E</em>) if Δ<em>E</em>&gt;0, where <em>P</em> is the probability of exchange, Δ<em>E</em> = <em>β<sub>n</sub></em> 〈<em>U<sub>n</sub></em>〉<em><sub>ts</sub></em>–<em>β<sub>m</sub></em> 〈<em>U<sub>m</sub></em>〉<em><sub>ts</sub></em>, <em>β</em> = 1/k<sub>B</sub><em>T</em>, <em>U</em> is potential energy, and <em>ts</em> is the MD smoothing time preceding the exchange over which the energies of the instantaneous configurations <em>x<sub>n</sub></em> and <em>x<sub>m</sub></em> are box averaged. Value of <em>ts</em> was tuned interactively to produce mean exchange acceptance rates of about 30%, while preserving approximately Bolzmann sampling, with <em>ts</em> of 200 fs used to generate the results described below. In the limit where the smoothing time is very long and the distribution of energy approaches the fluctuations in that time domain, e.g., <em>ts</em>&gt;600 fs (<a href="#pone-0000446-g002">Fig. 2C</a>), REMS is expected to produce ensembles with significantly reduced energy differences among states, allowing transitions between states that would otherwise by very different in their energies.</p>
<a id="article1.body1.sec3.sec2.p2" name="article1.body1.sec3.sec2.p2"></a><p>Upon each exchange of replicas neighboring in temperature, another exchange using the new pairs of neighboring replicas was attempted in order to maximize the tempering effect and the movement of replicas across the sampled temperature range. Upon a completed exchange, velocities of the exchanged configurations were rescaled to the new temperatures, another exchange was attempted 2 ps later, and the entire REMS simulation was produced for a total of 4,710 exchanges, while discarding 100 initial exchanges, corresponding to more than 0.3 µs of aggregate MD time, and sampling more than 150 million configurations.</p>
<a id="article1.body1.sec3.sec2.p3" name="article1.body1.sec3.sec2.p3"></a><p>Energy smoothing of REMS is equivalent to introducing an error into the calculation of the Metropolis criterion, and consequently produces non-stationary distributions of Markov chains of configurations. Though different in origin, this feature of REMS is analogous to the lack of stationary distributions produced by other tempering methods such as variants of Jump-walking (J-walking), where the conventional MC walker is allowed large transitions sampled from a different temperature ensemble, yielding generally non-stationary distributions of states <a href="#pone.0000446-Geyer1">[52]</a>, <a href="#pone.0000446-Opps1">[53]</a>. Nevertheless, it can be shown that approximately canonical distributions of states can be generated by using tempering MC methods such as J-walking when the jumping frequency is low compared to the total length of the MC walker, e.g., when the deformation of the free energy of the system is small relative to the differences in energy of major ensemble states <a href="#pone.0000446-Neirotti1">[54]</a>, <a href="#pone.0000446-Brown1">[55]</a>.</p>
<a id="article1.body1.sec3.sec2.p4" name="article1.body1.sec3.sec2.p4"></a><p>In order to evaluate the suitability of REMS to actually recover canonical energy distributions, we calculated the constant volume heat capacity of pure water: C<sub>v</sub> = (〈U<sup>2</sup>〉–〈U〉<sup>2</sup>)/k<sub>B</sub>T<sup>2</sup>. Because heat capacity reports squares of energy fluctuations, it is an extremely sensitive measure of the equipartition of energy that characterizes canonical ensembles. For this purpose, we used a 20×20×20 Å<sup>3</sup> box of equilibrated TIP3 water under periodic boundary conditions, simulated using MD in the canonical ensemble for 1 ns, using MD protocol as described above, at four different temperatures: 273, 285, 299, and 313 K. We carried out a REMS simulation of the same system, using replicas at 273, 285, 299, and 313 K, simulated for 1,000 exchanges attempted every 1 ps with <em>ts</em> of 200 fs, corresponding to aggregate MD time of 1 ns, equal to that of simulations using canonical MD without REMS. As can be seen from <a href="#pone-0000446-g003">Fig. 3</a>, usage of the energy smoothed Metropolis criterion does not lead to any significant distortions of the mean energy of water under near physiologic conditions of temperature and pressure, as seen from the comparison of the results of MD NVT and REMS simulations. More importantly, no statistically significant differences exist between the constant volume heat capacities of water calculated using exactly canonical MD NVT and approximately canonical REMS, both of which are in good agreement with the experimentally measured values (<a href="#pone-0000446-g003">Fig. 3</a>) <a href="#pone.0000446-Jorgensen1">[48]</a>.</p>
<a id="article1.body1.sec3.sec2.p5" name="article1.body1.sec3.sec2.p5"></a><p>To evaluate the computational efficiency of REMS, we calculated mean transition probabilities of exchanging pairs of replicas adjoining in temperature during the course of the simulation of the thermal folding of TC5b. As can be seen from <a href="#pone-0000446-g004">Fig. 4</a>, usage of REMS improves the otherwise system size-limited parallel tempering MC, yielding mean exchange acceptance ratios of about 30%, similar to traditional MC transition probabilities. To evaluate sampling efficiency, we calculated the evolution of the apparent self-diffusion coefficient <em>Δ<sub>0−N</sub></em> = (<em>f</em>(0) − <em>f</em>(<em>N</em>))/<em>f</em>(final), as a function of simulation length with respect to the number of replica exchanges <em>N</em>, where <em>f</em> is a phase space variable, such as the fraction of the native hydrophobic core of TC5b. If the sampling of phase space is ergodic, <em>Δ<sub>0−N</sub></em> decays to one at long <em>N</em>. This is a necessary but insufficient condition of ergodicity, since it depends on the choice of initial and final conditions. Due to the requirement of carrying out multiple independent simulations, we are unable to evaluate ergodicity directly <a href="#pone.0000446-Kentsis1">[21]</a>.</p>


<h4>Microscopic analysis and clustering</h4>
<a id="article1.body1.sec3.sec3.p1" name="article1.body1.sec3.sec3.p1"></a><p>In the analysis of structures of calculated ensembles, we use the term conformation to refer to geometries of individual interactions, and configuration to refer to molecular geometries of groups of interactions. Although canonical structures, such as α-helices and β-turns, have defined regular geometries, conformations in solution at ambient temperature exhibit considerable plasticity. Thus, we utilized a self-consistent method for defining conformational basins using a stepwise optimal clustering algorithm based on a self-organizing neural net, as implemented in ART-2 by Brooks and coworkers <a href="#pone.0000446-Karpen1">[56]</a>. Briefly, the cluster assignments of structural variables extracted from simulation ensembles were minimized subject to a constraint on a cluster radius, such that no member of a cluster was farther than a specified distance from the cluster center. Because the convergence of such minimizations is sensitive to initial conditions, we tested the robustness of assignments to conformational basins by recalculating cluster assignments using reshuffled trajectories (data not shown). Cluster occurrences and probabilities of sampling of conformational basins as defined in this manner were calculated using a set of home-built programs, available upon request.</p>
<a id="article1.body1.sec3.sec3.p2" name="article1.body1.sec3.sec3.p2"></a><p>In this manner, we examined the formation of the N-terminal α-helix by clustering (<em>Φ,ψ</em>) dihedral angles of the L<sup>2</sup>YIQWLK<sup>9</sup> polypeptide backbone and intrahelical hydrogen bond distances between backbone amide hydrogens and carbonyl oxygens, formation of the α-helical/secondary Q<sup>5</sup>:K<sup>8</sup> salt bridge by clustering the distance between side chain Q carboxamide oxygen and K amine nitrogen, formation of the β-turn by clustering (<em>Φ,ψ</em>) dihedral angles of S<sup>14</sup> and the hydrogen bond distance between backbone D<sup>9</sup> carbonyl oxygen and side chain S<sup>14</sup> hydroxyl hydrogen, formation of the β-turn/tertiary salt bridge D<sup>9</sup>:R<sup>16</sup> by clustering the distance between side chain D carboxylate carbon and R guanidino nitrogen, formation of the polyproline helix by clustering (<em>Φ,ψ,ω</em>) dihedral angles of P<sup>18</sup>, and lastly, formation of the hydrophobic core by clustering contact distances among side chain Y<sup>3</sup> phenol carbon ζ, W<sup>6</sup> indole carbon δ, and P<sup>19</sup> imido carbon δ. For all conformational variables, probabilities of forming native conformations were calculated by using clusters with near native centroids, as referenced to the NMR structure of TC5b.</p>


<h4>Folding manifold learning</h4>
<a id="article1.body1.sec3.sec4.p1" name="article1.body1.sec3.sec4.p1"></a><p>Because probabilities of forming structural configurations, such as folding intermediates, cannot be derived from conformational probabilities <em>a priori</em>, we examined their occurrence by direct enumeration of conditional probabilities of forming pairs of conformations. Apparent cooperativities of forming pairs of native interactions were calculated by using <em>P<sub>pair</sub></em> = <em>P</em> (<em>i</em>+<em>j</em>|<em>i</em> ; <em>j</em>), the probability of forming both interactions <em>i</em> and <em>j</em> under the condition that either <em>i</em> or <em>j</em> is formed.</p>
<a id="article1.body1.sec3.sec4.p2" name="article1.body1.sec3.sec4.p2"></a><p>In order to discover configurations that involve more than four-body interactions described above, we applied non-linear graph manifold learning techniques. Conventionally, study of high dimensional data such as atomic protein folding trajectories has been done using linear methods such as principal component analysis (PCA). PCA works by computing linear projections of greatest variance from the top eigenvectors of the data covariance matrix, thereby preserving the covariance structure of the data. However, because the global structure of high dimensional data is not necessarily linear, low dimensional linear principal components fail to capture this structure adequately (<a href="#pone-0000446-g006">Fig. 6</a>) <a href="#pone.0000446-Saul1">[57]</a>. Recently, graph based methods, including locally linear embedding (LLE), have been developed to preserve data neighbor relationships without enforcing global linearity <a href="#pone.0000446-Roweis1">[58]</a>. Simply put, such methods provide compact representations of complex data without imposing artificial constraints.</p>
<a id="article1.body1.sec3.sec4.p3" name="article1.body1.sec3.sec4.p3"></a><p>Our LLE input data set was dimensioned using the Cartesian coordinates of heavy atoms of TC5b (154 atoms × 3 (<em>x,y,z</em>) = 462 dimensions), and included 2,355 configurations sampled from the 363 K replica to model the unfolded state ensemble, or from the 310 K replica to model the folding ensemble. All coordinates were centered and oriented with respect to the NMR structure of TC5b (PDB model 1, see <a href="#s3">Methods</a>) in order to simplify the calculated manifolds, though this procedure is not required, in contrast to PCA <a href="#pone.0000446-Hunenberger1">[59]</a>, <a href="#pone.0000446-Prompers1">[60]</a>. LLE was carried out by calculating Euclidean distances between individual configurations, as defined by the Cartesian coordinates of their heavy atoms, and constructing nearest neighbor graphs using <em>k</em>-means clustering to define nodes of <em>k</em>-nearest neighbors. For the results shown below, we used <em>k</em> of 18. Varying <em>k</em> between 12 and 20 produced no qualitative differences in resulting manifolds (data not shown). The constructed graphs contained edges that specified nearest neighbor relations, as based on geometric similarity of Cartesian coordinates of heavy atoms, with edge weights computed by reconstructing each input configuration <em>x<sub>i</sub></em> from its <em>k</em>-nearest neighbors and minimizing the reconstruction error <em>ε<sub>W</sub></em> = Σ<em><sub>i</sub></em> (<em>x<sub>i</sub></em>−Σ<em><sub>j</sub></em> <em>W<sub>ij</sub>x<sub>j</sub></em>)<sup>2</sup>. The low dimensional manifold that preserved these locally linear neighbor relations was constructed by minimizing <em>ε<sub>ψ</sub></em> = Σ<em><sub>i</sub></em> (<em>ψ<sub>i</sub></em>−Σ<em><sub>j</sub></em> <em>W<sub>ij</sub>ψ<sub>j</sub></em>)<sup>2</sup>, where <em>ψ<sub>i</sub></em> is the low dimensional embedding of the high dimensional configuration <em>x<sub>i</sub></em>. Such a manifold preserves distance relationships of the data, subject to the constraints of the nearest-neighbor graph and the locally (but not globally) linear embeddings that describe it.</p>
<a id="article1.body1.sec3.sec4.p4" name="article1.body1.sec3.sec4.p4"></a><p>Our approach is related to other graph-based studies of molecular ensembles <a href="#pone.0000446-Singhal1">[61]</a>–<a href="#pone.0000446-Andrec1">[63]</a>, but instead of analyzing kinetic or energetic relations among states with respect to each other, we examine their geometric (dis)similarities with respect to the overall organization of the ensemble. In this way, projections of the high dimensional configurations <em>x<sub>i</sub></em>, as sampled from their molecular ensembles, onto the low dimensional manifold <em>ψ<sub>i</sub></em> reveal groups of geometric mesostates and their ensemble coordinates.</p>

</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>We thank Leslie Greengard for helpful discussions, Alex Proekt for comments on the manuscript, and Michael Feig for help with the MMTSB Tool Set. This study utilized the high-performance computational capabilities of the Biowulf PC/Linux cluster at the National Institutes of Health, Bethesda, MD (<a href="http://biowulf.nih.gov">http://biowulf.nih.gov</a>).</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: AK RO. Performed the experiments: AK. Analyzed the data: AK TG MM. Contributed reagents/materials/analysis tools: AK. Wrote the paper: AK RO.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0000446-Levinthal1" id="pone.0000446-Levinthal1"></a>Levinthal C (1968) Are there pathways for protein folding? Extrait du Journal de Chimie Physique  65: 44–45.  <ul class="find" data-citedArticleID="951670"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Are+there+pathways+for+protein+folding%3F&amp;auth=&amp;atitle=Are+there+pathways+for+protein+folding%3F" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Are+there+pathways+for+protein+folding%3F" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Are+there+pathways+for+protein+folding%3F%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0000446-Sosnick1" id="pone.0000446-Sosnick1"></a>Sosnick TR, Mayne L, Englander SW (1996) Molecular collapse: the rate-limiting step in two-state cytochrome c folding. Proteins  24: 413–426.  <ul class="find" data-citedArticleID="951726" data-doi="10.1002/(sici)1097-0134(199604)24:4<413::aid-prot1&gt;3.3.co;2-y"><li><a href="http://dx.doi.org/10.1002/(sici)1097-0134(199604)24:4<413::aid-prot1&gt;3.3.co;2-y" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Molecular+collapse%3A+the+rate-limiting+step+in+two-state+cytochrome+c+folding." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Molecular+collapse%3A+the+rate-limiting+step+in+two-state+cytochrome+c+folding.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0000446-Baldwin1" id="pone.0000446-Baldwin1"></a>Baldwin RL, Rose GD (1999) Is protein folding hierarchic? II. Folding intermediates and transition states. Trends Biochem Sci  24: 77–83.  <ul class="find" data-citedArticleID="951620" data-doi="10.1016/s0968-0004(98)01345-0"><li><a href="http://dx.doi.org/10.1016/s0968-0004(98)01345-0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Is+protein+folding+hierarchic%3F+II.+Folding+intermediates+and+transition+states." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Is+protein+folding+hierarchic%3F+II.+Folding+intermediates+and+transition+states.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0000446-Baldwin2" id="pone.0000446-Baldwin2"></a>Baldwin RL, Rose GD (1999) Is protein folding hierarchic? I. Local structure and peptide folding. Trends Biochem Sci  24: 26–33.  <ul class="find" data-citedArticleID="951622" data-doi="10.1016/s0968-0004(98)01346-2"><li><a href="http://dx.doi.org/10.1016/s0968-0004(98)01346-2" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Is+protein+folding+hierarchic%3F+I.+Local+structure+and+peptide+folding." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Is+protein+folding+hierarchic%3F+I.+Local+structure+and+peptide+folding.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0000446-Daggett1" id="pone.0000446-Daggett1"></a>Daggett V, Fersht AR (2003) Is there a unifying mechanism for protein folding? Trends Biochem Sci  28: 18–25.  <ul class="find" data-citedArticleID="951636" data-doi="10.1016/s0968-0004(02)00012-9"><li><a href="http://dx.doi.org/10.1016/s0968-0004(02)00012-9" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Is+there+a+unifying+mechanism+for+protein+folding%3F" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Is+there+a+unifying+mechanism+for+protein+folding%3F%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0000446-Neri1" id="pone.0000446-Neri1"></a>Neri D, Billeter M, Wider G, Wuthrich K (1992) NMR determination of residual structure in a urea-denatured protein, the 434-repressor. Science  257: 1559–1563.  <ul class="find" data-citedArticleID="951692" data-doi="10.1126/science.1523410"><li><a href="http://dx.doi.org/10.1126/science.1523410" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=NMR+determination+of+residual+structure+in+a+urea-denatured+protein%2C+the+434-repressor." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22NMR+determination+of+residual+structure+in+a+urea-denatured+protein%2C+the+434-repressor.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0000446-Logan1" id="pone.0000446-Logan1"></a>Logan TM, Theriault Y, Fesik SW (1994) Structural characterization of the FK506 binding protein unfolded in urea and guanidine hydrochloride. J Mol Biol  236: 637–648.  <ul class="find" data-citedArticleID="951674" data-doi="10.1006/jmbi.1994.1173"><li><a href="http://dx.doi.org/10.1006/jmbi.1994.1173" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Structural+characterization+of+the+FK506+binding+protein+unfolded+in+urea+and+guanidine+hydrochloride." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Structural+characterization+of+the+FK506+binding+protein+unfolded+in+urea+and+guanidine+hydrochloride.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0000446-Bai1" id="pone.0000446-Bai1"></a>Bai Y, Chung J, Dyson HJ, Wright PE (2001) Structural and dynamic characterization of an unfolded state of poplar apo-plastocyanin formed under nondenaturing conditions. Protein Sci  10: 1056–1066.  <ul class="find" data-citedArticleID="951618" data-doi="10.1110/ps.00601"><li><a href="http://dx.doi.org/10.1110/ps.00601" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Structural+and+dynamic+characterization+of+an+unfolded+state+of+poplar+apo-plastocyanin+formed+under+nondenaturing+conditions." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Structural+and+dynamic+characterization+of+an+unfolded+state+of+poplar+apo-plastocyanin+formed+under+nondenaturing+conditions.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0000446-Schwalbe1" id="pone.0000446-Schwalbe1"></a>Schwalbe H, Fiebig KM, Buck M, Jones JA, Grimshaw SB, et al.  (1997) Structural and dynamical properties of a denatured protein. Heteronuclear 3D NMR experiments and theoretical simulations of lysozyme in 8 M urea. Biochemistry  36: 8977–8991.  <ul class="find" data-citedArticleID="951718" data-doi="10.1021/bi970049q"><li><a href="http://dx.doi.org/10.1021/bi970049q" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Structural+and+dynamical+properties+of+a+denatured+protein.+Heteronuclear+3D+NMR+experiments+and+theoretical+simulations+of+lysozyme+in+8+M+urea." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Structural+and+dynamical+properties+of+a+denatured+protein.+Heteronuclear+3D+NMR+experiments+and+theoretical+simulations+of+lysozyme+in+8+M+urea.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0000446-KleinSeetharaman1" id="pone.0000446-KleinSeetharaman1"></a>Klein-Seetharaman J, Oikawa M, Grimshaw SB, Wirmer J, Duchardt E, et al.  (2002) Long-range interactions within a nonnative protein. Science  295: 1719–1722.  <ul class="find" data-citedArticleID="951664" data-doi="10.1126/science.1067680"><li><a href="http://dx.doi.org/10.1126/science.1067680" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Long-range+interactions+within+a+nonnative+protein." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Long-range+interactions+within+a+nonnative+protein.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0000446-Kortemme1" id="pone.0000446-Kortemme1"></a>Kortemme T, Kelly MJ, Kay LE, Forman-Kay J, Serrano L (2000) Similarities between the spectrin SH3 domain denatured state and its folding transition state. J Mol Biol  297: 1217–1229.  <ul class="find" data-citedArticleID="951666" data-doi="10.1006/jmbi.2000.3618"><li><a href="http://dx.doi.org/10.1006/jmbi.2000.3618" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Similarities+between+the+spectrin+SH3+domain+denatured+state+and+its+folding+transition+state." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Similarities+between+the+spectrin+SH3+domain+denatured+state+and+its+folding+transition+state.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0000446-Dyson1" id="pone.0000446-Dyson1"></a>Dyson HJ, Wright PE (2005) Elucidation of the protein folding landscape by NMR. Methods Enzymol  394: 299–321.  <ul class="find" data-citedArticleID="951638" data-doi="10.1016/s0076-6879(05)94011-1"><li><a href="http://dx.doi.org/10.1016/s0076-6879(05)94011-1" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Elucidation+of+the+protein+folding+landscape+by+NMR." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Elucidation+of+the+protein+folding+landscape+by+NMR.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0000446-Still1" id="pone.0000446-Still1"></a>Still WC, Tempczyk A, Hawley RC, Hendrickson T (1990) Semianalytical treatment of solvation for molecular mechanics and dynamics. J Am Chem Soc  112: 6127–6129.  <ul class="find" data-citedArticleID="951728" data-doi="10.1021/ja00172a038"><li><a href="http://dx.doi.org/10.1021/ja00172a038" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Semianalytical+treatment+of+solvation+for+molecular+mechanics+and+dynamics." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Semianalytical+treatment+of+solvation+for+molecular+mechanics+and+dynamics.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0000446-Zhou1" id="pone.0000446-Zhou1"></a>Zhou R, Berne BJ (2002) Can a continuum solvent model reproduce the free energy landscape of a beta-hairpin folding in water? Proc Natl Acad Sci U S A  99: 12777–12782.  <ul class="find" data-citedArticleID="951734" data-doi="10.1073/pnas.142430099"><li><a href="http://dx.doi.org/10.1073/pnas.142430099" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Can+a+continuum+solvent+model+reproduce+the+free+energy+landscape+of+a+beta-hairpin+folding+in+water%3F" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Can+a+continuum+solvent+model+reproduce+the+free+energy+landscape+of+a+beta-hairpin+folding+in+water%3F%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0000446-Lwin1" id="pone.0000446-Lwin1"></a>Lwin TZ, Zhou R, Luo R (2006) Is Poisson-Boltzmann theory insufficient for protein folding simulations? J Chem Phys  124: 034902.  <ul class="find" data-citedArticleID="951676" data-doi="10.1063/1.2161202"><li><a href="http://dx.doi.org/10.1063/1.2161202" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Is+Poisson-Boltzmann+theory+insufficient+for+protein+folding+simulations%3F" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Is+Poisson-Boltzmann+theory+insufficient+for+protein+folding+simulations%3F%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0000446-Chen1" id="pone.0000446-Chen1"></a>Chen J, Im W, Brooks CL Iii (2006) Balancing solvation and intramolecular interactions: toward a consistent generalized born force field. J Am Chem Soc  128: 3728–3736.  <ul class="find" data-citedArticleID="951630" data-doi="10.1021/ja057216r"><li><a href="http://dx.doi.org/10.1021/ja057216r" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Balancing+solvation+and+intramolecular+interactions%3A+toward+a+consistent+generalized+born+force+field." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Balancing+solvation+and+intramolecular+interactions%3A+toward+a+consistent+generalized+born+force+field.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0000446-Geney1" id="pone.0000446-Geney1"></a>Geney R, Layten M, Gomperts R, Hornak V, Simmerling C (2006) Investigation of Salt Bridge Stability in a Generalized Born Solvent Model. J Chem Theory Comput  2: 115–127.  <ul class="find" data-citedArticleID="951644" data-doi="10.1021/ct050183l"><li><a href="http://dx.doi.org/10.1021/ct050183l" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Investigation+of+Salt+Bridge+Stability+in+a+Generalized+Born+Solvent+Model." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Investigation+of+Salt+Bridge+Stability+in+a+Generalized+Born+Solvent+Model.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0000446-Hukushima1" id="pone.0000446-Hukushima1"></a>Hukushima K, Nemoto K (1996) Exchange Monte Carlo method and application to spin glass simulations. J Phys Soc Japan  65: 1604–1608.  <ul class="find" data-citedArticleID="951652" data-doi="10.1143/jpsj.65.1604"><li><a href="http://dx.doi.org/10.1143/jpsj.65.1604" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Exchange+Monte+Carlo+method+and+application+to+spin+glass+simulations." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Exchange+Monte+Carlo+method+and+application+to+spin+glass+simulations.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0000446-Hansmann1" id="pone.0000446-Hansmann1"></a>Hansmann UH (1997) Parallel-tempering algorithm for conformational studies of biological molecules. Chem Phys Lett  281: 140–150.  <ul class="find" data-citedArticleID="951650" data-doi="10.1016/s0009-2614(97)01198-6"><li><a href="http://dx.doi.org/10.1016/s0009-2614(97)01198-6" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Parallel-tempering+algorithm+for+conformational+studies+of+biological+molecules." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Parallel-tempering+algorithm+for+conformational+studies+of+biological+molecules.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0000446-Sugita1" id="pone.0000446-Sugita1"></a>Sugita Y, Okamoto Y (1999) Replica-exchange molecular dynamics method for protein folding. Chem Phys Lett  314: 141–151.  <ul class="find" data-citedArticleID="951730" data-doi="10.1016/s0009-2614(99)01123-9"><li><a href="http://dx.doi.org/10.1016/s0009-2614(99)01123-9" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Replica-exchange+molecular+dynamics+method+for+protein+folding." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Replica-exchange+molecular+dynamics+method+for+protein+folding.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0000446-Kentsis1" id="pone.0000446-Kentsis1"></a>Kentsis A, Mezei M, Gindin T, Osman R (2004) Unfolded state of polyalanine is a segmented polyproline II helix. Proteins  55: 493–501.  <ul class="find" data-citedArticleID="951660" data-doi="10.1002/prot.20051"><li><a href="http://dx.doi.org/10.1002/prot.20051" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Unfolded+state+of+polyalanine+is+a+segmented+polyproline+II+helix." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Unfolded+state+of+polyalanine+is+a+segmented+polyproline+II+helix.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0000446-Ulmschneider1" id="pone.0000446-Ulmschneider1"></a>Ulmschneider JP, Jorgensen WL (2004) Polypeptide Folding Using Monte Carlo Sampling, Concerted Rotation, and Continuum Solvation. J Am Chem Soc  126: 1849–1857.  <ul class="find" data-citedArticleID="951732" data-doi="10.1021/ja0378862"><li><a href="http://dx.doi.org/10.1021/ja0378862" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Polypeptide+Folding+Using+Monte+Carlo+Sampling%2C+Concerted+Rotation%2C+and+Continuum+Solvation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Polypeptide+Folding+Using+Monte+Carlo+Sampling%2C+Concerted+Rotation%2C+and+Continuum+Solvation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0000446-Mezei1" id="pone.0000446-Mezei1"></a>Mezei M (2003) Efficient Monte Carlo sampling for long molecular chains using local moves, tested on a solvated lipid bilayer. J Chem Phys  118: 3874–3880.  <ul class="find" data-citedArticleID="951682" data-doi="10.1063/1.1539839"><li><a href="http://dx.doi.org/10.1063/1.1539839" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Efficient+Monte+Carlo+sampling+for+long+molecular+chains+using+local+moves%2C+tested+on+a+solvated+lipid+bilayer." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Efficient+Monte+Carlo+sampling+for+long+molecular+chains+using+local+moves%2C+tested+on+a+solvated+lipid+bilayer.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0000446-Mitsutake1" id="pone.0000446-Mitsutake1"></a>Mitsutake A, Okamoto Y (2004) Replica-exchange extensions of simulated tempering method. J Chem Phys  121: 2491–2504.  <ul class="find" data-citedArticleID="951686" data-doi="10.1063/1.1766015"><li><a href="http://dx.doi.org/10.1063/1.1766015" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Replica-exchange+extensions+of+simulated+tempering+method." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Replica-exchange+extensions+of+simulated+tempering+method.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0000446-Nymeyer1" id="pone.0000446-Nymeyer1"></a>Nymeyer H, Gnanakaran S, Garcia AE (2004) Atomic simulations of protein folding, using the replica exchange algorithm. Methods Enzymol  383: 119–149.  <ul class="find" data-citedArticleID="951698" data-doi="10.1016/s0076-6879(04)83006-4"><li><a href="http://dx.doi.org/10.1016/s0076-6879(04)83006-4" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Atomic+simulations+of+protein+folding%2C+using+the+replica+exchange+algorithm." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Atomic+simulations+of+protein+folding%2C+using+the+replica+exchange+algorithm.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0000446-Cheng1" id="pone.0000446-Cheng1"></a>Cheng X, Cui G, Hornak V, Simmerling C (2005) Modified Replica Exchange Simulation Methods for Local Structure Refinement. J Phys Chem B  109: 8220–8230.  <ul class="find" data-citedArticleID="951632" data-doi="10.1021/jp045437y"><li><a href="http://dx.doi.org/10.1021/jp045437y" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Modified+Replica+Exchange+Simulation+Methods+for+Local+Structure+Refinement." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Modified+Replica+Exchange+Simulation+Methods+for+Local+Structure+Refinement.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0000446-Liu1" id="pone.0000446-Liu1"></a>Liu P, Kim B, Friesner RA, Berne BJ (2005) Replica exchange with solute tempering: a method for sampling biological systems in explicit water. Proc Natl Acad Sci U S A  102: 13749–13754.  <ul class="find" data-citedArticleID="951672" data-doi="10.1073/pnas.0506346102"><li><a href="http://dx.doi.org/10.1073/pnas.0506346102" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Replica+exchange+with+solute+tempering%3A+a+method+for+sampling+biological+systems+in+explicit+water." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Replica+exchange+with+solute+tempering%3A+a+method+for+sampling+biological+systems+in+explicit+water.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0000446-Neidigh1" id="pone.0000446-Neidigh1"></a>Neidigh JW, Fesinmeyer RM, Andersen NH (2002) Designing a 20-residue protein. Nat Struct Biol  9: 425–430.  <ul class="find" data-citedArticleID="951688" data-doi="10.1038/nsb798"><li><a href="http://dx.doi.org/10.1038/nsb798" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Designing+a+20-residue+protein." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Designing+a+20-residue+protein.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0000446-Zhou2" id="pone.0000446-Zhou2"></a>Zhou R (2003) Trp-cage: folding free energy landscape in explicit water. Proc Natl Acad Sci U S A  100: 13280–13285.  <ul class="find" data-citedArticleID="951736" data-doi="10.1073/pnas.2233312100"><li><a href="http://dx.doi.org/10.1073/pnas.2233312100" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Trp-cage%3A+folding+free+energy+landscape+in+explicit+water." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Trp-cage%3A+folding+free+energy+landscape+in+explicit+water.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0000446-Pitera1" id="pone.0000446-Pitera1"></a>Pitera JW, Swope W (2003) Understanding folding and design: replica-exchange simulations of “Trp-cage” miniproteins. Proc Natl Acad Sci U S A  100: 7587–7592.  <ul class="find" data-citedArticleID="951702" data-doi="10.1073/pnas.1330954100"><li><a href="http://dx.doi.org/10.1073/pnas.1330954100" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Understanding+folding+and+design%3A+replica-exchange+simulations+of+%26%23x201c%3BTrp-cage%26%23x201d%3B+miniproteins." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Understanding+folding+and+design%3A+replica-exchange+simulations+of+%26%23x201c%3BTrp-cage%26%23x201d%3B+miniproteins.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0000446-Schug1" id="pone.0000446-Schug1"></a>Schug A, Wenzel W (2004) All-atom folding of the trp-cage protein with an adaptive parallel tempering method. Europhys Lett  67: 307–313.  <ul class="find" data-citedArticleID="951716" data-doi="10.1209/epl/i2004-10056-2"><li><a href="http://dx.doi.org/10.1209/epl/i2004-10056-2" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=All-atom+folding+of+the+trp-cage+protein+with+an+adaptive+parallel+tempering+method." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22All-atom+folding+of+the+trp-cage+protein+with+an+adaptive+parallel+tempering+method.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0000446-Neuweiler1" id="pone.0000446-Neuweiler1"></a>Neuweiler H, Doose S, Sauer M (2005) A microscopic view of miniprotein folding: enhanced folding efficiency through formation of an intermediate. Proc Natl Acad Sci U S A  102: 16650–16655.  <ul class="find" data-citedArticleID="951694" data-doi="10.1073/pnas.0507351102"><li><a href="http://dx.doi.org/10.1073/pnas.0507351102" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+microscopic+view+of+miniprotein+folding%3A+enhanced+folding+efficiency+through+formation+of+an+intermediate." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+microscopic+view+of+miniprotein+folding%3A+enhanced+folding+efficiency+through+formation+of+an+intermediate.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0000446-Ahmed1" id="pone.0000446-Ahmed1"></a>Ahmed Z, Beta IA, Mikhonin AV, Asher SA (2005) UV-resonance raman thermal unfolding study of Trp-cage shows that it is not a simple two-state miniprotein. J Am Chem Soc  127: 10943–10950.  <ul class="find" data-citedArticleID="951614" data-doi="10.1021/ja050664e"><li><a href="http://dx.doi.org/10.1021/ja050664e" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=UV-resonance+raman+thermal+unfolding+study+of+Trp-cage+shows+that+it+is+not+a+simple+two-state+miniprotein." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22UV-resonance+raman+thermal+unfolding+study+of+Trp-cage+shows+that+it+is+not+a+simple+two-state+miniprotein.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0000446-Qiu1" id="pone.0000446-Qiu1"></a>Qiu L, Pabit SA, Roitberg AE, Hagen SJ (2002) Smaller and faster: the 20-residue Trp-cage protein folds in 4 micros. J Am Chem Soc  124: 12952–12953.  <ul class="find" data-citedArticleID="951706" data-doi="10.1021/ja0279141"><li><a href="http://dx.doi.org/10.1021/ja0279141" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Smaller+and+faster%3A+the+20-residue+Trp-cage+protein+folds+in+4+micros." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Smaller+and+faster%3A+the+20-residue+Trp-cage+protein+folds+in+4+micros.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0000446-Nikiforovich1" id="pone.0000446-Nikiforovich1"></a>Nikiforovich GV, Andersen NH, Fesinmeyer RM, Frieden C (2003) Possible locally driven folding pathways of TC5b, a 20-residue protein. Proteins  52: 292–302.  <ul class="find" data-citedArticleID="951696" data-doi="10.1002/prot.10409"><li><a href="http://dx.doi.org/10.1002/prot.10409" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Possible+locally+driven+folding+pathways+of+TC5b%2C+a+20-residue+protein." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Possible+locally+driven+folding+pathways+of+TC5b%2C+a+20-residue+protein.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">36.
              </span><a name="pone.0000446-Go1" id="pone.0000446-Go1"></a>Go N, Taketomi H (1978) Respective roles of short-and long-range interactions in protein folding. Proc Natl Acad Sci U S A  75: 559–563.  <ul class="find" data-citedArticleID="951648" data-doi="10.1073/pnas.75.2.559"><li><a href="http://dx.doi.org/10.1073/pnas.75.2.559" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Respective+roles+of+short-and+long-range+interactions+in+protein+folding." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Respective+roles+of+short-and+long-range+interactions+in+protein+folding.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">37.
              </span><a name="pone.0000446-Abkevich1" id="pone.0000446-Abkevich1"></a>Abkevich VI, Gutin AM, Shakhnovich EI (1995) Impact of local and non-local interactions on thermodynamics and kinetics of protein folding. J Mol Biol  252: 460–471.  <ul class="find" data-citedArticleID="951612" data-doi="10.1006/jmbi.1995.0511"><li><a href="http://dx.doi.org/10.1006/jmbi.1995.0511" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Impact+of+local+and+non-local+interactions+on+thermodynamics+and+kinetics+of+protein+folding." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Impact+of+local+and+non-local+interactions+on+thermodynamics+and+kinetics+of+protein+folding.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">38.
              </span><a name="pone.0000446-Faisca1" id="pone.0000446-Faisca1"></a>Faisca PF, Telo da Gama MM, Nunes A (2005) The Go model revisited: Native structure and the geometric coupling between local and long-range contacts. Proteins  60: 712–722.  <ul class="find" data-citedArticleID="951640" data-doi="10.1002/prot.20521"><li><a href="http://dx.doi.org/10.1002/prot.20521" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+Go+model+revisited%3A+Native+structure+and+the+geometric+coupling+between+local+and+long-range+contacts." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+Go+model+revisited%3A+Native+structure+and+the+geometric+coupling+between+local+and+long-range+contacts.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">39.
              </span><a name="pone.0000446-Cheung1" id="pone.0000446-Cheung1"></a>Cheung MS, Finke JM, Callahan B, Onuchic JN (2003) Exploring the Interplay between Topology and Secondary Structural Formation in the Protein Folding Problem. J Phys Chem B  107: 11193–11200.  <ul class="find" data-citedArticleID="951634" data-doi="10.1021/jp034441r"><li><a href="http://dx.doi.org/10.1021/jp034441r" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Exploring+the+Interplay+between+Topology+and+Secondary+Structural+Formation+in+the+Protein+Folding+Problem." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Exploring+the+Interplay+between+Topology+and+Secondary+Structural+Formation+in+the+Protein+Folding+Problem.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">40.
              </span><a name="pone.0000446-Shi1" id="pone.0000446-Shi1"></a>Shi Z, Olson CA, Rose GD, Baldwin RL, Kallenbach NR (2002) Polyproline II structure in a sequence of seven alanine residues. Proc Natl Acad Sci U S A  99: 9190–9195.  <ul class="find" data-citedArticleID="951720" data-doi="10.1073/pnas.112193999"><li><a href="http://dx.doi.org/10.1073/pnas.112193999" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Polyproline+II+structure+in+a+sequence+of+seven+alanine+residues." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Polyproline+II+structure+in+a+sequence+of+seven+alanine+residues.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">41.
              </span><a name="pone.0000446-Mezei2" id="pone.0000446-Mezei2"></a>Mezei M, Fleming PJ, Srinivasan R, Rose GD (2004) The solvation free energy of the peptide backbone is strongly conformation-dependent. Proteins  55: 502–507.  <ul class="find" data-citedArticleID="951684"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=The+solvation+free+energy+of+the+peptide+backbone+is+strongly+conformation-dependent.&amp;auth=&amp;atitle=The+solvation+free+energy+of+the+peptide+backbone+is+strongly+conformation-dependent." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+solvation+free+energy+of+the+peptide+backbone+is+strongly+conformation-dependent." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+solvation+free+energy+of+the+peptide+backbone+is+strongly+conformation-dependent.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">42.
              </span><a name="pone.0000446-Shoemaker1" id="pone.0000446-Shoemaker1"></a>Shoemaker BA, Wang J, Wolynes PG (1997) Structural correlations in protein folding funnels. Proc Natl Acad Sci U S A  94: 777–782.  <ul class="find" data-citedArticleID="951722" data-doi="10.1073/pnas.94.3.777"><li><a href="http://dx.doi.org/10.1073/pnas.94.3.777" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Structural+correlations+in+protein+folding+funnels." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Structural+correlations+in+protein+folding+funnels.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">43.
              </span><a name="pone.0000446-Krantz1" id="pone.0000446-Krantz1"></a>Krantz BA, Dothager RS, Sosnick TR (2004) Discerning the structure and energy of multiple transition states in protein folding using psi-analysis. J Mol Biol  337: 463–475.  <ul class="find" data-citedArticleID="951668" data-doi="10.1016/j.jmb.2004.01.018"><li><a href="http://dx.doi.org/10.1016/j.jmb.2004.01.018" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Discerning+the+structure+and+energy+of+multiple+transition+states+in+protein+folding+using+psi-analysis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Discerning+the+structure+and+energy+of+multiple+transition+states+in+protein+folding+using+psi-analysis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">44.
              </span><a name="pone.0000446-SchuelerFurman1" id="pone.0000446-SchuelerFurman1"></a>Schueler-Furman O, Wang C, Bradley P, Misura K, Baker D (2005) Progress in modeling of protein structures and interactions. Science  310: 638–642.  <ul class="find" data-citedArticleID="951714" data-doi="10.1126/science.1112160"><li><a href="http://dx.doi.org/10.1126/science.1112160" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Progress+in+modeling+of+protein+structures+and+interactions." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Progress+in+modeling+of+protein+structures+and+interactions.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">45.
              </span><a name="pone.0000446-Biswas1" id="pone.0000446-Biswas1"></a>Biswas P, Zou J, Saven JG (2005) Statistical theory for protein ensembles with designed energy landscapes. J Chem Phys  123: 154908.  <ul class="find" data-citedArticleID="951624" data-doi="10.1063/1.2062047"><li><a href="http://dx.doi.org/10.1063/1.2062047" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Statistical+theory+for+protein+ensembles+with+designed+energy+landscapes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Statistical+theory+for+protein+ensembles+with+designed+energy+landscapes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">46.
              </span><a name="pone.0000446-Kentsis2" id="pone.0000446-Kentsis2"></a>Kentsis A, Mezei M, Osman R (2005) Origin of the sequence-dependent polyproline II structure in unfolded peptides. Proteins  61: 769–776.  <ul class="find" data-citedArticleID="951662" data-doi="10.1002/prot.20655"><li><a href="http://dx.doi.org/10.1002/prot.20655" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Origin+of+the+sequence-dependent+polyproline+II+structure+in+unfolded+peptides." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Origin+of+the+sequence-dependent+polyproline+II+structure+in+unfolded+peptides.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">47.
              </span><a name="pone.0000446-Bunagan1" id="pone.0000446-Bunagan1"></a>Bunagan MR, Yang X, Saven JG, Gai F (2006) Ultrafast folding of a computationally designed trp-cage mutant: trp(2)-cage. J Phys Chem B Condens Matter Mater Surf Interfaces Biophys  110: 3759–3763.  <ul class="find" data-citedArticleID="951628" data-doi="10.1021/jp055288z"><li><a href="http://dx.doi.org/10.1021/jp055288z" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Ultrafast+folding+of+a+computationally+designed+trp-cage+mutant%3A+trp%282%29-cage." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Ultrafast+folding+of+a+computationally+designed+trp-cage+mutant%3A+trp%282%29-cage.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">48.
              </span><a name="pone.0000446-Jorgensen1" id="pone.0000446-Jorgensen1"></a>Jorgensen W, Chandrasekhar J, Madura JD, Impey RW, Klein ML (1983) Comparison of simple potential functions for simulating liquid water. J Chem Phys  79: 926–935.  <ul class="find" data-citedArticleID="951656" data-doi="10.1063/1.445869"><li><a href="http://dx.doi.org/10.1063/1.445869" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Comparison+of+simple+potential+functions+for+simulating+liquid+water." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Comparison+of+simple+potential+functions+for+simulating+liquid+water.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">49.
              </span><a name="pone.0000446-MacKerell1" id="pone.0000446-MacKerell1"></a>MacKerell AD, Bashford D, Bellott M, Dunbrack RL, Evanseck JD, et al.  (1998) All-atom empirical potential for molecular modeling and dynamics studies of proteins. J Phys Chem B  102: 3586–3616.  <ul class="find" data-citedArticleID="951678" data-doi="10.1021/jp973084f"><li><a href="http://dx.doi.org/10.1021/jp973084f" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=All-atom+empirical+potential+for+molecular+modeling+and+dynamics+studies+of+proteins." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22All-atom+empirical+potential+for+molecular+modeling+and+dynamics+studies+of+proteins.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">50.
              </span><a name="pone.0000446-Mackerell1" id="pone.0000446-Mackerell1"></a>Mackerell AD Jr, Feig M, Brooks CL 3rd (2004) Extending the treatment of backbone energetics in protein force fields: limitations of gas-phase quantum mechanics in reproducing protein conformational distributions in molecular dynamics simulations. J Comput Chem  25: 1400–1415.  <ul class="find" data-citedArticleID="951680" data-doi="10.1002/jcc.20065"><li><a href="http://dx.doi.org/10.1002/jcc.20065" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Extending+the+treatment+of+backbone+energetics+in+protein+force+fields%3A+limitations+of+gas-phase+quantum+mechanics+in+reproducing+protein+conformational+distributions+in+molecular+dynamics+simulations." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Extending+the+treatment+of+backbone+energetics+in+protein+force+fields%3A+limitations+of+gas-phase+quantum+mechanics+in+reproducing+protein+conformational+distributions+in+molecular+dynamics+simulations.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">51.
              </span><a name="pone.0000446-Feig1" id="pone.0000446-Feig1"></a>Feig M, Karanicolas J, Brooks CL 3rd (2004) MMTSB Tool Set: enhanced sampling and multiscale modeling methods for applications in structural biology. J Mol Graph Model  22: 377–395.  <ul class="find" data-citedArticleID="951642" data-doi="10.1016/j.jmgm.2003.12.005"><li><a href="http://dx.doi.org/10.1016/j.jmgm.2003.12.005" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=MMTSB+Tool+Set%3A+enhanced+sampling+and+multiscale+modeling+methods+for+applications+in+structural+biology." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22MMTSB+Tool+Set%3A+enhanced+sampling+and+multiscale+modeling+methods+for+applications+in+structural+biology.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">52.
              </span><a name="pone.0000446-Geyer1" id="pone.0000446-Geyer1"></a>Geyer CJ, Thompson EA (1995) Annealing Markov Chain Monte Carlo With Applications to Ancestral Inference. J Am Stat Assoc  90: 909.  <ul class="find" data-citedArticleID="951646" data-doi="10.1080/01621459.1995.10476590"><li><a href="http://dx.doi.org/10.1080/01621459.1995.10476590" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Annealing+Markov+Chain+Monte+Carlo+With+Applications+to+Ancestral+Inference." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Annealing+Markov+Chain+Monte+Carlo+With+Applications+to+Ancestral+Inference.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">53.
              </span><a name="pone.0000446-Opps1" id="pone.0000446-Opps1"></a>Opps SB, Schofield J (2001) Extended state-space Monte Carlo methods. Phys Rev E  63: 056701.  <ul class="find" data-citedArticleID="951700" data-doi="10.1103/physreve.63.056701"><li><a href="http://dx.doi.org/10.1103/physreve.63.056701" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Extended+state-space+Monte+Carlo+methods." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Extended+state-space+Monte+Carlo+methods.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">54.
              </span><a name="pone.0000446-Neirotti1" id="pone.0000446-Neirotti1"></a>Neirotti JP, Calvo F, Freeman DL, Doll JD (2000) Phase changes in 38-atom Lennard-Jones clusters. I. A parallel tempering study in the canonical ensemble. J Chem Phys  112: 10340–10349.  <ul class="find" data-citedArticleID="951690" data-doi="10.1063/1.481671"><li><a href="http://dx.doi.org/10.1063/1.481671" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Phase+changes+in+38-atom+Lennard-Jones+clusters.+I.+A+parallel+tempering+study+in+the+canonical+ensemble." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Phase+changes+in+38-atom+Lennard-Jones+clusters.+I.+A+parallel+tempering+study+in+the+canonical+ensemble.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">55.
              </span><a name="pone.0000446-Brown1" id="pone.0000446-Brown1"></a>Brown S, Head-Gordon T (2003) Cool walking: A new Markov chain Monte Carlo sampling method. J Comp Chem  24: 68–76.  <ul class="find" data-citedArticleID="951626" data-doi="10.1002/jcc.10181"><li><a href="http://dx.doi.org/10.1002/jcc.10181" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Cool+walking%3A+A+new+Markov+chain+Monte+Carlo+sampling+method." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Cool+walking%3A+A+new+Markov+chain+Monte+Carlo+sampling+method.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">56.
              </span><a name="pone.0000446-Karpen1" id="pone.0000446-Karpen1"></a>Karpen ME, Tobias DJ, Brooks CL 3rd (1993) Statistical clustering techniques for the analysis of long molecular dynamics trajectories: analysis of 2.2-ns trajectories of YPGDV. Biochemistry  32: 412–420.  <ul class="find" data-citedArticleID="951658" data-doi="10.1021/bi00053a005"><li><a href="http://dx.doi.org/10.1021/bi00053a005" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Statistical+clustering+techniques+for+the+analysis+of+long+molecular+dynamics+trajectories%3A+analysis+of+2.2-ns+trajectories+of+YPGDV." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Statistical+clustering+techniques+for+the+analysis+of+long+molecular+dynamics+trajectories%3A+analysis+of+2.2-ns+trajectories+of+YPGDV.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">57.
              </span><a name="pone.0000446-Saul1" id="pone.0000446-Saul1"></a>Saul LK, Weinberger KQ, Ham JH, Sha F, Lee DD (2005)  Spectral methods for dimensionality reduction. In: Schoelkopf B, Chapelle O, Zien A, editors. Semisupervised Learning. Cambridge, MA: MIT Press.   <ul class="find-nolinks"></ul></li><li><span class="label">58.
              </span><a name="pone.0000446-Roweis1" id="pone.0000446-Roweis1"></a>Roweis ST, Saul LK (2000) Nonlinear dimensionality reduction by locally linear embedding. Science  290: 2323–2326.  <ul class="find" data-citedArticleID="951710" data-doi="10.1126/science.290.5500.2323"><li><a href="http://dx.doi.org/10.1126/science.290.5500.2323" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Nonlinear+dimensionality+reduction+by+locally+linear+embedding." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Nonlinear+dimensionality+reduction+by+locally+linear+embedding.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">59.
              </span><a name="pone.0000446-Hunenberger1" id="pone.0000446-Hunenberger1"></a>Hunenberger PH, Mark AE, van Gunsteren WF (1995) Fluctuation and cross-correlation analysis of protein motions observed in nanosecond molecular dynamics simulations. J Mol Biol  252: 492–503.  <ul class="find" data-citedArticleID="951654" data-doi="10.1006/jmbi.1995.0514"><li><a href="http://dx.doi.org/10.1006/jmbi.1995.0514" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Fluctuation+and+cross-correlation+analysis+of+protein+motions+observed+in+nanosecond+molecular+dynamics+simulations." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Fluctuation+and+cross-correlation+analysis+of+protein+motions+observed+in+nanosecond+molecular+dynamics+simulations.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">60.
              </span><a name="pone.0000446-Prompers1" id="pone.0000446-Prompers1"></a>Prompers JJ, Bruschweiler R (2002) Dynamic and structural analysis of isotropically distributed molecular ensembles. Proteins  46: 177–189.  <ul class="find" data-citedArticleID="951704" data-doi="10.1002/prot.10025"><li><a href="http://dx.doi.org/10.1002/prot.10025" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Dynamic+and+structural+analysis+of+isotropically+distributed+molecular+ensembles." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Dynamic+and+structural+analysis+of+isotropically+distributed+molecular+ensembles.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">61.
              </span><a name="pone.0000446-Singhal1" id="pone.0000446-Singhal1"></a>Singhal N, Snow CD, Pande VS (2004) Using path sampling to build better Markovian state models: predicting the folding rate and mechanism of a tryptophan zipper beta hairpin. J Chem Phys  121: 415–425.  <ul class="find" data-citedArticleID="951724" data-doi="10.1063/1.1738647"><li><a href="http://dx.doi.org/10.1063/1.1738647" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Using+path+sampling+to+build+better+Markovian+state+models%3A+predicting+the+folding+rate+and+mechanism+of+a+tryptophan+zipper+beta+hairpin." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Using+path+sampling+to+build+better+Markovian+state+models%3A+predicting+the+folding+rate+and+mechanism+of+a+tryptophan+zipper+beta+hairpin.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">62.
              </span><a name="pone.0000446-Rao1" id="pone.0000446-Rao1"></a>Rao F, Caflisch A (2004) The protein folding network. J Mol Biol  342: 299–306.  <ul class="find" data-citedArticleID="951708" data-doi="10.1016/j.jmb.2004.06.063"><li><a href="http://dx.doi.org/10.1016/j.jmb.2004.06.063" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+protein+folding+network." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+protein+folding+network.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">63.
              </span><a name="pone.0000446-Andrec1" id="pone.0000446-Andrec1"></a>Andrec M, Felts AK, Gallicchio E, Levy RM (2005) Protein folding pathways from replica exchange simulations and a kinetic network model. Proc Natl Acad Sci U S A  102: 6801–6806.  <ul class="find" data-citedArticleID="951616" data-doi="10.1073/pnas.0408970102"><li><a href="http://dx.doi.org/10.1073/pnas.0408970102" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Protein+folding+pathways+from+replica+exchange+simulations+and+a+kinetic+network+model." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Protein+folding+pathways+from+replica+exchange+simulations+and+a+kinetic+network+model.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000446.XML" value="114918"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000446.PDF" value="395146"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000446.g001.PNG_L" value="735226"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000446.g001.PNG_M" value="336729"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000446.g001.PNG_S" value="8786"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000446.g001.TIF" value="1099182"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000446.g001.PNG_I" value="116508"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000446.g002.PNG_L" value="132010"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000446.g002.PNG_M" value="17364"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000446.g002.PNG_S" value="3579"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000446.g002.TIF" value="229484"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000446.g002.PNG_I" value="35480"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000446.g003.PNG_L" value="109583"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000446.g003.PNG_M" value="35649"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000446.g003.PNG_S" value="4331"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000446.g003.TIF" value="215320"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000446.g003.PNG_I" value="49576"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000446.g004.PNG_L" value="46554"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000446.g004.PNG_M" value="38259"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000446.g004.PNG_S" value="6015"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000446.g004.TIF" value="257782"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000446.g004.PNG_I" value="71400"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000446.g005.PNG_L" value="114340"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000446.g005.PNG_M" value="52464"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000446.g005.PNG_S" value="7461"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000446.g005.TIF" value="230956"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000446.g005.PNG_I" value="110488"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000446.g006.PNG_L" value="304389"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000446.g006.PNG_M" value="126519"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000446.g006.PNG_S" value="4429"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000446.g006.TIF" value="391382"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000446.g006.PNG_I" value="45332"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000446.t001.PNG_L" value="91933"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000446.t001.PNG_M" value="101142"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000446.t001.PNG_S" value="3587"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000446.t001.TIF" value="421844"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000446.t001.PNG_I" value="44899"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000446.g007.PNG_L" value="336701"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000446.g007.PNG_M" value="124352"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000446.g007.PNG_S" value="5037"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000446.g007.TIF" value="462058"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000446.g007.PNG_I" value="55934"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000446&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000446" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000446&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0000446&volume=&issue=&title=Calculation of the Free Energy and Cooperativity of Protein Folding&author_name=Alex%20Kentsis%2C%20Tatyana%20Gindin%2C%20Mihaly%20Mezei%2C%20Roman%20Osman&start_page=1&end_page=10" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000446" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000446" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0000446" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000446&amp;t=Calculation%20of%20the%20Free%20Energy%20and%20Cooperativity%20of%20Protein%20Folding" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000446&title=Calculation%20of%20the%20Free%20Energy%20and%20Cooperativity%20of%20Protein%20Folding&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000446&amp;title=Calculation%20of%20the%20Free%20Energy%20and%20Cooperativity%20of%20Protein%20Folding" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000446" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0000446&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Calculation of the Free Energy and Cooperativity of Protein Folding';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0000446';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Calculation%20of%20the%20Free%20Energy%20and%20Cooperativity%20of%20Protein%20Folding http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000446" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0000446" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">











          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Free+energy%22" title="Search for articles in the subject area:'Free energy'"><div class="flagText">Free energy</div></a>
              <div data-categoryid="18159" data-articleid="23730"
                   data-categoryname="Free energy"
                   class="flagImage" title="Flag 'Free energy' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Graphs%22" title="Search for articles in the subject area:'Graphs'"><div class="flagText">Graphs</div></a>
              <div data-categoryid="19267" data-articleid="23730"
                   data-categoryname="Graphs"
                   class="flagImage" title="Flag 'Graphs' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Manifolds%22" title="Search for articles in the subject area:'Manifolds'"><div class="flagText">Manifolds</div></a>
              <div data-categoryid="36747" data-articleid="23730"
                   data-categoryname="Manifolds"
                   class="flagImage" title="Flag 'Manifolds' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Potential+energy%22" title="Search for articles in the subject area:'Potential energy'"><div class="flagText">Potential energy</div></a>
              <div data-categoryid="23753" data-articleid="23730"
                   data-categoryname="Potential energy"
                   class="flagImage" title="Flag 'Potential energy' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Protein+folding%22" title="Search for articles in the subject area:'Protein folding'"><div class="flagText">Protein folding</div></a>
              <div data-categoryid="23597" data-articleid="23730"
                   data-categoryname="Protein folding"
                   class="flagImage" title="Flag 'Protein folding' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Protein+structure%22" title="Search for articles in the subject area:'Protein structure'"><div class="flagText">Protein structure</div></a>
              <div data-categoryid="31877" data-articleid="23730"
                   data-categoryname="Protein structure"
                   class="flagImage" title="Flag 'Protein structure' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Salt+bridges%22" title="Search for articles in the subject area:'Salt bridges'"><div class="flagText">Salt bridges</div></a>
              <div data-categoryid="32567" data-articleid="23730"
                   data-categoryname="Salt bridges"
                   class="flagImage" title="Flag 'Salt bridges' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Thermal+stability%22" title="Search for articles in the subject area:'Thermal stability'"><div class="flagText">Thermal stability</div></a>
              <div data-categoryid="24611" data-articleid="23730"
                   data-categoryname="Thermal stability"
                   class="flagImage" title="Flag 'Thermal stability' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=1313'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=7581'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=9954&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>

<div class="block sidebar-comments">
    <div class="header">
        <h3>Comments</h3>
    </div>
      <p><a href="/annotation/listThread.action?root=10473">REMC to study the thermodynamics of folding</a><br>Posted by dsingh</p>
</div>

</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
