

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Human GLI3 Intragenic Conserved Non-Coding Sequences Are Tissue-Specific Enhancers</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0000366"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0000366" />

    <meta name="citation_title" content="Human GLI3 Intragenic Conserved Non-Coding Sequences Are Tissue-Specific Enhancers"/>
    <meta itemprop="name" content="Human GLI3 Intragenic Conserved Non-Coding Sequences Are Tissue-Specific Enhancers"/>

      <meta name="citation_author" content="Amir Ali Abbasi"/>
            <meta name="citation_author_institution" content="Institute of Human Genetics, Philipps-University, Marburg, Germany"/>
      <meta name="citation_author" content="Zissis Paparidis"/>
            <meta name="citation_author_institution" content="Institute of Human Genetics, Philipps-University, Marburg, Germany"/>
      <meta name="citation_author" content="Sajid Malik"/>
            <meta name="citation_author_institution" content="Institute of Human Genetics, Philipps-University, Marburg, Germany"/>
      <meta name="citation_author" content="Debbie K. Goode"/>
            <meta name="citation_author_institution" content="School of Biological and Chemical Sciences, Queen Mary University of London, London, United Kingdom"/>
      <meta name="citation_author" content="Heather Callaway"/>
            <meta name="citation_author_institution" content="School of Biological and Chemical Sciences, Queen Mary University of London, London, United Kingdom"/>
      <meta name="citation_author" content="Greg Elgar"/>
            <meta name="citation_author_institution" content="School of Biological and Chemical Sciences, Queen Mary University of London, London, United Kingdom"/>
      <meta name="citation_author" content="Karl-Heinz Grzeschik"/>
            <meta name="citation_author_institution" content="Institute of Human Genetics, Philipps-University, Marburg, Germany"/>

    <meta name="citation_date" content="2007/4/11"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0000366.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e366"/>
    <meta name="citation_issue" content="4"/>
    <meta name="citation_volume" content="2"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=The phenotypic spectrum of GLI3 morphopathies includes autosomal dominant preaxial polydactyly type-IV and postaxial polydactyly type-A/B; No phenotype prediction from the position of GLI3 mutations.; citation_author=U Radhakrishna; citation_author=D Bornholdt; citation_author=HS Scott; citation_author=UC Patel; citation_author=C Rossier; citation_journal_title=Am J Hum Genet; citation_volume=65; citation_number=1; citation_pages=645-655; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=GLI3 zinc-finger gene interrupted by translocations in Greig syndrome families.; citation_author=A Vortkamp; citation_author=M Gessler; citation_author=KH Grzeschik; citation_journal_title=Nature; citation_volume=352; citation_number=2; citation_pages=539-540; citation_date=1991; " />
      <meta name="citation_reference" content="citation_title=Point mutations in human GLI3 cause Greig syndrome.; citation_author=A Wild; citation_author=M Kalff-Suske; citation_author=A Vortkamp; citation_author=D Bornholdt; citation_author=R Konig; citation_journal_title=Hum Mol Genet; citation_volume=6; citation_number=3; citation_pages=1979-1984; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Point mutations throughout the GLI3 gene cause Greig cephalopolysyndactyly syndrome.; citation_author=M Kalff-Suske; citation_author=A Wild; citation_author=J Topp; citation_author=M Wessling; citation_author=EM Jacobsen; citation_journal_title=Hum Mol Genet; citation_volume=8; citation_number=4; citation_pages=1769-1777; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=GLI3 frameshift mutations cause autosomal dominant Pallister-Hall syndrome.; citation_author=S Kang; citation_author=JM Graham; citation_author=AH Olney; citation_author=LG Biesecker; citation_journal_title=Nat Genet; citation_volume=15; citation_number=5; citation_pages=266-268; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Mutation in GLI3 in postaxial polydactyly type A.; citation_author=U Radhakrishna; citation_author=A Wild; citation_author=KH Grzeschik; citation_author=SE Antonarakis; citation_journal_title=Nat Genet; citation_volume=17; citation_number=6; citation_pages=269-271; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Evidence for allelism of the recessive insertional mutation add and the dominant mouse mutation extra-toes (Xt).; citation_author=TM Pohl; citation_author=MG Mattei; citation_author=U Ruther; citation_journal_title=Development; citation_volume=110; citation_number=7; citation_pages=1153-1157; citation_date=1990; " />
      <meta name="citation_reference" content="citation_title=Expression of the zinc finger gene Gli3 is affected in the morphogenetic mouse mutant extra-toes (Xt).; citation_author=T Schimmang; citation_author=M Lemaistre; citation_author=A Vortkamp; citation_author=U Ruther; citation_journal_title=Development; citation_volume=116; citation_number=8; citation_pages=799-804; citation_date=1992; " />
      <meta name="citation_reference" content="citation_title=A mouse model of greig cephalopolysyndactyly syndrome: the extra-toesJ mutation contains an intragenic deletion of the Gli3 gene.; citation_author=CC Hui; citation_author=AL Joyner; citation_journal_title=Nat Genet; citation_volume=3; citation_number=9; citation_pages=241-246; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=The mouse mutant Polydactyly Nagoya (Pdn) defines a novel allele of the zinc finger gene Gli3.; citation_author=T Schimmang; citation_author=SI Oda; citation_author=U Ruther; citation_journal_title=Mamm Genome; citation_volume=5; citation_number=10; citation_pages=384-386; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=Pallister-Hall syndrome.; citation_author=LG Biesecker; citation_author=F Ondrey; citation_journal_title=Otolaryngol Head Neck Surg; citation_volume=119; citation_number=11; citation_pages=556; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Molecular and clinical analyses of Greig cephalopolysyndactyly and Pallister-Hall syndromes: robust phenotype prediction from the type and position of GLI3 mutations.; citation_author=JJ Johnston; citation_author=I Olivos-Glander; citation_author=C Killoran; citation_author=E Elson; citation_author=JT Turner; citation_journal_title=Am J Hum Genet; citation_volume=76; citation_number=12; citation_pages=609-622; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Pallister-Hall syndrome phenotype in mice mutant for Gli3.; citation_author=J Bose; citation_author=L Grotewold; citation_author=U Ruther; citation_journal_title=Hum Mol Genet; citation_volume=11; citation_number=13; citation_pages=1129-1135; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=The emergent design of the neural tube: prepattern, SHH morphogen and GLI code.; citation_author=A Ruiz i Altaba; citation_author=V Nguyen; citation_author=V Palma; citation_journal_title=Curr Opin Genet Dev; citation_volume=13; citation_number=14; citation_pages=513-521; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Diminished Sonic hedgehog signaling and lack of floor plate differentiation in Gli2 mutant mice.; citation_author=Q Ding; citation_author=J Motoyama; citation_author=S Gasca; citation_author=R Mo; citation_author=H Sasaki; citation_journal_title=Development; citation_volume=125; citation_number=15; citation_pages=2533-2543; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Essential function of Gli2 and Gli3 in the formation of lung, trachea and oesophagus.; citation_author=J Motoyama; citation_author=J Liu; citation_author=R Mo; citation_author=Q Ding; citation_author=M Post; citation_journal_title=Nat Genet; citation_volume=20; citation_number=16; citation_pages=54-57; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Gli1 can rescue the in vivo function of Gli2.; citation_author=CB Bai; citation_author=AL Joyner; citation_journal_title=Development; citation_volume=128; citation_number=17; citation_pages=5161-5172; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Gli2, but not Gli1, is required for initial Shh signaling and ectopic activation of the Shh pathway.; citation_author=CB Bai; citation_author=W Auerbach; citation_author=JS Lee; citation_author=D Stephen; citation_author=AL Joyner; citation_journal_title=Development; citation_volume=129; citation_number=18; citation_pages=4753-4761; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Gli2 functions in FGF signaling during antero-posterior patterning.; citation_author=R Brewster; citation_author=JL Mullor; citation_author=A Ruiz i Altaba; citation_journal_title=Development; citation_volume=127; citation_number=19; citation_pages=4395-4405; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Interplays of Gli2 and Gli3 and their requirement in mediating Shh-dependent sclerotome induction.; citation_author=L Buttitta; citation_author=R Mo; citation_author=CC Hui; citation_author=CM Fan; citation_journal_title=Development; citation_volume=130; citation_number=20; citation_pages=6233-6243; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Dorsal-ventral patterning of the spinal cord requires Gli3 transcriptional repressor activity.; citation_author=M Persson; citation_author=D Stamataki; citation_author=P te Welscher; citation_author=E Andersson; citation_author=J Bose; citation_journal_title=Genes Dev; citation_volume=16; citation_number=21; citation_pages=2865-2878; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=A direct requirement for Hedgehog signaling for normal specification of all ventral progenitor domains in the presumptive mammalian spinal cord.; citation_author=M Wijgerde; citation_author=JA McMahon; citation_author=M Rule; citation_author=AP McMahon; citation_journal_title=Genes Dev; citation_volume=16; citation_number=22; citation_pages=2849-2864; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Parsing the prosencephalon.; citation_author=M Rallu; citation_author=JG Corbin; citation_author=G Fishell; citation_journal_title=Nat Rev Neurosci; citation_volume=3; citation_number=23; citation_pages=943-951; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Spatial pattern of sonic hedgehog signaling through Gli genes during cerebellum development.; citation_author=JD Corrales; citation_author=GL Rocco; citation_author=S Blaess; citation_author=Q Guo; citation_author=AL Joyner; citation_journal_title=Development; citation_volume=131; citation_number=24; citation_pages=5581-5590; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Anteroposterior patterning in the limb and digit specification: Contribution of mouse genetics.; citation_author=B Robert; citation_author=Y Lallemand; citation_journal_title=Dev Dyn; citation_volume=235; citation_number=25; citation_pages=2337-2352; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Gli3 acts as a repressor downstream of Ihh in regulating two distinct steps of chondrocyte differentiation.; citation_author=L Koziel; citation_author=M Wuelling; citation_author=S Schneider; citation_author=A Vortkamp; citation_journal_title=Development; citation_volume=132; citation_number=26; citation_pages=5249-5260; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Gli2 and Gli3 have redundant and context-dependent function in skeletal muscle formation.; citation_author=A McDermott; citation_author=M Gustafsson; citation_author=T Elsam; citation_author=CC Hui; citation_author=CP Emerson; citation_journal_title=Development; citation_volume=132; citation_number=27; citation_pages=345-357; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Sonic hedgehog-patched Gli signaling in the developing rat prostate gland: lobe-specific suppression by neonatal estrogens reduces ductal growth and branching.; citation_author=Y Pu; citation_author=L Huang; citation_author=GS Prins; citation_journal_title=Dev Biol; citation_volume=273; citation_number=28; citation_pages=257-275; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Transcription factor target practice.; citation_author=FC Holstege; citation_author=H Clevers; citation_journal_title=Cell; citation_volume=124; citation_number=29; citation_pages=21-23; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=A long-range Shh enhancer regulates expression in the developing limb and fin and is associated with preaxial polydactyly.; citation_author=LA Lettice; citation_author=SJ Heaney; citation_author=LA Purdie; citation_author=L Li; citation_author=P de Beer; citation_journal_title=Hum Mol Genet; citation_volume=12; citation_number=30; citation_pages=1725-1735; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Highly conserved non-coding sequences are associated with vertebrate development.; citation_author=A Woolfe; citation_author=M Goodson; citation_author=DK Goode; citation_author=P Snell; citation_author=GK McEwen; citation_journal_title=PLoS Biol; citation_volume=3; citation_number=31; citation_pages=e7; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=In vivo enhancer analysis of human conserved non-coding sequences.; citation_author=LA Pennacchio; citation_author=N Ahituv; citation_author=AM Moses; citation_author=S Prabhakar; citation_author=MA Nobrega; citation_journal_title=Nature; citation_volume=444; citation_number=32; citation_pages=499-502; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Evolution and functional classification of vertebrate gene deserts.; citation_author=I Ovcharenko; citation_author=GG Loots; citation_author=MA Nobrega; citation_author=RC Hardison; citation_author=W Miller; citation_journal_title=Genome Res; citation_volume=15; citation_number=33; citation_pages=137-145; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Ultraconserved elements in the human genome.; citation_author=G Bejerano; citation_author=M Pheasant; citation_author=I Makunin; citation_author=S Stephen; citation_author=WJ Kent; citation_journal_title=Science; citation_volume=304; citation_number=34; citation_pages=1321-1325; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Human chromosome 7: DNA sequence and biology.; citation_author=SW Scherer; citation_author=J Cheung; citation_author=JR MacDonald; citation_author=LR Osborne; citation_author=K Nakabayashi; citation_journal_title=Science; citation_volume=300; citation_number=35; citation_pages=767-772; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Greig syndrome in a large kindred due to reciprocal chromosome translocation t(6;7)(q27;p13).; citation_author=G Kruger; citation_author=J Gotz; citation_author=U Kvist; citation_author=H Dunker; citation_author=F Erfurth; citation_journal_title=Am J Med Genet; citation_volume=32; citation_number=36; citation_pages=411-416; citation_date=1989; " />
      <meta name="citation_reference" content="citation_title=Molecular linkage of the morphogenetic mutation add and the zinc finger gene Gli3.; citation_author=F van der Hoeven; citation_author=T Schimmang; citation_author=A Vortkamp; citation_author=U Ruther; citation_journal_title=Mamm Genome; citation_volume=4; citation_number=37; citation_pages=276-277; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=Sonic hedgehog expression in Gli3 depressed mouse embryo, Pdn/Pdn.; citation_author=E Ueta; citation_author=M Maekawa; citation_author=I Morimoto; citation_author=E Nanba; citation_author=I Naruse; citation_journal_title=Congenit Anom (Kyoto); citation_volume=44; citation_number=38; citation_pages=27-32; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Computational screening of conserved genomic DNA in search of functional noncoding elements.; citation_author=G Bejerano; citation_author=AC Siepel; citation_author=WJ Kent; citation_author=D Haussler; citation_journal_title=Nat Methods; citation_volume=2; citation_number=39; citation_pages=535-545; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Shuffling of cis-regulatory elements is a pervasive feature of the vertebrate lineage.; citation_author=R Sanges; citation_author=E Kalmar; citation_author=P Claudiani; citation_author=M D'Amato; citation_author=F Muller; citation_journal_title=Genome Biol; citation_volume=7; citation_number=40; citation_pages=R56; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Evaluation of the RET regulatory landscape reveals the biological relevance of a HSCR-implicated enhancer.; citation_author=EA Grice; citation_author=ES Rochelle; citation_author=ED Green; citation_author=A Chakravarti; citation_author=AS McCallion; citation_journal_title=Hum Mol Genet; citation_volume=14; citation_number=41; citation_pages=3837-3845; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Direct regulation of knot gene expression by Ultrabithorax and the evolution of cis-regulatory elements in Drosophila.; citation_author=BM Hersh; citation_author=SB Carroll; citation_journal_title=Development; citation_volume=132; citation_number=42; citation_pages=1567-1577; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=DNA binding sites: representation and discovery.; citation_author=GD Stormo; citation_journal_title=Bioinformatics; citation_volume=16; citation_number=43; citation_pages=16-23; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Comparative genomics at the vertebrate extremes.; citation_author=D Boffelli; citation_author=MA Nobrega; citation_author=EM Rubin; citation_journal_title=Nat Rev Genet; citation_volume=5; citation_number=44; citation_pages=456-465; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=In vivo characterization of a vertebrate ultraconserved enhancer.; citation_author=F Poulin; citation_author=MA Nobrega; citation_author=I Plajzer-Frick; citation_author=A Holt; citation_author=V Afzal; citation_journal_title=Genomics; citation_volume=85; citation_number=45; citation_pages=774-781; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=rVISTA 2.0: evolutionary analysis of transcription factor binding sites.; citation_author=GG Loots; citation_author=I Ovcharenko; citation_journal_title=Nucleic Acids Res; citation_volume=32; citation_number=46; citation_pages=W217-221; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=ConSite: web-based prediction of regulatory elements using cross-species comparison.; citation_author=A Sandelin; citation_author=WW Wasserman; citation_author=B Lenhard; citation_journal_title=Nucleic Acids Res; citation_volume=32; citation_number=47; citation_pages=W249-252; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Highly conserved regulatory elements around the SHH gene may contribute to the maintenance of conserved synteny across human chromosome 7q36.3.; citation_author=DK Goode; citation_author=P Snell; citation_author=SF Smith; citation_author=JE Cooke; citation_author=G Elgar; citation_journal_title=Genomics; citation_volume=86; citation_number=48; citation_pages=172-181; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Ancient duplicated conserved noncoding elements in vertebrates: a genomic and functional analysis.; citation_author=GK McEwen; citation_author=A Woolfe; citation_author=D Goode; citation_author=T Vavouri; citation_author=H Callaway; citation_journal_title=Genome Res; citation_volume=16; citation_number=49; citation_pages=451-465; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Intronic enhancers control expression of zebrafish sonic hedgehog in floor plate and notochord.; citation_author=F Muller; citation_author=B Chang; citation_author=S Albert; citation_author=N Fischer; citation_author=L Tora; citation_journal_title=Development; citation_volume=126; citation_number=50; citation_pages=2103-2116; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Zebrafish Gli3 functions as both an activator and a repressor in Hedgehog signaling.; citation_author=OV Tyurina; citation_author=B Guner; citation_author=E Popova; citation_author=J Feng; citation_author=AF Schier; citation_journal_title=Dev Biol; citation_volume=277; citation_number=51; citation_pages=537-556; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Gli function is essential for motor neuron induction in zebrafish.; citation_author=G Vanderlaan; citation_author=OV Tyurina; citation_author=RO Karlstrom; citation_author=A Chandrasekhar; citation_journal_title=Dev Biol; citation_volume=282; citation_number=52; citation_pages=550-570; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Conservation of RET regulatory function from human to zebrafish without sequence similarity.; citation_author=S Fisher; citation_author=EA Grice; citation_author=RM Vinton; citation_author=SL Bessling; citation_author=AS McCallion; citation_journal_title=Science; citation_volume=312; citation_number=53; citation_pages=276-279; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=New technologies, new findings, and new concepts in the study of vertebrate cis-regulatory sequences.; citation_author=JL Gomez-Skarmeta; citation_author=B Lenhard; citation_author=TS Becker; citation_journal_title=Dev Dyn; citation_volume=235; citation_number=54; citation_pages=870-885; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=The zebrafish book: A guide for the laboratory use of zebrafish (Danio rerio); citation_author=M Westerfield; citation_number=55; citation_date=2000; citation_publisher=University of Oregon Press; " />
      <meta name="citation_reference" content="citation_title=Techniques in neural development.; citation_author=CB Moens; citation_author=A Fritz; citation_journal_title=Methods Cell Biol; citation_volume=59; citation_number=56; citation_pages=253-272; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Glocal alignment: finding rearrangements during alignment.; citation_author=M Brudno; citation_author=S Malde; citation_author=A Poliakov; citation_author=CB Do; citation_author=O Couronne; citation_journal_title=Bioinformatics; citation_number=57; citation_pages=i54-62; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=VISTA : visualizing global DNA sequence alignments of arbitrary length.; citation_author=C Mayor; citation_author=M Brudno; citation_author=JR Schwartz; citation_author=A Poliakov; citation_author=EM Rubin; citation_journal_title=Bioinformatics; citation_volume=16; citation_number=58; citation_pages=1046-1047; citation_date=2000; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000366" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Human GLI3 Intragenic Conserved Non-Coding Sequences Are Tissue-Specific Enhancers"/>
    <meta name="twitter:description" content="The zinc-finger transcription factor GLI3 is a key regulator of development, acting as a primary transducer of Sonic hedgehog (SHH) signaling in a combinatorial context dependent fashion controlling multiple patterning steps in different tissues/organs. A tight temporal and spatial control of gene expression is indispensable, however, cis-acting sequence elements regulating GLI3 expression have not yet been reported. We show that 11 ancient genomic DNA signatures, conserved from the pufferfish Takifugu (Fugu) rubripes to man, are distributed throughout the introns of human GLI3. They map within larger conserved non-coding elements (CNEs) that are found in the tetrapod lineage. Full length CNEs transiently transfected into human cell cultures acted as cell type specific enhancers of gene transcription. The regulatory potential of these elements is conserved and was exploited to direct tissue specific expression of a reporter gene in zebrafish embryos. Assays of deletion constructs revealed that the human-Fugu conserved sequences within the GLI3 intronic CNEs were essential but not sufficient for full-scale transcriptional activation. The enhancer activity of the CNEs is determined by a combinatorial effect of a core sequence conserved between human and teleosts (Fugu) and flanking tetrapod-specific sequences, suggesting that successive clustering of sequences with regulatory potential around an ancient, highly conserved nucleus might be a possible mechanism for the evolution of cis-acting regulatory elements."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0000366.g007"/>

  <meta property="og:title" content="Human GLI3 Intragenic Conserved Non-Coding Sequences Are Tissue-Specific Enhancers" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000366" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=5087'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=9856'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=8904&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0000366">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000366" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000366&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Human <i>GLI3</i> Intragenic Conserved Non-Coding Sequences Are Tissue-Specific Enhancers
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Amir Ali Abbasi, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Institute of Human Genetics, Philipps-University, Marburg, Germany
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Zissis Paparidis, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Institute of Human Genetics, Philipps-University, Marburg, Germany
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Sajid Malik, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Institute of Human Genetics, Philipps-University, Marburg, Germany
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Debbie K. Goode, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  School of Biological and Chemical Sciences, Queen Mary University of London, London, United Kingdom
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Heather Callaway, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  School of Biological and Chemical Sciences, Queen Mary University of London, London, United Kingdom
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Greg Elgar, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  School of Biological and Chemical Sciences, Queen Mary University of London, London, United Kingdom
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Karl-Heinz Grzeschik
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:grzeschi@staff.uni-marburg.de">grzeschi@staff.uni-marburg.de</a></p>

                <p>Affiliation:
                  Institute of Human Genetics, Philipps-University, Marburg, Germany
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: April 11, 2007</li>
    <li>DOI: 10.1371/journal.pone.0000366</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0000366" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0000366" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000366" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0000366" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0000366-g001" data-doi="info:doi/10.1371/journal.pone.0000366" data-uri="info:doi/10.1371/journal.pone.0000366.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000366.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000366-t001" data-doi="info:doi/10.1371/journal.pone.0000366" data-uri="info:doi/10.1371/journal.pone.0000366.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000366.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000366-g002" data-doi="info:doi/10.1371/journal.pone.0000366" data-uri="info:doi/10.1371/journal.pone.0000366.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000366.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000366-g003" data-doi="info:doi/10.1371/journal.pone.0000366" data-uri="info:doi/10.1371/journal.pone.0000366.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000366.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000366-g004" data-doi="info:doi/10.1371/journal.pone.0000366" data-uri="info:doi/10.1371/journal.pone.0000366.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000366.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000366-g005" data-doi="info:doi/10.1371/journal.pone.0000366" data-uri="info:doi/10.1371/journal.pone.0000366.g005" title="Figure 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000366.g005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000366-g006" data-doi="info:doi/10.1371/journal.pone.0000366" data-uri="info:doi/10.1371/journal.pone.0000366.g006" title="Figure 6">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000366.g006&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000366-g007" data-doi="info:doi/10.1371/journal.pone.0000366" data-uri="info:doi/10.1371/journal.pone.0000366.g007" title="Figure 7">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000366.g007&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000366">Reader Comments (0)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0000366" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1"></a><p>The zinc-finger transcription factor GLI3 is a key regulator of development, acting as a primary transducer of Sonic hedgehog (<em>SHH</em>) signaling in a combinatorial context dependent fashion controlling multiple patterning steps in different tissues/organs. A tight temporal and spatial control of gene expression is indispensable, however, <em>cis</em>-acting sequence elements regulating <em>GLI3</em> expression have not yet been reported. We show that 11 ancient genomic DNA signatures, conserved from the pufferfish <em>Takifugu (Fugu) rubripes</em> to man, are distributed throughout the introns of human <em>GLI3.</em> They map within larger conserved non-coding elements (CNEs) that are found in the tetrapod lineage. Full length CNEs transiently transfected into human cell cultures acted as cell type specific enhancers of gene transcription. The regulatory potential of these elements is conserved and was exploited to direct tissue specific expression of a reporter gene in zebrafish embryos. Assays of deletion constructs revealed that the human-<em>Fugu</em> conserved sequences within the <em>GLI3</em> intronic CNEs were essential but not sufficient for full-scale transcriptional activation. The enhancer activity of the CNEs is determined by a combinatorial effect of a core sequence conserved between human and teleosts <em>(Fugu)</em> and flanking tetrapod-specific sequences, suggesting that successive clustering of sequences with regulatory potential around an ancient, highly conserved nucleus might be a possible mechanism for the evolution of cis-acting regulatory elements.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>Abbasi AA, Paparidis Z, Malik S, Goode DK, Callaway H, et al.  (2007) Human <em>GLI3</em> Intragenic Conserved Non-Coding Sequences Are Tissue-Specific Enhancers. PLoS ONE 2(4):
          e366.
            doi:10.1371/journal.pone.0000366</p><p><strong>Academic Editor: </strong>Jean-Nicolas Volff, Ecole Normale Supérieure de Lyon, France</p><p><strong>Received:</strong> February 2, 2007; <strong>Accepted:</strong> March 19, 2007; <strong>Published:</strong> April 11, 2007</p><p><strong>Copyright:</strong> © 2007 Abbasi et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>This work was funded by Deutsche Forschungsgemeinschaft (Gr373-21; GRK 767) and Stiftung P.E.Kempkes, Marburg, Germany. A.A.A. holds a fellowship of Higher Education Commission of Pakistan.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>Mutations in the human transcription factor GLI3 cause a variety of dominant developmental defect syndromes, subsumed under the term “GLI3 morphopathies” <a href="#pone.0000366-Radhakrishna1">[1]</a>, including Greig cephalopolysyndactyly syndrome (GCPS) <a href="#pone.0000366-Vortkamp1">[2]</a>–<a href="#pone.0000366-KalffSuske1">[4]</a>, Pallister-Hall syndrome (PHS) <a href="#pone.0000366-Kang1">[5]</a>, postaxial polydactyly type A (PAPA) <a href="#pone.0000366-Radhakrishna2">[6]</a>, and preaxial polydactyly type IV (PPD-IV) <a href="#pone.0000366-Radhakrishna1">[1]]</a>. Mutations affecting murine <em>Gli3</em>, such as extra toes (<em>Xt</em>), anterior digit deformity (<em>add</em>), and polydactyly Nagoya (<em>Pdn</em>) serve as models for GLI3 morphopathies <a href="#pone.0000366-Pohl1">[7]</a>–<a href="#pone.0000366-Schimmang2">[10]</a>. All GLI3 morphopathies show malformations of the autopod, i.e. polydactylies or syndactylies. In addition, craniofacial abnormalities are associated with GCPS, and in the most severe form, PHS, other developmental malformations occur, such as hypothalamic hamartoma, visceral anomalies, anus atresy, epiglottis and larynx defects <a href="#pone.0000366-Biesecker1">[11]</a>.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>Genotype-phenotype correlation has been reported for Pallister-Hall syndrome with mutations deleting the C-terminal part of GLI3, 3′of the zinc finger encoding domain, leaving the DNA-binding domain intact <a href="#pone.0000366-Johnston1">[12]</a>, <a href="#pone.0000366-Bose1">[13]</a>. Functional haploinsufficiency of <em>GLI3</em> appears to cause GCPS, since deletions or translocations eliminating one allele as well as missense or nonsense mutations distributed over the entire coding sequence are associated with this phenotype <a href="#pone.0000366-Vortkamp1">[2]</a>, <a href="#pone.0000366-KalffSuske1">[4]</a>, <a href="#pone.0000366-Johnston1">[12]</a>.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>The transcription factor GLI3, together with its paralogues GLI1 and GLI2, acts as a primary transducer of Sonic hedgehog (SHH) signaling in a context dependent combinatorial fashion <a href="#pone.0000366-RuiziAltaba1">[14]</a>. GLI3 and GLI2 can act both as transcriptional activators or repressors whereas GLI1, whose expression is transcriptionally regulated by GLI2 and GLI3, appears to play a secondary role in potentiating the SHH response <a href="#pone.0000366-Ding1">[15]</a>–<a href="#pone.0000366-Bai2">[18]</a>. In murine embryos, the locations affected in human GLI3 morphopathies, in particular the forebrain and the autopod, show strong <em>Gli3</em> expression <a href="#pone.0000366-Schimmang1">[8]</a>. In humans, a lower level of GLI3 in these locations due to haploinsufficiency is inadequate for normal development. Apparently, the amount of gene product produced by one <em>GLI3</em> allele is sufficient in most other locations.</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p>Mouse embryos with homozygous <em>Gli3</em> deficiency show pleiotropic and lethal congenital malformations with distinct preaxial limb polydactylies <a href="#pone.0000366-Schimmang1">[8]</a>, <a href="#pone.0000366-Hui1">[9]</a>. A multitude of studies in mice and other model organisms have proven that a GLI-code, the interplay of GLI proteins and the temporally fine tuned expression of the <em>GLI</em> genes in adjacent domains, together provide a basic tool that is used over and over again in embryonal development. This is applicable to patterning along the anteroposterior axis <a href="#pone.0000366-Brewster1">[19]</a>, induction of sclerotome <a href="#pone.0000366-Buttitta1">[20]</a>, morphogenesis of the neural tube <a href="#pone.0000366-Persson1">[21]</a>, <a href="#pone.0000366-RuiziAltaba1">[14]</a>, <a href="#pone.0000366-Wijgerde1">[22]</a>, the prosencephalon <a href="#pone.0000366-Rallu1">[23]</a>, and cerebellum <a href="#pone.0000366-Corrales1">[24]</a>, anterior-posterior limb patterning <a href="#pone.0000366-Robert1">[25]</a>, chondrocyte differentiation <a href="#pone.0000366-Koziel1">[26]</a>, skeletal muscle formation <a href="#pone.0000366-McDermott1">[27]</a> and prostate gland development <a href="#pone.0000366-Pu1">[28]</a>. These data indicate that GLI3 has essential functions controlling multiple patterning steps in different tissues/organs, and therefore a tight temporal and spatial control of gene expression is indispensable.</p>
<a id="article1.body1.sec1.p5" name="article1.body1.sec1.p5"></a><p>The identification of <em>cis</em>-acting regulatory elements interacting with the <em>GLI3</em> promoter could facilitate the detection of factors controlling the tissue specific availability of GLI3 <em>in trans</em> in Hedgehog (HH) target cells. In turn, identification of transcription factors for spatial and temporal control of <em>GLI3</em> expression would greatly enhance our understanding of the regulatory network that coordinates the multitude of patterning events associated with the HH signaling pathway. Mammalian enhancers can be defined by a combinatorial code for an assembly of transcription factor binding sites (TFBS), but <em>in silico</em> identification has proven difficult. This is firstly due to the paucity of information about TF binding specificity, confined to a set of loose consensus binding motifs. Secondly, transcription factors generally recognize only six to eight base-pair DNA motifs, and the distance over which they may be located around a particular gene could be vast <a href="#pone.0000366-Holstege1">[29]</a>. Enhancer elements have been observed at a distance of more than a megabase from their target gene <a href="#pone.0000366-Lettice1">[30]</a>. To narrow the sequence intervals to be scrutinized experimentally for <em>cis</em>-acting regulatory potential, multispecies highly conserved non-coding sequences (CNEs) have been targeted <a href="#pone.0000366-Woolfe1">[31]</a>. CNEs are much more conserved than the sequences of known enhancers, but many of these elements clearly regulate gene expression <a href="#pone.0000366-Pennacchio1">[32]</a>. They also might play a role in other processes, e.g. as repressors, replication origins or modulators of chromatin structure. The reason for the strong evolutionary constraint over extended lengths of DNA sequence is not known. Sequence conservation of <em>cis</em>-regulatory elements of transcription within CNEs might date back to the period in evolution when the new patterns that they determine were added to a basic body plan. Non-coding sequence elements conserved from <em>Fugu</em> to man might harbour enhancers directing a basic outline common to the two distantly related vertebrates, whereas tetrapod specific CNEs might only contain regulatory elements for later additions to the body plan, such as an autopod with digits.</p>
<a id="article1.body1.sec1.p6" name="article1.body1.sec1.p6"></a><p>As an initial attempt to identify and characterize the regulatory code directing human <em>GLI3</em> expression, we have applied reporter gene assays to test the regulatory potential of 11 intronic <em>Fugu</em>-human CNEs in cultured cells with or without endogenous <em>GLI3</em> expression. All elements are able to regulate expression in a cell type dependent fashion. The elements identified as potential enhancers extend beyond the <em>Fugu</em>-human highly conserved core sequences into flanking, less well conserved DNA. These core sequences are necessary but not sufficient for full regulatory potential. By expressing reporter genes under the control of the human <em>GLI3</em>-CNEs in zebrafish embryos, we demonstrate that the activating or repressor potential of CNEs observed in human cell culture transient transfection assays is retained <em>in vivo</em> in a teleost fish. Enhancers with activating potential differ in their tissue specificity, however, none of them direct expression exclusively in one tissue. Nevertheless, to a large extent reporter gene expression patterns mimic endogenous zebrafish <em>gli3</em> expression. We conclude that human-<em>Fugu</em> CNEs, located in the introns of <em>GLI3,</em> mark critical components of the cis-regulatory inventory for temporal and spatial expression control of this key developmental gene.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3>
<h4><em>GLI3</em> Tetrapod-Teleost Conserved Non-coding Elements (CNEs) are located exclusively within introns</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>The pufferfish <em>gli3</em> (scaffold_210; ENSEMBL genome browser) is tightly bordered by genes that are not orthologous to the human <em>GLI3</em> flanking regions. Therefore, it is more likely that non-coding sequences conserved between human and <em>Fugu</em> and which might be potential enhancers, are restricted to <em>GLI3</em> introns. <em>GLI3</em> is flanked by variable gene desert <a href="#pone.0000366-Ovcharenko1">[33]</a>. Comparison of approximately 1 Mb human genomic DNA sequence encompassing <em>GLI3</em> and extending up to the flanking genes with the complete assembly of the <em>Takifugu rubripes</em> genome sequence indicates that sequence homology is restricted to the gene region proper (<a href="#pone-0000366-g001">Figure 1</a>).</p>
<div class="figure" id="pone-0000366-g001"><div class="img"><a name="pone-0000366-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000366.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000366" data-uri="info:doi/10.1371/journal.pone.0000366.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000366.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000366.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000366.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000366.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000366.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000366.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000366.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000366.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000366.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1. </strong></p><a id="article1.body1.sec2.sec1.fig1.caption1.p1" name="article1.body1.sec2.sec1.fig1.caption1.p1"></a><p>Comparative Sequence Analysis of the <em>GLI3</em> Locus Detects Conserved Non-coding Sequence Elements. (A) Sequence alignments of the genomic interval containing the human <em>GLI3</em> locus and flanking human genes <em>INHBA</em> and <em>PSMA2</em> with orthologous counterparts from representative members of rodent, bird, amphibian, and fish lineages. These are shown as SLAGAN derived VISTA representations. Conserved coding sequences are depicted in blue and conserved non-coding sequences are in pink. Criteria of alignment were 60 bp window and 50% conservation cutoff. Conservation between human and <em>Fugu</em> (scaffold_210 ENSEMBL genome browser) is restricted to the <em>GLI3</em> gene. Red bars above the conservation plot depict the approximate length of intergenic regions flanking human <em>GLI3.</em> The blue arrow shows the length of the <em>GLI3</em> gene and the direction of transcription. A graphic representation showing exons and introns of <em>GLI3</em> is shown below the homology plot. Green vertical lines indicate the positions of alterations affecting the genomic structure of the locus which result in loss of GLI3 function: a translocation event associated with Greig cephalopolysyndactyly syndrome (GCPS) <a href="#pone.0000366-Kruger1">[36]</a>, and two insertions (ins) in mouse mutants anterior digit pattern deformity (<em>add</em>) <a href="#pone.0000366-Pohl1">[7]</a>, <a href="#pone.0000366-vanderHoeven1">[37]</a> and polydactyly Nagoya (<em>Pdn</em>) <a href="#pone.0000366-Ueta1">[38]</a>. (B) Magnified view of the human/<em>Fugu</em> conservation plot and the genomic structure of human <em>GLI3</em>. The red vertical bars below the plot show the position of human/<em>Fugu</em> highly conserved non-coding sequence elements (CNEs) that were functionally tested as putative enhancers.</p>
<span>doi:10.1371/journal.pone.0000366.g001</span></div><a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2"></a><p>Multi-species alignment of <em>GLI3</em> genomic sequences from mammals revealed extensive conservation, which obscured the identification of potentially functional elements embedded in intronic DNA (<a href="#pone-0000366-g001">Figure 1A</a>). However, in the transition from moderate (mammalian sequence comparison) to intermediate evolutionary distance (human vs birds/amphibia) the extent of neutrally evolving sequences dropped sharply, whilst sequence comparison at an extreme phylogenetic distance (human/teleost) reduced the number of candidates further. This allowed us to prioritize 11 CNEs for functional analysis.</p>
<a id="article1.body1.sec2.sec1.p3" name="article1.body1.sec2.sec1.p3"></a><p>These CNEs are distributed across almost the entire <em>GLI3</em> interval (<a href="#pone-0000366-g001">Figure 1B</a>), with 2 elements in each of introns 2, 3, 4, and 10 and one in each of introns 1, 6, and 13. The amplicons encompassing ancient signatures within flanking sequences highly conserved only in mammals are described in <a href="#pone-0000366-t001">Table 1</a>. CNE2 represents an ultraconserved element <a href="#pone.0000366-Bejerano1">[34]</a> (&gt;200 bp at 100% identity in human, rat and mouse) and will be described elsewhere. A further element, CNE8, located in intron 10 has not yet been tested functionally. Using both extrinsic and <em>ab initio</em> approaches embedded at the UCSC browser and the Ensembl gene build system, we found no evidence for overlap with putative protein coding regions or non-coding RNA genes. In each of the 11 CNEs we predict transcription factor binding sites (TFBSs) for established developmental regulators (<a href="#pone-0000366-t001">Table 1</a>) using the programs Consite and rVista v 2.</p>
<div class="figure" id="pone-0000366-t001"><div class="img"><a name="pone-0000366-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000366.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000366" data-uri="info:doi/10.1371/journal.pone.0000366.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000366.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000366.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000366.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000366.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000366.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000366.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000366.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000366.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000366.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>Tetrapod-Teleost Conserved Non-Coding elements (CNEs) from Introns of Human <em>GLI3</em> Selected for Functional Analysis</span></strong></p><span>doi:10.1371/journal.pone.0000366.t001</span></div>

<h4>Cell Based Reporter Assays Reveal a Context Dependent Dual Nature (Activator/Repressor) of CNEs</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>In order to test the selected subset of 10 sequence elements for their potential to regulate reporter gene expression, recombinant constructs with CNEs placed in either orientation upstream of a luciferase gene controlled by either the heterologous SV40 promoter or the human minimal <em>GLI3</em> promoter (<a href="#pone-0000366-g002">Figure 2A</a>), were transiently transfected into two human kidney fibroblast lines. The H661 cell line expresses endogenous <em>GLI3</em> whereas H441 does not express this gene (data not shown). In dual luciferase assays eight elements (CNE 1, 5, 6, 7, 9, 10, 11, and 12) showed activating potential in H661 cells whereas two elements (CNE3 and CNE4) repressed reporter gene expression below the level achieved by either promoter alone (<a href="#pone-0000366-g002">Figure 2B</a>).</p>
<div class="figure" id="pone-0000366-g002"><div class="img"><a name="pone-0000366-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000366.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000366" data-uri="info:doi/10.1371/journal.pone.0000366.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000366.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000366.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000366.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000366.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000366.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000366.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000366.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000366.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000366.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2. </strong></p><a id="article1.body1.sec2.sec2.fig1.caption1.p1" name="article1.body1.sec2.sec2.fig1.caption1.p1"></a><p>CNEs Regulate Luciferase Reporter Gene Expression in Transiently Transfected Human Cell Lines. (A) Diagrams of the reporter constructs employed to test the regulatory potential of CNEs from the introns of human <em>GLI3</em>. CNEs were cloned into in a pGL3-Basic vector containing either a minimal <em>GLI3</em> promoter (pGL3-CNE-prom<em>GLI3</em>-300-luc) or a heterologous SV40 promoter (pGL3-CNE promSV40-luc) upstream of a firefly luciferase gene. (B) Luciferase activity of reporter constructs in human H661 cells that express endogenous <em>GLI3</em>. (C) Luciferase activity of reporter constructs in human H441 cells that do not express endogenous <em>GLI3</em>. The pGL3-Basic vector, with no promoter/enhancer insert was used as a negative control. Luciferase activity in cells transiently transfected with the positive control, a construct containing a SV40 promoter upstream of the reporter gene (pGL3-promSV40-luc), was taken as 100% (blue dotted line). A plasmid expressing <em>Renilla</em> luciferase was co-transfected as a standard for transcription efficiency. Average firefly luciferase reporter activities relative to <em>Renilla</em> luciferase activity from three triplicate transfection experiments are depicted as percentage of activity obtained with the positive control vector (B, C). Standard errors of the mean are shown. Black dotted lines indicate the luciferase expression level reached in each cell line with the pGL3-prom<em>GLI3</em>-300-luc vector.</p>
<span>doi:10.1371/journal.pone.0000366.g002</span></div><a id="article1.body1.sec2.sec2.p2" name="article1.body1.sec2.sec2.p2"></a><p>In contrast, when tested in the H441 cell line, all CNEs exhibited a strong repressing activity (<a href="#pone-0000366-g002">Figure 2C</a>). Thus, the cell based reporter assay identified two categories of intra-<em>GLI3</em> regulatory elements: firstly context independent repressors and secondly enhancers with a context dependent dual nature, serving as activators in a GLI3 positive context and as repressors in cells without endogenous <em>GLI3</em> expression.</p>


<h4><em>In Vivo</em> Functional Analysis of CNEs with Transiently Transfected Zebrafish Embryos</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>The CNEs that have been tested <em>in vitro</em> were next tested <em>in vivo</em> using zebrafish as a model organism. CNEs were co-injected with a GFP reporter into zebrafish embryos and then monitored for enhancer activity at set time points.</p>
<a id="article1.body1.sec2.sec3.p2" name="article1.body1.sec2.sec3.p2"></a><p>With the exception of CNE5, the <em>in vitro</em> identified cellular context dependent enhancer elements drove GFP expression in significant proportion of microinjected zebrafish embryos (<a href="#pone-0000366-g003">Figures 3</a> and <a href="#pone-0000366-g004">4</a>), whereas neither CNE3 nor CNE4 could induce reporter gene expression in fish embryos. At day two of development (~26–33 hours post fertilization, hpf), CNE1 directs GFP expression prominently in various subdivisions of CNS, forebrain, midbrain and hindbrain with 22%, 32% and 58% of expressing embryos respectively. Within the cardiac chambers, GFP expression induced by CNE1 was observed in 10% of expressing embryos at day two and in 30% on day 3 of development (~50–54 hpf). Reporter gene expression was also observed in blood cells of day 2 embryos (12%) skin (19%) and developing median fin fold (32%).</p>
<div class="figure" id="pone-0000366-g003"><div class="img"><a name="pone-0000366-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000366.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000366" data-uri="info:doi/10.1371/journal.pone.0000366.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000366.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000366.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000366.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000366.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000366.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000366.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000366.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000366.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000366.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3. </strong></p><a id="article1.body1.sec2.sec3.fig1.caption1.p1" name="article1.body1.sec2.sec3.fig1.caption1.p1"></a><p>Sites of GFP Expression Induced by <em>GLI3</em>-Associated CNEs in Zebrafish Embryos. Upregulation of GFP by individual <em>GLI3</em>-associated CNEs (indicated by name and location in a <em>GLI3</em>-intron) depicted in schematic representations of day two, 24–33 hpf (D2) or day 3 (D3) embryos. N = the total number of positive embryos per CNE. Categories of cell type that were positive for a given element are color coded, and each dot represents a single GFP positive cell. These are mapped onto camera lucida drawings of the zebrafish embryo, and the overall results are overlaid. The percentage of positive embryos that show expression in each color coded tissue category are shown on the bar charts.</p>
<span>doi:10.1371/journal.pone.0000366.g003</span></div><div class="figure" id="pone-0000366-g004"><div class="img"><a name="pone-0000366-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000366.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000366" data-uri="info:doi/10.1371/journal.pone.0000366.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000366.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000366.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000366.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000366.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000366.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000366.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000366.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000366.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000366.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4. </strong></p><a id="article1.body1.sec2.sec3.fig2.caption1.p1" name="article1.body1.sec2.sec3.fig2.caption1.p1"></a><p>Tissue Type Specific Expression of GFP Reporter Gene in Zebrafish Embryos. Examples of GFP expression induced by CNEs 1, 9, 10, and 11 are shown in fixed tissues after wholemount anti-GFP immunostaining (bright field views A and F) or in live embryos by combined bright field and GFP fluorescence microscopy analyses (B, C, D, E, G and H). Arrowheads indicate GFP expressing cells. Embryos C and D are ~26–33 hpf, while embryos A, B, E, F, G, and H are 48–54 hpf. Lateral views, anterior to the left and dorsal to the top except for F where the dorsal view is shown. GFP positive cells were found in the following: (A) CNE1, heart chamber (B) CNE1, hindbrain neurons (C) CNE9, notochord (D) CNE9, spinal cord neuron (E) CNE10, lower jaw primordia and pericardial regions (F) CNE10, lens epithelial cell layer (G) CNE11, pectoral fin (H) CNE11, muscle. (e) Eye; (f) fin; (h) heart; (hb) hindbrain; (I) lens; (nc) notochord; (ov) otic vesicle; (r) retina; (s) spinal cord; (y) yolk.</p>
<span>doi:10.1371/journal.pone.0000366.g004</span></div><a id="article1.body1.sec2.sec3.p3" name="article1.body1.sec2.sec3.p3"></a><p>CNE10 directs reporter gene expression most frequently in eye (54% of expressing embryos), pericardial region (57%) and skin cells (48%). Within the eye, CNE10 mediated reporter expression in retinal ganglion cells, the photoreceptor layer at the retinal margin, the lens epithelial cell layer and the lens nuclear region. CNE10 also induces GFP expression in the lower jaw primordia or first pharyngeal arch (mandibular arch) region in significant proportion (24%) of day 2 (~26–33 hpf) expressing embryos.</p>
<a id="article1.body1.sec2.sec3.p4" name="article1.body1.sec2.sec3.p4"></a><p>CNE7 did induce reporter gene expression in different regions of day 2 embryos (~26–33 hpf) but the activity was not particularly strong in any one tissue/region of the embryo.</p>
<a id="article1.body1.sec2.sec3.p5" name="article1.body1.sec2.sec3.p5"></a><p>CNE6 drove GFP expression most prominently in the spinal cord neurons (21% of expressing embryos), and less frequently in hindbrain neurons immediately flanking the hindbrain/spinal cord boundary (10%), in blood cells (17%), and muscle fibers (10%).</p>
<a id="article1.body1.sec2.sec3.p6" name="article1.body1.sec2.sec3.p6"></a><p>CNE11 activity on day 2 of development was confined to skin cells (64% of expressing embryos), muscle fibers (30%) and heart (30%). In contrast to other elements, which drove expression mainly on day 2 (~26–33 hpf) of development, CNE11 also strongly enhanced reporter expression on day 3 (50–54 hpf) of development (<a href="#pone-0000366-g003">Figure 3</a>), within heart chambers (55%) skin cells (25%), muscle fibers (12%), with some expression in the pectoral fins.</p>
<a id="article1.body1.sec2.sec3.p7" name="article1.body1.sec2.sec3.p7"></a><p>A particularly prominent GFP expression domain for CNE9 injected embryos was in notochord cells (74% of GFP expressing embryos). In addition, reporter gene expression occurred in spinal cord (14%), forebrain (11%), hindbrain (11%), skin cells (20%), fin (14%), and muscle fibers (11%).</p>


<h4><em>In Vitro</em> Deletion Analysis of Selected Sub-set of CNEs</h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>In order to define functionally critical regions within CNEs and to understand the significance of strength of evolutionary constraints on defining their overall activity, we prioritized three elements CNE1, CNE5, and CNE6 for dissection and subsequent analysis of the fragments by transient transfection assays in H661 cells. CNE1 spans a 945 bp human/fish conserved track with overall human/fish sequence similarity of ~71%. Close inspection of CNE1 revealed a sequence block of ~125 bp (<em>hcCNE1-125bp</em>) under particularly strong negative selection, almost unaltered in human/mouse and human/chick sequence comparisons, whilst a human/<em>Fugu</em> comparison shows ~92% sequence identity (<a href="#pone-0000366-g005">Figure 5A</a>).</p>
<div class="figure" id="pone-0000366-g005"><div class="img"><a name="pone-0000366-g005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000366.g005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000366" data-uri="info:doi/10.1371/journal.pone.0000366.g005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000366.g005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000366.g005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000366.g005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000366.g005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000366.g005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000366.g005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000366.g005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000366.g005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000366.g005.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 5. </strong></p><a id="article1.body1.sec2.sec4.fig1.caption1.p1" name="article1.body1.sec2.sec4.fig1.caption1.p1"></a><p>Deletion Analysis Reveals a Critical Role of <em>hc-CNE1-125bp</em> for the Regulatory Potential of CNE1. (A) BLASTZ alignment of a human, mouse, chick, frog, and <em>Fugu</em> highly-conserved 125 bp sequence fragment embedded within CNE1 shown with predicted conserved TFBSs (above). (B) SLAGAN alignment plots of human, mouse, chick, frog and <em>Fugu</em> CNE1 using human sequence as the base line. (C) Architecture of CNE1 wild type and deletion constructs, The red bar depicts the highly conserved region, and less well conserved regions are shown in black. Luciferase activity obtained in H661 cells after transient transfection of reporter constructs is shown in the diagram at the right side. Reporter gene expression is driven by CNE1 fragments upstream of the human <em>GLI3</em> minimal promoter. The red bar depicts luciferase expression (100%) in H661 cells driven alone by the control <em>GLI3</em> minimal promoter (Prom-<em>GLI3</em>-300), while green bars represent the activity recorded for the vectors containing experimental reporter constructs.</p>
<span>doi:10.1371/journal.pone.0000366.g005</span></div><a id="article1.body1.sec2.sec4.p2" name="article1.body1.sec2.sec4.p2"></a><p>In order to test the functional significance of the <em>hcCNE1-125bp</em> track we generated two different deletion constructs. One contained <em>hcCNE1-125bp</em> alone with a minimal <em>GLI3</em> promoter. In the second construct, to investigate the activity of the moderately conserved human/<em>Fugu</em> flanking region, the <em>hcCNE1-125bp</em> fragment was deleted from the wild type CNE1(<a href="#pone-0000366-g005">Figure 5B</a>).</p>
<a id="article1.body1.sec2.sec4.p3" name="article1.body1.sec2.sec4.p3"></a><p>Deletion of the 125 bp highly conserved region reduced the activity by 67% compared to wild type construct, but it was still able to induce reporter gene expression 4-fold compared to the control vector, in which luciferase expression was driven alone by the minimal <em>GLI3</em> promoter. Tested alone, <em>hcCNE1-125bp</em> was unable to show any activating potential (<a href="#pone-0000366-g005">Figure 5C</a>).</p>
<a id="article1.body1.sec2.sec4.p4" name="article1.body1.sec2.sec4.p4"></a><p>CNE5 harbors two highly conserved blocks interrupted by a short less well-conserved fragment (<a href="#pone-0000366-g006">Figure 6B</a>). Within one of these blocks, 100% conserved in multispecies sequence comparison from human to fish, phylogenetic footprinting reveals contiguous binding sites for three developmentally important homeobox and paired box transcription factors, <em>PBX1, PAX2</em> and <em>MEIS1</em>. The <em>PAX2</em> and <em>MEIS1</em> binding sites overlap by one nucleotide.</p>
<div class="figure" id="pone-0000366-g006"><div class="img"><a name="pone-0000366-g006" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000366.g006&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000366" data-uri="info:doi/10.1371/journal.pone.0000366.g006"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000366.g006&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000366.g006/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000366.g006/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000366.g006/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000366.g006/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000366.g006.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000366.g006/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000366.g006/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000366.g006.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 6. </strong></p><a id="article1.body1.sec2.sec4.fig2.caption1.p1" name="article1.body1.sec2.sec4.fig2.caption1.p1"></a><p>Putative Binding Sites for Individual Trans-Acting Factors are Necessary but not Sufficient for Activating Potential of CNE5. (A) BLASTZ alignment of highly conserved fragments embedded within CNE5 along with predicted conserved TFBSs. (B) CNE5 alignment plot of human, mouse, chick, frog and <em>Fugu</em> sequences using human sequence as the base line. (C) Architecture of wild type and deletion constructs; the red portion of the bar depicts the highly conserved human/fish regions. Luciferase activity obtained in H661cells after transient transfection of reporter constructs is shown in the diagram at the right side. Reporter gene expression is driven by CNE5 fragments upstream of the human <em>GLI3</em> minimal promoter. The red bar depicts luciferase expression (100%) in H661 cells driven alone by the control <em>GLI3</em> minimal promoter (Prom<em>GLI3</em>-300), while the green bars represent the activity recorded for the vectors containing experimental reporter constructs, i.e. wild type CNE5 (wt 578bp), CNE5 with deleted <em>PBX1, PAX2</em> and <em>MEIS1</em> binding module (<em>CNE5Δ50bp</em>), and the 144 bp fragment (<em>hcCNE5-144bp</em>). Deletion of the 50 bp fragment almost entirely extinguishes the strong activating potential of CNE5. The isolated 144 bp fragment cannot activate expression.</p>
<span>doi:10.1371/journal.pone.0000366.g006</span></div><a id="article1.body1.sec2.sec4.p5" name="article1.body1.sec2.sec4.p5"></a><p>A 50 bp module encompassing the <em>PBX1, PAX2</em>, and <em>MEIS1</em> binding sites was excised from the wild type CNE5 fragment. Also, a 144 bp region encompassing both highly conserved blocks was isolated. Each of these were compared to wt CNE5 for their potential to enhance reporter gene transcription when transiently transfected into H661 cells. In contrast to wt CNE5, both elements were unable to activate basic transcription (<a href="#pone-0000366-g006">Figure 6B and 6C</a>).</p>
<a id="article1.body1.sec2.sec4.p6" name="article1.body1.sec2.sec4.p6"></a><p>The wild type 862 bp CNE6 fragment shows 87% sequence identity in a human/mouse comparison and contains a 179 bp moderately conserved region between human/<em>Fugu</em>. This track encompasses a 35 bp highly conserved site (<a href="#pone-0000366-g007">Figure 7A and 7B</a>). In order to investigate the role of both the 179 bp human/fish region and the flanking tetrapod conserved elements in the overall <em>in vitro</em> enhancer activity of wt CNE6, each region was investigated separately (<a href="#pone-0000366-g007">Figure 7C</a>).</p>
<div class="figure" id="pone-0000366-g007"><div class="img"><a name="pone-0000366-g007" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000366.g007&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000366" data-uri="info:doi/10.1371/journal.pone.0000366.g007"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000366.g007&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000366.g007/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000366.g007/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000366.g007/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000366.g007/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000366.g007.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000366.g007/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000366.g007/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000366.g007.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 7. </strong></p><a id="article1.body1.sec2.sec4.fig3.caption1.p1" name="article1.body1.sec2.sec4.fig3.caption1.p1"></a><p>CNE6 Sequences Flanking Human/Fish conserved Track Show Residual Enhancer Activity. (A) BLASTZ alignment of the highest conserved 35 bp along with two predicted conserved TFBSs from the human/<em>Fugu</em> conserved block within CNE6. (B) CNE6 alignment plot of human, mouse, chick, frog and <em>Fugu</em> sequences using human sequence as the base line. (C) Architecture of wild type and deletion constructs; the red bar depicts the highly conserved human/fish segment. Luciferase activity obtained in H661 cells after transient transfection of reporter constructs is shown in the diagram at the right side. Reporter gene expression is driven by CNE5 fragments upstream of the human <em>GLI3</em> minimal promoter. The red bar depicts luciferase expression (100%) in H661 cells driven alone by the control <em>GLI3</em> minimal promoter (Prom<em>GLI3</em>-300), whilst the green bars represent the activity recorded for the vectors containing experimental reporter constructs, i.e. wild type CNE6 (wt 862bp), CNE6 with deleted human/<em>Fugu</em> conserved block (<em>CNE6Δh/f-179bp</em>), and the 72% human/fish conserved fragment (<em>CNE6h/f-179bp</em>). <em>CNE6Δh/f-179bp</em> can still enhance reporter gene transcription more than two-fold. The isolated 179 bp fragment cannot activate expression.</p>
<span>doi:10.1371/journal.pone.0000366.g007</span></div><a id="article1.body1.sec2.sec4.p7" name="article1.body1.sec2.sec4.p7"></a><p>Deletion of the 179 bp element reduced the activity of CNE6 by ~70% compared to the wild type construct. However, compared to the control vector, this deleted CNE6 was still able to up-regulate the reporter gene expression by more than 2-fold. The 179 bp fragment upstream of the minimal <em>GLI3</em> promoter did not result in up-regulation of reporter gene expression compared to the control (<a href="#pone-0000366-g007">Figure 7B and 7C</a>).</p>

</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3>
<h4>Tetrapod-Teleost Conserved CNEs within Introns of <em>GLI3</em> Identify Enhancers</h4>
<a id="article1.body1.sec3.sec1.p1" name="article1.body1.sec3.sec1.p1"></a><p>Human <em>GLI3</em> extends over 260 kb on chromosome 7p14.1 (<a href="#pone-0000366-g001">Figure 1A</a>), a gene poor region, and is flanked by ~260 kb and ~700 kb intergenic intervals <a href="#pone.0000366-Scherer1">[35]</a>. <em>GLI3</em>-regulatory elements, potentially, could be located anywhere in this region. Considering observations with other developmental genes in gene poor regions, such as sonic hedgehog (<em>SHH</em>) <a href="#pone.0000366-Lettice1">[30]</a>, enhancers could even map within or beyond the neighbouring genes. In humans, the occurrence of distant regulatory elements can be heralded by cytogenetically detected translocations in patients with developmental malformations, causing the trait via separation of enhancer elements from their respective gene. In the case of <em>GLI3</em>, a translocation t(6;7)(q27;p13) truncating chromosome 7p14 about 10 kb downstream of the last exon results in a GCPS phenotype characteristic for functional deficiency of one <em>GLI3</em> allele <a href="#pone.0000366-Kruger1">[36]</a> (<a href="#pone-0000366-g001">Figure 1</a>). Silencing of the intact <em>GLI3</em> gene in this case could be caused by loss of cis-regulatory sequences distal to the breakpoint. In the mouse, a transgene insertion ~64 kb upstream of <em>Gli3</em> is associated with the phenotype of <em>anterior digit deformity</em> (add) <a href="#pone.0000366-Pohl1">[7]</a>, <a href="#pone.0000366-vanderHoeven1">[37]</a> (<a href="#pone-0000366-g001">Figure 1</a>). Here, the function of murine <em>Gli3</em>-enhancers located upstream of the insertion site may be disturbed. Thus, it is possible that <em>GLI3</em> might be among the genes regulated by distant enhancers. A further <em>Gli3</em> allele, the mouse mutant <em>Pdn</em>, results from insertion of a transposon into intron 3 (<a href="#pone-0000366-g001">Figure 1</a>) <a href="#pone.0000366-Ueta1">[38]</a>. In those mice <em>Gli3</em> expression appears to be possible, though at a reduced level. Based on these observations, <em>GLI3</em> regulatory elements might be found up- or downstream of the gene or within the introns.</p>
<a id="article1.body1.sec3.sec1.p2" name="article1.body1.sec3.sec1.p2"></a><p>Algorithms for the prediction of enhancers determining the temporal and spatial expression of human genes are increasingly powerful <a href="#pone.0000366-Bejerano2">[39]</a>, however, sound predictions of <em>GLI3</em> regulatory signatures have not yet been reported.</p>
<a id="article1.body1.sec3.sec1.p3" name="article1.body1.sec3.sec1.p3"></a><p>The region around <em>GLI3</em> is prohibitively large for using the painstaking strategy of stepwise deletions in reporter gene assays. Recently, it has been reported that there is a considerable overlap between experimentally verified enhancer elements and non-coding sequence elements (CNEs), that are evolutionarily conserved between distantly related species such as humans and the pufferfish <a href="#pone.0000366-Pennacchio1">[32]</a>. This suggests that CNEs around or within a gene are promising candidates for enhancers of expression. Different levels of stringency have been applied for the definition of CNEs <a href="#pone.0000366-Bejerano1">[34]</a>, <a href="#pone.0000366-Sanges1">[40]</a>, mostly with the intention to select a manageable number of candidate elements rather than with a biologically based rationale.</p>
<a id="article1.body1.sec3.sec1.p4" name="article1.body1.sec3.sec1.p4"></a><p>By employing multispecies sequence alignment we identified an ancient (tetrapod-teleost conserved) non-coding architecture within the introns of <em>GLI3</em>. The ancient, human/fish conserved signatures are embedded in larger sequence domains conserved in evolutionarily more recent species such as frog, chick or mouse (<a href="#pone-0000366-g001">Figure 1</a>). To test possible enhancers of expression we chose human/mouse CNEs encompassing &gt;60 bp tracks with more than 50% sequence similarity between human and <em>fugu</em>. These candidate elements represent sequence that is under ancient, strong evolutionary constraint operating to maintain a DNA sequence signature.</p>


<h4><em>In Vitro</em> Regulatory Activity of Intra-<em>GLI3</em> CNEs is Cell Type Specific</h4>
<a id="article1.body1.sec3.sec2.p1" name="article1.body1.sec3.sec2.p1"></a><p>The majority of intra-<em>GLI3</em> CNEs (8/10) exhibited a cellular context dependent dual nature. In the endogenous <em>GLI3</em> expressing environment (H661) they functioned as activators whilst in the GLI3 negative (H441) cellular context they actively repressed the transcription (<a href="#pone-0000366-g002">Figure 2</a>). This differential activity is strong evidence in favor of assigning <em>GLI3</em>-specific regulatory potential to these CNEs. Similar context dependent dual-nature regulatory activity is known for other transcription factors <a href="#pone.0000366-Grice1">[41]</a>. Our <em>in vitro</em> investigation also revealed two of the CNEs that had a repressing potential even in a GLI3 positive cellular context.</p>
<a id="article1.body1.sec3.sec2.p2" name="article1.body1.sec3.sec2.p2"></a><p>The most plausible scenario to explain the dual nature of a sub-set of intra-<em>GLI3</em> enhancers could be the interaction of each CNE with different subsets of trans-acting factors (either activators or repressors of transcription) in a cellular context dependent manner <a href="#pone.0000366-Hersh1">[42]</a>, whilst elements with repressing potential, even in a GLI3 positive context, suggest the existence of context independent regulation.</p>


<h4><em>In Vitro</em> Deletion Analysis Defines Functional Modules Within CNE1, 5 and 6</h4>
<a id="article1.body1.sec3.sec3.p1" name="article1.body1.sec3.sec3.p1"></a><p>We tested a subset of the CNEs, each associated with unique sequence features, as potential enhancers in transient transfection assays in H661 cells to see if the core elements conserved in human-<em>Fugu</em> represent functionally critical regulatory modules. CNE1 spans a human/<em>Fugu</em> conserved region of exceptionally extended length, 935 bp, and embedded within it a highly constraint interval of 125 bp almost 100% conserved down to chick, while depicting a 92% conservation in human/fish comparison. CNE5 encompasses human to fish 100% conserved contiguous binding sites for developmentally important TFs PBX1, PAX2 and MEIS1. CNE6 docks a small moderately conserved human/fish track of 179bp, within human/mouse 862 bp track with overall 87% conservation.</p>
<a id="article1.body1.sec3.sec3.p2" name="article1.body1.sec3.sec3.p2"></a><p>Considering the known degeneracy of transcription factor binding target sites <a href="#pone.0000366-Stormo1">[43]</a>, the high conservation of the 125 bp track within CNE1 (<a href="#pone-0000366-g005">Figure 5</a>) over the course of such an extreme phylogenetic separation (human/fish) is unexpected <a href="#pone.0000366-Boffelli1">[44]</a>. A previous <em>in vivo</em> attempt to resolve the significance of a tetrapod-conserved non-coding sequence element encompassing a human-<em>Fugu</em> conserved region around the human <em>DACH</em> gene had revealed that alone, neither a human-<em>Fugu</em> conserved core nor the less conserved flanking region, functioned as activators <a href="#pone.0000366-Poulin1">[45]</a>. The authors concluded that either the assay was not sufficiently sensitive or the core element might have an unknown biological function. In contrast, our <em>in vitro</em> deletion analysis with CNE1 provides evidence in favor of a quantitative participation of the conserved core to the overall activity of the enhancer, and suggests that this module embedded within CNE1 is essentially gene regulatory in function (<a href="#pone-0000366-g005">Figure 5</a>). The fact that the 125 bp sequence was unable to show any detectable activity in isolation reflects that this module is essential but not sufficient to uphold enhancer function on its own.</p>
<a id="article1.body1.sec3.sec3.p3" name="article1.body1.sec3.sec3.p3"></a><p>The manipulation of CNE5 (<a href="#pone-0000366-g006">Figure 6</a>) revealed that the deleted highly conserved element encompassing predicted contiguous TFBSs for developmental regulators <em>PBX1, PAX2</em> and <em>MEIS1</em> is necessary but not sufficient for the activating potential of this site.</p>
<a id="article1.body1.sec3.sec3.p4" name="article1.body1.sec3.sec3.p4"></a><p>The <em>in vitro</em> reporter gene expression data from deletion constructs of CNE6 suggest the existence of tetrapod specific functional constraints in the vicinity of an ancient fish specific element (<a href="#pone-0000366-g007">Figure 7</a>). It would appear that the overall enhancer activity is determined by the combinatorial affect of the ancient and the more recent sequences.</p>
<a id="article1.body1.sec3.sec3.p5" name="article1.body1.sec3.sec3.p5"></a><p>In all 3 examples, the excised elements had no activating potential when analyzed without the flanking sequences. However, we conclude from the sizeable reduction in activating potential in the absence of the core, that the human-<em>Fugu</em> conserved modules within the <em>GLI3</em>-CNEs are essential for transcriptional regulation. Mutagenesis of the predicted binding sites could show if transcription factors are involved in this function. The fact that flanking intervals of the human-<em>Fugu</em> conserved sequence elements contribute to the activity of the element, suggests that after the divergence of tetrapod-teleost lineages (450 Million years ago) there was a progressive gain of novel function centred around an ancient enhancer element. This possibly allowed fine-tuning of gene expression differentially in the tetrapod lineage, congruent with their complex developmental and anatomical needs.</p>


<h4>Can Transcription Factor Binding Sites Within CNEs Explain Their Evolutionary Conservation?</h4>
<a id="article1.body1.sec3.sec4.p1" name="article1.body1.sec3.sec4.p1"></a><p>A possible restraint causing the maintenance of CNEs involved in gene regulation throughout vertebrates could be a strict combinatorial code of TFBSs where order and distance are critical. We have screened for intra-CNE human-<em>Fugu</em> conserved putative TFBSs using the computer programs Consite and rVista v 2. In order to increase the sensitivity and to reduce the number of false positives, we have combined the TFBSs motif searches with phylogenetic footprinting of CNEs across distantly related species <a href="#pone.0000366-Loots1">[46]</a>, <a href="#pone.0000366-Sandelin1">[47]</a>. In each of the 11 sequences we identified human-<em>Fugu</em> conserved TFBSs for a number of developmental regulators (<a href="#pone-0000366-t001">Table. 1</a>). The prediction of binding sites for established developmental regulators under the highly stringent criteria in each of the tetrapod-teleost conserved intra-<em>GLI3</em> sequence tracks corroborates the conclusion from our experiments that the ancient elements contribute to the activity of the enhancer. However, TFBSs are known to allow considerable degeneracy and their overall density across each individual CNE is low. Unless strict maintenance of a combination of specific TFBSs and flanking sequence is required to retain tissue specifity of enhancer action, these sites may not contribute to the major constraint responsible for conservation of non-coding elements throughout evolution.</p>


<h4>Intra-<em>GLI3</em> CNEs Show Tissue Specific Regulatory Activity <em>In Vivo</em></h4>
<a id="article1.body1.sec3.sec5.p1" name="article1.body1.sec3.sec5.p1"></a><p>In order to address the <em>in vivo</em> role of <em>GLI3</em> associated conserved non-coding elements we selected a medium throughput strategy <a href="#pone.0000366-Woolfe1">[31]</a>, employing transient reporter gene expression from the human <em>β</em>-globin promoter under the influence of a putative enhancer element in zebrafish embryos. This approach exploiting the transparency and rapid development of zebrafish embryos has recently shown its immense potential for functionally testing enhancer elements among conserved non-coding regions <a href="#pone.0000366-Woolfe1">[31]</a>, <a href="#pone.0000366-Goode1">[48]</a>–<a href="#pone.0000366-Muller1">[50]</a>. Our results (<a href="#pone-0000366-g003">Figure 3</a>) indicate that the regulatory potential of most of the human CNEs defined in transient transfection assays of human cell cultures is similarly present in fish embryos. There is also a correlation between both enhancer and repressor activity <em>in vitro</em> and <em>in vivo</em>. Thus, we present evidence that both the sequence and the regulatory characteristics of <em>cis</em>-acting elements are conserved throughout evolution, from teleosts to man.</p>
<a id="article1.body1.sec3.sec5.p2" name="article1.body1.sec3.sec5.p2"></a><p>In mouse, GLI3 plays a prominent role in development of brain, ear, eye, craniofacial structures, limb and lung, and is also expressed in heart, kidney, skeletal muscles, fetal blood cells, epidermal cell layer of skin and other tissues (Mouse Genome Informatics <a href="http://www.informatics.jax.org">http://www.informatics.jax.org</a>). Zebrafish <em>gli3</em> is reported to be expressed in brain, dorsal spinal cord neurons, eye, and pectoral fin bud (Zebrafish Information Network; <a href="http://zfin.org">http://zfin.org</a>) <a href="#pone.0000366-Tyurina1">[51]</a>, <a href="#pone.0000366-Vanderlaan1">[52]</a>. However, exhaustive expression patterns throughout different stages of development have not been published.</p>
<a id="article1.body1.sec3.sec5.p3" name="article1.body1.sec3.sec5.p3"></a><p>A number of the positions in which transgene expression is observed coincide with known sites of GLI3 activity. For example CNE1 drives GFP expression predominantly in various subdivisions of the CNS, CNE10 activity was most frequent in the eye, pericardial region, lower jaw primordia and skin cells, CNE6 activity was more specific to hindbrain/spinal cord boundary neurons, muscle fibers and blood cells, and CNE11 driven reporter expression was largely restricted to cardiac chambers, skin cells and muscle fibers. Interestingly, CNE11 also induced GFP expression with low frequency within pectoral fins at day 3 of development which is consistent with the reported timing of zebrafish <em>gli3</em> expression in this tissue <a href="#pone.0000366-Tyurina1">[51]</a>. It can be seen that functional redundancy with respect to the site of expression was evident for all regulatory elements, a notion concordant with findings in other genes <a href="#pone.0000366-Fisher1">[53]</a>.</p>
<a id="article1.body1.sec3.sec5.p4" name="article1.body1.sec3.sec5.p4"></a><p>Some cell populations such as heart, the pericardial region, blood cells, muscle fibers, skin, and lower jaw primordial are domains of <em>Gli3</em> expression in mouse but not so far described in zebrafish. However, GLI3 functions appear to be conserved in mouse and zebrafish <a href="#pone.0000366-Tyurina1">[51]</a>. Therefore, the expression of g<em>li3</em> in zebrafish might be more extensive than reported so far. We observed expression in domains of the embryo where <em>gli3</em> is expressed neither in zebrafish nor in mouse. For example, CNE9 directed expression predominantly to the notochord, which is inconsistent with the reported endogenous <em>GLI3</em> expression in either species. This could reflect position effects upon the reporter-transgene inducing its expression at ectopic sites. The unexpected finding of a CNE within <em>GLI3</em>, which directs reporter gene expression at a site where GLI3 itself is never observed, stresses the importance of genomic context for the function of regulatory elements, as had been concluded by previous studies <a href="#pone.0000366-Woolfe1">[31]</a>, <a href="#pone.0000366-Fisher1">[53]</a>, <a href="#pone.0000366-GomezSkarmeta1">[54]</a>. We must therefore exercise caution when trying to draw conclusions on the normal regulatory potential of genomic fragments based on reporter construct studies, in both cell culture and transgenic animals.</p>


<h4>Conserved Regulatory Elements are Uncovered by Sequence Comparison at Extreme Phylogenetic Separation</h4>
<a id="article1.body1.sec3.sec6.p1" name="article1.body1.sec3.sec6.p1"></a><p>Most locations of reporter gene expression induced in transgenic zebrafish embryos by the human intronic <em>GLI3</em> CNEs represent prominent sites reported for endogenous zebrafish <em>gli3</em> <a href="#pone.0000366-Tyurina1">[51]</a>. However, zebrafish <em>gli3</em> expression in the pectoral fin bud has been reported to begin around 37 hpf, and by 44 h is expressed uniformly throughout the fin bud <a href="#pone.0000366-Tyurina1">[51]</a>. At this location only CNE 11 evoked signals in the pectoral fin (<a href="#pone-0000366-g003">Figure 3</a>), unlike the other enhancers, most of which ceased to act after ~28–33 hrs. It is possible that more focused analysis may reveal additional expression in the fin bud, but most probably the array of potential cis-acting regulatory elements chosen in this study did not cover the complete toolbox of elements required to orchestrate <em>gli3</em> expression during zebrafish development. We have pinpointed the regions to be analyzed as potential enhancers by the presence of a human-<em>Fugu</em> conserved sequence element, but the extent of the fragments included as CNEs was defined from human/mouse comparison. By this approach we addressed an ancient core as well sequences flanking each human-<em>Fugu</em>-conserved element, which may have evolved in tetrapods after its divergence from the teleost lineage. It is of note that these flanking sequences show little identity in teleost genomes, yet still function as enhancers in zebrafish. Homology among non-coding intra-<em>GLI3</em> sequences of tetrapods is not restricted to areas identified through comparison with <em>Fugu</em>. CNEs uncovered by sequence comparison within tetrapods could form a rich source of further regulatory elements patterning tetrapod-specific additions to the body plan. It will be interesting to test if and where enhancers regulating expression of more modern structures, such as digits, direct reporter expression in the fish.</p>

</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4>Reporter constructs</h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p>Candidate enhancer sequences (CNEs, <a href="#pone-0000366-t001">Table 1</a>) were PCR amplified using the high fidelity herculase enhanced DNA polymerase (Stratagene, Amsterdam, The Netherlands) with primers containing KpnI restriction site tags. Amplified DNA was purified using the QIAquick PCR purification kit (Qiagen, Hilden, Germany). Purified PCR products were then subjected to restriction site digestion with KpnI (New England Bio Labs, Ipswich, USA) and subsequently cloned in both orientations upstream of a minimal <em>GLI3</em> promoter or a heterologous SV40 promoter driving expression of the luciferase gene in the vector pGL3 (Promega, Madison, USA). The reporter constracts were designated pGL3-promGLI3-300-luc and pGL3-promSV40-luc, respectively. Recombinant reporter expression constructs were transfected into Top10 competent bacterial cells (Invitrogen, Karlsruhe, Germany) and subsequently isolated and purified using the Qiagen plasmid purification kit (Qiagen). To control the clones for presence of any point mutations generated during PCR amplification, appropriate DNA preparations were sequenced in ABI 377 automated sequencer (Applied Biosystems, Foster City, USA) and were analyzed with Sequencer software, Version 4.2.</p>


<h4>Deletion Mutants</h4>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1"></a><p>The deletion mutants of selected CNEs were made by PCR using the recombinant reporter construct of each of the respective wild type CNE as a template. The sequences flanking the segment to be deleted were PCR amplified with two different sets of primers. One member of each set was wt primer tagged with a KpnI restriction site, while the other member was designed from the immediate vicinity of the sequence to be deleted and tagged with a HindIII restriction site. Amplified products flanking the region to be deleted were purified using the QIAquick PCR purification kit (Qiagen) and digested by HindIII then subsequently ligated to one another. The ligated products were size fractioned on 2% agarose gel, and the DNA fragment of expected length was gel excised, purified by using a QIAquick gel extraction kit (Qiagen), digested by KpnI, and inserted into the pGL3-promGLI3-300-luc reporter plasmid. Sequence of each deleted recombinant construct was confirmed by sequencing (ABI 377 automated sequencer; Applied Biosystems). In order to avoid the <em>de-novo</em> creation of transcription factor binding sites, compared to wild type sequence, each of the deleted sequences were analyzed for potential TFBS with the TESS web tool (Transcription Element Search Software on <a href="http://www.cbil.upenn.edu/tess">http://www.cbil.upenn.edu/tess</a>).</p>


<h4>Cell Cultures</h4>
<a id="article1.body1.sec4.sec3.p1" name="article1.body1.sec4.sec3.p1"></a><p>The human lung tumor cell line H661 and the human bronchiolar epithelial cells H441 were obtained from the ATTC, USA, and grown under standard conditions in RPM1-1640 medium (Sigma Aldrich, Missouri, USA) containing 10% fetal calf serum, 1% non-essential amino acids, 2% penicillin/streptomycin and 1% L-glutamine (H661) or in modified RPMI-1640 medium (Sigma Aldrich) with 25mM HEPES and sodium bicarbonate, containing 4% fetal calf serum, 1% non-essential amino acids, 2% penicillin/streptomycin and 1% L-glutamine (H441), respectively.</p>


<h4>Transient Transfection and Dual Luciferase Assay</h4>
<a id="article1.body1.sec4.sec4.p1" name="article1.body1.sec4.sec4.p1"></a><p>The day before transfection, 4×10<sup>5</sup> H661 or 3×10<sup>5</sup> H441 cells were seeded into each well of a 12-well plate in 2 ml of the appropriate growth medium containing serum and antibiotics. After 24 hours of incubation at normal growth conditions, cells were transfected by using Effectene (Qiagen) according to the manufacturer's recommendations with the experimental firefly luciferase reporter constructs at a concentration of 200 ng/well, along with 100 ng/well of pRLSV40 (Promega) an expression vector containing cDNA encoding <em>Renilla</em> luciferase as an internal control reporter, and 200 ng/well of pGKBT7 (Clontech, Mountain View, USA) as a stuffer/carrier DNA.</p>
<a id="article1.body1.sec4.sec4.p2" name="article1.body1.sec4.sec4.p2"></a><p>48 hours after transfection, cells were assayed for luciferase activity with with the Dual-Luciferase Reporter Assay System (Promega) on an AutoLumat LB 953 luminometer (Berthold, Pforzheim, Germany). The activities of experimental reporter (firefly luciferase) were normalized to the activities of internal control reporter (<em>Renilla</em> luciferase). Triplicate assays were conducted three times.</p>


<h4>Zebrafish Enhancer/GFP Reporter Assay</h4>
<a id="article1.body1.sec4.sec5.p1" name="article1.body1.sec4.sec5.p1"></a><p>Zebrafish were bred and raised according to standard protocols <a href="#pone.0000366-Westerfield1">[55]</a>. CNEs for co-injection were either cut out from plasmids or amplified by PCR and then purified by QIAquick PCR purification kit (Qiagen). The reporter expression construct consisting of cDNA encoding enhanced green fluorescent protein (EGFP) under the control of minimal promoter from the human, β-globin gene was also PCR amplified from plasmid construct (Clontech). Element DNA (250–300 ng/ul) and reporter DNA fragment (25 ng/ul) were combined with tracer, i.e. phenol red (0.1%), and co-injected into the embryos produced from natural mating with a femtojet pressure injection system (Eppendorf, Hamburg, Germany) at the 1- to 8-cell stage, embryos developing abnormally were discarded after 2 to 3 hours of injection. Normal embryos were raised in 0.003% phenylthiocarbamide in embryo medium from tailbud stage. On the second day of microinjection (approximately 26–33 hpf) embryos were dechorionated using pronase E, anaesthetized in Tricaine and analysed under UV-light for GFP expression by using an IX81 motorised inverted microscope (Olympus, Tokyo, Japan). Images were captured using an FVII CCD monochrome digital camera and analySIS image-processing software.</p>
<a id="article1.body1.sec4.sec5.p2" name="article1.body1.sec4.sec5.p2"></a><p>GFP expressing cells were classified according to the following tissue categories: forebrain, midbrain, hindbrain, spinal cord, eye, ear, notochord, muscle, blood (circulating)/blood islands, heart/pericardial region, epidermis and fins. GFP expressing cells that were not localized unequivocally were classified as others. Location and tissue category of each GFP-expressing cell for each embryo was recorded schematically using Adobe Photoshop software (Adobe Systems, San Jose, USA), onto an overlay of a camera lucida drawing of 31-hpf embryo. For each CNE, the GFP expression data was collected from 20-50 expressing embryos. As a control, mean of 200 embryos were injected with conserved coding and non-conserved intronic sequences along with the reporter system and were found unable to show any significant GFP induction.</p>
<a id="article1.body1.sec4.sec5.p3" name="article1.body1.sec4.sec5.p3"></a><p>Combined schematised expression data for each CNE was compressed into a JPEG file and coupled with graphical depiction of expression domains to present an overall impression of the spatial pattern to which the element directs expression.</p>


<h4>Anti-GFP Immunostaining</h4>
<a id="article1.body1.sec4.sec6.p1" name="article1.body1.sec4.sec6.p1"></a><p>For immunostaining embryos were fixed in 4% paraformaldehyde overnight at 4°C and incubated with rabbit polyclonal anti-GFP (AMS Biotechnology, Abingdon Oxon, UK) using standard protocols <a href="#pone.0000366-Moens1">[56]</a> and the ABC amplification system (Vectastain; Vector laboratories, Burlingame, USA). Stained embryos were subsequently cleared in glycerol, flatmounted, and observed under bright field with Olympus IX81 motorised inverted microscope.</p>


<h4>Comparative Sequence Analysis</h4>
<a id="article1.body1.sec4.sec7.p1" name="article1.body1.sec4.sec7.p1"></a><p>Approximately 1 Mb of the human genome, encompassing <em>GLI</em>3 (ENSG00000106571) as well as <em>GLI3</em> orthologous sequences of mouse (ENSMUSG00000021318), chick (ENSGALG00000012329), frog (ENSXETG00000001856) and <em>Fugu</em> (SINFRUG00000153715) were obtained from Ensembl genome browser (<a href="http://www.ensembl.org">http://www.ensembl.org</a>). Multispecies sequence comparison was performed by using the glocal alignment program Shuffle-LAGAN <a href="#pone.0000366-Brudno1">[57]</a>. Human sequence was used as the baseline and annotated by using the exon/intron information available at Ensembl genome browser. Shuffle-LAGAN alignment was visualised with the VISTA visualization program <a href="#pone.0000366-Mayor1">[58]</a>. The alignment was performed using 60 bp window and a cutoff score of 50% identity.</p>


<h4><em>In Silico</em> Mapping of Conserved Transcription Factor Binding Sites</h4>
<a id="article1.body1.sec4.sec8.p1" name="article1.body1.sec4.sec8.p1"></a><p>Human-<em>Fugu</em> conserved transcription factor binding sites in each CNE were detected with ConSite (<a href="http:/www.phylofoot.org/consite">http:/www.phylofoot.org/consite</a>) and rVISTA.2.0 (<a href="http:/rvista.decode.org/">http:/rvista.decode.org/</a>). The ConSite screen for conserved TFBS was performed against the JASPAR database with 50% conservation cuttoff, 60 bp window size and 75% transcription factor score threshold settings.</p>
<a id="article1.body1.sec4.sec8.p2" name="article1.body1.sec4.sec8.p2"></a><p>rVISTA 2.0 searches for conserved TFBSs were performed against 500 vertebrate TF matrices from the TRANSFAC library, with matrix similarity cuttoff 0.85 by submitting a BLASTZ alignment file for each CNE to the rVISTA 2.0 site.</p>

</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>We thank Mrs. D. Bornholdt for expert technical assistance.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: KG. Performed the experiments: AA ZP DG HC. Analyzed the data: GE AA KG ZP SM DG. Wrote the paper: GE AA KG DG.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0000366-Radhakrishna1" id="pone.0000366-Radhakrishna1"></a>Radhakrishna U, Bornholdt D, Scott HS, Patel UC, Rossier C, et al.  (1999) The phenotypic spectrum of GLI3 morphopathies includes autosomal dominant preaxial polydactyly type-IV and postaxial polydactyly type-A/B; No phenotype prediction from the position of GLI3 mutations. Am J Hum Genet  65: 645–655.  <ul class="find" data-citedArticleID="943770" data-doi="10.1086/302557"><li><a href="http://dx.doi.org/10.1086/302557" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+phenotypic+spectrum+of+GLI3+morphopathies+includes+autosomal+dominant+preaxial+polydactyly+type-IV+and+postaxial+polydactyly+type-A%2FB%3B+No+phenotype+prediction+from+the+position+of+GLI3+mutations." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+phenotypic+spectrum+of+GLI3+morphopathies+includes+autosomal+dominant+preaxial+polydactyly+type-IV+and+postaxial+polydactyly+type-A%2FB%3B+No+phenotype+prediction+from+the+position+of+GLI3+mutations.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0000366-Vortkamp1" id="pone.0000366-Vortkamp1"></a>Vortkamp A, Gessler M, Grzeschik KH (1991) GLI3 zinc-finger gene interrupted by translocations in Greig syndrome families. Nature  352: 539–540.  <ul class="find" data-citedArticleID="943798" data-doi="10.1038/352539a0"><li><a href="http://dx.doi.org/10.1038/352539a0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=GLI3+zinc-finger+gene+interrupted+by+translocations+in+Greig+syndrome+families." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22GLI3+zinc-finger+gene+interrupted+by+translocations+in+Greig+syndrome+families.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0000366-Wild1" id="pone.0000366-Wild1"></a>Wild A, Kalff-Suske M, Vortkamp A, Bornholdt D, Konig R, et al.  (1997) Point mutations in human GLI3 cause Greig syndrome. Hum Mol Genet  6: 1979–1984.  <ul class="find" data-citedArticleID="943804" data-doi="10.1093/hmg/6.11.1979"><li><a href="http://dx.doi.org/10.1093/hmg/6.11.1979" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Point+mutations+in+human+GLI3+cause+Greig+syndrome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Point+mutations+in+human+GLI3+cause+Greig+syndrome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0000366-KalffSuske1" id="pone.0000366-KalffSuske1"></a>Kalff-Suske M, Wild A, Topp J, Wessling M, Jacobsen EM, et al.  (1999) Point mutations throughout the GLI3 gene cause Greig cephalopolysyndactyly syndrome. Hum Mol Genet  8: 1769–1777.  <ul class="find" data-citedArticleID="943734" data-doi="10.1093/hmg/8.9.1769"><li><a href="http://dx.doi.org/10.1093/hmg/8.9.1769" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Point+mutations+throughout+the+GLI3+gene+cause+Greig+cephalopolysyndactyly+syndrome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Point+mutations+throughout+the+GLI3+gene+cause+Greig+cephalopolysyndactyly+syndrome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0000366-Kang1" id="pone.0000366-Kang1"></a>Kang S, Graham JM Jr, Olney AH, Biesecker LG (1997) GLI3 frameshift mutations cause autosomal dominant Pallister-Hall syndrome. Nat Genet  15: 266–268.  <ul class="find" data-citedArticleID="943736" data-doi="10.1038/ng0397-266"><li><a href="http://dx.doi.org/10.1038/ng0397-266" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=GLI3+frameshift+mutations+cause+autosomal+dominant+Pallister-Hall+syndrome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22GLI3+frameshift+mutations+cause+autosomal+dominant+Pallister-Hall+syndrome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0000366-Radhakrishna2" id="pone.0000366-Radhakrishna2"></a>Radhakrishna U, Wild A, Grzeschik KH, Antonarakis SE (1997) Mutation in GLI3 in postaxial polydactyly type A. Nat Genet  17: 269–271.  <ul class="find" data-citedArticleID="943772" data-doi="10.1038/ng1197-269"><li><a href="http://dx.doi.org/10.1038/ng1197-269" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mutation+in+GLI3+in+postaxial+polydactyly+type+A." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mutation+in+GLI3+in+postaxial+polydactyly+type+A.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0000366-Pohl1" id="pone.0000366-Pohl1"></a>Pohl TM, Mattei MG, Ruther U (1990) Evidence for allelism of the recessive insertional mutation add and the dominant mouse mutation extra-toes (Xt). Development  110: 1153–1157.  <ul class="find" data-citedArticleID="943764"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Evidence+for+allelism+of+the+recessive+insertional+mutation+add+and+the+dominant+mouse+mutation+extra-toes+%28Xt%29.&amp;auth=&amp;atitle=Evidence+for+allelism+of+the+recessive+insertional+mutation+add+and+the+dominant+mouse+mutation+extra-toes+%28Xt%29." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evidence+for+allelism+of+the+recessive+insertional+mutation+add+and+the+dominant+mouse+mutation+extra-toes+%28Xt%29." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evidence+for+allelism+of+the+recessive+insertional+mutation+add+and+the+dominant+mouse+mutation+extra-toes+%28Xt%29.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0000366-Schimmang1" id="pone.0000366-Schimmang1"></a>Schimmang T, Lemaistre M, Vortkamp A, Ruther U (1992) Expression of the zinc finger gene Gli3 is affected in the morphogenetic mouse mutant extra-toes (Xt). Development  116: 799–804.  <ul class="find" data-citedArticleID="943786"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Expression+of+the+zinc+finger+gene+Gli3+is+affected+in+the+morphogenetic+mouse+mutant+extra-toes+%28Xt%29.&amp;auth=&amp;atitle=Expression+of+the+zinc+finger+gene+Gli3+is+affected+in+the+morphogenetic+mouse+mutant+extra-toes+%28Xt%29." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Expression+of+the+zinc+finger+gene+Gli3+is+affected+in+the+morphogenetic+mouse+mutant+extra-toes+%28Xt%29." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Expression+of+the+zinc+finger+gene+Gli3+is+affected+in+the+morphogenetic+mouse+mutant+extra-toes+%28Xt%29.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0000366-Hui1" id="pone.0000366-Hui1"></a>Hui CC, Joyner AL (1993) A mouse model of greig cephalopolysyndactyly syndrome: the extra-toesJ mutation contains an intragenic deletion of the Gli3 gene. Nat Genet  3: 241–246.  <ul class="find" data-citedArticleID="943730" data-doi="10.1038/ng0393-241"><li><a href="http://dx.doi.org/10.1038/ng0393-241" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+mouse+model+of+greig+cephalopolysyndactyly+syndrome%3A+the+extra-toesJ+mutation+contains+an+intragenic+deletion+of+the+Gli3+gene." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+mouse+model+of+greig+cephalopolysyndactyly+syndrome%3A+the+extra-toesJ+mutation+contains+an+intragenic+deletion+of+the+Gli3+gene.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0000366-Schimmang2" id="pone.0000366-Schimmang2"></a>Schimmang T, Oda SI, Ruther U (1994) The mouse mutant Polydactyly Nagoya (Pdn) defines a novel allele of the zinc finger gene Gli3. Mamm Genome  5: 384–386.  <ul class="find" data-citedArticleID="943788" data-doi="10.1007/bf00356560"><li><a href="http://dx.doi.org/10.1007/bf00356560" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+mouse+mutant+Polydactyly+Nagoya+%28Pdn%29+defines+a+novel+allele+of+the+zinc+finger+gene+Gli3." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+mouse+mutant+Polydactyly+Nagoya+%28Pdn%29+defines+a+novel+allele+of+the+zinc+finger+gene+Gli3.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0000366-Biesecker1" id="pone.0000366-Biesecker1"></a>Biesecker LG, Ondrey F (1998) Pallister-Hall syndrome. Otolaryngol Head Neck Surg  119: 556.  <ul class="find" data-citedArticleID="943702" data-doi="10.1016/s0194-5998(98)70126-3"><li><a href="http://dx.doi.org/10.1016/s0194-5998(98)70126-3" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Pallister-Hall+syndrome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Pallister-Hall+syndrome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0000366-Johnston1" id="pone.0000366-Johnston1"></a>Johnston JJ, Olivos-Glander I, Killoran C, Elson E, Turner JT, et al.  (2005) Molecular and clinical analyses of Greig cephalopolysyndactyly and Pallister-Hall syndromes: robust phenotype prediction from the type and position of GLI3 mutations. Am J Hum Genet  76: 609–622.  <ul class="find" data-citedArticleID="943732" data-doi="10.1086/429346"><li><a href="http://dx.doi.org/10.1086/429346" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Molecular+and+clinical+analyses+of+Greig+cephalopolysyndactyly+and+Pallister-Hall+syndromes%3A+robust+phenotype+prediction+from+the+type+and+position+of+GLI3+mutations." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Molecular+and+clinical+analyses+of+Greig+cephalopolysyndactyly+and+Pallister-Hall+syndromes%3A+robust+phenotype+prediction+from+the+type+and+position+of+GLI3+mutations.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0000366-Bose1" id="pone.0000366-Bose1"></a>Bose J, Grotewold L, Ruther U (2002) Pallister-Hall syndrome phenotype in mice mutant for Gli3. Hum Mol Genet  11: 1129–1135.  <ul class="find" data-citedArticleID="943706" data-doi="10.1093/hmg/11.9.1129"><li><a href="http://dx.doi.org/10.1093/hmg/11.9.1129" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Pallister-Hall+syndrome+phenotype+in+mice+mutant+for+Gli3." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Pallister-Hall+syndrome+phenotype+in+mice+mutant+for+Gli3.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0000366-RuiziAltaba1" id="pone.0000366-RuiziAltaba1"></a>Ruiz i Altaba A, Nguyen V, Palma V (2003) The emergent design of the neural tube: prepattern, SHH morphogen and GLI code. Curr Opin Genet Dev  13: 513–521.  <ul class="find" data-citedArticleID="943778" data-doi="10.1016/j.gde.2003.08.005"><li><a href="http://dx.doi.org/10.1016/j.gde.2003.08.005" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+emergent+design+of+the+neural+tube%3A+prepattern%2C+SHH+morphogen+and+GLI+code." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+emergent+design+of+the+neural+tube%3A+prepattern%2C+SHH+morphogen+and+GLI+code.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0000366-Ding1" id="pone.0000366-Ding1"></a>Ding Q, Motoyama J, Gasca S, Mo R, Sasaki H, et al.  (1998) Diminished Sonic hedgehog signaling and lack of floor plate differentiation in Gli2 mutant mice. Development  125: 2533–2543.  <ul class="find" data-citedArticleID="943716"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Diminished+Sonic+hedgehog+signaling+and+lack+of+floor+plate+differentiation+in+Gli2+mutant+mice.&amp;auth=&amp;atitle=Diminished+Sonic+hedgehog+signaling+and+lack+of+floor+plate+differentiation+in+Gli2+mutant+mice." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Diminished+Sonic+hedgehog+signaling+and+lack+of+floor+plate+differentiation+in+Gli2+mutant+mice." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Diminished+Sonic+hedgehog+signaling+and+lack+of+floor+plate+differentiation+in+Gli2+mutant+mice.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0000366-Motoyama1" id="pone.0000366-Motoyama1"></a>Motoyama J, Liu J, Mo R, Ding Q, Post M, et al.  (1998) Essential function of Gli2 and Gli3 in the formation of lung, trachea and oesophagus. Nat Genet  20: 54–57.  <ul class="find" data-citedArticleID="943754"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Essential+function+of+Gli2+and+Gli3+in+the+formation+of+lung%2C+trachea+and+oesophagus.&amp;auth=&amp;atitle=Essential+function+of+Gli2+and+Gli3+in+the+formation+of+lung%2C+trachea+and+oesophagus." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Essential+function+of+Gli2+and+Gli3+in+the+formation+of+lung%2C+trachea+and+oesophagus." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Essential+function+of+Gli2+and+Gli3+in+the+formation+of+lung%2C+trachea+and+oesophagus.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0000366-Bai1" id="pone.0000366-Bai1"></a>Bai CB, Joyner AL (2001) Gli1 can rescue the in vivo function of Gli2. Development  128: 5161–5172.  <ul class="find" data-citedArticleID="943694"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Gli1+can+rescue+the+in+vivo+function+of+Gli2.&amp;auth=&amp;atitle=Gli1+can+rescue+the+in+vivo+function+of+Gli2." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Gli1+can+rescue+the+in+vivo+function+of+Gli2." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Gli1+can+rescue+the+in+vivo+function+of+Gli2.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0000366-Bai2" id="pone.0000366-Bai2"></a>Bai CB, Auerbach W, Lee JS, Stephen D, Joyner AL (2002) Gli2, but not Gli1, is required for initial Shh signaling and ectopic activation of the Shh pathway. Development  129: 4753–4761.  <ul class="find" data-citedArticleID="943696"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Gli2%2C+but+not+Gli1%2C+is+required+for+initial+Shh+signaling+and+ectopic+activation+of+the+Shh+pathway.&amp;auth=&amp;atitle=Gli2%2C+but+not+Gli1%2C+is+required+for+initial+Shh+signaling+and+ectopic+activation+of+the+Shh+pathway." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Gli2%2C+but+not+Gli1%2C+is+required+for+initial+Shh+signaling+and+ectopic+activation+of+the+Shh+pathway." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Gli2%2C+but+not+Gli1%2C+is+required+for+initial+Shh+signaling+and+ectopic+activation+of+the+Shh+pathway.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0000366-Brewster1" id="pone.0000366-Brewster1"></a>Brewster R, Mullor JL, Ruiz i Altaba A (2000) Gli2 functions in FGF signaling during antero-posterior patterning. Development  127: 4395–4405.  <ul class="find" data-citedArticleID="943708"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Gli2+functions+in+FGF+signaling+during+antero-posterior+patterning.&amp;auth=&amp;atitle=Gli2+functions+in+FGF+signaling+during+antero-posterior+patterning." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Gli2+functions+in+FGF+signaling+during+antero-posterior+patterning." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Gli2+functions+in+FGF+signaling+during+antero-posterior+patterning.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0000366-Buttitta1" id="pone.0000366-Buttitta1"></a>Buttitta L, Mo R, Hui CC, Fan CM (2003) Interplays of Gli2 and Gli3 and their requirement in mediating Shh-dependent sclerotome induction. Development  130: 6233–6243.  <ul class="find" data-citedArticleID="943712" data-doi="10.1242/dev.00851"><li><a href="http://dx.doi.org/10.1242/dev.00851" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Interplays+of+Gli2+and+Gli3+and+their+requirement+in+mediating+Shh-dependent+sclerotome+induction." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Interplays+of+Gli2+and+Gli3+and+their+requirement+in+mediating+Shh-dependent+sclerotome+induction.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0000366-Persson1" id="pone.0000366-Persson1"></a>Persson M, Stamataki D, te Welscher P, Andersson E, Bose J, et al.  (2002) Dorsal-ventral patterning of the spinal cord requires Gli3 transcriptional repressor activity. Genes Dev  16: 2865–2878.  <ul class="find" data-citedArticleID="943762" data-doi="10.1101/gad.243402"><li><a href="http://dx.doi.org/10.1101/gad.243402" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Dorsal-ventral+patterning+of+the+spinal+cord+requires+Gli3+transcriptional+repressor+activity." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Dorsal-ventral+patterning+of+the+spinal+cord+requires+Gli3+transcriptional+repressor+activity.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0000366-Wijgerde1" id="pone.0000366-Wijgerde1"></a>Wijgerde M, McMahon JA, Rule M, McMahon AP (2002) A direct requirement for Hedgehog signaling for normal specification of all ventral progenitor domains in the presumptive mammalian spinal cord. Genes Dev  16: 2849–2864.  <ul class="find" data-citedArticleID="943802" data-doi="10.1101/gad.1025702"><li><a href="http://dx.doi.org/10.1101/gad.1025702" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+direct+requirement+for+Hedgehog+signaling+for+normal+specification+of+all+ventral+progenitor+domains+in+the+presumptive+mammalian+spinal+cord." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+direct+requirement+for+Hedgehog+signaling+for+normal+specification+of+all+ventral+progenitor+domains+in+the+presumptive+mammalian+spinal+cord.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0000366-Rallu1" id="pone.0000366-Rallu1"></a>Rallu M, Corbin JG, Fishell G (2002) Parsing the prosencephalon. Nat Rev Neurosci  3: 943–951.  <ul class="find" data-citedArticleID="943774" data-doi="10.1038/nrn989"><li><a href="http://dx.doi.org/10.1038/nrn989" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Parsing+the+prosencephalon." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Parsing+the+prosencephalon.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0000366-Corrales1" id="pone.0000366-Corrales1"></a>Corrales JD, Rocco GL, Blaess S, Guo Q, Joyner AL (2004) Spatial pattern of sonic hedgehog signaling through Gli genes during cerebellum development. Development  131: 5581–5590.  <ul class="find" data-citedArticleID="943714" data-doi="10.1242/dev.01438"><li><a href="http://dx.doi.org/10.1242/dev.01438" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Spatial+pattern+of+sonic+hedgehog+signaling+through+Gli+genes+during+cerebellum+development." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Spatial+pattern+of+sonic+hedgehog+signaling+through+Gli+genes+during+cerebellum+development.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0000366-Robert1" id="pone.0000366-Robert1"></a>Robert B, Lallemand Y (2006) Anteroposterior patterning in the limb and digit specification: Contribution of mouse genetics. Dev Dyn  235: 2337–2352.  <ul class="find" data-citedArticleID="943776" data-doi="10.1002/dvdy.20890"><li><a href="http://dx.doi.org/10.1002/dvdy.20890" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Anteroposterior+patterning+in+the+limb+and+digit+specification%3A+Contribution+of+mouse+genetics." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Anteroposterior+patterning+in+the+limb+and+digit+specification%3A+Contribution+of+mouse+genetics.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0000366-Koziel1" id="pone.0000366-Koziel1"></a>Koziel L, Wuelling M, Schneider S, Vortkamp A (2005) Gli3 acts as a repressor downstream of Ihh in regulating two distinct steps of chondrocyte differentiation. Development  132: 5249–5260.  <ul class="find" data-citedArticleID="943738" data-doi="10.1242/dev.02097"><li><a href="http://dx.doi.org/10.1242/dev.02097" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Gli3+acts+as+a+repressor+downstream+of+Ihh+in+regulating+two+distinct+steps+of+chondrocyte+differentiation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Gli3+acts+as+a+repressor+downstream+of+Ihh+in+regulating+two+distinct+steps+of+chondrocyte+differentiation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0000366-McDermott1" id="pone.0000366-McDermott1"></a>McDermott A, Gustafsson M, Elsam T, Hui CC, Emerson CP Jr, et al.  (2005) Gli2 and Gli3 have redundant and context-dependent function in skeletal muscle formation. Development  132: 345–357.  <ul class="find" data-citedArticleID="943748" data-doi="10.1242/dev.01537"><li><a href="http://dx.doi.org/10.1242/dev.01537" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Gli2+and+Gli3+have+redundant+and+context-dependent+function+in+skeletal+muscle+formation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Gli2+and+Gli3+have+redundant+and+context-dependent+function+in+skeletal+muscle+formation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0000366-Pu1" id="pone.0000366-Pu1"></a>Pu Y, Huang L, Prins GS (2004) Sonic hedgehog-patched Gli signaling in the developing rat prostate gland: lobe-specific suppression by neonatal estrogens reduces ductal growth and branching. Dev Biol  273: 257–275.  <ul class="find" data-citedArticleID="943768" data-doi="10.1016/j.ydbio.2004.06.002"><li><a href="http://dx.doi.org/10.1016/j.ydbio.2004.06.002" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Sonic+hedgehog-patched+Gli+signaling+in+the+developing+rat+prostate+gland%3A+lobe-specific+suppression+by+neonatal+estrogens+reduces+ductal+growth+and+branching." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Sonic+hedgehog-patched+Gli+signaling+in+the+developing+rat+prostate+gland%3A+lobe-specific+suppression+by+neonatal+estrogens+reduces+ductal+growth+and+branching.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0000366-Holstege1" id="pone.0000366-Holstege1"></a>Holstege FC, Clevers H (2006) Transcription factor target practice. Cell  124: 21–23.  <ul class="find" data-citedArticleID="943728" data-doi="10.1016/j.cell.2005.12.026"><li><a href="http://dx.doi.org/10.1016/j.cell.2005.12.026" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Transcription+factor+target+practice." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Transcription+factor+target+practice.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0000366-Lettice1" id="pone.0000366-Lettice1"></a>Lettice LA, Heaney SJ, Purdie LA, Li L, de Beer P, et al.  (2003) A long-range Shh enhancer regulates expression in the developing limb and fin and is associated with preaxial polydactyly. Hum Mol Genet  12: 1725–1735.  <ul class="find" data-citedArticleID="943742" data-doi="10.1093/hmg/ddg180"><li><a href="http://dx.doi.org/10.1093/hmg/ddg180" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+long-range+Shh+enhancer+regulates+expression+in+the+developing+limb+and+fin+and+is+associated+with+preaxial+polydactyly." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+long-range+Shh+enhancer+regulates+expression+in+the+developing+limb+and+fin+and+is+associated+with+preaxial+polydactyly.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0000366-Woolfe1" id="pone.0000366-Woolfe1"></a>Woolfe A, Goodson M, Goode DK, Snell P, McEwen GK, et al.  (2005) Highly conserved non-coding sequences are associated with vertebrate development. PLoS Biol  3: e7.  <ul class="find" data-citedArticleID="943806" data-doi="10.1371/journal.pbio.0030007"><li><a href="http://dx.doi.org/10.1371/journal.pbio.0030007" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Highly+conserved+non-coding+sequences+are+associated+with+vertebrate+development." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Highly+conserved+non-coding+sequences+are+associated+with+vertebrate+development.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0000366-Pennacchio1" id="pone.0000366-Pennacchio1"></a>Pennacchio LA, Ahituv N, Moses AM, Prabhakar S, Nobrega MA, et al.  (2006) In vivo enhancer analysis of human conserved non-coding sequences. Nature  444: 499–502.  <ul class="find" data-citedArticleID="943760" data-doi="10.1038/nature05295"><li><a href="http://dx.doi.org/10.1038/nature05295" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=In+vivo+enhancer+analysis+of+human+conserved+non-coding+sequences." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22In+vivo+enhancer+analysis+of+human+conserved+non-coding+sequences.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0000366-Ovcharenko1" id="pone.0000366-Ovcharenko1"></a>Ovcharenko I, Loots GG, Nobrega MA, Hardison RC, Miller W, et al.  (2005) Evolution and functional classification of vertebrate gene deserts. Genome Res  15: 137–145.  <ul class="find" data-citedArticleID="943758" data-doi="10.1101/gr.3015505"><li><a href="http://dx.doi.org/10.1101/gr.3015505" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evolution+and+functional+classification+of+vertebrate+gene+deserts." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evolution+and+functional+classification+of+vertebrate+gene+deserts.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0000366-Bejerano1" id="pone.0000366-Bejerano1"></a>Bejerano G, Pheasant M, Makunin I, Stephen S, Kent WJ, et al.  (2004) Ultraconserved elements in the human genome. Science  304: 1321–1325.  <ul class="find" data-citedArticleID="943698" data-doi="10.1126/science.1098119"><li><a href="http://dx.doi.org/10.1126/science.1098119" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Ultraconserved+elements+in+the+human+genome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Ultraconserved+elements+in+the+human+genome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0000366-Scherer1" id="pone.0000366-Scherer1"></a>Scherer SW, Cheung J, MacDonald JR, Osborne LR, Nakabayashi K, et al.  (2003) Human chromosome 7: DNA sequence and biology. Science  300: 767–772.  <ul class="find" data-citedArticleID="943784" data-doi="10.1126/science.1083423"><li><a href="http://dx.doi.org/10.1126/science.1083423" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Human+chromosome+7%3A+DNA+sequence+and+biology." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Human+chromosome+7%3A+DNA+sequence+and+biology.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">36.
              </span><a name="pone.0000366-Kruger1" id="pone.0000366-Kruger1"></a>Kruger G, Gotz J, Kvist U, Dunker H, Erfurth F, et al.  (1989) Greig syndrome in a large kindred due to reciprocal chromosome translocation t(6;7)(q27;p13). Am J Med Genet  32: 411–416.  <ul class="find" data-citedArticleID="943740" data-doi="10.1002/ajmg.1320320329"><li><a href="http://dx.doi.org/10.1002/ajmg.1320320329" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Greig+syndrome+in+a+large+kindred+due+to+reciprocal+chromosome+translocation+t%286%3B7%29%28q27%3Bp13%29." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Greig+syndrome+in+a+large+kindred+due+to+reciprocal+chromosome+translocation+t%286%3B7%29%28q27%3Bp13%29.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">37.
              </span><a name="pone.0000366-vanderHoeven1" id="pone.0000366-vanderHoeven1"></a>van der Hoeven F, Schimmang T, Vortkamp A, Ruther U (1993) Molecular linkage of the morphogenetic mutation add and the zinc finger gene Gli3. Mamm Genome  4: 276–277.  <ul class="find" data-citedArticleID="943808" data-doi="10.1007/bf00417435"><li><a href="http://dx.doi.org/10.1007/bf00417435" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Molecular+linkage+of+the+morphogenetic+mutation+add+and+the+zinc+finger+gene+Gli3." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Molecular+linkage+of+the+morphogenetic+mutation+add+and+the+zinc+finger+gene+Gli3.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">38.
              </span><a name="pone.0000366-Ueta1" id="pone.0000366-Ueta1"></a>Ueta E, Maekawa M, Morimoto I, Nanba E, Naruse I (2004) Sonic hedgehog expression in Gli3 depressed mouse embryo, Pdn/Pdn. Congenit Anom (Kyoto)  44: 27–32.  <ul class="find" data-citedArticleID="943794" data-doi="10.1111/j.1741-4520.2003.00005.x"><li><a href="http://dx.doi.org/10.1111/j.1741-4520.2003.00005.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Sonic+hedgehog+expression+in+Gli3+depressed+mouse+embryo%2C+Pdn%2FPdn." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Sonic+hedgehog+expression+in+Gli3+depressed+mouse+embryo%2C+Pdn%2FPdn.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">39.
              </span><a name="pone.0000366-Bejerano2" id="pone.0000366-Bejerano2"></a>Bejerano G, Siepel AC, Kent WJ, Haussler D (2005) Computational screening of conserved genomic DNA in search of functional noncoding elements. Nat Methods  2: 535–545.  <ul class="find" data-citedArticleID="943700" data-doi="10.1038/nmeth0705-535"><li><a href="http://dx.doi.org/10.1038/nmeth0705-535" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Computational+screening+of+conserved+genomic+DNA+in+search+of+functional+noncoding+elements." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Computational+screening+of+conserved+genomic+DNA+in+search+of+functional+noncoding+elements.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">40.
              </span><a name="pone.0000366-Sanges1" id="pone.0000366-Sanges1"></a>Sanges R, Kalmar E, Claudiani P, D'Amato M, Muller F, et al.  (2006) Shuffling of cis-regulatory elements is a pervasive feature of the vertebrate lineage. Genome Biol  7: R56.  <ul class="find" data-citedArticleID="943782" data-doi="10.1186/gb-2006-7-7-r56"><li><a href="http://dx.doi.org/10.1186/gb-2006-7-7-r56" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Shuffling+of+cis-regulatory+elements+is+a+pervasive+feature+of+the+vertebrate+lineage." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Shuffling+of+cis-regulatory+elements+is+a+pervasive+feature+of+the+vertebrate+lineage.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">41.
              </span><a name="pone.0000366-Grice1" id="pone.0000366-Grice1"></a>Grice EA, Rochelle ES, Green ED, Chakravarti A, McCallion AS (2005) Evaluation of the RET regulatory landscape reveals the biological relevance of a HSCR-implicated enhancer. Hum Mol Genet  14: 3837–3845.  <ul class="find" data-citedArticleID="943724" data-doi="10.1093/hmg/ddi408"><li><a href="http://dx.doi.org/10.1093/hmg/ddi408" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evaluation+of+the+RET+regulatory+landscape+reveals+the+biological+relevance+of+a+HSCR-implicated+enhancer." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evaluation+of+the+RET+regulatory+landscape+reveals+the+biological+relevance+of+a+HSCR-implicated+enhancer.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">42.
              </span><a name="pone.0000366-Hersh1" id="pone.0000366-Hersh1"></a>Hersh BM, Carroll SB (2005) Direct regulation of knot gene expression by Ultrabithorax and the evolution of cis-regulatory elements in Drosophila. Development  132: 1567–1577.  <ul class="find" data-citedArticleID="943726" data-doi="10.1242/dev.01737"><li><a href="http://dx.doi.org/10.1242/dev.01737" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Direct+regulation+of+knot+gene+expression+by+Ultrabithorax+and+the+evolution+of+cis-regulatory+elements+in+Drosophila." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Direct+regulation+of+knot+gene+expression+by+Ultrabithorax+and+the+evolution+of+cis-regulatory+elements+in+Drosophila.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">43.
              </span><a name="pone.0000366-Stormo1" id="pone.0000366-Stormo1"></a>Stormo GD (2000) DNA binding sites: representation and discovery. Bioinformatics  16: 16–23.  <ul class="find" data-citedArticleID="943790" data-doi="10.1093/bioinformatics/16.1.16"><li><a href="http://dx.doi.org/10.1093/bioinformatics/16.1.16" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=DNA+binding+sites%3A+representation+and+discovery." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22DNA+binding+sites%3A+representation+and+discovery.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">44.
              </span><a name="pone.0000366-Boffelli1" id="pone.0000366-Boffelli1"></a>Boffelli D, Nobrega MA, Rubin EM (2004) Comparative genomics at the vertebrate extremes. Nat Rev Genet  5: 456–465.  <ul class="find" data-citedArticleID="943704" data-doi="10.1038/nrg1350"><li><a href="http://dx.doi.org/10.1038/nrg1350" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Comparative+genomics+at+the+vertebrate+extremes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Comparative+genomics+at+the+vertebrate+extremes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">45.
              </span><a name="pone.0000366-Poulin1" id="pone.0000366-Poulin1"></a>Poulin F, Nobrega MA, Plajzer-Frick I, Holt A, Afzal V, et al.  (2005) In vivo characterization of a vertebrate ultraconserved enhancer. Genomics  85: 774–781.  <ul class="find" data-citedArticleID="943766" data-doi="10.1016/j.ygeno.2005.03.003"><li><a href="http://dx.doi.org/10.1016/j.ygeno.2005.03.003" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=In+vivo+characterization+of+a+vertebrate+ultraconserved+enhancer." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22In+vivo+characterization+of+a+vertebrate+ultraconserved+enhancer.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">46.
              </span><a name="pone.0000366-Loots1" id="pone.0000366-Loots1"></a>Loots GG, Ovcharenko I (2004) rVISTA 2.0: evolutionary analysis of transcription factor binding sites. Nucleic Acids Res  32: W217–221.  <ul class="find" data-citedArticleID="943744" data-doi="10.1093/nar/gkh383"><li><a href="http://dx.doi.org/10.1093/nar/gkh383" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=rVISTA+2.0%3A+evolutionary+analysis+of+transcription+factor+binding+sites." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22rVISTA+2.0%3A+evolutionary+analysis+of+transcription+factor+binding+sites.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">47.
              </span><a name="pone.0000366-Sandelin1" id="pone.0000366-Sandelin1"></a>Sandelin A, Wasserman WW, Lenhard B (2004) ConSite: web-based prediction of regulatory elements using cross-species comparison. Nucleic Acids Res  32: W249–252.  <ul class="find" data-citedArticleID="943780" data-doi="10.1093/nar/gkh372"><li><a href="http://dx.doi.org/10.1093/nar/gkh372" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=ConSite%3A+web-based+prediction+of+regulatory+elements+using+cross-species+comparison." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22ConSite%3A+web-based+prediction+of+regulatory+elements+using+cross-species+comparison.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">48.
              </span><a name="pone.0000366-Goode1" id="pone.0000366-Goode1"></a>Goode DK, Snell P, Smith SF, Cooke JE, Elgar G (2005) Highly conserved regulatory elements around the SHH gene may contribute to the maintenance of conserved synteny across human chromosome 7q36.3. Genomics  86: 172–181.  <ul class="find" data-citedArticleID="943722" data-doi="10.1016/j.ygeno.2005.04.006"><li><a href="http://dx.doi.org/10.1016/j.ygeno.2005.04.006" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Highly+conserved+regulatory+elements+around+the+SHH+gene+may+contribute+to+the+maintenance+of+conserved+synteny+across+human+chromosome+7q36.3." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Highly+conserved+regulatory+elements+around+the+SHH+gene+may+contribute+to+the+maintenance+of+conserved+synteny+across+human+chromosome+7q36.3.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">49.
              </span><a name="pone.0000366-McEwen1" id="pone.0000366-McEwen1"></a>McEwen GK, Woolfe A, Goode D, Vavouri T, Callaway H, et al.  (2006) Ancient duplicated conserved noncoding elements in vertebrates: a genomic and functional analysis. Genome Res  16: 451–465.  <ul class="find" data-citedArticleID="943750" data-doi="10.1101/gr.4143406"><li><a href="http://dx.doi.org/10.1101/gr.4143406" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Ancient+duplicated+conserved+noncoding+elements+in+vertebrates%3A+a+genomic+and+functional+analysis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Ancient+duplicated+conserved+noncoding+elements+in+vertebrates%3A+a+genomic+and+functional+analysis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">50.
              </span><a name="pone.0000366-Muller1" id="pone.0000366-Muller1"></a>Muller F, Chang B, Albert S, Fischer N, Tora L, et al.  (1999) Intronic enhancers control expression of zebrafish sonic hedgehog in floor plate and notochord. Development  126: 2103–2116.  <ul class="find" data-citedArticleID="943756"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Intronic+enhancers+control+expression+of+zebrafish+sonic+hedgehog+in+floor+plate+and+notochord.&amp;auth=&amp;atitle=Intronic+enhancers+control+expression+of+zebrafish+sonic+hedgehog+in+floor+plate+and+notochord." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Intronic+enhancers+control+expression+of+zebrafish+sonic+hedgehog+in+floor+plate+and+notochord." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Intronic+enhancers+control+expression+of+zebrafish+sonic+hedgehog+in+floor+plate+and+notochord.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">51.
              </span><a name="pone.0000366-Tyurina1" id="pone.0000366-Tyurina1"></a>Tyurina OV, Guner B, Popova E, Feng J, Schier AF, et al.  (2005) Zebrafish Gli3 functions as both an activator and a repressor in Hedgehog signaling. Dev Biol  277: 537–556.  <ul class="find" data-citedArticleID="943792" data-doi="10.1016/j.ydbio.2004.10.003"><li><a href="http://dx.doi.org/10.1016/j.ydbio.2004.10.003" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Zebrafish+Gli3+functions+as+both+an+activator+and+a+repressor+in+Hedgehog+signaling." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Zebrafish+Gli3+functions+as+both+an+activator+and+a+repressor+in+Hedgehog+signaling.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">52.
              </span><a name="pone.0000366-Vanderlaan1" id="pone.0000366-Vanderlaan1"></a>Vanderlaan G, Tyurina OV, Karlstrom RO, Chandrasekhar A (2005) Gli function is essential for motor neuron induction in zebrafish. Dev Biol  282: 550–570.  <ul class="find" data-citedArticleID="943796" data-doi="10.1016/j.ydbio.2005.04.010"><li><a href="http://dx.doi.org/10.1016/j.ydbio.2005.04.010" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Gli+function+is+essential+for+motor+neuron+induction+in+zebrafish." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Gli+function+is+essential+for+motor+neuron+induction+in+zebrafish.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">53.
              </span><a name="pone.0000366-Fisher1" id="pone.0000366-Fisher1"></a>Fisher S, Grice EA, Vinton RM, Bessling SL, McCallion AS (2006) Conservation of RET regulatory function from human to zebrafish without sequence similarity. Science  312: 276–279.  <ul class="find" data-citedArticleID="943718" data-doi="10.1126/science.1124070"><li><a href="http://dx.doi.org/10.1126/science.1124070" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Conservation+of+RET+regulatory+function+from+human+to+zebrafish+without+sequence+similarity." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Conservation+of+RET+regulatory+function+from+human+to+zebrafish+without+sequence+similarity.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">54.
              </span><a name="pone.0000366-GomezSkarmeta1" id="pone.0000366-GomezSkarmeta1"></a>Gomez-Skarmeta JL, Lenhard B, Becker TS (2006) New technologies, new findings, and new concepts in the study of vertebrate cis-regulatory sequences. Dev Dyn  235: 870–885.  <ul class="find" data-citedArticleID="943720" data-doi="10.1002/dvdy.20659"><li><a href="http://dx.doi.org/10.1002/dvdy.20659" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=New+technologies%2C+new+findings%2C+and+new+concepts+in+the+study+of+vertebrate+cis-regulatory+sequences." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22New+technologies%2C+new+findings%2C+and+new+concepts+in+the+study+of+vertebrate+cis-regulatory+sequences.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">55.
              </span><a name="pone.0000366-Westerfield1" id="pone.0000366-Westerfield1"></a>Westerfield M (2000) The zebrafish book: A guide for the laboratory use of zebrafish (<em>Danio rerio</em>). Oregon: University of Oregon Press.   <ul class="find-nolinks"></ul></li><li><span class="label">56.
              </span><a name="pone.0000366-Moens1" id="pone.0000366-Moens1"></a>Moens CB, Fritz A (1999) Techniques in neural development. Methods Cell Biol  59: 253–272.  <ul class="find" data-citedArticleID="943752" data-doi="10.1016/s0091-679x(08)61829-4"><li><a href="http://dx.doi.org/10.1016/s0091-679x(08)61829-4" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Techniques+in+neural+development." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Techniques+in+neural+development.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">57.
              </span><a name="pone.0000366-Brudno1" id="pone.0000366-Brudno1"></a>Brudno M, Malde S, Poliakov A, Do CB, Couronne O, et al.  (2003) Glocal alignment: finding rearrangements during alignment. Bioinformatics 19 Suppl 1i54–62.  <ul class="find" data-citedArticleID="943710" data-doi="10.1093/bioinformatics/btg1005"><li><a href="http://dx.doi.org/10.1093/bioinformatics/btg1005" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Glocal+alignment%3A+finding+rearrangements+during+alignment." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Glocal+alignment%3A+finding+rearrangements+during+alignment.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">58.
              </span><a name="pone.0000366-Mayor1" id="pone.0000366-Mayor1"></a>Mayor C, Brudno M, Schwartz JR, Poliakov A, Rubin EM, et al.  (2000) VISTA : visualizing global DNA sequence alignments of arbitrary length. Bioinformatics  16: 1046–1047.  <ul class="find" data-citedArticleID="943746" data-doi="10.1093/bioinformatics/16.11.1046"><li><a href="http://dx.doi.org/10.1093/bioinformatics/16.11.1046" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=VISTA+%3A+visualizing+global+DNA+sequence+alignments+of+arbitrary+length." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22VISTA+%3A+visualizing+global+DNA+sequence+alignments+of+arbitrary+length.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000366.XML" value="125907"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000366.PDF" value="611576"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000366.g001.PNG_L" value="616354"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000366.g001.PNG_M" value="48781"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000366.g001.PNG_S" value="3499"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000366.g001.TIF" value="2308870"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000366.g001.PNG_I" value="28572"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000366.t001.PNG_L" value="284431"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000366.t001.PNG_M" value="105894"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000366.t001.PNG_S" value="3794"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000366.t001.TIF" value="1196790"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000366.t001.PNG_I" value="42921"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000366.g002.PNG_L" value="705888"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000366.g002.PNG_M" value="57013"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000366.g002.PNG_S" value="6099"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000366.g002.TIF" value="1537606"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000366.g002.PNG_I" value="84747"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000366.g003.PNG_L" value="3189068"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000366.g003.PNG_M" value="128675"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000366.g003.PNG_S" value="4123"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000366.g003.TIF" value="4468558"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000366.g003.PNG_I" value="48018"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000366.g004.PNG_L" value="1327430"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000366.g004.PNG_M" value="172123"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000366.g004.PNG_S" value="5511"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000366.g004.TIF" value="1765372"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000366.g004.PNG_I" value="55360"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000366.g005.PNG_L" value="674498"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000366.g005.PNG_M" value="50169"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000366.g005.PNG_S" value="2874"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000366.g005.TIF" value="1376046"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000366.g005.PNG_I" value="23562"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000366.g006.PNG_L" value="636157"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000366.g006.PNG_M" value="41485"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000366.g006.PNG_S" value="2970"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000366.g006.TIF" value="1241706"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000366.g006.PNG_I" value="20942"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000366.g007.PNG_L" value="773706"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000366.g007.PNG_M" value="47519"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000366.g007.PNG_S" value="3040"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000366.g007.TIF" value="1530076"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000366.g007.PNG_I" value="21965"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000366&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000366" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000366&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0000366&volume=&issue=&title=Human GLI3 Intragenic Conserved Non-Coding Sequences Are Tissue-Specific Enhancers&author_name=Amir%20Ali%20Abbasi%2C%20Zissis%20Paparidis%2C%20Sajid%20Malik%2C%20Debbie%20K.%20Goode%2C%20Heather%20Callaway%2C%20Greg%20Elgar%2C%20Karl-Heinz%20Grzeschik&start_page=1&end_page=12" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000366" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000366" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0000366" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000366&amp;t=Human%20GLI3%20Intragenic%20Conserved%20Non-Coding%20Sequences%20Are%20Tissue-Specific%20Enhancers" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000366&title=Human%20GLI3%20Intragenic%20Conserved%20Non-Coding%20Sequences%20Are%20Tissue-Specific%20Enhancers&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000366&amp;title=Human%20GLI3%20Intragenic%20Conserved%20Non-Coding%20Sequences%20Are%20Tissue-Specific%20Enhancers" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000366" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0000366&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Human GLI3 Intragenic Conserved Non-Coding Sequences Are Tissue-Specific Enhancers';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0000366';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Human%20GLI3%20Intragenic%20Conserved%20Non-Coding%20Sequences%20Are%20Tissue-Specific%20Enhancers http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000366" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0000366" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">












          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22DNA+sequences%22" title="Search for articles in the subject area:'DNA sequences'"><div class="flagText">DNA sequences</div></a>
              <div data-categoryid="32003" data-articleid="23570"
                   data-categoryname="DNA sequences"
                   class="flagImage" title="Flag 'DNA sequences' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Embryos%22" title="Search for articles in the subject area:'Embryos'"><div class="flagText">Embryos</div></a>
              <div data-categoryid="17321" data-articleid="23570"
                   data-categoryname="Embryos"
                   class="flagImage" title="Flag 'Embryos' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Enhancer+elements%22" title="Search for articles in the subject area:'Enhancer elements'"><div class="flagText">Enhancer elements</div></a>
              <div data-categoryid="22445" data-articleid="23570"
                   data-categoryname="Enhancer elements"
                   class="flagImage" title="Flag 'Enhancer elements' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Gene+expression%22" title="Search for articles in the subject area:'Gene expression'"><div class="flagText">Gene expression</div></a>
              <div data-categoryid="17515" data-articleid="23570"
                   data-categoryname="Gene expression"
                   class="flagImage" title="Flag 'Gene expression' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Luciferase%22" title="Search for articles in the subject area:'Luciferase'"><div class="flagText">Luciferase</div></a>
              <div data-categoryid="31895" data-articleid="23570"
                   data-categoryname="Luciferase"
                   class="flagImage" title="Flag 'Luciferase' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Reporter+genes%22" title="Search for articles in the subject area:'Reporter genes'"><div class="flagText">Reporter genes</div></a>
              <div data-categoryid="21415" data-articleid="23570"
                   data-categoryname="Reporter genes"
                   class="flagImage" title="Flag 'Reporter genes' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Sequence+alignment%22" title="Search for articles in the subject area:'Sequence alignment'"><div class="flagText">Sequence alignment</div></a>
              <div data-categoryid="47429" data-articleid="23570"
                   data-categoryname="Sequence alignment"
                   class="flagImage" title="Flag 'Sequence alignment' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Zebrafish%22" title="Search for articles in the subject area:'Zebrafish'"><div class="flagText">Zebrafish</div></a>
              <div data-categoryid="32787" data-articleid="23570"
                   data-categoryname="Zebrafish"
                   class="flagImage" title="Flag 'Zebrafish' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=1284'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=7981'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=9522&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>


</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
