

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Comparative Expression Profiling of the Chlamydia trachomatis pmp Gene Family for Clinical and Reference Strains</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0000878"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0000878" />

    <meta name="citation_title" content="Comparative Expression Profiling of the Chlamydia trachomatis pmp Gene Family for Clinical and Reference Strains"/>
    <meta itemprop="name" content="Comparative Expression Profiling of the Chlamydia trachomatis pmp Gene Family for Clinical and Reference Strains"/>

      <meta name="citation_author" content="Alexandra Nunes"/>
            <meta name="citation_author_institution" content="Center for Immunobiology and Vaccine Development, Children's Hospital Oakland Research Institute, Oakland, California, United States of America"/>
            <meta name="citation_author_institution" content="Departamento de Bacteriologia, Instituto Nacional de Saúde, Lisboa, Portugal"/>
      <meta name="citation_author" content="João P. Gomes"/>
            <meta name="citation_author_institution" content="Center for Immunobiology and Vaccine Development, Children's Hospital Oakland Research Institute, Oakland, California, United States of America"/>
            <meta name="citation_author_institution" content="Departamento de Bacteriologia, Instituto Nacional de Saúde, Lisboa, Portugal"/>
      <meta name="citation_author" content="Sally Mead"/>
            <meta name="citation_author_institution" content="Center for Immunobiology and Vaccine Development, Children's Hospital Oakland Research Institute, Oakland, California, United States of America"/>
      <meta name="citation_author" content="Carlos Florindo"/>
            <meta name="citation_author_institution" content="Departamento de Bacteriologia, Instituto Nacional de Saúde, Lisboa, Portugal"/>
      <meta name="citation_author" content="Helena Correia"/>
            <meta name="citation_author_institution" content="Departamento de Bacteriologia, Instituto Nacional de Saúde, Lisboa, Portugal"/>
      <meta name="citation_author" content="Maria J. Borrego"/>
            <meta name="citation_author_institution" content="Departamento de Bacteriologia, Instituto Nacional de Saúde, Lisboa, Portugal"/>
      <meta name="citation_author" content="Deborah Dean"/>
            <meta name="citation_author_institution" content="Center for Immunobiology and Vaccine Development, Children's Hospital Oakland Research Institute, Oakland, California, United States of America"/>
            <meta name="citation_author_institution" content="Department of Bioengineering, University of California at Berkeley, Berkeley, California, United States of America"/>
            <meta name="citation_author_institution" content="Department of Medicine, School of Medicine, University of California at San Francisco, San Francisco, California, United States of America"/>

    <meta name="citation_date" content="2007/9/12"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0000878.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e878"/>
    <meta name="citation_issue" content="9"/>
    <meta name="citation_volume" content="2"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Interaction of chlamydiae and host cells in vitro.; citation_author=JW Moulder; citation_journal_title=Microbiol Rev; citation_volume=55; citation_number=1; citation_pages=143-190; citation_date=1991; " />
      <meta name="citation_reference" content="citation_title=The chlamydial development cycle.; citation_author=ME Ward; citation_number=2; citation_pages=71-97; citation_date=1988; citation_publisher=CRC Press, Inc; " />
      <meta name="citation_reference" content="citation_title=Chlamydia trachomatis Sexually Transmitted Diseases.; citation_author=D Dean; citation_number=3; citation_pages=473-490; citation_date=1997; citation_publisher=Appleton and Lange Publishers; " />
      <meta name="citation_reference" content="citation_title=Persistent Chlamydia trachomatis infections resist apoptotic stimuli.; citation_author=D Dean; citation_author=VC Powers; citation_journal_title=Infect Immun; citation_volume=69; citation_number=4; citation_pages=2442-2447; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=The Genus Chlamydia.; citation_author=PI Fields; citation_author=RC Barnes; citation_number=5; citation_pages=3691-3709; citation_date=1992; citation_publisher=Springer-Verlag; " />
      <meta name="citation_reference" content="citation_title=Phylogenetic analysis of the outer-membrane-protein genes of Chlamydiae, and its implication for vaccine development.; citation_author=WM Fitch; citation_author=EM Peterson; citation_author=LM de la Maza; citation_journal_title=Mol Biol Evol; citation_volume=10; citation_number=6; citation_pages=892-913; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=Molecular and mutation trends analyses of omp1 alleles for serovar E of Chlamydia trachomatis. Implications for the immunopathogenesis of disease.; citation_author=D Dean; citation_author=K Millman; citation_journal_title=J Clin Invest; citation_volume=99; citation_number=7; citation_pages=475-483; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Genome sequence of an obligate intracellular pathogen of humans: Chlamydia trachomatis.; citation_author=RS Stephens; citation_author=S Kalman; citation_author=C Lammel; citation_author=J Fan; citation_author=R Marathe; citation_journal_title=Science; citation_volume=282; citation_number=8; citation_pages=754-759; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Comparative genomic analysis of Chlamydia trachomatis oculotropic and genitotropic strains.; citation_author=JH Carlson; citation_author=SF Porcella; citation_author=G McClarty; citation_author=HD Caldwell; citation_journal_title=Infect Immun; citation_volume=73; citation_number=9; citation_pages=6407-6418; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Polymorphisms in Chlamydia trachomatis tryptophan synthase genes differentiate between genital and ocular isolates.; citation_author=HD Caldwell; citation_author=H Wood; citation_author=D Crane; citation_author=R Bailey; citation_author=RB Jones; citation_journal_title=J Clin Invest; citation_volume=111; citation_number=10; citation_pages=1757-1769; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Polymorphisms in the Chlamydia trachomatis cytotoxin locus associated with ocular and genital isolates.; citation_author=JH Carlson; citation_author=S Hughes; citation_author=D Hogan; citation_author=G Cieplak; citation_author=DE Sturdevant; citation_journal_title=Infect Immun; citation_volume=72; citation_number=11; citation_pages=7063-7072; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Polymorphisms in the Nine Polymorphic Membrane Proteins of Chlamydia trachomatis across All Serovars: Evidence for Serovar Da Recombination and Correlation with Tissue Tropism.; citation_author=JP Gomes; citation_author=A Nunes; citation_author=WJ Bruno; citation_author=MJ Borrego; citation_author=C Florindo; citation_journal_title=J Bacteriol; citation_volume=188; citation_number=12; citation_pages=275-286; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Polymorphic membrane protein H has evolved in parallel with the three disease-causing groups of Chlamydia trachomatis.; citation_author=DR Stothard; citation_author=GA Toth; citation_author=BE Batteiger; citation_journal_title=Infect Immun; citation_volume=71; citation_number=13; citation_pages=1200-1208; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Recombination in the genome of Chlamydia trachomatis involving the polymorphic membrane protein C gene relative to ompA and evidence for horizontal gene transfer.; citation_author=JP Gomes; citation_author=WJ Bruno; citation_author=MJ Borrego; citation_author=D Dean; citation_journal_title=J Bacteriol; citation_volume=186; citation_number=14; citation_pages=4295-4306; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Transcriptional activity of a sequence variable protein family in Chlamydia trachomatis.; citation_author=EA Lindquist; citation_author=RS Stephens; citation_number=15; citation_pages=259-262; citation_date=1998; citation_publisher=International Chlamydia Symposium; " />
      <meta name="citation_reference" content="citation_title=Genomic transcriptional profiling of the developmental cycle of Chlamydia trachomatis.; citation_author=RJ Belland; citation_author=G Zhong; citation_author=DD Crane; citation_author=D Hogan; citation_author=D Sturdevant; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=100; citation_number=16; citation_pages=8478-8483; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Immunoreactivity and differential developmental expression of known and putative Chlamydia trachomatis membrane proteins for biologically variant serovars representing distinct disease groups.; citation_author=JP Gomes; citation_author=RC Hsia; citation_author=S Mead; citation_author=MJ Borrego; citation_author=D Dean; citation_journal_title=Microbes Infect; citation_volume=7; citation_number=17; citation_pages=410-420; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Expression, Processing, and Localization of PmpD of Chlamydia trachomatis Serovar L2 during the Chlamydial Developmental Cycle.; citation_author=AO Kiselev; citation_author=WE Stamm; citation_author=JR Yates; citation_author=MF Lampe; citation_journal_title=PLoS ONE; citation_volume=2; citation_number=18; citation_pages=e568; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Shotgun proteomic analysis of Chlamydia trachomatis.; citation_author=P Skipp; citation_author=J Robinson; citation_author=CD O'Connor; citation_author=IN Clarke; citation_journal_title=Proteomics; citation_volume=5; citation_number=19; citation_pages=1558-1573; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Membrane proteins PmpG and PmpH are major constituents of Chlamydia trachomatis L2 outer membrane complex.; citation_author=PH Mygind; citation_author=G Christiansen; citation_author=P Roepstorff; citation_author=S Birkelund; citation_journal_title=FEMS Microbiol Lett; citation_volume=186; citation_number=20; citation_pages=163-169; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Comparative proteome analysis of Chlamydia trachomatis serovar A, D and L2.; citation_author=AC Shaw; citation_author=K Gevaert; citation_author=H Demol; citation_author=B Hoorelbeke; citation_author=J Vandekerckhove; citation_journal_title=Proteomics; citation_volume=2; citation_number=21; citation_pages=164-186; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Characterization of outer membrane proteins in Chlamydia trachomatis LGV serovar L2.; citation_author=RJ Tanzer; citation_author=TP Hatch; citation_journal_title=J Bacteriol; citation_volume=183; citation_number=22; citation_pages=2686-2690; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Differential immune response to polymorphic membrane proteins in STD patients.; citation_author=Rc Hsia; citation_author=I Ahmed; citation_author=B Batteiger; citation_author=O Sekkides; citation_author=G Ridgway; citation_number=23; citation_pages=219; citation_date=2000; citation_publisher=Societa Editrice Esculapio; " />
      <meta name="citation_reference" content="citation_title=Polymorphic proteins of Chlamydia spp.–autotransporters beyond the Proteobacteria.; citation_author=IR Henderson; citation_author=AC Lam; citation_journal_title=Trends Microbiol; citation_volume=9; citation_number=24; citation_pages=573-578; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=The great escape: structure and function of the autotransporter proteins.; citation_author=IR Henderson; citation_author=F Navarro-Garcia; citation_author=JP Nataro; citation_journal_title=Trends Microbiol; citation_volume=6; citation_number=25; citation_pages=370-378; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=From the inside out–processing of the Chlamydial autotransporter PmpD and its role in bacterial adhesion and activation of human host cells.; citation_author=W Wehrl; citation_author=V Brinkmann; citation_author=PR Jungblut; citation_author=TF Meyer; citation_author=AJ Szczepek; citation_journal_title=Mol Microbiol; citation_volume=51; citation_number=26; citation_pages=319-334; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Chlamydia trachomatis polymorphic membrane protein D is a species-common pan-neutralizing antigen.; citation_author=DD Crane; citation_author=JH Carlson; citation_author=ER Fischer; citation_author=P Bavoil; citation_author=RC Hsia; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=103; citation_number=27; citation_pages=1894-1899; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Evolution of Chlamydia trachomatis diversity occurs by widespread interstrain recombination involving hotspots.; citation_author=JP Gomes; citation_author=WJ Bruno; citation_author=A Nunes; citation_author=N Santos; citation_author=C Florindo; citation_journal_title=Genome Res; citation_volume=17; citation_number=28; citation_pages=50-60; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Evidence for long-term cervical persistence of Chlamydia trachomatis by omp1 genotyping.; citation_author=D Dean; citation_author=R Suchland; citation_author=W Stamm; citation_journal_title=J Infect Dis; citation_volume=182; citation_number=29; citation_pages=909-916; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Purification and partial characterization of the major outer membrane protein of Chlamydia trachomatis.; citation_author=HD Caldwell; citation_author=J Kromhout; citation_author=J Schachter; citation_journal_title=Infect Immun; citation_volume=31; citation_number=30; citation_pages=1161-1176; citation_date=1981; " />
      <meta name="citation_reference" content="citation_title=Mutational analysis of the Chlamydia trachomatis dnaK promoter defines the optimal -35 promoter element.; citation_author=CS Schaumburg; citation_author=M Tan; citation_journal_title=Nucleic Acids Res; citation_volume=31; citation_number=31; citation_pages=551-555; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=A positive cis-acting DNA element is required for high-level transcription in Chlamydia.; citation_author=CS Schaumburg; citation_author=M Tan; citation_journal_title=J Bacteriol; citation_volume=182; citation_number=32; citation_pages=5167-5171; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Mutagenesis of the P2 promoter of the major outer membrane protein gene of Chlamydia trachomatis.; citation_author=AL Douglas; citation_author=TP Hatch; citation_journal_title=J Bacteriol; citation_volume=178; citation_number=33; citation_pages=5573-5578; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Mutational analysis of the Chlamydia trachomatis rRNA P1 promoter defines four regions important for transcription in vitro.; citation_author=M Tan; citation_author=T Gaal; citation_author=RL Gourse; citation_author=JN Engel; citation_journal_title=J Bacteriol; citation_volume=180; citation_number=34; citation_pages=2359-2366; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=The 3′-terminal sequence of Escherichia coli 16S ribosomal RNA: complementarity to nonsense triplets and ribosome binding sites.; citation_author=J Shine; citation_author=L Dalgarno; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=71; citation_number=35; citation_pages=1342-1346; citation_date=1974; " />
      <meta name="citation_reference" content="citation_title=Developmental regulation of tandem promoters for the major outer membrane protein gene of Chlamydia trachomatis.; citation_author=RS Stephens; citation_author=EA Wagar; citation_author=U Edman; citation_journal_title=J Bacteriol; citation_volume=170; citation_number=36; citation_pages=744-750; citation_date=1988; " />
      <meta name="citation_reference" content="citation_title=Sequence diversity of the 60-kilodalton protein and of a putative 15-kilodalton protein between the trachoma and lymphogranuloma venereum biovars of Chlamydia trachomatis.; citation_author=LM de la Maza; citation_author=TJ Fielder; citation_author=EJ Carlson; citation_author=BA Markoff; citation_author=EM Peterson; citation_journal_title=Infect Immun; citation_volume=59; citation_number=37; citation_pages=1196-1201; citation_date=1991; " />
      <meta name="citation_reference" content="citation_title=Identification by sequence analysis of two-site posttranslational processing of the cysteine-rich outer membrane protein 2 of Chlamydia trachomatis serovar L2.; citation_author=JE Allen; citation_author=RS Stephens; citation_journal_title=J Bacteriol; citation_volume=171; citation_number=38; citation_pages=285-291; citation_date=1989; " />
      <meta name="citation_reference" content="citation_title=The 75-kilodalton protein of Chlamydia trachomatis: a member of the heat shock protein 70 family?; citation_author=SL Danilition; citation_author=IW Maclean; citation_author=R Peeling; citation_author=S Winston; citation_author=RC Brunham; citation_journal_title=Infect Immun; citation_volume=58; citation_number=39; citation_pages=189-196; citation_date=1990; " />
      <meta name="citation_reference" content="citation_title=Lost in translation: the influence of ribosomes on bacterial mRNA decay.; citation_author=A Deana; citation_author=JG Belasco; citation_journal_title=Genes Dev; citation_volume=19; citation_number=40; citation_pages=2526-2533; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Specificity of Escherichia coli endoribonuclease RNase E: in vivo and in vitro analysis of mutants in a bacteriophage T4 mRNA processing site.; citation_author=CP Ehretsmann; citation_author=AJ Carpousis; citation_author=HM Krisch; citation_journal_title=Genes Dev; citation_volume=6; citation_number=41; citation_pages=149-159; citation_date=1992; " />
      <meta name="citation_reference" content="citation_title=A nucleotide sequence from a ribonuclease III processing site in bacteriophage T7 RNA.; citation_author=HD Robertson; citation_author=E Dickson; citation_author=JJ Dunn; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=74; citation_number=42; citation_pages=822-826; citation_date=1977; " />
      <meta name="citation_reference" content="citation_title=A conserved sequence element in ribonuclease III processing signals is not required for accurate in vitro enzymatic cleavage.; citation_author=BS Chelladurai; citation_author=H Li; citation_author=AW Nicholson; citation_journal_title=Nucleic Acids Res; citation_volume=19; citation_number=43; citation_pages=1759-1766; citation_date=1991; " />
      <meta name="citation_reference" content="citation_title=Degradation of RNA in bacteria: comparison of mRNA and stable RNA.; citation_author=MP Deutscher; citation_journal_title=Nucleic Acids Res; citation_volume=34; citation_number=44; citation_pages=659-666; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Expression of the glycolytic gapA operon in Bacillus subtilis: differential syntheses of proteins encoded by the operon.; citation_author=C Meinken; citation_author=HM Blencke; citation_author=H Ludwig; citation_author=J Stulke; citation_journal_title=Microbiology; citation_volume=149; citation_number=45; citation_pages=751-761; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Transcriptional organization and posttranscriptional regulation of the Bacillus subtilis branched-chain amino acid biosynthesis genes.; citation_author=U Mader; citation_author=S Hennig; citation_author=M Hecker; citation_author=G Homuth; citation_journal_title=J Bacteriol; citation_volume=186; citation_number=46; citation_pages=2240-2252; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Prediction of transcriptional terminators in Bacillus subtilis and related species.; citation_author=MJ de Hoon; citation_author=Y Makita; citation_author=K Nakai; citation_author=S Miyano; citation_journal_title=PLoS Comput Biol; citation_volume=1; citation_number=47; citation_pages=e25; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=mRNA decay in prokaryotes and eukaryotes: different approaches to a similar problem.; citation_author=SR Kushner; citation_journal_title=IUBMB Life; citation_volume=56; citation_number=48; citation_pages=585-594; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Analysis of complete genomes suggests that many prokaryotes do not rely on hairpin formation in transcription termination.; citation_author=T Washio; citation_author=J Sasayama; citation_author=M Tomita; citation_journal_title=Nucleic Acids Res; citation_volume=26; citation_number=49; citation_pages=5456-5463; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Immune response to chlamydial 60-kilodalton heat shock protein in tears from Nepali trachoma patients.; citation_author=T Hessel; citation_author=SP Dhital; citation_author=R Plank; citation_author=D Dean; citation_journal_title=Infect Immun; citation_volume=69; citation_number=50; citation_pages=4996-5000; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Different Growth Rates of Chlamydia trachomatis Biovars Reflect Pathotype.; citation_author=I Miyairi; citation_author=OS Mahdi; citation_author=SP Ouellette; citation_author=RJ Belland; citation_author=GI Byrne; citation_journal_title=J Infect Dis; citation_volume=194; citation_number=51; citation_pages=350-357; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Surface location of PmpD, a polymorphic membrane protein of Chlamydia trachomatis serovar L2.; citation_author=AO Kiselev; citation_author=ML Johnson; citation_author=LB Ballweber; citation_author=WE Stamm; citation_author=MF Lampe; citation_number=52; citation_pages=567-570; citation_date=2002; citation_publisher=International Chlamydia Symposium; " />
      <meta name="citation_reference" content="citation_title=The expression, processing and localization of polymorphic membrane proteins in Chlamydia pneumoniae strain CWL029.; citation_author=BB Vandahl; citation_author=AS Pedersen; citation_author=K Gevaert; citation_author=A Holm; citation_author=J Vandekerckhove; citation_journal_title=BMC Microbiol; citation_volume=2; citation_number=53; citation_pages=36; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Computational analysis of the polymorphic membrane protein superfamily of Chlamydia trachomatis and Chlamydia pneumoniae.; citation_author=J Grimwood; citation_author=RS Stephens; citation_journal_title=Microb Comp Genomics; citation_volume=4; citation_number=54; citation_pages=187-201; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Processing endoribonucleases and mRNA degradation in bacteria.; citation_author=D Kennell; citation_journal_title=J Bacteriol; citation_volume=184; citation_number=55; citation_pages=4645-4657; discussion 4665; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Comparative sequence analysis of ribonucleases HII, III, II PH and D.; citation_author=IS Mian; citation_journal_title=Nucleic Acids Res; citation_volume=25; citation_number=56; citation_pages=3187-3195; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Staphylococcus aureus RNAIII and the endoribonuclease III coordinately regulate spa gene expression.; citation_author=E Huntzinger; citation_author=S Boisset; citation_author=C Saveanu; citation_author=Y Benito; citation_author=T Geissmann; citation_journal_title=Embo J; citation_volume=24; citation_number=57; citation_pages=824-835; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Both RNase E and RNase III control the stability of sodB mRNA upon translational inhibition by the small regulatory RNA RyhB.; citation_author=T Afonyushkin; citation_author=B Vecerek; citation_author=I Moll; citation_author=U Blasi; citation_author=VR Kaberdin; citation_journal_title=Nucleic Acids Res; citation_volume=33; citation_number=58; citation_pages=1678-1689; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=The Bacillus subtilis L-arabinose (ara) operon: nucleotide sequence, genetic organization and expression.; citation_author=I Sa-Nogueira; citation_author=TV Nogueira; citation_author=S Soares; citation_author=H de Lencastre; citation_journal_title=Microbiology; citation_volume=143 ( Pt 3); citation_number=59; citation_pages=957-969; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=The transcription of the cbb operon in Nitrosomonas europaea.; citation_author=X Wei; citation_author=LA Sayavedra-Soto; citation_author=DJ Arp; citation_journal_title=Microbiology; citation_volume=150; citation_number=60; citation_pages=1869-1879; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Borrelia burgdorferi outer surface protein (osp) B expression independent of ospA.; citation_author=FT Liang; citation_author=MJ Caimano; citation_author=JD Radolf; citation_author=E Fikrig; citation_journal_title=Microb Pathog; citation_volume=37; citation_number=61; citation_pages=35-40; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Expression of the bmpB gene of Borrelia burgdorferi is modulated by two distinct transcription termination events.; citation_author=R Ramamoorthy; citation_author=NA McClain; citation_author=A Gautam; citation_author=D Scholl-Meeker; citation_journal_title=J Bacteriol; citation_volume=187; citation_number=62; citation_pages=2592-2600; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Selective decay of Escherichia coli dnaG messenger RNA is initiated by RNase E.; citation_author=V Yajnik; citation_author=GN Godson; citation_journal_title=J Biol Chem; citation_volume=268; citation_number=63; citation_pages=13253-13260; citation_date=1993; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000878" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Comparative Expression Profiling of the Chlamydia trachomatis pmp Gene Family for Clinical and Reference Strains"/>
    <meta name="twitter:description" content="BackgroundChlamydia trachomatis, an obligate intracellular pathogen, is a leading worldwide cause of ocular and urogenital diseases. Advances have been made in our understanding of the nine-member polymorphic membrane protein (Pmp) gene (pmp) family of C. trachomatis. However, there is only limited information on their biologic role, especially for biological variants (biovar) and clinical strains.Methodology/Principal FindingsWe evaluated expression for pmps throughout development for reference strains E/Bour and L2/434, representing different biovars, and for clinical E and L2 strains. Immunoreactivity of patient sera to recombinant (r)Pmps was also determined. All pmps were expressed at two hours. pmpA had the lowest expression but was up-regulated at 12 h for all strains, indicating involvement in reticulate body development. For pmpD, expression peaked at 36 h. Additionally, 57.7% of sera from infected and 0% from uninfected adolescents were reactive to rPmpD (p&#8202;=&#8202;0.001), suggesting a role in immunogenicity. pmpF had the highest expression levels for all clinical strains and L2/434 with differential expression of the pmpFE operon for the same strains. Sera were nonreactive to rPmpF despite immunoreactivity to rMOMP and rPmpD, suggesting that PmpF is not associated with humoral immune responses. pmpFE sequences for clinical strains were identical to those of the respective reference strains. We identified the putative pmpFE promoter, which was, surprisingly, 100% conserved for all strains. Analyses of ribosomal binding sites, RNase E, and hairpin structures suggested complex regulatory mechanism(s) for this &gt;6 Kb operon.Conclusions/SignificanceThe dissimilar expression of the same pmp for different C. trachomatis strains may explain different strain-specific needs and phenotypic distinctions. This is further supported by the differential immunoreactivity to rPmpD and rPmpF of sera from patients infected with different strains. Furthermore, clinical E strains did not correlate with the E reference strain at the gene expression level, reinforcing the need for expansive studies of clinical strains."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0000878.g005"/>

  <meta property="og:title" content="Comparative Expression Profiling of the Chlamydia trachomatis pmp Gene Family for Clinical and Reference Strains" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000878" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=2395'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=2265'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=4793&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0000878">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000878" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000878&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Comparative Expression Profiling of the <i>Chlamydia trachomatis pmp</i> Gene Family for Clinical and Reference Strains
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Alexandra Nunes, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliations:
                  Center for Immunobiology and Vaccine Development, Children's Hospital Oakland Research Institute, Oakland, California, United States of America, 
                  Departamento de Bacteriologia, Instituto Nacional de Saúde, Lisboa, Portugal
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            João P. Gomes, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliations:
                  Center for Immunobiology and Vaccine Development, Children's Hospital Oakland Research Institute, Oakland, California, United States of America, 
                  Departamento de Bacteriologia, Instituto Nacional de Saúde, Lisboa, Portugal
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Sally Mead, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Center for Immunobiology and Vaccine Development, Children's Hospital Oakland Research Institute, Oakland, California, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Carlos Florindo, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Departamento de Bacteriologia, Instituto Nacional de Saúde, Lisboa, Portugal
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Helena Correia, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Departamento de Bacteriologia, Instituto Nacional de Saúde, Lisboa, Portugal
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Maria J. Borrego, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Departamento de Bacteriologia, Instituto Nacional de Saúde, Lisboa, Portugal
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Deborah Dean
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:ddean@chori.org">ddean@chori.org</a></p>

                <p>Affiliations:
                  Center for Immunobiology and Vaccine Development, Children's Hospital Oakland Research Institute, Oakland, California, United States of America, 
                  Department of Bioengineering, University of California at Berkeley, Berkeley, California, United States of America, 
                  Department of Medicine, School of Medicine, University of California at San Francisco, San Francisco, California, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: September 12, 2007</li>
    <li>DOI: 10.1371/journal.pone.0000878</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0000878" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0000878" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000878" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0000878" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0000878-t001" data-doi="info:doi/10.1371/journal.pone.0000878" data-uri="info:doi/10.1371/journal.pone.0000878.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000878.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000878-g001" data-doi="info:doi/10.1371/journal.pone.0000878" data-uri="info:doi/10.1371/journal.pone.0000878.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000878.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000878-g002" data-doi="info:doi/10.1371/journal.pone.0000878" data-uri="info:doi/10.1371/journal.pone.0000878.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000878.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000878-g003" data-doi="info:doi/10.1371/journal.pone.0000878" data-uri="info:doi/10.1371/journal.pone.0000878.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000878.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000878-g004" data-doi="info:doi/10.1371/journal.pone.0000878" data-uri="info:doi/10.1371/journal.pone.0000878.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000878.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000878-g005" data-doi="info:doi/10.1371/journal.pone.0000878" data-uri="info:doi/10.1371/journal.pone.0000878.g005" title="Figure 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000878.g005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000878-t002" data-doi="info:doi/10.1371/journal.pone.0000878" data-uri="info:doi/10.1371/journal.pone.0000878.t002" title="Table 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000878.t002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000878">Reader Comments (3)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0000878" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2>
<h3>Background</h3>
<a id="article1.front1.article-meta1.abstract1.sec1.p1" name="article1.front1.article-meta1.abstract1.sec1.p1"></a><p><em>Chlamydia trachomatis</em>, an obligate intracellular pathogen, is a leading worldwide cause of ocular and urogenital diseases. Advances have been made in our understanding of the nine-member polymorphic membrane protein (Pmp) gene (<em>pmp)</em> family of <em>C. trachomatis</em>. However, there is only limited information on their biologic role, especially for biological variants (biovar) and clinical strains.</p>


<h3>Methodology/Principal Findings</h3>
<a id="article1.front1.article-meta1.abstract1.sec2.p1" name="article1.front1.article-meta1.abstract1.sec2.p1"></a><p>We evaluated expression for <em>pmp</em>s throughout development for reference strains E/Bour and L<sub>2</sub>/434, representing different biovars, and for clinical E and L<sub>2 </sub>strains. Immunoreactivity of patient sera to recombinant (r)Pmps was also determined. All <em>pmp</em>s were expressed at two hours. <em>pmpA</em> had the lowest expression but was up-regulated at 12 h for all strains, indicating involvement in reticulate body development. For <em>pmpD</em>, expression peaked at 36 h. Additionally, 57.7% of sera from infected and 0% from uninfected adolescents were reactive to rPmpD (p = 0.001), suggesting a role in immunogenicity. <em>pmpF</em> had the highest expression levels for all clinical strains and L<sub>2</sub>/434 with differential expression of the <em>pmpFE</em> operon for the same strains. Sera were nonreactive to rPmpF despite immunoreactivity to rMOMP and rPmpD, suggesting that PmpF is not associated with humoral immune responses. <em>pmpFE</em> sequences for clinical strains were identical to those of the respective reference strains. We identified the putative <em>pmpFE</em> promoter, which was, surprisingly, 100% conserved for all strains. Analyses of ribosomal binding sites, RNase E, and hairpin structures suggested complex regulatory mechanism(s) for this &gt;6 Kb operon.</p>


<h3>Conclusions/Significance</h3>
<a id="article1.front1.article-meta1.abstract1.sec3.p1" name="article1.front1.article-meta1.abstract1.sec3.p1"></a><p>The dissimilar expression of the same <em>pmp</em> for different <em>C. trachomatis</em> strains may explain different strain-specific needs and phenotypic distinctions. This is further supported by the differential immunoreactivity to rPmpD and rPmpF of sera from patients infected with different strains. Furthermore, clinical E strains did not correlate with the E reference strain at the gene expression level, reinforcing the need for expansive studies of clinical strains.</p>

</div>


<div class="articleinfo"><p><strong>Citation: </strong>Nunes A, Gomes JP, Mead S, Florindo C, Correia H, et al.  (2007) Comparative Expression Profiling of the <em>Chlamydia trachomatis pmp</em> Gene Family for Clinical and Reference Strains. PLoS ONE 2(9):
          e878.
            doi:10.1371/journal.pone.0000878</p><p><strong>Academic Editor: </strong>Niyaz Ahmed, Centre for DNA Fingerprinting and Diagnostics, India</p><p><strong>Received:</strong> July 9, 2007; <strong>Accepted:</strong> August 18, 2007; <strong>Published:</strong> September 12, 2007</p><p><strong>Copyright:</strong> © 2007 Nunes et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>This work was supported by grants from Fundação Para a Ciência e Tecnologia and FEDER (POCTI/39822/MGI/2001) (to M.J. Borrego), and Public Health Service Grants from the National Institutes of Health, R01-AI39499 and R01-AI059647 (to D. Dean).</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p><em>Chlamydia trachomatis</em> is an obligate intracellular pathogen that is responsible for significant worldwide morbidity associated with ocular and sexually transmitted diseases (STD). The developmental cycle of the organism is biphasic beginning with the adhesion of the elementary body (EB), an infectious and metabolically inert form, to the host cell. After endocytosis, the EB differentiates ~2 to 12 h post infection (p.i.) into a larger, non-infectious and metabolically active reticulate body (RB), which initiates intracellular replication by binary fission within a vacuole called an inclusion body. At ~30 to 36 h p.i., RBs reorganize into new EBs, which are released by host cell lysis or exocytosis at 48 to 72 h p.i. that initiates another infectious cycle <a href="#pone.0000878-Moulder1">[1]</a>, <a href="#pone.0000878-Ward1">[2]</a>.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>The disease spectrum of <em>C. trachomatis</em> ranges from conjunctivitis and ocular trachoma to tubal factor infertility, ectopic pregnancy and infant pneumonitis <a href="#pone.0000878-Dean1">[3]</a>, <a href="#pone.0000878-Dean2">[4]</a>. <em>C. trachomatis</em> serological variants (serovars) are grouped within two human biological variants (biovars) according to characteristics of disease presentation: the trachoma biovar, including serovars A to C and Ba, which cause conjunctivitis and trachoma, and serovars D to K and Ba, Da, Ia and Ja, which cause conjunctivitis, urogenital infections and infant pneumonitis, and the lymphogranuloma venereum (LGV) biovar (serovars L<sub>1</sub> to L<sub>3</sub> and L<sub>2a</sub>). The latter serovars are more invasive, causing genital ulceration, lymphadenitis and proctitis <a href="#pone.0000878-Dean1">[3]</a>, <a href="#pone.0000878-Fields1">[5]</a>. However, serotyping of the major outer membrane protein (MOMP), and phylogenetic reconstructions of this protein and the corresponding gene (<em>ompA</em>) <a href="#pone.0000878-Fitch1">[6]</a>, <a href="#pone.0000878-Dean3">[7]</a> do not group serovars by trachoma, non-invasive urogenital or invasive LGV disease groups.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>The molecular mechanisms behind these biological differences among serovars (or strains) are not well understood. Genome sequences of reference strains D/UW-3 <a href="#pone.0000878-Stephens1">[8]</a> and A/Har-13 <a href="#pone.0000878-Carlson1">[9]</a>, as well as ongoing <em>C. trachomatis</em> genomic sequencing are providing information on specific genes and proteins that may explain tissue tropism and virulence differences for the three disease groups. <em>C. trachomatis</em> contains a partial tryptophan operon (<em>trpRBA</em>) where urogenital strains, but not trachoma strains, can synthesize tryptophan from mucosal substrates <a href="#pone.0000878-Caldwell1">[10]</a>. The toxin gene possesses an intact N-terminal region that encodes an active enzymatic domain for the urogenital strains but not for trachoma or LGV strains <a href="#pone.0000878-Carlson2">[11]</a>.</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p>Research on the nine member polymorphic membrane protein (Pmp) gene (<em>pmp</em>) family has revealed phylogenetic reconstructions where six <em>pmp</em>s (<em>pmpB</em>, <em>pmpC</em>, <em>pmpF</em>, <em>pmpG</em>, <em>pmpH</em> and <em>pmpI</em>) form clades that correspond to the three disease groups <a href="#pone.0000878-Gomes1">[12]</a>–<a href="#pone.0000878-Gomes2">[14]</a>. At the gene expression level, previous RT-PCR analyses of reference strains D/UW-3 and L<sub>2</sub>/434 <a href="#pone.0000878-Lindquist1">[15]</a>, and microarray analysis of D/UW-3 <a href="#pone.0000878-Belland1">[16]</a> found that all nine <em>pmp</em>s were transcribed starting at 8 h p.i. Yet, based on real-time quantitative (k)RT-PCR, we found expression as early as 2 h p.i. for <em>pmpC</em> for reference strains Ba/Apache-2, G/UW-57 and L<sub>2</sub>/434, and a differential expression profile with earlier up-regulation of <em>pmpC</em> for L<sub>2</sub>/434 <a href="#pone.0000878-Gomes3">[17]</a>. Also, another study based on kRT-PCR, Kiselev <em>et al.</em> <a href="#pone.0000878-Kiselev1">[18]</a> detected <em>pmpD</em> expression as early as 2h p.i. for L<sub>2</sub>/434. Proteomics analyses have also shown that all Pmps of L<sub>2</sub>/434 are detected as outer membrane constituents <a href="#pone.0000878-Skipp1">[19]</a>–<a href="#pone.0000878-Tanzer1">[22]</a>. There is also evidence that some Pmps are antigenic for human sera <a href="#pone.0000878-Lindquist1">[15]</a>, <a href="#pone.0000878-Hsia1">[23]</a>. We observed a heterogeneous immunoreactivity to recombinant (r)PmpC using sera from patients infected with different <em>C. trachomatis</em> strains, suggesting a role for PmpC in antigenic variation <a href="#pone.0000878-Gomes3">[17]</a>. More recently, Pmps have been considered autotransporters based on bioinformatics analyses <a href="#pone.0000878-Henderson1">[24]</a>, <a href="#pone.0000878-Henderson2">[25]</a>. Wehrl <em>et al</em>. <a href="#pone.0000878-Wehrl1">[26]</a> has experimentally demonstrated the autotransporter model for the <em>C. pneumoniae</em> ortholog of <em>C. trachomatis</em> PmpD, Pmp21. Further, using immunofluorescence microscopy, Western blotting and penicillin treatment, the results of Kiselev <em>et al.</em> <a href="#pone.0000878-Kiselev1">[18]</a> for L<sub>2</sub>/434 PmpD are in general agreement with the autotransporter model for this protein. PmpD has also been shown to be a species-common neutralizing antigen <a href="#pone.0000878-Crane1">[27]</a>, while PmpF has been implicated as a potential target of the host immune response as it contains several predicted major histocompatibility (MHC) epitopes within the N-terminal domain <a href="#pone.0000878-Carlson1">[9]</a>.</p>
<a id="article1.body1.sec1.p5" name="article1.body1.sec1.p5"></a><p>Despite the potential importance of Pmps in chlamydial biology, there is a lack of expression data for the <em>pmp</em> genes as well as an insufficient understanding of the host immune response to their proteins. Here, we profile the expression of all <em>pmp</em>s throughout development for reference strains E/Bour and L<sub>2</sub>/434, representing the two <em>C. trachomatis</em> biovars. We chose E/Bour because it is the most prevalent strain worldwide, although the mechanisms of its ecological success are not yet understood. L<sub>2</sub>/434 was selected as it has been widely studied with a plethora of biological information for comparative analyses. The biological uniqueness of these two strains <em>in vivo</em> is reflected in their differential tissue tropism, virulence and disease presentation. In light of our recent findings that reference strains do not represent the same genetic composition of clinical strains that are circulating among human populations today <a href="#pone.0000878-Gomes4">[28]</a>, we also compared the nine <em>pmp</em> expression levels for four C. <em>trachomatis</em> clinical strains, representing <em>ompA</em> genotypes of E and L<sub>2</sub>. Further, we examined the immunoreactivity of sera from adolescents with and without <em>C. trachomatis</em> urogenital infections against rPmps to further define their potential importance in human disease.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4><em>C. trachomatis</em> cell culture of reference strains and clinical strains</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p><em>C. trachomatis</em> reference strains E/Bour and L<sub>2</sub>/434, three clinical strains belonging to <em>ompA</em> genotype E (designated as E/537C-05, E/S-141 and E/CS-500-96) and one clinical strain belonging to <em>ompA</em> genotype L<sub>2 </sub>were evaluated in this study. E/537C-05 and E/S-141 were collected from patients with vaginal discharge, E/CS-500-96 from a patient with cervicitis, and L<sub>2</sub> from a patient with proctitis. Each was propagated in HeLa 229 cell monolayers using standard techniques as previously described <a href="#pone.0000878-Dean2">[4]</a>, <a href="#pone.0000878-Dean4">[29]</a>. EBs were harvested at 48–72 h p.i. and purified by discontinuous density centrifugation in Renografin <a href="#pone.0000878-Caldwell2">[30]</a>.</p>
<a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2"></a><p>Confluent HeLa cells were either mock-infected or infected with a multiplicity of infection of one for each reference strain or clinical strain in SPG prior to incubation with culture medium <a href="#pone.0000878-Dean2">[4]</a>, <a href="#pone.0000878-Dean4">[29]</a>. Eight T<sub>25</sub> flasks (one for each time point of 2, 6, 12, 18, 24, 36, and 48 h and mock-infected) per strain were inoculated and placed at 37°C in 5% CO<sub>2</sub> <a href="#pone.0000878-Gomes3">[17]</a>. Cultured cells were harvested at each time point, and total RNA was extracted as previously described <a href="#pone.0000878-Gomes3">[17]</a>.</p>


<h4>Reverse Transcription and Quantitative Real-Time PCR (kPCR)</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>RNA was quantified by O.D. measured at A<sub>260</sub>. cDNA was generated from 500 ng of each RNA sample using TaqMan RT Reagents (Applied Biosystems, Foster City, CA) and random hexamers, and was quantified by O.D. measured at A<sub>260</sub>.</p>
<a id="article1.body1.sec2.sec2.p2" name="article1.body1.sec2.sec2.p2"></a><p>Quantitation of <em>pmp</em> expression was achieved using the ABI 7000 SDS (Applied Biosystems), SYBR Green chemistry, and the standard curve method for relative quantitation, using reagents and thermocycling as previously described <a href="#pone.0000878-Gomes3">[17]</a>. <em>16SrRNA</em> was used as the endogenous control since normalizing the data against <em>16SrRNA</em> provides a control for the number of organisms (EBs and RBs) and, therefore, for the differential growth rate of each strain. <em>ompA</em> was included as a quality control for kRT-PCR results since it has been widely used for gene expression studies <a href="#pone.0000878-Belland1">[16]</a>, <a href="#pone.0000878-Gomes3">[17]</a>.</p>
<a id="article1.body1.sec2.sec2.p3" name="article1.body1.sec2.sec2.p3"></a><p>Primers for each of the nine <em>pmp</em>s (<a href="#pone-0000878-t001">Table 1</a>) were designed using Primer Express (Applied Biosystems). Primers for <em>ompA</em>, <em>16SrRNA</em>, and <em>pmpC</em> were used as previously designed (<a href="#pone-0000878-t001">Table 1</a>) <a href="#pone.0000878-Gomes3">[17]</a>.</p>
<div class="figure" id="pone-0000878-t001"><div class="img"><a name="pone-0000878-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000878.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000878" data-uri="info:doi/10.1371/journal.pone.0000878.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000878.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000878.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000878.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000878.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000878.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000878.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000878.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000878.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000878.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>Oligonucleotide primers used for kRT-PCR</span></strong></p><span>doi:10.1371/journal.pone.0000878.t001</span></div><a id="article1.body1.sec2.sec2.p4" name="article1.body1.sec2.sec2.p4"></a><p>Each plate contained two replicates of each sample cDNA, three different negative controls and standard curves for each gene as previously described (17). For all experiments, the amount of target and control gene was determined from the respective standard curve by conversion of the mean threshold cycle values. Normalization was obtained by dividing the quantity of the target gene by the quantity of the control gene. The specificity of the amplified products was verified by analysis of the dissociation curves generated by the ABI7000 software based on the specific melting temperature for each amplicon. The results were based on three independent experiments for reference strains E/Bour and L<sub>2</sub>/434, and for the four clinical strains.</p>


<h4>Genetic analysis of the <em>pmpFE</em> operon for <em>C. trachomatis</em> reference and clinical strains</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>Based on the considerable expression disparities between <em>pmpF</em> and <em>pmpE</em> (which belong to the same operon) for reference strain L<sub>2</sub>/434 and mostly for the clinical strains (see <a href="#s3">results</a> below), we sequenced the <em>pmpFE</em> operon as well as the upstream 164 base pair (bp) <em>pmpG/pmpF</em> intergenomic region (IGR) that likely contains the operon regulatory region. In the <em>C. trachomatis</em> chromosome, <em>pmpF</em> and <em>pmpE</em> are located on the minus strand; <em>pmpF</em> is located upstream of <em>pmpE</em>, with a 2 bp IGR . <em>pmpE</em> was sequenced for the six strains (Genbank Accession Numbers EF490370 for E/537C-05, EF490371 for E/CS-500-96, EF490372 for E/S-141, EF490373 for L2, EF490374 for E/Bour, and EF490375 for L<sub>2</sub>/434), while <em>pmpG/pmpF</em> IGR and <em>pmpF</em> were sequenced only for the clinical strains (Genbank Accession Numbers EF490366 for L2, EF490367 for E/CS-500-96, EF490368 for E/S-141, and EF490369 for E/537C-05), as the sequences for the reference strains were available from our previous study (GenBank Accession Number AY887650 for E/Bour and AY887660 for L<sub>2</sub>/434) <a href="#pone.0000878-Gomes1">[12]</a>. The amplification and sequencing strategies were performed as previously described <a href="#pone.0000878-Gomes1">[12]</a>, except for the <em>pmpG/pmpF</em> IGR, where we used primer 5′-ACT CGG ATC TCC TAT AAC AG-3′ for sequencing.</p>
<a id="article1.body1.sec2.sec3.p2" name="article1.body1.sec2.sec3.p2"></a><p>Since the transcription process can be strongly affected by the structure and sequence variability of promoter regions <a href="#pone.0000878-Schaumburg1">[31]</a>–<a href="#pone.0000878-Tan1">[34]</a>, a putative promoter search for the <em>pmpFE</em> operon was performed using EditSeq software (DNASTAR, Madison, WI) for sequences described in the literature and also by using two promoter prediction programs: <a href="http://www.fruitfly.org/seq_tools/promoter.html">http://www.fruitfly.org/seq_tools/promot​er.html</a> and <a href="http://www.prodoric.de/vfp/vfp_promoter.php">http://www.prodoric.de/vfp/vfp_promoter.​php</a>. Given the expression dissimilarities obtained in this study for <em>pmpF</em> and <em>pmpE</em>, we searched for putative Shine-Dalgarno ribosome binding sequences (RBS) <a href="#pone.0000878-Shine1">[35]</a> as well as previously described chlamydial RBS <a href="#pone.0000878-Stephens2">[36]</a>–<a href="#pone.0000878-Danilition1">[39]</a> within this operon, since ribosomes can either prolong or shorten the lifetime of mRNA in response to events that occur during translation or termination processes <a href="#pone.0000878-Deana1">[40]</a>. The existence of putative consensus cleavage sites for RNase E <a href="#pone.0000878-Ehretsmann1">[41]</a>–<a href="#pone.0000878-Chelladurai1">[43]</a>, the major endonuclease that generally initiates mRNA degradation in most bacteria <a href="#pone.0000878-Deutscher1">[44]</a>, was also examined within the <em>pmpFE</em> operon. These two analyses were performed using EditSeq software (DNASTAR). Putative stem-loop structures were searched throughout the <em>pmpFE</em> operon using GeneQuest software (DNASTAR) and RNAstructure software version 4.4 (<a href="http://rna.urmc.rochester.edu/rnastructure.html">http://rna.urmc.rochester.edu/rnastructu​re.html</a>) due to the regulatory or processing role of stem-loop structures in premature transcription termination as well as in mRNA degradation and maturation mechanisms <a href="#pone.0000878-Meinken1">[45]</a>–<a href="#pone.0000878-Washio1">[49]</a>, respectively.</p>


<h4>Immunoreactivity of patient sera against Pmp fusion proteins</h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>We generated fusion proteins for PmpD and PmpF because the latter displayed such high mRNA expression for L<sub>2</sub>/434 and the clinical strains, and the former was expressed late in development for all strains under study, being the last up-regulated protein for four of the six strains analyzed. Also, PmpD has been associated with neutralizing epitopes <a href="#pone.0000878-Crane1">[27]</a>. The rMOMP fusion protein was available from a previous study <a href="#pone.0000878-Hessel1">[50]</a>. The PET30 expression system (EMD Biosciences, San Diego, CA) was used for cloning PCR products containing <em>pmpD</em> or <em>pmpF</em> generated from strain E/Bour genomic DNA as we have described <a href="#pone.0000878-Gomes3">[17]</a>. The forward and reverse primers were 5′-GACGACGACAAGATGAGTTCCGAGAAAGATATA-3′ and 5′-AATGCTGGATTGCGATTGATCTTTTAACCGGGCTTCT​CCTC-3′for <em>pmpD</em>, respectively, and 5′-GACGACGACAAGATGATTAAAAGAACTTCTCTA-3′ and 5′-AATGCAGGAGGAGCTCTGGTCTTTTAACCGGGCTTCT​CCTC-3′for <em>pmpF</em>, respectively. Sequencing confirmed that the insert was in frame with the S-tag and His-tag as we have described previously for rPmpC <a href="#pone.0000878-Gomes3">[17]</a>. The clones were transformed into <em>E. coli</em> BL21 and induced using 0.1µM IPTG during the exponential growth phase. Ni-agarose (Sigma, St. Louis, MO) was used for fusion protein purification according to the package insert. Recombinant proteins were determined to be the correct molecular weight (calculated at ~160.6 kDa for rPmpD and ~112.3 kDa for rPmpF) by immunoblot using AP-conjugated S-protein, which binds to the S-tag peptide with a distinct band at the correct molecular weight for each. Optimal protein concentrations were determined and standardized using BCA (Pierce, Rockford, Ill) before analyzing the clinical sera. The optimal protein concentration for rPmpD was 50 ng and 100 ng for rPmpF. Sera from 39 consented female adolescents 14 to 19 years of age attending clinics in Oakland, CA, were used at a 1:50 dilution for immunoblotting as described previously <a href="#pone.0000878-Gomes3">[17]</a>. The Institutional Review Board of Children's Hospital Oakland approved the study, and all patients provided written consent for all clinical samples that were obtained and used in this study. The blots were blocked with Blotto prior to reacting with patient sera and alkaline phosphatase-conjugated anti-human IgG (R&amp;D Systems, Minneapolis, MN). The chemifluorescent substrate ECF (Amersham, Piscataway, NJ) was used to visualize reactive bands. Twenty six (67%) of the 39 adolescents were infected with a single <em>ompA</em> genotype: 3 Ba, 3 D, 8 E, 5 F, 1 G, 1 Ia, 2 J, and 3 K.</p>
<a id="article1.body1.sec2.sec4.p2" name="article1.body1.sec2.sec4.p2"></a><p>There was no evidence for mixed infections. The original cervical samples were used for sequencing (see ref 17) to best determine the presence of a mixed infection, since propagation may result in one strain overgrowing another. On inspection of the electropherograms, none of the samples had ambiguous results. All nucleotides were represented by single, clear peaks with extremely low background and without evidence for double peaks in a nucleotide position where different <em>ompA</em> genotypes differ, which might suggest a mixed infection.</p>

</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Results"></a><h3>Results</h3>
<h4>Real-time quantitation for <em>pmp</em> expression</h4>
<a id="article1.body1.sec3.sec1.p1" name="article1.body1.sec3.sec1.p1"></a><p>The results of specificity assays revealed no non-specific products, and indicated the presence of the expected amplicon for each gene. Standard curves for all 11 genes had slope values between -3.1 and -3.5, which represents efficiencies between 93 and 100%. There were only minor variations in the slope for each standard curve among independent experiments, indicating a highly reproducible kPCR as we have also shown in previous experiments <a href="#pone.0000878-Gomes3">[17]</a>. We defined the gene expression profile as the qualitative gene expression pattern throughout development where quantitative values are not considered. For example, one expression profile would show increasing expression up to a peak with tapering down of the expression after the peak. An expression peak was defined as the time point of the highest relative mRNA value. All quantitative expression comparisons refer to differences between the expression peak of each gene, even those occurring at different time points p.i.</p>


<h4>Expression profile of the nine <em>pmp</em> genes throughout development for E/Bour and L<sub>2</sub>/434</h4>
<a id="article1.body1.sec3.sec2.p1" name="article1.body1.sec3.sec2.p1"></a><p>L<sub>2</sub>/434 had strikingly different mRNA levels among some <em>pmp</em>s and also between different time points for the same <em>pmp</em> (<a href="#pone-0000878-g001">Fig. 1A</a>). <em>pmpF</em> had the highest relative mRNA expression, up to 11.5-fold higher than for <em>pmpA</em>, the least expressed gene. mRNA levels were detected at 2 h p.i. for all <em>pmps</em>, including <em>pmpA</em> where the scale limits visualization of the low mRNA expression, and peaked at different time points. For all <em>pmps</em> except <em>pmpA,</em> mRNA levels decreased consistently after the peak until 48 h.</p>
<div class="figure" id="pone-0000878-g001"><div class="img"><a name="pone-0000878-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000878.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000878" data-uri="info:doi/10.1371/journal.pone.0000878.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000878.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000878.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000878.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000878.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000878.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000878.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000878.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000878.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000878.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Expression profile of the nine <em>pmp</em> genes and <em>ompA</em> throughout the development of <em>C. trachomatis.</em></span></strong></p><a id="article1.body1.sec3.sec2.fig1.caption1.p1" name="article1.body1.sec3.sec2.fig1.caption1.p1"></a><p>Reference strain L<sub>2</sub>/434 is represented in panel A and E/Bour in panel B. Values represent the mean±SEM based on three independent experiments for time points of 2, 6, 12, 18, 24, 36, and 48 h post infection. See <a href="#s2">methods</a> for details.</p>
<span>doi:10.1371/journal.pone.0000878.g001</span></div><a id="article1.body1.sec3.sec2.p2" name="article1.body1.sec3.sec2.p2"></a><p>The <em>pmp</em> expression profiles for E/Bour were more homogeneous than for L<sub>2</sub>/434, and, in some cases, mRNA levels were lower than for the corresponding L<sub>2</sub>/434 <em>pmp</em> (<a href="#pone-0000878-g001">Fig. 1B</a>). Similar to L<sub>2</sub>/434, mRNA levels were detected at 2 h p.i. for all <em>pmp</em> genes. <em>pmpE, pmpF</em> and <em>pmpG</em> showed the highest expression levels for strain E/Bour with up to 3.1-fold higher mRNA levels than for <em>pmpA</em> and <em>pmpI</em>, the least expressed genes. As for L<sub>2</sub>/434, <em>pmpA</em> and <em>pmpD</em> were the earliest and the latest up-regulated genes, respectively. In contrast to L<sub>2</sub>/434, all <em>pmp</em> genes except <em>pmpA</em> and <em>pmpI</em> had stable mRNA levels after the expression peak until 48 h. For this reason, the expression peak for E/Bour <em>pmps</em> was defined as the time point at which a noticeable expression increase occurred.</p>
<a id="article1.body1.sec3.sec2.p3" name="article1.body1.sec3.sec2.p3"></a><p>For both reference strains, <em>ompA</em> had remarkably higher mRNA values at all time points than for the <em>pmp</em>s (<a href="#pone-0000878-g001">Fig. 1</a>). In contrast to most <em>pmp</em>s, <em>ompA</em> revealed a similar gene expression profile for both reference strains.</p>


<h4>Expression profile of the nine <em>pmp</em> genes throughout development for <em>C. trachomatis</em> clinical strains</h4>
<a id="article1.body1.sec3.sec3.p1" name="article1.body1.sec3.sec3.p1"></a><p>The four clinical strains had a similar <em>pmp</em> expression profile (<a href="#pone-0000878-g002">Fig. 2</a>), which showed decreasing mRNA levels after the expression peak to 48 h. mRNA levels were detected at 2 h p.i. for all <em>pmp</em>s, although the scale limits visualization. Overall, <em>pmp</em>s peaked at 18h for L<sub>2</sub> (<a href="#pone-0000878-g002">Fig. 2A</a>) and E/CS-500-96 (<a href="#pone-0000878-g002">Fig. 2D</a>), and at 36h for clinical strains E/537C-05 (<a href="#pone-0000878-g002">Fig. 2B</a>) and E/S-141 (<a href="#pone-0000878-g002">Fig. 2C</a>). Similar to reference strains E and L<sub>2</sub>, <em>pmpA</em> was the first up-regulated gene for all clinical strains. In addition, <em>pmpD</em> was the last up-regulated gene for clinical strains L<sub>2</sub> and E/CS-500-96 (<a href="#pone-0000878-g002">Fig. 2A and 2D</a>), and was also expressed late in development (together with other <em>pmp</em>s) for the other two clinical E strains under study.</p>
<div class="figure" id="pone-0000878-g002"><div class="img"><a name="pone-0000878-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000878.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000878" data-uri="info:doi/10.1371/journal.pone.0000878.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000878.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000878.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000878.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000878.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000878.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000878.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000878.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000878.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000878.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Expression profile of <em>pmp</em> and <em>ompA</em> genes throughout the development of <em>C. trachomatis</em> clinical strains.</span></strong></p><a id="article1.body1.sec3.sec3.fig1.caption1.p1" name="article1.body1.sec3.sec3.fig1.caption1.p1"></a><p>(A) Strain L<sub>2</sub> shares the same <em>ompA</em> genotype as L<sub>2</sub>/434; and strains E/537C-05 (B), E/S-141 (C) and E/CS-500-96 (D) share the same <em>ompA</em> genotype as E/Bour. Values represent the mean±SEM based on three independent experiments for time points of 2, 6, 12, 18, 24, 36, and 48 h post infection. See <a href="#s2">methods</a> for details.</p>
<span>doi:10.1371/journal.pone.0000878.g002</span></div><a id="article1.body1.sec3.sec3.p2" name="article1.body1.sec3.sec3.p2"></a><p><em>pmpF</em> had the highest expression among all of the <em>pmps</em> for the clinical strains (<a href="#pone-0000878-g002">Fig. 2</a>). In fact, there was a 27-fold higher expression of <em>pmpF</em> compared with the least expressed gene (<em>pmpD</em>) for L<sub>2</sub>. For clinical E strains, there was a 19.2- and 22.6-fold higher expression of <em>pmpF</em> compared with the least expressed gene (<em>pmpI</em>) for E/537C-05 and E/CS-500-96 respectively, and a 54.2-fold higher expression than <em>pmpA</em> for E/S-141 Although no relevant dissimilarities were observed for <em>pmpF</em> between clinical L<sub>2</sub> and L<sub>2</sub>/434, there were considerable expression differences among the clinical strains and E/Bour with up to 11.7-fold higher mRNA values for E/CS-500-96 than for E/Bour.</p>
<a id="article1.body1.sec3.sec3.p3" name="article1.body1.sec3.sec3.p3"></a><p>For <em>ompA</em>, mRNA levels peaked at 36 h for E/537C-05 and E/S-141, and at 18h for E/CS-500-96 and L<sub>2</sub>, declining thereafter (<a href="#pone-0000878-g002">Fig. 2</a>). The most striking example of differential mRNA levels between <em>ompA</em> and <em>pmp</em>s occurred for E/S-141, where <em>ompA</em> had a 232.0-fold higher value compared with the least expressed gene, <em>pmpA</em>. However, all clinical strains except E/CS-500-96 had lower <em>ompA</em> expression levels for all time points compared with the corresponding reference strains.</p>


<h4>Genetic analysis of <em>pmpFE</em> operon for <em>C. trachomatis</em> reference strains and clinical strains</h4>
<a id="article1.body1.sec3.sec4.p1" name="article1.body1.sec3.sec4.p1"></a><p>The <em>pmpF</em>, <em>pmpE</em> and <em>pmpG/pmpF</em> IGR sequences for the three clinical E strains were 100% similar to the corresponding E/Bour sequences, while L<sub>2</sub> showed 4 nucleotide (nt) differences to L<sub>2</sub>/434 but only for <em>pmpE</em>. Compared to both L<sub>2</sub> strains, the four E strains showed 317 (10.2%) nt and 106 (10.3%) amino acid (aa) differences for <em>pmpF</em> as well as 56 (1.9%) nt and 21 (20 to L<sub>2</sub>) (2.1%) aa differences for <em>pmpE</em>. For the <em>pmpG/pmpF</em> IGR, which comprises the ~164 bp upstream region of <em>pmpF</em>, there were 5 nt differences between the L<sub>2</sub> and the E strains, although none of them fell within the putative promoter region for the <em>pmpFE</em> operon (<a href="#pone-0000878-g003">Fig. 3</a>). The putative promoter is located within a 100% conserved stretch of the <em>pmpG/pmpF</em> IGR for both reference and all clinical strains (<a href="#pone-0000878-g003">Fig. 3</a>). The -10 promoter element (TAAAAT) identified in this study was identical to the one that was previously characterized for the L<sub>2</sub>/434 and D/UW-3 <em>ltuB</em> promoter, while the -35 region (TTGCAT) was 100% similar to the <em>hctA</em> promoter of the same chlamydial reference strains <a href="#pone.0000878-Schaumburg2">[32]</a>.</p>
<div class="figure" id="pone-0000878-g003"><div class="img"><a name="pone-0000878-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000878.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000878" data-uri="info:doi/10.1371/journal.pone.0000878.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000878.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000878.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000878.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000878.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000878.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000878.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000878.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000878.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000878.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Predicted <em>pmpF</em> promoter sequence for reference and clinical strains.</span></strong></p><a id="article1.body1.sec3.sec4.fig1.caption1.p1" name="article1.body1.sec3.sec4.fig1.caption1.p1"></a><p>Sequences are for reference strains E/Bour and L<sub>2</sub>/434, and clinical strains E/537C-05, E/S-141, E/CS-500-96, and L<sub>2</sub>. The predicted transcription promoter for <em>pmpF</em> is located within a 100% conserved region of the <em>pmpG/pmpF</em> IGR, where putative -10 and -35 elements are in blue characters and boxed. Potential A/T spacer region is underlined, and the predicted transcription start site is shown in a larger font below a red asterisk. The putative RBS for <em>pmpF</em> is in orange characters, and the putative RNase E cleavage sites are highlighted in grey. Numbers represent positions relative to the start codon of <em>pmpF</em> (highlighted in yellow). The start codon of <em>pmpG</em> is highlighted in blue.</p>
<span>doi:10.1371/journal.pone.0000878.g003</span></div><a id="article1.body1.sec3.sec4.p2" name="article1.body1.sec3.sec4.p2"></a><p>Analysis of the <em>pmpFE</em> operon sequence revealed several putative hairpin loop structures although the actual RNA folding in those regions functioning as a classic rho-independent type transcriptional terminator <a href="#pone.0000878-Washio1">[49]</a> cannot be assumed. At least 41 putative RNase E cleavage sites were identified throughout the <em>pmpFE</em> operon, 13 of which were not conserved between L<sub>2</sub>/434 (and L<sub>2</sub>) and the four E strains (<a href="#pone-0000878-g004">Fig. 4</a>). One of these non-conserved sites involved the <em>pmpF/pmpE</em> IGR, and is specific for the E strains. The search for an RBS revealed a perfect prokaryotic Shine-Dalgarno sequence (AGGAGG) located 17 nts upstream of the start codon of <em>pmpE,</em> which is approximately 3000 bp below the last bp in <a href="#pone-0000878-g003">Figure 3</a> and, therefore, is not shown. This RBS is in close proximity to the above-described putative RNase cleavage site shared only by the four E strains. However, the best approach for a putative RBS sequence for <em>pmpF</em> has two mismatches when compared with the ones described in the literature, and is unusually distant from the start codon (<a href="#pone-0000878-g003">Fig. 3</a>). Two additional putative RNase E cleavage sites, one of which was in close proximity to this RBS, were identified within the <em>pmpF</em> regulatory region (<a href="#pone-0000878-g003">Fig. 3</a>).</p>
<div class="figure" id="pone-0000878-g004"><div class="img"><a name="pone-0000878-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000878.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000878" data-uri="info:doi/10.1371/journal.pone.0000878.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000878.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000878.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000878.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000878.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000878.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000878.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000878.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000878.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000878.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>Distribution/Location of the putative RNase E cleavage sites within the <em>pmpFE</em> operon coding sequence.</span></strong></p><a id="article1.body1.sec3.sec4.fig2.caption1.p1" name="article1.body1.sec3.sec4.fig2.caption1.p1"></a><p>The sequence is for reference strains E/Bour and L<sub>2</sub>/434 and for clinical strains E/537C-05, E/S-141, E/CS-500-96 and L<sub>2</sub>. Black vertical lines represent all RNase E cleavage sites conserved among all strains under study; green vertical lines show the ones only conserved among the four “E” strains; orange vertical lines represent those specific solely for both L<sub>2</sub> strains. Numbers represent nucleotide positions relative to the start codon of <em>pmpF.</em></p>
<span>doi:10.1371/journal.pone.0000878.g004</span></div>

<h4>Immunoreactivity of patient sera with Pmp fusion proteins</h4>
<a id="article1.body1.sec3.sec5.p1" name="article1.body1.sec3.sec5.p1"></a><p><a href="#pone-0000878-t002">Table 2</a> shows the clinical and microbiologic characteristics of the 39 adolescents enrolled in the study and the results of their serum immunoreactivity to rPmpD and rPmpF. All sera from patients infected with chlamydial clinical strains Ba, E, F and K (n = 15; 57.7%), but none with D, Ia, J or G (n = 11; 42.3%), were reactive to rPmpD while sera from uninfected patients were nonreactive with rPmpD (p = 0.001). <a href="#pone-0000878-g005">Figure 5</a> shows the immunoblot results of representative sera from patients infected with Ba, D, E, F, G, Ia, J and K to rPmpD. Because <em>pmpD</em> is highly conserved among all reference strains <a href="#pone.0000878-Gomes1">[12]</a>, constructing rPmpD using the <em>pmpD</em> sequence of reference strain E/Bour should not have contributed to the observed differences in immunoreactivity. Further, cross-reactivity between strains was unlikely since the patients were infected with only a single strain, and sera that were reactive to rPmpD were not reactive to rPmpF. In our previous study, sera form the same individuals infected with clinical strains D, E and G reacted with rPmpC <a href="#pone.0000878-Gomes3">[17]</a>. Surprisingly, none of the sera reacted to rPmpF (<a href="#pone-0000878-g005">Figure 5</a>), not even sera from the eight patients infected with strain E, although all sera from infected patients and one uninfected patient reacted with rMOMP as previously shown <a href="#pone.0000878-Gomes3">[17]</a>.</p>
<div class="figure" id="pone-0000878-g005"><div class="img"><a name="pone-0000878-g005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000878.g005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000878" data-uri="info:doi/10.1371/journal.pone.0000878.g005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000878.g005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000878.g005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000878.g005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000878.g005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000878.g005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000878.g005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000878.g005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000878.g005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000878.g005.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 5.  <span>Dot-Blot of serum immunoreactivity against recombinant (r)PmpD and rPmpF.</span></strong></p><a id="article1.body1.sec3.sec5.fig1.caption1.p1" name="article1.body1.sec3.sec5.fig1.caption1.p1"></a><p>Sera was obtained from adolescents singly infected and uninfected with a different <em>C. trachomatis</em> clinical strain as described previously <a href="#pone.0000878-Gomes3">[17]</a> (see also <a href="#s2">methods</a>). rPmpD and rPmpF concentrations were standardized for use on the blots. Immunoreactivity to each fusion protein for sera from patients infected with strain Ba (n = 3), D (n = 3), E (n = 8), F (n = 5), G (n = 1), Ia (n = 1), J (n = 2) or K (n = 3) are shown. Of note is that immunoreactivity was consistent for sera from patients infected with the same clinical strain except for strain F (<a href="#pone-0000878-t002">Table 2</a>); all eight patients infected with strain E were reactive to rPmpD.</p>
<span>doi:10.1371/journal.pone.0000878.g005</span></div><div class="figure" id="pone-0000878-t002"><div class="img"><a name="pone-0000878-t002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000878.t002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000878" data-uri="info:doi/10.1371/journal.pone.0000878.t002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000878.t002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000878.t002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000878.t002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000878.t002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000878.t002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000878.t002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000878.t002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000878.t002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000878.t002.TIF"></span>)
                </a></li></ul></div><p><strong>Table 2.  <span>Clinical and microbiologic characteristics of female adolescents from whom sera was used for determining the immunoreactivity against rPmpD and rPmpF</span></strong></p><span>doi:10.1371/journal.pone.0000878.t002</span></div>
</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec4.p1" name="article1.body1.sec4.p1"></a><p>In this study, we determined the gene expression profile of the nine <em>pmp</em>s throughout development for reference strains L<sub>2</sub>/434 and E/Bour, and four clinical strains belonging to <em>ompA</em> genotypes E and L<sub>2</sub>. The reference strains had significant gene expression differences where E/Bour had relatively lower mRNA levels and generally sustained expression from 24 to 48 h compared with L<sub>2</sub>/434 (<a href="#pone-0000878-g001">Fig. 1</a>). Surprisingly, in contrast to clinical L<sub>2</sub>, the three clinical E strains showed a dissimilar <em>pmp</em> expression profile to E/Bour (<a href="#pone-0000878-g002">Fig. 2</a>). These remarkable expression dissimilarities are generally supported by our recent comparative genomics findings where the laboratory adapted reference strains did not reflect the same genetic make-up of strains that are circulating among human populations today and currently exposed to immune selection <a href="#pone.0000878-Gomes4">[28]</a>.</p>
<a id="article1.body1.sec4.p2" name="article1.body1.sec4.p2"></a><p>It is well known that the developmental stages for reference E strains occur at later time points than for reference L<sub>2</sub> strains <a href="#pone.0000878-Miyairi1">[51]</a><strong>.</strong> This is supported by the <em>ompA</em> expression for E/Bour, which is shifted ~6 hours later than for L<sub>2</sub>/434. However, the differential growth rate between these two reference strains does not explain the dissimilar <em>pmp</em> expression, as most <em>pmp</em>s were up-regulated at the same time point for both (<a href="#pone-0000878-g001">Fig. 1</a>). For E/Bour, almost all of the <em>pmp</em>s had increased expression during the second half of development with comparable mRNA levels at these stages, suggesting a similar involvement in RB division and RB to EB transformation. For L<sub>2</sub>/434, although most <em>pmp</em>s showed a general up-regulation of transcription at the exponential growth phase of RB division when new membranes are being formed, <em>pmpC, pmpE and pmpF</em> appeared to play a more important role during this phase. Thus, the gene expression results of both E/Bour and L<sub>2</sub>/434 suggest their potential importance in membrane integrity. However, some Pmps may have more specific functions than others, depending on the chlamydial strain. In support of this, a proteomics study by Shaw <em>et al.</em> <a href="#pone.0000878-Shaw1">[21]</a> detected five Pmps among reference strains A/HAR-13, D/UW-3, and L<sub>2</sub>/434, where PmpF was only detected for L<sub>2</sub>/434. In another proteomics study, only <em>pmpE</em>, <em>pmpG</em>, and <em>pmpH</em> were detected for L<sub>2</sub>/434 <a href="#pone.0000878-Tanzer1">[22]</a>. However, it is possible that these studies reflect a lack of sensitivity in detecting Pmps since a recent study was able to detect all Pmps for L<sub>2</sub>/434 <a href="#pone.0000878-Skipp1">[19]</a>.</p>
<a id="article1.body1.sec4.p3" name="article1.body1.sec4.p3"></a><p>Interestingly, <em>pmpA</em> had, in general, the lowest expression levels of all <em>pmp</em>s at each time point except that it had one of the highest levels at 12 h p.i. (<a href="#pone-0000878-g001">Fig. 1</a> and <a href="#pone-0000878-g002">2</a>), suggesting a greater importance of PmpA during early stages of development. This is supported by shotgun proteomics where Skipp <em>et al.</em> <a href="#pone.0000878-Skipp1">[19]</a> identified PmpA exclusively in RBs, whereas all other Pmps were detected in both RBs and EBs for L<sub>2</sub>/434. Additionally, for PmpD, the late up-regulation at 36 h corresponds to RB transformation into EBs, suggesting a role in EB outer membrane structure. In support of this, PmpD has a cysteine content considerably higher than any other Pmp <a href="#pone.0000878-Gomes1">[12]</a>. There are 26 conserved cysteine residues in PmpD for all 19 <em>C. trachomatis</em> reference strains, while the mean for all other Pmps is only 13.9 [SE 2.3]. Cysteine residues are responsible for the highly disulfide cross-linked proteins of the outer membrane complex of EBs. Previous studies found that PmpD is surface located and cross-linked in the chlamydial outer membrane complex through disulfide bonds <a href="#pone.0000878-Kiselev2">[52]</a>. Furthermore, the N-terminal domain of <em>C. pneumoniae</em> Pmp21, the <em>C. trachomatis</em> PmpD ortholog, was shown to be non-covalently bound to other components of the EB surface <a href="#pone.0000878-Wehrl1">[26]</a>. Additionally, PmpD has shown species-specific neutralizing activity <a href="#pone.0000878-Crane1">[27]</a>. These collective data are supported by our findings that sera from <em>C. trachomatis</em> infected patients were reactive to rPmpD (<a href="#pone-0000878-g005">Fig. 5</a>). Our results were remarkably consistent for sera from patients infected with the same strain. For example, sera from all eight patients infected with strain E were reactive as were sera from three patients infected with strain Ba and three infected with strain K, although only one of the five patients with strain F were reactive; none of patients with strains D, Ia, J or G were reactive. Additional research is required to determine epitopes on PmpD that may correlate with the differential immune responses we observed.</p>
<a id="article1.body1.sec4.p4" name="article1.body1.sec4.p4"></a><p>Overall, considering both reference strains and clinical strains, <em>pmpA</em> and <em>pmpI</em> were the least expressed genes, while <em>pmpF</em> was the most highly expressed, although <em>pmpE</em> and <em>pmpG</em> also had similar expression levels for E/Bour. We previously found that PmpF is the most polymorphic protein among the <em>C. trachomatis</em> Pmps for both reference and clinical strains <a href="#pone.0000878-Gomes4">[28]</a>, <a href="#pone.0000878-Gomes1">[12]</a>. Consistent with the observed protein diversity, phylogenetic analyses of PmpF grouped <em>C. trachomatis</em> strains by tissue tropism properties <a href="#pone.0000878-Gomes1">[12]</a>. Further, comparative analyses of PmpF reveal distinct domains that may be associated with a specific disease group.</p>
<a id="article1.body1.sec4.p5" name="article1.body1.sec4.p5"></a><p>The outer membrane exposure of the N-terminus has been experimentally demonstrated for some <em>C. pneumoniae</em> Pmps <a href="#pone.0000878-Wehrl1">[26]</a>, <a href="#pone.0000878-Vandahl1">[53]</a>, suggesting that these proteins may be subjected to host immune pressure. The N-terminal half for <em>C. trachomatis</em> PmpF also contains numerous non-synonymous amino acid changes at locations of predicted MHC epitopes <a href="#pone.0000878-Carlson1">[9]</a>, indicating that it may be involved in eliciting a cellular immune response. Our findings that none of the sera from infected patients reacted with rPmpF suggest that this protein is not associated with the humeral immune response. Strain origin (E/Bour) of rPmpF did not seem to be an issue as sera from the eight patients infected with strain E were non-reactive. Furthermore, sensitivity was unlikely to be an issue given the immunoreactivity of the same sera with rPmpC and rMOMP, as we have previously described <a href="#pone.0000878-Gomes3">[17]</a>, and with rPmpD in this study. The occurrence of highly repeated GGAI motifs in the N-terminus suggests that Pmps may be associated with cell adhesion <a href="#pone.0000878-Grimwood1">[54]</a>, which has been reported for Pmp21 of <em>C. pneumoniae</em> <a href="#pone.0000878-Wehrl1">[26]</a>. These cumulative findings suggest that Pmps are expressed with a differential immune response for patients infected with a specific strain. These findings and the remarkable <em>pmpF</em> expression dissimilarities among L<sub>2</sub>/434, E/Bour and the clinical strains suggest that there may be differential biological functions across strains and within the same strain for PmpF, either as a structural component to maintain membrane integrity, as part of a large pool of polymorphic antigens to elicit cellular immunity, or as an adhesin.</p>
<a id="article1.body1.sec4.p6" name="article1.body1.sec4.p6"></a><p>In our study, <strong>t</strong>he <em>pmpF</em> sequences for the three clinical E strains were found to be 100% similar to the E/Bour sequence as was L<sub>2</sub> to L<sub>2</sub>/434. Since it is highly unlikely that identical proteins have diverse functions, we hypothesized that there may be differential regulation at the promoter level or regulation involving variations in mRNA processing and/or degradation, which would yield distinct mRNA amounts according to strain-specific needs. It is well known that point mutations in regulatory regions, such as promoter regions and RBS, can affect transcription and translation levels. However, analysis of the putative promoter region and RBS for <em>pmpF</em> showed that they are 100% conserved for both reference and the clinical strains (<a href="#pone-0000878-g003">Fig. 3</a>), suggesting that the observed <em>pmpF</em> expression heterogeneity may result from variations in mRNA processing and/or degradation. In fact, regulatory systems of gene expression acting at both the transcriptional and translational levels are well represented in the chlamydial genome, including homologues of endoribonucleases E, III, G and P, exoribonucleases II and PNPase, and oligoribonuclease <a href="#pone.0000878-Stephens1">[8]</a>, <a href="#pone.0000878-Carlson1">[9]</a>. These are known to control mRNA stability and processing as well as translational efficiency in other bacteria, such as <em>Escherichia coli</em> and <em>Staphylococcus aureus</em> <a href="#pone.0000878-Deutscher1">[44]</a>, <a href="#pone.0000878-Kushner1">[48]</a>, <a href="#pone.0000878-Kennell1">[55]</a>–<a href="#pone.0000878-Huntzinger1">[57]</a>. The susceptibility of mRNA to ribonuclease attack may be influenced by events occurring not only at any stage during ribosome binding, but also during translation elongation or termination <a href="#pone.0000878-Deana1">[40]</a>.</p>
<a id="article1.body1.sec4.p7" name="article1.body1.sec4.p7"></a><p>We identified two conserved putative RNase E cleavage sites in the <em>pmpG/pmpF</em> IGR, one of which is in close proximity to the putative RBS (<a href="#pone-0000878-g003">Fig. 3</a>). It is known that RBS sequence variability and sequestering by competitive regulatory proteins or conformational impediments can affect ribosome binding/loading and, thus, mRNA lifetime <a href="#pone.0000878-Deana1">[40]</a>. Considering this, a hypothetical initial cleavage by RNase E could reduce the affinity of the <em>pmpF</em> translation initiation region for ribosomes, thereby allowing subsequent mRNA degradation/processing by endo- and exonucleases, preferentially for E/Bour when compared to the other strains. A similar regulation has already been reported for <em>sodB</em> mRNA of <em>E. coli</em> at low iron concentrations <a href="#pone.0000878-Afonyushkin1">[58]</a>. However, this hypothetical mechanism, although possible, is speculative and lacks experimental evidence.</p>
<a id="article1.body1.sec4.p8" name="article1.body1.sec4.p8"></a><p><em>pmpF</em> and <em>pmpE</em> belong to the same operon, yet had remarkably dissimilar mRNA levels for L<sub>2</sub>/434, and more so for all clinical strains with up to 8.4-fold higher expression for <em>pmpF</em> than for <em>pmpE</em> (<a href="#pone-0000878-g001">Fig. 1A</a> and <a href="#pone-0000878-g002">2</a>). This did not occur for the <em>pmpGH</em> operon. We speculated that the expression heterogeneity within the <em>pmpFE</em> operon may be generated by premature termination of transcription, rapid mRNA processing, or mRNA degradation primarily of the downstream gene (<em>pmpE</em>) of this large operon transcript (&gt;6 Kb). Similar regulatory mechanisms have already been suggested to explain the existence of multiple transcripts within other bacterial policistronic operons <a href="#pone.0000878-Kennell1">[55]</a>, such as those of <em>Bacillus subtilis ara</em> <a href="#pone.0000878-SaNogueira1">[59]</a>, <em>Nitrosomonas europaea cbb</em> <a href="#pone.0000878-Wei1">[60]</a>, and <em>Borrelia burgdorferi ospAB</em> and <em>bmpAB</em> <a href="#pone.0000878-Liang1">[61]</a>, <a href="#pone.0000878-Ramamoorthy1">[62]</a>.</p>
<a id="article1.body1.sec4.p9" name="article1.body1.sec4.p9"></a><p>Although we cannot assume that the putative stem-loop structures found within the <em>pmpFE</em> operon sequence may act as classic rho-independent type transcriptional terminators <a href="#pone.0000878-Washio1">[49]</a>, the possibility of hairpin formation (a common phenomenon in mRNA, mainly on large transcripts) cannot be ignored nor can its hypothetical processing role in mRNA degradation and maturation be discounted. Furthermore, several putative RNase E cleavage sites were identified throughout the <em>pmpFE</em> operon (<a href="#pone-0000878-g004">Fig. 4</a>), which is expected for policistronic operons, although it is well known that RNase E cleaves mRNA only at a limited number of sites <a href="#pone.0000878-Kennell1">[55]</a>. Interestingly, some of the RNase E sites were not conserved between L<sub>2</sub> and E strains, suggesting that targeted mRNA degradation or rapid processing events may occur in this large transcript. Curiously, one of these non-conserved recognition sites involved solely the <em>pmpF/pmpE</em> IGR of the four E strains. Thus, if RNase E uses this cleavage site, subsequent degradation or processing events from this point would only occur for E strains and could hypothetically yield an mRNA decay of <em>pmpE</em>. Yet, as above, this mechanism is speculative and lacks experimental evidence. However, in a previous study, differential transcript quantities were reported for the MMSO genes of <em>E. coli</em> that contained a consensus RNase E cleavage site in the intergenic regions of the operon, suggesting complex mRNA processing <a href="#pone.0000878-Yajnik1">[63]</a>.</p>
<a id="article1.body1.sec4.p10" name="article1.body1.sec4.p10"></a><p>Overall, the heterogeneous expression levels among <em>pmp</em>s and among strains highlight the importance of this gene family in chlamydial biology. In particular, the unique expression disparity for the <em>pmpFE</em> operon with relatively high <em>pmpF</em> mRNA levels for 5 of the 6 strains under study, as well as the differential immunoreactivity of patient sera to rPmpD, suggest that some Pmps may explain phenotypic differences among strains for antigenicity, virulence and tissue tropism. Furthermore, our findings that clinical E strains do not correlate with reference strain E/Bour at the gene expression level are supported by our previously reported genomic data <a href="#pone.0000878-Gomes4">[28]</a>, reinforcing the need to examine clinical along with reference strains to advance our understanding of the role of <em>pmp</em>s in chlamydial biology and disease pathogenesis.</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>We would like to express our gratitude to the clinicians, staff and patients (and their parents) of the teen clinics in Oakland for their willingness to participate in the study.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: DD AN JG MB. Performed the experiments: AN CF JG SM HC MB AN. Analyzed the data: DD AN JG MB AN. Contributed reagents/materials/analysis tools: DD MB. Wrote the paper: DD AN JG AN.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0000878-Moulder1" id="pone.0000878-Moulder1"></a>Moulder JW (1991) Interaction of chlamydiae and host cells in vitro. Microbiol Rev  55: 143–190.  <ul class="find" data-citedArticleID="990804"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Interaction+of+chlamydiae+and+host+cells+in+vitro.&amp;auth=&amp;atitle=Interaction+of+chlamydiae+and+host+cells+in+vitro." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Interaction+of+chlamydiae+and+host+cells+in+vitro." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Interaction+of+chlamydiae+and+host+cells+in+vitro.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0000878-Ward1" id="pone.0000878-Ward1"></a>Ward ME (1988)  The chlamydial development cycle. In: Barron AL, editor. The Microbiology of Chlamydia. Boca Raton, , FL: CRC Press, Inc. pp. 71–97.  <ul class="find-nolinks"></ul></li><li><span class="label">3.
              </span><a name="pone.0000878-Dean1" id="pone.0000878-Dean1"></a>Dean D (1997)  <em>Chlamydia trachomatis</em> Sexually Transmitted Diseases. In: Connor DH, Schwartz DA, Chandler FW, editors. Pathology of Infectious Diseases. Stamford, , CT: Appleton and Lange Publishers. pp. 473–490.  <ul class="find-nolinks"></ul></li><li><span class="label">4.
              </span><a name="pone.0000878-Dean2" id="pone.0000878-Dean2"></a>Dean D, Powers VC (2001) Persistent Chlamydia trachomatis infections resist apoptotic stimuli. Infect Immun  69: 2442–2447.  <ul class="find" data-citedArticleID="990746" data-doi="10.1128/iai.69.4.2442-2447.2001"><li><a href="http://dx.doi.org/10.1128/iai.69.4.2442-2447.2001" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Persistent+Chlamydia+trachomatis+infections+resist+apoptotic+stimuli." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Persistent+Chlamydia+trachomatis+infections+resist+apoptotic+stimuli.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0000878-Fields1" id="pone.0000878-Fields1"></a>Fields PI, Barnes RC (1992)  The Genus Chlamydia. In: Balows A, Truper HG, Dworkin M, Harder W, Schleifer KH, editors. The Prokaryotes. New York: Springer-Verlag. pp. 3691–3709.  <ul class="find-nolinks"></ul></li><li><span class="label">6.
              </span><a name="pone.0000878-Fitch1" id="pone.0000878-Fitch1"></a>Fitch WM, Peterson EM, de la Maza LM (1993) Phylogenetic analysis of the outer-membrane-protein genes of Chlamydiae, and its implication for vaccine development. Mol Biol Evol  10: 892–913.  <ul class="find" data-citedArticleID="990762"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Phylogenetic+analysis+of+the+outer-membrane-protein+genes+of+Chlamydiae%2C+and+its+implication+for+vaccine+development.&amp;auth=&amp;atitle=Phylogenetic+analysis+of+the+outer-membrane-protein+genes+of+Chlamydiae%2C+and+its+implication+for+vaccine+development." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Phylogenetic+analysis+of+the+outer-membrane-protein+genes+of+Chlamydiae%2C+and+its+implication+for+vaccine+development." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Phylogenetic+analysis+of+the+outer-membrane-protein+genes+of+Chlamydiae%2C+and+its+implication+for+vaccine+development.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0000878-Dean3" id="pone.0000878-Dean3"></a>Dean D, Millman K (1997) Molecular and mutation trends analyses of <em>omp1</em> alleles for serovar E of <em>Chlamydia trachomatis</em>. Implications for the immunopathogenesis of disease. J Clin Invest  99: 475–483.  <ul class="find" data-citedArticleID="990748" data-doi="10.1172/jci119182"><li><a href="http://dx.doi.org/10.1172/jci119182" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Molecular+and+mutation+trends+analyses+of+omp1+alleles+for+serovar+E+of+Chlamydia+trachomatis.+Implications+for+the+immunopathogenesis+of+disease." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Molecular+and+mutation+trends+analyses+of+omp1+alleles+for+serovar+E+of+Chlamydia+trachomatis.+Implications+for+the+immunopathogenesis+of+disease.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0000878-Stephens1" id="pone.0000878-Stephens1"></a>Stephens RS, Kalman S, Lammel C, Fan J, Marathe R, et al.  (1998) Genome sequence of an obligate intracellular pathogen of humans: <em>Chlamydia trachomatis</em>. Science  282: 754–759.  <ul class="find" data-citedArticleID="990824" data-doi="10.1126/science.282.5389.754"><li><a href="http://dx.doi.org/10.1126/science.282.5389.754" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genome+sequence+of+an+obligate+intracellular+pathogen+of+humans%3A+Chlamydia+trachomatis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genome+sequence+of+an+obligate+intracellular+pathogen+of+humans%3A+Chlamydia+trachomatis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0000878-Carlson1" id="pone.0000878-Carlson1"></a>Carlson JH, Porcella SF, McClarty G, Caldwell HD (2005) Comparative genomic analysis of Chlamydia trachomatis oculotropic and genitotropic strains. Infect Immun  73: 6407–6418.  <ul class="find" data-citedArticleID="990734" data-doi="10.1128/iai.73.10.6407-6418.2005"><li><a href="http://dx.doi.org/10.1128/iai.73.10.6407-6418.2005" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Comparative+genomic+analysis+of+Chlamydia+trachomatis+oculotropic+and+genitotropic+strains." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Comparative+genomic+analysis+of+Chlamydia+trachomatis+oculotropic+and+genitotropic+strains.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0000878-Caldwell1" id="pone.0000878-Caldwell1"></a>Caldwell HD, Wood H, Crane D, Bailey R, Jones RB, et al.  (2003) Polymorphisms in <em>Chlamydia trachomatis</em> tryptophan synthase genes differentiate between genital and ocular isolates. J Clin Invest  111: 1757–1769.  <ul class="find" data-citedArticleID="990730" data-doi="10.1172/jci17993"><li><a href="http://dx.doi.org/10.1172/jci17993" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Polymorphisms+in+Chlamydia+trachomatis+tryptophan+synthase+genes+differentiate+between+genital+and+ocular+isolates." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Polymorphisms+in+Chlamydia+trachomatis+tryptophan+synthase+genes+differentiate+between+genital+and+ocular+isolates.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0000878-Carlson2" id="pone.0000878-Carlson2"></a>Carlson JH, Hughes S, Hogan D, Cieplak G, Sturdevant DE, et al.  (2004) Polymorphisms in the Chlamydia trachomatis cytotoxin locus associated with ocular and genital isolates. Infect Immun  72: 7063–7072.  <ul class="find" data-citedArticleID="990736" data-doi="10.1128/iai.72.12.7063-7072.2004"><li><a href="http://dx.doi.org/10.1128/iai.72.12.7063-7072.2004" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Polymorphisms+in+the+Chlamydia+trachomatis+cytotoxin+locus+associated+with+ocular+and+genital+isolates." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Polymorphisms+in+the+Chlamydia+trachomatis+cytotoxin+locus+associated+with+ocular+and+genital+isolates.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0000878-Gomes1" id="pone.0000878-Gomes1"></a>Gomes JP, Nunes A, Bruno WJ, Borrego MJ, Florindo C, et al.  (2006) Polymorphisms in the Nine Polymorphic Membrane Proteins of Chlamydia trachomatis across All Serovars: Evidence for Serovar Da Recombination and Correlation with Tissue Tropism. J Bacteriol  188: 275–286.  <ul class="find" data-citedArticleID="990764" data-doi="10.1128/jb.188.1.275-286.2006"><li><a href="http://dx.doi.org/10.1128/jb.188.1.275-286.2006" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Polymorphisms+in+the+Nine+Polymorphic+Membrane+Proteins+of+Chlamydia+trachomatis+across+All+Serovars%3A+Evidence+for+Serovar+Da+Recombination+and+Correlation+with+Tissue+Tropism." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Polymorphisms+in+the+Nine+Polymorphic+Membrane+Proteins+of+Chlamydia+trachomatis+across+All+Serovars%3A+Evidence+for+Serovar+Da+Recombination+and+Correlation+with+Tissue+Tropism.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0000878-Stothard1" id="pone.0000878-Stothard1"></a>Stothard DR, Toth GA, Batteiger BE (2003) Polymorphic membrane protein H has evolved in parallel with the three disease-causing groups of Chlamydia trachomatis. Infect Immun  71: 1200–1208.  <ul class="find" data-citedArticleID="990828" data-doi="10.1128/iai.71.3.1200-1208.2003"><li><a href="http://dx.doi.org/10.1128/iai.71.3.1200-1208.2003" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Polymorphic+membrane+protein+H+has+evolved+in+parallel+with+the+three+disease-causing+groups+of+Chlamydia+trachomatis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Polymorphic+membrane+protein+H+has+evolved+in+parallel+with+the+three+disease-causing+groups+of+Chlamydia+trachomatis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0000878-Gomes2" id="pone.0000878-Gomes2"></a>Gomes JP, Bruno WJ, Borrego MJ, Dean D (2004) Recombination in the genome of <em>Chlamydia trachomatis</em> involving the polymorphic membrane protein C gene relative to <em>ompA</em> and evidence for horizontal gene transfer. J Bacteriol  186: 4295–4306.  <ul class="find" data-citedArticleID="990766" data-doi="10.1128/jb.186.13.4295-4306.2004"><li><a href="http://dx.doi.org/10.1128/jb.186.13.4295-4306.2004" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Recombination+in+the+genome+of+Chlamydia+trachomatis+involving+the+polymorphic+membrane+protein+C+gene+relative+to+ompA+and+evidence+for+horizontal+gene+transfer." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Recombination+in+the+genome+of+Chlamydia+trachomatis+involving+the+polymorphic+membrane+protein+C+gene+relative+to+ompA+and+evidence+for+horizontal+gene+transfer.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0000878-Lindquist1" id="pone.0000878-Lindquist1"></a>Lindquist EA, Stephens RS (1998)  Transcriptional activity of a sequence variable protein family in <em>Chlamydia trachomatis</em>. In: Stephens RS, Byrne GI, Christiansen G, Clarke IN, Grayston JT, et al., editors. Chlamydial Infections Proceedings of the Ninth International Symposium on Human Chlamydial Infection. Napa, , California: International Chlamydia Symposium. pp. 259–262.  <ul class="find-nolinks"></ul></li><li><span class="label">16.
              </span><a name="pone.0000878-Belland1" id="pone.0000878-Belland1"></a>Belland RJ, Zhong G, Crane DD, Hogan D, Sturdevant D, et al.  (2003) Genomic transcriptional profiling of the developmental cycle of Chlamydia trachomatis. Proc Natl Acad Sci U S A  100: 8478–8483.  <ul class="find" data-citedArticleID="990728" data-doi="10.1073/pnas.1331135100"><li><a href="http://dx.doi.org/10.1073/pnas.1331135100" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genomic+transcriptional+profiling+of+the+developmental+cycle+of+Chlamydia+trachomatis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genomic+transcriptional+profiling+of+the+developmental+cycle+of+Chlamydia+trachomatis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0000878-Gomes3" id="pone.0000878-Gomes3"></a>Gomes JP, Hsia RC, Mead S, Borrego MJ, Dean D (2005) Immunoreactivity and differential developmental expression of known and putative <em>Chlamydia trachomatis</em> membrane proteins for biologically variant serovars representing distinct disease groups. Microbes Infect  7: 410–420.  <ul class="find" data-citedArticleID="990768" data-doi="10.1016/j.micinf.2004.11.014"><li><a href="http://dx.doi.org/10.1016/j.micinf.2004.11.014" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Immunoreactivity+and+differential+developmental+expression+of+known+and+putative+Chlamydia+trachomatis+membrane+proteins+for+biologically+variant+serovars+representing+distinct+disease+groups." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Immunoreactivity+and+differential+developmental+expression+of+known+and+putative+Chlamydia+trachomatis+membrane+proteins+for+biologically+variant+serovars+representing+distinct+disease+groups.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0000878-Kiselev1" id="pone.0000878-Kiselev1"></a>Kiselev AO, Stamm WE, Yates JR, Lampe MF (2007) Expression, Processing, and Localization of PmpD of Chlamydia trachomatis Serovar L2 during the Chlamydial Developmental Cycle. PLoS ONE  2: e568.  <ul class="find" data-citedArticleID="990786" data-doi="10.1371/journal.pone.0000568"><li><a href="http://dx.doi.org/10.1371/journal.pone.0000568" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Expression%2C+Processing%2C+and+Localization+of+PmpD+of+Chlamydia+trachomatis+Serovar+L2+during+the+Chlamydial+Developmental+Cycle." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Expression%2C+Processing%2C+and+Localization+of+PmpD+of+Chlamydia+trachomatis+Serovar+L2+during+the+Chlamydial+Developmental+Cycle.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0000878-Skipp1" id="pone.0000878-Skipp1"></a>Skipp P, Robinson J, O'Connor CD, Clarke IN (2005) Shotgun proteomic analysis of Chlamydia trachomatis. Proteomics  5: 1558–1573.  <ul class="find" data-citedArticleID="990822" data-doi="10.1002/pmic.200401044"><li><a href="http://dx.doi.org/10.1002/pmic.200401044" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Shotgun+proteomic+analysis+of+Chlamydia+trachomatis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Shotgun+proteomic+analysis+of+Chlamydia+trachomatis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0000878-Mygind1" id="pone.0000878-Mygind1"></a>Mygind PH, Christiansen G, Roepstorff P, Birkelund S (2000) Membrane proteins PmpG and PmpH are major constituents of Chlamydia trachomatis L2 outer membrane complex. FEMS Microbiol Lett  186: 163–169.  <ul class="find" data-citedArticleID="990806" data-doi="10.1111/j.1574-6968.2000.tb09098.x"><li><a href="http://dx.doi.org/10.1111/j.1574-6968.2000.tb09098.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Membrane+proteins+PmpG+and+PmpH+are+major+constituents+of+Chlamydia+trachomatis+L2+outer+membrane+complex." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Membrane+proteins+PmpG+and+PmpH+are+major+constituents+of+Chlamydia+trachomatis+L2+outer+membrane+complex.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0000878-Shaw1" id="pone.0000878-Shaw1"></a>Shaw AC, Gevaert K, Demol H, Hoorelbeke B, Vandekerckhove J, et al.  (2002) Comparative proteome analysis of Chlamydia trachomatis serovar A, D and L2. Proteomics  2: 164–186.  <ul class="find" data-citedArticleID="990818" data-doi="10.1002/1615-9861(200202)2:2<164::aid-prot164&gt;3.0.co;2-u"><li><a href="http://dx.doi.org/10.1002/1615-9861(200202)2:2<164::aid-prot164&gt;3.0.co;2-u" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Comparative+proteome+analysis+of+Chlamydia+trachomatis+serovar+A%2C+D+and+L2." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Comparative+proteome+analysis+of+Chlamydia+trachomatis+serovar+A%2C+D+and+L2.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0000878-Tanzer1" id="pone.0000878-Tanzer1"></a>Tanzer RJ, Hatch TP (2001) Characterization of outer membrane proteins in Chlamydia trachomatis LGV serovar L2. J Bacteriol  183: 2686–2690.  <ul class="find" data-citedArticleID="990832" data-doi="10.1128/jb.183.8.2686-2690.2001"><li><a href="http://dx.doi.org/10.1128/jb.183.8.2686-2690.2001" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Characterization+of+outer+membrane+proteins+in+Chlamydia+trachomatis+LGV+serovar+L2." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Characterization+of+outer+membrane+proteins+in+Chlamydia+trachomatis+LGV+serovar+L2.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0000878-Hsia1" id="pone.0000878-Hsia1"></a>Hsia Rc, Ahmed I, Batteiger B, Sekkides O, Ridgway G, et al. 2000 Aug 20–23, 2000.  Differential immune response to polymorphic membrane proteins in STD patients. In: Saikku P, editor. Helsinki, Finland: Societa Editrice Esculapio.   <ul class="find-nolinks"></ul></li><li><span class="label">24.
              </span><a name="pone.0000878-Henderson1" id="pone.0000878-Henderson1"></a>Henderson IR, Lam AC (2001) Polymorphic proteins of Chlamydia spp.–autotransporters beyond the Proteobacteria. Trends Microbiol  9: 573–578.  <ul class="find" data-citedArticleID="990774" data-doi="10.1016/s0966-842x(01)02234-x"><li><a href="http://dx.doi.org/10.1016/s0966-842x(01)02234-x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Polymorphic+proteins+of+Chlamydia+spp.%26%23x2013%3Bautotransporters+beyond+the+Proteobacteria." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Polymorphic+proteins+of+Chlamydia+spp.%26%23x2013%3Bautotransporters+beyond+the+Proteobacteria.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0000878-Henderson2" id="pone.0000878-Henderson2"></a>Henderson IR, Navarro-Garcia F, Nataro JP (1998) The great escape: structure and function of the autotransporter proteins. Trends Microbiol  6: 370–378.  <ul class="find" data-citedArticleID="990776" data-doi="10.1016/s0966-842x(98)01318-3"><li><a href="http://dx.doi.org/10.1016/s0966-842x(98)01318-3" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+great+escape%3A+structure+and+function+of+the+autotransporter+proteins." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+great+escape%3A+structure+and+function+of+the+autotransporter+proteins.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0000878-Wehrl1" id="pone.0000878-Wehrl1"></a>Wehrl W, Brinkmann V, Jungblut PR, Meyer TF, Szczepek AJ (2004) From the inside out–processing of the Chlamydial autotransporter PmpD and its role in bacterial adhesion and activation of human host cells. Mol Microbiol  51: 319–334.  <ul class="find" data-citedArticleID="990840" data-doi="10.1046/j.1365-2958.2003.03838.x"><li><a href="http://dx.doi.org/10.1046/j.1365-2958.2003.03838.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=From+the+inside+out%26%23x2013%3Bprocessing+of+the+Chlamydial+autotransporter+PmpD+and+its+role+in+bacterial+adhesion+and+activation+of+human+host+cells." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22From+the+inside+out%26%23x2013%3Bprocessing+of+the+Chlamydial+autotransporter+PmpD+and+its+role+in+bacterial+adhesion+and+activation+of+human+host+cells.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0000878-Crane1" id="pone.0000878-Crane1"></a>Crane DD, Carlson JH, Fischer ER, Bavoil P, Hsia RC, et al.  (2006) Chlamydia trachomatis polymorphic membrane protein D is a species-common pan-neutralizing antigen. Proc Natl Acad Sci U S A  103: 1894–1899.  <ul class="find" data-citedArticleID="990740" data-doi="10.1073/pnas.0508983103"><li><a href="http://dx.doi.org/10.1073/pnas.0508983103" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Chlamydia+trachomatis+polymorphic+membrane+protein+D+is+a+species-common+pan-neutralizing+antigen." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Chlamydia+trachomatis+polymorphic+membrane+protein+D+is+a+species-common+pan-neutralizing+antigen.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0000878-Gomes4" id="pone.0000878-Gomes4"></a>Gomes JP, Bruno WJ, Nunes A, Santos N, Florindo C, et al.  (2007) Evolution of Chlamydia trachomatis diversity occurs by widespread interstrain recombination involving hotspots. Genome Res  17: 50–60.  <ul class="find" data-citedArticleID="990770" data-doi="10.1101/gr.5674706"><li><a href="http://dx.doi.org/10.1101/gr.5674706" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evolution+of+Chlamydia+trachomatis+diversity+occurs+by+widespread+interstrain+recombination+involving+hotspots." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evolution+of+Chlamydia+trachomatis+diversity+occurs+by+widespread+interstrain+recombination+involving+hotspots.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0000878-Dean4" id="pone.0000878-Dean4"></a>Dean D, Suchland R, Stamm W (2000) Evidence for long-term cervical persistence of <em>Chlamydia trachomatis</em> by <em>omp1</em> genotyping. J Infect Dis  182: 909–916.  <ul class="find" data-citedArticleID="990750" data-doi="10.1086/315778"><li><a href="http://dx.doi.org/10.1086/315778" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evidence+for+long-term+cervical+persistence+of+Chlamydia+trachomatis+by+omp1+genotyping." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evidence+for+long-term+cervical+persistence+of+Chlamydia+trachomatis+by+omp1+genotyping.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0000878-Caldwell2" id="pone.0000878-Caldwell2"></a>Caldwell HD, Kromhout J, Schachter J (1981) Purification and partial characterization of the major outer membrane protein of Chlamydia trachomatis. Infect Immun  31: 1161–1176.  <ul class="find" data-citedArticleID="990732"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Purification+and+partial+characterization+of+the+major+outer+membrane+protein+of+Chlamydia+trachomatis.&amp;auth=&amp;atitle=Purification+and+partial+characterization+of+the+major+outer+membrane+protein+of+Chlamydia+trachomatis." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Purification+and+partial+characterization+of+the+major+outer+membrane+protein+of+Chlamydia+trachomatis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Purification+and+partial+characterization+of+the+major+outer+membrane+protein+of+Chlamydia+trachomatis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0000878-Schaumburg1" id="pone.0000878-Schaumburg1"></a>Schaumburg CS, Tan M (2003) Mutational analysis of the Chlamydia trachomatis dnaK promoter defines the optimal -35 promoter element. Nucleic Acids Res  31: 551–555.  <ul class="find" data-citedArticleID="990814" data-doi="10.1093/nar/gkg150"><li><a href="http://dx.doi.org/10.1093/nar/gkg150" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mutational+analysis+of+the+Chlamydia+trachomatis+dnaK+promoter+defines+the+optimal+-35+promoter+element." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mutational+analysis+of+the+Chlamydia+trachomatis+dnaK+promoter+defines+the+optimal+-35+promoter+element.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0000878-Schaumburg2" id="pone.0000878-Schaumburg2"></a>Schaumburg CS, Tan M (2000) A positive cis-acting DNA element is required for high-level transcription in Chlamydia. J Bacteriol  182: 5167–5171.  <ul class="find" data-citedArticleID="990816" data-doi="10.1128/jb.182.18.5167-5171.2000"><li><a href="http://dx.doi.org/10.1128/jb.182.18.5167-5171.2000" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+positive+cis-acting+DNA+element+is+required+for+high-level+transcription+in+Chlamydia." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+positive+cis-acting+DNA+element+is+required+for+high-level+transcription+in+Chlamydia.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0000878-Douglas1" id="pone.0000878-Douglas1"></a>Douglas AL, Hatch TP (1996) Mutagenesis of the P2 promoter of the major outer membrane protein gene of Chlamydia trachomatis. J Bacteriol  178: 5573–5578.  <ul class="find" data-citedArticleID="990756"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Mutagenesis+of+the+P2+promoter+of+the+major+outer+membrane+protein+gene+of+Chlamydia+trachomatis.&amp;auth=&amp;atitle=Mutagenesis+of+the+P2+promoter+of+the+major+outer+membrane+protein+gene+of+Chlamydia+trachomatis." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mutagenesis+of+the+P2+promoter+of+the+major+outer+membrane+protein+gene+of+Chlamydia+trachomatis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mutagenesis+of+the+P2+promoter+of+the+major+outer+membrane+protein+gene+of+Chlamydia+trachomatis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0000878-Tan1" id="pone.0000878-Tan1"></a>Tan M, Gaal T, Gourse RL, Engel JN (1998) Mutational analysis of the Chlamydia trachomatis rRNA P1 promoter defines four regions important for transcription in vitro. J Bacteriol  180: 2359–2366.  <ul class="find" data-citedArticleID="990830"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Mutational+analysis+of+the+Chlamydia+trachomatis+rRNA+P1+promoter+defines+four+regions+important+for+transcription+in+vitro.&amp;auth=&amp;atitle=Mutational+analysis+of+the+Chlamydia+trachomatis+rRNA+P1+promoter+defines+four+regions+important+for+transcription+in+vitro." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mutational+analysis+of+the+Chlamydia+trachomatis+rRNA+P1+promoter+defines+four+regions+important+for+transcription+in+vitro." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mutational+analysis+of+the+Chlamydia+trachomatis+rRNA+P1+promoter+defines+four+regions+important+for+transcription+in+vitro.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0000878-Shine1" id="pone.0000878-Shine1"></a>Shine J, Dalgarno L (1974) The 3′-terminal sequence of Escherichia coli 16S ribosomal RNA: complementarity to nonsense triplets and ribosome binding sites. Proc Natl Acad Sci U S A  71: 1342–1346.  <ul class="find" data-citedArticleID="990820" data-doi="10.1073/pnas.71.4.1342"><li><a href="http://dx.doi.org/10.1073/pnas.71.4.1342" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+3%26%23x2032%3B-terminal+sequence+of+Escherichia+coli+16S+ribosomal+RNA%3A+complementarity+to+nonsense+triplets+and+ribosome+binding+sites." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+3%26%23x2032%3B-terminal+sequence+of+Escherichia+coli+16S+ribosomal+RNA%3A+complementarity+to+nonsense+triplets+and+ribosome+binding+sites.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">36.
              </span><a name="pone.0000878-Stephens2" id="pone.0000878-Stephens2"></a>Stephens RS, Wagar EA, Edman U (1988) Developmental regulation of tandem promoters for the major outer membrane protein gene of Chlamydia trachomatis. J Bacteriol  170: 744–750.  <ul class="find" data-citedArticleID="990826"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Developmental+regulation+of+tandem+promoters+for+the+major+outer+membrane+protein+gene+of+Chlamydia+trachomatis.&amp;auth=&amp;atitle=Developmental+regulation+of+tandem+promoters+for+the+major+outer+membrane+protein+gene+of+Chlamydia+trachomatis." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Developmental+regulation+of+tandem+promoters+for+the+major+outer+membrane+protein+gene+of+Chlamydia+trachomatis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Developmental+regulation+of+tandem+promoters+for+the+major+outer+membrane+protein+gene+of+Chlamydia+trachomatis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">37.
              </span><a name="pone.0000878-delaMaza1" id="pone.0000878-delaMaza1"></a>de la Maza LM, Fielder TJ, Carlson EJ, Markoff BA, Peterson EM (1991) Sequence diversity of the 60-kilodalton protein and of a putative 15-kilodalton protein between the trachoma and lymphogranuloma venereum biovars of Chlamydia trachomatis. Infect Immun  59: 1196–1201.  <ul class="find" data-citedArticleID="990848"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Sequence+diversity+of+the+60-kilodalton+protein+and+of+a+putative+15-kilodalton+protein+between+the+trachoma+and+lymphogranuloma+venereum+biovars+of+Chlamydia+trachomatis.&amp;auth=&amp;atitle=Sequence+diversity+of+the+60-kilodalton+protein+and+of+a+putative+15-kilodalton+protein+between+the+trachoma+and+lymphogranuloma+venereum+biovars+of+Chlamydia+trachomatis." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Sequence+diversity+of+the+60-kilodalton+protein+and+of+a+putative+15-kilodalton+protein+between+the+trachoma+and+lymphogranuloma+venereum+biovars+of+Chlamydia+trachomatis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Sequence+diversity+of+the+60-kilodalton+protein+and+of+a+putative+15-kilodalton+protein+between+the+trachoma+and+lymphogranuloma+venereum+biovars+of+Chlamydia+trachomatis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">38.
              </span><a name="pone.0000878-Allen1" id="pone.0000878-Allen1"></a>Allen JE, Stephens RS (1989) Identification by sequence analysis of two-site posttranslational processing of the cysteine-rich outer membrane protein 2 of Chlamydia trachomatis serovar L2. J Bacteriol  171: 285–291.  <ul class="find" data-citedArticleID="990726"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Identification+by+sequence+analysis+of+two-site+posttranslational+processing+of+the+cysteine-rich+outer+membrane+protein+2+of+Chlamydia+trachomatis+serovar+L2.&amp;auth=&amp;atitle=Identification+by+sequence+analysis+of+two-site+posttranslational+processing+of+the+cysteine-rich+outer+membrane+protein+2+of+Chlamydia+trachomatis+serovar+L2." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Identification+by+sequence+analysis+of+two-site+posttranslational+processing+of+the+cysteine-rich+outer+membrane+protein+2+of+Chlamydia+trachomatis+serovar+L2." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Identification+by+sequence+analysis+of+two-site+posttranslational+processing+of+the+cysteine-rich+outer+membrane+protein+2+of+Chlamydia+trachomatis+serovar+L2.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">39.
              </span><a name="pone.0000878-Danilition1" id="pone.0000878-Danilition1"></a>Danilition SL, Maclean IW, Peeling R, Winston S, Brunham RC (1990) The 75-kilodalton protein of Chlamydia trachomatis: a member of the heat shock protein 70 family? Infect Immun  58: 189–196.  <ul class="find" data-citedArticleID="990742"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=The+75-kilodalton+protein+of+Chlamydia+trachomatis%3A+a+member+of+the+heat+shock+protein+70+family%3F&amp;auth=&amp;atitle=The+75-kilodalton+protein+of+Chlamydia+trachomatis%3A+a+member+of+the+heat+shock+protein+70+family%3F" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+75-kilodalton+protein+of+Chlamydia+trachomatis%3A+a+member+of+the+heat+shock+protein+70+family%3F" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+75-kilodalton+protein+of+Chlamydia+trachomatis%3A+a+member+of+the+heat+shock+protein+70+family%3F%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">40.
              </span><a name="pone.0000878-Deana1" id="pone.0000878-Deana1"></a>Deana A, Belasco JG (2005) Lost in translation: the influence of ribosomes on bacterial mRNA decay. Genes Dev  19: 2526–2533.  <ul class="find" data-citedArticleID="990752" data-doi="10.1101/gad.1348805"><li><a href="http://dx.doi.org/10.1101/gad.1348805" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Lost+in+translation%3A+the+influence+of+ribosomes+on+bacterial+mRNA+decay." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Lost+in+translation%3A+the+influence+of+ribosomes+on+bacterial+mRNA+decay.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">41.
              </span><a name="pone.0000878-Ehretsmann1" id="pone.0000878-Ehretsmann1"></a>Ehretsmann CP, Carpousis AJ, Krisch HM (1992) Specificity of Escherichia coli endoribonuclease RNase E: in vivo and in vitro analysis of mutants in a bacteriophage T4 mRNA processing site. Genes Dev  6: 149–159.  <ul class="find" data-citedArticleID="990758" data-doi="10.1101/gad.6.1.149"><li><a href="http://dx.doi.org/10.1101/gad.6.1.149" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Specificity+of+Escherichia+coli+endoribonuclease+RNase+E%3A+in+vivo+and+in+vitro+analysis+of+mutants+in+a+bacteriophage+T4+mRNA+processing+site." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Specificity+of+Escherichia+coli+endoribonuclease+RNase+E%3A+in+vivo+and+in+vitro+analysis+of+mutants+in+a+bacteriophage+T4+mRNA+processing+site.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">42.
              </span><a name="pone.0000878-Robertson1" id="pone.0000878-Robertson1"></a>Robertson HD, Dickson E, Dunn JJ (1977) A nucleotide sequence from a ribonuclease III processing site in bacteriophage T7 RNA. Proc Natl Acad Sci U S A  74: 822–826.  <ul class="find" data-citedArticleID="990810" data-doi="10.1073/pnas.74.3.822"><li><a href="http://dx.doi.org/10.1073/pnas.74.3.822" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+nucleotide+sequence+from+a+ribonuclease+III+processing+site+in+bacteriophage+T7+RNA." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+nucleotide+sequence+from+a+ribonuclease+III+processing+site+in+bacteriophage+T7+RNA.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">43.
              </span><a name="pone.0000878-Chelladurai1" id="pone.0000878-Chelladurai1"></a>Chelladurai BS, Li H, Nicholson AW (1991) A conserved sequence element in ribonuclease III processing signals is not required for accurate in vitro enzymatic cleavage. Nucleic Acids Res  19: 1759–1766.  <ul class="find" data-citedArticleID="990738" data-doi="10.1093/nar/19.8.1759"><li><a href="http://dx.doi.org/10.1093/nar/19.8.1759" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+conserved+sequence+element+in+ribonuclease+III+processing+signals+is+not+required+for+accurate+in+vitro+enzymatic+cleavage." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+conserved+sequence+element+in+ribonuclease+III+processing+signals+is+not+required+for+accurate+in+vitro+enzymatic+cleavage.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">44.
              </span><a name="pone.0000878-Deutscher1" id="pone.0000878-Deutscher1"></a>Deutscher MP (2006) Degradation of RNA in bacteria: comparison of mRNA and stable RNA. Nucleic Acids Res  34: 659–666.  <ul class="find" data-citedArticleID="990754" data-doi="10.1093/nar/gkj472"><li><a href="http://dx.doi.org/10.1093/nar/gkj472" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Degradation+of+RNA+in+bacteria%3A+comparison+of+mRNA+and+stable+RNA." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Degradation+of+RNA+in+bacteria%3A+comparison+of+mRNA+and+stable+RNA.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">45.
              </span><a name="pone.0000878-Meinken1" id="pone.0000878-Meinken1"></a>Meinken C, Blencke HM, Ludwig H, Stulke J (2003) Expression of the glycolytic gapA operon in Bacillus subtilis: differential syntheses of proteins encoded by the operon. Microbiology  149: 751–761.  <ul class="find" data-citedArticleID="990798" data-doi="10.1099/mic.0.26078-0"><li><a href="http://dx.doi.org/10.1099/mic.0.26078-0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Expression+of+the+glycolytic+gapA+operon+in+Bacillus+subtilis%3A+differential+syntheses+of+proteins+encoded+by+the+operon." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Expression+of+the+glycolytic+gapA+operon+in+Bacillus+subtilis%3A+differential+syntheses+of+proteins+encoded+by+the+operon.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">46.
              </span><a name="pone.0000878-Mader1" id="pone.0000878-Mader1"></a>Mader U, Hennig S, Hecker M, Homuth G (2004) Transcriptional organization and posttranscriptional regulation of the Bacillus subtilis branched-chain amino acid biosynthesis genes. J Bacteriol  186: 2240–2252.  <ul class="find" data-citedArticleID="990796" data-doi="10.1128/jb.186.8.2240-2252.2004"><li><a href="http://dx.doi.org/10.1128/jb.186.8.2240-2252.2004" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Transcriptional+organization+and+posttranscriptional+regulation+of+the+Bacillus+subtilis+branched-chain+amino+acid+biosynthesis+genes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Transcriptional+organization+and+posttranscriptional+regulation+of+the+Bacillus+subtilis+branched-chain+amino+acid+biosynthesis+genes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">47.
              </span><a name="pone.0000878-deHoon1" id="pone.0000878-deHoon1"></a>de Hoon MJ, Makita Y, Nakai K, Miyano S (2005) Prediction of transcriptional terminators in Bacillus subtilis and related species. PLoS Comput Biol  1: e25.  <ul class="find" data-citedArticleID="990846" data-doi="10.1371/journal.pcbi.0010025.eor"><li><a href="http://dx.doi.org/10.1371/journal.pcbi.0010025.eor" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Prediction+of+transcriptional+terminators+in+Bacillus+subtilis+and+related+species." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Prediction+of+transcriptional+terminators+in+Bacillus+subtilis+and+related+species.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">48.
              </span><a name="pone.0000878-Kushner1" id="pone.0000878-Kushner1"></a>Kushner SR (2004) mRNA decay in prokaryotes and eukaryotes: different approaches to a similar problem. IUBMB Life  56: 585–594.  <ul class="find" data-citedArticleID="990790" data-doi="10.1080/15216540400022441"><li><a href="http://dx.doi.org/10.1080/15216540400022441" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=mRNA+decay+in+prokaryotes+and+eukaryotes%3A+different+approaches+to+a+similar+problem." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22mRNA+decay+in+prokaryotes+and+eukaryotes%3A+different+approaches+to+a+similar+problem.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">49.
              </span><a name="pone.0000878-Washio1" id="pone.0000878-Washio1"></a>Washio T, Sasayama J, Tomita M (1998) Analysis of complete genomes suggests that many prokaryotes do not rely on hairpin formation in transcription termination. Nucleic Acids Res  26: 5456–5463.  <ul class="find" data-citedArticleID="990838" data-doi="10.1093/nar/26.23.5456"><li><a href="http://dx.doi.org/10.1093/nar/26.23.5456" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Analysis+of+complete+genomes+suggests+that+many+prokaryotes+do+not+rely+on+hairpin+formation+in+transcription+termination." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Analysis+of+complete+genomes+suggests+that+many+prokaryotes+do+not+rely+on+hairpin+formation+in+transcription+termination.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">50.
              </span><a name="pone.0000878-Hessel1" id="pone.0000878-Hessel1"></a>Hessel T, Dhital SP, Plank R, Dean D (2001) Immune response to chlamydial 60-kilodalton heat shock protein in tears from Nepali trachoma patients. Infect Immun  69: 4996–5000.  <ul class="find" data-citedArticleID="990778" data-doi="10.1128/iai.69.8.4996-5000.2001"><li><a href="http://dx.doi.org/10.1128/iai.69.8.4996-5000.2001" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Immune+response+to+chlamydial+60-kilodalton+heat+shock+protein+in+tears+from+Nepali+trachoma+patients." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Immune+response+to+chlamydial+60-kilodalton+heat+shock+protein+in+tears+from+Nepali+trachoma+patients.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">51.
              </span><a name="pone.0000878-Miyairi1" id="pone.0000878-Miyairi1"></a>Miyairi I, Mahdi OS, Ouellette SP, Belland RJ, Byrne GI (2006) Different Growth Rates of Chlamydia trachomatis Biovars Reflect Pathotype. J Infect Dis  194: 350–357.  <ul class="find" data-citedArticleID="990802" data-doi="10.1086/505432"><li><a href="http://dx.doi.org/10.1086/505432" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Different+Growth+Rates+of+Chlamydia+trachomatis+Biovars+Reflect+Pathotype." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Different+Growth+Rates+of+Chlamydia+trachomatis+Biovars+Reflect+Pathotype.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">52.
              </span><a name="pone.0000878-Kiselev2" id="pone.0000878-Kiselev2"></a>Kiselev AO, Johnson ML, Ballweber LB, Stamm WE, Lampe MF (2002)  Surface location of PmpD, a polymorphic membrane protein of <em>Chlamydia trachomatis</em> serovar L2. In: Schachter J, Christiansen G, Clarke IN, Hammerschlag MR, Kaltenboeck B, et al., editors. San Francisco: International Chlamydia Symposium. pp. 567–570.  <ul class="find-nolinks"></ul></li><li><span class="label">53.
              </span><a name="pone.0000878-Vandahl1" id="pone.0000878-Vandahl1"></a>Vandahl BB, Pedersen AS, Gevaert K, Holm A, Vandekerckhove J, et al.  (2002) The expression, processing and localization of polymorphic membrane proteins in Chlamydia pneumoniae strain CWL029. BMC Microbiol  2: 36.  <ul class="find" data-citedArticleID="990834"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=The+expression%2C+processing+and+localization+of+polymorphic+membrane+proteins+in+Chlamydia+pneumoniae+strain+CWL029.&amp;auth=&amp;atitle=The+expression%2C+processing+and+localization+of+polymorphic+membrane+proteins+in+Chlamydia+pneumoniae+strain+CWL029." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+expression%2C+processing+and+localization+of+polymorphic+membrane+proteins+in+Chlamydia+pneumoniae+strain+CWL029." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+expression%2C+processing+and+localization+of+polymorphic+membrane+proteins+in+Chlamydia+pneumoniae+strain+CWL029.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">54.
              </span><a name="pone.0000878-Grimwood1" id="pone.0000878-Grimwood1"></a>Grimwood J, Stephens RS (1999) Computational analysis of the polymorphic membrane protein superfamily of Chlamydia trachomatis and Chlamydia pneumoniae. Microb Comp Genomics  4: 187–201.  <ul class="find" data-citedArticleID="990772" data-doi="10.1089/omi.1.1999.4.187"><li><a href="http://dx.doi.org/10.1089/omi.1.1999.4.187" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Computational+analysis+of+the+polymorphic+membrane+protein+superfamily+of+Chlamydia+trachomatis+and+Chlamydia+pneumoniae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Computational+analysis+of+the+polymorphic+membrane+protein+superfamily+of+Chlamydia+trachomatis+and+Chlamydia+pneumoniae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">55.
              </span><a name="pone.0000878-Kennell1" id="pone.0000878-Kennell1"></a>Kennell D (2002) Processing endoribonucleases and mRNA degradation in bacteria. J Bacteriol  184: 4645–4657; discussion 4665.  <ul class="find" data-citedArticleID="990784" data-doi="10.1128/jb.184.17.4645-4657.2002"><li><a href="http://dx.doi.org/10.1128/jb.184.17.4645-4657.2002" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Processing+endoribonucleases+and+mRNA+degradation+in+bacteria." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Processing+endoribonucleases+and+mRNA+degradation+in+bacteria.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">56.
              </span><a name="pone.0000878-Mian1" id="pone.0000878-Mian1"></a>Mian IS (1997) Comparative sequence analysis of ribonucleases HII, III, II PH and D. Nucleic Acids Res  25: 3187–3195.  <ul class="find" data-citedArticleID="990800" data-doi="10.1093/nar/25.16.3187"><li><a href="http://dx.doi.org/10.1093/nar/25.16.3187" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Comparative+sequence+analysis+of+ribonucleases+HII%2C+III%2C+II+PH+and+D." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Comparative+sequence+analysis+of+ribonucleases+HII%2C+III%2C+II+PH+and+D.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">57.
              </span><a name="pone.0000878-Huntzinger1" id="pone.0000878-Huntzinger1"></a>Huntzinger E, Boisset S, Saveanu C, Benito Y, Geissmann T, et al.  (2005) Staphylococcus aureus RNAIII and the endoribonuclease III coordinately regulate spa gene expression. Embo J  24: 824–835.  <ul class="find" data-citedArticleID="990782" data-doi="10.1038/sj.emboj.7600572"><li><a href="http://dx.doi.org/10.1038/sj.emboj.7600572" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Staphylococcus+aureus+RNAIII+and+the+endoribonuclease+III+coordinately+regulate+spa+gene+expression." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Staphylococcus+aureus+RNAIII+and+the+endoribonuclease+III+coordinately+regulate+spa+gene+expression.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">58.
              </span><a name="pone.0000878-Afonyushkin1" id="pone.0000878-Afonyushkin1"></a>Afonyushkin T, Vecerek B, Moll I, Blasi U, Kaberdin VR (2005) Both RNase E and RNase III control the stability of sodB mRNA upon translational inhibition by the small regulatory RNA RyhB. Nucleic Acids Res  33: 1678–1689.  <ul class="find" data-citedArticleID="990724" data-doi="10.1093/nar/gki313"><li><a href="http://dx.doi.org/10.1093/nar/gki313" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Both+RNase+E+and+RNase+III+control+the+stability+of+sodB+mRNA+upon+translational+inhibition+by+the+small+regulatory+RNA+RyhB." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Both+RNase+E+and+RNase+III+control+the+stability+of+sodB+mRNA+upon+translational+inhibition+by+the+small+regulatory+RNA+RyhB.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">59.
              </span><a name="pone.0000878-SaNogueira1" id="pone.0000878-SaNogueira1"></a>Sa-Nogueira I, Nogueira TV, Soares S, de Lencastre H (1997) The Bacillus subtilis L-arabinose (ara) operon: nucleotide sequence, genetic organization and expression. Microbiology  143 ( Pt 3): 957–969.  <ul class="find" data-citedArticleID="990812" data-doi="10.1099/00221287-143-3-957"><li><a href="http://dx.doi.org/10.1099/00221287-143-3-957" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+Bacillus+subtilis+L-arabinose+%28ara%29+operon%3A+nucleotide+sequence%2C+genetic+organization+and+expression." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+Bacillus+subtilis+L-arabinose+%28ara%29+operon%3A+nucleotide+sequence%2C+genetic+organization+and+expression.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">60.
              </span><a name="pone.0000878-Wei1" id="pone.0000878-Wei1"></a>Wei X, Sayavedra-Soto LA, Arp DJ (2004) The transcription of the cbb operon in Nitrosomonas europaea. Microbiology  150: 1869–1879.  <ul class="find" data-citedArticleID="990842" data-doi="10.1099/mic.0.26785-0"><li><a href="http://dx.doi.org/10.1099/mic.0.26785-0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+transcription+of+the+cbb+operon+in+Nitrosomonas+europaea." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+transcription+of+the+cbb+operon+in+Nitrosomonas+europaea.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">61.
              </span><a name="pone.0000878-Liang1" id="pone.0000878-Liang1"></a>Liang FT, Caimano MJ, Radolf JD, Fikrig E (2004) Borrelia burgdorferi outer surface protein (osp) B expression independent of ospA. Microb Pathog  37: 35–40.  <ul class="find" data-citedArticleID="990792" data-doi="10.1016/j.micpath.2004.02.007"><li><a href="http://dx.doi.org/10.1016/j.micpath.2004.02.007" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Borrelia+burgdorferi+outer+surface+protein+%28osp%29+B+expression+independent+of+ospA." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Borrelia+burgdorferi+outer+surface+protein+%28osp%29+B+expression+independent+of+ospA.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">62.
              </span><a name="pone.0000878-Ramamoorthy1" id="pone.0000878-Ramamoorthy1"></a>Ramamoorthy R, McClain NA, Gautam A, Scholl-Meeker D (2005) Expression of the bmpB gene of Borrelia burgdorferi is modulated by two distinct transcription termination events. J Bacteriol  187: 2592–2600.  <ul class="find" data-citedArticleID="990808" data-doi="10.1128/jb.187.8.2592-2600.2005"><li><a href="http://dx.doi.org/10.1128/jb.187.8.2592-2600.2005" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Expression+of+the+bmpB+gene+of+Borrelia+burgdorferi+is+modulated+by+two+distinct+transcription+termination+events." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Expression+of+the+bmpB+gene+of+Borrelia+burgdorferi+is+modulated+by+two+distinct+transcription+termination+events.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">63.
              </span><a name="pone.0000878-Yajnik1" id="pone.0000878-Yajnik1"></a>Yajnik V, Godson GN (1993) Selective decay of Escherichia coli dnaG messenger RNA is initiated by RNase E. J Biol Chem  268: 13253–13260.  <ul class="find" data-citedArticleID="990844"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Selective+decay+of+Escherichia+coli+dnaG+messenger+RNA+is+initiated+by+RNase+E.&amp;auth=&amp;atitle=Selective+decay+of+Escherichia+coli+dnaG+messenger+RNA+is+initiated+by+RNase+E." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Selective+decay+of+Escherichia+coli+dnaG+messenger+RNA+is+initiated+by+RNase+E." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Selective+decay+of+Escherichia+coli+dnaG+messenger+RNA+is+initiated+by+RNase+E.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000878.XML" value="132274"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000878.PDF" value="467522"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000878.t001.PNG_L" value="267345"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000878.t001.PNG_M" value="163636"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000878.t001.PNG_S" value="14911"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000878.t001.TIF" value="1217392"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000878.t001.PNG_I" value="51294"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000878.g001.PNG_L" value="495284"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000878.g001.PNG_M" value="55966"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000878.g001.PNG_S" value="8092"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000878.g001.TIF" value="865596"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000878.g001.PNG_I" value="19021"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000878.g002.PNG_L" value="585674"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000878.g002.PNG_M" value="58183"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000878.g002.PNG_S" value="9236"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000878.g002.TIF" value="1265798"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000878.g002.PNG_I" value="24525"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000878.g003.PNG_L" value="74363"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000878.g003.PNG_M" value="67483"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000878.g003.PNG_S" value="7444"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000878.g003.TIF" value="505962"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000878.g003.PNG_I" value="24869"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000878.g004.PNG_L" value="27752"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000878.g004.PNG_M" value="23251"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000878.g004.PNG_S" value="7389"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000878.g004.TIF" value="165034"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000878.g004.PNG_I" value="8643"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000878.g005.PNG_L" value="263590"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000878.g005.PNG_M" value="18123"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000878.g005.PNG_S" value="4250"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000878.g005.TIF" value="267444"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000878.g005.PNG_I" value="6991"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000878.t002.PNG_L" value="156336"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000878.t002.PNG_M" value="86524"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000878.t002.PNG_S" value="10246"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000878.t002.TIF" value="677484"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000878.t002.PNG_I" value="27253"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000878&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000878" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000878&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0000878&volume=&issue=&title=Comparative Expression Profiling of the Chlamydia trachomatis pmp Gene Family for Clinical and Reference Strains&author_name=Alexandra%20Nunes%2C%20Jo%C3%A3o%20P.%20Gomes%2C%20Sally%20Mead%2C%20Carlos%20Florindo%2C%20Helena%20Correia%2C%20Maria%20J.%20Borrego%2C%20Deborah%20Dean&start_page=1&end_page=10" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000878" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000878" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0000878" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000878&amp;t=Comparative%20Expression%20Profiling%20of%20the%20Chlamydia%20trachomatis%20pmp%20Gene%20Family%20for%20Clinical%20and%20Reference%20Strains" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000878&title=Comparative%20Expression%20Profiling%20of%20the%20Chlamydia%20trachomatis%20pmp%20Gene%20Family%20for%20Clinical%20and%20Reference%20Strains&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000878&amp;title=Comparative%20Expression%20Profiling%20of%20the%20Chlamydia%20trachomatis%20pmp%20Gene%20Family%20for%20Clinical%20and%20Reference%20Strains" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000878" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0000878&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Comparative Expression Profiling of the Chlamydia trachomatis pmp Gene Family for Clinical and Reference Strains';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0000878';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Comparative%20Expression%20Profiling%20of%20the%20Chlamydia%20trachomatis%20pmp%20Gene%20Family%20for%20Clinical%20and%20Reference%20Strains http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000878" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0000878" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">













          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Adolescents%22" title="Search for articles in the subject area:'Adolescents'"><div class="flagText">Adolescents</div></a>
              <div data-categoryid="41133" data-articleid="24594"
                   data-categoryname="Adolescents"
                   class="flagImage" title="Flag 'Adolescents' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Chlamydia%22" title="Search for articles in the subject area:'Chlamydia'"><div class="flagText">Chlamydia</div></a>
              <div data-categoryid="43359" data-articleid="24594"
                   data-categoryname="Chlamydia"
                   class="flagImage" title="Flag 'Chlamydia' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Chlamydia+trachomatis%22" title="Search for articles in the subject area:'Chlamydia trachomatis'"><div class="flagText">Chlamydia trachomatis</div></a>
              <div data-categoryid="42921" data-articleid="24594"
                   data-categoryname="Chlamydia trachomatis"
                   class="flagImage" title="Flag 'Chlamydia trachomatis' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Gene+expression%22" title="Search for articles in the subject area:'Gene expression'"><div class="flagText">Gene expression</div></a>
              <div data-categoryid="17515" data-articleid="24594"
                   data-categoryname="Gene expression"
                   class="flagImage" title="Flag 'Gene expression' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Messenger+RNA%22" title="Search for articles in the subject area:'Messenger RNA'"><div class="flagText">Messenger RNA</div></a>
              <div data-categoryid="18353" data-articleid="24594"
                   data-categoryname="Messenger RNA"
                   class="flagImage" title="Flag 'Messenger RNA' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Operons%22" title="Search for articles in the subject area:'Operons'"><div class="flagText">Operons</div></a>
              <div data-categoryid="21291" data-articleid="24594"
                   data-categoryname="Operons"
                   class="flagImage" title="Flag 'Operons' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Outer+membrane+proteins%22" title="Search for articles in the subject area:'Outer membrane proteins'"><div class="flagText">Outer membrane proteins</div></a>
              <div data-categoryid="20669" data-articleid="24594"
                   data-categoryname="Outer membrane proteins"
                   class="flagImage" title="Flag 'Outer membrane proteins' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Ribonucleases%22" title="Search for articles in the subject area:'Ribonucleases'"><div class="flagText">Ribonucleases</div></a>
              <div data-categoryid="46199" data-articleid="24594"
                   data-categoryname="Ribonucleases"
                   class="flagImage" title="Flag 'Ribonucleases' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=4272'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=5681'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=6168&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>

<div class="block sidebar-comments">
    <div class="header">
        <h3>Comments</h3>
    </div>
      <p><a href="/annotation/listThread.action?root=20737">This article evaluated on F1000Medicine</a><br>Posted by F1000Medicine</p>
      <p><a href="/annotation/listThread.action?root=43301">Academic Editor's Rating</a><br>Posted by NiyazAhmed</p>
      <p><a href="/annotation/listThread.action?root=14213">Academic Editor's comments</a><br>Posted by NiyazAhmed</p>
</div>

</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
