

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Transcriptome of Aphanomyces euteiches: New Oomycete Putative Pathogenicity Factors and Metabolic Pathways</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0001723"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0001723" />

    <meta name="citation_title" content="Transcriptome of Aphanomyces euteiches: New Oomycete Putative Pathogenicity Factors and Metabolic Pathways"/>
    <meta itemprop="name" content="Transcriptome of Aphanomyces euteiches: New Oomycete Putative Pathogenicity Factors and Metabolic Pathways"/>

      <meta name="citation_author" content="Elodie Gaulin"/>
            <meta name="citation_author_institution" content="UMR 5546 Centre National de la Recherche Scientifique (CNRS), Université Paul Sabatier Toulouse III, Université de Toulouse, Pôle de Biotechnologie Végétale, Castanet-Tolosan, France"/>
      <meta name="citation_author" content="Mohammed-Amine Madoui"/>
            <meta name="citation_author_institution" content="UMR 5546 Centre National de la Recherche Scientifique (CNRS), Université Paul Sabatier Toulouse III, Université de Toulouse, Pôle de Biotechnologie Végétale, Castanet-Tolosan, France"/>
      <meta name="citation_author" content="Arnaud Bottin"/>
            <meta name="citation_author_institution" content="UMR 5546 Centre National de la Recherche Scientifique (CNRS), Université Paul Sabatier Toulouse III, Université de Toulouse, Pôle de Biotechnologie Végétale, Castanet-Tolosan, France"/>
      <meta name="citation_author" content="Christophe Jacquet"/>
            <meta name="citation_author_institution" content="UMR 5546 Centre National de la Recherche Scientifique (CNRS), Université Paul Sabatier Toulouse III, Université de Toulouse, Pôle de Biotechnologie Végétale, Castanet-Tolosan, France"/>
      <meta name="citation_author" content="Catherine Mathé"/>
            <meta name="citation_author_institution" content="UMR 5546 Centre National de la Recherche Scientifique (CNRS), Université Paul Sabatier Toulouse III, Université de Toulouse, Pôle de Biotechnologie Végétale, Castanet-Tolosan, France"/>
      <meta name="citation_author" content="Arnaud Couloux"/>
            <meta name="citation_author_institution" content="Genoscope (CEA), Evry, France"/>
            <meta name="citation_author_institution" content="UMR 8030 Centre National de la Recherche Scientifique (CNRS), Evry, France"/>
            <meta name="citation_author_institution" content="Université d'Evry, Evry, France"/>
      <meta name="citation_author" content="Patrick Wincker"/>
            <meta name="citation_author_institution" content="Genoscope (CEA), Evry, France"/>
            <meta name="citation_author_institution" content="UMR 8030 Centre National de la Recherche Scientifique (CNRS), Evry, France"/>
            <meta name="citation_author_institution" content="Université d'Evry, Evry, France"/>
      <meta name="citation_author" content="Bernard Dumas"/>
            <meta name="citation_author_institution" content="UMR 5546 Centre National de la Recherche Scientifique (CNRS), Université Paul Sabatier Toulouse III, Université de Toulouse, Pôle de Biotechnologie Végétale, Castanet-Tolosan, France"/>

    <meta name="citation_date" content="2008/3/5"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0001723.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e1723"/>
    <meta name="citation_issue" content="3"/>
    <meta name="citation_volume" content="3"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Characterization of Aphanomyces euteiches isolates recovered from alfalfa in Wisconsin.; citation_author=PA Delwich; citation_author=CR Grau; citation_author=EB Holub; citation_author=JB Perry; citation_journal_title=Plant Dis; citation_volume=71; citation_number=1; citation_pages=155-161; citation_date=1987; " />
      <meta name="citation_reference" content="citation_title=Pathogenicity of Aphanomyces spp. from different leguminous crops in Sweden.; citation_author=JP Levenfors; citation_author=M Wikstrom; citation_author=L Persson; citation_author=B Gerhardson; citation_journal_title=Eur J Plant Pathol; citation_volume=109; citation_number=2; citation_pages=535-543; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Specific behaviour of French Aphanomyces euteiches Drechs. populations for virulence and aggressiveness on pea, related to isolates from Europe, America and New Zealand.; citation_author=E Wicker; citation_author=F Rouxel; citation_journal_title=Eur J Plant Pathol; citation_volume=107; citation_number=3; citation_pages=919-929; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Factors affecting growth and development of Aphanomyces euteiches.; citation_author=JE Mitchell; citation_author=CY Yang; citation_journal_title=Phytopathology; citation_volume=56; citation_number=4; citation_pages=917-922; citation_date=1966; " />
      <meta name="citation_reference" content="citation_title=Pathogenicity, morphology and isozyme variability among isolates of Aphanomyces spp. from weeds and various plants.; citation_author=M Larsson; citation_journal_title=Mycol Res; citation_volume=98; citation_number=5; citation_pages=231-240; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=Consistent quantitative trait loci in pea for partial resistance to Aphanomyces euteiches isolates from the United States and France.; citation_author=ML Pilet-Nayel; citation_author=FJ Muehlbauer; citation_author=RJ McGee; citation_author=JM Kraft; citation_author=A Baranger; citation_journal_title=Phytopathology; citation_volume=95; citation_number=6; citation_pages=1287-1293; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=The crayfish plague fungus: history and recent advances.; citation_author=K Söderhäll; citation_author=L Cerenius; citation_journal_title=Freshwater Crayfish; citation_volume=12; citation_number=7; citation_pages=11-35; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Aphanomyces invadans in Atlantic menhaden along the east coast of the United States.; citation_author=VS Blazer; citation_author=JH Lilley; citation_author=WB Schill; citation_author=Y Kiryu; citation_author=C Densmore; citation_journal_title=J Aquat Anim Health; citation_volume=14; citation_number=8; citation_pages=1-10; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Root rot disease of legumes caused by Aphanomyces euteiches.; citation_author=E Gaulin; citation_author=C Jacquet; citation_author=A Bottin; citation_author=B Dumas; citation_journal_title=Mol Plant Pathol; citation_volume=8; citation_number=9; citation_pages=539-548; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Management of soilborne diseases of beetroot in Australia: a review.; citation_author=HL Martin; citation_journal_title=Aust J Exp Agri; citation_volume=43; citation_number=10; citation_pages=1281-1292; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Nonhost resistance to Phytophthora: novel prospects for a classical problem.; citation_author=S Kamoun; citation_journal_title=Curr Opin Plant Biol; citation_volume=4; citation_number=11; citation_pages=295-300; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Large-scale gene discovery in the oomycete Phytophthora infestans reveals likely components of phytopathogenicity shared with true fungi.; citation_author=TA Randall; citation_author=RA Dwyer; citation_author=E Huitema; citation_author=K Beyer; citation_author=C Cvitanich; citation_journal_title=Mol Plant Microbe Interact; citation_volume=18; citation_number=12; citation_pages=229-243; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Phytophthora functional genomics database (PFGD): functional genomics of Phytophthora-plant interactions.; citation_author=K Gajendran; citation_author=MD Gonzales; citation_author=A Farmer; citation_author=E Archuleta; citation_author=J Win; citation_journal_title=Nucleic Acids Res; citation_volume=34; citation_number=13; citation_pages=465-470; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=The Phytophthora genome initiative database: informatics and analysis for distributed pathogenomic research.; citation_author=M Waugh; citation_author=P Hraber; citation_author=J Weller; citation_author=Y Wu; citation_author=G Chen; citation_journal_title=Nucleic Acids Res; citation_volume=28; citation_number=14; citation_pages=87-90; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Phytophthora genome sequences uncover evolutionary origins and mechanisms of pathogenesis.; citation_author=BM Tyler; citation_author=S Tripathy; citation_author=X Zhang; citation_author=P Dehal; citation_author=RH Jiang; citation_journal_title=Science; citation_volume=313; citation_number=15; citation_pages=1261-1266; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Expressed sequence tags from Phytophthora sojae reveal genes specific to development and infection.; citation_author=TA Torto-Alalibo; citation_author=S Tripathy; citation_author=BM Smith; citation_author=FD Arredondo; citation_author=L Zhou; citation_journal_title=Mol Plant Microbe Interact; citation_volume=20; citation_number=16; citation_pages=781-793; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Gene identification in the oomycete pathogen Phytophthora parasitica during in vitro vegetative growth through expressed sequence tags.; citation_author=F Panabières; citation_author=J Amselem; citation_author=E Galiana; citation_author=J Le Berre; citation_journal_title=Fungal Genet Biol; citation_volume=42; citation_number=17; citation_pages=611-623; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=A translocation signal for delivery of oomycete effector proteins into host plant cells.; citation_author=SC Whisson; citation_author=PC Boevink; citation_author=L Moleleki; citation_author=AO Avrova; citation_author=JG Morales; citation_journal_title=Nature; citation_volume=450; citation_number=18; citation_pages=115-118; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Some factors affecting growth of Aphanomyces euteiches in synthetic media.; citation_author=G Papavizas; citation_author=C Davey; citation_journal_title=Amer J Bot; citation_volume=47; citation_number=19; citation_pages=758-765; citation_date=1960; " />
      <meta name="citation_reference" content="citation_title=AphanoDB: a genomic resource for Aphanomyces pathogens.; citation_author=MA Madoui; citation_author=E Gaulin; citation_author=C Mathé; citation_author=H San Clemente; citation_author=A Couloux; citation_journal_title=BMC Genomics; citation_volume=8; citation_number=20; citation_pages=471; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Improved prediction of signal peptides: SignalP 3.0.; citation_author=JD Bendtsen; citation_author=H Nielsen; citation_author=G von Heijne; citation_author=S Brunak; citation_journal_title=J Mol Biol; citation_volume=340; citation_number=21; citation_pages=783-795; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=EMBOSS: the European Molecular Biology Open Software Suite.; citation_author=P Rice; citation_author=I Longden; citation_author=A Bleasby; citation_journal_title=Trends Genet; citation_volume=16; citation_number=22; citation_pages=276-277; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Profile hidden Markov models.; citation_author=SR Eddy; citation_journal_title=Bioinformatics; citation_volume=14; citation_number=23; citation_pages=755-763; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Adaptive evolution has targeted the C-terminal domain of the RXLR effectors of plant pathogenic oomycetes.; citation_author=J Win; citation_author=W Morgan; citation_author=J Bos; citation_author=KV Krasileva; citation_author=LM Cano; citation_journal_title=Plant Cell; citation_volume=19; citation_number=24; citation_pages=2349-2369; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=A host-targeting signal in virulence proteins reveals a secretome in malarial infection.; citation_author=NL Hiller; citation_author=S Bhattacharjee; citation_author=C van Ooij; citation_author=K Liolios; citation_author=T Harrison; citation_journal_title=Science; citation_volume=306; citation_number=25; citation_pages=1934-1937; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=CLUSTAL W: improving the sensitivity of progressive multiple sequence alignment through sequence weighting, position-specific gap penalties and weight matrix choice.; citation_author=JD Thompson; citation_author=DG Higgins; citation_author=TJ Gibson; citation_journal_title=Nucleic Acids Res; citation_volume=22; citation_number=26; citation_pages=4673-4680; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=WebLogo: A sequence logo generator.; citation_author=GE Crooks; citation_author=G Hon; citation_author=JM Chandonia; citation_author=SE Brenner; citation_journal_title=Genome Res; citation_volume=14; citation_number=27; citation_pages=1188-1190; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Gapped BLAST and PSI-BLAST: a new generation of protein database search programs.; citation_author=SF Altschul; citation_author=TL Madden; citation_author=AA Schaffer; citation_author=J Zhang; citation_author=Z Zhang; citation_journal_title=Nucleic Acids Res; citation_volume=25; citation_number=28; citation_pages=3389-3402; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=The Pfam protein families database.; citation_author=A Bateman; citation_author=L Coin; citation_author=R Durbin; citation_author=RD Finn; citation_author=V Hollich; citation_journal_title=Nucleic Acids Res; citation_volume=32; citation_number=29; citation_pages=D138-141; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Estimating and comparing the rates of gene discovery and expressed sequence tag (EST) frequencies in EST surveys.; citation_author=E Susko; citation_author=AJ Roger; citation_journal_title=Bioinformatics; citation_volume=20; citation_number=30; citation_pages=2279-2287; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Controlling the false discovery rate: a practical and powerful approach to multiple testing.; citation_author=Y Benjamini; citation_author=Y Hochberg; citation_journal_title=J R Stat Soc Ser B; citation_volume=57; citation_number=31; citation_pages=289-300; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Plant colonization by the vascular wilt fungus Fusarium oxysporum requires FOW1, a gene encoding a mitochondrial protein.; citation_author=I Inoue; citation_author=F Namiki; citation_author=T Tsuge; citation_journal_title=Plant Cell; citation_volume=14; citation_number=32; citation_pages=1869-1883; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Purification, elicitor activity, and cell wall localization of a glycoprotein from Phytophthora parasitica var. nicotianae, a fungal pathogen of tobacco.; citation_author=N Séjalon-Delmas; citation_author=F Villalba Mateos; citation_author=A Bottin; citation_author=M Rickauer; citation_author=R Dargent; citation_journal_title=Phytopathology; citation_volume=87; citation_number=33; citation_pages=899-909; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Cloning and characterization of a cDNA encoding an elicitor of Phytophthora parasitica var. nicotianae that shows cellulose-binding and lectin-like activities.; citation_author=F Villalba-Mateos; citation_author=M Rickauer; citation_author=MT Esquerré-Tugayé; citation_journal_title=Mol Plant-Microbe Interact; citation_volume=10; citation_number=34; citation_pages=1045-1053; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Cellulose binding domains of a Phytophthora cell wall protein are novel pathogen-associated molecular patterns.; citation_author=E Gaulin; citation_author=N Dramé; citation_author=C Lafitte; citation_author=T Torto-Alalibo; citation_author=Y Martinez; citation_journal_title=Plant Cell; citation_volume=18; citation_number=35; citation_pages=1766-1777; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=The PAN module: the N-terminal domains of plasminogen and hepathocyte growth factor are homologous with the apple domains of the prekallikrein family and with a novel domain found in numerous nematode proteins.; citation_author=H Tordai; citation_author=L Banyai; citation_author=L Patthy; citation_journal_title=FEBS Lett; citation_volume=461; citation_number=36; citation_pages=63-67; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Cyst germination proteins of the potato pathogen Phytophthora infestans share homology with human mucins.; citation_author=B Gornhardt; citation_author=I Rouhara; citation_author=E Schmelzer; citation_journal_title=Mol Plant Microbe Interact; citation_volume=13; citation_number=37; citation_pages=32-42; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=The toxoplasma micronemal protein MIC4 is an adhesin composed of six conserved apple domains.; citation_author=S Brecht; citation_author=V Carruthers; citation_author=D Ferguson; citation_author=O Giddings; citation_author=G Wang; citation_journal_title=J Biol Chem; citation_volume=276; citation_number=38; citation_pages=4119-4127; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Structure of AMA1 from Plasmodium falciparum reveals a clustering of polymorphisms that surround a conserved hydrophobic pocket.; citation_author=T Bai; citation_author=M Becker; citation_author=A Gupta; citation_author=P Strike; citation_author=VJ Murphy; citation_journal_title=Proc Natl Acad Sci USA; citation_volume=102; citation_number=39; citation_pages=12736-12741; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=During attachment Phytophthora spores secrete proteins containing thrombospondin type 1 repeats.; citation_author=AV Robold; citation_author=AR Hardham; citation_journal_title=Curr Genet; citation_volume=47; citation_number=40; citation_pages=307-315; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Adhesion molecules and other secreted host-interaction determinants in Apicomplexa: insights from comparative genomics.; citation_author=V Anantharaman; citation_author=LM Iyer; citation_author=S Balaji; citation_author=L Aravind; citation_journal_title=Int Rev Cytol; citation_volume=262; citation_number=41; citation_pages=1-74; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Structural and sequence-based classification of glycoside hydrolases.; citation_author=B Henrissat; citation_author=G Davies; citation_journal_title=Curr Opin Struct Biol; citation_volume=7; citation_number=42; citation_pages=637-644; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Expressed sequence tags from the oomycete fish pathogen Saprolegnia parasitica reveal putative virulence factors.; citation_author=T Torto-Alalibo; citation_author=M Tian; citation_author=K Gajendran; citation_author=M Waugh; citation_author=P van West; citation_journal_title=BMC Microbiol; citation_volume=2; citation_number=43; citation_pages=5-46; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Characterization of 1,3-β-glucanase and 1,3;1,4-β-glucanase genes from Phytophthora infestans.; citation_author=A McLeod; citation_author=CD Smart; citation_author=WE Fry; citation_journal_title=Fungal Genet Biol; citation_volume=38; citation_number=44; citation_pages=250-263; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Gene duplication event in family 12 glycosyl hydrolase from Phytophthora spp.; citation_author=S Costanzo; citation_author=MD Ospina-Giraldo; citation_author=KL Deahl; citation_author=CJ Baker; citation_author=RW Jones; citation_journal_title=Fungal Genet Biol; citation_volume=43; citation_number=45; citation_pages=707-714; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Characterization and evolutionary analysis of a large polygalacturonase gene family in the oomycete plant pathogen Phytophthora cinnamomi.; citation_author=A Gotesson; citation_author=JS Marshall; citation_author=DA Jones; citation_author=AR Hardham; citation_journal_title=Mol Plant Microbe Interact; citation_volume=15; citation_number=46; citation_pages=907-921; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Cloning and analysis of pppg1, an inducible endopolygalacturonase gene from the oomycete plant pathogen Phytophthora parasitica.; citation_author=HZ Yan; citation_author=RF Liou; citation_journal_title=Fungal Genet Biol; citation_volume=42; citation_number=47; citation_pages=339-350; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=EST mining and functional expression assays identify extracellular effector proteins from the plant pathogen Phytophthora.; citation_author=TA Torto; citation_author=S Li; citation_author=A Styer; citation_author=E Huitema; citation_author=A Testa; citation_journal_title=Genome Res; citation_volume=13; citation_number=48; citation_pages=1675-1685; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=A Kazal-like extracellular serine protease inhibitor from Phytophthora infestans targets the tomato pathogenesis-related protease P69B.; citation_author=M Tian; citation_author=E Huitema; citation_author=L Da Cunha; citation_author=T Torto-Alalibo; citation_author=S Kamoun; citation_journal_title=J Biol Chem; citation_volume=279; citation_number=49; citation_pages=26370-26375; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=A Phytophthora infestans cystatin-like protein targets a novel tomato papain-like apoplastic protease.; citation_author=M Tian; citation_author=J Win; citation_author=J Song; citation_author=R van der Hoorn; citation_author=E van der Knaap; citation_journal_title=Plant Physiol; citation_volume=143; citation_number=50; citation_pages=364-377; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=NPP1, a Phytophthora-associated trigger of plant defense in parsley and Arabidopsis.; citation_author=G Fellbrich; citation_author=A Romanski; citation_author=A Varet; citation_author=B Blume; citation_author=F Brunner; citation_journal_title=Plant J; citation_volume=32; citation_number=51; citation_pages=375-390; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Expression of a Phytophthora sojae necrosis-inducing protein occurs during transition from biotrophy to necrotrophy.; citation_author=D Qutob; citation_author=S Kamoun; citation_author=M Gijzen; citation_journal_title=Plant J; citation_volume=32; citation_number=52; citation_pages=361-373; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Are elicitins cryptograms in plant-Oomycete communications?; citation_author=M Ponchet; citation_author=F Panabières; citation_author=ML Milat; citation_author=V Mikes; citation_author=JL Montillet; citation_journal_title=Cell Mol Life Sci; citation_volume=56; citation_number=53; citation_pages=1020-1047; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Ancient origin of elicitin gene clusters in Phytophthora genomes.; citation_author=RH Jiang; citation_author=BM Tyler; citation_author=SC Whisson; citation_author=AR Hardham; citation_author=F Govers; citation_journal_title=Mol Biol Evol; citation_volume=23; citation_number=54; citation_pages=338-351; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Peptide transport in plants.; citation_author=G Stacey; citation_author=S Koh; citation_author=C Granger; citation_author=JM Becker; citation_journal_title=Trends Plant Sci; citation_volume=7; citation_number=55; citation_pages=257-263; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Multiplicity and regulation of genes encoding peptide transporters in Saccharomyces cerevisiae.; citation_author=M Hauser; citation_author=V Narita; citation_author=AM Donhardt; citation_author=F Naider; citation_author=JM Becker; citation_journal_title=Mol Membr Biol; citation_volume=18; citation_number=56; citation_pages=105-112; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=A loss-of-function mutation in AtYSL1 reveals its role in iron and nicotianamine seed loading.; citation_author=M Le Jean; citation_author=A Schikora; citation_author=S Mari; citation_author=JF Briat; citation_author=C Curie; citation_journal_title=Plant J; citation_volume=44; citation_number=57; citation_pages=769-782; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Maize Yellow Stripe1, an iron-phytosiderophore uptake transporter, is a member of the oligopeptide transporter (OPT) family.; citation_author=MR Yen; citation_author=YH Tseng; citation_author=MH Saier; citation_journal_title=Microbiology; citation_volume=147; citation_number=58; citation_pages=2881-2883; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Trypanothione: a novel bis(glutathionyl)spermidine cofactor for glutathione reductase in trypanosomatids.; citation_author=AH Fairlamb; citation_author=P Blackburn; citation_author=P Ulrich; citation_author=BT Chait; citation_author=A Cerami; citation_journal_title=Science; citation_volume=227; citation_number=59; citation_pages=1485-1487; citation_date=1985; " />
      <meta name="citation_reference" content="citation_title=A single enzyme catalyses formation of trypanothione from glutathione and spermidine in Trypanosoma cruzi.; citation_author=SL Oza; citation_author=E Tetaud; citation_author=MR Ariyanayagam; citation_author=SS Warnon; citation_author=AH Fairlamb; citation_journal_title=J Biol Chem; citation_volume=277; citation_number=60; citation_pages=35853-35861; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Trypanothione synthesis in Crithidia revisited.; citation_author=M Comini; citation_author=U Menge; citation_author=J Wissing; citation_author=L Flohe; citation_journal_title=J Biol Chem; citation_volume=280; citation_number=61; citation_pages=6850-6860; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Glutathionylspermidine metabolism in Escherichia coli. Purification, cloning, overproduction, and characterization of a bifunctional glutathionylspermidine synthetase/amidase.; citation_author=JM Bollinger; citation_author=DS Kwon; citation_author=GW Huisman; citation_author=R Kolter; citation_author=CT Walsh; citation_journal_title=J Biol Chem; citation_volume=270; citation_number=62; citation_pages=14031-14041; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=The high resolution crystal structure of recombinant Crithidia fasciculata tryparedoxin-I.; citation_author=MS Alphey; citation_author=GA Leonard; citation_author=DG Gourley; citation_author=E Tetaud; citation_author=AH Fairlamb; citation_journal_title=J Biol Chem; citation_volume=274; citation_number=63; citation_pages=25613-25622; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Functional analysis of all nonribosomal peptide synthetases in Cochliobolus heterostrophus reveals a factor, NPS6, involved in virulence and resistance to oxidative stress.; citation_author=BN Lee; citation_author=S Kroken; citation_author=DY Chou; citation_author=B Robbertse; citation_author=OC Yoder; citation_journal_title=Eukaryot Cell; citation_volume=4; citation_number=64; citation_pages=545-555; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Cloning and characterization of a cyclic peptide synthetase gene from Alternaria alternata apple pathotype whose product is involved in AM-toxin synthesis and pathogenicity.; citation_author=RD Johnson; citation_author=L Johnson; citation_author=Y Itoh; citation_author=M Kodama; citation_author=H Otani; citation_journal_title=Mol Plant Microbe Interact; citation_volume=13; citation_number=65; citation_pages=742-753; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Exploration of the late stages of the tomato-Phytophthora parasitica interactions through histological analysis and generation of expressed sequence tags.; citation_author=JY Le Berre; citation_author=G Engler; citation_author=F Panabieres; citation_journal_title=New Phytol; citation_number=66; citation_pages=1-11; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Elicitins, proteinaceous elicitors of plant defense, are a new class of sterol carrier proteins.; citation_author=V Mikes; citation_author=ML Milat; citation_author=M Ponchet; citation_author=F Panabieres; citation_author=P Ricci; citation_journal_title=Biochem Biophys Res Commun; citation_volume=245; citation_number=67; citation_pages=133-139; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Cell biology of plant-oomycete interactions.; citation_author=AR Hardham; citation_journal_title=Cell Microbiol; citation_volume=9; citation_number=68; citation_pages=31-39; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=The CBEL glycoprotein of Phytophthora parasitica var-nicotianae is involved in cell wall deposition and adhesion to cellulosic substrates.; citation_author=E Gaulin; citation_author=A Jauneau; citation_author=F Villalba; citation_author=M Rickauer; citation_author=MT Esquerré-Tugayé; citation_journal_title=J Cell Sci; citation_volume=115; citation_number=69; citation_pages=4565-4575; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Investigating the role of ABC transporters in multifungicide insensitivity in Phytophthora infestans.; citation_author=HS Judelson; citation_author=G Senthil; citation_journal_title=Mol Plant Pathol; citation_volume=7; citation_number=70; citation_pages=17-29; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Development of codominant simple sequence repeat, single nucleotide polymorphism and sequence characterized amplified region markers for the pea root rot pathogen, Aphanomyces euteiches.; citation_author=HO Akamatsu; citation_author=NJ Grunwald; citation_author=MI Chilvers; citation_author=LD Porter; citation_author=TL Peever; citation_journal_title=J Microbiol Methods; citation_volume=71; citation_number=71; citation_pages=82-86; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Trypanosomes lacking trypanothione reductase are avirulent and show increased sensitivity to oxidative stress.; citation_author=S Krieger; citation_author=W Schwarz; citation_author=MR Ariyanayagam; citation_author=AH Fairlamb; citation_author=RL Krauth-Siegel; citation_journal_title=Mol Microbiol; citation_volume=35; citation_number=72; citation_pages=542-552; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Evidence that trypanothione reductase is an essential enzyme in Leishmania by targeted replacement of the tryA gene locus.; citation_author=J Tovar; citation_author=S Wilkinson; citation_author=JC Mottram; citation_author=AH Fairlamb; citation_journal_title=Mol Microbiol; citation_volume=29; citation_number=73; citation_pages=653-660; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Oxidative burst: an early plant response to pathogen infection.; citation_author=P Wojtaszek; citation_journal_title=Biochem J; citation_volume=322; citation_number=74; citation_pages=681-692; citation_date=1997; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001723" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Transcriptome of Aphanomyces euteiches: New Oomycete Putative Pathogenicity Factors and Metabolic Pathways"/>
    <meta name="twitter:description" content="Aphanomyces euteiches is an oomycete pathogen that causes seedling blight and root rot of legumes, such as alfalfa and pea. The genus Aphanomyces is phylogenically distinct from well-studied oomycetes such as Phytophthora sp., and contains species pathogenic on plants and aquatic animals. To provide the first foray into gene diversity of A. euteiches, two cDNA libraries were constructed using mRNA extracted from mycelium grown in an artificial liquid medium or in contact to plant roots. A unigene set of 7,977 sequences was obtained from 18,864 high-quality expressed sequenced tags (ESTs) and characterized for potential functions. Comparisons with oomycete proteomes revealed major differences between the gene content of A. euteiches and those of Phytophthora species, leading to the identification of biosynthetic pathways absent in Phytophthora, of new putative pathogenicity genes and of expansion of gene families encoding extracellular proteins, notably different classes of proteases. Among the genes specific of A. euteiches are members of a new family of extracellular proteins putatively involved in adhesion, containing up to four protein domains similar to fungal cellulose binding domains. Comparison of A. euteiches sequences with proteomes of fully sequenced eukaryotic pathogens, including fungi, apicomplexa and trypanosomatids, allowed the identification of A. euteiches genes with close orthologs in these microorganisms but absent in other oomycetes sequenced so far, notably transporters and non-ribosomal peptide synthetases, and suggests the presence of a defense mechanism against oxidative stress which was initially characterized in the pathogenic trypanosomatids."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0001723.g007"/>

  <meta property="og:title" content="Transcriptome of Aphanomyces euteiches: New Oomycete Putative Pathogenicity Factors and Metabolic Pathways" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001723" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=7353'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=7624'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=7861&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0001723">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001723" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001723&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Transcriptome of <i>Aphanomyces euteiches</i>: New Oomycete Putative Pathogenicity Factors and Metabolic Pathways
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Elodie Gaulin
              <span class="corresponding">mail</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:gaulin@scsv.ups-tlse.fr">gaulin@scsv.ups-tlse.fr</a> (EG); <a href="mailto:dumas@scsv.ups-tlse.fr">dumas@scsv.ups-tlse.fr</a> (BD)</p>

                <p>Affiliation:
                  UMR 5546 Centre National de la Recherche Scientifique (CNRS), Université Paul Sabatier Toulouse III, Université de Toulouse, Pôle de Biotechnologie Végétale, Castanet-Tolosan, France
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Mohammed-Amine Madoui, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  UMR 5546 Centre National de la Recherche Scientifique (CNRS), Université Paul Sabatier Toulouse III, Université de Toulouse, Pôle de Biotechnologie Végétale, Castanet-Tolosan, France
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Arnaud Bottin, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  UMR 5546 Centre National de la Recherche Scientifique (CNRS), Université Paul Sabatier Toulouse III, Université de Toulouse, Pôle de Biotechnologie Végétale, Castanet-Tolosan, France
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Christophe Jacquet, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  UMR 5546 Centre National de la Recherche Scientifique (CNRS), Université Paul Sabatier Toulouse III, Université de Toulouse, Pôle de Biotechnologie Végétale, Castanet-Tolosan, France
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Catherine Mathé, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  UMR 5546 Centre National de la Recherche Scientifique (CNRS), Université Paul Sabatier Toulouse III, Université de Toulouse, Pôle de Biotechnologie Végétale, Castanet-Tolosan, France
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Arnaud Couloux, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliations:
                  Genoscope (CEA), Evry, France, 
                  UMR 8030 Centre National de la Recherche Scientifique (CNRS), Evry, France, 
                  Université d'Evry, Evry, France
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Patrick Wincker, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliations:
                  Genoscope (CEA), Evry, France, 
                  UMR 8030 Centre National de la Recherche Scientifique (CNRS), Evry, France, 
                  Université d'Evry, Evry, France
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Bernard Dumas
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:gaulin@scsv.ups-tlse.fr">gaulin@scsv.ups-tlse.fr</a> (EG); <a href="mailto:dumas@scsv.ups-tlse.fr">dumas@scsv.ups-tlse.fr</a> (BD)</p>

                <p>Affiliation:
                  UMR 5546 Centre National de la Recherche Scientifique (CNRS), Université Paul Sabatier Toulouse III, Université de Toulouse, Pôle de Biotechnologie Végétale, Castanet-Tolosan, France
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: March 05, 2008</li>
    <li>DOI: 10.1371/journal.pone.0001723</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0001723" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0001723" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001723" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0001723" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0001723-g001" data-doi="info:doi/10.1371/journal.pone.0001723" data-uri="info:doi/10.1371/journal.pone.0001723.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001723.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001723-t001" data-doi="info:doi/10.1371/journal.pone.0001723" data-uri="info:doi/10.1371/journal.pone.0001723.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001723.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001723-t002" data-doi="info:doi/10.1371/journal.pone.0001723" data-uri="info:doi/10.1371/journal.pone.0001723.t002" title="Table 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001723.t002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001723-g002" data-doi="info:doi/10.1371/journal.pone.0001723" data-uri="info:doi/10.1371/journal.pone.0001723.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001723.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001723-g003" data-doi="info:doi/10.1371/journal.pone.0001723" data-uri="info:doi/10.1371/journal.pone.0001723.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001723.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001723-g004" data-doi="info:doi/10.1371/journal.pone.0001723" data-uri="info:doi/10.1371/journal.pone.0001723.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001723.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001723-g005" data-doi="info:doi/10.1371/journal.pone.0001723" data-uri="info:doi/10.1371/journal.pone.0001723.g005" title="Figure 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001723.g005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001723-g006" data-doi="info:doi/10.1371/journal.pone.0001723" data-uri="info:doi/10.1371/journal.pone.0001723.g006" title="Figure 6">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001723.g006&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001723-g007" data-doi="info:doi/10.1371/journal.pone.0001723" data-uri="info:doi/10.1371/journal.pone.0001723.g007" title="Figure 7">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001723.g007&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001723-t003" data-doi="info:doi/10.1371/journal.pone.0001723" data-uri="info:doi/10.1371/journal.pone.0001723.t003" title="Table 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001723.t003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001723">Reader Comments (0)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0001723" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1"></a><p><em>Aphanomyces euteiches</em> is an oomycete pathogen that causes seedling blight and root rot of legumes, such as alfalfa and pea. The genus <em>Aphanomyces</em> is phylogenically distinct from well-studied oomycetes such as <em>Phytophthora</em> sp., and contains species pathogenic on plants and aquatic animals. To provide the first foray into gene diversity of <em>A. euteiches</em>, two cDNA libraries were constructed using mRNA extracted from mycelium grown in an artificial liquid medium or in contact to plant roots. A unigene set of 7,977 sequences was obtained from 18,864 high-quality expressed sequenced tags (ESTs) and characterized for potential functions. Comparisons with oomycete proteomes revealed major differences between the gene content of <em>A. euteiches</em> and those of <em>Phytophthora</em> species, leading to the identification of biosynthetic pathways absent in <em>Phytophthora</em>, of new putative pathogenicity genes and of expansion of gene families encoding extracellular proteins, notably different classes of proteases. Among the genes specific of <em>A. euteiches</em> are members of a new family of extracellular proteins putatively involved in adhesion, containing up to four protein domains similar to fungal cellulose binding domains. Comparison of <em>A. euteiches</em> sequences with proteomes of fully sequenced eukaryotic pathogens, including fungi, apicomplexa and trypanosomatids, allowed the identification of <em>A. euteiches</em> genes with close orthologs in these microorganisms but absent in other oomycetes sequenced so far, notably transporters and non-ribosomal peptide synthetases, and suggests the presence of a defense mechanism against oxidative stress which was initially characterized in the pathogenic trypanosomatids.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>Gaulin E, Madoui M-A, Bottin A, Jacquet C, Mathé C, et al.  (2008) Transcriptome of <em>Aphanomyces euteiches</em>: New Oomycete Putative Pathogenicity Factors and Metabolic Pathways. PLoS ONE 3(3):
          e1723.
            doi:10.1371/journal.pone.0001723</p><p><strong>Academic Editor: </strong>Cecile Fairhead, Pasteur Institute, France</p><p><strong>Received:</strong> December 28, 2007; <strong>Accepted:</strong> February 5, 2008; <strong>Published:</strong> March 5, 2008</p><p><strong>Copyright:</strong> © 2008 Gaulin et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>This work was funded by CNRS, University of Toulouse (Toulouse III) and Genoscope.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>The legume root rot disease caused by the oomycete pathogen <em>Aphanomyces euteiches</em> is one major yield reducing factor in legume production including pea and alfalfa throughout the world <a href="#pone.0001723-Delwich1">[1]</a>–<a href="#pone.0001723-Wicker1">[3]</a>. <em>A. euteiches</em> can infect plants at any age. The first symptoms on pea may be discerned on roots which are softened and water-soaked. The pathogen spreads rapidly through the cortical tissue and the fine branches of feeding rootlets are destroyed. Then the epicotyls become dark and eventually collapse. Leaves progressively turn yellow, starting at the bottom of the shoot. In severe cases, the plants collapse and die before forming any pod.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>Thick-walled oospores are resting stages and constitute the primary inoculum source in soil <a href="#pone.0001723-Mitchell1">[4]</a>. After germination, primary zoospores are released at the apex of sporangia. These bi-flagellate motile elements encyst at the root surface, germinate and infect the host root. After a few days, the pathogen can reproduce sexually in the host and form new oospores. These oospores can remain dormant in soil and organic debris for many years <a href="#pone.0001723-Larsson1">[5]</a>. Effective chemical controls for <em>Aphanomyces</em> root rot of legumes are not available. Crop rotation and bioassay methods to assess the inoculum potential in the soil remain the only effective method to avoid disease. Using recurrent selection-based strategies, pea germplasms with partial resistance or tolerance were obtained <a href="#pone.0001723-PiletNayel1">[6]</a>.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>Among Oomycetes, the genus <em>Aphanomyces</em> belongs to the Saprolegniales and comprises species that are destructive on plants, crustaceans and fishes such as <em>A. astaci</em> and <em>A. invadans.</em> Virulent genotypes of <em>A. astaci</em> were introduced in Europe giving rise to disease outbreaks among its crustacean host population, the freshwater crayfish <a href="#pone.0001723-Sderhll1">[7]</a>. This parasite is now considered as one of the 100 world's worst invasive species (Global Invasive Species Database, http//<a href="http://www.issg.org/database">www.issg.org/database</a>). For the last 30 years, the epizootic ulcerative syndrome (EUS) a disease caused by <em>A. invadans</em>, has affected wild and farmed fish in Asia and Indo-Pacific regions and has been recently detected in the eastern seaboard of the US <a href="#pone.0001723-Blazer1">[8]</a>. <em>Aphanomyces</em> genus also includes significant plant pathogens such as <em>A. euteiches</em> and <em>A. cochlioides</em> causing the most serious seedling disease of sugar beet in terms of yield loss, persistence of the oospores in soil, and difficulty of control <a href="#pone.0001723-Gaulin1">[9]</a>, <a href="#pone.0001723-Martin1">[10]</a>. By contrast to this diversity of hosts, some oomycete genera, such as the Peronosporale <em>Phytophthora</em> is composed of species which are only pathogenic to plants. Such diversity within the oomycetes could reflect different evolutionary histories and possibly different mechanisms of infection between Saprolegniales and Peronosporales <a href="#pone.0001723-Kamoun1">[11]</a>.</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p>Until recently, oomycetes genomics focussed only on <em>Phytophthora</em> spp. with the completion of genome sequencing projects and large scale identification of expressed sequences <a href="#pone.0001723-Randall1">[12]</a>–<a href="#pone.0001723-Panabires1">[17]</a>. These data, combined to functional studies, led to the identification of new factors of pathogenicity such as the large family of RxLR effectors which are thought to manipulate the plant cell upon their translocation into the host cytoplasm <a href="#pone.0001723-Whisson1">[18]</a>. However little is known about the biology and pathology of other oomycetes, notably those belonging to the Saprolegniales. To fill this gap, we have undertaken a large scale sequencing of <em>A. euteiches</em> cDNAs. This microorganism is of interest since it is able to interact with the model legume <em>Medicago truncatula,</em> offering the opportunity to engage genetic approaches for deciphering this interaction <a href="#pone.0001723-Gaulin1">[9]</a>. Here we report on the characterization of an EST collection comprising 18,684 EST assembled into 7,977 unique sequences from two different cDNA libraries. A systematic comparative strategy was engaged to identify specific <em>A. euteiches</em> gene sets absent in <em>Phytophthora</em> species. This work greatly expands our knowledge of this group of destructive pathogens and points to the large evolutionary divergence within the oomycete lineage.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4><em>Aphanomyces euteiches</em> growth conditions</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p><em>Aphanomyces euteiches</em> (ATCC201684) was maintained on Corn Meal Agar medium (8.5 g/l) at 28°C in the dark. Synthetic medium <a href="#pone.0001723-Papavizas1">[19]</a> supplemented with pectin from apple (10 g/l) was used to perform growth assay on pectin.</p>


<h4>cDNA libraries construction</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>Two unidirectional cDNA libraries from <em>A. euteiches</em> were prepared from mycelium grown on medium containing yeast extract/glucose (library MYC) and from mycelium in interaction with <em>Medicago truncatula</em> roots (library INT) <a href="#pone.0001723-Madoui1">[20]</a>. A total of 18,684 expressed sequence tags (9,224 from library MYC and 9,460 from library INT) were submitted to the EBI databank for accession number assignation and were assembled into 7,977 unigenes <a href="#pone.0001723-Madoui1">[20]</a>.</p>


<h4>Sequence analysis and database searches</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>Sequences describe here could be downloaded from the <em>Aphanomyces</em> database AphanoDB (<a href="#pone.0001723-Papavizas1">[19]</a>, <a href="http://www.polebio.scsv.ups-tlse.fr/aphano/">http://www.polebio.scsv.ups-tlse.fr/apha​no/</a>) and cDNA clones are available at the CNRGV (<a href="http://cnrgv.toulouse.inra.fr/ENG/">http://cnrgv.toulouse.inra.fr/ENG/</a>). Data searches and analyses have been conducted on AphanoDB using local tools.</p>
<a id="article1.body1.sec2.sec3.p2" name="article1.body1.sec2.sec3.p2"></a><p>For detection of RxLR effectors, all open reading frames (ORFs) encoding at least 100 amino acids (delimited by two stops) were extracted. This led to 136,617 putative ORFs. Signal peptides were predicted with SignalP version 3.0 <a href="#pone.0001723-Bendtsen1">[21]</a>, with a hidden Markov model probability cutoff of 0.8 and 10,131 candidate secreted proteins were obtained.</p>
<a id="article1.body1.sec2.sec3.p3" name="article1.body1.sec2.sec3.p3"></a><p>Three independent strategies were used to look for the RxLR motif over all these sequences:</p>


<ol class="simple">

<li>- Search for the regular expression RxLR-x(1,40)-[ED][ED][KR] (with the fuzzpro program from EMBOSS suite <a href="#pone.0001723-Rice1">[22]</a>) in the first 100 residues downstream of the signal peptide cleavage site.</li>

<li>This led to the identification of 5 sequences, among which only two were retained as being in the same frame as the BLASTX matches.</li>

<li>- Search with HMM profiles performed with the hmmsearch program from the HMMer 2.2 package <a href="#pone.0001723-Eddy1">[23]</a> with two alternative profiles, RdEER <a href="#pone.0001723-Win1">[24]</a> and cropped <a href="#pone.0001723-Whisson1">[18]</a>. From the 2 sequences obtained with this method none was in the frame in agreement with BLASTX matches.</li>

<li>- Search for the regular expression RxLx[EDQ]-x(1,40)-[ED][ED][KR] (with the fuzzpro program from EMBOSS suite) in the first 100 residues downstream of the signal peptide cleavage site. The RxLx[EDQ] motif was the original signal required for secretion identified in <em>Plasmodium falciparum</em> <a href="#pone.0001723-Hiller1">[25]</a>: 2 out of the 3 resulting sequences were discarded because of inconsistency of the strand relatively to similarity searches.</li>

</ol><a id="article1.body1.sec2.sec3.p4" name="article1.body1.sec2.sec3.p4"></a><p>Multiple alignments were conducted using the program CLUSTALW <a href="#pone.0001723-Thompson1">[26]</a> and visualized with BOXSHADE (<a href="http://www.ch.embnet.org/software/BOX_form.html">http://www.ch.embnet.org/software/BOX_fo​rm.html</a>). CLUSTALW results were submitted to the WebLogo server <a href="#pone.0001723-Crooks1">[27]</a> (<a href="http://weblogo.berkeley.edu">http://weblogo.berkeley.edu</a>).</p>

</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Results"></a><h3>Results</h3>
<h4>cDNA libraries, cDNA sequencing and functional annotation</h4>
<a id="article1.body1.sec3.sec1.p1" name="article1.body1.sec3.sec1.p1"></a><p>Two unidirectionnal cDNA libraries were constructed using mRNAs isolated from mycelium of <em>A. euteiches</em> cultivated either <em>in vitro</em> in a liquid medium (MYC library) or grown in contact to <em>M. truncatula</em> roots (INT library). A total of 18,684 high-quality sequences (9,224 for the MYC library and 9,460 for the INT library) corresponding to the 5′ end of cDNA inserts were acquired. A 7,977 unigene set was assembled from EST data. Among the unigenes, 2,843 consensus sequences were assembled from multiple sequence reads and 5,134 were singlets.</p>
<a id="article1.body1.sec3.sec1.p2" name="article1.body1.sec3.sec1.p2"></a><p>The 7,977 unigene set was annotated by comparison to the NCBI non-redundant (nr) protein database (5-17-2007 Version) using BLAST analyses <a href="#pone.0001723-Altschul1">[28]</a>. Proteomes of seven fully sequenced organisms were added to the analysis. These include the proteomes from two oomycetes (<em>P. sojae</em> and <em>P. ramorum</em>), the diatom <em>Thalassiosira pseudonana</em>, the pathogenic fungus <em>Nectria haematococca</em>, the model plant <em>Arabidopsis thaliana</em> and the apicomplexa parasites <em>Toxoplasma gondii</em> and <em>Plasmodium falciparum</em>. Overall, about 70% of the sequences showed homology to previously described genes in the NCBI non-redundant protein database using a BLASTX E value cut-off of 10<sup>−5</sup> (<a href="#pone-0001723-g001">Figure 1</a>). As expected, a large proportion of sequences (80%, E value cut-off of &lt;10<sup>−5</sup>) had significant similarities to <em>Phytophthora</em> predicted proteins. Interestingly, about the same fraction (50%, E value cut-off of 10<sup>−5</sup>) of <em>A. euteiches</em> sequences showed homology to a plant, diatom, apicomplexa or fungal sequence whereas these organisms are distantly related (<a href="#pone-0001723-g001">Figure 1</a>). The unigene set was then annotated by comparisons with the Pfam database of protein domains <a href="#pone.0001723-Bateman1">[29]</a>. 45% of the sequences showed homology with a protein domain with an E value&lt;10<sup>−5</sup>. All the data were stored in a public database called AphanoDB (<a href="http://www.polebio.scsv.ups-tlse.fr/aphano/">www.polebio.scsv.ups-tlse.fr/aphano/</a>) which is described elsewhere <a href="#pone.0001723-Madoui1">[20]</a>. Consensus and singlet sequences were named with a unique identifier with the prefix ‘Ae’ for the species, 3 digits for the number of EST included in the contig, the 2 digits ‘AL’ for the strain and 5 digits as unique number.</p>
<div class="figure" id="pone-0001723-g001"><div class="img"><a name="pone-0001723-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001723.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001723" data-uri="info:doi/10.1371/journal.pone.0001723.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001723.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001723.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001723.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001723.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001723.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001723.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001723.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001723.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001723.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Comparison of <em>A. euteiches</em> sequences to the non redundant protein database at NCBI and proteomes of fully sequenced organisms.</span></strong></p><a id="article1.body1.sec3.sec1.fig1.caption1.p1" name="article1.body1.sec3.sec1.fig1.caption1.p1"></a><p>Unigene sequences were blasted using the BLASTX software to the nr database and to the proteome of <em>P. sojae</em> (Ps), <em>A. thaliana</em> (At), <em>N. haematococca</em> (Nh), <em>T. pseudonana</em> (Tp) and <em>T. gondii</em> (Tg). For each class of e-value the percentage of unigenes showing homology is indicated.</p>
<span>doi:10.1371/journal.pone.0001723.g001</span></div>

<h4>Genes highly represented in the interaction library</h4>
<a id="article1.body1.sec3.sec2.p1" name="article1.body1.sec3.sec2.p1"></a><p>Determination of the number of sequences specific of each library provided a good indication that the two growth conditions used for library construction allowed the expression of divergent transcriptomes. More than 40% of the sequences appeared to be specific for one library (<a href="#pone-0001723-t001">Table 1</a>). To identify unigenes with a statistical difference in transcript abundance, we used a method described in <a href="#pone.0001723-Susko1">[30]</a> based upon the frequencies which genes occur in each library and false discovery rate control for multiple test corrections <a href="#pone.0001723-Benjamini1">[31]</a>. Eight genes abundantly represented (<em>p</em>-value&lt;B-H cutoff) in the interaction library were identified (<a href="#pone-0001723-t002">Table 2</a>). Interestingly, 4 genes encode putative proteins with significant similarity with transporters: 2 sucrose transporters (Ae_48AL7986 and Ae_29AL7339), a protein with an oligopeptide transporter domain (Ae_21AL7132; InterPro domain IPR004813), and a gene coding a probable mitochondrial substrate carrier (Ae_97AL5378). This latter gene is of interest since it has been shown that a gene encoding a mitochondrial carrier protein is required for pathogenicity of the soil-borne fungus <em>Fusarium oxysporum</em> <a href="#pone.0001723-Inoue1">[32]</a>.</p>
<div class="figure" id="pone-0001723-t001"><div class="img"><a name="pone-0001723-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001723.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001723" data-uri="info:doi/10.1371/journal.pone.0001723.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001723.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001723.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001723.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001723.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001723.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001723.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001723.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001723.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001723.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>Number of expressed sequence tags and unigenes specific to a given library.</span></strong></p><span>doi:10.1371/journal.pone.0001723.t001</span></div><div class="figure" id="pone-0001723-t002"><div class="img"><a name="pone-0001723-t002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001723.t002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001723" data-uri="info:doi/10.1371/journal.pone.0001723.t002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001723.t002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001723.t002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001723.t002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001723.t002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001723.t002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001723.t002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001723.t002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001723.t002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001723.t002.TIF"></span>)
                </a></li></ul></div><p><strong>Table 2.  <span>Unigenes highly represented in the INT library.</span></strong></p><span>doi:10.1371/journal.pone.0001723.t002</span></div>

<h4>Genes encoding potential pathogenicity proteins</h4>
<a id="article1.body1.sec3.sec3.p1" name="article1.body1.sec3.sec3.p1"></a><p>Plant pathogens express a large array of genes playing various roles in pathogenesis. These genes can be roughly classified into eight major categories according to a scheme established for <em>P. sojae</em> and <em>P. parasitica</em> <a href="#pone.0001723-TortoAlalibo1">[16]</a>, <a href="#pone.0001723-Panabires1">[17]</a>. To facilitate comparative analyses, <em>A. euteiches</em> unigenes were classified following the same criteria. The results obtained with the <em>A. euteiches</em> ESTs were compared to those obtained from <em>P. sojae</em> ESTs described in <a href="#pone.0001723-TortoAlalibo1">[16]</a> since in this later study about the same number of ESTs as in <em>A. euteiches</em> was analysed (26,943 ESTs assembled into 7,863 unigenes). This comparative approach revealed striking differences between <em>A. euteiches</em> and <em>P. sojae</em> (<a href="#pone-0001723-g002">Figure 2</a>). Gene families over-represented in <em>A. euteiches</em> compared to <em>P. sojae</em> include genes encoding proteins playing a role in adhesion (mucin-like proteins), in protein degradation and in drug resistance (ABC and PDR-like ABC transporter, cytochrome P450 enzymes).</p>
<div class="figure" id="pone-0001723-g002"><div class="img"><a name="pone-0001723-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001723.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001723" data-uri="info:doi/10.1371/journal.pone.0001723.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001723.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001723.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001723.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001723.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001723.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001723.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001723.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001723.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001723.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Classification of unigenes based on eight functional categories involved in pathogenicity.</span></strong></p><a id="article1.body1.sec3.sec3.fig1.caption1.p1" name="article1.body1.sec3.sec3.fig1.caption1.p1"></a><p><em>A. euteiches</em> unigenes (black bars) for which a putative function was assigned were classified into defined categories according to <a href="#pone.0001723-Panabires1">[17]</a> and <a href="#pone.0001723-TortoAlalibo1">[16]</a>. For each category the number of ESTs (A) and the number of unigene (B) is shown. For comparison to <em>Phytophthora</em>, analysis of a EST collection from <em>P. sojae</em> was added to the figure <a href="#pone.0001723-TortoAlalibo1">[16]</a> (white bars).</p>
<span>doi:10.1371/journal.pone.0001723.g002</span></div>

<h4>Adhesion</h4>
<a id="article1.body1.sec3.sec4.p1" name="article1.body1.sec3.sec4.p1"></a><p>One of the most well-characterized protein playing a key role in oomycete adhesion to plant surface component is a cell wall protein isolated from <em>P. parasitica</em> named CBEL (for Cellulose Binding Elicitor Lectin) <a href="#pone.0001723-SjalonDelmas1">[33]</a>, <a href="#pone.0001723-VillalbaMateos1">[34]</a>. CBEL contains two fungal type I Cellulose Binding Domains (CBD or CBM_1; InterPro domain IPR000254) which are involved in cellulose binding but which are also perceived by the host cells as pathogen associated molecular patterns <a href="#pone.0001723-Gaulin2">[35]</a>. CBEL also harbours two regions with similarities to the N/apple PAN domain (IPR000177), a conserved domain involved in protein-protein or protein-carbohydrate interactions <a href="#pone.0001723-Tordai1">[36]</a>. cDNA sequences showing typical CBEL features, <em>i.e</em> CBDs associated to N/apple PAN domains, were not found in <em>A. euteiches</em> ESTs. Accordingly, antigens were not detected with antibodies directed against CBEL, when mycelium extracts were probed by western blot analysis (data not shown). However, CBDs were detected in a large gene family coding proteins showing similarities to a mucin-like protein identified in the soybean root nematode <em>Heterodera glycines</em> (GenBank accession number AAC62109). More than 150 ESTs were assembled into 32 unigenes, the largest contig containing 39 ESTs (Ae_39AL5321). Similarity between <em>A. euteiches</em> and <em>H. glycines</em> proteins is restricted to a domain originally found in cyst germination proteins of <em>P. infestans</em> (<a href="#pone-0001723-g003">Figure 3</a>; <a href="#pone.0001723-Gornhardt1">[37]</a>). The full length sequence of Ae_10AL7886 was determined and domain organization of the predicted protein was deduced (<a href="#pone-0001723-g003">Figure 3</a>). SignalP analysis of the predicted protein identified a 15-amino acids putative signal peptide. Four domains showing typical features of fungal cellulose binding domains were found in the N-terminal and C-terminal ends of the protein. The central part showed similarities to the <em>H. glycines</em> mucin-like protein and the cyst germination proteins of <em>P. infestans</em>. The combination of these two modules was not detected so far and suggests a role of this protein in cell adhesion.</p>
<div class="figure" id="pone-0001723-g003"><div class="img"><a name="pone-0001723-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001723.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001723" data-uri="info:doi/10.1371/journal.pone.0001723.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001723.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001723.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001723.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001723.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001723.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001723.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001723.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001723.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001723.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Protein domain organisation of cell surface proteins from oomycetes and apicomplexa.</span></strong></p><a id="article1.body1.sec3.sec4.fig1.caption1.p1" name="article1.body1.sec3.sec4.fig1.caption1.p1"></a><p>Two proteins from <em>Phytophthora</em>, CBEL and CAR which are localized at the cell surface <a href="#pone.0001723-SjalonDelmas1">[33]</a>, <a href="#pone.0001723-Gornhardt1">[37]</a> are represented. CBEL contains two closely associated domains, a cellulose binding domain (CBD) and a N/apple PAN domain (PAN). CAR proteins are composed of a mucin-like region and a N-terminal region (CAR) with conserved cysteine residues. AMA1 is a <em>P. falciparum</em> protein with two PAN domains <a href="#pone.0001723-Bai1">[39]</a>.</p>
<span>doi:10.1371/journal.pone.0001723.g003</span></div><a id="article1.body1.sec3.sec4.p2" name="article1.body1.sec3.sec4.p2"></a><p>Eight distinct cDNA sequences showed similarity to CBEL centered on the N/apple PAN domain. This domain is of interest since it has been found in surface proteins of apicomplexan parasites essential for attachment to host cells <a href="#pone.0001723-Brecht1">[38]</a>. Whereas the role of this domain in oomycete proteins is still unclear, it can be speculated that it can interact with host targets to promote pathogenesis. cDNAs coding for proteins with N/Apple domains fell into two categories. The first category included predicted protein sequences for which no other protein domains were found except one or several N/Apple domains. A representative example is Ae_1AL0128 which was fully sequenced (<a href="#pone-0001723-g003">Figure 3</a>). It includes three N/Apple domains and a predicted transmembrane domain located at the C-terminus end. Interestingly, a transmembrane domain was also detected in the <em>P. falciparum</em> surface protein AMA1 which was shown to contain two N/Apple PAN domains (<a href="#pone-0001723-g003">Figure 3</a>, <a href="#pone.0001723-Bai1">[39]</a>). The second category included predicted protein sequences for which N/apple domains were associated with a catalytic domain. Two of these unigenes (Ae_7AL7989 and Ae_1AL5080) were predicted to encode proteins with a peptidase domain associated to one (Ae_1AL5080) or five (Ae_7AL7989) N/apple domains. Finally, three unigenes were identified showing several copies of a thrombospondin type I motif, a conserved 50 amino acid sequence found in surface proteins of <em>Phytophthora</em> and apicomplexan parasites <a href="#pone.0001723-Robold1">[40]</a>, <a href="#pone.0001723-Anantharaman1">[41]</a> and putatively involved in adhesion.</p>


<h4>Cell wall degrading enzymes (CWDEs)</h4>
<a id="article1.body1.sec3.sec5.p1" name="article1.body1.sec3.sec5.p1"></a><p>Sequences encoding enzymes potentially involved in degradation of plant cell walls were selected, by looking for the presence of glycoside hydrolase (GH) domains of enzymes hydrolysing substrates that can be found in plant cell walls, such as β-1,4- and β-1,3-glucanases, pectinases, xylosidases, arabinosidases and glucosidases of various specificities <a href="#pone.0001723-Henrissat1">[42]</a>. A collection of 53 potential CWDE unigenes out of 7977 unigenes, a proportion similar to what was recorded in <em>P. sojae</em> or <em>ramorum</em> <a href="#pone.0001723-Tyler1">[15]</a> was obtained. All but one <em>A. euteiches</em> CWDE unigenes had a <em>Phytophthora</em> homolog with an E value lower than 10<sup>−40</sup>.</p>
<a id="article1.body1.sec3.sec5.p2" name="article1.body1.sec3.sec5.p2"></a><p>β-1,4-glucanases and β-1,3-glucanases mainly of GH families 5 and 81 potentially involved in degradation of plant cellulose and callose were represented by 29 unigenes. Among the putative β-1,3-glucanases from <em>A. euteiches</em>, an homolog (Ae_2AL6121) of a previously described GH family 17 endo-1,3-β-glucanase from <em>Saprolegnia parasitica</em> <a href="#pone.0001723-TortoAlalibo2">[43]</a> and <em>P. infestans</em> (PiENDO1; <a href="#pone.0001723-McLeod1">[44]</a>) was found, as well as two unigenes (Ae_2AL7302 and Ae_12AL7443) showing high similarity to the GH family 5 exo-1,3-β-glucanase PiEXO3 from <em>P. infestans</em> <a href="#pone.0001723-McLeod1">[44]</a>. Interestingly, the latter unigenes are exclusively composed of ESTs from the interaction library, suggesting that they are involved in the interaction with the plant.</p>
<a id="article1.body1.sec3.sec5.p3" name="article1.body1.sec3.sec5.p3"></a><p>Other glycosidases from <em>A. euteiches</em>, potentially involved in degradation of plant cell wall polysaccharides such as xyloglucans, xylans or galacto(gluco)mannans, were represented by 24 unigenes encoding GH family 1, 3 and 30 enzymes. However, GH family 12 endoglucanases, potentially involved in xyloglucan degradation and represented by a family of 8 to 10 genes in <em>P. ramorum</em> and <em>P. sojae</em> <a href="#pone.0001723-Costanzo1">[45]</a>, and GH family 10 xylanases, represented by more than 5 genes in the <em>P. ramorum</em> and <em>P. sojae</em> genomes, were missing in the database.</p>
<a id="article1.body1.sec3.sec5.p4" name="article1.body1.sec3.sec5.p4"></a><p>Surprisingly, no EST related to pectin metabolism was found, despite the fact that <em>P. sojae</em> and <em>P. ramorum</em> genomes contain numerous pectinase genes <a href="#pone.0001723-Tyler1">[15]</a>, and <em>bona-fide</em> pectinase genes were found in <em>P. infestans</em>, <em>P. cinnamomi</em> and <em>P. parasitica</em> <a href="#pone.0001723-Randall1">[12]</a>, <a href="#pone.0001723-Gotesson1">[46]</a>–<a href="#pone.0001723-Torto1">[48]</a>. The lack of sequences related to pectin degradation is correlated with the lack of pectinase activity observed in yeast extract/glucose medium (data not shown) and the failure to grow <em>A. euteiches</em> on a synthetic medium containing pectin as sole carbon source, whereas <em>P. parasitica</em>, which produces an inducible polygalacturonase <a href="#pone.0001723-Yan1">[47]</a>, grew equally well on glucose or pectin medium (<a href="#pone-0001723-g004">Figure 4</a>). These results showed that pectinase genes were poorly expressed in the growth conditions we used, and that pectinase genes are differently expressed in <em>A. euteiches</em> and <em>Phytophthora</em> species.</p>
<div class="figure" id="pone-0001723-g004"><div class="img"><a name="pone-0001723-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001723.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001723" data-uri="info:doi/10.1371/journal.pone.0001723.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001723.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001723.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001723.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001723.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001723.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001723.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001723.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001723.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001723.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>Growth of <em>A. euteiches</em> on synthetic media.</span></strong></p><a id="article1.body1.sec3.sec5.fig1.caption1.p1" name="article1.body1.sec3.sec5.fig1.caption1.p1"></a><p><em>A. euteiches</em> and <em>Phytophthora parasitica</em> mycelium were grown on synthetic medium containing Glucose (SM-Glc) or Pectin (SM-Pect) as carbon source. Photos were taken 15 days after inoculation.</p>
<span>doi:10.1371/journal.pone.0001723.g004</span></div>

<h4>Proteases and protease inhibitors</h4>
<a id="article1.body1.sec3.sec6.p1" name="article1.body1.sec3.sec6.p1"></a><p>A large set of <em>A. euteiches</em> sequences were predicted to encode proteases (<a href="#pone-0001723-g002">Figure 2</a>). Serine, aspartyl and cysteine proteases were identified, the largest family being the cysteine protease family. Comparison with the <em>P. sojae</em> ESTs collection revealed that the number of protease genes is significantly higher in <em>A. euteiches</em> than in <em>P. sojae</em>. For example, 17 unigenes (67 ESTs) of <em>A. euteiches</em> were predicted to encode serine carboxypeptidase (IPR001563) whereas one unigene (1 EST) was found in <em>P. sojae</em>. Another example is represented by cysteine proteases (IPR000668) for which 25 unigenes (125 ESTs) were found whereas only 5 unigenes (41 ESTs) were detected in the <em>P. sojae</em> EST collection. Moreover, two types of proteases belonging to the cysteine protease family, peptidase c1b (IPR004134) and peptidase c14 (caspase, IPR011600) were found only in <em>A. euteiches</em> and not in other oomycetes. This analysis showed that <em>A. euteiches</em> is clearly distinct from <em>Phytophthora</em> species with respect to protease production, and this is in accordance with the detection of a high protease activity in culture medium (data not shown).</p>
<a id="article1.body1.sec3.sec6.p2" name="article1.body1.sec3.sec6.p2"></a><p>By contrast with the large number of proteases genes, only two genes (13 ESTs) encoding protease inhibitors were detected. These include a cystein protease inhibitor of the cystatin family (Ae_2AL5945; IPR000010) and a serine protease inhibitor (Ae_11AL6547) containing Kazal domains (IPR002350). These two types of protease inhibitors and their potential role in defense-counterdefense mechanisms were studied in <em>P. infestans</em> <a href="#pone.0001723-Tian1">[49]</a>, <a href="#pone.0001723-Tian2">[50]</a>. Five domains containing typical features of Kazal motifs were found in Ae_11AL6547. However, more domains could be present since the sequence appeared to be incomplete. The predicted translation product of Ae_2AL5945 were analysed with SignalP leading to the identification of a 17 amino acid signal peptide with a significant mean S value of 0.90 and hidden Markov probability of 1.0. Similarity searches of the predicted protein against the NCBI non-redundant database using the BLASTP program revealed a weak similarity (<em>E</em> value = 6.10<sup>−3</sup>) to the <em>P. infestans</em> cystatin EPIC4, centered on the predicted active site. Multiple alignment using ClustalW <a href="#pone.0001723-Thompson1">[26]</a> was done using cystatin domains from <em>P. infestans</em> (EPIC1, EPIC2A, EPIC2B, EPIC3 and EPIC4) and the two domains from Ae_2AL5945 (Ae_2AL5945_1 and Ae_2AL5945_2). Cystatin domains from <em>A. euteiches</em> are clearly distinct from <em>P. infestans</em> sequences except in the highly conserved sites of cystatins, <em>i.e.</em> the N terminal trunk and the L1 binding loop (<a href="#pone-0001723-g005">Figure 5</a>). However, the second binding loop is divergent in <em>A. euteiches</em> sequences.</p>
<div class="figure" id="pone-0001723-g005"><div class="img"><a name="pone-0001723-g005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001723.g005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001723" data-uri="info:doi/10.1371/journal.pone.0001723.g005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001723.g005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001723.g005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001723.g005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001723.g005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001723.g005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001723.g005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001723.g005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001723.g005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001723.g005.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 5.  <span>A cystatin-like protease inhibitor in <em>A. euteiches.</em></span></strong></p><a id="article1.body1.sec3.sec6.fig1.caption1.p1" name="article1.body1.sec3.sec6.fig1.caption1.p1"></a><p>A, domain organisation of the deduced protein sequence of Ae_2AL5945. A putative signal peptide (SP) and two domains similar to cystatin domain are shown. B, alignment of cystatin domains from <em>P. infestans</em> cystatins (EPIC1, EPIC2A, EPIC2B , EPIC3 and EPIC4) and from Ae_2AL5945. The active sites of cystatins, including the N-terminal trunk (NT), first binding loop (L1), and second binding loop (L2), are shown.</p>
<span>doi:10.1371/journal.pone.0001723.g005</span></div>

<h4>Effectors</h4>
<a id="article1.body1.sec3.sec7.p1" name="article1.body1.sec3.sec7.p1"></a><p>A class of oomycete effectors which could be delivered inside plant cells during infection was recently characterized. Effectors from <em>Hyaloperonospora parasitica</em>, <em>P. infestans</em> and <em>P. sojae</em> do not show extensive similarity except a conserved RxLR motif downstream of the signal peptide, followed by an EER motif. The <em>A. euteiches</em> unigene collection was mined for putative RxLR effectors looking for matches either to a sequence pattern or to HMM profiles. We end up with two unigenes containing the RxLR-dEER motif (Ae_1AL5059, Ae_1AL1390) and one with a RxLx[EDQ]-dEER motif (Ae_40AL5333), which is close to the <em>Plasmodium</em> motif. Ae_1AL5059 and Ae_1AL1390 showed homology to a putative <em>P. sojae</em> and <em>P. ramorum</em> genes but appeared to be incomplete. Ae_40AL5333 encodes a putative lysine rich protein with no homology to known protein sequence.</p>
<a id="article1.body1.sec3.sec7.p2" name="article1.body1.sec3.sec7.p2"></a><p>A distinct class of putative cytoplasmic effectors is the Crinkling and Necrosis (CRN) family first described in <em>P. infestans</em> <a href="#pone.0001723-Torto1">[48]</a>. While <em>Phytophthora</em> CRNs lack an RxLR motif, some <em>H. parasitica</em> CRNs show an RxRL sequence overlapping another conserved motif LxFLAK <a href="#pone.0001723-Win1">[24]</a>. Several sequences showing high similarity with <em>Phytophthora</em> CRN were detected in <em>A. euteiches</em>. The largest groups of CRNs showed strong similarity to <em>P. infestans</em> CRN5 (14 unigenes) or CRN13 (9 unigenes). Interestingly, the conserved LxLFLAK sequence was not present in <em>A. euteiches</em> genes, but a closely related motif, F/LxLYLALK, was detected (<a href="#pone-0001723-g006">Figure 6</a>). This is the first time that CRN genes are found in an organism distinct from Peronosporales suggesting that this class of effectors plays an important role in oomycete pathogenicity.</p>
<div class="figure" id="pone-0001723-g006"><div class="img"><a name="pone-0001723-g006" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001723.g006&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001723" data-uri="info:doi/10.1371/journal.pone.0001723.g006"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001723.g006&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001723.g006/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001723.g006/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001723.g006/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001723.g006/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001723.g006.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001723.g006/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001723.g006/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001723.g006.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 6.  <span>Identification of conserved amino acid residues at the N-terminal extremities of <em>P. infestans</em> and <em>A. euteiches</em> CRN sequences.</span></strong></p><a id="article1.body1.sec3.sec7.fig1.caption1.p1" name="article1.body1.sec3.sec7.fig1.caption1.p1"></a><p>A, multiple alignment of <em>A. euteiches</em> like CRNs and <em>P. infestans</em> CRNs (CRN1, CRN5, CRN11, CRN14) showing the highest homology to <em>A. euteiches</em> sequences. B, consensus sequence pattern calculated using Weblogo showing the most conserved amino acid residues.</p>
<span>doi:10.1371/journal.pone.0001723.g006</span></div><a id="article1.body1.sec3.sec7.p3" name="article1.body1.sec3.sec7.p3"></a><p>Other oomycete effector genes include genes coding necrosis-inducing peptides (NPP family; <a href="#pone.0001723-Fellbrich1">[51]</a>, <a href="#pone.0001723-Qutob1">[52]</a>) and elicitins <a href="#pone.0001723-Ponchet1">[53]</a>, <a href="#pone.0001723-Jiang1">[54]</a>. No ortholog of NPP was found and only one divergent sequence encoding a putative elicitin-like protein was detected (Ae_1AL4317).</p>


<h4>Transporters</h4>
<a id="article1.body1.sec3.sec8.p1" name="article1.body1.sec3.sec8.p1"></a><p>A large number of sequences were predicted to encode proteins involved in transport of molecules. The largest family encodes ABC transporters (84 unigenes and 342 ESTs) and PDR-like ABC transporters (28 unigenes and 86 ESTs). Interestingly, two oligopeptide transporter genes (Ae_21AL7132 and Ae_1AL0658) encoding putative proteins with an OPT domain (IPR004813) were found. Two distinct, proton-coupled peptide transport systems have been described in eukaryotic organisms: the PTR (peptide transport) system, specific for di- and tripeptides, and the OPT (oligopeptide transport) system, which transports tetra- and pentapeptides. Homolog of di-/tripeptide transporters are found in virtually all organisms, whereas the oligopeptide transporters are limited to fungi and plants (reviewed in <a href="#pone.0001723-Stacey1">[55]</a>, <a href="#pone.0001723-Hauser1">[56]</a>). While the PTR system is well represented in <em>Phytophthora</em> proteomes (8 genes in <em>P. sojae</em>), the OPT proteins are absent, suggesting that OPT dependant oligopeptide transport is not widely found in oomycetes. However, Ae_21AL7132 and Ae_1AL0658 show strong similarity with an <em>A. thaliana</em> sequence (E value = 6.10<sup>−72</sup> and 5.10<sup>−38</sup> respectively) encoding a metal-nicotianamine transporter. This class of plant transporters is involved in iron–phytosiderophore uptake by roots but exhibits also significant sequence similarity to several functionally characterized yeast transporters of the OPT family <a href="#pone.0001723-LeJean1">[57]</a>, <a href="#pone.0001723-Yen1">[58]</a>. Thus it cannot be excluded that Ae_21AL7132 and Ae_1AL0658 encoded proteins are involved in uptake of phytosiderophore-Fe complex. Interestingly, ESTs assembled into Ae_21AL7132 were found almost exclusively in the interaction library (20 ESTs in the interaction library and 1 in the mycelium library; <a href="#pone-0001723-t002">Table 2</a>) suggesting a role for this class of transporters in pathogenesis.</p>


<h4>Identification of <em>A. euteiches</em> genes without orthologs in <em>Phytophthora</em></h4>
<a id="article1.body1.sec3.sec9.p1" name="article1.body1.sec3.sec9.p1"></a><p>Identification of unigenes without any similarity to <em>Phytophthora</em> sequences but with similarity to sequences from trypanosamatid parasites led to the identification of a family of proteins putatively involved in the synthesis of trypanothione, a compound playing a key role in defense against oxidative stress in trypanosomatids <a href="#pone.0001723-Fairlamb1">[59]</a>. Trypanothione is synthesized by conjugation of the polyamine spermidine and the tripeptide glutathione. The biosynthesis of trypanothione starts with the formation of glutathionylspermidine (Gsp) which then reacts with a second GSH molecule to form trypanothione (<a href="#pone-0001723-g007">Figure 7A</a>). These two biosynthetic steps are catalysed by two distinct enzymes, glutathionylspermidine synthetases and trypanothione synthetases. However, it has been shown that trypanothione syntethase can catalyse the entire synthesis of trypanothione in <em>Trypanosoma cruzi</em> and <em>Crithidia fasciculata</em> <a href="#pone.0001723-Oza1">[60]</a>, <a href="#pone.0001723-Comini1">[61]</a>. Glutathionylspermidine has also been identified in <em>E. coli</em> where it could be involved in regulating the intracellular level of spermidine and glutathione. Six <em>A. euteiches</em> unigenes were found encoding proteins with a typical glutathionylspermidine syntethase domain (IPR005494) and similarity to trypanothione syntethase sequences from trypanosomatids (E value&lt;10<sup>−10</sup>). Moreover, one sequence (Ae_2AL7562) contains a second domain found in trypanothione synthetases and bacterial bifunctional glutathionylspermidine synthetase/amidase, the CHAP (cysteine, histidine dependant amidohydrolase/peptidase) domain (IPR007921). Alignment of the protein sequence with an <em>E. coli</em> bifunctional glutathionylspermidine synthetase <a href="#pone.0001723-Bollinger1">[62]</a> and a trypanothione synthetase from the insect parasite <em>C. fasciculata</em> <a href="#pone.0001723-Alphey1">[63]</a> is shown on <a href="#pone-0001723-g007">Figure 7B</a>. These results suggest the presence of a glutathionylspermidine based defense mechanism against oxidative stress in <em>A. euteiches</em> that is absent in other oomycetes such as <em>P. sojae</em> and <em>P. ramorum</em>.</p>
<div class="figure" id="pone-0001723-g007"><div class="img"><a name="pone-0001723-g007" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001723.g007&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001723" data-uri="info:doi/10.1371/journal.pone.0001723.g007"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001723.g007&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001723.g007/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001723.g007/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001723.g007/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001723.g007/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001723.g007.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001723.g007/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001723.g007/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001723.g007.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 7.  <span>Identification of <em>A. euteiches</em> genes putatively involved in trypanothione synthesis.</span></strong></p><a id="article1.body1.sec3.sec9.fig1.caption1.p1" name="article1.body1.sec3.sec9.fig1.caption1.p1"></a><p>A, trypanothione biosynthesis starts with the formation of glutathione catalysed by g-glutamyl synthetase and glutathione synthetase. Trypanothione is synthesized from glutathione and spermidine either in two successive steps catalysed by glutathionyl spermidine synthase and trypanothione synthase. It has been shown in some cases that trypanothione synthase is able to catalysed the two reactions. Unigenes predicted to code enzymes belonging to this pathway are indicated. B, multiple alignment of gluthationyl spermidine synthetase from <em>E. coli</em> (GSP_ECOLI), from <em>Crythidia fasciculata</em> (TRYS CRIFA) and Ae_2AL7562.</p>
<span>doi:10.1371/journal.pone.0001723.g007</span></div><a id="article1.body1.sec3.sec9.p2" name="article1.body1.sec3.sec9.p2"></a><p>To find <em>A. euteiches</em> genes which could be absent in other oomycetes but present in fungal plant pathogens, all the unigenes sequences were compared to the pea root pathogen <em>N. haematococca.</em> The sequences showing an E value greater than 0.1 to a <em>Phytophthora</em> sequence and lower than 10<sup>−10</sup> to a sequence from the <em>N. haematococca</em> proteomes were selected. About half of the sequences (19 unigenes) matched to sequences of unknown function and without known protein domains. Interestingly, 3 of the remaining sequences (Ae_1AL0272, Ae_2AL7741 and Ae_1AL3374) showed a strong similarity to hydrolytic enzymes (glucan hydrolases and lipase) and 3 (Ae_1AL1763, Ae_1AL3400 and Ae_1AL0931) to non-ribosomal peptide synthetases (NRPS). These latter enzymes are of outstanding importance since their products have been shown to be involved in fungal pathogenesis <a href="#pone.0001723-Lee1">[64]</a>, <a href="#pone.0001723-Johnson1">[65]</a> (<a href="#pone-0001723-t003">Table 3</a>). Four putative NRPS genes were detected in <em>P. sojae</em> and <em>P. ramorum</em> genomes <a href="#pone.0001723-Tyler1">[15]</a>. However these genes are not related to the <em>A. euteiches</em> sequences. These data suggests that <em>A. euteiches</em> could be able to produce secondary metabolites distinct from those synthesized by <em>Phytophthora</em> species.</p>
<div class="figure" id="pone-0001723-t003"><div class="img"><a name="pone-0001723-t003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001723.t003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001723" data-uri="info:doi/10.1371/journal.pone.0001723.t003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001723.t003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001723.t003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001723.t003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001723.t003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001723.t003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001723.t003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001723.t003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001723.t003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001723.t003.TIF"></span>)
                </a></li></ul></div><p><strong>Table 3.  <span><em>A. euteiches</em> unigenes showing a BLASTX match to fungal, plant or diatom proteins (E value&lt;1.10<sup>−10</sup>) and not to <em>Phytophthora</em> proteins (E value&gt;0.1).</span></strong></p><span>doi:10.1371/journal.pone.0001723.t003</span></div><a id="article1.body1.sec3.sec9.p3" name="article1.body1.sec3.sec9.p3"></a><p>Comparative analysis of <em>A. euteiches</em> sequences with a high BLASTX score to a <em>A. thaliana</em> or <em>T. pseudonana</em> sequences allowed the identification of two biosynthetic pathways which are absent in <em>Phytophthora</em> species but present in <em>Aphanomyces</em> (<a href="#pone-0001723-t003">Table 3</a>). One sequence (Ae_1AL0450) showed significant similarity to a thiamine biosynthetic enzyme from plants (top hit to protein AAP03875, E value = 1.10<sup>−110</sup>) and a thiamine biosynthesis Thi4 protein domain (IPR002922). This enzyme is involved in the synthesis of the thiamine precursor, thiazole. Members of the genus <em>Phytophthora</em> are thiamine auxotrophs and consequently lack this biosynthetic enzyme whereas <em>A. euteiches</em> and other Saprolegniales such as <em>Saprolegnia parasitica</em> are able to synthesize thiamine. Accordingly, the sequence from <em>A. euteiches</em> is highly similar to the <em>S. parasitica</em> gene (SPM2G2; 93% identity at the protein level) described in <a href="#pone.0001723-TortoAlalibo2">[43]</a>.</p>
<a id="article1.body1.sec3.sec9.p4" name="article1.body1.sec3.sec9.p4"></a><p><em>Phytophthora</em> species being sterol auxotrophs, key genes coding for enzymes involved in sterol biosynthesis were not identified in <em>Phytophthora</em> genomes <a href="#pone.0001723-Tyler1">[15]</a>. By contrast, Saprolegniales species such as <em>A. euteiches</em> are able to grow in minimal media without exogenous sterols. Accordingly, several genes coding for putative sterol biosynthesis enzymes were identified. These include two cycloartenol synthases (Ae_3AL7274; Ae_1AL0818) and a cytochrome P450 enzyme belonging to the CYP51 family (Ae_5AL7244) which strongly matched (E values&lt;10<sup>−70</sup>) plant enzymes. Each gene has also closed orthologs in the diatom <em>T. pseudonana</em> suggesting that the sterol biosynthetic is ancient in the oomycete lineage and has been lost by <em>Phytophthora</em> species.</p>

</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec4.p1" name="article1.body1.sec4.p1"></a><p>In this study we describe an extensive characterization of an EST collection of <em>A. euteiches</em>, an economically important plant pathogen. The closest oomycete specie which was the subject of a genomic approach is the fish pathogen <em>S. parasitica</em>, with the sequencing of 1,510 cDNA clones (1,279 unigenes) obtained from one library constructed from mycelium grown axenically <a href="#pone.0001723-TortoAlalibo2">[43]</a>. The current study is more expansive in terms of number of cDNA clones sequenced (18,684) and constructed libraries (2) and adds to the EST resources available for other oomycetes such as <em>P. sojae</em> (7,863 unigenes) <a href="#pone.0001723-TortoAlalibo1">[16]</a>, <em>P. infestans</em> (18,256 unigenes) <a href="#pone.0001723-Randall1">[12]</a> and <em>P. parasitica</em> (4734 unigenes from two libraries) <a href="#pone.0001723-Panabires1">[17]</a>, <a href="#pone.0001723-LeBerre1">[66]</a>. Comparison between the unigene contents of the two <em>A. euteiches</em> libraries showed that they reflected two divergent expression programs. The procedure used to cultivate <em>A. euteiches</em> mycelium in contact to <em>M. truncatula</em> roots avoided contamination of pathogen mRNAs with plant mRNAs <a href="#pone.0001723-Madoui1">[20]</a>. Nevertheless, perception of plant root tissue by the pathogen occurred, leading to the induction of a specific subset of genes. These include 4 transporter genes which could be involved in nutrient uptake during infection.</p>
<a id="article1.body1.sec4.p2" name="article1.body1.sec4.p2"></a><p>Comparative analysis of <em>A. euteiches</em> unigene sequences to proteomes derived from fully sequenced organisms provided a global view of sequence similarities distribution among organisms which are distantly related. As expected, most of sequences showed similarity to a <em>Phytophthora</em> predicted protein sequence. About 20% of unigenes did not have any ortholog in <em>P. sojae</em> or <em>P. ramorum</em>. More surprisingly, we found that similar proportions of unigenes have an ortholog in the plant <em>A. thaliana</em>, the fungus <em>N. haematococca</em>, the diatom <em>T. pseudonana</em> and the apicomplexa <em>T. brucei</em>. These sequences were examined in more detail with a particular emphasis to those with no orthologs in <em>Phytophthora</em> genomes. This revealed the presence of genes involved in biosynthetic pathways which are known to be absent in <em>Phytophthora</em>. For example, it is known that <em>Phytophthora</em> species, but not <em>Aphanomyces</em> species, are thiamine auxotrophs. Accordingly, a gene coding a thiamine biosynthetic enzyme was identified, showing homology with a <em>S. parasitica</em> sequence <a href="#pone.0001723-TortoAlalibo2">[43]</a>. Another highly significant example concerns sterol metabolism. Among oomycetes, members of the Saprolegniale order can synthesize sterols <em>de novo</em> whereas species belonging to Pythiales are sterol auxotrophs. We found several genes encoding proteins showing high similarity to enzymes involved in sterol biosynthesis. This pathway is a major target for chemical inhibitors and a large number of fungicides have been developed acting mainly on a demethylation step catalysed by a cytochrome p450 enzyme of the CYP51 family. The identification of this class of proteins in <em>A. euteiches</em> will allow the development of new compounds acting on Saprolegniale pathogens. The ability of <em>A. euteiches</em> to synthesize its own sterols could be correlated with the fact that one elicitin-like sequence has been detected and no antigen was detected with anti elicitin antibodies when culture filtrates were probed by western blot analysis (data not shown). Elicitins are proteins found only in oomycetes where they were proposed to act as sterol carriers <a href="#pone.0001723-Mikes1">[67]</a>. In <em>Phytophthora</em>, elicitins represent a major class of secreted protein and are highly expressed. For example, 210 ESTs, corresponding to 16 elicitin genes were detected in the <em>P. sojae</em> EST collection <a href="#pone.0001723-TortoAlalibo1">[16]</a>. Moreover, elicitins can be perceived by the host cell, leading to a strong induction of defense reactions, and are thought to play a role in plant-pathogen specificity <a href="#pone.0001723-Ponchet1">[53]</a>. Thus, the low expression of elicitins in <em>A. euteiches</em> could be linked to its ability to synthesize its own sterols.</p>
<a id="article1.body1.sec4.p3" name="article1.body1.sec4.p3"></a><p>Classification of putative pathogenicity factors into 8 categories revealed other striking differences between <em>A. euteiches</em> and <em>Phytophthora</em> species. These include qualitative and quantitative differences for gene encoding hydrolases, effectors or proteins involved in adhesion, drug resistance and oxidative stress. Adhesion of pathogens to the surface of host tissues is a critical step in the establishment of oomycete pathogenesis <a href="#pone.0001723-Hardham1">[68]</a>. Although several adhesins produced by fungal pathogens have been characterized, molecular mechanisms involved in adhesion of oomycetes plant pathogens are still largely unknown. These include cyst germination proteins from <em>P. infestans</em> which share homology with mucins <a href="#pone.0001723-Gornhardt1">[37]</a>, <em>Phytophthora</em> surface proteins containing thrombospondin repeats <a href="#pone.0001723-Robold1">[40]</a> and <em>P. parasitica</em> CBEL <a href="#pone.0001723-VillalbaMateos1">[34]</a>. A functional approach demonstrated the essential role of CBEL in adhesion. Inhibition of CBEL gene expression in transgenic <em>P. parasitica</em> strains led to a dramatic reduction in adhesion of the mycelium to cellulosic substrates <a href="#pone.0001723-Gaulin3">[69]</a>. Several sequences showing similarity with <em>Phytophthora</em> adhesins were found in <em>A. euteiches</em> but with specific features. Of particular interest is a highly expressed family of proteins composed of a combination of cellulose binding domains (CBM_1) similar to those present in CBEL, and acidic domains found in the N-terminal part of the <em>P. infestans</em> cyst germination proteins. These proteins can represent a novel class of oomycete adhesins.</p>
<a id="article1.body1.sec4.p4" name="article1.body1.sec4.p4"></a><p>Analysis of the EST collection revealed expansion of gene families in <em>A. euteiches</em> such as genes encoding proteases. Proteases and protease inhibitors play an important role in the molecular dialogue which establishes between the pathogen and its host. In <em>Phytophthora</em>, several protease inhibitors have been shown to target plant proteases during infection <a href="#pone.0001723-Tian1">[49]</a>, <a href="#pone.0001723-Tian2">[50]</a>. Expansion of proteases gene families in <em>A. euteiches</em> indicates that these proteins are probably major players of <em>A. euteiches</em> pathogenicity. Thus, protein degradation and utilization is probably a major metabolic pathway related to pathogenesis in <em>A. euteiches</em>. Others genes showing expansion and diversification in <em>A. euteiches</em> comprise ABC transporters whose role in mediating resistance to plant toxins and fungicide has been suggested in <em>Phytophthora</em> <a href="#pone.0001723-Judelson1">[70]</a>. By contrast, some gene families which are well represented in <em>Phytophthora</em> were not detected in <em>A. euteiches</em>. This is the case for pectinases for which their role as pathogenicity factors has been demonstrated for several plant pathogens. The lack of ESTs showing homology to pectinases is correlated to the weak production of extracellular pectinolytic activity <em>in vitro</em> and the absence of <em>A. euteiches</em> growth on a medium containing pectin as sole carbon source. Recently, sequencing of a 192 bp genomic DNA fragment of <em>A. euteiches</em> revealed similarity with a <em>P. capsici</em> polygalacturonase <a href="#pone.0001723-Akamatsu1">[71]</a>. Thus it cannot be definitively concluded from our results that pectinase genes are not present in <em>A. euteiches</em>.</p>
<a id="article1.body1.sec4.p5" name="article1.body1.sec4.p5"></a><p>Another group of oomycete pathogenicity factors are proteins which are translocated into the cytoplasm of the host cell during infection. These gene products all contain an RxLR motif which has been shown to be essential for the effective transport of these effectors <a href="#pone.0001723-Whisson1">[18]</a>. While no clear homolog of effectors belonging to the RxLR class were identified in <em>A. euteiches</em>, their presence cannot be fully excluded. It has been shown that these proteins are under a strong positive selection <a href="#pone.0001723-Win1">[24]</a> and can thus escape <em>in silico</em> analyses. However, genes showing strong similarity with a second class of effectors belonging to the crinkler family were identified. Up to now, these proteins were found only in <em>Phytophthora</em> species where they represent a large superfamily with 40 genes in the genome of <em>P. sojae</em> <a href="#pone.0001723-Tyler1">[15]</a>. CRNs were issued from a large screen aiming at identifying extracellular proteins from <em>P. infestans</em> able to induce symptoms when expressed in plant tissues <a href="#pone.0001723-Torto1">[48]</a>. While the exact function of CRNs is still unclear, conservation of these sequences in all pathogenic oomycetes examined so far indicates that they probably play a specific role in oomycete pathogenesis.</p>
<a id="article1.body1.sec4.p6" name="article1.body1.sec4.p6"></a><p>Finally, comparison of <em>A. euteiches</em> sequences to protein sequences from tryponosomatids led to the identification of protein sequences with strong similarity to enzymes involved in synthesis of antioxidant compounds, glutathionylspermidine and trypanothione. Trypanothione has been identified as an essential redox intermediate and its role in defense against oxidative stress has been demonstrated in the human pathogens <em>Trypanosoma brucei</em> and <em>Leishmania donovani,</em> <a href="#pone.0001723-Krieger1">[72]</a>, <a href="#pone.0001723-Tovar1">[73]</a>. Since the oxidative burst is one of the earliest observable plant defense response against pathogen invasion (for review see <a href="#pone.0001723-Wojtaszek1">[74]</a>), the production of antioxidant compounds will constitute an efficient defense mechanism for the pathogen.</p>
<a id="article1.body1.sec4.p7" name="article1.body1.sec4.p7"></a><p>In conclusion this work greatly expands our knowledge of a genus which includes plant and animal pathogens and more generally illustrates the diversity which can exist among the oomycete. The repertoire of expressed genes described here will be used as source of candidate genes for functional studies aiming at identifying essential factors of <em>A. euteiches</em> pathogenicity.</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>We thank Didier Laborie, David Allouche, Jean-Marc Larre and all members of the Bioinformatic Platform of the Genopole (Toulouse, France) for computing technical support.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: EG. Performed the experiments: PW AC EG MM AB CJ. Analyzed the data: PW BD MM AB CM. Contributed reagents/materials/analysis tools: PW AC MM AB CJ CM. Wrote the paper: BD.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0001723-Delwich1" id="pone.0001723-Delwich1"></a>Delwich PA, Grau CR, Holub EB, Perry JB (1987) Characterization of <em>Aphanomyces euteiches</em> isolates recovered from alfalfa in Wisconsin. Plant Dis  71: 155–161.  <ul class="find" data-citedArticleID="1067362" data-doi="10.1094/pd-71-0155"><li><a href="http://dx.doi.org/10.1094/pd-71-0155" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Characterization+of+Aphanomyces+euteiches+isolates+recovered+from+alfalfa+in+Wisconsin." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Characterization+of+Aphanomyces+euteiches+isolates+recovered+from+alfalfa+in+Wisconsin.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0001723-Levenfors1" id="pone.0001723-Levenfors1"></a>Levenfors JP, Wikstrom M, Persson L, Gerhardson B (2003) Pathogenicity of <em>Aphanomyces</em> spp. from different leguminous crops in Sweden. Eur J Plant Pathol  109: 535–543.  <ul class="find" data-citedArticleID="1067410" data-doi="10.1023/a:1024711428760"><li><a href="http://dx.doi.org/10.1023/a:1024711428760" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Pathogenicity+of+Aphanomyces+spp.+from+different+leguminous+crops+in+Sweden." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Pathogenicity+of+Aphanomyces+spp.+from+different+leguminous+crops+in+Sweden.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0001723-Wicker1" id="pone.0001723-Wicker1"></a>Wicker E, Rouxel F (2001) Specific behaviour of French <em>Aphanomyces euteiches</em> Drechs. populations for virulence and aggressiveness on pea, related to isolates from Europe, America and New Zealand. Eur J Plant Pathol  107: 919–929.  <ul class="find" data-citedArticleID="1067472"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Specific+behaviour+of+French+Aphanomyces+euteiches+Drechs.+populations+for+virulence+and+aggressiveness+on+pea%2C+related+to+isolates+from+Europe%2C+America+and+New+Zealand.&amp;auth=&amp;atitle=Specific+behaviour+of+French+Aphanomyces+euteiches+Drechs.+populations+for+virulence+and+aggressiveness+on+pea%2C+related+to+isolates+from+Europe%2C+America+and+New+Zealand." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Specific+behaviour+of+French+Aphanomyces+euteiches+Drechs.+populations+for+virulence+and+aggressiveness+on+pea%2C+related+to+isolates+from+Europe%2C+America+and+New+Zealand." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Specific+behaviour+of+French+Aphanomyces+euteiches+Drechs.+populations+for+virulence+and+aggressiveness+on+pea%2C+related+to+isolates+from+Europe%2C+America+and+New+Zealand.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0001723-Mitchell1" id="pone.0001723-Mitchell1"></a>Mitchell JE, Yang CY (1966) Factors affecting growth and development of <em>Aphanomyces euteiches.</em> Phytopathology  56: 917–922.  <ul class="find" data-citedArticleID="1067420"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Factors+affecting+growth+and+development+of+Aphanomyces+euteiches.&amp;auth=&amp;atitle=Factors+affecting+growth+and+development+of+Aphanomyces+euteiches." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Factors+affecting+growth+and+development+of+Aphanomyces+euteiches." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Factors+affecting+growth+and+development+of+Aphanomyces+euteiches.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0001723-Larsson1" id="pone.0001723-Larsson1"></a>Larsson M (1994) Pathogenicity, morphology and isozyme variability among isolates of <em>Aphanomyces</em> spp. from weeds and various plants. Mycol Res  98: 231–240.  <ul class="find" data-citedArticleID="1067402" data-doi="10.1016/s0953-7562(09)80191-3"><li><a href="http://dx.doi.org/10.1016/s0953-7562(09)80191-3" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Pathogenicity%2C+morphology+and+isozyme+variability+among+isolates+of+Aphanomyces+spp.+from+weeds+and+various+plants." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Pathogenicity%2C+morphology+and+isozyme+variability+among+isolates+of+Aphanomyces+spp.+from+weeds+and+various+plants.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0001723-PiletNayel1" id="pone.0001723-PiletNayel1"></a>Pilet-Nayel ML, Muehlbauer FJ, McGee RJ, Kraft JM, Baranger A, et al.  (2005) Consistent quantitative trait loci in pea for partial resistance to <em>Aphanomyces euteiches</em> isolates from the United States and France. Phytopathology  95: 1287–1293.  <ul class="find" data-citedArticleID="1067428" data-doi="10.1094/phyto-95-1287"><li><a href="http://dx.doi.org/10.1094/phyto-95-1287" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Consistent+quantitative+trait+loci+in+pea+for+partial+resistance+to+Aphanomyces+euteiches+isolates+from+the+United+States+and+France." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Consistent+quantitative+trait+loci+in+pea+for+partial+resistance+to+Aphanomyces+euteiches+isolates+from+the+United+States+and+France.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0001723-Sderhll1" id="pone.0001723-Sderhll1"></a>Söderhäll K, Cerenius L (1999) The crayfish plague fungus: history and recent advances. Freshwater Crayfish  12: 11–35.  <ul class="find" data-citedArticleID="1067440" data-doi="10.1016/0044-8486(92)90133-6"><li><a href="http://dx.doi.org/10.1016/0044-8486(92)90133-6" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+crayfish+plague+fungus%3A+history+and+recent+advances." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+crayfish+plague+fungus%3A+history+and+recent+advances.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0001723-Blazer1" id="pone.0001723-Blazer1"></a>Blazer VS, Lilley JH, Schill WB, Kiryu Y, Densmore C, et al.  (2002) <em>Aphanomyces invadans</em> in Atlantic menhaden along the east coast of the United States. J Aquat Anim Health  14: 1–10.  <ul class="find" data-citedArticleID="1067350" data-doi="10.1577/1548-8667(2002)014<0001:aiiama&gt;2.0.co;2"><li><a href="http://dx.doi.org/10.1577/1548-8667(2002)014<0001:aiiama&gt;2.0.co;2" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Aphanomyces+invadans+in+Atlantic+menhaden+along+the+east+coast+of+the+United+States." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Aphanomyces+invadans+in+Atlantic+menhaden+along+the+east+coast+of+the+United+States.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0001723-Gaulin1" id="pone.0001723-Gaulin1"></a>Gaulin E, Jacquet C, Bottin A, Dumas B (2007) Root rot disease of legumes caused by <em>Aphanomyces euteiches.</em> Mol Plant Pathol  8: 539–548.  <ul class="find" data-citedArticleID="1067372" data-doi="10.1111/j.1364-3703.2007.00413.x"><li><a href="http://dx.doi.org/10.1111/j.1364-3703.2007.00413.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Root+rot+disease+of+legumes+caused+by+Aphanomyces+euteiches." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Root+rot+disease+of+legumes+caused+by+Aphanomyces+euteiches.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0001723-Martin1" id="pone.0001723-Martin1"></a>Martin HL (2003) Management of soilborne diseases of beetroot in Australia: a review. Aust J Exp Agri  43: 1281–1292.  <ul class="find" data-citedArticleID="1067414" data-doi="10.1071/ea02150"><li><a href="http://dx.doi.org/10.1071/ea02150" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Management+of+soilborne+diseases+of+beetroot+in+Australia%3A+a+review." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Management+of+soilborne+diseases+of+beetroot+in+Australia%3A+a+review.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0001723-Kamoun1" id="pone.0001723-Kamoun1"></a>Kamoun S (2001) Nonhost resistance to <em>Phytophthora</em>: novel prospects for a classical problem. Curr Opin Plant Biol  4: 295–300.  <ul class="find" data-citedArticleID="1067398" data-doi="10.1016/s1369-5266(00)00176-x"><li><a href="http://dx.doi.org/10.1016/s1369-5266(00)00176-x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Nonhost+resistance+to+Phytophthora%3A+novel+prospects+for+a+classical+problem." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Nonhost+resistance+to+Phytophthora%3A+novel+prospects+for+a+classical+problem.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0001723-Randall1" id="pone.0001723-Randall1"></a>Randall TA, Dwyer RA, Huitema E, Beyer K, Cvitanich C, et al.  (2005) Large-scale gene discovery in the oomycete <em>Phytophthora</em> infestans reveals likely components of phytopathogenicity shared with true fungi. Mol Plant Microbe Interact  18: 229–243.  <ul class="find" data-citedArticleID="1067434" data-doi="10.1094/mpmi-18-0229"><li><a href="http://dx.doi.org/10.1094/mpmi-18-0229" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Large-scale+gene+discovery+in+the+oomycete+Phytophthora+infestans+reveals+likely+components+of+phytopathogenicity+shared+with+true+fungi." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Large-scale+gene+discovery+in+the+oomycete+Phytophthora+infestans+reveals+likely+components+of+phytopathogenicity+shared+with+true+fungi.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0001723-Gajendran1" id="pone.0001723-Gajendran1"></a>Gajendran K, Gonzales MD, Farmer A, Archuleta E, Win J, et al.  (2006) <em>Phytophthora</em> functional genomics database (PFGD): functional genomics of <em>Phytophthora</em>-plant interactions. Nucleic Acids Res  34: 465–470.  <ul class="find" data-citedArticleID="1067370" data-doi="10.1093/nar/gkj119"><li><a href="http://dx.doi.org/10.1093/nar/gkj119" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Phytophthora+functional+genomics+database+%28PFGD%29%3A+functional+genomics+of+Phytophthora-plant+interactions." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Phytophthora+functional+genomics+database+%28PFGD%29%3A+functional+genomics+of+Phytophthora-plant+interactions.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0001723-Waugh1" id="pone.0001723-Waugh1"></a>Waugh M, Hraber P, Weller J, Wu Y, Chen G, et al.  (2000) The <em>Phytophthora</em> genome initiative database: informatics and analysis for distributed pathogenomic research. Nucleic Acids Res  28: 87–90.  <ul class="find" data-citedArticleID="1067468" data-doi="10.1093/nar/28.1.87"><li><a href="http://dx.doi.org/10.1093/nar/28.1.87" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+Phytophthora+genome+initiative+database%3A+informatics+and+analysis+for+distributed+pathogenomic+research." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+Phytophthora+genome+initiative+database%3A+informatics+and+analysis+for+distributed+pathogenomic+research.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0001723-Tyler1" id="pone.0001723-Tyler1"></a>Tyler BM, Tripathy S, Zhang X, Dehal P, Jiang RH, et al.  (2006) <em>Phytophthora</em> genome sequences uncover evolutionary origins and mechanisms of pathogenesis. Science  313: 1261–1266.  <ul class="find" data-citedArticleID="1067464" data-doi="10.1126/science.1128796"><li><a href="http://dx.doi.org/10.1126/science.1128796" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Phytophthora+genome+sequences+uncover+evolutionary+origins+and+mechanisms+of+pathogenesis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Phytophthora+genome+sequences+uncover+evolutionary+origins+and+mechanisms+of+pathogenesis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0001723-TortoAlalibo1" id="pone.0001723-TortoAlalibo1"></a>Torto-Alalibo TA, Tripathy S, Smith BM, Arredondo FD, Zhou L, et al.  (2007) Expressed sequence tags from <em>Phytophthora sojae</em> reveal genes specific to development and infection. Mol Plant Microbe Interact  20: 781–793.  <ul class="find" data-citedArticleID="1067458" data-doi="10.1094/mpmi-20-7-0781"><li><a href="http://dx.doi.org/10.1094/mpmi-20-7-0781" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Expressed+sequence+tags+from+Phytophthora+sojae+reveal+genes+specific+to+development+and+infection." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Expressed+sequence+tags+from+Phytophthora+sojae+reveal+genes+specific+to+development+and+infection.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0001723-Panabires1" id="pone.0001723-Panabires1"></a>Panabières F, Amselem J, Galiana E, Le Berre J (2005) Gene identification in the oomycete pathogen <em>Phytophthora parasitica</em> during <em>in vitro</em> vegetative growth through expressed sequence tags. Fungal Genet Biol  42: 611–623.  <ul class="find" data-citedArticleID="1067424" data-doi="10.1016/j.fgb.2005.03.002"><li><a href="http://dx.doi.org/10.1016/j.fgb.2005.03.002" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Gene+identification+in+the+oomycete+pathogen+Phytophthora+parasitica+during+in+vitro+vegetative+growth+through+expressed+sequence+tags." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Gene+identification+in+the+oomycete+pathogen+Phytophthora+parasitica+during+in+vitro+vegetative+growth+through+expressed+sequence+tags.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0001723-Whisson1" id="pone.0001723-Whisson1"></a>Whisson SC, Boevink PC, Moleleki L, Avrova AO, Morales JG, et al.  (2007) A translocation signal for delivery of oomycete effector proteins into host plant cells. Nature  450: 115–118.  <ul class="find" data-citedArticleID="1067470" data-doi="10.1038/nature06203"><li><a href="http://dx.doi.org/10.1038/nature06203" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+translocation+signal+for+delivery+of+oomycete+effector+proteins+into+host+plant+cells." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+translocation+signal+for+delivery+of+oomycete+effector+proteins+into+host+plant+cells.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0001723-Papavizas1" id="pone.0001723-Papavizas1"></a>Papavizas G, Davey C (1960) Some factors affecting growth of <em>Aphanomyces euteiches</em> in synthetic media. Amer J Bot  47: 758–765.  <ul class="find" data-citedArticleID="1067426" data-doi="10.2307/2439112"><li><a href="http://dx.doi.org/10.2307/2439112" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Some+factors+affecting+growth+of+Aphanomyces+euteiches+in+synthetic+media." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Some+factors+affecting+growth+of+Aphanomyces+euteiches+in+synthetic+media.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0001723-Madoui1" id="pone.0001723-Madoui1"></a>Madoui MA, Gaulin E, Mathé C, San Clemente H, Couloux A, et al.  (2007) AphanoDB: a genomic resource for <em>Aphanomyces</em> pathogens. BMC Genomics  8: 471.  <ul class="find" data-citedArticleID="1067412" data-doi="10.1186/1471-2164-8-471"><li><a href="http://dx.doi.org/10.1186/1471-2164-8-471" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=AphanoDB%3A+a+genomic+resource+for+Aphanomyces+pathogens." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22AphanoDB%3A+a+genomic+resource+for+Aphanomyces+pathogens.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0001723-Bendtsen1" id="pone.0001723-Bendtsen1"></a>Bendtsen JD, Nielsen H, von Heijne G, Brunak S (2004) Improved prediction of signal peptides: SignalP 3.0. J Mol Biol  340: 783–795.  <ul class="find" data-citedArticleID="1067346" data-doi="10.1016/j.jmb.2004.05.028"><li><a href="http://dx.doi.org/10.1016/j.jmb.2004.05.028" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Improved+prediction+of+signal+peptides%3A+SignalP+3.0." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Improved+prediction+of+signal+peptides%3A+SignalP+3.0.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0001723-Rice1" id="pone.0001723-Rice1"></a>Rice P, Longden I, Bleasby A (2000) EMBOSS: the European Molecular Biology Open Software Suite. Trends Genet  16: 276–277.  <ul class="find" data-citedArticleID="1067436" data-doi="10.1016/s0168-9525(00)02024-2"><li><a href="http://dx.doi.org/10.1016/s0168-9525(00)02024-2" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=EMBOSS%3A+the+European+Molecular+Biology+Open+Software+Suite." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22EMBOSS%3A+the+European+Molecular+Biology+Open+Software+Suite.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0001723-Eddy1" id="pone.0001723-Eddy1"></a>Eddy SR (1998) Profile hidden Markov models. Bioinformatics  14: 755–763.  <ul class="find" data-citedArticleID="1067364" data-doi="10.1093/bioinformatics/14.9.755"><li><a href="http://dx.doi.org/10.1093/bioinformatics/14.9.755" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Profile+hidden+Markov+models." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Profile+hidden+Markov+models.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0001723-Win1" id="pone.0001723-Win1"></a>Win J, Morgan W, Bos J, Krasileva KV, Cano LM, et al.  (2007) Adaptive evolution has targeted the C-terminal domain of the RXLR effectors of plant pathogenic oomycetes. Plant Cell  19: 2349–2369.  <ul class="find" data-citedArticleID="1067474" data-doi="10.1105/tpc.107.051037"><li><a href="http://dx.doi.org/10.1105/tpc.107.051037" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Adaptive+evolution+has+targeted+the+C-terminal+domain+of+the+RXLR+effectors+of+plant+pathogenic+oomycetes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Adaptive+evolution+has+targeted+the+C-terminal+domain+of+the+RXLR+effectors+of+plant+pathogenic+oomycetes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0001723-Hiller1" id="pone.0001723-Hiller1"></a>Hiller NL, Bhattacharjee S, van Ooij C, Liolios K, Harrison T, et al.  (2004) A host-targeting signal in virulence proteins reveals a secretome in malarial infection. Science  306: 1934–1937.  <ul class="find" data-citedArticleID="1067388" data-doi="10.1126/science.1102737"><li><a href="http://dx.doi.org/10.1126/science.1102737" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+host-targeting+signal+in+virulence+proteins+reveals+a+secretome+in+malarial+infection." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+host-targeting+signal+in+virulence+proteins+reveals+a+secretome+in+malarial+infection.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0001723-Thompson1" id="pone.0001723-Thompson1"></a>Thompson JD, Higgins DG, Gibson TJ (1994) CLUSTAL W: improving the sensitivity of progressive multiple sequence alignment through sequence weighting, position-specific gap penalties and weight matrix choice. Nucleic Acids Res  22: 4673–4680.  <ul class="find" data-citedArticleID="1067448" data-doi="10.1093/nar/22.22.4673"><li><a href="http://dx.doi.org/10.1093/nar/22.22.4673" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=CLUSTAL+W%3A+improving+the+sensitivity+of+progressive+multiple+sequence+alignment+through+sequence+weighting%2C+position-specific+gap+penalties+and+weight+matrix+choice." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22CLUSTAL+W%3A+improving+the+sensitivity+of+progressive+multiple+sequence+alignment+through+sequence+weighting%2C+position-specific+gap+penalties+and+weight+matrix+choice.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0001723-Crooks1" id="pone.0001723-Crooks1"></a>Crooks GE, Hon G, Chandonia JM, Brenner SE (2004) WebLogo: A sequence logo generator. Genome Res  14: 1188–1190.  <ul class="find" data-citedArticleID="1067360" data-doi="10.1101/gr.849004"><li><a href="http://dx.doi.org/10.1101/gr.849004" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=WebLogo%3A+A+sequence+logo+generator." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22WebLogo%3A+A+sequence+logo+generator.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0001723-Altschul1" id="pone.0001723-Altschul1"></a>Altschul SF, Madden TL, Schaffer AA, Zhang J, Zhang Z, et al.  (1997) Gapped BLAST and PSI-BLAST: a new generation of protein database search programs. Nucleic Acids Res  25: 3389–3402.  <ul class="find" data-citedArticleID="1067338" data-doi="10.1093/nar/25.17.3389"><li><a href="http://dx.doi.org/10.1093/nar/25.17.3389" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Gapped+BLAST+and+PSI-BLAST%3A+a+new+generation+of+protein+database+search+programs." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Gapped+BLAST+and+PSI-BLAST%3A+a+new+generation+of+protein+database+search+programs.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0001723-Bateman1" id="pone.0001723-Bateman1"></a>Bateman A, Coin L, Durbin R, Finn RD, Hollich V, et al.  (2004) The Pfam protein families database. Nucleic Acids Res  32: D138–141.  <ul class="find" data-citedArticleID="1067344" data-doi="10.1093/nar/gkh121"><li><a href="http://dx.doi.org/10.1093/nar/gkh121" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+Pfam+protein+families+database." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+Pfam+protein+families+database.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0001723-Susko1" id="pone.0001723-Susko1"></a>Susko E, Roger AJ (2004) Estimating and comparing the rates of gene discovery and expressed sequence tag (EST) frequencies in EST surveys. Bioinformatics  20: 2279–2287.  <ul class="find" data-citedArticleID="1067446" data-doi="10.1093/bioinformatics/bth239"><li><a href="http://dx.doi.org/10.1093/bioinformatics/bth239" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Estimating+and+comparing+the+rates+of+gene+discovery+and+expressed+sequence+tag+%28EST%29+frequencies+in+EST+surveys." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Estimating+and+comparing+the+rates+of+gene+discovery+and+expressed+sequence+tag+%28EST%29+frequencies+in+EST+surveys.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0001723-Benjamini1" id="pone.0001723-Benjamini1"></a>Benjamini Y, Hochberg Y (1995) Controlling the false discovery rate: a practical and powerful approach to multiple testing. J R Stat Soc Ser B  57: 289–300.  <ul class="find" data-citedArticleID="1067348"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Controlling+the+false+discovery+rate%3A+a+practical+and+powerful+approach+to+multiple+testing.&amp;auth=&amp;atitle=Controlling+the+false+discovery+rate%3A+a+practical+and+powerful+approach+to+multiple+testing." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Controlling+the+false+discovery+rate%3A+a+practical+and+powerful+approach+to+multiple+testing." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Controlling+the+false+discovery+rate%3A+a+practical+and+powerful+approach+to+multiple+testing.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0001723-Inoue1" id="pone.0001723-Inoue1"></a>Inoue I, Namiki F, Tsuge T (2002) Plant colonization by the vascular wilt fungus <em>Fusarium oxysporum</em> requires <em>FOW1</em>, a gene encoding a mitochondrial protein. Plant Cell  14: 1869–1883.  <ul class="find" data-citedArticleID="1067390" data-doi="10.1105/tpc.002576"><li><a href="http://dx.doi.org/10.1105/tpc.002576" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Plant+colonization+by+the+vascular+wilt+fungus+Fusarium+oxysporum+requires+FOW1%2C+a+gene+encoding+a+mitochondrial+protein." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Plant+colonization+by+the+vascular+wilt+fungus+Fusarium+oxysporum+requires+FOW1%2C+a+gene+encoding+a+mitochondrial+protein.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0001723-SjalonDelmas1" id="pone.0001723-SjalonDelmas1"></a>Séjalon-Delmas N, Villalba Mateos F, Bottin A, Rickauer M, Dargent R, et al.  (1997) Purification, elicitor activity, and cell wall localization of a glycoprotein from <em>Phytophthora parasitica</em> var. <em>nicotianae</em>, a fungal pathogen of tobacco. Phytopathology  87: 899–909.  <ul class="find" data-citedArticleID="1067442" data-doi="10.1094/phyto.1997.87.9.899"><li><a href="http://dx.doi.org/10.1094/phyto.1997.87.9.899" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Purification%2C+elicitor+activity%2C+and+cell+wall+localization+of+a+glycoprotein+from+Phytophthora+parasitica+var.+nicotianae%2C+a+fungal+pathogen+of+tobacco." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Purification%2C+elicitor+activity%2C+and+cell+wall+localization+of+a+glycoprotein+from+Phytophthora+parasitica+var.+nicotianae%2C+a+fungal+pathogen+of+tobacco.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0001723-VillalbaMateos1" id="pone.0001723-VillalbaMateos1"></a>Villalba-Mateos F, Rickauer M, Esquerré-Tugayé MT (1997) Cloning and characterization of a cDNA encoding an elicitor of <em>Phytophthora parasitica</em> var. <em>nicotianae</em> that shows cellulose-binding and lectin-like activities. Mol Plant-Microbe Interact  10: 1045–1053.  <ul class="find" data-citedArticleID="1067466" data-doi="10.1094/mpmi.1997.10.9.1045"><li><a href="http://dx.doi.org/10.1094/mpmi.1997.10.9.1045" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Cloning+and+characterization+of+a+cDNA+encoding+an+elicitor+of+Phytophthora+parasitica+var.+nicotianae+that+shows+cellulose-binding+and+lectin-like+activities." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Cloning+and+characterization+of+a+cDNA+encoding+an+elicitor+of+Phytophthora+parasitica+var.+nicotianae+that+shows+cellulose-binding+and+lectin-like+activities.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0001723-Gaulin2" id="pone.0001723-Gaulin2"></a>Gaulin E, Dramé N, Lafitte C, Torto-Alalibo T, Martinez Y, et al.  (2006) Cellulose binding domains of a <em>Phytophthora</em> cell wall protein are novel pathogen-associated molecular patterns. Plant Cell  18: 1766–1777.  <ul class="find" data-citedArticleID="1067374" data-doi="10.1105/tpc.105.038687"><li><a href="http://dx.doi.org/10.1105/tpc.105.038687" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Cellulose+binding+domains+of+a+Phytophthora+cell+wall+protein+are+novel+pathogen-associated+molecular+patterns." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Cellulose+binding+domains+of+a+Phytophthora+cell+wall+protein+are+novel+pathogen-associated+molecular+patterns.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">36.
              </span><a name="pone.0001723-Tordai1" id="pone.0001723-Tordai1"></a>Tordai H, Banyai L, Patthy L (1999) The PAN module: the N-terminal domains of plasminogen and hepathocyte growth factor are homologous with the apple domains of the prekallikrein family and with a novel domain found in numerous nematode proteins. FEBS Lett  461: 63–67.  <ul class="find" data-citedArticleID="1067454" data-doi="10.1016/s0014-5793(99)01416-7"><li><a href="http://dx.doi.org/10.1016/s0014-5793(99)01416-7" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+PAN+module%3A+the+N-terminal+domains+of+plasminogen+and+hepathocyte+growth+factor+are+homologous+with+the+apple+domains+of+the+prekallikrein+family+and+with+a+novel+domain+found+in+numerous+nematode+proteins." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+PAN+module%3A+the+N-terminal+domains+of+plasminogen+and+hepathocyte+growth+factor+are+homologous+with+the+apple+domains+of+the+prekallikrein+family+and+with+a+novel+domain+found+in+numerous+nematode+proteins.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">37.
              </span><a name="pone.0001723-Gornhardt1" id="pone.0001723-Gornhardt1"></a>Gornhardt B, Rouhara I, Schmelzer E (2000) Cyst germination proteins of the potato pathogen <em>Phytophthora infestans</em> share homology with human mucins. Mol Plant Microbe Interact  13: 32–42.  <ul class="find" data-citedArticleID="1067378" data-doi="10.1094/mpmi.2000.13.1.32"><li><a href="http://dx.doi.org/10.1094/mpmi.2000.13.1.32" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Cyst+germination+proteins+of+the+potato+pathogen+Phytophthora+infestans+share+homology+with+human+mucins." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Cyst+germination+proteins+of+the+potato+pathogen+Phytophthora+infestans+share+homology+with+human+mucins.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">38.
              </span><a name="pone.0001723-Brecht1" id="pone.0001723-Brecht1"></a>Brecht S, Carruthers V, Ferguson D, Giddings O, Wang G, et al.  (2001) The toxoplasma micronemal protein MIC4 is an adhesin composed of six conserved apple domains. J Biol Chem  276: 4119–4127.  <ul class="find" data-citedArticleID="1067354" data-doi="10.1074/jbc.m008294200"><li><a href="http://dx.doi.org/10.1074/jbc.m008294200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+toxoplasma+micronemal+protein+MIC4+is+an+adhesin+composed+of+six+conserved+apple+domains." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+toxoplasma+micronemal+protein+MIC4+is+an+adhesin+composed+of+six+conserved+apple+domains.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">39.
              </span><a name="pone.0001723-Bai1" id="pone.0001723-Bai1"></a>Bai T, Becker M, Gupta A, Strike P, Murphy VJ, et al.  (2005) Structure of AMA1 from <em>Plasmodium falciparum</em> reveals a clustering of polymorphisms that surround a conserved hydrophobic pocket. Proc Natl Acad Sci USA  102: 12736–12741.  <ul class="find" data-citedArticleID="1067342" data-doi="10.1073/pnas.0501808102"><li><a href="http://dx.doi.org/10.1073/pnas.0501808102" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Structure+of+AMA1+from+Plasmodium+falciparum+reveals+a+clustering+of+polymorphisms+that+surround+a+conserved+hydrophobic+pocket." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Structure+of+AMA1+from+Plasmodium+falciparum+reveals+a+clustering+of+polymorphisms+that+surround+a+conserved+hydrophobic+pocket.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">40.
              </span><a name="pone.0001723-Robold1" id="pone.0001723-Robold1"></a>Robold AV, Hardham AR (2005) During attachment <em>Phytophthora</em> spores secrete proteins containing thrombospondin type 1 repeats. Curr Genet  47: 307–315.  <ul class="find" data-citedArticleID="1067438" data-doi="10.1007/s00294-004-0559-8"><li><a href="http://dx.doi.org/10.1007/s00294-004-0559-8" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=During+attachment+Phytophthora+spores+secrete+proteins+containing+thrombospondin+type+1+repeats." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22During+attachment+Phytophthora+spores+secrete+proteins+containing+thrombospondin+type+1+repeats.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">41.
              </span><a name="pone.0001723-Anantharaman1" id="pone.0001723-Anantharaman1"></a>Anantharaman V, Iyer LM, Balaji S, Aravind L (2007) Adhesion molecules and other secreted host-interaction determinants in Apicomplexa: insights from comparative genomics. Int Rev Cytol  262: 1–74.  <ul class="find" data-citedArticleID="1067340" data-doi="10.1016/s0074-7696(07)62001-4"><li><a href="http://dx.doi.org/10.1016/s0074-7696(07)62001-4" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Adhesion+molecules+and+other+secreted+host-interaction+determinants+in+Apicomplexa%3A+insights+from+comparative+genomics." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Adhesion+molecules+and+other+secreted+host-interaction+determinants+in+Apicomplexa%3A+insights+from+comparative+genomics.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">42.
              </span><a name="pone.0001723-Henrissat1" id="pone.0001723-Henrissat1"></a>Henrissat B, Davies G (1997) Structural and sequence-based classification of glycoside hydrolases. Curr Opin Struct Biol  7: 637–644.  <ul class="find" data-citedArticleID="1067386" data-doi="10.1016/s0959-440x(97)80072-3"><li><a href="http://dx.doi.org/10.1016/s0959-440x(97)80072-3" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Structural+and+sequence-based+classification+of+glycoside+hydrolases." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Structural+and+sequence-based+classification+of+glycoside+hydrolases.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">43.
              </span><a name="pone.0001723-TortoAlalibo2" id="pone.0001723-TortoAlalibo2"></a>Torto-Alalibo T, Tian M, Gajendran K, Waugh M, van West P, et al.  (2005) Expressed sequence tags from the oomycete fish pathogen <em>Saprolegnia parasitica</em> reveal putative virulence factors. BMC Microbiol  2: 5–46.  <ul class="find" data-citedArticleID="1067460" data-doi="10.1186/1471-2180-5-46"><li><a href="http://dx.doi.org/10.1186/1471-2180-5-46" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Expressed+sequence+tags+from+the+oomycete+fish+pathogen+Saprolegnia+parasitica+reveal+putative+virulence+factors." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Expressed+sequence+tags+from+the+oomycete+fish+pathogen+Saprolegnia+parasitica+reveal+putative+virulence+factors.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">44.
              </span><a name="pone.0001723-McLeod1" id="pone.0001723-McLeod1"></a>McLeod A, Smart CD, Fry WE (2003) Characterization of 1,3-β-glucanase and 1,3;1,4-β-glucanase genes from <em>Phytophthora infestans.</em> Fungal Genet Biol  38: 250–263.  <ul class="find" data-citedArticleID="1067416" data-doi="10.1016/s1087-1845(02)00523-6"><li><a href="http://dx.doi.org/10.1016/s1087-1845(02)00523-6" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Characterization+of+1%2C3-%CE%B2-glucanase+and+1%2C3%3B1%2C4-%CE%B2-glucanase+genes+from+Phytophthora+infestans." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Characterization+of+1%2C3-%CE%B2-glucanase+and+1%2C3%3B1%2C4-%CE%B2-glucanase+genes+from+Phytophthora+infestans.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">45.
              </span><a name="pone.0001723-Costanzo1" id="pone.0001723-Costanzo1"></a>Costanzo S, Ospina-Giraldo MD, Deahl KL, Baker CJ, Jones RW (2006) Gene duplication event in family 12 glycosyl hydrolase from <em>Phytophthora spp</em>. Fungal Genet Biol  43: 707–714.  <ul class="find" data-citedArticleID="1067358" data-doi="10.1016/j.fgb.2006.04.006"><li><a href="http://dx.doi.org/10.1016/j.fgb.2006.04.006" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Gene+duplication+event+in+family+12+glycosyl+hydrolase+from+Phytophthora+spp." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Gene+duplication+event+in+family+12+glycosyl+hydrolase+from+Phytophthora+spp.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">46.
              </span><a name="pone.0001723-Gotesson1" id="pone.0001723-Gotesson1"></a>Gotesson A, Marshall JS, Jones DA, Hardham AR (2002) Characterization and evolutionary analysis of a large polygalacturonase gene family in the oomycete plant pathogen <em>Phytophthora cinnamomi.</em> Mol Plant Microbe Interact  15: 907–921.  <ul class="find" data-citedArticleID="1067380" data-doi="10.1094/mpmi.2002.15.9.907"><li><a href="http://dx.doi.org/10.1094/mpmi.2002.15.9.907" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Characterization+and+evolutionary+analysis+of+a+large+polygalacturonase+gene+family+in+the+oomycete+plant+pathogen+Phytophthora+cinnamomi." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Characterization+and+evolutionary+analysis+of+a+large+polygalacturonase+gene+family+in+the+oomycete+plant+pathogen+Phytophthora+cinnamomi.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">47.
              </span><a name="pone.0001723-Yan1" id="pone.0001723-Yan1"></a>Yan HZ, Liou RF (2005) Cloning and analysis of <em>pppg</em>1, an inducible endopolygalacturonase gene from the oomycete plant pathogen <em>Phytophthora parasitica.</em> Fungal Genet Biol  42: 339–350.  <ul class="find" data-citedArticleID="1067478" data-doi="10.1016/j.fgb.2005.01.003"><li><a href="http://dx.doi.org/10.1016/j.fgb.2005.01.003" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Cloning+and+analysis+of+pppg1%2C+an+inducible+endopolygalacturonase+gene+from+the+oomycete+plant+pathogen+Phytophthora+parasitica." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Cloning+and+analysis+of+pppg1%2C+an+inducible+endopolygalacturonase+gene+from+the+oomycete+plant+pathogen+Phytophthora+parasitica.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">48.
              </span><a name="pone.0001723-Torto1" id="pone.0001723-Torto1"></a>Torto TA, Li S, Styer A, Huitema E, Testa A, et al.  (2003) EST mining and functional expression assays identify extracellular effector proteins from the plant pathogen <em>Phytophthora.</em> Genome Res  13: 1675–1685.  <ul class="find" data-citedArticleID="1067456" data-doi="10.1101/gr.910003"><li><a href="http://dx.doi.org/10.1101/gr.910003" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=EST+mining+and+functional+expression+assays+identify+extracellular+effector+proteins+from+the+plant+pathogen+Phytophthora." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22EST+mining+and+functional+expression+assays+identify+extracellular+effector+proteins+from+the+plant+pathogen+Phytophthora.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">49.
              </span><a name="pone.0001723-Tian1" id="pone.0001723-Tian1"></a>Tian M, Huitema E, Da Cunha L, Torto-Alalibo T, Kamoun S (2004) A Kazal-like extracellular serine protease inhibitor from <em>Phytophthora infestans</em> targets the tomato pathogenesis-related protease P69B. J Biol Chem  279: 26370–26375.  <ul class="find" data-citedArticleID="1067450" data-doi="10.1074/jbc.m400941200"><li><a href="http://dx.doi.org/10.1074/jbc.m400941200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+Kazal-like+extracellular+serine+protease+inhibitor+from+Phytophthora+infestans+targets+the+tomato+pathogenesis-related+protease+P69B." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+Kazal-like+extracellular+serine+protease+inhibitor+from+Phytophthora+infestans+targets+the+tomato+pathogenesis-related+protease+P69B.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">50.
              </span><a name="pone.0001723-Tian2" id="pone.0001723-Tian2"></a>Tian M, Win J, Song J, van der Hoorn R, van der Knaap E, et al.  (2007) A <em>Phytophthora infestans</em> cystatin-like protein targets a novel tomato papain-like apoplastic protease. Plant Physiol  143: 364–377.  <ul class="find" data-citedArticleID="1067452" data-doi="10.1104/pp.106.090050"><li><a href="http://dx.doi.org/10.1104/pp.106.090050" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+Phytophthora+infestans+cystatin-like+protein+targets+a+novel+tomato+papain-like+apoplastic+protease." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+Phytophthora+infestans+cystatin-like+protein+targets+a+novel+tomato+papain-like+apoplastic+protease.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">51.
              </span><a name="pone.0001723-Fellbrich1" id="pone.0001723-Fellbrich1"></a>Fellbrich G, Romanski A, Varet A, Blume B, Brunner F, et al.  (2002) NPP1, a <em>Phytophthora</em>-associated trigger of plant defense in parsley and <em>Arabidopsis.</em> Plant J  32: 375–390.  <ul class="find" data-citedArticleID="1067368" data-doi="10.1046/j.1365-313x.2002.01454.x"><li><a href="http://dx.doi.org/10.1046/j.1365-313x.2002.01454.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=NPP1%2C+a+Phytophthora-associated+trigger+of+plant+defense+in+parsley+and+Arabidopsis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22NPP1%2C+a+Phytophthora-associated+trigger+of+plant+defense+in+parsley+and+Arabidopsis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">52.
              </span><a name="pone.0001723-Qutob1" id="pone.0001723-Qutob1"></a>Qutob D, Kamoun S, Gijzen M (2002) Expression of a <em>Phytophthora sojae</em> necrosis-inducing protein occurs during transition from biotrophy to necrotrophy. Plant J  32: 361–373.  <ul class="find" data-citedArticleID="1067432" data-doi="10.1046/j.1365-313x.2002.01439.x"><li><a href="http://dx.doi.org/10.1046/j.1365-313x.2002.01439.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Expression+of+a+Phytophthora+sojae+necrosis-inducing+protein+occurs+during+transition+from+biotrophy+to+necrotrophy." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Expression+of+a+Phytophthora+sojae+necrosis-inducing+protein+occurs+during+transition+from+biotrophy+to+necrotrophy.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">53.
              </span><a name="pone.0001723-Ponchet1" id="pone.0001723-Ponchet1"></a>Ponchet M, Panabières F, Milat ML, Mikes V, Montillet JL, et al.  (1999) Are elicitins cryptograms in plant-Oomycete communications? Cell Mol Life Sci  56: 1020–1047.  <ul class="find" data-citedArticleID="1067430" data-doi="10.1007/s000180050491"><li><a href="http://dx.doi.org/10.1007/s000180050491" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Are+elicitins+cryptograms+in+plant-Oomycete+communications%3F" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Are+elicitins+cryptograms+in+plant-Oomycete+communications%3F%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">54.
              </span><a name="pone.0001723-Jiang1" id="pone.0001723-Jiang1"></a>Jiang RH, Tyler BM, Whisson SC, Hardham AR, Govers F (2006) Ancient origin of elicitin gene clusters in <em>Phytophthora</em> genomes. Mol Biol Evol  23: 338–351.  <ul class="find" data-citedArticleID="1067392" data-doi="10.1093/molbev/msj039"><li><a href="http://dx.doi.org/10.1093/molbev/msj039" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Ancient+origin+of+elicitin+gene+clusters+in+Phytophthora+genomes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Ancient+origin+of+elicitin+gene+clusters+in+Phytophthora+genomes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">55.
              </span><a name="pone.0001723-Stacey1" id="pone.0001723-Stacey1"></a>Stacey G, Koh S, Granger C, Becker JM (2002) Peptide transport in plants. Trends Plant Sci  7: 257–263.  <ul class="find" data-citedArticleID="1067444" data-doi="10.1016/s1360-1385(02)02249-5"><li><a href="http://dx.doi.org/10.1016/s1360-1385(02)02249-5" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Peptide+transport+in+plants." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Peptide+transport+in+plants.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">56.
              </span><a name="pone.0001723-Hauser1" id="pone.0001723-Hauser1"></a>Hauser M, Narita V, Donhardt AM, Naider F, Becker JM (2001) Multiplicity and regulation of genes encoding peptide transporters in <em>Saccharomyces cerevisiae.</em> Mol Membr Biol  18: 105–112.  <ul class="find" data-citedArticleID="1067384" data-doi="10.1080/09687680010029374"><li><a href="http://dx.doi.org/10.1080/09687680010029374" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Multiplicity+and+regulation+of+genes+encoding+peptide+transporters+in+Saccharomyces+cerevisiae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Multiplicity+and+regulation+of+genes+encoding+peptide+transporters+in+Saccharomyces+cerevisiae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">57.
              </span><a name="pone.0001723-LeJean1" id="pone.0001723-LeJean1"></a>Le Jean M, Schikora A, Mari S, Briat JF, Curie C (2005) A loss-of-function mutation in <em>AtYSL</em>1 reveals its role in iron and nicotianamine seed loading. Plant J  44: 769–782.  <ul class="find" data-citedArticleID="1067406" data-doi="10.1111/j.1365-313x.2005.02569.x"><li><a href="http://dx.doi.org/10.1111/j.1365-313x.2005.02569.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+loss-of-function+mutation+in+AtYSL1+reveals+its+role+in+iron+and+nicotianamine+seed+loading." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+loss-of-function+mutation+in+AtYSL1+reveals+its+role+in+iron+and+nicotianamine+seed+loading.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">58.
              </span><a name="pone.0001723-Yen1" id="pone.0001723-Yen1"></a>Yen MR, Tseng YH, Saier MH Jr (2001) <em>Maize Yellow Stripe</em>1, an iron-phytosiderophore uptake transporter, is a member of the oligopeptide transporter (OPT) family. Microbiology  147: 2881–2883.  <ul class="find" data-citedArticleID="1067480"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Maize+Yellow+Stripe1%2C+an+iron-phytosiderophore+uptake+transporter%2C+is+a+member+of+the+oligopeptide+transporter+%28OPT%29+family.&amp;auth=&amp;atitle=Maize+Yellow+Stripe1%2C+an+iron-phytosiderophore+uptake+transporter%2C+is+a+member+of+the+oligopeptide+transporter+%28OPT%29+family." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Maize+Yellow+Stripe1%2C+an+iron-phytosiderophore+uptake+transporter%2C+is+a+member+of+the+oligopeptide+transporter+%28OPT%29+family." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Maize+Yellow+Stripe1%2C+an+iron-phytosiderophore+uptake+transporter%2C+is+a+member+of+the+oligopeptide+transporter+%28OPT%29+family.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">59.
              </span><a name="pone.0001723-Fairlamb1" id="pone.0001723-Fairlamb1"></a>Fairlamb AH, Blackburn P, Ulrich P, Chait BT, Cerami A (1985) Trypanothione: a novel bis(glutathionyl)spermidine cofactor for glutathione reductase in trypanosomatids. Science  227: 1485–1487.  <ul class="find" data-citedArticleID="1067366" data-doi="10.1126/science.3883489"><li><a href="http://dx.doi.org/10.1126/science.3883489" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Trypanothione%3A+a+novel+bis%28glutathionyl%29spermidine+cofactor+for+glutathione+reductase+in+trypanosomatids." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Trypanothione%3A+a+novel+bis%28glutathionyl%29spermidine+cofactor+for+glutathione+reductase+in+trypanosomatids.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">60.
              </span><a name="pone.0001723-Oza1" id="pone.0001723-Oza1"></a>Oza SL, Tetaud E, Ariyanayagam MR, Warnon SS, Fairlamb AH (2002) A single enzyme catalyses formation of trypanothione from glutathione and spermidine in <em>Trypanosoma cruzi.</em> J Biol Chem  277: 35853–35861.  <ul class="find" data-citedArticleID="1067422" data-doi="10.1074/jbc.m204403200"><li><a href="http://dx.doi.org/10.1074/jbc.m204403200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+single+enzyme+catalyses+formation+of+trypanothione+from+glutathione+and+spermidine+in+Trypanosoma+cruzi." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+single+enzyme+catalyses+formation+of+trypanothione+from+glutathione+and+spermidine+in+Trypanosoma+cruzi.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">61.
              </span><a name="pone.0001723-Comini1" id="pone.0001723-Comini1"></a>Comini M, Menge U, Wissing J, Flohe L (2005) Trypanothione synthesis in <em>Crithidia</em> revisited. J Biol Chem  280: 6850–6860.  <ul class="find" data-citedArticleID="1067356" data-doi="10.1074/jbc.m404486200"><li><a href="http://dx.doi.org/10.1074/jbc.m404486200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Trypanothione+synthesis+in+Crithidia+revisited." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Trypanothione+synthesis+in+Crithidia+revisited.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">62.
              </span><a name="pone.0001723-Bollinger1" id="pone.0001723-Bollinger1"></a>Bollinger JM Jr, Kwon DS, Huisman GW, Kolter R, Walsh CT (1995) Glutathionylspermidine metabolism in <em>Escherichia coli</em>. Purification, cloning, overproduction, and characterization of a bifunctional glutathionylspermidine synthetase/amidase. J Biol Chem  270: 14031–14041.  <ul class="find" data-citedArticleID="1067352" data-doi="10.1074/jbc.270.23.14031"><li><a href="http://dx.doi.org/10.1074/jbc.270.23.14031" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Glutathionylspermidine+metabolism+in+Escherichia+coli.+Purification%2C+cloning%2C+overproduction%2C+and+characterization+of+a+bifunctional+glutathionylspermidine+synthetase%2Famidase." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Glutathionylspermidine+metabolism+in+Escherichia+coli.+Purification%2C+cloning%2C+overproduction%2C+and+characterization+of+a+bifunctional+glutathionylspermidine+synthetase%2Famidase.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">63.
              </span><a name="pone.0001723-Alphey1" id="pone.0001723-Alphey1"></a>Alphey MS, Leonard GA, Gourley DG, Tetaud E, Fairlamb AH, et al.  (1999) The high resolution crystal structure of recombinant <em>Crithidia fasciculata</em> tryparedoxin-I. J Biol Chem  274: 25613–25622.  <ul class="find" data-citedArticleID="1067336" data-doi="10.1074/jbc.274.36.25613"><li><a href="http://dx.doi.org/10.1074/jbc.274.36.25613" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+high+resolution+crystal+structure+of+recombinant+Crithidia+fasciculata+tryparedoxin-I." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+high+resolution+crystal+structure+of+recombinant+Crithidia+fasciculata+tryparedoxin-I.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">64.
              </span><a name="pone.0001723-Lee1" id="pone.0001723-Lee1"></a>Lee BN, Kroken S, Chou DY, Robbertse B, Yoder OC, et al.  (2005) Functional analysis of all nonribosomal peptide synthetases in <em>Cochliobolus heterostrophus</em> reveals a factor, NPS6, involved in virulence and resistance to oxidative stress. Eukaryot Cell  4: 545–555.  <ul class="find" data-citedArticleID="1067408" data-doi="10.1128/ec.4.3.545-555.2005"><li><a href="http://dx.doi.org/10.1128/ec.4.3.545-555.2005" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Functional+analysis+of+all+nonribosomal+peptide+synthetases+in+Cochliobolus+heterostrophus+reveals+a+factor%2C+NPS6%2C+involved+in+virulence+and+resistance+to+oxidative+stress." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Functional+analysis+of+all+nonribosomal+peptide+synthetases+in+Cochliobolus+heterostrophus+reveals+a+factor%2C+NPS6%2C+involved+in+virulence+and+resistance+to+oxidative+stress.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">65.
              </span><a name="pone.0001723-Johnson1" id="pone.0001723-Johnson1"></a>Johnson RD, Johnson L, Itoh Y, Kodama M, Otani H, et al.  (2000) Cloning and characterization of a cyclic peptide synthetase gene from <em>Alternaria alternata</em> apple pathotype whose product is involved in AM-toxin synthesis and pathogenicity. Mol Plant Microbe Interact  13: 742–753.  <ul class="find" data-citedArticleID="1067394" data-doi="10.1094/mpmi.2000.13.7.742"><li><a href="http://dx.doi.org/10.1094/mpmi.2000.13.7.742" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Cloning+and+characterization+of+a+cyclic+peptide+synthetase+gene+from+Alternaria+alternata+apple+pathotype+whose+product+is+involved+in+AM-toxin+synthesis+and+pathogenicity." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Cloning+and+characterization+of+a+cyclic+peptide+synthetase+gene+from+Alternaria+alternata+apple+pathotype+whose+product+is+involved+in+AM-toxin+synthesis+and+pathogenicity.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">66.
              </span><a name="pone.0001723-LeBerre1" id="pone.0001723-LeBerre1"></a>Le Berre JY, Engler G, Panabieres F (2007) Exploration of the late stages of the tomato-<em>Phytophthora parasitica</em> interactions through histological analysis and generation of expressed sequence tags. New Phytol 1–11.  <ul class="find" data-citedArticleID="1067404" data-doi="10.1111/j.1469-8137.2007.02269.x"><li><a href="http://dx.doi.org/10.1111/j.1469-8137.2007.02269.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Exploration+of+the+late+stages+of+the+tomato-Phytophthora+parasitica+interactions+through+histological+analysis+and+generation+of+expressed+sequence+tags." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Exploration+of+the+late+stages+of+the+tomato-Phytophthora+parasitica+interactions+through+histological+analysis+and+generation+of+expressed+sequence+tags.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">67.
              </span><a name="pone.0001723-Mikes1" id="pone.0001723-Mikes1"></a>Mikes V, Milat ML, Ponchet M, Panabieres F, Ricci P, et al.  (1998) Elicitins, proteinaceous elicitors of plant defense, are a new class of sterol carrier proteins. Biochem Biophys Res Commun  245: 133–139.  <ul class="find" data-citedArticleID="1067418" data-doi="10.1006/bbrc.1998.8341"><li><a href="http://dx.doi.org/10.1006/bbrc.1998.8341" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Elicitins%2C+proteinaceous+elicitors+of+plant+defense%2C+are+a+new+class+of+sterol+carrier+proteins." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Elicitins%2C+proteinaceous+elicitors+of+plant+defense%2C+are+a+new+class+of+sterol+carrier+proteins.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">68.
              </span><a name="pone.0001723-Hardham1" id="pone.0001723-Hardham1"></a>Hardham AR (2007) Cell biology of plant-oomycete interactions. Cell Microbiol  9: 31–39.  <ul class="find" data-citedArticleID="1067382" data-doi="10.1111/j.1462-5822.2006.00833.x"><li><a href="http://dx.doi.org/10.1111/j.1462-5822.2006.00833.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Cell+biology+of+plant-oomycete+interactions." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Cell+biology+of+plant-oomycete+interactions.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">69.
              </span><a name="pone.0001723-Gaulin3" id="pone.0001723-Gaulin3"></a>Gaulin E, Jauneau A, Villalba F, Rickauer M, Esquerré-Tugayé MT, et al.  (2002) The CBEL glycoprotein of <em>Phytophthora parasitica</em> var-<em>nicotianae</em> is involved in cell wall deposition and adhesion to cellulosic substrates. J Cell Sci  115: 4565–4575.  <ul class="find" data-citedArticleID="1067376" data-doi="10.1242/jcs.00138"><li><a href="http://dx.doi.org/10.1242/jcs.00138" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+CBEL+glycoprotein+of+Phytophthora+parasitica+var-nicotianae+is+involved+in+cell+wall+deposition+and+adhesion+to+cellulosic+substrates." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+CBEL+glycoprotein+of+Phytophthora+parasitica+var-nicotianae+is+involved+in+cell+wall+deposition+and+adhesion+to+cellulosic+substrates.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">70.
              </span><a name="pone.0001723-Judelson1" id="pone.0001723-Judelson1"></a>Judelson HS, Senthil G (2006) Investigating the role of ABC transporters in multifungicide insensitivity in <em>Phytophthora infestans.</em> Mol Plant Pathol  7: 17–29.  <ul class="find" data-citedArticleID="1067396" data-doi="10.1111/j.1364-3703.2005.00313.x"><li><a href="http://dx.doi.org/10.1111/j.1364-3703.2005.00313.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Investigating+the+role+of+ABC+transporters+in+multifungicide+insensitivity+in+Phytophthora+infestans." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Investigating+the+role+of+ABC+transporters+in+multifungicide+insensitivity+in+Phytophthora+infestans.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">71.
              </span><a name="pone.0001723-Akamatsu1" id="pone.0001723-Akamatsu1"></a>Akamatsu HO, Grunwald NJ, Chilvers MI, Porter LD, Peever TL (2007) Development of codominant simple sequence repeat, single nucleotide polymorphism and sequence characterized amplified region markers for the pea root rot pathogen, <em>Aphanomyces euteiches</em>. J Microbiol Methods  71: 82–86.  <ul class="find" data-citedArticleID="1067334" data-doi="10.1016/j.mimet.2007.08.001"><li><a href="http://dx.doi.org/10.1016/j.mimet.2007.08.001" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Development+of+codominant+simple+sequence+repeat%2C+single+nucleotide+polymorphism+and+sequence+characterized+amplified+region+markers+for+the+pea+root+rot+pathogen%2C+Aphanomyces+euteiches." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Development+of+codominant+simple+sequence+repeat%2C+single+nucleotide+polymorphism+and+sequence+characterized+amplified+region+markers+for+the+pea+root+rot+pathogen%2C+Aphanomyces+euteiches.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">72.
              </span><a name="pone.0001723-Krieger1" id="pone.0001723-Krieger1"></a>Krieger S, Schwarz W, Ariyanayagam MR, Fairlamb AH, Krauth-Siegel RL, et al.  (2000) Trypanosomes lacking trypanothione reductase are avirulent and show increased sensitivity to oxidative stress. Mol Microbiol  35: 542–552.  <ul class="find" data-citedArticleID="1067400" data-doi="10.1046/j.1365-2958.2000.01721.x"><li><a href="http://dx.doi.org/10.1046/j.1365-2958.2000.01721.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Trypanosomes+lacking+trypanothione+reductase+are+avirulent+and+show+increased+sensitivity+to+oxidative+stress." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Trypanosomes+lacking+trypanothione+reductase+are+avirulent+and+show+increased+sensitivity+to+oxidative+stress.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">73.
              </span><a name="pone.0001723-Tovar1" id="pone.0001723-Tovar1"></a>Tovar J, Wilkinson S, Mottram JC, Fairlamb AH (1998) Evidence that trypanothione reductase is an essential enzyme in Leishmania by targeted replacement of the tryA gene locus. Mol Microbiol  29: 653–660.  <ul class="find" data-citedArticleID="1067462" data-doi="10.1046/j.1365-2958.1998.00968.x"><li><a href="http://dx.doi.org/10.1046/j.1365-2958.1998.00968.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evidence+that+trypanothione+reductase+is+an+essential+enzyme+in+Leishmania+by+targeted+replacement+of+the+tryA+gene+locus." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evidence+that+trypanothione+reductase+is+an+essential+enzyme+in+Leishmania+by+targeted+replacement+of+the+tryA+gene+locus.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">74.
              </span><a name="pone.0001723-Wojtaszek1" id="pone.0001723-Wojtaszek1"></a>Wojtaszek P (1997) Oxidative burst: an early plant response to pathogen infection. Biochem J  322: 681–692.  <ul class="find" data-citedArticleID="1067476"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Oxidative+burst%3A+an+early+plant+response+to+pathogen+infection.&amp;auth=&amp;atitle=Oxidative+burst%3A+an+early+plant+response+to+pathogen+infection." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Oxidative+burst%3A+an+early+plant+response+to+pathogen+infection." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Oxidative+burst%3A+an+early+plant+response+to+pathogen+infection.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001723.XML" value="143475"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001723.PDF" value="1038183"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001723.g001.PNG_L" value="319725"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001723.g001.PNG_M" value="51681"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001723.g001.PNG_S" value="7332"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001723.g001.TIF" value="616000"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001723.g001.PNG_I" value="19544"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001723.t001.PNG_L" value="60510"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001723.t001.PNG_M" value="76003"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001723.t001.PNG_S" value="12567"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001723.t001.TIF" value="255652"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001723.t001.PNG_I" value="27079"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001723.t002.PNG_L" value="118111"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001723.t002.PNG_M" value="66853"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001723.t002.PNG_S" value="9727"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001723.t002.TIF" value="529366"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001723.t002.PNG_I" value="20362"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001723.g002.PNG_L" value="179020"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001723.g002.PNG_M" value="45505"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001723.g002.PNG_S" value="7937"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001723.g002.TIF" value="415854"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001723.g002.PNG_I" value="47535"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001723.g003.PNG_L" value="235549"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001723.g003.PNG_M" value="88826"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001723.g003.PNG_S" value="8253"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001723.g003.TIF" value="486602"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001723.g003.PNG_I" value="32586"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001723.g004.PNG_L" value="255089"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001723.g004.PNG_M" value="120451"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001723.g004.PNG_S" value="8318"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001723.g004.TIF" value="413020"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001723.g004.PNG_I" value="43669"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001723.g005.PNG_L" value="574801"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001723.g005.PNG_M" value="174031"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001723.g005.PNG_S" value="8592"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001723.g005.TIF" value="820390"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001723.g005.PNG_I" value="62202"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001723.g006.PNG_L" value="639978"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001723.g006.PNG_M" value="195400"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001723.g006.PNG_S" value="12120"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001723.g006.TIF" value="1030956"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001723.g006.PNG_I" value="71199"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001723.g007.PNG_L" value="1172060"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001723.g007.PNG_M" value="185696"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001723.g007.PNG_S" value="8645"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001723.g007.TIF" value="1448364"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001723.g007.PNG_I" value="61715"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001723.t003.PNG_L" value="216097"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001723.t003.PNG_M" value="119699"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001723.t003.PNG_S" value="12281"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001723.t003.TIF" value="873282"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001723.t003.PNG_I" value="36759"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001723&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001723" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001723&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0001723&volume=&issue=&title=Transcriptome of Aphanomyces euteiches: New Oomycete Putative Pathogenicity Factors and Metabolic Pathways&author_name=Elodie%20Gaulin%2C%20Mohammed-Amine%20Madoui%2C%20Arnaud%20Bottin%2C%20Christophe%20Jacquet%2C%20Catherine%20Math%C3%A9%2C%20Arnaud%20Couloux%2C%20Patrick%20Wincker%2C%20Bernard%20Dumas&start_page=1&end_page=12" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001723" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001723" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0001723" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001723&amp;t=Transcriptome%20of%20Aphanomyces%20euteiches%3A%20New%20Oomycete%20Putative%20Pathogenicity%20Factors%20and%20Metabolic%20Pathways" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001723&title=Transcriptome%20of%20Aphanomyces%20euteiches%3A%20New%20Oomycete%20Putative%20Pathogenicity%20Factors%20and%20Metabolic%20Pathways&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001723&amp;title=Transcriptome%20of%20Aphanomyces%20euteiches%3A%20New%20Oomycete%20Putative%20Pathogenicity%20Factors%20and%20Metabolic%20Pathways" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001723" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0001723&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Transcriptome of Aphanomyces euteiches: New Oomycete Putative Pathogenicity Factors and Metabolic Pathways';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0001723';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Transcriptome%20of%20Aphanomyces%20euteiches%3A%20New%20Oomycete%20Putative%20Pathogenicity%20Factors%20and%20Metabolic%20Pathways http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001723" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0001723" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">



















          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Expressed+sequence+tags%22" title="Search for articles in the subject area:'Expressed sequence tags'"><div class="flagText">Expressed sequence tags</div></a>
              <div data-categoryid="34427" data-articleid="26272"
                   data-categoryname="Expressed sequence tags"
                   class="flagImage" title="Flag 'Expressed sequence tags' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Oomycetes%22" title="Search for articles in the subject area:'Oomycetes'"><div class="flagText">Oomycetes</div></a>
              <div data-categoryid="44021" data-articleid="26272"
                   data-categoryname="Oomycetes"
                   class="flagImage" title="Flag 'Oomycetes' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Phytophthora%22" title="Search for articles in the subject area:'Phytophthora'"><div class="flagText">Phytophthora</div></a>
              <div data-categoryid="34957" data-articleid="26272"
                   data-categoryname="Phytophthora"
                   class="flagImage" title="Flag 'Phytophthora' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Plant+pathogens%22" title="Search for articles in the subject area:'Plant pathogens'"><div class="flagText">Plant pathogens</div></a>
              <div data-categoryid="34897" data-articleid="26272"
                   data-categoryname="Plant pathogens"
                   class="flagImage" title="Flag 'Plant pathogens' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Proteases%22" title="Search for articles in the subject area:'Proteases'"><div class="flagText">Proteases</div></a>
              <div data-categoryid="19447" data-articleid="26272"
                   data-categoryname="Proteases"
                   class="flagImage" title="Flag 'Proteases' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Sequence+databases%22" title="Search for articles in the subject area:'Sequence databases'"><div class="flagText">Sequence databases</div></a>
              <div data-categoryid="42787" data-articleid="26272"
                   data-categoryname="Sequence databases"
                   class="flagImage" title="Flag 'Sequence databases' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Sequence+motif+analysis%22" title="Search for articles in the subject area:'Sequence motif analysis'"><div class="flagText">Sequence motif analysis</div></a>
              <div data-categoryid="19657" data-articleid="26272"
                   data-categoryname="Sequence motif analysis"
                   class="flagImage" title="Flag 'Sequence motif analysis' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22cDNA+libraries%22" title="Search for articles in the subject area:'cDNA libraries'"><div class="flagText">cDNA libraries</div></a>
              <div data-categoryid="35307" data-articleid="26272"
                   data-categoryname="cDNA libraries"
                   class="flagImage" title="Flag 'cDNA libraries' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=1919'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=8051'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=1404&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>


</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
