

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: A Dimer of the Toll-Like Receptor 4 Cytoplasmic Domain Provides a Specific Scaffold for the Recruitment of Signalling Adaptor Proteins</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0000788"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0000788" />

    <meta name="citation_title" content="A Dimer of the Toll-Like Receptor 4 Cytoplasmic Domain Provides a Specific Scaffold for the Recruitment of Signalling Adaptor Proteins"/>
    <meta itemprop="name" content="A Dimer of the Toll-Like Receptor 4 Cytoplasmic Domain Provides a Specific Scaffold for the Recruitment of Signalling Adaptor Proteins"/>

      <meta name="citation_author" content="Ricardo Núñez Miguel"/>
            <meta name="citation_author_institution" content="Department of Biochemistry, University of Cambridge, Cambridge, United Kingdom"/>
      <meta name="citation_author" content="Joyce Wong"/>
            <meta name="citation_author_institution" content="Department of Biochemistry, University of Cambridge, Cambridge, United Kingdom"/>
      <meta name="citation_author" content="Julian F. Westoll"/>
            <meta name="citation_author_institution" content="Department of Biochemistry, University of Cambridge, Cambridge, United Kingdom"/>
      <meta name="citation_author" content="Heather J. Brooks"/>
            <meta name="citation_author_institution" content="Department of Veterinary Medicine, University of Cambridge, Cambridge, United Kingdom"/>
      <meta name="citation_author" content="Luke A. J. O'Neill"/>
            <meta name="citation_author_institution" content="School of Biochemistry and Immunology, Trinity College, Dublin, Ireland"/>
      <meta name="citation_author" content="Nicholas J. Gay"/>
            <meta name="citation_author_institution" content="Department of Biochemistry, University of Cambridge, Cambridge, United Kingdom"/>
      <meta name="citation_author" content="Clare E. Bryant"/>
            <meta name="citation_author_institution" content="Department of Veterinary Medicine, University of Cambridge, Cambridge, United Kingdom"/>
      <meta name="citation_author" content="Tom P. Monie"/>
            <meta name="citation_author_institution" content="Department of Biochemistry, University of Cambridge, Cambridge, United Kingdom"/>

    <meta name="citation_date" content="2007/8/29"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0000788.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e788"/>
    <meta name="citation_issue" content="8"/>
    <meta name="citation_volume" content="2"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Innate immune recognition.; citation_author=CA Janeway; citation_author=R Medzhitov; citation_journal_title=Annu Rev Immunol; citation_volume=20; citation_number=1; citation_pages=197-216; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Toll-like receptor signalling.; citation_author=S Akira; citation_author=K Takeda; citation_journal_title=Nat Rev Immunol; citation_volume=4; citation_number=2; citation_pages=499-511; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Structure and function of toll receptors and their ligands.; citation_author=NJ Gay; citation_author=M Gangloff; citation_journal_title=Annu Rev Biochem; citation_volume=76; citation_number=3; citation_pages=141-165; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Gram-negative endotoxin: an extraordinary lipid with profound effects on eukaryotic signal transduction.; citation_author=CR Raetz; citation_author=RJ Ulevitch; citation_author=SD Wright; citation_author=CH Sibley; citation_author=A Ding; citation_journal_title=Faseb J; citation_volume=5; citation_number=4; citation_pages=2652-2660; citation_date=1991; " />
      <meta name="citation_reference" content="citation_title=Sepsis under siege: a new understanding of sepsis might lead to the development of therapies to treat septic shock.; citation_author=P Hunter; citation_journal_title=EMBO Rep; citation_volume=7; citation_number=5; citation_pages=667-669; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Defective LPS signaling in C3H/HeJ and C57BL/10ScCr mice: mutations in Tlr4 gene.; citation_author=A Poltorak; citation_author=X He; citation_author=I Smirnova; citation_author=MY Liu; citation_author=C Van Huffel; citation_journal_title=Science; citation_volume=282; citation_number=6; citation_pages=2085-2088; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Essential role of MD-2 in LPS responsiveness and TLR4 distribution.; citation_author=Y Nagai; citation_author=S Akashi; citation_author=M Nagafuku; citation_author=M Ogata; citation_author=Y Iwakura; citation_journal_title=Nat Immunol; citation_volume=3; citation_number=7; citation_pages=667-672; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Molecular genetic analysis of an endotoxin nonresponder mutant cell line: a point mutation in a conserved region of MD-2 abolishes endotoxin-induced signaling.; citation_author=AB Schromm; citation_author=E Lien; citation_author=P Henneke; citation_author=JC Chow; citation_author=A Yoshimura; citation_journal_title=J Exp Med; citation_volume=194; citation_number=8; citation_pages=79-88; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=MD-2: the Toll ‘gatekeeper’ in endotoxin signalling.; citation_author=M Gangloff; citation_author=NJ Gay; citation_journal_title=Trends Biochem Sci; citation_volume=29; citation_number=9; citation_pages=294-300; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Innate recognition of lipopolysaccharide by Toll-like receptor 4-MD-2.; citation_author=K Miyake; citation_journal_title=Trends Microbiol; citation_volume=12; citation_number=10; citation_pages=186-192; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Lipid A antagonist, lipid IVa, is distinct from lipid A in interaction with Toll-like receptor 4 (TLR4)-MD-2 and ligand-induced TLR4 oligomerization.; citation_author=S Saitoh; citation_author=S Akashi; citation_author=T Yamada; citation_author=N Tanimura; citation_author=M Kobayashi; citation_journal_title=Int Immunol; citation_volume=16; citation_number=11; citation_pages=961-969; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Toll-like receptors as molecular switches.; citation_author=NJ Gay; citation_author=M Gangloff; citation_author=AN Weber; citation_journal_title=Nat Rev Immunol; citation_volume=6; citation_number=12; citation_pages=693-698; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Ligand-induced conformational changes allosterically activate Toll-like receptor 9.; citation_author=E Latz; citation_author=A Verma; citation_author=A Visintin; citation_author=M Gong; citation_author=CM Sirois; citation_journal_title=Nat Immunol; citation_volume=8; citation_number=13; citation_pages=772-779; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=The Toll-IL-1 receptor adaptor family grows to five members.; citation_author=LA O'Neill; citation_author=KA Fitzgerald; citation_author=AG Bowie; citation_journal_title=Trends Immunol; citation_volume=24; citation_number=14; citation_pages=286-290; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=A family of human receptors structurally related to Drosophila Toll.; citation_author=FL Rock; citation_author=G Hardiman; citation_author=JC Timans; citation_author=RA Kastelein; citation_author=JF Bazan; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=95; citation_number=15; citation_pages=588-593; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Structural complementarity of Toll/interleukin-1 receptor domains in Toll-like receptors and the adaptors Mal and MyD88.; citation_author=A Dunne; citation_author=M Ejdeback; citation_author=PL Ludidi; citation_author=LA O'Neill; citation_author=NJ Gay; citation_journal_title=J Biol Chem; citation_volume=278; citation_number=16; citation_pages=41443-41451; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Mal (MyD88-adapter-like) is required for Toll-like receptor-4 signal transduction.; citation_author=KA Fitzgerald; citation_author=EM Palsson-McDermott; citation_author=AG Bowie; citation_author=CA Jefferies; citation_author=AS Mansell; citation_journal_title=Nature; citation_volume=413; citation_number=17; citation_pages=78-83; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=LPS-TLR4 signaling to IRF-3/7 and NF-kappaB involves the toll adapters TRAM and TRIF.; citation_author=KA Fitzgerald; citation_author=DC Rowe; citation_author=BJ Barnes; citation_author=DR Caffrey; citation_author=A Visintin; citation_journal_title=J Exp Med; citation_volume=198; citation_number=18; citation_pages=1043-1055; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Structural basis for signal transduction by the Toll/interleukin-1 receptor domains.; citation_author=Y Xu; citation_author=X Tao; citation_author=B Shen; citation_author=T Horng; citation_author=R Medzhitov; citation_journal_title=Nature; citation_volume=408; citation_number=19; citation_pages=111-115; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Assessment of protein models with three-dimensional profiles.; citation_author=R Luthy; citation_author=JU Bowie; citation_author=D Eisenberg; citation_journal_title=Nature; citation_volume=356; citation_number=20; citation_pages=83-85; citation_date=1992; " />
      <meta name="citation_reference" content="citation_title=Evaluation of GRAMM low-resolution docking methodology on the hemagglutinin-antibody complex.; citation_author=IA Vakser; citation_journal_title=Proteins Suppl; citation_volume=1; citation_number=21; citation_pages=226-230; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=pyDock: electrostatics and desolvation for effective scoring of rigid-body protein-protein docking.; citation_author=T Man-Kuang Cheng; citation_author=TL Blundell; citation_author=J Fernandez-Recio; citation_journal_title=Proteins; citation_volume=68; citation_number=22; citation_pages=503-515; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Common interaction surfaces of the toll-like receptor 4 cytoplasmic domain stimulate multiple nuclear targets.; citation_author=T Ronni; citation_author=V Agarwal; citation_author=M Haykinson; citation_author=ME Haberland; citation_author=G Cheng; citation_journal_title=Mol Cell Biol; citation_volume=23; citation_number=23; citation_pages=2543-2555; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Cutting Edge: Differential inhibition of TLR signaling pathways by cell-permeable peptides representing BB loops of TLRs.; citation_author=VY Toshchakov; citation_author=MJ Fenton; citation_author=SN Vogel; citation_journal_title=J Immunol; citation_volume=178; citation_number=24; citation_pages=2655-2660; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Differential involvement of BB loops of toll-IL-1 resistance (TIR) domain-containing adapter proteins in TLR4- versus TLR2-mediated signal transduction.; citation_author=VU Toshchakov; citation_author=S Basu; citation_author=MJ Fenton; citation_author=SN Vogel; citation_journal_title=J Immunol; citation_volume=175; citation_number=25; citation_pages=494-500; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=MyD88 adapter-like (Mal) is phosphorylated by Bruton's tyrosine kinase during TLR2 and TLR4 signal transduction.; citation_author=P Gray; citation_author=A Dunne; citation_author=C Brikos; citation_author=CA Jefferies; citation_author=SL Doyle; citation_journal_title=J Biol Chem; citation_volume=281; citation_number=26; citation_pages=10489-10495; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=The crystal structure of the BAR domain from human Bin1/amphiphysin II and its implications for molecular recognition.; citation_author=E Casal; citation_author=L Federici; citation_author=W Zhang; citation_author=J Fernandez-Recio; citation_author=EM Priego; citation_journal_title=Biochemistry; citation_volume=45; citation_number=27; citation_pages=12917-12928; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Suppressor of cytokine signaling 1 negatively regulates Toll-like receptor signaling by mediating Mal degradation.; citation_author=A Mansell; citation_author=R Smith; citation_author=SL Doyle; citation_author=P Gray; citation_author=JE Fenner; citation_journal_title=Nat Immunol; citation_volume=7; citation_number=28; citation_pages=148-155; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=NF-kappaB activation by the Toll-IL-1 receptor domain protein MyD88 adapter-like is regulated by caspase-1.; citation_author=SM Miggin; citation_author=E Palsson-McDermott; citation_author=A Dunne; citation_author=C Jefferies; citation_author=E Pinteaux; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=104; citation_number=29; citation_pages=3372-3377; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=A Mal functional variant is associated with protection against invasive pneumococcal disease, bacteremia, malaria and tuberculosis.; citation_author=CC Khor; citation_author=SJ Chapman; citation_author=FO Vannberg; citation_author=A Dunne; citation_author=C Murphy; citation_journal_title=Nat Genet; citation_volume=39; citation_number=30; citation_pages=523-528; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=FUGUE: sequence-structure homology recognition using environment-specific substitution tables and structure-dependent gap penalties.; citation_author=J Shi; citation_author=TL Blundell; citation_author=K Mizuguchi; citation_journal_title=J Mol Biol; citation_volume=310; citation_number=31; citation_pages=243-257; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=HOMSTRAD: adding sequence information to structure-based alignments of homologous protein families.; citation_author=PI de Bakker; citation_author=A Bateman; citation_author=DF Burke; citation_author=RN Miguel; citation_author=K Mizuguchi; citation_journal_title=Bioinformatics; citation_volume=17; citation_number=32; citation_pages=748-749; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Comparative protein modelling by satisfaction of spatial restraints.; citation_author=A Sali; citation_author=TL Blundell; citation_journal_title=J Mol Biol; citation_volume=234; citation_number=33; citation_pages=779-815; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=CHARMM: a program for macromolecular energy minimisation and dynamics calculations.; citation_author=BR Brooks; citation_author=RE Bruccolesi; citation_author=BD Olafson; citation_author=S Swaminathan; citation_author=M Karplus; citation_journal_title=Journal of Computational Chemistry; citation_volume=4; citation_number=34; citation_pages=187-217; citation_date=1983; " />
      <meta name="citation_reference" content="citation_title=A program to check the sterochemical quality of protein structures.; citation_author=RA Laskowski; citation_author=MW MacArthur; citation_author=DS Moss; citation_author=JM Thornton; citation_journal_title=Journal of Applied Crystallography; citation_volume=26; citation_number=35; citation_pages=283-291; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=JOY: protein sequence-structure representation and analysis.; citation_author=K Mizuguchi; citation_author=CM Deane; citation_author=TL Blundell; citation_author=MS Johnson; citation_author=JP Overington; citation_journal_title=Bioinformatics; citation_volume=14; citation_number=36; citation_pages=617-623; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Principles of protein-protein interactions.; citation_author=S Jones; citation_author=JM Thornton; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=93; citation_number=37; citation_pages=13-20; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=The interpretation of protein structures: estimation of static accessibility.; citation_author=B Lee; citation_author=FM Richards; citation_journal_title=J Mol Biol; citation_volume=55; citation_number=38; citation_pages=379-400; citation_date=1971; " />
      <meta name="citation_reference" content="citation_title=Solvent accessible surface area and excluded volume in proteins. Analytical equations for overlapping spheres and implications for the hydrophobic effect.; citation_author=TJ Richmond; citation_journal_title=J Mol Biol; citation_volume=178; citation_number=39; citation_pages=63-89; citation_date=1984; " />
      <meta name="citation_reference" content="citation_title=Satisfying hydrogen bonding potential in proteins.; citation_author=IK McDonald; citation_author=JM Thornton; citation_journal_title=J Mol Biol; citation_volume=238; citation_number=40; citation_pages=777-793; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=Molecular surface recognition: determination of geometric fit between proteins and their ligands by correlation techniques.; citation_author=E Katchalski-Katzir; citation_author=I Shariv; citation_author=M Eisenstein; citation_author=AA Friesem; citation_author=C Aflalo; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=89; citation_number=41; citation_pages=2195-2199; citation_date=1992; " />
      <meta name="citation_reference" content="citation_title=Sequence patterns derived from the automated prediction of functional residues in structurally-aligned homologous protein families.; citation_author=RN Miguel; citation_journal_title=Bioinformatics; citation_volume=20; citation_number=42; citation_pages=2380-2389; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=IKKepsilon and TBK1 are essential components of the IRF3 signaling pathway.; citation_author=KA Fitzgerald; citation_author=SM McWhirter; citation_author=KL Faia; citation_author=DC Rowe; citation_author=E Latz; citation_journal_title=Nat Immunol; citation_volume=4; citation_number=43; citation_pages=491-496; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Essential role of interferon regulatory factor 3 in direct activation of RANTES chemokine transcription.; citation_author=R Lin; citation_author=C Heylbroeck; citation_author=P Genin; citation_author=PM Pitha; citation_author=J Hiscott; citation_journal_title=Mol Cell Biol; citation_volume=19; citation_number=44; citation_pages=959-966; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Ligand-receptor and receptor-receptor interactions act in concert to activate signaling in the Drosophila toll pathway.; citation_author=AN Weber; citation_author=MC Moncrieffe; citation_author=M Gangloff; citation_author=JL Imler; citation_author=NJ Gay; citation_journal_title=J Biol Chem; citation_volume=280; citation_number=45; citation_pages=22793-22799; citation_date=2005; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000788" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="A Dimer of the Toll-Like Receptor 4 Cytoplasmic Domain Provides a Specific Scaffold for the Recruitment of Signalling Adaptor Proteins"/>
    <meta name="twitter:description" content="The Toll-like receptor 4 (TLR4) is a class I transmembrane receptor expressed on the surface of immune system cells. TLR4 is activated by exposure to lipopolysaccharides derived from the outer membrane of Gram negative bacteria and forms part of the innate immune response in mammals. Like other class 1 receptors, TLR4 is activated by ligand induced dimerization, and recent studies suggest that this causes concerted conformational changes in the receptor leading to self association of the cytoplasmic Toll/Interleukin 1 receptor (TIR) signalling domain. This homodimerization event is proposed to provide a new scaffold that is able to bind downstream signalling adaptor proteins. TLR4 uses two different sets of adaptors; TRAM and TRIF, and Mal and MyD88. These adaptor pairs couple two distinct signalling pathways leading to the activation of interferon response factor 3 (IRF-3) and nuclear factor &kappa;B (NF&kappa;B) respectively. In this paper we have generated a structural model of the TLR4 TIR dimer and used molecular docking to probe for potential sites of interaction between the receptor homodimer and the adaptor molecules. Remarkably, both the Mal and TRAM adaptors are strongly predicted to bind at two symmetry-related sites at the homodimer interface. This model of TLR4 activation is supported by extensive functional studies involving site directed mutagenesis, inhibition by cell permeable peptides and stable protein phosphorylation of receptor and adaptor TIR domains. Our results also suggest a molecular mechanism for two recent findings, the caspase 1 dependence of Mal signalling and the protective effects conferred by the Mal polymorphism Ser180Leu."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0000788.g005"/>

  <meta property="og:title" content="A Dimer of the Toll-Like Receptor 4 Cytoplasmic Domain Provides a Specific Scaffold for the Recruitment of Signalling Adaptor Proteins" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000788" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=1553'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=6915'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=8367&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0000788">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000788" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000788&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    A Dimer of the Toll-Like Receptor 4 Cytoplasmic Domain Provides a Specific Scaffold for the Recruitment of Signalling Adaptor Proteins
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Ricardo Núñez Miguel
              <span class="equal-contrib"
                    title="These authors contributed equally to this work">equal contributor</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">

                <p><span class="equal-contrib" title="These authors contributed equally to this work">equal contributor</span>
                Contributed equally to this work with: Ricardo Núñez Miguel, Joyce Wong</p>

              
              

                <p>Affiliation:
                  Department of Biochemistry, University of Cambridge, Cambridge, United Kingdom
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Joyce Wong
              <span class="equal-contrib"
                    title="These authors contributed equally to this work">equal contributor</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">

                <p><span class="equal-contrib" title="These authors contributed equally to this work">equal contributor</span>
                Contributed equally to this work with: Ricardo Núñez Miguel, Joyce Wong</p>

              
              

                <p>Affiliation:
                  Department of Biochemistry, University of Cambridge, Cambridge, United Kingdom
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Julian F. Westoll, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Biochemistry, University of Cambridge, Cambridge, United Kingdom
                </p>

                <p>
                    Current address: University of Stellenbosch, Matieland, South Africa
                </p>

              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Heather J. Brooks, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Veterinary Medicine, University of Cambridge, Cambridge, United Kingdom
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Luke A. J. O'Neill, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  School of Biochemistry and Immunology, Trinity College, Dublin, Ireland
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Nicholas J. Gay
              <span class="corresponding">mail</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:njg11@cam.ac.uk">njg11@cam.ac.uk</a> (NG), <a href="mailto:tpm22@cam.ac.uk ceb27@cam.ac.uk">tpm22@cam.ac.uk ceb27@cam.ac.uk</a> (TPM), <a href="mailto:ceb27@cam.ac.uk">ceb27@cam.ac.uk</a> (CB)</p>

                <p>Affiliation:
                  Department of Biochemistry, University of Cambridge, Cambridge, United Kingdom
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Clare E. Bryant
              <span class="corresponding">mail</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:njg11@cam.ac.uk">njg11@cam.ac.uk</a> (NG), <a href="mailto:tpm22@cam.ac.uk ceb27@cam.ac.uk">tpm22@cam.ac.uk ceb27@cam.ac.uk</a> (TPM), <a href="mailto:ceb27@cam.ac.uk">ceb27@cam.ac.uk</a> (CB)</p>

                <p>Affiliation:
                  Department of Veterinary Medicine, University of Cambridge, Cambridge, United Kingdom
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Tom P. Monie
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:njg11@cam.ac.uk">njg11@cam.ac.uk</a> (NG), <a href="mailto:tpm22@cam.ac.uk ceb27@cam.ac.uk">tpm22@cam.ac.uk ceb27@cam.ac.uk</a> (TPM), <a href="mailto:ceb27@cam.ac.uk">ceb27@cam.ac.uk</a> (CB)</p>

                <p>Affiliation:
                  Department of Biochemistry, University of Cambridge, Cambridge, United Kingdom
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: August 29, 2007</li>
    <li>DOI: 10.1371/journal.pone.0000788</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0000788" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0000788" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000788" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0000788" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0000788-g001" data-doi="info:doi/10.1371/journal.pone.0000788" data-uri="info:doi/10.1371/journal.pone.0000788.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000788.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000788-g002" data-doi="info:doi/10.1371/journal.pone.0000788" data-uri="info:doi/10.1371/journal.pone.0000788.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000788.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000788-g003" data-doi="info:doi/10.1371/journal.pone.0000788" data-uri="info:doi/10.1371/journal.pone.0000788.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000788.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000788-t001" data-doi="info:doi/10.1371/journal.pone.0000788" data-uri="info:doi/10.1371/journal.pone.0000788.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000788.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000788-g004" data-doi="info:doi/10.1371/journal.pone.0000788" data-uri="info:doi/10.1371/journal.pone.0000788.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000788.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000788-t002" data-doi="info:doi/10.1371/journal.pone.0000788" data-uri="info:doi/10.1371/journal.pone.0000788.t002" title="Table 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000788.t002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000788-t003" data-doi="info:doi/10.1371/journal.pone.0000788" data-uri="info:doi/10.1371/journal.pone.0000788.t003" title="Table 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000788.t003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000788-t004" data-doi="info:doi/10.1371/journal.pone.0000788" data-uri="info:doi/10.1371/journal.pone.0000788.t004" title="Table 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000788.t004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000788-t005" data-doi="info:doi/10.1371/journal.pone.0000788" data-uri="info:doi/10.1371/journal.pone.0000788.t005" title="Table 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000788.t005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000788-g005" data-doi="info:doi/10.1371/journal.pone.0000788" data-uri="info:doi/10.1371/journal.pone.0000788.g005" title="Figure 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000788.g005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000788">Reader Comments (0)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0000788" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1"></a><p>The Toll-like receptor 4 (TLR4) is a class I transmembrane receptor expressed on the surface of immune system cells. TLR4 is activated by exposure to lipopolysaccharides derived from the outer membrane of Gram negative bacteria and forms part of the innate immune response in mammals. Like other class 1 receptors, TLR4 is activated by ligand induced dimerization, and recent studies suggest that this causes concerted conformational changes in the receptor leading to self association of the cytoplasmic Toll/Interleukin 1 receptor (TIR) signalling domain. This homodimerization event is proposed to provide a new scaffold that is able to bind downstream signalling adaptor proteins. TLR4 uses two different sets of adaptors; TRAM and TRIF, and Mal and MyD88. These adaptor pairs couple two distinct signalling pathways leading to the activation of interferon response factor 3 (IRF-3) and nuclear factor κB (NFκB) respectively. In this paper we have generated a structural model of the TLR4 TIR dimer and used molecular docking to probe for potential sites of interaction between the receptor homodimer and the adaptor molecules. Remarkably, both the Mal and TRAM adaptors are strongly predicted to bind at two symmetry-related sites at the homodimer interface. This model of TLR4 activation is supported by extensive functional studies involving site directed mutagenesis, inhibition by cell permeable peptides and stable protein phosphorylation of receptor and adaptor TIR domains. Our results also suggest a molecular mechanism for two recent findings, the caspase 1 dependence of Mal signalling and the protective effects conferred by the Mal polymorphism Ser180Leu.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>Núñez Miguel R, Wong J, Westoll JF, Brooks HJ, O'Neill LAJ, et al.  (2007) A Dimer of the Toll-Like Receptor 4 Cytoplasmic Domain Provides a Specific Scaffold for the Recruitment of Signalling Adaptor Proteins. PLoS ONE 2(8):
          e788.
            doi:10.1371/journal.pone.0000788</p><p><strong>Academic Editor: </strong>Juha Klefstrom, University of Helsinki, Finland</p><p><strong>Received:</strong> May 9, 2007; <strong>Accepted:</strong> July 24, 2007; <strong>Published:</strong> August 29, 2007</p><p><strong>Copyright:</strong> © 2007 Núñez Miguel et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>This work was supported by grants to NJG from the UK Medical Research Council and Biotechnology and Biological Sciences Research Council.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>In humans and other vertebrates initial responses to infection by pathogenic microorganisms such as viruses and bacteria are mediated by a highly developed innate immune response<a href="#pone.0000788-Janeway1">[1]</a>. Pattern recognition receptors (PRRs) expressed by immune system cells such as macrophages and dendritic cells are able to detect conserved microbial structures. These cells then generate the innate immune responses that are required to fight the infection and promote the development of adaptive immunity. The Toll-like receptors (TLRs) are an important group of PRRs that respond to a range of microbial products such as lipopeptides and non-self nucleic acids <a href="#pone.0000788-Akira1">[2]</a>. The TLRs are type I transmembrane receptors and consist of an extracellular domain made up mainly of leucine rich repeat motifs, a single transmembrane spanning segment and a globular cytoplasmic domain, the Toll/interleukin 1 receptor domain (TIR) <a href="#pone.0000788-Gay1">[3]</a>. There are ten Toll-like receptors encoded in the human genome and each of these respond to specific microbial products.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>One of the most important innate immune stimuli is lipopolysaccharide (LPS) or endotoxin found in the outer membrane of Gram-negative bacteria <a href="#pone.0000788-Raetz1">[4]</a>. LPS is one of the most powerful immunostimulators known and is responsible not only for the induction of innate immunity but also for the dangerous condition endotoxic shock which often develops during Gram negative septicaemia. Endotoxic shock is a severe inflammatory disease that leads rapidly to multi organ failure and death. This condition accounts for about 200,000 deaths per annum in Europe and thus understanding the mechanism of action of LPS mediated immune activation is an important objective in medical research <a href="#pone.0000788-Hunter1">[5]</a>. In 1998 TLR4 was identified as the signalling receptor for LPS. Mice that lack functional TLR4 are hyposensitive to LPS and consequently more sensitive to infection by Gram negative bacteria <a href="#pone.0000788-Poltorak1">[6]</a>. Subsequent studies showed that MD-2, a co-receptor protein of TLR4, was also essential for LPS induced signalling <a href="#pone.0000788-Nagai1">[7]</a>, <a href="#pone.0000788-Schromm1">[8]</a>. MD-2 is a member of a small class of lipid binding proteins and interacts directly with the lipid A moiety of LPS <a href="#pone.0000788-Gangloff1">[9]</a>, <a href="#pone.0000788-Miyake1">[10]</a>.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>Like other class I receptors the initial step in signal transduction by TLR4 involves dimerization or oligomerization of two receptor chains induced by binding of MD-2 to the lipid A moiety of LPS <a href="#pone.0000788-Saitoh1">[11]</a>. This in turn probably causes protein conformational changes in the receptor resulting in the association of two receptor TIR domains (<a href="#pone-0000788-g001">Figure 1</a>) <a href="#pone.0000788-Gay2">[12]</a>. Alternatively, the receptor may be present in the cell as a preformed but inactive dimer and ligand binding may cause reorientation of the TIR domains. Consistent with this idea, a recent study using FRET (fluorescence resonance energy transfer) microscopy showed that the TLR9 TIR domains undergo a large positional change on ligand binding <a href="#pone.0000788-Latz1">[13]</a>. In either case association of the receptor TIRs would provide a new scaffold that allows the recruitment of specific adaptor proteins to form a post-receptor signalling complex. There are five adaptor proteins, all of which contain TIR domains, that function in TLR signalling; MyD88 (Myeloid differentiation primary response protein 88), Mal (MyD88 adaptor like; also known as TIRAP), TRIF (TIR domain-containing adaptor protein inducing interferon-β; also known as TICAM1), TRAM (TRIF-related adaptor molecule; also known as TICAM2), and SARM (sterile α- and armadillo-motif-containing protein) <a href="#pone.0000788-ONeill1">[14]</a>. The TIR domain forms into an α−β structure and the sequence conservation observed reflects the structural requirements of this fold <a href="#pone.0000788-Rock1">[15]</a>. On the other hand, the loops that connect the secondary structure elements of the TIR domain and the surface electrostatic properties are more variable and these properties may confer specificity for homo- and heterotypic interactions between different TIR domains <a href="#pone.0000788-Dunne1">[16]</a>.</p>
<div class="figure" id="pone-0000788-g001"><div class="img"><a name="pone-0000788-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000788.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000788" data-uri="info:doi/10.1371/journal.pone.0000788.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000788.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000788.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000788.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000788.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000788.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000788.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000788.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000788.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000788.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>The TLR4 signalling pathway.</span></strong></p><a id="article1.body1.sec1.fig1.caption1.p1" name="article1.body1.sec1.fig1.caption1.p1"></a><p>(A) Overview of the TLR4 signalling pathway. Both the NF-κB and the interferon pathways are induced by stimulation with lipopolysaccharide. Adapted from <a href="#pone.0000788-Akira1">[2]</a>, <a href="#pone.0000788-Gay1">[3]</a>. (B) Mechanism of signal transduction by TLR4. The curved ectodomains (ECD) are illustrated in light blue and the co-receptor protein MD-2 in grey. The TIR domains are shown in yellow and red respectively. M = membrane, L = LPS. (i) Prior to activation, receptor molecules are able to diffuse in the membrane and may form transient dimers. The ectodomains are rigidly connected to the cytoplasmic TIRs by the transmembrane helix. (ii) Receptor dimerization following activation by LPS binding to MD2. By analogy with <em>Drosophila</em> Toll (see <a href="#pone.0000788-Weber1">[45]</a>), which is activated by a dimeric protein ligand, the receptor complexes are likely to be symmetrical. Conformational rearrangements constrain the TIR domains to interact through equivalent surfaces forming a symmetrical dimer. (iii) The dimerized TIRs provide a new molecular surface that can bind to the ‘bridging adaptor’ molecules TRAM and Mal with high affinity. Interaction with the downstream adaptors TRIF and MyD88 leads to NFκB and IRF3 mediated signalling respectively.</p>
<span>doi:10.1371/journal.pone.0000788.g001</span></div><a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p>Unlike the other TLRs, activated TLR4 signals via two distinct sets of adaptor proteins, Mal and MyD88, and TRAM and TRIF <a href="#pone.0000788-Fitzgerald1">[17]</a>, <a href="#pone.0000788-Fitzgerald2">[18]</a> (see <a href="#pone-0000788-g001">Figure 1</a>). For each pathway Mal and TRAM are thought to engage directly with the TLR4 receptor dimer and subsequently act as ‘bridging adaptors’ for the recruitment of MyD88 and TRIF respectively. Mal is required for rapid activation of the ΝFκΒ transcription factor and the production of pro-inflammatory cytokines such as TNFα and IL12. The TRAM adaptor on the other hand stimulates a different pathway leading to activation of the interferon response factor, IRF-3. IRF-3 induces expression of a distinct set of genes to NFκB, such as interferon β and the chemokine RANTES. It is not known whether a single activated TLR4 dimer can recruit both TRAM and Mal simultaneously or whether binding is mutually exclusive.</p>
<a id="article1.body1.sec1.p5" name="article1.body1.sec1.p5"></a><p>In this paper we have used modelling, docking and mutagenesis studies to investigate the nature of the initial TLR4 receptor dimer-‘bridging adaptor’ complexes. We show that both Mal and TRAM are strongly predicted to bind at the interface of a TLR4 TIR homodimer, a result consistent with multiple other studies using mutagenesis of receptor and adaptor TIRs and blocking peptides.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3>
<h4>Predicted structure of the TLR4 TIR domain homodimer and the Mal and TRAM adaptors</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>The structure of the receptor TIR domains from TLR1 and 2 was solved previously by X-ray crystallography <a href="#pone.0000788-Xu1">[19]</a>. These modules behave as monomers in solution and the packing of the molecules in the crystal lattice does not suggest a likely arrangement for the functional homodimer generated during receptor activation. More recently in the course of a structural genomics project, the TIR domain of TLR10 has been solved (<a href="http://sgc.utoronto.ca/SGC-WebPages/StructureDescription/2J67.php">http://sgc.utoronto.ca/SGC-WebPages/Stru​ctureDescription/2J67.php</a>). In contrast to TLR1 and 2, the TLR10 TIR domain is present as a dimer in the asymmetric unit. The molecules are related by a two-fold axis of symmetry and there is an extensive dimer interface. In addition, the region of the TIR domain involved in dimer formation includes a surface structural element called the BB-loop that links the second β-sheet to the second α-helix. The BB loop is thought to be important for homodimer formation because mutations of a conserved proline residue inactivate signalling and exert dominant negative effects, indicating the formation of non-functional heterodimers of the wild-type and mutant receptor chain. Thus it is likely that the TLR10 TIR structure represents a physiologically relevant conformation.</p>
<a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2"></a><p>In light of this, and the widely held view that signal induced dimerization of the receptor cytoplasmic domains is required for signalling <a href="#pone.0000788-Gay2">[12]</a>, we decided to model the TLR4 TIR domain as a homodimer, reasoning that this structure would approximate the configuration adopted in the activated receptor complex. This approach allows significant advances to be made in the interpretation of our modelling results over previous work in which the TLR4 TIR was modelled as a monomer <a href="#pone.0000788-Dunne1">[16]</a>. The crystal structures of TLR1 and TLR2 (Pdb: 1FYV and 1FYW) together with the recently released crystal structure of the human TLR10 dimer (Pdb: 2J67) were used as templates to build the TLR4 structural model (<a href="#pone-0000788-g002">Figure 2</a>). In the alignments used for the modelling sequence identity between TLR4 and TLR1, TLR2 and TLR10 was 35.0%, 40.5% and 35.0%, respectively. The secondary structure of the modelled TLR4 TIR domain is similar to that of the templates. It is predicted to contain five β strands and six α helices, with the BB loop adopting a similar conformation to that in the template structures (<a href="#pone-0000788-g002">Figures 2</a>, <a href="#pone-0000788-g003">3A and 3B</a>). Analysis of the structural model of the TLR4 dimer reveals that 94.2% of the phi-psi dihedral angles are found in the most favourable regions of the Ramachandran plot. The remaining 5.8% are found within the allowed regions. This highlights the excellent geometry of the model. Verify3D <a href="#pone.0000788-Luthy1">[20]</a> reports no values below 0.21, further indicating that all the residues are located in favourable structural environments. The JOY output also shows that the residues in the model are in environments similar to those of the templates (<a href="#pone-0000788-g002">Figure 2</a>).</p>
<div class="figure" id="pone-0000788-g002"><div class="img"><a name="pone-0000788-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000788.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000788" data-uri="info:doi/10.1371/journal.pone.0000788.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000788.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000788.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000788.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000788.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000788.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000788.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000788.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000788.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000788.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Structure based sequence alignments of TIR domains.</span></strong></p><a id="article1.body1.sec2.sec1.fig1.caption1.p1" name="article1.body1.sec2.sec1.fig1.caption1.p1"></a><p>The program JOY was used to annotate the alignments for TLR1, TLR2 TLR4, TLR10, Mal and TRAM. Numbers on top of amino acid sequences are alignment positions. The key to JOY annotations is as follows (a graphical version is viewable as <a href="#pone.0000788.s001">Table S1</a>); solvent inaccessible – UPPER CASE; solvent accessible – lower case; α-helix – dark grey shaded; β-strand – mid-grey shaded; 3<sub>10</sub> helix – light grey shaded; hydrogen bond to main chain amide – bold; hydrogen bond to main chain carbonyl – <span class="underline">underline</span>; hydrogen bond to other sidechain – tilde; disulphide bond – cedilla; positive φ---&gt;φ - <em>italic</em>; <em>cis</em>-peptide – breve.</p>
<span>doi:10.1371/journal.pone.0000788.g002</span></div><a id="article1.body1.sec2.sec1.p3" name="article1.body1.sec2.sec1.p3"></a><p>The surface area buried at the dimer interface of the TLR4 homodimer has contributions of 657 Å<sup>2</sup> and 665 Å<sup>2</sup> from each of the two protomers. <a href="#pone-0000788-t001">Table 1</a> lists the 18 residues that make van der Waals contacts with residues from the other chain in the dimer interface. Interestingly, some interactions are non-reciprocal. For example, Arg745 from chain A makes an interaction that buries 42.4 Å<sup>2</sup> of surface but the corresponding residue in chain B buries only 8.6 Å<sup>2</sup>. Similarly, Arg780 from chain A buries a surface of 30.8 Å<sup>2</sup> whilst the same residue from chain B does not interact. Weak inter-subunit electrostatic interactions are also observed. The strongest is between Glu750 from one chain and Arg780 from the other. However, the distance between charged atoms is 9.6 Å. Significantly the BB loops from the two protomers also interact with each other. In particular, the residue Phe712 from each subunit forms an aromatic interaction with one another. Interestingly, the dimer has a flat but slightly curved surface (<a href="#pone-0000788-g003">Figure 3B – side view</a>) which may be the top (or membrane proximal) surface of the structure (see <a href="#s3">discussion</a>).</p>
<div class="figure" id="pone-0000788-g003"><div class="img"><a name="pone-0000788-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000788.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000788" data-uri="info:doi/10.1371/journal.pone.0000788.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000788.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000788.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000788.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000788.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000788.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000788.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000788.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000788.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000788.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Structural modelling of the TLR4 TIR domain homodimer, Mal and TRAM.</span></strong></p><a id="article1.body1.sec2.sec1.fig2.caption1.p1" name="article1.body1.sec2.sec1.fig2.caption1.p1"></a><p>The BB loops of the two TLR4 protomers are coloured blue and yellow respectively. For Mal and TRAM they are coloured green (A) TLR4 top view. (B) TLR4 side view. (C) Mal. (D) TRAM.</p>
<span>doi:10.1371/journal.pone.0000788.g003</span></div><div class="figure" id="pone-0000788-t001"><div class="img"><a name="pone-0000788-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000788.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000788" data-uri="info:doi/10.1371/journal.pone.0000788.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000788.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000788.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000788.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000788.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000788.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000788.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000788.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000788.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000788.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>Interacting residues in the dimer interface of TLR4.</span></strong></p><span>doi:10.1371/journal.pone.0000788.t001</span></div><a id="article1.body1.sec2.sec1.p4" name="article1.body1.sec2.sec1.p4"></a><p>We have also generated high quality models of the Mal and TRAM adaptor TIR domains (<a href="#pone-0000788-g003">Figure 3C, D</a>), using the crystal structures of TLR1 and 2 as templates (<a href="#pone-0000788-g002">Figure 2</a>). In the alignments used Mal displays amino acid sequence identity with TLR1 and TLR2 of 21.1% and 23.0%, respectively. As expected the secondary structure of the Mal TIR domain is similar to that of the templates with the exception of a nine amino acid deletion before the start of the 4<sup>th</sup> helical segment (alignment position 114 to 122 in <a href="#pone-0000788-g002">Figure 2</a>). This deletion results in structural distortion and loss of the αD helix and a small β strand. The model of the Mal TIR domain also has good geometry. None of its phi-psi dihedral angles are found within disallowed regions of the Ramachandran plot, whilst 90.6% are in geometrically favoured regions. The JOY and Verify3D outputs show that the environments of residues in the model of Mal are similar to those of the templates and are not energetically unfavourable (<a href="#pone-0000788-g002">Figure 2</a>). Overall the structure of Mal is a four-stranded β sheet surrounded by five α helices in which the BB loop adopts a fixed conformation similar to that seen in TLR1 and 2. A similar strategy was used to model the structure of the TRAM adaptor. In this instance the amino acid sequence identity between the modelled regions of TRAM and TLR1 and TLR2 are 15.5% and 15.9%, respectively. With the exception of a ten amino acid insertion before the last small β strand (alignment position 136 to 145 in <a href="#pone-0000788-g002">Figure 2</a>) the secondary structure of the TRAM TIR domain is similar to that of the templates. The phi-psi dihedral angles of the TRAM model show good geometry. The Ramachandran plot contains 88.5% in the most favourable regions, 10.9% in allowed regions and only a single residue (0.6%) in disallowed regions. As with Mal and TLR4 the residues in the TRAM model are predicted to be in structurally favourable environments.</p>


<h4>The MAL and TRAM adaptors are predicted to bind at the TLR4 dimer interface</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>We have used GRAMM <a href="#pone.0000788-Vakser1">[21]</a>, a molecular docking programme that uses shape complementarity to assess the interaction of protein molecules, to probe the likely binding sites in the TLR4 dimer for the adaptors Mal and TRAM (<a href="#pone-0000788-g004">Figure 4A, B</a>). Consideration of TLR4 as a dimer creates a more physiologically relevant receptor docking template compared with earlier studies in which the TLR4 receptor was modelled as a monomer <a href="#pone.0000788-Dunne1">[16]</a>. Remarkably, the 100 best low resolution docking solutions for Mal lie at the TLR4 dimer interface indicating that dimer formation creates two specific symmetry related binding sites for this molecule. The TRAM adaptor is predicted to bind to the same site as Mal. The interaction surfaces on the TLR4 dimer interface are at either side of the structure rather than at the top, a region that would be sterically hindered by the membrane.</p>
<div class="figure" id="pone-0000788-g004"><div class="img"><a name="pone-0000788-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000788.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000788" data-uri="info:doi/10.1371/journal.pone.0000788.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000788.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000788.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000788.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000788.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000788.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000788.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000788.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000788.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000788.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>Docking studies predict that the adaptors bind at the side of the TLR4 homodimer interface.</span></strong></p><a id="article1.body1.sec2.sec2.fig1.caption1.p1" name="article1.body1.sec2.sec2.fig1.caption1.p1"></a><p>The TLR4 protomers, represented as ribbon diagrams are in green and cyan. Docked Mal and TRAM are represented as stick models and the 50 best docking solutions generated by GRAMM for either Mal (A) or TRAM (B) have been superimposed upon one another. (C) High resolution complex of TLR4 dimer (green and cyan), Mal (pink) and TRAM (yellow). The position of each BB loop is labelled.</p>
<span>doi:10.1371/journal.pone.0000788.g004</span></div><a id="article1.body1.sec2.sec2.p2" name="article1.body1.sec2.sec2.p2"></a><p>We then carried out high resolution docking experiments using the programme PyDock <a href="#pone.0000788-ManKuangCheng1">[22]</a>. These experiments were guided through the use of new, and previously published, information from mutagenesis studies. In particular we took account of mutants located on the side surface of the dimer (His728Asp, Trp757Ala, Gln683Ala, IleIle723-724AlaAla, His724Ala, GlyPhe726-728AlaAla, Gln758Ala, Arg763Ala; see <a href="#pone-0000788-t002">Table 2</a>, <a href="#pone-0000788-t003">3</a>; and ref <a href="#pone.0000788-Ronni1">[23]</a>). We also considered the impact of reported contributions from Mal (BB loop residues and a tyrosine phosphorylation site, Tyr86, located close to the BB loop, see <a href="#s3">discussion</a>). The high resolution model of the TLR4 homodimer with Mal and TRAM bound is shown in <a href="#pone-0000788-g004">Figure 4C</a> with Mal in magenta and TRAM in yellow.</p>
<div class="figure" id="pone-0000788-t002"><div class="img"><a name="pone-0000788-t002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000788.t002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000788" data-uri="info:doi/10.1371/journal.pone.0000788.t002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000788.t002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000788.t002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000788.t002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000788.t002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000788.t002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000788.t002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000788.t002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000788.t002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000788.t002.TIF"></span>)
                </a></li></ul></div><p><strong>Table 2.  <span>Potential structural impact of TLR4 TIR mutations.</span></strong></p><span>doi:10.1371/journal.pone.0000788.t002</span></div><div class="figure" id="pone-0000788-t003"><div class="img"><a name="pone-0000788-t003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000788.t003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000788" data-uri="info:doi/10.1371/journal.pone.0000788.t003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000788.t003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000788.t003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000788.t003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000788.t003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000788.t003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000788.t003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000788.t003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000788.t003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000788.t003.TIF"></span>)
                </a></li></ul></div><p><strong>Table 3.  <span>Summary of effects of mutation on NFκB and IFN-β activation.</span></strong></p><span>doi:10.1371/journal.pone.0000788.t003</span></div><a id="article1.body1.sec2.sec2.p3" name="article1.body1.sec2.sec2.p3"></a><p>The buried surface at the interface of the TLR4 dimer-Mal complex constitutes 1824 Å<sup>2</sup> from the TLR4 dimer and 1761 Å<sup>2</sup> from Mal. There are 17 residues from TLR4 chain A and 28 residues from chain B that make contacts (ΔASA&gt;1 Å<sup>2</sup>) with 49 residues from Mal. <a href="#pone-0000788-t004">Table 4</a> shows those residues that produce the strongest van der Waals interactions (ΔASA&gt;40 Å<sup>2</sup>) in the complex interface. For TLR4 this is Trp757, and for Mal Glu167. Seven hydrogen bonds are present in the interface of the TLR4 dimer-Mal complex, donated by three residues from each TLR4 chain and six residues from Mal. This includes a double hydrogen bond between TLR4 His728 and Mal Thr166. Strong electrostatic interactions are present between charged residues from both components of the complex. The strongest salt bridge is between the TLR4 Arg763 and Mal Glu167. Also important is the ion pair between TLR4 Lys819 and Mal Glu167.</p>
<div class="figure" id="pone-0000788-t004"><div class="img"><a name="pone-0000788-t004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000788.t004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000788" data-uri="info:doi/10.1371/journal.pone.0000788.t004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000788.t004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000788.t004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000788.t004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000788.t004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000788.t004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000788.t004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000788.t004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000788.t004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000788.t004.TIF"></span>)
                </a></li></ul></div><p><strong>Table 4.  <span>Residues that produce strong interactions (ΔASA&gt;40 Å<sup>2</sup>) in the interface of the TLR4 dimer-Mal complex.</span></strong></p><span>doi:10.1371/journal.pone.0000788.t004</span></div><a id="article1.body1.sec2.sec2.p4" name="article1.body1.sec2.sec2.p4"></a><p>The interface of the TLR4 dimer-TRAM complex results in a buried surface of 1729 Å<sup>2</sup> from the TLR4 dimer and 1805 Å<sup>2</sup> from TRAM. TLR4 chain A and B contribute 33 and 9 residues respectively, which form contacts (ΔASA&gt;1 Å<sup>2</sup>) with 43 residues from TRAM. <a href="#pone-0000788-t005">Table 5</a> shows those residues that produce strong van der Waals interactions (ΔASA&gt;40 Å<sup>2</sup>) in the complex interface. As with the Mal complex, TLR4 Trp757 makes the strongest interaction with TRAM, whilst TRAM Asn 159 interacts most strongly with TLR4. In this complex only three hydrogen bonds are present in the interface. TLR4 chain A donates three residues and TRAM two, of which one, Asp164, produces two strong hydrogen bonds. Strong electrostatic interactions are observed between charged residues from both components of the complex. Salt bridges are formed between TLR4 Glu824 and TRAM Arg231 and TLR4 Glu685 and TRAM Arg119. Also important is the ion pair between TLR4 Arg780 and TRAM Glu75.</p>
<div class="figure" id="pone-0000788-t005"><div class="img"><a name="pone-0000788-t005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000788.t005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000788" data-uri="info:doi/10.1371/journal.pone.0000788.t005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000788.t005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000788.t005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000788.t005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000788.t005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000788.t005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000788.t005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000788.t005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000788.t005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000788.t005.TIF"></span>)
                </a></li></ul></div><p><strong>Table 5.  <span>Residues that produce strong interactions (ΔASA&gt;40 Å<sup>2</sup>) in the interface of the TLR4 dimer-TRAM complex.</span></strong></p><span>doi:10.1371/journal.pone.0000788.t005</span></div><a id="article1.body1.sec2.sec2.p5" name="article1.body1.sec2.sec2.p5"></a><p>We next used the FPSPD program (see <a href="#s4">Methods</a>) to predict functional sites in TLR4, Mal and TRAM. The patterns found for TLR4 were: residues 677–698, 705–730, 734–759 and 795–815. The first functional pattern 677–698 contains residues involved in Mal binding (<a href="#pone-0000788-t004">Table 4</a>). Pattern 705–730, which includes the BB loop, possesses residues for homodimerization, and TRAM binding (<a href="#pone-0000788-t001">Tables 1</a> and <a href="#pone-0000788-t005">5</a>) and pattern 734–759 is implicated in homodimerization and Mal binding (<a href="#pone-0000788-t001">Tables 1</a> and <a href="#pone-0000788-t004">4</a>). Finally, pattern 795–815 does not contribute to any of the interactions identified in this study but may represent a binding site for a second adaptor molecule such as MyD88. In the case of Mal, the functional patterns predicted by FPSPD are: residues 89–104, 108–122, 121–139 and 146–169. All these are directly involved in TLR4 binding except pattern 108–122. However, this one is spatially close to the TLR4 pattern 795–815 in the TLR4 dimer-Mal complex. Thus these patterns could both contribute to a binding site that is only formed after initial recruitment of Mal. In the case of TRAM the patterns identified are: residues 72–94, 117–127, 136–164, 171–181 and 218–226. Patterns 72–94 and 136–164 have involvement in TLR4 binding, see (<a href="#pone-0000788-t005">Table 5</a>). Only one residue from pattern 117–127 interacts with TLR4, and neither pattern 171–181 nor 218–226 contain residues that interact with TLR4. These regions may potentially be involved in binding of a second adaptor, for example TRIF. Interestingly, pattern 117–127 is spatially equivalent to the Mal pattern 108–122.</p>
<a id="article1.body1.sec2.sec2.p6" name="article1.body1.sec2.sec2.p6"></a><p>In conclusion, this modelling study predicts that homodimerization of the TLR4 TIR domain creates a scaffold for the recruitment of adaptor molecules into a post receptor complex. Remarkably the study predicts that the ‘BB’ loop structures of all three molecules are critical determinants of binding specificity (<a href="#pone-0000788-g004">Figure 4C</a>).</p>


<h4>Functional studies of the TLR4, MAL and TRAM TIR domains support the proposed mechanism of receptor activation</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>Although experimental structural analysis of TLR4 has proved difficult a number of studies have identified regions of the receptor and adaptor TIRs that are important for signalling function. In particular these studies have utilised mutagenesis, the use of cell permeable blocking peptides and studies of Mal tyrosine phosphorylation.</p>
<a id="article1.body1.sec2.sec3.p2" name="article1.body1.sec2.sec3.p2"></a><p><a href="#pone-0000788-t002">Table 2</a> summarises the results of mutagenesis studies on the TIR domain of TLR4 and the effect that the mutations are predicted to have on the receptor-adaptor complexes. This includes previously unpublished data together with the results reported by Ronni et al <a href="#pone.0000788-Ronni1">[23]</a>. In general the effect of mutating individual residues is variable, ranging from a substantial loss of function, such as the BB loop proline residue, to partial or no loss of signalling. Our analysis indicates that the mutants that impair function can be grouped into three classes. The first are those such as P714 that are likely to impair receptor homodimerization or cause the formation of non-functional receptor dimers that are not able to bind adaptors. These residues predominantly lie in two discrete regions of the molecule, residues 708–718, containing the BB-loop, and residues 744–755 which form the αC helix/CD loop. The second group are residues that lie in the putative adaptor binding surface created by the dimer interface. The modelling study identifies two other surfaces that may be interaction sites with adaptors or other protein factors involved in TLR4 signalling, one on the sides of the dimer and one on the bottom. These regions may represent secondary sites of interaction for MyD88 and TRIF and interestingly a previous study pointed to the CD loop located on the bottom surface as a binding site for MyD88 <a href="#pone.0000788-Dunne1">[16]</a>. The third group of mutants are those that make important intra-chain contacts and are probably required for the stability of the TIR domain fold. We have also assayed a number of mutant receptors for signalling to both NFκB (a Mal directed target) and IFNβ (a TRAM target). The results (<a href="#pone-0000788-t003">Table 3</a>) show that mutations on the whole have similar effects on both arms of the TLR4 pathway consistent with the Mal and TRAM adaptors binding to the same or overlapping site on the activated receptor.</p>
<a id="article1.body1.sec2.sec3.p3" name="article1.body1.sec2.sec3.p3"></a><p>In addition to mutagenesis, studies with cell permeable blocking peptides also provide support for the model proposed here. Peptides corresponding to the BB-loop of TLR4 strongly inhibit LPS induced responses mediated by both the TRAM and the Mal adaptors <a href="#pone.0000788-Toshchakov1">[24]</a>. According to our model this peptide would compete with the BB loops in the receptor TIRs and prevent or disrupt the formation of the homodimer. BB-loop peptides from Mal and TRAM also inhibit LPS signalling, again affecting both arms of the pathway <a href="#pone.0000788-Toshchakov2">[25]</a>. This result is also consistent with our model as it provides strong evidence that the BB-loops of Mal and TRAM play a critical role in binding to the activated receptor. As the peptides block both NFκB and IRF-3 directed responses this result also indicated that the binding sites for the two adaptors overlap. The same study also found that a BB-loop peptide from MyD88 inhibited signalling by LPS. This finding suggests that initial binding of the Mal or TRAM adaptors creates a new surface for association of MyD88 and TRIF with determinants contributed by both the receptor and the adaptor TIRs, possibly involving the bottom surface of the homodimer.</p>
<a id="article1.body1.sec2.sec3.p4" name="article1.body1.sec2.sec3.p4"></a><p>A third line of evidence in support of the model comes from studies of tyrosine phosphorylation. Mal is phosphorylated by the Bruton tyrosine kinase at positions 86, 106 and 187 and substitution of tyrosine with phenylalanine at positions 86 and 187 impairs LPS mediated signalling to NFκB <a href="#pone.0000788-Gray1">[26]</a>. As shown in <a href="#pone-0000788-g005">Figure 5</a>, Tyr86 is located close to the BB-loop and should interfere with binding of Mal to TLR4. In our complex Tyr86 makes an interaction that buries 33.0 Å<sup>2</sup> of surface. Interestingly, co-immunoprecipitation experiments indicate that the phospho forms of Mal do not bind to TLR4.</p>
<div class="figure" id="pone-0000788-g005"><div class="img"><a name="pone-0000788-g005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000788.g005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000788" data-uri="info:doi/10.1371/journal.pone.0000788.g005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000788.g005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000788.g005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000788.g005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000788.g005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000788.g005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000788.g005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000788.g005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000788.g005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000788.g005.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 5.  <span>Modelling suggests a molecular explanation for the caspase 1 dependence of Mal and the malfunctional human polymorphism Ser180Leu.</span></strong></p><a id="article1.body1.sec2.sec3.fig1.caption1.p1" name="article1.body1.sec2.sec3.fig1.caption1.p1"></a><p>The models are shown as van der Waal surface representations. (A) Side view showing the position of the BB loop (green) and the phosphorylated tyrosine, Tyr86. In the complex this part of Mal forms the interface with TLR4. The position of the α-E helix (red) which is cleaved out by caspase 1 is shown on the opposite surface to the BB loop. (B) Back view of Mal (rotated 90° to the right relative to (A)). (C), (D) Mal with the α-E helix removed highlighting the deep groove created and the exposed position of the otherwise buried Ser180 residue (yellow).</p>
<span>doi:10.1371/journal.pone.0000788.g005</span></div>
</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p>The structural modelling and functional studies presented here provide strong support for theoretical models of signal transduction by TLR4 and other Toll family receptors <a href="#pone.0000788-Gay2">[12]</a>. In this view, stimulus induced dimerization of the receptor extracellular domains leads to concerted protein conformational changes that in turn lead to self-association or rearrangement of the receptor TIRs thereby creating a new molecular surface for the recruitment of signalling adaptor proteins. In our model of the TLR4 homodimer the interface has extensive interactions involving the BB loops of the two subunits. The conserved proline residue will confer a rigid conformation on the BB-loop and substitution by other residues would cause considerable distortion in the geometry of the homodimer interface. Another important conclusion of this study is that the receptor TIRs associate with a 2-fold axis of symmetry. This implies that the linkers between the membrane and the TIR domains have rotational flexibility. This seems plausible as the linkers are fairly long (about 20 amino acids) and contain glycine residues that can adopt a wider range of dihedral angles than the chiral amino acids. Another interesting feature of the TLR4 dimer is the flat but slightly curved surface predicted to form the top or membrane proximal surface of the structure (<a href="#pone-0000788-g003">Figure 3B</a>). This architecture is seen in other proteins that interact with membrane surfaces, for example the BAR domain of amphiphysin <a href="#pone.0000788-Casal1">[27]</a>.</p>
<a id="article1.body1.sec3.p2" name="article1.body1.sec3.p2"></a><p>Our model also predicts that Mal and TRAM bind to the same region in the TLR4 dimer interface. This explains why cell permeable blocking peptides compete out both Mal and TRAM directed responses simultaneously <a href="#pone.0000788-Toshchakov1">[24]</a>, <a href="#pone.0000788-Toshchakov2">[25]</a>. However, the model does not resolve the question of whether a single activated receptor dimer can stimulate both the Mal and TRAM directed pathways simultaneously or whether adaptor engagement is mutually exclusive, something that would require positive cooperativity. Whilst we are currently addressing this question, it is clear however, that each activated receptor will have two symmetry related adaptor binding sites so in principle either hypothesis is feasible.</p>
<a id="article1.body1.sec3.p3" name="article1.body1.sec3.p3"></a><p>Previous studies have attempted to model the interactions of Mal and MyD88 with TLR4 and TLR2 <a href="#pone.0000788-Dunne1">[16]</a>. However, these used monomeric receptors in the modelling process. It is now widely believed that receptor activation leads to ligand induced dimerization <a href="#pone.0000788-Gay2">[12]</a>. Consequently our current studies have modelled the activated TLR4 receptor as a dimer and are therefore likely to be more physiologically relevant. This may explain the altered orientation of Mal binding compared to the previous study. Our observations are however broadly consistent with the earlier predicted location of a binding site for MyD88 on TLR4. Further, our work suggests that initial binding of the ‘bridging adaptor’ proteins may contribute to formation of an additional surface for secondary adaptor binding.</p>
<a id="article1.body1.sec3.p4" name="article1.body1.sec3.p4"></a><p>The predicted adaptor/receptor binding surface is extensive and is contributed to by many residues from both the receptor and the adaptors. This may explain why many mutations in the receptor TIR only partially impair signalling (see <a href="#pone-0000788-t002">Tables 2</a> and <a href="#pone-0000788-t003">3</a> and ref <a href="#pone.0000788-Ronni1">[23]</a>). Changing a single favourable receptor-adaptor contact might result in a decrease in affinity of the complex but would not abolish function completely.</p>
<a id="article1.body1.sec3.p5" name="article1.body1.sec3.p5"></a><p>The docking results suggest that Mal binds the TLR4 homodimer with a closer proximity and greater number of direct contacts than TRAM does (<a href="#pone-0000788-t004">Tables 4</a> and <a href="#pone-0000788-t005">5</a>). This may suggest a higher affinity for Mal binding and would perhaps explain why TRAM is not subjected to such high levels of cellular regulation and also why Mal/MyD88 pathways trigger early NF-κB expression, but TRAM/TRIF signalling results in late NF-κB expression. Moreover, phosphorylation by Bruton tyrosine kinase of Mal at Tyr86, a residue predicted to interact strongly with the TLR4 homodimer may render Mal-TLR4 interaction sterically unfeasible in addition to facilitating SOCS-1 mediated Mal polyubiquitination and degradation <a href="#pone.0000788-Mansell1">[28]</a>. Interestingly, the equivalent residue in TRAM is Phe78 suggesting that the binding between TRAM and the receptor dimer may be a relatively weaker hydrophobic interaction.</p>
<a id="article1.body1.sec3.p6" name="article1.body1.sec3.p6"></a><p>Two other recently published results concerning Mal are also of interest in the context of this model. Firstly, Miggin et al. <a href="#pone.0000788-Miggin1">[29]</a> have shown that Mal is cleaved at position 198 by the cysteine protease caspase 1 and that this processing is required for Mal to function in signalling by both TLR2 and TLR4. The cleavage releases a 4 kDa fragment from the C-terminus of Mal (see <a href="#pone-0000788-g002">Figure 2</a> and <a href="#pone-0000788-g005">5</a>) corresponding to the ‘EE’ loop and the last (E) α-helix of the TIR fold. As shown in <a href="#pone-0000788-g005">Figure 5</a>, the site of proteolysis is on the surface opposite the BB-loop and thus should not affect binding of Mal to the receptor dimer. One explanation for this finding might be that in the absence of caspase cleavage MyD88 cannot be recruited into the postreceptor complex by Mal. Plausibly MyD88 might displace the ‘E’ helix from Mal and bind into the cleft that is exposed (<a href="#pone-0000788-g005">Figure 5A–D</a>). The second study concerns a variant of Mal, Ser180Leu, which is common in the human population. When heterozygous this polymorphism protects against a range of microbial infections and cell based assays show that the mutation significantly impairs signalling by both TLR2 and TLR4 <a href="#pone.0000788-Khor1">[30]</a>. In our model of Mal this residue is buried in the structure but would be exposed after cleavage with caspase 1 (<a href="#pone-0000788-g005">Figure 5C, D</a>). Thus it is possible that the observed loss of function associated with Ser180Leu is caused by a defect in recruitment of MyD88 to caspase-1 cleaved Mal. With the replacement of serine by a large, hydrophobic leucine residue introducing an unfavourable interaction in the caspase-1 cleaved adaptor.</p>
<a id="article1.body1.sec3.p7" name="article1.body1.sec3.p7"></a><p>In conclusion, the current study provides a basis for future structural and functional studies of TLR4 activation. A long term objective is to carry out experimental structural analysis of activated membrane receptor complexes of TLR4 using low resolution techniques such image reconstruction of electron micrographs as well as protein crystallography.</p>
</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4>Molecular modelling</h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p>The amino acid sequences and three dimensional structures of the homologous proteins used as templates for comparative modelling were obtained from the Protein Data Bank (<a href="http://www.rcsb.org/pdb">http://www.rcsb.org/pdb</a>). Initial alignments between target protein and its templates were obtained using the program FUGUE <a href="#pone.0000788-Shi1">[31]</a>. FUGUE produces alignments by comparison of sequence profiles with structural profiles of homologous protein families taken from the HOMSTRAD database <a href="#pone.0000788-deBakker1">[32]</a>.</p>
<a id="article1.body1.sec4.sec1.p2" name="article1.body1.sec4.sec1.p2"></a><p>Models were produced using the program MODELLER <a href="#pone.0000788-Sali1">[33]</a>. MODELLER produces comparative models satisfying spatial restraints with simultaneous optimization of CHARMM energies <a href="#pone.0000788-Brooks1">[34]</a>. This method uses conjugate gradients and molecular dynamics with simulated annealing <a href="#pone.0000788-Sali1">[33]</a>. The form of the spatial restraints was obtained by statistical analysis of the relationship between pairs of homologous structures from a database of sequence alignments for 416 proteins of known 3D structure in 105 families. Comparative models were verified by the validation programs PROCHECK <a href="#pone.0000788-Laskowski1">[35]</a>, VERIFY3D <a href="#pone.0000788-Luthy1">[20]</a> and JOY <a href="#pone.0000788-Mizuguchi1">[36]</a>. The alignments were then manually modified as required and the modelling and validation process repeated. The process of modelling, validation and realignment was repeated until models with good geometry and conformation had been obtained.</p>
<a id="article1.body1.sec4.sec1.p3" name="article1.body1.sec4.sec1.p3"></a><p>Contact residues in the TLR4 homodimer and docked complexes were defined as the residues that possessed an interface solvent accessible surface area (ASA) that decreased (ΔASA) by more than 1 Å<sup>2</sup> on complexation <a href="#pone.0000788-Jones1">[37]</a>. The ASA was calculated using the Lee and Richards algorithm <a href="#pone.0000788-Lee1">[38]</a> developed by Richmond <a href="#pone.0000788-Richmond1">[39]</a>. HBPLUS was used for hydrogen bond definition <a href="#pone.0000788-McDonald1">[40]</a>. Electrostatic interactions were calculated using an in house program (ELECINT, R.N. Miguel, unpublished) using a dielectric constant distance dependent for electrostatic field calculation and pH = 7.4 for the calculation of charges of side chain atoms of charged residues using the Henderson-Hasselbalch equation.</p>


<h4>Protein docking and binding site prediction</h4>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1"></a><p>Low resolution protein-protein docking was carried out using Global Range Molecular Matching (GRAMM) methodology <a href="#pone.0000788-KatchalskiKatzir1">[41]</a>. GRAMM methodology is an empirical approach to smoothing the intermolecular energy function by changing the range of the atom-atom potentials. Low resolution docking is useful for determining possible relative positions of the two proteins in the complex. High resolution protein-protein docking was performed by the program PyDock <a href="#pone.0000788-ManKuangCheng1">[22]</a>. PyDock is a method for rigid-body protein-protein docking. It explores either FTDOCK or ZDOCK methods to generate conformations of complexes and uses a scoring function that applies electrostatics and desolvation energies to select the best solutions.</p>
<a id="article1.body1.sec4.sec2.p2" name="article1.body1.sec4.sec2.p2"></a><p>The FPSPD program <a href="#pone.0000788-Miguel1">[42]</a> was used for the identification of functional sites. FPSPD utilises environment-dependent substitution tables and evolutionary trace analysis to identify residues from a structurally-aligned homologous family of proteins that are unusually highly conserved. Solvent accessibility calculations are used to estimate the probability of residues and molecular patterns being directly involved in functional interactions. The three-dimensional structure is used to estimate the borders of the functional patterns.</p>


<h4>Plasmids and reagents</h4>
<a id="article1.body1.sec4.sec3.p1" name="article1.body1.sec4.sec3.p1"></a><p>pcDNA3TLR4, pcDNA3CD14, pEFIREMD2, pNFBluc and pRantes were as previously described <a href="#pone.0000788-Fitzgerald3">[43]</a>, <a href="#pone.0000788-Lin1">[44]</a>. Lipopolysaccharide (<em>Escherichia coli</em> serotype 0127;B8) was obtained from Sigma; recombinant human TNF and IFN were from R&amp;D Systems. The phRGTK plasmid (Promega) was used as a transfection control. Site-directed mutagenesis was performed on pCDNA3TLR4 using the Stratagene Quikchange procedure.</p>


<h4>Cell culture, transfections and luciferase assay</h4>
<a id="article1.body1.sec4.sec4.p1" name="article1.body1.sec4.sec4.p1"></a><p>HEK293 cells were maintained in Dulbecco's modified Eagle's medium supplemented with 10% fetal calf serum, 100 µg/ml penicillin/streptomycin and 2 mM L-glutamine at 37°C and 5% CO<sub>2</sub>. Cells were seeded at 10<sup>5</sup> ml<sup>−1</sup> 24 hr prior to transfection using Polyfect (Qiagen). Cells were stimulated 48 hrs post transfection. The Dual-Glo system from Promega was used for luciferase assays. Luminescence readings were obtained using a Lumister luminometer (BMG Labtech).</p>

</div>

<div id="section5" class="section"><a id="s5" name="s5" toc="s5" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0000788"></div><a name="pone.0000788.s001" id="pone.0000788.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000788.s001">Table S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1"></a><p class="preSiDOI">Joy alignment key</p>
<p class="siDoi">doi:10.1371/journal.pone.0000788.s001</p><a id="article1.body1.sec5.supplementary-material1.caption1.p2" name="article1.body1.sec5.supplementary-material1.caption1.p2"></a><p class="postSiDOI">(0.06 MB DOC)</p>
</div>



<div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: LO TM NG CB. Performed the experiments: TM RN JW JW HB. Analyzed the data: TM RN JW CB. Wrote the paper: TM RN JW NG CB.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0000788-Janeway1" id="pone.0000788-Janeway1"></a>Janeway CA Jr, Medzhitov R (2002) Innate immune recognition. Annu Rev Immunol  20: 197–216.  <ul class="find" data-citedArticleID="982364" data-doi="10.1146/annurev.immunol.20.083001.084359"><li><a href="http://dx.doi.org/10.1146/annurev.immunol.20.083001.084359" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Innate+immune+recognition." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Innate+immune+recognition.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0000788-Akira1" id="pone.0000788-Akira1"></a>Akira S, Takeda K (2004) Toll-like receptor signalling. Nat Rev Immunol  4: 499–511.  <ul class="find" data-citedArticleID="982340" data-doi="10.1038/nri1391"><li><a href="http://dx.doi.org/10.1038/nri1391" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Toll-like+receptor+signalling." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Toll-like+receptor+signalling.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0000788-Gay1" id="pone.0000788-Gay1"></a>Gay NJ, Gangloff M (2007) Structure and function of toll receptors and their ligands. Annu Rev Biochem  76: 141–165.  <ul class="find" data-citedArticleID="982356" data-doi="10.1146/annurev.biochem.76.060305.151318"><li><a href="http://dx.doi.org/10.1146/annurev.biochem.76.060305.151318" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Structure+and+function+of+toll+receptors+and+their+ligands." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Structure+and+function+of+toll+receptors+and+their+ligands.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0000788-Raetz1" id="pone.0000788-Raetz1"></a>Raetz CR, Ulevitch RJ, Wright SD, Sibley CH, Ding A, et al.  (1991) Gram-negative endotoxin: an extraordinary lipid with profound effects on eukaryotic signal transduction. Faseb J  5: 2652–2660.  <ul class="find" data-citedArticleID="982402"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Gram-negative+endotoxin%3A+an+extraordinary+lipid+with+profound+effects+on+eukaryotic+signal+transduction.&amp;auth=&amp;atitle=Gram-negative+endotoxin%3A+an+extraordinary+lipid+with+profound+effects+on+eukaryotic+signal+transduction." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Gram-negative+endotoxin%3A+an+extraordinary+lipid+with+profound+effects+on+eukaryotic+signal+transduction." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Gram-negative+endotoxin%3A+an+extraordinary+lipid+with+profound+effects+on+eukaryotic+signal+transduction.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0000788-Hunter1" id="pone.0000788-Hunter1"></a>Hunter P (2006) Sepsis under siege: a new understanding of sepsis might lead to the development of therapies to treat septic shock. EMBO Rep  7: 667–669.  <ul class="find" data-citedArticleID="982362" data-doi="10.1038/sj.embor.7400742"><li><a href="http://dx.doi.org/10.1038/sj.embor.7400742" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Sepsis+under+siege%3A+a+new+understanding+of+sepsis+might+lead+to+the+development+of+therapies+to+treat+septic+shock." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Sepsis+under+siege%3A+a+new+understanding+of+sepsis+might+lead+to+the+development+of+therapies+to+treat+septic+shock.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0000788-Poltorak1" id="pone.0000788-Poltorak1"></a>Poltorak A, He X, Smirnova I, Liu MY, Van Huffel C, et al.  (1998) Defective LPS signaling in C3H/HeJ and C57BL/10ScCr mice: mutations in Tlr4 gene. Science  282: 2085–2088.  <ul class="find" data-citedArticleID="982400" data-doi="10.1126/science.282.5396.2085"><li><a href="http://dx.doi.org/10.1126/science.282.5396.2085" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Defective+LPS+signaling+in+C3H%2FHeJ+and+C57BL%2F10ScCr+mice%3A+mutations+in+Tlr4+gene." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Defective+LPS+signaling+in+C3H%2FHeJ+and+C57BL%2F10ScCr+mice%3A+mutations+in+Tlr4+gene.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0000788-Nagai1" id="pone.0000788-Nagai1"></a>Nagai Y, Akashi S, Nagafuku M, Ogata M, Iwakura Y, et al.  (2002) Essential role of MD-2 in LPS responsiveness and TLR4 distribution. Nat Immunol  3: 667–672.  <ul class="find" data-citedArticleID="982396" data-doi="10.1038/ni809"><li><a href="http://dx.doi.org/10.1038/ni809" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Essential+role+of+MD-2+in+LPS+responsiveness+and+TLR4+distribution." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Essential+role+of+MD-2+in+LPS+responsiveness+and+TLR4+distribution.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0000788-Schromm1" id="pone.0000788-Schromm1"></a>Schromm AB, Lien E, Henneke P, Chow JC, Yoshimura A, et al.  (2001) Molecular genetic analysis of an endotoxin nonresponder mutant cell line: a point mutation in a conserved region of MD-2 abolishes endotoxin-induced signaling. J Exp Med  194: 79–88.  <ul class="find" data-citedArticleID="982414" data-doi="10.1084/jem.194.1.79"><li><a href="http://dx.doi.org/10.1084/jem.194.1.79" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Molecular+genetic+analysis+of+an+endotoxin+nonresponder+mutant+cell+line%3A+a+point+mutation+in+a+conserved+region+of+MD-2+abolishes+endotoxin-induced+signaling." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Molecular+genetic+analysis+of+an+endotoxin+nonresponder+mutant+cell+line%3A+a+point+mutation+in+a+conserved+region+of+MD-2+abolishes+endotoxin-induced+signaling.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0000788-Gangloff1" id="pone.0000788-Gangloff1"></a>Gangloff M, Gay NJ (2004) MD-2: the Toll ‘gatekeeper’ in endotoxin signalling. Trends Biochem Sci  29: 294–300.  <ul class="find" data-citedArticleID="982354" data-doi="10.1016/j.tibs.2004.04.008"><li><a href="http://dx.doi.org/10.1016/j.tibs.2004.04.008" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=MD-2%3A+the+Toll+%26%23x2018%3Bgatekeeper%26%23x2019%3B+in+endotoxin+signalling." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22MD-2%3A+the+Toll+%26%23x2018%3Bgatekeeper%26%23x2019%3B+in+endotoxin+signalling.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0000788-Miyake1" id="pone.0000788-Miyake1"></a>Miyake K (2004) Innate recognition of lipopolysaccharide by Toll-like receptor 4-MD-2. Trends Microbiol  12: 186–192.  <ul class="find" data-citedArticleID="982392" data-doi="10.1016/j.tim.2004.02.009"><li><a href="http://dx.doi.org/10.1016/j.tim.2004.02.009" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Innate+recognition+of+lipopolysaccharide+by+Toll-like+receptor+4-MD-2." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Innate+recognition+of+lipopolysaccharide+by+Toll-like+receptor+4-MD-2.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0000788-Saitoh1" id="pone.0000788-Saitoh1"></a>Saitoh S, Akashi S, Yamada T, Tanimura N, Kobayashi M, et al.  (2004) Lipid A antagonist, lipid IVa, is distinct from lipid A in interaction with Toll-like receptor 4 (TLR4)-MD-2 and ligand-induced TLR4 oligomerization. Int Immunol  16: 961–969.  <ul class="find" data-citedArticleID="982410" data-doi="10.1093/intimm/dxh097"><li><a href="http://dx.doi.org/10.1093/intimm/dxh097" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Lipid+A+antagonist%2C+lipid+IVa%2C+is+distinct+from+lipid+A+in+interaction+with+Toll-like+receptor+4+%28TLR4%29-MD-2+and+ligand-induced+TLR4+oligomerization." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Lipid+A+antagonist%2C+lipid+IVa%2C+is+distinct+from+lipid+A+in+interaction+with+Toll-like+receptor+4+%28TLR4%29-MD-2+and+ligand-induced+TLR4+oligomerization.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0000788-Gay2" id="pone.0000788-Gay2"></a>Gay NJ, Gangloff M, Weber AN (2006) Toll-like receptors as molecular switches. Nat Rev Immunol  6: 693–698.  <ul class="find" data-citedArticleID="982358" data-doi="10.1038/nri1916"><li><a href="http://dx.doi.org/10.1038/nri1916" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Toll-like+receptors+as+molecular+switches." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Toll-like+receptors+as+molecular+switches.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0000788-Latz1" id="pone.0000788-Latz1"></a>Latz E, Verma A, Visintin A, Gong M, Sirois CM, et al.  (2007) Ligand-induced conformational changes allosterically activate Toll-like receptor 9. Nat Immunol  8: 772–779.  <ul class="find" data-citedArticleID="982374" data-doi="10.1038/ni1479"><li><a href="http://dx.doi.org/10.1038/ni1479" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Ligand-induced+conformational+changes+allosterically+activate+Toll-like+receptor+9." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Ligand-induced+conformational+changes+allosterically+activate+Toll-like+receptor+9.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0000788-ONeill1" id="pone.0000788-ONeill1"></a>O'Neill LA, Fitzgerald KA, Bowie AG (2003) The Toll-IL-1 receptor adaptor family grows to five members. Trends Immunol  24: 286–290.  <ul class="find" data-citedArticleID="982398" data-doi="10.1016/s1471-4906(03)00115-7"><li><a href="http://dx.doi.org/10.1016/s1471-4906(03)00115-7" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+Toll-IL-1+receptor+adaptor+family+grows+to+five+members." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+Toll-IL-1+receptor+adaptor+family+grows+to+five+members.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0000788-Rock1" id="pone.0000788-Rock1"></a>Rock FL, Hardiman G, Timans JC, Kastelein RA, Bazan JF (1998) A family of human receptors structurally related to Drosophila Toll. Proc Natl Acad Sci U S A  95: 588–593.  <ul class="find" data-citedArticleID="982406" data-doi="10.1073/pnas.95.2.588"><li><a href="http://dx.doi.org/10.1073/pnas.95.2.588" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+family+of+human+receptors+structurally+related+to+Drosophila+Toll." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+family+of+human+receptors+structurally+related+to+Drosophila+Toll.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0000788-Dunne1" id="pone.0000788-Dunne1"></a>Dunne A, Ejdeback M, Ludidi PL, O'Neill LA, Gay NJ (2003) Structural complementarity of Toll/interleukin-1 receptor domains in Toll-like receptors and the adaptors Mal and MyD88. J Biol Chem  278: 41443–41451.  <ul class="find" data-citedArticleID="982346" data-doi="10.1074/jbc.m301742200"><li><a href="http://dx.doi.org/10.1074/jbc.m301742200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Structural+complementarity+of+Toll%2Finterleukin-1+receptor+domains+in+Toll-like+receptors+and+the+adaptors+Mal+and+MyD88." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Structural+complementarity+of+Toll%2Finterleukin-1+receptor+domains+in+Toll-like+receptors+and+the+adaptors+Mal+and+MyD88.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0000788-Fitzgerald1" id="pone.0000788-Fitzgerald1"></a>Fitzgerald KA, Palsson-McDermott EM, Bowie AG, Jefferies CA, Mansell AS, et al.  (2001) Mal (MyD88-adapter-like) is required for Toll-like receptor-4 signal transduction. Nature  413: 78–83.  <ul class="find" data-citedArticleID="982348"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Mal+%28MyD88-adapter-like%29+is+required+for+Toll-like+receptor-4+signal+transduction.&amp;auth=&amp;atitle=Mal+%28MyD88-adapter-like%29+is+required+for+Toll-like+receptor-4+signal+transduction." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mal+%28MyD88-adapter-like%29+is+required+for+Toll-like+receptor-4+signal+transduction." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mal+%28MyD88-adapter-like%29+is+required+for+Toll-like+receptor-4+signal+transduction.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0000788-Fitzgerald2" id="pone.0000788-Fitzgerald2"></a>Fitzgerald KA, Rowe DC, Barnes BJ, Caffrey DR, Visintin A, et al.  (2003) LPS-TLR4 signaling to IRF-3/7 and NF-kappaB involves the toll adapters TRAM and TRIF. J Exp Med  198: 1043–1055.  <ul class="find" data-citedArticleID="982350" data-doi="10.1084/jem.20031023"><li><a href="http://dx.doi.org/10.1084/jem.20031023" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=LPS-TLR4+signaling+to+IRF-3%2F7+and+NF-kappaB+involves+the+toll+adapters+TRAM+and+TRIF." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22LPS-TLR4+signaling+to+IRF-3%2F7+and+NF-kappaB+involves+the+toll+adapters+TRAM+and+TRIF.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0000788-Xu1" id="pone.0000788-Xu1"></a>Xu Y, Tao X, Shen B, Horng T, Medzhitov R, et al.  (2000) Structural basis for signal transduction by the Toll/interleukin-1 receptor domains. Nature  408: 111–115.  <ul class="find" data-citedArticleID="982426" data-doi="10.1038/35040600"><li><a href="http://dx.doi.org/10.1038/35040600" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Structural+basis+for+signal+transduction+by+the+Toll%2Finterleukin-1+receptor+domains." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Structural+basis+for+signal+transduction+by+the+Toll%2Finterleukin-1+receptor+domains.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0000788-Luthy1" id="pone.0000788-Luthy1"></a>Luthy R, Bowie JU, Eisenberg D (1992) Assessment of protein models with three-dimensional profiles. Nature  356: 83–85.  <ul class="find" data-citedArticleID="982380" data-doi="10.1038/356083a0"><li><a href="http://dx.doi.org/10.1038/356083a0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Assessment+of+protein+models+with+three-dimensional+profiles." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Assessment+of+protein+models+with+three-dimensional+profiles.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0000788-Vakser1" id="pone.0000788-Vakser1"></a>Vakser IA (1997) Evaluation of GRAMM low-resolution docking methodology on the hemagglutinin-antibody complex. Proteins Suppl  1: 226–230.  <ul class="find" data-citedArticleID="982422" data-doi="10.1002/(sici)1097-0134(1997)1+<226::aid-prot31&gt;3.3.co;2-0"><li><a href="http://dx.doi.org/10.1002/(sici)1097-0134(1997)1+<226::aid-prot31&gt;3.3.co;2-0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evaluation+of+GRAMM+low-resolution+docking+methodology+on+the+hemagglutinin-antibody+complex." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evaluation+of+GRAMM+low-resolution+docking+methodology+on+the+hemagglutinin-antibody+complex.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0000788-ManKuangCheng1" id="pone.0000788-ManKuangCheng1"></a>Man-Kuang Cheng T, Blundell TL, Fernandez-Recio J (2007) pyDock: electrostatics and desolvation for effective scoring of rigid-body protein-protein docking. Proteins  68: 503–515.  <ul class="find" data-citedArticleID="982382" data-doi="10.1002/prot.21419"><li><a href="http://dx.doi.org/10.1002/prot.21419" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=pyDock%3A+electrostatics+and+desolvation+for+effective+scoring+of+rigid-body+protein-protein+docking." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22pyDock%3A+electrostatics+and+desolvation+for+effective+scoring+of+rigid-body+protein-protein+docking.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0000788-Ronni1" id="pone.0000788-Ronni1"></a>Ronni T, Agarwal V, Haykinson M, Haberland ME, Cheng G, et al.  (2003) Common interaction surfaces of the toll-like receptor 4 cytoplasmic domain stimulate multiple nuclear targets. Mol Cell Biol  23: 2543–2555.  <ul class="find" data-citedArticleID="982408" data-doi="10.1128/mcb.23.7.2543-2555.2003"><li><a href="http://dx.doi.org/10.1128/mcb.23.7.2543-2555.2003" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Common+interaction+surfaces+of+the+toll-like+receptor+4+cytoplasmic+domain+stimulate+multiple+nuclear+targets." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Common+interaction+surfaces+of+the+toll-like+receptor+4+cytoplasmic+domain+stimulate+multiple+nuclear+targets.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0000788-Toshchakov1" id="pone.0000788-Toshchakov1"></a>Toshchakov VY, Fenton MJ, Vogel SN (2007) Cutting Edge: Differential inhibition of TLR signaling pathways by cell-permeable peptides representing BB loops of TLRs. J Immunol  178: 2655–2660.  <ul class="find" data-citedArticleID="982418"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Cutting+Edge%3A+Differential+inhibition+of+TLR+signaling+pathways+by+cell-permeable+peptides+representing+BB+loops+of+TLRs.&amp;auth=&amp;atitle=Cutting+Edge%3A+Differential+inhibition+of+TLR+signaling+pathways+by+cell-permeable+peptides+representing+BB+loops+of+TLRs." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Cutting+Edge%3A+Differential+inhibition+of+TLR+signaling+pathways+by+cell-permeable+peptides+representing+BB+loops+of+TLRs." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Cutting+Edge%3A+Differential+inhibition+of+TLR+signaling+pathways+by+cell-permeable+peptides+representing+BB+loops+of+TLRs.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0000788-Toshchakov2" id="pone.0000788-Toshchakov2"></a>Toshchakov VU, Basu S, Fenton MJ, Vogel SN (2005) Differential involvement of BB loops of toll-IL-1 resistance (TIR) domain-containing adapter proteins in TLR4- versus TLR2-mediated signal transduction. J Immunol  175: 494–500.  <ul class="find" data-citedArticleID="982420"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Differential+involvement+of+BB+loops+of+toll-IL-1+resistance+%28TIR%29+domain-containing+adapter+proteins+in+TLR4-+versus+TLR2-mediated+signal+transduction.&amp;auth=&amp;atitle=Differential+involvement+of+BB+loops+of+toll-IL-1+resistance+%28TIR%29+domain-containing+adapter+proteins+in+TLR4-+versus+TLR2-mediated+signal+transduction." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Differential+involvement+of+BB+loops+of+toll-IL-1+resistance+%28TIR%29+domain-containing+adapter+proteins+in+TLR4-+versus+TLR2-mediated+signal+transduction." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Differential+involvement+of+BB+loops+of+toll-IL-1+resistance+%28TIR%29+domain-containing+adapter+proteins+in+TLR4-+versus+TLR2-mediated+signal+transduction.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0000788-Gray1" id="pone.0000788-Gray1"></a>Gray P, Dunne A, Brikos C, Jefferies CA, Doyle SL, et al.  (2006) MyD88 adapter-like (Mal) is phosphorylated by Bruton's tyrosine kinase during TLR2 and TLR4 signal transduction. J Biol Chem  281: 10489–10495.  <ul class="find" data-citedArticleID="982360" data-doi="10.1074/jbc.m508892200"><li><a href="http://dx.doi.org/10.1074/jbc.m508892200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=MyD88+adapter-like+%28Mal%29+is+phosphorylated+by+Bruton%27s+tyrosine+kinase+during+TLR2+and+TLR4+signal+transduction." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22MyD88+adapter-like+%28Mal%29+is+phosphorylated+by+Bruton%27s+tyrosine+kinase+during+TLR2+and+TLR4+signal+transduction.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0000788-Casal1" id="pone.0000788-Casal1"></a>Casal E, Federici L, Zhang W, Fernandez-Recio J, Priego EM, et al.  (2006) The crystal structure of the BAR domain from human Bin1/amphiphysin II and its implications for molecular recognition. Biochemistry  45: 12917–12928.  <ul class="find" data-citedArticleID="982344" data-doi="10.1021/bi060717k"><li><a href="http://dx.doi.org/10.1021/bi060717k" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+crystal+structure+of+the+BAR+domain+from+human+Bin1%2Famphiphysin+II+and+its+implications+for+molecular+recognition." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+crystal+structure+of+the+BAR+domain+from+human+Bin1%2Famphiphysin+II+and+its+implications+for+molecular+recognition.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0000788-Mansell1" id="pone.0000788-Mansell1"></a>Mansell A, Smith R, Doyle SL, Gray P, Fenner JE, et al.  (2006) Suppressor of cytokine signaling 1 negatively regulates Toll-like receptor signaling by mediating Mal degradation. Nat Immunol  7: 148–155.  <ul class="find" data-citedArticleID="982384" data-doi="10.1038/ni1299"><li><a href="http://dx.doi.org/10.1038/ni1299" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Suppressor+of+cytokine+signaling+1+negatively+regulates+Toll-like+receptor+signaling+by+mediating+Mal+degradation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Suppressor+of+cytokine+signaling+1+negatively+regulates+Toll-like+receptor+signaling+by+mediating+Mal+degradation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0000788-Miggin1" id="pone.0000788-Miggin1"></a>Miggin SM, Palsson-McDermott E, Dunne A, Jefferies C, Pinteaux E, et al.  (2007) NF-kappaB activation by the Toll-IL-1 receptor domain protein MyD88 adapter-like is regulated by caspase-1. Proc Natl Acad Sci U S A  104: 3372–3377.  <ul class="find" data-citedArticleID="982388" data-doi="10.1073/pnas.0608100104"><li><a href="http://dx.doi.org/10.1073/pnas.0608100104" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=NF-kappaB+activation+by+the+Toll-IL-1+receptor+domain+protein+MyD88+adapter-like+is+regulated+by+caspase-1." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22NF-kappaB+activation+by+the+Toll-IL-1+receptor+domain+protein+MyD88+adapter-like+is+regulated+by+caspase-1.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0000788-Khor1" id="pone.0000788-Khor1"></a>Khor CC, Chapman SJ, Vannberg FO, Dunne A, Murphy C, et al.  (2007) A Mal functional variant is associated with protection against invasive pneumococcal disease, bacteremia, malaria and tuberculosis. Nat Genet  39: 523–528.  <ul class="find" data-citedArticleID="982370" data-doi="10.1038/ng1976"><li><a href="http://dx.doi.org/10.1038/ng1976" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+Mal+functional+variant+is+associated+with+protection+against+invasive+pneumococcal+disease%2C+bacteremia%2C+malaria+and+tuberculosis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+Mal+functional+variant+is+associated+with+protection+against+invasive+pneumococcal+disease%2C+bacteremia%2C+malaria+and+tuberculosis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0000788-Shi1" id="pone.0000788-Shi1"></a>Shi J, Blundell TL, Mizuguchi K (2001) FUGUE: sequence-structure homology recognition using environment-specific substitution tables and structure-dependent gap penalties. J Mol Biol  310: 243–257.  <ul class="find" data-citedArticleID="982416" data-doi="10.1006/jmbi.2001.4762"><li><a href="http://dx.doi.org/10.1006/jmbi.2001.4762" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=FUGUE%3A+sequence-structure+homology+recognition+using+environment-specific+substitution+tables+and+structure-dependent+gap+penalties." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22FUGUE%3A+sequence-structure+homology+recognition+using+environment-specific+substitution+tables+and+structure-dependent+gap+penalties.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0000788-deBakker1" id="pone.0000788-deBakker1"></a>de Bakker PI, Bateman A, Burke DF, Miguel RN, Mizuguchi K, et al.  (2001) HOMSTRAD: adding sequence information to structure-based alignments of homologous protein families. Bioinformatics  17: 748–749.  <ul class="find" data-citedArticleID="982428" data-doi="10.1093/bioinformatics/17.8.748"><li><a href="http://dx.doi.org/10.1093/bioinformatics/17.8.748" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=HOMSTRAD%3A+adding+sequence+information+to+structure-based+alignments+of+homologous+protein+families." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22HOMSTRAD%3A+adding+sequence+information+to+structure-based+alignments+of+homologous+protein+families.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0000788-Sali1" id="pone.0000788-Sali1"></a>Sali A, Blundell TL (1993) Comparative protein modelling by satisfaction of spatial restraints. J Mol Biol  234: 779–815.  <ul class="find" data-citedArticleID="982412" data-doi="10.1006/jmbi.1993.1626"><li><a href="http://dx.doi.org/10.1006/jmbi.1993.1626" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Comparative+protein+modelling+by+satisfaction+of+spatial+restraints." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Comparative+protein+modelling+by+satisfaction+of+spatial+restraints.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0000788-Brooks1" id="pone.0000788-Brooks1"></a>Brooks BR, Bruccolesi RE, Olafson BD, Swaminathan S, Karplus M (1983) CHARMM: a program for macromolecular energy minimisation and dynamics calculations. Journal of Computational Chemistry  4: 187–217.  <ul class="find" data-citedArticleID="982342" data-doi="10.1002/jcc.540040211"><li><a href="http://dx.doi.org/10.1002/jcc.540040211" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=CHARMM%3A+a+program+for+macromolecular+energy+minimisation+and+dynamics+calculations." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22CHARMM%3A+a+program+for+macromolecular+energy+minimisation+and+dynamics+calculations.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0000788-Laskowski1" id="pone.0000788-Laskowski1"></a>Laskowski RA, MacArthur MW, Moss DS, Thornton JM (1993) A program to check the sterochemical quality of protein structures. Journal of Applied Crystallography  26: 283–291.  <ul class="find" data-citedArticleID="982372" data-doi="10.1107/s0021889892009944"><li><a href="http://dx.doi.org/10.1107/s0021889892009944" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+program+to+check+the+sterochemical+quality+of+protein+structures." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+program+to+check+the+sterochemical+quality+of+protein+structures.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">36.
              </span><a name="pone.0000788-Mizuguchi1" id="pone.0000788-Mizuguchi1"></a>Mizuguchi K, Deane CM, Blundell TL, Johnson MS, Overington JP (1998) JOY: protein sequence-structure representation and analysis. Bioinformatics  14: 617–623.  <ul class="find" data-citedArticleID="982394" data-doi="10.1093/bioinformatics/14.7.617"><li><a href="http://dx.doi.org/10.1093/bioinformatics/14.7.617" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=JOY%3A+protein+sequence-structure+representation+and+analysis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22JOY%3A+protein+sequence-structure+representation+and+analysis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">37.
              </span><a name="pone.0000788-Jones1" id="pone.0000788-Jones1"></a>Jones S, Thornton JM (1996) Principles of protein-protein interactions. Proc Natl Acad Sci U S A  93: 13–20.  <ul class="find" data-citedArticleID="982366"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Principles+of+protein-protein+interactions.&amp;auth=&amp;atitle=Principles+of+protein-protein+interactions." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Principles+of+protein-protein+interactions." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Principles+of+protein-protein+interactions.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">38.
              </span><a name="pone.0000788-Lee1" id="pone.0000788-Lee1"></a>Lee B, Richards FM (1971) The interpretation of protein structures: estimation of static accessibility. J Mol Biol  55: 379–400.  <ul class="find" data-citedArticleID="982376" data-doi="10.1016/0022-2836(71)90324-x"><li><a href="http://dx.doi.org/10.1016/0022-2836(71)90324-x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+interpretation+of+protein+structures%3A+estimation+of+static+accessibility." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+interpretation+of+protein+structures%3A+estimation+of+static+accessibility.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">39.
              </span><a name="pone.0000788-Richmond1" id="pone.0000788-Richmond1"></a>Richmond TJ (1984) Solvent accessible surface area and excluded volume in proteins. Analytical equations for overlapping spheres and implications for the hydrophobic effect. J Mol Biol  178: 63–89.  <ul class="find" data-citedArticleID="982404" data-doi="10.1016/0022-2836(84)90231-6"><li><a href="http://dx.doi.org/10.1016/0022-2836(84)90231-6" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Solvent+accessible+surface+area+and+excluded+volume+in+proteins.+Analytical+equations+for+overlapping+spheres+and+implications+for+the+hydrophobic+effect." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Solvent+accessible+surface+area+and+excluded+volume+in+proteins.+Analytical+equations+for+overlapping+spheres+and+implications+for+the+hydrophobic+effect.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">40.
              </span><a name="pone.0000788-McDonald1" id="pone.0000788-McDonald1"></a>McDonald IK, Thornton JM (1994) Satisfying hydrogen bonding potential in proteins. J Mol Biol  238: 777–793.  <ul class="find" data-citedArticleID="982386" data-doi="10.1006/jmbi.1994.1334"><li><a href="http://dx.doi.org/10.1006/jmbi.1994.1334" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Satisfying+hydrogen+bonding+potential+in+proteins." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Satisfying+hydrogen+bonding+potential+in+proteins.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">41.
              </span><a name="pone.0000788-KatchalskiKatzir1" id="pone.0000788-KatchalskiKatzir1"></a>Katchalski-Katzir E, Shariv I, Eisenstein M, Friesem AA, Aflalo C, et al.  (1992) Molecular surface recognition: determination of geometric fit between proteins and their ligands by correlation techniques. Proc Natl Acad Sci U S A  89: 2195–2199.  <ul class="find" data-citedArticleID="982368" data-doi="10.1073/pnas.89.6.2195"><li><a href="http://dx.doi.org/10.1073/pnas.89.6.2195" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Molecular+surface+recognition%3A+determination+of+geometric+fit+between+proteins+and+their+ligands+by+correlation+techniques." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Molecular+surface+recognition%3A+determination+of+geometric+fit+between+proteins+and+their+ligands+by+correlation+techniques.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">42.
              </span><a name="pone.0000788-Miguel1" id="pone.0000788-Miguel1"></a>Miguel RN (2004) Sequence patterns derived from the automated prediction of functional residues in structurally-aligned homologous protein families. Bioinformatics  20: 2380–2389.  <ul class="find" data-citedArticleID="982390" data-doi="10.1093/bioinformatics/bth255"><li><a href="http://dx.doi.org/10.1093/bioinformatics/bth255" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Sequence+patterns+derived+from+the+automated+prediction+of+functional+residues+in+structurally-aligned+homologous+protein+families." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Sequence+patterns+derived+from+the+automated+prediction+of+functional+residues+in+structurally-aligned+homologous+protein+families.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">43.
              </span><a name="pone.0000788-Fitzgerald3" id="pone.0000788-Fitzgerald3"></a>Fitzgerald KA, McWhirter SM, Faia KL, Rowe DC, Latz E, et al.  (2003) IKKepsilon and TBK1 are essential components of the IRF3 signaling pathway. Nat Immunol  4: 491–496.  <ul class="find" data-citedArticleID="982352" data-doi="10.1038/ni921"><li><a href="http://dx.doi.org/10.1038/ni921" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=IKKepsilon+and+TBK1+are+essential+components+of+the+IRF3+signaling+pathway." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22IKKepsilon+and+TBK1+are+essential+components+of+the+IRF3+signaling+pathway.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">44.
              </span><a name="pone.0000788-Lin1" id="pone.0000788-Lin1"></a>Lin R, Heylbroeck C, Genin P, Pitha PM, Hiscott J (1999) Essential role of interferon regulatory factor 3 in direct activation of RANTES chemokine transcription. Mol Cell Biol  19: 959–966.  <ul class="find" data-citedArticleID="982378"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Essential+role+of+interferon+regulatory+factor+3+in+direct+activation+of+RANTES+chemokine+transcription.&amp;auth=&amp;atitle=Essential+role+of+interferon+regulatory+factor+3+in+direct+activation+of+RANTES+chemokine+transcription." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Essential+role+of+interferon+regulatory+factor+3+in+direct+activation+of+RANTES+chemokine+transcription." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Essential+role+of+interferon+regulatory+factor+3+in+direct+activation+of+RANTES+chemokine+transcription.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">45.
              </span><a name="pone.0000788-Weber1" id="pone.0000788-Weber1"></a>Weber AN, Moncrieffe MC, Gangloff M, Imler JL, Gay NJ (2005) Ligand-receptor and receptor-receptor interactions act in concert to activate signaling in the Drosophila toll pathway. J Biol Chem  280: 22793–22799.  <ul class="find" data-citedArticleID="982424" data-doi="10.1074/jbc.m502074200"><li><a href="http://dx.doi.org/10.1074/jbc.m502074200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Ligand-receptor+and+receptor-receptor+interactions+act+in+concert+to+activate+signaling+in+the+Drosophila+toll+pathway." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Ligand-receptor+and+receptor-receptor+interactions+act+in+concert+to+activate+signaling+in+the+Drosophila+toll+pathway.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000788.XML" value="127249"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000788.PDF" value="562835"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000788.g001.PNG_L" value="733987"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000788.g001.PNG_M" value="80292"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000788.g001.PNG_S" value="9209"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000788.g001.TIF" value="1335956"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000788.g001.PNG_I" value="28856"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000788.g002.PNG_L" value="129757"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000788.g002.PNG_M" value="105725"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000788.g002.PNG_S" value="10943"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000788.g002.TIF" value="217176"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000788.g003.PNG_L" value="2933496"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000788.g003.PNG_M" value="323808"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000788.g003.PNG_S" value="18359"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000788.g003.TIF" value="4115088"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000788.g003.PNG_I" value="108297"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000788.t001.PNG_L" value="115278"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000788.t001.PNG_M" value="68646"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000788.t001.PNG_S" value="11913"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000788.t001.TIF" value="697084"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000788.t001.PNG_I" value="20699"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000788.g004.PNG_L" value="4089055"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000788.g004.PNG_M" value="293145"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000788.g004.PNG_S" value="12204"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000788.g004.TIF" value="5673550"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000788.g004.PNG_I" value="91777"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000788.t002.PNG_L" value="453731"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000788.t002.PNG_M" value="147363"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000788.t002.PNG_S" value="14731"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000788.t002.TIF" value="2173364"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000788.t002.PNG_I" value="74146"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000788.t003.PNG_L" value="72739"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000788.t003.PNG_M" value="92032"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000788.t003.PNG_S" value="13932"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000788.t003.TIF" value="361044"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000788.t003.PNG_I" value="31264"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000788.t004.PNG_L" value="116584"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000788.t004.PNG_M" value="64539"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000788.t004.PNG_S" value="11268"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000788.t004.TIF" value="665972"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000788.t004.PNG_I" value="19456"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000788.t005.PNG_L" value="105129"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000788.t005.PNG_M" value="59571"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000788.t005.PNG_S" value="10679"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000788.t005.TIF" value="569118"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000788.t005.PNG_I" value="17903"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000788.g005.PNG_L" value="3912067"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000788.g005.PNG_M" value="271665"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000788.g005.PNG_S" value="19008"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000788.g005.TIF" value="5509592"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000788.g005.PNG_I" value="128822"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000788.s001.DOC" value="64000"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000788&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000788" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000788&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0000788&volume=&issue=&title=A Dimer of the Toll-Like Receptor 4 Cytoplasmic Domain Provides a Specific Scaffold for the Recruitment of Signalling Adaptor Proteins&author_name=Ricardo%20N%C3%BA%C3%B1ez%20Miguel%2C%20Joyce%20Wong%2C%20Julian%20F.%20Westoll%2C%20Heather%20J.%20Brooks%2C%20Luke%20A.%20J.%20O'Neill%2C%20Nicholas%20J.%20Gay%2C%20Clare%20E.%20Bryant%2C%20Tom%20P.%20Monie&start_page=1&end_page=12" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000788" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000788" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0000788" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000788&amp;t=A%20Dimer%20of%20the%20Toll-Like%20Receptor%204%20Cytoplasmic%20Domain%20Provides%20a%20Specific%20Scaffold%20for%20the%20Recruitment%20of%20Signalling%20Adaptor%20Proteins" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000788&title=A%20Dimer%20of%20the%20Toll-Like%20Receptor%204%20Cytoplasmic%20Domain%20Provides%20a%20Specific%20Scaffold%20for%20the%20Recruitment%20of%20Signalling%20Adaptor%20Proteins&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000788&amp;title=A%20Dimer%20of%20the%20Toll-Like%20Receptor%204%20Cytoplasmic%20Domain%20Provides%20a%20Specific%20Scaffold%20for%20the%20Recruitment%20of%20Signalling%20Adaptor%20Proteins" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000788" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0000788&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'A Dimer of the Toll-Like Receptor 4 Cytoplasmic Domain Provides a Specific Scaffold for the Recruitment of Signalling Adaptor Proteins';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0000788';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20A%20Dimer%20of%20the%20Toll-Like%20Receptor%204%20Cytoplasmic%20Domain%20Provides%20a%20Specific%20Scaffold%20for%20the%20Recruitment%20of%20Signalling%20Adaptor%20Proteins http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000788" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0000788" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">















          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Crystal+structure%22" title="Search for articles in the subject area:'Crystal structure'"><div class="flagText">Crystal structure</div></a>
              <div data-categoryid="17301" data-articleid="24414"
                   data-categoryname="Crystal structure"
                   class="flagImage" title="Flag 'Crystal structure' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Electrostatics%22" title="Search for articles in the subject area:'Electrostatics'"><div class="flagText">Electrostatics</div></a>
              <div data-categoryid="17995" data-articleid="24414"
                   data-categoryname="Electrostatics"
                   class="flagImage" title="Flag 'Electrostatics' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Hydrogen+bonding%22" title="Search for articles in the subject area:'Hydrogen bonding'"><div class="flagText">Hydrogen bonding</div></a>
              <div data-categoryid="23891" data-articleid="24414"
                   data-categoryname="Hydrogen bonding"
                   class="flagImage" title="Flag 'Hydrogen bonding' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Immune+receptor+signaling%22" title="Search for articles in the subject area:'Immune receptor signaling'"><div class="flagText">Immune receptor signaling</div></a>
              <div data-categoryid="19743" data-articleid="24414"
                   data-categoryname="Immune receptor signaling"
                   class="flagImage" title="Flag 'Immune receptor signaling' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Receptor+physiology%22" title="Search for articles in the subject area:'Receptor physiology'"><div class="flagText">Receptor physiology</div></a>
              <div data-categoryid="23845" data-articleid="24414"
                   data-categoryname="Receptor physiology"
                   class="flagImage" title="Flag 'Receptor physiology' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Sequence+alignment%22" title="Search for articles in the subject area:'Sequence alignment'"><div class="flagText">Sequence alignment</div></a>
              <div data-categoryid="18051" data-articleid="24414"
                   data-categoryname="Sequence alignment"
                   class="flagImage" title="Flag 'Sequence alignment' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Transmembrane+receptors%22" title="Search for articles in the subject area:'Transmembrane receptors'"><div class="flagText">Transmembrane receptors</div></a>
              <div data-categoryid="20299" data-articleid="24414"
                   data-categoryname="Transmembrane receptors"
                   class="flagImage" title="Flag 'Transmembrane receptors' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Tyrosine%22" title="Search for articles in the subject area:'Tyrosine'"><div class="flagText">Tyrosine</div></a>
              <div data-categoryid="40505" data-articleid="24414"
                   data-categoryname="Tyrosine"
                   class="flagImage" title="Flag 'Tyrosine' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=7906'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=8349'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=7583&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>


</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
