

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Acoel Flatworms Are Not Platyhelminthes: Evidence from Phylogenomics</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0000717"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0000717" />

    <meta name="citation_title" content="Acoel Flatworms Are Not Platyhelminthes: Evidence from Phylogenomics"/>
    <meta itemprop="name" content="Acoel Flatworms Are Not Platyhelminthes: Evidence from Phylogenomics"/>

      <meta name="citation_author" content="Hervé Philippe"/>
            <meta name="citation_author_institution" content="Canadian Institute for Advanced Research, Centre Robert-Cedergren, Département de Biochimie, Université de Montréal, Montréal, Québec, Canada"/>
      <meta name="citation_author" content="Henner Brinkmann"/>
            <meta name="citation_author_institution" content="Canadian Institute for Advanced Research, Centre Robert-Cedergren, Département de Biochimie, Université de Montréal, Montréal, Québec, Canada"/>
      <meta name="citation_author" content="Pedro Martinez"/>
            <meta name="citation_author_institution" content="Departament de Genètica, Facultat de Biologia, Universitat de Barcelona, Barcelona, Spain"/>
      <meta name="citation_author" content="Marta Riutort"/>
            <meta name="citation_author_institution" content="Departament de Genètica, Facultat de Biologia, Universitat de Barcelona, Barcelona, Spain"/>
      <meta name="citation_author" content="Jaume Baguñà"/>
            <meta name="citation_author_institution" content="Departament de Genètica, Facultat de Biologia, Universitat de Barcelona, Barcelona, Spain"/>

    <meta name="citation_date" content="2007/8/8"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0000717.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e717"/>
    <meta name="citation_issue" content="8"/>
    <meta name="citation_volume" content="2"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=The Invertebrates. Vol. II. Platyhelminthes and Rhynchocoela.; citation_author=L Hyman; citation_number=1; citation_date=1951; citation_publisher=McGraw-Hill; " />
      <meta name="citation_reference" content="citation_title=Animal phylogeny and the ancestry of bilaterians: inferences from morphology and 18S rDNA gene sequences.; citation_author=KJ Peterson; citation_author=DJ Eernisse; citation_journal_title=Evolution&Development; citation_volume=3; citation_number=2; citation_pages=170-205; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Is the Turbellaria polyphyletic?; citation_author=JPS Smith; citation_author=S Tyler; citation_author=RM Rieger; citation_journal_title=Hydrobiologia; citation_volume=132; citation_number=3; citation_pages=71-78; citation_date=1986; " />
      <meta name="citation_reference" content="citation_title=The dawn of bilaterian animals: the case of acoelomorph flatworms.; citation_author=J Baguñà; citation_author=M Riutort; citation_journal_title=Bioessays; citation_volume=26; citation_number=4; citation_pages=1046-1057; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Acoel flatworms: earliest extant bilaterian Metazoans, not members of Platyhelminthes.; citation_author=I Ruiz-Trillo; citation_author=M Riutort; citation_author=DT Littlewood; citation_author=EA Herniou; citation_author=J Baguñà; citation_journal_title=Science; citation_volume=283; citation_number=5; citation_pages=1919-1923; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=The Nemertodermatida are basal bilaterians not members of Platyhelminthes.; citation_author=U Jondelius; citation_author=I Ruiz-Trillo; citation_author=J Baguñà; citation_author=M Riutort; citation_journal_title=Zool Scr; citation_volume=31; citation_number=6; citation_pages=201-215; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Combined large and small subunit ribosomal RNA phylogenies support a basal position of the acoelomorph flatworms.; citation_author=MJ Telford; citation_author=AE Lockyer; citation_author=C Cartwright-Finch; citation_author=DT Littlewood; citation_journal_title=Proc R Soc Lond B Biol Sci; citation_volume=270; citation_number=7; citation_pages=1077-1083; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=A phylogenetic analysis of myosin heavy chain type II sequences corroborates that Acoela and Nemertodermatida are basal bilaterians.; citation_author=I Ruiz-Trillo; citation_author=J Paps; citation_author=M Loukota; citation_author=C Ribera; citation_author=U Jondelius; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=99; citation_number=8; citation_pages=11246-11251; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Mitochondrial genome data support the basal position of Acoelomorpha and the polyphyly of the Platyhelminthes.; citation_author=I Ruiz-Trillo; citation_author=M Riutort; citation_author=HM Fourcade; citation_author=J Baguñà; citation_author=JL Boore; citation_journal_title=Mol Phylogenet Evol; citation_volume=33; citation_number=9; citation_pages=321-332; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Cases in which parsimony or compatibility methods will be positively misleading.; citation_author=J Felsenstein; citation_journal_title=Syst Zool; citation_volume=27; citation_number=10; citation_pages=401-410; citation_date=1978; " />
      <meta name="citation_reference" content="citation_title=The new animal phylogeny: reliability and implications.; citation_author=A Adoutte; citation_author=G Balavoine; citation_author=N Lartillot; citation_author=O Lespinet; citation_author=B Prud'homme; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=97; citation_number=11; citation_pages=4453-4456; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=The new view of animal phylogeny.; citation_author=KM Halanych; citation_journal_title=Annual Review of Ecology Evolution and Systematics; citation_volume=35; citation_number=12; citation_pages=229-256; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Phylogenomics and the reconstruction of the tree of life.; citation_author=F Delsuc; citation_author=H Brinkmann; citation_author=H Philippe; citation_journal_title=Nat Rev Genet; citation_volume=6; citation_number=13; citation_pages=361-375; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Lack of resolution in the animal phylogeny: closely spaced cladogeneses or undetected systematic errors?; citation_author=D Baurain; citation_author=H Brinkmann; citation_author=H Philippe; citation_journal_title=Mol Biol Evol; citation_volume=24; citation_number=14; citation_pages=6-9; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Large-scale sequencing and the new animal phylogeny.; citation_author=H Philippe; citation_author=MJ Telford; citation_journal_title=Trends Ecol Evol; citation_volume=21; citation_number=15; citation_pages=614-620; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Tunicates and not cephalochordates are the closest living relatives of vertebrates.; citation_author=F Delsuc; citation_author=H Brinkmann; citation_author=D Chourrout; citation_author=H Philippe; citation_journal_title=Nature; citation_volume=439; citation_number=16; citation_pages=965-968; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Heterotachy and long-branch attraction in phylogenetics.; citation_author=H Philippe; citation_author=Y Zhou; citation_author=H Brinkmann; citation_author=N Rodrigue; citation_author=F Delsuc; citation_journal_title=BMC Evol Biol; citation_volume=5; citation_number=17; citation_pages=50; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Evolutionary trees from DNA sequences: a maximum likelihood approach.; citation_author=J Felsenstein; citation_journal_title=J Mol Evol; citation_volume=17; citation_number=18; citation_pages=368-376; citation_date=1981; " />
      <meta name="citation_reference" content="citation_title=A Bayesian mixture model for across-site heterogeneities in the amino-acid replacement process.; citation_author=N Lartillot; citation_author=H Philippe; citation_journal_title=Mol Biol Evol; citation_volume=21; citation_number=19; citation_pages=1095-1109; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Suppression of long-branch attraction artefacts in the animal phylogeny using a site-heterogeneous model.; citation_author=N Lartillot; citation_author=H Brinkmann; citation_author=H Philippe; citation_journal_title=BMC Evol Biol; citation_volume=7; citation_number=20; citation_pages=S4; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Deuterostome phylogeny reveals monophyletic chordates and the new phylum Xenoturbellida.; citation_author=SJ Bourlat; citation_author=T Juliusdottir; citation_author=CJ Lowe; citation_author=R Freeman; citation_author=J Aronowicz; citation_journal_title=Nature; citation_volume=444; citation_number=21; citation_pages=85-88; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Broad taxon and gene sampling indicate that chaetognaths are protostomes.; citation_author=DQ Matus; citation_author=RR Copley; citation_author=CW Dunn; citation_author=A Hejnol; citation_author=H Eccleston; citation_journal_title=Curr Biol; citation_volume=16; citation_number=22; citation_pages=R575-576; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Multigene analyses of bilaterian animals corroborate the monophyly of Ecdysozoa, Lophotrochozoa, and Protostomia.; citation_author=H Philippe; citation_author=N Lartillot; citation_author=H Brinkmann; citation_journal_title=Mol Biol Evol; citation_volume=22; citation_number=23; citation_pages=1246-1253; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Chaetognath phylogenomics: a protostome with deuterostome-like development.; citation_author=F Marletaz; citation_author=E Martin; citation_author=Y Perez; citation_author=D Papillon; citation_author=X Caubit; citation_journal_title=Curr Biol; citation_volume=16; citation_number=24; citation_pages=R577-578; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=An empirical assessment of long-branch attraction artefacts in deep eukaryotic phylogenomics.; citation_author=H Brinkmann; citation_author=M Giezen; citation_author=Y Zhou; citation_author=GP Raucourt; citation_author=H Philippe; citation_journal_title=Syst Biol; citation_volume=54; citation_number=25; citation_pages=743-757; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Multicellular Animals.; citation_author=P Ax; citation_number=26; citation_pages=225; citation_date=1996; citation_publisher=Springer; " />
      <meta name="citation_reference" content="citation_title=Das Phylogenetische System der Plathelminthes.; citation_author=U Ehlers; citation_number=27; citation_pages=317; citation_date=1985; citation_publisher=G. Fischer; " />
      <meta name="citation_reference" content="citation_title=The epidermal ciliary rootlets of Xenoturbella bocki (Xenoturbellida) revisited: new support for a possible kinship with the Acoelomorpha (Platyhelminthes).; citation_author=K Lundin; citation_journal_title=Zoologica Scripta; citation_volume=27; citation_number=28; citation_pages=263-270; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Phylogenomics of eukaryotes: impact of missing data on large alignments.; citation_author=H Philippe; citation_author=EA Snell; citation_author=E Bapteste; citation_author=P Lopez; citation_author=PW Holland; citation_journal_title=Mol Biol Evol; citation_volume=21; citation_number=29; citation_pages=1740-1752; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=MUST, a computer package of Management Utilities for Sequences and Trees.; citation_author=H Philippe; citation_journal_title=Nucleic Acids Res; citation_volume=21; citation_number=30; citation_pages=5264-5272; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=Selection of Conserved Blocks from Multiple Alignments for Their Use in Phylogenetic Analysis.; citation_author=J Castresana; citation_journal_title=Mol Biol Evol; citation_volume=17; citation_number=31; citation_pages=540-552; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=SCaFoS: a tool for Selection, Concatenation and Fusion of Sequences for phylogenomics.; citation_author=B Roure; citation_author=N Rodriguez-Ezpeleta; citation_author=H Philippe; citation_journal_title=BMC Evol Biol; citation_volume=7; citation_number=32; citation_pages=S2; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=TREE-PUZZLE: maximum likelihood phylogenetic analysis using quartets and parallel computing.; citation_author=HA Schmidt; citation_author=K Strimmer; citation_author=M Vingron; citation_author=A von Haeseler; citation_journal_title=Bioinformatics; citation_volume=18; citation_number=33; citation_pages=502-504; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=PHYLIP (Phylogene Inference Package).; citation_author=J Felsenstein; citation_number=34; citation_date=2001; citation_publisher=Department of Genetics, University of Washington; " />
      <meta name="citation_reference" content="citation_title=PAUP*: Phylogenetic Analysis Using Parsimony and other methods.; citation_author=DL Swofford; citation_number=35; citation_date=2000; citation_publisher=Sinauer; " />
      <meta name="citation_reference" content="citation_title=CONSEL: for assessing the confidence of phylogenetic tree selection.; citation_author=H Shimodaira; citation_author=M Hasegawa; citation_journal_title=Bioinformatics; citation_volume=17; citation_number=36; citation_pages=1246-1247; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=PAML: a program package for phylogenetic analysis by maximum likelihood.; citation_author=Z Yang; citation_journal_title=Comput Appl Biosci; citation_volume=13; citation_number=37; citation_pages=555-556; citation_date=1997; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000717" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Acoel Flatworms Are Not Platyhelminthes: Evidence from Phylogenomics"/>
    <meta name="twitter:description" content="Acoel flatworms are small marine worms traditionally considered to belong to the phylum Platyhelminthes. However, molecular phylogenetic analyses suggest that acoels are not members of Platyhelminthes, but are rather extant members of the earliest diverging Bilateria. This result has been called into question, under suspicions of a long branch attraction (LBA) artefact. Here we re-examine this problem through a phylogenomic approach using 68 different protein-coding genes from the acoel Convoluta pulchra and 51 metazoan species belonging to 15 different phyla. We employ a mixture model, named CAT, previously found to overcome LBA artefacts where classical models fail. Our results unequivocally show that acoels are not part of the classically defined Platyhelminthes, making the latter polyphyletic. Moreover, they indicate a deuterostome affinity for acoels, potentially as a sister group to all deuterostomes, to Xenoturbellida, to Ambulacraria, or even to chordates. However, the weak support found for most deuterostome nodes, together with the very fast evolutionary rate of the acoel Convoluta pulchra, call for more data from slowly evolving acoels (or from its sister-group, the Nemertodermatida) to solve this challenging phylogenetic problem."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0000717.g001"/>

  <meta property="og:title" content="Acoel Flatworms Are Not Platyhelminthes: Evidence from Phylogenomics" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000717" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=5453'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=7842'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=6415&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0000717">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000717" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000717&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Acoel Flatworms Are Not Platyhelminthes: Evidence from Phylogenomics
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Hervé Philippe
              <span class="corresponding">mail</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:herve.philippe@umontreal.ca">herve.philippe@umontreal.ca</a> (HP); <a href="mailto:jbaguna@ub.edu">jbaguna@ub.edu</a> (JB)</p>

                <p>Affiliation:
                  Canadian Institute for Advanced Research, Centre Robert-Cedergren, Département de Biochimie, Université de Montréal, Montréal, Québec, Canada
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Henner Brinkmann, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Canadian Institute for Advanced Research, Centre Robert-Cedergren, Département de Biochimie, Université de Montréal, Montréal, Québec, Canada
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Pedro Martinez, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Departament de Genètica, Facultat de Biologia, Universitat de Barcelona, Barcelona, Spain
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Marta Riutort, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Departament de Genètica, Facultat de Biologia, Universitat de Barcelona, Barcelona, Spain
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Jaume Baguñà
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:herve.philippe@umontreal.ca">herve.philippe@umontreal.ca</a> (HP); <a href="mailto:jbaguna@ub.edu">jbaguna@ub.edu</a> (JB)</p>

                <p>Affiliation:
                  Departament de Genètica, Facultat de Biologia, Universitat de Barcelona, Barcelona, Spain
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: August 08, 2007</li>
    <li>DOI: 10.1371/journal.pone.0000717</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0000717" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0000717" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000717" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0000717" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0000717-g001" data-doi="info:doi/10.1371/journal.pone.0000717" data-uri="info:doi/10.1371/journal.pone.0000717.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000717.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000717">Reader Comments (4)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0000717" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1"></a><p>Acoel flatworms are small marine worms traditionally considered to belong to the phylum Platyhelminthes. However, molecular phylogenetic analyses suggest that acoels are not members of Platyhelminthes, but are rather extant members of the earliest diverging Bilateria. This result has been called into question, under suspicions of a long branch attraction (LBA) artefact. Here we re-examine this problem through a phylogenomic approach using 68 different protein-coding genes from the acoel <em>Convoluta pulchra</em> and 51 metazoan species belonging to 15 different phyla. We employ a mixture model, named CAT, previously found to overcome LBA artefacts where classical models fail. Our results unequivocally show that acoels are not part of the classically defined Platyhelminthes, making the latter polyphyletic. Moreover, they indicate a deuterostome affinity for acoels, potentially as a sister group to all deuterostomes, to Xenoturbellida, to Ambulacraria, or even to chordates. However, the weak support found for most deuterostome nodes, together with the very fast evolutionary rate of the acoel <em>Convoluta pulchra</em>, call for more data from slowly evolving acoels (or from its sister-group, the Nemertodermatida) to solve this challenging phylogenetic problem.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>Philippe H, Brinkmann H, Martinez P, Riutort M, Baguñà J (2007) Acoel Flatworms Are Not Platyhelminthes: Evidence from Phylogenomics. PLoS ONE 2(8):
          e717.
            doi:10.1371/journal.pone.0000717</p><p><strong>Academic Editor: </strong>Jean-Nicolas Volff, Ecole Normale Supérieure de Lyon, France</p><p><strong>Received:</strong> May 4, 2007; <strong>Accepted:</strong> July 4, 2007; <strong>Published:</strong> August 8, 2007</p><p><strong>Copyright:</strong> © 2007 Philippe et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>HP acknowledges Génome Québec, the Canadian Research Chair and the Université de Montréal for financial support, PM, MR and JB are supported by grants from ICREA, Generalitat de Catalunya and the Ministerio de Educación y Ciencia.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>Acoelomorph flatworms (Acoela+Nemertodermatida) constitute a small group of bilaterian marine worms that recently came to the limelight. Several morphological similarities suggest that acoelomorphs belong to the Platyhelminthes <a href="#pone.0000717-Hyman1">[1]</a>–<a href="#pone.0000717-Smith1">[3]</a>; however, these characters are often ill-defined or mere symplesiomorphies <a href="#pone.0000717-Bagu1">[4]</a>, leaving the status of the phylum Platyhelminthes unsettled. Molecular phylogenies based on SSU rRNA <a href="#pone.0000717-RuizTrillo1">[5]</a>, <a href="#pone.0000717-Jondelius1">[6]</a>, combined SSU and LSU rRNA <a href="#pone.0000717-Telford1">[7]</a>, myosin II <a href="#pone.0000717-RuizTrillo2">[8]</a> and mitochondrial genomes <a href="#pone.0000717-RuizTrillo3">[9]</a>, provide strong statistical support for excluding acoelomorphs from Platyhelminthes. Rather, these markers locate them as a sister-group to all remaining Bilateria.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>However, because acoelomorphs evolve at a very high rate (except perhaps for myosin II), their basal emergence can be easily explained by a long branch attraction (LBA) <a href="#pone.0000717-Felsenstein1">[10]</a> artefact triggered by the distantly related outgroup (Cnidaria). Although careful approaches (e.g. selection of the slowest evolving rRNA sequence among 18 acoels <a href="#pone.0000717-RuizTrillo1">[5]</a>) have been used in an attempt to avoid LBA artefacts, the position of acoelomorphs remains unsettled. In particular, acoelomorphs are often considered as secondarily simplified organisms <a href="#pone.0000717-Adoutte1">[11]</a>, <a href="#pone.0000717-Halanych1">[12]</a>, potentially explaining their fast evolutionary rate. However, finding acoelomorphs at or close to the base of Bilateria, instead of belonging to Platyhelminthes, would allow to polarize several key bilaterian characters including the brain, the coelom, the nephridium, and the possession of a primary larval stage.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>We therefore decided to reanalyse this important question by applying the powerful simultaneous analysis of multiple orthologous nuclear protein-coding genes <a href="#pone.0000717-Delsuc1">[13]</a> and using up-to-date tree reconstruction methods in order to enhance the phylogenetic signal <a href="#pone.0000717-Baurain1">[14]</a>.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results and Discussion"></a><h3>Results and Discussion</h3><a id="article1.body1.sec2.p1" name="article1.body1.sec2.p1"></a><p>We sequenced 2304 ESTs from the acoel <em>Convoluta pulchra</em>, a number that has been suggested to provide a sufficient amount of homologous positions <a href="#pone.0000717-Philippe1">[15]</a>. A total of 68 different protein-coding genes was unequivocally assigned to a dataset of conserved single copy genes previously cloned from other multicellular animals and related taxa <a href="#pone.0000717-Baurain1">[14]</a>, <a href="#pone.0000717-Delsuc2">[16]</a>. A rich taxon sampling of 51 species containing the main bilaterian lineages (Deuterostomia, Ecdysozoa and Lophotrochozoa) and several close and slow-evolving outgroup species (four Cnidaria, three Porifera, three Choanoflagellata and three Ichthyosporea) was selected to reduce as much as possible potential LBA artefacts. After the removal of ambiguously aligned positions and portions that have not been sequenced in <em>Convoluta</em>, an alignment of 11,959 positions was obtained.</p>
<a id="article1.body1.sec2.p2" name="article1.body1.sec2.p2"></a><p>The sequences from <em>Convoluta</em> are extremely divergent, which make its placement likely to be subject to the LBA artefact. Indeed, when parsimony, a method highly sensitive to this artefact <a href="#pone.0000717-Felsenstein1">[10]</a>, is used, <em>Convoluta</em> is invariantly attracted by the longest unbroken branch: the tunicate <em>Oikopleura</em> with a Bootstrap Support (BS) of 96% (<a href="#pone.0000717.s004">Figure S1</a>), or the Platyhelminthes (BS = 72%) when <em>Oikopleura</em> is excluded (<a href="#pone.0000717.s005">Figure S2</a>). We therefore turned to probabilistic methods that are less sensitive to LBA <a href="#pone.0000717-Philippe2">[17]</a>, <a href="#pone.0000717-Felsenstein2">[18]</a>. Moreover, we used the CAT model <a href="#pone.0000717-Lartillot1">[19]</a> because it has been shown to overcome the LBA artefact when other models fail <a href="#pone.0000717-Baurain1">[14]</a>, <a href="#pone.0000717-Lartillot2">[20]</a>. The resulting tree (<a href="#pone-0000717-g001">Figure 1</a>) is in excellent agreement with current knowledge <a href="#pone.0000717-Halanych1">[12]</a> and with recent large scale analyses <a href="#pone.0000717-Baurain1">[14]</a>, <a href="#pone.0000717-Delsuc2">[16]</a>, <a href="#pone.0000717-Bourlat1">[21]</a>–<a href="#pone.0000717-Marletaz1">[24]</a>: the monophyly of all animal phyla and of most super-phyla (Bilateria, Protostomia, Ecdysozoa, and Lophotrochozoa) is recovered with strong support (BS&gt;99%). Importantly, <em>Convoluta</em> does neither cluster with Platyhelminthes nor with <em>Oikopleura</em>, albeit these two groups display the longest branches. Although the precise position of <em>Convoluta</em> is not robustly resolved, two highly supported nodes (Protostomia and Lophotrochozoa) separate acoels from Platyhelminthes. The rejection of the traditional morphological hypothesis is therefore very strong.</p>
<div class="figure" id="pone-0000717-g001"><div class="img"><a name="pone-0000717-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000717.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000717" data-uri="info:doi/10.1371/journal.pone.0000717.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000717.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000717.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000717.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000717.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000717.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000717.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000717.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000717.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000717.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Phylogenetic analysis of genomic data strongly rejects the grouping of <em>Convoluta</em> and Platyhelminthes.</span></strong></p><a id="article1.body1.sec2.fig1.caption1.p1" name="article1.body1.sec2.fig1.caption1.p1"></a><p>Bayesian tree obtained from the analysis of 11,959 aligned amino-acid positions with the CAT model. Bootstrap values obtained are indicated when &lt;100%, otherwise a bullet is present on the node. The scale bar indicates the number of changes per site.</p>
<span>doi:10.1371/journal.pone.0000717.g001</span></div><a id="article1.body1.sec2.p3" name="article1.body1.sec2.p3"></a><p>However, the use of the standard WAG model provides strong support for the grouping of the two long branches platyhelminths and acoels (<a href="#pone.0000717.s006">Figure S3</a>). To further confirm that this grouping is an artefact of both parsimony and the WAG model, we looked for genes that are present in <em>Convoluta</em> and in deuterostomes, but absent from protostomes (see supplementary for details). Despite having analysed only 1,664 contigs from <em>Convoluta</em> (mainly encoding universal proteins such as ribosomal proteins), we found one gene, the guanidinoacetate N-methyltransferase, that is present in sponges, cnidarians, deuterostomes and <em>Convoluta</em>, but absent from all protostomes, except the basal chaetognaths <a href="#pone.0000717-Marletaz1">[24]</a>. The tree of the WAG model (<a href="#pone.0000717.s006">Figure S3</a>), which locates acoels as a sister-group to platyhelminths, implies at least three independent losses, whereas the tree of <a href="#pone-0000717-g001">Figure 1</a> implies a single loss in the protostomes, after the emergence of chaetognaths. The congruence of the tree inferred by the CAT model and of the distribution of guanidinoacetate N-methyltransferase strongly argues that the tree inferred using the WAG model is biased by an LBA artefact, pointing towards a reduced sensitivity of the CAT model to this artefact <a href="#pone.0000717-Baurain1">[14]</a>, <a href="#pone.0000717-Lartillot2">[20]</a>.</p>
<a id="article1.body1.sec2.p4" name="article1.body1.sec2.p4"></a><p>Although our phylogenomic tree of Bilateria is generally well resolved, three nodes within protostomes, the positions of chaetognaths, tardigrades and platyhelminths, are poorly supported. More strikingly, most of the relationships among deuterostomes are unsupported. Except for the monophyly of Olfactores <a href="#pone.0000717-Delsuc2">[16]</a> and of Ambulacraria <a href="#pone.0000717-Bourlat1">[21]</a>, the remaining nodes, including deuterostomes and chordates, receive BS below 55%. The very fast evolving <em>Convoluta</em> emerges in this part of the tree, as a sister-group to deuterostomes (<a href="#pone-0000717-g001">Figure 1</a>, BS = 14%), to chordates (BS = 33%), or to <em>Xenoturbella</em> (BS = 20%). <em>Convoluta</em> shows affinity with deuterostomes in 90% of the bootstrap replicates, being basal to Bilateria in only 7% of the replicates and to Protostomia in only 3%. It should be mentioned that the lack of support within deuterostomes is not due to the presence of the long branch of <em>Convoluta</em>, since the BS remains low (between 42% and 73%) when <em>Convoluta</em> is removed (<a href="#pone.0000717.s007">Figure S4</a>). In consequence, obtaining a reliable placement of <em>Convoluta</em> is hampered by two difficulties: (i) its very fast evolutionary rate, and (ii) a location in, or close to, deuterostomes, a region for which the powerful phylogenomic approach only yields poorly supported results <a href="#pone.0000717-Delsuc2">[16]</a>, <a href="#pone.0000717-Bourlat1">[21]</a>, <a href="#pone.0000717-Philippe3">[23]</a>.</p>
<a id="article1.body1.sec2.p5" name="article1.body1.sec2.p5"></a><p>To enhance the phylogenetic signal <a href="#pone.0000717-Brinkmann1">[25]</a>, we removed the outgroup (i.e. non-bilaterian species). Surprisingly, a single topological change occurs (<a href="#pone.0000717.s008">Figure S5</a>): <em>Xenoturbella</em> becomes the sister-group to <em>Convoluta</em>, albeit with low support (41%). Furthermore, the weak support for the recently proposed <em>Xenoturbella</em>+<em>Ambulacraria</em> <a href="#pone.0000717-Bourlat1">[21]</a> is not due to the presence of <em>Convoluta</em> (<a href="#pone.0000717.s009">Figure S6</a>). In summary, our large alignment of 11,959 positions strongly rejects the grouping of acoels with platyhelminths, and more generally protostomes. It favours deuterostome affinity for acoels, potentially as a sister-group to xenoturbellids. However, more data, especially from slowly evolving acoels, are needed to solve this challenging phylogenetic problem.</p>
<a id="article1.body1.sec2.p6" name="article1.body1.sec2.p6"></a><p>The rejection of the grouping of acoels with platyhelminths <em>sensu stricto</em> (see <a href="#pone.0000717-Bagu1">[4]</a> for a thorough taxonomical discussion) have several morphological implications. First, the lack of protonephridia in acoels, traditionally regarded as derived by the loss from a platyhelminth ancestor bearing them, may now be considered the retention of a primitive condition, a state shared with diploblasts. A similar argument could be applied to the sack-like gut of acoels (and of its sister group, the Nemertodermatida; <a href="#pone.0000717-Ax1">[26]</a>, <a href="#pone.0000717-Ehlers1">[27]</a>) which may now be a symplesiomorphy shared with the similar state in diploblasts and independent from the similar sack-like condition of the Platyhelminthes <em>sensu stricto</em> within the Lophotrochozoa. In addition, the peculiar duet-spiral cleavage of acoels, also traditionally considered to be derived from the quartet spiral cleavage of other platyhelminths, could now be considered as having originated either from a form of radial or biradial cleavage characteristic of more primitive metazoans. Interestingly, the grouping of Acoelomorpha and Xenoturbellida (<a href="#pone.0000717.s008">Figure S5</a>) has been proposed on the basis of the ultrastructures of epidermal locomotory ciliary rootlets <a href="#pone.0000717-Lundin1">[28]</a> and deserves future studies.</p>
<a id="article1.body1.sec2.p7" name="article1.body1.sec2.p7"></a><p>Our results contradict not only the morphological view but also previous molecular phylogenies that strongly support an emergence of acoels at the base of Bilateria <a href="#pone.0000717-RuizTrillo1">[5]</a>–<a href="#pone.0000717-RuizTrillo3">[9]</a>, since this position receives here a bootstrap support of only 7%. But, such a basal emergence is expected to be reinforced by a LBA artefact between the long branch of <em>Convoluta</em> and the one of the distantly related outgroup. The use of the CAT model, which is less sensitive to LBA <a href="#pone.0000717-Baurain1">[14]</a>, <a href="#pone.0000717-Lartillot2">[20]</a>, partly explains the observed lack of support for a basal emergence, since the latter support increases when the WAG model is used (<a href="#pone.0000717.s010">Figure S7</a>). However, the main reason for the discrepancy with previous molecular studies <a href="#pone.0000717-RuizTrillo1">[5]</a>–<a href="#pone.0000717-RuizTrillo3">[9]</a> is likely that the outgroup is less distantly related in our dataset, thereby reducing the LBA artefact: the distance from cnidarians to the last common ancestor of Bilateria represents 55.5% of the cnidarians/vertebrates distance for our nuclear proteins (<a href="#pone-0000717-g001">Figure 1</a>), but 63.9% for mitochondrial proteins <a href="#pone.0000717-RuizTrillo3">[9]</a> and 73.7% for 18S rRNA <a href="#pone.0000717-RuizTrillo1">[5]</a>.</p>
<a id="article1.body1.sec2.p8" name="article1.body1.sec2.p8"></a><p>In more conceptual terms, the position of acoels out of the Platyhelminthes should warn us against the naive view that considers some features as ‘lost’, ‘absent’, or ‘reduced’ in clades (e.g. acoels) than might never have had them in the first place. Indeed, the possible position of acoels as a sister-group to Xenoturbellida, another group composed of simple organisms, at the base of deuterostomes leaves open the question of the evolutionary origin of their morphological simplicity.</p>
</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4>Gene cloning</h4>
<a id="article1.body1.sec3.sec1.p1" name="article1.body1.sec3.sec1.p1"></a><p>A cDNA library was constructed from adult tissue of the acoel <em>Convoluta pulchra</em> using standard methodologies. The cDNAs were cloned in the plasmid vector pSPORT1. A library of several thousand clones was generated, with an average insert size of 1.5 kb. From the generated collection, 2,304 clones were selected and sequenced using the T7 primer. All sequences were grouped and unigenes plus singletons were selected for the phylogenetic analysis.</p>


<h4>Data assembly</h4>
<a id="article1.body1.sec3.sec2.p1" name="article1.body1.sec3.sec2.p1"></a><p>Each of the gene alignments used in previous studies <a href="#pone.0000717-Baurain1">[14]</a>, <a href="#pone.0000717-Delsuc2">[16]</a>, <a href="#pone.0000717-Philippe3">[23]</a>, <a href="#pone.0000717-Philippe4">[29]</a> was updated with, in addition to <em>Convoluta</em> sequences, newly available sequences downloaded from the Trace Archive (<a href="http://www.ncbi.nlm.nih.gov/Traces/">http://www.ncbi.nlm.nih.gov/Traces/</a>) and the EST Database (<a href="http://www.ncbi.nlm.nih.gov/dbEST/">http://www.ncbi.nlm.nih.gov/dbEST/</a>) of GenBank at the National Center for Biotechnology Information (<a href="http://www.ncbi.nlm.nih.gov/">http://www.ncbi.nlm.nih.gov/</a>) using new features of the program ED from the MUST package <a href="#pone.0000717-Philippe5">[30]</a> (see <a href="#pone.0000717.s001">Table S1</a> for the list of species). Ambiguously aligned regions were automatically detected and removed using the program Gblocks <a href="#pone.0000717-Castresana1">[31]</a> and this selection was manually refined using the program ED.</p>
<a id="article1.body1.sec3.sec2.p2" name="article1.body1.sec3.sec2.p2"></a><p>The concatenation of the 68 genes for which <em>Convoluta</em> sequences were available was done by the program S<span class="small-caps">ca</span>F<span class="small-caps">os</span> <a href="#pone.0000717-Roure1">[<span class="small-caps">32</span>]</a>. S<span class="small-caps">ca</span>F<span class="small-caps">os</span> allows the selection of slowly evolving sequences according to their degree of divergence using ML distances computed under a WAG+F model by T<span class="small-caps">ree-</span>P<span class="small-caps">uzzle</span> <a href="#pone.0000717-Schmidt1">[<span class="small-caps">33</span>]</a>. It also permits to reduce the percentage of missing data per taxon by creating chimerical sequences from species belonging to the same predefined taxonomic group (see <a href="#pone.0000717.s002">Table S2</a>). Only genes that are represented for by at least 42 species were considered. The resulting alignment consists of 68 genes, 52 species and 15,554 unambiguously aligned positions. We further removed all positions for which <em>Convoluta</em> was not present (due to partial gene sequences), resulting in a final alignment of 11,959 positions. Alignments are available upon request from H.P. (<a href="mailto:herve.philippe@umontreal.ca">herve.philippe@umontreal.ca</a>).</p>


<h4>Chimerical Operational Taxonomic Units (OTUs)</h4>
<a id="article1.body1.sec3.sec3.p1" name="article1.body1.sec3.sec3.p1"></a><p>To increase the amount of information, we created chimerical sequences by using closely related taxa in cases where full length sequences were missing. The list of chimerical species is shown in <a href="#pone.0000717.s002">Table S2</a>. Above the species level OTUs have been named after the most frequently represented species in the data set for that inclusive taxonomic group. The list of the 68 genes used in this study as well as the species missing for each gene is given in <a href="#pone.0000717.s003">Table S3</a>.</p>


<h4>Phylogenetic reconstruction</h4>
<a id="article1.body1.sec3.sec4.p1" name="article1.body1.sec3.sec4.p1"></a><p>PhyloBayes analyses were performed with the CAT mixture model, which accounts for across-site heterogeneities in the amino-acid replacement process <a href="#pone.0000717-Lartillot1">[19]</a>. Two independent runs were performed with a total length of 15,000 cycles (250 topological moves per cycle) with the same operators as in Lartillot et al. <a href="#pone.0000717-Lartillot2">[20]</a>. The first 500 points were discarded as burn-in, and the posterior consensus was computed on the 14,500 remaining trees. Bootstrap proportions were obtained after 100 pseudo-replicates generated with S<span class="small-caps">eq</span>B<span class="small-caps">oot</span> <a href="#pone.0000717-Felsenstein3">[<span class="small-caps">34</span>]</a>. For computation time reasons, we performed only 2,000 cycles. We verified that 2,000 cycles gave virtually identical results than 14,500 for the complete dataset. In addition, we used a conservative burn-in value of 1,000 (manual verification of a few replicates indicates that the burn-in is less than 500). Trees were collected after the burn-in and for each replicate the consensus of these 1,000 trees was computed by phylobayes. These 100 consensus trees fed to CONSENSE <a href="#pone.0000717-Felsenstein3">[<span class="small-caps">34</span>]</a> in order to compute the bootstrap support values for each node.</p>
<a id="article1.body1.sec3.sec4.p2" name="article1.body1.sec3.sec4.p2"></a><p>MP heuristic searches were conducted using P<span class="small-caps">aup*</span> <a href="#pone.0000717-Swofford1">[<span class="small-caps">35</span>]</a> with 10 random additions of species, TBR branch swapping and MAXTREES = 1000. MP bootstrap percentages were obtained after 1,000 replications using the same heuristic search strategy using P<span class="small-caps">aup*</span>.</p>
<a id="article1.body1.sec3.sec4.p3" name="article1.body1.sec3.sec4.p3"></a><p>Likelihood-based tests of alternative topologies were calculated using C<span class="small-caps">onsel </span><a href="#pone.0000717-Shimodaira1">[36]</a>. The topology obtained with the CAT model was used as a backbone on which all possible positions of <em>Convoluta</em> were added, yielding 99 (<a href="#pone.0000717.s006">Figure S3</a>) and 91 trees when Platyhelminthes were discarded (<a href="#pone.0000717.s010">Figure S7</a>). ML branch lengths of alternative topologies were first inferred assuming a concatenated WAG+F+Γ<sub>4</sub> model using T<span class="small-caps">ree-</span>P<span class="small-caps">uzzle </span><a href="#pone.0000717-Schmidt1">[33]</a>, site-wise log-likelihood values were then computed with C<span class="small-caps">odeml</span> <a href="#pone.0000717-Yang1">[37]</a> and p-values of the different likelihood-based tests were calculated with C<span class="small-caps">onsel</span>.</p>


<h4>Blast search for gene signature</h4>
<a id="article1.body1.sec3.sec5.p1" name="article1.body1.sec3.sec5.p1"></a><p>We searched for genes that could be informative for the phylogenetic position of <em>Convoluta</em>, i.e. present in all animals including <em>Convoluta</em> but absent, or very divergent, from Platyhelminthes. Given the lack of complete genomes from Lophotrochozoa, we looked for genes absent in protostomes. We used five complete genomes from deuterostomes (<em>Xenopus tropicalis</em>, <em>Homo sapiens</em>, <em>Takifugu rubripes</em>, <em>Ciona intestinalis</em>, and <em>Strongylocentrotus purpuratus</em>) and four from protostomes (<em>Caenorhabditis elegans</em>, <em>Apis melifera</em>, <em>Anopheles gambiae</em>, and <em>Drosophila melanogaster</em>). We run similarity search, using blastx, for the 1664 contigs from <em>Convoluta</em> against these genomes and retained 80 sequences that display 10% more similarity to deuterostomes than to protostomes and ten that display 10% more similarity to protostomes than to deuterostomes. A difference of 10% ensures that sequence similarity is a good indicator of phylogenetic affinity (unpublished observation). These 90 sequences were blasted (blastx) against the nr section of NCBI. A manual inspection of the output allowed to eliminate numerous false positives (e.g. genes lost in insects or nematodes, but present in Platyhelminthes or molluscs), yielding 40 genes for a more careful analysis. We then added sequences from additional species (all the species used for our phylogenomic tree). Only one gene was present in <em>Convoluta</em> and only in deuterostomes but not in protostomes: the guanidinoacetate N-methyltransferase. We performed a similar analysis to test a basal position of <em>Convoluta</em> <a href="#pone.0000717-RuizTrillo1">[5]</a>, <a href="#pone.0000717-RuizTrillo2">[8]</a>, <a href="#pone.0000717-RuizTrillo3">[9]</a>. We looked for genes that are present in <em>Convoluta</em> and in <em>Nematostella</em> (<a href="http://genome.jgi-psf.org/Nemve1/Nemve1.home.html">http://genome.jgi-psf.org/Nemve1/Nemve1.​home.html</a>) but absent from Bilateria. However no genes that would be in favour of a sister-group of <em>Convoluta</em> to all Bilateria were found.</p>

</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0000717"></div><a name="pone.0000717.s001" id="pone.0000717.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000717.s001">Table S1. </a></strong></p><a id="article1.body1.sec4.supplementary-material1.caption1.p1" name="article1.body1.sec4.supplementary-material1.caption1.p1"></a><p class="preSiDOI">List of the species for which new sequence data have been incorporated in the protein alignments.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000717.s001</p><a id="article1.body1.sec4.supplementary-material1.caption1.p2" name="article1.body1.sec4.supplementary-material1.caption1.p2"></a><p class="postSiDOI">(0.01 MB PDF)</p>
<a name="pone.0000717.s002" id="pone.0000717.s002"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000717.s002">Table S2. </a></strong></p><a id="article1.body1.sec4.supplementary-material2.caption1.p1" name="article1.body1.sec4.supplementary-material2.caption1.p1"></a><p class="preSiDOI">List of chimerical Operational Taxonomic Units (OTUs).</p>
<p class="siDoi">doi:10.1371/journal.pone.0000717.s002</p><a id="article1.body1.sec4.supplementary-material2.caption1.p2" name="article1.body1.sec4.supplementary-material2.caption1.p2"></a><p class="postSiDOI">(0.02 MB PDF)</p>
<a name="pone.0000717.s003" id="pone.0000717.s003"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000717.s003">Table S3. </a></strong></p><a id="article1.body1.sec4.supplementary-material3.caption1.p1" name="article1.body1.sec4.supplementary-material3.caption1.p1"></a><p class="preSiDOI">Summary of the occurrence of missing data per taxa in the complete dataset.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000717.s003</p><a id="article1.body1.sec4.supplementary-material3.caption1.p2" name="article1.body1.sec4.supplementary-material3.caption1.p2"></a><p class="postSiDOI">(0.02 MB PDF)</p>
<a name="pone.0000717.s004" id="pone.0000717.s004"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000717.s004">Figure S1. </a></strong></p><a id="article1.body1.sec4.supplementary-material4.caption1.p1" name="article1.body1.sec4.supplementary-material4.caption1.p1"></a><p class="preSiDOI">Maximum parsimony tree inferred from 11,959 unambiguously aligned amino acid positions with PAUP. The robustness of the phylogenetic inference was estimated by 1000 bootstrap replicates. Nodes supported by 100% bootstrap are denoted by black circles while lower values are given explicitly. The scale bar indicates the number of changes.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000717.s004</p><a id="article1.body1.sec4.supplementary-material4.caption1.p2" name="article1.body1.sec4.supplementary-material4.caption1.p2"></a><p class="postSiDOI">(0.02 MB PDF)</p>
<a name="pone.0000717.s005" id="pone.0000717.s005"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000717.s005">Figure S2. </a></strong></p><a id="article1.body1.sec4.supplementary-material5.caption1.p1" name="article1.body1.sec4.supplementary-material5.caption1.p1"></a><p class="preSiDOI">Maximum parsimony tree inferred from 11,959 unambiguously aligned amino acid positions without the fast evolving tunicate Oikopleura. The robustness of the phylogenetic inference was estimated by 1000 bootstrap replicates. Nodes supported by 100% bootstrap are denoted by black circles while lower values are given explicitly. The scale bar indicates the number of changes.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000717.s005</p><a id="article1.body1.sec4.supplementary-material5.caption1.p2" name="article1.body1.sec4.supplementary-material5.caption1.p2"></a><p class="postSiDOI">(0.02 MB PDF)</p>
<a name="pone.0000717.s006" id="pone.0000717.s006"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000717.s006">Figure S3. </a></strong></p><a id="article1.body1.sec4.supplementary-material6.caption1.p1" name="article1.body1.sec4.supplementary-material6.caption1.p1"></a><p class="preSiDOI">Comparison of all possible placements of Convoluta within Holozoa. The possible positions of Convoluta were tested using the program CONSEL (Shimodaira &amp; Hasegawa 2001) on the alignment of 11,959 positions with the WAG+Γ model. Among 99 possible positions, only three were not rejected by the AU test. The number of the topology indicated in second first column is reported on the topology to indicate the position of Convoluta.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000717.s006</p><a id="article1.body1.sec4.supplementary-material6.caption1.p2" name="article1.body1.sec4.supplementary-material6.caption1.p2"></a><p class="postSiDOI">(0.02 MB PDF)</p>
<a name="pone.0000717.s007" id="pone.0000717.s007"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000717.s007">Figure S4. </a></strong></p><a id="article1.body1.sec4.supplementary-material7.caption1.p1" name="article1.body1.sec4.supplementary-material7.caption1.p1"></a><p class="preSiDOI">Bayesian tree inferred from 11,959 unambiguously aligned amino acid positions without Convoluta using the CAT model. The robustness of the phylogenetic inference was estimated by 100 bootstrap replicates. Nodes supported by bootstrap values of 100% are denoted by black circles while lower values are given explicitly. The scale bar indicates the number of changes per site.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000717.s007</p><a id="article1.body1.sec4.supplementary-material7.caption1.p2" name="article1.body1.sec4.supplementary-material7.caption1.p2"></a><p class="postSiDOI">(0.02 MB PDF)</p>
<a name="pone.0000717.s008" id="pone.0000717.s008"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000717.s008">Figure S5. </a></strong></p><a id="article1.body1.sec4.supplementary-material8.caption1.p1" name="article1.body1.sec4.supplementary-material8.caption1.p1"></a><p class="preSiDOI">Bayesian tree inferred from 11,959 unambiguously aligned amino acid positions without the outgroup (Ichthyosporea, Choanoflagellata, Porifera and Cnidaria) using the CAT model. The robustness of the phylogenetic inference was estimated by 100 bootstrap replicates. Nodes supported by bootstrap values of 100% are denoted by black circles while lower values are given explicitly. The scale bar indicates the number of changes per site.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000717.s008</p><a id="article1.body1.sec4.supplementary-material8.caption1.p2" name="article1.body1.sec4.supplementary-material8.caption1.p2"></a><p class="postSiDOI">(0.01 MB PDF)</p>
<a name="pone.0000717.s009" id="pone.0000717.s009"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000717.s009">Figure S6. </a></strong></p><a id="article1.body1.sec4.supplementary-material9.caption1.p1" name="article1.body1.sec4.supplementary-material9.caption1.p1"></a><p class="preSiDOI">Bayesian tree inferred from 11,959 unambiguously aligned amino acid positions without Convoluta and the outgroup (Ichthyosporea, Choanoflagellata, Porifera and Cnidaria) using the CAT model. The robustness of the phylogenetic inference was estimated through 100 bootstrap replicates. Nodes supported by bootstrap values of 100% are denoted by black circles while lower values are given explicitly. The scale bar indicates the number of changes per site.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000717.s009</p><a id="article1.body1.sec4.supplementary-material9.caption1.p2" name="article1.body1.sec4.supplementary-material9.caption1.p2"></a><p class="postSiDOI">(0.01 MB PDF)</p>
<a name="pone.0000717.s010" id="pone.0000717.s010"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000717.s010">Figure S7. </a></strong></p><a id="article1.body1.sec4.supplementary-material10.caption1.p1" name="article1.body1.sec4.supplementary-material10.caption1.p1"></a><p class="preSiDOI">Comparison of all possible placements of Convoluta within Holozoa when Platyhelminthes are removed. The possible positions of Convoluta were tested using the program CONSEL (Shimodaira &amp; Hasegawa 2001) on the alignment of 11,959 positions with the WAG+Γ model. Among 91 possible positions, six were not rejected by the AU test. The number of the topology indicated in the second column is reported on the topology to indicate the position of Convoluta.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000717.s010</p><a id="article1.body1.sec4.supplementary-material10.caption1.p2" name="article1.body1.sec4.supplementary-material10.caption1.p2"></a><p class="postSiDOI">(0.02 MB PDF)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>We would like to thank Peter Ladurner (University of Innsbruck, Austria) for providing specimens of <em>Convoluta pulchra</em> and Ketil Malde (University of Bergen, Norway) for helping in the EST data analysis. The Réseau Québecois de Calcul de Haute Performance, the Barcelona Supercomputing Centre (Mare nostrum) and the Centre de supercomputació de Catalunya are acknowledged for computational resources.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: HP JB. Performed the experiments: PM MR. Analyzed the data: HP HB. Contributed reagents/materials/analysis tools: HP JB PM MR. Wrote the paper: HP JB.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0000717-Hyman1" id="pone.0000717-Hyman1"></a>Hyman L (1951) The Invertebrates. Vol. II. Platyhelminthes and Rhynchocoela. New York: McGraw-Hill.   <ul class="find-nolinks"></ul></li><li><span class="label">2.
              </span><a name="pone.0000717-Peterson1" id="pone.0000717-Peterson1"></a>Peterson KJ, Eernisse DJ (2001) Animal phylogeny and the ancestry of bilaterians: inferences from morphology and 18S rDNA gene sequences. Evolution&amp;Development  3: 170–205.  <ul class="find" data-citedArticleID="975652" data-doi="10.1046/j.1525-142x.2001.003003170.x"><li><a href="http://dx.doi.org/10.1046/j.1525-142x.2001.003003170.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Animal+phylogeny+and+the+ancestry+of+bilaterians%3A+inferences+from+morphology+and+18S+rDNA+gene+sequences." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Animal+phylogeny+and+the+ancestry+of+bilaterians%3A+inferences+from+morphology+and+18S+rDNA+gene+sequences.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0000717-Smith1" id="pone.0000717-Smith1"></a>Smith JPS, Tyler S, Rieger RM (1986) Is the Turbellaria polyphyletic? Hydrobiologia  132: 71–78.  <ul class="find" data-citedArticleID="975676" data-doi="10.1007/bf00046231"><li><a href="http://dx.doi.org/10.1007/bf00046231" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Is+the+Turbellaria+polyphyletic%3F" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Is+the+Turbellaria+polyphyletic%3F%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0000717-Bagu1" id="pone.0000717-Bagu1"></a>Baguñà J, Riutort M (2004) The dawn of bilaterian animals: the case of acoelomorph flatworms. Bioessays  26: 1046–1057.  <ul class="find" data-citedArticleID="975614" data-doi="10.1002/bies.20113"><li><a href="http://dx.doi.org/10.1002/bies.20113" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+dawn+of+bilaterian+animals%3A+the+case+of+acoelomorph+flatworms." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+dawn+of+bilaterian+animals%3A+the+case+of+acoelomorph+flatworms.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0000717-RuizTrillo1" id="pone.0000717-RuizTrillo1"></a>Ruiz-Trillo I, Riutort M, Littlewood DT, Herniou EA, Baguñà J (1999) Acoel flatworms: earliest extant bilaterian Metazoans, not members of Platyhelminthes. Science  283: 1919–1923.  <ul class="find" data-citedArticleID="975666" data-doi="10.1126/science.283.5409.1919"><li><a href="http://dx.doi.org/10.1126/science.283.5409.1919" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Acoel+flatworms%3A+earliest+extant+bilaterian+Metazoans%2C+not+members+of+Platyhelminthes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Acoel+flatworms%3A+earliest+extant+bilaterian+Metazoans%2C+not+members+of+Platyhelminthes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0000717-Jondelius1" id="pone.0000717-Jondelius1"></a>Jondelius U, Ruiz-Trillo I, Baguñà J, Riutort M (2002) The Nemertodermatida are basal bilaterians not members of Platyhelminthes. Zool Scr  31: 201–215.  <ul class="find" data-citedArticleID="975640" data-doi="10.1046/j.1463-6409.2002.00090.x"><li><a href="http://dx.doi.org/10.1046/j.1463-6409.2002.00090.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+Nemertodermatida+are+basal+bilaterians+not+members+of+Platyhelminthes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+Nemertodermatida+are+basal+bilaterians+not+members+of+Platyhelminthes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0000717-Telford1" id="pone.0000717-Telford1"></a>Telford MJ, Lockyer AE, Cartwright-Finch C, Littlewood DT (2003) Combined large and small subunit ribosomal RNA phylogenies support a basal position of the acoelomorph flatworms. Proc R Soc Lond B Biol Sci  270: 1077–1083.  <ul class="find" data-citedArticleID="975680" data-doi="10.1098/rspb.2003.2342"><li><a href="http://dx.doi.org/10.1098/rspb.2003.2342" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Combined+large+and+small+subunit+ribosomal+RNA+phylogenies+support+a+basal+position+of+the+acoelomorph+flatworms." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Combined+large+and+small+subunit+ribosomal+RNA+phylogenies+support+a+basal+position+of+the+acoelomorph+flatworms.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0000717-RuizTrillo2" id="pone.0000717-RuizTrillo2"></a>Ruiz-Trillo I, Paps J, Loukota M, Ribera C, Jondelius U, et al.  (2002) A phylogenetic analysis of myosin heavy chain type II sequences corroborates that Acoela and Nemertodermatida are basal bilaterians. Proc Natl Acad Sci U S A  99: 11246–11251.  <ul class="find" data-citedArticleID="975668" data-doi="10.1073/pnas.172390199"><li><a href="http://dx.doi.org/10.1073/pnas.172390199" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+phylogenetic+analysis+of+myosin+heavy+chain+type+II+sequences+corroborates+that+Acoela+and+Nemertodermatida+are+basal+bilaterians." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+phylogenetic+analysis+of+myosin+heavy+chain+type+II+sequences+corroborates+that+Acoela+and+Nemertodermatida+are+basal+bilaterians.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0000717-RuizTrillo3" id="pone.0000717-RuizTrillo3"></a>Ruiz-Trillo I, Riutort M, Fourcade HM, Baguñà J, Boore JL (2004) Mitochondrial genome data support the basal position of Acoelomorpha and the polyphyly of the Platyhelminthes. Mol Phylogenet Evol  33: 321–332.  <ul class="find" data-citedArticleID="975670" data-doi="10.1016/j.ympev.2004.06.002"><li><a href="http://dx.doi.org/10.1016/j.ympev.2004.06.002" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mitochondrial+genome+data+support+the+basal+position+of+Acoelomorpha+and+the+polyphyly+of+the+Platyhelminthes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mitochondrial+genome+data+support+the+basal+position+of+Acoelomorpha+and+the+polyphyly+of+the+Platyhelminthes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0000717-Felsenstein1" id="pone.0000717-Felsenstein1"></a>Felsenstein J (1978) Cases in which parsimony or compatibility methods will be positively misleading. Syst Zool  27: 401–410.  <ul class="find" data-citedArticleID="975630" data-doi="10.2307/2412923"><li><a href="http://dx.doi.org/10.2307/2412923" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Cases+in+which+parsimony+or+compatibility+methods+will+be+positively+misleading." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Cases+in+which+parsimony+or+compatibility+methods+will+be+positively+misleading.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0000717-Adoutte1" id="pone.0000717-Adoutte1"></a>Adoutte A, Balavoine G, Lartillot N, Lespinet O, Prud'homme B, et al.  (2000) The new animal phylogeny: reliability and implications. Proc Natl Acad Sci U S A  97: 4453–4456.  <ul class="find" data-citedArticleID="975610" data-doi="10.1073/pnas.97.9.4453"><li><a href="http://dx.doi.org/10.1073/pnas.97.9.4453" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+new+animal+phylogeny%3A+reliability+and+implications." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+new+animal+phylogeny%3A+reliability+and+implications.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0000717-Halanych1" id="pone.0000717-Halanych1"></a>Halanych KM (2004) The new view of animal phylogeny. Annual Review of Ecology Evolution and Systematics  35: 229–256.  <ul class="find" data-citedArticleID="975636" data-doi="10.1146/annurev.ecolsys.35.112202.130124"><li><a href="http://dx.doi.org/10.1146/annurev.ecolsys.35.112202.130124" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+new+view+of+animal+phylogeny." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+new+view+of+animal+phylogeny.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0000717-Delsuc1" id="pone.0000717-Delsuc1"></a>Delsuc F, Brinkmann H, Philippe H (2005) Phylogenomics and the reconstruction of the tree of life. Nat Rev Genet  6: 361–375.  <ul class="find" data-citedArticleID="975624" data-doi="10.1038/nrg1603"><li><a href="http://dx.doi.org/10.1038/nrg1603" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Phylogenomics+and+the+reconstruction+of+the+tree+of+life." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Phylogenomics+and+the+reconstruction+of+the+tree+of+life.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0000717-Baurain1" id="pone.0000717-Baurain1"></a>Baurain D, Brinkmann H, Philippe H (2007) Lack of resolution in the animal phylogeny: closely spaced cladogeneses or undetected systematic errors? Mol Biol Evol  24: 6–9.  <ul class="find" data-citedArticleID="975616" data-doi="10.1093/molbev/msl137"><li><a href="http://dx.doi.org/10.1093/molbev/msl137" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Lack+of+resolution+in+the+animal+phylogeny%3A+closely+spaced+cladogeneses+or+undetected+systematic+errors%3F" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Lack+of+resolution+in+the+animal+phylogeny%3A+closely+spaced+cladogeneses+or+undetected+systematic+errors%3F%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0000717-Philippe1" id="pone.0000717-Philippe1"></a>Philippe H, Telford MJ (2006) Large-scale sequencing and the new animal phylogeny. Trends Ecol Evol  21: 614–620.  <ul class="find" data-citedArticleID="975654" data-doi="10.1016/j.tree.2006.08.004"><li><a href="http://dx.doi.org/10.1016/j.tree.2006.08.004" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Large-scale+sequencing+and+the+new+animal+phylogeny." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Large-scale+sequencing+and+the+new+animal+phylogeny.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0000717-Delsuc2" id="pone.0000717-Delsuc2"></a>Delsuc F, Brinkmann H, Chourrout D, Philippe H (2006) Tunicates and not cephalochordates are the closest living relatives of vertebrates. Nature  439: 965–968.  <ul class="find" data-citedArticleID="975626" data-doi="10.1038/nature04336"><li><a href="http://dx.doi.org/10.1038/nature04336" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Tunicates+and+not+cephalochordates+are+the+closest+living+relatives+of+vertebrates." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Tunicates+and+not+cephalochordates+are+the+closest+living+relatives+of+vertebrates.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0000717-Philippe2" id="pone.0000717-Philippe2"></a>Philippe H, Zhou Y, Brinkmann H, Rodrigue N, Delsuc F (2005) Heterotachy and long-branch attraction in phylogenetics. BMC Evol Biol  5: 50.  <ul class="find" data-citedArticleID="975656" data-doi="10.1186/1471-2148-5-50"><li><a href="http://dx.doi.org/10.1186/1471-2148-5-50" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Heterotachy+and+long-branch+attraction+in+phylogenetics." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Heterotachy+and+long-branch+attraction+in+phylogenetics.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0000717-Felsenstein2" id="pone.0000717-Felsenstein2"></a>Felsenstein J (1981) Evolutionary trees from DNA sequences: a maximum likelihood approach. J Mol Evol  17: 368–376.  <ul class="find" data-citedArticleID="975632" data-doi="10.1007/bf01734359"><li><a href="http://dx.doi.org/10.1007/bf01734359" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evolutionary+trees+from+DNA+sequences%3A+a+maximum+likelihood+approach." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evolutionary+trees+from+DNA+sequences%3A+a+maximum+likelihood+approach.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0000717-Lartillot1" id="pone.0000717-Lartillot1"></a>Lartillot N, Philippe H (2004) A Bayesian mixture model for across-site heterogeneities in the amino-acid replacement process. Mol Biol Evol  21: 1095–1109.  <ul class="find" data-citedArticleID="975642" data-doi="10.1093/molbev/msh112"><li><a href="http://dx.doi.org/10.1093/molbev/msh112" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+Bayesian+mixture+model+for+across-site+heterogeneities+in+the+amino-acid+replacement+process." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+Bayesian+mixture+model+for+across-site+heterogeneities+in+the+amino-acid+replacement+process.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0000717-Lartillot2" id="pone.0000717-Lartillot2"></a>Lartillot N, Brinkmann H, Philippe H (2007) Suppression of long-branch attraction artefacts in the animal phylogeny using a site-heterogeneous model. BMC Evol Biol  7: Suppl 1S4.  <ul class="find" data-citedArticleID="975644" data-doi="10.1186/1471-2148-7-s1-s4"><li><a href="http://dx.doi.org/10.1186/1471-2148-7-s1-s4" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Suppression+of+long-branch+attraction+artefacts+in+the+animal+phylogeny+using+a+site-heterogeneous+model." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Suppression+of+long-branch+attraction+artefacts+in+the+animal+phylogeny+using+a+site-heterogeneous+model.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0000717-Bourlat1" id="pone.0000717-Bourlat1"></a>Bourlat SJ, Juliusdottir T, Lowe CJ, Freeman R, Aronowicz J, et al.  (2006) Deuterostome phylogeny reveals monophyletic chordates and the new phylum Xenoturbellida. Nature  444: 85–88.  <ul class="find" data-citedArticleID="975618" data-doi="10.1038/nature05241"><li><a href="http://dx.doi.org/10.1038/nature05241" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Deuterostome+phylogeny+reveals+monophyletic+chordates+and+the+new+phylum+Xenoturbellida." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Deuterostome+phylogeny+reveals+monophyletic+chordates+and+the+new+phylum+Xenoturbellida.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0000717-Matus1" id="pone.0000717-Matus1"></a>Matus DQ, Copley RR, Dunn CW, Hejnol A, Eccleston H, et al.  (2006) Broad taxon and gene sampling indicate that chaetognaths are protostomes. Curr Biol  16: R575–576.  <ul class="find" data-citedArticleID="975650" data-doi="10.1016/j.cub.2006.07.017"><li><a href="http://dx.doi.org/10.1016/j.cub.2006.07.017" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Broad+taxon+and+gene+sampling+indicate+that+chaetognaths+are+protostomes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Broad+taxon+and+gene+sampling+indicate+that+chaetognaths+are+protostomes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0000717-Philippe3" id="pone.0000717-Philippe3"></a>Philippe H, Lartillot N, Brinkmann H (2005) Multigene analyses of bilaterian animals corroborate the monophyly of Ecdysozoa, Lophotrochozoa, and Protostomia. Mol Biol Evol  22: 1246–1253.  <ul class="find" data-citedArticleID="975658" data-doi="10.1093/molbev/msi111"><li><a href="http://dx.doi.org/10.1093/molbev/msi111" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Multigene+analyses+of+bilaterian+animals+corroborate+the+monophyly+of+Ecdysozoa%2C+Lophotrochozoa%2C+and+Protostomia." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Multigene+analyses+of+bilaterian+animals+corroborate+the+monophyly+of+Ecdysozoa%2C+Lophotrochozoa%2C+and+Protostomia.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0000717-Marletaz1" id="pone.0000717-Marletaz1"></a>Marletaz F, Martin E, Perez Y, Papillon D, Caubit X, et al.  (2006) Chaetognath phylogenomics: a protostome with deuterostome-like development. Curr Biol  16: R577–578.  <ul class="find" data-citedArticleID="975648" data-doi="10.1016/j.cub.2006.07.016"><li><a href="http://dx.doi.org/10.1016/j.cub.2006.07.016" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Chaetognath+phylogenomics%3A+a+protostome+with+deuterostome-like+development." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Chaetognath+phylogenomics%3A+a+protostome+with+deuterostome-like+development.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0000717-Brinkmann1" id="pone.0000717-Brinkmann1"></a>Brinkmann H, Giezen M, Zhou Y, Raucourt GP, Philippe H (2005) An empirical assessment of long-branch attraction artefacts in deep eukaryotic phylogenomics. Syst Biol  54: 743–757.  <ul class="find" data-citedArticleID="975620"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=An+empirical+assessment+of+long-branch+attraction+artefacts+in+deep+eukaryotic+phylogenomics.&amp;auth=&amp;atitle=An+empirical+assessment+of+long-branch+attraction+artefacts+in+deep+eukaryotic+phylogenomics." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=An+empirical+assessment+of+long-branch+attraction+artefacts+in+deep+eukaryotic+phylogenomics." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22An+empirical+assessment+of+long-branch+attraction+artefacts+in+deep+eukaryotic+phylogenomics.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0000717-Ax1" id="pone.0000717-Ax1"></a>Ax P (1996) Multicellular Animals. A New Approach to the Phylogenetic Order in Nature. Berlin: Springer.   <ul class="find-nolinks"></ul></li><li><span class="label">27.
              </span><a name="pone.0000717-Ehlers1" id="pone.0000717-Ehlers1"></a>Ehlers U (1985) Das Phylogenetische System der Plathelminthes. Stuttgart, New York: G. Fischer.   <ul class="find-nolinks"></ul></li><li><span class="label">28.
              </span><a name="pone.0000717-Lundin1" id="pone.0000717-Lundin1"></a>Lundin K (1998) The epidermal ciliary rootlets of Xenoturbella bocki (Xenoturbellida) revisited: new support for a possible kinship with the Acoelomorpha (Platyhelminthes). Zoologica Scripta  27: 263–270.  <ul class="find" data-citedArticleID="975646" data-doi="10.1111/j.1463-6409.1998.tb00440.x"><li><a href="http://dx.doi.org/10.1111/j.1463-6409.1998.tb00440.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+epidermal+ciliary+rootlets+of+Xenoturbella+bocki+%28Xenoturbellida%29+revisited%3A+new+support+for+a+possible+kinship+with+the+Acoelomorpha+%28Platyhelminthes%29." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+epidermal+ciliary+rootlets+of+Xenoturbella+bocki+%28Xenoturbellida%29+revisited%3A+new+support+for+a+possible+kinship+with+the+Acoelomorpha+%28Platyhelminthes%29.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0000717-Philippe4" id="pone.0000717-Philippe4"></a>Philippe H, Snell EA, Bapteste E, Lopez P, Holland PW, et al.  (2004) Phylogenomics of eukaryotes: impact of missing data on large alignments. Mol Biol Evol  21: 1740–1752.  <ul class="find" data-citedArticleID="975660" data-doi="10.1093/molbev/msh182"><li><a href="http://dx.doi.org/10.1093/molbev/msh182" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Phylogenomics+of+eukaryotes%3A+impact+of+missing+data+on+large+alignments." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Phylogenomics+of+eukaryotes%3A+impact+of+missing+data+on+large+alignments.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0000717-Philippe5" id="pone.0000717-Philippe5"></a>Philippe H (1993) MUST, a computer package of Management Utilities for Sequences and Trees. Nucleic Acids Res  21: 5264–5272.  <ul class="find" data-citedArticleID="975662" data-doi="10.1093/nar/21.22.5264"><li><a href="http://dx.doi.org/10.1093/nar/21.22.5264" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=MUST%2C+a+computer+package+of+Management+Utilities+for+Sequences+and+Trees." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22MUST%2C+a+computer+package+of+Management+Utilities+for+Sequences+and+Trees.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0000717-Castresana1" id="pone.0000717-Castresana1"></a>Castresana J (2000) Selection of Conserved Blocks from Multiple Alignments for Their Use in Phylogenetic Analysis. Mol Biol Evol  17: 540–552.  <ul class="find" data-citedArticleID="975622" data-doi="10.1093/oxfordjournals.molbev.a026334"><li><a href="http://dx.doi.org/10.1093/oxfordjournals.molbev.a026334" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Selection+of+Conserved+Blocks+from+Multiple+Alignments+for+Their+Use+in+Phylogenetic+Analysis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Selection+of+Conserved+Blocks+from+Multiple+Alignments+for+Their+Use+in+Phylogenetic+Analysis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0000717-Roure1" id="pone.0000717-Roure1"></a>Roure B, Rodriguez-Ezpeleta N, Philippe H (2007) SCaFoS: a tool for Selection, Concatenation and Fusion of Sequences for phylogenomics. BMC Evol Biol  7: Suppl 1S2.  <ul class="find" data-citedArticleID="975664" data-doi="10.1186/1471-2148-7-s1-s2"><li><a href="http://dx.doi.org/10.1186/1471-2148-7-s1-s2" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=SCaFoS%3A+a+tool+for+Selection%2C+Concatenation+and+Fusion+of+Sequences+for+phylogenomics." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22SCaFoS%3A+a+tool+for+Selection%2C+Concatenation+and+Fusion+of+Sequences+for+phylogenomics.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0000717-Schmidt1" id="pone.0000717-Schmidt1"></a>Schmidt HA, Strimmer K, Vingron M, von Haeseler A (2002) TREE-PUZZLE: maximum likelihood phylogenetic analysis using quartets and parallel computing. Bioinformatics  18: 502–504.  <ul class="find" data-citedArticleID="975672" data-doi="10.1093/bioinformatics/18.3.502"><li><a href="http://dx.doi.org/10.1093/bioinformatics/18.3.502" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=TREE-PUZZLE%3A+maximum+likelihood+phylogenetic+analysis+using+quartets+and+parallel+computing." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22TREE-PUZZLE%3A+maximum+likelihood+phylogenetic+analysis+using+quartets+and+parallel+computing.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0000717-Felsenstein3" id="pone.0000717-Felsenstein3"></a>Felsenstein J (2001) PHYLIP (Phylogene Inference Package). Seattle: Department of Genetics, University of Washington.   3.6 ed: Distributed by the author.  <ul class="find-nolinks"></ul></li><li><span class="label">35.
              </span><a name="pone.0000717-Swofford1" id="pone.0000717-Swofford1"></a>Swofford DL (2000) PAUP*: Phylogenetic Analysis Using Parsimony and other methods. Sunderland, MA: Sinauer.   4b10 ed:.  <ul class="find-nolinks"></ul></li><li><span class="label">36.
              </span><a name="pone.0000717-Shimodaira1" id="pone.0000717-Shimodaira1"></a>Shimodaira H, Hasegawa M (2001) CONSEL: for assessing the confidence of phylogenetic tree selection. Bioinformatics  17: 1246–1247.  <ul class="find" data-citedArticleID="975674" data-doi="10.1093/bioinformatics/17.12.1246"><li><a href="http://dx.doi.org/10.1093/bioinformatics/17.12.1246" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=CONSEL%3A+for+assessing+the+confidence+of+phylogenetic+tree+selection." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22CONSEL%3A+for+assessing+the+confidence+of+phylogenetic+tree+selection.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">37.
              </span><a name="pone.0000717-Yang1" id="pone.0000717-Yang1"></a>Yang Z (1997) PAML: a program package for phylogenetic analysis by maximum likelihood. Comput Appl Biosci  13: 555–556.  <ul class="find" data-citedArticleID="975682" data-doi="10.1093/bioinformatics/13.5.555"><li><a href="http://dx.doi.org/10.1093/bioinformatics/13.5.555" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=PAML%3A+a+program+package+for+phylogenetic+analysis+by+maximum+likelihood." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22PAML%3A+a+program+package+for+phylogenetic+analysis+by+maximum+likelihood.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000717.XML" value="65361"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000717.PDF" value="162268"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000717.g001.PNG_L" value="76805"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000717.g001.PNG_M" value="64789"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000717.g001.PNG_S" value="9968"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000717.g001.TIF" value="124418"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000717.g001.PNG_I" value="25731"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000717.s001.PDF" value="10982"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000717.s002.PDF" value="15220"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000717.s003.PDF" value="19705"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000717.s004.PDF" value="15576"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000717.s005.PDF" value="15590"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000717.s006.PDF" value="17581"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000717.s007.PDF" value="15672"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000717.s008.PDF" value="14811"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000717.s009.PDF" value="14536"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000717.s010.PDF" value="17172"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000717&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000717" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000717&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0000717&volume=&issue=&title=Acoel Flatworms Are Not Platyhelminthes: Evidence from Phylogenomics&author_name=Herv%C3%A9%20Philippe%2C%20Henner%20Brinkmann%2C%20Pedro%20Martinez%2C%20Marta%20Riutort%2C%20Jaume%20Bagu%C3%B1%C3%A0&start_page=1&end_page=5" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000717" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000717" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0000717" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000717&amp;t=Acoel%20Flatworms%20Are%20Not%20Platyhelminthes%3A%20Evidence%20from%20Phylogenomics" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000717&title=Acoel%20Flatworms%20Are%20Not%20Platyhelminthes%3A%20Evidence%20from%20Phylogenomics&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000717&amp;title=Acoel%20Flatworms%20Are%20Not%20Platyhelminthes%3A%20Evidence%20from%20Phylogenomics" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000717" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0000717&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Acoel Flatworms Are Not Platyhelminthes: Evidence from Phylogenomics';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0000717';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Acoel%20Flatworms%20Are%20Not%20Platyhelminthes%3A%20Evidence%20from%20Phylogenomics http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000717" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0000717" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">













          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Animal+genomics%22" title="Search for articles in the subject area:'Animal genomics'"><div class="flagText">Animal genomics</div></a>
              <div data-categoryid="17455" data-articleid="24272"
                   data-categoryname="Animal genomics"
                   class="flagImage" title="Flag 'Animal genomics' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Animal+phylogenetics%22" title="Search for articles in the subject area:'Animal phylogenetics'"><div class="flagText">Animal phylogenetics</div></a>
              <div data-categoryid="46223" data-articleid="24272"
                   data-categoryname="Animal phylogenetics"
                   class="flagImage" title="Flag 'Animal phylogenetics' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Flatworms%22" title="Search for articles in the subject area:'Flatworms'"><div class="flagText">Flatworms</div></a>
              <div data-categoryid="43295" data-articleid="24272"
                   data-categoryname="Flatworms"
                   class="flagImage" title="Flag 'Flatworms' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Invertebrate+genomics%22" title="Search for articles in the subject area:'Invertebrate genomics'"><div class="flagText">Invertebrate genomics</div></a>
              <div data-categoryid="17221" data-articleid="24272"
                   data-categoryname="Invertebrate genomics"
                   class="flagImage" title="Flag 'Invertebrate genomics' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Mammalian+genomics%22" title="Search for articles in the subject area:'Mammalian genomics'"><div class="flagText">Mammalian genomics</div></a>
              <div data-categoryid="17625" data-articleid="24272"
                   data-categoryname="Mammalian genomics"
                   class="flagImage" title="Flag 'Mammalian genomics' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Phylogenetic+analysis%22" title="Search for articles in the subject area:'Phylogenetic analysis'"><div class="flagText">Phylogenetic analysis</div></a>
              <div data-categoryid="17851" data-articleid="24272"
                   data-categoryname="Phylogenetic analysis"
                   class="flagImage" title="Flag 'Phylogenetic analysis' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Sequence+alignment%22" title="Search for articles in the subject area:'Sequence alignment'"><div class="flagText">Sequence alignment</div></a>
              <div data-categoryid="47429" data-articleid="24272"
                   data-categoryname="Sequence alignment"
                   class="flagImage" title="Flag 'Sequence alignment' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Topology%22" title="Search for articles in the subject area:'Topology'"><div class="flagText">Topology</div></a>
              <div data-categoryid="32019" data-articleid="24272"
                   data-categoryname="Topology"
                   class="flagImage" title="Flag 'Topology' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=4576'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=8101'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=6650&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>

<div class="block sidebar-comments">
    <div class="header">
        <h3>Comments</h3>
    </div>
      <p><a href="/annotation/listThread.action?root=20381">Corrigendum</a><br>Posted by PLoS_ONE_Group</p>
      <p><a href="/annotation/listThread.action?root=8819">Referee Comments: Referee 1 (Tim Littlewood)</a><br>Posted by PLoS_ONE_Group</p>
      <p><a href="/annotation/listThread.action?root=42487">convincing result</a><br>Posted by jstajich</p>
</div>

</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
