

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Recombination and Population Mosaic of a Multifunctional Viral Gene, Adeno-Associated Virus cap</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0001634"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0001634" />

    <meta name="citation_title" content="Recombination and Population Mosaic of a Multifunctional Viral Gene, Adeno-Associated Virus cap"/>
    <meta itemprop="name" content="Recombination and Population Mosaic of a Multifunctional Viral Gene, Adeno-Associated Virus cap"/>

      <meta name="citation_author" content="Yasuhiro Takeuchi"/>
            <meta name="citation_author_institution" content="Medical Reserach Council (MRC)/UCL Centre for Medical Molecular Virology, Division of Infection and Immunology, University College London, London, United Kingdom"/>
      <meta name="citation_author" content="Richard Myers"/>
            <meta name="citation_author_institution" content="Medical Reserach Council (MRC)/UCL Centre for Medical Molecular Virology, Division of Infection and Immunology, University College London, London, United Kingdom"/>
            <meta name="citation_author_institution" content="Centre for Infections, Health Protection Agency, London, United Kingdom"/>
      <meta name="citation_author" content="Olivier Danos"/>
            <meta name="citation_author_institution" content="Medical Reserach Council (MRC)/UCL Centre for Medical Molecular Virology, Division of Infection and Immunology, University College London, London, United Kingdom"/>
            <meta name="citation_author_institution" content="Institut National de la Santé et de la Recherche Médicale (INSERM), U781, Institut Fédératif de Recherche Necker Enfants Malades (IFR 94), Hôpital Necker Enfants Malades, Paris, France"/>

    <meta name="citation_date" content="2008/2/20"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0001634.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e1634"/>
    <meta name="citation_issue" content="2"/>
    <meta name="citation_volume" content="3"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Parvovirus host range, cell tropism and evolution.; citation_author=K Hueffer; citation_author=CR Parrish; citation_journal_title=Current Opinion in Microbiology; citation_volume=6; citation_number=1; citation_pages=392; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=High rate of viral evolution associated with the emergence of carnivore parvovirus.; citation_author=LA Shackelton; citation_author=CR Parrish; citation_author=U Truyen; citation_author=EC Holmes; citation_journal_title=PNAS; citation_volume=102; citation_number=2; citation_pages=379-384; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Adeno-associated Virus Serotypes: Vector Toolkit for Human Gene Therapy.; citation_author=Z Wu; citation_author=A Asokan; citation_author=RJ Samulski; citation_journal_title=Molecular Therapy; citation_volume=14; citation_number=3; citation_pages=316-327; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=The atomic structure of adeno-associated virus (AAV-2), a vector for human gene therapy.; citation_author=Q Xie; citation_author=W Bu; citation_author=S Bhatia; citation_author=J Hare; citation_author=T Somasundaram; citation_journal_title=PNAS; citation_volume=99; citation_number=4; citation_pages=10405-10410; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Surface-Exposed Adeno-Associated Virus Vp1-NLS Capsid Fusion Protein Rescues Infectivity of Noninfectious Wild-Type Vp2/Vp3 and Vp3-Only Capsids but Not That of Fivefold Pore Mutant Virions.; citation_author=JC Grieger; citation_author=JS Johnson; citation_author=B Gurda-Whitaker; citation_author=M Agbandje-McKenna; citation_author=RJ Samulski; citation_journal_title=J Virol; citation_volume=81; citation_number=5; citation_pages=7833-7843; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=A Conformational Change in the Adeno-Associated Virus Type 2 Capsid Leads to the Exposure of Hidden VP1 N Termini.; citation_author=S Kronenberg; citation_author=B Bottcher; citation_author=CW von der Lieth; citation_author=S Bleker; citation_author=JA Kleinschmidt; citation_journal_title=J Virol; citation_volume=79; citation_number=6; citation_pages=5296-5303; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Adeno-Associated Virus Type 2 Capsids with Externalized VP1/VP2 Trafficking Domains Are Generated prior to Passage through the Cytoplasm and Are Maintained until Uncoating Occurs in the Nucleus.; citation_author=F Sonntag; citation_author=S Bleker; citation_author=B Leuchs; citation_author=R Fischer; citation_author=JA Kleinschmidt; citation_journal_title=J Virol; citation_volume=80; citation_number=7; citation_pages=11040-11054; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Separate Basic Region Motifs within the Adeno-Associated Virus Capsid Proteins Are Essential for Infectivity and Assembly.; citation_author=JC Grieger; citation_author=S Snowdy; citation_author=RJ Samulski; citation_journal_title=J Virol; citation_volume=80; citation_number=8; citation_pages=5199-5210; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=A Viral Phospholipase A2 Is Required for Parvovirus Infectivity.; citation_author=Z Zadori; citation_author=J Szelei; citation_author=M-C Lacoste; citation_author=Y Li; citation_author=S Gariepy; citation_journal_title=Developmental Cell; citation_volume=1; citation_number=9; citation_pages=291-302; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Clades of Adeno-Associated Viruses Are Widely Disseminated in Human Tissues.; citation_author=G Gao; citation_author=LH Vandenberghe; citation_author=MR Alvira; citation_author=Y Lu; citation_author=R Calcedo; citation_journal_title=J Virol; citation_volume=78; citation_number=10; citation_pages=6381-6388; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Molecular Characterization of Adeno-Associated Viruses Infecting Children.; citation_author=C-L Chen; citation_author=RL Jensen; citation_author=BC Schnepp; citation_author=MJ Connell; citation_author=R Shell; citation_journal_title=J Virol; citation_volume=79; citation_number=11; citation_pages=14781-14792; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Adeno-associated viruses undergo substantial evolution in primates during natural infections.; citation_author=G Gao; citation_author=MR Alvira; citation_author=S Somanathan; citation_author=Y Lu; citation_author=LH Vandenberghe; citation_journal_title=PNAS; citation_volume=100; citation_number=12; citation_pages=6081-6086; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Gene Therapy Vectors Based on Adeno-Associated Virus Type 1.; citation_author=W Xiao; citation_author=N Chirmule; citation_author=SC Berta; citation_author=B McCullough; citation_author=G Gao; citation_journal_title=J Virol; citation_volume=73; citation_number=13; citation_pages=3994-4003; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Evidence and consequence of porcine endogenous retrovirus recombination.; citation_author=B Bartosch; citation_author=D Stefanidis; citation_author=R Myers; citation_author=R Weiss; citation_author=C Patience; citation_journal_title=J Virol; citation_volume=78; citation_number=14; citation_pages=13880-13890; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=DNA amplification of adeno-associated virus as a response to cellular genotoxic stress.; citation_author=AO Yalkinoglu; citation_author=R Heilbronn; citation_author=A Burkle; citation_author=JR Schlehofer; citation_author=H zur Hausen; citation_journal_title=Cancer Res; citation_volume=48; citation_number=15; citation_pages=3123-3129; citation_date=1988; " />
      <meta name="citation_reference" content="citation_title=Recruitment of wild-type and recombinant adeno-associated virus into adenovirus replication centers.; citation_author=MD Weitzman; citation_author=KJ Fisher; citation_author=JM Wilson; citation_journal_title=J Virol; citation_volume=70; citation_number=16; citation_pages=1845-1854; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Virus-mediated killing of cells that lack p53 activity.; citation_author=K Raj; citation_author=P Ogston; citation_author=P Beard; citation_journal_title=Nature; citation_volume=412; citation_number=17; citation_pages=914; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Inverted Terminal Repeat Sequences Are Important for Intermolecular Recombination and Circularization of Adeno-Associated Virus Genomes.; citation_author=Z Yan; citation_author=R Zak; citation_author=Y Zhang; citation_author=JF Engelhardt; citation_journal_title=J Virol; citation_volume=79; citation_number=18; citation_pages=364-379; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Involvement of Cellular Double-Stranded DNA Break Binding Proteins in Processing of the Recombinant Adeno-Associated Virus Genome.; citation_author=L Zentilin; citation_author=A Marcello; citation_author=M Giacca; citation_journal_title=J Virol; citation_volume=75; citation_number=19; citation_pages=12279-12287; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Protein building blocks preserved by recombination.; citation_author=CA Voigt; citation_author=C Martinez; citation_author=ZG Wang; citation_author=SL Mayo; citation_author=FH Arnold; citation_journal_title=Nat Struct Biol; citation_volume=9; citation_number=20; citation_pages=553-558; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Successful production of pseudotyped rAAV vectors using a modified baculovirus expression system.; citation_author=E Kohlbrenner; citation_author=G Aslanidi; citation_author=K Nash; citation_author=S Shklyaev; citation_author=M Campbell-Thompson; citation_journal_title=Mol Ther; citation_volume=12; citation_number=21; citation_pages=1217-1225; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Intron positions correlate with module boundaries in ancient proteins.; citation_author=SJ de Souza; citation_author=M Long; citation_author=L Schoenbach; citation_author=SW Roy; citation_author=W Gilbert; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=93; citation_number=22; citation_pages=14632-14636; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Origin of genes.; citation_author=W Gilbert; citation_author=SJ de Souza; citation_author=M Long; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=94; citation_number=23; citation_pages=7698-7703; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Update on the prevalence of serum antibodies (IgG and IgM) to adeno-associated virus (AAV).; citation_author=K Erles; citation_author=P Sebökovà; citation_author=J Schlehofer; citation_journal_title=Journal of Medical Virology; citation_volume=59; citation_number=24; citation_pages=406-411; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=CLUSTAL W: improving the sensitivity of progressive multiple sequence alignment through sequence weighting, position-specific gap penalties and weight matrix choice.; citation_author=JD Thompson; citation_author=DG Higgins; citation_author=TJ Gibson; citation_journal_title=Nucleic Acids Res; citation_volume=22; citation_number=25; citation_pages=4673-4680; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=A simple, fast, and accurate algorithm to estimate large phylogenies by maximum likelihood.; citation_author=S Guindon; citation_author=O Gascuel; citation_journal_title=Syst Biol; citation_volume=52; citation_number=26; citation_pages=696-704; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=MODELTEST: testing the model of DNA substitution.; citation_author=D Posada; citation_author=KA Crandall; citation_journal_title=Bioinformatics; citation_volume=14; citation_number=27; citation_pages=817-818; citation_date=1998; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001634" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Recombination and Population Mosaic of a Multifunctional Viral Gene, Adeno-Associated Virus cap"/>
    <meta name="twitter:description" content="Homologous recombination is a dominant force in evolution and results in genetic mosaics. To detect evidence of recombination events and assess the biological significance of genetic mosaics, genome sequences for various viral populations of reasonably large size are now available in the GenBank. We studied a multi-functional viral gene, the adeno-associated virus (AAV) cap gene, which codes for three capsid proteins, VP1, VP2 and VP3. VP1-3 share a common C-terminal domain corresponding to VP3, which forms the viral core structure, while the VP1 unique N-terminal part contains an enzymatic domain with phospholipase A2 activity. Our recombinant detection program (RecI) revealed five novel recombination events, four of which have their cross-over points in the N-terminal, VP1 and VP2 unique region. Comparison of phylogenetic trees for different cap gene regions confirmed discordant phylogenies for the recombinant sequences. Furthermore, differences in the phylogenetic tree structures for the VP1 unique (VP1u) region and the rest of cap highlighted the mosaic nature of cap gene in the AAV population: two dominant forms of VP1u sequences were identified and these forms are linked to diverse sequences in the rest of cap gene. This observation together with the finding of frequent recombination in the VP1 and 2 unique regions suggests that this region is a recombination hot spot. Recombination events in this region preserve protein blocks of distinctive functions and contribute to convergence in VP1u and divergence of the rest of cap. Additionally the possible biological significance of two dominant VP1u forms is inferred."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0001634.g003"/>

  <meta property="og:title" content="Recombination and Population Mosaic of a Multifunctional Viral Gene, Adeno-Associated Virus cap" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001634" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=7506'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=4215'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=8372&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0001634">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001634" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001634&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Recombination and Population Mosaic of a Multifunctional Viral Gene, Adeno-Associated Virus <i>cap</i>
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Yasuhiro Takeuchi
              <span class="corresponding">mail</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p>*E-mail: <a href="mailto:y.takeuchi@ucl.ac.uk">y.takeuchi@ucl.ac.uk</a></p>

                <p>Affiliation:
                  Medical Reserach Council (MRC)/UCL Centre for Medical Molecular Virology, Division of Infection and Immunology, University College London, London, United Kingdom
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Richard Myers, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliations:
                  Medical Reserach Council (MRC)/UCL Centre for Medical Molecular Virology, Division of Infection and Immunology, University College London, London, United Kingdom, 
                  Centre for Infections, Health Protection Agency, London, United Kingdom
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Olivier Danos
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliations:
                  Medical Reserach Council (MRC)/UCL Centre for Medical Molecular Virology, Division of Infection and Immunology, University College London, London, United Kingdom, 
                  Institut National de la Santé et de la Recherche Médicale (INSERM), U781, Institut Fédératif de Recherche Necker Enfants Malades (IFR 94), Hôpital Necker Enfants Malades, Paris, France
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: February 20, 2008</li>
    <li>DOI: 10.1371/journal.pone.0001634</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0001634" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0001634" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001634" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0001634" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0001634-g001" data-doi="info:doi/10.1371/journal.pone.0001634" data-uri="info:doi/10.1371/journal.pone.0001634.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001634.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001634-g002" data-doi="info:doi/10.1371/journal.pone.0001634" data-uri="info:doi/10.1371/journal.pone.0001634.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001634.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001634-g003" data-doi="info:doi/10.1371/journal.pone.0001634" data-uri="info:doi/10.1371/journal.pone.0001634.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001634.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001634-t001" data-doi="info:doi/10.1371/journal.pone.0001634" data-uri="info:doi/10.1371/journal.pone.0001634.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001634.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001634">Reader Comments (0)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0001634" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1"></a><p>Homologous recombination is a dominant force in evolution and results in genetic mosaics. To detect evidence of recombination events and assess the biological significance of genetic mosaics, genome sequences for various viral populations of reasonably large size are now available in the GenBank. We studied a multi-functional viral gene, the adeno-associated virus (AAV) <em>cap</em> gene, which codes for three capsid proteins, VP1, VP2 and VP3. VP1-3 share a common C-terminal domain corresponding to VP3, which forms the viral core structure, while the VP1 unique N-terminal part contains an enzymatic domain with phospholipase A2 activity. Our recombinant detection program (RecI) revealed five novel recombination events, four of which have their cross-over points in the N-terminal, VP1 and VP2 unique region. Comparison of phylogenetic trees for different <em>cap</em> gene regions confirmed discordant phylogenies for the recombinant sequences. Furthermore, differences in the phylogenetic tree structures for the VP1 unique (VP1u) region and the rest of <em>cap</em> highlighted the mosaic nature of <em>cap</em> gene in the AAV population: two dominant forms of VP1u sequences were identified and these forms are linked to diverse sequences in the rest of <em>cap</em> gene. This observation together with the finding of frequent recombination in the VP1 and 2 unique regions suggests that this region is a recombination hot spot. Recombination events in this region preserve protein blocks of distinctive functions and contribute to convergence in VP1u and divergence of the rest of <em>cap</em>. Additionally the possible biological significance of two dominant VP1u forms is inferred.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>Takeuchi Y, Myers R, Danos O (2008) Recombination and Population Mosaic of a Multifunctional Viral Gene, Adeno-Associated Virus <em>cap</em>. PLoS ONE 3(2):
          e1634.
            doi:10.1371/journal.pone.0001634</p><p><strong>Academic Editor: </strong>Jean Carr, Institute of Human Virology, United States of America</p><p><strong>Received:</strong> November 30, 2007; <strong>Accepted:</strong> January 25, 2008; <strong>Published:</strong> February 20, 2008</p><p><strong>Copyright:</strong> © 2008 Takeuchi et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>This work was funded by UCL Clinical Research and Development Committee. This sponsor played no role in the design and conduct of the study, in the collection, analysis, and interpretation of the data, and in the preparation, review, or approval of the manuscript.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>Parvoviruses are eukaryotic viruses that infect a wide variety of hosts from insects to human and non-human primates. They are composed of a small icosahedral capsid, 22 to 24 nm in diameter into which is packaged a single stranded DNA genome of less than 5.6 kilobases with terminal palindromic sequences. In recent years, studies of the carnivore parvovirus subgroup have allowed the observation of cross species transmission between cat and dog populations. This host range shift was due to a high mutation rate in the capsid protein gene that enabled rapid adaptation to a new cell surface receptor <a href="#pone.0001634-Hueffer1">[1]</a>, <a href="#pone.0001634-Shackelton1">[2]</a>. Host range variation has also been studied among Adeno-Associated Viruses (AAV), which are helper-dependent Parvoviruses (dependoviruses) with great potential as gene transfer vectors <a href="#pone.0001634-Wu1">[3]</a>. The capsid of AAV is formed by three proteins, VP1, VP2 and VP3 which are encoded by the same <em>cap</em> gene and differ in their N-terminal domain (<a href="#pone-0001634-g001">Fig 1A</a>). The common C-terminal domain corresponds to VP3 and folds into capsomer units that display a typical beta-barrel structure <a href="#pone.0001634-Xie1">[4]</a> with solvent exposed loops of variable amino-acids sequences. These variable regions of VP3 form the structural elements that recognize cell surface receptors. About one tenth of the capsomers are VP1 or VP2 and display additional N-terminal domains.</p>
<div class="figure" id="pone-0001634-g001"><div class="img"><a name="pone-0001634-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001634.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001634" data-uri="info:doi/10.1371/journal.pone.0001634.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001634.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001634.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001634.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001634.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001634.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001634.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001634.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001634.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001634.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Profiles and cross-over points of novel recombination in the AAV cap gene.</span></strong></p><a id="article1.body1.sec1.fig1.caption1.p1" name="article1.body1.sec1.fig1.caption1.p1"></a><p>A. Schematic drawing of the <em>cap</em> gene products (VP1-3) with five recombination cross-over points inferred by LOHA analyses below (triangles colour-coordinated with Panels B-F) is shown. Viral phospholipase A and antigenic GH loop domains are indicated. B–F. Recombination profiles for representative combinations of 3 sequences (recombinant/parent 1×parent 2) were visualised by the LOHA analysis. Percent mismatch across a sliding window of 41 Nt positions between recombinant/parent pairs were plotted against the nucleotide positions along the multiple alignment with a step interval of 1 position. The x-axis also indicates three regions (R1-3) separately analysed in the following studies (see <a href="#pone-0001634-g002">Fig 2</a> and <a href="#pone-0001634-g003">3</a>).</p>
<span>doi:10.1371/journal.pone.0001634.g001</span></div><a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>Beyond receptor recognition, AAV capsid proteins have been involved in the complex processes of viral entry and trafficking to the nucleus. The additional domains of VP1 and VP2 are buried inside the capsid in the native virions and become surface exposed during trafficking through the endosomal system <a href="#pone.0001634-Grieger1">[5]</a>–<a href="#pone.0001634-Sonntag1">[7]</a>. This conformational change reveals basic amino acids which have been found to enhance nuclear targeting <a href="#pone.0001634-Sonntag1">[7]</a>, <a href="#pone.0001634-Grieger2">[8]</a>. It also activates a Phospholipase A2 (PLA2) which is encoded by the VP1 unique region (VP1u) and is conserved in all parvoviruses <a href="#pone.0001634-Zadori1">[9]</a>. PLA2 activity is essential for endosomal escape and entry of the viral DNA into the nucleus <a href="#pone.0001634-Sonntag1">[7]</a>.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>Over one hundred different AAV isolates have been characterized in humans and non-human primates. They display a high degree of sequence variation in the <em>cap</em> gene regions encoding exposed loops and the receptor binding domain <a href="#pone.0001634-Gao1">[10]</a>. This variability allows AAV to escape the host immune responses and to adapt to different receptors. Phylogenetic studies of the cap sequences indicate that the numerous AAV isolates have probably evolved from a single parent of avian origin and are now distributed in clades, A-F <a href="#pone.0001634-Gao1">[10]</a>. Initial sequence comparisons have suggested that certain isolates could be the result of recombination between divergent parental genomes: for example Clade C. a predominantly human clade, has been proposed to be a hybrid of two AAV serotypes, AAV2 (Clade B) and AAV3 <a href="#pone.0001634-Gao1">[10]</a>–<a href="#pone.0001634-Xiao1">[13]</a>. It is therefore possible that the structurally and functionally distinct regions of the cap gene have different evolutionary histories. In this study we reveal the mosaic genetic nature of the <em>cap</em> gene in individual virus isolates as well as in the AAV population for which sequence data are available. We discuss the relationship between such genetic mosaicism and the <em>cap</em> gene functions, as well as the implications for AAV biology and epidemiology.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3><a id="article1.body1.sec2.p1" name="article1.body1.sec2.p1"></a><p>Available AAV <em>cap</em> nucleotide sequences, including those of human, mammalian, avian and reptilian origins, were retrieved from Genbank and aligned (n = 156). This alignment was used for recombinant sequence screening (<a href="#pone-0001634-g001">Fig 1</a>) and generation of phylogenetic trees for different regions of the <em>cap</em> gene (<a href="#pone-0001634-g002">Fig 2</a>) as described in <a href="#s4">Materials and Methods</a>. Representative recombinant and parent relationships for the five recombination events were analysed using LOHA <a href="#pone.0001634-Bartosch1">[14]</a> (<a href="#pone-0001634-g001">Fig 1</a>, Panels B–F). For example, Panel B shows pair-wise comparisons generated by plotting the percentage of mismatches between hu 31 and AAV2 (blue) and between hu 31 and AAV9 (pink) (Y axis) for 41 base pair (bp) sliding windows along the alignment (X-axis). The first 150 bp of hu 31 are almost identical to those of AAV2 and different from AAV9, while this relationship is reversed in the rest of the genome which is nearly identical to AAV9 and different from AAV2. We infer from this result that hu 31 is a recombinant containing AAV2 and AAV9 like sequences in mosaic. While Panels B, C and E show similar mosaic patterns in <em>cap</em>, involving two different parent viruses, panels D and F show a more complex picture, with segments of low sequence homology for which no close parent can be identified. Three other sequences, rh 32–34, were closely related to AAV11 and showed similar LOHA profiles, while hu 31 and hu 44 shared the recombinant features of hu32 and hu46 respectively (data not shown, see <a href="#pone-0001634-g002">Fig 2</a>). It is likely that these sequences with the same recombination patterns have resulted from single recombination events. Recombination screening detected five recombination cross-over points the positions of which are indicated on the cap gene in <a href="#pone-0001634-g001">Fig 1A</a>. Our recombination screening also detected the previously reported recombination events observed in a single rhesus macaque (Tulane/F953) isolate <a href="#pone.0001634-Gao2">[12]</a>. Unlike the Tulane/F953 derived sequences, some of these novel recombinants, if not all, are likely to represent established isolates/strains and represent recombination events that resulted in viable, naturally selected recombinant forms of AAV. Interestingly, the five cross over points are not evenly distributed along the cap gene, as four of them occur in the N-terminus proximal part of cap that encode unique VP1 and VP2 sequences, suggesting that this part may be a region of frequent recombination.</p>
<div class="figure" id="pone-0001634-g002"><div class="img"><a name="pone-0001634-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001634.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001634" data-uri="info:doi/10.1371/journal.pone.0001634.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001634.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001634.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001634.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001634.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001634.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001634.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001634.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001634.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001634.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Local phylogenetic trees between 53 AAV cap sequences.</span></strong></p><a id="article1.body1.sec2.fig1.caption1.p1" name="article1.body1.sec2.fig1.caption1.p1"></a><p>Neighbour-joining trees were produced based on the nucleotide alignment for 3 regions: R1, VP1 unique; R2, between VP2 and VP3 N-termini; R3, VP3 coding region shared by all VP 1-3 (see <a href="#pone-0001634-g001">Fig 1</a>). Phylogenetic discordance between the three regions was observed for recombinant sequences. Tree structure difference between regions, particularly R1 versus R3, demonstrates the mosaic nature of the <em>cap</em> gene. Fifty-three sequences were selected by excluding sequences of non-mammalian origin (except an avian AAV used as the root), sequences derived from Tulane/F953 monkey and a number of ‘redundant’ sequences as indicated in <a href="#pone.0001634.s001">Table S1</a>. Bootstrap scores greater than 90 % are shown. Recombinant sequences depicted in <a href="#pone-0001634-g001">Fig 1B–F</a> are colour-boxed. A branch length scale common for all three trees is shown.</p>
<span>doi:10.1371/journal.pone.0001634.g002</span></div><a id="article1.body1.sec2.p2" name="article1.body1.sec2.p2"></a><p>The generation of local phylogenetic trees confirmed the finding of the recombinant and mosaic nature of the <em>cap</em> gene. <a href="#pone-0001634-g002">Figure 2</a> shows local trees for the alignments of selected sequences in VP1 unique region (R1), the region between VP2 and VP3 initiation (R2) and the C-terminal region shared by all (R3). Sequences of non-mammalian origin, Tulane/F953 derived sequences and a number of ‘redundant’ sequences of human origin were excluded in this analysis. Phylogenetic discordance between the three regions was observed, such that recombinant sequences belonged to one cluster in a tree but moved to another cluster separated from the former cluster with high probability (high bootstrap score) in another tree. Such sequence jumps between branches are demonstrated for sequences, hu 44 and hu 46 (red in R1 and R2 trees), AAV11 and rh 32–34 (blue in R1 and R2), and hu 43 (pink in R2 and R3). The association of hu31 and hu32 (green) with AAV9 is evident in R2 and R3, but not R1, while the phylogenetic discordance of rh 60 (yellow) within R3 is not obvious but reflected by its long terminal branch in R3 tree (<a href="#pone-0001634-g002">Fig 2</a>).</p>
<a id="article1.body1.sec2.p3" name="article1.body1.sec2.p3"></a><p><a href="#pone-0001634-g002">Figure 2</a> also shows a striking tree structure difference between regions, particularly R1 versus R3, demonstrating the mosaic nature of the <em>cap</em> gene within these AAV sequences. The R1 tree, corresponding to the VP1 unique/PLA coding region has few long branches with tightly populated clusters, whereas the R3 tree corresponding to the capsid structure has many more long branches with less populated clusters. This observation can be interpreted simply as a consequence of different levels of conservation between these regions; R1 is a homologous, conserved region and R3 a variable antigenic region. However the conservation of R1 is not uniform within the population, as sequences separated by long branches appear to be substantially divergent in both R1 and R3: for example the mismatch between AAV1 and AAV2 in R1 and R3 is 16.8% and 22.5%, respectively. It is possible that R1 can take only few divergent forms due to functional restrictions, while varying forms of R3 are tolerated and/or necessary.</p>
<a id="article1.body1.sec2.p4" name="article1.body1.sec2.p4"></a><p>To further define this regional difference in the phylogenetic relationship within the AAV population, we analysed a data set selected by excluding apparent recombinant sequences and redundant sequences. For this we selected one representative of each cluster of sequences with more than 98.5 % homology over the full <em>cap</em> gene. Four sequences, AAV3, AAV4, AAV9 and ch 5, were included as outsiders in this analysis. <a href="#pone-0001634-g003">Figure 3A and B</a> shows the resulting trees for R1 and the full <em>cap</em> gene respectively. In these trees, it is apparent that there are two dominant forms of R1. Sequences containing these two dominant forms were classified into Group 1 (G1) and 2 (G2), represented by AAV1 and AAV2, respectively (<a href="#pone-0001634-g003">Fig 3A, G</a>1 of 15 members and G2 of 10 members). R1 sequences in both G1 and G2 form tight clusters with high bootstrap values and for the most part low intra-cluster branch-lengths (less than 0.02 substitutions) (<a href="#pone-0001634-g003">Fig 3A</a>). The cluster structure of G1 was not absolutely conserved due to variation of the placement of sequences cy 6 and pi 1 using both maximal likelihood (ML) and neighbour joining (NJ) methodologies. The outlying sequence (ch 5) was shown to cluster robustly (bootstrap values 100, ML and NJ) with G2 sequences but with a far greater branch length than other members of this cluster. ML and NJ analysis of the full cap gene illustrated that the sequences within G1 and G2 remain clustered, but show substantial diversity depicted by long intra-cluster branches (<a href="#pone-0001634-g003">Fig 3B</a>).</p>
<div class="figure" id="pone-0001634-g003"><div class="img"><a name="pone-0001634-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001634.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001634" data-uri="info:doi/10.1371/journal.pone.0001634.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001634.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001634.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001634.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001634.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001634.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001634.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001634.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001634.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001634.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Identification of two sequence groups (G1 and G2) tightly clustered in the VP1 unique region.</span></strong></p><a id="article1.body1.sec2.fig2.caption1.p1" name="article1.body1.sec2.fig2.caption1.p1"></a><p>A. A Phyml maximum likelihood (ML) tree for R1 region was generated for a dataset of 29 sequences (see selection in <a href="#pone.0001634.s001">Table S1</a>). AAV4 was used as the outgroup and the bootstrap scores greater than 80 % are shown. Tightly clustered sequence groups, G1 of 15 sequences and G2 of 10 sequences, were identified and indicated. A similar tree was obtained using neighbour-joining method and its bootstrap scores that support G1 and G2 clustering are shown in parentheses. B. An ML tree for the full-length cap was obtained and shown in the same way as the R1 tree (Panel A). C. Average group LOHA profiles based on the nucleotide alignment are shown. Percent mismatch scores across a sliding window of 101 Nt positions for all possible pairs between all 29 sequences (Control), 15 G1 sequences (G1) and 10 G2 sequences (G2) were averaged and plotted against the nucleotide positions. D. Similar average LOHA profiles with 33 a.a. window size were obtained based on the amino acid sequence alignment.</p>
<span>doi:10.1371/journal.pone.0001634.g003</span></div><a id="article1.body1.sec2.p5" name="article1.body1.sec2.p5"></a><p>To reveal where the members of G1 and G2 are homologous within the group and where they divert, pair-wise LOHA analyses were performed between 29 selected sequences (total 406 possible pairs). <a href="#pone-0001634-g003">Figure 3C</a> shows the averaged LOHA plots (101 bp sliding window) for the DNA alignment of all 406 sequence pairs, for 105 sequence pairs of G1 and 45 G2 sequence pairs. The 406 sequence pair average was included as a control indicating the diversity within the 29 sequence population along the alignment. While this graph shows the greatest variability in the middle part of VP3 (ca. 1300–1900 Nts), all 101 bp windows including VP1 unique region (R1, 1–415 Nts) have more than 5% mismatches in the average DNA LOHA for 406 sequence pair (<a href="#pone-0001634-g003">Fig 3C</a>). In contrast, the average for 105 G1 pairs showed low scores of less than 5 % at 1–403 positions, while the rest of the graph possesses a similar pattern to the control, with a sharp increase around the position 400. Because the window at position 400 covers bp positions that extend to bp 450, the region represented by 1–400 in this analysis roughly corresponds to R1. The ratio of these two plots (G1/Control) scored less than 30 % at positions 4–390, but most positions in the rest were 40–100 % (data not shown), demonstrating the peculiar, mosaic diversity pattern of the G1 sequences. The average for G2 pairs also shows a similar pattern to G1 average, as positions ca. 1–400 have low scores with an increase downstream, while this population has another region of low scores (bp 900–1200).</p>
<a id="article1.body1.sec2.p6" name="article1.body1.sec2.p6"></a><p>Average plots were also obtained for protein alignments of 29 predicted Cap amino acid sequences, using a 33 a.a. window (<a href="#pone-0001634-g003">Fig. 3D</a>). A pattern similar to the one described above for DNA LOHA graphs was observed. These results indicate that there are two dominant forms (G1 and G2) of VP1 unique domains which are structurally distinct from each other and different from those found in the outsider AAV isolates. This indicates that the genetic mosaic feature is indeed phenotypically expressed at the protein level.</p>
<a id="article1.body1.sec2.p7" name="article1.body1.sec2.p7"></a><p>Following the identification of two dominant forms of VP1 unique (R1) region, we searched for potential distinction of biological properties between G1 and G2 members. It was noted that G2 members analysed in <a href="#pone-0001634-g003">Fig 3</a> are all of human origin, whilst G1 included isolates of both human and non-human primate (NHP) origin. Since this suggested that G1 and G2 might have different host ranges, the sample origin of all available primate AAV sequences were verified in their original reports. These sequences were classified into G1 (clustered with AAV1), G2 (clustered with AAV2) and others by making a phylogenetic tree for R1 (not shown) and summarised in <a href="#pone-0001634-t001">Table 1</a>. The majority of human AAV (50 sequences) belong to either AAV2-like clade B or AAV2/3 hybrid clade C which have been reported to be frequently found in the human population <a href="#pone.0001634-Gao1">[10]</a>, <a href="#pone.0001634-Chen1">[11]</a>. These sequences have the G2 form of R1 (<a href="#pone-0001634-t001">Table 1</a>). In contrast the majority of NHP AAV contain the G1 form of R1 and no NHP AAV has the G2 form (<a href="#pone-0001634-t001">Table 1</a>). There are a substantial number (11 sequences) of human AAV with the G1 form (<a href="#pone-0001634-t001">Table 1</a>). These sequences clustered together with NHP AAV sequences in a mixed fashion along the whole cap gene (see <a href="#pone-0001634-g002">Fig 2</a> and Fig3B). These results suggest that the G1 form of R1 is widely prevalent in primates and that G1 members of human origin may represent multiple cases of recent cross-species infection. Among the sequences classified as others, four sequences, hu 31, hu 32, hu 44 and hu 46, are apparent recombinant sequences identified in this study (<a href="#pone-0001634-t001">Table 1</a>). It is noteworthy that these sequences are of human origin and appear to have adopted G2 like forms of R1 (<a href="#pone-0001634-g001">Fig 1B, 1C</a>, <a href="#pone-0001634-g002">Fig 2</a> R1 tree).</p>
<div class="figure" id="pone-0001634-t001"><div class="img"><a name="pone-0001634-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001634.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001634" data-uri="info:doi/10.1371/journal.pone.0001634.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001634.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001634.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001634.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001634.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001634.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001634.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001634.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001634.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001634.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>Host distribution of G1/G2 forms.</span></strong></p><span>doi:10.1371/journal.pone.0001634.t001</span></div></div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p>Our screening for recombinant sequences within 156 AAV <em>cap</em> sequences newly identified evidence for five recombination events. These events appear to be relatively recent recombination where the recombinant sequence and at least one parent sequence are closely related (<a href="#pone-0001634-g002">Fig 2</a>). This result, along with previous reports on AAV recombination <a href="#pone.0001634-Gao1">[10]</a>–<a href="#pone.0001634-Xiao1">[13]</a> suggests that homologous recombination of AAV genomes occurs frequently. While the molecular mechanism of AAV recombination is still unclear, AAV has a number of properties that may facilitate recombination. In the presence of helper virus or upon genotoxic stress, the genome is efficiently converted into double-stranded DNA and amplified at high copy numbers <a href="#pone.0001634-Yalkinoglu1">[15]</a>, <a href="#pone.0001634-Weitzman1">[16]</a>. In addition, the inverted terminal repeats have a recombinogenic structure that can potentially attract cellular recombination and repair enzymes to the AAV genome <a href="#pone.0001634-Raj1">[17]</a>–<a href="#pone.0001634-Zentilin1">[19]</a>. Four out of the five crossover points identified here fall in and near the VP1 unique sequence, suggesting that recombination occurs frequently in this region. It is therefore likely that recombination has contributed to the mosaic structure of the <em>cap</em> gene within the AAV population in cases where the VP1 unique N-terminal segment and the C-terminal domain have different phylogenetic origins.</p>
<a id="article1.body1.sec3.p2" name="article1.body1.sec3.p2"></a><p>Two dominant forms of VP1 unique sequence were identified, whilst the remainder of the Cap protein has many diverse forms. The dominant AAV1-like form of VP1 unique region is prevalent in a wide range of primate AAV and linked to various forms of VP2/3 coding region (G1 sequences). The other dominant form represented by AAV2 is found in two previously reported lineages of human AAV, AAV2-like clade B represented by AAV2 and clade C represented by AAV2/3 hybrid isolates <a href="#pone.0001634-Gao1">[10]</a>, <a href="#pone.0001634-Chen1">[11]</a>. The VP1 unique region encodes a phospholipase domain <a href="#pone.0001634-Sonntag1">[7]</a>, which is enzymatically and structurally distinct from the VP3 capsomer unit. Our results suggest that dominant VP1 unique forms are restricted by enzymatic function and have been slowly evolving in their hosts. These forms could have become prevalent by convergent mutations and recombination purging less fit VP1 unique sequences. For example AAV11 and rh 32–34 sequences might have lost an unknown form of VP1 unique sequence by adopting AAV1 (also AAV10)-like form by recombination (<a href="#pone-0001634-g001">Fig 1D</a> and <a href="#pone-0001634-g002">2</a>, R1 tree). In this context, frequent recombination around the VP2 initiation site (R1 and R2 junction) like those found in this study can contribute to the rise of dominant forms of VP1 unique region while enhancing diversification of the capsid structure domain. In other words, frequent, localised recombination within the <em>cap</em> gene may have resulted in conservation of one part and divergence in the other part, and in a genetic mosaic.</p>
<a id="article1.body1.sec3.p3" name="article1.body1.sec3.p3"></a><p>The idea that the dominant forms of VP1 unique/PLA domain have risen partly by recombination is consistent with the theory that protein building blocks are preserved by recombination <a href="#pone.0001634-Voigt1">[20]</a>. The observed mosaicism within the AAV <em>cap</em> sequences suggests that recombination occurs around the junction of VP1 unique region more easily than in the middle of the <em>cap</em> domains. Indeed, artificial swapping of this region between AAV2 and AAV8 leads to a functional protein <a href="#pone.0001634-Kohlbrenner1">[21]</a>. We therefore speculate that this junction may well be a recombination hot spot, representing a naturally occurring example of preservation of protein building blocks <a href="#pone.0001634-Voigt1">[20]</a>. This will be interesting to consider in the scope of general evolutionary issues including the exon swapping theory <a href="#pone.0001634-Voigt1">[20]</a>, <a href="#pone.0001634-deSouza1">[22]</a>, <a href="#pone.0001634-Gilbert1">[23]</a>. Identification and swapping of a domain/region in artificial recombinant proteins may be useful for designing AAV vector systems. It will be interesting to study the behaviour of chimeric AAV <em>cap</em> genes where either of the two dominant forms of the VP1 unique domain are combined with various forms of capsid structural domains, in the context of both gene transfer vector and replicative infection systems.</p>
<a id="article1.body1.sec3.p4" name="article1.body1.sec3.p4"></a><p>The biological significance for the presence of two distinctive, dominant forms of VP1 unique sequence is unknown. One possibility is that this region affects AAV host range, as the AAV2-like form has been found exclusively in sequences derived from human samples whereas AAV1-like form has been found in both human and NHP samples (<a href="#pone-0001634-t001">Table 1</a>). These data however require cautious interpretation, because the sample number and range is limited. Notably most human isolates were characterized from samples collected in the Eastern part of the United States <a href="#pone.0001634-Gao1">[10]</a>, <a href="#pone.0001634-Chen1">[11]</a>, and therefore originate from a relatively small geographic area. However, the AAV2 epidemic in the human population may be global, as suggested by serological studies where it was found to be the prevalent type <a href="#pone.0001634-Erles1">[24]</a> and may be explained by a founder effect. Our finding that two lines of recombinant <em>cap</em> gene hu 31/32 and hu 44/46 have acquired AAV2 like VP1 unique sequence (<a href="#pone-0001634-g001">Fig 1</a> and <a href="#pone-0001634-g002">2</a>) may reflect a proliferation of this dominant form in human AAV population. It is also noteworthy that among NHP AAV sequences, a single sequence derived from chimpanzee, the ape species closest to humans has a VP1 unique sequence close to the AAV2-like form (<a href="#pone-0001634-g002">Fig 2</a> R1 tree ch 5 sequence). Further investigation is warranted to test the possibility that this region affects the AAV host range. It is also possible that the distinctive VP1 unique forms coevolved with the upstream gene <em>rep</em>. Further sequencing of a wider range of AAV samples, ideally the whole genome including <em>rep</em> and host range studies on artificial AAV recombinants with the two dominant forms will test this hypothesis.</p>
</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4>Retrieval and alignment of sequences</h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p>The capsid sequence of 156 AAV isolates derived from human, non-human primate and non-primate species were downloaded from GenBank (<a href="http://www.ncbi.nlm.nih.gov/">http://www.ncbi.nlm.nih.gov/</a>). These sequences listed in <a href="#pone.0001634.s001"><strong>Table S1</strong></a> were aligned using ClustalW <a href="#pone.0001634-Thompson1">[25]</a> and manually curated to improve the quality of the alignment.</p>


<h4>Recombination detection and analyses</h4>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1"></a><p>Recombination detection was performed initially on the entire 156 sequence dataset using a high throughput recombination detection method developed in-house. The Recombination Index (RecI) program used all possible triplets of sequences (<strong><em>x, y, z</em></strong>), determined the likelihood that one sequence was recombinant and that it was parented by the other two. The Rec program identified recombinants by scoring the percentage mismatch between sequence pairs using a sliding window. The question of whether sequence <strong><em>x</em></strong> was a recombinant between sequences <strong><em>y</em></strong> and <strong><em>z</em></strong> was answered by comparing <strong><em>x</em></strong> vs <strong><em>y</em></strong> and <strong><em>x</em></strong> vs <strong><em>z</em></strong>. If sequence <strong><em>x</em></strong> was a recombinant, then sequence <strong><em>x</em></strong> would contain at least one region more like sequence <strong><em>y</em></strong> than sequence <strong><em>z</em></strong> and another region more like sequence <strong><em>z</em></strong> than sequence <strong><em>y</em></strong>. These relationships were used to produce indices for all possible combinations of 3 sequences (recombinant/parents) from a multiple sequence data set. The details of the algorithm and the program executable are available on request.</p>
<a id="article1.body1.sec4.sec2.p2" name="article1.body1.sec4.sec2.p2"></a><p>The program LOHA <a href="#pone.0001634-Bartosch1">[14]</a> visualised conservation or divergence within and between sequence groups and was used to analyse potential recombination events indicated by the Rec analysis. LOHA was generated in-house and used a multiple alignment of sequences as its input. LOHA compared each of the possible pairs of aligned sequences with gaps and calculated the average percent mismatch across a sliding window of defined size (typically 41 Nt positions) and a step interval of 1 position. Positions with gaps for both sequences were counted as matched, while positions with a gap for only one sequence were counted as mismatched. The percent mismatch between a pair of sequences was plotted against the nucleotide positions using Microsoft Excel. This allowed regions of similarity or divergence to be visualised. Comparison of 3 LOHA plots (<strong><em>x</em></strong> vs <strong><em>y</em></strong>, <strong><em>x</em></strong> vs <strong><em>z</em></strong>, <strong><em>y</em></strong> vs <strong><em>z)</em></strong> allowed possible recombination events between two parents and a progeny sequence to be determined. The output from a LOHA analysis of a multiple alignment was also grouped such that the average of the mismatch within a LOHA window was generated using all pairs of sequences input into the program. This output generated a single score per sliding window along the length of the alignment that was used to determine the level of nucleotide variation within a group of sequences (average group LOHA), as opposed to pairwise comparisons within the alignment (LOHA).</p>


<h4>Phylogenetic analyses</h4>
<a id="article1.body1.sec4.sec3.p1" name="article1.body1.sec4.sec3.p1"></a><p>Phylogenetic analysis of AAV capsid sequences was performed to identify the evolutionary relationships between AAV strains, to identify clusters of highly related and therefore redundant sequences, and to verify the recombination events detected by the RecI and LOHA programs. Two methods of phylogenetic analysis were performed. The initial capsid phylogenies were generated using a bootstrapped (1000 replicates) neighbour-joining methodology (ClustalW). Trees for selected sequences were also generated using bootstrapped (1000 replicates) Phyml maximum likelihood trees <a href="#pone.0001634-Guindon1">[26]</a> following estimation of an evolutionary model using Modeltest (Paup) <a href="#pone.0001634-Posada1">[27]</a> to confirm that this algorithm would generate similar phylogenies to those produced by a simpler evolutionary analysis. Bootstrapped neighbour-joining phylogenies were also performed on the partial alignments used to analyse recombinant breakpoints.</p>

</div>

<div id="section5" class="section"><a id="s5" name="s5" toc="s5" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0001634"></div><a name="pone.0001634.s001" id="pone.0001634.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001634.s001">Table S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1"></a><p class="preSiDOI">List of 156 AAV cap gene sequences. A table to list up AAV cap gene sequenced used.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001634.s001</p><a id="article1.body1.sec5.supplementary-material1.caption1.p2" name="article1.body1.sec5.supplementary-material1.caption1.p2"></a><p class="postSiDOI">(0.15 MB DOC)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>We thank Robin Weiss, Stephan Hue and William Saurin for critical reading of this manuscript.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: YT OD. Performed the experiments: YT RM. Analyzed the data: YT RM. Contributed reagents/materials/analysis tools: RM. Wrote the paper: YT RM OD.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0001634-Hueffer1" id="pone.0001634-Hueffer1"></a>Hueffer K, Parrish CR (2003) Parvovirus host range, cell tropism and evolution. Current Opinion in Microbiology  6: 392.  <ul class="find" data-citedArticleID="1059208" data-doi="10.1016/s1369-5274(03)00083-3"><li><a href="http://dx.doi.org/10.1016/s1369-5274(03)00083-3" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Parvovirus+host+range%2C+cell+tropism+and+evolution." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Parvovirus+host+range%2C+cell+tropism+and+evolution.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0001634-Shackelton1" id="pone.0001634-Shackelton1"></a>Shackelton LA, Parrish CR, Truyen U, Holmes EC (2005) High rate of viral evolution associated with the emergence of carnivore parvovirus. PNAS  102: 379–384.  <ul class="find" data-citedArticleID="1059218" data-doi="10.1073/pnas.0406765102"><li><a href="http://dx.doi.org/10.1073/pnas.0406765102" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=High+rate+of+viral+evolution+associated+with+the+emergence+of+carnivore+parvovirus." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22High+rate+of+viral+evolution+associated+with+the+emergence+of+carnivore+parvovirus.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0001634-Wu1" id="pone.0001634-Wu1"></a>Wu Z, Asokan A, Samulski RJ (2006) Adeno-associated Virus Serotypes: Vector Toolkit for Human Gene Therapy. Molecular Therapy  14: 316–327.  <ul class="find" data-citedArticleID="1059228" data-doi="10.1016/j.ymthe.2006.05.009"><li><a href="http://dx.doi.org/10.1016/j.ymthe.2006.05.009" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Adeno-associated+Virus+Serotypes%3A+Vector+Toolkit+for+Human+Gene+Therapy." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Adeno-associated+Virus+Serotypes%3A+Vector+Toolkit+for+Human+Gene+Therapy.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0001634-Xie1" id="pone.0001634-Xie1"></a>Xie Q, Bu W, Bhatia S, Hare J, Somasundaram T, et al.  (2002) The atomic structure of adeno-associated virus (AAV-2), a vector for human gene therapy. PNAS  99: 10405–10410.  <ul class="find" data-citedArticleID="1059232" data-doi="10.1073/pnas.162250899"><li><a href="http://dx.doi.org/10.1073/pnas.162250899" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+atomic+structure+of+adeno-associated+virus+%28AAV-2%29%2C+a+vector+for+human+gene+therapy." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+atomic+structure+of+adeno-associated+virus+%28AAV-2%29%2C+a+vector+for+human+gene+therapy.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0001634-Grieger1" id="pone.0001634-Grieger1"></a>Grieger JC, Johnson JS, Gurda-Whitaker B, Agbandje-McKenna M, Samulski RJ (2007) Surface-Exposed Adeno-Associated Virus Vp1-NLS Capsid Fusion Protein Rescues Infectivity of Noninfectious Wild-Type Vp2/Vp3 and Vp3-Only Capsids but Not That of Fivefold Pore Mutant Virions. J Virol  81: 7833–7843.  <ul class="find" data-citedArticleID="1059202" data-doi="10.1128/jvi.00580-07"><li><a href="http://dx.doi.org/10.1128/jvi.00580-07" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Surface-Exposed+Adeno-Associated+Virus+Vp1-NLS+Capsid+Fusion+Protein+Rescues+Infectivity+of+Noninfectious+Wild-Type+Vp2%2FVp3+and+Vp3-Only+Capsids+but+Not+That+of+Fivefold+Pore+Mutant+Virions." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Surface-Exposed+Adeno-Associated+Virus+Vp1-NLS+Capsid+Fusion+Protein+Rescues+Infectivity+of+Noninfectious+Wild-Type+Vp2%2FVp3+and+Vp3-Only+Capsids+but+Not+That+of+Fivefold+Pore+Mutant+Virions.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0001634-Kronenberg1" id="pone.0001634-Kronenberg1"></a>Kronenberg S, Bottcher B, von der Lieth CW, Bleker S, Kleinschmidt JA (2005) A Conformational Change in the Adeno-Associated Virus Type 2 Capsid Leads to the Exposure of Hidden VP1 N Termini. J Virol  79: 5296–5303.  <ul class="find" data-citedArticleID="1059212" data-doi="10.1128/jvi.79.9.5296-5303.2005"><li><a href="http://dx.doi.org/10.1128/jvi.79.9.5296-5303.2005" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+Conformational+Change+in+the+Adeno-Associated+Virus+Type+2+Capsid+Leads+to+the+Exposure+of+Hidden+VP1+N+Termini." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+Conformational+Change+in+the+Adeno-Associated+Virus+Type+2+Capsid+Leads+to+the+Exposure+of+Hidden+VP1+N+Termini.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0001634-Sonntag1" id="pone.0001634-Sonntag1"></a>Sonntag F, Bleker S, Leuchs B, Fischer R, Kleinschmidt JA (2006) Adeno-Associated Virus Type 2 Capsids with Externalized VP1/VP2 Trafficking Domains Are Generated prior to Passage through the Cytoplasm and Are Maintained until Uncoating Occurs in the Nucleus. J Virol  80: 11040–11054.  <ul class="find" data-citedArticleID="1059220" data-doi="10.1128/jvi.01056-06"><li><a href="http://dx.doi.org/10.1128/jvi.01056-06" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Adeno-Associated+Virus+Type+2+Capsids+with+Externalized+VP1%2FVP2+Trafficking+Domains+Are+Generated+prior+to+Passage+through+the+Cytoplasm+and+Are+Maintained+until+Uncoating+Occurs+in+the+Nucleus." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Adeno-Associated+Virus+Type+2+Capsids+with+Externalized+VP1%2FVP2+Trafficking+Domains+Are+Generated+prior+to+Passage+through+the+Cytoplasm+and+Are+Maintained+until+Uncoating+Occurs+in+the+Nucleus.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0001634-Grieger2" id="pone.0001634-Grieger2"></a>Grieger JC, Snowdy S, Samulski RJ (2006) Separate Basic Region Motifs within the Adeno-Associated Virus Capsid Proteins Are Essential for Infectivity and Assembly. J Virol  80: 5199–5210.  <ul class="find" data-citedArticleID="1059204" data-doi="10.1128/jvi.02723-05"><li><a href="http://dx.doi.org/10.1128/jvi.02723-05" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Separate+Basic+Region+Motifs+within+the+Adeno-Associated+Virus+Capsid+Proteins+Are+Essential+for+Infectivity+and+Assembly." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Separate+Basic+Region+Motifs+within+the+Adeno-Associated+Virus+Capsid+Proteins+Are+Essential+for+Infectivity+and+Assembly.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0001634-Zadori1" id="pone.0001634-Zadori1"></a>Zadori Z, Szelei J, Lacoste M-C, Li Y, Gariepy S, et al.  (2001) A Viral Phospholipase A2 Is Required for Parvovirus Infectivity. Developmental Cell  1: 291–302.  <ul class="find" data-citedArticleID="1059238" data-doi="10.1016/s1534-5807(01)00031-4"><li><a href="http://dx.doi.org/10.1016/s1534-5807(01)00031-4" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+Viral+Phospholipase+A2+Is+Required+for+Parvovirus+Infectivity." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+Viral+Phospholipase+A2+Is+Required+for+Parvovirus+Infectivity.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0001634-Gao1" id="pone.0001634-Gao1"></a>Gao G, Vandenberghe LH, Alvira MR, Lu Y, Calcedo R, et al.  (2004) Clades of Adeno-Associated Viruses Are Widely Disseminated in Human Tissues. J Virol  78: 6381–6388.  <ul class="find" data-citedArticleID="1059196" data-doi="10.1128/jvi.78.12.6381-6388.2004"><li><a href="http://dx.doi.org/10.1128/jvi.78.12.6381-6388.2004" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Clades+of+Adeno-Associated+Viruses+Are+Widely+Disseminated+in+Human+Tissues." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Clades+of+Adeno-Associated+Viruses+Are+Widely+Disseminated+in+Human+Tissues.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0001634-Chen1" id="pone.0001634-Chen1"></a>Chen C-L, Jensen RL, Schnepp BC, Connell MJ, Shell R, et al.  (2005) Molecular Characterization of Adeno-Associated Viruses Infecting Children. J Virol  79: 14781–14792.  <ul class="find" data-citedArticleID="1059192" data-doi="10.1128/jvi.79.23.14781-14792.2005"><li><a href="http://dx.doi.org/10.1128/jvi.79.23.14781-14792.2005" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Molecular+Characterization+of+Adeno-Associated+Viruses+Infecting+Children." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Molecular+Characterization+of+Adeno-Associated+Viruses+Infecting+Children.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0001634-Gao2" id="pone.0001634-Gao2"></a>Gao G, Alvira MR, Somanathan S, Lu Y, Vandenberghe LH, et al.  (2003) Adeno-associated viruses undergo substantial evolution in primates during natural infections. PNAS  100: 6081–6086.  <ul class="find" data-citedArticleID="1059198" data-doi="10.1073/pnas.0937739100"><li><a href="http://dx.doi.org/10.1073/pnas.0937739100" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Adeno-associated+viruses+undergo+substantial+evolution+in+primates+during+natural+infections." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Adeno-associated+viruses+undergo+substantial+evolution+in+primates+during+natural+infections.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0001634-Xiao1" id="pone.0001634-Xiao1"></a>Xiao W, Chirmule N, Berta SC, McCullough B, Gao G, et al.  (1999) Gene Therapy Vectors Based on Adeno-Associated Virus Type 1. J Virol  73: 3994–4003.  <ul class="find" data-citedArticleID="1059230"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Gene+Therapy+Vectors+Based+on+Adeno-Associated+Virus+Type+1.&amp;auth=&amp;atitle=Gene+Therapy+Vectors+Based+on+Adeno-Associated+Virus+Type+1." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Gene+Therapy+Vectors+Based+on+Adeno-Associated+Virus+Type+1." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Gene+Therapy+Vectors+Based+on+Adeno-Associated+Virus+Type+1.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0001634-Bartosch1" id="pone.0001634-Bartosch1"></a>Bartosch B, Stefanidis D, Myers R, Weiss R, Patience C, et al.  (2004) Evidence and consequence of porcine endogenous retrovirus recombination. J Virol  78: 13880–13890.  <ul class="find" data-citedArticleID="1059190" data-doi="10.1128/jvi.78.24.13880-13890.2004"><li><a href="http://dx.doi.org/10.1128/jvi.78.24.13880-13890.2004" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evidence+and+consequence+of+porcine+endogenous+retrovirus+recombination." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evidence+and+consequence+of+porcine+endogenous+retrovirus+recombination.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0001634-Yalkinoglu1" id="pone.0001634-Yalkinoglu1"></a>Yalkinoglu AO, Heilbronn R, Burkle A, Schlehofer JR, zur Hausen H (1988) DNA amplification of adeno-associated virus as a response to cellular genotoxic stress. Cancer Res  48: 3123–3129.  <ul class="find" data-citedArticleID="1059234"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=DNA+amplification+of+adeno-associated+virus+as+a+response+to+cellular+genotoxic+stress.&amp;auth=&amp;atitle=DNA+amplification+of+adeno-associated+virus+as+a+response+to+cellular+genotoxic+stress." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=DNA+amplification+of+adeno-associated+virus+as+a+response+to+cellular+genotoxic+stress." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22DNA+amplification+of+adeno-associated+virus+as+a+response+to+cellular+genotoxic+stress.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0001634-Weitzman1" id="pone.0001634-Weitzman1"></a>Weitzman MD, Fisher KJ, Wilson JM (1996) Recruitment of wild-type and recombinant adeno-associated virus into adenovirus replication centers. J Virol  70: 1845–1854.  <ul class="find" data-citedArticleID="1059226"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Recruitment+of+wild-type+and+recombinant+adeno-associated+virus+into+adenovirus+replication+centers.&amp;auth=&amp;atitle=Recruitment+of+wild-type+and+recombinant+adeno-associated+virus+into+adenovirus+replication+centers." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Recruitment+of+wild-type+and+recombinant+adeno-associated+virus+into+adenovirus+replication+centers." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Recruitment+of+wild-type+and+recombinant+adeno-associated+virus+into+adenovirus+replication+centers.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0001634-Raj1" id="pone.0001634-Raj1"></a>Raj K, Ogston P, Beard P (2001) Virus-mediated killing of cells that lack p53 activity. Nature  412: 914.  <ul class="find" data-citedArticleID="1059216" data-doi="10.1038/416202a"><li><a href="http://dx.doi.org/10.1038/416202a" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Virus-mediated+killing+of+cells+that+lack+p53+activity." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Virus-mediated+killing+of+cells+that+lack+p53+activity.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0001634-Yan1" id="pone.0001634-Yan1"></a>Yan Z, Zak R, Zhang Y, Engelhardt JF (2005) Inverted Terminal Repeat Sequences Are Important for Intermolecular Recombination and Circularization of Adeno-Associated Virus Genomes. J Virol  79: 364–379.  <ul class="find" data-citedArticleID="1059236" data-doi="10.1128/jvi.79.1.364-379.2005"><li><a href="http://dx.doi.org/10.1128/jvi.79.1.364-379.2005" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Inverted+Terminal+Repeat+Sequences+Are+Important+for+Intermolecular+Recombination+and+Circularization+of+Adeno-Associated+Virus+Genomes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Inverted+Terminal+Repeat+Sequences+Are+Important+for+Intermolecular+Recombination+and+Circularization+of+Adeno-Associated+Virus+Genomes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0001634-Zentilin1" id="pone.0001634-Zentilin1"></a>Zentilin L, Marcello A, Giacca M (2001) Involvement of Cellular Double-Stranded DNA Break Binding Proteins in Processing of the Recombinant Adeno-Associated Virus Genome. J Virol  75: 12279–12287.  <ul class="find" data-citedArticleID="1059240" data-doi="10.1128/jvi.75.24.12279-12287.2001"><li><a href="http://dx.doi.org/10.1128/jvi.75.24.12279-12287.2001" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Involvement+of+Cellular+Double-Stranded+DNA+Break+Binding+Proteins+in+Processing+of+the+Recombinant+Adeno-Associated+Virus+Genome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Involvement+of+Cellular+Double-Stranded+DNA+Break+Binding+Proteins+in+Processing+of+the+Recombinant+Adeno-Associated+Virus+Genome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0001634-Voigt1" id="pone.0001634-Voigt1"></a>Voigt CA, Martinez C, Wang ZG, Mayo SL, Arnold FH (2002) Protein building blocks preserved by recombination. Nat Struct Biol  9: 553–558.  <ul class="find" data-citedArticleID="1059224" data-doi="10.1038/nsb805"><li><a href="http://dx.doi.org/10.1038/nsb805" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Protein+building+blocks+preserved+by+recombination." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Protein+building+blocks+preserved+by+recombination.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0001634-Kohlbrenner1" id="pone.0001634-Kohlbrenner1"></a>Kohlbrenner E, Aslanidi G, Nash K, Shklyaev S, Campbell-Thompson M, et al.  (2005) Successful production of pseudotyped rAAV vectors using a modified baculovirus expression system. Mol Ther  12: 1217–1225.  <ul class="find" data-citedArticleID="1059210" data-doi="10.1016/j.ymthe.2005.08.018"><li><a href="http://dx.doi.org/10.1016/j.ymthe.2005.08.018" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Successful+production+of+pseudotyped+rAAV+vectors+using+a+modified+baculovirus+expression+system." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Successful+production+of+pseudotyped+rAAV+vectors+using+a+modified+baculovirus+expression+system.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0001634-deSouza1" id="pone.0001634-deSouza1"></a>de Souza SJ, Long M, Schoenbach L, Roy SW, Gilbert W (1996) Intron positions correlate with module boundaries in ancient proteins. Proc Natl Acad Sci U S A  93: 14632–14636.  <ul class="find" data-citedArticleID="1059242" data-doi="10.1016/s0168-9525(97)83365-3"><li><a href="http://dx.doi.org/10.1016/s0168-9525(97)83365-3" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Intron+positions+correlate+with+module+boundaries+in+ancient+proteins." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Intron+positions+correlate+with+module+boundaries+in+ancient+proteins.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0001634-Gilbert1" id="pone.0001634-Gilbert1"></a>Gilbert W, de Souza SJ, Long M (1997) Origin of genes. Proc Natl Acad Sci U S A  94: 7698–7703.  <ul class="find" data-citedArticleID="1059200" data-doi="10.1073/pnas.94.15.7698"><li><a href="http://dx.doi.org/10.1073/pnas.94.15.7698" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Origin+of+genes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Origin+of+genes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0001634-Erles1" id="pone.0001634-Erles1"></a>Erles K, Sebökovà P, Schlehofer J (1999) Update on the prevalence of serum antibodies (IgG and IgM) to adeno-associated virus (AAV). Journal of Medical Virology  59: 406–411.  <ul class="find" data-citedArticleID="1059194" data-doi="10.1002/(sici)1096-9071(199911)59:3<406::aid-jmv22&gt;3.0.co;2-n"><li><a href="http://dx.doi.org/10.1002/(sici)1096-9071(199911)59:3<406::aid-jmv22&gt;3.0.co;2-n" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Update+on+the+prevalence+of+serum+antibodies+%28IgG+and+IgM%29+to+adeno-associated+virus+%28AAV%29." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Update+on+the+prevalence+of+serum+antibodies+%28IgG+and+IgM%29+to+adeno-associated+virus+%28AAV%29.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0001634-Thompson1" id="pone.0001634-Thompson1"></a>Thompson JD, Higgins DG, Gibson TJ (1994) CLUSTAL W: improving the sensitivity of progressive multiple sequence alignment through sequence weighting, position-specific gap penalties and weight matrix choice. Nucleic Acids Res  22: 4673–4680.  <ul class="find" data-citedArticleID="1059222" data-doi="10.1093/nar/22.22.4673"><li><a href="http://dx.doi.org/10.1093/nar/22.22.4673" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=CLUSTAL+W%3A+improving+the+sensitivity+of+progressive+multiple+sequence+alignment+through+sequence+weighting%2C+position-specific+gap+penalties+and+weight+matrix+choice." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22CLUSTAL+W%3A+improving+the+sensitivity+of+progressive+multiple+sequence+alignment+through+sequence+weighting%2C+position-specific+gap+penalties+and+weight+matrix+choice.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0001634-Guindon1" id="pone.0001634-Guindon1"></a>Guindon S, Gascuel O (2003) A simple, fast, and accurate algorithm to estimate large phylogenies by maximum likelihood. Syst Biol  52: 696–704.  <ul class="find" data-citedArticleID="1059206"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=A+simple%2C+fast%2C+and+accurate+algorithm+to+estimate+large+phylogenies+by+maximum+likelihood.&amp;auth=&amp;atitle=A+simple%2C+fast%2C+and+accurate+algorithm+to+estimate+large+phylogenies+by+maximum+likelihood." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+simple%2C+fast%2C+and+accurate+algorithm+to+estimate+large+phylogenies+by+maximum+likelihood." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+simple%2C+fast%2C+and+accurate+algorithm+to+estimate+large+phylogenies+by+maximum+likelihood.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0001634-Posada1" id="pone.0001634-Posada1"></a>Posada D, Crandall KA (1998) MODELTEST: testing the model of DNA substitution. Bioinformatics  14: 817–818.  <ul class="find" data-citedArticleID="1059214" data-doi="10.1093/bioinformatics/14.9.817"><li><a href="http://dx.doi.org/10.1093/bioinformatics/14.9.817" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=MODELTEST%3A+testing+the+model+of+DNA+substitution." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22MODELTEST%3A+testing+the+model+of+DNA+substitution.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001634.XML" value="64416"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001634.PDF" value="354478"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001634.g001.PNG_L" value="1731437"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001634.g001.PNG_M" value="134904"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001634.g001.PNG_S" value="20150"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001634.g001.TIF" value="3147004"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001634.g001.PNG_I" value="139032"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001634.g002.PNG_L" value="1109389"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001634.g002.PNG_M" value="95671"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001634.g002.PNG_S" value="10755"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001634.g002.TIF" value="2433798"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001634.g002.PNG_I" value="31045"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001634.g003.PNG_L" value="389717"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001634.g003.PNG_M" value="120069"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001634.g003.PNG_S" value="13496"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001634.g003.TIF" value="1113044"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001634.g003.PNG_I" value="41559"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001634.t001.PNG_L" value="49658"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001634.t001.PNG_M" value="62986"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001634.t001.PNG_S" value="11190"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001634.t001.TIF" value="241382"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001634.t001.PNG_I" value="21496"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001634.s001.DOC" value="150528"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001634&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001634" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001634&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0001634&volume=&issue=&title=Recombination and Population Mosaic of a Multifunctional Viral Gene, Adeno-Associated Virus cap&author_name=Yasuhiro%20Takeuchi%2C%20Richard%20Myers%2C%20Olivier%20Danos&start_page=1&end_page=7" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001634" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001634" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0001634" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001634&amp;t=Recombination%20and%20Population%20Mosaic%20of%20a%20Multifunctional%20Viral%20Gene%2C%20Adeno-Associated%20Virus%20cap" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001634&title=Recombination%20and%20Population%20Mosaic%20of%20a%20Multifunctional%20Viral%20Gene%2C%20Adeno-Associated%20Virus%20cap&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001634&amp;title=Recombination%20and%20Population%20Mosaic%20of%20a%20Multifunctional%20Viral%20Gene%2C%20Adeno-Associated%20Virus%20cap" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001634" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0001634&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Recombination and Population Mosaic of a Multifunctional Viral Gene, Adeno-Associated Virus cap';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0001634';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Recombination%20and%20Population%20Mosaic%20of%20a%20Multifunctional%20Viral%20Gene%2C%20Adeno-Associated%20Virus%20cap http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001634" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0001634" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">

















          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Animal+phylogenetics%22" title="Search for articles in the subject area:'Animal phylogenetics'"><div class="flagText">Animal phylogenetics</div></a>
              <div data-categoryid="20335" data-articleid="26094"
                   data-categoryname="Animal phylogenetics"
                   class="flagImage" title="Flag 'Animal phylogenetics' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22DNA+sequence+analysis%22" title="Search for articles in the subject area:'DNA sequence analysis'"><div class="flagText">DNA sequence analysis</div></a>
              <div data-categoryid="47935" data-articleid="26094"
                   data-categoryname="DNA sequence analysis"
                   class="flagImage" title="Flag 'DNA sequence analysis' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22DNA+sequences%22" title="Search for articles in the subject area:'DNA sequences'"><div class="flagText">DNA sequences</div></a>
              <div data-categoryid="32003" data-articleid="26094"
                   data-categoryname="DNA sequences"
                   class="flagImage" title="Flag 'DNA sequences' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Homologous+recombination%22" title="Search for articles in the subject area:'Homologous recombination'"><div class="flagText">Homologous recombination</div></a>
              <div data-categoryid="22257" data-articleid="26094"
                   data-categoryname="Homologous recombination"
                   class="flagImage" title="Flag 'Homologous recombination' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Phylogenetic+analysis%22" title="Search for articles in the subject area:'Phylogenetic analysis'"><div class="flagText">Phylogenetic analysis</div></a>
              <div data-categoryid="17851" data-articleid="26094"
                   data-categoryname="Phylogenetic analysis"
                   class="flagImage" title="Flag 'Phylogenetic analysis' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Recombinant+proteins%22" title="Search for articles in the subject area:'Recombinant proteins'"><div class="flagText">Recombinant proteins</div></a>
              <div data-categoryid="18471" data-articleid="26094"
                   data-categoryname="Recombinant proteins"
                   class="flagImage" title="Flag 'Recombinant proteins' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Sequence+alignment%22" title="Search for articles in the subject area:'Sequence alignment'"><div class="flagText">Sequence alignment</div></a>
              <div data-categoryid="18051" data-articleid="26094"
                   data-categoryname="Sequence alignment"
                   class="flagImage" title="Flag 'Sequence alignment' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Sequence+analysis%22" title="Search for articles in the subject area:'Sequence analysis'"><div class="flagText">Sequence analysis</div></a>
              <div data-categoryid="47475" data-articleid="26094"
                   data-categoryname="Sequence analysis"
                   class="flagImage" title="Flag 'Sequence analysis' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=4877'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=201'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=2448&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>


</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
