

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Nullomers: Really a Matter of Natural Selection?</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0001022"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0001022" />

    <meta name="citation_title" content="Nullomers: Really a Matter of Natural Selection?"/>
    <meta itemprop="name" content="Nullomers: Really a Matter of Natural Selection?"/>

      <meta name="citation_author" content="Claudia Acquisti"/>
            <meta name="citation_author_institution" content="Center for Evolutionary Functional Genomics, Arizona State University, Tempe, Arizona, United States of America"/>
            <meta name="citation_author_institution" content="The Biodesign Institute and the School of Life Sciences, Arizona State University, Tempe, Arizona, United States of America"/>
      <meta name="citation_author" content="George Poste"/>
            <meta name="citation_author_institution" content="The Biodesign Institute and the School of Life Sciences, Arizona State University, Tempe, Arizona, United States of America"/>
      <meta name="citation_author" content="David Curtiss"/>
            <meta name="citation_author_institution" content="Center for Evolutionary Functional Genomics, Arizona State University, Tempe, Arizona, United States of America"/>
      <meta name="citation_author" content="Sudhir Kumar"/>
            <meta name="citation_author_institution" content="Center for Evolutionary Functional Genomics, Arizona State University, Tempe, Arizona, United States of America"/>
            <meta name="citation_author_institution" content="The Biodesign Institute and the School of Life Sciences, Arizona State University, Tempe, Arizona, United States of America"/>

    <meta name="citation_date" content="2007/10/10"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0001022.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e1022"/>
    <meta name="citation_issue" content="10"/>
    <meta name="citation_volume" content="2"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Absent sequences: nullomers and primes.; citation_author=G Hampikian; citation_author=T Andersen; citation_journal_title=Pacific Sym. on Biocomputing; citation_volume=12; citation_number=1; citation_pages=355-366; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=How independent are the appearances of n-mers in different genomes?; citation_author=Y Fofanov; citation_author=Y Luo; citation_author=C Katili; citation_author=J Wang; citation_author=Y Belosludtsev; citation_journal_title=Bioinformatics; citation_volume=20; citation_number=2; citation_pages=2421-2428; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Genome comparison without alignment using shortest unique substrings.; citation_author=B Haubold; citation_author=N Pierstorff; citation_author=F Möller; citation_author=T Wiehe; citation_journal_title=BMC Bioinformatics; citation_volume=6; citation_number=3; citation_pages=123; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Transposable elements and the evolution of genome size in eukaryotes.; citation_author=MG Kidwell; citation_journal_title=Genetica; citation_volume=115; citation_number=4; citation_pages=49-63; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=The DNA so dangerous it does not exist.; citation_author=L Geddes; citation_journal_title=New Scientist; citation_volume=2585; citation_number=5; citation_pages=12; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Genome-scale compositional comparisons in eukaryotes.; citation_author=AJ Gentles; citation_author=S Karlin; citation_journal_title=Genome Res.; citation_volume=11; citation_number=6; citation_pages=540-546; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Enzymatic Synthesis of Deoxyribonucleic Acid. VIII. Frequencies of nearest neighbor base sequences in deoxyribonucleic acid.; citation_author=J Josse; citation_author=AD Kaiser; citation_author=A Kornberg; citation_journal_title=J. Biol. Chem.; citation_volume=236; citation_number=7; citation_pages=864-875; citation_date=1961; " />
      <meta name="citation_reference" content="citation_title=The expected equilibrium of the CpG dinucleotide in vertebrate genomes under a mutation model.; citation_author=J Sved; citation_author=A Bird; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=87; citation_number=8; citation_pages=4692-4696; citation_date=1990; " />
      <meta name="citation_reference" content="citation_title=Neutral substitutions occur at a faster rate in exons than in noncoding DNA in primate genomes.; citation_author=S Subramanian; citation_author=S Kumar; citation_journal_title=Genome Res.; citation_volume=13; citation_number=9; citation_pages=838-844; citation_date=2003; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001022" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Nullomers: Really a Matter of Natural Selection?"/>
    <meta name="twitter:description" content="BackgroundNullomers are short DNA sequences that are absent from the genomes of humans and other species. Assuming that nullomers are the signatures of natural selection against deleterious sequences in humans, the use of nullomers in drug target identification, pesticide development, environmental monitoring, and forensic applications has been envisioned.ResultsHere, we show that the hypermutability of CpG dinucleotides, rather than the natural selection against the nullomer sequences, is likely the reason for the phenomenal event of short sequence motifs becoming nullomers. Furthermore, many reported human nullomers differ by only one nucleotide, which reinforces the role of mutation in the evolution of the constellation of nullomers in populations and species. The known nullomers in chimpanzee, cow, dog, and mouse genomes show patterns that are consistent with those seen in humans.ConclusionsThe role of mutations, instead of selection, in generating nullomers cast doubt on the utility of nullomers in many envisioned applications, because of their dependence on the role of lethal selection on the origin of nullomers."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0001022.g002"/>

  <meta property="og:title" content="Nullomers: Really a Matter of Natural Selection?" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001022" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=6843'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=8190'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=7622&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0001022">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001022" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001022&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Nullomers: Really a Matter of Natural Selection?
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Claudia Acquisti, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliations:
                  Center for Evolutionary Functional Genomics, Arizona State University, Tempe, Arizona, United States of America, 
                  The Biodesign Institute and the School of Life Sciences, Arizona State University, Tempe, Arizona, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            George Poste, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  The Biodesign Institute and the School of Life Sciences, Arizona State University, Tempe, Arizona, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            David Curtiss, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Center for Evolutionary Functional Genomics, Arizona State University, Tempe, Arizona, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Sudhir Kumar
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:s.kumar@asu.edu">s.kumar@asu.edu</a></p>

                <p>Affiliations:
                  Center for Evolutionary Functional Genomics, Arizona State University, Tempe, Arizona, United States of America, 
                  The Biodesign Institute and the School of Life Sciences, Arizona State University, Tempe, Arizona, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: October 10, 2007</li>
    <li>DOI: 10.1371/journal.pone.0001022</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0001022" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0001022" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001022" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0001022" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0001022-g001" data-doi="info:doi/10.1371/journal.pone.0001022" data-uri="info:doi/10.1371/journal.pone.0001022.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001022.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001022-t001" data-doi="info:doi/10.1371/journal.pone.0001022" data-uri="info:doi/10.1371/journal.pone.0001022.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001022.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001022-g002" data-doi="info:doi/10.1371/journal.pone.0001022" data-uri="info:doi/10.1371/journal.pone.0001022.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001022.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001022">Reader Comments (1)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0001022" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2>
<h3>Background</h3>
<a id="article1.front1.article-meta1.abstract1.sec1.p1" name="article1.front1.article-meta1.abstract1.sec1.p1"></a><p>Nullomers are short DNA sequences that are absent from the genomes of humans and other species. Assuming that nullomers are the signatures of natural selection against deleterious sequences in humans, the use of nullomers in drug target identification, pesticide development, environmental monitoring, and forensic applications has been envisioned.</p>


<h3>Results</h3>
<a id="article1.front1.article-meta1.abstract1.sec2.p1" name="article1.front1.article-meta1.abstract1.sec2.p1"></a><p>Here, we show that the hypermutability of CpG dinucleotides, rather than the natural selection against the nullomer sequences, is likely the reason for the phenomenal event of short sequence motifs becoming nullomers. Furthermore, many reported human nullomers differ by only one nucleotide, which reinforces the role of mutation in the evolution of the constellation of nullomers in populations and species. The known nullomers in chimpanzee, cow, dog, and mouse genomes show patterns that are consistent with those seen in humans.</p>


<h3>Conclusions</h3>
<a id="article1.front1.article-meta1.abstract1.sec3.p1" name="article1.front1.article-meta1.abstract1.sec3.p1"></a><p>The role of mutations, instead of selection, in generating nullomers cast doubt on the utility of nullomers in many envisioned applications, because of their dependence on the role of lethal selection on the origin of nullomers.</p>

</div>


<div class="articleinfo"><p><strong>Citation: </strong>Acquisti C, Poste G, Curtiss D, Kumar S (2007) Nullomers: Really a Matter of Natural Selection? PLoS ONE 2(10):
          e1022.
            doi:10.1371/journal.pone.0001022</p><p><strong>Academic Editor: </strong>Steven Salzberg, University of Maryland, United States of America</p><p><strong>Received:</strong> August 27, 2007; <strong>Accepted:</strong> September 19, 2007; <strong>Published:</strong> October 10, 2007</p><p><strong>Copyright:</strong> © 2007 Acquisti et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>This work was supported in part by research grants from National Institutes of Health and National Science Foundation (SK).</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>With the number of completely sequenced genomes approaching a landmark of 1000, it is becoming possible to look for similarities and differences between genomes to make evolutionary and functional inferences. For instance, using the complete genome sequences, investigators are now identifying short sequences that are missing from one or more genomes <a href="#pone.0001022-Hampikian1">[1]</a>–<a href="#pone.0001022-Haubold1">[3]</a>. The discovery of the absence of very short sequences (called nullomers) is rather unexpected when we consider that the genomes of higher organisms are extremely large, with a majority consisting of non-coding and non-repetitive sequences <a href="#pone.0001022-Kidwell1">[4]</a>. For example, over half of the haploid human genome (~1.5 billion base pairs) is occupied by unique sequences with no known function, and one would expect to see all possible sequences of up to a length of ~15 in the human genome.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>However, many 11 bp sequence motifs were found to be missing from the human genome <a href="#pone.0001022-Hampikian1">[1]</a>. Immediately, these nullomers have been assumed to be caused by the act of negative selection, and thus considered useful for a variety of basic science and application scenarios, including drug target identification, pesticide development, environmental monitoring, and forensics <a href="#pone.0001022-Hampikian1">[1]</a>, <a href="#pone.0001022-Geddes1">[5]</a>. Is the natural selection against nullomers really the primary cause of their absence in our genomes?</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Analysis and Discussion"></a><h3>Analysis and Discussion</h3><a id="article1.body1.sec2.p1" name="article1.body1.sec2.p1"></a><p>We examined whether the well-known mutational characteristics of the human genome may create the observed deficit of some short sequence motifs. At the outset, we observed that ~50% of the nucleotides in the 80 reported human nullomers of length 11 <a href="#pone.0001022-Hampikian1">[1]</a> participate in the CpG dinucleotides (CpGs). This number is 40 times that seen for nucleotides in the non-coding regions of the human genome <a href="#pone.0001022-Gentles1">[6]</a>.</p>
<a id="article1.body1.sec2.p2" name="article1.body1.sec2.p2"></a><p>A deficit of sequence motifs containing CpGs in vertebrates' genomes has been known for over four decades <a href="#pone.0001022-Josse1">[7]</a>; it is caused by the hypermutability of CpGs, which mutate at a rate 10–20 times higher than the other point mutations <a href="#pone.0001022-Sved1">[e.g.8]</a>, <a href="#pone.0001022-Subramanian1">[9]</a>. Because all reported human nullomers contain multiple CpGs, their absence from the human genome may be caused by the hyper mutation of positions involved in CpG dinucleotides. In this case, we expect to see many 11 bp sequences that show C→T and G→A differences from nullomer sequences, which correspond to the CpG → TpG and CpG → CpA mutations, respectively. This prediction is confirmed by the analysis of the human genome (<a href="#pone-0001022-g001">Fig. 1A, B</a>). In fact, motifs that differ in one, two and three base pairs from the four never-found human nullomers occur with increasing frequencies consistent with the effects of CpG hypermutability. Nullomer-alternatives with all the CpGs mutated occur with the greatest frequencies, and there is an exponential negative relationship between the number of CpGs contained in a motif and its frequency in the human genome (<a href="#pone-0001022-g001">Fig. 1A</a>).</p>
<div class="figure" id="pone-0001022-g001"><div class="img"><a name="pone-0001022-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001022.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001022" data-uri="info:doi/10.1371/journal.pone.0001022.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001022.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001022.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001022.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001022.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001022.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001022.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001022.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001022.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001022.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Frequencies of mutated forms of the nullomers in the human genome.</span></strong></p><a id="article1.body1.sec2.fig1.caption1.p1" name="article1.body1.sec2.fig1.caption1.p1"></a><p>(A) The counts of CpG-mutated forms of nullomers (with one, two, and three CpGs mutated to TpG or CpA) found in the non-repetitive portion of the human genome sequence (<a href="http://hgdownload.cse.ucsc.edu/goldenPath/hg18/bigZips">http://hgdownload.cse.ucsc.edu/goldenPat​h/hg18/bigZips</a>). The results are shown for the four never-found nullomers in the human genome and polymorphic sequences databases (see Table 3 in ref. <a href="#pone.0001022-Hampikian1">[1]</a>). (B) Chromosomal means and standard deviations of the numbers of fully-CpG-mutated forms of the 76 nullomers, which occur with low frequency <a href="#pone.0001022-Hampikian1">[1]</a>. Estimates were derived by randomly mutating all CpG positions to either CpA or TpG for each nullomers and then scanning each human chromosome to find their frequency. The set of mutants analyzed contained 76 sequences, because each nullomer contributed one mutated form to the set . (C) The ratio of the average counts of the original and CpG-mutated nullomer sequences. Results are shown for one, two, and three mutations for all nullomers that contain three CpGs (see panel A). Original and CpG mutants with the same A+T content were compared.</p>
<span>doi:10.1371/journal.pone.0001022.g001</span></div><a id="article1.body1.sec2.p3" name="article1.body1.sec2.p3"></a><p>The nullomer-alternatives with one, two, and three CpG mutations contain increasingly larger A+T content, because of C→T and G→A changes in the nullomer mutants. Therefore, we investigated whether the increasing A+T content of the CpG mutated nullomers may cause the trend seen in <a href="#pone-0001022-g001">Fig. 1A</a>, because A+T-rich motifs are more abundant in the genome. For making valid comparisons within the context of the nullomer sequences, we computed the average number of motifs occurrences where all the CpGs in the nullomer sequence were replaced by dinucleotides other than CpA and TpG (non-CpG mutants). We found the ratios of the non-CpG mutants to the CpG mutants for each A+T content category to be close to one when all CpG dinucleotides are mutated, but it becomes 60 times higher when only one CpG is mutated (<a href="#pone-0001022-g001">Fig. 1C</a>). This trend suggests that hypermutability of CpGs is the primary factor responsible for causing a deficit of motifs, which are identified as nullomers.</p>
<a id="article1.body1.sec2.p4" name="article1.body1.sec2.p4"></a><p>If CpG hypermutability is a major factor in causing nullomers, we expect to see an overabundance of CpG dinucleotides in the nullomers found in other mammalian genomes. This is indeed the case, as over 50% base pairs in the nullomers identified in chimpanzee, mouse, dog, and cow, genomes are involved in CpGs (<a href="#pone-0001022-t001">Table 1</a>). Interestingly, out of a total of 530 nullomers, not a single one is CpG-free (<a href="#pone-0001022-t001">Table 1</a>). In fact, all nullomers contain at least two CpG dinucleotides (<a href="#pone-0001022-t001">Table 1</a>).</p>
<div class="figure" id="pone-0001022-t001"><div class="img"><a name="pone-0001022-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001022.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001022" data-uri="info:doi/10.1371/journal.pone.0001022.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001022.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001022.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001022.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001022.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001022.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001022.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001022.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001022.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001022.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>Proponderance of CpG dinucleotides in the mammalian nullomers of length 11 base pairs.</span></strong></p><span>doi:10.1371/journal.pone.0001022.t001</span></div><a id="article1.body1.sec2.p5" name="article1.body1.sec2.p5"></a><p>In addition to hypermutable CpG dinucleotides, regular point mutations appear to have played an important role in generating the observed constellation of human nullomers. This is clearly evident from the comparative sequence analysis of nullomers within human and between human and chimpanzee genomes (<a href="#pone-0001022-g002">Fig. 2</a>). Within humans, 14 out of 80 reported nullomers differ in only one base pair. In addition, humans share 28 nullomers with chimpanzees, and 14 human nullomers differ in one base pair from chimpanzee nullomers. This means that the human set contains nullomers inherited from the common ancestor of human and chimpanzee, in addition to those that have arisen within the human lineage. Furthermore, the human genome shares more nullomers with its closest evolutionary relative chimpanzee than with distantly-related mammals, including mouse (two), cow (none) and dog (none).</p>
<div class="figure" id="pone-0001022-g002"><div class="img"><a name="pone-0001022-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001022.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001022" data-uri="info:doi/10.1371/journal.pone.0001022.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001022.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001022.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001022.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001022.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001022.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001022.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001022.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001022.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001022.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Similarities and differences in human and chimpanzee nullomers.</span></strong></p><a id="article1.body1.sec2.fig2.caption1.p1" name="article1.body1.sec2.fig2.caption1.p1"></a><p>(A) Seven pairs of human nullomers that differ in only one nucleotide. Note that several nullomers are listed twice in ref. 1, as some sequences are reverse complement of others (e.g., atatCGCGCGt and aCGCGCGatat). (B) Set of nullomers that are found in both human and chimpanzee genomes (data from <a href="http://trac.boisestate.edu/dna/applets/SeqCount.html">http://trac.boisestate.edu/dna/applets/S​eqCount.html</a>) <a href="#pone.0001022-Hampikian1">[1]</a>.</p>
<span>doi:10.1371/journal.pone.0001022.g002</span></div><a id="article1.body1.sec2.p6" name="article1.body1.sec2.p6"></a><p>In summary, our results suggest that the collection of human and other mammalian nullomers are likely a by-product of the mutational characteristics of the genome, with hypermutable CpG dinucleotides playing a major role. This means that similarities in the mutational patterns among species will lead to the discovery of similar (or common) sets of nullomers among species. Therefore, mutational characteristics, rather than selection pressures, lead to the rare occurrence and absence of certain motifs. This fact will adversely impact envisioned uses of these nullomers, which are contingent on the presupposition that lethality of nullomer sequences to the host individuals is the primary cause of their rarity in our genomes <a href="#pone.0001022-Hampikian1">[e.g.1]</a>, <a href="#pone.0001022-Geddes1">[5]</a>.</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>We thank Dr. Paul Davies for comments on an earlier draft of this manuscript and Kristi Garboushian for editorial support.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: SK GP. Performed the experiments: CA. Analyzed the data: CA. Contributed reagents/materials/analysis tools: CA DC. Wrote the paper: SK CA.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0001022-Hampikian1" id="pone.0001022-Hampikian1"></a>Hampikian G, Andersen T (2007) Absent sequences: nullomers and primes. Pacific Sym. on Biocomputing  12: 355–366.  <ul class="find" data-citedArticleID="1004276" data-doi="10.1142/9789812772435_0034"><li><a href="http://dx.doi.org/10.1142/9789812772435_0034" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Absent+sequences%3A+nullomers+and+primes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Absent+sequences%3A+nullomers+and+primes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0001022-Fofanov1" id="pone.0001022-Fofanov1"></a>Fofanov Y, Luo Y, Katili C, Wang J, Belosludtsev Y, et al.  (2004) How independent are the appearances of n-mers in different genomes? Bioinformatics  20: 2421–2428.  <ul class="find" data-citedArticleID="1004270" data-doi="10.1093/bioinformatics/bth266"><li><a href="http://dx.doi.org/10.1093/bioinformatics/bth266" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=How+independent+are+the+appearances+of+n-mers+in+different+genomes%3F" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22How+independent+are+the+appearances+of+n-mers+in+different+genomes%3F%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0001022-Haubold1" id="pone.0001022-Haubold1"></a>Haubold B, Pierstorff N, Möller F, Wiehe T (2005) Genome comparison without alignment using shortest unique substrings. BMC Bioinformatics  6: 123.  <ul class="find" data-citedArticleID="1004278" data-doi="10.1186/1471-2105-6-123"><li><a href="http://dx.doi.org/10.1186/1471-2105-6-123" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genome+comparison+without+alignment+using+shortest+unique+substrings." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genome+comparison+without+alignment+using+shortest+unique+substrings.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0001022-Kidwell1" id="pone.0001022-Kidwell1"></a>Kidwell MG (2002) Transposable elements and the evolution of genome size in eukaryotes. Genetica  115: 49–63.  <ul class="find" data-citedArticleID="1004282" data-doi="10.1023/a:1016072014259"><li><a href="http://dx.doi.org/10.1023/a:1016072014259" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Transposable+elements+and+the+evolution+of+genome+size+in+eukaryotes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Transposable+elements+and+the+evolution+of+genome+size+in+eukaryotes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0001022-Geddes1" id="pone.0001022-Geddes1"></a>Geddes L (2007) The DNA so dangerous it does not exist. New Scientist  2585: 12.  <ul class="find" data-citedArticleID="1004272" data-doi="10.1016/s0262-4079(07)60013-x"><li><a href="http://dx.doi.org/10.1016/s0262-4079(07)60013-x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+DNA+so+dangerous+it+does+not+exist." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+DNA+so+dangerous+it+does+not+exist.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0001022-Gentles1" id="pone.0001022-Gentles1"></a>Gentles AJ, Karlin S (2001) Genome-scale compositional comparisons in eukaryotes. Genome Res.  11: 540–546.  <ul class="find" data-citedArticleID="1004274" data-doi="10.1101/gr.163101"><li><a href="http://dx.doi.org/10.1101/gr.163101" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genome-scale+compositional+comparisons+in+eukaryotes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genome-scale+compositional+comparisons+in+eukaryotes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0001022-Josse1" id="pone.0001022-Josse1"></a>Josse J, Kaiser AD, Kornberg A (1961) Enzymatic Synthesis of Deoxyribonucleic Acid. VIII. Frequencies of nearest neighbor base sequences in deoxyribonucleic acid. J. Biol. Chem.  236: 864–875.  <ul class="find" data-citedArticleID="1004280"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Enzymatic+Synthesis+of+Deoxyribonucleic+Acid.+VIII.+Frequencies+of+nearest+neighbor+base+sequences+in+deoxyribonucleic+acid.&amp;auth=&amp;atitle=Enzymatic+Synthesis+of+Deoxyribonucleic+Acid.+VIII.+Frequencies+of+nearest+neighbor+base+sequences+in+deoxyribonucleic+acid." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Enzymatic+Synthesis+of+Deoxyribonucleic+Acid.+VIII.+Frequencies+of+nearest+neighbor+base+sequences+in+deoxyribonucleic+acid." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Enzymatic+Synthesis+of+Deoxyribonucleic+Acid.+VIII.+Frequencies+of+nearest+neighbor+base+sequences+in+deoxyribonucleic+acid.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0001022-Sved1" id="pone.0001022-Sved1"></a>Sved J, Bird A (1990) The expected equilibrium of the CpG dinucleotide in vertebrate genomes under a mutation model. Proc Natl Acad Sci U S A  87: 4692–4696.  <ul class="find" data-citedArticleID="1004286" data-doi="10.1073/pnas.87.12.4692"><li><a href="http://dx.doi.org/10.1073/pnas.87.12.4692" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+expected+equilibrium+of+the+CpG+dinucleotide+in+vertebrate+genomes+under+a+mutation+model." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+expected+equilibrium+of+the+CpG+dinucleotide+in+vertebrate+genomes+under+a+mutation+model.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0001022-Subramanian1" id="pone.0001022-Subramanian1"></a>Subramanian S, Kumar S (2003) Neutral substitutions occur at a faster rate in exons than in noncoding DNA in primate genomes. Genome Res.  13: 838–844.  <ul class="find" data-citedArticleID="1004284" data-doi="10.1101/gr.1152803"><li><a href="http://dx.doi.org/10.1101/gr.1152803" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Neutral+substitutions+occur+at+a+faster+rate+in+exons+than+in+noncoding+DNA+in+primate+genomes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Neutral+substitutions+occur+at+a+faster+rate+in+exons+than+in+noncoding+DNA+in+primate+genomes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001022.XML" value="26381"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001022.PDF" value="285310"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001022.g001.PNG_L" value="1283444"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001022.g001.PNG_M" value="178932"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001022.g001.PNG_S" value="14726"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001022.g001.TIF" value="2026620"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001022.g001.PNG_I" value="92290"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001022.t001.PNG_L" value="72463"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001022.t001.PNG_M" value="95497"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001022.t001.PNG_S" value="14727"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001022.t001.TIF" value="299616"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001022.t001.PNG_I" value="32392"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001022.g002.PNG_L" value="363751"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001022.g002.PNG_M" value="227539"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001022.g002.PNG_S" value="13674"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001022.g002.TIF" value="447304"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001022.g002.PNG_I" value="77226"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001022&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001022" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001022&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0001022&volume=&issue=&title=Nullomers: Really a Matter of Natural Selection?&author_name=Claudia%20Acquisti%2C%20George%20Poste%2C%20David%20Curtiss%2C%20Sudhir%20Kumar&start_page=1&end_page=3" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001022" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001022" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0001022" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001022&amp;t=Nullomers%3A%20Really%20a%20Matter%20of%20Natural%20Selection%3F" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001022&title=Nullomers%3A%20Really%20a%20Matter%20of%20Natural%20Selection%3F&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001022&amp;title=Nullomers%3A%20Really%20a%20Matter%20of%20Natural%20Selection%3F" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001022" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0001022&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Nullomers: Really a Matter of Natural Selection?';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0001022';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Nullomers%3A%20Really%20a%20Matter%20of%20Natural%20Selection%3F http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001022" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0001022" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">
















          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Chimpanzees%22" title="Search for articles in the subject area:'Chimpanzees'"><div class="flagText">Chimpanzees</div></a>
              <div data-categoryid="43093" data-articleid="24878"
                   data-categoryname="Chimpanzees"
                   class="flagImage" title="Flag 'Chimpanzees' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22DNA+methylation%22" title="Search for articles in the subject area:'DNA methylation'"><div class="flagText">DNA methylation</div></a>
              <div data-categoryid="32275" data-articleid="24878"
                   data-categoryname="DNA methylation"
                   class="flagImage" title="Flag 'DNA methylation' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Genome+evolution%22" title="Search for articles in the subject area:'Genome evolution'"><div class="flagText">Genome evolution</div></a>
              <div data-categoryid="17207" data-articleid="24878"
                   data-categoryname="Genome evolution"
                   class="flagImage" title="Flag 'Genome evolution' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Genomic+databases%22" title="Search for articles in the subject area:'Genomic databases'"><div class="flagText">Genomic databases</div></a>
              <div data-categoryid="42727" data-articleid="24878"
                   data-categoryname="Genomic databases"
                   class="flagImage" title="Flag 'Genomic databases' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Human+genomics%22" title="Search for articles in the subject area:'Human genomics'"><div class="flagText">Human genomics</div></a>
              <div data-categoryid="17253" data-articleid="24878"
                   data-categoryname="Human genomics"
                   class="flagImage" title="Flag 'Human genomics' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Mammalian+genomics%22" title="Search for articles in the subject area:'Mammalian genomics'"><div class="flagText">Mammalian genomics</div></a>
              <div data-categoryid="17625" data-articleid="24878"
                   data-categoryname="Mammalian genomics"
                   class="flagImage" title="Flag 'Mammalian genomics' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Mutation%22" title="Search for articles in the subject area:'Mutation'"><div class="flagText">Mutation</div></a>
              <div data-categoryid="31809" data-articleid="24878"
                   data-categoryname="Mutation"
                   class="flagImage" title="Flag 'Mutation' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Sequence+motif+analysis%22" title="Search for articles in the subject area:'Sequence motif analysis'"><div class="flagText">Sequence motif analysis</div></a>
              <div data-categoryid="19657" data-articleid="24878"
                   data-categoryname="Sequence motif analysis"
                   class="flagImage" title="Flag 'Sequence motif analysis' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=4418'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=523'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=440&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>

<div class="block sidebar-comments">
    <div class="header">
        <h3>Comments</h3>
    </div>
      <p><a href="/annotation/listThread.action?root=5671">A nice counterpoint to a recent over-interpretation</a><br>Posted by salzberg</p>
</div>

</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
