

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Physiology, Pathology and Relatedness of Human Tissues from Gene Expression Meta-Analysis</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0001880"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0001880" />

    <meta name="citation_title" content="Physiology, Pathology and Relatedness of Human Tissues from Gene Expression Meta-Analysis"/>
    <meta itemprop="name" content="Physiology, Pathology and Relatedness of Human Tissues from Gene Expression Meta-Analysis"/>

      <meta name="citation_author" content="Dario Greco"/>
            <meta name="citation_author_institution" content="Institute of Biotechnology, University of Helsinki, Helsinki, Finland"/>
      <meta name="citation_author" content="Panu Somervuo"/>
            <meta name="citation_author_institution" content="Institute of Biotechnology, University of Helsinki, Helsinki, Finland"/>
      <meta name="citation_author" content="Antonio Di Lieto"/>
            <meta name="citation_author_institution" content="Neuroscience Center, University of Helsinki, Helsinki, Finland"/>
      <meta name="citation_author" content="Tuomas Raitila"/>
            <meta name="citation_author_institution" content="Institute of Biotechnology, University of Helsinki, Helsinki, Finland"/>
      <meta name="citation_author" content="Lucio Nitsch"/>
            <meta name="citation_author_institution" content="Dipartimento di Biologia e Patologia Cellulare e Molecolare, Università “Federico II”, Napoli, Italy"/>
      <meta name="citation_author" content="Eero Castrén"/>
            <meta name="citation_author_institution" content="Neuroscience Center, University of Helsinki, Helsinki, Finland"/>
      <meta name="citation_author" content="Petri Auvinen"/>
            <meta name="citation_author_institution" content="Institute of Biotechnology, University of Helsinki, Helsinki, Finland"/>

    <meta name="citation_date" content="2008/4/2"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0001880.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e1880"/>
    <meta name="citation_issue" content="4"/>
    <meta name="citation_volume" content="3"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Compactness of human housekeeping genes: selection for economy or genomic design?; citation_author=AE Vinogradov; citation_journal_title=Trends Genet; citation_volume=20(5); citation_number=1; citation_pages=248-253; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Selection for short introns in highly expressed genes.; citation_author=CI Castillo-Davis; citation_author=SL Mekhedov; citation_author=DL Hartl; citation_author=EV Koonin; citation_author=FA Kondrashov; citation_journal_title=Nat Genet; citation_volume=31(4); citation_number=2; citation_pages=415-418; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Isochores and tissue-specificity.; citation_author=AE Vinogradov; citation_journal_title=Nucleic Acids Res; citation_volume=31(17); citation_number=3; citation_pages=5212-5220; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Determinants of substitution rates in mammalian genes: expression pattern affects selection intensity but not mutation rate.; citation_author=L Duret; citation_author=D Mouchiroud; citation_journal_title=Mol Biol Evol; citation_volume=17; citation_number=4; citation_pages=68-74; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Elevated rates of protein secretion, evolution, and disease among tissue-specific genes.; citation_author=EE Winter; citation_author=L Goodstadt; citation_author=CP Ponting; citation_journal_title=Genome research; citation_volume=14; citation_number=5; citation_pages=54-61; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Detecting and profiling tissue-selective genes.; citation_author=S Liang; citation_author=Y Li; citation_author=X Be; citation_author=S Howes; citation_author=W Liu; citation_journal_title=Physiological genomics; citation_volume=26(2); citation_number=6; citation_pages=158-162; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=High density synthetic oligonucleotide arrays.; citation_author=RJ Lipshutz; citation_author=SP Fodor; citation_author=TR Gingeras; citation_author=DJ Lockhart; citation_journal_title=Nature genetics; citation_volume=21; citation_number=7; citation_pages=20-24; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=A gene atlas of the mouse and human protein-encoding transcriptomes.; citation_author=AI Su; citation_author=T Wiltshire; citation_author=S Batalov; citation_author=H Lapp; citation_author=KA Ching; citation_journal_title=Proceedings of the National Academy of Sciences of the United States of America; citation_volume=101(16); citation_number=8; citation_pages=6062-6067; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Evolving gene/transcript definitions significantly alter the interpretation of GeneChip data.; citation_author=M Dai; citation_author=P Wang; citation_author=AD Boyd; citation_author=G Kostov; citation_author=B Athey; citation_journal_title=Nucleic acids research; citation_volume=33(20); citation_number=9; citation_pages=e175; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Improved precision and accuracy for microarrays using updated probe set definitions.; citation_author=R Sandberg; citation_author=O Larsson; citation_journal_title=BMC Bioinformatics; citation_volume=8; citation_number=10; citation_pages=48; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Redefinition of Affymetrix probe sets by sequence overlap with cDNA microarray probes reduces cross-platform inconsistencies in cancer-associated gene expression measurements.; citation_author=SL Carter; citation_author=AC Eklund; citation_author=BH Mecham; citation_author=IS Kohane; citation_author=Z Szallasi; citation_journal_title=BMC Bioinformatics; citation_volume=6; citation_number=11; citation_pages=107; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=GATA4 mutations cause human congenital heart defects and reveal an interaction with TBX5.; citation_author=V Garg; citation_author=IS Kathiriya; citation_author=R Barnes; citation_author=MK Schluterman; citation_author=IN King; citation_journal_title=Nature; citation_volume=424(6947); citation_number=12; citation_pages=443-447; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Phenotypes with GATA4 or NKX2.5 mutations in familial atrial septal defect.; citation_author=K Hirayama-Yamada; citation_author=M Kamisago; citation_author=K Akimoto; citation_author=H Aotsuka; citation_author=Y Nakamura; citation_journal_title=American journal of medical genetics; citation_volume=135; citation_number=13; citation_pages=47-52; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Mutations in the cardiac transcription factor NKX2.5 affect diverse cardiac developmental pathways.; citation_author=DW Benson; citation_author=GM Silberbach; citation_author=A Kavanaugh-McHugh; citation_author=C Cottrill; citation_author=Y Zhang; citation_journal_title=J Clin Invest; citation_volume=104(11); citation_number=14; citation_pages=1567-1573; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=NKX2.5 mutations in patients with tetralogy of fallot.; citation_author=E Goldmuntz; citation_author=E Geiger; citation_author=DW Benson; citation_journal_title=Circulation; citation_volume=104(21); citation_number=15; citation_pages=2565-2568; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Congenital heart disease caused by mutations in the transcription factor NKX2-5.; citation_author=JJ Schott; citation_author=DW Benson; citation_author=CT Basson; citation_author=W Pease; citation_author=GM Silberbach; citation_journal_title=Science; citation_volume=281(5373); citation_number=16; citation_pages=108-111; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Two novel frameshift mutations in NKX2.5 result in novel features including visceral inversus and sinus venosus type ASD.; citation_author=Y Watanabe; citation_author=DW Benson; citation_author=S Yano; citation_author=T Akagi; citation_author=M Yoshino; citation_author=JC Murray; citation_journal_title=J Med Genet; citation_volume=39(11); citation_number=17; citation_pages=807-811; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Capillary malformation-arteriovenous malformation, a new clinical and genetic disorder caused by RASA1 mutations.; citation_author=I Eerola; citation_author=LM Boon; citation_author=JB Mulliken; citation_author=PE Burrows; citation_author=A Dompmartin; citation_journal_title=Am J Hum Genet; citation_volume=73(6); citation_number=18; citation_pages=1240-1249; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=The amygdala and its place in the cerebral hemisphere.; citation_author=LW Swanson; citation_journal_title=Annals of the New York Academy of Sciences; citation_volume=985; citation_number=19; citation_pages=174-184; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=What is the amygdala?; citation_author=LW Swanson; citation_author=GD Petrovich; citation_journal_title=Trends in neurosciences; citation_volume=21(8); citation_number=20; citation_pages=323-331; citation_date=1998; " />
      <meta name="citation_reference" content="citation_number=21; citation_date=2001; citation_publisher=Garland; " />
      <meta name="citation_reference" content="citation_number=22; citation_date=2000; citation_publisher=Saunders; " />
      <meta name="citation_reference" content="citation_title=Hemopoiesis in human fetal and embryonic liver.; citation_author=W Timens; citation_author=WA Kamps; citation_journal_title=Microsc Res Tech; citation_volume=39(5); citation_number=23; citation_pages=387-397; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Mammalian neural stem cells.; citation_author=FH Gage; citation_journal_title=Science; citation_volume=287(5457); citation_number=24; citation_pages=1433-1438; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=How widespread is adult neurogenesis in mammals?; citation_author=E Gould; citation_journal_title=Nat Rev Neurosci; citation_volume=8(6); citation_number=25; citation_pages=481-488; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Relationship between the tissue-specificity of mouse gene expression and the evolutionary origin and function of the proteins.; citation_author=S Freilich; citation_author=T Massingham; citation_author=S Bhattacharyya; citation_author=H Ponsting; citation_author=PA Lyons; citation_journal_title=Genome biology; citation_volume=6(7); citation_number=26; citation_pages=R56; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Computational analysis of tissue-specific combinatorial gene regulation: predicting interaction between transcription factors in human tissues.; citation_author=X Yu; citation_author=J Lin; citation_author=DJ Zack; citation_author=J Qian; citation_journal_title=Nucleic Acids Res; citation_volume=34(17); citation_number=27; citation_pages=4925-4936; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Identification of regulatory regions which confer muscle-specific gene expression.; citation_author=WW Wasserman; citation_author=JW Fickett; citation_journal_title=J Mol Biol; citation_volume=278; citation_number=28; citation_pages=167-181; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=A predictive model for regulatory sequences directing liver-specific transcription.; citation_author=W Krivan; citation_author=WW Wasserman; citation_journal_title=Genome Res; citation_volume=11(9); citation_number=29; citation_pages=1559-1566; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Gene regulatory networks in the evolution and development of the heart.; citation_author=EN Olson; citation_journal_title=Science; citation_volume=313(5795); citation_number=30; citation_pages=1922-1927; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=The E2F family: specific functions and overlapping interests.; citation_author=C Attwooll; citation_author=E Lazzerini Denchi; citation_author=K Helin; citation_journal_title=EMBO J 8;; citation_volume=23(24); citation_number=31; citation_pages=4709-16; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Sibling rivalry in the E2F family.; citation_author=JM Trimarchi; citation_author=JA Lees; citation_journal_title=Nat Rev Mol Cell Biol; citation_volume=3; citation_number=32; citation_pages=11-20; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Regulation of neuronal survival and death by E2F-dependent gene repression and derepression.; citation_author=DX Liu; citation_author=LA Greene; citation_journal_title=Neuron; citation_volume=32(3); citation_number=33; citation_pages=425-438; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Baltimore D: Physiological functions for brain NF-kappaB.; citation_author=MK Meffert; citation_journal_title=Trends Neurosci; citation_volume=28; citation_number=34; citation_pages=37-43; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Members of the NF-kappaB family expressed in zones of active neurogenesis in the postnatal and adult mouse brain.; citation_author=S Denis-Donini; citation_author=A Caprini; citation_author=C Frassoni; citation_author=M Grilli; citation_journal_title=Brain Res Dev Brain Res; citation_volume=154; citation_number=35; citation_pages=81-89; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Exploration, normalization, and summaries of high density oligonucleotide array probe level data.; citation_author=RA Irizarry; citation_author=B Hobbs; citation_author=F Collin; citation_author=YD Beazer-Barclay; citation_author=KJ Antonellis; citation_journal_title=Biostatistics; citation_volume=4(2); citation_number=36; citation_pages=249-264; citation_date=2003; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001880" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Physiology, Pathology and Relatedness of Human Tissues from Gene Expression Meta-Analysis"/>
    <meta name="twitter:description" content="BackgroundDevelopment and maintenance of the identity of tissues is of central importance for multicellular organisms. Based on gene expression profiles, it is possible to divide genes in housekeeping genes and those whose expression is preferential in one or a few tissues and which provide specialized functions that have a strong effect on the physiology of the whole organism.ResultsWe have surveyed the gene expression in 78 normal human tissues integrating publicly available microarray gene expression data. A total amount of 1601 genes were identified as selectively expressed in one or more tissues. The tissue-selective genes covered a wide range of cellular and molecular functions, and could be linked to 361 human diseases with Mendelian inheritance. Based on the gene expression profiles, we were able to form a network of tissues reflecting their functional relatedness and, to certain extent, their development. Using co-citation driven gene network technique and promoter analysis, we predicted a transcriptional module where the co-operation of the transcription factors E2F and NF-kappaB can possibly regulate a number of genes involved in the neurogenesis that takes place in the adult hippocampus.ConclusionsHere we propose that integration of gene expression data from Affymetrix GeneChip experiments is possible through re-annotation and commonly used pre-processing methods. We suggest that some functional aspects of the tissues can be explained by the co-operation of multiple transcription factors that regulate the expression of selected groups of genes."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0001880.g001"/>

  <meta property="og:title" content="Physiology, Pathology and Relatedness of Human Tissues from Gene Expression Meta-Analysis" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001880" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=841'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=6886'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=8850&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0001880">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001880" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001880&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Physiology, Pathology and Relatedness of Human Tissues from Gene Expression Meta-Analysis
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Dario Greco
              <span class="corresponding">mail</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:dario.greco@helsinki.fi">dario.greco@helsinki.fi</a></p>

                <p>Affiliation:
                  Institute of Biotechnology, University of Helsinki, Helsinki, Finland
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Panu Somervuo, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Institute of Biotechnology, University of Helsinki, Helsinki, Finland
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Antonio Di Lieto, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Neuroscience Center, University of Helsinki, Helsinki, Finland
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Tuomas Raitila, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Institute of Biotechnology, University of Helsinki, Helsinki, Finland
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Lucio Nitsch, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Dipartimento di Biologia e Patologia Cellulare e Molecolare, Università “Federico II”, Napoli, Italy
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Eero Castrén, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Neuroscience Center, University of Helsinki, Helsinki, Finland
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Petri Auvinen
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Institute of Biotechnology, University of Helsinki, Helsinki, Finland
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: April 02, 2008</li>
    <li>DOI: 10.1371/journal.pone.0001880</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0001880" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0001880" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001880" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0001880" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0001880-g001" data-doi="info:doi/10.1371/journal.pone.0001880" data-uri="info:doi/10.1371/journal.pone.0001880.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001880.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001880-t001" data-doi="info:doi/10.1371/journal.pone.0001880" data-uri="info:doi/10.1371/journal.pone.0001880.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001880.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001880">Reader Comments (1)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0001880" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2>
<h3>Background</h3>
<a id="article1.front1.article-meta1.abstract1.sec1.p1" name="article1.front1.article-meta1.abstract1.sec1.p1"></a><p>Development and maintenance of the identity of tissues is of central importance for multicellular organisms. Based on gene expression profiles, it is possible to divide genes in housekeeping genes and those whose expression is preferential in one or a few tissues and which provide specialized functions that have a strong effect on the physiology of the whole organism.</p>


<h3>Results</h3>
<a id="article1.front1.article-meta1.abstract1.sec2.p1" name="article1.front1.article-meta1.abstract1.sec2.p1"></a><p>We have surveyed the gene expression in 78 normal human tissues integrating publicly available microarray gene expression data. A total amount of 1601 genes were identified as selectively expressed in one or more tissues. The tissue-selective genes covered a wide range of cellular and molecular functions, and could be linked to 361 human diseases with Mendelian inheritance. Based on the gene expression profiles, we were able to form a network of tissues reflecting their functional relatedness and, to certain extent, their development. Using co-citation driven gene network technique and promoter analysis, we predicted a transcriptional module where the co-operation of the transcription factors E2F and NF-kappaB can possibly regulate a number of genes involved in the neurogenesis that takes place in the adult hippocampus.</p>


<h3>Conclusions</h3>
<a id="article1.front1.article-meta1.abstract1.sec3.p1" name="article1.front1.article-meta1.abstract1.sec3.p1"></a><p>Here we propose that integration of gene expression data from Affymetrix GeneChip experiments is possible through re-annotation and commonly used pre-processing methods. We suggest that some functional aspects of the tissues can be explained by the co-operation of multiple transcription factors that regulate the expression of selected groups of genes.</p>

</div>


<div class="articleinfo"><p><strong>Citation: </strong>Greco D, Somervuo P, Di Lieto A, Raitila T, Nitsch L, et al.  (2008) Physiology, Pathology and Relatedness of Human Tissues from Gene Expression Meta-Analysis. PLoS ONE 3(4):
          e1880.
            doi:10.1371/journal.pone.0001880</p><p><strong>Editor: </strong>Ulrich Mueller, University of Giessen, Germany</p><p><strong>Received:</strong> November 23, 2007; <strong>Accepted:</strong> February 25, 2008; <strong>Published:</strong> April 2, 2008</p><p><strong>Copyright:</strong> © 2008 Greco et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>This research was funded by the Institute of Biotechnology, University of Helsinki, and by the Academy of Finland (grant 117968).</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>The human body consists of numerous cell types that are highly organized into functional units constituting tissues and organs. Expression patterns of genes have been under selection for eons and, as a result, cell types and tissues differ from each other both morphologically and functionally. The mechanisms leading to the development, differentiation, and maintenance of tissues have been under intensive investigation by generations of scientists.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>A generally accepted view of gene expression programs divides genes in two main categories: i) housekeeping genes that are virtually always expressed in every tissue and work to maintain basic cellular functions; and ii) genes whose expression is preferential in one or a few tissues and which provide specialized functions that have a strong effect on the physiology of the whole organism.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>Compared to the housekeeping genes, tissue-specific genes have been described as longer <a href="#pone.0001880-Vinogradov1">[1]</a>, with longer introns <a href="#pone.0001880-CastilloDavis1">[2]</a>, a lower GC content <a href="#pone.0001880-Vinogradov2">[3]</a>, and lower substitution rates at non synonymous sites <a href="#pone.0001880-Duret1">[4]</a>. Moreover, tissue-specific genes seem to evolve faster and they are more likely to be mutated in genetic diseases with Mendelian inheritance <a href="#pone.0001880-Winter1">[5]</a>.</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p>In terms of gene expression, tissue specificity can be addressed in strict terms of genes that are exclusively transcribed in only one particular tissue type, but there is evidence indicating that most tissues with similar function share many expression patterns. Therefore, the concept of tissue-selectivity, which considers those genes whose expression is enriched in one or more similar tissues <a href="#pone.0001880-Liang1">[6]</a>, might be more useful. Affymetrix high-density oligonucleotide arrays <a href="#pone.0001880-Lipshutz1">[7]</a> have been already used for investigating tissue-specific expression patterns <a href="#pone.0001880-Liang1">[6]</a>, <a href="#pone.0001880-Su1">[8]</a>. However, there are several problematic aspects in the GeneChip technology, related especially to the mis-annotation of many probes. Dai and collaborators <a href="#pone.0001880-Dai1">[9]</a> have observed that updating the probe annotation for all the Affymetrix chipsets affects a large number of the probe sets. More recently, it has been shown that updating the definitions of the Affymetrix probes leads to more precise and accurate results as compared with the original annotations provided by the manufacturer <a href="#pone.0001880-Sandberg1">[10]</a>. Re-annotation of the Affymetrix probes has been also shown to improve the cross-platform reproducibility and meta-analysis of independent microarray experiments <a href="#pone.0001880-Carter1">[11]</a>.</p>
<a id="article1.body1.sec1.p5" name="article1.body1.sec1.p5"></a><p>The aim of this study was to investigate tissue-selective expression patterns, integrating publicly available gene expression data. A total of 195 images of Affymetrix GeneChips were collected from the GEO database (<a href="http://www.ncbi.nlm.nih.gov/geo/">http://www.ncbi.nlm.nih.gov/geo/</a>). All probes present on the chipset were re-annotated according to the latest release of the Entrez Gene database (<a href="http://www.ncbi.nlm.nih.gov/sites/entrezdbgene">http://www.ncbi.nlm.nih.gov/sites/entrez​dbgene</a>). After extended quality control and preprocessing, we explored the tissue-selective expression patterns.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3>
<h4>Identification of tissue-selective genes</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>We searched for genes expressed in a tissue-selective manner. A tissue-selectivity score was computed for each gene and used as a weight for the expression values. After permutation test we could identify 1601 genes selectively expressed in one or more tissues. About 35% of 1601 genes were selectively expressed only in one tissue, 20% were shared by two, and 13% by three tissues. Ten percent of the tissue-selective genes were shared by six or more tissues. The majority of tissue-selective genes shared by ten or more tissues were expressed in neural system tissues. The majority of the tissue selective genes were found in the immune system (32% of 1601), followed by central and peripheral nervous systems (17%), muscles (15%) and reproductive organs (9%). Altogether, the other categories accounted 27% of selective genes.</p>


<h4>Functions of tissue-selective genes</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>The 1601 tissue-selective genes covered a wide range of cellular and molecular functions as they could be annotated into 1694 distinct Biological Process, 1094 Molecular Function and 290 Cellular Component functional families from the three gene ontology classifications (<a href="#pone.0001880.s004">File S1</a>, tables 0.2, 0.3 and 0.4). The gene ontology classification revealed a suggestive distribution of the 1601 tissue-selective genes into functional families: 19% of them were classified in the Molecular Function family “signal transducer activity”, and about 8% in the group of “receptor binding” proteins. Moreover, when the same genes were grouped according to the Cellular Component ontology, about 18% were annotated under the family “extracellular region”.</p>
<a id="article1.body1.sec2.sec2.p2" name="article1.body1.sec2.sec2.p2"></a><p>The classification of the 1601 tissue-selective genes according to the Biological Process ontology also showed that about 16% were associated with the term “development”, and almost 14% to the term “immune response”.</p>
<a id="article1.body1.sec2.sec2.p3" name="article1.body1.sec2.sec2.p3"></a><p>The gene ontology annotation was also used to characterize the genes identified in each of the 78 tissues separately (details in <a href="#pone.0001880.s004">File S1</a>). For example, “blood coagulation”, “iron ion homeostasis”, “lipid metabolism”, and “gluconeogenesis” were found over-represented in liver with p&lt;1.0E-4 (<a href="#pone.0001880.s004">File S1</a>, table 44.2). Similarly, “male gamete generation” and “spermatogenesis” were found in testis with p&lt;1.0E-18 (<a href="#pone.0001880.s004">File S1</a>, table 55.2). Generally, the selective genes in all the analyzed tissues showed excellent correlation with the known physiological functions.</p>


<h4>Tissue-selective genes and human diseases</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>It has been suggested that slow-evolving housekeeping genes are underrepresented among disease genes, due to a higher chance of embryonic lethality when mutated <a href="#pone.0001880-Winter1">[5]</a>. The 1601 tissue-selective genes were enriched in disease-genes as they were linked to 361 diseases described in the OMIM database (<a href="http://www.ncbi.nlm.nih.gov/sites/entrezdbOMIM">http://www.ncbi.nlm.nih.gov/sites/entrez​dbOMIM</a>). We also observed that, in most of the cases, the 1601 tissue-selective genes are associated to pathological phenotype in the tissues or organs where they are found selectively expressed.</p>
<a id="article1.body1.sec2.sec3.p2" name="article1.body1.sec2.sec3.p2"></a><p>For instance, among the genes we found selectively expressed in fetal heart, GATA4 and NKX2.5 have been associated with Atrial Septal Defect 2 <a href="#pone.0001880-Garg1">[12]</a>, <a href="#pone.0001880-HirayamaYamada1">[13]</a>. In addition, mutations of the gene NKX2.5 have been described in Tetralogy of Fallot <a href="#pone.0001880-Benson1">[14]</a>, <a href="#pone.0001880-Goldmuntz1">[15]</a> and in Atrial Septal Defect with Atrioventricular Conduction Defect <a href="#pone.0001880-Schott1">[16]</a>, <a href="#pone.0001880-Watanabe1">[17]</a>. The placenta-selective gene RASA1 is reported in two diseases characterized by aberrations of blood vessels: the Parkes Weber Syndrome and Capillary Malformation-Arteriovenous Malformation <a href="#pone.0001880-Eerola1">[18]</a>. Several muscle-selective genes have been described in a number of myopathies, as well as several gland-selective genes are associated with syndromes of the endocrine system and metabolic diseases. More extensive listing of diseases is available in the supplemental results (<a href="#pone.0001880.s004">File S1</a>, table 0.5).</p>


<h4>Tissue connectome</h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>We wanted to investigate the hypothesis that tissues and organs sharing tissue-selective genes might present some degree of relatedness.</p>
<a id="article1.body1.sec2.sec4.p2" name="article1.body1.sec2.sec4.p2"></a><p>For this propose, we built a network of tissues that we named connectome. In the connectome, each node represented a tissue and the genes selectively expressed in two or more tissues formed the edges between the nodes. Each edge was thus associated with the number of shared genes.</p>
<a id="article1.body1.sec2.sec4.p3" name="article1.body1.sec2.sec4.p3"></a><p>The number of edges in the network was computed as a function of the number of shared genes between the tissues and three cutoff values (30 or more genes, 20 or more genes, and 5 or more genes) were chosen as representative of different degrees of connectivity (<a href="#pone-0001880-g001">Figure 1d</a>).</p>
<div class="figure" id="pone-0001880-g001"><div class="img"><a name="pone-0001880-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001880.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001880" data-uri="info:doi/10.1371/journal.pone.0001880.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001880.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001880.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001880.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001880.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001880.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001880.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001880.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001880.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001880.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Tissue Connectome.</span></strong></p><a id="article1.body1.sec2.sec4.fig1.caption1.p1" name="article1.body1.sec2.sec4.fig1.caption1.p1"></a><p>Color code: Central Nervous System RED; Peripheral Nervous System ORANGE; Testis YELLOW; Muscles GREEN; immune cells LIGHT BLUE; Immune Organs DARK BLUE; Respiratory System PINK; Pancreas and Islets DARK PINK; Adrenal Gland and Adrenal Cortex LIGHT PINK; Thyroid and Fetal Thyroid SEPIA; Others WHITE. The edges have been drawn between tissue nodes sharing: a) 30 or more genes; b) 20 or more genes; c) 5 or more genes. d) the number of edges as a function of tissue-selective genes shared by two or more tissues. The tissue indexes are reported in <a href="#pone-0001880-t001">Table 1</a>.</p>
<span>doi:10.1371/journal.pone.0001880.g001</span></div><h5>The 30 sharing genes connectome.</h5><a id="article1.body1.sec2.sec4.sec1.p1" name="article1.body1.sec2.sec4.sec1.p1"></a><p>It was possible to observe separate networks of seven central nervous system (CNS) tissues, five testis tissues, twelve immune cells, and six muscles (<a href="#pone-0001880-g001">Figure 1a</a>). Moreover, a chain-like connection was found between fetal lung – fetal liver – liver – kidney. The forebrain structures, consisting of telencephalic and diencephalic structures, clustered together. The central tissue in the CNS cluster was amygdala, which is consistent with the view that amygdala, rather than being a structural and developmental unit, is a collection of adjacent cell groups within the forebrain <a href="#pone.0001880-Swanson1">[19]</a>, <a href="#pone.0001880-Swanson2">[20]</a>. Whole blood had extended connections with several circulating cells, such as T cells, B cells, natural killer cells, monocytes, and BDCA4-dendritic cells. On the other hand, tight connections were also formed between the cells resident in the bone marrow, such as early erythroid cells, B lymphoblasts, endothelial cells, and CD34 clones.</p>

<h5>The 20 sharing genes connectome.</h5><a id="article1.body1.sec2.sec4.sec2.p1" name="article1.body1.sec2.sec4.sec2.p1"></a><p>Two larger sub-networks emerged, constituted respectively of CNS tissues, and immune cells and tissues, muscles, excretory organs, and thyroid (<a href="#pone-0001880-g001">Figure 1b</a>). Tonsil bridged the connection between immune cells and muscles. This makes sense as in the tonsil there are both myoepithelial and immune cells. Tonsil presented also connections with B cells, BDCA4-dendritic cells, and lymph node. Histologic studies of the tonsil showed that they are lymphoid structures consisting mainly of B-lymphocytes, but they are occupied also by T-lymphocytes, activated B-lymphocytes and other cells of the immune system. Tonsil shares histological features with lymph nodes as its cells are supported by a fine network of reticular fibers and high-endothelial venules function in the “homing” of circulating lymphocytes <a href="#pone.0001880-CA1">[21]</a>, <a href="#pone.0001880-AK1">[22]</a>. During the fetal life massive erythropoiesis happens in several tissues such as the liver <a href="#pone.0001880-Timens1">[23]</a>. Fetal liver, in fact, connected early erythroid cells and CD105-endothelial cells that are precursors of circulating cells and which reside in the bone marrow after the birth. The testis tissues still appeared unconnected to any other tissue. The cluster of 15 CNS tissues included structures spanning from the spinal cord to telencephalic structures, forming the center of this network.</p>

<h5>The 5 sharing genes connectome.</h5><a id="article1.body1.sec2.sec4.sec3.p1" name="article1.body1.sec2.sec4.sec3.p1"></a><p>All CNS tissues, including the olfactory bulb clustered together (<a href="#pone-0001880-g001">Figure 1c</a>). In addition, some peripheral nervous system structures joined this cluster. This sub-network was no longer distinct but shared links with other structures, particularly fetal tissues such as heart and lung. In addition, muscles shared links with the CNS cluster, which may be due to the innervation of the muscle samples as well as to the genes involved in ion homeostasis, which are expressed both in neurons and muscular cells. Fetal brain, hippocampus and olfactory bulb bridged the CNS cluster with other tissues. Neurogenesis, production of new neurons, continues in adult hippocampus and olfactory bulb and the genes expressed in these newborn neurons may give them immature characteristics, which are shared by fetal brain and other fetal tissues <a href="#pone.0001880-Gage1">[24]</a>. Hypothalamus and pituitary gland, which are anatomically connected, linked together.</p>



<h4>The hippocampus regulatory gene network</h4>
<a id="article1.body1.sec2.sec5.p1" name="article1.body1.sec2.sec5.p1"></a><p>One of the goals of our analysis was to find clear correspondence between the tissue-selective gene expression programs and specific functions of tissues. Within the CNS, neurogenesis during adult life takes place in the hippocampus during normal and pathological conditions. The analysis of the connectome showed interesting links of the hippocampus with other anatomical structures where cell duplication and differentiation are known to happen. We wanted to test the hypothesis that genes selectively expressed in hippocampus would form a transcriptional network underlying this specific function. For this, we built a network of the hippocampus-selective genes based on their co-citation into the Pubmed database as well as the presence of specific transcription factor binding sites (TFBS) in their promoter regions ( <a href="#pone.0001880.s002">Figure S1</a>). The transcription factor NF-kappaB, which was not among the hippocampus-selective genes, presented an interesting topological position as it had connections with a number of hippocampus-selective genes (<a href="#pone.0001880.s002">Figure S1</a>). Detailed analysis of the promoters of the NF-kappaB interactors revealed the presence of a significantly conserved binding sequence for E2F and, 92–115 bp downstream, one for NF-kappaB (<a href="#pone.0001880.s003">Figure S2</a>). Screening the whole set of known human promoters, we found the E2F-NF-kappaB module in 1901 regulatory sequences, suggesting a common mechanism of transcriptional regulation. The gene ontology clustering of these genes showed significant over-representation of the families “nervous system development”, “cell adhesion”, “transmembrane receptor protein tyrosine kinase signaling pathway”, and “retinoic acid receptor activity” (details in <a href="#pone.0001880.s005">File S2</a>). In addition to the hippocampus-selective genes, also some fetal brain-selective, amygdala-selective, and prefrontal cortex-selective genes could be regulated by the E2F-NF-kappaB module. All these areas have been extensively investigated for neurogenesis <a href="#pone.0001880-Gould1">[25]</a>.</p>

</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p>We have integrated microarray data produced in several laboratories for exploring the tissue-selective expression patterns in 78 normal human tissues.</p>
<a id="article1.body1.sec3.p2" name="article1.body1.sec3.p2"></a><p>One of the interests about the tissue-specific genes concerns their functional role in normal and pathological conditions. We observed that the group of tissue-selective genes is enriched in “signal transducer activity”, “receptor binding”, and “extracellular region”, as well as “development” and “immune response” functional families. Our results are largely concordant with the findings of Winter and collaborators, who have reported that genes encoding secreted proteins highly correlate with tissue specificity <a href="#pone.0001880-Winter1">[5]</a>. Freilich and collaborators also reported that genes with tissue-specific expression patterns mainly encode for regulatory proteins involved in signal transduction activity <a href="#pone.0001880-Freilich1">[26]</a>. In addition, they observed that the tissue-specific group is possibly enriched in transcription factors encoding genes <a href="#pone.0001880-Freilich1">[26]</a>. We found only 59 transcription factor genes expressed in a tissue-selective manner. Accordingly, Yu and collaborators have reported that ubiquitously expressed transcription factors can combine with other factors contributing to tissue-specificity. In addition, they observed that individual transcription factors can participate in tissue-specific gene regulation by interacting with distinct partners in different tissues <a href="#pone.0001880-Yu1">[27]</a>. We propose that many transcription factors are evenly expressed in many tissues and that several stimuli mediated by tissue-specific signal transduction machineries mediate the functional activation of specific combinations of them at the protein level only in certain tissues and in defined temporal windows.</p>
<a id="article1.body1.sec3.p3" name="article1.body1.sec3.p3"></a><p>The connectome shows a novel and intriguing way to investigate the relatedness of human tissues and organs. While interpreting these results, it should be also taken into account that many human tissues have a complex architecture, as they consist of a certain number of specialized cells with variant transcriptional profiles. Microarrays can reliably detect cRNA species at the concentration of a few pico-molars, but it can be problematic to assign a certain gene expression event to the correct cell subpopulation of complex tissues. Nevertheless, we believe that the tissues should be always studied as functional entities and their global gene expression should be target of interest. When thinking of the liver, for instance, it should be considered that the identity of this organ is given by the combination of expression programs in several kinds of cells, more than just hepatocytes. Moreover, it is reasonable to think that in samples of many tissues some amount of blood cells are also present. This can easily explain the wide connections that circulating cells form with other kinds of anatomical structures in terms of shared gene expression. However, we observed a very tight intra-connectivity within the groups of nervous tissues, blood cells, testis tissues, and muscles, suggesting that the identity of these anatomical structures is determined by the differential expression of many genes. This is also evident from several clustering analyses we preformed on the data (details in <a href="#pone.0001880.s006">File S3</a>). The connectome of tissues also shows interesting interactions largely explained by the functional and morphological similarity of some tissues. It is the case, for instance, of the connection between the tonsil and lymph node. In other cases, the topological features of certain tissues in the network are suggestive of developmental mechanisms, as for the central position of the amygdala within the central nervous system connectome.</p>
<a id="article1.body1.sec3.p4" name="article1.body1.sec3.p4"></a><p>Increasing attention is being oriented to the inference of transcriptional regulatory networks from high throughput gene expression screenings. These approaches aim to link gene expression data to the activity of transcription factors in cause-effect models. Some effort has been already put also into the investigation of regulatory gene networks of tissue-specific genes having a central role in the physiology and development of specific anatomical structures <a href="#pone.0001880-Wasserman1">[28]</a>, <a href="#pone.0001880-Krivan1">[29]</a>, <a href="#pone.0001880-Olson1">[30]</a>. We investigated in detail the expression patterns that might play a role in determining some functional aspect of the hippocampus. We computationally predicted a promoter module formed by conserved consensus motifs for the transcription factors E2F and NF-kappaB present on 1901 human known promoters. The gene ontology classification suggests that these genes are directly involved in neurogenesis and central nervous system development.</p>
<a id="article1.body1.sec3.p5" name="article1.body1.sec3.p5"></a><p>The E2F family of transcription factors, by interaction with several partners such as pRb, p107 or p130, are thought to regulate the cell cycle <a href="#pone.0001880-Attwooll1">[31]</a>, <a href="#pone.0001880-Trimarchi1">[32]</a> and trigger signals that also either promote cellular growth, cell cycle exit, or terminal differentiation in neurons <a href="#pone.0001880-Trimarchi1">[32]</a>, <a href="#pone.0001880-Liu1">[33]</a>. NF-kappaB has been described as playing an important role in synaptic activity and plasticity, neuroprotection, and in behavioral aspects of learning and memory <a href="#pone.0001880-Meffert1">[34]</a>. Moreover, members of NF-kappaB family have been found to be expressed in areas of active neurogenesis in post-natal and adult mouse brains <a href="#pone.0001880-DenisDonini1">[35]</a>.</p>
<a id="article1.body1.sec3.p6" name="article1.body1.sec3.p6"></a><p>Altogether, these results expand our understanding of how gene expression programs determine the functional identity of human tissues.</p>
</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Methods"></a><h3>Methods</h3>
<h4>Data collection</h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p>Total number of 195 Affymetrix HG-U133A CEL files was collected from the GEO database (<a href="http://www.ncbi.nlm.nih.gov/geo/">http://www.ncbi.nlm.nih.gov/geo/</a>) from 6 different data sets. All the gathered arrays had been hybridized to normal adult or fetal human tissues or cell types for a total of 78 different classes (<a href="#pone-0001880-t001">Table 1</a>). Data were selected according to the following criteria: i) all the experiments had been documented according to the MIAME standard; ii) all the arrays had been hybridized with samples isolated from human tissues and experiments were not carried out with cell lines; iii) all the samples came from healthy control subjects or from reference RNA samples; iv) the raw array images (CEL files) were available for download; and v) the Affymetrix chipset used for the hybridization was the human HGU-133A. A quality check of the data was performed using the recommendations of the manufacturer. Altogether, 195 sets of individual array data were used for further analysis.</p>
<div class="figure" id="pone-0001880-t001"><div class="img"><a name="pone-0001880-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001880.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001880" data-uri="info:doi/10.1371/journal.pone.0001880.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001880.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001880.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001880.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001880.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001880.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001880.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001880.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001880.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001880.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>Summary of the data set analyzed.</span></strong></p><span>doi:10.1371/journal.pone.0001880.t001</span></div>

<h4>Data Pre-processing</h4>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1"></a><p>Sequence-based re-annotation of the Affymetrix probes on an HGU-133A chipset <a href="#pone.0001880-Dai1">[9]</a> according to the latest release of the Entrez Gene database was used (<a href="http://www.ncbi.nlm.nih.gov/sites/entrezdbgene">http://www.ncbi.nlm.nih.gov/sites/entrez​dbgene</a>). The expression values for each gene were calculated using the RMA algorithm <a href="#pone.0001880-Irizarry1">[36]</a>.</p>


<h4>Tissue-selectivity analysis</h4>
<a id="article1.body1.sec4.sec3.p1" name="article1.body1.sec4.sec3.p1"></a><p>A tissue-selectivity score was computed for each tissue-gene pair from the expression data matrix. Permutation test was performed to define a significance threshold. Details in <a href="#pone.0001880.s001">Table S1</a>.</p>


<h4>Gene ontology analysis</h4>
<a id="article1.body1.sec4.sec4.p1" name="article1.body1.sec4.sec4.p1"></a><p>Fisher's exact test was performed in order to select over-represented gene ontology classes in the tissue-selective genes. The functional families presenting p&lt;0.01 were considered as significantly represented.</p>


<h4>Gene network and promoter analysis</h4>
<a id="article1.body1.sec4.sec5.p1" name="article1.body1.sec4.sec5.p1"></a><p>The hippocampus-selective genes were processed in the software Bibiosphere to build up gene networks based on their co-citation in the literature as well as the presence of TFBS for known transcription factors in their promoter regions (<a href="http://www.genomatix.de/products/BiblioSphere/">http://www.genomatix.de/products/BiblioS​phere/</a>). Because of the extensive connectivity of NF-KappaB within the network, the genes presenting a significant TFBS for NF-KappaB were selected for further analysis. The promoter sequences of these genes were retrieved using the Gene2Promoter software (<a href="http://www.genomatix.de/online">http://www.genomatix.de/online</a> help/help eldorado/Gene2Promoter Intro.html) and analyzed with FrameWorker (<a href="http://www.genomatix.de/online">http://www.genomatix.de/online</a> help/help gems/FrameWorker.html) to search for common models containing at least two TFBS. Finally, the significant model constituted by E2F and NF-KappaB was screened for in the whole set of known human promoters using ModelInspector (<a href="http://www.genomatix.de/online">http://www.genomatix.de/online</a> help/help fastm/modelinspector help.html).</p>

</div>

<div id="section5" class="section"><a id="s5" name="s5" toc="s5" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0001880"></div><a name="pone.0001880.s001" id="pone.0001880.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001880.s001">Table S1. </a></strong></p><p class="siDoi">doi:10.1371/journal.pone.0001880.s001</p><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1"></a><p class="postSiDOI">(0.06 MB DOC)</p>
<a name="pone.0001880.s002" id="pone.0001880.s002"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001880.s002">Figure S1. </a></strong></p><p class="siDoi">doi:10.1371/journal.pone.0001880.s002</p><a id="article1.body1.sec5.supplementary-material2.caption1.p1" name="article1.body1.sec5.supplementary-material2.caption1.p1"></a><p class="postSiDOI">(0.12 MB PDF)</p>
<a name="pone.0001880.s003" id="pone.0001880.s003"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001880.s003">Figure S2. </a></strong></p><p class="siDoi">doi:10.1371/journal.pone.0001880.s003</p><a id="article1.body1.sec5.supplementary-material3.caption1.p1" name="article1.body1.sec5.supplementary-material3.caption1.p1"></a><p class="postSiDOI">(0.04 MB PDF)</p>
<a name="pone.0001880.s004" id="pone.0001880.s004"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001880.s004">File S1. </a></strong></p><p class="siDoi">doi:10.1371/journal.pone.0001880.s004</p><a id="article1.body1.sec5.supplementary-material4.caption1.p1" name="article1.body1.sec5.supplementary-material4.caption1.p1"></a><p class="postSiDOI">(7.50 MB PDF)</p>
<a name="pone.0001880.s005" id="pone.0001880.s005"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001880.s005">File S2. </a></strong></p><p class="siDoi">doi:10.1371/journal.pone.0001880.s005</p><a id="article1.body1.sec5.supplementary-material5.caption1.p1" name="article1.body1.sec5.supplementary-material5.caption1.p1"></a><p class="postSiDOI">(0.82 MB PDF)</p>
<a name="pone.0001880.s006" id="pone.0001880.s006"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001880.s006">File S3. </a></strong></p><p class="siDoi">doi:10.1371/journal.pone.0001880.s006</p><a id="article1.body1.sec5.supplementary-material6.caption1.p1" name="article1.body1.sec5.supplementary-material6.caption1.p1"></a><p class="postSiDOI">(0.17 MB PDF)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>The authors would like to thank Dr. John R. Walker (Novartis Research Foundation, USA) for kindly providing the raw data for the GEO dataset GDS596, Dr. Alan Schulman (University of Helsinki, Finland) and Dr. Roy Siddall (University of Helsinki, Finland) for comments on the manuscript.</p>
<a id="article1.back1.ack1.p2" name="article1.back1.ack1.p2"></a><p>The authors are also grateful to the Finnish IT Centre for Science for providing the supercomputing facilities, Dr. Martin Seifert (Genomatix Software GmbH, Germany) for helpful advice concerning the Genomatix software.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: DG. Analyzed the data: EC DG PS AD TR LN PA. Wrote the paper: EC DG PS AD PA.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0001880-Vinogradov1" id="pone.0001880-Vinogradov1"></a>Vinogradov AE (2004) Compactness of human housekeeping genes: selection for economy or genomic design? Trends Genet  20(5): 248–253.  <ul class="find" data-citedArticleID="1081886" data-doi="10.1016/j.tig.2004.03.006"><li><a href="http://dx.doi.org/10.1016/j.tig.2004.03.006" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Compactness+of+human+housekeeping+genes%3A+selection+for+economy+or+genomic+design%3F" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Compactness+of+human+housekeeping+genes%3A+selection+for+economy+or+genomic+design%3F%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0001880-CastilloDavis1" id="pone.0001880-CastilloDavis1"></a>Castillo-Davis CI, Mekhedov SL, Hartl DL, Koonin EV, Kondrashov FA (2002) Selection for short introns in highly expressed genes. Nat Genet  31(4): 415–418.  <ul class="find" data-citedArticleID="1081836" data-doi="10.1038/ng940"><li><a href="http://dx.doi.org/10.1038/ng940" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Selection+for+short+introns+in+highly+expressed+genes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Selection+for+short+introns+in+highly+expressed+genes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0001880-Vinogradov2" id="pone.0001880-Vinogradov2"></a>Vinogradov AE (2003) Isochores and tissue-specificity. Nucleic Acids Res  31(17): 5212–5220.  <ul class="find" data-citedArticleID="1081888" data-doi="10.1093/nar/gkg699"><li><a href="http://dx.doi.org/10.1093/nar/gkg699" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Isochores+and+tissue-specificity." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Isochores+and+tissue-specificity.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0001880-Duret1" id="pone.0001880-Duret1"></a>Duret L, Mouchiroud D (2000) Determinants of substitution rates in mammalian genes: expression pattern affects selection intensity but not mutation rate. Mol Biol Evol  17: 68–74.  <ul class="find" data-citedArticleID="1081842" data-doi="10.1093/oxfordjournals.molbev.a026239"><li><a href="http://dx.doi.org/10.1093/oxfordjournals.molbev.a026239" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Determinants+of+substitution+rates+in+mammalian+genes%3A+expression+pattern+affects+selection+intensity+but+not+mutation+rate." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Determinants+of+substitution+rates+in+mammalian+genes%3A+expression+pattern+affects+selection+intensity+but+not+mutation+rate.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0001880-Winter1" id="pone.0001880-Winter1"></a>Winter EE, Goodstadt L, Ponting CP (2004) Elevated rates of protein secretion, evolution, and disease among tissue-specific genes. Genome research  14: 54–61.  <ul class="find" data-citedArticleID="1081894" data-doi="10.1101/gr.1924004"><li><a href="http://dx.doi.org/10.1101/gr.1924004" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Elevated+rates+of+protein+secretion%2C+evolution%2C+and+disease+among+tissue-specific+genes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Elevated+rates+of+protein+secretion%2C+evolution%2C+and+disease+among+tissue-specific+genes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0001880-Liang1" id="pone.0001880-Liang1"></a>Liang S, Li Y, Be X, Howes S, Liu W (2006) Detecting and profiling tissue-selective genes. Physiological genomics  26(2): 158–162.  <ul class="find" data-citedArticleID="1081862" data-doi="10.1152/physiolgenomics.00313.2005"><li><a href="http://dx.doi.org/10.1152/physiolgenomics.00313.2005" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Detecting+and+profiling+tissue-selective+genes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Detecting+and+profiling+tissue-selective+genes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0001880-Lipshutz1" id="pone.0001880-Lipshutz1"></a>Lipshutz RJ, Fodor SP, Gingeras TR, Lockhart DJ (1999) High density synthetic oligonucleotide arrays. Nature genetics  21: (1 Suppl)20–24.  <ul class="find" data-citedArticleID="1081864" data-doi="10.1038/4447"><li><a href="http://dx.doi.org/10.1038/4447" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=High+density+synthetic+oligonucleotide+arrays." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22High+density+synthetic+oligonucleotide+arrays.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0001880-Su1" id="pone.0001880-Su1"></a>Su AI, Wiltshire T, Batalov S, Lapp H, Ching KA, et al.  (2004) A gene atlas of the mouse and human protein-encoding transcriptomes. Proceedings of the National Academy of Sciences of the United States of America  101(16): 6062–6067.  <ul class="find" data-citedArticleID="1081876" data-doi="10.1073/pnas.0400782101"><li><a href="http://dx.doi.org/10.1073/pnas.0400782101" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+gene+atlas+of+the+mouse+and+human+protein-encoding+transcriptomes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+gene+atlas+of+the+mouse+and+human+protein-encoding+transcriptomes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0001880-Dai1" id="pone.0001880-Dai1"></a>Dai M, Wang P, Boyd AD, Kostov G, Athey B, et al.  (2005) Evolving gene/transcript definitions significantly alter the interpretation of GeneChip data. Nucleic acids research  33(20): e175.  <ul class="find" data-citedArticleID="1081838" data-doi="10.1093/nar/gni179"><li><a href="http://dx.doi.org/10.1093/nar/gni179" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evolving+gene%2Ftranscript+definitions+significantly+alter+the+interpretation+of+GeneChip+data." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evolving+gene%2Ftranscript+definitions+significantly+alter+the+interpretation+of+GeneChip+data.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0001880-Sandberg1" id="pone.0001880-Sandberg1"></a>Sandberg R, Larsson O (2007) Improved precision and accuracy for microarrays using updated probe set definitions. BMC Bioinformatics  8: 48.  <ul class="find" data-citedArticleID="1081872"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Improved+precision+and+accuracy+for+microarrays+using+updated+probe+set+definitions.&amp;auth=&amp;atitle=Improved+precision+and+accuracy+for+microarrays+using+updated+probe+set+definitions." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Improved+precision+and+accuracy+for+microarrays+using+updated+probe+set+definitions." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Improved+precision+and+accuracy+for+microarrays+using+updated+probe+set+definitions.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0001880-Carter1" id="pone.0001880-Carter1"></a>Carter SL, Eklund AC, Mecham BH, Kohane IS, Szallasi Z (2005) Redefinition of Affymetrix probe sets by sequence overlap with cDNA microarray probes reduces cross-platform inconsistencies in cancer-associated gene expression measurements. BMC Bioinformatics  6: 107.  <ul class="find" data-citedArticleID="1081834"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Redefinition+of+Affymetrix+probe+sets+by+sequence+overlap+with+cDNA+microarray+probes+reduces+cross-platform+inconsistencies+in+cancer-associated+gene+expression+measurements.&amp;auth=&amp;atitle=Redefinition+of+Affymetrix+probe+sets+by+sequence+overlap+with+cDNA+microarray+probes+reduces+cross-platform+inconsistencies+in+cancer-associated+gene+expression+measurements." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Redefinition+of+Affymetrix+probe+sets+by+sequence+overlap+with+cDNA+microarray+probes+reduces+cross-platform+inconsistencies+in+cancer-associated+gene+expression+measurements." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Redefinition+of+Affymetrix+probe+sets+by+sequence+overlap+with+cDNA+microarray+probes+reduces+cross-platform+inconsistencies+in+cancer-associated+gene+expression+measurements.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0001880-Garg1" id="pone.0001880-Garg1"></a>Garg V, Kathiriya IS, Barnes R, Schluterman MK, King IN, et al.  (2003) GATA4 mutations cause human congenital heart defects and reveal an interaction with TBX5. Nature  424(6947): 443–447.  <ul class="find" data-citedArticleID="1081850" data-doi="10.1038/nature01827"><li><a href="http://dx.doi.org/10.1038/nature01827" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=GATA4+mutations+cause+human+congenital+heart+defects+and+reveal+an+interaction+with+TBX5." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22GATA4+mutations+cause+human+congenital+heart+defects+and+reveal+an+interaction+with+TBX5.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0001880-HirayamaYamada1" id="pone.0001880-HirayamaYamada1"></a>Hirayama-Yamada K, Kamisago M, Akimoto K, Aotsuka H, Nakamura Y, et al.  (2005) Phenotypes with GATA4 or NKX2.5 mutations in familial atrial septal defect. American journal of medical genetics  135: 47–52.  <ul class="find" data-citedArticleID="1081856" data-doi="10.1002/ajmg.a.30684"><li><a href="http://dx.doi.org/10.1002/ajmg.a.30684" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Phenotypes+with+GATA4+or+NKX2.5+mutations+in+familial+atrial+septal+defect." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Phenotypes+with+GATA4+or+NKX2.5+mutations+in+familial+atrial+septal+defect.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0001880-Benson1" id="pone.0001880-Benson1"></a>Benson DW, Silberbach GM, Kavanaugh-McHugh A, Cottrill C, Zhang Y, et al.  (1999) Mutations in the cardiac transcription factor NKX2.5 affect diverse cardiac developmental pathways. J Clin Invest  104(11): 1567–1573.  <ul class="find" data-citedArticleID="1081830" data-doi="10.1172/jci8154"><li><a href="http://dx.doi.org/10.1172/jci8154" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mutations+in+the+cardiac+transcription+factor+NKX2.5+affect+diverse+cardiac+developmental+pathways." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mutations+in+the+cardiac+transcription+factor+NKX2.5+affect+diverse+cardiac+developmental+pathways.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0001880-Goldmuntz1" id="pone.0001880-Goldmuntz1"></a>Goldmuntz E, Geiger E, Benson DW (2001) NKX2.5 mutations in patients with tetralogy of fallot. Circulation  104(21): 2565–2568.  <ul class="find" data-citedArticleID="1081852" data-doi="10.1161/hc4601.098427"><li><a href="http://dx.doi.org/10.1161/hc4601.098427" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=NKX2.5+mutations+in+patients+with+tetralogy+of+fallot." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22NKX2.5+mutations+in+patients+with+tetralogy+of+fallot.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0001880-Schott1" id="pone.0001880-Schott1"></a>Schott JJ, Benson DW, Basson CT, Pease W, Silberbach GM, et al.  (1998) Congenital heart disease caused by mutations in the transcription factor NKX2-5. Science  281(5373): 108–111.  <ul class="find" data-citedArticleID="1081874" data-doi="10.1126/science.281.5373.108"><li><a href="http://dx.doi.org/10.1126/science.281.5373.108" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Congenital+heart+disease+caused+by+mutations+in+the+transcription+factor+NKX2-5." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Congenital+heart+disease+caused+by+mutations+in+the+transcription+factor+NKX2-5.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0001880-Watanabe1" id="pone.0001880-Watanabe1"></a>Watanabe Y, Benson DW, Yano S, Akagi T, Yoshino M, Murray JC (2002) Two novel frameshift mutations in NKX2.5 result in novel features including visceral inversus and sinus venosus type ASD. J Med Genet  39(11): 807–811.  <ul class="find" data-citedArticleID="1081892" data-doi="10.1136/jmg.39.11.807"><li><a href="http://dx.doi.org/10.1136/jmg.39.11.807" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Two+novel+frameshift+mutations+in+NKX2.5+result+in+novel+features+including+visceral+inversus+and+sinus+venosus+type+ASD." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Two+novel+frameshift+mutations+in+NKX2.5+result+in+novel+features+including+visceral+inversus+and+sinus+venosus+type+ASD.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0001880-Eerola1" id="pone.0001880-Eerola1"></a>Eerola I, Boon LM, Mulliken JB, Burrows PE, Dompmartin A, et al.  (2003) Capillary malformation-arteriovenous malformation, a new clinical and genetic disorder caused by RASA1 mutations. Am J Hum Genet  73(6): 1240–1249.  <ul class="find" data-citedArticleID="1081844" data-doi="10.1086/379793"><li><a href="http://dx.doi.org/10.1086/379793" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Capillary+malformation-arteriovenous+malformation%2C+a+new+clinical+and+genetic+disorder+caused+by+RASA1+mutations." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Capillary+malformation-arteriovenous+malformation%2C+a+new+clinical+and+genetic+disorder+caused+by+RASA1+mutations.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0001880-Swanson1" id="pone.0001880-Swanson1"></a>Swanson LW (2003) The amygdala and its place in the cerebral hemisphere. Annals of the New York Academy of Sciences  985: 174–184.  <ul class="find" data-citedArticleID="1081878" data-doi="10.1111/j.1749-6632.2003.tb07081.x"><li><a href="http://dx.doi.org/10.1111/j.1749-6632.2003.tb07081.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+amygdala+and+its+place+in+the+cerebral+hemisphere." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+amygdala+and+its+place+in+the+cerebral+hemisphere.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0001880-Swanson2" id="pone.0001880-Swanson2"></a>Swanson LW, Petrovich GD (1998) What is the amygdala? Trends in neurosciences  21(8): 323–331.  <ul class="find" data-citedArticleID="1081880" data-doi="10.1016/s0166-2236(98)01265-x"><li><a href="http://dx.doi.org/10.1016/s0166-2236(98)01265-x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=What+is+the+amygdala%3F" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22What+is+the+amygdala%3F%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0001880-CA1" id="pone.0001880-CA1"></a>CA Janeway MWMS P Travers (2001) Immunobiology, 5th edition. New York: Garland.   <ul class="find-nolinks"></ul></li><li><span class="label">22.
              </span><a name="pone.0001880-AK1" id="pone.0001880-AK1"></a>AK Abbas JP AH Lichtman (2000) Cellular and Molecular Immunology, 4th edition. Philadelphia: Saunders.   <ul class="find-nolinks"></ul></li><li><span class="label">23.
              </span><a name="pone.0001880-Timens1" id="pone.0001880-Timens1"></a>Timens W, Kamps WA (1997) Hemopoiesis in human fetal and embryonic liver. Microsc Res Tech  39(5): 387–397.  <ul class="find" data-citedArticleID="1081882" data-doi="10.1002/(sici)1097-0029(19971201)39:5<387::aid-jemt1&gt;3.0.co;2-e"><li><a href="http://dx.doi.org/10.1002/(sici)1097-0029(19971201)39:5<387::aid-jemt1&gt;3.0.co;2-e" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Hemopoiesis+in+human+fetal+and+embryonic+liver." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Hemopoiesis+in+human+fetal+and+embryonic+liver.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0001880-Gage1" id="pone.0001880-Gage1"></a>Gage FH (2000) Mammalian neural stem cells. Science  287(5457): 1433–1438.  <ul class="find" data-citedArticleID="1081848" data-doi="10.1126/science.287.5457.1433"><li><a href="http://dx.doi.org/10.1126/science.287.5457.1433" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mammalian+neural+stem+cells." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mammalian+neural+stem+cells.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0001880-Gould1" id="pone.0001880-Gould1"></a>Gould E (2007) How widespread is adult neurogenesis in mammals? Nat Rev Neurosci  8(6): 481–488.  <ul class="find" data-citedArticleID="1081854" data-doi="10.1038/nrn2147"><li><a href="http://dx.doi.org/10.1038/nrn2147" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=How+widespread+is+adult+neurogenesis+in+mammals%3F" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22How+widespread+is+adult+neurogenesis+in+mammals%3F%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0001880-Freilich1" id="pone.0001880-Freilich1"></a>Freilich S, Massingham T, Bhattacharyya S, Ponsting H, Lyons PA, et al.  (2005) Relationship between the tissue-specificity of mouse gene expression and the evolutionary origin and function of the proteins. Genome biology  6(7): R56.  <ul class="find" data-citedArticleID="1081846" data-doi="10.1186/gb-2005-6-7-r56"><li><a href="http://dx.doi.org/10.1186/gb-2005-6-7-r56" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Relationship+between+the+tissue-specificity+of+mouse+gene+expression+and+the+evolutionary+origin+and+function+of+the+proteins." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Relationship+between+the+tissue-specificity+of+mouse+gene+expression+and+the+evolutionary+origin+and+function+of+the+proteins.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0001880-Yu1" id="pone.0001880-Yu1"></a>Yu X, Lin J, Zack DJ, Qian J (2006) Computational analysis of tissue-specific combinatorial gene regulation: predicting interaction between transcription factors in human tissues. Nucleic Acids Res  34(17): 4925–4936.  <ul class="find" data-citedArticleID="1081896" data-doi="10.1093/nar/gkl595"><li><a href="http://dx.doi.org/10.1093/nar/gkl595" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Computational+analysis+of+tissue-specific+combinatorial+gene+regulation%3A+predicting+interaction+between+transcription+factors+in+human+tissues." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Computational+analysis+of+tissue-specific+combinatorial+gene+regulation%3A+predicting+interaction+between+transcription+factors+in+human+tissues.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0001880-Wasserman1" id="pone.0001880-Wasserman1"></a>Wasserman WW, Fickett JW (1998) Identification of regulatory regions which confer muscle-specific gene expression. J Mol Biol  278: 167–181.  <ul class="find" data-citedArticleID="1081890" data-doi="10.1006/jmbi.1998.1700"><li><a href="http://dx.doi.org/10.1006/jmbi.1998.1700" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Identification+of+regulatory+regions+which+confer+muscle-specific+gene+expression." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Identification+of+regulatory+regions+which+confer+muscle-specific+gene+expression.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0001880-Krivan1" id="pone.0001880-Krivan1"></a>Krivan W, Wasserman WW (2001) A predictive model for regulatory sequences directing liver-specific transcription. Genome Res  11(9): 1559–1566.  <ul class="find" data-citedArticleID="1081860" data-doi="10.1101/gr.180601"><li><a href="http://dx.doi.org/10.1101/gr.180601" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+predictive+model+for+regulatory+sequences+directing+liver-specific+transcription." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+predictive+model+for+regulatory+sequences+directing+liver-specific+transcription.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0001880-Olson1" id="pone.0001880-Olson1"></a>Olson EN (2006) Gene regulatory networks in the evolution and development of the heart. Science  313(5795): 1922–1927.  <ul class="find" data-citedArticleID="1081870" data-doi="10.1126/science.1132292"><li><a href="http://dx.doi.org/10.1126/science.1132292" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Gene+regulatory+networks+in+the+evolution+and+development+of+the+heart." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Gene+regulatory+networks+in+the+evolution+and+development+of+the+heart.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0001880-Attwooll1" id="pone.0001880-Attwooll1"></a>Attwooll C, Lazzerini Denchi E, Helin K (2004) The E2F family: specific functions and overlapping interests. EMBO J 8;  23(24): 4709–16.  <ul class="find" data-citedArticleID="1081828" data-doi="10.1038/sj.emboj.7600481"><li><a href="http://dx.doi.org/10.1038/sj.emboj.7600481" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+E2F+family%3A+specific+functions+and+overlapping+interests." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+E2F+family%3A+specific+functions+and+overlapping+interests.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0001880-Trimarchi1" id="pone.0001880-Trimarchi1"></a>Trimarchi JM, Lees JA (2002) Sibling rivalry in the E2F family. Nat Rev Mol Cell Biol  3: 11–20.  <ul class="find" data-citedArticleID="1081884" data-doi="10.1038/nrm714"><li><a href="http://dx.doi.org/10.1038/nrm714" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Sibling+rivalry+in+the+E2F+family." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Sibling+rivalry+in+the+E2F+family.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0001880-Liu1" id="pone.0001880-Liu1"></a>Liu DX, Greene LA (2001) Regulation of neuronal survival and death by E2F-dependent gene repression and derepression. Neuron  32(3): 425–438.  <ul class="find" data-citedArticleID="1081866" data-doi="10.1016/s0896-6273(01)00495-0"><li><a href="http://dx.doi.org/10.1016/s0896-6273(01)00495-0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Regulation+of+neuronal+survival+and+death+by+E2F-dependent+gene+repression+and+derepression." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Regulation+of+neuronal+survival+and+death+by+E2F-dependent+gene+repression+and+derepression.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0001880-Meffert1" id="pone.0001880-Meffert1"></a>Meffert MK (2005) Baltimore D: Physiological functions for brain NF-kappaB. Trends Neurosci  28: 37–43.  <ul class="find" data-citedArticleID="1081868" data-doi="10.1016/j.tins.2004.11.002"><li><a href="http://dx.doi.org/10.1016/j.tins.2004.11.002" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Baltimore+D%3A+Physiological+functions+for+brain+NF-kappaB." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Baltimore+D%3A+Physiological+functions+for+brain+NF-kappaB.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0001880-DenisDonini1" id="pone.0001880-DenisDonini1"></a>Denis-Donini S, Caprini A, Frassoni C, Grilli M (2005) Members of the NF-kappaB family expressed in zones of active neurogenesis in the postnatal and adult mouse brain. Brain Res Dev Brain Res  154: 81–89.  <ul class="find" data-citedArticleID="1081840" data-doi="10.1016/j.devbrainres.2004.10.010"><li><a href="http://dx.doi.org/10.1016/j.devbrainres.2004.10.010" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Members+of+the+NF-kappaB+family+expressed+in+zones+of+active+neurogenesis+in+the+postnatal+and+adult+mouse+brain." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Members+of+the+NF-kappaB+family+expressed+in+zones+of+active+neurogenesis+in+the+postnatal+and+adult+mouse+brain.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">36.
              </span><a name="pone.0001880-Irizarry1" id="pone.0001880-Irizarry1"></a>Irizarry RA, Hobbs B, Collin F, Beazer-Barclay YD, Antonellis KJ, et al.  (2003) Exploration, normalization, and summaries of high density oligonucleotide array probe level data. Biostatistics  4(2): 249–264.  <ul class="find" data-citedArticleID="1081858" data-doi="10.1093/biostatistics/4.2.249"><li><a href="http://dx.doi.org/10.1093/biostatistics/4.2.249" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Exploration%2C+normalization%2C+and+summaries+of+high+density+oligonucleotide+array+probe+level+data." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Exploration%2C+normalization%2C+and+summaries+of+high+density+oligonucleotide+array+probe+level+data.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001880.XML" value="78276"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001880.PDF" value="518079"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001880.g001.PNG_L" value="586630"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001880.g001.PNG_M" value="114464"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001880.g001.PNG_S" value="9750"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001880.g001.TIF" value="755438"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001880.g001.PNG_I" value="36031"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001880.t001.PNG_L" value="328553"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001880.t001.PNG_M" value="63579"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001880.t001.PNG_S" value="25914"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001880.t001.TIF" value="2068656"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001880.t001.PNG_I" value="203688"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001880.s001.DOC" value="55296"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001880.s002.PDF" value="119927"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001880.s003.PDF" value="35598"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001880.s004.PDF" value="7504586"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001880.s005.PDF" value="818496"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001880.s006.PDF" value="169441"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001880&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001880" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001880&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0001880&volume=&issue=&title=Physiology, Pathology and Relatedness of Human Tissues from Gene Expression Meta-Analysis&author_name=Dario%20Greco%2C%20Panu%20Somervuo%2C%20Antonio%20Di%20Lieto%2C%20Tuomas%20Raitila%2C%20Lucio%20Nitsch%2C%20Eero%20Castr%C3%A9n%2C%20Petri%20Auvinen&start_page=1&end_page=6" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001880" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001880" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0001880" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001880&amp;t=Physiology%2C%20Pathology%20and%20Relatedness%20of%20Human%20Tissues%20from%20Gene%20Expression%20Meta-Analysis" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001880&title=Physiology%2C%20Pathology%20and%20Relatedness%20of%20Human%20Tissues%20from%20Gene%20Expression%20Meta-Analysis&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001880&amp;title=Physiology%2C%20Pathology%20and%20Relatedness%20of%20Human%20Tissues%20from%20Gene%20Expression%20Meta-Analysis" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001880" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0001880&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Physiology, Pathology and Relatedness of Human Tissues from Gene Expression Meta-Analysis';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0001880';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Physiology%2C%20Pathology%20and%20Relatedness%20of%20Human%20Tissues%20from%20Gene%20Expression%20Meta-Analysis http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001880" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0001880" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">

















          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Central+nervous+system%22" title="Search for articles in the subject area:'Central nervous system'"><div class="flagText">Central nervous system</div></a>
              <div data-categoryid="47337" data-articleid="26584"
                   data-categoryname="Central nervous system"
                   class="flagImage" title="Flag 'Central nervous system' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Gene+expression%22" title="Search for articles in the subject area:'Gene expression'"><div class="flagText">Gene expression</div></a>
              <div data-categoryid="17515" data-articleid="26584"
                   data-categoryname="Gene expression"
                   class="flagImage" title="Flag 'Gene expression' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Gene+ontologies%22" title="Search for articles in the subject area:'Gene ontologies'"><div class="flagText">Gene ontologies</div></a>
              <div data-categoryid="40557" data-articleid="26584"
                   data-categoryname="Gene ontologies"
                   class="flagImage" title="Flag 'Gene ontologies' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Genetic+networks%22" title="Search for articles in the subject area:'Genetic networks'"><div class="flagText">Genetic networks</div></a>
              <div data-categoryid="31807" data-articleid="26584"
                   data-categoryname="Genetic networks"
                   class="flagImage" title="Flag 'Genetic networks' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Hippocampus%22" title="Search for articles in the subject area:'Hippocampus'"><div class="flagText">Hippocampus</div></a>
              <div data-categoryid="47339" data-articleid="26584"
                   data-categoryname="Hippocampus"
                   class="flagImage" title="Flag 'Hippocampus' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Muscle+tissue%22" title="Search for articles in the subject area:'Muscle tissue'"><div class="flagText">Muscle tissue</div></a>
              <div data-categoryid="49177" data-articleid="26584"
                   data-categoryname="Muscle tissue"
                   class="flagImage" title="Flag 'Muscle tissue' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Tonsils%22" title="Search for articles in the subject area:'Tonsils'"><div class="flagText">Tonsils</div></a>
              <div data-categoryid="50317" data-articleid="26584"
                   data-categoryname="Tonsils"
                   class="flagImage" title="Flag 'Tonsils' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Transcription+factors%22" title="Search for articles in the subject area:'Transcription factors'"><div class="flagText">Transcription factors</div></a>
              <div data-categoryid="17335" data-articleid="26584"
                   data-categoryname="Transcription factors"
                   class="flagImage" title="Flag 'Transcription factors' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=87'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=1989'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=7390&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>

<div class="block sidebar-comments">
    <div class="header">
        <h3>Comments</h3>
    </div>
      <p><a href="/annotation/listThread.action?root=13093">Referee comments: Referee 1</a><br>Posted by PLoS_ONE_Group</p>
</div>

</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
