

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Monitoring the T-Cell Receptor Repertoire at Single-Clone Resolution</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0000055"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0000055" />

    <meta name="citation_title" content="Monitoring the T-Cell Receptor Repertoire at Single-Clone Resolution"/>
    <meta itemprop="name" content="Monitoring the T-Cell Receptor Repertoire at Single-Clone Resolution"/>

      <meta name="citation_author" content="Hendrik P.J. Bonarius"/>
            <meta name="citation_author_institution" content="Division of Clinical Immunology and Rheumatology, Academic Medical Centre, University of Amsterdam, Amsterdam, Netherlands"/>
      <meta name="citation_author" content="Frank Baas"/>
            <meta name="citation_author_institution" content="Department of Neurogenetics, Academic Medical Centre, University of Amsterdam, Amsterdam, Netherlands"/>
      <meta name="citation_author" content="Ester B.M. Remmerswaal"/>
            <meta name="citation_author_institution" content="Department of Experimental Immunology, Academic Medical Centre, University of Amsterdam, Amsterdam, Netherlands"/>
      <meta name="citation_author" content="RenÃ© A.W. van Lier"/>
            <meta name="citation_author_institution" content="Department of Experimental Immunology, Academic Medical Centre, University of Amsterdam, Amsterdam, Netherlands"/>
      <meta name="citation_author" content="Ineke J.M. ten Berge"/>
            <meta name="citation_author_institution" content="Division of Nephrology, Department of Internal Medicine, Academic Medical Centre, University of Amsterdam, Amsterdam, Netherlands"/>
      <meta name="citation_author" content="Paul P. Tak"/>
            <meta name="citation_author_institution" content="Division of Clinical Immunology and Rheumatology, Academic Medical Centre, University of Amsterdam, Amsterdam, Netherlands"/>
      <meta name="citation_author" content="Niek de Vries"/>
            <meta name="citation_author_institution" content="Division of Clinical Immunology and Rheumatology, Academic Medical Centre, University of Amsterdam, Amsterdam, Netherlands"/>

    <meta name="citation_date" content="2006/12/20"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0000055.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e55"/>
    <meta name="citation_issue" content="1"/>
    <meta name="citation_volume" content="1"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=T cell antigen receptor genes and T cell recognition.; citation_author=MM Davis; citation_author=PJ Bjorkman; citation_journal_title=Nature; citation_volume=334; citation_number=1; citation_pages=395-402; citation_date=1988; " />
      <meta name="citation_reference" content="citation_title=A direct estimate of the human alphabeta T cell receptor diversity.; citation_author=TP Arstila; citation_author=A Casrouge; citation_author=V Baron; citation_author=J Even; citation_author=J Kanellopoulos; citation_journal_title=Science; citation_volume=286; citation_number=2; citation_pages=958-961; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Diversity of human alphabeta T cell receptors.; citation_author=TP Arstila; citation_author=A Casrouge; citation_author=V Baron; citation_author=J Even; citation_author=J Kanellopoulos; citation_journal_title=Science; citation_volume=288; citation_number=3; citation_pages=1135; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=The sizes of the CDR3 hypervariable regions of the murine T-cell receptor beta chains vary as a function of the recombined germ-line segments.; citation_author=C Pannetier; citation_author=M Cochet; citation_author=S Darche; citation_author=A Casrouge; citation_author=M Zoller; citation_author=P Kourilsky; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=90; citation_number=4; citation_pages=4319-4323; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=Characterization of functional and phenotypic changes in anti-Gag vaccine-induced T cell responses and their role in protection after HIV-1 infection; citation_author=MR Betts; citation_author=B Exley; citation_author=DA Price; citation_author=A Bansal; citation_author=ZT Camacho; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=102; citation_number=5; citation_pages=4512-4517; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=LMO2-associated clonal T cell proliferation in two patients after gene therapy for SCID-X1.; citation_author=S Hacein-Bey-Abina; citation_author=C Von Kalle; citation_author=M Schmidt; citation_author=MP McCormack; citation_author=N Wulffraat; citation_journal_title=Science; citation_volume=302; citation_number=6; citation_pages=415-419; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=T cell receptor recognition motifs govern immune escape patterns in acute SIV infection.; citation_author=DA Price; citation_author=SM West; citation_author=MR Betts; citation_author=LE Ruff; citation_author=JM Brenchley; citation_journal_title=Immunity; citation_volume=21; citation_number=7; citation_pages=793-803; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Avidity for antigen shapes clonal dominance in CD8+ T cell populations specific for persistent DNA viruses; citation_author=DA Price; citation_author=JM Brenchley; citation_author=LE Ruff; citation_author=MR Betts; citation_author=BJ Hill; citation_journal_title=J Exp Med; citation_volume=202; citation_number=8; citation_pages=1349-1361; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Direct recognition by cytolytic T cells of Hfe, a MHC class Ib molecule without antigen-presenting function.; citation_author=PS Rohrlich; citation_author=N Fazilleau; citation_author=F Ginhoux; citation_author=H Firat; citation_author=F Michel; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=102; citation_number=9; citation_pages=12855-12860; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=T-cell repertoire diversity and clonal expansions in normal and clinical samples.; citation_author=C Pannetier; citation_author=J Even; citation_author=P Kourilsky; citation_journal_title=Immunol Today; citation_volume=16; citation_number=10; citation_pages=176-181; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=The molecular genetics of the T-cell antigen receptor and T-cell antigen recognition.; citation_author=M Kronenberg; citation_author=G Siu; citation_author=LE Hood; citation_author=NA Shastri; citation_journal_title=Ann Rev Imm; citation_volume=4; citation_number=11; citation_pages=529-591; citation_date=1986; " />
      <meta name="citation_reference" content="citation_title=Human T-cell receptor variable gene segment families.; citation_author=B Arden; citation_author=SP Clark; citation_author=D Kabelitz; citation_author=TW Mak; citation_journal_title=Immunogenetics; citation_volume=42; citation_number=12; citation_pages=455-500; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=IMGT the international ImMunoGeneTics database.; citation_author=M Ruiz; citation_author=V Giudicelli; citation_author=C Ginestoux; citation_author=P Stoehr; citation_author=J Robinson; citation_journal_title=Nucleic Acids Res; citation_volume=28; citation_number=13; citation_pages=219-221; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Expression profiling using a hexamer-based universal microarray.; citation_author=ME Roth; citation_author=L Feng; citation_author=KJ McConnell; citation_author=PJ Schaffer; citation_author=CE Guerra; citation_author=JP Affourtit; citation_journal_title=Nature Biotechnol; citation_volume=22; citation_number=14; citation_pages=418-426; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Hunting (auto)immune T cells in neuroimmunological diseases.; citation_author=R Hohlfeld; citation_journal_title=Brain; citation_volume=126; citation_number=15; citation_pages=2-4; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=The human cytotoxic T-lymphocyte (CTL) response to cytomegalovirus is dominated by structural protein pp65: frequency specificity and T- cell receptor usage of pp65-specific CTL.; citation_author=MR Wills; citation_author=AJ Carmichael; citation_author=K Mynard; citation_author=X Jin; citation_author=MP Weekes; citation_journal_title=J Virol; citation_volume=70; citation_number=16; citation_pages=7569-7579; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Tolerance of T-cell clones is associated with membrane antigen changes.; citation_author=ED Zanders; citation_author=JR Lamb; citation_author=M Feldmann; citation_author=N Green; citation_author=PC Beverley; citation_journal_title=Nature; citation_volume=303; citation_number=17; citation_pages=625-627; citation_date=1983; " />
      <meta name="citation_reference" content="citation_title=Strong selection of virus-specific cytotoxic CD4+ T cell clones during primary human cytomegalovirus infection.; citation_author=EM van Leeuwen; citation_author=EB Remmerswaal; citation_author=MH Heemskerk; citation_author=IJ ten Berge; citation_author=RA van Lier; citation_journal_title=Blood; citation_number=18; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Reverse transcriptase template switching: A SMART (TM) approach for full-length cDNA library construction.; citation_author=YY Zhu; citation_author=EM Machleder; citation_author=A Chenchik; citation_author=R Li; citation_author=PD Siebert; citation_journal_title=BioTechniques; citation_volume=30; citation_number=19; citation_pages=892-897; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=A novel approach to the analysis of specificity clonality and frequency of HIV-specific T cell responses reveals a potential mechanism for control of viral escape.; citation_author=DC Douek; citation_author=MR Betts; citation_author=JM Brenchley; citation_author=BJ Hill; citation_author=DR Ambrozak; citation_journal_title=J Immunol; citation_volume=168; citation_number=20; citation_pages=3099-3104; citation_date=2002; " />
      <meta name="citation_reference" content="citation_author=C Janeway; citation_author=P Travers; citation_author=M Walport; citation_author=M Shlomchik; citation_number=21; citation_pages=800; citation_date=2004; citation_publisher=Garland Science Textbooks; " />
      <meta name="citation_reference" content="citation_title=The fidelity of template-directed oligonucleotide ligation and its relevance to DNA computation.; citation_author=KD James; citation_author=AR Boles; citation_author=D Henckel; citation_author=AD Ellington; citation_journal_title=Nucleic Acids Res; citation_volume=26; citation_number=22; citation_pages=5203-5211; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Molecular tracking of antigen-specific T cell clones in neurological immune-mediated disorders.; citation_author=PA Muraro; citation_author=KP Wandinger; citation_author=B Bielekova; citation_author=B Gran; citation_author=A Marques; citation_journal_title=Brain; citation_volume=126; citation_number=23; citation_pages=20-31; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Adoptive T cell therapy using antigen-specific CD8+ T cell clones for the treatment of patients with metastatic melanoma.; citation_author=C Yee; citation_author=JA Thompson; citation_author=D Byrd; citation_author=SR Riddell; citation_author=P Roche; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=99; citation_number=24; citation_pages=16168-16173; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=A novel high-performance random array platform for quantitative gene expression profiling.; citation_author=K Kuhn; citation_author=SC Baker; citation_author=E Chudin; citation_author=MH Lieu; citation_author=S Oeser; citation_journal_title=Genome Research; citation_volume=14; citation_number=25; citation_pages=2347-2356; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Modulation of the T cell receptor beta chain repertoire after heart transplantation.; citation_author=K Doumaid; citation_author=PP van Miert; citation_author=LM Vaessen; citation_author=EB Remmerswaal; citation_author=W Weimar; citation_journal_title=Transpl Immunol; citation_volume=8; citation_number=26; citation_pages=83-94; citation_date=2000; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000055" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Monitoring the T-Cell Receptor Repertoire at Single-Clone Resolution"/>
    <meta name="twitter:description" content="The adaptive immune system recognizes billions of unique antigens using highly variable T-cell receptors. The &alpha;&beta; T-cell receptor repertoire includes an estimated 106 different rearranged &beta; chains per individual. This paper describes a novel micro-array based method that monitors the &beta; chain repertoire with a resolution of a single T-cell clone. These T-arrays are quantitative and detect T-cell clones at a frequency of less than one T cell in a million, which is 2 logs more sensitive than spectratyping (immunoscope), the current standard in repertoire analysis. Using T-arrays we detected CMV-specific CD4+ and CD8+ T-cell clones that expanded early after viral antigen stimulation in vitro and in vivo. This approach will be useful in monitoring individual T-cell clones in diverse experimental settings, and in identification of T-cell clones associated with infectious disease, autoimmune disease and cancer."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0000055.g006"/>

  <meta property="og:title" content="Monitoring the T-Cell Receptor Repertoire at Single-Clone Resolution" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000055" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=2585'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=5505'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=5744&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0000055">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000055" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000055&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Monitoring the T-Cell Receptor Repertoire at Single-Clone Resolution
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Hendrik P.J. Bonarius, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Division of Clinical Immunology and Rheumatology, Academic Medical Centre, University of Amsterdam, Amsterdam, Netherlands
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Frank Baas, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Neurogenetics, Academic Medical Centre, University of Amsterdam, Amsterdam, Netherlands
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Ester B.M. Remmerswaal, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Experimental Immunology, Academic Medical Centre, University of Amsterdam, Amsterdam, Netherlands
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            RenÃ© A.W. van Lier, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Experimental Immunology, Academic Medical Centre, University of Amsterdam, Amsterdam, Netherlands
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Ineke J.M. ten Berge, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Division of Nephrology, Department of Internal Medicine, Academic Medical Centre, University of Amsterdam, Amsterdam, Netherlands
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Paul P. Tak, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Division of Clinical Immunology and Rheumatology, Academic Medical Centre, University of Amsterdam, Amsterdam, Netherlands
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Niek de Vries
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:Niek.deVries@amc.uva.nl">Niek.deVries@amc.uva.nl</a>
            </p>

                <p>Affiliation:
                  Division of Clinical Immunology and Rheumatology, Academic Medical Centre, University of Amsterdam, Amsterdam, Netherlands
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: December 20, 2006</li>
    <li>DOI: 10.1371/journal.pone.0000055</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0000055" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0000055" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000055" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0000055" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0000055-g001" data-doi="info:doi/10.1371/journal.pone.0000055" data-uri="info:doi/10.1371/journal.pone.0000055.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000055.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000055-g002" data-doi="info:doi/10.1371/journal.pone.0000055" data-uri="info:doi/10.1371/journal.pone.0000055.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000055.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000055-g003" data-doi="info:doi/10.1371/journal.pone.0000055" data-uri="info:doi/10.1371/journal.pone.0000055.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000055.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000055-g004" data-doi="info:doi/10.1371/journal.pone.0000055" data-uri="info:doi/10.1371/journal.pone.0000055.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000055.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000055-g005" data-doi="info:doi/10.1371/journal.pone.0000055" data-uri="info:doi/10.1371/journal.pone.0000055.g005" title="Figure 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000055.g005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000055-g006" data-doi="info:doi/10.1371/journal.pone.0000055" data-uri="info:doi/10.1371/journal.pone.0000055.g006" title="Figure 6">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000055.g006&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000055">Reader Comments (2)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0000055" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1"></a><p>The adaptive immune system recognizes billions of unique antigens using highly variable T-cell receptors. The Î±Î² T-cell receptor repertoire includes an estimated 10<sup>6</sup> different rearranged Î² chains per individual. This paper describes a novel micro-array based method that monitors the Î² chain repertoire with a resolution of a single T-cell clone. These T-arrays are quantitative and detect T-cell clones at a frequency of less than one T cell in a million, which is 2 logs more sensitive than spectratyping (immunoscope), the current standard in repertoire analysis. Using T-arrays we detected CMV-specific CD4+ and CD8+ T-cell clones that expanded early after viral antigen stimulation <em>in vitro</em> and <em>in vivo</em>. This approach will be useful in monitoring individual T-cell clones in diverse experimental settings, and in identification of T-cell clones associated with infectious disease, autoimmune disease and cancer.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>Bonarius HP, Baas F, Remmerswaal EB, Lier RAv, Berge IJt, et al.  (2006) Monitoring the T-Cell Receptor Repertoire at Single-Clone Resolution. PLoS ONE 1(1):
          e55.
            doi:10.1371/journal.pone.0000055</p><p><strong>Academic Editor: </strong>Etienne Joly, UniversitÃ© de Toulouse, France</p><p><strong>Received:</strong> September 12, 2006; <strong>Accepted:</strong> October 13, 2006; <strong>Published:</strong> December 20, 2006</p><p><strong>Copyright:</strong> Â© 2006 de Vries et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>The Netherlands Genomics Initiative (NGI: <a href="http://www.genomics.nl">www.genomics.nl</a>) funded the position of H.P.J.B. in a Horizon Breakthrough Grant to N.V. The NGI did not play a role in the conduct of the study, in the collection, analysis, and interpretation of the data, or in the preparation, review, or approval of the manuscript. There are no other financial or other relations that might be relevant to the submission of this manuscript.</p><p><strong>Competing interests:</strong> The AMC at the University of Amsterdam, the employer of all authors, filed for a patent application listing N.V. as inventor. In case the system becomes profitable, personal revenues are maximized at â¬50.000 a year; the department may receive up to 10% of the financial rewards. Neither the AMC nor the transfer office were involved in the conception, editing or the decision to submit the manuscript.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>T cells are key players in the antigen specific immune responses. Antigen specificity is provided by the T-cell receptor (TCR), which is unique for each T-cell clone. Upon antigen recognition, individual T cell clones generally expand and acquire differential effector properties. Although the number of potential TCRs has been estimated at 10<sup>15</sup> different Î±/Î² combinations <a href="#pone.0000055-Davis1">[1]</a>, the actual Î±Î² TCR repertoire per individual is estimated to include 10<sup>6</sup> different Î² chains <a href="#pone.0000055-Arstila1">[2]</a>, each pairing with a limited number of Î± chains <a href="#pone.0000055-Arstila2">[3]</a>. There is no rapid technology available that can sensitively and quantitatively monitor this highly diverse T-cell receptor repertoire.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>Current technology for screening the TCR repertoire for expanded T-cell clones relies on âspectratypingâ <a href="#pone.0000055-Pannetier1">[4]</a>, often referred to as immunoscope, and/or individual cloning and sequencing of a sample of the T-cell population <a href="#pone.0000055-Arstila1">[2]</a>, <a href="#pone.0000055-Betts1">[5]</a>â<a href="#pone.0000055-Rohrlich1">[9]</a>]. In spectratyping analysis, PCR amplified TCR DNA is separated on size of the CDR3 region. This approach separates the TCRÎ² repertoire in approximately 230 fractions, resulting from the use of ~23 primers for all functional VÎ² families and about 8 different CDR3 lengths per VÎ² <a href="#pone.0000055-Pannetier2">[10]</a>. A higher resolution can be attained when V- and J-region primers are used; however, this requires 23 Â· 13 individual PCR reactions, and results in a resolution of approximately 23 Â· 13 Â· 8 peaks (<a href="#pone.0000055.s004">
               <strong>Table S2</strong>
            </a>).</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>Spectratyping itself generally does not identify individual T cell clones, and is therefore often followed by repetitive cloning and sequencing. Clonal peaks identified in the spectratype patterns are sequenced, typically 10<sup>2</sup> clones and maximally 10<sup>4</sup> clones per sample <a href="#pone.0000055-Arstila1">[2]</a>, <a href="#pone.0000055-Betts1">[5]</a>â<a href="#pone.0000055-Rohrlich1">[9]</a>] in previous publications. The sensitivity of this combined approach depends on the sensitivity of spectratyping for identification of clonal peaks, and on the number of T-cell-receptor rearrangements cloned and sequenced. Thus, although the combination of spectratyping with sequencing can attain sufficient resolution to analyze TCR diversity, the approach is laborious and time consuming as it requires PCR amplification, isolation of individual bands based on DNA size, purification, followed by repeated cloning and sequencing.</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p>Here, we explore a novel approach which exploits the high capacity of DNA microarrays to monitor the expression of many T-cell receptor rearrangements in parallel. At present, it can be used to follow T cell responses in cases where type of VÎ²/JÎ² and length of JÎ²-gene segment are available, e.g. from prior immunoscope (spectratyping) experiments. The feasibility of this approach is shown, and validated both in vitro and in vivo. We show that T arrays quantitatively monitor the expansion of T-cell clones after viral infection with high sensitivity (1 in 10<sup>6</sup> cells), and with sufficient resolution to identify single clones in a background of polyclonal peripheral blood T-cells. While at present it allows monitoring a VÎ²/JÎ²-specific fraction of 0.03% of the T-cell receptor repertoire on a single 4000-spot slide, the microarray-based method can be scaled up to monitor and screen a large pool of the T-cell repertoire for dominant clonotypes. We envision that this sensitive and rapid technology will be useful for monitoring and screening of clonal T-cell expansions for many applications in medical research.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3>
<h4>Creating single-clone resolution</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>To create adequate resolution between different potential TCRs we focussed on the highly variable complementarity determining region 3 (CDR3) of the TCR beta chain. This region consists of one out of 40â48 functional VÎ² and one out of 13 functional JÎ² segments, joined by DÎ² gene segments <a href="#pone.0000055-Kronenberg1">[11]</a>â<a href="#pone.0000055-Ruiz1">[13]</a>. The CDR3 is generated during VDJ-recombination by random deletion and addition of nucleotides at the V-, D-, and J-junctions <a href="#pone.0000055-Davis1">[1]</a> and produces the hypervariable NDN region, which can be used as a signature for each TCR (<a href="#pone-0000055-g001">
                  <strong>Fig. 1A</strong>
               </a>). We developed a T-array protocol (<a href="#pone-0000055-g001">
                  <strong>Fig. 1C</strong>
               </a>) to interrogate the first six nucleotides of the NDN region and the length of the JÎ²-gene segment. Resolution is created in three subsequent steps by: <em>i</em>) VÎ²-specific PCR amplification of the CDR3Î² (<a href="#pone-0000055-g001">
                  <strong>Fig. 1C1 and C2</strong>
               </a>), <em>ii</em>) hybridization of a labelled oligonucleotide (âannealerâ) specific for the JÎ²-family and for the number of JÎ²-nucleotides deleted (<a href="#pone-0000055-g001">
                  <strong>Fig. 1C4</strong>
               </a>), and (<em>iii</em>) a ligation reaction specific for the first six nucleotides of the hypervariable NDN region on a universal hexamer microarray <a href="#pone.0000055-Roth1">[14]</a>, encoding all permutations of a hexamer nucleotide (<a href="#pone-0000055-g001">
                  <strong>Fig. 1C5</strong>
               </a>). In this way, the hexamer sequences on the array complementary to the first six nucleotides of the NDN region of a T-cell clone are ligated to the fluorescent annealer probes (<a href="#pone-0000055-g001">
                  <strong>Fig. 1c6â7</strong>
               </a>). The fluorescent signal of each hexamer sequence on a single microarray chip, quantitatively reflects the expansion of a certain T-cell clone. It should be noted that an annealer designed for a JÎ² gene with n nucleotides deleted from the germline sequence will also give a signal for TCRs with less than n nucleotides deleted. The latter TCRs will reveal part of the germline sequence of JÎ² in their hexamer sequences (<strong>1C4B</strong>).</p>
<div class="figure" id="pone-0000055-g001"><div class="img"><a name="pone-0000055-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000055.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000055" data-uri="info:doi/10.1371/journal.pone.0000055.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000055.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000055.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000055.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000055.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000055.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000055.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000055.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000055.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000055.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>The T-array protocol.</span></strong></p><a id="article1.body1.sec2.sec1.fig1.caption1.p1" name="article1.body1.sec2.sec1.fig1.caption1.p1"></a><p>
                     <strong>(A)</strong> During development, VDJ recombination causes enormous variability in TCRÎ² chain by randomly selecting different combinations of 23 V, 2 D, and 13 J gene segments, by nucleotide insertion (<span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000055.e001&amp;representation=PNG" class="inline-graphic"></span>), and by nucleotide deletion from V (<span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000055.e002&amp;representation=PNG" class="inline-graphic"></span>), D, and J (<span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000055.e003&amp;representation=PNG" class="inline-graphic"></span>) genes.</p>
<a id="article1.body1.sec2.sec1.fig1.caption1.p2" name="article1.body1.sec2.sec1.fig1.caption1.p2"></a><p>This results in a diversity of an estimated 10<sup>6</sup> different Î² chains per individual.</p>
<a id="article1.body1.sec2.sec1.fig1.caption1.p3" name="article1.body1.sec2.sec1.fig1.caption1.p3"></a><p>
                     <strong>(B)</strong> N-deletion causes shortening of the VÎ² and JÎ² segments.</p>
<a id="article1.body1.sec2.sec1.fig1.caption1.p4" name="article1.body1.sec2.sec1.fig1.caption1.p4"></a><p>The number of nucleotides deleted from VÎ² and JÎ² germline DNA is limited.</p>
<a id="article1.body1.sec2.sec1.fig1.caption1.p5" name="article1.body1.sec2.sec1.fig1.caption1.p5"></a><p>N-deletion of 192 published TCRÎ² mRNAs was determined.</p>
<a id="article1.body1.sec2.sec1.fig1.caption1.p6" name="article1.body1.sec2.sec1.fig1.caption1.p6"></a><p>The figure shows the cumulative percentage of CDR3Î²s for the number of nucleotides deleted.</p>
<a id="article1.body1.sec2.sec1.fig1.caption1.p7" name="article1.body1.sec2.sec1.fig1.caption1.p7"></a><p>TCRÎ²'s with n nucleotides deleted represent approximately 10% of the repertoire if n = 0 to 6, and 5%, if n = 7 to 9.</p>
<a id="article1.body1.sec2.sec1.fig1.caption1.p8" name="article1.body1.sec2.sec1.fig1.caption1.p8"></a><p>
                     <strong>(C)</strong> The T-array protocol: <strong>(C1)</strong> cDNA from T-cells is generated.</p>
<a id="article1.body1.sec2.sec1.fig1.caption1.p9" name="article1.body1.sec2.sec1.fig1.caption1.p9"></a><p>
                     <strong>(C2)</strong> CDR3Î² regions are PCR amplified using biotinylated VÎ²-specific (<span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000055.e004&amp;representation=PNG" class="inline-graphic"></span>) or VÎ²-generic primers (not shown here).</p>
<a id="article1.body1.sec2.sec1.fig1.caption1.p10" name="article1.body1.sec2.sec1.fig1.caption1.p10"></a><p>
                     <strong>(C3)</strong> Biotinylated strands are removed after alkaline denaturation using streptavidin-coated beads.</p>
<a id="article1.body1.sec2.sec1.fig1.caption1.p11" name="article1.body1.sec2.sec1.fig1.caption1.p11"></a><p>
                     <strong>(c4)</strong> Single-strands of polyclonal TCRs are aliquoted and hybridized to fluorescently labeled annealers (<span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000055.e005&amp;representation=PNG" class="inline-graphic"></span>) complementary to the NDN-adjacent end of a JÎ² gene.</p>
<a id="article1.body1.sec2.sec1.fig1.caption1.p12" name="article1.body1.sec2.sec1.fig1.caption1.p12"></a><p>A specific number of JÎ²-gene nucleotides (n) is deleted for each annealer, accounting for N-deletion during the VDJ recombination process.</p>
<a id="article1.body1.sec2.sec1.fig1.caption1.p13" name="article1.body1.sec2.sec1.fig1.caption1.p13"></a><p>Insert (C4): Each annealer will hybridize to TCRÎ² rearrangements where n nucleotides are deleted from the JÎ²-germline gene segment (<strong>C4A</strong>) or where less than n nucleotides are deleted (<strong>C4B</strong>).</p>
<a id="article1.body1.sec2.sec1.fig1.caption1.p14" name="article1.body1.sec2.sec1.fig1.caption1.p14"></a><p>
                     <strong>(C5)</strong> The annealer-hybridized fractions are loaded on universal hexamer arrays for <strong>(C6)</strong> T-cell-clone-specific ligation and, <strong>(C7)</strong> subsequently washed, scanned and analyzed.</p>
<span>doi:10.1371/journal.pone.0000055.g001</span></div><a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2"></a><p>The resolution of this T-array protocol depends on the number of VÎ² and JÎ² segments, the size of the microarrays, and the number of JÎ²-nucleotides deleted. To predict the potential resolution of the assay we analyzed the distribution of N-deletion in a random selection of 192 published CDR3Î² mRNA sequences (<a href="#pone.0000055.s003">
                  <strong>Table S1</strong>
               </a>). For 99% of the sequences a maximum of 10 nucleotides is deleted from VÎ² genes, and a maximum of 11 nucleotides from JÎ² genes (<a href="#pone-0000055-g001">
                  <strong>Fig. 1B</strong>
               </a>). Within these limits, an almost uniform distribution of the TCRs was observed over the number of nucleotides deleted. This enabled us to predict the potential resolution of the assay. Although the theoretical size of the TCR repertoire is estimated at 10<sup>15</sup>, extensive cloning experiments have shown that within one individual the beta-chain repertoire contains approximately 10<sup>6</sup> unique sequences <a href="#pone.0000055-Arstila1">[2]</a>, each of which pairs with a limited number of Î± chains <a href="#pone.0000055-Arstila1">[2]</a>, <a href="#pone.0000055-Arstila2">[3]</a>. Based on these numbers we estimate that after VÎ²/JÎ²-specific amplification on average 10<sup>6</sup>/1.4 Â· 10<sup>7</sup> = 0.07 CDR3Î² sequences from the complete repertoire of a human individual will ligate to a single sequence on the universal hexamer microarray (see also <a href="#pone.0000055.s004">
                  <strong>Table S2C</strong>
               </a>). In theory, the assay should therefore have sufficient resolution to detect single CDR3Î² sequences.</p>


<h4>Testing sequence specificity and validity</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>The specificity of the protocol was tested using the T-cell clone Jurkat E6-1, for which the CDR3 is known. After PCR amplification of the Jurkat CDR3Î² region, we isolated the antisense strand and hybridized it to a fluorescently labelled oligonucleotide encoding the NDN-oriented end of the JÎ²1-2 sequence. Specificity of the ligation reaction for Jurkat NDN sequence was then tested with hexamers either complementary or not complementary to this NDN sequence (<a href="#pone-0000055-g002">
                  <strong>Fig. 2BâC</strong>
               </a>). Only in the presence of the complementary hexamer sequence (5â²-GTTCGG-3â²) the annealer oligonucleotide was elongated, indicating that the ligation is sequence specific for the Jurkat CDR3Î². Similarly, when the sense strand was used as a template, the annealer was elongated only with the hexamer sequence complementary to the 5â²-end of NDNÎ² (<a href="#pone-0000055-g002">
                  <strong>Fig. 2DâE</strong>
               </a>). When tested on a universal hexamer array, out of all 4096 possible sequences the Jurkat NDN sequence GTTCGG gave the strongest signal (<a href="#pone-0000055-g002">
                  <strong>Fig. 2FâH</strong>
               </a>). This shows that the protocol is sequence specific for the T-cell clone analyzed. However, some other spots did produce positive signals, albeit at much lower signal intensities, notably if the encoded sequence was identical in the 3â²-end nucleotides (NNTCGG). This suggests that, apart from the strongest signal at hexamer GTTCCG, the Jurkat NDN sequence ligated to hexamers with a 5â²-end mismatch.</p>
<div class="figure" id="pone-0000055-g002"><div class="img"><a name="pone-0000055-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000055.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000055" data-uri="info:doi/10.1371/journal.pone.0000055.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000055.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000055.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000055.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000055.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000055.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000055.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000055.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000055.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000055.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Ligation experiments with Jurkat CDR3Î² amplicons.</span></strong></p><a id="article1.body1.sec2.sec2.fig1.caption1.p1" name="article1.body1.sec2.sec2.fig1.caption1.p1"></a><p>
                     <strong>(A)</strong> VDJ rearrangement of Jurkat TCRÎ².</p>
<a id="article1.body1.sec2.sec2.fig1.caption1.p2" name="article1.body1.sec2.sec2.fig1.caption1.p2"></a><p>In this CDR3 sequence, there are 3 and 0 nucleotides deleted form the germline VÎ² and JÎ², respectively.</p>
<a id="article1.body1.sec2.sec2.fig1.caption1.p3" name="article1.body1.sec2.sec2.fig1.caption1.p3"></a><p>
                     <strong>(BâE)</strong> Capillary-electrophoresis chromatograms showing length and signal of fluorescent annealers (<span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000055.e006&amp;representation=PNG" class="inline-graphic"></span>).</p>
<a id="article1.body1.sec2.sec2.fig1.caption1.p4" name="article1.body1.sec2.sec2.fig1.caption1.p4"></a><p>
                     <strong>(B)</strong> A hexamer (GTTCGG) complementary to the Jurkat 3-end NDN sequence elongated (<strong>*</strong>) the JÎ²1-2-specific annealer (â¦).</p>
<a id="article1.body1.sec2.sec2.fig1.caption1.p5" name="article1.body1.sec2.sec2.fig1.caption1.p5"></a><p>
                     <strong>(C)</strong> The non-complementary 5â²-end hexamer failed to cause elongation.</p>
<a id="article1.body1.sec2.sec2.fig1.caption1.p6" name="article1.body1.sec2.sec2.fig1.caption1.p6"></a><p>Black peaks: Cy5 signal; Red peaks: FAM signal of size standards.</p>
<a id="article1.body1.sec2.sec2.fig1.caption1.p7" name="article1.body1.sec2.sec2.fig1.caption1.p7"></a><p>
                     <strong>(D, E)</strong> Similarly, a Cy5-labeled oligonucleotide complementary to the 3â² coding end of VÎ²12, was elongated (<strong>*</strong>) only in the presence of the hexamer GTCGAG which is complementary to the 5â²-end of the NDN sequence.</p>
<a id="article1.body1.sec2.sec2.fig1.caption1.p8" name="article1.body1.sec2.sec2.fig1.caption1.p8"></a><p>
                     <strong>(F)</strong> Detail of universal hexamer array after ligation of a Cy5-labeled oligonucleotide complementary to the 5â² coding end of JÎ²1-2 in the presence of the antisense strand template of Jurkat CDR3Î².</p>
<a id="article1.body1.sec2.sec2.fig1.caption1.p9" name="article1.body1.sec2.sec2.fig1.caption1.p9"></a><p>
                     <strong>(G)</strong> Signal intensities of all 4096 spots of the same array experiment.</p>
<a id="article1.body1.sec2.sec2.fig1.caption1.p10" name="article1.body1.sec2.sec2.fig1.caption1.p10"></a><p>The arrow in (F) and (G) indicates the microarray spot with sequence GTTCGG.</p>
<a id="article1.body1.sec2.sec2.fig1.caption1.p11" name="article1.body1.sec2.sec2.fig1.caption1.p11"></a><p>
                     <strong>(H)</strong> List of 20 strongest spots with their hexamer sequence and Cy5-fluorescent signal.</p>
<span>doi:10.1371/journal.pone.0000055.g002</span></div>

<h4>Determination of sensitivity T-array</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>Having shown that the T-array is specific for the NDN sequence of the analyzed T-cell clone we then compared the sensitivity of the assay to that of spectratyping by diluting decreasing proportions of Jurkat cells in a background of peripheral blood CD4<sup>+</sup> T-cells. Semi-quantitative PCR showed that TCR transcripts in Jurkat cells were not more abundant than in CD4+ cells obtained from a healthy blood donor (<a href="#pone.0000055.s001">
                  <strong>Fig. S1</strong>
               </a>). For immunoscope, Jurkat/CD4+ mixtures were then PCR amplified with a VÎ²12-sense primer and a fluorescamine-labelled CÎ² reverse primer and size separated by capillary electrophoresis (<a href="#pone-0000055-g003">
                  <strong>Fig. 3A</strong>
               </a>). As expected, the size difference between DNA peaks was 3 bp and the peak signals were normally (Gaussian) distributed <a href="#pone.0000055-Betts1">[5]</a>. Only in the case of a dilution of 1 Jurkat cell per 10<sup>4</sup> CD4<sup>+</sup> blood cells, the peak associated with Jurkat CDR3Î² length (14 amino acids) is 47% of the total peak area. At dilutions of 1 in 10<sup>5</sup> to 1 in 10<sup>7</sup> the peak associated with 14 amino acids is 7â8% of the total peak area, and the peak distribution remains normally distributed, indicating no dominance of any CDR3 length in the VÎ²12 compartment. These results show that the sensitivity of spectratyping is approximately 1 T-cells in 10<sup>4</sup> cells, which is in agreement with other reports <a href="#pone.0000055-Hohlfeld1">[15]</a>.</p>
<div class="figure" id="pone-0000055-g003"><div class="img"><a name="pone-0000055-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000055.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000055" data-uri="info:doi/10.1371/journal.pone.0000055.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000055.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000055.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000055.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000055.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000055.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000055.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000055.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000055.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000055.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Spectratyping and T-array for Jurkat T-cell clone mixed with peripheral blood CD4<sup>+</sup> T-cells.</span></strong></p><a id="article1.body1.sec2.sec3.fig1.caption1.p1" name="article1.body1.sec2.sec3.fig1.caption1.p1"></a><p>Jurkat cells were added in different dilutions to a background of peripheral blood CD4<sup>+</sup> T-cells.</p>
<a id="article1.body1.sec2.sec3.fig1.caption1.p2" name="article1.body1.sec2.sec3.fig1.caption1.p2"></a><p>
                     <strong>(panel A)</strong> CDR3 spectratyping with VÎ²12-specific primer.</p>
<a id="article1.body1.sec2.sec3.fig1.caption1.p3" name="article1.body1.sec2.sec3.fig1.caption1.p3"></a><p>The arrow indicates a length identical to 14 amino acids, which is the length of the Jurkat CDR3Î².</p>
<a id="article1.body1.sec2.sec3.fig1.caption1.p4" name="article1.body1.sec2.sec3.fig1.caption1.p4"></a><p>
                     <strong>(panel B)</strong> A detail of the T-array scans.</p>
<a id="article1.body1.sec2.sec3.fig1.caption1.p5" name="article1.body1.sec2.sec3.fig1.caption1.p5"></a><p>White arrows indicate hexamer sequence GTTCGG, which is complementary to the first six nucleotides Jurkat NDNÎ² region.</p>
<a id="article1.body1.sec2.sec3.fig1.caption1.p6" name="article1.body1.sec2.sec3.fig1.caption1.p6"></a><p>
                     <strong>(panel C)</strong> Signal intensities of all 4096 spots of the T-array.</p>
<a id="article1.body1.sec2.sec3.fig1.caption1.p7" name="article1.body1.sec2.sec3.fig1.caption1.p7"></a><p>Black arrows indicate hexamer sequence GTTCGG.</p>
<span>doi:10.1371/journal.pone.0000055.g003</span></div><a id="article1.body1.sec2.sec3.p2" name="article1.body1.sec2.sec3.p2"></a><p>For T-arrays, the antisense strands were isolated and hybridized to the JÎ²-1-2-specific, Cy5-labeled oligonucleotide mentioned earlier and ligated on a universal hexamer microarray (<a href="#pone-0000055-g003">
                  <strong>Fig. 3BâC</strong>
               </a>). In the case of a 1 in 10<sup>5</sup> to 1 in 10<sup>7</sup> Jurkat/CD4<sup>+</sup> ratio, the hexamer sequence GTTCGG, which is complementary to the 3â² end of Jurkat NDN region, was quantitatively picked up (<a href="#pone-0000055-g003">
                  <strong>Fig. 3C</strong>
               </a>). In the case of a 1 in 10<sup>7</sup> dilution, the GTTCGG signal was similarly intense as the hexamer spots TGTCGG and CTTCGG. These sequences only differ in the nucleotides at the terminal end of the ligation product, suggesting that these are 5â² mismatch ligations of the Jurkat sequence. These results show that in this format of the T-array protocol, individual T-cell clones are picked up with a sensitivity of 1 clone in 10<sup>6</sup> T-cells.</p>


<h4>Detection of expanding T-cell clones after viral antigen stimulation</h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>To test whether the T-array protocol would allow identification of T-cell clones that expand upon antigen activation an in vitro stimulation experiment was performed. Human peripheral blood cells from a healthy HLA-A2<sup>+</sup> donor latently infected with the Î² herpes virus CMV were isolated and stimulated with the CMV-peptide NLVPMVATV. This 9-amino acid motif from the viral structural protein pp65 dominates the cytotoxic T-lymphocyte response against CMV <a href="#pone.0000055-Hohlfeld1">[15]</a>. In HLA-A2<sup>+</sup> individuals, the CD8<sup>+</sup> response to NLVPMVATV is VÎ²-restricted, in particular for but not limited to VÎ²13<sup>+</sup> T-cells <a href="#pone.0000055-Wills1">[16]</a>, which was in agreement with spectratyping analysis of our donor (data not shown). FACS analysis, using HLA-A2-NLV tetramer staining showed that the fraction of antigen specific T cells in the cytotoxic T-cell pool increased after stimulation with NLV peptide (<a href="#pone-0000055-g004">
                  <strong>Fig. 4A</strong>
               </a>). Before stimulation (Day 0) a fraction of ~5% of the CD8<sup>+</sup> cells was tetramer positive confirming CMV latency. Three days after peptide stimulation CMV-reactive T-cells were not detectable by FACS using tetramers, which can be attributed to TCR internalization after MHC/peptide recognition <a href="#pone.0000055-Zanders1">[17]</a>. During the next 10 days, the fraction of tetramer-positive cells slowly increased to ~60%. From day 6, spectratyping analysis revealed that the VÎ²13<sup>+</sup> compartment became restricted to a CDR3 length correlating to 14 amino acids (<a href="#pone-0000055-g004">
                  <strong>Fig. 4B</strong>
               </a>), suggesting that either a single T-cell clone or only a limited number of clones in the VÎ²13 compartment had expanded.</p>
<div class="figure" id="pone-0000055-g004"><div class="img"><a name="pone-0000055-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000055.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000055" data-uri="info:doi/10.1371/journal.pone.0000055.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000055.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000055.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000055.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000055.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000055.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000055.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000055.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000055.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000055.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>Clonal expansion of T-cells from a CMV<sup>+</sup> donor after antigen specific stimulation.</span></strong></p><a id="article1.body1.sec2.sec4.fig1.caption1.p1" name="article1.body1.sec2.sec4.fig1.caption1.p1"></a><p>
                     <strong>(panel A)</strong> The fraction of antigen specific T-cell clones determined with NLVPMVATV-loaded tetramers.</p>
<a id="article1.body1.sec2.sec4.fig1.caption1.p2" name="article1.body1.sec2.sec4.fig1.caption1.p2"></a><p>
                     <strong>(panel B)</strong> Spectratyping of unsorted VÎ²13<sup>+</sup> and VÎ²13<sup>+</sup>/JÎ²1-2<sup>+</sup> fraction.</p>
<a id="article1.body1.sec2.sec4.fig1.caption1.p3" name="article1.body1.sec2.sec4.fig1.caption1.p3"></a><p>
                     <strong>(panel C)</strong> CDR3 sequence of the clone identified compared to the JÎ²1-2 germline sequence.</p>
<a id="article1.body1.sec2.sec4.fig1.caption1.p4" name="article1.body1.sec2.sec4.fig1.caption1.p4"></a><p>
                     <strong>(panel D)</strong> T-array of unsorted VÎ²13<sup>+</sup>/JÎ²1-2<sup>+</sup> fraction.</p>
<a id="article1.body1.sec2.sec4.fig1.caption1.p5" name="article1.body1.sec2.sec4.fig1.caption1.p5"></a><p>The annealer oligonucleotide used in this T-array experiment had the sequence ACTATGGCTACACCTTCGGTT, allowing detection of rearrangements with 3 or less nucleotides deleted from the JÎ²1-2 germline.</p>
<a id="article1.body1.sec2.sec4.fig1.caption1.p6" name="article1.body1.sec2.sec4.fig1.caption1.p6"></a><p>The arrow indicates sequence CCTTTT, the first nucleotides of the NDNÎ² region of the dominant T-cell clone that was identified in this screen.</p>
<span>doi:10.1371/journal.pone.0000055.g004</span></div><a id="article1.body1.sec2.sec4.p2" name="article1.body1.sec2.sec4.p2"></a><p>While VÎ²-13/CÎ²-spectratyping (<a href="#pone-0000055-g004">
                  <strong>Fig. 4B</strong>
               </a>) or tetramer analysis (<a href="#pone-0000055-g004">
                  <strong>Fig. 4A</strong>
               </a>) only detected antigen-specific clones for CMV at day 6, this clone was detected at day 3 using a more specific VÎ²-13/JÎ²-1-2 spectratyping approach. The T-array, which was performed on non-sorted T-cells, identified the CMV-specific T-cell clone characterised by the NDN sequence CCTTTT already at day 0 (<a href="#pone-0000055-g004">
                  <strong>Fig. 4C</strong>
               </a>). To exclude aspecific effects of the primer and annealer sequences used a T-array experiment was performed using the same primers and annealer oligonucleotide on a different sample; in this case no signal above background was observed at the CCTTTT hexamer (data not shown). Further validation of the CCTTTT hexamer sequence was acquired using extensive cloning and sequencing (See below).</p>
<a id="article1.body1.sec2.sec4.p3" name="article1.body1.sec2.sec4.p3"></a><p>Hexamer sequences other than CCTTTT gave signals above background intensity (<a href="#pone.0000055.s005">
                  <strong>Table S3</strong>
               </a>, <a href="#pone-0000055-g004">
                  <strong>Fig. 4C</strong>
               </a>). Of the top 100 signals 19 had a 3â² CTA-end, identical to the terminal germline sequence of the JÎ²-1-2 gene segment. These sequences derive from TCRs that have no nucleotides deleted from the germline JÎ²-1-2 gene (<a href="#pone.0000055.s002">
                  <strong>Fig. S2</strong>
               </a>), and therefore give 5â²-(NNN)CTA-3â² signals. Similarly, sequences that have a (NNNN)TA-end (34 out of 100) or (NNNNN)A-end (62 out of 100) derive from TCRs encoding the germline JÎ²-1-2 gene with only one resp. two terminal nucleotide deleted. Indeed, these signals derive from such TCR sequences as shown by complete sequencing of these TCRÎ²'s (see also below). They can all be identified based on the terminal germline nucleotide (here âAâ) in the hexamer sequence.</p>


<h4>Validating T-array data by sequencing of multiple T-cell clones</h4>
<a id="article1.body1.sec2.sec5.p1" name="article1.body1.sec2.sec5.p1"></a><p>To test whether the T-array signal matches the frequency of these T-cell clones as estimated by repetitive cloning and sequencing, we sequenced VÎ²13<sup>+</sup>/JÎ²1-2<sup>+</sup> TCRs from three samples of the experiment shown in <a href="#pone-0000055-g004">Figure 4</a> (<a href="#pone.0000055.s006">
                  <strong>Table S4</strong>
               </a>). Out of 52 clones sequenced from the Day 0 sample, 27 were found to have 3 or less nucleotides deleted from the JÎ²-1-2 gene, and can therefore be detected in the T-array shown in <a href="#pone-0000055-g004">Figure 4</a>. Fourteen of these sequences were unique. The two clones that were found at high frequency (7/52) gave the strongest signals on the T-array (hexamer sequences CCTTTT and GGACCG). One clone which was detected at lower frequency (CAGCTA, frequency 2/52) also gave a T-array signal well above background. Eleven clones were detected with a frequency of only 1 out of 52. Three of these gave T array signals above background (<a href="#pone.0000055.s007">
                  <strong>Table S5</strong>
               </a>). Eight clones gave signals similar to background, suggesting that the concentration of these clones in the blood sample is below the detection limit of the T-array. The clonal frequencies measured at day 3 were in agreement with the expansion measured by the T-array, showing that the T-array protocol quantitatively detects clonal expansion.</p>


<h4>Application of T-array protocol for <em>in vivo</em> detection</h4>
<a id="article1.body1.sec2.sec6.p1" name="article1.body1.sec2.sec6.p1"></a><p>To test the applicability of the T-array protocol for detection of clones <em>in vivo,</em> we analyzed a well-characterized sample of FACS sorted, CMV-specific, IFNÎ³-secreting CD4<sup>+</sup> T-cells from a renal transplant recipient 9 weeks after primary CMV infection, at the peak of viral load <a href="#pone.0000055-vanLeeuwen1">[18]</a>. This sample of 11,600 sorted CMV-specific T-cells was pre-amplified by anchored PCR <a href="#pone.0000055-Zhu1">[19]</a>, <a href="#pone.0000055-Douek1">[20]</a>, which was used here as pre- amplification step to generate sufficient cDNA from a relatively small amount of RNA (<a href="#pone-0000055-g005">
                  <strong>Fig. 5AâB</strong>
               </a>). Spectratyping indicated a relatively broad repertoire <a href="#pone.0000055-vanLeeuwen1">[18]</a>. Within the repertoire 11 VÎ² families were extensively analyzed by cloning and sequencing <a href="#pone.0000055-vanLeeuwen1">[18]</a>. In the VÎ²6.1 pool, 60 clones were sequenced, revealing 12 unique sequences of which 4 were JÎ²2.7<sup>+</sup>. A T-array was performed to screen the VÎ²6.1- JÎ²2.7 subpopulation with an annealer oligonucleotide that detects JÎ²2.7 sequences with 3 or less nucleotides deleted from the JÎ²2.7 gene (<a href="#pone-0000055-g005">
                  <strong>Fig. 5F</strong>
               </a>). All 3 clones that meet these criteria were picked up with the T-array (<a href="#pone-0000055-g005">
                  <strong>Fig. 5E</strong>
               </a>, <a href="#pone.0000055.s007">
                  <strong>Table S5</strong>
               </a>). In addition, the T-array signal matched the clonal frequency of the T-cell clones identified. The clone with hexamer CGGCTC which was picked up in 5 out of 60 sequences gave the strongest signal, followed by clone GAGGAA (3 out of 60), and clone CCAGTC (1 out of 60), respectively. These data show that the T-array can detect <em>in vivo</em> expanded T-cell clones in a quantitative way.</p>
<div class="figure" id="pone-0000055-g005"><div class="img"><a name="pone-0000055-g005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000055.g005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000055" data-uri="info:doi/10.1371/journal.pone.0000055.g005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000055.g005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000055.g005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000055.g005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000055.g005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000055.g005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000055.g005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000055.g005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000055.g005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000055.g005.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 5.  <span>T-array analysis of CMV-specific cells <em>in vivo.</em>
                  </span></strong></p><a id="article1.body1.sec2.sec6.fig1.caption1.p1" name="article1.body1.sec2.sec6.fig1.caption1.p1"></a><p>
                     <strong>(A)</strong> Peripheral blood was drawn 9 weeks after primary CMV infection of a CMV-seronegative renal transplant recipient, and sorted for CMV+ IFNÎ³-producing T-cells.</p>
<a id="article1.body1.sec2.sec6.fig1.caption1.p2" name="article1.body1.sec2.sec6.fig1.caption1.p2"></a><p>
                     <strong>(B)</strong> cDNA was amplified using anchored PCR (<em>
                        <a href="#pone.0000055-Douek1">ref. 20</a>
                     </em>).</p>
<a id="article1.body1.sec2.sec6.fig1.caption1.p3" name="article1.body1.sec2.sec6.fig1.caption1.p3"></a><p>
                     <strong>(C)</strong> VÎ²'s were analyzed by spectratyping and 11 VÎ²-families were cloned and sequenced (<em>
                        <a href="#pone.0000055-vanLeeuwen1">ref. 18</a>
                     </em>).</p>
<a id="article1.body1.sec2.sec6.fig1.caption1.p4" name="article1.body1.sec2.sec6.fig1.caption1.p4"></a><p>
                     <strong>(D)</strong> The most prominent VÎ²/JÎ² combination as identified by (C), i.e. the VÎ²6.1/JÎ²2.7+pool, was amplified by VÎ²6.1-and JÎ²2.7-primers and loaded on a hexamer array.</p>
<a id="article1.body1.sec2.sec6.fig1.caption1.p5" name="article1.body1.sec2.sec6.fig1.caption1.p5"></a><p>
                     <strong>(E)</strong> CDR3 regions, clonal frequencies, and joining sequences of the identified VÎ²6.1/JÎ²2.7+T-cell clones.</p>
<a id="article1.body1.sec2.sec6.fig1.caption1.p6" name="article1.body1.sec2.sec6.fig1.caption1.p6"></a><p>
                     <strong>(F)</strong> VÎ²6.1/JÎ²2.7-specific T-array with the annealer CTACGAGCAGTACTTCGGG, which matches the germline JÎ²-2-7 sequence with 3 nucleotides deleted.</p>
<span>doi:10.1371/journal.pone.0000055.g005</span></div>
</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p>The diverse repertoire of TCR rearrangements can potentially be analyzed using microarrays, which have a high capacity to differentiate and monitor many unique DNA rearrangements in parallel. However, the size of the TCRÎ² repertoire at the DNA level is too large for full TCR repertoire analysis at single-clone resolution on a single microarray. The Î±Î² receptor diversity is estimated at 10<sup>15</sup> to 10<sup>18</sup> rearrangements <a href="#pone.0000055-Davis1">[1]</a>, <a href="#pone.0000055-Janeway1">[21]</a>, which is formed for a relatively large part by the Î² chain. Within one individual, however, the size of the Î² chain repertoire is much more limited (10<sup>6</sup>) <a href="#pone.0000055-Arstila1">[2]</a>, and microarrays could produce sufficient resolution to distinguish single T cell clones in the repertoire of one individual. Here, we use universal microarrays for this concept and show that this is feasible. In the design presented, T-arrays tag individual clones based on the sequence information in the NDN-J or NDN-V junction. The tag for each clone consists of the J- or V-family used, the number of terminal nucleotides deleted from this J- or V-gene segment and the first six nucleotides of the NDN region. This design creates tags that are specific for one in more than a million clones, which in theory allows single-clone analysis of the complete TCRÎ² repertoire on high-density microarrays.</p>
<a id="article1.body1.sec3.p2" name="article1.body1.sec3.p2"></a><p>The validity of the T-array protocol was shown in several experiments. Firstly, the PCR fragments derived from the TCR of Jurkat cells were selectively ligated to hexamer oligonucleotides complementary to its NDN sequence both in solution and using hexamer arrays (<a href="#pone-0000055-g002">
               <strong>Fig. 2</strong>
            </a>). Second, the protocol allowed early, highly specific identification of an expanding T cell clone after in vitro stimulation with CMV-peptide (<a href="#pone-0000055-g004">
               <strong>Fig. 4</strong>
            </a>). Third, T-cell clones from blood taken form CMV-infected individuals that were identified using T-arrays, were also detected by multiple cloning and sequencing (<a href="#pone.0000055.s006">
               <strong>Table S4</strong>
            </a>). Lastly, in FACS-sorted CMV-specific, IFNÎ³-secreting CD4+ T cells from a renal transplant patient 9 weeks after CMV infection T-arrays detected the dominant VÎ²6.1<sup>+</sup>/JÎ²2.7<sup>+</sup> clones identified earlier by extensive cloning and sequencing (<a href="#pone-0000055-g005">
               <strong>Fig. 5</strong>
            </a>).</p>
<a id="article1.body1.sec3.p3" name="article1.body1.sec3.p3"></a><p>The sensitivity of the protocol was determined after mixing a Jurkat T-cell clone in a background of peripheral blood CD4+ T-cells in a range of dilutions. The data show that the Jurkat TCR rearrangement was detected in a ratio of at least 1 in 10<sup>6</sup> (<a href="#pone-0000055-g003">
               <strong>Fig. 3</strong>
            </a>). This is 2 logs more sensitive than VÎ²-CÎ² spectratyping <a href="#pone.0000055-Pannetier1">[4]</a>, <a href="#pone.0000055-Hohlfeld1">[15]</a>, which can detect a T-cell clone in 1 in 10<sup>4</sup>. VÎ²-JÎ² spectratyping, an alternative approach which is not widely used, is theoretically 12-fold higher than that of VÎ² spectratyping and therefore 10-fold less sensitive than the T-array approach. The superior sensitivity of the T array was confirmed by detection of a CMV-specific T-cell clone which was identified in the unstimulated population of circulating T cells obtained directly from a CMV-infected donor (<a href="#pone-0000055-g003">
               <strong>Fig. 3</strong>
            </a>). This clone was only detected after 3 days of antigen stimulation by VÎ²-JÎ² spectratyping and after 6 days by VÎ²-CÎ² spectratyping (<a href="#pone-0000055-g004">
               <strong>Fig. 4</strong>
            </a>). Thus T-arrays make highly sensitive detection and tracking of T cells possible. <a href="#pone-0000055-g006">Figure 6</a> illustrates the sensitivity of various methods for the analysis of the TCRÎ² repertoire.</p>
<div class="figure" id="pone-0000055-g006"><div class="img"><a name="pone-0000055-g006" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000055.g006&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000055" data-uri="info:doi/10.1371/journal.pone.0000055.g006"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000055.g006&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000055.g006/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000055.g006/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000055.g006/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000055.g006/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000055.g006.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000055.g006/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000055.g006/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000055.g006.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 6.  <span>Comparison of the diversity of the human TCRb repertoire and the sensitivity of various methods for repertoire analysis.</span></strong></p><a id="article1.body1.sec3.fig1.caption1.p1" name="article1.body1.sec3.fig1.caption1.p1"></a><p>
                  <strong>(A)</strong> The human T-cell repertoire contains an estimated 10<sup>6</sup> rearrangements per individual (<em>
                     <a href="#pone.0000055-Arstila1">ref. 2</a>
                  </em>).</p>
<a id="article1.body1.sec3.fig1.caption1.p2" name="article1.body1.sec3.fig1.caption1.p2"></a><p>
                  <strong>(B)</strong> The sensitivity of VÎ²/CÎ² immunoscope is approximately 2 in 10<sup>4</sup> (<em>
                     <a href="#pone.0000055-Hohlfeld1">ref. 15</a> and </em>
                  <a href="#pone-0000055-g003">
                     <em>Fig. 3A</em>
                  </a>).</p>
<a id="article1.body1.sec3.fig1.caption1.p3" name="article1.body1.sec3.fig1.caption1.p3"></a><p>
                  <strong>(C)</strong> Based on the number of JÎ² primers, it is estimated that VÎ²/JÎ² immunoscope is 12-fold more sensitive than VÎ²-CÎ² immunoscope.</p>
<a id="article1.body1.sec3.fig1.caption1.p4" name="article1.body1.sec3.fig1.caption1.p4"></a><p>
                  <strong>(D,E)</strong> The sensitivity by which TCR rearrangements are picked by indivual cloning, depends on the number of clones sequenced (<em>
                     <a href="#pone.0000055-Betts1">ref. 5</a>â<a href="#pone.0000055-Rohrlich1">9</a>
                  </em>).</p>
<a id="article1.body1.sec3.fig1.caption1.p5" name="article1.body1.sec3.fig1.caption1.p5"></a><p>
                  <strong>(F)</strong> The sensitivity of the T-array is approximately 1 in 10<sup>6</sup> rearrangements (<a href="#pone-0000055-g003">
                     <em>Fig. 3C</em>
                  </a>).</p>
<span>doi:10.1371/journal.pone.0000055.g006</span></div><a id="article1.body1.sec3.p4" name="article1.body1.sec3.p4"></a><p>In addition, the protocol allowed quantitative monitoring of T cell clones. With decreasing numbers of Jurkat cells in a CD4 background the signal clearly decreased (<a href="#pone-0000055-g003">
               <strong>Fig. 3</strong>
            </a>). The increasing frequency of the CMV-specific clone in the in vitro experiment as evidenced by tetramer staining, and by spectratyping, was also reflected in signal intensities on the arrays (<a href="#pone-0000055-g004">
               <strong>Fig. 4</strong>
            </a>). Likewise, the observed clonal frequencies of the CMV-specific clones in the in vivo experiment (<a href="#pone.0000055.s007">
               <strong>Table S4</strong>
            </a>) were quantitatively reflected in the T-array data.</p>
<a id="article1.body1.sec3.p5" name="article1.body1.sec3.p5"></a><p>Although the ligation reaction is highly specific for the correct hexamer sequence, ligation mismatches did occur. However, in every instance true positives gave the strongest signal, even in complex mixtures. <a href="#pone-0000055-g002">Figure 2G</a> supports previous data <a href="#pone.0000055-Roth1">[14]</a>, <a href="#pone.0000055-James1">[22]</a> which show that mismatches occur mainly in the two nucleotide positions opposite of the site of ligation. Algorithms, based on known ligation patterns <a href="#pone.0000055-Roth1">[14]</a>, <a href="#pone.0000055-James1">[22]</a>, have been developed that identify false positives and reduce the loss of resolution when complex mixtures such as full-genome transcripts are analyzed on hexamer arrays <a href="#pone.0000055-Roth1">[14]</a>. Such algorithms may help to minimize the effect of cross ligations on the resolution of T-arrays and help to detect less frequent clones.</p>
<a id="article1.body1.sec3.p6" name="article1.body1.sec3.p6"></a><p>The technology described here can be applied to monitor a small selection of the TCRÎ² repertoire quantitatively, and to track a subset of T-cell clones sensitively and quantitatively. While the combination of spectratyping, cloning and sequencing may take several weeks, the T-array method takes only a single day including scanning and quantification. Furthermore it is sensitive, and allows monitoring of growth kinetics at the clonal level. This rapid and sensitive method may find applications in the study of the relation between clonal expansion of T cells and autoimmune phenomena, e.g. responses to immunotherapy, retrospectively and prospectively. Recurrence of autoimmune disease could be predicted in the case of previously identified clones <a href="#pone.0000055-Muraro1">[23]</a>, or the fate of T-cells in adoptive therapy against cancer <a href="#pone.0000055-Yee1">[24]</a> could be monitored at single-clone level.</p>
<a id="article1.body1.sec3.p7" name="article1.body1.sec3.p7"></a><p>One of the prospects of this technology is that it could possibly be developed into a tool that screens the complete TCRÎ² repertoire on a single array. The format presented here screens only 1/23 Â· 1/144 = 0.03% of the repertoire (<a href="#pone.0000055.s004">
               <strong>Table S2C</strong>
            </a>). Recently, we successfully explored the feasibility of a protocol in which T-array analysis is preceded by simultaneous amplification of all VÎ² families in one PCR reaction using anchored PCR as described earlier <a href="#pone.0000055-Zhu1">[19]</a>, <a href="#pone.0000055-Douek1">[20]</a> (data not shown). The resulting 144 arrays can then be housed in a high-density matrix of multiple arrays that can be individually loaded. Such matrices have recently become available <a href="#pone.0000055-Kuhn1">[25]</a>. Rapid, quantitative and sensitive full repertoire screening would have significant impact in immunological research and on the development of immunotherapeutics. Identical arrays might be built for the analysis of the TCRÎ±, -Î³ and -Î´ repertoires and of the B-cell receptor repertoire in humans and other species.</p>
<a id="article1.body1.sec3.p8" name="article1.body1.sec3.p8"></a><p>In conclusion, here we show proof of concept of an approach to sensitively monitor changes in the frequency of unique TCR rearrangements using microarrays. The protocol is rapid and universal for the detection of all T- and B-cell receptor rearrangements. We propose that this technology will be useful for monitoring of clonal T- and B-cell expansions for many applications in medical research.</p>
</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4>Analysis of CDR3 sequences from public database</h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p>TCR Î²-CDR3 mRNA sequences of human T-cell clones were collected from the public database of NCBI at NIH. VÎ²-, JÎ²-, DÎ²-segments were identified using the V-QUEST algorithm from the international ImMunoGeneTics information system <a href="#pone.0000055-Ruiz1">[13]</a>. A number of 50 sequences were validated manually, and assignment errors were identified only for N-deletions larger than 8 nucleotides. To exclude other assignment errors all CDR3Î² sequences with N-deletions larger than 7 nucleotides were therefore assigned manually.</p>


<h4>Cells and flow cytometry</h4>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1"></a><p>Jurkat cell line clone E6-1 (ATCC, Manassas, VA) was grown in DMEM culture medium (Sigma-Aldrich, St. Louis, MO) supplemented with 5% FCS. Human peripheral blood mononuclear cells (PBMC) (<a href="#pone-0000055-g003">Figure 3</a>) were isolated from buffy coats of healthy blood donors by density centrifugation with Ficoll-Isopaque (Pharmacia Biotech, Uppsala, S). Informed consent was obtained from blood donors. CD4<sup>+</sup> T cells were isolated by using anti-CD4 microbeads (Miltenyi Biotec, Bergisch Gladbach, D), followed by positive selection with the VarioMACS (Miltenyi Biotec) according to the manufacturer's protocol. The purity of the CD4+ cells isolated was measured using anti-CD4 PerCP-conjugated antibodies (BD Biosciences, San Jose, CA).</p>
<a id="article1.body1.sec4.sec2.p2" name="article1.body1.sec4.sec2.p2"></a><p>Thawed PBMCs (<a href="#pone-0000055-g004">Figure 4</a>) were resuspended in IMDM (BioWhittaker, Verviers, Belgium), containing 10% FCS and antibiotics (100 U/ml sodium penicillin G and 100 Âµg/ml streptomycin sulfate). Cells were washed in PBS containing 0.01% (w/v) NaN<sub>3</sub> and 0.5% (w/v) BSA (PBA). A total of 250,000 PBMCs were incubated with an appropriate concentration of tetrameric complexes in a small volume for 10 min at 4Â°C. Subsequently, fluorescently labelled conjugated mAbs (concentrations according to manufacturer's instructions) were added and incubated for 30 min at 4Â°C. For analysis of expression of surface markers, the following Abs were used: the allophycocyanin-conjugated HLA-A2 tetramer loaded with the CMV pp65-derived NLVPMVATV peptide <a href="#pone.0000055-Hohlfeld1">[15]</a>, and anti-CD8 PerCPâconjugated antibodies (BD Biosciences, San Jose, CA).</p>
<a id="article1.body1.sec4.sec2.p3" name="article1.body1.sec4.sec2.p3"></a><p>CMV-specific IFN-Î³-producing CD4+ cells from a renal transplant recipient were isolated using IFNÎ³ Secretion Assay Detection Kit (PE) (Miltenyi Biotec, Amsterdam, The Netherlands) according to the manufacturer's conditions. At the moment of peak viral load, 9 weeks after transplantation, PBMCs were isolated and stimulated for 16 hours with CMV antigen (10 Âµl/ml) and incubated with IFNÎ³ Catch Reagent for 5 minutes at 4Â°C, incubated with IFNÎ³ Detection Antibody (PE), CD4 APC (BD Pharmingen, San Diego, USA) and sorted using FACsARIA (BD). The patient had given written informed consent, and the local medical ethics committee had approved the study.</p>


<h4>Expansion of virus-specific autologous cytotoxic T-lymphocytes</h4>
<a id="article1.body1.sec4.sec3.p1" name="article1.body1.sec4.sec3.p1"></a><p>PBMCs from a CMV seropositive, HLA-A2<sup>+</sup> healthy volunteer donor were used for expansion of CMV specific CD8<sup>+</sup> cells. Informed consent was obtained from the blood donor. PBMCs were stimulated in IMDM supplemented with 10% human pool serum, antibiotics, and 2-ME with CMVpp65-A2 peptide NLVPMVATV (1.25 Âµg/ml) and IL-2 (50 U/ml Biotest, Dreieich, D) in 24-well plates. After one week, cells were restimulated on a weekly basis with irradiated (30 Gy) CMV-pp65-A2 peptide loaded EBV transformed cell-lines expressing HLA-A2<sup>+</sup> (5Ã10<sup>4</sup> cells/ml) in the presence of IL-2.</p>


<h4>Cloning and sequencing</h4>
<a id="article1.body1.sec4.sec4.p1" name="article1.body1.sec4.sec4.p1"></a><p>VÎ² PCR products were purified and ligated into pGEM-T Easy Vector (Promega, Madison, WI) and cloned by transformation of competent DH5Î± <em>E. coli</em>. Selected colonies were amplified by PCR using M13 primers (Invitrogen - Life Technologies, Breda, NL) and then sequenced on the ABI Prism 3730 DNA automatic sequencer (Applied Biosystem, Foster City, CA) using the dye terminator cycle sequencing chemistry (v1.1) (Perkin Elmer, Foster City, CA). Clones that did not yield a PCR product using direct colony-PCR, were cultured in LB medium, plasmid DNA was purified using the Wizard Plus Minipreps DNA purification system (Promega, Madison, WI), and plasmids were sequenced similarly as described above.</p>


<h4>PCR and Spectratyping analysis</h4>
<a id="article1.body1.sec4.sec5.p1" name="article1.body1.sec4.sec5.p1"></a><p>RNA was isolated using the GenElute Mammalian Total RNA Kit (Sigma-Aldrich, Zwijndrecht, NL). For experiments shown in <a href="#pone-0000055-g002">Figures 2</a>, <a href="#pone-0000055-g003">3</a> and <a href="#pone-0000055-g004">4</a>, cDNA was synthesized using Superscript RT II and oligo-dT primers (Sigma-Aldrich) according to the manufacturer's protocol (InVitrogen - Life Technologies, Breda, NL). For experiments shown in <a href="#pone-0000055-g005">Figure 5</a>, cDNA was synthesized using the Super Smartâ¢ and the Smartâ¢ cDNA synthesis kit (Clontech, Mountain View, CA), respectively. PCR was performed with TCR VÎ² primers <a href="#pone.0000055-Doumaid1">[26]</a> in combination with a TCR CÎ² primer, labelled with fluorescent dye fluorophore fluorescamine (FAM). Each amplification reaction was performed with 4 Î¼l cDNA in the presence of 25 pmol 5â² sense TCR VÎ² primer, 25 pmol 3â² antisense TCR CÎ² primer, 0.5 mM MgCl<sub>2</sub>, 0.5 mM dNTP, 10 mM TrisâHCl (pH 8.4), 50 mM KCl, 4 mM KCl, 2.5 units AmpliTaq DNA polymerase (Perkin Elmer/Roche Molecular Systems Inc., Branchburg, NJ) in a total volume of 40 Âµl. PCR cycles were performed in a T1 Thermocycler (Biometra, Goettingen, D). The FAM-labelled PCR products were run on the ABI Prism 3100 Genetic Analyzer capillary system (Applied Biosystem, Foster City, CA) using POP6 as separation matrix, filter set D for the detection of fluorescent signals, and ROX500 as internal size standard. Genescan Software (Applied Biosystem, Foster City, CA) was used for size determination and quantification.</p>


<h4>T-array protocol</h4>
<a id="article1.body1.sec4.sec6.p1" name="article1.body1.sec4.sec6.p1"></a><p>After cDNA synthesis (1C1), the T-array protocol follows PCR amplification (1C2), isolation of polyclonal, single strands (1C3), hybridization of annealer oligonucleotides (1C4), and ligation, washing, scanning, and quantification of hexamer arrays (1C5âC7).</p>
<h5>PCR amplification (<a href="#pone-0000055-g001">Fig. 1C2</a>).</h5><a id="article1.body1.sec4.sec6.sec1.p1" name="article1.body1.sec4.sec6.sec1.p1"></a><p>Biotinylated PCR products were obtained using sense biotinylated VÎ² primers against reverse, antisense CÎ² or JÎ² primers (PCR conditions as described above). For the analysis of CMV-cells <em>in vivo</em> (<a href="#pone-0000055-g005">
                     <strong>Fig. 5</strong>
                  </a>), cDNA was synthesized using the smart PCR cDNA synthesis kit (Clontech, Mountain View, CA).</p>

<h5>Isolation of single strands (<a href="#pone-0000055-g001">Fig. 1C3</a>).</h5><a id="article1.body1.sec4.sec6.sec2.p1" name="article1.body1.sec4.sec6.sec2.p1"></a><p>1.0 mg streptavidin-coated magnetic beads (M-280 Dynabeads, Dynal Biotech, Oslo, N) were washed twice in B&amp;W buffer (Dynal Biotech, Oslo, N) and biotinylated PCR products were linked to the magnetic beads according to the suppliers protocol. Non-bound DNA and nucleotides were removed by washing in 1x and subsequently 0.4Ã B&amp;W buffer. The non-biotinylated single strands were released by 10 minutes incubation in 0.15 N NaOH. After magnetic separation, supernatant containing the non-biotinylated single strands was pH neutralized using neutralization buffer (0.75 HCl, 0.125 M Tris, 16.7 mM MgCl<sub>2</sub>, 1.67 mM DTT).</p>

<h5>Hybridization of annealer oligonucleotides (<a href="#pone-0000055-g001">Fig. 1C4</a>).</h5><a id="article1.body1.sec4.sec6.sec3.p1" name="article1.body1.sec4.sec6.sec3.p1"></a><p>Single strands were then incubated with 1 pmol Cy5-labeled, 5â² phosphorylated annealer oligonucleotide (Biolegio, Malden, NL) at a starting temperature of 90Â°C. The heated water bath was (passively) cooled to ambient temperature. Sequences of used annealer oligonucleotides are CTAACTATGGC-TACACCTTCGGTTT (<a href="#pone-0000055-g002">Fig. 2</a>, <a href="#pone-0000055-g003">3</a>), AAACTGCTGGCACAGAAGTACACTT (<a href="#pone-0000055-g002">Fig 2d,e</a>), ACTATGGCTACACCTTCGGTT (<a href="#pone-0000055-g004">Fig. 4</a>) and CTACGAGCAGTACTTCGGG (<a href="#pone-0000055-g005">Fig. 5</a>).</p>

<h5>Ligation, washing, scanning, and quantification (<a href="#pone-0000055-g001">Fig. 1C5âC7</a>).</h5><a id="article1.body1.sec4.sec6.sec4.p1" name="article1.body1.sec4.sec6.sec4.p1"></a><p>Ligation on arrays (Accessarray, Expresson Biosystems, Roslin, UK) was performed at 30Â°C in a volume of 125 Âµl in 1Ã BSA (NEB, Ipswich, MA), 25 Âµl 5Ã DNA Ligase buffer, 12 units T4 DNA ligase. After ligation slides were washed in 0.1% SDS at 90Â°C, ddH2O at RT, and dried by 500Ã <em>g</em> centrifugation for 3 minutes. Ligated arrays were scanned with a GSI Lumonics ScanArray 5000 (Perkin-Elmer Life Sciences, Boston, MA). Spot intensities were quantified using with ArrayVision 6.0 software (Image Research, St. Catharines, Ontario, CDN).</p>
<a id="article1.body1.sec4.sec6.sec4.p2" name="article1.body1.sec4.sec6.sec4.p2"></a><p>T-array data in a format according to the MIAME guidelines checklist <a href="http://www.mged.org/Workgroups/MIAME/miame_checklist.html">www.mged.org/Workgroups/MIAME/miame_checâklist.html</a> are available on request.</p>



<h4>Ligation in <em>solution</em>
            </h4>
<a id="article1.body1.sec4.sec7.p1" name="article1.body1.sec4.sec7.p1"></a><p>For experiments shown in <a href="#pone-0000055-g002">
                  <strong>Fig. 2BâE</strong>
               </a>, ligation was performed <em>in solution</em> with single hexamer oligonucleotides. 1 pmol of hexamers, 4 units of T4 DNA ligase and 2 Âµl 5Ã DNA Ligase buffer (In vitrogen - Life Technologies, Breda, NL) and template/annealer complex were added in a total volume of 10 Âµl and incubated for 45 minutes at 16Â°C, followed by a 10 minutes denaturation step at 65Â°C. Ligation products were analyzed on the ABI Prism 3100 Genetic Analyzer capillary system and Genescan software as described above.</p>

</div>

<div id="section5" class="section"><a id="s5" name="s5" toc="s5" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0000055"></div><a name="pone.0000055.s001" id="pone.0000055.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000055.s001">Figure S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1"></a><p class="preSiDOI">Semi-quantitative PCR Jurkat cells and CD4+ cells.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000055.s001</p><a id="article1.body1.sec5.supplementary-material1.caption1.p2" name="article1.body1.sec5.supplementary-material1.caption1.p2"></a><p class="postSiDOI">(0.06 MB DOC)</p>
<a name="pone.0000055.s002" id="pone.0000055.s002"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000055.s002">Figure S2. </a></strong></p><a id="article1.body1.sec5.supplementary-material2.caption1.p1" name="article1.body1.sec5.supplementary-material2.caption1.p1"></a><p class="preSiDOI">Germ line signals in T-array.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000055.s002</p><a id="article1.body1.sec5.supplementary-material2.caption1.p2" name="article1.body1.sec5.supplementary-material2.caption1.p2"></a><p class="postSiDOI">(0.05 MB DOC)</p>
<a name="pone.0000055.s003" id="pone.0000055.s003"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000055.s003">Table S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material3.caption1.p1" name="article1.body1.sec5.supplementary-material3.caption1.p1"></a><p class="preSiDOI">N-deletion in 192 TCRÎ² sequences from public databases.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000055.s003</p><a id="article1.body1.sec5.supplementary-material3.caption1.p2" name="article1.body1.sec5.supplementary-material3.caption1.p2"></a><p class="postSiDOI">(0.51 MB DOC)</p>
<a name="pone.0000055.s004" id="pone.0000055.s004"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000055.s004">Table S2. </a></strong></p><a id="article1.body1.sec5.supplementary-material4.caption1.p1" name="article1.body1.sec5.supplementary-material4.caption1.p1"></a><p class="preSiDOI">Maximum resolution for spectratyping and T-array protocol.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000055.s004</p><a id="article1.body1.sec5.supplementary-material4.caption1.p2" name="article1.body1.sec5.supplementary-material4.caption1.p2"></a><p class="postSiDOI">(0.06 MB DOC)</p>
<a name="pone.0000055.s005" id="pone.0000055.s005"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000055.s005">Table S3. </a></strong></p><a id="article1.body1.sec5.supplementary-material5.caption1.p1" name="article1.body1.sec5.supplementary-material5.caption1.p1"></a><p class="preSiDOI">Signal intensity T arrays.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000055.s005</p><a id="article1.body1.sec5.supplementary-material5.caption1.p2" name="article1.body1.sec5.supplementary-material5.caption1.p2"></a><p class="postSiDOI">(0.12 MB DOC)</p>
<a name="pone.0000055.s006" id="pone.0000055.s006"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000055.s006">Table S4. </a></strong></p><a id="article1.body1.sec5.supplementary-material6.caption1.p1" name="article1.body1.sec5.supplementary-material6.caption1.p1"></a><p class="preSiDOI">Comparing T-array signals to results from cloning and sequencing in the ex vivo CMV-stimulation experiment.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000055.s006</p><a id="article1.body1.sec5.supplementary-material6.caption1.p2" name="article1.body1.sec5.supplementary-material6.caption1.p2"></a><p class="postSiDOI">(0.29 MB DOC)</p>
<a name="pone.0000055.s007" id="pone.0000055.s007"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000055.s007">Table S5. </a></strong></p><a id="article1.body1.sec5.supplementary-material7.caption1.p1" name="article1.body1.sec5.supplementary-material7.caption1.p1"></a><p class="preSiDOI">T-array analysis of CMV-specific cells <em>in vivo</em>.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000055.s007</p><a id="article1.body1.sec5.supplementary-material7.caption1.p2" name="article1.body1.sec5.supplementary-material7.caption1.p2"></a><p class="postSiDOI">(0.07 MB DOC)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>We thank F. van der Velde for assistance with bioinformatics.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: PT RV Nd HB FB ER It. Performed the experiments: Nd HB It. Analyzed the data: PT RV Nd HB FB It. Contributed reagents/materials/analysis tools: PT RV HB FB ER It. Wrote the paper: PT RV Nd HB FB ER It.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0000055-Davis1" id="pone.0000055-Davis1"></a>Davis MM, Bjorkman PJ (1988) T cell antigen receptor genes and T cell recognition. Nature  334: 395â402.  <ul class="find" data-citedArticleID="917438" data-doi="10.1038/334395a0"><li><a href="http://dx.doi.org/10.1038/334395a0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=T+cell+antigen+receptor+genes+and+T+cell+recognition." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22T+cell+antigen+receptor+genes+and+T+cell+recognition.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0000055-Arstila1" id="pone.0000055-Arstila1"></a>Arstila TP, Casrouge A, Baron V, Even J, Kanellopoulos J, et al.  (1999) A direct estimate of the human alphabeta T cell receptor diversity. Science  286: 958â961.  <ul class="find" data-citedArticleID="917432" data-doi="10.1126/science.286.5441.958"><li><a href="http://dx.doi.org/10.1126/science.286.5441.958" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+direct+estimate+of+the+human+alphabeta+T+cell+receptor+diversity." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+direct+estimate+of+the+human+alphabeta+T+cell+receptor+diversity.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0000055-Arstila2" id="pone.0000055-Arstila2"></a>Arstila TP, Casrouge A, Baron V, Even J, Kanellopoulos J, et al.  (2000) Diversity of human alphabeta T cell receptors. Science  288: 1135.  <ul class="find" data-citedArticleID="917434"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Diversity+of+human+alphabeta+T+cell+receptors.&amp;auth=&amp;atitle=Diversity+of+human+alphabeta+T+cell+receptors." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Diversity+of+human+alphabeta+T+cell+receptors." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Diversity+of+human+alphabeta+T+cell+receptors.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0000055-Pannetier1" id="pone.0000055-Pannetier1"></a>Pannetier C, Cochet M, Darche S, Casrouge A, Zoller M, Kourilsky P (1993) The sizes of the CDR3 hypervariable regions of the murine T-cell receptor beta chains vary as a function of the recombined germ-line segments. Proc Natl Acad Sci U S A  90: 4319â4323.  <ul class="find" data-citedArticleID="917458" data-doi="10.1073/pnas.90.9.4319"><li><a href="http://dx.doi.org/10.1073/pnas.90.9.4319" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+sizes+of+the+CDR3+hypervariable+regions+of+the+murine+T-cell+receptor+beta+chains+vary+as+a+function+of+the+recombined+germ-line+segments." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+sizes+of+the+CDR3+hypervariable+regions+of+the+murine+T-cell+receptor+beta+chains+vary+as+a+function+of+the+recombined+germ-line+segments.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0000055-Betts1" id="pone.0000055-Betts1"></a>Betts MR, Exley B, Price DA, Bansal A, Camacho ZT, et al.  (2005) Characterization of functional and phenotypic changes in anti-Gag vaccine-induced T cell responses and their role in protection after HIV-1 infection Proc Natl Acad Sci U S A  102: 4512â4517.  <ul class="find" data-citedArticleID="917436" data-doi="10.1073/pnas.0408773102"><li><a href="http://dx.doi.org/10.1073/pnas.0408773102" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Characterization+of+functional+and+phenotypic+changes+in+anti-Gag+vaccine-induced+T+cell+responses+and+their+role+in+protection+after+HIV-1+infection" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Characterization+of+functional+and+phenotypic+changes+in+anti-Gag+vaccine-induced+T+cell+responses+and+their+role+in+protection+after+HIV-1+infection%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0000055-HaceinBeyAbina1" id="pone.0000055-HaceinBeyAbina1"></a>Hacein-Bey-Abina S, Von Kalle C, Schmidt M, McCormack MP, Wulffraat N, et al.  (2003) LMO2-associated clonal T cell proliferation in two patients after gene therapy for SCID-X1. Science  302: 415â419.  <ul class="find" data-citedArticleID="917444" data-doi="10.1126/science.1088547"><li><a href="http://dx.doi.org/10.1126/science.1088547" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=LMO2-associated+clonal+T+cell+proliferation+in+two+patients+after+gene+therapy+for+SCID-X1." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22LMO2-associated+clonal+T+cell+proliferation+in+two+patients+after+gene+therapy+for+SCID-X1.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0000055-Price1" id="pone.0000055-Price1"></a>Price DA, West SM, Betts MR, Ruff LE, Brenchley JM, et al.  (2004) T cell receptor recognition motifs govern immune escape patterns in acute SIV infection. Immunity  21: 793â803.  <ul class="find" data-citedArticleID="917462" data-doi="10.1016/j.immuni.2004.10.010"><li><a href="http://dx.doi.org/10.1016/j.immuni.2004.10.010" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=T+cell+receptor+recognition+motifs+govern+immune+escape+patterns+in+acute+SIV+infection." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22T+cell+receptor+recognition+motifs+govern+immune+escape+patterns+in+acute+SIV+infection.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0000055-Price2" id="pone.0000055-Price2"></a>Price DA, Brenchley JM, Ruff LE, Betts MR, Hill BJ, et al.  (2005) Avidity for antigen shapes clonal dominance in CD8+ T cell populations specific for persistent DNA viruses J Exp Med  202: 1349â1361.  <ul class="find" data-citedArticleID="917464" data-doi="10.1084/jem.20051357"><li><a href="http://dx.doi.org/10.1084/jem.20051357" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Avidity+for+antigen+shapes+clonal+dominance+in+CD8%2B+T+cell+populations+specific+for+persistent+DNA+viruses" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Avidity+for+antigen+shapes+clonal+dominance+in+CD8%2B+T+cell+populations+specific+for+persistent+DNA+viruses%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0000055-Rohrlich1" id="pone.0000055-Rohrlich1"></a>Rohrlich PS, Fazilleau N, Ginhoux F, Firat H, Michel F, et al.  (2005) Direct recognition by cytolytic T cells of Hfe, a MHC class Ib molecule without antigen-presenting function. Proc Natl Acad Sci U S A  102: 12855â12860.  <ul class="find" data-citedArticleID="917466" data-doi="10.1073/pnas.0502309102"><li><a href="http://dx.doi.org/10.1073/pnas.0502309102" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Direct+recognition+by+cytolytic+T+cells+of+Hfe%2C+a+MHC+class+Ib+molecule+without+antigen-presenting+function." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Direct+recognition+by+cytolytic+T+cells+of+Hfe%2C+a+MHC+class+Ib+molecule+without+antigen-presenting+function.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0000055-Pannetier2" id="pone.0000055-Pannetier2"></a>Pannetier C, Even J, Kourilsky P (1995) T-cell repertoire diversity and clonal expansions in normal and clinical samples. Immunol Today  16: 176â181.  <ul class="find" data-citedArticleID="917460" data-doi="10.1016/0167-5699(95)80117-0"><li><a href="http://dx.doi.org/10.1016/0167-5699(95)80117-0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=T-cell+repertoire+diversity+and+clonal+expansions+in+normal+and+clinical+samples." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22T-cell+repertoire+diversity+and+clonal+expansions+in+normal+and+clinical+samples.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0000055-Kronenberg1" id="pone.0000055-Kronenberg1"></a>Kronenberg M, Siu G, Hood LE, Shastri NA (1986) The molecular genetics of the T-cell antigen receptor and T-cell antigen recognition. Ann Rev Imm  4: 529â591.  <ul class="find" data-citedArticleID="917452" data-doi="10.1146/annurev.iy.04.040186.002525"><li><a href="http://dx.doi.org/10.1146/annurev.iy.04.040186.002525" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+molecular+genetics+of+the+T-cell+antigen+receptor+and+T-cell+antigen+recognition." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+molecular+genetics+of+the+T-cell+antigen+receptor+and+T-cell+antigen+recognition.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0000055-Arden1" id="pone.0000055-Arden1"></a>Arden B, Clark SP, Kabelitz D, Mak TW (1995) Human T-cell receptor variable gene segment families. Immunogenetics  42: 455â500.  <ul class="find" data-citedArticleID="917430" data-doi="10.1007/bf00172176"><li><a href="http://dx.doi.org/10.1007/bf00172176" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Human+T-cell+receptor+variable+gene+segment+families." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Human+T-cell+receptor+variable+gene+segment+families.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0000055-Ruiz1" id="pone.0000055-Ruiz1"></a>Ruiz M, Giudicelli V, Ginestoux C, Stoehr P, Robinson J, et al.  (2000) IMGT the international ImMunoGeneTics database. Nucleic Acids Res  28: 219â221.  <ul class="find" data-citedArticleID="917470" data-doi="10.1093/nar/28.1.219"><li><a href="http://dx.doi.org/10.1093/nar/28.1.219" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=IMGT+the+international+ImMunoGeneTics+database." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22IMGT+the+international+ImMunoGeneTics+database.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0000055-Roth1" id="pone.0000055-Roth1"></a>Roth ME, Feng L, McConnell KJ, Schaffer PJ, Guerra CE, Affourtit JP, et al.  (2004) Expression profiling using a hexamer-based universal microarray. Nature Biotechnol  22: 418â426.  <ul class="find" data-citedArticleID="917468" data-doi="10.1038/nbt948"><li><a href="http://dx.doi.org/10.1038/nbt948" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Expression+profiling+using+a+hexamer-based+universal+microarray." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Expression+profiling+using+a+hexamer-based+universal+microarray.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0000055-Hohlfeld1" id="pone.0000055-Hohlfeld1"></a>Hohlfeld R (2003) Hunting (auto)immune T cells in neuroimmunological diseases. Brain  126: 2â4.  <ul class="find" data-citedArticleID="917446" data-doi="10.1093/brain/awg001"><li><a href="http://dx.doi.org/10.1093/brain/awg001" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Hunting+%28auto%29immune+T+cells+in+neuroimmunological+diseases." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Hunting+%28auto%29immune+T+cells+in+neuroimmunological+diseases.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0000055-Wills1" id="pone.0000055-Wills1"></a>Wills MR, Carmichael AJ, Mynard K, Jin X, Weekes MP, et al.  (1996) The human cytotoxic T-lymphocyte (CTL) response to cytomegalovirus is dominated by structural protein pp65: frequency specificity and T- cell receptor usage of pp65-specific CTL. J Virol  70: 7569â7579.  <ul class="find" data-citedArticleID="917472"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=The+human+cytotoxic+T-lymphocyte+%28CTL%29+response+to+cytomegalovirus+is+dominated+by+structural+protein+pp65%3A+frequency+specificity+and+T-+cell+receptor+usage+of+pp65-specific+CTL.&amp;auth=&amp;atitle=The+human+cytotoxic+T-lymphocyte+%28CTL%29+response+to+cytomegalovirus+is+dominated+by+structural+protein+pp65%3A+frequency+specificity+and+T-+cell+receptor+usage+of+pp65-specific+CTL." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+human+cytotoxic+T-lymphocyte+%28CTL%29+response+to+cytomegalovirus+is+dominated+by+structural+protein+pp65%3A+frequency+specificity+and+T-+cell+receptor+usage+of+pp65-specific+CTL." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+human+cytotoxic+T-lymphocyte+%28CTL%29+response+to+cytomegalovirus+is+dominated+by+structural+protein+pp65%3A+frequency+specificity+and+T-+cell+receptor+usage+of+pp65-specific+CTL.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0000055-Zanders1" id="pone.0000055-Zanders1"></a>Zanders ED, Lamb JR, Feldmann M, Green N, Beverley PC (1983) Tolerance of T-cell clones is associated with membrane antigen changes. Nature  303: 625â627.  <ul class="find" data-citedArticleID="917476" data-doi="10.1038/303625a0"><li><a href="http://dx.doi.org/10.1038/303625a0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Tolerance+of+T-cell+clones+is+associated+with+membrane+antigen+changes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Tolerance+of+T-cell+clones+is+associated+with+membrane+antigen+changes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0000055-vanLeeuwen1" id="pone.0000055-vanLeeuwen1"></a>van Leeuwen EM, Remmerswaal EB, Heemskerk MH, ten Berge IJ, van Lier RA (2006) Strong selection of virus-specific cytotoxic CD4+ T cell clones during primary human cytomegalovirus infection. Blood.   In press, published online DOI 10.1182/blood-2006-03-006809.  <ul class="find" data-citedArticleID="917480" data-doi="10.1182/blood-2006-03-006809"><li><a href="http://dx.doi.org/10.1182/blood-2006-03-006809" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Strong+selection+of+virus-specific+cytotoxic+CD4%2B+T+cell+clones+during+primary+human+cytomegalovirus+infection." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Strong+selection+of+virus-specific+cytotoxic+CD4%2B+T+cell+clones+during+primary+human+cytomegalovirus+infection.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0000055-Zhu1" id="pone.0000055-Zhu1"></a>Zhu YY, Machleder EM, Chenchik A, Li R, Siebert PD (2001) Reverse transcriptase template switching: A SMART (TM) approach for full-length cDNA library construction. BioTechniques  30: 892â897.  <ul class="find" data-citedArticleID="917478"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Reverse+transcriptase+template+switching%3A+A+SMART+%28TM%29+approach+for+full-length+cDNA+library+construction.&amp;auth=&amp;atitle=Reverse+transcriptase+template+switching%3A+A+SMART+%28TM%29+approach+for+full-length+cDNA+library+construction." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Reverse+transcriptase+template+switching%3A+A+SMART+%28TM%29+approach+for+full-length+cDNA+library+construction." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Reverse+transcriptase+template+switching%3A+A+SMART+%28TM%29+approach+for+full-length+cDNA+library+construction.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0000055-Douek1" id="pone.0000055-Douek1"></a>Douek DC, Betts MR, Brenchley JM, Hill BJ, Ambrozak DR, et al.  (2002) A novel approach to the analysis of specificity clonality and frequency of HIV-specific T cell responses reveals a potential mechanism for control of viral escape. J Immunol  168: 3099â3104.  <ul class="find" data-citedArticleID="917440"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=A+novel+approach+to+the+analysis+of+specificity+clonality+and+frequency+of+HIV-specific+T+cell+responses+reveals+a+potential+mechanism+for+control+of+viral+escape.&amp;auth=&amp;atitle=A+novel+approach+to+the+analysis+of+specificity+clonality+and+frequency+of+HIV-specific+T+cell+responses+reveals+a+potential+mechanism+for+control+of+viral+escape." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+novel+approach+to+the+analysis+of+specificity+clonality+and+frequency+of+HIV-specific+T+cell+responses+reveals+a+potential+mechanism+for+control+of+viral+escape." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+novel+approach+to+the+analysis+of+specificity+clonality+and+frequency+of+HIV-specific+T+cell+responses+reveals+a+potential+mechanism+for+control+of+viral+escape.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0000055-Janeway1" id="pone.0000055-Janeway1"></a>Janeway C, Travers P, Walport M, Shlomchik M (2004) Immunobiology,. Edinburgh: Garland Science Textbooks.   6<sup>th</sup> edition.  <ul class="find-nolinks"></ul></li><li><span class="label">22.
              </span><a name="pone.0000055-James1" id="pone.0000055-James1"></a>James KD, Boles AR, Henckel D, Ellington AD (1998) The fidelity of template-directed oligonucleotide ligation and its relevance to DNA computation. Nucleic Acids Res  26: 5203â5211.  <ul class="find" data-citedArticleID="917448" data-doi="10.1093/nar/26.22.5203"><li><a href="http://dx.doi.org/10.1093/nar/26.22.5203" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+fidelity+of+template-directed+oligonucleotide+ligation+and+its+relevance+to+DNA+computation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+fidelity+of+template-directed+oligonucleotide+ligation+and+its+relevance+to+DNA+computation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0000055-Muraro1" id="pone.0000055-Muraro1"></a>Muraro PA, Wandinger KP, Bielekova B, Gran B, Marques A, et al.  (2003) Molecular tracking of antigen-specific T cell clones in neurological immune-mediated disorders. Brain  126: 20â31.  <ul class="find" data-citedArticleID="917456" data-doi="10.1093/brain/awg021"><li><a href="http://dx.doi.org/10.1093/brain/awg021" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Molecular+tracking+of+antigen-specific+T+cell+clones+in+neurological+immune-mediated+disorders." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Molecular+tracking+of+antigen-specific+T+cell+clones+in+neurological+immune-mediated+disorders.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0000055-Yee1" id="pone.0000055-Yee1"></a>Yee C, Thompson JA, Byrd D, Riddell SR, Roche P, et al.  (2002) Adoptive T cell therapy using antigen-specific CD8+ T cell clones for the treatment of patients with metastatic melanoma. Proc Natl Acad Sci U S A  99: 16168â16173.  <ul class="find" data-citedArticleID="917474" data-doi="10.1073/pnas.242600099"><li><a href="http://dx.doi.org/10.1073/pnas.242600099" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Adoptive+T+cell+therapy+using+antigen-specific+CD8%2B+T+cell+clones+for+the+treatment+of+patients+with+metastatic+melanoma." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Adoptive+T+cell+therapy+using+antigen-specific+CD8%2B+T+cell+clones+for+the+treatment+of+patients+with+metastatic+melanoma.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0000055-Kuhn1" id="pone.0000055-Kuhn1"></a>Kuhn K, Baker SC, Chudin E, Lieu MH, Oeser S, et al.  (2004) A novel high-performance random array platform for quantitative gene expression profiling. Genome Research  14: 2347â2356.  <ul class="find" data-citedArticleID="917454" data-doi="10.1101/gr.2739104"><li><a href="http://dx.doi.org/10.1101/gr.2739104" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+novel+high-performance+random+array+platform+for+quantitative+gene+expression+profiling." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+novel+high-performance+random+array+platform+for+quantitative+gene+expression+profiling.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0000055-Doumaid1" id="pone.0000055-Doumaid1"></a>Doumaid K, van Miert PP, Vaessen LM, Remmerswaal EB, Weimar W, et al.  (2000) Modulation of the T cell receptor beta chain repertoire after heart transplantation. Transpl Immunol  8: 83â94.  <ul class="find" data-citedArticleID="917442" data-doi="10.1016/s0966-3274(00)00008-3"><li><a href="http://dx.doi.org/10.1016/s0966-3274(00)00008-3" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Modulation+of+the+T+cell+receptor+beta+chain+repertoire+after+heart+transplantation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Modulation+of+the+T+cell+receptor+beta+chain+repertoire+after+heart+transplantation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000055.XML" value="104264"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000055.PDF" value="652541"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000055.e001.PNG" value="934"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000055.e001.TIF" value="3428"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000055.e002.PNG" value="934"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000055.e002.TIF" value="3432"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000055.e003.PNG" value="922"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000055.e003.TIF" value="3440"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000055.e004.PNG" value="1015"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000055.e004.TIF" value="3608"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000055.e005.PNG" value="943"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000055.e005.TIF" value="3540"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000055.g001.PNG_L" value="1216733"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000055.g001.PNG_M" value="120901"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000055.g001.PNG_S" value="5476"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000055.g001.TIF" value="1772726"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000055.g001.PNG_I" value="48780"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000055.e006.PNG" value="931"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000055.e006.TIF" value="3540"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000055.g002.PNG_L" value="1319593"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000055.g002.PNG_M" value="127825"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000055.g002.PNG_S" value="4738"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000055.g002.TIF" value="1945052"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000055.g002.PNG_I" value="48099"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000055.g003.PNG_L" value="2552362"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000055.g003.PNG_M" value="237236"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000055.g003.PNG_S" value="5691"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000055.g003.TIF" value="3463982"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000055.g003.PNG_I" value="73687"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000055.g004.PNG_L" value="1316204"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000055.g004.PNG_M" value="114673"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000055.g004.PNG_S" value="4495"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000055.g004.TIF" value="2242404"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000055.g004.PNG_I" value="59685"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000055.g005.PNG_L" value="558136"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000055.g005.PNG_M" value="58169"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000055.g005.PNG_S" value="3245"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000055.g005.TIF" value="861904"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000055.g005.PNG_I" value="24022"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000055.g006.PNG_L" value="594823"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000055.g006.PNG_M" value="59269"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000055.g006.PNG_S" value="2833"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000055.g006.TIF" value="724378"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000055.g006.PNG_I" value="21975"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000055.s001.DOC" value="56832"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000055.s002.DOC" value="54784"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000055.s003.DOC" value="506368"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000055.s004.DOC" value="57344"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000055.s005.DOC" value="117760"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000055.s006.DOC" value="295424"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000055.s007.DOC" value="74240"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000055&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000055" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000055&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0000055&volume=&issue=&title=Monitoring the T-Cell Receptor Repertoire at Single-Clone Resolution&author_name=Hendrik%20P.J.%20Bonarius%2C%20Frank%20Baas%2C%20Ester%20B.M.%20Remmerswaal%2C%20Ren%C3%A9%20A.W.%20van%20Lier%2C%20Ineke%20J.M.%20ten%20Berge%2C%20Paul%20P.%20Tak%2C%20Niek%20de%20Vries&start_page=1&end_page=10" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000055" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000055" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0000055" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000055&amp;t=Monitoring%20the%20T-Cell%20Receptor%20Repertoire%20at%20Single-Clone%20Resolution" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000055&title=Monitoring%20the%20T-Cell%20Receptor%20Repertoire%20at%20Single-Clone%20Resolution&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000055&amp;title=Monitoring%20the%20T-Cell%20Receptor%20Repertoire%20at%20Single-Clone%20Resolution" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000055" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0000055&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Monitoring the T-Cell Receptor Repertoire at Single-Clone Resolution';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0000055';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Monitoring%20the%20T-Cell%20Receptor%20Repertoire%20at%20Single-Clone%20Resolution http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000055" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0000055" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">




















          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Cloning%22" title="Search for articles in the subject area:'Cloning'"><div class="flagText">Cloning</div></a>
              <div data-categoryid="23551" data-articleid="22950"
                   data-categoryname="Cloning"
                   class="flagImage" title="Flag 'Cloning' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22DNA+sequence+analysis%22" title="Search for articles in the subject area:'DNA sequence analysis'"><div class="flagText">DNA sequence analysis</div></a>
              <div data-categoryid="47935" data-articleid="22950"
                   data-categoryname="DNA sequence analysis"
                   class="flagImage" title="Flag 'DNA sequence analysis' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22DNA+sequences%22" title="Search for articles in the subject area:'DNA sequences'"><div class="flagText">DNA sequences</div></a>
              <div data-categoryid="32003" data-articleid="22950"
                   data-categoryname="DNA sequences"
                   class="flagImage" title="Flag 'DNA sequences' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Nucleotide+sequencing%22" title="Search for articles in the subject area:'Nucleotide sequencing'"><div class="flagText">Nucleotide sequencing</div></a>
              <div data-categoryid="19663" data-articleid="22950"
                   data-categoryname="Nucleotide sequencing"
                   class="flagImage" title="Flag 'Nucleotide sequencing' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Oligonucleotides%22" title="Search for articles in the subject area:'Oligonucleotides'"><div class="flagText">Oligonucleotides</div></a>
              <div data-categoryid="19655" data-articleid="22950"
                   data-categoryname="Oligonucleotides"
                   class="flagImage" title="Flag 'Oligonucleotides' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Polymerase+chain+reaction%22" title="Search for articles in the subject area:'Polymerase chain reaction'"><div class="flagText">Polymerase chain reaction</div></a>
              <div data-categoryid="26251" data-articleid="22950"
                   data-categoryname="Polymerase chain reaction"
                   class="flagImage" title="Flag 'Polymerase chain reaction' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22T+cell+receptors%22" title="Search for articles in the subject area:'T cell receptors'"><div class="flagText">T cell receptors</div></a>
              <div data-categoryid="35041" data-articleid="22950"
                   data-categoryname="T cell receptors"
                   class="flagImage" title="Flag 'T cell receptors' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22T+cells%22" title="Search for articles in the subject area:'T cells'"><div class="flagText">T cells</div></a>
              <div data-categoryid="40147" data-articleid="22950"
                   data-categoryname="T cells"
                   class="flagImage" title="Flag 'T cells' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=1138'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=684'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=9473&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>

<div class="block sidebar-comments">
    <div class="header">
        <h3>Comments</h3>
    </div>
      <p><a href="/annotation/listThread.action?root=2137">Using gene chips to measure TCR diversity</a><br>Posted by pbaum</p>
      <p><a href="/annotation/listThread.action?root=1273">Addendum to figure 5F</a><br>Posted by niek_de_vries</p>
</div>

</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
