

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Proteomic Shifts in Embryonic Stem Cells with Gene Dose Modifications Suggest the Presence of Balancer Proteins in Protein Regulatory Networks</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0001218"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0001218" />

    <meta name="citation_title" content="Proteomic Shifts in Embryonic Stem Cells with Gene Dose Modifications Suggest the Presence of Balancer Proteins in Protein Regulatory Networks"/>
    <meta itemprop="name" content="Proteomic Shifts in Embryonic Stem Cells with Gene Dose Modifications Suggest the Presence of Balancer Proteins in Protein Regulatory Networks"/>

      <meta name="citation_author" content="Lei Mao"/>
            <meta name="citation_author_institution" content="Institute for Human Genetics, Charité-University Medicine Berlin, Germany"/>
            <meta name="citation_author_institution" content="Max Planck Institute for Molecular Genetics, Berlin, Germany"/>
      <meta name="citation_author" content="Claus Zabel"/>
            <meta name="citation_author_institution" content="Institute for Human Genetics, Charité-University Medicine Berlin, Germany"/>
      <meta name="citation_author" content="Marion Herrmann"/>
            <meta name="citation_author_institution" content="Institute for Human Genetics, Charité-University Medicine Berlin, Germany"/>
      <meta name="citation_author" content="Tobias Nolden"/>
            <meta name="citation_author_institution" content="Max Planck Institute for Molecular Genetics, Berlin, Germany"/>
      <meta name="citation_author" content="Florian Mertes"/>
            <meta name="citation_author_institution" content="Max Planck Institute for Molecular Genetics, Berlin, Germany"/>
      <meta name="citation_author" content="Laetitia Magnol"/>
            <meta name="citation_author_institution" content="Institut de Transgénose, IEM, UMR6218, CNRS Uni Orléans, Orléans, France"/>
      <meta name="citation_author" content="Caroline Chabert"/>
            <meta name="citation_author_institution" content="EA 3508, Université Paris Diderot-Paris 7, Paris, France"/>
      <meta name="citation_author" content="Daniela Hartl"/>
            <meta name="citation_author_institution" content="Institute for Human Genetics, Charité-University Medicine Berlin, Germany"/>
      <meta name="citation_author" content="Yann Herault"/>
            <meta name="citation_author_institution" content="Institut de Transgénose, IEM, UMR6218, CNRS Uni Orléans, Orléans, France"/>
      <meta name="citation_author" content="Jean Maurice Delabar"/>
            <meta name="citation_author_institution" content="EA 3508, Université Paris Diderot-Paris 7, Paris, France"/>
      <meta name="citation_author" content="Thomas Manke"/>
            <meta name="citation_author_institution" content="Max Planck Institute for Molecular Genetics, Berlin, Germany"/>
      <meta name="citation_author" content="Heinz Himmelbauer"/>
            <meta name="citation_author_institution" content="Max Planck Institute for Molecular Genetics, Berlin, Germany"/>
      <meta name="citation_author" content="Joachim Klose"/>
            <meta name="citation_author_institution" content="Institute for Human Genetics, Charité-University Medicine Berlin, Germany"/>

    <meta name="citation_date" content="2007/11/28"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0001218.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e1218"/>
    <meta name="citation_issue" content="11"/>
    <meta name="citation_volume" content="2"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Parkin-deficient mice exhibit nigrostriatal deficits but not loss of dopaminergic neurons.; citation_author=MS Goldberg; citation_author=SM Fleming; citation_author=JJ Palacino; citation_author=C Cepeda; citation_author=HA Lam; citation_journal_title=J Biol Chem; citation_volume=278; citation_number=1; citation_pages=43628-43635; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Exon 1 of the HD gene with an expanded CAG repeat is sufficient to cause a progressive neurological phenotype in transgenic mice.; citation_author=L Mangiarini; citation_author=K Sathasivam; citation_author=M Seller; citation_author=B Cozens; citation_author=A Harper; citation_journal_title=Cell; citation_volume=87; citation_number=2; citation_pages=493-506; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Mitochondrial dysfunction and oxidative damage in parkin-deficient mice.; citation_author=JJ Palacino; citation_author=D Sagi; citation_author=MS Goldberg; citation_author=S Krauss; citation_author=C Motz; citation_journal_title=J Biol Chem; citation_volume=279; citation_number=3; citation_pages=18614-18622; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Influence of Huntington's Disease on the Human and Mouse Proteome.; citation_author=C Zabel; citation_author=J Klose; citation_journal_title=Int Rev Neurobiol; citation_volume=61; citation_number=4; citation_pages=241-283; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Proteomic analysis of parkin knockout mice: alterations in energy metabolism, protein handling and synaptic function.; citation_author=M Periquet; citation_author=O Corti; citation_author=S Jacquier; citation_author=A Brice; citation_journal_title=J Neurochem; citation_volume=95; citation_number=5; citation_pages=1259-1276; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Two amyloid precursor protein transgenic mouse models with Alzheimer disease-like pathology.; citation_author=C Sturchler-Pierrat; citation_author=D Abramowski; citation_author=M Duke; citation_author=KH Wiederhold; citation_author=C Mistl; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=94; citation_number=6; citation_pages=13287-13292; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Two-dimensional electrophoresis of proteins: an updated protocol and implications for a functional analysis of the genome.; citation_author=J Klose; citation_author=U Kobalz; citation_journal_title=Electrophoresis; citation_volume=16; citation_number=7; citation_pages=1034-1059; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Large-gel 2-D electrophoresis. Fractionated extraction of total tissue proteins from mouse and human for 2-D electrophoresis.; citation_author=J Klose; citation_journal_title=Methods Mol Biol; citation_volume=112; citation_number=8; citation_pages=147-172; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Wild-type and mutant alpha-synuclein induce a multi-component gene expression profile consistent with shared pathophysiology in different transgenic mouse models of PD.; citation_author=RM Miller; citation_author=GL Kiser; citation_author=T Kaysser-Kranich; citation_author=C Casaceli; citation_author=E Colla; citation_journal_title=Exp Neurol; citation_volume=23; citation_number=9; citation_pages=23; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Microarray analysis of gene expression in Men1 knockout embryoid body reveals genetic events involved in early mouse embryonic development.; citation_author=HL Zhang; citation_author=TH Luo; citation_author=L Feng; citation_author=Y Zhao; citation_author=WY Li; citation_journal_title=Biochem Biophys Res Commun; citation_volume=352; citation_number=10; citation_pages=456-462; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Comparative proteomics in neurodegenerative and non-neurodegenerative diseases suggest nodal point proteins in regulatory networking.; citation_author=C Zabel; citation_author=D Sagi; citation_author=AM Kaindl; citation_author=N Steireif; citation_author=Y Klare; citation_journal_title=J Proteome Res; citation_volume=5; citation_number=11; citation_pages=1948-1958; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Functional amyloid formation within mammalian tissue.; citation_author=DM Fowler; citation_author=AV Koulov; citation_author=C Alory-Jost; citation_author=MS Marks; citation_author=WE Balch; citation_journal_title=PLoS Biol; citation_volume=4; citation_number=12; citation_pages=e6; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=The differentially expressed C21orf5 gene in the medial temporal-lobe system could play a role in mental retardation in Down syndrome and transgenic mice.; citation_author=C Lopes; citation_author=Z Chettouh; citation_author=JM Delabar; citation_author=M Rachidi; citation_journal_title=Biochem Biophys Res Commun; citation_volume=305; citation_number=13; citation_pages=915-924; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Transgenic mouse in vivo library of human Down syndrome critical region 1: association between DYRK1A overexpression, brain development abnormalities, and cell cycle protein alteration.; citation_author=I Branchi; citation_author=Z Bichler; citation_author=L Minghetti; citation_author=JM Delabar; citation_author=F Malchiodi-Albedi; citation_journal_title=J Neuropathol Exp Neurol; citation_volume=63; citation_number=14; citation_pages=429-440; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Modeling chromosomes in mouse to explore the function of genes, genomic disorders, and chromosomal organization.; citation_author=V Brault; citation_author=P Pereira; citation_author=A Duchon; citation_author=Y Herault; citation_journal_title=PLoS Genet; citation_volume=2; citation_number=15; citation_pages=e86; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Evolutionary conservation of motif constituents in the yeast protein interaction network.; citation_author=S Wuchty; citation_author=ZN Oltvai; citation_author=AL Barabasi; citation_journal_title=Nat Genet; citation_volume=35; citation_number=16; citation_pages=176-179; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Topological units of environmental signal processing in the transcriptional regulatory network of Escherichia coli.; citation_author=G Balazsi; citation_author=AL Barabasi; citation_author=ZN Oltvai; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=102; citation_number=17; citation_pages=7841-7846; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Total cell protein concentration as an evolutionary constraint on the metabolic control distribution in cells.; citation_author=GC Brown; citation_journal_title=J Theor Biol; citation_volume=153; citation_number=18; citation_pages=195-203; citation_date=1991; " />
      <meta name="citation_reference" content="citation_title=Implications of macromolecular crowding for protein assembly.; citation_author=AP Minton; citation_journal_title=Curr Opin Struct Biol; citation_volume=10; citation_number=19; citation_pages=34-39; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=The influence of macromolecular crowding and macromolecular confinement on biochemical reactions in physiological media.; citation_author=AP Minton; citation_journal_title=J Biol Chem; citation_volume=276; citation_number=20; citation_pages=10577-10580; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Membrane proteins and proteomics: un amour impossible?; citation_author=V Santoni; citation_author=M Molloy; citation_author=T Rabilloud; citation_journal_title=Electrophoresis; citation_volume=21; citation_number=21; citation_pages=1054-1070; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Parallel patterns of evolution in the genomes and transcriptomes of humans and chimpanzees.; citation_author=P Khaitovich; citation_author=I Hellmann; citation_author=W Enard; citation_author=K Nowick; citation_author=M Leinweber; citation_journal_title=Science; citation_volume=309; citation_number=22; citation_pages=1850-1854; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=A human protein-protein interaction network: a resource for annotating the proteome.; citation_author=U Stelzl; citation_author=U Worm; citation_author=M Lalowski; citation_author=C Haenig; citation_author=FH Brembeck; citation_journal_title=Cell; citation_volume=122; citation_number=23; citation_pages=957-968; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Complexity and demographic stability in population models.; citation_author=L Demetrius; citation_author=VM Gundlach; citation_author=G Ochs; citation_journal_title=Theor Popul Biol; citation_volume=65; citation_number=24; citation_pages=211-225; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=An entropic characterization of protein interaction networks and cellular robustness.; citation_author=T Manke; citation_author=L Demetrius; citation_author=M Vingron; citation_journal_title=J R Soc Interface; citation_volume=3; citation_number=25; citation_pages=843-850; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Relationship between insertion/deletion (indel) frequency of proteins and essentiality.; citation_author=SK Chan; citation_author=M Hsing; citation_author=F Hormozdiari; citation_author=A Cherkasov; citation_journal_title=BMC Bioinformatics; citation_volume=8; citation_number=26; citation_pages=227; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=High-betweenness proteins in the yeast protein interaction network.; citation_author=MP Joy; citation_author=A Brock; citation_author=DE Ingber; citation_author=S Huang; citation_journal_title=J Biomed Biotechnol; citation_volume=2005; citation_number=27; citation_pages=96-103; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Homeostasis and differentiation in random genetic control networks.; citation_author=S Kauffman; citation_journal_title=Nature; citation_volume=224; citation_number=28; citation_pages=177-178; citation_date=1969; " />
      <meta name="citation_reference" content="citation_title=Shape-dependent control of cell growth, differentiation, and apoptosis: switching between attractors in cell regulatory networks.; citation_author=S Huang; citation_author=DE Ingber; citation_journal_title=Exp Cell Res; citation_volume=261; citation_number=29; citation_pages=91-103; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=A non-genetic basis for cancer progression and metastasis: self-organizing attractors in cell regulatory networks.; citation_author=S Huang; citation_author=DE Ingber; citation_journal_title=Breast Dis; citation_volume=26; citation_number=30; citation_pages=27-54; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Self-renewal of teratocarcinoma and embryonic stem cells.; citation_author=I Chambers; citation_author=A Smith; citation_journal_title=Oncogene; citation_volume=23; citation_number=31; citation_pages=7150-7160; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Establishment of a gene-trap sequence tag library to generate mutant mice from embryonic stem cells.; citation_author=MV Wiles; citation_author=F Vauti; citation_author=J Otte; citation_author=EM Fuchtbauer; citation_author=P Ruiz; citation_journal_title=Nat Genet; citation_volume=24; citation_number=32; citation_pages=13-14; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Analysis of different promoter systems for efficient transgene expression in mouse embryonic stem cell lines.; citation_author=S Chung; citation_author=T Andersson; citation_author=KC Sonntag; citation_author=L Bjorklund; citation_author=O Isacson; citation_journal_title=Stem Cells; citation_volume=20; citation_number=33; citation_pages=139-145; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Mutagenic insertion and chromosome engineering resource (MICER).; citation_author=DJ Adams; citation_author=PJ Biggs; citation_author=T Cox; citation_author=R Davies; citation_author=L van der Weyden; citation_journal_title=Nat Genet; citation_volume=36; citation_number=34; citation_pages=867-871; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=A new mouse embryonic stem cell line with good germ line contribution and gene targeting frequency.; citation_author=TM Magin; citation_author=J McWhir; citation_author=DW Melton; citation_journal_title=Nucleic Acids Res; citation_volume=20; citation_number=35; citation_pages=3795-3796; citation_date=1992; " />
      <meta name="citation_reference" content="citation_title=BAC engineering for the generation of ES cell-targeting constructs and mouse transgenes.; citation_author=G Testa; citation_author=K Vintersten; citation_author=Y Zhang; citation_author=V Benes; citation_author=JP Muyrers; citation_journal_title=Methods Mol Biol; citation_volume=256; citation_number=36; citation_pages=123-139; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=The in vitro development of blastocyst-derived embryonic stem cell lines: formation of visceral yolk sac, blood islands and myocardium.; citation_author=TC Doetschman; citation_author=H Eistetter; citation_author=M Katz; citation_author=W Schmidt; citation_author=R Kemler; citation_journal_title=J Embryol Exp Morphol; citation_volume=87; citation_number=37; citation_pages=27-45; citation_date=1985; " />
      <meta name="citation_reference" content="citation_title=Improved silver staining procedure for fast staining in PhastSystem Development Unit. I. Staining of sodium dodecyl sulfate gels.; citation_author=J Heukeshoven; citation_author=R Dernick; citation_journal_title=Electrophoresis; citation_volume=9; citation_number=38; citation_pages=28-32; citation_date=1988; " />
      <meta name="citation_reference" content="citation_title=Estimation of the mtDNA mutation rate in aging mice by proteome analysis and mathematical modeling.; citation_author=L Mao; citation_author=C Zabel; citation_author=MA Wacker; citation_author=G Nebrich; citation_author=D Sagi; citation_journal_title=Exp Gerontol; citation_volume=41; citation_number=39; citation_pages=11-24; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=High MS-compatibility of silver nitrate-stained protein spots from 2-DE gels using ZipPlates and AnchorChips for successful protein identification.; citation_author=G Nebrich; citation_author=M Herrmann; citation_author=D Sagi; citation_author=J Klose; citation_author=P Giavalisco; citation_journal_title=Electrophoresis; citation_volume=19; citation_number=40; citation_pages=19; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Rapid identification of proteins by peptide-mass fingerprinting.; citation_author=DJC Pappin; citation_author=P Hojrup; citation_author=AJ Bleasby; citation_journal_title=Curr Biol; citation_volume=3; citation_number=41; citation_pages=327-332; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=Gene ontology: tool for the unification of biology. The Gene Ontology Consortium.; citation_author=M Ashburner; citation_author=CA Ball; citation_author=JA Blake; citation_author=D Botstein; citation_author=H Butler; citation_journal_title=Nat Genet; citation_volume=25; citation_number=42; citation_pages=25-29; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Cytoscape: a software environment for integrated models of biomolecular interaction networks.; citation_author=P Shannon; citation_author=A Markiel; citation_author=O Ozier; citation_author=NS Baliga; citation_author=JT Wang; citation_journal_title=Genome Res; citation_volume=13; citation_number=43; citation_pages=2498-2504; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Significance analysis of microarrays applied to the ionizing radiation response.; citation_author=VG Tusher; citation_author=R Tibshirani; citation_author=G Chu; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=98; citation_number=44; citation_pages=5116-5121; citation_date=2001; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001218" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Proteomic Shifts in Embryonic Stem Cells with Gene Dose Modifications Suggest the Presence of Balancer Proteins in Protein Regulatory Networks"/>
    <meta name="twitter:description" content="Large numbers of protein expression changes are usually observed in mouse models for neurodegenerative diseases, even when only a single gene was mutated in each case. To study the effect of gene dose alterations on the cellular proteome, we carried out a proteomic investigation on murine embryonic stem cells that either overexpressed individual genes or displayed aneuploidy over a genomic region encompassing 14 genes. The number of variant proteins detected per cell line ranged between 70 and 110, and did not correlate with the number of modified genes. In cell lines with single gene mutations, up and down-regulated proteins were always in balance in comparison to parental cell lines regarding number as well as concentration of differentially expressed proteins. In contrast, dose alteration of 14 genes resulted in an unequal number of up and down-regulated proteins, though the balance was kept at the level of protein concentration. We propose that the observed protein changes might partially be explained by a proteomic network response. Hence, we hypothesize the existence of a class of &ldquo;balancer&rdquo; proteins within the proteomic network, defined as proteins that buffer or cushion a system, and thus oppose multiple system disturbances. Through database queries and resilience analysis of the protein interaction network, we found that potential balancer proteins are of high cellular abundance, possess a low number of direct interaction partners, and show great allelic variation. Moreover, balancer proteins contribute more heavily to the network entropy, and thus are of high importance in terms of system resilience. We propose that the &ldquo;elasticity&rdquo; of the proteomic regulatory network mediated by balancer proteins may compensate for changes that occur under diseased conditions."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0001218.g005"/>

  <meta property="og:title" content="Proteomic Shifts in Embryonic Stem Cells with Gene Dose Modifications Suggest the Presence of Balancer Proteins in Protein Regulatory Networks" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001218" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=3883'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=4191'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=384&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0001218">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001218" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001218&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Proteomic Shifts in Embryonic Stem Cells with Gene Dose Modifications Suggest the Presence of Balancer Proteins in Protein Regulatory Networks
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Lei Mao
              <span class="corresponding">mail</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:lei.mao@charite.de">lei.mao@charite.de</a></p>

                <p>Affiliations:
                  Institute for Human Genetics, Charité-University Medicine Berlin, Germany, 
                  Max Planck Institute for Molecular Genetics, Berlin, Germany
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Claus Zabel, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Institute for Human Genetics, Charité-University Medicine Berlin, Germany
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Marion Herrmann, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Institute for Human Genetics, Charité-University Medicine Berlin, Germany
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Tobias Nolden, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Max Planck Institute for Molecular Genetics, Berlin, Germany
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Florian Mertes, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Max Planck Institute for Molecular Genetics, Berlin, Germany
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Laetitia Magnol, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Institut de Transgénose, IEM, UMR6218, CNRS Uni Orléans, Orléans, France
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Caroline Chabert, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  EA 3508, Université Paris Diderot-Paris 7, Paris, France
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Daniela Hartl, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Institute for Human Genetics, Charité-University Medicine Berlin, Germany
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Yann Herault, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Institut de Transgénose, IEM, UMR6218, CNRS Uni Orléans, Orléans, France
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Jean Maurice Delabar, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  EA 3508, Université Paris Diderot-Paris 7, Paris, France
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Thomas Manke, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Max Planck Institute for Molecular Genetics, Berlin, Germany
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Heinz Himmelbauer, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Max Planck Institute for Molecular Genetics, Berlin, Germany
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Joachim Klose
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Institute for Human Genetics, Charité-University Medicine Berlin, Germany
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: November 28, 2007</li>
    <li>DOI: 10.1371/journal.pone.0001218</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0001218" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0001218" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001218" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0001218" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0001218-g001" data-doi="info:doi/10.1371/journal.pone.0001218" data-uri="info:doi/10.1371/journal.pone.0001218.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001218.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001218-t001" data-doi="info:doi/10.1371/journal.pone.0001218" data-uri="info:doi/10.1371/journal.pone.0001218.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001218.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001218-t002" data-doi="info:doi/10.1371/journal.pone.0001218" data-uri="info:doi/10.1371/journal.pone.0001218.t002" title="Table 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001218.t002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001218-g002" data-doi="info:doi/10.1371/journal.pone.0001218" data-uri="info:doi/10.1371/journal.pone.0001218.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001218.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001218-g003" data-doi="info:doi/10.1371/journal.pone.0001218" data-uri="info:doi/10.1371/journal.pone.0001218.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001218.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001218-g004" data-doi="info:doi/10.1371/journal.pone.0001218" data-uri="info:doi/10.1371/journal.pone.0001218.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001218.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001218-t003" data-doi="info:doi/10.1371/journal.pone.0001218" data-uri="info:doi/10.1371/journal.pone.0001218.t003" title="Table 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001218.t003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001218-g005" data-doi="info:doi/10.1371/journal.pone.0001218" data-uri="info:doi/10.1371/journal.pone.0001218.g005" title="Figure 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001218.g005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001218-t004" data-doi="info:doi/10.1371/journal.pone.0001218" data-uri="info:doi/10.1371/journal.pone.0001218.t004" title="Table 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001218.t004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001218-t005" data-doi="info:doi/10.1371/journal.pone.0001218" data-uri="info:doi/10.1371/journal.pone.0001218.t005" title="Table 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001218.t005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001218">Reader Comments (1)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0001218" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1"></a><p>Large numbers of protein expression changes are usually observed in mouse models for neurodegenerative diseases, even when only a single gene was mutated in each case. To study the effect of gene dose alterations on the cellular proteome, we carried out a proteomic investigation on murine embryonic stem cells that either overexpressed individual genes or displayed aneuploidy over a genomic region encompassing 14 genes. The number of variant proteins detected per cell line ranged between 70 and 110, and did not correlate with the number of modified genes. In cell lines with single gene mutations, up and down-regulated proteins were always in balance in comparison to parental cell lines regarding number as well as concentration of differentially expressed proteins. In contrast, dose alteration of 14 genes resulted in an unequal number of up and down-regulated proteins, though the balance was kept at the level of protein concentration. We propose that the observed protein changes might partially be explained by a proteomic network response. Hence, we hypothesize the existence of a class of “balancer” proteins within the proteomic network, defined as proteins that buffer or cushion a system, and thus oppose multiple system disturbances. Through database queries and resilience analysis of the protein interaction network, we found that potential balancer proteins are of high cellular abundance, possess a low number of direct interaction partners, and show great allelic variation. Moreover, balancer proteins contribute more heavily to the network entropy, and thus are of high importance in terms of system resilience. We propose that the “elasticity” of the proteomic regulatory network mediated by balancer proteins may compensate for changes that occur under diseased conditions.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>Mao L, Zabel C, Herrmann M, Nolden T, Mertes F, et al.  (2007) Proteomic Shifts in Embryonic Stem Cells with Gene Dose Modifications Suggest the Presence of Balancer Proteins in Protein Regulatory Networks. PLoS ONE 2(11):
          e1218.
            doi:10.1371/journal.pone.0001218</p><p><strong>Academic Editor: </strong>Sui Huang, Children's Hospital Boston, United States of America</p><p><strong>Received:</strong> July 2, 2007; <strong>Accepted:</strong> November 1, 2007; <strong>Published:</strong> November 28, 2007</p><p><strong>Copyright:</strong> © 2007 Mao et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>This work was co-supported by the German National Genome Research Network (NGFN) program established by the German Ministry for Education and Research (BMBF), and by the EU grant 37627 “AnEUploidy”. C. Chabert and L. Magnol were supported by a Lejeune foundation grant. T. Manke was supported by a grant from the Volkswagen Foundation.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>Investigations of etiology and pathogenesis of human diseases are frequently performed using suitable animals as a model system. Most commonly mice are employed where a gene of particular interest is knocked out, mutated or overexpressed. When the effect caused by genome modification is subsequently studied in these mice at the molecular level, usually a large number of changes are observed on the mRNA and protein levels, in spite of the fact that only a single gene was altered. For example, in protein patterns obtained by two-dimensional gel electrophoresis (2-DE) of brain proteins from a mouse model for Parkinson's disease deficient of the parkin protein <a href="#pone.0001218-Goldberg1">[1]</a> and from a transgenic mouse model for Huntington's disease <a href="#pone.0001218-Mangiarini1">[2]</a>, we detected 15 and 40 variant proteins, respectively <a href="#pone.0001218-Palacino1">[3]</a>, <a href="#pone.0001218-Zabel1">[4]</a>. Using more sensitive protein detection methods, such as the differential in-gel electrophoresis (DIGE) technique and analyzing two different brain regions at two different age stages, 87 quantitatively variant proteins were detected in the parkin knock-out mouse <a href="#pone.0001218-Periquet1">[5]</a>. In investigations of a transgenic mouse model for Alzheimer's disease that overexpressed mutated human amyloid precursor protein (<em>App</em>) <a href="#pone.0001218-SturchlerPierrat1">[6]</a> using our large-gel 2-DE <a href="#pone.0001218-Klose1">[7]</a>, <a href="#pone.0001218-Klose2">[8]</a> and DIGE technique, we detected more than one hundred variant proteins (Hartl D. et. al., unpublished results). On the mRNA level, Miller and colleagues observed over 600 changes in a single gene modified Parkinson disease mouse model <a href="#pone.0001218-Miller1">[9]</a>. Similar results were also obtained in other single gene knock-out mouse models <a href="#pone.0001218-Zhang1">[10]</a>.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>Apparently, the molecular response to a single gene mutation is of considerable complexity, and certainly much more complex than detectable using current experimental approaches. We have previously compared the protein changes detected in mouse models for different neurodegenerative diseases and, in addition, mouse models of non-neurodegenerative disorders <a href="#pone.0001218-Zabel2">[11]</a>. We found that up to 36% of variant proteins were shared among these different disease models and hypothesized that these protein alterations were not disease-specific. Unexpectedly, when we compared wild-type mice of different inbred strains, we found that most of these putative disease-unspecific protein alterations also occurred as polymorphisms that distinguished strains of mice. This suggested that some, if not most of the protein changes observed when investigating disease models might not be genuinely informative regarding etiology or pathogenesis of the disease under consideration.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>To investigate the significance of protein changes under disease conditions, we have chosen a more systematic and simplified approach by using mouse embryonic stem (ES) cells with highly defined modifications in a controlled environment. Six mutant cell lines were investigated. All of them contained gene modifications relevant to neurodegenerative diseases. Four cell lines contained one single overexpressed gene, i.e. <em>App</em> (a cell surface receptor), <em>Snca</em> with changes relevant to Alzheimer's and Parkinson's disease, respectively <a href="#pone.0001218-Miller1">[9]</a>, <a href="#pone.0001218-Fowler1">[12]</a> and <em>Dyrk1a</em> (a nuclear kinase) as well as <em>Dopey2</em> (a leucine zipper-like protein) both relevant to Down syndrome <a href="#pone.0001218-Lopes1">[13]</a>, <a href="#pone.0001218-Branchi1">[14]</a>. In two other cell lines, a segment encompassing 14 genes relevant to Down syndrome was duplicated (trisomic) in one case and deleted (monosomic) in the other <a href="#pone.0001218-Brault1">[15]</a>. The six mutant cell lines were investigated by 2-DE and altered protein expression was recorded by comparison with the respective parental lines. Many variant proteins showing up or down-regulation were observed. Profound quantitative analysis of protein changes led us to the hypothesis that the cellular proteome is kept quantitatively in balance by a particular class of proteins to which we refer as “balancer proteins”. Accordingly, we assume that when the quantitative arrangement of the proteome is perturbed by gene dosage effects, it will be subjected to a rearrangement in order to achieve a new balance. Thus, the many protein changes observed may reflect the rearrangement of the proteome to protect the cell from deleterious effects of gene dosage mutations.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3><a id="article1.body1.sec2.p1" name="article1.body1.sec2.p1"></a><p>Proteins expressed in ES cells were separated by large-gel 2-DE. On a representative 2-DE pattern of total protein extract from ES cells, a total of 4958 protein spots could be scored visually (<a href="#pone-0001218-g001">Figure 1</a>). Using Delta2D imaging software (see <a href="#s4">Methods</a>), over 5500 protein spots were detected. Six different transgenic cell lines were investigated in this study. These comprised two cell lines in which one single gene was duplicated (<em>mES_Dyrk1a_Tris</em> or <em>mES_Dopey2_Tris</em>), and two cell lines in which one gene was overexpressed (5.5 times more than wild-type in <em>mES_hAPP</em> or 1.6 times in <em>mES_Snca</em>). In two cell lines gene dosage was altered over a chromosomal region that spanned 14 genes on mouse chromosome 17. A hemizygous deletion line was monosomic for the interval (<em>mES_14_Mono</em>). The other line contained an engineered duplication of the segment, and thus was trisomic (<em>mES_14_Tris</em>). No difference was observed between transgenic and parental cell lines with respect to cellular morphology and growth behavior. The six cell lines were compared to their parental cell lines with regard to their protein expression profiles. The number of proteins that showed significantly increased or decreased expression, when compared to their expression in parental cell lines, was in the range of 70 to 110 variants per cell line (<a href="#pone-0001218-t001">Table 1</a>). In total, 255 distinct variant proteins were observed in the six cell lines (<a href="#pone.0001218.s001">Table S1</a>). The data-adjusted modified t-test SAM (Significance Analysis of Microarrays) was used to calculate that the false discovery rate for obtaining a comparable result was less than 1 %.</p>
<div class="figure" id="pone-0001218-g001"><div class="img"><a name="pone-0001218-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001218.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001218" data-uri="info:doi/10.1371/journal.pone.0001218.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001218.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001218.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001218.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001218.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001218.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001218.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001218.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001218.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001218.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Representative protein expression pattern of mouse embryonic cell lines as revealed by large-gel 2D-electrophoresis.</span></strong></p><a id="article1.body1.sec2.fig1.caption1.p1" name="article1.body1.sec2.fig1.caption1.p1"></a><p>Over 5500 proteins (including protein isoforms) were resolved on a single gel. Highlighted spots correspond to spot ID of candidate balancer proteins detailed in <a href="#pone-0001218-t002">Table 2</a>.</p>
<span>doi:10.1371/journal.pone.0001218.g001</span></div><div class="figure" id="pone-0001218-t001"><div class="img"><a name="pone-0001218-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001218.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001218" data-uri="info:doi/10.1371/journal.pone.0001218.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001218.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001218.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001218.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001218.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001218.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001218.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001218.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001218.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001218.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>Number of quantitatively variant proteins in six transgenic mouse embryonic stem cell lines.</span></strong></p><span>doi:10.1371/journal.pone.0001218.t001</span></div><div class="figure" id="pone-0001218-t002"><div class="img"><a name="pone-0001218-t002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001218.t002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001218" data-uri="info:doi/10.1371/journal.pone.0001218.t002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001218.t002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001218.t002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001218.t002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001218.t002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001218.t002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001218.t002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001218.t002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001218.t002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001218.t002.TIF"></span>)
                </a></li></ul></div><p><strong>Table 2.  <span>Proteins changed in more than three transgenic ES cell lines (proposed balancer proteins).</span></strong></p><span>doi:10.1371/journal.pone.0001218.t002</span></div><a id="article1.body1.sec2.p2" name="article1.body1.sec2.p2"></a><p>In the four cell lines that overexpressed a single gene, 40 to 50 proteins were up-regulated. This was always accompanied by a similar number of down-regulated proteins. A quite different situation was found for the two cell lines with the dosage alteration in 14 genes: If duplicated, 60% of proteins were up-regulated and 40% were down-regulated (40%). In case of deletion, a similar imbalance was found, but in the opposite direction, i.e. about 60% of the variant proteins showed decreased expression, while only about 40% were over-expressed (<a href="#pone-0001218-g002">Figure 2A</a>). The observations described above were based on the number of proteins showing altered expression profiles in the transgenic cell lines. In the next step, we investigated the total protein amount showing altered expression within each cell line by determining relative protein concentrations (protein spot volumes) across all altered proteins. This resulted in a balanced picture, i.e. no significant difference could be detected in the protein amount undergoing up and down-regulation (<a href="#pone-0001218-g002">Figure 2B</a>). Most importantly, this was even true for the two cell lines with 14 genes altered, which showed a drastic imbalance in the number of proteins that underwent up or down-regulation (see above).</p>
<div class="figure" id="pone-0001218-g002"><div class="img"><a name="pone-0001218-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001218.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001218" data-uri="info:doi/10.1371/journal.pone.0001218.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001218.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001218.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001218.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001218.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001218.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001218.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001218.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001218.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001218.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Proteins that showed altered expression in transgenic ES cell lines.</span></strong></p><a id="article1.body1.sec2.fig2.caption1.p1" name="article1.body1.sec2.fig2.caption1.p1"></a><p>(A) Number of altered proteins in each transgenic cell line, expressed as percentile of total number of altered proteins. (B) Amount of proteins that underwent altered expression in each cell line, represented as percent of total spot volume that was up or down-regulated in transgenic cell lines. Dose alteration of 14 genes could no longer be balanced by an equivalent number of variant proteins. However, a balance remained at the level of protein concentration.</p>
<span>doi:10.1371/journal.pone.0001218.g002</span></div><a id="article1.body1.sec2.p3" name="article1.body1.sec2.p3"></a><p>When we compared proteins that showed quantitative changes among the six cell lines, we found that many of these proteins were altered in several cell lines. Specifically, 38 proteins showed changes in more than three cell lines. Among them, the expression of three proteins changed in all six cell lines, eight proteins changed in five, while 27 proteins changed in four of six different transgenic ES cell lines. In contrast, 114 proteins were altered only in one cell line. In order to test to which extent changes of expression in the same proteins may occur by chance in multiple cell lines independently, the numbers of observed co-changed proteins in different numbers of cell lines were compared to theoretical numbers of co-changed proteins, assuming that a total of 800 protein spots were investigated, among which 10% were differentially expressed in transgenic and control cell lines (<a href="#pone-0001218-g003">Figure 3</a>). Our calculation showed that the occurrence of the same protein alteration in more than three cell lines was unlikely to be coincidental (p&lt;0.001).</p>
<div class="figure" id="pone-0001218-g003"><div class="img"><a name="pone-0001218-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001218.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001218" data-uri="info:doi/10.1371/journal.pone.0001218.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001218.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001218.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001218.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001218.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001218.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001218.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001218.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001218.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001218.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Comparison of observed number of co-changed proteins against a theoretical calculation of co-changed proteins across six different transgenic cell lines.</span></strong></p><a id="article1.body1.sec2.fig3.caption1.p1" name="article1.body1.sec2.fig3.caption1.p1"></a><p>It was assumed that a total of 800 protein spots were investigated, among which 10% of the proteins change in their expression profile. This comparison shows that the occurrence of the same protein alteration in more than three cell lines is unlikely to be coincidental.</p>
<span>doi:10.1371/journal.pone.0001218.g003</span></div><a id="article1.body1.sec2.p4" name="article1.body1.sec2.p4"></a><p>An interesting observation was made when we considered proteins that were only altered in both <em>mES_14_Mono</em> and <em>mES_14_Tris</em>: Two thirds of them showed the same change tendency, i.e., either up-regulated in both cell lines, or down-regulated in both cell lines, despite opposite gene dose alteration (trisomy versus monosomy). This suggested that many changes could be unrelated with respect to the gene(s) that caused the dosage imbalance. Hence, we hypothesize that the proteins showing changes in several cell lines (38 proteins, see above) represent a particular class of proteins, which we propose to call “balancer” proteins (<a href="#pone-0001218-t002">Table 2</a>). Different from that, proteins that were altered only in a single cell line are called here “cell line-specific proteins” to denote protein alterations specific to a cell line characterized by a distinct genetic alteration (114 proteins, see above).</p>
<a id="article1.body1.sec2.p5" name="article1.body1.sec2.p5"></a><p>Among the candidate balancer proteins, seven of them were always increased in their expression in our experiment (<a href="#pone-0001218-t002">Table 2</a>). They are: <em>Atp6v1c1</em>, <em>Ccdc25</em>, <em>Eno1</em>, <em>Nudt16l1</em>, <em>Psmb7</em>, <em>Ranbp5</em> and <em>S100a11</em>. On the other hand, three balancer proteins (<em>Bat2d</em>, <em>Psmb6</em> and <em>Tceb2</em>) were consistently down-regulated in their expression. One protein (<em>Psme1</em>) was down-regulated in three cell lines with transgene overexpression (<em>mES_14_Tris</em>, <em>mES_Dyrk1a_Tris</em> and <em>mES_Snca</em>), while it was up-regulated in <em>mES_14_Mono</em>. To determine whether putative balancer or cell line-specific proteins might be direct interaction partners of genes mutated in the six cell lines, we queried all mutated genes, balancer proteins and cell line-specific proteins in the KEGG pathway database. No overlapping KEGG pathway entries were detected between balancers and mutated genes. On the contrary, four KEGG pathway terms of cell line-specific protein overlapped with that of mutated genes in our six ES cell lines. These included methionine metabolism, selenoamino acid metabolism, ABC transporters and purine metabolism. Similar conclusions could be drawn from Biocarta pathway database queries.</p>
<a id="article1.body1.sec2.p6" name="article1.body1.sec2.p6"></a><p>In order to investigate whether balancers and cell line-specific proteins might represent two different classes of proteins with certain biochemical and biophysical properties, we compared these two sets according to different parameters. The spectrum of biochemical and biophysical criteria selected for characterization included molecular weight, isoelectric point, predicted protein instability, aliphatic index, hydrophobicity, cellular abundance, polymorphisms (i.e. allelic diversity) and number of direct protein interaction partners. As summarized in <a href="#pone-0001218-t003">Table 3</a>, balancers and cell line-specific proteins showed no perceivable difference in their molecular weights and isoelectric points, neither in their instability, nor regarding aliphatic index or hydrophobicity. However, balancer proteins were significantly more abundant in the cell (p = 0.008). Furthermore, cell line-specific proteins were found to have twice as many interaction partners as balancer proteins (p = 0.004) (<a href="#pone-0001218-g004">Figure 4B</a>). We then queried the Mouse Genome Informatics Database (<a href="http://www.informatics.jax.org">www.informatics.jax.org</a>) for the occurrence of single nucleotide polymorphisms (SNPs) in balancer and cell line-specific proteins as a measure of their allelic diversity. Interestingly, the potential balancers had significantly more non-synonymous SNPs in coding regions than potential cell line-specific proteins (<a href="#pone-0001218-t003">Table 3</a>), while no significant difference could be established for other SNP evaluations (total number of SNPs, proportion of synonymous SNPs in the coding regions and the frequency of SNPs in the 5′-UTR, 3′-UTR, introns and sequences flanking upstream and downstream of a locus).</p>
<div class="figure" id="pone-0001218-g004"><div class="img"><a name="pone-0001218-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001218.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001218" data-uri="info:doi/10.1371/journal.pone.0001218.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001218.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001218.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001218.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001218.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001218.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001218.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001218.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001218.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001218.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>Cumulative fraction plots of “balancer”-“cell line-specific protein” comparison.</span></strong></p><a id="article1.body1.sec2.fig4.caption1.p1" name="article1.body1.sec2.fig4.caption1.p1"></a><p>(A) Entropic contribution.; (B) Number of direct protein interaction partners. Compared to cell line-specific proteins, balancers possess significantly higher values of entropic contribution and a low number of direct interaction partners.</p>
<span>doi:10.1371/journal.pone.0001218.g004</span></div><div class="figure" id="pone-0001218-t003"><div class="img"><a name="pone-0001218-t003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001218.t003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001218" data-uri="info:doi/10.1371/journal.pone.0001218.t003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001218.t003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001218.t003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001218.t003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001218.t003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001218.t003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001218.t003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001218.t003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001218.t003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001218.t003.TIF"></span>)
                </a></li></ul></div><p><strong>Table 3.  <span>Comparison of protein properties of balancer and cell line-specific proteins.</span></strong></p><span>doi:10.1371/journal.pone.0001218.t003</span></div><a id="article1.body1.sec2.p7" name="article1.body1.sec2.p7"></a><p>To assign functional categories, a Gene Ontology (GO) term enrichment analysis was performed. <a href="#pone-0001218-t004">Tables 4</a> and <a href="#pone-0001218-t005">5</a> give a summary of GO-terms that occurred more frequently in balancers or cell line-specific proteins, respectively, based on human GOA database searches (see <a href="#s4">Methods</a> for details). Eight GO-terms were specifically over-represented in balancer candidates. They comprise protein degradation, disulfide modification and electron carrier processes. In contrast, over 33 GO-terms were enriched in cell line-specific proteins. Notably, a large part of them were involved in mRNA processing and related functions. These two GO-term sets overlap by participating in protein chaperoning of catabolism processes.</p>
<a id="article1.body1.sec2.p8" name="article1.body1.sec2.p8"></a><p>Next, we undertook an analysis of protein-protein interactions that balancer and cell line-specific proteins participate in, chiefly based on the Human Reference Protein Database (see <a href="#s4">methods</a> for details). The protein-protein interaction graph constructed from our ES cell data comprised 2677 nodes (distinct proteins, indicated by gene symbol). This interaction graph shared the common feature of scale-free geometry with other protein interaction networks, such as that of <em>E. coli</em> or <em>Saccharomyces cerevisiae</em> <a href="#pone.0001218-Wuchty1">[16]</a>, <a href="#pone.0001218-Balazsi1">[17]</a>. Among all protein nodes, 2565 (96%) of them could be linked to a giant network component with heterogeneous degree distribution. The remaining 112 proteins formed 41 isolated components, with the number of nodes varying from one to twelve. <a href="#pone-0001218-g005">Figure 5</a> shows a subset of the protein-protein interaction network centered around the proteasome subunits. In the entropy analysis of the network, we focused on the giant network component, since the network entropy is only defined for the strongly connected components of the network. All 38 balancer proteins belonged to the giant network component, as well as 79 out of 114 cell line-specific proteins.</p>
<div class="figure" id="pone-0001218-g005"><div class="img"><a name="pone-0001218-g005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001218.g005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001218" data-uri="info:doi/10.1371/journal.pone.0001218.g005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001218.g005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001218.g005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001218.g005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001218.g005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001218.g005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001218.g005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001218.g005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001218.g005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001218.g005.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 5.  <span>A protein-protein interaction subgraph showing the proteasome subunits, where nodes denote proteins and the edges describe protein-protein interaction.</span></strong></p><a id="article1.body1.sec2.fig5.caption1.p1" name="article1.body1.sec2.fig5.caption1.p1"></a><p>Two local hub proteins of this subgraph (Psma2 and Psma3) belong to cell line-specific proteins, while a candidate balancer protein (Psmb6) represents a connection between these two modules (see <a href="#s3">discussion</a> for details). This supports our assumption that balancer proteins could be connective hubs between different modules. Protein marked in green: Psma2; yellow: Psmb6; magenta: Psma3.</p>
<span>doi:10.1371/journal.pone.0001218.g005</span></div><a id="article1.body1.sec2.p9" name="article1.body1.sec2.p9"></a><p>As network entropy is a measure of system homeostasis, we may expect high-ranking proteins to be affected more frequently as the cell responds to various stimuli. Through a direct comparison of balancers to cell line-specific proteins using their entropic contribution, we found that balancers, on average, possess significantly higher values of entropic contribution than cell line-specific proteins (p = 0.02, Wilcoxon rank test, <a href="#pone-0001218-g004">Figure 4</a>). Alternatively, we ask to what extent the entropic measurement can distinguish between cell line-specific proteins and balancers within the background of all proteins in the giant component. To this end, we took the same number of top-ranking proteins based on their entropic contribution and studied their overlap with our 38 balancers or 79 cell line-specific proteins, respectively. Assuming a hypergeometric distribution over a total of 2526 proteins, this corresponds to p = 0.018 and p = 0.094 for balancers and cell line-specific proteins, respectively. This illustrates that the entropic ranking of proteins selects balancers preferentially, thus it validates our previous observation that proteins with high contribution to network entropy are enriched in the set of balancer proteins.</p>
</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p>We investigated the effect of gene dosage alterations on the proteome of mouse embryonic stem (ES) cells. Using our large-gel 2-DE, extraordinary in its high resolution and reproducibility <a href="#pone.0001218-Klose1">[7]</a>, total protein extracts from six different ES cell lines were analyzed. In four of them, one single gene was overexpressed either by gene duplication (<em>Dyrk1a</em>, <em>Dopey2</em>) or by conventional stable gene transfection (<em>App</em>, <em>Snca</em>). In two other cell lines, the dosage of a whole set of 14 genes was altered so that the segment was either duplicated (trisomic) or deleted (monosomic). According to our observations, dose alteration of a single gene led to quantitative changes in a large number of proteins. Surprisingly however, altering the dosage of 14 genes instead of one gene did not increase the number of altered proteins accordingly. In effect, the frequency of protein variations induced by one or 14 altered gene dosages was in a similar range. Hence, we propose that the protein changes observed might not completely reflect reactions of proteins functionally linked with the genes whose dosage was altered. Rather, these changes may at least be partially explained as a global response of the cellular proteome to the gene dosage defect.</p>
<a id="article1.body1.sec3.p2" name="article1.body1.sec3.p2"></a><p>Considering the protein changes observed in our ES cell lines in more detail, we found that in all cases where a single gene was overexpressed, the number of proteins which were up-regulated was always in equilibrium with the number of down-regulated proteins (<a href="#pone-0001218-g002">Figure 2A</a>). Moreover, when we measured up and down-regulation of proteins in terms of protein amount instead of number of proteins, a balance in up and down-regulation was also observed. The situation was different in the two cell lines carrying alterations in 14 genes. Here, the number of proteins up or down-regulated was no longer in equilibrium: In <em>mES_14_Tris</em>, about 60% of the altered proteins were up-regulated, whereas about 40% of the proteins were down-regulated. The changes in the <em>mES_14_Mono</em> showed the same ratio, but in reversed direction (ca. 60% down, 40% up). However, regarding the protein variations at the level of protein amount, a balance reoccurred even in cell lines with 14 genes altered (<a href="#pone-0001218-g002">Figure 2B</a>).</p>
<a id="article1.body1.sec3.p3" name="article1.body1.sec3.p3"></a><p>We therefore hypothesize the existence of a proteome-wide acting regulatory mechanism that leads to a compensation of an imbalance in the quantitative arrangement of the cellular proteome. Within the proteome of a cell, the relative concentration of each particular protein should be precisely arranged and well balanced. In consequence, aberrant quantitative changes, even in a single protein, may alter the relative concentration of many other proteins, thereby disturbing the overall proteomic balance. In this situation, the first response of the cell could be towards restoring the balance in the cellular proteome in order to maintain normal cellular operations. As a result, below certain thresholds, a rebuild of system homeostasis by quantitative rearrangement of the proteome may be achieved.</p>
<a id="article1.body1.sec3.p4" name="article1.body1.sec3.p4"></a><p>Several considerations that originate from theoretical biology and experimental model systems are in line with our hypothesis outlined above: The theory of protein minimization <a href="#pone.0001218-Brown1">[18]</a> states that all protein levels within a cell are maintained at the minimum level compatible with function, while metabolic pathway fluxes are maintained at the maximum. This is explained as a consequence of an increasing number of proteins occurring in the course of evolution, e.g. by gene duplication, that needed to be accommodated in the cells. Since the resources of a cell (such as space, energy, metabolites e.g. amino acids and unbound water to allow diffusion) did not increase accordingly, the occurrence of new proteins in evolution was always accompanied by a concentration reduction of proteins that already present. In order to keep cellular functions intact in spite of protein concentration reduction, the functional efficiency of the already established proteins (e.g. the specific activity of enzymes) had to increase. Another theory, the excluded volume theory established by A.P. Minton <a href="#pone.0001218-Minton1">[19]</a>, <a href="#pone.0001218-Minton2">[20]</a> deals with the high degree of macromolecular crowding in cells. If a protein is overexpressed in a cell, movement of this and surrounding proteins becomes restricted due to excluded volume. Thus the distance between protein molecules becomes smaller than the diameter of moving protein molecules. Proteins react to this situation with conformational changes and tend to aggregate and to lose their function. Apparently, active or passive regulatory mechanisms exist that keep the cellular protein concentrations within a physiologically buffered range.</p>
<a id="article1.body1.sec3.p5" name="article1.body1.sec3.p5"></a><p>When the relative cellular proteome composition is disturbed, probably not all proteins are changed in their quantitative occurrence to the same extent. Regarding our hypothesis, we suggest the existence of a special class of proteins that are particularly effective in such rebalancing approaches. This led us to propose discrimination between balancer and cell line-specific proteins. We hypothesize that balancers are proteins that buffer or cushion a cellular system by common properties, i.e., properties not necessarily related to their specific functions. Accordingly, the same proteins may change when different system disturbances have caused protein imbalance. In line with these definitions, we found no considerable overlapping functions between balancer proteins and the transgenes. In contrast, the expression alterations of cell line-specific proteins could more likely have been directly induced by gene dosage modifications. This finding further supports the notion that the changes of balancer proteins represent more peripheral cellular affairs.</p>
<a id="article1.body1.sec3.p6" name="article1.body1.sec3.p6"></a><p>To find out whether balancer proteins might have further distinct properties, we analyzed them using multiple categories outlined in <a href="#pone-0001218-t003">tables 3</a>, <a href="#pone-0001218-t004">4</a> and <a href="#pone-0001218-t005">5</a>. We found that potential balancers seem to be of high cellular abundance. This is plausible as very low abundance proteins (e.g. regulatory proteins, transcription factors and receptors) are possibly present only in a few copies per cell and thus have no real buffer capacity to compensate imbalance at the proteomic scale. In retrospect, it is known that all proteins visible on 2-DE patterns are relatively abundant <a href="#pone.0001218-Santoni1">[21]</a>. Still, even under these preconditions, balancer proteins seem to be more abundant than cell line-specific proteins. Moreover, potential balancers turned out to be more polymorphic in their coding regions than cell line-specific proteins. Protein polymorphisms indicate proteins which became less constrained in the course of evolution <a href="#pone.0001218-Khaitovich1">[22]</a>. As a consequence, proteins bearing a higher degree of polymorphisms (including balancers) may tend to be more flexible in quantitative changes, whereas cell line-specific proteins may require a stronger connection between expression level and function.</p>
<a id="article1.body1.sec3.p7" name="article1.body1.sec3.p7"></a><p>Another trait of our candidates for balancer proteins was found by screening a protein-protein interaction database available online (HPRD). Here, balancers possessed less direct interaction partners than cell line-specific proteins. Interestingly, in the protein interaction network published by Stelzl et al. for human proteins <a href="#pone.0001218-Stelzl1">[23]</a>, disease-related proteins annotated in the <em>Online Mendelian Inheritance in Man</em> database (OMIM) were located almost exclusively in the area of low connective level. This correlation may indicate a particular role of balancer proteins in diseases conditions, but at the same time raises our suspicion that balancer proteins are more likely to be identified as disease-associated proteins partially due to their frequent and reproducible alterations.</p>
<a id="article1.body1.sec3.p8" name="article1.body1.sec3.p8"></a><p>Towards understanding how balancer proteins in their functional properties may impart elasticity to the proteomic system, we queried what kind of shared functional categories these proteins may possess (biological process and molecular function GO terms). Compared to the candidate balancers, cell line-specific proteins were associated with a much broader spectrum of GO-categories (<a href="#pone-0001218-t004">Table 4</a> and <a href="#pone-0001218-t005">5</a>). In addition, cell line-specific proteins but not balancers were highly involved in mRNA-related processes. This is in line with the fact that these processes are tightly regulated. Proteins involved therein are thus prone to concentration alteration, a property incompatible with a role as balancers. Moreover, our set of putative balancer proteins was enriched in stress and metabolic proteins compared to the remaining proteins altered. The physiological activity of a significant subset of cellular proteins is modified by the redox state of regulatory thiol groups. The cellular redox homeostasis depends on the balance between oxidation of thiols through oxygen and reactive oxygen species and reduction by thiol-disulfide transfer reactions. In this respect, it would make sense that potential balancer proteins are enriched in GO categories implicated in disulfide oxidoreductase and thiol disulfide exchange.</p>
<div class="figure" id="pone-0001218-t004"><div class="img"><a name="pone-0001218-t004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001218.t004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001218" data-uri="info:doi/10.1371/journal.pone.0001218.t004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001218.t004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001218.t004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001218.t004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001218.t004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001218.t004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001218.t004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001218.t004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001218.t004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001218.t004.TIF"></span>)
                </a></li></ul></div><p><strong>Table 4.  <span>GO-terms enriched among balancer proteins detected in the proteome of mouse ES cells.</span></strong></p><span>doi:10.1371/journal.pone.0001218.t004</span></div><div class="figure" id="pone-0001218-t005"><div class="img"><a name="pone-0001218-t005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001218.t005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001218" data-uri="info:doi/10.1371/journal.pone.0001218.t005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001218.t005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001218.t005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001218.t005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001218.t005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001218.t005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001218.t005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001218.t005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001218.t005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001218.t005.TIF"></span>)
                </a></li></ul></div><p><strong>Table 5.  <span>Enriched GO-terms among cell line-specific proteins detected in the proteome of mouse embryonic stem cells.</span></strong></p><span>doi:10.1371/journal.pone.0001218.t005</span></div><a id="article1.body1.sec3.p9" name="article1.body1.sec3.p9"></a><p>One particularly important feature of a living system is its resilience against external and internal changes, which, at the molecular level, amounts to perturbations in network parameters. In an attempt to analyze this robustness of the cellular system, we applied a network analysis, which is motivated by concepts from statistical mechanics and dynamical systems theory. Our approach is based on the assumption that biological processes often operate at steady state, which corresponds to the observed phenotype <a href="#pone.0001218-Demetrius1">[24]</a>. It has been shown that changes in network entropy, a fundamental statistical property, are positively correlated with system robustness. In turn, the entropic contribution of a protein describes its impact on network integrity. Removal of nodes with high entropic contribution more often result in lethal phenotypes from yeast and <em>C. elegans</em> <a href="#pone.0001218-Manke1">[25]</a>. Our ranking analysis shows that there is a difference between balancers and cell line-specific proteins: Compared to cell line-specific proteins, balancers possessed a higher entropic contribution. This structural property suggests that balancers might be able to attenuate system disturbance more efficiently. The existence of balancer proteins could therefore be responsible for the elasticity of a cellular system.</p>
<a id="article1.body1.sec3.p10" name="article1.body1.sec3.p10"></a><p>For example, a number of proteins representing proteasome subunits showed altered expression in our transgenic cell lines. Five of them belong to balancer candidates, while three other proteasome subunits belong to cell line-specific proteins. Considering the proteasome sub-interaction network in detail (<a href="#pone-0001218-g005">Figure 5</a>), we noticed that <em>Psma2</em> and <em>Psma3</em>, which are local hubs in the subgraph, both belong to candidate cell line-specific proteins. On the other hand, <em>Psmb6</em> is a candidate balancer protein connecting between two different nodes of a higher order. This example supports our assumption that balancer proteins could be connective hubs between different modules. Such “bridges” are probably heavily utilized during balancing processes. It is worth noting that the concept of “bridges” discussed here resembles that of “high betweenness” of previous studies on protein interaction networks using graph theory <a href="#pone.0001218-Chan1">[26]</a>, <a href="#pone.0001218-Joy1">[27]</a>. If two clusters of interacting proteins are joined together only through a mutual interacting protein, this protein would have a “high betweeness” measure. “High betweeness” thus indicates the importance of a node within the wider context of the holistic network <a href="#pone.0001218-Joy1">[27]</a>. Here, the entropic contribution captures this property not in terms of shortest pats (as betweenness), but in terms of random walks inside the network. In this sense, network entropy and entropic contribution provide a conceptual framework to understand the role of the heuristic centrality indices, such as node degree and betweenness.</p>
<a id="article1.body1.sec3.p11" name="article1.body1.sec3.p11"></a><p>We are aware that our conclusion provides only one qualitative interpretation of the experimental observations. Under the assumption that gene dosage modifications in the ES cell lines represent small perturbations to the cellular system, more detailed theoretical interpretations can also be sought. For example, previous studies have described that cellular fates such as differentiation, growth, quiescence, or apoptosis may represent the convergence of stochastic cellular program onto a small set of common self-stabilizing “attractors” states <a href="#pone.0001218-Kauffman1">[28]</a>–<a href="#pone.0001218-Huang2">[30]</a>. These attractor states, which are robust to small perturbations, may also explain our observation that the transgenic ES cells remained in their original steady state as undifferentiated ES cells. However, we are cautious with respect to such a general conclusion, considering that our sample set is very limited, both in terms of sample dimension and its representative nature. Importantly, most of the current network data is of purely structural character, and does not allow for a more detailed understanding of the underlying dynamics, or even its logical abstraction. Moreover, the protein property information was obtained from current protein database entries that are incomplete and may be biased towards intensively studied proteins. Furthermore, due to our small sample sizes, the p-value estimations are not very robust, and may affect our assignment of significance for observed differences. Possible future experiments to test our hypothesis could be, for example, to analyze transgenic cell lines overexpressing one of the candidate balancer proteins in the same <em>in vitro</em> system.</p>
<a id="article1.body1.sec3.p12" name="article1.body1.sec3.p12"></a><p>In summary, based on our results we hypothesize that the large number of variant proteins detected in mutant ES cells does not necessarily reflect disease-related dysfunctions of these proteins, but rather a quantitative rearrangement of the proteome in response to a disturbance induced by gene dosage alterations. We postulated a regulatory mechanism established in a cell that protects it from deleterious effects of mutations by keeping the macromolecular composition of a cell quantitatively in balance.</p>
</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4>Transgenic ES cell line construction</h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p>Pluripotent mouse ES cells were genetically manipulated on single or a set of genes involved in neurodegenerative diseases. A plasmid-mediated gene insertion protocol was used to generate <em>App</em> and <em>Snca</em>-overexpressing cell lines (<em>mES_hAPP</em> and <em>mES_Snca</em>, respectively), with CGR8 as parental line <a href="#pone.0001218-Chambers1">[31]</a>, <a href="#pone.0001218-Wiles1">[32]</a>. For this purpose, a transfer vector based on pMSCV (BD BioSciences Clontech Heidelberg, Germany), which contained a puromycin resistance gene, was modified by inserting a 1.3 kb fragment of the rat promoter for translation elongation factor 1 alpha 1 (<em>Eef1a1</em>). This promoter has been shown to be suitable for protein overexpression in ES cells <a href="#pone.0001218-Chung1">[33]</a>. cDNA of a target gene (human <em>App</em> or mouse <em>Snca</em>) was inserted in frame with the initiating methionine specified by the rat <em>Eef1a1</em> promoter. The vector was electroporated into the ES cells at standard conditions (250V, 500 µF). 24h after electroporation, seven days of puromycin selection followed to select stably transformed ES cell lines. Western blotting was carried out to confirm protein overexpression (monoclonal mouse IgG against human amyloid β peptide, amino acids 1–17, clone 6E10; monoclonal mouse IgG against α-synuclein, clone 5D6, Signet Laboratories, Berkeley, USA). The MICER strategy was used to generate ES cell lines bearing segmental deletion or duplication of <em>Abcg1</em>-<em>U2af1</em> on mouse chromosome 17 (30333543 to 31387432 bp), using ES cell line HM-1 as parental line <a href="#pone.0001218-Lopes1">[13]</a>, <a href="#pone.0001218-Brault1">[15]</a>, <a href="#pone.0001218-Adams1">[34]</a>, <a href="#pone.0001218-Magin1">[35]</a> (<em>mES_14_Mono</em> and <em>mES_14_Tris</em>, respectively). This chromosome segment contains the following genes: <em>Abcg1</em>, <em>Tff3</em>, <em>Tff2</em>, <em>Tff1</em>, <em>Tmprss3</em>, <em>Ubash3a</em>, <em>Tsga2</em>, <em>LOC667056</em>, <em>Slc37a1</em>, <em>Pde9a</em>, <em>Wdr4</em>, <em>1500032D14Rik</em>, <em>Pknox1</em>, <em>Cbs</em> and <em>LOC623242</em>. The bordering gene <em>Abcg1</em> was deleted in the monosomy, but unaffected in the trisomy cell line. ES cell lines trisomic for murine <em>Dyrk1a</em> (<em>mES_Dyrk1a_Tris</em>) or murine <em>Dopey2</em> (<em>mES_Dopey2_Tris</em>) were generated using a BAC gene transfer protocol <a href="#pone.0001218-Testa1">[36]</a>, with D3 as parental cell line (for <em>Dyrk1a:</em> BAC 189N10 from the CT7 library, pBeloBac11 vector, 94672437 to 94823558 bp on MMU16; for <em>Dopey2:</em> PAC 186P4 from the RP21 library, pPAC4 vector, 93576842 to 93751423 bp on MMU16) <a href="#pone.0001218-Doetschman1">[37]</a>. All ES cell lines were able to give germ-line transmission <a href="#pone.0001218-Branchi1">[14]</a>, <a href="#pone.0001218-Brault1">[15]</a>, except for the CGR8 subclone used, which is primarily intended for work <em>in vitro</em> (Savatier, personal communication).</p>


<h4>Maintenance of ES cells</h4>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1"></a><p>ES cell lines were grown in Dulbecco's Modified Eagle Medium (DMEM; Invitrogen, Karlsruhe, Germany) supplemented with 15% fetal calf serum (Biochrom, Berlin Germany), 2mM L-glutamine (Invitrogen), 0.1mM non-essential amino acids (Invitrogen), 1mM sodium pyruvate (Invitrogen), 0.1 mM 2-mercaptoethanol (Invitrogen) and 100U/ml leukemia inhibitory factor (LIF, Chemicon, Hampshire UK) under standard cell culture conditions (37°C, 5% CO2, 95% humidity). Modified and control cell lines were always cultured in parallel. CGR8-derived ES cell lines were maintained on gelatine-coated (0.1% v/v) cell culture plates. ES cells with E3 or HM-1 as parental line were maintained on mitotically inactivated murine embryonic fibroblasts. Prior to cell harvest, these cells were grown for three further passages on gelatine-coated plates to eliminate feeder cells. Cells were grown to 70–90% confluence and met morphological criteria for undifferentiated ES cells at the time of harvest (tightly packed cells forming round colonies). ES cells of three independent 10cm culture dishes were gently dissociated in ice cold PBS containing 5mM EDTA. This resulted in three biological replicates for each cell line. Trypsinization was avoided to preclude protein alteration artifacts.</p>


<h4>2D-Electrophoresis</h4>
<a id="article1.body1.sec4.sec3.p1" name="article1.body1.sec4.sec3.p1"></a><p>ES cell total protein extraction was carried out using our standard protocol <a href="#pone.0001218-Klose2">[8]</a>. 70 µg of protein was separated in each 2-DE-run as described previously <a href="#pone.0001218-Klose1">[7]</a>. Transgenic and their parental cell lines were always run in parallel. Two technical repeats were conducted for each cellular protein extract. Silver staining protocol was employed to visualize protein spots <a href="#pone.0001218-Heukeshoven1">[38]</a>. Computer-assisted manual gel evaluation was performed after scanning of the gel images (600 dpi, UMAX, Willich Germany) (Delta2D version 3.4, Decodon, Greifswald Germany) <a href="#pone.0001218-Mao1">[39]</a>. Briefly, corresponding gel images were first warped using “exact mode” (manual vector setting combined with automatic warping). A fusion gel image was subsequently generated using union mode, which is a weighted arithmetic mean across the entire gel series. Spot detection was carried out on this fusion image automatically, followed by manual spot editing. Subsequently, spots were transferred from fusion image to all gels. The signal intensities of each spot was computed as a weighted sum of all pixel intensities (volume of protein spot). Percent volume of spot intensities calculated as a fraction of the total spot volume of the parent gel was used for quantitative analysis of protein expression level. Ninety-five percent of the protein spots on the 2D gels that did not vary in their concentration and spot intensity served as reference. Thus, the balancing phenomenon is not due to a normalization artifact that could have arisen from global normalization to a mean or median. Normalized values after local background subtraction were subsequently exported from Delta2D in spreadsheet format for statistical analysis.</p>


<h4>Mass spectrometric protein identification</h4>
<a id="article1.body1.sec4.sec4.p1" name="article1.body1.sec4.sec4.p1"></a><p>For protein identification by mass spectrometry, 2-DE gels were stained with a mass spectrometry compatible silver staining protocol <a href="#pone.0001218-Nebrich1">[40]</a>. Protein spots of interest were excised from 2-DE gels and subjected to in-gel trypsin digestion without reduction and alkylation. Tryptic fragments were analyzed on a LCQ Deca XP nano HPLC/ESI ion trap mass spectrometer (Thermo Fisher Scientific, Waltham, MA, USA) as described previously <a href="#pone.0001218-Zabel2">[11]</a>. For database-assisted protein mass finger printing, monoisotopic mass values of peptides were searched against NCBI-nr (version 20061206, taxonomy: <em>Mus musculus</em>), allowing one missed cleavage. Peptide mass tolerance and fragment mass tolerance were set at 0.8 Dalton. Oxidation of methionine and arylamide adducts on cysteine (propionaide) were considered as variable peptide modifications. Criteria for positive identification of proteins were set according to the scoring algorithm delineated in Mascot (Matrix Science, London, UK) <a href="#pone.0001218-Pappin1">[41]</a>, with individual ion score cut-off threshold corresponding to p&lt;0.05.</p>


<h4>Annotation of biochemical properties and functional categories to proteins</h4>
<a id="article1.body1.sec4.sec5.p1" name="article1.body1.sec4.sec5.p1"></a><p>Public database queries were performed for the characterization of proteins with altered expression profiles in transgenic ES cells. For this purpose, <em>GOstat</em> (<a href="http://gostat.wehi.edu.au">http://gostat.wehi.edu.au</a>) was employed to annotate and search against the human GOA database (<a href="http://www.ebi.ac.uk/GOA">www.ebi.ac.uk/GOA</a>) in order to determine highly represented functional categories for our proteins of interests <a href="#pone.0001218-Ashburner1">[42]</a>. This tool integrates a Fisher's exact test that decides whether the observed GO-term over-representation is significant. p&lt;0.01 was set as statistical significance threshold. ProtParam was used to predict the protein instability, aliphatic index and Gravy scores (<a href="http://www.expasy.org/tools/protparam">www.expasy.org/tools/protparam</a>). The Human Reference Protein Database (<a href="http://www.hprd.org">www.hprd.org</a>) was used to access the number of direct protein interaction partners. Batch searches of overall protein-protein interaction network information were performed via the meta-database UniHi (<a href="http://theoderich.fb3.mdc-berlin.de:8080/unihi">http://theoderich.fb3.mdc-berlin.de:8080​/unihi</a>). Subsequently, information originating from HPRD, BIND, DIP and Reactome, which are curated manually, was extracted. The visualization of the protein interaction network was performed using Cytoscape (<a href="http://www.cytoscape.org">www.cytoscape.org</a>) <a href="#pone.0001218-Shannon1">[43]</a>. The Mouse Genome informatics database (MGI 3.5) was used to access the number of SNPs across 86 inbred mouse strains (<a href="http://www.informatics.jax.org">www.informatics.jax.org</a>). Biological pathway analyses were performed using KEGG (<a href="http://www.genome.ad.jp/kegg">www.genome.ad.jp/kegg</a>) and Biocarta pathway databases (<a href="http://www.biocarta.com">www.biocarta.com</a>). Protein abundance information was extracted from 2-DE data.</p>


<h4>Statistics</h4>
<a id="article1.body1.sec4.sec6.p1" name="article1.body1.sec4.sec6.p1"></a><p>To assess statistical significance of expression differences between transgenic and control cell lines, Student's T-test was carried out for control vs. transgenic groups (pair-wise, two sided, n = 6). p&lt;0.05 was used as statistical significance threshold. Only protein expression changes over 30% compared to control were retained for further analysis. As a post hoc control analysis, protein expression data generated from 2DE were scrutinized using the Significance Analysis of Microarrays tool (SAM, www-stat.stanford.edu) to identify the false detective rate required to gain the comparable set of altered proteins (100 permutations) <a href="#pone.0001218-Tusher1">[44]</a>. Protein expression alteration (fold change against wild-type controls) was reported with standard error of means (SEM). Due to their non-parametric distribution nature, protein property data (protein cellular abundance, biochemical properties and entropic contribution) were visualized as cumulative fraction plots. Differences of datasets between balancer and cell line-specific proteins were assessed with the Wilcoxon sum rank test (p&lt;0.05).</p>


<h4>Network-based approach for system robustness analysis</h4>
<a id="article1.body1.sec4.sec7.p1" name="article1.body1.sec4.sec7.p1"></a><p>Many aspects of cellular behavior are mainly determined by the structural properties of the underlying molecular network. In order to characterize the macroscopic resilience properties of the proteomic system, we adopted a network approach which is based on molecular protein interactions. This approach utilizes a fluctuation theorem <a href="#pone.0001218-Demetrius1">[24]</a>, which states that the resilience of macroscoptic network observables is positively correlated with the pathway diversity, a property which can be measured by network entropy. In this context, network entropy appears as the dynamical entropy of a stochastic process defined on the network, i.e the weighted-average Shannon entropy, <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001218.e001&amp;representation=PNG" class="inline-graphic"></span>, where π<sub>i</sub> is the stationary distribution of the stochastic process (<em>P<sub>ij</sub></em>) and <em>H<sub>i</sub></em> is the standard Shannon entropy defined by:<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001218.e002&amp;representation=PNG" class="inline-graphic"></span><br>i.e., the uncertainty about the next step of a random walk operating on the network. The stochastic process, <em>P<sub>ij</sub></em>, is defined through a variational principle for the leading eigenvalue, which, for unweighted networks, maximizes the overall network entropy <a href="#pone.0001218-Demetrius1">[24]</a>. Thus, “<em>H</em>” denotes the network entropy of the whole protein-protein interaction network, whereas “<em>H<sub>i</sub></em>” denotes the entropic contribution of each individual protein (see <a href="#pone.0001218.s002">Methods S1</a> for details). This entropic characterization leads to a natural importance ranking of proteins within the context of resilience of the global protein interaction network <a href="#pone.0001218-Manke1">[25]</a>. For this purpose, a protein-protein interaction network was generated from all proteins identified from the 2-DE protein pattern of ES cells. This generates an undirected, un-weighted information transfer graph where nodes denote proteins and the edges describe protein-protein interaction. The topological structure of the graph can be described by an NxN adjacency matrix A = (a<sub>ij</sub>). The entropic contribution of each protein to the global network entropy was calculated as in <a href="#pone.0001218-Manke1">[25]</a>.</p>

</div>

<div id="section5" class="section"><a id="s5" name="s5" toc="s5" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0001218"></div><a name="pone.0001218.s001" id="pone.0001218.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001218.s001">Table S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1"></a><p class="preSiDOI">Protein expression profile changes in transgenic ES cell lines.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001218.s001</p><a id="article1.body1.sec5.supplementary-material1.caption1.p2" name="article1.body1.sec5.supplementary-material1.caption1.p2"></a><p class="postSiDOI">(0.11 MB XLS)</p>
<a name="pone.0001218.s002" id="pone.0001218.s002"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001218.s002">Methods S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material2.caption1.p1" name="article1.body1.sec5.supplementary-material2.caption1.p1"></a><p class="preSiDOI">Supplementary method of network entropy calculation</p>
<p class="siDoi">doi:10.1371/journal.pone.0001218.s002</p><a id="article1.body1.sec5.supplementary-material2.caption1.p2" name="article1.body1.sec5.supplementary-material2.caption1.p2"></a><p class="postSiDOI">(0.05 MB PDF)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>We thank Mrs. Silke Becker for her excellent technical assistance with the mass spectrometric analysis, Pierre Savatier for CGR8 cells, Hans-Jörg Warnatz for the kind gift of cDNA for amyloid beta precursor protein, and Axel Kowald for the fruitful discussions. We would like to thank Mrs. Deirdre West for editorial assistance.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: LM CZ JK. Performed the experiments: LM MH TN FM. Analyzed the data: TM LM DH. Contributed reagents/materials/analysis tools: JD LM CC YH. Wrote the paper: TM LM HH JK.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0001218-Goldberg1" id="pone.0001218-Goldberg1"></a>Goldberg MS, Fleming SM, Palacino JJ, Cepeda C, Lam HA, et al.  (2003) Parkin-deficient mice exhibit nigrostriatal deficits but not loss of dopaminergic neurons. J Biol Chem  278: 43628–43635.  Epub 42003 Aug 43620.  <ul class="find" data-citedArticleID="1021510" data-doi="10.1074/jbc.m308947200"><li><a href="http://dx.doi.org/10.1074/jbc.m308947200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Parkin-deficient+mice+exhibit+nigrostriatal+deficits+but+not+loss+of+dopaminergic+neurons." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Parkin-deficient+mice+exhibit+nigrostriatal+deficits+but+not+loss+of+dopaminergic+neurons.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0001218-Mangiarini1" id="pone.0001218-Mangiarini1"></a>Mangiarini L, Sathasivam K, Seller M, Cozens B, Harper A, et al.  (1996) Exon 1 of the HD gene with an expanded CAG repeat is sufficient to cause a progressive neurological phenotype in transgenic mice. Cell  87: 493–506.  <ul class="find" data-citedArticleID="1021532" data-doi="10.1016/s0092-8674(00)81369-0"><li><a href="http://dx.doi.org/10.1016/s0092-8674(00)81369-0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Exon+1+of+the+HD+gene+with+an+expanded+CAG+repeat+is+sufficient+to+cause+a+progressive+neurological+phenotype+in+transgenic+mice." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Exon+1+of+the+HD+gene+with+an+expanded+CAG+repeat+is+sufficient+to+cause+a+progressive+neurological+phenotype+in+transgenic+mice.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0001218-Palacino1" id="pone.0001218-Palacino1"></a>Palacino JJ, Sagi D, Goldberg MS, Krauss S, Motz C, et al.  (2004) Mitochondrial dysfunction and oxidative damage in parkin-deficient mice. J Biol Chem  279: 18614–18622.  <ul class="find" data-citedArticleID="1021546" data-doi="10.1074/jbc.m401135200"><li><a href="http://dx.doi.org/10.1074/jbc.m401135200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mitochondrial+dysfunction+and+oxidative+damage+in+parkin-deficient+mice." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mitochondrial+dysfunction+and+oxidative+damage+in+parkin-deficient+mice.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0001218-Zabel1" id="pone.0001218-Zabel1"></a>Zabel C, Klose J (2004) Influence of Huntington's Disease on the Human and Mouse Proteome. Int Rev Neurobiol  61: 241–283.  <ul class="find" data-citedArticleID="1021568" data-doi="10.1016/s0074-7742(04)61010-5"><li><a href="http://dx.doi.org/10.1016/s0074-7742(04)61010-5" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Influence+of+Huntington%27s+Disease+on+the+Human+and+Mouse+Proteome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Influence+of+Huntington%27s+Disease+on+the+Human+and+Mouse+Proteome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0001218-Periquet1" id="pone.0001218-Periquet1"></a>Periquet M, Corti O, Jacquier S, Brice A (2005) Proteomic analysis of parkin knockout mice: alterations in energy metabolism, protein handling and synaptic function. J Neurochem  95: 1259–1276.  Epub 2005 Sep 1257.  <ul class="find" data-citedArticleID="1021550" data-doi="10.1111/j.1471-4159.2005.03442.x"><li><a href="http://dx.doi.org/10.1111/j.1471-4159.2005.03442.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Proteomic+analysis+of+parkin+knockout+mice%3A+alterations+in+energy+metabolism%2C+protein+handling+and+synaptic+function." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Proteomic+analysis+of+parkin+knockout+mice%3A+alterations+in+energy+metabolism%2C+protein+handling+and+synaptic+function.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0001218-SturchlerPierrat1" id="pone.0001218-SturchlerPierrat1"></a>Sturchler-Pierrat C, Abramowski D, Duke M, Wiederhold KH, Mistl C, et al.  (1997) Two amyloid precursor protein transgenic mouse models with Alzheimer disease-like pathology. Proc Natl Acad Sci U S A  94: 13287–13292.  <ul class="find" data-citedArticleID="1021558" data-doi="10.1073/pnas.94.24.13287"><li><a href="http://dx.doi.org/10.1073/pnas.94.24.13287" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Two+amyloid+precursor+protein+transgenic+mouse+models+with+Alzheimer+disease-like+pathology." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Two+amyloid+precursor+protein+transgenic+mouse+models+with+Alzheimer+disease-like+pathology.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0001218-Klose1" id="pone.0001218-Klose1"></a>Klose J, Kobalz U (1995) Two-dimensional electrophoresis of proteins: an updated protocol and implications for a functional analysis of the genome. Electrophoresis  16: 1034–1059.  <ul class="find" data-citedArticleID="1021524" data-doi="10.1002/elps.11501601175"><li><a href="http://dx.doi.org/10.1002/elps.11501601175" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Two-dimensional+electrophoresis+of+proteins%3A+an+updated+protocol+and+implications+for+a+functional+analysis+of+the+genome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Two-dimensional+electrophoresis+of+proteins%3A+an+updated+protocol+and+implications+for+a+functional+analysis+of+the+genome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0001218-Klose2" id="pone.0001218-Klose2"></a>Klose J (1999) Large-gel 2-D electrophoresis. Fractionated extraction of total tissue proteins from mouse and human for 2-D electrophoresis. Methods Mol Biol  112: 147–172.  <ul class="find" data-citedArticleID="1021526" data-doi="10.1385/1-59259-584-7:67"><li><a href="http://dx.doi.org/10.1385/1-59259-584-7:67" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Large-gel+2-D+electrophoresis.+Fractionated+extraction+of+total+tissue+proteins+from+mouse+and+human+for+2-D+electrophoresis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Large-gel+2-D+electrophoresis.+Fractionated+extraction+of+total+tissue+proteins+from+mouse+and+human+for+2-D+electrophoresis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0001218-Miller1" id="pone.0001218-Miller1"></a>Miller RM, Kiser GL, Kaysser-Kranich T, Casaceli C, Colla E, et al.  (2007) Wild-type and mutant alpha-synuclein induce a multi-component gene expression profile consistent with shared pathophysiology in different transgenic mouse models of PD. Exp Neurol  23: 23.  <ul class="find" data-citedArticleID="1021538" data-doi="10.1016/j.expneurol.2006.12.005"><li><a href="http://dx.doi.org/10.1016/j.expneurol.2006.12.005" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Wild-type+and+mutant+alpha-synuclein+induce+a+multi-component+gene+expression+profile+consistent+with+shared+pathophysiology+in+different+transgenic+mouse+models+of+PD." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Wild-type+and+mutant+alpha-synuclein+induce+a+multi-component+gene+expression+profile+consistent+with+shared+pathophysiology+in+different+transgenic+mouse+models+of+PD.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0001218-Zhang1" id="pone.0001218-Zhang1"></a>Zhang HL, Luo TH, Feng L, Zhao Y, Li WY, et al.  (2007) Microarray analysis of gene expression in Men1 knockout embryoid body reveals genetic events involved in early mouse embryonic development. Biochem Biophys Res Commun  352: 456–462.  Epub 2006 Nov 2016.  <ul class="find" data-citedArticleID="1021572" data-doi="10.1016/j.bbrc.2006.11.031"><li><a href="http://dx.doi.org/10.1016/j.bbrc.2006.11.031" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Microarray+analysis+of+gene+expression+in+Men1+knockout+embryoid+body+reveals+genetic+events+involved+in+early+mouse+embryonic+development." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Microarray+analysis+of+gene+expression+in+Men1+knockout+embryoid+body+reveals+genetic+events+involved+in+early+mouse+embryonic+development.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0001218-Zabel2" id="pone.0001218-Zabel2"></a>Zabel C, Sagi D, Kaindl AM, Steireif N, Klare Y, et al.  (2006) Comparative proteomics in neurodegenerative and non-neurodegenerative diseases suggest nodal point proteins in regulatory networking. J Proteome Res  5: 1948–1958.  <ul class="find" data-citedArticleID="1021570" data-doi="10.1021/pr0601077"><li><a href="http://dx.doi.org/10.1021/pr0601077" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Comparative+proteomics+in+neurodegenerative+and+non-neurodegenerative+diseases+suggest+nodal+point+proteins+in+regulatory+networking." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Comparative+proteomics+in+neurodegenerative+and+non-neurodegenerative+diseases+suggest+nodal+point+proteins+in+regulatory+networking.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0001218-Fowler1" id="pone.0001218-Fowler1"></a>Fowler DM, Koulov AV, Alory-Jost C, Marks MS, Balch WE, et al.  (2006) Functional amyloid formation within mammalian tissue. PLoS Biol  4: e6.  <ul class="find" data-citedArticleID="1021508" data-doi="10.1371/journal.pbio.0040006"><li><a href="http://dx.doi.org/10.1371/journal.pbio.0040006" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Functional+amyloid+formation+within+mammalian+tissue." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Functional+amyloid+formation+within+mammalian+tissue.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0001218-Lopes1" id="pone.0001218-Lopes1"></a>Lopes C, Chettouh Z, Delabar JM, Rachidi M (2003) The differentially expressed C21orf5 gene in the medial temporal-lobe system could play a role in mental retardation in Down syndrome and transgenic mice. Biochem Biophys Res Commun  305: 915–924.  <ul class="find" data-citedArticleID="1021528" data-doi="10.1016/s0006-291x(03)01163-x"><li><a href="http://dx.doi.org/10.1016/s0006-291x(03)01163-x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+differentially+expressed+C21orf5+gene+in+the+medial+temporal-lobe+system+could+play+a+role+in+mental+retardation+in+Down+syndrome+and+transgenic+mice." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+differentially+expressed+C21orf5+gene+in+the+medial+temporal-lobe+system+could+play+a+role+in+mental+retardation+in+Down+syndrome+and+transgenic+mice.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0001218-Branchi1" id="pone.0001218-Branchi1"></a>Branchi I, Bichler Z, Minghetti L, Delabar JM, Malchiodi-Albedi F, et al.  (2004) Transgenic mouse in vivo library of human Down syndrome critical region 1: association between DYRK1A overexpression, brain development abnormalities, and cell cycle protein alteration. J Neuropathol Exp Neurol  63: 429–440.  <ul class="find" data-citedArticleID="1021492"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Transgenic+mouse+in+vivo+library+of+human+Down+syndrome+critical+region+1%3A+association+between+DYRK1A+overexpression%2C+brain+development+abnormalities%2C+and+cell+cycle+protein+alteration.&amp;auth=&amp;atitle=Transgenic+mouse+in+vivo+library+of+human+Down+syndrome+critical+region+1%3A+association+between+DYRK1A+overexpression%2C+brain+development+abnormalities%2C+and+cell+cycle+protein+alteration." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Transgenic+mouse+in+vivo+library+of+human+Down+syndrome+critical+region+1%3A+association+between+DYRK1A+overexpression%2C+brain+development+abnormalities%2C+and+cell+cycle+protein+alteration." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Transgenic+mouse+in+vivo+library+of+human+Down+syndrome+critical+region+1%3A+association+between+DYRK1A+overexpression%2C+brain+development+abnormalities%2C+and+cell+cycle+protein+alteration.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0001218-Brault1" id="pone.0001218-Brault1"></a>Brault V, Pereira P, Duchon A, Herault Y (2006) Modeling chromosomes in mouse to explore the function of genes, genomic disorders, and chromosomal organization. PLoS Genet  2: e86.  <ul class="find" data-citedArticleID="1021494" data-doi="10.1371/journal.pgen.0020086"><li><a href="http://dx.doi.org/10.1371/journal.pgen.0020086" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Modeling+chromosomes+in+mouse+to+explore+the+function+of+genes%2C+genomic+disorders%2C+and+chromosomal+organization." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Modeling+chromosomes+in+mouse+to+explore+the+function+of+genes%2C+genomic+disorders%2C+and+chromosomal+organization.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0001218-Wuchty1" id="pone.0001218-Wuchty1"></a>Wuchty S, Oltvai ZN, Barabasi AL (2003) Evolutionary conservation of motif constituents in the yeast protein interaction network. Nat Genet  35: 176–179.  Epub 2003 Sep 2014.  <ul class="find" data-citedArticleID="1021566" data-doi="10.1038/ng1242"><li><a href="http://dx.doi.org/10.1038/ng1242" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evolutionary+conservation+of+motif+constituents+in+the+yeast+protein+interaction+network." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evolutionary+conservation+of+motif+constituents+in+the+yeast+protein+interaction+network.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0001218-Balazsi1" id="pone.0001218-Balazsi1"></a>Balazsi G, Barabasi AL, Oltvai ZN (2005) Topological units of environmental signal processing in the transcriptional regulatory network of Escherichia coli. Proc Natl Acad Sci U S A  102: 7841–7846.  Epub 2005 May 7820.  <ul class="find" data-citedArticleID="1021490" data-doi="10.1073/pnas.0500365102"><li><a href="http://dx.doi.org/10.1073/pnas.0500365102" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Topological+units+of+environmental+signal+processing+in+the+transcriptional+regulatory+network+of+Escherichia+coli." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Topological+units+of+environmental+signal+processing+in+the+transcriptional+regulatory+network+of+Escherichia+coli.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0001218-Brown1" id="pone.0001218-Brown1"></a>Brown GC (1991) Total cell protein concentration as an evolutionary constraint on the metabolic control distribution in cells. J Theor Biol  153: 195–203.  <ul class="find" data-citedArticleID="1021496" data-doi="10.1016/s0022-5193(05)80422-9"><li><a href="http://dx.doi.org/10.1016/s0022-5193(05)80422-9" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Total+cell+protein+concentration+as+an+evolutionary+constraint+on+the+metabolic+control+distribution+in+cells." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Total+cell+protein+concentration+as+an+evolutionary+constraint+on+the+metabolic+control+distribution+in+cells.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0001218-Minton1" id="pone.0001218-Minton1"></a>Minton AP (2000) Implications of macromolecular crowding for protein assembly. Curr Opin Struct Biol  10: 34–39.  <ul class="find" data-citedArticleID="1021540" data-doi="10.1016/s0959-440x(99)00045-7"><li><a href="http://dx.doi.org/10.1016/s0959-440x(99)00045-7" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Implications+of+macromolecular+crowding+for+protein+assembly." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Implications+of+macromolecular+crowding+for+protein+assembly.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0001218-Minton2" id="pone.0001218-Minton2"></a>Minton AP (2001) The influence of macromolecular crowding and macromolecular confinement on biochemical reactions in physiological media. J Biol Chem  276: 10577–10580.  Epub 12001 Feb 10515.  <ul class="find" data-citedArticleID="1021542" data-doi="10.1074/jbc.r100005200"><li><a href="http://dx.doi.org/10.1074/jbc.r100005200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+influence+of+macromolecular+crowding+and+macromolecular+confinement+on+biochemical+reactions+in+physiological+media." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+influence+of+macromolecular+crowding+and+macromolecular+confinement+on+biochemical+reactions+in+physiological+media.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0001218-Santoni1" id="pone.0001218-Santoni1"></a>Santoni V, Molloy M, Rabilloud T (2000) Membrane proteins and proteomics: un amour impossible? Electrophoresis  21: 1054–1070.  <ul class="find" data-citedArticleID="1021552" data-doi="10.1002/(sici)1522-2683(20000401)21:6<1054::aid-elps1054&gt;3.0.co;2-8"><li><a href="http://dx.doi.org/10.1002/(sici)1522-2683(20000401)21:6<1054::aid-elps1054&gt;3.0.co;2-8" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Membrane+proteins+and+proteomics%3A+un+amour+impossible%3F" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Membrane+proteins+and+proteomics%3A+un+amour+impossible%3F%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0001218-Khaitovich1" id="pone.0001218-Khaitovich1"></a>Khaitovich P, Hellmann I, Enard W, Nowick K, Leinweber M, et al.  (2005) Parallel patterns of evolution in the genomes and transcriptomes of humans and chimpanzees. Science  309: 1850–1854.  Epub 2005 Sep 1851.  <ul class="find" data-citedArticleID="1021522" data-doi="10.1126/science.1108296"><li><a href="http://dx.doi.org/10.1126/science.1108296" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Parallel+patterns+of+evolution+in+the+genomes+and+transcriptomes+of+humans+and+chimpanzees." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Parallel+patterns+of+evolution+in+the+genomes+and+transcriptomes+of+humans+and+chimpanzees.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0001218-Stelzl1" id="pone.0001218-Stelzl1"></a>Stelzl U, Worm U, Lalowski M, Haenig C, Brembeck FH, et al.  (2005) A human protein-protein interaction network: a resource for annotating the proteome. Cell  122: 957–968.  <ul class="find" data-citedArticleID="1021556" data-doi="10.1016/j.cell.2005.08.029"><li><a href="http://dx.doi.org/10.1016/j.cell.2005.08.029" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+human+protein-protein+interaction+network%3A+a+resource+for+annotating+the+proteome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+human+protein-protein+interaction+network%3A+a+resource+for+annotating+the+proteome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0001218-Demetrius1" id="pone.0001218-Demetrius1"></a>Demetrius L, Gundlach VM, Ochs G (2004) Complexity and demographic stability in population models. Theor Popul Biol  65: 211–225.  <ul class="find" data-citedArticleID="1021504" data-doi="10.1016/j.tpb.2003.12.002"><li><a href="http://dx.doi.org/10.1016/j.tpb.2003.12.002" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Complexity+and+demographic+stability+in+population+models." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Complexity+and+demographic+stability+in+population+models.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0001218-Manke1" id="pone.0001218-Manke1"></a>Manke T, Demetrius L, Vingron M (2006) An entropic characterization of protein interaction networks and cellular robustness. J R Soc Interface  3: 843–850.  <ul class="find" data-citedArticleID="1021534" data-doi="10.1098/rsif.2006.0140"><li><a href="http://dx.doi.org/10.1098/rsif.2006.0140" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=An+entropic+characterization+of+protein+interaction+networks+and+cellular+robustness." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22An+entropic+characterization+of+protein+interaction+networks+and+cellular+robustness.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0001218-Chan1" id="pone.0001218-Chan1"></a>Chan SK, Hsing M, Hormozdiari F, Cherkasov A (2007) Relationship between insertion/deletion (indel) frequency of proteins and essentiality. BMC Bioinformatics  8: 227.  <ul class="find" data-citedArticleID="1021500" data-doi="10.1186/1471-2105-8-227"><li><a href="http://dx.doi.org/10.1186/1471-2105-8-227" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Relationship+between+insertion%2Fdeletion+%28indel%29+frequency+of+proteins+and+essentiality." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Relationship+between+insertion%2Fdeletion+%28indel%29+frequency+of+proteins+and+essentiality.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0001218-Joy1" id="pone.0001218-Joy1"></a>Joy MP, Brock A, Ingber DE, Huang S (2005) High-betweenness proteins in the yeast protein interaction network. J Biomed Biotechnol  2005: 96–103.  <ul class="find" data-citedArticleID="1021518" data-doi="10.1155/jbb.2005.96"><li><a href="http://dx.doi.org/10.1155/jbb.2005.96" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=High-betweenness+proteins+in+the+yeast+protein+interaction+network." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22High-betweenness+proteins+in+the+yeast+protein+interaction+network.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0001218-Kauffman1" id="pone.0001218-Kauffman1"></a>Kauffman S (1969) Homeostasis and differentiation in random genetic control networks. Nature  224: 177–178.  <ul class="find" data-citedArticleID="1021520" data-doi="10.1038/224177a0"><li><a href="http://dx.doi.org/10.1038/224177a0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Homeostasis+and+differentiation+in+random+genetic+control+networks." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Homeostasis+and+differentiation+in+random+genetic+control+networks.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0001218-Huang1" id="pone.0001218-Huang1"></a>Huang S, Ingber DE (2000) Shape-dependent control of cell growth, differentiation, and apoptosis: switching between attractors in cell regulatory networks. Exp Cell Res  261: 91–103.  <ul class="find" data-citedArticleID="1021514" data-doi="10.1006/excr.2000.5044"><li><a href="http://dx.doi.org/10.1006/excr.2000.5044" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Shape-dependent+control+of+cell+growth%2C+differentiation%2C+and+apoptosis%3A+switching+between+attractors+in+cell+regulatory+networks." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Shape-dependent+control+of+cell+growth%2C+differentiation%2C+and+apoptosis%3A+switching+between+attractors+in+cell+regulatory+networks.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0001218-Huang2" id="pone.0001218-Huang2"></a>Huang S, Ingber DE (2006) A non-genetic basis for cancer progression and metastasis: self-organizing attractors in cell regulatory networks. Breast Dis  26: 27–54.  <ul class="find" data-citedArticleID="1021516"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=A+non-genetic+basis+for+cancer+progression+and+metastasis%3A+self-organizing+attractors+in+cell+regulatory+networks.&amp;auth=&amp;atitle=A+non-genetic+basis+for+cancer+progression+and+metastasis%3A+self-organizing+attractors+in+cell+regulatory+networks." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+non-genetic+basis+for+cancer+progression+and+metastasis%3A+self-organizing+attractors+in+cell+regulatory+networks." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+non-genetic+basis+for+cancer+progression+and+metastasis%3A+self-organizing+attractors+in+cell+regulatory+networks.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0001218-Chambers1" id="pone.0001218-Chambers1"></a>Chambers I, Smith A (2004) Self-renewal of teratocarcinoma and embryonic stem cells. Oncogene  23: 7150–7160.  <ul class="find" data-citedArticleID="1021498" data-doi="10.1038/sj.onc.1207930"><li><a href="http://dx.doi.org/10.1038/sj.onc.1207930" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Self-renewal+of+teratocarcinoma+and+embryonic+stem+cells." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Self-renewal+of+teratocarcinoma+and+embryonic+stem+cells.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0001218-Wiles1" id="pone.0001218-Wiles1"></a>Wiles MV, Vauti F, Otte J, Fuchtbauer EM, Ruiz P, et al.  (2000) Establishment of a gene-trap sequence tag library to generate mutant mice from embryonic stem cells. Nat Genet  24: 13–14.  <ul class="find" data-citedArticleID="1021564"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Establishment+of+a+gene-trap+sequence+tag+library+to+generate+mutant+mice+from+embryonic+stem+cells.&amp;auth=&amp;atitle=Establishment+of+a+gene-trap+sequence+tag+library+to+generate+mutant+mice+from+embryonic+stem+cells." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Establishment+of+a+gene-trap+sequence+tag+library+to+generate+mutant+mice+from+embryonic+stem+cells." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Establishment+of+a+gene-trap+sequence+tag+library+to+generate+mutant+mice+from+embryonic+stem+cells.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0001218-Chung1" id="pone.0001218-Chung1"></a>Chung S, Andersson T, Sonntag KC, Bjorklund L, Isacson O, et al.  (2002) Analysis of different promoter systems for efficient transgene expression in mouse embryonic stem cell lines. Stem Cells  20: 139–145.  <ul class="find" data-citedArticleID="1021502" data-doi="10.1634/stemcells.20-2-139"><li><a href="http://dx.doi.org/10.1634/stemcells.20-2-139" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Analysis+of+different+promoter+systems+for+efficient+transgene+expression+in+mouse+embryonic+stem+cell+lines." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Analysis+of+different+promoter+systems+for+efficient+transgene+expression+in+mouse+embryonic+stem+cell+lines.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0001218-Adams1" id="pone.0001218-Adams1"></a>Adams DJ, Biggs PJ, Cox T, Davies R, van der Weyden L, et al.  (2004) Mutagenic insertion and chromosome engineering resource (MICER). Nat Genet  36: 867–871.  Epub 2004 Jul 2004.  <ul class="find" data-citedArticleID="1021486" data-doi="10.1038/ng1388"><li><a href="http://dx.doi.org/10.1038/ng1388" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mutagenic+insertion+and+chromosome+engineering+resource+%28MICER%29." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mutagenic+insertion+and+chromosome+engineering+resource+%28MICER%29.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0001218-Magin1" id="pone.0001218-Magin1"></a>Magin TM, McWhir J, Melton DW (1992) A new mouse embryonic stem cell line with good germ line contribution and gene targeting frequency. Nucleic Acids Res  20: 3795–3796.  <ul class="find" data-citedArticleID="1021530" data-doi="10.1093/nar/20.14.3795"><li><a href="http://dx.doi.org/10.1093/nar/20.14.3795" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+new+mouse+embryonic+stem+cell+line+with+good+germ+line+contribution+and+gene+targeting+frequency." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+new+mouse+embryonic+stem+cell+line+with+good+germ+line+contribution+and+gene+targeting+frequency.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">36.
              </span><a name="pone.0001218-Testa1" id="pone.0001218-Testa1"></a>Testa G, Vintersten K, Zhang Y, Benes V, Muyrers JP, et al.  (2004) BAC engineering for the generation of ES cell-targeting constructs and mouse transgenes. Methods Mol Biol  256: 123–139.  <ul class="find" data-citedArticleID="1021560" data-doi="10.1385/1-59259-753-x:123"><li><a href="http://dx.doi.org/10.1385/1-59259-753-x:123" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=BAC+engineering+for+the+generation+of+ES+cell-targeting+constructs+and+mouse+transgenes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22BAC+engineering+for+the+generation+of+ES+cell-targeting+constructs+and+mouse+transgenes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">37.
              </span><a name="pone.0001218-Doetschman1" id="pone.0001218-Doetschman1"></a>Doetschman TC, Eistetter H, Katz M, Schmidt W, Kemler R (1985) The in vitro development of blastocyst-derived embryonic stem cell lines: formation of visceral yolk sac, blood islands and myocardium. J Embryol Exp Morphol  87: 27–45.  <ul class="find" data-citedArticleID="1021506"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=The+in+vitro+development+of+blastocyst-derived+embryonic+stem+cell+lines%3A+formation+of+visceral+yolk+sac%2C+blood+islands+and+myocardium.&amp;auth=&amp;atitle=The+in+vitro+development+of+blastocyst-derived+embryonic+stem+cell+lines%3A+formation+of+visceral+yolk+sac%2C+blood+islands+and+myocardium." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+in+vitro+development+of+blastocyst-derived+embryonic+stem+cell+lines%3A+formation+of+visceral+yolk+sac%2C+blood+islands+and+myocardium." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+in+vitro+development+of+blastocyst-derived+embryonic+stem+cell+lines%3A+formation+of+visceral+yolk+sac%2C+blood+islands+and+myocardium.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">38.
              </span><a name="pone.0001218-Heukeshoven1" id="pone.0001218-Heukeshoven1"></a>Heukeshoven J, Dernick R (1988) Improved silver staining procedure for fast staining in PhastSystem Development Unit. I. Staining of sodium dodecyl sulfate gels. Electrophoresis  9: 28–32.  <ul class="find" data-citedArticleID="1021512" data-doi="10.1002/elps.1150090106"><li><a href="http://dx.doi.org/10.1002/elps.1150090106" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Improved+silver+staining+procedure+for+fast+staining+in+PhastSystem+Development+Unit.+I.+Staining+of+sodium+dodecyl+sulfate+gels." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Improved+silver+staining+procedure+for+fast+staining+in+PhastSystem+Development+Unit.+I.+Staining+of+sodium+dodecyl+sulfate+gels.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">39.
              </span><a name="pone.0001218-Mao1" id="pone.0001218-Mao1"></a>Mao L, Zabel C, Wacker MA, Nebrich G, Sagi D, et al.  (2006) Estimation of the mtDNA mutation rate in aging mice by proteome analysis and mathematical modeling. Exp Gerontol  41: 11–24.  Epub 2005 Nov 2022.  <ul class="find" data-citedArticleID="1021536" data-doi="10.1016/j.exger.2005.09.012"><li><a href="http://dx.doi.org/10.1016/j.exger.2005.09.012" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Estimation+of+the+mtDNA+mutation+rate+in+aging+mice+by+proteome+analysis+and+mathematical+modeling." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Estimation+of+the+mtDNA+mutation+rate+in+aging+mice+by+proteome+analysis+and+mathematical+modeling.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">40.
              </span><a name="pone.0001218-Nebrich1" id="pone.0001218-Nebrich1"></a>Nebrich G, Herrmann M, Sagi D, Klose J, Giavalisco P (2007) High MS-compatibility of silver nitrate-stained protein spots from 2-DE gels using ZipPlates and AnchorChips for successful protein identification. Electrophoresis  19: 19.  <ul class="find" data-citedArticleID="1021544" data-doi="10.1002/elps.200600656"><li><a href="http://dx.doi.org/10.1002/elps.200600656" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=High+MS-compatibility+of+silver+nitrate-stained+protein+spots+from+2-DE+gels+using+ZipPlates+and+AnchorChips+for+successful+protein+identification." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22High+MS-compatibility+of+silver+nitrate-stained+protein+spots+from+2-DE+gels+using+ZipPlates+and+AnchorChips+for+successful+protein+identification.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">41.
              </span><a name="pone.0001218-Pappin1" id="pone.0001218-Pappin1"></a>Pappin DJC, Hojrup P, Bleasby AJ (1993) Rapid identification of proteins by peptide-mass fingerprinting. Curr Biol  3: 327–332.  <ul class="find" data-citedArticleID="1021548" data-doi="10.1016/0960-9822(93)90195-t"><li><a href="http://dx.doi.org/10.1016/0960-9822(93)90195-t" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Rapid+identification+of+proteins+by+peptide-mass+fingerprinting." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Rapid+identification+of+proteins+by+peptide-mass+fingerprinting.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">42.
              </span><a name="pone.0001218-Ashburner1" id="pone.0001218-Ashburner1"></a>Ashburner M, Ball CA, Blake JA, Botstein D, Butler H, et al.  (2000) Gene ontology: tool for the unification of biology. The Gene Ontology Consortium. Nat Genet  25: 25–29.  <ul class="find" data-citedArticleID="1021488"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Gene+ontology%3A+tool+for+the+unification+of+biology.+The+Gene+Ontology+Consortium.&amp;auth=&amp;atitle=Gene+ontology%3A+tool+for+the+unification+of+biology.+The+Gene+Ontology+Consortium." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Gene+ontology%3A+tool+for+the+unification+of+biology.+The+Gene+Ontology+Consortium." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Gene+ontology%3A+tool+for+the+unification+of+biology.+The+Gene+Ontology+Consortium.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">43.
              </span><a name="pone.0001218-Shannon1" id="pone.0001218-Shannon1"></a>Shannon P, Markiel A, Ozier O, Baliga NS, Wang JT, et al.  (2003) Cytoscape: a software environment for integrated models of biomolecular interaction networks. Genome Res  13: 2498–2504.  <ul class="find" data-citedArticleID="1021554" data-doi="10.1101/gr.1239303"><li><a href="http://dx.doi.org/10.1101/gr.1239303" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Cytoscape%3A+a+software+environment+for+integrated+models+of+biomolecular+interaction+networks." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Cytoscape%3A+a+software+environment+for+integrated+models+of+biomolecular+interaction+networks.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">44.
              </span><a name="pone.0001218-Tusher1" id="pone.0001218-Tusher1"></a>Tusher VG, Tibshirani R, Chu G (2001) Significance analysis of microarrays applied to the ionizing radiation response. Proc Natl Acad Sci U S A  98: 5116–5121.  Epub 2001 Apr 5117.  <ul class="find" data-citedArticleID="1021562" data-doi="10.1073/pnas.091062498"><li><a href="http://dx.doi.org/10.1073/pnas.091062498" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Significance+analysis+of+microarrays+applied+to+the+ionizing+radiation+response." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Significance+analysis+of+microarrays+applied+to+the+ionizing+radiation+response.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001218.XML" value="136282"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001218.PDF" value="450492"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001218.g001.PNG_L" value="2104411"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001218.g001.PNG_M" value="253118"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001218.g001.PNG_S" value="18070"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001218.g001.TIF" value="3371142"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001218.g001.PNG_I" value="83727"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001218.t001.PNG_L" value="58030"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001218.t001.PNG_M" value="35869"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001218.t001.PNG_S" value="8173"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001218.t001.TIF" value="274532"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001218.t001.PNG_I" value="10862"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001218.t002.PNG_L" value="358986"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001218.t002.PNG_M" value="185628"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001218.t002.PNG_S" value="17333"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001218.t002.TIF" value="1629202"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001218.t002.PNG_I" value="59450"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001218.g002.PNG_L" value="289936"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001218.g002.PNG_M" value="130221"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001218.g002.PNG_S" value="17378"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001218.g002.TIF" value="762134"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001218.g002.PNG_I" value="44739"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001218.g003.PNG_L" value="42875"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001218.g003.PNG_M" value="32833"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001218.g003.PNG_S" value="10238"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001218.g003.TIF" value="272650"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001218.g003.PNG_I" value="11084"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001218.g004.PNG_L" value="295776"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001218.g004.PNG_M" value="62273"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001218.g004.PNG_S" value="16019"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001218.g004.TIF" value="925816"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001218.g004.PNG_I" value="53449"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001218.t003.PNG_L" value="184339"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001218.t003.PNG_M" value="101579"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001218.t003.PNG_S" value="11613"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001218.t003.TIF" value="830450"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001218.t003.PNG_I" value="31153"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001218.g005.PNG_L" value="1544088"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001218.g005.PNG_M" value="259749"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001218.g005.PNG_S" value="14194"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001218.g005.TIF" value="2810928"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001218.g005.PNG_I" value="78622"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001218.t004.PNG_L" value="120213"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001218.t004.PNG_M" value="65772"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001218.t004.PNG_S" value="9423"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001218.t004.TIF" value="523674"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001218.t004.PNG_I" value="19813"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001218.t005.PNG_L" value="344995"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001218.t005.PNG_M" value="188717"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001218.t005.PNG_S" value="17279"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001218.t005.TIF" value="1542294"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001218.t005.PNG_I" value="56872"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001218.e001.PNG" value="6958"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001218.e001.TIF" value="22884"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001218.e002.PNG" value="8467"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001218.e002.TIF" value="24360"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001218.s001.XLS" value="114176"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001218.s002.PDF" value="53050"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001218&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001218" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001218&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0001218&volume=&issue=&title=Proteomic Shifts in Embryonic Stem Cells with Gene Dose Modifications Suggest the Presence of Balancer Proteins in Protein Regulatory Networks&author_name=Lei%20Mao%2C%20Claus%20Zabel%2C%20Marion%20Herrmann%2C%20Tobias%20Nolden%2C%20Florian%20Mertes%2C%20Laetitia%20Magnol%2C%20Caroline%20Chabert%2C%20Daniela%20Hartl%2C%20Yann%20Herault%2C%20Jean%20Maurice%20Delabar%2C%20Thomas%20Manke%2C%20Heinz%20Himmelbauer%2C%20Joachim%20Klose&start_page=1&end_page=12" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001218" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001218" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0001218" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001218&amp;t=Proteomic%20Shifts%20in%20Embryonic%20Stem%20Cells%20with%20Gene%20Dose%20Modifications%20Suggest%20the%20Presence%20of%20Balancer%20Proteins%20in%20Protein%20Regulatory%20Networks" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001218&title=Proteomic%20Shifts%20in%20Embryonic%20Stem%20Cells%20with%20Gene%20Dose%20Modifications%20Suggest%20the%20Presence%20of%20Balancer%20Proteins%20in%20Protein%20Regulatory%20Networks&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001218&amp;title=Proteomic%20Shifts%20in%20Embryonic%20Stem%20Cells%20with%20Gene%20Dose%20Modifications%20Suggest%20the%20Presence%20of%20Balancer%20Proteins%20in%20Protein%20Regulatory%20Networks" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001218" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0001218&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Proteomic Shifts in Embryonic Stem Cells with Gene Dose Modifications Suggest the Presence of Balancer Proteins in Protein Regulatory Networks';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0001218';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Proteomic%20Shifts%20in%20Embryonic%20Stem%20Cells%20with%20Gene%20Dose%20Modifications%20Suggest%20the%20Presence%20of%20Balancer%20Proteins%20in%20Protein%20Regulatory%20Networks http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001218" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0001218" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">










          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Cell+metabolism%22" title="Search for articles in the subject area:'Cell metabolism'"><div class="flagText">Cell metabolism</div></a>
              <div data-categoryid="18083" data-articleid="25268"
                   data-categoryname="Cell metabolism"
                   class="flagImage" title="Flag 'Cell metabolism' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22DNA-binding+proteins%22" title="Search for articles in the subject area:'DNA-binding proteins'"><div class="flagText">DNA-binding proteins</div></a>
              <div data-categoryid="17441" data-articleid="25268"
                   data-categoryname="DNA-binding proteins"
                   class="flagImage" title="Flag 'DNA-binding proteins' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Entropy%22" title="Search for articles in the subject area:'Entropy'"><div class="flagText">Entropy</div></a>
              <div data-categoryid="21749" data-articleid="25268"
                   data-categoryname="Entropy"
                   class="flagImage" title="Flag 'Entropy' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Metabolic+networks%22" title="Search for articles in the subject area:'Metabolic networks'"><div class="flagText">Metabolic networks</div></a>
              <div data-categoryid="21289" data-articleid="25268"
                   data-categoryname="Metabolic networks"
                   class="flagImage" title="Flag 'Metabolic networks' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Protein+expression%22" title="Search for articles in the subject area:'Protein expression'"><div class="flagText">Protein expression</div></a>
              <div data-categoryid="17513" data-articleid="25268"
                   data-categoryname="Protein expression"
                   class="flagImage" title="Flag 'Protein expression' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Protein+interaction+networks%22" title="Search for articles in the subject area:'Protein interaction networks'"><div class="flagText">Protein interaction networks</div></a>
              <div data-categoryid="31803" data-articleid="25268"
                   data-categoryname="Protein interaction networks"
                   class="flagImage" title="Flag 'Protein interaction networks' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Protein+metabolism%22" title="Search for articles in the subject area:'Protein metabolism'"><div class="flagText">Protein metabolism</div></a>
              <div data-categoryid="19757" data-articleid="25268"
                   data-categoryname="Protein metabolism"
                   class="flagImage" title="Flag 'Protein metabolism' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Proteomes%22" title="Search for articles in the subject area:'Proteomes'"><div class="flagText">Proteomes</div></a>
              <div data-categoryid="18649" data-articleid="25268"
                   data-categoryname="Proteomes"
                   class="flagImage" title="Flag 'Proteomes' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=7672'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=2224'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=5460&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>

<div class="block sidebar-comments">
    <div class="header">
        <h3>Comments</h3>
    </div>
      <p><a href="/annotation/listThread.action?root=4017">Referee Comments: Referee 1</a><br>Posted by PLoS_ONE_Group</p>
</div>

</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
