

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Screen for Localized Proteins in Caulobacter crescentus</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0001756"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0001756" />

    <meta name="citation_title" content="Screen for Localized Proteins in Caulobacter crescentus"/>
    <meta itemprop="name" content="Screen for Localized Proteins in Caulobacter crescentus"/>

      <meta name="citation_author" content="Jay H. Russell"/>
            <meta name="citation_author_institution" content="Department of Biochemistry and Molecular Biology, The Pennsylvania State University, University Park, Pennsylvania, United States of America"/>
      <meta name="citation_author" content="Kenneth C. Keiler"/>
            <meta name="citation_author_institution" content="Department of Biochemistry and Molecular Biology, The Pennsylvania State University, University Park, Pennsylvania, United States of America"/>

    <meta name="citation_date" content="2008/3/12"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0001756.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e1756"/>
    <meta name="citation_issue" content="3"/>
    <meta name="citation_volume" content="3"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Generating and exploiting polarity in bacteria.; citation_author=L Shapiro; citation_author=HH McAdams; citation_author=R Losick; citation_journal_title=Science; citation_volume=298; citation_number=1; citation_pages=1942-1946; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Temporal and spatial regulation in prokaryotic cell cycle progression and development.; citation_author=KR Ryan; citation_author=L Shapiro; citation_journal_title=Annu Rev Biochem; citation_volume=72; citation_number=2; citation_pages=367-394; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Regulatory proteins with a sense of direction: cell cycle signalling network in Caulobacter.; citation_author=C Jacobs-Wagner; citation_journal_title=Mol Microbiol; citation_volume=51; citation_number=3; citation_pages=7-13; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=A bacterial cell-cycle regulatory network operating in time and space.; citation_author=HH McAdams; citation_author=L Shapiro; citation_journal_title=Science; citation_volume=301; citation_number=4; citation_pages=1874-1877; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Cytokinesis in bacteria.; citation_author=J Errington; citation_author=RA Daniel; citation_author=DJ Scheffers; citation_journal_title=Microbiol Mol Biol Rev; citation_volume=67; citation_number=5; citation_pages=52-65; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Polar Location of the chemoreceptor complex in the Escherichia coli cell.; citation_author=JR Maddock; citation_author=L Shapiro; citation_journal_title=Science; citation_volume=259; citation_number=6; citation_pages=1717-1723; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=Peptide signals encode protein localization.; citation_author=JH Russell; citation_author=KC Keiler; citation_journal_title=J Bacteriol; citation_volume=189; citation_number=7; citation_pages=7581-7585; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=The bacterial cytoskeleton.; citation_author=YL Shih; citation_author=L Rothfield; citation_journal_title=Microbiol Mol Biol Rev; citation_volume=70; citation_number=8; citation_pages=729-754; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Complete set of ORF clones of Escherichia coli ASKA library (a complete set of E. coli K-12 ORF archive): unique resources for biological research.; citation_author=M Kitagawa; citation_author=T Ara; citation_author=M Arifuzzaman; citation_author=T Ioka-Nakamichi; citation_author=E Inamoto; citation_journal_title=DNA Res; citation_volume=12; citation_number=9; citation_pages=291-299; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Envelope-associated nucleoid from Caulobacter crescentus stalked and swarmer cells.; citation_author=M Evinger; citation_author=N Agabian; citation_journal_title=J Bacteriol; citation_volume=132; citation_number=10; citation_pages=294-301; citation_date=1977; " />
      <meta name="citation_reference" content="citation_title=Spatial and temporal control of differentiation and cell cycle progression in Caulobacter crescentus.; citation_author=N Ausmees; citation_author=C Jacobs-Wagner; citation_journal_title=Annu Rev Microbiol; citation_volume=57; citation_number=11; citation_pages=225-247; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Genetics of Caulobacter crescentus.; citation_author=B Ely; citation_journal_title=Methods Enzymol; citation_volume=204; citation_number=12; citation_pages=372-384; citation_date=1991; " />
      <meta name="citation_reference" content="citation_title=Positive selection procedure for entrapment of insertion sequence elements in gram-negative bacteria.; citation_author=P Gay; citation_author=D Le Coq; citation_author=M Steinmetz; citation_author=T Berkelman; citation_author=CI Kado; citation_journal_title=J Bacteriol; citation_volume=164; citation_number=13; citation_pages=918-921; citation_date=1985; " />
      <meta name="citation_reference" content="citation_title=tmRNA is required for correct timing of DNA replication in Caulobacter crescentus.; citation_author=KC Keiler; citation_author=L Shapiro; citation_journal_title=J Bacteriol; citation_volume=185; citation_number=14; citation_pages=573-580; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Mini-Tn5 transposon derivatives for insertion mutagenesis, promoter probing, and chromosomal insertion of cloned DNA in gram-negative eubacteria.; citation_author=V de Lorenzo; citation_author=M Herrero; citation_author=U Jakubzik; citation_author=KN Timmis; citation_journal_title=J Bacteriol; citation_volume=172; citation_number=15; citation_pages=6568-6572; citation_date=1990; " />
      <meta name="citation_reference" content="citation_title=A short polypeptide marker sequence useful for recombinant protein identification and purification.; citation_author=TP Hopp; citation_author=KS Prickett; citation_author=VL Price; citation_author=RT Libby; citation_author=CJ March; citation_volume=6; citation_number=16; citation_pages=1204-1210; citation_date=1988; " />
      <meta name="citation_reference" content="citation_title=Isolation and characterization of a xylose-dependent promoter from Caulobacter crescentus.; citation_author=AC Meisenzahl; citation_author=L Shapiro; citation_author=U Jenal; citation_journal_title=J Bacteriol; citation_volume=179; citation_number=17; citation_pages=592-600; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=A comprehensive set of plasmids for vanillate- and xylose-inducible gene expression in Caulobacter crescentus.; citation_author=M Thanbichler; citation_author=AA Iniesta; citation_author=L Shapiro; citation_journal_title=Nucleic Acids Res; citation_volume=35; citation_number=18; citation_pages=e137; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=MipZ, a spatial regulator coordinating chromosome segregation with cell division in Caulobacter.; citation_author=M Thanbichler; citation_author=L Shapiro; citation_journal_title=Cell; citation_volume=126; citation_number=19; citation_pages=147-162; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Fingerprinting genomes using PCR with arbitrary primers.; citation_author=J Welsh; citation_author=M McClelland; citation_journal_title=Nucleic Acids Res; citation_volume=18; citation_number=20; citation_pages=7213-7218; citation_date=1990; " />
      <meta name="citation_reference" content="citation_title=Genetic applications of an inverse polymerase chain reaction.; citation_author=H Ochman; citation_author=AS Gerber; citation_author=DL Hartl; citation_journal_title=Genetics; citation_volume=120; citation_number=21; citation_pages=621-623; citation_date=1988; " />
      <meta name="citation_reference" content="citation_title=Stabilization of polar localization of a chemoreceptor via its covalent modifications and its communication with a different chemoreceptor.; citation_author=D Shiomi; citation_author=S Banno; citation_author=M Homma; citation_author=I Kawagishi; citation_journal_title=J Bacteriol; citation_volume=187; citation_number=22; citation_pages=7647-7654; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=The receptor binding site for the methyltransferase of bacterial chemotaxis is distinct from the sites of methylation.; citation_author=J Wu; citation_author=J Li; citation_author=G Li; citation_author=DG Long; citation_author=RM Weis; citation_journal_title=Biochemistry; citation_volume=35; citation_number=23; citation_pages=4984-4993; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Polar localization of a bacterial chemoreceptor.; citation_author=MR Alley; citation_author=JR Maddock; citation_author=L Shapiro; citation_journal_title=Genes Dev; citation_volume=6; citation_number=24; citation_pages=825-836; citation_date=1992; " />
      <meta name="citation_reference" content="citation_title=The Caulobacter crescentus paracrystalline S-layer protein is secreted by an ABC transporter (type I) secretion apparatus.; citation_author=P Awram; citation_author=J Smit; citation_journal_title=J Bacteriol; citation_volume=180; citation_number=25; citation_pages=3062-3069; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Secretion of the Caulobacter crescentus S-layer protein: further localization of the C-terminal secretion signal and its use for secretion of recombinant proteins.; citation_author=WH Bingle; citation_author=JF Nomellini; citation_author=J Smit; citation_journal_title=J Bacteriol; citation_volume=182; citation_number=26; citation_pages=3298-3301; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Subcellular sites for bacterial protein export.; citation_author=N Campo; citation_author=H Tjalsma; citation_author=G Buist; citation_author=D Stepniak; citation_author=M Meijer; citation_journal_title=Mol Microbiol; citation_volume=53; citation_number=27; citation_pages=1583-1599; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=An actin-like gene can determine cell polarity in bacteria.; citation_author=Z Gitai; citation_author=N Dye; citation_author=L Shapiro; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=101; citation_number=28; citation_pages=8643-8648; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Division site selection in Escherichia coli involves dynamic redistribution of Min proteins within coiled structures that extend between the two cell poles.; citation_author=YL Shih; citation_author=T Le; citation_author=L Rothfield; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=100; citation_number=29; citation_pages=7865-7870; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=FtsZ-spirals and -arcs determine the shape of the invaginating septa in some mutants of Escherichia coli.; citation_author=SG Addinall; citation_author=J Lutkenhaus; citation_journal_title=Mol Microbiol; citation_volume=22; citation_number=30; citation_pages=231-237; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=RNaseE and the other constituents of the RNA degradosome are components of the bacterial cytoskeleton.; citation_author=A Taghbalout; citation_author=L Rothfield; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=104; citation_number=31; citation_pages=1667-1672; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=MreB actin-mediated segregation of a specific region of a bacterial chromosome.; citation_author=Z Gitai; citation_author=NA Dye; citation_author=A Reisenauer; citation_author=M Wachi; citation_author=L Shapiro; citation_journal_title=Cell; citation_volume=120; citation_number=32; citation_pages=329-341; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=A Transducing Bacteriophage for Caulobacter crescentus Uses the Paracrystalline Surface Layer Protein as a Receptor.; citation_author=P Edwards; citation_author=J Smit; citation_journal_title=J Bacteriol; citation_volume=173; citation_number=33; citation_pages=5568-5572; citation_date=1991; " />
      <meta name="citation_reference" content="citation_title=Complete genome sequence of Caulobacter crescentus.; citation_author=WC Nierman; citation_author=TV Feldblyum; citation_author=MT Laub; citation_author=IT Paulsen; citation_author=KE Nelson; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=98; citation_number=34; citation_pages=4136-4141; citation_date=2001; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001756" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Screen for Localized Proteins in Caulobacter crescentus"/>
    <meta name="twitter:description" content="Precise localization of individual proteins is required for processes such as motility, chemotaxis, cell-cycle progression, and cell division in bacteria, but the number of proteins that are localized in bacterial species is not known. A screen based on transposon mutagenesis and fluorescence activated cell sorting was devised to identify large numbers of localized proteins, and employed in Caulobacter crescentus. From a sample of the clones isolated in the screen, eleven proteins with no previously characterized localization in C. crescentus were identified, including six hypothetical proteins. The localized hypothetical proteins included one protein that was localized in a helix-like structure, and two proteins for which the localization changed as a function of the cell cycle, suggesting that complex three-dimensional patterns and cell cycle-dependent localization are likely to be common in bacteria. Other mutants produced localized fusion proteins even though the transposon has inserted near the 5&prime; end of a gene, demonstrating that short peptides can contain sufficient information to localize bacterial proteins. The screen described here could be used in most bacterial species."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0001756.g009"/>

  <meta property="og:title" content="Screen for Localized Proteins in Caulobacter crescentus" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001756" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=7414'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=3231'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=2582&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0001756">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001756" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001756&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Screen for Localized Proteins in <i>Caulobacter crescentus</i>
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Jay H. Russell, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Biochemistry and Molecular Biology, The Pennsylvania State University, University Park, Pennsylvania, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Kenneth C. Keiler
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">* E-mail:</span> <a href="mailto:kkeiler@psu.edu">kkeiler@psu.edu</a></p>

                <p>Affiliation:
                  Department of Biochemistry and Molecular Biology, The Pennsylvania State University, University Park, Pennsylvania, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: March 12, 2008</li>
    <li>DOI: 10.1371/journal.pone.0001756</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0001756" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0001756" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001756" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0001756" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0001756-t001" data-doi="info:doi/10.1371/journal.pone.0001756" data-uri="info:doi/10.1371/journal.pone.0001756.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001756.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001756-g001" data-doi="info:doi/10.1371/journal.pone.0001756" data-uri="info:doi/10.1371/journal.pone.0001756.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001756.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001756-g002" data-doi="info:doi/10.1371/journal.pone.0001756" data-uri="info:doi/10.1371/journal.pone.0001756.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001756.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001756-g003" data-doi="info:doi/10.1371/journal.pone.0001756" data-uri="info:doi/10.1371/journal.pone.0001756.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001756.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001756-g004" data-doi="info:doi/10.1371/journal.pone.0001756" data-uri="info:doi/10.1371/journal.pone.0001756.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001756.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001756-g005" data-doi="info:doi/10.1371/journal.pone.0001756" data-uri="info:doi/10.1371/journal.pone.0001756.g005" title="Figure 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001756.g005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001756-g006" data-doi="info:doi/10.1371/journal.pone.0001756" data-uri="info:doi/10.1371/journal.pone.0001756.g006" title="Figure 6">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001756.g006&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001756-g007" data-doi="info:doi/10.1371/journal.pone.0001756" data-uri="info:doi/10.1371/journal.pone.0001756.g007" title="Figure 7">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001756.g007&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001756-g008" data-doi="info:doi/10.1371/journal.pone.0001756" data-uri="info:doi/10.1371/journal.pone.0001756.g008" title="Figure 8">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001756.g008&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001756-g009" data-doi="info:doi/10.1371/journal.pone.0001756" data-uri="info:doi/10.1371/journal.pone.0001756.g009" title="Figure 9">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001756.g009&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001756">Reader Comments (0)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0001756" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1"></a><p>Precise localization of individual proteins is required for processes such as motility, chemotaxis, cell-cycle progression, and cell division in bacteria, but the number of proteins that are localized in bacterial species is not known. A screen based on transposon mutagenesis and fluorescence activated cell sorting was devised to identify large numbers of localized proteins, and employed in <em>Caulobacter crescentus.</em> From a sample of the clones isolated in the screen, eleven proteins with no previously characterized localization in <em>C. crescentus</em> were identified, including six hypothetical proteins. The localized hypothetical proteins included one protein that was localized in a helix-like structure, and two proteins for which the localization changed as a function of the cell cycle, suggesting that complex three-dimensional patterns and cell cycle-dependent localization are likely to be common in bacteria. Other mutants produced localized fusion proteins even though the transposon has inserted near the 5′ end of a gene, demonstrating that short peptides can contain sufficient information to localize bacterial proteins. The screen described here could be used in most bacterial species.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>Russell JH, Keiler KC (2008) Screen for Localized Proteins in <em>Caulobacter crescentus</em>. PLoS ONE 3(3):
          e1756.
            doi:10.1371/journal.pone.0001756</p><p><strong>Academic Editor: </strong>Christophe Herman, Baylor College of Medicine, United States of America</p><p><strong>Received:</strong> December 7, 2007; <strong>Accepted:</strong> February 7, 2008; <strong>Published:</strong> March 12, 2008</p><p><strong>Copyright:</strong> © 2008 Russell, Keiler. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>This work was supported by NIGMS grant GM068720. The funders played no role in any aspect of the research or manuscript.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>A fundamental regulatory challenge for all cells is to make the correct amount of protein at the proper time, and place it at a precise location. Several proteins in bacteria have been shown to be localized in an area measured in the tens of nanometers and not merely to domains such as the cytoplasm, periplasm, or membrane <a href="#pone.0001756-Shapiro1">[1]</a>–<a href="#pone.0001756-JacobsWagner1">[3]</a>. Critical processes such as cell division and differentiation depend on correct protein localization <a href="#pone.0001756-McAdams1">[4]</a>, <a href="#pone.0001756-Errington1">[5]</a>, yet for most localized proteins the mechanisms responsible for localization are unknown. In some cases, such as chemoreceptor complexes <a href="#pone.0001756-Maddock1">[6]</a>, extensive protein-protein interactions are required for localization. Localization information can also be encoded in relatively short peptide sequences, or locons, that have been shown to direct localization of exogenous proteins <a href="#pone.0001756-Russell1">[7]</a>. Several protein filaments similar to elements of the eukaryotic cytoskeleton have been discovered in bacteria <a href="#pone.0001756-Shih1">[8]</a>, but it is not yet clear if these filaments play a role in localization of most bacterial proteins. To understand the importance of these various mechanisms and to gauge the degree of order within a bacterial cell, it is necessary to identify as many localized proteins as possible.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>One approach to identify localized proteins is to clone and express each gene as a fusion to a reporter such as <em>gfp</em>, and observe each clone. When this approach was used in <em>E. coli</em>, approximately 20% of the proteins were localized <a href="#pone.0001756-Kitagawa1">[9]</a>. We have devised another approach, using a transposon to insert <em>gfp</em> at arbitrary sites in the genome. The use of transposon mutagenesis greatly decreases the resources required for examining a large number of genes. In addition, because GFP fusions are made at many sites within each gene, it is possible to identify short peptides that contain localization information and uncover localization determinants that are masked in a full-length protein. In this study, transposon mutagenesis was used to identify localized proteins and short localization sequences in the model bacterial system <em>Caulobacter crescentus</em>.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p><em>C. crescentus</em> has an asymmetric morphology that facilitates identification of specific subcellular locations <a href="#pone.0001756-Ryan1">[2]</a>. Cell division in <em>C. crescentus</em> produces two different daughter cells, a stalked cell, with a prosthesis at the old pole, and a swarmer cell with a single flagellum at the old pole. Swarmer cells must differentiate into stalked cells before they can replicate their DNA and divide. The stalk and flagellum can be used as markers for the poles of the cell, so that the newer pole can be distinguished from the older pole. Another significant advantage of <em>C. crescentus</em> is the ability to study changes in localization in response to development and cell cycle progression. <em>C. crescentus</em> swarmer cells can be isolated in a density gradient and these cells will pass synchronously through the cell cycle and developmental program <a href="#pone.0001756-Evinger1">[10]</a>. Synchronized cultures have been used to detect cell-cycle dependent localization of several structural and signaling proteins <a href="#pone.0001756-Ryan1">[2]</a>, <a href="#pone.0001756-Ausmees1">[11]</a>, but the number of proteins whose localization changes through the cell cycle is not known.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Methods"></a><h3>Methods</h3>
<h4>Plasmids and Strains</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>The wild-type <em>C. crescentus</em> strain was NA1000 <a href="#pone.0001756-Evinger1">[10]</a>. <em>C. crescentus</em> cells were grown in PYE medium or M2G medium at 30°C as previously described <a href="#pone.0001756-Ely1">[12]</a>. Synchronized cultures of <em>C. crescentus</em> were obtained by isolating swarmer cells from a Ludox density gradient <a href="#pone.0001756-Evinger1">[10]</a>. In-frame deletions of <em>CC0572</em>, <em>CC2233</em>, and <em>CC3691</em> were generated using plasmid pNPTS138 with the <em>sacB</em> counterselection method <a href="#pone.0001756-Gay1">[13]</a> as previously described <a href="#pone.0001756-Keiler1">[14]</a>.</p>
<a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2"></a><p>The mini-Tn5-GFP transposon was constructed by cloning the <em>egfp</em> gene from plasmid pEGFP-N2 (Novartis) into the Not I site of mini-Tn5 <em>Km2</em> in pUT <a href="#pone.0001756-deLorenzo1">[15]</a>. This plasmid was mobilized into <em>C. crescentus</em> by electroporation <a href="#pone.0001756-Ely1">[12]</a>. To generate M2-tagged proteins, the pM2C plasmid was constructed by cloning sequence encoding the M2 epitope <a href="#pone.0001756-Hopp1">[16]</a> downstream of the xylose-inducible promoter <a href="#pone.0001756-Meisenzahl1">[17]</a> in pML80 (M. Laub, unpublished). The coding sequence for <em>CC0572</em>, <em>CC3691</em>, or <em>ahpC</em> was then inserted between the promoter and the M2 sequence in pM2C. Production of an M2 fusion protein was induced in <em>C. crescentus</em> cells containing a pM2C-derived plasmid by addition of xylose to 0.3%. To produce full-length proteins fused to GFP, the coding sequence for CheR or CC2233 was inserted upstream of the <em>egfp</em> gene in pMT426 <a href="#pone.0001756-Thanbichler1">[18]</a> under control of the vanillate-inducible promoter <a href="#pone.0001756-Thanbichler2">[19]</a> and expression was induced in cells bearing the plasmid by addition of vanillate to 0.5 mM.</p>


<h4>Localization screen</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p><em>C. crescentus</em> cells were mutagenized by introduction of mini-Tn5-GFP and grown on PYE agar medium with 5 µg/ml kanamycin to select for mutants. Colonies (~20,000) were scraped into PYE medium, grown at 30°C for one hour, and cells with fluorescence above background levels were isolated by FACS (Stanford Shared FACS Facility) and grown on PYE agar medium. To screen for localized fluorescence signal, cells from individual colonies were grown in liquid medium and observed on wet mounts by epifluorescence microscopy.</p>
<a id="article1.body1.sec2.sec2.p2" name="article1.body1.sec2.sec2.p2"></a><p>To identify transposon insertion sites, mutant genomic DNA was isolated (Purgene) and fragments adjacent to <em>egfp</em> were cloned by arbitrary PCR <a href="#pone.0001756-Welsh1">[20]</a> with a specific primer within the <em>egfp</em> gene, ArbC (<span class="gene">GGCCACGCGTCGACTAGTAC</span>) and arbitrary primers Arb1 (<span class="gene">GGCCACGCGTCGACTAGTACNNNNNNNNNNGAT</span>) and Arb6 (<span class="gene">GGCCACGCGTCGACTAGTACNNNNNNNNNNACGCC</span>). Alternatively, for inverse PCR <a href="#pone.0001756-Ochman1">[21]</a>, genomic DNA was digested with Not I, incubated with T4 ligase at 16°C overnight, and amplified by PCR using primers GFP644Fwd (<span class="gene">GAGAAGCGCGATCACATGGTC</span>) and GFP133Rev (<span class="gene">GGGTCAGCTTGCCGTAGGTGGCATCGC</span>). Amplified DNA was cloned into pGEM T-Easy (Promega) and sequenced.</p>


<h4>Microscopy</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>Cells were immobilized on a 1% agar pad and imaged using an Eclipse E600 microscope (Nikon) with a 100× Plan Fluor oil N. A. 1.3 objective in conjunction with a CoolSNAP <em>fx</em> CCD camera (Photometrics) controlled by Image-Pro Discovery software (Media Cybernetics). Single microscopy images were deconvolved using the 2D Blind Deconvolution algorithm in AutoQuant software (Media Cybernetics).</p>
<a id="article1.body1.sec2.sec3.p2" name="article1.body1.sec2.sec3.p2"></a><p>For optical sectioning, an IX70 laser scanning confocal microscope (Olympus) with 100× UPlanFL oil N. A. 1.3 objective and Fluoview software was used to obtain a Z-series of confocal images at increments of 0.15 µM. Each Z-series was deconvolved and reconstituted using AutoQuant software.</p>
<a id="article1.body1.sec2.sec3.p3" name="article1.body1.sec2.sec3.p3"></a><p>For immunofluorescence microscopy, cells producing an M2 fusion protein were harvested at OD<sub>660</sub> 0.3–0.4 and prepared as previously described <a href="#pone.0001756-Maddock1">[6]</a>. Cells were stained with DAPI and probed with anti-FLAG M2-Cy3 antibody (Sigma), or anti-FLAG antibody (Sigma) followed by Alexa Fluor 488 goat anti–mouse IgG (Invitrogen). Images were obtained using an Eclipse 90i microscope (Nikon) with a 60× TIRF N. A. 1.4 objective with a Nikon CoolSNAP HQ CCD camera controlled by Simple PCi (Compix, Inc.).</p>
<a id="article1.body1.sec2.sec3.p4" name="article1.body1.sec2.sec3.p4"></a><p>To quantify the prevalence of localization patterns, at least 3 independent microscopy fields were chosen at random and intact cells were identified by DIC (for epifluorescence) or DAPI staining (for immunofluorescence). The signal in at least 80 intact cells was scored and the results were reported as a percentage of intact cells</p>

</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Results"></a><h3>Results</h3>
<h4>Transposon Mutagenesis to Identify Localized Proteins</h4>
<a id="article1.body1.sec3.sec1.p1" name="article1.body1.sec3.sec1.p1"></a><p>To identify proteins in <em>C. crescentus</em> that are localized, the mini-Tn5-GFP transposon was engineered to produce fusions of chromosomally-encoded proteins to GFP. A copy of the <em>egfp</em> gene was cloned near the O sequence of the mini-Tn5 <em>Km2</em> transposon <a href="#pone.0001756-deLorenzo1">[15]</a>, such that recombination of the transposon in the correct reading frame within a gene would result in a C-terminal fusion of the encoded protein to a 13 amino acid linker followed by GFP. Mutagenesis of <em>C. crescentus</em> with mini-Tn5-GFP produced approximately 20,000 kanamycin-resistant colonies.</p>
<a id="article1.body1.sec3.sec1.p2" name="article1.body1.sec3.sec1.p2"></a><p>Cells without GFP were expected to comprise about 90% of the colonies, because they could be produced from integration of the mini-Tn5-GFP in a non-coding region, out of the correct reading frame, or in a gene that was not expressed at high enough levels to produce observable GFP signal. Therefore, before examining cells for localized fusion proteins, clones that did not produce observable amounts of a GFP fusion protein were eliminated. Mutant colonies were pooled and cells producing GFP signal above background were isolated by fluorescence activated cell sorting (FACS). Approximately 2×10<sup>5</sup> cells were sorted, and 6,100 cells producing detectable GFP signal were isolated and grown on agar medium.</p>
<a id="article1.body1.sec3.sec1.p3" name="article1.body1.sec3.sec1.p3"></a><p>To identify clones with localized fluorescence, cells from each colony were grown to exponential phase in liquid medium and observed using epifluorescence microscopy. The majority of the clones produced an even distribution of cytoplasmic fluorescence, similar to <em>C. crescentus</em> producing GFP with no fusion (not shown). However, over 1,000 clones with localized GFP fluorescence were recovered. Twenty-four clones with representative localization patterns were chosen for further investigation, and the transposon insertion site was determined by either arbitrary PCR or inverse PCR, followed by DNA sequencing. The identity of each GFP fusion protein was determined by conceptual translation of the DNA sequence. Of the sequenced clones, seven independent insertions were recovered in each of two genes, <em>rsaA</em> and <em>serA</em> and one insertion was recovered in each of ten different genes (<a href="#pone-0001756-t001">Table 1</a>).</p>
<div class="figure" id="pone-0001756-t001"><div class="img"><a name="pone-0001756-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001756.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001756" data-uri="info:doi/10.1371/journal.pone.0001756.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001756.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001756.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001756.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001756.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001756.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001756.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001756.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001756.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001756.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>Localized GFP fusion proteins.</span></strong></p><span>doi:10.1371/journal.pone.0001756.t001</span></div><a id="article1.body1.sec3.sec1.p4" name="article1.body1.sec3.sec1.p4"></a><p>The transposon screen identified five proteins predicted to be localized based on similarity to proteins from other species, or on bioinformatic algorithms. CheR, a protein methyl transferase component of the chemosensory system, was found in a single focus at either the pole (47%) or mid-cell (40%) (<a href="#pone-0001756-g001">Fig. 1A</a>). Although CheR has not been studied in <em>C. crescentus</em>, it is localized in a focus near the cell pole in other bacteria <a href="#pone.0001756-Shiomi1">[22]</a>, <a href="#pone.0001756-Wu1">[23]</a>, and other chemosensory proteins are localized to the flagellar pole in <em>C. crescentus</em> <a href="#pone.0001756-Alley1">[24]</a>. The transposon insertion in <em>cheR</em> resulted in fusion of the first 267 residues of the 293 residue protein to GFP. To ensure that the observed localization of CheR was not due to truncation of the protein, the full-length <em>cheR</em> gene was cloned as a fusion with <em>gfp</em> and expressed from a vanillate-inducible promoter. The full-length CheR-GFP localized with the same pattern observed in the transposon-generated mutant (<a href="#pone-0001756-g001">Fig. 1B</a>). SerA and DnaK fusions also have localization patterns in <em>C. crescentus</em> similar to those observed in <em>E. coli</em> <a href="#pone.0001756-Kitagawa1">[9]</a>. SerA, which catalyzes the first reaction dedicated to serine biosynthesis, was localized to a focus that was frequently at the stalked pole or near mid-cell. DnaK, a protein chaperone, was found concentrated at both cell poles (<a href="#pone-0001756-t001">Table 1</a>). Two proteins, CoxA and CC2362, were found around the periphery of the cell. CoxA (cytochrome <em>c</em> oxidase) is an integral membrane protein, and is localized to the cell periphery in <em>E. coli</em> <a href="#pone.0001756-Kitagawa1">[9]</a>. CC2362 is predicted to be a peripheral membrane protein (22), and the transposon-generated fusion confirms this prediction. These data indicate that the transposon mutagenesis strategy is capable of identifying proteins expected to be localized.</p>
<div class="figure" id="pone-0001756-g001"><div class="img"><a name="pone-0001756-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001756.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001756" data-uri="info:doi/10.1371/journal.pone.0001756.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001756.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001756.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001756.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001756.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001756.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001756.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001756.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001756.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001756.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Localization of CheR.</span></strong></p><a id="article1.body1.sec3.sec1.fig1.caption1.p1" name="article1.body1.sec3.sec1.fig1.caption1.p1"></a><p><em>C. crescentus</em> producing CheR(1–267)-GFP generated by insertion of mini-Tn5-GFP (A) or full-length CheR-GFP from a low copy-number plasmid (B) were imaged by epifluorescence (left panels) and differential interference contrast (DIC) microscopy (center panels). Schematic diagrams (right panels) indicate location of fluorescence signal within the cell. Quantification of images like those shown in panel A indicated 47% of cells had a focus at the pole and 40% had a focus at mid-cell (n = 100). Scale bars indicating 1 µm are shown.</p>
<span>doi:10.1371/journal.pone.0001756.g001</span></div><a id="article1.body1.sec3.sec1.p5" name="article1.body1.sec3.sec1.p5"></a><p>Seven independent fusions to RsaA, the S-layer protein, were localized to a single focus at the cell pole (<a href="#pone-0001756-t001">Table 1</a>). These results were unexpected, because RsaA is secreted by a dedicated type I transporter, and the mature form of the protein is extracellular <a href="#pone.0001756-Awram1">[25]</a>. However, the polar localization of C-terminal GFP fusions was not inconsistent with previous studies of RsaA transport. C-terminal truncations of RsaA are not exported because the extreme C terminus contains the export signal, but it has been proposed that the N-terminal region of the protein contains information to target RsaA to the transporter sites <a href="#pone.0001756-Bingle1">[26]</a>. The localization of RsaA-GFP to the cell poles may indicate that the transporter complex is located at the cell poles.</p>
<a id="article1.body1.sec3.sec1.p6" name="article1.body1.sec3.sec1.p6"></a><p>Two of the localized GFP fusions were to proteins with known functions but for which no intracellular localization had been reported. AhpC, a subunit of alkyl hydroperoxide reductase, was localized to a single focus at the stalked pole (70%) or mid-cell (30%) when fused to GFP (<a href="#pone-0001756-t001">Table 1</a>, <a href="#pone-0001756-g002">Fig. 2A</a>). AhpC is involved in oxidative stress response, and there is no obvious reason why it would be localized to a subdomain of the cytoplasm. To confirm the epifluorescence results and ensure that truncation of the protein was not responsible for the localization pattern, immunofluorescence microscopy was used to observe the localization of full-length AhpC with an M2 epitope at the C terminus. Immunofluorescence images also showed AhpC in a focus at the stalked pole or mid-cell (<a href="#pone-0001756-g002">Fig. 2B</a>). The second protein, alanine dehydrogenase, was localized in a focus near mid-cell (<a href="#pone-0001756-t001">Table 1</a>). As for AhpC, there is no clear reason why alanine dehydrogenase would be localized. These data indicate that some proteins that do not appear to function in localized reactions or processes are nevertheless concentrated at specific locations in the cell.</p>
<div class="figure" id="pone-0001756-g002"><div class="img"><a name="pone-0001756-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001756.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001756" data-uri="info:doi/10.1371/journal.pone.0001756.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001756.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001756.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001756.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001756.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001756.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001756.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001756.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001756.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001756.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Localization of AhpC.</span></strong></p><a id="article1.body1.sec3.sec1.fig2.caption1.p1" name="article1.body1.sec3.sec1.fig2.caption1.p1"></a><p>(A) <em>C. crescentus</em> producing AhpC(1–36)-GFP generated by insertion of mini-Tn5-GFP were imaged by epifluorescence (left panel) and differential interference contrast (DIC) microscopy (center panel). Schematic diagram (right panel) indicates location of fluorescence signal within the cell. Quantification of images like those shown in panel A indicated 70% of cells had a focus at the stalked pole and 30% had a focus at mid-cell (n = 129). (B) <em>C. crescentus</em> producing full-length AhpC fused to M2 were imaged by immunofluorescence microscopy using anti-FLAG M2-Cy3 antibody (fuchsia) and DAPI (blue). Scale bars indicating 1 µm are shown.</p>
<span>doi:10.1371/journal.pone.0001756.g002</span></div><a id="article1.body1.sec3.sec1.p7" name="article1.body1.sec3.sec1.p7"></a><p>Four localized GFP fusion proteins were annotated as hypothetical or conserved hypothetical proteins, in addition to CC2362. CC3691-GFP was located in two foci, one at the stalked pole and one at mid-cell in 94% of cells, even though the fusion contains only 28 residues of CC3691 (<a href="#pone-0001756-t001">Table 1</a>, <a href="#pone-0001756-g003">Fig. 3A</a>). Immunofluorescence microscopy of cells producing a full-length CC3691 with an M2 epitope at the C terminus also showed two foci in 98% of cells (<a href="#pone-0001756-g003">Fig. 3B</a>), confirming the localization pattern and indicating that the information directing protein localization is contained within the N-terminal 28 residues. Fusions to CC3385 and CC2233 were localized to the periphery of the cell, indicating that these proteins are associated with the membrane, even though bioinformatic predictions did not suggest membrane interactions. The localization patterns of both CC0572 and CC2233 appeared to depend on the cell type, so these proteins were investigated in more detail.</p>
<div class="figure" id="pone-0001756-g003"><div class="img"><a name="pone-0001756-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001756.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001756" data-uri="info:doi/10.1371/journal.pone.0001756.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001756.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001756.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001756.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001756.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001756.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001756.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001756.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001756.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001756.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Localization of CC3691.</span></strong></p><a id="article1.body1.sec3.sec1.fig3.caption1.p1" name="article1.body1.sec3.sec1.fig3.caption1.p1"></a><p>(A) <em>C. crescentus</em> producing CC3691(1–28)-GFP generated by insertion of mini-Tn5-GFP were imaged by epifluorescence (left panel) and differential interference contrast (DIC) microscopy (center panel). Schematic diagram (right panel) indicates location of fluorescence signal within the cell. Quantification of images like those shown in panel A indicated 94% of cells had one focus at the stalked pole and one focus at mid-cell (n = 143). (B) <em>C. crescentus</em> producing full-length CC3691 fused to M2 were imaged by immunofluorescence microscopy using anti-FLAG M2-Cy3 antibody (fuchsia) and DAPI (blue). Quantification of images like those shown in panel B indicated 98% of cells had one focus at the stalked pole and one focus at mid-cell (n = 80). Scale bars indicating 1 µm are shown.</p>
<span>doi:10.1371/journal.pone.0001756.g003</span></div>

<h4>CC0572 is localized to a cell cycle-dependent helix-like structure</h4>
<a id="article1.body1.sec3.sec2.p1" name="article1.body1.sec3.sec2.p1"></a><p>Epifluorescence images of CC0572 fused to GFP revealed a regular array of spots and bands in over 50% of cells (<a href="#pone-0001756-g004">Fig. 4A</a>). Arrays of spots or bands have been seen in fluorescence images of proteins that form a helix within the cell <a href="#pone.0001756-Shih1">[8]</a>, <a href="#pone.0001756-Campo1">[27]</a>–<a href="#pone.0001756-Taghbalout1">[31]</a>. To confirm the localization pattern of CC0572, immunofluorescence images were obtained from cells producing a fusion of the full-length CC0572 protein to M2 (<a href="#pone-0001756-g004">Fig. 4B</a>). As seen for other helical filaments (13, 29), CC0572-M2 produced a pattern of connected bands. To determine if CC0572 was localized in a helix or other three-dimensional structure, optical sections were obtained using confocal microscopy. When these sections were computationally reconstituted to produce a three-dimensional model of the fluorescence signal, a helix-like pattern of CC0572-GFP that extended the length of the cell was apparent (<a href="#pone-0001756-g005">Fig. 5</a>).</p>
<div class="figure" id="pone-0001756-g004"><div class="img"><a name="pone-0001756-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001756.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001756" data-uri="info:doi/10.1371/journal.pone.0001756.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001756.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001756.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001756.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001756.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001756.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001756.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001756.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001756.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001756.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>Localization of CC0572.</span></strong></p><a id="article1.body1.sec3.sec2.fig1.caption1.p1" name="article1.body1.sec3.sec2.fig1.caption1.p1"></a><p>(A) <em>C. crescentus</em> producing CC0572(1–314)-GFP generated by insertion of mini-Tn5-GFP were imaged by epifluorescence (left panel) and differential interference contrast (DIC) microscopy (center panel). Schematic diagram (right panel) indicates location of fluorescence signal within the cell. (B) <em>C. crescentus</em> producing full-length CC0572 fused to M2 were imaged by immunofluorescence using anti-FLAG antibody and Alexa Fluor 488 anti-mouse antibody (green). Scale bars indicating 1 µm are shown.</p>
<span>doi:10.1371/journal.pone.0001756.g004</span></div><div class="figure" id="pone-0001756-g005"><div class="img"><a name="pone-0001756-g005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001756.g005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001756" data-uri="info:doi/10.1371/journal.pone.0001756.g005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001756.g005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001756.g005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001756.g005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001756.g005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001756.g005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001756.g005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001756.g005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001756.g005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001756.g005.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 5.  <span>CC0572 is localized to a helix-like structure.</span></strong></p><a id="article1.body1.sec3.sec2.fig2.caption1.p1" name="article1.body1.sec3.sec2.fig2.caption1.p1"></a><p>(A) Confocal microscopy was used to acquire a Z-series of fluorescence images of <em>C. crescentus</em> producing CC0572(1–314)-GFP. Schematic diagrams indicate the relative position of the optical section. (B) Images from the Z-series were computationally reconstituted to produce a composite image.</p>
<span>doi:10.1371/journal.pone.0001756.g005</span></div><a id="article1.body1.sec3.sec2.p2" name="article1.body1.sec3.sec2.p2"></a><p>For cells producing CC0572-GFP in exponentially growing cultures, which contained all cell types, only stalked cells clearly had the helix-like fluorescence pattern. To determine if the pattern changed through the cell cycle, swarmer cells producing CC0572-GFP were isolated and the fluorescence was observed as the cells passed synchronously through the cell cycle (<a href="#pone-0001756-g006">Fig. 6</a>). Swarmer cells (0 min) frequently had one bright spot of fluorescence at the cell pole. As the cells differentiated into stalked cells, this spot was lost and the helix-like fluorescence pattern could be seen (30–75 min). In late pre-divisional cells (90 min), the helix-like pattern was maintained, and after cell division, swarmer cells had a single focus of fluorescence and stalked cells had a helix-like fluorescence pattern. Thus, the localization of CC0572 changed as a function of the cell cycle. Western blots showed that the amount of CC0572 protein in the cell changed by less than 20% during the cell cycle (not shown), so the reorganization of the fluorescence signal is unlikely to be due to changes in protein concentration.</p>
<div class="figure" id="pone-0001756-g006"><div class="img"><a name="pone-0001756-g006" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001756.g006&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001756" data-uri="info:doi/10.1371/journal.pone.0001756.g006"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001756.g006&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001756.g006/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001756.g006/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001756.g006/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001756.g006/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001756.g006.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001756.g006/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001756.g006/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001756.g006.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 6.  <span>CC0572 localization changes during the cell cycle.</span></strong></p><a id="article1.body1.sec3.sec2.fig3.caption1.p1" name="article1.body1.sec3.sec2.fig3.caption1.p1"></a><p>Swarmer cells were isolated from cultures of <em>C. crescentus</em> producing CC0572(1–314)-GFP and the cells were imaged by epifluorescence (left panels) and DIC microscopy (right panels) during synchronous growth in PYE medium. The time after synchronization is indicated. Schematic diagram indicates the stage of the cell cycle at each time point. Quantification of images like those shown indicated that 43% of cells had a focus at the cell pole at 0 min (n = 105); from 30–75 min a helix-like pattern was seen in 61% of cells (n = 176); and after 90 min, 47% of septating cells retained the helix-like pattern (n = 110). Scale bars indicating 1 µm are shown.</p>
<span>doi:10.1371/journal.pone.0001756.g006</span></div><a id="article1.body1.sec3.sec2.p3" name="article1.body1.sec3.sec2.p3"></a><p>Cell-cycle regulated helical localization patterns have been reported for the actin-like protein MreB in <em>C. crescentus</em> <a href="#pone.0001756-Gitai1">[28]</a>. To determine if CC0572 required MreB for localization, the MreB structure was disrupted using the inhibitor A22 <a href="#pone.0001756-Gitai2">[32]</a>. In cells treated with A22, fluorescence from a GFP-MreB fusion was rapidly delocalized (<a href="#pone-0001756-g007">Fig. 7</a>). The helical pattern produced by CC0572-GFP was unaffected under similar conditions (<a href="#pone-0001756-g007">Fig. 7</a>), indicating that localization is independent of the MreB structure.</p>
<div class="figure" id="pone-0001756-g007"><div class="img"><a name="pone-0001756-g007" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001756.g007&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001756" data-uri="info:doi/10.1371/journal.pone.0001756.g007"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001756.g007&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001756.g007/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001756.g007/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001756.g007/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001756.g007/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001756.g007.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001756.g007/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001756.g007/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001756.g007.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 7.  <span>Localization of CC0572 does not depend on the MreB helix.</span></strong></p><a id="article1.body1.sec3.sec2.fig4.caption1.p1" name="article1.body1.sec3.sec2.fig4.caption1.p1"></a><p><em>C. crescentus</em> producing CC0572(1–314)-GFP or GFP-MreB were incubated without A22 (top) or with A22 (bottom) for 30 min and imaged by epifluorescence (left panels) and DIC microscopy (right panels). Under these conditions, the GFP-MreB helix was completely disrupted, but the CC0572(1–314)-GFP helix was still present. Scale bars indicating 1 µm are shown.</p>
<span>doi:10.1371/journal.pone.0001756.g007</span></div><a id="article1.body1.sec3.sec2.p4" name="article1.body1.sec3.sec2.p4"></a><p>Localization to a cell-cycle dependent structure suggests a role for CC0572 in a cell-cycle regulated process, but no function for this protein is known. CC0572 has some sequence similarity to chitinases, suggesting that it might be involved in exopolysaccharide metabolism. Consistent with this hypothesis, cells lacking CC0572 could not be infected with phage ΦCr30, which uses the S-layer protein attached to the exopolysaccharide as a receptor <a href="#pone.0001756-Edwards1">[33]</a>. To determine if CC0572 was required for normal cell structure, a deletion of the CC0572 gene was engineered and cell growth and morphology were assessed under a variety of nutrient and growth conditions. No defects were seen for these cells during lag phase, exponential growth, or stationary phase in complex medium or defined media. Likewise, no defects in colony morphology were seen for growth on solid media. These results indicate that although CC0572 is produced and localized during exponential growth in culture, it is not required for most processes under laboratory conditions. CC0572 may play an auxiliary role or provide a redundant enzymatic activity, so that absence of CC0572 is not deleterious. Alternatively, CC0572 may be necessary only under particular growth conditions that were not assayed here.</p>


<h4>CC2233 Moves to the Division Plane During the Cell Cycle</h4>
<a id="article1.body1.sec3.sec3.p1" name="article1.body1.sec3.sec3.p1"></a><p>The transposon insertion in <em>CC2233</em> resulted in a fusion protein containing only 62 amino acids of CC2233, which was localized to the periphery of the cells in 100% of cells (<a href="#pone-0001756-g008">Fig. 8</a>). Bioinformatic predictions did not suggest that CC2233 is a membrane protein, so to ensure that the full-length protein was also localized, the full-length <em>CC2233</em> gene fused to <em>gfp</em> was cloned under the control of a vanillate-inducible promoter and expressed in <em>C. crescentus.</em> The fluorescence patterns in these cells were indistinguishable from those in the transposon-generated fusion (<a href="#pone-0001756-g009">Fig. 9</a>). Cells producing CC2233-GFP were synchronized and observed to determine if the localization pattern changed as a function of the cell cycle. In swarmer and stalked cells, the fluorescence signal was localized around the periphery of the cell, but in pre-divisional cells the majority of the fluorescence moved to the division plane (<a href="#pone-0001756-g009">Fig. 9</a>). This localization pattern suggests that CC2233 might play some role in the late processes of cell division. However, deletion of CC2233 did not result in any defects in cell division, morphology, or cell growth. It is possible that CC2233 is localized to the division plane through interactions with a cell division protein, but CC2233 itself is not important for cytokinesis. Alternatively, CC2233 may be involved in a non-essential process at the division plane. The function of both CC0572 and CC2233 remain to be established, but their localization patterns suggest that many proteins with complex and dynamic localization patterns remain to be identified.</p>
<div class="figure" id="pone-0001756-g008"><div class="img"><a name="pone-0001756-g008" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001756.g008&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001756" data-uri="info:doi/10.1371/journal.pone.0001756.g008"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001756.g008&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001756.g008/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001756.g008/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001756.g008/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001756.g008/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001756.g008.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001756.g008/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001756.g008/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001756.g008.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 8.  <span>Localization of CC2233.</span></strong></p><a id="article1.body1.sec3.sec3.fig1.caption1.p1" name="article1.body1.sec3.sec3.fig1.caption1.p1"></a><p>(A) <em>C. crescentus</em> producing CC2233(1–62)-GFP generated by insertion of mini-Tn5-GFP were imaged by epifluorescence (left panel) and differential interference contrast (DIC) microscopy (center panel). Schematic diagram (right panel) indicates location of fluorescence signal within the cell. Quantification of images as in (A) indicated 100% of cells had fluorescence localized to the periphery (n = 500). (B) <em>C. crescentus</em> producing full-length CC2233 fused to GFP were imaged by epifluorescence. Quantification of images as in (B) indicated 100% of cells had fluorescence localized to the periphery (n = 100). Scale bars indicating 1 µm are shown.</p>
<span>doi:10.1371/journal.pone.0001756.g008</span></div><div class="figure" id="pone-0001756-g009"><div class="img"><a name="pone-0001756-g009" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001756.g009&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001756" data-uri="info:doi/10.1371/journal.pone.0001756.g009"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001756.g009&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001756.g009/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001756.g009/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001756.g009/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001756.g009/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001756.g009.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001756.g009/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001756.g009/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001756.g009.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 9.  <span>Localization of CC2233 changes during the cell cycle.</span></strong></p><a id="article1.body1.sec3.sec3.fig2.caption1.p1" name="article1.body1.sec3.sec3.fig2.caption1.p1"></a><p>Swarmer cells were isolated from cultures of <em>C. crescentus</em> producing CC2233(1–62)-GFP and the cells were imaged by epifluorescence (left panels) and DIC microscopy (right panels) during synchronous growth in M2G medium. The time after synchronization is shown (cell cycle progression is slower than in <a href="#pone-0001756-g004">Figure 4</a> because cells were grown in minimal medium). Schematic diagram indicates the cell cycle stage at each time point. CC2233-GFP was located around the periphery until the predivisional stage (105–135 min), when it was concentrated at the division plane (arrowheads). Scale bars indicating 1 µm are shown.</p>
<span>doi:10.1371/journal.pone.0001756.g009</span></div>
</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec4.p1" name="article1.body1.sec4.p1"></a><p>The transposon-based strategy employed here successfully identified localized proteins in <em>C. crescentus</em>, and could be adapted for any species that is compatible with transposon mutagenesis. The first sequencing efforts identified eleven previously unknown localized proteins out of 24 clones that were sequenced. Because over 1000 clones with localized GFP signal were obtained, there are likely to be many more localized proteins in <em>C. crescentus</em> than have been described to date. This point is accentuated by two groups of localized proteins, those involved in processes for which there is no obvious requirement for localized function, such as metabolism and stress response, and hypothetical proteins. SerA and alanine dehydrogenase are each localized, even though the substrates and products of amino acid metabolism can presumably diffuse rapidly throughout the cell. SerA is also localized in <em>E. coli</em> <a href="#pone.0001756-Kitagawa1">[9]</a>, so it is unlikely that this phenomenon is peculiar to <em>C. crescentus</em>. One possible rationale for localizing biosynthetic enzymes is to limit diffusion of unstable or harmful reaction intermediates. Six of the localized proteins were annotated as hypothetical, a category that comprises 37% of the <em>C. crescentus</em> proteome <a href="#pone.0001756-Nierman1">[34]</a>. The recovery of these hypothetical proteins at a high frequency from the screen indicates that localization may not be rare. In particular, the identification of two hypothetical proteins with cell-cycle dependent localization patterns, one of which is localized to a helix-like structure, indicate that dynamic localization and complex three-dimensional structures may be common in <em>C. crescentus</em>.</p>
<a id="article1.body1.sec4.p2" name="article1.body1.sec4.p2"></a><p>For two proteins, the mini-Tn5-GFP screen identified a short peptide that must contain sufficient information to target the protein to its location. The AhpC-GFP fusion contained only 36 residues of AhpC, and CC3691-GFP contained 28 residues of CC3691, yet each of these fusions was localized in the same pattern as the full-length proteins. It has been demonstrated that short peptides, or locons, can target an exogenous protein to a precise location in a bacterial cell <a href="#pone.0001756-Russell1">[7]</a>. These data indicate that endogenous proteins use locon signals as well.</p>
<a id="article1.body1.sec4.p3" name="article1.body1.sec4.p3"></a><p>In principle, proteins in aggregates or inclusion bodies could appear to be localized in one or more foci, and it is possible that some of the clones isolated from the screen have localized GFP signal due to aggregation or inclusion body formation. However, potential aggregation or inclusion body formation due to protein over-production was limited for the screen used here, because proteins were produced from their wild-type promoter at the normal chromosomal locus. In addition, the foci of CheR, AhpC, and CC3691 were not caused by aggregation of truncated GFP-fusion proteins, or by dimerization or aggregation of the GFP sequence, because identical patterns were seen with full-length versions of the proteins fused to the M2 epitope (<a href="#pone-0001756-g001">Figs. 1</a>, <a href="#pone-0001756-g002">2</a>, and <a href="#pone-0001756-g003">3</a>).</p>
<a id="article1.body1.sec4.p4" name="article1.body1.sec4.p4"></a><p>A mutagenesis-based screen such as the one used here is significantly faster and less expensive than constructing fusions for each gene, and because the same localization patterns were observed for fusions of full-length versions of five of the proteins to M2 or GFP, the results for many proteins would be identical. However, the mutagenesis approach will clearly miss some localized proteins. Because Tn5 clearly has some sequence bias for insertion, the recovery of localized proteins could be expanded by using additional transposons, such as <em>mariner</em> and <em>mu,</em> engineered to insert <em>gfp</em>. Proteins that are essential and cannot tolerate a C-terminal fusion, and proteins that require a free C terminus for localization will not be recovered using transposon mutagenesis. Polar effects caused by transposon insertion may also prevent recovery of some localized proteins. Nevertheless, many new localized proteins can be found with this technique.</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>Thanks to Elaine Kunze and Nicole Bem at the Center for Quantitative Cell Analysis at the Huck Institutes of the Life Sciences, Penn State University, for assistance with confocal microscopy and visualization software, and to Patrick Viollier for providing A22 and the strain producing GFP-MreB.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: KK JR. Performed the experiments: KK JR. Analyzed the data: KK JR. Wrote the paper: KK JR.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0001756-Shapiro1" id="pone.0001756-Shapiro1"></a>Shapiro L, McAdams HH, Losick R (2002) Generating and exploiting polarity in bacteria. Science  298: 1942–1946.  <ul class="find" data-citedArticleID="1070508" data-doi="10.1126/science.1072163"><li><a href="http://dx.doi.org/10.1126/science.1072163" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Generating+and+exploiting+polarity+in+bacteria." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Generating+and+exploiting+polarity+in+bacteria.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0001756-Ryan1" id="pone.0001756-Ryan1"></a>Ryan KR, Shapiro L (2003) Temporal and spatial regulation in prokaryotic cell cycle progression and development. Annu Rev Biochem  72: 367–394.  <ul class="find" data-citedArticleID="1070506"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Temporal+and+spatial+regulation+in+prokaryotic+cell+cycle+progression+and+development.&amp;auth=&amp;atitle=Temporal+and+spatial+regulation+in+prokaryotic+cell+cycle+progression+and+development." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Temporal+and+spatial+regulation+in+prokaryotic+cell+cycle+progression+and+development." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Temporal+and+spatial+regulation+in+prokaryotic+cell+cycle+progression+and+development.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0001756-JacobsWagner1" id="pone.0001756-JacobsWagner1"></a>Jacobs-Wagner C (2004) Regulatory proteins with a sense of direction: cell cycle signalling network in <em>Caulobacter</em>. Mol Microbiol  51: 7–13.  <ul class="find" data-citedArticleID="1070488" data-doi="10.1046/j.1365-2958.2003.03828.x"><li><a href="http://dx.doi.org/10.1046/j.1365-2958.2003.03828.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Regulatory+proteins+with+a+sense+of+direction%3A+cell+cycle+signalling+network+in+Caulobacter." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Regulatory+proteins+with+a+sense+of+direction%3A+cell+cycle+signalling+network+in+Caulobacter.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0001756-McAdams1" id="pone.0001756-McAdams1"></a>McAdams HH, Shapiro L (2003) A bacterial cell-cycle regulatory network operating in time and space. Science  301: 1874–1877.  <ul class="find" data-citedArticleID="1070496" data-doi="10.1126/science.1087694"><li><a href="http://dx.doi.org/10.1126/science.1087694" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+bacterial+cell-cycle+regulatory+network+operating+in+time+and+space." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+bacterial+cell-cycle+regulatory+network+operating+in+time+and+space.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0001756-Errington1" id="pone.0001756-Errington1"></a>Errington J, Daniel RA, Scheffers DJ (2003) Cytokinesis in bacteria. Microbiol Mol Biol Rev  67: 52–65.  <ul class="find" data-citedArticleID="1070476" data-doi="10.1128/mmbr.67.1.52-65.2003"><li><a href="http://dx.doi.org/10.1128/mmbr.67.1.52-65.2003" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Cytokinesis+in+bacteria." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Cytokinesis+in+bacteria.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0001756-Maddock1" id="pone.0001756-Maddock1"></a>Maddock JR, Shapiro L (1993) Polar Location of the chemoreceptor complex in the <em>Escherichia coli</em> cell. Science  259: 1717–1723.  <ul class="find" data-citedArticleID="1070494" data-doi="10.1126/science.8456299"><li><a href="http://dx.doi.org/10.1126/science.8456299" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Polar+Location+of+the+chemoreceptor+complex+in+the+Escherichia+coli+cell." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Polar+Location+of+the+chemoreceptor+complex+in+the+Escherichia+coli+cell.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0001756-Russell1" id="pone.0001756-Russell1"></a>Russell JH, Keiler KC (2007) Peptide signals encode protein localization. J Bacteriol  189: 7581–7585.  <ul class="find" data-citedArticleID="1070504" data-doi="10.1128/jb.00981-07"><li><a href="http://dx.doi.org/10.1128/jb.00981-07" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Peptide+signals+encode+protein+localization." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Peptide+signals+encode+protein+localization.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0001756-Shih1" id="pone.0001756-Shih1"></a>Shih YL, Rothfield L (2006) The bacterial cytoskeleton. Microbiol Mol Biol Rev  70: 729–754.  <ul class="find" data-citedArticleID="1070510" data-doi="10.1128/mmbr.00017-06"><li><a href="http://dx.doi.org/10.1128/mmbr.00017-06" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+bacterial+cytoskeleton." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+bacterial+cytoskeleton.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0001756-Kitagawa1" id="pone.0001756-Kitagawa1"></a>Kitagawa M, Ara T, Arifuzzaman M, Ioka-Nakamichi T, Inamoto E, et al.  (2005) Complete set of ORF clones of Escherichia coli ASKA library (a complete set of <em>E</em>. <em>coli</em> K-12 ORF archive): unique resources for biological research. DNA Res  12: 291–299.  <ul class="find" data-citedArticleID="1070492" data-doi="10.1093/dnares/dsi012"><li><a href="http://dx.doi.org/10.1093/dnares/dsi012" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Complete+set+of+ORF+clones+of+Escherichia+coli+ASKA+library+%28a+complete+set+of+E.+coli+K-12+ORF+archive%29%3A+unique+resources+for+biological+research." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Complete+set+of+ORF+clones+of+Escherichia+coli+ASKA+library+%28a+complete+set+of+E.+coli+K-12+ORF+archive%29%3A+unique+resources+for+biological+research.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0001756-Evinger1" id="pone.0001756-Evinger1"></a>Evinger M, Agabian N (1977) Envelope-associated nucleoid from <em>Caulobacter</em> crescentus stalked and swarmer cells. J Bacteriol  132: 294–301.  <ul class="find" data-citedArticleID="1070478"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Envelope-associated+nucleoid+from+Caulobacter+crescentus+stalked+and+swarmer+cells.&amp;auth=&amp;atitle=Envelope-associated+nucleoid+from+Caulobacter+crescentus+stalked+and+swarmer+cells." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Envelope-associated+nucleoid+from+Caulobacter+crescentus+stalked+and+swarmer+cells." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Envelope-associated+nucleoid+from+Caulobacter+crescentus+stalked+and+swarmer+cells.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0001756-Ausmees1" id="pone.0001756-Ausmees1"></a>Ausmees N, Jacobs-Wagner C (2003) Spatial and temporal control of differentiation and cell cycle progression in <em>Caulobacter crescentus</em>. Annu Rev Microbiol  57: 225–247.  <ul class="find" data-citedArticleID="1070464"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Spatial+and+temporal+control+of+differentiation+and+cell+cycle+progression+in+Caulobacter+crescentus.&amp;auth=&amp;atitle=Spatial+and+temporal+control+of+differentiation+and+cell+cycle+progression+in+Caulobacter+crescentus." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Spatial+and+temporal+control+of+differentiation+and+cell+cycle+progression+in+Caulobacter+crescentus." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Spatial+and+temporal+control+of+differentiation+and+cell+cycle+progression+in+Caulobacter+crescentus.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0001756-Ely1" id="pone.0001756-Ely1"></a>Ely B (1991) Genetics of <em>Caulobacter crescentus</em>. Methods Enzymol  204: 372–384.  <ul class="find" data-citedArticleID="1070474" data-doi="10.1016/0076-6879(91)04019-k"><li><a href="http://dx.doi.org/10.1016/0076-6879(91)04019-k" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genetics+of+Caulobacter+crescentus." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genetics+of+Caulobacter+crescentus.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0001756-Gay1" id="pone.0001756-Gay1"></a>Gay P, Le Coq D, Steinmetz M, Berkelman T, Kado CI (1985) Positive selection procedure for entrapment of insertion sequence elements in gram-negative bacteria. J Bacteriol  164: 918–921.  <ul class="find" data-citedArticleID="1070480"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Positive+selection+procedure+for+entrapment+of+insertion+sequence+elements+in+gram-negative+bacteria.&amp;auth=&amp;atitle=Positive+selection+procedure+for+entrapment+of+insertion+sequence+elements+in+gram-negative+bacteria." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Positive+selection+procedure+for+entrapment+of+insertion+sequence+elements+in+gram-negative+bacteria." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Positive+selection+procedure+for+entrapment+of+insertion+sequence+elements+in+gram-negative+bacteria.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0001756-Keiler1" id="pone.0001756-Keiler1"></a>Keiler KC, Shapiro L (2003) tmRNA is required for correct timing of DNA replication in <em>Caulobacter crescentus</em>. J Bacteriol  185: 573–580.  <ul class="find" data-citedArticleID="1070490" data-doi="10.1128/jb.185.2.573-580.2003"><li><a href="http://dx.doi.org/10.1128/jb.185.2.573-580.2003" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=tmRNA+is+required+for+correct+timing+of+DNA+replication+in+Caulobacter+crescentus." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22tmRNA+is+required+for+correct+timing+of+DNA+replication+in+Caulobacter+crescentus.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0001756-deLorenzo1" id="pone.0001756-deLorenzo1"></a>de Lorenzo V, Herrero M, Jakubzik U, Timmis KN (1990) Mini-Tn5 transposon derivatives for insertion mutagenesis, promoter probing, and chromosomal insertion of cloned DNA in gram-negative eubacteria. J Bacteriol  172: 6568–6572.  <ul class="find" data-citedArticleID="1070526"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Mini-Tn5+transposon+derivatives+for+insertion+mutagenesis%2C+promoter+probing%2C+and+chromosomal+insertion+of+cloned+DNA+in+gram-negative+eubacteria.&amp;auth=&amp;atitle=Mini-Tn5+transposon+derivatives+for+insertion+mutagenesis%2C+promoter+probing%2C+and+chromosomal+insertion+of+cloned+DNA+in+gram-negative+eubacteria." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mini-Tn5+transposon+derivatives+for+insertion+mutagenesis%2C+promoter+probing%2C+and+chromosomal+insertion+of+cloned+DNA+in+gram-negative+eubacteria." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mini-Tn5+transposon+derivatives+for+insertion+mutagenesis%2C+promoter+probing%2C+and+chromosomal+insertion+of+cloned+DNA+in+gram-negative+eubacteria.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0001756-Hopp1" id="pone.0001756-Hopp1"></a>Hopp TP, Prickett KS, Price VL, Libby RT, March CJ, et al.  (1988) A short polypeptide marker sequence useful for recombinant protein identification and purification. 6: 1204–1210.  <ul class="find" data-citedArticleID="1070486" data-doi="10.1038/nbt1088-1204"><li><a href="http://dx.doi.org/10.1038/nbt1088-1204" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+short+polypeptide+marker+sequence+useful+for+recombinant+protein+identification+and+purification." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+short+polypeptide+marker+sequence+useful+for+recombinant+protein+identification+and+purification.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0001756-Meisenzahl1" id="pone.0001756-Meisenzahl1"></a>Meisenzahl AC, Shapiro L, Jenal U (1997) Isolation and characterization of a xylose-dependent promoter from <em>Caulobacter crescentus</em>. J Bacteriol  179: 592–600.  <ul class="find" data-citedArticleID="1070498"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Isolation+and+characterization+of+a+xylose-dependent+promoter+from+Caulobacter+crescentus.&amp;auth=&amp;atitle=Isolation+and+characterization+of+a+xylose-dependent+promoter+from+Caulobacter+crescentus." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Isolation+and+characterization+of+a+xylose-dependent+promoter+from+Caulobacter+crescentus." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Isolation+and+characterization+of+a+xylose-dependent+promoter+from+Caulobacter+crescentus.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0001756-Thanbichler1" id="pone.0001756-Thanbichler1"></a>Thanbichler M, Iniesta AA, Shapiro L (2007) A comprehensive set of plasmids for vanillate- and xylose-inducible gene expression in <em>Caulobacter crescentus</em>. Nucleic Acids Res  35: e137.  <ul class="find" data-citedArticleID="1070518" data-doi="10.1093/nar/gkm818"><li><a href="http://dx.doi.org/10.1093/nar/gkm818" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+comprehensive+set+of+plasmids+for+vanillate-+and+xylose-inducible+gene+expression+in+Caulobacter+crescentus." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+comprehensive+set+of+plasmids+for+vanillate-+and+xylose-inducible+gene+expression+in+Caulobacter+crescentus.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0001756-Thanbichler2" id="pone.0001756-Thanbichler2"></a>Thanbichler M, Shapiro L (2006) MipZ, a spatial regulator coordinating chromosome segregation with cell division in <em>Caulobacter</em>. Cell  126: 147–162.  <ul class="find" data-citedArticleID="1070520" data-doi="10.1016/j.cell.2006.05.038"><li><a href="http://dx.doi.org/10.1016/j.cell.2006.05.038" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=MipZ%2C+a+spatial+regulator+coordinating+chromosome+segregation+with+cell+division+in+Caulobacter." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22MipZ%2C+a+spatial+regulator+coordinating+chromosome+segregation+with+cell+division+in+Caulobacter.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0001756-Welsh1" id="pone.0001756-Welsh1"></a>Welsh J, McClelland M (1990) Fingerprinting genomes using PCR with arbitrary primers. Nucleic Acids Res  18: 7213–7218.  <ul class="find" data-citedArticleID="1070522" data-doi="10.1093/nar/18.24.7213"><li><a href="http://dx.doi.org/10.1093/nar/18.24.7213" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Fingerprinting+genomes+using+PCR+with+arbitrary+primers." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Fingerprinting+genomes+using+PCR+with+arbitrary+primers.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0001756-Ochman1" id="pone.0001756-Ochman1"></a>Ochman H, Gerber AS, Hartl DL (1988) Genetic applications of an inverse polymerase chain reaction. Genetics  120: 621–623.  <ul class="find" data-citedArticleID="1070502"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Genetic+applications+of+an+inverse+polymerase+chain+reaction.&amp;auth=&amp;atitle=Genetic+applications+of+an+inverse+polymerase+chain+reaction." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genetic+applications+of+an+inverse+polymerase+chain+reaction." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genetic+applications+of+an+inverse+polymerase+chain+reaction.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0001756-Shiomi1" id="pone.0001756-Shiomi1"></a>Shiomi D, Banno S, Homma M, Kawagishi I (2005) Stabilization of polar localization of a chemoreceptor via its covalent modifications and its communication with a different chemoreceptor. J Bacteriol  187: 7647–7654.  <ul class="find" data-citedArticleID="1070514" data-doi="10.1128/jb.187.22.7647-7654.2005"><li><a href="http://dx.doi.org/10.1128/jb.187.22.7647-7654.2005" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Stabilization+of+polar+localization+of+a+chemoreceptor+via+its+covalent+modifications+and+its+communication+with+a+different+chemoreceptor." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Stabilization+of+polar+localization+of+a+chemoreceptor+via+its+covalent+modifications+and+its+communication+with+a+different+chemoreceptor.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0001756-Wu1" id="pone.0001756-Wu1"></a>Wu J, Li J, Li G, Long DG, Weis RM (1996) The receptor binding site for the methyltransferase of bacterial chemotaxis is distinct from the sites of methylation. Biochemistry  35: 4984–4993.  <ul class="find" data-citedArticleID="1070524" data-doi="10.1021/bi9530189"><li><a href="http://dx.doi.org/10.1021/bi9530189" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+receptor+binding+site+for+the+methyltransferase+of+bacterial+chemotaxis+is+distinct+from+the+sites+of+methylation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+receptor+binding+site+for+the+methyltransferase+of+bacterial+chemotaxis+is+distinct+from+the+sites+of+methylation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0001756-Alley1" id="pone.0001756-Alley1"></a>Alley MR, Maddock JR, Shapiro L (1992) Polar localization of a bacterial chemoreceptor. Genes Dev  6: 825–836.  <ul class="find" data-citedArticleID="1070462" data-doi="10.1101/gad.6.5.825"><li><a href="http://dx.doi.org/10.1101/gad.6.5.825" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Polar+localization+of+a+bacterial+chemoreceptor." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Polar+localization+of+a+bacterial+chemoreceptor.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0001756-Awram1" id="pone.0001756-Awram1"></a>Awram P, Smit J (1998) The <em>Caulobacter crescentus</em> paracrystalline S-layer protein is secreted by an ABC transporter (type I) secretion apparatus. J Bacteriol  180: 3062–3069.  <ul class="find" data-citedArticleID="1070466"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=The+Caulobacter+crescentus+paracrystalline+S-layer+protein+is+secreted+by+an+ABC+transporter+%28type+I%29+secretion+apparatus.&amp;auth=&amp;atitle=The+Caulobacter+crescentus+paracrystalline+S-layer+protein+is+secreted+by+an+ABC+transporter+%28type+I%29+secretion+apparatus." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+Caulobacter+crescentus+paracrystalline+S-layer+protein+is+secreted+by+an+ABC+transporter+%28type+I%29+secretion+apparatus." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+Caulobacter+crescentus+paracrystalline+S-layer+protein+is+secreted+by+an+ABC+transporter+%28type+I%29+secretion+apparatus.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0001756-Bingle1" id="pone.0001756-Bingle1"></a>Bingle WH, Nomellini JF, Smit J (2000) Secretion of the <em>Caulobacter crescentus</em> S-layer protein: further localization of the C-terminal secretion signal and its use for secretion of recombinant proteins. J Bacteriol  182: 3298–3301.  <ul class="find" data-citedArticleID="1070468" data-doi="10.1128/jb.182.11.3298-3301.2000"><li><a href="http://dx.doi.org/10.1128/jb.182.11.3298-3301.2000" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Secretion+of+the+Caulobacter+crescentus+S-layer+protein%3A+further+localization+of+the+C-terminal+secretion+signal+and+its+use+for+secretion+of+recombinant+proteins." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Secretion+of+the+Caulobacter+crescentus+S-layer+protein%3A+further+localization+of+the+C-terminal+secretion+signal+and+its+use+for+secretion+of+recombinant+proteins.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0001756-Campo1" id="pone.0001756-Campo1"></a>Campo N, Tjalsma H, Buist G, Stepniak D, Meijer M, et al.  (2004) Subcellular sites for bacterial protein export. Mol Microbiol  53: 1583–1599.  <ul class="find" data-citedArticleID="1070470" data-doi="10.1111/j.1365-2958.2004.04278.x"><li><a href="http://dx.doi.org/10.1111/j.1365-2958.2004.04278.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Subcellular+sites+for+bacterial+protein+export." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Subcellular+sites+for+bacterial+protein+export.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0001756-Gitai1" id="pone.0001756-Gitai1"></a>Gitai Z, Dye N, Shapiro L (2004) An actin-like gene can determine cell polarity in bacteria. Proc Natl Acad Sci U S A  101: 8643–8648.  <ul class="find" data-citedArticleID="1070482" data-doi="10.1073/pnas.0402638101"><li><a href="http://dx.doi.org/10.1073/pnas.0402638101" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=An+actin-like+gene+can+determine+cell+polarity+in+bacteria." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22An+actin-like+gene+can+determine+cell+polarity+in+bacteria.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0001756-Shih2" id="pone.0001756-Shih2"></a>Shih YL, Le T, Rothfield L (2003) Division site selection in <em>Escherichia coli</em> involves dynamic redistribution of Min proteins within coiled structures that extend between the two cell poles. Proc Natl Acad Sci U S A  100: 7865–7870.  <ul class="find" data-citedArticleID="1070512" data-doi="10.1073/pnas.1232225100"><li><a href="http://dx.doi.org/10.1073/pnas.1232225100" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Division+site+selection+in+Escherichia+coli+involves+dynamic+redistribution+of+Min+proteins+within+coiled+structures+that+extend+between+the+two+cell+poles." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Division+site+selection+in+Escherichia+coli+involves+dynamic+redistribution+of+Min+proteins+within+coiled+structures+that+extend+between+the+two+cell+poles.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0001756-Addinall1" id="pone.0001756-Addinall1"></a>Addinall SG, Lutkenhaus J (1996) FtsZ-spirals and -arcs determine the shape of the invaginating septa in some mutants of <em>Escherichia coli</em>. Mol Microbiol  22: 231–237.  <ul class="find" data-citedArticleID="1070460" data-doi="10.1046/j.1365-2958.1996.00100.x"><li><a href="http://dx.doi.org/10.1046/j.1365-2958.1996.00100.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=FtsZ-spirals+and+-arcs+determine+the+shape+of+the+invaginating+septa+in+some+mutants+of+Escherichia+coli." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22FtsZ-spirals+and+-arcs+determine+the+shape+of+the+invaginating+septa+in+some+mutants+of+Escherichia+coli.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0001756-Taghbalout1" id="pone.0001756-Taghbalout1"></a>Taghbalout A, Rothfield L (2007) RNaseE and the other constituents of the RNA degradosome are components of the bacterial cytoskeleton. Proc Natl Acad Sci U S A  104: 1667–1672.  <ul class="find" data-citedArticleID="1070516" data-doi="10.1073/pnas.0610491104"><li><a href="http://dx.doi.org/10.1073/pnas.0610491104" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=RNaseE+and+the+other+constituents+of+the+RNA+degradosome+are+components+of+the+bacterial+cytoskeleton." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22RNaseE+and+the+other+constituents+of+the+RNA+degradosome+are+components+of+the+bacterial+cytoskeleton.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0001756-Gitai2" id="pone.0001756-Gitai2"></a>Gitai Z, Dye NA, Reisenauer A, Wachi M, Shapiro L (2005) MreB actin-mediated segregation of a specific region of a bacterial chromosome. Cell  120: 329–341.  <ul class="find" data-citedArticleID="1070484" data-doi="10.1016/j.cell.2005.01.007"><li><a href="http://dx.doi.org/10.1016/j.cell.2005.01.007" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=MreB+actin-mediated+segregation+of+a+specific+region+of+a+bacterial+chromosome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22MreB+actin-mediated+segregation+of+a+specific+region+of+a+bacterial+chromosome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0001756-Edwards1" id="pone.0001756-Edwards1"></a>Edwards P, Smit J (1991) A Transducing Bacteriophage for <em>Caulobacter crescentus</em> Uses the Paracrystalline Surface Layer Protein as a Receptor. J Bacteriol  173: 5568–5572.  <ul class="find" data-citedArticleID="1070472"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=A+Transducing+Bacteriophage+for+Caulobacter+crescentus+Uses+the+Paracrystalline+Surface+Layer+Protein+as+a+Receptor.&amp;auth=&amp;atitle=A+Transducing+Bacteriophage+for+Caulobacter+crescentus+Uses+the+Paracrystalline+Surface+Layer+Protein+as+a+Receptor." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+Transducing+Bacteriophage+for+Caulobacter+crescentus+Uses+the+Paracrystalline+Surface+Layer+Protein+as+a+Receptor." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+Transducing+Bacteriophage+for+Caulobacter+crescentus+Uses+the+Paracrystalline+Surface+Layer+Protein+as+a+Receptor.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0001756-Nierman1" id="pone.0001756-Nierman1"></a>Nierman WC, Feldblyum TV, Laub MT, Paulsen IT, Nelson KE, et al.  (2001) Complete genome sequence of <em>Caulobacter crescentus</em>. Proc Natl Acad Sci U S A  98: 4136–4141.  <ul class="find" data-citedArticleID="1070500" data-doi="10.1073/pnas.061029298"><li><a href="http://dx.doi.org/10.1073/pnas.061029298" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Complete+genome+sequence+of+Caulobacter+crescentus." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Complete+genome+sequence+of+Caulobacter+crescentus.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001756.XML" value="73287"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001756.PDF" value="438494"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001756.t001.PNG_L" value="150711"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001756.t001.PNG_M" value="83571"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001756.t001.PNG_S" value="10079"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001756.t001.TIF" value="678852"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001756.t001.PNG_I" value="25428"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001756.g001.PNG_L" value="641515"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001756.g001.PNG_M" value="102766"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001756.g001.PNG_S" value="7328"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001756.g001.TIF" value="2086056"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001756.g001.PNG_I" value="34443"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001756.g002.PNG_L" value="687323"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001756.g002.PNG_M" value="66833"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001756.g002.PNG_S" value="5840"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001756.g002.TIF" value="1879662"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001756.g002.PNG_I" value="22434"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001756.g003.PNG_L" value="612365"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001756.g003.PNG_M" value="154787"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001756.g003.PNG_S" value="9820"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001756.g003.TIF" value="2368254"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001756.g003.PNG_I" value="54829"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001756.g004.PNG_L" value="610515"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001756.g004.PNG_M" value="148261"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001756.g004.PNG_S" value="11411"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001756.g004.TIF" value="1934356"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001756.g004.PNG_I" value="54349"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001756.g005.PNG_L" value="60108"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001756.g005.PNG_M" value="25577"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001756.g005.PNG_S" value="8278"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001756.g005.TIF" value="661456"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001756.g005.PNG_I" value="29442"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001756.g006.PNG_L" value="1294747"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001756.g006.PNG_M" value="149865"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001756.g006.PNG_S" value="13920"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001756.g006.TIF" value="3029114"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001756.g006.PNG_I" value="79395"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001756.g007.PNG_L" value="948167"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001756.g007.PNG_M" value="123397"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001756.g007.PNG_S" value="8118"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001756.g007.TIF" value="2204018"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001756.g007.PNG_I" value="41301"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001756.g008.PNG_L" value="569368"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001756.g008.PNG_M" value="104694"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001756.g008.PNG_S" value="10260"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001756.g008.TIF" value="1634154"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001756.g008.PNG_I" value="40375"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001756.g009.PNG_L" value="1539799"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001756.g009.PNG_M" value="165996"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001756.g009.PNG_S" value="11957"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001756.g009.TIF" value="3261398"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001756.g009.PNG_I" value="69492"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001756&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001756" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001756&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0001756&volume=&issue=&title=Screen for Localized Proteins in Caulobacter crescentus&author_name=Jay%20H.%20Russell%2C%20Kenneth%20C.%20Keiler&start_page=1&end_page=11" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001756" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001756" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0001756" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001756&amp;t=Screen%20for%20Localized%20Proteins%20in%20Caulobacter%20crescentus" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001756&title=Screen%20for%20Localized%20Proteins%20in%20Caulobacter%20crescentus&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001756&amp;title=Screen%20for%20Localized%20Proteins%20in%20Caulobacter%20crescentus" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001756" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0001756&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Screen for Localized Proteins in Caulobacter crescentus';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0001756';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Screen%20for%20Localized%20Proteins%20in%20Caulobacter%20crescentus http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001756" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0001756" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">











          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Caulobacter+crescentus%22" title="Search for articles in the subject area:'Caulobacter crescentus'"><div class="flagText">Caulobacter crescentus</div></a>
              <div data-categoryid="34361" data-articleid="26336"
                   data-categoryname="Caulobacter crescentus"
                   class="flagImage" title="Flag 'Caulobacter crescentus' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Cell+cycle+and+cell+division%22" title="Search for articles in the subject area:'Cell cycle and cell division'"><div class="flagText">Cell cycle and cell division</div></a>
              <div data-categoryid="17353" data-articleid="26336"
                   data-categoryname="Cell cycle and cell division"
                   class="flagImage" title="Flag 'Cell cycle and cell division' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Cell+fusion%22" title="Search for articles in the subject area:'Cell fusion'"><div class="flagText">Cell fusion</div></a>
              <div data-categoryid="17857" data-articleid="26336"
                   data-categoryname="Cell fusion"
                   class="flagImage" title="Flag 'Cell fusion' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Fluorescence+imaging%22" title="Search for articles in the subject area:'Fluorescence imaging'"><div class="flagText">Fluorescence imaging</div></a>
              <div data-categoryid="17303" data-articleid="26336"
                   data-categoryname="Fluorescence imaging"
                   class="flagImage" title="Flag 'Fluorescence imaging' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Fluorescence+microscopy%22" title="Search for articles in the subject area:'Fluorescence microscopy'"><div class="flagText">Fluorescence microscopy</div></a>
              <div data-categoryid="17505" data-articleid="26336"
                   data-categoryname="Fluorescence microscopy"
                   class="flagImage" title="Flag 'Fluorescence microscopy' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Interference+microscopy%22" title="Search for articles in the subject area:'Interference microscopy'"><div class="flagText">Interference microscopy</div></a>
              <div data-categoryid="21643" data-articleid="26336"
                   data-categoryname="Interference microscopy"
                   class="flagImage" title="Flag 'Interference microscopy' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Transposable+elements%22" title="Search for articles in the subject area:'Transposable elements'"><div class="flagText">Transposable elements</div></a>
              <div data-categoryid="17435" data-articleid="26336"
                   data-categoryname="Transposable elements"
                   class="flagImage" title="Flag 'Transposable elements' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Transposon+mutagenesis%22" title="Search for articles in the subject area:'Transposon mutagenesis'"><div class="flagText">Transposon mutagenesis</div></a>
              <div data-categoryid="50167" data-articleid="26336"
                   data-categoryname="Transposon mutagenesis"
                   class="flagImage" title="Flag 'Transposon mutagenesis' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=6374'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=4813'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=5799&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>


</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
