

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Austro-Asiatic Tribes of Northeast India Provide Hitherto Missing Genetic Link between South and Southeast Asia</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0001141"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0001141" />

    <meta name="citation_title" content="Austro-Asiatic Tribes of Northeast India Provide Hitherto Missing Genetic Link between South and Southeast Asia"/>
    <meta itemprop="name" content="Austro-Asiatic Tribes of Northeast India Provide Hitherto Missing Genetic Link between South and Southeast Asia"/>

      <meta name="citation_author" content="B. Mohan Reddy"/>
            <meta name="citation_author_institution" content="Biological Anthropology Unit, Indian Statistical Institute, Hyderabad, India"/>
      <meta name="citation_author" content="B. T. Langstieh"/>
            <meta name="citation_author_institution" content="Biological Anthropology Unit, Indian Statistical Institute, Hyderabad, India"/>
            <meta name="citation_author_institution" content="Department of Anthropology, Northeast Hill University, Shillong, India"/>
      <meta name="citation_author" content="Vikrant Kumar"/>
            <meta name="citation_author_institution" content="Biological Anthropology Unit, Indian Statistical Institute, Hyderabad, India"/>
            <meta name="citation_author_institution" content="Genome Institute of Singapore, Singapore, Singapore"/>
      <meta name="citation_author" content="T. Nagaraja"/>
            <meta name="citation_author_institution" content="Biological Anthropology Unit, Indian Statistical Institute, Hyderabad, India"/>
      <meta name="citation_author" content="A. N. S. Reddy"/>
            <meta name="citation_author_institution" content="Biological Anthropology Unit, Indian Statistical Institute, Hyderabad, India"/>
      <meta name="citation_author" content="Aruna Meka"/>
            <meta name="citation_author_institution" content="Biological Anthropology Unit, Indian Statistical Institute, Hyderabad, India"/>
      <meta name="citation_author" content="A. G. Reddy"/>
            <meta name="citation_author_institution" content="Centre for Cellular and Molecular Biology, Hyderabad, India"/>
      <meta name="citation_author" content="K. Thangaraj"/>
            <meta name="citation_author_institution" content="Centre for Cellular and Molecular Biology, Hyderabad, India"/>
      <meta name="citation_author" content="Lalji Singh"/>
            <meta name="citation_author_institution" content="Centre for Cellular and Molecular Biology, Hyderabad, India"/>

    <meta name="citation_date" content="2007/11/7"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0001141.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e1141"/>
    <meta name="citation_issue" content="11"/>
    <meta name="citation_volume" content="2"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=The history and geography of human genes.; citation_author=LL Cavalli-Sforza; citation_author=P Menozzi; citation_author=A Piazza; citation_number=1; citation_date=1994; citation_publisher=University Press; " />
      <meta name="citation_reference" content="citation_title=Natives or immigrants: modern human origin in East Asia.; citation_author=L Jin; citation_author=B Su; citation_journal_title=Nat Rev Genet; citation_volume=1; citation_number=2; citation_pages=126-133; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Reconstructing the origin of Andaman Islanders.; citation_author=K Thangaraj; citation_author=G Chaubey; citation_author=T Kivisild; citation_author=AG Reddy; citation_author=VK Singh; citation_journal_title=Science; citation_volume=308; citation_number=3; citation_pages=996; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Single, rapid coastal settlement of Asia revealed by analysis of complete mitochondrial genomes.; citation_author=V Macaulay; citation_author=C Hill; citation_author=A Achilli; citation_author=C Rengo; citation_author=D Clarke; citation_journal_title=Science; citation_volume=308; citation_number=4; citation_pages=1034-1036; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Status of Austro-Asiatic groups in the peopling of India: An exploratory study based on the available prehistoric, Linguistic and Bioligcal evidences.; citation_author=V Kumar; citation_author=BM Reddy; citation_journal_title=J Biosci; citation_volume=28; citation_number=5; citation_pages=507-522; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=The northeast Indian passageway: a barrier or corridor for human migrations?; citation_author=R Cordaux; citation_author=G Weiss; citation_author=N Saha; citation_author=M Stoneking; citation_journal_title=Mol Biol Evol; citation_volume=21; citation_number=6; citation_pages=1525-1533; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Y chromosome haplotypes reveal prehistorical migrations to the Himalayas.; citation_author=B Su; citation_author=C Xiao; citation_author=R Deka; citation_author=MT Seielstad; citation_author=D Kangwanpong; citation_journal_title=Hum. Genet; citation_volume=107; citation_number=7; citation_pages=582-590; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Antiquity, geographic contiguity and genetic affinity among Tibeto-Burman populations of India: a microsatellite study; citation_author=S Krithika; citation_author=R Trivedi; citation_author=VK Kashyap; citation_author=P Bharati; citation_author=TS Vasulu; citation_journal_title=Ann Hum Biol; citation_volume=33; citation_number=8; citation_pages=26-42; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Researches on the prehistoric archaeology of Assam.; citation_author=TC Sharma; citation_journal_title=Jr Assam Sc Soc; citation_volume=IX; citation_number=9; citation_pages=1-11; citation_date=1966; " />
      <meta name="citation_reference" content="citation_title=Prehistoric archaeology in North Eastern India-A review in progress.; citation_author=TC Sharma; citation_number=10; citation_pages=102-107; citation_date=1980; citation_publisher=Cosmo Publications; " />
      <meta name="citation_reference" content="citation_title=North East India in prehistoric times.; citation_author=TC Sharma; citation_number=11; citation_pages=1-23; citation_date=1984; citation_publisher=Vendrame Missiological Institute, Centre of Indigenous Culture; " />
      <meta name="citation_reference" content="citation_title=Who are the pre-historic dwellers of the Meghalaya plateau?; citation_author=Z Hussain; citation_number=12; citation_pages=74-85; citation_date=1991; citation_publisher=Vikas Publishing House Pvt. Ltd; " />
      <meta name="citation_reference" content="citation_title=The contribution of linguistic palaeontology to the homeland of Austro-Asiatic.; citation_author=G Diffloth; citation_number=13; citation_pages=77-81; citation_date=2005; citation_publisher=Routledge Curzon; " />
      <meta name="citation_reference" content="citation_title=Molecular evolutionary genetics.; citation_author=M Nei; citation_number=14; citation_date=1987; citation_publisher=Columbia University Press; " />
      <meta name="citation_reference" content="citation_title=Polarity and temporality of high-resolution y-chromosome distributions in India identify both indigenous and exogenous expansions and reveal minor genetic influence of central asian pastoralists.; citation_author=S Sengupta; citation_author=LA Zhivotovsky; citation_author=R King; citation_author=SQ Mehdi; citation_author=CA Edmonds; citation_journal_title=Am J Hum Genet; citation_volume=78; citation_number=15; citation_pages=202-21; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=The dazzling array of basal branches in the mtDNA macrohaplogroup M from India as inferred from complete genomes.; citation_author=C Sun; citation_author=QP Kong; citation_author=MG Palanichamy; citation_author=S Agrawal; citation_author=HJ Bandelt; citation_journal_title=Mol Biol Evol; citation_volume=23; citation_number=16; citation_pages=683-690; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Phylogeny of East Asian mitochondrial DNA lineages inferred from complete sequences.; citation_author=QP Kong; citation_author=YG Yao; citation_author=C Sun; citation_author=HJ Bandelt; citation_author=CL Zhu; citation_journal_title=Am J Hum Genet; citation_volume=73; citation_number=17; citation_pages=671-676; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Multiplexed SNP typing of ancient DNA clarifies the origin of andaman mtDNA haplogroups amongst south Asian tribal populations.; citation_author=P Endicott; citation_author=M Metspalu; citation_author=C Stringer; citation_author=V Macaulay; citation_author=A Cooper; citation_journal_title=PLoS ONE; citation_volume=1; citation_number=18; citation_pages=e81; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Comment on “Reconstructing the origin of Andaman islanders”.; citation_author=MG Palanichamy; citation_author=S Agrawal; citation_author=YG Yao; citation_author=QP Kong; citation_author=C Sun; citation_journal_title=Science; citation_volume=311; citation_number=19; citation_pages=470; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Genetic affinities among the lower castes and tribal groups of India: inference from Y chromosome and mitochondrial DNA.; citation_author=I Thanseem; citation_author=K Thangaraj; citation_author=G Chaubey; citation_author=VK Singh; citation_author=LV Bhaskar; citation_journal_title=BMC Genet; citation_volume=7; citation_number=20; citation_pages=42; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Natural selection shaped regional mtDNA variation in humans.; citation_author=D Mishmar; citation_author=E Ruiz-Pesini; citation_author=P Golik; citation_author=V Macaulay; citation_author=AG Clark; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=7; citation_number=21; citation_pages=171-6; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Y-chromosome evidence suggests a common paternal heritage of Austro-Asiatic populations.; citation_author=V Kumar; citation_author=AN Reddy; citation_author=JP Babu; citation_author=TN Rao; citation_author=BT Langstieh; citation_journal_title=BMC Evol Biol.; citation_volume=7; citation_number=22; citation_pages=47; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Genomic structures and population histories of linguistically distinct tribal groups of India.; citation_author=S Roychoudhury; citation_author=S Roy; citation_author=A Basu; citation_author=R Banerjee; citation_author=H Vishwanathan; citation_journal_title=Hum Genet; citation_volume=3; citation_number=23; citation_pages=339-350; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=The genetic heritage of the earliest settlers persists both in Indian. tribal and caste populations.; citation_author=T Kivisild; citation_author=S Rootsi; citation_author=M Metspalu; citation_author=S Mastana; citation_author=K Kaldma; citation_journal_title=Am J Hum Genet; citation_volume=72; citation_number=24; citation_pages=313-332; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Most of the extant mtDNA boundaries in south and southwest Asia were likely shaped during the initial settlement of Eurasia by anatomically modern humans.; citation_author=M Metspalu; citation_author=T Kivisild; citation_author=E Metspalu; citation_author=J Parik; citation_author=G Hudjashov; citation_journal_title=BMC Genet; citation_volume=5; citation_number=25; citation_pages=e26; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Reanalysis and revision of the Cambridge reference sequence for human mitochondrial DNA.; citation_author=RM Andrews; citation_author=I Kubacka; citation_author=PF Chinnery; citation_author=RN Lightowlers; citation_author=DM Turnbull; citation_journal_title=Nat Genet; citation_volume=23; citation_number=26; citation_pages=147; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=The emerging tree of West Eurasian mtDNAs: a synthesis of control-region sequences and RFLPs.; citation_author=V Macaulay; citation_author=M Richards; citation_author=E Hickey; citation_author=E Vega; citation_author=F Cruciani; citation_journal_title=Am J Hum Genet; citation_volume=64; citation_number=27; citation_pages=232-249; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Genetic evidence of an early exit of Homo sapies sapiens from Africa through eastern Africa.; citation_author=L Qunitana-Murci; citation_author=O Semino; citation_author=HJ Bandelt; citation_author=G Passarino; citation_author=K McElreavey; citation_journal_title=Nat Genet; citation_volume=23; citation_number=28; citation_pages=437-441; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Tracing European founder lineages in the Near Eastern mtDNA pool.; citation_author=M Richards; citation_author=V Macaulay; citation_author=E Hickey; citation_author=E Vega; citation_author=B Sykes; citation_journal_title=Am J Hum Genet; citation_volume=67; citation_number=29; citation_pages=1251-1276; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Phylogeny of mitochondrial DNA macrohaplogroup N in India, based on complete sequencing: implications for the peopling of South Asia.; citation_author=MG Palanichamy; citation_author=C Sun; citation_author=S Agrawal; citation_author=HJ Bandelt; citation_author=QP Kong; citation_journal_title=Am J Hum Genet; citation_volume=75; citation_number=30; citation_pages=966-978; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=The phylogeography of Y chromosome binary haplotypes and the origins of modern human populations.; citation_author=PA Underhill; citation_author=G Passarino; citation_author=AA Lin; citation_author=P Shen; citation_author=M Mirazon Lahr; citation_journal_title=Ann Hum Genet; citation_volume=65; citation_number=31; citation_pages=43-62; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=A nomenclature system for the tree of human Y-chromosomal binary haplogroups.; citation_journal_title=Genome Res; citation_volume=12; citation_number=32; citation_pages=339-348; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=The human Y chromosome: an evolutionary marker comes of age.; citation_author=MA Jobling; citation_author=C Tyler-Smith; citation_journal_title=Nat Rev Genet; citation_volume=8; citation_number=33; citation_pages=598-612; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=A novel multiplex for simultaneous amplification of 20 Y chromosome STR markers.; citation_author=JM Butler; citation_author=R Schoske; citation_author=PM Vallone; citation_author=MC Kline; citation_author=AJ Redd; citation_journal_title=Forensic Sci Int; citation_volume=129; citation_number=34; citation_pages=10-24; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Arlequin ver. 3.0: An integrated software package for population genetics data analysis.; citation_author=L Excoffier; citation_author=G Laval; citation_author=S Schneider; citation_journal_title=Evolutionary Bioinformatics Online; citation_volume=1; citation_number=35; citation_pages=47-50; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Median joining networks for inferring intraspecific phylogenies.; citation_author=H Bandelt; citation_author=P Forster; citation_author=A Rohl; citation_journal_title=Mol Biol Evol; citation_volume=16; citation_number=36; citation_pages=37-48; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Genetic affinities of the andaman islanders, a vanishing human population.; citation_author=K Thangaraj; citation_author=L Singh; citation_author=A Reddy; citation_author=V Rao; citation_author=S Sehgal; citation_journal_title=Curr Biol; citation_volume=13; citation_number=37; citation_pages=86-93; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Paternal population history of East Asia: sources, patterns, and microevolutionary processes.; citation_author=T Karafet; citation_author=L Xu; citation_author=R Du; citation_author=W Wang; citation_author=S Feng; citation_journal_title=Am J Hum Genet; citation_volume=69; citation_number=38; citation_pages=615-628; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Mitochondrial DNA polymorphisms in Thailand.; citation_author=G Fucharoen; citation_author=S Fucharoen; citation_author=S Horai; citation_journal_title=J Hum Genet; citation_volume=3; citation_number=39; citation_pages=115-125; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Human mtDNA and Y-chromosome variation is correlated with matrilocal versus patrilocal residence.; citation_author=H Oota; citation_author=W Settheetham-Ishida; citation_author=D Tiwawech; citation_author=T Ishida; citation_author=M Stoneking; citation_journal_title=Nat Genet; citation_volume=29; citation_number=40; citation_pages=20-1; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Mitochondrial DNA variation in Nicobarese Islanders.; citation_author=BV Prasad; citation_author=CE Ricker; citation_author=WS Watkins; citation_author=ME Dixon; citation_author=BB Rao; citation_journal_title=Hum Biol; citation_volume=5; citation_number=41; citation_pages=715-725; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Genetic relationship of Chinese ethnic populations revealed by mtDNA sequence diversity.; citation_author=Y Yao; citation_author=L Nie; citation_author=H Harpending; citation_author=Y Fu; citation_author=Z Yuan; citation_journal_title=Am J Phys Anthropol; citation_volume=118; citation_number=42; citation_pages=63-76; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Phylogeographic differentiation of mitochondrial DNA in Han Chinese.; citation_author=YG Yao; citation_author=QP Kong; citation_author=HJ Bandelt; citation_author=T Kivisild; citation_author=YP Zhang; citation_journal_title=Am J Hum Genet; citation_volume=70; citation_number=43; citation_pages=635-651; citation_date=2002; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001141" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Austro-Asiatic Tribes of Northeast India Provide Hitherto Missing Genetic Link between South and Southeast Asia"/>
    <meta name="twitter:description" content="Northeast India, the only region which currently forms a land bridge between the Indian subcontinent and Southeast Asia, has been proposed as an important corridor for the initial peopling of East Asia. Given that the Austro-Asiatic linguistic family is considered to be the oldest and spoken by certain tribes in India, Northeast India and entire Southeast Asia, we expect that populations of this family from Northeast India should provide the signatures of genetic link between Indian and Southeast Asian populations. In order to test this hypothesis, we analyzed mtDNA and Y-Chromosome SNP and STR data of the eight groups of the Austro-Asiatic Khasi from Northeast India and the neighboring Garo and compared with that of other relevant Asian populations. The results suggest that the Austro-Asiatic Khasi tribes of Northeast India represent a genetic continuity between the populations of South and Southeast Asia, thereby advocating that northeast India could have been a major corridor for the movement of populations from India to East/Southeast Asia."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0001141.g006"/>

  <meta property="og:title" content="Austro-Asiatic Tribes of Northeast India Provide Hitherto Missing Genetic Link between South and Southeast Asia" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001141" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=6054'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=2448'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=7622&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0001141">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001141" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001141&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Austro-Asiatic Tribes of Northeast India Provide Hitherto Missing Genetic Link between South and Southeast Asia
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            B. Mohan Reddy
              <span class="corresponding">mail</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:bmr@isical.ac.in">bmr@isical.ac.in</a></p>

                <p>Affiliation:
                  Biological Anthropology Unit, Indian Statistical Institute, Hyderabad, India
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            B. T. Langstieh, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliations:
                  Biological Anthropology Unit, Indian Statistical Institute, Hyderabad, India, 
                  Department of Anthropology, Northeast Hill University, Shillong, India
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Vikrant Kumar, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliations:
                  Biological Anthropology Unit, Indian Statistical Institute, Hyderabad, India, 
                  Genome Institute of Singapore, Singapore, Singapore
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            T. Nagaraja, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Biological Anthropology Unit, Indian Statistical Institute, Hyderabad, India
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            A. N. S. Reddy, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Biological Anthropology Unit, Indian Statistical Institute, Hyderabad, India
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Aruna Meka, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Biological Anthropology Unit, Indian Statistical Institute, Hyderabad, India
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            A. G. Reddy, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Centre for Cellular and Molecular Biology, Hyderabad, India
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            K. Thangaraj, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Centre for Cellular and Molecular Biology, Hyderabad, India
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Lalji Singh
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Centre for Cellular and Molecular Biology, Hyderabad, India
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: November 07, 2007</li>
    <li>DOI: 10.1371/journal.pone.0001141</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0001141" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0001141" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001141" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0001141" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0001141-g001" data-doi="info:doi/10.1371/journal.pone.0001141" data-uri="info:doi/10.1371/journal.pone.0001141.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001141.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001141-t001" data-doi="info:doi/10.1371/journal.pone.0001141" data-uri="info:doi/10.1371/journal.pone.0001141.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001141.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001141-g002" data-doi="info:doi/10.1371/journal.pone.0001141" data-uri="info:doi/10.1371/journal.pone.0001141.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001141.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001141-t002" data-doi="info:doi/10.1371/journal.pone.0001141" data-uri="info:doi/10.1371/journal.pone.0001141.t002" title="Table 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001141.t002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001141-g003" data-doi="info:doi/10.1371/journal.pone.0001141" data-uri="info:doi/10.1371/journal.pone.0001141.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001141.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001141-t003" data-doi="info:doi/10.1371/journal.pone.0001141" data-uri="info:doi/10.1371/journal.pone.0001141.t003" title="Table 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001141.t003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001141-g004" data-doi="info:doi/10.1371/journal.pone.0001141" data-uri="info:doi/10.1371/journal.pone.0001141.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001141.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001141-g005" data-doi="info:doi/10.1371/journal.pone.0001141" data-uri="info:doi/10.1371/journal.pone.0001141.g005" title="Figure 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001141.g005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001141-g006" data-doi="info:doi/10.1371/journal.pone.0001141" data-uri="info:doi/10.1371/journal.pone.0001141.g006" title="Figure 6">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001141.g006&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001141-t004" data-doi="info:doi/10.1371/journal.pone.0001141" data-uri="info:doi/10.1371/journal.pone.0001141.t004" title="Table 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001141.t004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001141-t005" data-doi="info:doi/10.1371/journal.pone.0001141" data-uri="info:doi/10.1371/journal.pone.0001141.t005" title="Table 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001141.t005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001141-t006" data-doi="info:doi/10.1371/journal.pone.0001141" data-uri="info:doi/10.1371/journal.pone.0001141.t006" title="Table 6">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001141.t006&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001141">Reader Comments (0)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0001141" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1"></a><p>Northeast India, the only region which currently forms a land bridge between the Indian subcontinent and Southeast Asia, has been proposed as an important corridor for the initial peopling of East Asia. Given that the Austro-Asiatic linguistic family is considered to be the oldest and spoken by certain tribes in India, Northeast India and entire Southeast Asia, we expect that populations of this family from Northeast India should provide the signatures of genetic link between Indian and Southeast Asian populations. In order to test this hypothesis, we analyzed mtDNA and Y-Chromosome SNP and STR data of the eight groups of the Austro-Asiatic Khasi from Northeast India and the neighboring Garo and compared with that of other relevant Asian populations. The results suggest that the Austro-Asiatic Khasi tribes of Northeast India represent a genetic continuity between the populations of South and Southeast Asia, thereby advocating that northeast India could have been a major corridor for the movement of populations from India to East/Southeast Asia.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>Reddy BM, Langstieh BT, Kumar V, Nagaraja T, Reddy ANS, et al.  (2007) Austro-Asiatic Tribes of Northeast India Provide Hitherto Missing Genetic Link between South and Southeast Asia. PLoS ONE 2(11):
          e1141.
            doi:10.1371/journal.pone.0001141</p><p><strong>Academic Editor: </strong>Philip Awadalla, University of Montreal, Canada</p><p><strong>Received:</strong> March 20, 2007; <strong>Accepted:</strong> October 16, 2007; <strong>Published:</strong> November 7, 2007</p><p><strong>Copyright:</strong> © 2007 Reddy et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>This project was funded by Indian Statistical Institute.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>Two major routes have been proposed for the initial peopling of East Asia; one via Central Asia to Northeast Asia, which subsequently expanded towards Southeast Asia and beyond, and the other through India to Southeast Asia and further to different regions of East Asia <a href="#pone.0001141-CavalliSforza1">[1]</a>. It is pertinent in this context that the Indian subcontinent has been considered as a major corridor for the migration of human populations to East Asia <a href="#pone.0001141-Jin1">[2]</a>–<a href="#pone.0001141-Macaulay1">[4]</a>. Given its unique geographic position, Northeast India is the only region which currently forms a land bridge between the Indian subcontinent and Southeast Asia, hence hypothesized as an important passage for the initial peopling of East Asia. This region is inhabited by populations belonging to Indo-European, Tibeto-Burman and Austro-Asiatic linguistic families. Whereas Indo-European populations are also found in other parts of India, West Asia and Europe but absent in East Asia, Tibeto-Burman populations are otherwise found only in East Asia. However, Austro-Asiatic speakers, hypothesized as probably the earliest settlers in the Indian subcontinent (<a href="#pone.0001141-Kumar1">[5]</a> and references their in), are also found in other parts of India as well as in East/Southeast Asia. Therefore, if Northeast India had served as an initial corridor, it is likely that the Austro-Asiatic tribes of this region should provide hitherto missing genetic link, which may reflect genetic continuity between Indian and East/Southeast Asian populations. Based on mitochondrial DNA (mtDNA) and Y-chromosome markers, Cordaux et al. <a href="#pone.0001141-Cordaux1">[6]</a> observed genetic discontinuity between the Indian and southeast Asian populations and inferred that Northeast India might have acted as a barrier rather than the facilitator of the movement of populations both into and out of India. However, this study included only a few Tibeto-Burman populations of Northeast India whose distribution is restricted only to this region in India <a href="#pone.0001141-Su1">[7]</a>–<a href="#pone.0001141-Krithika1">[8]</a>, besides a few other populations from other parts of India, possibly with no genetic link with East Asians. It is therefore imperative that the framework of testing such a hypothesis should include adequate representation of these people from Northeast India. Further evidence is needed by way of determining the mtDNA and Y-chromosome haplogroups/lineages of the Austro-Asiatic tribes of the northeastern region and their comparison with appropriate set of South and Southeast Asian populations.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>The Northeastern part of India is sandwiched by the marked presence of the young fold mountains of Eastern Himalayas on the northern side and the Indian Ocean on its southern side. The archaeological evidences, based on stone tools from the Garo hills of the Meghalaya region of Northeast India (<a href="#pone-0001141-g001">Figure 1</a>), suggest that this region might have been inhabited as early as in the Paleolithic period <a href="#pone.0001141-Sharma1">[9]</a>–<a href="#pone.0001141-Hussain1">[12]</a>. Within this Meghalaya region, one finds Khasi tribes whose language belongs to the Khasi-Khmic subfamily of the Austro-Asiatics <a href="#pone.0001141-Diffloth1">[13]</a>, the other two branches of Austro-Asiatics being Mundari and Mon-Khmer, amidst the ethnic majority of Tibeto-Burman populations. Further, the Austro-Asiatic populations of Meghalaya and one of the contiguous Tibeto-Burman tribes namely, Garo, practice matrilineal pattern of descent and matrilocal pattern of residence while the other populations of this region are patrilineal and patrilocal. Such extreme cultural and linguistic diversity of this region may also imply high degree of genetic heterogeneity possibly due to passage of diverse populations through this region. Despite a possible major role played in the population dispersal by the Northeastern region, as transect between India and East Asia, the extent and nature of mtDNA and Y-chromosome diversity of this region is not adequately studied. Therefore, we present results based on the analyses of mtDNA and Y-Chromosome Single Nucleotide Polymorphisms (SNPs) and Short Tandem Repeats (STRs) data of the 8 subgroups of the Austro-Asiatic Khasi from Northeast India (<a href="#pone-0001141-t001">Table 1</a> and <a href="#pone-0001141-g001">Figure 1</a>), probably for the first time, along with that of the other relevant populations to explore the missing genetic link between the Indian and Southeast Asian populations.</p>
<div class="figure" id="pone-0001141-g001"><div class="img"><a name="pone-0001141-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001141.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001141" data-uri="info:doi/10.1371/journal.pone.0001141.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001141.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001141.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001141.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001141.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001141.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001141.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001141.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001141.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001141.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Map of Meghlaya showing the core areas of geographic distribution of the Khasi and Garo populations.</span></strong></p><span>doi:10.1371/journal.pone.0001141.g001</span></div><div class="figure" id="pone-0001141-t001"><div class="img"><a name="pone-0001141-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001141.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001141" data-uri="info:doi/10.1371/journal.pone.0001141.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001141.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001141.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001141.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001141.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001141.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001141.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001141.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001141.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001141.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>Areas of sampling and the linguistic affiliations of the Meghalayan populations along with the number of samples typed for Y-chromosome and mtDNA.</span></strong></p><span>doi:10.1371/journal.pone.0001141.t001</span></div></div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3>
<h4>Distribution and diversity of Y-chromosome</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>Nei's <a href="#pone.0001141-Nei1">[14]</a> diversity statistic, <em>h,</em> based on the frequency of different haplogroups (<a href="#pone-0001141-t002">Table 2</a>), ranges from 77% in the Maram to 86.2% in the Pnar among the Khasi-Khmuic Austro-Asiatic groups, whereas it is 77.5% in the Tibeto-Burman Garo. For Y-STR haplotypes, while it ranges from 96.1% in Nongtrai to 99.9% in Khynriam in the Khasi-Khmic populations, it is 99.3% for the Garo. Out of the 26 potential haplogroups defined by the markers used in this study a total of 12 haplogroups were found in these populations (<a href="#pone-0001141-g002">Figure 2</a>). O-M95, with its frequency ranging from 17% in War-Khasi to 42% in War-Jaintia, was the most common haplogroup in all the Austro-Asiatic populations followed by the undifferentiated O-M122 (ranging from 11% in Nongtrai to 34% in Bhoi) where as in the Tibeto-Burman Garo the frequency of O-M134 and undifferentiated O-M122 haplogroups (23% and 17%, respectively) were the most common. H-M69 and its subclade H-M82 which is reported to be in high frequency in most of the Indo-European populations <a href="#pone.0001141-Sengupta1">[15]</a> are present with an average frequency of only 3% among them.</p>
<div class="figure" id="pone-0001141-g002"><div class="img"><a name="pone-0001141-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001141.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001141" data-uri="info:doi/10.1371/journal.pone.0001141.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001141.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001141.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001141.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001141.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001141.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001141.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001141.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001141.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001141.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Rooted maximum-parsimony tree of Y-chromosome haplogroups defined by binary markers along with their frequency in Nine Meghalayan Populations.</span></strong></p><span>doi:10.1371/journal.pone.0001141.g002</span></div><div class="figure" id="pone-0001141-t002"><div class="img"><a name="pone-0001141-t002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001141.t002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001141" data-uri="info:doi/10.1371/journal.pone.0001141.t002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001141.t002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001141.t002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001141.t002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001141.t002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001141.t002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001141.t002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001141.t002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001141.t002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001141.t002.TIF"></span>)
                </a></li></ul></div><p><strong>Table 2.  <span>Genetic Diversity (in percentage) based on Y-Chromosome and mtDNA analysis of populations from Meghalaya</span></strong></p><span>doi:10.1371/journal.pone.0001141.t002</span></div>

<h4>Population structure based on Y-chromosome</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>Based on the multidimensional scaling (MDS) of the Pairwise F<sub>ST</sub> distances computed using haplogroup frequencies of Austro-Asiatic (Khasi from northeast India and others) and neighboring non-Austro-Asiatic populations, the two-dimensional MDS plot is furnished in <a href="#pone-0001141-g003">Figure 3</a>. A good fit between the two-dimensional MDS plot and the source data (pairwise value of F<sub>ST</sub>) was obtained (stress value of 18%). Broadly speaking, most of the Austro-Asiatic populations, including all the three linguistic sub-families of Austro-Asiatics i.e Mundari, Khasi-Khmuic and Mon-Khmer tribes, irrespective of their geographic affiliations, are placed in the upper right quadrant; Nicobarese, Ho, Santhal, She and Zhuang are somewhat removed from the others. On the other hand, most of the Tibeto-Burman populations are differentiated from the Austro-Asiatic populations and the Indo-European populations (clustered in the lower right quadrant) on the 1<sup>st</sup> and 2<sup>nd</sup> dimension, respectively. The Khasi-Khmuic populations, which form a compact cluster near the centroid, do not cluster with the Tibeto-Burman populations of Northeast India, barring the Garo of Meghalaya which has contiguous geographic distribution and marital interaction with them. Overall, the populations of the same linguistic family seem to cluster together, with few exceptions such as the Austro-Asiatic Lodha, which is placed among the Indo-European populations.</p>
<div class="figure" id="pone-0001141-g003"><div class="img"><a name="pone-0001141-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001141.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001141" data-uri="info:doi/10.1371/journal.pone.0001141.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001141.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001141.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001141.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001141.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001141.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001141.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001141.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001141.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001141.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Plot on the first two dimensions derived from the multidimensional scaling of the pairwise F<sub>ST</sub> distances of the populations based on Y-haplogroups.</span></strong></p><a id="article1.body1.sec2.sec2.fig1.caption1.p1" name="article1.body1.sec2.sec2.fig1.caption1.p1"></a><p>Reference to comparative data is given at <a href="#pone-0001141-t005">Table 5</a>. SEA, Southeast Asian; AA, Austro-Asiatic; IE, Indo-European, TB, Tibeto-Burman; Khy, Khynriam; Lyn, Lyngngam; Wkhasi, War Khasi; Viet, Vietnamese;</p>
<span>doi:10.1371/journal.pone.0001141.g003</span></div><a id="article1.body1.sec2.sec2.p2" name="article1.body1.sec2.sec2.p2"></a><p>The analysis of molecular variance (AMOVA) yielded a significant but low F<sub>ST</sub> values for both Y-SNPs (0.02) and STRs (0.02), suggesting a probable recent differentiation of the Khasi-Khmuic populations (<a href="#pone-0001141-t003">Table 3</a>). For Y-SNPs, whereas among group differentiation between the Khasi and Southeast Asian Austro-Asiatic populations is low (0.03) and non-significant it is relatively high and significant between the Khasi and Mundari populations (0.08). On the other hand, the F<sub>CT</sub> value between Khasi-Khmuic and Indian Tibeto-Burman populations is very high and significant (0.30) while between Khasi-Khmuic and Southeast Asian Tibeto-Burman it was relatively low and non-significant (0.03). Although there is virtually no difference in the haplogroupic composition of the Tibeto-Burman Garo from Meghalaya and Southeast Asian Tibeto-Burman populations as suggested by the F<sub>CT</sub> (−0.01627), it is surprisingly high (0.17975) between the Garo and the other Indian Tibeto-Burman populations.</p>
<div class="figure" id="pone-0001141-t003"><div class="img"><a name="pone-0001141-t003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001141.t003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001141" data-uri="info:doi/10.1371/journal.pone.0001141.t003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001141.t003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001141.t003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001141.t003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001141.t003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001141.t003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001141.t003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001141.t003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001141.t003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001141.t003.TIF"></span>)
                </a></li></ul></div><p><strong>Table 3.  <span>Analysis of Molecular Variance using Y-SNPs/STRs between groups of populations categorized on the basis of geography and languages</span></strong></p><span>doi:10.1371/journal.pone.0001141.t003</span></div>

<h4>Profile of new mtDNA haplogroups</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>Based on Hypervariable segment (HVS) I and the known coding region SNPs most of the individuals could be assigned to specific haplogroups/lineages. However, there were still many individuals who could not be assigned to any existing lineages. Based on their HVS-I motif we could group these samples into 6 broad clades, and resequenced complete mtDNA of 1-2 samples from each of those clades to assign them to a known or new haplogroups (<a href="#pone-0001141-g004">Fig 4</a>). We also resequenced complete mtDNA for the samples falling in haplogroup B as none of the defining mutations for the subhaplogroups of B were found. The analysis of complete mtDNA suggests the presence of four new haplogroups which we have designated as M48, M49, M50 and B7. All the motifs in the coding region of the M48, except for 6336, which defines M30a <a href="#pone.0001141-Sun1">[16]</a> have not been reported and therefore we assign all these samples a new lineage. While the average frequency of M48 is 11% among the Austro-Asiatic Khasi groups, ranging from zero in War-Jaintia to as high as 26% in Lyngngam, it is present with a frequency of 4% among the Garo. Although haplogroups M49 and M50 are found with an average frequency of about 3% each in the Khasi populations, they could not be traced in the Garo as well as in some of the subgroups of Khasi. A subset of mutations at 150-9452-12950-13928C of our B-haplogroup samples has been reported in one of the samples (SD10313) of Han Chinese <a href="#pone.0001141-Kong1">[17]</a> which also falls in undifferentiated haplogroup B. We have proposed to name it as haplogroup B7 including the Han Chinese samples.</p>
<div class="figure" id="pone-0001141-g004"><div class="img"><a name="pone-0001141-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001141.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001141" data-uri="info:doi/10.1371/journal.pone.0001141.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001141.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001141.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001141.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001141.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001141.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001141.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001141.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001141.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001141.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>Phylogenetic tree of new haplogroups based on full mtDNA along with the TMRCA and associated 95% Confidence Interval.</span></strong></p><a id="article1.body1.sec2.sec3.fig1.caption1.p1" name="article1.body1.sec2.sec3.fig1.caption1.p1"></a><p>The samples names are on the tip of the haplogroups. The comparative data, for which sample names are written in capital letters, are from Kong et al. <a href="#pone.0001141-Kong1">[17]</a>, Thangaraj et al. <a href="#pone.0001141-Thangaraj1">[3]</a>, Sun et al. <a href="#pone.0001141-Sun1">[16]</a> and Palanichamy et al. <a href="#pone.0001141-Palanichamy1">[19]</a>.</p>
<span>doi:10.1371/journal.pone.0001141.g004</span></div><a id="article1.body1.sec2.sec3.p2" name="article1.body1.sec2.sec3.p2"></a><p>In addition to these four new haplogroups we propose two new sub-haplogroups –M33b- within M33, and M31c within M31 (<a href="#pone-0001141-g004">Fig 4</a>). The samples falling in M31c has all the defining mutations of M31 but do not share any of the coding region motifs with either M31a which has been reported in the Andamanese of Andaman and Nicobar island <a href="#pone.0001141-Thangaraj1">[3]</a> and other Tribal populations of India <a href="#pone.0001141-Endicott1">[18]</a>, or M31b found in Rajbanshis (SW1) of Northeast India <a href="#pone.0001141-Palanichamy1">[19]</a>. Therefore, we propose a new haplogroup, i.e. M31c. While this haplogroup is absent in the Garo, it is found with an average frequency of ~5% in the Austro-Asiatic Khasi populations with a maximum frequency of ~17% among the Bhoi. The samples of M33b have mutations which define M33 and it also shares mutations at positions 1719-3221-16293-16324 with the Rajbanshi sample (SW23) which is now re-designated as M33b. The frequency of M33b, with the exception of Pnar (~22%) is low and found only in Lyngngam, Khynriam and Garo (~2, ~3 and ~3%, respectively). On the other hand, M33a which were found to be in extremely high frequency in the Garo (~55%) and with an average frequency of~5% in Khasi-Khumic populations has been also reported in the Brahmins of Uttar Pradesh, India <a href="#pone.0001141-Sun1">[16]</a> and in the two populations of South India <a href="#pone.0001141-Thanseem1">[20]</a>. It is interesting to note that all the samples of this study, except one Khynriam sample, forms a single sublineage defined by 16316 HVS-I motif which distinguishes it from other M33a lineages found in other parts of India.</p>


<h4>Distribution and diversity of mtDNA haplotypes/lineages</h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>In the 444 samples representing the 8 Khasi-Khmuic Austro-Asiatic tribes and a Tibeto-Burman Garo a total of 117 distinct HVSI haplotypes were found. Among these, 67 haplotypes are unique, each represented by single individual. Of the remaining, 37 are shared at least by two different tribes out of which only 10 are shared between Garo and Khasi subtribes. Based on the phylogenetic analysis of mtDNA control and coding region SNPs, 37 distinct haplogroups and subhaplogroups were observed among the studied populations (<a href="#pone-0001141-g005">Fig 5</a>). The samples that still remained unclassified in M and R are only ~6%, and 0.5%, respectively. Among the Austro-Asiatic Khasi, ~80% of the variation is accounted for by a set of 10 haplogroups–M*, M4a, M9a, M31c, M33a, M33b, M48, MD, MD4 and U2, whereas in the Garo a subset of only 3 haplogroups–M*, M33a and U2-accounted for ~80% of the total sample. However, these 3 haplogroups account for only ~18% of the sampled individuals from the neighbouring Austro-Asiatic Khasi populations.</p>
<div class="figure" id="pone-0001141-g005"><div class="img"><a name="pone-0001141-g005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001141.g005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001141" data-uri="info:doi/10.1371/journal.pone.0001141.g005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001141.g005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001141.g005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001141.g005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001141.g005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001141.g005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001141.g005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001141.g005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001141.g005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001141.g005.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 5.  <span>Tree Drawn from a Median-Joining Network of mtDNA Haplogroups Observed in Nine Meghalayan Populations.</span></strong></p><a id="article1.body1.sec2.sec4.fig1.caption1.p1" name="article1.body1.sec2.sec4.fig1.caption1.p1"></a><p>The haplogroups box in yellow colour are either new or redefined haplogroups.</p>
<span>doi:10.1371/journal.pone.0001141.g005</span></div><a id="article1.body1.sec2.sec4.p2" name="article1.body1.sec2.sec4.p2"></a><p>The mtDNA haplogroup diversity (<a href="#pone-0001141-t002">Table 2</a>) among the Austro-Asiatic groups is low and ranges from 83.1% in War-Jaintia to 93.6% in Bhoi whereas in Garo the diversity is extremely low (66.9%). Similarly, the haplotype diversity (<a href="#pone-0001141-t002">Table 2</a>) for the Austro-Asiatic groups ranges from 86.8% in War-Jaintia to 96.1% in Khynriam where as in the Garo it is 68.1%.</p>


<h4>Population relationships based on mtDNA haplogroups</h4>
<a id="article1.body1.sec2.sec5.p1" name="article1.body1.sec2.sec5.p1"></a><p>The two dimensional plot of the multidimensional scaling of the genetic distance matrix of the 40 populations, including 8 Khasi subtribes and Garo of the present study and other relevant populations from the South and southeast Asia, is shown in <a href="#pone-0001141-g006">Figure 6</a>. The plot depicts the Tibeto-Burman Garo and Austro-Asiatic Nicobarese (a Mon-Khmer population) and Sakai as extreme outliers. As expected, the Mundari Austro-Asiatic populations, with predominantly South Asian mtDNA haplogroups, are placed as outliers aligning with the two Indian Indo-European populations on the extreme right corner of the plot. Although the Khasi-Khmuic Austro-Asiatic populations, except for Nongtrai, Lyngngam, form a constellation near to the left of centroid, it also has other populations such as Han, Lisu and Bai as part of this constellation. The Southeast Asian Tibeto-Burman populations is scattered along the 1<sup>st</sup> axis. Similarly, the Indian Tibeto-Burmans do not form its own cluster. Overall, the three different sub-families of Austro-Asiatic populations do not form a homogeneous cluster, unlike in the case of Y-chromosome.</p>
<div class="figure" id="pone-0001141-g006"><div class="img"><a name="pone-0001141-g006" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001141.g006&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001141" data-uri="info:doi/10.1371/journal.pone.0001141.g006"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001141.g006&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001141.g006/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001141.g006/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001141.g006/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001141.g006/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001141.g006.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001141.g006/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001141.g006/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001141.g006.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 6.  <span>Plot on the first two dimensions derived from the multidimensional scaling of the pairwise F<sub>ST</sub> distances of the populations based on mtDNA haplogroups.</span></strong></p><a id="article1.body1.sec2.sec5.fig1.caption1.p1" name="article1.body1.sec2.sec5.fig1.caption1.p1"></a><p>Reference to comparative data is given at <a href="#pone-0001141-t006">Table 6</a>. SEA, Southeast Asian; AA, Austro-Asiatic; IE, Indo-European; TB, Tibeto-Burman; Wk, WarKhasi; Ch, Chong;</p>
<span>doi:10.1371/journal.pone.0001141.g006</span></div><a id="article1.body1.sec2.sec5.p2" name="article1.body1.sec2.sec5.p2"></a><p>Although the AMOVA suggests low F<sub>ST</sub> value (0.05), hence low differentiation among the Khasi-Khmuic populations (<a href="#pone-0001141-t004">Table 4</a>), it is quite high between them and the Garo (0.12). The differentiation of Khasi-Khmuic tribes with Southeast Asian Austro-Asiatic populations is moderate (0.05) but is much higher with those of Mundari populations (0.12). Surprisingly, the Tibeto-Burman Garo of Meghalaya shows high degree of differentiation with the other Tibeto-Burman populations of India (0.17) as well as Southeast Asia (0.13).</p>
<div class="figure" id="pone-0001141-t004"><div class="img"><a name="pone-0001141-t004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001141.t004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001141" data-uri="info:doi/10.1371/journal.pone.0001141.t004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001141.t004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001141.t004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001141.t004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001141.t004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001141.t004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001141.t004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001141.t004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001141.t004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001141.t004.TIF"></span>)
                </a></li></ul></div><p><strong>Table 4.  <span>Analysis of Molecular Variance using mtDNA haplogroups between groups of populations categorized on the basis of geography and languages</span></strong></p><span>doi:10.1371/journal.pone.0001141.t004</span></div>

<h4>Time to Most Recent Common Ancestors (TMRCA)</h4>
<a id="article1.body1.sec2.sec6.p1" name="article1.body1.sec2.sec6.p1"></a><p>The TMRCA was calculated based on mtDNA coding region (nucleotide position 577-16023) with the average sequence evolution rate as 1.26±0.08×10<sup>−8</sup> base substitutions per nucleotide per year <a href="#pone.0001141-Mishmar1">[21]</a>. The TMRCA of the haplogroups based on the full mtDNA sequence (<a href="#pone-0001141-g004">Fig 4</a>) suggest a younger age of Khasi/Northeast Indian haplogroup M (41,000 YBP) compared to what has been obtained in the other studies <a href="#pone.0001141-Sun1">[16]</a> for Indian M haplogroup (54,000 YBP). This is because of the very low age contribution from the M48 haplogroup. Reanalyzing the data by removing M48 increases the age to ~50,000 YBP which is close to what has been obtained in the other studies. The TMRCA of haplogroup M31 and M33 is ~40,000 YBP and ~50,000 YBP suggesting that M33 like M31 is an archaic lineage. The age of B7 suggests that this haplogroup has originated ~28,000 YBP in East Asia where all the other sub-haplogroups of B have been hypothesized to have originated.</p>

</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3>
<h4>Origin and spread of predominant Y-haplogroups of Northeast India</h4>
<a id="article1.body1.sec3.sec1.p1" name="article1.body1.sec3.sec1.p1"></a><p>The present study provides a comprehensive genetic analysis of the populations of Austro-Asiatic linguistic family inhabiting the Northeast Indian region, which has probably served as the corridor for the peopling of Southeast Asia. Two major haplogroups O-M95 and O-M122 and its subhaplogroups dominate the haplogroup composition of the Austro-Asiatic and Tibeto-Burman populations of Northeast India and East and Southeast Asia (<a href="#pone-0001141-t005">Table 5</a>). Indo-European groups of Northeast India lack these haplogroups. Kumar et al. <a href="#pone.0001141-Kumar2">[22]</a> suggest that the haplogroup O-M95 had its origin probably in the ancestors of Mundari, one of the major subfamilies of the Indian Austro-Asiatics inhabiting Eastern and Central regions of the country, and correlated its spread with the movement of these populations to Southeast Asia and to other parts inhabited by the Austro-Asiatics via the Northeast Indian corridor. Our results are consistent with this observation as we not only find relatively high frequency O-M95 (<a href="#pone-0001141-t005">Table 5</a>) in all the subgroups of Khasi from Northeast India, but also find a decreasing gradient of O-M95 average frequency from Mundari (53%) to Khasi (~31%) to Southeast Asian Austro-Asiatics (~23%), suggesting diffusion of O-M95 from India to Southeast Asia. Further, O-M95 is either absent or, if present, only with negligible frequency in the other linguistic groups of India. It is found to be either absent or has a negligible presence in the other Tibeto-Burman populations of Northeast India and the moderate frequency of O-M95 in the Tibeto-Burman Garo may be because of the gene flow from the neighboring Khasi which is facilitated by the practice of matrilocality in these populations. This suggests that the haplogroup O-M95 is, by and large, restricted to Austro-Asiatic populations in India and supports the contention of Kumar et al. <a href="#pone.0001141-Kumar2">[22]</a> that probably there was a concomitant spread of the ancestors of the present day AA people with proto-Austro-Asiatic language and haplogroup O-M95.</p>
<div class="figure" id="pone-0001141-t005"><div class="img"><a name="pone-0001141-t005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001141.t005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001141" data-uri="info:doi/10.1371/journal.pone.0001141.t005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001141.t005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001141.t005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001141.t005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001141.t005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001141.t005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001141.t005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001141.t005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001141.t005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001141.t005.TIF"></span>)
                </a></li></ul></div><p><strong>Table 5.  <span>Y-haplogroup frequencies in different linguistic populations of South and Southeast Asia.</span></strong></p><span>doi:10.1371/journal.pone.0001141.t005</span></div><a id="article1.body1.sec3.sec1.p2" name="article1.body1.sec3.sec1.p2"></a><p>Haplogroup O-M122 is found to be in high frequency in the Garo as well as in Khasi-Khmuic populations. However, further typing of O-M122 chromosomes suggests a high frequency of undifferentiated O-M122 among the Khasi-Khmic populations, whereas the frequency of O-M134 (one of the subhaplogroups of O-M122) is found to be much higher among the Garo. Incidentally, O-M134 is found to be in much higher frequency compared to the undifferentiated O-M122 in the other Indian Tibeto-Burman populations as well. Further, Tibeto-Burman populations of Southeast Asia also have relatively much higher frequency O-M134 compared to the Austro-Asiatics there whose undifferentiated O-M122 samples fall mostly into subhaplogroup O-M159 (<a href="#pone-0001141-t005">Table 5</a>). The presence of O-M134 in high frequency among the Tibeto-Burman populations, both from India and East/southeast Asia, strongly suggests possibility of its correlation with the migration and spread of Tibeto-Burman populations into India.</p>


<h4>Distinct origin of Khasi-Khmuic tribes</h4>
<a id="article1.body1.sec3.sec2.p1" name="article1.body1.sec3.sec2.p1"></a><p>Khasi-Khumic groups are surrounded by the predominant but ethnically similar Tibeto-Burman populations in the region and a comparison with them, particularly with the Garo which is also matrilocal like the Khasi and with whom they have contiguous distribution in Meghalaya, suggests a distinct genetic origin of the Khasi-Khmuic populations. For example, unlike the Khasi, the Tibeto-Burman Garo shows a very low frequency of O-M95 and undifferentiated O-M122 Y-haplogroups but a high frequency of differentiated O-M122 i.e. O-M134 and O-M133. Similarly, the other Tibeto-Burman groups from India show a low frequency of O-M95 and undifferentiated O-M122 but a very high frequency of O-M134 (<a href="#pone-0001141-t005">Table 5</a>). Further, while the Garo shows a high frequency of M33a (55%), it lacks M9a, MD and MD4 mtDNA haplogroups which are in high frequency among the Khasi. The other Tibeto-Burman groups from this region, although has 30% of unclassified M*, show a high presence F and its subhaplogroup, M8c, A and its subhaplogroups most of which have a negligible presence in the Austro-Asiatic Khasi populations. Moreover, M9a, MD and MD4 mtDNA haplogroups which account for ~30% of Khasi-Khmuic samples are found only with a frequency of ~10% among the Tibeto-Burman groups. Most importantly, Khasi-khumic group has ~25% new haplogroups (M48, M49, M50, M31c and M33b) which has not been reported so far from any of the Northeast Indian groups except Garo which has M48 and M33b with a combined frequency of ~7%. Thus, the composition of both the mtDNA and Y haplogroups in the Austro-Asiatic Khasi as a whole suggests their distinct origin and a separate migration vis-à-vis the Tibeto-Burman groups of this region.</p>


<h4>mtDNA landscape of the Meghalaya populations</h4>
<a id="article1.body1.sec3.sec3.p1" name="article1.body1.sec3.sec3.p1"></a><p>The Khasi-Khmuic and the Garo populations of Meghalaya essentially have three kinds of haplogroups: the commonly found South Asian haplogroups, East Asian haplogroups and new haplogroups (<a href="#pone-0001141-t006">Table 6</a> and <a href="#pone-0001141-g005">Fig 5</a>). Among the new haplogroups, B7 is found only in Khasi-Khmuic populations and it has been otherwise reported as unclassified B in Han Chinese <a href="#pone.0001141-Kong1">[17]</a> suggesting its probable origin in East Asia, as is the case with other haplogroups of B. However, all the other new haplogroups viz. M48, M49 and M50 or the redefined subhaplogroups i.e. M31c and M33b have not yet been reported from East Asia and the neighboring East Asian populations and these haplogroups needs to be assessed, especially from Myanmar region, to trace their origin and movement. Further, these haplogroups have not been reported among the 23 Indo-European and 25 Dravidian populations of the Indian subcontinent <a href="#pone.0001141-Roychoudhury1">[23]</a>–<a href="#pone.0001141-Metspalu1">[25]</a> confirming the absence of this haplogroups in other parts of India.</p>
<div class="figure" id="pone-0001141-t006"><div class="img"><a name="pone-0001141-t006" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001141.t006&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001141" data-uri="info:doi/10.1371/journal.pone.0001141.t006"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001141.t006&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001141.t006/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001141.t006/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001141.t006/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001141.t006/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001141.t006.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001141.t006/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001141.t006/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001141.t006.TIF"></span>)
                </a></li></ul></div><p><strong>Table 6.  <span>mtDNA haplogroup frequencies in different linguistic populations of South and Southeast Asia.</span></strong></p><span>doi:10.1371/journal.pone.0001141.t006</span></div><a id="article1.body1.sec3.sec3.p2" name="article1.body1.sec3.sec3.p2"></a><p>The phylogeny of M33 and M31 is quite intriguing. It is striking that these two lineages have evolved into ethnic specific branches, separated by a number of mutations, suggesting their deep antiquity. Haplogroup M31 has three subhpalogroups-M31a, M31b and M31c (<a href="#pone-0001141-g004">Fig 4</a>). While M31a is reported with a high frequency in the Greater Andmanese <a href="#pone.0001141-Thangaraj1">[3]</a> and Lodha, Chenchu and Lambadi tribal groups of India <a href="#pone.0001141-Endicott1">[18]</a>, M31b and M31c are found in Northeast India, M31b among the Rajbanshis <a href="#pone.0001141-Palanichamy1">[19]</a> and M31c with an average frequency of ~5% in the Khasi-Khmuic populations (<a href="#pone-0001141-g005">Fig 5</a>). However, this haplogroup is absent in the Tibeto-Burmans of Northeast India or among the other Indian populations. Haplogroup M33b has a total frequency of ~4% among the Khasi and otherwise it has been reported only from Rajbanshi from this region. On the other hand, haplogroup M33a, which has been reported in the Brahmins of Uttar Pradesh, India <a href="#pone.0001141-Sun1">[16]</a> and in the two populations of South India <a href="#pone.0001141-Thanseem1">[20]</a> bifurcates into two branches-one, without the motif 16316, identified in only 1 Khynriam sample and another defined by 16316 HVS-I motif and found in all the samples of M33a of Garo (~55%) and Khasi-Khumic populations of Meghalaya (~5%). However, none of the other Indian Austro-Asiatic populations (Mundari) shows M33 or its subhaplogroup (Kumar et al., unpublished results). Although we need to screen more populations in order to resolve the origin and distribution of M31 and M33, both these lineages are very old (&gt;40,000 YBP) and probably originated in mainland India as their presence has been reported only from the Indian subcontinent. Overall, the new and the redefined haplogroups, excluding M33a, account for ~25% of the total frequency in the Khasi-Khmuic populations and ~4% in the Garo, while M33a accounts for ~55% in Garo. The rest of the haplogroups are of either South Asian or East Asian types.</p>
<a id="article1.body1.sec3.sec3.p3" name="article1.body1.sec3.sec3.p3"></a><p>The East Asian haplogroups with high frequency in the Khasi-Khmuic populations (M9a, MD and MD4) account for ~35% of the samples. Along with B7, the presence of very different set of East Asian haplogroups among them, as compared to the Tibeto-Burmans of India, may suggest a strong possibility of their admixture with or assimilation of certain East Asian populations, other than the Indian Tibeto-Burmans. This is also reflected in case of Y-chromosome with high frequency of undifferentiated O-M122. The South Asian haplogroups found in Khasi-Khmuic populations are M2 and M2a, M18, M25, M3, M4a, M5, R, R8, U2 and U2abc, and U3 which account for a total of ~22% among them. However, the frequency of all these haplogroups is quite low except for M4a and U2, which account for ~7 and ~8%, respectively. The low frequency of South Asian haplogroups might be either due to admixture with the surrounding Indo-European populations or due to initial splintering of their gene pool from that of the Mundari tribes as both these linguistic groups show predominantly South Asian mtDNA haplogroups.</p>


<h4>The Austro-Asiatic tribes of Northeast India: Genetic link/continuity between South and Southeast Asian populations</h4>
<a id="article1.body1.sec3.sec4.p1" name="article1.body1.sec3.sec4.p1"></a><p>A rapid human migration through Southern route ~60,000 YBP is suggested to have brought undifferentiated M and R into South and East Asia and subsequently differentiated into different subhaplogroups in different regions <a href="#pone.0001141-Kong1">[17]</a>. For example, the undifferentiated M evolved into different haplogroups such as M2, M3, M5 M6, M18 etc. in South Asia, whereas in East Asia, primarily haplogroup M9, MD, MG etc. evolved. In this backdrop, it is interesting that the Mundari Austro-Asiatic tribes of central and eastern India have mostly South Asian haplogroups (Kumar et al. unpublished results) with a high frequency of M2b, M40a, R6 and R7, which are considered to be the old lineages. This may support the migration of these tribes into India by Southern route and using the Western Indian corridor, bringing with them the undifferentiated M and R ~50,000 YBP. The predominant lineages found among them might have evolved subsequently. However, these mtDNA haplogroups are with very low frequencies in the Khasi-Khmuic Austro-Asiatic populations from Northeast India who, nonetheless, have ~25% of their haplogroups as new ones (<a href="#pone-0001141-t006">Table 6</a>). This would be feasible only if a section of them had separated soon after the ancestors of Austro-Asiatics had come to India, and migrated quite rapidly to Northeast India and further to Southeast Asia before the differentiation of M and R into any of the haplogroups found in South Asia. The undifferentiated M and R haplogroups carried by this group might have evolved into certain characteristic haplogroups such as M48, M49 and M50 in the Khasi but not in the Mundari groups around 40,000 YBP. A section of these initial migrants to Northeast India had probably moved further to Southeast Asia in rapid succession carrying with them the undifferentiated M (before the Khasi- specific haplogroups could evolve), which might have evolved into typical East Asian haplogroups. It is evident from the foregoing discussion that the Austro-Asiatic Khasi of Northeast India represents genetic continuity, linking the populations of South and Southeast Asia. Therefore, our findings reinforce the suggestion that Northeast India has acted as a corridor for initial movement of populations, not as a barrier as suggested in a recent study <a href="#pone.0001141-Cordaux1">[6]</a>.</p>

</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4>Collection of blood samples</h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p>About 5 ml of intravenous blood samples were collected in 5 ml Tarson tubes containing EDTA as an anticoagulant from a total of 444 healthy unrelated volunteers, both males and females, after obtaining the informed written consent. Prior approval for the study was obtained from the ‘Indian Statistical Institute Review Committee for Protection of Research Risk to Humans’. These samples represent 8 subgroups of the Austro-Asiatic Khasi, besides the neighbouring Tibeto-Burman Garo tribes from Meghalaya in the Northeastern part of India (<a href="#pone-0001141-t001">Table 1</a> and <a href="#pone-0001141-g001">Figure 1</a>). While the samples for the different dialectical groups of Khasi tribe were drawn from almost all the areas of their distribution in Khasi hills, Garo samples were represented in bulk from the contiguous areas of the Lyngngam tribe, such as Rongjeng, Khonjoy, Shallang, etc. with a few samples drawn from all over the capital city of Meghalaya, Shillong.</p>


<h4>mtDNA typing</h4>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1"></a><p>The mtDNA genomes were amplified and sequenced by means of the procedures described in a recent study <a href="#pone.0001141-Kong1">[17]</a>. Sequences were edited and mutations scored relative to the revised Cambridge Reference Sequence <a href="#pone.0001141-Andrews1">[26]</a> (rCRS). Initially, HVS-I (nucleotide positions [nps] 160001–16400) was sequenced, besides typing SNPs at 10398 and 10400. The published HVS-I sequences <a href="#pone.0001141-Sun1">[16]</a>, <a href="#pone.0001141-Kivisild1">[24]</a>, <a href="#pone.0001141-Macaulay2">[27]</a>–<a href="#pone.0001141-Palanichamy2">[30]</a> were compared with our HVS-I sequences from Meghalaya in combination with the typed SNPs of this study to identify the relevant coding region SNPs, which are diagnostic of the main haplogroups and subhaplogroups within the mtDNA phylogeny. These coding regions were then selectively assayed by sequencing to obtain haplogroups and their derivatives (<a href="#pone-0001141-g005">Fig 5</a>). Representative samples from those that could not be assigned into known haplogroups were carefully chosen and complete mtDNA sequencing was done to assign/designate them to the new (sub)haplogroups (<a href="#pone-0001141-g004">Fig 4</a>).</p>


<h4>Y-chromosome typing</h4>
<a id="article1.body1.sec4.sec3.p1" name="article1.body1.sec4.sec3.p1"></a><p>The following 25 Y-SNPs which are known to detect variations in Asia were screened: RPS4Y, M210, M38, M8, M217, M93, M86, M89, M69, M82, M172, M9, M175, M122, M7, M164, M159, M121, M134, M133, M162, M95, M11, M45 and M173 <a href="#pone.0001141-Underhill1">[31]</a>–<a href="#pone.0001141-Jobling1">[33]</a>. The rooted maximum-parsimony trees of the haplogroups defined by these markers are presented in <a href="#pone-0001141-g002">Figure 2</a>. Many of the samples were typed with all the binary markers for internal check on the reliability of the typing and also to detect recurrent mutations. The nomenclature as suggested and followed by Y-chromosome consortium <a href="#pone.0001141-YCC1">[32]</a> was used. The following 6 Y-STRs loci were also typed: DYS19, DYS389I, DYS389b, DYS390, DYS391 and DYS393 (for data refer to <a href="#pone.0001141.s001">Dataset S1</a>) and the details of these loci are given at Butler et al <a href="#pone.0001141-Butler1">[34]</a>.</p>


<h4>Statistical Analysis</h4>
<a id="article1.body1.sec4.sec4.p1" name="article1.body1.sec4.sec4.p1"></a><p>Since the DYS389II allele length also contains DYS389I, for all statistical analyses a simple subtraction of DYS389I allele length from that of DYS389II was done to avoid the double-counting variation at DYS389I. The subtracted DYS389II allele is named as DYS389b. The Y-SNP and modified Y-STR data were then analyzed for haplogroup and haplotype diversity, respectively, along with their associated Standard Error by means of the software package ARLEQUIN 3.01 <a href="#pone.0001141-Excoffier1">[35]</a>. The frequencies of haplogroups constructed by binaray markers were used to compute pairwise F<sub>ST</sub> genetic distance matrix. Based on the distance matrix, MDS analysis was performed using SPSS package. The genetic structure as reflected in the distribution of Y-SNPs and STRs was further explored through AMOVA, by grouping populations based on their geography and linguistic affiliations</p>
<a id="article1.body1.sec4.sec4.p2" name="article1.body1.sec4.sec4.p2"></a><p>Haplotype and Haplogroup diversity of mtDNA and AMOVA based on haplogroup frequencies were calculated using the ARLEQUIN 3.01 <a href="#pone.0001141-Excoffier1">[35]</a>. Phylogenetic relationships between the observed haplogroups were first drawn by hand and then confirmed by using the NETWORK program <a href="#pone.0001141-Bandelt1">[36]</a>. The frequencies of haplogroups were used as input vector to compute pairwise F<sub>ST</sub> genetic distance matrix using Arlequin 3.01. Based on this distance matrix, MDS analysis was performed using SPSS package. The TMRCA of the clades and subclades and their associated SEs were calculated using their full mtDNA sequences <a href="#pone.0001141-Mishmar1">[21]</a>.</p>

</div>

<div id="section5" class="section"><a id="s5" name="s5" toc="s5" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0001141"></div><a name="pone.0001141.s001" id="pone.0001141.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001141.s001">Dataset S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1"></a><p class="preSiDOI">Six Y-STR data of all the 9 populations</p>
<p class="siDoi">doi:10.1371/journal.pone.0001141.s001</p><a id="article1.body1.sec5.supplementary-material1.caption1.p2" name="article1.body1.sec5.supplementary-material1.caption1.p2"></a><p class="postSiDOI">(0.05 MB DOC)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>This study is essentially a part of the Indian Statistical Institute project undertaken by BMR in collaboration with Centre for Cellular &amp; Molecular Biology, Hyderabad. BMR thanks Directors of both the Institutes for logistic support.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: BR BL. Performed the experiments: AR BL TN AR. Analyzed the data: VK AM. Contributed reagents/materials/analysis tools: KT BR LS. Wrote the paper: BR VK. Other: Collected blood samples: BL. Commented on the manuscript and participated in the discussions: KT LS.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0001141-CavalliSforza1" id="pone.0001141-CavalliSforza1"></a>Cavalli-Sforza LL, Menozzi P, Piazza A (1994) The history and geography of human genes. Princeton, NJ: University Press.   <ul class="find-nolinks"></ul></li><li><span class="label">2.
              </span><a name="pone.0001141-Jin1" id="pone.0001141-Jin1"></a>Jin L, Su B (2000) Natives or immigrants: modern human origin in East Asia. Nat Rev Genet  1: 126–133.  <ul class="find" data-citedArticleID="1014478"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Natives+or+immigrants%3A+modern+human+origin+in+East+Asia.&amp;auth=&amp;atitle=Natives+or+immigrants%3A+modern+human+origin+in+East+Asia." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Natives+or+immigrants%3A+modern+human+origin+in+East+Asia." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Natives+or+immigrants%3A+modern+human+origin+in+East+Asia.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0001141-Thangaraj1" id="pone.0001141-Thangaraj1"></a>Thangaraj K, Chaubey G, Kivisild T, Reddy AG, Singh VK, et al.  (2005) Reconstructing the origin of Andaman Islanders. Science  308: 996.  <ul class="find" data-citedArticleID="1014530" data-doi="10.1126/science.1109987"><li><a href="http://dx.doi.org/10.1126/science.1109987" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Reconstructing+the+origin+of+Andaman+Islanders." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Reconstructing+the+origin+of+Andaman+Islanders.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0001141-Macaulay1" id="pone.0001141-Macaulay1"></a>Macaulay V, Hill C, Achilli A, Rengo C, Clarke D, et al.  (2005) Single, rapid coastal settlement of Asia revealed by analysis of complete mitochondrial genomes. Science  308: 1034–1036.  <ul class="find" data-citedArticleID="1014494" data-doi="10.1126/science.1109792"><li><a href="http://dx.doi.org/10.1126/science.1109792" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Single%2C+rapid+coastal+settlement+of+Asia+revealed+by+analysis+of+complete+mitochondrial+genomes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Single%2C+rapid+coastal+settlement+of+Asia+revealed+by+analysis+of+complete+mitochondrial+genomes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0001141-Kumar1" id="pone.0001141-Kumar1"></a>Kumar V, Reddy BM (2003) Status of Austro-Asiatic groups in the peopling of India: An exploratory study based on the available prehistoric, Linguistic and Bioligcal evidences. J Biosci  28: 507–522.  <ul class="find" data-citedArticleID="1014490" data-doi="10.1007/bf02705125"><li><a href="http://dx.doi.org/10.1007/bf02705125" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Status+of+Austro-Asiatic+groups+in+the+peopling+of+India%3A+An+exploratory+study+based+on+the+available+prehistoric%2C+Linguistic+and+Bioligcal+evidences." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Status+of+Austro-Asiatic+groups+in+the+peopling+of+India%3A+An+exploratory+study+based+on+the+available+prehistoric%2C+Linguistic+and+Bioligcal+evidences.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0001141-Cordaux1" id="pone.0001141-Cordaux1"></a>Cordaux R, Weiss G, Saha N, Stoneking M (2004) The northeast Indian passageway: a barrier or corridor for human migrations? Mol Biol Evol  21: 1525–1533.  <ul class="find" data-citedArticleID="1014466" data-doi="10.1093/molbev/msh151"><li><a href="http://dx.doi.org/10.1093/molbev/msh151" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+northeast+Indian+passageway%3A+a+barrier+or+corridor+for+human+migrations%3F" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+northeast+Indian+passageway%3A+a+barrier+or+corridor+for+human+migrations%3F%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0001141-Su1" id="pone.0001141-Su1"></a>Su B, Xiao C, Deka R, Seielstad MT, Kangwanpong D, et al.  (2000) Y chromosome haplotypes reveal prehistorical migrations to the Himalayas. Hum. Genet  107: 582–590.  <ul class="find" data-citedArticleID="1014526" data-doi="10.1007/s004390000406"><li><a href="http://dx.doi.org/10.1007/s004390000406" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Y+chromosome+haplotypes+reveal+prehistorical+migrations+to+the+Himalayas." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Y+chromosome+haplotypes+reveal+prehistorical+migrations+to+the+Himalayas.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0001141-Krithika1" id="pone.0001141-Krithika1"></a>Krithika S, Trivedi R, Kashyap VK, Bharati P, Vasulu TS (2006) Antiquity, geographic contiguity and genetic affinity among Tibeto-Burman populations of India: a microsatellite study Ann Hum Biol  33: 26–42.  <ul class="find" data-citedArticleID="1014488" data-doi="10.1080/03014460500424043"><li><a href="http://dx.doi.org/10.1080/03014460500424043" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Antiquity%2C+geographic+contiguity+and+genetic+affinity+among+Tibeto-Burman+populations+of+India%3A+a+microsatellite+study" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Antiquity%2C+geographic+contiguity+and+genetic+affinity+among+Tibeto-Burman+populations+of+India%3A+a+microsatellite+study%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0001141-Sharma1" id="pone.0001141-Sharma1"></a>Sharma TC (1966) Researches on the prehistoric archaeology of Assam. Jr Assam Sc Soc  IX: 1–11.  <ul class="find" data-citedArticleID="1014520"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Researches+on+the+prehistoric+archaeology+of+Assam.&amp;auth=&amp;atitle=Researches+on+the+prehistoric+archaeology+of+Assam." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Researches+on+the+prehistoric+archaeology+of+Assam." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Researches+on+the+prehistoric+archaeology+of+Assam.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0001141-Sharma2" id="pone.0001141-Sharma2"></a>Sharma TC (1980)  Prehistoric archaeology in North Eastern India-A review in progress. In: Sharma TC, Majumdar DN, editors. Eastern Himalayas. New Delhi: Cosmo Publications. pp. 102–107.  <ul class="find-nolinks"></ul></li><li><span class="label">11.
              </span><a name="pone.0001141-Sharma3" id="pone.0001141-Sharma3"></a>Sharma TC (1984)  North East India in prehistoric times. In: Karotemprel S, editor. Tribes of North East India. Shillong: Vendrame Missiological Institute, Centre of Indigenous Culture. pp. 1–23.  <ul class="find-nolinks"></ul></li><li><span class="label">12.
              </span><a name="pone.0001141-Hussain1" id="pone.0001141-Hussain1"></a>Hussain Z (1991)  Who are the pre-historic dwellers of the Meghalaya plateau? In: Singh JP, Sengupta G, editors. Archaeology of North East India. New Delhi: Vikas Publishing House Pvt. Ltd. pp. 74–85.  <ul class="find-nolinks"></ul></li><li><span class="label">13.
              </span><a name="pone.0001141-Diffloth1" id="pone.0001141-Diffloth1"></a>Diffloth G (2005)  The contribution of linguistic palaeontology to the homeland of Austro-Asiatic. In: Sagart L, Blench R, Sanchez-Mazas A, editors. The Peopling of East Asia: Putting Together Archaeology, Linguistics and Genetics. London: Routledge Curzon. pp. 77–81.  <ul class="find-nolinks"></ul></li><li><span class="label">14.
              </span><a name="pone.0001141-Nei1" id="pone.0001141-Nei1"></a>Nei M (1987) Molecular evolutionary genetics. New York: Columbia University Press.   <ul class="find-nolinks"></ul></li><li><span class="label">15.
              </span><a name="pone.0001141-Sengupta1" id="pone.0001141-Sengupta1"></a>Sengupta S, Zhivotovsky LA, King R, Mehdi SQ, Edmonds CA, et al.  (2006) Polarity and temporality of high-resolution y-chromosome distributions in India identify both indigenous and exogenous expansions and reveal minor genetic influence of central asian pastoralists. Am J Hum Genet  78: 202–21.  <ul class="find" data-citedArticleID="1014518" data-doi="10.1086/499411"><li><a href="http://dx.doi.org/10.1086/499411" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Polarity+and+temporality+of+high-resolution+y-chromosome+distributions+in+India+identify+both+indigenous+and+exogenous+expansions+and+reveal+minor+genetic+influence+of+central+asian+pastoralists." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Polarity+and+temporality+of+high-resolution+y-chromosome+distributions+in+India+identify+both+indigenous+and+exogenous+expansions+and+reveal+minor+genetic+influence+of+central+asian+pastoralists.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0001141-Sun1" id="pone.0001141-Sun1"></a>Sun C, Kong QP, Palanichamy MG, Agrawal S, Bandelt HJ, et al.  (2006) The dazzling array of basal branches in the mtDNA macrohaplogroup M from India as inferred from complete genomes. Mol Biol Evol  23: 683–690.  <ul class="find" data-citedArticleID="1014528" data-doi="10.1093/molbev/msj078"><li><a href="http://dx.doi.org/10.1093/molbev/msj078" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+dazzling+array+of+basal+branches+in+the+mtDNA+macrohaplogroup+M+from+India+as+inferred+from+complete+genomes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+dazzling+array+of+basal+branches+in+the+mtDNA+macrohaplogroup+M+from+India+as+inferred+from+complete+genomes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0001141-Kong1" id="pone.0001141-Kong1"></a>Kong QP, Yao YG, Sun C, Bandelt HJ, Zhu CL, et al.  (2003) Phylogeny of East Asian mitochondrial DNA lineages inferred from complete sequences. Am J Hum Genet  73: 671–676.  <ul class="find" data-citedArticleID="1014486" data-doi="10.1086/377718"><li><a href="http://dx.doi.org/10.1086/377718" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Phylogeny+of+East+Asian+mitochondrial+DNA+lineages+inferred+from+complete+sequences." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Phylogeny+of+East+Asian+mitochondrial+DNA+lineages+inferred+from+complete+sequences.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0001141-Endicott1" id="pone.0001141-Endicott1"></a>Endicott P, Metspalu M, Stringer C, Macaulay V, Cooper A, et al.  (2006) Multiplexed SNP typing of ancient DNA clarifies the origin of andaman mtDNA haplogroups amongst south Asian tribal populations. PLoS ONE  1: e81.  <ul class="find" data-citedArticleID="1014470" data-doi="10.1371/journal.pone.0000081"><li><a href="http://dx.doi.org/10.1371/journal.pone.0000081" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Multiplexed+SNP+typing+of+ancient+DNA+clarifies+the+origin+of+andaman+mtDNA+haplogroups+amongst+south+Asian+tribal+populations." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Multiplexed+SNP+typing+of+ancient+DNA+clarifies+the+origin+of+andaman+mtDNA+haplogroups+amongst+south+Asian+tribal+populations.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0001141-Palanichamy1" id="pone.0001141-Palanichamy1"></a>Palanichamy MG, Agrawal S, Yao YG, Kong QP, Sun C, et al.  (2006) Comment on “Reconstructing the origin of Andaman islanders”. Science  311: 470.  <ul class="find" data-citedArticleID="1014506" data-doi="10.1126/science.1120176"><li><a href="http://dx.doi.org/10.1126/science.1120176" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Comment+on+%E2%80%9CReconstructing+the+origin+of+Andaman+islanders%E2%80%9D." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Comment+on+%E2%80%9CReconstructing+the+origin+of+Andaman+islanders%E2%80%9D.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0001141-Thanseem1" id="pone.0001141-Thanseem1"></a>Thanseem I, Thangaraj K, Chaubey G, Singh VK, Bhaskar LV, et al.  (2006) Genetic affinities among the lower castes and tribal groups of India: inference from Y chromosome and mitochondrial DNA. BMC Genet  7: 42.  <ul class="find" data-citedArticleID="1014534"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Genetic+affinities+among+the+lower+castes+and+tribal+groups+of+India%3A+inference+from+Y+chromosome+and+mitochondrial+DNA.&amp;auth=&amp;atitle=Genetic+affinities+among+the+lower+castes+and+tribal+groups+of+India%3A+inference+from+Y+chromosome+and+mitochondrial+DNA." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genetic+affinities+among+the+lower+castes+and+tribal+groups+of+India%3A+inference+from+Y+chromosome+and+mitochondrial+DNA." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genetic+affinities+among+the+lower+castes+and+tribal+groups+of+India%3A+inference+from+Y+chromosome+and+mitochondrial+DNA.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0001141-Mishmar1" id="pone.0001141-Mishmar1"></a>Mishmar D, Ruiz-Pesini E, Golik P, Macaulay V, Clark AG, et al.  (2003) Natural selection shaped regional mtDNA variation in humans. Proc Natl Acad Sci U S A  7: 171–6.  <ul class="find" data-citedArticleID="1014500" data-doi="10.1073/pnas.0136972100"><li><a href="http://dx.doi.org/10.1073/pnas.0136972100" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Natural+selection+shaped+regional+mtDNA+variation+in+humans." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Natural+selection+shaped+regional+mtDNA+variation+in+humans.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0001141-Kumar2" id="pone.0001141-Kumar2"></a>Kumar V, Reddy AN, Babu JP, Rao TN, Langstieh BT, et al.  (2007) Y-chromosome evidence suggests a common paternal heritage of Austro-Asiatic populations. BMC Evol Biol.  7: 47.  <ul class="find" data-citedArticleID="1014492" data-doi="10.1186/1471-2148-7-47"><li><a href="http://dx.doi.org/10.1186/1471-2148-7-47" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Y-chromosome+evidence+suggests+a+common+paternal+heritage+of+Austro-Asiatic+populations." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Y-chromosome+evidence+suggests+a+common+paternal+heritage+of+Austro-Asiatic+populations.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0001141-Roychoudhury1" id="pone.0001141-Roychoudhury1"></a>Roychoudhury S, Roy S, Basu A, Banerjee R, Vishwanathan H, et al.  (2001) Genomic structures and population histories of linguistically distinct tribal groups of India. Hum Genet  3: 339–350.  <ul class="find" data-citedArticleID="1014516" data-doi="10.1007/s004390100577"><li><a href="http://dx.doi.org/10.1007/s004390100577" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genomic+structures+and+population+histories+of+linguistically+distinct+tribal+groups+of+India." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genomic+structures+and+population+histories+of+linguistically+distinct+tribal+groups+of+India.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0001141-Kivisild1" id="pone.0001141-Kivisild1"></a>Kivisild T, Rootsi S, Metspalu M, Mastana S, Kaldma K, et al.  (2003) The genetic heritage of the earliest settlers persists both in Indian. tribal and caste populations. Am J Hum Genet  72: 313–332.  <ul class="find" data-citedArticleID="1014484" data-doi="10.1086/346068"><li><a href="http://dx.doi.org/10.1086/346068" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+genetic+heritage+of+the+earliest+settlers+persists+both+in+Indian.+tribal+and+caste+populations." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+genetic+heritage+of+the+earliest+settlers+persists+both+in+Indian.+tribal+and+caste+populations.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0001141-Metspalu1" id="pone.0001141-Metspalu1"></a>Metspalu M, Kivisild T, Metspalu E, Parik J, Hudjashov G, et al.  (2004) Most of the extant mtDNA boundaries in south and southwest Asia were likely shaped during the initial settlement of Eurasia by anatomically modern humans. BMC Genet  5: e26.  <ul class="find" data-citedArticleID="1014498"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Most+of+the+extant+mtDNA+boundaries+in+south+and+southwest+Asia+were+likely+shaped+during+the+initial+settlement+of+Eurasia+by+anatomically+modern+humans.&amp;auth=&amp;atitle=Most+of+the+extant+mtDNA+boundaries+in+south+and+southwest+Asia+were+likely+shaped+during+the+initial+settlement+of+Eurasia+by+anatomically+modern+humans." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Most+of+the+extant+mtDNA+boundaries+in+south+and+southwest+Asia+were+likely+shaped+during+the+initial+settlement+of+Eurasia+by+anatomically+modern+humans." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Most+of+the+extant+mtDNA+boundaries+in+south+and+southwest+Asia+were+likely+shaped+during+the+initial+settlement+of+Eurasia+by+anatomically+modern+humans.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0001141-Andrews1" id="pone.0001141-Andrews1"></a>Andrews RM, Kubacka I, Chinnery PF, Lightowlers RN, Turnbull DM, et al.  (1999) Reanalysis and revision of the Cambridge reference sequence for human mitochondrial DNA. Nat Genet  23: 147.  <ul class="find" data-citedArticleID="1014458" data-doi="10.1038/13779"><li><a href="http://dx.doi.org/10.1038/13779" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Reanalysis+and+revision+of+the+Cambridge+reference+sequence+for+human+mitochondrial+DNA." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Reanalysis+and+revision+of+the+Cambridge+reference+sequence+for+human+mitochondrial+DNA.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0001141-Macaulay2" id="pone.0001141-Macaulay2"></a>Macaulay V, Richards M, Hickey E, Vega E, Cruciani F, et al.  (1999) The emerging tree of West Eurasian mtDNAs: a synthesis of control-region sequences and RFLPs. Am J Hum Genet  64: 232–249.  <ul class="find" data-citedArticleID="1014496" data-doi="10.1086/302204"><li><a href="http://dx.doi.org/10.1086/302204" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+emerging+tree+of+West+Eurasian+mtDNAs%3A+a+synthesis+of+control-region+sequences+and+RFLPs." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+emerging+tree+of+West+Eurasian+mtDNAs%3A+a+synthesis+of+control-region+sequences+and+RFLPs.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0001141-QunitanaMurci1" id="pone.0001141-QunitanaMurci1"></a>Qunitana-Murci L, Semino O, Bandelt HJ, Passarino G, McElreavey K, et al.  (1999) Genetic evidence of an early exit of Homo sapies sapiens from Africa through eastern Africa. Nat Genet  23: 437–441.  <ul class="find" data-citedArticleID="1014512" data-doi="10.1038/70550"><li><a href="http://dx.doi.org/10.1038/70550" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genetic+evidence+of+an+early+exit+of+Homo+sapies+sapiens+from+Africa+through+eastern+Africa." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genetic+evidence+of+an+early+exit+of+Homo+sapies+sapiens+from+Africa+through+eastern+Africa.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0001141-Richards1" id="pone.0001141-Richards1"></a>Richards M, Macaulay V, Hickey E, Vega E, Sykes B, et al.  (2000) Tracing European founder lineages in the Near Eastern mtDNA pool. Am J Hum Genet  67: 1251–1276.  <ul class="find" data-citedArticleID="1014514" data-doi="10.1086/321197"><li><a href="http://dx.doi.org/10.1086/321197" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Tracing+European+founder+lineages+in+the+Near+Eastern+mtDNA+pool." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Tracing+European+founder+lineages+in+the+Near+Eastern+mtDNA+pool.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0001141-Palanichamy2" id="pone.0001141-Palanichamy2"></a>Palanichamy MG, Sun C, Agrawal S, Bandelt HJ, Kong QP, et al.  (2004) Phylogeny of mitochondrial DNA macrohaplogroup N in India, based on complete sequencing: implications for the peopling of South Asia. Am J Hum Genet  75: 966–978.  <ul class="find" data-citedArticleID="1014508" data-doi="10.1086/425871"><li><a href="http://dx.doi.org/10.1086/425871" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Phylogeny+of+mitochondrial+DNA+macrohaplogroup+N+in+India%2C+based+on+complete+sequencing%3A+implications+for+the+peopling+of+South+Asia." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Phylogeny+of+mitochondrial+DNA+macrohaplogroup+N+in+India%2C+based+on+complete+sequencing%3A+implications+for+the+peopling+of+South+Asia.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0001141-Underhill1" id="pone.0001141-Underhill1"></a>Underhill PA, Passarino G, Lin AA, Shen P, Mirazon Lahr M, et al.  (2001) The phylogeography of Y chromosome binary haplotypes and the origins of modern human populations. Ann Hum Genet  65: 43–62.  <ul class="find" data-citedArticleID="1014536" data-doi="10.1046/j.1469-1809.2001.6510043.x"><li><a href="http://dx.doi.org/10.1046/j.1469-1809.2001.6510043.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+phylogeography+of+Y+chromosome+binary+haplotypes+and+the+origins+of+modern+human+populations." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+phylogeography+of+Y+chromosome+binary+haplotypes+and+the+origins+of+modern+human+populations.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0001141-YCC1" id="pone.0001141-YCC1"></a>YCC (The Y Chromosome Consortium) (2002) A nomenclature system for the tree of human Y-chromosomal binary haplogroups. Genome Res  12: 339–348.  <ul class="find" data-citedArticleID="1014538" data-doi="10.1101/gr.217602"><li><a href="http://dx.doi.org/10.1101/gr.217602" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+nomenclature+system+for+the+tree+of+human+Y-chromosomal+binary+haplogroups." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+nomenclature+system+for+the+tree+of+human+Y-chromosomal+binary+haplogroups.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0001141-Jobling1" id="pone.0001141-Jobling1"></a>Jobling MA, Tyler-Smith C (2003) The human Y chromosome: an evolutionary marker comes of age. Nat Rev Genet  8: 598–612.  <ul class="find" data-citedArticleID="1014480" data-doi="10.1038/nrg1124"><li><a href="http://dx.doi.org/10.1038/nrg1124" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+human+Y+chromosome%3A+an+evolutionary+marker+comes+of+age." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+human+Y+chromosome%3A+an+evolutionary+marker+comes+of+age.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0001141-Butler1" id="pone.0001141-Butler1"></a>Butler JM, Schoske R, Vallone PM, Kline MC, Redd AJ, et al.  (2002) A novel multiplex for simultaneous amplification of 20 Y chromosome STR markers. Forensic Sci Int  129: 10–24.  <ul class="find" data-citedArticleID="1014462" data-doi="10.1016/s0379-0738(02)00195-0"><li><a href="http://dx.doi.org/10.1016/s0379-0738(02)00195-0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+novel+multiplex+for+simultaneous+amplification+of+20+Y+chromosome+STR+markers." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+novel+multiplex+for+simultaneous+amplification+of+20+Y+chromosome+STR+markers.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0001141-Excoffier1" id="pone.0001141-Excoffier1"></a>Excoffier L, Laval G, Schneider S (2005) Arlequin ver. 3.0: An integrated software package for population genetics data analysis. Evolutionary Bioinformatics Online  1: 47–50.  <ul class="find" data-citedArticleID="1014472"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Arlequin+ver.+3.0%3A+An+integrated+software+package+for+population+genetics+data+analysis.&amp;auth=&amp;atitle=Arlequin+ver.+3.0%3A+An+integrated+software+package+for+population+genetics+data+analysis." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Arlequin+ver.+3.0%3A+An+integrated+software+package+for+population+genetics+data+analysis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Arlequin+ver.+3.0%3A+An+integrated+software+package+for+population+genetics+data+analysis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">36.
              </span><a name="pone.0001141-Bandelt1" id="pone.0001141-Bandelt1"></a>Bandelt H, Forster P, Rohl A (1999) Median joining networks for inferring intraspecific phylogenies. Mol Biol Evol  16: 37–48.  <ul class="find" data-citedArticleID="1014460" data-doi="10.1093/oxfordjournals.molbev.a026036"><li><a href="http://dx.doi.org/10.1093/oxfordjournals.molbev.a026036" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Median+joining+networks+for+inferring+intraspecific+phylogenies." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Median+joining+networks+for+inferring+intraspecific+phylogenies.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">37.
              </span><a name="pone.0001141-Thangaraj2" id="pone.0001141-Thangaraj2"></a>Thangaraj K, Singh L, Reddy A, Rao V, Sehgal S, et al.  (2003) Genetic affinities of the andaman islanders, a vanishing human population. Curr Biol  13: 86–93.  <ul class="find" data-citedArticleID="1014532" data-doi="10.1016/s0960-9822(02)01336-2"><li><a href="http://dx.doi.org/10.1016/s0960-9822(02)01336-2" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genetic+affinities+of+the+andaman+islanders%2C+a+vanishing+human+population." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genetic+affinities+of+the+andaman+islanders%2C+a+vanishing+human+population.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">38.
              </span><a name="pone.0001141-Karafet1" id="pone.0001141-Karafet1"></a>Karafet T, Xu L, Du R, Wang W, Feng S, et al.  (2001) Paternal population history of East Asia: sources, patterns, and microevolutionary processes. Am J Hum Genet  69: 615–628.  <ul class="find" data-citedArticleID="1014482" data-doi="10.1086/323299"><li><a href="http://dx.doi.org/10.1086/323299" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Paternal+population+history+of+East+Asia%3A+sources%2C+patterns%2C+and+microevolutionary+processes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Paternal+population+history+of+East+Asia%3A+sources%2C+patterns%2C+and+microevolutionary+processes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">39.
              </span><a name="pone.0001141-Fucharoen1" id="pone.0001141-Fucharoen1"></a>Fucharoen G, Fucharoen S, Horai S (2001) Mitochondrial DNA polymorphisms in Thailand. J Hum Genet  3: 115–125.  <ul class="find" data-citedArticleID="1014474" data-doi="10.1007/s100380170098"><li><a href="http://dx.doi.org/10.1007/s100380170098" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mitochondrial+DNA+polymorphisms+in+Thailand." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mitochondrial+DNA+polymorphisms+in+Thailand.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">40.
              </span><a name="pone.0001141-Oota1" id="pone.0001141-Oota1"></a>Oota H, Settheetham-Ishida W, Tiwawech D, Ishida T, Stoneking M (2001) Human mtDNA and Y-chromosome variation is correlated with matrilocal versus patrilocal residence. Nat Genet  29: 20–1.  <ul class="find" data-citedArticleID="1014504" data-doi="10.1038/ng711"><li><a href="http://dx.doi.org/10.1038/ng711" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Human+mtDNA+and+Y-chromosome+variation+is+correlated+with+matrilocal+versus+patrilocal+residence." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Human+mtDNA+and+Y-chromosome+variation+is+correlated+with+matrilocal+versus+patrilocal+residence.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">41.
              </span><a name="pone.0001141-Prasad1" id="pone.0001141-Prasad1"></a>Prasad BV, Ricker CE, Watkins WS, Dixon ME, Rao BB, et al.  (2001) Mitochondrial DNA variation in Nicobarese Islanders. Hum Biol  5: 715–725.  <ul class="find" data-citedArticleID="1014510" data-doi="10.1353/hub.2001.0072"><li><a href="http://dx.doi.org/10.1353/hub.2001.0072" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mitochondrial+DNA+variation+in+Nicobarese+Islanders." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mitochondrial+DNA+variation+in+Nicobarese+Islanders.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">42.
              </span><a name="pone.0001141-Yao1" id="pone.0001141-Yao1"></a>Yao Y, Nie L, Harpending H, Fu Y, Yuan Z, et al.  (2002) Genetic relationship of Chinese ethnic populations revealed by mtDNA sequence diversity. Am J Phys Anthropol  118: 63–76.  <ul class="find" data-citedArticleID="1014540" data-doi="10.1002/ajpa.10052"><li><a href="http://dx.doi.org/10.1002/ajpa.10052" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genetic+relationship+of+Chinese+ethnic+populations+revealed+by+mtDNA+sequence+diversity." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genetic+relationship+of+Chinese+ethnic+populations+revealed+by+mtDNA+sequence+diversity.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">43.
              </span><a name="pone.0001141-Yao2" id="pone.0001141-Yao2"></a>Yao YG, Kong QP, Bandelt HJ, Kivisild T, Zhang YP (2002) Phylogeographic differentiation of mitochondrial DNA in Han Chinese. Am J Hum Genet  70: 635–651.  <ul class="find" data-citedArticleID="1014542" data-doi="10.1086/338999"><li><a href="http://dx.doi.org/10.1086/338999" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Phylogeographic+differentiation+of+mitochondrial+DNA+in+Han+Chinese." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Phylogeographic+differentiation+of+mitochondrial+DNA+in+Han+Chinese.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001141.XML" value="114851"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001141.PDF" value="627884"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001141.g001.PNG_L" value="4365299"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001141.g001.PNG_M" value="267371"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001141.g001.PNG_S" value="10226"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001141.g001.TIF" value="6269988"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001141.g001.PNG_I" value="81051"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001141.t001.PNG_L" value="94204"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001141.t001.PNG_M" value="53285"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001141.t001.PNG_S" value="9829"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001141.t001.TIF" value="497490"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001141.t001.PNG_I" value="16532"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001141.g002.PNG_L" value="5461509"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001141.g002.PNG_M" value="247695"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001141.g002.PNG_S" value="17041"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001141.g002.TIF" value="8333636"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001141.g002.PNG_I" value="142127"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001141.t002.PNG_L" value="98461"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001141.t002.PNG_M" value="57968"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001141.t002.PNG_S" value="9317"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001141.t002.TIF" value="465844"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001141.t002.PNG_I" value="17844"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001141.g003.PNG_L" value="2202774"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001141.g003.PNG_M" value="155099"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001141.g003.PNG_S" value="10108"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001141.g003.TIF" value="3859924"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001141.g003.PNG_I" value="50330"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001141.t003.PNG_L" value="90576"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001141.t003.PNG_M" value="125286"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001141.t003.PNG_S" value="16747"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001141.t003.TIF" value="407392"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001141.t003.PNG_I" value="42741"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001141.g004.PNG_L" value="4281375"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001141.g004.PNG_M" value="213896"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001141.g004.PNG_S" value="10533"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001141.g004.TIF" value="7114118"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001141.g004.PNG_I" value="67278"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001141.g005.PNG_L" value="6512233"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001141.g005.PNG_M" value="308725"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001141.g005.PNG_S" value="16171"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001141.g005.TIF" value="9829650"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001141.g005.PNG_I" value="107723"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001141.g006.PNG_L" value="2228918"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001141.g006.PNG_M" value="122109"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001141.g006.PNG_S" value="8312"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001141.g006.TIF" value="4261128"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001141.g006.PNG_I" value="38956"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001141.t004.PNG_L" value="74404"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001141.t004.PNG_M" value="104353"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001141.t004.PNG_S" value="14744"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001141.t004.TIF" value="331032"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001141.t004.PNG_I" value="34738"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001141.t005.PNG_L" value="163699"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001141.t005.PNG_M" value="96530"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001141.t005.PNG_S" value="12670"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001141.t005.TIF" value="745644"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001141.t005.PNG_I" value="30113"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001141.t006.PNG_L" value="160427"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001141.t006.PNG_M" value="90857"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001141.t006.PNG_S" value="10808"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001141.t006.TIF" value="632528"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001141.t006.PNG_I" value="28905"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001141.s001.DOC" value="52224"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001141&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001141" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001141&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0001141&volume=&issue=&title=Austro-Asiatic Tribes of Northeast India Provide Hitherto Missing Genetic Link between South and Southeast Asia&author_name=B.%20Mohan%20Reddy%2C%20B.%20T.%20Langstieh%2C%20Vikrant%20Kumar%2C%20T.%20Nagaraja%2C%20A.%20N.%20S.%20Reddy%2C%20Aruna%20Meka%2C%20A.%20G.%20Reddy%2C%20K.%20Thangaraj%2C%20Lalji%20Singh&start_page=1&end_page=12" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001141" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001141" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0001141" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001141&amp;t=Austro-Asiatic%20Tribes%20of%20Northeast%20India%20Provide%20Hitherto%20Missing%20Genetic%20Link%20between%20South%20and%20Southeast%20Asia" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001141&title=Austro-Asiatic%20Tribes%20of%20Northeast%20India%20Provide%20Hitherto%20Missing%20Genetic%20Link%20between%20South%20and%20Southeast%20Asia&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001141&amp;title=Austro-Asiatic%20Tribes%20of%20Northeast%20India%20Provide%20Hitherto%20Missing%20Genetic%20Link%20between%20South%20and%20Southeast%20Asia" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001141" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0001141&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Austro-Asiatic Tribes of Northeast India Provide Hitherto Missing Genetic Link between South and Southeast Asia';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0001141';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Austro-Asiatic%20Tribes%20of%20Northeast%20India%20Provide%20Hitherto%20Missing%20Genetic%20Link%20between%20South%20and%20Southeast%20Asia http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001141" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0001141" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">














          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Asia%22" title="Search for articles in the subject area:'Asia'"><div class="flagText">Asia</div></a>
              <div data-categoryid="40353" data-articleid="25114"
                   data-categoryname="Asia"
                   class="flagImage" title="Flag 'Asia' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22DNA+sequences%22" title="Search for articles in the subject area:'DNA sequences'"><div class="flagText">DNA sequences</div></a>
              <div data-categoryid="32003" data-articleid="25114"
                   data-categoryname="DNA sequences"
                   class="flagImage" title="Flag 'DNA sequences' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Haplogroups%22" title="Search for articles in the subject area:'Haplogroups'"><div class="flagText">Haplogroups</div></a>
              <div data-categoryid="43141" data-articleid="25114"
                   data-categoryname="Haplogroups"
                   class="flagImage" title="Flag 'Haplogroups' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22India%22" title="Search for articles in the subject area:'India'"><div class="flagText">India</div></a>
              <div data-categoryid="40329" data-articleid="25114"
                   data-categoryname="India"
                   class="flagImage" title="Flag 'India' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Mitochondrial+DNA%22" title="Search for articles in the subject area:'Mitochondrial DNA'"><div class="flagText">Mitochondrial DNA</div></a>
              <div data-categoryid="32451" data-articleid="25114"
                   data-categoryname="Mitochondrial DNA"
                   class="flagImage" title="Flag 'Mitochondrial DNA' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Population+genetics%22" title="Search for articles in the subject area:'Population genetics'"><div class="flagText">Population genetics</div></a>
              <div data-categoryid="32359" data-articleid="25114"
                   data-categoryname="Population genetics"
                   class="flagImage" title="Flag 'Population genetics' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Sociolinguistics%22" title="Search for articles in the subject area:'Sociolinguistics'"><div class="flagText">Sociolinguistics</div></a>
              <div data-categoryid="23083" data-articleid="25114"
                   data-categoryname="Sociolinguistics"
                   class="flagImage" title="Flag 'Sociolinguistics' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Y+chromosomes%22" title="Search for articles in the subject area:'Y chromosomes'"><div class="flagText">Y chromosomes</div></a>
              <div data-categoryid="17981" data-articleid="25114"
                   data-categoryname="Y chromosomes"
                   class="flagImage" title="Flag 'Y chromosomes' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=5739'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=1418'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=6311&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>


</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
