

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: The Origin of Phenotypic Heterogeneity in a Clonal Cell Population In Vitro</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0000394"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0000394" />

    <meta name="citation_title" content="The Origin of Phenotypic Heterogeneity in a Clonal Cell Population In Vitro"/>
    <meta itemprop="name" content="The Origin of Phenotypic Heterogeneity in a Clonal Cell Population In Vitro"/>

      <meta name="citation_author" content="Daniel Stockholm"/>
            <meta name="citation_author_institution" content="GENETHON–Centre National de la Recherche Scientifique (CNRS), UMR 8115, Evry, France"/>
      <meta name="citation_author" content="Rachid Benchaouir"/>
            <meta name="citation_author_institution" content="GENETHON–Centre National de la Recherche Scientifique (CNRS), UMR 8115, Evry, France"/>
      <meta name="citation_author" content="Julien Picot"/>
            <meta name="citation_author_institution" content="GENETHON–Centre National de la Recherche Scientifique (CNRS), UMR 8115, Evry, France"/>
      <meta name="citation_author" content="Philippe Rameau"/>
            <meta name="citation_author_institution" content="GENETHON–Centre National de la Recherche Scientifique (CNRS), UMR 8115, Evry, France"/>
      <meta name="citation_author" content="Thi My Anh Neildez"/>
            <meta name="citation_author_institution" content="GENETHON–Centre National de la Recherche Scientifique (CNRS), UMR 8115, Evry, France"/>
            <meta name="citation_author_institution" content="Ecole Pratique des Hautes Etudes, Paris, France"/>
      <meta name="citation_author" content="Gabriel Landini"/>
            <meta name="citation_author_institution" content="Oral Pathology Unit, School of Dentistry, The University of Birmingham, Birmingham, England"/>
      <meta name="citation_author" content="Corinne Laplace-Builhé"/>
            <meta name="citation_author_institution" content="GENETHON–Centre National de la Recherche Scientifique (CNRS), UMR 8115, Evry, France"/>
      <meta name="citation_author" content="Andras Paldi"/>
            <meta name="citation_author_institution" content="GENETHON–Centre National de la Recherche Scientifique (CNRS), UMR 8115, Evry, France"/>
            <meta name="citation_author_institution" content="Ecole Pratique des Hautes Etudes, Paris, France"/>

    <meta name="citation_date" content="2007/4/25"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0000394.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e394"/>
    <meta name="citation_issue" content="4"/>
    <meta name="citation_volume" content="2"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Cell individuality: the bistability of competence development.; citation_author=S Avery; citation_journal_title=Trends in Microbiology; citation_volume=13; citation_number=1; citation_pages=459-462; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Dynamic clustering of bacterial population.; citation_author=E Ko; citation_author=T Yomo; citation_author=I Urabe; citation_journal_title=Physica D; citation_volume=75; citation_number=2; citation_pages=81-88; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title='Spontaneous' transformation as aberrant epigenesis.; citation_author=H Rubin; citation_journal_title=Differentiation; citation_volume=53; citation_number=3; citation_pages=123-137; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=Cellular epigenetics: effects of passage history on competence of cells for "spontaneous" transformation.; citation_author=H Rubin; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=90; citation_number=4; citation_pages=10715-10719; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=Selected cell and selective microenvironment in neoplastic development.; citation_author=H Rubin; citation_journal_title=Cancer Res; citation_volume=61; citation_number=5; citation_pages=799-807; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Degrees and kinds of selection in spontaneous neoplastic transformation: an operational analysis.; citation_author=H Rubin; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=102; citation_number=6; citation_pages=9276-9281; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=High frequency of tumour cell reversion to non-tumorigenic phenotype.; citation_author=VA Lavrovsky; citation_author=MA Guvakova; citation_author=YV Lavrovsky; citation_journal_title=Eur J Cancer; citation_volume=28; citation_number=7; citation_pages=17-21; citation_date=1992; " />
      <meta name="citation_reference" content="citation_title=Reversion of UVC-induced tumorigenic human hybrid cells to the non-tumorigenic phenotype.; citation_author=C Sun; citation_author=RJ Antonionio; citation_author=JL Redpath; citation_journal_title=Eur J Cancer; citation_volume=32A; citation_number=8; citation_pages=322-327; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Evidence for a resident subset of cells with SP phenotype in the C2C12 myogenic line: a tool to explore muscle stem cell biology.; citation_author=R Benchaouir; citation_author=P Rameau; citation_author=C Decraene; citation_author=P Dreyfus; citation_author=D Israeli; citation_journal_title=Exp Cell Res; citation_volume=294; citation_number=9; citation_pages=254-268; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Global transcriptional characterization of SP and MP cells from the myogenic C2C12 cell line: effect of FGF6.; citation_author=C Decraene; citation_author=R Benchaouir; citation_author=MA Dillies; citation_author=D Israeli; citation_author=S Bortoli; citation_journal_title=Physiol Genomics; citation_volume=23; citation_number=10; citation_pages=132-149; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Exponential distribution of locomotion activity in cell cultures.; citation_author=A Czirok; citation_author=K Schlett; citation_author=E Madarasz; citation_author=T Vicsek; citation_journal_title=Physical Review Letters; citation_volume=81; citation_number=11; citation_pages=3038-3041; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Single cell motion in aggregates of embryonic cells.; citation_author=J Mombach; citation_author=J Glazier; citation_journal_title=Physical Review Letters; citation_volume=76; citation_number=12; citation_pages=3032-3035; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Inhibition of desmin expression blocks myoblast fusion and interferes with the myogenic regulators MyoD and myogenin.; citation_author=H Li; citation_author=SK Choudhary; citation_author=DJ Milner; citation_author=MI Munir; citation_author=IR Kuisk; citation_journal_title=J Cell Biol; citation_volume=124; citation_number=13; citation_pages=827-841; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=Isolation of human myoblasts with the fluorescence-activated cell sorter.; citation_author=C Webster; citation_author=GK Pavlath; citation_author=DR Parks; citation_author=FS Walsh; citation_author=HM Blau; citation_journal_title=Exp Cell Res; citation_volume=174; citation_number=14; citation_pages=252-265; citation_date=1988; " />
      <meta name="citation_reference" content="citation_title=A probabilist theory for cell differentiation, embryonic mortality and DNA C-value paradox.; citation_author=J Kupiec; citation_journal_title=Specul Sci Technol; citation_volume=6; citation_number=15; citation_pages=471-478; citation_date=1983; " />
      <meta name="citation_reference" content="citation_title=Stochastic gene expression during cell differentiation: order from disorder?; citation_author=A Paldi; citation_journal_title=Cellular and Molecular Life Sciences; citation_volume=60; citation_number=16; citation_pages=1775-1779; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Conceptual models to understand tissue stem cell organization.; citation_author=M Loeffler; citation_author=I Roeder; citation_journal_title=Current Opinion in Hematology; citation_volume=11; citation_number=17; citation_pages=81-87; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Morphogenesis, plasticity and irreversibility.; citation_author=C Furusawa; citation_author=K Kaneko; citation_journal_title=Int J Dev Biol; citation_volume=50; citation_number=18; citation_pages=223-232; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Modeling embryogenesis and cancer: an approach based on an equilibrium between the autostabilization of stochastic gene expression and the interdependence of cells for proliferation.; citation_author=B Laforge; citation_author=D Guez; citation_author=M Martinez; citation_author=JJ Kupiec; citation_journal_title=Prog Biophys Mol Biol; citation_volume=89; citation_number=19; citation_pages=93-120; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Clonal heterogeneity in the germinal zone of the developing rat telencephalon.; citation_author=SE Acklin; citation_author=D van der Kooy; citation_journal_title=Development; citation_volume=118; citation_number=20; citation_pages=175-192; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=The somatic mutation theory of cancer: growing problems with the paradigm?; citation_author=AM Soto; citation_author=C Sonnenschein; citation_journal_title=Bioessays; citation_volume=26; citation_number=21; citation_pages=1097-1107; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Stem cells and their niches.; citation_author=K Moore; citation_author=I Lemischka; citation_journal_title=Science; citation_volume=311; citation_number=22; citation_pages=1880-1885; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Combination of quantification and observation methods for study of ”side population”cells in their “in vitro” microenvironment.; citation_author=RP Benchaouir; citation_author=J Greppo; citation_author=N Rameau; citation_author=PH Stockholm; citation_author=D Garcia1; citation_journal_title=Cytometry A;; citation_number=23; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Modelling spatial patterns (with discussion).; citation_author=B Ripley; citation_journal_title=J. Roy. Stat. Soc. B.; citation_volume=39; citation_number=24; citation_pages=172-212; citation_date=1977; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000394" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="The Origin of Phenotypic Heterogeneity in a Clonal Cell Population In Vitro"/>
    <meta name="twitter:description" content="BackgroundThe spontaneous emergence of phenotypic heterogeneity in clonal populations of mammalian cells in vitro is a rule rather than an exception. We consider two simple, mutually non-exclusive models that explain the generation of diverse cell types in a homogeneous population. In the first model, the phenotypic switch is the consequence of extrinsic factors. Initially identical cells may become different because they encounter different local environments that induce adaptive responses. According to the second model, the phenotypic switch is intrinsic to the cells that may occur even in homogeneous environments.Principal FindingsWe have investigated the &ldquo;extrinsic&rdquo; and the &ldquo;intrinsic&rdquo; mechanisms using computer simulations and experimentation. First, we simulated in silico the emergence of two cell types in a clonal cell population using a multiagent model. Both mechanisms produced stable phenotypic heterogeneity, but the distribution of the cell types was different. The &ldquo;intrinsic&rdquo; model predicted an even distribution of the rare phenotype cells, while in the &ldquo;extrinsic&rdquo; model these cells formed small clusters. The key predictions of the two models were confronted with the results obtained experimentally using a myogenic cell line.ConclusionsThe observations emphasize the importance of the &ldquo;ecological&rdquo; context and suggest that, consistently with the &ldquo;extrinsic&rdquo; model, local stochastic interactions between phenotypically identical cells play a key role in the initiation of phenotypic switch. Nevertheless, the &ldquo;intrinsic&rdquo; model also shows some other aspects of reality: The phenotypic switch is not triggered exclusively by the local environmental variations, but also depends to some extent on the phenotypic intrinsic robustness of the cells."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0000394.g007"/>

  <meta property="og:title" content="The Origin of Phenotypic Heterogeneity in a Clonal Cell Population In Vitro" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000394" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=2373'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=1416'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=7284&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0000394">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000394" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000394&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    The Origin of Phenotypic Heterogeneity in a Clonal Cell Population In Vitro
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Daniel Stockholm
              <span class="equal-contrib"
                    title="These authors contributed equally to this work">equal contributor</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">

                <p><span class="equal-contrib" title="These authors contributed equally to this work">equal contributor</span>
                Contributed equally to this work with: Daniel Stockholm, Rachid Benchaouir</p>

              
              

                <p>Affiliation:
                  GENETHON–Centre National de la Recherche Scientifique (CNRS), UMR 8115, Evry, France
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Rachid Benchaouir
              <span class="equal-contrib"
                    title="These authors contributed equally to this work">equal contributor</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">

                <p><span class="equal-contrib" title="These authors contributed equally to this work">equal contributor</span>
                Contributed equally to this work with: Daniel Stockholm, Rachid Benchaouir</p>

              
              

                <p>Affiliation:
                  GENETHON–Centre National de la Recherche Scientifique (CNRS), UMR 8115, Evry, France
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Julien Picot, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  GENETHON–Centre National de la Recherche Scientifique (CNRS), UMR 8115, Evry, France
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Philippe Rameau, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  GENETHON–Centre National de la Recherche Scientifique (CNRS), UMR 8115, Evry, France
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Thi My Anh Neildez, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliations:
                  GENETHON–Centre National de la Recherche Scientifique (CNRS), UMR 8115, Evry, France, 
                  Ecole Pratique des Hautes Etudes, Paris, France
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Gabriel Landini, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Oral Pathology Unit, School of Dentistry, The University of Birmingham, Birmingham, England
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Corinne Laplace-Builhé, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  GENETHON–Centre National de la Recherche Scientifique (CNRS), UMR 8115, Evry, France
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Andras Paldi
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:paldi@genethon.fr">paldi@genethon.fr</a></p>

                <p>Affiliations:
                  GENETHON–Centre National de la Recherche Scientifique (CNRS), UMR 8115, Evry, France, 
                  Ecole Pratique des Hautes Etudes, Paris, France
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: April 25, 2007</li>
    <li>DOI: 10.1371/journal.pone.0000394</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0000394" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0000394" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000394" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0000394" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0000394-g001" data-doi="info:doi/10.1371/journal.pone.0000394" data-uri="info:doi/10.1371/journal.pone.0000394.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000394.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000394-g002" data-doi="info:doi/10.1371/journal.pone.0000394" data-uri="info:doi/10.1371/journal.pone.0000394.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000394.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000394-g003" data-doi="info:doi/10.1371/journal.pone.0000394" data-uri="info:doi/10.1371/journal.pone.0000394.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000394.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000394-g004" data-doi="info:doi/10.1371/journal.pone.0000394" data-uri="info:doi/10.1371/journal.pone.0000394.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000394.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000394-g005" data-doi="info:doi/10.1371/journal.pone.0000394" data-uri="info:doi/10.1371/journal.pone.0000394.g005" title="Figure 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000394.g005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000394-g006" data-doi="info:doi/10.1371/journal.pone.0000394" data-uri="info:doi/10.1371/journal.pone.0000394.g006" title="Figure 6">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000394.g006&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000394-g007" data-doi="info:doi/10.1371/journal.pone.0000394" data-uri="info:doi/10.1371/journal.pone.0000394.g007" title="Figure 7">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000394.g007&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000394">Reader Comments (1)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0000394" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2>
<h3>Background</h3>
<a id="article1.front1.article-meta1.abstract1.sec1.p1" name="article1.front1.article-meta1.abstract1.sec1.p1"></a><p>The spontaneous emergence of phenotypic heterogeneity in clonal populations of mammalian cells <em>in vitro</em> is a rule rather than an exception. We consider two simple, mutually non-exclusive models that explain the generation of diverse cell types in a homogeneous population. In the first model, the phenotypic switch is the consequence of extrinsic factors. Initially identical cells may become different because they encounter different local environments that induce adaptive responses. According to the second model, the phenotypic switch is intrinsic to the cells that may occur even in homogeneous environments.</p>


<h3>Principal Findings</h3>
<a id="article1.front1.article-meta1.abstract1.sec2.p1" name="article1.front1.article-meta1.abstract1.sec2.p1"></a><p>We have investigated the “extrinsic” and the “intrinsic” mechanisms using computer simulations and experimentation. First, we simulated <em>in silico</em> the emergence of two cell types in a clonal cell population using a multiagent model. Both mechanisms produced stable phenotypic heterogeneity, but the distribution of the cell types was different. The “intrinsic” model predicted an even distribution of the rare phenotype cells, while in the “extrinsic” model these cells formed small clusters. The key predictions of the two models were confronted with the results obtained experimentally using a myogenic cell line.</p>


<h3>Conclusions</h3>
<a id="article1.front1.article-meta1.abstract1.sec3.p1" name="article1.front1.article-meta1.abstract1.sec3.p1"></a><p>The observations emphasize the importance of the “ecological” context and suggest that, consistently with the “extrinsic” model, local stochastic interactions between phenotypically identical cells play a key role in the initiation of phenotypic switch. Nevertheless, the “intrinsic” model also shows some other aspects of reality: The phenotypic switch is not triggered exclusively by the local environmental variations, but also depends to some extent on the phenotypic intrinsic robustness of the cells.</p>

</div>


<div class="articleinfo"><p><strong>Citation: </strong>Stockholm D, Benchaouir R, Picot J, Rameau P, Neildez TMA, et al.  (2007) The Origin of Phenotypic Heterogeneity in a Clonal Cell Population In Vitro. PLoS ONE 2(4):
          e394.
            doi:10.1371/journal.pone.0000394</p><p><strong>Academic Editor: </strong>Sui Huang, Children's Hospital Boston, United States of America</p><p><strong>Received:</strong> September 18, 2006; <strong>Accepted:</strong> April 2, 2007; <strong>Published:</strong> April 25, 2007</p><p><strong>Copyright:</strong> © 2007 Stockholm et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>This work was funded by the Association Française contre les Myopathies, Genopole (Evry), Conseil Régional de l'Essonne and the Fondation pour la Recherche Médicale.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>Phenotypic heterogeneity in genetically homogenous cell populations is frequently observed in <em>in vitro</em> cell cultures. Some cells within genetically uniform populations of various organisms such as bacteria or yeast exhibit striking phenotypic variability <a href="#pone.0000394-Avery1">[1]</a>, <a href="#pone.0000394-Ko1">[2]</a>. In <em>in vitro</em> mammalian cellular systems the, spontaneous emergence of phenotypic heterogeneity is a rule rather than an exception. Phenotypic heterogeneity is systematically observed in cultures of various lines of genetically identical cells even in controlled environments. These different phenotypes can be strikingly different; for example, non malignant cells can spontaneously produce neoplastic subclones at a frequency dependent on the culture conditions. <a href="#pone.0000394-Rubin1">[3]</a>–<a href="#pone.0000394-Rubin4">[6]</a>. Conversely, cells in isogenic populations of malignant cell lines in culture can spontaneously revert to stable non-malignant phenotypes.<a href="#pone.0000394-Lavrovsky1">[7]</a>, <a href="#pone.0000394-Sun1">[8]</a>.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>Since cell lines and primary cells are commonly used to investigate regulatory processes and gene expression during cell differentiation, understanding phenotypic differentiation in clonal populations is a fundamental problem in biology. There are two mutually non exclusive basic models for this phenomenon. In the first model the phenotypic switch is thought to occur as the consequence of extrinsic factors. Two initially identical cells may become different because they encounter different local environments that induce alternative adaptive responses. By changing its phenotype the cell itself contributes changes of the local microenvironment and thus elicits responses from the surrounding cells that are likely to lead to continuous dynamic changes in the population. According to the second model, the phenotype switch is intrinsic to the cells. The phenotypic changes therefore may occur even in an homogenous environment and may result from asymmetric segregation of intrinsic fate determinants during cell division that lead to the change in gene expression patterns.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>These two hypotheses can be investigated in cell cultures. One example studied in our laboratory showed that two subpopulations appear spontaneously in proliferating C2C12 mouse myogenic cells <a href="#pone.0000394-Benchaouir1">[9]</a>. The main population (MP) represents 95 to 99% of the cells and is characterized by a relatively low level of <em>Mdr1</em> activity. The remaining 5 to 1% of the cells have relatively high levels of <em>Mdr1</em> activity and constitute the so-called side population (SP). Transcriptome analyses showed distinct expression levels of several genes in the two cell subtypes <a href="#pone.0000394-Decraene1">[10]</a> suggesting that the SP and the MP cells may represent different stages in the differentiation process. Phenotypically, the SP cells of the C2C12 cell line display a number of features reminiscent of stem cells (similarly to other SP cells which were identified by the same criteria of fluorescent dye exclusion in various cell lines or primary cell cultures). On the other hand, MP cells are more like the cells committed to differentiate into myotubes.</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p>In the present study we investigated the “intrinsic” and “extrinsic” hypotheses of phenotypic differentiation using simple multi-agent computer modelling. In this approach each cell is considered as an autonomous “agent”. Only the rules determining the action of individual agents are defined, while the behaviour of the whole system emerges from the collective behaviour of all agents. The computer simulations showed that both the “intrinsic” and “extrinsic” models could in principle produce heterogeneous populations, but the spatial distribution of the different cell types was different across the two models. The predictions made by the models concerning the distribution of the different cell types were compared with the spatial distribution of the SP and MP cells in the C2C12 cell line. The results indicate that both models capture certain aspects of the <em>in vitro</em> situation, and at the same time underline the importance of the microenvironment and local intercellular interactions that may be related to phenotypic switching. .</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3><a id="article1.body1.sec2.p1" name="article1.body1.sec2.p1"></a><p>We have designed a simple agent-based model to analyse the possible outcomes of the “extrinsic” and the “intrinsic” mechanisms of phenotypic differentiation. Agent-based models follow a bottom-up simulation strategy, defining simple rules that govern the behavior of individual agents (the cells, in our case) without a global view of the whole system. In the next sections we the describe the general model, followed by the intrinsic and extrinsic variants of the growth model.</p>

<h4>The basic parameters in the model</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>Our models are based on a limited number of simplified assumptions about how individual cells migrate, interact with each other, divide and die. These universal features were deduced from direct observations on real life cultures of myogenic C2C12 cells and their control values (i.e. model parameters) were chosen to represent observed realistic ranges.</p>
<a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2"></a><p>Since cell migration plays a crucial role in the model, the migration characteristics were defined on the basis of videomicroscopic observations of growing C2C12 cell cultures (more than 3000 cell velocity values over an 18h period). In accordance with earlier reports <a href="#pone.0000394-Czirok1">[11]</a>, <a href="#pone.0000394-Mombach1">[12]</a> we found that cells migrate randomly (<a href="#pone-0000394-g001">Fig. 1B</a>) and that the cumulative velocity magnitudes (at a given time) as well as the velocities of the same cell over long periods, follow an exponential distribution (<a href="#pone-0000394-g001">Fig. 1C</a>). Therefore, the migration velocities of the simulated cells were generated using an exponential probability distribution function.</p>
<div class="figure" id="pone-0000394-g001"><div class="img"><a name="pone-0000394-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000394.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000394" data-uri="info:doi/10.1371/journal.pone.0000394.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000394.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000394.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000394.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000394.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000394.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000394.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000394.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000394.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000394.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>The basic parameters in the model.</span></strong></p><a id="article1.body1.sec2.sec1.fig1.caption1.p1" name="article1.body1.sec2.sec1.fig1.caption1.p1"></a><p><strong>A</strong>: Multiagent computer simulation of the “extrinsic” and “intrinsic” mechanisms. Cells migrate and divide in the same way in the two models, and cell death is a function of the local cell density. The phenotypic switch of each cell is either dependent on the local cell density in the “extrinsic” (left) or a fixed probability in the “intrinsic” (right) model.” N” is the number of neighbours in the circle with a radius “R”; “p<sub>d</sub>” is the probability of division; p<sub>1</sub> and p<sub>2</sub> are the probabilities of the A and B type cells to change their phenotype in the “intrinsic” model; N<sub>ex</sub> is the threshold, given by the number of neighbours. <strong>B</strong>: Characteristics of the cell migration in the computer model and <em>in vitro</em>, as observed in the cultures of the C2C12 cell line. The trajectories of a single cell simulated <em>in silico</em> (left) and its <em>in vitro</em> counterpart (right, determined by video microscopy), are shown. <strong>C</strong>: The exponential distribution of the cumulative velocity magnitudes in the simulation (left) and <em>in vitro</em>, as determined experimentally (right).</p>
<span>doi:10.1371/journal.pone.0000394.g001</span></div><a id="article1.body1.sec2.sec1.p3" name="article1.body1.sec2.sec1.p3"></a><p>In the model, all cells divided at each iteration step, but the survival of the daughter cells depended on the local cell density. The highest local density, N<sub>max death</sub>, above which the daughter cell cannot survive (i.e. it dies) was defined as the number of neighbours within a circular region around the cell with radius R. The number of neighbours for each cell is therefore determined at each iteration step. In all simulations the arbitrary value of R = 1 was used-it is important to note that the actual value of R gains a meaning only in relation to the value of N<sub>max death</sub> because the cells <em>in silico</em> have no size (i.e. they are represented by a point). As a result, the values of R = 1 and N<sub>max death</sub> = D give similar results to R = 2 and N<sub>max death</sub> = 4xD. In an early version of the model the cells always died when the value of N<sub>max death</sub> was reached. This led to the formation of large empty patches (never observed in real cell cultures), so in order to make the model more realistic, we implemented cell death as a probabilistic event. The probability that a cell will die, p<sub>death</sub> increases from 0 to 1 following a sigmoidal curve as a function of N<sub>max death</sub>. As a result, the excessive cell density variations in the virtual cell population disappeared. Although not confirmed by direct observation, the assumption that cell death will depend on the local cell density is intuitively easy to understand: cell density correlates with the gradient of nutrients, oxygen and toxic metabolites making less likely in dense regions than in sparse ones.</p>
<a id="article1.body1.sec2.sec1.p4" name="article1.body1.sec2.sec1.p4"></a><p>We explored the parameter space defined by the migration velocity and N<sub>max death</sub>. Values from 40 to100 cells/R for N<sub>max death</sub> provided a cell density that approximated the observed maximal number of neighbours in a circle with a radius of 8 to 20 µm in real cultures of C2C12 cells (close to confluence). The average cell migration velocity emerged as a crucial parameter for the model. Low average velocity values (&lt;0.2) led to the formation of small clusters of cells separated by empty strips. Because such patterns were never observed in real cultures and the real cell average velocity was higher than 0.2, we programmed velocity values between 0.3 and 1.0. The observed cell distribution pattern was random at all N<sub>max death</sub> values, as estimated by the nearest neighbour distance method (see <a href="#s4">Materials and Methods</a>).</p>
<a id="article1.body1.sec2.sec1.p5" name="article1.body1.sec2.sec1.p5"></a><p>The “extrinsic” and the “intrinsic” hypotheses for the generation of cellular heterogeneity were implemented varying the parameters defined above. We assumed the existence of two cell types: A and B. In both cases, the same rules for migration, division and death (as described above) were applied to type A (representing the SP cells) and B (representing the MP cells).Type A cells can differentiate into B cells and <em>vice versa</em> under conditions defined by the hypothesis under test (“intrinsic” <em>versus</em> “extrinsic”; <a href="#pone-0000394-g001">Fig. 1A</a>). In spite of the generality of the model, we focused our attention on a range of model parameters where the A cells are in minority, because in the experimental system used for the verification of the predictions in the second part of this work one cell type, the stem cell-like SP cells, represent only 1 to 2% of the whole population.</p>


<h4>The “intrinsic” model</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>In the “intrinsic” model we addressed the question of cell autonomy of the phenotypic switch. At each simulation step, type A cells have a fixed probability p<sub>AtoB</sub> to transform into B cells, and B cells have a probability p<sub>BtoA</sub> to become A cells (the environment plays no role in the switching). The simulations start with a single A cell which replicates to fill the available space, reaching a maximum size at an equilibrium between growth and death. B cells appear with a frequency determined by p<sub>AtoB</sub>. We explored the effects of varying the values of p<sub>AtoB</sub> and p<sub>BtoA</sub> between 0 and 0.5. If p<sub>BtoA</sub> = 0, B cells overgrow A cells and the whole population becomes B type. If B cells can switch to A (p<sub>BtoA</sub>≠0), the size of the two subpopulations, [A] and [B], reaches a steady state equilibrium with a ratio [A]/[B] that depends on p<sub>BtoA</sub>/p<sub>AtoB</sub>. (<a href="#pone-0000394-g002">Fig. 2A</a>). The spacial randomness of cell distribution was estimated by the calculation of the standardized nearest neighbour distance (<em>w</em>). The two cell types are distributed randomly in the population both during a) the growth phase and b) at the equilibrium when velocity values are higher than 0.2 (<a href="#pone-0000394-g002">Fig. 2B</a>). This suggests that the random element in cell type spatial distribution is a generic property of the “intrinsic” model.</p>
<div class="figure" id="pone-0000394-g002"><div class="img"><a name="pone-0000394-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000394.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000394" data-uri="info:doi/10.1371/journal.pone.0000394.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000394.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000394.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000394.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000394.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000394.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000394.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000394.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000394.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000394.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>The “intrinsic” model.</span></strong></p><a id="article1.body1.sec2.sec2.fig1.caption1.p1" name="article1.body1.sec2.sec2.fig1.caption1.p1"></a><p><strong>A</strong>: Results of the “intrinsic” model simulations for large values for p<sub>BtoA</sub>. Note corresponding increase in proportion of type A cells (in red) and their random distribution. <strong>B</strong>: Analysis of type A (left panel) and type B (right panel) cell distributions in the “intrinsic” model as a function of average migration velocity using the standardized nearest neighbour distance (<em>w</em>). If <em>w</em> = 1, the cells are randomly distributed. Small standardized nearest neighbour distances (<em>w</em>&lt;1) indicate clustering; this is only observed for B cells with very low average migration velocities (&lt;0.2). In these examples p<sub>AtoB</sub> = 0.7 and p<sub>BtoA</sub> = 0.02, but similar results were obtained for other values of p.</p>
<span>doi:10.1371/journal.pone.0000394.g002</span></div><a id="article1.body1.sec2.sec2.p2" name="article1.body1.sec2.sec2.p2"></a><p>We were most interested in the case where the ratio [A]/[B] is small, because it corresponds to the (real) cultured examined (see later). The <a href="#pone-0000394-g003">Fig. 3A</a> shows the results of a typical run, where we set p<sub>AtoB</sub> = 0.70 and p<sub>BtoA</sub> = 0.02. As a result of the high probability of A to B conversion and the relatively low probability of the B to A reversion, A cells become the rare phenotype, representing a small fraction of the population. Obviously, this is symmetrical to the case where p<sub>AtoB</sub> = 0.02 and p<sub>BtoA</sub> = 0.70. As indicated above, the A and B cells are distributed randomly at equilibrium (<a href="#pone-0000394-g003">Fig. 3C</a>). As expected, there was no significant difference in the number of neighbours between the A and B cells (<a href="#pone-0000394-g003">Fig. 3B</a>). The lack of significant clustering of the A and B cells was confirmed by the Ripley's L function analysis.</p>
<div class="figure" id="pone-0000394-g003"><div class="img"><a name="pone-0000394-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000394.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000394" data-uri="info:doi/10.1371/journal.pone.0000394.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000394.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000394.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000394.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000394.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000394.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000394.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000394.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000394.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000394.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Results of a typical simulations.</span></strong></p><a id="article1.body1.sec2.sec2.fig2.caption1.p1" name="article1.body1.sec2.sec2.fig2.caption1.p1"></a><p><strong>A</strong>: Results of a typical run at the exponential growth phase and at equilibrium (type A cells are red and type B cells are green). Left panels: “intrinsic” phenotypic switch; right panels: “extrinsic” phenotypic switch.. In all simulations N<sub>max death</sub> = 40 was used. <strong>B</strong>: The distribution of the number of neighbours around the A and B cells (upper and lower panels, respectively) in the “extrinsic” (left) and “intrinsic” (right) models. The average number of neighbours and the standard deviation is indicated for each panel. <strong>C</strong>: Analysis of the spatial randomness of cell distribution at equilibrium using Ripley's L statistics. Ripley's L a point pattern with those generated by a homogeneous Poisson process. The plot shows the estimate of L(h) for various values of R ( = radius of a circle around the cell), and compares them to the line y = 0 (expected in a homogeneous Poisson process). An envelope defining the confidence interval is obtained from the maximum and minimum L(h) estimates of a large number of Monte Carlo simulations. The point set is significantly clustered in the range of scales where the estimated L(h) values are larger than 0 and lay outside the region defined by the envelope. This is the case for type A cells when considering small distances in the “extrinsic” model (red line). However, there is no significant clustering of type A cells (red line) in the “intrinsic” model and no clustering of type B cells (green line) in any of the two models, because all L(h) values are close to 0 and lay inside the region defined by the envelope (black lines).</p>
<span>doi:10.1371/journal.pone.0000394.g003</span></div>

<h4>The “extrinsic” model</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>In the “extrinsic” variant of the model, the phenotypic switch was programmed as a consequence of the change in the cellular microenvironment. Type A cells switch to type B if the number of neighbours in a circular region with a radius R around them is higher than a fixed number N<sub>ex</sub>. Type B cells switch to type A if the local density drops below the limit N<sub>ex</sub>. The local cell density, therefore, plays a role in the induction of the phenotypic switch correspond which is similar to cell survival: cell density here is also expected to correlate with the gradient of nutrients, secreted factors, oxygen, toxic metabolites, etc. Therefore, in our context, A and B cells represent two forms of phenotypic adaptation to high- and low density environments. However, we do not make explicit hypotheses concerning the precise mechanism underlying this phenotypic switch.</p>
<a id="article1.body1.sec2.sec3.p2" name="article1.body1.sec2.sec3.p2"></a><p>A typical simulation starts with a single A cell, with type B cells appearing first at the centre of the growing population (where the cell density is the highest). Local fluctuations in cell density due to random cell movement and cell death sometimes allow B cells to switch back to type A. During the growth phase, A cells are observed on the periphery of the population (<a href="#pone-0000394-g003">Fig. 3A</a>) but when the cells fill all the available space, the subpopulations reach a dynamic equilibrium with [A]/[B] determined by the N<sub>ex</sub>/R ratio (see <a href="#pone-0000394-g004">Fig. 4 A</a>). When A cells switch to B type at low density (during the equilibrium phase) the majority of the population is composed of B cells. However, if N<sub>ex</sub> is close to N<sub>max death</sub>, the number of A and B cells is nearly equal. When compared to the “intrinsic” model, the spatial distribution of the cells in this “extrinsic” model is markedly different. The rare phenotype A cells typically form groups or clusters in areas with low local cell density for all tested values of N<sub>ex</sub>. The B cells are distributed randomly when N<sub>ex</sub>&lt;N<sub>max death</sub>, but some clustering appears when N<sub>ex</sub> approaches N<sub>max death</sub> and the sizes of the two subpopulations become comparable (not shown). We also examined the effect of migration velocity on the spatial distribution of cells and found that clustering of A cells was not affected by this parameter (<a href="#pone-0000394-g004">Fig. 4B</a>). Therefore, cluster formation seems to be a generic feature of the “extrinsic” model and robust in the range of the parameters considered.</p>
<div class="figure" id="pone-0000394-g004"><div class="img"><a name="pone-0000394-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000394.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000394" data-uri="info:doi/10.1371/journal.pone.0000394.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000394.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000394.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000394.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000394.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000394.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000394.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000394.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000394.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000394.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>The “extrinsic” model.</span></strong></p><a id="article1.body1.sec2.sec3.fig1.caption1.p1" name="article1.body1.sec2.sec3.fig1.caption1.p1"></a><p><strong>A</strong>: Simulations with the “extrinsic” model using increasing values of N<sub>ex</sub>. Note the increasing proportion and clustering of type A (red) cells with increasing N<sub>ex</sub>. In all simulations N<sub>max death</sub> = 40 was used. <strong>B</strong>: Analysis of type A cell distribution as a function of average migration velocity and varying N<sub>ex</sub> using the standardized nearest neighbour distance. Type A cells were clustered (<em>w</em>&lt;1) at all but small average velocity values at all N<sub>ex</sub> values analysed.</p>
<span>doi:10.1371/journal.pone.0000394.g004</span></div><a id="article1.body1.sec2.sec3.p3" name="article1.body1.sec2.sec3.p3"></a><p>The results of a typical run are shown in <a href="#pone-0000394-g002">Fig 2</a>. The average number of neighbours around type A and type B cells (a measure of the local cell density) is significantly different (<a href="#pone-0000394-g003">Fig. 3B</a>). The distribution of neighbours around B cells is bell-shaped, while the distribution of neighbours around A cells is truncated above the value of N<sub>ex</sub>. While the non-random distribution of type A cells in the periphery of the exponential growth phase is evident, the clustering of A cells at the equilibrium phase was demonstrated by the standardized nearest neighbour distance method (<em>w</em>). If <em>w</em> = 1, the cells are randomly distributed, small standardized nearest neighbour distances (<em>w</em>&lt;1) indicate clustering. The Ripley's test confirmed that A cells were significantly clustered, but the spatial distribution of the B cells was found random over all cell-cell distances (<a href="#pone-0000394-g003">Fig. 3C</a>).</p>


<h4>Experimental analysis</h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>The computer simulations show that both the “intrinsic” and “extrinsic” mechanisms are able to generate heterogeneous populations of cells with a stable proportion of the two cell types. The comparison of the “intrinsic” and “extrinsic” models provided testable predictions. Type A cells in the “extrinsic” model had on average fewer neighbours than B cells (<a href="#pone-0000394-g003">Fig. 3B</a>). Since the A to B switch was defined as a function of the local cell density, this was expected. An unforeseen consequence, however, was that the rare phenotype A tended to form clusters while the B cells are distributed randomly. The “intrinsic” model instead produced A cells distributed randomly and which had on average the same number of neighbours as B cells. These distinctive features in the two models was investigated experimentally using the C2C12 myogenic cell line. There are two subpopulations in the cultures of these cells: the rare stem-like SP (A cells in the model) and the myoblast-like MP cells (B cells in the model). The SP cells can differentiate into MP cells and <em>vice versa</em>. Analysis of the spatial distribution of the SP cells provides an experimental test for the model prediction, because clustering of the A cells in the “extrinsic” model was more apparent when the frequency of these cells was low. We then examined the distribution of the SP cells in cultures of C2C12 cells. To do this, cells were grown <em>in vitro</em> and stained using Hoechst 33342 dye in the culture dish. Instead of analysing the fluorescence intensity of the nuclei by cytometry, we acquired nuclear images by two photon microscopy. Since the size of the cell population was considerably large, images from separate fields were acquired and analysed as statistical samples of the whole population. The analysis was performed using image segmentation software (developed in our laboratory) which makes possible the automatic measurement of the fluorescence intensity of the nuclei while also recording the position within the culture. Since SP cells represent only a small fraction of the population, it is necessary to analyse large numbers of cells to find enough SP cells for statistical analysis. The blue and red Hoechst 33342 fluorescence intensity of 5900 cells was analysed. Only 71 (1.2%) were identified as SP cells within the limits defined by the usual criteria. The results are shown in <a href="#pone-0000394-g005">Fig. 5A</a>. Many SP cells formed small groups in the regions with relatively low cell density on the periphery of the growing cell population. However, numerous MP cells were also found in these low dense regions, indicating that low density <em>per se</em> may not be sufficient to generate the SP phenotype. On the other hand, many SP cells were found in the regions of high cell density. These cells did not form groups; they were dispersed in a high density MP environment (<a href="#pone-0000394-g003">Fig. 3A</a>). The number of neighbours around the SP cells had a bimodal frequency distribution (<a href="#pone-0000394-g005">Fig. 5B</a>), suggesting the existence of two distinct subpopulations. The frequency distribution of the neighbours (a measure of the local cell density) for SP and MP cells is significantly different (Wilcoxon-Rank-Sum test, p&lt;0.001). In order to determine whether the spatial distribution of SP cells shows significant clustering, we used Ripley's L-statistics. Since the whole cell population in the culture-dish can not be analysed on a single image due to its large size, the analysis was done on separate sample images. Significant clustering was found only for SP cells in low density regions, while the distribution in the high density regions it did not significantly differ from the uniform pattern. On <a href="#pone-0000394-g005">Fig. 5C</a> we show two examples of statistics for clustering and two for homogeneous distributions of SP cells. Interestingly, the MP cells also show clustering over a wide scale of distances, suggesting that the randomization of the pattern by cell migration and death was incomplete in the analysed growing population.</p>
<div class="figure" id="pone-0000394-g005"><div class="img"><a name="pone-0000394-g005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000394.g005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000394" data-uri="info:doi/10.1371/journal.pone.0000394.g005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000394.g005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000394.g005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000394.g005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000394.g005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000394.g005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000394.g005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000394.g005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000394.g005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000394.g005.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 5.  <span>Localization of the SP cells in the growing population of C2C12 cells.</span></strong></p><a id="article1.body1.sec2.sec4.fig1.caption1.p1" name="article1.body1.sec2.sec4.fig1.caption1.p1"></a><p><strong>A</strong>: The SP cells were identified on the basis of their capacity to exclude the fluorescent dye Hoechst 33342. Four representative images (<em>a, b, c</em> and <em>d</em>) are shown from regions with different cell densities. The nuclei are shown in false color (red for SP cells and green for MP cells). <strong>B</strong>: The frequency distribution of the number of neighbours is significantly different for SP and MP cells (upper and lower panel). The average number of neighbours is shown on the left side of each panel. Note the bimodal distribution for SP cells. The number of neighbours for each cell in a circle of R = 15 µm was calculated on the basis of the digitalized images. The two distributions were found to be significantly different (p&lt;0.001) as analysed by the non-parametric Wilcoxon-Rank-Sum test. <strong>C</strong>: Analysis of the spatial randomness of SP cell distribution using Ripley's L statistics of the four images shown in Fig. 5A. The red lines indicate the L-functions for the SP cells over a range of <em>r = </em>100. The black lines show the upper and lower limits of the envelope functions for the images analysed. The <em>c</em> and <em>d</em> patterns are significantly different from a random pattern, because the values of the observed L-function are larger than the upper envelope function while the two other L-functions (panels a and b) indicate homogeneously distributed SP cells on the corresponding images.</p>
<span>doi:10.1371/journal.pone.0000394.g005</span></div>

<h4>The “hybrid” model</h4>
<a id="article1.body1.sec2.sec5.p1" name="article1.body1.sec2.sec5.p1"></a><p>The dissimilarity in the distribution of the rare phenotype SP cells in culture and the type A cells in both the “extrinsic” and “intrinsic” models indicates that the phenotypic switch (at least in our system) may follow an intermediate scheme, where each model emulates reality only partially. One possibility is that low density <em>per se</em> is not sufficient to generate the SP phenotype in all cells because the MP and SP phenotypes are robust and are able to resist, to some extent, microenvironmental fluctuations. To test this assumption we designed a third “hybrid” model that combined the “extrinsic” and “intrinsic” assumptions. In the “hybrid” model the phenotypic switch is triggered by the microenvironment: type A cells can switch to the type B phenotype if they have more than N<sub>ex</sub> neighbours (again, within a circular region of radius R). Type B cells can switch back only if the local cell density becomes lower than the limit N<sub>ex</sub>. However, the cells encountering a favourable microenvironment undergo phenotypic change with an intrinsic probability p<sub>AtoB</sub> and p<sub>BtoA</sub> , so a fraction of cells keep their original phenotype even in a permissive microenvironment (<a href="#pone-0000394-g006">Fig. 6A</a>). If p<sub>AtoB</sub> and p<sub>BtoA</sub> = 1, the model is equivalent to the “extrisic” version. The results shown in <a href="#pone-0000394-g006">Fig. 6B</a> were obtained by using N<sub>ex</sub> = 30 values (the same as in <a href="#pone-0000394-g003">Fig. 3A</a>) and the p values were fixed at p<sub>AtoB</sub> = 0.7 and p<sub>BtoA</sub> = 0.4. Although the distribution of the number of neighbours is no longer bimodal, the simulations show a type A cell neighbour distribution (<a href="#pone-0000394-g006">Fig. 6C</a>) reminiscent of that observed for the SP cells. This could be interpreted as an indication that the “hybrid” model matches reality better.</p>
<div class="figure" id="pone-0000394-g006"><div class="img"><a name="pone-0000394-g006" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000394.g006&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000394" data-uri="info:doi/10.1371/journal.pone.0000394.g006"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000394.g006&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000394.g006/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000394.g006/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000394.g006/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000394.g006/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000394.g006.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000394.g006/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000394.g006/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000394.g006.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 6.  <span>The “hybrid extrinsic-intrinsic” model.</span></strong></p><a id="article1.body1.sec2.sec5.fig1.caption1.p1" name="article1.body1.sec2.sec5.fig1.caption1.p1"></a><p><strong>A.</strong> Cells migrate, divide and die under the same conditions as in the “extrinsic” and “intrinsic” models. The phenotypic switch of each cell is dependent on the local cell density as in the “extrinsic” model, but the cells encountering a favourable microenvironment undergo phenotypic change with probabilities p<sub>AtoB</sub> and p<sub>BtoA</sub>. B: Results of a typical simulation of the “hybrid” model during the growth phase and at equilibrium. Note the simultaneous presence of small clusters and dispersed single type A cells. p<sub>AtoB</sub> = 0.7 and p<sub>BtoA</sub> = 0.4. C: The distribution of the number of neighbours around the A and B cells (left and right respectively) in the hybrid model. The average number of neighbours and the standard deviation are indicated for each panel. Note the more dispersed distribution of type A cell neighbours. D: Analysis of the spatial distribution randomness of SP and MP cells using Ripley's L statistics. The upper panel shows the type A cell L-function (red line) with values larger than 0 and outside the range defined by the upper-and lower-envelope functions (black line) (this indicates significant clustering of type A cells at small R distances). The type B cells (green line) are randomly distributed, because the L(h) values are close to 0 at all scales (R).</p>
<span>doi:10.1371/journal.pone.0000394.g006</span></div>

<h4>Primary myoblast culture</h4>
<a id="article1.body1.sec2.sec6.p1" name="article1.body1.sec2.sec6.p1"></a><p>In order to clarify whether the non-random distribution of the rare phenotype cells is observed only in the C2C12 line or whether it is a general feature, we analysed clonal populations of primary human myoblasts. These were obtained by cloning of individual cells from a primary myoblast (muscle biopsy) culture. The cultures were allowed to grow to several hundred cells, fixed and immunostained with an anti-desmin antibody (a muscle-specific intermediate filament protein and one of the earliest markers of activated muscle precursor cells). Desmin is present in all myoblasts, but its expression level depends on the degree of commitment of the cell to the myogenic differentiation path <a href="#pone.0000394-Li1">[13]</a>. This marker is frequently used to isolate pure myoblast populations <a href="#pone.0000394-Webster1">[14]</a>. A typical isogenic population is shown on <a href="#pone-0000394-g007">Fig 7</a>. Although no accurate quantitative measures were performed, it is obvious that the level of the desmin protein was highly variable within the cluture. Cells with the lowest expression levels (presumed to be less engaged in differentiation) were observed preferentially on the periphery. These observations show that phenotypic heterogeneity can also emerge spontaneously in a clonal population of primary cells and the uneven distribution of the different cells is reminiscent of the results obtained using the C2C12 cell line.</p>
<div class="figure" id="pone-0000394-g007"><div class="img"><a name="pone-0000394-g007" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000394.g007&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000394" data-uri="info:doi/10.1371/journal.pone.0000394.g007"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000394.g007&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000394.g007/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000394.g007/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000394.g007/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000394.g007/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000394.g007.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000394.g007/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000394.g007/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000394.g007.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 7.  <span>Clonal population derived from a human primary myoblast.</span></strong></p><a id="article1.body1.sec2.sec6.fig1.caption1.p1" name="article1.body1.sec2.sec6.fig1.caption1.p1"></a><p><strong>A</strong>: The cells were fixed and immunostained with an anti-desmin antibody. A series of 55 pictures were obtained and compiled into a single picture showing the whole population. The variation in the intensity of desmin immunostain is higher at the periphery of the growing population. The left panel shows 4 high resolution images of different parts of the population pointed by the blue arrows. <strong>B</strong>: Colour coded image of the same population as on the A panel. The colour code is based on the intensity of the pixels (red: low, green: high intensity). On the left are shown 2 high resolution images with the same color code. In the upper left, a region of interest is indicated in white with the corresponding pixel histogram shown beneath. Two additional histograms show the pixel intensities of two regions of interest: high-(green) and a low-desmin expressing cells (red). Note that the low desmin expressing cells are more frequent at the periphery of the culture.</p>
<span>doi:10.1371/journal.pone.0000394.g007</span></div>
</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p>The basic model presented in this paper simulates the growth of a clonal cell population and provides a simple framework for testing the “intrinsic” and “extrinsic” hypotheses on the origin of phenotypic heterogeneity. The values of the parameters controlling cell motility, cell cycle length and cell death were deduced from observations on living cell cultures and were the same in both the “intrinsic” and the “extrinsic” versions. One could call into question the assumption that both A and B cell types follow the same rules of proliferation and migration. However, the opposite case, i.e. different proliferation and/or migration behaviour, would be more restrictive in the context of our model because it would be based on the assumption that proliferation and migration differences are part of the phenotypic differences between the two cell types. The results presented here show that even in the more general and parsimonious case of identity of proliferation and migration, phenotypic heterogeneity in a clonal cell population is possible. Differences in these characteristics between the two cell types would act to reinforce the heterogeneity.</p>
<a id="article1.body1.sec3.p2" name="article1.body1.sec3.p2"></a><p>We have examined two different hypotheses for the origin of phenotypic heterogeneity by computer simulation and experimental analysis. Computer simulations show that in the “extrinsic” model, the rare A phenotype cells were clustered in regions of low local cell densities, whereas the “intrinsic” model predicted random distribution of all cells in the cell culture.</p>
<a id="article1.body1.sec3.p3" name="article1.body1.sec3.p3"></a><p>According to the “extrinsic” model, cell differentiation depends on external cues. In the “intrinsic” model the spontaneous phenotypic switch occurs in a cell autonomous way with a given probability where intracellular interactions do not play a role. In the “intrinsic” model, since the phenotype switch is a relatively rare event, each cell type produces two daughter cells of the same type. Whether these cells remain close to each other depends on cell migratation. Random cell migration acts as a homogenizing factor that can disrupt clusters formed by cells of the same lineage. However, it also acts as the source of local heterogeneity of cell density. This constitutes a source of variability in the immediate environment of each cell such as the number of cell to cell contacts, the establishment of local concentration gradients of nutrients, oxygen or molecules secreted by the cells. Hence, in the “extrinsic” model random cell migration locally promotes the phenotypic switch.</p>
<a id="article1.body1.sec3.p4" name="article1.body1.sec3.p4"></a><p>The higher incidence of the A type cells in the low density regions in the “extrinsic” model was expected, because the phenotypic switch was conditioned by the low number of neighbours. However, clustering of the A cells in the “extrinsic” model and even distributions of these cells in the “intrinsic” model could not be predicted, based on the model's initial conditions, because the key parameter, cell migration, is stochastic. Clustering of the A cells is, therefore, a distinctive key prediction that could be analysed experimentally. It is worth noting, that in our models stable subpopulations of A and B type cells are maintained only if the phenotypic switch is reversible.</p>
<a id="article1.body1.sec3.p5" name="article1.body1.sec3.p5"></a><p>A computer simulation model of a complex phenomenon such as cell differentiation is inevitably general and based on a number of simplifications. On the other hand, an experimental system is always unique, so the experimental testing of the predictions of an abstract model must reconcile these two facets. The C2C12 cell line appeared as a good compromise for the experimental analysis of the clustering prediction. Reversible differentiation of the rare stem-like SP cells into myoblast-like MP cells and <em>vice versa</em> has been described <a href="#pone.0000394-Benchaouir1">[9]</a>. The SP–MP transformation is a true differentiation step, because both cell types differ in their morphology, gene expression pattern and biological characteristics <a href="#pone.0000394-Benchaouir1">[9]</a>, <a href="#pone.0000394-Decraene1">[10]</a>. Therefore, the A cells in the simulation share basic characteristics with the SP cells and the B cells are similar to the MP cells.</p>
<a id="article1.body1.sec3.p6" name="article1.body1.sec3.p6"></a><p>The observation that the SP cells had the tendency to be grouped in regions of low local cell density is consistent with the “extrinsic” model and stresses the importance of the local microenvironment in the initiation of the phenotypic switch. This conclusion was corroborated by the observation of position-dependent emergence of phenotypic heterogeneity in a population of primary myoblasts. Nevertheless, the “intrinsic” model also captures a part of reality, because rare phenotype cells were also found dispersed in regions of high cell density. Therefore, the “intrinsic” and “extrinsic” mechanisms are likely to correspond to two idealized solutions that act together. This is confirmed by the “hybrid” model, which provides a phenotype distribution in the population similar to that observed in the real cell culture.</p>
<a id="article1.body1.sec3.p7" name="article1.body1.sec3.p7"></a><p>What is the biological meaning underlying the “intrinsic” and “extrinsic” characteristics? We think that the “extrinsic” model refers to the cell's capacity to react to fluctuations in the environment by changing its gene expression pattern conditioned by the local concentration gradients of factors and metabolites secreted by the cells, nutrients, oxygen etc. However, the phenotypic change is not a continous transition initiated automatically by the variation of the local environment, rather it is like switch of a multistable system from one stable state to another. The cellular phenotype is robust and can resist small stochastic variation of the environment which can only induce fluctuations around the stable phenotypic state. Large fluctuations, however, destabilize the cell and generate changes. In this sense, the “intrinsic” probability in the hybrid model refers to the phenotypic robustness based on epigenetic mechanisms and transcription regulation networks rather than to a spontaneous propensity for differentiation. As a consequence, adaptation of a cell to the local microenvironment may constitute the first step in the emergence of a new cellular phenotype. Cell-to-cell interactions could then act to preferentially stabilize the new state. As a result, the cell type composition of the originally homogenous population becomes heterogeneous and tends to a steady-state equilibrium, in spite of the fact that the phenotype of each individual cell may vary. Stochastic processes are usually considered as a deleterious noise, but as it was suggested earlier <a href="#pone.0000394-Kupiec1">[15]</a>, <a href="#pone.0000394-Paldi1">[16]</a>, they can play a positive role in the process of differentiation. Our observations also provide support for this view: the population level stability is based, on one hand, on the stochastic fluctuation of the cells induced by local microenvironment and, on the other hand, on the stabilizing forces of cell interactions.</p>
<a id="article1.body1.sec3.p8" name="article1.body1.sec3.p8"></a><p>Current conceptual models tend to abandon the classical assumption of a strict hierarchy during differentiation and understand cell differentiation as a dynamic process <a href="#pone.0000394-Loeffler1">[17]</a>. For example, Kaneko and co-workers proposed on the basis of modelling studies that the differentiation of the cells is realized through an “isologous diversification” process, a mechanism by which identical state cells can diversify through the interplay between internal dynamics and external interactions <a href="#pone.0000394-Furusawa1">[18]</a>. Another modelling study has also demonstrated that auto-stabilization of stochastic processes by interdependence of cells for proliferation is essential for differentiation and pattern formation <a href="#pone.0000394-Laforge1">[19]</a>.</p>
<a id="article1.body1.sec3.p9" name="article1.body1.sec3.p9"></a><p>Although our experiments were carried out on <em>in vitro</em> cell cultures, our results may contribute to the better understanding of <em>in vivo</em> processes also. Differentiation of various cell types from a stem cell pool is a typical example of the process of phenotypic diversification <a href="#pone.0000394-Acklin1">[20]</a>. The importance of the local cellular environment and cell-to-cell communication is increasingly recognized in normal differentiation and in neoplastic transformation <a href="#pone.0000394-Soto1">[21]</a>. Terms like “niche” borrowed from ecology are more and more used to describe the impact of tissue context and microenvironment on cell differentiation <em>in vivo</em> <a href="#pone.0000394-Moore1">[22]</a>. Our results suggest that phenotypic differentiation and niche formation are tightly linked and interdependent processes; both emerge from the multitude of local interactions between the participating cells. A practical consequence of this interdependence is the inherent difficulty in establishing cultures of cells with homogenous phenotype.</p>
</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4>Analysis of cell migration velocity</h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p>Records of cell migration were done using a Zeiss Axiovert 100M confocal microscope. C2C12 cells were maintained under controlled CO<sub>2</sub> atmosphere and temperature. Bright field images were recorded every 10 minutes for 18 hours. Image acquisition was done at high resolution (1024×1024 pixels) with aid of Zeiss LSM 510 software for PC. Migration velocities were calculated with a “Manual Tracking” plug-in (Fabrice Cordelières, Institut Curie, Orsay, France) of ImageJ software (<a href="http://rsb.info.nih.gov/ij/">http://rsb.info.nih.gov/ij/</a>).</p>


<h4>Computer simulations</h4>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1"></a><p>The computer simulation was performed using the Netlogo language, specifically designed to make simple agent-based models (Wilensky, U. 1999. NetLogo. <a href="http://ccl.northwestern.edu/netlogo/">http://ccl.northwestern.edu/netlogo/</a>. Center for Connected Learning and Computer-Based Modeling, Northwestern University. Evanston, IL). The migration, division and death parameters were implemented identically in the “extrinsic” and “intrinsic” models. To implement cell migration, we defined the migration velocity as the linear distance the cell moves from its position in each step of the simulation and calculated this distance for each cell on the basis of an exponential probability distribution function determined from experimental observations. The direction of migration was considered to be random. The cells are allowed to divide at each iteration step. The probability of cell death, p<sub>death</sub> is calculated for each cell after the division. A value N<sub>max death </sub>and a standard deviation, SD of N<sub>max death</sub> is defined before each run for the maximal number of neighbours in a circle with a radius R. The value of p<sub>death</sub> increases from 0 to 1 as a function of this parameter following a sigmoidal curve. As a result, the distribution of the life lengths of the cells follows a Gaussian distribution.</p>
<a id="article1.body1.sec4.sec2.p2" name="article1.body1.sec4.sec2.p2"></a><p>The total size of the population was determined by the sum of the divisions and deaths. Although no rule was defined for growth of the whole population, it followed a typical logistic kinetics and the maximal density of the cell population in any circular area with a radius R of the virtual culture dish oscillates around the value of N<sub>max death</sub>.</p>


<h4>Cell cultures</h4>
<a id="article1.body1.sec4.sec3.p1" name="article1.body1.sec4.sec3.p1"></a><p>C2C12 cells were obtained from the American Type Culture Collection (CRL 1772) and routinely propagated in proliferation Dulbecco's Modified Eagle's Medium (DMEM, Gibco BRL) with 4,5 g/ml of glucose, supplemented with 20% (v/v) foetal calf serum (FCS, Hyclone), 100 U/ml penicillin and 100 µg/ml streptomycin. The cultures were grown at 37°C under a humidified atmosphere of air with 7% CO2. Initial plating density was between 2 .10<sup>3</sup> and 3.10<sup>3</sup> cell/cm<sup>2</sup> and the cells were cultured for 6 days. 35 mm glass bottom culture dishes (Mat Tek Corporation) were used for cell culture and two photon analyses.</p>
<a id="article1.body1.sec4.sec3.p2" name="article1.body1.sec4.sec3.p2"></a><p>The primary human mononuclear cells from muscle biopsies were obtained from the Genethon's cell bank and cultured in Skeletal muscle cell growth medium (Promocell–C-23060) supplemented with 10% calf serum (PAA–A15-043), glutamax (Gibco–35050.038) and 50 µg/ml gentamicine (Gibco–15751.037). Individual cells were cloned by cell sorter (MoFlo–Dako) and allow to expand in a colony of several hundred cells.</p>


<h4>Phenotype detection</h4>
<a id="article1.body1.sec4.sec4.p1" name="article1.body1.sec4.sec4.p1"></a><p>The C2C12 cells were stained by the DNA dye Hoechst 33342 (Sigma-Aldrich B-2261) at a final concentration of 11 µg/ml for 90 min at 37°C under mild shaking. The controls were incubated in the presence of 100 µM of verapamil (Sigma V-106 batch 28 H4699). The images were collected by a two-photon microscope under conditions of Hoechst excitation and processed by a software <a href="#pone.0000394-Benchaouir2">[23]</a> that allows the automated segmentation of the nuclei, the measurement of the fluorescence intensity of each nucleus and the recording of their position in the plate. The cells with a given fluorescence intensity can then be located easily on the recorded image of the culture plate for further analysis.</p>
<a id="article1.body1.sec4.sec4.p2" name="article1.body1.sec4.sec4.p2"></a><p>The colonies of primary cells were fixed with pure methanol (for 60 min at−20°C) and immunostained by an anti-desmin monoclonal antibody (1/40-Sigma–clone DE-U-10 Product n°D1033).</p>


<h4>Statistical analysis</h4>
<a id="article1.body1.sec4.sec5.p1" name="article1.body1.sec4.sec5.p1"></a><p>The number of neighbours around each cell in the simulations and the cell cultures was calculated automatically on the basis of the digitalized images and the distributions of the frequencies were compared by two sided t-test. The results were confirmed by the non-parametric Wilcoxon-Rank-Sum test.</p>
<a id="article1.body1.sec4.sec5.p2" name="article1.body1.sec4.sec5.p2"></a><p>The standardized nearest neighbour distance method was used to describe the degree of spatial clustering of the A and B cell distributions obtained in the “extrinsic” and “intrinsic” models. This method uses the average distance from every cell to its nearest neighbour to determine if the cells are clustered, distributed homogenously or dispersed. The standardized nearest neighbour distance, <em>w</em>, is calculated as the ratio of the average nearest neigbour distance (W) and the expected nearest neighbour distance if the cell distribution were homogenous (E[W]): <em>w</em> = W/E[W]. (W is calculated as the average of all distances to the nearest point from any point <em>i</em>. The expectation of the nearest neighbour distance of point s under the hypothesis of complete spatial randomness is a function of point density: E[W] = 1/√λ , where λ is the cell density.) If <em>w</em> = 1, the cells are randomly distributed. Small standardized nearest neighbour distances (w&lt;1) indicate clustering and <em>w</em>&gt;1 indicates dispersion.</p>
<a id="article1.body1.sec4.sec5.p3" name="article1.body1.sec4.sec5.p3"></a><p>In order to test the significance spatial clustering of the cells in the simulations and in the cell cultures the Ripley's L-function was used <a href="#pone.0000394-Ripley1">[24]</a>. It was computed using the software program Programita kindly provided by Thorsten Wiegand. Ripley's L function provides a summary of spatial dependence over a wide range of scales of pattern, including all event-event distances, not just the nearest neighbor distances. It is used to compare a point pattern with point patterns generated by known processes, e.g. a homogenous Poisson process. The test gives a plot of the estimate of L(h) at different values of R ( = radius of a circle around the cell), and compares this to the line y = 1 expected under a homogenous Poisson model. An envelope defining the confidence interval and obtained from the maximum and minimum L(h) estimates of a large number of Monte Carlo simulations allows a simple visualisation of the range of scales where clusters are observed.</p>

</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>The authors are grateful to Susan Elsevier Saint-Juste and Otto Merten for critical reading of the manuscript.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: AP DS RB TN CL. Performed the experiments: PR DS RB JP TN. Analyzed the data: AP PR DS JP CL GL. Wrote the paper: AP DS CL GL.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0000394-Avery1" id="pone.0000394-Avery1"></a>Avery S (2005) Cell individuality: the bistability of competence development. Trends in Microbiology  13: 459–462.  <ul class="find" data-citedArticleID="946618" data-doi="10.1016/j.tim.2005.08.006"><li><a href="http://dx.doi.org/10.1016/j.tim.2005.08.006" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Cell+individuality%3A+the+bistability+of+competence+development." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Cell+individuality%3A+the+bistability+of+competence+development.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0000394-Ko1" id="pone.0000394-Ko1"></a>Ko E, Yomo T, Urabe I (1994) Dynamic clustering of bacterial population. Physica D  75: 81–88.  <ul class="find" data-citedArticleID="946630" data-doi="10.1016/0167-2789(94)90276-3"><li><a href="http://dx.doi.org/10.1016/0167-2789(94)90276-3" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Dynamic+clustering+of+bacterial+population." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Dynamic+clustering+of+bacterial+population.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0000394-Rubin1" id="pone.0000394-Rubin1"></a>Rubin H (1993) 'Spontaneous' transformation as aberrant epigenesis. Differentiation  53: 123–137.  <ul class="find" data-citedArticleID="946650" data-doi="10.1111/j.1432-0436.1993.tb00652.x"><li><a href="http://dx.doi.org/10.1111/j.1432-0436.1993.tb00652.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=%27Spontaneous%27+transformation+as+aberrant+epigenesis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22%27Spontaneous%27+transformation+as+aberrant+epigenesis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0000394-Rubin2" id="pone.0000394-Rubin2"></a>Rubin H (1993) Cellular epigenetics: effects of passage history on competence of cells for "spontaneous" transformation. Proc Natl Acad Sci U S A  90: 10715–10719.  <ul class="find" data-citedArticleID="946652"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Cellular+epigenetics%3A+effects+of+passage+history+on+competence+of+cells+for+%22spontaneous%22+transformation.&amp;auth=&amp;atitle=Cellular+epigenetics%3A+effects+of+passage+history+on+competence+of+cells+for+%22spontaneous%22+transformation." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Cellular+epigenetics%3A+effects+of+passage+history+on+competence+of+cells+for+%22spontaneous%22+transformation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Cellular+epigenetics%3A+effects+of+passage+history+on+competence+of+cells+for+%22spontaneous%22+transformation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0000394-Rubin3" id="pone.0000394-Rubin3"></a>Rubin H (2001) Selected cell and selective microenvironment in neoplastic development. Cancer Res  61: 799–807.  <ul class="find" data-citedArticleID="946654"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Selected+cell+and+selective+microenvironment+in+neoplastic+development.&amp;auth=&amp;atitle=Selected+cell+and+selective+microenvironment+in+neoplastic+development." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Selected+cell+and+selective+microenvironment+in+neoplastic+development." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Selected+cell+and+selective+microenvironment+in+neoplastic+development.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0000394-Rubin4" id="pone.0000394-Rubin4"></a>Rubin H (2005) Degrees and kinds of selection in spontaneous neoplastic transformation: an operational analysis. Proc Natl Acad Sci U S A  102: 9276–9281.  <ul class="find" data-citedArticleID="946656" data-doi="10.1073/pnas.0503688102"><li><a href="http://dx.doi.org/10.1073/pnas.0503688102" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Degrees+and+kinds+of+selection+in+spontaneous+neoplastic+transformation%3A+an+operational+analysis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Degrees+and+kinds+of+selection+in+spontaneous+neoplastic+transformation%3A+an+operational+analysis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0000394-Lavrovsky1" id="pone.0000394-Lavrovsky1"></a>Lavrovsky VA, Guvakova MA, Lavrovsky YV (1992) High frequency of tumour cell reversion to non-tumorigenic phenotype. Eur J Cancer  28: 17–21.  <ul class="find" data-citedArticleID="946636" data-doi="10.1016/0959-8049(92)90375-c"><li><a href="http://dx.doi.org/10.1016/0959-8049(92)90375-c" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=High+frequency+of+tumour+cell+reversion+to+non-tumorigenic+phenotype." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22High+frequency+of+tumour+cell+reversion+to+non-tumorigenic+phenotype.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0000394-Sun1" id="pone.0000394-Sun1"></a>Sun C, Antonionio RJ, Redpath JL (1996) Reversion of UVC-induced tumorigenic human hybrid cells to the non-tumorigenic phenotype. Eur J Cancer  32A: 322–327.  <ul class="find" data-citedArticleID="946660" data-doi="10.1016/0959-8049(95)00581-1"><li><a href="http://dx.doi.org/10.1016/0959-8049(95)00581-1" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Reversion+of+UVC-induced+tumorigenic+human+hybrid+cells+to+the+non-tumorigenic+phenotype." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Reversion+of+UVC-induced+tumorigenic+human+hybrid+cells+to+the+non-tumorigenic+phenotype.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0000394-Benchaouir1" id="pone.0000394-Benchaouir1"></a>Benchaouir R, Rameau P, Decraene C, Dreyfus P, Israeli D, et al.  (2004) Evidence for a resident subset of cells with SP phenotype in the C2C12 myogenic line: a tool to explore muscle stem cell biology. Exp Cell Res  294: 254–268.  <ul class="find" data-citedArticleID="946620" data-doi="10.1016/j.yexcr.2003.11.005"><li><a href="http://dx.doi.org/10.1016/j.yexcr.2003.11.005" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evidence+for+a+resident+subset+of+cells+with+SP+phenotype+in+the+C2C12+myogenic+line%3A+a+tool+to+explore+muscle+stem+cell+biology." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evidence+for+a+resident+subset+of+cells+with+SP+phenotype+in+the+C2C12+myogenic+line%3A+a+tool+to+explore+muscle+stem+cell+biology.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0000394-Decraene1" id="pone.0000394-Decraene1"></a>Decraene C, Benchaouir R, Dillies MA, Israeli D, Bortoli S, et al.  (2005) Global transcriptional characterization of SP and MP cells from the myogenic C2C12 cell line: effect of FGF6. Physiol Genomics  23: 132–149.  <ul class="find" data-citedArticleID="946626" data-doi="10.1152/physiolgenomics.00141.2004"><li><a href="http://dx.doi.org/10.1152/physiolgenomics.00141.2004" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Global+transcriptional+characterization+of+SP+and+MP+cells+from+the+myogenic+C2C12+cell+line%3A+effect+of+FGF6." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Global+transcriptional+characterization+of+SP+and+MP+cells+from+the+myogenic+C2C12+cell+line%3A+effect+of+FGF6.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0000394-Czirok1" id="pone.0000394-Czirok1"></a>Czirok A, Schlett K, Madarasz E, Vicsek T (1998) Exponential distribution of locomotion activity in cell cultures. Physical Review Letters  81: 3038–3041.  <ul class="find" data-citedArticleID="946624" data-doi="10.1103/physrevlett.81.3038"><li><a href="http://dx.doi.org/10.1103/physrevlett.81.3038" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Exponential+distribution+of+locomotion+activity+in+cell+cultures." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Exponential+distribution+of+locomotion+activity+in+cell+cultures.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0000394-Mombach1" id="pone.0000394-Mombach1"></a>Mombach J, Glazier J (1996) Single cell motion in aggregates of embryonic cells. Physical Review Letters  76: 3032–3035.  <ul class="find" data-citedArticleID="946642" data-doi="10.1103/physrevlett.76.3032"><li><a href="http://dx.doi.org/10.1103/physrevlett.76.3032" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Single+cell+motion+in+aggregates+of+embryonic+cells." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Single+cell+motion+in+aggregates+of+embryonic+cells.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0000394-Li1" id="pone.0000394-Li1"></a>Li H, Choudhary SK, Milner DJ, Munir MI, Kuisk IR, et al.  (1994) Inhibition of desmin expression blocks myoblast fusion and interferes with the myogenic regulators MyoD and myogenin. J Cell Biol  124: 827–841.  <ul class="find" data-citedArticleID="946638" data-doi="10.1083/jcb.124.5.827"><li><a href="http://dx.doi.org/10.1083/jcb.124.5.827" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Inhibition+of+desmin+expression+blocks+myoblast+fusion+and+interferes+with+the+myogenic+regulators+MyoD+and+myogenin." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Inhibition+of+desmin+expression+blocks+myoblast+fusion+and+interferes+with+the+myogenic+regulators+MyoD+and+myogenin.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0000394-Webster1" id="pone.0000394-Webster1"></a>Webster C, Pavlath GK, Parks DR, Walsh FS, Blau HM (1988) Isolation of human myoblasts with the fluorescence-activated cell sorter. Exp Cell Res  174: 252–265.  <ul class="find" data-citedArticleID="946662" data-doi="10.1016/0014-4827(88)90159-0"><li><a href="http://dx.doi.org/10.1016/0014-4827(88)90159-0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Isolation+of+human+myoblasts+with+the+fluorescence-activated+cell+sorter." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Isolation+of+human+myoblasts+with+the+fluorescence-activated+cell+sorter.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0000394-Kupiec1" id="pone.0000394-Kupiec1"></a>Kupiec J (1983) A probabilist theory for cell differentiation, embryonic mortality and DNA C-value paradox. Specul Sci Technol  6: 471–478.  <ul class="find" data-citedArticleID="946632"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=A+probabilist+theory+for+cell+differentiation%2C+embryonic+mortality+and+DNA+C-value+paradox.&amp;auth=&amp;atitle=A+probabilist+theory+for+cell+differentiation%2C+embryonic+mortality+and+DNA+C-value+paradox." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+probabilist+theory+for+cell+differentiation%2C+embryonic+mortality+and+DNA+C-value+paradox." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+probabilist+theory+for+cell+differentiation%2C+embryonic+mortality+and+DNA+C-value+paradox.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0000394-Paldi1" id="pone.0000394-Paldi1"></a>Paldi A (2003) Stochastic gene expression during cell differentiation: order from disorder? Cellular and Molecular Life Sciences  60: 1775–1779.  <ul class="find" data-citedArticleID="946646" data-doi="10.1007/s00018-003-23147-z"><li><a href="http://dx.doi.org/10.1007/s00018-003-23147-z" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Stochastic+gene+expression+during+cell+differentiation%3A+order+from+disorder%3F" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Stochastic+gene+expression+during+cell+differentiation%3A+order+from+disorder%3F%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0000394-Loeffler1" id="pone.0000394-Loeffler1"></a>Loeffler M, Roeder I (2004) Conceptual models to understand tissue stem cell organization. Current Opinion in Hematology  11: 81–87.  <ul class="find" data-citedArticleID="946640" data-doi="10.1097/01.moh.0000133648.83991.af"><li><a href="http://dx.doi.org/10.1097/01.moh.0000133648.83991.af" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Conceptual+models+to+understand+tissue+stem+cell+organization." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Conceptual+models+to+understand+tissue+stem+cell+organization.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0000394-Furusawa1" id="pone.0000394-Furusawa1"></a>Furusawa C, Kaneko K (2006) Morphogenesis, plasticity and irreversibility. Int J Dev Biol  50: 223–232.  <ul class="find" data-citedArticleID="946628" data-doi="10.1387/ijdb.052051cf"><li><a href="http://dx.doi.org/10.1387/ijdb.052051cf" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Morphogenesis%2C+plasticity+and+irreversibility." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Morphogenesis%2C+plasticity+and+irreversibility.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0000394-Laforge1" id="pone.0000394-Laforge1"></a>Laforge B, Guez D, Martinez M, Kupiec JJ (2005) Modeling embryogenesis and cancer: an approach based on an equilibrium between the autostabilization of stochastic gene expression and the interdependence of cells for proliferation. Prog Biophys Mol Biol  89: 93–120.  <ul class="find" data-citedArticleID="946634" data-doi="10.1016/j.pbiomolbio.2004.11.004"><li><a href="http://dx.doi.org/10.1016/j.pbiomolbio.2004.11.004" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Modeling+embryogenesis+and+cancer%3A+an+approach+based+on+an+equilibrium+between+the+autostabilization+of+stochastic+gene+expression+and+the+interdependence+of+cells+for+proliferation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Modeling+embryogenesis+and+cancer%3A+an+approach+based+on+an+equilibrium+between+the+autostabilization+of+stochastic+gene+expression+and+the+interdependence+of+cells+for+proliferation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0000394-Acklin1" id="pone.0000394-Acklin1"></a>Acklin SE, van der Kooy D (1993) Clonal heterogeneity in the germinal zone of the developing rat telencephalon. Development  118: 175–192.  <ul class="find" data-citedArticleID="946616"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Clonal+heterogeneity+in+the+germinal+zone+of+the+developing+rat+telencephalon.&amp;auth=&amp;atitle=Clonal+heterogeneity+in+the+germinal+zone+of+the+developing+rat+telencephalon." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Clonal+heterogeneity+in+the+germinal+zone+of+the+developing+rat+telencephalon." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Clonal+heterogeneity+in+the+germinal+zone+of+the+developing+rat+telencephalon.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0000394-Soto1" id="pone.0000394-Soto1"></a>Soto AM, Sonnenschein C (2004) The somatic mutation theory of cancer: growing problems with the paradigm? Bioessays  26: 1097–1107.  <ul class="find" data-citedArticleID="946658" data-doi="10.1002/bies.20087"><li><a href="http://dx.doi.org/10.1002/bies.20087" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+somatic+mutation+theory+of+cancer%3A+growing+problems+with+the+paradigm%3F" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+somatic+mutation+theory+of+cancer%3A+growing+problems+with+the+paradigm%3F%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0000394-Moore1" id="pone.0000394-Moore1"></a>Moore K, Lemischka I (2006) Stem cells and their niches. Science  311: 1880–1885.  <ul class="find" data-citedArticleID="946644" data-doi="10.1126/science.1110542"><li><a href="http://dx.doi.org/10.1126/science.1110542" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Stem+cells+and+their+niches." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Stem+cells+and+their+niches.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0000394-Benchaouir2" id="pone.0000394-Benchaouir2"></a>Benchaouir RP, Greppo J, Rameau N, Stockholm PH, Garcia1 D, et al.  (2006) Combination of quantification and observation methods for study of ”side population”cells in their “in vitro” microenvironment. Cytometry A;.   in press. <ul class="find" data-citedArticleID="946622" data-doi="10.1002/cyto.a.20367"><li><a href="http://dx.doi.org/10.1002/cyto.a.20367" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Combination+of+quantification+and+observation+methods+for+study+of+%26%23x201d%3Bside+population%26%23x201d%3Bcells+in+their+%26%23x201c%3Bin+vitro%26%23x201d%3B+microenvironment." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Combination+of+quantification+and+observation+methods+for+study+of+%26%23x201d%3Bside+population%26%23x201d%3Bcells+in+their+%26%23x201c%3Bin+vitro%26%23x201d%3B+microenvironment.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0000394-Ripley1" id="pone.0000394-Ripley1"></a>Ripley B (1977) Modelling spatial patterns (with discussion). J. Roy. Stat. Soc. B.  39: 172–212.  <ul class="find" data-citedArticleID="946648"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Modelling+spatial+patterns+%28with+discussion%29.&amp;auth=&amp;atitle=Modelling+spatial+patterns+%28with+discussion%29." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Modelling+spatial+patterns+%28with+discussion%29." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Modelling+spatial+patterns+%28with+discussion%29.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000394.XML" value="77241"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000394.PDF" value="1627842"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000394.g001.PNG_L" value="285442"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000394.g001.PNG_M" value="62915"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000394.g001.PNG_S" value="4833"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000394.g001.TIF" value="737146"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000394.g001.PNG_I" value="51889"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000394.g002.PNG_L" value="2704749"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000394.g002.PNG_M" value="268305"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000394.g002.PNG_S" value="8594"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000394.g002.TIF" value="4713908"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000394.g002.PNG_I" value="128005"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000394.g003.PNG_L" value="1479474"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000394.g003.PNG_M" value="155071"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000394.g003.PNG_S" value="7216"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000394.g003.TIF" value="2498308"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000394.g003.PNG_I" value="91079"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000394.g004.PNG_L" value="1597176"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000394.g004.PNG_M" value="275559"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000394.g004.PNG_S" value="7810"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000394.g004.TIF" value="2496514"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000394.g004.PNG_I" value="108675"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000394.g005.PNG_L" value="2699627"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000394.g005.PNG_M" value="217182"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000394.g005.PNG_S" value="7874"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000394.g005.TIF" value="3874286"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000394.g005.PNG_I" value="107691"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000394.g006.PNG_L" value="810762"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000394.g006.PNG_M" value="105032"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000394.g006.PNG_S" value="5930"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000394.g006.TIF" value="1492594"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000394.g006.PNG_I" value="70831"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000394.g007.PNG_L" value="3443691"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000394.g007.PNG_M" value="266738"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000394.g007.PNG_S" value="11486"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000394.g007.TIF" value="4534156"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000394.g007.PNG_I" value="159919"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000394&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000394" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000394&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0000394&volume=&issue=&title=The Origin of Phenotypic Heterogeneity in a Clonal Cell Population In Vitro&author_name=Daniel%20Stockholm%2C%20Rachid%20Benchaouir%2C%20Julien%20Picot%2C%20Philippe%20Rameau%2C%20Thi%20My%20Anh%20Neildez%2C%20Gabriel%20Landini%2C%20Corinne%20Laplace-Builh%C3%A9%2C%20Andras%20Paldi&start_page=1&end_page=13" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000394" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000394" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0000394" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000394&amp;t=The%20Origin%20of%20Phenotypic%20Heterogeneity%20in%20a%20Clonal%20Cell%20Population%20In%20Vitro" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000394&title=The%20Origin%20of%20Phenotypic%20Heterogeneity%20in%20a%20Clonal%20Cell%20Population%20In%20Vitro&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000394&amp;title=The%20Origin%20of%20Phenotypic%20Heterogeneity%20in%20a%20Clonal%20Cell%20Population%20In%20Vitro" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000394" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0000394&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'The Origin of Phenotypic Heterogeneity in a Clonal Cell Population In Vitro';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0000394';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20The%20Origin%20of%20Phenotypic%20Heterogeneity%20in%20a%20Clonal%20Cell%20Population%20In%20Vitro http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000394" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0000394" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">















          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22B+cells%22" title="Search for articles in the subject area:'B cells'"><div class="flagText">B cells</div></a>
              <div data-categoryid="31979" data-articleid="23626"
                   data-categoryname="B cells"
                   class="flagImage" title="Flag 'B cells' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Cell+cycle+and+cell+division%22" title="Search for articles in the subject area:'Cell cycle and cell division'"><div class="flagText">Cell cycle and cell division</div></a>
              <div data-categoryid="17353" data-articleid="23626"
                   data-categoryname="Cell cycle and cell division"
                   class="flagImage" title="Flag 'Cell cycle and cell division' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Cell+death%22" title="Search for articles in the subject area:'Cell death'"><div class="flagText">Cell death</div></a>
              <div data-categoryid="19339" data-articleid="23626"
                   data-categoryname="Cell death"
                   class="flagImage" title="Flag 'Cell death' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Cell+differentiation%22" title="Search for articles in the subject area:'Cell differentiation'"><div class="flagText">Cell differentiation</div></a>
              <div data-categoryid="17923" data-articleid="23626"
                   data-categoryname="Cell differentiation"
                   class="flagImage" title="Flag 'Cell differentiation' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Cell+migration%22" title="Search for articles in the subject area:'Cell migration'"><div class="flagText">Cell migration</div></a>
              <div data-categoryid="17279" data-articleid="23626"
                   data-categoryname="Cell migration"
                   class="flagImage" title="Flag 'Cell migration' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Computer+modeling%22" title="Search for articles in the subject area:'Computer modeling'"><div class="flagText">Computer modeling</div></a>
              <div data-categoryid="2283" data-articleid="23626"
                   data-categoryname="Computer modeling"
                   class="flagImage" title="Flag 'Computer modeling' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Phenotypes%22" title="Search for articles in the subject area:'Phenotypes'"><div class="flagText">Phenotypes</div></a>
              <div data-categoryid="40235" data-articleid="23626"
                   data-categoryname="Phenotypes"
                   class="flagImage" title="Flag 'Phenotypes' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Spatial+distribution%22" title="Search for articles in the subject area:'Spatial distribution'"><div class="flagText">Spatial distribution</div></a>
              <div data-categoryid="31855" data-articleid="23626"
                   data-categoryname="Spatial distribution"
                   class="flagImage" title="Flag 'Spatial distribution' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=6618'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=4703'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=5796&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>

<div class="block sidebar-comments">
    <div class="header">
        <h3>Comments</h3>
    </div>
      <p><a href="/annotation/listThread.action?root=44549">interesting question, but ...</a><br>Posted by suihuang</p>
</div>

</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
