

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Homology Modeling of Dissimilatory APS Reductases (AprBA) of Sulfur-Oxidizing and Sulfate-Reducing Prokaryotes</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0001514"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0001514" />

    <meta name="citation_title" content="Homology Modeling of Dissimilatory APS Reductases (AprBA) of Sulfur-Oxidizing and Sulfate-Reducing Prokaryotes"/>
    <meta itemprop="name" content="Homology Modeling of Dissimilatory APS Reductases (AprBA) of Sulfur-Oxidizing and Sulfate-Reducing Prokaryotes"/>

      <meta name="citation_author" content="Birte Meyer"/>
            <meta name="citation_author_institution" content="Max Planck Institute for Marine Microbiology, Bremen, Germany"/>
      <meta name="citation_author" content="Jan Kuever"/>
            <meta name="citation_author_institution" content="Max Planck Institute for Marine Microbiology, Bremen, Germany"/>

    <meta name="citation_date" content="2008/1/30"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0001514.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e1514"/>
    <meta name="citation_issue" content="1"/>
    <meta name="citation_volume" content="3"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Dissimilatory sulfate- and sulfur-reducing prokaryotes.; citation_author=R Rabus; citation_author=TA Hansen; citation_author=F Widdel; citation_number=1; citation_pages=1-87; citation_date=1999; citation_publisher=Springer; " />
      <meta name="citation_reference" content="citation_title=Metabolism of sulfate-reducing prokaryotes.; citation_author=TA Hansen; citation_journal_title=Antonie Van Leeuwenhoek International Journal of General and Molecular Microbiology; citation_volume=66; citation_number=2; citation_pages=165-185; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=The biological sulfur cycle.; citation_author=T Brüser; citation_author=PNL Lens; citation_author=HG Trüper; citation_number=3; citation_pages=47-86; citation_date=2000; citation_publisher=IWA Publishing; " />
      <meta name="citation_reference" content="citation_title=Sulfite reductase and APS reductase from Archaeoglobus fulgidus.; citation_author=C Dahl; citation_author=HG Trüper; citation_journal_title=Methods in Enzymology; citation_number=4; citation_pages=427-441; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Towards the phylogeny of APS reductases and sirohaem sulfite reductases in sulfate-reducing and sulfur-oxidizing prokaryotes.; citation_author=WM Hipp; citation_author=AS Pott; citation_author=N Thum-Schmitz; citation_author=I Faath; citation_author=C Dahl; citation_journal_title=Microbiology-Uk; citation_volume=143; citation_number=5; citation_pages=2891-2902; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Anaerobic oxidation of sulfur-compounds as electron-donors for bacterial photosynthesis.; citation_author=HG Trüper; citation_author=U Fischer; citation_journal_title=Philosophical Transactions of the Royal Society of London Series B-Biological Sciences; citation_volume=298; citation_number=6; citation_pages=529-542; citation_date=1982; " />
      <meta name="citation_reference" content="citation_title=Enzymes of dissimilatory sulfide oxidation in phototrophic sulfur bacteria.; citation_author=C Dahl; citation_author=HG Trüper; citation_journal_title=Methods in Enzymology; citation_number=7; citation_pages=400-421; citation_date=1994; " />
      <meta name="citation_reference" content="citation_author=BF Taylor; citation_number=8; citation_pages=393-400; citation_date=1994; citation_publisher=Academic Press; " />
      <meta name="citation_reference" content="citation_title=Anaerobic oxidation of thiosulfate and elemental sulfur in Thiobacillus denitrificans.; citation_author=M Schedel; citation_author=HG Trüper; citation_journal_title=Archives of Microbiology; citation_volume=124; citation_number=9; citation_pages=205-210; citation_date=1980; " />
      <meta name="citation_reference" content="citation_title=Sulfur compounds as photosynthetic electron donors.; citation_author=DC Brune; citation_number=10; citation_pages=847-870; citation_date=1995; citation_publisher=Kluwer Academic Publishers; " />
      <meta name="citation_reference" content="citation_title=Dissimilatory ATP sulfurylase from Archaeoglobus fulgidus.; citation_author=D Sperling; citation_author=U Kappler; citation_author=HG Truper; citation_author=C Dahl; citation_journal_title=Methods in Enzymology; citation_number=11; citation_pages=419-427; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Adenylylsulfate reductases from sulfate-reducing bacteria.; citation_author=J Lampreia; citation_author=AS Pereira; citation_author=JJG Moura; citation_journal_title=Methods in Enzymology; citation_number=12; citation_pages=241-260; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=Adenylylsulfate reductases from archaea and bacteria are 1∶1 alpha beta-heterodimeric iron-sulfur flavoenzymes - high similarity of molecular properties emphasizes their central role in sulfur metabolism.; citation_author=G Fritz; citation_author=T Buchert; citation_author=H Huber; citation_author=KO Stetter; citation_author=PMH Kroneck; citation_journal_title=FEBS Letters; citation_volume=473; citation_number=13; citation_pages=63-66; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=A novel membrane-bound respiratory complex from Desulfovibrio desulfuricans ATCC 27774.; citation_author=RH Pires; citation_author=AI Lourenco; citation_author=F Morais; citation_author=M Teixeira; citation_author=AV Xavier; citation_journal_title=Biochimica Et Biophysica Acta-Bioenergetics; citation_volume=1605; citation_number=14; citation_pages=67-82; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Physiological and gene expression analysis of inhibition of Desulfovibrio vulgaris Hildenborough by nitrite.; citation_author=SA Haveman; citation_author=EA Greene; citation_author=CP Stilwell; citation_author=JK Voordouw; citation_author=G Voordouw; citation_journal_title=Journal of Bacteriology; citation_volume=186; citation_number=15; citation_pages=7944-7950; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Phylogeny of the alpha and beta subunits of the dissimilatory adenosine-5′-phosphosulfate (APS) reductase from sulfate-reducing prokaryotes - origin and evolution of the dissimilatory sulfate-reduction pathway.; citation_author=B Meyer; citation_author=J Kuever; citation_journal_title=Microbiology; citation_volume=153; citation_number=16; citation_pages=2026-2044; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Molecular analysis of the distribution and phylogeny of dissimilatory adenosine-5′-phosphosulfate reductase-encoding genes (aprBA) among sulfur-oxidizing prokaryotes.; citation_author=B Meyer; citation_author=J Kuever; citation_journal_title=Microbiology; citation_volume=153; citation_number=17; citation_pages=3478-3498; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Structure of adenylylsulfate reductase from the hyperthermophilic Archaeoglobus fulgidus at 1.6-A resolution.; citation_author=G Fritz; citation_author=A Roth; citation_author=A Schiffer; citation_author=T Buchert; citation_author=G Bourenkov; citation_journal_title=Proceedings of the National Academy of Sciences of the United States of America; citation_volume=99; citation_number=18; citation_pages=1836-1841; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=The structure of Wolinella succinogenes quinol: Fumarate reductase and its relevance to the superfamily of succinate: Quinone oxidoreductases.; citation_author=CRD Lancaster; citation_journal_title=Membrane Proteins; citation_volume=63; citation_number=19; citation_pages=131-149; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Wolinella succinogenes quinol : fumarate reductase and its comparison to E. coli succinate : quinone reductase.; citation_author=CRD Lancaster; citation_journal_title=FEBS Letters; citation_volume=555; citation_number=20; citation_pages=21-28; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Studies on Adenosine 5′-Phosphosulfate Reductase from Desulfovibrio desulfuricans and Thiobacillus thioparus. I. Assay and Purification.; citation_author=HD Peck; citation_author=TE Deacon; citation_author=JT Davidson; citation_journal_title=Biochimica Et Biophysica Acta; citation_volume=96; citation_number=21; citation_pages=429-446; citation_date=1965; " />
      <meta name="citation_reference" content="citation_title=A Flavin-Sulfite Adduct as an Intermediate in Reaction Catalyzed by Adenylyl Sulfate Reductase from Desulfovibrio vulgaris.; citation_author=GB Michaels; citation_author=JT Davidson; citation_author=HD Peck; citation_journal_title=Biochemical and Biophysical Research Communications; citation_volume=39; citation_number=22; citation_pages=321-&; citation_date=1970; " />
      <meta name="citation_reference" content="citation_title=Reaction mechanism of the iron-sulfur flavoenzyme adenosine-5′-phosphosulfate reductase based on the structural characterization of different enzymatic states.; citation_author=A Schiffer; citation_author=G Fritz; citation_author=PMH Kroneck; citation_author=U Ermler; citation_journal_title=Biochemistry; citation_volume=45; citation_number=23; citation_pages=2960-2967; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=The function of the [4Fe-4S] clusters and FAD in bacterial and archaeal adenylylsulfate reductases - Evidence for flavin-catalyzed reduction of adenosine 5′-phosphosulfate.; citation_author=G Fritz; citation_author=T Buchert; citation_author=PMH Kroneck; citation_journal_title=Journal of Biological Chemistry; citation_volume=277; citation_number=24; citation_pages=26066-26073; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Comparative protein structure modeling of genes and genomes.; citation_author=MA Marti-Renom; citation_author=AC Stuart; citation_author=A Fiser; citation_author=R Sanchez; citation_author=F Melo; citation_journal_title=Annual Review of Biophysics and Biomolecular Structure; citation_volume=29; citation_number=25; citation_pages=291-325; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Comparative protein structure modeling.; citation_author=A Fiser; citation_author=R Sanchez; citation_author=F Melo; citation_author=A Sali; citation_number=26; citation_pages=275-312; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Homology modeling.; citation_author=E Krieger; citation_author=SB Nabuurs; citation_author=G Vriend; citation_number=27; citation_date=2003; citation_publisher=Wiley; " />
      <meta name="citation_reference" content="citation_title=The response of protein structures to amino-acid sequence changes.; citation_author=AM Lesk; citation_author=C Cothia; citation_journal_title=Philosophical Transactions of the Royal Society of London B Biological Sciences; citation_volume=317; citation_number=28; citation_pages=345-356; citation_date=1986; " />
      <meta name="citation_reference" content="citation_title=3DCoffee@igs: a web server for combining sequences and structures into a multiple sequence alignment.; citation_author=O Poirot; citation_author=K Suhre; citation_author=C Abergel; citation_author=E O'Toole; citation_author=C Notredame; citation_journal_title=Nucleic Acids Research; citation_volume=32; citation_number=29; citation_pages=W37-W40; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Tcoffee@igs: a web server for computing, evaluating and combining multiple sequence alignments.; citation_author=O Poirot; citation_author=E O'Toole; citation_author=C Notredame; citation_journal_title=Nucleic Acids Research; citation_volume=31; citation_number=30; citation_pages=3503-3506; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=PHYML Online -a web server for fast maximum likelihood-based phylogenetic inference.; citation_author=S Guindon; citation_author=F Lethiec; citation_author=P Duroux; citation_author=O Gascuel; citation_journal_title=Nucl Acids Res; citation_volume=33; citation_number=31; citation_pages=W557-559; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Clustered genes related to sulfate respiration in uncultured prokaryotes support the theory of their concomitant horizontal transfer.; citation_author=M Mussmann; citation_author=M Richter; citation_author=T Lombardot; citation_author=A Meyerdierks; citation_author=J Kuever; citation_journal_title=Journal of Bacteriology; citation_volume=187; citation_number=32; citation_pages=7126-7137; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=SWISS-MODEL and the Swiss-PdbViewer: An environment for comparative protein modeling.; citation_author=N Guex; citation_author=MC Peitsch; citation_journal_title=Electrophoresis; citation_volume=18; citation_number=33; citation_pages=2714-2723; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=SWISS-MODEL: an automated protein homology-modeling server.; citation_author=T Schwede; citation_author=J Kopp; citation_author=N Guex; citation_author=MC Peitsch; citation_journal_title=Nucl Acids Res; citation_volume=31; citation_number=34; citation_pages=3381-3385; citation_date=2003; " />
      <meta name="citation_reference" content="citation_author=WF van Gunsteren; citation_author=SR Billeter; citation_author=AA Eising; citation_author=PH Hünenberger; citation_author=P Krüger; citation_number=35; citation_date=1996; citation_publisher=VdF Hochschulverlag ETHZ; " />
      <meta name="citation_reference" content="citation_title=Verification of protein structures: sidechain planarity.; citation_author=RWW Hooft; citation_author=C Sander; citation_author=G Vriend; citation_journal_title=Journal of Applied Crystallography; citation_volume=29; citation_number=36; citation_pages=714-716; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Recognition of errors in three-dimensional structures of proteins.; citation_author=MJ Sippl; citation_journal_title=Proteins; citation_volume=17; citation_number=37; citation_pages=355-362; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=Assessing protein structures with a non-local atomic interaction energy.; citation_author=F Melo; citation_author=E Feytmans; citation_journal_title=Journal of Molecular Biology; citation_volume=277; citation_number=38; citation_pages=1141-1152; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Assessment of protein models with three-dimensional profiles.; citation_author=R Luthy; citation_author=JU Bowie; citation_author=D Eisenberg; citation_journal_title=Nature; citation_volume=356; citation_number=39; citation_pages=83-85; citation_date=1992; " />
      <meta name="citation_reference" content="citation_title=Protein structure computing in the genomic era.; citation_author=T Schwede; citation_author=A Diemand; citation_author=N Guex; citation_author=MC Peitsch; citation_journal_title=Research in Microbiology; citation_volume=151; citation_number=40; citation_pages=107-112; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=A novel, fully saturated cyclic menaquinone in the archaeabacterium Pyrobaculum organotrophum.; citation_author=BJ Tindall; citation_author=V Wray; citation_author=R Huber; citation_author=MD Collins; citation_journal_title=Systematic and Applied Microbiology; citation_volume=14; citation_number=41; citation_pages=218-221; citation_date=1991; " />
      <meta name="citation_reference" content="citation_title=Taxonomy of nonmethanogenic hyperthermophilic and related thermophilic archaea.; citation_author=T Itoh; citation_journal_title=Journal of Bioscience and Bioengineering; citation_volume=96; citation_number=42; citation_pages=203-212; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=The photosynthetic reaction centre from the purple bacterium Rhodopseudomonas viridis.; citation_author=J Deisenhofer; citation_author=H Michel; citation_journal_title=EMBO (European Molecular Biology Organization) Journal; citation_volume=8; citation_number=43; citation_pages=2149-2170; citation_date=1989; " />
      <meta name="citation_reference" content="citation_title=Crystal structure of a complex between electron transfer partners, cytochrome c peroxidase and cytochrome c.; citation_author=H Pelletier; citation_author=J Kraut; citation_journal_title=Science; citation_volume=258; citation_number=44; citation_pages=1748-1755; citation_date=1992; " />
      <meta name="citation_reference" content="citation_title=Thermoproteales.; citation_author=H Huber; citation_author=R Huber; citation_author=KO Stetter; citation_number=45; citation_date=2002; citation_publisher=Fischer Verlag; " />
      <meta name="citation_reference" content="citation_title=Pyrobaculum calidifontis sp. nov., a novel hyperthermophilic archaeon that grows in atmospheric air.; citation_author=T Amo; citation_author=MLF Paje; citation_author=A Inagaki; citation_author=S Ezaki; citation_author=H Atomi; citation_journal_title=Archaea; citation_volume=1; citation_number=46; citation_pages=113-121; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Genome sequence of the hyperthermophilic crenarchaeon Pyrobaculum aerophilum.; citation_author=ST Fitz-Gibbon; citation_author=H Ladner; citation_author=UJ Kim; citation_author=KO Stetter; citation_author=MI Simon; citation_journal_title=Proceedings of the National Academy of Sciences of the United States of America; citation_volume=99; citation_number=47; citation_pages=984-989; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Crystal structure of refined thioredoxin reductase from Escherichia coli: Structural flexibility in the isoalloxazine ring of the flavin adenine dinucleotide cofactor.; citation_author=BW Lennon; citation_author=CH Williams; citation_author=ML Ludwig; citation_journal_title=Protein Science; citation_volume=8; citation_number=48; citation_pages=2366-2379; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Isotope fractionation by sulfate-reducing natural populations and the isotopic composition of sulfide in marine sediments.; citation_author=KS Habicht; citation_author=DE Canfield; citation_journal_title=Geology; citation_volume=29; citation_number=49; citation_pages=555-558; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Effect of low sulfate concentrations on lactate oxidation and isotope Fractionation during sulfate reduction by Archaeoglobus fulgidus strain Z.; citation_author=KS Habicht; citation_author=LL Salling; citation_author=B Thamdrup; citation_author=DE Canfield; citation_journal_title=Applied and Environmental Microbiology; citation_volume=71; citation_number=50; citation_pages=3770-3777; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Biogeochemistry of sulfur isotopes.; citation_author=DE Canfield; citation_number=51; citation_pages=607-636; citation_date=2001; citation_publisher=Mineralogy Society of America; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001514" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Homology Modeling of Dissimilatory APS Reductases (AprBA) of Sulfur-Oxidizing and Sulfate-Reducing Prokaryotes"/>
    <meta name="twitter:description" content="BackgroundThe dissimilatory adenosine-5&prime;-phosphosulfate (APS) reductase (cofactors flavin adenine dinucleotide, FAD, and two [4Fe-4S] centers) catalyzes the transformation of APS to sulfite and AMP in sulfate-reducing prokaryotes (SRP); in sulfur-oxidizing bacteria (SOB) it has been suggested to operate in the reverse direction. Recently, the three-dimensional structure of the Archaeoglobus fulgidus enzyme has been determined in different catalytically relevant states providing insights into its reaction cycle.Methodology/Principal FindingsFull-length AprBA sequences from 20 phylogenetically distinct SRP and SOB species were used for homology modeling. In general, the average accuracy of the calculated models was sufficiently good to allow a structural and functional comparison between the beta- and alpha-subunit structures (78.8&ndash;99.3% and 89.5&ndash;96.8% of the AprB and AprA main chain atoms, respectively, had root mean square deviations below 1 &Aring; with respect to the template structures). Besides their overall conformity, the SRP- and SOB-derived models revealed the existence of individual adaptations at the electron-transferring AprB protein surface presumably resulting from docking to different electron donor/acceptor proteins. These structural alterations correlated with the protein phylogeny (three major phylogenetic lineages: (1) SRP including LGT-affected Archaeoglobi and SOB of Apr lineage II, (2) crenarchaeal SRP Caldivirga and Pyrobaculum, and (3) SOB of the distinct Apr lineage I) and the presence of potential APS reductase-interacting redox complexes. The almost identical protein matrices surrounding both [4Fe-4S] clusters, the FAD cofactor, the active site channel and center within the AprB/A models of SRP and SOB point to a highly similar catalytic process of APS reduction/sulfite oxidation independent of the metabolism type the APS reductase is involved in and the species it has been originated from.ConclusionsBased on the comparative models, there are no significant structural differences between dissimilatory APS reductases from SRP and SOB; this might be indicative for a similar catalytic process of APS reduction/sulfite oxidation."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0001514.g010"/>

  <meta property="og:title" content="Homology Modeling of Dissimilatory APS Reductases (AprBA) of Sulfur-Oxidizing and Sulfate-Reducing Prokaryotes" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001514" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=1897'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=2137'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=1430&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0001514">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001514" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001514&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Homology Modeling of Dissimilatory APS Reductases (AprBA) of Sulfur-Oxidizing and Sulfate-Reducing Prokaryotes
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Birte Meyer, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Max Planck Institute for Marine Microbiology, Bremen, Germany
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Jan Kuever
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:kuever@mpa-bremen.de">kuever@mpa-bremen.de</a></p>

                <p>Affiliation:
                  Max Planck Institute for Marine Microbiology, Bremen, Germany
                </p>

                <p>
                    Current address: Bremen Institute for Materials Testing, Bremen, Germany
                </p>

              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: January 30, 2008</li>
    <li>DOI: 10.1371/journal.pone.0001514</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0001514" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0001514" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001514" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0001514" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0001514-g001" data-doi="info:doi/10.1371/journal.pone.0001514" data-uri="info:doi/10.1371/journal.pone.0001514.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001514.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001514-g002" data-doi="info:doi/10.1371/journal.pone.0001514" data-uri="info:doi/10.1371/journal.pone.0001514.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001514.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001514-g003" data-doi="info:doi/10.1371/journal.pone.0001514" data-uri="info:doi/10.1371/journal.pone.0001514.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001514.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001514-t001" data-doi="info:doi/10.1371/journal.pone.0001514" data-uri="info:doi/10.1371/journal.pone.0001514.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001514.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001514-g004" data-doi="info:doi/10.1371/journal.pone.0001514" data-uri="info:doi/10.1371/journal.pone.0001514.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001514.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001514-g005" data-doi="info:doi/10.1371/journal.pone.0001514" data-uri="info:doi/10.1371/journal.pone.0001514.g005" title="Figure 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001514.g005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001514-t002" data-doi="info:doi/10.1371/journal.pone.0001514" data-uri="info:doi/10.1371/journal.pone.0001514.t002" title="Table 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001514.t002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001514-g006" data-doi="info:doi/10.1371/journal.pone.0001514" data-uri="info:doi/10.1371/journal.pone.0001514.g006" title="Figure 6">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001514.g006&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001514-g007" data-doi="info:doi/10.1371/journal.pone.0001514" data-uri="info:doi/10.1371/journal.pone.0001514.g007" title="Figure 7">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001514.g007&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001514-g008" data-doi="info:doi/10.1371/journal.pone.0001514" data-uri="info:doi/10.1371/journal.pone.0001514.g008" title="Figure 8">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001514.g008&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001514-g009" data-doi="info:doi/10.1371/journal.pone.0001514" data-uri="info:doi/10.1371/journal.pone.0001514.g009" title="Figure 9">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001514.g009&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001514-g010" data-doi="info:doi/10.1371/journal.pone.0001514" data-uri="info:doi/10.1371/journal.pone.0001514.g010" title="Figure 10">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001514.g010&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001514">Reader Comments (1)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0001514" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2>
<h3>Background</h3>
<a id="article1.front1.article-meta1.abstract1.sec1.p1" name="article1.front1.article-meta1.abstract1.sec1.p1"></a><p>The dissimilatory adenosine-5′-phosphosulfate (APS) reductase (cofactors flavin adenine dinucleotide, FAD, and two [4Fe-4S] centers) catalyzes the transformation of APS to sulfite and AMP in sulfate-reducing prokaryotes (SRP); in sulfur-oxidizing bacteria (SOB) it has been suggested to operate in the reverse direction. Recently, the three-dimensional structure of the <em>Archaeoglobus fulgidus</em> enzyme has been determined in different catalytically relevant states providing insights into its reaction cycle.</p>


<h3>Methodology/Principal Findings</h3>
<a id="article1.front1.article-meta1.abstract1.sec2.p1" name="article1.front1.article-meta1.abstract1.sec2.p1"></a><p>Full-length AprBA sequences from 20 phylogenetically distinct SRP and SOB species were used for homology modeling. In general, the average accuracy of the calculated models was sufficiently good to allow a structural and functional comparison between the beta- and alpha-subunit structures (78.8–99.3% and 89.5–96.8% of the AprB and AprA main chain atoms, respectively, had root mean square deviations below 1 Å with respect to the template structures). Besides their overall conformity, the SRP- and SOB-derived models revealed the existence of individual adaptations at the electron-transferring AprB protein surface presumably resulting from docking to different electron donor/acceptor proteins. These structural alterations correlated with the protein phylogeny (three major phylogenetic lineages: (1) SRP including LGT-affected Archaeoglobi and SOB of Apr lineage II, (2) crenarchaeal SRP <em>Caldivirga</em> and <em>Pyrobaculum</em>, and (3) SOB of the distinct Apr lineage I) and the presence of potential APS reductase-interacting redox complexes. The almost identical protein matrices surrounding both [4Fe-4S] clusters, the FAD cofactor, the active site channel and center within the AprB/A models of SRP and SOB point to a highly similar catalytic process of APS reduction/sulfite oxidation independent of the metabolism type the APS reductase is involved in and the species it has been originated from.</p>


<h3>Conclusions</h3>
<a id="article1.front1.article-meta1.abstract1.sec3.p1" name="article1.front1.article-meta1.abstract1.sec3.p1"></a><p>Based on the comparative models, there are no significant structural differences between dissimilatory APS reductases from SRP and SOB; this might be indicative for a similar catalytic process of APS reduction/sulfite oxidation.</p>

</div>


<div class="articleinfo"><p><strong>Citation: </strong>Meyer B, Kuever J (2008) Homology Modeling of Dissimilatory APS Reductases (AprBA) of Sulfur-Oxidizing and Sulfate-Reducing Prokaryotes. PLoS ONE 3(1):
          e1514.
            doi:10.1371/journal.pone.0001514</p><p><strong>Academic Editor: </strong>Hany El-Shemy, Cairo University, Egypt</p><p><strong>Received:</strong> October 9, 2007; <strong>Accepted:</strong> December 17, 2007; <strong>Published:</strong> January 30, 2008</p><p><strong>Copyright:</strong> © 2008 Meyer, Kuever. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>This study was funded by the Max-Planck-Society and the German Research Foundation.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>ATP sulfurylase, adenosine-5′-phosphosulfate (APS) reductase and sulfite reductase mediate the process of dissimilatory sulfate reduction and, therefore, represent key enzymes in the energy metabolism of the phylogenetically diverse sulfate-reducing prokaryotes (SRP). Sulfate is activated to APS by ATP sulfurylase (Sat) at the expense of ATP; hereafter, APS reductase (Apr) converts APS to AMP and sulfite which is subsequently reduced to sulfide by the activity of sulfite reductase (Dsr) <a href="#pone.0001514-Rabus1">[1]</a>–<a href="#pone.0001514-Dahl1">[4]</a>. Homologous proteins have been shown to exist in several anoxygenic phototrophic and chemotrophic, facultative anaerobic sulfur-oxidizing bacteria (SOB) <a href="#pone.0001514-Brser1">[3]</a>, <a href="#pone.0001514-Hipp1">[5]</a>–<a href="#pone.0001514-Sperling1">[11]</a> in which they were suggested to operate in the reverse direction oxidizing sulfide to sulfate. In SRP as well as in SOB, these enzymes appeared to be located in the cytoplasm or at the cytoplasmic site of the inner membrane <a href="#pone.0001514-Brser1">[3]</a>, <a href="#pone.0001514-Dahl2">[7]</a>, <a href="#pone.0001514-Taylor1">[8]</a>, <a href="#pone.0001514-Brune1">[10]</a>, <a href="#pone.0001514-Sperling1">[11]</a>.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>The enzyme APS reductase has been investigated from several SRP and SOB with respect to the molecular parameters, e.g. mass, subunit composition and cofactor stoichiometry; the results confirmed its general conformity although the enzyme was involved in different metabolic pathways (oxidative/reductive) and was isolated from phylogenetically distant species <a href="#pone.0001514-Dahl2">[7]</a>, <a href="#pone.0001514-Taylor1">[8]</a>, <a href="#pone.0001514-Lampreia1">[12]</a>, <a href="#pone.0001514-Fritz1">[13]</a>. Its functional unit is suggested to be a 1:1 heterodimeric complex composed of a FAD-containing alpha-subunit with a molecular mass of 70–75 kDa and a two [4Fe-4S] clusters-containing beta-subunit of 18–23 kDa <a href="#pone.0001514-Lampreia1">[12]</a>, <a href="#pone.0001514-Fritz1">[13]</a>. There is increasing evidence that a quinone-interacting membrane-bound oxidoreductase, QmoABC, represents the native electron donor in SRP <a href="#pone.0001514-Pires1">[14]</a>, <a href="#pone.0001514-Haveman1">[15]</a>. Recent phylogenetic analyses of AprBA sequences of sulfate-reducers and sulfur-oxidizers demonstrated that their encoding genes, <em>aprBA</em>, have been frequently affected by lateral gene transfer (LGT) events <a href="#pone.0001514-Meyer1">[16]</a>, <a href="#pone.0001514-Meyer2">[17]</a> which are reflected in the existence of two distantly related oxidative APS reductase types in SOB: Most members of the <em>Chromatiaceae</em> possess a homologue (SOB Apr lineage I) distinct to the reductive protein type of the SRP, whereas members of certain SOB families and genera, e.g. <em>Chlorobiaceae</em> and betaproteobacterial <em>Thiobacillus</em>, harbor SRB-related, putatively LGT-acquired <em>aprBA</em> genes (SOB Apr lineage II) <a href="#pone.0001514-Meyer2">[17]</a>. Genome data demonstrated that the composition/arrangement of the <em>apr</em> gene loci (occurrence of <em>qmoABC</em> or <em>aprM</em> genes) of the distinct SOB lineages I and II correlates with the AprBA phylogeny. Based on the concomitant presence of the <em>apr</em> and <em>qmo</em> genes, the Qmo redox complex has been suggested to act as a functional link between the APS reductase and the membrane quinone pool also in those SOB that harbor SRP-related <em>apr</em> gene loci (SOB Apr lineage II) <a href="#pone.0001514-Meyer2">[17]</a> as proposed for the SRP <a href="#pone.0001514-Pires1">[14]</a>. The native electron donor of the SOB Apr lineage I type APS reductase is still unknown; interestingly, the corresponding genes are always co-transcribed with a conserved ORF, <em>aprM</em>, coding for a membrane-integral protein <a href="#pone.0001514-Meyer2">[17]</a>.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>The X-ray structure of the dissimilatory APS reductase isolated from <em>Archaeoglobus fulgidus</em> has been elucidated at 1.6 Å resolution by Fritz and coworkers <a href="#pone.0001514-Fritz2">[18]</a>: Its beta-subunit can be subdivided in three segments comprising a bacterial ferredoxin-like segment that envelopes both [4Fe-4S] clusters (amino acids B1-B68), followed by a three-stranded antiparallel beta-sheet (B69-B104) and a tail with a length of 50 Å (B105-B148) (see <a href="#pone-0001514-g001">Fig. 1</a>). The structure of the alpha-subunit can be grouped into the FAD cofactor-binding (amino acids A2-A261 and A394-A487), the capping (A262-A393) and the helical domains (A488-A643) (see <a href="#pone-0001514-g001">Fig. 1</a>); its overall structure classifies this subunit of the APS reductase as member of the fumarate reductase family <a href="#pone.0001514-Fritz2">[18]</a>–<a href="#pone.0001514-Lancaster2">[20]</a>. The global part of the beta-subunit is embedded into a broad cleft of the alpha-subunit, while its long tail wraps around the latter increasing the contact surface between both subunits <a href="#pone.0001514-Fritz2">[18]</a> (see <a href="#pone-0001514-g001">Fig. 1</a>). The reaction mechanism of APS reductase has been under debate <a href="#pone.0001514-Peck1">[21]</a>, <a href="#pone.0001514-Michaels1">[22]</a>; Schiffers and coworkers recently determined the X-ray structures of <em>A. fulgidus</em> APS reductases in different enzymatic states <a href="#pone.0001514-Schiffer1">[23]</a> that confirmed the proposed catalytic mechanism via a nucleophilic attack of the N5 atom of reduced FAD on the sulfur of APS. A covalent FAD-APS intermediate is formed that decomposes spontaneously to AMP and to the FAD-sulfite adduct which is subsequently cleaved, and sulfite is finally liberated <a href="#pone.0001514-Schiffer1">[23]</a>, <a href="#pone.0001514-Fritz3">[24]</a>. The two electrons required for the reduction of APS were postulated to be transferred one by one over 30 Å via [4Fe-4S] cluster II at the surface of the protein, cluster I and Trp-B48 to the isoalloxazine ring of the buried FAD <a href="#pone.0001514-Fritz2">[18]</a>, <a href="#pone.0001514-Schiffer1">[23]</a>, <a href="#pone.0001514-Fritz3">[24]</a> (see <a href="#pone-0001514-g001">Fig. 1</a>).</p>
<div class="figure" id="pone-0001514-g001"><div class="img"><a name="pone-0001514-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001514.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001514" data-uri="info:doi/10.1371/journal.pone.0001514.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001514.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001514.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001514.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001514.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001514.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001514.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001514.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001514.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001514.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Three-dimensional ribbon structure of APS reductase from <em>A. fulgidus.</em></span></strong></p><a id="article1.body1.sec1.fig1.caption1.p1" name="article1.body1.sec1.fig1.caption1.p1"></a><p>The beta-subunit segments are colored red (ferredoxin segment), blue (3 antiparallel beta-sheets segment), and green (tail segment); the alpha-subunit domains are colored light blue and orange (FAD-binding domain I and II), pink (capping domain), and grey (helical domain). The [4Fe-4S] clusters, FAD and substrate APS are shown as ball-and-stick representations; tryptophan Trp-B48 of AprB is highlighted by violet color. Ribbon structure is shown from (A) top view, (B) bottom view (substrate channel), (C) front view, and (D) back view.</p>
<span>doi:10.1371/journal.pone.0001514.g001</span></div><a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p>The aim of this study was to investigate the structural differences of oxidative and reductive APS reductases present in the phylogenetically and physiologically diverse SRP and SOB based on homology modeling-built protein models. The theoretical protein structure prediction method homology (or comparative) modeling <a href="#pone.0001514-MartiRenom1">[25]</a>–<a href="#pone.0001514-Krieger1">[27]</a> relies on the observation that the structural conformation of a protein is more highly conserved than its amino acid sequence, and that small or medium changes in sequence typically result in only small changes in the 3D structure <a href="#pone.0001514-Lesk1">[28]</a>. In general, reliable homology models are constructed if the sequence identity exceeds 30% <a href="#pone.0001514-MartiRenom1">[25]</a>–<a href="#pone.0001514-Krieger1">[27]</a>. The results are discussed in context to the AprBA-based phylogeny and the presence/absence of the Qmo redox complex-encoding genes.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Analysis"></a><h3>Analysis</h3>
<h4>Sequence analysis and phylogenetics</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>PSI-BLAST (<a href="http://www.ncbi.nlm.nih.gov/BLAST/">www.ncbi.nlm.nih.gov/BLAST/</a>) was used to search the non-redundant version of current public databases of the National Center for Biotechnology Information (NCBI) for homologous, full-length sequences of QmoABC and AprBA. The AprBA sequences were implemented into the persisting multiple alignment of SRP- and SOB-derived sequences <a href="#pone.0001514-Meyer1">[16]</a>, <a href="#pone.0001514-Meyer2">[17]</a> using the Bioedit (version 7.0.5) sequence alignment editor (<a href="http://www.mbio.ncsu.edu/BioEdit/bioedit.html">http://www.mbio.ncsu.edu/BioEdit/bioedit​.html</a>). The QmoA, QmoB, and QmoC proteins were automatically aligned with the web server <a href="mailto:3DCoffeeigs">3DCoffeeigs</a> (<a href="http://igs-server.cnrs-mrs.fr/Tcoffee/">http://igs-server.cnrs-mrs.fr/Tcoffee/</a>) <a href="#pone.0001514-Poirot1">[29]</a>, <a href="#pone.0001514-Poirot2">[30]</a>; the initial alignments were refined manually by visual inspection. Phylogenetic analysis of the AprBA, QmoA, QmoB, and QmoC data sets (119, 21, 22, and 18 sequences) was performed with the online version of PhyML (<a href="http://atgc.lirmm.fr/phyml">http://atgc.lirmm.fr/phyml</a>) <a href="#pone.0001514-Guindon1">[31]</a>. Maximum likelihood-based phylogenetic trees were constructed using the global rearrangement and randomized species input order options as well as the JTT matrix as amino acid replacement model; regions of insertions and deletions (indels) were omitted from the calculations yielding finally 701, 392, 715, and 357 compared amino acid positions, respectively. Statistical support for the protein trees is given by bootstrap analysis with 100 resamplings.</p>


<h4>Homology modeling</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>The AprBA sequences of <em>Pyrobaculum calidifontis</em> (YP_929769-70), <em>Pyrobaculum aerophilum</em> (NP_560098-101), <em>Caldivirga maquilingensis</em> (ZP_01711818-9), <em>Chlorobaculum tepidum</em> (NP_661758-9), <em>Pelagibacter ubique</em> HTCC1062 (YP_266256-7), uncultured alphaproteobacterium EBAC2C11 (AAV31645-6), <em>Thiobacillus denitrificans</em> ATCC 25259 (YP_314630-1, YP_316040-1), <em>Allochromatium vinosum</em> (AAC23620-1), <em>Candidatus</em> Ruthia magnifica (YP_903357-8), <em>Desulfovibrio vulgaris</em> str. Hildenborough (YP_010067-8), <em>Desulfovibrio desulfuricans</em> ATCC 29577 (AAF36689-90), <em>Desulfotalea psychrophila</em> (YP_064840-1), <em>Desulfobulbus</em> sp. str. MLMS-1 (ZP_01288426-7), <em>Olavius algarvensis</em> Delta1 symbiont (AASZ_01000974), <em>Syntrophobacter fumaroxidans</em> (YP_845176-7), uncultured sulfate-reducing bacterium fosws7f8 and fosws39f7 (CAJ31201-2; CAJ31179-80) <a href="#pone.0001514-Mussmann1">[32]</a>, <em>Desulfotomaculum reducens</em> str. MI-1 (ZP_011499890-1), <em>Thermodesulfobacterium commune</em> and <em>Thermodesulfovibrio yellowstonii</em> (genome sequencing database of The Institute of Genome Research, <a href="http://www.tigr.org">http://www.tigr.org</a>) were selected to create three-dimensional models of dissimilatory APS reductases from a phylogenetically and physiologically diverse range of SRB and SOB. Comparative modeling by assembly of rigid bodies <a href="#pone.0001514-MartiRenom1">[25]</a>–<a href="#pone.0001514-Krieger1">[27]</a> was performed with the SWISS-MODEL/ProModII online server for automated protein homology modeling (<a href="http://swissmodel.expasy.org">http://swissmodel.expasy.org</a>) <a href="#pone.0001514-Guex1">[33]</a>, <a href="#pone.0001514-Schwede1">[34]</a> using the recently determined AprBA structures of <em>Archaeoglobus fulgidus</em> <a href="#pone.0001514-Fritz2">[18]</a>, <a href="#pone.0001514-Schiffer1">[23]</a> as templates (Brookhaven Protein Data Bank, PDB, entries 1jnrA/B, 2FJA, 2FJB, 2FJD, 2FJE). The input alignments (SWISS-MODEL “alignment mode”) were based on a structural multiple alignment calculated by 3DCoffee which was manually refined to result in optimal pair wise target-template alignments (to be modeled SOB/SRB AprBA versus the <em>A. fulgidus</em> sequence). Model construction by the ProModII program included complete backbone and side chain building, loop building, verification of model quality, including packing, and subsequent energy minimization <a href="#pone.0001514-Guex1">[33]</a>, <a href="#pone.0001514-Schwede1">[34]</a> using the Gromos96 force field <a href="#pone.0001514-vanGunsteren1">[35]</a>. The stereochemical and energetic parameters of the initial 3D protein models were evaluated by the WHATCHECK <a href="#pone.0001514-Hooft1">[36]</a>, PROSAII <a href="#pone.0001514-Sippl1">[37]</a>, ANOLEA <a href="#pone.0001514-Melo1">[38]</a>, and Verify3D <a href="#pone.0001514-Luthy1">[39]</a> analysis reports provided by the SWISS-MODEL server. The protein structure visualization program DeepView (Swiss-PdbViewer, version 3.7.; available from the ExPASy server <a href="http://www.expasy.org/spdbv">http://www.expasy.org/spdbv</a>) <a href="#pone.0001514-Guex1">[33]</a>, <a href="#pone.0001514-Schwede2">[40]</a> was used to optimize the placement of the insertions and deletions considering the template structure context and conservation of structural features with a functional role. The “project mode” of SWISS-MODEL was used to iteratively improve the output 3D models of each SOB and SRB species on the basis of the evaluation software embedded within ProModII. Docking of the [4Fe-4S] clusters, FAD cofactor and substrate APS into the structure of modeled dissimilatory APS reductase was performed by superimposition with the template. The quality of the final model was determined from the B-score values provided by SWISS-MODEL and the root mean square (RMS) deviation values between equivalenced atoms in the amino acids of modeled and template protein. The models were also analyzed for violations of main chain Phi/Psi dihedral bond angle ratios and backbone/side chain steric conflicts. The molecular surface including the electrostatic potential as well as the accessible surface areas were calculated using the corresponding tools implemented into the DeepView program. The figures were generated using the Render3D mode of DeepView.</p>

</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Results and Discussion"></a><h3>Results and Discussion</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p>In general, the average homology model accuracy is a function of the template-target sequence similarity: The dispersion of the model-target structural overlap increases with the decrease in sequence identity between template and target. Errors in models can be divided into five categories: (1) errors in side chain packing, (2) distortions and shifts of a region that is aligned correctly with the template structure, (3) distortions and shifts of a region that does not have an equivalent segment in any of the template structures, (4) distortions and shifts of a region that is aligned incorrectly with the template structures, and (5) a misfolded structure that results from using an incorrect template. Errors 3–5 are relatively infrequent when sequences with more than 40% identity to the templates are modeled; in such a case, approximately 90% of the main chain atoms are likely to be modeled with an RMS deviation of about 1 Å between template and target proteins; the structural differences are usually limited to loops and side chains <a href="#pone.0001514-MartiRenom1">[25]</a>–<a href="#pone.0001514-Krieger1">[27]</a>. Overall, these models are almost as good as medium-resolution experimental structures, because the proteins at this level of similarity are likely to be as similar to each other as are the structures for the same protein determined by different experimental techniques under different conditions, see <a href="#pone.0001514-MartiRenom1">[25]</a>–<a href="#pone.0001514-Krieger1">[27]</a> and references therein. When the sequence identity is between 30 and 40%, the structural differences become larger resulting in an increase of the main chain RMS errors to about 1.5 Å for about 80% of residues. The rest of the residues are modeled with large errors because the homology modeling methods generally fail to model structural distortions and rigid body shifts, and are unable to recover from misalignments <a href="#pone.0001514-MartiRenom1">[25]</a>–<a href="#pone.0001514-Krieger1">[27]</a>.</p>
<a id="article1.body1.sec3.p2" name="article1.body1.sec3.p2"></a><p>In this study, full-length Apr sequences from 20 phylogenetically distinct SRP and SOB species were used for homology modeling that ranged in their sequence identity values to the <em>A. fulgidus</em> templates between 38.6 to 62.6% (beta-subunit) and 47.6 to 60.7% (alpha-subunit) (see supplementary data material <a href="#pone.0001514.s005">Table S1</a> and <a href="#pone.0001514.s007">S3</a>). The lowest identity values were found for the sequences of SOB Apr lineage-I members, e.g. <em>Pelagibacter ubique</em>, (AprB: 38.6 to 45.8%; AprA: 47.7 to 51.5%) and the sequences of crenarchaeal <em>Pyrobaculum</em> spp. (38.7 to 42.4%; AprA: 47.6%) which is in accordance to the AprBA phylogeny (see <a href="#pone-0001514-g002">Fig. 2A</a>). The APS reductases of SRB species possessed sequence identities to the <em>A. fulgidus</em> templates that ranged between 53.1 to 62.6% (AprB) and 49.5 to 60.7% (AprA) with the highest values received for the Gram-positive SRB and the LGT-affected deltaproteobacterial members <a href="#pone.0001514-Meyer1">[16]</a>; the APS reductases of SOB Apr lineage II had 49.0 to 61.4% (AprB) and 50.2 to 54.0% (AprA) sequence identity to the templates. The overall accuracy of all AprA and most AprB comparative models could be assumed to be sufficiently good to allow their structural and functional comparison (in the models, 78.8 to 99.3% (AprB) and 89.5 to 96.8% (AprA) of the main chain atoms had RMS deviations below 1 Å, see supplementary data material <a href="#pone.0001514.s005">Table S1</a> and <a href="#pone.0001514.s007">S3</a>).</p>
<div class="figure" id="pone-0001514-g002"><div class="img"><a name="pone-0001514-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001514.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001514" data-uri="info:doi/10.1371/journal.pone.0001514.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001514.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001514.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001514.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001514.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001514.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001514.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001514.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001514.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001514.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Phylogenetic trees based on (A) AprBA, (B) QmoA, (C) QmoB, and (D) QmoC sequences.</span></strong></p><a id="article1.body1.sec3.fig1.caption1.p1" name="article1.body1.sec3.fig1.caption1.p1"></a><p>The trees were inferred using PhyML (maximum likelihood method). The SOB Apr lineage-I sequence group (A) and the <em>Archaeglobus fulgidus</em> QmoABC sequences (B–D) were used as outgroup, respectively. The scale bar corresponds to 10% estimated sequence divergence. Branching orders that were only supported by bootstrap resampling values below 50% are shown as multifurcations; percentages greater than 50% of bootstrap resampling supporting a topological element are indicated near the nodes.</p>
<span>doi:10.1371/journal.pone.0001514.g002</span></div>
<h4>Structural comparison of the beta-subunit (AprB) comparative models of the dissimilatory APS reductase among SRP and SOB</h4>
<a id="article1.body1.sec3.sec1.p1" name="article1.body1.sec3.sec1.p1"></a><p>The three segment-subdivision of the <em>A. fulgidus</em> beta-subunit described by Fritz and coworkers <a href="#pone.0001514-Fritz2">[18]</a>, <a href="#pone.0001514-Schiffer1">[23]</a> is reflected in all comparative models of the dissimilatory APS reductase irrespective of species metabolism type and phylogenetic affiliation. While the presence and orientation of the secondary structure elements are strictly conserved in the functionally important first and second segment of the models (ferredoxin-like and alpha-subunit interface region), the third, the tail segment, is more variable among the investigated SRP and SOB (see <a href="#pone-0001514-g003">Fig. 3</a> for AprB models of selected SRP/SOB; all AprB models are presented in supplementary material <a href="#pone.0001514.s005">Table S1</a> and <a href="#pone.0001514.s001">Figure S1</a>). Significantly higher main chain atom RMS deviations of up to 3.28 Å to the <em>A. fulgidus</em> template were present in this protein region in comparison with the low values of the first segments that ranged between 0.00 and 0.84 Å (see <a href="#pone-0001514-t001">Table 1</a>). The tail region has been proposed to be responsible for the tightening of the subunit interaction and, thus, a stable heterodimer formation by increasing the contact surface between both subunits <a href="#pone.0001514-Fritz2">[18]</a>, <a href="#pone.0001514-Schiffer1">[23]</a>. The differing presence of secondary structure elements in the tail segments of the models is a result of the high variability in sequence and length. The structural dispersion of the tail regions might reflect the process of speciation by individual structural adaptations at the interacting surface between the beta- and alpha-subunit of each species. Indeed, the tail segment is the only sequence section of the beta-subunit that contains sufficient phylogenetic information to allow inter- and intrafamily differentiation among the SRP and SOB sequences.</p>
<div class="figure" id="pone-0001514-g003"><div class="img"><a name="pone-0001514-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001514.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001514" data-uri="info:doi/10.1371/journal.pone.0001514.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001514.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001514.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001514.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001514.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001514.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001514.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001514.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001514.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001514.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Three-dimensional structure of AprB from <em>A. fulgidus</em> (A) and selected, homology modeling-based AprB models from <em>Allochromatium vinosum</em> (B) and <em>Pelagibacter ubique</em> (C) (as representatives of SOB from Apr lineage-I), <em>Pyrobaculum calidifontis</em> (D) (as representative of crenarchaeal SRP), <em>Desulfotomaculum reducens</em> (E) (as representative of Gram-positive SRB and LGT-affected deltaproteobacterial SRB), <em>Desulfovibrio vulgaris</em> (F) (as representative of non-LGT-affected deltaproteobacterial SRB), <em>Chlorobaculum tepidum</em> (G) and <em>Thiobacillus denitrificans</em> (H) (as representatives of LGT-affected SOB from Apr lineage-II).</span></strong></p><a id="article1.body1.sec3.sec1.fig1.caption1.p1" name="article1.body1.sec3.sec1.fig1.caption1.p1"></a><p>Ribbon structure shown from front view (positions of [4Fe-4S] clusters indicated in <em>A. fulgidus</em> AprB). Ribbon structure of <em>A. fulgidus</em> AprB (A) colored by secondary structure elements; ribbon structures of AprB models (B–H) colored by model confidence factor provided by SWISS-MODEL (green, respective region of model and reference structure superpose; red, respective region of model deviates from the reference structure). The missing flexible loop between Cys-B13 and Gly-B19 (enumeration based on <em>A. fulgidus</em> sequence) in models of SOB from Apr lineage-I and <em>Pyrobaculum</em> spp. is marked by red arrows.</p>
<span>doi:10.1371/journal.pone.0001514.g003</span></div><div class="figure" id="pone-0001514-t001"><div class="img"><a name="pone-0001514-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001514.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001514" data-uri="info:doi/10.1371/journal.pone.0001514.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001514.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001514.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001514.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001514.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001514.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001514.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001514.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001514.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001514.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>Root mean square deviations (RMSD) of AprB and AprA comparative models from SRP and SOB with respect to the <em>A. fulgidus</em> template structure.</span></strong></p><span>doi:10.1371/journal.pone.0001514.t001</span></div><a id="article1.body1.sec3.sec1.p2" name="article1.body1.sec3.sec1.p2"></a><p>Besides their overall conformity in the secondary structure element positioning, the first and the second segment of the comparative models showed significant differences in distinct loop regions resulting from insertions and deletions (see <a href="#pone-0001514-g003">Fig. 3</a> and <a href="#pone-0001514-g004">4</a>; for details see supplementary data material <a href="#pone.0001514.s005">Table S1</a> and <a href="#pone.0001514.s001">Figure S1</a>). Interestingly, these structural differences reflected the AprBA-based phylogeny in its separation into three major phylogenetic AprBA clusters: (1) the SRB including the LGT-affected Archaeoglobi and SOB of Apr lineage II, (2) the <em>Caldivirga</em>-<em>Pyrobaculum</em> group of putative crenarchaeal sulfate-reducers, and (3) the SOB of distinct Apr lineage I (see <a href="#pone-0001514-g002">Fig. 2A</a>). Furthermore, the presence/absence of certain loops among the AprB models of each cluster correlated with the absence/presence of the Qmo redox complex and AprM protein encoding genes in the investigated SRP and SOB genomes <a href="#pone.0001514-Meyer1">[16]</a>, <a href="#pone.0001514-Meyer2">[17]</a> and might reflect their different functional linkage to the electron transport chain in the membrane. (1) In all comparative AprB models of SRB, <em>A. fulgidus</em> and the SOB of Apr lineage II, there is a flexible loop conservatively located between Cys-B13 and Gly-B19 which comprised predominantly charged amino acids of the following general sequence, K G X D/E K/R (see <a href="#pone-0001514-g003">Fig. 3</a> and <a href="#pone-0001514-g004">4</a>; for sequence details see supplementary data material <a href="#pone.0001514.s005">Table S1</a>). This loop is absent in the beta-subunit models of SOB from Apr lineage I and the <em>Caldivirga</em>-<em>Pyrobaculum</em> group (see <a href="#pone-0001514-g003">Fig. 3B–D</a> and supplementary data material <a href="#pone.0001514.s001">Figure S1</a>). Especially the conserved Lys-B14 and Glu/Asp-B17 are located at exposed, solvent-accessible positions in this loop (see <a href="#pone-0001514-g004">Fig. 4</a> and supplementary data material <a href="#pone.0001514.s001">Figures S1</a>, panels G/H); they might be responsible for the functional docking of the putative redox partner QmoABC/QmoAB(-HdrBC) to the AprB protein surface adjacent to the [4Fe-4S] cluster II. The Qmo complex presumably operates as physiological electron carrier between the membrane-integral quinone/quinol pool and the cytoplasmic APS reductase in the SRP as well as in the sulfite-oxidizing <em>Chlorobiaceae</em> and <em>Beta</em>- and <em>Gammaproteobacteria</em> that harbor a SRB-related enzyme (SOB Apr lineage II) <a href="#pone.0001514-Meyer2">[17]</a>. <em>Thiobacillus denitrificans</em> possessed a second exposed loop of six amino acids positioned between the beta-alpha-beta structure motifs of the ferredoxin-like segment; however, its functional or structural role is not apparent. (2) The crenarchaeal <em>Caldivirga maquilingensis</em> and putative sulfate-reducing <em>Pyrobaculum</em> spp. did not harbor <em>qmo</em> homologous sequences in their genomes <a href="#pone.0001514-Meyer2">[17]</a>; in agreement to the previous proposal, an elongated loop between secondary structure element 2 and 3 was absent in their AprB comparative models (see <a href="#pone-0001514-g003">Fig. 3D</a> and supplementary material <a href="#pone.0001514.s001">Figure S1</a>). The putative physiological electron donor for the dissimilatory APS reductase in these species is unknown; however, naphtoquinones have been described from <em>Pyrobaculum</em> species <a href="#pone.0001514-Tindall1">[41]</a>, <a href="#pone.0001514-Itoh1">[42]</a>. The electrostatic potential at the protein surface of the crenarchaeal AprB models differed significantly from the other SRP and SOB which might be an indication that the potential interacting redox partner and the electron transfer process of the aforementioned are different and unrelated to the latter (see <a href="#pone-0001514-g004">Fig. 4</a> and supplementary data material <a href="#pone.0001514.s001">Figures S1</a>, panels E/F). Interestingly, both <em>Pyrobaculum</em> APS reductases missed the strictly conserved, electron transfer-relevant Trp-B48 that was substituted by an Ala-B43 residue at the corresponding AprB model position (see <a href="#pone-0001514-g004">Fig. 4 panel G</a> for the <em>Pyrobaculum calidifontis</em> AprB model). In all other Apr comparative models, the Trp-B48/43 was located between the S3 of cluster I and the methyl C8M of FAD (distance 12.4 Å) and in van der Waals contact with both redox centers; its indole ring was locked in its position by Thr-A233 and Arg-A232. The electron transfer function of trytophan residues located between two redox centers has been frequently documented <a href="#pone.0001514-Deisenhofer1">[43]</a>, <a href="#pone.0001514-Pelletier1">[44]</a>; an analogous functional role, however, has not been reported for alanine residues. In consequence, the <em>Pyrobaculum</em> spp. APS reductases might have lost their structural ability for electron transfer between both subunits (essential for APS reduction) and, thus, their enzymes will not be functional anymore (see also the structural comparison of the alpha-subunits). Indeed, no cultivated <em>Pyrobaculum</em> species has been described to be capable of dissimilatory sulfate reduction <a href="#pone.0001514-Huber1">[45]</a>, <a href="#pone.0001514-Amo1">[46]</a>; notably, the <em>aprBA</em> sequences of the <em>Pyrobaculum aerophilum</em> genome are frame-shifted <a href="#pone.0001514-FitzGibbon1">[47]</a> which might be a result of elevated mutation rate in irrelevant enzymes/metabolic pathways. (3) Like the beta-subunits of the crenarchaeal putative sulfate-reducers, the AprB comparative models of the SOB Apr lineage-I sulfur-oxidizers did not contain an elongated loop between secondary structure element 1 and 2 (see <a href="#pone-0001514-g003">Fig. 3B and C</a> and supplementary material <a href="#pone.0001514.s001">Figure S1</a>). Consistently, the genomes of the respective species did not include <em>qmo</em> homologues; however, their <em>aprBA</em> genes were always preceded and co-transcribed by a membrane-integral protein encoding gene, <em>aprM</em>. Interestingly, all AprB models of the SOB Apr lineage I possessed in the antiparallel beta-sheet segment an elongated loop by amino acid insertion between the secondary structure element 8 and 9 that comprised predominantly negatively charged residues (see <a href="#pone-0001514-g004">Fig. 4 panels C and E</a>; supplementary data material <a href="#pone.0001514.s005">Table S1</a> and <a href="#pone.0001514.s001">Figure S1</a>, compare panels E/F). The latter were exposed to the solvent (see <a href="#pone-0001514-g004">Fig. 4 panels D and F</a> and supplementary data material <a href="#pone.0001514.s001">Figure S1</a>, compare panels G/H) and might function as an interface region for docking the AprBA enzyme to the AprM protein that anchors the dissimilatory APS reductase to the membrane and enables its physical contact to the yet unknown electron receptor. The presumed differing functional linkage of the cytoplasmic SOB Apr lineage I-type APS reductases to the membrane (not involving Qmo complex homologues) was also reflected in the deviating electrostatic potential at the protein surface when compared to the SRB-type AprB models (supplementary data material <a href="#pone.0001514.s001">Figure S1</a>, compare panels E/F).</p>
<div class="figure" id="pone-0001514-g004"><div class="img"><a name="pone-0001514-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001514.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001514" data-uri="info:doi/10.1371/journal.pone.0001514.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001514.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001514.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001514.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001514.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001514.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001514.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001514.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001514.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001514.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>Three-dimensional structure of AprB from <em>A. fulgidus</em> (A, B) and selected, homology modeling-based AprB models from <em>Allochromatium vinosum</em> (C, D), <em>Pelagibacter ubique</em> (E, F), <em>Pyrobaculum calidifontis</em> (G, H), <em>Desulfotomaculum reducens</em> (I, J), <em>Desulfovibrio vulgaris</em> (K, L), <em>Chlorobaculum tepidum</em> (M, N) and <em>Thiobacillus denitrificans</em> (O, P).</span></strong></p><a id="article1.body1.sec3.sec1.fig2.caption1.p1" name="article1.body1.sec3.sec1.fig2.caption1.p1"></a><p>Protein molecular surface colored by calculated electrostatic potential are shown in panels A, C, E, G, I, K, M, O (electric charge at the molecular surface is colored with a red (negative), white (neutral), and blue (positive) color gradient; electric field extending into the solvent is shown); the differently present, negatively charged loops in the models of SRP and SOB are marked by yellow color (the additional loop of <em>Thiobacillus denitrificans</em> is shown in violet); the electron-transferring Trp-B43/-B48 is marked by yellow color (<em>Pyrobaculum calidifontis</em>: Trp-substituting Ala-B43 is highlighted in G and H). Protein molecular surface colored by calculated solvent accessibility are shown in panels B, D, F, H, J, L, N, P.</p>
<span>doi:10.1371/journal.pone.0001514.g004</span></div>

<h4>The [4Fe-4S] clusters surrounding protein matrix in the AprB models of SRP and SOB</h4>
<a id="article1.body1.sec3.sec2.p1" name="article1.body1.sec3.sec2.p1"></a><p>The [4Fe-4S] clusters of APS reductases from several sulfate reducing strains have been documented to differ significantly in their reduction potential (approximately −500 mV for cluster II and −60 mV for cluster I) <a href="#pone.0001514-Fritz2">[18]</a>, <a href="#pone.0001514-Schiffer1">[23]</a>, <a href="#pone.0001514-Fritz3">[24]</a>. Based on the X-ray structure of <em>A. fulgidus</em>, the large reduction potential difference of the two clusters was explained by their distinctly different surrounding protein matrix (see <a href="#pone-0001514-g005">Fig. 5 panel A</a>). Generally, local dipole in close proximity to the acid-labile sulfur and cysteinyl sulfur atoms modulate the reduction potential of a [4Fe-4S] cluster: Its reduced state can be stabilized by NH-S hydrogen bonds and backbone amide dipoles that shift the reduction potential of an iron-sulfur center to a more positive value. In the <em>A. fulgidus</em> APS reductase, the number of polar interactions between the sulfur atoms of cluster I compared to cluster II and the backbone amides at a distance of less than 3.5 Å (17 versus 7) is substantially increased and proposed to be responsible for the high reduction potential of cluster I. In contrast, the close proximity of the negatively charged Asp-B11-carboxylate group to an acid-labile sulfur of the cluster II was suggested to stabilize its oxidized state with the result of a low reduction potential <a href="#pone.0001514-Fritz2">[18]</a>, <a href="#pone.0001514-Schiffer1">[23]</a> (see <a href="#pone-0001514-g005">Fig. 5 panel A</a>; see also supplementary material <a href="#pone.0001514.s002">Figure S2</a> for details).</p>
<div class="figure" id="pone-0001514-g005"><div class="img"><a name="pone-0001514-g005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001514.g005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001514" data-uri="info:doi/10.1371/journal.pone.0001514.g005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001514.g005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001514.g005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001514.g005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001514.g005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001514.g005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001514.g005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001514.g005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001514.g005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001514.g005.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 5.  <span>AprB protein matrix surrounding the [4Fe-4S] cluster I and II (residues in a distance of less than 5.0 Å are shown) in the three-dimensional structure from <em>A. fulgidus</em> (A) and selected, homology modeling-based models from <em>Allochromatium vinosum</em> (B), <em>Pelagibacter ubique</em> (C), <em>Pyrobaculum calidifontis</em> (D), <em>Desulfotomaculum reducens</em> (E), <em>Desulfovibrio vulgaris</em> (F), <em>Chlorobaculum tepidum</em> (G), and <em>Thiobacillus denitrificans</em> (H).</span></strong></p><a id="article1.body1.sec3.sec2.fig1.caption1.p1" name="article1.body1.sec3.sec2.fig1.caption1.p1"></a><p>Charged and polar residues are marked (positively charged AA, blue; negatively charged AA, red; polar AA, yellow); tryptophan (Trp-B48/-B43) and cysteine residues are highlighted by violet and green color.</p>
<span>doi:10.1371/journal.pone.0001514.g005</span></div><a id="article1.body1.sec3.sec2.p2" name="article1.body1.sec3.sec2.p2"></a><p>The DeepView program does not allow the calculation of potential interactions between the modeled protein and enzyme-associated cofactors; the structural comparison of the different comparative models and the <em>A. fulgidus</em> template was therefore restricted to distance analyses between the [4Fe-4S] clusters and their surrounding protein matrix. Notably, there were no significant structural differences between the comparative models of SRP and those of SOB (see <a href="#pone-0001514-t002">Table 2</a>, see <a href="#pone-0001514-g005">Fig. 5</a> for AprB models of selected SRP/SOB; for details see supplementary data material <a href="#pone.0001514.s002">Figure S2</a> and <a href="#pone.0001514.s006">Table S2</a>). Indeed, the protein environments of the clusters from sulfur-oxidizers of Apr lineage II, e.g. <em>Chlorobaculum tepidum</em> and <em>Thiobacillus denitrificans</em>, were more similar to those of SRP than to those of Apr lineage I sulfur-oxidizers (compare residues in a distance of less than 3.5 Å to the clusters, <a href="#pone-0001514-t002">Table 2</a>). In consequence, there was no increased number of polar interactions between cluster II compared with cluster I and backbone amides in the aforementioned SOB that would modulate the iron-sulfur center reduction potentials in order to favor the reverse “oxidative” electron transfer in these species. According to the models, the structural differences in the [4Fe-4S] ambient protein matrix rather correlated with the AprBA phylogeny: In contrast to the models of SRB including affiliated Archaeoglobi and SOB of Apr lineage II, the Apr lineage I SOB and crenarchaeal SRP models revealed a deviating successive appearance of amino acids near both clusters (see supplementary data material <a href="#pone.0001514.s002">Figure S2</a> and <a href="#pone.0001514.s006">Table S2</a>). As a result of the missing elongated loop between secondary structure 1 and 2, Ile-B24 and Thr-B3 came very close (2.0 and 3.0 Å, respectively) to the [4Fe-4S] cluster I, whereas cluster II exhibited an increased number of backbone amide contacts at a distance of less than 3.0 Å in contrast to cluster I. Besides the carboxylate group of Asp-B11 (in some cases substituted by Glu-B54), the side chain of a histidine residue (His-B15/54/55) was positioned in proximity (3.5 Å) to cluster II in most AprB models of Apr lineage I SOB (<a href="#pone-0001514-t002">Table 2</a> and supplementary data material <a href="#pone.0001514.s002">Figure S2</a> and <a href="#pone.0001514.s006">Table S2</a>). However, if this positively charged surrounding is appropriate to stabilize the reduced state of cluster II and, thus, to shift its reduction potential to more positive values remains to be proven experimentally. Noteworthy, both <em>Pyrobaculum</em> spp. AprB models contained two positively charged amino acids adjacent to the acid labile sulfur and cysteinyl sulfur atoms of cluster II that would not favor oxidized state of the latter as proposed to be essential for the intrinsic electron transfer in the SRP-type proteins (see <a href="#pone-0001514-g005">Fig. 5 panel D</a> for <em>Pyrobaculum calidifontis</em>). Indeed, this feature might be another indication for the presence of a non-functional enzyme in the investigated <em>Pyrobaculum</em> species.</p>
<div class="figure" id="pone-0001514-t002"><div class="img"><a name="pone-0001514-t002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001514.t002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001514" data-uri="info:doi/10.1371/journal.pone.0001514.t002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001514.t002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001514.t002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001514.t002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001514.t002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001514.t002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001514.t002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001514.t002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001514.t002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001514.t002.TIF"></span>)
                </a></li></ul></div><p><strong>Table 2.  <span>AprB amino acid residues surrounding the [4Fe-4S] clusters and cysteinyl sulfur atoms in a distance of less than 3.5 Å (sorted by chemical nature of the amino acid side chain)</span></strong></p><span>doi:10.1371/journal.pone.0001514.t002</span></div>

<h4>Structural comparison of the alpha-subunit (AprA) comparative models of the dissimilatory APS reductase among SRP and SOB</h4>
<a id="article1.body1.sec3.sec3.p1" name="article1.body1.sec3.sec3.p1"></a><p>The <em>A. fulgidus</em> alpha-subunit has been described to be subdivided into three domains, the FAD-binding (A2-261, A394-487), capping (A262-393), and the helical domain (A488-643) (see <a href="#pone-0001514-g001">Fig. 1</a> and <a href="#pone-0001514-g006">6</a>). The FAD-binding domain constitutes the center, while the capping and helical domains form the periphery of the alpha-subunit (the helical domain is firmly attached to the first whereas the capping domain is partly exposed from the core region). The FAD-binding domain is composed of a central six-stranded parallel beta-sheet (sec. str. elm. 1, 3, 11, 12, 13, 14) that is flanked by four alpha-helices on one side (sec. str. elm. 2, 6, 10, 36) and by a four-stranded mixed beta-sheet on the other (sec. str. elm. 15, 17, 19, 32) (see <a href="#pone-0001514-g006">Fig. 6 panel B</a>). The capping domain is inserted into the polypeptide chain of the aforementioned domain and consists of a four-stranded antiparallel beta-sheet (sec. str. elm. 20. 22, 27, 31) surrounded by eight (mostly short) alpha-helices (see <a href="#pone-0001514-g006">Fig. 6 panel C</a>), whereas the helical domain is primarily composed of three long alpha-helices (sec. str. elm. 39, 41, 42) (see <a href="#pone-0001514-g006">Fig. 6 panel D</a>) <a href="#pone.0001514-Fritz2">[18]</a>, <a href="#pone.0001514-Schiffer1">[23]</a>. This general fold scheme was present in all comparative AprA models of APS reductases from SOB and SRP (irrespective of species metabolism type and protein phylogenetic affiliation) (see <a href="#pone-0001514-g007">Fig. 7</a>, for details see supplementary data material <a href="#pone.0001514.s007">Table S3</a> and <a href="#pone.0001514.s003">Figure S3</a>). The conserved nature of the AprA structure was reflected in the low main chain atom RMS deviations from the template structure that ranged between 0.66 Å (uncultured SRB fosws7f8) and 1.20 Å (<em>O. algarvensis</em> Delta 1 symbiont) in the SRB and SOB Apr lineage II models (except the model of <em>Desulfovibrio vulgaris</em> with 1.54 Å RMSD), between 0.79 Å (<em>Thiobacillus denitrificans</em>) and 1.46 Å (environmental sequence EBAC2C11) in the SOB Apr lineage I, and in the crenarchaeal SRP group between 0.88 Å (<em>Caldivirga maquilingensis</em>) and 1.16 Å (<em>Pyrobaculum calidifontis</em>) (see <a href="#pone-0001514-t001">Table 1</a>). The helical domain which has been suggested to mainly build up the interface region between two αβ-heterodimers <a href="#pone.0001514-Fritz2">[18]</a>, <a href="#pone.0001514-Schiffer1">[23]</a> constituted the most highly conserved protein region in the AprA models with regard to the presence and orientation of the secondary structure elements (RMSD from 0.06 to 0.99 Å; see supplementary data material <a href="#pone.0001514.s007">Table S3</a> and <a href="#pone.0001514.s003">Figure S3</a>). In contrast, the FAD-binding and the capping domains of the comparative AprA models showed more structural differences; nevertheless, these alterations were predominantly restricted to certain loop amino acid stretches (e.g. between sec. str. elm. 10 and 11 or 36 and 37) and short secondary structure elements (e.g. alpha-helices, sec. str. elm. 23, 24 or 33) located at the protein surface (see supplementary data material <a href="#pone.0001514.s007">Table S3</a> and <a href="#pone.0001514.s003">Figure S3</a>). As mentioned in a previous section, the higher structural variability among the latter might reflect the individual adaptations at the contact areas between both APS reductase subunits of each species because these domains comprise the AprA interface areas to the second and third segment of AprB. Overall, the alpha-subunit core region was conserved and structurally uniform among the enzymes of sulfur-oxidizers and sulfate-reducers. Notably, the comparative models of <em>Thermodesulfovibrio yellowstonii</em>, <em>Thermodesulfobacterium commune</em>, the non-LGT-affected deltaproteobacterial SRB and the SOB of Apr lineage II contained 17 to 21 amino acids long insertions between the secondary structure elements 7 and 8; the inserted amino acid stretches were predicted by SWISS-MODEL to primarily form two extended, antiparallel beta-sheets that are exposed from the core region (see <a href="#pone-0001514-g007">Fig. 7 panel E to H</a>, see also supplementary data material <a href="#pone.0001514.s007">Table S3</a> and <a href="#pone.0001514.s003">Figure S3</a>). The accuracy of the proposed structures in this AprA model area is highly speculative because of the current computational limitations in calculating the correct fold of loop regions encompassing more than eight residues. Its solvent-exposed location at the AprB-interacting site of the alpha-subunit might be an indication for an involvement in stabilizing the contact between the native electron donor/acceptor protein and the electron-transferring smaller subunit of APS reductase.</p>
<div class="figure" id="pone-0001514-g006"><div class="img"><a name="pone-0001514-g006" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001514.g006&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001514" data-uri="info:doi/10.1371/journal.pone.0001514.g006"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001514.g006&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001514.g006/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001514.g006/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001514.g006/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001514.g006/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001514.g006.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001514.g006/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001514.g006/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001514.g006.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 6.  <span>Structure of AprA from <em>A. fulgidus.</em></span></strong></p><a id="article1.body1.sec3.sec3.fig1.caption1.p1" name="article1.body1.sec3.sec3.fig1.caption1.p1"></a><p>(A–E) Ribbon structure colored by secondary structure elements (position of FAD cofactor indicated), (A) entire model from front view, (B) only FAD-binding domain, (C) only capping domain, (D) only helical domain, and (E) entire protein from top view (position of APS molecule indicated additionally); (F) protein molecular surface colored by calculated solvent accessibility shown from top view; (G, H) protein molecular surface colored by calculated electrostatic potential shown from top and back view (electric charge at the molecular surface is colored with a red (negative), white (neutral, and blue (positive) color gradient).</p>
<span>doi:10.1371/journal.pone.0001514.g006</span></div><div class="figure" id="pone-0001514-g007"><div class="img"><a name="pone-0001514-g007" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001514.g007&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001514" data-uri="info:doi/10.1371/journal.pone.0001514.g007"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001514.g007&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001514.g007/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001514.g007/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001514.g007/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001514.g007/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001514.g007.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001514.g007/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001514.g007/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001514.g007.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 7.  <span>Selected, homology modeling-based AprA models from <em>Allochromatium vinosum</em> (A) and <em>Pelagibacter ubique</em> (B) (as representatives of SOB from Apr lineage-I), <em>Pyrobaculum calidifontis</em> (C) (as representative of crenarchaeal SRP), <em>Desulfotomaculum reducens</em> (D) (as representative of Gram-positive SRB and LGT-affected deltaproteobacterial SRB), <em>Thermodesulfobacterium commune</em> (E) (as representative of thermophilic SRB), <em>Desulfovibrio vulgaris</em> (F) (as representative of non-LGT-affected deltaproteobacterial SRB), <em>Chlorobaculum tepidum</em> (G) and <em>Thiobacillus denitrificans</em> (H) (as representatives of LGT-affected SOB from Apr lineage-II).</span></strong></p><a id="article1.body1.sec3.sec3.fig2.caption1.p1" name="article1.body1.sec3.sec3.fig2.caption1.p1"></a><p>Ribbon structure shown from front view (position of FAD cofactor and substrate APS are indicated). Ribbon structure of AprA models colored by model confidence factor provided by SWISS-MODEL (green, respective region of model and reference structure superpose; red, respective region of model deviates from the reference structure).</p>
<span>doi:10.1371/journal.pone.0001514.g007</span></div>

<h4>FAD cofactor surrounding protein matrix in the AprA models of SRP and SOB</h4>
<a id="article1.body1.sec3.sec4.p1" name="article1.body1.sec3.sec4.p1"></a><p>The reductive reaction mechanism of the dissimilatory APS reductase proceeds via a nucleophilic attack of the N5 atom (isoalloxazine moiety) of the reduced FAD on the sulfur of APS <a href="#pone.0001514-Fritz2">[18]</a>, <a href="#pone.0001514-Schiffer1">[23]</a>, <a href="#pone.0001514-Fritz3">[24]</a>. According to the AprA comparative models, 35 to 36 mostly conserved amino acids are located at a distance of less than 4.1 Å to the FAD cofactor (comprising residues of sec. str. elm. 1 to 2, 3 to 4, 11, 14 to 15, 16 to 18, 31 to 32, and 35 to 36 including the interjacent loop regions, see <a href="#pone-0001514-g008">Fig. 8</a>; for details see also supplementary data material <a href="#pone.0001514.s003">Figure S3</a>, panels G/H, and supplementary data material <a href="#pone.0001514.s008">Table S4</a>); thus, the cofactor-surrounding protein matrices are highly conserved among the APS reductases of SOB and SRP. As revealed by the <em>A. fulgidus</em> 3D protein structure, a pronounced feature of the FADH<sup>−</sup> state in the enzyme is the substantial bend of the isoalloxazine ring along the N5-N10 axis by an angle of 25° (<a href="#pone-0001514-g009">Fig. 9</a> and <a href="#pone-0001514-g010">10</a>). In general, the protein matrix could have a considerable influence on the bending angle of the latter and thereby affect the redox potential of FAD; a flat conformation of FAD (0–10°) favors the oxidized state and a bent “butterfly” conformation (15–30°) favors the reduced state. In the <em>A. fulgidus</em> APS reductase, the side chains of Asn-A74 and Trp-A234 that are located at the <em>re</em>-face of FAD enforce a shift of the dimethylbenzene and pyrimidine rings toward the <em>si</em>-face of the FAD, whereas the pyrazine ring is held in position by Leu-A70 which protrudes toward the <em>si</em>-face of FAD (<a href="#pone-0001514-g009">Fig. 9 panels A and B</a>). The induced stabilization of the reduced form of FAD agrees with the experimentally determined higher reduction potentials of ~−45 mV in APS reductases of several SRP compared to ~−220 mV of free FAD <a href="#pone.0001514-Fritz2">[18]</a>, <a href="#pone.0001514-Schiffer1">[23]</a>, <a href="#pone.0001514-Fritz3">[24]</a>. Notably, the aforementioned residues are strictly conserved and strongly fixed at their positions in all AprA models of the sulfate reducers (except <em>Pyrobaculum calidifontis</em>, see <a href="#pone-0001514-g009">Fig. 9 panels C and D</a>) and even the sulfur-oxidizers (see supplementary data material <a href="#pone.0001514.s004">Figure S4</a> for details) which might indicate that this cofactor is also present in a bent conformation in the SOB-type APS reductases although the coplanar arrangement of the three aromatic rings would be the energetical favorable conformation in the oxidized state. Indeed, the isoalloxazine ring system of <em>A. fulgidus</em> APS reductase revealed an identical bending angle in the oxidized as in the reduced FAD state <a href="#pone.0001514-Fritz2">[18]</a>, <a href="#pone.0001514-Schiffer1">[23]</a>, <a href="#pone.0001514-Fritz3">[24]</a> i.e. the butterfly conformation is maintained in the <em>A. fulgidus</em> protein independent of the redox potential of the FAD cofactor and direction of catalytic reaction (<a href="#pone-0001514-g010">Fig. 10</a>). This is also demonstrated by the AprA models of this study that encompassed reductive and oxidative type APS reductases from distinct SRP and SOB. In contrast, other enzymes e.g. thioredoxin reductase <a href="#pone.0001514-Lennon1">[48]</a>, alternate the FAD conformation with respect to their oxidized and reduced states. The AprA model of <em>Pyrobaculum calidifontis</em> lacked the structurally essential Asn-A63 at the re-face of FAD (see <a href="#pone-0001514-g009">Fig. 9 panels C and D</a>). However, the strained conformation of the isoalloxazine moiety was suggested to be important for efficient electron flow between the redox centers by facilitating the reduction of the oxidized FAD via the [4Fe-4S] clusters. In agreement with the AprB model-derived results, this missing structural feature might be another indication for the non-functionality of the APS reductase present in the investigated <em>Pyrobaculum</em> species.</p>
<div class="figure" id="pone-0001514-g008"><div class="img"><a name="pone-0001514-g008" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001514.g008&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001514" data-uri="info:doi/10.1371/journal.pone.0001514.g008"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001514.g008&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001514.g008/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001514.g008/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001514.g008/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001514.g008/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001514.g008.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001514.g008/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001514.g008/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001514.g008.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 8.  <span>AprA protein matrix surrounding the FAD cofactor (residues in a distance of less than 4.1 Å are shown) in the three-dimensional structure from <em>A. fulgidus</em> (A) and selected, homology modeling-based models from <em>Allochromatium vinosum</em> (B), <em>Pelagibacter ubique</em> (C), <em>Pyrobaculum calidifontis</em> (D), <em>Desulfotomaculum reducens</em> (E), <em>Desulfovibrio vulgaris</em> (F), <em>Chlorobaculum tepidum</em> (G), and <em>Thiobacillus denitrificans</em> (H).</span></strong></p><a id="article1.body1.sec3.sec4.fig1.caption1.p1" name="article1.body1.sec3.sec4.fig1.caption1.p1"></a><p>Charged and polar residues are marked (positively charged AA, blue; negatively charged AA, red; polar AA, yellow; uncharged/-polar AA, grey).</p>
<span>doi:10.1371/journal.pone.0001514.g008</span></div><div class="figure" id="pone-0001514-g009"><div class="img"><a name="pone-0001514-g009" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001514.g009&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001514" data-uri="info:doi/10.1371/journal.pone.0001514.g009"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001514.g009&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001514.g009/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001514.g009/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001514.g009/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001514.g009/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001514.g009.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001514.g009/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001514.g009/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001514.g009.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 9.  <span>AprA active center from the protein of <em>A. fulgidus</em> (A, B) and the homology modeling-based model of <em>Pyrobaculum calidifontis</em> (C, D) (residues in a distance of less than 6.5 Å to the N5 atom of FAD cofactor are shown).</span></strong></p><a id="article1.body1.sec3.sec4.fig2.caption1.p1" name="article1.body1.sec3.sec4.fig2.caption1.p1"></a><p>FAD cofactor and substrate APS (in two conformations) are shown as ball-and-stick representations. Residues involved in the isoalloxazine binding (e.g. <em>A. fulgidus</em>: Leu-A70, Asn-A74, Trp-A234) are highlighted by green color (missing Asn-A63 in the <em>Pyrobaculum calidifontis</em> AprA model is marked by an arrow), the invariant, positively charged residues His-A398 and Arg-A265 of the active site are blue colored (other AA are colored in grey). Distances are given in Å (B, D). (E) The position of the electron-transferring [4Fe-4S] cluster I and Trp-B48 (highlighted by violet color) of AprB to the FAD cofactor in the AprA protein of <em>A. fulgidus</em> is shown (ribbon structure is colored in grey).</p>
<span>doi:10.1371/journal.pone.0001514.g009</span></div><div class="figure" id="pone-0001514-g010"><div class="img"><a name="pone-0001514-g010" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001514.g010&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001514" data-uri="info:doi/10.1371/journal.pone.0001514.g010"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001514.g010&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001514.g010/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001514.g010/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001514.g010/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001514.g010/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001514.g010.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001514.g010/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001514.g010/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001514.g010.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 10.  <span>Reaction cycle of the dissimilatory APS reductase from <em>A. fulgidus</em> (Schiffer et al., 2006).</span></strong></p><span>doi:10.1371/journal.pone.0001514.g010</span></div>

<h4>Active site channel and active center in the AprA models of SRP and SOB</h4>
<a id="article1.body1.sec3.sec5.p1" name="article1.body1.sec3.sec5.p1"></a><p>According to the <em>A. fulgidus</em> 3D structure, the active site of APS reductase is deeply buried into the protein interior and is only accessible from the outside through a 17 Å long channel of 10 Å diameter which is formed at the interface between the FAD-binding and capping domains <a href="#pone.0001514-Fritz2">[18]</a>, <a href="#pone.0001514-Schiffer1">[23]</a>, <a href="#pone.0001514-Fritz3">[24]</a> (see <a href="#pone-0001514-g001">Fig. 1B</a> and <a href="#pone-0001514-g009">9E</a>). The entrance of the substrate-binding channel is surrounded by five positively charged residues (Arg-A85, Lys-A281, Lys-A283, Arg-A294, and Arg-A317) attracting negatively charged molecules such as APS, sulfite and AMP. With the exception of Lys-A281 and the AMP-moiety-interacting Arg-A317, the previous residues were not well-conserved among the AprA comparative models of SRP and SOB but were substituted by other positively charged residues of varying positions at the opening of the channel (data not shown). The channel itself has been proposed to be pre-built prior to substrate binding by a hydrophobic cluster of residues (Tyr-A95, Trp-A144, Trp-A234, Phe-A261, Val-A273, Gly-A274, Phe-A277, Leu-A278, Phe-A448) <a href="#pone.0001514-Fritz2">[18]</a>, <a href="#pone.0001514-Schiffer1">[23]</a>, <a href="#pone.0001514-Fritz3">[24]</a> that would allow fast catalysis which is essential especially for SRP that rely on efficient transformation of sulfate to sulfite for energy conservation. Indeed, the aforementioned residues are strictly conserved among the AprA models (note: Tyr-A95 and Phe-A261 are substituted by the hydrophobic amino acids methionine in most deltaproteobacterial SRB and isoleucine in SOB of Apr lineage I proteins; see supplementary data material <a href="#pone.0001514.s009">Table S5</a>). Structural analyses of different states of the <em>A. fulgidus</em> enzyme indicated, that substrate binding induces a shift of the isoalloxazine ring towards the channel bottom thereby producing a compressed enzyme-substrate complex. While the conformations of the adenine and ribose ring of APS are well-defined in the active site channel, phosphate and, particularly, the sulfate group appeared to be conformationally rather flexible. Before reduction, the molecule APS is present in a strained conformation with its sulfate moiety directly positioned in front of the pyrazine ring of FADH<sup>−</sup> (distance of ~3.4 Å between the sulfur and the N5 atoms). In this position, the sulfate O1 atom has been predicted to interact with the ND2 atom of Asn-A74, its O2 atom with NE2 atom of His-A398, and its O3 atom via a water molecule with Arg-A265 and Trp-A234 and via two water molecules with Glu-A141, Asp-A361, and Asn-A74 (<a href="#pone-0001514-g010">Fig. 10</a>) <a href="#pone.0001514-Fritz2">[18]</a>, <a href="#pone.0001514-Schiffer1">[23]</a>, <a href="#pone.0001514-Fritz3">[24]</a>. Indeed, Asn-A74, Trp-A234, Arg-A265, and His-A398 corresponding residues were predicted to be located in the active sites of all comparative models (except <em>Pyrobaculum calidifontis</em>, see previous section) at nearly identical positions and distances to the sulfate group of APS/sulfite and the N5 atom of FAD (see supplementary data material <a href="#pone.0001514.s004">Figure S4</a> and <a href="#pone.0001514.s010">Table S6</a>). In agreement with the <em>A. fulgidus</em> structure, the phosphate and ribose moiety of APS/AMP might be connected with Arg-A265, Val-A273, and Gly-A274 as well as Tyr-A95 and His-A446 corresponding residues in the substrate-binding channel of SRP- and SOB-type models. The adenine ring of APS/AMP is most likely fixed in its position in the models as proposed for the APS reductase of <em>A. fulgidus</em> (clamped between Arg-A317 and Leu-A278) <a href="#pone.0001514-Fritz2">[18]</a>, <a href="#pone.0001514-Schiffer1">[23]</a>, <a href="#pone.0001514-Fritz3">[24]</a> (see supplementary material <a href="#pone.0001514.s009">Table S5</a>). Upon APS/AMP binding of the <em>A. fulgidus</em> APS reductase, the Arg-A317 residues has been predicted to largely change its conformation by swinging into the channel to form a coplanar arrangement of its guanidine group to the adenine ring <a href="#pone.0001514-Fritz2">[18]</a>, <a href="#pone.0001514-Schiffer1">[23]</a>, <a href="#pone.0001514-Fritz3">[24]</a>. In conclusion, the active site channel including the active center appeared to be structural highly conserved among the APS reductases of SRP and even SOB with only minor alterations by single amino acid substitutions.</p>


<h4>Reaction mechanism of APS reduction and sulfite oxidation</h4>
<a id="article1.body1.sec3.sec6.p1" name="article1.body1.sec3.sec6.p1"></a><p>The enzyme-bound APS is held in a strained conformation by the Arg-A317/Leu-A278 clamp fixation of the adenine ring and the curved APS conformation. This energy-rich state has been postulated to become relaxed during the sulfonation reaction via a nucleophilic attack of the N5 atom of reduced FAD on the sulfur atom of APS; in consequence, a covalent FAD-APS intermediate is formed. The energy liberated upon cleavage of the S-O-P mixed anhydride bond of APS is high (80 kJ mol<sup>−1</sup>); catalysis will be supported by an increase in the nucleophilicity of N5 due to the deprotonated N1 atom and of the electrophilicity of the sulfur due to hydrogen bonds between the four sulfate oxygens of APS and Asn-A74, Trp-A234, Arg-A265, and His-A398. Rearrangements of electrons result in the release of AMP accompanied by formation of the N5-sulfite adduct. Subsequent dissociation of the latter to yield oxidized FAD and sulfite might be enzymatically triggered by protonation of the sulfite moiety (via His-A398) (see <a href="#pone-0001514-g010">Fig. 10</a>). Finally, the product sulfite is released and oxidized FAD reduced via the two [4Fe-4S] clusters. Concerning the backward reaction, the oxidative formation of APS from sulfite and AMP, it is assumed that the nucleophile sulfite adds to the N5 atom of oxidized FAD as most electrophilic position of the cofactor followed by a shift of the negative charge on the sulfite sulfur toward the flavin ring. This enables a nucleophilic attack of the charged phosphate oxygen of AMP on the sulfur atom of the FAD-sulfite adduct. After rearrangement of electrons, APS is eliminated and FAD becomes reduced by two electrons, which are subsequently transferred to the iron-sulfur clusters <a href="#pone.0001514-Fritz2">[18]</a>, <a href="#pone.0001514-Schiffer1">[23]</a>, <a href="#pone.0001514-Fritz3">[24]</a>.</p>
<a id="article1.body1.sec3.sec6.p2" name="article1.body1.sec3.sec6.p2"></a><p>With respect to the comparative models-based assumption of nearly identical protein matrices surrounding the FAD cofactor, the active site channel and center in the SRP- and SOB-type alpha-subunits, the overall catalytic process of APS reduction/sulfite oxidation in SRP and SOB appeared to be identical irrespective of metabolism type the APS reductase is involved in or species it has been originated from. Indeed, as experimentally proven by enzyme assays <a href="#pone.0001514-Lampreia1">[12]</a>, <a href="#pone.0001514-Fritz2">[18]</a>, <a href="#pone.0001514-Peck1">[21]</a>, <a href="#pone.0001514-Schiffer1">[23]</a>, <a href="#pone.0001514-Fritz3">[24]</a> and demonstrated by the common occurrence of sulfur isotope fractionation during microbial-mediated sulfate reduction <a href="#pone.0001514-Habicht1">[49]</a>–<a href="#pone.0001514-Canfield1">[51]</a>, the forward but also the backward reaction is generally catalyzed by enzymes of the SRP-type (note the sulfite-oxidation unfavorable redox potential of cofactors as determined for the <em>A. fulgidus</em> protein). However, the data of UV-vis difference spectra of APS reductases from <em>A. fulgidus</em>, <em>Desulfovibrio desulfuricans</em>, and <em>Desulfovibrio vulgaris</em> (upon addition of sulfite and AMP) indicated the presence of an anionic flavosemiquinone after FAD-sulfite adduct decay and reduction of the flavin. In contrast, the enzyme isolated from the sulfur-oxidizer <em>Thiobacillus denitrificans</em> (DSM 807) did not form a stable flavosemiquinone under the same conditions, and the FAD and [4Fe-4S] clusters became immediately reduced. It has been postulated that the presence/absence of this anionic flavin radical might reflect the preferential direction of catalysis in sulfate-reducing and sulfur-oxidizing organisms <a href="#pone.0001514-Fritz2">[18]</a>, <a href="#pone.0001514-Schiffer1">[23]</a>, <a href="#pone.0001514-Fritz3">[24]</a>. On the basis of the comparative models, there appeared to be no significant structural differences between APS reductases of sulfate-reducers and sulfur-oxidizers that would be indicative for a favored oxidation reaction in the SOB-type enzymes (e.g. reverse modulation of redox potential of FAD and iron-sulfur centers) in comparison to the SRP-type proteins.</p>

</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0001514"></div><a name="pone.0001514.s001" id="pone.0001514.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001514.s001">Figure S1. </a></strong></p><p class="siDoi">doi:10.1371/journal.pone.0001514.s001</p><a id="article1.body1.sec4.supplementary-material1.caption1.p1" name="article1.body1.sec4.supplementary-material1.caption1.p1"></a><p class="postSiDOI">(8.23 MB DOC)</p>
<a name="pone.0001514.s002" id="pone.0001514.s002"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001514.s002">Figure S2. </a></strong></p><p class="siDoi">doi:10.1371/journal.pone.0001514.s002</p><a id="article1.body1.sec4.supplementary-material2.caption1.p1" name="article1.body1.sec4.supplementary-material2.caption1.p1"></a><p class="postSiDOI">(2.17 MB DOC)</p>
<a name="pone.0001514.s003" id="pone.0001514.s003"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001514.s003">Figure S3. </a></strong></p><p class="siDoi">doi:10.1371/journal.pone.0001514.s003</p><a id="article1.body1.sec4.supplementary-material3.caption1.p1" name="article1.body1.sec4.supplementary-material3.caption1.p1"></a><p class="postSiDOI">(8.75 MB DOC)</p>
<a name="pone.0001514.s004" id="pone.0001514.s004"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001514.s004">Figure S4. </a></strong></p><p class="siDoi">doi:10.1371/journal.pone.0001514.s004</p><a id="article1.body1.sec4.supplementary-material4.caption1.p1" name="article1.body1.sec4.supplementary-material4.caption1.p1"></a><p class="postSiDOI">(1.17 MB DOC)</p>
<a name="pone.0001514.s005" id="pone.0001514.s005"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001514.s005">Table S1. </a></strong></p><p class="siDoi">doi:10.1371/journal.pone.0001514.s005</p><a id="article1.body1.sec4.supplementary-material5.caption1.p1" name="article1.body1.sec4.supplementary-material5.caption1.p1"></a><p class="postSiDOI">(0.76 MB DOC)</p>
<a name="pone.0001514.s006" id="pone.0001514.s006"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001514.s006">Table S2. </a></strong></p><p class="siDoi">doi:10.1371/journal.pone.0001514.s006</p><a id="article1.body1.sec4.supplementary-material6.caption1.p1" name="article1.body1.sec4.supplementary-material6.caption1.p1"></a><p class="postSiDOI">(0.21 MB DOC)</p>
<a name="pone.0001514.s007" id="pone.0001514.s007"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001514.s007">Table S3. </a></strong></p><p class="siDoi">doi:10.1371/journal.pone.0001514.s007</p><a id="article1.body1.sec4.supplementary-material7.caption1.p1" name="article1.body1.sec4.supplementary-material7.caption1.p1"></a><p class="postSiDOI">(2.10 MB DOC)</p>
<a name="pone.0001514.s008" id="pone.0001514.s008"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001514.s008">Table S4. </a></strong></p><p class="siDoi">doi:10.1371/journal.pone.0001514.s008</p><a id="article1.body1.sec4.supplementary-material8.caption1.p1" name="article1.body1.sec4.supplementary-material8.caption1.p1"></a><p class="postSiDOI">(0.18 MB DOC)</p>
<a name="pone.0001514.s009" id="pone.0001514.s009"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001514.s009">Table S5. </a></strong></p><p class="siDoi">doi:10.1371/journal.pone.0001514.s009</p><a id="article1.body1.sec4.supplementary-material9.caption1.p1" name="article1.body1.sec4.supplementary-material9.caption1.p1"></a><p class="postSiDOI">(0.18 MB DOC)</p>
<a name="pone.0001514.s010" id="pone.0001514.s010"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001514.s010">Table S6. </a></strong></p><p class="siDoi">doi:10.1371/journal.pone.0001514.s010</p><a id="article1.body1.sec4.supplementary-material10.caption1.p1" name="article1.body1.sec4.supplementary-material10.caption1.p1"></a><p class="postSiDOI">(0.19 MB DOC)</p>
</div>



<div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: JK BM. Performed the experiments: BM. Analyzed the data: JK BM. Contributed reagents/materials/analysis tools: JK. Wrote the paper: JK BM.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0001514-Rabus1" id="pone.0001514-Rabus1"></a>Rabus R, Hansen TA, Widdel F (1999)  Dissimilatory sulfate- and sulfur-reducing prokaryotes. In: Dworkin M, Schleifer K-H, Stackebrandt E, editors. The prokaryotes: an evolving electronic database for the microbiological community. New York, N. Y.: Springer. pp. 1–87.  <ul class="find-nolinks"></ul></li><li><span class="label">2.
              </span><a name="pone.0001514-Hansen1" id="pone.0001514-Hansen1"></a>Hansen TA (1994) Metabolism of sulfate-reducing prokaryotes. Antonie Van Leeuwenhoek International Journal of General and Molecular Microbiology  66: 165–185.  <ul class="find" data-citedArticleID="1048356" data-doi="10.1007/bf00871638"><li><a href="http://dx.doi.org/10.1007/bf00871638" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Metabolism+of+sulfate-reducing+prokaryotes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Metabolism+of+sulfate-reducing+prokaryotes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0001514-Brser1" id="pone.0001514-Brser1"></a>Brüser T, Lens PNL, Trüper HG (2000)  The biological sulfur cycle. In: Lens PNL, Pol LH, editors. Environmental Technologies to Treat Sulfur Pollution. London: IWA Publishing. pp. 47–86.  <ul class="find-nolinks"></ul></li><li><span class="label">4.
              </span><a name="pone.0001514-Dahl1" id="pone.0001514-Dahl1"></a>Dahl C, Trüper HG (2001) Sulfite reductase and APS reductase from <em>Archaeoglobus fulgidus</em>. Methods in Enzymology 427–441.  <ul class="find" data-citedArticleID="1048332" data-doi="10.1016/s0076-6879(01)31074-1"><li><a href="http://dx.doi.org/10.1016/s0076-6879(01)31074-1" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Sulfite+reductase+and+APS+reductase+from+Archaeoglobus+fulgidus." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Sulfite+reductase+and+APS+reductase+from+Archaeoglobus+fulgidus.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0001514-Hipp1" id="pone.0001514-Hipp1"></a>Hipp WM, Pott AS, Thum-Schmitz N, Faath I, Dahl C, et al.  (1997) Towards the phylogeny of APS reductases and sirohaem sulfite reductases in sulfate-reducing and sulfur-oxidizing prokaryotes. Microbiology-Uk  143: 2891–2902.  <ul class="find" data-citedArticleID="1048360" data-doi="10.1099/00221287-143-9-2891"><li><a href="http://dx.doi.org/10.1099/00221287-143-9-2891" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Towards+the+phylogeny+of+APS+reductases+and+sirohaem+sulfite+reductases+in+sulfate-reducing+and+sulfur-oxidizing+prokaryotes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Towards+the+phylogeny+of+APS+reductases+and+sirohaem+sulfite+reductases+in+sulfate-reducing+and+sulfur-oxidizing+prokaryotes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0001514-Trper1" id="pone.0001514-Trper1"></a>Trüper HG, Fischer U (1982) Anaerobic oxidation of sulfur-compounds as electron-donors for bacterial photosynthesis. Philosophical Transactions of the Royal Society of London Series B-Biological Sciences  298: 529–542.  <ul class="find" data-citedArticleID="1048422" data-doi="10.1098/rstb.1982.0095"><li><a href="http://dx.doi.org/10.1098/rstb.1982.0095" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Anaerobic+oxidation+of+sulfur-compounds+as+electron-donors+for+bacterial+photosynthesis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Anaerobic+oxidation+of+sulfur-compounds+as+electron-donors+for+bacterial+photosynthesis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0001514-Dahl2" id="pone.0001514-Dahl2"></a>Dahl C, Trüper HG (1994) Enzymes of dissimilatory sulfide oxidation in phototrophic sulfur bacteria. Methods in Enzymology 400–421.  <ul class="find" data-citedArticleID="1048334" data-doi="10.1016/0076-6879(94)43030-6"><li><a href="http://dx.doi.org/10.1016/0076-6879(94)43030-6" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Enzymes+of+dissimilatory+sulfide+oxidation+in+phototrophic+sulfur+bacteria." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Enzymes+of+dissimilatory+sulfide+oxidation+in+phototrophic+sulfur+bacteria.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0001514-Taylor1" id="pone.0001514-Taylor1"></a>Taylor BF (1994) Adenylylsulfate reductases from Thiobacilli. Methods in Enzymology. San Diego, California: Academic Press. pp. 393–400.  <ul class="find-nolinks"></ul></li><li><span class="label">9.
              </span><a name="pone.0001514-Schedel1" id="pone.0001514-Schedel1"></a>Schedel M, Trüper HG (1980) Anaerobic oxidation of thiosulfate and elemental sulfur in <em>Thiobacillus denitrificans</em>. Archives of Microbiology  124: 205–210.  <ul class="find" data-citedArticleID="1048406" data-doi="10.1007/bf00427728"><li><a href="http://dx.doi.org/10.1007/bf00427728" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Anaerobic+oxidation+of+thiosulfate+and+elemental+sulfur+in+Thiobacillus+denitrificans." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Anaerobic+oxidation+of+thiosulfate+and+elemental+sulfur+in+Thiobacillus+denitrificans.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0001514-Brune1" id="pone.0001514-Brune1"></a>Brune DC (1995)  Sulfur compounds as photosynthetic electron donors. In: Blankenship RE, Madigan MT, Bauer CE, editors. Anoxygenic Photosynthetic Bacteria. Dordrecht: Kluwer Academic Publishers. pp. 847–870.  <ul class="find-nolinks"></ul></li><li><span class="label">11.
              </span><a name="pone.0001514-Sperling1" id="pone.0001514-Sperling1"></a>Sperling D, Kappler U, Truper HG, Dahl C (2001) Dissimilatory ATP sulfurylase from <em>Archaeoglobus fulgidus</em>. Methods in Enzymology 419–427.  <ul class="find" data-citedArticleID="1048416" data-doi="10.1016/s0076-6879(01)31073-x"><li><a href="http://dx.doi.org/10.1016/s0076-6879(01)31073-x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Dissimilatory+ATP+sulfurylase+from+Archaeoglobus+fulgidus." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Dissimilatory+ATP+sulfurylase+from+Archaeoglobus+fulgidus.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0001514-Lampreia1" id="pone.0001514-Lampreia1"></a>Lampreia J, Pereira AS, Moura JJG (1994) Adenylylsulfate reductases from sulfate-reducing bacteria. Methods in Enzymology 241–260.  <ul class="find" data-citedArticleID="1048370" data-doi="10.1016/0076-6879(94)43018-7"><li><a href="http://dx.doi.org/10.1016/0076-6879(94)43018-7" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Adenylylsulfate+reductases+from+sulfate-reducing+bacteria." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Adenylylsulfate+reductases+from+sulfate-reducing+bacteria.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0001514-Fritz1" id="pone.0001514-Fritz1"></a>Fritz G, Buchert T, Huber H, Stetter KO, Kroneck PMH (2000) Adenylylsulfate reductases from archaea and bacteria are 1:1 alpha beta-heterodimeric iron-sulfur flavoenzymes - high similarity of molecular properties emphasizes their central role in sulfur metabolism. FEBS Letters  473: 63–66.  <ul class="find" data-citedArticleID="1048342" data-doi="10.1016/s0014-5793(00)01500-3"><li><a href="http://dx.doi.org/10.1016/s0014-5793(00)01500-3" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Adenylylsulfate+reductases+from+archaea+and+bacteria+are+1%E2%88%B61+alpha+beta-heterodimeric+iron-sulfur+flavoenzymes+-+high+similarity+of+molecular+properties+emphasizes+their+central+role+in+sulfur+metabolism." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Adenylylsulfate+reductases+from+archaea+and+bacteria+are+1%E2%88%B61+alpha+beta-heterodimeric+iron-sulfur+flavoenzymes+-+high+similarity+of+molecular+properties+emphasizes+their+central+role+in+sulfur+metabolism.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0001514-Pires1" id="pone.0001514-Pires1"></a>Pires RH, Lourenco AI, Morais F, Teixeira M, Xavier AV, et al.  (2003) A novel membrane-bound respiratory complex from <em>Desulfovibrio desulfuricans</em> ATCC 27774. Biochimica Et Biophysica Acta-Bioenergetics  1605: 67–82.  <ul class="find" data-citedArticleID="1048398" data-doi="10.1016/s0005-2728(03)00065-3"><li><a href="http://dx.doi.org/10.1016/s0005-2728(03)00065-3" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+novel+membrane-bound+respiratory+complex+from+Desulfovibrio+desulfuricans+ATCC+27774." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+novel+membrane-bound+respiratory+complex+from+Desulfovibrio+desulfuricans+ATCC+27774.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0001514-Haveman1" id="pone.0001514-Haveman1"></a>Haveman SA, Greene EA, Stilwell CP, Voordouw JK, Voordouw G (2004) Physiological and gene expression analysis of inhibition of <em>Desulfovibrio vulgaris</em> Hildenborough by nitrite. Journal of Bacteriology  186: 7944–7950.  <ul class="find" data-citedArticleID="1048358" data-doi="10.1128/jb.186.23.7944-7950.2004"><li><a href="http://dx.doi.org/10.1128/jb.186.23.7944-7950.2004" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Physiological+and+gene+expression+analysis+of+inhibition+of+Desulfovibrio+vulgaris+Hildenborough+by+nitrite." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Physiological+and+gene+expression+analysis+of+inhibition+of+Desulfovibrio+vulgaris+Hildenborough+by+nitrite.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0001514-Meyer1" id="pone.0001514-Meyer1"></a>Meyer B, Kuever J (2007) Phylogeny of the alpha and beta subunits of the dissimilatory adenosine-5′-phosphosulfate (APS) reductase from sulfate-reducing prokaryotes - origin and evolution of the dissimilatory sulfate-reduction pathway. Microbiology  153: 2026–2044.  <ul class="find" data-citedArticleID="1048386" data-doi="10.1099/mic.0.2006/003152-0"><li><a href="http://dx.doi.org/10.1099/mic.0.2006/003152-0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Phylogeny+of+the+alpha+and+beta+subunits+of+the+dissimilatory+adenosine-5%E2%80%B2-phosphosulfate+%28APS%29+reductase+from+sulfate-reducing+prokaryotes+-+origin+and+evolution+of+the+dissimilatory+sulfate-reduction+pathway." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Phylogeny+of+the+alpha+and+beta+subunits+of+the+dissimilatory+adenosine-5%E2%80%B2-phosphosulfate+%28APS%29+reductase+from+sulfate-reducing+prokaryotes+-+origin+and+evolution+of+the+dissimilatory+sulfate-reduction+pathway.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0001514-Meyer2" id="pone.0001514-Meyer2"></a>Meyer B, Kuever J (2007) Molecular analysis of the distribution and phylogeny of dissimilatory adenosine-5′-phosphosulfate reductase-encoding genes (<em>aprBA</em>) among sulfur-oxidizing prokaryotes. Microbiology  153: 3478–3498.  <ul class="find" data-citedArticleID="1048388" data-doi="10.1099/mic.0.2007/008250-0"><li><a href="http://dx.doi.org/10.1099/mic.0.2007/008250-0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Molecular+analysis+of+the+distribution+and+phylogeny+of+dissimilatory+adenosine-5%E2%80%B2-phosphosulfate+reductase-encoding+genes+%28aprBA%29+among+sulfur-oxidizing+prokaryotes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Molecular+analysis+of+the+distribution+and+phylogeny+of+dissimilatory+adenosine-5%E2%80%B2-phosphosulfate+reductase-encoding+genes+%28aprBA%29+among+sulfur-oxidizing+prokaryotes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0001514-Fritz2" id="pone.0001514-Fritz2"></a>Fritz G, Roth A, Schiffer A, Buchert T, Bourenkov G, et al.  (2002) Structure of adenylylsulfate reductase from the hyperthermophilic <em>Archaeoglobus fulgidus</em> at 1.6-A resolution. Proceedings of the National Academy of Sciences of the United States of America  99: 1836–1841.  <ul class="find" data-citedArticleID="1048344" data-doi="10.1073/pnas.042664399"><li><a href="http://dx.doi.org/10.1073/pnas.042664399" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Structure+of+adenylylsulfate+reductase+from+the+hyperthermophilic+Archaeoglobus+fulgidus+at+1.6-A+resolution." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Structure+of+adenylylsulfate+reductase+from+the+hyperthermophilic+Archaeoglobus+fulgidus+at+1.6-A+resolution.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0001514-Lancaster1" id="pone.0001514-Lancaster1"></a>Lancaster CRD (2003) The structure of Wolinella succinogenes quinol: Fumarate reductase and its relevance to the superfamily of succinate: Quinone oxidoreductases. Membrane Proteins  63: 131–149.  <ul class="find" data-citedArticleID="1048372" data-doi="10.1016/s0065-3233(03)63006-8"><li><a href="http://dx.doi.org/10.1016/s0065-3233(03)63006-8" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+structure+of+Wolinella+succinogenes+quinol%3A+Fumarate+reductase+and+its+relevance+to+the+superfamily+of+succinate%3A+Quinone+oxidoreductases." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+structure+of+Wolinella+succinogenes+quinol%3A+Fumarate+reductase+and+its+relevance+to+the+superfamily+of+succinate%3A+Quinone+oxidoreductases.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0001514-Lancaster2" id="pone.0001514-Lancaster2"></a>Lancaster CRD (2003) <em>Wolinella succinogenes</em> quinol : fumarate reductase and its comparison to <em>E. coli</em> succinate : quinone reductase. FEBS Letters  555: 21–28.  <ul class="find" data-citedArticleID="1048374" data-doi="10.1016/s0014-5793(03)01100-1"><li><a href="http://dx.doi.org/10.1016/s0014-5793(03)01100-1" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Wolinella+succinogenes+quinol+%3A+fumarate+reductase+and+its+comparison+to+E.+coli+succinate+%3A+quinone+reductase." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Wolinella+succinogenes+quinol+%3A+fumarate+reductase+and+its+comparison+to+E.+coli+succinate+%3A+quinone+reductase.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0001514-Peck1" id="pone.0001514-Peck1"></a>Peck HD, Deacon TE, Davidson JT (1965) Studies on Adenosine 5′-Phosphosulfate Reductase from <em>Desulfovibrio desulfuricans</em> and <em>Thiobacillus thioparus</em>. I. Assay and Purification. Biochimica Et Biophysica Acta  96: 429–446.  <ul class="find" data-citedArticleID="1048394" data-doi="10.1016/0005-2787(65)90561-7"><li><a href="http://dx.doi.org/10.1016/0005-2787(65)90561-7" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Studies+on+Adenosine+5%E2%80%B2-Phosphosulfate+Reductase+from+Desulfovibrio+desulfuricans+and+Thiobacillus+thioparus.+I.+Assay+and+Purification." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Studies+on+Adenosine+5%E2%80%B2-Phosphosulfate+Reductase+from+Desulfovibrio+desulfuricans+and+Thiobacillus+thioparus.+I.+Assay+and+Purification.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0001514-Michaels1" id="pone.0001514-Michaels1"></a>Michaels GB, Davidson JT, Peck HD (1970) A Flavin-Sulfite Adduct as an Intermediate in Reaction Catalyzed by Adenylyl Sulfate Reductase from <em>Desulfovibrio vulgaris</em>. Biochemical and Biophysical Research Communications  39: 321–&amp;.  <ul class="find" data-citedArticleID="1048390" data-doi="10.1016/0006-291x(70)90579-6"><li><a href="http://dx.doi.org/10.1016/0006-291x(70)90579-6" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+Flavin-Sulfite+Adduct+as+an+Intermediate+in+Reaction+Catalyzed+by+Adenylyl+Sulfate+Reductase+from+Desulfovibrio+vulgaris." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+Flavin-Sulfite+Adduct+as+an+Intermediate+in+Reaction+Catalyzed+by+Adenylyl+Sulfate+Reductase+from+Desulfovibrio+vulgaris.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0001514-Schiffer1" id="pone.0001514-Schiffer1"></a>Schiffer A, Fritz G, Kroneck PMH, Ermler U (2006) Reaction mechanism of the iron-sulfur flavoenzyme adenosine-5′-phosphosulfate reductase based on the structural characterization of different enzymatic states. Biochemistry  45: 2960–2967.  <ul class="find" data-citedArticleID="1048408" data-doi="10.1021/bi0521689"><li><a href="http://dx.doi.org/10.1021/bi0521689" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Reaction+mechanism+of+the+iron-sulfur+flavoenzyme+adenosine-5%E2%80%B2-phosphosulfate+reductase+based+on+the+structural+characterization+of+different+enzymatic+states." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Reaction+mechanism+of+the+iron-sulfur+flavoenzyme+adenosine-5%E2%80%B2-phosphosulfate+reductase+based+on+the+structural+characterization+of+different+enzymatic+states.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0001514-Fritz3" id="pone.0001514-Fritz3"></a>Fritz G, Buchert T, Kroneck PMH (2002) The function of the [4Fe-4S] clusters and FAD in bacterial and archaeal adenylylsulfate reductases - Evidence for flavin-catalyzed reduction of adenosine 5′-phosphosulfate. Journal of Biological Chemistry  277: 26066–26073.  <ul class="find" data-citedArticleID="1048346" data-doi="10.1074/jbc.m203397200"><li><a href="http://dx.doi.org/10.1074/jbc.m203397200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+function+of+the+%5B4Fe-4S%5D+clusters+and+FAD+in+bacterial+and+archaeal+adenylylsulfate+reductases+-+Evidence+for+flavin-catalyzed+reduction+of+adenosine+5%E2%80%B2-phosphosulfate." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+function+of+the+%5B4Fe-4S%5D+clusters+and+FAD+in+bacterial+and+archaeal+adenylylsulfate+reductases+-+Evidence+for+flavin-catalyzed+reduction+of+adenosine+5%E2%80%B2-phosphosulfate.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0001514-MartiRenom1" id="pone.0001514-MartiRenom1"></a>Marti-Renom MA, Stuart AC, Fiser A, Sanchez R, Melo F, et al.  (2000) Comparative protein structure modeling of genes and genomes. Annual Review of Biophysics and Biomolecular Structure  29: 291–325.  <ul class="find" data-citedArticleID="1048382" data-doi="10.1146/annurev.biophys.29.1.291"><li><a href="http://dx.doi.org/10.1146/annurev.biophys.29.1.291" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Comparative+protein+structure+modeling+of+genes+and+genomes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Comparative+protein+structure+modeling+of+genes+and+genomes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0001514-Fiser1" id="pone.0001514-Fiser1"></a>Fiser A, Sanchez R, Melo F, Sali A (2001)  Comparative protein structure modeling. In: Watanabe M, Roux B, MacKerell AD, Becker O, editors. Computational Biochemistry and Biophysics. pp. 275–312.  <ul class="find-nolinks"></ul></li><li><span class="label">27.
              </span><a name="pone.0001514-Krieger1" id="pone.0001514-Krieger1"></a>Krieger E, Nabuurs SB, Vriend G (2003)  Homology modeling. In: Bourne PE, Weissig H, editors. Structural Bioinformatics. Wiley.   <ul class="find-nolinks"></ul></li><li><span class="label">28.
              </span><a name="pone.0001514-Lesk1" id="pone.0001514-Lesk1"></a>Lesk AM, Cothia C (1986) The response of protein structures to amino-acid sequence changes. Philosophical Transactions of the Royal Society of London B Biological Sciences  317: 345–356.  <ul class="find" data-citedArticleID="1048378" data-doi="10.1098/rsta.1986.0044"><li><a href="http://dx.doi.org/10.1098/rsta.1986.0044" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+response+of+protein+structures+to+amino-acid+sequence+changes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+response+of+protein+structures+to+amino-acid+sequence+changes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0001514-Poirot1" id="pone.0001514-Poirot1"></a>Poirot O, Suhre K, Abergel C, O'Toole E, Notredame C (2004) 3DCoffee@igs: a web server for combining sequences and structures into a multiple sequence alignment. Nucleic Acids Research  32: W37–W40.  <ul class="find" data-citedArticleID="1048400" data-doi="10.1093/nar/gkh382"><li><a href="http://dx.doi.org/10.1093/nar/gkh382" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=3DCoffee%40igs%3A+a+web+server+for+combining+sequences+and+structures+into+a+multiple+sequence+alignment." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%223DCoffee%40igs%3A+a+web+server+for+combining+sequences+and+structures+into+a+multiple+sequence+alignment.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0001514-Poirot2" id="pone.0001514-Poirot2"></a>Poirot O, O'Toole E, Notredame C (2003) Tcoffee@igs: a web server for computing, evaluating and combining multiple sequence alignments. Nucleic Acids Research  31: 3503–3506.  <ul class="find" data-citedArticleID="1048402" data-doi="10.1093/nar/gkg522"><li><a href="http://dx.doi.org/10.1093/nar/gkg522" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Tcoffee%40igs%3A+a+web+server+for+computing%2C+evaluating+and+combining+multiple+sequence+alignments." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Tcoffee%40igs%3A+a+web+server+for+computing%2C+evaluating+and+combining+multiple+sequence+alignments.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0001514-Guindon1" id="pone.0001514-Guindon1"></a>Guindon S, Lethiec F, Duroux P, Gascuel O (2005) PHYML Online -a web server for fast maximum likelihood-based phylogenetic inference. Nucl Acids Res  33: W557–559.  <ul class="find" data-citedArticleID="1048350" data-doi="10.1093/nar/gki352"><li><a href="http://dx.doi.org/10.1093/nar/gki352" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=PHYML+Online+-a+web+server+for+fast+maximum+likelihood-based+phylogenetic+inference." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22PHYML+Online+-a+web+server+for+fast+maximum+likelihood-based+phylogenetic+inference.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0001514-Mussmann1" id="pone.0001514-Mussmann1"></a>Mussmann M, Richter M, Lombardot T, Meyerdierks A, Kuever J, et al.  (2005) Clustered genes related to sulfate respiration in uncultured prokaryotes support the theory of their concomitant horizontal transfer. Journal of Bacteriology  187: 7126–7137.  <ul class="find" data-citedArticleID="1048392" data-doi="10.1128/jb.187.20.7126-7137.2005"><li><a href="http://dx.doi.org/10.1128/jb.187.20.7126-7137.2005" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Clustered+genes+related+to+sulfate+respiration+in+uncultured+prokaryotes+support+the+theory+of+their+concomitant+horizontal+transfer." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Clustered+genes+related+to+sulfate+respiration+in+uncultured+prokaryotes+support+the+theory+of+their+concomitant+horizontal+transfer.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0001514-Guex1" id="pone.0001514-Guex1"></a>Guex N, Peitsch MC (1997) SWISS-MODEL and the Swiss-PdbViewer: An environment for comparative protein modeling. Electrophoresis  18: 2714–2723.  <ul class="find" data-citedArticleID="1048348" data-doi="10.1002/elps.1150181505"><li><a href="http://dx.doi.org/10.1002/elps.1150181505" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=SWISS-MODEL+and+the+Swiss-PdbViewer%3A+An+environment+for+comparative+protein+modeling." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22SWISS-MODEL+and+the+Swiss-PdbViewer%3A+An+environment+for+comparative+protein+modeling.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0001514-Schwede1" id="pone.0001514-Schwede1"></a>Schwede T, Kopp J, Guex N, Peitsch MC (2003) SWISS-MODEL: an automated protein homology-modeling server. Nucl Acids Res  31: 3381–3385.  <ul class="find" data-citedArticleID="1048410" data-doi="10.1093/nar/gkg520"><li><a href="http://dx.doi.org/10.1093/nar/gkg520" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=SWISS-MODEL%3A+an+automated+protein+homology-modeling+server." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22SWISS-MODEL%3A+an+automated+protein+homology-modeling+server.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0001514-vanGunsteren1" id="pone.0001514-vanGunsteren1"></a>van Gunsteren WF, Billeter SR, Eising AA, Hünenberger PH, Krüger P, et al.  (1996) Biomolecular simulations: GROMOS96 Manual and User Guide. Zurich, Switzerland: VdF Hochschulverlag ETHZ.   <ul class="find-nolinks"></ul></li><li><span class="label">36.
              </span><a name="pone.0001514-Hooft1" id="pone.0001514-Hooft1"></a>Hooft RWW, Sander C, Vriend G (1996) Verification of protein structures: sidechain planarity. Journal of Applied Crystallography  29: 714–716.  <ul class="find" data-citedArticleID="1048362" data-doi="10.1107/s0021889896008631"><li><a href="http://dx.doi.org/10.1107/s0021889896008631" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Verification+of+protein+structures%3A+sidechain+planarity." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Verification+of+protein+structures%3A+sidechain+planarity.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">37.
              </span><a name="pone.0001514-Sippl1" id="pone.0001514-Sippl1"></a>Sippl MJ (1993) Recognition of errors in three-dimensional structures of proteins. Proteins  17: 355–362.  <ul class="find" data-citedArticleID="1048414" data-doi="10.1002/prot.340170404"><li><a href="http://dx.doi.org/10.1002/prot.340170404" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Recognition+of+errors+in+three-dimensional+structures+of+proteins." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Recognition+of+errors+in+three-dimensional+structures+of+proteins.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">38.
              </span><a name="pone.0001514-Melo1" id="pone.0001514-Melo1"></a>Melo F, Feytmans E (1998) Assessing protein structures with a non-local atomic interaction energy. Journal of Molecular Biology  277: 1141–1152.  <ul class="find" data-citedArticleID="1048384" data-doi="10.1006/jmbi.1998.1665"><li><a href="http://dx.doi.org/10.1006/jmbi.1998.1665" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Assessing+protein+structures+with+a+non-local+atomic+interaction+energy." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Assessing+protein+structures+with+a+non-local+atomic+interaction+energy.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">39.
              </span><a name="pone.0001514-Luthy1" id="pone.0001514-Luthy1"></a>Luthy R, Bowie JU, Eisenberg D (1992) Assessment of protein models with three-dimensional profiles. Nature  356: 83–85.  <ul class="find" data-citedArticleID="1048380" data-doi="10.1038/356083a0"><li><a href="http://dx.doi.org/10.1038/356083a0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Assessment+of+protein+models+with+three-dimensional+profiles." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Assessment+of+protein+models+with+three-dimensional+profiles.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">40.
              </span><a name="pone.0001514-Schwede2" id="pone.0001514-Schwede2"></a>Schwede T, Diemand A, Guex N, Peitsch MC (2000) Protein structure computing in the genomic era. Research in Microbiology  151: 107–112.  <ul class="find" data-citedArticleID="1048412" data-doi="10.1016/s0923-2508(00)00121-2"><li><a href="http://dx.doi.org/10.1016/s0923-2508(00)00121-2" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Protein+structure+computing+in+the+genomic+era." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Protein+structure+computing+in+the+genomic+era.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">41.
              </span><a name="pone.0001514-Tindall1" id="pone.0001514-Tindall1"></a>Tindall BJ, Wray V, Huber R, Collins MD (1991) A novel, fully saturated cyclic menaquinone in the archaeabacterium <em>Pyrobaculum organotrophum</em>. Systematic and Applied Microbiology  14: 218–221.  <ul class="find" data-citedArticleID="1048420" data-doi="10.1016/s0723-2020(11)80371-1"><li><a href="http://dx.doi.org/10.1016/s0723-2020(11)80371-1" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+novel%2C+fully+saturated+cyclic+menaquinone+in+the+archaeabacterium+Pyrobaculum+organotrophum." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+novel%2C+fully+saturated+cyclic+menaquinone+in+the+archaeabacterium+Pyrobaculum+organotrophum.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">42.
              </span><a name="pone.0001514-Itoh1" id="pone.0001514-Itoh1"></a>Itoh T (2003) Taxonomy of nonmethanogenic hyperthermophilic and related thermophilic archaea. Journal of Bioscience and Bioengineering  96: 203–212.  <ul class="find" data-citedArticleID="1048366" data-doi="10.1016/s1389-1723(03)80183-4"><li><a href="http://dx.doi.org/10.1016/s1389-1723(03)80183-4" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Taxonomy+of+nonmethanogenic+hyperthermophilic+and+related+thermophilic+archaea." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Taxonomy+of+nonmethanogenic+hyperthermophilic+and+related+thermophilic+archaea.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">43.
              </span><a name="pone.0001514-Deisenhofer1" id="pone.0001514-Deisenhofer1"></a>Deisenhofer J, Michel H (1989) The photosynthetic reaction centre from the purple bacterium <em>Rhodopseudomonas viridis</em>. EMBO (European Molecular Biology Organization) Journal  8: 2149–2170.  <ul class="find" data-citedArticleID="1048336" data-doi="10.1007/978-1-4615-7598-6_55"><li><a href="http://dx.doi.org/10.1007/978-1-4615-7598-6_55" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+photosynthetic+reaction+centre+from+the+purple+bacterium+Rhodopseudomonas+viridis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+photosynthetic+reaction+centre+from+the+purple+bacterium+Rhodopseudomonas+viridis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">44.
              </span><a name="pone.0001514-Pelletier1" id="pone.0001514-Pelletier1"></a>Pelletier H, Kraut J (1992) Crystal structure of a complex between electron transfer partners, cytochrome c peroxidase and cytochrome c. Science  258: 1748–1755.  <ul class="find" data-citedArticleID="1048396" data-doi="10.1126/science.1334573"><li><a href="http://dx.doi.org/10.1126/science.1334573" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Crystal+structure+of+a+complex+between+electron+transfer+partners%2C+cytochrome+c+peroxidase+and+cytochrome+c." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Crystal+structure+of+a+complex+between+electron+transfer+partners%2C+cytochrome+c+peroxidase+and+cytochrome+c.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">45.
              </span><a name="pone.0001514-Huber1" id="pone.0001514-Huber1"></a>Huber H, Huber R, Stetter KO (2002)  <em>Thermoproteales</em>. In: Dworkin M, Falkow E, Rosenberg E, Schleifer K-H, Stackebrandt E, editors. The prokaryotes An evolving electronic resource for the microbial community. New York, N. Y.: Fischer Verlag.   <ul class="find-nolinks"></ul></li><li><span class="label">46.
              </span><a name="pone.0001514-Amo1" id="pone.0001514-Amo1"></a>Amo T, Paje MLF, Inagaki A, Ezaki S, Atomi H, et al.  (2002) <em>Pyrobaculum calidifontis</em> sp. nov., a novel hyperthermophilic archaeon that grows in atmospheric air. Archaea  1: 113–121.  <ul class="find" data-citedArticleID="1048324" data-doi="10.1155/2002/616075"><li><a href="http://dx.doi.org/10.1155/2002/616075" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Pyrobaculum+calidifontis+sp.+nov.%2C+a+novel+hyperthermophilic+archaeon+that+grows+in+atmospheric+air." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Pyrobaculum+calidifontis+sp.+nov.%2C+a+novel+hyperthermophilic+archaeon+that+grows+in+atmospheric+air.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">47.
              </span><a name="pone.0001514-FitzGibbon1" id="pone.0001514-FitzGibbon1"></a>Fitz-Gibbon ST, Ladner H, Kim UJ, Stetter KO, Simon MI, et al.  (2002) Genome sequence of the hyperthermophilic crenarchaeon <em>Pyrobaculum aerophilum</em>. Proceedings of the National Academy of Sciences of the United States of America  99: 984–989.  <ul class="find" data-citedArticleID="1048340" data-doi="10.1073/pnas.241636498"><li><a href="http://dx.doi.org/10.1073/pnas.241636498" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genome+sequence+of+the+hyperthermophilic+crenarchaeon+Pyrobaculum+aerophilum." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genome+sequence+of+the+hyperthermophilic+crenarchaeon+Pyrobaculum+aerophilum.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">48.
              </span><a name="pone.0001514-Lennon1" id="pone.0001514-Lennon1"></a>Lennon BW, Williams CH, Ludwig ML (1999) Crystal structure of refined thioredoxin reductase from <em>Escherichia coli</em>: Structural flexibility in the isoalloxazine ring of the flavin adenine dinucleotide cofactor. Protein Science  8: 2366–2379.  <ul class="find" data-citedArticleID="1048376" data-doi="10.1110/ps.8.11.2366"><li><a href="http://dx.doi.org/10.1110/ps.8.11.2366" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Crystal+structure+of+refined+thioredoxin+reductase+from+Escherichia+coli%3A+Structural+flexibility+in+the+isoalloxazine+ring+of+the+flavin+adenine+dinucleotide+cofactor." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Crystal+structure+of+refined+thioredoxin+reductase+from+Escherichia+coli%3A+Structural+flexibility+in+the+isoalloxazine+ring+of+the+flavin+adenine+dinucleotide+cofactor.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">49.
              </span><a name="pone.0001514-Habicht1" id="pone.0001514-Habicht1"></a>Habicht KS, Canfield DE (2001) Isotope fractionation by sulfate-reducing natural populations and the isotopic composition of sulfide in marine sediments. Geology  29: 555–558.  <ul class="find" data-citedArticleID="1048352" data-doi="10.1130/0091-7613(2001)029<0555:ifbsrn&gt;2.0.co;2"><li><a href="http://dx.doi.org/10.1130/0091-7613(2001)029<0555:ifbsrn&gt;2.0.co;2" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Isotope+fractionation+by+sulfate-reducing+natural+populations+and+the+isotopic+composition+of+sulfide+in+marine+sediments." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Isotope+fractionation+by+sulfate-reducing+natural+populations+and+the+isotopic+composition+of+sulfide+in+marine+sediments.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">50.
              </span><a name="pone.0001514-Habicht2" id="pone.0001514-Habicht2"></a>Habicht KS, Salling LL, Thamdrup B, Canfield DE (2005) Effect of low sulfate concentrations on lactate oxidation and isotope Fractionation during sulfate reduction by <em>Archaeoglobus fulgidus</em> strain Z. Applied and Environmental Microbiology  71: 3770–3777.  <ul class="find" data-citedArticleID="1048354" data-doi="10.1128/aem.71.7.3770-3777.2005"><li><a href="http://dx.doi.org/10.1128/aem.71.7.3770-3777.2005" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Effect+of+low+sulfate+concentrations+on+lactate+oxidation+and+isotope+Fractionation+during+sulfate+reduction+by+Archaeoglobus+fulgidus+strain+Z." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Effect+of+low+sulfate+concentrations+on+lactate+oxidation+and+isotope+Fractionation+during+sulfate+reduction+by+Archaeoglobus+fulgidus+strain+Z.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">51.
              </span><a name="pone.0001514-Canfield1" id="pone.0001514-Canfield1"></a>Canfield DE (2001)  Biogeochemistry of sulfur isotopes. In: Valley JW, Cole DR, editors. Stable Isotope Geochemistry: Reviews in Mineralogy and Geochemistry. Washington, DC: Mineralogy Society of America. pp. 607–636.  <ul class="find-nolinks"></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001514.XML" value="151068"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001514.PDF" value="5964025"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001514.g001.PNG_L" value="350980"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001514.g001.PNG_M" value="284384"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001514.g001.PNG_S" value="13162"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001514.g001.TIF" value="511492"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001514.g001.PNG_I" value="92956"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001514.g002.PNG_L" value="69774"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001514.g002.PNG_M" value="82030"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001514.g002.PNG_S" value="9299"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001514.g002.TIF" value="104190"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001514.g002.PNG_I" value="33664"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001514.g003.PNG_L" value="615736"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001514.g003.PNG_M" value="188533"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001514.g003.PNG_S" value="14887"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001514.g003.TIF" value="1120296"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001514.g003.PNG_I" value="107314"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001514.t001.PNG_L" value="356207"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001514.t001.PNG_M" value="210821"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001514.t001.PNG_S" value="17711"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001514.t001.TIF" value="1442280"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001514.t001.PNG_I" value="64508"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001514.e001.PNG" value="4868"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001514.e001.TIF" value="20820"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001514.e002.PNG" value="4888"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001514.e002.TIF" value="21164"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001514.e003.PNG" value="4887"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001514.e003.TIF" value="21164"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001514.g004.PNG_L" value="2442805"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001514.g004.PNG_M" value="276136"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001514.g004.PNG_S" value="18781"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001514.g004.TIF" value="3684426"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001514.g004.PNG_I" value="86188"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001514.g005.PNG_L" value="812359"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001514.g005.PNG_M" value="256491"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001514.g005.PNG_S" value="18934"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001514.g005.TIF" value="1283248"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001514.g005.PNG_I" value="152690"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001514.t002.PNG_L" value="387939"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001514.t002.PNG_M" value="182566"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001514.t002.PNG_S" value="15563"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001514.t002.TIF" value="1873188"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001514.t002.PNG_I" value="55454"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001514.g006.PNG_L" value="683695"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001514.g006.PNG_M" value="228569"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001514.g006.PNG_S" value="17321"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001514.g006.TIF" value="1111112"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001514.g006.PNG_I" value="137598"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001514.g007.PNG_L" value="857268"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001514.g007.PNG_M" value="293964"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001514.g007.PNG_S" value="20672"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001514.g007.TIF" value="1363570"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001514.g007.PNG_I" value="178422"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001514.g008.PNG_L" value="870496"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001514.g008.PNG_M" value="292690"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001514.g008.PNG_S" value="19968"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001514.g008.TIF" value="1347528"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001514.g008.PNG_I" value="172982"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001514.g009.PNG_L" value="736768"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001514.g009.PNG_M" value="288007"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001514.g009.PNG_S" value="17467"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001514.g009.TIF" value="1159218"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001514.g009.PNG_I" value="120643"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001514.g010.PNG_L" value="138046"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001514.g010.PNG_M" value="71929"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001514.g010.PNG_S" value="8135"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001514.g010.TIF" value="178598"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001514.g010.PNG_I" value="22969"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001514.s001.DOC" value="8252928"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001514.s002.DOC" value="2166272"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001514.s003.DOC" value="8765952"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001514.s004.DOC" value="1168896"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001514.s005.DOC" value="802304"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001514.s006.DOC" value="211968"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001514.s007.DOC" value="2243584"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001514.s008.DOC" value="179200"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001514.s009.DOC" value="184320"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001514.s010.DOC" value="196608"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001514&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001514" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001514&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0001514&volume=&issue=&title=Homology Modeling of Dissimilatory APS Reductases (AprBA) of Sulfur-Oxidizing and Sulfate-Reducing Prokaryotes&author_name=Birte%20Meyer%2C%20Jan%20Kuever&start_page=1&end_page=17" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001514" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001514" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0001514" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001514&amp;t=Homology%20Modeling%20of%20Dissimilatory%20APS%20Reductases%20(AprBA)%20of%20Sulfur-Oxidizing%20and%20Sulfate-Reducing%20Prokaryotes" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001514&title=Homology%20Modeling%20of%20Dissimilatory%20APS%20Reductases%20(AprBA)%20of%20Sulfur-Oxidizing%20and%20Sulfate-Reducing%20Prokaryotes&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001514&amp;title=Homology%20Modeling%20of%20Dissimilatory%20APS%20Reductases%20(AprBA)%20of%20Sulfur-Oxidizing%20and%20Sulfate-Reducing%20Prokaryotes" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001514" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0001514&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Homology Modeling of Dissimilatory APS Reductases (AprBA) of Sulfur-Oxidizing and Sulfate-Reducing Prokaryotes';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0001514';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Homology%20Modeling%20of%20Dissimilatory%20APS%20Reductases%20(AprBA)%20of%20Sulfur-Oxidizing%20and%20Sulfate-Reducing%20Prokaryotes http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001514" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0001514" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">













          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Cofactors+%28biochemistry%29%22" title="Search for articles in the subject area:'Cofactors (biochemistry)'"><div class="flagText">Cofactors (biochemistry)</div></a>
              <div data-categoryid="31901" data-articleid="25854"
                   data-categoryname="Cofactors (biochemistry)"
                   class="flagImage" title="Flag 'Cofactors (biochemistry)' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Oxidation-reduction+reactions%22" title="Search for articles in the subject area:'Oxidation-reduction reactions'"><div class="flagText">Oxidation-reduction reactions</div></a>
              <div data-categoryid="20673" data-articleid="25854"
                   data-categoryname="Oxidation-reduction reactions"
                   class="flagImage" title="Flag 'Oxidation-reduction reactions' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Phylogenetics%22" title="Search for articles in the subject area:'Phylogenetics'"><div class="flagText">Phylogenetics</div></a>
              <div data-categoryid="46093" data-articleid="25854"
                   data-categoryname="Phylogenetics"
                   class="flagImage" title="Flag 'Phylogenetics' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Protein+structure%22" title="Search for articles in the subject area:'Protein structure'"><div class="flagText">Protein structure</div></a>
              <div data-categoryid="31877" data-articleid="25854"
                   data-categoryname="Protein structure"
                   class="flagImage" title="Flag 'Protein structure' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Sequence+alignment%22" title="Search for articles in the subject area:'Sequence alignment'"><div class="flagText">Sequence alignment</div></a>
              <div data-categoryid="47429" data-articleid="25854"
                   data-categoryname="Sequence alignment"
                   class="flagImage" title="Flag 'Sequence alignment' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Sulfates%22" title="Search for articles in the subject area:'Sulfates'"><div class="flagText">Sulfates</div></a>
              <div data-categoryid="20973" data-articleid="25854"
                   data-categoryname="Sulfates"
                   class="flagImage" title="Flag 'Sulfates' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Sulfites%22" title="Search for articles in the subject area:'Sulfites'"><div class="flagText">Sulfites</div></a>
              <div data-categoryid="21307" data-articleid="25854"
                   data-categoryname="Sulfites"
                   class="flagImage" title="Flag 'Sulfites' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Sulfur%22" title="Search for articles in the subject area:'Sulfur'"><div class="flagText">Sulfur</div></a>
              <div data-categoryid="20755" data-articleid="25854"
                   data-categoryname="Sulfur"
                   class="flagImage" title="Flag 'Sulfur' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=8390'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=6135'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=1326&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>

<div class="block sidebar-comments">
    <div class="header">
        <h3>Comments</h3>
    </div>
      <p><a href="/annotation/listThread.action?root=20709">Referee comments: Referee 1</a><br>Posted by PLoS_ONE_Group</p>
</div>

</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
