

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Gene Sampling Strategies for Multi-Locus Population Estimates of Genetic Diversity (θ)</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0000160"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0000160" />

    <meta name="citation_title" content="Gene Sampling Strategies for Multi-Locus Population Estimates of Genetic Diversity (θ)"/>
    <meta itemprop="name" content="Gene Sampling Strategies for Multi-Locus Population Estimates of Genetic Diversity (θ)"/>

      <meta name="citation_author" content="Matthew D. Carling"/>
            <meta name="citation_author_institution" content="Museum of Natural Science, Louisiana State University, Baton Rouge, Louisiana, United States of America"/>
            <meta name="citation_author_institution" content="Department of Biological Sciences, Louisiana State University, Baton Rouge, Louisiana, United States of America"/>
      <meta name="citation_author" content="Robb T. Brumfield"/>
            <meta name="citation_author_institution" content="Museum of Natural Science, Louisiana State University, Baton Rouge, Louisiana, United States of America"/>

    <meta name="citation_date" content="2007/1/17"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0000160.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e160"/>
    <meta name="citation_issue" content="1"/>
    <meta name="citation_volume" content="2"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Coalescing into the 21st century: An overview and prospects of coalescent theory.; citation_author=YX Fu; citation_author=WH Li; citation_journal_title=Theor Popul Biol; citation_volume=56; citation_number=1; citation_pages=1-10; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Inferring phylogenies from mtDNA variation - Mitochondrial-gene trees versus nuclear-gene trees.; citation_author=WS Moore; citation_journal_title=Evolution; citation_volume=49; citation_number=2; citation_pages=718-726; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Mitochondrial-gene trees versus nuclear-gene trees, a reply to Hoelzer.; citation_author=WS Moore; citation_journal_title=Evolution; citation_volume=51; citation_number=3; citation_pages=627-629; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Mechanisms of population differentiation in marbled murrelets: Historical versus contemporary processes.; citation_author=BC Congdon; citation_author=JF Piatt; citation_author=K Martin; citation_author=VL Friesen; citation_journal_title=Evolution; citation_volume=54; citation_number=4; citation_pages=974-986; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Testing models of migration and isolation among populations of chinook salmon (Oncorhynchus tschawytscha).; citation_author=MJ Ford; citation_journal_title=Evolution; citation_volume=52; citation_number=5; citation_pages=539-557; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Nuclear DNA analyses in genetic studies of populations: Practice, problems and prospects.; citation_author=DX Zhang; citation_author=GM Hewitt; citation_journal_title=Mol Ecol; citation_volume=12; citation_number=6; citation_pages=563-584; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Prospects for nuclear gene phylogeography.; citation_author=MP Hare; citation_journal_title=Trends Ecol Evol; citation_volume=16; citation_number=7; citation_pages=700-706; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Revealing the demographic histories of species using DNA sequences.; citation_author=BC Emerson; citation_author=E Paradis; citation_author=C Thebaud; citation_journal_title=Trends Ecol Evol; citation_volume=16; citation_number=8; citation_pages=707-716; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Statistical phylogeography.; citation_author=LL Knowles; citation_author=WP Maddison; citation_journal_title=Mol Ecol; citation_volume=11; citation_number=9; citation_pages=2623-2635; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Genealogical trees, coalescent theory and the analysis of genetic polymorphisms.; citation_author=NA Rosenberg; citation_author=M Nordborg; citation_journal_title=Nature Rev Genet; citation_volume=3; citation_number=10; citation_pages=380-390; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Perspective: Gene divergence, population divergence, and the variance in coalescence time in phylogeographic studies.; citation_author=SV Edwards; citation_author=P Beerli; citation_journal_title=Evolution; citation_volume=54; citation_number=11; citation_pages=1839-1854; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Usefulness of single nucleotide polymorphism data for estimating population parameters.; citation_author=MK Kuhner; citation_author=P Beerli; citation_author=J Yamato; citation_author=J Felsenstein; citation_journal_title=Genetics; citation_volume=156; citation_number=12; citation_pages=439-447; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Estimating ancestral population parameters.; citation_author=J Wakeley; citation_author=J Hey; citation_journal_title=Genetics; citation_volume=145; citation_number=13; citation_pages=847-855; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Optimal sequencing strategies for surveying molecular genetic diversity.; citation_author=A Pluzhnikov; citation_author=P Donnelly; citation_journal_title=Genetics; citation_volume=144; citation_number=14; citation_pages=1247-1262; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Divergence time and population-size in the lineage leading to modern humans.; citation_author=N Takahata; citation_author=Y Satta; citation_author=J Klein; citation_journal_title=Theor Popul Biol; citation_volume=48; citation_number=15; citation_pages=198-221; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=On the estimation of ancestral population sizes of modern humans.; citation_author=ZH Yang; citation_journal_title=Genet Res; citation_volume=69; citation_number=16; citation_pages=111-116; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Accuracy of coalescent likelihood estimates: Do we need more sites, more sequences, or more loci?; citation_author=J Felsenstein; citation_journal_title=Mol Biol Evol; citation_volume=23; citation_number=17; citation_pages=691-700; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Estimation of population parameters and recombination rates from single nucleotide polymorphisms.; citation_author=R Nielsen; citation_journal_title=Genetics; citation_volume=154; citation_number=18; citation_pages=931-942; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Number of segregating sites in genetic models without recombination.; citation_author=GA Watterson; citation_journal_title=Theor Popul Biol; citation_volume=7; citation_number=19; citation_pages=256-276; citation_date=1975; " />
      <meta name="citation_reference" content="citation_title=Population dynamics of HIV-1 inferred from gene sequences.; citation_author=NC Grassly; citation_author=PH Harvey; citation_author=EC Holmes; citation_journal_title=Genetics; citation_volume=151; citation_number=20; citation_pages=427-438; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=A hidden Markov Model approach to variation among sites in rate of evolution.; citation_author=J Felsenstein; citation_author=GA Churchill; citation_journal_title=Mol Biol Evol; citation_volume=13; citation_number=21; citation_pages=93-104; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Huge populations and old species of Costa Rican and Panamanian dirt frogs inferred from mitochondrial and nuclear gene sequences.; citation_author=AJ Crawford; citation_journal_title=Mol Ecol; citation_volume=12; citation_number=22; citation_pages=2525-2540; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Multilocus analysis of introgression between two sympatric sister species of Drosophila: Drosophila yakuba and D. santomea.; citation_author=A Llopert; citation_author=D Lachaise; citation_author=JA Coyne; citation_journal_title=Genetics; citation_volume=171; citation_number=23; citation_pages=197-210; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Conservation genetics and demographic history of the endangered Cape Fear shiner (Notropis mekistocholas).; citation_author=E Saillant; citation_author=JC Patton; citation_author=KE Ross; citation_author=JR Gold; citation_journal_title=Mol Ecol; citation_volume=13; citation_number=24; citation_pages=2947-2958; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Phylogeography of sexual Heteronotia binoei (Gekkonidae) in the Australian arid zone: Climatic cycling and repetitive hybridization.; citation_author=JL Strasburg; citation_author=M Kearney; citation_journal_title=Mol Ecol; citation_volume=14; citation_number=25; citation_pages=2755-2772; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Arlequin ver 3.0: An intergrated software package for population genetics data analysis.; citation_author=L Excoffier; citation_author=G Laval; citation_author=S Schneider; citation_journal_title=Evol Bioinform Online; citation_volume=1; citation_number=26; citation_pages=47-50; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=GeneTree: Comparing gene and species phylogenies using reconciled trees.; citation_author=RDM Page; citation_journal_title=Bioinformatics; citation_volume=14; citation_number=27; citation_pages=819-820; citation_date=1998; " />
      <meta name="citation_reference" content="citation_author=AJ Drummond; citation_author=A Rambaut; citation_number=28; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Estimating effective population-size from samples of sequences - Inefficiency of pairwise and segregating sites as compared to phylogenetic estimates.; citation_author=J Felsenstein; citation_journal_title=Genet Res; citation_volume=59; citation_number=29; citation_pages=139-147; citation_date=1992; " />
      <meta name="citation_reference" content="citation_title=Maximum-likelihood-estimation of population parameters.; citation_author=YX Fu; citation_author=WH Li; citation_journal_title=Genetics; citation_volume=134; citation_number=30; citation_pages=1261-1270; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=Maximum-likelihood estimation of migration rates and effective population numbers in two populations using a coalescent approach.; citation_author=P Beerli; citation_author=J Felsenstein; citation_journal_title=Genetics; citation_volume=152; citation_number=31; citation_pages=763-773; citation_date=1999; " />
      <meta name="citation_reference" content="citation_author=RR Sokal; citation_author=FJ Rohlf; citation_number=32; citation_pages=887; citation_date=1995; citation_publisher=W. H. Freeman and Company; " />
      <meta name="citation_reference" content="citation_title=Estimating effective population size and mutation rate from sequence data using Metropolis-Hastings sampling.; citation_author=MK Kuhner; citation_author=J Yamato; citation_author=J Felsenstein; citation_journal_title=Genetics; citation_volume=140; citation_number=33; citation_pages=1421-1430; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Statistical tests of neutrality of mutations.; citation_author=YX Fu; citation_author=WH Li; citation_journal_title=Genetics; citation_volume=133; citation_number=34; citation_pages=693-709; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=Genetic evidence on the demography of speciation in allopatric dolphin species.; citation_author=MP Hare; citation_author=F Cipriano; citation_author=SR Palumbi; citation_journal_title=Evolution; citation_volume=56; citation_number=35; citation_pages=804-816; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Natural selection and the genetic differentiation of coastal and Arctic populations of the Atlantic cod in northern Norway: A test involving nucleotide sequence variation at the pantophysin (PanI) locus.; citation_author=GH Pogson; citation_author=SE Fevolden; citation_journal_title=Mol Ecol; citation_volume=12; citation_number=36; citation_pages=63-74; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Single-nucleotide polymorphism characterization in species with limited available sequence information: High nucleotide diversity revealed in the avian genome.; citation_author=CR Primmer; citation_author=T Borge; citation_author=J Lindell; citation_author=GP Saetre; citation_journal_title=Mol Ecol; citation_volume=11; citation_number=37; citation_pages=603-612; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Genome-scale approaches to resolving incongruence in molecular phylogenies.; citation_author=A Rokas; citation_author=BL Williams; citation_author=N King; citation_author=SB Carroll; citation_journal_title=Nature; citation_volume=425; citation_number=38; citation_pages=798-804; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Perspective: Highly variable loci and their interpretation in evolution and conservation.; citation_author=PW Hedrick; citation_journal_title=Evolution; citation_volume=53; citation_number=39; citation_pages=313-318; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Recent trends in population genetics: More data! More math! Simple models?; citation_author=J Wakeley; citation_journal_title=J Hered; citation_volume=95; citation_number=40; citation_pages=397-405; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=The population genetics of the origin and divergence of the Drosophila simulans complex species.; citation_author=RM Kliman; citation_author=P Andolfatto; citation_author=JA Coyne; citation_author=F Depaulis; citation_author=M Kreitman; citation_journal_title=Genetics; citation_volume=156; citation_number=41; citation_pages=1913-1931; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Genetic evidence for complex speciation of humans and chimpanzees.; citation_author=N Patterson; citation_author=DJ Richter; citation_author=S Gnerre; citation_author=ES Lander; citation_author=D Reich; citation_journal_title=Nature; citation_volume=441; citation_number=42; citation_pages=1103-1108; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Patterns of DNA variability at X-linked loci in Mus domesticus.; citation_author=MW Nachman; citation_journal_title=Genetics; citation_volume=147; citation_number=43; citation_pages=1303-1316; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=DNA variability and recombination rates at X-linked loci in humans.; citation_author=MW Nachman; citation_author=VL Bauer; citation_author=SL Crowell; citation_author=CF Aquadro; citation_journal_title=Genetics; citation_volume=150; citation_number=44; citation_pages=1133-1141; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Human SNP variability and mutation rate are higher in regions of high recombination.; citation_author=MJ Lercher; citation_author=LD Hurst; citation_journal_title=Trends Genet; citation_volume=18; citation_number=45; citation_pages=337-340; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Molecular population-genetics of the distal portion of the X-chromosome in Drosophila - Evidence for genetic hitchhiking of the yellow-achaete region.; citation_author=DJ Begun; citation_author=CF Aquadro; citation_journal_title=Genetics; citation_volume=129; citation_number=46; citation_pages=1147-1158; citation_date=1991; " />
      <meta name="citation_reference" content="citation_title=Lack of polymorphism on the Drosophila 4th chromosome resulting from selection.; citation_author=AJ Berry; citation_author=JW Ajioka; citation_author=M Kreitman; citation_journal_title=Genetics; citation_volume=129; citation_number=47; citation_pages=1111-1117; citation_date=1991; " />
      <meta name="citation_reference" content="citation_title=Single nucleotide polymorphisms and recombination rate in humans.; citation_author=MW Nachman; citation_journal_title=Trends Genet; citation_volume=17; citation_number=48; citation_pages=481-485; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Variation in recombination rate across the genome: Evidence and implications.; citation_author=MW Nachman; citation_journal_title=Curr Opin Genetics Devel; citation_volume=12; citation_number=49; citation_pages=657-663; citation_date=2002; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000160" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Gene Sampling Strategies for Multi-Locus Population Estimates of Genetic Diversity (θ)"/>
    <meta name="twitter:description" content="BackgroundTheoretical work suggests that data from multiple nuclear loci provide better estimates of population genetic parameters than do single loci, but just how many loci are needed and how much sequence is required from each has been little explored.Methodology/Principle FindingsTo investigate how much data is required to estimate the population genetic parameter &theta; (4Ne&mu;) accurately under ideal circumstances, we simulated datasets of DNA sequences under three values of &theta; per site (0.1, 0.01, 0.001), varying in both the total number of base pairs sequenced per individual and the number of equal-length loci. From these datasets we estimated &theta; using the maximum likelihood coalescent framework implemented in the computer program Migrate. Our results corroborated the theoretical expectation that increasing the number of loci impacted the accuracy of the estimate more than increasing the sequence length at single loci. However, when the value of &theta; was low (0.001), the per-locus sequence length was also important for estimating &theta; accurately, something that has not been emphasized in previous work.Conclusions/SignificanceAccurate estimation of &theta; required data from at least 25 independently evolving loci. Beyond this, there was little added benefit in terms of decreasing the squared coefficient of variation of the coalescent estimates relative to the extra effort required to sample more loci."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0000160.g002"/>

  <meta property="og:title" content="Gene Sampling Strategies for Multi-Locus Population Estimates of Genetic Diversity (θ)" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000160" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=4086'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=3308'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=9211&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0000160">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000160" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000160&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Gene Sampling Strategies for Multi-Locus Population Estimates of Genetic Diversity (<i>θ</i>)
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Matthew D. Carling
              <span class="corresponding">mail</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:mcarli1@lsu.edu">mcarli1@lsu.edu</a>
            </p>

                <p>Affiliations:
                  Museum of Natural Science, Louisiana State University, Baton Rouge, Louisiana, United States of America, 
                  Department of Biological Sciences, Louisiana State University, Baton Rouge, Louisiana, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Robb T. Brumfield
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Museum of Natural Science, Louisiana State University, Baton Rouge, Louisiana, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: January 17, 2007</li>
    <li>DOI: 10.1371/journal.pone.0000160</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0000160" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0000160" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000160" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0000160" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0000160-t001" data-doi="info:doi/10.1371/journal.pone.0000160" data-uri="info:doi/10.1371/journal.pone.0000160.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000160.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000160-g001" data-doi="info:doi/10.1371/journal.pone.0000160" data-uri="info:doi/10.1371/journal.pone.0000160.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000160.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000160-g002" data-doi="info:doi/10.1371/journal.pone.0000160" data-uri="info:doi/10.1371/journal.pone.0000160.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000160.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000160">Reader Comments (0)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0000160" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2>
<h3>Background</h3>
<a id="article1.front1.article-meta1.abstract1.sec1.p1" name="article1.front1.article-meta1.abstract1.sec1.p1"></a><p>Theoretical work suggests that data from multiple nuclear loci provide better estimates of population genetic parameters than do single loci, but just how many loci are needed and how much sequence is required from each has been little explored.</p>


<h3>Methodology/Principle Findings</h3>
<a id="article1.front1.article-meta1.abstract1.sec2.p1" name="article1.front1.article-meta1.abstract1.sec2.p1"></a><p>To investigate how much data is required to estimate the population genetic parameter <em>θ</em> (4<em>N<sub>e</sub>μ</em>) accurately under ideal circumstances, we simulated datasets of DNA sequences under three values of <em>θ</em> per site (0.1, 0.01, 0.001), varying in both the total number of base pairs sequenced per individual and the number of equal-length loci. From these datasets we estimated <em>θ</em> using the maximum likelihood coalescent framework implemented in the computer program M<span class="small-caps">igrate</span>. Our results corroborated the theoretical expectation that increasing the number of loci impacted the accuracy of the estimate more than increasing the sequence length at single loci. However, when the value of <em>θ</em> was low (0.001), the per-locus sequence length was also important for estimating <em>θ</em> accurately, something that has not been emphasized in previous work.</p>


<h3>Conclusions/Significance</h3>
<a id="article1.front1.article-meta1.abstract1.sec3.p1" name="article1.front1.article-meta1.abstract1.sec3.p1"></a><p>Accurate estimation of <em>θ</em> required data from at least 25 independently evolving loci. Beyond this, there was little added benefit in terms of decreasing the squared coefficient of variation of the coalescent estimates relative to the extra effort required to sample more loci.</p>

</div>


<div class="articleinfo"><p><strong>Citation: </strong>Carling MD, Brumfield RT (2007) Gene Sampling Strategies for Multi-Locus Population Estimates of Genetic Diversity (<em>θ</em>). PLoS ONE 2(1):
          e160.
            doi:10.1371/journal.pone.0000160</p><p><strong>Academic Editor: </strong>Konrad Scheffler, University of Cape Town, South Africa</p><p><strong>Received:</strong> November 9, 2006; <strong>Accepted:</strong> December 15, 2006; <strong>Published:</strong> January 17, 2007</p><p><strong>Copyright:</strong> © 2007 Carling, Brumfield. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>This work was funded by NSF grant DEB-0332093 to RTB.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>Coalescent theory, placed in a maximum-likelihood framework, allows researchers to explicitly test hypotheses concerning the processes that shape patterns of genetic variation in natural populations <a href="#pone.0000160-Fu1">[1]</a>. Catalyzed by automated sequencing technologies as well as the availability of universal PCR primers (a direct benefit of genome projects), population genetic studies of natural populations have recently experienced a large-scale shift from single-locus investigations of cytoplasmic markers (e.g. mitochondrial, chloroplast) to multi-locus studies of autosomal and sex-linked loci. Although single-locus cytoplasmic gene trees are expected to outperform nuclear gene trees in some instances <a href="#pone.0000160-Moore1">[2]</a>, <a href="#pone.0000160-Moore2">[3]</a>, the statistical confidence of the inferences drawn from such trees is limited <a href="#pone.0000160-Congdon1">[4]</a>–<a href="#pone.0000160-Zhang1">[6]</a>. Consequently, the recombining histories of multiple nuclear loci are being used increasingly in empirical estimates of population genetic parameters <a href="#pone.0000160-Hare1">[7]</a>–<a href="#pone.0000160-Knowles1">[9]</a>.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>It is well understood that coalescent estimates of population genetic parameters (e.g. effective population size, recombination rates) are often estimated with large errors when calculated from single-locus datasets <a href="#pone.0000160-Rosenberg1">[10]</a>, <a href="#pone.0000160-Edwards1">[11]</a>. Simulation studies have demonstrated that the error rates associated with such parameter estimations can be reduced by analyzing data from multiple independent loci <a href="#pone.0000160-Kuhner1">[12]</a>, <a href="#pone.0000160-Wakeley1">[13]</a>. Statistically, adding additional loci corresponds to gaining independent replicates of the underlying evolutionary processes <a href="#pone.0000160-Pluzhnikov1">[14]</a>. Additionally, in some situations, greater precision of a population genetic parameter estimate can be achieved by sampling both shared polymorphisms and fixed differences between two populations, which can only be observed by sampling loci that have different genealogies <a href="#pone.0000160-Wakeley1">[e.g. 13]</a>. Multiple loci are also required in some methods for estimating the effective population size of ancestral populations <a href="#pone.0000160-Takahata1">[15]</a>, <a href="#pone.0000160-Yang1">[16]</a>.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>With the development of analytical methods for multiple-locus parameter estimation and the relative ease of collecting large amounts of sequence data, researchers with limited resources are concerned increasingly with how much data is required to estimate the parameters of interest accurately <a href="#pone.0000160-Felsenstein1">[17]</a>. If more loci are better, as previous work has alluded to <a href="#pone.0000160-Kuhner1">[12]</a>, <a href="#pone.0000160-Nielsen1">[18]</a>, how many loci are needed and how many base pairs from each locus? Until an analytical solution to this question is developed, simulated data can provide insights into whether, given a fixed amount of sequencing effort, it is preferable to increase the number of loci or the sequence length of individual loci. Theory suggests that the number of independently segregating loci is crucial to the accurate estimation of the population genetic parameter θ (4<em>N<sub>e</sub>
            </em>μ) <a href="#pone.0000160-Pluzhnikov1">[14]</a>, <a href="#pone.0000160-Felsenstein1">[17]</a>, which is usually interpreted as a scaled measure of the neutral mutation rate per site or as the proportion of polymorphic sites in a population <a href="#pone.0000160-Watterson1">[19]</a>.</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p>Our primary motivation was to investigate the accuracy and precision of estimates of θ, using a coalescent-based analytical method, across simulated datasets which vary in both the number of loci and the total number of base pairs sequenced per individual (e.g. 10 kb sequence/individual at one locus or distributed evenly among 10 loci). Whereas previous work in this area has focused primarily on making theoretical predictions of optimal sampling strategies <a href="#pone.0000160-Pluzhnikov1">[14]</a>, <a href="#pone.0000160-Felsenstein1">[17]</a>, our work uses extensive data simulations, over a broad range of both number of loci sampled and total sequence length sampled, to explore the validity of those predictions.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Materials and methods"></a><h3>Materials and methods</h3>
<h4>Simulated Data</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>For three different per-site values of θ (0.1, 0.01, 0.001), we simulated DNA sequence data using Treevolve version 1.3 <a href="#pone.0000160-Grassly1">[20]</a> according to scenarios that varied in both total amount of sequence per individual, as well as in the number of loci. Treevolve simulates DNA sequence evolution under a coalescent model. Although Treevolve can simulate sequences under a variety of population dynamic models, our data were simulated under a simple model with no selection, no intra-locus recombination, no population subdivision, and no migration. A population sample size of 10, which corresponds to phased nuclear haplotype data from five diploid individuals, was used in all simulations. We adjusted the Treevolve input parameters as follows: sequence length (varied according to scenario, see <a href="#pone-0000160-t001">Table 1</a>); mutation rate (1×10<sup>−6</sup> for θ = 0.1; 1×10<sup>−7</sup> for θ = 0.01, 0.001); number of loci sampled per individual (varied according to scenario, see <a href="#pone-0000160-t001">Table 1</a>); ploidy (haploid); generation time/variance in offspring number (1.0, corresponds to Wright-Fisher reproduction of non-overlapping generations and sampling with replacement); length of period (1×10<sup>12</sup> upper limit on coalescent time, our value ensures all simulated sequences coalesce); population size (25000 for θ = 0.1, 0.01; 2500 for θ = 0.001); subdivision (1, corresponds to no population subdivision); and recombination (0.0). Sequences were simulated under the Felsenstein '84 <a href="#pone.0000160-Felsenstein2">[21]</a> model of sequence evolution, with a transition/transversion ratio of 2.0 to model a moderate transition bias.</p>
<div class="figure" id="pone-0000160-t001"><div class="img"><a name="pone-0000160-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000160.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000160" data-uri="info:doi/10.1371/journal.pone.0000160.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000160.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000160.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000160.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000160.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000160.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000160.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000160.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000160.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000160.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>Simulation conditions and summary statistics for <em>è</em> calculations.</span></strong></p><span>doi:10.1371/journal.pone.0000160.t001</span></div><a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2"></a><p>By manipulating the user-defined mutation rate and population size, we were able to simulate sequences under known values of θ. With θ = 0.1 (μ = 1×10<sup>−6</sup>, <em>N<sub>e</sub>
               </em> = 25000), we first simulated 100 replicate datasets, each containing 10 single-locus DNA sequences 0.5 kb in length. Keeping θ = 0.1, we then simulated 100 more datasets that differed from the first datasets only in the length of each DNA sequence, which was increased to 1 kb. Next, 100 datasets were generated that contained DNA sequences from two unlinked loci, each 0.5 kb in length, sampled from 10 individuals. In these datasets the amount of DNA sequence per individual was equal to the previous datasets (1 kb), but was partitioned into two independent (i.e. no intra-genic recombination, but free inter-genic recombination) loci instead of being sampled from a single locus. In this way, we generated 100 replicates of datasets which varied in total DNA sequence length per individual from 0.5 kb to 100 kb, and the number of equal-length loci from which the sequences were sampled: 1 to 100 (15 total datasets, <a href="#pone-0000160-t001">Table 1</a>). We repeated the simulations with θ = 0.01 (μ = 1×10<sup>−7</sup>, <em>N<sub>e</sub>
               </em> = 25000; <a href="#pone-0000160-t001">Table 1</a>). The same 15 scenarios, plus three additional ones (25 kb per individual divided among five equal length loci, 50 kb among 10 loci, 100 kb among 20 loci), were simulated with θ = 0.001 (μ = 1×10<sup>−7</sup>, <em>N<sub>e</sub>
               </em> = 2500; <a href="#pone-0000160-t001">Table 1</a>). Output files were manipulated into the input format required by M<span class="small-caps">igrate</span> (see below) using a Perl script we wrote. We chose the values of θ to span the extremes of variation observed in natural populations <a href="#pone.0000160-Crawford1">[22]</a>–<a href="#pone.0000160-Strasburg1">[25]</a>. Simulated datasets and the Perl scripts are available from the corresponding author.</p>


<h4>Calculations and Analyses</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>A number of software packages and analytical methods are available for estimating θ, such as such as SITES <a href="#pone.0000160-Wakeley1">[13]</a>, Arlequin <a href="#pone.0000160-Excoffier1">[26]</a>, GeneTree <a href="#pone.0000160-Page1">[27]</a>, and B<span class="small-caps">east</span> 
               <a href="#pone.0000160-Drummond1">[28]</a>. Because methods that incorporate phylogenetic structure into the parameter estimation procedure have been demonstrated to have less bias <a href="#pone.0000160-Fu1">[1]</a>, <a href="#pone.0000160-Felsenstein3">[29]</a>, <a href="#pone.0000160-Fu2">[30]</a>, we calculated θ using the coalescent framework implemented in M<span class="small-caps">igrate</span> version 1.7.5 <a href="#pone.0000160-Beerli1">[31]</a>, but our results should be applicable to other methods of θ estimation. Using Markov chain Monte Carlo (MCMC) methods to approximate the likelihood distribution, M<span class="small-caps">igrate</span> calculates maximum likelihood estimates of population parameters under a coalescent framework. After discarding the first 10000 genealogies in each chain as “burn-in”, we sampled every 20 genealogies for both the 10 short chains (1000 total genealogies sampled), and the three long chains (10000 total genealogies sampled). We set the transition/transversion ratio at 2.0; other parameters were left at their default settings.</p>
<a id="article1.body1.sec2.sec2.p2" name="article1.body1.sec2.sec2.p2"></a><p>To test for bias in the parameter estimates we investigated whether any mean value of θ differed from the mean values for other scenarios within a given simulated value of θ. Statistical analyses were performed using SAS version 9.0 (SAS Institute Inc., Cary, NC 2002). We used the Tukey multiple comparison adjustment to test all pairwise combinations <a href="#pone.0000160-Sokal1">[32]</a>.</p>

</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Results"></a><h3>Results</h3>
<h4>Number of loci</h4>
<a id="article1.body1.sec3.sec1.p1" name="article1.body1.sec3.sec1.p1"></a><p>As predicted by theory <a href="#pone.0000160-Pluzhnikov1">[14]</a>, <a href="#pone.0000160-Felsenstein1">[17]</a> and demonstrated in previous analyses of simulated data <a href="#pone.0000160-Beerli1">[31]</a>, <a href="#pone.0000160-Kuhner2">[33]</a>, estimates of θ were greatly improved by increasing the number of loci sequenced per individual (<a href="#pone-0000160-g001">Figure 1</a>). The accuracy, measured as the square of the mean estimate divided by the variance of the estimate, increased proportionately with the number of sampled loci (<a href="#pone-0000160-g001">Figure 1A</a>) and concomitantly, the square of the coefficient of variation (standard deviation/mean) decreased considerably over the same sampling regime (<a href="#pone-0000160-g001">Figure 1B</a>). For a given value of θ, sampling additional loci always increased the accuracy and decreased the coefficient of variation (<a href="#pone-0000160-t001">Table 1</a>). Further, our calculations of accuracy were largely congruent with theoretical predictions of accuracy calculated using Felsenstein's <a href="#pone.0000160-Felsenstein1">[17]</a> modification of formulas developed by Fu and Li <a href="#pone.0000160-Fu3">[34]</a>. Discrepancies between our calculated accuracies and the predicted accuracy values arose when M<span class="small-caps">igrate</span> produced biased estimates of θ (see below)</p>
<div class="figure" id="pone-0000160-g001"><div class="img"><a name="pone-0000160-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000160.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000160" data-uri="info:doi/10.1371/journal.pone.0000160.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000160.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000160.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000160.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000160.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000160.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000160.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000160.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000160.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000160.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1. </strong></p><a id="article1.body1.sec3.sec1.fig1.caption1.p1" name="article1.body1.sec3.sec1.fig1.caption1.p1"></a><p>Influence of increasing the number of loci sampled per individual on the coalescent estimates of θ (0.1, 0.01, 0.001): (A) improvement in accuracy (mean<sup>2</sup>/variance), loci sampled are 1 kb in length; (B) improvement in squared coefficient of variation ((standard deviation/mean)<sup>2</sup>), loci sampled are 1 kb in length; (C) accuracy, loci sampled are 5 kb in length, θ = 0.001 (see text); (D) squared coefficient of variation, loci are 5 kb in length (see text).</p>
<span>doi:10.1371/journal.pone.0000160.g001</span></div><a id="article1.body1.sec3.sec1.p2" name="article1.body1.sec3.sec1.p2"></a><p>In terms of further decreasing the squared coefficient of variation, little additional improvement was gained by sampling more than 25 loci (<a href="#pone-0000160-g001">Figure 1B</a>). For example, when θ = 0.01, the total improvement in the coalescent estimate, measured by subtracting the squared coefficient of variation for the 100 loci scenario (0.00111) from the squared coefficient of variation for the 1 locus scenario (0.15965) was 0.15854. Roughly 81% of the improvement was accounted for by increasing the number of loci sampled from 1 to 5, and nearly 98% of the improvement could be explained by increasing the number of sampled loci from 1 to 25; adding the last 75 loci accounted for less than 3% of the total improvement in the squared coefficient of variation. As in accuracy (above), the decrease in the squared coefficients of variation was as predicted by theory <a href="#pone.0000160-Felsenstein1">[17]</a>. Similar results were obtained when θ = 0.1. Approximately 78% of the total improvement could be explained by increasing the number of sampled loci from 1 to 5 and nearly 97% was accounted for by increasing the number of loci sampled from 1 to 25.</p>
<a id="article1.body1.sec3.sec1.p3" name="article1.body1.sec3.sec1.p3"></a><p>Although increasing the number of sampled loci improved accuracy and decreased the squared coefficient of variation when θ = 0.001, the means of the estimates decreased (see below), precluding calculations of the improvement gained by sampling additional loci.</p>


<h4>Number of base pairs per locus</h4>
<a id="article1.body1.sec3.sec2.p1" name="article1.body1.sec3.sec2.p1"></a><p>In contrast to increasing the number of loci sampled, increasing the length of sequence at a particular locus had relatively little impact on improving the accuracy associated with estimating θ, except when the known value of θ was small (<a href="#pone-0000160-g002">Figure 2</a>). Increasing the length of sequence from 0.5 kb to 100 kb resulted in a total improvement in the estimate accuracy of 5.2824, when θ = 0.01, which is less than 0.6% of the total improvement in accuracy gained by increasing the number of 1 kb loci from one to 100. More dramatically, when the known value of θ was 0.001, point estimates calculated from 1 kb of sequence from a single locus were anywhere from 1.71 (min/0.001×100) to 291.3% (max/0.001×100) of the actual value (<a href="#pone-0000160-t001">Table 1</a>). These results demonstrate that parameter estimates based on single-locus population genetic data have large deviations regardless of sequence length or true value of the parameter of interest.</p>
<div class="figure" id="pone-0000160-g002"><div class="img"><a name="pone-0000160-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000160.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000160" data-uri="info:doi/10.1371/journal.pone.0000160.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000160.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000160.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000160.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000160.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000160.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000160.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000160.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000160.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000160.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2. </strong></p><a id="article1.body1.sec3.sec2.fig1.caption1.p1" name="article1.body1.sec3.sec2.fig1.caption1.p1"></a><p>Influence on increasing the sequence length of a single sampled locus on (A) accuracy (mean<sup>2</sup>/variance) of the coalescent estimates of θ (0.1, 0.01, 0.001); (B) squared coefficient of variation ((standard deviation/mean)<sup>2</sup>).</p>
<span>doi:10.1371/journal.pone.0000160.g002</span></div><a id="article1.body1.sec3.sec2.p2" name="article1.body1.sec3.sec2.p2"></a><p>On average, across 100 replicate datasets, M<span class="small-caps">igrate</span> performed well at recovering the known value of θ (<a href="#pone-0000160-t001">Table 1</a>). However, M<span class="small-caps">igrate</span> produced biased estimates of θ when the known parameter value was small (0.001). The mean values of θ under some scenarios were significantly less than the true value of 0.001 (one-way ANOVA, <em>P</em>&lt;0.05, <a href="#pone-0000160-t001">Table 1</a>). The downward bias resulted from the lack of information (i.e. polymorphic sites) available when the known value of θ was small (0.001 here), especially when the length of the sequence was less than 1 kb. With little information at each locus, the likelihood surfaces were very flat, leading to poor estimates of θ <a href="#pone.0000160-Beerli1">[31]</a>. We addressed this problem by increasing the per-locus sequence length to 5 kb (<a href="#pone-0000160-g001">Figures 1C and 1D</a>), such that we generated additional datasets, which had 25 kb of sequence divided among five loci of 5 kb each, 50 kb divided equally among 10 loci, and 100 kb divided equally among 20 loci. Increasing the per-locus sequence length eliminated the downward bias (<a href="#pone-0000160-t001">Table 1</a>), and also increased the accuracy of the parameter calculations when compared to the values obtained for shorter sequence lengths (contrast <a href="#pone-0000160-g001">Figures 1A and 1B</a> with <a href="#pone-0000160-g001">Figures 1C and 1D</a>). Since the simulations were not extended to include datasets containing 50 and 100 loci, each 5 kb in length for θ = 0.001, we did not investigate the improvement gained by adding loci.</p>

</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec4.p1" name="article1.body1.sec4.p1"></a><p>Our simulations demonstrated clearly the utility of multi-locus datasets in estimating population genetic parameters under a coalescent framework, validating a key component of theoretical investigations of optimal sampling strategies <a href="#pone.0000160-Pluzhnikov1">[14]</a>, <a href="#pone.0000160-Felsenstein1">[17]</a>. An important consideration is that there are two sources of error that contribute to the variance in coalescent-based parameter estimates, error associated with calculating the probability of the genealogy and error associated with calculating the probability of the coalescent. M<span class="small-caps">igrate</span> and other coalescent-based software packages attempt to marginalize the effects of the genealogical error.</p>
<a id="article1.body1.sec4.p2" name="article1.body1.sec4.p2"></a><p>The number of loci is crucial to reducing the coefficient of variation, even more so than increasing the length of the sequence at any one locus (<a href="#pone-0000160-g001">Figures 1</a>, <a href="#pone-0000160-g002">2</a>). In our data, the most reliable way to reduce the variation in the parameter estimates was to increase the number of loci sequenced per individual. Previous work has mentioned this issue <a href="#pone.0000160-Edwards1">[e.g. 11]</a>, but our results provide additional insights into how the precision of parameter estimation varies with the amount of genetic information available.</p>
<a id="article1.body1.sec4.p3" name="article1.body1.sec4.p3"></a><p>Other investigations into the optimal sampling strategy for coalescent-based estimates of population genetic parameters have used a fixed-cost approach <a href="#pone.0000160-Pluzhnikov1">[14]</a>, <a href="#pone.0000160-Felsenstein1">[17]</a>. In general, they sought to identify the ratio of individuals, number of loci, and per-locus sequence length that would maximize the accuracy of the estimate. Since they operated under a fixed-cost model, there are trade-offs; for example, an increase in the number of individuals sampled necessitated a decrease the in the number of loci sampled. One important conclusion of the pioneering theoretical work by Pluzhnikov and Donnelly <a href="#pone.0000160-Pluzhnikov1">[14]</a> was that to increase precision one should always choose to move to an independent region rather than extend the length of the current region. Felsenstein <a href="#pone.0000160-Felsenstein1">[17]</a> came to similar conclusions, and described the optimal sampling strategy under a cost-per-base model as one in which large numbers of loci, each a single base long, are employed. A more realistic cost-per-read model still advocates a many-locus, few individual design, a result supported by our simulations. However, our simulations focus attention on an important caveat to the many-locus sampling strategy – the loss of information at short loci (0.5 kb in our simulations) when θ is small (0.001 in our simulations).</p>
<a id="article1.body1.sec4.p4" name="article1.body1.sec4.p4"></a><p>Under small values of θ (0.001 here), which are commonly observed in empirical data <a href="#pone.0000160-Crawford1">[22]</a>, <a href="#pone.0000160-Hare2">[35]</a>–<a href="#pone.0000160-Primmer1">[37]</a>, the paucity of informative sites in short sequences may introduce a downward bias in the estimates (<a href="#pone-0000160-t001">Table 1</a>). Examining the results from the simulated datasets where the total sequence data from each individual was 10 kb reveals the magnitude of this problem. At large and moderate values of θ (0.1 and 0.01), the accuracy of the estimates was highest when the 10 kb of sequence was partitioned into 20 loci, each 0.5 kb in length (<a href="#pone-0000160-t001">Table 1</a>). In contrast, the most accurate sampling strategy when θ = 0.001 was to sample only two loci, each 5 kb in length (<a href="#pone-0000160-t001">Table 1</a>). Further, when θ was 0.001, the mean value of the estimates was 0.00039 when the data were partitioned into 20 equal-length loci. This low estimate represents a statistically significant reduction of ~60% from the known value (<a href="#pone-0000160-t001">Table 1</a>). Note that theoretical predictions encourage sampling more loci even at small values of θ; a 20 loci sampling strategy produced the highest predicted accuracy for each value of θ when each individual was sequenced at 10 kb. At our smallest simulated value of θ (0.001), the mean estimates of seven of the 18 datasets were significantly smaller than the known value, a potential problem with the “more loci, shorter sequence” sample design. The sampling scenarios that resulted in a biased estimate of θ were the same scenarios for which our calculated accuracy values differ greatly from the predicted values (<a href="#pone-0000160-t001">Table 1</a>). The predicted values are based on an unbiased coalescent estimator, which our data suggest is not the case when θ was small. The downward bias in the coalescent estimates likely accounts for the differences between our calculated accuracies and the predicted values. Increasing the sequence length at each particular locus may alleviate the problem. In our simulations, none of the datasets that contained loci 5 kb in length or longer showed the downward bias (<a href="#pone-0000160-t001">Table 1</a>). Further work is needed to investigate the minimum amount of sequence data per locus required to ameliorate the problem. The problem of a downward bias is well documented <a href="#pone.0000160-Felsenstein1">[17]</a>, <a href="#pone.0000160-Beerli1">[31]</a>, but it has not been emphasized in previous work of optimal sampling strategies. The results of our study underscore the need to consider both the per locus sequence length and number of sampled loci.</p>
<a id="article1.body1.sec4.p5" name="article1.body1.sec4.p5"></a><p>Accurate estimation of θ required data from at least 25 independently evolving loci. Beyond this, there was little added benefit in terms of decreasing the squared coefficient of variation of the coalescent estimates relative to the extra effort required to sample more loci. Interestingly, a recent paper concluded that to accurately recover phylogenetic relationships with maximum support, researchers should use at least 20 genes <a href="#pone.0000160-Rokas1">[38]</a>, suggesting that 20 loci will not only provide a robust estimate of population genetic parameters, but also of phylogeny. While sampling 25 loci will increase the statistical confidence in population genetic parameters, researchers interested in comparing parameter estimates between taxa should remain cautious in attributing biological significance to statistical differences <a href="#pone.0000160-Hedrick1">[39]</a>. These differences might be small enough to render any corresponding biological differences irrelevant.</p>
<a id="article1.body1.sec4.p6" name="article1.body1.sec4.p6"></a><p>Our analyses were based on a large, panmictic population at equilibrium, and without recombination or selection. Application of our results to empirical data must be considered in light of the restrictive assumptions under which the data were simulated. Multi-locus empirical datasets have many more sources of variation that would elevate the number of loci needed to estimate θ accurately. Another important source of variation not considered here were the theoretical limits of current coalescent methods to correctly infer population processes <a href="#pone.0000160-Wakeley2">[40]</a>.</p>
<a id="article1.body1.sec4.p7" name="article1.body1.sec4.p7"></a><p>The calculations based on the simulated datasets were focused on generating average, multi-locus estimates of θ. Evolutionary forces (e.g. different mutation rates, levels of selection) whose impacts are heterogeneous across the genome further complicate the interpretation of empirical estimates of θ measured from loci distributed across the genome <a href="#pone.0000160-Kliman1">[41]</a>, <a href="#pone.0000160-Patterson1">[42]</a>. For example, genetic diversity and recombination are positively correlated; genomic regions with high rates of recombination often show high nucleotide diversity <a href="#pone.0000160-Nachman1">[43]</a>–<a href="#pone.0000160-Lercher1">[45]</a> and vice-versa <a href="#pone.0000160-Begun1">[46]</a>, <a href="#pone.0000160-Berry1">[47]</a>, where selection typically reduces variation. Therefore, differences in genome-wide recombination rates <a href="#pone.0000160-Nachman3">[48]</a>, <a href="#pone.0000160-Nachman4">[49]</a> could inflate levels of variation at some loci, beyond what might be expected for neutrally evolving loci as were sampled in our simulations.</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>We thank P. Beerli and M. Kuhner for their assistance with M<span class="small-caps">igrate</span> and R<span class="small-caps">ecombine</span>. D. Pollock provided computational resources and helped write Perl scripts. This manuscript was greatly improved by comments from Z. Cheviron, B. Counterman, A. Crawford, S. Dixon-Schully, S. Edwards, J. Felsenstein, R. Harrison, M. Hellberg, J. Hey, R. Kliman, D. Ortiz-Barrientos, M. Noor, M. Taylor, C. Witt, and an anonymous reviewer. J. Felsenstein generously provided unpublished results.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: RB. Performed the experiments: RB MC. Analyzed the data: MC. Wrote the paper: MC.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0000160-Fu1" id="pone.0000160-Fu1"></a>Fu YX, Li WH (1999) Coalescing into the 21st century: An overview and prospects of coalescent theory. Theor Popul Biol  56: 1–10.  <ul class="find" data-citedArticleID="926348" data-doi="10.1006/tpbi.1999.1421"><li><a href="http://dx.doi.org/10.1006/tpbi.1999.1421" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Coalescing+into+the+21st+century%3A+An+overview+and+prospects+of+coalescent+theory." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Coalescing+into+the+21st+century%3A+An+overview+and+prospects+of+coalescent+theory.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0000160-Moore1" id="pone.0000160-Moore1"></a>Moore WS (1995) Inferring phylogenies from mtDNA variation - Mitochondrial-gene trees versus nuclear-gene trees. Evolution  49: 718–726.  <ul class="find" data-citedArticleID="926374" data-doi="10.2307/2410325"><li><a href="http://dx.doi.org/10.2307/2410325" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Inferring+phylogenies+from+mtDNA+variation+-+Mitochondrial-gene+trees+versus+nuclear-gene+trees." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Inferring+phylogenies+from+mtDNA+variation+-+Mitochondrial-gene+trees+versus+nuclear-gene+trees.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0000160-Moore2" id="pone.0000160-Moore2"></a>Moore WS (1997) Mitochondrial-gene trees versus nuclear-gene trees, a reply to Hoelzer. Evolution  51: 627–629.  <ul class="find" data-citedArticleID="926376" data-doi="10.2307/2411137"><li><a href="http://dx.doi.org/10.2307/2411137" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mitochondrial-gene+trees+versus+nuclear-gene+trees%2C+a+reply+to+Hoelzer." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mitochondrial-gene+trees+versus+nuclear-gene+trees%2C+a+reply+to+Hoelzer.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0000160-Congdon1" id="pone.0000160-Congdon1"></a>Congdon BC, Piatt JF, Martin K, Friesen VL (2000) Mechanisms of population differentiation in marbled murrelets: Historical versus contemporary processes. Evolution  54: 974–986.  <ul class="find" data-citedArticleID="926328" data-doi="10.1554/0014-3820(2000)054[0974:mopdim]2.3.co;2"><li><a href="http://dx.doi.org/10.1554/0014-3820(2000)054[0974:mopdim]2.3.co;2" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mechanisms+of+population+differentiation+in+marbled+murrelets%3A+Historical+versus+contemporary+processes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mechanisms+of+population+differentiation+in+marbled+murrelets%3A+Historical+versus+contemporary+processes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0000160-Ford1" id="pone.0000160-Ford1"></a>Ford MJ (1998) Testing models of migration and isolation among populations of chinook salmon (<em>Oncorhynchus tschawytscha</em>). Evolution  52: 539–557.  <ul class="find" data-citedArticleID="926346" data-doi="10.2307/2411089"><li><a href="http://dx.doi.org/10.2307/2411089" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Testing+models+of+migration+and+isolation+among+populations+of+chinook+salmon+%28Oncorhynchus+tschawytscha%29." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Testing+models+of+migration+and+isolation+among+populations+of+chinook+salmon+%28Oncorhynchus+tschawytscha%29.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0000160-Zhang1" id="pone.0000160-Zhang1"></a>Zhang DX, Hewitt GM (2003) Nuclear DNA analyses in genetic studies of populations: Practice, problems and prospects. Mol Ecol  12: 563–584.  <ul class="find" data-citedArticleID="926418" data-doi="10.1046/j.1365-294x.2003.01773.x"><li><a href="http://dx.doi.org/10.1046/j.1365-294x.2003.01773.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Nuclear+DNA+analyses+in+genetic+studies+of+populations%3A+Practice%2C+problems+and+prospects." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Nuclear+DNA+analyses+in+genetic+studies+of+populations%3A+Practice%2C+problems+and+prospects.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0000160-Hare1" id="pone.0000160-Hare1"></a>Hare MP (2001) Prospects for nuclear gene phylogeography. Trends Ecol Evol  16: 700–706.  <ul class="find" data-citedArticleID="926356" data-doi="10.1016/s0169-5347(01)02326-6"><li><a href="http://dx.doi.org/10.1016/s0169-5347(01)02326-6" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Prospects+for+nuclear+gene+phylogeography." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Prospects+for+nuclear+gene+phylogeography.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0000160-Emerson1" id="pone.0000160-Emerson1"></a>Emerson BC, Paradis E, Thebaud C (2001) Revealing the demographic histories of species using DNA sequences. Trends Ecol Evol  16: 707–716.  <ul class="find" data-citedArticleID="926336" data-doi="10.1016/s0169-5347(01)02305-9"><li><a href="http://dx.doi.org/10.1016/s0169-5347(01)02305-9" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Revealing+the+demographic+histories+of+species+using+DNA+sequences." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Revealing+the+demographic+histories+of+species+using+DNA+sequences.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0000160-Knowles1" id="pone.0000160-Knowles1"></a>Knowles LL, Maddison WP (2002) Statistical phylogeography. Mol Ecol  11: 2623–2635.  <ul class="find" data-citedArticleID="926364" data-doi="10.1046/j.1365-294x.2002.01637.x"><li><a href="http://dx.doi.org/10.1046/j.1365-294x.2002.01637.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Statistical+phylogeography." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Statistical+phylogeography.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0000160-Rosenberg1" id="pone.0000160-Rosenberg1"></a>Rosenberg NA, Nordborg M (2002) Genealogical trees, coalescent theory and the analysis of genetic polymorphisms. Nature Rev Genet  3: 380–390.  <ul class="find" data-citedArticleID="926400" data-doi="10.1038/nrg795"><li><a href="http://dx.doi.org/10.1038/nrg795" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genealogical+trees%2C+coalescent+theory+and+the+analysis+of+genetic+polymorphisms." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genealogical+trees%2C+coalescent+theory+and+the+analysis+of+genetic+polymorphisms.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0000160-Edwards1" id="pone.0000160-Edwards1"></a>Edwards SV, Beerli P (2000) Perspective: Gene divergence, population divergence, and the variance in coalescence time in phylogeographic studies. Evolution  54: 1839–1854.  <ul class="find" data-citedArticleID="926334" data-doi="10.1111/j.0014-3820.2000.tb01231.x"><li><a href="http://dx.doi.org/10.1111/j.0014-3820.2000.tb01231.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Perspective%3A+Gene+divergence%2C+population+divergence%2C+and+the+variance+in+coalescence+time+in+phylogeographic+studies." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Perspective%3A+Gene+divergence%2C+population+divergence%2C+and+the+variance+in+coalescence+time+in+phylogeographic+studies.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0000160-Kuhner1" id="pone.0000160-Kuhner1"></a>Kuhner MK, Beerli P, Yamato J, Felsenstein J (2000) Usefulness of single nucleotide polymorphism data for estimating population parameters. Genetics  156: 439–447.  <ul class="find" data-citedArticleID="926366"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Usefulness+of+single+nucleotide+polymorphism+data+for+estimating+population+parameters.&amp;auth=&amp;atitle=Usefulness+of+single+nucleotide+polymorphism+data+for+estimating+population+parameters." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Usefulness+of+single+nucleotide+polymorphism+data+for+estimating+population+parameters." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Usefulness+of+single+nucleotide+polymorphism+data+for+estimating+population+parameters.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0000160-Wakeley1" id="pone.0000160-Wakeley1"></a>Wakeley J, Hey J (1997) Estimating ancestral population parameters. Genetics  145: 847–855.  <ul class="find" data-citedArticleID="926410"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Estimating+ancestral+population+parameters.&amp;auth=&amp;atitle=Estimating+ancestral+population+parameters." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Estimating+ancestral+population+parameters." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Estimating+ancestral+population+parameters.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0000160-Pluzhnikov1" id="pone.0000160-Pluzhnikov1"></a>Pluzhnikov A, Donnelly P (1996) Optimal sequencing strategies for surveying molecular genetic diversity. Genetics  144: 1247–1262.  <ul class="find" data-citedArticleID="926392"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Optimal+sequencing+strategies+for+surveying+molecular+genetic+diversity.&amp;auth=&amp;atitle=Optimal+sequencing+strategies+for+surveying+molecular+genetic+diversity." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Optimal+sequencing+strategies+for+surveying+molecular+genetic+diversity." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Optimal+sequencing+strategies+for+surveying+molecular+genetic+diversity.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0000160-Takahata1" id="pone.0000160-Takahata1"></a>Takahata N, Satta Y, Klein J (1995) Divergence time and population-size in the lineage leading to modern humans. Theor Popul Biol  48: 198–221.  <ul class="find" data-citedArticleID="926408" data-doi="10.1006/tpbi.1995.1026"><li><a href="http://dx.doi.org/10.1006/tpbi.1995.1026" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Divergence+time+and+population-size+in+the+lineage+leading+to+modern+humans." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Divergence+time+and+population-size+in+the+lineage+leading+to+modern+humans.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0000160-Yang1" id="pone.0000160-Yang1"></a>Yang ZH (1997) On the estimation of ancestral population sizes of modern humans. Genet Res  69: 111–116.  <ul class="find" data-citedArticleID="926416" data-doi="10.1017/s001667239700270x"><li><a href="http://dx.doi.org/10.1017/s001667239700270x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=On+the+estimation+of+ancestral+population+sizes+of+modern+humans." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22On+the+estimation+of+ancestral+population+sizes+of+modern+humans.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0000160-Felsenstein1" id="pone.0000160-Felsenstein1"></a>Felsenstein J (2006) Accuracy of coalescent likelihood estimates: Do we need more sites, more sequences, or more loci? Mol Biol Evol  23: 691–700.  <ul class="find" data-citedArticleID="926340" data-doi="10.1093/molbev/msj079"><li><a href="http://dx.doi.org/10.1093/molbev/msj079" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Accuracy+of+coalescent+likelihood+estimates%3A+Do+we+need+more+sites%2C+more+sequences%2C+or+more+loci%3F" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Accuracy+of+coalescent+likelihood+estimates%3A+Do+we+need+more+sites%2C+more+sequences%2C+or+more+loci%3F%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0000160-Nielsen1" id="pone.0000160-Nielsen1"></a>Nielsen R (2000) Estimation of population parameters and recombination rates from single nucleotide polymorphisms. Genetics  154: 931–942.  <ul class="find" data-citedArticleID="926386"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Estimation+of+population+parameters+and+recombination+rates+from+single+nucleotide+polymorphisms.&amp;auth=&amp;atitle=Estimation+of+population+parameters+and+recombination+rates+from+single+nucleotide+polymorphisms." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Estimation+of+population+parameters+and+recombination+rates+from+single+nucleotide+polymorphisms." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Estimation+of+population+parameters+and+recombination+rates+from+single+nucleotide+polymorphisms.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0000160-Watterson1" id="pone.0000160-Watterson1"></a>Watterson GA (1975) Number of segregating sites in genetic models without recombination. Theor Popul Biol  7: 256–276.  <ul class="find" data-citedArticleID="926414" data-doi="10.1016/0040-5809(75)90020-9"><li><a href="http://dx.doi.org/10.1016/0040-5809(75)90020-9" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Number+of+segregating+sites+in+genetic+models+without+recombination." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Number+of+segregating+sites+in+genetic+models+without+recombination.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0000160-Grassly1" id="pone.0000160-Grassly1"></a>Grassly NC, Harvey PH, Holmes EC (1999) Population dynamics of HIV-1 inferred from gene sequences. Genetics  151: 427–438.  <ul class="find" data-citedArticleID="926354"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Population+dynamics+of+HIV-1+inferred+from+gene+sequences.&amp;auth=&amp;atitle=Population+dynamics+of+HIV-1+inferred+from+gene+sequences." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Population+dynamics+of+HIV-1+inferred+from+gene+sequences." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Population+dynamics+of+HIV-1+inferred+from+gene+sequences.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0000160-Felsenstein2" id="pone.0000160-Felsenstein2"></a>Felsenstein J, Churchill GA (1996) A hidden Markov Model approach to variation among sites in rate of evolution. Mol Biol Evol  13: 93–104.  <ul class="find" data-citedArticleID="926342" data-doi="10.1093/oxfordjournals.molbev.a025575"><li><a href="http://dx.doi.org/10.1093/oxfordjournals.molbev.a025575" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+hidden+Markov+Model+approach+to+variation+among+sites+in+rate+of+evolution." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+hidden+Markov+Model+approach+to+variation+among+sites+in+rate+of+evolution.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0000160-Crawford1" id="pone.0000160-Crawford1"></a>Crawford AJ (2003) Huge populations and old species of Costa Rican and Panamanian dirt frogs inferred from mitochondrial and nuclear gene sequences. Mol Ecol  12: 2525–2540.  <ul class="find" data-citedArticleID="926330" data-doi="10.1046/j.1365-294x.2003.01910.x"><li><a href="http://dx.doi.org/10.1046/j.1365-294x.2003.01910.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Huge+populations+and+old+species+of+Costa+Rican+and+Panamanian+dirt+frogs+inferred+from+mitochondrial+and+nuclear+gene+sequences." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Huge+populations+and+old+species+of+Costa+Rican+and+Panamanian+dirt+frogs+inferred+from+mitochondrial+and+nuclear+gene+sequences.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0000160-Llopert1" id="pone.0000160-Llopert1"></a>Llopert A, Lachaise D, Coyne JA (2005) Multilocus analysis of introgression between two sympatric sister species of Drosophila: <em>Drosophila yakuba</em> and <em>D. santomea</em>. Genetics  171: 197–210.  <ul class="find" data-citedArticleID="926372" data-doi="10.1534/genetics.104.033597"><li><a href="http://dx.doi.org/10.1534/genetics.104.033597" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Multilocus+analysis+of+introgression+between+two+sympatric+sister+species+of+Drosophila%3A+Drosophila+yakuba+and+D.+santomea." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Multilocus+analysis+of+introgression+between+two+sympatric+sister+species+of+Drosophila%3A+Drosophila+yakuba+and+D.+santomea.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0000160-Saillant1" id="pone.0000160-Saillant1"></a>Saillant E, Patton JC, Ross KE, Gold JR (2005) Conservation genetics and demographic history of the endangered Cape Fear shiner (<em>Notropis mekistocholas</em>). Mol Ecol  13: 2947–2958.  <ul class="find" data-citedArticleID="926402" data-doi="10.1111/j.1365-294x.2004.02303.x"><li><a href="http://dx.doi.org/10.1111/j.1365-294x.2004.02303.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Conservation+genetics+and+demographic+history+of+the+endangered+Cape+Fear+shiner+%28Notropis+mekistocholas%29." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Conservation+genetics+and+demographic+history+of+the+endangered+Cape+Fear+shiner+%28Notropis+mekistocholas%29.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0000160-Strasburg1" id="pone.0000160-Strasburg1"></a>Strasburg JL, Kearney M (2005) Phylogeography of sexual <em>Heteronotia binoei</em> (Gekkonidae) in the Australian arid zone: Climatic cycling and repetitive hybridization. Mol Ecol  14: 2755–2772.  <ul class="find" data-citedArticleID="926406" data-doi="10.1111/j.1365-294x.2005.02627.x"><li><a href="http://dx.doi.org/10.1111/j.1365-294x.2005.02627.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Phylogeography+of+sexual+Heteronotia+binoei+%28Gekkonidae%29+in+the+Australian+arid+zone%3A+Climatic+cycling+and+repetitive+hybridization." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Phylogeography+of+sexual+Heteronotia+binoei+%28Gekkonidae%29+in+the+Australian+arid+zone%3A+Climatic+cycling+and+repetitive+hybridization.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0000160-Excoffier1" id="pone.0000160-Excoffier1"></a>Excoffier L, Laval G, Schneider S (2005) Arlequin ver 3.0: An intergrated software package for population genetics data analysis. Evol Bioinform Online  1: 47–50.  <ul class="find" data-citedArticleID="926338"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Arlequin+ver+3.0%3A+An+intergrated+software+package+for+population+genetics+data+analysis.&amp;auth=&amp;atitle=Arlequin+ver+3.0%3A+An+intergrated+software+package+for+population+genetics+data+analysis." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Arlequin+ver+3.0%3A+An+intergrated+software+package+for+population+genetics+data+analysis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Arlequin+ver+3.0%3A+An+intergrated+software+package+for+population+genetics+data+analysis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0000160-Page1" id="pone.0000160-Page1"></a>Page RDM (1998) GeneTree: Comparing gene and species phylogenies using reconciled trees. Bioinformatics  14: 819–820.  <ul class="find" data-citedArticleID="926388" data-doi="10.1093/bioinformatics/14.9.819"><li><a href="http://dx.doi.org/10.1093/bioinformatics/14.9.819" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=GeneTree%3A+Comparing+gene+and+species+phylogenies+using+reconciled+trees." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22GeneTree%3A+Comparing+gene+and+species+phylogenies+using+reconciled+trees.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0000160-Drummond1" id="pone.0000160-Drummond1"></a>Drummond AJ, Rambaut A (2003) BEAST v1.0.   Available at <a href="http://evolve.zoo.ox.ac.uk/beast/">http://evolve.zoo.ox.ac.uk/beast/</a>
               .  <ul class="find-nolinks"></ul></li><li><span class="label">29.
              </span><a name="pone.0000160-Felsenstein3" id="pone.0000160-Felsenstein3"></a>Felsenstein J (1992) Estimating effective population-size from samples of sequences - Inefficiency of pairwise and segregating sites as compared to phylogenetic estimates. Genet Res  59: 139–147.  <ul class="find" data-citedArticleID="926344" data-doi="10.1017/s0016672300030354"><li><a href="http://dx.doi.org/10.1017/s0016672300030354" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Estimating+effective+population-size+from+samples+of+sequences+-+Inefficiency+of+pairwise+and+segregating+sites+as+compared+to+phylogenetic+estimates." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Estimating+effective+population-size+from+samples+of+sequences+-+Inefficiency+of+pairwise+and+segregating+sites+as+compared+to+phylogenetic+estimates.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0000160-Fu2" id="pone.0000160-Fu2"></a>Fu YX, Li WH (1993) Maximum-likelihood-estimation of population parameters. Genetics  134: 1261–1270.  <ul class="find" data-citedArticleID="926350"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Maximum-likelihood-estimation+of+population+parameters.&amp;auth=&amp;atitle=Maximum-likelihood-estimation+of+population+parameters." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Maximum-likelihood-estimation+of+population+parameters." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Maximum-likelihood-estimation+of+population+parameters.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0000160-Beerli1" id="pone.0000160-Beerli1"></a>Beerli P, Felsenstein J (1999) Maximum-likelihood estimation of migration rates and effective population numbers in two populations using a coalescent approach. Genetics  152: 763–773.  <ul class="find" data-citedArticleID="926322"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Maximum-likelihood+estimation+of+migration+rates+and+effective+population+numbers+in+two+populations+using+a+coalescent+approach.&amp;auth=&amp;atitle=Maximum-likelihood+estimation+of+migration+rates+and+effective+population+numbers+in+two+populations+using+a+coalescent+approach." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Maximum-likelihood+estimation+of+migration+rates+and+effective+population+numbers+in+two+populations+using+a+coalescent+approach." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Maximum-likelihood+estimation+of+migration+rates+and+effective+population+numbers+in+two+populations+using+a+coalescent+approach.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0000160-Sokal1" id="pone.0000160-Sokal1"></a>Sokal RR, Rohlf FJ (1995) Biometry. New York: W. H. Freeman and Company.   <ul class="find-nolinks"></ul></li><li><span class="label">33.
              </span><a name="pone.0000160-Kuhner2" id="pone.0000160-Kuhner2"></a>Kuhner MK, Yamato J, Felsenstein J (1995) Estimating effective population size and mutation rate from sequence data using Metropolis-Hastings sampling. Genetics  140: 1421–1430.  <ul class="find" data-citedArticleID="926368"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Estimating+effective+population+size+and+mutation+rate+from+sequence+data+using+Metropolis-Hastings+sampling.&amp;auth=&amp;atitle=Estimating+effective+population+size+and+mutation+rate+from+sequence+data+using+Metropolis-Hastings+sampling." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Estimating+effective+population+size+and+mutation+rate+from+sequence+data+using+Metropolis-Hastings+sampling." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Estimating+effective+population+size+and+mutation+rate+from+sequence+data+using+Metropolis-Hastings+sampling.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0000160-Fu3" id="pone.0000160-Fu3"></a>Fu YX, Li WH (1993) Statistical tests of neutrality of mutations. Genetics  133: 693–709.  <ul class="find" data-citedArticleID="926352"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Statistical+tests+of+neutrality+of+mutations.&amp;auth=&amp;atitle=Statistical+tests+of+neutrality+of+mutations." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Statistical+tests+of+neutrality+of+mutations." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Statistical+tests+of+neutrality+of+mutations.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0000160-Hare2" id="pone.0000160-Hare2"></a>Hare MP, Cipriano F, Palumbi SR (2002) Genetic evidence on the demography of speciation in allopatric dolphin species. Evolution  56: 804–816.  <ul class="find" data-citedArticleID="926358" data-doi="10.1554/0014-3820(2002)056[0804:geotdo]2.0.co;2"><li><a href="http://dx.doi.org/10.1554/0014-3820(2002)056[0804:geotdo]2.0.co;2" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genetic+evidence+on+the+demography+of+speciation+in+allopatric+dolphin+species." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genetic+evidence+on+the+demography+of+speciation+in+allopatric+dolphin+species.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">36.
              </span><a name="pone.0000160-Pogson1" id="pone.0000160-Pogson1"></a>Pogson GH, Fevolden SE (2003) Natural selection and the genetic differentiation of coastal and Arctic populations of the Atlantic cod in northern Norway: A test involving nucleotide sequence variation at the pantophysin (PanI) locus. Mol Ecol  12: 63–74.  <ul class="find" data-citedArticleID="926394" data-doi="10.1046/j.1365-294x.2003.01713.x"><li><a href="http://dx.doi.org/10.1046/j.1365-294x.2003.01713.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Natural+selection+and+the+genetic+differentiation+of+coastal+and+Arctic+populations+of+the+Atlantic+cod+in+northern+Norway%3A+A+test+involving+nucleotide+sequence+variation+at+the+pantophysin+%28PanI%29+locus." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Natural+selection+and+the+genetic+differentiation+of+coastal+and+Arctic+populations+of+the+Atlantic+cod+in+northern+Norway%3A+A+test+involving+nucleotide+sequence+variation+at+the+pantophysin+%28PanI%29+locus.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">37.
              </span><a name="pone.0000160-Primmer1" id="pone.0000160-Primmer1"></a>Primmer CR, Borge T, Lindell J, Saetre GP (2002) Single-nucleotide polymorphism characterization in species with limited available sequence information: High nucleotide diversity revealed in the avian genome. Mol Ecol  11: 603–612.  <ul class="find" data-citedArticleID="926396" data-doi="10.1046/j.0962-1083.2001.01452.x"><li><a href="http://dx.doi.org/10.1046/j.0962-1083.2001.01452.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Single-nucleotide+polymorphism+characterization+in+species+with+limited+available+sequence+information%3A+High+nucleotide+diversity+revealed+in+the+avian+genome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Single-nucleotide+polymorphism+characterization+in+species+with+limited+available+sequence+information%3A+High+nucleotide+diversity+revealed+in+the+avian+genome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">38.
              </span><a name="pone.0000160-Rokas1" id="pone.0000160-Rokas1"></a>Rokas A, Williams BL, King N, Carroll SB (2003) Genome-scale approaches to resolving incongruence in molecular phylogenies. Nature  425: 798–804.  <ul class="find" data-citedArticleID="926398" data-doi="10.1038/nature02053"><li><a href="http://dx.doi.org/10.1038/nature02053" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genome-scale+approaches+to+resolving+incongruence+in+molecular+phylogenies." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genome-scale+approaches+to+resolving+incongruence+in+molecular+phylogenies.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">39.
              </span><a name="pone.0000160-Hedrick1" id="pone.0000160-Hedrick1"></a>Hedrick PW (1999) Perspective: Highly variable loci and their interpretation in evolution and conservation. Evolution  53: 313–318.  <ul class="find" data-citedArticleID="926360" data-doi="10.2307/2640768"><li><a href="http://dx.doi.org/10.2307/2640768" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Perspective%3A+Highly+variable+loci+and+their+interpretation+in+evolution+and+conservation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Perspective%3A+Highly+variable+loci+and+their+interpretation+in+evolution+and+conservation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">40.
              </span><a name="pone.0000160-Wakeley2" id="pone.0000160-Wakeley2"></a>Wakeley J (2004) Recent trends in population genetics: More data! More math! Simple models? J Hered  95: 397–405.  <ul class="find" data-citedArticleID="926412" data-doi="10.1093/jhered/esh062"><li><a href="http://dx.doi.org/10.1093/jhered/esh062" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Recent+trends+in+population+genetics%3A+More+data%21+More+math%21+Simple+models%3F" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Recent+trends+in+population+genetics%3A+More+data%21+More+math%21+Simple+models%3F%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">41.
              </span><a name="pone.0000160-Kliman1" id="pone.0000160-Kliman1"></a>Kliman RM, Andolfatto P, Coyne JA, Depaulis F, Kreitman M, et al.  (2000) The population genetics of the origin and divergence of the <em>Drosophila simulans</em> complex species. Genetics  156: 1913–1931.  <ul class="find" data-citedArticleID="926362"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=The+population+genetics+of+the+origin+and+divergence+of+the+Drosophila+simulans+complex+species.&amp;auth=&amp;atitle=The+population+genetics+of+the+origin+and+divergence+of+the+Drosophila+simulans+complex+species." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+population+genetics+of+the+origin+and+divergence+of+the+Drosophila+simulans+complex+species." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+population+genetics+of+the+origin+and+divergence+of+the+Drosophila+simulans+complex+species.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">42.
              </span><a name="pone.0000160-Patterson1" id="pone.0000160-Patterson1"></a>Patterson N, Richter DJ, Gnerre S, Lander ES, Reich D (2006) Genetic evidence for complex speciation of humans and chimpanzees. Nature  441: 1103–1108.  <ul class="find" data-citedArticleID="926390" data-doi="10.1038/nature04789"><li><a href="http://dx.doi.org/10.1038/nature04789" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genetic+evidence+for+complex+speciation+of+humans+and+chimpanzees." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genetic+evidence+for+complex+speciation+of+humans+and+chimpanzees.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">43.
              </span><a name="pone.0000160-Nachman1" id="pone.0000160-Nachman1"></a>Nachman MW (1997) Patterns of DNA variability at X-linked loci in <em>Mus domesticus</em>. Genetics  147: 1303–1316.  <ul class="find" data-citedArticleID="926378"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Patterns+of+DNA+variability+at+X-linked+loci+in+Mus+domesticus.&amp;auth=&amp;atitle=Patterns+of+DNA+variability+at+X-linked+loci+in+Mus+domesticus." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Patterns+of+DNA+variability+at+X-linked+loci+in+Mus+domesticus." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Patterns+of+DNA+variability+at+X-linked+loci+in+Mus+domesticus.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">44.
              </span><a name="pone.0000160-Nachman2" id="pone.0000160-Nachman2"></a>Nachman MW, Bauer VL, Crowell SL, Aquadro CF (1998) DNA variability and recombination rates at X-linked loci in humans. Genetics  150: 1133–1141.  <ul class="find" data-citedArticleID="926380"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=DNA+variability+and+recombination+rates+at+X-linked+loci+in+humans.&amp;auth=&amp;atitle=DNA+variability+and+recombination+rates+at+X-linked+loci+in+humans." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=DNA+variability+and+recombination+rates+at+X-linked+loci+in+humans." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22DNA+variability+and+recombination+rates+at+X-linked+loci+in+humans.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">45.
              </span><a name="pone.0000160-Lercher1" id="pone.0000160-Lercher1"></a>Lercher MJ, Hurst LD (2002) Human SNP variability and mutation rate are higher in regions of high recombination. Trends Genet  18: 337–340.  <ul class="find" data-citedArticleID="926370" data-doi="10.1016/s0168-9525(02)02669-0"><li><a href="http://dx.doi.org/10.1016/s0168-9525(02)02669-0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Human+SNP+variability+and+mutation+rate+are+higher+in+regions+of+high+recombination." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Human+SNP+variability+and+mutation+rate+are+higher+in+regions+of+high+recombination.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">46.
              </span><a name="pone.0000160-Begun1" id="pone.0000160-Begun1"></a>Begun DJ, Aquadro CF (1991) Molecular population-genetics of the distal portion of the X-chromosome in Drosophila - Evidence for genetic hitchhiking of the yellow-achaete region. Genetics  129: 1147–1158.  <ul class="find" data-citedArticleID="926324"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Molecular+population-genetics+of+the+distal+portion+of+the+X-chromosome+in+Drosophila+-+Evidence+for+genetic+hitchhiking+of+the+yellow-achaete+region.&amp;auth=&amp;atitle=Molecular+population-genetics+of+the+distal+portion+of+the+X-chromosome+in+Drosophila+-+Evidence+for+genetic+hitchhiking+of+the+yellow-achaete+region." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Molecular+population-genetics+of+the+distal+portion+of+the+X-chromosome+in+Drosophila+-+Evidence+for+genetic+hitchhiking+of+the+yellow-achaete+region." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Molecular+population-genetics+of+the+distal+portion+of+the+X-chromosome+in+Drosophila+-+Evidence+for+genetic+hitchhiking+of+the+yellow-achaete+region.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">47.
              </span><a name="pone.0000160-Berry1" id="pone.0000160-Berry1"></a>Berry AJ, Ajioka JW, Kreitman M (1991) Lack of polymorphism on the Drosophila 4th chromosome resulting from selection. Genetics  129: 1111–1117.  <ul class="find" data-citedArticleID="926326"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Lack+of+polymorphism+on+the+Drosophila+4th+chromosome+resulting+from+selection.&amp;auth=&amp;atitle=Lack+of+polymorphism+on+the+Drosophila+4th+chromosome+resulting+from+selection." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Lack+of+polymorphism+on+the+Drosophila+4th+chromosome+resulting+from+selection." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Lack+of+polymorphism+on+the+Drosophila+4th+chromosome+resulting+from+selection.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">48.
              </span><a name="pone.0000160-Nachman3" id="pone.0000160-Nachman3"></a>Nachman MW (2001) Single nucleotide polymorphisms and recombination rate in humans. Trends Genet  17: 481–485.  <ul class="find" data-citedArticleID="926382" data-doi="10.1016/s0168-9525(01)02409-x"><li><a href="http://dx.doi.org/10.1016/s0168-9525(01)02409-x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Single+nucleotide+polymorphisms+and+recombination+rate+in+humans." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Single+nucleotide+polymorphisms+and+recombination+rate+in+humans.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">49.
              </span><a name="pone.0000160-Nachman4" id="pone.0000160-Nachman4"></a>Nachman MW (2002) Variation in recombination rate across the genome: Evidence and implications. Curr Opin Genetics Devel  12: 657–663.  <ul class="find" data-citedArticleID="926384" data-doi="10.1016/s0959-437x(02)00358-1"><li><a href="http://dx.doi.org/10.1016/s0959-437x(02)00358-1" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Variation+in+recombination+rate+across+the+genome%3A+Evidence+and+implications." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Variation+in+recombination+rate+across+the+genome%3A+Evidence+and+implications.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000160.XML" value="125107"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000160.PDF" value="157096"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000160.t001.PNG_L" value="455447"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000160.t001.PNG_M" value="45960"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000160.t001.PNG_S" value="4025"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000160.t001.TIF" value="21764330"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000160.t001.PNG_I" value="30048"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000160.g001.PNG_L" value="790912"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000160.g001.PNG_M" value="27137"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000160.g001.PNG_S" value="1925"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000160.g001.TIF" value="1136588"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000160.g001.PNG_I" value="11561"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000160.g002.PNG_L" value="34275"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000160.g002.PNG_M" value="10373"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000160.g002.PNG_S" value="1585"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000160.g002.TIF" value="55342"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000160.g002.PNG_I" value="6418"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000160&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000160" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000160&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0000160&volume=&issue=&title=Gene Sampling Strategies for Multi-Locus Population Estimates of Genetic Diversity (θ)&author_name=Matthew%20D.%20Carling%2C%20Robb%20T.%20Brumfield&start_page=1&end_page=6" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000160" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000160" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0000160" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000160&amp;t=Gene%20Sampling%20Strategies%20for%20Multi-Locus%20Population%20Estimates%20of%20Genetic%20Diversity%20(%CE%B8)" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000160&title=Gene%20Sampling%20Strategies%20for%20Multi-Locus%20Population%20Estimates%20of%20Genetic%20Diversity%20(%CE%B8)&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000160&amp;title=Gene%20Sampling%20Strategies%20for%20Multi-Locus%20Population%20Estimates%20of%20Genetic%20Diversity%20(%CE%B8)" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000160" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0000160&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Gene Sampling Strategies for Multi-Locus Population Estimates of Genetic Diversity (θ)';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0000160';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Gene%20Sampling%20Strategies%20for%20Multi-Locus%20Population%20Estimates%20of%20Genetic%20Diversity%20(%CE%B8) http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000160" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0000160" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">
















          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22DNA+recombination%22" title="Search for articles in the subject area:'DNA recombination'"><div class="flagText">DNA recombination</div></a>
              <div data-categoryid="33483" data-articleid="23158"
                   data-categoryname="DNA recombination"
                   class="flagImage" title="Flag 'DNA recombination' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22DNA+sequences%22" title="Search for articles in the subject area:'DNA sequences'"><div class="flagText">DNA sequences</div></a>
              <div data-categoryid="32003" data-articleid="23158"
                   data-categoryname="DNA sequences"
                   class="flagImage" title="Flag 'DNA sequences' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Genetic+loci%22" title="Search for articles in the subject area:'Genetic loci'"><div class="flagText">Genetic loci</div></a>
              <div data-categoryid="17713" data-articleid="23158"
                   data-categoryname="Genetic loci"
                   class="flagImage" title="Flag 'Genetic loci' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Genome+evolution%22" title="Search for articles in the subject area:'Genome evolution'"><div class="flagText">Genome evolution</div></a>
              <div data-categoryid="17207" data-articleid="23158"
                   data-categoryname="Genome evolution"
                   class="flagImage" title="Flag 'Genome evolution' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Mutation%22" title="Search for articles in the subject area:'Mutation'"><div class="flagText">Mutation</div></a>
              <div data-categoryid="31809" data-articleid="23158"
                   data-categoryname="Mutation"
                   class="flagImage" title="Flag 'Mutation' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Phylogenetic+analysis%22" title="Search for articles in the subject area:'Phylogenetic analysis'"><div class="flagText">Phylogenetic analysis</div></a>
              <div data-categoryid="17851" data-articleid="23158"
                   data-categoryname="Phylogenetic analysis"
                   class="flagImage" title="Flag 'Phylogenetic analysis' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Population+genetics%22" title="Search for articles in the subject area:'Population genetics'"><div class="flagText">Population genetics</div></a>
              <div data-categoryid="32359" data-articleid="23158"
                   data-categoryname="Population genetics"
                   class="flagImage" title="Flag 'Population genetics' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Sequence+analysis%22" title="Search for articles in the subject area:'Sequence analysis'"><div class="flagText">Sequence analysis</div></a>
              <div data-categoryid="47475" data-articleid="23158"
                   data-categoryname="Sequence analysis"
                   class="flagImage" title="Flag 'Sequence analysis' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=1482'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=2147'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=7042&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>


</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
