

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: SiDCoN: A Tool to Aid Scoring of DNA Copy Number Changes in SNP Chip Data</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0001093"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0001093" />

    <meta name="citation_title" content="SiDCoN: A Tool to Aid Scoring of DNA Copy Number Changes in SNP Chip Data"/>
    <meta itemprop="name" content="SiDCoN: A Tool to Aid Scoring of DNA Copy Number Changes in SNP Chip Data"/>

      <meta name="citation_author" content="Derek J. Nancarrow"/>
            <meta name="citation_author_institution" content="Oncogenomics, Queensland Institute of Medical Research, Herston, Queensland, Australia"/>
      <meta name="citation_author" content="Herlina Y. Handoko"/>
            <meta name="citation_author_institution" content="Oncogenomics, Queensland Institute of Medical Research, Herston, Queensland, Australia"/>
      <meta name="citation_author" content="Mitchell S. Stark"/>
            <meta name="citation_author_institution" content="Oncogenomics, Queensland Institute of Medical Research, Herston, Queensland, Australia"/>
      <meta name="citation_author" content="David C. Whiteman"/>
            <meta name="citation_author_institution" content="Cancer and Population Studies, Queensland Institute of Medical Research, Herston, Queensland, Australia"/>
      <meta name="citation_author" content="Nicholas K. Hayward"/>
            <meta name="citation_author_institution" content="Oncogenomics, Queensland Institute of Medical Research, Herston, Queensland, Australia"/>

    <meta name="citation_date" content="2007/10/31"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0001093.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e1093"/>
    <meta name="citation_issue" content="10"/>
    <meta name="citation_volume" content="2"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Frequent occurrence of uniparental disomy in colorectal cancer.; citation_author=CL Andersen; citation_author=C Wiuf; citation_author=M Kruhoffer; citation_author=M Korsgaard; citation_author=S Laurberg; citation_journal_title=Carcinogenesis; citation_volume=28; citation_number=1; citation_pages=38-48; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=QuantiSNP: an Objective Bayes Hidden-Markov Model to detect and accurately map copy number variation using SNP genotyping data.; citation_author=S Colella; citation_author=C Yau; citation_author=JM Taylor; citation_author=G Mirza; citation_author=H Butler; citation_journal_title=Nucleic Acids Res; citation_volume=35; citation_number=2; citation_pages=2013-2025; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Application of Genome-Wide Single Nucleotide Polymorphism Typing: Simple Association and Beyond.; citation_author=JR Gibbs; citation_author=A Singleton; citation_journal_title=PLoS Genetics; citation_volume=2; citation_number=3; citation_pages=e150; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=High-resolution genomic profiling of chromosomal aberrations using Infinium whole-genome genotyping.; citation_author=DA Peiffer; citation_author=JM Le; citation_author=FJ Steemers; citation_author=W Chang; citation_author=T Jenniges; citation_journal_title=Genome Res; citation_volume=16; citation_number=4; citation_pages=1136-1148; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=CARAT: a novel method for allelic detection of DNA copy number changes using high density oligonucleotide arrays.; citation_author=J Huang; citation_author=W Wei; citation_author=J Chen; citation_author=J Zhang; citation_author=G Liu; citation_journal_title=BMC Bioinformatics; citation_volume=7; citation_number=5; citation_pages=83; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Genome-wide loss of heterozygosity and copy number analysis in melanoma using high-density single-nucleotide polymorphism arrays.; citation_author=M Stark; citation_author=N Hayward; citation_journal_title=Cancer Res; citation_volume=67; citation_number=6; citation_pages=2632-2642; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=A robust algorithm for copy number detection using high-density oligonucleotide single nucleotide polymorphism genotyping arrays.; citation_author=Y Nannya; citation_author=M Sanada; citation_author=K Nakazaki; citation_author=N Hosoya; citation_author=L Wang; citation_journal_title=Cancer Res; citation_volume=65; citation_number=7; citation_pages=6071-6079; citation_date=2005; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001093" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="SiDCoN: A Tool to Aid Scoring of DNA Copy Number Changes in SNP Chip Data"/>
    <meta name="twitter:description" content="The recent application of genome-wide, single nucleotide polymorphism (SNP) microarrays to investigate DNA copy number aberrations in cancer has provided unparalleled sensitivity for identifying genomic changes. In some instances the complexity of these changes makes them difficult to interpret, particularly when tumour samples are contaminated with normal (stromal) tissue. Current automated scoring algorithms require considerable manual data checking and correction, especially when assessing uncultured tumour specimens. To address these limitations we have developed a visual tool to aid in the analysis of DNA copy number data. Simulated DNA Copy Number (SiDCoN) is a spreadsheet-based application designed to simulate the appearance of B-allele and logR plots for all known types of tumour DNA copy number changes, in the presence or absence of stromal contamination. The system allows the user to determine the level of stromal contamination, as well as specify up to 3 different DNA copy number aberrations for up to 5000 data points (representing individual SNPs). This allows users great flexibility to assess simple or complex DNA copy number combinations. We demonstrate how this utility can be used to estimate the level of stromal contamination within tumour samples and its application in deciphering the complex heterogeneous copy number changes we have observed in a series of tumours. We believe this tool will prove useful to others working in the area, both as a training tool, and to aid in the interpretation of complex copy number changes."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0001093.g005"/>

  <meta property="og:title" content="SiDCoN: A Tool to Aid Scoring of DNA Copy Number Changes in SNP Chip Data" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001093" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=9896'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=4365'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=7098&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0001093">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001093" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001093&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    SiDCoN: A Tool to Aid Scoring of DNA Copy Number Changes in SNP Chip Data
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Derek J. Nancarrow
              <span class="corresponding">mail</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:derekN@qimr.edu.au">derekN@qimr.edu.au</a></p>

                <p>Affiliation:
                  Oncogenomics, Queensland Institute of Medical Research, Herston, Queensland, Australia
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Herlina Y. Handoko, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Oncogenomics, Queensland Institute of Medical Research, Herston, Queensland, Australia
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Mitchell S. Stark, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Oncogenomics, Queensland Institute of Medical Research, Herston, Queensland, Australia
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            David C. Whiteman, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Cancer and Population Studies, Queensland Institute of Medical Research, Herston, Queensland, Australia
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Nicholas K. Hayward
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Oncogenomics, Queensland Institute of Medical Research, Herston, Queensland, Australia
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: October 31, 2007</li>
    <li>DOI: 10.1371/journal.pone.0001093</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0001093" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0001093" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001093" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0001093" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0001093-g001" data-doi="info:doi/10.1371/journal.pone.0001093" data-uri="info:doi/10.1371/journal.pone.0001093.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001093.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001093-g002" data-doi="info:doi/10.1371/journal.pone.0001093" data-uri="info:doi/10.1371/journal.pone.0001093.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001093.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001093-g003" data-doi="info:doi/10.1371/journal.pone.0001093" data-uri="info:doi/10.1371/journal.pone.0001093.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001093.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001093-g004" data-doi="info:doi/10.1371/journal.pone.0001093" data-uri="info:doi/10.1371/journal.pone.0001093.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001093.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001093-g005" data-doi="info:doi/10.1371/journal.pone.0001093" data-uri="info:doi/10.1371/journal.pone.0001093.g005" title="Figure 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001093.g005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001093">Reader Comments (0)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0001093" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1"></a><p>The recent application of genome-wide, single nucleotide polymorphism (SNP) microarrays to investigate DNA copy number aberrations in cancer has provided unparalleled sensitivity for identifying genomic changes. In some instances the complexity of these changes makes them difficult to interpret, particularly when tumour samples are contaminated with normal (stromal) tissue. Current automated scoring algorithms require considerable manual data checking and correction, especially when assessing uncultured tumour specimens. To address these limitations we have developed a visual tool to aid in the analysis of DNA copy number data. <span class="underline">Si</span>mulated <span class="underline">D</span>NA <span class="underline">Co</span>py <span class="underline">N</span>umber (SiDCoN) is a spreadsheet-based application designed to simulate the appearance of B-allele and logR plots for all known types of tumour DNA copy number changes, in the presence or absence of stromal contamination. The system allows the user to determine the level of stromal contamination, as well as specify up to 3 different DNA copy number aberrations for up to 5000 data points (representing individual SNPs). This allows users great flexibility to assess simple or complex DNA copy number combinations. We demonstrate how this utility can be used to estimate the level of stromal contamination within tumour samples and its application in deciphering the complex heterogeneous copy number changes we have observed in a series of tumours. We believe this tool will prove useful to others working in the area, both as a training tool, and to aid in the interpretation of complex copy number changes.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>Nancarrow DJ, Handoko HY, Stark MS, Whiteman DC, Hayward NK (2007) SiDCoN: A Tool to Aid Scoring of DNA Copy Number Changes in SNP Chip Data. PLoS ONE 2(10):
          e1093.
            doi:10.1371/journal.pone.0001093</p><p><strong>Academic Editor: </strong>Anja-Katrin Bielinsky, University of Minnesota, United States of America</p><p><strong>Received:</strong> September 12, 2007; <strong>Accepted:</strong> October 4, 2007; <strong>Published:</strong> October 31, 2007</p><p><strong>Copyright:</strong> © 2007 Nancarrow et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>This research was supported by grant number CA 001833-03 from the United States National Cancer Institute, the Queensland Cancer Fund and the National Health and Medical Research Council (NHMRC) of Australia (Program no. 199600). David Whiteman is supported by Senior Research Fellowships from the National Health and Medical Research Council of Australia. The funding bodies played no role in the design or conduct of the study; the collection, management, analysis, or interpretation of the data; or the preparation, review, or approval of the manuscript.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>Single nucleotide polymorphism (SNP) microarrays provide data on both genotype and signal intensity, the combination of which can be used to generate information on chromosomal segment copy number. An increasing number of studies utilise whole-genome high density SNP chips to generate DNA copy number profiles for a variety of tumour types. Kits and software tools are now commercially available for this purpose from a number of suppliers. This emerging technology has distinct advantages over previous karyotype-based comparative genome hybridization (CGH) methods <a href="#pone.0001093-Andersen1">[1]</a> and analytic methods are evolving rapidly. When applying these SNP microarrays (SNP-aCGH) to cancer research, the aim is to synthesize a comprehensive DNA copy number profile which maps aberrations across the entire genome within individual tumour samples.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>There are several method papers devoted to the analysis of DNA copy number using SNP array platforms <a href="#pone.0001093-Colella1">[2]</a>–<a href="#pone.0001093-Huang1">[5]</a> and dedicated software functions are available in commercial applications. There are two broad approaches to this work: 1) identifying statistically significant genomic regions of change (e.g. Colella and coworkers <a href="#pone.0001093-Colella1">[2]</a>); 2) developing tools to auto-analyse the data to generate genome-wide, sample specific DNA copy number profiles.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>The success of SNP-aCGH for mapping sample specific DNA copy number changes stems from the ability to combine CGH and loss of heterozygosity (LOH) studies in the same analysis. As is often the case with new biotechnology, the analysis procedures lag behind the experimental advancements in terms of simplicity and flexibility. While commercially available software applications provide analysis algorithms to identify significant regions of change, we <a href="#pone.0001093-Stark1">[6]</a> have found this to be inadequate for generating a whole-genome view of DNA copy number changes without heavy manual interpretation.</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p>In SNP-aCGH analyses the resulting genotype data consist of intensity values for two channels corresponding to the fluorophors associated with the A &amp; B alleles (attached to specific oligos/beads). Data can be plotted as raw A versus raw B intensity plots, however several refined data presentation methods have proven more useful. One of these, log 2 of the sample intensity to reference intensity ratio (logR), provides a continuous measure of the CGH component of the data. In this case, the signal intensity of each SNP in the target sample is expressed as a ratio over that of the normal sample or reference pool. Log 2 of this ratio provides an effective means to curtail the range of outlying values. While the variability of individual logR values is large, due to variances in PCR conditions and primer sequences, modified algorithms such as that of Nannya and coworkers <a href="#pone.0001093-Nannya1">[7]</a> and the Illumina proprietary method, as well as the application of a moving average, are available to reduce the effects of this variation across a chromosomal region. These features, including a proprietary algorithm for SNP normalisation, are built in to the Illumina Beadstudio 2&amp;3 software packages.</p>
<a id="article1.body1.sec1.p5" name="article1.body1.sec1.p5"></a><p>Another key SNP-aCGH data presentation track, Allele B frequency (Ballele), visualises the LOH component. By adjusting the theta (allelic intensity ratio) value for cluster position (assessed in a panel of normal samples) the relative composition of each SNP allele can be determined <a href="#pone.0001093-Peiffer1">[4]</a>. The Allele B frequency represents the proportion contributed by one SNP allele (B), expressed as a range from 0 to 1. Thus in a normal heterozygous sample (with equal amounts of allele A and allele B for a given SNP) Ballele would be 0.5. Homozygosity is represented by either 0 (AA) or 1 (BB).</p>
<a id="article1.body1.sec1.p6" name="article1.body1.sec1.p6"></a><p>As with traditional LOH analysis by microsatellite or SNP markers, a run of consecutive homozygous markers indicates a region of potential DNA loss or copy neutral LOH. Intermediate values (&gt;0 &amp; &lt;0.5 or &gt;0.5 &amp; &lt;1) represent variation in the amount of allele B relative to the total (A+B) SNP intensity. This can be seen, for example, as a result of incomplete loss of one allele. Ballele plots tend to be less variable than those of logR <a href="#pone.0001093-Peiffer1">[4]</a>. The methods for calculating both Ballele and logR are described in detail by Peiffer and coworkers <a href="#pone.0001093-Peiffer1">[4]</a>.</p>
<a id="article1.body1.sec1.p7" name="article1.body1.sec1.p7"></a><p>When considered together, Ballele and logR plots of SNP-aCGH data allow the relative amounts of each parental marker to be estimated across each chromosome. It is unnecessary to type parents for SNP-aCGH, thus the maternal or paternal origin of an individual chromosome is unknown. However, as one delves into deeper levels of DNA copy number complexity there is a need to be able to clearly express which genotype is being referred to. As an example, a 3n genotype (simple amplification) can consist of chromosome segments from the same parent (<em>AAA</em>) or an imbalance of chromosomes from both parents (<em>AAB, ABB</em>). SNP-aCGH can distinguish <em>AAA</em> from the other two forms of 3n amplification (which would require parental information to be separated), and there may be a biological distinction; does a target gene within an <em>AAA</em> region function in the context of tumour suppression, or as an oncogene? In this manuscript we have designated <em>A</em> and <em>B</em> to distinguish between paired chromosomes to allow a more detailed description of the observable genotypes. Note that italics have been used to distinguish <em>A</em> &amp; <em>B</em> parental chromosomes from the A and B alleles at an individual SNP. We chose this nomenclature since “A” and “B” have historically been used extensively in both instances.</p>
<a id="article1.body1.sec1.p8" name="article1.body1.sec1.p8"></a><p>Interpreting Ballele and logR plots together provides an estimate of the average copy number of <em>A</em> and <em>B</em> versions of each autosome, thus providing a sample specific genome-wide profile of DNA copy number. These views essentially correspond to data generated from the original CGH (~logR) and LOH (~Ballele) methodologies and it is the ability to simultaneously interpret both in a very dense map that makes SNP-aCGH so powerful.</p>
<a id="article1.body1.sec1.p9" name="article1.body1.sec1.p9"></a><p>When compared to cancer cell lines, DNA from a tumour biopsy often generates a distinctively different pattern (Ballele &amp; logR) for a variety of the common copy number changes observed when using SNP microarrays. This is due to the presence of non-cancer cells within the biopsy, including inflammatory cells, connective tissues and other components referred to as stromal contamination (<strong>normal 2n</strong> chromosomal complement). Even a small amount of non-cancer tissue (5–10%) will substantially alter the appearance of logR and Ballele plots of certain changes, such as LOH and homozygously deleted (HD) regions (discussed in detail below). Since the proprietary algorithms have been developed and optimized for use with cell lines they perform poorly for tumour biopsies contaminated with stromal tissue.</p>
<a id="article1.body1.sec1.p10" name="article1.body1.sec1.p10"></a><p>Several problems arise when attempting to provide a detailed genome-wide profile, particularly in tumour biopsies, all of which can be related to the ability to interpret the critical data captured within the Ballele and logR plots, whether manually, or by some algorithm. Here we describe a simple application which simulates all possible DNA copy number changes in terms of Ballele and logR plots. We believe this tool, <span class="underline">Si</span>mulated <span class="underline">D</span>NA <span class="underline">Co</span>py <span class="underline">N</span>umber (SiDCoN), will be helpful for interpreting complex regions of change, as well as for training researchers to accurately score whole-genome profiles in the presence of significant stromal contamination. Furthermore SiDCoN allows the user to estimate the level of stromal contamination within a tumour sample.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results and Discussion"></a><h3>Results and Discussion</h3><a id="article1.body1.sec2.p1" name="article1.body1.sec2.p1"></a><p>SiDCoN was used to generate simulated data for all common DNA copy number changes, assuming either no stromal component to the sample, such as seen for a tumour cell line (<a href="#pone-0001093-g001">Figure 1a</a>), or assuming a 20% stromal component, as might be expected for an average tumour biopsy (<a href="#pone-0001093-g001">Figure 1b</a>). The following sections describe the defining features of each DNA copy number variation presented in <a href="#pone-0001093-g001">Figure 1</a>, highlighting the effect the presence of stroma has on the profiles.</p>
<div class="figure" id="pone-0001093-g001"><div class="img"><a name="pone-0001093-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001093.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001093" data-uri="info:doi/10.1371/journal.pone.0001093.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001093.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001093.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001093.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001093.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001093.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001093.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001093.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001093.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001093.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>A comparison of DNA copy number genotypes showing Ballele (top) and logR (bottom) simulated data for a) tumour cell line showing 100% changes in each case, and b) tumour biopsy with 80% tumour and 20% stromal (normal) DNA.</span></strong></p><a id="article1.body1.sec2.fig1.caption1.p1" name="article1.body1.sec2.fig1.caption1.p1"></a><p>In each case all common DNA copy number changes are represented (separated by vertical lines) and discussed sequentially within the text.</p>
<span>doi:10.1371/journal.pone.0001093.g001</span></div>
<h4>Observable DNA Copy Number Genotypes</h4>
<h5>Normal [2n, AB].</h5><a id="article1.body1.sec2.sec1.sec1.p1" name="article1.body1.sec2.sec1.sec1.p1"></a><p>Since stroma is assumed to be normal 2n chromosomal complement, Ballele and logR plots are identical for tumour cell lines (<a href="#pone-0001093-g001">Figure 1a</a>) and tumour biopsies (<a href="#pone-0001093-g001">Figure 1b</a>). The B allele frequencies for each SNP are either close to zero (no B allele) or 1 (2 B alleles) indicating uninformativeness, or close to 0.5 indicating a heterozygous signal with equal proportions of both alleles. Deviations from 0.5 are the result of “random” fluctuations in the experimental system. Across a chromosomal region (multiple SNPs) the relative B-allele intensity sits around 0.5 and the logR is expected to be close to zero; equal proportions of sample and reference DNA resulting in a ratio of 1 and a log2 value of zero. When scoring a region as normal 2n, with a B-allele at 0.5, it is important that the logR be at zero since there are several alterations which have a normal B allele (see HD and 4n normal below). This is one of the major benefits of using a matched normal sample as a reference, rather than the Illumina reference cluster, as described by Peiffer and coworkers <a href="#pone.0001093-Peiffer1">[4]</a>.</p>

<h5>Loss of Heterozygosity (LOH) [1n, A or B].</h5><a id="article1.body1.sec2.sec1.sec2.p1" name="article1.body1.sec2.sec1.sec2.p1"></a><p>LOH is one of the most common DNA copy number changes seen in cancer and its occurrence can help localize the position of a tumour suppressor gene. As can be seen from <a href="#pone-0001093-g001">Figure 1</a> the presence of stromal tissue changes the appearance of the Ballele plot. When the sample contains 100% tumour cells, all with LOH in the region of study, the Ballele plot consists of values either close to zero or close to 1, since there is only one allele present in the sample (<a href="#pone-0001093-g001">Figure 1a</a>). We can label this as pure LOH. However, when stromal tissue is present the allele B frequency for each polymorphic SNP shifts towards 0.5 (<a href="#pone-0001093-g001">Figure 1b</a>). The extent of the shift can be directly proportional to the level of stroma and this phenomenon is described in detail under “Estimating Stromal Contamination” (<a href="#pone-0001093-g002">Figure 2a</a>). Pure LOH results in an expected logR value of minus 0.54 (Illumina Beadstudio manual) as shown in <a href="#pone-0001093-g001">Figure 1a</a>, and, as can be seen in <a href="#pone-0001093-g002">Figure 2a</a>, the presence of stroma makes this signal weaker (closer to zero) in a proportional manner.</p>
<div class="figure" id="pone-0001093-g002"><div class="img"><a name="pone-0001093-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001093.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001093" data-uri="info:doi/10.1371/journal.pone.0001093.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001093.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001093.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001093.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001093.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001093.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001093.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001093.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001093.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001093.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Simulated Ballele (top) and logR (bottom) plots showing serial dilution of the a) loss of heterozygousity, and b) amplification (3n) tumour genotypes in the presence of increasing levels of stromal (normal).</span></strong></p><a id="article1.body1.sec2.sec1.sec2.fig1.caption1.p1" name="article1.body1.sec2.sec1.sec2.fig1.caption1.p1"></a><p>In each case stromal levels of 0% (pure tumour) to 80% are represented in steps of 10% (separated by vertical lines), along with the normal 2n genotype.</p>
<span>doi:10.1371/journal.pone.0001093.g002</span></div>
<h5>Copy Neutral LOH (N-LOH) [2n, AA or BB].</h5><a id="article1.body1.sec2.sec1.sec3.p1" name="article1.body1.sec2.sec1.sec3.p1"></a><p>N-LOH, or uniparental disomy, is actually the result of two events, first LOH occurs, removing one copy of the target chromosomal region and then this region is duplicated. Alternatively amplification followed by loss of the non-amplified chromosome can result in N-LOH. The resulting genotype is a normal (2n) DNA complement which consists of the same chromosomal version: <em>AA</em> or <em>BB</em> instead of the <em>AB</em> seen in a normal heterozygous region. One of the advantages of using a SNP platform to document DNA copy number changes in tumours is that, unlike karyotype-based CGH methods, N-LOH can be distinguished from normal 2n <a href="#pone.0001093-Andersen1">[1]</a>. With standard LOH methods N-LOH would be seen as LOH, thus specific characterization of N-LOH events is relatively new. N-LOH is surprisingly common in a variety of tumours <a href="#pone.0001093-Andersen1">[1]</a>. As with pure LOH, pure N-LOH results in a Ballele plot that consists of values close to zero or 1, since all the SNPs are homozygous (<a href="#pone-0001093-g001">Figure 1a</a>). When stromal tissue is present the Ballele plot does shift towards 0.5 as seen with LOH, however the degree of this shift is different, but predictable (<a href="#pone-0001093-g001">Figure 1b</a>). The N-LOH genotype results in a logR value that is close to zero (the same as normal 2n tissue) in either the presence or absence of stroma (<a href="#pone-0001093-g001">Figure 1</a>).</p>

<h5>Amplified LOH (AMP-LOH) [3n, AAA or BBB].</h5><a id="article1.body1.sec2.sec1.sec4.p1" name="article1.body1.sec2.sec1.sec4.p1"></a><p>The formation of AMP-LOH is similar to that of N-LOH, except that the chromosomal region resulting from LOH is duplicated more than once. <a href="#pone-0001093-g001">Figure 1</a> only depicts AAA amplification (3n), although it is possible to use SiDCoN to simulate as many n as required. As with both LOH and N-LOH, pure AMP-LOH will present a B-allele plot consisting of 0 or 1 values (with the system variation) irrespective of how many times the region has been duplicated, and the logR value will approximate that of 3n amplification (~0.34 – Illumina Beadstudio manual) if there are 3 versions of the target chromosomal region, <em>AAA</em> (<a href="#pone-0001093-g001">Figure 1a</a>). When stromal tissue is present the Ballele plot shifts towards 0.5, and again, the degree of shift relates to the amount of normal tissue present (<a href="#pone-0001093-g001">Figure 1b</a>). The level of stromal contamination also controls the strength of the gain observed in the logR. Given the small shift (0.34) expected for pure 3n genotypes, increasing levels of stroma make the logR signal closer and closer to zero. With the degree of variation within the logR signal it is often difficult to distinguish AMP-LOH from N-LOH in the presence of more than 60% stroma.</p>

<h5>Homozygous Deletion [0n, 0].</h5><a id="article1.body1.sec2.sec1.sec5.p1" name="article1.body1.sec2.sec1.sec5.p1"></a><p>The most apparent change associated with a pure homozygous deletion (HD) in cell lines is a very low logR score, below −2 and often dipping below −4. In the presence of stroma, however, the magnitude of the change can be substantially smaller, depending upon the amount of non-cancer tissue present. In the case of complete HD within a cell line, where essentially no DNA remains for the chromosomal region in question, there is not enough signal for an accurate determination of the allele B frequency of each SNP, resulting in essentially random numbers for the Ballele plot (<a href="#pone-0001093-g001">Figure 1a</a>). In the presence of even a trace of non-cancer cells there is sufficient material to gain an accurate allele B frequency, however, since this arises solely from non-cancer tissue the Ballele plot appears as normal 2n. Therefore, unlike pure HD, HD in the presence of stroma can only be distinguished from normal 2n by the drop in logR (<a href="#pone-0001093-g001">Figure 1b</a>). It should be noted that in tumours HD is often mixed with LOH, as discussed below.</p>
<a id="article1.body1.sec2.sec1.sec5.p2" name="article1.body1.sec2.sec1.sec5.p2"></a><p>When scoring HD on the X chromosome of male subjects, it should be remembered that the normal state (one X chromosome) has a 1 or 0 Ballele pattern. Thus the loss of the remaining allele is only evident by the drop in logR value. The difference between HD in the presence or absence of non-cancer cells is only evidenced by the degree of logR drop (data not shown).</p>

<h5>4n Balanced Amplification [4n, AABB].</h5><a id="article1.body1.sec2.sec1.sec6.p1" name="article1.body1.sec2.sec1.sec6.p1"></a><p>When the normal chromosomal complement (2n) is duplicated within the target region the logR plot is representative of 4n, while the Ballele plot shows a normal 0.5 signal (<a href="#pone-0001093-g001">Figure 1</a>). In the case where there is a mixture of normal stroma and target-specific tetraploid cells, the strength of the logR signal increase is reduced, while the B allele is unchanged (<a href="#pone-0001093-g001">Figure 1b</a> vs <a href="#pone-0001093-g001">Figure 1a</a>). If a proportion of the tumour sample is tetraploid (4n total chromosome complement), which often happens in a number of cancer types, this is not detectable by current SNP-aCGH analyses, due to the necessity to normalize DNA content between sample and reference.</p>

<h5>Simple Amplification (3nAMP) [3n, AAB or ABB].</h5><a id="article1.body1.sec2.sec1.sec7.p1" name="article1.body1.sec2.sec1.sec7.p1"></a><p>In combination with LOH, regions of genomic amplification are considered hallmarks of the tumour genotype. While LOH often denotes the presence of a tumour suppressor, amplification is indicative of gene(s) that have an oncogenic effect. Given the small shift of 0.34 expected for pure 3n amplified genotypes (<a href="#pone-0001093-g001">Figure 1a</a>), the presence of a moderate proportion of stroma shifts the logR signal close to zero (<a href="#pone-0001093-g001">Figure 1b</a>), such that it is often difficult to distinguish 3n from 2n using logR alone (<a href="#pone-0001093-g002">Figure 2b</a>). In pure 3nAMP the Ballele plot has a characteristic split, with clusterings at 0.66 and 0.34 amongst polymorphic SNPs (along with nonpolymorphic SNPs which cluster at either 1 or 0), indicating that either 2/3 or 1/3 of the alleles are B at individual SNPs (<a href="#pone-0001093-g001">Figure 1a</a>). Consistent with the other changes described above, the presence of stroma shifts the Ballele signal back towards 0.5. The degree of shift is proportional to the percentage of cells with a normal genotype, as shown in <a href="#pone-0001093-g002">Figure 2b</a>. Note the distinctions in Ballele plot between 3nAMP (<em>AAB</em>) and AMP-LOH (<em>AAA</em>) making it possible to clearly distinguish these two genotypes, even in the presence of stroma (<a href="#pone-0001093-g001">Figure 1</a>).</p>

<h5>Complex Amplification [e.g. 4n, AAAB].</h5><a id="article1.body1.sec2.sec1.sec8.p1" name="article1.body1.sec2.sec1.sec8.p1"></a><p>Higher levels of amplification are reasonably common in tumour samples, e.g. around oncogenes such as ERBB2 and MYC. They differ from 3nAMP in the degree of shift towards 1 in the Ballele plot and increasing logR values, as the ratio of amplified to normal alleles increases in relation to the total number of DNA copies. As the number of copies increases, the possible number of genotype combinations also grows, although generally one allele is chosen to be overrepresented. <a href="#pone-0001093-g001">Figure 1</a> illustrates 4nAMP (<em>AAAB</em>), 5nAMP (<em>AAAAB</em>) and 6nAMP (<em>AAAAB</em>). Note the clear distinction in B allele plot for 4nAMP (<em>AAAB</em>) and 4n-ploidy (<em>AABB</em>) in <a href="#pone-0001093-g001">Figure 1</a>. In the presence of stroma it becomes very difficult to unambiguously distinguish 3nAMP from 4nAMP unless the degree of stromal contamination is known (data not shown). Using SiDCoN one can demonstrate that the allelic mixtures for higher levels of amplification (&gt;6n) are difficult to distinguish, especially in the presence of stroma (data now shown).</p>



<h4>Esimating Stromal Contamination in Tumour Samples</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>Cancer researchers often only have access to small biopsies of tumours, from which accurate estimates of stromal contamination can be difficult. Visual estimates based on the histologic appearance of a neighbouring tissue section can be used as a guide, but there is no guarantee that the section is representative. If the biopsy is small, halving it for pathology considerably reduces yield (often disproportionately) and the tumour content estimate is still only a guide. For both expression and DNA copy-number profiling the presence of large amounts of stromal tissue will reduce the number of differences observed; introducing a type 2 error.</p>
<a id="article1.body1.sec2.sec2.p2" name="article1.body1.sec2.sec2.p2"></a><p>As can be seen in <a href="#pone-0001093-g002">Figure 2</a>, the amount of non-cancer tissue present within a tumour alters both the B allele and logR signals for several copy number changes. We propose that at the site of any of the simple copy number changes seen in tumours (LOH, N-LOH &amp; 3nAMP) the degree of shift can be used to estimate the percentage of cells that have a normal 2n genotype. Using SiDCoN we have been able to assess the tumour/normal DNA mixing experiment presented in Figure 5 (plates A-E) of Peiffer et al <a href="#pone.0001093-Peiffer1">[4]</a>. By simulating various ratios of normal and LOH genotypes it is possible to estimate the tumour DNA content in each of the 75%, 50% and 25% mixtures generated by Peifer and coworkers as approximately 71%, 45% and 22% respectively, based on the Ballele plots as presented. In addition, the tumour content in figure 5, plate F of Peiffer et al <a href="#pone.0001093-Peiffer1">[4]</a> can also be estimated from the LOH stretch present across the later half of chromosome 13. We estimate that the 0.67 Ballele split corresponds to 52% tumour (or ~50% from the serial dilution presented in <a href="#pone-0001093-g002">Figure 2a</a>), rather than 67% as described in their legend to figure 5 <a href="#pone.0001093-Peiffer1">[4]</a>. Peiffer and coworkers have not accounted for the fact that the LOH genotype contains only one chromosome (<em>A</em>), while the normal 2n (stromal) component contains both <em>A</em> and <em>B</em> chromosomes. Thus, the calculation actually is 2 – (1/0.67), rather than 0.67/1, as they assume. We thought it important to highlight this point since this is the only other published example of applying Ballele and logR plots to estimating stromal contamination level.</p>
<a id="article1.body1.sec2.sec2.p3" name="article1.body1.sec2.sec2.p3"></a><p><a href="#pone-0001093-g003">Figure 3</a> demonstrates the use of SiDCoN (<a href="#pone-0001093-g003">Figure 3b</a>) to determine the region-specific level of involvement in a specific DNA sample (<a href="#pone-0001093-g003">Figure 3a</a>). In this example, the sample shows 40% normal cells for both the indicated 3nAMP (p arm) and LOH (q arm), based on the visual similarity of these profiles to that of the actual data; particularly the Ballele plot. Using this visual approach, the level of cellular involvement can be estimated for each DNA copy number change present in a sample.</p>
<div class="figure" id="pone-0001093-g003"><div class="img"><a name="pone-0001093-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001093.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001093" data-uri="info:doi/10.1371/journal.pone.0001093.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001093.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001093.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001093.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001093.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001093.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001093.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001093.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001093.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001093.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Estimating AMP &amp; LOH levels in a) real data using b) SiDCoN.</span></strong></p><a id="article1.body1.sec2.sec2.fig1.caption1.p1" name="article1.body1.sec2.sec2.fig1.caption1.p1"></a><p>Comparison to simulated data indicates that ~60% of cells are AMP or LOH for the indicated regions (thus 40% of cells are normal 2n in each case). These data suggest that this tumour biopsy contains 40% stroma, although more copy number changes across the genome are needed to confirm this.</p>
<span>doi:10.1371/journal.pone.0001093.g003</span></div><a id="article1.body1.sec2.sec2.p4" name="article1.body1.sec2.sec2.p4"></a><p>Furthermore, when multiple DNA copy number changes are present in a sample it is possible to estimate the degree of stromal contamination present in a given tumour biopsy, with the aid of SiDCoN. There are two major complications to this procedure:</p>
<h5>1) Multiple regions of DNA copy number change are required.</h5><a id="article1.body1.sec2.sec2.sec1.p1" name="article1.body1.sec2.sec2.sec1.p1"></a><p>At any given region of the genome a tumour might be heterogeneous for a DNA copy number change, such that a proportion of the tumour cells show a change and the remainder are normal or show a different DNA copy number change. Thus, estimating the stromal contamination rate from a single DNA copy number change may be inaccurate. In addition, across a given chromosome, the pattern of DNA copy number change is likely to vary. One must be certain to choose consistent regions of change. For example across a given chromosome arm the level of LOH may vary. The region of LOH which is closest to the Ballele plot expected for pure LOH (<a href="#pone-0001093-g001">Figure 1a</a>) is most likely to give an accurate estimation of the percentage of stroma present. Note that the X chromosome should be excluded from this estimation in samples of male origin.</p>

<h5>2) Scoring ambiguity.</h5><a id="article1.body1.sec2.sec2.sec2.p1" name="article1.body1.sec2.sec2.sec2.p1"></a><p>As discussed above, at higher levels of stromal contamination it becomes difficult to clearly determine the specific tumour genotype in a given region. Estimating the level of stroma from LOH is the most accurate, simply because the Ballele pattern is the most different from normal. Experimentally we find we can score LOH down to 30% of the sample (<a href="#pone-0001093-g002">Figure 2a</a>), however this is dependent on the ability to discriminate LOH from N-LOH on the logR plot, and thus a straight baseline is required.</p>
<a id="article1.body1.sec2.sec2.sec2.p2" name="article1.body1.sec2.sec2.sec2.p2"></a><p>Taking these factors into consideration, when a primary tumour sample exhibits a number of unambiguous LOH changes across the genome, we can use the Ballele plot level of these to estimate the level of contaminating normal stromal material, as shown from the set of simulations provided in <a href="#pone-0001093-g002">Figure 2</a> where LOH and AMP changes are provided in 10% steps. Practically, we limit our serial steps to 5% when using SiDCoN in this manner, although either 5 or 10% steps provide a good estimate, depending on the number of scorable changes per sample (down to 30% as described above).</p>
<a id="article1.body1.sec2.sec2.sec2.p3" name="article1.body1.sec2.sec2.sec2.p3"></a><p>While the amount of stromal contamination does effect how much the logR mean value does shift towards zero, it is not as accurate as using the Allele B frequency for estimating the stromal contamination level.</p>



<h4>Investigating region-specific DNA copy number heterogeneity in tumour samples</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p><a href="#pone-0001093-g004">Figure 4</a> demonstrates the application of SiDCoN to interpreting the level of DNA copy number change in the presence of stroma (4b and 4c), and estimating the mixture levels of multiple copy number changes (4a and 4b). At the beginning of the melanoma cell line chromosome excerpt depicted in <a href="#pone-0001093-g004">Figure 4a</a> (top) is a region consisting of a combination of LOH and HD cells. Using the simulator (4a bottom) it can be estimated from the logR values that this is the result of 70% of the cell line population being LOH for this region, while the remaining 30% are homozygously deleted. This is consistent with the fact that within this region of 70% LOH/30% HD, is a 100% HD region which contains a known melanoma tumour suppressor gene (CDKN2A). Similarly, in <a href="#pone-0001093-g004">Figure 4b</a> (top) the central section of the chromosomal excerpt from an oesophageal adenocarcinoma tumour biopsy shows a region with a mixture of LOH and HD tumour cells, however in this case normal 2n (stromal) cell population is also present. Note that, at the beginning of this excerpt there is a region of LOH alone, in the presence of a normal 2n population. Using the simulator in the manner demonstrated in <a href="#pone-0001093-g003">Figure 3</a> we estimate that in each case the normal 2n population is ~25% and that the 75% LOH region moves to a mixture of 50% LOH and 25% HD within the central region of the section shown (<a href="#pone-0001093-g004">Figure 4b</a>). By looking at LOH regions across all autosomes for this sample, we confirmed that the stromal contamination rate within the biopsy sample is ~25% (data not shown) suggesting that all tumour cells are involved in the changes described.</p>
<div class="figure" id="pone-0001093-g004"><div class="img"><a name="pone-0001093-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001093.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001093" data-uri="info:doi/10.1371/journal.pone.0001093.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001093.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001093.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001093.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001093.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001093.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001093.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001093.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001093.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001093.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>Some examples of observed vs simulated chromosomal excerpts showing mixed populations of DNA copy number changes (top) and the manually adjusted simulations of these changes (bottom).</span></strong></p><a id="article1.body1.sec2.sec3.fig1.caption1.p1" name="article1.body1.sec2.sec3.fig1.caption1.p1"></a><p>a) a melanoma cell line with changes including HD and mix of LOH &amp; HD. b) an EAC tumour biopsy profile which includes LOH, N-LOH and a mix of HD and LOH. The simulator is particularly useful for explaining LOH/HD combinations in the presence of stroma as seen here. c) another EAC tumour biopsy with changes on a higher background of stroma/normal cells. Manually adjusting the simulator is useful for determining the level of tumour cell involvement in each change.</p>
<span>doi:10.1371/journal.pone.0001093.g004</span></div><a id="article1.body1.sec2.sec3.p2" name="article1.body1.sec2.sec3.p2"></a><p>As the application of SNP-aCGH to investigate DNA copy number changes becomes more popular, investigators will want to extract as much information as possible from these data. We present here a simple simulation-based method to investigate complex DNA copy number changes to a level beyond current analytical methods. While the application of our method is manual, in relation to users adjusting the simulated parameters to obtain a visual match for individual data, SiDCoN provides researchers with the flexibility to assess a wide variety of SNP-aCGH data interpretations. We have demonstrated how this approach can be used to estimate the stromal contamination rate within tumour biopsies, and to describe mixed DNA copy number populations, in the presence or absence of normal 2n cell populations. There is also the potential to implement the basic principles of our application into next-generation autoscoring programs which may save researchers considerable time.</p>

</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Methods"></a><h3>Methods</h3>
<h4>Test samples</h4>
<a id="article1.body1.sec3.sec1.p1" name="article1.body1.sec3.sec1.p1"></a><p>DNA was extracted from melanoma cell lines and oesophageal adenocarcinoma biopsies using Qiagen nucleic acid column purification technology as set out in the manufacturer's instructions (Qiagen, Hilden, Germany). The resulting genomic DNA was quantitated using a Nanodrop and 750ug applied to Infinium II Whole Genome Genotyping HumanHap300 Beadarray chips as per manufacturer's instructions (Illumina, San Diego, CA, USA). The chip image data was then processed through Beadstation and Beadstudio 2 software applications to generate B-allele and logR plots. In the case of melanoma cell lines, the Illumina reference pool was used as a reference. For the adenocarcinoma biopsies a normal squamous oesophageal biopsy (from one of the cancer patients) was used as a reference. The normal squamous tissue was compared to the Illumina reference pool to look for any anomalous regions. This highlighted a small region of change on 6q which was excluded from the analyses of the oesophageal tumour biopsies. We found, as reported by Peiffer and co-workers <a href="#pone.0001093-Peiffer1">[4]</a>, that using a local normal tissue in this manner gave a much cleaner baseline signal compared to using the Illumina pooled reference.</p>


<h4>Using SiDCoN</h4>
<a id="article1.body1.sec3.sec2.p1" name="article1.body1.sec3.sec2.p1"></a><p>SiDCoN (<a href="#pone.0001093.s001">Supplement S1</a>) is an Microsoft EXCEL spreadsheet application that allows users to enter up to 3 copy number genotypes, along with a stromal proportion, for up to 5000 observable datapoints (SNPs). <a href="#pone.0001093.s002">Supplement S2</a> is a help file outlining its usage. Generally, we envisage its use will be restricted to estimating one genotype and stromal contamination, or a mixture of 2 copy number variants; however we wanted the tool to be as flexible as possible. Each copy number genotype is entered into the main “datasheet”, along with the proportion of cells (0–1) with this type (<a href="#pone-0001093-g005">Figure 5a</a>). If the total of the 3 genotypes for a SNP is less than 1, the remaining portion is assumed to be stroma (normal 2n, AB). The relative proportion each entered DNA copy number genotype makes towards the overall pattern is determined using predetermined values from the “lookup” sheet, shown in <a href="#pone-0001093-g005">Figure 5b</a>, and the entered fractions. From these calculations the expected Ballele and logR values are determined for each SNP (columns V and W on the right hand side of <a href="#pone-0001093-g005">Figure 5a</a>). A randomization algorithm is then applied to each datapoint, using the EXCEL “RAND()” function, to allow the simulated data plots to appear visually similar to real data. The resulting Ballele and logR plots are then presented as EXCEL graphs on separate sheets (not shown), or combined on the “both graphs” sheet (<a href="#pone-0001093-g005">Figure 5c</a>). The results (<a href="#pone-0001093-g001">Figure 1</a>) provide a means to generate reasonable facsimiles of actual DNA copy number data, enabling the user to determine how best to interpret patterns observed in the real data.</p>
<div class="figure" id="pone-0001093-g005"><div class="img"><a name="pone-0001093-g005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001093.g005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001093" data-uri="info:doi/10.1371/journal.pone.0001093.g005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001093.g005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001093.g005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001093.g005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001093.g005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001093.g005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001093.g005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001093.g005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001093.g005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001093.g005.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 5.  <span>Screen grabs from SiDCoN showing a) the main “datasheet” interface with space for three DNA copy number genotypes and a stromal component for each SNP, b) the lookup sheet containing information needed for calculations dependent on the copy number genotypes entered and c) the Ballele and logR output, implementing randomised values to visually simulate the look of actual data.</span></strong></p><span>doi:10.1371/journal.pone.0001093.g005</span></div><a id="article1.body1.sec3.sec2.p2" name="article1.body1.sec3.sec2.p2"></a><p>Ballele frequency graphs focus around 3 (normal 2n) or 4 lines which correspond to whether the B allele for each SNP is informative or uninformative with either no copies (0) or all copies (1) of allele B. When the B allele is informative it can either be the predominant SNP allele (&gt;0.5), the minimal SNP allele (&lt;0.5) or in equal proportion with A (0.5). The latter case, results in 3 focus lines (1, 0.5 &amp; 0) (see <a href="#pone-0001093-g001">Figure 1a</a>, normal &amp; 4-ploid). Instances where there is only 1 parental chromosome present result in only 2 lines of focus (0 &amp; 1) (see <a href="#pone-0001093-g001">Figure 1a</a>, LOH, N-LOH and AMP-LOH) while all other instances result in 4 lines of focus. Since all SNPs in the current assay have only two possible alleles (A &amp; B) this is always the case (even for sex chromosomes), unless there are more than 2 parental chromosomes present. To be consistent with Peiffer and coworkers <a href="#pone.0001093-Peiffer1">[4]</a>, we represent Ballele as the position of the upper informative line of focus, giving a range of 0.5 to 1. A value of 1 means only one parental chromosome is present such that no SNPs are informative (lines of focus at 0 and 1) as is the case for complete LOH. A value of 0.5 indicates a normal (2n) Ballele appearance with lines of focus at 1, 0 and 0.5, while other values (between 1 and 0.5 indicate there are 4 lines of focus 1, 0, Ballele and 1-Ballele).</p>
<a id="article1.body1.sec3.sec2.p3" name="article1.body1.sec3.sec2.p3"></a><p>SiDCoN generates simulated mean logR values using the formula:<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001093.e001&amp;representation=PNG" class="inline-graphic"></span><br>We expected that log2 would yield the correct result, but for some reason log10, in the simulations, provides much closer values to observed logR values. Presumably this is the result of using simulated R values.</p>

</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0001093"></div><a name="pone.0001093.s001" id="pone.0001093.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001093.s001">Supplement S1. </a></strong></p><a id="article1.body1.sec4.supplementary-material1.caption1.p1" name="article1.body1.sec4.supplementary-material1.caption1.p1"></a><p class="preSiDOI">Simulated DNA Copy Number (SiDCoN) - a spreadsheet application designed to simulate the appearance of B-allele and logR plots for all known types of tumour DNA copy number changes in the presence or absence of stromal contamination</p>
<p class="siDoi">doi:10.1371/journal.pone.0001093.s001</p><a id="article1.body1.sec4.supplementary-material1.caption1.p2" name="article1.body1.sec4.supplementary-material1.caption1.p2"></a><p class="postSiDOI">(4.89 MB XLS)</p>
<a name="pone.0001093.s002" id="pone.0001093.s002"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001093.s002">Supplement S2. </a></strong></p><a id="article1.body1.sec4.supplementary-material2.caption1.p1" name="article1.body1.sec4.supplementary-material2.caption1.p1"></a><p class="preSiDOI">Instructions for using SiDCoN</p>
<p class="siDoi">doi:10.1371/journal.pone.0001093.s002</p><a id="article1.body1.sec4.supplementary-material2.caption1.p2" name="article1.body1.sec4.supplementary-material2.caption1.p2"></a><p class="postSiDOI">(0.03 MB DOC)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>We wish to acknowledge the contribution of the other Study of Digestive Health investigators; Adele C. Green, Peter G. Parsons, David M. Purdie, Penelope M. Webb (Queensland Institute of Medical Research), Sandra J. Pavey (Princess Alexandra Hospital), David Gotley, B. Mark Smithers (University of Queensland/Princess Alexandra Hospital), Paul Drew, Glyn Jamieson (University of Adelaide), David Watson (Flinders University of South Australia), Andrew Clouston (Envoi Pathology).</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: DN DW NH. Performed the experiments: HH MS. Analyzed the data: DN HH. Wrote the paper: DN. Other: Critical review of manuscript: HH MS DW NH.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0001093-Andersen1" id="pone.0001093-Andersen1"></a>Andersen CL, Wiuf C, Kruhoffer M, Korsgaard M, Laurberg S, et al.  (2007) Frequent occurrence of uniparental disomy in colorectal cancer. Carcinogenesis  28: 38–48.  <ul class="find" data-citedArticleID="1010510" data-doi="10.1093/carcin/bgl086"><li><a href="http://dx.doi.org/10.1093/carcin/bgl086" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Frequent+occurrence+of+uniparental+disomy+in+colorectal+cancer." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Frequent+occurrence+of+uniparental+disomy+in+colorectal+cancer.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0001093-Colella1" id="pone.0001093-Colella1"></a>Colella S, Yau C, Taylor JM, Mirza G, Butler H, et al.  (2007) QuantiSNP: an Objective Bayes Hidden-Markov Model to detect and accurately map copy number variation using SNP genotyping data. Nucleic Acids Res  35: 2013–2025.  <ul class="find" data-citedArticleID="1010512" data-doi="10.1093/nar/gkm076"><li><a href="http://dx.doi.org/10.1093/nar/gkm076" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=QuantiSNP%3A+an+Objective+Bayes+Hidden-Markov+Model+to+detect+and+accurately+map+copy+number+variation+using+SNP+genotyping+data." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22QuantiSNP%3A+an+Objective+Bayes+Hidden-Markov+Model+to+detect+and+accurately+map+copy+number+variation+using+SNP+genotyping+data.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0001093-Gibbs1" id="pone.0001093-Gibbs1"></a>Gibbs JR, Singleton A (2006) Application of Genome-Wide Single Nucleotide Polymorphism Typing: Simple Association and Beyond. PLoS Genetics  2: e150.  <ul class="find" data-citedArticleID="1010514" data-doi="10.1371/journal.pgen.0020150"><li><a href="http://dx.doi.org/10.1371/journal.pgen.0020150" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Application+of+Genome-Wide+Single+Nucleotide+Polymorphism+Typing%3A+Simple+Association+and+Beyond." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Application+of+Genome-Wide+Single+Nucleotide+Polymorphism+Typing%3A+Simple+Association+and+Beyond.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0001093-Peiffer1" id="pone.0001093-Peiffer1"></a>Peiffer DA, Le JM, Steemers FJ, Chang W, Jenniges T, et al.  (2006) High-resolution genomic profiling of chromosomal aberrations using Infinium whole-genome genotyping. Genome Res  16: 1136–1148.  <ul class="find" data-citedArticleID="1010520" data-doi="10.1101/gr.5402306"><li><a href="http://dx.doi.org/10.1101/gr.5402306" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=High-resolution+genomic+profiling+of+chromosomal+aberrations+using+Infinium+whole-genome+genotyping." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22High-resolution+genomic+profiling+of+chromosomal+aberrations+using+Infinium+whole-genome+genotyping.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0001093-Huang1" id="pone.0001093-Huang1"></a>Huang J, Wei W, Chen J, Zhang J, Liu G, et al.  (2006) CARAT: a novel method for allelic detection of DNA copy number changes using high density oligonucleotide arrays. BMC Bioinformatics  7: 83.  <ul class="find" data-citedArticleID="1010516"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=CARAT%3A+a+novel+method+for+allelic+detection+of+DNA+copy+number+changes+using+high+density+oligonucleotide+arrays.&amp;auth=&amp;atitle=CARAT%3A+a+novel+method+for+allelic+detection+of+DNA+copy+number+changes+using+high+density+oligonucleotide+arrays." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=CARAT%3A+a+novel+method+for+allelic+detection+of+DNA+copy+number+changes+using+high+density+oligonucleotide+arrays." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22CARAT%3A+a+novel+method+for+allelic+detection+of+DNA+copy+number+changes+using+high+density+oligonucleotide+arrays.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0001093-Stark1" id="pone.0001093-Stark1"></a>Stark M, Hayward N (2007) Genome-wide loss of heterozygosity and copy number analysis in melanoma using high-density single-nucleotide polymorphism arrays. Cancer Res  67: 2632–2642.  <ul class="find" data-citedArticleID="1010522" data-doi="10.1158/0008-5472.can-06-4152"><li><a href="http://dx.doi.org/10.1158/0008-5472.can-06-4152" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genome-wide+loss+of+heterozygosity+and+copy+number+analysis+in+melanoma+using+high-density+single-nucleotide+polymorphism+arrays." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genome-wide+loss+of+heterozygosity+and+copy+number+analysis+in+melanoma+using+high-density+single-nucleotide+polymorphism+arrays.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0001093-Nannya1" id="pone.0001093-Nannya1"></a>Nannya Y, Sanada M, Nakazaki K, Hosoya N, Wang L, et al.  (2005) A robust algorithm for copy number detection using high-density oligonucleotide single nucleotide polymorphism genotyping arrays. Cancer Res  65: 6071–6079.  <ul class="find" data-citedArticleID="1010518" data-doi="10.1158/0008-5472.can-05-0465"><li><a href="http://dx.doi.org/10.1158/0008-5472.can-05-0465" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+robust+algorithm+for+copy+number+detection+using+high-density+oligonucleotide+single+nucleotide+polymorphism+genotyping+arrays." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+robust+algorithm+for+copy+number+detection+using+high-density+oligonucleotide+single+nucleotide+polymorphism+genotyping+arrays.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001093.XML" value="52877"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001093.PDF" value="863877"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001093.g001.PNG_L" value="227788"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001093.g001.PNG_M" value="164442"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001093.g001.PNG_S" value="11218"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001093.g001.TIF" value="748558"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001093.g001.PNG_I" value="56773"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001093.g002.PNG_L" value="2469439"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001093.g002.PNG_M" value="164930"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001093.g002.PNG_S" value="10759"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001093.g002.TIF" value="3572624"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001093.g002.PNG_I" value="57237"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001093.g003.PNG_L" value="4909234"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001093.g003.PNG_M" value="295357"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001093.g003.PNG_S" value="12733"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001093.g003.TIF" value="6947724"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001093.g003.PNG_I" value="96273"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001093.g004.PNG_L" value="3901618"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001093.g004.PNG_M" value="262498"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001093.g004.PNG_S" value="12833"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001093.g004.TIF" value="6493536"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001093.g004.PNG_I" value="86981"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001093.g005.PNG_L" value="3551270"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001093.g005.PNG_M" value="273625"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001093.g005.PNG_S" value="11128"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001093.g005.TIF" value="5897058"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001093.g005.PNG_I" value="90073"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001093.e001.PNG" value="11025"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001093.e001.TIF" value="27752"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001093.s001.XLS" value="4893696"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001093.s002.DOC" value="33792"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001093&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001093" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001093&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0001093&volume=&issue=&title=SiDCoN: A Tool to Aid Scoring of DNA Copy Number Changes in SNP Chip Data&author_name=Derek%20J.%20Nancarrow%2C%20Herlina%20Y.%20Handoko%2C%20Mitchell%20S.%20Stark%2C%20David%20C.%20Whiteman%2C%20Nicholas%20K.%20Hayward&start_page=1&end_page=8" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001093" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001093" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0001093" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001093&amp;t=SiDCoN%3A%20A%20Tool%20to%20Aid%20Scoring%20of%20DNA%20Copy%20Number%20Changes%20in%20SNP%20Chip%20Data" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001093&title=SiDCoN%3A%20A%20Tool%20to%20Aid%20Scoring%20of%20DNA%20Copy%20Number%20Changes%20in%20SNP%20Chip%20Data&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001093&amp;title=SiDCoN%3A%20A%20Tool%20to%20Aid%20Scoring%20of%20DNA%20Copy%20Number%20Changes%20in%20SNP%20Chip%20Data" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001093" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0001093&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'SiDCoN: A Tool to Aid Scoring of DNA Copy Number Changes in SNP Chip Data';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0001093';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20SiDCoN%3A%20A%20Tool%20to%20Aid%20Scoring%20of%20DNA%20Copy%20Number%20Changes%20in%20SNP%20Chip%20Data http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001093" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0001093" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">











          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Alleles%22" title="Search for articles in the subject area:'Alleles'"><div class="flagText">Alleles</div></a>
              <div data-categoryid="46071" data-articleid="25020"
                   data-categoryname="Alleles"
                   class="flagImage" title="Flag 'Alleles' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Biopsy%22" title="Search for articles in the subject area:'Biopsy'"><div class="flagText">Biopsy</div></a>
              <div data-categoryid="19453" data-articleid="25020"
                   data-categoryname="Biopsy"
                   class="flagImage" title="Flag 'Biopsy' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Chromatin+immunoprecipitation%22" title="Search for articles in the subject area:'Chromatin immunoprecipitation'"><div class="flagText">Chromatin immunoprecipitation</div></a>
              <div data-categoryid="40169" data-articleid="25020"
                   data-categoryname="Chromatin immunoprecipitation"
                   class="flagImage" title="Flag 'Chromatin immunoprecipitation' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Chromosomes%22" title="Search for articles in the subject area:'Chromosomes'"><div class="flagText">Chromosomes</div></a>
              <div data-categoryid="17251" data-articleid="25020"
                   data-categoryname="Chromosomes"
                   class="flagImage" title="Flag 'Chromosomes' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Computer+software%22" title="Search for articles in the subject area:'Computer software'"><div class="flagText">Computer software</div></a>
              <div data-categoryid="2287" data-articleid="25020"
                   data-categoryname="Computer software"
                   class="flagImage" title="Flag 'Computer software' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22DNA%22" title="Search for articles in the subject area:'DNA'"><div class="flagText">DNA</div></a>
              <div data-categoryid="32449" data-articleid="25020"
                   data-categoryname="DNA"
                   class="flagImage" title="Flag 'DNA' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Genome+analysis%22" title="Search for articles in the subject area:'Genome analysis'"><div class="flagText">Genome analysis</div></a>
              <div data-categoryid="40541" data-articleid="25020"
                   data-categoryname="Genome analysis"
                   class="flagImage" title="Flag 'Genome analysis' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Variant+genotypes%22" title="Search for articles in the subject area:'Variant genotypes'"><div class="flagText">Variant genotypes</div></a>
              <div data-categoryid="40547" data-articleid="25020"
                   data-categoryname="Variant genotypes"
                   class="flagImage" title="Flag 'Variant genotypes' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=1801'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=6675'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=5719&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>


</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
