

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: HLA Alleles Associated with Slow Progression to AIDS Truly Prefer to Present HIV-1 p24</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0000920"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0000920" />

    <meta name="citation_title" content="HLA Alleles Associated with Slow Progression to AIDS Truly Prefer to Present HIV-1 p24"/>
    <meta itemprop="name" content="HLA Alleles Associated with Slow Progression to AIDS Truly Prefer to Present HIV-1 p24"/>

      <meta name="citation_author" content="José A. M. Borghans"/>
            <meta name="citation_author_institution" content="Theoretical Biology/Bioinformatics, Utrecht University, Utrecht, The Netherlands"/>
            <meta name="citation_author_institution" content="Department of Immunology, University Medical Center Utrecht, Utrecht, The Netherlands"/>
      <meta name="citation_author" content="Anne Mølgaard"/>
            <meta name="citation_author_institution" content="Center for Biological Sequence Analysis, Technical University of Denmark, Lyngby, Denmark"/>
      <meta name="citation_author" content="Rob J. de Boer"/>
            <meta name="citation_author_institution" content="Theoretical Biology/Bioinformatics, Utrecht University, Utrecht, The Netherlands"/>
      <meta name="citation_author" content="Can Keşmir"/>
            <meta name="citation_author_institution" content="Theoretical Biology/Bioinformatics, Utrecht University, Utrecht, The Netherlands"/>
            <meta name="citation_author_institution" content="Center for Biological Sequence Analysis, Technical University of Denmark, Lyngby, Denmark"/>
            <meta name="citation_author_institution" content="Academic Biomedical Centre, Utrecht University, Utrecht, The Netherlands"/>

    <meta name="citation_date" content="2007/9/19"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0000920.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e920"/>
    <meta name="citation_issue" content="9"/>
    <meta name="citation_volume" content="2"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=The influence of HLA genotype on AIDS.; citation_author=M Carrington; citation_author=SJ O'Brien; citation_journal_title=Annu Rev Med; citation_volume=4; citation_number=1; citation_pages=535-551; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Evidence of HIV-1 adaptation to HLA-restricted immune responses at a population level.; citation_author=CB Moore; citation_author=M John; citation_author=IR James; citation_author=FT Christiansen; citation_author=CS Witt; citation_journal_title=Science; citation_volume=296; citation_number=2; citation_pages=1439-1443; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=HIV evolution: CTL escape mutation and reversion after transmission.; citation_author=AJ Leslie; citation_author=KJ Pfafferott; citation_author=P Chetty; citation_author=R Draenert; citation_author=MM Addo; citation_journal_title=Nat Med; citation_volume=10; citation_number=3; citation_pages=282-289; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Reversion of CTL escape-variant immunodeficiency viruses in vivo.; citation_author=TC Friedrich; citation_author=EJ Dodds; citation_author=LJ Yant; citation_author=L Vojnov; citation_author=R Rudersdorf; citation_journal_title=Nat Med; citation_volume=10; citation_number=4; citation_pages=275-281; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Characterization of HLA-B57-restricted human immunodeficiency virus type 1 Gag- and RT-specific cytotoxic T lymphocyte responses.; citation_author=MR Klein; citation_author=SH Van der Burg; citation_author=E Hovenkamp; citation_author=AM Holwerda; citation_author=JW Drijfhout; citation_journal_title=J Gen Virol; citation_volume=79; citation_number=5; citation_pages=2191-2201; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Cross-reactive cytotoxic T lymphocytes against a HIV-1 p24 epitope in slow progressors with B*57.; citation_author=GM Gillespie; citation_author=R Kaul; citation_author=T Dong; citation_author=HB Yang; citation_author=T Rostron; citation_journal_title=AIDS; citation_volume=16; citation_number=6; citation_pages=961-972; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Maintenance of viral suppression in HIV-1-infected HLA-B*57+ elite suppressors despite CTL escape mutations.; citation_author=JR Bailey; citation_author=TM Williams; citation_author=RF Siliciano; citation_author=JN Blankson; citation_journal_title=J Exp Med; citation_volume=203; citation_number=7; citation_pages=1357-1369; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=High responsiveness of HLA-B57-restricted Gag-specific CD8+ T cells in vitro may contribute to the protective effect of HLA-B57 in HIV-infection.; citation_author=CA Jansen; citation_author=S Kostense; citation_author=K Vandenberghe; citation_author=NM Nanlohy; citation_author=IM De Cuyper; citation_journal_title=Eur J Immunol; citation_volume=35; citation_number=8; citation_pages=150-158; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Influence of HLA-B57 on clinical presentation and viral control during acute HIV-1 infection.; citation_author=M Altfeld; citation_author=MM Addo; citation_author=ES Rosenberg; citation_author=FM Hecht; citation_author=PK Lee; citation_journal_title=AIDS; citation_volume=17; citation_number=9; citation_pages=2581-2591; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=HLA alleles associated with delayed progression to AIDS contribute strongly to the initial CD8+ T cell response against HIV-1.; citation_author=M Altfeld; citation_author=ET Kalife; citation_author=Y Qi; citation_author=H Streeck; citation_author=M Lichterfeld; citation_journal_title=PLoS Med; citation_volume=3; citation_number=10; citation_pages=e403; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=HLA-B63 presents HLA-B57/B58-restricted cytotoxic T-lymphocyte epitopes and is associated with low human immunodeficiency virus load.; citation_author=N Frahm; citation_author=S Adams; citation_author=P Kiepiela; citation_author=CH Linde; citation_author=HS Hewitt; citation_journal_title=J Virol; citation_volume=79; citation_number=11; citation_pages=10218-10225; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Identification of a novel HLA-B*2705-restricted cytotoxic T-lymphocyte epitope within a conserved region of HIV-1 Nef.; citation_author=PJ Goulder; citation_author=A Edwards; citation_author=RE Phillips; citation_author=AJ McMichael; citation_journal_title=AIDS; citation_volume=11; citation_number=12; citation_pages=536-538; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Effect of a single amino acid change in MHC class I molecules on the rate of progression to AIDS.; citation_author=X Gao; citation_author=GW Nelson; citation_author=P Karacki; citation_author=MP Martin; citation_author=J Phair; citation_journal_title=N Engl J Med; citation_volume=344; citation_number=13; citation_pages=1668-1675; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=HLA and HIV-1: heterozygote advantage and B*35-Cw*04 disadvantage.; citation_author=M Carrington; citation_author=GW Nelson; citation_author=MP Martin; citation_author=T Kissner; citation_author=D Vlahov; citation_journal_title=Science; citation_volume=283; citation_number=14; citation_pages=1748-1752; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Advantage of rare HLA supertype in HIV disease progression.; citation_author=E Trachtenberg; citation_author=B Korber; citation_author=C Sollars; citation_author=TB Kepler; citation_author=PT Hraber; citation_journal_title=Nat Med; citation_volume=9; citation_number=15; citation_pages=928-935; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Quantifiable cytotoxic T lymphocyte responses and HLA-related risk of progression to AIDS.; citation_author=A Scherer; citation_author=J Frater; citation_author=A Oxenius; citation_author=J Agudelo; citation_author=DA Price; citation_journal_title=Proc Natl Acad Sci USA; citation_volume=101; citation_number=16; citation_pages=12266-12270; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Control of human immunodeficiency virus replication by cytotoxic T lymphocytes targeting subdominant epitopes.; citation_author=N Frahm; citation_author=P Kiepiela; citation_author=S Adams; citation_author=CH Linde; citation_author=HS Hewitt; citation_journal_title=Nat Immunol; citation_volume=7; citation_number=17; citation_pages=173-178; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Inverse correlation between memory Gag-specific cytotoxic T lymphocytes and viral replication in human immunodeficiency virus-infected children.; citation_author=F Buseyne; citation_author=J Le Chenadec; citation_author=B Corre; citation_author=F Porrot; citation_author=M Burgard; citation_journal_title=J Infect Dis; citation_volume=186; citation_number=18; citation_pages=1589-1596; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Magnitude of functional CD8+ T-cell responses to the gag protein of human immunodeficiency virus type 1 correlates inversely with viral load in plasma.; citation_author=BH Edwards; citation_author=A Bansal; citation_author=S Sabbaj; citation_author=J Bakari; citation_author=MJ Mulligan; citation_journal_title=J Virol; citation_volume=76; citation_number=19; citation_pages=2298-2305; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Association between virus-specific T-cell responses and plasma viral load in human immunodeficiency virus type 1 subtype C infection.; citation_author=V Novitsky; citation_author=P Gilbert; citation_author=T Peter; citation_author=MF McLane; citation_author=S Gaolekwe; citation_journal_title=J Virol; citation_volume=77; citation_number=20; citation_pages=882-890; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Hierarchical targeting of subtype C human immunodeficiency virus type 1 proteins by CD8+ T cells: correlation with viral load.; citation_author=A Masemola; citation_author=T Mashishi; citation_author=G Khoury; citation_author=P Mohube; citation_author=P Mokgotho; citation_journal_title=J Virol; citation_volume=78; citation_number=21; citation_pages=3233-3243; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Relative dominance of Gag p24-specific cytotoxic T lymphocytes is associated with human immunodeficiency virus control.; citation_author=R Zuniga; citation_author=A Lucchetti; citation_author=P Galvan; citation_author=S Sanchez; citation_author=C Sanchez; citation_journal_title=J Virol; citation_volume=80; citation_number=22; citation_pages=3122-3125; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=CD8+ T-cell responses to different HIV proteins have discordant associations with viral load.; citation_author=P Kiepiela; citation_author=K Ngumbela; citation_author=C Thobakgale; citation_author=D Ramduth; citation_author=I Honeyborne; citation_journal_title=Nat Med; citation_volume=13; citation_number=23; citation_pages=46-53; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Control of human immunodeficiency virus type 1 is associated with HLA-B*13 and targeting of multiple gag-specific CD8+ T-cell epitopes.; citation_author=I Honeyborne; citation_author=A Prendergast; citation_author=F Pereyra; citation_author=A Leslie; citation_author=H Crawford; citation_journal_title=J Virol; citation_volume=81; citation_number=24; citation_pages=3667-3672; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Recognition of a defined region within p24 Gag by CD8+ T cells during primary human immunodeficiency virus type 1 infection in individuals expressing protective HLA class I alleles.; citation_author=H Streeck; citation_author=M Lichterfeld; citation_author=G Alter; citation_author=A Meier; citation_author=N Teigen; citation_journal_title=J Virol; citation_volume=81; citation_number=25; citation_pages=7725-7731; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Consistent cytotoxic T-lymphocyte targeting of immunodominant regions in human immunodeficiency virus across multiple ethnicities.; citation_author=N Frahm; citation_author=BT Korber; citation_author=CM Adams; citation_author=JJ Szinger; citation_author=R Draenert; citation_journal_title=J Virol; citation_volume=78; citation_number=26; citation_pages=2187-2200; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Generating quantitative models describing the sequence specificity of biological processes with the stabilized matrix method.; citation_author=B Peters; citation_author=A Sette; citation_journal_title=BMC Bioinformatics; citation_volume=6; citation_number=27; citation_pages=132; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Sensitive quantitative predictions of peptide-MHC binding by a ‘Query by Committee’ artificial neural network approach.; citation_author=S Buus; citation_author=SL Lauemoller; citation_author=P Worning; citation_author=C Keşmir; citation_author=T Frimurer; citation_journal_title=Tissue Antigens; citation_volume=62; citation_number=28; citation_pages=378-384; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Reliable prediction of T-cell epitopes using neural networks with novel sequence representations.; citation_author=M Nielsen; citation_author=C Lundegaard; citation_author=P Worning; citation_author=SL Lauemoller; citation_author=K Lamberth; citation_journal_title=Protein Sci; citation_volume=12; citation_number=29; citation_pages=1007-1017; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Generation of tissue-specific and promiscuous HLA ligand databases using DNA microarrays and virtual HLA class II matrices.; citation_author=T Sturniolo; citation_author=E Bono; citation_author=J Ding; citation_author=L Raddrizzani; citation_author=O Tuereci; citation_journal_title=Nat Biotechnol; citation_volume=17; citation_number=30; citation_pages=555-561; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Sequence logos: a new way to display consensus sequences.; citation_author=TD Schneider; citation_author=RM Stephens; citation_journal_title=Nucleic Acids Res; citation_volume=18; citation_number=31; citation_pages=6097-6100; citation_date=1990; " />
      <meta name="citation_reference" content="citation_title=A mathematical theory of communication.; citation_author=CE Shannon; citation_journal_title=Bell System Tech J; citation_volume=27; citation_number=32; citation_pages=379-423; citation_date=1948; " />
      <meta name="citation_reference" content="citation_title=An improved Bonferroni procedure for multiple tests of significance.; citation_author=RJ Simes; citation_journal_title=Biometrika; citation_volume=73; citation_number=33; citation_pages=751-754; citation_date=1986; " />
      <meta name="citation_reference" content="citation_title=A quantitative analysis of the variables affecting the repertoire of T cell specificities recognized after vaccinia virus infection.; citation_author=E Assarsson; citation_author=J Sidney; citation_author=C Oseroff; citation_author=V Pasquetto; citation_author=HH Bui; citation_journal_title=J Immunol; citation_volume=178; citation_number=34; citation_pages=7890-7901; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Prediction of proteasome cleavage motifs by neural networks.; citation_author=C Keşmir; citation_author=AK Nussbaum; citation_author=H Schild; citation_author=V Detours; citation_author=S Brunak; citation_journal_title=Protein Eng; citation_volume=15; citation_number=35; citation_pages=287-296; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Identifying MHC class I epitopes by predicting the TAP transport efficiency of epitope precursors.; citation_author=B Peters; citation_author=S Bulik; citation_author=R Tampe; citation_author=PM Van Endert; citation_author=HG Holzhutter; citation_journal_title=J Immunol; citation_volume=171; citation_number=36; citation_pages=1741-1749; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Timing the ancestor of the HIV-1 pandemic strains.; citation_author=B Korber; citation_author=M Muldoon; citation_author=J Theiler; citation_author=F Gao; citation_author=R Gupta; citation_journal_title=Science; citation_volume=288; citation_number=37; citation_pages=1789-1796; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Nine major HLA class I supertypes account for the vast preponderance of HLA-A and -B polymorphism.; citation_author=A Sette; citation_author=J Sidney; citation_journal_title=Immunogenetics; citation_volume=50; citation_number=38; citation_pages=201-212; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Human immunodeficiency virus type 1 subtype C molecular phylogeny: consensus sequence for an AIDS vaccine design?; citation_author=V Novitsky; citation_author=UR Smith; citation_author=P Gilbert; citation_author=MF McLane; citation_author=P Chigwedere; citation_journal_title=J Virol; citation_volume=76; citation_number=39; citation_pages=5435-5451; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Functional surfaces of the human immunodeficiency virus type 1 capsid protein.; citation_author=UK Von Schwedler; citation_author=KM Stray; citation_author=JE Garrus; citation_author=WI Sundquist; citation_journal_title=J Virol; citation_volume=77; citation_number=40; citation_pages=5439-5450; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Fitness costs limit viral escape from cytotoxic T lymphocytes at a structurally constrained epitope.; citation_author=FW Peyerl; citation_author=HS Bazick; citation_author=MH Newberg; citation_author=DH Barouch; citation_author=J Sodroski; citation_journal_title=J Virol; citation_volume=78; citation_number=41; citation_pages=13901-13910; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=HIV-1 Nef is preferentially recognized by CD8 T cells in primary HIV-1 infection despite a relatively high degree of genetic diversity.; citation_author=M Lichterfeld; citation_author=XG Yu; citation_author=D Cohen; citation_author=MM Addo; citation_author=J Malenfant; citation_journal_title=AIDS; citation_volume=18; citation_number=42; citation_pages=1383-1392; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Compensatory mutation partially restores fitness and delays reversion of escape mutation within the immunodominant HLA-B*5703-restricted Gag epitope in chronic human immunodeficiency virus type 1 infection.; citation_author=H Crawford; citation_author=JG Prado; citation_author=A Leslie; citation_author=S Hue; citation_author=I Honeyborne; citation_journal_title=J Virol; citation_volume=81; citation_number=43; citation_pages=8346-8351; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=The Protein Data Bank.; citation_author=HM Berman; citation_author=J Westbrook; citation_author=Z Feng; citation_author=G Gilliland; citation_author=TN Bhat; citation_journal_title=Nucleic Acids Res; citation_volume=28; citation_number=44; citation_pages=235-242; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Crystal structure of dimeric HIV-1 capsid protein.; citation_author=C Momany; citation_author=LC Kovari; citation_author=AJ Prongay; citation_author=W Keller; citation_author=RK Gitti; citation_journal_title=Nat Struct Biol; citation_volume=3; citation_number=45; citation_pages=763-770; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Efficient particle production by minimal Gag constructs which retain the carboxy-terminal domain of human immunodeficiency virus type 1 capsid-p2 and a late assembly domain.; citation_author=MA Accola; citation_author=B Strack; citation_author=HG Gottlinger; citation_journal_title=J Virol; citation_volume=74; citation_number=46; citation_pages=5395-5402; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Nucleic acid-independent retrovirus assembly can be driven by dimerization.; citation_author=MC Johnson; citation_author=HM Scobie; citation_author=YM Ma; citation_author=VM Vogt; citation_journal_title=J Virol; citation_volume=76; citation_number=47; citation_pages=11177-11185; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Analysis of the assembly function of the human immunodeficiency virus type 1 gag protein nucleocapsid domain.; citation_author=Y Zhang; citation_author=H Qian; citation_author=Z Love; citation_author=E Barklis; citation_journal_title=J Virol; citation_volume=72; citation_number=48; citation_pages=1782-1789; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=HIV-1-specific mucosal CD8+ lymphocyte responses in the cervix of HIV-1-resistant prostitutes in Nairobi.; citation_author=R Kaul; citation_author=FA Plummer; citation_author=J Kimani; citation_author=T Dong; citation_author=P Kiama; citation_journal_title=J Immunol; citation_volume=164; citation_number=49; citation_pages=1602-1611; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Fitness cost of escape mutations in p24 Gag in association with control of human immunodeficiency virus type 1.; citation_author=J Martinez-Picado; citation_author=JG Prado; citation_author=EE Fry; citation_author=K Pfafferott; citation_author=A Leslie; citation_journal_title=J Virol; citation_volume=80; citation_number=50; citation_pages=3617-3623; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Conserved CTL epitopes shared between HIV-infected human long-term survivors and chimpanzees.; citation_author=SS Balla-Jhagjhoorsingh; citation_author=G Koopman; citation_author=P Mooij; citation_author=TG Haaksma; citation_author=VJ Teeuwsen; citation_journal_title=J Immunol; citation_volume=162; citation_number=51; citation_pages=2308-2314; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Structure of the carboxyl-terminal dimerization domain of the HIV-1 capsid protein.; citation_author=TR Gamble; citation_author=S Yoo; citation_author=FF Vajdos; citation_author=UK Von Schwedler; citation_author=DK Worthylake; citation_journal_title=Science; citation_volume=278; citation_number=52; citation_pages=849-853; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Molecular and functional analysis of a conserved CTL epitope in HIV-1 p24 recognized from a long-term nonprogressor: constraints on immune escape associated with targeting a sequence essential for viral replication.; citation_author=R Wagner; citation_author=B Leschonsky; citation_author=E Harrer; citation_author=C Paulus; citation_author=C Weber; citation_journal_title=J Immunol; citation_volume=162; citation_number=53; citation_pages=3727-3734; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=The stoichiometry of Gag protein in HIV-1.; citation_author=JA Briggs; citation_author=MN Simon; citation_author=I Gross; citation_author=HG Krausslich; citation_author=SD Fuller; citation_journal_title=Nat Struct Mol Biol; citation_volume=11; citation_number=54; citation_pages=672-675; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Gag-specific CD8+ T lymphocytes recognize infected cells before AIDS-virus integration and viral protein expression.; citation_author=JB Sacha; citation_author=C Chung; citation_author=EG Rakasz; citation_author=SP Spencer; citation_author=AK Jonas; citation_journal_title=J Immunol; citation_volume=178; citation_number=55; citation_pages=2746-2754; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Effective T-cell responses select human immunodeficiency virus mutants and slow disease progression.; citation_author=AJ Frater; citation_author=H Brown; citation_author=A Oxenius; citation_author=HF Gunthard; citation_author=B Hirschel; citation_journal_title=J Virol; citation_volume=81; citation_number=56; citation_pages=6742-6751; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=HIV-1 gag-specific cytotoxic T lymphocytes defined with recombinant vaccinia virus and synthetic peptides.; citation_author=DF Nixon; citation_author=AR Townsend; citation_author=JG Elvin; citation_author=CR Rizza; citation_author=J Gallwey; citation_journal_title=Nature; citation_volume=336; citation_number=57; citation_pages=484-487; citation_date=1988; " />
      <meta name="citation_reference" content="citation_title=Immune escape precedes breakthrough human immunodeficiency virus type 1 viremia and broadening of the cytotoxic T-lymphocyte response in an HLA-B27-positive long-term-nonprogressing child.; citation_author=ME Feeney; citation_author=Y Tang; citation_author=KA Roosevelt; citation_author=AJ Leslie; citation_author=K McIntosh; citation_journal_title=J Virol; citation_volume=78; citation_number=58; citation_pages=8927-8930; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Definition of supertypes for HLA molecules using clustering of specificity matrices.; citation_author=O Lund; citation_author=M Nielsen; citation_author=C Keşmir; citation_author=AG Petersen; citation_author=C Lundegaard; citation_journal_title=Immunogenetics; citation_volume=55; citation_number=59; citation_pages=797-810; citation_date=2004; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000920" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="HLA Alleles Associated with Slow Progression to AIDS Truly Prefer to Present HIV-1 p24"/>
    <meta name="twitter:description" content="BackgroundThe mechanism behind the association between human leukocyte antigen (HLA) molecules and the rate of HIV-1 disease progression is still poorly understood. Recent data suggest that &ldquo;protective&rdquo; HLA molecules, i.e. those associated with a low HIV-1 viral load and relatively slow disease progression, tend to present epitopes from the Gag capsid protein. Although this suggests that preferential targeting of Gag delays disease progression, the apparent preference for Gag could also be a side-effect of the relatively high immunogenicity of the protein.Methods and FindingsTo separate cause and effect, we predicted HIV-1 epitopes from the whole genome of HIV-1, and found that protective HLA alleles have a true preference for the p24 Gag protein, while non-protective HLA alleles preferentially target HIV-1 Nef. In line with this, we found a significant negative correlation between the predicted affinity of the best-binding p24 epitopes and the relative hazard of HIV-1 disease progression for a large number of HLA molecules. When the epitopes targeted by protective HLA alleles were mapped to the known p24 structure, we found that mutations in these epitopes are likely to disturb the p24 dimer structure, which is expected to severely reduce the fitness of the virus.ConclusionsOur results suggest that the intrinsic preference of different HLA molecules to present p24 peptides explains why some HLA molecules are more protective than others."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0000920.g004"/>

  <meta property="og:title" content="HLA Alleles Associated with Slow Progression to AIDS Truly Prefer to Present HIV-1 p24" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000920" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=57'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=416'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=3924&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0000920">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000920" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000920&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    HLA Alleles Associated with Slow Progression to AIDS Truly Prefer to Present HIV-1 p24
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            José A. M. Borghans, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliations:
                  Theoretical Biology/Bioinformatics, Utrecht University, Utrecht, The Netherlands, 
                  Department of Immunology, University Medical Center Utrecht, Utrecht, The Netherlands
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Anne Mølgaard, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Center for Biological Sequence Analysis, Technical University of Denmark, Lyngby, Denmark
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Rob J. de Boer, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Theoretical Biology/Bioinformatics, Utrecht University, Utrecht, The Netherlands
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Can Keşmir
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:c.kesmir@uu.nl">c.kesmir@uu.nl</a></p>

                <p>Affiliations:
                  Theoretical Biology/Bioinformatics, Utrecht University, Utrecht, The Netherlands, 
                  Center for Biological Sequence Analysis, Technical University of Denmark, Lyngby, Denmark, 
                  Academic Biomedical Centre, Utrecht University, Utrecht, The Netherlands
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: September 19, 2007</li>
    <li>DOI: 10.1371/journal.pone.0000920</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0000920" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0000920" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000920" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0000920" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0000920-g001" data-doi="info:doi/10.1371/journal.pone.0000920" data-uri="info:doi/10.1371/journal.pone.0000920.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000920.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000920-g002" data-doi="info:doi/10.1371/journal.pone.0000920" data-uri="info:doi/10.1371/journal.pone.0000920.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000920.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000920-g003" data-doi="info:doi/10.1371/journal.pone.0000920" data-uri="info:doi/10.1371/journal.pone.0000920.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000920.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000920-t001" data-doi="info:doi/10.1371/journal.pone.0000920" data-uri="info:doi/10.1371/journal.pone.0000920.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000920.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000920-g004" data-doi="info:doi/10.1371/journal.pone.0000920" data-uri="info:doi/10.1371/journal.pone.0000920.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000920.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000920">Reader Comments (1)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0000920" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2>
<h3>Background</h3>
<a id="article1.front1.article-meta1.abstract1.sec1.p1" name="article1.front1.article-meta1.abstract1.sec1.p1"></a><p>The mechanism behind the association between human leukocyte antigen (HLA) molecules and the rate of HIV-1 disease progression is still poorly understood. Recent data suggest that “protective” HLA molecules, i.e. those associated with a low HIV-1 viral load and relatively slow disease progression, tend to present epitopes from the Gag capsid protein. Although this suggests that preferential targeting of Gag delays disease progression, the apparent preference for Gag could also be a side-effect of the relatively high immunogenicity of the protein.</p>


<h3>Methods and Findings</h3>
<a id="article1.front1.article-meta1.abstract1.sec2.p1" name="article1.front1.article-meta1.abstract1.sec2.p1"></a><p>To separate cause and effect, we predicted HIV-1 epitopes from the whole genome of HIV-1, and found that protective HLA alleles have a true preference for the p24 Gag protein, while non-protective HLA alleles preferentially target HIV-1 Nef. In line with this, we found a significant negative correlation between the predicted affinity of the best-binding p24 epitopes and the relative hazard of HIV-1 disease progression for a large number of HLA molecules. When the epitopes targeted by protective HLA alleles were mapped to the known p24 structure, we found that mutations in these epitopes are likely to disturb the p24 dimer structure, which is expected to severely reduce the fitness of the virus.</p>


<h3>Conclusions</h3>
<a id="article1.front1.article-meta1.abstract1.sec3.p1" name="article1.front1.article-meta1.abstract1.sec3.p1"></a><p>Our results suggest that the intrinsic preference of different HLA molecules to present p24 peptides explains why some HLA molecules are more protective than others.</p>

</div>


<div class="articleinfo"><p><strong>Citation: </strong>Borghans JAM, Mølgaard A, de Boer RJ, Keşmir C (2007) HLA Alleles Associated with Slow Progression to AIDS Truly Prefer to Present HIV-1 p24. PLoS ONE 2(9):
          e920.
            doi:10.1371/journal.pone.0000920</p><p><strong>Academic Editor: </strong>Marcus Altfeld, Massachusetts General Hospital, United States of America</p><p><strong>Received:</strong> July 9, 2007; <strong>Accepted:</strong> August 16, 2007; <strong>Published:</strong> September 19, 2007</p><p><strong>Copyright:</strong> © 2007 Borghans et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>This work was financially supported by the Netherlands Organization for Scientific Research (NWO, grants 050.50.202, 916.36.003 and 016.048.603), and by AIDS Fonds Netherlands (grant 7011).</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>There is increasing evidence that human leukocyte antigen (HLA) molecules influence the rate of disease progression after HIV-1 infection (reviewed in <a href="#pone.0000920-Carrington1">[1]</a>). Cytotoxic T lymphocyte (CTL) responses against HIV-1 assert a strong evolutionary pressure on the virus, causing selection of HIV-1 variants that successfully escape recognition by CTL <a href="#pone.0000920-Moore1">[2]</a>–<a href="#pone.0000920-Friedrich1">[4]</a>. Due to the large polymorphism of HLA molecules, different individuals are able to respond to specific immunodominant HIV-1 epitopes depending on their HLA background. Since disease progression is influenced by an individual's HLA background, some CTL responses apparently control the virus better than others.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>HLA-B57 molecules have been found to have the strongest association with immune control of HIV-1 (see e.g. <a href="#pone.0000920-Klein1">[5]</a>, <a href="#pone.0000920-Gillespie1">[6]</a>). Among elite suppressors of HIV, i.e. HIV-infected individuals with normal CD4<sup>+</sup> T cell counts and viral loads below the detection level without therapy, HLA-B57 molecules are significantly overrepresented <a href="#pone.0000920-Bailey1">[7]</a>. HLA-B57 restricted CTL responses tend to be more responsive and immunodominant than other CTL responses <a href="#pone.0000920-Jansen1">[8]</a>–<a href="#pone.0000920-Altfeld2">[10]</a>. HLA-B58 and HLA-B63 have binding motifs that are very similar to HLA-B57 and are also associated with good immune control of HIV-1 <a href="#pone.0000920-Carrington1">[1]</a>, <a href="#pone.0000920-Frahm1">[11]</a>. The protective effect of HLA-B27, another HLA type associated with long term non-progression <a href="#pone.0000920-Carrington1">[1]</a>, has been proposed to be due to the very conserved Gag epitope it presents <a href="#pone.0000920-Goulder1">[12]</a>. HLA-B35 and HLA-B53, on the other hand, tend to be associated with relatively rapid progression to AIDS <a href="#pone.0000920-Carrington1">[1]</a>, although different associations have been found for different HLA-subtypes: while HLA-B3503 is associated with rapid progression to AIDS, the rate of disease progression of individuals with HLA-B3501 (which has only a slightly different peptide-binding motif) does not differ from the population average <a href="#pone.0000920-Gao1">[13]</a>.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>The frequency of HLA molecules in the human population has been proposed to play a role in the association between HLA molecules and the rate of disease progression. Firstly, individuals with rare HLA molecules are more likely to be heterozygous at the HLA loci, and are thereby expected to induce an immune response against a larger diversity of peptides than homozygous individuals. Indeed, HLA-heterozygosity is associated with relatively slow disease progression in HIV-1 infection <a href="#pone.0000920-Gao1">[13]</a>, <a href="#pone.0000920-Carrington2">[14]</a>. Secondly, since HIV-1 evolution occurs in the context of the HLA background of the human population, HIV-1 may be better adapted to common than to rare HLA molecules <a href="#pone.0000920-Trachtenberg1">[15]</a>–<a href="#pone.0000920-Frahm2">[17]</a>. However, a large study establishing the relative hazard of HIV-1 disease progression for different HLA molecules found no correlation between the relative hazard and the population frequency of HLA molecules <a href="#pone.0000920-Gao1">[13]</a>.</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p>Alternatively, qualitative differences between HLA molecules in terms of the specific HIV peptides that they present may explain the association between HLA molecules and the rate of HIV disease progression. It has recently become clear that individuals with slow HIV-1 disease progression tend to make broad and strong CTL responses against HIV-1 Gag, while individuals with rapid disease progression and high HIV-1 viral loads make strong CTL responses to Env and accessory/regulatory proteins <a href="#pone.0000920-Bailey1">[7]</a>, <a href="#pone.0000920-Buseyne1">[18]</a>–<a href="#pone.0000920-Streeck1">[25]</a>. It is tempting to conclude from these data that certain HLA alleles provide better protection against HIV-1 disease progression because of their tendency to induce CTL responses to HIV-1 Gag. The observed association between the presence of CTL to Gag and slow disease progression could, however, also be due to the relatively high immunogenicity of HIV-Gag <a href="#pone.0000920-Frahm3">[26]</a>, which would give a bias towards the detection of Gag-specific responses in individuals with low viral loads. In the latter case, the observed association between HLA molecules with a low relative hazard and CTL responses against Gag would not be the cause but the consequence of low viral loads.</p>
<a id="article1.body1.sec1.p5" name="article1.body1.sec1.p5"></a><p>To separate cause and effect, we studied the binding preferences of different HLA alleles for the different HIV proteins using HLA-peptide prediction tools. The use of prediction tools, instead of clinical data, is crucial in this analysis, because prediction tools will reveal unbiased preferences of HLA alleles for different HIV-1 proteins, whereas HLA-restricted HIV-1 epitopes reported in the HIV-1 database may be biased, because some HIV-1 proteins are studied more extensively than others. We found that HLA molecules associated with slow HIV disease progression have an intrinsic preference to present epitopes from the p24 Gag capsid protein of HIV-1. Analysis of the structure of the p24 protein pointed out that CTL escape mutations restricted by protective HLA molecules are likely to disturb the ability of p24 to form a dimer and thereby to result in a drastic viral fitness loss. Taken together, our analyses suggest that differences between HLA molecules in the tendency to present peptides from p24 underlie the association between HLA molecules and the rate of HIV-1 disease progression.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Methods"></a><h3>Methods</h3>
<h4>Data</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>The consensus and ancestral sequences for different HIV proteins, as well as 59 individual world-wide HIV-1 clade B p24 sequences with known year of sampling were downloaded from the Los Alamos HIV database at <a href="http://www.hiv.lanl.gov/">www.hiv.lanl.gov/</a> (consensus August 2004).</p>


<h4>HLA-peptide binding predictions</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>We used two different algorithms to predict the binding affinities of HIV-1 peptides (of length 9 and 10) to HLA molecules: the stabilized matrix method (SMM), available at <a href="http://www.immuneepitope.org">www.immuneepitope.org</a> <a href="#pone.0000920-Peters1">[27]</a>, and an artificial neural network (ANN), NetMHC3.0 available at <a href="http://www.cbs.dtu.dk/services/NetMHC-3.0/">www.cbs.dtu.dk/services/NetMHC-3.0/</a> <a href="#pone.0000920-Buus1">[28]</a>, <a href="#pone.0000920-Nielsen1">[29]</a>. For HLA-B3503 we used the only method that is currently available, i.e. the general matrix method at <a href="http://www.cbs.dtu.dk/services/NetMHC-3.0/">www.cbs.dtu.dk/services/NetMHC-3.0/</a>. Since prediction scores from these algorithms for different HLA molecules cannot be compared directly, we either used the ranks of these peptides among all HIV-1 peptides, or applied an HLA-binding affinity normalization as previously proposed <a href="#pone.0000920-Sturniolo1">[30]</a>. Briefly, the HLA-binding affinities of HIV-1 peptides were divided by an HLA-specific threshold, which was based on the top 1% binding affinities of a large set of non-HIV peptides.</p>


<h4>Sequence logos</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>Sequence logos <a href="#pone.0000920-Schneider1">[31]</a> were made to visualize i) to what extent a position in a sequence is conserved (given by the height of a bar, i.e. the information content) and ii) which amino acids are most frequently found at a particular position (the height of each amino acid in the logo is proportional to the frequency of occurrence at that position). Sequence logos were generated using the Shannon information content <a href="#pone.0000920-Shannon1">[32]</a>. The maximum information content is log <sub>2</sub> 20 = 4.3, which is obtained if the same amino acid is always observed at a particular position. The minimum Shannon information is zero, which is obtained if all amino acids occur at the same frequency at a position.</p>

</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Results"></a><h3>Results</h3>
<h4>Protective HLA alleles preferentially target the p24 protein</h4>
<a id="article1.body1.sec3.sec1.p1" name="article1.body1.sec3.sec1.p1"></a><p>To investigate whether the recently reported association between HLA molecules conferring slow HIV-1 disease progression and CTL responses against Gag <a href="#pone.0000920-Bailey1">[7]</a>, <a href="#pone.0000920-Buseyne1">[18]</a>–<a href="#pone.0000920-Honeyborne1">[24]</a> is due to intrinsic properties of HLA molecules or is a side-effect of the high immunogenicity of Gag, we analyzed HLA-peptide binding predictions for a large number of different HLA molecules. We studied the HLA-binding affinities of HIV-1 peptides for HLA-B5701, B5801, and B2705, three HLA alleles with a clearly low relative hazard (RH) of HIV disease progression <a href="#pone.0000920-Carrington1">[1]</a> (<a href="#pone-0000920-g001">Figure 1</a>, Low RH, open and shaded symbols), and for HLA-B3503 and B5301, which are both associated with relatively rapid HIV-1 disease progression <a href="#pone.0000920-Carrington1">[1]</a> (<a href="#pone-0000920-g001">Figure 1</a>, High RH, solid symbols). Predictions were made for all peptides of length nine and ten from the consensus sequence of HIV-1 clade B, using the stabilized matrix method <a href="#pone.0000920-Peters1">[27]</a>, and an artificial neural network <a href="#pone.0000920-Buus1">[28]</a>, <a href="#pone.0000920-Nielsen1">[29]</a> (see <a href="#s2">Methods</a>). The HIV-1 epitopes were subsequently ranked by their predicted HLA-binding affinity within all HIV proteins. We plotted the ranks of the three best-binding peptides from each HIV-1 protein among all other HIV epitopes, because Kiepiela <em>et al.</em> <a href="#pone.0000920-Kiepiela1">[23]</a> reported that the presence of CTL responses against at least two Gag-epitopes is associated with a low HIV-1 viral load.</p>
<div class="figure" id="pone-0000920-g001"><div class="img"><a name="pone-0000920-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000920.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000920" data-uri="info:doi/10.1371/journal.pone.0000920.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000920.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000920.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000920.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000920.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000920.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000920.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000920.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000920.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000920.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Predicted ranks of epitopes from different HIV-1 proteins.</span></strong></p><a id="article1.body1.sec3.sec1.fig1.caption1.p1" name="article1.body1.sec3.sec1.fig1.caption1.p1"></a><p>HLA-binding epitopes were predicted for HLA alleles with a low RH (HLA-B5701, HLA-B5801 and HLA-B2705) and HLA alleles with a high RH (HLA-B3503 and B5301), and their rank among all other HIV-1 epitopes was plotted. Low ranks correspond to good binding peptides. The analyses were limited to the three best-binding epitopes from each HIV-1 protein. HLA alleles with a low RH were found to have a significantly higher preference for peptides from p24 than HLA alleles with a high RH (<em>p</em> = 0.01, Mann-Whitney test), while HLA alleles with a high RH had a significantly higher preference for Nef (<em>p</em> = 0.02). A significant difference between low and high RH HLA molecules was also observed for peptides from Vpu (<em>p</em> = 0.01), p17 (<em>p</em> = 0.02), Vif (<em>p</em> = 0.02), and Rev (<em>p</em> = 0.01), but the median ranks of the best-binding peptides from these proteins are so high that these differences are probably not physiologically important. Epitopes for HLA-B2705 are depicted as shaded circles. All predictions were based on a neural-network based predictor (NetMHC); the use of matrix methods gave similar results (results not shown). The number of amino acids in each of the HIV proteins is: p24 231, Nef 206, Env 856, Pol 947, Vpr 96, Vpu 82, Tat 101, p17 132, Vif 192, and Rev 116 amino acids.</p>
<span>doi:10.1371/journal.pone.0000920.g001</span></div><a id="article1.body1.sec3.sec1.p2" name="article1.body1.sec3.sec1.p2"></a><p><a href="#pone-0000920-g001">Figure 1</a> shows that the three HLA alleles with a low RH have a significantly stronger preference for peptides from p24 than the two HLA alleles with a high RH (Mann-Whitney, <em>p</em> = 0.01), while the latter have a stronger preference for peptides from Nef (Mann-Whitney, <em>p</em> = 0.02). These differences remain significant when correcting for multiple tests of significance using the improved Bonferroni procedure <a href="#pone.0000920-Simes1">[33]</a>. It is also evident from <a href="#pone-0000920-g001">Figure 1</a> that, despite the relatively small size of the p24 protein (7.5% of the HIV-1 proteome), many of the best-binding peptides for HLA alleles with a low RH come from p24. For HLA molecules with a low RH, the average rank of epitopes from p24 is comparable to the epitopes from the much larger proteins Env and Pol. The other relatively small HIV-1 proteins (p17, Vpr, Vpu, Tat, Vif and Rev) contain hardly any of the high ranking epitopes. Of note, both HLA-B5701 and B5801 appeared to have three very good-binding p24 epitopes, all ranking within the first ten among all HIV peptides. In contrast, HLA-B2705 was found to have only a single good-binding p24 epitope (see <a href="#pone-0000920-g001">Figure 1</a>, shaded symbols).</p>
<a id="article1.body1.sec3.sec1.p3" name="article1.body1.sec3.sec1.p3"></a><p>For a large number of HLA molecules, we plotted the predicted affinity score of the three best-binding p24 epitopes as a function of the relative hazard of the corresponding HLA type as defined by Gao <em>et al.</em> <a href="#pone.0000920-Gao1">[13]</a> (see <a href="#pone-0000920-g002">Figure 2</a>). It was previously established that these relative hazards do not significantly correlate with the frequencies of the HLA molecules in the human population (Kendall's <em>τ</em> = −0.08, <em>p</em> = 0.52, <a href="#pone.0000920-Gao1">[13]</a>). In contrast, we found a clear correlation between the affinity score of the best-binding p24 epitopes from 59 different individual HIV-1 clade B sequences and the RH of HIV-1 disease progression for 27 different HLA types (<a href="#pone-0000920-g002">Figure 2A</a>). Since the data from different HIV-1 clade B sequences are not independent, we used the consensus HIV-1 clade B sequence to test the significance of this correlation. This revealed a significant correlation between the binding affinity of epitopes from HIV-1 clade B p24 and the risk of HIV-1 disease progression (Kendall's <em>τ</em> = −0.17, <em>p</em> = 0.029, see <a href="#pone-0000920-g002">Figure 2B</a>). The best-binding HIV-1 clade B p24 peptides restricted by the protective HLA types B58 and B57 turned out to have significantly higher predicted affinity scores (purple and blue circles in <a href="#pone-0000920-g002">Figure 2B</a>) than those restricted by all other HLA alleles (Mann-Whitney, <em>p</em> = 0.0009). For none of the other HIV-1 clade B proteins, a significant correlation between the affinity score of the best-binding peptides and the relative hazard of disease progression of the different HLA types was found (not shown). Obviously, the HLA binding affinity does not have to correlate directly with immunogenicity or antiviral CTL response. However, as pointed out recently <a href="#pone.0000920-Assarsson1">[34]</a>, HLA-binding is the most restrictive step in determining immunodominance. Taken together, these results suggest that the experimentally observed association between protective HLA alleles and responses against Gag is caused by an intrinsic preference of these HLA alleles for peptides from p24, and is not merely a side-effect of the high immunogenicity of Gag. The antigen processing efficacy (here measured as a combination of proteasome cleavage predictions <a href="#pone.0000920-Kemir1">[35]</a> and predicted TAP binding affinity <a href="#pone.0000920-Peters2">[36]</a>) does not influence this intrinsic preference (results not shown).</p>
<div class="figure" id="pone-0000920-g002"><div class="img"><a name="pone-0000920-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000920.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000920" data-uri="info:doi/10.1371/journal.pone.0000920.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000920.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000920.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000920.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000920.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000920.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000920.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000920.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000920.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000920.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Relationship between relative hazard and predicted HLA-binding affinity for p24 epitopes.</span></strong></p><a id="article1.body1.sec3.sec1.fig2.caption1.p1" name="article1.body1.sec3.sec1.fig2.caption1.p1"></a><p>The relationship between the relative hazard of HIV-1 disease progression and the predicted affinity of p24 peptides plotted for different HLA alleles. The graphs show the normalized predicted affinity of the three best-binding p24 epitopes for each HLA allele. Data were based on (A) the Caucasian RH of 27 different HLA alleles <a href="#pone.0000920-Gao1">[13]</a> and 59 individual HIV-1 clade B sequences downloaded from the Los Alamos database, (B) the Caucasian RH of 27 different HLA alleles <a href="#pone.0000920-Gao1">[13]</a> and the consensus HIV-1 clade B sequence, and (C) the African RH of 24 different HLA alleles for which non-trivial RH have been determined <a href="#pone.0000920-Gao1">[13]</a> and the consensus HIV-1 clade C sequence. The RH values represent the relative hazard of the 2-digit HLA serotype (except for HLA-B3503 and B3501 for which separate RH are known <a href="#pone.0000920-Gao1">[13]</a>), while the predicted affinities were based on prediction tools for the 4-digit HLA allele that dominates the 2-digit HLA serotype (except for HLA-A0205 and HLA-B5802, which dominate the HLA serotypes HLA-A02 and HLA-B58 in Africans, but for which no high quality prediction tools are available; when they were omitted from the analysis in panel (C), the correlation remained significant (<em>p</em> = 0.014)). Because binding motifs of HLA molecules are not always independent, we repeated the analysis by randomly selecting only one peptide, when a peptide binds more than one allele. In all cases the analysis for HIV-1 clade B and clade C remained significant (<em>p</em>&lt;0.02). All HLA-peptide binding predictions were based on matrix methods; very similar results were obtained when using NetMHC, or when we confined our analyses to the best-binding p24 peptide for each HLA type.</p>
<span>doi:10.1371/journal.pone.0000920.g002</span></div><a id="article1.body1.sec3.sec1.p4" name="article1.body1.sec3.sec1.p4"></a><p>If HIV-1 were adapting to the most common HLA alleles in the human population, one could argue that the current HIV-1 consensus sequence in the Caucasian population should contain more CTL escape mutations for common HLA types, such as HLA-B35, than for rare HLA types, such as HLA-B57, B58 and B27. We therefore repeated our analysis using the predicted ancestral sequence for HIV-1 subtype B and the HXB2 strain dating from 1983 (see <a href="#s2">Methods</a> and <a href="#pone.0000920-Korber1">[37]</a>), which should contain no or only few CTL escape mutations. Both the HXB2 and the ancestral sequence differed by only 2 amino acids from the consensus HIV-1 clade B sequence; as a consequence there was hardly any difference between the predicted p24 epitopes from the consensus and the ancestral HIV-1 clade B sequences, and HXB2 for the various HLA types (results not shown). Additionally, the binding affinities of p24 epitopes from the consensus sequence of HIV-1 clade C, the major HIV-1 subtype in large parts of Africa, correlated significantly with the relative hazard of these HLA alleles in the Afro-American population <a href="#pone.0000920-Gao1">[13]</a> (<a href="#pone-0000920-g002">Figure 2C</a>). These observations demonstrate that our results are not due to frequency-dependent adaptation of HIV-1 to the most common HLA alleles in the human population, and thereby confirm that HLA types with a low RH have an intrinsic preference for peptides from p24.</p>


<h4>Preferential binding of p24 is due to sequence patterns</h4>
<a id="article1.body1.sec3.sec2.p1" name="article1.body1.sec3.sec2.p1"></a><p>The differential targeting of p24 and Nef by protective and non-protective HLA alleles could be due to differences in the amino acid distributions or in the sequence patterns between the two proteins. When the amino acid residues occurring in p24 and Nef were completely shuffled (without changing the amino acid frequencies of the two proteins), both proteins were no longer preferentially targeted by any of the HLA alleles (results not shown). This suggests that p24 and Nef carry distinct sequence patterns that are preferred by HLA alleles with a low or high RH, respectively. To study the sequence patterns preferred by protective and non-protective HLA alleles, we devised sequence logos for both groups of HLA alleles, based on a large set of experimentally verified good-binding peptides (data available at <a href="http://www.immuneepitope.org">www.immuneepitope.org</a>). Since HLA-B57 and B58, and HLA-B35 and B53 are known to have similar binding motifs <a href="#pone.0000920-Sette1">[38]</a> we generated combined binding logos for the HLA alleles with low and high RH (see <a href="#pone-0000920-g003">Figure 3</a>). The sequence logos point out that the two groups of alleles have non-overlapping preferences, and target very different HIV-1 peptides. Taken together, the HLA-peptide binding data suggest that different sequence patterns occurring in HIV-1 p24 and Nef are causing the preferential binding of p24 and Nef peptides by HLA alleles associated with slow and rapid disease progression, respectively.</p>
<div class="figure" id="pone-0000920-g003"><div class="img"><a name="pone-0000920-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000920.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000920" data-uri="info:doi/10.1371/journal.pone.0000920.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000920.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000920.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000920.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000920.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000920.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000920.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000920.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000920.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000920.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Binding motifs of HLA alleles with a low or high RH.</span></strong></p><a id="article1.body1.sec3.sec2.fig1.caption1.p1" name="article1.body1.sec3.sec2.fig1.caption1.p1"></a><p>The sequence logos displaying the preferences of HLA alleles with a low (A) or high (B) RH of HIV disease progression. Experimentally verified good-binding peptides for HLA-B5701 and B5801 were used to make the low RH logo (A), while the good binders of HLA-B3501 and B5301 were used for the high RH logo (B, data from <a href="http://www.immuneepitope.org">www.immuneepitope.org</a>). These sequence logos visualize i) to what extent a position in a sequence is conserved (given by the height of a bar, i.e. the information content) and ii) which amino acids are most frequently found at a particular position (the height of each amino acid in the logo is proportional to the frequency of occurrence at that position). The information content may vary between log <sub>2</sub> 20 = 4.3, which is obtained if the same amino acid is always observed at a particular position, and zero, which is obtained if all amino acids occur at the same frequency at a position. Amino acids are colour coded according to their physicochemical characteristics. Neutral and polar, green; basic, blue; acidic, red; neutral and hydrophobic, black.</p>
<span>doi:10.1371/journal.pone.0000920.g003</span></div>

<h4>Predicted associations with slow progression</h4>
<a id="article1.body1.sec3.sec3.p1" name="article1.body1.sec3.sec3.p1"></a><p>Based on the suggestion that CTL responses directed against p24 slow down disease progression, we investigated which other HLA molecules and CTL epitopes are likely to be associated with slow disease progression, by predicting the p24 epitopes (based on consensus HIV-1 clade B) for 19 different (4-digit) HLA-A and 16 HLA-B alleles. The p24 peptides that are predicted to be among the 10 best-binding HIV peptides for each HLA molecule, which are all expected to be very good HLA binders, are summarized in <a href="#pone-0000920-t001">Table 1</a>. None of the HLA alleles with a high RH had a p24 epitope with a rank lower than 20. One forth of all epitopes in <a href="#pone-0000920-t001">Table 1</a> are presented by HLA alleles with a binding motif very similar to HLA-B58 (belonging to the B58 supertype <a href="#pone.0000920-Sette1">[38]</a>). Interestingly, none of the good-binding p24 peptides came from HLA-B5802, while 4 good-binding p24 peptides came from HLA-B5801, which is fully in line with the recent observation that HLA-B5801 is associated with slow disease progression, while B-5802 is not <a href="#pone.0000920-Kiepiela1">[23]</a>. We predict HLA-A6901 to be a protective HLA allele, having three very good, possibly immunodominant, p24 epitopes (see <a href="#pone-0000920-t001">Table 1</a>). This relatively rare HLA allele is not widely reported to be associated with slow HIV disease progression, even though HLA-A6901 has a RH as low as 0.47 <a href="#pone.0000920-Gao1">[13]</a>, which is even lower than the RH of HLA-B57. Some of the other alleles in <a href="#pone-0000920-t001">Table 1</a>, including A0301, A2402, A2403, B0801 and B4501, have been reported to have a low RH among African-Americans <a href="#pone.0000920-Gao1">[13]</a>. In principle, all 18 HLA alleles and all CTL epitopes listed in <a href="#pone-0000920-t001">Table 1</a> are predicted to be associated with slow disease progression. Thirty percent of these CTL epitopes remain to be experimentally confirmed.</p>
<div class="figure" id="pone-0000920-t001"><div class="img"><a name="pone-0000920-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000920.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000920" data-uri="info:doi/10.1371/journal.pone.0000920.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000920.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000920.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000920.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000920.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000920.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000920.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000920.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000920.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000920.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>Predicted p24 epitopes with a high ranking<sup>a</sup> HLA-binding affinity score.</span></strong></p><span>doi:10.1371/journal.pone.0000920.t001</span></div>

<h4>Why is preferential targeting of p24 protective?</h4>
<a id="article1.body1.sec3.sec4.p1" name="article1.body1.sec3.sec4.p1"></a><p>Why would preferential targeting of the p24 protein provide better protection against HIV disease progression than immune responses to other HIV-1 proteins? A likely contributor is the fact that the p24 capsid protein is one of the most functionally and structurally constrained proteins of HIV-1 <a href="#pone.0000920-Leslie1">[3]</a>, <a href="#pone.0000920-Frahm3">[26]</a>, <a href="#pone.0000920-Novitsky2">[39]</a>. It has been shown that point mutations in the capsid surface markedly reduce viral fitness <a href="#pone.0000920-Leslie1">[3]</a>, <a href="#pone.0000920-VonSchwedler1">[40]</a>, <a href="#pone.0000920-Peyerl1">[41]</a>. In contrast to such constraints on p24, the Nef protein is known to be polymorphic <a href="#pone.0000920-Frahm3">[26]</a>. During acute infection, immune responses to Nef are typically replaced by responses to more conserved regions of HIV-1 <a href="#pone.0000920-Lichterfeld1">[42]</a>. The level of protection conferred by different HLA molecules may correlate with the loss of viral fitness resulting from CTL escape; a large viral fitness cost may either lead to less frequent CTL escape or to viral attenuation upon CTL escape.</p>
<a id="article1.body1.sec3.sec4.p2" name="article1.body1.sec3.sec4.p2"></a><p>The epitopes targeted by protective HLA alleles indeed lie in relatively constrained regions of the p24 protein. The epitope TSTLQEQIGW (TW10, p24 positions 108 to 117) was found to trigger an immunodominant CTL response in individuals expressing HLA-B57 or B58 <a href="#pone.0000920-Altfeld1">[9]</a>. Mutations in this epitope allow for CTL escape, but are associated with a large viral fitness cost. For example, the Thr→Asn mutation at p24–110 causes a 10-fold reduction in the replication rate of the virus <a href="#pone.0000920-Leslie1">[3]</a>. Similarly, Ala→Glu escape in another HLA-B57 immunodominant epitope, KAFSPEVIPMF (KF11, p24 positions 30 to 40), severely reduces the viral replicative capacity <a href="#pone.0000920-Crawford1">[43]</a>. In the crystal structure of p24 <a href="#pone.0000920-Berman1">[44]</a>, the N-terminal domain can be seen to form a homo-dimer <a href="#pone.0000920-Momany1">[45]</a>, which is known to be important in Gag assembly <a href="#pone.0000920-Accola1">[46]</a>–<a href="#pone.0000920-Zhang1">[48]</a>. The two monomers have contact at three extended regions (see <a href="#pone-0000920-g004">Figure 4A</a>), two of which have been shown to contain CTL epitopes that induce protective responses in HLA-B57 or B58-positive individuals (i.e. TW10 and ISPRTLNAW, IW9, p24 positions 15-23 <a href="#pone.0000920-Klein1">[5]</a>). The third region at the dimer interface (QDLNMMLNIVGG, p24 positions 50-61) contains an epitope restricted by HLA-B14 (DLNMMLNIV, DV9) <a href="#pone.0000920-Kaul1">[49]</a>, which is also associated with a low relative hazard of HIV disease progression (RH = 0.7 <a href="#pone.0000920-Gao1">[13]</a>). Recently, Martinez-Picado <em>et al.</em> <a href="#pone.0000920-MartinezPicado1">[50]</a> proposed that the reduced fitness of the T110N mutant is due to the loss of the hydrogen bond between the side chain of the hydroxyl group of T110 and the backbone amide of E113, which would destabilize helix 6 of the N-terminal domain of p24. Analysis of the dimeric structure of p24 shows that T110 also has a very central position at the protein-protein interface, being one of three residues that are in contact with their corresponding residue in the other monomer (see <a href="#pone-0000920-g004">Figure 4B</a>). If the threonine residue is replaced by an asparagine, the symmetric hydrophobic contact is replaced by a polar contact, which is expected to disturb the dimeric interface, and thereby to destabilize the HIV capsid. In combination with the intramolecular destabilizing effect on helix 6, this destabilization of the HIV capsid may explain why CTL escape mutations at these sites are associated with a large viral fitness cost.</p>
<div class="figure" id="pone-0000920-g004"><div class="img"><a name="pone-0000920-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000920.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000920" data-uri="info:doi/10.1371/journal.pone.0000920.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000920.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000920.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000920.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000920.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000920.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000920.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000920.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000920.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000920.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>Crystal structure of a p24 dimer.</span></strong></p><a id="article1.body1.sec3.sec4.fig1.caption1.p1" name="article1.body1.sec3.sec4.fig1.caption1.p1"></a><p>Entry 1AFV from the protein data bank is shown (PDB, <a href="http://www.rcsb.org/pdb">www.rcsb.org/pdb</a>) <a href="#pone.0000920-Berman1">[44]</a>, <a href="#pone.0000920-Momany1">[45]</a>. (A) One monomer is shown in combined cartoon and surface representation to show the extent of the surface exposed part of the epitopes. For clarity, the other monomer is shown only in cartoon representation. The epitopes located in the dimer interface are highlighted in red (IW9), yellow (TW10) and green (DV9), and are also shown in stick representation. (B) A close-up of the dimer interface. The two monomers are shown in dark blue and cyan, and the TW10 epitopes are shown in green and magenta stick representations. The hydrophobic contact between T110 from one monomer to T110 in the other monomer is indicated by a dashed line.</p>
<span>doi:10.1371/journal.pone.0000920.g004</span></div>
</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec4.p1" name="article1.body1.sec4.p1"></a><p>Our results demonstrate that the association between HLA molecules conferring slow HIV-1 disease progression and CTL responses to HIV-1 Gag is due to an intrinsic preference of these HLA alleles for peptides from the p24 protein, and not merely a side-effect of the high immunogenicity of the Gag protein. Several recent studies have also established a link between low HIV viral loads and CTL responses against Gag or p24 <a href="#pone.0000920-Bailey1">[7]</a>, <a href="#pone.0000920-Buseyne1">[18]</a>–<a href="#pone.0000920-Honeyborne1">[24]</a>. Bailey <em>et al.</em> <a href="#pone.0000920-Bailey1">[7]</a> reported that CTL responses in elite suppressors of HIV infection focus on HLA-B57 restricted Gag epitopes. Moreover, a significant negative correlation between the magnitude of the CD8 T cell response and HIV-1 viral load was only found for responses against p24 and not for responses against other HIV-1 proteins <a href="#pone.0000920-Novitsky1">[20]</a>, <a href="#pone.0000920-Zuniga1">[22]</a>, <a href="#pone.0000920-Kiepiela1">[23]</a>. When we performed a partial correlation on data published by Frahm <em>et al.</em> <a href="#pone.0000920-Frahm3">[26]</a>, who determined CTL responses to a large panel of HIV-1 peptides spanning all HIV-1 proteins in 150 infected individuals, we found that the number of CTL responses directed against p24 correlated positively with the CD4<sup>+</sup> T cell count (<em>p</em> = 0.04) and negatively with viral load (<em>p</em> = 0.01), while the number of CTL responses against Nef correlated <em>positively</em> with viral load (<em>p</em> = 0.03) and not with the CD4<sup>+</sup> T cell count (<em>p</em> = 0.96) (unpublished results). Interestingly, the main HIV-1 specific CTL responses in chimpanzees, which have a low viral load and do not develop AIDS, have been found to be directed against the very same peptides from p24 that are targeted by human individuals with HLA-B57 or B27, who tend to be long-term non-progressors <a href="#pone.0000920-Carrington1">[1]</a>, <a href="#pone.0000920-BallaJhagjhoorsingh1">[51]</a>. All these data suggest that preferential targeting of p24 delays disease progression. However, previous studies could never rule out the possibility that the apparent preference for p24 was a side-effect of the high immunogenicity of p24, causing immune responses to p24 to be better maintained than other responses in individuals with low viral load. Our analyses based on HLA-peptide binding predictions conclusively show that HLA alleles associated with slow HIV disease progression have an intrinsic preference for peptides from the HIV p24 protein. Our results thereby suggest that the class I restricted CTL immune response, particularly against p24, plays a key role in controlling HIV-1 infection.</p>
<a id="article1.body1.sec4.p2" name="article1.body1.sec4.p2"></a><p>Why then is preferential targeting of p24 beneficial? The first reason that comes to mind is that p24 is one of the most functionally and structurally constrained proteins of HIV. P24 contains a stretch of 20 amino acids which is conserved across retroviruses, and is essential for viral assembly, maturation, and infectivity <a href="#pone.0000920-Gamble1">[52]</a>. This region contains a B14 epitope (a low RH allele), and Wagner <em>et al.</em> <a href="#pone.0000920-Wagner1">[53]</a> found that all mutations abrogating the CTL response to this epitope drastically reduced the replication capacity of the virus. In line with this, we found several HIV epitopes that are presented by protective HLA molecules to be located at the dimer interface of p24, which is expected to be sensitive to mutations. The peptide binding motifs of HLA alleles with low and high relative hazards (<a href="#pone-0000920-g003">Figure 3</a>) also suggest that low RH alleles tend to present peptides that are more sensitive to mutations, because they prefer tryptophan (W) at position 9. Tryptophan is the only amino acid coded by a single triplet. Any mutation in a tryptophan triplet will thus lead to an amino acid substitution or a stop-codon. Due to the unusual side chain properties of tryptophan, such amino acid substitutions tend to affect the protein structure and function. As a consequence, tryptophan is the most conserved amino acid. HLA alleles with a high RH, on the other hand, prefer proline (P) at position 2, which is coded by 4 triplets, and is known to be less conserved than tryptophan. HLA alleles with a low RH thus seem to preferentially bind the parts of the HIV genome that are most sensitive to mutations. This may in part explain why they confer better protection against disease progression.</p>
<a id="article1.body1.sec4.p3" name="article1.body1.sec4.p3"></a><p>Since HIV-1 proteins other than p24 may also contain functionally and structurally constrained regions, we investigated if there is a general correlation between the relative hazard of HLA alleles and the tendency to target constrained parts of the HIV-1 proteome. We predicted the HIV-1 epitopes for a large number of HLA alleles and used the entropy of each predicted epitope as a measure of functional and structural constraint. The Shannon entropy <a href="#pone.0000920-Shannon1">[32]</a> of each residue in the HIV-1 proteome was calculated using the HIV-1 protein alignments for clade B available in the Los Alamos HIV database (September 2005). Surprisingly, the average entropies of the three predicted best-binding epitopes for HLA alleles with a low RH were not significantly different from the ones presented by HLA alleles with a high RH (Mann-Whitney, <em>p</em> = 0.87, unpublished results). There was also no significant correlation between the entropy score of the best-binding predicted epitopes and the RH of the HLA presenting the epitope (<em>p</em> = 0.79, unpublished results). Presentation of conserved HIV epitopes is thus not explaining the difference between HLA alleles associated with slow and rapid disease progression.</p>
<a id="article1.body1.sec4.p4" name="article1.body1.sec4.p4"></a><p>There are two non-mutually exclusive explanations for this surprising finding. The first is that targeting conserved epitopes is required but not sufficient to delay disease progression (see also <a href="#pone.0000920-Kiepiela1">[23]</a>). Apart from its conservedness, p24 may have other properties that explain why CTL responses against p24 are most beneficial. Possible factors include the fact that p24 is one of the most immunogenic and abundant HIV proteins. An immature HIV particle contains approximately 1500 copies of p24, while other HIV proteins are present at much lower copy numbers <a href="#pone.0000920-Briggs1">[54]</a>. P24 epitopes are therefore expected to induce stronger immune responses than other HIV epitopes. Additionally, it was recently shown that p24 can be detected within two hours after the infection of a target cell, which is well before other HIV-1 proteins are produced, and before Nef can down-regulate HLA expression <a href="#pone.0000920-Sacha1">[55]</a>. The origin of the early expressed p24 is probably the large amount of p24 packaged in viral particles <a href="#pone.0000920-Briggs1">[54]</a>. Preferential targeting of conserved epitopes from an early and abundantly expressed, highly immunogenic protein, may hence be the clue to slowing down disease progression.</p>
<a id="article1.body1.sec4.p5" name="article1.body1.sec4.p5"></a><p>A second explanation is that entropy is not the correct measure of functional and structural constraints. Indeed, peptides that are hardly constrained may nevertheless have a low entropy if there is no strong CTL pressure on the peptide, or if the HLA molecule by which the peptide is restricted is very rare in the human population. Conversely, peptides may have a high entropy despite functional or structural constraint if the CTL pressure on the peptide is so high that the peptide mutates despite a high viral fitness cost <a href="#pone.0000920-Frater1">[56]</a>. The latter is exactly what has been described for p24 epitopes targeted by elite suppressors of HIV-1 <a href="#pone.0000920-Bailey1">[7]</a>. Despite the lack of correlation between the relative hazard of HIV disease progression and the tendency to present peptides with low entropy, preferential targeting of functionally and structurally constrained regions of HIV-1 may thus be key to slowing down disease progression.</p>
<a id="article1.body1.sec4.p6" name="article1.body1.sec4.p6"></a><p>HLA-B27 was the first HLA that was described to be associated with slow HIV-1 disease progression <a href="#pone.0000920-Nixon1">[57]</a>. Our analyses reveal an important difference between HLA-B27 on the one hand and HLA-B57 and B58 on the other. While the latter two HLA alleles target at least 3 different p24 peptides with high affinity, HLA-B27 has only one good-binding p24 peptide (see <a href="#pone-0000920-g001">Figure 1</a>). It has been reported that CTL escape of this epitope caused a rapid increase of viremia in an HLA-B27-positive long-term non-progressing child <a href="#pone.0000920-Feeney1">[58]</a>. Our analyses suggest that this abrupt breakage of protection may be caused by the absence of other protective anti-p24 CTL responses.</p>
<a id="article1.body1.sec4.p7" name="article1.body1.sec4.p7"></a><p>The differences between HLA alleles associated with slow and rapid disease progression have also been sought in their frequencies in the human population. Trachtenberg <em>et al.</em> <a href="#pone.0000920-Trachtenberg1">[15]</a> demonstrated a significant correlation between the population frequency of HLA supertypes <a href="#pone.0000920-Sette1">[38]</a>, <a href="#pone.0000920-Lund1">[59]</a> and the HIV-1 viral load at set point in a large group of HIV-1 infected homosexual men. Scherer <em>et al.</em> <a href="#pone.0000920-Scherer1">[16]</a> extended these results by showing that common HLA alleles are associated with a lack of CTL responses to known HIV-1 epitopes, further supporting the idea that HIV-1 is adapting to the most common HLA molecules in the human population. Frahm <em>et al.</em> <a href="#pone.0000920-Frahm2">[17]</a> demonstrated that an HLA molecule associated with a low viral load in a population in which the HLA is rare, lacked this association in another population where the HLA is common. Data are conflicting, however, because no significant correlation could be found between the relative hazard of HIV disease progression and the population frequency of HLA molecules <a href="#pone.0000920-Gao1">[13]</a>.</p>
<a id="article1.body1.sec4.p8" name="article1.body1.sec4.p8"></a><p>Whatever the effect of the population frequency of HLA molecules on the rate of HIV disease progression is, the current study shows that qualitative differences in the epitopes targeted by different HLA molecules contribute to the association between HLA molecules and the rate of HIV-1 disease progression.</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>We thank Frank Miedema for valuable discussions, Morten Nielsen and Claus Lundegaard for extensive discussions on MHC-peptide binding predictions, Almut Scherer for valuable comments regarding an earlier version of the manuscript, and Frahm <em>et al.</em> <a href="#pone.0000920-Frahm3">[26]</a> for making their dataset publicly available.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Analyzed the data: CK AM. Wrote the paper: JB. Other: Performed the bio-informatic analyses: CK. Made the crystal structure analyses: AM. Contributed to the study design: Rd CK JB. Contributed to the interpretation of the data: AM Rd CK JB. Contributed to the writing of the manuscript: AM Rd JB CK.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0000920-Carrington1" id="pone.0000920-Carrington1"></a>Carrington M, O'Brien SJ (2003) The influence of HLA genotype on AIDS. Annu Rev Med  4: 535–551.  <ul class="find" data-citedArticleID="994510" data-doi="10.1146/annurev.med.54.101601.152346"><li><a href="http://dx.doi.org/10.1146/annurev.med.54.101601.152346" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+influence+of+HLA+genotype+on+AIDS." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+influence+of+HLA+genotype+on+AIDS.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0000920-Moore1" id="pone.0000920-Moore1"></a>Moore CB, John M, James IR, Christiansen FT, Witt CS, et al.  (2002) Evidence of HIV-1 adaptation to HLA-restricted immune responses at a population level. Science  296: 1439–1443.  <ul class="find" data-citedArticleID="994566" data-doi="10.1126/science.1069660"><li><a href="http://dx.doi.org/10.1126/science.1069660" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evidence+of+HIV-1+adaptation+to+HLA-restricted+immune+responses+at+a+population+level." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evidence+of+HIV-1+adaptation+to+HLA-restricted+immune+responses+at+a+population+level.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0000920-Leslie1" id="pone.0000920-Leslie1"></a>Leslie AJ, Pfafferott KJ, Chetty P, Draenert R, Addo MM, et al.  (2004) HIV evolution: CTL escape mutation and reversion after transmission. Nat Med  10: 282–289.  <ul class="find" data-citedArticleID="994554" data-doi="10.1038/nm992"><li><a href="http://dx.doi.org/10.1038/nm992" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=HIV+evolution%3A+CTL+escape+mutation+and+reversion+after+transmission." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22HIV+evolution%3A+CTL+escape+mutation+and+reversion+after+transmission.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0000920-Friedrich1" id="pone.0000920-Friedrich1"></a>Friedrich TC, Dodds EJ, Yant LJ, Vojnov L, Rudersdorf R, et al.  (2004) Reversion of CTL escape-variant immunodeficiency viruses in vivo. Nat Med  10: 275–281.  <ul class="find" data-citedArticleID="994528" data-doi="10.1038/nm998"><li><a href="http://dx.doi.org/10.1038/nm998" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Reversion+of+CTL+escape-variant+immunodeficiency+viruses+in+vivo." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Reversion+of+CTL+escape-variant+immunodeficiency+viruses+in+vivo.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0000920-Klein1" id="pone.0000920-Klein1"></a>Klein MR, Van der Burg SH, Hovenkamp E, Holwerda AM, Drijfhout JW, et al.  (1998) Characterization of HLA-B57-restricted human immunodeficiency virus type 1 Gag- and RT-specific cytotoxic T lymphocyte responses. J Gen Virol  79: 2191–2201.  <ul class="find" data-citedArticleID="994550"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Characterization+of+HLA-B57-restricted+human+immunodeficiency+virus+type+1+Gag-+and+RT-specific+cytotoxic+T+lymphocyte+responses.&amp;auth=&amp;atitle=Characterization+of+HLA-B57-restricted+human+immunodeficiency+virus+type+1+Gag-+and+RT-specific+cytotoxic+T+lymphocyte+responses." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Characterization+of+HLA-B57-restricted+human+immunodeficiency+virus+type+1+Gag-+and+RT-specific+cytotoxic+T+lymphocyte+responses." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Characterization+of+HLA-B57-restricted+human+immunodeficiency+virus+type+1+Gag-+and+RT-specific+cytotoxic+T+lymphocyte+responses.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0000920-Gillespie1" id="pone.0000920-Gillespie1"></a>Gillespie GM, Kaul R, Dong T, Yang HB, Rostron T, et al.  (2002) Cross-reactive cytotoxic T lymphocytes against a HIV-1 p24 epitope in slow progressors with B*57. AIDS  16: 961–972.  <ul class="find" data-citedArticleID="994534" data-doi="10.1097/00002030-200205030-00002"><li><a href="http://dx.doi.org/10.1097/00002030-200205030-00002" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Cross-reactive+cytotoxic+T+lymphocytes+against+a+HIV-1+p24+epitope+in+slow+progressors+with+B*57." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Cross-reactive+cytotoxic+T+lymphocytes+against+a+HIV-1+p24+epitope+in+slow+progressors+with+B*57.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0000920-Bailey1" id="pone.0000920-Bailey1"></a>Bailey JR, Williams TM, Siliciano RF, Blankson JN (2006) Maintenance of viral suppression in HIV-1-infected HLA-B*57<sup>+</sup> elite suppressors despite CTL escape mutations. J Exp Med  203: 1357–1369.  <ul class="find" data-citedArticleID="994498" data-doi="10.1084/jem.20052319"><li><a href="http://dx.doi.org/10.1084/jem.20052319" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Maintenance+of+viral+suppression+in+HIV-1-infected+HLA-B*57%2B+elite+suppressors+despite+CTL+escape+mutations." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Maintenance+of+viral+suppression+in+HIV-1-infected+HLA-B*57%2B+elite+suppressors+despite+CTL+escape+mutations.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0000920-Jansen1" id="pone.0000920-Jansen1"></a>Jansen CA, Kostense S, Vandenberghe K, Nanlohy NM, De Cuyper IM, et al.  (2005) High responsiveness of HLA-B57-restricted Gag-specific CD8<sup>+</sup> T cells in vitro may contribute to the protective effect of HLA-B57 in HIV-infection. Eur J Immunol  35: 150–158.  <ul class="find" data-citedArticleID="994540" data-doi="10.1002/eji.200425487"><li><a href="http://dx.doi.org/10.1002/eji.200425487" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=High+responsiveness+of+HLA-B57-restricted+Gag-specific+CD8%2B+T+cells+in+vitro+may+contribute+to+the+protective+effect+of+HLA-B57+in+HIV-infection." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22High+responsiveness+of+HLA-B57-restricted+Gag-specific+CD8%2B+T+cells+in+vitro+may+contribute+to+the+protective+effect+of+HLA-B57+in+HIV-infection.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0000920-Altfeld1" id="pone.0000920-Altfeld1"></a>Altfeld M, Addo MM, Rosenberg ES, Hecht FM, Lee PK, et al.  (2003) Influence of HLA-B57 on clinical presentation and viral control during acute HIV-1 infection. AIDS  17: 2581–2591.  <ul class="find" data-citedArticleID="994492" data-doi="10.1097/00002030-200312050-00005"><li><a href="http://dx.doi.org/10.1097/00002030-200312050-00005" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Influence+of+HLA-B57+on+clinical+presentation+and+viral+control+during+acute+HIV-1+infection." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Influence+of+HLA-B57+on+clinical+presentation+and+viral+control+during+acute+HIV-1+infection.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0000920-Altfeld2" id="pone.0000920-Altfeld2"></a>Altfeld M, Kalife ET, Qi Y, Streeck H, Lichterfeld M, et al.  (2006) HLA alleles associated with delayed progression to AIDS contribute strongly to the initial CD8<sup>+</sup> T cell response against HIV-1. PLoS Med  3: e403.  <ul class="find" data-citedArticleID="994494" data-doi="10.1371/journal.pmed.0030403"><li><a href="http://dx.doi.org/10.1371/journal.pmed.0030403" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=HLA+alleles+associated+with+delayed+progression+to+AIDS+contribute+strongly+to+the+initial+CD8%2B+T+cell+response+against+HIV-1." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22HLA+alleles+associated+with+delayed+progression+to+AIDS+contribute+strongly+to+the+initial+CD8%2B+T+cell+response+against+HIV-1.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0000920-Frahm1" id="pone.0000920-Frahm1"></a>Frahm N, Adams S, Kiepiela P, Linde CH, Hewitt HS, et al.  (2005) HLA-B63 presents HLA-B57/B58-restricted cytotoxic T-lymphocyte epitopes and is associated with low human immunodeficiency virus load. J Virol  79: 10218–10225.  <ul class="find" data-citedArticleID="994520" data-doi="10.1128/jvi.79.16.10218-10225.2005"><li><a href="http://dx.doi.org/10.1128/jvi.79.16.10218-10225.2005" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=HLA-B63+presents+HLA-B57%2FB58-restricted+cytotoxic+T-lymphocyte+epitopes+and+is+associated+with+low+human+immunodeficiency+virus+load." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22HLA-B63+presents+HLA-B57%2FB58-restricted+cytotoxic+T-lymphocyte+epitopes+and+is+associated+with+low+human+immunodeficiency+virus+load.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0000920-Goulder1" id="pone.0000920-Goulder1"></a>Goulder PJ, Edwards A, Phillips RE, McMichael AJ (1997) Identification of a novel HLA-B*2705-restricted cytotoxic T-lymphocyte epitope within a conserved region of HIV-1 Nef. AIDS  11: 536–538.  <ul class="find" data-citedArticleID="994536" data-doi="10.1097/00002030-199715000-00015"><li><a href="http://dx.doi.org/10.1097/00002030-199715000-00015" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Identification+of+a+novel+HLA-B*2705-restricted+cytotoxic+T-lymphocyte+epitope+within+a+conserved+region+of+HIV-1+Nef." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Identification+of+a+novel+HLA-B*2705-restricted+cytotoxic+T-lymphocyte+epitope+within+a+conserved+region+of+HIV-1+Nef.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0000920-Gao1" id="pone.0000920-Gao1"></a>Gao X, Nelson GW, Karacki P, Martin MP, Phair J, et al.  (2001) Effect of a single amino acid change in MHC class I molecules on the rate of progression to AIDS. N Engl J Med  344: 1668–1675.  <ul class="find" data-citedArticleID="994532" data-doi="10.1056/nejm200105313442203"><li><a href="http://dx.doi.org/10.1056/nejm200105313442203" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Effect+of+a+single+amino+acid+change+in+MHC+class+I+molecules+on+the+rate+of+progression+to+AIDS." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Effect+of+a+single+amino+acid+change+in+MHC+class+I+molecules+on+the+rate+of+progression+to+AIDS.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0000920-Carrington2" id="pone.0000920-Carrington2"></a>Carrington M, Nelson GW, Martin MP, Kissner T, Vlahov D, et al.  (1999) HLA and HIV-1: heterozygote advantage and B*35-Cw*04 disadvantage. Science  283: 1748–1752.  <ul class="find" data-citedArticleID="994512" data-doi="10.1126/science.283.5408.1748"><li><a href="http://dx.doi.org/10.1126/science.283.5408.1748" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=HLA+and+HIV-1%3A+heterozygote+advantage+and+B*35-Cw*04+disadvantage." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22HLA+and+HIV-1%3A+heterozygote+advantage+and+B*35-Cw*04+disadvantage.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0000920-Trachtenberg1" id="pone.0000920-Trachtenberg1"></a>Trachtenberg E, Korber B, Sollars C, Kepler TB, Hraber PT, et al.  (2003) Advantage of rare HLA supertype in HIV disease progression. Nat Med  9: 928–935.  <ul class="find" data-citedArticleID="994598" data-doi="10.1038/nm893"><li><a href="http://dx.doi.org/10.1038/nm893" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Advantage+of+rare+HLA+supertype+in+HIV+disease+progression." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Advantage+of+rare+HLA+supertype+in+HIV+disease+progression.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0000920-Scherer1" id="pone.0000920-Scherer1"></a>Scherer A, Frater J, Oxenius A, Agudelo J, Price DA, et al.  (2004) Quantifiable cytotoxic T lymphocyte responses and HLA-related risk of progression to AIDS. Proc Natl Acad Sci USA  101: 12266–12270.  <ul class="find" data-citedArticleID="994584" data-doi="10.1073/pnas.0404091101"><li><a href="http://dx.doi.org/10.1073/pnas.0404091101" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Quantifiable+cytotoxic+T+lymphocyte+responses+and+HLA-related+risk+of+progression+to+AIDS." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Quantifiable+cytotoxic+T+lymphocyte+responses+and+HLA-related+risk+of+progression+to+AIDS.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0000920-Frahm2" id="pone.0000920-Frahm2"></a>Frahm N, Kiepiela P, Adams S, Linde CH, Hewitt HS, et al.  (2006) Control of human immunodeficiency virus replication by cytotoxic T lymphocytes targeting subdominant epitopes. Nat Immunol  7: 173–178.  <ul class="find" data-citedArticleID="994522" data-doi="10.1038/ni1281"><li><a href="http://dx.doi.org/10.1038/ni1281" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Control+of+human+immunodeficiency+virus+replication+by+cytotoxic+T+lymphocytes+targeting+subdominant+epitopes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Control+of+human+immunodeficiency+virus+replication+by+cytotoxic+T+lymphocytes+targeting+subdominant+epitopes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0000920-Buseyne1" id="pone.0000920-Buseyne1"></a>Buseyne F, Le Chenadec J, Corre B, Porrot F, Burgard M, et al.  (2002) Inverse correlation between memory Gag-specific cytotoxic T lymphocytes and viral replication in human immunodeficiency virus-infected children. J Infect Dis  186: 1589–1596.  <ul class="find" data-citedArticleID="994506" data-doi="10.1086/345482"><li><a href="http://dx.doi.org/10.1086/345482" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Inverse+correlation+between+memory+Gag-specific+cytotoxic+T+lymphocytes+and+viral+replication+in+human+immunodeficiency+virus-infected+children." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Inverse+correlation+between+memory+Gag-specific+cytotoxic+T+lymphocytes+and+viral+replication+in+human+immunodeficiency+virus-infected+children.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0000920-Edwards1" id="pone.0000920-Edwards1"></a>Edwards BH, Bansal A, Sabbaj S, Bakari J, Mulligan MJ, et al.  (2002) Magnitude of functional CD8<sup>+</sup> T-cell responses to the gag protein of human immunodeficiency virus type 1 correlates inversely with viral load in plasma. J Virol  76: 2298–2305.  <ul class="find" data-citedArticleID="994516" data-doi="10.1128/jvi.76.5.2298-2305.2002"><li><a href="http://dx.doi.org/10.1128/jvi.76.5.2298-2305.2002" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Magnitude+of+functional+CD8%2B+T-cell+responses+to+the+gag+protein+of+human+immunodeficiency+virus+type+1+correlates+inversely+with+viral+load+in+plasma." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Magnitude+of+functional+CD8%2B+T-cell+responses+to+the+gag+protein+of+human+immunodeficiency+virus+type+1+correlates+inversely+with+viral+load+in+plasma.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0000920-Novitsky1" id="pone.0000920-Novitsky1"></a>Novitsky V, Gilbert P, Peter T, McLane MF, Gaolekwe S, et al.  (2003) Association between virus-specific T-cell responses and plasma viral load in human immunodeficiency virus type 1 subtype C infection. J Virol  77: 882–890.  <ul class="find" data-citedArticleID="994572" data-doi="10.1128/jvi.77.2.882-890.2003"><li><a href="http://dx.doi.org/10.1128/jvi.77.2.882-890.2003" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Association+between+virus-specific+T-cell+responses+and+plasma+viral+load+in+human+immunodeficiency+virus+type+1+subtype+C+infection." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Association+between+virus-specific+T-cell+responses+and+plasma+viral+load+in+human+immunodeficiency+virus+type+1+subtype+C+infection.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0000920-Masemola1" id="pone.0000920-Masemola1"></a>Masemola A, Mashishi T, Khoury G, Mohube P, Mokgotho P, et al.  (2004) Hierarchical targeting of subtype C human immunodeficiency virus type 1 proteins by CD8<sup>+</sup> T cells: correlation with viral load. J Virol  78: 3233–3243.  <ul class="find" data-citedArticleID="994562" data-doi="10.1128/jvi.78.7.3233-3243.2004"><li><a href="http://dx.doi.org/10.1128/jvi.78.7.3233-3243.2004" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Hierarchical+targeting+of+subtype+C+human+immunodeficiency+virus+type+1+proteins+by+CD8%2B+T+cells%3A+correlation+with+viral+load." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Hierarchical+targeting+of+subtype+C+human+immunodeficiency+virus+type+1+proteins+by+CD8%2B+T+cells%3A+correlation+with+viral+load.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0000920-Zuniga1" id="pone.0000920-Zuniga1"></a>Zuniga R, Lucchetti A, Galvan P, Sanchez S, Sanchez C, et al.  (2006) Relative dominance of Gag p24-specific cytotoxic T lymphocytes is associated with human immunodeficiency virus control. J Virol  80: 3122–3125.  <ul class="find" data-citedArticleID="994606" data-doi="10.1128/jvi.80.6.3122-3125.2006"><li><a href="http://dx.doi.org/10.1128/jvi.80.6.3122-3125.2006" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Relative+dominance+of+Gag+p24-specific+cytotoxic+T+lymphocytes+is+associated+with+human+immunodeficiency+virus+control." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Relative+dominance+of+Gag+p24-specific+cytotoxic+T+lymphocytes+is+associated+with+human+immunodeficiency+virus+control.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0000920-Kiepiela1" id="pone.0000920-Kiepiela1"></a>Kiepiela P, Ngumbela K, Thobakgale C, Ramduth D, Honeyborne I, et al.  (2007) CD8<sup>+</sup> T-cell responses to different HIV proteins have discordant associations with viral load. Nat Med  13: 46–53.  <ul class="find" data-citedArticleID="994548" data-doi="10.1038/nm1520"><li><a href="http://dx.doi.org/10.1038/nm1520" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=CD8%2B+T-cell+responses+to+different+HIV+proteins+have+discordant+associations+with+viral+load." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22CD8%2B+T-cell+responses+to+different+HIV+proteins+have+discordant+associations+with+viral+load.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0000920-Honeyborne1" id="pone.0000920-Honeyborne1"></a>Honeyborne I, Prendergast A, Pereyra F, Leslie A, Crawford H, et al.  (2007) Control of human immunodeficiency virus type 1 is associated with HLA-B*13 and targeting of multiple gag-specific CD8<sup>+</sup> T-cell epitopes. J Virol  81: 3667–3672.  <ul class="find" data-citedArticleID="994538" data-doi="10.1128/jvi.02689-06"><li><a href="http://dx.doi.org/10.1128/jvi.02689-06" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Control+of+human+immunodeficiency+virus+type+1+is+associated+with+HLA-B*13+and+targeting+of+multiple+gag-specific+CD8%2B+T-cell+epitopes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Control+of+human+immunodeficiency+virus+type+1+is+associated+with+HLA-B*13+and+targeting+of+multiple+gag-specific+CD8%2B+T-cell+epitopes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0000920-Streeck1" id="pone.0000920-Streeck1"></a>Streeck H, Lichterfeld M, Alter G, Meier A, Teigen N, et al.  (2007) Recognition of a defined region within p24 Gag by CD8<sup>+</sup> T cells during primary human immunodeficiency virus type 1 infection in individuals expressing protective HLA class I alleles. J Virol  81: 7725–7731.  <ul class="find" data-citedArticleID="994594" data-doi="10.1128/jvi.00708-07"><li><a href="http://dx.doi.org/10.1128/jvi.00708-07" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Recognition+of+a+defined+region+within+p24+Gag+by+CD8%2B+T+cells+during+primary+human+immunodeficiency+virus+type+1+infection+in+individuals+expressing+protective+HLA+class+I+alleles." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Recognition+of+a+defined+region+within+p24+Gag+by+CD8%2B+T+cells+during+primary+human+immunodeficiency+virus+type+1+infection+in+individuals+expressing+protective+HLA+class+I+alleles.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0000920-Frahm3" id="pone.0000920-Frahm3"></a>Frahm N, Korber BT, Adams CM, Szinger JJ, Draenert R, et al.  (2004) Consistent cytotoxic T-lymphocyte targeting of immunodominant regions in human immunodeficiency virus across multiple ethnicities. J Virol  78: 2187–2200.  <ul class="find" data-citedArticleID="994524" data-doi="10.1128/jvi.78.5.2187-2200.2004"><li><a href="http://dx.doi.org/10.1128/jvi.78.5.2187-2200.2004" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Consistent+cytotoxic+T-lymphocyte+targeting+of+immunodominant+regions+in+human+immunodeficiency+virus+across+multiple+ethnicities." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Consistent+cytotoxic+T-lymphocyte+targeting+of+immunodominant+regions+in+human+immunodeficiency+virus+across+multiple+ethnicities.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0000920-Peters1" id="pone.0000920-Peters1"></a>Peters B, Sette A (2005) Generating quantitative models describing the sequence specificity of biological processes with the stabilized matrix method. BMC Bioinformatics  6: 132.  <ul class="find" data-citedArticleID="994576"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Generating+quantitative+models+describing+the+sequence+specificity+of+biological+processes+with+the+stabilized+matrix+method.&amp;auth=&amp;atitle=Generating+quantitative+models+describing+the+sequence+specificity+of+biological+processes+with+the+stabilized+matrix+method." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Generating+quantitative+models+describing+the+sequence+specificity+of+biological+processes+with+the+stabilized+matrix+method." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Generating+quantitative+models+describing+the+sequence+specificity+of+biological+processes+with+the+stabilized+matrix+method.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0000920-Buus1" id="pone.0000920-Buus1"></a>Buus S, Lauemoller SL, Worning P, Keşmir C, Frimurer T, et al.  (2003) Sensitive quantitative predictions of peptide-MHC binding by a ‘Query by Committee’ artificial neural network approach. Tissue Antigens  62: 378–384.  <ul class="find" data-citedArticleID="994508" data-doi="10.1034/j.1399-0039.2003.00112.x"><li><a href="http://dx.doi.org/10.1034/j.1399-0039.2003.00112.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Sensitive+quantitative+predictions+of+peptide-MHC+binding+by+a+%26%23x2018%3BQuery+by+Committee%26%23x2019%3B+artificial+neural+network+approach." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Sensitive+quantitative+predictions+of+peptide-MHC+binding+by+a+%26%23x2018%3BQuery+by+Committee%26%23x2019%3B+artificial+neural+network+approach.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0000920-Nielsen1" id="pone.0000920-Nielsen1"></a>Nielsen M, Lundegaard C, Worning P, Lauemoller SL, Lamberth K, et al.  (2003) Reliable prediction of T-cell epitopes using neural networks with novel sequence representations. Protein Sci  12: 1007–1017.  <ul class="find" data-citedArticleID="994568" data-doi="10.1110/ps.0239403"><li><a href="http://dx.doi.org/10.1110/ps.0239403" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Reliable+prediction+of+T-cell+epitopes+using+neural+networks+with+novel+sequence+representations." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Reliable+prediction+of+T-cell+epitopes+using+neural+networks+with+novel+sequence+representations.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0000920-Sturniolo1" id="pone.0000920-Sturniolo1"></a>Sturniolo T, Bono E, Ding J, Raddrizzani L, Tuereci O, et al.  (1999) Generation of tissue-specific and promiscuous HLA ligand databases using DNA microarrays and virtual HLA class II matrices. Nat Biotechnol  17: 555–561.  <ul class="find" data-citedArticleID="994596"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Generation+of+tissue-specific+and+promiscuous+HLA+ligand+databases+using+DNA+microarrays+and+virtual+HLA+class+II+matrices.&amp;auth=&amp;atitle=Generation+of+tissue-specific+and+promiscuous+HLA+ligand+databases+using+DNA+microarrays+and+virtual+HLA+class+II+matrices." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Generation+of+tissue-specific+and+promiscuous+HLA+ligand+databases+using+DNA+microarrays+and+virtual+HLA+class+II+matrices." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Generation+of+tissue-specific+and+promiscuous+HLA+ligand+databases+using+DNA+microarrays+and+virtual+HLA+class+II+matrices.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0000920-Schneider1" id="pone.0000920-Schneider1"></a>Schneider TD, Stephens RM (1990) Sequence logos: a new way to display consensus sequences. Nucleic Acids Res  18: 6097–6100.  <ul class="find" data-citedArticleID="994586" data-doi="10.1093/nar/18.20.6097"><li><a href="http://dx.doi.org/10.1093/nar/18.20.6097" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Sequence+logos%3A+a+new+way+to+display+consensus+sequences." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Sequence+logos%3A+a+new+way+to+display+consensus+sequences.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0000920-Shannon1" id="pone.0000920-Shannon1"></a>Shannon CE (1948) A mathematical theory of communication. Bell System Tech J  27: 379–423.623–656.  <ul class="find" data-citedArticleID="994590" data-doi="10.1002/j.1538-7305.1948.tb01338.x"><li><a href="http://dx.doi.org/10.1002/j.1538-7305.1948.tb01338.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+mathematical+theory+of+communication." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+mathematical+theory+of+communication.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0000920-Simes1" id="pone.0000920-Simes1"></a>Simes RJ (1986) An improved Bonferroni procedure for multiple tests of significance. Biometrika  73: 751–754.  <ul class="find" data-citedArticleID="994592" data-doi="10.1093/biomet/73.3.751"><li><a href="http://dx.doi.org/10.1093/biomet/73.3.751" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=An+improved+Bonferroni+procedure+for+multiple+tests+of+significance." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22An+improved+Bonferroni+procedure+for+multiple+tests+of+significance.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0000920-Assarsson1" id="pone.0000920-Assarsson1"></a>Assarsson E, Sidney J, Oseroff C, Pasquetto V, Bui HH, et al.  (2007) A quantitative analysis of the variables affecting the repertoire of T cell specificities recognized after vaccinia virus infection. J Immunol  178: 7890–7901.  <ul class="find" data-citedArticleID="994496"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=A+quantitative+analysis+of+the+variables+affecting+the+repertoire+of+T+cell+specificities+recognized+after+vaccinia+virus+infection.&amp;auth=&amp;atitle=A+quantitative+analysis+of+the+variables+affecting+the+repertoire+of+T+cell+specificities+recognized+after+vaccinia+virus+infection." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+quantitative+analysis+of+the+variables+affecting+the+repertoire+of+T+cell+specificities+recognized+after+vaccinia+virus+infection." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+quantitative+analysis+of+the+variables+affecting+the+repertoire+of+T+cell+specificities+recognized+after+vaccinia+virus+infection.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0000920-Kemir1" id="pone.0000920-Kemir1"></a>Keşmir C, Nussbaum AK, Schild H, Detours V, Brunak S (2002) Prediction of proteasome cleavage motifs by neural networks. Protein Eng  15: 287–296.  <ul class="find" data-citedArticleID="994546" data-doi="10.1093/protein/15.4.287"><li><a href="http://dx.doi.org/10.1093/protein/15.4.287" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Prediction+of+proteasome+cleavage+motifs+by+neural+networks." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Prediction+of+proteasome+cleavage+motifs+by+neural+networks.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">36.
              </span><a name="pone.0000920-Peters2" id="pone.0000920-Peters2"></a>Peters B, Bulik S, Tampe R, Van Endert PM, Holzhutter HG (2003) Identifying MHC class I epitopes by predicting the TAP transport efficiency of epitope precursors. J Immunol  171: 1741–1749.  <ul class="find" data-citedArticleID="994578"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Identifying+MHC+class+I+epitopes+by+predicting+the+TAP+transport+efficiency+of+epitope+precursors.&amp;auth=&amp;atitle=Identifying+MHC+class+I+epitopes+by+predicting+the+TAP+transport+efficiency+of+epitope+precursors." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Identifying+MHC+class+I+epitopes+by+predicting+the+TAP+transport+efficiency+of+epitope+precursors." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Identifying+MHC+class+I+epitopes+by+predicting+the+TAP+transport+efficiency+of+epitope+precursors.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">37.
              </span><a name="pone.0000920-Korber1" id="pone.0000920-Korber1"></a>Korber B, Muldoon M, Theiler J, Gao F, Gupta R, et al.  (2000) Timing the ancestor of the HIV-1 pandemic strains. Science  288: 1789–1796.  <ul class="find" data-citedArticleID="994552" data-doi="10.1126/science.288.5472.1789"><li><a href="http://dx.doi.org/10.1126/science.288.5472.1789" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Timing+the+ancestor+of+the+HIV-1+pandemic+strains." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Timing+the+ancestor+of+the+HIV-1+pandemic+strains.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">38.
              </span><a name="pone.0000920-Sette1" id="pone.0000920-Sette1"></a>Sette A, Sidney J (1999) Nine major HLA class I supertypes account for the vast preponderance of HLA-A and -B polymorphism. Immunogenetics  50: 201–212.  <ul class="find" data-citedArticleID="994588" data-doi="10.1007/s002510050594"><li><a href="http://dx.doi.org/10.1007/s002510050594" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Nine+major+HLA+class+I+supertypes+account+for+the+vast+preponderance+of+HLA-A+and+-B+polymorphism." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Nine+major+HLA+class+I+supertypes+account+for+the+vast+preponderance+of+HLA-A+and+-B+polymorphism.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">39.
              </span><a name="pone.0000920-Novitsky2" id="pone.0000920-Novitsky2"></a>Novitsky V, Smith UR, Gilbert P, McLane MF, Chigwedere P, et al.  (2002) Human immunodeficiency virus type 1 subtype C molecular phylogeny: consensus sequence for an AIDS vaccine design? J Virol  76: 5435–5451.  <ul class="find" data-citedArticleID="994574" data-doi="10.1128/jvi.76.11.5435-5451.2002"><li><a href="http://dx.doi.org/10.1128/jvi.76.11.5435-5451.2002" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Human+immunodeficiency+virus+type+1+subtype+C+molecular+phylogeny%3A+consensus+sequence+for+an+AIDS+vaccine+design%3F" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Human+immunodeficiency+virus+type+1+subtype+C+molecular+phylogeny%3A+consensus+sequence+for+an+AIDS+vaccine+design%3F%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">40.
              </span><a name="pone.0000920-VonSchwedler1" id="pone.0000920-VonSchwedler1"></a>Von Schwedler UK, Stray KM, Garrus JE, Sundquist WI (2003) Functional surfaces of the human immunodeficiency virus type 1 capsid protein. J Virol  77: 5439–5450.  <ul class="find" data-citedArticleID="994600" data-doi="10.1128/jvi.77.9.5439-5450.2003"><li><a href="http://dx.doi.org/10.1128/jvi.77.9.5439-5450.2003" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Functional+surfaces+of+the+human+immunodeficiency+virus+type+1+capsid+protein." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Functional+surfaces+of+the+human+immunodeficiency+virus+type+1+capsid+protein.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">41.
              </span><a name="pone.0000920-Peyerl1" id="pone.0000920-Peyerl1"></a>Peyerl FW, Bazick HS, Newberg MH, Barouch DH, Sodroski J, et al.  (2004) Fitness costs limit viral escape from cytotoxic T lymphocytes at a structurally constrained epitope. J Virol  78: 13901–13910.  <ul class="find" data-citedArticleID="994580" data-doi="10.1128/jvi.78.24.13901-13910.2004"><li><a href="http://dx.doi.org/10.1128/jvi.78.24.13901-13910.2004" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Fitness+costs+limit+viral+escape+from+cytotoxic+T+lymphocytes+at+a+structurally+constrained+epitope." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Fitness+costs+limit+viral+escape+from+cytotoxic+T+lymphocytes+at+a+structurally+constrained+epitope.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">42.
              </span><a name="pone.0000920-Lichterfeld1" id="pone.0000920-Lichterfeld1"></a>Lichterfeld M, Yu XG, Cohen D, Addo MM, Malenfant J, et al.  (2004) HIV-1 Nef is preferentially recognized by CD8 T cells in primary HIV-1 infection despite a relatively high degree of genetic diversity. AIDS  18: 1383–1392.  <ul class="find" data-citedArticleID="994556" data-doi="10.1097/01.aids.0000131329.51633.a3"><li><a href="http://dx.doi.org/10.1097/01.aids.0000131329.51633.a3" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=HIV-1+Nef+is+preferentially+recognized+by+CD8+T+cells+in+primary+HIV-1+infection+despite+a+relatively+high+degree+of+genetic+diversity." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22HIV-1+Nef+is+preferentially+recognized+by+CD8+T+cells+in+primary+HIV-1+infection+despite+a+relatively+high+degree+of+genetic+diversity.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">43.
              </span><a name="pone.0000920-Crawford1" id="pone.0000920-Crawford1"></a>Crawford H, Prado JG, Leslie A, Hue S, Honeyborne I, et al.  (2007) Compensatory mutation partially restores fitness and delays reversion of escape mutation within the immunodominant HLA-B*5703-restricted Gag epitope in chronic human immunodeficiency virus type 1 infection. J Virol  81: 8346–8351.  <ul class="find" data-citedArticleID="994514" data-doi="10.1128/jvi.00465-07"><li><a href="http://dx.doi.org/10.1128/jvi.00465-07" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Compensatory+mutation+partially+restores+fitness+and+delays+reversion+of+escape+mutation+within+the+immunodominant+HLA-B*5703-restricted+Gag+epitope+in+chronic+human+immunodeficiency+virus+type+1+infection." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Compensatory+mutation+partially+restores+fitness+and+delays+reversion+of+escape+mutation+within+the+immunodominant+HLA-B*5703-restricted+Gag+epitope+in+chronic+human+immunodeficiency+virus+type+1+infection.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">44.
              </span><a name="pone.0000920-Berman1" id="pone.0000920-Berman1"></a>Berman HM, Westbrook J, Feng Z, Gilliland G, Bhat TN, et al.  (2000) The Protein Data Bank. Nucleic Acids Res  28: 235–242.  <ul class="find" data-citedArticleID="994502" data-doi="10.1093/nar/28.1.235"><li><a href="http://dx.doi.org/10.1093/nar/28.1.235" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+Protein+Data+Bank." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+Protein+Data+Bank.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">45.
              </span><a name="pone.0000920-Momany1" id="pone.0000920-Momany1"></a>Momany C, Kovari LC, Prongay AJ, Keller W, Gitti RK, et al.  (1996) Crystal structure of dimeric HIV-1 capsid protein. Nat Struct Biol  3: 763–770.  <ul class="find" data-citedArticleID="994564" data-doi="10.1038/nsb0996-763"><li><a href="http://dx.doi.org/10.1038/nsb0996-763" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Crystal+structure+of+dimeric+HIV-1+capsid+protein." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Crystal+structure+of+dimeric+HIV-1+capsid+protein.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">46.
              </span><a name="pone.0000920-Accola1" id="pone.0000920-Accola1"></a>Accola MA, Strack B, Gottlinger HG (2000) Efficient particle production by minimal Gag constructs which retain the carboxy-terminal domain of human immunodeficiency virus type 1 capsid-p2 and a late assembly domain. J Virol  74: 5395–5402.  <ul class="find" data-citedArticleID="994490" data-doi="10.1128/jvi.74.12.5395-5402.2000"><li><a href="http://dx.doi.org/10.1128/jvi.74.12.5395-5402.2000" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Efficient+particle+production+by+minimal+Gag+constructs+which+retain+the+carboxy-terminal+domain+of+human+immunodeficiency+virus+type+1+capsid-p2+and+a+late+assembly+domain." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Efficient+particle+production+by+minimal+Gag+constructs+which+retain+the+carboxy-terminal+domain+of+human+immunodeficiency+virus+type+1+capsid-p2+and+a+late+assembly+domain.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">47.
              </span><a name="pone.0000920-Johnson1" id="pone.0000920-Johnson1"></a>Johnson MC, Scobie HM, Ma YM, Vogt VM (2002) Nucleic acid-independent retrovirus assembly can be driven by dimerization. J Virol  76: 11177–11185.  <ul class="find" data-citedArticleID="994542" data-doi="10.1128/jvi.76.22.11177-11185.2002"><li><a href="http://dx.doi.org/10.1128/jvi.76.22.11177-11185.2002" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Nucleic+acid-independent+retrovirus+assembly+can+be+driven+by+dimerization." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Nucleic+acid-independent+retrovirus+assembly+can+be+driven+by+dimerization.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">48.
              </span><a name="pone.0000920-Zhang1" id="pone.0000920-Zhang1"></a>Zhang Y, Qian H, Love Z, Barklis E (1998) Analysis of the assembly function of the human immunodeficiency virus type 1 gag protein nucleocapsid domain. J Virol  72: 1782–1789.  <ul class="find" data-citedArticleID="994604"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Analysis+of+the+assembly+function+of+the+human+immunodeficiency+virus+type+1+gag+protein+nucleocapsid+domain.&amp;auth=&amp;atitle=Analysis+of+the+assembly+function+of+the+human+immunodeficiency+virus+type+1+gag+protein+nucleocapsid+domain." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Analysis+of+the+assembly+function+of+the+human+immunodeficiency+virus+type+1+gag+protein+nucleocapsid+domain." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Analysis+of+the+assembly+function+of+the+human+immunodeficiency+virus+type+1+gag+protein+nucleocapsid+domain.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">49.
              </span><a name="pone.0000920-Kaul1" id="pone.0000920-Kaul1"></a>Kaul R, Plummer FA, Kimani J, Dong T, Kiama P, et al.  (2000) HIV-1-specific mucosal CD8<sup>+</sup> lymphocyte responses in the cervix of HIV-1-resistant prostitutes in Nairobi. J Immunol  164: 1602–1611.  <ul class="find" data-citedArticleID="994544" data-doi="10.1172/jci10714"><li><a href="http://dx.doi.org/10.1172/jci10714" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=HIV-1-specific+mucosal+CD8%2B+lymphocyte+responses+in+the+cervix+of+HIV-1-resistant+prostitutes+in+Nairobi." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22HIV-1-specific+mucosal+CD8%2B+lymphocyte+responses+in+the+cervix+of+HIV-1-resistant+prostitutes+in+Nairobi.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">50.
              </span><a name="pone.0000920-MartinezPicado1" id="pone.0000920-MartinezPicado1"></a>Martinez-Picado J, Prado JG, Fry EE, Pfafferott K, Leslie A, et al.  (2006) Fitness cost of escape mutations in p24 Gag in association with control of human immunodeficiency virus type 1. J Virol  80: 3617–3623.  <ul class="find" data-citedArticleID="994560" data-doi="10.1128/jvi.80.7.3617-3623.2006"><li><a href="http://dx.doi.org/10.1128/jvi.80.7.3617-3623.2006" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Fitness+cost+of+escape+mutations+in+p24+Gag+in+association+with+control+of+human+immunodeficiency+virus+type+1." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Fitness+cost+of+escape+mutations+in+p24+Gag+in+association+with+control+of+human+immunodeficiency+virus+type+1.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">51.
              </span><a name="pone.0000920-BallaJhagjhoorsingh1" id="pone.0000920-BallaJhagjhoorsingh1"></a>Balla-Jhagjhoorsingh SS, Koopman G, Mooij P, Haaksma TG, Teeuwsen VJ, et al.  (1999) Conserved CTL epitopes shared between HIV-infected human long-term survivors and chimpanzees. J Immunol  162: 2308–2314.  <ul class="find" data-citedArticleID="994500"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Conserved+CTL+epitopes+shared+between+HIV-infected+human+long-term+survivors+and+chimpanzees.&amp;auth=&amp;atitle=Conserved+CTL+epitopes+shared+between+HIV-infected+human+long-term+survivors+and+chimpanzees." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Conserved+CTL+epitopes+shared+between+HIV-infected+human+long-term+survivors+and+chimpanzees." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Conserved+CTL+epitopes+shared+between+HIV-infected+human+long-term+survivors+and+chimpanzees.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">52.
              </span><a name="pone.0000920-Gamble1" id="pone.0000920-Gamble1"></a>Gamble TR, Yoo S, Vajdos FF, Von Schwedler UK, Worthylake DK, et al.  (1997) Structure of the carboxyl-terminal dimerization domain of the HIV-1 capsid protein. Science  278: 849–853.  <ul class="find" data-citedArticleID="994530" data-doi="10.1126/science.278.5339.849"><li><a href="http://dx.doi.org/10.1126/science.278.5339.849" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Structure+of+the+carboxyl-terminal+dimerization+domain+of+the+HIV-1+capsid+protein." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Structure+of+the+carboxyl-terminal+dimerization+domain+of+the+HIV-1+capsid+protein.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">53.
              </span><a name="pone.0000920-Wagner1" id="pone.0000920-Wagner1"></a>Wagner R, Leschonsky B, Harrer E, Paulus C, Weber C, et al.  (1999) Molecular and functional analysis of a conserved CTL epitope in HIV-1 p24 recognized from a long-term nonprogressor: constraints on immune escape associated with targeting a sequence essential for viral replication. J Immunol  162: 3727–3734.  <ul class="find" data-citedArticleID="994602"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Molecular+and+functional+analysis+of+a+conserved+CTL+epitope+in+HIV-1+p24+recognized+from+a+long-term+nonprogressor%3A+constraints+on+immune+escape+associated+with+targeting+a+sequence+essential+for+viral+replication.&amp;auth=&amp;atitle=Molecular+and+functional+analysis+of+a+conserved+CTL+epitope+in+HIV-1+p24+recognized+from+a+long-term+nonprogressor%3A+constraints+on+immune+escape+associated+with+targeting+a+sequence+essential+for+viral+replication." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Molecular+and+functional+analysis+of+a+conserved+CTL+epitope+in+HIV-1+p24+recognized+from+a+long-term+nonprogressor%3A+constraints+on+immune+escape+associated+with+targeting+a+sequence+essential+for+viral+replication." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Molecular+and+functional+analysis+of+a+conserved+CTL+epitope+in+HIV-1+p24+recognized+from+a+long-term+nonprogressor%3A+constraints+on+immune+escape+associated+with+targeting+a+sequence+essential+for+viral+replication.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">54.
              </span><a name="pone.0000920-Briggs1" id="pone.0000920-Briggs1"></a>Briggs JA, Simon MN, Gross I, Krausslich HG, Fuller SD, et al.  (2004) The stoichiometry of Gag protein in HIV-1. Nat Struct Mol Biol  11: 672–675.  <ul class="find" data-citedArticleID="994504" data-doi="10.1038/nsmb785"><li><a href="http://dx.doi.org/10.1038/nsmb785" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+stoichiometry+of+Gag+protein+in+HIV-1." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+stoichiometry+of+Gag+protein+in+HIV-1.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">55.
              </span><a name="pone.0000920-Sacha1" id="pone.0000920-Sacha1"></a>Sacha JB, Chung C, Rakasz EG, Spencer SP, Jonas AK, et al.  (2007) Gag-specific CD8<sup>+</sup> T lymphocytes recognize infected cells before AIDS-virus integration and viral protein expression. J Immunol  178: 2746–2754.  <ul class="find" data-citedArticleID="994582"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Gag-specific+CD8%2B+T+lymphocytes+recognize+infected+cells+before+AIDS-virus+integration+and+viral+protein+expression.&amp;auth=&amp;atitle=Gag-specific+CD8%2B+T+lymphocytes+recognize+infected+cells+before+AIDS-virus+integration+and+viral+protein+expression." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Gag-specific+CD8%2B+T+lymphocytes+recognize+infected+cells+before+AIDS-virus+integration+and+viral+protein+expression." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Gag-specific+CD8%2B+T+lymphocytes+recognize+infected+cells+before+AIDS-virus+integration+and+viral+protein+expression.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">56.
              </span><a name="pone.0000920-Frater1" id="pone.0000920-Frater1"></a>Frater AJ, Brown H, Oxenius A, Gunthard HF, Hirschel B, et al.  (2007) Effective T-cell responses select human immunodeficiency virus mutants and slow disease progression. J Virol  81: 6742–6751.  <ul class="find" data-citedArticleID="994526" data-doi="10.1128/jvi.00022-07"><li><a href="http://dx.doi.org/10.1128/jvi.00022-07" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Effective+T-cell+responses+select+human+immunodeficiency+virus+mutants+and+slow+disease+progression." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Effective+T-cell+responses+select+human+immunodeficiency+virus+mutants+and+slow+disease+progression.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">57.
              </span><a name="pone.0000920-Nixon1" id="pone.0000920-Nixon1"></a>Nixon DF, Townsend AR, Elvin JG, Rizza CR, Gallwey J, et al.  (1988) HIV-1 gag-specific cytotoxic T lymphocytes defined with recombinant vaccinia virus and synthetic peptides. Nature  336: 484–487.  <ul class="find" data-citedArticleID="994570" data-doi="10.1038/336484a0"><li><a href="http://dx.doi.org/10.1038/336484a0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=HIV-1+gag-specific+cytotoxic+T+lymphocytes+defined+with+recombinant+vaccinia+virus+and+synthetic+peptides." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22HIV-1+gag-specific+cytotoxic+T+lymphocytes+defined+with+recombinant+vaccinia+virus+and+synthetic+peptides.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">58.
              </span><a name="pone.0000920-Feeney1" id="pone.0000920-Feeney1"></a>Feeney ME, Tang Y, Roosevelt KA, Leslie AJ, McIntosh K, et al.  (2004) Immune escape precedes breakthrough human immunodeficiency virus type 1 viremia and broadening of the cytotoxic T-lymphocyte response in an HLA-B27-positive long-term-nonprogressing child. J Virol  78: 8927–8930.  <ul class="find" data-citedArticleID="994518" data-doi="10.1128/jvi.78.16.8927-8930.2004"><li><a href="http://dx.doi.org/10.1128/jvi.78.16.8927-8930.2004" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Immune+escape+precedes+breakthrough+human+immunodeficiency+virus+type+1+viremia+and+broadening+of+the+cytotoxic+T-lymphocyte+response+in+an+HLA-B27-positive+long-term-nonprogressing+child." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Immune+escape+precedes+breakthrough+human+immunodeficiency+virus+type+1+viremia+and+broadening+of+the+cytotoxic+T-lymphocyte+response+in+an+HLA-B27-positive+long-term-nonprogressing+child.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">59.
              </span><a name="pone.0000920-Lund1" id="pone.0000920-Lund1"></a>Lund O, Nielsen M, Keşmir C, Petersen AG, Lundegaard C, et al.  (2004) Definition of supertypes for HLA molecules using clustering of specificity matrices. Immunogenetics  55: 797–810.  <ul class="find" data-citedArticleID="994558" data-doi="10.1007/s00251-004-0647-4"><li><a href="http://dx.doi.org/10.1007/s00251-004-0647-4" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Definition+of+supertypes+for+HLA+molecules+using+clustering+of+specificity+matrices." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Definition+of+supertypes+for+HLA+molecules+using+clustering+of+specificity+matrices.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000920.XML" value="122558"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000920.PDF" value="259333"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000920.g001.PNG_L" value="48258"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000920.g001.PNG_M" value="90407"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000920.g001.PNG_S" value="13940"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000920.g001.TIF" value="154670"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000920.g001.PNG_I" value="32092"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000920.g002.PNG_L" value="106590"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000920.g002.PNG_M" value="99192"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000920.g002.PNG_S" value="11893"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000920.g002.TIF" value="451952"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000920.g002.PNG_I" value="34231"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000920.g003.PNG_L" value="55110"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000920.g003.PNG_M" value="73638"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000920.g003.PNG_S" value="10935"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000920.g003.TIF" value="111520"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000920.g003.PNG_I" value="27404"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000920.t001.PNG_L" value="327962"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000920.t001.PNG_M" value="176512"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000920.t001.PNG_S" value="18306"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000920.t001.TIF" value="1604788"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000920.t001.PNG_I" value="58695"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000920.g004.PNG_L" value="1312732"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000920.g004.PNG_M" value="203692"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000920.g004.PNG_S" value="12089"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000920.g004.TIF" value="2059168"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000920.g004.PNG_I" value="69423"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000920&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000920" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000920&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0000920&volume=&issue=&title=HLA Alleles Associated with Slow Progression to AIDS Truly Prefer to Present HIV-1 p24&author_name=Jos%C3%A9%20A.%20M.%20Borghans%2C%20Anne%20M%C3%B8lgaard%2C%20Rob%20J.%20de%20Boer%2C%20Can%20Ke%C5%9Fmir&start_page=1&end_page=9" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000920" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000920" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0000920" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000920&amp;t=HLA%20Alleles%20Associated%20with%20Slow%20Progression%20to%20AIDS%20Truly%20Prefer%20to%20Present%20HIV-1%20p24" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000920&title=HLA%20Alleles%20Associated%20with%20Slow%20Progression%20to%20AIDS%20Truly%20Prefer%20to%20Present%20HIV-1%20p24&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000920&amp;title=HLA%20Alleles%20Associated%20with%20Slow%20Progression%20to%20AIDS%20Truly%20Prefer%20to%20Present%20HIV-1%20p24" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000920" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0000920&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'HLA Alleles Associated with Slow Progression to AIDS Truly Prefer to Present HIV-1 p24';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0000920';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20HLA%20Alleles%20Associated%20with%20Slow%20Progression%20to%20AIDS%20Truly%20Prefer%20to%20Present%20HIV-1%20p24 http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000920" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0000920" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">


























          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Antigens%22" title="Search for articles in the subject area:'Antigens'"><div class="flagText">Antigens</div></a>
              <div data-categoryid="32115" data-articleid="24678"
                   data-categoryname="Antigens"
                   class="flagImage" title="Flag 'Antigens' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22HIV%22" title="Search for articles in the subject area:'HIV'"><div class="flagText">HIV</div></a>
              <div data-categoryid="42663" data-articleid="24678"
                   data-categoryname="HIV"
                   class="flagImage" title="Flag 'HIV' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22HIV-1%22" title="Search for articles in the subject area:'HIV-1'"><div class="flagText">HIV-1</div></a>
              <div data-categoryid="42863" data-articleid="24678"
                   data-categoryname="HIV-1"
                   class="flagImage" title="Flag 'HIV-1' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Immune+response%22" title="Search for articles in the subject area:'Immune response'"><div class="flagText">Immune response</div></a>
              <div data-categoryid="17773" data-articleid="24678"
                   data-categoryname="Immune response"
                   class="flagImage" title="Flag 'Immune response' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Microbial+mutation%22" title="Search for articles in the subject area:'Microbial mutation'"><div class="flagText">Microbial mutation</div></a>
              <div data-categoryid="20011" data-articleid="24678"
                   data-categoryname="Microbial mutation"
                   class="flagImage" title="Flag 'Microbial mutation' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Sequence+databases%22" title="Search for articles in the subject area:'Sequence databases'"><div class="flagText">Sequence databases</div></a>
              <div data-categoryid="47501" data-articleid="24678"
                   data-categoryname="Sequence databases"
                   class="flagImage" title="Flag 'Sequence databases' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Sequence+motif+analysis%22" title="Search for articles in the subject area:'Sequence motif analysis'"><div class="flagText">Sequence motif analysis</div></a>
              <div data-categoryid="19657" data-articleid="24678"
                   data-categoryname="Sequence motif analysis"
                   class="flagImage" title="Flag 'Sequence motif analysis' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Viral+load%22" title="Search for articles in the subject area:'Viral load'"><div class="flagText">Viral load</div></a>
              <div data-categoryid="21755" data-articleid="24678"
                   data-categoryname="Viral load"
                   class="flagImage" title="Flag 'Viral load' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=8177'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=1038'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=4574&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>

<div class="block sidebar-comments">
    <div class="header">
        <h3>Comments</h3>
    </div>
      <p><a href="/annotation/listThread.action?root=17937">Updated Figure 3</a><br>Posted by PLoS_ONE_Group</p>
</div>

</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
