

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: The Evolution of Gene Expression QTL in Saccharomyces cerevisiae</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0000678"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0000678" />

    <meta name="citation_title" content="The Evolution of Gene Expression QTL in Saccharomyces cerevisiae"/>
    <meta itemprop="name" content="The Evolution of Gene Expression QTL in Saccharomyces cerevisiae"/>

      <meta name="citation_author" content="James Ronald"/>
            <meta name="citation_author_institution" content="Department of Genome Sciences, University of Washington, Seattle, Washington, United States of America"/>
      <meta name="citation_author" content="Joshua M. Akey"/>
            <meta name="citation_author_institution" content="Department of Genome Sciences, University of Washington, Seattle, Washington, United States of America"/>

    <meta name="citation_date" content="2007/8/1"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0000678.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e678"/>
    <meta name="citation_issue" content="8"/>
    <meta name="citation_volume" content="2"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Evolution of gene expression in the Drosophila melanogaster subgroup.; citation_author=SA Rifkin; citation_author=J Kim; citation_author=KP White; citation_journal_title=Nature Genetics; citation_volume=33; citation_number=1; citation_pages=138-144; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Population genetic variation in genome-wide gene expression.; citation_author=JP Townsend; citation_author=D Cavalieri; citation_author=DL Hartl; citation_journal_title=Molecular Biology and Evolution; citation_volume=20; citation_number=2; citation_pages=955-963; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Population genetic variation in gene expression is associated with phenotypic variation in Saccharomyces cerevisiae.; citation_author=JC Fay; citation_author=HL McCullough; citation_author=PD Sniegowski; citation_author=MB Eisen; citation_journal_title=Genome Biology; citation_volume=5; citation_number=3; citation_pages=R26; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=A neutral model of transcriptome evolution.; citation_author=P Khaitovich; citation_author=G Weiss; citation_author=M Lachmann; citation_author=I Hellmann; citation_author=W Enard; citation_journal_title=PLoS Biology; citation_volume=2; citation_number=4; citation_pages=e132; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=The transcriptional consequences of mutation and natural selection in Caenorhabditis elegans.; citation_author=DR Denver; citation_author=K Morris; citation_author=JT Streelman; citation_author=SK Kim; citation_author=M Lynch; citation_journal_title=Nature Genetics; citation_volume=37; citation_number=5; citation_pages=544-548; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Parallel patterns of evolution in the genomes and transcriptomes of humans and chimpanzees.; citation_author=P Khaitovich; citation_author=I Hellmann; citation_author=W Enard; citation_author=K Nowich; citation_author=M Leinweber; citation_journal_title=Science; citation_volume=309; citation_number=6; citation_pages=1805-1854; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=A mutation accumulation assay reveals a broad capacity for rapid evolution of gene expression.; citation_author=SA Rifkin; citation_author=D Houle; citation_author=J Kim; citation_author=KP White; citation_journal_title=Nature; citation_volume=438; citation_number=7; citation_pages=220-223; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Expression profiling in primates reveals a rapid evolution of human transcription factors.; citation_author=Y Gilad; citation_author=A Oshlack; citation_author=GK Smyth; citation_author=TP Speed; citation_author=KP White; citation_journal_title=Nature; citation_volume=440; citation_number=8; citation_pages=242-245; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Neutral and adaptive variation in gene expression.; citation_author=A Whitehead; citation_author=DL Crawford; citation_journal_title=Proceedings of the National Academy of Sciences U S A; citation_volume=103; citation_number=9; citation_pages=5425-5430; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Variation within and among species in gene expression: raw material for evolution.; citation_author=A Whitehead; citation_author=DL Crawford; citation_journal_title=Molecular Ecology; citation_volume=15; citation_number=10; citation_pages=1197-1211; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Genetic dissection of transcriptional regulation in budding yeast.; citation_author=RB Brem; citation_author=G Yvert; citation_author=R Clinton; citation_author=L Kruglyak; citation_journal_title=Science; citation_volume=296; citation_number=11; citation_pages=752-755; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Genetics of gene expression surveyed in maize, mouse and man.; citation_author=EE Schadt; citation_author=SA Monks; citation_author=TA Drake; citation_author=AJ Lusis; citation_author=N Che; citation_journal_title=Nature; citation_volume=422; citation_number=12; citation_pages=297-302; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Genetic analysis of genome-wide variation in human gene expression.; citation_author=M Morley; citation_author=CM Molony; citation_author=TM Weber; citation_author=JL Devlin; citation_author=KG Ewens; citation_journal_title=Nature; citation_volume=430; citation_number=13; citation_pages=743-747; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Genetic inheritance of gene expression in human cell lines.; citation_author=SA Monks; citation_author=A Leonardson; citation_author=H Zhu; citation_author=P Cundiff; citation_author=P Pietrusiak; citation_journal_title=American Journal of Human Genetics; citation_volume=75; citation_number=14; citation_pages=1094-1105; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Uncovering regulatory pathways that affect hematopoietic stem cell function using ‘genetical genomics’.; citation_author=L Bystrykh; citation_author=E Weersing; citation_author=B Dontje; citation_author=S Sutton; citation_author=MT Pletcher; citation_journal_title=Nature Genetics; citation_volume=37; citation_number=15; citation_pages=225-232; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Complex trait analysis of gene expression uncovers polygenic and pleiotropic networks that modulate nervous system function.; citation_author=EJ Chesler; citation_author=L Lu; citation_author=S Shou; citation_author=Y Qu; citation_author=J Gu; citation_journal_title=Nature Genetics; citation_volume=37; citation_number=16; citation_pages=233-242; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Integrated transcriptional profiling and linkage analysis for identification of genes underlying disease.; citation_author=N Hubner; citation_author=CA Wallace; citation_author=H Zimdahl; citation_author=E Petretto; citation_author=H Schulz; citation_journal_title=Nature Genetics; citation_volume=37; citation_number=17; citation_pages=243-253; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Combined expression trait correlations and expression quantitative trait locus mapping.; citation_author=H Lan; citation_author=M Chen; citation_author=JB Flowers; citation_author=BS Yandell; citation_author=DS Stapleton; citation_journal_title=PLoS Genetics; citation_volume=2; citation_number=18; citation_pages=e6; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Heritability and tissue specificity of expression quantitative trait loci.; citation_author=E Petretto; citation_author=J Mangion; citation_author=NJ Dickens; citation_author=SA Cook; citation_author=MK Kumaran; citation_journal_title=PLoS Genetics; citation_volume=2; citation_number=19; citation_pages=e172; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Transacting regulatory variation in Saccharomyces cerevisiae and the role of transcription factors.; citation_author=G Yvert; citation_author=RB Brem; citation_author=J Whittle; citation_author=JM Akey; citation_author=E Foss; citation_journal_title=Nature Genetics; citation_volume=35; citation_number=20; citation_pages=57-64; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Genetical genomics analysis of a yeast segregant population for transcription network inference.; citation_author=N Bing; citation_author=I Hoeschele; citation_journal_title=Genetics; citation_volume=170; citation_number=21; citation_pages=533-542; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Multiple locus linkage analysis of genomewide expression in yeast.; citation_author=JD Storey; citation_author=JM Akey; citation_author=L Kruglyak; citation_journal_title=PLoS Biology; citation_volume=3; citation_number=22; citation_pages=e267; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Local regulatory variation in Saccharomyces cerevisiae.; citation_author=J Ronald; citation_author=RB Brem; citation_author=J Whittle; citation_author=L Kruglyak; citation_journal_title=PLoS Genetics; citation_volume=1; citation_number=23; citation_pages=e25; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Identifying regulatory mechanisms using individual variation reveals key role for chromatin modification.; citation_author=SI Lee; citation_author=D Pe'er; citation_author=AM Dudley; citation_author=GM Church; citation_author=D Koller; citation_journal_title=Proceedings of the National Academy of Sciences U S A; citation_volume=103; citation_number=24; citation_pages=14062-14067; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Elevated evolutionary rates in the laboratory strain of Saccharomyces cerevisiae.; citation_author=Z Gu; citation_author=L David; citation_author=D Petrov; citation_author=T Jones; citation_author=RW Davis; citation_journal_title=Proceedings of the National Academy of Sciences U S A; citation_volume=102; citation_number=25; citation_pages=1092-1097; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Sequencing and comparison of yeast species to identify genes and regulatory elements.; citation_author=M Kellis; citation_author=N Patterson; citation_author=M Endrizzi; citation_author=B Birren; citation_author=ES Lander; citation_journal_title=Nature; citation_volume=423; citation_number=26; citation_pages=241-254; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Identification of functional transcription factor binding sites using closely related Saccharomyces species.; citation_author=SW Doniger; citation_author=J Huh; citation_author=JC Fay; citation_journal_title=Genome Research; citation_volume=15; citation_number=27; citation_pages=701-709; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Analysis of chimeric mRNAs derived from the STE3 mRNA identifies multiple regions within yeast mRNAs that modulate mRNA decay.; citation_author=B Heaton; citation_author=C Decker; citation_author=D Muhlrad; citation_author=J Donahue; citation_author=A Jacobson; citation_journal_title=Nucleic Acids Research; citation_volume=20; citation_number=28; citation_pages=5365-5373; citation_date=1992; " />
      <meta name="citation_reference" content="citation_title=Mutations affecting stability and deadenylation of the yeast MFA2 transcript.; citation_author=D Muhlrad; citation_author=R Parker; citation_journal_title=Genes and Development; citation_volume=6; citation_number=29; citation_pages=2100-2111; citation_date=1992; " />
      <meta name="citation_reference" content="citation_title=Turnover mechanisms of the stable yeast PGK1 mRNA.; citation_author=D Muhlrad; citation_author=CJ Decker; citation_author=R Parker; citation_journal_title=Molecular and Cellular Biology; citation_volume=15; citation_number=30; citation_pages=2145-2156; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Genome-wide evolutionary rates in laboratory and wild yeast.; citation_author=J Ronald; citation_author=H Tang; citation_author=R Brem; citation_journal_title=Genetics; citation_volume=174; citation_number=31; citation_pages=541-544; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Population genomic analysis of outcrossing and recombination in yeast.; citation_author=DM Ruderfer; citation_author=SC Pratt; citation_author=HS Seidel; citation_author=L Kruglyak; citation_journal_title=Nature Genetics; citation_volume=38; citation_number=32; citation_pages=1077-1081; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Positive and negative selection on the human genome.; citation_author=JC Fay; citation_author=GJ Wyckoff; citation_author=CI Wu; citation_journal_title=Genetics; citation_volume=158; citation_number=33; citation_pages=1227-1234; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Widespread purifying selection at polymorphic sites in human protein coding loci.; citation_author=AL Hughes; citation_author=B Packer; citation_author=R Welsh; citation_author=AW Bergen; citation_author=SJ Chanock; citation_journal_title=Proceedings of the National Academy of Sciences U S A; citation_volume=100; citation_number=34; citation_pages=15754-15757; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Evidence for abundant slightly deleterious polymorphisms in bacterial populations.; citation_author=AL Hughes; citation_journal_title=Genetics; citation_volume=169; citation_number=35; citation_pages=533-538; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Finding functional features in Saccharomyces genomes by phylogenetic footprinting.; citation_author=P Cliften; citation_author=P Sudarsanam; citation_author=A Desikan; citation_author=L Fulton; citation_author=B Fulton; citation_journal_title=Science; citation_volume=301; citation_number=36; citation_pages=71-76; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Transcriptional regulatory code of a eukaryotic genome.; citation_author=CT Harbison; citation_author=DB Gordon; citation_author=TI Lee; citation_author=NJ Rinaldi; citation_author=KD Macisaac; citation_journal_title=Nature; citation_volume=431; citation_number=37; citation_pages=99-104; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=The genealogy of samples in models with selection.; citation_author=C Neuhauser; citation_author=SM Krone; citation_journal_title=Genetics; citation_volume=145; citation_number=38; citation_pages=519-534; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Ancestral processes with selection.; citation_author=SM Krone; citation_author=C Neuhauser; citation_journal_title=Theoretical Population Biology; citation_volume=51; citation_number=39; citation_pages=210-237; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Detecting selection in noncoding regions of nucleotide sequences.; citation_author=WS Wong; citation_author=R Nielsen; citation_journal_title=Genetics; citation_volume=167; citation_number=40; citation_pages=949-958; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Genealogy of principle strains of the yeast genetic stock center.; citation_author=RK Mortimer; citation_author=JR Johnston; citation_journal_title=Genetics; citation_volume=113; citation_number=41; citation_pages=35-43; citation_date=1986; " />
      <meta name="citation_reference" content="citation_title=Near-neutrality in evolution of genes and gene regulation.; citation_author=T Ohta; citation_journal_title=Proceedings of the National Academy of Sciences U S A; citation_volume=99; citation_number=42; citation_pages=16134-16137; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Rates of divergence in gene expression profiles of primates, mice, and flies: stabilizing selection and variability among functional categories.; citation_author=B Lemos; citation_author=CD Meiklejohn; citation_author=M Caceres; citation_author=DL Hartl; citation_journal_title=Evolution: Internation Journal of Organic Evolution; citation_volume=59; citation_number=43; citation_pages=126-137; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Compensatory cis-trans evolution and the dysregulation of gene expression in interspecific hybrids of Drosophila.; citation_author=CR Landry; citation_author=PJ Wittkopp; citation_author=CH Taubes; citation_author=JM Ranz; citation_author=AG Clark; citation_journal_title=Genetics; citation_volume=171; citation_number=44; citation_pages=1813-1822; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=The landscape of genetic complexity across 5,700 gene expression traits in yeast.; citation_author=RB Brem; citation_author=L Kruglyak; citation_journal_title=Proceedings of the National Academy of Sciences U S A; citation_volume=102; citation_number=45; citation_pages=1572-1577; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Detecting natural selection on cis-regulatory DNA.; citation_author=MW Hahn; citation_journal_title=Genetica; citation_volume=129; citation_number=46; citation_pages=7-18; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=LAGAN and Multi-LAGAN: efficient tools for large-scale multiple alignment of genomic DNA.; citation_author=M Brudno; citation_author=C Do; citation_author=G Cooper; citation_author=MF Kim; citation_author=E Davydov; citation_journal_title=Genome Research; citation_volume=13; citation_number=47; citation_pages=721-731; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=CLUSTALW: Improving the sensitivity of progressive multiple alignment through sequence weighting, position-specific gap penalties and weight matrix choice.; citation_author=JD Thompson; citation_author=DG Higgins; citation_author=TJ Gibson; citation_journal_title=Nucleic Acids Research; citation_volume=22; citation_number=48; citation_pages=4673-4680; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=Statistical significance for genomewide studies.; citation_author=JD Storey; citation_author=R Tibshirani; citation_journal_title=Proceedings of the National Academy of Sciences U S A; citation_volume=100; citation_number=49; citation_pages=9440-9445; citation_date=2003; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000678" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="The Evolution of Gene Expression QTL in Saccharomyces cerevisiae"/>
    <meta name="twitter:description" content="Understanding the evolutionary forces that influence patterns of gene expression variation will provide insights into the mechanisms of evolutionary change and the molecular basis of phenotypic diversity. To date, studies of gene expression evolution have primarily been made by analyzing how gene expression levels vary within and between species. However, the fundamental unit of heritable variation in transcript abundance is the underlying regulatory allele, and as a result it is necessary to understand gene expression evolution at the level of DNA sequence variation. Here we describe the evolutionary forces shaping patterns of genetic variation for 1206 cis-regulatory QTL identified in a cross between two divergent strains of Saccharomyces cerevisiae. We demonstrate that purifying selection against mildly deleterious alleles is the dominant force governing cis-regulatory evolution in S. cerevisiae and estimate the strength of selection. We also find that essential genes and genes with larger codon bias are subject to slightly stronger cis-regulatory constraint and that positive selection has played a role in the evolution of major trans-acting QTL."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0000678.g006"/>

  <meta property="og:title" content="The Evolution of Gene Expression QTL in Saccharomyces cerevisiae" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000678" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=4528'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=8020'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=7420&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0000678">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000678" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000678&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    The Evolution of Gene Expression QTL in <i>Saccharomyces cerevisiae</i>
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            James Ronald, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Genome Sciences, University of Washington, Seattle, Washington, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Joshua M. Akey
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:akeyj@u.washington.edu">akeyj@u.washington.edu</a></p>

                <p>Affiliation:
                  Department of Genome Sciences, University of Washington, Seattle, Washington, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: August 01, 2007</li>
    <li>DOI: 10.1371/journal.pone.0000678</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0000678" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0000678" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000678" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0000678" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0000678-g001" data-doi="info:doi/10.1371/journal.pone.0000678" data-uri="info:doi/10.1371/journal.pone.0000678.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000678.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000678-g002" data-doi="info:doi/10.1371/journal.pone.0000678" data-uri="info:doi/10.1371/journal.pone.0000678.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000678.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000678-g003" data-doi="info:doi/10.1371/journal.pone.0000678" data-uri="info:doi/10.1371/journal.pone.0000678.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000678.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000678-g004" data-doi="info:doi/10.1371/journal.pone.0000678" data-uri="info:doi/10.1371/journal.pone.0000678.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000678.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000678-t001" data-doi="info:doi/10.1371/journal.pone.0000678" data-uri="info:doi/10.1371/journal.pone.0000678.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000678.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000678-g005" data-doi="info:doi/10.1371/journal.pone.0000678" data-uri="info:doi/10.1371/journal.pone.0000678.g005" title="Figure 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000678.g005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000678-g006" data-doi="info:doi/10.1371/journal.pone.0000678" data-uri="info:doi/10.1371/journal.pone.0000678.g006" title="Figure 6">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000678.g006&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000678">Reader Comments (1)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0000678" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1"></a><p>Understanding the evolutionary forces that influence patterns of gene expression variation will provide insights into the mechanisms of evolutionary change and the molecular basis of phenotypic diversity. To date, studies of gene expression evolution have primarily been made by analyzing how gene expression levels vary within and between species. However, the fundamental unit of heritable variation in transcript abundance is the underlying regulatory allele, and as a result it is necessary to understand gene expression evolution at the level of DNA sequence variation. Here we describe the evolutionary forces shaping patterns of genetic variation for 1206 <em>cis</em>-regulatory QTL identified in a cross between two divergent strains of <em>Saccharomyces cerevisiae</em>. We demonstrate that purifying selection against mildly deleterious alleles is the dominant force governing <em>cis</em>-regulatory evolution in <em>S. cerevisiae</em> and estimate the strength of selection. We also find that essential genes and genes with larger codon bias are subject to slightly stronger <em>cis</em>-regulatory constraint and that positive selection has played a role in the evolution of major <em>trans</em>-acting QTL.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>Ronald J, Akey JM (2007) The Evolution of Gene Expression QTL in <em>Saccharomyces cerevisiae</em>. PLoS ONE 2(8):
          e678.
            doi:10.1371/journal.pone.0000678</p><p><strong>Academic Editor: </strong>Matthew Hahn, Indiana University, United States of America</p><p><strong>Received:</strong> May 17, 2007; <strong>Accepted:</strong> June 20, 2007; <strong>Published:</strong> August 1, 2007</p><p><strong>Copyright:</strong> © 2007 Ronald, Akey. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>JMA is an Alfred P. Sloan research fellow. The sponsors played no role in any aspect of the project other than funding.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>Gene expression is the primary intermediate between information encoded by the genome and higher order phenotypes, and as a result expression variation is thought to be an important source of phenotypic diversity. Considerable effort has been devoted to characterizing patterns of natural variation in and the evolutionary trajectories of gene expression levels both within and between species <a href="#pone.0000678-Rifkin1">[1]</a>–<a href="#pone.0000678-Whitehead1">[9]</a>. A reoccurring observation in these studies is that transcript abundance varies considerably, with a significant amount of this variation attributable to heritable genetic changes that affect gene expression levels in a quantitative manner <a href="#pone.0000678-Whitehead2">[10]</a>. A powerful paradigm to emerge in studies of gene expression variation has been the combination of microarray technology and genetic mapping, allowing many gene expression QTL to be identified <a href="#pone.0000678-Brem1">[11]</a>–<a href="#pone.0000678-Petretto1">[19]</a>. Because these QTL point to regulatory polymorphisms that are the underlying units of heritable variation in transcript abundance, understanding the forces governing their evolution can provide detailed insights into gene expression diversity within populations and divergence between species.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>To study the evolutionary forces acting on regulatory polymorphisms, we took advantage of a large, well-studied data set of gene expression QTL discovered between the <em>S. cerevisiae</em> laboratory strain BY4716 (BY, isogenic to S288C) and the wild vineyard strain RM11-1a (RM) <a href="#pone.0000678-Brem1">[11]</a>, <a href="#pone.0000678-Yvert1">[20]</a>–<a href="#pone.0000678-Lee1">[24]</a>. We leveraged the available whole genome sequences of BY, RM, the clinical isolate YJM789 (YJM) <a href="#pone.0000678-Gu1">[25]</a>, and the outgroup <em>Saccharomyces paradoxus</em> <a href="#pone.0000678-Kellis1">[26]</a> to make inferences about the evolutionary forces acting on DNA sequence variation underlying regulatory QTL. Furthermore, we made use of the known ancestral history of S288C to identify a key recent departure from mutation-purifying selection-drift equilibrium in the regulatory program of laboratory yeast.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>Our analyses represent a first step toward applying population genetics models to a large set of QTL underlying variation in functional genomics phenotypes. As it becomes feasible to collect these data in large and cosmopolitan samples within a species, population genetics approaches and evolutionary modeling will become increasingly important and informative. Finally, although we focus on gene expression levels, our approaches provide the conceptual foundation for understanding the evolutionary forces shaping extant patterns of variation in other genetically tractable functional genomics phenotypes.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3>
<h4>Genomic distribution of regulatory QTL</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>In segregants derived from a cross between the BY and RM strains, a large number of gene expression levels show significant linkage to markers throughout the genome <a href="#pone.0000678-Brem1">[11]</a>, <a href="#pone.0000678-Yvert1">[20]</a>, <a href="#pone.0000678-Ronald1">[23]</a>. <a href="#pone-0000678-g001">Figure 1</a> shows the location of 2368 genes (out of 5067 total) that demonstrate linkage at a false discovery rate (FDR) ≤0.05 (see <a href="#pone.0000678.s001">Text S1</a>). Vertical bands indicate single major <em>trans</em>-acting QTL that influence large numbers of gene expression levels throughout the genome. The diagonal band represents QTL that are located coincident with the gene under inspection. We previously used allele-specific expression measurements and comparative sequence analysis to show that the majority of these QTL are due to <em>cis</em>-acting polymorphisms in the promoter and 3′ UTR of the corresponding gene <a href="#pone.0000678-Ronald1">[23]</a>. In this paper, we refer to genes whose expression levels show linkage coincident with their own location as <em>cis</em>-acting QTL and, although there appears to be a minor role for local <em>trans</em>-acting polymorphisms at these loci, we refer to the causative polymorphisms at these loci as <em>cis</em>-acting regulatory polymorphisms. In addition, our models are constructed to account for the contribution of nearby <em>trans</em>-acting QTL that occur on the same chromosome as the expression trait of interest, thus producing linkage that mimics <em>cis</em>-acting QTL. For convenience we refer to genes whose expression levels fail to show significant linkage to their own loci as genes without <em>cis</em>-regulatory variation, although in reality a sizable fraction of these genes are expected to harbor undetected <em>cis</em>-regulatory polymorphisms due to the incomplete power of the data set (see below). In the analyses described below we focus primarily on <em>cis</em>-acting QTL because each represents an independent evolutionary event, they are more abundant than <em>trans</em>-acting QTL (1206 <em>cis</em>-acting QTL, see <a href="#pone.0000678.s001">Text S1</a>, versus approximately 100–200 <em>trans</em>-acting QTL <a href="#pone.0000678-Yvert1">[20]</a>), are detected more reliably <a href="#pone.0000678-Petretto1">[19]</a>, and in contrast to <em>trans</em>-acting QTL their locations are known with more precision. Finally, it is important to emphasize that in contrast to an observed DNA polymorphism, a QTL is an estimated quantity, defined by statistically significant linkage between a trait and a particular genomic region. As a result, our models are constructed to account for the uncertainty inherent in the QTL detection process. Furthermore, we show that our estimates of key parameters are robust to various QTL detection thresholds.</p>
<div class="figure" id="pone-0000678-g001"><div class="img"><a name="pone-0000678-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000678.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000678" data-uri="info:doi/10.1371/journal.pone.0000678.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000678.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000678.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000678.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000678.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000678.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000678.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000678.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000678.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000678.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Genomic distribution of regulatory QTL between BY and RM.</span></strong></p><a id="article1.body1.sec2.sec1.fig1.caption1.p1" name="article1.body1.sec2.sec1.fig1.caption1.p1"></a><p>Location of the gene whose expression level is under inspection (vertical axis) versus marker location (horizontal axis) for 2368 trait marker pairs (points) with significant linkage at an experiment-wide permutation based FDR≤0.05. Identities of known major <em>trans</em>-acting QTL are listed above. Lower panels show the synonymous site substitution rate for the two chromosomal regions indicated by boxes. Breaks in the curves are due to the absence of synonymous sites in intergenic regions. As previously described, regions with low neutral substitution rates contain fewer <em>cis</em>-acting QTL <a href="#pone.0000678-Ronald1">[23]</a>.</p>
<span>doi:10.1371/journal.pone.0000678.g001</span></div>

<h4>Rate of accumulation of <em>cis</em>-acting QTL</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>A commonly used strategy to detect deviations from neutrality is to compare the rate of accumulation of putatively functional changes (e.g. non-synonymous substitutions) to putatively neutral changes (e.g. synonymous substitutions). Using this approach, we compared the observed rate of accumulation of <em>cis</em>-acting QTL to what is expected under neutrality. Loci throughout the genome show different levels of neutral substitution between BY and RM due to ancestral recombination (see <a href="#pone.0000678.s001">Text S1</a>), so if <em>cis</em>-acting QTL are selectively neutral, we expect them to accumulate at a clock-like rate based on their locus-specific coalescence time. Assuming that there are (on average) <em>n</em> regulatory sites per gene and that the coalescence time at locus <em>i</em> is <em>t<sub>i</sub></em> (measured in units of <em>N<sub>e</sub></em> generations), then the probability that a gene shows <em>cis</em>-acting expression variation is the probability that any regulatory site undergoes mutation<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000678.e001&amp;representation=PNG" class="inline-graphic"><span class="note">(1)</span></span><br>where θ = 2μ<em>N<sub>e</sub></em>. If some fraction, δ, of genes do not tolerate or have lost regulatory polymorphisms due to purifying selection, then the probability that a randomly chosen gene shows <em>cis</em>-acting expression variation is<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000678.e002&amp;representation=PNG" class="inline-graphic"><span class="note">(2)</span></span><br>For each locus the divergence time can be estimated via the synonymous substitution rate (see <a href="#s4">Methods</a> and <a href="#pone.0000678.s001">Text S1</a>), but the precise value of the locus-specific divergence time is unknown. Therefore, we compute the joint probability of observing <em>m<sub>i</sub></em> substitutions at <em>M<sub>i</sub></em> neutral sites and a <em>cis</em>-acting expression change at locus <em>i</em> by integrating over the full range of coalescence times, weighted by their probability density function <em>e<sup>−t</sup></em><a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000678.e003&amp;representation=PNG" class="inline-graphic"><span class="note">(3)</span></span><br>Similarly, the joint probability of observing <em>m<sub>i</sub></em> substitutions at <em>M<sub>i</sub></em> neutral sites and no <em>cis</em>-acting expression change at locus <em>i</em> is given by<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000678.e004&amp;representation=PNG" class="inline-graphic"><span class="note">(4)</span></span><br>Combining Equations 3 and 4 and the power, Prob(linkage|<em>cis</em>-acting QTL) (estimated to be 0.504, see <a href="#s4">Methods</a>), and false positive rate, Prob(linkage|no <em>cis</em>-acting QTL) (estimated to be 0.039, see <a href="#s4">Methods</a>), gives the likelihood of the pattern of genes showing significant linkage to <em>cis</em>-acting QTL<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000678.e005&amp;representation=PNG" class="inline-graphic"><span class="note">(5)</span></span><br>The neutral model (δ = 0) is nested in the parameter space at the boundary (0≤δ≤1). Therefore, twice the difference between the log<em><sub>e</sub></em> likelihoods of the purifying selection model and the neutral model is distributed as a <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000678.e006&amp;representation=PNG" class="inline-graphic"></span> distribution.</p>
<a id="article1.body1.sec2.sec2.p2" name="article1.body1.sec2.sec2.p2"></a><p>Using this approach, the maximum likelihood estimates of δ, <em>n</em>, and θ were 0.24 (95% CI 0.13–0.32, <a href="#pone-0000678-g002">Figure 2</a>), 144, and 0.009, with the purifying selection model representing a significant improvement over the fully neutral model (<em>p</em> = 1.2×10<sup>−4</sup>) where δ was constrained to zero and the estimates of <em>n</em> and θ were 85 and 0.009. Other QTL detection thresholds yielded quantitatively similar estimates (see <a href="#pone.0000678.s001">Text S1</a>). Note that an estimate of 144 regulatory sites per gene under the purifying selection model, although perhaps large, is in reasonable agreement with the observations that polymorphisms in both the promoter and 3′ UTR contribute to regulatory variation <a href="#pone.0000678-Ronald1">[23]</a>, that approximately 40% of intergenic sites in <em>S. cerevisiae</em> are subject to purifying selection <a href="#pone.0000678-Doniger1">[27]</a>, and that numerous sequence features appear to contribute to message stability <a href="#pone.0000678-Heaton1">[28]</a>–<a href="#pone.0000678-Muhlrad2">[30]</a>. To provide a more intuitive and graphical representation of the models, we fit Equations 1 and 2 to the data by a simple regression based alternative to the likelihood approach as shown in <a href="#pone-0000678-g003">Figure 3</a> (see <a href="#s4">Methods</a>).</p>
<div class="figure" id="pone-0000678-g002"><div class="img"><a name="pone-0000678-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000678.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000678" data-uri="info:doi/10.1371/journal.pone.0000678.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000678.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000678.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000678.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000678.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000678.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000678.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000678.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000678.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000678.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Likelihood surface for δ and <em>n</em>.</span></strong></p><a id="article1.body1.sec2.sec2.fig1.caption1.p1" name="article1.body1.sec2.sec2.fig1.caption1.p1"></a><p>Black contours show the 63%, 95%, and 99% confidence intervals for the joint value of δ and <em>n</em> (based on the χ<sup>2</sup><sub>2</sub> distribution with Δ log<em><sub>e</sub></em> likelihood = 1, 3, and 5, respectively). Gray contours correspond to Δ log<em><sub>e</sub></em> likelihood = 10, 15, 20, …, 100.</p>
<span>doi:10.1371/journal.pone.0000678.g002</span></div><a id="article1.body1.sec2.sec2.p3" name="article1.body1.sec2.sec2.p3"></a><p>We considered other possible explanations for the poor fit of the neutral model relative to the purifying selection model, including inability to detect linkages for some genes due to low expression, variation in the number of regulatory sites per gene, and microarray hybridization artifact, but these alternative models appear to be less plausible than the purifying selection model (see <a href="#pone.0000678.s001">Text S1</a>). It is important to note that the estimate of δ was sensitive to the estimated power to detect <em>cis</em>-acting QTL by linkage analysis and to a lesser extent to the estimated false-positive rate. This sensitivity is to be expected, since the power and false-positive rate determine how well the observed pattern of significant linkages captures the true underlying pattern of <em>cis</em>-acting regulatory variation. However, as described in <a href="#pone.0000678.s001">Text S1</a>, further analyses suggest that our estimate of the power is conservative making the estimate of δ an underestimate.</p>


<h4>Allele frequency distribution of <em>cis</em>-regulatory polymorphisms</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>The estimated value of δ does not imply that 24% of genes are without <em>cis</em>-acting regulatory variation. Instead, this estimate reflects that regulatory evolution is proceeding slower than the neutral prediction and that at any given level of divergence, approximately 24% fewer genes show <em>cis</em>-acting expression variation than would be expected if these expression changes were selectively neutral. This deficiency could be due to strong purifying selection against expression changes in a subset of genes, persistent weak purifying selection against expression changes in most genes, or a combination of these possibilities.</p>
<a id="article1.body1.sec2.sec3.p2" name="article1.body1.sec2.sec3.p2"></a><p>In order to understand the relative contributions of these processes, we evaluated the allele frequency distribution of existing regulatory polymorphisms segregating in 932 out of the 1206 genes with <em>cis</em>-acting regulatory variation for which we could identify orthologs in BY, RM, and YJM (which can be regarded as a randomly mating, recombining population <a href="#pone.0000678-Ronald2">[31]</a>, <a href="#pone.0000678-Ruderfer1">[32]</a>; see also <a href="#pone.0000678.s002">Figure S1</a> and <a href="#pone.0000678.s001">Text S1</a>) and the outgroup <em>S. paradoxus</em>. We determined whether the frequency distribution of <em>cis</em>-acting regulatory polymorphisms was skewed toward rare derived alleles, which tend to be recent and occur in sites otherwise conserved in both <em>Saccharomyces</em> lineages. Such an approach has previously been used as an indicator of weak purifying selection <a href="#pone.0000678-Fay2">[33]</a>-<a href="#pone.0000678-Hughes2">[35]</a>. We classified derived alleles between BY and RM as rare if either the BY or RM allele was observed in both YJM and <em>S. paradoxus</em> or as common if both the BY and RM alleles were observed in YJM and <em>S. paradoxus</em> (<a href="#pone-0000678-g004">Figure 4</a>).</p>
<div class="figure" id="pone-0000678-g003"><div class="img"><a name="pone-0000678-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000678.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000678" data-uri="info:doi/10.1371/journal.pone.0000678.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000678.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000678.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000678.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000678.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000678.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000678.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000678.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000678.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000678.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Rate of accumulation of <em>cis</em>-acting QTL.</span></strong></p><a id="article1.body1.sec2.sec3.fig1.caption1.p1" name="article1.body1.sec2.sec3.fig1.caption1.p1"></a><p>Genes were divided into bins based on their locus-specific maximum likelihood estimate of θ×<em>t<sub>i</sub></em>. The rate of <em>cis</em>-acting QTL in each bin (points, 95% CIs shown in vertical lines) was estimated based on the observed number of genes with linkage and the estimated power and false positive rate of linkage analysis. The least squares fit of the purifying selection model to the points (solid line) results in estimates of δ and <em>n</em> of 0.31 and 181 which are somewhat larger but not significantly different from the estimates obtained under the likelihood based approach. The dashed line shows the least squares fit of the neutral model, yielding <em>n</em> = 71.</p>
<span>doi:10.1371/journal.pone.0000678.g003</span></div><div class="figure" id="pone-0000678-g004"><div class="img"><a name="pone-0000678-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000678.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000678" data-uri="info:doi/10.1371/journal.pone.0000678.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000678.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000678.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000678.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000678.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000678.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000678.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000678.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000678.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000678.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>Illustration of common and rare derived alleles between BY and RM.</span></strong></p><a id="article1.body1.sec2.sec3.fig2.caption1.p1" name="article1.body1.sec2.sec3.fig2.caption1.p1"></a><p>The three possible rooted tree topologies for the <em>S. cerevisiae</em> strains are shown with branch lengths approximately to scale. Hypothetical genotypes for a polymorphism between BY and RM are given below. Orange and blue points represent mutations between BY and RM that result in common and rare derived alleles, respectively. Note that repeat mutation leads to apparently common derived alleles as illustrated for the right most topology.</p>
<span>doi:10.1371/journal.pone.0000678.g004</span></div><a id="article1.body1.sec2.sec3.p3" name="article1.body1.sec2.sec3.p3"></a><p>We found that polymorphisms in the key <em>cis</em>-acting regulatory regions of these genes (the promoter region from 101 to 200 bases upstream of translation start <a href="#pone.0000678-Ronald1">[23]</a>, <a href="#pone.0000678-Cliften1">[36]</a>, <a href="#pone.0000678-Harbison1">[37]</a> and the 3′ UTR from 1 to 100 bases downstream of translation stop <a href="#pone.0000678-Ronald1">[23]</a>) were skewed toward rare derived alleles relative to synonymous site polymorphisms in the same 932 genes, consistent with the action of purifying selection (<a href="#pone-0000678-t001">Table 1</a>). Approximately 68% of promoter and 3′ UTR derived alleles between BY and RM in genes with <em>cis</em>-acting regulatory variation are classified as rare compared to 61% of synonymous derived alleles (Fisher's exact test, <em>p</em> = 9.0×10<sup>−5</sup>). This skew drops off in intergenic regions around the consensus yeast promoter and is absent in more distal downstream intergenic regions (<a href="#pone-0000678-t001">Table 1</a>). In addition, we found that for promoter polymorphisms the skew toward derived alleles was larger in genes with <em>cis</em>-regulatory changes, suggesting that purifying selection acts more strongly to restrict large perturbations in gene expression (<a href="#pone.0000678.s003">Figure S2</a>).</p>
<div class="figure" id="pone-0000678-t001"><div class="img"><a name="pone-0000678-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000678.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000678" data-uri="info:doi/10.1371/journal.pone.0000678.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000678.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000678.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000678.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000678.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000678.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000678.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000678.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000678.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000678.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1. </strong></p><span>doi:10.1371/journal.pone.0000678.t001</span></div><a id="article1.body1.sec2.sec3.p4" name="article1.body1.sec2.sec3.p4"></a><p>For comparison, we performed the same analysis on 932 genes without linkage to <em>cis</em>-acting QTL located within 5 kb of each of the genes with <em>cis</em>-regulatory variation to approximate the same distribution of tree topologies and branch lengths in the two sets of genes (<a href="#pone-0000678-t001">Table 1</a>). We also performed the analysis for 2352 genes without <em>cis</em>-regulatory variation located elsewhere in the genome. Although there is a significant excess of rare derived alleles in the promoter and 3′ UTR relative to synonymous sites in these genes, some skew is to be expected given our power to detect <em>cis</em>-regulatory effects is incomplete. In addition, it is likely that some of these polymorphisms lead to <em>cis</em>-acting expression variation between BY and RM under other growth and environmental conditions, and a skew toward rare derived alleles is consistent with the action of weak purifying selection on such environmentally dependent regulatory sites. Interestingly, the skew towards rare alleles is more extreme for both intergenic and non-synonymous changes in the set of 2352 genes located distant from detected <em>cis</em>-acting QTL. The explanation for this effect is presumably due to the shallower genealogies across these 2352 loci, in which the mean number of changes per synonymous site was 0.0067 as compared to 0.012 in the 932 genes with <em>cis</em>-regulatory variation and the 0.011 in the 932 adjacent genes without <em>cis</em>-regulatory variation. These shallower genealogies imply approximately half the level neutral variation, and hence a higher ratio of mildly deleterious to neutral changes at these loci as evidenced by the significantly elevated rare derived allele skew in non-synonymous changes.</p>
<a id="article1.body1.sec2.sec3.p5" name="article1.body1.sec2.sec3.p5"></a><p>In spite of the skew toward rare derived alleles in putative regulatory regions of genes without statistically significant <em>cis</em>-regulatory variation, it is notable that the effect is significantly less extreme than for non-synonymous changes in these genes. In genes with detectable <em>cis</em>-regulatory variation, derived alleles show a skew similar to non-synonymous changes, suggesting that the regulatory mutations we detect may be associated with fitness costs commensurate with non-synonymous mutations.</p>


<h4>Modeling the allele frequency skew under the ancestral selection graph</h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>To estimate the strength of selection giving rise to the observed rare derived allele skew, we performed simulations under the ancestral selection graph, an extension of the coalescent that incorporates natural selection <a href="#pone.0000678-Neuhauser1">[38]</a>, <a href="#pone.0000678-Krone1">[39]</a> (see <a href="#s4">Methods</a> and <a href="#pone-0000678-g005">Figure 5</a>). Each simulation included a selected site (with fitness values in terms of the selection coefficient 2<em>N<sub>e</sub>s</em>) representing a <em>cis</em>-regulatory site and a linked neutral site representing a synonymous site. For each realized genealogy in the simulations, if the two sampled individuals representing BY and RM were polymorphic with respect to each other at the selected (or neutral) site, we determined whether the derived allele was common or rare as defined above. <a href="#pone-0000678-g006">Figure 6</a> shows the average proportion of rare derived alleles at the selected and neutral sites as a function of the scaled fitness difference between selective classes. The value of the scaled fitness difference between selective classes which best fits the observed skew for regulatory polymorphisms is 2.1 (95% CI based on the binomial distribution approximately 1.6–2.7). Note that this is likely an underestimate because in the observed data some synonymous sites may not be selectively neutral and because the magnitude of the observed allele frequency skew may be diminished by neutral polymorphisms in the promoter region and 3′ UTR that do not affect regulatory sites.</p>
<div class="figure" id="pone-0000678-g005"><div class="img"><a name="pone-0000678-g005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000678.g005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000678" data-uri="info:doi/10.1371/journal.pone.0000678.g005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000678.g005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000678.g005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000678.g005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000678.g005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000678.g005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000678.g005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000678.g005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000678.g005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000678.g005.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 5.  <span>Ancestral selection graph simulation scheme.</span></strong></p><a id="article1.body1.sec2.sec4.fig1.caption1.p1" name="article1.body1.sec2.sec4.fig1.caption1.p1"></a><p>The left panel shows a percolation diagram illustrating the underlying Moran model with neutral births realized by all individuals (δ arrows) and extra births realized only by fitter individuals (<em>2</em>, <em>3</em>, and <em>4</em> arrows) (see <a href="#pone.0000678-Neuhauser1">[38]</a>, <a href="#pone.0000678-Krone1">[39]</a> for detailed discussion). The right panel shows a realization of the reverse time simulation process for four sampled individuals representing the three <em>S. cerevisiae</em> strains and <em>S. paradoxus</em>. After mutations have been placed on the graph, branching events are resolved depending on the fitness of the two potential ancestors. Resolution of branching events produces a typical coalescent tree but introduces a bias towards advantageous alleles.</p>
<span>doi:10.1371/journal.pone.0000678.g005</span></div><div class="figure" id="pone-0000678-g006"><div class="img"><a name="pone-0000678-g006" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000678.g006&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000678" data-uri="info:doi/10.1371/journal.pone.0000678.g006"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000678.g006&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000678.g006/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000678.g006/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000678.g006/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000678.g006/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000678.g006.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000678.g006/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000678.g006/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000678.g006.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 6.  <span>Strength of purifying selection against <em>cis</em>-acting regulatory changes.</span></strong></p><a id="article1.body1.sec2.sec4.fig2.caption1.p1" name="article1.body1.sec2.sec4.fig2.caption1.p1"></a><p>Light gray shaded areas indicate that 95% CIs for the proportion of rare derived alleles (vertical axis) in synonymous sites and in the promoter and 3′ UTR. Ninety-five percent CIs for the expected proportion of rare derived alleles at selected sites (dark gray shading) and linked neutral sites (black shading) are shown as a function of the scaled fitness difference between selective classes (horizontal axis). The dashed line indicates the scaled purifying selection coefficient (2.1) that is most likely to have produced the observed allele frequency skew based on linear interpolation between 2<em>N<sub>e</sub>s</em> = 2.0 and 2.2. The rate of substitution at the selected site relative to the linked neutral site, denoted by ζ, is indicated along with the 95% CI.</p>
<span>doi:10.1371/journal.pone.0000678.g006</span></div><a id="article1.body1.sec2.sec4.p2" name="article1.body1.sec2.sec4.p2"></a><p>These simulations also allowed us to ask at what rate mildly deleterious alleles are lost between BY and RM. Comparing the number of times the selected site was polymorphic between BY and RM to the number of times the linked neutral site was polymorphic, we estimate that with a scaled fitness difference between selective classes of 2.1, the substitution rate at selected sites relative to neutral sites (analogous to <em>d<sub>I</sub></em>/<em>d<sub>S</sub></em> , the intergenic regulatory substitution rate to synonymous substitution rate and denoted by ζ <a href="#pone.0000678-Wong1">[40]</a>) was 69% (95% CI 60%–78% based on the 95% CI for 2<em>N<sub>e</sub>s</em>; see <a href="#s4">Methods</a>). We considered additional selection models (see <a href="#pone.0000678.s001">Text S1</a>) and found that under these other models estimates of ζ were at most 0.73. If the substitution rate at regulatory sites is approximately 0.7 times the neutral substitution rate, then the probability that a gene shows <em>cis</em>-acting regulatory variation would be<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000678.e007&amp;representation=PNG" class="inline-graphic"></span><br> Note that this differs from Equation 2 because in analyzing the rate of accumulation of <em>cis</em>-acting QTL it is not possible to estimate both the number of regulatory sites and a mutation rate multiplier at these sites simultaneously. Both models are similar, however, in that they describe a slower rate of accumulation of <em>cis</em>-acting QTL relative to neutrality. Using the estimated values of <em>n</em> and θ obtained above, the expected deficiency in the rate of <em>cis</em>-acting QTL across all loci would be<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000678.e008&amp;representation=PNG" class="inline-graphic"></span><br> Therefore, based on the mutation-weak purifying selection-drift equilibrium predicted by the patterns of genetic variation apparent at existing <em>cis</em>-acting QTL, we predict there should be 16% fewer such QTL than under neutrality. Above, we estimated a 24% deficiency in the number of <em>cis</em>-acting QTL based on their rate of accumulation across all genes. Thus, the strength of purifying selective acting on extant <em>cis</em>-acting polymorphisms would be expected to produce nearly the same deficiency in the number of <em>cis</em>-acting QTL observed across all genes.</p>


<h4>Features associated with slower <em>cis</em>-regulatory evolution</h4>
<a id="article1.body1.sec2.sec5.p1" name="article1.body1.sec2.sec5.p1"></a><p>As described above, weak purifying selection appears to be sufficient to explain much of the deficiency in <em>cis</em>-acting QTL. Nonetheless, in order to determine whether certain classes of genes were subject to stronger selective pressure against expression changes than others, we determined whether any particular features were associated with <em>cis</em>-acting expression changes. There was a slight but significant deficiency in the number of essential genes among genes with <em>cis</em>-regulatory variation (16% versus 20% in genes lacking <em>cis</em>-regulatory variation, Fisher's exact test, <em>p</em> = 0.0034). There was also significantly less codon bias among genes with <em>cis</em>-regulatory variation compared to genes without (mean codon bias 0.095 versus 0.125, Wilcoxon rank test, <em>p</em> = 3.4×10<sup>−4</sup>). We did not find a significant difference in tolerance to amino acid change (comparing <em>d<sub>N</sub></em>/<em>d<sub>S</sub></em> among <em>Saccharomyces sensu stricto</em> species <a href="#pone.0000678-Kellis1">[26]</a> in genes with versus without <em>cis</em>-regulatory variation, data not shown) suggesting that structural constraint does not necessarily predict gene expression constraint.</p>


<h4>Evolution of major <em>trans</em>-acting regulatory QTL</h4>
<a id="article1.body1.sec2.sec6.p1" name="article1.body1.sec2.sec6.p1"></a><p>In contrast to our estimates of a 16%–24% deficiency in the number of <em>cis</em>-acting QTL, approximately 94% of genes show heritable expression variation between BY and RM (see <a href="#pone.0000678.s001">Text S1</a>), with many linking to major <em>trans</em>-acting QTL as shown in <a href="#pone-0000678-g001">Figure 1</a>. If most individual expression changes (as mediated by <em>cis</em>-acting polymorphisms) are mildly deleterious, then we expect that the cumulative selective effects against major <em>trans</em>-acting QTL would be so strong that they would be rapidly eliminated from the population. The existence of such major <em>trans</em>-acting regulatory QTL therefore suggests that alleles at these loci may confer some selective benefit to mitigate the deleterious effects on gene expression. There are two examples of such QTL that support this hypothesis. The first is the trivial case of the <em>leu2</em>Δ<em>0</em> allele (<a href="#pone-0000678-g001">Figure 1</a>), which experienced selection in the laboratory as an auxotrophic marker. The second is <em>AMN1</em> in which the D368V loss of function mutation in BY is responsible for widespread gene expression changes <a href="#pone.0000678-Yvert1">[20]</a>, <a href="#pone.0000678-Ronald1">[23]</a>. Loss of function of <em>AMN1</em> also causes cellular dispersal in BY, rather than the clumpy growth observed in RM <a href="#pone.0000678-Yvert1">[20]</a>. BY is derived from the strain S288C, and as described by Robert Mortimer, “Conditions established for this strain [S288C] were that it be nonclumpy (nonflocculent) - i.e., dispersed as single cells in liquid culture…” <a href="#pone.0000678-Mortimer1">[41]</a>. We sequenced <em>AMN1</em> in the available S288C natural isolate progenitor strains EM93, EM126, NRLL YB-210, and “Yeast Foam” <a href="#pone.0000678-Mortimer1">[41]</a> (which account for approximately 95% of the S288C ancestry) and found that they possess the aspartate allele at residue 368 rather than valine, suggesting that this novel major <em>trans</em>-acting regulatory allele is unlikely to be found in the wild but instead was fixed during Mortimer's selection for cellular dispersal. Thus, the phenotypic benefits of leucine requirement and cellular dispersal in the laboratory apparently facilitated the emergence of widespread gene expression changes, which might otherwise be intolerable to the cell in the wild.</p>

</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p>By analyzing a large set of <em>cis</em>-acting QTL discovered between divergent yeast strains, we have provided an initial description of the evolutionary forces acting on gene expression QTL. We have shown that <em>cis</em>-acting QTL accumulate more slowly than expected under neutrality and that the underlying regulatory polymorphisms are skewed toward rare derived alleles. Thus, weak purifying selection against expression polymorphisms appears to be a pervasive force acting on gene expression levels for yeast in log phase growth in rich media. We estimated a scaled selection coefficient of ≈2 for typical <em>cis</em>-regulatory changes, indicating that selection, though detectable, is rather weak with stochastic forces playing a significant role in shaping patterns of within population <em>cis</em>-regulatory diversity. Under such a nearly neutral regime, unpreferred <em>cis</em>-regulatory alleles are expected to be present at appreciable frequencies, and an interplay of forces, including changes in effective population size, linkage disequilibrium among selected alleles, and epistatic selection <a href="#pone.0000678-Ohta1">[42]</a>, is expected to figure prominently in <em>cis</em>-regulatory evolution over longer time periods.</p>
<a id="article1.body1.sec3.p2" name="article1.body1.sec3.p2"></a><p>Given the widespread weak purifying selection on gene expression, the existence of major <em>trans</em>-acting regulatory alleles that affect hundreds of genes throughout the genome is surprising. We proposed that deviations from the normal mutation-purifying selection-drift regime may allow such alleles to persist in the population. At these loci, we hypothesized that the deleterious transcriptional effects of novel major <em>trans</em>-acting regulatory alleles tend to be balanced by beneficial phenotypic effects of these same alleles.</p>
<a id="article1.body1.sec3.p3" name="article1.body1.sec3.p3"></a><p>There are several important caveats to our analyses. First, as described above and in <a href="#pone.0000678.s001">Text S1</a> our analyses of the rate of accumulation of <em>cis</em>-acting QTL were sensitive to the estimated power of the linkage analyses and to a lesser extent to the estimated false-positive rate. As larger expression QTL data sets are collected, better estimates of these quantities can be obtained, allowing for more precise estimation of the extent to which purifying selection affects gene expression QTL. Second, although we interpreted the observed excess of rare derived <em>cis</em>-regulatory alleles as further evidence of weak purifying selection, some fraction of these rare alleles may be due to fixation of beneficial regulatory changes within divergent <em>S. cerevisiae</em> strains due to positive selection. As described below, we expect that weak selection to maintain gene expression stability may result in positive selection on certain compensatory regulatory changes in spite of the fact that most novel alleles are likely to be deleterious.</p>
<a id="article1.body1.sec3.p4" name="article1.body1.sec3.p4"></a><p>Although our analyses were based on a small number of yeast strains, they make several predictions about the pattern of expression QTL that might be observed among strains in the yeast population. First, based on our conclusion that most <em>cis</em>-acting QTL are mildly deleterious, we expect that among any pairwise comparison of strains, fewer <em>cis</em>-acting QTL would be present than predicted under neutrality. Second, selection against extant <em>cis</em>-regulatory alleles is rather weak, we would not expect to observe a set of genes that have invariant <em>cis</em>-regulation across numerous yeast strains. Instead, we predict that most or all genes are likely to show <em>cis</em>-regulatory polymorphism in the global population of yeast strains. Finally, we provided anecdotal evidence that positive selection may allow major <em>trans</em>-acting regulatory QTL to emerge and persist in the population. In addition to positive selection, demographic perturbations such as bottlenecks or population structure may lead to the emergence of novel major regulatory alleles. However, in the absence of such forces, we would expect few major <em>trans</em>-acting regulatory QTL. Indeed, in outbred populations such as humans, the existence of major <em>trans</em>-acting QTL is controversial (contrast <a href="#pone.0000678-Morley1">[13]</a> with <a href="#pone.0000678-Monks1">[14]</a>).</p>
<a id="article1.body1.sec3.p5" name="article1.body1.sec3.p5"></a><p>In the broader context of gene expression evolution, recent studies in a variety of species have found widespread signatures of purifying selection in patterns of gene expression variation <a href="#pone.0000678-Rifkin1">[1]</a>, <a href="#pone.0000678-Denver1">[5]</a>, <a href="#pone.0000678-Rifkin2">[7]</a>, <a href="#pone.0000678-Lemos1">[43]</a>. Our results provide the first comprehensive evaluation of the evolutionary forces acting upon regulatory QTL and suggest that the evolutionarily stable expression patterns observed at the level of overall transcript abundance are due to persistent weak purifying selection acting against novel regulatory alleles in most genes. Studying gene expression evolution in terms of the underlying regulatory QTL is an important first step towards a more detailed and quantitative understanding of the forces governing regulatory evolution and allows new hypotheses to be explored. For example, if selection acts to constrain gene expression levels to an optimal level but novel regulatory alleles persist with long sojourn times due to weak purifying selection, then compensatory regulatory evolution <a href="#pone.0000678-Landry1">[44]</a> may be common. Compensatory fixation of additive regulatory alleles with opposing effects would be expected to result high levels of transgressive segregation, consistent with previous observations <a href="#pone.0000678-Brem2">[45]</a>. In addition to coevolution among regulatory alleles, it has also been suggested that <em>cis</em>-regulatory alleles may coevolve with alleles of the associated protein as a mechanism for titrating gene activity <a href="#pone.0000678-Hahn1">[46]</a>. More generally, our results confirm theoretical predictions that mildly deleterious regulatory QTL segregate in natural populations <a href="#pone.0000678-Ohta1">[42]</a> and raise the possibility that these polymorphisms contribute to phenotypic diversity.</p>
</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4>Strains, expression data, and sequence analysis</h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p>Strains BY4716, RM11-1a, and YJM789 and <em>S. paradoxus</em> have been described elsewhere <a href="#pone.0000678-Brem1">[11]</a>, <a href="#pone.0000678-Yvert1">[20]</a>, <a href="#pone.0000678-Gu1">[25]</a>, <a href="#pone.0000678-Kellis1">[26]</a>. Whole genome sequences for BY (isogenic to S288C), RM and <em>S. paradoxus</em>, and YJM were obtained from the Saccharomyces Genome Database (<a href="http://www.yeastgenome.org/">http://www.yeastgenome.org/</a>), the Broad Institute (<a href="http://www.broad.mit.edu/annotation/fungi/fgi/">http://www.broad.mit.edu/annotation/fung​i/fgi/</a>), and the Stanford Genome Technology Center (<a href="http://www-sequence.stanford.edu/">http://www-sequence.stanford.edu/</a> yjm789 public/), respectively. Gene expression measurements and genotypes in 112 segregants in the cross between BY and RM are from Brem and Kruglyak <a href="#pone.0000678-Brem2">[45]</a>. Whole genome linkage analyses and tests for <em>cis</em>-acting regulatory variation are described in <a href="#pone.0000678.s001">Text S1</a> and in Ronald et al. (2005) <a href="#pone.0000678-Ronald1">[23]</a>. To estimate the locus-specific coalescence time for each gene we created whole chromosome alignments for BY and RM using LAGAN <a href="#pone.0000678-Brudno1">[47]</a> as described in <a href="#pone.0000678.s001">Text S1</a>. We identified orthologous genes by reciprocal best match using CROSSMATCH (<a href="http://bozeman.mbt.washington">http://bozeman.mbt.washington</a>. edu/phrap.docs/phrap.html) and performed alignments of BY, RM, YJM, and <em>S. paradoxus</em> genes and intergenic regions using CLUSTALW <a href="#pone.0000678-Thompson1">[48]</a>. We purchased strains EM93, EM126, NRLL YB-210, and “Yeast Foam” from America Type Culture Collection (catalog #204501), Herman J Phaff Culture Collection (#40-126), and Centraalbureau voor Schimmelcultures (#6333 and #1428), respectively. Primer sequences for <em>AMN1</em> were 5′-CCAAAGGAAAGACCATGCTT-3′ and 5′-CTAGCGCGACCAGTGAGAC-3′.</p>


<h4>Power and false-positive rate to detect <em>cis</em>-acting QTL by linkage analysis</h4>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1"></a><p>Using linkage analysis to detect <em>cis</em>-acting QTL involves a false-positive and false-negative rate. In order to estimate these quantities, we must compare the number of statistically significant linkage tests with the estimated number of truly null and truly alternative linkage tests. We must also account for apparent linkages to <em>cis</em>-acting polymorphisms which occur instead because of linked <em>trans</em>-acting regulatory genes that are on the same chromosome but distinct from the locus in question. Such QTL are problematic for the model because their locations, and hence their locus-specific coalescence times, are unknown. To estimate these quantities, we used the method of Storey and Tibshirani to analyze the complete distribution of <em>p</em>-values to estimate the overall proportion of truly alternative tests <a href="#pone.0000678-Storey2">[49]</a>.</p>
<a id="article1.body1.sec4.sec2.p2" name="article1.body1.sec4.sec2.p2"></a><p>First, we estimated the proportion of truly null tests (denoted π<sub>0</sub>) and truly alternative tests (1−π<sub>0</sub>) across all 5067 single marker linkage tests performed at the marker closest to the locus of the gene in question. For this set of tests, π<sub>0</sub>≈0.514 suggesting that approximately 2464/5067 of tests are truly alternative in the sense that a regulatory QTL affecting the expression level of the gene in question is linked to the marker locus. These regulatory QTL include both the <em>cis</em>-acting QTL that we are interested in as well as linked <em>trans</em>-acting QTL on the same chromosome. Assuming that <em>trans</em>-acting regulatory QTL are distributed randomly with respect to their target genes, we estimated the rate at which these single marker linkage tests detect linked <em>trans</em>-acting regulatory QTL by testing each gene expression trait for linkage to randomly chosen markers in the genome <a href="#pone.0000678-Ronald1">[23]</a>. The rate at which regulatory QTL are detected among these tests provides an estimate of the genome-wide prevalence of polymorphic <em>trans</em>-acting regulatory loci between BY and RM. The estimate of π<sub>0</sub> for this set of tests is approximately 0.898, indicating that about 518/5067 of these single marker linkage tests at random loci detect true <em>trans</em>-acting QTL. Thus,</p>
<pre>
	Prob(<em>cis</em>-acting QTL ∪ linked <em>trans</em>-acting QTL) = 
       	Prob(<em>cis</em>-acting QTL) + Prob(linked <em>trans</em>-acting QTL)
       	- Prob(<em>cis</em>-acting QTL) · Prob(linked <em>trans</em>-acting QTL) ≈ 2464/5067

	Prob(linked <em>trans</em>-acting QTL) ≈ 518/5067
	Prob(<em>cis</em>-acting QTL) ≈ 0.428
</pre><a id="article1.body1.sec4.sec2.p3" name="article1.body1.sec4.sec2.p3"></a><p>In order to estimate the rate of true <em>cis</em>-acting QTL among the set of genes with statistically significant linkage, we must subtract out two types of false signals: linkages to nearby <em>trans</em>-acting QTL as described above and statistical false-positives in which neither a <em>cis</em>-acting nor a linked <em>trans</em>-acting QTL exists, but instead the gene expression level shows a spurious correlation with the marker. The rate at which statistical false-positives occur can be estimated by the FDR. For the 1206 linkage tests significant at LOD≥1.37, the FDR estimated by Storey's and Tibshirani's method was 0.026. Thus, approximately 1175 out of these 1206 significant tests are expected to be truly alternative in the sense that a <em>cis</em>-acting or <em>trans</em>-acting QTL is linked to the marker in question. There were 157 linkages called significant at LOD≥1.37 among the 5067 random marker linkage tests, and the associated FDR was ≈0.342. Thus, approximately 103 true-positive <em>trans</em>-acting QTL are detected in the 5067 random single marker linkage tests. Using these estimates the rate at which true <em>cis</em>-acting QTL are called significant can be calculated as follows</p>
<pre>
	Prob(significant linkage∩(<em>cis</em>-acting QTL ∪ linked <em>trans</em>-acting QTL)) = 
       Prob(linkage∩<em>cis</em>-acting QTL)+Prob(linkage∩linked<em>trans</em>-acting QTL)
       – Prob(linkage∩<em>cis</em>-acting QTL) ·Prob(linkage∩linked <em>trans</em>-acting QTL)≈1175/5067  Prob(significant linkage∩linked <em>trans</em>-acting QTL)≈103/5067  Prob(significant linkage∩<em>cis</em>-acting QTL)≈0.216
</pre><a id="article1.body1.sec4.sec2.p4" name="article1.body1.sec4.sec2.p4"></a><p>The estimated power to detect <em>cis</em>-acting QTL is then Prob(significant linkage∩<em>cis</em>-acting QTL)÷Prob(<em>cis</em>-acting QTL)≈0.504 and the estimated false-positive rate is (Prob(significant linkage)-Prob(significant linkage∩<em>cis</em>-acting QTL))÷(1-Prob(<em>cis</em>-acting QTL))≈0.039. As described in <a href="#pone.0000678.s001">Text S1</a>, we expect that these estimates are conservative.</p>


<h4>Regression based approach for the rate of accumulation of <em>cis</em>-acting QTL</h4>
<a id="article1.body1.sec4.sec3.p1" name="article1.body1.sec4.sec3.p1"></a><p>As described above, the probability that a gene shows <em>cis</em>-acting regulatory variation depends on the coalescence time <em>t<sub>i</sub></em> between the two copies of the gene. For each locus, the value of <em>t<sub>i</sub></em> is unknown, but it is straightforward to obtain a point estimate of <em>t<sub>i</sub></em> based on the observed locus-specific synonymous site substitution rate. If at locus <em>i</em> there are <em>m<sub>i</sub></em> synonymous substitutions observed between BY and RM among a total of <em>M<sub>i</sub></em> synonymous sites, then<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000678.e009&amp;representation=PNG" class="inline-graphic"><span class="note">(6)</span></span><br>and the maximum likelihood estimate of θ<em>t<sub>i</sub></em> is <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000678.e010&amp;representation=PNG" class="inline-graphic"></span>. These locus-specific estimates could then be plugged into Equation 2, yielding<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000678.e011&amp;representation=PNG" class="inline-graphic"><span class="note">(7)</span></span><br>Grouping genes based on the observed <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000678.e012&amp;representation=PNG" class="inline-graphic"></span> allows for a simple regression of Prob(<em>cis</em>-acting QTL) on the estimated θ<em>t<sub>i</sub></em> as a function of the parameters of interest δ and <em>n</em>, as given by Equation 7 and as shown in <a href="#pone-0000678-g003">Figure 3</a>.</p>


<h4>Ancestral selection graph simulations</h4>
<a id="article1.body1.sec4.sec4.p1" name="article1.body1.sec4.sec4.p1"></a><p>We used the ancestral selection graph to estimate the scaled selection coefficient that would give rise to the observed skew towards rare derived alleles at regulatory polymorphisms. We modeled BY, RM, and YJM as sampled individuals from a common <em>S. cerevisiae</em> population with <em>S. paradoxus</em> as the outgroup (diverged 37<em>N<sub>e</sub></em> generations ago). This scheme is shown in <a href="#pone-0000678-g005">Figure 5</a>. Each simulation included a neutral site and a linked selected site at which the scaled selection coefficients for the four nucleotides were σ<em><sub>T</sub></em> = 0, σ<em><sub>C</sub></em> = 2<em>N<sub>e</sub>s</em>, σ<em><sub>G</sub></em> = 2×2<em>N<sub>e</sub>s</em>, and σ<em><sub>A</sub></em> = 3×2<em>N<sub>e</sub>s</em>. Our implementation of the ancestral selection graph applies to evolution in haploid populations or diploid populations in which selection acts additively. Mutation occurred at both loci at a rate of θ/2 along each branch with θ = 2μ<em>N<sub>e</sub></em> = 0.01 (based on the observed synonymous site substitution rate) according to the Kimura two parameter model with a transition to transversion ratio of 4.4 (estimated from the observed data). Note that by allowing mutation and selection to occur continuously in both <em>Saccharomyces</em> lineages, we correctly account for the effect of repeat mutations on our parsimony based counts of rare derived alleles under both the null hypothesis (neutral evolution) and the alternative hypothesis (purifying selection). We performed 1×10<sup>7</sup> simulations for 2<em>N<sub>e</sub>s</em> = 0.0, 0.2, 0.4, …, 3.0. We also considered other selection models in which transition mutations lead to a larger decrease in the fitness (see <a href="#pone.0000678.s004">Figure S3</a> and <a href="#pone.0000678.s005">Figure S4</a> and <a href="#pone.0000678.s001">Text S1</a>). All other aspects of the ancestral selection graph simulation were performed according to the algorithms outlined in Neuhauser and Krone <a href="#pone.0000678-Neuhauser1">[38]</a>.</p>
<a id="article1.body1.sec4.sec4.p2" name="article1.body1.sec4.sec4.p2"></a><p>At a scaled fitness difference between selective classes of 2<em>N<sub>e</sub>s</em> = 2.0 (2.2), there were 69379 (66022) simulations out of 1×10<sup>7</sup> for which the two sampled individuals representing BY and RM were polymorphic with respect to each other at the selected site. Of these polymorphisms 45427 (44130) were classified as having rare derived alleles whereas 22277 (20507) were classified as common. As in the observed data, unclassified polymorphisms occur because more than two alleles are observed among the four <em>Saccharomyces</em> sequences at a single site. The scaled selection coefficient that most closely reproduced the observed skew was estimated to be 2.1 by linear interpolation between 2<em>N<sub>e</sub>s</em> equal to 2.0 and 2.2. For 2<em>N<sub>e</sub>s</em> = 2.0 (2.2), the linked neutral site was polymorphic 98247 (97017) times (slightly less than 1×10<sup>7</sup>×θ due to repeat mutation), and 52844 (52508) of these polymorphisms were categorized as having rare derived alleles whereas 35513 (34819) were classified as common. This proportion of rare derived alleles is not significantly different from that observed at synonymous sites, supporting the simple demographic model employed in these simulations. The decreased rate of polymorphism between BY and RM at the selected site relative to the neutral site (ζ = 69379/98247≈71% at 2<em>N<sub>e</sub>s</em> = 2.0; ζ = 66022/97017≈68% at 2<em>N<sub>e</sub>s</em> = 2.2) reflects the elimination of mildly deleterious alleles by purifying selection. Linear interpolation between these values provides an estimate of the deficiency in regulatory polymorphism due to weak purifying selection.</p>

</div>

<div id="section5" class="section"><a id="s5" name="s5" toc="s5" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0000678"></div><a name="pone.0000678.s001" id="pone.0000678.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000678.s001">Text S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1"></a><p class="preSiDOI">Supplementary materials describing in detail the methods and results for supporting analyses.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000678.s001</p><a id="article1.body1.sec5.supplementary-material1.caption1.p2" name="article1.body1.sec5.supplementary-material1.caption1.p2"></a><p class="postSiDOI">(0.11 MB PDF)</p>
<a name="pone.0000678.s002" id="pone.0000678.s002"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000678.s002">Figure S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material2.caption1.p1" name="article1.body1.sec5.supplementary-material2.caption1.p1"></a><p class="preSiDOI">Observed synonymous site substitution rate autocorrelation function (blue points) and 10 realizations of the autocorrelation function from simulated yeast genomes (black lines). We imposed the same pattern of missing data on the simulated data as was present in the observed data (due to gaps, to low quality regions of the alignments, and to the absence of synonymous sites in intergenic regions and because of overlapping ORFs). In both the simulated and observed data the autocorrelation function was calculated for each chromosome and then averaged across the 16 chromosomes.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000678.s002</p><a id="article1.body1.sec5.supplementary-material2.caption1.p2" name="article1.body1.sec5.supplementary-material2.caption1.p2"></a><p class="postSiDOI">(0.17 MB TIF)</p>
<a name="pone.0000678.s003" id="pone.0000678.s003"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000678.s003">Figure S2. </a></strong></p><a id="article1.body1.sec5.supplementary-material3.caption1.p1" name="article1.body1.sec5.supplementary-material3.caption1.p1"></a><p class="preSiDOI">Magnitude of the skew toward derived alleles as a function of <em>cis</em>-regulatory effect size. The proportion of derived alleles classified as rare are shown for synonymous polymorphisms (black), promoter polymorphisms (blue), and 3' UTR polymorphisms (orange) in genes with <em>cis</em>-regulatory variation.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000678.s003</p><a id="article1.body1.sec5.supplementary-material3.caption1.p2" name="article1.body1.sec5.supplementary-material3.caption1.p2"></a><p class="postSiDOI">(0.11 MB TIF)</p>
<a name="pone.0000678.s004" id="pone.0000678.s004"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000678.s004">Figure S3. </a></strong></p><a id="article1.body1.sec5.supplementary-material4.caption1.p1" name="article1.body1.sec5.supplementary-material4.caption1.p1"></a><p class="preSiDOI">Strength of purifying selection against <em>cis</em>-acting regulatory changes with Σ<sub>T</sub> = 0, Σ<sub>G</sub> = 2N<sub>e</sub>s, Σ<sub>C</sub> = 2×2N<sub>e</sub>s, and Σ<sub>A</sub> = 3×2N<sub>e</sub>s. The relative rate of substitution at the selected site relative to the linked neutral site, denoted by ζ, is indicated along with the 95% CI.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000678.s004</p><a id="article1.body1.sec5.supplementary-material4.caption1.p2" name="article1.body1.sec5.supplementary-material4.caption1.p2"></a><p class="postSiDOI">(0.85 MB TIF)</p>
<a name="pone.0000678.s005" id="pone.0000678.s005"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000678.s005">Figure S4. </a></strong></p><a id="article1.body1.sec5.supplementary-material5.caption1.p1" name="article1.body1.sec5.supplementary-material5.caption1.p1"></a><p class="preSiDOI">Strength of purifying selection against <em>cis</em>-acting regulatory changes with Σ<sub>G</sub> = 0, Σ<sub>T</sub> = 2N<sub>e</sub>s, Σ<sub>C</sub> = 2×2N<sub>e</sub>s, and Σ<sub>A</sub> = 3×2N<sub>e</sub>s. The relative rate of substitution at the selected site relative to the linked neutral site, denoted by ζ, is indicated along with the 95% CI.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000678.s005</p><a id="article1.body1.sec5.supplementary-material5.caption1.p2" name="article1.body1.sec5.supplementary-material5.caption1.p2"></a><p class="postSiDOI">(0.84 MB TIF)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>We thank John Storey and Phil Green for critiquing drafts of the manuscript, Joe Felsenstein for suggestions on likelihood estimation, and Leonid Kruglyak for making the data used in this study publicly available.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: JA JR. Performed the experiments: JR. Analyzed the data: JA JR. Wrote the paper: JA JR.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0000678-Rifkin1" id="pone.0000678-Rifkin1"></a>Rifkin SA, Kim J, White KP (2003) Evolution of gene expression in the <em>Drosophila melanogaster</em> subgroup. Nature Genetics  33: 138–144.  <ul class="find" data-citedArticleID="971872" data-doi="10.1038/ng1086"><li><a href="http://dx.doi.org/10.1038/ng1086" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evolution+of+gene+expression+in+the+Drosophila+melanogaster+subgroup." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evolution+of+gene+expression+in+the+Drosophila+melanogaster+subgroup.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0000678-Townsend1" id="pone.0000678-Townsend1"></a>Townsend JP, Cavalieri D, Hartl DL (2003) Population genetic variation in genome-wide gene expression. Molecular Biology and Evolution  20: 955–963.  <ul class="find" data-citedArticleID="971890" data-doi="10.1093/molbev/msg106"><li><a href="http://dx.doi.org/10.1093/molbev/msg106" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Population+genetic+variation+in+genome-wide+gene+expression." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Population+genetic+variation+in+genome-wide+gene+expression.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0000678-Fay1" id="pone.0000678-Fay1"></a>Fay JC, McCullough HL, Sniegowski PD, Eisen MB (2004) Population genetic variation in gene expression is associated with phenotypic variation in <em>Saccharomyces cerevisiae</em>. Genome Biology  5: R26.  <ul class="find" data-citedArticleID="971820" data-doi="10.1186/gb-2004-5-4-r26"><li><a href="http://dx.doi.org/10.1186/gb-2004-5-4-r26" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Population+genetic+variation+in+gene+expression+is+associated+with+phenotypic+variation+in+Saccharomyces+cerevisiae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Population+genetic+variation+in+gene+expression+is+associated+with+phenotypic+variation+in+Saccharomyces+cerevisiae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0000678-Khaitovich1" id="pone.0000678-Khaitovich1"></a>Khaitovich P, Weiss G, Lachmann M, Hellmann I, Enard W, et al.  (2004) A neutral model of transcriptome evolution. PLoS Biology  2: e132.  <ul class="find" data-citedArticleID="971842" data-doi="10.1371/journal.pbio.0020132"><li><a href="http://dx.doi.org/10.1371/journal.pbio.0020132" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+neutral+model+of+transcriptome+evolution." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+neutral+model+of+transcriptome+evolution.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0000678-Denver1" id="pone.0000678-Denver1"></a>Denver DR, Morris K, Streelman JT, Kim SK, Lynch M, et al.  (2005) The transcriptional consequences of mutation and natural selection in <em>Caenorhabditis elegans</em>. Nature Genetics  37: 544–548.  <ul class="find" data-citedArticleID="971816" data-doi="10.1038/ng1554"><li><a href="http://dx.doi.org/10.1038/ng1554" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+transcriptional+consequences+of+mutation+and+natural+selection+in+Caenorhabditis+elegans." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+transcriptional+consequences+of+mutation+and+natural+selection+in+Caenorhabditis+elegans.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0000678-Khaitovich2" id="pone.0000678-Khaitovich2"></a>Khaitovich P, Hellmann I, Enard W, Nowich K, Leinweber M, et al.  (2005) Parallel patterns of evolution in the genomes and transcriptomes of humans and chimpanzees. Science  309: 1805–1854.  <ul class="find" data-citedArticleID="971844" data-doi="10.1126/science.1108296"><li><a href="http://dx.doi.org/10.1126/science.1108296" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Parallel+patterns+of+evolution+in+the+genomes+and+transcriptomes+of+humans+and+chimpanzees." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Parallel+patterns+of+evolution+in+the+genomes+and+transcriptomes+of+humans+and+chimpanzees.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0000678-Rifkin2" id="pone.0000678-Rifkin2"></a>Rifkin SA, Houle D, Kim J, White KP (2005) A mutation accumulation assay reveals a broad capacity for rapid evolution of gene expression. Nature  438: 220–223.  <ul class="find" data-citedArticleID="971874" data-doi="10.1038/nature04114"><li><a href="http://dx.doi.org/10.1038/nature04114" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+mutation+accumulation+assay+reveals+a+broad+capacity+for+rapid+evolution+of+gene+expression." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+mutation+accumulation+assay+reveals+a+broad+capacity+for+rapid+evolution+of+gene+expression.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0000678-Gilad1" id="pone.0000678-Gilad1"></a>Gilad Y, Oshlack A, Smyth GK, Speed TP, White KP (2006) Expression profiling in primates reveals a rapid evolution of human transcription factors. Nature  440: 242–245.  <ul class="find" data-citedArticleID="971824" data-doi="10.1038/nature04559"><li><a href="http://dx.doi.org/10.1038/nature04559" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Expression+profiling+in+primates+reveals+a+rapid+evolution+of+human+transcription+factors." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Expression+profiling+in+primates+reveals+a+rapid+evolution+of+human+transcription+factors.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0000678-Whitehead1" id="pone.0000678-Whitehead1"></a>Whitehead A, Crawford DL (2006) Neutral and adaptive variation in gene expression. Proceedings of the National Academy of Sciences U S A  103: 5425–5430.  <ul class="find" data-citedArticleID="971892" data-doi="10.1073/pnas.0507648103"><li><a href="http://dx.doi.org/10.1073/pnas.0507648103" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Neutral+and+adaptive+variation+in+gene+expression." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Neutral+and+adaptive+variation+in+gene+expression.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0000678-Whitehead2" id="pone.0000678-Whitehead2"></a>Whitehead A, Crawford DL (2006) Variation within and among species in gene expression: raw material for evolution. Molecular Ecology  15: 1197–1211.  <ul class="find" data-citedArticleID="971894" data-doi="10.1111/j.1365-294x.2006.02868.x"><li><a href="http://dx.doi.org/10.1111/j.1365-294x.2006.02868.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Variation+within+and+among+species+in+gene+expression%3A+raw+material+for+evolution." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Variation+within+and+among+species+in+gene+expression%3A+raw+material+for+evolution.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0000678-Brem1" id="pone.0000678-Brem1"></a>Brem RB, Yvert G, Clinton R, Kruglyak L (2002) Genetic dissection of transcriptional regulation in budding yeast. Science  296: 752–755.  <ul class="find" data-citedArticleID="971804" data-doi="10.1126/science.1069516"><li><a href="http://dx.doi.org/10.1126/science.1069516" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genetic+dissection+of+transcriptional+regulation+in+budding+yeast." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genetic+dissection+of+transcriptional+regulation+in+budding+yeast.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0000678-Schadt1" id="pone.0000678-Schadt1"></a>Schadt EE, Monks SA, Drake TA, Lusis AJ, Che N, et al.  (2003) Genetics of gene expression surveyed in maize, mouse and man. Nature  422: 297–302.  <ul class="find" data-citedArticleID="971882" data-doi="10.1038/nature01434"><li><a href="http://dx.doi.org/10.1038/nature01434" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genetics+of+gene+expression+surveyed+in+maize%2C+mouse+and+man." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genetics+of+gene+expression+surveyed+in+maize%2C+mouse+and+man.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0000678-Morley1" id="pone.0000678-Morley1"></a>Morley M, Molony CM, Weber TM, Devlin JL, Ewens KG, et al.  (2004) Genetic analysis of genome-wide variation in human gene expression. Nature  430: 743–747.  <ul class="find" data-citedArticleID="971858" data-doi="10.1038/nature02797"><li><a href="http://dx.doi.org/10.1038/nature02797" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genetic+analysis+of+genome-wide+variation+in+human+gene+expression." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genetic+analysis+of+genome-wide+variation+in+human+gene+expression.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0000678-Monks1" id="pone.0000678-Monks1"></a>Monks SA, Leonardson A, Zhu H, Cundiff P, Pietrusiak P, et al.  (2004) Genetic inheritance of gene expression in human cell lines. American Journal of Human Genetics  75: 1094–1105.  <ul class="find" data-citedArticleID="971856" data-doi="10.1086/426461"><li><a href="http://dx.doi.org/10.1086/426461" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genetic+inheritance+of+gene+expression+in+human+cell+lines." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genetic+inheritance+of+gene+expression+in+human+cell+lines.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0000678-Bystrykh1" id="pone.0000678-Bystrykh1"></a>Bystrykh L, Weersing E, Dontje B, Sutton S, Pletcher MT, et al.  (2005) Uncovering regulatory pathways that affect hematopoietic stem cell function using ‘genetical genomics’. Nature Genetics  37: 225–232.  <ul class="find" data-citedArticleID="971810" data-doi="10.1038/ng1497"><li><a href="http://dx.doi.org/10.1038/ng1497" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Uncovering+regulatory+pathways+that+affect+hematopoietic+stem+cell+function+using+%26%23x2018%3Bgenetical+genomics%26%23x2019%3B." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Uncovering+regulatory+pathways+that+affect+hematopoietic+stem+cell+function+using+%26%23x2018%3Bgenetical+genomics%26%23x2019%3B.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0000678-Chesler1" id="pone.0000678-Chesler1"></a>Chesler EJ, Lu L, Shou S, Qu Y, Gu J, et al.  (2005) Complex trait analysis of gene expression uncovers polygenic and pleiotropic networks that modulate nervous system function. Nature Genetics  37: 233–242.  <ul class="find" data-citedArticleID="971812" data-doi="10.1038/ng1518"><li><a href="http://dx.doi.org/10.1038/ng1518" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Complex+trait+analysis+of+gene+expression+uncovers+polygenic+and+pleiotropic+networks+that+modulate+nervous+system+function." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Complex+trait+analysis+of+gene+expression+uncovers+polygenic+and+pleiotropic+networks+that+modulate+nervous+system+function.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0000678-Hubner1" id="pone.0000678-Hubner1"></a>Hubner N, Wallace CA, Zimdahl H, Petretto E, Schulz H, et al.  (2005) Integrated transcriptional profiling and linkage analysis for identification of genes underlying disease. Nature Genetics  37: 243–253.  <ul class="find" data-citedArticleID="971834" data-doi="10.1038/ng1522"><li><a href="http://dx.doi.org/10.1038/ng1522" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Integrated+transcriptional+profiling+and+linkage+analysis+for+identification+of+genes+underlying+disease." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Integrated+transcriptional+profiling+and+linkage+analysis+for+identification+of+genes+underlying+disease.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0000678-Lan1" id="pone.0000678-Lan1"></a>Lan H, Chen M, Flowers JB, Yandell BS, Stapleton DS, et al.  (2006) Combined expression trait correlations and expression quantitative trait locus mapping. PLoS Genetics  2: e6.  <ul class="find" data-citedArticleID="971848" data-doi="10.1371/journal.pgen.0020006"><li><a href="http://dx.doi.org/10.1371/journal.pgen.0020006" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Combined+expression+trait+correlations+and+expression+quantitative+trait+locus+mapping." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Combined+expression+trait+correlations+and+expression+quantitative+trait+locus+mapping.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0000678-Petretto1" id="pone.0000678-Petretto1"></a>Petretto E, Mangion J, Dickens NJ, Cook SA, Kumaran MK, et al.  (2006) Heritability and tissue specificity of expression quantitative trait loci. PLoS Genetics  2: e172.  <ul class="find" data-citedArticleID="971870" data-doi="10.1371/journal.pgen.0020172"><li><a href="http://dx.doi.org/10.1371/journal.pgen.0020172" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Heritability+and+tissue+specificity+of+expression+quantitative+trait+loci." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Heritability+and+tissue+specificity+of+expression+quantitative+trait+loci.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0000678-Yvert1" id="pone.0000678-Yvert1"></a>Yvert G, Brem RB, Whittle J, Akey JM, Foss E, et al.  (2003) Transacting regulatory variation in <em>Saccharomyces cerevisiae</em> and the role of transcription factors. Nature Genetics  35: 57–64.  <ul class="find" data-citedArticleID="971898" data-doi="10.1038/ng1222"><li><a href="http://dx.doi.org/10.1038/ng1222" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Transacting+regulatory+variation+in+Saccharomyces+cerevisiae+and+the+role+of+transcription+factors." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Transacting+regulatory+variation+in+Saccharomyces+cerevisiae+and+the+role+of+transcription+factors.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0000678-Bing1" id="pone.0000678-Bing1"></a>Bing N, Hoeschele I (2005) Genetical genomics analysis of a yeast segregant population for transcription network inference. Genetics  170: 533–542.  <ul class="find" data-citedArticleID="971802" data-doi="10.1534/genetics.105.041103"><li><a href="http://dx.doi.org/10.1534/genetics.105.041103" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genetical+genomics+analysis+of+a+yeast+segregant+population+for+transcription+network+inference." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genetical+genomics+analysis+of+a+yeast+segregant+population+for+transcription+network+inference.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0000678-Storey1" id="pone.0000678-Storey1"></a>Storey JD, Akey JM, Kruglyak L (2005) Multiple locus linkage analysis of genomewide expression in yeast. PLoS Biology  3: e267.  <ul class="find" data-citedArticleID="971884" data-doi="10.1371/journal.pbio.0030267"><li><a href="http://dx.doi.org/10.1371/journal.pbio.0030267" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Multiple+locus+linkage+analysis+of+genomewide+expression+in+yeast." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Multiple+locus+linkage+analysis+of+genomewide+expression+in+yeast.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0000678-Ronald1" id="pone.0000678-Ronald1"></a>Ronald J, Brem RB, Whittle J, Kruglyak L (2005) Local regulatory variation in <em>Saccharomyces cerevisiae</em>. PLoS Genetics  1: e25.  <ul class="find" data-citedArticleID="971876" data-doi="10.1371/journal.pgen.0010025"><li><a href="http://dx.doi.org/10.1371/journal.pgen.0010025" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Local+regulatory+variation+in+Saccharomyces+cerevisiae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Local+regulatory+variation+in+Saccharomyces+cerevisiae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0000678-Lee1" id="pone.0000678-Lee1"></a>Lee SI, Pe'er D, Dudley AM, Church GM, Koller D (2006) Identifying regulatory mechanisms using individual variation reveals key role for chromatin modification. Proceedings of the National Academy of Sciences U S A  103: 14062–14067.  <ul class="find" data-citedArticleID="971852" data-doi="10.1073/pnas.0601852103"><li><a href="http://dx.doi.org/10.1073/pnas.0601852103" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Identifying+regulatory+mechanisms+using+individual+variation+reveals+key+role+for+chromatin+modification." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Identifying+regulatory+mechanisms+using+individual+variation+reveals+key+role+for+chromatin+modification.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0000678-Gu1" id="pone.0000678-Gu1"></a>Gu Z, David L, Petrov D, Jones T, Davis RW, et al.  (2005) Elevated evolutionary rates in the laboratory strain of <em>Saccharomyces cerevisiae</em>. Proceedings of the National Academy of Sciences U S A  102: 1092–1097.  <ul class="find" data-citedArticleID="971826" data-doi="10.1073/pnas.0409159102"><li><a href="http://dx.doi.org/10.1073/pnas.0409159102" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Elevated+evolutionary+rates+in+the+laboratory+strain+of+Saccharomyces+cerevisiae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Elevated+evolutionary+rates+in+the+laboratory+strain+of+Saccharomyces+cerevisiae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0000678-Kellis1" id="pone.0000678-Kellis1"></a>Kellis M, Patterson N, Endrizzi M, Birren B, Lander ES (2003) Sequencing and comparison of yeast species to identify genes and regulatory elements. Nature  423: 241–254.  <ul class="find" data-citedArticleID="971840" data-doi="10.1038/nature01644"><li><a href="http://dx.doi.org/10.1038/nature01644" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Sequencing+and+comparison+of+yeast+species+to+identify+genes+and+regulatory+elements." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Sequencing+and+comparison+of+yeast+species+to+identify+genes+and+regulatory+elements.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0000678-Doniger1" id="pone.0000678-Doniger1"></a>Doniger SW, Huh J, Fay JC (2005) Identification of functional transcription factor binding sites using closely related <em>Saccharomyces</em> species. Genome Research  15: 701–709.  <ul class="find" data-citedArticleID="971818" data-doi="10.1101/gr.3578205"><li><a href="http://dx.doi.org/10.1101/gr.3578205" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Identification+of+functional+transcription+factor+binding+sites+using+closely+related+Saccharomyces+species." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Identification+of+functional+transcription+factor+binding+sites+using+closely+related+Saccharomyces+species.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0000678-Heaton1" id="pone.0000678-Heaton1"></a>Heaton B, Decker C, Muhlrad D, Donahue J, Jacobson A, et al.  (1992) Analysis of chimeric mRNAs derived from the <em>STE3</em> mRNA identifies multiple regions within yeast mRNAs that modulate mRNA decay. Nucleic Acids Research  20: 5365–5373.  <ul class="find" data-citedArticleID="971832" data-doi="10.1093/nar/20.20.5365"><li><a href="http://dx.doi.org/10.1093/nar/20.20.5365" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Analysis+of+chimeric+mRNAs+derived+from+the+STE3+mRNA+identifies+multiple+regions+within+yeast+mRNAs+that+modulate+mRNA+decay." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Analysis+of+chimeric+mRNAs+derived+from+the+STE3+mRNA+identifies+multiple+regions+within+yeast+mRNAs+that+modulate+mRNA+decay.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0000678-Muhlrad1" id="pone.0000678-Muhlrad1"></a>Muhlrad D, Parker R (1992) Mutations affecting stability and deadenylation of the yeast <em>MFA2</em> transcript. Genes and Development  6: 2100–2111.  <ul class="find" data-citedArticleID="971862" data-doi="10.1101/gad.6.11.2100"><li><a href="http://dx.doi.org/10.1101/gad.6.11.2100" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mutations+affecting+stability+and+deadenylation+of+the+yeast+MFA2+transcript." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mutations+affecting+stability+and+deadenylation+of+the+yeast+MFA2+transcript.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0000678-Muhlrad2" id="pone.0000678-Muhlrad2"></a>Muhlrad D, Decker CJ, Parker R (1995) Turnover mechanisms of the stable yeast <em>PGK1</em> mRNA. Molecular and Cellular Biology  15: 2145–2156.  <ul class="find" data-citedArticleID="971864"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Turnover+mechanisms+of+the+stable+yeast+PGK1+mRNA.&amp;auth=&amp;atitle=Turnover+mechanisms+of+the+stable+yeast+PGK1+mRNA." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Turnover+mechanisms+of+the+stable+yeast+PGK1+mRNA." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Turnover+mechanisms+of+the+stable+yeast+PGK1+mRNA.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0000678-Ronald2" id="pone.0000678-Ronald2"></a>Ronald J, Tang H, Brem R (2006) Genome-wide evolutionary rates in laboratory and wild yeast. Genetics  174: 541–544.  <ul class="find" data-citedArticleID="971878" data-doi="10.1534/genetics.106.060863"><li><a href="http://dx.doi.org/10.1534/genetics.106.060863" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genome-wide+evolutionary+rates+in+laboratory+and+wild+yeast." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genome-wide+evolutionary+rates+in+laboratory+and+wild+yeast.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0000678-Ruderfer1" id="pone.0000678-Ruderfer1"></a>Ruderfer DM, Pratt SC, Seidel HS, Kruglyak L (2006) Population genomic analysis of outcrossing and recombination in yeast. Nature Genetics  38: 1077–1081.  <ul class="find" data-citedArticleID="971880" data-doi="10.1038/ng1859"><li><a href="http://dx.doi.org/10.1038/ng1859" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Population+genomic+analysis+of+outcrossing+and+recombination+in+yeast." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Population+genomic+analysis+of+outcrossing+and+recombination+in+yeast.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0000678-Fay2" id="pone.0000678-Fay2"></a>Fay JC, Wyckoff GJ, Wu CI (2001) Positive and negative selection on the human genome. Genetics  158: 1227–1234.  <ul class="find" data-citedArticleID="971822"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Positive+and+negative+selection+on+the+human+genome.&amp;auth=&amp;atitle=Positive+and+negative+selection+on+the+human+genome." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Positive+and+negative+selection+on+the+human+genome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Positive+and+negative+selection+on+the+human+genome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0000678-Hughes1" id="pone.0000678-Hughes1"></a>Hughes AL, Packer B, Welsh R, Bergen AW, Chanock SJ, et al.  (2003) Widespread purifying selection at polymorphic sites in human protein coding loci. Proceedings of the National Academy of Sciences U S A  100: 15754–15757.  <ul class="find" data-citedArticleID="971836" data-doi="10.1073/pnas.2536718100"><li><a href="http://dx.doi.org/10.1073/pnas.2536718100" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Widespread+purifying+selection+at+polymorphic+sites+in+human+protein+coding+loci." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Widespread+purifying+selection+at+polymorphic+sites+in+human+protein+coding+loci.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0000678-Hughes2" id="pone.0000678-Hughes2"></a>Hughes AL (2005) Evidence for abundant slightly deleterious polymorphisms in bacterial populations. Genetics  169: 533–538.  <ul class="find" data-citedArticleID="971838" data-doi="10.1534/genetics.104.036939"><li><a href="http://dx.doi.org/10.1534/genetics.104.036939" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evidence+for+abundant+slightly+deleterious+polymorphisms+in+bacterial+populations." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evidence+for+abundant+slightly+deleterious+polymorphisms+in+bacterial+populations.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">36.
              </span><a name="pone.0000678-Cliften1" id="pone.0000678-Cliften1"></a>Cliften P, Sudarsanam P, Desikan A, Fulton L, Fulton B, et al.  (2003) Finding functional features in <em>Saccharomyces</em> genomes by phylogenetic footprinting. Science  301: 71–76.  <ul class="find" data-citedArticleID="971814" data-doi="10.1126/science.1084337"><li><a href="http://dx.doi.org/10.1126/science.1084337" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Finding+functional+features+in+Saccharomyces+genomes+by+phylogenetic+footprinting." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Finding+functional+features+in+Saccharomyces+genomes+by+phylogenetic+footprinting.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">37.
              </span><a name="pone.0000678-Harbison1" id="pone.0000678-Harbison1"></a>Harbison CT, Gordon DB, Lee TI, Rinaldi NJ, Macisaac KD, et al.  (2004) Transcriptional regulatory code of a eukaryotic genome. Nature  431: 99–104.  <ul class="find" data-citedArticleID="971830" data-doi="10.1038/nature02800"><li><a href="http://dx.doi.org/10.1038/nature02800" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Transcriptional+regulatory+code+of+a+eukaryotic+genome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Transcriptional+regulatory+code+of+a+eukaryotic+genome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">38.
              </span><a name="pone.0000678-Neuhauser1" id="pone.0000678-Neuhauser1"></a>Neuhauser C, Krone SM (1997) The genealogy of samples in models with selection. Genetics  145: 519–534.  <ul class="find" data-citedArticleID="971866"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=The+genealogy+of+samples+in+models+with+selection.&amp;auth=&amp;atitle=The+genealogy+of+samples+in+models+with+selection." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+genealogy+of+samples+in+models+with+selection." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+genealogy+of+samples+in+models+with+selection.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">39.
              </span><a name="pone.0000678-Krone1" id="pone.0000678-Krone1"></a>Krone SM, Neuhauser C (1997) Ancestral processes with selection. Theoretical Population Biology  51: 210–237.  <ul class="find" data-citedArticleID="971846" data-doi="10.1006/tpbi.1997.1299"><li><a href="http://dx.doi.org/10.1006/tpbi.1997.1299" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Ancestral+processes+with+selection." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Ancestral+processes+with+selection.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">40.
              </span><a name="pone.0000678-Wong1" id="pone.0000678-Wong1"></a>Wong WS, Nielsen R (2004) Detecting selection in noncoding regions of nucleotide sequences. Genetics  167: 949–958.  <ul class="find" data-citedArticleID="971896" data-doi="10.1534/genetics.102.010959"><li><a href="http://dx.doi.org/10.1534/genetics.102.010959" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Detecting+selection+in+noncoding+regions+of+nucleotide+sequences." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Detecting+selection+in+noncoding+regions+of+nucleotide+sequences.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">41.
              </span><a name="pone.0000678-Mortimer1" id="pone.0000678-Mortimer1"></a>Mortimer RK, Johnston JR (1986) Genealogy of principle strains of the yeast genetic stock center. Genetics  113: 35–43.  <ul class="find" data-citedArticleID="971860"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Genealogy+of+principle+strains+of+the+yeast+genetic+stock+center.&amp;auth=&amp;atitle=Genealogy+of+principle+strains+of+the+yeast+genetic+stock+center." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genealogy+of+principle+strains+of+the+yeast+genetic+stock+center." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genealogy+of+principle+strains+of+the+yeast+genetic+stock+center.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">42.
              </span><a name="pone.0000678-Ohta1" id="pone.0000678-Ohta1"></a>Ohta T (2002) Near-neutrality in evolution of genes and gene regulation. Proceedings of the National Academy of Sciences U S A  99: 16134–16137.  <ul class="find" data-citedArticleID="971868" data-doi="10.1073/pnas.252626899"><li><a href="http://dx.doi.org/10.1073/pnas.252626899" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Near-neutrality+in+evolution+of+genes+and+gene+regulation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Near-neutrality+in+evolution+of+genes+and+gene+regulation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">43.
              </span><a name="pone.0000678-Lemos1" id="pone.0000678-Lemos1"></a>Lemos B, Meiklejohn CD, Caceres M, Hartl DL (2005) Rates of divergence in gene expression profiles of primates, mice, and flies: stabilizing selection and variability among functional categories. Evolution: Internation Journal of Organic Evolution  59: 126–137.  <ul class="find" data-citedArticleID="971854" data-doi="10.1554/04-251"><li><a href="http://dx.doi.org/10.1554/04-251" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Rates+of+divergence+in+gene+expression+profiles+of+primates%2C+mice%2C+and+flies%3A+stabilizing+selection+and+variability+among+functional+categories." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Rates+of+divergence+in+gene+expression+profiles+of+primates%2C+mice%2C+and+flies%3A+stabilizing+selection+and+variability+among+functional+categories.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">44.
              </span><a name="pone.0000678-Landry1" id="pone.0000678-Landry1"></a>Landry CR, Wittkopp PJ, Taubes CH, Ranz JM, Clark AG, et al.  (2005) Compensatory <em>cis</em>-<em>trans</em> evolution and the dysregulation of gene expression in interspecific hybrids of <em>Drosophila</em>. Genetics  171: 1813–1822.  <ul class="find" data-citedArticleID="971850" data-doi="10.1534/genetics.105.047449"><li><a href="http://dx.doi.org/10.1534/genetics.105.047449" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Compensatory+cis-trans+evolution+and+the+dysregulation+of+gene+expression+in+interspecific+hybrids+of+Drosophila." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Compensatory+cis-trans+evolution+and+the+dysregulation+of+gene+expression+in+interspecific+hybrids+of+Drosophila.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">45.
              </span><a name="pone.0000678-Brem2" id="pone.0000678-Brem2"></a>Brem RB, Kruglyak L (2005) The landscape of genetic complexity across 5,700 gene expression traits in yeast. Proceedings of the National Academy of Sciences U S A  102: 1572–1577.  <ul class="find" data-citedArticleID="971806" data-doi="10.1073/pnas.0408709102"><li><a href="http://dx.doi.org/10.1073/pnas.0408709102" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+landscape+of+genetic+complexity+across+5%2C700+gene+expression+traits+in+yeast." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+landscape+of+genetic+complexity+across+5%2C700+gene+expression+traits+in+yeast.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">46.
              </span><a name="pone.0000678-Hahn1" id="pone.0000678-Hahn1"></a>Hahn MW (2007) Detecting natural selection on <em>cis</em>-regulatory DNA. Genetica  129: 7–18.  <ul class="find" data-citedArticleID="971828" data-doi="10.1007/s10709-006-0029-y"><li><a href="http://dx.doi.org/10.1007/s10709-006-0029-y" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Detecting+natural+selection+on+cis-regulatory+DNA." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Detecting+natural+selection+on+cis-regulatory+DNA.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">47.
              </span><a name="pone.0000678-Brudno1" id="pone.0000678-Brudno1"></a>Brudno M, Do C, Cooper G, Kim MF, Davydov E, et al.  (2003) LAGAN and Multi-LAGAN: efficient tools for large-scale multiple alignment of genomic DNA. Genome Research  13: 721–731.  <ul class="find" data-citedArticleID="971808" data-doi="10.1101/gr.926603"><li><a href="http://dx.doi.org/10.1101/gr.926603" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=LAGAN+and+Multi-LAGAN%3A+efficient+tools+for+large-scale+multiple+alignment+of+genomic+DNA." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22LAGAN+and+Multi-LAGAN%3A+efficient+tools+for+large-scale+multiple+alignment+of+genomic+DNA.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">48.
              </span><a name="pone.0000678-Thompson1" id="pone.0000678-Thompson1"></a>Thompson JD, Higgins DG, Gibson TJ (1994) CLUSTALW: Improving the sensitivity of progressive multiple alignment through sequence weighting, position-specific gap penalties and weight matrix choice. Nucleic Acids Research  22: 4673–4680.  <ul class="find" data-citedArticleID="971888" data-doi="10.1093/nar/22.22.4673"><li><a href="http://dx.doi.org/10.1093/nar/22.22.4673" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=CLUSTALW%3A+Improving+the+sensitivity+of+progressive+multiple+alignment+through+sequence+weighting%2C+position-specific+gap+penalties+and+weight+matrix+choice." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22CLUSTALW%3A+Improving+the+sensitivity+of+progressive+multiple+alignment+through+sequence+weighting%2C+position-specific+gap+penalties+and+weight+matrix+choice.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">49.
              </span><a name="pone.0000678-Storey2" id="pone.0000678-Storey2"></a>Storey JD, Tibshirani R (2003) Statistical significance for genomewide studies. Proceedings of the National Academy of Sciences U S A  100: 9440–9445.  <ul class="find" data-citedArticleID="971886" data-doi="10.1073/pnas.1530509100"><li><a href="http://dx.doi.org/10.1073/pnas.1530509100" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Statistical+significance+for+genomewide+studies." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Statistical+significance+for+genomewide+studies.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000678.XML" value="110595"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000678.PDF" value="440107"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000678.g001.PNG_L" value="164631"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000678.g001.PNG_M" value="43260"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000678.g001.PNG_S" value="6303"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000678.g001.TIF" value="250388"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000678.g001.PNG_I" value="15783"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000678.e001.PNG" value="9087"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000678.e001.TIF" value="25144"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000678.e002.PNG" value="9549"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000678.e002.TIF" value="25668"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000678.e003.PNG" value="14212"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000678.e003.TIF" value="30240"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000678.e004.PNG" value="14520"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000678.e004.TIF" value="30472"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000678.e005.PNG" value="27596"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000678.e005.TIF" value="44784"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000678.e006.PNG" value="7040"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000678.e006.TIF" value="23016"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000678.g002.PNG_L" value="291936"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000678.g002.PNG_M" value="50865"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000678.g002.PNG_S" value="5000"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000678.g002.TIF" value="398178"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000678.g002.PNG_I" value="20313"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000678.g003.PNG_L" value="60313"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000678.g003.PNG_M" value="17003"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000678.g003.PNG_S" value="4653"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000678.g003.TIF" value="124736"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000678.g003.PNG_I" value="6583"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000678.g004.PNG_L" value="116002"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000678.g004.PNG_M" value="29604"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000678.g004.PNG_S" value="5181"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000678.g004.TIF" value="343422"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000678.g004.PNG_I" value="11413"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000678.t001.PNG_L" value="210170"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000678.t001.PNG_M" value="121759"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000678.t001.PNG_S" value="11840"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000678.t001.TIF" value="833898"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000678.t001.PNG_I" value="37254"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000678.g005.PNG_L" value="122541"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000678.g005.PNG_M" value="30668"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000678.g005.PNG_S" value="6283"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000678.g005.TIF" value="290514"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000678.g005.PNG_I" value="10932"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000678.g006.PNG_L" value="87991"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000678.g006.PNG_M" value="25077"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000678.g006.PNG_S" value="6243"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000678.g006.TIF" value="149402"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000678.g006.PNG_I" value="9285"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000678.e007.PNG" value="9302"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000678.e007.TIF" value="25408"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000678.e008.PNG" value="15060"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000678.e008.TIF" value="31168"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000678.e009.PNG" value="14675"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000678.e009.TIF" value="31060"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000678.e010.PNG" value="8183"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000678.e010.TIF" value="24052"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000678.e011.PNG" value="14500"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000678.e011.TIF" value="30292"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000678.e012.PNG" value="5792"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000678.e012.TIF" value="21764"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000678.s001.PDF" value="109087"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000678.s002.TIF" value="141815"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000678.s003.TIF" value="76499"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000678.s004.TIF" value="415114"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000678.s005.TIF" value="413304"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000678&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000678" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000678&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0000678&volume=&issue=&title=The Evolution of Gene Expression QTL in Saccharomyces cerevisiae&author_name=James%20Ronald%2C%20Joshua%20M.%20Akey&start_page=1&end_page=11" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000678" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000678" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0000678" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000678&amp;t=The%20Evolution%20of%20Gene%20Expression%20QTL%20in%20Saccharomyces%20cerevisiae" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000678&title=The%20Evolution%20of%20Gene%20Expression%20QTL%20in%20Saccharomyces%20cerevisiae&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000678&amp;title=The%20Evolution%20of%20Gene%20Expression%20QTL%20in%20Saccharomyces%20cerevisiae" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000678" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0000678&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'The Evolution of Gene Expression QTL in Saccharomyces cerevisiae';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0000678';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20The%20Evolution%20of%20Gene%20Expression%20QTL%20in%20Saccharomyces%20cerevisiae http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000678" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0000678" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">













          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Alleles%22" title="Search for articles in the subject area:'Alleles'"><div class="flagText">Alleles</div></a>
              <div data-categoryid="46071" data-articleid="24194"
                   data-categoryname="Alleles"
                   class="flagImage" title="Flag 'Alleles' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Evolutionary+genetics%22" title="Search for articles in the subject area:'Evolutionary genetics'"><div class="flagText">Evolutionary genetics</div></a>
              <div data-categoryid="17209" data-articleid="24194"
                   data-categoryname="Evolutionary genetics"
                   class="flagImage" title="Flag 'Evolutionary genetics' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Fungal+evolution%22" title="Search for articles in the subject area:'Fungal evolution'"><div class="flagText">Fungal evolution</div></a>
              <div data-categoryid="17551" data-articleid="24194"
                   data-categoryname="Fungal evolution"
                   class="flagImage" title="Flag 'Fungal evolution' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Gene+expression%22" title="Search for articles in the subject area:'Gene expression'"><div class="flagText">Gene expression</div></a>
              <div data-categoryid="17515" data-articleid="24194"
                   data-categoryname="Gene expression"
                   class="flagImage" title="Flag 'Gene expression' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Gene+regulation%22" title="Search for articles in the subject area:'Gene regulation'"><div class="flagText">Gene regulation</div></a>
              <div data-categoryid="17553" data-articleid="24194"
                   data-categoryname="Gene regulation"
                   class="flagImage" title="Flag 'Gene regulation' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Genetic+loci%22" title="Search for articles in the subject area:'Genetic loci'"><div class="flagText">Genetic loci</div></a>
              <div data-categoryid="17713" data-articleid="24194"
                   data-categoryname="Genetic loci"
                   class="flagImage" title="Flag 'Genetic loci' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Genome+evolution%22" title="Search for articles in the subject area:'Genome evolution'"><div class="flagText">Genome evolution</div></a>
              <div data-categoryid="17207" data-articleid="24194"
                   data-categoryname="Genome evolution"
                   class="flagImage" title="Flag 'Genome evolution' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Natural+selection%22" title="Search for articles in the subject area:'Natural selection'"><div class="flagText">Natural selection</div></a>
              <div data-categoryid="34433" data-articleid="24194"
                   data-categoryname="Natural selection"
                   class="flagImage" title="Flag 'Natural selection' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=4375'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=1823'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=2193&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>

<div class="block sidebar-comments">
    <div class="header">
        <h3>Comments</h3>
    </div>
      <p><a href="/annotation/listThread.action?root=20715">Needs to be defined</a><br>Posted by shearer</p>
</div>

</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
