

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: The PDZ Domain as a Complex Adaptive System</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0000953"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0000953" />

    <meta name="citation_title" content="The PDZ Domain as a Complex Adaptive System"/>
    <meta itemprop="name" content="The PDZ Domain as a Complex Adaptive System"/>

      <meta name="citation_author" content="Alexei Kurakin"/>
            <meta name="citation_author_institution" content="The Buck Institute for Age Research, Novato, California, United States of America"/>
      <meta name="citation_author" content="Andrzej Swistowski"/>
            <meta name="citation_author_institution" content="The Buck Institute for Age Research, Novato, California, United States of America"/>
      <meta name="citation_author" content="Susan C. Wu"/>
            <meta name="citation_author_institution" content="The Buck Institute for Age Research, Novato, California, United States of America"/>
      <meta name="citation_author" content="Dale E. Bredesen"/>
            <meta name="citation_author_institution" content="The Buck Institute for Age Research, Novato, California, United States of America"/>

    <meta name="citation_date" content="2007/9/26"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0000953.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e953"/>
    <meta name="citation_issue" content="9"/>
    <meta name="citation_volume" content="2"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Modular peptide recognition domains in eukaryotic signaling.; citation_author=J Kuriyan; citation_author=D Cowburn; citation_journal_title=Annu Rev Biophys Biomol Struct; citation_volume=26; citation_number=1; citation_pages=259-288; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=From Src Homology domains to other signaling modules: proposal of the ‘protein recognition code’.; citation_author=M Sudol; citation_journal_title=Oncogene; citation_volume=17; citation_number=2; citation_pages=1469-1474; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Selectivity and promiscuity in the interaction network mediated by protein recognition modules.; citation_author=L Castagnoli; citation_author=A Costantini; citation_author=C Dall'Armi; citation_author=S Gonfloni; citation_author=L Montecchi-Palazzi; citation_journal_title=FEBS Lett; citation_volume=567; citation_number=3; citation_pages=74-79; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Convergent evolution with combinatorial peptides.; citation_author=BK Kay; citation_author=J Kasanov; citation_author=S Knight; citation_author=A Kurakin; citation_journal_title=FEBS Lett; citation_volume=480; citation_number=4; citation_pages=55-62; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Domain repertoires as a tool to derive protein recognition rules.; citation_author=A Zucconi; citation_author=S Panni; citation_author=S Paoluzi; citation_author=L Castagnoli; citation_author=L Dente; citation_journal_title=FEBS Lett; citation_volume=480; citation_number=5; citation_pages=49-54; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=The rat brain postsynaptic density fraction contains a homolog of the Drosophila discs-large tumor suppressor protein.; citation_author=KO Cho; citation_author=CA Hunt; citation_author=MB Kennedy; citation_journal_title=Neuron; citation_volume=9; citation_number=6; citation_pages=929-942; citation_date=1992; " />
      <meta name="citation_reference" content="citation_title=Initial sequencing and analysis of the human genome.; citation_author=ES Lander; citation_author=LM Linton; citation_author=B Birren; citation_author=C Nusbaum; citation_author=MC Zody; citation_journal_title=Nature; citation_volume=409; citation_number=7; citation_pages=860-921; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Mechanism and role of PDZ domains in signaling complex assembly.; citation_author=BZ Harris; citation_author=WA Lim; citation_journal_title=J Cell Sci; citation_volume=114; citation_number=8; citation_pages=3219-3231; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=PDZ domains: structural modules for protein complex assembly.; citation_author=AY Hung; citation_author=M Sheng; citation_journal_title=J Biol Chem; citation_volume=277; citation_number=9; citation_pages=5699-5702; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Signal-processing machines at the postsynaptic density.; citation_author=MB Kennedy; citation_journal_title=Science; citation_volume=290; citation_number=10; citation_pages=750-754; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=PDZ domain proteins of synapses.; citation_author=E Kim; citation_author=M Sheng; citation_journal_title=Nat Rev Neurosci; citation_volume=5; citation_number=11; citation_pages=771-781; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=SAP family proteins.; citation_author=A Fujita; citation_author=Y Kurachi; citation_journal_title=Biochem Biophys Res Commun; citation_volume=269; citation_number=12; citation_pages=1-6; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Classification of PDZ domains.; citation_author=I Bezprozvanny; citation_author=A Maximov; citation_journal_title=FEBS Lett; citation_volume=509; citation_number=13; citation_pages=457-462; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=PDZ domains: troubles in classification.; citation_author=P Vaccaro; citation_author=L Dente; citation_journal_title=FEBS Lett; citation_volume=512; citation_number=14; citation_pages=345-349; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Making protein interactions druggable: targeting PDZ domains.; citation_author=KK Dev; citation_journal_title=Nat Rev Drug Discov; citation_volume=3; citation_number=15; citation_pages=1047-1056; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Recognition of unique carboxyl-terminal motifs by distinct PDZ domains.; citation_author=Z Songyang; citation_author=AS Fanning; citation_author=C Fu; citation_author=J Xu; citation_author=SM Marfatia; citation_journal_title=Science; citation_volume=275; citation_number=16; citation_pages=73-77; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=CRIPT, a novel postsynaptic protein that binds to the third PDZ domain of PSD-95/SAP90.; citation_author=M Niethammer; citation_author=JG Valtschanoff; citation_author=TM Kapoor; citation_author=DW Allison; citation_author=RJ Weinberg; citation_journal_title=Neuron; citation_volume=20; citation_number=17; citation_pages=693-707; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Selectivity and promiscuity of the first and second PDZ domains of PSD-95 and synapse-associated protein 102.; citation_author=IA Lim; citation_author=DD Hall; citation_author=JW Hell; citation_journal_title=J Biol Chem; citation_volume=277; citation_number=18; citation_pages=21697-21711; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Thermodynamic basis for promiscuity and selectivity in protein-protein interactions: PDZ domains, a case study.; citation_author=N Basdevant; citation_author=H Weinstein; citation_author=M Ceruso; citation_journal_title=J Am Chem Soc; citation_volume=128; citation_number=19; citation_pages=12766-12777; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Crystal structures of a complexed and peptide-free membrane protein-binding domain: molecular basis of peptide recognition by PDZ.; citation_author=DA Doyle; citation_author=A Lee; citation_author=J Lewis; citation_author=E Kim; citation_author=M Sheng; citation_journal_title=Cell; citation_volume=85; citation_number=20; citation_pages=1067-1076; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Quantification of PDZ domain specificity, prediction of ligand affinity and rational design of super-binding peptides.; citation_author=U Wiedemann; citation_author=P Boisguerin; citation_author=R Leben; citation_author=D Leitner; citation_author=G Krause; citation_journal_title=J Mol Biol; citation_volume=343; citation_number=21; citation_pages=703-718; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Novel mode of ligand recognition by the Erbin PDZ domain.; citation_author=G Birrane; citation_author=J Chung; citation_author=JA Ladias; citation_journal_title=J Biol Chem; citation_volume=278; citation_number=22; citation_pages=1399-1402; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Comparative structural analysis of the Erbin PDZ domain and the first PDZ domain of ZO-1. Insights into determinants of PDZ domain specificity.; citation_author=BA Appleton; citation_author=Y Zhang; citation_author=P Wu; citation_author=JP Yin; citation_author=W Hunziker; citation_journal_title=J Biol Chem; citation_volume=281; citation_number=23; citation_pages=22312-22320; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Searching for specificity in SH domains.; citation_author=JE Ladbury; citation_author=S Arold; citation_journal_title=Chem Biol; citation_volume=7; citation_number=24; citation_pages=R3-8; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Target-Assisted Iterative Screening reveals novel interactors for PSD95, Nedd4, Src, Abl and Crk proteins.; citation_author=A Kurakin; citation_author=D Bredesen; citation_journal_title=J Biomol Struct Dyn; citation_volume=19; citation_number=25; citation_pages=1-15; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Target-assisted iterative screening of phage surface display cDNA libraries.; citation_author=A Kurakin; citation_author=S Wu; citation_author=DE Bredesen; citation_journal_title=Methods Mol Biol; citation_volume=264; citation_number=26; citation_pages=47-60; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Filamentous fusion phage: novel expression vectors that display cloned antigens on the virion surface.; citation_author=GP Smith; citation_journal_title=Science; citation_volume=228; citation_number=27; citation_pages=1315-1317; citation_date=1985; " />
      <meta name="citation_reference" content="citation_title=Distinct ligand preferences of Src homology 3 domains from Src, Yes, Abl, Cortactin, p53bp2, PLCgamma, Crk, and Grb2.; citation_author=AB Sparks; citation_author=JE Rider; citation_author=NG Hoffman; citation_author=DM Fowlkes; citation_author=LA Quillam; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=93; citation_number=28; citation_pages=1540-1544; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Domain interaction between NMDA receptor subunits and the postsynaptic density protein PSD-95.; citation_author=HC Kornau; citation_author=LT Schenker; citation_author=MB Kennedy; citation_author=PH Seeburg; citation_journal_title=Science; citation_volume=269; citation_number=29; citation_pages=1737-1740; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Clustering of Shaker-type K+ channels by interaction with a family of membrane-associated guanylate kinases.; citation_author=E Kim; citation_author=M Niethammer; citation_author=A Rothschild; citation_author=YN Jan; citation_author=M Sheng; citation_journal_title=Nature; citation_volume=378; citation_number=30; citation_pages=85-88; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=An unconventional IAP-binding motif revealed by target-assisted iterative screening (TAIS) of the BIR3-cIAP1 domain. J Mol Recognit.; citation_author=A Kurakin; citation_author=DE Bredesen; citation_volume=20(1); citation_number=31; citation_pages=39-50; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=The PDZ1 domain of SAP90. Characterization of structure and binding.; citation_author=A Piserchio; citation_author=M Pellegrini; citation_author=S Mehta; citation_author=SM Blackman; citation_author=EP Garcia; citation_journal_title=J Biol Chem; citation_volume=277; citation_number=32; citation_pages=6967-6973; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=The dynamic organization of postsynaptic proteins: translocating molecules regulate synaptic function.; citation_author=A Inoue; citation_author=S Okabe; citation_journal_title=Curr Opin Neurobiol; citation_volume=13; citation_number=33; citation_pages=332-340; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Rapid redistribution of synaptic PSD-95 in the neocortex in vivo.; citation_author=NW Gray; citation_author=RM Weimer; citation_author=I Bureau; citation_author=K Svoboda; citation_journal_title=PLoS Biol; citation_volume=4; citation_number=34; citation_pages=e370; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=The concept of self-organization in cellular architecture.; citation_author=T Misteli; citation_journal_title=J Cell Biol; citation_volume=155; citation_number=35; citation_pages=181-185; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Self-organization versus Watchmaker: stochastic dynamics of cellular organization.; citation_author=A Kurakin; citation_journal_title=Biol Chem; citation_volume=386; citation_number=36; citation_pages=247-254; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=The dynamics of chromosome organization and gene regulation.; citation_author=DL Spector; citation_journal_title=Annu Rev Biochem; citation_volume=72; citation_number=37; citation_pages=573-608; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=SH3 domains: complexity in moderation.; citation_author=BJ Mayer; citation_journal_title=J Cell Sci; citation_volume=114; citation_number=38; citation_pages=1253-1263; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Self-Organization versus Watchmaker: ambiguity of molecular recognition and design charts of cellular circuitry.; citation_author=A Kurakin; citation_journal_title=J Mol Recognit; citation_volume=20; citation_number=39; citation_pages=In press. DOI: 10.1002/jmr.839; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Atypical recognition consensus of CIN85/SETA/Ruk SH3 domains revealed by target-assisted iterative screening.; citation_author=AV Kurakin; citation_author=S Wu; citation_author=DE Bredesen; citation_journal_title=J Biol Chem; citation_volume=278; citation_number=40; citation_pages=34102-34109; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=The classification of amino acid conservation.; citation_author=WR Taylor; citation_journal_title=J Theor Biol; citation_volume=119; citation_number=41; citation_pages=205-218; citation_date=1986; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000953" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="The PDZ Domain as a Complex Adaptive System"/>
    <meta name="twitter:description" content="Specific protein associations define the wiring of protein interaction networks and thus control the organization and functioning of the cell as a whole. Peptide recognition by PDZ and other protein interaction domains represents one of the best-studied classes of specific protein associations. However, a mechanistic understanding of the relationship between selectivity and promiscuity commonly observed in the interactions mediated by peptide recognition modules as well as its functional meaning remain elusive. To address these questions in a comprehensive manner, two large populations of artificial and natural peptide ligands of six archetypal PDZ domains from the synaptic proteins PSD95 and SAP97 were generated by target-assisted iterative screening (TAIS) of combinatorial peptide libraries and by synthesis of proteomic fragments, correspondingly. A comparative statistical analysis of affinity-ranked artificial and natural ligands yielded a comprehensive picture of known and novel PDZ ligand specificity determinants, revealing a hitherto unappreciated combination of specificity and adaptive plasticity inherent to PDZ domain recognition. We propose a reconceptualization of the PDZ domain in terms of a complex adaptive system representing a flexible compromise between the rigid order of exquisite specificity and the chaos of unselective promiscuity, which has evolved to mediate two mutually contradictory properties required of such higher order sub-cellular organizations as synapses, cell junctions, and others &ndash; organizational structure and organizational plasticity/adaptability. The generalization of this reconceptualization in regard to other protein interaction modules and specific protein associations is consistent with the image of the cell as a complex adaptive macromolecular system as opposed to clockwork."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0000953.g005"/>

  <meta property="og:title" content="The PDZ Domain as a Complex Adaptive System" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000953" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=9201'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=5674'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=25&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0000953">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000953" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000953&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    The PDZ Domain as a Complex Adaptive System
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Alexei Kurakin
              <span class="corresponding">mail</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:ak@alexeikurakin.org">ak@alexeikurakin.org</a></p>

                <p>Affiliation:
                  The Buck Institute for Age Research, Novato, California, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Andrzej Swistowski, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  The Buck Institute for Age Research, Novato, California, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Susan C. Wu, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  The Buck Institute for Age Research, Novato, California, United States of America
                </p>

                <p>
                    Current address: Department of Biochemistry and Biophysics, Lineberger Comprehensive Cancer Center, University of North Carolina at Chapel Hill, Chapel Hill, North Carolina, United States of America
                </p>

              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Dale E. Bredesen
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  The Buck Institute for Age Research, Novato, California, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: September 26, 2007</li>
    <li>DOI: 10.1371/journal.pone.0000953</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0000953" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0000953" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000953" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0000953" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0000953-g001" data-doi="info:doi/10.1371/journal.pone.0000953" data-uri="info:doi/10.1371/journal.pone.0000953.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000953.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000953-g002" data-doi="info:doi/10.1371/journal.pone.0000953" data-uri="info:doi/10.1371/journal.pone.0000953.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000953.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000953-g003" data-doi="info:doi/10.1371/journal.pone.0000953" data-uri="info:doi/10.1371/journal.pone.0000953.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000953.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000953-g004" data-doi="info:doi/10.1371/journal.pone.0000953" data-uri="info:doi/10.1371/journal.pone.0000953.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000953.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000953-g005" data-doi="info:doi/10.1371/journal.pone.0000953" data-uri="info:doi/10.1371/journal.pone.0000953.g005" title="Figure 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000953.g005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000953">Reader Comments (2)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0000953" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1"></a><p>Specific protein associations define the wiring of protein interaction networks and thus control the organization and functioning of the cell as a whole. Peptide recognition by PDZ and other protein interaction domains represents one of the best-studied classes of specific protein associations. However, a mechanistic understanding of the relationship between selectivity and promiscuity commonly observed in the interactions mediated by peptide recognition modules as well as its functional meaning remain elusive. To address these questions in a comprehensive manner, two large populations of artificial and natural peptide ligands of six archetypal PDZ domains from the synaptic proteins PSD95 and SAP97 were generated by target-assisted iterative screening (TAIS) of combinatorial peptide libraries and by synthesis of proteomic fragments, correspondingly. A comparative statistical analysis of affinity-ranked artificial and natural ligands yielded a comprehensive picture of known and novel PDZ ligand specificity determinants, revealing a hitherto unappreciated combination of specificity and adaptive plasticity inherent to PDZ domain recognition. We propose a reconceptualization of the PDZ domain in terms of a complex adaptive system representing a flexible compromise between the rigid order of exquisite specificity and the chaos of unselective promiscuity, which has evolved to mediate two mutually contradictory properties required of such higher order sub-cellular organizations as synapses, cell junctions, and others – organizational structure and organizational plasticity/adaptability. The generalization of this reconceptualization in regard to other protein interaction modules and specific protein associations is consistent with the image of the cell as a complex adaptive macromolecular system as opposed to clockwork.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>Kurakin A, Swistowski A, Wu SC, Bredesen DE (2007) The PDZ Domain as a Complex Adaptive System. PLoS ONE 2(9):
          e953.
            doi:10.1371/journal.pone.0000953</p><p><strong>Academic Editor: </strong>Wenqing Xu, University of Washington, United States of America</p><p><strong>Received:</strong> July 17, 2007; <strong>Accepted:</strong> August 30, 2007; <strong>Published:</strong> September 26, 2007</p><p><strong>Copyright:</strong> © 2007 Kurakin et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>Supported in part by NS33376 from the NIH to D.E.B.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>Protein interaction modules, such as PDZ, SH3, WW, EH, SH2 and other domains, mediate protein-protein interactions by recognizing and binding short and usually linear peptide epitopes within their interacting partners <a href="#pone.0000953-Kuriyan1">[1]</a>–<a href="#pone.0000953-Kay1">[4]</a>. The importance of this particular class of protein interactions <em>in vivo</em> is underscored by the estimates suggesting that a substantial fraction of all specific protein interactions in the cell may involve peptide recognition domains <a href="#pone.0000953-Castagnoli1">[3]</a>, <a href="#pone.0000953-Zucconi1">[5]</a>.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>PDZ domain is a prototypical and one of the best-characterized protein interaction modules. Approximately 90 amino acids long, PDZ domain was first discovered as sequence repeats in the primary structures of the post-synaptic density 95 (<span class="underline">P</span>SD95), disk-large (<span class="underline">D</span>lg) and zona occludens-1 (<span class="underline">Z</span>O-1) proteins <a href="#pone.0000953-Cho1">[6]</a>. Later it was identified in many other proteins and the first draft of the human genome ranked the PDZ domain family as number 19 among the most abundant domain families <a href="#pone.0000953-Lander1">[7]</a>. More than 400 different PDZ domains are currently estimated to exist in humans or in mice. PDZ domains often occur in multiple copies within proteins, as well as in various combinations with other types of protein interaction modules and/or functional domains. The abundance of PDZ domains in metazoan genomes together with the scarcity of canonical PDZ domains in non-metazoans indicates a possibly critical function of PDZ domains in multicellular organization <a href="#pone.0000953-Harris1">[8]</a>.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>While able to interact with internal amino acid sequences properly constrained within secondary structure, in their canonical and by far the most common mode of interaction PDZ domains recognize and bind short specific sequences at the extreme C-termini of their interacting partners <a href="#pone.0000953-Hung1">[9]</a>. Recognition of C-termini represents a form of non-invasive interaction well suited to mediate organization of transport, localization, sorting and spatial arrangement of proteins using their individual C-terminal tails recognized and handled by various PDZ domains. Perhaps not surprisingly, many PDZ domain proteins, especially those containing multiple copies of PDZ domains, function as scaffolds at the specialized membrane regions in the cell, where they manage organization and maintenance of large macromolecular complexes, such as signal-processing machinery at post-synaptic densities (PSD) <a href="#pone.0000953-Kennedy1">[10]</a>, <a href="#pone.0000953-Kim1">[11]</a>.</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p><span class="underline">P</span>ost-<span class="underline">s</span>ynaptic <span class="underline">d</span>ensity protein 95 (PSD95) is an archetypal member of the <span class="underline">s</span>ynapse-<span class="underline">a</span>ssociated <span class="underline">p</span>rotein (SAP) family of scaffolding molecules comprising PSD95/SAP90, SAP97, SAP102 and PSD93/chapsyn110. SAP proteins function as key organizers that control synaptic composition, organization and function <a href="#pone.0000953-Kim1">[11]</a>, <a href="#pone.0000953-Fujita1">[12]</a>. The members of the SAP family share the same overall domain organization with three N-terminal PDZ domains followed by an SH3 domain and a guanylate homology domain at the C-terminus (<a href="#pone-0000953-g001">Fig. 1</a>). All five domains appear to function as protein interaction modules mediating associations of SAP scaffolds with their multiple interacting partners <a href="#pone.0000953-Kim1">[11]</a>.</p>
<div class="figure" id="pone-0000953-g001"><div class="img"><a name="pone-0000953-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000953.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000953" data-uri="info:doi/10.1371/journal.pone.0000953.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000953.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000953.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000953.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000953.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000953.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000953.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000953.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000953.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000953.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Binding of 95 artificial phage-displayed ligands to six PDZ domains of PSD95 and SAP97.</span></strong></p><a id="article1.body1.sec1.fig1.caption1.p1" name="article1.body1.sec1.fig1.caption1.p1"></a><p>Binding histograms were obtained by individual phage ELISA performed on purified GST fusions of the indicated domains immobilized in micro-titer plate wells <a href="#pone.0000953-Kurakin6">[40]</a>. For accurate relative affinity evaluations and cross-domain comparison, the slopes of individual ELISA kinetics were determined and normalized by the highest slope value in each of the six sets shown. The 96<sup>th</sup> well in each set was loaded with a library aliquot to indicate background. Axis X indicates identification (i.d.) numbers of individual artificial ligands (see individual ligand sequences together with their i.d. numbers in <a href="#pone.0000953.s004">Table S1</a>). Axis Y indicates the normalized relative affinity. The domain organization of the PSD95 and SAP97 proteins is shown above histograms.</p>
<span>doi:10.1371/journal.pone.0000953.g001</span></div><a id="article1.body1.sec1.p5" name="article1.body1.sec1.p5"></a><p>Over the 15 years since the discovery of PDZ domains, the biochemistry and structural basis of PDZ domain recognition as well as the biology of PDZ domain-containing proteins have been subjects of numerous studies, which are summarized in a number of reviews <a href="#pone.0000953-Harris1">[8]</a>, <a href="#pone.0000953-Hung1">[9]</a>, <a href="#pone.0000953-Kim1">[11]</a>. There are, however, three major uncertainties that appear to persist in the PDZ domain field, despite extensive research efforts to clarify them. These are 1) the degree of selectivity of individual PDZ domains, 2) the significance of the ligand residues situated upstream of the last four C-terminal amino acids and 3) the physiological affinity range of PDZ domain interactions.</p>
<a id="article1.body1.sec1.p6" name="article1.body1.sec1.p6"></a><p>The first uncertainty is illustrated by the continual but so far failed attempts to classify PDZ domains in accord with their specificities (see the examples of at least five different classifications in Refs. <a href="#pone.0000953-Harris1">[8]</a>, <a href="#pone.0000953-Hung1">[9]</a>, <a href="#pone.0000953-Bezprozvanny1">[13]</a>-<a href="#pone.0000953-Dev1">[15]</a>). It is generally agreed that there are two major classes of PDZ domains – class I PDZ domains recognize and bind the C-termini of proteins conforming to the consensus sequence X-(S/T)-X-(V/I/L)-COOH, while class II PDZ domains interact with the C-terminal consensus X-Φ-X-Φ-COOH, where X is any amino acid and Φ stands for hydrophobic residue <a href="#pone.0000953-Songyang1">[16]</a>. In the last ten years following the identification of these two major classes, PDZ domain classification became increasingly complicated, mainly due to the discoveries that a) there are a number of, and potentially many, distinct PDZ domain specificity classes, b) at least some of the PDZ domains can be classified into more than one class, as they are able to interact with the peptide ligands that do not share a common consensus at their C-termini and c) some of the same class PDZ domains can clearly differentiate between the ligands sharing a class-defining consensus, such as X-(S/T)-X-(V/I/L)-COOH, for example <a href="#pone.0000953-Niethammer1">[17]</a>, <a href="#pone.0000953-Lim1">[18]</a>. At the same time, PDZ domains became notorious for their apparent promiscuity <a href="#pone.0000953-Lim1">[18]</a>, <a href="#pone.0000953-Basdevant1">[19]</a>.</p>
<a id="article1.body1.sec1.p7" name="article1.body1.sec1.p7"></a><p>The second major uncertainty pertains to the contribution of the ligand residues that are situated upstream of the last three to four C-terminal amino acids. Since the initial structural studies implicating only the few carboxy-terminal ligand residues in direct interactions with PDZ domains, it has been assumed that the influence of the upstream residues in PDZ ligands is inconsequential for PDZ domain interactions, and the occasional experimental evidence to the contrary is normally regarded as exceptional, relevant only for a particular domain or even a particular domain-ligand pair <a href="#pone.0000953-Dev1">[15]</a>, <a href="#pone.0000953-Doyle1">[20]</a>, <a href="#pone.0000953-Wiedemann1">[21]</a>. The generality of this assumption, however, becomes increasingly questionable as the examples demonstrating involvement of the upstream ligand residues continue to accumulate <a href="#pone.0000953-Niethammer1">[17]</a>, <a href="#pone.0000953-Birrane1">[22]</a>, <a href="#pone.0000953-Appleton1">[23]</a>.</p>
<a id="article1.body1.sec1.p8" name="article1.body1.sec1.p8"></a><p>The widely diverse affinities reported for PDZ domain-peptide interactions, spanning more than three orders of magnitude, represent another source of confusion. Because in-solution methods, such as fluorescence polarization (FP), tend to estimate PDZ domain-peptide interaction affinities in the low micromolar range, well within the affinity range expected from protein interaction domains mediating transient specific associations inside the cell, they are generally perceived as more trustworthy than the <em>K<sub>D</sub></em> values in the low to medium nanomolar range obtained by solid phase methods, such as surface plasmon resonance (SPR) or enzyme-linked immunosorbent assays (ELISA) <a href="#pone.0000953-Harris1">[8]</a>, <a href="#pone.0000953-Hung1">[9]</a>. However, since both PDZ domain-containing proteins and pertinent PDZ domain ligands are often found clustered at the specialized membrane regions, such as postsynaptic densities or cellular junctions, a question has been raised as to which method approximates the <em>in vivo</em> situation better and what affinities should be considered as physiologically relevant for PDZ domain-mediated interactions <a href="#pone.0000953-Hung1">[9]</a>.</p>
<a id="article1.body1.sec1.p9" name="article1.body1.sec1.p9"></a><p>It should be emphasized that the ambiguities detailed above for the PDZ domain family are common, to a larger or smaller degree, to all the peptide recognition domain families <a href="#pone.0000953-Castagnoli1">[3]</a>, <a href="#pone.0000953-Ladbury1">[24]</a>, thus creating an apparent paradox – how the cell achieves its highly organized state while relying on the molecular interactions of limited selectivity?</p>
<a id="article1.body1.sec1.p10" name="article1.body1.sec1.p10"></a><p>To address the above-mentioned questions in a comprehensive manner, we applied a number of novel biochemical and statistical approaches to generate and analyze large populations of peptide ligands for a number of well-studied PDZ domains. The results of this study reveal a hitherto unappreciated combination of specificity and adaptive plasticity inherent to PDZ domain recognition. The complexity of PDZ domain recognition and the seemingly contradictory and/or confusing observations accumulated in the field are reconciled within a novel, if unexpected, image of the PDZ domain emerging as a complex adaptive system evolved to ensure both structure and organizational plasticity of higher order dynamic macromolecular systems such as synapses, cell junctions, and others.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3>
<h4>Analysis of PDZ domain recognition: artificial ligands</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>This study capitalizes on distinct advantages of the novel screening format for phage-displayed peptide libraries, target-assisted iterative screening (TAIS), introduced recently and described elsewhere <a href="#pone.0000953-Kurakin1">[25]</a>, <a href="#pone.0000953-Kurakin2">[26]</a>. Omitting competition between individual binders and switching molecular context in target presentation, TAIS allows for selection of specific peptide binders to a given protein target in a wide range of affinities with no false positives, and thus provides unique and unexploited opportunity to generate large datasets for analysis of individual binders on the one-by-one basis, rather than, as is done traditionally, considering population averages (synthetic peptide library screens) <a href="#pone.0000953-Songyang1">[16]</a> or a few of the best binders only (conventional panning) <a href="#pone.0000953-Smith1">[27]</a>, <a href="#pone.0000953-Sparks1">[28]</a>. TAIS was applied to cDNA and random 16-mer peptide libraries in a search for peptide ligands of various PDZ domains of the SAP family of proteins. The relative affinities of the 95 artificial ligands isolated from peptide libraries towards six PDZ domains of the PSD95 and SAP97 proteins are shown in <a href="#pone-0000953-g001">Fig. 1</a> (see individual ligand sequences together with their i.d. numbers in <a href="#pone.0000953.s004">Table S1</a>).</p>
<a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2"></a><p>Visual examination of binding histograms suggests that 1) the recognition specificities of all six domains examined are very similar, albeit not identical; 2) the second domains of both proteins are noticeably more promiscuous than their first and third domains; and 3) the specificities of homologous domains across different proteins appear to be more similar than the specificities of the domains belonging to the same protein. The first two observations are in agreement with the established body of experimental evidence <a href="#pone.0000953-Niethammer1">[17]</a>, <a href="#pone.0000953-Lim1">[18]</a>, <a href="#pone.0000953-Kornau1">[29]</a>, <a href="#pone.0000953-Kim2">[30]</a>. The third observation suggests a likely evolutionary scenario, in which the SAP family of proteins originated by duplication-divergence of an individual PDZ domain within an ancestor protein, followed by duplication-divergence of a whole protein to generate the family members characterized by overlapping but distinct biological roles/functions <a href="#pone.0000953-Kim1">[11]</a>, <a href="#pone.0000953-Fujita1">[12]</a>.</p>
<a id="article1.body1.sec2.sec1.p3" name="article1.body1.sec2.sec1.p3"></a><p>To delineate recognition preferences of the target PDZ domains, the last sixteen C-terminal amino acids of the peptide ligands selected in TAIS screens were analyzed using the <span class="underline">r</span>esidue-<span class="underline">f</span>requency-<span class="underline">p</span>atterning (RFP) algorithm described recently <a href="#pone.0000953-Kurakin3">[31]</a>. The RFP procedure includes an analysis of statistical biases in relative frequencies of amino acid residues within a given set of peptide sequences followed by a search for patterns in the positioning of over-(under-)represented residues within individual peptides. The observed-to-expected ratios of individual amino acid frequencies within the whole set of artificial ligands are shown in <a href="#pone-0000953-g002">Fig. 2A</a>.</p>
<div class="figure" id="pone-0000953-g002"><div class="img"><a name="pone-0000953-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000953.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000953" data-uri="info:doi/10.1371/journal.pone.0000953.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000953.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000953.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000953.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000953.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000953.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000953.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000953.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000953.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000953.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>RFP (<span class="underline">r</span>esidue <span class="underline">f</span>requency <span class="underline">p</span>atterning) analysis of artificial ligands.</span></strong></p><a id="article1.body1.sec2.sec1.fig1.caption1.p1" name="article1.body1.sec2.sec1.fig1.caption1.p1"></a><p><em>A</em>, The observed-to-expected ratios of individual amino acid frequencies within the whole set of artificial peptide ligands isolated from cDNA and random peptide libraries by TAIS using SAP PDZ domains as targets. <em>B</em>, The frequency distributions of the indicated amino acids within the last sixteen C-terminal positions of aligned artificial peptide ligands. For both <em>A</em> and <em>B</em>: axis Y indicates the observed-to-expected frequency ratio values. The dotted line corresponds to the expected frequency value. Statistically significant overrepresentation is indicated by star symbols. <em>n</em> is sample size, <em>p</em> indicates the chi-square (A) or binomial (B) tests <em>P</em>-values. <em>C</em>, The aligned sequences of artificial peptide ligands are arranged in four groups based on their relative affinities to the PSD95-PDZ1. The numbers in parentheses indicate the range of normalized phage ELISA values within a given affinity group. The ligand positions from “−4” to “−7” are boxed to indicate the area of relative concentration of positively charged residues. Arginines and lysines are highlighted green, while aspartic and glutamic acids are red. The unique i.d. numbers of artificial ligands are indicated on the left from their sequences. The digits above columns indicate the C-terminal position numbering of ligand residues. Analogous arrangements of ligands for other five PDZ domains are shown in <a href="#pone.0000953.s001">Fig. S1A</a>, <a href="#pone.0000953.s002">Fig. S1B</a> and <a href="#pone.0000953.s003">Fig. S1C</a>.</p>
<span>doi:10.1371/journal.pone.0000953.g002</span></div><a id="article1.body1.sec2.sec1.p4" name="article1.body1.sec2.sec1.p4"></a><p>The relative frequencies of four residues, valine, arginine, threonine and serine are twice as high as expected. Since the probability of such frequencies arising by chance is vanishingly low (about 2.8E-15, assuming Bernoulli trials approximation), it is fair to hypothesize that V, R, T and S are the ligand residues that are preferred at the domain-ligand interaction interface and thus are likely to be important for binding to the target PDZ domains. The relative overrepresentation of V, T and S does not come as a surprise, as the known minimal recognition consensus of the PSD95 PDZ domains is X-(S/T)-X-(V/I/L)-COOH <a href="#pone.0000953-Lim1">[18]</a>. However, the essential role of the ligand's arginines in the SAP PDZ domain recognition has not been described previously. Analysis of the distribution of overrepresented residues within the last sixteen amino acids of artificial ligands (<a href="#pone-0000953-g002">Fig. 2B</a>) reveals that arginines are concentrated mainly at the ligand positions from “−4” to “−7”, suggesting important contributions of these positions to specificity and/or affinity of the interactions studied (by convention, residues of PDZ ligands are numbered starting with the last C-terminal residue as occupying the position “0”, penultimate residue as occupying the position “−1” and so forth, moving along the ligand sequence from C- to N-terminus).</p>
<a id="article1.body1.sec2.sec1.p5" name="article1.body1.sec2.sec1.p5"></a><p>To explore the relationships between amino acid frequencies at specific ligand positions and the strength of PDZ domain-ligand interactions we arranged peptide ligands into four groups in accord with their relative affinities: 1) best binders (normalized phage ELISA signal from 0.8 to 1.0); 2) good binders (0.6 to 0.8 ELISA signal); 3) moderate binders (0.4 to 0.6 ELISA signal) and 4) weak binders (0.2 to 0.4 ELISA signal) (see <a href="#pone-0000953-g002">Fig. 2C</a>). Independent of affinity ranking, the “−2” and “0” positions of ligands are almost invariably occupied by threonine and valine residues, respectively. Of note are the overwhelming preference for threonine over serine at the position “−2” and the practically complete dominance of valine over other hydrophobic residues in the role of the last C-terminal residue (<a href="#pone-0000953-g002">Fig. 2C</a>). The same statistical biases with respect to the “−2” and “0” positions have been observed for all six domains examined (see <a href="#pone.0000953.s001">Fig. S1A</a>, <a href="#pone.0000953.s002">Fig. S1B</a> and <a href="#pone.0000953.s003">Fig. S1C</a>), which is suggestive that both T and V are essential for either strong or relatively weak interactions with the PDZ domains examined, and that their substitution by other amino acids is likely to be either disruptive, or, as in the case of conservative substitutions such as T -&gt;S or V -&gt;I or L, significantly detrimental for binding. This conclusion is consistent with the previously reported mutational analysis of a PSD95 and SAP102 natural ligand, demonstrating that the substitution of S by T at the “−2” ligand position results in at least two-fold increase in binding affinity, and that valine as the last C-terminal residue is by far superior for binding to the first two PDZ domains of PSD95 and SAP102 in comparison to isoleucine or leucine <a href="#pone.0000953-Lim1">[18]</a>. The third overrepresented residue, serine, is rather evenly dispersed along peptide ligands with the only obvious preference for the position “−2”, as expected, where it occurs more often than in other positions, albeit infrequently relative to threonine. The fourth overrepresented residue, arginine, tends to be absent within the last four C-terminal positions of peptide ligands, in all affinity groups. Instead, arginine concentrates at the ligand positions from “−4 to −7”, with noticeable preference for the position “−4” in strong binders. In addition, visual inspection of aligned peptide ligands indicates that glutamate is clearly a preferred residue at the position “−3”, even though glutamate is not an overrepresented residue overall (<a href="#pone-0000953-g002">Fig. 2C</a>). The significant decrease in the relative frequencies of arginine and glutamate at the ligand positions “−4” and “−3”, correspondingly, when one compares statistics of strongest versus weakest binders, suggests that the presence of arginine and glutamate in these positions is essential for strong interactions with the target PDZ domains (see <a href="#pone.0000953.s001">Fig. S1A</a>, <a href="#pone.0000953.s002">Fig. S1B</a> and <a href="#pone.0000953.s003">Fig. S1C</a>).</p>
<a id="article1.body1.sec2.sec1.p6" name="article1.body1.sec2.sec1.p6"></a><p>As the described positional patterns of overrepresented residues hold for all six target domains (not shown), it is fair to conclude that the general recognition consensus of the PSD95 and SAP97 PDZ domains is X-R-E-(T/S)-X-V-COOH. Indeed, this inferred consensus represents a refinement of the well-known minimal recognition consensus of the class I PDZ domains, X-(S/T)-X-(V/I/L)-COOH, first defined for the PSD95 PDZ domains through analysis of C-terminal sequences in the PSD95 interacting partners <a href="#pone.0000953-Kornau1">[29]</a>, <a href="#pone.0000953-Kim2">[30]</a> and later confirmed by structural and biochemical studies <a href="#pone.0000953-Songyang1">[16]</a>, <a href="#pone.0000953-Lim1">[18]</a>, <a href="#pone.0000953-Doyle1">[20]</a>, <a href="#pone.0000953-Piserchio1">[32]</a>. However, the prediction of natural ligands of the PSD95 PDZ domains in protein databases using this refined consensus as a query poses the following problem. Consider, as an example, the affinity-sorted sets of the PSD95-PDZ1 domain ligands shown in <a href="#pone-0000953-g002">Fig. 2C</a>. Notice that approximately 36% of the best binders and 68% of good binders do not feature arginine at their “-4” positions, and thus the query X-R-E-(T/S)-X-V-COOH is likely to miss a very significant fraction of natural ligands in protein databases. Relaxing the consensus to X-E-(T/S)-X-V-COOH is not very helpful either, for only about 52% of the best and good binders selected in our screens feature glutamate residues at their “−3” positions (<a href="#pone-0000953-g002">Fig. 2C</a>). Therefore, in an attempt to extract from protein databases as many interactors of the PSD95 PDZ domains as possible, while minimizing spurious hits, we decided to consider as putative PSD95 PDZ interactors only those proteins that present at their C-termini amino acid sequences matching either a) the consensus X-E-(T/S)-X-V-COOH, b) the last four amino acids of all the best and good binders of the PSD95 PDZ domains, or c) the last four amino acid residues of the known interacting partners of the PSD95 PDZ domains reported in web-based protein interaction databases such as MINT, PPID and IntAct. The decision to focus only on the last four C-terminal positions was driven by the prevailing assumption that only the last three to four amino acids of peptide ligands are essential for PDZ domain-mediated interactions <a href="#pone.0000953-Dev1">[15]</a>, <a href="#pone.0000953-Doyle1">[20]</a>, <a href="#pone.0000953-Wiedemann1">[21]</a>.</p>


<h4>Analysis of PDZ domain recognition: natural ligands</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>A search of SWISS-PROT and TrEMBL databases with the described above queries gave 126 potential interacting partners for the PSD95 PDZ domains (see the individual C-terminal sequences of putative interactors together with their i.d. numbers in <a href="#pone.0000953.s005">Table S2</a>). In order to verify the predicted interactions and to evaluate relative affinities of individual natural ligands to the target domains, we synthesized 126 N-terminally biotinylated 15-mer proteomic fragments corresponding to our hits and assayed them for binding to each of the target domains <em>in vitro</em>, using the peptide ELISA assay <a href="#pone.0000953-Kurakin3">[31]</a> (<a href="#pone-0000953-g003">Fig. 3</a>).</p>
<div class="figure" id="pone-0000953-g003"><div class="img"><a name="pone-0000953-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000953.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000953" data-uri="info:doi/10.1371/journal.pone.0000953.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000953.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000953.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000953.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000953.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000953.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000953.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000953.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000953.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000953.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Binding of 126 proteomic fragments (natural ligands) to six PDZ domains of PSD95 and SAP97.</span></strong></p><a id="article1.body1.sec2.sec2.fig1.caption1.p1" name="article1.body1.sec2.sec2.fig1.caption1.p1"></a><p>Binding histograms were obtained by peptide ELISA performed on purified GST fusions of the indicated domains immobilized in micro-titer plate wells as described previously <a href="#pone.0000953-Kurakin3">[31]</a>. The X axis indicates the i.d. numbers of natural ligands (see <a href="#pone.0000953.s005">Table S2</a>). The Y axis indicates the peptide ELISA kinetics slope value in arbitrary units (a.u.). The 127<sup>th</sup> and 128<sup>th</sup> wells in each of the six sets were loaded with irrelevant peptides to indicate background signal. To illustrate internal consistency of the affinity evaluations obtained by peptide ELISA and their external consistency with the previously published affinity measurements, the reported affinities of the five PDZ domain-ligand pairs obtained by three different research groups using fluorescence polarization <a href="#pone.0000953-Niethammer1">[17]</a>, <a href="#pone.0000953-Lim1">[18]</a>, <a href="#pone.0000953-Piserchio1">[32]</a> are shown. The previously reported affinities provide calibration, suggesting that the individual signals that are higher than 200 a.u. roughly correspond to the interaction affinities of 15 µM <em>K<sub>D</sub></em> or stronger.</p>
<span>doi:10.1371/journal.pone.0000953.g003</span></div><a id="article1.body1.sec2.sec2.p2" name="article1.body1.sec2.sec2.p2"></a><p>From visual inspection of the binding histograms shown in <a href="#pone-0000953-g003">Fig. 3</a>, it is evident that the highest degree of promiscuity exhibited by the second PDZ domains of both proteins towards natural ligands and the more pronounced similarities in ligand preferences between homologous domains across different proteins rather than between different PDZ domains within the same protein recapitulate the patterns previously observed for artificial ligands (<a href="#pone-0000953-g001">Fig.1</a>). Paradoxically, however, the third PDZ domains of both proteins, PSD95 and SAP97, appear to be significantly more selective toward natural ligands than one would expect from their rather promiscuous interactions with artificial ligands (compare <a href="#pone-0000953-g001">Fig. 1</a> and <a href="#pone-0000953-g003">Fig. 3</a>).</p>
<a id="article1.body1.sec2.sec2.p3" name="article1.body1.sec2.sec2.p3"></a><p>To pinpoint the molecular determinants in natural ligands that are responsible for strong interactions with the target PDZ domains we looked for statistical biases in the relative amino acid frequencies within the positional window “−4 to −7”, the fully degenerate positions in our queries. It is worth emphasizing that all 126 natural ligands had been selected based on their match with the last four C-terminal amino acids of artificial ligands only. In other words, if the residues upstream of the last four amino acids in natural ligands were relatively unimportant for interactions with the target PDZ domains, one would expect no significant biases in amino acid frequencies at those positions. If, on the contrary, they are both essential and specific for the target domains, then the statistical biases within this region in natural ligands should be analogous to, or at least reminiscent of, the amino acid frequency patterns observed within the same positional window in artificial ligands.</p>
<a id="article1.body1.sec2.sec2.p4" name="article1.body1.sec2.sec2.p4"></a><p>The histograms in <a href="#pone-0000953-g004">Fig. 4</a> summarize and compare the amino acid compositional biases observed within the positional window “−4 to −7” of various natural and artificial ligand sets of the PSD95-PDZ1 domain. Essentially the same compositional biases were observed for natural and artificial ligands of the PSD95-PDZ2 domain and the first two SAP97 PDZ domains (not shown). Altogether, the results suggest that the positions upstream of the last four C-terminal residues in natural ligands of the first two PDZ domains do exhibit statistical biases in amino acid frequencies, and that these statistical biases do match closely the amino acid frequency patterns observed within the same positions in artificial ligands. The overrepresentation of positively charged residues, arginine and lysine, within the “−4 to −7” positional window of peptide ligands, both natural and artificial, correlates with high affinity of domain-ligand interactions, while the presence of negatively charged residues, aspartate and glutamate, within the same positional window correlates with poor or no binding at all. Noticeably, positively charged residues, especially arginine, while dispersed within the “−4 to −7” positional window in natural binders, tend to concentrate at the position “−4” in the strongest natural binders, faithfully recapitulating the arginine positional pattern observed in artificial ligands (compare <a href="#pone-0000953-g002">Fig. 2C</a> and <a href="#pone-0000953-g004">Fig. 4C</a>). Therefore, we conclude that the ligand positions “−4 to −7” are essential for selectivity and affinity of PDZ domain interactions, at least in the specific case of the first two PDZ domains of the PSD95 and SAP97 proteins. The paucity of strong natural binders for the third PDZ domains precluded analogous analysis of their natural ligands. However, the overall recognition pattern of the PSD95-PDZ3 domain is likely to be very similar to that of the first two PSD95 PDZ domains, as evidenced by the analysis of artificial ligands of the third PDZ domains discussed previously and by the comparative analysis of PSD95-PDZ2 and PSD95-PDZ3 natural ligands that follows next.</p>
<div class="figure" id="pone-0000953-g004"><div class="img"><a name="pone-0000953-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000953.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000953" data-uri="info:doi/10.1371/journal.pone.0000953.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000953.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000953.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000953.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000953.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000953.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000953.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000953.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000953.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000953.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>Analysis of amino acid frequency biases within the positional window “−4 to −7” in artificial and natural binders of the PSD95-PDZ1 domain.</span></strong></p><a id="article1.body1.sec2.sec2.fig2.caption1.p1" name="article1.body1.sec2.sec2.fig2.caption1.p1"></a><p><em>A</em>, The observed-to-expected ratios of individual (top) and grouped (bottom) amino acid frequencies within the positional window “−4 to −7” in the 32 best artificial (open bars) and 32 best natural (filled bars) peptide ligands. <em>B</em>, The observed-to-expected ratios of individual (top) and grouped (bottom) amino acid frequencies within the positional window “−4 to −7” in the 32 worst natural peptide ligands. Grouping of twenty natural amino acids into eleven conserved physicochemical classes <a href="#pone.0000953-Taylor1">[41]</a> is shown below histograms. Statistically significant over-(under)representation is indicated by star symbols. <em>n</em> is sample size, <em>p</em> indicates the binomial test <em>P</em>-values. <em>C</em>, Comparison of the best (on the left) and worst (on the right) natural binders of the PSD95-PDZ1 domain. The i.d. numbers of natural ligands are indicated on the left from their sequences. Arginines and lysines are highlighted green, while aspartates and glutamates are red. Notice how the relative abundance of negatively charged residues at the “−1” position in the best binders mirrors the relative abundance of positively charged residues at the “−1” position in the worst binders.</p>
<span>doi:10.1371/journal.pone.0000953.g004</span></div>

<h4>Differential recognition by the same class PDZ domains</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>In order to gain insight into the pattern recognition differences of the PDZ domains that belong to the same class but are able to differentiate between peptide ligands sharing the class-defining C-terminal consensus, we investigated a particular case of this general puzzle, namely, the differences in recognition preferences between the second and third PDZ domains of PSD95, which are classified as type I PDZ domains but are known to discriminate between various X-(S/T)-X-(V/I/L)-COOH ligands <a href="#pone.0000953-Niethammer1">[17]</a>, <a href="#pone.0000953-Lim1">[18]</a>. In <a href="#pone-0000953-g005">Fig. 5A</a> we compare two sets of C-terminal sequences. The first set represents the sequences of natural ligands that bound strongly to the PSD95-PDZ2 domain but showed poor or no binding to the PSD95-PDZ3 domain. The second set shows the sequences of the PSD95-PDZ3 strongest binders. From this comparison, one can discern the following patterns: the PSD95-PDZ3 domain appears 1) to disfavor D at the “−1” ligand position; 2) to prefer T over S at the “−2” position; 3) to prefer R or K, while disfavoring I, V or L at the “−4” ligand position and 4) to favor positively charged residues in the “−4 to −7” positional window and thus, by inference, to be especially sensitive to negatively charged residues at those positions. Correspondingly, the PSD95-PDZ2 domain appears to rely on hydrophobic interactions mainly, both within the “−4 to −7” ligand positions and at the “−1” position. While a significantly larger number of different ligand sequences is needed for formal statistical validation of these tentative patterns, they are good enough to rationalize why certain PSD95 interacting partners, such as, for example, NMDA (<em>N</em>-metyl-D-aspartate) receptor subunits NR2A and NR2B with their C-terminal sequences KKMPSIESDV-COOH and EKLSSIESDV-COOH, respectively, or voltage-gated potassium channel subunits Kv1.1, Kv1.2 and Kv1.3 (C-termini VNKSKLLTDV-COOH, VNITKMLTDV-COOH and VNIKKIFTDV-COOH, correspondingly), bind to the first two PSD95 PDZ domains but fail to interact with the third PSD95 PDZ domain <a href="#pone.0000953-Lim1">[18]</a>, <a href="#pone.0000953-Kornau1">[29]</a>, <a href="#pone.0000953-Kim2">[30]</a>. It is also likely that these patterns, tentative as they are, possess reasonable predictive power. One could hypothesize, for example, that receptor-type tyrosine protein phosphatases gamma (SWISS-PROT i.d.#P23470, C-terminus DPAESMESLV-COOH, peptide #17 in the natural ligand dataset) and zeta (SWISS-PROT i.d.# P23471, C-terminus NIAESLESLV-COOH, peptide #18 in the natural ligand dataset) would bind to the second PDZ domains of PSD95 and SAP97 but not to their third PDZ domains, as they did (<a href="#pone-0000953-g003">Fig. 3</a>), and so forth.</p>
<div class="figure" id="pone-0000953-g005"><div class="img"><a name="pone-0000953-g005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000953.g005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000953" data-uri="info:doi/10.1371/journal.pone.0000953.g005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000953.g005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000953.g005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000953.g005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000953.g005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000953.g005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000953.g005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000953.g005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000953.g005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000953.g005.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 5.  <span>Pattern recognition differences between individual PDZ domains of PSD95.</span></strong></p><a id="article1.body1.sec2.sec3.fig1.caption1.p1" name="article1.body1.sec2.sec3.fig1.caption1.p1"></a><p><em>A</em>, Comparison of the sequences that exhibited the highest differential ratios in their relative affinities to PDZ2 domain versus PDZ3 domain (on the left) to the sequences of the best PDZ3 binders (on the right). <em>B</em>, Comparison of the sequences that showed the highest differential ratios in their relative affinities to PDZ2 domain versus PDZ1 domain (on the left) to the sequences of best PDZ1 binders (on the right). Arginines and lysines are shown green, aspartates and glutamates are red.</p>
<span>doi:10.1371/journal.pone.0000953.g005</span></div><a id="article1.body1.sec2.sec3.p2" name="article1.body1.sec2.sec3.p2"></a><p>To generalize, we suggest that by increasing statistical power one is likely to detect and define robust differences in pattern recognition between any pair of individual PDZ domains. Indeed, the comparison of the natural ligands exhibiting several-fold preference, in terms of relative affinity, for the PSD95-PDZ2 domain over the PSD95-PDZ1 domain with the best binders of the latter clearly suggests that these domains are also capable of differential recognition (<a href="#pone-0000953-g005">Fig. 5B</a>). It appears that the accumulation of sub-optimal residues within the last 6 or 7 positions renders the first domain sensitive to the negatively charged residues at and upstream of the “−6” or “−7” positions, while the second domain, relying on hydrophobic interactions, retains significant affinity for such ligands. Speaking of methodological advantages, it should be noted that, in addition to providing high-resolution power, statistical analysis of large datasets of binders is relatively insensitive to experimental errors and variations. Perhaps even more importantly, this type of analysis is free from certain limiting assumptions implicit in more traditional approaches, thus allowing for discovery of novel unanticipated patterns. For example, the conventional “lock-and-key”-type interpretations assume independence in energetic contributions of individual ligand residues, thus implying that the inferences made from the analysis of population averages or of a few ligands are valid for all individual ligands. Whether this assumption is true or not can only be answered through systematic analysis of large populations of the “isogenic” interactions on the one-by-one basis, which have been notably lacking, mainly due to the very same assumption.</p>

</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p>Summarizing the recognition patterns of the PSD95 and SAP97 PDZ domains inferred from the comparative analysis of amino acid organization of their affinity-ranked artificial and natural ligands, the PSD95 and SAP97 PDZ domains prefer positively charged residues, lysine and arginine, in the positional window “−4 to −7”, while strongly favoring lysine or arginine at the position “−4”, glutamate at the position “−3”, threonine at the position “−2” and valine at the position “0”. Even though the ligand position “−1” appears to accept various residues, it is apparently used for discrimination between individual PDZ domains – lysines and arginines are disfavored in this position by the first two PDZ domains (<a href="#pone-0000953-g004">Fig. 4C</a> and not shown), which is in agreement with the reported mutational analysis of the NR2B C-terminus <a href="#pone.0000953-Lim1">[18]</a>, while aspartate at the “−1” position is not well tolerated by the third PDZ domain, in agreement with the mutational analysis of the CRIPT C-terminal peptide <a href="#pone.0000953-Niethammer1">[17]</a>. We also noticed that none of the natural binders and none of the artificial ligands feature proline at the “−4” position, suggesting a possible advantage of keeping the main chain of ligands unconstrained at this position. The relative promiscuity of the second PDZ domain can be explained by its apparent reliance on hydrophobic interactions within the positional window “−4 to −7” and, likely, at the position “−1”, which makes this domain less sensitive to lysine/arginine versus aspartate/glutamate compositional biases, thus allowing for a much greater variety of amino acids acceptable at these positions. Indeed, a number of strong binders of the second PDZ domain feature leucine, isoleucine, valine or even aspartate in the positions often occupied by lysine or arginine in the strong binders of the first and third PDZ domains (<a href="#pone-0000953-g005">Fig. 5</a> and not shown).</p>
<a id="article1.body1.sec3.p2" name="article1.body1.sec3.p2"></a><p>On the whole, it appears that PDZ domain interactions are driven by the interdependent contributions of multiple ligand positions to the overall energy of interaction, which may span the last eight or more C-terminal amino acids of ligands. The unexpected plasticity and complexity of PDZ recognition are rooted in an apparently integral nature of the individual ligand residue contributions. Sub-optimal amino acids at some of the ligand positions can be compensated by optimal amino acids at other positions to preserve the strength of interaction. At the same time, even the major favorable energetic contributions of threonine and valine at the “−2” and “0” positions can be compromised by delinquent residues acting somewhere else along the chain. In the ligands featuring sub-optimal amino acids within the last four or five C-terminal ligand positions the individual contributions of upstream residues may become critical, thus allowing for highly differential recognition of such ligands by very similar PDZ domains.</p>
<a id="article1.body1.sec3.p3" name="article1.body1.sec3.p3"></a><p>The paradoxical behavior of the third PDZ domains of PSD95 and SAP97, which appear to be exquisitely selective towards natural ligands, but promiscuous toward artificial ligands, is unlikely to find its explanation in the physicochemical idiosyncrasies of the third domains only. Instead, we suggest that what appears as the exquisite selectivity of the third domain towards natural ligands may simply reflect the selective pressures imposed by evolution on functional organization of the postsynaptic density, which led to a relatively limited number of the PDZ3 ligands encoded in the genome. In this regard, a few examples from the artificial ligands dataset are most illustrative. Tryptophan is a significantly overrepresented amino acid at the “−1” position in artificial ligands, in all affinity groups, suggesting that the presence of tryptophan at the “−1” ligand position is not detrimental for interaction <em>per se</em>, with any of the target domains (<a href="#pone-0000953-g002">Fig. 2C</a>, <a href="#pone.0000953.s001">Fig. S1A</a>, <a href="#pone.0000953.s002">Fig. S1B</a> and <a href="#pone.0000953.s003">Fig. S1C</a>). If anything, it appears to be advantageous. However, only nine proteins out of the 14550 human protein entries in the SWISS-PROT database have the C-termini matching the consensus X-(S/T)-W-V-COOH. As another example, the PSD95-PDZ3 domain has selected a set of unusual peptides with the C-terminus W-Y-H-S-F-COOH (see <a href="#pone.0000953.s004">Table S1</a>), with which it interacts selectively and with reasonable affinity <em>in vitro</em> (<a href="#pone-0000953-g001">Fig. 1</a>, peptides # 17, 18, 19 and 20, and not shown). None of the human proteins appears to have such C-termini. Apparently, SAP PDZ domains are open to a much larger spectrum of interactions than that encoded in the genomic sequences.</p>
<a id="article1.body1.sec3.p4" name="article1.body1.sec3.p4"></a><p>What is then the biological meaning of adaptive plasticity in PDZ domain recognition? And what are physiologically relevant affinities of PDZ domain interactions? We speculate that both the adaptive plasticity and the wide range of interaction affinities of SAP PDZ domains are directly linked to the managerial/scaffolding role of SAP proteins in synapse organization. It is fair to suggest that both a large ligand sequence space and a wide affinity range of scaffold-mediated interactions are beneficial, if not essential, for synapse plasticity, because they define the spatio-temporal ranges within which the synapse organizational dynamics operate. The imaging studies of molecular dynamics in living cells, tissues and animals indicate that synapses <a href="#pone.0000953-Inoue1">[33]</a>, <a href="#pone.0000953-Gray1">[34]</a>, as well as many other, and maybe all, sub-cellular structures <a href="#pone.0000953-Misteli1">[35]</a>, <a href="#pone.0000953-Kurakin4">[36]</a>, are maintained as steady-state metastable molecular organizations by continuous flux of their resident components entering and leaving organizations, with individual components following individual dynamics, from very slow to very fast, as defined by specific protein associations taking place within a given sub-cellular organization. We suggest that within this image/framework of dynamic synapse, where both molecular composition and organization of the synapse at any given moment are defined to a large extent by available PDZ domain-containing scaffolds, by a particular assortment of the C-termini present at the synaptic site and by <em>the competition</em> between available C-termini for available PDZ domains, the adaptive plasticity and wide affinity range of PDZ scaffold-mediated interactions emerge as essential pre-requisites of synaptic compositional and organizational flexibility. The selectivity of PDZ domain interactions, on the other hand, ensures a certain degree of order and organizational structure required to perform synaptic functions.</p>
<a id="article1.body1.sec3.p5" name="article1.body1.sec3.p5"></a><p>We also speculate that the synaptic environmental and organizational invariants are encoded in the genome in the form of matching spectra of synapse-associated PDZ domains and their cognate ligands. In this way the genome loosely specifies the overall schematics and principles of synapse organization, while maturation, fine-tuning, and adaptation of individual synaptic structures take place as a result of their individual development and experience. In the same sense as neuronal organization of every newborn brain has been shaped by evolution to recognize certain perceptual/environmental invariants, but is not limited to recognition of those patterns only, the PDZ domains have been shaped by evolution to recognize certain C-terminal sequences present in a given proteome, but are not limited to the recognition of those sequences only. In this way, the composition, organization, and functioning of individual synapses remain open for evolution at both ontogenetic and phylogenetic levels, accommodating novel C-terminal sequences that can potentially arise from a plethora of the epigenetic and genetic molecular mechanisms known to generate molecular diversity, including posttranslational modifications, regulated proteolysis, RNA splicing, mutations, DNA rearrangements, protein splicing, and others. In short, we suggest that the adaptive plasticity of SAP PDZ domain recognition and the wide affinity range of SAP PDZ domain interactions are evolutionarily enforced by requirements of synapse plasticity and reflect the managerial role of SAP scaffolds in synaptic organizational dynamics.</p>
<a id="article1.body1.sec3.p6" name="article1.body1.sec3.p6"></a><p>It should be emphasized that the proposed conceptualization of the PDZ domain as a complex adaptive system evolved to ensure both structure and organizational flexibility of higher order macromolecular organizations not only resolves the uncertainties pertaining to PDZ domain recognition, but also suggests a fundamental molecular mechanism underlying the adaptive plasticity of sub-cellular molecular organization revealed in a number of the recent studies in which advanced imaging techniques were used to address molecular dynamics in living cells <a href="#pone.0000953-Inoue1">[33]</a>, <a href="#pone.0000953-Misteli1"></a>,<a href="#pone.0000953-Kurakin4"></a>,<a href="#pone.0000953-Spector1">[35−37]</a>. Specifically, we suggest that peptide recognition modules, such as PDZ, SH3, SH2, WW, EH and other domains, which show both selectivity and promiscuity in their interactions <a href="#pone.0000953-Castagnoli1">[3]</a>, <a href="#pone.0000953-Basdevant1">[19]</a>, <a href="#pone.0000953-Ladbury1">[24]</a>, <a href="#pone.0000953-Mayer1">[38]</a>, function as adaptive molecular “synapses” of cellular protein interaction networks, rather than perform as Lego Block-like elements for assembly of pre-defined and immutable structures. The combination of selectivity and plasticity may constitute inherent property of all specific protein interactions, for it has clear evolutionary advantages over mechanistic self-assembly, allowing the cell 1) to capitalize on the evolutionary memory manifested as the limited selectivity of specific protein associations, 2) to adapt its organization to a given environmental context and 3) to explore new variants of intracellular molecular organization in the course of biological evolution.</p>
<a id="article1.body1.sec3.p7" name="article1.body1.sec3.p7"></a><p>In the same sense as the overall organization of a newborn brain represents, essentially, a form of evolutionary memory, subject to both ontogenetic and phylogenetic development and maturation, the overall organization of cellular protein interaction networks encoded in the matching spectra of peptide interaction modules and their cognate ligands within a given genome may represent an evolutionary memory that is subject to ontogenetic and phylogenetic development, maturation, and adaptation (see reference <a href="#pone.0000953-Kurakin5">[39]</a> introducing the concept of evolutionary memory).</p>
</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Materials and methods"></a><h3>Materials and methods</h3>
<h4>Phage display libraries, peptides, proteins and antibodies</h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p>The 16-mer random peptide library was generated in-house using the T7 phage display library construction kit from Novagen. The human brain cDNA library was purchased from Novagen. The GST fusion protein expression constructs of the PSD95-PDZ2, PSD95-PDZ3, SAP97-PDZ1 and SAP97-PDZ2 domains were kindly provided by Dr. B. K. Kay (The University of Illinois at Chicago). The GST fusion constructs of the PDS95-PDZ1 and SAP97-PDZ3 domains were generated by PCR amplification of the corresponding PDZ domain coding regions from SAP cDNAs (generously provided by Dr. David S. Bredt (University of California, San Francisco)) followed by cloning into the pGEX2TK expression vector (Amersham Pharmacia). The PDZ domain borders were defined by the SMART software tools (<a href="http://smart.embl-heidelberg.de/">http://smart.embl-heidelberg.de/</a>). All the constructs were verified by sequencing. Expression and protein purification of GST fusions were performed in accord with manufacturer's instructions. The detailed protein purification protocols used in this work can be found at <a href="http://www.buckinstitute.org/TAIS">http://www.buckinstitute.org/TAIS</a>. The synthetic biotinylated peptides corresponding to natural ligand sequences were purchased from JPT Peptide Technologies GmbH (Berlin, Germany). Rabbit polyclonal anti-T7 antibodies were a generously gift from Dr. F. W. Studier, Brookhaven National Laboratory. Donkey anti-rabbit antibodies conjugated to horseradish peroxidase were purchased from Amersham Biosciences.</p>


<h4>Target-assisted iterative screening (TAIS)</h4>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1"></a><p>A detailed description of the TAIS method is presented in Kurakin et al. (<em>20</em>). The TAIS flowchart and protocols can be found on the Internet (<a href="http://www.buckinstitute.org/TAIS">http://www.buckinstitute.org/TAIS</a>). Briefly, 30 µg of a GST-PDZ domain fusion immobilized on sepharose beads were blocked with 0.5% bovine serum albumin (BSA) in TBS-T (Tris-buffered saline, pH 7.4+0.1% Tween 20) and incubated with a phage-displayed peptide library aliquot (approx. 10<sup>8</sup>–10<sup>9</sup> pfu). After 90 minutes of incubation at room temperature (RT) the beads were thoroughly washed with TBS-T and bound phages were eluted with 200 µl of 1% SDS for 15 min at RT. Following elution, the phages were immediately mixed with a molten 0.6% top agarose containing host cells and plated onto two pre-warmed 150 mm agar plates. When phage plaques became visible, the plates were cooled down for 30 min at 4°C and overlaid with 132 mm nitrocellulose membranes (Schleicher &amp; Schuell) for 5 min. Following plaque lift, the membranes were blocked in 1% BSA in TBS for 1 hour at RT and incubated overnight in 25 ml of TBS-T on a rocker at 4°C with 10 µg of the target PDZ domain that had been cleaved from the GST moiety, biotinylated and complexed with streptavidin – alkaline phosphatase (STRAP) at a ratio of 4:1. After extensive washing with TBS-T, positive plaques were developed on the membranes with insoluble alkaline phosphatase (AP) substrate BCIP/NBT (Sigma). Individual positive plaques were identified on the plates and phages from these plaques were propagated separately in the appropriate host for production of individual phage lysates. The identities of phage-displayed peptides were inferred by sequencing the library-specific DNA inserts amplified by PCR from the T7 phage display vector.</p>


<h4>Phage ELISA</h4>
<a id="article1.body1.sec4.sec3.p1" name="article1.body1.sec4.sec3.p1"></a><p>GST fusion-coated microtiter ELISA plates (COSTAR) were prepared by passive immobilization of 1 µg of the indicated GST fusion proteins (<a href="#pone-0000953-g001">Fig. 1</a>) per well in 200 µl of 0.1M NaHCO<sub>3</sub>, pH 8.0, overnight at 4°C. Following protein coating, plates were blocked by adding 150 µl of 1% BSA in TBS for 1 hour at RT. Following incubation, the wells with immobilized target proteins were washed 5 (x1ml) times with TBS-T. One hundred µl of freshly prepared individual phage lysate was added to the ELISA plate wells and incubated for 1 hour at RT. Following incubation, unbound phages were washed away with TBS-T and the amount of retained phages was determined with polyclonal T7 phage-specific antibodies followed by monoclonal anti-rabbit antibodies conjugated to horseradish peroxidase (HRP) (Amersham Pharmacia). The individual phage ELISA kinetics were followed and quantified colorimetrically using soluble HRP substrate (ABTS/H<sub>2</sub>O<sub>2</sub>). ELISA readings were taken on a SpectraMAX190 plate reader (Molecular Devices) at 405 nm. To ensure reproducibility, each of the individual phage ELISA binding histograms presented was obtained at least three times in at least three separate experiments performed by two different experimenters. The representative sets of binding histograms are shown in <a href="#pone-0000953-g001">Fig. 1</a>.</p>


<h4>Peptide ELISA</h4>
<a id="article1.body1.sec4.sec4.p1" name="article1.body1.sec4.sec4.p1"></a><p>Wells of microtiter plates were coated with 1 µg of the indicated GST-PDZ domain fusions, washed with TBS-T and blocked with 1% BSA in the same way as described above for phage ELISA. Individual biotinylated peptides (30 ng) were pre-incubated with 1 µg of streptavidin-HRP conjugate (Pierce) in 300 µl of TBS-T for 30 min at RT. One hundred µl of the peptide-streptavidin-HRP conjugate were added to 100 µl of TBS-T left in each coated well after the final wash of the protein immobilization/blocking procedure. Microtiter plates were incubated for 1 hour at RT, and then washed 5 (x1mL) times with TBS-T. The amounts of peptides retained were quantified colorimetrically by adding soluble HRP substrate (ABTS/H<sub>2</sub>O<sub>2</sub>) and measuring ELISA kinetic slopes. ELISA readings were taken on a SpectraMAX190 plate reader (Molecular Devices) at 405 nm. To ensure reproducibility, all peptide ELISA experiments presented were repeated at least three times in at least three separate experiments performed by two different experimenters. The representative sets of binding histograms are shown in <a href="#pone-0000953-g003">Fig. 3</a>.</p>


<h4>Statistical analysis</h4>
<a id="article1.body1.sec4.sec5.p1" name="article1.body1.sec4.sec5.p1"></a><p>Evaluation of statistical significance of amino acid frequency biases was based on the Bernoulli trials approximation, i.e. on the assumption of random and independent sampling of individual amino acids from a population with specified amino acid frequencies. In the case of artificial ligands obtained from cDNA library (about 50% of ligands) and from random peptide library (another 50%), we assumed that sampling was done from a population where individual amino acids are equally represented. We believe it is a reasonable, albeit coarse-grained, approximation both for random peptide library and for the cDNA library used, considering that about 90% of the peptides isolated from the latter represented frameshifts. It was assumed that natural ligands were sampled from a population where individual amino acid are distributed in accord with their average occurrence in the SWISS-PROT database (the v. 51.1 issue statistics). The chi-square and binomial tests <em>P</em>-values shown in the <a href="#pone-0000953-g002">Fig. 2</a> and <a href="#pone-0000953-g004">Fig. 4</a> were calculated and corrected for multiple testing using the GraphPad Online Calculators at <a href="http://www.graphpad.com">http://www.graphpad.com</a></p>

</div>

<div id="section5" class="section"><a id="s5" name="s5" toc="s5" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0000953"></div><a name="pone.0000953.s001" id="pone.0000953.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000953.s001">Figure S1A. </a></strong></p><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1"></a><p class="preSiDOI">Distribution of charged residues within affinity-ranked artificial ligands of the PSD95-PDZ2 and PSD95-PDZ3 domains. The aligned sequences of artificial peptide ligands are arranged in four groups based on their relative affinities to the indicated PDZ domains. The numbers in parentheses indicate the range of normalized phage ELISA values within a given affinity group. Arginines and lysines are highlighted green, while aspartic and glutamic acids are red. Upper panel - PSD95-PDZ2 ligands; lower panel - PSD95-PDZ3 ligands.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000953.s001</p><a id="article1.body1.sec5.supplementary-material1.caption1.p2" name="article1.body1.sec5.supplementary-material1.caption1.p2"></a><p class="postSiDOI">(0.86 MB TIF)</p>
<a name="pone.0000953.s002" id="pone.0000953.s002"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000953.s002">Figure S1B. </a></strong></p><a id="article1.body1.sec5.supplementary-material2.caption1.p1" name="article1.body1.sec5.supplementary-material2.caption1.p1"></a><p class="preSiDOI">Distribution of charged residues within affinity-ranked artificial ligands of the SAP97-PDZ1 and SAP97-PDZ2 domains. The aligned sequences of artificial peptide ligands are arranged in four groups based on their relative affinities to SAP PDZ domains. The numbers in parentheses indicate the range of normalized phage ELISA values within a given affinity group. Arginines and lysines are highlighted green, while aspartic and glutamic acids are red. Upper panel - SAP97-PDZ1 ligands; lower panel - SAP97-PDZ2 ligands.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000953.s002</p><a id="article1.body1.sec5.supplementary-material2.caption1.p2" name="article1.body1.sec5.supplementary-material2.caption1.p2"></a><p class="postSiDOI">(0.80 MB TIF)</p>
<a name="pone.0000953.s003" id="pone.0000953.s003"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000953.s003">Figure S1C. </a></strong></p><a id="article1.body1.sec5.supplementary-material3.caption1.p1" name="article1.body1.sec5.supplementary-material3.caption1.p1"></a><p class="preSiDOI">Distribution of charged residues within affinity-ranked artificial ligands of the SAP97-PDZ3 domain. The aligned sequences of artificial peptide ligands are arranged in four groups based on their relative affinities to SAP PDZ domains. The numbers in parentheses indicate the range of normalized phage ELISA values within a given affinity group. Arginines and lysines are highlighted green, while aspartic and glutamic acids are red.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000953.s003</p><a id="article1.body1.sec5.supplementary-material3.caption1.p2" name="article1.body1.sec5.supplementary-material3.caption1.p2"></a><p class="postSiDOI">(0.41 MB TIF)</p>
<a name="pone.0000953.s004" id="pone.0000953.s004"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000953.s004">Table S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material4.caption1.p1" name="article1.body1.sec5.supplementary-material4.caption1.p1"></a><p class="preSiDOI">Artificial peptide ligands isolated from phage-displayed random peptide and cDNA libraries by TAIS using various SAP PDZ domains as targets</p>
<p class="siDoi">doi:10.1371/journal.pone.0000953.s004</p><a id="article1.body1.sec5.supplementary-material4.caption1.p2" name="article1.body1.sec5.supplementary-material4.caption1.p2"></a><p class="postSiDOI">(0.05 MB DOC)</p>
<a name="pone.0000953.s005" id="pone.0000953.s005"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000953.s005">Table S2. </a></strong></p><a id="article1.body1.sec5.supplementary-material5.caption1.p1" name="article1.body1.sec5.supplementary-material5.caption1.p1"></a><p class="preSiDOI">Putative natural peptide ligands of SAP PDZ domains</p>
<p class="siDoi">doi:10.1371/journal.pone.0000953.s005</p><a id="article1.body1.sec5.supplementary-material5.caption1.p2" name="article1.body1.sec5.supplementary-material5.caption1.p2"></a><p class="postSiDOI">(0.06 MB DOC)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>We thank Dr. B. K. Kay (The University of Illinois at Chicago), Dr. F. W. Studier (Brookhaven National Laboratory) and Dr. David S. Bredt (University of California, San Francisco) for providing research reagents. We express our gratitude to Dr. D. A. Greenberg (Buck Institute) for critical reading of the manuscript. We thank Afanasy Kurakin (Moscow State University) for software tools development and Dr. M.-Y. Chou (Industrial Technology Research Institute, Taiwan) for technical assistance. We thank Dr. M. Zhang (Hong Kong University of Science and Technology) and anonymous reviewer for their comments and suggestions.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: AK. Performed the experiments: AK SW AS. Analyzed the data: AK SW AS. Contributed reagents/materials/analysis tools: DB AK. Wrote the paper: AK. Other: Provided funding, edited manuscript: DB.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0000953-Kuriyan1" id="pone.0000953-Kuriyan1"></a>Kuriyan J, Cowburn D (1997) Modular peptide recognition domains in eukaryotic signaling. Annu Rev Biophys Biomol Struct  26: 259–288.  <ul class="find" data-citedArticleID="997420" data-doi="10.1146/annurev.biophys.26.1.259"><li><a href="http://dx.doi.org/10.1146/annurev.biophys.26.1.259" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Modular+peptide+recognition+domains+in+eukaryotic+signaling." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Modular+peptide+recognition+domains+in+eukaryotic+signaling.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0000953-Sudol1" id="pone.0000953-Sudol1"></a>Sudol M (1998) From Src Homology domains to other signaling modules: proposal of the ‘protein recognition code’. Oncogene  17: 1469–1474.  <ul class="find" data-citedArticleID="997444" data-doi="10.1038/sj.onc.1202182"><li><a href="http://dx.doi.org/10.1038/sj.onc.1202182" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=From+Src+Homology+domains+to+other+signaling+modules%3A+proposal+of+the+%E2%80%98protein+recognition+code%E2%80%99." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22From+Src+Homology+domains+to+other+signaling+modules%3A+proposal+of+the+%E2%80%98protein+recognition+code%E2%80%99.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0000953-Castagnoli1" id="pone.0000953-Castagnoli1"></a>Castagnoli L, Costantini A, Dall'Armi C, Gonfloni S, Montecchi-Palazzi L, et al.  (2004) Selectivity and promiscuity in the interaction network mediated by protein recognition modules. FEBS Lett  567: 74–79.  <ul class="find" data-citedArticleID="997380" data-doi="10.1016/j.febslet.2004.03.116"><li><a href="http://dx.doi.org/10.1016/j.febslet.2004.03.116" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Selectivity+and+promiscuity+in+the+interaction+network+mediated+by+protein+recognition+modules." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Selectivity+and+promiscuity+in+the+interaction+network+mediated+by+protein+recognition+modules.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0000953-Kay1" id="pone.0000953-Kay1"></a>Kay BK, Kasanov J, Knight S, Kurakin A (2000) Convergent evolution with combinatorial peptides. FEBS Lett  480: 55–62.  <ul class="find" data-citedArticleID="997398" data-doi="10.1016/s0014-5793(00)01778-6"><li><a href="http://dx.doi.org/10.1016/s0014-5793(00)01778-6" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Convergent+evolution+with+combinatorial+peptides." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Convergent+evolution+with+combinatorial+peptides.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0000953-Zucconi1" id="pone.0000953-Zucconi1"></a>Zucconi A, Panni S, Paoluzi S, Castagnoli L, Dente L, et al.  (2000) Domain repertoires as a tool to derive protein recognition rules. FEBS Lett  480: 49–54.  <ul class="find" data-citedArticleID="997452" data-doi="10.1016/s0014-5793(00)01777-4"><li><a href="http://dx.doi.org/10.1016/s0014-5793(00)01777-4" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Domain+repertoires+as+a+tool+to+derive+protein+recognition+rules." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Domain+repertoires+as+a+tool+to+derive+protein+recognition+rules.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0000953-Cho1" id="pone.0000953-Cho1"></a>Cho KO, Hunt CA, Kennedy MB (1992) The rat brain postsynaptic density fraction contains a homolog of the Drosophila discs-large tumor suppressor protein. Neuron  9: 929–942.  <ul class="find" data-citedArticleID="997382" data-doi="10.1016/0896-6273(92)90245-9"><li><a href="http://dx.doi.org/10.1016/0896-6273(92)90245-9" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+rat+brain+postsynaptic+density+fraction+contains+a+homolog+of+the+Drosophila+discs-large+tumor+suppressor+protein." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+rat+brain+postsynaptic+density+fraction+contains+a+homolog+of+the+Drosophila+discs-large+tumor+suppressor+protein.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0000953-Lander1" id="pone.0000953-Lander1"></a>Lander ES, Linton LM, Birren B, Nusbaum C, Zody MC, et al.  (2001) Initial sequencing and analysis of the human genome. Nature  409: 860–921.  <ul class="find" data-citedArticleID="997424"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Initial+sequencing+and+analysis+of+the+human+genome.&amp;auth=&amp;atitle=Initial+sequencing+and+analysis+of+the+human+genome." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Initial+sequencing+and+analysis+of+the+human+genome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Initial+sequencing+and+analysis+of+the+human+genome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0000953-Harris1" id="pone.0000953-Harris1"></a>Harris BZ, Lim WA (2001) Mechanism and role of PDZ domains in signaling complex assembly. J Cell Sci  114: 3219–3231.  <ul class="find" data-citedArticleID="997392"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Mechanism+and+role+of+PDZ+domains+in+signaling+complex+assembly.&amp;auth=&amp;atitle=Mechanism+and+role+of+PDZ+domains+in+signaling+complex+assembly." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mechanism+and+role+of+PDZ+domains+in+signaling+complex+assembly." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mechanism+and+role+of+PDZ+domains+in+signaling+complex+assembly.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0000953-Hung1" id="pone.0000953-Hung1"></a>Hung AY, Sheng M (2002) PDZ domains: structural modules for protein complex assembly. J Biol Chem  277: 5699–5702.  <ul class="find" data-citedArticleID="997394" data-doi="10.1074/jbc.r100065200"><li><a href="http://dx.doi.org/10.1074/jbc.r100065200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=PDZ+domains%3A+structural+modules+for+protein+complex+assembly." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22PDZ+domains%3A+structural+modules+for+protein+complex+assembly.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0000953-Kennedy1" id="pone.0000953-Kennedy1"></a>Kennedy MB (2000) Signal-processing machines at the postsynaptic density. Science  290: 750–754.  <ul class="find" data-citedArticleID="997400" data-doi="10.1126/science.290.5492.750"><li><a href="http://dx.doi.org/10.1126/science.290.5492.750" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Signal-processing+machines+at+the+postsynaptic+density." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Signal-processing+machines+at+the+postsynaptic+density.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0000953-Kim1" id="pone.0000953-Kim1"></a>Kim E, Sheng M (2004) PDZ domain proteins of synapses. Nat Rev Neurosci  5: 771–781.  <ul class="find" data-citedArticleID="997402" data-doi="10.1038/nrn1517"><li><a href="http://dx.doi.org/10.1038/nrn1517" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=PDZ+domain+proteins+of+synapses." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22PDZ+domain+proteins+of+synapses.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0000953-Fujita1" id="pone.0000953-Fujita1"></a>Fujita A, Kurachi Y (2000) SAP family proteins. Biochem Biophys Res Commun  269: 1–6.  <ul class="find" data-citedArticleID="997388" data-doi="10.1006/bbrc.1999.1893"><li><a href="http://dx.doi.org/10.1006/bbrc.1999.1893" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=SAP+family+proteins." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22SAP+family+proteins.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0000953-Bezprozvanny1" id="pone.0000953-Bezprozvanny1"></a>Bezprozvanny I, Maximov A (2001) Classification of PDZ domains. FEBS Lett  509: 457–462.  <ul class="find" data-citedArticleID="997376" data-doi="10.1016/s0014-5793(01)03214-8"><li><a href="http://dx.doi.org/10.1016/s0014-5793(01)03214-8" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Classification+of+PDZ+domains." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Classification+of+PDZ+domains.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0000953-Vaccaro1" id="pone.0000953-Vaccaro1"></a>Vaccaro P, Dente L (2002) PDZ domains: troubles in classification. FEBS Lett  512: 345–349.  <ul class="find" data-citedArticleID="997448" data-doi="10.1016/s0014-5793(02)02220-2"><li><a href="http://dx.doi.org/10.1016/s0014-5793(02)02220-2" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=PDZ+domains%3A+troubles+in+classification." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22PDZ+domains%3A+troubles+in+classification.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0000953-Dev1" id="pone.0000953-Dev1"></a>Dev KK (2004) Making protein interactions druggable: targeting PDZ domains. Nat Rev Drug Discov  3: 1047–1056.  <ul class="find" data-citedArticleID="997384" data-doi="10.1038/nrd1578"><li><a href="http://dx.doi.org/10.1038/nrd1578" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Making+protein+interactions+druggable%3A+targeting+PDZ+domains." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Making+protein+interactions+druggable%3A+targeting+PDZ+domains.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0000953-Songyang1" id="pone.0000953-Songyang1"></a>Songyang Z, Fanning AS, Fu C, Xu J, Marfatia SM, et al.  (1997) Recognition of unique carboxyl-terminal motifs by distinct PDZ domains. Science  275: 73–77.  <ul class="find" data-citedArticleID="997438" data-doi="10.1126/science.275.5296.73"><li><a href="http://dx.doi.org/10.1126/science.275.5296.73" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Recognition+of+unique+carboxyl-terminal+motifs+by+distinct+PDZ+domains." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Recognition+of+unique+carboxyl-terminal+motifs+by+distinct+PDZ+domains.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0000953-Niethammer1" id="pone.0000953-Niethammer1"></a>Niethammer M, Valtschanoff JG, Kapoor TM, Allison DW, Weinberg RJ, et al.  (1998) CRIPT, a novel postsynaptic protein that binds to the third PDZ domain of PSD-95/SAP90. Neuron  20: 693–707.  <ul class="find" data-citedArticleID="997432" data-doi="10.1016/s0896-6273(00)81009-0"><li><a href="http://dx.doi.org/10.1016/s0896-6273(00)81009-0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=CRIPT%2C+a+novel+postsynaptic+protein+that+binds+to+the+third+PDZ+domain+of+PSD-95%2FSAP90." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22CRIPT%2C+a+novel+postsynaptic+protein+that+binds+to+the+third+PDZ+domain+of+PSD-95%2FSAP90.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0000953-Lim1" id="pone.0000953-Lim1"></a>Lim IA, Hall DD, Hell JW (2002) Selectivity and promiscuity of the first and second PDZ domains of PSD-95 and synapse-associated protein 102. J Biol Chem  277: 21697–21711.  <ul class="find" data-citedArticleID="997426" data-doi="10.1074/jbc.m112339200"><li><a href="http://dx.doi.org/10.1074/jbc.m112339200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Selectivity+and+promiscuity+of+the+first+and+second+PDZ+domains+of+PSD-95+and+synapse-associated+protein+102." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Selectivity+and+promiscuity+of+the+first+and+second+PDZ+domains+of+PSD-95+and+synapse-associated+protein+102.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0000953-Basdevant1" id="pone.0000953-Basdevant1"></a>Basdevant N, Weinstein H, Ceruso M (2006) Thermodynamic basis for promiscuity and selectivity in protein-protein interactions: PDZ domains, a case study. J Am Chem Soc  128: 12766–12777.  <ul class="find" data-citedArticleID="997374" data-doi="10.1021/ja060830y"><li><a href="http://dx.doi.org/10.1021/ja060830y" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Thermodynamic+basis+for+promiscuity+and+selectivity+in+protein-protein+interactions%3A+PDZ+domains%2C+a+case+study." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Thermodynamic+basis+for+promiscuity+and+selectivity+in+protein-protein+interactions%3A+PDZ+domains%2C+a+case+study.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0000953-Doyle1" id="pone.0000953-Doyle1"></a>Doyle DA, Lee A, Lewis J, Kim E, Sheng M, et al.  (1996) Crystal structures of a complexed and peptide-free membrane protein-binding domain: molecular basis of peptide recognition by PDZ. Cell  85: 1067–1076.  <ul class="find" data-citedArticleID="997386" data-doi="10.1016/s0092-8674(00)81307-0"><li><a href="http://dx.doi.org/10.1016/s0092-8674(00)81307-0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Crystal+structures+of+a+complexed+and+peptide-free+membrane+protein-binding+domain%3A+molecular+basis+of+peptide+recognition+by+PDZ." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Crystal+structures+of+a+complexed+and+peptide-free+membrane+protein-binding+domain%3A+molecular+basis+of+peptide+recognition+by+PDZ.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0000953-Wiedemann1" id="pone.0000953-Wiedemann1"></a>Wiedemann U, Boisguerin P, Leben R, Leitner D, Krause G, et al.  (2004) Quantification of PDZ domain specificity, prediction of ligand affinity and rational design of super-binding peptides. J Mol Biol  343: 703–718.  <ul class="find" data-citedArticleID="997450" data-doi="10.1016/j.jmb.2004.08.064"><li><a href="http://dx.doi.org/10.1016/j.jmb.2004.08.064" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Quantification+of+PDZ+domain+specificity%2C+prediction+of+ligand+affinity+and+rational+design+of+super-binding+peptides." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Quantification+of+PDZ+domain+specificity%2C+prediction+of+ligand+affinity+and+rational+design+of+super-binding+peptides.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0000953-Birrane1" id="pone.0000953-Birrane1"></a>Birrane G, Chung J, Ladias JA (2003) Novel mode of ligand recognition by the Erbin PDZ domain. J Biol Chem  278: 1399–1402.  <ul class="find" data-citedArticleID="997378" data-doi="10.1074/jbc.c200571200"><li><a href="http://dx.doi.org/10.1074/jbc.c200571200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Novel+mode+of+ligand+recognition+by+the+Erbin+PDZ+domain." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Novel+mode+of+ligand+recognition+by+the+Erbin+PDZ+domain.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0000953-Appleton1" id="pone.0000953-Appleton1"></a>Appleton BA, Zhang Y, Wu P, Yin JP, Hunziker W, et al.  (2006) Comparative structural analysis of the Erbin PDZ domain and the first PDZ domain of ZO-1. Insights into determinants of PDZ domain specificity. J Biol Chem  281: 22312–22320.  <ul class="find" data-citedArticleID="997372" data-doi="10.1074/jbc.m602901200"><li><a href="http://dx.doi.org/10.1074/jbc.m602901200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Comparative+structural+analysis+of+the+Erbin+PDZ+domain+and+the+first+PDZ+domain+of+ZO-1.+Insights+into+determinants+of+PDZ+domain+specificity." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Comparative+structural+analysis+of+the+Erbin+PDZ+domain+and+the+first+PDZ+domain+of+ZO-1.+Insights+into+determinants+of+PDZ+domain+specificity.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0000953-Ladbury1" id="pone.0000953-Ladbury1"></a>Ladbury JE, Arold S (2000) Searching for specificity in SH domains. Chem Biol  7: R3–8.  <ul class="find" data-citedArticleID="997422" data-doi="10.1016/s1074-5521(00)00067-3"><li><a href="http://dx.doi.org/10.1016/s1074-5521(00)00067-3" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Searching+for+specificity+in+SH+domains." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Searching+for+specificity+in+SH+domains.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0000953-Kurakin1" id="pone.0000953-Kurakin1"></a>Kurakin A, Bredesen D (2002) Target-Assisted Iterative Screening reveals novel interactors for PSD95, Nedd4, Src, Abl and Crk proteins. J Biomol Struct Dyn  19: 1–15.  <ul class="find" data-citedArticleID="997408" data-doi="10.1080/07391102.2002.10506805"><li><a href="http://dx.doi.org/10.1080/07391102.2002.10506805" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Target-Assisted+Iterative+Screening+reveals+novel+interactors+for+PSD95%2C+Nedd4%2C+Src%2C+Abl+and+Crk+proteins." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Target-Assisted+Iterative+Screening+reveals+novel+interactors+for+PSD95%2C+Nedd4%2C+Src%2C+Abl+and+Crk+proteins.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0000953-Kurakin2" id="pone.0000953-Kurakin2"></a>Kurakin A, Wu S, Bredesen DE (2004) Target-assisted iterative screening of phage surface display cDNA libraries. Methods Mol Biol  264: 47–60.  <ul class="find" data-citedArticleID="997410" data-doi="10.1385/1-59259-759-9:047"><li><a href="http://dx.doi.org/10.1385/1-59259-759-9:047" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Target-assisted+iterative+screening+of+phage+surface+display+cDNA+libraries." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Target-assisted+iterative+screening+of+phage+surface+display+cDNA+libraries.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0000953-Smith1" id="pone.0000953-Smith1"></a>Smith GP (1985) Filamentous fusion phage: novel expression vectors that display cloned antigens on the virion surface. Science  228: 1315–1317.  <ul class="find" data-citedArticleID="997436" data-doi="10.1126/science.4001944"><li><a href="http://dx.doi.org/10.1126/science.4001944" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Filamentous+fusion+phage%3A+novel+expression+vectors+that+display+cloned+antigens+on+the+virion+surface." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Filamentous+fusion+phage%3A+novel+expression+vectors+that+display+cloned+antigens+on+the+virion+surface.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0000953-Sparks1" id="pone.0000953-Sparks1"></a>Sparks AB, Rider JE, Hoffman NG, Fowlkes DM, Quillam LA, et al.  (1996) Distinct ligand preferences of Src homology 3 domains from Src, Yes, Abl, Cortactin, p53bp2, PLCgamma, Crk, and Grb2. Proc Natl Acad Sci U S A  93: 1540–1544.  <ul class="find" data-citedArticleID="997440" data-doi="10.1073/pnas.93.4.1540"><li><a href="http://dx.doi.org/10.1073/pnas.93.4.1540" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Distinct+ligand+preferences+of+Src+homology+3+domains+from+Src%2C+Yes%2C+Abl%2C+Cortactin%2C+p53bp2%2C+PLCgamma%2C+Crk%2C+and+Grb2." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Distinct+ligand+preferences+of+Src+homology+3+domains+from+Src%2C+Yes%2C+Abl%2C+Cortactin%2C+p53bp2%2C+PLCgamma%2C+Crk%2C+and+Grb2.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0000953-Kornau1" id="pone.0000953-Kornau1"></a>Kornau HC, Schenker LT, Kennedy MB, Seeburg PH (1995) Domain interaction between NMDA receptor subunits and the postsynaptic density protein PSD-95. Science  269: 1737–1740.  <ul class="find" data-citedArticleID="997406" data-doi="10.1126/science.7569905"><li><a href="http://dx.doi.org/10.1126/science.7569905" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Domain+interaction+between+NMDA+receptor+subunits+and+the+postsynaptic+density+protein+PSD-95." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Domain+interaction+between+NMDA+receptor+subunits+and+the+postsynaptic+density+protein+PSD-95.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0000953-Kim2" id="pone.0000953-Kim2"></a>Kim E, Niethammer M, Rothschild A, Jan YN, Sheng M (1995) Clustering of Shaker-type K+ channels by interaction with a family of membrane-associated guanylate kinases. Nature  378: 85–88.  <ul class="find" data-citedArticleID="997404" data-doi="10.1038/378085a0"><li><a href="http://dx.doi.org/10.1038/378085a0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Clustering+of+Shaker-type+K%2B+channels+by+interaction+with+a+family+of+membrane-associated+guanylate+kinases." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Clustering+of+Shaker-type+K%2B+channels+by+interaction+with+a+family+of+membrane-associated+guanylate+kinases.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0000953-Kurakin3" id="pone.0000953-Kurakin3"></a>Kurakin A, Bredesen DE (2006) An unconventional IAP-binding motif revealed by target-assisted iterative screening (TAIS) of the BIR3-cIAP1 domain. J Mol Recognit. 20(1): 39–50.  <ul class="find" data-citedArticleID="997412" data-doi="10.1002/jmr.809"><li><a href="http://dx.doi.org/10.1002/jmr.809" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=An+unconventional+IAP-binding+motif+revealed+by+target-assisted+iterative+screening+%28TAIS%29+of+the+BIR3-cIAP1+domain.+J+Mol+Recognit." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22An+unconventional+IAP-binding+motif+revealed+by+target-assisted+iterative+screening+%28TAIS%29+of+the+BIR3-cIAP1+domain.+J+Mol+Recognit.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0000953-Piserchio1" id="pone.0000953-Piserchio1"></a>Piserchio A, Pellegrini M, Mehta S, Blackman SM, Garcia EP, et al.  (2002) The PDZ1 domain of SAP90. Characterization of structure and binding. J Biol Chem  277: 6967–6973.  <ul class="find" data-citedArticleID="997434" data-doi="10.1074/jbc.m109453200"><li><a href="http://dx.doi.org/10.1074/jbc.m109453200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+PDZ1+domain+of+SAP90.+Characterization+of+structure+and+binding." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+PDZ1+domain+of+SAP90.+Characterization+of+structure+and+binding.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0000953-Inoue1" id="pone.0000953-Inoue1"></a>Inoue A, Okabe S (2003) The dynamic organization of postsynaptic proteins: translocating molecules regulate synaptic function. Curr Opin Neurobiol  13: 332–340.  <ul class="find" data-citedArticleID="997396" data-doi="10.1016/s0959-4388(03)00077-1"><li><a href="http://dx.doi.org/10.1016/s0959-4388(03)00077-1" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+dynamic+organization+of+postsynaptic+proteins%3A+translocating+molecules+regulate+synaptic+function." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+dynamic+organization+of+postsynaptic+proteins%3A+translocating+molecules+regulate+synaptic+function.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0000953-Gray1" id="pone.0000953-Gray1"></a>Gray NW, Weimer RM, Bureau I, Svoboda K (2006) Rapid redistribution of synaptic PSD-95 in the neocortex in vivo. PLoS Biol  4: e370.  <ul class="find" data-citedArticleID="997390" data-doi="10.1371/journal.pbio.0040370"><li><a href="http://dx.doi.org/10.1371/journal.pbio.0040370" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Rapid+redistribution+of+synaptic+PSD-95+in+the+neocortex+in+vivo." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Rapid+redistribution+of+synaptic+PSD-95+in+the+neocortex+in+vivo.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0000953-Misteli1" id="pone.0000953-Misteli1"></a>Misteli T (2001) The concept of self-organization in cellular architecture. J Cell Biol  155: 181–185.  <ul class="find" data-citedArticleID="997430" data-doi="10.1083/jcb.200108110"><li><a href="http://dx.doi.org/10.1083/jcb.200108110" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+concept+of+self-organization+in+cellular+architecture." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+concept+of+self-organization+in+cellular+architecture.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">36.
              </span><a name="pone.0000953-Kurakin4" id="pone.0000953-Kurakin4"></a>Kurakin A (2005) Self-organization versus Watchmaker: stochastic dynamics of cellular organization. Biol Chem  386: 247–254.  <ul class="find" data-citedArticleID="997414" data-doi="10.1515/bc.2005.030"><li><a href="http://dx.doi.org/10.1515/bc.2005.030" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Self-organization+versus+Watchmaker%3A+stochastic+dynamics+of+cellular+organization." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Self-organization+versus+Watchmaker%3A+stochastic+dynamics+of+cellular+organization.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">37.
              </span><a name="pone.0000953-Spector1" id="pone.0000953-Spector1"></a>Spector DL (2003) The dynamics of chromosome organization and gene regulation. Annu Rev Biochem  72: 573–608.  <ul class="find" data-citedArticleID="997442"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=The+dynamics+of+chromosome+organization+and+gene+regulation.&amp;auth=&amp;atitle=The+dynamics+of+chromosome+organization+and+gene+regulation." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+dynamics+of+chromosome+organization+and+gene+regulation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+dynamics+of+chromosome+organization+and+gene+regulation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">38.
              </span><a name="pone.0000953-Mayer1" id="pone.0000953-Mayer1"></a>Mayer BJ (2001) SH3 domains: complexity in moderation. J Cell Sci  114: 1253–1263.  <ul class="find" data-citedArticleID="997428"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=SH3+domains%3A+complexity+in+moderation.&amp;auth=&amp;atitle=SH3+domains%3A+complexity+in+moderation." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=SH3+domains%3A+complexity+in+moderation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22SH3+domains%3A+complexity+in+moderation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">39.
              </span><a name="pone.0000953-Kurakin5" id="pone.0000953-Kurakin5"></a>Kurakin A (2007) Self-Organization versus Watchmaker: ambiguity of molecular recognition and design charts of cellular circuitry. J Mol Recognit  20: In press. DOI: 10.1002/jmr.839.  <ul class="find" data-citedArticleID="997416" data-doi="10.1002/jmr.839"><li><a href="http://dx.doi.org/10.1002/jmr.839" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Self-Organization+versus+Watchmaker%3A+ambiguity+of+molecular+recognition+and+design+charts+of+cellular+circuitry." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Self-Organization+versus+Watchmaker%3A+ambiguity+of+molecular+recognition+and+design+charts+of+cellular+circuitry.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">40.
              </span><a name="pone.0000953-Kurakin6" id="pone.0000953-Kurakin6"></a>Kurakin AV, Wu S, Bredesen DE (2003) Atypical recognition consensus of CIN85/SETA/Ruk SH3 domains revealed by target-assisted iterative screening. J Biol Chem  278: 34102–34109.  <ul class="find" data-citedArticleID="997418" data-doi="10.1074/jbc.m305264200"><li><a href="http://dx.doi.org/10.1074/jbc.m305264200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Atypical+recognition+consensus+of+CIN85%2FSETA%2FRuk+SH3+domains+revealed+by+target-assisted+iterative+screening." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Atypical+recognition+consensus+of+CIN85%2FSETA%2FRuk+SH3+domains+revealed+by+target-assisted+iterative+screening.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">41.
              </span><a name="pone.0000953-Taylor1" id="pone.0000953-Taylor1"></a>Taylor WR (1986) The classification of amino acid conservation. J Theor Biol  119: 205–218.  <ul class="find" data-citedArticleID="997446" data-doi="10.1016/s0022-5193(86)80075-3"><li><a href="http://dx.doi.org/10.1016/s0022-5193(86)80075-3" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+classification+of+amino+acid+conservation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+classification+of+amino+acid+conservation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000953.XML" value="100859"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000953.PDF" value="692631"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000953.g001.PNG_L" value="749420"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000953.g001.PNG_M" value="62375"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000953.g001.PNG_S" value="12681"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000953.g001.TIF" value="2138030"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000953.g001.PNG_I" value="26251"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000953.g002.PNG_L" value="2667928"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000953.g002.PNG_M" value="294434"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000953.g002.PNG_S" value="19246"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000953.g002.TIF" value="4053774"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000953.g002.PNG_I" value="101731"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000953.g003.PNG_L" value="447752"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000953.g003.PNG_M" value="59920"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000953.g003.PNG_S" value="10703"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000953.g003.TIF" value="1010782"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000953.g003.PNG_I" value="21077"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000953.g004.PNG_L" value="1635703"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000953.g004.PNG_M" value="188817"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000953.g004.PNG_S" value="18149"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000953.g004.TIF" value="2857422"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000953.g004.PNG_I" value="97078"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000953.g005.PNG_L" value="670867"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000953.g005.PNG_M" value="264042"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000953.g005.PNG_S" value="18878"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000953.g005.TIF" value="993254"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000953.g005.PNG_I" value="106121"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000953.s001.TIF" value="1290204"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000953.s002.TIF" value="1229270"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000953.s003.TIF" value="602052"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000953.s004.DOC" value="48640"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000953.s005.DOC" value="57856"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000953&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000953" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000953&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0000953&volume=&issue=&title=The PDZ Domain as a Complex Adaptive System&author_name=Alexei%20Kurakin%2C%20Andrzej%20Swistowski%2C%20Susan%20C.%20Wu%2C%20Dale%20E.%20Bredesen&start_page=1&end_page=11" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000953" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000953" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0000953" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000953&amp;t=The%20PDZ%20Domain%20as%20a%20Complex%20Adaptive%20System" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000953&title=The%20PDZ%20Domain%20as%20a%20Complex%20Adaptive%20System&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000953&amp;title=The%20PDZ%20Domain%20as%20a%20Complex%20Adaptive%20System" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000953" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0000953&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'The PDZ Domain as a Complex Adaptive System';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0000953';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20The%20PDZ%20Domain%20as%20a%20Complex%20Adaptive%20System http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000953" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0000953" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">













          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Amino+acid+analysis%22" title="Search for articles in the subject area:'Amino acid analysis'"><div class="flagText">Amino acid analysis</div></a>
              <div data-categoryid="47251" data-articleid="24742"
                   data-categoryname="Amino acid analysis"
                   class="flagImage" title="Flag 'Amino acid analysis' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Arginine%22" title="Search for articles in the subject area:'Arginine'"><div class="flagText">Arginine</div></a>
              <div data-categoryid="40219" data-articleid="24742"
                   data-categoryname="Arginine"
                   class="flagImage" title="Flag 'Arginine' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Bacteriophages%22" title="Search for articles in the subject area:'Bacteriophages'"><div class="flagText">Bacteriophages</div></a>
              <div data-categoryid="43083" data-articleid="24742"
                   data-categoryname="Bacteriophages"
                   class="flagImage" title="Flag 'Bacteriophages' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Binders%22" title="Search for articles in the subject area:'Binders'"><div class="flagText">Binders</div></a>
              <div data-categoryid="22201" data-articleid="24742"
                   data-categoryname="Binders"
                   class="flagImage" title="Flag 'Binders' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Genome+evolution%22" title="Search for articles in the subject area:'Genome evolution'"><div class="flagText">Genome evolution</div></a>
              <div data-categoryid="17207" data-articleid="24742"
                   data-categoryname="Genome evolution"
                   class="flagImage" title="Flag 'Genome evolution' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Peptide+libraries%22" title="Search for articles in the subject area:'Peptide libraries'"><div class="flagText">Peptide libraries</div></a>
              <div data-categoryid="20137" data-articleid="24742"
                   data-categoryname="Peptide libraries"
                   class="flagImage" title="Flag 'Peptide libraries' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Peptides%22" title="Search for articles in the subject area:'Peptides'"><div class="flagText">Peptides</div></a>
              <div data-categoryid="17847" data-articleid="24742"
                   data-categoryname="Peptides"
                   class="flagImage" title="Flag 'Peptides' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Protein+interactions%22" title="Search for articles in the subject area:'Protein interactions'"><div class="flagText">Protein interactions</div></a>
              <div data-categoryid="17661" data-articleid="24742"
                   data-categoryname="Protein interactions"
                   class="flagImage" title="Flag 'Protein interactions' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=1066'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=6336'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=7582&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>

<div class="block sidebar-comments">
    <div class="header">
        <h3>Comments</h3>
    </div>
      <p><a href="/annotation/listThread.action?root=13907">Webcast at SciVee</a><br>Posted by akurakin</p>
      <p><a href="/annotation/listThread.action?root=12193">Pubcast at SciVee</a><br>Posted by akurakin</p>
</div>

</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
