

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Optimization of Ribosome Structure and Function by rRNA Base Modification</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0000174"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0000174" />

    <meta name="citation_title" content="Optimization of Ribosome Structure and Function by rRNA Base Modification"/>
    <meta itemprop="name" content="Optimization of Ribosome Structure and Function by rRNA Base Modification"/>

      <meta name="citation_author" content="Jennifer L. Baxter-Roshek"/>
            <meta name="citation_author_institution" content="Department of Cell Biology and Molecular Genetics, University of Maryland, College Park, Maryland, United States of America"/>
      <meta name="citation_author" content="Alexey N. Petrov"/>
            <meta name="citation_author_institution" content="Department of Cell Biology and Molecular Genetics, University of Maryland, College Park, Maryland, United States of America"/>
      <meta name="citation_author" content="Jonathan D. Dinman"/>
            <meta name="citation_author_institution" content="Department of Cell Biology and Molecular Genetics, University of Maryland, College Park, Maryland, United States of America"/>

    <meta name="citation_date" content="2007/1/24"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0000174.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e174"/>
    <meta name="citation_issue" content="1"/>
    <meta name="citation_volume" content="2"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=The complete atomic structure of the large ribosomal subunit at 2.4 A resolution.; citation_author=N Ban; citation_author=P Nissen; citation_author=J Hansen; citation_author=PB Moore; citation_author=TA Steitz; citation_journal_title=Science; citation_volume=289; citation_number=1; citation_pages=905-920; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Crystal Structure of the Ribosome at 5.5 A Resolution.; citation_author=MM Yusupov; citation_author=GZ Yusupova; citation_author=A Baucom; citation_author=K Lieberman; citation_author=TN Earnest; citation_journal_title=Science; citation_volume=292; citation_number=2; citation_pages=883-896; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=A comprehensive database for the small nucleolar RNAs from Saccharomyces cerevisiae.; citation_author=DA Samarsky; citation_author=MJ Fournier; citation_journal_title=Nucleic Acids Res; citation_volume=27; citation_number=3; citation_pages=161-164; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=rRNA modifications and ribosome function.; citation_author=WA Decatur; citation_author=MJ Fournier; citation_journal_title=Trends Biochem Sci; citation_volume=27; citation_number=4; citation_pages=344-351; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=The pseudouridine residues of ribosomal RNA.; citation_author=J Ofengand; citation_author=A Bakin; citation_author=J Wrzesinski; citation_author=K Nurse; citation_author=BG Lane; citation_journal_title=Biochem Cell Biol; citation_volume=73; citation_number=5; citation_pages=915-924; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=In vitro complementation analysis localizes 23S rRNA posttranscriptional modifications that are required for Escherichia coli 50S ribosomal subunit assembly and function.; citation_author=R Green; citation_author=HF Noller; citation_journal_title=RNA; citation_volume=2; citation_number=6; citation_pages=1011-1021; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Temperature-sensitive mutations demonstrate roles for yeast fibrillarin in pre-rRNA processing, pre-rRNA methylation, and ribosome assembly.; citation_author=D Tollervey; citation_author=H Lehtonen; citation_author=R Jansen; citation_author=H Kern; citation_author=EC Hurt; citation_journal_title=Cell; citation_volume=72; citation_number=7; citation_pages=443-457; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=Point mutations in yeast CBF5 can abolish in vivo pseudouridylation of rRNA.; citation_author=Y Zebarjadian; citation_author=T King; citation_author=MJ Fournier; citation_author=L Clarke; citation_author=J Carbon; citation_journal_title=Mol Cell Biol; citation_volume=19; citation_number=8; citation_pages=7461-7472; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=A computational screen for methylation guide snoRNAs in yeast.; citation_author=TM Lowe; citation_author=SR Eddy; citation_journal_title=Science; citation_volume=283; citation_number=9; citation_pages=1168-1171; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=A pseudouridine synthase required for the formation of two universally conserved pseudouridines in ribosomal RNA is essential for normal growth of Escherichia coli.; citation_author=S Raychaudhuri; citation_author=J Conrad; citation_author=BG Hall; citation_author=J Ofengand; citation_journal_title=RNA; citation_volume=4; citation_number=10; citation_pages=1407-1417; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Functional effect of deletion and mutation of the Escherichia coli ribosomal RNA and tRNA pseudouridine synthase RluA.; citation_author=S Raychaudhuri; citation_author=L Niu; citation_author=J Conrad; citation_author=BG Lane; citation_author=J Ofengand; citation_journal_title=J Biol Chem; citation_volume=274; citation_number=11; citation_pages=18880-18886; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Pseudouridine in RNA: what, where, how, and why.; citation_author=M Charette; citation_author=MW Gray; citation_journal_title=IUBMB Life; citation_volume=49; citation_number=12; citation_pages=341-351; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Post-transcriptional nucleotide modification and alternative folding of RNA.; citation_author=M Helm; citation_journal_title=Nucleic Acids Res; citation_volume=34; citation_number=13; citation_pages=721-733; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Thermodynamics of 2′-ribose substitutions in UUCG tetraloops.; citation_author=DJ Williams; citation_author=JL Boots; citation_author=KB Hall; citation_journal_title=RNA; citation_volume=7; citation_number=14; citation_pages=44-53; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Unique structural and stabilizing roles for the individual pseudouridine residues in the 1920 region of Escherichia coli 23S rRNA.; citation_author=M Meroueh; citation_author=PJ Grohar; citation_author=J Qiu; citation_author=J SantaLucia; citation_author=SA Scaringe; citation_journal_title=Nucleic Acids Res; citation_volume=28; citation_number=15; citation_pages=2075-2083; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Effects of nucleotide substitution and modification on the stability and structure of helix 69 from 28S rRNA.; citation_author=M Sumita; citation_author=JP Desaulniers; citation_author=YC Chang; citation_author=HM Chui; citation_author=L Clos; citation_journal_title=RNA; citation_volume=11; citation_number=16; citation_pages=1420-1429; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Identification of the rrmA gene encoding the 23S rRNA m1G745 methyltransferase in Escherichia coli and characterization of an m1G745-deficient mutant.; citation_author=C Gustafsson; citation_author=BC Persson; citation_journal_title=J Bacteriol; citation_volume=180; citation_number=17; citation_pages=359-365; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Ribosome structure and activity are altered in cells lacking snoRNPs that form pseudouridines in the peptidyl transferase center.; citation_author=TH King; citation_author=B Liu; citation_author=RR McCully; citation_author=MJ Fournier; citation_journal_title=Molecular Cell; citation_volume=11; citation_number=18; citation_pages=425-435; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Functional redundancy of Spb1p and a snR52-dependent mechanism for the 2′-O-ribose methylation of a conserved rRNA position in yeast.; citation_author=C Bonnerot; citation_author=L Pintard; citation_author=G Lutfalla; citation_journal_title=Molecular Cell; citation_volume=12; citation_number=19; citation_pages=1309-1315; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Spb1p-directed formation of Gm2922 in the ribosome catalytic center occurs at a late processing stage.; citation_author=B Lapeyre; citation_author=SK Purushothaman; citation_journal_title=Mol Cell; citation_volume=16; citation_number=20; citation_pages=663-669; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=The FtsJ/RrmJ heat shock protein of Escherichia coli is a 23 S ribosomal RNA methyltransferase.; citation_author=T Caldas; citation_author=E Binet; citation_author=P Bouloc; citation_author=A Costa; citation_author=J Desgres; citation_journal_title=J Biol Chem; citation_volume=275; citation_number=21; citation_pages=16414-16419; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=RNA methylation under heat shock control.; citation_author=H Bugl; citation_author=EB Fauman; citation_author=BL Staker; citation_author=F Zheng; citation_author=SR Kushner; citation_journal_title=Mol Cell; citation_volume=6; citation_number=22; citation_pages=349-360; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Maintenance of the correct open reading frame by the ribosome.; citation_author=TM Hansen; citation_author=PV Baranov; citation_author=IP Ivanov; citation_author=RF Gesteland; citation_author=JF Atkins; citation_journal_title=Embo Reports; citation_volume=4; citation_number=23; citation_pages=499-504; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=The tricodermin group of antibiotics, inhibitors of peptide bond formation by eukaryotic ribosomes.; citation_author=L Carrasco; citation_author=M Barbacid; citation_author=D Vazquez; citation_journal_title=Biochim et Biophys Acta; citation_volume=312; citation_number=24; citation_pages=368-376; citation_date=1973; " />
      <meta name="citation_reference" content="citation_title=Inhibitors of protein biosynthesis. II. Mode of action of anisomycin.; citation_author=AP Grollman; citation_journal_title=J Biol Chem; citation_volume=242; citation_number=25; citation_pages=3226-3233; citation_date=1967; " />
      <meta name="citation_reference" content="citation_title=Two classes of inhibitors of peptidyltransferase activity in eukaryotes.; citation_author=D Schindler; citation_journal_title=Nature; citation_volume=249; citation_number=26; citation_pages=38-41; citation_date=1974; " />
      <meta name="citation_reference" content="citation_title=Structures of five antibiotics bound at the peptidyl transferase center of the large ribosomal subunit.; citation_author=JL Hansen; citation_author=PB Moore; citation_author=TA Steitz; citation_journal_title=J Mol Biol; citation_volume=330; citation_number=27; citation_pages=1061-1075; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Stabilization of N-acetylphenylalanyl transfer ribonucleic acid binding to ribosomes by sparsomycin.; citation_author=AE Herner; citation_author=IH Goldberg; citation_author=LB Cohen; citation_journal_title=Biochemistry; citation_volume=8; citation_number=28; citation_pages=1335-1344; citation_date=1969; " />
      <meta name="citation_reference" content="citation_title=Localization of sparsomycin action to the peptide-bond-forming step.; citation_author=J Jayaraman; citation_author=IH Goldberg; citation_journal_title=Biochemistry; citation_volume=7; citation_number=29; citation_pages=418-421; citation_date=1968; " />
      <meta name="citation_reference" content="citation_title=Sites of interaction of the CCA end of peptidyl-tRNA with 23S rRNA.; citation_author=D Moazed; citation_author=HF Noller; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=88; citation_number=30; citation_pages=3725-3728; citation_date=1991; " />
      <meta name="citation_reference" content="citation_title=Ribosomal frameshifting efficiency and Gag/Gag-pol ratio are critical for yeast M1 double-stranded RNA virus propagation.; citation_author=JD Dinman; citation_author=RB Wickner; citation_journal_title=J Virology; citation_volume=66; citation_number=31; citation_pages=3669-3676; citation_date=1992; " />
      <meta name="citation_reference" content="citation_title=Double-stranded RNA viruses of Saccharomyces cerevisiae.; citation_author=RB Wickner; citation_journal_title=Microbiol Rev; citation_volume=60; citation_number=32; citation_pages=250-265; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=An in vivo dual-luciferase assay system for studying translational recoding in the yeast Saccharomyces cerevisiae.; citation_author=JW Harger; citation_author=JD Dinman; citation_journal_title=RNA; citation_volume=9; citation_number=33; citation_pages=1019-1024; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Structures of the bacterial ribosome at 3.5 A resolution.; citation_author=BS Schuwirth; citation_author=MA Borovinskaya; citation_author=CW Hau; citation_author=W Zhang; citation_author=A Vila-Sanjurjo; citation_journal_title=Science; citation_volume=310; citation_number=34; citation_pages=827-834; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Important contribution to catalysis of peptide bond formation by a single ionizing group within the ribosome.; citation_author=VI Katunin; citation_author=GW Muth; citation_author=SA Strobel; citation_author=W Wintermeyer; citation_author=MV Rodnina; citation_journal_title=Mol Cell; citation_volume=10; citation_number=35; citation_pages=339-346; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Simulating movement of tRNA into the ribosome during decoding.; citation_author=KY Sanbonmatsu; citation_author=S Joseph; citation_author=CS Tung; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=102; citation_number=36; citation_pages=15854-15859; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Translational maintenance of frame: mutants of Saccharomyces cerevisiae with altered −1 ribosomal frameshifting efficiencies.; citation_author=JD Dinman; citation_author=RB Wickner; citation_journal_title=Genetics; citation_volume=136; citation_number=37; citation_pages=75-86; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=Yeast virus propagation depends critically on free 60S ribosomal subunit concentration.; citation_author=Y Ohtake; citation_author=RB Wickner; citation_journal_title=Mol Cell Biol; citation_volume=15; citation_number=38; citation_pages=2772-2781; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Ribosomal frameshifting in the yeast retrotransposon Ty: tRNAs induce slippage on a 7 nucleotide minimal site.; citation_author=MF Belcourt; citation_author=PJ Farabaugh; citation_journal_title=Cell; citation_volume=62; citation_number=39; citation_pages=339-352; citation_date=1990; " />
      <meta name="citation_reference" content="citation_title=Conserved loop sequence of helix 69 in Escherichia coli 23 S rRNA is involved in A-site tRNA binding and translational fidelity.; citation_author=N Hirabayashi; citation_author=NS Sato; citation_author=T Suzuki; citation_journal_title=J Biol Chem; citation_volume=281; citation_number=40; citation_pages=17203-17211; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Mouse dyskerin mutations affect accumulation of telomerase RNA and small nucleolar RNA, telomerase activity, and ribosomal RNA processing.; citation_author=Y Mochizuki; citation_author=J He; citation_author=S Kulkarni; citation_author=M Bessler; citation_author=PJ Mason; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=101; citation_number=41; citation_pages=10756-10761; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Impaired control of IRES-mediated translation in X-linked dyskeratosis congenita.; citation_author=A Yoon; citation_author=G Peng; citation_author=Y Brandenburger; citation_author=O Zollo; citation_author=W Xu; citation_journal_title=Science; citation_volume=312; citation_number=42; citation_pages=902-906; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=High efficiency transformation of Escherichia coli with plasmids.; citation_author=H Inoue; citation_author=H Nojima; citation_author=H Okayama; citation_journal_title=Gene; citation_volume=96; citation_number=43; citation_pages=23-28; citation_date=1990; " />
      <meta name="citation_reference" content="citation_title=Transformation of intact yeast cells treated with alkali cations.; citation_author=H Ito; citation_author=Y Fukuda; citation_author=K Murata; citation_author=A Kimura; citation_journal_title=J Bacteriol; citation_volume=153; citation_number=44; citation_pages=163-168; citation_date=1983; " />
      <meta name="citation_reference" content="citation_title=Systematic analysis of bicistronic reporter assay data.; citation_author=JL Jacobs; citation_author=JD Dinman; citation_journal_title=Nucleic Acids Res; citation_volume=32; citation_number=45; citation_pages=e160-e170; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Ribosomal protein L5 helps anchor peptidyl-tRNA to the P-site in Saccharomyces cerevisiae.; citation_author=A Meskauskas; citation_author=JD Dinman; citation_journal_title=RNA; citation_volume=7; citation_number=46; citation_pages=1084-1096; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Identification of functionally important amino acids of ribosomal protein L3 by saturation mutagenesis.; citation_author=A Meskauskas; citation_author=AN Petrov; citation_author=JD Dinman; citation_journal_title=Molecular & Cellular Biology; citation_volume=25; citation_number=47; citation_pages=10863-10874; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Structural and functional analysis of 5S rRNA.; citation_author=S Kiparisov; citation_author=A Petrov; citation_author=A Meskauskas; citation_author=PV Sergiev; citation_author=OA Dontsova; citation_journal_title=Molecular Genetics and Genomics; citation_volume=27; citation_number=48; citation_pages=235-247; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=An arc of unpaired "hinge bases" facilitates information exchange among functional centers of the ribosome.; citation_author=R Rakauskaite; citation_author=JD Dinman; citation_journal_title=Mol Cell Biol; citation_volume=26; citation_number=49; citation_pages=8992-9002; citation_date=2006; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000174" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Optimization of Ribosome Structure and Function by rRNA Base Modification"/>
    <meta name="twitter:description" content="BackgroundTranslating mRNA sequences into functional proteins is a fundamental process necessary for the viability of organisms throughout all kingdoms of life. The ribosome carries out this process with a delicate balance between speed and accuracy. This work investigates how ribosome structure and function are affected by rRNA base modification. The prevailing view is that rRNA base modifications serve to fine tune ribosome structure and function.Methodology/Principal FindingsTo test this hypothesis, yeast strains deficient in rRNA modifications in the ribosomal peptidyltransferase center were monitored for changes in and translational fidelity. These studies revealed allele-specific sensitivity to translational inhibitors, changes in reading frame maintenance, nonsense suppression and aa-tRNA selection. Ribosomes isolated from two mutants with the most pronounced phenotypic changes had increased affinities for aa-tRNA, and surprisingly, increased rates of peptidyltransfer as monitored by the puromycin assay. rRNA chemical analyses of one of these mutants identified structural changes in five specific bases associated with the ribosomal A-site.Conclusions/SignificanceTogether, the data suggest that modification of these bases fine tune the structure of the A-site region of the large subunit so as to assure correct positioning of critical rRNA bases involved in aa-tRNA accommodation into the PTC, of the eEF-1A&bull;aa-tRNA&bull;GTP ternary complex with the GTPase associated center, and of the aa-tRNA in the A-site. These findings represent a direct demonstration in support of the prevailing hypothesis that rRNA modifications serve to optimize rRNA structure for production of accurate and efficient ribosomes."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0000174.g005"/>

  <meta property="og:title" content="Optimization of Ribosome Structure and Function by rRNA Base Modification" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000174" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=7326'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=167'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=7387&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0000174">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000174" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000174&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Optimization of Ribosome Structure and Function by rRNA Base Modification
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Jennifer L. Baxter-Roshek, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Cell Biology and Molecular Genetics, University of Maryland, College Park, Maryland, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Alexey N. Petrov, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Cell Biology and Molecular Genetics, University of Maryland, College Park, Maryland, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Jonathan D. Dinman
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:dinman@umd.edu">dinman@umd.edu</a>
            </p>

                <p>Affiliation:
                  Department of Cell Biology and Molecular Genetics, University of Maryland, College Park, Maryland, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: January 24, 2007</li>
    <li>DOI: 10.1371/journal.pone.0000174</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0000174" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0000174" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000174" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0000174" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0000174-g001" data-doi="info:doi/10.1371/journal.pone.0000174" data-uri="info:doi/10.1371/journal.pone.0000174.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000174.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000174-g002" data-doi="info:doi/10.1371/journal.pone.0000174" data-uri="info:doi/10.1371/journal.pone.0000174.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000174.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000174-t001" data-doi="info:doi/10.1371/journal.pone.0000174" data-uri="info:doi/10.1371/journal.pone.0000174.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000174.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000174-g003" data-doi="info:doi/10.1371/journal.pone.0000174" data-uri="info:doi/10.1371/journal.pone.0000174.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000174.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000174-g004" data-doi="info:doi/10.1371/journal.pone.0000174" data-uri="info:doi/10.1371/journal.pone.0000174.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000174.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000174-g005" data-doi="info:doi/10.1371/journal.pone.0000174" data-uri="info:doi/10.1371/journal.pone.0000174.g005" title="Figure 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000174.g005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000174">Reader Comments (6)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0000174" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2>
<h3>Background</h3>
<a id="article1.front1.article-meta1.abstract1.sec1.p1" name="article1.front1.article-meta1.abstract1.sec1.p1"></a><p>Translating mRNA sequences into functional proteins is a fundamental process necessary for the viability of organisms throughout all kingdoms of life. The ribosome carries out this process with a delicate balance between speed and accuracy. This work investigates how ribosome structure and function are affected by rRNA base modification. The prevailing view is that rRNA base modifications serve to fine tune ribosome structure and function.</p>


<h3>Methodology/Principal Findings</h3>
<a id="article1.front1.article-meta1.abstract1.sec2.p1" name="article1.front1.article-meta1.abstract1.sec2.p1"></a><p>To test this hypothesis, yeast strains deficient in rRNA modifications in the ribosomal peptidyltransferase center were monitored for changes in and translational fidelity. These studies revealed allele-specific sensitivity to translational inhibitors, changes in reading frame maintenance, nonsense suppression and aa-tRNA selection. Ribosomes isolated from two mutants with the most pronounced phenotypic changes had increased affinities for aa-tRNA, and surprisingly, increased rates of peptidyltransfer as monitored by the puromycin assay. rRNA chemical analyses of one of these mutants identified structural changes in five specific bases associated with the ribosomal A-site.</p>


<h3>Conclusions/Significance</h3>
<a id="article1.front1.article-meta1.abstract1.sec3.p1" name="article1.front1.article-meta1.abstract1.sec3.p1"></a><p>Together, the data suggest that modification of these bases fine tune the structure of the A-site region of the large subunit so as to assure correct positioning of critical rRNA bases involved in aa-tRNA accommodation into the PTC, of the eEF-1A•aa-tRNA•GTP ternary complex with the GTPase associated center, and of the aa-tRNA in the A-site. These findings represent a direct demonstration in support of the prevailing hypothesis that rRNA modifications serve to optimize rRNA structure for production of accurate and efficient ribosomes.</p>

</div>


<div class="articleinfo"><p><strong>Citation: </strong>Baxter-Roshek JL, Petrov AN, Dinman JD (2007) Optimization of Ribosome Structure and Function by rRNA Base Modification. PLoS ONE 2(1):
          e174.
            doi:10.1371/journal.pone.0000174</p><p><strong>Academic Editor: </strong>Thomas Preiss, Victor Chang Cardiac Research Institute, Australia</p><p><strong>Received:</strong> December 14, 2006; <strong>Accepted:</strong> December 21, 2006; <strong>Published:</strong> January 24, 2007</p><p><strong>Copyright:</strong> © 2007 Baxter-Roshek et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>This work was supported by a grant to JDD from the National Institutes of Health (R01 GM58859). JLB-R was supported in part by a Virology Training Grant from the National Institutes of Health (T32 AI051967).</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>Modification of ribonucleotides represents a way to expand the topological potentials of RNA molecules beyond those afforded by each of the four bases alone. Extensive research into rRNA modification has resulted in mapping of the majority of pseudouridine (<em>Ψ</em>) and 2′-O-methylation (Nm) residues in eukaryotic and archaeal ribosomes, and in identification of the snoRNA molecules that guide their modification. Despite this, little is understood about the functional roles of nucleotide modification. It is known that nucleotide modifications within the ribosome are not located randomly. This is most clearly seen in the ribosomal large subunit, where modifications cluster in highly conserved areas of the ribosome devoted to peptidyl transfer, sites of A- and P tRNA binding, the peptide exit tunnel and intersubunit bridges <a href="#pone.0000174-Ban1">[1]</a>–<a href="#pone.0000174-Decatur1">[4]</a>. This clustering is conserved in organisms ranging from <em>E. coli</em> to humans with the number of modifications increasing with evolutionary complexity <a href="#pone.0000174-Ofengand1">[5]</a>. <em>In vitro</em> reconstituted <em>E. coli</em> ribosomes lacking rRNA modifications were severely defective in catalytic activity <a href="#pone.0000174-Green1">[6]</a>, and global disruption of <em>Ψ</em> or Nm formation <em>in vivo</em> resulted in strong growth defects in yeast <a href="#pone.0000174-Tollervey1">[7]</a>, <a href="#pone.0000174-Zebarjadian1">[8]</a>. These essential modifications tend to be performed by snoRNPs that also harbor components essential for rRNA processing. However, most snoRNAs responsible for guiding rRNA modification can be individually deleted with minimal detriment to the organism <a href="#pone.0000174-Samarsky1">[3]</a>, <a href="#pone.0000174-Lowe1">[9]</a>. In fact, disruption/deletion of pseudouridine synthase proteins responsible for modification of only two or three residues in <em>E. coli</em> did not produce discernable differences in exponential growth rates between wild-type and mutant stains <em>in vivo</em>. However, rRNA modification mutants were strongly out competed by isogenic wild-type strains in competition experiments, suggesting a growth advantage conferred by the modifications <a href="#pone.0000174-Raychaudhuri1">[10]</a>, <a href="#pone.0000174-Raychaudhuri2">[11]</a>. The prevailing hypothesis is that, although rRNA modifications are individually dispensable for survival, together they may serve to optimize rRNA structure for production of accurate and efficient ribosomes.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>Based on the chemical properties of <em>Ψ</em> and Nm residues, their possible functional roles can be inferred but not established. It has been suggested that <em>Ψ</em> residues may contribute to RNA stability by altering potentials for base stacking, and by offering an extra hydrogen bond donor as compared to uridine <a href="#pone.0000174-Charette1">[12]</a>, <a href="#pone.0000174-Helm1">[13]</a>. Nm residues offer protection against hydrolysis by bases and nucleases and can promote RNA structural changes by changing the hydration sphere around the 2′ oxygen, blocking sugar edge interactions and favoring the 3′endo ribose configuration <a href="#pone.0000174-Helm1">[13]</a>, <a href="#pone.0000174-Williams1">[14]</a>. Thermodynamic and NMR based studies revealed that a <em>Ψ</em> residue can stabilize an RNA hairpin structure when located at a stem loop junction, and is slightly destabilizing when located in single-stranded loop regions <a href="#pone.0000174-Meroueh1">[15]</a>. Recent NMR studies of the highly conserved and highly modified large subunit rRNA (LSU) helix 69 of the human ribosome observed discernable but subtle secondary structure differences between rRNA with and without the modifications <a href="#pone.0000174-Sumita1">[16]</a>. Functional and structural studies have shown that rRNA modification defects can impact on translation rates and ribosome integrity. In <em>E. coli</em>, mutants lacking methylation of the m<sup>1</sup>G745 residue located in the LSU exhibited decreased growth rates, decreased rates of polypeptide chain elongation, defects in ribosome profiles, and resistance to viomycin <a href="#pone.0000174-Gustafsson1">[17]</a>. In yeast, knockout strains were made lacking each of six snoRNA genes that guide pseudouridylation of residues in the PTC of the ribosome, as well as one strain that lacked all six genes <a href="#pone.0000174-King1">[18]</a>. Only one individual mutant, the snR10 deletion strain, had phenotypic defects, but deletion of all six snoRNA genes promoted moderate defects in growth and translation rates, paromomycin hypersensitivity, and changes in ribosome profiles. <em>In vivo</em> DMS studies also revealed altered LSU rRNA structure for the multiple snoRNA deletion strain. Other functional studies have centered around two methylated nucleotides, mU<sub>2920</sub> and mG<sub>2921</sub>, in the A loop of the yeast ribosome. There are two components thought to be involved in the methylation of these rRNA residues: the guide snoRNA snR52, and the site-specific methyltransferase Spb1p, an essential yeast nucleolar protein. Primer extension analysis revealed a functionally redundant pathway whereby snR52 or Spb1p could methylate residue Um<sub>2920</sub> 
            <a href="#pone.0000174-Bonnerot1">[19]</a>. Later thin layer chromatography experiments revealed a different mechanism whereby Spb1p and snR52 were responsible for methylation of Gm<sub>2921</sub> and Um<sub>2920</sub> respectively, and showing that Spb1p could methylate residue Um<sub>2920</sub> in the absence of snR52 <a href="#pone.0000174-Lapeyre1">[20]</a>. Despite this discrepancy, it is clear that deleting both snR52 and Spb1p resulted in strong defects in growth rates, altered polysome profiles, and paromomycin hypersensitivity <a href="#pone.0000174-Bonnerot1">[19]</a>, making Spb1p an important exception to the snoRNA guided modification rule in eukaryotes. The <em>E. coli</em> homolog of Spb1p, FtsJ/RrmJ, methylates 23S rRNA residue Um<sub>2552</sub> the equivalent of yeast Um<sub>2920</sub> 
            <a href="#pone.0000174-Caldas1">[21]</a>, and deletion of this protein in <em>E. coli</em> resulted in severe growth defects, temperature sensitivity, and altered ribosome profiles <a href="#pone.0000174-Bugl1">[22]</a>.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>Despite their high level of conservation and distribution in functionally important areas of the ribosome, the functions of individual rRNA modifications belie their importance with a lack of defects in their absence. However, the changes in ribosome profiles and rRNA structures in multiple mutants suggest the intriguing possibility that they may each contribute to refining the structure and function of the translational apparatus. In order to more fully understand this, several strains harboring single deletion mutations, and one containing two gene deletions of previously characterized snoRNAs known to modify the PTC of the yeast ribosome were first characterized using a wide variety of genetic assays designed to assess translational fidelity. The results show that defects in rRNA modification produce allele specific mutant phenotypes including increased sensitivity to translational inhibitors; defects in virus propagation; changes in translational fidelity as monitored by +1 and −1 PRF, discrimination between cognate- and near-cognate aa-tRNAs, and recognition of termination codons. These analyses led to more detailed biochemical characterization of two mutants, demonstrating their increased affinities for aa-tRNAs and decreased rates of peptidyltransfer. rRNA chemical deprotection studies using a mutant defective in its ability to Nm Gm2921 and Um2920 identified structural changes in five positions. Specifically, at U2923 in the 25S rRNA A-loop in the peptidyltransferase center, at A2932 and A2933, which help to coordinate correct folding of the helix 90 – 92 structure, and at C2848 where the tip of helix 89 interacts with the GTPase-associated center, and at C2851, where helix 89 interacts with the T-stem of aa-tRNA. Together, the data suggest that modification of these bases fine tune the structure of the A-site region of the large subunit so as to assure correct positioning of critical rRNA bases involved in aa-tRNA accommodation into the PTC, of the eEF-1A•aa-tRNA•GTP ternary complex with the GTPase associated center, and of the aa-tRNA in the A-site. These findings represent the first direct demonstration in support of the prevailing hypothesis that rRNA modifications serve to optimize rRNA structure for production of accurate and efficient ribosomes.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3><a id="article1.body1.sec2.p1" name="article1.body1.sec2.p1"></a><p>In order to more precisely determine the role of rRNA modifications in the translational fidelity of the ribosome, yeast strains lacking several previously characterized snoRNAs and one protein that modify residues around the peptidyltransferase center of the ribosome were chosen for characterization. The strains contain single knockouts of the snoRNAs snR10, snR34, snR37, snR42, and snR46 which together pseudouridylate six rRNA residues in the PTC of the yeast ribosome, with snR34 modifying two of those residues. Single and double knockout strain snr52 and a methylase deficient mutant of the essential yeast protein Spb1, which are responsible for methylation of mG<sub>2921</sub> and mU<sub>2920</sub>, were also used in this study. Since Spb1p is an essential yeast protein, a methylase deficient mutant with a D to A substitution affecting the AdoMet-binding site was used <a href="#pone.0000174-Bonnerot1">[19]</a>, <a href="#pone.0000174-Lapeyre1">[20]</a>. Mutant strains <em>snr10</em>
            <em>Δ</em>, <em>spb1DA</em>, and <em>spb1DA</em>/<em>snr52</em>
            <em>Δ</em> have slow growth phenotypes. The locations of the modified bases modified are shown in <a href="#pone-0000174-g001">figure 1</a>.</p>
<div class="figure" id="pone-0000174-g001"><div class="img"><a name="pone-0000174-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000174.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000174" data-uri="info:doi/10.1371/journal.pone.0000174.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000174.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000174.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000174.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000174.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000174.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000174.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000174.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000174.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000174.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>25S rRNA in the peptiptidyl transferase center of yeast.</span></strong></p><a id="article1.body1.sec2.fig1.caption1.p1" name="article1.body1.sec2.fig1.caption1.p1"></a><p>(A) Secondary structure of yeast 25S rRNA in the PTC. snoRNAs targeted for this study are indicated along with the residues they modify. <em>Ψ</em> – pseudouridylated residue; Nm – 2′-<em>O</em>-ribose methylated residue. Helices are numbered in black. (B) Three dimensional representation of the <em>E. coli</em> PTC <a href="#pone.0000174-Schuwirth1">[34]</a>. Modified residues are labeled by the colors indicated in panel A. <em>Left</em>: view into the PTC from the top of the LSU, <em>righ</em>t: 90° rotation of <em>Left</em>. Helices and tRNAs are labeled.</p>
<span>doi:10.1371/journal.pone.0000174.g001</span></div>
<h4>rRNA modification mutants show sensitivity to translation inhibitors</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>Protein translation inhibitors that specifically interact with the ribosome provide sensitive and convenient probes for changes in ribosome function. Anisomycin, which binds the A-site crevice that normally accepts the amino acid side-chains of A-site bound aminoacyl-tRNAs <a href="#pone.0000174-Hansen1">[23]</a> interfering with the binding of 3′ end of the aa-tRNA <a href="#pone.0000174-Carrasco1">[24]</a>–<a href="#pone.0000174-Schindler1">[26]</a>, was used to probe the A-site of the peptidyltransferase center (PTC). Sparsomycin, which binds on top of the CCA end of a P-site bound substrate and interacts with it interfering <a href="#pone.0000174-Hansen2">[27]</a> with the binding of the 3′ end of the peptidyl-tRNA <a href="#pone.0000174-Herner1">[28]</a>–<a href="#pone.0000174-Moazed1">[30]</a> was also used as a probe for functional changes in the PTC. Previous studies reported that <em>spb1DA</em> mutants were sensitive to paromomycin, and that sparsomycin had no effect <a href="#pone.0000174-Bonnerot1">[19]</a>. Further, the <em>snr10</em>
               <em>Δ</em> mutant was also shown to be sensitive to paromomycin <a href="#pone.0000174-King1">[18]</a>. To obtain drug sensitivity profiles for all of the mutants, standard 10-fold dilution spot assays were performed in the presence of anisomycin or sparsomycin (20 µg/ml each). As shown in <a href="#pone-0000174-g002">Figure 2</a> and summarized in <a href="#pone-0000174-t001">Table 1</a>, <em>spb1DA</em>/<em>snr52</em>
               <em>Δ</em> cells were anisomycin hypersensitive, and <em>snr34</em>
               <em>Δ</em> and <em>snr46</em>
               <em>Δ</em> strains were hypersensitive to sparsomycin.</p>
<div class="figure" id="pone-0000174-g002"><div class="img"><a name="pone-0000174-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000174.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000174" data-uri="info:doi/10.1371/journal.pone.0000174.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000174.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000174.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000174.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000174.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000174.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000174.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000174.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000174.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000174.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Sensitivity of rRNA base modification mutants to Translational Inhibitors.</span></strong></p><a id="article1.body1.sec2.sec1.fig1.caption1.p1" name="article1.body1.sec2.sec1.fig1.caption1.p1"></a><p>Mutant and isogenic wild-type yeast strains were spotted as ten fold dilutions from 10<sup>5</sup> to 10<sup>1</sup> CFU onto YPAD media containing 20 µg/ml anisomycin or sparsomycin. Cells were incubated for 3 days at 30°C, and growth was monitored as compared to growth on plates in the absence of drug. Each strain and drug was assayed at least twice.</p>
<span>doi:10.1371/journal.pone.0000174.g002</span></div><div class="figure" id="pone-0000174-t001"><div class="img"><a name="pone-0000174-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000174.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000174" data-uri="info:doi/10.1371/journal.pone.0000174.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000174.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000174.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000174.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000174.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000174.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000174.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000174.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000174.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000174.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>Summary of rRNA modification mutant phenotypes.</span></strong></p><span>doi:10.1371/journal.pone.0000174.t001</span></div>

<h4>Virus propagation in rRNA modification mutants</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>The yeast killer virus system composed of the dsRNA L-A helper virus and M<sub>1</sub> ‘killer” satellite viruses, provides a highly sensitive assay for small defects in ribosome function. The L-A viral genome contains two overlapping ORFs, <em>gag</em> and <em>pol</em>, which encoded the structural protein and the RNA dependent RNA polymerase respectively. The two ORFs are joined by a programmed −1 ribosomal frameshift (−1 PRF) signal, and a −1 PRF event is required for synthesis a Gag-pol fusion protein. The M<sub>1</sub> satellite virus dsRNA genome encodes a secreted toxin. The pre-toxin provides the infected cell with immunity to the toxin, while secretion of the mature toxin results in death of uninfected yeast cells. Alterations in −1 PRF frequencies alter the ratio of structural to enzymatic viral proteins produced for particle assembly thereby interfering with the ability of yeast to maintain the L-A helper and M<sub>1</sub> satellite viruses <a href="#pone.0000174-Dinman1">[31]</a>. In addition, M<sub>1</sub> propagation is highly sensitive to changes in levels of free large subunits in yeast, and mutants with altered amounts of free ribosomal LSU fail to maintain the M<sub>1</sub> virus <a href="#pone.0000174-Wickner1">[32]</a>. To assess the effects of the mutants on virus propoagation, the L-A and M<sub>1</sub> viruses were introduced into isogenic wild-type and mutant cells by cytoplasmic mixing, and cells were assayed for their abilities to maintain the killer phenotype (<a href="#pone-0000174-g003">Fig. 3A</a>). The wild-type strains and several of the mutants (<em>snr42<em>Δ</em>, snr 34<em>Δ</em>,</em> and <em>snr52<em>Δ</em>
               </em>) were able to stably maintain the killer virus (K<sup>+</sup>). However, several of the mutants displayed defective killer phenotypes. The <em>snr37</em>
               <em>Δ</em> and <em>snr46</em>
               <em>Δ</em> mutants stably maintained the virus, but showed reductions in zones of killer activity phenotypes (K<sup>w</sup>). Although the killer phenotype could be initially established in <em>snr10</em>
               <em>Δ</em> cells, it was rapidly lost, resulting in K<sup>−</sup> phenotype. The killer maintenance defect was most severe in the <em>spb1DA</em> and <em>spb1DA</em>/<em>snr52</em>
               <em>Δ</em> mutants where infection could not be established. Previously published data indicates altered ribosome profiles for mutants <em>snr10</em>
               <em>Δ</em> 
               <a href="#pone.0000174-King1">[18]</a> and <em>spb1DA</em>/<em>snr52</em>
               <em>Δ</em> 
               <a href="#pone.0000174-Bonnerot1">[19]</a>, which could be a contributing factor to the observed virus propagation defects. To rule out the possibility that defects in the processing or secretion of the killer toxin <a href="#pone.0000174-Wickner1">[32]</a> were responsible for the observed killer phenotypes, double-stranded viral RNA was extracted from wild-type and mutant cells and visualized (<a href="#pone-0000174-g003">Fig. 3B</a>). The analysis revealed that M<sub>1</sub> dsRNA abundance correlated with the observed killer phenotypes; i.e. M<sub>1</sub> dsRNA was observed in the strains which showed the K<sup>+</sup> phenotype and was absent or faint in strains that showed K<sup>−</sup> or K<sup>w</sup> phenotypes respectively.</p>
<div class="figure" id="pone-0000174-g003"><div class="img"><a name="pone-0000174-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000174.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000174" data-uri="info:doi/10.1371/journal.pone.0000174.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000174.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000174.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000174.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000174.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000174.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000174.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000174.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000174.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000174.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Many of the rRNA base modification mutants have M<sub>1</sub> virus propagation defects.</span></strong></p><a id="article1.body1.sec2.sec2.fig1.caption1.p1" name="article1.body1.sec2.sec2.fig1.caption1.p1"></a><p>Yeast rRNA modification mutants were tested for their ability to maintain the L-A and M<sub>1</sub> viruses. (A) Mutant and isogenic wild-type yeast strains were spotted onto YPAD plates, and allowed to grow at 30°C, and then replica plated to a seeded lawn of 5X47 indicator cells. Plates were incubated at room temperature for 3–5 days until a zone of inhibition was clearly visible for wild-type cells. (B) Total RNAs were extracted from mutant and isogenic wild-type yeast strains and digested with RNase A under high salt conditions. The resulting double-stranded RNA was separated on a 1% agarose gel and visualized with ethidium bromide. L-A and M<sub>1</sub> dsRNAs are indicated. The image was inverted for clarity.</p>
<span>doi:10.1371/journal.pone.0000174.g003</span></div>

<h4>rRNA modification mutants cause allele-specific defects in translational fidelity</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>Assays designed to monitor various aspects of translational fidelity were employed to more precisely determine the role of rRNA modifications in translational fidelity. An <em>in vivo</em> bicistronic dual-luciferase reporter system was used to quantitatively monitor changes in −1 and +1 PRF, suppression of nonsense codons, and fidelity of aa-tRNA selection <a href="#pone.0000174-Harger1">[33]</a>. The control reporter is a yeast expression vector containing <em>Renilla</em> and firefly luciferase genes, which yields active <em>Renilla</em> and firefly luciferase proteins. Programmed −1 and +1 frameshifting test reporters were constructed by inserting a frameshift signal, L-A or Ty<em>1</em> respectively, between the <em>Renilla</em> and firefly genes such that firefly luciferase can only be produced in the event of a frameshift. <em>Renilla</em> luciferase serves as an internal control, eliminating effects due to differences in mRNA abundance, mRNA stability or translation rates between the test and control reporters. Nonsense suppression test reporters contained a stop codon (UAA, UAG, or UGA) six codons into the firefly luciferase gene, so that firefly luciferase can only be produced consequent to nonsense suppression. The misincorporation test reporters were created by mutating the firefly luciferase catalytic residue R218 from the wild-type AGA codon to either the near-cognate AGC leucine codon, or the non-cognate TCT leucine codon, so that active firefly luciferase can only synthesized when the incorrect tRNA<sup>Arg</sup> is selected. Recoding efficiencies were measured for each mutant as described in the <a href="#s4">Materials and Methods</a>. These translational fidelity data are summarized in <a href="#pone-0000174-t001">Table 1</a>.</p>
<a id="article1.body1.sec2.sec3.p2" name="article1.body1.sec2.sec3.p2"></a><p>The <em>spb1DA</em>/<em>snr52</em>
               <em>Δ</em> double mutant showed a significant increase in −1 PRF efficiency (1.5 fold of wild-type, <em>p</em> = 2.7×10<sup>−15</sup>), consistent with the inability of this strain to propagate the M<sub>1</sub> killer virus. None of the other mutants significantly affected −1 PRF (1.2 fold or less than wild type). Similarly, none of the mutants had significant effects on +1 PRF (1.2 fold or less). The most dramatic effects of the mutants were observed in their abilities to recognize termination codons. With the exception of <em>snr42</em>
               <em>Δ</em>, which was hyperaccurate at UAA and UGA, but not at UAG codons, similar trends for changes in nonsense suppression were observed for all three stop codons in the other mutant strains. For example, the <em>snr37</em>
               <em>Δ</em>, <em>snr10</em>
               <em>Δ</em>, and the double mutant <em>spb1DA</em>/<em>snr52</em>
               <em>Δ</em> strains were hyperaccurate with respect to their ability to recognize all three termination codons, while a significant increase in general nonsense suppression was displayed in the <em>snr46</em>
               <em>Δ</em> mutant strain. The mutants were also assayed with regard to their ability to discriminate between sense and missense codons by using reporters described in the Supporting Information <a href="#pone.0000174.s001">Materials and Methods S1</a>. The <em>snr10</em>
               <em>Δ</em> mutant strain slightly enhanced misreading of both near and non-cognate codons (1.3 fold&gt;wild-type), and <em>snr46</em>
               <em>Δ</em> showed a small increase in selection of non-cognate aa-tRNA (1.3 fold of wild type). The double mutant <em>spb1DA</em>/<em>snr52</em>
               <em>Δ</em> had a very significant effect on near cognate aa-tRNA misreading (1.9 fold wild-type), but did not significantly affect non-cognate aa-tRNA selection. The <em>spb1DA</em> single mutant actually exhibited a slight (0.8 fold) decrease in non-cognate aa-tRNA selection when compared to its wild-type strain with no effect on near-cognate selection events. The <em>snr52</em>
               <em>Δ</em> single mutant showed no change in non-cognate aa-tRNA selection events, but did display a 1.4 fold increase in near-cognate values.</p>


<h4>Changes in aminoacyl-tRNA binding and peptidyltransfer rates</h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>Defects in translational fidelity could possibly be due to changes in tRNA binding to the ribosome or rates of peptidyltransfer. The two mutants with the most dramatic phenotypic effects, <em>snr46</em>
               <em>Δ</em> and <em>spb1DA</em>/<em>snr52</em>
               <em>Δ</em>, were selected for more detailed biochemical characterization. Ribosomes were isolated from the isogenic wild-type and mutant strains and their affinities for aa- and peptidyl-tRNA as well as peptidyltransfer rates were determined as described in the Supporting Information <a href="#pone.0000174.s001">Materials and Methods S1</a>. Ribosomes from both mutants had increased affinities for [<sup>14</sup>C]Phe-tRNA (<a href="#pone-0000174-g004">Fig. 4A</a>). Specifically, the Kd<sub>app</sub> for <em>snr46</em>
               <em>Δ</em> ribosomes was 0.3 µM ±0.05, as compared to 1.8 µM ±0.4 by ribosomes isolated from the corresponding isogenic wild-type strain. Similarly, ribosomes isolated from the <em>spb1DA</em>/<em>snr52</em>
               <em>Δ</em> mutant had a Kd<sub>app</sub> of 0.3 µM ±0.1 as compared to 0.6 µM ±0.2 for the corresponding isogenic wild-type <em>SPB1 SNR52</em> strain. In contrast, neither mutant affected binding of Ac-[<sup>14</sup>C]Phe-tRNA to the ribosomal P-site (<a href="#pone-0000174-g004">Fig 4B</a>). Peptidyltransfer rates were measured using the puromycin reaction as described in the methods. Surprisingly, both mutant strains promoted increased rates of peptidyltransfer as compared to isogenic wild-type values (<a href="#pone-0000174-g004">Fig. 4C</a>). Specifically, the K<sub>app</sub> of <em>snr46</em>
               <em>Δ</em> ribosomes was 0.06 min<sup>−1</sup> ±0.003 while its isogenic wild-type showed a K<sub>app</sub> value of 0.04 min<sup>−1</sup> ±0.003. The wild-type <em>SPB1 SNR52</em> was shown to have a K<sub>app</sub> of 0.02 min<sup>−1</sup> ±0.004 while the mutant strain <em>spb1DA</em>/<em>snr52</em>
               <em>Δ</em> showed a K<sub>app</sub> of 0.04 min<sup>−1</sup> ±0.007.</p>
<div class="figure" id="pone-0000174-g004"><div class="img"><a name="pone-0000174-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000174.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000174" data-uri="info:doi/10.1371/journal.pone.0000174.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000174.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000174.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000174.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000174.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000174.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000174.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000174.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000174.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000174.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>Ribosome biochemistry.</span></strong></p><a id="article1.body1.sec2.sec4.fig1.caption1.p1" name="article1.body1.sec2.sec4.fig1.caption1.p1"></a><p>Mutant strain <em>snr46</em>
                     <em>Δ</em> and the isogenic wild-type are shown in the top row, and mutant strain <em>spb1DA/snr52</em>
                     <em>Δ</em> and the isogenic wild-type are show in the bottom row. Error bars represent standard error for all graphs. A. [<sup>14</sup>C]Phe-tRNA binding to the A-site of the ribosome. One site binding curves of bound tRNA as analyzed using GraphPad Prism software. Data are reported as a percentage of the total tRNA bound. B. Ac-[<sup>14</sup>C]Phe-tRNA binding to the P-site of the ribosome. One site binding curves of bound tRNA as analyzed using GraphPad Prism software. Data are reported as a percentage of the total tRNA bound. C. Peptidyltransfer. Timecourse assays of peptidyltransferase activities as measured by the puromycin reaction.</p>
<span>doi:10.1371/journal.pone.0000174.g004</span></div>

<h4>The spb1DA/snr52<em>Δ</em> mutant promotes changes in key rRNA structural elements that interact with aa-tRNA</h4>
<a id="article1.body1.sec2.sec5.p1" name="article1.body1.sec2.sec5.p1"></a><p>It has been speculated that the post-transcriptional rRNA modification may serve to increase the stability of the local RNA structure or decrease risk of degradation <a href="#pone.0000174-Decatur1">[4]</a>, <a href="#pone.0000174-Helm1">[13]</a>. With this in mind, <em>in vitro</em> rRNA structure probing was performed on the wild-type and mutant ribosomes biochemically characterized in the previous section. Mutants <em>snr46</em>
               <em>Δ</em> and <em>spb1DA</em>/<em>snr52</em>
               <em>Δ</em> and isogenic wild-type puromycin treated ribosomes were incubated with the chemically modifying agents CMCT, kethoxal and DMS <em>in vitro</em>. rRNAs were extracted and primer extension analyses performed using primers sufficient to transverse the entire PTC i.e. helices 89-93. <a href="#pone-0000174-g005">Figure 5A</a> shows a representative autoradiogram for the wild-type and mutant <em>spb1DA</em>/<em>snr52</em>
               <em>Δ</em> strains. Differences between wild-type and mutant protection patterns and their nucleotide locations are indicated. Residues C2843 and C2851 in helix 89 were deprotected from DMS, and residue U2923 in the A-loop showed increased protection from CMCT. Weaker, but consistent deprotection patterns of A2932 and A2933 were also observed. These are all mapped into the context of the 2-dimensional map of the yeast 25S rRNA (<a href="#pone-0000174-g005">Fig. 5B</a>), and within the atomic resolution 3-dimensional structure of the <em>E. coli</em> ribosome (<a href="#pone-0000174-g005">Fig. 5C</a>) <a href="#pone.0000174-Schuwirth1">[34]</a>. The increased intensity corresponding to U2845 (marked by * in <a href="#pone-0000174-g005">Fig. 5A</a>) is not DMS-specific, and was not repeatable. No significant differences in protection patterns were observed between isogenic wild-type and mutant <em>snr46</em>
               <em>Δ</em> ribosomes (data not shown).</p>
<div class="figure" id="pone-0000174-g005"><div class="img"><a name="pone-0000174-g005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000174.g005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000174" data-uri="info:doi/10.1371/journal.pone.0000174.g005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000174.g005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000174.g005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000174.g005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000174.g005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000174.g005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000174.g005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000174.g005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000174.g005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000174.g005.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 5.  <span>25S rRNA structure probing analysis in <em>spb1DA/snr52</em>
                     <em>Δ</em> mutant ribosomes.</span></strong></p><a id="article1.body1.sec2.sec5.fig1.caption1.p1" name="article1.body1.sec2.sec5.fig1.caption1.p1"></a><p>(A) Puromycin treated ribosomes isolated from isogenic wild-type and <em>spb1DA/snr52</em>
                     <em>Δ</em> mutant strains were used for <em>in vitro</em> chemical probing of the structure around the peptidyltransferase center of the ribosome, specifically helices 89-93. Reactions were performed in triplicate, representative autoradiographs are shown. U - untreated; C - CMCT; D – DMS; K – Kethoxal. Residues with changes in banding pattern labeled. Strongly modified bases at positions U2923, C2843, and C2851 are indicated, as are the more weakly deprotected A2932 and A2933. The increased intensity corresponding to U2845 (marked by *) is not DMS-specific. (B–D). Locations of residues demonstrating strong changes in protection patterns mapped to LSU rRNA structures. Um2920 and Gm2921 are indicated and color coded. Bases with altered protection patterns are circled in orange. aa-tRNA accommodation ‘gate’ bases <a href="#pone.0000174-Sanbonmatsu1">[36]</a> are indicated with purple (gate 1) and green (gate 2). The “catalytic” A base (equivalent to <em>E. coli</em> 23S rRNA A2451) is incicated with red. Individual LSU helices are numbered and color coded. (B) Secondary structure of yeast 25S rRNA around the PTC. (C) Three dimensional representation of rRNA bases of interest mapped onto the <em>E. coli</em> PTC <a href="#pone.0000174-Schuwirth1">[34]</a>. Arrow represents the path the 3′ end of the aa-tRNA travels when entering the A-site. (D) Rotation and zoom in of Panel C. Shows the path into the A-site from the aa-tRNA perspective. Residues demonstrating changes in protection patterns are labeled in orange. Locations of modified bases and A-site ‘gate’ residues are labeled.</p>
<span>doi:10.1371/journal.pone.0000174.g005</span></div>
</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p>It is presently thought that rRNA base modification serves to fine tune the ribosome structure so as to optimize ribosome biogenesis and the various functions carried out by mature ribosomes. The current study has tested this hypothesis by focusing on modified bases in the A-site region of the large subunit. The data presented here provide the most detailed structure/function analysis to date, showing how minor changes in rRNA structure assure correct positioning of critical rRNA bases involved in guiding and placement of aa-tRNAs into the ribosomal A-site.</p>
<a id="article1.body1.sec3.p2" name="article1.body1.sec3.p2"></a><p>The observation of increased rates of peptidyltransfer in the <em>spb1DA</em>/<em>snr52</em>
            <em>Δ</em> and <em>snr46</em>
            <em>Δ</em> ribosomes may provide the key to understanding the function of these modified bases. Such an observation is unusual because this reaction is normally nearly instantaneous (&gt;50/sec) <a href="#pone.0000174-Katunin1">[35]</a>. To understand this, it is necessary to examine the reaction in the context of the puromyucin assay, which is approximately 3 orders of magnitude slower than peptidyltransferase assays performed with whole aa-tRNAs. One of the rate limiting aspects of the puromycin reaction is the requirement for this small molecule to diffuse into the A-site of ribosomes that are pre-loaded with peptidyl-tRNA. Thus, a simple way to increase the rate of this reaction would be to increase rates of puromycin diffusion into the active site. This could be accomplished by widening the path through which puromycin must travel in order to access the A-site. The structure probing data from <em>spb1DA</em>/<em>snr52</em>
            <em>Δ</em> ribosomes reveals that C2843 and C2851, two critical bases lying along the path taken by the 3′ end of the aa-tRNA during accommodation <a href="#pone.0000174-Sanbonmatsu1">[36]</a>, are deprotected (<a href="#pone-0000174-g005">Fig. 5A–C</a>). This is consistent with the model of this channel being more open in this mutant. Accommodating aa-tRNA slides along the side of the helix 90 – 92 structure, and A2032 and A2933 play important roles in coordinating proper folding of this structure. Thus, the observed mild deprotection of these two bases is also consistent with the ‘open accommodation channel’ model. In addition, hyperprotection of U2923 may be due to this base collapsing into the space normally occupied by the methyl groups attached to the nearby Um2920 and Gm2921 (see <a href="#pone-0000174-g005">Fig. 5D</a>). This movement would drag C2922 along with it. C2922 is one of two bases that form the first “gate” through which accommodating aa-tRNA must pass <a href="#pone.0000174-Sanbonmatsu1">[36]</a> (shown as orange arrows in <a href="#pone-0000174-g005">Figs. 5C and 5D</a>). Thus, repositioning it toward the 5′ side of the A-loop would serve to open this gate, further lessening steric hindrance to puromycin.</p>
<a id="article1.body1.sec3.p3" name="article1.body1.sec3.p3"></a><p>This structural model can account for all of the other biochemical and genetic phenotypes associated with the double mutant. Opening of the aa-tRNA gates would enhance rates of diffusion of small molecules, e.g. anisomycin and sparsomycin, into the peptidyltransferase center, resulting in the hypersensitivity to these drugs observed with the <em>spb1DA</em>/<em>snr52</em>
            <em>Δ</em> and <em>snr46</em>
            <em>Δ</em> mutants respectively. This could also enhance rates of aa-tRNA accommodation, resulting in increased affinity for aa-tRNA, and increased rates of aa-tRNA misincorporation at near-cognate and termination codons. Increased −1 PRF, which involves aa-tRNA slippage could be due to loss of the interaction between C2851 and the T-stem of aa-tRNA, i.e. the aa-tRNA may not be as well fitted into the LSU, perhaps making it more prone to slip. In turn, the inability to propagate the killer virus is consistent with increased −1 PRF <a href="#pone.0000174-Dinman2">[37]</a>, although it should be noted that propagation of the M<sub>1</sub> satellite virus is extremely sensitive to defects in the translational apparatus independent of changes in −1 PRF <a href="#pone.0000174-Ohtake1">[38]</a>. Importantly, +1 PRF, which only involves peptidyl-tRNA slippage <a href="#pone.0000174-Belcourt1">[39]</a>, was not affected by these mutants, consistent with observation that none of the mutants affected binding of Ac-[<sup>14</sup>C]Phe to the P-site.</p>
<a id="article1.body1.sec3.p4" name="article1.body1.sec3.p4"></a><p>We speculate that the <em>spb1DA/snr52<em>Δ</em>
            </em> double mutant displayed the most severe phenotypic defects because loss of methyl groups on 2 adjacent bases had the greatest effects on local hydration spheres, thus producing the most dramatic changes in rRNA structure. This suggests that the other mutants may also alter rRNA structure, but that these were too slight to be detected by the methods employed. However, we suggest that the effects of small structural changes are amplified through chains of biological events, thus producing the observed phenotypes. In particular, the ability of ribosomes to recognize termination codons, and the ability of cells to maintain the killer virus appear to be exquisitely sensitive indicators of subtle changes in the translational apparatus. Our conclusion is that rRNA base modification serves to fine-tune ribosome structure so as to best coordinate the structure of the molecule with its functional requirements.</p>
<a id="article1.body1.sec3.p5" name="article1.body1.sec3.p5"></a><p>There are other functionally important regions of the ribosome that contain high densities of rRNA modifications. For example, the loop of helix 69 has at least three modified residues, makes intersubunit contacts, and interacts with both A- and P-site tRNAs <a href="#pone.0000174-Yusupov1">[2]</a>. Functional studies in <em>E. coli</em> revealed defects in translational fidelity associated with helix 69 mutants <a href="#pone.0000174-Hirabayashi1">[40]</a>. Extensive genetic, biochemical and structural analysis of this helix would likely provide a wealth of information concerning translational fidelity and subunit association. Another interesting facet of rRNA modification is its possible role in human disease. X-linked dyskeratosis congenital (X-linked DC), marked by skin and bone marrow failure in humans, is caused by point mutations in the gene encoding the nucleolar protein dyskerin. Dyskerin is present in both the telomerase complex and in ribonucleoparticles that pseudouridylate rRNA residues. Mutations in dyskerin are associated with severe telomere dysfunction and defects in pre-rRNA processing <a href="#pone.0000174-Mochizuki1">[41]</a>. Interestingly, cells with mutant dyskerin activity also demonstrate a defect in translation of messenger RNAs containing IRES elements <a href="#pone.0000174-Yoon1">[42]</a>. It will be interesting to explore the possible relationship between rRNA modification and regulation of IRES dependent translation.</p>
</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Materials and Methods"></a><h3>Materials and Methods</h3><a id="article1.body1.sec4.p1" name="article1.body1.sec4.p1"></a><p>Detailed descriptions of materials and methods are available in the Supporting Information <a href="#pone.0000174.s001">Materials and Methods S1</a>.</p>

<h4>Strains, media, and genetic methods</h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p>The <em>S. cerevisiae</em> strains used in this study are presented in Table S1. <em>Escherichia coli</em> strain DH5α was used to amplify plasmids (listed in Table S2), and <em>E. coli</em> transformations were performed using the high-efficiency transformation method <a href="#pone.0000174-Inoue1">[43]</a>. Yeast cells were transformed using the alkali cation method <a href="#pone.0000174-Ito1">[44]</a>. YPAD and synthetic complete medium (H-), as well as YPG, SD, and 4.7 MB plates used for testing the killer phenotype were prepared and used as described previously <a href="#pone.0000174-Dinman2">[37]</a>. Oligonucleotide primers were purchased from IDT (Coralville, IA) and are listed in Table S3. Yeast deletion strains <em>snr10</em>
               <em>Δ</em>, <em>34</em>
               <em>Δ</em>, <em>37</em>
               <em>Δ</em>, <em>42</em>
               <em>Δ</em>, <em>46</em>
               <em>Δ</em> and isogenic wild-type were provided by M.J. Fournier. Yeast strains s<em>nr52</em>
               <em>Δ</em>, s<em>pb1DA</em>, the double mutant and an isogenic wild-type were provided by G. Lutfalla. Cytoduction of the L-A and M<sub>1</sub> killer virus into snoRNA knockout strains and subsequent killer virus assays were carried out as previously described <a href="#pone.0000174-Dinman1">[31]</a>. Total RNAs extracted from cytoduced wild-type and snoRNA knockout strains were analyzed for the presence of the L-A and M<sub>1</sub> dsRNAs, and dual luciferase assays to quantitatively monitor translational recoding in yeast were performed as previously described <a href="#pone.0000174-Harger1">[33]</a>. The latter involved use of a 0-frame control reporter in combination with −1, or +1 ribosomal frameshift, nonsense suppression, or misincorporation test reporter constructs. Recoding efficiencies and statistical analyses were performed as previously described <a href="#pone.0000174-Jacobs1">[45]</a>. Ten-fold dilution spot assays to monitor sensitivity to anisomycin or sparsomycin (20 µg/mL) were performed as previously described <a href="#pone.0000174-Meskauskas1">[46]</a>.</p>


<h4>Ribosome biochemistry</h4>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1"></a><p>
               <em>S. cerevisiae</em> ribosomes were isolated, yeast aminoacyl-tRNA synthetases were purified, yeast phenylalanyl-tRNAs were aminoacylated with [<sup>14</sup>C]Phe, and Ac-[<sup>14</sup>C]tRNA was synthesized as previously described <a href="#pone.0000174-Meskauskas2">[47]</a>. [<sup>14</sup>C]Phe-tRNA and Ac-[<sup>14</sup>C]tRNA were purified by HPLC. Peptidyltransfer assays using Ac-[<sup>14</sup>C]Phe-tRNA and puromycin, and equilibrium binding studies of [<sup>14</sup>C]Phe-tRNA binding to the ribosomal A-site, and of Ac-[<sup>14</sup>C]Phe-tRNA to the ribosomal P-site were carried out as previously described <a href="#pone.0000174-Meskauskas2">[47]</a>. The data were fitted to a one site binding model using Prism Graph Pad software. Chemical protections studies of 25S rRNA employed puromycin treated ribosomes incubated with DMS (dimethyl sulfate), kethoxal or CMCT (1-cyclohexyl-3-(2-morpholinoethyl) carbodiimide metho-p-toluene), and the rRNA modifications were visualized by primer extension reactions using AMV reverse transcriptase (Roche, Mannheim, Germany) and the <sup>32</sup>P-end-labeled primers shown in Table S3 were performed as previously described <a href="#pone.0000174-Kiparisov1">[48]</a>, <a href="#pone.0000174-Rakauskaite1">[49]</a>.</p>

</div>

<div id="section5" class="section"><a id="s5" name="s5" toc="s5" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0000174"></div><a name="pone.0000174.s001" id="pone.0000174.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000174.s001">Materials and Methods S1. </a></strong></p><p class="siDoi">doi:10.1371/journal.pone.0000174.s001</p><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1"></a><p class="postSiDOI">(0.09 MB DOC)</p>
<a name="pone.0000174.s002" id="pone.0000174.s002"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000174.s002">Table S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material2.caption1.p1" name="article1.body1.sec5.supplementary-material2.caption1.p1"></a><p class="preSiDOI">Yeast strains used in this study.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000174.s002</p><a id="article1.body1.sec5.supplementary-material2.caption1.p2" name="article1.body1.sec5.supplementary-material2.caption1.p2"></a><p class="postSiDOI">(0.04 MB DOC)</p>
<a name="pone.0000174.s003" id="pone.0000174.s003"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000174.s003">Table S2. </a></strong></p><a id="article1.body1.sec5.supplementary-material3.caption1.p1" name="article1.body1.sec5.supplementary-material3.caption1.p1"></a><p class="preSiDOI">Plasmid list.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000174.s003</p><a id="article1.body1.sec5.supplementary-material3.caption1.p2" name="article1.body1.sec5.supplementary-material3.caption1.p2"></a><p class="postSiDOI">(0.05 MB DOC)</p>
<a name="pone.0000174.s004" id="pone.0000174.s004"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000174.s004">Table S3. </a></strong></p><a id="article1.body1.sec5.supplementary-material4.caption1.p1" name="article1.body1.sec5.supplementary-material4.caption1.p1"></a><p class="preSiDOI">Oligonucleotides</p>
<p class="siDoi">doi:10.1371/journal.pone.0000174.s004</p><a id="article1.body1.sec5.supplementary-material4.caption1.p2" name="article1.body1.sec5.supplementary-material4.caption1.p2"></a><p class="postSiDOI">(0.03 MB DOC)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>The authors thank Drs. M.J. Fournier and G. Lutfalla for generously providing us with yeast strains. Thanks to John Russ for help with molecular modeling, and to Rasa Rakauskatie, Arturas Meskauskas, and Kristi Muldoon-Jacobs for overall assistance.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: JD JB. Performed the experiments: JB AP. Analyzed the data: JD JB. Contributed reagents/materials/analysis tools: JB AP. Wrote the paper: JD JB.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0000174-Ban1" id="pone.0000174-Ban1"></a>Ban N, Nissen P, Hansen J, Moore PB, Steitz TA (2000) The complete atomic structure of the large ribosomal subunit at 2.4 A resolution. Science  289: 905–920.  <ul class="find" data-citedArticleID="927590" data-doi="10.1126/science.289.5481.905"><li><a href="http://dx.doi.org/10.1126/science.289.5481.905" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+complete+atomic+structure+of+the+large+ribosomal+subunit+at+2.4+A+resolution." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+complete+atomic+structure+of+the+large+ribosomal+subunit+at+2.4+A+resolution.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0000174-Yusupov1" id="pone.0000174-Yusupov1"></a>Yusupov MM, Yusupova GZ, Baucom A, Lieberman K, Earnest TN, et al.  (2001) Crystal Structure of the Ribosome at 5.5 A Resolution. Science  292: 883–896.  <ul class="find" data-citedArticleID="927684" data-doi="10.1126/science.1060089"><li><a href="http://dx.doi.org/10.1126/science.1060089" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Crystal+Structure+of+the+Ribosome+at+5.5+A+Resolution." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Crystal+Structure+of+the+Ribosome+at+5.5+A+Resolution.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0000174-Samarsky1" id="pone.0000174-Samarsky1"></a>Samarsky DA, Fournier MJ (1999) A comprehensive database for the small nucleolar RNAs from Saccharomyces cerevisiae. Nucleic Acids Res  27: 161–164.  <ul class="find" data-citedArticleID="927666" data-doi="10.1093/nar/27.1.161"><li><a href="http://dx.doi.org/10.1093/nar/27.1.161" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+comprehensive+database+for+the+small+nucleolar+RNAs+from+Saccharomyces+cerevisiae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+comprehensive+database+for+the+small+nucleolar+RNAs+from+Saccharomyces+cerevisiae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0000174-Decatur1" id="pone.0000174-Decatur1"></a>Decatur WA, Fournier MJ (2002) rRNA modifications and ribosome function. Trends Biochem Sci  27: 344–351.  <ul class="find" data-citedArticleID="927604" data-doi="10.1016/s0968-0004(02)02109-6"><li><a href="http://dx.doi.org/10.1016/s0968-0004(02)02109-6" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=rRNA+modifications+and+ribosome+function." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22rRNA+modifications+and+ribosome+function.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0000174-Ofengand1" id="pone.0000174-Ofengand1"></a>Ofengand J, Bakin A, Wrzesinski J, Nurse K, Lane BG (1995) The pseudouridine residues of ribosomal RNA. Biochem Cell Biol  73: 915–924.  <ul class="find" data-citedArticleID="927656" data-doi="10.1139/o95-099"><li><a href="http://dx.doi.org/10.1139/o95-099" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+pseudouridine+residues+of+ribosomal+RNA." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+pseudouridine+residues+of+ribosomal+RNA.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0000174-Green1" id="pone.0000174-Green1"></a>Green R, Noller HF (1996) In vitro complementation analysis localizes 23S rRNA posttranscriptional modifications that are required for Escherichia coli 50S ribosomal subunit assembly and function. RNA  2: 1011–1021.  <ul class="find" data-citedArticleID="927610"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=In+vitro+complementation+analysis+localizes+23S+rRNA+posttranscriptional+modifications+that+are+required+for+Escherichia+coli+50S+ribosomal+subunit+assembly+and+function.&amp;auth=&amp;atitle=In+vitro+complementation+analysis+localizes+23S+rRNA+posttranscriptional+modifications+that+are+required+for+Escherichia+coli+50S+ribosomal+subunit+assembly+and+function." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=In+vitro+complementation+analysis+localizes+23S+rRNA+posttranscriptional+modifications+that+are+required+for+Escherichia+coli+50S+ribosomal+subunit+assembly+and+function." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22In+vitro+complementation+analysis+localizes+23S+rRNA+posttranscriptional+modifications+that+are+required+for+Escherichia+coli+50S+ribosomal+subunit+assembly+and+function.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0000174-Tollervey1" id="pone.0000174-Tollervey1"></a>Tollervey D, Lehtonen H, Jansen R, Kern H, Hurt EC (1993) Temperature-sensitive mutations demonstrate roles for yeast fibrillarin in pre-rRNA processing, pre-rRNA methylation, and ribosome assembly. Cell  72: 443–457.  <ul class="find" data-citedArticleID="927676" data-doi="10.1016/0092-8674(93)90120-f"><li><a href="http://dx.doi.org/10.1016/0092-8674(93)90120-f" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Temperature-sensitive+mutations+demonstrate+roles+for+yeast+fibrillarin+in+pre-rRNA+processing%2C+pre-rRNA+methylation%2C+and+ribosome+assembly." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Temperature-sensitive+mutations+demonstrate+roles+for+yeast+fibrillarin+in+pre-rRNA+processing%2C+pre-rRNA+methylation%2C+and+ribosome+assembly.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0000174-Zebarjadian1" id="pone.0000174-Zebarjadian1"></a>Zebarjadian Y, King T, Fournier MJ, Clarke L, Carbon J (1999) Point mutations in yeast CBF5 can abolish in vivo pseudouridylation of rRNA. Mol Cell Biol  19: 7461–7472.  <ul class="find" data-citedArticleID="927686"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Point+mutations+in+yeast+CBF5+can+abolish+in+vivo+pseudouridylation+of+rRNA.&amp;auth=&amp;atitle=Point+mutations+in+yeast+CBF5+can+abolish+in+vivo+pseudouridylation+of+rRNA." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Point+mutations+in+yeast+CBF5+can+abolish+in+vivo+pseudouridylation+of+rRNA." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Point+mutations+in+yeast+CBF5+can+abolish+in+vivo+pseudouridylation+of+rRNA.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0000174-Lowe1" id="pone.0000174-Lowe1"></a>Lowe TM, Eddy SR (1999) A computational screen for methylation guide snoRNAs in yeast. Science  283: 1168–1171.  <ul class="find" data-citedArticleID="927644" data-doi="10.1126/science.283.5405.1168"><li><a href="http://dx.doi.org/10.1126/science.283.5405.1168" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+computational+screen+for+methylation+guide+snoRNAs+in+yeast." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+computational+screen+for+methylation+guide+snoRNAs+in+yeast.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0000174-Raychaudhuri1" id="pone.0000174-Raychaudhuri1"></a>Raychaudhuri S, Conrad J, Hall BG, Ofengand J (1998) A pseudouridine synthase required for the formation of two universally conserved pseudouridines in ribosomal RNA is essential for normal growth of Escherichia coli. RNA  4: 1407–1417.  <ul class="find" data-citedArticleID="927662" data-doi="10.1017/s1355838298981146"><li><a href="http://dx.doi.org/10.1017/s1355838298981146" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+pseudouridine+synthase+required+for+the+formation+of+two+universally+conserved+pseudouridines+in+ribosomal+RNA+is+essential+for+normal+growth+of+Escherichia+coli." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+pseudouridine+synthase+required+for+the+formation+of+two+universally+conserved+pseudouridines+in+ribosomal+RNA+is+essential+for+normal+growth+of+Escherichia+coli.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0000174-Raychaudhuri2" id="pone.0000174-Raychaudhuri2"></a>Raychaudhuri S, Niu L, Conrad J, Lane BG, Ofengand J (1999) Functional effect of deletion and mutation of the Escherichia coli ribosomal RNA and tRNA pseudouridine synthase RluA. J Biol Chem  274: 18880–18886.  <ul class="find" data-citedArticleID="927664" data-doi="10.1074/jbc.274.27.18880"><li><a href="http://dx.doi.org/10.1074/jbc.274.27.18880" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Functional+effect+of+deletion+and+mutation+of+the+Escherichia+coli+ribosomal+RNA+and+tRNA+pseudouridine+synthase+RluA." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Functional+effect+of+deletion+and+mutation+of+the+Escherichia+coli+ribosomal+RNA+and+tRNA+pseudouridine+synthase+RluA.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0000174-Charette1" id="pone.0000174-Charette1"></a>Charette M, Gray MW (2000) Pseudouridine in RNA: what, where, how, and why. IUBMB Life  49: 341–351.  <ul class="find" data-citedArticleID="927602" data-doi="10.1080/152165400410182"><li><a href="http://dx.doi.org/10.1080/152165400410182" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Pseudouridine+in+RNA%3A+what%2C+where%2C+how%2C+and+why." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Pseudouridine+in+RNA%3A+what%2C+where%2C+how%2C+and+why.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0000174-Helm1" id="pone.0000174-Helm1"></a>Helm M (2006) Post-transcriptional nucleotide modification and alternative folding of RNA. Nucleic Acids Res  34: 721–733.  <ul class="find" data-citedArticleID="927622" data-doi="10.1093/nar/gkj471"><li><a href="http://dx.doi.org/10.1093/nar/gkj471" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Post-transcriptional+nucleotide+modification+and+alternative+folding+of+RNA." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Post-transcriptional+nucleotide+modification+and+alternative+folding+of+RNA.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0000174-Williams1" id="pone.0000174-Williams1"></a>Williams DJ, Boots JL, Hall KB (2001) Thermodynamics of 2′-ribose substitutions in UUCG tetraloops. RNA  7: 44–53.  <ul class="find" data-citedArticleID="927680" data-doi="10.1017/s1355838201001558"><li><a href="http://dx.doi.org/10.1017/s1355838201001558" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Thermodynamics+of+2%26%23x2032%3B-ribose+substitutions+in+UUCG+tetraloops." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Thermodynamics+of+2%26%23x2032%3B-ribose+substitutions+in+UUCG+tetraloops.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0000174-Meroueh1" id="pone.0000174-Meroueh1"></a>Meroueh M, Grohar PJ, Qiu J, SantaLucia J Jr, Scaringe SA, et al.  (2000) Unique structural and stabilizing roles for the individual pseudouridine residues in the 1920 region of Escherichia coli 23S rRNA. Nucleic Acids Res  28: 2075–2083.  <ul class="find" data-citedArticleID="927646" data-doi="10.1093/nar/28.10.2075"><li><a href="http://dx.doi.org/10.1093/nar/28.10.2075" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Unique+structural+and+stabilizing+roles+for+the+individual+pseudouridine+residues+in+the+1920+region+of+Escherichia+coli+23S+rRNA." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Unique+structural+and+stabilizing+roles+for+the+individual+pseudouridine+residues+in+the+1920+region+of+Escherichia+coli+23S+rRNA.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0000174-Sumita1" id="pone.0000174-Sumita1"></a>Sumita M, Desaulniers JP, Chang YC, Chui HM, Clos L, et al.  (2005) Effects of nucleotide substitution and modification on the stability and structure of helix 69 from 28S rRNA. RNA  11: 1420–1429.  <ul class="find" data-citedArticleID="927674" data-doi="10.1261/rna.2320605"><li><a href="http://dx.doi.org/10.1261/rna.2320605" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Effects+of+nucleotide+substitution+and+modification+on+the+stability+and+structure+of+helix+69+from+28S+rRNA." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Effects+of+nucleotide+substitution+and+modification+on+the+stability+and+structure+of+helix+69+from+28S+rRNA.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0000174-Gustafsson1" id="pone.0000174-Gustafsson1"></a>Gustafsson C, Persson BC (1998) Identification of the rrmA gene encoding the 23S rRNA m1G745 methyltransferase in Escherichia coli and characterization of an m1G745-deficient mutant. J Bacteriol  180: 359–365.  <ul class="find" data-citedArticleID="927614"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Identification+of+the+rrmA+gene+encoding+the+23S+rRNA+m1G745+methyltransferase+in+Escherichia+coli+and+characterization+of+an+m1G745-deficient+mutant.&amp;auth=&amp;atitle=Identification+of+the+rrmA+gene+encoding+the+23S+rRNA+m1G745+methyltransferase+in+Escherichia+coli+and+characterization+of+an+m1G745-deficient+mutant." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Identification+of+the+rrmA+gene+encoding+the+23S+rRNA+m1G745+methyltransferase+in+Escherichia+coli+and+characterization+of+an+m1G745-deficient+mutant." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Identification+of+the+rrmA+gene+encoding+the+23S+rRNA+m1G745+methyltransferase+in+Escherichia+coli+and+characterization+of+an+m1G745-deficient+mutant.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0000174-King1" id="pone.0000174-King1"></a>King TH, Liu B, McCully RR, Fournier MJ (2003) Ribosome structure and activity are altered in cells lacking snoRNPs that form pseudouridines in the peptidyl transferase center. Molecular Cell  11: 425–435.  <ul class="find" data-citedArticleID="927638" data-doi="10.1016/s1097-2765(03)00040-6"><li><a href="http://dx.doi.org/10.1016/s1097-2765(03)00040-6" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Ribosome+structure+and+activity+are+altered+in+cells+lacking+snoRNPs+that+form+pseudouridines+in+the+peptidyl+transferase+center." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Ribosome+structure+and+activity+are+altered+in+cells+lacking+snoRNPs+that+form+pseudouridines+in+the+peptidyl+transferase+center.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0000174-Bonnerot1" id="pone.0000174-Bonnerot1"></a>Bonnerot C, Pintard L, Lutfalla G (2003) Functional redundancy of Spb1p and a snR52-dependent mechanism for the 2′-O-ribose methylation of a conserved rRNA position in yeast. Molecular Cell  12: 1309–1315.  <ul class="find" data-citedArticleID="927594" data-doi="10.1016/s1097-2765(03)00435-0"><li><a href="http://dx.doi.org/10.1016/s1097-2765(03)00435-0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Functional+redundancy+of+Spb1p+and+a+snR52-dependent+mechanism+for+the+2%26%23x2032%3B-O-ribose+methylation+of+a+conserved+rRNA+position+in+yeast." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Functional+redundancy+of+Spb1p+and+a+snR52-dependent+mechanism+for+the+2%26%23x2032%3B-O-ribose+methylation+of+a+conserved+rRNA+position+in+yeast.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0000174-Lapeyre1" id="pone.0000174-Lapeyre1"></a>Lapeyre B, Purushothaman SK (2004) Spb1p-directed formation of Gm2922 in the ribosome catalytic center occurs at a late processing stage. Mol Cell  16: 663–669.  <ul class="find" data-citedArticleID="927642" data-doi="10.1016/j.molcel.2004.10.022"><li><a href="http://dx.doi.org/10.1016/j.molcel.2004.10.022" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Spb1p-directed+formation+of+Gm2922+in+the+ribosome+catalytic+center+occurs+at+a+late+processing+stage." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Spb1p-directed+formation+of+Gm2922+in+the+ribosome+catalytic+center+occurs+at+a+late+processing+stage.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0000174-Caldas1" id="pone.0000174-Caldas1"></a>Caldas T, Binet E, Bouloc P, Costa A, Desgres J, et al.  (2000) The FtsJ/RrmJ heat shock protein of Escherichia coli is a 23 S ribosomal RNA methyltransferase. J Biol Chem  275: 16414–16419.  <ul class="find" data-citedArticleID="927598" data-doi="10.1074/jbc.m001854200"><li><a href="http://dx.doi.org/10.1074/jbc.m001854200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+FtsJ%2FRrmJ+heat+shock+protein+of+Escherichia+coli+is+a+23+S+ribosomal+RNA+methyltransferase." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+FtsJ%2FRrmJ+heat+shock+protein+of+Escherichia+coli+is+a+23+S+ribosomal+RNA+methyltransferase.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0000174-Bugl1" id="pone.0000174-Bugl1"></a>Bugl H, Fauman EB, Staker BL, Zheng F, Kushner SR, et al.  (2000) RNA methylation under heat shock control. Mol Cell  6: 349–360.  <ul class="find" data-citedArticleID="927596" data-doi="10.1016/s1097-2765(00)00035-6"><li><a href="http://dx.doi.org/10.1016/s1097-2765(00)00035-6" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=RNA+methylation+under+heat+shock+control." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22RNA+methylation+under+heat+shock+control.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0000174-Hansen1" id="pone.0000174-Hansen1"></a>Hansen TM, Baranov PV, Ivanov IP, Gesteland RF, Atkins JF (2003) Maintenance of the correct open reading frame by the ribosome. Embo Reports  4: 499–504.  <ul class="find" data-citedArticleID="927616" data-doi="10.1038/sj.embor.embor825"><li><a href="http://dx.doi.org/10.1038/sj.embor.embor825" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Maintenance+of+the+correct+open+reading+frame+by+the+ribosome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Maintenance+of+the+correct+open+reading+frame+by+the+ribosome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0000174-Carrasco1" id="pone.0000174-Carrasco1"></a>Carrasco L, Barbacid M, Vazquez D (1973) The tricodermin group of antibiotics, inhibitors of peptide bond formation by eukaryotic ribosomes. Biochim et Biophys Acta  312: 368–376.  <ul class="find" data-citedArticleID="927600" data-doi="10.1016/0005-2787(73)90381-x"><li><a href="http://dx.doi.org/10.1016/0005-2787(73)90381-x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+tricodermin+group+of+antibiotics%2C+inhibitors+of+peptide+bond+formation+by+eukaryotic+ribosomes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+tricodermin+group+of+antibiotics%2C+inhibitors+of+peptide+bond+formation+by+eukaryotic+ribosomes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0000174-Grollman1" id="pone.0000174-Grollman1"></a>Grollman AP (1967) Inhibitors of protein biosynthesis. II. Mode of action of anisomycin. J Biol Chem  242: 3226–3233.  <ul class="find" data-citedArticleID="927612"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Inhibitors+of+protein+biosynthesis.+II.+Mode+of+action+of+anisomycin.&amp;auth=&amp;atitle=Inhibitors+of+protein+biosynthesis.+II.+Mode+of+action+of+anisomycin." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Inhibitors+of+protein+biosynthesis.+II.+Mode+of+action+of+anisomycin." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Inhibitors+of+protein+biosynthesis.+II.+Mode+of+action+of+anisomycin.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0000174-Schindler1" id="pone.0000174-Schindler1"></a>Schindler D (1974) Two classes of inhibitors of peptidyltransferase activity in eukaryotes. Nature  249: 38–41.  <ul class="find" data-citedArticleID="927670" data-doi="10.1038/249038a0"><li><a href="http://dx.doi.org/10.1038/249038a0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Two+classes+of+inhibitors+of+peptidyltransferase+activity+in+eukaryotes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Two+classes+of+inhibitors+of+peptidyltransferase+activity+in+eukaryotes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0000174-Hansen2" id="pone.0000174-Hansen2"></a>Hansen JL, Moore PB, Steitz TA (2003) Structures of five antibiotics bound at the peptidyl transferase center of the large ribosomal subunit. J Mol Biol  330: 1061–1075.  <ul class="find" data-citedArticleID="927618" data-doi="10.1016/s0022-2836(03)00668-5"><li><a href="http://dx.doi.org/10.1016/s0022-2836(03)00668-5" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Structures+of+five+antibiotics+bound+at+the+peptidyl+transferase+center+of+the+large+ribosomal+subunit." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Structures+of+five+antibiotics+bound+at+the+peptidyl+transferase+center+of+the+large+ribosomal+subunit.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0000174-Herner1" id="pone.0000174-Herner1"></a>Herner AE, Goldberg IH, Cohen LB (1969) Stabilization of N-acetylphenylalanyl transfer ribonucleic acid binding to ribosomes by sparsomycin. Biochemistry  8: 1335–1344.  <ul class="find" data-citedArticleID="927624" data-doi="10.1021/bi00832a006"><li><a href="http://dx.doi.org/10.1021/bi00832a006" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Stabilization+of+N-acetylphenylalanyl+transfer+ribonucleic+acid+binding+to+ribosomes+by+sparsomycin." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Stabilization+of+N-acetylphenylalanyl+transfer+ribonucleic+acid+binding+to+ribosomes+by+sparsomycin.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0000174-Jayaraman1" id="pone.0000174-Jayaraman1"></a>Jayaraman J, Goldberg IH (1968) Localization of sparsomycin action to the peptide-bond-forming step. Biochemistry  7: 418–421.  <ul class="find" data-citedArticleID="927634" data-doi="10.1021/bi00841a053"><li><a href="http://dx.doi.org/10.1021/bi00841a053" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Localization+of+sparsomycin+action+to+the+peptide-bond-forming+step." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Localization+of+sparsomycin+action+to+the+peptide-bond-forming+step.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0000174-Moazed1" id="pone.0000174-Moazed1"></a>Moazed D, Noller HF (1991) Sites of interaction of the CCA end of peptidyl-tRNA with 23S rRNA. Proc Natl Acad Sci U S A  88: 3725–3728.  <ul class="find" data-citedArticleID="927652" data-doi="10.1073/pnas.88.9.3725"><li><a href="http://dx.doi.org/10.1073/pnas.88.9.3725" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Sites+of+interaction+of+the+CCA+end+of+peptidyl-tRNA+with+23S+rRNA." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Sites+of+interaction+of+the+CCA+end+of+peptidyl-tRNA+with+23S+rRNA.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0000174-Dinman1" id="pone.0000174-Dinman1"></a>Dinman JD, Wickner RB (1992) Ribosomal frameshifting efficiency and Gag/Gag-pol ratio are critical for yeast M<sub>1</sub> double-stranded RNA virus propagation. J Virology  66: 3669–3676.  <ul class="find" data-citedArticleID="927606"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Ribosomal+frameshifting+efficiency+and+Gag%2FGag-pol+ratio+are+critical+for+yeast+M1+double-stranded+RNA+virus+propagation.&amp;auth=&amp;atitle=Ribosomal+frameshifting+efficiency+and+Gag%2FGag-pol+ratio+are+critical+for+yeast+M1+double-stranded+RNA+virus+propagation." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Ribosomal+frameshifting+efficiency+and+Gag%2FGag-pol+ratio+are+critical+for+yeast+M1+double-stranded+RNA+virus+propagation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Ribosomal+frameshifting+efficiency+and+Gag%2FGag-pol+ratio+are+critical+for+yeast+M1+double-stranded+RNA+virus+propagation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0000174-Wickner1" id="pone.0000174-Wickner1"></a>Wickner RB (1996) Double-stranded RNA viruses of <em>Saccharomyces cerevisiae</em>. Microbiol Rev  60: 250–265.  <ul class="find" data-citedArticleID="927678" data-doi="10.1146/annurev.mi.46.100192.002023"><li><a href="http://dx.doi.org/10.1146/annurev.mi.46.100192.002023" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Double-stranded+RNA+viruses+of+Saccharomyces+cerevisiae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Double-stranded+RNA+viruses+of+Saccharomyces+cerevisiae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0000174-Harger1" id="pone.0000174-Harger1"></a>Harger JW, Dinman JD (2003) An in vivo dual-luciferase assay system for studying translational recoding in the yeast <em>Saccharomyces cerevisiae</em>. RNA  9: 1019–1024.  <ul class="find" data-citedArticleID="927620" data-doi="10.1261/rna.5930803"><li><a href="http://dx.doi.org/10.1261/rna.5930803" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=An+in+vivo+dual-luciferase+assay+system+for+studying+translational+recoding+in+the+yeast+Saccharomyces+cerevisiae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22An+in+vivo+dual-luciferase+assay+system+for+studying+translational+recoding+in+the+yeast+Saccharomyces+cerevisiae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0000174-Schuwirth1" id="pone.0000174-Schuwirth1"></a>Schuwirth BS, Borovinskaya MA, Hau CW, Zhang W, Vila-Sanjurjo A, et al.  (2005) Structures of the bacterial ribosome at 3.5 A resolution. Science  310: 827–834.  <ul class="find" data-citedArticleID="927672" data-doi="10.1126/science.1117230"><li><a href="http://dx.doi.org/10.1126/science.1117230" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Structures+of+the+bacterial+ribosome+at+3.5+A+resolution." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Structures+of+the+bacterial+ribosome+at+3.5+A+resolution.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0000174-Katunin1" id="pone.0000174-Katunin1"></a>Katunin VI, Muth GW, Strobel SA, Wintermeyer W, Rodnina MV (2002) Important contribution to catalysis of peptide bond formation by a single ionizing group within the ribosome. Mol Cell  10: 339–346.  <ul class="find" data-citedArticleID="927636" data-doi="10.1016/s1097-2765(02)00566-x"><li><a href="http://dx.doi.org/10.1016/s1097-2765(02)00566-x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Important+contribution+to+catalysis+of+peptide+bond+formation+by+a+single+ionizing+group+within+the+ribosome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Important+contribution+to+catalysis+of+peptide+bond+formation+by+a+single+ionizing+group+within+the+ribosome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">36.
              </span><a name="pone.0000174-Sanbonmatsu1" id="pone.0000174-Sanbonmatsu1"></a>Sanbonmatsu KY, Joseph S, Tung CS (2005) Simulating movement of tRNA into the ribosome during decoding. Proc Natl Acad Sci U S A  102: 15854–15859.  <ul class="find" data-citedArticleID="927668" data-doi="10.1073/pnas.0503456102"><li><a href="http://dx.doi.org/10.1073/pnas.0503456102" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Simulating+movement+of+tRNA+into+the+ribosome+during+decoding." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Simulating+movement+of+tRNA+into+the+ribosome+during+decoding.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">37.
              </span><a name="pone.0000174-Dinman2" id="pone.0000174-Dinman2"></a>Dinman JD, Wickner RB (1994) Translational maintenance of frame: mutants of <em>Saccharomyces cerevisiae</em> with altered −1 ribosomal frameshifting efficiencies. Genetics  136: 75–86.  <ul class="find" data-citedArticleID="927608"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Translational+maintenance+of+frame%3A+mutants+of+Saccharomyces+cerevisiae+with+altered+%26%23x2212%3B1+ribosomal+frameshifting+efficiencies.&amp;auth=&amp;atitle=Translational+maintenance+of+frame%3A+mutants+of+Saccharomyces+cerevisiae+with+altered+%26%23x2212%3B1+ribosomal+frameshifting+efficiencies." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Translational+maintenance+of+frame%3A+mutants+of+Saccharomyces+cerevisiae+with+altered+%26%23x2212%3B1+ribosomal+frameshifting+efficiencies." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Translational+maintenance+of+frame%3A+mutants+of+Saccharomyces+cerevisiae+with+altered+%26%23x2212%3B1+ribosomal+frameshifting+efficiencies.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">38.
              </span><a name="pone.0000174-Ohtake1" id="pone.0000174-Ohtake1"></a>Ohtake Y, Wickner RB (1995) Yeast virus propagation depends critically on free 60S ribosomal subunit concentration. Mol Cell Biol  15: 2772–2781.  <ul class="find" data-citedArticleID="927658"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Yeast+virus+propagation+depends+critically+on+free+60S+ribosomal+subunit+concentration.&amp;auth=&amp;atitle=Yeast+virus+propagation+depends+critically+on+free+60S+ribosomal+subunit+concentration." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Yeast+virus+propagation+depends+critically+on+free+60S+ribosomal+subunit+concentration." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Yeast+virus+propagation+depends+critically+on+free+60S+ribosomal+subunit+concentration.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">39.
              </span><a name="pone.0000174-Belcourt1" id="pone.0000174-Belcourt1"></a>Belcourt MF, Farabaugh PJ (1990) Ribosomal frameshifting in the yeast retrotransposon Ty: tRNAs induce slippage on a 7 nucleotide minimal site. Cell  62: 339–352.  <ul class="find" data-citedArticleID="927592" data-doi="10.1016/0092-8674(90)90371-k"><li><a href="http://dx.doi.org/10.1016/0092-8674(90)90371-k" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Ribosomal+frameshifting+in+the+yeast+retrotransposon+Ty%3A+tRNAs+induce+slippage+on+a+7+nucleotide+minimal+site." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Ribosomal+frameshifting+in+the+yeast+retrotransposon+Ty%3A+tRNAs+induce+slippage+on+a+7+nucleotide+minimal+site.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">40.
              </span><a name="pone.0000174-Hirabayashi1" id="pone.0000174-Hirabayashi1"></a>Hirabayashi N, Sato NS, Suzuki T (2006) Conserved loop sequence of helix 69 in Escherichia coli 23 S rRNA is involved in A-site tRNA binding and translational fidelity. J Biol Chem  281: 17203–17211.  <ul class="find" data-citedArticleID="927626" data-doi="10.1074/jbc.m511728200"><li><a href="http://dx.doi.org/10.1074/jbc.m511728200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Conserved+loop+sequence+of+helix+69+in+Escherichia+coli+23+S+rRNA+is+involved+in+A-site+tRNA+binding+and+translational+fidelity." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Conserved+loop+sequence+of+helix+69+in+Escherichia+coli+23+S+rRNA+is+involved+in+A-site+tRNA+binding+and+translational+fidelity.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">41.
              </span><a name="pone.0000174-Mochizuki1" id="pone.0000174-Mochizuki1"></a>Mochizuki Y, He J, Kulkarni S, Bessler M, Mason PJ (2004) Mouse dyskerin mutations affect accumulation of telomerase RNA and small nucleolar RNA, telomerase activity, and ribosomal RNA processing. Proc Natl Acad Sci U S A  101: 10756–10761.  <ul class="find" data-citedArticleID="927654" data-doi="10.1073/pnas.0402560101"><li><a href="http://dx.doi.org/10.1073/pnas.0402560101" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mouse+dyskerin+mutations+affect+accumulation+of+telomerase+RNA+and+small+nucleolar+RNA%2C+telomerase+activity%2C+and+ribosomal+RNA+processing." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mouse+dyskerin+mutations+affect+accumulation+of+telomerase+RNA+and+small+nucleolar+RNA%2C+telomerase+activity%2C+and+ribosomal+RNA+processing.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">42.
              </span><a name="pone.0000174-Yoon1" id="pone.0000174-Yoon1"></a>Yoon A, Peng G, Brandenburger Y, Zollo O, Xu W, et al.  (2006) Impaired control of IRES-mediated translation in X-linked dyskeratosis congenita. Science  312: 902–906.  <ul class="find" data-citedArticleID="927682" data-doi="10.1126/science.1123835"><li><a href="http://dx.doi.org/10.1126/science.1123835" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Impaired+control+of+IRES-mediated+translation+in+X-linked+dyskeratosis+congenita." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Impaired+control+of+IRES-mediated+translation+in+X-linked+dyskeratosis+congenita.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">43.
              </span><a name="pone.0000174-Inoue1" id="pone.0000174-Inoue1"></a>Inoue H, Nojima H, Okayama H (1990) High efficiency transformation of Escherichia coli with plasmids. Gene  96: 23–28.  <ul class="find" data-citedArticleID="927628" data-doi="10.1016/0378-1119(90)90336-p"><li><a href="http://dx.doi.org/10.1016/0378-1119(90)90336-p" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=High+efficiency+transformation+of+Escherichia+coli+with+plasmids." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22High+efficiency+transformation+of+Escherichia+coli+with+plasmids.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">44.
              </span><a name="pone.0000174-Ito1" id="pone.0000174-Ito1"></a>Ito H, Fukuda Y, Murata K, Kimura A (1983) Transformation of intact yeast cells treated with alkali cations. J Bacteriol  153: 163–168.  <ul class="find" data-citedArticleID="927630"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Transformation+of+intact+yeast+cells+treated+with+alkali+cations.&amp;auth=&amp;atitle=Transformation+of+intact+yeast+cells+treated+with+alkali+cations." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Transformation+of+intact+yeast+cells+treated+with+alkali+cations." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Transformation+of+intact+yeast+cells+treated+with+alkali+cations.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">45.
              </span><a name="pone.0000174-Jacobs1" id="pone.0000174-Jacobs1"></a>Jacobs JL, Dinman JD (2004) Systematic analysis of bicistronic reporter assay data. Nucleic Acids Res  32: e160–e170.  <ul class="find" data-citedArticleID="927632" data-doi="10.1093/nar/gnh157"><li><a href="http://dx.doi.org/10.1093/nar/gnh157" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Systematic+analysis+of+bicistronic+reporter+assay+data." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Systematic+analysis+of+bicistronic+reporter+assay+data.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">46.
              </span><a name="pone.0000174-Meskauskas1" id="pone.0000174-Meskauskas1"></a>Meskauskas A, Dinman JD (2001) Ribosomal protein L5 helps anchor peptidyl-tRNA to the P-site in Saccharomyces cerevisiae. RNA  7: 1084–1096.  <ul class="find" data-citedArticleID="927648" data-doi="10.1017/s1355838201001480"><li><a href="http://dx.doi.org/10.1017/s1355838201001480" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Ribosomal+protein+L5+helps+anchor+peptidyl-tRNA+to+the+P-site+in+Saccharomyces+cerevisiae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Ribosomal+protein+L5+helps+anchor+peptidyl-tRNA+to+the+P-site+in+Saccharomyces+cerevisiae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">47.
              </span><a name="pone.0000174-Meskauskas2" id="pone.0000174-Meskauskas2"></a>Meskauskas A, Petrov AN, Dinman JD (2005) Identification of functionally important amino acids of ribosomal protein L3 by saturation mutagenesis. Molecular &amp; Cellular Biology  25: 10863–10874.  <ul class="find" data-citedArticleID="927650" data-doi="10.1128/mcb.25.24.10863-10874.2005"><li><a href="http://dx.doi.org/10.1128/mcb.25.24.10863-10874.2005" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Identification+of+functionally+important+amino+acids+of+ribosomal+protein+L3+by+saturation+mutagenesis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Identification+of+functionally+important+amino+acids+of+ribosomal+protein+L3+by+saturation+mutagenesis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">48.
              </span><a name="pone.0000174-Kiparisov1" id="pone.0000174-Kiparisov1"></a>Kiparisov S, Petrov A, Meskauskas A, Sergiev PV, Dontsova OA, et al.  (2005) Structural and functional analysis of 5S rRNA. Molecular Genetics and Genomics  27: 235–247.  <ul class="find" data-citedArticleID="927640" data-doi="10.1007/s00438-005-0020-9"><li><a href="http://dx.doi.org/10.1007/s00438-005-0020-9" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Structural+and+functional+analysis+of+5S+rRNA." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Structural+and+functional+analysis+of+5S+rRNA.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">49.
              </span><a name="pone.0000174-Rakauskaite1" id="pone.0000174-Rakauskaite1"></a>Rakauskaite R, Dinman JD (2006) An arc of unpaired "hinge bases" facilitates information exchange among functional centers of the ribosome. Mol Cell Biol  26: 8992–9002.  <ul class="find" data-citedArticleID="927660"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=An+arc+of+unpaired+%22hinge+bases%22+facilitates+information+exchange+among+functional+centers+of+the+ribosome.&amp;auth=&amp;atitle=An+arc+of+unpaired+%22hinge+bases%22+facilitates+information+exchange+among+functional+centers+of+the+ribosome." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=An+arc+of+unpaired+%22hinge+bases%22+facilitates+information+exchange+among+functional+centers+of+the+ribosome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22An+arc+of+unpaired+%22hinge+bases%22+facilitates+information+exchange+among+functional+centers+of+the+ribosome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000174.XML" value="156273"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000174.PDF" value="378756"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000174.g001.PNG_L" value="1419001"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000174.g001.PNG_M" value="126235"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000174.g001.PNG_S" value="3451"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000174.g001.TIF" value="1655060"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000174.g001.PNG_I" value="42707"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000174.g002.PNG_L" value="1361534"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000174.g002.PNG_M" value="149408"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000174.g002.PNG_S" value="4128"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000174.g002.TIF" value="1331222"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000174.g002.PNG_I" value="46702"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000174.t001.PNG_L" value="364680"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000174.t001.PNG_M" value="37588"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000174.t001.PNG_S" value="2752"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000174.t001.TIF" value="21221798"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000174.t001.PNG_I" value="15689"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000174.g003.PNG_L" value="608327"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000174.g003.PNG_M" value="84178"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000174.g003.PNG_S" value="3296"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000174.g003.TIF" value="597450"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000174.g003.PNG_I" value="29842"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000174.g004.PNG_L" value="74207"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000174.g004.PNG_M" value="22207"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000174.g004.PNG_S" value="2081"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000174.g004.TIF" value="120028"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000174.g004.PNG_I" value="13206"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000174.g005.PNG_L" value="6994351"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000174.g005.PNG_M" value="274521"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000174.g005.PNG_S" value="7581"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000174.g005.TIF" value="8877564"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000174.g005.PNG_I" value="97220"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000174.s001.DOC" value="87040"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000174.s002.DOC" value="44544"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000174.s003.DOC" value="53248"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000174.s004.DOC" value="30208"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000174&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000174" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000174&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0000174&volume=&issue=&title=Optimization of Ribosome Structure and Function by rRNA Base Modification&author_name=Jennifer%20L.%20Baxter-Roshek%2C%20Alexey%20N.%20Petrov%2C%20Jonathan%20D.%20Dinman&start_page=1&end_page=9" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000174" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000174" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0000174" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000174&amp;t=Optimization%20of%20Ribosome%20Structure%20and%20Function%20by%20rRNA%20Base%20Modification" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000174&title=Optimization%20of%20Ribosome%20Structure%20and%20Function%20by%20rRNA%20Base%20Modification&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000174&amp;title=Optimization%20of%20Ribosome%20Structure%20and%20Function%20by%20rRNA%20Base%20Modification" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000174" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0000174&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Optimization of Ribosome Structure and Function by rRNA Base Modification';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0000174';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Optimization%20of%20Ribosome%20Structure%20and%20Function%20by%20rRNA%20Base%20Modification http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000174" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0000174" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">










          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Codons%22" title="Search for articles in the subject area:'Codons'"><div class="flagText">Codons</div></a>
              <div data-categoryid="31811" data-articleid="23186"
                   data-categoryname="Codons"
                   class="flagImage" title="Flag 'Codons' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Methylation%22" title="Search for articles in the subject area:'Methylation'"><div class="flagText">Methylation</div></a>
              <div data-categoryid="18179" data-articleid="23186"
                   data-categoryname="Methylation"
                   class="flagImage" title="Flag 'Methylation' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Mutant+strains%22" title="Search for articles in the subject area:'Mutant strains'"><div class="flagText">Mutant strains</div></a>
              <div data-categoryid="33025" data-articleid="23186"
                   data-categoryname="Mutant strains"
                   class="flagImage" title="Flag 'Mutant strains' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Protein+translation%22" title="Search for articles in the subject area:'Protein translation'"><div class="flagText">Protein translation</div></a>
              <div data-categoryid="24691" data-articleid="23186"
                   data-categoryname="Protein translation"
                   class="flagImage" title="Flag 'Protein translation' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Ribosomal+RNA%22" title="Search for articles in the subject area:'Ribosomal RNA'"><div class="flagText">Ribosomal RNA</div></a>
              <div data-categoryid="21371" data-articleid="23186"
                   data-categoryname="Ribosomal RNA"
                   class="flagImage" title="Flag 'Ribosomal RNA' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Ribosomes%22" title="Search for articles in the subject area:'Ribosomes'"><div class="flagText">Ribosomes</div></a>
              <div data-categoryid="32243" data-articleid="23186"
                   data-categoryname="Ribosomes"
                   class="flagImage" title="Flag 'Ribosomes' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Transfer+RNA%22" title="Search for articles in the subject area:'Transfer RNA'"><div class="flagText">Transfer RNA</div></a>
              <div data-categoryid="21239" data-articleid="23186"
                   data-categoryname="Transfer RNA"
                   class="flagImage" title="Flag 'Transfer RNA' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Yeast%22" title="Search for articles in the subject area:'Yeast'"><div class="flagText">Yeast</div></a>
              <div data-categoryid="43039" data-articleid="23186"
                   data-categoryname="Yeast"
                   class="flagImage" title="Flag 'Yeast' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=7452'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=2973'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=3338&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>

<div class="block sidebar-comments">
    <div class="header">
        <h3>Comments</h3>
    </div>
      <p><a href="/annotation/listThread.action?root=6281">Or resistance?</a><br>Posted by ribonux</p>
      <p><a href="/annotation/listThread.action?root=3195">snoRNAs of yeast database</a><br>Posted by WayneDecatur</p>
      <p><a href="/annotation/listThread.action?root=3165">Clarification of terms.</a><br>Posted by dinman</p>
</div>

</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
