

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: The Effect of Central Loops in miRNA:MRE Duplexes on the Efficiency of miRNA-Mediated Gene Regulation</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0001719"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0001719" />

    <meta name="citation_title" content="The Effect of Central Loops in miRNA:MRE Duplexes on the Efficiency of miRNA-Mediated Gene Regulation"/>
    <meta itemprop="name" content="The Effect of Central Loops in miRNA:MRE Duplexes on the Efficiency of miRNA-Mediated Gene Regulation"/>

      <meta name="citation_author" content="Wenbin Ye"/>
            <meta name="citation_author_institution" content="Department of Biological Science and Biotechnology, Tsinghua University, Beijing, People's Republic of China"/>
            <meta name="citation_author_institution" content="Division of Life Science, Graduate School at Shenzhen, Tsinghua University, Shenzhen, People's Republic of China"/>
      <meta name="citation_author" content="Qing Lv"/>
            <meta name="citation_author_institution" content="Department of Biological Science and Biotechnology, Tsinghua University, Beijing, People's Republic of China"/>
            <meta name="citation_author_institution" content="Division of Life Science, Graduate School at Shenzhen, Tsinghua University, Shenzhen, People's Republic of China"/>
      <meta name="citation_author" content="Chung-Kwun Amy Wong"/>
            <meta name="citation_author_institution" content="Sunnybrook Research Institute, Sunnybrook Health Sciences Centre, Department of Laboratory Medicine and Pathology, University of Toronto, Toronto, Canada"/>
      <meta name="citation_author" content="Sean Hu"/>
            <meta name="citation_author_institution" content="Shenzhen Beike Biotechnology Co., Ltd., Shenzhen, People's Republic of China"/>
      <meta name="citation_author" content="Chao Fu"/>
            <meta name="citation_author_institution" content="Department of Biological Science and Biotechnology, Tsinghua University, Beijing, People's Republic of China"/>
            <meta name="citation_author_institution" content="Division of Life Science, Graduate School at Shenzhen, Tsinghua University, Shenzhen, People's Republic of China"/>
      <meta name="citation_author" content="Zhong Hua"/>
            <meta name="citation_author_institution" content="Division of Life Science, Graduate School at Shenzhen, Tsinghua University, Shenzhen, People's Republic of China"/>
      <meta name="citation_author" content="Guoping Cai"/>
            <meta name="citation_author_institution" content="Division of Life Science, Graduate School at Shenzhen, Tsinghua University, Shenzhen, People's Republic of China"/>
      <meta name="citation_author" content="Guoxi Li"/>
            <meta name="citation_author_institution" content="Shenzhen Beike Biotechnology Co., Ltd., Shenzhen, People's Republic of China"/>
      <meta name="citation_author" content="Burton B. Yang"/>
            <meta name="citation_author_institution" content="Sunnybrook Research Institute, Sunnybrook Health Sciences Centre, Department of Laboratory Medicine and Pathology, University of Toronto, Toronto, Canada"/>
      <meta name="citation_author" content="Yaou Zhang"/>
            <meta name="citation_author_institution" content="Division of Life Science, Graduate School at Shenzhen, Tsinghua University, Shenzhen, People's Republic of China"/>

    <meta name="citation_date" content="2008/3/5"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0001719.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e1719"/>
    <meta name="citation_issue" content="3"/>
    <meta name="citation_volume" content="3"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Specificity of microRNA target selection in translational repression.; citation_author=JG Doench; citation_author=PA Sharp; citation_journal_title=Genes Dev; citation_volume=18; citation_number=1; citation_pages=504-511; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Brain expressed microRNAs implicated in schizophrenia etiology.; citation_author=T Hansen; citation_author=L Olsen; citation_author=M Lindow; citation_author=KD Jakobsen; citation_author=H Ullum; citation_journal_title=PLoS ONE; citation_volume=2; citation_number=2; citation_pages=e873; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Two genetic circuits repress the Caenorhabditis elegans heterochronic gene lin-28 after translation initiation.; citation_author=K Seggerson; citation_author=L Tang; citation_author=EG Moss; citation_journal_title=Dev Biol; citation_volume=243; citation_number=3; citation_pages=215-225; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Both natural and designed micro RNAs can inhibit the expression of cognate mRNAs when expressed in human cells.; citation_author=Y Zeng; citation_author=EJ Wagner; citation_author=BR Cullen; citation_journal_title=Mol Cell; citation_volume=9; citation_number=4; citation_pages=1327-1333; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=MicroRNA-directed cleavage of HOXB8 mRNA.; citation_author=S Yekta; citation_author=IH Shih; citation_author=DP Bartel; citation_journal_title=Science; citation_volume=304; citation_number=5; citation_pages=594-596; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=A microRNA in a multiple-turnover RNAi enzyme complex.; citation_author=G Hutvagner; citation_author=PD Zamore; citation_journal_title=Science; citation_volume=297; citation_number=6; citation_pages=2056-2060; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Substrate requirements for let-7 function in the developing zebrafish embryo.; citation_author=WP Kloosterman; citation_author=E Wienholds; citation_author=RF Ketting; citation_author=RH Plasterk; citation_journal_title=Nucleic Acids Res; citation_volume=32; citation_number=7; citation_pages=6284-6291; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=miR-150, a microRNA expressed in mature B and T cells, blocks early B cell development when expressed prematurely.; citation_author=B Zhou; citation_author=S Wang; citation_author=C Mayr; citation_author=DP Bartel; citation_author=HF Lodish; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=104; citation_number=8; citation_pages=7080-7085; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=MicroRNA-34b and MicroRNA-34c are targets of p53 and cooperate in control of cell proliferation and adhesion-independent growth.; citation_author=DC Corney; citation_author=A Flesken-Nikitin; citation_author=AK Godwin; citation_author=W Wang; citation_author=AY Nikitin; citation_journal_title=Cancer Res; citation_volume=67; citation_number=9; citation_pages=8433-8438; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=The role of microRNA-1 and microRNA-133 in skeletal muscle proliferation and differentiation.; citation_author=JF Chen; citation_author=EM Mandel; citation_author=JM Thomson; citation_author=Q Wu; citation_author=TE Callis; citation_journal_title=Nat Genet; citation_volume=38; citation_number=10; citation_pages=228-233; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=The let-7 microRNA represses cell proliferation pathways in human cells.; citation_author=CD Johnson; citation_author=A Esquela-Kerscher; citation_author=G Stefani; citation_author=M Byrom; citation_author=K Kelnar; citation_journal_title=Cancer Res; citation_volume=67; citation_number=11; citation_pages=7713-7722; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=The microRNA miR-181 targets the homeobox protein Hox-A11 during mammalian myoblast differentiation.; citation_author=I Naguibneva; citation_author=M Ameyar-Zazoua; citation_author=A Polesskaya; citation_author=S Ait-Si-Ali; citation_author=R Groisman; citation_journal_title=Nat Cell Biol; citation_volume=8; citation_number=12; citation_pages=278-284; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=A microRNA mediates EGF receptor signaling and promotes photoreceptor differentiation in the Drosophila eye.; citation_author=X Li; citation_author=RW Carthew; citation_journal_title=Cell; citation_volume=123; citation_number=13; citation_pages=1267-1277; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=MicroRNA control of Nodal signalling.; citation_author=G Martello; citation_author=L Zacchigna; citation_author=M Inui; citation_author=M Montagner; citation_author=M Adorno; citation_journal_title=Nature; citation_volume=449; citation_number=14; citation_pages=183-188; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=A MicroRNA Feedback Circuit in Midbrain Dopamine Neurons.; citation_author=J Kim; citation_author=K Inoue; citation_author=J Ishii; citation_author=WB Vanti; citation_author=SV Voronov; citation_journal_title=Science; citation_volume=317; citation_number=15; citation_pages=1220-1224; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=miRNA Profiling of Naive, Effector and Memory CD8 T Cells.; citation_author=H Wu; citation_author=JR Neilson; citation_author=P Kumar; citation_author=M Manocha; citation_author=P Shankar; citation_journal_title=PLoS ONE; citation_volume=2; citation_number=16; citation_pages=e1020; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=MicroRNA-21 is an antiapoptotic factor in human glioblastoma cells.; citation_author=JA Chan; citation_author=AM Krichevsky; citation_author=KS Kosik; citation_journal_title=Cancer Res; citation_volume=65; citation_number=17; citation_pages=6029-6033; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Differential patterns of microRNA expression in neuroblastoma are correlated with prognosis, differentiation, and apoptosis.; citation_author=Y Chen; citation_author=RL Stallings; citation_journal_title=Cancer Res; citation_volume=67; citation_number=18; citation_pages=976-983; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=bantam encodes a developmentally regulated microRNA that controls cell proliferation and regulates the proapoptotic gene hid in Drosophila.; citation_author=J Brennecke; citation_author=DR Hipfner; citation_author=A Stark; citation_author=RB Russell; citation_author=SM Cohen; citation_journal_title=Cell; citation_volume=113; citation_number=19; citation_pages=25-36; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=c-Myc-regulated microRNAs modulate E2F1 expression.; citation_author=KA O'Donnell; citation_author=EA Wentzel; citation_author=KI Zeller; citation_author=CV Dang; citation_author=JT Mendell; citation_journal_title=Nature; citation_volume=435; citation_number=20; citation_pages=839-843; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=The let-7 MicroRNA family members mir-48, mir-84, and mir-241 function together to regulate developmental timing in Caenorhabditis elegans.; citation_author=AL Abbott; citation_author=E Alvarez-Saavedra; citation_author=EA Miska; citation_author=NC Lau; citation_author=DP Bartel; citation_journal_title=Dev Cell; citation_volume=9; citation_number=21; citation_pages=403-414; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=MicroRNA biogenesis and cancer.; citation_author=RI Gregory; citation_author=R Shiekhattar; citation_journal_title=Cancer Res; citation_volume=65; citation_number=22; citation_pages=3509-3512; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Orthologous MicroRNA Genes Are Located in Cancer-Associated Genomic Regions in Human and Mouse.; citation_author=IV Makunin; citation_author=M Pheasant; citation_author=C Simons; citation_author=JS Mattick; citation_journal_title=PLoS ONE; citation_volume=2; citation_number=23; citation_pages=e1133; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=A microRNA expression signature of human solid tumors defines cancer gene targets.; citation_author=S Volinia; citation_author=GA Calin; citation_author=CG Liu; citation_author=S Ambs; citation_author=A Cimmino; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=103; citation_number=24; citation_pages=2257-2261; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=MicroRNA-cancer connection: the beginning of a new tale.; citation_author=GA Calin; citation_author=CM Croce; citation_journal_title=Cancer Res; citation_volume=66; citation_number=25; citation_pages=7390-7394; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=MicroRNA-378 promotes cell survival, tumor growth, and angiogenesis by targeting SuFu and Fus-1 expression.; citation_author=DY Lee; citation_author=Z Deng; citation_author=CH Wang; citation_author=BB Yang; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=104; citation_number=26; citation_pages=20350-20355; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=MicroRNAs: novel regulators involved in the pathogenesis of Psoriasis?; citation_author=E Sonkoly; citation_author=T Wei; citation_author=PC Janson; citation_author=A Saaf; citation_author=L Lundeberg; citation_journal_title=PLoS ONE; citation_volume=2; citation_number=27; citation_pages=e610; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Mammalian MicroRNA Prediction through a Support Vector Machine Model of Sequence and Structure.; citation_author=Y Sheng; citation_author=PG Engstrom; citation_author=B Lenhard; citation_journal_title=PLoS ONE; citation_volume=2; citation_number=28; citation_pages=e946; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=RAS is regulated by the let-7 microRNA family.; citation_author=SM Johnson; citation_author=H Grosshans; citation_author=J Shingara; citation_author=M Byrom; citation_author=R Jarvis; citation_journal_title=Cell; citation_volume=120; citation_number=29; citation_pages=635-647; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=The cold shock domain protein LIN-28 controls developmental timing in C. elegans and is regulated by the lin-4 RNA.; citation_author=EG Moss; citation_author=RC Lee; citation_author=V Ambros; citation_journal_title=Cell; citation_volume=88; citation_number=30; citation_pages=637-646; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Prediction of mammalian microRNA targets.; citation_author=BP Lewis; citation_author=IH Shih; citation_author=MW Jones-Rhoades; citation_author=DP Bartel; citation_author=CB Burge; citation_journal_title=Cell; citation_volume=115; citation_number=31; citation_pages=787-798; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=A combined computational-experimental approach predicts human microRNA targets.; citation_author=M Kiriakidou; citation_author=PT Nelson; citation_author=A Kouranov; citation_author=P Fitziev; citation_author=C Bouyioukos; citation_journal_title=Genes Dev; citation_volume=18; citation_number=32; citation_pages=1165-1178; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Computational identification of plant microRNAs and their targets, including a stress-induced miRNA.; citation_author=MW Jones-Rhoades; citation_author=DP Bartel; citation_journal_title=Mol Cell; citation_volume=14; citation_number=33; citation_pages=787-799; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=MicroRNA targets in Drosophila.; citation_author=AJ Enright; citation_author=B John; citation_author=U Gaul; citation_author=T Tuschl; citation_author=C Sander; citation_journal_title=Genome Biol; citation_volume=5; citation_number=34; citation_pages=R1; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Fast and effective prediction of microRNA/target duplexes.; citation_author=M Rehmsmeier; citation_author=P Steffen; citation_author=M Hochsmann; citation_author=R Giegerich; citation_journal_title=Rna; citation_volume=10; citation_number=35; citation_pages=1507-1517; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Identification of Drosophila MicroRNA targets.; citation_author=A Stark; citation_author=J Brennecke; citation_author=RB Russell; citation_author=SM Cohen; citation_journal_title=PLoS Biol; citation_volume=1; citation_number=36; citation_pages=E60; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Human MicroRNA targets.; citation_author=B John; citation_author=AJ Enright; citation_author=A Aravin; citation_author=T Tuschl; citation_author=C Sander; citation_journal_title=PLoS Biol; citation_volume=2; citation_number=37; citation_pages=e363; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Identification of Novel Drosophila melanogaster MicroRNAs.; citation_author=T Sandmann; citation_author=SM Cohen; citation_journal_title=PLoS ONE; citation_volume=2; citation_number=38; citation_pages=e1265; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Architecture of a validated microRNA::target interaction.; citation_author=MC Vella; citation_author=K Reinert; citation_author=FJ Slack; citation_journal_title=Chem Biol; citation_volume=11; citation_number=39; citation_pages=1619-1623; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=MiRNA-directed regulation of VEGF and other angiogenic factors under hypoxia.; citation_author=Z Hua; citation_author=Q Lv; citation_author=W Ye; citation_author=CK Wong; citation_author=G Cai; citation_journal_title=PLoS ONE; citation_volume=1; citation_number=40; citation_pages=e116; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=The biology of VEGF and its receptors.; citation_author=N Ferrara; citation_author=HP Gerber; citation_author=J LeCouter; citation_journal_title=Nat Med; citation_volume=9; citation_number=41; citation_pages=669-676; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Perfect seed pairing is not a generally reliable predictor for miRNA-target interactions.; citation_author=D Didiano; citation_author=O Hobert; citation_journal_title=Nat Struct Mol Biol; citation_volume=13; citation_number=42; citation_pages=849-851; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=A pancreatic islet-specific microRNA regulates insulin secretion.; citation_author=MN Poy; citation_author=L Eliasson; citation_author=J Krutzfeldt; citation_author=S Kuwajima; citation_author=X Ma; citation_journal_title=Nature; citation_volume=432; citation_number=43; citation_pages=226-230; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Serum response factor regulates a muscle-specific microRNA that targets Hand2 during cardiogenesis.; citation_author=Y Zhao; citation_author=E Samal; citation_author=D Srivastava; citation_journal_title=Nature; citation_volume=436; citation_number=44; citation_pages=214-220; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Combinatorial microRNA target predictions.; citation_author=A Krek; citation_author=D Grun; citation_author=MN Poy; citation_author=R Wolf; citation_author=L Rosenberg; citation_journal_title=Nat Genet; citation_volume=37; citation_number=45; citation_pages=495-500; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=miR-15 and miR-16 induce apoptosis by targeting BCL2.; citation_author=A Cimmino; citation_author=GA Calin; citation_author=M Fabbri; citation_author=MV Iorio; citation_author=M Ferracin; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=102; citation_number=46; citation_pages=13944-13949; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=A genetic screen implicates miRNA-372 and miRNA-373 as oncogenes in testicular germ cell tumors.; citation_author=PM Voorhoeve; citation_author=C le Sage; citation_author=M Schrier; citation_author=AJ Gillis; citation_author=H Stoop; citation_journal_title=Cell; citation_volume=124; citation_number=47; citation_pages=1169-1181; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Microrna-155 regulates human angiotensin II type 1 receptor expression in fibroblasts.; citation_author=MM Martin; citation_author=EJ Lee; citation_author=JA Buckenberger; citation_author=TD Schmittgen; citation_author=TS Elton; citation_journal_title=J Biol Chem; citation_number=48; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=MicroRNA fingerprints during human megakaryocytopoiesis.; citation_author=R Garzon; citation_author=F Pichiorri; citation_author=T Palumbo; citation_author=R Iuliano; citation_author=A Cimmino; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=103; citation_number=49; citation_pages=5078-5083; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Vienna RNA secondary structure server.; citation_author=IL Hofacker; citation_journal_title=Nucleic Acids Res; citation_volume=31; citation_number=50; citation_pages=3429-3431; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Partition function and base pairing probabilities of RNA heterodimers.; citation_author=SH Bernhart; citation_author=H Tafer; citation_author=U Muckstein; citation_author=C Flamm; citation_author=PF Stadler; citation_journal_title=Algorithms Mol Biol; citation_volume=1; citation_number=51; citation_pages=3; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Sequence variants in SLITRK1 are associated with Tourette's syndrome.; citation_author=JF Abelson; citation_author=KY Kwan; citation_author=BJ O'Roak; citation_author=DY Baek; citation_author=AA Stillman; citation_journal_title=Science; citation_volume=310; citation_number=52; citation_pages=317-320; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Conserved seed pairing, often flanked by adenosines, indicates that thousands of human genes are microRNA targets.; citation_author=BP Lewis; citation_author=CB Burge; citation_author=DP Bartel; citation_journal_title=Cell; citation_volume=120; citation_number=53; citation_pages=15-20; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Principles of microRNA-target recognition.; citation_author=J Brennecke; citation_author=A Stark; citation_author=RB Russell; citation_author=SM Cohen; citation_journal_title=PLoS Biol; citation_volume=3; citation_number=54; citation_pages=e85; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Microarray analysis shows that some microRNAs downregulate large numbers of target mRNAs.; citation_author=LP Lim; citation_author=NC Lau; citation_author=P Garrett-Engele; citation_author=A Grimson; citation_author=JM Schelter; citation_journal_title=Nature; citation_volume=433; citation_number=55; citation_pages=769-773; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=MicroRNA targeting specificity in mammals: determinants beyond seed pairing.; citation_author=A Grimson; citation_author=KK Farh; citation_author=WK Johnston; citation_author=P Garrett-Engele; citation_author=LP Lim; citation_journal_title=Mol Cell; citation_volume=27; citation_number=56; citation_pages=91-105; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Potent effect of target structure on microRNA function.; citation_author=D Long; citation_author=R Lee; citation_author=P Williams; citation_author=CY Chan; citation_author=V Ambros; citation_journal_title=Nat Struct Mol Biol; citation_volume=14; citation_number=57; citation_pages=287-294; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=The role of site accessibility in microRNA target recognition.; citation_author=M Kertesz; citation_author=N Iovino; citation_author=U Unnerstall; citation_author=U Gaul; citation_author=E Segal; citation_journal_title=Nat Genet; citation_volume=39; citation_number=58; citation_pages=1278-1284; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=MicroRNAs: genomics, biogenesis, mechanism, and function.; citation_author=DP Bartel; citation_journal_title=Cell; citation_volume=116; citation_number=59; citation_pages=281-297; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=RNAi: the nuts and bolts of the RISC machine.; citation_author=W Filipowicz; citation_journal_title=Cell; citation_volume=122; citation_number=60; citation_pages=17-20; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=miRNP:mRNA association in polyribosomes in a human neuronal cell line.; citation_author=PT Nelson; citation_author=AG Hatzigeorgiou; citation_author=Z Mourelatos; citation_journal_title=Rna; citation_volume=10; citation_number=61; citation_pages=387-394; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Human let-7a miRNA blocks protein production on actively translating polyribosomes.; citation_author=S Nottrott; citation_author=MJ Simard; citation_author=JD Richter; citation_journal_title=Nat Struct Mol Biol; citation_volume=13; citation_number=62; citation_pages=1108-1114; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Evidence that microRNAs are associated with translating messenger RNAs in human cells.; citation_author=PA Maroney; citation_author=Y Yu; citation_author=J Fisher; citation_author=TW Nilsen; citation_journal_title=Nat Struct Mol Biol; citation_volume=13; citation_number=63; citation_pages=1102-1107; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=The microRNA Registry.; citation_author=S Griffiths-Jones; citation_journal_title=Nucleic Acids Res; citation_volume=32; citation_number=64; citation_pages=D109-111; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=An improved algorithm for matching biological sequences.; citation_author=O Gotoh; citation_journal_title=J Mol Biol; citation_volume=162; citation_number=65; citation_pages=705-708; citation_date=1982; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001719" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="The Effect of Central Loops in miRNA:MRE Duplexes on the Efficiency of miRNA-Mediated Gene Regulation"/>
    <meta name="twitter:description" content="MicroRNAs (miRNAs) guide posttranscriptional repression of mRNAs. Hundreds of miRNAs have been identified but the target identification of mammalian mRNAs is still a difficult task due to a poor understanding of the interaction between miRNAs and the miRNA recognizing element (MRE). In recent research, the importance of the 5&prime; end of the miRNA:MRE duplex has been emphasized and the effect of the tail region addressed, but the role of the central loop has largely remained unexplored. Here we examined the effect of the loop region in miRNA:MRE duplexes and found that the location of the central loop is one of the important factors affecting the efficiency of gene regulation mediated by miRNAs. It was further determined that the addition of a loop score combining both location and size as a new criterion for predicting MREs and their cognate miRNAs significantly decreased the false positive rates and increased the specificity of MRE prediction."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0001719.g005"/>

  <meta property="og:title" content="The Effect of Central Loops in miRNA:MRE Duplexes on the Efficiency of miRNA-Mediated Gene Regulation" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001719" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=7248'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=4435'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=2676&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0001719">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001719" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001719&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    The Effect of Central Loops in miRNA:MRE Duplexes on the Efficiency of miRNA-Mediated Gene Regulation
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Wenbin Ye
              <span class="equal-contrib"
                    title="These authors contributed equally to this work">equal contributor</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">

                <p><span class="equal-contrib" title="These authors contributed equally to this work">equal contributor</span>
                Contributed equally to this work with: Wenbin Ye, Qing Lv</p>

              
              

                <p>Affiliations:
                  Department of Biological Science and Biotechnology, Tsinghua University, Beijing, People's Republic of China, 
                  Division of Life Science, Graduate School at Shenzhen, Tsinghua University, Shenzhen, People's Republic of China
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Qing Lv
              <span class="equal-contrib"
                    title="These authors contributed equally to this work">equal contributor</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">

                <p><span class="equal-contrib" title="These authors contributed equally to this work">equal contributor</span>
                Contributed equally to this work with: Wenbin Ye, Qing Lv</p>

              
              

                <p>Affiliations:
                  Department of Biological Science and Biotechnology, Tsinghua University, Beijing, People's Republic of China, 
                  Division of Life Science, Graduate School at Shenzhen, Tsinghua University, Shenzhen, People's Republic of China
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Chung-Kwun Amy Wong, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Sunnybrook Research Institute, Sunnybrook Health Sciences Centre, Department of Laboratory Medicine and Pathology, University of Toronto, Toronto, Canada
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Sean Hu, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Shenzhen Beike Biotechnology Co., Ltd., Shenzhen, People's Republic of China
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Chao Fu, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliations:
                  Department of Biological Science and Biotechnology, Tsinghua University, Beijing, People's Republic of China, 
                  Division of Life Science, Graduate School at Shenzhen, Tsinghua University, Shenzhen, People's Republic of China
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Zhong Hua, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Division of Life Science, Graduate School at Shenzhen, Tsinghua University, Shenzhen, People's Republic of China
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Guoping Cai, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Division of Life Science, Graduate School at Shenzhen, Tsinghua University, Shenzhen, People's Republic of China
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Guoxi Li, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Shenzhen Beike Biotechnology Co., Ltd., Shenzhen, People's Republic of China
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Burton B. Yang
              <span class="corresponding">mail</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:byang@sri.utoronto.ca">byang@sri.utoronto.ca</a> (BBY); <a href="mailto:zhangyo@sz.tsinghua.edu.cn">zhangyo@sz.tsinghua.edu.cn</a> (YZ)</p>

                <p>Affiliation:
                  Sunnybrook Research Institute, Sunnybrook Health Sciences Centre, Department of Laboratory Medicine and Pathology, University of Toronto, Toronto, Canada
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Yaou Zhang
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:byang@sri.utoronto.ca">byang@sri.utoronto.ca</a> (BBY); <a href="mailto:zhangyo@sz.tsinghua.edu.cn">zhangyo@sz.tsinghua.edu.cn</a> (YZ)</p>

                <p>Affiliation:
                  Division of Life Science, Graduate School at Shenzhen, Tsinghua University, Shenzhen, People's Republic of China
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: March 05, 2008</li>
    <li>DOI: 10.1371/journal.pone.0001719</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0001719" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0001719" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001719" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0001719" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0001719-g001" data-doi="info:doi/10.1371/journal.pone.0001719" data-uri="info:doi/10.1371/journal.pone.0001719.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001719.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001719-g002" data-doi="info:doi/10.1371/journal.pone.0001719" data-uri="info:doi/10.1371/journal.pone.0001719.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001719.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001719-g003" data-doi="info:doi/10.1371/journal.pone.0001719" data-uri="info:doi/10.1371/journal.pone.0001719.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001719.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001719-g004" data-doi="info:doi/10.1371/journal.pone.0001719" data-uri="info:doi/10.1371/journal.pone.0001719.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001719.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001719-g005" data-doi="info:doi/10.1371/journal.pone.0001719" data-uri="info:doi/10.1371/journal.pone.0001719.g005" title="Figure 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001719.g005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001719-t001" data-doi="info:doi/10.1371/journal.pone.0001719" data-uri="info:doi/10.1371/journal.pone.0001719.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001719.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001719-t002" data-doi="info:doi/10.1371/journal.pone.0001719" data-uri="info:doi/10.1371/journal.pone.0001719.t002" title="Table 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001719.t002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001719-t003" data-doi="info:doi/10.1371/journal.pone.0001719" data-uri="info:doi/10.1371/journal.pone.0001719.t003" title="Table 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001719.t003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001719-t004" data-doi="info:doi/10.1371/journal.pone.0001719" data-uri="info:doi/10.1371/journal.pone.0001719.t004" title="Table 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001719.t004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001719">Reader Comments (1)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0001719" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1"></a><p>MicroRNAs (miRNAs) guide posttranscriptional repression of mRNAs. Hundreds of miRNAs have been identified but the target identification of mammalian mRNAs is still a difficult task due to a poor understanding of the interaction between miRNAs and the miRNA recognizing element (MRE). In recent research, the importance of the 5′ end of the miRNA:MRE duplex has been emphasized and the effect of the tail region addressed, but the role of the central loop has largely remained unexplored. Here we examined the effect of the loop region in miRNA:MRE duplexes and found that the location of the central loop is one of the important factors affecting the efficiency of gene regulation mediated by miRNAs. It was further determined that the addition of a loop score combining both location and size as a new criterion for predicting MREs and their cognate miRNAs significantly decreased the false positive rates and increased the specificity of MRE prediction.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>Ye W, Lv Q, Wong C-KA, Hu S, Fu C, et al.  (2008) The Effect of Central Loops in miRNA:MRE Duplexes on the Efficiency of miRNA-Mediated Gene Regulation. PLoS ONE 3(3):
          e1719.
            doi:10.1371/journal.pone.0001719</p><p><strong>Academic Editor: </strong>Mark Isalan, Center for Genomic Regulation, Spain</p><p><strong>Received:</strong> December 5, 2007; <strong>Accepted:</strong> January 28, 2008; <strong>Published:</strong> March 5, 2008</p><p><strong>Copyright:</strong> © 2008 Ye et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>This work was supported by a China-Canada Joint Health Research Grant (from Canadian Institutes of Health Research and the National Natural Science Foundation of China to Burton B. Yang and Yaou Zhang), Grant MOP-62729 from Canadian Institutes of Health Research to Burton B. Yang, Grant 30611120519 from the National Natural Science Foundation of China to Yaou Zhang, and Grant 387 2005-k2-046 from the Shenzhen Bureau of Science Technology &amp; Information to Yaou Zhang. Burton B. Yang is the recipient of a Career Investigator Award (CI 5958) from the Heart and Stroke Foundation of Ontario.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>MicroRNAs (miRNAs) are small non-coding RNAs which guide translational repression<a href="#pone.0001719-Doench1">[1]</a>–<a href="#pone.0001719-Zeng1">[4]</a> or RNA cleavage<a href="#pone.0001719-Yekta1">[5]</a>, <a href="#pone.0001719-Hutvagner1">[6]</a> by binding partially or with perfect complementarity with their target sites in the 3′-untranslated region (3′-UTR) of mRNAs. Having been studied for about a decade, miRNAs have been demonstrated to have important roles in development<a href="#pone.0001719-Kloosterman1">[7]</a>, <a href="#pone.0001719-Zhou1">[8]</a>, cell proliferation <a href="#pone.0001719-Corney1">[9]</a>–<a href="#pone.0001719-Johnson1">[11]</a>, differentiation <a href="#pone.0001719-Naguibneva1">[12]</a>–<a href="#pone.0001719-Wu1">[16]</a>, apoptosis<a href="#pone.0001719-Chan1">[17]</a>, <a href="#pone.0001719-Chen2">[18]</a>, cell cycle progression <a href="#pone.0001719-Brennecke1">[19]</a>–<a href="#pone.0001719-Abbott1">[21]</a>, tumorigenesis<a href="#pone.0001719-Gregory1">[22]</a>–<a href="#pone.0001719-Lee1">[26]</a> and many other physiological or pathological processes<a href="#pone.0001719-Hansen1">[2]</a>, <a href="#pone.0001719-Wu1">[16]</a>, <a href="#pone.0001719-Sonkoly1">[27]</a>. Hundreds of miRNAs have been identified by experimental or bioinformatics methods in animals, plants, and viruses, and the number of miRNAs are still increasing<a href="#pone.0001719-Sheng1">[28]</a>. However, the comprehension of miRNA function remains preliminary, because few miRNA targets have been identified in spite of the enormous quantity of identified miRNAs<a href="#pone.0001719-Yekta1">[5]</a>, <a href="#pone.0001719-Johnson2">[29]</a>, <a href="#pone.0001719-Moss1">[30]</a>. It therefore appears that progress in our understanding of miRNA function has been limited by the difficulty in accurate target prediction and validation.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>Identification of miRNA targets or the miRNA recognizing element (MRE) is the first step towards understanding the biological function of miRNAs. Since experiments used for the identification of miRNA targets are laborious and are hard to apply for large-scale investigations, many computational approaches have been recently developed to predict miRNA targets in vertebrates and <em>Drosophila</em><a href="#pone.0001719-Lewis1">[31]</a>–<a href="#pone.0001719-Sandmann1">[38]</a>. However, computational algorithms predicting miRNA targets still have high false-positive rates, because of the poor understanding of the interaction between miRNAs and their cognate mRNAs. Generally, miRNA:MRE duplexes consist of a 5′ end seed region, central bulges or loops (loop region or mismatches), and a 3′ end tail region. The importance of the seed region in miRNA-mediated gene regulation is highly emphasized by most researchers, while the effect of the tail region has been addressed computationally by Enright <em>et al.</em> and John <em>et al.</em>, and also experimentally by Vella <em>et al.</em>, as well as others <a href="#pone.0001719-Enright1">[34]</a>, <a href="#pone.0001719-John1">[37]</a>, <a href="#pone.0001719-Vella1">[39]</a>. However, the role of the loops in the middle region of the miRNA:MRE duplex is not very clear, though Kiriakidou and coworkers have reported that the size of the loop or bulge might affect the efficiency of miRNA-mediated gene regulation<a href="#pone.0001719-Kiriakidou1">[32]</a>.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>In this study, we used vascular endothelial growth factor (VEGF) as a model to examine the effect of the loop region in miRNA:MRE duplexes on miRNA-mediated gene regulation, and found that the location of the central loop is another one of the important factors in miRNA functioning. VEGF is an ideal model because the numbers of miRNA binding sites and the various types of miRNA:MRE structures were predicted in the VEGF 3′-UTR in our previous investigation<a href="#pone.0001719-Hua1">[40]</a>. The addition of loop scores combining the effects of both location and size as a new criterion for the prediction of MREs and their cognate miRNAs to <em>FindTar,</em> a computational algorithm developed in our lab, decreased the false positive rates and increased the accuracy of MRE prediction significantly.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3>
<h4>Putative miRNAs for VEGF regulation</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>VEGF is an important regulator of physiological or pathological angiogenesis and is involved in embryogenesis, wound healing, and tumorigenesis <a href="#pone.0001719-Ferrara1">[41]</a>. In this study, we used VEGF as a model to investigate the effect of the secondary structure of the non-seed region in the miRNA:MRE duplex on miRNA-mediated gene regulation. Previously, we developed an algorithm, <em>FindTar</em> version 1.0, using the generally recognized criteria to predict the binding interactions of miRNA:MREs and their secondary structures. The cut-offs −24 and −15 kcal/mol were chosen subjectively to avoid predicting too few or too many putative miRNAs targeting VEGF. When strict screening criteria were used, which included a minimal free energy (ΔG) below −24 kcal/mol, tail scores of miRNA:MREs duplexes above 15, seed windows at nt2–7 with conservation without G-U wobble, 16 MREs for 15 potential miRNAs were predicted as Group 1 (see <a href="#pone.0001719.s001">Table S1A</a>) in the VEGF 3′-UTR. When we relaxed the screening criteria to ΔG between −24 and −15 kcal/mol, tail score above 5, plus elastic seed windows (nt1–6, nt2–7, or nt3–8) with tolerance for a single G-U wobble in the seed region<a href="#pone.0001719-Didiano1">[42]</a>, hundreds of MREs and miRNAs were predicted. From these miRNAs, we randomly chose 15 miRNAs as the experimental Group 2 (<a href="#pone.0001719.s001">Table S1B</a>). Hypoxia-induced CNE cells (nasopharyngeal carcinoma cell line) were transfected with synthetic miRNA duplexes, including all the miRNAs in Group 1 and 2. A random sequence (NC), m<em>iR-224</em>, mutated <em>miR-16</em>, and <em>miR-20a</em> that did not have MREs in the VEGF 3′-UTR were used as negative controls. VEGF expression was measured by ELISA. The experiments indicated that 23 of the 30 miRNAs significantly repressed VEGF expression by over 20%, compared to negative controls (<a href="#pone-0001719-g001">Figure 1</a>).</p>
<div class="figure" id="pone-0001719-g001"><div class="img"><a name="pone-0001719-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001719.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001719" data-uri="info:doi/10.1371/journal.pone.0001719.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001719.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001719.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001719.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001719.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001719.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001719.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001719.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001719.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001719.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Validation of VEGF regulation by putative miRNAs.</span></strong></p><a id="article1.body1.sec2.sec1.fig1.caption1.p1" name="article1.body1.sec2.sec1.fig1.caption1.p1"></a><p>The effect of putative VEGF-regulative miRNAs on VEGF expression was tested in hypoxia-induced CNE cells by transfecting the cells with siRNA duplexes homologous in sequence to the miRNAs in group 1 (A) and 2 (B). VEGF expression was determined by ELISA. MiRNAs in group 1 were selected using a strict criteria: free energy less than −24 kcal/mol, nt2–7 perfectly pairing with the 5′-end of miRNAs, and sequence conservation of target sites across five vertebrate species. MiRNAs in group 2 were chosen from miRNAs that only met relatively relaxed criteria: free energy between −24 kcal/mol and −15 kcal/mol, and an elastic seed window tolerating one G:U wobble in successive 6-mers. Repressive ratio = (1–ELISA value of miRNA/ELISA value of blank)×100%. The blank is the sample from cells without transfection, providing a control for protein expression in the absence of regulation. A random sequence (NC), <em>miR-224</em>, mutated <em>miR-16</em> (<em>miR-16-M</em>), and <em>miR-20a</em> (<em>miR-20a-M</em>) that do not have MREs in the VEGF 3′-UTR were used as negative controls.</p>
<span>doi:10.1371/journal.pone.0001719.g001</span></div><a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2"></a><p>To confirm the repressive action of the miRNAs through forming miRNA:MRE duplexes, we inserted two fragments of the VEGF 3′-UTR into a luciferase expression vector, generating the luciferase reporter constructs pRL-VEGF-Con1 and pRL-VEGF-Con2. pRL-VEGF-Con1 contains a fragment located at nt31-216 of the VEGF 3′-UTR, while pRL-VEGF-Con2 contains a fragment at nt703-944. Of the 30 miRNAs tested in the ELISA assay, 21 harbored a single MRE in pRL-VEGF-Con1 and/or pRL-VEGF-Con2. These miRNAs were co-transfected with their respective reporters into COS-7 cells and the levels of <em>Renilla</em> luciferase (RL) were measured to determine the repressive effects of these miRNAs. These experiments indicated that most of the selected miRNAs produced a direct effect with significant repression of luciferase activity as compared with the controls (<a href="#pone-0001719-g002">Figure 2</a>).</p>
<div class="figure" id="pone-0001719-g002"><div class="img"><a name="pone-0001719-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001719.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001719" data-uri="info:doi/10.1371/journal.pone.0001719.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001719.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001719.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001719.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001719.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001719.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001719.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001719.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001719.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001719.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Luciferase activity assay.</span></strong></p><a id="article1.body1.sec2.sec1.fig2.caption1.p1" name="article1.body1.sec2.sec1.fig2.caption1.p1"></a><p>COS-7 cells were co-transfected with the luciferase reporter vector, which contained either the VEGF 3′-UTR fragment nt31-216 (pRL-VEGF-Con1) or nt703-944 (pRL-VEGF-Con2), and an miRNA which has a putative binding site in either pRL-VEGF-Con1 (A) or pRL-VEGF-Con2 (B). Luciferase activity was measured to determine the effect of these miRNAs on luciferase translation. All miRNAs have only one predicted MRE in the corresponding report vector. Repressive ratio = (1–LA value of miRNA/LA value of blank)×100%. LA: luciferase activity. The blank is the sample from cells without transfection, providing a control for protein expression in the absence of regulation. A random sequence (NC), <em>miR-29b</em>, <em>miR-150</em>, <em>miR-106b</em>, and <em>miR-134</em> that do not have MREs in the VEGF 3′-UTR were used as negative controls.</p>
<span>doi:10.1371/journal.pone.0001719.g002</span></div>

<h4>The relationship between the central loops of miRNA:MRE duplexes and the repressive efficiency of miRNAs</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>We analyzed the relationship between the central loop of miRNA-mRNA duplexes and the repressive efficiency of miRNAs in Groups 1 and 2 on VEGF expression, and found that the location of the central loop in miRNA:MRE duplexes appeared to play an important role. The loops of the miRNA:MRE duplexes in <a href="#pone.0001719.s001">Table S1</a> were divided into three types according to their location in the duplex. Loops starting between nt9 and nt11 inclusive of the miRNAs were designated standard loops, before nt9 as Type I decentered loops, and after nt11 as Type II decentered loops (<a href="#pone-0001719-g003">Figure 3A</a>). Thus, of the 30 total miRNA:MREs tested, there were 21 miRNAs with standard central loops, 2 with Type I decentered loops, and 7 with Type II decentered loops. Of the 24 functional miRNAs with a repression ratio of VEGF expression &gt;20%, 19 harbored a standard central loop, and only 5 contained Type II decentered loops. On the other hand, amongst the 7 non-functional miRNAs, 2 had Type II decentered loops, 2 had Type I decentered loops, and only 3 had standard central loops. Our analysis indicates that most of the MREs with a decentered loop were either false positives or highly inefficient. According to these data, we hypothesize that the central loop is one of the important factors that guide miRNA-MRE interactions. However, our data only included interactions between miRNAs and the 3′-UTR of VEGF, and many VEGF regulatory miRNAs have more than one MRE in the 3′-UTR of VEGF with an overlap of target sites, and there is a degree of similarity between different microRNAs to each other. Therefore, more data was needed to confirm our hypothesis on the importance of the loop in the miRNA:MRE duplexes in miRNA functioning.</p>
<div class="figure" id="pone-0001719-g003"><div class="img"><a name="pone-0001719-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001719.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001719" data-uri="info:doi/10.1371/journal.pone.0001719.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001719.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001719.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001719.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001719.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001719.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001719.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001719.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001719.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001719.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Correlation and linear regression analysis of central loops of miRNA:MRE duplexes.</span></strong></p><a id="article1.body1.sec2.sec2.fig1.caption1.p1" name="article1.body1.sec2.sec2.fig1.caption1.p1"></a><p>A, Central loops (or bulges) of miRNA:MRE duplexes were divided into three categories: standard loop, type I decentered loop, and type II decentered loop. Standard loops start at between nt9 and nt11 of the miRNA:MRE duplex counting from the 5′ end of the miRNAs. The type I decentered loop starts before nt9, with the type II decentered loop opening after nt11. The loop score is designated according to the algorithm introduced in the <a href="#s4">Methods</a>. B–D, Relationship between miRNA repression efficiency and the central loop score (B), tail score (C), and minimum free energy (D). Compared with the tail score and minimum free energy, the central loop score has a closer correlative relationship with miRNA repression efficiency (r = 0.646, p&lt;0.001).</p>
<span>doi:10.1371/journal.pone.0001719.g003</span></div><a id="article1.body1.sec2.sec2.p2" name="article1.body1.sec2.sec2.p2"></a><p>The published data on miRNA target identification by experiments and bioinformatics approaches, which were available on the website DIANA Tarbase (<a href="http://www.diana.pcbi.upenn.edu/tarbase.html">www.diana.pcbi.upenn.edu/tarbase.html</a>)<a href="#pone.0001719-Lewis1">[31]</a>, <a href="#pone.0001719-Kiriakidou1">[32]</a>, <a href="#pone.0001719-Poy1">[43]</a>–<a href="#pone.0001719-Garzon1">[49]</a>, support our hypothesis. We used these data to validate the effect of the loop structure of miRNA:MRE duplexes on the repression of gene expression. To avoid the incongruency brought by different standards or outcomes of RNA secondary structure and free energy predicted by distinct algorithms, we re-calculated the secondary structures of miRNA:MRE duplexes and their free energy for all the predicted miRNAs obtain from the online database by RNAcofold, a program for the prediction of hybrid structures of two RNA sequences (<a href="http://www.tbi.univie.ac.at/ivo/RNA/">www.tbi.univie.ac.at/ivo/RNA/</a>)<a href="#pone.0001719-Hofacker1">[50]</a>–<a href="#pone.0001719-Abelson1">[52]</a>. Thus, we provided a relatively equal platform or prerequisite to compare the structure of all these microRNA targets. To exclude the interference of multiple MREs of a single miRNA for one gene, we only took those miRNAs that have single MREs on one gene into account, and a total of 20 miRNAs were selected (<a href="#pone.0001719.s002">Table S2</a>). Analysis of the effect of the loop structure of miRNA:MRE duplexes on the repression of gene expression showed that 18 of the 20 microRNAs had their loop starting from p9 to p11, while only 2 had Type I decentered loops. Therefore, while much attention has focused on the seed region in the identification of miRNA targets<a href="#pone.0001719-Lewis2">[53]</a>, <a href="#pone.0001719-Brennecke2">[54]</a>, it seems that other factors such as the location of the loops also have a great impact on gene repression.</p>
<a id="article1.body1.sec2.sec2.p3" name="article1.body1.sec2.sec2.p3"></a><p>The miRNAs with a single MRE in pRL-VEGF-Con1 and/or pRL-VEGF-Con2 (<a href="#pone.0001719.s003">Table S3</a>) were selected to investigate the relationship between miRNA repression efficiency and the secondary structure of miRNA:MRE duplexes. The central loops are scored as follows: (i) to get a location coefficient according to the locations of loops: 1 for standard loop, 0.75 for type II decentered loop, 0 or 0.5 for type I decentered loops starting before nt7 inclusive or at nt8; (ii) to get the size coefficient according to the sizes of the loops: 10 for 1 bp, 20 for 2 bp, and 25 for 3 bp. If the size of the loop was more then 3 bp, the size coefficient would drop to 20. The total loop scores were calculated by multiplying the location coefficient with the size coefficient. The relationship between miRNA efficiency and the central loop was analyzed using correlation and linear regression. The central loop scores demonstrated a significant correlative relationship with miRNA repressive efficiency (<a href="#pone-0001719-g003">Figure. 3B</a>). The tail score (<a href="#pone-0001719-g003">Figure 3C</a>) and minimum free energy (<a href="#pone-0001719-g003">Figure 3D</a>) did not show significant relationships with gene repression.</p>


<h4>Effect of loop location on miRNA action</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>Although the results of co-transfection of miRNAs and the reporter pRL-VEGF-Con1 and/or pRL-VEGF-Con2 suggested a tight connection between the loop structure and the repressive effect of miRNAs, direct evidence was still needed to confirm our hypothesis. To do so, we performed site mutation assays (<a href="#pone-0001719-g004">Figure 4</a>). Mutations of <em>miR-17-5p</em> (<a href="#pone-0001719-g004">Figure 4A</a>) and <em>miR-372</em> (<a href="#pone-0001719-g004">Figure 4B</a>) moved the central loops of the miRNA:MRE duplexes to Type I and Type II decentered loops while keeping the minimum free energy of the miRNA:MREs at similar levels, and the mutation of MREs of <em>miR-17-5p</em> and <em>miR-372</em> in pRL-VEGF-Con1 and pRL-VEGF-Con2 also formed Type I and Type II decentered loops (<a href="#pone-0001719-g004">Figure 4D and E</a>). COS-7 cells were then transfected with wild type or mutated miRNAs with different report vectors. The levels of luciferase activity were measured to determine the repressive effects of different miRNA:MRE duplexes. Duplexes with type I or II decentered loops produced significantly lower repressive effects, compared with the miRNA:MRE duplexes with standard loops (<a href="#pone-0001719-g004">Figure 4C and F</a>).</p>
<div class="figure" id="pone-0001719-g004"><div class="img"><a name="pone-0001719-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001719.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001719" data-uri="info:doi/10.1371/journal.pone.0001719.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001719.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001719.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001719.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001719.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001719.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001719.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001719.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001719.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001719.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>Mutation assays.</span></strong></p><a id="article1.body1.sec2.sec3.fig1.caption1.p1" name="article1.body1.sec2.sec3.fig1.caption1.p1"></a><p>Site-directed mutagenesis of <em>miR-17-5p</em> (A) , <em>miR-372</em> (B), pRL-VEGF-Con1 (D), and pRL-VEGF-Con2 (E) caused movement of the standard loops of miRNA:MRE duplexes, forming Type I and Type II decentered loops instead, while the free energy of miRNA:MREs was kept at similar levels. COS-7 cells were transfected with wild type or mutated miRNAs and different report vectors. The levels of luciferase activity decreased significantly due to the change in loop location (C and F).</p>
<span>doi:10.1371/journal.pone.0001719.g004</span></div><a id="article1.body1.sec2.sec3.p2" name="article1.body1.sec2.sec3.p2"></a><p>We then investigated the effect of loop location on miRNA functioning using naturally existing sites (<a href="#pone-0001719-g005">Figure 5</a>). We changed the fragments of the VEGF 3′-UTR in pRL-VEGF-Con1 or 2 to fragments of c-MET or COX2 3′-UTR, resulting in the constructs pRL-CMET-Con3 and pRL-COX2-Con4. Change of the 3′-UTR fragments allowed us to compare miRNA:MRE duplexes bearing standard central loops (<a href="#pone-0001719-g005">Figure 5A and D</a>) with those bearing Type I (<a href="#pone-0001719-g005">Figure 5B</a>) or Type II (<a href="#pone-0001719-g005">Figure 5E</a>) decentered loops. COS-7 cell were co-transfected with miRNAs and different report constructs, and the levels of luciferase activity measured. The experiments indicated that miRNA:MRE duplexes with Type I or Type II decentered loops caused by changing the fragments showed significantly lower repressive effects, compared with the miRNA:MRE duplexes with standard loops (<a href="#pone-0001719-g005">Figure 5C and F</a>). The results confirmed the importance of the location of central loops in miRNA:MRE in the repressive activity of miRNAs.</p>
<div class="figure" id="pone-0001719-g005"><div class="img"><a name="pone-0001719-g005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001719.g005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001719" data-uri="info:doi/10.1371/journal.pone.0001719.g005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001719.g005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001719.g005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001719.g005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001719.g005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001719.g005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001719.g005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001719.g005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001719.g005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001719.g005.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 5.  <span>Effect of loop location on miRNA functioning.</span></strong></p><a id="article1.body1.sec2.sec3.fig2.caption1.p1" name="article1.body1.sec2.sec3.fig2.caption1.p1"></a><p>The insert fragments of the VEGF 3′-UTR in pRL-VEGF-Con1 and/or pRL-VEGF-Con2 were changed to fragments of the c-Met 3′-UTR or COX-2 3′-UTR to create pRL-CMET-Con3 and pRL-COX2-Con4. These changes allowed us to compare miRNA:MRE duplexes bearing standard central loops (A and D) in miRNA:MRE duplexes with the ones bearing Type I (B) and Type II (E) decentered loops. Luciferase activity assays indicated that changes to Type I and Type II decentered loops decreased the repressive effect of miRNAs significantly (C and F).</p>
<span>doi:10.1371/journal.pone.0001719.g005</span></div>

<h4>The effect of adding loop scores as a screening criterion on MRE prediction</h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>Since the central loop of miRNA:MRE appeared to be important for miRNA-mediated gene regulation, addition of a loop score as a criterion for MRE prediction should improve the results. To confirm this, all the single MREs in this investigation and MREs in the results (<a href="#pone-0001719-g001">Figure 1</a>, <a href="#pone-0001719-g003">3</a>, and 6) of Kiriakidou's publication<a href="#pone.0001719-Kiriakidou1">[32]</a> were included to re-calculate the secondary structures of the miRNA:MRE duplexes by RNAcofold and screened by <em>FindTar</em> version 1.0 and <em>FindTar</em> version 2.0 (<a href="#pone.0001719.s004">Table S4</a>). <em>FindTar</em> version 1.0 uses the old criteria of seed homology plus free energy to predict MREs. FindTar version 2.0 uses new criteria, which adopts the central loop law as an additional criterion for MRE prediction. The MREs predicted by computational algorithms with or without the loop score were compared with the Gold Standard, the results of luciferase activity assays (<a href="#pone-0001719-t001">Table 1</a>). The sensitivity, specificity, and Youden Index were calculated (<a href="#pone-0001719-t001">Table 1</a>). Addition of the central loop scores increased specificity, decreased false positive rates, and reached Youden Index 1. However, it increased false negative rate slightly. Youden Index is a measure of the accuracy of a test based on its false positive and false negative rates, with the ideal value being 1.0. The results confirmed the effect of the loop scores in reducing the false positives and increasing the accuracy and specificity of MRE prediction.</p>
<div class="figure" id="pone-0001719-t001"><div class="img"><a name="pone-0001719-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001719.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001719" data-uri="info:doi/10.1371/journal.pone.0001719.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001719.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001719.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001719.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001719.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001719.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001719.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001719.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001719.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001719.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>Statistical analyses of the results in MRE prediction before and after filtering with the central loop score</span></strong></p><span>doi:10.1371/journal.pone.0001719.t001</span></div><a id="article1.body1.sec2.sec4.p2" name="article1.body1.sec2.sec4.p2"></a><p>Finally, we validated the importance of the loop score using published data<a href="#pone.0001719-Lim1">[55]</a>, <a href="#pone.0001719-Grimson1">[56]</a>. Lim LP and coworkers introduced chemically synthesized <em>miR-1</em>, <em>miR-124</em>, <em>miR-373</em>, <em>mir7</em>, <em>mir9</em>, <em>mir122a</em>, <em>mir128a</em>, <em>mir132</em>, <em>mir133a</em>, <em>mir142</em>, <em>mir148b</em>, and <em>mir181a</em> duplexes into HeLa cells, resulting in down-regulation of many mRNAs. The majority of the down-regulated mRNAs exhibit seed homologies with the introduced miRNAs. The experiment also showed that there are many mRNAs which have seeds homologies but are not under the regulation of the transfected miRNAs (false MREs). We used our revised <em>FindTar</em> algorithm (<em>FindTar</em> version 2.0) to calculate loop scores for the miRNA:MREs for all seed-containing mRNAs which were up- or down-regulated by the miRNAs used, and re-analyzed their microarray data (<a href="#pone-0001719-t002">Table 2</a>). Genes down-regulated or up-regulated at a p-value &lt;0.05 at both 12hr and 24hr were scanned by FindTar for MREs of <em>miR-1</em>, <em>miR-124</em>, <em>miR-373</em>, <em>mir7</em>, <em>mir9</em>, <em>mir122a</em>, <em>mir128a</em>, <em>mir132</em>, <em>mir133a</em>, <em>mir142</em>, <em>mir148b</em>, or <em>mir181a</em>. The MRE criteria for <em>FindTar</em> version 1.0 includes seed homologies and free energy &lt;−15, and <em>FindTar</em> version 2.0 includes seed homologies, free energy &lt;−15, and loop score ≥20. The total number of bases in the 3′-UTR of the seed-containing mRNAs which were up- or down-regulated by the miRNAs used, were counted. The ratio of bases per MRE-Up/bases per MRE-D was used to evaluate the specificity of miRNA-mediated gene down regulation. Bases per MRE were calculated by the total number of bases in the 3′-UTR sequence of all affected mRNAs divided by the number of total MREs in these mRNAs. If all the mRNAs had seed homologies with the introduced miRNAs just by chance (the extreme case), bases per MRE-Up should be equal to bases per MRE-D. In this case, the ratio of bases per MRE-Up/bases per MRE-D would be close to 1, and the down-regulation of the messages would not have a relationship with the miRNAs. Otherwise, the ratio would be over 1 and the down-regulation of the messages would be considered at least partially mediated by the miRNAs used. This would be due to the presence of more MREs in the down-regulated mRNAs than in the up-regulated mRNAs, because of the specificity of the MREs. In this report, all the ratios are over 1.5, as shown in Table II. Furthermore, the ratios in group screened with the loop scores (L) are higher than those in unscreened group (NL). These results suggest that, besides seed sequence, loop scores also affect the specificity of miRNA-mediated gene regulation.</p>
<div class="figure" id="pone-0001719-t002"><div class="img"><a name="pone-0001719-t002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001719.t002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001719" data-uri="info:doi/10.1371/journal.pone.0001719.t002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001719.t002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001719.t002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001719.t002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001719.t002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001719.t002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001719.t002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001719.t002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001719.t002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001719.t002.TIF"></span>)
                </a></li></ul></div><p><strong>Table 2.  <span>Re-analysis of Lim <em>et al</em> and Grimson <em>et al.</em> microarray data from miRNAs transfected in HeLa cells.</span></strong></p><span>doi:10.1371/journal.pone.0001719.t002</span></div>
</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p>In the past few years, complementarity between miRNA and the 3′-UTR of target mRNAs, the conservation of target 3′-UTR sequences across different species, and the thermodynamics of the miRNA:MREs were widely used as the criteria to predict MREs in different computational algorithms, including miRanda <a href="#pone.0001719-Enright1">[34]</a>, <a href="#pone.0001719-John1">[37]</a>, TargetScan <a href="#pone.0001719-Lewis1">[31]</a>, DIANA-microT <a href="#pone.0001719-Kiriakidou1">[32]</a>, and RNAhybrid <a href="#pone.0001719-Rehmsmeier1">[35]</a>. The number of binding sites, the distance between sites, the cost of disruption to existing mRNA UTR secondary structure <a href="#pone.0001719-Long1">[57]</a> and target-site accessibility <a href="#pone.0001719-Kertesz1">[58]</a> are also included as factors which might affect the accuracy of MRE prediction. However, the high levels of false positive prediction and the poor overlap of the predicted MREs between different computational algorithms were still common problems. We believed this was because some important rules governing miRNA:MRE interaction were not included.</p>
<a id="article1.body1.sec3.p2" name="article1.body1.sec3.p2"></a><p>Our results suggest that the location of central loop may be one of the important factors that have not been included. It is well known that binding of miRNAs with their targets is strongly affected by the nucleotides in the 5′ end of the miRNA:MRE duplex<a href="#pone.0001719-Lewis2">[53]</a>, <a href="#pone.0001719-Bartel1">[59]</a>, and that, in contrast to the strict requirement for base pairing at the proximal region, nucleotide mismatch at the distal region seems to be highly tolerated. However, there is hardly any previous discussion about the role of the loop region of the miRNA:MRE duplex on miRNA function, although Kiriakidou and coworkers have addressed the effect of the loop size on the efficiency of miRNA-mediated gene regulation<a href="#pone.0001719-Kiriakidou1">[32]</a>. In this investigation, we found that the location of the loop in the miRNA:mRNA duplexes is an important factor which affects the efficiency of gene regulation mediated by miRNAs. Loop scores combining both the size and location of the loops may be used as a new criterion for predicting MREs and their cognate miRNAs, resulting in a significant decrease in false positives.</p>
<a id="article1.body1.sec3.p3" name="article1.body1.sec3.p3"></a><p>The microarray data from Lim <em>et al.</em> and Grimson <em>et al.</em> provide strong support for the importance of the central loop rule<a href="#pone.0001719-Lim1">[55]</a>, <a href="#pone.0001719-Grimson1">[56]</a>. This can be shown through a mathematics approach. The number of predicted MREs in down-regulated genes is assigned to the T<sub>D</sub> variable, with loop scores &gt; = 20 as S<sub>D</sub>. The number of predicted MREs in up-regulated genes is then appointed as T<sub>U,</sub> with loop scores &gt; = 20 as S<sub>U</sub>. The ratio of positive MREs in down-regulated genes is named α, and the ratio in up-regulated genes β. Because miRNAs are down-regulative factors of gene expression, α should be bigger than β (α&gt;β). Amongst the true positive MREs, the ratio of MREs with loop scores &gt; = 20 is appointed m, and the false positive rate as n. According to the data in the <a href="#pone-0001719-t003">Table 3</a>, S<sub>D</sub> = [αm+(1-α)n]T<sub>D</sub>, S<sub>U</sub> = [βm+(1-β)n]T<sub>U.</sub> The total number of bases in the 3′-UTR in down-regulated genes is then assigned as B<sub>D </sub>and the total number of bases in up-regulated genes as B<sub>U. </sub>The ratio of bases per MRE-Up/bases per MRE-D in the group screened for their loop scores (loop score&gt; = 20) is bigger than that in a non-screened group (NL), so:<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001719.e003&amp;representation=PNG" class="inline-graphic"></span><br>α&gt;β, therefore m&gt;n, meaning the ratio of MREs with loop scores &gt; = 20 amongst true MREs is bigger than that in false MREs, so using the loop score as a criterion for MRE prediction is reasonable.</p>
<div class="figure" id="pone-0001719-t003"><div class="img"><a name="pone-0001719-t003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001719.t003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001719" data-uri="info:doi/10.1371/journal.pone.0001719.t003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001719.t003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001719.t003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001719.t003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001719.t003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001719.t003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001719.t003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001719.t003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001719.t003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001719.t003.TIF"></span>)
                </a></li></ul></div><p><strong>Table 3.  <span>Prediction of true and false MER with <em>FindTar</em></span></strong></p><span>doi:10.1371/journal.pone.0001719.t003</span></div><a id="article1.body1.sec3.p4" name="article1.body1.sec3.p4"></a><p>Because miRNA:mRNA duplexes with type I decentered loops usually have a slightly shorter seed region with 6–7 nucleotides, it may be argued that the decreased repressive efficiency of miRNAs with type I decentered loops is the result of their short seed regions causing a decreased binding affinity, rather than the location of the central loop. To address this issue, we compared the repressive effects of two groups of miRNAs on gene expression in <a href="#pone-0001719-t004">Table 4</a>. The miRNAs in both groups have a short seed region of 6–7 nucleotides, with group 1 having standard loops and group 2 with type I decentered loops. The results demonstrate that the repressive efficiency of group 1 is much stronger than group 2. These results suggest that loop location indeed affect the efficiency of miRNA regulation.</p>
<div class="figure" id="pone-0001719-t004"><div class="img"><a name="pone-0001719-t004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001719.t004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001719" data-uri="info:doi/10.1371/journal.pone.0001719.t004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001719.t004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001719.t004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001719.t004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001719.t004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001719.t004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001719.t004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001719.t004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001719.t004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001719.t004.TIF"></span>)
                </a></li></ul></div><p><strong>Table 4.  <span>Comparison of repressive ratio.</span></strong></p><span>doi:10.1371/journal.pone.0001719.t004</span></div><a id="article1.body1.sec3.p5" name="article1.body1.sec3.p5"></a><p>Evidence has shown that Ago, the major component of RISCs, includes both PAZ and PIWI domains. The PIWI domain of Ago, containing the triad of acidic amino acids DDH, is responsible for mRNA cleavage in RISCs, in which mRNA cleavage occurs between residues base-paired to nucleotides 10 and 11 of the siRNA, which is equivalent to the location of the standard loop of miRNA:mRNA duplexes. In contrast to the mRNA-cleaving RISC however, little is known about miRNPs or miRISCs. However, the position of the miRNA within the mature complex must be similar to that of the siRNA guide strand, because miRNPs can cleave mRNA when presented with highly complementary targets <a href="#pone.0001719-Filipowicz1">[60]</a>. What factors decide whether mRNA will be cleaved or translationally repressed when bound by miRNP or an siRNA RISC is not clear, but we believe that the loop of the miRNA:mRNA duplex may be one of the important factors. One element is the location of the standard loop. It is known that perfect complementarity in the central part of the siRNA:mRNA duplexes allows A-form helix formation in the region facing the DDH triad, which is mandatory for cleavage <a href="#pone.0001719-Filipowicz1">[60]</a>. The presence of a loop in this region effectively interferes with A-helix formation and would probably prevent cleavage. The other reason for the importance of a central loop in switching miRNPs or miRISCs from cleavage of the target to translational repression, is that the loop of the miRNA:mRNA duplex might affect the conformation of Ago. When siRNA or miRNA interact with Ago, its 5′ end binds to PIWI and the 3′ end to the PAZ domain of Ago. If there is flexible loop between the two ends, it would be easy for Ago to change its conformation to a new one which is necessary for translational repression.</p>
<a id="article1.body1.sec3.p6" name="article1.body1.sec3.p6"></a><p>Recently released data demonstrates that there are very complex interactions between miRNAs, mRNAs, miRNP proteins, and polyribosomes. It is reported that miRNP proteins, including Ago, are physically associated with the polyribosome, and that most miRNAs are also associated with polysomes. Association of miRNAs with polysomes is mRNA-mediated, with miRNAs associated with translating mRNAs, and a specific miRNA-targeted mRNA present in polysomes <a href="#pone.0001719-Nelson1">[61]</a>–<a href="#pone.0001719-Maroney1">[63]</a>. An miRNA:mRNA duplex with a flexible loop would have the ability to form different conformations to meet all of the needs in the complex process of translational repression.</p>
</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4>MicroRNA and mRNA 3-′UTR Database</h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p>Human microRNA sequences (version 8.2) were retrieved from the miRBase website<a href="#pone.0001719-GriffithsJones1">[64]</a> (<a href="http://microrna.sanger.ac.uk/sequences/index.shtml">http://microrna.sanger.ac.uk/sequences/i​ndex.shtml</a>). Sequences of the 3′-untranslated region (3′-UTR) of VEGF were retrieved from human (<em>Homo sapiens</em>), mouse (<em>Mus musculus</em>), rat (<em>Rattus norvegicus</em>), chimpanzee (<em>Pan troglodytes</em>), and Cow (<em>Bos taurus</em>) using the Ensembl Database (<a href="http://www.ensembl.org">http://www.ensembl.org</a>).</p>


<h4>Algorithm of FindTar Version1.0</h4>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1"></a><p>We defined a relatively relaxed criterion for the seed region of the MRE so that more possible miRNA targets could be included in the primary screening. A flexible miRNA “seed window” (nt1–6, 2–7 and 3–8 counted from the 5′ end of the miRNA) was used to scan the 3′-UTRs of a gene in the order of 5′-3′ for potential target sites. In the seed windows, we searched for a perfectly Watson-Crick–base-paired stretch of 6 nt but tolerated one G-U wobble.</p>
<a id="article1.body1.sec4.sec2.p2" name="article1.body1.sec4.sec2.p2"></a><p>After the seed regions were identified, the upstream flanking region of the seed and the seed region itself was extracted for Dynamic programming. The miRNA:mRNA duplex and the MRE sequences of the VEGF 3′-UTR were identified with Dynamic hybridization. The distal sequences in the miRNA:mRNA duplex were scored with revised Gotoh's and Marks' methods <a href="#pone.0001719-John1">[37]</a>, <a href="#pone.0001719-Gotoh1">[65]</a>. A score of +5 was given to G:C and A:T pairs, +2 to G:U wobble pairs, and −3 to mismatch pairs, with the gap-open and gap elongation parameters set to −8.0 and −2.0 respectively. The final scores of the miRNA:mRNA duplexes were the sum of single residue-pair match scores <a href="#pone.0001719-John1">[37]</a>. Then the sequences of MREs identified by Dynamic hybridization in the VEGF 3′-UTR and potential miRNA were used as input in RNAcofold to predict the RNA secondary structures of the miRNA:mRNA duplexes. If the length of the MRE identified by Dynamic programming was not long enough to form the miRNA:mRNA duplexes calculated by RNAcofold, the duplex would be simulated again using the same mRNA sequence. This time, the sequence used would begin further upstream, such that the region used was 40nt in length including the seed region.</p>
<a id="article1.body1.sec4.sec2.p3" name="article1.body1.sec4.sec2.p3"></a><p>RNAcofold most lately released in the Vienna RNA Package 1.5 beta version (<a href="http://www.tbi.univie.ac.at/ivo/RNA/">www.tbi.univie.ac.at/ivo/RNA/</a>) was incorporated into the <em>FindTar</em> version 1.0 algorithm. Sequences of MREs in the VEGF 3′-UTR and potential miRNA were input to precisely predict the RNA secondary structures of the miRNA:mRNA duplexes by calculating the minimum free energy (ΔG) of the whole miRNA:mRNA duplex.</p>


<h4>Algorithm of FindTar Version2.0</h4>
<a id="article1.body1.sec4.sec3.p1" name="article1.body1.sec4.sec3.p1"></a><p>In order to decrease the number of false positives in the prediction of putative miRNA targets, the central loop law was introduced into the <em>FindTar</em> version 1.0 to produce <em>FindTar Version2.0</em> (<a href="http://bio.sz.tsinghua.edu.cn/findtar/">http://bio.sz.tsinghua.edu.cn/findtar/</a>). There were three elements in the central loop law: loop position, loop size, and loop priority. (1) Loop position coefficient: 0 for loops starting before nt7 inclusive, 0.5 for loops starting at nt8, both named Type I decentered loops; 1 for loops starting between nt9 and nt11 inclusive, named the standard loop; 0.75 for loops starting after nt12 inclusive, named the Type II decentered loop. (2) Loop size coefficient: 10 for loops with a size of 1 bp, 20 for 2 bp, 25 for 3 bp. When the loop size is above 4 bp inclusive, the coefficient is back to 20. (3) Loop priority law: this criterion was defined to avoid the “noise” bulge which may confuse the central loop scoring system. In order to make it clear which loop should be scored and which should not, the priority for scoring a loop was defined as the first loop which met the following criteria: (<em>a</em>) the first loop starting before nt9 of more than 1 bp; (<em>b</em>) the first loop starting between nt9 and nt11 of any size–if there is more than one loop, then the longest loop will be chosen; (<em>c</em>) the first loop starting after nt11 of any size. Finally, the total loop score was calculated by multiplying the location coefficient with the size coefficient. The central loop score was applied as a filter after all the miRNA:MREs were predicted. All these duplexes were scored, with a higher score equivalent to a higher probability of functioning.</p>


<h4>Preparation of Reporter Vectors</h4>
<a id="article1.body1.sec4.sec4.p1" name="article1.body1.sec4.sec4.p1"></a><p>The construction of luciferase report vectors, named pRL-VEGF-Con1 and pRL-VEGF-Con2, has been described previously<a href="#pone.0001719-Hua1">[40]</a>. Briefly, two fragments of the VEGF 3′-UTR with multiple MREs were PCR-amplified using primers with the following sequences: construct 1 sense, <span class="gene">5′-cgttctagagtttcgggaaccagatctc</span>; antisense, <span class="gene">5′-aacactagtaatgcttccgccggagt</span>; construct 2 sense, <span class="gene">5′-tcttctagacaggtcagacggacag</span>; antisense, <span class="gene">5′-acaactagtctcttctcttcgccgg</span>. Two copies of each fragment were cloned downstream of the stop codon in pRL-TK (Promega Corp., Madison, WI, USA). pRL-VEGF-Con1 contains nucleotides nt31-216 of the VEGF 3′-UTR, while pRL-VEGF-Con2 contains nucleotides nt703-944 of the VEGF 3′-UTR. The site-specific mutants of pRL-VEGF-Con1 and pRL-VEGF-Con2 were also established by using Takara MutanBEST Kit (Takara Corp., Dalian, PRC) according the method provided by the manual.</p>


<h4>Cell Culture and Hypoxia Induction</h4>
<a id="article1.body1.sec4.sec5.p1" name="article1.body1.sec4.sec5.p1"></a><p>COS-7 cells were cultured in Dulbecco's modified Eagle medium (DMEM) (GIBCO, Carlsbad, CA, USA) containing 10% fetal bovine serum (GIBCO, Carlsbad, CA, USA), and the CNE cells obtained from Kunming Cell Bank (Kunming, China) were cultured in RPMI 1640 (GIBCO, Carlsbad, CA, USA) containing 10% fetal bovine serum. Hypoxia induction was performed by adding desferrioxamine mesylate (DFOM) (Sigma-Aldrich Co., Saint Louis, MO, USA) in the culture medium to a final concentration of 130 µM.</p>


<h4>Preparation of an siRNA duplex homologous to the miRNA sequence, transfection and dual luciferase assay</h4>
<a id="article1.body1.sec4.sec6.p1" name="article1.body1.sec4.sec6.p1"></a><p>SiRNA duplex homologous to the miRNA sequences were synthesized and purified by Shanghai GenePharma Co. Ltd (Shanghai, China). pRL-TK plasmids bearing MREs in the 3′-UTR (400 ng for CNE, 100 ng for COS-7) were co-transfected with the pGL-3 reporter plasmid (100 ng for CNE, 5 ng for COS-7) and 20 pmol of siRNA/miRNAs into CNE or COS-7 cells (~3×10<sup>4</sup>, 24-well cell culture plates) using Lipofectamine 2000 (Invitrogen, Carlsbad, CA) in accordance with the manufacturer's instructions. 26–32 hours after transfection, luciferase activity was determined using the Dual Luciferase Reporter Assay System (Promega Corp., Madison, WI, USA).</p>


<h4>ELISA assay</h4>
<a id="article1.body1.sec4.sec7.p1" name="article1.body1.sec4.sec7.p1"></a><p>The supernatant harvested from cell culture 30 hrs after transfection was assayed using human VEGF ELISA kits from R&amp;D Systems (Minneapolis, MN, USA) and read by GENios ELISA Microplate Reader (TECAN, Austria). All experiments were repeated in triplicate.</p>

</div>

<div id="section5" class="section"><a id="s5" name="s5" toc="s5" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0001719"></div><a name="pone.0001719.s001" id="pone.0001719.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001719.s001">Table S1. </a></strong></p><p class="siDoi">doi:10.1371/journal.pone.0001719.s001</p><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1"></a><p class="postSiDOI">(0.07 MB DOC)</p>
<a name="pone.0001719.s002" id="pone.0001719.s002"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001719.s002">Table S2. </a></strong></p><p class="siDoi">doi:10.1371/journal.pone.0001719.s002</p><a id="article1.body1.sec5.supplementary-material2.caption1.p1" name="article1.body1.sec5.supplementary-material2.caption1.p1"></a><p class="postSiDOI">(0.09 MB DOC)</p>
<a name="pone.0001719.s003" id="pone.0001719.s003"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001719.s003">Table S3. </a></strong></p><p class="siDoi">doi:10.1371/journal.pone.0001719.s003</p><a id="article1.body1.sec5.supplementary-material3.caption1.p1" name="article1.body1.sec5.supplementary-material3.caption1.p1"></a><p class="postSiDOI">(0.07 MB DOC)</p>
<a name="pone.0001719.s004" id="pone.0001719.s004"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001719.s004">Table S4. </a></strong></p><p class="siDoi">doi:10.1371/journal.pone.0001719.s004</p><a id="article1.body1.sec5.supplementary-material4.caption1.p1" name="article1.body1.sec5.supplementary-material4.caption1.p1"></a><p class="postSiDOI">(0.14 MB DOC)</p>
</div>



<div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: QL WY YZ. Performed the experiments: ZH QL WY. Analyzed the data: BY YZ CW. Contributed reagents/materials/analysis tools: GC SH CF GL. Wrote the paper: BY YZ CW.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0001719-Doench1" id="pone.0001719-Doench1"></a>Doench JG, Sharp PA (2004) Specificity of microRNA target selection in translational repression. Genes Dev  18: 504–511.  <ul class="find" data-citedArticleID="1066892" data-doi="10.1101/gad.1184404"><li><a href="http://dx.doi.org/10.1101/gad.1184404" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Specificity+of+microRNA+target+selection+in+translational+repression." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Specificity+of+microRNA+target+selection+in+translational+repression.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0001719-Hansen1" id="pone.0001719-Hansen1"></a>Hansen T, Olsen L, Lindow M, Jakobsen KD, Ullum H, et al.  (2007) Brain expressed microRNAs implicated in schizophrenia etiology. PLoS ONE  2: e873.  <ul class="find" data-citedArticleID="1066910" data-doi="10.1371/journal.pone.0000873"><li><a href="http://dx.doi.org/10.1371/journal.pone.0000873" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Brain+expressed+microRNAs+implicated+in+schizophrenia+etiology." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Brain+expressed+microRNAs+implicated+in+schizophrenia+etiology.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0001719-Seggerson1" id="pone.0001719-Seggerson1"></a>Seggerson K, Tang L, Moss EG (2002) Two genetic circuits repress the Caenorhabditis elegans heterochronic gene lin-28 after translation initiation. Dev Biol  243: 215–225.  <ul class="find" data-citedArticleID="1066972" data-doi="10.1006/dbio.2001.0563"><li><a href="http://dx.doi.org/10.1006/dbio.2001.0563" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Two+genetic+circuits+repress+the+Caenorhabditis+elegans+heterochronic+gene+lin-28+after+translation+initiation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Two+genetic+circuits+repress+the+Caenorhabditis+elegans+heterochronic+gene+lin-28+after+translation+initiation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0001719-Zeng1" id="pone.0001719-Zeng1"></a>Zeng Y, Wagner EJ, Cullen BR (2002) Both natural and designed micro RNAs can inhibit the expression of cognate mRNAs when expressed in human cells. Mol Cell  9: 1327–1333.  <ul class="find" data-citedArticleID="1066990" data-doi="10.1016/s1097-2765(02)00541-5"><li><a href="http://dx.doi.org/10.1016/s1097-2765(02)00541-5" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Both+natural+and+designed+micro+RNAs+can+inhibit+the+expression+of+cognate+mRNAs+when+expressed+in+human+cells." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Both+natural+and+designed+micro+RNAs+can+inhibit+the+expression+of+cognate+mRNAs+when+expressed+in+human+cells.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0001719-Yekta1" id="pone.0001719-Yekta1"></a>Yekta S, Shih IH, Bartel DP (2004) MicroRNA-directed cleavage of HOXB8 mRNA. Science  304: 594–596.  <ul class="find" data-citedArticleID="1066988" data-doi="10.1126/science.1097434"><li><a href="http://dx.doi.org/10.1126/science.1097434" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=MicroRNA-directed+cleavage+of+HOXB8+mRNA." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22MicroRNA-directed+cleavage+of+HOXB8+mRNA.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0001719-Hutvagner1" id="pone.0001719-Hutvagner1"></a>Hutvagner G, Zamore PD (2002) A microRNA in a multiple-turnover RNAi enzyme complex. Science  297: 2056–2060.  <ul class="find" data-citedArticleID="1066916" data-doi="10.1126/science.1073827"><li><a href="http://dx.doi.org/10.1126/science.1073827" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+microRNA+in+a+multiple-turnover+RNAi+enzyme+complex." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+microRNA+in+a+multiple-turnover+RNAi+enzyme+complex.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0001719-Kloosterman1" id="pone.0001719-Kloosterman1"></a>Kloosterman WP, Wienholds E, Ketting RF, Plasterk RH (2004) Substrate requirements for let-7 function in the developing zebrafish embryo. Nucleic Acids Res  32: 6284–6291.  <ul class="find" data-citedArticleID="1066932" data-doi="10.1093/nar/gkh968"><li><a href="http://dx.doi.org/10.1093/nar/gkh968" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Substrate+requirements+for+let-7+function+in+the+developing+zebrafish+embryo." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Substrate+requirements+for+let-7+function+in+the+developing+zebrafish+embryo.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0001719-Zhou1" id="pone.0001719-Zhou1"></a>Zhou B, Wang S, Mayr C, Bartel DP, Lodish HF (2007) miR-150, a microRNA expressed in mature B and T cells, blocks early B cell development when expressed prematurely. Proc Natl Acad Sci U S A  104: 7080–7085.  <ul class="find" data-citedArticleID="1066994" data-doi="10.1073/pnas.0702409104"><li><a href="http://dx.doi.org/10.1073/pnas.0702409104" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=miR-150%2C+a+microRNA+expressed+in+mature+B+and+T+cells%2C+blocks+early+B+cell+development+when+expressed+prematurely." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22miR-150%2C+a+microRNA+expressed+in+mature+B+and+T+cells%2C+blocks+early+B+cell+development+when+expressed+prematurely.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0001719-Corney1" id="pone.0001719-Corney1"></a>Corney DC, Flesken-Nikitin A, Godwin AK, Wang W, Nikitin AY (2007) MicroRNA-34b and MicroRNA-34c are targets of p53 and cooperate in control of cell proliferation and adhesion-independent growth. Cancer Res  67: 8433–8438.  <ul class="find" data-citedArticleID="1066888" data-doi="10.1158/0008-5472.can-07-1585"><li><a href="http://dx.doi.org/10.1158/0008-5472.can-07-1585" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=MicroRNA-34b+and+MicroRNA-34c+are+targets+of+p53+and+cooperate+in+control+of+cell+proliferation+and+adhesion-independent+growth." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22MicroRNA-34b+and+MicroRNA-34c+are+targets+of+p53+and+cooperate+in+control+of+cell+proliferation+and+adhesion-independent+growth.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0001719-Chen1" id="pone.0001719-Chen1"></a>Chen JF, Mandel EM, Thomson JM, Wu Q, Callis TE, et al.  (2006) The role of microRNA-1 and microRNA-133 in skeletal muscle proliferation and differentiation. Nat Genet  38: 228–233.  <ul class="find" data-citedArticleID="1066882" data-doi="10.1038/ng1725"><li><a href="http://dx.doi.org/10.1038/ng1725" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+role+of+microRNA-1+and+microRNA-133+in+skeletal+muscle+proliferation+and+differentiation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+role+of+microRNA-1+and+microRNA-133+in+skeletal+muscle+proliferation+and+differentiation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0001719-Johnson1" id="pone.0001719-Johnson1"></a>Johnson CD, Esquela-Kerscher A, Stefani G, Byrom M, Kelnar K, et al.  (2007) The let-7 microRNA represses cell proliferation pathways in human cells. Cancer Res  67: 7713–7722.  <ul class="find" data-citedArticleID="1066920" data-doi="10.1158/0008-5472.can-07-1083"><li><a href="http://dx.doi.org/10.1158/0008-5472.can-07-1083" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+let-7+microRNA+represses+cell+proliferation+pathways+in+human+cells." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+let-7+microRNA+represses+cell+proliferation+pathways+in+human+cells.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0001719-Naguibneva1" id="pone.0001719-Naguibneva1"></a>Naguibneva I, Ameyar-Zazoua M, Polesskaya A, Ait-Si-Ali S, Groisman R, et al.  (2006) The microRNA miR-181 targets the homeobox protein Hox-A11 during mammalian myoblast differentiation. Nat Cell Biol  8: 278–284.  <ul class="find" data-citedArticleID="1066958" data-doi="10.1038/ncb1373"><li><a href="http://dx.doi.org/10.1038/ncb1373" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+microRNA+miR-181+targets+the+homeobox+protein+Hox-A11+during+mammalian+myoblast+differentiation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+microRNA+miR-181+targets+the+homeobox+protein+Hox-A11+during+mammalian+myoblast+differentiation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0001719-Li1" id="pone.0001719-Li1"></a>Li X, Carthew RW (2005) A microRNA mediates EGF receptor signaling and promotes photoreceptor differentiation in the Drosophila eye. Cell  123: 1267–1277.  <ul class="find" data-citedArticleID="1066942" data-doi="10.1016/j.cell.2005.10.040"><li><a href="http://dx.doi.org/10.1016/j.cell.2005.10.040" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+microRNA+mediates+EGF+receptor+signaling+and+promotes+photoreceptor+differentiation+in+the+Drosophila+eye." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+microRNA+mediates+EGF+receptor+signaling+and+promotes+photoreceptor+differentiation+in+the+Drosophila+eye.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0001719-Martello1" id="pone.0001719-Martello1"></a>Martello G, Zacchigna L, Inui M, Montagner M, Adorno M, et al.  (2007) MicroRNA control of Nodal signalling. Nature  449: 183–188.  <ul class="find" data-citedArticleID="1066952" data-doi="10.1038/nature06100"><li><a href="http://dx.doi.org/10.1038/nature06100" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=MicroRNA+control+of+Nodal+signalling." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22MicroRNA+control+of+Nodal+signalling.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0001719-Kim1" id="pone.0001719-Kim1"></a>Kim J, Inoue K, Ishii J, Vanti WB, Voronov SV, et al.  (2007) A MicroRNA Feedback Circuit in Midbrain Dopamine Neurons. Science  317: 1220–1224.  <ul class="find" data-citedArticleID="1066928" data-doi="10.1126/science.1140481"><li><a href="http://dx.doi.org/10.1126/science.1140481" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+MicroRNA+Feedback+Circuit+in+Midbrain+Dopamine+Neurons." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+MicroRNA+Feedback+Circuit+in+Midbrain+Dopamine+Neurons.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0001719-Wu1" id="pone.0001719-Wu1"></a>Wu H, Neilson JR, Kumar P, Manocha M, Shankar P, et al.  (2007) miRNA Profiling of Naive, Effector and Memory CD8 T Cells. PLoS ONE  2: e1020.  <ul class="find" data-citedArticleID="1066986" data-doi="10.1371/journal.pone.0001020"><li><a href="http://dx.doi.org/10.1371/journal.pone.0001020" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=miRNA+Profiling+of+Naive%2C+Effector+and+Memory+CD8+T+Cells." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22miRNA+Profiling+of+Naive%2C+Effector+and+Memory+CD8+T+Cells.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0001719-Chan1" id="pone.0001719-Chan1"></a>Chan JA, Krichevsky AM, Kosik KS (2005) MicroRNA-21 is an antiapoptotic factor in human glioblastoma cells. Cancer Res  65: 6029–6033.  <ul class="find" data-citedArticleID="1066880" data-doi="10.1158/0008-5472.can-05-0137"><li><a href="http://dx.doi.org/10.1158/0008-5472.can-05-0137" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=MicroRNA-21+is+an+antiapoptotic+factor+in+human+glioblastoma+cells." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22MicroRNA-21+is+an+antiapoptotic+factor+in+human+glioblastoma+cells.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0001719-Chen2" id="pone.0001719-Chen2"></a>Chen Y, Stallings RL (2007) Differential patterns of microRNA expression in neuroblastoma are correlated with prognosis, differentiation, and apoptosis. Cancer Res  67: 976–983.  <ul class="find" data-citedArticleID="1066884" data-doi="10.1158/0008-5472.can-06-3667"><li><a href="http://dx.doi.org/10.1158/0008-5472.can-06-3667" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Differential+patterns+of+microRNA+expression+in+neuroblastoma+are+correlated+with+prognosis%2C+differentiation%2C+and+apoptosis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Differential+patterns+of+microRNA+expression+in+neuroblastoma+are+correlated+with+prognosis%2C+differentiation%2C+and+apoptosis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0001719-Brennecke1" id="pone.0001719-Brennecke1"></a>Brennecke J, Hipfner DR, Stark A, Russell RB, Cohen SM (2003) bantam encodes a developmentally regulated microRNA that controls cell proliferation and regulates the proapoptotic gene hid in Drosophila. Cell  113: 25–36.  <ul class="find" data-citedArticleID="1066874" data-doi="10.1016/s0092-8674(03)00231-9"><li><a href="http://dx.doi.org/10.1016/s0092-8674(03)00231-9" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=bantam+encodes+a+developmentally+regulated+microRNA+that+controls+cell+proliferation+and+regulates+the+proapoptotic+gene+hid+in+Drosophila." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22bantam+encodes+a+developmentally+regulated+microRNA+that+controls+cell+proliferation+and+regulates+the+proapoptotic+gene+hid+in+Drosophila.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0001719-ODonnell1" id="pone.0001719-ODonnell1"></a>O'Donnell KA, Wentzel EA, Zeller KI, Dang CV, Mendell JT (2005) c-Myc-regulated microRNAs modulate E2F1 expression. Nature  435: 839–843.  <ul class="find" data-citedArticleID="1066964" data-doi="10.1038/nature03677"><li><a href="http://dx.doi.org/10.1038/nature03677" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=c-Myc-regulated+microRNAs+modulate+E2F1+expression." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22c-Myc-regulated+microRNAs+modulate+E2F1+expression.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0001719-Abbott1" id="pone.0001719-Abbott1"></a>Abbott AL, Alvarez-Saavedra E, Miska EA, Lau NC, Bartel DP, et al.  (2005) The let-7 MicroRNA family members mir-48, mir-84, and mir-241 function together to regulate developmental timing in Caenorhabditis elegans. Dev Cell  9: 403–414.  <ul class="find" data-citedArticleID="1066866" data-doi="10.1016/j.devcel.2005.07.009"><li><a href="http://dx.doi.org/10.1016/j.devcel.2005.07.009" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+let-7+MicroRNA+family+members+mir-48%2C+mir-84%2C+and+mir-241+function+together+to+regulate+developmental+timing+in+Caenorhabditis+elegans." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+let-7+MicroRNA+family+members+mir-48%2C+mir-84%2C+and+mir-241+function+together+to+regulate+developmental+timing+in+Caenorhabditis+elegans.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0001719-Gregory1" id="pone.0001719-Gregory1"></a>Gregory RI, Shiekhattar R (2005) MicroRNA biogenesis and cancer. Cancer Res  65: 3509–3512.  <ul class="find" data-citedArticleID="1066904" data-doi="10.1158/0008-5472.can-05-0298"><li><a href="http://dx.doi.org/10.1158/0008-5472.can-05-0298" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=MicroRNA+biogenesis+and+cancer." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22MicroRNA+biogenesis+and+cancer.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0001719-Makunin1" id="pone.0001719-Makunin1"></a>Makunin IV, Pheasant M, Simons C, Mattick JS (2007) Orthologous MicroRNA Genes Are Located in Cancer-Associated Genomic Regions in Human and Mouse. PLoS ONE  2: e1133.  <ul class="find" data-citedArticleID="1066948" data-doi="10.1371/journal.pone.0001133"><li><a href="http://dx.doi.org/10.1371/journal.pone.0001133" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Orthologous+MicroRNA+Genes+Are+Located+in+Cancer-Associated+Genomic+Regions+in+Human+and+Mouse." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Orthologous+MicroRNA+Genes+Are+Located+in+Cancer-Associated+Genomic+Regions+in+Human+and+Mouse.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0001719-Volinia1" id="pone.0001719-Volinia1"></a>Volinia S, Calin GA, Liu CG, Ambs S, Cimmino A, et al.  (2006) A microRNA expression signature of human solid tumors defines cancer gene targets. Proc Natl Acad Sci U S A  103: 2257–2261.  <ul class="find" data-citedArticleID="1066982" data-doi="10.1073/pnas.0510565103"><li><a href="http://dx.doi.org/10.1073/pnas.0510565103" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+microRNA+expression+signature+of+human+solid+tumors+defines+cancer+gene+targets." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+microRNA+expression+signature+of+human+solid+tumors+defines+cancer+gene+targets.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0001719-Calin1" id="pone.0001719-Calin1"></a>Calin GA, Croce CM (2006) MicroRNA-cancer connection: the beginning of a new tale. Cancer Res  66: 7390–7394.  <ul class="find" data-citedArticleID="1066878" data-doi="10.1158/0008-5472.can-06-0800"><li><a href="http://dx.doi.org/10.1158/0008-5472.can-06-0800" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=MicroRNA-cancer+connection%3A+the+beginning+of+a+new+tale." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22MicroRNA-cancer+connection%3A+the+beginning+of+a+new+tale.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0001719-Lee1" id="pone.0001719-Lee1"></a>Lee DY, Deng Z, Wang CH, Yang BB (2007) MicroRNA-378 promotes cell survival, tumor growth, and angiogenesis by targeting SuFu and Fus-1 expression. Proc Natl Acad Sci U S A  104: 20350–20355.  <ul class="find" data-citedArticleID="1066936" data-doi="10.1073/pnas.0706901104"><li><a href="http://dx.doi.org/10.1073/pnas.0706901104" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=MicroRNA-378+promotes+cell+survival%2C+tumor+growth%2C+and+angiogenesis+by+targeting+SuFu+and+Fus-1+expression." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22MicroRNA-378+promotes+cell+survival%2C+tumor+growth%2C+and+angiogenesis+by+targeting+SuFu+and+Fus-1+expression.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0001719-Sonkoly1" id="pone.0001719-Sonkoly1"></a>Sonkoly E, Wei T, Janson PC, Saaf A, Lundeberg L, et al.  (2007) MicroRNAs: novel regulators involved in the pathogenesis of Psoriasis? PLoS ONE  2: e610.  <ul class="find" data-citedArticleID="1066976" data-doi="10.1371/journal.pone.0000610"><li><a href="http://dx.doi.org/10.1371/journal.pone.0000610" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=MicroRNAs%3A+novel+regulators+involved+in+the+pathogenesis+of+Psoriasis%3F" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22MicroRNAs%3A+novel+regulators+involved+in+the+pathogenesis+of+Psoriasis%3F%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0001719-Sheng1" id="pone.0001719-Sheng1"></a>Sheng Y, Engstrom PG, Lenhard B (2007) Mammalian MicroRNA Prediction through a Support Vector Machine Model of Sequence and Structure. PLoS ONE  2: e946.  <ul class="find" data-citedArticleID="1066974" data-doi="10.1371/journal.pone.0000946"><li><a href="http://dx.doi.org/10.1371/journal.pone.0000946" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mammalian+MicroRNA+Prediction+through+a+Support+Vector+Machine+Model+of+Sequence+and+Structure." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mammalian+MicroRNA+Prediction+through+a+Support+Vector+Machine+Model+of+Sequence+and+Structure.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0001719-Johnson2" id="pone.0001719-Johnson2"></a>Johnson SM, Grosshans H, Shingara J, Byrom M, Jarvis R, et al.  (2005) RAS is regulated by the let-7 microRNA family. Cell  120: 635–647.  <ul class="find" data-citedArticleID="1066922" data-doi="10.1016/j.cell.2005.01.014"><li><a href="http://dx.doi.org/10.1016/j.cell.2005.01.014" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=RAS+is+regulated+by+the+let-7+microRNA+family." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22RAS+is+regulated+by+the+let-7+microRNA+family.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0001719-Moss1" id="pone.0001719-Moss1"></a>Moss EG, Lee RC, Ambros V (1997) The cold shock domain protein LIN-28 controls developmental timing in C. elegans and is regulated by the lin-4 RNA. Cell  88: 637–646.  <ul class="find" data-citedArticleID="1066956" data-doi="10.1016/s0092-8674(00)81906-6"><li><a href="http://dx.doi.org/10.1016/s0092-8674(00)81906-6" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+cold+shock+domain+protein+LIN-28+controls+developmental+timing+in+C.+elegans+and+is+regulated+by+the+lin-4+RNA." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+cold+shock+domain+protein+LIN-28+controls+developmental+timing+in+C.+elegans+and+is+regulated+by+the+lin-4+RNA.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0001719-Lewis1" id="pone.0001719-Lewis1"></a>Lewis BP, Shih IH, Jones-Rhoades MW, Bartel DP, Burge CB (2003) Prediction of mammalian microRNA targets. Cell  115: 787–798.  <ul class="find" data-citedArticleID="1066938" data-doi="10.1016/s0092-8674(03)01018-3"><li><a href="http://dx.doi.org/10.1016/s0092-8674(03)01018-3" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Prediction+of+mammalian+microRNA+targets." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Prediction+of+mammalian+microRNA+targets.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0001719-Kiriakidou1" id="pone.0001719-Kiriakidou1"></a>Kiriakidou M, Nelson PT, Kouranov A, Fitziev P, Bouyioukos C, et al.  (2004) A combined computational-experimental approach predicts human microRNA targets. Genes Dev  18: 1165–1178.  <ul class="find" data-citedArticleID="1066930" data-doi="10.1101/gad.1184704"><li><a href="http://dx.doi.org/10.1101/gad.1184704" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+combined+computational-experimental+approach+predicts+human+microRNA+targets." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+combined+computational-experimental+approach+predicts+human+microRNA+targets.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0001719-JonesRhoades1" id="pone.0001719-JonesRhoades1"></a>Jones-Rhoades MW, Bartel DP (2004) Computational identification of plant microRNAs and their targets, including a stress-induced miRNA. Mol Cell  14: 787–799.  <ul class="find" data-citedArticleID="1066924" data-doi="10.1016/j.molcel.2004.05.027"><li><a href="http://dx.doi.org/10.1016/j.molcel.2004.05.027" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Computational+identification+of+plant+microRNAs+and+their+targets%2C+including+a+stress-induced+miRNA." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Computational+identification+of+plant+microRNAs+and+their+targets%2C+including+a+stress-induced+miRNA.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0001719-Enright1" id="pone.0001719-Enright1"></a>Enright AJ, John B, Gaul U, Tuschl T, Sander C, et al.  (2003) MicroRNA targets in Drosophila. Genome Biol  5: R1.  <ul class="find" data-citedArticleID="1066894" data-doi="10.1186/gb-2003-5-1-r1"><li><a href="http://dx.doi.org/10.1186/gb-2003-5-1-r1" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=MicroRNA+targets+in+Drosophila." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22MicroRNA+targets+in+Drosophila.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0001719-Rehmsmeier1" id="pone.0001719-Rehmsmeier1"></a>Rehmsmeier M, Steffen P, Hochsmann M, Giegerich R (2004) Fast and effective prediction of microRNA/target duplexes. Rna  10: 1507–1517.  <ul class="find" data-citedArticleID="1066968" data-doi="10.1261/rna.5248604"><li><a href="http://dx.doi.org/10.1261/rna.5248604" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Fast+and+effective+prediction+of+microRNA%2Ftarget+duplexes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Fast+and+effective+prediction+of+microRNA%2Ftarget+duplexes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">36.
              </span><a name="pone.0001719-Stark1" id="pone.0001719-Stark1"></a>Stark A, Brennecke J, Russell RB, Cohen SM (2003) Identification of Drosophila MicroRNA targets. PLoS Biol  1: E60.  <ul class="find" data-citedArticleID="1066978" data-doi="10.1371/journal.pbio.0000060"><li><a href="http://dx.doi.org/10.1371/journal.pbio.0000060" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Identification+of+Drosophila+MicroRNA+targets." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Identification+of+Drosophila+MicroRNA+targets.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">37.
              </span><a name="pone.0001719-John1" id="pone.0001719-John1"></a>John B, Enright AJ, Aravin A, Tuschl T, Sander C, et al.  (2004) Human MicroRNA targets. PLoS Biol  2: e363.  <ul class="find" data-citedArticleID="1066918" data-doi="10.1371/journal.pbio.0020363"><li><a href="http://dx.doi.org/10.1371/journal.pbio.0020363" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Human+MicroRNA+targets." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Human+MicroRNA+targets.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">38.
              </span><a name="pone.0001719-Sandmann1" id="pone.0001719-Sandmann1"></a>Sandmann T, Cohen SM (2007) Identification of Novel Drosophila melanogaster MicroRNAs. PLoS ONE  2: e1265.  <ul class="find" data-citedArticleID="1066970" data-doi="10.1371/journal.pone.0001265"><li><a href="http://dx.doi.org/10.1371/journal.pone.0001265" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Identification+of+Novel+Drosophila+melanogaster+MicroRNAs." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Identification+of+Novel+Drosophila+melanogaster+MicroRNAs.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">39.
              </span><a name="pone.0001719-Vella1" id="pone.0001719-Vella1"></a>Vella MC, Reinert K, Slack FJ (2004) Architecture of a validated microRNA::target interaction. Chem Biol  11: 1619–1623.  <ul class="find" data-citedArticleID="1066980" data-doi="10.1016/j.chembiol.2004.09.010"><li><a href="http://dx.doi.org/10.1016/j.chembiol.2004.09.010" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Architecture+of+a+validated+microRNA%3A%3Atarget+interaction." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Architecture+of+a+validated+microRNA%3A%3Atarget+interaction.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">40.
              </span><a name="pone.0001719-Hua1" id="pone.0001719-Hua1"></a>Hua Z, Lv Q, Ye W, Wong CK, Cai G, et al.  (2006) MiRNA-directed regulation of VEGF and other angiogenic factors under hypoxia. PLoS ONE  1: e116.  <ul class="find" data-citedArticleID="1066914" data-doi="10.1371/journal.pone.0000116"><li><a href="http://dx.doi.org/10.1371/journal.pone.0000116" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=MiRNA-directed+regulation+of+VEGF+and+other+angiogenic+factors+under+hypoxia." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22MiRNA-directed+regulation+of+VEGF+and+other+angiogenic+factors+under+hypoxia.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">41.
              </span><a name="pone.0001719-Ferrara1" id="pone.0001719-Ferrara1"></a>Ferrara N, Gerber HP, LeCouter J (2003) The biology of VEGF and its receptors. Nat Med  9: 669–676.  <ul class="find" data-citedArticleID="1066896" data-doi="10.1038/nm0603-669"><li><a href="http://dx.doi.org/10.1038/nm0603-669" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+biology+of+VEGF+and+its+receptors." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+biology+of+VEGF+and+its+receptors.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">42.
              </span><a name="pone.0001719-Didiano1" id="pone.0001719-Didiano1"></a>Didiano D, Hobert O (2006) Perfect seed pairing is not a generally reliable predictor for miRNA-target interactions. Nat Struct Mol Biol  13: 849–851.  <ul class="find" data-citedArticleID="1066890" data-doi="10.1038/nsmb1138"><li><a href="http://dx.doi.org/10.1038/nsmb1138" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Perfect+seed+pairing+is+not+a+generally+reliable+predictor+for+miRNA-target+interactions." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Perfect+seed+pairing+is+not+a+generally+reliable+predictor+for+miRNA-target+interactions.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">43.
              </span><a name="pone.0001719-Poy1" id="pone.0001719-Poy1"></a>Poy MN, Eliasson L, Krutzfeldt J, Kuwajima S, Ma X, et al.  (2004) A pancreatic islet-specific microRNA regulates insulin secretion. Nature  432: 226–230.  <ul class="find" data-citedArticleID="1066966" data-doi="10.1038/nature03076"><li><a href="http://dx.doi.org/10.1038/nature03076" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+pancreatic+islet-specific+microRNA+regulates+insulin+secretion." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+pancreatic+islet-specific+microRNA+regulates+insulin+secretion.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">44.
              </span><a name="pone.0001719-Zhao1" id="pone.0001719-Zhao1"></a>Zhao Y, Samal E, Srivastava D (2005) Serum response factor regulates a muscle-specific microRNA that targets Hand2 during cardiogenesis. Nature  436: 214–220.  <ul class="find" data-citedArticleID="1066992" data-doi="10.1038/nature03817"><li><a href="http://dx.doi.org/10.1038/nature03817" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Serum+response+factor+regulates+a+muscle-specific+microRNA+that+targets+Hand2+during+cardiogenesis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Serum+response+factor+regulates+a+muscle-specific+microRNA+that+targets+Hand2+during+cardiogenesis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">45.
              </span><a name="pone.0001719-Krek1" id="pone.0001719-Krek1"></a>Krek A, Grun D, Poy MN, Wolf R, Rosenberg L, et al.  (2005) Combinatorial microRNA target predictions. Nat Genet  37: 495–500.  <ul class="find" data-citedArticleID="1066934" data-doi="10.1038/ng1536"><li><a href="http://dx.doi.org/10.1038/ng1536" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Combinatorial+microRNA+target+predictions." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Combinatorial+microRNA+target+predictions.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">46.
              </span><a name="pone.0001719-Cimmino1" id="pone.0001719-Cimmino1"></a>Cimmino A, Calin GA, Fabbri M, Iorio MV, Ferracin M, et al.  (2005) miR-15 and miR-16 induce apoptosis by targeting BCL2. Proc Natl Acad Sci U S A  102: 13944–13949.  <ul class="find" data-citedArticleID="1066886" data-doi="10.1073/pnas.0506654102"><li><a href="http://dx.doi.org/10.1073/pnas.0506654102" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=miR-15+and+miR-16+induce+apoptosis+by+targeting+BCL2." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22miR-15+and+miR-16+induce+apoptosis+by+targeting+BCL2.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">47.
              </span><a name="pone.0001719-Voorhoeve1" id="pone.0001719-Voorhoeve1"></a>Voorhoeve PM, le Sage C, Schrier M, Gillis AJ, Stoop H, et al.  (2006) A genetic screen implicates miRNA-372 and miRNA-373 as oncogenes in testicular germ cell tumors. Cell  124: 1169–1181.  <ul class="find" data-citedArticleID="1066984" data-doi="10.1016/j.cell.2006.02.037"><li><a href="http://dx.doi.org/10.1016/j.cell.2006.02.037" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+genetic+screen+implicates+miRNA-372+and+miRNA-373+as+oncogenes+in+testicular+germ+cell+tumors." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+genetic+screen+implicates+miRNA-372+and+miRNA-373+as+oncogenes+in+testicular+germ+cell+tumors.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">48.
              </span><a name="pone.0001719-Martin1" id="pone.0001719-Martin1"></a>Martin MM, Lee EJ, Buckenberger JA, Schmittgen TD, Elton TS (2006) Microrna-155 regulates human angiotensin II type 1 receptor expression in fibroblasts. J Biol Chem.   <ul class="find" data-citedArticleID="1066954" data-doi="10.1074/jbc.m601496200"><li><a href="http://dx.doi.org/10.1074/jbc.m601496200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Microrna-155+regulates+human+angiotensin+II+type+1+receptor+expression+in+fibroblasts." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Microrna-155+regulates+human+angiotensin+II+type+1+receptor+expression+in+fibroblasts.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">49.
              </span><a name="pone.0001719-Garzon1" id="pone.0001719-Garzon1"></a>Garzon R, Pichiorri F, Palumbo T, Iuliano R, Cimmino A, et al.  (2006) MicroRNA fingerprints during human megakaryocytopoiesis. Proc Natl Acad Sci U S A  103: 5078–5083.  <ul class="find" data-citedArticleID="1066900" data-doi="10.1073/pnas.0600587103"><li><a href="http://dx.doi.org/10.1073/pnas.0600587103" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=MicroRNA+fingerprints+during+human+megakaryocytopoiesis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22MicroRNA+fingerprints+during+human+megakaryocytopoiesis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">50.
              </span><a name="pone.0001719-Hofacker1" id="pone.0001719-Hofacker1"></a>Hofacker IL (2003) Vienna RNA secondary structure server. Nucleic Acids Res  31: 3429–3431.  <ul class="find" data-citedArticleID="1066912" data-doi="10.1093/nar/gkg599"><li><a href="http://dx.doi.org/10.1093/nar/gkg599" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Vienna+RNA+secondary+structure+server." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Vienna+RNA+secondary+structure+server.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">51.
              </span><a name="pone.0001719-Bernhart1" id="pone.0001719-Bernhart1"></a>Bernhart SH, Tafer H, Muckstein U, Flamm C, Stadler PF, et al.  (2006) Partition function and base pairing probabilities of RNA heterodimers. Algorithms Mol Biol  1: 3.  <ul class="find" data-citedArticleID="1066872" data-doi="10.1186/1748-7188-1-3"><li><a href="http://dx.doi.org/10.1186/1748-7188-1-3" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Partition+function+and+base+pairing+probabilities+of+RNA+heterodimers." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Partition+function+and+base+pairing+probabilities+of+RNA+heterodimers.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">52.
              </span><a name="pone.0001719-Abelson1" id="pone.0001719-Abelson1"></a>Abelson JF, Kwan KY, O'Roak BJ, Baek DY, Stillman AA, et al.  (2005) Sequence variants in SLITRK1 are associated with Tourette's syndrome. Science  310: 317–320.  <ul class="find" data-citedArticleID="1066868" data-doi="10.1126/science.1116502"><li><a href="http://dx.doi.org/10.1126/science.1116502" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Sequence+variants+in+SLITRK1+are+associated+with+Tourette%27s+syndrome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Sequence+variants+in+SLITRK1+are+associated+with+Tourette%27s+syndrome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">53.
              </span><a name="pone.0001719-Lewis2" id="pone.0001719-Lewis2"></a>Lewis BP, Burge CB, Bartel DP (2005) Conserved seed pairing, often flanked by adenosines, indicates that thousands of human genes are microRNA targets. Cell  120: 15–20.  <ul class="find" data-citedArticleID="1066940" data-doi="10.1016/j.cell.2004.12.035"><li><a href="http://dx.doi.org/10.1016/j.cell.2004.12.035" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Conserved+seed+pairing%2C+often+flanked+by+adenosines%2C+indicates+that+thousands+of+human+genes+are+microRNA+targets." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Conserved+seed+pairing%2C+often+flanked+by+adenosines%2C+indicates+that+thousands+of+human+genes+are+microRNA+targets.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">54.
              </span><a name="pone.0001719-Brennecke2" id="pone.0001719-Brennecke2"></a>Brennecke J, Stark A, Russell RB, Cohen SM (2005) Principles of microRNA-target recognition. PLoS Biol  3: e85.  <ul class="find" data-citedArticleID="1066876" data-doi="10.1371/journal.pbio.0030085"><li><a href="http://dx.doi.org/10.1371/journal.pbio.0030085" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Principles+of+microRNA-target+recognition." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Principles+of+microRNA-target+recognition.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">55.
              </span><a name="pone.0001719-Lim1" id="pone.0001719-Lim1"></a>Lim LP, Lau NC, Garrett-Engele P, Grimson A, Schelter JM, et al.  (2005) Microarray analysis shows that some microRNAs downregulate large numbers of target mRNAs. Nature  433: 769–773.  <ul class="find" data-citedArticleID="1066944" data-doi="10.1038/nature03315"><li><a href="http://dx.doi.org/10.1038/nature03315" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Microarray+analysis+shows+that+some+microRNAs+downregulate+large+numbers+of+target+mRNAs." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Microarray+analysis+shows+that+some+microRNAs+downregulate+large+numbers+of+target+mRNAs.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">56.
              </span><a name="pone.0001719-Grimson1" id="pone.0001719-Grimson1"></a>Grimson A, Farh KK, Johnston WK, Garrett-Engele P, Lim LP, et al.  (2007) MicroRNA targeting specificity in mammals: determinants beyond seed pairing. Mol Cell  27: 91–105.  <ul class="find" data-citedArticleID="1066908" data-doi="10.1016/j.molcel.2007.06.017"><li><a href="http://dx.doi.org/10.1016/j.molcel.2007.06.017" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=MicroRNA+targeting+specificity+in+mammals%3A+determinants+beyond+seed+pairing." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22MicroRNA+targeting+specificity+in+mammals%3A+determinants+beyond+seed+pairing.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">57.
              </span><a name="pone.0001719-Long1" id="pone.0001719-Long1"></a>Long D, Lee R, Williams P, Chan CY, Ambros V, et al.  (2007) Potent effect of target structure on microRNA function. Nat Struct Mol Biol  14: 287–294.  <ul class="find" data-citedArticleID="1066946" data-doi="10.1038/nsmb1226"><li><a href="http://dx.doi.org/10.1038/nsmb1226" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Potent+effect+of+target+structure+on+microRNA+function." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Potent+effect+of+target+structure+on+microRNA+function.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">58.
              </span><a name="pone.0001719-Kertesz1" id="pone.0001719-Kertesz1"></a>Kertesz M, Iovino N, Unnerstall U, Gaul U, Segal E (2007) The role of site accessibility in microRNA target recognition. Nat Genet  39: 1278–1284.  <ul class="find" data-citedArticleID="1066926" data-doi="10.1038/ng2135"><li><a href="http://dx.doi.org/10.1038/ng2135" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+role+of+site+accessibility+in+microRNA+target+recognition." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+role+of+site+accessibility+in+microRNA+target+recognition.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">59.
              </span><a name="pone.0001719-Bartel1" id="pone.0001719-Bartel1"></a>Bartel DP (2004) MicroRNAs: genomics, biogenesis, mechanism, and function. Cell  116: 281–297.  <ul class="find" data-citedArticleID="1066870" data-doi="10.1016/s0092-8674(04)00045-5"><li><a href="http://dx.doi.org/10.1016/s0092-8674(04)00045-5" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=MicroRNAs%3A+genomics%2C+biogenesis%2C+mechanism%2C+and+function." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22MicroRNAs%3A+genomics%2C+biogenesis%2C+mechanism%2C+and+function.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">60.
              </span><a name="pone.0001719-Filipowicz1" id="pone.0001719-Filipowicz1"></a>Filipowicz W (2005) RNAi: the nuts and bolts of the RISC machine. Cell  122: 17–20.  <ul class="find" data-citedArticleID="1066898" data-doi="10.1016/j.cell.2005.06.023"><li><a href="http://dx.doi.org/10.1016/j.cell.2005.06.023" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=RNAi%3A+the+nuts+and+bolts+of+the+RISC+machine." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22RNAi%3A+the+nuts+and+bolts+of+the+RISC+machine.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">61.
              </span><a name="pone.0001719-Nelson1" id="pone.0001719-Nelson1"></a>Nelson PT, Hatzigeorgiou AG, Mourelatos Z (2004) miRNP:mRNA association in polyribosomes in a human neuronal cell line. Rna  10: 387–394.  <ul class="find" data-citedArticleID="1066960" data-doi="10.1261/rna.5181104"><li><a href="http://dx.doi.org/10.1261/rna.5181104" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=miRNP%3AmRNA+association+in+polyribosomes+in+a+human+neuronal+cell+line." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22miRNP%3AmRNA+association+in+polyribosomes+in+a+human+neuronal+cell+line.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">62.
              </span><a name="pone.0001719-Nottrott1" id="pone.0001719-Nottrott1"></a>Nottrott S, Simard MJ, Richter JD (2006) Human let-7a miRNA blocks protein production on actively translating polyribosomes. Nat Struct Mol Biol  13: 1108–1114.  <ul class="find" data-citedArticleID="1066962" data-doi="10.1038/nsmb1173"><li><a href="http://dx.doi.org/10.1038/nsmb1173" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Human+let-7a+miRNA+blocks+protein+production+on+actively+translating+polyribosomes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Human+let-7a+miRNA+blocks+protein+production+on+actively+translating+polyribosomes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">63.
              </span><a name="pone.0001719-Maroney1" id="pone.0001719-Maroney1"></a>Maroney PA, Yu Y, Fisher J, Nilsen TW (2006) Evidence that microRNAs are associated with translating messenger RNAs in human cells. Nat Struct Mol Biol  13: 1102–1107.  <ul class="find" data-citedArticleID="1066950" data-doi="10.1038/nsmb1174"><li><a href="http://dx.doi.org/10.1038/nsmb1174" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evidence+that+microRNAs+are+associated+with+translating+messenger+RNAs+in+human+cells." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evidence+that+microRNAs+are+associated+with+translating+messenger+RNAs+in+human+cells.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">64.
              </span><a name="pone.0001719-GriffithsJones1" id="pone.0001719-GriffithsJones1"></a>Griffiths-Jones S (2004) The microRNA Registry. Nucleic Acids Res  32: D109–111.  <ul class="find" data-citedArticleID="1066906" data-doi="10.1093/nar/gkh023"><li><a href="http://dx.doi.org/10.1093/nar/gkh023" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+microRNA+Registry." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+microRNA+Registry.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">65.
              </span><a name="pone.0001719-Gotoh1" id="pone.0001719-Gotoh1"></a>Gotoh O (1982) An improved algorithm for matching biological sequences. J Mol Biol  162: 705–708.  <ul class="find" data-citedArticleID="1066902" data-doi="10.1016/0022-2836(82)90398-9"><li><a href="http://dx.doi.org/10.1016/0022-2836(82)90398-9" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=An+improved+algorithm+for+matching+biological+sequences." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22An+improved+algorithm+for+matching+biological+sequences.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001719.XML" value="128948"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001719.PDF" value="738148"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001719.g001.PNG_L" value="1194448"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001719.g001.PNG_M" value="95046"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001719.g001.PNG_S" value="10567"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001719.g001.TIF" value="1645300"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001719.g001.PNG_I" value="31105"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001719.g002.PNG_L" value="1144965"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001719.g002.PNG_M" value="87804"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001719.g002.PNG_S" value="10116"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001719.g002.TIF" value="1614916"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001719.g002.PNG_I" value="28333"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001719.g003.PNG_L" value="1033228"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001719.g003.PNG_M" value="99149"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001719.g003.PNG_S" value="9120"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001719.g003.TIF" value="1286838"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001719.g003.PNG_I" value="29246"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001719.g004.PNG_L" value="3200704"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001719.g004.PNG_M" value="297742"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001719.g004.PNG_S" value="12878"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001719.g004.TIF" value="4329502"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001719.g004.PNG_I" value="91242"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001719.g005.PNG_L" value="3656887"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001719.g005.PNG_M" value="317556"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001719.g005.PNG_S" value="13987"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001719.g005.TIF" value="4941130"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001719.g005.PNG_I" value="102745"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001719.t001.PNG_L" value="124087"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001719.t001.PNG_M" value="69550"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001719.t001.PNG_S" value="9881"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001719.t001.TIF" value="606726"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001719.t001.PNG_I" value="21612"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001719.e001.PNG" value="8013"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001719.e001.TIF" value="23948"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001719.e002.PNG" value="8111"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001719.e002.TIF" value="24068"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001719.t002.PNG_L" value="252106"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001719.t002.PNG_M" value="143835"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001719.t002.PNG_S" value="13706"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001719.t002.TIF" value="1009960"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001719.t002.PNG_I" value="45531"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001719.e003.PNG" value="21095"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001719.e003.TIF" value="37540"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001719.t003.PNG_L" value="44352"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001719.t003.PNG_M" value="62665"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001719.t003.PNG_S" value="11534"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001719.t003.TIF" value="209866"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001719.t003.PNG_I" value="21269"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001719.t004.PNG_L" value="217632"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001719.t004.PNG_M" value="140617"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001719.t004.PNG_S" value="14318"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001719.t004.TIF" value="1199314"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001719.t004.PNG_I" value="43169"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001719.s001.DOC" value="68608"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001719.s002.DOC" value="93184"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001719.s003.DOC" value="72704"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001719.s004.DOC" value="136192"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001719&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001719" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001719&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0001719&volume=&issue=&title=The Effect of Central Loops in miRNA:MRE Duplexes on the Efficiency of miRNA-Mediated Gene Regulation&author_name=Wenbin%20Ye%2C%20Qing%20Lv%2C%20Chung-Kwun%20Amy%20Wong%2C%20Sean%20Hu%2C%20Chao%20Fu%2C%20Zhong%20Hua%2C%20Guoping%20Cai%2C%20Guoxi%20Li%2C%20Burton%20B.%20Yang%2C%20Yaou%20Zhang&start_page=1&end_page=12" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001719" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001719" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0001719" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001719&amp;t=The%20Effect%20of%20Central%20Loops%20in%20miRNA%3AMRE%20Duplexes%20on%20the%20Efficiency%20of%20miRNA-Mediated%20Gene%20Regulation" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001719&title=The%20Effect%20of%20Central%20Loops%20in%20miRNA%3AMRE%20Duplexes%20on%20the%20Efficiency%20of%20miRNA-Mediated%20Gene%20Regulation&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001719&amp;title=The%20Effect%20of%20Central%20Loops%20in%20miRNA%3AMRE%20Duplexes%20on%20the%20Efficiency%20of%20miRNA-Mediated%20Gene%20Regulation" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001719" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0001719&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'The Effect of Central Loops in miRNA:MRE Duplexes on the Efficiency of miRNA-Mediated Gene Regulation';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0001719';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20The%20Effect%20of%20Central%20Loops%20in%20miRNA%3AMRE%20Duplexes%20on%20the%20Efficiency%20of%20miRNA-Mediated%20Gene%20Regulation http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001719" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0001719" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">












          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Algorithms%22" title="Search for articles in the subject area:'Algorithms'"><div class="flagText">Algorithms</div></a>
              <div data-categoryid="32621" data-articleid="26264"
                   data-categoryname="Algorithms"
                   class="flagImage" title="Flag 'Algorithms' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Free+energy%22" title="Search for articles in the subject area:'Free energy'"><div class="flagText">Free energy</div></a>
              <div data-categoryid="18159" data-articleid="26264"
                   data-categoryname="Free energy"
                   class="flagImage" title="Flag 'Free energy' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Gene+regulation%22" title="Search for articles in the subject area:'Gene regulation'"><div class="flagText">Gene regulation</div></a>
              <div data-categoryid="17553" data-articleid="26264"
                   data-categoryname="Gene regulation"
                   class="flagImage" title="Flag 'Gene regulation' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Gene+targeting%22" title="Search for articles in the subject area:'Gene targeting'"><div class="flagText">Gene targeting</div></a>
              <div data-categoryid="17701" data-articleid="26264"
                   data-categoryname="Gene targeting"
                   class="flagImage" title="Flag 'Gene targeting' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Luciferase%22" title="Search for articles in the subject area:'Luciferase'"><div class="flagText">Luciferase</div></a>
              <div data-categoryid="31895" data-articleid="26264"
                   data-categoryname="Luciferase"
                   class="flagImage" title="Flag 'Luciferase' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Luciferase+assay%22" title="Search for articles in the subject area:'Luciferase assay'"><div class="flagText">Luciferase assay</div></a>
              <div data-categoryid="21481" data-articleid="26264"
                   data-categoryname="Luciferase assay"
                   class="flagImage" title="Flag 'Luciferase assay' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Messenger+RNA%22" title="Search for articles in the subject area:'Messenger RNA'"><div class="flagText">Messenger RNA</div></a>
              <div data-categoryid="18353" data-articleid="26264"
                   data-categoryname="Messenger RNA"
                   class="flagImage" title="Flag 'Messenger RNA' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22MicroRNA%22" title="Search for articles in the subject area:'MicroRNA'"><div class="flagText">MicroRNA</div></a>
              <div data-categoryid="19821" data-articleid="26264"
                   data-categoryname="MicroRNA"
                   class="flagImage" title="Flag 'MicroRNA' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=9809'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=1442'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=8590&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>

<div class="block sidebar-comments">
    <div class="header">
        <h3>Comments</h3>
    </div>
      <p><a href="/annotation/listThread.action?root=13043">Publisher's Note: Correction to Affiliation 3</a><br>Posted by PLoS_ONE_Group</p>
</div>

</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
