

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Complexity Reduction of Polymorphic Sequences (CRoPS™): A Novel Approach for Large-Scale Polymorphism Discovery in Complex Genomes</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0001172"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0001172" />

    <meta name="citation_title" content="Complexity Reduction of Polymorphic Sequences (CRoPS™): A Novel Approach for Large-Scale Polymorphism Discovery in Complex Genomes"/>
    <meta itemprop="name" content="Complexity Reduction of Polymorphic Sequences (CRoPS™): A Novel Approach for Large-Scale Polymorphism Discovery in Complex Genomes"/>

      <meta name="citation_author" content="Nathalie J. van Orsouw"/>
            <meta name="citation_author_institution" content="Keygene NV, Wageningen, The Netherlands"/>
      <meta name="citation_author" content="René C. J. Hogers"/>
            <meta name="citation_author_institution" content="Keygene NV, Wageningen, The Netherlands"/>
      <meta name="citation_author" content="Antoine Janssen"/>
            <meta name="citation_author_institution" content="Keygene NV, Wageningen, The Netherlands"/>
      <meta name="citation_author" content="Feyruz Yalcin"/>
            <meta name="citation_author_institution" content="Keygene NV, Wageningen, The Netherlands"/>
      <meta name="citation_author" content="Sandor Snoeijers"/>
            <meta name="citation_author_institution" content="Keygene NV, Wageningen, The Netherlands"/>
      <meta name="citation_author" content="Esther Verstege"/>
            <meta name="citation_author_institution" content="Keygene NV, Wageningen, The Netherlands"/>
      <meta name="citation_author" content="Harrie Schneiders"/>
            <meta name="citation_author_institution" content="Keygene NV, Wageningen, The Netherlands"/>
      <meta name="citation_author" content="Hein van der Poel"/>
            <meta name="citation_author_institution" content="Keygene NV, Wageningen, The Netherlands"/>
      <meta name="citation_author" content="Jan van Oeveren"/>
            <meta name="citation_author_institution" content="Keygene NV, Wageningen, The Netherlands"/>
      <meta name="citation_author" content="Harold Verstegen"/>
            <meta name="citation_author_institution" content="Keygene NV, Wageningen, The Netherlands"/>
      <meta name="citation_author" content="Michiel J. T. van Eijk"/>
            <meta name="citation_author_institution" content="Keygene NV, Wageningen, The Netherlands"/>

    <meta name="citation_date" content="2007/11/14"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0001172.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e1172"/>
    <meta name="citation_issue" content="11"/>
    <meta name="citation_volume" content="2"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Analysis of the genome sequence of the flowering plant Arabidopsis thaliana.; citation_journal_title=Nature; citation_volume=408; citation_number=1; citation_pages=796-815; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=The map-based sequence of the rice genome.; citation_journal_title=Nature; citation_volume=436; citation_number=2; citation_pages=793-800; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Nested retrotransposons in the intergenic regions of the maize genome.; citation_author=P SanMiguel; citation_author=A Tikhonov; citation_author=Y-K Jin; citation_author=N Motchoulskaia; citation_author=D Zakharov; citation_journal_title=Science; citation_volume=274; citation_number=3; citation_pages=765-768; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Sequence composition, organization, and evolution of the core Triticeae genome.; citation_author=W Li; citation_author=P Zhang; citation_author=JP Fellers; citation_author=B Friebe; citation_author=BS Gill; citation_journal_title=Plant J.; citation_volume=40; citation_number=4; citation_pages=500-511; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Global repeat discovery and estimation of genomic copy number in a large, complex genome using a high-throughput 454 sequence survey.; citation_author=K Swaminathan; citation_author=K Varala; citation_author=ME Hudson; citation_journal_title=BMC Genomics; citation_volume=8; citation_number=5; citation_pages=1471-2164; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Analysis of red pepper (Capsicum annuum) genome.; citation_author=CS An; citation_author=SC Kim; citation_author=SL Go; citation_journal_title=J. Plant Biol.; citation_volume=39; citation_number=6; citation_pages=57-61; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=BeadArray™ technology: enabling an accurate, cost-effective approach to high-throughput genotyping.; citation_author=A Oliphant; citation_author=DL Barker; citation_author=JR Stuelpnagel; citation_author=MS Chee; citation_journal_title=Biotechniques; citation_volume=32; citation_number=7; citation_pages=S56-S61; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Multiplexed genotyping with sequence-tagged molecular inversion probes.; citation_author=P Hardenbol; citation_author=J Banér; citation_author=M Jain; citation_author=M Nilsson; citation_author=E Namsaraev; citation_journal_title=Nat. Biotechnol.; citation_volume=21; citation_number=8; citation_pages=673-678; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Large-scale genotyping of complex DNA.; citation_author=GC Kennedy; citation_author=H Matsuzaki; citation_author=S Dong; citation_author=W-M Liu; citation_author=J Huang; citation_journal_title=Nat. Biotechnol.; citation_volume=21; citation_number=9; citation_pages=1233-1237; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Parallel genotyping of over 10,000 SNPs using a one-primer assay on a high-density oligonucleotide array.; citation_author=H Matsuzaki; citation_author=H Loi; citation_author=S Dong; citation_author=Y-Y Tsai; citation_author=J Fang; citation_journal_title=Genome Res.; citation_volume=14; citation_number=10; citation_pages=414-425; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=A haplotype map of the human genome.; citation_journal_title=Nature; citation_volume=437; citation_number=11; citation_pages=1299-1320; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Sensitive mutation detection in heterogeneous cancer specimens by massive parallel picoliter reactor sequencing.; citation_author=RK Thomas; citation_author=E Nickerson; citation_author=JF Simons; citation_author=PA Jänne; citation_author=T Tengs; citation_journal_title=Nat. Med.; citation_volume=12; citation_number=12; citation_pages=852-855; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Nuclear DNA content of some important plant species.; citation_author=K Arumuganathan; citation_author=ED Earle; citation_journal_title=Plant Mol. Biol. Rep.; citation_volume=9; citation_number=13; citation_pages=208; citation_date=1991; " />
      <meta name="citation_reference" content="citation_title=Abundance, distribution, and transcriptional activity of repetitive elements in the maize genome.; citation_author=BC Meyers; citation_author=SV Tingey; citation_author=M Morgante; citation_journal_title=Genome Res.; citation_volume=11; citation_number=14; citation_pages=1660-1676; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Efficient capture of unique sequences from eukaryotic genomes.; citation_author=DG Peterson; citation_author=SR Wessler; citation_author=AH Paterson; citation_journal_title=Trends Genet.; citation_volume=18; citation_number=15; citation_pages=547-550; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Enrichment of gene-coding sequences in maize by genome filtration.; citation_author=CA Whitelaw; citation_author=WB Barbazuk; citation_author=G Pertea; citation_author=AP Chan; citation_author=F Cheung; citation_journal_title=Science; citation_volume=302; citation_number=16; citation_pages=2118-2120; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Gene enrichment in maize with hypomethylated partial restriction (HMPR) libraries.; citation_author=J Emberton; citation_author=J Ma; citation_author=Y Yuan; citation_author=P SanMiguel; citation_author=JL Bennetzen; citation_journal_title=Genome Res.; citation_volume=15; citation_number=17; citation_pages=1441-1446; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Active maize genes are unmodified and flanked by diverse classes of modified, highly repetitive DNA.; citation_author=JL Bennetzen; citation_author=K Schrick; citation_author=PS Springer; citation_author=WE Brown; citation_author=P SanMiguel; citation_journal_title=Genome; citation_volume=37; citation_number=18; citation_pages=565-576; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=Differential methylation of genes and retrotransposons facilitates shotgun sequencing of the maize genome.; citation_author=PD Rabinowicz; citation_author=K Schutz; citation_author=N Dedhia; citation_author=C Yordan; citation_author=LD Parnell; citation_journal_title=Nature Genet.; citation_volume=23; citation_number=19; citation_pages=305-308; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=AFLP: a new technique for DNA fingerprinting.; citation_author=P Vos; citation_author=R Hogers; citation_author=M Bleeker; citation_author=M Reijans; citation_author=T van de Lee; citation_journal_title=Nucleic Acids Res.; citation_volume=23; citation_number=20; citation_pages=4407-4414; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Selective restriction fragment amplification; a general method for DNA fingerprinting.; citation_author=M Zabeau; citation_author=P Vos; citation_journal_title=EP 0534858-A1,B1; US patent 6045994.; citation_number=21; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=AFLP technology for DNA fingerprinting.; citation_author=M Vuylsteke; citation_author=JD Peleman; citation_author=MJT van Eijk; citation_journal_title=Nature protocols; citation_volume=2; citation_number=22; citation_pages=1387-1398; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Genome sequencing in microfabricated high-density picolitre reactors.; citation_author=M Margulies; citation_author=M Egholm; citation_author=WE Altman; citation_author=S Attiya; citation_author=JS Bader; citation_journal_title=Nature; citation_volume=437; citation_number=23; citation_pages=376-380; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=SNPWave™: a flexible multiplexed SNP genotyping technology.; citation_author=MJT Van Eijk; citation_author=JLN Broekhof; citation_author=HJA van der Poel; citation_author=RCJ Hogers; citation_author=H Schneiders; citation_journal_title=Nucleic Acids Res.; citation_volume=32; citation_number=24; citation_pages=e47; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Sampling the Arabidopsis transcriptome with massively parallel pyrosequencing.; citation_author=APM Weber; citation_author=KL Weber; citation_author=K Carr; citation_author=C Wilkerson; citation_author=JB Ohlrogge; citation_journal_title=Plant Physiol.; citation_volume=144; citation_number=25; citation_pages=32-42; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=TIGR Gene Indices clustering tools (TGICL): a software system for fast clustering of large EST datasets.; citation_author=G Pertea; citation_author=X Huang; citation_author=F Liang; citation_author=V Antonescu; citation_author=R Sultana; citation_journal_title=Bioinformatics; citation_volume=19; citation_number=26; citation_pages=651-652; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=The use of coded PCR primers enables high-throughput sequencing of multiple homolog amplification products by 454 parallel sequencing.; citation_author=J Binladen; citation_author=MTP Gilbert; citation_author=JP Bollback; citation_author=F Panitz; citation_author=C Bendixen; citation_journal_title=Plos One; citation_volume=2; citation_number=27; citation_pages=e197; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=SNP discovery via 454 transcriptome sequencing.; citation_author=WB Barbazuk; citation_author=SJ Emrich; citation_author=HD Chen; citation_author=PS Schnable; citation_journal_title=Plant J.; citation_volume=51; citation_number=28; citation_pages=910-918; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=A rapid CTAB DNA isolation technique useful for RAPD fingerprinting and other PCR applications.; citation_author=CN Stuart; citation_author=LE Via; citation_journal_title=Biotechniques; citation_volume=14; citation_number=29; citation_pages=748-750; citation_date=1993; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001172" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Complexity Reduction of Polymorphic Sequences (CRoPS™): A Novel Approach for Large-Scale Polymorphism Discovery in Complex Genomes"/>
    <meta name="twitter:description" content="Application of single nucleotide polymorphisms (SNPs) is revolutionizing human bio-medical research. However, discovery of polymorphisms in low polymorphic species is still a challenging and costly endeavor, despite widespread availability of Sanger sequencing technology. We present CRoPS&trade; as a novel approach for polymorphism discovery by combining the power of reproducible genome complexity reduction of AFLP&reg; with Genome Sequencer (GS) 20/GS FLX next-generation sequencing technology. With CRoPS, hundreds-of-thousands of sequence reads derived from complexity-reduced genome sequences of two or more samples are processed and mined for SNPs using a fully-automated bioinformatics pipeline. We show that over 75% of putative maize SNPs discovered using CRoPS are successfully converted to SNPWave&reg; assays, confirming them to be true SNPs derived from unique (single-copy) genome sequences. By using CRoPS, polymorphism discovery will become affordable in organisms with high levels of repetitive DNA in the genome and/or low levels of polymorphism in the (breeding) germplasm without the need for prior sequence information."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0001172.g006"/>

  <meta property="og:title" content="Complexity Reduction of Polymorphic Sequences (CRoPS™): A Novel Approach for Large-Scale Polymorphism Discovery in Complex Genomes" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001172" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=2684'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=6440'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=9960&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0001172">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001172" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001172&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Complexity Reduction of Polymorphic Sequences (CRoPS™): A Novel Approach for Large-Scale Polymorphism Discovery in Complex Genomes
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Nathalie J. van Orsouw
              <span class="corresponding">mail</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:nathalie.van-orsouw@keygene.com">nathalie.van-orsouw@keygene.com</a></p>

                <p>Affiliation:
                  Keygene NV, Wageningen, The Netherlands
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            René C. J. Hogers, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Keygene NV, Wageningen, The Netherlands
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Antoine Janssen, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Keygene NV, Wageningen, The Netherlands
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Feyruz Yalcin, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Keygene NV, Wageningen, The Netherlands
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Sandor Snoeijers, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Keygene NV, Wageningen, The Netherlands
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Esther Verstege, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Keygene NV, Wageningen, The Netherlands
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Harrie Schneiders, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Keygene NV, Wageningen, The Netherlands
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Hein van der Poel, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Keygene NV, Wageningen, The Netherlands
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Jan van Oeveren, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Keygene NV, Wageningen, The Netherlands
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Harold Verstegen, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Keygene NV, Wageningen, The Netherlands
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Michiel J. T. van Eijk
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Keygene NV, Wageningen, The Netherlands
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: November 14, 2007</li>
    <li>DOI: 10.1371/journal.pone.0001172</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0001172" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0001172" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001172" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0001172" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0001172-t001" data-doi="info:doi/10.1371/journal.pone.0001172" data-uri="info:doi/10.1371/journal.pone.0001172.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001172.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001172-g001" data-doi="info:doi/10.1371/journal.pone.0001172" data-uri="info:doi/10.1371/journal.pone.0001172.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001172.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001172-g002" data-doi="info:doi/10.1371/journal.pone.0001172" data-uri="info:doi/10.1371/journal.pone.0001172.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001172.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001172-g003" data-doi="info:doi/10.1371/journal.pone.0001172" data-uri="info:doi/10.1371/journal.pone.0001172.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001172.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001172-g004" data-doi="info:doi/10.1371/journal.pone.0001172" data-uri="info:doi/10.1371/journal.pone.0001172.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001172.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001172-g005" data-doi="info:doi/10.1371/journal.pone.0001172" data-uri="info:doi/10.1371/journal.pone.0001172.g005" title="Figure 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001172.g005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001172-g006" data-doi="info:doi/10.1371/journal.pone.0001172" data-uri="info:doi/10.1371/journal.pone.0001172.g006" title="Figure 6">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001172.g006&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001172">Reader Comments (1)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0001172" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1"></a><p>Application of single nucleotide polymorphisms (SNPs) is revolutionizing human bio-medical research. However, discovery of polymorphisms in low polymorphic species is still a challenging and costly endeavor, despite widespread availability of Sanger sequencing technology. We present CRoPS™ as a novel approach for polymorphism discovery by combining the power of reproducible genome complexity reduction of AFLP® with Genome Sequencer (GS) 20/GS FLX next-generation sequencing technology. With CRoPS, hundreds-of-thousands of sequence reads derived from complexity-reduced genome sequences of two or more samples are processed and mined for SNPs using a fully-automated bioinformatics pipeline. We show that over 75% of putative maize SNPs discovered using CRoPS are successfully converted to SNPWave® assays, confirming them to be true SNPs derived from unique (single-copy) genome sequences. By using CRoPS, polymorphism discovery will become affordable in organisms with high levels of repetitive DNA in the genome and/or low levels of polymorphism in the (breeding) germplasm without the need for prior sequence information.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>van Orsouw NJ, Hogers RCJ, Janssen A, Yalcin F, Snoeijers S, et al.  (2007) Complexity Reduction of Polymorphic Sequences (CRoPS™): A Novel Approach for Large-Scale Polymorphism Discovery in Complex Genomes. PLoS ONE 2(11):
          e1172.
            doi:10.1371/journal.pone.0001172</p><p><strong>Academic Editor: </strong>Ivan Baxter, Purdue University, United States of America</p><p><strong>Received:</strong> September 25, 2007; <strong>Accepted:</strong> October 25, 2007; <strong>Published:</strong> November 14, 2007</p><p><strong>Copyright:</strong> © 2007 van Orsouw et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>This work was funded by Keygene N.V.</p><p><strong>Competing interests:</strong> Paid employment: All authors and co-authors with the exception of S. Snoeijers are employees of Keygene N.V. S. Snoeijers was a Keygene N.V. employee during the execution of this project but he has left Keygene N.V. on December 31, 2006. Patent application: The CRoPS technology is subject to patent applications owned by Keygene N.V. See also acknowledgements.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>SNP discovery is an important area of molecular genetics research aimed at collecting sufficient exploitable sequence polymorphisms to enable high-resolution, high-throughput genotyping at lower costs in the future. However, for many crop species the efficiency of the SNP discovery process is often hampered by the fact that limited amounts of genome sequences are available compared to e.g. <em>Arabidopsis</em> and rice, for which draft genome sequences have been completed <a href="#pone.0001172-The1">[1]</a>, <a href="#pone.0001172-International1">[2]</a>. Furthermore, the occurrence of (highly) duplicated genome sequences in crops such as maize <a href="#pone.0001172-SanMiguel1">[3]</a>, wheat <a href="#pone.0001172-Li1">[4]</a>, soybean <a href="#pone.0001172-Swaminathan1">[5]</a> and pepper <a href="#pone.0001172-An1">[6]</a> impedes conversion of identified polymorphisms into genotyping assays for application in breeding. As a result, available high-throughput SNP genotyping technologies <a href="#pone.0001172-Oliphant1">[7]</a>–<a href="#pone.0001172-Matsuzaki1">[10]</a> can not be fully exploited in plant breeding at present due to lack of suitable “content”. This is unlike the situation in humans where several millions of SNPs are known and being utilized in population genetic analysis <a href="#pone.0001172-The2">[11]</a> and medical diagnostics <a href="#pone.0001172-Thomas1">[12]</a>. Hence, there is a need for efficient polymorphism discovery technologies which target unique genome regions in organisms lacking extensive genome sequence information.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>The maize (<em>Zea mays</em>) genome comprises 2300 to 2700 Mb <a href="#pone.0001172-Arumuganathan1">[13]</a>. Approximately 80% of the total nuclear genome of maize consists of highly repetitive sequences interspersed with single-copy, gene-rich regions. The majority of the repeats are classified as long terminal repeat (LTR)-retrotransposon families that vary in copy number <a href="#pone.0001172-Meyers1">[14]</a>. As a consequence of these genome characteristics, SNP discovery in maize is not straightforward since it is not always obvious how to distinguish a true SNP from sequence differences between duplicated sequences occurring within the genome. Various techniques have been employed to enrich for single-copy sequences in maize, such as High C<sub>o</sub><em>t</em> selection <a href="#pone.0001172-Peterson1">[15]</a>, methylation filtering <a href="#pone.0001172-Whitelaw1">[16]</a> and hypomethylated partial restriction (HMPR) <a href="#pone.0001172-Emberton1">[17]</a>. HMPR utilizes methylation-sensitive restriction enzymes, thereby relying on the observation that in maize genes often remain unmethylated, whereas most LTR retrotransposons are methylated <a href="#pone.0001172-Bennetzen1">[18]</a>, <a href="#pone.0001172-Rabinowicz1">[19]</a>. Especially HMPR has been shown to be exceptional in depleting retrotransposons to less than 5% <a href="#pone.0001172-Emberton1">[17]</a> of the original content. However, despite the fact that these methods enrich for low-copy sequences, for economical reasons further genome complexity reduction is required to engage in comparative sequencing.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>The AFLP® technology <a href="#pone.0001172-Vos1">[20]</a>–<a href="#pone.0001172-Vuylsteke1">[22]</a> is a powerful DNA fingerprinting technology which has found widespread application in many organisms of diverse origin, including plants, animals, micro-organisms and human. AFLP is based on the selective PCR amplification of restriction fragments from a digest of whole genomic DNA. Its main features are that no prior sequence information is needed and multiplexing levels can be controlled by the choice (and number) of restriction endonucleases and by varying the number of selective bases of the primers used in the amplification process. Besides its many applications as genetic marker technology <a href="#pone.0001172-Vuylsteke1">[22]</a>, AFLP is therefore also a robust and scalable method for genome complexity reduction. This feature of the AFLP technology can be exploited to expedite polymorphism discovery by generating in parallel highly similar genome representations of multiple accessions of crop species for high-throughput sequencing.</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p>Here we describe the CRoPS™ technology (acronym for <span class="underline">C</span>omplexity <span class="underline">R</span>eduction <span class="underline">o</span>f <span class="underline">P</span>olymorphic <span class="underline">S</span>equences) and its application in maize. With CRoPS, tagged complexity-reduced libraries of two or more genetically diverse samples are prepared by AFLP, preferably using a methylation-sensitive restriction enzyme. Next, AFLP fragment libraries are sequenced at 5 to10-fold average redundancy in microfabricated, high-density picoliter reactions using the GS system <a href="#pone.0001172-Margulies1">[23]</a>. Resulting sequences are clustered and aligned, and the alignments are mined for SNPs using custom-developed bio-informatics tools. Rigorous quality measures are applied to separate PCR amplification and/or sequence errors from true polymorphisms. The fact that CRoPS is AFLP-based enables its application in many organisms, irrespective of genome complexity and size. The use of homozygous lines in the CRoPS process enables selection of SNPs which are located in low- or single copy genome sequences and therefore have a high conversion rate to genotyping assays for medium to large-scale genotyping.</p>
<a id="article1.body1.sec1.p5" name="article1.body1.sec1.p5"></a><p>The CRoPS technology has been applied for polymorphism discovery between the maize lines B73 and Mo17, using AFLP enzyme combination <em>Hpa</em>II/<em>Mse</em>I. Using a fully automated bioinformatics pipeline we mined more than 1200 high quality putative SNPs and show that 23 out of 30 SNPs were successfully converted into SNPWave assays <a href="#pone.0001172-VanEijk1">[24]</a>. We propose CRoPS as a generic approach to significantly enhance polymorphism discovery in vegetable and field crops.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3>
<h4>GS 20 sequence analysis</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>After completion of one single GS 20 run, a first bioinformatics analysis was performed using the GS 20 software (i.e. “on-rig” software). A total of 754,199 reads (“totalRawWells”) were obtained. The number of reads after the first filtering for Key sequences (“totalKeyPass”) was 739,042. Of these, 399,252 GS 20 raw sequencing reads remained after the final filtering by the GS 20 software. This number of sequence reads is higher than the specifications of the GS 20 but in line with other runs we performed earlier (data not shown) as well as results reported by others <a href="#pone.0001172-Weber1">[25]</a>. Their average read length was 103 nt (<a href="#pone-0001172-t001">Table 1</a>).</p>
<div class="figure" id="pone-0001172-t001"><div class="img"><a name="pone-0001172-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001172.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001172" data-uri="info:doi/10.1371/journal.pone.0001172.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001172.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001172.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001172.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001172.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001172.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001172.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001172.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001172.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001172.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>Overview of results of one GS 20 CRoPS run in maize</span></strong></p><span>doi:10.1371/journal.pone.0001172.t001</span></div><a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2"></a><p>Further bioinformatics analysis took place “off-rig” (i.e. on a separate server) using the CRoPS pipeline (<a href="#pone-0001172-g001">Fig. 1</a>). The GS 20 raw sequencing reads were trimmed (adapter removal) and 383,566 (96%) sequences remained (i.e. sequences for which a significant match with a tagged AFLP primer was found). The reasons for rejection of the remaining 15,686 (4%) reads (classified as faulty reads) were three-fold: 1) AFLP adapter not found, 2) conflict in adapter position (concatamers), and 3) sample identification tag conflict, i.e. a sequence with sample identification tag of one sample at one end and with sample identification tag of the second sample at the other end of the sequence read (so called “mixed fragments”, see further below).</p>
<div class="figure" id="pone-0001172-g001"><div class="img"><a name="pone-0001172-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001172.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001172" data-uri="info:doi/10.1371/journal.pone.0001172.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001172.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001172.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001172.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001172.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001172.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001172.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001172.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001172.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001172.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Bioinformatics pipeline for high-throughput analysis of CRoPS sequence runs.</span></strong></p><span>doi:10.1371/journal.pone.0001172.g001</span></div><a id="article1.body1.sec2.sec1.p3" name="article1.body1.sec2.sec1.p3"></a><p>Using the TIGR Gene Indices clustering tool (TGICL) <a href="#pone.0001172-Pertea1">[26]</a>, the remaining 383,566 sequences were clustered and assembled. Among these were two very large clusters (119,717 and 23,608 reads respectively) containing heavily repeated sequences. Homology searches using the Basic Local Alignment Search Tool (BLAST) revealed that the sequences within these two clusters were in fact chloroplast sequences. These two clusters were excluded from further processing. Subsequently, sequences within the remaining clusters were assembled into multiple sequence alignments (MSAs) (<a href="#pone-0001172-t001">Table 1</a>). In addition to the 18,989 MSAs containing 211,100 sequence reads, 29,141 (7.6%) singletons were found, i.e. sequences that were not assembled into an MSA.</p>
<a id="article1.body1.sec2.sec1.p4" name="article1.body1.sec2.sec1.p4"></a><p>Finally, SNPs were mined between the reads contained in an MSA. Parameters for SNP mining were set to include only SNPs for which both alleles were observed at least twice and SNPs not being part of homopolymers larger than 3 bases. The threshold for minimal distance to a neighboring SNP was initially set at one base, i.e. all SNPs were selected irrespective of their distance to (a) neighboring SNP(s). In addition, and importantly, SNPs were mined according to sample origin, i.e. only SNPs “segregating homozygously” between the two maize lines were included. By doing so, a strong filter was created to select against “false” SNPs resulting from alignment of highly homologous duplicated sequences as opposed to genuine SNPs derived from single-copy sequences in the sequenced genome fraction (<a href="#pone-0001172-g002">Fig. 2</a>). As a result, 1262 putative SNPs, including 37 putative indels were mined (<a href="#pone-0001172-t001">Table 1</a>).</p>
<div class="figure" id="pone-0001172-g002"><div class="img"><a name="pone-0001172-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001172.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001172" data-uri="info:doi/10.1371/journal.pone.0001172.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001172.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001172.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001172.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001172.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001172.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001172.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001172.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001172.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001172.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Example of a multiple sequence alignment (MSA) with SNP and sample related properties.</span></strong></p><a id="article1.body1.sec2.sec1.fig2.caption1.p1" name="article1.body1.sec2.sec1.fig2.caption1.p1"></a><p>SNP properties include sequence depth (sd), the count on the number of reads at the polymorphic position, the relative position of the SNP on the consensus sequence, the distance to the neighboring SNP, flanking sequence size and homopolymeric region information. Sample related properties were derived from the Oracle database. The ratio sample sequence depth to MSA sequence depth is calculated.</p>
<span>doi:10.1371/journal.pone.0001172.g002</span></div>

<h4>Effect of search parameters on SNP mining</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>To investigate the relationship between the number of putative SNPs and SNP mining parameters, SNPs were mined under different parameter settings regarding the minimal available sequence information flanking the target SNP (1, 2, 3, 4, 6 or 12 bases), and the minimal interval of flanking sequence that must be devoid of additional SNPs (1, 2, 3, 4, 6 or 12 bases). SNP mining was performed by varying these two parameters in all 36 (6 times 6) possible combinations, while keeping the other SNP mining parameters, including minimal representation of both alleles at least twice, homopolymer settings and segregation according to sample origin constant. As expected, the number of SNPs mined according to these more stringent criteria decreased to less than 50% (from 1262 to 591; <a href="#pone-0001172-g003">Fig. 3</a>). This selection of 591 SNPs was available for subsequent assay design.</p>
<div class="figure" id="pone-0001172-g003"><div class="img"><a name="pone-0001172-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001172.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001172" data-uri="info:doi/10.1371/journal.pone.0001172.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001172.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001172.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001172.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001172.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001172.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001172.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001172.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001172.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001172.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Number of putative SNPs and indels as a function of the minimal length of flanking sequences surrounding the SNP and the minimal interval devoid of additional SNPs/indels.</span></strong></p><span>doi:10.1371/journal.pone.0001172.g003</span></div>

<h4>Validation of putative CRoPS SNPs</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>Small-scale validation of putative SNPs was carried out using the SNPWave® technology <a href="#pone.0001172-VanEijk1">[24]</a>. From the selection of 591 putative SNPs mined according to the stringent criteria mentioned above (including a minimal of 12 bases flanking sequence surrounding the target SNP and minimal interval of 12 bases devoid of additional SNPs), 30 SNPs were randomly selected. Two 15-plex SNPWave assays were designed and tested using two parental lines and 94 recombinant inbred lines (RIL) offspring of the ISU (B73×Mo17) maize mapping population. For 23 out of 30 tested loci (77%) clear SNPWave reactions products were observed for both alleles, while for the remaining 7 loci one or both alleles were not observed (conversion failure). For all 23 SNP loci functioning properly in the SNPWave assay, the parental lines B73 and Mo17 were polymorphic and segregation was observed among RIL lines (<a href="#pone-0001172-g004">Fig. 4</a>), indicative of a high proportion of mined SNPs being derived from single-copy regions in the genome.</p>
<div class="figure" id="pone-0001172-g004"><div class="img"><a name="pone-0001172-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001172.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001172" data-uri="info:doi/10.1371/journal.pone.0001172.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001172.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001172.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001172.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001172.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001172.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001172.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001172.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001172.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001172.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>Pseudo-gel image visualizations of two SNPWave assays in maize detected by capillary electrophoresis.</span></strong></p><a id="article1.body1.sec2.sec3.fig1.caption1.p1" name="article1.body1.sec2.sec3.fig1.caption1.p1"></a><p>Left panel: 13-plex SNPWave assay; right panel: 10-plex SNPWave assay. Number 1-9 represent different recombinant inbred line offspring of B73 and Mo17.</p>
<span>doi:10.1371/journal.pone.0001172.g004</span></div>
</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p>We have applied CRoPS technology for polymorphism discovery in maize and have mined more than 1200 high quality putative SNPs from a single GS 20 sequencing run. We speculate that the stringent but user-definable parameter settings of the bioinformatics pipeline as well as the use of <em>Hpa</em>II as one of the restriction enzymes for AFLP template preparation effectively enrich for SNPs located in low-copy or unique genome sequences which have a high success rate of conversion. Since SNPWave is a ligation-based multiplexed SNP genotyping technology <a href="#pone.0001172-VanEijk1">[24]</a>, we expect conversion rates to be similar when SNPs mined using CRoPS are converted using other ligation-based SNP genotyping technologies <a href="#pone.0001172-Oliphant1">[7]</a>, <a href="#pone.0001172-Hardenbol1">[8]</a>.</p>
<a id="article1.body1.sec3.p2" name="article1.body1.sec3.p2"></a><p>During the development of CRoPS, which led to the current sample preparation protocol, we have made several modifications (see <a href="#s4">Methods</a>) to the original protocol for GS 20 sequencing <a href="#pone.0001172-Margulies1">[23]</a> which was conceived for library preparation of a single sample. These modifications were introduced after the observation of so called “mixed-fragments” in earlier CRoPS runs (results not shown). “Mixed fragments” are sequence reads containing a sample identification tag of one sample at one end and the sample identification tag of another sample at the other end (<a href="#pone-0001172-g005">Fig. 5</a>). In earlier experiments we observed these “mixed fragments” at frequencies between 0.1 and 16% of all obtained reads per run with higher frequencies when more than two samples were involved (data not shown). We suspected that “mixed-fragments” arose from the combination of the enzymatic (3′-5′ exonuclease) mediated recession of free 3′ termini of sample DNA and concomitant fill-in using <em>Bst</em> polymerase to create blunt ends for GS 20 adapter ligation as per the original protocol. When this procedure is applied to a mixture of short PCR products containing single-stranded fragments (such as in case of CRoPS), heteroduplex fragments are formed upon mixing the two (or more) samples at this step. Since the different samples contain different four base sample identification tags at their 5′ ends, we suspected that the 3′ ends (which do not match the four base sample identification tags at the 5′ ends of the opposite strand of the heteroduplexes) are removed and filled-in with the opposite strand as template for polymerization. The net result of such an event is a sample identification tag switch (<a href="#pone-0001172-g005">Fig. 5</a>). Therefore, we omitted the end-polishing step and modified the GS 20 adapters A and B by adding a 5′ T nucleotide to allow T/A ligation as commonly performed in PCR product cloning (<a href="#pone-0001172-g006">Fig. 6</a>). This modification was also expected to prevent possible concatamer formation of PCR products. Indeed, these modifications reduced the occurrence of “mixed fragments” to negligible levels (less than 0.00025% of reads) in the CRoPS run reported here.</p>
<div class="figure" id="pone-0001172-g005"><div class="img"><a name="pone-0001172-g005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001172.g005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001172" data-uri="info:doi/10.1371/journal.pone.0001172.g005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001172.g005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001172.g005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001172.g005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001172.g005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001172.g005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001172.g005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001172.g005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001172.g005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001172.g005.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 5.  <span>Composition and hypothesized cause of “mixed fragments”.</span></strong></p><a id="article1.body1.sec3.fig1.caption1.p1" name="article1.body1.sec3.fig1.caption1.p1"></a><p>“Mixed fragments” are characterized by the occurrence of the sample identification tag of sample 1 on one side and the sample identification tag of sample 2 on the other side. (A) Schematic representation of observed homoduplex and heteroduplex fragment types containing expected tags and “mixed fragments”. (B) “Mixed fragments” are formed when (1) a heteroduplex is formed between complementary strands of samples 1 and 2, (2) 3′-5′ exonuclease activity of T4 DNA polymerase removes the sequence tags at the 3′ ends, (3) polymerase activity of T4 DNA polymerase extends the 3′ ends using the opposite strand as template, resulting in incorporation of the “wrong” sequence tag, i.e. the observation of “mixed fragments”.</p>
<span>doi:10.1371/journal.pone.0001172.g005</span></div><div class="figure" id="pone-0001172-g006"><div class="img"><a name="pone-0001172-g006" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001172.g006&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001172" data-uri="info:doi/10.1371/journal.pone.0001172.g006"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001172.g006&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001172.g006/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001172.g006/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001172.g006/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001172.g006/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001172.g006.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001172.g006/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001172.g006/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001172.g006.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 6.  <span>Protocol modification to avoid “mixed fragments”.</span></strong></p><a id="article1.body1.sec3.fig2.caption1.p1" name="article1.body1.sec3.fig2.caption1.p1"></a><p>(A) Blunt-end adapter ligation as per the original GS 20 library preparation protocol. (B) T/A ligation as applied in the CRoPS protocol. Amplification using a polymerase lacking 3′-5′ exonuclease (proofreading) activity is performed resulting in A-addition to the AFLP fragments, after which the T-adapters can be ligated. (C) Flowcharts of the original GS 20 library preparation protocol and the CRoPS library preparation protocol.</p>
<span>doi:10.1371/journal.pone.0001172.g006</span></div><a id="article1.body1.sec3.p3" name="article1.body1.sec3.p3"></a><p>Although it was attempted to carefully pool AFLP products of both samples in equal amounts, a somewhat skewed sample distribution in terms of reads per sample (39% sample 1 and 61% sample 2) was obtained. With varying levels of deviation from equal sample representation, this has also been observed in at least six other GS 20 runs (data not shown), despite our attempts to pool equimolar amounts of AFLP products. Clearly it would be beneficial to reach more equal sample representation to increase the number of SNPs mined per run. The same observation was made by Binladen and co-workers <a href="#pone.0001172-Binladen1">[27]</a>. In their study the effect of 5′-tag sequences was suggested as a likely explanation for the single molecule sequence variations. We support this explanation as we have observed in this GS 20 run and other subsequent runs (data not shown) that 5′ tag sequences containing “AC” bases at the 5′ end are significantly underrepresented when equimolar amounts of tagged sample DNAs are pooled. Therefore, in retrospect the choice of a sample identification tag containing “5′-AC” has contributed to the observed skewed sample distribution.</p>
<a id="article1.body1.sec3.p4" name="article1.body1.sec3.p4"></a><p>Other optimization steps expected to increase the output of CRoPS further include selection against plastid (chloroplast sequences) co-isolated with genomic DNA, the use of normalized genomic or cDNA <a href="#pone.0001172-Barbazuk1">[28]</a> libraries or other methods of enrichment for unique, single-copy sequences such as High C<sub>o</sub>t selection <a href="#pone.0001172-Peterson1">[15]</a> or methylfiltration <a href="#pone.0001172-Whitelaw1">[16]</a>, prior to AFLP amplification. The use of such enrichment methods contribute to losing as little as possible sequence capacity to (highly) repeated sequences. Obviously, the output of a CRoPS run will also increase considerably as a result of the recent introduction of the GS FLX which has output specifications of over 400,000 reads with average read length of 240 nt. The increased read length does not only increase the amount of basepairs per run but also reduces the fraction of SNPs that can not be exploited due to insufficient flanking sequence information available for assay development. In conclusion, CRoPS is a powerful technology for random genetic marker development, which meets the shortcomings intrinsic to many plant species, i.e. the lack of available sequence information, large genomes containing high proportions of duplicated sequences and/or low levels of polymorphism. In the absence of whole-genome draft sequences, high-throughput sequencing of genome representations of multiple accessions in parallel using CRoPS will supply sufficient genetic (single nucleotide) polymorphisms to allow marker-assisted selection using existing genotyping platforms. It is our expectation that these developments will allow high-resolution sequence-based breeding using thousands of genetic markers to become reality in the nearby future.</p>
</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4>AFLP target preparation</h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p>Total genomic DNA was isolated from leaf material of the two parental lines (i.e. B73 &amp; Mo17) of the ISU mapping population (<a href="http://www.maizegdb.org">www.maizegdb.org</a>), using a modified CTAB procedure <a href="#pone.0001172-Stuart1">[29]</a>. These 2 parental lines were chosen to be able to validate and map the discovered SNPs in the ISU mapping population.</p>
<a id="article1.body1.sec4.sec1.p2" name="article1.body1.sec4.sec1.p2"></a><p>AFLP templates were prepared as described previously <a href="#pone.0001172-Vos1">[20]</a>. In short, 100–500 ng total genomic DNA was digested using 5 units <em>Hpa</em>II and 2 units <em>Mse</em>I for at least 1 hour at 37°C. After digestion, the mixture was heated at 80°C for 10 min. Next, AFLP adapter ligation using <em>Hpa</em>II and <em>Mse</em>I adapters was carried out for 3 hours at 37°C. The restriction-ligation (RL) mixture was subsequently diluted 10-fold with T<sub>10</sub>E<sub>0.1</sub> and 5 µl diluted mix was used as a template in a selective pre-amplification step, the so-called +1/+1 pre-amplification. Primer sequences for the +1/+1 pre-amplification were 5′-GTAGACTGCGTACACGGA-3′ (<em>Hpa</em>II site, including 1 selective nucleotide “A”) and 5′-GATGAGTCCTGAGTAAC-3′ (<em>Mse</em>I site, including 1 selective nucleotide “C”). Twenty µl PCRs were performed containing 5 µl diluted RL mixture, 30 ng <em>Hpa</em>II primer, 30 ng <em>Mse</em>I primer, 0.2 mM dNTP, 0.4 U AmpliTaq® (Applied Biosystems) and 1× AmpliTaq buffer. PCR was performed for 20 cycles with the following cycle profile: 30 sec 94°C, 60 sec 56°C, 60 sec 72°C, followed by cooling down to 4°C.</p>
<a id="article1.body1.sec4.sec1.p3" name="article1.body1.sec4.sec1.p3"></a><p>The +1/+1 pre-amplification reaction was diluted 20-fold with T<sub>10</sub>E<sub>0.1</sub>, and used for the second selective amplification step, the so-called +1/+2 selective amplification. Primer sequences for the +1/+2 selective amplification were 5′-‘P-ACACGTAGACTGCGTACACGGA-3′ (<em>Hpa</em>II site, including 1 selective nucleotide “A”) and 5′-‘P-ACACGATGAGTCCTGAGTAACT-3′ (<em>Mse</em>I site, including 2 selective nucleotides “CT”) for sample B73. The four most 5′ bases of these primers serve as sample identification tag (KeyGene™ SeqTag technology). These 4-nt sample identification tags were selected from a collection of 4-nt sequences differing by at least 2 nt to exclude the possibility that a single nucleotide substitution error could cause incorrect assignment of the sequence to a sample. Similarly, primer sequences for the +1/+2 selective amplification of the Mo17 sample were 5′-‘P-AGCTGTAGACTGCGTACACGGA-3′ (<em>Hpa</em>II site, including 1 selective nucleotide “A”) and 5′-‘P-AGCTGATGAGTCCTGAGTAACT-3′ (<em>Mse</em>I site, including 2 selective nucleotides “CT”). Fifty µl PCRs were performed containing 5 µl diluted +1/+1 pre-amplification mixture, 75 ng <em>Hpa</em>II primer, 75 ng <em>Mse</em>I primer, 0.2 mM dNTP, 1 U AmpliTaq (Applied Biosystems) and 1× AmpliTaq buffer. PCR was performed for 30 cycles with the following cycle profile: 30 sec 94°C, 60 sec 56°C, 60 sec 72°C, followed by cooling down to 4°C.</p>
<a id="article1.body1.sec4.sec1.p4" name="article1.body1.sec4.sec1.p4"></a><p>Next, 100 µl of PCR products of each sample were purified using the QIAquick PCR Purification Kit (Qiagen). Concentrations of both samples were determined using the Nanodrop ND-1000 (Nanodrop Technologies), after which equal amounts of the two samples were pooled and further treated as one fragment library sample. This saves costs and prevents relying on physical compartmentalization to separate both samples. Furthermore this approach provides flexibility regarding processing multiple samples.</p>


<h4>GS 20 library preparation &amp; titration</h4>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1"></a><p>3.45 µg of the fragment library sample (i.e. pooled, purified and tagged AFLP products) were used as input for GS 20 library construction. The use of tagged and pooled PCR products, however, necessitated several adaptations in the published GS 20 library construction protocol <a href="#pone.0001172-Margulies1">[23]</a>. First, no shearing was carried out. Second, the end-polishing step was omitted, and modified A and B adapters were used as follows: adapter A-upper strand: 5′-CCATCTCATCCCTGCGTGTCCCATCTGTTCCCTCCCT​GTCTCAGT-3′,adapter A-lower strand: 5′-CTGAGACAGGGAGGGAACAGATGG-3′, adapter B-upper strand: 5′-BIO-TEG-CCTATCCCCTGTGTGCCTTGCCTATCCCC​TGTTGCGTGTCTCAGT-3′and adapter B-lower strand: 5′-P-CTGAGACACGCAACAGGGGATAGGCAAGGCACACA​GGGGATAGG-3′.Finally, the <em>Bst</em> DNA polymerase fill-in step of the published protocol was left out.</p>
<a id="article1.body1.sec4.sec2.p2" name="article1.body1.sec4.sec2.p2"></a><p>After library construction, a titration run was carried out using 16, 64, 256 and 512 copies per bead. The copies per bead ratio to be used in the titration run is estimated based on the concentration of sstDNA (single stranded AB library). Therefore, the outcome of the titration run determines the ratio which needs to be applied in the actual sequencing run. Based on the titration run carried out for this experiment, a 48 copies per bead ratio was selected, founded on the “Predicted Recovery” of approximately 2.10<sup>6</sup> enriched beads, &gt; 60% PassFilter, &lt;20% Mixed+Dots and approximately 6000 Keypass reads.</p>


<h4>GS 20 sequencing</h4>
<a id="article1.body1.sec4.sec3.p1" name="article1.body1.sec4.sec3.p1"></a><p>Emulsion PCR and bead enrichment were carried out according to the standard GS 20 protocol (Roche Applied Science). One full picotiterplate (PTP) (70×75 mm) with two regions was used. Enriched beads were divided over both regions. Sequencing was performed according to the manufacturer's instructions (Roche Applied Science).</p>


<h4>CRoPS bioinformatics pipeline</h4>
<a id="article1.body1.sec4.sec4.p1" name="article1.body1.sec4.sec4.p1"></a><p>The basecalled reads from both regions were added together in one file and further processed for SNP mining using a fully automated pipeline (Keygene N.V.). The web based pipeline was written in Perl 5.8.0 and runs via an Apache web server on a Linux platform. Microsoft Internet Explorer was used as client. An Oracle 10<em>g</em> relational database served as the central repository for all raw and processed data and the material and process definition.</p>
<a id="article1.body1.sec4.sec4.p2" name="article1.body1.sec4.sec4.p2"></a><p>The SNP discovery process consisted of four parts, namely (1) GS 20 data processing, (2) CRoPS pre-processing, (3) the CRoPS analysis, and (4) CRoPS SNP mining (<a href="#pone-0001172-g001">Fig. 1</a>).</p>
<a id="article1.body1.sec4.sec4.p3" name="article1.body1.sec4.sec4.p3"></a><p>GS 20 data processing was performed on-rig using the standard GS software. This process resulted in the “GS 20 raw sequence reads” that were directly used for further processing in the CRoPS pre-processing step. During pre-processing, the origin of the reads was identified according to their four base sample identification tags. The implementation of this step was based on the internal BLAST function in Oracle 10<em>g</em> (Oracle). Furthermore, the AFLP adapter sequences were trimmed. Pre-processed reads were saved to the database. In the CRoPS analysis step, reads were clustered and assembled using the TGICL tool <a href="#pone.0001172-Pertea1">[26]</a>. Clustering was performed using the following variable parameters: minimum percent identity for overlaps (94%), minimum overlap length (30 nt) and maximum length of unmatched overhangs (30 nt). Again, all data obtained were subsequently saved to the Oracle database.</p>
<a id="article1.body1.sec4.sec4.p4" name="article1.body1.sec4.sec4.p4"></a><p>Polymorphisms were selected during the SNP mining step. For each putative SNP a number of features were recorded, including relative position in the consensus sequence, sample count in the MSA, allele count per sample, the MSA depth (number of reads at the SNP position), distance to flanking SNPs, flanking size around SNP and the presence of a homopolymer stretch in which a SNP may occur. Mining rules were created from these features and defined as follows: 1) each allele should be present at least twice in a MSA, 2) SNPs should not be part of homopolymers larger than 3 bases, and 3) SNPs should be segregating according to sample origin. SNPs that passed the filters were selected as the best candidates for conversion into genotyping assays.</p>
<a id="article1.body1.sec4.sec4.p5" name="article1.body1.sec4.sec4.p5"></a><p>The pre-processed sequence data will be deposited at the NCBI Short Read Archive (SRA) as soon as this archive is ready to accept the data (expected at the end of 2007). Until then, the data can be requested from the authors.</p>


<h4>SNPWave</h4>
<a id="article1.body1.sec4.sec5.p1" name="article1.body1.sec4.sec5.p1"></a><p>Probes were designed for 30 putative SNPs in two multiplex (15-plex) SNPWave assays using ProbeDesigner software (Keygene N.V.) as described previously <a href="#pone.0001172-VanEijk1">[24]</a>. SNPWave reactions were carried out as described previously <a href="#pone.0001172-VanEijk1">[24]</a>. In short, ligation reactions were carried out in 10 µl volume containing 200–400 ng total genomic DNA, 1×Taq DNA ligase buffer [20 mM tris-HCl, 25 mM KAc, 10 mM MgAc<sub>2</sub>, 10 mM dithiothreitol (DTT), 1 mM NAD, 0.1% triton X-100; pH 7.6 at 25°C; New England Biolabs Inc], 2 U Taq DNA ligase (New England Biolabs Inc) and 0.5 fmol of each of the ligation probes. Next, 10 cycles of repeated denaturation, probe hybridization and ligation were performed in a Perkin Elmer 9700 thermal cycler (Applied Biosystems) using the following profile: initial denaturation for 2 min at 94°C, followed by 10 cycles of 15 s at 94°C and 60 min at 60°C, and storage at 4°C. After ligation, the mixture was diluted with 30 µl of 1×Taq DNA ligase buffer to 40 µl.</p>
<a id="article1.body1.sec4.sec5.p2" name="article1.body1.sec4.sec5.p2"></a><p>Ten µl of diluted ligation reaction was subsequently amplified in a 20 µl mixture containing 1×GeneAmp® PCR buffer (Applied Biosystems), 0.2 mM of each dNTP, 0.4 U AmpliTaq Gold DNA polymerase (Applied Biosystems), and 30 ng unlabeled forward primer (5-GACTGCGTACCAATTC-3) and 30 ng FAM-labeled reverse primer (5-GATGAGTCCTGAGTAA-3). The amplification profile was 12 min at 94°C, followed by 13 cycles of 30 s at 94°C, 30 s at 65°C with a reduction of 0.7°C per cycle to 56°C in cycle 13, followed by 1 min at 72°C. This was followed by 23 cycles of 30 s at 94°C, 30 s at 56°C and 1 min at 72°C, and storage at 4°C.</p>
<a id="article1.body1.sec4.sec5.p3" name="article1.body1.sec4.sec5.p3"></a><p>Purification of diluted SNPWave PCR products and subsequent detection on the MegaBACE 1000 (Amersham Biosciences/GE Healthcare Life Sciences) were as described previously <a href="#pone.0001172-VanEijk1">[24]</a>.</p>
<a id="article1.body1.sec4.sec5.p4" name="article1.body1.sec4.sec5.p4"></a><p>SNPs and flanking sequences can be found in the supplementary file (<a href="#pone.0001172.s001">Table S1</a>). Probe sequences are available upon request from the authors.</p>

</div>

<div id="section5" class="section"><a id="s5" name="s5" toc="s5" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0001172"></div><a name="pone.0001172.s001" id="pone.0001172.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001172.s001">Table S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1"></a><p class="preSiDOI">SNPs and flanking sequences used for the 13-plex and 10-plex SNPWave assays</p>
<p class="siDoi">doi:10.1371/journal.pone.0001172.s001</p><a id="article1.body1.sec5.supplementary-material1.caption1.p2" name="article1.body1.sec5.supplementary-material1.caption1.p2"></a><p class="postSiDOI">(0.04 MB DOC)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>We thank Rolf Mank and the staff of the MegaBACE unit at Keygene for excellent technical assistance.</p>
<a id="article1.back1.ack1.p2" name="article1.back1.ack1.p2"></a><p>The AFLP®, CRoPS™, SNPWave® and KeyGene™ SeqTag technologies are covered by patents and patent applications owned by Keygene N.V. AFLP and SNPWave are registered trademarks of Keygene NV. Applications for trademark registration for CRoPS and KeyGene have been filed by Keygene N.V. Other (registered) trademarks are the property of the respective owners.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: Nv RH Mv. Performed the experiments: SS EV HS Hv. Analyzed the data: AJ FY Jv. Contributed reagents/materials/analysis tools: AJ FY Jv HV. Wrote the paper: Nv Mv.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0001172-The1" id="pone.0001172-The1"></a>The Arabidopsis Initiative (2000) Analysis of the genome sequence of the flowering plant Arabidopsis thaliana. Nature  408: 796–815.  <ul class="find" data-citedArticleID="1017222" data-doi="10.1038/35048692"><li><a href="http://dx.doi.org/10.1038/35048692" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Analysis+of+the+genome+sequence+of+the+flowering+plant+Arabidopsis+thaliana." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Analysis+of+the+genome+sequence+of+the+flowering+plant+Arabidopsis+thaliana.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0001172-International1" id="pone.0001172-International1"></a>International Rice Genome Sequencing Project (2005) The map-based sequence of the rice genome. Nature  436: 793–800.  <ul class="find" data-citedArticleID="1017196" data-doi="10.1038/nature03895"><li><a href="http://dx.doi.org/10.1038/nature03895" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+map-based+sequence+of+the+rice+genome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+map-based+sequence+of+the+rice+genome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0001172-SanMiguel1" id="pone.0001172-SanMiguel1"></a>SanMiguel P, Tikhonov A, Jin Y-K, Motchoulskaia N, Zakharov D, et al.  (1996) Nested retrotransposons in the intergenic regions of the maize genome. Science  274: 765–768.  <ul class="find" data-citedArticleID="1017216" data-doi="10.1126/science.274.5288.765"><li><a href="http://dx.doi.org/10.1126/science.274.5288.765" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Nested+retrotransposons+in+the+intergenic+regions+of+the+maize+genome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Nested+retrotransposons+in+the+intergenic+regions+of+the+maize+genome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0001172-Li1" id="pone.0001172-Li1"></a>Li W, Zhang P, Fellers JP, Friebe B, Gill BS (2004) Sequence composition, organization, and evolution of the core Triticeae genome. Plant J.  40: 500–511.  <ul class="find" data-citedArticleID="1017200" data-doi="10.1111/j.1365-313x.2004.02228.x"><li><a href="http://dx.doi.org/10.1111/j.1365-313x.2004.02228.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Sequence+composition%2C+organization%2C+and+evolution+of+the+core+Triticeae+genome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Sequence+composition%2C+organization%2C+and+evolution+of+the+core+Triticeae+genome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0001172-Swaminathan1" id="pone.0001172-Swaminathan1"></a>Swaminathan K, Varala K, Hudson ME (2007) Global repeat discovery and estimation of genomic copy number in a large, complex genome using a high-throughput 454 sequence survey. BMC Genomics  8: 1471–2164.  <ul class="find" data-citedArticleID="1017220" data-doi="10.1186/1471-2164-8-132"><li><a href="http://dx.doi.org/10.1186/1471-2164-8-132" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Global+repeat+discovery+and+estimation+of+genomic+copy+number+in+a+large%2C+complex+genome+using+a+high-throughput+454+sequence+survey." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Global+repeat+discovery+and+estimation+of+genomic+copy+number+in+a+large%2C+complex+genome+using+a+high-throughput+454+sequence+survey.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0001172-An1" id="pone.0001172-An1"></a>An CS, Kim SC, Go SL (1996) Analysis of red pepper (<em>Capsicum annuum</em>) genome. J. Plant Biol.  39: 57–61.  <ul class="find" data-citedArticleID="1017182"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Analysis+of+red+pepper+%28Capsicum+annuum%29+genome.&amp;auth=&amp;atitle=Analysis+of+red+pepper+%28Capsicum+annuum%29+genome." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Analysis+of+red+pepper+%28Capsicum+annuum%29+genome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Analysis+of+red+pepper+%28Capsicum+annuum%29+genome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0001172-Oliphant1" id="pone.0001172-Oliphant1"></a>Oliphant A, Barker DL, Stuelpnagel JR, Chee MS (2002) BeadArray™ technology: enabling an accurate, cost-effective approach to high-throughput genotyping. Biotechniques  32: S56–S61.  <ul class="find" data-citedArticleID="1017208"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=BeadArray%3F+technology%3A+enabling+an+accurate%2C+cost-effective+approach+to+high-throughput+genotyping.&amp;auth=&amp;atitle=BeadArray%3F+technology%3A+enabling+an+accurate%2C+cost-effective+approach+to+high-throughput+genotyping." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=BeadArray%E2%84%A2+technology%3A+enabling+an+accurate%2C+cost-effective+approach+to+high-throughput+genotyping." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22BeadArray%E2%84%A2+technology%3A+enabling+an+accurate%2C+cost-effective+approach+to+high-throughput+genotyping.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0001172-Hardenbol1" id="pone.0001172-Hardenbol1"></a>Hardenbol P, Banér J, Jain M, Nilsson M, Namsaraev E, et al.  (2003) Multiplexed genotyping with sequence-tagged molecular inversion probes. Nat. Biotechnol.  21: 673–678.  <ul class="find" data-citedArticleID="1017194" data-doi="10.1038/nbt821"><li><a href="http://dx.doi.org/10.1038/nbt821" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Multiplexed+genotyping+with+sequence-tagged+molecular+inversion+probes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Multiplexed+genotyping+with+sequence-tagged+molecular+inversion+probes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0001172-Kennedy1" id="pone.0001172-Kennedy1"></a>Kennedy GC, Matsuzaki H, Dong S, Liu W-M, Huang J, et al.  (2003) Large-scale genotyping of complex DNA. Nat. Biotechnol.  21: 1233–1237.  <ul class="find" data-citedArticleID="1017198" data-doi="10.1038/nbt869"><li><a href="http://dx.doi.org/10.1038/nbt869" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Large-scale+genotyping+of+complex+DNA." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Large-scale+genotyping+of+complex+DNA.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0001172-Matsuzaki1" id="pone.0001172-Matsuzaki1"></a>Matsuzaki H, Loi H, Dong S, Tsai Y-Y, Fang J, et al.  (2004) Parallel genotyping of over 10,000 SNPs using a one-primer assay on a high-density oligonucleotide array. Genome Res.  14: 414–425.  <ul class="find" data-citedArticleID="1017204" data-doi="10.1101/gr.2014904"><li><a href="http://dx.doi.org/10.1101/gr.2014904" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Parallel+genotyping+of+over+10%2C000+SNPs+using+a+one-primer+assay+on+a+high-density+oligonucleotide+array." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Parallel+genotyping+of+over+10%2C000+SNPs+using+a+one-primer+assay+on+a+high-density+oligonucleotide+array.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0001172-The2" id="pone.0001172-The2"></a>The international HapMap Consortium (2005) A haplotype map of the human genome. Nature  437: 1299–1320.  <ul class="find" data-citedArticleID="1017224" data-doi="10.1038/nature04226"><li><a href="http://dx.doi.org/10.1038/nature04226" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+haplotype+map+of+the+human+genome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+haplotype+map+of+the+human+genome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0001172-Thomas1" id="pone.0001172-Thomas1"></a>Thomas RK, Nickerson E, Simons JF, Jänne PA, Tengs T, et al.  (2006) Sensitive mutation detection in heterogeneous cancer specimens by massive parallel picoliter reactor sequencing. Nat. Med.  12: 852–855.  <ul class="find" data-citedArticleID="1017226" data-doi="10.1038/nm1437"><li><a href="http://dx.doi.org/10.1038/nm1437" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Sensitive+mutation+detection+in+heterogeneous+cancer+specimens+by+massive+parallel+picoliter+reactor+sequencing." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Sensitive+mutation+detection+in+heterogeneous+cancer+specimens+by+massive+parallel+picoliter+reactor+sequencing.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0001172-Arumuganathan1" id="pone.0001172-Arumuganathan1"></a>Arumuganathan K, Earle ED (1991) Nuclear DNA content of some important plant species. Plant Mol. Biol. Rep.  9: 208.  <ul class="find" data-citedArticleID="1017184" data-doi="10.1007/bf02672069"><li><a href="http://dx.doi.org/10.1007/bf02672069" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Nuclear+DNA+content+of+some+important+plant+species." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Nuclear+DNA+content+of+some+important+plant+species.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0001172-Meyers1" id="pone.0001172-Meyers1"></a>Meyers BC, Tingey SV, Morgante M (2001) Abundance, distribution, and transcriptional activity of repetitive elements in the maize genome. Genome Res.  11: 1660–1676.  <ul class="find" data-citedArticleID="1017206" data-doi="10.1101/gr.188201"><li><a href="http://dx.doi.org/10.1101/gr.188201" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Abundance%2C+distribution%2C+and+transcriptional+activity+of+repetitive+elements+in+the+maize+genome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Abundance%2C+distribution%2C+and+transcriptional+activity+of+repetitive+elements+in+the+maize+genome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0001172-Peterson1" id="pone.0001172-Peterson1"></a>Peterson DG, Wessler SR, Paterson AH (2002) Efficient capture of unique sequences from eukaryotic genomes. Trends Genet.  18: 547–550.  <ul class="find" data-citedArticleID="1017212" data-doi="10.1016/s0168-9525(02)02764-6"><li><a href="http://dx.doi.org/10.1016/s0168-9525(02)02764-6" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Efficient+capture+of+unique+sequences+from+eukaryotic+genomes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Efficient+capture+of+unique+sequences+from+eukaryotic+genomes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0001172-Whitelaw1" id="pone.0001172-Whitelaw1"></a>Whitelaw CA, Barbazuk WB, Pertea G, Chan AP, Cheung F, et al.  (2003) Enrichment of gene-coding sequences in maize by genome filtration. Science  302: 2118–2120.  <ul class="find" data-citedArticleID="1017236" data-doi="10.1126/science.1090047"><li><a href="http://dx.doi.org/10.1126/science.1090047" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Enrichment+of+gene-coding+sequences+in+maize+by+genome+filtration." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Enrichment+of+gene-coding+sequences+in+maize+by+genome+filtration.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0001172-Emberton1" id="pone.0001172-Emberton1"></a>Emberton J, Ma J, Yuan Y, SanMiguel P, Bennetzen JL (2005) Gene enrichment in maize with hypomethylated partial restriction (HMPR) libraries. Genome Res.  15: 1441–1446.  <ul class="find" data-citedArticleID="1017192" data-doi="10.1101/gr.3362105"><li><a href="http://dx.doi.org/10.1101/gr.3362105" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Gene+enrichment+in+maize+with+hypomethylated+partial+restriction+%28HMPR%29+libraries." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Gene+enrichment+in+maize+with+hypomethylated+partial+restriction+%28HMPR%29+libraries.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0001172-Bennetzen1" id="pone.0001172-Bennetzen1"></a>Bennetzen JL, Schrick K, Springer PS, Brown WE, SanMiguel P (1994) Active maize genes are unmodified and flanked by diverse classes of modified, highly repetitive DNA. Genome  37: 565–576.  <ul class="find" data-citedArticleID="1017188" data-doi="10.1139/g94-081"><li><a href="http://dx.doi.org/10.1139/g94-081" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Active+maize+genes+are+unmodified+and+flanked+by+diverse+classes+of+modified%2C+highly+repetitive+DNA." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Active+maize+genes+are+unmodified+and+flanked+by+diverse+classes+of+modified%2C+highly+repetitive+DNA.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0001172-Rabinowicz1" id="pone.0001172-Rabinowicz1"></a>Rabinowicz PD, Schutz K, Dedhia N, Yordan C, Parnell LD, et al.  (1999) Differential methylation of genes and retrotransposons facilitates shotgun sequencing of the maize genome. Nature Genet.  23: 305–308.  <ul class="find" data-citedArticleID="1017214" data-doi="10.1038/15479"><li><a href="http://dx.doi.org/10.1038/15479" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Differential+methylation+of+genes+and+retrotransposons+facilitates+shotgun+sequencing+of+the+maize+genome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Differential+methylation+of+genes+and+retrotransposons+facilitates+shotgun+sequencing+of+the+maize+genome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0001172-Vos1" id="pone.0001172-Vos1"></a>Vos P, Hogers R, Bleeker M, Reijans M, van de Lee T, et al.  (1995) AFLP: a new technique for DNA fingerprinting. Nucleic Acids Res.  23: 4407–4414.  <ul class="find" data-citedArticleID="1017230" data-doi="10.1093/nar/23.21.4407"><li><a href="http://dx.doi.org/10.1093/nar/23.21.4407" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=AFLP%3A+a+new+technique+for+DNA+fingerprinting." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22AFLP%3A+a+new+technique+for+DNA+fingerprinting.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0001172-Zabeau1" id="pone.0001172-Zabeau1"></a>Zabeau M, Vos P (1993) Selective restriction fragment amplification; a general method for DNA fingerprinting. EP 0534858-A1,B1; US patent 6045994..   <ul class="find" data-citedArticleID="1017238"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Selective+restriction+fragment+amplification%3B+a+general+method+for+DNA+fingerprinting.&amp;auth=&amp;atitle=Selective+restriction+fragment+amplification%3B+a+general+method+for+DNA+fingerprinting." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Selective+restriction+fragment+amplification%3B+a+general+method+for+DNA+fingerprinting." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Selective+restriction+fragment+amplification%3B+a+general+method+for+DNA+fingerprinting.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0001172-Vuylsteke1" id="pone.0001172-Vuylsteke1"></a>Vuylsteke M, Peleman JD, van Eijk MJT (2007) AFLP technology for DNA fingerprinting. Nature protocols  2: 1387–1398.  <ul class="find" data-citedArticleID="1017232" data-doi="10.1038/nprot.2007.175"><li><a href="http://dx.doi.org/10.1038/nprot.2007.175" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=AFLP+technology+for+DNA+fingerprinting." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22AFLP+technology+for+DNA+fingerprinting.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0001172-Margulies1" id="pone.0001172-Margulies1"></a>Margulies M, Egholm M, Altman WE, Attiya S, Bader JS, et al.  (2005) Genome sequencing in microfabricated high-density picolitre reactors. Nature  437: 376–380.  <ul class="find" data-citedArticleID="1017202"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Genome+sequencing+in+microfabricated+high-density+picolitre+reactors.&amp;auth=&amp;atitle=Genome+sequencing+in+microfabricated+high-density+picolitre+reactors." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genome+sequencing+in+microfabricated+high-density+picolitre+reactors." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genome+sequencing+in+microfabricated+high-density+picolitre+reactors.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0001172-VanEijk1" id="pone.0001172-VanEijk1"></a>Van Eijk MJT, Broekhof JLN, van der Poel HJA, Hogers RCJ, Schneiders H, et al.  (2004) SNPWave™: a flexible multiplexed SNP genotyping technology. Nucleic Acids Res.  32: e47.  <ul class="find" data-citedArticleID="1017228"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=SNPWave%3F%3A+a+flexible+multiplexed+SNP+genotyping+technology.&amp;auth=&amp;atitle=SNPWave%3F%3A+a+flexible+multiplexed+SNP+genotyping+technology." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=SNPWave%E2%84%A2%3A+a+flexible+multiplexed+SNP+genotyping+technology." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22SNPWave%E2%84%A2%3A+a+flexible+multiplexed+SNP+genotyping+technology.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0001172-Weber1" id="pone.0001172-Weber1"></a>Weber APM, Weber KL, Carr K, Wilkerson C, Ohlrogge JB (2007) Sampling the Arabidopsis transcriptome with massively parallel pyrosequencing. Plant Physiol.  144: 32–42.  <ul class="find" data-citedArticleID="1017234" data-doi="10.1104/pp.107.096677"><li><a href="http://dx.doi.org/10.1104/pp.107.096677" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Sampling+the+Arabidopsis+transcriptome+with+massively+parallel+pyrosequencing." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Sampling+the+Arabidopsis+transcriptome+with+massively+parallel+pyrosequencing.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0001172-Pertea1" id="pone.0001172-Pertea1"></a>Pertea G, Huang X, Liang F, Antonescu V, Sultana R, et al.  (2003) TIGR Gene Indices clustering tools (TGICL): a software system for fast clustering of large EST datasets. Bioinformatics  19: 651–652.  <ul class="find" data-citedArticleID="1017210" data-doi="10.1093/bioinformatics/btg034"><li><a href="http://dx.doi.org/10.1093/bioinformatics/btg034" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=TIGR+Gene+Indices+clustering+tools+%28TGICL%29%3A+a+software+system+for+fast+clustering+of+large+EST+datasets." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22TIGR+Gene+Indices+clustering+tools+%28TGICL%29%3A+a+software+system+for+fast+clustering+of+large+EST+datasets.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0001172-Binladen1" id="pone.0001172-Binladen1"></a>Binladen J, Gilbert MTP, Bollback JP, Panitz F, Bendixen C, et al.  (2007) The use of coded PCR primers enables high-throughput sequencing of multiple homolog amplification products by 454 parallel sequencing. Plos One  2: e197.  <ul class="find" data-citedArticleID="1017190" data-doi="10.1371/journal.pone.0000197"><li><a href="http://dx.doi.org/10.1371/journal.pone.0000197" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+use+of+coded+PCR+primers+enables+high-throughput+sequencing+of+multiple+homolog+amplification+products+by+454+parallel+sequencing." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+use+of+coded+PCR+primers+enables+high-throughput+sequencing+of+multiple+homolog+amplification+products+by+454+parallel+sequencing.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0001172-Barbazuk1" id="pone.0001172-Barbazuk1"></a>Barbazuk WB, Emrich SJ, Chen HD, Schnable PS (2007) SNP discovery via 454 transcriptome sequencing. Plant J.  51: 910–918.  <ul class="find" data-citedArticleID="1017186" data-doi="10.1111/j.1365-313x.2007.03193.x"><li><a href="http://dx.doi.org/10.1111/j.1365-313x.2007.03193.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=SNP+discovery+via+454+transcriptome+sequencing." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22SNP+discovery+via+454+transcriptome+sequencing.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0001172-Stuart1" id="pone.0001172-Stuart1"></a>Stuart CN, Via LE (1993) A rapid CTAB DNA isolation technique useful for RAPD fingerprinting and other PCR applications. Biotechniques  14: 748–750.  <ul class="find" data-citedArticleID="1017218"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=A+rapid+CTAB+DNA+isolation+technique+useful+for+RAPD+fingerprinting+and+other+PCR+applications.&amp;auth=&amp;atitle=A+rapid+CTAB+DNA+isolation+technique+useful+for+RAPD+fingerprinting+and+other+PCR+applications." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+rapid+CTAB+DNA+isolation+technique+useful+for+RAPD+fingerprinting+and+other+PCR+applications." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+rapid+CTAB+DNA+isolation+technique+useful+for+RAPD+fingerprinting+and+other+PCR+applications.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001172.XML" value="69751"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001172.PDF" value="694574"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001172.t001.PNG_L" value="132182"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001172.t001.PNG_M" value="159930"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001172.t001.PNG_S" value="19111"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001172.t001.TIF" value="610206"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001172.t001.PNG_I" value="65035"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001172.g001.PNG_L" value="96604"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001172.g001.PNG_M" value="66258"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001172.g001.PNG_S" value="10373"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001172.g001.TIF" value="191330"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001172.g001.PNG_I" value="23382"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001172.g002.PNG_L" value="641719"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001172.g002.PNG_M" value="283658"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001172.g002.PNG_S" value="16188"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001172.g002.TIF" value="1652050"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001172.g002.PNG_I" value="121657"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001172.g003.PNG_L" value="627942"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001172.g003.PNG_M" value="103196"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001172.g003.PNG_S" value="11687"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001172.g003.TIF" value="1895026"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001172.g003.PNG_I" value="36029"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001172.g004.PNG_L" value="664272"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001172.g004.PNG_M" value="74839"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001172.g004.PNG_S" value="10739"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001172.g004.TIF" value="722438"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001172.g005.PNG_L" value="73771"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001172.g005.PNG_M" value="70738"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001172.g005.PNG_S" value="10877"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001172.g005.TIF" value="475178"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001172.g005.PNG_I" value="47670"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001172.g006.PNG_L" value="93727"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001172.g006.PNG_M" value="88776"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001172.g006.PNG_S" value="12432"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001172.g006.TIF" value="560224"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001172.g006.PNG_I" value="71477"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001172.s001.DOC" value="43520"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001172&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001172" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001172&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0001172&volume=&issue=&title=Complexity Reduction of Polymorphic Sequences (CRoPS™): A Novel Approach for Large-Scale Polymorphism Discovery in Complex Genomes&author_name=Nathalie%20J.%20van%20Orsouw%2C%20Ren%C3%A9%20C.%20J.%20Hogers%2C%20Antoine%20Janssen%2C%20Feyruz%20Yalcin%2C%20Sandor%20Snoeijers%2C%20Esther%20Verstege%2C%20Harrie%20Schneiders%2C%20Hein%20van%20der%20Poel%2C%20Jan%20van%20Oeveren%2C%20Harold%20Verstegen%2C%20Michiel%20J.%20T.%20van%20Eijk&start_page=1&end_page=10" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001172" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001172" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0001172" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001172&amp;t=Complexity%20Reduction%20of%20Polymorphic%20Sequences%20(CRoPS%E2%84%A2)%3A%20A%20Novel%20Approach%20for%20Large-Scale%20Polymorphism%20Discovery%20in%20Complex%20Genomes" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001172&title=Complexity%20Reduction%20of%20Polymorphic%20Sequences%20(CRoPS%E2%84%A2)%3A%20A%20Novel%20Approach%20for%20Large-Scale%20Polymorphism%20Discovery%20in%20Complex%20Genomes&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001172&amp;title=Complexity%20Reduction%20of%20Polymorphic%20Sequences%20(CRoPS%E2%84%A2)%3A%20A%20Novel%20Approach%20for%20Large-Scale%20Polymorphism%20Discovery%20in%20Complex%20Genomes" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001172" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0001172&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Complexity Reduction of Polymorphic Sequences (CRoPS™): A Novel Approach for Large-Scale Polymorphism Discovery in Complex Genomes';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0001172';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Complexity%20Reduction%20of%20Polymorphic%20Sequences%20(CRoPS%E2%84%A2)%3A%20A%20Novel%20Approach%20for%20Large-Scale%20Polymorphism%20Discovery%20in%20Complex%20Genomes http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001172" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0001172" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">



















          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Amplified+fragment+length+polymorphism%22" title="Search for articles in the subject area:'Amplified fragment length polymorphism'"><div class="flagText">Amplified fragment length polymorphism</div></a>
              <div data-categoryid="47885" data-articleid="25176"
                   data-categoryname="Amplified fragment length polymorphism"
                   class="flagImage" title="Flag 'Amplified fragment length polymorphism' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22DNA+sequences%22" title="Search for articles in the subject area:'DNA sequences'"><div class="flagText">DNA sequences</div></a>
              <div data-categoryid="32003" data-articleid="25176"
                   data-categoryname="DNA sequences"
                   class="flagImage" title="Flag 'DNA sequences' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Maize%22" title="Search for articles in the subject area:'Maize'"><div class="flagText">Maize</div></a>
              <div data-categoryid="43001" data-articleid="25176"
                   data-categoryname="Maize"
                   class="flagImage" title="Flag 'Maize' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Multiple+alignment+calculation%22" title="Search for articles in the subject area:'Multiple alignment calculation'"><div class="flagText">Multiple alignment calculation</div></a>
              <div data-categoryid="20427" data-articleid="25176"
                   data-categoryname="Multiple alignment calculation"
                   class="flagImage" title="Flag 'Multiple alignment calculation' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Plant+genomics%22" title="Search for articles in the subject area:'Plant genomics'"><div class="flagText">Plant genomics</div></a>
              <div data-categoryid="33365" data-articleid="25176"
                   data-categoryname="Plant genomics"
                   class="flagImage" title="Flag 'Plant genomics' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Polymerase+chain+reaction%22" title="Search for articles in the subject area:'Polymerase chain reaction'"><div class="flagText">Polymerase chain reaction</div></a>
              <div data-categoryid="26251" data-articleid="25176"
                   data-categoryname="Polymerase chain reaction"
                   class="flagImage" title="Flag 'Polymerase chain reaction' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Sequence+alignment%22" title="Search for articles in the subject area:'Sequence alignment'"><div class="flagText">Sequence alignment</div></a>
              <div data-categoryid="18051" data-articleid="25176"
                   data-categoryname="Sequence alignment"
                   class="flagImage" title="Flag 'Sequence alignment' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Sequence+assembly+tools%22" title="Search for articles in the subject area:'Sequence assembly tools'"><div class="flagText">Sequence assembly tools</div></a>
              <div data-categoryid="40553" data-articleid="25176"
                   data-categoryname="Sequence assembly tools"
                   class="flagImage" title="Flag 'Sequence assembly tools' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=5664'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=2193'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=3465&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>

<div class="block sidebar-comments">
    <div class="header">
        <h3>Comments</h3>
    </div>
      <p><a href="/annotation/listThread.action?root=21591">Referee Comments: Referee 1 (J. Antoni Rafalski)</a><br>Posted by PLoS_ONE_Group</p>
</div>

</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
