

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Heteropolymeric Triplex-Based Genomic Assay® to Detect Pathogens or Single-Nucleotide Polymorphisms in Human Genomic Samples</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0000305"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0000305" />

    <meta name="citation_title" content="Heteropolymeric Triplex-Based Genomic Assay® to Detect Pathogens or Single-Nucleotide Polymorphisms in Human Genomic Samples"/>
    <meta itemprop="name" content="Heteropolymeric Triplex-Based Genomic Assay® to Detect Pathogens or Single-Nucleotide Polymorphisms in Human Genomic Samples"/>

      <meta name="citation_author" content="Jasmine I. Daksis"/>
            <meta name="citation_author_institution" content="Ingeneus Research, Mississauga, Ontario, Canada"/>
      <meta name="citation_author" content="Glen H. Erikson"/>
            <meta name="citation_author_institution" content="Ingeneus Research, Mississauga, Ontario, Canada"/>

    <meta name="citation_date" content="2007/3/21"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0000305.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e305"/>
    <meta name="citation_issue" content="3"/>
    <meta name="citation_volume" content="2"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=DNA supercoiling and unusual structures.; citation_author=MD Frank-Kamenetskii; citation_number=1; citation_pages=185-215; citation_date=1990; citation_publisher=Cold Spring Harbor Laboratory Press; " />
      <meta name="citation_reference" content="citation_title=Triple helix formation inhibits transcription elongation in vitro.; citation_author=SL Young; citation_author=SH Krawczyk; citation_author=MD Matteucci; citation_author=JJ Toole; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=88; citation_number=2; citation_pages=10023-10026; citation_date=1991; " />
      <meta name="citation_reference" content="citation_title=A triple helix-forming oligonucleotide-intercalator conjugate acts as a transcriptional repressor via inhibition of NF kappa B binding to interlelukin-2 receptor alpha-regulatory sequence.; citation_author=M Grigoriev; citation_author=D Praseuth; citation_author=P Robin; citation_author=A Hemar; citation_author=T Saison-Behmoaras; citation_journal_title=J Biol Chem; citation_volume=267; citation_number=3; citation_pages=3389-3395; citation_date=1992; " />
      <meta name="citation_reference" content="citation_title=Control of gene expression by triple helix-forming oligonucleotides. The antigene strategy.; citation_author=C Helene; citation_author=NT Thuong; citation_author=A Harel-Bellan; citation_journal_title=Ann NY Acad Sci; citation_volume=660; citation_number=4; citation_pages=27-36; citation_date=1992; " />
      <meta name="citation_reference" content="citation_title=Analysis of promoter-specific repression by triple-helical DNA complexes in a eukaryotic cell-free transcription system.; citation_author=LJ Maher; citation_author=PB Dervan; citation_author=B Wold; citation_journal_title=Biochemistry; citation_volume=31; citation_number=5; citation_pages=70-81; citation_date=1992; " />
      <meta name="citation_reference" content="citation_title=Triplex DNA: fundamentals, advances, and potential applications for gene therapy.; citation_author=PP Chan; citation_author=PM Glazer; citation_journal_title=J Mol Med; citation_volume=75; citation_number=6; citation_pages=267-282; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Triplex-directed modification of genes and gene activity.; citation_author=KM Vasquez; citation_author=JH Wilson; citation_journal_title=Trends Biochem Sci; citation_volume=23; citation_number=7; citation_pages=4-9; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=The potential for gene repair via triple helix formation.; citation_author=MM Seidman; citation_author=PM Glazer; citation_journal_title=J Clin Invest; citation_volume=112; citation_number=8; citation_pages=487-494; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Oligodeoxyribonucleotide length and sequence effects on intermolecular purine-purine-pyrimidine triple-helix formation.; citation_author=AJ Cheng; citation_author=MW Van Dyke; citation_journal_title=Nucleic Acids Res; citation_volume=22; citation_number=9; citation_pages=4742-4747; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=Sequence specificity in triple-helix formation: experimental and theoretical studies of the effect of mismatches on triplex stability.; citation_author=JL Mergny; citation_author=JS Sun; citation_author=M Rougee; citation_author=T Montenay-Garestier; citation_author=F Barcelo; citation_journal_title=Biochemistry; citation_volume=30; citation_number=10; citation_pages=9791-9798; citation_date=1991; " />
      <meta name="citation_reference" content="citation_title=Influence of pH on the equilibrium association constants for oligodeoxyribonucleotide-directed triple helix formation at single DNA sites.; citation_author=SF Singleton; citation_author=PB Dervan; citation_journal_title=Biochemistry; citation_volume=31; citation_number=11; citation_pages=10995-11003; citation_date=1992; " />
      <meta name="citation_reference" content="citation_title=Triplex DNA structures.; citation_author=MD Frank-Kamenetskii; citation_author=SM Mirkin; citation_journal_title=Annu Rev Biochem; citation_volume=64; citation_number=12; citation_pages=65-95; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Sequence-dependent stability of intramolecular DNA triple helices.; citation_author=D Leitner; citation_author=K Weisz; citation_journal_title=J Biomolec Struc and Dynamics; citation_volume=17; citation_number=13; citation_pages=993-1000; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Specific triplex binding capacity of mixed base sequence duplex nucleic acids used for single-nucleotide polymorphism detection.; citation_author=JI Daksis; citation_author=GH Erikson; citation_journal_title=Genetic Testing; citation_volume=9; citation_number=14; citation_pages=111-120; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=External quality assurance program for PCR amplification of genomic DNA: an Italian experience.; citation_author=CC Raggi; citation_author=P Pinzani; citation_author=A Paradiso; citation_author=M Pazzagli; citation_author=C Orlando; citation_journal_title=Clin Chem; citation_volume=49; citation_number=15; citation_pages=782-791; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Generic scheme for independent performance assessment in the molecular biology laboratory.; citation_author=L Birch; citation_author=CA English; citation_author=M Burns; citation_author=JT Keer; citation_journal_title=Clin Chem; citation_volume=50; citation_number=16; citation_pages=1553-1559; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=The drosophila erg K+ channel polypeptide is encoded by the seizure locus.; citation_author=SA Titus; citation_author=JW Warmke; citation_author=B Ganetzky; citation_journal_title=J Neurosci; citation_volume=17; citation_number=17; citation_pages=875-881; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=A molecular basis for cardiac arrhythmia: HERG mutations cause long QT syndrome.; citation_author=ME Curran; citation_author=I Splawski; citation_author=KW Timothy; citation_author=M Vincent; citation_author=ED Green; citation_journal_title=Cell; citation_volume=80; citation_number=18; citation_pages=795-804; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Laboratory standards and guidelines for population-based cystic fibrosis carrier screening.; citation_author=WW Grody; citation_author=GR Cutting; citation_author=KW Klinger; citation_author=CS Richards; citation_author=MS Watson; citation_journal_title=Genet Med; citation_volume=3; citation_number=19; citation_pages=149-154; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Mutation in blood coagulation factor V associated with resistance to activated protein C.; citation_author=RM Bertina; citation_author=BPC Koeleman; citation_author=T Koster; citation_author=FR Rosendaal; citation_author=RJ Dirven; citation_journal_title=Nature; citation_volume=369; citation_number=20; citation_pages=64-67; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=Resistance to activated protein C as a basis for venous thrombosis.; citation_author=PJ Svensson; citation_author=B Dahlback; citation_journal_title=N Engl J Med; citation_volume=330; citation_number=21; citation_pages=517-522; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=Human methylenetetrahydrofolate reductase: isolation of cDNA, mapping and mutation identification.; citation_author=P Goyette; citation_author=JS Sumner; citation_author=R Milos; citation_author=AM Duncan; citation_author=DS Rosenblatt; citation_journal_title=Nat Genet; citation_volume=7; citation_number=22; citation_pages=195-200; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=1H NMR studies of the bis-intercalation of a homodimeric oxazole yellow dye in DNA oligonucleotides.; citation_author=F Johansen; citation_author=JP Jacobsen; citation_journal_title=J Biomolec Struc and Dynamics; citation_volume=16; citation_number=23; citation_pages=205-222; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=The elongation of duplex DNA in complexes with RecA protein.; citation_author=A Stasiak; citation_author=E Di Capua; citation_author=T Koller; citation_journal_title=J Mol Biol; citation_volume=151; citation_number=24; citation_pages=557-564; citation_date=1981; " />
      <meta name="citation_reference" content="citation_title=What do x-ray crystallographic and electron microscopic structural studies of the RecA protein tell us about recombination?; citation_author=EH Egelman; citation_journal_title=Curr Opin Struct Biol; citation_volume=3; citation_number=25; citation_pages=189-197; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=A parallel DNA triplex as a model for the intermediate in homologous recombination.; citation_author=VB Zhurkin; citation_author=G Raghunathan; citation_author=NB Ulyanov; citation_author=RD Camerini-Otero; citation_author=RL Jernigan; citation_journal_title=J Mol Biol; citation_volume=239; citation_number=26; citation_pages=181-200; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=Probing the structure of a putative intermediate in homologous recombination: the third strand in the parallel DNA triplex is in contact with the major groove of the duplex.; citation_author=MG Kim; citation_author=VB Zhurkin; citation_author=RL Jernigan; citation_author=RD Camerini-Otero; citation_journal_title=J Mol Biol; citation_volume=247; citation_number=27; citation_pages=874-889; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=DNA stretching and compression: large-scale simulations of double helical structures.; citation_author=KM Kosikov; citation_author=AA Gorin; citation_author=VB Zhurkin; citation_author=WK Olson; citation_journal_title=J Mol Biol; citation_volume=289; citation_number=28; citation_pages=1301-1326; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Radioprobing of a RecA-three-stranded DNA complex with iodine 125: evidence for recognition of homology in the major groove of the target duplex.; citation_author=VA Malkov; citation_author=IG Panyutin; citation_author=RD Neumann; citation_author=VB Zhurkin; citation_author=RD Camerini-Otero; citation_journal_title=J Mol Biol; citation_volume=299; citation_number=29; citation_pages=629-640; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=A mechanism for RecA-promoted sequence homology recognition and strand exchange between single-stranded DNA and duplex DNA, via triple-helical intermediates.; citation_author=G Bertucat; citation_author=R Lavery; citation_author=C Prevost; citation_journal_title=J Biomolec Struc and Dynamics Conversation; citation_volume=11; citation_number=30; citation_pages=147-153; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Competitive hybridization kinetics reveals unexpected behavior patterns.; citation_author=Y Zhang; citation_author=DA Hammer; citation_author=DJ Graves; citation_journal_title=Biophysical J; citation_volume=89; citation_number=31; citation_pages=2950-2959; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Crystal structure of restriction endonuclease BglI bound to its interrupted DNA recognition sequence.; citation_author=M Newman; citation_author=K Lunnen; citation_author=G Wilson; citation_author=J Greci; citation_author=I Schildkraut; citation_journal_title=EMBO J; citation_volume=17; citation_number=32; citation_pages=5466-5476; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Genomic DNA sequence of the cystic fibrosis transmembrane conductance regulator (CFTR) gene.; citation_author=J Zielenski; citation_author=R Rozmahel; citation_author=D Bozon; citation_author=B Kerem; citation_author=Z Grzelczak; citation_journal_title=Genomics; citation_volume=10; citation_number=33; citation_pages=214-228; citation_date=1991; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000305" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Heteropolymeric Triplex-Based Genomic Assay® to Detect Pathogens or Single-Nucleotide Polymorphisms in Human Genomic Samples"/>
    <meta name="twitter:description" content="Human genomic samples are complex and are considered difficult to assay directly without denaturation or PCR amplification. We report the use of a base-specific heteropolymeric triplex, formed by native duplex genomic target and an oligonucleotide third strand probe, to assay for low copy pathogen genomes present in a sample also containing human genomic duplex DNA, or to assay human genomic duplex DNA for Single Nucleotide Polymorphisms (SNP), without PCR amplification. Wild-type and mutant probes are used to identify triplexes containing FVL G1691A, MTHFR C677T and CFTR mutations. The specific triplex structure forms rapidly at room temperature in solution and may be detected without a separation step. YOYO-1, a fluorescent bis-intercalator, promotes and signals the formation of the specific triplex. Genomic duplexes may be assayed homogeneously with single base pair resolution. The specific triple-stranded structures of the assay may approximate homologous recombination intermediates, which various models suggest may form in either the major or minor groove of the duplex. The bases of the stable duplex target are rendered specifically reactive to the bases of the probe because of the activity of intercalated YOYO-1, which is known to decondense duplex locally 1.3 fold. This may approximate the local decondensation effected by recombination proteins such as RecA in vivo. Our assay, while involving triplex formation, is sui generis, as it is not homopurine sequence-dependent, as are &ldquo;canonical triplexes&rdquo;. Rather, the base pair-specific heteropolymeric triplex of the assay is conformation-dependent. The highly sensitive diagnostic assay we present allows for the direct detection of base sequence in genomic duplex samples, including those containing human genomic duplex DNA, thereby bypassing the inherent problems and cost associated with conventional PCR based diagnostic assays."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0000305.g005"/>

  <meta property="og:title" content="Heteropolymeric Triplex-Based Genomic Assay® to Detect Pathogens or Single-Nucleotide Polymorphisms in Human Genomic Samples" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000305" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=5080'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=955'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=5135&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0000305">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000305" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000305&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Heteropolymeric Triplex-Based Genomic Assay® to Detect Pathogens or Single-Nucleotide Polymorphisms in Human Genomic Samples
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Jasmine I. Daksis
              <span class="corresponding">mail</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:jasmine@ingeneus.net">jasmine@ingeneus.net</a></p>

                <p>Affiliation:
                  Ingeneus Research, Mississauga, Ontario, Canada
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Glen H. Erikson
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Ingeneus Research, Mississauga, Ontario, Canada
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: March 21, 2007</li>
    <li>DOI: 10.1371/journal.pone.0000305</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0000305" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0000305" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000305" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0000305" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0000305-g001" data-doi="info:doi/10.1371/journal.pone.0000305" data-uri="info:doi/10.1371/journal.pone.0000305.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000305.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000305-t001" data-doi="info:doi/10.1371/journal.pone.0000305" data-uri="info:doi/10.1371/journal.pone.0000305.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000305.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000305-g002" data-doi="info:doi/10.1371/journal.pone.0000305" data-uri="info:doi/10.1371/journal.pone.0000305.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000305.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000305-g003" data-doi="info:doi/10.1371/journal.pone.0000305" data-uri="info:doi/10.1371/journal.pone.0000305.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000305.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000305-t002" data-doi="info:doi/10.1371/journal.pone.0000305" data-uri="info:doi/10.1371/journal.pone.0000305.t002" title="Table 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000305.t002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000305-t003" data-doi="info:doi/10.1371/journal.pone.0000305" data-uri="info:doi/10.1371/journal.pone.0000305.t003" title="Table 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000305.t003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000305-g004" data-doi="info:doi/10.1371/journal.pone.0000305" data-uri="info:doi/10.1371/journal.pone.0000305.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000305.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000305-t004" data-doi="info:doi/10.1371/journal.pone.0000305" data-uri="info:doi/10.1371/journal.pone.0000305.t004" title="Table 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000305.t004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000305-g005" data-doi="info:doi/10.1371/journal.pone.0000305" data-uri="info:doi/10.1371/journal.pone.0000305.g005" title="Figure 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000305.g005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000305">Reader Comments (0)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0000305" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1"></a><p>Human genomic samples are complex and are considered difficult to assay directly without denaturation or PCR amplification. We report the use of a base-specific heteropolymeric triplex, formed by native duplex genomic target and an oligonucleotide third strand probe, to assay for low copy pathogen genomes present in a sample also containing human genomic duplex DNA, or to assay human genomic duplex DNA for Single Nucleotide Polymorphisms (SNP), without PCR amplification. Wild-type and mutant probes are used to identify triplexes containing <em>FVL</em> G1691A, <em>MTHFR</em> C677T and <em>CFTR</em> mutations. The specific triplex structure forms rapidly at room temperature in solution and may be detected without a separation step. YOYO-1, a fluorescent bis-intercalator, promotes and signals the formation of the specific triplex. Genomic duplexes may be assayed homogeneously with single base pair resolution. The specific triple-stranded structures of the assay may approximate homologous recombination intermediates, which various models suggest may form in either the major or minor groove of the duplex. The bases of the stable duplex target are rendered specifically reactive to the bases of the probe because of the activity of intercalated YOYO-1, which is known to decondense duplex locally 1.3 fold. This may approximate the local decondensation effected by recombination proteins such as RecA <em>in vivo</em>. Our assay, while involving triplex formation, is <em>sui generis</em>, as it is not homopurine sequence-dependent, as are “canonical triplexes”. Rather, the base pair-specific heteropolymeric triplex of the assay is conformation-dependent. The highly sensitive diagnostic assay we present allows for the direct detection of base sequence in genomic duplex samples, including those containing human genomic duplex DNA, thereby bypassing the inherent problems and cost associated with conventional PCR based diagnostic assays.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>Daksis JI, Erikson GH (2007) Heteropolymeric Triplex-Based Genomic Assay® to Detect Pathogens or Single-Nucleotide Polymorphisms in Human Genomic Samples. PLoS ONE 2(3):
          e305.
            doi:10.1371/journal.pone.0000305</p><p><strong>Academic Editor: </strong>Cecile Fairhead, Pasteur Institute, France</p><p><strong>Received:</strong> February 16, 2007; <strong>Accepted:</strong> February 25, 2007; <strong>Published:</strong> March 21, 2007</p><p><strong>Copyright:</strong> © 2007 Daksis, Erikson. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>Ingeneus Inc.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>Triplex nucleic acids have attracted attention due to their potential regulation of DNA transcription, replication, recombination, chromosome packing, and potential applications in gene therapy <a href="#pone.0000305-FrankKamenetskii1">[1]</a>–<a href="#pone.0000305-Seidman1">[8]</a>. The “canonical triplex” structures of these studies have consisted of long stretches of uninterrupted homopyrimidine or homopurine strands, typically 12–14 bases in length, that bind in the major groove of duplex DNA, forming Hoogsteen or reverse Hoogsteen hydrogen bonds with bases in the homopurine or purine-rich strand of the duplex. Substantial deviations from strict binding rules are understood to preclude triplex formation <a href="#pone.0000305-Cheng1">[9]</a>–<a href="#pone.0000305-Leitner1">[13]</a>. The utility of such “canonical triplexes” is severely limited due to the requirement of long uninterrupted homopurine sequences in the double-stranded DNA (dsDNA). Such sequence requirements have been understood to preclude the development of triplex-based molecular diagnostics.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>The discovery that dsDNA targets, under readily achieved conditions, may specifically bind third-strand DNA oligonucleotides of any base sequence, was accordingly a significant advance <a href="#pone.0000305-Daksis1">[14]</a>. Third-strand binding, facilitated and stabilized by YOYO-1, a cationic bis-intercalator, occurs within 5 minutes at room temperature with great facility, over a wide range of temperatures, under mild and varied incubation conditions and at physiological pH. Specific heteropolymeric triplex formation between non-denatured dsDNA and a ssDNA of mixed base sequence can occur without strand displacement, duplex invasion or the action of recombination proteins, such as recA. This triplex binding capability was used to demonstrate that the Triplex Assay® can accurately detect 1-, 2-, or 3-bp mutations, deletions or insertions in synthetic dsDNA targets and in homozygous or heterozygous polymerase chain reaction (PCR) amplified dsDNA amplicons of varied GC content <a href="#pone.0000305-Daksis1">[14]</a>.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>PCR amplification based molecular diagnostics are not optimal, due to complexity in optimization, inherent replication error rate, risk of contaminant amplification and cost. The predictable and unpredictable variables that negatively affect PCR amplification based diagnostic methods in real world settings have been recognized as significant <a href="#pone.0000305-Raggi1">[15]</a>, <a href="#pone.0000305-Birch1">[16]</a>. In this report we disclose our advances in SNP and pathogen detection in complex genomic samples using the Triplex Assay® technology. We demonstrate the Genomic Assay®, carried out homogeneously in bulk solution, whereby SNPs or pathogens in samples containing human genomic dsDNA are directly detected by fluorescent signal. We do not have recourse to PCR amplification.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3>
<h4>Genomic Assay® used to detect pathogens at low copy number</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>Molecular diagnostic testing for infectious diseases, bacterial and viral, commonly depend upon PCR based methodologies. High copy numbers of the pathogen, such as 10<sup>6</sup> to 10<sup>8</sup> colony forming units (CFU)/ml, may be required for successful PCR amplification by diagnostic labs <a href="#pone.0000305-Birch1">[16]</a>. Use of the Genomic Assay® to detect bacterial genomic dsDNA targets at low concentrations was investigated.</p>
<a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2"></a><p>Sequences of the ssDNA probes used are listed in <a href="#pone-0000305-t001">Table 1</a>. Perfectly matched 25-mer triplexes formed in reaction mixtures containing 100 copies of <em>Bacillus globigii</em> (BG) genomic dsDNA and probe bglIR-WT25C (probe 1) during a 5 minute incubation in the presence of 300 nM YOYO-1 (<a href="#pone-0000305-g001">Figure 1</a>). All incubations referred to in this report were carried out at room temperature (RT). Duplex DNA employed in our experiments was never subjected to denaturing conditions. The perfect match reaction mixture signal emission intensities were all significantly greater than the combined fluorescence signals emitted by target BG genomic dsDNA and YOYO-1 control intensities and probe and YOYO-1 control intensities (<a href="#pone.0000305.s007">Supplemental Table S1</a>). BG genomic dsDNA controls, which could not be detected at a 32% PMT setting on the Genexus Analyzer, were however detected at the more sensitive 36% PMT setting, consistent with the very low copy number of BG genomic DNA being assayed. To identify triplex-associated signal, all fluorescence emission values from reaction mixtures were corrected for background fluorescence by subtracting the appropriate probe control emission intensity. After initial assay, emissions from all reactions were monitored over time to obtain additional support for the diagnostic conclusions. The efficiency of triplex formation slightly improved during the first 35 minutes of incubation and remained relatively constant thereafter (<a href="#pone-0000305-g001">Figure 1</a>). Other BG genomic DNA samples were prepared as 10-fold serial dilutions, ranging in amount from 10<sup>6</sup> copies to 10<sup>2</sup> copies. All were successfully detected using probe 1 and YOYO-1 (data not shown).</p>
<div class="figure" id="pone-0000305-g001"><div class="img"><a name="pone-0000305-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000305.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000305" data-uri="info:doi/10.1371/journal.pone.0000305.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000305.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000305.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000305.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000305.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000305.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000305.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000305.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000305.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000305.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Detection of <em>Bacillus globigii</em> genomic dsDNA at low copy number.</span></strong></p><a id="article1.body1.sec2.sec1.fig1.caption1.p1" name="article1.body1.sec2.sec1.fig1.caption1.p1"></a><p>100 copies of BG genomic dsDNA (0.46 pg) were reacted with 3.2 pmoles of 25-mer ssDNA probe in the presence of 0.5×TBE, 40 mM TMA-Cl, and 300 nM YOYO-1 in either the presence or absence of 2 ng of human genomic dsDNA (approximately 302 copies). Fluorescent emissions of the reaction mixtures (80 µl) were monitored with the Genexus® Analyzer at a setting of 32% PMT after 5, 15, 25, 35, 45, 55 and 65 minutes of incubation at RT. Intensity of triplex-associated fluorescence is plotted as a function of incubation time for the BG triplexes formed in the presence of either 300 nM YOYO-1 (1) or 300 nM YOYO-1 and human genomic dsDNA (2).</p>
<span>doi:10.1371/journal.pone.0000305.g001</span></div><div class="figure" id="pone-0000305-t001"><div class="img"><a name="pone-0000305-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000305.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000305" data-uri="info:doi/10.1371/journal.pone.0000305.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000305.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000305.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000305.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000305.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000305.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000305.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000305.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000305.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000305.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>25-mer and 15-mer ssDNA probe sequences.</span></strong></p><span>doi:10.1371/journal.pone.0000305.t001</span></div><a id="article1.body1.sec2.sec1.p3" name="article1.body1.sec2.sec1.p3"></a><p>Even more remarkable was the ability of the Genomic Assay® to specifically detect 100 copies of bacterial genomic DNA in a sample also containing human genomic dsDNA. Heteropolymeric perfectly matched triplexes were detectably formed between the 100 copies of BG genomic dsDNA and probe 1 in the presence of 300 nM YOYO-1, 40 mM TMA-Cl and 2 ng (302 copies) of human genomic dsDNA (hgDNA) (<a href="#pone-0000305-g001">Figure 1</a>). Moreover, periodic monitoring showed there was a small, progressive increase in triplex-associated fluorescence emission throughout the first 65 minutes of incubation. The fluorescence emissions of the reaction mixtures, whose emissions are shown in <a href="#pone-0000305-g001">Figure 1</a>, were also monitored after 24 hours of incubation. While the BG triplex-associated fluorescence observed in the absence of background hgDNA slightly decreased after 24 hours of incubation, the BG triplex-associated fluorescence observed in the sample also containing hgDNA continued to increase throughout the 24 hours of incubation (<a href="#pone.0000305.s007">Supplemental Table S1</a>). It is noteworthy that the triplex-associated fluorescence emission remained stable over an extended incubation period of 24 hours. The ability to homogeneously assay 0.33 copies of bacterial genomic DNA for each copy of hgDNA present in the sample demonstrates the extreme sensitivity of the Genomic Assay® in detecting pathogens in 80 µl bulk solution.</p>
<a id="article1.body1.sec2.sec1.p4" name="article1.body1.sec2.sec1.p4"></a><p>In this paper we present the use of the Genomic Assay® to detect nucleic acid sequences in various types of samples. Throughout, detection occurs by means of fluorescent emissions from YOYO-1 molecules. The assay may be carried out homogeneously, without a separation step because YOYO-1 molecules, when suitably stimulated, emit with varying intensities dependent upon the nucleic acid structures with which they are complexed and also dependent upon whether certain other reagents are present at very low concentrations. In addition to data in support of these previously unrecognized characteristics of YOYO-1 emissions, we will below, under the “YOYO-1 emissions examined” section of the Results, present data for the unexpected observation that certain lesser concentrations of YOYO-1 will emit much more brightly than greater concentrations of YOYO-1 in the presence of uniform concentrations of nucleic acid oligonucleotides, short duplex DNA or human genomic dsDNA.</p>


<h4>Genomic Assay® identifies 1 bp SNPs in <em>drosophila</em> genomic dsDNA</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>The ability of the Genomic Assay® to identify 1 bp mismatches in non-PCR amplified, non-denatured <em>drosophila</em> genomic dsDNA was investigated. The <em>drosophila</em> genome was selected as an assay target because its size at 165 million bp is greater than that of an average human chromosome. The mutant <em>drosophila</em> genomic dsDNA (ts1 gDNA) contains a 1 bp A to T substitution within the <em>erg</em> gene (a seizure locus) <a href="#pone.0000305-Titus1">[17]</a>. Similar mutations in the human <em>erg</em> gene (<em>HERG</em>) cause long-QT syndrome, a type of cardiac arrhythmia <a href="#pone.0000305-Curran1">[18]</a>.</p>
<a id="article1.body1.sec2.sec2.p2" name="article1.body1.sec2.sec2.p2"></a><p>The highest triplex-associated fluorescent emission intensity was achieved when wild-type <em>drosophila</em> genomic dsDNA (wt gDNA) was reacted with probe 2, indicating that heteropolymeric triplexes, composed of genomic dsDNA bound to a ssDNA probe, can readily form and easily be detected in a homogeneous bulk solution after just 5 min of incubation at RT (<a href="#pone-0000305-g002">Figure 2</a>). The fluorescent intensities attributed to the wt gDNA and YOYO-1 control sample and the wt ssDNA and YOYO-1 control sample were 12% and 1%, respectively, of the total fluorescent intensity emitted from the sample containing perfectly matched triplexes after 5 min of incubation. When the maximum fluorescent emission values were corrected for background fluorescence to account for triplex-associated fluorescent emission, the mismatched dsDNA:ssDNA complexes containing a 1 bp A-A mismatch (ts1 gDNA+probe 2) emitted a triplex-associated fluorescent intensity that was 67% lower than the triplex-associated fluorescent intensity emitted from the perfectly matched dsDNA:ssDNA complexes (wt gDNA+probe 2) after 5 min of incubation (<a href="#pone-0000305-g002">Figure 2</a>). A marginal increase in discrimination between perfectly matched and 1 bp mismatched triplexes was observed upon extended incubation, indicating that the matched triplex complexes formed in the first 5 min, were stable throughout the 90 min incubation period which was periodically monitored. These results demonstrate that the YOYO-1 mediated Genomic Assay® can be employed to assay for 1 bp SNPs in non-denatured genomes of the complexity of <em>drosophila</em>, homogeneously in bulk solution.</p>
<div class="figure" id="pone-0000305-g002"><div class="img"><a name="pone-0000305-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000305.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000305" data-uri="info:doi/10.1371/journal.pone.0000305.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000305.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000305.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000305.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000305.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000305.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000305.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000305.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000305.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000305.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Assay of <em>drosophila</em> genomic dsDNA for the ts1 mutation in the <em>erg</em> gene.</span></strong></p><a id="article1.body1.sec2.sec2.fig1.caption1.p1" name="article1.body1.sec2.sec2.fig1.caption1.p1"></a><p>Fifty ng of either wild-type <em>drosophila</em> genomic dsDNA (wt gDNA) or mutant <em>drosophila</em> genomic dsDNA (ts1 gDNA) were reacted at RT for 5 min with 4 pmoles of wild-type probe 2 in the presence of 0.5×TBE and 500 nM YOYO-1. Fluorescent emissions of the reaction mixtures (100 µl) were monitored with the Genexus® Analyzer at a setting of 30% PMT after 5, 15, 30, 45, 60, 75 and 90 minutes of incubation. The intensity of fluorescence is plotted as a function of incubation time for each sample analyzed. The samples consist of YOYO-1 only with no DNA present (1), wt gDNA with no added ssDNA probe (2), ts1 gDNA with no added ssDNA probe (3), ssDNA probe 2 (4), perfectly matched triplex (5) and 1 bp A–A mismatched triplex (6), as indicated.</p>
<span>doi:10.1371/journal.pone.0000305.g002</span></div>

<h4>Genomic Assay® identifies 1 bp CFTR SNPs and FVL G1691A in human genomic dsDNA</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>The ability of the Genomic Assay® to identify clinically relevant SNPs in the <em>CFTR</em> gene and the factor V gene in non-PCR amplified, non-denatured human genomic dsDNA was investigated. The <em>CFTR</em> SNPs were chosen from the 25 most prevalent <em>CFTR</em> mutations identified by the American College of Medical Genetics based on data derived from Grody <em>et al</em>. <a href="#pone.0000305-Grody1">[19]</a>. The Factor V Leiden (<em>FVL</em>) mutation, G1691A, results in the substitution of glutamine for arginine at position 506 in the amino acid sequence of the coagulation factor V protein <a href="#pone.0000305-Bertina1">[20]</a>. <em>FVL</em> G1691A is a SNP responsible for greater than 90 percent of inherited venous thromboembolism attributed to mutations in factor V <a href="#pone.0000305-Svensson1">[21]</a>.</p>
<a id="article1.body1.sec2.sec3.p2" name="article1.body1.sec2.sec3.p2"></a><p>Typically the ssDNA probe controls produced higher levels of fluorescent emission in the presence of YOYO-1 as compared to that emitted from the genomic dsDNA target control comprising an identical amount of YOYO-1. On occasion, significantly different fluorescence emission levels were observed for the wild-type and mutant ssDNA probe plus YOYO-1 controls, due to different levels of self hybridization characteristic of various probe sequences in the presence of YOYO-1. Data relevant to these observations are set forth below in the “YOYO-1 emissions examined” section of the Results.</p>
<a id="article1.body1.sec2.sec3.p3" name="article1.body1.sec2.sec3.p3"></a><p>30-mer, 25-mer, 20-mer and 15-mer ssDNA probe sequences were tested for use in the Genomic Assay® (<a href="#pone.0000305.s008">Supplemental Tables S2</a> and <a href="#pone.0000305.s009">S3</a>). Surprisingly, 25-mer probes were empirically determined to be optimal for maximum discrimination between perfectly matched and 1 bp mismatched triplex-associated fluorescent emissions when hgDNA was being assayed, under the conditions of our testing. To identify triplex-associated signal, fluorescence emission values from the reaction mixtures were corrected for background fluorescence by subtracting the appropriate probe control emission value from the relevant reaction mixture emissions, all emission intensities being acquired after equal periods of incubation.</p>
<a id="article1.body1.sec2.sec3.p4" name="article1.body1.sec2.sec3.p4"></a><p>Heteropolymeric perfectly matched DNA triplexes in reaction mixtures comprised of wild-type hgDNA and wild-type probe delta F508-WT25C (probe 3), formed and were detected after as little as a 5 min incubation in the presence of 600 nM YOYO-1 (<a href="#pone-0000305-g003">Figure 3A</a> and <a href="#pone-0000305-t002">Table 2</a>). It is truly remarkable that perfect match triplex-associated emissions can be significantly greater than the combined emissions of a hgDNA target control with YOYO-1 and a probe control with YOYO-1 (<a href="#pone-0000305-t002">Table 2</a>). The percent variation of triplex-associated fluorescence (TAF) signals was 0.2–5.0% when triplicate samples prepared at the same time were assayed on the same plate concurrently. The signal of perfect match triplex formation increased dramatically during the first 15 min of incubation, after which a plateau in fluorescence emission level was observed (<a href="#pone-0000305-g003">Figure 3A</a>). The triplex-associated fluorescent emissions produced by a 3 bp deletion triplex in reaction mixtures comprising either 500 pg or 250 pg hgDNA and mutant probe delta F508-MUT25C (probe 4) were 95% and 100% lower, respectively, than those emitted by the perfectly matched triplexes comprising comparable amounts of hgDNA and probe 4 after 5 min of incubation, and remained at this low level throughout the 75 min incubation period monitored (<a href="#pone-0000305-g003">Figure 3A</a> and <a href="#pone-0000305-t002">Table 2</a>).</p>
<div class="figure" id="pone-0000305-g003"><div class="img"><a name="pone-0000305-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000305.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000305" data-uri="info:doi/10.1371/journal.pone.0000305.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000305.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000305.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000305.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000305.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000305.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000305.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000305.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000305.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000305.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Assay of <em>CFTR</em> and <em>FVL</em> mutations in human genomic dsDNA.</span></strong></p><a id="article1.body1.sec2.sec3.fig1.caption1.p1" name="article1.body1.sec2.sec3.fig1.caption1.p1"></a><p>Human genomic dsDNA was extracted from blood (A, B and D) or saliva (C) as described in the text. (A) 500 pg of wild-type human genomic dsDNA (approximately 75 copies) was reacted at RT with 3.2 pmoles of either wild-type or mutant ssDNA probe in the presence of 0.5×TBE and 600 nM YOYO-1. (B–D) 1 ng, 2 ng or 200 pg of wild-type human genomic dsDNA (approximately 151 copies, 302 copies or 30 copies, respectively) was reacted at RT with 3.2 pmoles of either wild-type or mutant ssDNA probe in the presence of 0.5×TBE and 500 nM YOYO-1. (A–D) Reaction mixtures (80 µl) were irradiated as described in the text and analyzed for fluorescent emission. The intensity of triplex-associated fluorescence is plotted as a function of incubation time for each sample analyzed. The samples consist of perfectly matched triplex (1) and mismatched triplex (2) as indicated for <em>CFTR</em> delta F508 (A), <em>CFTR</em> 3659delC (B), <em>CFTR</em> 2789+5G→A (C) and <em>FVL</em> G1691A (D).</p>
<span>doi:10.1371/journal.pone.0000305.g003</span></div><div class="figure" id="pone-0000305-t002"><div class="img"><a name="pone-0000305-t002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000305.t002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000305" data-uri="info:doi/10.1371/journal.pone.0000305.t002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000305.t002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000305.t002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000305.t002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000305.t002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000305.t002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000305.t002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000305.t002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000305.t002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000305.t002.TIF"></span>)
                </a></li></ul></div><p><strong>Table 2.  <span>Triplex assays of human genomic dsDNA for <em>CFTR</em> delta F508 (3 bp AAG deletion).</span></strong></p><span>doi:10.1371/journal.pone.0000305.t002</span></div><a id="article1.body1.sec2.sec3.p5" name="article1.body1.sec2.sec3.p5"></a><p>When 1 ng of wild-type hgDNA was reacted with wild-type probe 3659delC-WT25C (probe 7) in the presence of 500 nM YOYO-1, triplex-associated signals were detected after 5 min of incubation (<a href="#pone-0000305-g003">Figure 3B</a>). A slight increase in triplex-associated fluorescence was observed over the 60 min of incubation. The triplex-associated fluorescent emissions from a 1 bp deletion triplex in a reaction mixture comprised of 1 ng hgDNA and mutant probe 3659delC-MUT25C (probe 8) remained constant throughout the 60 min incubation period and were 81% lower than those emitted from the reaction mixture containing the perfectly matched triplexes (<a href="#pone-0000305-g003">Figure 3B</a>).</p>
<a id="article1.body1.sec2.sec3.p6" name="article1.body1.sec2.sec3.p6"></a><p>The highest fluorescent emission intensity was achieved when wild-type hgDNA was reacted with wild-type probe 2789+5G→A-WT25C (probe 11), indicating perfect match triplex formation after 5 min of incubation in the presence of 500 nM YOYO-1 (<a href="#pone-0000305-g003">Figure 3C</a> and <a href="#pone-0000305-t003">Table 3</a>). The emissions from these perfectly matched DNA triplexes remained stable throughout the 60 min incubation period. When 2 ng of hgDNA was reacted with mutant probe 2789+5G→A-MUT25C (probe 12) to form a 1 bp T–G mismatch triplex, the triplex-associated fluorescent emission was 100% lower than that emitted by the reaction mixture containing the perfectly matched triplex after 5 min incubation and remained constant thereafter (<a href="#pone-0000305-g003">Figure 3C</a> and <a href="#pone-0000305-t003">Table 3</a>). In this instance the fluorescent emissions from reaction mixtures containing mismatched genomic DNA triplexes were less than the emissions from the comparable probe and YOYO-1 control. We hypothesize that this result occurs because, while YOYO-1 facilitates and signals self-association of the 25-mer probes when no genomic dsDNA is present, such self-associated probe complexes tend to disperse, along with any complexed YOYO-1, upon introduction of the genomic target and that only a small offsetting gain in fluorescence occurs when YOYO-1 intercalates into the duplex genomic target or into imperfect triple stranded structures. When emissions from samples containing mismatched triplexes are less intense than emissions from the relevant probe and YOYO-1 control, very desirable assay conditions have been achieved.</p>
<div class="figure" id="pone-0000305-t003"><div class="img"><a name="pone-0000305-t003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000305.t003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000305" data-uri="info:doi/10.1371/journal.pone.0000305.t003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000305.t003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000305.t003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000305.t003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000305.t003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000305.t003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000305.t003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000305.t003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000305.t003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000305.t003.TIF"></span>)
                </a></li></ul></div><p><strong>Table 3.  <span>Triplex assays of human genomic dsDNA (purified from saliva) for <em>CFTR</em> 2789+5G→A (1 bp T–G mismatch).</span></strong></p><span>doi:10.1371/journal.pone.0000305.t003</span></div><a id="article1.body1.sec2.sec3.p7" name="article1.body1.sec2.sec3.p7"></a><p>Heteropolymeric perfectly matched triplexes were detected in reaction mixtures containing either 2 ng, 1 ng, 500 pg or 200 pg hgDNA and probe FVL G1691A-WT25C (probe 33), formed during a 5 min incubation in the presence of 500 nM YOYO-1 (<a href="#pone-0000305-g003">Figure 3D</a> and <a href="#pone.0000305.s010">Supplemental Table S4</a>). The perfect match triplex signal levels were all significantly greater than the combined fluorescence signals of target amounts of hgDNA with YOYO-1 control and wild-type probe with YOYO-1 control (<a href="#pone.0000305.s010">Supplemental Table S4</a>). As was usual, the triplex-associated emissions from the reaction mixtures were stable throughout the 60 min incubation period. The triplex-associated fluorescent emissions from reaction mixtures containing between 2 ng and 200 pg of genomic target and probe FVL G1691A-MUT25C (probe 34) and producing 1 bp G–T mismatched triplexes were consistently less than probe 34 plus YOYO-1 control values after 5, 15, 30, 45 and 60 minute incubations (<a href="#pone.0000305.s010">Supplemental Table S4</a>). Therefore the reactions produced no detected triplex-associated fluorescence. Comparable results were observed when 500 pg, 200 pg, 100 pg or 75 pg of hgDNA were reacted with 3.2 pmoles of either wild-type probe CFTR 2789+5G→A-WT25C (probe 11) or mutant probe CFTR 2789+5G→A-MUT25C (probe 12) (<a href="#pone.0000305.s011">Supplemental Table S5</a>).</p>
<a id="article1.body1.sec2.sec3.p8" name="article1.body1.sec2.sec3.p8"></a><p>These results collectively demonstrate the high level of efficiency and specificity of the Genomic Assay® when practised in bulk solution with 25-mer ssDNA probes and YOYO-1, to directly detect mismatches between probes and non-denatured, non-amplified human genomic dsDNA targets, over a broad range of genomic target concentrations. SNPs in human genomic targets ranging from 4 ng to 75 pg, approximately 604 copies to 11 copies, were identified in the Genomic Assay® using varied YOYO-1 concentrations, homogeneously and in an 80 µl solution.</p>
<a id="article1.body1.sec2.sec3.p9" name="article1.body1.sec2.sec3.p9"></a><p>The human genomic DNA samples assayed in <a href="#pone-0000305-g003">Figures 3A, 3B and 3D</a> were purified from blood. We assayed for the CFTR 2789+5G→A mutation in both human genomic dsDNA purified from blood (<a href="#pone.0000305.s012">Supplemental Table S6</a>) and purified from treated saliva/sputum samples (<a href="#pone-0000305-g003">Figure 3C</a> and <a href="#pone-0000305-t003">Table 3</a>), which had been stored at RT for 1 week following incubation at 50°C for 30 min. The 50°C incubation is recommended by the manufacturer of the Oragene oral sample acquisition kit to allow long-term storage at RT. In side by side comparisons, triplex-associated fluorescent emissions from the reaction mixtures containing hgDNA purified from blood or from treated saliva/sputum, indicated that perfectly matched triplexes formed with equivalent efficiency after 5 min incubation in the presence of 500 nM YOYO-1 (<a href="#pone-0000305-t003">Table 3</a> and <a href="#pone.0000305.s012">Supplemental Table S6</a>). The triplex-associated emissions remained stable throughout a 60 min incubation during which they were monitored. The triplex-associated fluorescent emissions from reaction mixtures containing 1 bp T-G mismatched triplexes were very low, regardless of the origin of the sample or which of the two methods of purification of the sample had been employed (<a href="#pone-0000305-t003">Table 3</a> and <a href="#pone.0000305.s012">Supplemental Table S6</a>).</p>
<a id="article1.body1.sec2.sec3.p10" name="article1.body1.sec2.sec3.p10"></a><p>This demonstrates that the Genomic Assay® of hgDNA purified from saliva/sputum in the Oragene DNA collection kit can be as efficient and specific as one in which the hgDNA is purified from blood. Purification of genomic DNA, as occurs when using Qiagen kits, is therefore not a requirement for the Genomic Assay® of samples containing hgDNA. The acquisition of hgDNA for molecular assay from oral samples offers clear advantages in cost, handling, storing and shipping over acquisition of samples from blood. It would also be greatly preferred by sample donors. It therefore opens the way for convenient point of care testing employing the Genomic Assay®. In side by side comparisons, 1 ml of peripheral blood, purified by Qiagen kit, yielded more than enough human genomic DNA to perform over 9,000 assays as described herein, if 1 ng were assayed per reaction. A typical saliva/sputum sample, purified with an Oragene kit, produced enough human genomic DNA for over 7,000 assays, if 1 ng were needed per reaction.</p>


<h4>Genomic Assay® identifies MTHFR C677T in homozygous and heterozygous human genomic dsDNA samples</h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>The ability of the Genomic Assay® to identify the methylenetetrahydrofolate reductase (<em>MTHFR</em>) mutation, C677T, in non-PCR amplified, non-denatured, human genomic dsDNA was investigated. The <em>MTHFR</em> 677 polymorphism is a cytosine to thymine substitution in the sense strand of the <em>MTHFR</em> gene, resulting in an alanine to valine substitution in the MTHFR enzyme <a href="#pone.0000305-Goyette1">[22]</a>.</p>
<a id="article1.body1.sec2.sec4.p2" name="article1.body1.sec2.sec4.p2"></a><p>Different fluorescence emission levels were observed for the wild-type probe MTHFR C677T-WT25C (probe 31) plus YOYO-1 control and the mutant probe MTHFR C677T-MUT25C (probe 32) plus YOYO-1 control, due to differences in levels of self hybridization which are characteristic of each probe sequence in the presence of YOYO-1 (<a href="#pone.0000305.s013">Supplemental Tables S7</a> and <a href="#pone.0000305.s014">S8</a>). To identify triplex-associated signal, fluorescence emission values from the reaction mixtures, containing hgDNA, probe and YOYO-1, were corrected for background fluorescence by subtracting the appropriate probe control emission value from the relevant reaction mixture emission, both of which were measured after the same duration of incubation.</p>
<a id="article1.body1.sec2.sec4.p3" name="article1.body1.sec2.sec4.p3"></a><p>Perfectly matched DNA triplexes in reaction mixtures comprised of either wild-type homozygous hgDNA and wild-type probe MTHFR C677T-WT25C (probe 31) or mutant homozygous hgDNA and mutant probe MTHFR C677T-MUT25C (probe 32), formed efficiently and were detected after just 5 min of incubation in the presence of 500 nM YOYO-1 (samples 1 and 4, respectively, in <a href="#pone-0000305-g004">Figure 4</a>). The efficiency of matched triplex formation, signaled by the appearance of triplex-associated fluorescence, was slightly greater in the reaction mixtures comprising 1 ng mutant homozygous hgDNA and mutant probe 32, than that comprising 1 ng wild-type homozygous hgDNA and wild-type probe 31, after 5 min of incubation. The triplex-associated fluorescent emission intensities observed from reaction mixtures containing either of the two perfectly matched triplexes were more similar when 2 ng, instead of 1 ng, of hgDNA was present in the 80 µl bulk solution reaction mixtures (<a href="#pone.0000305.s013">Supplemental Tables S7</a> and <a href="#pone.0000305.s014">S8</a>).</p>
<div class="figure" id="pone-0000305-g004"><div class="img"><a name="pone-0000305-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000305.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000305" data-uri="info:doi/10.1371/journal.pone.0000305.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000305.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000305.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000305.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000305.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000305.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000305.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000305.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000305.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000305.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>Assay of <em>MTHFR</em> C677T in homozygous and heterozygous human genomic dsDNA samples.</span></strong></p><a id="article1.body1.sec2.sec4.fig1.caption1.p1" name="article1.body1.sec2.sec4.fig1.caption1.p1"></a><p>Human genomic dsDNA that was either wild-type homozygous, mutant heterozygous or mutant homozygous with respect to <em>MTHFR</em> C677T, was extracted from blood as described in the text. One ng of human genomic dsDNA (approximately 151 copies) was reacted at RT with 3.2 pmoles of either wild-type or mutant ssDNA probe in the presence of 0.5×TBE and 500 nM YOYO-1. Reaction mixtures (80 µl) were irradiated as described in the text and analyzed for fluorescent emission. The intensity of triplex-associated fluorescence is plotted as a function of incubation time for each sample analyzed. The samples consist of perfectly matched triplexes (1 and 4), mutant heterozygous triplexes (3) and mismatched triplexes (2 and 5) as indicated for <em>MTHFR</em> C677T.</p>
<span>doi:10.1371/journal.pone.0000305.g004</span></div><a id="article1.body1.sec2.sec4.p4" name="article1.body1.sec2.sec4.p4"></a><p>When 1 ng mutant heterozygous hgDNA was reacted with mutant probe 32 (<a href="#pone-0000305-g004">sample 3, Figure 4</a>), the triplex-associated fluorescent emissions were 77% less after 5 min incubation and 90% less after 60 min incubation, than those emitted from the reaction mixture containing the perfectly matched triplexes comprised of 1 ng wild-type homozygous hgDNA and wild-type probe 31 (<a href="#pone-0000305-g004">sample 1, Figure 4</a>), each evaluated after comparable periods of incubation.</p>
<a id="article1.body1.sec2.sec4.p5" name="article1.body1.sec2.sec4.p5"></a><p>The triplex-associated fluorescent emissions from 1 bp C–A mismatch triplexes in a reaction mixture comprised of 1 ng wild-type homozygous hgDNA and mutant probe 32 (<a href="#pone-0000305-g004">sample 2, Figure 4</a>) were 92% less after 5 min and 98% less after 60 min incubation, than those emitted from the reaction mixture containing the perfectly matched triplexes comprised of 1 ng wild-type homozygous hgDNA and wild-type probe 31 (<a href="#pone-0000305-g004">sample 1, Figure 4</a>), each evaluated after comparable periods of incubation. The triplex-associated fluorescent emissions from 1 bp C–A mismatch triplexes in a reaction mixture comprised of 1 ng mutant homozygous hgDNA and wild-type probe 31 (<a href="#pone-0000305-g004">sample 5, Figure 4</a>) were consistently less than the wild-type probe 31 plus YOYO-1 control value. Therefore the reaction produced no detected triplex-associated fluorescence.</p>
<a id="article1.body1.sec2.sec4.p6" name="article1.body1.sec2.sec4.p6"></a><p>These results demonstrate the high efficiency and specificity of the Genomic Assay® in assaying for SNPs in wild-type homozygous, mutant heterozygous or mutant homozygous hgDNA samples homogeneously and in bulk solution. The results shown were obtained using antisense 25-mer wild-type and mutant probes for <em>MTHFR</em> C677T. Comparable results were observed when analogous reactions were performed using sense 25-mer wild-type and mutant probes for <em>MTHFR</em> C677T (data not shown). The sensitivity of the Genomic Assay® to assay for <em>MTHFR</em> C677T was further demonstrated when human genomic dsDNA targets, ranging in weight from 2 ng to 200 pg, were successfully assayed with wild-type probe 31 and mutant probe 32 in a final reaction volume of 80 µl (<a href="#pone.0000305.s015">Supplemental Table S9</a>).</p>
<a id="article1.body1.sec2.sec4.p7" name="article1.body1.sec2.sec4.p7"></a><p>Numerous SNPs, in regions having widely disparate GC contents, have been successfully and repeatedly assayed in hgDNA using the Genomic Assay® (<a href="#pone-0000305-t004">Table 4</a>). On occasion, variations in reagent concentrations or protocol were advantageous. The triplex-associated fluorescent emissions observed from the mismatched triplexes varied between 100% to 54% lower than those observed from the respective perfectly matched triplexes after 5 min of incubation (<a href="#pone-0000305-t004">Table 4</a>).</p>
<div class="figure" id="pone-0000305-t004"><div class="img"><a name="pone-0000305-t004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000305.t004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000305" data-uri="info:doi/10.1371/journal.pone.0000305.t004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000305.t004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000305.t004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000305.t004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000305.t004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000305.t004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000305.t004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000305.t004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000305.t004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000305.t004.TIF"></span>)
                </a></li></ul></div><p><strong>Table 4.  <span>Data from Assays of Human Genomic DNA Samples.</span></strong></p><span>doi:10.1371/journal.pone.0000305.t004</span></div>

<h4>The Genomic Assay® can employ the waxing and/or waning of triplex-associated fluorescent emissions</h4>
<a id="article1.body1.sec2.sec5.p1" name="article1.body1.sec2.sec5.p1"></a><p>The effect of addition of various kosmotropic agents to the reaction mixture when assaying to identify various SNPs in human genomic dsDNA was investigated. In the presence of 40 mM of the kosmotropic cation, tetramethylammonium chloride (TMA-Cl), the fluorescent emissions from the wild-type probe CFTR 3849+10kbC→T-WT25C (probe 9) and mutant probe CFTR 3849+10kbC→T-MUT25C (probe 10) controls were very similar (<a href="#pone.0000305.s016">Supplemental Table S10</a>). Duplicate triplex reaction mixtures were assembled. In one instance YOYO-1 was added last, with the mixture incubated for 5 min before irradiation (<a href="#pone-0000305-g005">Figure 5A</a>). In the other instance all components of the reaction mixture (except hgDNA) had been mixed, incubated for 5 min and irradiated once before hgDNA was added, followed by a further 5 min incubation and another irradiation (<a href="#pone-0000305-g005">Figure 5B</a>). To identify triplex-associated emissions, fluorescence emission values from the reaction mixtures were corrected for background fluorescence by subtracting the appropriate probe control emission values from the relevant reaction mixture emissions, all of which were acquired after comparable periods of incubation.</p>
<div class="figure" id="pone-0000305-g005"><div class="img"><a name="pone-0000305-g005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000305.g005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000305" data-uri="info:doi/10.1371/journal.pone.0000305.g005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000305.g005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000305.g005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000305.g005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000305.g005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000305.g005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000305.g005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000305.g005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000305.g005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000305.g005.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 5.  <span>Waxing and waning of triplex-associated fluorescent emissions can enhance a homogeneous in solution assay.</span></strong></p><a id="article1.body1.sec2.sec5.fig1.caption1.p1" name="article1.body1.sec2.sec5.fig1.caption1.p1"></a><p>(A, B) Human genomic dsDNA was extracted from blood as described in the text. Two ng of wild-type human genomic dsDNA (approximately 302 copies) was reacted at RT with 3.2 pmoles of either wild-type or mutant ssDNA probe in the presence of 0.5×TBE, 40 mM TMA-Cl and 500 nM YOYO-1. (A) YOYO-1 was added last to the reaction mixtures. (B) Human genomic dsDNA was added last to the reaction mixtures. (A, B) Reaction mixtures (80 µl) were irradiated as described in the text and analyzed for fluorescent emission. The intensity of triplex-associated fluorescence is plotted as a function of incubation time for each sample analyzed. The samples consist of perfectly matched triplex (1) and mismatched triplex (2) as indicated for <em>CFTR</em> 3849+10kbC→T.</p>
<span>doi:10.1371/journal.pone.0000305.g005</span></div><a id="article1.body1.sec2.sec5.p2" name="article1.body1.sec2.sec5.p2"></a><p>The triplex-associated fluorescent emissions from perfectly matched triplexes in reaction mixtures consisting of hgDNA and wild-type probe 3849+10kbC→T-WT25C (probe 9), monitored after a 15 min incubation in the presence of 40 mM TMA-Cl and 500 nM YOYO-1, were similar, whether YOYO-1 or hgDNA had been added last to the reaction mixtures (<a href="#pone-0000305-g005">Figures 5A and 5B</a>). Under both protocols, the perfectly matched triplex-associated fluorescent emissions increased when monitored over the first 60 min of incubation.</p>
<a id="article1.body1.sec2.sec5.p3" name="article1.body1.sec2.sec5.p3"></a><p>In the reaction mixture containing 40 mM TMA-Cl, to which YOYO-1 had been added last, the triplex-associated emissions from 1 bp A–C mismatched triplexes, consisting of hgDNA and mutant probe 3849+10kbC→T-MUT25C (probe 10), were 70% less after 5 min of incubation and were 100% less after 30 min of incubation, than those emitted from a similarly monitored reaction mixture containing the perfectly matched triplexes (<a href="#pone-0000305-g005">Figure 5A</a>). The triplex-associated emissions from the 1 bp A–C mismatched triplexes in these reaction mixtures remained constant between 30 min and 60 min of incubation. In the reaction mixture containing 40 mM TMA-Cl, to which hgDNA had been added last, the triplex-associated emissions from 1 bp A–C mismatched triplexes consisting of hgDNA and mutant probe 3849+10kbC→T-MUT25C (probe 10) were 49% less after 15 min of incubation and were 88% less after 60 min of incubation, than those emitted from a similarly monitored reaction mixture containing the perfectly matched triplexes (<a href="#pone-0000305-g005">Figure 5B</a>).</p>
<a id="article1.body1.sec2.sec5.p4" name="article1.body1.sec2.sec5.p4"></a><p>These results demonstrate that the addition of 40 mM TMA-Cl to the triplex reaction mixture can result in an enhanced progressive increase in triplex-associated fluorescent emission levels from perfectly matched triplexes and an enhanced progressive decrease in triplex-associated fluorescent emission levels from mismatched triplexes, which pattern of emission changes we refer to as “waxing and waning”. Increases in discrimination levels between perfect match binding and mismatch binding can be observed over time. A pronounced waxing and waning emission pattern was also observed in Genomic Assays® of hgDNA samples for <em>CFTR</em> 2789+5G→A, <em>CFTR</em> delta F508 or <em>FVL</em> G1691A detection when 45 mM TMA-Cl, 40 mM TMA-Cl or both 50 mM TMA-Cl and 20 mM NaCl were included, respectively, in the reaction mixtures containing 500 nM YOYO-1 (data not shown).</p>
<a id="article1.body1.sec2.sec5.p5" name="article1.body1.sec2.sec5.p5"></a><p>Numerous kosmotropic cations have been used by us in conjuction with YOYO-1 to generally improve the specificity of the Genomic Assay® and to enhance the waxing and waning of triplex-associated emissions during SNP detection (data not shown). These include 50 to 80 mM NaCl, 10 to 60 mM Na<sub>2</sub>SO<sub>4</sub>, 50 mM Na<sub>2</sub>HPO<sub>4</sub>, 125 to 250 mM (NH<sub>4</sub>)<sub>2</sub>SO<sub>4</sub>, 30 mM TriMA-Cl, 30 to 52.5 mM TMA-Cl, each added separately, or 50 mM TMA-Cl in combination with 10 to 20 mM NaCl. The benefits of adding one or more kosmotropic cations to a Genomic Assay® reaction mixture and the best concentrations for addition under any set of selected assay conditions must be determined experimentally.</p>


<h4>YOYO-1 emissions examined</h4>
<a id="article1.body1.sec2.sec6.p1" name="article1.body1.sec2.sec6.p1"></a><p>YOYO-1 will intercalate into a variety of multi-stranded nucleic acid complexes. It is fluorescent when intercalated and suitably irradiated. In the Genomic Assay® we used it to promote and signal the formation of heteropolymeric, specific triplex useful in diagnostics and analysis. As shown throughout this paper, the emissions from YOYO-1 complexed in triple stranded structures were detectable over and above the emissions generated by YOYO-1 complexed with genomic duplex DNA present in the reaction mixture and very significant emissions by YOYO-1 complexed with probe:probe binding. Our triplex assay reaction mixtures typically contained concentrations of between 300 to 600 nM YOYO-1 when assaying genomic samples. We conclude that YOYO-1 allowed for such assaying because it did not produce a constant emission per intercalated molecule. Triplex intercalated YOYO-1's fluoresced much brighter than did duplex intercalated YOYO-1's. We also observed that YOYO-1 molecules were much brighter when intercalated into complexes of identical short oligonucleotides, such as the probes used in our assay, than when a like number of YOYO-1 molecules were in the presence of genomic duplex DNA.</p>
<a id="article1.body1.sec2.sec6.p2" name="article1.body1.sec2.sec6.p2"></a><p>Probe:probe associated fluorescence constituted the major source of unwanted background signal in our YOYO-1 promoted triplex assay conducted homogeneously in solution. Such probe:probe complex emissions were quantified in the “probe plus YOYO-1” control which accompanied each triplex assay. Emissions from probe:probe complexes must be determined individually, as each probe base sequence has a characteristic ability to not only form hairpin structures and self-dimer hybridized complexes, but also to form parallel homologous structures stabilized and signalled by intercalated YOYO-1 molecules (Daksis and Erikson, unpublished observations). YOYO-1's non-specific intercalation into duplex genomic DNA also produced unwanted background signal in our assay, albeit of a vastly lesser intensity than that emitted from YOYO-1s intercalated into probe:probe complexes. This observation was unexpected, but consistently observed. We investigated such background emissions by having a “duplex target plus YOYO-1” control which accompanied each triplex assay. Fortunately, the signal emitted from triple-stranded nucleic acids and complexed YOYO-1 molecules was so intense, that the triple-stranded structure could be readily detected homogeneously in solution. When genomic dsDNA was added last to a reaction mixture, we believe probe:probe complexes dissolved and intercalated YOYO-1 molecules were released.</p>
<a id="article1.body1.sec2.sec6.p3" name="article1.body1.sec2.sec6.p3"></a><p>To demonstrate YOYO-1's ability to promote and signal the formation of probe:probe complexes, we monitored fluorescent emissions from 80 µl final volume hybridization mixtures containing 0.5×TBE, pH 8.3, 40 mM TMA-Cl, 3.2 pmole of 25-mer heteropolymeric DNA oligo probe bglIR-WT25C (probe 1) and various concentrations of YOYO-1, ranging from 50 nM to 1000 nM, at 50 nM intervals. There was no emission from YOYO-1 over this range of concentrations, if no nucleic acid was present. The emissions were acquired at various time points after commencement of incubation, ranging from 5 minutes to 24 hours using the Genexus® Analyzer. <a href="#pone.0000305.s001">Supplemental Figure S1</a> showed the 5 minute, 65 minute and 24 hour emissions as a function of YOYO-1 concentration.</p>
<a id="article1.body1.sec2.sec6.p4" name="article1.body1.sec2.sec6.p4"></a><p>At concentrations between 50 and 350 nM, YOYO-1 facilitated probe:probe structures which formed by the 5 minute time point and were stable thereafter for at least 24 hours, as indicated by near uniform fluorescent emissions. At concentrations between 350 and 900 nM, YOYO-1 facilitated and signalled probe:probe structures which appeared to dissipate over time. At concentrations of 900 nM YOYO-1 or more, probe:probe complexes appeared not to occur, or fluorescent emission from such complexes was abolished, should they have formed. After 24 hours of incubation, concentrations of 700 nM YOYO-1 or more appeared to abolish probe:probe complexes, or emissions from them, should they have remained. We speak in absolute terms for convenience and to express the general meaning of our data. We do not suggest that better instrumentation or more precise experiments will not generate data, which will, at the margin, qualify our statements.</p>
<a id="article1.body1.sec2.sec6.p5" name="article1.body1.sec2.sec6.p5"></a><p>The <a href="#pone.0000305.s001">Supplemental Figure S1</a> data will be very disturbing, if one's only experience with intercalators has been to use a fixed amount of intercalator to quantitate an unknown amount of duplex DNA. In addition to the non-linear, fluorescence pattern alluded to above, it is as well surprising that a lesser quantity of YOYO-1 might fluoresce more brightly than a greater quantity in the presence of a constant amount of DNA. Nevertheless, it was an observable, highly reproducible and hence incontrovertible result.</p>
<a id="article1.body1.sec2.sec6.p6" name="article1.body1.sec2.sec6.p6"></a><p>The maximum fluorescent emission in the presence of 3.2 pmoles of probe oligos arose when 100 nM of YOYO-1 was present. Molecules of YOYO-1 intercalated with duplex genomic DNA, as shown below, did not emit very intensely per molecule, as compared to those intercalated in probe:probe complexes. The inference may be fairly drawn that slight amounts of YOYO-1 complexed with homologously paired probes in a triplex assay test mixture will produce unwanted background emissions which may be significantly more intense than emissions from like amounts of YOYO-1 intercalated with duplex genomic DNA. Such probe:probe associated emissions will tend to minimize the triplex-associated fluorescence (TAF) which can only be ascertained after subtraction of the corresponding “probe plus YOYO-1” control emission value. So as to not double count YOYO-1 background emissions, we arrived at TAF by subtracting from the total emission from the reaction mixture the concurrently acquired “probe plus YOYO-1” control emission intensity, while not subtracting the “target plus YOYO-1” control emission intensity.</p>
<a id="article1.body1.sec2.sec6.p7" name="article1.body1.sec2.sec6.p7"></a><p>The data presented in <a href="#pone.0000305.s001">Supplemental Figure S1</a> was obtained from a reaction mixture containing 40 mM TMA-Cl, a kosmotropic cation. <a href="#pone.0000305.s002">Supplemental Figure S2</a> compared YOYO-1 emissions acquired after 5 minutes of incubation in the presence or absence of 40 mM TMA-Cl, when constant amounts of the DNA oligo probes were present and concentrations of YOYO-1 varied. TMA-Cl is a water structuring agent. It is understood to co-ordinate and bind multiple water molecules. <a href="#pone.0000305.s002">Supplemental Figure S2</a> disclosed some other surprising results.</p>
<a id="article1.body1.sec2.sec6.p8" name="article1.body1.sec2.sec6.p8"></a><p>The presence of TMA-Cl in the concentration employed, facilitated a smooth and largely uninterrupted decline in YOYO-1 emissions from reaction mixtures containing increasing YOYO-1 concentrations, greater than 100 nM. Absent TMA-Cl, the reaction mixtures exhibited a constant low level of YOYO-1 emission at YOYO-1 concentrations of 400 nM or more, suggesting the absence of probe:probe complexes in the presence of those concentrations of YOYO-1 or the inability of those concentrations of intercalated YOYO-1s to emit light upon irradiation. It is surprising that a low concentration of TMA-Cl could have such a striking effect on YOYO-1 emissions and/or probe:probe complex formation in the presence of an overwhelming greater number of water molecules. From this data it appeared that, absent TMA-Cl, YOYO-1, at concentrations above 400 nM, disrupted the ability of probes to bind one another homologously to create an anhydrous environment hospitable to YOYO-1 intercalation and hence YOYO-1 emissions.</p>
<a id="article1.body1.sec2.sec6.p9" name="article1.body1.sec2.sec6.p9"></a><p>
          <a href="#pone.0000305.s003">Supplemental Figure S3</a> displayed 5 minute incubation data from reaction mixtures, with and without 40 mM TMA-Cl, where 3.2 pmoles of <span class="underline">ds</span>DNA bglIR probes were incubated with YOYO-1 concentrations from 50 nM to 1000 nM, at 50 nM intervals. The dsDNA probes were comprised of antisense probe 1 and a complementary 25-mer antiparallel sense strand to which it was annealed in the presence of 100 mM NaCl. Here the objective was to plot the emissions from varying YOYO-1 concentrations present with a defined amount of identical <span class="underline">short</span> duplex DNA. The data for the 65 minute incubation time point in each experiment were virtually identical and the 24 hour time point data were similar in each experiment. In <a href="#pone.0000305.s003">Supplemental Figure S3</a> we saw, as we did with the experiments with ssDNA oligo probes, that maximal emission levels occurred, surprisingly, with low YOYO-1 concentrations present and hence low levels of YOYO-1 intercalation. At increasing YOYO-1 concentrations above 200 nM, TMA-Cl facilitated a steady, essentially linear, decline in emission intensities. This decline was unexpected. We did not conclude that the short duplexes were saturated in the presence of 200 nM YOYO-1. In the reaction mixtures without TMA-Cl, increasing YOYO-1 concentrations from 50 to 150 nM YOYO-1 resulted in increased emissions but there was an abrupt emission decline when concentrations between 200 and 300 nM YOYO-1 were present, and in the presence of yet greater YOYO-1 concentrations, declining emissions were produced.</p>
<a id="article1.body1.sec2.sec6.p10" name="article1.body1.sec2.sec6.p10"></a><p>
          <a href="#pone.0000305.s004">Supplemental Figure S4</a> disclosed data from experiments where reaction mixtures, with and without 40 mM TMA-Cl, contained 2 ng purified human genomic duplex DNA (gDNA). Concentrations of YOYO-1 varying from 50 to 1000 nM, at 50 nM intervals were added. The <a href="#pone.0000305.s004">Supplemental Figure S4</a> data was obtained after 5 minutes of incubation. The 5 minute emission intensities from reaction mixtures containing various concentrations of YOYO-1 were virtually identical at the 24 hour time point (data not shown). In the presence of TMA-Cl, the data generally indicated emissions, whose intensities increased, more or less linearly, by about 50% over the tested range of increasing YOYO-1 concentrations. In the absence of TMA-Cl, the emissions from a constant amount of human gDNA in the presence of increasing YOYO-1 concentrations above 200 nM, tended to decrease at a generally linear way. This data too would not be expected by anyone conversant with using a constant amount of intercalating dye to quantitate an unknown amount of duplex DNA. It showed how TMA-Cl, at slight concentration, may affect emission from duplex intercalated YOYO-1s. All of the foregoing data provides much to be pondered. Almost nothing related to YOYO-1 emissions from low concentrations appears to occur as one would expect. Fortunately, in spite of all this, YOYO-1 makes possible a homogeneous triplex SNP assay of duplex genomic DNA sample which is directly detectable.</p>
<a id="article1.body1.sec2.sec6.p11" name="article1.body1.sec2.sec6.p11"></a><p>
          <a href="#pone.0000305.s005">Supplemental Figure S5</a> disclosed data from reaction mixtures in which perfect or 1 bp mismatch triplexes were formed using our standard 3.2 pmole/80 µl concentration of 25-mer oligo probes and 2 ng of wild-type human genomic dsDNA, 40 mM TMA-Cl and 600 nM YOYO-1. To identify triplex-associated emissions, fluorescence emission values from the reaction mixtures were corrected for background fluorescence by subtracting the “probe plus YOYO-1” control emission intensities acquired at the incubation time points at which reaction mixture emissions were also acquired. The emission values giving rise to <a href="#pone.0000305.s005">Supplemental Figure S5</a> data are produced in <a href="#pone.0000305.s017">Supplemental Table S11</a>. The data disclosed that the triplex-associated emissions of the mismatched triplex, under the conditions of the reaction, were less than that of the matched triplex. <a href="#pone.0000305.s006">Supplemental Figure S6</a> presented the triplex-associated emissions from triplex assay mixtures when 500 nM YOYO-1 and 45 mM TMA-Cl were present. The emission values giving rise to <a href="#pone.0000305.s006">Supplemental Figure S6</a> are produced in <a href="#pone.0000305.s018">Supplemental Table S12</a>. <a href="#pone.0000305.s006">Supplemental Figure S6</a> disclosed very significant TAF single base pair assay discrimination, far in excess of that produced when 600 nM YOYO-1 and 40 mM TMA-Cl concentrations were used. This data disclosed that YOYO-1 concentration must be matched with other reagents present in the reaction mixture to carry out optimally sensitive and optimally signalled triplex SNP assays of genomic samples employing YOYO-1 as the promoter and reporter of the assay. All such matching must be explored empirically.</p>
<a id="article1.body1.sec2.sec6.p12" name="article1.body1.sec2.sec6.p12"></a><p>The foregoing provides data disclosing some of the puzzling but adventitious aspects of YOYO-1 emissions. Complexed YOYO-1 emissions can be remarkably intense at very dilute concentration which may contribute high non-specific background emissions. YOYO-1 emission variability produced by the class of nucleic acid complex into which it is intercalated makes possible a very sensitive and homogeneous assay for the presence of low copy pathogens or SNPs in human genomic samples. It is of great importance to realize that the YOYO-1 emission per molecule is immensely variable depending on the nucleic acid structure into which it is intercalated, and depends as well upon relative and local YOYO-1 and reagent concentrations. It is readily apparent that the YOYO-1 promoted and signalled triplex assay is a phenomenon strongly related to hydration because kosmotropic agents, such as TMA-Cl, can contribute very significantly to the TAF. YOYO-1 causes whatever nucleic acids are present and capable of providing multi-stranded anhydrous environments for YOYO-1 intercalation, to set up in 5 minutes or less at room temperature and thereafter to remain remarkably stable.</p>


<h4>Discussion</h4>
<a id="article1.body1.sec2.sec7.p1" name="article1.body1.sec2.sec7.p1"></a><p>Molecular diagnostics of nucleic acids currently rely almost exclusively on a PCR amplification step. Due to the complexity of PCR optimization, the risk of contaminant amplification, the presence of known or unknown PCR inhibitors in reaction mixtures, and the inherent replication error rate, the potential for misdiagnosis when using PCR based diagnostic methodologies is significant <a href="#pone.0000305-Raggi1">[15]</a>, <a href="#pone.0000305-Birch1">[16]</a>. Overcoming or coping with all of these factors adds to the cost of using the PCR method.</p>
<a id="article1.body1.sec2.sec7.p2" name="article1.body1.sec2.sec7.p2"></a><p>The well known sequence requirements of “canonical triplex” formation have been heretofore understood to preclude development of molecular diagnostic methods whereby base pairs in stable heteropolymeric duplex samples may be assayed with base specific resolution using third strand probes.</p>
<a id="article1.body1.sec2.sec7.p3" name="article1.body1.sec2.sec7.p3"></a><p>We have developed a Genomic Assay® providing direct detection of heteropolymeric triplex formation between native genomic dsDNA and ssDNA probes, which eliminates reliance on PCR amplification of genomic samples. The Genomic Assay® results presented here clearly demonstrate the capability to assay human genomic dsDNA homogeneously, in solution, at RT, after as little as 5 minutes of incubation, using unlabeled ssDNA probes and YOYO-1. Such heteropolymeric triplexes can be formed to detect sequences, SNPs, deletions, insertions and repeated sequences in genomic samples, or to detect the presence or genotype of another organism or pathogen in a sample also containing human genomic dsDNA. The assay provides for the direct detection of a signal related to the base specific triplex complex formed by the duplex target sequence and probe in a homogeneous solution.</p>
<a id="article1.body1.sec2.sec7.p4" name="article1.body1.sec2.sec7.p4"></a><p>The Genomic Assay® reported herein depends upon YOYO-1 bis-intercalation rendering the duplex target into a locally de-condensed conformation <a href="#pone.0000305-Johansen1">[23]</a>, which allows for specific base recognition and detectable binding to occur between a heteropolymeric base sequence of a strand in the duplex and the heteropolymeric base sequence of the probe strand. The Ingeneus Triplex® and the Genomic Assay® are therefore conformation dependent, whereas the “canonical triplex” is homopurine or homopyrimidine sequence dependent. The base specific heteropolymeric triplex assay we report is therefore <em>sui generis</em>. The base specific heteropolymeric complex formed in the assay reported here approximates the triple strand intermediates which are postulated by various models of homologous recombination. Homologous recombination <em>in vivo</em> requires the action of recombination proteins, such as RecA, to locally de-condense the duplex so as to facilitate specific heteropolymeric recognition and binding of the mixed base third strand. RecA's effect on binding duplex is to locally de-condense the duplex 1.5 fold <a href="#pone.0000305-Stasiak1">[24]</a>, <a href="#pone.0000305-Egelman1">[25]</a>. YOYO-1, which is a non-specific bis-intercalator as used by us, is known to locally de-condense the duplex 1.3 fold <a href="#pone.0000305-Johansen1">[23]</a> upon intercalation. We accordingly propose that YOYO-1 causes duplex DNA to be rendered into a conformation suitable for the specific recognition and binding of a heteropolymeric probe strand to a sequence of bases present in the duplex target.</p>
<a id="article1.body1.sec2.sec7.p5" name="article1.body1.sec2.sec7.p5"></a><p>There are two models of homologous recombination which seem relevant to the understanding of the Genomic Assay® and the base specific Ingeneus Triplex® upon which it depends. The model first proposed involves third strand recognition of duplex base sequence in the major groove of the duplex <a href="#pone.0000305-Zhurkin1">[26]</a>. That model has been elaborated since <a href="#pone.0000305-Kim1">[27]</a>–<a href="#pone.0000305-Malkov1">[29]</a>. The other model involves third strand recognition of duplex base sequence in the minor grove of the duplex <a href="#pone.0000305-Bertucat1">[30]</a>. Our references regarding each of the two homologous recombination models referred to are representative only and are not exhaustive of the groups working on such models and their published results.</p>
<a id="article1.body1.sec2.sec7.p6" name="article1.body1.sec2.sec7.p6"></a><p>In most cases, the triplex-associated fluorescent emission signal we observe could be detected within the first 5 min of incubation (<a href="#pone-0000305-g001">Figures 1</a>–<a href="#pone-0000305-g002"></a><a href="#pone-0000305-g003"></a><a href="#pone-0000305-g004"></a><a href="#pone-0000305-g005">5</a>). While triplex-associated fluorescent emissions from homogeneous reaction mixtures containing perfectly matched genomic triplexes were relatively high, the triplex-associated fluorescent emissions from reaction mixtures comprising mismatched triplexes were lower and could be very low (<a href="#pone-0000305-g002">Figures 2</a>–<a href="#pone-0000305-g003"></a><a href="#pone-0000305-g004"></a><a href="#pone-0000305-g005">5</a> and <a href="#pone-0000305-t002">Tables 2</a>–<a href="#pone-0000305-t003"></a><a href="#pone-0000305-t004">4</a>). The triplex-associated fluorescent emissions observed from SNP assay mismatched triplexes were often between 80% to 100% less than those observed from the respective perfectly matched triplexes. The observation that the emissions attributable to triplex complexes formed with hgDNA samples after 5 min of incubation, and were stably detected throughout extended incubations, strongly suggests that equilibrium of triplex formation had been substantially achieved after only five min of incubation at RT. Time constraints imposed by manual handling steps have precluded our monitoring of emissions attributable to triplex formation after incubations shorter than 5 min.</p>
<a id="article1.body1.sec2.sec7.p7" name="article1.body1.sec2.sec7.p7"></a><p>Reaction mixtures and protocols can be selected in which changes in fluorescent emission levels between perfect match triplexes and mismatch triplexes can allow for better discrimination based upon “waxing and waning” of emitted signals monitored over time. The waxing and waning emission pattern may be enhanced by the addition of selected concentrations of one or more kosmotropic cations, added separately or in combination, to an assay reaction mixture (<a href="#pone-0000305-g005">Figure 5</a> and data not shown). Additionally, there are advantages in carrying out assays in duplicate employing varied protocols or reagents, so as to obtain signals whose respective characteristics confirm the scoring of the sample. When assaying genomic targets, it is very useful to have fluorescence based methods, which are not dependent merely on relative strength of emission intensities, but which also encompass rates and direction of change of emissions over time.</p>
<a id="article1.body1.sec2.sec7.p8" name="article1.body1.sec2.sec7.p8"></a><p>The use of mathematical models to understand the thermodynamics and kinetics of matched and mismatched duplex hybridization, as time and concentration dependent, has recently been reported <a href="#pone.0000305-Zhang1">[31]</a>. Transitory phases in which mismatched duplexes are initially more numerous than matched duplexes are theorized with mismatched duplexes disassociating as the reaction mixture comes to equilibrium. Our waxing and waning methods are congruent with such observations and theorizing.</p>
<a id="article1.body1.sec2.sec7.p9" name="article1.body1.sec2.sec7.p9"></a><p>The Genomic Assay® can be very efficient and specific, regardless whether YOYO-1 or genomic dsDNA has been added last to the reaction mixtures (<a href="#pone-0000305-g005">Figures 5A and 5B</a>). This will allow the Genomic Assay® to be carried out using kits containing all reagents specific for the assay to be conducted, to which the test genomic dsDNA is added last.</p>
<a id="article1.body1.sec2.sec7.p10" name="article1.body1.sec2.sec7.p10"></a><p>It is remarkable how easily 25-mer heteropolymeric specific triplexes form in the presence of YOYO-1 and that they may be directly and homogeneously detected in a reaction mixture containing human genomic dsDNA. The number of YOYO-1 molecules present in a typical Genomic Assay® reaction mixture as presented in this report is limiting and the number of potential binding sites in a human genomic target is in excess, as is the potential of ssDNA oligonucleotide probes to form parallel homologous complexes stabilized and signaled by intercalated YOYO-1s (Daksis and Erikson, unpublished observations). Upon introduction of human genomic dsDNA to a reaction mixture containing YOYO-1 and ssDNA oligonucleotide probes, it appears that YOYO-1 intercalation into duplex DNA and matched triplexes is favoured over YOYO-1 interaction with probe:probe complexes. It is possible that the greatly increased fluorescence emitted by perfectly matched triplex structures, detectable above the YOYO-1 emission from genomic dsDNA background, may be a result of mutual donor:donor energy migration between stimulated YOYO-1 molecules incorporated in the enhanced anhydrous conditions of the triplex structure.</p>
<a id="article1.body1.sec2.sec7.p11" name="article1.body1.sec2.sec7.p11"></a><p>The Genomic Assay® may also be performed with labels other than YOYO-1 used free in solution, or in formats other than in bulk solution (Daksis and Erikson, unpublished observations).</p>
<a id="article1.body1.sec2.sec7.p12" name="article1.body1.sec2.sec7.p12"></a><p>In conclusion, we have developed a rapid, homogeneous assay based on detectable heteropolymeric triplex formation between native genomic dsDNA and ssDNA probes, which can eliminate recourse to PCR amplification based methods. The Genomic Assay® allows the direct detection of low copy pathogen genomes present in a sample also containing human genomic duplex DNA, or the direct detection of SNPs in human genomic duplex DNA. Our assay constitutes a hybridization assay which is fundamentally different than all previously developed hybridization assays, since no denaturation of the duplex DNA is required prior to binding of the ssDNA probe. Furthermore, the presence of long stretches of homopurine or homopyrimidine sequences in the binding region of the duplex DNA, as necessary for “canonical triplex” formation, is not required in our assay. Provided a suitable local de-condensation of the duplex target is achieved, specific heteropolymeric triplexes readily form with third strand probes with base pair resolution. The Genomic Assay® herein presented depends upon YOYO-1, a bis-intercalator, to both de-condense the duplex and to signal the presence of the specific triplex of the assay. The heteropolymeric triplexes promoted by YOYO-1 approximate the heteropolymeric homologous recombination triplex intermediates, which occur <em>in vivo</em> under the action of recombination proteins, such as RecA. Homologous recombination models propose that mixed base ssDNA sequences bind either in the major or minor groove of the duplex. It is accordingly no longer necessary to denature genomic duplex DNA to be able to assay it for the presence of a sequence of interest or a SNP, nor is it necessary to perform a separation step if one wishes to employ optical signals to assay a highly complex target in bulk solution.</p>

</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4>Single-stranded oligonucleotide probes</h4>
<a id="article1.body1.sec3.sec1.p1" name="article1.body1.sec3.sec1.p1"></a><p>Sequences of the ssDNA probes used are listed in <a href="#pone-0000305-t001">Table 1</a>. All oligonucleotide probes were synthesized on a DNA synthesizer, cartridge purified and dissolved in ddH<sub>2</sub>O at a concentration of 1 pmole/µl. Probe 1 was complementary to a segment of the sense strand of the Bgl I restriction endonuclease (<em>bglIR</em>) gene in <em>Bacillus globigii</em><a href="#pone.0000305-Newman1">[32]</a>. Probe 2 was complementary to a segment of the antisense strand of the <em>erg</em> gene in <em>Drosophila melanogaster</em><a href="#pone.0000305-Titus1">[17]</a>. Probes 3 to 30 were derived from various exons and/or introns of the human cystic fibrosis transmembrane conductance regulator gene (<em>CFTR</em>) <a href="#pone.0000305-Zielenski1">[33]</a>. Probes 31 and 32 were derived from exon 4 of the human methylenetetrahydrofolate reductase (<em>MTHFR</em>) gene <a href="#pone.0000305-Goyette1">[22]</a>. Probes 33 and 34 were derived from exon 10 of the human factor V gene <a href="#pone.0000305-Bertina1">[20]</a>.</p>


<h4>Genomic sample isolation</h4>
<a id="article1.body1.sec3.sec2.p1" name="article1.body1.sec3.sec2.p1"></a><p>
          <em>Bacillus globigii</em> (BG) genomic dsDNA was isolated from lysed vegetative BG cells using a QIAamp DNA blood mini-purification kit as per the manufacturer's instructions (Qiagen, Mississauga, Canada). <em>Drosophila</em> genomic dsDNA was purified from <em>Drosophila</em> wild-type Canton-S and mutant <em>sei<sup>ts1</sup></em> strains by multiple phenol-chloroform extractions and kindly provided by Dr. Barry Ganetzky (University of Wisconsin, Madison, WI, USA). Human genomic dsDNA was extracted from human blood using a QIAamp DNA blood purification kit (Qiagen, Mississauga, Canada) or from human saliva using an Oragene DNA collection kit (DNA Genotek, Ottawa, Canada) as per the manufacturers' instructions. The concentration of each genomic dsDNA sample was determined by UV spectroscopy.</p>


<h4>Genomic Assay®</h4>
<a id="article1.body1.sec3.sec3.p1" name="article1.body1.sec3.sec3.p1"></a><p>Different concentrations of genomic dsDNA were assayed, dependent on the source of genomic DNA. Unless otherwise indicated, the triplex binding reaction mixtures (80 µl) contained the following: between 4 ng and 75 pg of human genomic dsDNA, 3.2 pmoles of 25-mer ssDNA probe, 22.5 mM Tris-borate, pH 8.3, 0.5 mM EDTA (0.5×TBE) and 500 nM YOYO-1. YOYO-1, a hydrophobic bis-intercalator quenched by aqueous medium, was supplied by Molecular Probes (Eugene, Oregon, USA). Following a 5 min incubation at room temperature (RT) (22°C), the reaction mixtures were placed into separate wells of a Corning No Bind Surface 384-well plate (black with clear bottom, with universal lid) and irradiated through the bottom of the multi-well plate by the Genexus® Analyzer. The Genexus® Analyzer employs a Melles Griot 15 mW argon ion laser, Model 35-IMA-415-120, configured to deliver about 84 W/cm<sup>2</sup>/sec radiation at a wavelength of 488 nm to each sample. The Genexus® Analyzer, manufactured by Biomedical Photometrics Inc. (Waterloo, Ontario, Canada), utilizes Macroview software, which controls the scanning parameters of the confocal laser and provides quantitative analysis of scanned samples in the multi-well plates. Irradiation occurred at a sampling interval of 60 microns at settings of 20 hertz, 30–34% PMT and 10 µA/V sensitivity. At these scanning parameters, approximately 44,000 pixel emissions were measured per assay and averaged. Fluorescent emissions were also monitored over time. The Genexus® Analyzer allows 384 test samples to be scanned in approximately 2 min. Care must be taken in selecting means of stimulating YOYO-1 emission. YOYO-1 acts as a photocleavage agent if stimulated with overly energetic radiation. We have empirically determined that light-emitted diode (LED) irradiation, or laser irradiation of 84 W/cm<sup>2</sup>/sec can be optimal. Instruments powered by “bright light” sources, such as Xenon bulbs, that generate pulsed or continuous bright light, which is subsequently filtered, have not proved useful to carry out the Triplex Assay® using YOYO-1.</p>

</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0000305"></div><a name="pone.0000305.s001" id="pone.0000305.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000305.s001">Figure S1. </a></strong></p><a id="article1.body1.sec4.supplementary-material1.caption1.p1" name="article1.body1.sec4.supplementary-material1.caption1.p1"></a><p class="preSiDOI">Fluorescence emissions from varied YOYO-1 concentrations and a 25-mer ssDNA probe. 3.2 pmoles of a 25-mer ssDNA probe was reacted with various concentrations of YOYO-1, ranging from 50 nM to 1000 nM, at 50 nM intervals, in the presence of 0.5×TBE and 40 mM TMA-Cl. Fluorescent emissions of the reaction mixtures (80 ul) were monitored with the Genexus Analyzer at a setting of 32% PMT after 5, 15, 25, 35, 45, 55 and 65 minutes, and 24 hours of incubation at RT. Intensity of fluorescence is plotted as a function of YOYO-1 concentration for the probe:probe complexes formed after 5 min of incubation (1), 65 min of incubation (2) or 24 hours of incubation (3).</p>
<p class="siDoi">doi:10.1371/journal.pone.0000305.s001</p><a id="article1.body1.sec4.supplementary-material1.caption1.p2" name="article1.body1.sec4.supplementary-material1.caption1.p2"></a><p class="postSiDOI">(0.01 MB PDF)</p>
<a name="pone.0000305.s002" id="pone.0000305.s002"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000305.s002">Figure S2. </a></strong></p><a id="article1.body1.sec4.supplementary-material2.caption1.p1" name="article1.body1.sec4.supplementary-material2.caption1.p1"></a><p class="preSiDOI">Comparison of the fluorescence emissions from varied YOYO-1 concentrations and a 25-mer ssDNA probe in the presence or absence of a kosmotropic agent. 3.2 pmoles of a 25-mer ssDNA probe was reacted with various concentrations of YOYO-1, ranging from 50 nM to 1000 nM, at 50 nM intervals, in the presence of 0.5×TBE, and in the presence or absence of 40 mM TMA-Cl. Fluorescent emissions of the reaction mixtures (80 ul) were monitored with the Genexus Analyzer at a setting of 32% PMT after 5, 15, 25, 35, 45, 55 and 65 minutes, and 24 hours of incubation at RT. Intensity of fluorescence is plotted as a function of YOYO-1 concentration for the probe:probe complexes formed after 5 min of incubation in the presence (1) or absence (2) of 40 mM TMA-Cl.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000305.s002</p><a id="article1.body1.sec4.supplementary-material2.caption1.p2" name="article1.body1.sec4.supplementary-material2.caption1.p2"></a><p class="postSiDOI">(0.01 MB PDF)</p>
<a name="pone.0000305.s003" id="pone.0000305.s003"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000305.s003">Figure S3. </a></strong></p><a id="article1.body1.sec4.supplementary-material3.caption1.p1" name="article1.body1.sec4.supplementary-material3.caption1.p1"></a><p class="preSiDOI">Comparison of the fluorescence emissions from varied YOYO-1 concentrations and a 25-mer dsDNA in the presence or absence of a kosmotropic agent. 3.2 pmoles of a 25-mer dsDNA was reacted with various concentrations of YOYO-1, ranging from 50 nM to 1000 nM, at 50 nM intervals, in the presence of 0.5×TBE, and in the presence or absence of 40 mM TMA-Cl. Fluorescent emissions of the reaction mixtures (80 ul) were monitored with the Genexus Analyzer at a setting of 32% PMT after 5, 15, 25, 35, 45, 55 and 65 minutes, and 24 hours of incubation at RT. Intensity of fluorescence is plotted as a function of YOYO-1 concentration for the dsDNA:YOYO-1 complexes formed after 5 min of incubation in the presence (1) or absence (2) of 40 mM TMA-Cl.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000305.s003</p><a id="article1.body1.sec4.supplementary-material3.caption1.p2" name="article1.body1.sec4.supplementary-material3.caption1.p2"></a><p class="postSiDOI">(0.01 MB PDF)</p>
<a name="pone.0000305.s004" id="pone.0000305.s004"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000305.s004">Figure S4. </a></strong></p><a id="article1.body1.sec4.supplementary-material4.caption1.p1" name="article1.body1.sec4.supplementary-material4.caption1.p1"></a><p class="preSiDOI">Comparison of the fluorescence emissions from varied YOYO-1 concentrations and human genomic dsDNA in the presence or absence of a kosmotropic agent. Human genomic dsDNA was extracted from blood as described in the text. Two ng of wild-type human genomic dsDNA (approximately 302 copies) was reacted with various concentrations of YOYO-1, ranging from 50 nM to 1000 nM, at 50 nM intervals, in the presence of 0.5×TBE, and in the presence or absence of 40 mM TMA-Cl. Fluorescent emissions of the reaction mixtures (80 ul) were monitored with the Genexus Analyzer at a setting of 32% PMT after 5, 15, 25, 35, 45, 55 and 65 minutes, and 24 hours of incubation at RT. Intensity of fluorescence is plotted as a function of YOYO-1 concentration for the dsDNA:YOYO-1 complexes formed after 5 min of incubation in the presence (1) or absence (2) of 40 mM TMA-Cl.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000305.s004</p><a id="article1.body1.sec4.supplementary-material4.caption1.p2" name="article1.body1.sec4.supplementary-material4.caption1.p2"></a><p class="postSiDOI">(0.01 MB PDF)</p>
<a name="pone.0000305.s005" id="pone.0000305.s005"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000305.s005">Figure S5. </a></strong></p><a id="article1.body1.sec4.supplementary-material5.caption1.p1" name="article1.body1.sec4.supplementary-material5.caption1.p1"></a><p class="preSiDOI">Assays of human genomic dsDNA for CFTR 3849+10kbC→T (1 bp A–C mismatch) in the presence of 600 nM YOYO-1 and 40 mM TMA-Cl. Human genomic dsDNA was extracted from blood as described in the text. Two ng of wild-type human genomic dsDNA (approximately 302 copies) was reacted at RT with 3.2 pmoles of either wild-type or mutant ssDNA probe in the presence of 0.5×TBE, 40 mM TMA-Cl and 600 nM YOYO-1. YOYO-1 was added last to the reaction mixtures. Reaction mixtures (80 ul) were irradiated as described in the text and analyzed for fluorescent emission. The intensity of triplex-associated fluorescence is plotted as a function of incubation time for each sample analyzed. The samples consist of perfectly matched triplex (1) and mismatched triplex (2) as indicated for CFTR 3849+10kbC→T.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000305.s005</p><a id="article1.body1.sec4.supplementary-material5.caption1.p2" name="article1.body1.sec4.supplementary-material5.caption1.p2"></a><p class="postSiDOI">(0.01 MB PDF)</p>
<a name="pone.0000305.s006" id="pone.0000305.s006"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000305.s006">Figure S6. </a></strong></p><a id="article1.body1.sec4.supplementary-material6.caption1.p1" name="article1.body1.sec4.supplementary-material6.caption1.p1"></a><p class="preSiDOI">Assays of human genomic dsDNA for CFTR 3849+10kbC→T (1 bp A–C mismatch) in the presence of 500 nM YOYO-1 and 45 mM TMA-Cl. Human genomic dsDNA was extracted from blood as described in the text. Two ng of wild-type human genomic dsDNA (approximately 302 copies) was reacted at RT with 3.2 pmoles of either wild-type or mutant ssDNA probe in the presence of 0.5×TBE, 45 mM TMA-Cl and 500 nM YOYO-1. YOYO-1 was added last to the reaction mixtures. Reaction mixtures (80 ul) were irradiated as described in the text and analyzed for fluorescent emission. The intensity of triplex-associated fluorescence is plotted as a function of incubation time for each sample analyzed. The samples consist of perfectly matched triplex (1) and mismatched triplex (2) as indicated for CFTR 3849+10kbC→T.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000305.s006</p><a id="article1.body1.sec4.supplementary-material6.caption1.p2" name="article1.body1.sec4.supplementary-material6.caption1.p2"></a><p class="postSiDOI">(0.01 MB PDF)</p>
<a name="pone.0000305.s007" id="pone.0000305.s007"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000305.s007">Table S1. </a></strong></p><a id="article1.body1.sec4.supplementary-material7.caption1.p1" name="article1.body1.sec4.supplementary-material7.caption1.p1"></a><p class="preSiDOI">Assays of Bacillus globigii genomic dsDNA in the presence or absence of 2 ng human genomic dsDNA. 100 copies of Bacillus globigii genomic dsDNA are detected by triplex assay in reaction mixtures containing 300 nM YOYO-1 in the presence or absence of 2 ng human genomic dsDNA.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000305.s007</p><a id="article1.body1.sec4.supplementary-material7.caption1.p2" name="article1.body1.sec4.supplementary-material7.caption1.p2"></a><p class="postSiDOI">(0.05 MB DOC)</p>
<a name="pone.0000305.s008" id="pone.0000305.s008"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000305.s008">Table S2. </a></strong></p><a id="article1.body1.sec4.supplementary-material8.caption1.p1" name="article1.body1.sec4.supplementary-material8.caption1.p1"></a><p class="preSiDOI">Comparison of triplex assays of human genomic dsDNA using 15-mer, 20-mer or 25-mer ssDNA probes. Triplex assays of human genomic dsDNA for CFTR delta F508 using 15-mer, 20-mer or 25-mer ssDNA probes demonstrate optimal probe length under the conditions employed, to be 25 bases.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000305.s008</p><a id="article1.body1.sec4.supplementary-material8.caption1.p2" name="article1.body1.sec4.supplementary-material8.caption1.p2"></a><p class="postSiDOI">(0.06 MB DOC)</p>
<a name="pone.0000305.s009" id="pone.0000305.s009"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000305.s009">Table S3. </a></strong></p><a id="article1.body1.sec4.supplementary-material9.caption1.p1" name="article1.body1.sec4.supplementary-material9.caption1.p1"></a><p class="preSiDOI">Comparison of triplex assays of human genomic dsDNA using 20-mer, 25-mer or 30-mer ssDNA probes. Triplex assays of human genomic dsDNA for MTHFR C677T using 20-mer, 25-mer or 30-mer ssDNA probes demonstrate optimal probe length under the conditions employed, to be 25 bases.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000305.s009</p><a id="article1.body1.sec4.supplementary-material9.caption1.p2" name="article1.body1.sec4.supplementary-material9.caption1.p2"></a><p class="postSiDOI">(0.06 MB DOC)</p>
<a name="pone.0000305.s010" id="pone.0000305.s010"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000305.s010">Table S4. </a></strong></p><a id="article1.body1.sec4.supplementary-material10.caption1.p1" name="article1.body1.sec4.supplementary-material10.caption1.p1"></a><p class="preSiDOI">Assays of varying concentrations of human genomic dsDNA for Factor V Leiden (1 bp G–T mismatch). The specificity of the triplex assay in detecting FVL G1691A in mismatched triplexes is demonstrated over a broad range of human genomic dsDNA concentrations.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000305.s010</p><a id="article1.body1.sec4.supplementary-material10.caption1.p2" name="article1.body1.sec4.supplementary-material10.caption1.p2"></a><p class="postSiDOI">(0.07 MB DOC)</p>
<a name="pone.0000305.s011" id="pone.0000305.s011"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000305.s011">Table S5. </a></strong></p><a id="article1.body1.sec4.supplementary-material11.caption1.p1" name="article1.body1.sec4.supplementary-material11.caption1.p1"></a><p class="preSiDOI">Assays of varying concentrations of human genomic dsDNA for CFTR 2789+5G→A (1 bp T–G mismatch). The specificity of the triplex assay in detecting CFTR 2789+5G→A in mismatched triplexes is demonstrated over a broad range of human genomic dsDNA concentrations.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000305.s011</p><a id="article1.body1.sec4.supplementary-material11.caption1.p2" name="article1.body1.sec4.supplementary-material11.caption1.p2"></a><p class="postSiDOI">(0.08 MB DOC)</p>
<a name="pone.0000305.s012" id="pone.0000305.s012"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000305.s012">Table S6. </a></strong></p><a id="article1.body1.sec4.supplementary-material12.caption1.p1" name="article1.body1.sec4.supplementary-material12.caption1.p1"></a><p class="preSiDOI">Assays of human genomic dsDNA (purified from blood) for CFTR 2789+5G→A (1 bp T–G mismatch). The specificity of the triplex assay in detecting CFTR 2789+5G→A in mismatched triplexes comprising human genomic dsDNA purified from blood is demonstrated.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000305.s012</p><a id="article1.body1.sec4.supplementary-material12.caption1.p2" name="article1.body1.sec4.supplementary-material12.caption1.p2"></a><p class="postSiDOI">(0.04 MB DOC)</p>
<a name="pone.0000305.s013" id="pone.0000305.s013"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000305.s013">Table S7. </a></strong></p><a id="article1.body1.sec4.supplementary-material13.caption1.p1" name="article1.body1.sec4.supplementary-material13.caption1.p1"></a><p class="preSiDOI">Assays of human genomic dsDNA (wild-type homozygous or mutant heterozygous samples purified from blood) for MTHFR C677T. The specificity of the triplex assay in assaying wild-type homozygous and mutant heterozygous human genomic dsDNA samples for MTHFR C677T is demonstrated.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000305.s013</p><a id="article1.body1.sec4.supplementary-material13.caption1.p2" name="article1.body1.sec4.supplementary-material13.caption1.p2"></a><p class="postSiDOI">(0.06 MB DOC)</p>
<a name="pone.0000305.s014" id="pone.0000305.s014"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000305.s014">Table S8. </a></strong></p><a id="article1.body1.sec4.supplementary-material14.caption1.p1" name="article1.body1.sec4.supplementary-material14.caption1.p1"></a><p class="preSiDOI">Assays of human genomic dsDNA (mutant homozygous sample purified from blood) for MTHFR C677T. The specificity of the triplex assay in assaying mutant homozygous human genomic dsDNA samples for MTHFR C677T is demonstrated.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000305.s014</p><a id="article1.body1.sec4.supplementary-material14.caption1.p2" name="article1.body1.sec4.supplementary-material14.caption1.p2"></a><p class="postSiDOI">(0.05 MB DOC)</p>
<a name="pone.0000305.s015" id="pone.0000305.s015"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000305.s015">Table S9. </a></strong></p><a id="article1.body1.sec4.supplementary-material15.caption1.p1" name="article1.body1.sec4.supplementary-material15.caption1.p1"></a><p class="preSiDOI">Assays of varying concentrations of human genomic dsDNA for MTHFR C677T (1 bp C–A mismatch). The specificity of the triplex assay in detecting MTHFR C677T in mismatched triplexes is demonstrated over a broad range of human genomic dsDNA concentrations.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000305.s015</p><a id="article1.body1.sec4.supplementary-material15.caption1.p2" name="article1.body1.sec4.supplementary-material15.caption1.p2"></a><p class="postSiDOI">(0.07 MB DOC)</p>
<a name="pone.0000305.s016" id="pone.0000305.s016"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000305.s016">Table S10. </a></strong></p><a id="article1.body1.sec4.supplementary-material16.caption1.p1" name="article1.body1.sec4.supplementary-material16.caption1.p1"></a><p class="preSiDOI">Assays of human genomic dsDNA employing varying protocols. The specificity of the triplex assay in detecting CFTR 3849+10kbC→T in mismatched triplexes is demonstrated in reaction mixtures containing a kosmotropic agent. Two reaction protocols are compared.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000305.s016</p><a id="article1.body1.sec4.supplementary-material16.caption1.p2" name="article1.body1.sec4.supplementary-material16.caption1.p2"></a><p class="postSiDOI">(0.05 MB DOC)</p>
<a name="pone.0000305.s017" id="pone.0000305.s017"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000305.s017">Table S11. </a></strong></p><a id="article1.body1.sec4.supplementary-material17.caption1.p1" name="article1.body1.sec4.supplementary-material17.caption1.p1"></a><p class="preSiDOI">Assays of human genomic dsDNA for CFTR 3849+10kbC→T (1 bp A–C mismatch) in the presence of 600 nM YOYO-1 and 40 mM TMA-Cl. The emission values giving rise to <a href="#pone.0000305.s005">Supplementary Figure S5</a> data are shown. The specificity of the triplex assay in detecting CFTR 3849+10kbC→T in mismatched triplexes is demonstrated in reaction mixtures containing 600 nM YOYO-1 and 40 mM TMA-Cl.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000305.s017</p><a id="article1.body1.sec4.supplementary-material17.caption1.p2" name="article1.body1.sec4.supplementary-material17.caption1.p2"></a><p class="postSiDOI">(0.05 MB DOC)</p>
<a name="pone.0000305.s018" id="pone.0000305.s018"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000305.s018">Table S12. </a></strong></p><a id="article1.body1.sec4.supplementary-material18.caption1.p1" name="article1.body1.sec4.supplementary-material18.caption1.p1"></a><p class="preSiDOI">Assays of human genomic dsDNA for CFTR 3849+10kbC→T (1 bp A–C mismatch) in the presence of 500 nM YOYO-1 and 45 mM TMA-Cl. The emission values giving rise to <a href="#pone.0000305.s006">Supplementary Figure S6</a> data are shown. The specificity of the triplex assay in detecting CFTR 3849+10kbC→T in mismatched triplexes is demonstrated in reaction mixtures containing 500 nM YOYO-1 and 45 mM TMA-Cl.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000305.s018</p><a id="article1.body1.sec4.supplementary-material18.caption1.p2" name="article1.body1.sec4.supplementary-material18.caption1.p2"></a><p class="postSiDOI">(0.05 MB DOC)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>We thank I. Kandic and Dr. B. Tamot for excellent technical assistance performing experiments, and Dr. B. Ganetzky for generously providing genomic <em>drosophila</em> samples. We are indebted to Dr. C. Preston for critical review of the manuscript. Ingeneus Triplex®, Triplex Assay®, Genomic Assay® and Genexus® are registered trademarks of Ingeneus Inc. and are used under license.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: JD GE. Performed the experiments: JD. Analyzed the data: JD GE. Wrote the paper: JD GE.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0000305-FrankKamenetskii1" id="pone.0000305-FrankKamenetskii1"></a>Frank-Kamenetskii MD (1990)  DNA supercoiling and unusual structures. In: Cozzarelli NR, Wang JC, editors. DNA Topology and its Biological Effects. Cold Spring Harbor: Cold Spring Harbor Laboratory Press. pp. 185–215. <ul class="find-nolinks"></ul></li><li><span class="label">2.
              </span><a name="pone.0000305-Young1" id="pone.0000305-Young1"></a>Young SL, Krawczyk SH, Matteucci MD, Toole JJ (1991) Triple helix formation inhibits transcription elongation in vitro. Proc Natl Acad Sci U S A  88: 10023–10026.  <ul class="find" data-citedArticleID="938554" data-doi="10.1073/pnas.88.22.10023"><li><a href="http://dx.doi.org/10.1073/pnas.88.22.10023" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Triple+helix+formation+inhibits+transcription+elongation+in+vitro." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Triple+helix+formation+inhibits+transcription+elongation+in+vitro.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0000305-Grigoriev1" id="pone.0000305-Grigoriev1"></a>Grigoriev M, Praseuth D, Robin P, Hemar A, Saison-Behmoaras T, et al.  (1992) A triple helix-forming oligonucleotide-intercalator conjugate acts as a transcriptional repressor via inhibition of NF kappa B binding to interlelukin-2 receptor alpha-regulatory sequence. J Biol Chem  267: 3389–3395.  <ul class="find" data-citedArticleID="938518"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=A+triple+helix-forming+oligonucleotide-intercalator+conjugate+acts+as+a+transcriptional+repressor+via+inhibition+of+NF+kappa+B+binding+to+interlelukin-2+receptor+alpha-regulatory+sequence.&amp;auth=&amp;atitle=A+triple+helix-forming+oligonucleotide-intercalator+conjugate+acts+as+a+transcriptional+repressor+via+inhibition+of+NF+kappa+B+binding+to+interlelukin-2+receptor+alpha-regulatory+sequence." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+triple+helix-forming+oligonucleotide-intercalator+conjugate+acts+as+a+transcriptional+repressor+via+inhibition+of+NF+kappa+B+binding+to+interlelukin-2+receptor+alpha-regulatory+sequence." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+triple+helix-forming+oligonucleotide-intercalator+conjugate+acts+as+a+transcriptional+repressor+via+inhibition+of+NF+kappa+B+binding+to+interlelukin-2+receptor+alpha-regulatory+sequence.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0000305-Helene1" id="pone.0000305-Helene1"></a>Helene C, Thuong NT, Harel-Bellan A (1992) Control of gene expression by triple helix-forming oligonucleotides. The antigene strategy. Ann NY Acad Sci  660: 27–36.  <ul class="find" data-citedArticleID="938522" data-doi="10.1111/j.1749-6632.1992.tb21054.x"><li><a href="http://dx.doi.org/10.1111/j.1749-6632.1992.tb21054.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Control+of+gene+expression+by+triple+helix-forming+oligonucleotides.+The+antigene+strategy." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Control+of+gene+expression+by+triple+helix-forming+oligonucleotides.+The+antigene+strategy.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0000305-Maher1" id="pone.0000305-Maher1"></a>Maher LJ 3rd, Dervan PB, Wold B (1992) Analysis of promoter-specific repression by triple-helical DNA complexes in a eukaryotic cell-free transcription system. Biochemistry  31: 70–81.  <ul class="find" data-citedArticleID="938532" data-doi="10.1021/bi00116a012"><li><a href="http://dx.doi.org/10.1021/bi00116a012" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Analysis+of+promoter-specific+repression+by+triple-helical+DNA+complexes+in+a+eukaryotic+cell-free+transcription+system." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Analysis+of+promoter-specific+repression+by+triple-helical+DNA+complexes+in+a+eukaryotic+cell-free+transcription+system.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0000305-Chan1" id="pone.0000305-Chan1"></a>Chan PP, Glazer PM (1997) Triplex DNA: fundamentals, advances, and potential applications for gene therapy. J Mol Med  75: 267–282.  <ul class="find" data-citedArticleID="938502" data-doi="10.1007/s001090050112"><li><a href="http://dx.doi.org/10.1007/s001090050112" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Triplex+DNA%3A+fundamentals%2C+advances%2C+and+potential+applications+for+gene+therapy." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Triplex+DNA%3A+fundamentals%2C+advances%2C+and+potential+applications+for+gene+therapy.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0000305-Vasquez1" id="pone.0000305-Vasquez1"></a>Vasquez KM, Wilson JH (1998) Triplex-directed modification of genes and gene activity. Trends Biochem Sci  23: 4–9.  <ul class="find" data-citedArticleID="938552" data-doi="10.1016/s0968-0004(97)01158-4"><li><a href="http://dx.doi.org/10.1016/s0968-0004(97)01158-4" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Triplex-directed+modification+of+genes+and+gene+activity." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Triplex-directed+modification+of+genes+and+gene+activity.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0000305-Seidman1" id="pone.0000305-Seidman1"></a>Seidman MM, Glazer PM (2003) The potential for gene repair via triple helix formation. J Clin Invest  112: 487–494.  <ul class="find" data-citedArticleID="938542" data-doi="10.1172/jci19552"><li><a href="http://dx.doi.org/10.1172/jci19552" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+potential+for+gene+repair+via+triple+helix+formation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+potential+for+gene+repair+via+triple+helix+formation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0000305-Cheng1" id="pone.0000305-Cheng1"></a>Cheng AJ, Van Dyke MW (1994) Oligodeoxyribonucleotide length and sequence effects on intermolecular purine-purine-pyrimidine triple-helix formation. Nucleic Acids Res  22: 4742–4747.  <ul class="find" data-citedArticleID="938504" data-doi="10.1093/nar/22.22.4742"><li><a href="http://dx.doi.org/10.1093/nar/22.22.4742" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Oligodeoxyribonucleotide+length+and+sequence+effects+on+intermolecular+purine-purine-pyrimidine+triple-helix+formation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Oligodeoxyribonucleotide+length+and+sequence+effects+on+intermolecular+purine-purine-pyrimidine+triple-helix+formation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0000305-Mergny1" id="pone.0000305-Mergny1"></a>Mergny JL, Sun JS, Rougee M, Montenay-Garestier T, Barcelo F, et al.  (1991) Sequence specificity in triple-helix formation: experimental and theoretical studies of the effect of mismatches on triplex stability. Biochemistry  30: 9791–9798.  <ul class="find" data-citedArticleID="938536" data-doi="10.1021/bi00104a031"><li><a href="http://dx.doi.org/10.1021/bi00104a031" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Sequence+specificity+in+triple-helix+formation%3A+experimental+and+theoretical+studies+of+the+effect+of+mismatches+on+triplex+stability." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Sequence+specificity+in+triple-helix+formation%3A+experimental+and+theoretical+studies+of+the+effect+of+mismatches+on+triplex+stability.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0000305-Singleton1" id="pone.0000305-Singleton1"></a>Singleton SF, Dervan PB (1992) Influence of pH on the equilibrium association constants for oligodeoxyribonucleotide-directed triple helix formation at single DNA sites. Biochemistry  31: 10995–11003.  <ul class="find" data-citedArticleID="938544" data-doi="10.1021/bi00160a008"><li><a href="http://dx.doi.org/10.1021/bi00160a008" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Influence+of+pH+on+the+equilibrium+association+constants+for+oligodeoxyribonucleotide-directed+triple+helix+formation+at+single+DNA+sites." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Influence+of+pH+on+the+equilibrium+association+constants+for+oligodeoxyribonucleotide-directed+triple+helix+formation+at+single+DNA+sites.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0000305-FrankKamenetskii2" id="pone.0000305-FrankKamenetskii2"></a>Frank-Kamenetskii MD, Mirkin SM (1995) Triplex DNA structures. Annu Rev Biochem  64: 65–95.  <ul class="find" data-citedArticleID="938514" data-doi="10.1146/annurev.bi.64.070195.000433"><li><a href="http://dx.doi.org/10.1146/annurev.bi.64.070195.000433" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Triplex+DNA+structures." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Triplex+DNA+structures.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0000305-Leitner1" id="pone.0000305-Leitner1"></a>Leitner D, Weisz K (2000) Sequence-dependent stability of intramolecular DNA triple helices. J Biomolec Struc and Dynamics  17: 993–1000.  <ul class="find" data-citedArticleID="938530" data-doi="10.1080/07391102.2000.10506587"><li><a href="http://dx.doi.org/10.1080/07391102.2000.10506587" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Sequence-dependent+stability+of+intramolecular+DNA+triple+helices." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Sequence-dependent+stability+of+intramolecular+DNA+triple+helices.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0000305-Daksis1" id="pone.0000305-Daksis1"></a>Daksis JI, Erikson GH (2005) Specific triplex binding capacity of mixed base sequence duplex nucleic acids used for single-nucleotide polymorphism detection. Genetic Testing  9: 111–120.  <ul class="find" data-citedArticleID="938508" data-doi="10.1089/gte.2005.9.111"><li><a href="http://dx.doi.org/10.1089/gte.2005.9.111" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Specific+triplex+binding+capacity+of+mixed+base+sequence+duplex+nucleic+acids+used+for+single-nucleotide+polymorphism+detection." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Specific+triplex+binding+capacity+of+mixed+base+sequence+duplex+nucleic+acids+used+for+single-nucleotide+polymorphism+detection.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0000305-Raggi1" id="pone.0000305-Raggi1"></a>Raggi CC, Pinzani P, Paradiso A, Pazzagli M, Orlando C (2003) External quality assurance program for PCR amplification of genomic DNA: an Italian experience. Clin Chem  49: 782–791.  <ul class="find" data-citedArticleID="938540" data-doi="10.1373/49.5.782"><li><a href="http://dx.doi.org/10.1373/49.5.782" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=External+quality+assurance+program+for+PCR+amplification+of+genomic+DNA%3A+an+Italian+experience." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22External+quality+assurance+program+for+PCR+amplification+of+genomic+DNA%3A+an+Italian+experience.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0000305-Birch1" id="pone.0000305-Birch1"></a>Birch L, English CA, Burns M, Keer JT (2004) Generic scheme for independent performance assessment in the molecular biology laboratory. Clin Chem  50: 1553–1559.  <ul class="find" data-citedArticleID="938500" data-doi="10.1373/clinchem.2003.029454"><li><a href="http://dx.doi.org/10.1373/clinchem.2003.029454" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Generic+scheme+for+independent+performance+assessment+in+the+molecular+biology+laboratory." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Generic+scheme+for+independent+performance+assessment+in+the+molecular+biology+laboratory.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0000305-Titus1" id="pone.0000305-Titus1"></a>Titus SA, Warmke JW, Ganetzky B (1997) The <em>drosophila erg</em> K<sup>+</sup> channel polypeptide is encoded by the seizure locus. J Neurosci  17: 875–881.  <ul class="find" data-citedArticleID="938550"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=The+drosophila+erg+K%2B+channel+polypeptide+is+encoded+by+the+seizure+locus.&amp;auth=&amp;atitle=The+drosophila+erg+K%2B+channel+polypeptide+is+encoded+by+the+seizure+locus." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+drosophila+erg+K%2B+channel+polypeptide+is+encoded+by+the+seizure+locus." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+drosophila+erg+K%2B+channel+polypeptide+is+encoded+by+the+seizure+locus.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0000305-Curran1" id="pone.0000305-Curran1"></a>Curran ME, Splawski I, Timothy KW, Vincent M, Green ED, et al.  (1995) A molecular basis for cardiac arrhythmia: <em>HERG</em> mutations cause long QT syndrome. Cell  80: 795–804.  <ul class="find" data-citedArticleID="938506" data-doi="10.1016/0092-8674(95)90358-5"><li><a href="http://dx.doi.org/10.1016/0092-8674(95)90358-5" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+molecular+basis+for+cardiac+arrhythmia%3A+HERG+mutations+cause+long+QT+syndrome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+molecular+basis+for+cardiac+arrhythmia%3A+HERG+mutations+cause+long+QT+syndrome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0000305-Grody1" id="pone.0000305-Grody1"></a>Grody WW, Cutting GR, Klinger KW, Richards CS, Watson MS, et al.  (2001) Laboratory standards and guidelines for population-based cystic fibrosis carrier screening. Genet Med  3: 149–154.  <ul class="find" data-citedArticleID="938520" data-doi="10.1097/00125817-200103000-00010"><li><a href="http://dx.doi.org/10.1097/00125817-200103000-00010" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Laboratory+standards+and+guidelines+for+population-based+cystic+fibrosis+carrier+screening." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Laboratory+standards+and+guidelines+for+population-based+cystic+fibrosis+carrier+screening.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0000305-Bertina1" id="pone.0000305-Bertina1"></a>Bertina RM, Koeleman BPC, Koster T, Rosendaal FR, Dirven RJ, et al.  (1994) Mutation in blood coagulation factor V associated with resistance to activated protein C. Nature  369: 64–67.  <ul class="find" data-citedArticleID="938496" data-doi="10.1038/369064a0"><li><a href="http://dx.doi.org/10.1038/369064a0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mutation+in+blood+coagulation+factor+V+associated+with+resistance+to+activated+protein+C." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mutation+in+blood+coagulation+factor+V+associated+with+resistance+to+activated+protein+C.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0000305-Svensson1" id="pone.0000305-Svensson1"></a>Svensson PJ, Dahlback B (1994) Resistance to activated protein C as a basis for venous thrombosis. N Engl J Med  330: 517–522.  <ul class="find" data-citedArticleID="938548" data-doi="10.1056/nejm199402243300801"><li><a href="http://dx.doi.org/10.1056/nejm199402243300801" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Resistance+to+activated+protein+C+as+a+basis+for+venous+thrombosis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Resistance+to+activated+protein+C+as+a+basis+for+venous+thrombosis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0000305-Goyette1" id="pone.0000305-Goyette1"></a>Goyette P, Sumner JS, Milos R, Duncan AM, Rosenblatt DS, et al.  (1994) Human methylenetetrahydrofolate reductase: isolation of cDNA, mapping and mutation identification. Nat Genet  7: 195–200.  <ul class="find" data-citedArticleID="938516" data-doi="10.1038/ng0694-195"><li><a href="http://dx.doi.org/10.1038/ng0694-195" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Human+methylenetetrahydrofolate+reductase%3A+isolation+of+cDNA%2C+mapping+and+mutation+identification." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Human+methylenetetrahydrofolate+reductase%3A+isolation+of+cDNA%2C+mapping+and+mutation+identification.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0000305-Johansen1" id="pone.0000305-Johansen1"></a>Johansen F, Jacobsen JP (1998) 1H NMR studies of the bis-intercalation of a homodimeric oxazole yellow dye in DNA oligonucleotides. J Biomolec Struc and Dynamics  16: 205–222.  <ul class="find" data-citedArticleID="938524" data-doi="10.1080/07391102.1998.10508240"><li><a href="http://dx.doi.org/10.1080/07391102.1998.10508240" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=1H+NMR+studies+of+the+bis-intercalation+of+a+homodimeric+oxazole+yellow+dye+in+DNA+oligonucleotides." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%221H+NMR+studies+of+the+bis-intercalation+of+a+homodimeric+oxazole+yellow+dye+in+DNA+oligonucleotides.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0000305-Stasiak1" id="pone.0000305-Stasiak1"></a>Stasiak A, Di Capua E, Koller T (1981) The elongation of duplex DNA in complexes with RecA protein. J Mol Biol  151: 557–564.  <ul class="find" data-citedArticleID="938546" data-doi="10.1016/0022-2836(81)90010-3"><li><a href="http://dx.doi.org/10.1016/0022-2836(81)90010-3" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+elongation+of+duplex+DNA+in+complexes+with+RecA+protein." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+elongation+of+duplex+DNA+in+complexes+with+RecA+protein.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0000305-Egelman1" id="pone.0000305-Egelman1"></a>Egelman EH (1993) What do x-ray crystallographic and electron microscopic structural studies of the RecA protein tell us about recombination? Curr Opin Struct Biol  3: 189–197.  <ul class="find" data-citedArticleID="938510" data-doi="10.1016/s0959-440x(05)80151-4"><li><a href="http://dx.doi.org/10.1016/s0959-440x(05)80151-4" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=What+do+x-ray+crystallographic+and+electron+microscopic+structural+studies+of+the+RecA+protein+tell+us+about+recombination%3F" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22What+do+x-ray+crystallographic+and+electron+microscopic+structural+studies+of+the+RecA+protein+tell+us+about+recombination%3F%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0000305-Zhurkin1" id="pone.0000305-Zhurkin1"></a>Zhurkin VB, Raghunathan G, Ulyanov NB, Camerini-Otero RD, Jernigan RL (1994) A parallel DNA triplex as a model for the intermediate in homologous recombination. J Mol Biol  239: 181–200.  <ul class="find" data-citedArticleID="938558" data-doi="10.1006/jmbi.1994.1362"><li><a href="http://dx.doi.org/10.1006/jmbi.1994.1362" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+parallel+DNA+triplex+as+a+model+for+the+intermediate+in+homologous+recombination." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+parallel+DNA+triplex+as+a+model+for+the+intermediate+in+homologous+recombination.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0000305-Kim1" id="pone.0000305-Kim1"></a>Kim MG, Zhurkin VB, Jernigan RL, Camerini-Otero RD (1995) Probing the structure of a putative intermediate in homologous recombination: the third strand in the parallel DNA triplex is in contact with the major groove of the duplex. J Mol Biol  247: 874–889.  <ul class="find" data-citedArticleID="938526" data-doi="10.1006/jmbi.1994.0187"><li><a href="http://dx.doi.org/10.1006/jmbi.1994.0187" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Probing+the+structure+of+a+putative+intermediate+in+homologous+recombination%3A+the+third+strand+in+the+parallel+DNA+triplex+is+in+contact+with+the+major+groove+of+the+duplex." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Probing+the+structure+of+a+putative+intermediate+in+homologous+recombination%3A+the+third+strand+in+the+parallel+DNA+triplex+is+in+contact+with+the+major+groove+of+the+duplex.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0000305-Kosikov1" id="pone.0000305-Kosikov1"></a>Kosikov KM, Gorin AA, Zhurkin VB, Olson WK (1999) DNA stretching and compression: large-scale simulations of double helical structures. J Mol Biol  289: 1301–1326.  <ul class="find" data-citedArticleID="938528" data-doi="10.1006/jmbi.1999.2798"><li><a href="http://dx.doi.org/10.1006/jmbi.1999.2798" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=DNA+stretching+and+compression%3A+large-scale+simulations+of+double+helical+structures." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22DNA+stretching+and+compression%3A+large-scale+simulations+of+double+helical+structures.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0000305-Malkov1" id="pone.0000305-Malkov1"></a>Malkov VA, Panyutin IG, Neumann RD, Zhurkin VB, Camerini-Otero RD (2000) Radioprobing of a RecA-three-stranded DNA complex with iodine 125: evidence for recognition of homology in the major groove of the target duplex. J Mol Biol  299: 629–640.  <ul class="find" data-citedArticleID="938534" data-doi="10.1006/jmbi.2000.3770"><li><a href="http://dx.doi.org/10.1006/jmbi.2000.3770" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Radioprobing+of+a+RecA-three-stranded+DNA+complex+with+iodine+125%3A+evidence+for+recognition+of+homology+in+the+major+groove+of+the+target+duplex." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Radioprobing+of+a+RecA-three-stranded+DNA+complex+with+iodine+125%3A+evidence+for+recognition+of+homology+in+the+major+groove+of+the+target+duplex.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0000305-Bertucat1" id="pone.0000305-Bertucat1"></a>Bertucat G, Lavery R, Prevost C (2000) A mechanism for RecA-promoted sequence homology recognition and strand exchange between single-stranded DNA and duplex DNA, via triple-helical intermediates. J Biomolec Struc and Dynamics Conversation  11: 147–153.  <ul class="find" data-citedArticleID="938498" data-doi="10.1080/07391102.2000.10506615"><li><a href="http://dx.doi.org/10.1080/07391102.2000.10506615" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+mechanism+for+RecA-promoted+sequence+homology+recognition+and+strand+exchange+between+single-stranded+DNA+and+duplex+DNA%2C+via+triple-helical+intermediates." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+mechanism+for+RecA-promoted+sequence+homology+recognition+and+strand+exchange+between+single-stranded+DNA+and+duplex+DNA%2C+via+triple-helical+intermediates.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0000305-Zhang1" id="pone.0000305-Zhang1"></a>Zhang Y, Hammer DA, Graves DJ (2005) Competitive hybridization kinetics reveals unexpected behavior patterns. Biophysical J  89: 2950–2959.  <ul class="find" data-citedArticleID="938556" data-doi="10.1529/biophysj.104.058552"><li><a href="http://dx.doi.org/10.1529/biophysj.104.058552" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Competitive+hybridization+kinetics+reveals+unexpected+behavior+patterns." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Competitive+hybridization+kinetics+reveals+unexpected+behavior+patterns.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0000305-Newman1" id="pone.0000305-Newman1"></a>Newman M, Lunnen K, Wilson G, Greci J, Schildkraut I, et al.  (1998) Crystal structure of restriction endonuclease BglI bound to its interrupted DNA recognition sequence. EMBO J  17: 5466–5476.  <ul class="find" data-citedArticleID="938538" data-doi="10.1093/emboj/17.18.5466"><li><a href="http://dx.doi.org/10.1093/emboj/17.18.5466" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Crystal+structure+of+restriction+endonuclease+BglI+bound+to+its+interrupted+DNA+recognition+sequence." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Crystal+structure+of+restriction+endonuclease+BglI+bound+to+its+interrupted+DNA+recognition+sequence.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0000305-Zielenski1" id="pone.0000305-Zielenski1"></a>Zielenski J, Rozmahel R, Bozon D, Kerem B, Grzelczak Z, et al.  (1991) Genomic DNA sequence of the cystic fibrosis transmembrane conductance regulator (<em>CFTR</em>) gene. Genomics  10: 214–228.  <ul class="find" data-citedArticleID="938560" data-doi="10.1016/0888-7543(91)90503-7"><li><a href="http://dx.doi.org/10.1016/0888-7543(91)90503-7" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genomic+DNA+sequence+of+the+cystic+fibrosis+transmembrane+conductance+regulator+%28CFTR%29+gene." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genomic+DNA+sequence+of+the+cystic+fibrosis+transmembrane+conductance+regulator+%28CFTR%29+gene.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000305.XML" value="155966"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000305.PDF" value="234856"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000305.g001.PNG_L" value="22911"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000305.g001.PNG_M" value="27082"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000305.g001.PNG_S" value="2257"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000305.g001.TIF" value="44070"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000305.g001.PNG_I" value="13603"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000305.t001.PNG_L" value="295100"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000305.t001.PNG_M" value="116849"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000305.t001.PNG_S" value="12889"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000305.t001.TIF" value="1360584"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000305.t001.PNG_I" value="203371"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000305.g002.PNG_L" value="28594"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000305.g002.PNG_M" value="32861"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000305.g002.PNG_S" value="2456"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000305.g002.TIF" value="51832"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000305.g002.PNG_I" value="16177"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000305.g003.PNG_L" value="94922"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000305.g003.PNG_M" value="29513"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000305.g003.PNG_S" value="2302"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000305.g003.TIF" value="162102"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000305.g003.PNG_I" value="16659"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000305.t002.PNG_L" value="368533"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000305.t002.PNG_M" value="133823"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000305.t002.PNG_S" value="5854"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000305.t002.TIF" value="1658232"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000305.t002.PNG_I" value="54943"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000305.t003.PNG_L" value="267287"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000305.t003.PNG_M" value="94041"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000305.t003.PNG_S" value="4629"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000305.t003.TIF" value="1143610"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000305.t003.PNG_I" value="38081"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000305.g004.PNG_L" value="33304"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000305.g004.PNG_M" value="39542"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000305.g004.PNG_S" value="2522"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000305.g004.TIF" value="59114"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000305.g004.PNG_I" value="19768"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000305.t004.PNG_L" value="163021"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000305.t004.PNG_M" value="54531"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000305.t004.PNG_S" value="2917"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000305.t004.TIF" value="786042"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000305.t004.PNG_I" value="22845"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000305.g005.PNG_L" value="47871"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000305.g005.PNG_M" value="36609"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000305.g005.PNG_S" value="2943"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000305.g005.TIF" value="84682"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000305.g005.PNG_I" value="28327"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000305.s001.PDF" value="10816"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000305.s002.PDF" value="10117"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000305.s003.PDF" value="10231"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000305.s004.PDF" value="10008"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000305.s005.PDF" value="9418"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000305.s006.PDF" value="9400"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000305.s007.DOC" value="46592"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000305.s008.DOC" value="58368"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000305.s009.DOC" value="58368"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000305.s010.DOC" value="66048"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000305.s011.DOC" value="78848"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000305.s012.DOC" value="37376"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000305.s013.DOC" value="60928"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000305.s014.DOC" value="50688"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000305.s015.DOC" value="66048"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000305.s016.DOC" value="50176"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000305.s017.DOC" value="52736"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000305.s018.DOC" value="52736"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000305&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000305" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000305&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0000305&volume=&issue=&title=Heteropolymeric Triplex-Based Genomic Assay® to Detect Pathogens or Single-Nucleotide Polymorphisms in Human Genomic Samples&author_name=Jasmine%20I.%20Daksis%2C%20Glen%20H.%20Erikson&start_page=1&end_page=14" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000305" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000305" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0000305" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000305&amp;t=Heteropolymeric%20Triplex-Based%20Genomic%20Assay%C2%AE%20to%20Detect%20Pathogens%20or%20Single-Nucleotide%20Polymorphisms%20in%20Human%20Genomic%20Samples" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000305&title=Heteropolymeric%20Triplex-Based%20Genomic%20Assay%C2%AE%20to%20Detect%20Pathogens%20or%20Single-Nucleotide%20Polymorphisms%20in%20Human%20Genomic%20Samples&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000305&amp;title=Heteropolymeric%20Triplex-Based%20Genomic%20Assay%C2%AE%20to%20Detect%20Pathogens%20or%20Single-Nucleotide%20Polymorphisms%20in%20Human%20Genomic%20Samples" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000305" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0000305&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Heteropolymeric Triplex-Based Genomic Assay® to Detect Pathogens or Single-Nucleotide Polymorphisms in Human Genomic Samples';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0000305';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Heteropolymeric%20Triplex-Based%20Genomic%20Assay%C2%AE%20to%20Detect%20Pathogens%20or%20Single-Nucleotide%20Polymorphisms%20in%20Human%20Genomic%20Samples http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000305" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0000305" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">
















          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Bacterial+genomics%22" title="Search for articles in the subject area:'Bacterial genomics'"><div class="flagText">Bacterial genomics</div></a>
              <div data-categoryid="17227" data-articleid="23448"
                   data-categoryname="Bacterial genomics"
                   class="flagImage" title="Flag 'Bacterial genomics' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Blood%22" title="Search for articles in the subject area:'Blood'"><div class="flagText">Blood</div></a>
              <div data-categoryid="47189" data-articleid="23448"
                   data-categoryname="Blood"
                   class="flagImage" title="Flag 'Blood' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Fluorescence%22" title="Search for articles in the subject area:'Fluorescence'"><div class="flagText">Fluorescence</div></a>
              <div data-categoryid="17783" data-articleid="23448"
                   data-categoryname="Fluorescence"
                   class="flagImage" title="Flag 'Fluorescence' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Genome+complexity%22" title="Search for articles in the subject area:'Genome complexity'"><div class="flagText">Genome complexity</div></a>
              <div data-categoryid="18711" data-articleid="23448"
                   data-categoryname="Genome complexity"
                   class="flagImage" title="Flag 'Genome complexity' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Genomic+signal+processing%22" title="Search for articles in the subject area:'Genomic signal processing'"><div class="flagText">Genomic signal processing</div></a>
              <div data-categoryid="18635" data-articleid="23448"
                   data-categoryname="Genomic signal processing"
                   class="flagImage" title="Flag 'Genomic signal processing' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Human+genomics%22" title="Search for articles in the subject area:'Human genomics'"><div class="flagText">Human genomics</div></a>
              <div data-categoryid="17253" data-articleid="23448"
                   data-categoryname="Human genomics"
                   class="flagImage" title="Flag 'Human genomics' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Invertebrate+genomics%22" title="Search for articles in the subject area:'Invertebrate genomics'"><div class="flagText">Invertebrate genomics</div></a>
              <div data-categoryid="17221" data-articleid="23448"
                   data-categoryname="Invertebrate genomics"
                   class="flagImage" title="Flag 'Invertebrate genomics' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Polymerase+chain+reaction%22" title="Search for articles in the subject area:'Polymerase chain reaction'"><div class="flagText">Polymerase chain reaction</div></a>
              <div data-categoryid="26251" data-articleid="23448"
                   data-categoryname="Polymerase chain reaction"
                   class="flagImage" title="Flag 'Polymerase chain reaction' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=2643'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=2704'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=9830&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>


</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
