

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Plant Carbohydrate Scavenging through TonB-Dependent Receptors: A Feature Shared by Phytopathogenic and Aquatic Bacteria</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0000224"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0000224" />

    <meta name="citation_title" content="Plant Carbohydrate Scavenging through TonB-Dependent Receptors: A Feature Shared by Phytopathogenic and Aquatic Bacteria"/>
    <meta itemprop="name" content="Plant Carbohydrate Scavenging through TonB-Dependent Receptors: A Feature Shared by Phytopathogenic and Aquatic Bacteria"/>

      <meta name="citation_author" content="Servane Blanvillain"/>
            <meta name="citation_author_institution" content="Laboratoire des Interactions Plantes-Microorganismes, Centre National de la Recherche Scientifique (CNRS)/Institut National de la Recherche Agronomique (INRA) UMR2594/441, Castanet-Tolosan, France"/>
      <meta name="citation_author" content="Damien Meyer"/>
            <meta name="citation_author_institution" content="Laboratoire des Interactions Plantes-Microorganismes, Centre National de la Recherche Scientifique (CNRS)/Institut National de la Recherche Agronomique (INRA) UMR2594/441, Castanet-Tolosan, France"/>
      <meta name="citation_author" content="Alice Boulanger"/>
            <meta name="citation_author_institution" content="Laboratoire des Interactions Plantes-Microorganismes, Centre National de la Recherche Scientifique (CNRS)/Institut National de la Recherche Agronomique (INRA) UMR2594/441, Castanet-Tolosan, France"/>
      <meta name="citation_author" content="Martine Lautier"/>
            <meta name="citation_author_institution" content="Laboratoire des Interactions Plantes-Microorganismes, Centre National de la Recherche Scientifique (CNRS)/Institut National de la Recherche Agronomique (INRA) UMR2594/441, Castanet-Tolosan, France"/>
            <meta name="citation_author_institution" content="Université Paul Sabatier, Toulouse III, Toulouse, France"/>
      <meta name="citation_author" content="Catherine Guynet"/>
            <meta name="citation_author_institution" content="Laboratoire des Interactions Plantes-Microorganismes, Centre National de la Recherche Scientifique (CNRS)/Institut National de la Recherche Agronomique (INRA) UMR2594/441, Castanet-Tolosan, France"/>
      <meta name="citation_author" content="Nicolas Denancé"/>
            <meta name="citation_author_institution" content="Laboratoire des Interactions Plantes-Microorganismes, Centre National de la Recherche Scientifique (CNRS)/Institut National de la Recherche Agronomique (INRA) UMR2594/441, Castanet-Tolosan, France"/>
      <meta name="citation_author" content="Jacques Vasse"/>
            <meta name="citation_author_institution" content="Laboratoire des Interactions Plantes-Microorganismes, Centre National de la Recherche Scientifique (CNRS)/Institut National de la Recherche Agronomique (INRA) UMR2594/441, Castanet-Tolosan, France"/>
      <meta name="citation_author" content="Emmanuelle Lauber"/>
            <meta name="citation_author_institution" content="Laboratoire des Interactions Plantes-Microorganismes, Centre National de la Recherche Scientifique (CNRS)/Institut National de la Recherche Agronomique (INRA) UMR2594/441, Castanet-Tolosan, France"/>
      <meta name="citation_author" content="Matthieu Arlat"/>
            <meta name="citation_author_institution" content="Laboratoire des Interactions Plantes-Microorganismes, Centre National de la Recherche Scientifique (CNRS)/Institut National de la Recherche Agronomique (INRA) UMR2594/441, Castanet-Tolosan, France"/>
            <meta name="citation_author_institution" content="Université Paul Sabatier, Toulouse III, Toulouse, France"/>

    <meta name="citation_date" content="2007/2/21"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0000224.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e224"/>
    <meta name="citation_issue" content="2"/>
    <meta name="citation_volume" content="2"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Bacterial signal transduction network in a genomic perspective.; citation_author=MY Galperin; citation_journal_title=Environ Microbiol; citation_volume=6; citation_number=1; citation_pages=552-567; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Genetically modified organisms for the environment: stories of success and failure and what we have learned from them.; citation_author=I Cases; citation_author=V de Lorenzo; citation_journal_title=Int Microbiol; citation_volume=8; citation_number=2; citation_pages=213-222; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Proteomic identification of a two-component regulatory system in Pseudoalteromonas haloplanktis TAC125.; citation_author=R Papa; citation_author=S Glagla; citation_author=A Danchin; citation_author=T Schweder; citation_author=G Marino; citation_journal_title=Extremophiles; citation_volume=10; citation_number=3; citation_pages=483-91; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Black rot: a continuing threat to world crucifers.; citation_author=PH Williams; citation_journal_title=Plant Disease; citation_volume=64; citation_number=4; citation_pages=736-742; citation_date=1980; " />
      <meta name="citation_reference" content="citation_title=Black rot of crucifers.; citation_author=AM Alvarez; citation_number=5; citation_pages=21-52; citation_date=2000; citation_publisher=Kluwer Academic Publications; " />
      <meta name="citation_reference" content="citation_title=Nucleotide sequence of the engXCA gene encoding the major endoglucanase of Xanthomonas campestris pv. campestris.; citation_author=CL Gough; citation_author=JM Dow; citation_author=J Keen; citation_author=B Henrissat; citation_author=MJ Daniels; citation_journal_title=Gene; citation_volume=89; citation_number=6; citation_pages=53-59; citation_date=1990; " />
      <meta name="citation_reference" content="citation_title=Cloning of genes involved in negative regulation of production of extracellular enzymes and polysaccharide of Xanthomonas campestris pathovar campestris.; citation_author=JL Tang; citation_author=CL Gough; citation_author=MJ Daniels; citation_journal_title=Mol Gen Genet; citation_volume=222; citation_number=7; citation_pages=157-160; citation_date=1990; " />
      <meta name="citation_reference" content="citation_title=Extracellular proteases from Xanthomonas campestris pv. campestris, the black rot pathogen.; citation_author=JM Dow; citation_author=BR Clarke; citation_author=DE Milligan; citation_author=JL Tang; citation_author=MJ Daniels; citation_journal_title=Appl Environ Microbiol; citation_volume=56; citation_number=8; citation_pages=2994-2998; citation_date=1990; " />
      <meta name="citation_reference" content="citation_title=A two-component system involving an HD-GYP domain protein links cell-cell signalling to pathogenicity gene expression in Xanthomonas campestris.; citation_author=H Slater; citation_author=A Alvarez-Morales; citation_author=CE Barber; citation_author=MJ Daniels; citation_author=JM Dow; citation_journal_title=Mol Microbiol; citation_volume=38; citation_number=9; citation_pages=986-1003; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Cell-cell signaling in Xanthomonas campestris involves an HD-GYP domain protein that functions in cyclic di-GMP turnover.; citation_author=RP Ryan; citation_author=Y Fouhy; citation_author=JF Lucey; citation_author=LC Crossman; citation_author=S Spiro; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=103; citation_number=10; citation_pages=6712-6717; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Biofilm dispersal in Xanthomonas campestris is controlled by cell-cell signaling and is required for full virulence to plants.; citation_author=JM Dow; citation_author=L Crossman; citation_author=K Findlay; citation_author=YQ He; citation_author=JX Feng; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=100; citation_number=11; citation_pages=10995-11000; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Xanthomonas campestris contains a cluster of hrp genes related to the larger hrp cluster of Pseudomonas solanacearum.; citation_author=M Arlat; citation_author=CL Gough; citation_author=CE Barber; citation_author=C Boucher; citation_author=MJ Daniels; citation_journal_title=Mol Plant Microbe Interact; citation_volume=4; citation_number=12; citation_pages=593-601; citation_date=1991; " />
      <meta name="citation_reference" content="citation_title=PopF1 and PopF2, two proteins secreted by the type III protein secretion system of Ralstonia solanacearum, are translocators belonging to the HrpF/NopX family.; citation_author=D Meyer; citation_author=S Cunnac; citation_author=M Gueneron; citation_author=C Declercq; citation_author=F Van Gijsegem; citation_journal_title=J Bacteriol; citation_volume=188; citation_number=13; citation_pages=4903-4917; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Comparison of the genomes of two Xanthomonas pathogens with differing host specificities.; citation_author=ACR da Silva; citation_author=JA Ferro; citation_author=FC Reinach; citation_author=CS Farah; citation_author=LR Furlan; citation_journal_title=Nature; citation_volume=417; citation_number=14; citation_pages=459-463; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Comparative and functional genomic analyses of the pathogenicity of phytopathogen Xanthomonas campestris pv. campestris.; citation_author=W Qian; citation_author=Y Jia; citation_author=SX Ren; citation_author=YQ He; citation_author=JX Feng; citation_journal_title=Genome Res; citation_volume=15; citation_number=15; citation_pages=757-767; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Insights into genome plasticity and pathogenicity of the plant pathogenic bacterium Xanthomonas campestris pv. vesicatoria revealed by the complete genome sequence.; citation_author=F Thieme; citation_author=R Koebnik; citation_author=T Bekel; citation_author=C Berger; citation_author=J Boch; citation_journal_title=J Bacteriol; citation_volume=187; citation_number=16; citation_pages=7254-7266; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=The genome sequence of Xanthomonas oryzae pathovar oryzae KACC10331, the bacterial blight pathogen of rice.; citation_author=BM Lee; citation_author=YJ Park; citation_author=DS Park; citation_author=HW Kang; citation_author=JG Kim; citation_journal_title=Nucleic Acids Res; citation_volume=33; citation_number=17; citation_pages=577-586; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Genome sequence of Xanthomonas oryzae pv. oryzae suggests contribution of large numbers of effector genes and insertion sequences to its race diversity.; citation_author=H Ochiai; citation_author=Y Inoue; citation_author=M Takeya; citation_author=A Sasaki; citation_author=H Kaku; citation_journal_title=Japan Agricultural Research Quarterly; citation_volume=39; citation_number=18; citation_pages=275-287; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Touch and go: tying TonB to transport.; citation_author=K Postle; citation_author=RJ Kadner; citation_journal_title=Mol Microbiol; citation_volume=49; citation_number=19; citation_pages=869-882; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Ferric uptake regulation protein acts as a repressor, employing iron (II) as a cofactor to bind the operator of an iron transport operon in Escherichia coli.; citation_author=A Bagg; citation_author=JB Neilands; citation_journal_title=Biochemistry; citation_volume=26; citation_number=20; citation_pages=5471-5477; citation_date=1987; " />
      <meta name="citation_reference" content="citation_title=Characterization of the exbBD operon of Escherichia coli and the role of ExbB and ExbD in TonB function and stability.; citation_author=BM Ahmer; citation_author=MG Thomas; citation_author=RA Larsen; citation_author=K Postle; citation_journal_title=J Bacteriol; citation_volume=177; citation_number=21; citation_pages=4742-4747; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=ExbBD-dependent transport of maltodextrins through the novel MalA protein across the outer membrane of Caulobacter crescentus.; citation_author=H Neugebauer; citation_author=C Herrmann; citation_author=W Kammer; citation_author=G Schwarz; citation_author=A Nordheim; citation_journal_title=J Bacteriol; citation_volume=187; citation_number=22; citation_pages=8300-8311; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Substrate-induced transmembrane signaling in the cobalamin transporter BtuB.; citation_author=DP Chimento; citation_author=AK Mohanty; citation_author=RJ Kadner; citation_author=MC Wiener; citation_journal_title=Nat Struct Biol; citation_volume=10; citation_number=23; citation_pages=394-401; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Structural basis of gating by the outer membrane transporter FecA.; citation_author=AD Ferguson; citation_author=R Chakraborty; citation_author=BS Smith; citation_author=L Esser; citation_author=D van der Helm; citation_journal_title=Science; citation_volume=295; citation_number=24; citation_pages=1715-1719; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Structural evidence for iron-free citrate and ferric citrate binding to the TonB-dependent outer membrane transporter FecA.; citation_author=WW Yue; citation_author=S Grizot; citation_author=SK Buchanan; citation_journal_title=J Mol Biol; citation_volume=332; citation_number=25; citation_pages=353-368; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Structure of TonB in complex with FhuA, E. coli outer membrane receptor.; citation_author=PD Pawelek; citation_author=N Croteau; citation_author=C Ng-Thow-Hing; citation_author=CM Khursigara; citation_author=N Moiseeva; citation_journal_title=Science; citation_volume=312; citation_number=26; citation_pages=1399-1402; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Outer membrane active transport: structure of the BtuB:TonB complex.; citation_author=DD Shultis; citation_author=MD Purdy; citation_author=CN Banchs; citation_author=MC Wiener; citation_journal_title=Science; citation_volume=312; citation_number=27; citation_pages=1396-1399; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=The Pfam protein families database.; citation_author=A Bateman; citation_author=L Coin; citation_author=R Durbin; citation_author=RD Finn; citation_author=V Hollich; citation_journal_title=Nucleic Acids Res; citation_volume=32; citation_number=28; citation_pages=D138-141; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Nucleotide sequence of the gene for the ferrienterochelin receptor FepA in Escherichia coli. Homology among outer membrane receptors that interact with TonB.; citation_author=MD Lundrigan; citation_author=RJ Kadner; citation_journal_title=J Biol Chem; citation_volume=261; citation_number=29; citation_pages=10797-10801; citation_date=1986; " />
      <meta name="citation_reference" content="citation_title=Carboxy-terminal phenylalanine is essential for the correct assembly of a bacterial outer membrane protein.; citation_author=M Struyve; citation_author=M Moons; citation_author=J Tommassen; citation_journal_title=J Mol Biol; citation_volume=218; citation_number=30; citation_pages=141-148; citation_date=1991; " />
      <meta name="citation_reference" content="citation_title=Structural organization of TonB-dependent receptors.; citation_author=R Koebnik; citation_journal_title=Trends Microbiol; citation_volume=1; citation_number=31; citation_pages=201; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=Surface signaling: novel transcription initiation mechanism starting from the cell surface.; citation_author=V Braun; citation_journal_title=Arch Microbiol; citation_volume=167; citation_number=32; citation_pages=325-331; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=TonB-dependent trans-envelope signalling: the exception or the rule?; citation_author=R Koebnik; citation_journal_title=Trends Microbiol; citation_volume=13; citation_number=33; citation_pages=343-347; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Operator sequences of the aerobactin operon of plasmid ColV-K30 binding the ferric uptake regulation (fur) repressor.; citation_author=V de Lorenzo; citation_author=S Wee; citation_author=M Herrero; citation_author=JB Neilands; citation_journal_title=J Bacteriol; citation_volume=169; citation_number=34; citation_pages=2624-2630; citation_date=1987; " />
      <meta name="citation_reference" content="citation_title=Metal ion regulation of gene expression. Fur repressor-operator interaction at the promoter region of the aerobactin system of pColV-K30.; citation_author=V de Lorenzo; citation_author=F Giovannini; citation_author=M Herrero; citation_author=JB Neilands; citation_journal_title=J Mol Biol; citation_volume=203; citation_number=35; citation_pages=875-884; citation_date=1988; " />
      <meta name="citation_reference" content="citation_title=Bacterial genes induced within the nodule during the Rhizobium-legume symbiosis.; citation_author=V Oke; citation_author=SR Long; citation_journal_title=Mol Microbiol; citation_volume=32; citation_number=36; citation_pages=837-849; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Broad-host-range cre-lox system for antibiotic marker recycling in gram-negative bacteria.; citation_author=CJ Marx; citation_author=ME Lidstrom; citation_journal_title=Biotechniques; citation_volume=33; citation_number=37; citation_pages=1062-1067; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Ralstonia solanacearum requires F-box-like domain-containing type III effectors to promote disease on several host plants.; citation_author=A Angot; citation_author=N Peeters; citation_author=E Lechner; citation_author=F Vailleau; citation_author=C Baud; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=103; citation_number=38; citation_pages=14620-14625; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Growth deficiency of a Xanthomonas oryzae pv. oryzae fur mutant in rice leaves is rescued by ascorbic acid supplementation.; citation_author=S Subramoni; citation_author=RV Sonti; citation_journal_title=Mol Plant Microbe Interact; citation_volume=18; citation_number=39; citation_pages=644-651; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Characterization of a ferric uptake regulator (fur) gene from Xanthomonas campestris pv. phaseoli with unusual primary structure, genome organization, and expression patterns.; citation_author=S Loprasert; citation_author=R Sallabhan; citation_author=S Atichartpongkul; citation_author=S Mongkolsuk; citation_journal_title=Gene; citation_volume=239; citation_number=40; citation_pages=251-258; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=A higher-order background model improves the detection of promoter regulatory elements by Gibbs sampling.; citation_author=G Thijs; citation_author=M Lescot; citation_author=K Marchal; citation_author=S Rombauts; citation_author=B De Moor; citation_journal_title=Bioinformatics; citation_volume=17; citation_number=41; citation_pages=1113-1122; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=A Gibbs sampling method to detect overrepresented motifs in the upstream regions of coexpressed genes.; citation_author=G Thijs; citation_author=K Marchal; citation_author=M Lescot; citation_author=S Rombauts; citation_author=B De Moor; citation_journal_title=J Comput Biol; citation_volume=9; citation_number=42; citation_pages=447-464; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Transmembrane transcriptional control (surface signalling) of the Escherichia coli Fec type.; citation_author=V Braun; citation_author=S Mahren; citation_journal_title=FEMS Microbiol Rev; citation_volume=29; citation_number=43; citation_pages=673-684; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Binding of the fur (ferric uptake regulator) repressor of Escherichia coli to arrays of the GATAAT sequence.; citation_author=L Escolar; citation_author=J Perez-Martin; citation_author=V de Lorenzo; citation_journal_title=J Mol Biol; citation_volume=283; citation_number=44; citation_pages=537-547; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Transcriptomic and proteomic characterization of the Fur modulon in the metal-reducing bacterium Shewanella oneidensis.; citation_author=XF Wan; citation_author=NC Verberkmoes; citation_author=LA McCue; citation_author=D Stanek; citation_author=H Connelly; citation_journal_title=J Bacteriol; citation_volume=186; citation_number=45; citation_pages=8385-8400; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Recognition of DNA by Fur: a reinterpretation of the Fur box consensus sequence.; citation_author=N Baichoo; citation_author=JD Helmann; citation_journal_title=J Bacteriol; citation_volume=184; citation_number=46; citation_pages=5826-5832; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Searching for patterns in genomic data.; citation_author=M Dsouza; citation_author=N Larsen; citation_author=R Overbeek; citation_journal_title=Trends Genet; citation_volume=13; citation_number=47; citation_pages=497-498; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Feo–transport of ferrous iron into bacteria.; citation_author=ML Cartron; citation_author=S Maddocks; citation_author=P Gillingham; citation_author=CJ Craven; citation_author=SC Andrews; citation_journal_title=Biometals; citation_volume=19; citation_number=48; citation_pages=143-157; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Iron and fur regulation in Vibrio cholerae and the role of fur in virulence.; citation_author=AR Mey; citation_author=EE Wyckoff; citation_author=V Kanukurthy; citation_author=CR Fisher; citation_author=SM Payne; citation_journal_title=Infect Immun; citation_volume=73; citation_number=49; citation_pages=8167-8178; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=A Streptococcus pneumoniae pathogenicity island encoding an ABC transporter involved in iron uptake and virulence.; citation_author=JS Brown; citation_author=SM Gilliland; citation_author=DW Holden; citation_journal_title=Mol Microbiol; citation_volume=40; citation_number=50; citation_pages=572-585; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=cDNA-AFLP analysis unravels a genome-wide hrpG-regulon in the plant pathogen Xanthomonas campestris pv. vesicatoria.; citation_author=L Noel; citation_author=F Thieme; citation_author=D Nennstiel; citation_author=U Bonas; citation_journal_title=Mol Microbiol; citation_volume=41; citation_number=51; citation_pages=1271-1281; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Specific binding of the Xanthomonas campestris pv. vesicatoria AraC-type transcriptional activator HrpX to plant-inducible promoter boxes.; citation_author=R Koebnik; citation_author=A Kruger; citation_author=F Thieme; citation_author=A Urban; citation_author=U Bonas; citation_journal_title=J Bacteriol; citation_volume=188; citation_number=52; citation_pages=7652-7660; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Gene involved in transcriptional activation of the hrp regulatory gene hrpG in Xanthomonas oryzae pv. oryzae.; citation_author=S Tsuge; citation_author=T Nakayama; citation_author=S Terashima; citation_author=H Ochiai; citation_author=A Furutani; citation_journal_title=J Bacteriol; citation_volume=188; citation_number=53; citation_pages=4158-4162; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Molecular and functional characterization of a unique sucrose hydrolase from Xanthomonas axonopodis pv. glycines.; citation_author=HS Kim; citation_author=HJ Park; citation_author=S Heu; citation_author=J Jung; citation_journal_title=J Bacteriol; citation_volume=186; citation_number=54; citation_pages=411-418; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Transport of vitamin B 12 in Escherichia coli.; citation_author=PM Di Girolamo; citation_author=C Bradbeer; citation_journal_title=J Bacteriol; citation_volume=106; citation_number=55; citation_pages=745-750; citation_date=1971; " />
      <meta name="citation_reference" content="citation_title=Vitamin B12 transport in Escherichia coli: energy coupling between membranes.; citation_author=RJ Kadner; citation_journal_title=Mol Microbiol; citation_volume=4; citation_number=56; citation_pages=2027-2033; citation_date=1990; " />
      <meta name="citation_reference" content="citation_title=Transport of vitamin B12 in Escherichia coli: energy dependence.; citation_author=C Bradbeer; citation_author=ML Woodrow; citation_journal_title=J Bacteriol; citation_volume=128; citation_number=57; citation_pages=99-104; citation_date=1976; " />
      <meta name="citation_reference" content="citation_title=Characterization of HasB, a Serratia marcescens TonB-like protein specifically involved in the haemophore-dependent haem acquisition system.; citation_author=A Paquelin; citation_author=JM Ghigo; citation_author=S Bertin; citation_author=C Wandersman; citation_journal_title=Mol Microbiol; citation_volume=42; citation_number=58; citation_pages=995-1005; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=The two TonB systems of Vibrio cholerae: redundant and specific functions.; citation_author=SS Seliger; citation_author=AR Mey; citation_author=AM Valle; citation_author=SM Payne; citation_journal_title=Mol Microbiol; citation_volume=39; citation_number=59; citation_pages=801-812; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=tonB3 is required for normal twitching motility and extracellular assembly of type IV pili.; citation_author=B Huang; citation_author=K Ru; citation_author=Z Yuan; citation_author=CB Whitchurch; citation_author=JS Mattick; citation_journal_title=J Bacteriol; citation_volume=186; citation_number=60; citation_pages=4387-4389; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Mutational analysis of the TonB1 energy coupler of Pseudomonas aeruginosa.; citation_author=Q Zhao; citation_author=K Poole; citation_journal_title=J Bacteriol; citation_volume=184; citation_number=61; citation_pages=1503-1513; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=A family of bacterial regulators homologous to Gal and Lac repressors.; citation_author=MJ Weickert; citation_author=S Adhya; citation_journal_title=J Biol Chem; citation_volume=267; citation_number=62; citation_pages=15869-15874; citation_date=1992; " />
      <meta name="citation_reference" content="citation_title=The salCAB operon of Azospirillum irakense, required for growth on salicin, is repressed by SalR, a transcriptional regulator that belongs to the Lacl/GalR family.; citation_author=E Somers; citation_author=V Keijers; citation_author=D Ptacek; citation_author=M Halvorsen Ottoy; citation_author=M Srinivasan; citation_journal_title=Mol Gen Genet; citation_volume=263; citation_number=63; citation_pages=1038-1046; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=CLUSTAL W: improving the sensitivity of progressive multiple sequence alignment through sequence weighting, position-specific gap penalties and weight matrix choice.; citation_author=JD Thompson; citation_author=DG Higgins; citation_author=TJ Gibson; citation_journal_title=Nucleic Acids Res; citation_volume=22; citation_number=64; citation_pages=4673-4680; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=Reclassification of Xanthomonas.; citation_author=L Vauterin; citation_author=B Hoste; citation_author=K Kersters; citation_author=J Swings; citation_journal_title=International Journal of Systematic Bacteriology; citation_volume=45; citation_number=65; citation_pages=472-489; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Phylogenetic affiliation of the pseudomonads based on 16S rRNA sequence.; citation_author=Y Anzai; citation_author=H Kim; citation_author=JY Park; citation_author=H Wakabayashi; citation_author=H Oyaizu; citation_journal_title=Int J Syst Evol Microbiol; citation_volume=50 Pt 4; citation_number=66; citation_pages=1563-1589; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=The leucine-responsive regulatory protein (lrp) gene for characterization of the relationship among Xanthomonas species.; citation_author=J Cubero; citation_author=JH Graham; citation_journal_title=Int J Syst Evol Microbiol; citation_volume=54; citation_number=67; citation_pages=429-437; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Identification and characterization of genes required for biosynthesis and transport of the siderophore vibrioferrin in Vibrio parahaemolyticus.; citation_author=T Tanabe; citation_author=T Funahashi; citation_author=H Nakao; citation_author=S Miyoshi; citation_author=S Shinoda; citation_journal_title=J Bacteriol; citation_volume=185; citation_number=68; citation_pages=6938-6949; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Transcriptional profiling of Caulobacter crescentus during growth on complex and minimal media.; citation_author=AK Hottes; citation_author=M Meewan; citation_author=D Yang; citation_author=N Arana; citation_author=P Romero; citation_journal_title=J Bacteriol; citation_volume=186; citation_number=69; citation_pages=1448-1461; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Sucrose utilisation in bacteria: genetic organisation and regulation.; citation_author=SJ Reid; citation_author=VR Abratt; citation_journal_title=Appl Microbiol Biotechnol; citation_volume=67; citation_number=70; citation_pages=312-321; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Molecular basis of bacterial outer membrane permeability revisited.; citation_author=H Nikaido; citation_journal_title=Microbiol Mol Biol Rev; citation_volume=67; citation_number=71; citation_pages=593-656; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=The heterologous siderophores ferrioxamine B and ferrichrome activate signaling pathways in Pseudomonas aeruginosa.; citation_author=MA Llamas; citation_author=M Sparrius; citation_author=R Kloet; citation_author=CR Jimenez; citation_author=C Vandenbroucke-Grauls; citation_journal_title=J Bacteriol; citation_volume=188; citation_number=72; citation_pages=1882-1891; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Unusual structure of the tonB-exb DNA region of Xanthomonas campestris pv. campestris: tonB, exbB, and exbD1 are essential for ferric iron uptake, but exbD2 is not.; citation_author=HG Wiggerich; citation_author=B Klauke; citation_author=R Koplin; citation_author=UB Priefer; citation_author=A Puhler; citation_journal_title=J Bacteriol; citation_volume=179; citation_number=73; citation_pages=7103-7110; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=The exbD2 gene as well as the iron-uptake genes tonB, exbB and exbD1 of Xanthomonas campestris pv. campestris are essential for the induction of a hypersensitive response on pepper (Capsicum annuum).; citation_author=HG Wiggerich; citation_author=A Puhler; citation_journal_title=Microbiology 146 ( Pt; citation_volume=5); citation_number=74; citation_pages=1053-1060; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Molecular analysis of sucrose metabolism of Erwinia amylovora and influence on bacterial virulence.; citation_author=J Bogs; citation_author=K Geider; citation_journal_title=J Bacteriol; citation_volume=182; citation_number=75; citation_pages=5351-5358; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Molecular analysis of two ScrR repressors and of a ScrR-FruR hybrid repressor for sucrose and D-fructose specific regulons from enteric bacteria.; citation_author=K Jahreis; citation_author=JW Lengeler; citation_journal_title=Mol Microbiol; citation_volume=9; citation_number=76; citation_pages=195-209; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=Plasmid-mediated uptake and metabolism of sucrose by Escherichia coli K-12.; citation_author=K Schmid; citation_author=M Schupfner; citation_author=R Schmitt; citation_journal_title=J Bacteriol; citation_volume=151; citation_number=77; citation_pages=68-76; citation_date=1982; " />
      <meta name="citation_reference" content="citation_title=Analysis of sucrose catabolism in Klebsiella pneumoniae and in Scr+ derivatives of Escherichia coli K12.; citation_author=GA Sprenger; citation_author=JW Lengeler; citation_journal_title=J Gen Microbiol; citation_volume=134; citation_number=78; citation_pages=1635-1644; citation_date=1988; " />
      <meta name="citation_reference" content="citation_title=Sucrose transport through maltoporin mutants of Escherichia coli.; citation_author=P Van Gelder; citation_author=R Dutzler; citation_author=F Dumas; citation_author=R Koebnik; citation_author=T Schirmer; citation_journal_title=Protein Eng; citation_volume=14; citation_number=79; citation_pages=943-948; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=The sugar-specific outer membrane channel ScrY contains functional characteristics of general diffusion pores and substrate-specific porins.; citation_author=K Schulein; citation_author=K Schmid; citation_author=R Benzl; citation_journal_title=Mol Microbiol; citation_volume=5; citation_number=80; citation_pages=2233-2241; citation_date=1991; " />
      <meta name="citation_reference" content="citation_title=What can bacterial genome research teach us about bacteria-plant interactions?; citation_author=A Puhler; citation_author=M Arlat; citation_author=A Becker; citation_author=M Gottfert; citation_author=JP Morrissey; citation_journal_title=Curr Opin Plant Biol; citation_volume=7; citation_number=81; citation_pages=137-147; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Non-gamma-proteobacteria gene islands contribute to the Xanthomonas genome.; citation_author=WC Lima; citation_author=MA Van Sluys; citation_author=CF Menck; citation_journal_title=Omics; citation_volume=9; citation_number=82; citation_pages=160-172; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Ecochemical studies of interrelationships between epiphytic bacteria and host plants via secondary metabolites.; citation_author=Y Hashidoko; citation_journal_title=Biosci Biotechnol Biochem; citation_volume=69; citation_number=83; citation_pages=1427-1441; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Structure and function of bacterial super-biosystem responsible for import and depolymerization of macromolecules.; citation_author=W Hashimoto; citation_author=K Momma; citation_author=Y Maruyama; citation_author=M Yamasaki; citation_author=B Mikami; citation_journal_title=Biosci Biotechnol Biochem; citation_volume=69; citation_number=84; citation_pages=673-692; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Proteomics-based identification of outer-membrane proteins responsible for import of macromolecules in Sphingomonas sp. A1: alginate-binding flagellin on the cell surface.; citation_author=W Hashimoto; citation_author=J He; citation_author=Y Wada; citation_author=H Nankai; citation_author=B Mikami; citation_journal_title=Biochemistry; citation_volume=44; citation_number=85; citation_pages=13783-13794; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Engineered membrane superchannel improves bioremediation potential of dioxin-degrading bacteria.; citation_author=Y Aso; citation_author=Y Miyamoto; citation_author=KM Harada; citation_author=K Momma; citation_author=S Kawai; citation_journal_title=Nat Biotechnol; citation_volume=24; citation_number=86; citation_pages=188-189; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Complete genome sequence of Caulobacter crescentus.; citation_author=WC Nierman; citation_author=TV Feldblyum; citation_author=MT Laub; citation_author=IT Paulsen; citation_author=KE Nelson; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=98; citation_number=87; citation_pages=4136-4141; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Analysis of the outer membrane proteome of Caulobacter crescentus by two-dimensional electrophoresis and mass spectrometry.; citation_author=ND Phadke; citation_author=MP Molloy; citation_author=SA Steinhoff; citation_author=PJ Ulintz; citation_author=PC Andrews; citation_journal_title=Proteomics; citation_volume=1; citation_number=88; citation_pages=705-720; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Profiling the alkaline membrane proteome of Caulobacter crescentus with two-dimensional electrophoresis and mass spectrometry.; citation_author=MP Molloy; citation_author=ND Phadke; citation_author=H Chen; citation_author=R Tyldesley; citation_author=DE Garfin; citation_journal_title=Proteomics; citation_volume=2; citation_number=89; citation_pages=899-910; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Proteomic analysis of the Caulobacter crescentus stalk indicates competence for nutrient uptake.; citation_author=MM Ireland; citation_author=JA Karty; citation_author=EM Quardokus; citation_author=JP Reilly; citation_author=YV Brun; citation_journal_title=Mol Microbiol; citation_volume=45; citation_number=90; citation_pages=1029-1041; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Coexistence among Epiphytic Bacterial Populations Mediated through Nutritional Resource Partitioning.; citation_author=M Wilson; citation_author=SE Lindow; citation_journal_title=Appl Environ Microbiol; citation_volume=60; citation_number=91; citation_pages=4468-4477; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=Role of leaf surface sugars in colonization of plants by bacterial epiphytes.; citation_author=J Mercier; citation_author=SE Lindow; citation_journal_title=Appl Environ Microbiol; citation_volume=66; citation_number=92; citation_pages=369-374; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Appetite of an epiphyte: quantitative monitoring of bacterial sugar consumption in the phyllosphere.; citation_author=JH Leveau; citation_author=SE Lindow; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=98; citation_number=93; citation_pages=3446-3453; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=A bacterial sensor of plant cell contact controls the transcriptional induction of Ralstonia solanacearum pathogenicity genes.; citation_author=D Aldon; citation_author=B Brito; citation_author=C Boucher; citation_author=S Genin; citation_journal_title=Embo J; citation_volume=19; citation_number=94; citation_pages=2304-2314; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=PrhA controls a novel regulatory pathway required for the specific induction of Ralstonia solanacearum hrp genes in the presence of plant cells.; citation_author=M Marenda; citation_author=B Brito; citation_author=D Callard; citation_author=S Genin; citation_author=P Barberis; citation_journal_title=Mol Microbiol; citation_volume=27; citation_number=95; citation_pages=437-453; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=A signal transfer system through three compartments transduces the plant cell contact-dependent signal controlling Ralstonia solanacearum hrp genes.; citation_author=B Brito; citation_author=D Aldon; citation_author=P Barberis; citation_author=C Boucher; citation_author=S Genin; citation_journal_title=Mol Plant Microbe Interact; citation_volume=15; citation_number=96; citation_pages=109-119; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Elucidation of the hrp clusters of Xanthomonas oryzae pv. oryzicola that control the hypersensitive response in nonhost tobacco and pathogenicity in susceptible host rice.; citation_author=LF Zou; citation_author=XP Wang; citation_author=Y Xiang; citation_author=B Zhang; citation_author=YR Li; citation_journal_title=Appl Environ Microbiol; citation_volume=72; citation_number=97; citation_pages=6212-6224; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=An evolutionary alternative system for aryl beta-glucosides assimilation in bacteria.; citation_author=D Faure; citation_author=MH Saier; citation_author=J Vanderleyden; citation_journal_title=J Mol Microbiol Biotechnol; citation_volume=3; citation_number=98; citation_pages=467-470; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Whole genome analysis of the marine Bacteroidetes ‘Gramella forsetii’ reveals adaptations to degradation of polymeric organic matter.; citation_author=M Bauer; citation_author=M Kube; citation_author=H Teeling; citation_author=M Richter; citation_author=T Lombardot; citation_journal_title=Environ Microbiol; citation_volume=8; citation_number=99; citation_pages=2201-2213; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Host-bacterial mutualism in the human intestine.; citation_author=F Backhed; citation_author=RE Ley; citation_author=JL Sonnenburg; citation_author=DA Peterson; citation_author=JI Gordon; citation_journal_title=Science; citation_volume=307; citation_number=100; citation_pages=1915-1920; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Glycan foraging in vivo by an intestine-adapted bacterial symbiont.; citation_author=JL Sonnenburg; citation_author=J Xu; citation_author=DD Leip; citation_author=CH Chen; citation_author=BP Westover; citation_journal_title=Science; citation_volume=307; citation_number=101; citation_pages=1955-1959; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Inaugural Article: Honor thy symbionts.; citation_author=J Xu; citation_author=JI Gordon; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=100; citation_number=102; citation_pages=10452-10459; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Characterization of four outer membrane proteins involved in binding starch to the cell surface of Bacteroides thetaiotaomicron.; citation_author=JA Shipman; citation_author=JE Berleman; citation_author=AA Salyers; citation_journal_title=J Bacteriol; citation_volume=182; citation_number=103; citation_pages=5365-5372; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=A genomic view of the human-Bacteroides thetaiotaomicron symbiosis.; citation_author=J Xu; citation_author=MK Bjursell; citation_author=J Himrod; citation_author=S Deng; citation_author=LK Carmichael; citation_journal_title=Science; citation_volume=299; citation_number=104; citation_pages=2074-2076; citation_date=2003; " />
      <meta name="citation_reference" content="citation_author=J Miller; citation_number=105; citation_date=1972; citation_publisher=Cold Spring Harbor Laboratory Press; " />
      <meta name="citation_reference" content="citation_title=Selection procedure for deregulated iron transport mutants (fur) in Escherichia coli K 12: fur not only affects iron metabolism.; citation_author=K Hantke; citation_journal_title=Mol Gen Genet; citation_volume=210; citation_number=106; citation_pages=135-139; citation_date=1987; " />
      <meta name="citation_reference" content="citation_title=Universal chemical assay for the detection and determination of siderophores.; citation_author=B Schwyn; citation_author=JB Neilands; citation_journal_title=Anal Biochem; citation_volume=160; citation_number=107; citation_pages=47-56; citation_date=1987; " />
      <meta name="citation_reference" content="citation_title=Inventory and functional analysis of the large Hrp regulon in Ralstonia solanacearum: identification of novel effector proteins translocated to plant host cells through the type III secretion system.; citation_author=S Cunnac; citation_author=A Occhialini; citation_author=P Barberis; citation_author=C Boucher; citation_author=S Genin; citation_journal_title=Mol Microbiol; citation_volume=53; citation_number=108; citation_pages=115-128; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Bacterial blight of soybean: regulation of a pathogen gene determining host cultivar specificity.; citation_author=TV Huynh; citation_author=D Dahlbeck; citation_author=BJ Staskawicz; citation_journal_title=Science; citation_volume=245; citation_number=109; citation_pages=1374-1377; citation_date=1989; " />
      <meta name="citation_reference" content="citation_title=Stable RK2-derived cloning vectors for the analysis of gene expression and gene function in gram-negative bacteria.; citation_author=B Dombrecht; citation_author=J Vanderleyden; citation_author=J Michiels; citation_journal_title=Mol Plant Microbe Interact; citation_volume=14; citation_number=110; citation_pages=426-430; citation_date=2001; " />
      <meta name="citation_reference" content="citation_author=J Miller; citation_number=111; citation_date=1992; citation_publisher=Cold Spring Harbor Laboratory Press; " />
      <meta name="citation_reference" content="citation_title=GUS fusions: beta-glucuronidase as a sensitive and versatile gene fusion marker in higher plants.; citation_author=RA Jefferson; citation_author=TA Kavanagh; citation_author=MW Bevan; citation_journal_title=Embo J; citation_volume=6; citation_number=112; citation_pages=3901-3907; citation_date=1987; " />
      <meta name="citation_reference" content="citation_title=Functional analysis of the early chlorosis factor gene.; citation_author=CQ Morales; citation_author=J Posada; citation_author=E Macneale; citation_author=D Franklin; citation_author=I Rivas; citation_journal_title=Mol Plant Microbe Interact; citation_volume=18; citation_number=113; citation_pages=477-486; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Optimization of pathogenicity assays to study the Arabidopsis thaliana-Xanthomonas campestris pv. campestris pathosystem.; citation_author=D Meyer; citation_author=E Lauber; citation_author=D Roby; citation_author=M Arlat; citation_author=T Kroj; citation_journal_title=Molecular Plant Pathology; citation_volume=6; citation_number=114; citation_pages=327-333; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Improved prediction of signal peptides: SignalP 3.0.; citation_author=JD Bendtsen; citation_author=H Nielsen; citation_author=G von Heijne; citation_author=S Brunak; citation_journal_title=J Mol Biol; citation_volume=340; citation_number=115; citation_pages=783-795; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Multiple sequence alignment with hierarchical clustering.; citation_author=F Corpet; citation_journal_title=Nucleic Acids Res; citation_volume=16; citation_number=116; citation_pages=10881-10890; citation_date=1988; " />
      <meta name="citation_reference" content="citation_title=Protein secondary structure prediction based on position-specific scoring matrices.; citation_author=DT Jones; citation_journal_title=J Mol Biol; citation_volume=292; citation_number=117; citation_pages=195-202; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Protein structure prediction servers at University College London.; citation_author=K Bryson; citation_author=LJ McGuffin; citation_author=RL Marsden; citation_author=JJ Ward; citation_author=JS Sodhi; citation_journal_title=Nucleic Acids Res; citation_volume=33; citation_number=118; citation_pages=W36-38; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=The PSIPRED protein structure prediction server.; citation_author=LJ McGuffin; citation_author=K Bryson; citation_author=DT Jones; citation_journal_title=Bioinformatics; citation_volume=16; citation_number=119; citation_pages=404-405; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Sequence and expression analysis of the hrpB pathogenicity operon of Xanthomonas campestris pv. vesicatoria which encodes eight proteins with similarity to components of the Hrp, Ysc, Spa, and Fli secretion systems.; citation_author=S Fenselau; citation_author=U Bonas; citation_journal_title=Mol Plant Microbe Interact; citation_volume=8; citation_number=120; citation_pages=845-854; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Characterization of the cis-acting regulatory element controlling HrpB-mediated activation of the type III secretion system and effector genes in Ralstonia solanacearum.; citation_author=S Cunnac; citation_author=C Boucher; citation_author=S Genin; citation_journal_title=J Bacteriol; citation_volume=186; citation_number=121; citation_pages=2309-2318; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=The CLUSTAL_X windows interface: flexible strategies for multiple sequence alignment aided by quality analysis tools.; citation_author=JD Thompson; citation_author=TJ Gibson; citation_author=F Plewniak; citation_author=F Jeanmougin; citation_author=DG Higgins; citation_journal_title=Nucleic Acids Res; citation_volume=25; citation_number=122; citation_pages=4876-4882; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=WebLogo: a sequence logo generator.; citation_author=GE Crooks; citation_author=G Hon; citation_author=JM Chandonia; citation_author=SE Brenner; citation_journal_title=Genome Res; citation_volume=14; citation_number=123; citation_pages=1188-1190; citation_date=2004; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000224" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Plant Carbohydrate Scavenging through TonB-Dependent Receptors: A Feature Shared by Phytopathogenic and Aquatic Bacteria"/>
    <meta name="twitter:description" content="TonB-dependent receptors (TBDRs) are outer membrane proteins mainly known for the active transport of iron siderophore complexes in Gram-negative bacteria. Analysis of the genome of the phytopathogenic bacterium Xanthomonas campestris pv. campestris (Xcc), predicts 72 TBDRs. Such an overrepresentation is common in Xanthomonas species but is limited to only a small number of bacteria. Here, we show that one Xcc TBDR transports sucrose with a very high affinity, suggesting that it might be a sucrose scavenger. This TBDR acts with an inner membrane transporter, an amylosucrase and a regulator to utilize sucrose, thus defining a new type of carbohydrate utilization locus, named CUT locus, involving a TBDR for the transport of substrate across the outer membrane. This sucrose CUT locus is required for full pathogenicity on Arabidopsis, showing its importance for the adaptation to host plants. A systematic analysis of Xcc TBDR genes and a genome context survey suggested that several Xcc TBDRs belong to other CUT loci involved in the utilization of various plant carbohydrates. Interestingly, several Xcc TBDRs and CUT loci are conserved in aquatic bacteria such as Caulobacter crescentus, Colwellia psychrerythraea, Saccharophagus degradans, Shewanella spp., Sphingomonas spp. or Pseudoalteromonas spp., which share the ability to degrade a wide variety of complex carbohydrates and display TBDR overrepresentation. We therefore propose that TBDR overrepresentation and the presence of CUT loci designate the ability to scavenge carbohydrates. Thus CUT loci, which seem to participate to the adaptation of phytopathogenic bacteria to their host plants, might also play a very important role in the biogeochemical cycling of plant-derived nutrients in marine environments. Moreover, the TBDRs and CUT loci identified in this study are clearly different from those characterized in the human gut symbiont Bacteroides thetaiotaomicron, which allow glycan foraging, suggesting a convergent evolution of TBDRs in Proteobacteria and Bacteroidetes."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0000224.g012"/>

  <meta property="og:title" content="Plant Carbohydrate Scavenging through TonB-Dependent Receptors: A Feature Shared by Phytopathogenic and Aquatic Bacteria" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000224" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=6859'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=9065'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=7158&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0000224">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000224" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000224&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Plant Carbohydrate Scavenging through TonB-Dependent Receptors: A Feature Shared by Phytopathogenic and Aquatic Bacteria
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Servane Blanvillain
              <span class="equal-contrib"
                    title="These authors contributed equally to this work">equal contributor</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">

                <p><span class="equal-contrib" title="These authors contributed equally to this work">equal contributor</span>
                Contributed equally to this work with: Servane Blanvillain, Damien Meyer, Emmanuelle Lauber, Matthieu Arlat</p>

              
              

                <p>Affiliation:
                  Laboratoire des Interactions Plantes-Microorganismes, Centre National de la Recherche Scientifique (CNRS)/Institut National de la Recherche Agronomique (INRA) UMR2594/441, Castanet-Tolosan, France
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Damien Meyer
              <span class="equal-contrib"
                    title="These authors contributed equally to this work">equal contributor</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">

                <p><span class="equal-contrib" title="These authors contributed equally to this work">equal contributor</span>
                Contributed equally to this work with: Servane Blanvillain, Damien Meyer, Emmanuelle Lauber, Matthieu Arlat</p>

              
              

                <p>Affiliation:
                  Laboratoire des Interactions Plantes-Microorganismes, Centre National de la Recherche Scientifique (CNRS)/Institut National de la Recherche Agronomique (INRA) UMR2594/441, Castanet-Tolosan, France
                </p>

                <p>
                    Current address: UMR PaVe, Centre INRA, Beaucouze, France
                </p>

              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Alice Boulanger, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Laboratoire des Interactions Plantes-Microorganismes, Centre National de la Recherche Scientifique (CNRS)/Institut National de la Recherche Agronomique (INRA) UMR2594/441, Castanet-Tolosan, France
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Martine Lautier, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliations:
                  Laboratoire des Interactions Plantes-Microorganismes, Centre National de la Recherche Scientifique (CNRS)/Institut National de la Recherche Agronomique (INRA) UMR2594/441, Castanet-Tolosan, France, 
                  Université Paul Sabatier, Toulouse III, Toulouse, France
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Catherine Guynet, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Laboratoire des Interactions Plantes-Microorganismes, Centre National de la Recherche Scientifique (CNRS)/Institut National de la Recherche Agronomique (INRA) UMR2594/441, Castanet-Tolosan, France
                </p>

                <p>
                    Current address: Laboratoire de Microbiologie et Génétique Moléculaires, Centre National de la Recherche Scientifique (CNRS) UMR5100, Toulouse, France
                </p>

              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Nicolas Denancé, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Laboratoire des Interactions Plantes-Microorganismes, Centre National de la Recherche Scientifique (CNRS)/Institut National de la Recherche Agronomique (INRA) UMR2594/441, Castanet-Tolosan, France
                </p>

                <p>
                    Current address: Signalisation chez les Végétaux, Pole de Biotechnologie Végétale, Centre National de la Recherche Scientifique (CNRS) UMR5546, Castanet-Tolosan, France
                </p>

              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Jacques Vasse, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Laboratoire des Interactions Plantes-Microorganismes, Centre National de la Recherche Scientifique (CNRS)/Institut National de la Recherche Agronomique (INRA) UMR2594/441, Castanet-Tolosan, France
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Emmanuelle Lauber
              <span class="equal-contrib"
                    title="These authors contributed equally to this work">equal contributor</span>
              <span class="corresponding">mail</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">

                <p><span class="equal-contrib" title="These authors contributed equally to this work">equal contributor</span>
                Contributed equally to this work with: Servane Blanvillain, Damien Meyer, Emmanuelle Lauber, Matthieu Arlat</p>

              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:elauber@toulouse.inra.fr">elauber@toulouse.inra.fr</a> (EL); <a href="mailto:arlat@toulouse.inra.fr">arlat@toulouse.inra.fr</a> (MA)</p>

                <p>Affiliation:
                  Laboratoire des Interactions Plantes-Microorganismes, Centre National de la Recherche Scientifique (CNRS)/Institut National de la Recherche Agronomique (INRA) UMR2594/441, Castanet-Tolosan, France
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Matthieu Arlat
              <span class="equal-contrib"
                    title="These authors contributed equally to this work">equal contributor</span>
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">

                <p><span class="equal-contrib" title="These authors contributed equally to this work">equal contributor</span>
                Contributed equally to this work with: Servane Blanvillain, Damien Meyer, Emmanuelle Lauber, Matthieu Arlat</p>

              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:elauber@toulouse.inra.fr">elauber@toulouse.inra.fr</a> (EL); <a href="mailto:arlat@toulouse.inra.fr">arlat@toulouse.inra.fr</a> (MA)</p>

                <p>Affiliations:
                  Laboratoire des Interactions Plantes-Microorganismes, Centre National de la Recherche Scientifique (CNRS)/Institut National de la Recherche Agronomique (INRA) UMR2594/441, Castanet-Tolosan, France, 
                  Université Paul Sabatier, Toulouse III, Toulouse, France
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: February 21, 2007</li>
    <li>DOI: 10.1371/journal.pone.0000224</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0000224" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0000224" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000224" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0000224" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0000224-g001" data-doi="info:doi/10.1371/journal.pone.0000224" data-uri="info:doi/10.1371/journal.pone.0000224.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000224.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000224-g002" data-doi="info:doi/10.1371/journal.pone.0000224" data-uri="info:doi/10.1371/journal.pone.0000224.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000224.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000224-g003" data-doi="info:doi/10.1371/journal.pone.0000224" data-uri="info:doi/10.1371/journal.pone.0000224.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000224.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000224-t001" data-doi="info:doi/10.1371/journal.pone.0000224" data-uri="info:doi/10.1371/journal.pone.0000224.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000224.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000224-g004" data-doi="info:doi/10.1371/journal.pone.0000224" data-uri="info:doi/10.1371/journal.pone.0000224.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000224.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000224-g005" data-doi="info:doi/10.1371/journal.pone.0000224" data-uri="info:doi/10.1371/journal.pone.0000224.g005" title="Figure 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000224.g005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000224-g006" data-doi="info:doi/10.1371/journal.pone.0000224" data-uri="info:doi/10.1371/journal.pone.0000224.g006" title="Figure 6">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000224.g006&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000224-g007" data-doi="info:doi/10.1371/journal.pone.0000224" data-uri="info:doi/10.1371/journal.pone.0000224.g007" title="Figure 7">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000224.g007&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000224-g008" data-doi="info:doi/10.1371/journal.pone.0000224" data-uri="info:doi/10.1371/journal.pone.0000224.g008" title="Figure 8">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000224.g008&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000224-t002" data-doi="info:doi/10.1371/journal.pone.0000224" data-uri="info:doi/10.1371/journal.pone.0000224.t002" title="Table 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000224.t002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000224-g009" data-doi="info:doi/10.1371/journal.pone.0000224" data-uri="info:doi/10.1371/journal.pone.0000224.g009" title="Figure 9">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000224.g009&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000224-t003" data-doi="info:doi/10.1371/journal.pone.0000224" data-uri="info:doi/10.1371/journal.pone.0000224.t003" title="Table 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000224.t003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000224-t004" data-doi="info:doi/10.1371/journal.pone.0000224" data-uri="info:doi/10.1371/journal.pone.0000224.t004" title="Table 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000224.t004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000224-g010" data-doi="info:doi/10.1371/journal.pone.0000224" data-uri="info:doi/10.1371/journal.pone.0000224.g010" title="Figure 10">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000224.g010&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000224-g011" data-doi="info:doi/10.1371/journal.pone.0000224" data-uri="info:doi/10.1371/journal.pone.0000224.g011" title="Figure 11">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000224.g011&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000224-g012" data-doi="info:doi/10.1371/journal.pone.0000224" data-uri="info:doi/10.1371/journal.pone.0000224.g012" title="Figure 12">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000224.g012&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000224">Reader Comments (2)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0000224" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1"></a><p>TonB-dependent receptors (TBDRs) are outer membrane proteins mainly known for the active transport of iron siderophore complexes in Gram-negative bacteria. Analysis of the genome of the phytopathogenic bacterium <em>Xanthomonas campestris</em> pv. <em>campestris</em> (<em>Xcc</em>), predicts 72 TBDRs. Such an overrepresentation is common in <em>Xanthomonas</em> species but is limited to only a small number of bacteria. Here, we show that one <em>Xcc</em> TBDR transports sucrose with a very high affinity, suggesting that it might be a sucrose scavenger. This TBDR acts with an inner membrane transporter, an amylosucrase and a regulator to utilize sucrose, thus defining a new type of carbohydrate utilization locus, named CUT locus, involving a TBDR for the transport of substrate across the outer membrane. This sucrose CUT locus is required for full pathogenicity on <em>Arabidopsis</em>, showing its importance for the adaptation to host plants. A systematic analysis of <em>Xcc</em> TBDR genes and a genome context survey suggested that several <em>Xcc</em> TBDRs belong to other CUT loci involved in the utilization of various plant carbohydrates. Interestingly, several <em>Xcc</em> TBDRs and CUT loci are conserved in aquatic bacteria such as <em>Caulobacter crescentus, Colwellia psychrerythraea, Saccharophagus degradans, Shewanella</em> spp., <em>Sphingomonas</em> spp. or <em>Pseudoalteromonas</em> spp., which share the ability to degrade a wide variety of complex carbohydrates and display TBDR overrepresentation. We therefore propose that TBDR overrepresentation and the presence of CUT loci designate the ability to scavenge carbohydrates. Thus CUT loci, which seem to participate to the adaptation of phytopathogenic bacteria to their host plants, might also play a very important role in the biogeochemical cycling of plant-derived nutrients in marine environments. Moreover, the TBDRs and CUT loci identified in this study are clearly different from those characterized in the human gut symbiont <em>Bacteroides thetaiotaomicron</em>, which allow glycan foraging, suggesting a convergent evolution of TBDRs in Proteobacteria and Bacteroidetes.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>Blanvillain S, Meyer D, Boulanger A, Lautier M, Guynet C, et al.  (2007) Plant Carbohydrate Scavenging through TonB-Dependent Receptors: A Feature Shared by Phytopathogenic and Aquatic Bacteria. PLoS ONE 2(2):
          e224.
            doi:10.1371/journal.pone.0000224</p><p><strong>Academic Editor: </strong>Rosemary Redfield, University of British Columbia, Canada</p><p><strong>Received:</strong> December 20, 2006; <strong>Accepted:</strong> January 26, 2007; <strong>Published:</strong> February 21, 2007</p><p><strong>Copyright:</strong> © 2007 Blanvillain et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>S. Blanvillain, D. Meyer and A. Boulanger were supported by grants from the Ministère de la Recherche et de l'Enseignement Supérieur. We gratefully acknowledge financial support from the Département Santé des Plantes et Environnement de L'Institut National de la Recherche Agronomique.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>Bacteria are able to colonize a wide variety of habitats, including the most extreme environments and even living organisms. This remarkable feature likely reflects a high degree of adaptability and the presence of specific genetic programs devoted to the exploitation of nutrients present in these diverse habitats. The bacterial ability to use defined carbohydrates to support cell survival or growth implies the availability of these substrates in their habitat. Moreover, it requires the recognition of these molecules and the coordinated induction of particular uptake systems and of metabolic enzymes <a href="#pone.0000224-Galperin1">[1]</a>, <a href="#pone.0000224-Cases1">[2]</a>. Characterization of the repertoire of genes involved in signal perception or transduction and in carbohydrate utilization, in conjunction with analysis of the regulation of their expression, is likely to provide key information about the interaction and adaptation of bacteria with their environment <a href="#pone.0000224-Papa1">[3]</a>.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>The analysis and comparison of complete genomic sequences of numerous bacteria from diverse phylogenies and habitats have shown a relationship between the ecological niches that a bacterium occupies and the proportion of genes involved in signal perception and transduction. Bacteria that inhabit stable environments (extremophiles, obligate parasites or symbionts), which generally have small genomes, possess fewer sensory and regulatory genes than free-living bacteria found in complex and changing environments such as those living in soil or in association with plants <a href="#pone.0000224-Cases1">[2]</a>. Therefore, it was proposed that sensors and regulators can be used as “descriptors of bacterial lifestyle” <a href="#pone.0000224-Cases1">[2]</a>.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>With the aim to study the molecular mechanisms controlling adaptation of phytopathogenic bacteria to their host-plants, we undertook a global analysis of receptors and regulators of <em>Xanthomonas campestris</em> pv<em>. campestris</em> (<em>Xcc</em>), the causal agent of black rot of crucifers. This pathogen infects a wide range of <em>Brassicaceae</em> plants of economic interest, including cabbage, cauliflower and radish as well as the model plant <em>Arabidopsis thaliana</em>. This epiphytic bacterium naturally infects host plants <em>via</em> wounds in the leaves or hydathodes which are specialized pores on the leaf margins of higher plants that connect to the vascular system. Then bacteria multiply and progress in vascular tissues <a href="#pone.0000224-Williams1">[4]</a>, <a href="#pone.0000224-Alvarez1">[5]</a>. During the past two decades, classical molecular and genetic studies led to the characterization of several determinants controlling pathogenicity of <em>Xcc</em>, such as secretion of extracellular plant cell wall degrading enzymes <a href="#pone.0000224-Gough1">[6]</a>–<a href="#pone.0000224-Dow1">[8]</a>, cell-cell signaling <a href="#pone.0000224-Slater1">[9]</a>, <a href="#pone.0000224-Ryan1">[10]</a>, biofilm formation <a href="#pone.0000224-Dow2">[11]</a> and <em>hrp</em> genes coding for a type III secretion system <a href="#pone.0000224-Arlat1">[12]</a>, <a href="#pone.0000224-Meyer1">[13]</a>. The characterization of new virulence factors should now be greatly facilitated by the availability of complete genome sequences of two <em>Xcc</em> strains (strain ATCC33913 <a href="#pone.0000224-daSilva1">[14]</a> and strain 8004 <a href="#pone.0000224-Qian1">[15]</a>). Moreover, comparative genomics would improve the analysis of virulence and host adaptation in <em>Xanthomonas</em>, since the genomic sequences of four other <em>Xanthomonas</em> strains displaying different host specificities and representing three other species are also available, i.e. <em>Xanthomonas axonopodis</em> pv. <em>citri</em> (<em>Xac</em>), the causal agent of citrus canker <a href="#pone.0000224-daSilva1">[14]</a>, <em>Xanthomonas campestris</em> pv. <em>vesicatoria</em> (<em>Xcv</em>), the causative agent of bacterial spot disease on pepper and tomato plants <a href="#pone.0000224-Thieme1">[16]</a>, and <em>Xanthomonas oryzae</em> pv. <em>oryzae</em> (<em>Xoo</em>) (strain Kacc10331 <a href="#pone.0000224-Lee1">[17]</a> and strain MAFF311018 <a href="#pone.0000224-Ochiai1">[18]</a>), the causal agent of bacterial blight of rice. Thus, the <em>Xanthomonas</em> genus, which affects two major model plants (<em>Arabidopsis</em> and rice), constitutes a very attractive model to study plant-pathogen interactions.</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p>Our analysis of the <em>Xcc</em> (ATCC33913) genome revealed an overrepresentation of a particular family of receptors, named TonB-dependent receptors (TBDRs). These proteins are located in the outer membrane of Gram-negative bacteria and are mainly known to transport iron-siderophore complexes and vitamin B12 into the periplasm <a href="#pone.0000224-Postle1">[19]</a>. In most cases, the expression of the genes encoding these receptors is under the control of the Fur (<span class="underline">F</span>erric <span class="underline">u</span>ptake <span class="underline">r</span>egulator) repressor and activated under conditions of iron starvation <a href="#pone.0000224-Bagg1">[20]</a>. In contrast to diffusion through porins, transport <em>via</em> TBDRs requires energy which is provided by the inner membrane energy-coupling TonB-ExbB-ExbD protein complex <a href="#pone.0000224-Ahmer1">[21]</a>.</p>
<a id="article1.body1.sec1.p5" name="article1.body1.sec1.p5"></a><p>The exploration of complete genome sequences of 226 Gram negative bacteria showed that the overrepresentation of TBDRs is restricted to a small proportion of these bacteria, but is a common trait of all sequenced <em>Xanthomonas</em> species. Interestingly, most of the bacteria displaying this particularity have diverse lifestyles and belong to different taxonomical lineages, but they all share the ability to exploit complex carbohydrates. Therefore, we postulated that some <em>Xcc</em> TBDRs might be involved in the transport of plant-derived molecules. This hypothesis was recently reinforced with the characterization of a TBDR, named MalA, in the oligotrophic aquatic bacterium <em>Caulobacter crescentus</em>, which transports maltodextrins <a href="#pone.0000224-Neugebauer1">[22]</a>.</p>
<a id="article1.body1.sec1.p6" name="article1.body1.sec1.p6"></a><p>A systematic study of <em>Xcc</em> TBDRs, based on mutagenesis, expression analyses and transport assays identified one <em>Xcc</em> TBDR involved in the transport of sucrose, a major plant sugar. This TBDR gene is required for full virulence on <em>Arabidopsis</em> and is associated with genes required for sucrose metabolism, thus forming a “sucrose utilization locus”. Our study also suggests the existence of other TBDR-containing loci involved in the utilization of plant cell wall compounds which are conserved in a wide range of bacteria displaying TBDR overrepresentation.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3>
<h4>Identification of <em>Xcc</em> TBDRs</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>TBDR proteins consist of two domains, with a C-terminal membrane embedded β-barrel domain that is sealed by the N-terminal plug domain <a href="#pone.0000224-Chimento1">[23]</a>–<a href="#pone.0000224-Shultis1">[27]</a>. Hidden Markov models (HMMs), PF00593 and PF07715, corresponding to these two domains, are available in the Pfam database <a href="#pone.0000224-Bateman1">[28]</a>. Seventy-six proteins carrying one or both domains were detected in the proteome of the <em>Xcc</em> ATCC33913 strain (<a href="#pone.0000224.s003">Table S1</a>). Among these proteins, 64 possess both domains. The remaining 12 proteins, which possess only one of the two domains, can be divided into three groups: 3 proteins (<em>XCC2208, XCC4131</em> and <em>XCC4132</em>) which do not seem to have a canonical plug domain, one protein (<em>XCC2497</em>) which has no domain detected by the PF00593 HMM, and finally 8 proteins (<em>XCC0304-0305, XCC1750-1751, XCC3215-3216</em> and <em>XCC3270-3271</em>) which are truncated and correspond to 4 pseudogenes having a frame-shift in their coding region (<a href="#pone.0000224.s003">Table S1</a>). Each of these 4 pseudogenes was thereafter studied as a unique entity.</p>
<a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2"></a><p>Other conserved features of TBDRs were then used to further characterize these identified proteins. In the N-terminal part of the plug domain, TBDRs display a conserved sequence, the TonB-box, that interacts with the TonB protein <a href="#pone.0000224-Pawelek1">[26]</a>, <a href="#pone.0000224-Shultis1">[27]</a>, <a href="#pone.0000224-Lundrigan1">[29]</a>. A TonB-box, with the <em>Xcc</em> consensus sequence tLDXVXV (lower case indicates less highly conserved amino acid, X indicates any amino acid) was detected in all studied proteins (<a href="#pone.0000224.s003">Table S1</a>). The localization of TBDRs in the outer membrane implies their transport across the inner membrane and thus the presence of a signal peptide at their N-terminal end. The proteins were therefore analyzed for the presence of such a motif. TBDRs which were not predicted to possess a signal peptide in the annotation proposed by da Silva and colleagues <a href="#pone.0000224-daSilva1">[14]</a> were manually re-annotated, thus revealing putative signal peptides (<a href="#pone.0000224.s003">Table S1</a>). Out of 72 proteins, 1 TBDR (XCC2385) does not have a signal peptide. Finally, the 12 C-terminal amino acids of outer membrane proteins (OMPs) form a membrane anchoring β-sheet with the last residue being aromatic (F in the vast majority of OMPs) <a href="#pone.0000224-Struyve1">[30]</a>, <a href="#pone.0000224-Koebnik1">[31]</a>. The 12 C-terminal amino acid of 55 of the proteins studied here are predicted to form a potential β-sheet, ending with an aromatic residue (<a href="#pone.0000224.s003">Table S1</a>). Among the TBDRs which did not display a typical C-terminal sequence, 8 have a β-sheet ending with an aromatic amino acid followed by a short extension of 1 to 4 amino acids (<a href="#pone.0000224.s003">Table S1</a>).</p>
<a id="article1.body1.sec2.sec1.p3" name="article1.body1.sec2.sec1.p3"></a><p>Taking all this information together, 48 proteins possess all typical features of TBDRs in <em>Xcc</em> ATCC33913 and were thus considered as putative functional TBDRs. The other proteins lacking or having degenerated TBDR domains were also included in our study and then named Ps-TBDRs (for Pseudo-TBDRs) (<a href="#pone.0000224.s003">Table S1</a>).</p>
<a id="article1.body1.sec2.sec1.p4" name="article1.body1.sec2.sec1.p4"></a><p>Finally, some TBDRs contain a N-terminal extension, located between the signal peptide and the plug domain <a href="#pone.0000224-Braun1">[32]</a>. Two types of N-terminal extension have been identified: the transducer and the Oar-like extensions <a href="#pone.0000224-Koebnik2">[33]</a>. In <em>Xcc</em>, among the 9 TBDRs having an N-terminal extension, one is a member of the TBDR-transducer subclass, seven are in the Oar-like subclass and one displays a N-terminal extension that does not correspond to the two types already identified (<a href="#pone.0000224.s003">Table S1</a>). Among these TBDRs, only two have all canonical features of TBDRs.</p>


<h4>TBDRs are overrepresented in <em>Xanthomonas</em> spp. and in bacteria scavenging complex carbohydrates</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>A survey of TBDRs was performed in 226 eubacterial completely sequenced genomes, by identifying proteins detected by both PF07715 and PF00593 HMMs. This analysis showed that most bacteria (71%) hold less than 14 TBDRs per proteome, whereas the remaining bacteria have a very broad variation in TBDR number, ranging from 14 to 120 (<a href="#pone-0000224-g001">Figure 1</a> and <a href="#pone.0000224.s004">Table S2</a>). In fact, 27% of the bacteria studied here have no TBDR proteins and 43% possess between 1 and 13 TBDRs. Only a very small proportion of bacteria (15.5%) possesses more than 30 TBDRs, thus forming a particular class in which TBDRs seem to be over represented. It is worth noting that most bacteria in this class either belong to the α or γ-Proteobacteria classes or to the <em>Bacteroides</em> genus (<a href="#pone-0000224-g001">Figure 1</a> and <a href="#pone.0000224.s004">Table S2</a>). The ratios between the number of TBDRs found in each proteome and the genome size or the number of annotated coding sequences (CDS) displayed a very similar distribution pattern (<a href="#pone.0000224.s004">Table S2</a>), thus showing that there was no major bias due to annotation. Therefore, the TBDR/Mbp ratio was used to define several bacterial classes: bacteria displaying a ratio higher than 5 were considered as members of a class showing TBDR overrepresentation, while those having a ratio ranging from 3 to 5 belong to an intermediary class. The 4 <em>Xanthomonas</em> species whose genomes have been sequenced belong to the class displaying TBDR overrepresentation (<a href="#pone.0000224.s004">Table S2</a>). It is worth noting that phytopathogenic bacteria, such as <em>Pseudomonas syringae</em> pathovars or <em>Erwinia carotovora</em> subsp. <em>Atroseptica</em> belong to the intermediary class.</p>
<div class="figure" id="pone-0000224-g001"><div class="img"><a name="pone-0000224-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000224.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000224" data-uri="info:doi/10.1371/journal.pone.0000224.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000224.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000224.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000224.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000224.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000224.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000224.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000224.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000224.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000224.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1. </strong></p><a id="article1.body1.sec2.sec2.fig1.caption1.p1" name="article1.body1.sec2.sec2.fig1.caption1.p1"></a><p>Occurrence of TonB-dependent receptor (TBDR) genes in 226 completely sequenced Gram negative bacterial genomes, in ecological (A) or phylogenetic (B) contexts. TBDRs were detected by screening the Pfam database and the UniprotKB protein knowledge database using the two Pfam HMMs (PF07715, Plug; PF00593, TonB_dep_Rec). Only proteins displaying the two Pfam domains were considered. The habitat of bacteria (A) and their phylogeny (B) are represented by colour codes (see the legend boxes).</p>
<span>doi:10.1371/journal.pone.0000224.g001</span></div>

<h4>Seven <em>Xcc</em> TBDR and two Ps-TBDR genes are Fur-regulated</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>As TBDRs are mainly known to be involved in iron uptake, it was important to determine how many of these receptors are assigned to this function in <em>Xcc</em>. In most cases, the expression of genes encoding TBDRs involved in iron transport is regulated by the iron status in the medium. These genes are activated under iron depletion conditions and repressed under iron repletion conditions <em>via</em> the Fur repressor (<span class="underline">F</span>erric-<span class="underline">u</span>ptake <span class="underline">r</span>egulator), that binds to a specific DNA sequence element called the “Fur-box”, which is found in the target promoters of iron-regulated genes <a href="#pone.0000224-Bagg1">[20]</a>, <a href="#pone.0000224-deLorenzo1">[34]</a>, <a href="#pone.0000224-deLorenzo2">[35]</a>. These features led us to analyze the regulation by the iron status for all identified <em>Xcc</em> TBDR and Ps-TBDR genes.</p>
<a id="article1.body1.sec2.sec3.p2" name="article1.body1.sec2.sec3.p2"></a><p>The 76 <em>Xcc</em> TBDR and Ps-TBDR genes were mutated by insertion of the suicide plasmid pVO155 <a href="#pone.0000224-Oke1">[36]</a>, leading to transcriptional fusion with the promoterless <em>uidA</em> reporter gene. Two insertions (in <em>XCC1990</em> and <em>XCC3209</em>) were found unstable. Thus, we constructed deletion-mutations in these genes using the <em>cre-lox</em> system <a href="#pone.0000224-Marx1">[37]</a>, <a href="#pone.0000224-Angot1">[38]</a>. In order to analyze the expression of these 2 deleted TBDR genes, their promoter regions were cloned upstream of a promoterless <em>lacZ</em> gene in a reporter plasmid (see <a href="#s4">Materials and Methods</a>), and these constructions were introduced into the wild-type strain. Using β-glucuronidase or β-galactosidase expression assays, we monitored the expression of all <em>Xcc</em> TBDRs and Ps-TBDRs in iron-repleted and -depleted media. Seven genes (<em>XCC0158, XCC0768, XCC1391, XCC2772, XCC3050, XCC3518</em>, and <em>XCC4162</em>) and 2 Ps-TBDR genes (<em>XCC3216</em>-<em>3215 and XCC3595</em>) are induced by iron starvation, compared to iron-replete conditions (<a href="#pone-0000224-g002">Figure 2A</a>). With the exception of Ps-TBDR <em>XCC3216-3215</em>, this regulation pattern was confirmed by quantitative real-time reverse transcriptase polymerase chain reaction (qRT-PCR) in a wild-type background (data not shown). We then checked whether these genes are under the control of the Fur regulator. XCC1470, the orthologue of the Fur regulator identified in <em>Xoo</em> <a href="#pone.0000224-Subramoni1">[39]</a> and in <em>Xanthomonas campestris</em> pv. <em>phaseoli</em> (<em>Xap</em>) <a href="#pone.0000224-Loprasert1">[40]</a>, was mutated using the manganese method (see <a href="#s4">Materials and Methods</a>). A point mutant in this latter gene was obtained (<em>fur</em>1). As expected for a <em>fur</em> mutation, this mutant produced more siderophores than the wild-type strain and this production remained unaffected in response to increased iron levels (from 1 to 50 µM), as determined by using the chrome azurol S assay (data not shown). It is worth noting that the <em>fur</em> mutant is unable to induce any disease symptoms (data not shown), as already reported in <em>Xoo</em> <a href="#pone.0000224-Subramoni1">[39]</a>.</p>
<div class="figure" id="pone-0000224-g002"><div class="img"><a name="pone-0000224-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000224.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000224" data-uri="info:doi/10.1371/journal.pone.0000224.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000224.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000224.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000224.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000224.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000224.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000224.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000224.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000224.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000224.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2. </strong></p><a id="article1.body1.sec2.sec3.fig1.caption1.p1" name="article1.body1.sec2.sec3.fig1.caption1.p1"></a><p>Expression levels of 9 <em>Xanthomonas campestris</em> pv. <em>campestris</em> TonB-dependent receptor genes. (A) β-glucuronidase assays, performed in at least two independent experiments with pVO155 insertion mutant strains, cultivated in minimal medium supplemented with iron (100 µM FeSO<sub>4</sub>) or containing an iron chelator (100 µM 2,2′-dipyridyl; DPD). (B) Real-time quantitative RT-PCR (qRT-PCR), performed in at least three independent experiments, on RNA extracted from the wild-type strain, a <em>fur</em> mutant strain (<em>fur</em>1) or from a complemented strain [<em>fur</em>1 (pL-<em>fur</em>)], cultivated in minimal medium containing 100 µM FeSO<sub>4</sub>. Calculation of relative expression includes normalization against the endogenous control gene 16S RNA.</p>
<span>doi:10.1371/journal.pone.0000224.g002</span></div><a id="article1.body1.sec2.sec3.p3" name="article1.body1.sec2.sec3.p3"></a><p>We then compared by qRT-PCR analysis the relative expression of the TBDR and Ps-TBDR genes induced by iron starvation, in the wild-type strain <em>versus</em> a <em>fur</em> mutant, in an iron-containing medium. As presented in <a href="#pone-0000224-g002">Figure 2B</a>, the expression of all these genes was repressed by the Fur protein. However, the repression level of the 2 Ps-TBDR genes was low (less than twofold). We confirmed that the deregulation phenotype of the <em>fur</em> mutant was the result of mutation in the <em>fur</em> gene by complementation experiments: the repression by iron of the 7 TBDR and the 2 Ps-TBDR genes was restored by providing the wild-type Fur protein on the pL-<em>XCC1470</em> plasmid (see <a href="#s4">Materials and Methods</a>) in the <em>fur</em> mutant (<a href="#pone-0000224-g002">Figure 2B</a>).</p>


<h4>The 9 Fur-regulated TBDR/Ps-TBDR genes, but not the other <em>Xcc</em> TBDR genes, possess a putative Fur-box</h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>The DNA regions located upstream of the 9 Fur-repressed TBDR and Ps-TBDR genes were analyzed using the MotifSampler program <a href="#pone.0000224-Thijs1">[41]</a>, <a href="#pone.0000224-Thijs2">[42]</a> to identify putative Fur-boxes. For genes arranged in putative operons (<em>XCC3050</em> and <em>XCC0768</em>), the DNA region located upstream of the first gene of the putative operon was also studied. <em>XCC3595</em>, which belongs to the TBDR transducer subclass, is preceded by two genes displaying significant similarities with the <em>fecI</em> (<em>XCC3593</em>) and <em>fecR</em> (<em>XCC3594</em>) genes, coding for a sigma factor of the ECF subfamily and its anti-sigma factor, respectively. In most cases, the FecI/FecR system is associated with a TBDR of the transducer subclass and is involved in iron signaling and transport by regulating the expression of the TBDR gene and other “iron-associated” genes (for review see <a href="#pone.0000224-Koebnik2">[33]</a>, <a href="#pone.0000224-Braun2">[43]</a>). We thus also analyzed the DNA regions upstream of these two genes. This analysis allowed the identification of a single significant 19-bp palindromic motif (9-1-9 inverted repeat) in the upstream region of most genes or operons studied here (<a href="#pone-0000224-g003">Figure 3A</a>, <a href="#pone-0000224-t001">Table 1</a>). For <em>XCC3595</em>, the palindromic motif was detected in the upstream region of <em>XCC3593</em> (<em>fecI</em>). For <em>XCC0768</em>, two motifs were identified: one in the upstream region of <em>XCC0767</em> and one in the coding region of this gene (<a href="#pone-0000224-t001">Table 1</a>). This motif (AATGAgAATcATTctCATT; lower case indicates less highly conserved) only showed a weak similarity with the <em>E. coli</em> 19 bp Fur-box consensus sequence (GATAATGATAATCATTATC) <a href="#pone.0000224-Escolar1">[44]</a>: it matched this sequence in 10 bp positions out of 19 (52% identity). However, the conservation was significantly higher with Fur-box consensus sequences identified in <em>Shewanella oneidensis</em> (AATGATAATAATTATCATT; 84% identity) <a href="#pone.0000224-Wan1">[45]</a> and <em>Bacillus subtilis</em> (aaTGAtAATnATTaTCAtt; 84% identity) <a href="#pone.0000224-Baichoo1">[46]</a>. Interestingly, a multiple alignment of these 4 Fur-box consensus sequences showed that although the <em>E. coli</em> consensus sequence seems more divergent, it perfectly matches the 3 other sequences after a 3 bp shift (<a href="#pone-0000224-g003">Figure 3B and C</a>). These results suggest that the palindromic motif identified in this study might correspond to a possible <em>Xcc</em> Fur-box. We then explored DNA sequences located upstream of the other <em>Xcc</em> TBDR or Ps-TBDR genes as well as the entire <em>Xcc</em> genome sequence to detect putative Fur-boxes, using the MotifScanner program <a href="#pone.0000224-Thijs2">[42]</a> or the PatScan pattern matcher software <a href="#pone.0000224-Dsouza1">[47]</a>. No putative Fur-box was detected in the promoter region of any other TBDR gene. These analyses identified more than 20 genes having candidate Fur-boxes in the region from −300 to +20 relative to the start of translation (data not shown). Several of these genes are orthologs or homologs of genes regulated by Fur in other bacteria, such as <em>feoA</em> (<em>XCC1834</em>) <a href="#pone.0000224-Cartron1">[48]</a> and <em>bfd</em> (<em>XCC0481</em>) <a href="#pone.0000224-Mey1">[49]</a>, or involved in iron storage such as <em>piuB</em> (<em>XCC3955</em>) <a href="#pone.0000224-Brown1">[50]</a>, thus reinforcing the validity of our Fur-box consensus sequence. Altogether, this analysis suggests that only 7 TBDR and 2 Ps-TBDR are directly associated with iron uptake in <em>Xcc</em>. The role of the other TBDRs remained to be deciphered.</p>
<div class="figure" id="pone-0000224-g003"><div class="img"><a name="pone-0000224-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000224.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000224" data-uri="info:doi/10.1371/journal.pone.0000224.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000224.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000224.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000224.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000224.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000224.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000224.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000224.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000224.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000224.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3. </strong></p><a id="article1.body1.sec2.sec4.fig1.caption1.p1" name="article1.body1.sec2.sec4.fig1.caption1.p1"></a><p>Palindromic consensus Fur-box sequence identified upstream of 9 <em>Xanthomonas campestris</em> pv. <em>campestris</em> Fur regulated TonB-dependent receptor genes. (A) Sequence logos generated by WebLogo (<a href="http://weblogo.berkeley.edu/">http://weblogo.berkeley.edu/</a>, <a href="#pone.0000224-Crooks1">[123]</a>) of the <em>Xcc</em> Fur-box motif as predicted by the MotifSampler program <a href="#pone.0000224-Thijs1">[41]</a>, <a href="#pone.0000224-Thijs2">[42]</a>. (B and C) Multiple alignement generated with Multalin program <a href="#pone.0000224-Corpet1">[116]</a> with previously proposed Fur-box consensus sequences of <em>Bacillus subtilis</em> <a href="#pone.0000224-Baichoo1">[46]</a>, <em>Shewanella oneidensis</em> <a href="#pone.0000224-Wan1">[45]</a>, <em>X. campestris</em> pv. <em>campestris</em> and <em>Escherichia coli</em> <a href="#pone.0000224-Escolar1">[44]</a> (B) and after a 3 bp shift of the <em>E. coli</em> sequence (C).</p>
<span>doi:10.1371/journal.pone.0000224.g003</span></div><div class="figure" id="pone-0000224-t001"><div class="img"><a name="pone-0000224-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000224.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000224" data-uri="info:doi/10.1371/journal.pone.0000224.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000224.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000224.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000224.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000224.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000224.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000224.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000224.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000224.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000224.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>Candidate Fur-boxes identified in the promoter regions of <em>Xanthomonas campestris</em> pv. <em>campestris</em> Fur regulated TonB-dependent receptor (TBDR) genes, TBDR containing operons and the XCC3593 FecI sigma factor gene.</span></strong></p><span>doi:10.1371/journal.pone.0000224.t001</span></div>

<h4>Two TBDR genes belong to the <em>hrp</em> regulon of <em>Xcc</em></h4>
<a id="article1.body1.sec2.sec5.p1" name="article1.body1.sec2.sec5.p1"></a><p>During the study of TBDR gene promoters, we identified a pip/hrp<sub>II</sub> box in the promoter of 2 TBDR genes: <em>XCC1041</em> and <em>XCC1719</em>. In <em>Xanthomonas, hrp</em> genes coding for a TTSS, as well as other functions related to pathogenicity are positively regulated by <em>hrpG/hrpX</em> regulatory genes <a href="#pone.0000224-Noel1">[51]</a>. HrpG regulates the expression of <em>hrpX</em>, which controls the expression of genes possessing a pip-box (TTCGCN<sub>15</sub>TTCGC) or a hrp<sub>II</sub>-box (TTCGN<sub>16</sub>TTCG) <a href="#pone.0000224-Koebnik3">[52]</a>, <a href="#pone.0000224-Tsuge1">[53]</a>. By qRT-PCR experiments, we showed that <em>XCC1041</em> and <em>XCC1719</em> TBDR genes are activated by HrpG and HrpX (<a href="#pone-0000224-g004">Figure 4</a>). Interestingly, a mutant in the <em>XCC1719</em> TBDR gene is weakly affected in pathogenicity (data not shown, see below). This gene seems to be specific to <em>Xcc</em> as it is absent in the genomes of <em>Xcv, Xac, Xoo</em> and <em>Xf</em>, although its surrounding genes are conserved in syntenic regions in all these strains (<a href="#pone.0000224.s005">Table S3</a>). On the other hand, <em>XCC1041</em> is present in all <em>Xanthomonas</em> species (but absent in <em>Xf</em>) and encodes a TBDR with an atypical N-terminal extension.</p>
<div class="figure" id="pone-0000224-g004"><div class="img"><a name="pone-0000224-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000224.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000224" data-uri="info:doi/10.1371/journal.pone.0000224.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000224.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000224.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000224.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000224.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000224.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000224.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000224.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000224.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000224.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4. </strong></p><a id="article1.body1.sec2.sec5.fig1.caption1.p1" name="article1.body1.sec2.sec5.fig1.caption1.p1"></a><p>Regulation of <em>Xanthomonas campestris</em> pv. <em>campestris</em> TonB-dependent receptor genes <em>XCC1041</em> and <em>XCC1719</em> by <em>hrp</em> regulators. The relative expression was analyzed by real-time quantitaive RT-PCR (qRT-PCR), perfomed on RNA extracted from the wild-type strain and <em>hrpG</em> or <em>hrpX</em> insertion mutants (strains XP082 and XP083, respectively). Experiments were repeated at least three times. Calculation of relative expression includes normalization against the endogenous control gene 16S RNA.</p>
<span>doi:10.1371/journal.pone.0000224.g004</span></div>

<h4><em>Xcc</em> TBDRs and plant interactions: <em>XCC3358</em> plays a major role in pathogenicity</h4>
<a id="article1.body1.sec2.sec6.p1" name="article1.body1.sec2.sec6.p1"></a><p>In order to assess whether <em>Xcc</em> TBDRs control the interaction with plants, we tested the 76 TBDR mutants constructed in this study for pathogenicity on the <em>Arabidopsis thaliana Sf-2</em> ecotype. Among these mutants, 17 were altered in symptom development (data not shown). However, the alteration was weak and was not reproducible in all the tests we performed, suggesting either that their involvement is not crucial or that functional redundancy mask their individual contribution. However, the <em>XCC3358</em> insertion mutant was clearly and reproducibly altered in pathogenicity, showing a clear delay in symptom development in comparison to the wild-type strain (<a href="#pone-0000224-g005">Figure 5A</a>). The <em>XCC3358</em> gene is likely to form an operon with the downstream gene, <em>XCC3359</em> (<a href="#pone-0000224-g006">Figure 6</a>). The ortholog of the <em>XCC3359</em> gene, named <em>suh</em>, has been characterized in <em>Xanthomonas axonopodis</em> pv. <em>glycines</em> (<em>Xag</em>), the causal agent of bacterial pustule disease on soybean. This gene codes for a sucrose hydrolase, SUH, which plays a major role in sucrose metabolism in <em>Xag</em>. A mutant in this gene is moderately affected in pathogenicity on soybean <a href="#pone.0000224-Kim1">[54]</a>. This prompted us to construct a non polar deletion-mutation in <em>XCC3358</em> (<em>XCC3358</em>Δ1, see <a href="#s4">Materials and Methods</a> and <a href="#pone-0000224-g006">Figure 6</a>). A deletion-mutant was also generated into <em>XCC3359</em> (<em>XCC3359</em>Δ1, see <a href="#s4">Materials and Methods</a> and <a href="#pone-0000224-g006">Figure 6</a>). These two mutants showed an altered phenotype, i.e. delayed symptom development, similar to that observed with the pVO155 insertion mutant in <em>XCC3358</em> (<a href="#pone-0000224-g005">Figure 5B</a>). The non polar mutation in <em>XCC3358</em> was complemented by introducing the plasmid pL-<em>XCC3358</em>, which carries a functional <em>XCC3358</em> gene, into <em>XCC3358</em>Δ1 mutant (<a href="#pone-0000224-g005">Figure 5B</a>). This result confirms that the <em>XCC3358</em> TBDR plays a role in virulence. Similarly, the deleted mutant in <em>XCC3359</em> was complemented by a plasmid carrying this gene (<a href="#pone-0000224-g005">Figure 5B</a>).</p>
<div class="figure" id="pone-0000224-g005"><div class="img"><a name="pone-0000224-g005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000224.g005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000224" data-uri="info:doi/10.1371/journal.pone.0000224.g005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000224.g005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000224.g005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000224.g005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000224.g005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000224.g005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000224.g005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000224.g005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000224.g005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000224.g005.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 5. </strong></p><a id="article1.body1.sec2.sec6.fig1.caption1.p1" name="article1.body1.sec2.sec6.fig1.caption1.p1"></a><p>Quantitative analysis of the interaction between <em>Xanthomonas campestris</em> pv. <em>campestris</em> (<em>Xcc</em>) and <em>Arabidopsis thaliana Sf</em>-2 plants. (A) Pathogenicity tests with the <em>Xcc</em> wild-type strain and the <em>XCC3358</em> insertion mutant (<em>XCC3358</em>::pVO). (B) Pathogenicity and complementation tests with the <em>Xcc</em> wild-type strain, the <em>XCC3356</em> and <em>XCC3357</em> insertion mutants <em>(XCC3356</em>::pVO and <em>XCC3357</em>::pVO, respectively), the <em>XCC3358</em> and <em>XCC3359</em> deleted mutants (<em>XCC3358</em>D1 and <em>XCC3359</em>D1, respectively) and their corresponding complemented strains [<em>XCC3358</em>D1 (pL-<em>XCC3358</em>) and <em>XCC3359</em>D1 (pL-<em>XCC3359</em>), respectively]. Disease symptoms were scored 5 to 8 days after inoculation. Each inoculated leaf was individually scored as: no symptom = 0; weak chlorosis surrounding the wound sites = 1; strong V-shaped chlorosis = 2; developing necrosis = 3; leaf death = 4. The represented average disease scores and the standard deviations were calculated from the values of four plants with four inoculated leaves per plant.</p>
<span>doi:10.1371/journal.pone.0000224.g005</span></div><div class="figure" id="pone-0000224-g006"><div class="img"><a name="pone-0000224-g006" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000224.g006&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000224" data-uri="info:doi/10.1371/journal.pone.0000224.g006"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000224.g006&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000224.g006/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000224.g006/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000224.g006/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000224.g006/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000224.g006.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000224.g006/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000224.g006/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000224.g006.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 6. </strong></p><a id="article1.body1.sec2.sec6.fig2.caption1.p1" name="article1.body1.sec2.sec6.fig2.caption1.p1"></a><p>Genetic organization of the <em>Xanthomonas campestris</em> pv. <em>campestris sux</em> locus. Location of mutations are indicated above the map: arrowheads indicate pVO155 insertions and deleted sequences are represented by horizontal dotted bars. Regions cloned in plasmids are indicated below the map: horizontal thick black bars indicate sequences used for plasmid constructions.</p>
<span>doi:10.1371/journal.pone.0000224.g006</span></div>

<h4><em>XCC3358</em> TBDR belongs to a locus required for full pathogenicity</h4>
<a id="article1.body1.sec2.sec7.p1" name="article1.body1.sec2.sec7.p1"></a><p>In <em>Xcc</em> genome, <em>XCC3358</em> and <em>XCC3359</em> are preceded by two other genes that might be related to sugar metabolism: <em>XCC3356</em> codes for a putative transcriptional repressor of the LacI/GalR-family and <em>XCC3357</em> encodes a putative sugar transporter of the major facilitator family, allowing the transport of substrate molecules through the inner membrane. We constructed insertion mutants in these two genes, presuming that they are monocistronic (<a href="#pone-0000224-g006">Figure 6</a>). These insertions lead to transcriptional fusions with the promoterless <em>uidA</em> reporter gene. The insertion mutant in <em>XCC3357</em> showed an altered phenotype on the <em>A. thaliana</em> Sf-2 ecotype, resembling that using <em>XCC3358</em> or <em>XCC3359</em> mutants, whereas the insertion mutant in <em>XCC3356</em> was not affected in pathogenicity (<a href="#pone-0000224-g005">Figure 5B</a>). These results suggest that <em>XCC3357</em>, <em>XCC3358</em> and <em>XCC3359</em> are required in the same process related to sucrose metabolism and/or transport. The phenotype of a mutant in <em>XCC3356</em> was not surprising since this gene encodes a putative repressor, which might negatively control the expression of the other genes of the locus. A mutation in this gene should induce the constitutive expression of the other genes without affecting their function (see below).</p>


<h4>The <em>XCC3356</em>-<em>3359</em> locus is required for sucrose utilization</h4>
<a id="article1.body1.sec2.sec8.p1" name="article1.body1.sec2.sec8.p1"></a><p>Insertion mutants in <em>XCC3357, XCC3358</em> and <em>XCC3359</em> and the deletion mutant <em>XCC3359</em>Δ1 grew like the wild-type strain on minimal medium containing glucose or fructose (data not shown), but were all affected in growth on sucrose (20 mM) (<a href="#pone-0000224-g007">Figure 7A</a>). The growth of the insertion mutant in <em>XCC3356</em> and the <em>XCC3358</em>Δ1 mutant was not impaired in minimal medium containing sucrose. The fact that the insertion mutant in <em>XCC3358</em> was impaired in growth on sucrose whereas the deletion mutant in the same gene was not, suggested that the phenotype of the insertion mutant was due to a polar effect of the insertion on <em>XCC3359</em> gene expression. This hypothesis was confirmed by complementation experiments. The growth of insertion mutants in <em>XCC3358</em> and <em>XCC3359</em> on sucrose media was restored when XCC3359 was supplied <em>in trans</em> on the expression plasmid pL-<em>XCC3359</em>, which allows constitutive expression of the <em>XCC3359</em> gene. No complementation was observed when the <em>XCC3358</em> gene was supplied <em>in trans</em> in the <em>XCC3358</em> insertion mutant (<a href="#pone-0000224-g007">Figure 7B</a>). These results confirmed that <em>XCC3358</em> and <em>XCC3359</em> form an operon. They also pointed out that this putative operon and the entire locus is important for sucrose utilization in <em>Xcc</em>. Thus, we renamed this locus <em>sux</em>, for <span class="underline">s</span>ucrose <span class="underline">u</span>tilization in <em><span class="underline">X</span>anthomonas</em>. For the purpose of this study, the TBDR-amylosucrase operon was renamed <em>suxAB</em>, the putative sugar transporter gene, <em>suxC</em>, and the putative regulatory gene, <em>suxR</em> (see <a href="#pone-0000224-g006">Figure 6</a>). However, the role of the TBDR SuxA in sucrose utilization remained elusive: this transporter gene was not required for growth on sucrose, but a non polar mutant in this gene was altered in pathogenicity, like <em>suxB</em> and <em>suxC</em> insertion mutants. This observation suggested that this <em>suxA</em> gene might have a particular function.</p>
<div class="figure" id="pone-0000224-g007"><div class="img"><a name="pone-0000224-g007" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000224.g007&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000224" data-uri="info:doi/10.1371/journal.pone.0000224.g007"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000224.g007&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000224.g007/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000224.g007/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000224.g007/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000224.g007/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000224.g007.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000224.g007/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000224.g007/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000224.g007.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 7. </strong></p><a id="article1.body1.sec2.sec8.fig1.caption1.p1" name="article1.body1.sec2.sec8.fig1.caption1.p1"></a><p>Growth of <em>Xanthomonas campestris</em> pv. <em>campestris</em> wild-type strain and mutants in minimal medium containing 20 mM sucrose. (A) Bacterial growth of insertion mutants <em>XCC3356</em>::pVO to <em>XCC3359</em>::pVO or deletion mutants <em>XCC3358</em>Δ1 and <em>XCC3359</em>Δ1, compared to the wild-type strain. (B) Growth of strains carrying plamids pL-<em>XCC3358</em> or pL-<em>XCC3359</em> allowing the constitutive expression of <em>XCC3358</em> and <em>XCC3359</em> genes, respectively. Bars represent standard deviations from 3 independent experiments.</p>
<span>doi:10.1371/journal.pone.0000224.g007</span></div>

<h4>SuxA and SuxC allow sucrose uptake</h4>
<a id="article1.body1.sec2.sec9.p1" name="article1.body1.sec2.sec9.p1"></a><p>To clarify the role of the SuxA TBDR, we investigated its involvement in sucrose transport, by performing sucrose uptake experiments using [<sup>14</sup>C]sucrose.</p>
<a id="article1.body1.sec2.sec9.p2" name="article1.body1.sec2.sec9.p2"></a><p>First, the uptake rates into the <em>Xcc</em> wild-type strain were compared after an overnight preculture in the presence (induced) or absence (uninduced) of sucrose. As shown in <a href="#pone-0000224-g008">figure 8A</a>, induced cells took-up [<sup>14</sup>C]sucrose quicker than uninduced cells, but the values obtained after 2 hours of incubation were lower for induced cells. For further experiments, we worked with uninduced cells, in order to study the effect of different mutations on the transport induction.</p>
<div class="figure" id="pone-0000224-g008"><div class="img"><a name="pone-0000224-g008" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000224.g008&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000224" data-uri="info:doi/10.1371/journal.pone.0000224.g008"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000224.g008&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000224.g008/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000224.g008/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000224.g008/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000224.g008/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000224.g008.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000224.g008/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000224.g008/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000224.g008.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 8. </strong></p><a id="article1.body1.sec2.sec9.fig1.caption1.p1" name="article1.body1.sec2.sec9.fig1.caption1.p1"></a><p>[<sup>14</sup>C]sucrose transport into <em>Xanthomonas campestris</em> pv. <em>campestris</em> (<em>Xcc</em>). (A) Transport of [<sup>14</sup>C]sucrose over 120 minutes into the <em>Xcc</em> wild-type strain precultured in minimal medium without sugar (uninduced) or supplemented with 20 mM sucrose (induced). (B and C) Transport of [<sup>14</sup>C]sucrose over 8 minutes into the <em>Xcc</em> wild-type strain, the insertion mutants in <em>suxR</em> and in <em>suxC</em> (<em>suxR</em>::pVO and <em>suxC</em>::pVO, respectively), the deleted mutants in <em>suxA</em> and <em>suxB</em> (Δ<em>suxA</em> and Δ<em>suxB</em>, respectively) and their corresponding complemented strains [Δ<em>suxA</em> (pL-<em>suxA</em>) and Δ<em>suxB</em> (pL-<em>suxB</em>), respectively]. Cells were grown in minimal medium without sucrose. Transport was measured using 0.5 µM [<sup>14</sup>C]sucrose.</p>
<span>doi:10.1371/journal.pone.0000224.g008</span></div><a id="article1.body1.sec2.sec9.p3" name="article1.body1.sec2.sec9.p3"></a><p>Competition experiments were performed to test transport specificity. A 10-fold excess of unlabelled sucrose reduced the transport rate to 52% of the level obtained without unlabelled sucrose, and with a 100-fold or 200-fold excess, the transport rates were reduced to 10.5 and 7% respectively (<a href="#pone-0000224-t002">Table 2</a>). Moreover, the addition of unlabelled fructose or glucose (sucrose degradation sub-products) slightly affected sucrose transport; a 200-fold excess of these carbon sources reduced the sucrose transport to about 80% of the control level (<a href="#pone-0000224-t002">Table 2</a>).</p>
<div class="figure" id="pone-0000224-t002"><div class="img"><a name="pone-0000224-t002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000224.t002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000224" data-uri="info:doi/10.1371/journal.pone.0000224.t002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000224.t002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000224.t002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000224.t002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000224.t002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000224.t002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000224.t002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000224.t002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000224.t002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000224.t002.TIF"></span>)
                </a></li></ul></div><p><strong>Table 2.  <span>[<sup>14</sup>C]sucrose transport rates related to uninduced <em>Xanthomonas campestris</em> pv. <em>campestris</em> wild-type strain (<em>Xcc</em>568).</span></strong></p><span>doi:10.1371/journal.pone.0000224.t002</span></div><a id="article1.body1.sec2.sec9.p4" name="article1.body1.sec2.sec9.p4"></a><p>The involvement of the SuxA TBDR in sucrose transport was then checked by comparing the uptake rate of [<sup>14</sup>C]sucrose into the wild-type strain and into a <em>suxA</em> non polar mutant (Δ<em>suxA</em>). Sucrose transport was much lower in the Δ<em>suxA</em> strain, compared to the wild-type strain (<a href="#pone-0000224-g008">Figure 8B</a> and <a href="#pone-0000224-t002">Table 2</a>): after 8 minutes, it reached only 3.5% of the transport rate into the wild-type strain. When <em>suxA</em> was supplied <em>in trans</em> on a constitutive expression plasmid, sucrose transport was more efficient and rapid, with a transport rate value at 8 minutes corresponding to 484% of that of the wild-type strain. This result confirms that SuxA is required for sucrose entry into <em>Xcc</em>.</p>
<a id="article1.body1.sec2.sec9.p5" name="article1.body1.sec2.sec9.p5"></a><p>We then studied sucrose transport in <em>suxR</em> and <em>suxC</em> insertion mutants (<em>suxR</em>::pVO and <em>suxC</em>::pVO, respectively), and in a <em>suxB</em> deletion mutant (Δ<em>suxB</em>). When the inner membrane transporter encoded by <em>suxC</em> was absent, the sucrose transport rate reached only 2.1% of the value obtained for the wild-type strain. Thus, this protein is necessary for sucrose entry into <em>Xcc</em>. On the contrary, sucrose transport was greatly enhanced in the <em>suxR</em> repressor mutant and in the Δ<em>suxB</em> amylosucrase mutant (488% and 241% respectively of the value obtained for the wild-type strain after 8 minutes) (<a href="#pone-0000224-g008">Figure 8C</a> and <a href="#pone-0000224-t002">Table 2</a>). These results confirm the repressor function of SuxR (see below) and suggest that SuxB also has a repressor activity on <em>suxA</em> expression. When <em>suxB</em> was supplied <em>in trans</em> in the Δ<em>suxB</em> mutant, the sucrose transport was reduced to about the same level as the wild-type strain (76.4% of the transport rate level of the wild-type strain).</p>
<a id="article1.body1.sec2.sec9.p6" name="article1.body1.sec2.sec9.p6"></a><p>Concentration-dependent sucrose transport experiments showed a biphasic kinetics (<a href="#pone-0000224-g009">Figure 9</a>), with a fast rate between 0.01 and 0.1 µM sucrose (<em>K<sub>d</sub></em> = 0.033 µM), and a slow rate between 0.25 and 5 µM sucrose (<em>K<sub>d</sub></em> = 0.59 µM). This biphasic pattern is very similar to those observed in energy coupled transports of vitamin B12 into <em>E. coli</em> through the BtuB TBDR <a href="#pone.0000224-DiGirolamo1">[55]</a>, <a href="#pone.0000224-Kadner1">[56]</a> and more recently of maltose transport into <em>C. crescentus</em> through the MalA TBDR <a href="#pone.0000224-Neugebauer1">[22]</a>. In both systems, it was concluded that the first phase reflects binding of the transported molecule to the OM TBDR and that the second slower phase reflects binding to a cytoplasmic membrane transporter. Thus, we presume that the low <em>K<sub>d</sub></em> value (0.033 µM) mainly reflects binding to SuxA and that the higher <em>K<sub>d</sub></em> value (0.59 µM) binding to SuxC.</p>
<div class="figure" id="pone-0000224-g009"><div class="img"><a name="pone-0000224-g009" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000224.g009&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000224" data-uri="info:doi/10.1371/journal.pone.0000224.g009"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000224.g009&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000224.g009/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000224.g009/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000224.g009/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000224.g009/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000224.g009.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000224.g009/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000224.g009/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000224.g009.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 9. </strong></p><a id="article1.body1.sec2.sec9.fig2.caption1.p1" name="article1.body1.sec2.sec9.fig2.caption1.p1"></a><p>Concentration-dependent [<sup>14</sup>C]sucrose transport into wild-type <em>Xanthomonas campestris</em> pv. <em>campestris</em>. Cells were grown in minimal medium without sucrose and transport was measured for 15 sec at the indicated [<sup>14</sup>C]sucrose concentrations. The insert shows the Scatchard transformation of the binding data.</p>
<span>doi:10.1371/journal.pone.0000224.g009</span></div><a id="article1.body1.sec2.sec9.p7" name="article1.body1.sec2.sec9.p7"></a><p>Altogether, these data indicate that both SuxA and SuxC are required for sucrose transport. It is worth noting that if the Δ<em>suxA</em> mutant and the <em>suxC</em> insertion mutant have a different phenotype regarding their growth ability in presence of sucrose (see <a href="#pone-0000224-g007">figure 7A</a>), both mutants show a very similar alteration in sucrose transport rate after 8 minutes. However, after 240 minutes, sucrose transport rate in the <em>suxA</em>-deleted mutant reached almost 23% of the value obtained for the wild-type strain, whereas in the <em>suxC</em> insertion mutant, this rate was less than 7% of that of the wild-type strain. This result suggested the existence of two sucrose uptake pathways, an active and a passive pathway, both requiring transport through SuxC. The active transport pathway depends on SuxA for translocation through the outer membrane, whereas the passive pathway does not. It is assumed that the sucrose uptake rate observed into the <em>suxA</em>-deleted mutant is determined by slow and passive diffusion of sucrose across the outer membrane, which is sufficient to support growth in the presence of 20 mM sucrose in the medium.</p>


<h4>Sucrose uptake is energy-dependent</h4>
<a id="article1.body1.sec2.sec10.p1" name="article1.body1.sec2.sec10.p1"></a><p>Active iron transport through TBDRs depends on the proton motive force (PMF) <a href="#pone.0000224-Kadner1">[56]</a>, <a href="#pone.0000224-Bradbeer1">[57]</a>. Thus, we performed sucrose transport experiments in the presence of a PMF inhibitor, carbonyl cyanide 3-chlorophenyl-hydrazone (CCCP). Addition of 20 µM CCCP inhibited [<sup>14</sup>C]sucrose transport rate to 1.13% of the transport rate in the absence of CCCP (<a href="#pone-0000224-t002">Table 2</a>), demonstrating that sucrose transport through SuxA and/or SuxC is dependent on the proton motive force and does not occur by facilitated diffusion. These experiments did not permit us to conclude whether SuxA-mediated transport depended on the PMF, since SuxC belongs to the Na<sup>+</sup>-melibiose cotransporter and H<sup>+</sup> cotransporters family, which require the transmembrane potential for transport across the cytoplasmic membrane. We then tested whether TonB was required for sucrose uptake. Eight genes coding for proteins displaying significant similarities to TonB are predicted in the genome of <em>Xcc</em> (<a href="#pone-0000224-t002">Table 2</a>). These 8 genes were mutated by insertion of the pVO155 plasmid. As the insertion in <em>XCC2927</em> was found to be unstable, we constructed a deletion mutant in this gene (<a href="#s4">Materials and Methods</a>). Sucrose transport was not significantly impaired in the 8 putative <em>tonB</em> mutants (<a href="#pone-0000224-t002">Table 2</a>). This result suggests a complete or partial functional redundancy between at least two <em>Xcc</em> TonB proteins. Such a redundancy has already been observed in various other bacteria, e.g. <em>Serratia marcescens</em> <a href="#pone.0000224-Paquelin1">[58]</a>, <em>Vibrio cholerae</em> <a href="#pone.0000224-Seliger1">[59]</a> and <em>Pseudomonas aeruginosa</em> <a href="#pone.0000224-Huang1">[60]</a>, <a href="#pone.0000224-Zhao1">[61]</a>.</p>


<h4>The expression of <em>sux</em> genes is induced by sucrose and repressed by SuxR</h4>
<a id="article1.body1.sec2.sec11.p1" name="article1.body1.sec2.sec11.p1"></a><p>Using qRT-PCR analyses, we observed that the expression of <em>suxR</em>, <em>suxC</em> and <em>suxAB</em> is specifically induced by the presence of sucrose in the medium (<a href="#pone-0000224-t003">Table 3</a>).</p>
<div class="figure" id="pone-0000224-t003"><div class="img"><a name="pone-0000224-t003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000224.t003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000224" data-uri="info:doi/10.1371/journal.pone.0000224.t003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000224.t003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000224.t003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000224.t003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000224.t003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000224.t003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000224.t003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000224.t003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000224.t003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000224.t003.TIF"></span>)
                </a></li></ul></div><p><strong>Table 3.  <span>Relative expression ratios of <em>suxR, suxC, suxA</em> and <em>suxB</em> in the wild-type strain, in minimal medium containing 20 mM sucrose <em>vs</em> minimal medium without sucrose.</span></strong></p><span>doi:10.1371/journal.pone.0000224.t003</span></div><a id="article1.body1.sec2.sec11.p2" name="article1.body1.sec2.sec11.p2"></a><p>As <em>SuxR</em> has a high degree of similarity with members of the LacI/GalR family of transcriptional repressors <a href="#pone.0000224-Weickert1">[62]</a>, <a href="#pone.0000224-Somers1">[63]</a>, we tested whether this protein regulates the expression of <em>sux</em> genes by comparing their expression by qRT-PCR in the <em>suxR</em> insertion mutant and in the wild-type strain, cultivated in minimal medium with or without sucrose. <em>suxAB, suxC</em> and <em>suxR</em> were over-expressed in a <em>suxR</em> mutant as compared to the wild-type strain in the absence of sucrose (<a href="#pone-0000224-t004">Table 4</a>), suggesting a negative and effector-dependent control of <em>sux</em> genes by SuxR.</p>
<div class="figure" id="pone-0000224-t004"><div class="img"><a name="pone-0000224-t004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000224.t004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000224" data-uri="info:doi/10.1371/journal.pone.0000224.t004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000224.t004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000224.t004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000224.t004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000224.t004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000224.t004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000224.t004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000224.t004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000224.t004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000224.t004.TIF"></span>)
                </a></li></ul></div><p><strong>Table 4.  <span>Relative expression ratios of <em>suxR, suxC, suxA</em> and <em>suxB</em> in the <em>suxR</em> insertion mutant (<em>suxR</em>::pVO) <em>vs</em> wild-type strain.</span></strong></p><span>doi:10.1371/journal.pone.0000224.t004</span></div><a id="article1.body1.sec2.sec11.p3" name="article1.body1.sec2.sec11.p3"></a><p>We also studied the expression of the <em>suxAB</em> operon in the presence of different sucrose concentrations. For this purpose, the promoter region of this operon was cloned upstream of the promoterless <em>LacZ</em> gene in a reporter plasmid (see <a href="#s4">Materials and Methods</a>). This plasmid, named pPr-<em>suxA</em>, was introduced into the <em>Xcc</em> wild-type strain, the <em>suxR</em> and <em>suxC</em> insertion mutants (<em>suxR</em>::pVO and <em>suxC</em>::pVO, respectively), and the <em>suxA</em> and <em>suxB</em> deletion mutants (Δ<em>suxA</em> and Δ<em>suxB</em>, respectively).</p>
<a id="article1.body1.sec2.sec11.p4" name="article1.body1.sec2.sec11.p4"></a><p>These strains were used to perform β-galactosidase assays after 6 hours growth in minimal medium containing a range of sucrose concentrations (<a href="#pone-0000224-g010">Figure 10A</a>). Induction of the expression of the reporter gene was detected for sucrose concentrations ranging from 20 µM to 20 mM. Maximal induction (3.5 to 4-fold induction) was observed for concentrations higher than 200 µM in the wild-type background (<a href="#pone-0000224-g010">Figure 10A</a>). As expected, the expression of <em>suxA</em> was highly induced in the <em>suxR</em> mutant, confirming the repressor activity of this gene. However, we observed a significant diminution of the induction level at 2 and 20 mM sucrose in this mutant, suggesting the existence of a second level of control. A high and constitutive induction of <em>suxA</em> expression was also observed in the <em>suxB</em> deletion mutant, suggesting that the amylosucrase gene plays a role in the repression of the <em>suxAB</em> operon. At 2 mM sucrose concentration, a reduction in <em>suxA</em> expression level was also observed in this mutant, but it was smaller than that detected in the <em>suxR</em> mutant. The deregulation of <em>suxA</em> expression in the <em>suxB</em> mutant is in agreement with transport studies which showed a higher sucrose transport rate in this mutant (<a href="#pone-0000224-g008">Figure 8C</a>). No induction of <em>suxA</em> expression was observed in the <em>suxC</em> mutant, whatever sucrose concentration was used. This lack of induction certainly reflects the absence of sucrose entry into the cytoplasm of this mutant, thus preventing the alleviation of SuxR repression. This result clearly shows that the induction of <em>suxA</em> expression by sucrose requires the entry of this molecule into the cell. The pattern and level of expression of <em>suxA</em> observed in the <em>suxA</em> mutant were very similar to those observed in the wild-type strain, except that this expression was significantly higher in the mutant in the presence of 2 mM sucrose. This difference was reproducibly observed and further investigations are needed to understand this phenomenon.</p>
<div class="figure" id="pone-0000224-g010"><div class="img"><a name="pone-0000224-g010" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000224.g010&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000224" data-uri="info:doi/10.1371/journal.pone.0000224.g010"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000224.g010&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000224.g010/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000224.g010/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000224.g010/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000224.g010/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000224.g010.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000224.g010/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000224.g010/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000224.g010.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 10. </strong></p><a id="article1.body1.sec2.sec11.fig1.caption1.p1" name="article1.body1.sec2.sec11.fig1.caption1.p1"></a><p>Expression of the <em>Xanthomonas campestris</em> pv. <em>campestris suxA</em> gene in the presence of sucrose, fructose or glucose. The pPr-<em>suxA</em> plasmid carrying the promoterless <em>LacZ</em> reporter gene under the <em>suxA</em> promoter region was used to monitor <em>suxA</em> expression in different genetic backgrounds. Expression was measured in minimal medium and cells were harvested at the indicated times. (A) Expression after 6 hours induction in the presence of different sucrose concentrations, in the wild-type background, the <em>suxR</em> and <em>suxC</em> insertion mutant backgrounds <em>(suxR</em>::pVO and <em>suxC</em>::pVO, respectively), and the <em>suxA</em> and <em>suxB</em> deletion mutant backgrounds <em>(</em>Δ<em>suxA</em> and Δ<em>suxB</em>, respectively). (B) Kinetics of <em>LacZ</em> expression in the wild-type background or in the <em>suxA</em> deletion mutant background (Δ<em>suxA</em>), in presence of 20 mM (triangles) or 100 µM (squares) sucrose. (C) Kinetics of <em>LacZ</em> expression in the wild-type background in the presence of 100 µM sucrose, fructose or glucose.</p>
<span>doi:10.1371/journal.pone.0000224.g010</span></div><a id="article1.body1.sec2.sec11.p5" name="article1.body1.sec2.sec11.p5"></a><p>Expression assays suggested that sucrose transport through SuxA is not high enough to influence the expression of the <em>suxAB</em> operon. Sucrose entry across the outer membrane by slow diffusion seems to be sufficient to promote induction of <em>sux</em> genes after 6 hours growth in the presence of sucrose.</p>


<h4><em>suxA</em> is required for its rapid induction as revealed by using low sucrose concentrations</h4>
<a id="article1.body1.sec2.sec12.p1" name="article1.body1.sec2.sec12.p1"></a><p>We performed time-course experiments comparing <em>suxA</em> expression in the wild-type strain and the <em>suxA</em> deletion mutant grown in presence of different sucrose concentrations (0, 20 µM, 50 µM, 100 µM, 200 µM 500 µM, 1 mM, 2 mM, 5 mM and 20 mM). For clarity, only results obtained with 100 µM and 20 mM are shown in <a href="#pone-0000224-g010">Figure 10B</a>. In the presence of 20 mM sucrose (triangles on <a href="#pone-0000224-g010">Figure 10B</a>), <em>suxA</em> expression was not significantly different after 6 hours growth in the wild-type and the <em>suxA</em> mutant backgrounds. However, a slight difference was observed in the induction level of <em>suxA</em>, from 60 to 120 minutes after sucrose addition. The expression level of <em>suxA</em> was 1.5 to 1.6 fold higher in the wild type strain than in the <em>suxA</em> mutant. Later, this difference tended to decrease and the ratio reached a value of 1.1 after 6 hours. A reproducible and clear difference in curve profiles was noticed in the presence of 100 µM sucrose (squares on <a href="#pone-0000224-g010">Figure 10B</a>). A rapid induction was observed in the wild-type context, from 30 to 120 minutes after sucrose addition, followed by a plateauing, in which expression seems to remain constant. On the other hand, in the <em>suxA</em> deletion mutant background, the induction was linear and less rapid. Between 30 and 120 minutes after sucrose addition, the expression level of <em>suxA</em> was 2 fold higher in the wild-type strain background than in the <em>suxA</em> deletion mutant background, although after 360 minutes, the expression levels were identical in both strains. Similar results were obtained with sucrose concentrations ranging from 20 to 200 µM (data not shown). We propose that these results could reflect the existence of two pathways controlling <em>suxA</em> expression, which can be clearly differentiated with low sucrose concentrations. One of these pathways is controlled by SuxA. These results can be related to the transport analyses showing the existence of two types of sucrose transport across the outer membrane, one being SuxA-dependent and the other one, slower, being dependent on passive diffusion. We thus postulate that the induction of <em>suxA</em> indirectly reflects these two means of transport. Moreover, this induction is specific to sucrose and is not due to its degradation products glucose and fructose, as no induction was observed following addition of 100 µM of these molecules in the wild-type background (<a href="#pone-0000224-g010">Figure 10C</a>). Therefore, these experiments suggest that SuxA plays a role in sucrose transport which might be crucial at low concentrations of this sugar.</p>


<h4>Identification of other <em>Xcc</em> TBDRs in <em>loci</em> probably involved in plant carbohydrates utilization</h4>
<a id="article1.body1.sec2.sec13.p1" name="article1.body1.sec2.sec13.p1"></a><p>The genome of <em>Xcc</em> was then explored to see whether other TBDRs could belong to loci involved in the utilization of other plant compounds. <em>Xcc</em> has an extensive repertoire of plant cell-wall degrading enzymes, with cellulolytic, pectinolytic and hemicellulolytic activities <a href="#pone.0000224-daSilva1">[14]</a>. The analysis of carbohydrate active enzymes identified in the predicted proteomes of 209 Gram negative bacteria and referenced in the CAZy database (<a href="http://afmb.cnrs-mrs.fr/CAZY/">http://afmb.cnrs-mrs.fr/CAZY/</a>) showed that, after <em>Bacteroides</em> sp. and <em>S. degradans</em>, which are well known specialists for polysaccharide degradation, <em>Xcc</em> has one of the highest number of genes involved in polysaccharide metabolism per megabase (29.9, total 152) (<a href="#pone.0000224.s004">Table S2</a>). These genes encode 82 predicted glycosyl hydrolases, 45 glycosyl transferases, 5 polysaccharide lyases, 18 carbohydrate esterases and 2 carbohydrate binding proteins. Interestingly, 46 of these proteins are encoded in the vicinity of 24 TBDR/Ps-TBDR genes, thus suggesting the existence of 19 new loci putatively involved in carbohydrate utilization (<a href="#pone.0000224.s005">Table S3</a>). Among those loci, 7 also contain an inner membrane transporter coding gene and a regulatory gene. For 3 loci identified in this analysis, the substrate probably utilized and transported could be easily deduced from the nature of the degradative enzymes: the <em>XCC0120</em> TBDR gene is localized upstream of genes coding for a pectin methyl esterase and a pectate lyase and might belong to a locus involved in pectin utilization (<a href="#pone-0000224-g011">Figure 11A</a>); the <em>XCC4120</em> TBDR gene is found in a cluster of genes probably involved in xylan metabolism (<a href="#pone-0000224-g011">Figure 11B</a>); and the <em>XCC2469</em> TBDR gene might be related to maltose/maltodextrin utilization, since it is associated with genes coding for a cyclomaltodextrin glucanotransferase, two α-glucosidases, a sugar transporter and a maltose transport gene repressor (<a href="#pone-0000224-g011">Figure 11C</a>).</p>
<a id="article1.body1.sec2.sec13.p2" name="article1.body1.sec2.sec13.p2"></a><p>To verify whether these TBDRs are really associated with carbohydrate utilization, we studied their regulation by plant compounds. For this purpose, we performed β-glucuronidase expression assays using all <em>Xcc</em> TBDR pVO155 insertion mutants. Cells were grown in rich medium or in minimal medium supplemented or not with polygalacturonic acid (PGA), arabinose, glucose, maltose, sucrose, xylose or xylan. Most TBDR (or Ps-TBDR) genes (44 out of 74) were repressed in rich medium, compared to minimal medium (<a href="#pone.0000224.s006">Table S4</a>). We also observed that in minimal medium, 48 <em>Xcc</em> TBDR genes are repressed in the presence of sucrose, xylose, arabinose and/or glucose, suggesting that these genes are submitted to catabolic repression (<a href="#pone.0000224.s006">Table S4</a>). However, the repression patterns were variable, suggesting the existence of several repression pathways. It is worth noting that most Fur-regulated TBDR genes were submitted to catabolic repression.</p>
<div class="figure" id="pone-0000224-g011"><div class="img"><a name="pone-0000224-g011" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000224.g011&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000224" data-uri="info:doi/10.1371/journal.pone.0000224.g011"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000224.g011&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000224.g011/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000224.g011/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000224.g011/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000224.g011/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000224.g011.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000224.g011/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000224.g011/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000224.g011.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 11. </strong></p><a id="article1.body1.sec2.sec13.fig1.caption1.p1" name="article1.body1.sec2.sec13.fig1.caption1.p1"></a><p>Genetic organization of putative <em>Xanthomonas campestris</em> pv. <em>campestris</em> CUT loci and specific induction of the TonB-dependent receptor gene by plant carbohydrates. Putative pectin (A), xylan (B) and maltodextrin (C) utilization CUT loci and specific induction of the corresponding TBDR gene in the presence of PGA, xylose and maltose, respectively. β-glucuronidase assays were performed in at least two independent experiments with TBDR insertion mutants cultivated in minimal medium supplemented with glutamate 20 mM (GT) or specific carbohydrates (PGA: polygalacturonic acid 0.125%; XYL: xylose 20 mM; MAL: maltose 20 mM). TBDR: TonB-dependent receptor; PME: pectin methyl esterase; PL: pectate lyase; CGTase: cyclomaltodextrin glucanotransferase.</p>
<span>doi:10.1371/journal.pone.0000224.g011</span></div><a id="article1.body1.sec2.sec13.p3" name="article1.body1.sec2.sec13.p3"></a><p>Three TBDR/Ps-TBDR genes were induced by PGA (<em>XCC0120, XCC1749</em> and <em>XCC1750-1751</em>), one by maltose (<em>XCC2469</em>), seven by arabinose (<em>XCC0050, XCC1749, XCC1750, XCC1892, XCC2828, XCC4120</em> and <em>XCC4222</em>), three by xylan (<em>XCC2828, XCC4120</em> and <em>XCC4237</em>) and finally two by xylose (<em>XCC2828</em> and <em>XCC4120</em>) (<a href="#pone.0000224.s006">Table S4</a>). Interestingly, the 2 TBDRs induced by xylose were also induced by xylan and arabinose, and <em>XCC1749</em> and <em>XCC1750</em> were both induced by PGA and arabinose. <em>suxA</em> is the only TBDR gene for which expression is specifically induced by sucrose, and <em>XCC2469</em>, the orthologue of <em>C. crescentus malA</em> TBDR gene <a href="#pone.0000224-Neugebauer1">[22]</a>, is the only one induced by maltose. It is worth noting that the expression of the <em>XCC0120, XCC4120</em> and <em>XCC2469</em> TBDR genes was specifically induced by the postulated substrate of the associated CAZy referenced enzymes (see <a href="#pone-0000224-g011">Figure 11</a>). Altogether, these data suggest that several TBDRs are part of loci which seem to be involved in carbohydrate utilization. We thus propose the existence of 6 putative loci named <span class="underline">c</span>arbohydrate <span class="underline">u</span>tilization containing <span class="underline">T</span>BDRs (CUT) loci, which were defined by the presence of genes coding for carbohydrate degradative enzymes, inner membrane transporters and sugar related regulators beside TBDR genes. We also identified 15 putative partial CUT loci in the <em>Xcc</em> genome (see <a href="#pone.0000224.s005">Table S3</a>).</p>


<h4>Conservation and distribution of TBDRs and CUT <em>loci</em> in Xanthomonads</h4>
<a id="article1.body1.sec2.sec14.p1" name="article1.body1.sec2.sec14.p1"></a><p>When compared with proteins in the databases using BlastP and the “distance tree of results option” displayed on the BlastP result page, the best homologous genes of <em>Xcc</em> TBDRs/Ps-TBDRs were putative TBDRs of other <em>Xanthomonas</em> species and in some cases of <em>Xyllela</em> strains. These conserved genes might be considered as orthologs. We compared the distribution of <em>Xcc</em> TBDRs among Xanthomonads strains for which genome sequences are available, i.e. <em>Xcc</em> strain 8004, <em>Xcv</em> (strain 85–10), <em>Xac</em> (strain 306), <em>Xoo</em> (strains KACC10331 and MAFF311018) and <em>Xf</em> (strains 9a5c and PD). A search similar to that used for <em>Xcc</em> allowed us to identify 72 TBDR/Ps-TBDR in the proteome of <em>Xcc</em> strain 8004, 74 in that of <em>Xac</em>, 61 in <em>Xcv</em>, 41 and 42 in <em>Xoo</em> strains KACC10331 and MAFF311018 respectively, and finally 10 in each <em>Xf</em> strain. To further study the relationship between these TBDRs and <em>Xcc</em> TBDRs, we performed a comparative study using alignments generated by ClustalW <a href="#pone.0000224-Thompson1">[64]</a>. To carry out this comparative analysis, we only used TBDRs which seemed to be complete and Ps-TBDRs over 600 amino acids, in order to avoid bias in alignments. Examination of the phylogenetic tree, coupled with Blast results analysis, clearly confirmed that most <em>Xcc</em> TBDRs are well conserved in other <em>Xanthomonas</em> strains (<a href="#pone.0000224.s001">Figure S1</a> and <a href="#pone.0000224.s005">Table S3</a>). All TBDR or Ps-TBDR genes detected in <em>Xcc</em> strain ATCC33913 are present in strain 8004; 55 seem to have orthologs in <em>Xac</em>; 49 are conserved in <em>Xcv</em> and 34 only are conserved in <em>Xoo</em> strains which possess significantly less TBDRs. In most instances, the grouping and distribution pattern of orthologous TBDRs matched the phylogenetic classification of <em>Xanthomonas</em> species <a href="#pone.0000224-Vauterin1">[65]</a>–<a href="#pone.0000224-Cubero1">[67]</a>. This observation suggests that this protein family is ancient in the <em>Xanthomonas</em> genus. It is worth noting that branch lengths were variable, suggesting differential evolution rates. Moreover, we noticed that some truncated <em>Xcc</em> Ps-TBDRs could be functional in other strains: the Ps-TBDR XCC1750-1751 doublet corresponds to unique and complete TBDR proteins in <em>Xcv, Xac</em> and <em>Xcc</em> strain 8004. Similarly, the 2 Ps-TBDRs XCC3215-3216 and XCC3270-3271, which are also truncated in <em>Xcc</em> strain 8004, correspond to unique proteins in both <em>Xac</em> and <em>Xcv</em>. Interestingly, <em>Xac, Xcv</em> or <em>Xoo</em> TBDR orthologs of <em>Xcc</em> Ps-TBDR having non-canonical C-terminal regions (β-sheet ended by an aromatic amino acid followed by a short extension of 1 to 4 amino acids), also displayed this feature (data not shown). In most cases, these non canonical C-terminal domains were identical in all strains. This conservation might reflect a specific functional feature of <em>Xanthomonas</em> TBDRs.</p>
<a id="article1.body1.sec2.sec14.p2" name="article1.body1.sec2.sec14.p2"></a><p>The analysis of the phylogenetic tree also showed that Xanthomonads TBDRs can be divided into three main groups, with most TBDRs belonging to group 1. This group can be divided into 8 subgroups (1A to 1H). We noticed that (i) <em>Xcc</em> Oar-TBDRs are clustered in subgroup 1D, (ii) most plant carbohydrate induced TBDRs (8 out of 11) are grouped in subgroup 1C, and (iii) most <em>Xcc</em> Fur-regulated TBDRs (6 out of 8) are grouped in subgroup 1F. Among these Fur-regulated TBDR genes, <em>XCC3518</em>, <em>XCC3595</em> and <em>XCC4162</em>, which belong to subgroup 1F, seem specific to <em>Xcc</em> strains.</p>
<a id="article1.body1.sec2.sec14.p3" name="article1.body1.sec2.sec14.p3"></a><p>The comparison of genes located adjacent to orthologous TBDR genes in the different <em>Xanthomonas</em> strains showed that in most cases, these regions are syntenic (<a href="#pone.0000224.s005">Table S3</a>). Thus, all putative CUT loci identified in <em>Xcc</em> genome are well conserved in <em>Xac</em> and <em>Xcv</em> but only 2 out of 6 were found in <em>Xoo</em>. Similarly, almost all putative partial CUT loci are present in both <em>Xcv</em> and <em>Xac</em>, but only 9 out of 15 are conserved in <em>Xoo</em> strains. The putative polygalacturonate utilization locus (<em>XCC0120-XCC0122</em>) seems unique to <em>Xcc</em>. The genes bordering this locus are conserved and contiguous in <em>Xcv</em> and <em>Xac</em>, thus suggesting insertion or deletion events (<a href="#pone.0000224.s005">Table S3</a>).</p>
<a id="article1.body1.sec2.sec14.p4" name="article1.body1.sec2.sec14.p4"></a><p>Finally, among the 10 TBDRs identified in <em>Xf</em> strains, 6 are conserved in <em>Xcc, Xac, Xcv</em> and <em>Xoo</em>, 1 is conserved in <em>Xcc, Xac</em> and <em>Xcv</em>, whereas only 1 is present in <em>Xcc</em>. The 2 remaining <em>Xf</em> TBDRs (XF0339/PD1711; XF0599/PD1552) seem more divergent and thus specific to <em>Xf</em> strains. However, XF0599 displays weak similarities with XCC2772, a Fur-regulated TBDR of <em>Xcc</em>. None of the other <em>Xf</em> TBDRs are related to <em>Xcc</em> Fur-regulated TBDRs nor to <em>Xcc</em> TBDRs belonging to putative CUT loci identified in this study. Only 2 <em>Xf</em> TBDR genes (<em>XF2713</em> and <em>XF1036</em>) correspond to <em>Xcc</em> TBDRs present in partial CUT loci.</p>


<h4><em>Xcc</em> TBDRs are conserved in aquatic bacteria and/or phytopathogenic bacteria</h4>
<a id="article1.body1.sec2.sec15.p1" name="article1.body1.sec2.sec15.p1"></a><p>For each <em>Xcc</em> TBDR included in the phylogenetic study, we analyzed the BlastP results obtained on the nr databank, to characterize the next best homologous genes after Xanthomonads orthologs. This analysis allowed us to cluster <em>Xcc</em> TBDRs on the basis of the bacterial origin of the homologous TBDR. Thus, <em>Xcc</em> TBDRs could be divided into 4 main groups (<a href="#pone.0000224.s007">Table S5</a>). The first group corresponds to TBDRs showing their next best homologies (after Xanthomonads orthologs) with other <em>Xanthomonas</em> TBDRs rather than with TBDRs from other genera. This observation mainly concerned TBDRs found in subgroups 1B, 1C and 1D of the phylogenetic tree. Their positions in the tree suggested that they could be generated by successive duplication events that occurred before <em>Xanthomonas</em> speciation (<a href="#pone.0000224.s001">Figure S1</a>). The second group corresponds to TBDRs which display high similarities with TBDRs of β-Proteobacteria and/or <em>Pseudomonas</em> species. Interestingly most of these TBDRs are clustered in subgroup 3 of the phylogenetic tree. Moreover, some of these TBDRs showed high similarities with TBDRs of phytopathogenic bacteria such as <em>Acidovorax avenae</em> subsp. <em>citrulli, Pseudomonas syringae</em> pathovars or <em>Ralstonia solanacearum</em>. The third group contains two <em>Xcc</em> TBDRs proposed to be involved in iron uptake and showing significant similarities with putative TBDRs from <em>Cyanobacteria</em>. The fourth group, which is the largest one (with 35 TBDRs out of 70), corresponds to TBDRs which displayed very significant homologies with putative TBDRs of a wide range of aquatic bacteria belonging to the α or γ classes of Proteobacteria. It is worth noting that in most cases, <em>Xcc</em> TBDRs were not affiliated to TBDRs of a specific bacterial class, but there was rather a variety of origins. Thus, for 16 <em>Xcc</em> TBDRs the best similarities (after Xanthomonads similarities) were obtained with putative TBDRs of <em>C. crescentus</em> CB15 and <em>Caulobacter</em> sp. K31 strains, which are aquatic oligotrophs belonging to the Caulobacterales order of the α-Proteobacteria. Homologies were also observed with TBDRs of other α-Proteobacteria living in aquatic habitats, such as <em>Oceanicaulis alexandrii, Maricaulis maris</em> or <em>Parvilarcula bermudensis</em> HTCC2503, or found in multiple environments like <em>Sphingomonas</em> sp. SKA58, <em>Sphingopyxis alaskensis</em> or <em>Novosphingobium aromacitovorans</em>. Significant similarities were also obtained with putative TBDRs of aquatic γ-Proteobacteria classified in the <em>Alteromonadales</em>, including <em>Alteromonas macleodii, Saccharophagus degradans</em> 2–40, <em>Pseudoalteromonas</em> and <em>Shewanella</em> species. One common trait of most of these bacteria is that they show TBDRs overrepresentation with values of TBDR number per megabase ranging from 7.4 to 15.7 (<a href="#pone.0000224.s004">Table S2</a>). This raised the question of whether homologies between these TBDRs were fortuitous and a consequence of their large number or whether they reflect common biological functions.</p>


<h4>Conservation of TBDR regions and CUT loci in Gram-negative bacteria</h4>
<a id="article1.body1.sec2.sec16.p1" name="article1.body1.sec2.sec16.p1"></a><p>Some regions surrounding TBDR genes were found to be conserved in closely related bacteria such as <em>Vibrio parahaemolyticus</em> or <em>P. aeruginosa</em>. Thus, the XCC3050 Fur-regulated TBDR belongs to a cluster of 6 genes showing similarities with the <em>pvuA</em>-<em>pvsABCDE</em> gene cluster of <em>V. parahaemolyticus</em>, involved in the uptake and biosynthesis of the siderophore vibrioferrin <a href="#pone.0000224-Tanabe1">[68]</a> (<a href="#pone.0000224.s005">Table S3</a>). Similarly, a group of proteins encompassing XCC3067 TBDR and putatively involved in cobalamin uptake and biosynthesis in <em>Xcc</em> was conserved in <em>Pseudomonas</em> species (<a href="#pone.0000224.s005">Table S3</a>). Preliminary experiments showed that the expression of this TBDR gene is repressed by the presence of vitamin B12 in <em>Xcc</em>, thus suggesting that this cluster is functional (data not shown).</p>
<a id="article1.body1.sec2.sec16.p2" name="article1.body1.sec2.sec16.p2"></a><p>Several CUT loci identified in <em>Xcc</em> were not conserved in taxonomically related bacteria but rather in the group of aquatic bacteria showing TBDR conservation and in particular in <em>C. crescentus</em>. The <em>sux</em> locus, 3 of the putative CUT loci and 4 of the putative partial CUT loci identified in this study were entirely or partially conserved in this group of bacteria (<a href="#pone.0000224.s005">Table S3</a>). Thus, the putative maltose CUT locus was partially conserved with the <em>mal</em> locus recently identified in <em>C. crescentus</em>, which contains the MalA TBDR proposed to be involved in the uptake of maltodextrins <a href="#pone.0000224-Neugebauer1">[22]</a> (<a href="#pone.0000224.s002">Figure S2B</a>). Interestingly, MalA shows significant similarities with the <em>XCC2469</em> TBDR whose expression is induced by maltose and maps in the maltose CUT locus (<a href="#pone-0000224-g011">Figure 11C</a>).</p>
<a id="article1.body1.sec2.sec16.p3" name="article1.body1.sec2.sec16.p3"></a><p>Similarly, the putative xylose locus, containing the <em>XCC2828</em> TBDR gene, is also very well conserved in <em>C. crescentus</em> (<a href="#pone.0000224.s002">Figure S2C</a>). The corresponding <em>CC0999</em> TBDR gene was shown to be induced by xylose in this bacterium <a href="#pone.0000224-Hottes1">[69]</a>. Moreover, <em>CC2832</em>, which was also shown to be xylose-induced in <em>C. crescentus</em>, displays significant homologies with the xylose-induced TBDR gene <em>XCC4120</em> belonging to a xylose CUT locus. Furthermore, the 20-bp palindromic motif conserved upstream of genes of the <em>C. crescentus</em> xylose regulon, was also found upstream of genes putatively involved in xylose metabolism in <em>Xcc</em>, as well as in the promoter region of <em>XCC2828</em> and <em>XCC4120</em> TBDR genes <a href="#pone.0000224-Hottes1">[69]</a>.</p>
<a id="article1.body1.sec2.sec16.p4" name="article1.body1.sec2.sec16.p4"></a><p>The <em>sux</em> CUT locus seems less well conserved and showed some degree of variation (<a href="#pone.0000224.s002">Figure S2D</a>). The amylosucrase gene (<em>XCC3359</em>) is only well conserved in the corresponding <em>C. crescentus</em> locus. In the loci conserved in other bacteria, the degradation of sucrose seems to involve more classical pathways (for review see <a href="#pone.0000224-Reid1">[70]</a>). Moreover, the MFS transporter of the <em>Xcc</em> locus is different from that of the <em>C. crescentus</em> sucrose locus encoded by <em>CC1133</em>, although it is well conserved in both <em>Sphingomonas</em> SK58 and <em>Erythrobacter litoralis</em>. These differences show the existence of some degree of plasticity in the evolution of the putative CUT loci. Further investigations are needed to see whether all these partially conserved loci are involved in the utilization of the same molecule. However, this wide conservation of CUT loci is in favor of their existence. These results also confirm that the similarities observed between TBDRs are probably not fortuitous.</p>

</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3>
<h4>Overview</h4>
<a id="article1.body1.sec3.sec1.p1" name="article1.body1.sec3.sec1.p1"></a><p>The outer membrane (OM) of Gram negative bacteria serves as a selective permeation barrier, excluding hydrophilic solutes, including most nutrients. However, OMs contain embedded integral proteins, named outer membrane proteins (OMPs), which allow sensing and entry of nutrients into the cell. A major class of OMP with a certain substrate specificity, called porins, allows the translocation of hydrophilic solutes through the OM. Another class of OMPs, the TonB-dependent receptors (TBDRs), is mainly known to be involved in iron or vitamin B12 uptake <a href="#pone.0000224-Nikaido1">[71]</a>. Recently, the MalA TBDR of <em>C. crescentus</em> was shown to be involved in the ExbBD-dependent uptake of maltodextrins <a href="#pone.0000224-Neugebauer1">[22]</a>. Here, we show that several Gram negative bacteria, belonging to different lineages and having diverse habitats, display an overrepresentation of TBDRs, which might be related to the uptake of plant-derived carbohydrates.</p>


<h4><em>Xcc</em> TBDRs belonging to the Fur regulon</h4>
<a id="article1.body1.sec3.sec2.p1" name="article1.body1.sec3.sec2.p1"></a><p>Our global study of <em>Xcc</em> TBDRs showed that only a small fraction of them seems to be involved in iron uptake. Among the 72 TBDR/Ps-TBDRs identified in the <em>Xcc</em> genome, only 9 are up-regulated under iron-limiting conditions. We established that these 9 genes are repressed by the Fur repressor and that they are the only TBDR genes having a Fur-box in their promoter region. Recently, a proteomic approach carried out in <em>P. aeruginausa</em> PAO1, which contains 34 putative TBDR genes, identified a very similar number of TBDR genes regulated by iron-stavation. In fact, in this bacterium, 7 TBDRs are produced under iron-starvation conditions and 4 others are specifically induced by the presence of heterologous siderophores, under iron-restricted conditions <a href="#pone.0000224-Llamas1">[72]</a>. Therefore, although it is possible that some <em>Xcc</em> TBDR genes need specific heterologous siderophores for their expression, as observed in <em>P. aeruginosa</em>, the number of TBDR genes involved in iron uptake seems very comparable in both bacteria. This suggests that the other putative TBDRs/Ps-TBDRs might be required for different biological functions.</p>


<h4>The identification of CUT loci/systems in <em>Xcc</em> suggests a relationships between TBDRs and carbohydrate utilization</h4>
<a id="article1.body1.sec3.sec3.p1" name="article1.body1.sec3.sec3.p1"></a><p>Our data reveal that several <em>Xcc</em> TBDRs are related to plant carbohydrate utilization. A large proportion of <em>Xcc</em> TBDR genes are coupled with carbohydrate active enzymes. For 6 of them, an inner membrane transporter gene and a regulatory gene are also present in the same region, thus defining the existence of putative CUT loci involved in the uptake and utilization of plant carbohydrates. Moreover, 15 <em>Xcc</em> TBDR genes belong to partial CUT loci, thus suggesting the existence of CUT systems composed of different parts scattered in the genome. The existence of such multipartite CUT systems was supported by the observation that 11 TBDR/Ps-TBDR genes are specifically induced by plant carbohydrates such as sucrose or plant cell wall derived compounds including arabinose, xylan/xylose, pectin/polygalacturonate or maltose. Moreover, in 5 cases, there is a correlation between the inducing carbohydrate and the degradative enzyme(s) present in the CUT locus. This allows us to propose the existence of a sucrose CUT locus as well as loci involved in the utilization of complex carbohydrates such as pectin, xylan or starch.</p>


<h4>The <em>sux</em> CUT locus is functional</h4>
<a id="article1.body1.sec3.sec4.p1" name="article1.body1.sec3.sec4.p1"></a><p>The existence of functional CUT loci was confirmed by the detailed study of the sucrose CUT locus, which showed its involvement in the entry and utilization of sucrose in <em>Xcc</em>. This locus comprises four genes, <em>suxA, suxB, suxC</em> and <em>suxR</em>, coding for a TBDR, an amylosucrase, a sugar inner membrane transporter and a regulatory protein, respectively. [<sup>14</sup>C]sucrose uptake experiments showed that SuxA and SuxC are both required for sucrose entry into the cell. Concentration-dependent sucrose transport experiments showed a biphasic kinetic, with a similar pattern observed for vitamin B12 uptake into <em>E. coli</em> through the BtuB TBDR <a href="#pone.0000224-DiGirolamo1">[55]</a>, and more recently for maltose transport into <em>C. crescentus</em> through the MalA TBDR <a href="#pone.0000224-Neugebauer1">[22]</a>. In both of these systems, it has been concluded that the first phase reflects the binding of the transported molecule to the OM TBDR, and that the second slower phase reflects the binding to a cytoplasmic membrane transporter. Thus, we presume in our experiments that the low <em>K<sub>d</sub></em> value (0.033 µM) mainly reflects binding to SuxA and that the higher <em>K<sub>d</sub></em> value (0.59 µM) binding to SuxC. Sucrose transport was significantly lower in the <em>suxC</em> mutant than in the <em>suxA</em> non polar mutant. This difference might explain the differential phenotype of these two mutants observed for growth on sucrose: the <em>suxA</em> non polar mutant was not impaired in growth on media containing sucrose, whereas the <em>suxC</em> mutant was unable to grow under these conditions. These observations suggest the existence of an alternative pathway that supports facilitated diffusion of sucrose across the OM, whereas it seems that there is a unique route for crossing the inner membrane, depending on SuxC. The existence of this alternative pathway was also revealed by expression analyses of the <em>sux</em> locus. The regulation of <em>sux</em> genes seems to follow a classical inducer/repressor control, mediated by the SuxR repressor, sucrose being the inducer. The expression of <em>sux</em> genes is induced by sucrose but not by fructose or glucose. This induction was detected with sucrose concentrations ranging from 20 µM to 20 mM. These experiments showed that SuxA transport influenced sucrose induction of <em>suxA</em> at low sucrose concentrations (20 to 200 µM), whereas this effect is masked at higher concentrations, probably by interference by passive diffusion.</p>
<a id="article1.body1.sec3.sec4.p2" name="article1.body1.sec3.sec4.p2"></a><p>Our data also indicated that sucrose transport through the <em>sux</em> system is active and depends on the proton motive force. However, we could not conclude whether it depends on the TonB-ExbBD energy coupling system. In addition to TonB, the <em>Xcc</em> genome harbours 7 TonB-like proteins, which might substitute one for another. Another <em>Xcc</em> feature is the presence of a second <em>exbD</em> gene, named <em>exbD2</em>, mapping downstream of the <em>tonB-exbBD1</em> locus. Interestingly, <em>exbD2</em> is not required for iron uptake but is essential for HR induction, whereas <em>tonB</em>, <em>exbB</em> and <em>exbD1</em> genes are necessary for both processes <a href="#pone.0000224-Wiggerich1">[73]</a>, <a href="#pone.0000224-Wiggerich2">[74]</a>. This differential behavior might be related to the existence of at least two classes of TBDRs in <em>Xcc</em>, one involved in iron uptake and the other one in transport of plant compounds. Further work is needed to address this hypothesis.</p>


<h4>The <em>sux</em> locus represents a new sucrose utilization system</h4>
<a id="article1.body1.sec3.sec5.p1" name="article1.body1.sec3.sec5.p1"></a><p>The <em>Xcc sux</em> locus is clearly different from other sucrose utilization loci already found in bacteria (for review see <a href="#pone.0000224-Reid1">[70]</a>). In particular, it differs from the <em>scr</em> sucrose-utilization system from enteric bacteria, which is also present in the plant pathogenic bacterium <em>Erwinia amylovora</em>, where it plays a major role in plant colonization <a href="#pone.0000224-Bogs1">[75]</a>. The differences concern regulation, sucrose utilization and transport. The <em>scr</em> genes are regulated by a LacI/GalR family repressor, but fructose is the inducer <a href="#pone.0000224-Jahreis1">[76]</a>. In this system, the degradation of sucrose uses a sucrose-6-phosphate hydrolase/fructokinase degradation pathway <a href="#pone.0000224-Reid1">[70]</a>. On the contrary, the <em>Xcc sux</em> locus contains an amylosucrase orthologous to SUH, a unique sucrose hydrolase previously characterized in <em>Xag</em> <a href="#pone.0000224-Kim1">[54]</a>. This enzyme is responsible for intracellular sucrose hydrolysis. It is active on sucrose but not on sucrose-6-phosphate. In the s<em>cr</em> system, sucrose metabolism and uptake are coupled by the phosphoeneolpyruvate-dependent carbohydrate:phosphotransferase system (PTS), which controls crossing of the inner membrane and sucrose phosphorylation. The transport across the outer membrane is mediated <em>via</em> a porin, named ScrY <a href="#pone.0000224-Schmid1">[77]</a>, <a href="#pone.0000224-Sprenger1">[78]</a>. Thus, the transport of sucrose is very different in <em>sux</em> and <em>scr</em> systems. Interestingly, the <em>K<sub>d</sub></em> value of sucrose binding to SuxA is 1500- to 3000-fold lower than that of the <em>E. coli</em> ScrY sucrose porin, which varies from 13 mM <a href="#pone.0000224-VanGelder1">[79]</a> to 50 mM <a href="#pone.0000224-Schulein1">[80]</a>. Similarly, <em>C. crescentus</em> MalA TBDR transports maltodextrins with <em>K<sub>d</sub></em> values 1000-fold lower than those of the LamB porin, which facilitates the passive diffusion of maltodextrin <a href="#pone.0000224-Neugebauer1">[22]</a>. It is worth noting that the <em>K<sub>d</sub></em> values of sucrose binding to SuxA and maltodextrins binding to MalA are comparable. Thus, SuxA and MalA represents a new class of outer membrane carbohydrate transporters showing a much higher affinity for their substrate than porins.</p>
<a id="article1.body1.sec3.sec5.p2" name="article1.body1.sec3.sec5.p2"></a><p>The importance of the <em>sux</em> locus in <em>Xcc</em> is highlighted by the fact that it is required for full virulence on <em>Arabidopsis thaliana</em>. The phenotype of <em>suxB</em> and <em>suxC</em> mutants on plants can be related to their inability to grow on medium containing sucrose (20 mM). However, the non polar <em>suxA</em> mutant, which grows as well as the wild type strain on medium containing sucrose, was also affected in pathogenicity. Thus, it appears that the ability to scavenge sucrose with a very high affinity plays a key role during the interaction with host plants.</p>
<a id="article1.body1.sec3.sec5.p3" name="article1.body1.sec3.sec5.p3"></a><p>In conclusion, data obtained on this sucrose CUT locus strongly support the existence of the other CUT loci identified in <em>Xcc</em>, which might have a similar mode of action (see model presented in <a href="#pone-0000224-g012">Figure 12</a>). This suggests the presence of several systems which seem to partially overlap and which are involved in the scavenging of plant molecules. They might form a complex network required for the exploitation of plant resources but which might also participate in signaling.</p>
<div class="figure" id="pone-0000224-g012"><div class="img"><a name="pone-0000224-g012" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000224.g012&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000224" data-uri="info:doi/10.1371/journal.pone.0000224.g012"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000224.g012&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000224.g012/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000224.g012/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000224.g012/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000224.g012/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000224.g012.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000224.g012/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000224.g012/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000224.g012.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 12. </strong></p><a id="article1.body1.sec3.sec5.fig1.caption1.p1" name="article1.body1.sec3.sec5.fig1.caption1.p1"></a><p>Model of CUT loci functioning based on the <em>Xanthomonas campestris</em> pv. <em>campestris sux</em> locus. This scheme shows sucrose outer membrane transport <em>via</em> the SuxA TBDR or by passive diffusion through a putative porin. After crossing the inner membrane through the SuxC transporter, sucrose is proposed to interact with the SuxR repressor (thus allowing <em>sux</em> gene induction) and also to serve as a substrate for the SuxB amylosucrase. The large double headed arrow below the <em>sux</em> locus represents the balance between metabolic adaptation and virulence control putatively mediated by CUT loci.</p>
<span>doi:10.1371/journal.pone.0000224.g012</span></div>

<h4>TBDRs and CUT loci are ancient in the <em>Xanthomonas</em> genus</h4>
<a id="article1.body1.sec3.sec6.p1" name="article1.body1.sec3.sec6.p1"></a><p>The importance of TBDRs and CUT loci is not restricted to <em>Xcc</em> since they are well conserved in <em>Xac</em> and <em>Xcv</em>. A phylogenetic study of Xanthomonads TBDRs suggests that these proteins are an ancient class of proteins in the genus. Moreover, the grouping pattern observed in our phylogenetic study seems to correlate with functional features of <em>Xcc</em> TBDRs, thus suggesting the existence of structure/function relationships. However, it seems that there is some degree of variation in the repertoire of TBDR genes and CUT loci among <em>Xanthomonads</em>, as the number of genes and loci was lower in <em>Xoo</em> strains and also in <em>Xylella</em>. This latter species is considered as a minimal pathogen in the Xanthomonads with a restricted habitat and a reduced genome <a href="#pone.0000224-Puhler1">[81]</a>. <em>Xylella</em> possesses a significantly lower number of TBDRs and only one partially conserved CUT locus.</p>


<h4>Conservation of <em>Xcc</em> TBDRs and CUT loci reveals carbohydrate scavenging abilities in other bacteria</h4>
<a id="article1.body1.sec3.sec7.p1" name="article1.body1.sec3.sec7.p1"></a><p>One of the main surprises of this work was the observation that the large majority of <em>Xcc</em> TBDRs (35 out of 72) display significant similarities with TBDRs of bacteria which are mostly found in aquatic habitats and which are not closely related to Xanthomonads (see <a href="#pone.0000224.s007">Table S5</a>). Phylogenetic studies place Xanthomonads as a deep branch in the γ-Proteobacteria class, close to that of the β-Proteobacteria <a href="#pone.0000224-Anzai1">[66]</a>, <a href="#pone.0000224-Lima1">[82]</a>. The bacteria possessing similar TBDRs belong either to the α-Proteobacteria class or to the Alteromonadales order of the γ-Proteobacteria. Most bacteria in this latter order have been collected from diverse aquatic environments and belong mainly to 6 different genera: <em>Alteromonas, Colwellia, Idiomarina, Pseudoalteromonas, Saccharophagus</em> and <em>Shewanella</em>. The α-Proteobacteria mentioned in this study are also found mostly in aquatic habitats and are members of several orders, including Caulobacterales, Sphingomonadales, Rhodospirillales and Rhodobacterales. Similarities were not restricted to TBDRs genes, and genome context analyses showed that at least 8 putative (partial) CUT loci identified in <em>Xanthomonas</em> species were conserved among members of these aquatic bacteria. Several of them are found in association with abiotic or biotic surfaces, such as the surface of alga or shellfish. Some are adapted to live in oligotrophic environments, while others are abundant in nutrient-rich habitats or in association with particulate of organic matter. Some others are associated with decaying tissues of plant or algae. Although <em>Xanthomonas</em> seems to be very different from these bacteria in terms of habitat, lifestyle and taxonomy, we identified several common traits. All these bacteria belong to the class showing a TBDR overrepresentation (with TBDRs/Mbp ratios&gt;7.4). Most of them are able to degrade a wide variety of plant molecules or other complex carbohydrates such as chitin, alginate, as well as various aromatic compounds. This combination of characters and the similarities with <em>Xcc</em> suggest the existence of a shared biology which might be related to the ability to scavenge carbohydrates. If the situation in these bacteria is similar to that observed in <em>Xcc</em>, we can speculate that there is a significant proportion of TBDRs involved in the uptake of molecules other than iron-siderophores complexes. Moreover, as previously noticed <a href="#pone.0000224-Koebnik2">[33]</a>, we observed that bacteria having a restricted habitat, such as obligate parasites or symbionts, have no or only a very small number of TBDRs. Thus, these proteins might be considered as good indicators of bacterial lifestyle.</p>


<h4>Association between TBDRs and carbohydrate utilization in other Proteobacteria</h4>
<a id="article1.body1.sec3.sec8.p1" name="article1.body1.sec3.sec8.p1"></a><p>A specific role for TBDRs has already been attributed to bacteria displaying an overrepresentation of this protein family. Sphingomonads are widely distributed in nature and are mostly found in soils and aquatic environments. Some strains are associated with plants <a href="#pone.0000224-Hashidoko1">[83]</a>. Strains like <em>Sphingomonas</em> sp. A1 are able to take up and metabolize macromolecules such as alginate produced by brown seaweed and certain bacteria. This uptake is mediated by large structures, so called superchannels, which form a pit in the outer membrane, and which function as a funnel or concentrator (for review, see <a href="#pone.0000224-Hashimoto1">[84]</a>). Four TBDRs seem to be part of this superstructure and are thus proposed to participate in alginate transport <a href="#pone.0000224-Hashimoto2">[85]</a>. Recently, the manipulation of these superchannels revealed the importance of these structures in bioremediation <a href="#pone.0000224-Aso1">[86]</a>.</p>
<a id="article1.body1.sec3.sec8.p2" name="article1.body1.sec3.sec8.p2"></a><p>In <em>C. crescentus</em>, beside the prediction of 67 TBDRs, a genome analysis revealed the presence of several genes for the breakdown of plant polysaccharides as well as transport systems, suggesting that plant polymers are a significant source of nutriment for this organism <a href="#pone.0000224-Nierman1">[87]</a>. Interestingly, several <em>C. crescentus</em> TBDR genes have been shown or proposed to be associated with the uptake of plant compounds. As described above, Neugebauer and colleagues showed that <em>C. crescentus</em> can grow on maltodextrins and that the transport of these molecules is mediated by the MalA TBDR <a href="#pone.0000224-Neugebauer1">[22]</a>. Proteomic and transcriptomic studies have shown that TBDR genes are expressed at higher levels in minimal medium than in rich medium <a href="#pone.0000224-Hottes1">[69]</a>, <a href="#pone.0000224-Phadke1">[88]</a>–<a href="#pone.0000224-Ireland1">[90]</a>, as we observed in <em>Xcc</em>. Moreover, several TBDR genes are specifically induced by the presence of xylose <a href="#pone.0000224-Hottes1">[69]</a>. Interestingly, 2 TBDRs induced by xylose are conserved between <em>C. crescentus</em> and <em>Xcc</em>. Strikingly, putative regulatory <em>cis</em>-element motifs are conserved in the promoters of these homologous genes in both species. Moreover, one of these 2 <em>Xcc</em> TBDR genes (<em>XCC2828</em>) belongs to a putative partial CUT locus which is conserved in <em>C. crescentus</em> (<a href="#pone.0000224.s002">Figure S2C</a>). This suggests a possible lineage between these loci. Moreover, 3 other <em>Xcc</em> CUT loci are entirely or partially conserved in the <em>C. crescentus</em> oligotrophic bacterium, including the maltose and sucrose CUT loci. TBDRs in oligotrophs like <em>C. crescentus</em> might play a very important role by allowing the foraging of carbohydrates in nutrient poor environments.</p>


<h4>Common themes between <em>Xanthomonas</em> and aquatic bacteria</h4>
<a id="article1.body1.sec3.sec9.p1" name="article1.body1.sec3.sec9.p1"></a><p>It is clear that the life cycle of <em>Xanthomonas</em> spp. presents common features with the different lifestyles of bacteria described above. The leaf surfaces encountered by <em>Xanthomonas</em> during their epiphytic development might correspond to an oligotrophic environment. Plant leaves release secondary metabolites which can be used by epiphytic microorganisms <a href="#pone.0000224-Hashidoko1">[83]</a>. However, carbon resources have been shown to be the most limiting resource on plants <a href="#pone.0000224-Wilson1">[91]</a>. Simple sugars, like glucose, fructose and sucrose, are the most dominant carbon sources on plants that have been examined <a href="#pone.0000224-Mercier1">[92]</a>. Studies with bacterial biosensors <em>in situ</em> on plants revealed a high heterogeneity of sucrose availability, with an average accessibility of only about 20 µM on moist bean leaves <a href="#pone.0000224-Leveau1">[93]</a>. Interestingly, this sucrose concentration allowed the induction of <em>suxA</em> gene in our expression assays, thus suggesting that the <em>sux</em> uptake system might function on plant surfaces. The involvement of <em>sux</em> locus in epiphytic life is now being investigated. Furthermore, TBDRs might facilitate the exploitation of plant debris generated during disease development, thus resembling bacteria living on dead tissues. This phenomenon might have an impact on <em>Xanthomonas</em> life cycle by increasing <em>Xanthomonas</em> population size and thus facilitating new infections.</p>


<h4>TBDRs and virulence on plants</h4>
<a id="article1.body1.sec3.sec10.p1" name="article1.body1.sec3.sec10.p1"></a><p>It appears that in <em>Xanthomonas</em>, some TBDRs such as SuxA play an additional and specific role by controlling virulence. Therefore, it is tempting to propose that a strategy shared with non-pathogenic bacteria and based on the active uptake of plant-derived nutrients, could have been diverted by <em>Xanthomonas</em> for the control of pathogenicity. It is worth noting that several phytopathogenic bacteria such as <em>P. syringae</em> or <em>E. carotovora</em> subsp. <em>Atroseptica</em>, as well as <em>Pseudomonas</em> species associated with plants, have an intermediate overrepresentation of TBDRs, suggesting that this feature could be shared by other bacteria interacting with plants. In <em>R. solanacearum</em>, the expression of <em>hrp</em> genes coding for a type III secretion system (TTSS) which controls disease and HR development, is specifically induced upon contact with plant cells <a href="#pone.0000224-Aldon1">[94]</a>. This signaling is mediated by PrhA, a TBDR belonging to the transducer subclass. This receptor senses contact with plant cells and transduces this signal into the cytoplasm via PrhI and PrhR, which are FecI/FecR homologs <a href="#pone.0000224-Aldon1">[94]</a>–<a href="#pone.0000224-Brito1">[96]</a>. Apparently, this regulatory system does not require the transport of plant molecules <a href="#pone.0000224-Aldon1">[94]</a>. Although <em>Xac, Xcc, Xoo</em> or <em>Xcv</em> do not carry any TBDR showing a high homology with PrhA, recently a TBDR controlling both HR and pathogenicity was described in <em>X. oryzae</em> pv. <em>oryzicola</em>, the causal agent of bacterial leaf streak of rice. This TBDR does not belong to the transducer subclass and only shows a weak similarity with PrhA <a href="#pone.0000224-Zou1">[97]</a>. This TBDR gene is highly conserved in <em>Xoo</em> strain MAFF 311018 (<em>XOO0785</em>), <em>Xcv</em> (<em>XCV3654</em>) and <em>Xcc</em> (<em>XCC0674</em>). Interestingly, in <em>Xoo</em> it is located close to <em>trh</em> (<em>XOO0783</em>), a regulatory gene controlling the expression of <em>hrp</em> genes <a href="#pone.0000224-Tsuge1">[53]</a>. The orthologue of this regulatory gene is also conserved in <em>Xcc</em> (<em>XCC0672</em>), in the vicinity of the <em>XCC0674</em> TBDR gene. Thus, it is possible to speculate that both genes might be involved in a common circuit controlling the expression of <em>hrp</em> genes in <em>Xanthomonas</em> spp. In our study, a mutation in <em>XCC0674</em> did not affect the pathogenicity of <em>Xcc</em>. However, as there are more TBDRs in <em>Xcc</em> than in <em>Xoo</em> strains, we can speculate that functional redundancy might have masked the effect of the mutation. Further work is needed to see whether these genes regulate <em>hrp</em> genes in <em>Xcc</em>. Nevertheless, we established a link between <em>hrp</em> genes and TBDRs in <em>Xcc</em>. Indeed, we observed that 2 <em>Xcc</em> TBDR genes are regulated by the <em>hrpG</em> and <em>hrpX</em> regulatory genes. Therefore, it seems that there is an overlap between <em>hrp</em> genes and at least 2 TBDR genes. What is the function of these two TBDRs? Are they specifically required during the infection of plants, through the action of the <em>hrp</em> regulon, to exploit specific released plant molecules?</p>


<h4>What other functions for <em>Xcc</em> TBDRs?</h4>
<a id="article1.body1.sec3.sec11.p1" name="article1.body1.sec3.sec11.p1"></a><p>We have been able to define a putative role for 21 TBDRs out of 72 identified in <em>Xcc</em>. The nature of the molecules putatively transported by the other TBDRs remains to be discovered. It is probable that TBDRs are not restricted to the transport of carbohydrates and that they can take up various other molecules produced by plants. <em>Azospirillum irakensis</em>, a plant associated bacterium, is able to metabolize salicin, a phenolic glycoside produced by plants. Interestingly, the <em>sal</em> operon, which controls the degradation of salicin, contains a TBDR gene which was proposed to be involved in salicin uptake <a href="#pone.0000224-Faure1">[98]</a>. Thus, secondary metabolites including phenolic compounds might be assimilated through TBDRs. We are now trying to characterize which new molecules may be transported by <em>Xcc</em> TBDRs, to better understand the adaptation of this pathogen to host plants.</p>


<h4>TBDRs, CUT loci and evolution in Gram-negative bacteria</h4>
<a id="article1.body1.sec3.sec12.p1" name="article1.body1.sec3.sec12.p1"></a><p>This work has identified the existence of an ensemble of bacteria that have an overrepresentation of TBDR genes, and that share specific loci for the scavenging and utilization of carbohydrates. They belong to very different lineages in Proteobacteria and this raises the question of the origin of these TBDRs and CUT loci. Did they arise by convergent evolution or were they transferred from species to species by lateral gene transfer? Our data suggest that this latter hypothesis is most likely. Recently, a genomic comparative study established that <em>Xcc</em> and <em>Xac</em> have close to 40% of their genes showing highest similarities to genes from non γ-Proteobacteria, especially from α-Proteobacteria (20%). These genes seem to belong to genomic islands, denominated “unusual best-match islands” (UBIs) <a href="#pone.0000224-Lima1">[82]</a>. Interestingly, among 35 UBIs thus identified in <em>Xcc</em> genome, 14 contain TBDRs and/or CUT loci. However, there was no significant difference between the GC content of each of these loci and the overall content of the genome. It is therefore possible that these loci were acquired very early in the evolution of Xanthomonads and thus played a key role in their adaptation to plants. Most bacteria showing a TBDR overrepresentation possess TBDRs or CUT loci conserved with <em>Xcc</em>. However, there is a main exception with members of the Bacteroidetes phylum, as none of the TBDRs characterized in this phylum showed very significant similarities with those identified in <em>Xcc</em>. Bacteroidetes can be encountered in two very different niches, the marine environment and the human intestine <a href="#pone.0000224-Bauer1">[99]</a>, <a href="#pone.0000224-Backhed1">[100]</a>. Marine Bacteroidetes such as <em>Gramella forsetii</em> are associated with particulate of organic matter <a href="#pone.0000224-Bauer1">[99]</a>, whereas those found in the intestine are assembled on partially digested food particles <a href="#pone.0000224-Sonnenburg1">[101]</a>. In both cases, these bacteria are able to consume biopolymers. <em>Bacteroides thetaiotaomicron</em>, which is a prominent mutualist in the distal intestine of adult humans, has the largest ensemble of TBDR genes and glycobiome yet reported (<a href="#pone.0000224.s004">Table S2</a>). Studies have shown that this bacterium has a carbohydrate foraging behavior <a href="#pone.0000224-Backhed1">[100]</a>–<a href="#pone.0000224-Xu1">[102]</a>. It is well known to bind starch through a protein complex of the outer membrane, which comprises SusD and the SusC TBDR protein <a href="#pone.0000224-Shipman1">[103]</a>. One hundred and six paralogs of SusC and fifty three paralogs of SusD were predicted in the <em>B. thetaiotaomicron</em> genome <a href="#pone.0000224-Xu2">[104]</a>. In our study, we did not detect in <em>Xanthomonas</em> genomes any protein displaying similarities with SusD. Moreover, none of the TBDRs identified in <em>Xcc</em> showed strong similarities with SusC. A Blast analysis suggested that the SusC TBDR and the SusD OMP are specifically conserved in Bacteroidetes (data not shown). These results suggest that TBDRs involved in carbohydrate uptake evolved independently in Proteobacteria and Bacteroidetes. The analysis of the function and evolution of TBDRs in these phyla will certainly help us to better understand the adaptation of bacteria to their environment. This knowledge, which concerns the utilization of plant molecules that are widespread in the environment, will have a major impact not only in plant pathology, but also in human health as well as in the cycling of carbon and geobiology in marine environments.</p>

</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4>Bacterial strains, plasmids and growth conditions</h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p>The <em>Xanthomonas campestris</em> pv. <em>campestris</em> (<em>Xcc</em>) strains and plasmids used in this study are listed in <a href="#pone.0000224.s008">Table S6</a>. <em>Xcc</em> cells were grown at 30°C in MOKA rich medium (Yeast Extract 4 g/l, Casamino acids 8 g/l, K<sub>2</sub>HPO<sub>4</sub> 2 g/l, MgSO<sub>4</sub>.7H<sub>2</sub>O 0.3 g/l) or in MME minimal medium <a href="#pone.0000224-Arlat1">[12]</a>. <em>E. coli</em> cells were grown on LB medium <a href="#pone.0000224-Miller1">[105]</a>. Antibiotics were used at the following concentrations for <em>Xcc</em>: rifampicin, 50 µg/ml; kanamycin: 50 µg/ml; tetracycline: 5 µg/ml. Antibiotics were used at the following concentrations for <em>E. coli</em>: ampicillin, 50 µg/ml; kanamycin: 50 µg/ml; tetracycline: 10 µg/ml.</p>
<a id="article1.body1.sec4.sec1.p2" name="article1.body1.sec4.sec1.p2"></a><p>Growth curves were generated using the Bioscreen C instrument (Labsystems, Helsinki, Finland) in three independent experiments. Growth measurements were realized in 200-well microtiter plates on 350 µl volumes of a minimal medium containing 20 mM sucrose, inoculated at an OD<sub>600</sub> = 0.15 from a washed starter culture. Non-inoculated wells were used as asepsis controls. Optical densities at 600 nm values were measured every 30 min over a period of 2 to 3 days at 28°C. The microplates were shaken for 5 sec before each measurement.</p>


<h4>Construction of <em>Xanthomonas campestris</em> pv. <em>campestris</em> mutants</h4>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1"></a><p>Insertion mutants were constructed using the suicide plasmid pVO155 <a href="#pone.0000224-Oke1">[36]</a>. Oligonucleotide primers used for PCR amplification will be provided upon request. Amplicons were 300 bp in average. Location of insertions are indicated in <a href="#pone.0000224.s008">Table S6</a>. Deletion mutants in <em>XCC3358, XCC3359, XCC1990, XCC3209</em> and <em>XCC2927</em> were constructed using the <em>cre-lox</em> system adapted from Marx and colleagues <a href="#pone.0000224-Marx1">[37]</a>, <a href="#pone.0000224-Angot1">[38]</a>. Deleted regions are indicated in <a href="#pone.0000224.s008">Table S6</a>.</p>
<a id="article1.body1.sec4.sec2.p2" name="article1.body1.sec4.sec2.p2"></a><p>A <em>fur</em> mutant strain was obtained using the manganese mutagenesis method <a href="#pone.0000224-Hantke1">[106]</a> in LB medium containing 5 mM MnCl<sub>2</sub>. After incubation for 48 h at 30°C, surviving colonies were harvested for siderophores over-expression on CAS agar plates <a href="#pone.0000224-Schwyn1">[107]</a> adapted for <em>Xanthomonas</em> (K<sub>2</sub>HPO<sub>4</sub> 1 g/l; MgSO<sub>4</sub> 1 mM; Casamino acids 0.15 g/l; (NH<sub>4</sub>)<sub>2</sub>SO<sub>4</sub> 1 g/l; sucrose 20 mM; CAS 60.5 mg/l; HDTMA 72.9 mg/l; FeCl<sub>3</sub> 10 µM). A resistant strain over-expressing siderophores was selected; the <em>fur</em> gene was sequenced and contains a point mutation (T212A leading to L71Q).</p>


<h4>Plasmid constructions</h4>
<a id="article1.body1.sec4.sec3.p1" name="article1.body1.sec4.sec3.p1"></a><p>The <em>XCC3358, XCC3359</em> and <em>XCC1470</em> genes (<em>suxA, suxB</em> and <em>fur</em> respectively, see <a href="#pone-0000224-g006">Figure 6</a> and <a href="#pone.0000224.s008">Table S6</a>) were amplified by PCR using appropriately designed primers (Oligonucleotide primers used for PCR amplification will be provided upon request). PCR products corresponding to <em>suxA</em> and <em>suxB</em> genes were cloned into pCZ525, a derivative of pSC154 <a href="#pone.0000224-Cunnac1">[108]</a>, without <em>cyaA</em>' coding sequence. The obtained plasmids were cloned into pLAFR6 <a href="#pone.0000224-Huynh1">[109]</a> to give pL-<em>XCC3358</em> and pL-<em>XCC3359</em> respectively. The PCR product corresponding to the <em>fur</em> gene with its promoter and terminator sequences was cloned into pFAJ1700 <a href="#pone.0000224-Dombrecht1">[110]</a> to give pL-<em>XCC1470</em>.</p>
<a id="article1.body1.sec4.sec3.p2" name="article1.body1.sec4.sec3.p2"></a><p>The <em>XCC3358, XCC1990</em> and <em>XCC3209</em> promoter regions (see <a href="#pone.0000224.s008">Table S6</a>) were PCR amplified with appropriately designed primers. These promoter regions were cloned as <em>Hind</em>III-<em>Xba</em>I fragments, into the pCZ750 plasmid, a pFAJ1700 <a href="#pone.0000224-Dombrecht1">[110]</a> derivative containing the <em>Kpn</em>I-<em>Asc</em>I <em>lacZ</em> gene from the pCZ367 plasmid <a href="#pone.0000224-Cunnac1">[108]</a>.</p>


<h4>Expression studies</h4>
<a id="article1.body1.sec4.sec4.p1" name="article1.body1.sec4.sec4.p1"></a><p>β-galactosidase and β-glucuronidase assays: bacterial cultures in the appropriate medium were harvested at different time points and β-galactosidase and β-glucuronidase assays were performed as previously described <a href="#pone.0000224-Miller2">[111]</a>, <a href="#pone.0000224-Jefferson1">[112]</a>.</p>
<a id="article1.body1.sec4.sec4.p2" name="article1.body1.sec4.sec4.p2"></a><p>Quantitative RT-PCR (qRT-PCR): a 6 hour bacterial culture in the appropriate medium was harvested at an OD<sub>600</sub> = 0.4 to 0.6. RNA were extracted using the Rneasy Mini Kit (QIAGEN). One µg of RNA was treated with RNase-free DNase I (GE-Healthcare) for 15 min at 37°C. After DNase inactivation (10 min at 75°C), RNA were reverse-transcribed by Superscript II (Invitrogen) using random hexamers (Biolabs), for 2 min at 25°C followed by 1 h at 42°C. Quantitative-PCR amplification was performed on Light Cycler (Roche): 10 min 95°C, 1 cycle; 10 sec 95°C, 10 sec 65°C, 20 sec 72°C, 40 to 50 cycles). Experiments were carried out in three independent biological experiments. Oligonucleotide primers used for quantitative-PCR amplification will be provided upon request. As a control for real-time PCR, we used the 16S rRNA as described <a href="#pone.0000224-Morales1">[113]</a>. The 16S rRNA forward primer (5′-TGACGGTACCCAAAGAATAAGCA-3′) and 16S rRNA reverse primer (5′-ACGCTTGCACCCTTCGTATTA-3′) amplicon was 72 bp in length.</p>


<h4>Pathogenicity tests</h4>
<a id="article1.body1.sec4.sec5.p1" name="article1.body1.sec4.sec5.p1"></a><p>Pathogenicity tests were conducted on <em>Arabidopsis thaliana</em> Sf-2 ecotype as previously described <a href="#pone.0000224-Meyer2">[114]</a>. Each strain was tested on sets of 4 plants with 4 leaves per plant. Disease development was scored at days 5, 7 and 9 post-inoculation using a disease index ranging from 0 (no symptom), to 4 (leaf death).</p>


<h4>[<sup>14</sup>C]sucrose transport experiments</h4>
<a id="article1.body1.sec4.sec6.p1" name="article1.body1.sec4.sec6.p1"></a><p>Overnight cultures in minimal medium (MME) without carbon source (uninduced) or with 20 mM sucrose (induced) were centrifuged. Pellets were resuspended in MME and the OD<sub>600</sub> was adjusted to 1. [<sup>14</sup>C]sucrose (PerkinElmer, specific activity of 21.8 GBq/mmol) was added to a final concentration of 0.5 µM. For competition experiments, sucrose, fructose or glucose was added to [<sup>14</sup>C]sucrose at a final concentration ranging from 0.5 to 100 µM. After different times, from 20 sec to 2 hours, samples of 0.2 ml were collected on cellulose nitrate filters, washed with 10 ml water, dried, and finally, the radioactivity was determined in a liquid scintillation counter.</p>
<a id="article1.body1.sec4.sec6.p2" name="article1.body1.sec4.sec6.p2"></a><p>The concentration-dependent initial sucrose transport was determined using the rapid dilution method as described <a href="#pone.0000224-Neugebauer1">[22]</a>. Cells were precultured in minimal medium without sugar. After centrifugation and adjustment to an OD<sub>600</sub> of 1, cells were incubated for 15 sec in presence of 0.01, 0.025, 0.05, 0.1, 0.25, 0.5, 1, 2.5 and 5 µM [<sup>14</sup>C]sucrose and 0.2 ml samples were diluted into 5 ml MME supplemented with 0.1 mM sucrose. Cells were collected by filtration, washed with 10 ml MME supplemented with 10 mM sucrose, dried and the radioactivity was determined.</p>
<a id="article1.body1.sec4.sec6.p3" name="article1.body1.sec4.sec6.p3"></a><p>For inhibition of the proton motive force (PMF) with carbonyl cyanide 3-chlorophenyl-hydrazone (CCCP), cells were incubated for 10 minutes at 30°C with 20 µM CCCP prior to the addition of [<sup>14</sup>C]sucrose.</p>


<h4><em>In silico</em> analysis</h4>
<a id="article1.body1.sec4.sec7.p1" name="article1.body1.sec4.sec7.p1"></a><p>Location of the signal sequence responsible for the outer membrane localization was determined using the SignalP 3.0 server <a href="#pone.0000224-Bendtsen1">[115]</a> (<a href="http://www.cbs.dtu.dk/services/SignalP/">http://www.cbs.dtu.dk/services/SignalP/</a>) with default parameters for Gram-negative bacteria.</p>
<a id="article1.body1.sec4.sec7.p2" name="article1.body1.sec4.sec7.p2"></a><p>Comparison alignments used to identify the TonB-box were realized using ClustalW <a href="#pone.0000224-Thompson1">[64]</a> (<a href="http://www.ebi.ac.uk/clustalw/">http://www.ebi.ac.uk/clustalw/</a>) or Multalin <a href="#pone.0000224-Corpet1">[116]</a> (<a href="http://prodes.toulouse.inra.fr/multalin/multalin.html">http://prodes.toulouse.inra.fr/multalin/​multalin.html</a>) softwares.</p>
<a id="article1.body1.sec4.sec7.p3" name="article1.body1.sec4.sec7.p3"></a><p>β-sheets in the last 50 amino-acids of <em>Xcc</em> TBDRs were located using the secondary structure prediction method <a href="#pone.0000224-Jones1">[117]</a> on the PSIPRED Protein Structure Prediction Server <a href="#pone.0000224-Bryson1">[118]</a>, <a href="#pone.0000224-McGuffin1">[119]</a> (<a href="http://bioinf.cs.ucl.ac.uk/psipred/">http://bioinf.cs.ucl.ac.uk/psipred/</a>).</p>
<a id="article1.body1.sec4.sec7.p4" name="article1.body1.sec4.sec7.p4"></a><p>The MotifSampler program <a href="#pone.0000224-Thijs1">[41]</a>, <a href="#pone.0000224-Thijs2">[42]</a> was used to identify a motif corresponding to the <em>Xcc</em> Fux-box upstream of the 9 Fur-repressed TBDR and Ps-TBDR genes. MotifScanner program <a href="#pone.0000224-Thijs2">[42]</a> or the PatScan pattern matcher software <a href="#pone.0000224-Dsouza1">[47]</a> were used with the identified motif to locate all the Fur-boxes in the <em>Xcc</em> genome.</p>
<a id="article1.body1.sec4.sec7.p5" name="article1.body1.sec4.sec7.p5"></a><p>Pip boxes (TTCGCN<sub>15</sub>TTCGC) <a href="#pone.0000224-Fenselau1">[120]</a> and hrp<sub>II</sub> boxes (TTCGN<sub>16</sub>TTCG) <a href="#pone.0000224-Cunnac2">[121]</a> were identified in the <em>Xcc</em> genome using the PatScan software <a href="#pone.0000224-Dsouza1">[47]</a> <a href="http://www-unix.mcs.anl.gov/compbio/PatScan/HTML/patscan.html">http://www-unix.mcs.anl.gov/compbio/PatS​can/HTML/patscan.html</a>).</p>
<a id="article1.body1.sec4.sec7.p6" name="article1.body1.sec4.sec7.p6"></a><p>For phylogenetic analysis, amino acid sequences were aligned and phylogenetic trees were reconstructed by the neighbor-joining method as implemented in ClustalX <a href="#pone.0000224-Thompson2">[122]</a>.</p>

</div>

<div id="section5" class="section"><a id="s5" name="s5" toc="s5" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0000224"></div><a name="pone.0000224.s001" id="pone.0000224.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000224.s001">Figure S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1"></a><p class="preSiDOI">Phylogenetic tree of the family of TonB-dependent receptor proteins from Xanthomonas campestris pv. campestris strains ATCC33913 and 8004, Xanthomonas axonopodis pv. citri strain 306, Xanthomonas campestris pv. vesicatoria strain 85–10, Xanthomonas oryzae pv. oryzae strains KACC10331 and MAFF311018 and Xylella fastidiosa strains 9a5c and PD.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000224.s001</p><a id="article1.body1.sec5.supplementary-material1.caption1.p2" name="article1.body1.sec5.supplementary-material1.caption1.p2"></a><p class="postSiDOI">(0.04 MB PDF)</p>
<a name="pone.0000224.s002" id="pone.0000224.s002"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000224.s002">Figure S2. </a></strong></p><a id="article1.body1.sec5.supplementary-material2.caption1.p1" name="article1.body1.sec5.supplementary-material2.caption1.p1"></a><p class="preSiDOI">Genome context of genes associated with TonB-dependent receptors present in putative (partial) CUT loci of Xanthomonas campestris pv. campestris (Xcc) and conservation in non related bacteria. Homologous genes are marked by matching colors. White color indicates non conserved genes. For conserved genes, percentages of identity and similarity to the corresponding Xcc gene are indicated beneath.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000224.s002</p><a id="article1.body1.sec5.supplementary-material2.caption1.p2" name="article1.body1.sec5.supplementary-material2.caption1.p2"></a><p class="postSiDOI">(0.24 MB PDF)</p>
<a name="pone.0000224.s003" id="pone.0000224.s003"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000224.s003">Table S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material3.caption1.p1" name="article1.body1.sec5.supplementary-material3.caption1.p1"></a><p class="preSiDOI">Regions and domains of Xanthomonas campestris pv. campestris putative TonB-dependent receptors (TBDRs).</p>
<p class="siDoi">doi:10.1371/journal.pone.0000224.s003</p><a id="article1.body1.sec5.supplementary-material3.caption1.p2" name="article1.body1.sec5.supplementary-material3.caption1.p2"></a><p class="postSiDOI">(0.04 MB XLS)</p>
<a name="pone.0000224.s004" id="pone.0000224.s004"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000224.s004">Table S2. </a></strong></p><a id="article1.body1.sec5.supplementary-material4.caption1.p1" name="article1.body1.sec5.supplementary-material4.caption1.p1"></a><p class="preSiDOI">Distribution of TonB-dependent receptors in 226 sequenced Gram negative bacterial genomes. The EMBL-EBI, Integr8 web portal (<a href="http://www.ebi.ac.uk/integr8/EBI-Integr8-HomePage.do">http://www.ebi.ac.uk/integr8/EBI-Integr8​-HomePage.do</a>) and the NCBI ENTREZ Genome Project database (<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?">http://www.ncbi.nlm.nih.gov/entrez/query​.fcgi?</a>db = genomeprj) were used to select sequenced bacterial genomes. TonB-dependent receptors (TBDRs) were detected by screening the Pfam database and the UniprotKB protein knowledge database using the two Pfam domains (PF07715, Plug; PF00593, TonB_dep_Rec). Only proteins displaying the two Pfam domains were considered.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000224.s004</p><a id="article1.body1.sec5.supplementary-material4.caption1.p2" name="article1.body1.sec5.supplementary-material4.caption1.p2"></a><p class="postSiDOI">(0.08 MB XLS)</p>
<a name="pone.0000224.s005" id="pone.0000224.s005"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000224.s005">Table S3. </a></strong></p><a id="article1.body1.sec5.supplementary-material5.caption1.p1" name="article1.body1.sec5.supplementary-material5.caption1.p1"></a><p class="preSiDOI">Genome context of Xanthomonas campestris pv. campestris (Xcc) TonB-dependent receptor (TBDR) genes and definition of putative CUT loci, conservation in Xanthomonads or in other genera, and TBDR conservation in Pseudomonas aeruginosa PAO1.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000224.s005</p><a id="article1.body1.sec5.supplementary-material5.caption1.p2" name="article1.body1.sec5.supplementary-material5.caption1.p2"></a><p class="postSiDOI">(0.18 MB XLS)</p>
<a name="pone.0000224.s006" id="pone.0000224.s006"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000224.s006">Table S4. </a></strong></p><a id="article1.body1.sec5.supplementary-material6.caption1.p1" name="article1.body1.sec5.supplementary-material6.caption1.p1"></a><p class="preSiDOI">Differential expression ratios of Xanthomonas campestris pv. campestris TonB-dependent receptor (TBDR) pVO155 insertion mutants.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000224.s006</p><a id="article1.body1.sec5.supplementary-material6.caption1.p2" name="article1.body1.sec5.supplementary-material6.caption1.p2"></a><p class="postSiDOI">(0.03 MB XLS)</p>
<a name="pone.0000224.s007" id="pone.0000224.s007"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000224.s007">Table S5. </a></strong></p><a id="article1.body1.sec5.supplementary-material7.caption1.p1" name="article1.body1.sec5.supplementary-material7.caption1.p1"></a><p class="preSiDOI">Conservation of Xanthomonas campestris pv. campestris (Xcc) TonB-dependent receptors (TBDRs) beside Xanthomonads orthologies.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000224.s007</p><a id="article1.body1.sec5.supplementary-material7.caption1.p2" name="article1.body1.sec5.supplementary-material7.caption1.p2"></a><p class="postSiDOI">(0.04 MB XLS)</p>
<a name="pone.0000224.s008" id="pone.0000224.s008"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000224.s008">Table S6. </a></strong></p><a id="article1.body1.sec5.supplementary-material8.caption1.p1" name="article1.body1.sec5.supplementary-material8.caption1.p1"></a><p class="preSiDOI">List of plasmids and Xanthomonas campestris pv. campestris strains used or generated in this study.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000224.s008</p><a id="article1.body1.sec5.supplementary-material8.caption1.p2" name="article1.body1.sec5.supplementary-material8.caption1.p2"></a><p class="postSiDOI">(0.03 MB XLS)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>We would like to thank J. Cullimore for critical comments on the manuscript, J. Gouzy and S. Carrere for help in sequence analysis.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: EL SB DM MA. Performed the experiments: EL SB DM AB ML CG ND JV. Analyzed the data: EL SB DM MA. Contributed reagents/materials/analysis tools: EL SB DM MA. Wrote the paper: EL MA.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0000224-Galperin1" id="pone.0000224-Galperin1"></a>Galperin MY (2004) Bacterial signal transduction network in a genomic perspective. Environ Microbiol  6: 552–567.  <ul class="find" data-citedArticleID="931264" data-doi="10.1111/j.1462-2920.2004.00633.x"><li><a href="http://dx.doi.org/10.1111/j.1462-2920.2004.00633.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Bacterial+signal+transduction+network+in+a+genomic+perspective." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Bacterial+signal+transduction+network+in+a+genomic+perspective.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0000224-Cases1" id="pone.0000224-Cases1"></a>Cases I, de Lorenzo V (2005) Genetically modified organisms for the environment: stories of success and failure and what we have learned from them. Int Microbiol  8: 213–222.  <ul class="find" data-citedArticleID="931232"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Genetically+modified+organisms+for+the+environment%3A+stories+of+success+and+failure+and+what+we+have+learned+from+them.&amp;auth=&amp;atitle=Genetically+modified+organisms+for+the+environment%3A+stories+of+success+and+failure+and+what+we+have+learned+from+them." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genetically+modified+organisms+for+the+environment%3A+stories+of+success+and+failure+and+what+we+have+learned+from+them." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genetically+modified+organisms+for+the+environment%3A+stories+of+success+and+failure+and+what+we+have+learned+from+them.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0000224-Papa1" id="pone.0000224-Papa1"></a>Papa R, Glagla S, Danchin A, Schweder T, Marino G, et al.  (2006) Proteomic identification of a two-component regulatory system in <em>Pseudoalteromonas haloplanktis</em> TAC125. Extremophiles  10: 483–91.  <ul class="find" data-citedArticleID="931346" data-doi="10.1007/s00792-006-0525-0"><li><a href="http://dx.doi.org/10.1007/s00792-006-0525-0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Proteomic+identification+of+a+two-component+regulatory+system+in+Pseudoalteromonas+haloplanktis+TAC125." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Proteomic+identification+of+a+two-component+regulatory+system+in+Pseudoalteromonas+haloplanktis+TAC125.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0000224-Williams1" id="pone.0000224-Williams1"></a>Williams PH (1980) Black rot: a continuing threat to world crucifers. Plant Disease  64: 736–742.  <ul class="find" data-citedArticleID="931416" data-doi="10.1094/pd-64-736"><li><a href="http://dx.doi.org/10.1094/pd-64-736" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Black+rot%3A+a+continuing+threat+to+world+crucifers." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Black+rot%3A+a+continuing+threat+to+world+crucifers.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0000224-Alvarez1" id="pone.0000224-Alvarez1"></a>Alvarez AM (2000)  Black rot of crucifers. In: AJSe al, editor. Mechanisms of resistance to plant diseases. Dordrecht, Netherlands: Kluwer Academic Publications. pp. 21–52.  <ul class="find-nolinks"></ul></li><li><span class="label">6.
              </span><a name="pone.0000224-Gough1" id="pone.0000224-Gough1"></a>Gough CL, Dow JM, Keen J, Henrissat B, Daniels MJ (1990) Nucleotide sequence of the engXCA gene encoding the major endoglucanase of <em>Xanthomonas campestris</em> pv. <em>campestris</em>. Gene  89: 53–59.  <ul class="find" data-citedArticleID="931266" data-doi="10.1016/0378-1119(90)90205-6"><li><a href="http://dx.doi.org/10.1016/0378-1119(90)90205-6" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Nucleotide+sequence+of+the+engXCA+gene+encoding+the+major+endoglucanase+of+Xanthomonas+campestris+pv.+campestris." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Nucleotide+sequence+of+the+engXCA+gene+encoding+the+major+endoglucanase+of+Xanthomonas+campestris+pv.+campestris.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0000224-Tang1" id="pone.0000224-Tang1"></a>Tang JL, Gough CL, Daniels MJ (1990) Cloning of genes involved in negative regulation of production of extracellular enzymes and polysaccharide of <em>Xanthomonas campestris</em> pathovar <em>campestris</em>. Mol Gen Genet  222: 157–160.  <ul class="find" data-citedArticleID="931390" data-doi="10.1007/bf00260653"><li><a href="http://dx.doi.org/10.1007/bf00260653" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Cloning+of+genes+involved+in+negative+regulation+of+production+of+extracellular+enzymes+and+polysaccharide+of+Xanthomonas+campestris+pathovar+campestris." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Cloning+of+genes+involved+in+negative+regulation+of+production+of+extracellular+enzymes+and+polysaccharide+of+Xanthomonas+campestris+pathovar+campestris.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0000224-Dow1" id="pone.0000224-Dow1"></a>Dow JM, Clarke BR, Milligan DE, Tang JL, Daniels MJ (1990) Extracellular proteases from <em>Xanthomonas campestris</em> pv. <em>campestris</em>, the black rot pathogen. Appl Environ Microbiol  56: 2994–2998.  <ul class="find" data-citedArticleID="931250"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Extracellular+proteases+from+Xanthomonas+campestris+pv.+campestris%2C+the+black+rot+pathogen.&amp;auth=&amp;atitle=Extracellular+proteases+from+Xanthomonas+campestris+pv.+campestris%2C+the+black+rot+pathogen." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Extracellular+proteases+from+Xanthomonas+campestris+pv.+campestris%2C+the+black+rot+pathogen." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Extracellular+proteases+from+Xanthomonas+campestris+pv.+campestris%2C+the+black+rot+pathogen.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0000224-Slater1" id="pone.0000224-Slater1"></a>Slater H, Alvarez-Morales A, Barber CE, Daniels MJ, Dow JM (2000) A two-component system involving an HD-GYP domain protein links cell-cell signalling to pathogenicity gene expression in <em>Xanthomonas campestris</em>. Mol Microbiol  38: 986–1003.  <ul class="find" data-citedArticleID="931376" data-doi="10.1046/j.1365-2958.2000.02196.x"><li><a href="http://dx.doi.org/10.1046/j.1365-2958.2000.02196.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+two-component+system+involving+an+HD-GYP+domain+protein+links+cell-cell+signalling+to+pathogenicity+gene+expression+in+Xanthomonas+campestris." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+two-component+system+involving+an+HD-GYP+domain+protein+links+cell-cell+signalling+to+pathogenicity+gene+expression+in+Xanthomonas+campestris.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0000224-Ryan1" id="pone.0000224-Ryan1"></a>Ryan RP, Fouhy Y, Lucey JF, Crossman LC, Spiro S, et al.  (2006) Cell-cell signaling in <em>Xanthomonas campestris</em> involves an HD-GYP domain protein that functions in cyclic di-GMP turnover. Proc Natl Acad Sci U S A  103: 6712–6717.  <ul class="find" data-citedArticleID="931362" data-doi="10.1073/pnas.0600345103"><li><a href="http://dx.doi.org/10.1073/pnas.0600345103" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Cell-cell+signaling+in+Xanthomonas+campestris+involves+an+HD-GYP+domain+protein+that+functions+in+cyclic+di-GMP+turnover." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Cell-cell+signaling+in+Xanthomonas+campestris+involves+an+HD-GYP+domain+protein+that+functions+in+cyclic+di-GMP+turnover.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0000224-Dow2" id="pone.0000224-Dow2"></a>Dow JM, Crossman L, Findlay K, He YQ, Feng JX, et al.  (2003) Biofilm dispersal in <em>Xanthomonas campestris</em> is controlled by cell-cell signaling and is required for full virulence to plants. Proc Natl Acad Sci U S A  100: 10995–11000.  <ul class="find" data-citedArticleID="931252" data-doi="10.1073/pnas.1833360100"><li><a href="http://dx.doi.org/10.1073/pnas.1833360100" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Biofilm+dispersal+in+Xanthomonas+campestris+is+controlled+by+cell-cell+signaling+and+is+required+for+full+virulence+to+plants." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Biofilm+dispersal+in+Xanthomonas+campestris+is+controlled+by+cell-cell+signaling+and+is+required+for+full+virulence+to+plants.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0000224-Arlat1" id="pone.0000224-Arlat1"></a>Arlat M, Gough CL, Barber CE, Boucher C, Daniels MJ (1991) <em>Xanthomonas campestris</em> contains a cluster of <em>hrp</em> genes related to the larger hrp cluster of <em>Pseudomonas solanacearum</em>. Mol Plant Microbe Interact  4: 593–601.  <ul class="find" data-citedArticleID="931200" data-doi="10.1094/mpmi-4-593"><li><a href="http://dx.doi.org/10.1094/mpmi-4-593" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Xanthomonas+campestris+contains+a+cluster+of+hrp+genes+related+to+the+larger+hrp+cluster+of+Pseudomonas+solanacearum." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Xanthomonas+campestris+contains+a+cluster+of+hrp+genes+related+to+the+larger+hrp+cluster+of+Pseudomonas+solanacearum.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0000224-Meyer1" id="pone.0000224-Meyer1"></a>Meyer D, Cunnac S, Gueneron M, Declercq C, Van Gijsegem F, et al.  (2006) PopF1 and PopF2, two proteins secreted by the type III protein secretion system of <em>Ralstonia solanacearum</em>, are translocators belonging to the HrpF/NopX family. J Bacteriol  188: 4903–4917.  <ul class="find" data-citedArticleID="931322" data-doi="10.1128/jb.00180-06"><li><a href="http://dx.doi.org/10.1128/jb.00180-06" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=PopF1+and+PopF2%2C+two+proteins+secreted+by+the+type+III+protein+secretion+system+of+Ralstonia+solanacearum%2C+are+translocators+belonging+to+the+HrpF%2FNopX+family." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22PopF1+and+PopF2%2C+two+proteins+secreted+by+the+type+III+protein+secretion+system+of+Ralstonia+solanacearum%2C+are+translocators+belonging+to+the+HrpF%2FNopX+family.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0000224-daSilva1" id="pone.0000224-daSilva1"></a>da Silva ACR, Ferro JA, Reinach FC, Farah CS, Furlan LR, et al.  (2002) Comparison of the genomes of two <em>Xanthomonas</em> pathogens with differing host specificities. Nature  417: 459–463.  <ul class="find" data-citedArticleID="931430" data-doi="10.1038/417459a"><li><a href="http://dx.doi.org/10.1038/417459a" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Comparison+of+the+genomes+of+two+Xanthomonas+pathogens+with+differing+host+specificities." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Comparison+of+the+genomes+of+two+Xanthomonas+pathogens+with+differing+host+specificities.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0000224-Qian1" id="pone.0000224-Qian1"></a>Qian W, Jia Y, Ren SX, He YQ, Feng JX, et al.  (2005) Comparative and functional genomic analyses of the pathogenicity of phytopathogen <em>Xanthomonas campestris</em> pv. <em>campestris</em>. Genome Res  15: 757–767.  <ul class="find" data-citedArticleID="931358" data-doi="10.1101/gr.3378705"><li><a href="http://dx.doi.org/10.1101/gr.3378705" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Comparative+and+functional+genomic+analyses+of+the+pathogenicity+of+phytopathogen+Xanthomonas+campestris+pv.+campestris." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Comparative+and+functional+genomic+analyses+of+the+pathogenicity+of+phytopathogen+Xanthomonas+campestris+pv.+campestris.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0000224-Thieme1" id="pone.0000224-Thieme1"></a>Thieme F, Koebnik R, Bekel T, Berger C, Boch J, et al.  (2005) Insights into genome plasticity and pathogenicity of the plant pathogenic bacterium <em>Xanthomonas campestris</em> pv. <em>vesicatoria</em> revealed by the complete genome sequence. J Bacteriol  187: 7254–7266.  <ul class="find" data-citedArticleID="931392" data-doi="10.1128/jb.187.21.7254-7266.2005"><li><a href="http://dx.doi.org/10.1128/jb.187.21.7254-7266.2005" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Insights+into+genome+plasticity+and+pathogenicity+of+the+plant+pathogenic+bacterium+Xanthomonas+campestris+pv.+vesicatoria+revealed+by+the+complete+genome+sequence." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Insights+into+genome+plasticity+and+pathogenicity+of+the+plant+pathogenic+bacterium+Xanthomonas+campestris+pv.+vesicatoria+revealed+by+the+complete+genome+sequence.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0000224-Lee1" id="pone.0000224-Lee1"></a>Lee BM, Park YJ, Park DS, Kang HW, Kim JG, et al.  (2005) The genome sequence of <em>Xanthomonas oryzae</em> pathovar <em>oryzae</em> KACC10331, the bacterial blight pathogen of rice. Nucleic Acids Res  33: 577–586.  <ul class="find" data-citedArticleID="931300" data-doi="10.1093/nar/gki206"><li><a href="http://dx.doi.org/10.1093/nar/gki206" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+genome+sequence+of+Xanthomonas+oryzae+pathovar+oryzae+KACC10331%2C+the+bacterial+blight+pathogen+of+rice." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+genome+sequence+of+Xanthomonas+oryzae+pathovar+oryzae+KACC10331%2C+the+bacterial+blight+pathogen+of+rice.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0000224-Ochiai1" id="pone.0000224-Ochiai1"></a>Ochiai H, Inoue Y, Takeya M, Sasaki A, Kaku H (2005) Genome sequence of <em>Xanthomonas oryzae</em> pv. <em>oryzae</em> suggests contribution of large numbers of effector genes and insertion sequences to its race diversity. Japan Agricultural Research Quarterly  39: 275–287.  <ul class="find" data-citedArticleID="931342" data-doi="10.6090/jarq.39.275"><li><a href="http://dx.doi.org/10.6090/jarq.39.275" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genome+sequence+of+Xanthomonas+oryzae+pv.+oryzae+suggests+contribution+of+large+numbers+of+effector+genes+and+insertion+sequences+to+its+race+diversity." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genome+sequence+of+Xanthomonas+oryzae+pv.+oryzae+suggests+contribution+of+large+numbers+of+effector+genes+and+insertion+sequences+to+its+race+diversity.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0000224-Postle1" id="pone.0000224-Postle1"></a>Postle K, Kadner RJ (2003) Touch and go: tying TonB to transport. Mol Microbiol  49: 869–882.  <ul class="find" data-citedArticleID="931354" data-doi="10.1046/j.1365-2958.2003.03629.x"><li><a href="http://dx.doi.org/10.1046/j.1365-2958.2003.03629.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Touch+and+go%3A+tying+TonB+to+transport." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Touch+and+go%3A+tying+TonB+to+transport.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0000224-Bagg1" id="pone.0000224-Bagg1"></a>Bagg A, Neilands JB (1987) Ferric uptake regulation protein acts as a repressor, employing iron (II) as a cofactor to bind the operator of an iron transport operon in <em>Escherichia coli</em>. Biochemistry  26: 5471–5477.  <ul class="find" data-citedArticleID="931206" data-doi="10.1021/bi00391a039"><li><a href="http://dx.doi.org/10.1021/bi00391a039" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Ferric+uptake+regulation+protein+acts+as+a+repressor%2C+employing+iron+%28II%29+as+a+cofactor+to+bind+the+operator+of+an+iron+transport+operon+in+Escherichia+coli." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Ferric+uptake+regulation+protein+acts+as+a+repressor%2C+employing+iron+%28II%29+as+a+cofactor+to+bind+the+operator+of+an+iron+transport+operon+in+Escherichia+coli.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0000224-Ahmer1" id="pone.0000224-Ahmer1"></a>Ahmer BM, Thomas MG, Larsen RA, Postle K (1995) Characterization of the <em>exbBD</em> operon of <em>Escherichia coli</em> and the role of ExbB and ExbD in TonB function and stability. J Bacteriol  177: 4742–4747.  <ul class="find" data-citedArticleID="931190"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Characterization+of+the+exbBD+operon+of+Escherichia+coli+and+the+role+of+ExbB+and+ExbD+in+TonB+function+and+stability.&amp;auth=&amp;atitle=Characterization+of+the+exbBD+operon+of+Escherichia+coli+and+the+role+of+ExbB+and+ExbD+in+TonB+function+and+stability." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Characterization+of+the+exbBD+operon+of+Escherichia+coli+and+the+role+of+ExbB+and+ExbD+in+TonB+function+and+stability." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Characterization+of+the+exbBD+operon+of+Escherichia+coli+and+the+role+of+ExbB+and+ExbD+in+TonB+function+and+stability.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0000224-Neugebauer1" id="pone.0000224-Neugebauer1"></a>Neugebauer H, Herrmann C, Kammer W, Schwarz G, Nordheim A, et al.  (2005) ExbBD-dependent transport of maltodextrins through the novel MalA protein across the outer membrane of <em>Caulobacter crescentus</em>. J Bacteriol  187: 8300–8311.  <ul class="find" data-citedArticleID="931334" data-doi="10.1128/jb.187.24.8300-8311.2005"><li><a href="http://dx.doi.org/10.1128/jb.187.24.8300-8311.2005" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=ExbBD-dependent+transport+of+maltodextrins+through+the+novel+MalA+protein+across+the+outer+membrane+of+Caulobacter+crescentus." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22ExbBD-dependent+transport+of+maltodextrins+through+the+novel+MalA+protein+across+the+outer+membrane+of+Caulobacter+crescentus.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0000224-Chimento1" id="pone.0000224-Chimento1"></a>Chimento DP, Mohanty AK, Kadner RJ, Wiener MC (2003) Substrate-induced transmembrane signaling in the cobalamin transporter BtuB. Nat Struct Biol  10: 394–401.  <ul class="find" data-citedArticleID="931234" data-doi="10.1038/nsb914"><li><a href="http://dx.doi.org/10.1038/nsb914" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Substrate-induced+transmembrane+signaling+in+the+cobalamin+transporter+BtuB." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Substrate-induced+transmembrane+signaling+in+the+cobalamin+transporter+BtuB.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0000224-Ferguson1" id="pone.0000224-Ferguson1"></a>Ferguson AD, Chakraborty R, Smith BS, Esser L, van der Helm D, et al.  (2002) Structural basis of gating by the outer membrane transporter FecA. Science  295: 1715–1719.  <ul class="find" data-citedArticleID="931262" data-doi="10.1126/science.1067313"><li><a href="http://dx.doi.org/10.1126/science.1067313" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Structural+basis+of+gating+by+the+outer+membrane+transporter+FecA." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Structural+basis+of+gating+by+the+outer+membrane+transporter+FecA.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0000224-Yue1" id="pone.0000224-Yue1"></a>Yue WW, Grizot S, Buchanan SK (2003) Structural evidence for iron-free citrate and ferric citrate binding to the TonB-dependent outer membrane transporter FecA. J Mol Biol  332: 353–368.  <ul class="find" data-citedArticleID="931424" data-doi="10.1016/s0022-2836(03)00855-6"><li><a href="http://dx.doi.org/10.1016/s0022-2836(03)00855-6" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Structural+evidence+for+iron-free+citrate+and+ferric+citrate+binding+to+the+TonB-dependent+outer+membrane+transporter+FecA." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Structural+evidence+for+iron-free+citrate+and+ferric+citrate+binding+to+the+TonB-dependent+outer+membrane+transporter+FecA.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0000224-Pawelek1" id="pone.0000224-Pawelek1"></a>Pawelek PD, Croteau N, Ng-Thow-Hing C, Khursigara CM, Moiseeva N, et al.  (2006) Structure of TonB in complex with FhuA, <em>E. coli</em> outer membrane receptor. Science  312: 1399–1402.  <ul class="find" data-citedArticleID="931350" data-doi="10.1126/science.1128057"><li><a href="http://dx.doi.org/10.1126/science.1128057" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Structure+of+TonB+in+complex+with+FhuA%2C+E.+coli+outer+membrane+receptor." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Structure+of+TonB+in+complex+with+FhuA%2C+E.+coli+outer+membrane+receptor.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0000224-Shultis1" id="pone.0000224-Shultis1"></a>Shultis DD, Purdy MD, Banchs CN, Wiener MC (2006) Outer membrane active transport: structure of the BtuB:TonB complex. Science  312: 1396–1399.  <ul class="find" data-citedArticleID="931374" data-doi="10.1126/science.1127694"><li><a href="http://dx.doi.org/10.1126/science.1127694" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Outer+membrane+active+transport%3A+structure+of+the+BtuB%3ATonB+complex." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Outer+membrane+active+transport%3A+structure+of+the+BtuB%3ATonB+complex.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0000224-Bateman1" id="pone.0000224-Bateman1"></a>Bateman A, Coin L, Durbin R, Finn RD, Hollich V, et al.  (2004) The Pfam protein families database. Nucleic Acids Res  32: D138–141.  <ul class="find" data-citedArticleID="931210" data-doi="10.1093/nar/gkh121"><li><a href="http://dx.doi.org/10.1093/nar/gkh121" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+Pfam+protein+families+database." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+Pfam+protein+families+database.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0000224-Lundrigan1" id="pone.0000224-Lundrigan1"></a>Lundrigan MD, Kadner RJ (1986) Nucleotide sequence of the gene for the ferrienterochelin receptor FepA in <em>Escherichia coli</em>. Homology among outer membrane receptors that interact with TonB. J Biol Chem  261: 10797–10801.  <ul class="find" data-citedArticleID="931310"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Nucleotide+sequence+of+the+gene+for+the+ferrienterochelin+receptor+FepA+in+Escherichia+coli.+Homology+among+outer+membrane+receptors+that+interact+with+TonB.&amp;auth=&amp;atitle=Nucleotide+sequence+of+the+gene+for+the+ferrienterochelin+receptor+FepA+in+Escherichia+coli.+Homology+among+outer+membrane+receptors+that+interact+with+TonB." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Nucleotide+sequence+of+the+gene+for+the+ferrienterochelin+receptor+FepA+in+Escherichia+coli.+Homology+among+outer+membrane+receptors+that+interact+with+TonB." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Nucleotide+sequence+of+the+gene+for+the+ferrienterochelin+receptor+FepA+in+Escherichia+coli.+Homology+among+outer+membrane+receptors+that+interact+with+TonB.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0000224-Struyve1" id="pone.0000224-Struyve1"></a>Struyve M, Moons M, Tommassen J (1991) Carboxy-terminal phenylalanine is essential for the correct assembly of a bacterial outer membrane protein. J Mol Biol  218: 141–148.  <ul class="find" data-citedArticleID="931384" data-doi="10.1016/0022-2836(91)90880-f"><li><a href="http://dx.doi.org/10.1016/0022-2836(91)90880-f" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Carboxy-terminal+phenylalanine+is+essential+for+the+correct+assembly+of+a+bacterial+outer+membrane+protein." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Carboxy-terminal+phenylalanine+is+essential+for+the+correct+assembly+of+a+bacterial+outer+membrane+protein.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0000224-Koebnik1" id="pone.0000224-Koebnik1"></a>Koebnik R (1993) Structural organization of TonB-dependent receptors. Trends Microbiol  1: 201.  <ul class="find" data-citedArticleID="931294" data-doi="10.1016/0966-842x(93)90092-6"><li><a href="http://dx.doi.org/10.1016/0966-842x(93)90092-6" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Structural+organization+of+TonB-dependent+receptors." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Structural+organization+of+TonB-dependent+receptors.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0000224-Braun1" id="pone.0000224-Braun1"></a>Braun V (1997) Surface signaling: novel transcription initiation mechanism starting from the cell surface. Arch Microbiol  167: 325–331.  <ul class="find" data-citedArticleID="931220" data-doi="10.1007/s002030050451"><li><a href="http://dx.doi.org/10.1007/s002030050451" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Surface+signaling%3A+novel+transcription+initiation+mechanism+starting+from+the+cell+surface." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Surface+signaling%3A+novel+transcription+initiation+mechanism+starting+from+the+cell+surface.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0000224-Koebnik2" id="pone.0000224-Koebnik2"></a>Koebnik R (2005) TonB-dependent trans-envelope signalling: the exception or the rule? Trends Microbiol  13: 343–347.  <ul class="find" data-citedArticleID="931296" data-doi="10.1016/j.tim.2005.06.005"><li><a href="http://dx.doi.org/10.1016/j.tim.2005.06.005" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=TonB-dependent+trans-envelope+signalling%3A+the+exception+or+the+rule%3F" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22TonB-dependent+trans-envelope+signalling%3A+the+exception+or+the+rule%3F%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0000224-deLorenzo1" id="pone.0000224-deLorenzo1"></a>de Lorenzo V, Wee S, Herrero M, Neilands JB (1987) Operator sequences of the aerobactin operon of plasmid ColV-K30 binding the ferric uptake regulation (fur) repressor. J Bacteriol  169: 2624–2630.  <ul class="find" data-citedArticleID="931432"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Operator+sequences+of+the+aerobactin+operon+of+plasmid+ColV-K30+binding+the+ferric+uptake+regulation+%28fur%29+repressor.&amp;auth=&amp;atitle=Operator+sequences+of+the+aerobactin+operon+of+plasmid+ColV-K30+binding+the+ferric+uptake+regulation+%28fur%29+repressor." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Operator+sequences+of+the+aerobactin+operon+of+plasmid+ColV-K30+binding+the+ferric+uptake+regulation+%28fur%29+repressor." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Operator+sequences+of+the+aerobactin+operon+of+plasmid+ColV-K30+binding+the+ferric+uptake+regulation+%28fur%29+repressor.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0000224-deLorenzo2" id="pone.0000224-deLorenzo2"></a>de Lorenzo V, Giovannini F, Herrero M, Neilands JB (1988) Metal ion regulation of gene expression. Fur repressor-operator interaction at the promoter region of the aerobactin system of pColV-K30. J Mol Biol  203: 875–884.  <ul class="find" data-citedArticleID="931434"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Metal+ion+regulation+of+gene+expression.+Fur+repressor-operator+interaction+at+the+promoter+region+of+the+aerobactin+system+of+pColV-K30.&amp;auth=&amp;atitle=Metal+ion+regulation+of+gene+expression.+Fur+repressor-operator+interaction+at+the+promoter+region+of+the+aerobactin+system+of+pColV-K30." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Metal+ion+regulation+of+gene+expression.+Fur+repressor-operator+interaction+at+the+promoter+region+of+the+aerobactin+system+of+pColV-K30." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Metal+ion+regulation+of+gene+expression.+Fur+repressor-operator+interaction+at+the+promoter+region+of+the+aerobactin+system+of+pColV-K30.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">36.
              </span><a name="pone.0000224-Oke1" id="pone.0000224-Oke1"></a>Oke V, Long SR (1999) Bacterial genes induced within the nodule during the <em>Rhizobium</em>-legume symbiosis. Mol Microbiol  32: 837–849.  <ul class="find" data-citedArticleID="931344" data-doi="10.1046/j.1365-2958.1999.01402.x"><li><a href="http://dx.doi.org/10.1046/j.1365-2958.1999.01402.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Bacterial+genes+induced+within+the+nodule+during+the+Rhizobium-legume+symbiosis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Bacterial+genes+induced+within+the+nodule+during+the+Rhizobium-legume+symbiosis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">37.
              </span><a name="pone.0000224-Marx1" id="pone.0000224-Marx1"></a>Marx CJ, Lidstrom ME (2002) Broad-host-range cre-lox system for antibiotic marker recycling in gram-negative bacteria. Biotechniques  33: 1062–1067.  <ul class="find" data-citedArticleID="931314"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Broad-host-range+cre-lox+system+for+antibiotic+marker+recycling+in+gram-negative+bacteria.&amp;auth=&amp;atitle=Broad-host-range+cre-lox+system+for+antibiotic+marker+recycling+in+gram-negative+bacteria." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Broad-host-range+cre-lox+system+for+antibiotic+marker+recycling+in+gram-negative+bacteria." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Broad-host-range+cre-lox+system+for+antibiotic+marker+recycling+in+gram-negative+bacteria.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">38.
              </span><a name="pone.0000224-Angot1" id="pone.0000224-Angot1"></a>Angot A, Peeters N, Lechner E, Vailleau F, Baud C, et al.  (2006) <em>Ralstonia solanacearum</em> requires F-box-like domain-containing type III effectors to promote disease on several host plants. Proc Natl Acad Sci U S A  103: 14620–14625.  <ul class="find" data-citedArticleID="931196" data-doi="10.1073/pnas.0509393103"><li><a href="http://dx.doi.org/10.1073/pnas.0509393103" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Ralstonia+solanacearum+requires+F-box-like+domain-containing+type+III+effectors+to+promote+disease+on+several+host+plants." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Ralstonia+solanacearum+requires+F-box-like+domain-containing+type+III+effectors+to+promote+disease+on+several+host+plants.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">39.
              </span><a name="pone.0000224-Subramoni1" id="pone.0000224-Subramoni1"></a>Subramoni S, Sonti RV (2005) Growth deficiency of a <em>Xanthomonas oryzae</em> pv. <em>oryzae</em> fur mutant in rice leaves is rescued by ascorbic acid supplementation. Mol Plant Microbe Interact  18: 644–651.  <ul class="find" data-citedArticleID="931386" data-doi="10.1094/mpmi-18-0644"><li><a href="http://dx.doi.org/10.1094/mpmi-18-0644" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Growth+deficiency+of+a+Xanthomonas+oryzae+pv.+oryzae+fur+mutant+in+rice+leaves+is+rescued+by+ascorbic+acid+supplementation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Growth+deficiency+of+a+Xanthomonas+oryzae+pv.+oryzae+fur+mutant+in+rice+leaves+is+rescued+by+ascorbic+acid+supplementation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">40.
              </span><a name="pone.0000224-Loprasert1" id="pone.0000224-Loprasert1"></a>Loprasert S, Sallabhan R, Atichartpongkul S, Mongkolsuk S (1999) Characterization of a ferric uptake regulator (<em>fur</em>) gene from <em>Xanthomonas campestris</em> pv. <em>phaseoli</em> with unusual primary structure, genome organization, and expression patterns. Gene  239: 251–258.  <ul class="find" data-citedArticleID="931308" data-doi="10.1016/s0378-1119(99)00412-6"><li><a href="http://dx.doi.org/10.1016/s0378-1119(99)00412-6" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Characterization+of+a+ferric+uptake+regulator+%28fur%29+gene+from+Xanthomonas+campestris+pv.+phaseoli+with+unusual+primary+structure%2C+genome+organization%2C+and+expression+patterns." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Characterization+of+a+ferric+uptake+regulator+%28fur%29+gene+from+Xanthomonas+campestris+pv.+phaseoli+with+unusual+primary+structure%2C+genome+organization%2C+and+expression+patterns.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">41.
              </span><a name="pone.0000224-Thijs1" id="pone.0000224-Thijs1"></a>Thijs G, Lescot M, Marchal K, Rombauts S, De Moor B, et al.  (2001) A higher-order background model improves the detection of promoter regulatory elements by Gibbs sampling. Bioinformatics  17: 1113–1122.  <ul class="find" data-citedArticleID="931394" data-doi="10.1093/bioinformatics/17.12.1113"><li><a href="http://dx.doi.org/10.1093/bioinformatics/17.12.1113" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+higher-order+background+model+improves+the+detection+of+promoter+regulatory+elements+by+Gibbs+sampling." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+higher-order+background+model+improves+the+detection+of+promoter+regulatory+elements+by+Gibbs+sampling.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">42.
              </span><a name="pone.0000224-Thijs2" id="pone.0000224-Thijs2"></a>Thijs G, Marchal K, Lescot M, Rombauts S, De Moor B, et al.  (2002) A Gibbs sampling method to detect overrepresented motifs in the upstream regions of coexpressed genes. J Comput Biol  9: 447–464.  <ul class="find" data-citedArticleID="931396" data-doi="10.1089/10665270252935566"><li><a href="http://dx.doi.org/10.1089/10665270252935566" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+Gibbs+sampling+method+to+detect+overrepresented+motifs+in+the+upstream+regions+of+coexpressed+genes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+Gibbs+sampling+method+to+detect+overrepresented+motifs+in+the+upstream+regions+of+coexpressed+genes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">43.
              </span><a name="pone.0000224-Braun2" id="pone.0000224-Braun2"></a>Braun V, Mahren S (2005) Transmembrane transcriptional control (surface signalling) of the <em>Escherichia coli</em> Fec type. FEMS Microbiol Rev  29: 673–684.  <ul class="find" data-citedArticleID="931222" data-doi="10.1016/j.femsre.2004.10.001"><li><a href="http://dx.doi.org/10.1016/j.femsre.2004.10.001" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Transmembrane+transcriptional+control+%28surface+signalling%29+of+the+Escherichia+coli+Fec+type." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Transmembrane+transcriptional+control+%28surface+signalling%29+of+the+Escherichia+coli+Fec+type.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">44.
              </span><a name="pone.0000224-Escolar1" id="pone.0000224-Escolar1"></a>Escolar L, Perez-Martin J, de Lorenzo V (1998) Binding of the fur (ferric uptake regulator) repressor of Escherichia coli to arrays of the GATAAT sequence. J Mol Biol  283: 537–547.  <ul class="find" data-citedArticleID="931256" data-doi="10.1006/jmbi.1998.2119"><li><a href="http://dx.doi.org/10.1006/jmbi.1998.2119" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Binding+of+the+fur+%28ferric+uptake+regulator%29+repressor+of+Escherichia+coli+to+arrays+of+the+GATAAT+sequence." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Binding+of+the+fur+%28ferric+uptake+regulator%29+repressor+of+Escherichia+coli+to+arrays+of+the+GATAAT+sequence.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">45.
              </span><a name="pone.0000224-Wan1" id="pone.0000224-Wan1"></a>Wan XF, Verberkmoes NC, McCue LA, Stanek D, Connelly H, et al.  (2004) Transcriptomic and proteomic characterization of the Fur modulon in the metal-reducing bacterium <em>Shewanella oneidensis</em>. J Bacteriol  186: 8385–8400.  <ul class="find" data-citedArticleID="931408" data-doi="10.1128/jb.186.24.8385-8400.2004"><li><a href="http://dx.doi.org/10.1128/jb.186.24.8385-8400.2004" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Transcriptomic+and+proteomic+characterization+of+the+Fur+modulon+in+the+metal-reducing+bacterium+Shewanella+oneidensis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Transcriptomic+and+proteomic+characterization+of+the+Fur+modulon+in+the+metal-reducing+bacterium+Shewanella+oneidensis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">46.
              </span><a name="pone.0000224-Baichoo1" id="pone.0000224-Baichoo1"></a>Baichoo N, Helmann JD (2002) Recognition of DNA by Fur: a reinterpretation of the Fur box consensus sequence. J Bacteriol  184: 5826–5832.  <ul class="find" data-citedArticleID="931208" data-doi="10.1128/jb.184.21.5826-5832.2002"><li><a href="http://dx.doi.org/10.1128/jb.184.21.5826-5832.2002" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Recognition+of+DNA+by+Fur%3A+a+reinterpretation+of+the+Fur+box+consensus+sequence." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Recognition+of+DNA+by+Fur%3A+a+reinterpretation+of+the+Fur+box+consensus+sequence.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">47.
              </span><a name="pone.0000224-Dsouza1" id="pone.0000224-Dsouza1"></a>Dsouza M, Larsen N, Overbeek R (1997) Searching for patterns in genomic data. Trends Genet  13: 497–498.  <ul class="find" data-citedArticleID="931254" data-doi="10.1016/s0168-9525(97)01347-4"><li><a href="http://dx.doi.org/10.1016/s0168-9525(97)01347-4" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Searching+for+patterns+in+genomic+data." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Searching+for+patterns+in+genomic+data.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">48.
              </span><a name="pone.0000224-Cartron1" id="pone.0000224-Cartron1"></a>Cartron ML, Maddocks S, Gillingham P, Craven CJ, Andrews SC (2006) Feo–transport of ferrous iron into bacteria. Biometals  19: 143–157.  <ul class="find" data-citedArticleID="931230" data-doi="10.1007/s10534-006-0003-2"><li><a href="http://dx.doi.org/10.1007/s10534-006-0003-2" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Feo%26%23x2013%3Btransport+of+ferrous+iron+into+bacteria." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Feo%26%23x2013%3Btransport+of+ferrous+iron+into+bacteria.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">49.
              </span><a name="pone.0000224-Mey1" id="pone.0000224-Mey1"></a>Mey AR, Wyckoff EE, Kanukurthy V, Fisher CR, Payne SM (2005) Iron and fur regulation in <em>Vibrio cholerae</em> and the role of fur in virulence. Infect Immun  73: 8167–8178.  <ul class="find" data-citedArticleID="931320" data-doi="10.1128/iai.73.12.8167-8178.2005"><li><a href="http://dx.doi.org/10.1128/iai.73.12.8167-8178.2005" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Iron+and+fur+regulation+in+Vibrio+cholerae+and+the+role+of+fur+in+virulence." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Iron+and+fur+regulation+in+Vibrio+cholerae+and+the+role+of+fur+in+virulence.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">50.
              </span><a name="pone.0000224-Brown1" id="pone.0000224-Brown1"></a>Brown JS, Gilliland SM, Holden DW (2001) A <em>Streptococcus pneumoniae</em> pathogenicity island encoding an ABC transporter involved in iron uptake and virulence. Mol Microbiol  40: 572–585.  <ul class="find" data-citedArticleID="931226" data-doi="10.1046/j.1365-2958.2001.02414.x"><li><a href="http://dx.doi.org/10.1046/j.1365-2958.2001.02414.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+Streptococcus+pneumoniae+pathogenicity+island+encoding+an+ABC+transporter+involved+in+iron+uptake+and+virulence." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+Streptococcus+pneumoniae+pathogenicity+island+encoding+an+ABC+transporter+involved+in+iron+uptake+and+virulence.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">51.
              </span><a name="pone.0000224-Noel1" id="pone.0000224-Noel1"></a>Noel L, Thieme F, Nennstiel D, Bonas U (2001) cDNA-AFLP analysis unravels a genome-wide <em>hrpG</em>-regulon in the plant pathogen <em>Xanthomonas campestris</em> pv. <em>vesicatoria</em>. Mol Microbiol  41: 1271–1281.  <ul class="find" data-citedArticleID="931340" data-doi="10.1046/j.1365-2958.2001.02567.x"><li><a href="http://dx.doi.org/10.1046/j.1365-2958.2001.02567.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=cDNA-AFLP+analysis+unravels+a+genome-wide+hrpG-regulon+in+the+plant+pathogen+Xanthomonas+campestris+pv.+vesicatoria." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22cDNA-AFLP+analysis+unravels+a+genome-wide+hrpG-regulon+in+the+plant+pathogen+Xanthomonas+campestris+pv.+vesicatoria.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">52.
              </span><a name="pone.0000224-Koebnik3" id="pone.0000224-Koebnik3"></a>Koebnik R, Kruger A, Thieme F, Urban A, Bonas U (2006) Specific binding of the <em>Xanthomonas campestris</em> pv. <em>vesicatoria</em> AraC-type transcriptional activator HrpX to plant-inducible promoter boxes. J Bacteriol  188: 7652–7660.  <ul class="find" data-citedArticleID="931298" data-doi="10.1128/jb.00795-06"><li><a href="http://dx.doi.org/10.1128/jb.00795-06" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Specific+binding+of+the+Xanthomonas+campestris+pv.+vesicatoria+AraC-type+transcriptional+activator+HrpX+to+plant-inducible+promoter+boxes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Specific+binding+of+the+Xanthomonas+campestris+pv.+vesicatoria+AraC-type+transcriptional+activator+HrpX+to+plant-inducible+promoter+boxes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">53.
              </span><a name="pone.0000224-Tsuge1" id="pone.0000224-Tsuge1"></a>Tsuge S, Nakayama T, Terashima S, Ochiai H, Furutani A, et al.  (2006) Gene involved in transcriptional activation of the <em>hrp</em> regulatory gene <em>hrpG</em> in <em>Xanthomonas oryzae</em> pv. <em>oryzae</em>. J Bacteriol  188: 4158–4162.  <ul class="find" data-citedArticleID="931402" data-doi="10.1128/jb.00006-06"><li><a href="http://dx.doi.org/10.1128/jb.00006-06" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Gene+involved+in+transcriptional+activation+of+the+hrp+regulatory+gene+hrpG+in+Xanthomonas+oryzae+pv.+oryzae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Gene+involved+in+transcriptional+activation+of+the+hrp+regulatory+gene+hrpG+in+Xanthomonas+oryzae+pv.+oryzae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">54.
              </span><a name="pone.0000224-Kim1" id="pone.0000224-Kim1"></a>Kim HS, Park HJ, Heu S, Jung J (2004) Molecular and functional characterization of a unique sucrose hydrolase from <em>Xanthomonas axonopodis</em> pv. <em>glycines</em>. J Bacteriol  186: 411–418.  <ul class="find" data-citedArticleID="931292" data-doi="10.1128/jb.186.2.411-418.2004"><li><a href="http://dx.doi.org/10.1128/jb.186.2.411-418.2004" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Molecular+and+functional+characterization+of+a+unique+sucrose+hydrolase+from+Xanthomonas+axonopodis+pv.+glycines." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Molecular+and+functional+characterization+of+a+unique+sucrose+hydrolase+from+Xanthomonas+axonopodis+pv.+glycines.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">55.
              </span><a name="pone.0000224-DiGirolamo1" id="pone.0000224-DiGirolamo1"></a>Di Girolamo PM, Bradbeer C (1971) Transport of vitamin B 12 in <em>Escherichia coli</em>. J Bacteriol  106: 745–750.  <ul class="find" data-citedArticleID="931246"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Transport+of+vitamin+B+12+in+Escherichia+coli.&amp;auth=&amp;atitle=Transport+of+vitamin+B+12+in+Escherichia+coli." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Transport+of+vitamin+B+12+in+Escherichia+coli." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Transport+of+vitamin+B+12+in+Escherichia+coli.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">56.
              </span><a name="pone.0000224-Kadner1" id="pone.0000224-Kadner1"></a>Kadner RJ (1990) Vitamin B12 transport in <em>Escherichia coli</em>: energy coupling between membranes. Mol Microbiol  4: 2027–2033.  <ul class="find" data-citedArticleID="931290" data-doi="10.1111/j.1365-2958.1990.tb00562.x"><li><a href="http://dx.doi.org/10.1111/j.1365-2958.1990.tb00562.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Vitamin+B12+transport+in+Escherichia+coli%3A+energy+coupling+between+membranes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Vitamin+B12+transport+in+Escherichia+coli%3A+energy+coupling+between+membranes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">57.
              </span><a name="pone.0000224-Bradbeer1" id="pone.0000224-Bradbeer1"></a>Bradbeer C, Woodrow ML (1976) Transport of vitamin B12 in <em>Escherichia coli</em>: energy dependence. J Bacteriol  128: 99–104.  <ul class="find" data-citedArticleID="931218"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Transport+of+vitamin+B12+in+Escherichia+coli%3A+energy+dependence.&amp;auth=&amp;atitle=Transport+of+vitamin+B12+in+Escherichia+coli%3A+energy+dependence." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Transport+of+vitamin+B12+in+Escherichia+coli%3A+energy+dependence." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Transport+of+vitamin+B12+in+Escherichia+coli%3A+energy+dependence.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">58.
              </span><a name="pone.0000224-Paquelin1" id="pone.0000224-Paquelin1"></a>Paquelin A, Ghigo JM, Bertin S, Wandersman C (2001) Characterization of HasB, a <em>Serratia marcescens</em> TonB-like protein specifically involved in the haemophore-dependent haem acquisition system. Mol Microbiol  42: 995–1005.  <ul class="find" data-citedArticleID="931348" data-doi="10.1046/j.1365-2958.2001.02628.x"><li><a href="http://dx.doi.org/10.1046/j.1365-2958.2001.02628.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Characterization+of+HasB%2C+a+Serratia+marcescens+TonB-like+protein+specifically+involved+in+the+haemophore-dependent+haem+acquisition+system." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Characterization+of+HasB%2C+a+Serratia+marcescens+TonB-like+protein+specifically+involved+in+the+haemophore-dependent+haem+acquisition+system.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">59.
              </span><a name="pone.0000224-Seliger1" id="pone.0000224-Seliger1"></a>Seliger SS, Mey AR, Valle AM, Payne SM (2001) The two TonB systems of <em>Vibrio cholerae</em>: redundant and specific functions. Mol Microbiol  39: 801–812.  <ul class="find" data-citedArticleID="931370" data-doi="10.1046/j.1365-2958.2001.02273.x"><li><a href="http://dx.doi.org/10.1046/j.1365-2958.2001.02273.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+two+TonB+systems+of+Vibrio+cholerae%3A+redundant+and+specific+functions." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+two+TonB+systems+of+Vibrio+cholerae%3A+redundant+and+specific+functions.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">60.
              </span><a name="pone.0000224-Huang1" id="pone.0000224-Huang1"></a>Huang B, Ru K, Yuan Z, Whitchurch CB, Mattick JS (2004) tonB3 is required for normal twitching motility and extracellular assembly of type IV pili. J Bacteriol  186: 4387–4389.  <ul class="find" data-citedArticleID="931278" data-doi="10.1128/jb.186.13.4387-4389.2004"><li><a href="http://dx.doi.org/10.1128/jb.186.13.4387-4389.2004" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=tonB3+is+required+for+normal+twitching+motility+and+extracellular+assembly+of+type+IV+pili." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22tonB3+is+required+for+normal+twitching+motility+and+extracellular+assembly+of+type+IV+pili.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">61.
              </span><a name="pone.0000224-Zhao1" id="pone.0000224-Zhao1"></a>Zhao Q, Poole K (2002) Mutational analysis of the TonB1 energy coupler of <em>Pseudomonas aeruginosa</em>. J Bacteriol  184: 1503–1513.  <ul class="find" data-citedArticleID="931426" data-doi="10.1128/jb.184.6.1503-1513.2002"><li><a href="http://dx.doi.org/10.1128/jb.184.6.1503-1513.2002" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mutational+analysis+of+the+TonB1+energy+coupler+of+Pseudomonas+aeruginosa." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mutational+analysis+of+the+TonB1+energy+coupler+of+Pseudomonas+aeruginosa.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">62.
              </span><a name="pone.0000224-Weickert1" id="pone.0000224-Weickert1"></a>Weickert MJ, Adhya S (1992) A family of bacterial regulators homologous to Gal and Lac repressors. J Biol Chem  267: 15869–15874.  <ul class="find" data-citedArticleID="931410"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=A+family+of+bacterial+regulators+homologous+to+Gal+and+Lac+repressors.&amp;auth=&amp;atitle=A+family+of+bacterial+regulators+homologous+to+Gal+and+Lac+repressors." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+family+of+bacterial+regulators+homologous+to+Gal+and+Lac+repressors." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+family+of+bacterial+regulators+homologous+to+Gal+and+Lac+repressors.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">63.
              </span><a name="pone.0000224-Somers1" id="pone.0000224-Somers1"></a>Somers E, Keijers V, Ptacek D, Halvorsen Ottoy M, Srinivasan M, et al.  (2000) The salCAB operon of <em>Azospirillum irakense</em>, required for growth on salicin, is repressed by SalR, a transcriptional regulator that belongs to the Lacl/GalR family. Mol Gen Genet  263: 1038–1046.  <ul class="find" data-citedArticleID="931378" data-doi="10.1007/pl00008692"><li><a href="http://dx.doi.org/10.1007/pl00008692" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+salCAB+operon+of+Azospirillum+irakense%2C+required+for+growth+on+salicin%2C+is+repressed+by+SalR%2C+a+transcriptional+regulator+that+belongs+to+the+Lacl%2FGalR+family." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+salCAB+operon+of+Azospirillum+irakense%2C+required+for+growth+on+salicin%2C+is+repressed+by+SalR%2C+a+transcriptional+regulator+that+belongs+to+the+Lacl%2FGalR+family.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">64.
              </span><a name="pone.0000224-Thompson1" id="pone.0000224-Thompson1"></a>Thompson JD, Higgins DG, Gibson TJ (1994) CLUSTAL W: improving the sensitivity of progressive multiple sequence alignment through sequence weighting, position-specific gap penalties and weight matrix choice. Nucleic Acids Res  22: 4673–4680.  <ul class="find" data-citedArticleID="931398" data-doi="10.1093/nar/22.22.4673"><li><a href="http://dx.doi.org/10.1093/nar/22.22.4673" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=CLUSTAL+W%3A+improving+the+sensitivity+of+progressive+multiple+sequence+alignment+through+sequence+weighting%2C+position-specific+gap+penalties+and+weight+matrix+choice." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22CLUSTAL+W%3A+improving+the+sensitivity+of+progressive+multiple+sequence+alignment+through+sequence+weighting%2C+position-specific+gap+penalties+and+weight+matrix+choice.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">65.
              </span><a name="pone.0000224-Vauterin1" id="pone.0000224-Vauterin1"></a>Vauterin L, Hoste B, Kersters K, Swings J (1995) Reclassification of <em>Xanthomonas</em>. International Journal of Systematic Bacteriology  45: 472–489.  <ul class="find" data-citedArticleID="931406" data-doi="10.1099/00207713-45-3-472"><li><a href="http://dx.doi.org/10.1099/00207713-45-3-472" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Reclassification+of+Xanthomonas." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Reclassification+of+Xanthomonas.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">66.
              </span><a name="pone.0000224-Anzai1" id="pone.0000224-Anzai1"></a>Anzai Y, Kim H, Park JY, Wakabayashi H, Oyaizu H (2000) Phylogenetic affiliation of the pseudomonads based on 16S rRNA sequence. Int J Syst Evol Microbiol  50 Pt 4: 1563–1589.  <ul class="find" data-citedArticleID="931198" data-doi="10.1099/00207713-50-4-1563"><li><a href="http://dx.doi.org/10.1099/00207713-50-4-1563" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Phylogenetic+affiliation+of+the+pseudomonads+based+on+16S+rRNA+sequence." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Phylogenetic+affiliation+of+the+pseudomonads+based+on+16S+rRNA+sequence.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">67.
              </span><a name="pone.0000224-Cubero1" id="pone.0000224-Cubero1"></a>Cubero J, Graham JH (2004) The leucine-responsive regulatory protein (<em>lrp</em>) gene for characterization of the relationship among <em>Xanthomonas</em> species. Int J Syst Evol Microbiol  54: 429–437.  <ul class="find" data-citedArticleID="931240" data-doi="10.1099/ijs.0.02784-0"><li><a href="http://dx.doi.org/10.1099/ijs.0.02784-0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+leucine-responsive+regulatory+protein+%28lrp%29+gene+for+characterization+of+the+relationship+among+Xanthomonas+species." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+leucine-responsive+regulatory+protein+%28lrp%29+gene+for+characterization+of+the+relationship+among+Xanthomonas+species.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">68.
              </span><a name="pone.0000224-Tanabe1" id="pone.0000224-Tanabe1"></a>Tanabe T, Funahashi T, Nakao H, Miyoshi S, Shinoda S, et al.  (2003) Identification and characterization of genes required for biosynthesis and transport of the siderophore vibrioferrin in <em>Vibrio parahaemolyticus</em>. J Bacteriol  185: 6938–6949.  <ul class="find" data-citedArticleID="931388" data-doi="10.1128/jb.185.23.6938-6949.2003"><li><a href="http://dx.doi.org/10.1128/jb.185.23.6938-6949.2003" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Identification+and+characterization+of+genes+required+for+biosynthesis+and+transport+of+the+siderophore+vibrioferrin+in+Vibrio+parahaemolyticus." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Identification+and+characterization+of+genes+required+for+biosynthesis+and+transport+of+the+siderophore+vibrioferrin+in+Vibrio+parahaemolyticus.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">69.
              </span><a name="pone.0000224-Hottes1" id="pone.0000224-Hottes1"></a>Hottes AK, Meewan M, Yang D, Arana N, Romero P, et al.  (2004) Transcriptional profiling of <em>Caulobacter crescentus</em> during growth on complex and minimal media. J Bacteriol  186: 1448–1461.  <ul class="find" data-citedArticleID="931276" data-doi="10.1128/jb.186.5.1448-1461.2004"><li><a href="http://dx.doi.org/10.1128/jb.186.5.1448-1461.2004" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Transcriptional+profiling+of+Caulobacter+crescentus+during+growth+on+complex+and+minimal+media." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Transcriptional+profiling+of+Caulobacter+crescentus+during+growth+on+complex+and+minimal+media.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">70.
              </span><a name="pone.0000224-Reid1" id="pone.0000224-Reid1"></a>Reid SJ, Abratt VR (2005) Sucrose utilisation in bacteria: genetic organisation and regulation. Appl Microbiol Biotechnol  67: 312–321.  <ul class="find" data-citedArticleID="931360" data-doi="10.1007/s00253-004-1885-y"><li><a href="http://dx.doi.org/10.1007/s00253-004-1885-y" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Sucrose+utilisation+in+bacteria%3A+genetic+organisation+and+regulation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Sucrose+utilisation+in+bacteria%3A+genetic+organisation+and+regulation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">71.
              </span><a name="pone.0000224-Nikaido1" id="pone.0000224-Nikaido1"></a>Nikaido H (2003) Molecular basis of bacterial outer membrane permeability revisited. Microbiol Mol Biol Rev  67: 593–656.  <ul class="find" data-citedArticleID="931338" data-doi="10.1128/mmbr.67.4.593-656.2003"><li><a href="http://dx.doi.org/10.1128/mmbr.67.4.593-656.2003" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Molecular+basis+of+bacterial+outer+membrane+permeability+revisited." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Molecular+basis+of+bacterial+outer+membrane+permeability+revisited.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">72.
              </span><a name="pone.0000224-Llamas1" id="pone.0000224-Llamas1"></a>Llamas MA, Sparrius M, Kloet R, Jimenez CR, Vandenbroucke-Grauls C, et al.  (2006) The heterologous siderophores ferrioxamine B and ferrichrome activate signaling pathways in <em>Pseudomonas aeruginosa</em>. J Bacteriol  188: 1882–1891.  <ul class="find" data-citedArticleID="931306" data-doi="10.1128/jb.188.5.1882-1891.2006"><li><a href="http://dx.doi.org/10.1128/jb.188.5.1882-1891.2006" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+heterologous+siderophores+ferrioxamine+B+and+ferrichrome+activate+signaling+pathways+in+Pseudomonas+aeruginosa." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+heterologous+siderophores+ferrioxamine+B+and+ferrichrome+activate+signaling+pathways+in+Pseudomonas+aeruginosa.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">73.
              </span><a name="pone.0000224-Wiggerich1" id="pone.0000224-Wiggerich1"></a>Wiggerich HG, Klauke B, Koplin R, Priefer UB, Puhler A (1997) Unusual structure of the <em>tonB-exb</em> DNA region of <em>Xanthomonas campestris</em> pv. <em>campestris</em>: tonB, exbB, and exbD1 are essential for ferric iron uptake, but exbD2 is not. J Bacteriol  179: 7103–7110.  <ul class="find" data-citedArticleID="931412"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Unusual+structure+of+the+tonB-exb+DNA+region+of+Xanthomonas+campestris+pv.+campestris%3A+tonB%2C+exbB%2C+and+exbD1+are+essential+for+ferric+iron+uptake%2C+but+exbD2+is+not.&amp;auth=&amp;atitle=Unusual+structure+of+the+tonB-exb+DNA+region+of+Xanthomonas+campestris+pv.+campestris%3A+tonB%2C+exbB%2C+and+exbD1+are+essential+for+ferric+iron+uptake%2C+but+exbD2+is+not." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Unusual+structure+of+the+tonB-exb+DNA+region+of+Xanthomonas+campestris+pv.+campestris%3A+tonB%2C+exbB%2C+and+exbD1+are+essential+for+ferric+iron+uptake%2C+but+exbD2+is+not." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Unusual+structure+of+the+tonB-exb+DNA+region+of+Xanthomonas+campestris+pv.+campestris%3A+tonB%2C+exbB%2C+and+exbD1+are+essential+for+ferric+iron+uptake%2C+but+exbD2+is+not.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">74.
              </span><a name="pone.0000224-Wiggerich2" id="pone.0000224-Wiggerich2"></a>Wiggerich HG, Puhler A (2000) The <em>exbD2</em> gene as well as the iron-uptake genes <em>tonB, exbB</em> and <em>exbD1</em> of <em>Xanthomonas campestris</em> pv. <em>campestris</em> are essential for the induction of a hypersensitive response on pepper (<em>Capsicum annuum</em>). Microbiology 146 ( Pt  5): 1053–1060.  <ul class="find" data-citedArticleID="931414"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=The+exbD2+gene+as+well+as+the+iron-uptake+genes+tonB%2C+exbB+and+exbD1+of+Xanthomonas+campestris+pv.+campestris+are+essential+for+the+induction+of+a+hypersensitive+response+on+pepper+%28Capsicum+annuum%29.&amp;auth=&amp;atitle=The+exbD2+gene+as+well+as+the+iron-uptake+genes+tonB%2C+exbB+and+exbD1+of+Xanthomonas+campestris+pv.+campestris+are+essential+for+the+induction+of+a+hypersensitive+response+on+pepper+%28Capsicum+annuum%29." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+exbD2+gene+as+well+as+the+iron-uptake+genes+tonB%2C+exbB+and+exbD1+of+Xanthomonas+campestris+pv.+campestris+are+essential+for+the+induction+of+a+hypersensitive+response+on+pepper+%28Capsicum+annuum%29." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+exbD2+gene+as+well+as+the+iron-uptake+genes+tonB%2C+exbB+and+exbD1+of+Xanthomonas+campestris+pv.+campestris+are+essential+for+the+induction+of+a+hypersensitive+response+on+pepper+%28Capsicum+annuum%29.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">75.
              </span><a name="pone.0000224-Bogs1" id="pone.0000224-Bogs1"></a>Bogs J, Geider K (2000) Molecular analysis of sucrose metabolism of <em>Erwinia amylovora</em> and influence on bacterial virulence. J Bacteriol  182: 5351–5358.  <ul class="find" data-citedArticleID="931216" data-doi="10.1128/jb.182.19.5351-5358.2000"><li><a href="http://dx.doi.org/10.1128/jb.182.19.5351-5358.2000" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Molecular+analysis+of+sucrose+metabolism+of+Erwinia+amylovora+and+influence+on+bacterial+virulence." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Molecular+analysis+of+sucrose+metabolism+of+Erwinia+amylovora+and+influence+on+bacterial+virulence.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">76.
              </span><a name="pone.0000224-Jahreis1" id="pone.0000224-Jahreis1"></a>Jahreis K, Lengeler JW (1993) Molecular analysis of two ScrR repressors and of a ScrR-FruR hybrid repressor for sucrose and D-fructose specific regulons from enteric bacteria. Mol Microbiol  9: 195–209.  <ul class="find" data-citedArticleID="931284" data-doi="10.1111/j.1365-2958.1993.tb01681.x"><li><a href="http://dx.doi.org/10.1111/j.1365-2958.1993.tb01681.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Molecular+analysis+of+two+ScrR+repressors+and+of+a+ScrR-FruR+hybrid+repressor+for+sucrose+and+D-fructose+specific+regulons+from+enteric+bacteria." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Molecular+analysis+of+two+ScrR+repressors+and+of+a+ScrR-FruR+hybrid+repressor+for+sucrose+and+D-fructose+specific+regulons+from+enteric+bacteria.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">77.
              </span><a name="pone.0000224-Schmid1" id="pone.0000224-Schmid1"></a>Schmid K, Schupfner M, Schmitt R (1982) Plasmid-mediated uptake and metabolism of sucrose by <em>Escherichia coli</em> K-12. J Bacteriol  151: 68–76.  <ul class="find" data-citedArticleID="931364"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Plasmid-mediated+uptake+and+metabolism+of+sucrose+by+Escherichia+coli+K-12.&amp;auth=&amp;atitle=Plasmid-mediated+uptake+and+metabolism+of+sucrose+by+Escherichia+coli+K-12." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Plasmid-mediated+uptake+and+metabolism+of+sucrose+by+Escherichia+coli+K-12." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Plasmid-mediated+uptake+and+metabolism+of+sucrose+by+Escherichia+coli+K-12.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">78.
              </span><a name="pone.0000224-Sprenger1" id="pone.0000224-Sprenger1"></a>Sprenger GA, Lengeler JW (1988) Analysis of sucrose catabolism in <em>Klebsiella pneumoniae</em> and in Scr+ derivatives of <em>Escherichia coli</em> K12. J Gen Microbiol  134: 1635–1644.  <ul class="find" data-citedArticleID="931382" data-doi="10.1099/00221287-134-6-1635"><li><a href="http://dx.doi.org/10.1099/00221287-134-6-1635" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Analysis+of+sucrose+catabolism+in+Klebsiella+pneumoniae+and+in+Scr%2B+derivatives+of+Escherichia+coli+K12." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Analysis+of+sucrose+catabolism+in+Klebsiella+pneumoniae+and+in+Scr%2B+derivatives+of+Escherichia+coli+K12.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">79.
              </span><a name="pone.0000224-VanGelder1" id="pone.0000224-VanGelder1"></a>Van Gelder P, Dutzler R, Dumas F, Koebnik R, Schirmer T (2001) Sucrose transport through maltoporin mutants of <em>Escherichia coli</em>. Protein Eng  14: 943–948.  <ul class="find" data-citedArticleID="931404" data-doi="10.1093/protein/14.11.943"><li><a href="http://dx.doi.org/10.1093/protein/14.11.943" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Sucrose+transport+through+maltoporin+mutants+of+Escherichia+coli." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Sucrose+transport+through+maltoporin+mutants+of+Escherichia+coli.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">80.
              </span><a name="pone.0000224-Schulein1" id="pone.0000224-Schulein1"></a>Schulein K, Schmid K, Benzl R (1991) The sugar-specific outer membrane channel ScrY contains functional characteristics of general diffusion pores and substrate-specific porins. Mol Microbiol  5: 2233–2241.  <ul class="find" data-citedArticleID="931366" data-doi="10.1111/j.1365-2958.1991.tb02153.x"><li><a href="http://dx.doi.org/10.1111/j.1365-2958.1991.tb02153.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+sugar-specific+outer+membrane+channel+ScrY+contains+functional+characteristics+of+general+diffusion+pores+and+substrate-specific+porins." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+sugar-specific+outer+membrane+channel+ScrY+contains+functional+characteristics+of+general+diffusion+pores+and+substrate-specific+porins.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">81.
              </span><a name="pone.0000224-Puhler1" id="pone.0000224-Puhler1"></a>Puhler A, Arlat M, Becker A, Gottfert M, Morrissey JP, et al.  (2004) What can bacterial genome research teach us about bacteria-plant interactions? Curr Opin Plant Biol  7: 137–147.  <ul class="find" data-citedArticleID="931356" data-doi="10.1016/j.pbi.2004.01.009"><li><a href="http://dx.doi.org/10.1016/j.pbi.2004.01.009" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=What+can+bacterial+genome+research+teach+us+about+bacteria-plant+interactions%3F" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22What+can+bacterial+genome+research+teach+us+about+bacteria-plant+interactions%3F%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">82.
              </span><a name="pone.0000224-Lima1" id="pone.0000224-Lima1"></a>Lima WC, Van Sluys MA, Menck CF (2005) Non-gamma-proteobacteria gene islands contribute to the <em>Xanthomonas</em> genome. Omics  9: 160–172.  <ul class="find" data-citedArticleID="931304" data-doi="10.1089/omi.2005.9.160"><li><a href="http://dx.doi.org/10.1089/omi.2005.9.160" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Non-gamma-proteobacteria+gene+islands+contribute+to+the+Xanthomonas+genome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Non-gamma-proteobacteria+gene+islands+contribute+to+the+Xanthomonas+genome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">83.
              </span><a name="pone.0000224-Hashidoko1" id="pone.0000224-Hashidoko1"></a>Hashidoko Y (2005) Ecochemical studies of interrelationships between epiphytic bacteria and host plants via secondary metabolites. Biosci Biotechnol Biochem  69: 1427–1441.  <ul class="find" data-citedArticleID="931270" data-doi="10.1271/bbb.69.1427"><li><a href="http://dx.doi.org/10.1271/bbb.69.1427" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Ecochemical+studies+of+interrelationships+between+epiphytic+bacteria+and+host+plants+via+secondary+metabolites." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Ecochemical+studies+of+interrelationships+between+epiphytic+bacteria+and+host+plants+via+secondary+metabolites.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">84.
              </span><a name="pone.0000224-Hashimoto1" id="pone.0000224-Hashimoto1"></a>Hashimoto W, Momma K, Maruyama Y, Yamasaki M, Mikami B, et al.  (2005) Structure and function of bacterial super-biosystem responsible for import and depolymerization of macromolecules. Biosci Biotechnol Biochem  69: 673–692.  <ul class="find" data-citedArticleID="931272" data-doi="10.1271/bbb.69.673"><li><a href="http://dx.doi.org/10.1271/bbb.69.673" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Structure+and+function+of+bacterial+super-biosystem+responsible+for+import+and+depolymerization+of+macromolecules." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Structure+and+function+of+bacterial+super-biosystem+responsible+for+import+and+depolymerization+of+macromolecules.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">85.
              </span><a name="pone.0000224-Hashimoto2" id="pone.0000224-Hashimoto2"></a>Hashimoto W, He J, Wada Y, Nankai H, Mikami B, et al.  (2005) Proteomics-based identification of outer-membrane proteins responsible for import of macromolecules in <em>Sphingomonas</em> sp. A1: alginate-binding flagellin on the cell surface. Biochemistry  44: 13783–13794.  <ul class="find" data-citedArticleID="931274" data-doi="10.1021/bi050873b"><li><a href="http://dx.doi.org/10.1021/bi050873b" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Proteomics-based+identification+of+outer-membrane+proteins+responsible+for+import+of+macromolecules+in+Sphingomonas+sp.+A1%3A+alginate-binding+flagellin+on+the+cell+surface." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Proteomics-based+identification+of+outer-membrane+proteins+responsible+for+import+of+macromolecules+in+Sphingomonas+sp.+A1%3A+alginate-binding+flagellin+on+the+cell+surface.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">86.
              </span><a name="pone.0000224-Aso1" id="pone.0000224-Aso1"></a>Aso Y, Miyamoto Y, Harada KM, Momma K, Kawai S, et al.  (2006) Engineered membrane superchannel improves bioremediation potential of dioxin-degrading bacteria. Nat Biotechnol  24: 188–189.  <ul class="find" data-citedArticleID="931202" data-doi="10.1038/nbt1181"><li><a href="http://dx.doi.org/10.1038/nbt1181" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Engineered+membrane+superchannel+improves+bioremediation+potential+of+dioxin-degrading+bacteria." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Engineered+membrane+superchannel+improves+bioremediation+potential+of+dioxin-degrading+bacteria.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">87.
              </span><a name="pone.0000224-Nierman1" id="pone.0000224-Nierman1"></a>Nierman WC, Feldblyum TV, Laub MT, Paulsen IT, Nelson KE, et al.  (2001) Complete genome sequence of <em>Caulobacter crescentus</em>. Proc Natl Acad Sci U S A  98: 4136–4141.  <ul class="find" data-citedArticleID="931336" data-doi="10.1073/pnas.061029298"><li><a href="http://dx.doi.org/10.1073/pnas.061029298" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Complete+genome+sequence+of+Caulobacter+crescentus." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Complete+genome+sequence+of+Caulobacter+crescentus.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">88.
              </span><a name="pone.0000224-Phadke1" id="pone.0000224-Phadke1"></a>Phadke ND, Molloy MP, Steinhoff SA, Ulintz PJ, Andrews PC, et al.  (2001) Analysis of the outer membrane proteome of <em>Caulobacter crescentus</em> by two-dimensional electrophoresis and mass spectrometry. Proteomics  1: 705–720.  <ul class="find" data-citedArticleID="931352" data-doi="10.1002/1615-9861(200104)1:5<705::aid-prot705&gt;3.0.co;2-n"><li><a href="http://dx.doi.org/10.1002/1615-9861(200104)1:5<705::aid-prot705&gt;3.0.co;2-n" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Analysis+of+the+outer+membrane+proteome+of+Caulobacter+crescentus+by+two-dimensional+electrophoresis+and+mass+spectrometry." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Analysis+of+the+outer+membrane+proteome+of+Caulobacter+crescentus+by+two-dimensional+electrophoresis+and+mass+spectrometry.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">89.
              </span><a name="pone.0000224-Molloy1" id="pone.0000224-Molloy1"></a>Molloy MP, Phadke ND, Chen H, Tyldesley R, Garfin DE, et al.  (2002) Profiling the alkaline membrane proteome of <em>Caulobacter crescentus</em> with two-dimensional electrophoresis and mass spectrometry. Proteomics  2: 899–910.  <ul class="find" data-citedArticleID="931330" data-doi="10.1002/1615-9861(200207)2:7<899::aid-prot899&gt;3.0.co;2-y"><li><a href="http://dx.doi.org/10.1002/1615-9861(200207)2:7<899::aid-prot899&gt;3.0.co;2-y" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Profiling+the+alkaline+membrane+proteome+of+Caulobacter+crescentus+with+two-dimensional+electrophoresis+and+mass+spectrometry." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Profiling+the+alkaline+membrane+proteome+of+Caulobacter+crescentus+with+two-dimensional+electrophoresis+and+mass+spectrometry.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">90.
              </span><a name="pone.0000224-Ireland1" id="pone.0000224-Ireland1"></a>Ireland MM, Karty JA, Quardokus EM, Reilly JP, Brun YV (2002) Proteomic analysis of the <em>Caulobacter crescentus</em> stalk indicates competence for nutrient uptake. Mol Microbiol  45: 1029–1041.  <ul class="find" data-citedArticleID="931282" data-doi="10.1046/j.1365-2958.2002.03071.x"><li><a href="http://dx.doi.org/10.1046/j.1365-2958.2002.03071.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Proteomic+analysis+of+the+Caulobacter+crescentus+stalk+indicates+competence+for+nutrient+uptake." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Proteomic+analysis+of+the+Caulobacter+crescentus+stalk+indicates+competence+for+nutrient+uptake.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">91.
              </span><a name="pone.0000224-Wilson1" id="pone.0000224-Wilson1"></a>Wilson M, Lindow SE (1994) Coexistence among Epiphytic Bacterial Populations Mediated through Nutritional Resource Partitioning. Appl Environ Microbiol  60: 4468–4477.  <ul class="find" data-citedArticleID="931418"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Coexistence+among+Epiphytic+Bacterial+Populations+Mediated+through+Nutritional+Resource+Partitioning.&amp;auth=&amp;atitle=Coexistence+among+Epiphytic+Bacterial+Populations+Mediated+through+Nutritional+Resource+Partitioning." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Coexistence+among+Epiphytic+Bacterial+Populations+Mediated+through+Nutritional+Resource+Partitioning." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Coexistence+among+Epiphytic+Bacterial+Populations+Mediated+through+Nutritional+Resource+Partitioning.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">92.
              </span><a name="pone.0000224-Mercier1" id="pone.0000224-Mercier1"></a>Mercier J, Lindow SE (2000) Role of leaf surface sugars in colonization of plants by bacterial epiphytes. Appl Environ Microbiol  66: 369–374.  <ul class="find" data-citedArticleID="931318" data-doi="10.1128/aem.66.1.369-374.2000"><li><a href="http://dx.doi.org/10.1128/aem.66.1.369-374.2000" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Role+of+leaf+surface+sugars+in+colonization+of+plants+by+bacterial+epiphytes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Role+of+leaf+surface+sugars+in+colonization+of+plants+by+bacterial+epiphytes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">93.
              </span><a name="pone.0000224-Leveau1" id="pone.0000224-Leveau1"></a>Leveau JH, Lindow SE (2001) Appetite of an epiphyte: quantitative monitoring of bacterial sugar consumption in the phyllosphere. Proc Natl Acad Sci U S A  98: 3446–3453.  <ul class="find" data-citedArticleID="931302" data-doi="10.1073/pnas.061629598"><li><a href="http://dx.doi.org/10.1073/pnas.061629598" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Appetite+of+an+epiphyte%3A+quantitative+monitoring+of+bacterial+sugar+consumption+in+the+phyllosphere." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Appetite+of+an+epiphyte%3A+quantitative+monitoring+of+bacterial+sugar+consumption+in+the+phyllosphere.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">94.
              </span><a name="pone.0000224-Aldon1" id="pone.0000224-Aldon1"></a>Aldon D, Brito B, Boucher C, Genin S (2000) A bacterial sensor of plant cell contact controls the transcriptional induction of <em>Ralstonia solanacearum</em> pathogenicity genes. Embo J  19: 2304–2314.  <ul class="find" data-citedArticleID="931192" data-doi="10.1093/emboj/19.10.2304"><li><a href="http://dx.doi.org/10.1093/emboj/19.10.2304" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+bacterial+sensor+of+plant+cell+contact+controls+the+transcriptional+induction+of+Ralstonia+solanacearum+pathogenicity+genes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+bacterial+sensor+of+plant+cell+contact+controls+the+transcriptional+induction+of+Ralstonia+solanacearum+pathogenicity+genes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">95.
              </span><a name="pone.0000224-Marenda1" id="pone.0000224-Marenda1"></a>Marenda M, Brito B, Callard D, Genin S, Barberis P, et al.  (1998) PrhA controls a novel regulatory pathway required for the specific induction of <em>Ralstonia solanacearum hrp</em> genes in the presence of plant cells. Mol Microbiol  27: 437–453.  <ul class="find" data-citedArticleID="931312" data-doi="10.1046/j.1365-2958.1998.00692.x"><li><a href="http://dx.doi.org/10.1046/j.1365-2958.1998.00692.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=PrhA+controls+a+novel+regulatory+pathway+required+for+the+specific+induction+of+Ralstonia+solanacearum+hrp+genes+in+the+presence+of+plant+cells." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22PrhA+controls+a+novel+regulatory+pathway+required+for+the+specific+induction+of+Ralstonia+solanacearum+hrp+genes+in+the+presence+of+plant+cells.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">96.
              </span><a name="pone.0000224-Brito1" id="pone.0000224-Brito1"></a>Brito B, Aldon D, Barberis P, Boucher C, Genin S (2002) A signal transfer system through three compartments transduces the plant cell contact-dependent signal controlling <em>Ralstonia solanacearum hrp</em> genes. Mol Plant Microbe Interact  15: 109–119.  <ul class="find" data-citedArticleID="931224" data-doi="10.1094/mpmi.2002.15.2.109"><li><a href="http://dx.doi.org/10.1094/mpmi.2002.15.2.109" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+signal+transfer+system+through+three+compartments+transduces+the+plant+cell+contact-dependent+signal+controlling+Ralstonia+solanacearum+hrp+genes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+signal+transfer+system+through+three+compartments+transduces+the+plant+cell+contact-dependent+signal+controlling+Ralstonia+solanacearum+hrp+genes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">97.
              </span><a name="pone.0000224-Zou1" id="pone.0000224-Zou1"></a>Zou LF, Wang XP, Xiang Y, Zhang B, Li YR, et al.  (2006) Elucidation of the <em>hrp</em> clusters of <em>Xanthomonas oryzae</em> pv. <em>oryzicola</em> that control the hypersensitive response in nonhost tobacco and pathogenicity in susceptible host rice. Appl Environ Microbiol  72: 6212–6224.  <ul class="find" data-citedArticleID="931428" data-doi="10.1128/aem.00511-06"><li><a href="http://dx.doi.org/10.1128/aem.00511-06" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Elucidation+of+the+hrp+clusters+of+Xanthomonas+oryzae+pv.+oryzicola+that+control+the+hypersensitive+response+in+nonhost+tobacco+and+pathogenicity+in+susceptible+host+rice." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Elucidation+of+the+hrp+clusters+of+Xanthomonas+oryzae+pv.+oryzicola+that+control+the+hypersensitive+response+in+nonhost+tobacco+and+pathogenicity+in+susceptible+host+rice.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">98.
              </span><a name="pone.0000224-Faure1" id="pone.0000224-Faure1"></a>Faure D, Saier MH Jr, Vanderleyden J (2001) An evolutionary alternative system for aryl beta-glucosides assimilation in bacteria. J Mol Microbiol Biotechnol  3: 467–470.  <ul class="find" data-citedArticleID="931258"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=An+evolutionary+alternative+system+for+aryl+beta-glucosides+assimilation+in+bacteria.&amp;auth=&amp;atitle=An+evolutionary+alternative+system+for+aryl+beta-glucosides+assimilation+in+bacteria." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=An+evolutionary+alternative+system+for+aryl+beta-glucosides+assimilation+in+bacteria." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22An+evolutionary+alternative+system+for+aryl+beta-glucosides+assimilation+in+bacteria.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">99.
              </span><a name="pone.0000224-Bauer1" id="pone.0000224-Bauer1"></a>Bauer M, Kube M, Teeling H, Richter M, Lombardot T, et al.  (2006) Whole genome analysis of the marine Bacteroidetes ‘<em>Gramella forsetii</em>’ reveals adaptations to degradation of polymeric organic matter. Environ Microbiol  8: 2201–2213.  <ul class="find" data-citedArticleID="931212" data-doi="10.1111/j.1462-2920.2006.01152.x"><li><a href="http://dx.doi.org/10.1111/j.1462-2920.2006.01152.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Whole+genome+analysis+of+the+marine+Bacteroidetes+%26%23x2018%3BGramella+forsetii%26%23x2019%3B+reveals+adaptations+to+degradation+of+polymeric+organic+matter." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Whole+genome+analysis+of+the+marine+Bacteroidetes+%26%23x2018%3BGramella+forsetii%26%23x2019%3B+reveals+adaptations+to+degradation+of+polymeric+organic+matter.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">100.
              </span><a name="pone.0000224-Backhed1" id="pone.0000224-Backhed1"></a>Backhed F, Ley RE, Sonnenburg JL, Peterson DA, Gordon JI (2005) Host-bacterial mutualism in the human intestine. Science  307: 1915–1920.  <ul class="find" data-citedArticleID="931204" data-doi="10.1126/science.1104816"><li><a href="http://dx.doi.org/10.1126/science.1104816" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Host-bacterial+mutualism+in+the+human+intestine." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Host-bacterial+mutualism+in+the+human+intestine.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">101.
              </span><a name="pone.0000224-Sonnenburg1" id="pone.0000224-Sonnenburg1"></a>Sonnenburg JL, Xu J, Leip DD, Chen CH, Westover BP, et al.  (2005) Glycan foraging in vivo by an intestine-adapted bacterial symbiont. Science  307: 1955–1959.  <ul class="find" data-citedArticleID="931380" data-doi="10.1126/science.1109051"><li><a href="http://dx.doi.org/10.1126/science.1109051" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Glycan+foraging+in+vivo+by+an+intestine-adapted+bacterial+symbiont." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Glycan+foraging+in+vivo+by+an+intestine-adapted+bacterial+symbiont.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">102.
              </span><a name="pone.0000224-Xu1" id="pone.0000224-Xu1"></a>Xu J, Gordon JI (2003) Inaugural Article: Honor thy symbionts. Proc Natl Acad Sci U S A  100: 10452–10459.  <ul class="find" data-citedArticleID="931420" data-doi="10.1073/pnas.1734063100"><li><a href="http://dx.doi.org/10.1073/pnas.1734063100" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Inaugural+Article%3A+Honor+thy+symbionts." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Inaugural+Article%3A+Honor+thy+symbionts.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">103.
              </span><a name="pone.0000224-Shipman1" id="pone.0000224-Shipman1"></a>Shipman JA, Berleman JE, Salyers AA (2000) Characterization of four outer membrane proteins involved in binding starch to the cell surface of <em>Bacteroides thetaiotaomicron</em>. J Bacteriol  182: 5365–5372.  <ul class="find" data-citedArticleID="931372" data-doi="10.1128/jb.182.19.5365-5372.2000"><li><a href="http://dx.doi.org/10.1128/jb.182.19.5365-5372.2000" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Characterization+of+four+outer+membrane+proteins+involved+in+binding+starch+to+the+cell+surface+of+Bacteroides+thetaiotaomicron." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Characterization+of+four+outer+membrane+proteins+involved+in+binding+starch+to+the+cell+surface+of+Bacteroides+thetaiotaomicron.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">104.
              </span><a name="pone.0000224-Xu2" id="pone.0000224-Xu2"></a>Xu J, Bjursell MK, Himrod J, Deng S, Carmichael LK, et al.  (2003) A genomic view of the human-<em>Bacteroides thetaiotaomicron</em> symbiosis. Science  299: 2074–2076.  <ul class="find" data-citedArticleID="931422" data-doi="10.1126/science.1080029"><li><a href="http://dx.doi.org/10.1126/science.1080029" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+genomic+view+of+the+human-Bacteroides+thetaiotaomicron+symbiosis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+genomic+view+of+the+human-Bacteroides+thetaiotaomicron+symbiosis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">105.
              </span><a name="pone.0000224-Miller1" id="pone.0000224-Miller1"></a>Miller J (1972) Experiments in Molecular Genetics. Cold Spring Harbor, N.Y.: Cold Spring Harbor Laboratory Press.   <ul class="find-nolinks"></ul></li><li><span class="label">106.
              </span><a name="pone.0000224-Hantke1" id="pone.0000224-Hantke1"></a>Hantke K (1987) Selection procedure for deregulated iron transport mutants (fur) in <em>Escherichia coli</em> K 12: fur not only affects iron metabolism. Mol Gen Genet  210: 135–139.  <ul class="find" data-citedArticleID="931268" data-doi="10.1007/bf00337769"><li><a href="http://dx.doi.org/10.1007/bf00337769" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Selection+procedure+for+deregulated+iron+transport+mutants+%28fur%29+in+Escherichia+coli+K+12%3A+fur+not+only+affects+iron+metabolism." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Selection+procedure+for+deregulated+iron+transport+mutants+%28fur%29+in+Escherichia+coli+K+12%3A+fur+not+only+affects+iron+metabolism.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">107.
              </span><a name="pone.0000224-Schwyn1" id="pone.0000224-Schwyn1"></a>Schwyn B, Neilands JB (1987) Universal chemical assay for the detection and determination of siderophores. Anal Biochem  160: 47–56.  <ul class="find" data-citedArticleID="931368" data-doi="10.1016/0003-2697(87)90612-9"><li><a href="http://dx.doi.org/10.1016/0003-2697(87)90612-9" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Universal+chemical+assay+for+the+detection+and+determination+of+siderophores." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Universal+chemical+assay+for+the+detection+and+determination+of+siderophores.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">108.
              </span><a name="pone.0000224-Cunnac1" id="pone.0000224-Cunnac1"></a>Cunnac S, Occhialini A, Barberis P, Boucher C, Genin S (2004) Inventory and functional analysis of the large Hrp regulon in <em>Ralstonia solanacearum</em>: identification of novel effector proteins translocated to plant host cells through the type III secretion system. Mol Microbiol  53: 115–128.  <ul class="find" data-citedArticleID="931242" data-doi="10.1111/j.1365-2958.2004.04118.x"><li><a href="http://dx.doi.org/10.1111/j.1365-2958.2004.04118.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Inventory+and+functional+analysis+of+the+large+Hrp+regulon+in+Ralstonia+solanacearum%3A+identification+of+novel+effector+proteins+translocated+to+plant+host+cells+through+the+type+III+secretion+system." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Inventory+and+functional+analysis+of+the+large+Hrp+regulon+in+Ralstonia+solanacearum%3A+identification+of+novel+effector+proteins+translocated+to+plant+host+cells+through+the+type+III+secretion+system.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">109.
              </span><a name="pone.0000224-Huynh1" id="pone.0000224-Huynh1"></a>Huynh TV, Dahlbeck D, Staskawicz BJ (1989) Bacterial blight of soybean: regulation of a pathogen gene determining host cultivar specificity. Science  245: 1374–1377.  <ul class="find" data-citedArticleID="931280" data-doi="10.1126/science.2781284"><li><a href="http://dx.doi.org/10.1126/science.2781284" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Bacterial+blight+of+soybean%3A+regulation+of+a+pathogen+gene+determining+host+cultivar+specificity." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Bacterial+blight+of+soybean%3A+regulation+of+a+pathogen+gene+determining+host+cultivar+specificity.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">110.
              </span><a name="pone.0000224-Dombrecht1" id="pone.0000224-Dombrecht1"></a>Dombrecht B, Vanderleyden J, Michiels J (2001) Stable RK2-derived cloning vectors for the analysis of gene expression and gene function in gram-negative bacteria. Mol Plant Microbe Interact  14: 426–430.  <ul class="find" data-citedArticleID="931248" data-doi="10.1094/mpmi.2001.14.3.426"><li><a href="http://dx.doi.org/10.1094/mpmi.2001.14.3.426" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Stable+RK2-derived+cloning+vectors+for+the+analysis+of+gene+expression+and+gene+function+in+gram-negative+bacteria." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Stable+RK2-derived+cloning+vectors+for+the+analysis+of+gene+expression+and+gene+function+in+gram-negative+bacteria.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">111.
              </span><a name="pone.0000224-Miller2" id="pone.0000224-Miller2"></a>Miller J (1992) A Short Course in Bacterial Genetics. A Laboratory Manual and Handbook for Escherichia coliand Related Bacteria. Cold Spring Harbor, N.Y.: Cold Spring Harbor Laboratory Press.   <ul class="find-nolinks"></ul></li><li><span class="label">112.
              </span><a name="pone.0000224-Jefferson1" id="pone.0000224-Jefferson1"></a>Jefferson RA, Kavanagh TA, Bevan MW (1987) GUS fusions: beta-glucuronidase as a sensitive and versatile gene fusion marker in higher plants. Embo J  6: 3901–3907.  <ul class="find" data-citedArticleID="931286"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=GUS+fusions%3A+beta-glucuronidase+as+a+sensitive+and+versatile+gene+fusion+marker+in+higher+plants.&amp;auth=&amp;atitle=GUS+fusions%3A+beta-glucuronidase+as+a+sensitive+and+versatile+gene+fusion+marker+in+higher+plants." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=GUS+fusions%3A+beta-glucuronidase+as+a+sensitive+and+versatile+gene+fusion+marker+in+higher+plants." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22GUS+fusions%3A+beta-glucuronidase+as+a+sensitive+and+versatile+gene+fusion+marker+in+higher+plants.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">113.
              </span><a name="pone.0000224-Morales1" id="pone.0000224-Morales1"></a>Morales CQ, Posada J, Macneale E, Franklin D, Rivas I, et al.  (2005) Functional analysis of the early chlorosis factor gene. Mol Plant Microbe Interact  18: 477–486.  <ul class="find" data-citedArticleID="931332" data-doi="10.1094/mpmi-18-0477"><li><a href="http://dx.doi.org/10.1094/mpmi-18-0477" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Functional+analysis+of+the+early+chlorosis+factor+gene." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Functional+analysis+of+the+early+chlorosis+factor+gene.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">114.
              </span><a name="pone.0000224-Meyer2" id="pone.0000224-Meyer2"></a>Meyer D, Lauber E, Roby D, Arlat M, Kroj T (2005) Optimization of pathogenicity assays to study the <em>Arabidopsis thaliana</em>-<em>Xanthomonas campestris</em> pv. <em>campestris</em> pathosystem. Molecular Plant Pathology  6: 327–333.  <ul class="find" data-citedArticleID="931324" data-doi="10.1111/j.1364-3703.2005.00287.x"><li><a href="http://dx.doi.org/10.1111/j.1364-3703.2005.00287.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Optimization+of+pathogenicity+assays+to+study+the+Arabidopsis+thaliana-Xanthomonas+campestris+pv.+campestris+pathosystem." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Optimization+of+pathogenicity+assays+to+study+the+Arabidopsis+thaliana-Xanthomonas+campestris+pv.+campestris+pathosystem.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">115.
              </span><a name="pone.0000224-Bendtsen1" id="pone.0000224-Bendtsen1"></a>Bendtsen JD, Nielsen H, von Heijne G, Brunak S (2004) Improved prediction of signal peptides: SignalP 3.0. J Mol Biol  340: 783–795.  <ul class="find" data-citedArticleID="931214" data-doi="10.1016/j.jmb.2004.05.028"><li><a href="http://dx.doi.org/10.1016/j.jmb.2004.05.028" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Improved+prediction+of+signal+peptides%3A+SignalP+3.0." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Improved+prediction+of+signal+peptides%3A+SignalP+3.0.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">116.
              </span><a name="pone.0000224-Corpet1" id="pone.0000224-Corpet1"></a>Corpet F (1988) Multiple sequence alignment with hierarchical clustering. Nucleic Acids Res  16: 10881–10890.  <ul class="find" data-citedArticleID="931236" data-doi="10.1093/nar/16.22.10881"><li><a href="http://dx.doi.org/10.1093/nar/16.22.10881" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Multiple+sequence+alignment+with+hierarchical+clustering." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Multiple+sequence+alignment+with+hierarchical+clustering.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">117.
              </span><a name="pone.0000224-Jones1" id="pone.0000224-Jones1"></a>Jones DT (1999) Protein secondary structure prediction based on position-specific scoring matrices. J Mol Biol  292: 195–202.  <ul class="find" data-citedArticleID="931288" data-doi="10.1006/jmbi.1999.3091"><li><a href="http://dx.doi.org/10.1006/jmbi.1999.3091" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Protein+secondary+structure+prediction+based+on+position-specific+scoring+matrices." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Protein+secondary+structure+prediction+based+on+position-specific+scoring+matrices.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">118.
              </span><a name="pone.0000224-Bryson1" id="pone.0000224-Bryson1"></a>Bryson K, McGuffin LJ, Marsden RL, Ward JJ, Sodhi JS, et al.  (2005) Protein structure prediction servers at University College London. Nucleic Acids Res  33: W36–38.  <ul class="find" data-citedArticleID="931228" data-doi="10.1093/nar/gki410"><li><a href="http://dx.doi.org/10.1093/nar/gki410" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Protein+structure+prediction+servers+at+University+College+London." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Protein+structure+prediction+servers+at+University+College+London.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">119.
              </span><a name="pone.0000224-McGuffin1" id="pone.0000224-McGuffin1"></a>McGuffin LJ, Bryson K, Jones DT (2000) The PSIPRED protein structure prediction server. Bioinformatics  16: 404–405.  <ul class="find" data-citedArticleID="931316" data-doi="10.1093/bioinformatics/16.4.404"><li><a href="http://dx.doi.org/10.1093/bioinformatics/16.4.404" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+PSIPRED+protein+structure+prediction+server." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+PSIPRED+protein+structure+prediction+server.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">120.
              </span><a name="pone.0000224-Fenselau1" id="pone.0000224-Fenselau1"></a>Fenselau S, Bonas U (1995) Sequence and expression analysis of the <em>hrpB</em> pathogenicity operon of <em>Xanthomonas campestris</em> pv. <em>vesicatoria</em> which encodes eight proteins with similarity to components of the Hrp, Ysc, Spa, and Fli secretion systems. Mol Plant Microbe Interact  8: 845–854.  <ul class="find" data-citedArticleID="931260" data-doi="10.1094/mpmi-8-0845"><li><a href="http://dx.doi.org/10.1094/mpmi-8-0845" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Sequence+and+expression+analysis+of+the+hrpB+pathogenicity+operon+of+Xanthomonas+campestris+pv.+vesicatoria+which+encodes+eight+proteins+with+similarity+to+components+of+the+Hrp%2C+Ysc%2C+Spa%2C+and+Fli+secretion+systems." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Sequence+and+expression+analysis+of+the+hrpB+pathogenicity+operon+of+Xanthomonas+campestris+pv.+vesicatoria+which+encodes+eight+proteins+with+similarity+to+components+of+the+Hrp%2C+Ysc%2C+Spa%2C+and+Fli+secretion+systems.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">121.
              </span><a name="pone.0000224-Cunnac2" id="pone.0000224-Cunnac2"></a>Cunnac S, Boucher C, Genin S (2004) Characterization of the cis-acting regulatory element controlling HrpB-mediated activation of the type III secretion system and effector genes in <em>Ralstonia solanacearum</em>. J Bacteriol  186: 2309–2318.  <ul class="find" data-citedArticleID="931244" data-doi="10.1128/jb.186.8.2309-2318.2004"><li><a href="http://dx.doi.org/10.1128/jb.186.8.2309-2318.2004" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Characterization+of+the+cis-acting+regulatory+element+controlling+HrpB-mediated+activation+of+the+type+III+secretion+system+and+effector+genes+in+Ralstonia+solanacearum." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Characterization+of+the+cis-acting+regulatory+element+controlling+HrpB-mediated+activation+of+the+type+III+secretion+system+and+effector+genes+in+Ralstonia+solanacearum.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">122.
              </span><a name="pone.0000224-Thompson2" id="pone.0000224-Thompson2"></a>Thompson JD, Gibson TJ, Plewniak F, Jeanmougin F, Higgins DG (1997) The CLUSTAL_X windows interface: flexible strategies for multiple sequence alignment aided by quality analysis tools. Nucleic Acids Res  25: 4876–4882.  <ul class="find" data-citedArticleID="931400" data-doi="10.1093/nar/25.24.4876"><li><a href="http://dx.doi.org/10.1093/nar/25.24.4876" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+CLUSTAL_X+windows+interface%3A+flexible+strategies+for+multiple+sequence+alignment+aided+by+quality+analysis+tools." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+CLUSTAL_X+windows+interface%3A+flexible+strategies+for+multiple+sequence+alignment+aided+by+quality+analysis+tools.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">123.
              </span><a name="pone.0000224-Crooks1" id="pone.0000224-Crooks1"></a>Crooks GE, Hon G, Chandonia JM, Brenner SE (2004) WebLogo: a sequence logo generator. Genome Res  14: 1188–1190.  <ul class="find" data-citedArticleID="931238" data-doi="10.1101/gr.849004"><li><a href="http://dx.doi.org/10.1101/gr.849004" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=WebLogo%3A+a+sequence+logo+generator." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22WebLogo%3A+a+sequence+logo+generator.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000224.XML" value="265273"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000224.PDF" value="879649"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000224.g001.PNG_L" value="1979306"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000224.g001.PNG_M" value="191261"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000224.g001.PNG_S" value="5158"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000224.g001.TIF" value="2397866"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000224.g001.PNG_I" value="65301"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000224.g002.PNG_L" value="507076"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000224.g002.PNG_M" value="82267"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000224.g002.PNG_S" value="3206"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000224.g002.TIF" value="980370"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000224.g002.PNG_I" value="34245"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000224.g003.PNG_L" value="251810"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000224.g003.PNG_M" value="186467"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000224.g003.PNG_S" value="6022"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000224.g003.TIF" value="446054"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000224.g003.PNG_I" value="73354"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000224.t001.PNG_L" value="166337"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000224.t001.PNG_M" value="62976"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000224.t001.PNG_S" value="3057"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000224.t001.TIF" value="711604"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000224.t001.PNG_I" value="27341"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000224.g004.PNG_L" value="107861"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000224.g004.PNG_M" value="18855"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000224.g004.PNG_S" value="2363"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000224.g004.TIF" value="233304"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000224.g004.PNG_I" value="14358"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000224.g005.PNG_L" value="336952"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000224.g005.PNG_M" value="17626"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000224.g005.PNG_S" value="2250"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000224.g005.TIF" value="639894"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000224.g005.PNG_I" value="8771"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000224.g006.PNG_L" value="24483"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000224.g006.PNG_M" value="26636"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000224.g006.PNG_S" value="1880"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000224.g006.TIF" value="39328"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000224.g006.PNG_I" value="12280"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000224.g007.PNG_L" value="1579008"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000224.g007.PNG_M" value="114664"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000224.g007.PNG_S" value="3296"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000224.g007.TIF" value="1830842"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000224.g007.PNG_I" value="41307"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000224.g008.PNG_L" value="66530"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000224.g008.PNG_M" value="19616"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000224.g008.PNG_S" value="2076"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000224.g008.TIF" value="111166"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000224.g008.PNG_I" value="11364"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000224.t002.PNG_L" value="232983"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000224.t002.PNG_M" value="76564"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000224.t002.PNG_S" value="3494"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000224.t002.TIF" value="1276640"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000224.t002.PNG_I" value="31647"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000224.g009.PNG_L" value="26564"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000224.g009.PNG_M" value="18668"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000224.g009.PNG_S" value="2171"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000224.g009.TIF" value="53742"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000224.g009.PNG_I" value="16875"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000224.t003.PNG_L" value="63505"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000224.t003.PNG_M" value="64520"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000224.t003.PNG_S" value="2863"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000224.t003.TIF" value="303972"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000224.t003.PNG_I" value="29810"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000224.t004.PNG_L" value="74872"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000224.t004.PNG_M" value="77043"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000224.t004.PNG_S" value="3281"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000224.t004.TIF" value="350756"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000224.t004.PNG_I" value="34656"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000224.g010.PNG_L" value="233980"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000224.g010.PNG_M" value="88451"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000224.g010.PNG_S" value="4879"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000224.g010.TIF" value="480542"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000224.g010.PNG_I" value="56521"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000224.g011.PNG_L" value="339408"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000224.g011.PNG_M" value="47883"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000224.g011.PNG_S" value="2784"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000224.g011.TIF" value="505314"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000224.g011.PNG_I" value="19419"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000224.g012.PNG_L" value="795833"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000224.g012.PNG_M" value="287366"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000224.g012.PNG_S" value="8410"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000224.g012.TIF" value="1084962"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000224.g012.PNG_I" value="104327"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000224.s001.PDF" value="36333"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000224.s002.PDF" value="239574"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000224.s003.XLS" value="35328"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000224.s004.XLS" value="81920"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000224.s005.XLS" value="175104"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000224.s006.XLS" value="34816"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000224.s007.XLS" value="37888"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000224.s008.XLS" value="34304"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000224&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000224" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000224&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0000224&volume=&issue=&title=Plant Carbohydrate Scavenging through TonB-Dependent Receptors: A Feature Shared by Phytopathogenic and Aquatic Bacteria&author_name=Servane%20Blanvillain%2C%20Damien%20Meyer%2C%20Alice%20Boulanger%2C%20Martine%20Lautier%2C%20Catherine%20Guynet%2C%20Nicolas%20Denanc%C3%A9%2C%20Jacques%20Vasse%2C%20Emmanuelle%20Lauber%2C%20Matthieu%20Arlat&start_page=1&end_page=21" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000224" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000224" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0000224" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000224&amp;t=Plant%20Carbohydrate%20Scavenging%20through%20TonB-Dependent%20Receptors%3A%20A%20Feature%20Shared%20by%20Phytopathogenic%20and%20Aquatic%20Bacteria" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000224&title=Plant%20Carbohydrate%20Scavenging%20through%20TonB-Dependent%20Receptors%3A%20A%20Feature%20Shared%20by%20Phytopathogenic%20and%20Aquatic%20Bacteria&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000224&amp;title=Plant%20Carbohydrate%20Scavenging%20through%20TonB-Dependent%20Receptors%3A%20A%20Feature%20Shared%20by%20Phytopathogenic%20and%20Aquatic%20Bacteria" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000224" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0000224&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Plant Carbohydrate Scavenging through TonB-Dependent Receptors: A Feature Shared by Phytopathogenic and Aquatic Bacteria';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0000224';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Plant%20Carbohydrate%20Scavenging%20through%20TonB-Dependent%20Receptors%3A%20A%20Feature%20Shared%20by%20Phytopathogenic%20and%20Aquatic%20Bacteria http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000224" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0000224" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">














          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Bacterial+genomics%22" title="Search for articles in the subject area:'Bacterial genomics'"><div class="flagText">Bacterial genomics</div></a>
              <div data-categoryid="17227" data-articleid="23286"
                   data-categoryname="Bacterial genomics"
                   class="flagImage" title="Flag 'Bacterial genomics' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Carbohydrates%22" title="Search for articles in the subject area:'Carbohydrates'"><div class="flagText">Carbohydrates</div></a>
              <div data-categoryid="18141" data-articleid="23286"
                   data-categoryname="Carbohydrates"
                   class="flagImage" title="Flag 'Carbohydrates' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Caulobacter+crescentus%22" title="Search for articles in the subject area:'Caulobacter crescentus'"><div class="flagText">Caulobacter crescentus</div></a>
              <div data-categoryid="34361" data-articleid="23286"
                   data-categoryname="Caulobacter crescentus"
                   class="flagImage" title="Flag 'Caulobacter crescentus' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Genetic+loci%22" title="Search for articles in the subject area:'Genetic loci'"><div class="flagText">Genetic loci</div></a>
              <div data-categoryid="17713" data-articleid="23286"
                   data-categoryname="Genetic loci"
                   class="flagImage" title="Flag 'Genetic loci' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Sequence+motif+analysis%22" title="Search for articles in the subject area:'Sequence motif analysis'"><div class="flagText">Sequence motif analysis</div></a>
              <div data-categoryid="19657" data-articleid="23286"
                   data-categoryname="Sequence motif analysis"
                   class="flagImage" title="Flag 'Sequence motif analysis' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Sucrose%22" title="Search for articles in the subject area:'Sucrose'"><div class="flagText">Sucrose</div></a>
              <div data-categoryid="32241" data-articleid="23286"
                   data-categoryname="Sucrose"
                   class="flagImage" title="Flag 'Sucrose' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Xanthomonas%22" title="Search for articles in the subject area:'Xanthomonas'"><div class="flagText">Xanthomonas</div></a>
              <div data-categoryid="44729" data-articleid="23286"
                   data-categoryname="Xanthomonas"
                   class="flagImage" title="Flag 'Xanthomonas' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Xanthomonas+campestris%22" title="Search for articles in the subject area:'Xanthomonas campestris'"><div class="flagText">Xanthomonas campestris</div></a>
              <div data-categoryid="44093" data-articleid="23286"
                   data-categoryname="Xanthomonas campestris"
                   class="flagImage" title="Flag 'Xanthomonas campestris' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=696'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=4690'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=2182&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>

<div class="block sidebar-comments">
    <div class="header">
        <h3>Comments</h3>
    </div>
      <p><a href="/annotation/listThread.action?root=3625">Comment from the Academic Editor</a><br>Posted by redfield</p>
      <p><a href="/annotation/listThread.action?root=41963">Well done</a><br>Posted by Damien</p>
</div>

</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
