

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Alignment of Non-Covalent Interactions at Protein-Protein Interfaces</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0001926"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0001926" />

    <meta name="citation_title" content="Alignment of Non-Covalent Interactions at Protein-Protein Interfaces"/>
    <meta itemprop="name" content="Alignment of Non-Covalent Interactions at Protein-Protein Interfaces"/>

      <meta name="citation_author" content="Hongbo Zhu"/>
            <meta name="citation_author_institution" content="Max-Planck-Institut für Informatik, Saarbrücken, Germany"/>
      <meta name="citation_author" content="Ingolf Sommer"/>
            <meta name="citation_author_institution" content="Max-Planck-Institut für Informatik, Saarbrücken, Germany"/>
      <meta name="citation_author" content="Thomas Lengauer"/>
            <meta name="citation_author_institution" content="Max-Planck-Institut für Informatik, Saarbrücken, Germany"/>
      <meta name="citation_author" content="Francisco S. Domingues"/>
            <meta name="citation_author_institution" content="Max-Planck-Institut für Informatik, Saarbrücken, Germany"/>

    <meta name="citation_date" content="2008/4/2"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0001926.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e1926"/>
    <meta name="citation_issue" content="4"/>
    <meta name="citation_volume" content="3"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Principles of protein-protein interactions.; citation_author=S Jones; citation_author=J Thornton; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=93; citation_number=1; citation_pages=13-20; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=The atomic structure of protein-protein recognition sites.; citation_author=L Lo Conte; citation_author=C Chothia; citation_author=J Janin; citation_journal_title=J Mol Biol; citation_volume=285(5); citation_number=2; citation_pages=2177-98; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Electrostatic aspects of protein-protein interactions.; citation_author=FB Sheinerman; citation_author=R Norel; citation_author=B Honig; citation_journal_title=Curr Opin Struct Biol; citation_volume=10(2); citation_number=3; citation_pages=153-159; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Hydration of protein-protein interfaces.; citation_author=F Rodier; citation_author=RP Bahadur; citation_author=P Chakrabarti; citation_author=J Janin; citation_journal_title=Proteins; citation_volume=60; citation_number=4; citation_pages=36-45; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Diversity of protein-protein interactions.; citation_author=IMA Nooren; citation_author=JM Thornton; citation_journal_title=EMBO J; citation_volume=22(14); citation_number=5; citation_pages=3486-92; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=A dissection of specific and non-specific protein-protein interfaces.; citation_author=RP Bahadur; citation_author=P Chakrabarti; citation_author=F Rodier; citation_author=J Janin; citation_journal_title=J Mol Biol; citation_volume=336(4); citation_number=6; citation_pages=943-55; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Structure, function, and evolution of transient and obligate protein-protein interactions.; citation_author=J Mintseris; citation_author=Z Weng; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=102(31); citation_number=7; citation_pages=10930-10935; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=NOXclass: prediction of protein-protein interaction types.; citation_author=H Zhu; citation_author=F Domingues; citation_author=I Sommer; citation_author=T Lengauer; citation_journal_title=BMC Bioinformatics; citation_volume=7; citation_number=8; citation_pages=27; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=The Protein Data Bank.; citation_author=HM Berman; citation_author=J Westbrook; citation_author=Z Feng; citation_author=G Gilliland; citation_author=TN Bhat; citation_author=H Weissig; citation_author=IN Shindyalov; citation_author=PE Bourne; citation_journal_title=Nucleic Acids Res; citation_volume=28; citation_number=9; citation_pages=235-42; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=The relationship between sequence and interaction divergence in proteins.; citation_author=P Aloy; citation_author=H Ceulemans; citation_author=A Stark; citation_author=RB Russell; citation_journal_title=J Mol Biol; citation_volume=332(5); citation_number=10; citation_pages=989-98; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=The Many Faces of Protein-protein Interactions: A Compendium of Interface Geometry.; citation_author=WK Kim; citation_author=A Henschel; citation_author=C Winter; citation_author=M Schroeder; citation_journal_title=PLoS Comput Biol; citation_volume=2(9); citation_number=11; citation_pages=e124; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=SCOPPI: a structural classification of protein-protein interfaces.; citation_author=C Winter; citation_author=A Henschel; citation_author=WK Kim; citation_author=M Schroeder; citation_journal_title=Nucleic Acids Res; citation_volume=34(Database issue); citation_number=12; citation_pages=D310-D314; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Equivalent binding sites reveal convergently evolved interaction motifs.; citation_author=A Henschel; citation_author=WK Kim; citation_author=M Schroeder; citation_journal_title=Bioinformatics; citation_volume=22(5); citation_number=13; citation_pages=550-555; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=A dataset of protein-protein inter faces generated with a sequence order independent comparison technique.; citation_author=CJ Tsai; citation_author=SL Lin; citation_author=HJ Wolfson; citation_author=R Nussinov; citation_journal_title=J Mol Biol; citation_volume=260(4); citation_number=14; citation_pages=604-620; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=A new, structurally nonredundant, diverse data set of protein-protein interfaces and its implications.; citation_author=O Keskin; citation_author=CJ Tsai; citation_author=H Wolfson; citation_author=R Nussinov; citation_journal_title=Protein Sci; citation_volume=13(4); citation_number=15; citation_pages=1043-55; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Favorable scaffolds: proteins with different sequence, structure and function may associate in similar ways.; citation_author=O Keskin; citation_author=R Nussinov; citation_journal_title=Protein Eng Des Sel; citation_volume=18; citation_number=16; citation_pages=11-24; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Protein-Protein Interfaces: Recognition of Similar Spatial and Chemical Organizations.; citation_author=A Shulman-Peleg; citation_author=S Mintz; citation_author=R Nussinov; citation_author=HJ Wolfson; citation_number=17; citation_pages=194-205; citation_date=2004; citation_publisher=Springer; " />
      <meta name="citation_reference" content="citation_title=Generation and analysis of a protein-protein interface data set with similar chemical and spatial patterns of interactions.; citation_author=S Mintz; citation_author=A Shulman-Peleg; citation_author=HJ Wolfson; citation_author=R Nussinov; citation_journal_title=Proteins; citation_volume=61; citation_number=18; citation_pages=6-20; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=MAPPIS: Multiple 3D Alignment of Protein-Protein Interfaces.; citation_author=A Shulman-Peleg; citation_author=M Shatsky; citation_author=R Nussinov; citation_author=HJ Wolfson; citation_number=19; citation_pages=91-103; citation_date=2005; citation_publisher=Springer; " />
      <meta name="citation_reference" content="citation_title=A new method to detect related function among proteins independent of sequence and fold homology.; citation_author=S Schmitt; citation_author=D Kuhn; citation_author=G Klebe; citation_journal_title=J Mol Biol; citation_volume=323(2); citation_number=20; citation_pages=387-406; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Analysis of binding site similarity, small molecule similarity and experimental binding profiles in the human cytosolic sulfotransferase family.; citation_author=RJ Najmanovich; citation_author=A Allali Hassani; citation_author=RJ Morris; citation_author=L Dombrovsky; citation_author=PW Pan; citation_journal_title=Bioinformatics; citation_volume=23(2); citation_number=21; citation_pages=e104-e109; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Inferring Protein Function from Protein Structure.; citation_author=FS Domingues; citation_author=T Lengauer; citation_number=22; citation_pages=1211-1252; citation_date=2007; citation_publisher=Wiley VCH; " />
      <meta name="citation_reference" content="citation_title=Crystal structures of the RNA dependent RNA polymerase genotype 2a of hepatitis C virus reveal two conformations and suggest mechanisms of inhibition by non-nucleoside inhibitors.; citation_author=BK Biswal; citation_author=MM Cherney; citation_author=M Wang; citation_author=L Chan; citation_author=CG Yannopoulos; citation_journal_title=J Biol Chem; citation_volume=280(18); citation_number=23; citation_pages=18202-18210; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Structural interaction fingerprint (SIFt): a novel method for analyzing three-dimensional protein-ligand binding interactions.; citation_author=Z Deng; citation_author=C Chuaqui; citation_author=J Singh; citation_journal_title=J Med Chem; citation_volume=47(2); citation_number=24; citation_pages=337-344; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Using networks to identify fine structural differences between functionally distinct protein states.; citation_author=L Swint-Kruse; citation_journal_title=Biochemistry; citation_volume=43(34); citation_number=25; citation_pages=10886-10895; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Resmap: automated representation of macro molecular interfaces as two dimensional networks.; citation_author=L Swint-Kruse; citation_author=CS Brown; citation_journal_title=Bioinformatics; citation_volume=21(15); citation_number=26; citation_pages=3327-3328; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Similar binding sites and different partners: implications to shared proteins in cellular pathways.; citation_author=O Keskin; citation_author=R Nussinov; citation_journal_title=Structure; citation_volume=15(3); citation_number=27; citation_pages=341-354; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Asparagine and glutamine: using hydrogen atom contacts in the choice of side chain amide orientation.; citation_author=JM Word; citation_author=SC Lovell; citation_author=JS Richardson; citation_author=DC Richardson; citation_journal_title=J Mol Biol; citation_volume=285(4); citation_number=28; citation_pages=1735-1747; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Satisfying hydrogen bonding potential in proteins.; citation_author=IK McDonald; citation_author=JM Thornton; citation_journal_title=J Mol Biol; citation_volume=238(5); citation_number=29; citation_pages=777-793; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=Identification of tertiary structure resemblance in proteins using a maximal common subgraph isomorphism algorithm.; citation_author=HM Grindley; citation_author=PJ Artymiuk; citation_author=DW Rice; citation_author=P Willett; citation_journal_title=J Mol Biol; citation_volume=229(3); citation_number=30; citation_pages=707-721; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=An algorithm for finding maximal common subtopologies in a set of protein structures.; citation_author=I Koch; citation_author=T Lengauer; citation_author=E Wanke; citation_journal_title=J Comput Biol; citation_volume=3(2); citation_number=31; citation_pages=289-306; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=A note on the derivation of maximal common subgraphs of two directed or undirected graphs.; citation_author=G Levi; citation_journal_title=Calcolo; citation_volume=9(4); citation_number=32; citation_pages=341-352; citation_date=1972; " />
      <meta name="citation_reference" content="citation_title=Algorithm 457: finding all cliques of an undirected graph. Commun.; citation_author=C Bron; citation_author=J Kerbosch; citation_journal_title=ACM; citation_volume=16(9); citation_number=33; citation_pages=575-577; citation_date=1973; " />
      <meta name="citation_reference" content="citation_title=SCOP: a structural classification of proteins database for the investigation of sequences and structures.; citation_author=AG Murzin; citation_author=SE Brenner; citation_author=T Hubbard; citation_author=C Chothia; citation_journal_title=J Mol Biol; citation_volume=247(4); citation_number=34; citation_pages=536-40; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=DaliLite workbench for protein structure comparison.; citation_author=L Holm; citation_author=J Park; citation_journal_title=Bioinformatics; citation_volume=16(6); citation_number=35; citation_pages=566-567; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Hydrogen bonds and salt bridges across protein-protein interfaces.; citation_author=D Xu; citation_author=CJ Tsai; citation_author=R Nussinov; citation_journal_title=Protein Eng; citation_volume=10(9); citation_number=36; citation_pages=999-1012; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Derivation of 3D coordinate templates for searching structural databases: application to Ser-His-Asp catalytic triads in the serine proteinases and lipases.; citation_author=AC Wallace; citation_author=RA Laskowski; citation_author=JM Thornton; citation_journal_title=Protein Sci; citation_volume=5(6); citation_number=37; citation_pages=1001-1013; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Scorpion-toxin mimics of CD4 in complex with human immunodeficiency virus gp120 crystal structures, molecular mimicry, and neutralization breadth.; citation_author=C Huang; citation_author=F Stricher; citation_author=L Martin; citation_author=JM Decker; citation_author=S Majeed; citation_journal_title=Structure; citation_volume=13(5); citation_number=38; citation_pages=755-768; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Hot-spot mimicry of a cytokine receptor by a small molecule.; citation_author=CD Thanos; citation_author=WL DeLano; citation_author=JA Wells; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=103(42); citation_number=39; citation_pages=15422-15427; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Biochemistry.; citation_author=JM Berg; citation_author=JL Tymoczko; citation_author=L Stryer; citation_number=40; citation_date=2002; citation_publisher=W. H. Freeman and Company, fifth edition; " />
      <meta name="citation_reference" content="citation_title=The catalytic triad of serine peptidases.; citation_author=L Polgár; citation_journal_title=Cell Mol Life Sci; citation_volume=62(1920); citation_number=41; citation_pages=2161-2172; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Introduction to Protein Structure.; citation_author=C Branden; citation_author=J Tooze; citation_number=42; citation_pages=205; citation_date=1999; citation_publisher=Garland Publishing, Inc., 2nd edition; " />
      <meta name="citation_reference" content="citation_title=Potent small-molecule binding to a dynamic hot spot on IL-2.; citation_author=CD Thanos; citation_author=M Randal; citation_author=JA Wells; citation_journal_title=J Am Chem Soc; citation_volume=125(50); citation_number=43; citation_pages=15280-15281; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Anatomy of hot spots in protein interfaces.; citation_author=AA Bogan; citation_author=KS Thorn; citation_journal_title=J Mol Biol; citation_volume=280; citation_number=44; citation_pages=1-9; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Spatial chemical conservation of hot spot interactions in protein-protein complexes.; citation_author=A Shulman-Peleg; citation_author=M Shatsky; citation_author=R Nussinov; citation_author=HJ Wolfson; citation_journal_title=BMC Biol; citation_volume=5; citation_number=45; citation_pages=43; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=The PyMOL Molecular Graphics System.; citation_author=WL DeLano; citation_number=46; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001926" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Alignment of Non-Covalent Interactions at Protein-Protein Interfaces"/>
    <meta name="twitter:description" content="BackgroundThe study and comparison of protein-protein interfaces is essential for the understanding of the mechanisms of interaction between proteins. While there are many methods for comparing protein structures and protein binding sites, so far no methods have been reported for comparing the geometry of non-covalent interactions occurring at protein-protein interfaces.Methodology/Principal FindingsHere we present a method for aligning non-covalent interactions between different protein-protein interfaces. The method aligns the vector representations of van der Waals interactions and hydrogen bonds based on their geometry. The method has been applied to a dataset which comprises a variety of protein-protein interfaces. The alignments are consistent to a large extent with the results obtained using two other complementary approaches. In addition, we apply the method to three examples of protein mimicry. The method successfully aligns respective interfaces and allows for recognizing conserved interface regions.Conclusions/SignificanceThe Galinter method has been validated in the comparison of interfaces in which homologous subunits are involved, including cases of mimicry. The method is also applicable to comparing interfaces involving non-peptidic compounds. Galinter assists users in identifying local interface regions with similar patterns of non-covalent interactions. This is particularly relevant to the investigation of the molecular basis of interaction mimicry."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0001926.g003"/>

  <meta property="og:title" content="Alignment of Non-Covalent Interactions at Protein-Protein Interfaces" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001926" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=281'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=7399'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=509&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0001926">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001926" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001926&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Alignment of Non-Covalent Interactions at Protein-Protein Interfaces
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Hongbo Zhu
              <span class="corresponding">mail</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:hzhu@mpi-inf.mpg.de">hzhu@mpi-inf.mpg.de</a></p>

                <p>Affiliation:
                  Max-Planck-Institut für Informatik, Saarbrücken, Germany
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Ingolf Sommer, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Max-Planck-Institut für Informatik, Saarbrücken, Germany
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Thomas Lengauer, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Max-Planck-Institut für Informatik, Saarbrücken, Germany
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Francisco S. Domingues
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Max-Planck-Institut für Informatik, Saarbrücken, Germany
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: April 02, 2008</li>
    <li>DOI: 10.1371/journal.pone.0001926</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0001926" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0001926" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001926" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0001926" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0001926-g001" data-doi="info:doi/10.1371/journal.pone.0001926" data-uri="info:doi/10.1371/journal.pone.0001926.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001926.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001926-g002" data-doi="info:doi/10.1371/journal.pone.0001926" data-uri="info:doi/10.1371/journal.pone.0001926.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001926.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001926-g003" data-doi="info:doi/10.1371/journal.pone.0001926" data-uri="info:doi/10.1371/journal.pone.0001926.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001926.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001926">Reader Comments (1)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0001926" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2>
<h3>Background</h3>
<a id="article1.front1.article-meta1.abstract1.sec1.p1" name="article1.front1.article-meta1.abstract1.sec1.p1"></a><p>The study and comparison of protein-protein interfaces is essential for the understanding of the mechanisms of interaction between proteins. While there are many methods for comparing protein structures and protein binding sites, so far no methods have been reported for comparing the geometry of non-covalent interactions occurring at protein-protein interfaces.</p>


<h3>Methodology/Principal Findings</h3>
<a id="article1.front1.article-meta1.abstract1.sec2.p1" name="article1.front1.article-meta1.abstract1.sec2.p1"></a><p>Here we present a method for aligning non-covalent interactions between different protein-protein interfaces. The method aligns the vector representations of van der Waals interactions and hydrogen bonds based on their geometry. The method has been applied to a dataset which comprises a variety of protein-protein interfaces. The alignments are consistent to a large extent with the results obtained using two other complementary approaches. In addition, we apply the method to three examples of protein mimicry. The method successfully aligns respective interfaces and allows for recognizing conserved interface regions.</p>


<h3>Conclusions/Significance</h3>
<a id="article1.front1.article-meta1.abstract1.sec3.p1" name="article1.front1.article-meta1.abstract1.sec3.p1"></a><p>The Galinter method has been validated in the comparison of interfaces in which homologous subunits are involved, including cases of mimicry. The method is also applicable to comparing interfaces involving non-peptidic compounds. Galinter assists users in identifying local interface regions with similar patterns of non-covalent interactions. This is particularly relevant to the investigation of the molecular basis of interaction mimicry.</p>

</div>


<div class="articleinfo"><p><strong>Citation: </strong>Zhu H, Sommer I, Lengauer T, Domingues FS (2008) Alignment of Non-Covalent Interactions at Protein-Protein Interfaces. PLoS ONE 3(4):
          e1926.
            doi:10.1371/journal.pone.0001926</p><p><strong>Editor: </strong>Martin Egli, Vanderbilt University, United States of America</p><p><strong>Received:</strong> January 11, 2008; <strong>Accepted:</strong> February 28, 2008; <strong>Published:</strong> April 2, 2008</p><p><strong>Copyright:</strong> © 2008 Zhu et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>HZ is supported by the International Max Planck Research School for Computer Science (IMPRS-CS). FD is supported by the EU Network of Excellence BioSapiens (EU grant No. LSHG-CT-2003-503265).</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>Protein-protein interactions are involved in most cellular processes as many proteins carry out their functions by forming complexes. These protein complexes consist of interacting polypeptide chains (subunits). The interfaces in such complexes are composed of complementary binding sites from the respective subunits.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>The characterization of protein interfaces provides insights into protein interaction mechanisms. Such analysis is expected to have an impact on the prediction of interaction partners, as well as to assist in the design and engineering of protein interactions and interaction inhibitors. The physico-chemical properties of protein-protein interfaces have been previously investigated <a href="#pone.0001926-Jones1">[1]</a>–<a href="#pone.0001926-Rodier1">[4]</a>. Interactions between proteins have been classified according to different criteria; in a review, Nooren and Thornton use the criteria composition, affinity, and lifetime to classify interactions as homo or hetero, obligate or non-obligate, and permanent or transient, respectively <a href="#pone.0001926-Nooren1">[5]</a>. Methods have been developed for distinguishing different interaction types based on interface properties <a href="#pone.0001926-Bahadur1">[6]</a>–<a href="#pone.0001926-Zhu1">[8]</a>.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>Detailed comparison of protein-protein interfaces is fundamental for their better characterization and for structure-based classification of protein complexes. With an increasing amount of structural models for protein complexes available in the Protein Data Bank (PDB) <a href="#pone.0001926-Berman1">[9]</a>, protein complexes can now be compared systematically at the structural level. The structure similarity of protein complexes may be assessed at two levels: the similarity of the orientation of the binding sites relative to the folds of the subunits, and the local structure similarity of interfaces, as detailed in the next two paragraphs.</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p>In a comprehensive study, Aloy <em>et al.</em> have analyzed the relationship between protein sequence similarity and the spatial orientation of protein interaction <a href="#pone.0001926-Aloy1">[10]</a>. They discovered that among proteins with high sequence similarities the orientation of protein interaction tends to be conserved. Kim and colleagues have put forward a method for objectively comparing the orientations of interacting domains in two complexes <a href="#pone.0001926-Kim1">[11]</a>. They have divided protein domain-domain interfaces into different groups (face types), resulting in SCOPPI, a structural classification of protein-protein interfaces <a href="#pone.0001926-Winter1">[12]</a>. They have shown that similar protein domains may interact with distinct partners (non-homologous structures) using similar face types, but similar domains might also interact via different face types. Recently, using a similar method, Henschel <em>et al.</em> have identified cases of protein interaction mimicry, meaning that homologous subunits interact with non-homologous partners in the same relative orientation <a href="#pone.0001926-Henschel1">[13]</a>.</p>
<a id="article1.body1.sec1.p5" name="article1.body1.sec1.p5"></a><p>Local structure comparison of interfaces has been the focus of several other studies. Nussinov and colleagues have clustered all known protein-protein interfaces in the PDB by comparing the binding site C<sub>α</sub> atoms using a geometric hashing procedure <a href="#pone.0001926-Tsai1">[14]</a>, <a href="#pone.0001926-Keskin1">[15]</a>. Based on the analysis of the resulting clusters, they observed that proteins with different folds and functions may associate to yield interfaces of similar local structures <a href="#pone.0001926-Keskin2">[16]</a>. Shulman-Peleg <em>et al.</em> have developed I2I-SiteEngine and MAPPIS, programs that compare and align the functional groups at a pair or set of interacting binding sites using a geometric hashing algorithm <a href="#pone.0001926-ShulmanPeleg1">[17]</a>–<a href="#pone.0001926-ShulmanPeleg2">[19]</a>. Similar methods have been developed for comparing protein binding sites for small molecules <a href="#pone.0001926-Schmitt1">[20]</a>, <a href="#pone.0001926-Najmanovich1">[21]</a>, and they have been recently reviewed <a href="#pone.0001926-Domingues1">[22]</a>.</p>
<a id="article1.body1.sec1.p6" name="article1.body1.sec1.p6"></a><p>Protein complexes are stabilized by non-covalent interactions formed across interfaces (when we speak of non-covalent interaction we mean interactions between specific functional groups; when we speak of interaction, in general, we mean interactions between whole proteins composed of many non-covalent interactions). Non-covalent interactions at protein-protein or protein-ligand interfaces are often compared in order to characterize binding modes and to identify detailed structural differences. Biswal and colleagues have manually examined van der Waals (vdW) interactions and hydrogen bonds at two interfaces corresponding to a polymerase binding to two different inhibitors <a href="#pone.0001926-Biswal1">[23]</a>. Deng <em>et al.</em> have represented interactions at a protein-ligand interface as a one-dimensional fingerprint descriptor for studying different docking results on the same protein <a href="#pone.0001926-Deng1">[24]</a>. Swint-Kruse has compared the interfaces of dimeric LacI complexes in distinct functional states <a href="#pone.0001926-SwintKruse1">[25]</a>. The differences in fine structures of the interfaces have been identified by representing the set of non-covalent interactions as two-dimensional networks formed between interface residues <a href="#pone.0001926-SwintKruse2">[26]</a>. Recently, Keskin and Nussinov have shown that proteins may interact with variable partners via collections of structurally conserved non-covalent interactions <a href="#pone.0001926-Keskin3">[27]</a>. All of the above approaches require pre-computed sequence alignments or structure-based alignments of backbone atoms, and do not directly align the non-covalent interactions according to their conserved geometry.</p>
<a id="article1.body1.sec1.p7" name="article1.body1.sec1.p7"></a><p>Here, we present a novel method, Galinter, for aligning protein-protein interfaces. To our knowledge, this is the first method for explicitly comparing the geometry of non-covalent interactions at interfaces. The explicit comparison of non-covalent interactions provides an intuitive method of comparative analysis and visualization of binding modes, and for investigating the degree of conservation between interfaces. We have tested Galinter on a published dataset of interfaces, and have also applied the method to analyzing three medically relevant cases of protein mimicry.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Methods"></a><h3>Methods</h3>
<h4>Method workflow</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>In this study, two types of non-covalent interactions are considered: van der Waals interactions and hydrogen bonds. These non-covalent interactions are represented as vectors (NCIVs) connecting the centers of two interacting atoms. The goal of the method is to find the largest set of NCIVs that can be superposed (structurally aligned) in similar geometric orientations. Two NCIVs (each from one interface) are matched in the alignment if they represent the same type of non-covalent interactions, and have similar distances and relative orientations to the other matched NCIVs within the respective interfaces. A graph-based method is applied for aligning NCIVs. The complete procedure is implemented in Galinter (<span class="underline">G</span>raph-based <span class="underline">al</span>ignment of protein-protein <span class="underline">inter</span>faces). The workflow of the method is composed of the following five steps. <a href="#pone-0001926-g001">Figure 1</a> provides a schematic overview.</p>
<div class="figure" id="pone-0001926-g001"><div class="img"><a name="pone-0001926-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001926.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001926" data-uri="info:doi/10.1371/journal.pone.0001926.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001926.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001926.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001926.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001926.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001926.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001926.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001926.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001926.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001926.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Flow chart of Galinter.</span></strong></p><a id="article1.body1.sec2.sec1.fig1.caption1.p1" name="article1.body1.sec2.sec1.fig1.caption1.p1"></a><p>(NCIV: non-covalent interaction vector; CVec: contact vector)</p>
<span>doi:10.1371/journal.pone.0001926.g001</span></div><h5>Identifying NCIVs.</h5><a id="article1.body1.sec2.sec1.sec1.p1" name="article1.body1.sec2.sec1.sec1.p1"></a><p>For two protein complexes with known structures, two types of NCIVs between the interacting proteins are distinguished. Contact vectors (CVecs) are detected based on a distance criterion and represent van der Waals interactions. A CVec connects two heavy atoms if the distance between them is less than the sum of their respective van der Waals radii plus 1.0 Å. The user specifies one of the two binding sites as the head site and the other as the tail site. All CVecs point from the tail to the head site. Hydrogen bond vectors (HVecs) are the second type of NCIV. These are determined by adding hydrogen atoms to the protein structures with the REDUCE program <a href="#pone.0001926-Word1">[28]</a> and by then applying a set of geometric criteria <a href="#pone.0001926-McDonald1">[29, see Figure S1 in the supplement]</a>. The directions of the HVecs encode the hydrogen bonding donor-acceptor direction. The distance between a pair of NCIVs is defined as the Euclidean distance between the middle points of the head and tail points of the two vectors, respectively.</p>

<h5>Clustering NCIVs.</h5><a id="article1.body1.sec2.sec1.sec2.p1" name="article1.body1.sec2.sec1.sec2.p1"></a><p>In this step, two CVecs are grouped into the same cluster if they are closer than 2.0 Å and if the angle between their orientations is at most 45°. Subsequently, a consensus vector is computed and then used as representative for each cluster. A complete linkage hierarchical clustering algorithm is employed to cluster the NCIVs. HVecs are not clustered and are directly taken as representatives. The distance between representatives is defined in the same way as the distance between NCIVs.</p>
<a id="article1.body1.sec2.sec1.sec2.p2" name="article1.body1.sec2.sec1.sec2.p2"></a><p>This clustering step is based on the observation that often there are small groups (size of 2–4) of CVecs with similar orientations (angle difference at most 45°). Clustering NCIVs also reduces the size of the alignment problem and enables Galinter to obtain results in reasonable run time (within minutes).</p>

<h5>Generating a graph representation for protein-protein interfaces.</h5><a id="article1.body1.sec2.sec1.sec3.p1" name="article1.body1.sec2.sec1.sec3.p1"></a><p>In this step, each protein-protein interface is modeled as an undirected node-labeled edge-labeled graph <em>G</em>(<em>V</em>,<em>E</em>). Node set <em>V</em> consists of all the NCIV representatives obtained in the previous step. Each node is labeled as either a CVec representative, or a HVec representative. Two nodes <em>u</em>, <em>v</em> are connected by an edge if the distance between the corresponding NCIVs is in the range from 2.0 to 40.0 Å. Each edge is labeled with a 5-tuple <em>EdgeLabel</em>. In every <em>EdgeLabel</em>, the first value is the distance between the corresponding NCIVs, and the other four values are the distances between each pair of endpoints of these two NCIVs. We have chosen 2.0 Å as lower bound because in the previous clustering step the cluster radius is also 2.0 Å. The upper bound of 40.0 Å excludes less than 5% of the NCIVs, since more than 95% of the distances between all CVecs in a structurally non-redundant dataset <a href="#pone.0001926-Keskin1">[15]</a> are at most 40.0 Å (data not shown).</p>

<h5>Aligning representatives.</h5><a id="article1.body1.sec2.sec1.sec4.p1" name="article1.body1.sec2.sec1.sec4.p1"></a><p>Given two graphs <em>G</em><sub>1</sub>(<em>V</em><sub>1</sub>,<em>E</em><sub>1</sub>) and <em>G</em><sub>2</sub>(<em>V</em><sub>2</sub>,<em>E</em><sub>2</sub>) representing two protein-protein interfaces, the goal is to find all maximum common subgraphs <em>H</em><sub>1</sub> and <em>H</em><sub>2</sub> such that i) <em>H</em><sub>1</sub>⊆<em>G</em><sub>1</sub>, <em>H</em><sub>2</sub>⊆<em>G</em><sub>2</sub>, <em>H</em><sub>1</sub> and <em>H</em><sub>2</sub> are isomorphic <em>H</em><sub>1</sub>≡<em>H</em><sub>2</sub>, and ii) there is no pair (<em>H</em><sub>1</sub>′, <em>H</em><sub>2</sub>′) and such that <em>H</em><sub>1</sub>⊆<em>H</em><sub>1</sub>′⊆<em>G</em><sub>1</sub>, <em>H</em><sub>2</sub>⊆<em>H</em><sub>2</sub>′⊆<em>G</em><sub>2</sub>, <em>H</em><sub>1</sub>′≡<em>H</em><sub>2</sub>′, and <em>H</em><sub>1</sub>′, <em>H</em><sub>2</sub>′ have more nodes than <em>H</em><sub>1</sub> and <em>H</em><sub>2</sub>, respectively.</p>
<a id="article1.body1.sec2.sec1.sec4.p2" name="article1.body1.sec2.sec1.sec4.p2"></a><p>The maximum common subgraph problem is transformed to the maximum clique problem in the traditional fashion <a href="#pone.0001926-Grindley1">[30]</a>, <a href="#pone.0001926-Koch1">[31]</a>. Maximal common subgraphs in <em>G</em><sub>1</sub> and <em>G</em><sub>2</sub> are identified by searching for maximal cliques in a product graph of <em>G</em><sub>1</sub> and <em>G</em><sub>2</sub> <a href="#pone.0001926-Koch1">[31]</a>, <a href="#pone.0001926-Levi1">[32]</a>. The product graph <em>P</em>(<em>V<sub>P</sub></em>,<em>E<sub>P</sub></em>) has a node set <em>V<sub>P</sub></em> = {(<em>u</em><sub>1</sub>,<em>u</em><sub>2</sub>) | <em>V</em><sub>1</sub>×<em>V</em><sub>2</sub> and <em>label</em>(<em>u</em><sub>1</sub>) = <em>label</em>(<em>u</em><sub>2</sub>)}. In <em>P</em>, two nodes (<em>u</em><sub>1</sub>,<em>u</em><sub>2</sub>) and (<em>v</em><sub>1</sub>,<em>v</em><sub>2</sub>) are connected if and only if (<em>u</em><sub>1</sub>,<em>u</em><sub>2</sub>) and (<em>v</em><sub>1</sub>,<em>v</em><sub>2</sub>) are different, <em>u</em><sub>1</sub>,<em>v</em><sub>1</sub> are connected in <em>G</em><sub>1</sub> and <em>u</em><sub>2</sub>,<em>v</em><sub>2</sub> are connected in <em>G</em><sub>2</sub> and for each <em>i</em>Є(1,…,5):<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001926.e001&amp;representation=PNG" class="inline-graphic"></span><br>where <em>TOL<sub>rep</sub></em> is a tolerance function defined as:<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001926.e002&amp;representation=PNG" class="inline-graphic"></span><br>The function enforces an upper limit on the difference of two distances, which has been derived from the analysis of a set of protein-protein interfaces (unpublished).</p>
<a id="article1.body1.sec2.sec1.sec4.p3" name="article1.body1.sec2.sec1.sec4.p3"></a><p>After obtaining the product graph, maximal cliques are detected <a href="#pone.0001926-Bron1">[33]</a>. The cliques in the product graph correspond to aligned representatives. Only the largest alignments of representatives are consider in the following step.</p>

<h5>Extending aligned representatives to NCIVs.</h5><a id="article1.body1.sec2.sec1.sec5.p1" name="article1.body1.sec2.sec1.sec5.p1"></a><p>Up to this stage, the alignment consists of aligned representatives of NCIV clusters. In this step these aligned representatives are used as “anchors” for deriving the alignment between the original sets of NCIVs.</p>
<a id="article1.body1.sec2.sec1.sec5.p2" name="article1.body1.sec2.sec1.sec5.p2"></a><p>First, in an <em>expanding</em> procedure, two NCIVs are matched if i) they are of the same type, ii) they have similar orientations (the angle between them is at most 45°) after the transformation based on the superposition of the anchors, and iii) they have similar distances to the anchors. A tolerance function for distance difference is defined as:<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001926.e003&amp;representation=PNG" class="inline-graphic"></span><br>where <em>a</em> and <em>b</em> are the distances to be compared. <em>TOL<sub>vec</sub></em> is more restrictive than <em>TOL<sub>rep</sub></em>, as it is applied to actual NCIVs instead of representatives.</p>
<a id="article1.body1.sec2.sec1.sec5.p3" name="article1.body1.sec2.sec1.sec5.p3"></a><p>After finding all the potential alignments of NCIVs, a <em>filtering</em> procedure is performed. A pair of aligned NCIVs found in the <em>expanding</em> procedure is discarded if the difference of their distances to any other pair of aligned NCIVs exceeds the tolerance defined in <em>TOL<sub>vec</sub></em>.</p>
<a id="article1.body1.sec2.sec1.sec5.p4" name="article1.body1.sec2.sec1.sec5.p4"></a><p>The resulting matched NCIVs replace the aligned representatives as new anchors, and the <em>expanding</em> and <em>filtering</em> procedures are repeated. Newly found matches of NCIVs are added to the anchors, until no more NCIVs can be matched in the <em>expanding</em> procedure. All resulting alignments of NCIVs are sorted according to alignment size (number of matched NCIVs). Only the largest alignments are reported. Alignments with a size below 90% of the largest one are discarded.</p>

<h5>Availability.</h5><a id="article1.body1.sec2.sec1.sec6.p1" name="article1.body1.sec2.sec1.sec6.p1"></a><p>The source code of Galinter is available upon request from the authors.</p>



<h4>Comparison of alignments</h4>
<h5>Pilot Dataset.</h5><a id="article1.body1.sec2.sec2.sec1.p1" name="article1.body1.sec2.sec2.sec1.p1"></a><p>We have applied Galinter to the pilot dataset which was used for testing I2I-SiteEngine <a href="#pone.0001926-ShulmanPeleg1">[17]</a>. This dataset consists of 64 protein-protein interfaces clustered into 22 groups according to I2I-SiteEngine alignment results (see <a href="#pone.0001926.s002">Figure S2</a> in the supplement). It is composed of a variety of protein complexes, including antigen-antibody, protease-inhibitor, protein-peptide, and protein-protein dimers. There are both homo- and hetero-dimers in the dataset. We excluded eight singleton groups from the dataset. This analysis is restricted to the remaining 14 non-singleton groups.</p>
<a id="article1.body1.sec2.sec2.sec1.p2" name="article1.body1.sec2.sec2.sec1.p2"></a><p>For any pair of complexes to be compared, if at least one subunit of one complex is homologous to at least one subunit of the other complex, then the two complexes are labeled as S/D-homologous (single- or double-sided homologous). Otherwise the two complexes are labeled as non-homologous. Two subunit structures are considered to be homologous if they belong to the same superfamily in SCOP <a href="#pone.0001926-Murzin1">[34]</a>. In nine of the 14 groups, all complexes are S/D-homologous to each other within the group. The remaining groups also contain some complexes not related by homology. See <a href="#pone.0001926.s003">Figure S3</a>, <a href="#pone.0001926.s004">S4</a>, <a href="#pone.0001926.s005">S5</a> in the supplement for more details.</p>

<h5>Comparing Galinter to I2I-SiteEngine and DaliLite.</h5><a id="article1.body1.sec2.sec2.sec2.p1" name="article1.body1.sec2.sec2.sec2.p1"></a><p>On the pilot dataset, Galinter alignments were compared to the alignments generated by the I2I-SiteEngine interface comparison method. I2I-SiteEngine matches chemical functional groups and associated residues at the binding sites of different interfaces. In addition, we compared the results of both Galinter and I2I-SiteEngine to alignments based on backbone structure, generated with DaliLite <a href="#pone.0001926-Holm1">[35]</a>. Using DaliLite, subunit structures are compared individually at both sides of interfaces. A subsequent alignment of interface residues can be derived based on the most significant DaliLite alignment of subunit structures as detailed in <a href="#pone.0001926.s006">Figure S6</a> in the supplement.</p>

<h5>Assessing the agreement of the results.</h5><a id="article1.body1.sec2.sec2.sec3.p1" name="article1.body1.sec2.sec2.sec3.p1"></a><p>In this work, we define interface residues as those which contain at least one interface atom, where interface atoms are the atoms involved in interface NCIVs. We compared the alignment of interfaces from the different methods (Galinter, I2I-SiteEngine, and DaliLite) by examining the deviation of C<sub>α</sub> atom coordinates of interface residues after corresponding transformations. Given two interface residue sets <em>I<sub>1</sub></em> and <em>I<sub>2</sub></em> and two alignment methods <em>M<sub>a</sub></em> and <em>M<sub>b</sub></em>, let <em>I<sub>2a</sub></em> correspond to the transformed set <em>I<sub>2</sub></em> according to the optimal superposition based on the alignment from method <em>M<sub>a</sub></em>. Analogously, <em>I<sub>2</sub></em> is transformed to <em>I<sub>2b</sub></em> based on the alignment from method <em>M<sub>b</sub></em>. Then, the root-mean-square deviation (RMSD) for all C<sub>α</sub> atoms of interface residues in <em>I<sub>2a</sub></em> and <em>I<sub>2b</sub></em> is calculated to assess the agreement between the two methods <em>M<sub>a</sub></em> and <em>M<sub>b</sub></em>. This measure is defined as irRMSD (interface residue RMSD). See <a href="#pone.0001926.s007">Figure S7</a> in the supplement for an illustration of the calculation of irRMSD.</p>


</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Results"></a><h3>Results</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p>To assess whether Galinter produces valid interface alignments, we compared the results of Galinter to the alignments generated by other approaches. One of these approaches aligns functional chemical groups at interfaces (I2I-SiteEngine) and the other approach aligns backbone structures (DaliLite).</p>
<a id="article1.body1.sec3.p2" name="article1.body1.sec3.p2"></a><p>In the second part of this section, we present the application of Galinter to three mimicry cases, for which the interfaces have been manually compared before.</p>

<h4>Application results on the pilot dataset</h4>
<h5>Comparison between Galinter, I2I-SiteEngine, and DaliLite.</h5><a id="article1.body1.sec3.sec1.sec1.p1" name="article1.body1.sec3.sec1.sec1.p1"></a><p>We have applied Galinter to every pair of interfaces within each of the 14 groups from the pilot dataset. There are 240 comparisons in total. The mean run time is 138.5 seconds (median run time 71.5 seconds) on a normal desktop (3.0 GHz CPU, 1GB memory) for these comparisons. The alignment results are compared to those of I2I-SiteEngine and DaliLite. The extent of agreement is measured using irRMSD values as described in section “<strong><em>Assessing the agreement of the results</em></strong>”.</p>
<a id="article1.body1.sec3.sec1.sec1.p2" name="article1.body1.sec3.sec1.sec1.p2"></a><p>I2I-SiteEngine compares interfaces by aligning the functional groups at binding sites, instead of aligning molecular interactions within the interface like Galinter. Galinter and I2I-SiteEngine can be regarded as complementary approaches as they use different properties to compare interfaces.</p>
<a id="article1.body1.sec3.sec1.sec1.p3" name="article1.body1.sec3.sec1.sec1.p3"></a><p>Backbone structure comparison methods like DaliLite can be used to generate interface alignments indirectly. These alignments are indirect in the sense that they do not take the structural similarities of the interfaces into account explicitly. When the interaction orientations of subunits are conserved between S/D-homologous complexes, these indirect alignments provide a coarse way of validating alignments from direct methods like Galinter and I2I-SiteEngine. The alignments based on backbone structures are expected to agree with explicit alignments of non-covalent interactions within the interfaces to some extent but not necessarily to match them.</p>
<a id="article1.body1.sec3.sec1.sec1.p4" name="article1.body1.sec3.sec1.sec1.p4"></a><p><a href="#pone-0001926-g002">Figure 2</a> provides a summary of the irRMSD values obtained in the analysis. All pairwise comparisons of interfaces are separated into two groups according to whether the corresponding complexes are S/D-homologous or non-homologous. Of the 240 pairs of interfaces compared, 114 are S/D-homologous and the remaining 126 pairs are non-homologous. For the alignments of non-homologous interfaces, only irRMSD values for the comparison between Galinter and I2I-SiteEngine are shown, because most non-homologous interfaces cannot be aligned using DaliLite as there is no backbone structure similarity between the respective protein complexes.</p>
<div class="figure" id="pone-0001926-g002"><div class="img"><a name="pone-0001926-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001926.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001926" data-uri="info:doi/10.1371/journal.pone.0001926.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001926.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001926.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001926.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001926.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001926.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001926.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001926.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001926.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001926.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Overview of irRMSD values for pairwise comparison of protein-protein interfaces.</span></strong></p><a id="article1.body1.sec3.sec1.sec1.fig1.caption1.p1" name="article1.body1.sec3.sec1.sec1.fig1.caption1.p1"></a><p>Most interfaces for non-homologous complexes cannot be compared using backbone alignment method. Thus for the alignments of non-homologous complex interfaces, only an overview of irRMSD values for the comparison between Galinter and I2I-SiteEngine are shown. (Hm: S/D-homologous; NonHm: non-homologous; Gal: Galinter; I2I: I2I-SiteEngine; Dal: DaliLite)</p>
<span>doi:10.1371/journal.pone.0001926.g002</span></div><a id="article1.body1.sec3.sec1.sec1.p5" name="article1.body1.sec3.sec1.sec1.p5"></a><p><a href="#pone-0001926-g002">Figure 2</a> shows that for S/D-homologous interfaces, Galinter alignments usually agree with I2I-SiteEngine alignments. The alignments are similar (irRMSD≤2 Å) for 66% of the cases. Galinter and I2I-SiteEngine both produce similar alignments to DaliLite if the interfaces are S/D-homologous. But the agreement between Galinter and DaliLite is higher, in general, than that between I2I-SiteEngine and DaliLite. For non-homologous interfaces, Galinter and I2I-SiteEngine generate very different alignments. Less than 40% of the 126 comparisons have irRMSD values below 8 Å. The supplement contains the results for each comparison (<a href="#pone.0001926.s003">Figure S3</a>, <a href="#pone.0001926.s004">S4</a>, <a href="#pone.0001926.s005">S5</a>).</p>
<a id="article1.body1.sec3.sec1.sec1.p6" name="article1.body1.sec3.sec1.sec1.p6"></a><p>We have explored possible causes for the disagreements between the alignments of different methods. For non-homologous interfaces, most of the disagreements are observed in groups 19 and 5. Group 19 consists of coiled-coil interfaces. More than a single solution is expected for the alignment of these repetitive structures. Therefore it is not surprising that the alignments from different methods disagree. In general, the alignments of both methods result in reasonable superimposition of the helix backbones. Nevertheless, visual inspection reveals that for some of these pairs one of the methods generates better superposition of the interacting helices. Galinter produces better superposition in five pairs (1ic2CD <em>vs.</em> 1gl2BC, 1ic2CD <em>vs.</em> 1gk4AB, 1gl2AB <em>vs.</em> 1gk4AB, 1gl2BC <em>vs.</em> 1gk4AB, 1gk4AB <em>vs.</em> 1if3AB), and I2I-SiteEngine in three cases (1ic2CD <em>vs.</em> 1if3AB, 1gl2AB <em>vs.</em> 1if3AB, 1gl2BC <em>vs.</em> 1if3AB). For example, in the comparison of 1gl2AB and 1gk4AB, chain B of 1gl2 has 16 helix turns and they are all superposed based on the Galinter alignment, while only 8 helix turns are superposed based on the I2I-SiteEngine alignment.</p>
<a id="article1.body1.sec3.sec1.sec1.p7" name="article1.body1.sec3.sec1.sec1.p7"></a><p>In group 5, there are relatively few similarities between the subunits from different complexes. There seems to be no obvious alignment solution in terms of either structure or evolution. The only evident common feature in these interfaces is that they include two interacting β-strands. The assessment of the results in this group is thus challenging. Bearing this in mind, we have investigated the quality of the results by visual inspection of the superposition of the two strands at the interfaces. We have found that for 15 pairs Galinter provides better superposition of the interface β-strands, and for five pairs I2I-SiteEngines leads to better superposition of these strands.</p>
<a id="article1.body1.sec3.sec1.sec1.p8" name="article1.body1.sec3.sec1.sec1.p8"></a><p>The disagreements between Galinter and I2I-SiteEngine for S/D-homologous interfaces arise mainly from group 10, and also to a lesser extent, from the smaller group 4. Interestingly, for these two groups, the Galinter alignments agree with those based on DaliLite.</p>
<a id="article1.body1.sec3.sec1.sec1.p9" name="article1.body1.sec3.sec1.sec1.p9"></a><p>In general, the three methods agree to a large extent, especially when the interfaces are related by homology. Nevertheless, it is not surprising to observe disagreements in the non-homologous groups, considering both that Galinter and I2I-SiteEngine are based on different interface properties and that there are no unique solutions in these groups.</p>

<h5>Contribution of different types of non-covalent interactions to the alignment.</h5><a id="article1.body1.sec3.sec1.sec2.p1" name="article1.body1.sec3.sec1.sec2.p1"></a><p>The current implementation of Galinter aligns vdW interactions and hydrogen bonds at interfaces. However, there are other types of non-covalent atomic interactions, especially electrostatic interactions between positively and negatively charged atoms. Thus we have explored the contribution of short-range electrostatic interactions to the alignment of protein-protein interfaces. Using a definition by Xu <em>et al.</em> <a href="#pone.0001926-Xu1">[36]</a>, we have identified fewer than three short-range electrostatic interactions on average for each of the 64 interfaces in the pilot dataset used in the manuscript. This is only 1% of the number of vdW interactions. In addition, we have re-ranked the alignment results by assigning the larger weight of 3 to short-range electrostatic interactions (versus a weight of 1 to vdW interactions and hydrogen bonds). Except for four cases (1okvBE vs. 1okuBF, 10gsAB vs. 1axdAB, 1axdAB vs. 10gsAB, 1g0uOP vs. 1iruFG), the top-ranking alignments for the pilot dataset remain the same. Even for these four cases, the new results exhibit considerable similarity to the original alignments (half or more of the aligned NCIVs are the same).</p>
<a id="article1.body1.sec3.sec1.sec2.p2" name="article1.body1.sec3.sec1.sec2.p2"></a><p>These results indicate that the current method seems to be robust with respect to different weighting of the various types of interactions. Nevertheless, a thorough investigation is required on how to weight different types of non-covalent interactions for interface alignment, which will be the focus of future work.</p>



<h4>Analysis of mimicry cases</h4>
<a id="article1.body1.sec3.sec2.p1" name="article1.body1.sec3.sec2.p1"></a><p>Protein mimicry is relevant in the design of protein inhibitors. These inhibitors are frequently designed such that their binding mode is similar to that of a wild-type protein-protein interaction. Their development process is expected to benefit from detailed comparisons of the non-covalent interactions. We have applied Galinter to studying the protein-protein interaction mechanisms of three cases of protein mimicry: i) Chymotrypsin and subtilisin interact with the same type of inhibitors, an example of convergent evolution <a href="#pone.0001926-Wallace1">[37]</a>; ii) A scorpion-toxin derived compound (CD4M33-F23) mimics CD4 in complex with gp120, a mimicry case relevant to HIV therapy <a href="#pone.0001926-Huang1">[38]</a>; iii) A non-peptidic compound SP4206 mimics IL-2Rα in binding to IL-2 <a href="#pone.0001926-Thanos1">[39]</a>.</p>
<a id="article1.body1.sec3.sec2.p2" name="article1.body1.sec3.sec2.p2"></a><p>In each of these three cases, the subunits are homologous only on one side of the interface. In the third case, one of the interacting partners is not even a protein.</p>
<h5>Comparison of two protease-inhibitor interfaces.</h5><a id="article1.body1.sec3.sec2.sec1.p1" name="article1.body1.sec3.sec2.sec1.p1"></a><p>The Ser–His–Asp catalytic triad present in many proteases has been intensively analyzed <a href="#pone.0001926-Berg1">[40]</a>, <a href="#pone.0001926-Polgr1">[41]</a>. This catalytic triad occurs in several protein families which are non-homologous, and therefore have no significant backbone structure similarity <a href="#pone.0001926-Branden1">[42]</a>. Specifically, the trypsin-like serine proteases chymotrypsin and subtilisin belong to different SCOP superfamilies (<em>sccs</em> codes: b.47.1.2 and c.41.1.1, respectively). Although they lack obvious sequence or structure similarity, they have been found to share as many as three inhibitors <a href="#pone.0001926-Henschel1">[13]</a>.</p>
<a id="article1.body1.sec3.sec2.sec1.p2" name="article1.body1.sec3.sec2.sec1.p2"></a><p>We have analyzed the interactions formed between chymotrypsin and leech proteinase inhibitor eglin c (PDB code: 1acb, chains E and I), and subtilisin with chymotrypsin inhibitor 2 (PDB code: 1lw6, chains E and I). The two protease inhibitors have similar backbone structures and belong to the same SCOP family (b.40.1.1). The two interfaces contain 299 and 332 NCIVs, respectively. The longest Galinter alignment consists of 117 aligned NCIVs, and the results are visualized in <a href="#pone-0001926-g003">Figure 3A and 3B</a>. According to this alignment, the two catalytic triads are superposed with an RMSD of 0.5 Å (<a href="#pone-0001926-g003">Figure 3A</a>). The RMSD is computed for the overall functional template atoms of the catalytic triads as defined in Wallace <em>et al.</em> <a href="#pone.0001926-Wallace1">[37]</a>. <a href="#pone-0001926-g003">Figure 3B</a> displays superposed NCIVs according to Galinter at the two interfaces. It is noticeable that the NCIVs involving the catalytic serine and histidine residues are well conserved.</p>
<div class="figure" id="pone-0001926-g003"><div class="img"><a name="pone-0001926-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001926.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001926" data-uri="info:doi/10.1371/journal.pone.0001926.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001926.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001926.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001926.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001926.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001926.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001926.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001926.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001926.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001926.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Analysis of mimicry cases.</span></strong></p><a id="article1.body1.sec3.sec2.sec1.fig1.caption1.p1" name="article1.body1.sec3.sec2.sec1.fig1.caption1.p1"></a><p>Every example is shown with two representations in the same orientation. In all representations, the homologous side is in light blue and light yellow at the top, the mimic side is shown in dark blue and orange at the bottom. NCIVs at interfaces are shown as thin lines. A) Superposed inhibitors and catalytic triads for chymotrypsin (1acb) and subtilisin (1lw6) according to the Galinter alignment. The inhibitor for Chymotrypsin is shown in light blue and the inhibitor for subtilisin is shown in light yellow. The catalytic triads of chymotrypsin and subtilisin are shown as sticks in dark blue and orange, respectively. The chymotrypsin binding site is shown as a gray surface. B) Superposed NCIVs for chymotrypsin/inhibitor interface (1acbEI) and subtilisin/inhibitor interface (1lw6EI) according to the Galinter alignment. Only matched NCIVs are shown. Chymotrypsin/inhibitor NCIVs are shown in cyan, and subtilisin/inhibitor NCIVs are shown in yellow. C) Superposed NCIVs for CD4/gp120 interface (1rzjCG) and CD4M33-F23/gp120 interface (1yymMG) according to the Galinter alignment. CD4 is shown in dark blue and CD4M33-F23 is in orange. Only matched NCIVs are shown. CD4/gp120 NCIVs are shown in cyan, and CD4M33-F23/gp120 NCIVs are in yellow. Hydrogen bonds are shown as thick lines. D) An enlarged view of the matched NCIVs involving the hot spot phenylalanines. E) Superposed NCIVs according to the Galinter alignment of IL-2Rα/IL-2 interface (1z92BA) in dark and light blue, and of SP4206/IL-2 interface (1py2_A) in orange and light yellow. Only matched NCIVs are shown. IL-2Rα/IL-2 NCIVs are shown in cyan, SP4206/IL-2 NCIVs are in yellow. The hot spot residues Phe42, Tyr45, and Glu62 in IL-2 are shown as sticks. F) An enlarged view of the <em>mimic spot</em> around residue Glu62 in IL-2. PyMOL <a href="#pone.0001926-DeLano1">[46]</a> has been used to produce the representations.</p>
<span>doi:10.1371/journal.pone.0001926.g003</span></div><a id="article1.body1.sec3.sec2.sec1.p3" name="article1.body1.sec3.sec2.sec1.p3"></a><p>We have also compared the two interfaces based on inhibitor backbone alignment. First the inhibitor structures of the two complexes have been aligned using DaliLite. Then the two proteases have been superposed accordingly. This way an alignment of the interfaces is obtained indirectly. This indirect alignment agrees with the Galinter alignment to a considerable extent (irRMSD = 2.7 Å). Based on this indirect alignment, the RMSD for the overall functional template atoms of the catalytic triads is much larger than the one obtained based on the Galinter alignment (2.2 Å <em>vs.</em> 0.5 Å). This is not surprising given that these catalytic residues are not used by DaliLite when computing the alignment. Meanwhile, these results also indicate that to compare protein-protein interfaces, an explicit interface alignment approach is more adequate than an approach based on backbone structure.</p>

<h5>Analysis of a scorpion-toxin derived mimic of CD4 in complex with gp120.</h5><a id="article1.body1.sec3.sec2.sec2.p1" name="article1.body1.sec3.sec2.sec2.p1"></a><p>In order for HIV to infect host cells, the HIV envelope glycoprotein gp120 binds CD4 receptors located on the target cell surfaces. The CD4 binding site for gp120 has been engineered onto a scorpion-toxin protein, resulting in CD4M33-F23. Recently, the mimicked interaction of CD4M33-F23 in complex with gp120 has been investigated in detail and compared to the native complex structure of CD4 and gp120 <a href="#pone.0001926-Huang1">[38]</a>. In particular, Huang and colleagues analyzed the difference distance matrix between the two complexes for gp120 residues surrounding the hot spot residue Phe43 of CD4.</p>
<a id="article1.body1.sec3.sec2.sec2.p2" name="article1.body1.sec3.sec2.sec2.p2"></a><p>We have compared the natural complex interfaces (PDB code: 1rzj, chains C and G) and mimicry interface (PDB code: 1yym, chains M and G) using Galinter. The numbers of NCIVs are 364 for 1rzjCG and 166 for 1yymMG. In spite of the lack of similarities between the overall folds of CD4 and CD4M33-F23, about 80% (133 NCIVs) of the NCIVs at the CD4M33-F23/gp120 interface have been aligned to those at the CD4/gp120 interface. In addition, three of the four interface hydrogen bonds aligned as described in Huang <em>et al.</em> <a href="#pone.0001926-Huang1">[38]</a> are also aligned in the same way by Galinter (<a href="#pone-0001926-g003">Figure 3C</a>).</p>
<a id="article1.body1.sec3.sec2.sec2.p3" name="article1.body1.sec3.sec2.sec2.p3"></a><p>We have also observed that the hot spot residue Phe43 in CD4 (or equivalent residue Phe23 in CD4M33-F23) is in contact with eight residues of gp120 (Asp368, Glu370, Ile371, Asn425, Met426, Trp427, Gly473, and Met475) via 46 vdW interactions of 133 total aligned NCIVs in both interfaces. All these NCIVs have been aligned by Galinter successfully (<a href="#pone-0001926-g003">Figure 3D</a>).</p>

<h5>SP4206 mimic of IL-2Rα in binding to IL-2.</h5><a id="article1.body1.sec3.sec2.sec3.p1" name="article1.body1.sec3.sec2.sec3.p1"></a><p>Thanos <em>et al.</em> <a href="#pone.0001926-Thanos2">[43]</a> have published the structure of the small compound SP4206 binding to an IL-2 cytokine, which in turn blocks the natural interaction of IL-2 and its receptor IL-2Rα. Interestingly, although the interface size of SP4206 and IL-2 is only half as large as that between IL-2Rα and IL-2, SP4206 and IL-2Rα bind to IL-2 with similar affinities. Thanos and colleagues have discovered that this is mainly because SP4206 utilizes the same hot spot residues as IL-2Rα when interacting with IL-2 <a href="#pone.0001926-Thanos1">[39]</a>.</p>
<a id="article1.body1.sec3.sec2.sec3.p2" name="article1.body1.sec3.sec2.sec3.p2"></a><p>We have compared the interface of IL-2Rα and IL-2 (PDB code: 1z92, chains B and A), with the interface formed between SP4206 and IL-2 (PDB code: 1py2, FRH and chain A) using Galinter. The protocol has been slightly modified in order to identify hydrogen bonds between a non-peptidic molecule and a protein. HBPLUS <a href="#pone.0001926-McDonald1">[29]</a> has been used to infer hydrogen bonds within the interface between SP4206 and IL-2. We have identified 330 NCIVs for IL-2Rα/IL-2 interface, and 176 NCIVs for SP4206/IL-2 interface. The alignment results are shown in <a href="#pone-0001926-g003">Figure 3E</a>. Only a small number (35) of the interface NCIVs are aligned by Galinter. We have found that the main reason for this relatively short alignment is that the IL-2 binding sites adopt different conformations when binding the two partners. Particularly, two of the three hot spot residues on IL-2 binding sites (Phe42 and Tyr45) adopt different side chain formations in the interfaces. Only Glu62 is structurally conserved. In IL-2Rα/IL-2, this residue forms salt bridges with the guanido group of residue Arg36 in IL-2Rα. In SP4206/IL-2, we observe similar interactions between the carboxyl group of IL-2 Glu62 and the guanido group in SP4206 <a href="#pone.0001926-Thanos1">[39]</a>. Galinter correctly identifies these conserved interactions (see <a href="#pone-0001926-g003">Figure 3F</a>). Apparently the similarities are not uniformly distributed along the interfaces. It is noticeable that in proximity of residue Glu62 the NCIVs are conserved, while NCIVs are only sparsely aligned in the rest of the interfaces. We label this conserved interface region a <em>mimic spot</em>, in analogy to the concept of hot spot, which refers to residues contributing to a large fraction of the binding energy <a href="#pone.0001926-Bogan1">[44]</a>.</p>

<h5>Comparison to I2I-SiteEngine results.</h5><a id="article1.body1.sec3.sec2.sec4.p1" name="article1.body1.sec3.sec2.sec4.p1"></a><p>We have applied I2I-SiteEngine to align the three pairs of mimicry interfaces. In the case of the two protease-inhibitor interfaces, I2I-SiteEngine generates a similar alignment to Galinter with an irRMSD of 1.0 Å. The RMSD for the overall functional template atoms of the two catalytic triads is worse than that calculated based on Galinter alignment (1.1 Å <em>vs.</em> 0.5 Å). In addition, the RMSD for the two inhibitors is 4.2 Å which is higher than that obtained based on Galinter result (2.9 Å). For the second mimicry case, the I2I-SiteEngine alignment agrees with the Galinter result, with an irRMSD of only 0.4 Å. In the third mimicry case, one of the subunits participating in the interaction is a non-peptidic molecule (SP4206) and we could not obtain I2I-SiteEngine alignment. I2I-SiteEngine is only applicable to interfaces consisting of interacting proteins as it relies on the definition of functional groups of amino acids. This definition is not available for non-peptidic molecules. In this respect Galinter is more general than I2I-SiteEngine as it can also be applied to interfaces involving non-peptidic molecules.</p>


</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec4.p1" name="article1.body1.sec4.p1"></a><p>We have presented Galinter, a novel method for explicitly comparing interfaces based on the geometry and type of non-covalent interactions. The proposed method complements existing approaches to the analysis of protein-protein interfaces. The method was applied to the pilot dataset <a href="#pone.0001926-ShulmanPeleg1">[17]</a> and compared to an interface alignment method and to a backbone structure alignment method. It is reassuring that for S/D-homologous complexes we have obtained consistent results with the three methods. For non-homologous complexes, Galinter provides alternative solutions that tend to match common secondary structure elements at the interfaces. In addition, Galinter has been applied to comparing mimicry examples, and the results are consistent with previous human-curated analyses. The results also suggest that Galinter has the potential of assisting in the design of interaction inhibitors. In addition, as shown in the IL-2Rα mimicry example, Galinter is more general than existing approaches as it can compare interfaces in which non-peptidic molecules are involved.</p>
<a id="article1.body1.sec4.p2" name="article1.body1.sec4.p2"></a><p>Currently, the final Galinter alignments of NCIVs are ranked by their size in terms of the number of involved NCIVs, but a more comprehensive scoring function for alignments is desirable. Geometric and chemical similarity of matched NCIVs should be taken into account when computing alignment scores. Ideally such a scoring function should provide a statistical significance value for each alignment as well. This will be the focus of future work.</p>
<a id="article1.body1.sec4.p3" name="article1.body1.sec4.p3"></a><p>We have demonstrated the application of Galinter to the comparison of protein-protein interfaces, and also to the comparison of a protein-protein interface with an interface between a protein and a non-peptidic molecule (ligand). Galinter may also be applied for comparing protein-ligand to protein-ligand interfaces. But for this purpose the approach needs to be further tested. In addition, the interfaces in the current work have been defined between different polypeptide chains. However, the method is also applicable to the comparison of interfaces formed between protein domains along the same chain.</p>
<a id="article1.body1.sec4.p4" name="article1.body1.sec4.p4"></a><p>In the comparison of SP4206/IL-2 and IL-2Rα/IL-2, we have observed a non-uniform distribution of conserved NCIVs throughout the two interfaces. The NCIVs involving residue Arg36 on IL-2Rα and its counterpart guanido group on SP4206 are highly conserved. Similar results have also been observed in the first and second case studies. In the case of the protease/inhibitor interfaces, a large fraction of aligned NCIVs involve the two catalytic residues serine and histidine. At CD4/gp120 and CD4M33-F23/gp120 interfaces, Phe43 in CD4 and Phe23 in CD4M33-F23, respectively, form 46 NCIVs with eight surrounding residues (see <a href="#pone-0001926-g003">Figure 3D</a>). All these NCIVs are aligned and account for 35% of the final alignment. We call these conserved interface regions <em>mimic spots</em>. We plan to extend the functionality of Galinter to the automatic detection of conserved interface regions, as in the case of <em>mimic spots</em>. The relationship between conserved interface regions, <em>mimic spots</em> and hot spots is another interesting topic deserving further study. Recent results indicate that conserved regions and hot spots overlap to a considerable extent <a href="#pone.0001926-ShulmanPeleg3">[45]</a>.</p>
</div>

<div id="section5" class="section"><a id="s5" name="s5" toc="s5" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0001926"></div><a name="pone.0001926.s001" id="pone.0001926.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001926.s001">Figure S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1"></a><p class="preSiDOI">Geometric criteria for identifying hydrogen bonds.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001926.s001</p><a id="article1.body1.sec5.supplementary-material1.caption1.p2" name="article1.body1.sec5.supplementary-material1.caption1.p2"></a><p class="postSiDOI">(1.17 MB TIF)</p>
<a name="pone.0001926.s002" id="pone.0001926.s002"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001926.s002">Figure S2. </a></strong></p><a id="article1.body1.sec5.supplementary-material2.caption1.p1" name="article1.body1.sec5.supplementary-material2.caption1.p1"></a><p class="preSiDOI">Pilot dataset.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001926.s002</p><a id="article1.body1.sec5.supplementary-material2.caption1.p2" name="article1.body1.sec5.supplementary-material2.caption1.p2"></a><p class="postSiDOI">(0.60 MB TIF)</p>
<a name="pone.0001926.s003" id="pone.0001926.s003"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001926.s003">Figure S3. </a></strong></p><a id="article1.body1.sec5.supplementary-material3.caption1.p1" name="article1.body1.sec5.supplementary-material3.caption1.p1"></a><p class="preSiDOI">Galinter vs. I2I-SiteEngine. Heat maps for irRMSD values of interface residues. Only the 14 non-singleton groups in the pilot dataset are shown. The heat maps are sorted by size. The columns and rows for each heat map represent interfaces identified by their PDB code and chain names constituting the interfaces. The diagonal grids of all heat maps have been left blank. For S/D-homologous complexes, S/D-homology is indicated in corresponding grids by either a plus sign (+) for double-sided homology, or a minus sign (−) for single-sided homology. The heat maps have been produced using R (<a href="http://www.R-project.org">http://www.R-project.org</a>).</p>
<p class="siDoi">doi:10.1371/journal.pone.0001926.s003</p><a id="article1.body1.sec5.supplementary-material3.caption1.p2" name="article1.body1.sec5.supplementary-material3.caption1.p2"></a><p class="postSiDOI">(0.78 MB TIF)</p>
<a name="pone.0001926.s004" id="pone.0001926.s004"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001926.s004">Figure S4. </a></strong></p><a id="article1.body1.sec5.supplementary-material4.caption1.p1" name="article1.body1.sec5.supplementary-material4.caption1.p1"></a><p class="preSiDOI">Galinter vs. DaliLite. Heat maps for irRMSD values of interface residues. Only the 14 non-singleton groups in the pilot dataset are shown. The heat maps are sorted by size. The columns and rows for each heat map represent interfaces identified by their PDB code and chain names constituting the interfaces. The diagonal grids of all heat maps have been left blank. For S/D-homologous complexes, S/D-homology is indicated in corresponding grids by either a plus sign (+) for double-sided homology, or a minus sign (−) for single-sided homology. The heat maps have been produced using R (<a href="http://www.R-project.org">http://www.R-project.org</a>).</p>
<p class="siDoi">doi:10.1371/journal.pone.0001926.s004</p><a id="article1.body1.sec5.supplementary-material4.caption1.p2" name="article1.body1.sec5.supplementary-material4.caption1.p2"></a><p class="postSiDOI">(0.77 MB TIF)</p>
<a name="pone.0001926.s005" id="pone.0001926.s005"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001926.s005">Figure S5. </a></strong></p><a id="article1.body1.sec5.supplementary-material5.caption1.p1" name="article1.body1.sec5.supplementary-material5.caption1.p1"></a><p class="preSiDOI">I2I-SiteEngine vs. DaliLite. Heat maps for irRMSD values of interface residues. Only the 14 non-singleton groups in the pilot dataset are shown. The heat maps are sorted by size. The columns and rows for each heat map represent interfaces identified by their PDB code and chain names constituting the interfaces. The diagonal grids of all heat maps have been left blank. For S/D-homologous complexes, S/D-homology is indicated in corresponding grids by either a plus sign (+) for double-sided homology, or a minus sign (−) for single-sided homology. The heat maps have been produced using R (<a href="http://www.R-project.org">http://www.R-project.org</a>).</p>
<p class="siDoi">doi:10.1371/journal.pone.0001926.s005</p><a id="article1.body1.sec5.supplementary-material5.caption1.p2" name="article1.body1.sec5.supplementary-material5.caption1.p2"></a><p class="postSiDOI">(0.77 MB TIF)</p>
<a name="pone.0001926.s006" id="pone.0001926.s006"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001926.s006">Figure S6. </a></strong></p><a id="article1.body1.sec5.supplementary-material6.caption1.p1" name="article1.body1.sec5.supplementary-material6.caption1.p1"></a><p class="preSiDOI">Alignment of interfaces based on backbone structure. Using DaliLite, subunit structures are compared individually at both sides of interfaces. A subsequent alignment of interface residues can be derived based on the most significant DaliLite alignment of subunit structures.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001926.s006</p><a id="article1.body1.sec5.supplementary-material6.caption1.p2" name="article1.body1.sec5.supplementary-material6.caption1.p2"></a><p class="postSiDOI">(1.38 MB TIF)</p>
<a name="pone.0001926.s007" id="pone.0001926.s007"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001926.s007">Figure S7. </a></strong></p><a id="article1.body1.sec5.supplementary-material7.caption1.p1" name="article1.body1.sec5.supplementary-material7.caption1.p1"></a><p class="preSiDOI">Comparison of interface alignments using irRMSD measure. Given two interface residue sets <em>I</em><sub>1</sub> and <em>I</em><sub>2</sub> and two alignment methods <em>M</em><sub>a</sub> and <em>M</em><sub>b</sub>, let <em>I</em><sub>2a</sub> correspond to the transformed <em>I</em><sub>2</sub> according to the optimal superposition based on the alignment from method <em>M</em><sub>a</sub>. Analogously, <em>I</em><sub>2</sub> is transformed to <em>I</em><sub>2b</sub> based on the alignment from method <em>M</em><sub>b</sub>. Then, the root-mean-square deviation (RMSD) for all C<sub>α</sub> atoms of interface residues in <em>I</em><sub>2a</sub> and <em>I</em><sub>2b</sub> is calculated and reported as irRMSD to assess the agreement between the two methods. (NCIV: non-covalent interaction vector)</p>
<p class="siDoi">doi:10.1371/journal.pone.0001926.s007</p><a id="article1.body1.sec5.supplementary-material7.caption1.p2" name="article1.body1.sec5.supplementary-material7.caption1.p2"></a><p class="postSiDOI">(1.30 MB TIF)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>We are grateful to Oliver Sander and Andreas Steffen for fruitful discussions. We thank Gabriele Mayr and Mario Albrecht for helpful comments on the manuscript.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: HZ IS FD TL. Performed the experiments: HZ. Analyzed the data: HZ IS FD TL. Wrote the paper: HZ IS FD TL.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0001926-Jones1" id="pone.0001926-Jones1"></a>Jones S, Thornton J (1996) Principles of protein-protein interactions. Proc Natl Acad Sci U S A  93: 13–20.  <ul class="find" data-citedArticleID="1086102"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Principles+of+protein-protein+interactions.&amp;auth=&amp;atitle=Principles+of+protein-protein+interactions." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Principles+of+protein-protein+interactions." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Principles+of+protein-protein+interactions.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0001926-LoConte1" id="pone.0001926-LoConte1"></a>Lo Conte L, Chothia C, Janin J (1999) The atomic structure of protein-protein recognition sites. J Mol Biol  285(5): 2177–98.  <ul class="find" data-citedArticleID="1086116" data-doi="10.1006/jmbi.1998.2439"><li><a href="http://dx.doi.org/10.1006/jmbi.1998.2439" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+atomic+structure+of+protein-protein+recognition+sites." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+atomic+structure+of+protein-protein+recognition+sites.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0001926-Sheinerman1" id="pone.0001926-Sheinerman1"></a>Sheinerman FB, Norel R, Honig B (2000) Electrostatic aspects of protein-protein interactions. Curr Opin Struct Biol  10(2): 153–159.  <ul class="find" data-citedArticleID="1086136" data-doi="10.1016/s0959-440x(00)00065-8"><li><a href="http://dx.doi.org/10.1016/s0959-440x(00)00065-8" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Electrostatic+aspects+of+protein-protein+interactions." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Electrostatic+aspects+of+protein-protein+interactions.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0001926-Rodier1" id="pone.0001926-Rodier1"></a>Rodier F, Bahadur RP, Chakrabarti P, Janin J (2005) Hydration of protein-protein interfaces. Proteins  60: 36–45.  <ul class="find" data-citedArticleID="1086132" data-doi="10.1002/prot.20478"><li><a href="http://dx.doi.org/10.1002/prot.20478" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Hydration+of+protein-protein+interfaces." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Hydration+of+protein-protein+interfaces.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0001926-Nooren1" id="pone.0001926-Nooren1"></a>Nooren IMA, Thornton JM (2003) Diversity of protein-protein interactions. EMBO J  22(14): 3486–92.  <ul class="find" data-citedArticleID="1086128" data-doi="10.1093/emboj/cdg359"><li><a href="http://dx.doi.org/10.1093/emboj/cdg359" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Diversity+of+protein-protein+interactions." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Diversity+of+protein-protein+interactions.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0001926-Bahadur1" id="pone.0001926-Bahadur1"></a>Bahadur RP, Chakrabarti P, Rodier F, Janin J (2004) A dissection of specific and non-specific protein-protein interfaces. J Mol Biol  336(4): 943–55.  <ul class="find" data-citedArticleID="1086074" data-doi="10.1016/j.jmb.2003.12.073"><li><a href="http://dx.doi.org/10.1016/j.jmb.2003.12.073" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+dissection+of+specific+and+non-specific+protein-protein+interfaces." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+dissection+of+specific+and+non-specific+protein-protein+interfaces.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0001926-Mintseris1" id="pone.0001926-Mintseris1"></a>Mintseris J, Weng Z (2005) Structure, function, and evolution of transient and obligate protein-protein interactions. Proc Natl Acad Sci U S A  102(31): 10930–10935.  <ul class="find" data-citedArticleID="1086120" data-doi="10.1073/pnas.0502667102"><li><a href="http://dx.doi.org/10.1073/pnas.0502667102" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Structure%2C+function%2C+and+evolution+of+transient+and+obligate+protein-protein+interactions." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Structure%2C+function%2C+and+evolution+of+transient+and+obligate+protein-protein+interactions.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0001926-Zhu1" id="pone.0001926-Zhu1"></a>Zhu H, Domingues F, Sommer I, Lengauer T (2006) NOXclass: prediction of protein-protein interaction types. BMC Bioinformatics  7: 27.  <ul class="find" data-citedArticleID="1086162" data-doi="10.1186/1471-2105-7-27"><li><a href="http://dx.doi.org/10.1186/1471-2105-7-27" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=NOXclass%3A+prediction+of+protein-protein+interaction+types." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22NOXclass%3A+prediction+of+protein-protein+interaction+types.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0001926-Berman1" id="pone.0001926-Berman1"></a>Berman HM, Westbrook J, Feng Z, Gilliland G, Bhat TN, Weissig H, Shindyalov IN, Bourne PE (2000) The Protein Data Bank. Nucleic Acids Res  28: 235–42.  <ul class="find" data-citedArticleID="1086078" data-doi="10.1107/97809553602060000722"><li><a href="http://dx.doi.org/10.1107/97809553602060000722" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+Protein+Data+Bank." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+Protein+Data+Bank.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0001926-Aloy1" id="pone.0001926-Aloy1"></a>Aloy P, Ceulemans H, Stark A, Russell RB (2003) The relationship between sequence and interaction divergence in proteins. J Mol Biol  332(5): 989–98.  <ul class="find" data-citedArticleID="1086072" data-doi="10.1016/j.jmb.2003.07.006"><li><a href="http://dx.doi.org/10.1016/j.jmb.2003.07.006" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+relationship+between+sequence+and+interaction+divergence+in+proteins." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+relationship+between+sequence+and+interaction+divergence+in+proteins.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0001926-Kim1" id="pone.0001926-Kim1"></a>Kim WK, Henschel A, Winter C, Schroeder M (2006) The Many Faces of Protein-protein Interactions: A Compendium of Interface Geometry. PLoS Comput Biol  2(9): e124.  <ul class="find" data-citedArticleID="1086110" data-doi="10.1371/journal.pcbi.0020124.eor"><li><a href="http://dx.doi.org/10.1371/journal.pcbi.0020124.eor" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+Many+Faces+of+Protein-protein+Interactions%3A+A+Compendium+of+Interface+Geometry." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+Many+Faces+of+Protein-protein+Interactions%3A+A+Compendium+of+Interface+Geometry.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0001926-Winter1" id="pone.0001926-Winter1"></a>Winter C, Henschel A, Kim WK, Schroeder M (2006) SCOPPI: a structural classification of protein-protein interfaces. Nucleic Acids Res  34(Database issue): D310–D314.  <ul class="find" data-citedArticleID="1086156" data-doi="10.1093/nar/gkj099"><li><a href="http://dx.doi.org/10.1093/nar/gkj099" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=SCOPPI%3A+a+structural+classification+of+protein-protein+interfaces." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22SCOPPI%3A+a+structural+classification+of+protein-protein+interfaces.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0001926-Henschel1" id="pone.0001926-Henschel1"></a>Henschel A, Kim WK, Schroeder M (2006) Equivalent binding sites reveal convergently evolved interaction motifs. Bioinformatics  22(5): 550–555.  <ul class="find" data-citedArticleID="1086096" data-doi="10.1093/bioinformatics/bti782"><li><a href="http://dx.doi.org/10.1093/bioinformatics/bti782" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Equivalent+binding+sites+reveal+convergently+evolved+interaction+motifs." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Equivalent+binding+sites+reveal+convergently+evolved+interaction+motifs.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0001926-Tsai1" id="pone.0001926-Tsai1"></a>Tsai CJ, Lin SL, Wolfson HJ, Nussinov R (1996) A dataset of protein-protein inter faces generated with a sequence order independent comparison technique. J Mol Biol  260(4): 604–620.  <ul class="find" data-citedArticleID="1086152" data-doi="10.1006/jmbi.1996.0424"><li><a href="http://dx.doi.org/10.1006/jmbi.1996.0424" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+dataset+of+protein-protein+inter+faces+generated+with+a+sequence+order+independent+comparison+technique." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+dataset+of+protein-protein+inter+faces+generated+with+a+sequence+order+independent+comparison+technique.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0001926-Keskin1" id="pone.0001926-Keskin1"></a>Keskin O, Tsai CJ, Wolfson H, Nussinov R (2004) A new, structurally nonredundant, diverse data set of protein-protein interfaces and its implications. Protein Sci  13(4): 1043–55.  <ul class="find" data-citedArticleID="1086104" data-doi="10.1110/ps.03484604"><li><a href="http://dx.doi.org/10.1110/ps.03484604" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+new%2C+structurally+nonredundant%2C+diverse+data+set+of+protein-protein+interfaces+and+its+implications." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+new%2C+structurally+nonredundant%2C+diverse+data+set+of+protein-protein+interfaces+and+its+implications.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0001926-Keskin2" id="pone.0001926-Keskin2"></a>Keskin O, Nussinov R (2005) Favorable scaffolds: proteins with different sequence, structure and function may associate in similar ways. Protein Eng Des Sel  18: 11–24.  <ul class="find" data-citedArticleID="1086106" data-doi="10.1093/protein/gzh095"><li><a href="http://dx.doi.org/10.1093/protein/gzh095" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Favorable+scaffolds%3A+proteins+with+different+sequence%2C+structure+and+function+may+associate+in+similar+ways." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Favorable+scaffolds%3A+proteins+with+different+sequence%2C+structure+and+function+may+associate+in+similar+ways.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0001926-ShulmanPeleg1" id="pone.0001926-ShulmanPeleg1"></a>Shulman-Peleg A, Mintz S, Nussinov R, Wolfson HJ (2004)  Protein-Protein Interfaces: Recognition of Similar Spatial and Chemical Organizations. In: Jonassen I, Kim J, editors. Proceedings of the Fourth International Workshop on Algorithms in Bioinformatics, Volume 3240 of Lecture Notes in Computer Science. Berlin / Heidelberg: Springer. pp. 194–205.  <ul class="find-nolinks"></ul></li><li><span class="label">18.
              </span><a name="pone.0001926-Mintz1" id="pone.0001926-Mintz1"></a>Mintz S, Shulman-Peleg A, Wolfson HJ, Nussinov R (2005) Generation and analysis of a protein-protein interface data set with similar chemical and spatial patterns of interactions. Proteins  61: 6–20.  <ul class="find" data-citedArticleID="1086122" data-doi="10.1002/prot.20580"><li><a href="http://dx.doi.org/10.1002/prot.20580" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Generation+and+analysis+of+a+protein-protein+interface+data+set+with+similar+chemical+and+spatial+patterns+of+interactions." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Generation+and+analysis+of+a+protein-protein+interface+data+set+with+similar+chemical+and+spatial+patterns+of+interactions.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0001926-ShulmanPeleg2" id="pone.0001926-ShulmanPeleg2"></a>Shulman-Peleg A, Shatsky M, Nussinov R, Wolfson HJ (2005)  MAPPIS: Multiple 3D Alignment of Protein-Protein Interfaces. In: Berthold MR, Glen RC, Diederichs K, Kohlbacher O, Fischer I, editors. CompLife, Volume 3695 of Lecture Notes in Computer Science. Berlin/Heidelberg: Springer. pp. 91–103.  <ul class="find-nolinks"></ul></li><li><span class="label">20.
              </span><a name="pone.0001926-Schmitt1" id="pone.0001926-Schmitt1"></a>Schmitt S, Kuhn D, Klebe G (2002) A new method to detect related function among proteins independent of sequence and fold homology. J Mol Biol  323(2): 387–406.  <ul class="find" data-citedArticleID="1086134" data-doi="10.1016/s0022-2836(02)00811-2"><li><a href="http://dx.doi.org/10.1016/s0022-2836(02)00811-2" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+new+method+to+detect+related+function+among+proteins+independent+of+sequence+and+fold+homology." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+new+method+to+detect+related+function+among+proteins+independent+of+sequence+and+fold+homology.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0001926-Najmanovich1" id="pone.0001926-Najmanovich1"></a>Najmanovich RJ, Allali Hassani A, Morris RJ, Dombrovsky L, Pan PW, et al.  (2007) Analysis of binding site similarity, small molecule similarity and experimental binding profiles in the human cytosolic sulfotransferase family. Bioinformatics  23(2): e104–e109.  <ul class="find" data-citedArticleID="1086126" data-doi="10.1093/bioinformatics/btl292"><li><a href="http://dx.doi.org/10.1093/bioinformatics/btl292" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Analysis+of+binding+site+similarity%2C+small+molecule+similarity+and+experimental+binding+profiles+in+the+human+cytosolic+sulfotransferase+family." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Analysis+of+binding+site+similarity%2C+small+molecule+similarity+and+experimental+binding+profiles+in+the+human+cytosolic+sulfotransferase+family.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0001926-Domingues1" id="pone.0001926-Domingues1"></a>Domingues FS, Lengauer T (2007)  Inferring Protein Function from Protein Structure. In: Lengauer T, editor. Bioinformatics From Genomes to Therapies, Volume 3. Weinheim: Wiley VCH. pp. 1211–1252.  <ul class="find-nolinks"></ul></li><li><span class="label">23.
              </span><a name="pone.0001926-Biswal1" id="pone.0001926-Biswal1"></a>Biswal BK, Cherney MM, Wang M, Chan L, Yannopoulos CG, et al.  (2005) Crystal structures of the RNA dependent RNA polymerase genotype 2a of hepatitis C virus reveal two conformations and suggest mechanisms of inhibition by non-nucleoside inhibitors. J Biol Chem  280(18): 18202–18210.  <ul class="find" data-citedArticleID="1086080" data-doi="10.1074/jbc.m413410200"><li><a href="http://dx.doi.org/10.1074/jbc.m413410200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Crystal+structures+of+the+RNA+dependent+RNA+polymerase+genotype+2a+of+hepatitis+C+virus+reveal+two+conformations+and+suggest+mechanisms+of+inhibition+by+non-nucleoside+inhibitors." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Crystal+structures+of+the+RNA+dependent+RNA+polymerase+genotype+2a+of+hepatitis+C+virus+reveal+two+conformations+and+suggest+mechanisms+of+inhibition+by+non-nucleoside+inhibitors.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0001926-Deng1" id="pone.0001926-Deng1"></a>Deng Z, Chuaqui C, Singh J (2004) Structural interaction fingerprint (SIFt): a novel method for analyzing three-dimensional protein-ligand binding interactions. J Med Chem  47(2): 337–344.  <ul class="find" data-citedArticleID="1086090" data-doi="10.1021/jm030331x"><li><a href="http://dx.doi.org/10.1021/jm030331x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Structural+interaction+fingerprint+%28SIFt%29%3A+a+novel+method+for+analyzing+three-dimensional+protein-ligand+binding+interactions." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Structural+interaction+fingerprint+%28SIFt%29%3A+a+novel+method+for+analyzing+three-dimensional+protein-ligand+binding+interactions.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0001926-SwintKruse1" id="pone.0001926-SwintKruse1"></a>Swint-Kruse L (2004) Using networks to identify fine structural differences between functionally distinct protein states. Biochemistry  43(34): 10886–10895.  <ul class="find" data-citedArticleID="1086144" data-doi="10.1021/bi049450k"><li><a href="http://dx.doi.org/10.1021/bi049450k" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Using+networks+to+identify+fine+structural+differences+between+functionally+distinct+protein+states." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Using+networks+to+identify+fine+structural+differences+between+functionally+distinct+protein+states.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0001926-SwintKruse2" id="pone.0001926-SwintKruse2"></a>Swint-Kruse L, Brown CS (2005) Resmap: automated representation of macro molecular interfaces as two dimensional networks. Bioinformatics  21(15): 3327–3328.  <ul class="find" data-citedArticleID="1086146" data-doi="10.1093/bioinformatics/bti511"><li><a href="http://dx.doi.org/10.1093/bioinformatics/bti511" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Resmap%3A+automated+representation+of+macro+molecular+interfaces+as+two+dimensional+networks." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Resmap%3A+automated+representation+of+macro+molecular+interfaces+as+two+dimensional+networks.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0001926-Keskin3" id="pone.0001926-Keskin3"></a>Keskin O, Nussinov R (2007) Similar binding sites and different partners: implications to shared proteins in cellular pathways. Structure  15(3): 341–354.  <ul class="find" data-citedArticleID="1086108" data-doi="10.1016/j.str.2007.01.007"><li><a href="http://dx.doi.org/10.1016/j.str.2007.01.007" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Similar+binding+sites+and+different+partners%3A+implications+to+shared+proteins+in+cellular+pathways." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Similar+binding+sites+and+different+partners%3A+implications+to+shared+proteins+in+cellular+pathways.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0001926-Word1" id="pone.0001926-Word1"></a>Word JM, Lovell SC, Richardson JS, Richardson DC (1999) Asparagine and glutamine: using hydrogen atom contacts in the choice of side chain amide orientation. J Mol Biol  285(4): 1735–1747.  <ul class="find" data-citedArticleID="1086158" data-doi="10.1006/jmbi.1998.2401"><li><a href="http://dx.doi.org/10.1006/jmbi.1998.2401" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Asparagine+and+glutamine%3A+using+hydrogen+atom+contacts+in+the+choice+of+side+chain+amide+orientation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Asparagine+and+glutamine%3A+using+hydrogen+atom+contacts+in+the+choice+of+side+chain+amide+orientation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0001926-McDonald1" id="pone.0001926-McDonald1"></a>McDonald IK, Thornton JM (1994) Satisfying hydrogen bonding potential in proteins. J Mol Biol  238(5): 777–793.  <ul class="find" data-citedArticleID="1086118" data-doi="10.1006/jmbi.1994.1334"><li><a href="http://dx.doi.org/10.1006/jmbi.1994.1334" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Satisfying+hydrogen+bonding+potential+in+proteins." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Satisfying+hydrogen+bonding+potential+in+proteins.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0001926-Grindley1" id="pone.0001926-Grindley1"></a>Grindley HM, Artymiuk PJ, Rice DW, Willett P (1993) Identification of tertiary structure resemblance in proteins using a maximal common subgraph isomorphism algorithm. J Mol Biol  229(3): 707–721.  <ul class="find" data-citedArticleID="1086094" data-doi="10.1006/jmbi.1993.1074"><li><a href="http://dx.doi.org/10.1006/jmbi.1993.1074" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Identification+of+tertiary+structure+resemblance+in+proteins+using+a+maximal+common+subgraph+isomorphism+algorithm." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Identification+of+tertiary+structure+resemblance+in+proteins+using+a+maximal+common+subgraph+isomorphism+algorithm.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0001926-Koch1" id="pone.0001926-Koch1"></a>Koch I, Lengauer T, Wanke E (1996) An algorithm for finding maximal common subtopologies in a set of protein structures. J Comput Biol  3(2): 289–306.  <ul class="find" data-citedArticleID="1086112" data-doi="10.1089/cmb.1996.3.289"><li><a href="http://dx.doi.org/10.1089/cmb.1996.3.289" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=An+algorithm+for+finding+maximal+common+subtopologies+in+a+set+of+protein+structures." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22An+algorithm+for+finding+maximal+common+subtopologies+in+a+set+of+protein+structures.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0001926-Levi1" id="pone.0001926-Levi1"></a>Levi G (1972) A note on the derivation of maximal common subgraphs of two directed or undirected graphs. Calcolo  9(4): 341–352.  <ul class="find" data-citedArticleID="1086114" data-doi="10.1007/bf02575586"><li><a href="http://dx.doi.org/10.1007/bf02575586" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+note+on+the+derivation+of+maximal+common+subgraphs+of+two+directed+or+undirected+graphs." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+note+on+the+derivation+of+maximal+common+subgraphs+of+two+directed+or+undirected+graphs.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0001926-Bron1" id="pone.0001926-Bron1"></a>Bron C, Kerbosch J (1973) Algorithm 457: finding all cliques of an undirected graph. Commun. ACM  16(9): 575–577.  <ul class="find" data-citedArticleID="1086086" data-doi="10.1145/362342.362367"><li><a href="http://dx.doi.org/10.1145/362342.362367" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Algorithm+457%3A+finding+all+cliques+of+an+undirected+graph.+Commun." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Algorithm+457%3A+finding+all+cliques+of+an+undirected+graph.+Commun.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0001926-Murzin1" id="pone.0001926-Murzin1"></a>Murzin AG, Brenner SE, Hubbard T, Chothia C (1995) SCOP: a structural classification of proteins database for the investigation of sequences and structures. J Mol Biol  247(4): 536–40.  <ul class="find" data-citedArticleID="1086124" data-doi="10.1016/s0022-2836(05)80134-2"><li><a href="http://dx.doi.org/10.1016/s0022-2836(05)80134-2" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=SCOP%3A+a+structural+classification+of+proteins+database+for+the+investigation+of+sequences+and+structures." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22SCOP%3A+a+structural+classification+of+proteins+database+for+the+investigation+of+sequences+and+structures.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0001926-Holm1" id="pone.0001926-Holm1"></a>Holm L, Park J (2000) DaliLite workbench for protein structure comparison. Bioinformatics  16(6): 566–567.  <ul class="find" data-citedArticleID="1086098" data-doi="10.1093/bioinformatics/16.6.566"><li><a href="http://dx.doi.org/10.1093/bioinformatics/16.6.566" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=DaliLite+workbench+for+protein+structure+comparison." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22DaliLite+workbench+for+protein+structure+comparison.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">36.
              </span><a name="pone.0001926-Xu1" id="pone.0001926-Xu1"></a>Xu D, Tsai CJ, Nussinov R (1997) Hydrogen bonds and salt bridges across protein-protein interfaces. Protein Eng  10(9): 999–1012.  <ul class="find" data-citedArticleID="1086160" data-doi="10.1093/protein/10.9.999"><li><a href="http://dx.doi.org/10.1093/protein/10.9.999" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Hydrogen+bonds+and+salt+bridges+across+protein-protein+interfaces." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Hydrogen+bonds+and+salt+bridges+across+protein-protein+interfaces.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">37.
              </span><a name="pone.0001926-Wallace1" id="pone.0001926-Wallace1"></a>Wallace AC, Laskowski RA, Thornton JM (1996) Derivation of 3D coordinate templates for searching structural databases: application to Ser-His-Asp catalytic triads in the serine proteinases and lipases. Protein Sci  5(6): 1001–1013.  <ul class="find" data-citedArticleID="1086154" data-doi="10.1002/pro.5560050603"><li><a href="http://dx.doi.org/10.1002/pro.5560050603" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Derivation+of+3D+coordinate+templates+for+searching+structural+databases%3A+application+to+Ser-His-Asp+catalytic+triads+in+the+serine+proteinases+and+lipases." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Derivation+of+3D+coordinate+templates+for+searching+structural+databases%3A+application+to+Ser-His-Asp+catalytic+triads+in+the+serine+proteinases+and+lipases.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">38.
              </span><a name="pone.0001926-Huang1" id="pone.0001926-Huang1"></a>Huang C, Stricher F, Martin L, Decker JM, Majeed S, et al.  (2005) Scorpion-toxin mimics of CD4 in complex with human immunodeficiency virus gp120 crystal structures, molecular mimicry, and neutralization breadth. Structure  13(5): 755–768.  <ul class="find" data-citedArticleID="1086100" data-doi="10.1016/j.str.2005.03.006"><li><a href="http://dx.doi.org/10.1016/j.str.2005.03.006" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Scorpion-toxin+mimics+of+CD4+in+complex+with+human+immunodeficiency+virus+gp120+crystal+structures%2C+molecular+mimicry%2C+and+neutralization+breadth." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Scorpion-toxin+mimics+of+CD4+in+complex+with+human+immunodeficiency+virus+gp120+crystal+structures%2C+molecular+mimicry%2C+and+neutralization+breadth.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">39.
              </span><a name="pone.0001926-Thanos1" id="pone.0001926-Thanos1"></a>Thanos CD, DeLano WL, Wells JA (2006) Hot-spot mimicry of a cytokine receptor by a small molecule. Proc Natl Acad Sci U S A  103(42): 15422–15427.  <ul class="find" data-citedArticleID="1086148" data-doi="10.1073/pnas.0607058103"><li><a href="http://dx.doi.org/10.1073/pnas.0607058103" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Hot-spot+mimicry+of+a+cytokine+receptor+by+a+small+molecule." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Hot-spot+mimicry+of+a+cytokine+receptor+by+a+small+molecule.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">40.
              </span><a name="pone.0001926-Berg1" id="pone.0001926-Berg1"></a>Berg JM, Tymoczko JL, Stryer L (2002) Biochemistry. New York: W. H. Freeman and Company, fifth edition.   <ul class="find-nolinks"></ul></li><li><span class="label">41.
              </span><a name="pone.0001926-Polgr1" id="pone.0001926-Polgr1"></a>Polgár L (2005) The catalytic triad of serine peptidases. Cell Mol Life Sci  62(1920): 2161–2172.  <ul class="find" data-citedArticleID="1086130" data-doi="10.1007/s00018-005-5160-x"><li><a href="http://dx.doi.org/10.1007/s00018-005-5160-x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+catalytic+triad+of+serine+peptidases." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+catalytic+triad+of+serine+peptidases.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">42.
              </span><a name="pone.0001926-Branden1" id="pone.0001926-Branden1"></a>Branden C, Tooze J (1999) Introduction to Protein Structure. New York: Garland Publishing, Inc., 2nd edition.   <ul class="find-nolinks"></ul></li><li><span class="label">43.
              </span><a name="pone.0001926-Thanos2" id="pone.0001926-Thanos2"></a>Thanos CD, Randal M, Wells JA (2003) Potent small-molecule binding to a dynamic hot spot on IL-2. J Am Chem Soc  125(50): 15280–15281.  <ul class="find" data-citedArticleID="1086150" data-doi="10.1021/ja0382617"><li><a href="http://dx.doi.org/10.1021/ja0382617" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Potent+small-molecule+binding+to+a+dynamic+hot+spot+on+IL-2." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Potent+small-molecule+binding+to+a+dynamic+hot+spot+on+IL-2.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">44.
              </span><a name="pone.0001926-Bogan1" id="pone.0001926-Bogan1"></a>Bogan AA, Thorn KS (1998) Anatomy of hot spots in protein interfaces. J Mol Biol  280: 1–9.  <ul class="find" data-citedArticleID="1086082" data-doi="10.1006/jmbi.1998.1843"><li><a href="http://dx.doi.org/10.1006/jmbi.1998.1843" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Anatomy+of+hot+spots+in+protein+interfaces." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Anatomy+of+hot+spots+in+protein+interfaces.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">45.
              </span><a name="pone.0001926-ShulmanPeleg3" id="pone.0001926-ShulmanPeleg3"></a>Shulman-Peleg A, Shatsky M, Nussinov R, Wolfson HJ (2007) Spatial chemical conservation of hot spot interactions in protein-protein complexes. BMC Biol  5: 43.  <ul class="find" data-citedArticleID="1086142" data-doi="10.1186/1741-7007-5-43"><li><a href="http://dx.doi.org/10.1186/1741-7007-5-43" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Spatial+chemical+conservation+of+hot+spot+interactions+in+protein-protein+complexes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Spatial+chemical+conservation+of+hot+spot+interactions+in+protein-protein+complexes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">46.
              </span><a name="pone.0001926-DeLano1" id="pone.0001926-DeLano1"></a>DeLano WLThe PyMOL Molecular Graphics System.   on World Wide Web <a href="http://www.pymol.org">http://www.pymol.org</a> 2002.  <ul class="find-nolinks"></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001926.XML" value="95218"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001926.PDF" value="310346"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001926.g001.PNG_L" value="174429"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001926.g001.PNG_M" value="55989"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001926.g001.PNG_S" value="4166"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001926.g001.TIF" value="332108"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001926.g001.PNG_I" value="27057"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001926.e001.PNG" value="16756"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001926.e001.TIF" value="33588"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001926.e002.PNG" value="13584"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001926.e002.TIF" value="29624"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001926.e003.PNG" value="13350"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001926.e003.TIF" value="29500"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001926.g002.PNG_L" value="142040"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001926.g002.PNG_M" value="27253"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001926.g002.PNG_S" value="3278"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001926.g002.TIF" value="373298"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001926.g002.PNG_I" value="10436"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001926.g003.PNG_L" value="3469807"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001926.g003.PNG_M" value="331387"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001926.g003.PNG_S" value="17839"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001926.g003.TIF" value="4769526"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001926.g003.PNG_I" value="111675"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001926.s001.TIF" value="708850"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001926.s002.TIF" value="1022418"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001926.s003.TIF" value="2011452"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001926.s004.TIF" value="2011802"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001926.s005.TIF" value="2007930"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001926.s006.TIF" value="856124"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001926.s007.TIF" value="1095692"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001926&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001926" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001926&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0001926&volume=&issue=&title=Alignment of Non-Covalent Interactions at Protein-Protein Interfaces&author_name=Hongbo%20Zhu%2C%20Ingolf%20Sommer%2C%20Thomas%20Lengauer%2C%20Francisco%20S.%20Domingues&start_page=1&end_page=9" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001926" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001926" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0001926" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001926&amp;t=Alignment%20of%20Non-Covalent%20Interactions%20at%20Protein-Protein%20Interfaces" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001926&title=Alignment%20of%20Non-Covalent%20Interactions%20at%20Protein-Protein%20Interfaces&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001926&amp;title=Alignment%20of%20Non-Covalent%20Interactions%20at%20Protein-Protein%20Interfaces" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001926" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0001926&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Alignment of Non-Covalent Interactions at Protein-Protein Interfaces';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0001926';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Alignment%20of%20Non-Covalent%20Interactions%20at%20Protein-Protein%20Interfaces http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001926" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0001926" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">











          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Chymotrypsin%22" title="Search for articles in the subject area:'Chymotrypsin'"><div class="flagText">Chymotrypsin</div></a>
              <div data-categoryid="24415" data-articleid="26676"
                   data-categoryname="Chymotrypsin"
                   class="flagImage" title="Flag 'Chymotrypsin' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Graphs%22" title="Search for articles in the subject area:'Graphs'"><div class="flagText">Graphs</div></a>
              <div data-categoryid="19267" data-articleid="26676"
                   data-categoryname="Graphs"
                   class="flagImage" title="Flag 'Graphs' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Hydrogen+bonding%22" title="Search for articles in the subject area:'Hydrogen bonding'"><div class="flagText">Hydrogen bonding</div></a>
              <div data-categoryid="23891" data-articleid="26676"
                   data-categoryname="Hydrogen bonding"
                   class="flagImage" title="Flag 'Hydrogen bonding' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Protein+complexes%22" title="Search for articles in the subject area:'Protein complexes'"><div class="flagText">Protein complexes</div></a>
              <div data-categoryid="17297" data-articleid="26676"
                   data-categoryname="Protein complexes"
                   class="flagImage" title="Flag 'Protein complexes' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Protein+interactions%22" title="Search for articles in the subject area:'Protein interactions'"><div class="flagText">Protein interactions</div></a>
              <div data-categoryid="17661" data-articleid="26676"
                   data-categoryname="Protein interactions"
                   class="flagImage" title="Flag 'Protein interactions' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Protein+structure%22" title="Search for articles in the subject area:'Protein structure'"><div class="flagText">Protein structure</div></a>
              <div data-categoryid="31877" data-articleid="26676"
                   data-categoryname="Protein structure"
                   class="flagImage" title="Flag 'Protein structure' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Protein+structure+comparison%22" title="Search for articles in the subject area:'Protein structure comparison'"><div class="flagText">Protein structure comparison</div></a>
              <div data-categoryid="20777" data-articleid="26676"
                   data-categoryname="Protein structure comparison"
                   class="flagImage" title="Flag 'Protein structure comparison' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Protein-protein+interactions%22" title="Search for articles in the subject area:'Protein-protein interactions'"><div class="flagText">Protein-protein interactions</div></a>
              <div data-categoryid="18067" data-articleid="26676"
                   data-categoryname="Protein-protein interactions"
                   class="flagImage" title="Flag 'Protein-protein interactions' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=3949'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=5511'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=6263&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>

<div class="block sidebar-comments">
    <div class="header">
        <h3>Comments</h3>
    </div>
      <p><a href="/annotation/listThread.action?root=21649">Referee Comments: Referee 1 (Michael Schroeder)</a><br>Posted by PLoS_ONE_Group</p>
</div>

</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
