

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: A New Perspective on Transcriptional System Regulation (TSR): Towards TSR Profiling</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0001656"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0001656" />

    <meta name="citation_title" content="A New Perspective on Transcriptional System Regulation (TSR): Towards TSR Profiling"/>
    <meta itemprop="name" content="A New Perspective on Transcriptional System Regulation (TSR): Towards TSR Profiling"/>

      <meta name="citation_author" content="Rudolf S. N. Fehrmann"/>
            <meta name="citation_author_institution" content="Department of Genetics, University Medical Center Groningen, University of Groningen, Groningen, The Netherlands"/>
            <meta name="citation_author_institution" content="Department of Gynecology, University Medical Center Groningen, University of Groningen, Groningen, The Netherlands"/>
            <meta name="citation_author_institution" content="Department of Medical Oncology, University Medical Center Groningen, University of Groningen, Groningen, The Netherlands"/>
      <meta name="citation_author" content="Hendrik J. M. de Jonge"/>
            <meta name="citation_author_institution" content="Division of Pediatric Oncology, Department of Pediatrics, Beatrix Children's Hospital, University Medical Center Groningen, University of Groningen, Groningen, The Netherlands"/>
      <meta name="citation_author" content="Arja ter Elst"/>
            <meta name="citation_author_institution" content="Division of Pediatric Oncology, Department of Pediatrics, Beatrix Children's Hospital, University Medical Center Groningen, University of Groningen, Groningen, The Netherlands"/>
      <meta name="citation_author" content="AndrÃ© de Vries"/>
            <meta name="citation_author_institution" content="Department of Genetics, University Medical Center Groningen, University of Groningen, Groningen, The Netherlands"/>
      <meta name="citation_author" content="Anne G. P. Crijns"/>
            <meta name="citation_author_institution" content="Department of Gynecology, University Medical Center Groningen, University of Groningen, Groningen, The Netherlands"/>
      <meta name="citation_author" content="Alida C. Weidenaar"/>
            <meta name="citation_author_institution" content="Division of Pediatric Oncology, Department of Pediatrics, Beatrix Children's Hospital, University Medical Center Groningen, University of Groningen, Groningen, The Netherlands"/>
      <meta name="citation_author" content="Frans Gerbens"/>
            <meta name="citation_author_institution" content="Department of Genetics, University Medical Center Groningen, University of Groningen, Groningen, The Netherlands"/>
      <meta name="citation_author" content="Steven de Jong"/>
            <meta name="citation_author_institution" content="Department of Medical Oncology, University Medical Center Groningen, University of Groningen, Groningen, The Netherlands"/>
      <meta name="citation_author" content="Ate G. J. van der Zee"/>
            <meta name="citation_author_institution" content="Department of Gynecology, University Medical Center Groningen, University of Groningen, Groningen, The Netherlands"/>
      <meta name="citation_author" content="Elisabeth G. E. de Vries"/>
            <meta name="citation_author_institution" content="Department of Medical Oncology, University Medical Center Groningen, University of Groningen, Groningen, The Netherlands"/>
      <meta name="citation_author" content="Willem A. Kamps"/>
            <meta name="citation_author_institution" content="Division of Pediatric Oncology, Department of Pediatrics, Beatrix Children's Hospital, University Medical Center Groningen, University of Groningen, Groningen, The Netherlands"/>
      <meta name="citation_author" content="Robert M. W. Hofstra"/>
            <meta name="citation_author_institution" content="Department of Genetics, University Medical Center Groningen, University of Groningen, Groningen, The Netherlands"/>
      <meta name="citation_author" content="Gerard J. te Meerman"/>
            <meta name="citation_author_institution" content="Department of Genetics, University Medical Center Groningen, University of Groningen, Groningen, The Netherlands"/>
      <meta name="citation_author" content="Eveline S. J. M. de Bont"/>
            <meta name="citation_author_institution" content="Division of Pediatric Oncology, Department of Pediatrics, Beatrix Children's Hospital, University Medical Center Groningen, University of Groningen, Groningen, The Netherlands"/>

    <meta name="citation_date" content="2008/2/20"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0001656.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e1656"/>
    <meta name="citation_issue" content="2"/>
    <meta name="citation_volume" content="3"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Module networks: identifying regulatory modules and their condition-specific regulators from gene expression data.; citation_author=E Segal; citation_author=M Shapira; citation_author=A Regev; citation_author=D Peter; citation_author=D Botstein; citation_journal_title=Nat Genet; citation_volume=34; citation_number=1; citation_pages=166-176; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Cluster analysis and display of genome-wide expression patterns.; citation_author=MB Eisen; citation_author=PT Spellman; citation_author=PO Brown; citation_author=D Botstein; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=95; citation_number=2; citation_pages=14863-14868; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Broad patterns of gene expression revealed by clustering analysis of tumor and normal colon tissues probed by oligonucleotide arrays.; citation_author=U Alon; citation_author=N Barkai; citation_author=DA Notterman; citation_author=K Gish; citation_author=S Ybarra; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=96; citation_number=3; citation_pages=6745-6750; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Beyond the sequence: cellular organization of genome function.; citation_author=T Misteli; citation_journal_title=Cell; citation_volume=128; citation_number=4; citation_pages=787-800; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Exploring the conditional coregulation of yeast gene expression through fuzzy k-means clustering.; citation_author=AP Gasch; citation_author=MB Eisen; citation_journal_title=Genome Biol; citation_volume=3; citation_number=5; citation_pages=RESEARCH0059; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Revealing modular organization in the yeast transcriptional network.; citation_author=J Ihmels; citation_author=G Friedlander; citation_author=S Bergmann; citation_author=O Sarig; citation_author=Y Ziv; citation_journal_title=Nat Genet; citation_volume=31; citation_number=6; citation_pages=370-377; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Pitfalls in the use of DNA microarray data for diagnostic and prognostic classification.; citation_author=R Simon; citation_author=MD Radmacher; citation_author=K Dobbin; citation_author=LM McShane; citation_journal_title=J Natl Cancer Inst; citation_volume=95; citation_number=7; citation_pages=14-18; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Reliability and reproducibility issues in DNA microarray measurements.; citation_author=S Draghici; citation_author=P Khatri; citation_author=AC Eklund; citation_author=Z Szallasi; citation_journal_title=Trends Genet; citation_volume=22; citation_number=8; citation_pages=101-109; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Singular value decomposition for genome-wide expression data processing and modeling.; citation_author=O Alter; citation_author=PO Brown; citation_author=D Botstein; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=97; citation_number=9; citation_pages=10101-10106; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Insulators: exploiting transcriptional and epigenetic mechanisms.; citation_author=M Gaszner; citation_author=G Felsenfeld; citation_journal_title=Nat Rev Genet; citation_volume=7; citation_number=10; citation_pages=703-713; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=DNA methylation and histone deacetylation in the control of gene expression: basic biochemistry to human development and disease.; citation_author=A El-Osta; citation_author=AP Wolffe; citation_journal_title=Gene Expr; citation_volume=9; citation_number=11; citation_pages=63-75; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Replication and transcription: shaping the landscape of the genome.; citation_author=L Chakalova; citation_author=E Debrand; citation_author=JA Mitchell; citation_author=CS Osborne; citation_author=P Fraser; citation_journal_title=Nat Rev Genet; citation_volume=6; citation_number=12; citation_pages=669-677; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=The organization of replication and transcription.; citation_author=PR Cook; citation_journal_title=Science; citation_volume=284; citation_number=13; citation_pages=1790-1795; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Chromosome territories, nuclear architecture and gene regulation in mammalian cells.; citation_author=T Cremer; citation_author=C Cremer; citation_journal_title=Nat Rev Genet; citation_volume=2; citation_number=14; citation_pages=292-301; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Cell biology: chromosome territories.; citation_author=KJ Meaburn; citation_author=T Misteli; citation_journal_title=Nature; citation_volume=445; citation_number=15; citation_pages=379-781; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=The human transcriptome map: clustering of highly expressed genes in chromosomal domains.; citation_author=H Caron; citation_author=B van Schaik; citation_author=M van der Mee; citation_author=F Baas; citation_author=G Riggins; citation_journal_title=Science; citation_volume=291; citation_number=16; citation_pages=1289-1292; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Gene Expression Omnibus: NCBI gene expression and hybridization array data repository.; citation_author=R Edgar; citation_author=M Domrachev; citation_author=AE Lash; citation_journal_title=Nucleic Acids Res; citation_volume=30; citation_number=17; citation_pages=207-210; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=A comparison of normalization methods for high density oligonucleotide array data based on variance and bias.; citation_author=BM Bolstad; citation_author=RA Irizarry; citation_author=M Astrand; citation_author=TP Speed; citation_journal_title=Bioinformatics; citation_volume=19; citation_number=18; citation_pages=185-193; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Multivariate Statistical Inference and Applications.; citation_author=A Rencher; citation_number=19; citation_date=1998; citation_publisher=John Wiley & Sons, Inc; " />
      <meta name="citation_reference" content="citation_title=A tutorial on principal component analysis.; citation_author=LI Smith; citation_number=20; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Conditions of Measurement Affecting Reliability in Statistical Therories Of Mental Test Scores, edited by Frederick Mosteller Addison-Wesley Pub. Co., Reading, Massachusetts.; citation_author=FM Lord; citation_author=MR Novick; citation_number=21; citation_pages=129-150; citation_date=1968; " />
      <meta name="citation_reference" content="citation_title=GeneTrailâadvanced gene set enrichment analysis.; citation_author=C Backes; citation_author=A Keller; citation_author=J Kuentzer; citation_author=B Kneissl; citation_author=N Comtesse; citation_journal_title=Nucleic Acids Res; citation_number=22; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Gene set enrichment analysis: a knowledge-based approach for interpreting genome-wide expression profiles.; citation_author=A Subramanian; citation_author=P Tamayo; citation_author=VK Mootha; citation_author=S Mukherjee; citation_author=BL Ebert; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=102; citation_number=23; citation_pages=15545-15550; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Controlling the False Discovery Rate - A Practical and Powerful Approach to Multiple Testing.; citation_author=Y Benjamini; citation_author=Y Hochberg; citation_journal_title=Journal of the Royal Statistical Society Series B-Methodological; citation_volume=57; citation_number=24; citation_pages=289-300; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Gene ontology: tool for the unification of biology. The Gene Ontology Consortium.; citation_author=M Ashburner; citation_author=CA Ball; citation_author=JA Blake; citation_author=D Botstein; citation_author=H Butler; citation_journal_title=Nat Genet; citation_volume=25; citation_number=25; citation_pages=25-29; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Open source clustering software.; citation_author=MJ de Hoon; citation_author=S Imoto; citation_author=J Nolan; citation_author=S Miyano; citation_journal_title=Bioinformatics; citation_volume=20; citation_number=26; citation_pages=1453-1454; citation_date=2004; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001656" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="A New Perspective on Transcriptional System Regulation (TSR): Towards TSR Profiling"/>
    <meta name="twitter:description" content="It has been hypothesized that the net expression of a gene is determined by the combined effects of various transcriptional system regulators (TSRs). However, characterizing the complexity of regulation of the transcriptome is a major challenge. Principal component analysis on 17,550 heterogeneous human microarray experiments revealed that 50 orthogonal factors (hereafter called TSRs) are able to capture 64% of the variability in expression in a wide range of experimental conditions and tissues. We identified gene clusters controlled in the same direction and show that gene expression can be conceptualized as a process influenced by a fairly limited set of TSRs. Furthermore, TSRs can be linked to biological functions, as we demonstrate a strong relation between TSR-related gene clusters and biological functionality as well as cellular localization, i.e. gene products of similarly regulated genes by a specific TSR are located in identical parts of a cell. Using 3,934 diverse mouse microarray experiments we found striking similarities in transcriptional system regulation between human and mouse. Our results give biological insights into regulation of the cellular transcriptome and provide a tool to characterize expression profiles with highly reliable TSRs instead of thousands of individual genes, leading to a &gt;500-fold reduction of complexity with just 50 TSRs. This might open new avenues for those performing gene expression profiling studies."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0001656.g007"/>

  <meta property="og:title" content="A New Perspective on Transcriptional System Regulation (TSR): Towards TSR Profiling" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001656" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=7917'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=1524'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=6691&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0001656">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001656" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001656&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    A New Perspective on Transcriptional System Regulation (TSR): Towards TSR Profiling
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Rudolf S. N. Fehrmann
              <span class="equal-contrib"
                    title="These authors contributed equally to this work">equal contributor</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">

                <p><span class="equal-contrib" title="These authors contributed equally to this work">equal contributor</span>
                Contributed equally to this work with: Rudolf S. N. Fehrmann, Hendrik J. M. de Jonge, Gerard J. te Meerman, Eveline S. J. M. de Bont</p>

              
              

                <p>Affiliations:
                  Department of Genetics, University Medical Center Groningen, University of Groningen, Groningen, The Netherlands, 
                  Department of Gynecology, University Medical Center Groningen, University of Groningen, Groningen, The Netherlands, 
                  Department of Medical Oncology, University Medical Center Groningen, University of Groningen, Groningen, The Netherlands
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Hendrik J. M. de Jonge
              <span class="equal-contrib"
                    title="These authors contributed equally to this work">equal contributor</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">

                <p><span class="equal-contrib" title="These authors contributed equally to this work">equal contributor</span>
                Contributed equally to this work with: Rudolf S. N. Fehrmann, Hendrik J. M. de Jonge, Gerard J. te Meerman, Eveline S. J. M. de Bont</p>

              
              

                <p>Affiliation:
                  Division of Pediatric Oncology, Department of Pediatrics, Beatrix Children's Hospital, University Medical Center Groningen, University of Groningen, Groningen, The Netherlands
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Arja ter Elst, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Division of Pediatric Oncology, Department of Pediatrics, Beatrix Children's Hospital, University Medical Center Groningen, University of Groningen, Groningen, The Netherlands
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            AndrÃ© de Vries, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Genetics, University Medical Center Groningen, University of Groningen, Groningen, The Netherlands
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Anne G. P. Crijns, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Gynecology, University Medical Center Groningen, University of Groningen, Groningen, The Netherlands
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Alida C. Weidenaar, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Division of Pediatric Oncology, Department of Pediatrics, Beatrix Children's Hospital, University Medical Center Groningen, University of Groningen, Groningen, The Netherlands
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Frans Gerbens, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Genetics, University Medical Center Groningen, University of Groningen, Groningen, The Netherlands
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Steven de Jong, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Medical Oncology, University Medical Center Groningen, University of Groningen, Groningen, The Netherlands
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Ate G. J. van der Zee, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Gynecology, University Medical Center Groningen, University of Groningen, Groningen, The Netherlands
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Elisabeth G. E. de Vries, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Medical Oncology, University Medical Center Groningen, University of Groningen, Groningen, The Netherlands
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Willem A. Kamps, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Division of Pediatric Oncology, Department of Pediatrics, Beatrix Children's Hospital, University Medical Center Groningen, University of Groningen, Groningen, The Netherlands
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Robert M. W. Hofstra, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Genetics, University Medical Center Groningen, University of Groningen, Groningen, The Netherlands
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Gerard J. te Meerman
              <span class="equal-contrib"
                    title="These authors contributed equally to this work">equal contributor</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">

                <p><span class="equal-contrib" title="These authors contributed equally to this work">equal contributor</span>
                Contributed equally to this work with: Rudolf S. N. Fehrmann, Hendrik J. M. de Jonge, Gerard J. te Meerman, Eveline S. J. M. de Bont</p>

              
              

                <p>Affiliation:
                  Department of Genetics, University Medical Center Groningen, University of Groningen, Groningen, The Netherlands
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Eveline S. J. M. de Bont
              <span class="equal-contrib"
                    title="These authors contributed equally to this work">equal contributor</span>
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">

                <p><span class="equal-contrib" title="These authors contributed equally to this work">equal contributor</span>
                Contributed equally to this work with: Rudolf S. N. Fehrmann, Hendrik J. M. de Jonge, Gerard J. te Meerman, Eveline S. J. M. de Bont</p>

              
              <p>*E-mail: <a href="mailto:e.de.bont@bkk.umcg.nl">e.de.bont@bkk.umcg.nl</a></p>

                <p>Affiliation:
                  Division of Pediatric Oncology, Department of Pediatrics, Beatrix Children's Hospital, University Medical Center Groningen, University of Groningen, Groningen, The Netherlands
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: February 20, 2008</li>
    <li>DOI: 10.1371/journal.pone.0001656</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0001656" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0001656" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001656" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0001656" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0001656-g001" data-doi="info:doi/10.1371/journal.pone.0001656" data-uri="info:doi/10.1371/journal.pone.0001656.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001656.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001656-g002" data-doi="info:doi/10.1371/journal.pone.0001656" data-uri="info:doi/10.1371/journal.pone.0001656.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001656.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001656-g003" data-doi="info:doi/10.1371/journal.pone.0001656" data-uri="info:doi/10.1371/journal.pone.0001656.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001656.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001656-g004" data-doi="info:doi/10.1371/journal.pone.0001656" data-uri="info:doi/10.1371/journal.pone.0001656.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001656.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001656-g005" data-doi="info:doi/10.1371/journal.pone.0001656" data-uri="info:doi/10.1371/journal.pone.0001656.g005" title="Figure 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001656.g005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001656-g006" data-doi="info:doi/10.1371/journal.pone.0001656" data-uri="info:doi/10.1371/journal.pone.0001656.g006" title="Figure 6">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001656.g006&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001656-g007" data-doi="info:doi/10.1371/journal.pone.0001656" data-uri="info:doi/10.1371/journal.pone.0001656.g007" title="Figure 7">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001656.g007&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001656">Reader Comments (0)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0001656" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1"></a><p>It has been hypothesized that the net expression of a gene is determined by the combined effects of various transcriptional system regulators (TSRs). However, characterizing the complexity of regulation of the transcriptome is a major challenge. Principal component analysis on 17,550 heterogeneous human microarray experiments revealed that 50 orthogonal factors (hereafter called TSRs) are able to capture 64% of the variability in expression in a wide range of experimental conditions and tissues. We identified gene clusters controlled in the same direction and show that gene expression can be conceptualized as a process influenced by a fairly limited set of TSRs. Furthermore, TSRs can be linked to biological functions, as we demonstrate a strong relation between TSR-related gene clusters and biological functionality as well as cellular localization, <em>i.e.</em> gene products of similarly regulated genes by a specific TSR are located in identical parts of a cell. Using 3,934 diverse mouse microarray experiments we found striking similarities in transcriptional system regulation between human and mouse. Our results give biological insights into regulation of the cellular transcriptome and provide a tool to characterize expression profiles with highly reliable TSRs instead of thousands of individual genes, leading to a &gt;500-fold reduction of complexity with just 50 TSRs. This might open new avenues for those performing gene expression profiling studies.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>Fehrmann RSN, de Jonge HJM, ter Elst A, de Vries A, Crijns AGP, et al.  (2008) A New Perspective on Transcriptional System Regulation (TSR): Towards TSR Profiling. PLoS ONE 3(2):
          e1656.
            doi:10.1371/journal.pone.0001656</p><p><strong>Academic Editor: </strong>Stefan Maas, Lehigh University, United States of America</p><p><strong>Received:</strong> November 26, 2007; <strong>Accepted:</strong> January 17, 2008; <strong>Published:</strong> February 20, 2008</p><p><strong>Copyright:</strong> Â© 2008 Fehrmann et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>H.J.M. de J is partially supported by the Foundation for Pediatric Oncology Groningen and E.S.J.M. de B is supported by a grant from the Dutch Cancer Society (grant number 3661). The funders had no role in the design and conduct of the study, in the collection, analysis, and interpretation of the data, and in the preparation, review, or approval of the manuscript. An european patent application has been filed concerning diagnostic use of TSR's by the University of Groningen and the University Medical Center Groningen. The first and the last two authors could receive partial benefits of patent revenu in the future.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>Biological systems have a layered complexity and it is known that a cell's activity is modulated by a network of co-regulated gene clusters.<a href="#pone.0001656-Segal1">[1]</a> Such modules are characterized by clusters of transcriptionally correlated genes, most often with related functions.<a href="#pone.0001656-Eisen1">[2]</a> A number of studies using clustering algorithms based on similar expression patterns provided valuable clues about which strongly expressed genes are co-regulated in a small, specific set of experimental conditions.<a href="#pone.0001656-Segal1">[1]</a>â<a href="#pone.0001656-Alon1">[3]</a></p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>However, clustering algorithms are less effective when applied to large datasets of heterogeneous material. Basic clustering algorithms assign each gene to a single cluster of co-regulated genes, whereas it is hypothesized that the net expression of a gene is determined by the combined effects of various transcriptional system regulators (TSRs).<a href="#pone.0001656-Misteli1">[4]</a>â<a href="#pone.0001656-Ihmels1">[6]</a> In addition, each level of transcriptional regulation may only be active in certain phenotypes and the remaining phenotypes will contribute to noise.<a href="#pone.0001656-Ihmels1">[6]</a> In contrast, principal component analysis (PCA) on a large heterogeneous set could enable us to use correlation structures of not only strong but also weakly expressed genes and could provide a global picture of the dynamics of gene expression on various transcriptional regulation levels. It could allow individual genes to be classified into groups that are similarly controlled by a specific TSR.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>Unraveling the complexity of regulation of the transcriptome is a major challenge; as in principle an infinite number of TSRs could be needed to control the expression of thousands of genes ultimately leading to the large diversity seen in cellular phenotypes. In this study we identified a structure of transcriptional regulation by analyzing 17,550 heterogeneous microarray experiments. We found that the number of orthogonal factors needed to explain most of the variability in expression is fairly limited, even in a wide range of experimental conditions, tissues and even across species. Furthermore, using several different models, we show that these TSRs have biological relevance and yield reliable summary measurements of gene expression that are applicable to different tissue types as well as organisms.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3>
<h4>Transcriptional system regulators</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>Insight into the complexity of the regulation of the transcriptome was revealed by PCA on the expression correlation matrix of 13,032 genes in 17,550 human miscellaneous expression arrays. PCA demonstrated that 64% of the variance in expression of 13,032 genes was explained by only 50 orthogonal factors, called TSRs, which means a &gt;500-fold reduction in complexity (<a href="#pone-0001656-g001">Fig. 1A</a>). Similar results were observed in mice where 50 TSRs explained 71% of the variance in expression of 9,062 genes in 3,934 arrays (<a href="#pone-0001656-g001">Fig. 1A</a>). Moreover, <a href="#pone-0001656-g001">Figure 1A</a> shows that the pattern of the percentage explained variance per TSR is highly similar between human and mouse. <a href="#pone.0001656.s001">Tables S1</a> and <a href="#pone.0001656.s002">S2</a> give factor loadings for the first 50 TSRs in human and mouse, respectively.</p>
<div class="figure" id="pone-0001656-g001"><div class="img"><a name="pone-0001656-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001656.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001656" data-uri="info:doi/10.1371/journal.pone.0001656.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001656.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001656.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001656.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001656.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001656.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001656.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001656.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001656.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001656.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Explained variance and reliability of the first 50 transcriptional system regulators (TSRs).</span></strong></p><a id="article1.body1.sec2.sec1.fig1.caption1.p1" name="article1.body1.sec2.sec1.fig1.caption1.p1"></a><p>Panel A shows the explained variance for the first 50 TSRs in human and mouse. The percentage explained variance is depicted for each TSR. The cumulative percentage explained variance for the first 50 TSRs is 64% in human and 71% in mouse. Panel B shows a heat map where each box represents the Pearson correlation coefficient between TSRs generated from sets A and B, each using half of the data. Correlation coefficients of 0 and 1 are represented by black and red, respectively. Panel C shows the split-half correlations for the first 50 TSRs. (A TSR is a weighted sum of genes and a so-called TSR score can be calculated for each sample. In the split-half method, genes of a TSR score are split into two random parts and the resulting partial TSR scores are correlated (Pearson).)</p>
<span>doi:10.1371/journal.pone.0001656.g001</span></div>

<h4>Reliability of TSRs</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>To evaluate whether the identified TSRs depend on the specific set of selected microarray experiments, the human microarray data were randomly split into two halves and then two sets (A and B) of TSRs were generated, each using only half of the samples. <a href="#pone-0001656-g001">Figure 1B</a> contains a heat map showing correlation coefficients between TSRs generated in sets A and B. TSR1 generated in set A and TSR1 in set B correlated significantly (<em>R</em> = .999; <em>P</em>&lt;1.0Ã10<sup>â16</sup>), indicating highly similar control of identical genes by TSR1 in both sets. Furthermore, the diagonal line in <a href="#pone-0001656-g001">Figure 1B</a> shows that TSRs generated in both sets were highly similar in their control of identical genes. For a few TSRs, the relative position in the order they were found was seen to be switched, but the same directions of variation were identified. These results indicate that the TSRs were reliably identified and were not artifacts due to sample selection. To further investigate whether the identified TSRs were not artifacts due to gene selection, we applied the split-half method to each TSR. <a href="#pone-0001656-g001">Figure 1C</a> shows the split-half correlations for the first 50 TSRs, which were high (&gt;0.91), indicating their high internal consistency. In sum, the identified TSRs were robust and not artifacts due to selection of genes and/or tissues.</p>


<h4>Biological significance of TSRs</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>To validate that the identified TSRs are not merely mathematical constructs but contain biological coherence and to gain more biological insight into the regulation of the transcriptome, we performed Gene Set Enrichment Analysis (GSEA). The hypothesis is that our identified TSRs are related to known biologically related gene clusters represented by GO ontologies. Here we describe the GSEA results for the first, second and fiftieth TSR as examples. GSEA results for the first 25 TSRs are available in the supplementary data online. Among the genes most strongly influenced by TSR1, TSR2 and TSR50, many GO ontologies were significantly enriched (n = 488, n = 1157 and n = 119, respectively). <a href="#pone-0001656-g002">Figure 2</a> shows the most significant biological processes per TSR according to the GO ontology classification on either side of it. The graphs show the enrichment score as a function of the index in the list (<em>x</em>-axis) of genes ranked according to the correlation between their expression and a TSR score (factor loading). Red graphs with a âmountain-like shapeâ illustrate a specific GO ontology predominantly containing top ranked genes. In contrast, green graphs with a âvalley-like shapeâ illustrate a specific GO ontology predominantly containing bottom ranked genes. When genes belonging to a specific GO ontology are not top or bottom ranked but randomly distributed in the ranked list the graph will have a âzigzagâ shape. Furthermore, <a href="#pone-0001656-g002">Figure 2</a> shows that TSRs have the capacity to influence the expression of genes involved in specific biological processes in opposite directions, <em>e.g.</em> TSR1 regulates genes belonging to GO ontology âprogression through M phaseâ <em>vs.</em> âion transportâ, TSR2 genes belonging to GO ontology âcell cycle checkpointâ <em>vs.</em> âthe cell morphogenesisâ and TSR50 genes belonging to GO ontology âstriated muscle contractionâ <em>vs.</em> âcomplement activationâ. Biological processes represented by GO ontologies can theoretically be influenced by more then one TSR; the contrast seen in TSR1 âprogression through M phaseâ <em>vs.</em> âion transportâ was also seen in TSR3 for example. All enrichments for GO ontologies representing these six biological processes were highly significant; <em>P</em>&lt;10<sup>â8</sup>. In addition, the encoded proteins of genes controlled in opposite directions (<em>i.e</em>. top vs. bottom ranked genes) by a TSR are generally located in other compartments of a cell (<em>e.g.</em> Golgi, mitochondrion, nucleus, etc.), as shown for TSR1 in <a href="#pone-0001656-g003">Figure 3</a> for example. Panel A shows that the encoded proteins of the top-ranked genes of TSR1 (the limit of 200 was arbitrarily chosen) are generally located within the plasma membrane, whereas the 200 bottom-ranked genes of TSR1 are generally located within the nucleus of the cell. This is also visualized in Panel B, where the green âvalley-like shapeâ graph shows that the GO ontology for cellular localization ânucleusâ is enriched among the bottom-ranked genes. In contrast the red âmountain-like shapeâ graph shows the enrichment of the GO ontology âplasma membraneâ at the top-ranked genes.</p>
<div class="figure" id="pone-0001656-g002"><div class="img"><a name="pone-0001656-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001656.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001656" data-uri="info:doi/10.1371/journal.pone.0001656.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001656.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001656.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001656.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001656.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001656.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001656.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001656.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001656.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001656.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Biological significance of the transcriptional system regulators (TSRs).</span></strong></p><a id="article1.body1.sec2.sec3.fig1.caption1.p1" name="article1.body1.sec2.sec3.fig1.caption1.p1"></a><p>The most significant biological processes for either side of the respective TSR according to the GO ontology classification are shown for TSR1, TSR2 and TSR50 in panels A, B and C, respectively. Graphs depict the running sum statistics when applying gene set enrichment analysis. The running sum (<em>y</em>-axis) is shown as a function of the index in the list (<em>x</em>-axis) of genes ranked in ascending order according to their factor loadings within TSR. The red vs. green graphs show the biological coherence of opposing regulated gene clusters controlled by TSRs.</p>
<span>doi:10.1371/journal.pone.0001656.g002</span></div><div class="figure" id="pone-0001656-g003"><div class="img"><a name="pone-0001656-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001656.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001656" data-uri="info:doi/10.1371/journal.pone.0001656.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001656.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001656.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001656.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001656.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001656.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001656.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001656.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001656.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001656.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Relationship between cellular localization of genes controlled in opposite directions by TSR1.</span></strong></p><a id="article1.body1.sec2.sec3.fig2.caption1.p1" name="article1.body1.sec2.sec3.fig2.caption1.p1"></a><p>Panel A shows the cellular localization of the 400 most strongly controlled genes (highest factor loadings). The colors represent genes with positive (green) and negative (red) factor loadings. Panel B shows the results of gene set enrichment analysis with regard to cellular localization of the gene products. Graphs show the running sum statistics when applying gene set enrichment analysis. The running sum (<em>y</em>-axis) is shown as a function of the index in the list (<em>x</em>-axis) of genes ranked in ascending order according to their factor loadings within a TSR.</p>
<span>doi:10.1371/journal.pone.0001656.g003</span></div>

<h4>Similarities in transcriptional system regulation between human and mouse</h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>PCA on the combined two-species dataset consisting of 3,934 human and 3,934 mouse arrays was performed to assess the similarity of the structure in transcription regulation between human and mouse. PCA revealed that again 50 principal components (PCs) explained ~73% of the total variance in combined human and mouse gene expression. In this specific two-species dataset the first PC (PC1) explained ~25% of the total variance in expression. The distribution of PC1 scores for human samples showed no overlap with PC1 scores for mouse samples, suggesting that PC1 summarizes the variation in expression caused by species differences (<a href="#pone.0001656.s004">Figure S1</a>). Biological processes such as RNA processing, ion transport and primary metabolism were enriched in PC1 using GSEA analysis. Except for PC1, all PCs showed a strong overlap between human and mouse PC scores, suggesting that gene expression in human and mouse is similarly influenced by regulatory processes influencing evolutionary related gene clusters in the same direction.</p>


<h4>Mapping human and mouse TSRs</h4>
<a id="article1.body1.sec2.sec5.p1" name="article1.body1.sec2.sec5.p1"></a><p>The results above suggest a high similarity in the structure of transcriptional regulation between human and mouse. To assess similarities in regulation of biological processes, we mapped TSRs generated in the human dataset to TSRs generated in the mouse dataset. The Spearman rank correlation coefficients between human and mouse TSRs showed that human TSR1 was most strongly correlated with mouse TSR2 (<em>R</em> = .489, <em>P</em> = 1.0Ã10<sup>â6</sup>). Correlation coefficients between the first 25 human and mouse TSRs are given in <a href="#pone.0001656.s003">Table S3</a>. As an example of the strong resemblance in transcriptional system regulation between human and mouse, <a href="#pone-0001656-g004">Figure 4</a> shows that identical biological processes are enriched and similarly controlled in one direction (human TSR1 <em>vs</em>. mouse TSR2).</p>
<div class="figure" id="pone-0001656-g004"><div class="img"><a name="pone-0001656-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001656.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001656" data-uri="info:doi/10.1371/journal.pone.0001656.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001656.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001656.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001656.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001656.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001656.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001656.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001656.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001656.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001656.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>Similarities in transcriptional system regulation between human and mouse.</span></strong></p><a id="article1.body1.sec2.sec5.fig1.caption1.p1" name="article1.body1.sec2.sec5.fig1.caption1.p1"></a><p>Panels A and B show the results of gene set enrichment analysis for six biological processes for human TSR1 and mouse TSR2, respectively. Graphs show the running sum statistics. The running sum (<em>y</em>-axis) is shown as a function of the index in the list (<em>x</em>-axis) of genes ranked in ascending order according to their factor loadings with a TSR. The red vs. green graphs show the biological coherence of regulated gene clusters controlled by TSRs in both species.</p>
<span>doi:10.1371/journal.pone.0001656.g004</span></div>

<h4>Regional control of chromosomal domains</h4>
<a id="article1.body1.sec2.sec6.p1" name="article1.body1.sec2.sec6.p1"></a><p>Next, in order to further characterize the transcriptional system regulation, we assessed whether genes similarly controlled by a specific TSR also cluster within chromosomal regions. <a href="#pone-0001656-g005">Figure 5</a> shows regional factor loading profiles for TSR1. GSEA results in terms of chromosomal distribution of similarly controlled gene regions for other TSRs are available in the supplementary data. An application of a moving median with a window size of 20 genes clearly shows differences for the chromosomal regions. The Y-chromosome is not depicted as the number of genes in the dataset located on the Y-chromosome was less than 20, so a moving median could not be calculated. Several chromosomes (<em>e.g.</em> 1 and 11) have large regions of genes with predominantly positive factor loadings, interspersed with regions where genes have predominantly negative factor loadings. In contrast, chromosomes 4, 13 and 22 show hardly any regions of genes inversely controlled by TSR1, but, more remarkably, all these gene regions seem to be regulated in only one direction.</p>
<div class="figure" id="pone-0001656-g005"><div class="img"><a name="pone-0001656-g005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001656.g005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001656" data-uri="info:doi/10.1371/journal.pone.0001656.g005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001656.g005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001656.g005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001656.g005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001656.g005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001656.g005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001656.g005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001656.g005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001656.g005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001656.g005.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 5.  <span>Regional factor loading profiles for 23 chromosomes.</span></strong></p><a id="article1.body1.sec2.sec6.fig1.caption1.p1" name="article1.body1.sec2.sec6.fig1.caption1.p1"></a><p>Factor loadings for TSR1 are shown on chromosomes as a moving median with a window size of 20 genes. Chromosome position is depicted on the <em>x</em>-axis and factor loadings for gene regions are given on the <em>y</em>-axis. Bars above or below the middle line represent inversely regulated chromosomal regions.</p>
<span>doi:10.1371/journal.pone.0001656.g005</span></div>

<h4>Gene clustering based on TSR interaction</h4>
<a id="article1.body1.sec2.sec7.p1" name="article1.body1.sec2.sec7.p1"></a><p>We clustered genes based on factor loadings with the first 50 TSRs in order to elucidate the dynamics of gene expression regulation, in which individual genes are classified into groups with similar regulation patterns. Gene clusters with distinct patterns of regulation were observed; clusters predominantly controlled by the first TSRs and clusters with a more diffuse pattern of TSR regulation (<em>e.g.</em> genes of which the products are involved in the biosynthesis of proteins from mRNA molecules). Furthermore, as functionally related genes often exhibit expression patterns that are correlated, we expected to observe clusters of functionally related genes based on TSR interaction. For example, <a href="#pone-0001656-g006">Figure 6A and 6B</a> show gene clusters with a strong biological relationship, <em>i.e.</em> the human leukocyte antigen system (HLA). All clustering results are available as supplementary data and can be depicted with Java TreeView, which can be downloaded from <a href="http://jtreeview.sourceforge.net/">http://jtreeview.sourceforge.net/</a>.</p>
<div class="figure" id="pone-0001656-g006"><div class="img"><a name="pone-0001656-g006" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001656.g006&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001656" data-uri="info:doi/10.1371/journal.pone.0001656.g006"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001656.g006&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001656.g006/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001656.g006/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001656.g006/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001656.g006/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001656.g006.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001656.g006/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001656.g006/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001656.g006.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 6.  <span>Heat map examples of gene clusters based on factor loadings with the first 50 transcriptional system regulators (TSRs).</span></strong></p><a id="article1.body1.sec2.sec7.fig1.caption1.p1" name="article1.body1.sec2.sec7.fig1.caption1.p1"></a><p>Each box in the heat map represents the factor loading of a gene with one of the TSRs (negative = red; positive = green). The first row shows the average factor loadings for the depicted cluster.</p>
<span>doi:10.1371/journal.pone.0001656.g006</span></div>

<h4>Sample clustering based on TSR scores</h4>
<a id="article1.body1.sec2.sec8.p1" name="article1.body1.sec2.sec8.p1"></a><p>Samples from the publicly available human body index were clustered to assess whether similar tissue samples have similar patterns in TSR scores. Clustering results for all samples are given in the supplementary data. Samples with identical tissue origins showed strong clustering, <em>e.g.</em> <a href="#pone-0001656-g007">Figure 7</a> shows clustering of liver tissue samples as well as kidney samples.</p>
<div class="figure" id="pone-0001656-g007"><div class="img"><a name="pone-0001656-g007" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001656.g007&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001656" data-uri="info:doi/10.1371/journal.pone.0001656.g007"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001656.g007&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001656.g007/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001656.g007/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001656.g007/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001656.g007/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001656.g007.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001656.g007/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001656.g007/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001656.g007.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 7.  <span>Heat map example of sample clusters based on transcriptional system regulator (TSR) scores.</span></strong></p><a id="article1.body1.sec2.sec8.fig1.caption1.p1" name="article1.body1.sec2.sec8.fig1.caption1.p1"></a><p>Each box in the heat map represents the score for a sample (negative = red; positive = green). The first row shows the average TSR scores for the depicted cluster.</p>
<span>doi:10.1371/journal.pone.0001656.g007</span></div>
</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p>Principal component analysis (PCA) on a large number of heterogeneous microarray experiments showed that a maximum of 50 statistically independent transcriptional system regulators (TSRs) can explain the vast majority of biological variance in gene expression in human as well as in mouse. Furthermore, we identified clusters of genes which expression is influenced in the same direction by specific TSRs and showed that gene expression can be conceptualized as a process influenced by a limited set of TSRs.</p>
<a id="article1.body1.sec3.p2" name="article1.body1.sec3.p2"></a><p>In microarray studies small sample sizes often present a major problem, such as overfitting, <em>i.e.</em> finding a discriminatory pattern by chance, which may occur when large numbers of genes are used to discriminate a small number of phenotypes.<a href="#pone.0001656-Simon1">[7]</a> Since a TSR is a weighted sum of genes, a TSR score can be calculated for each observed expression array. This data reduction allows us to characterize the expression profile of an individual microarray with 50 highly reliable TSR scores (TSR-profiling) instead of using thousands of individual genes (the software to compress expression array data in TSR scores is available at the supplementary <a href="http://website: http://129.125.155.240/tsr/">website: http://129.125.155.240/tsr/</a>). As an example we showed that similar tissues clustered together based on the first 50 TSR scores (<a href="#pone-0001656-g007">Fig. 7</a>). These results strongly suggest that the origin of tissues can be uncovered with the use of TSR scores. This could be applied for the identification of cancer metastases of unknown origin. The advantage is that a limited set of TSRs makes it is possible to analyze even small gene expression datasets with much less danger of false-positive results due to overfitting. Furthermore, TSR scores can be used to identify systematic changes of expression in gene clusters consisting of genes with small but systematic fold changes between phenotypes. This is important as we often do not know whether large fold changes in individual genes will have more biological relevance than smaller but coordinate fold changes in a set of genes.</p>
<a id="article1.body1.sec3.p3" name="article1.body1.sec3.p3"></a><p>Inherent to microarray experiments the measurement of gene expression is composed of biological signals and experimental noise.<a href="#pone.0001656-Draghici1">[8]</a> We assumed that at least the first 50 TSRs, accounting for ~70% of the information present in the entire dataset, capture the majority of relevant biological signals and that subsequent ones are likely to consist of noise and experimental artifacts, a principle which has also been described by Alter et al.<a href="#pone.0001656-Alter1">[9]</a> Normalizing gene expression data by filtering out these latter TSRs enables a meaningful comparison to be made of the expression of different genes across different arrays in different experiments (<a href="#pone-0001656-g006">Fig. 6</a>). Moreover, in addition to compare genes on the basis of their net expression, genes can now also be compared by their similarity in regulation of any chosen subset of TSRs. Therefore, the function of genes with a provisional status could be elucidated by looking at genes with known biological functions which are similarly influenced by our identified TSRs.</p>
<a id="article1.body1.sec3.p4" name="article1.body1.sec3.p4"></a><p>TSRs can also be linked to biological functions as we have shown a strong relation between TSR-related gene clusters and biological coherence in terms of functionality as well as cellular localization (<a href="#pone-0001656-g002">Figures 2</a> and <a href="#pone-0001656-g003">3</a>). Genes encoding for proteins located in the nucleus are influenced in opposite directions compared to genes encoding for proteins located in the plasma membrane. This might suggest that there is a regulatory process which could down regulate the expression of ânuclearâ genes when up-regulating âplasma membraneâ genes and vice versa. However, of note, a biological process represented by a GO ontology is not necessarily influenced by one specific TSR alone but can be influenced by more then one TSR; e.g. the contrast nucleus <em>vs.</em> membrane was seen in several TSRs. Although we do not know the nature of the transcriptional regulatory processes represented by our TSRs, this study provides insights into which specific biological processes are influenced in opposite directions. Further studies are needed to unravel the underlying nature and interplay of these TSRs.</p>
<a id="article1.body1.sec3.p5" name="article1.body1.sec3.p5"></a><p>Expression profiling of relevant disease tissues might help in candidate gene selection. However, such studies are often problematic, as relevant human tissue is hard to obtain. To overcome these sampling problems, the use of comparable mouse tissues seems a good option. <a href="#pone-0001656-g004">Figure 4</a> shows a highly similar pattern of influence on gene expression between TSR1 in human and TSR2 in mice. Identical opposing GO ontologies are influenced in the same directions. Our results suggest a high homology between the transcriptional regulation of human and mouse. It is noteworthy that the first principal component from the PCA on the combined two-species dataset representing species differences revealed that GO ontologies such as primary metabolism and RNA processing were highly enriched. This is in line with expected metabolic differences between human and mouse. By filtering out the first principal component that represents expression differences between the species and/or platforms, will enable one with the remaining differences to translate mouse data to the human situation and <em>vice versa</em>, because species-specific variance in expression can be recognized and eliminated by subtraction.</p>
<a id="article1.body1.sec3.p6" name="article1.body1.sec3.p6"></a><p>So far several levels in transcriptional system regulation have been reported, <em>e.g.</em> classical DNA sequence regulators, epigenetic modifications, spatial and temporal organization of nuclear processes and chromosomes, organization of chromatin into higher-order domains, transcription factors and microRNAs.<a href="#pone.0001656-Gaszner1">[10]</a>â<a href="#pone.0001656-Meaburn1">[15]</a> These known levels in transcriptional system regulation may be represented by a combination of the TSRs identified in this study. In addition, some of the identified TSRs may represent other, as yet undiscovered levels of transcriptional system regulation. Interestingly, our work demonstrated that TSRs regulate genes from chromosomal regions predominantly in the same direction. This observation was most strongly pronounced for TSR1 (<a href="#pone-0001656-g005">Fig. 5</a>), suggesting it is strongly related to chromosomally related transcriptome regulation. A higher-order organization of transcriptome regulation in terms of chromosomal domains is also suggested by Caron et al.<a href="#pone.0001656-Caron1">[16]</a></p>
<a id="article1.body1.sec3.p7" name="article1.body1.sec3.p7"></a><p>Our results give biological insights into regulation of the cellular transcriptome and provide a tool to characterize expression profiles with highly reliable TSRs instead of thousands of individual genes, leading to a &gt;500-fold reduction of complexity with just 50 TSRs. This might open new avenues for those performing gene expression profiling studies.</p>
</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4>Data acquisition</h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p>Publicly available microarray expression data of 17,550 human samples hybridized to HG-U133A or HG-U133 Plus 2.0 Genechips (Affymetrix, Santa Clara, California, USA) and 3,934 mouse samples hybridized to MG-U74A Genechip (Affymetrix, Santa Clara, California, USA) were obtained from the Gene Expression Omnibus.<a href="#pone.0001656-Edgar1">[17]</a> These datasets contained a wide range of heterogeneous tissues (primary patient material, cell lines, normal tissues, etc.) and covered a multitude of different experimental conditions (transfected/transduced, stimulated or treated cells, etc.). For the human dataset, probes available on both platforms were selected for further analysis. Then the probes for the human as well as the mouse dataset were converted to official gene symbols, averaging log transformed expression values of multiple probes targeting the same gene. This resulted in 13,032 and 9,062 unique genes for the human and mouse datasets, respectively. Next, quantile normalization was applied separately to the log2 transformed expression values in each dataset.<a href="#pone.0001656-Bolstad1">[18]</a></p>


<h4>Principal component analysis</h4>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1"></a><p>Correlations between genes were calculated based on expression patterns across the diverse samples in both the human as well as the mouse datasets. Principal component analysis (PCA) was performed on the resulting correlation matrices, which is equivalent to factor analysis, leading to reduced dimensionality in gene space. We developed software to perform this task based on JAMA/C++, a translation of the Java Matrix Library, developed by the Mathworks and NIST, into C++ (<a href="http://math.nist.gov/tnt/">http://math.nist.gov/tnt/</a>). PCA is a method to condense a multi-dimensional dataset into a set of lower dimensions, in order to reveal the simplified linear structure of the data that often underlies it.<a href="#pone.0001656-Rencher1">[19]</a> In this study PCA represents a transformation of a set of correlated genes into sets of uncorrelated linear additions of gene expression signals called principal components (PCs). PCs are able to summarize expression information and, for this application, can be interpreted as statistically uncorrelated transcriptional system regulators (TSRs).<a href="#pone.0001656-Alter1">[9]</a> In each TSR all genes are present but the weight of individual genes in the linear addition varies among TSRs. TSRs are constructed in such a way that the first TSR explains the largest amount of variance in expression and each subsequent TSR explains the largest amount of the remaining variance in expression while remaining uncorrelated with previously constructed TSRs. Since a TSR is a weighted sum of genes, TSR scores can be calculated for each observed expression array (TSR profiling). A TSR score can be seen as the degree of activity of the regulator in different cellular states or phenotypes. We provide a software tool capable of calculating individual TSR scores for observed expression arrays (see supplementary information online). Subsequently, the correlation between individual gene expression and TSR scores can be calculated (<em>i.e.</em> a factor loading). A factor loading can be seen as the amount of control a specific TSR has on the net expression of a particular gene. A high positive or negative factor loading with a TSR indicates that a gene's expression is strongly influenced by this specific TSR. Clusters of genes with contrasting factor loading signs (<em>i.e.</em> positive <em>vs.</em> negative) are inversely regulated by a specific TSR. For further reading on PCA we recommend a publicly available tutorial.<a href="#pone.0001656-Smith1">[20]</a></p>


<h4>Reliability of transcriptional system regulators</h4>
<a id="article1.body1.sec4.sec3.p1" name="article1.body1.sec4.sec3.p1"></a><p>To investigate whether our method gives results that depend on the presence of specific arrays, we randomly divided the human dataset into two equally sized sets and then generated new TSRs using PCA, each based on only half of the data. To assess whether these separately generated sets were comparable, we calculated Pearson correlations between the factor loadings with TSRs from the two separate sets.</p>
<a id="article1.body1.sec4.sec3.p2" name="article1.body1.sec4.sec3.p2"></a><p>Furthermore, to validate that the identified TSRs were not artifacts of gene selection, we applied the split-half method on each TSR.<a href="#pone.0001656-Lord1">[21]</a> As described above, a TSR is a weighted sum of genes and a so-called TSR score can be calculated for each sample. In the split-half method, the genes of a TSR score are split into two random partitions and the resulting TSR scores of both parts are correlated (Pearson). High correlation indicates that TSR scores can be reliably calculated and that information from different genes is indeed identical and indicative of the same underlying TSR score.</p>


<h4>Gene set enrichment analysis (GSEA)</h4>
<a id="article1.body1.sec4.sec4.p1" name="article1.body1.sec4.sec4.p1"></a><p>To investigate whether our identified statistically uncorrelated TSRs are related to biologically related gene clusters represented by known GO ontologies we used GeneTrail, a software program recently developed by a German team (<a href="http://genetrail.bioinf.uni-sb.de">http://genetrail.bioinf.uni-sb.de</a>).<a href="#pone.0001656-Backes1">[22]</a>. This web-based application scores a sorted list of genes with respect to their enrichment of functional categories.<a href="#pone.0001656-Subramanian1">[23]</a> For each TSR we ranked the genes according to ascending factor loadings, <em>i.e.</em> from most negative to most positive factor loading. A factor loading is the correlation between individual gene expression and a specific TSR score (degree of regulator activation). The ranked list of genes, of which some belong to a functional set <em>S</em>, is then processed from top to bottom. Genes at the top and bottom (<em>i.e.</em> genes with high negative and positive factorloadings respectively) are genes most strongly controlled by our defined TSR. Whenever a gene belonging to the functional set <em>S</em> is found, an enrichment statistic (ES) is increased by a certain amount, otherwise the ES is decreased. This ES is depicted in graphs showing whether the genes that comprise a functional set <em>S</em> are accumulated at the top (red graph) or bottom (green graph) of the ranked list (see <a href="#pone-0001656-g002">Figs 2</a> and <a href="#pone-0001656-g004">4</a>). The red <em>vs.</em> green graphs show the biological coherence of opposing regulated gene clusters controlled by TSRs, <em>i.e.</em> gene clusters with positive <em>vs.</em> negative factor loadings. The minimum and maximum of this ES are used to estimate the significance of the enrichment; the more significant a functional set <em>S</em> is, the more important a TSR is in regulating the expression of genes belonging to <em>S</em>. <a href="#pone.0001656-Backes1">[22]</a> To adjust for multiple testing problems, we tested for the false discovery rate (FDR) according to Benjamini and Hochberg's method.<a href="#pone.0001656-Benjamini1">[24]</a> A significance threshold of <em>P</em>&lt;0.05 after FDR correction was maintained. Although GeneTrail reveals many biological categories, we focused our analysis on Gene Ontology (GO) and chromosomal location.<a href="#pone.0001656-Ashburner1">[25]</a> We analyzed 5,760 GO categories containing more than two genes and also assessed 24 chromosomes (including X and Y) for enrichment in co-regulated genes.</p>


<h4>Similarities in transcriptional system regulation between human and mouse</h4>
<a id="article1.body1.sec4.sec5.p1" name="article1.body1.sec4.sec5.p1"></a><p>To assess the similarity of the structure in transcription regulation between human and mouse, we applied a PCA to a combined two-species dataset containing 3,934 mouse arrays and 3,934 randomly selected human arrays. Expression data of 6,610 orthologus genes between human and mouse was selected for this dataset. If the structure of transcriptional regulation is similar between human and mouse, we would expect a similar, limited number of TSRs to be needed to summarize the same amount of variance in total expression as seen in the PCA performed on human and mouse expression data separately.</p>


<h4>Mapping human and mouse TSRs</h4>
<a id="article1.body1.sec4.sec6.p1" name="article1.body1.sec4.sec6.p1"></a><p>To map TSRs identified in 17,550 human and 3,934 mouse arrays separately, in order to assess similarities in regulation of biological processes between human and mouse, we selected the factor loadings of the first 50 TSRs for 6,610 identified homolog genes, based on similar gene symbol identifiers. Then we calculated Spearman rank correlations between factor loadings from human and mouse TSRs. High correlation between a human- and a mouse TSR indicates that these TSRs control identical gene clusters in human and mouse in the same way.</p>


<h4>Gene clustering based on gene-TSR correlation</h4>
<a id="article1.body1.sec4.sec7.p1" name="article1.body1.sec4.sec7.p1"></a><p>In microarray experiments, gene expression is composed of biological signals and experimental noise.<a href="#pone.0001656-Draghici1">[8]</a> In our model the first 50 TSRs capture most of the biologically relevant signals and subsequent TSRs capture noise and experimental artifacts. Clustering genes according to their factor loadings with the first 50 TSRs instead of net expression patterns, of which a part is experimental noise, might be a more robust approach and could give more insight into the dynamics of gene expression regulation, in which individual genes are classified into multiple groups of similar regulation. We used average linkage hierarchical clustering according to the Euclidean distance measure (square root of the sum of the squared differences in each dimension) by using the Cluster 3.0 software.<a href="#pone.0001656-deHoon1">[26]</a></p>


<h4>Sample clustering based on TSR scores</h4>
<a id="article1.body1.sec4.sec8.p1" name="article1.body1.sec4.sec8.p1"></a><p>Since a TSR is a weighted sum of genes, a so-called TSR score can be calculated for each individual sample. To assess whether similar tissue samples have identical TSR scores, we applied average linkage hierarchical clustering according to the Euclidean distance measure on the first 50 TSR scores. We selected 621 samples, representing over 90 distinct tissue types from the publicly available human body index dataset from the Gene Expression Omnibus (Accession number: GSE7307).</p>

</div>

<div id="section5" class="section"><a id="s5" name="s5" toc="s5" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0001656"></div><a name="pone.0001656.s001" id="pone.0001656.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001656.s001">Table S1. </a></strong></p><p class="siDoi">doi:10.1371/journal.pone.0001656.s001</p><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1"></a><p class="postSiDOI">(17.09 MB XLS)</p>
<a name="pone.0001656.s002" id="pone.0001656.s002"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001656.s002">Table S2. </a></strong></p><p class="siDoi">doi:10.1371/journal.pone.0001656.s002</p><a id="article1.body1.sec5.supplementary-material2.caption1.p1" name="article1.body1.sec5.supplementary-material2.caption1.p1"></a><p class="postSiDOI">(12.07 MB XLS)</p>
<a name="pone.0001656.s003" id="pone.0001656.s003"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001656.s003">Table S3. </a></strong></p><p class="siDoi">doi:10.1371/journal.pone.0001656.s003</p><a id="article1.body1.sec5.supplementary-material3.caption1.p1" name="article1.body1.sec5.supplementary-material3.caption1.p1"></a><p class="postSiDOI">(0.02 MB XLS)</p>
<a name="pone.0001656.s004" id="pone.0001656.s004"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001656.s004">Figure S1. </a></strong></p><p class="siDoi">doi:10.1371/journal.pone.0001656.s004</p><a id="article1.body1.sec5.supplementary-material4.caption1.p1" name="article1.body1.sec5.supplementary-material4.caption1.p1"></a><p class="postSiDOI">(1.14 MB tif)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>We thank J.L. Senior for critically editing our manuscript.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: At Hd RF Gt Ed. Performed the experiments: Hd RF Gt. Analyzed the data: At Hd RF RH FG Gt Ed. Contributed reagents/materials/analysis tools: Hd RF Gt. Wrote the paper: Ed At Hd RF RH FG WK Av Gt Ed Ad AC AW Sd.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0001656-Segal1" id="pone.0001656-Segal1"></a>Segal E, Shapira M, Regev A, Peter D, Botstein D, et al.  (2003) Module networks: identifying regulatory modules and their condition-specific regulators from gene expression data. Nat Genet  34: 166â176.  <ul class="find" data-citedArticleID="1061134" data-doi="10.1038/ng1165"><li><a href="http://dx.doi.org/10.1038/ng1165" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Module+networks%3A+identifying+regulatory+modules+and+their+condition-specific+regulators+from+gene+expression+data." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Module+networks%3A+identifying+regulatory+modules+and+their+condition-specific+regulators+from+gene+expression+data.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0001656-Eisen1" id="pone.0001656-Eisen1"></a>Eisen MB, Spellman PT, Brown PO, Botstein D (1998) Cluster analysis and display of genome-wide expression patterns. Proc Natl Acad Sci U S A  95: 14863â14868.  <ul class="find" data-citedArticleID="1061116" data-doi="10.1073/pnas.95.25.14863"><li><a href="http://dx.doi.org/10.1073/pnas.95.25.14863" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Cluster+analysis+and+display+of+genome-wide+expression+patterns." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Cluster+analysis+and+display+of+genome-wide+expression+patterns.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0001656-Alon1" id="pone.0001656-Alon1"></a>Alon U, Barkai N, Notterman DA, Gish K, Ybarra S, et al.  (1999) Broad patterns of gene expression revealed by clustering analysis of tumor and normal colon tissues probed by oligonucleotide arrays. Proc Natl Acad Sci U S A  96: 6745â6750.  <ul class="find" data-citedArticleID="1061092"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Broad+patterns+of+gene+expression+revealed+by+clustering+analysis+of+tumor+and+normal+colon+tissues+probed+by+oligonucleotide+arrays.&amp;auth=&amp;atitle=Broad+patterns+of+gene+expression+revealed+by+clustering+analysis+of+tumor+and+normal+colon+tissues+probed+by+oligonucleotide+arrays." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Broad+patterns+of+gene+expression+revealed+by+clustering+analysis+of+tumor+and+normal+colon+tissues+probed+by+oligonucleotide+arrays." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Broad+patterns+of+gene+expression+revealed+by+clustering+analysis+of+tumor+and+normal+colon+tissues+probed+by+oligonucleotide+arrays.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0001656-Misteli1" id="pone.0001656-Misteli1"></a>Misteli T (2007) Beyond the sequence: cellular organization of genome function. Cell  128: 787â800.  <ul class="find" data-citedArticleID="1061130" data-doi="10.1016/j.cell.2007.01.028"><li><a href="http://dx.doi.org/10.1016/j.cell.2007.01.028" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Beyond+the+sequence%3A+cellular+organization+of+genome+function." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Beyond+the+sequence%3A+cellular+organization+of+genome+function.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0001656-Gasch1" id="pone.0001656-Gasch1"></a>Gasch AP, Eisen MB (2002) Exploring the conditional coregulation of yeast gene expression through fuzzy k-means clustering. Genome Biol  3: RESEARCH0059.  <ul class="find" data-citedArticleID="1061120"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Exploring+the+conditional+coregulation+of+yeast+gene+expression+through+fuzzy+k-means+clustering.&amp;auth=&amp;atitle=Exploring+the+conditional+coregulation+of+yeast+gene+expression+through+fuzzy+k-means+clustering." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Exploring+the+conditional+coregulation+of+yeast+gene+expression+through+fuzzy+k-means+clustering." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Exploring+the+conditional+coregulation+of+yeast+gene+expression+through+fuzzy+k-means+clustering.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0001656-Ihmels1" id="pone.0001656-Ihmels1"></a>Ihmels J, Friedlander G, Bergmann S, Sarig O, Ziv Y, et al.  (2002) Revealing modular organization in the yeast transcriptional network. Nat Genet  31: 370â377.  <ul class="find" data-citedArticleID="1061124" data-doi="10.1038/ng941"><li><a href="http://dx.doi.org/10.1038/ng941" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Revealing+modular+organization+in+the+yeast+transcriptional+network." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Revealing+modular+organization+in+the+yeast+transcriptional+network.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0001656-Simon1" id="pone.0001656-Simon1"></a>Simon R, Radmacher MD, Dobbin K, McShane LM (2003) Pitfalls in the use of DNA microarray data for diagnostic and prognostic classification. J Natl Cancer Inst  95: 14â18.  <ul class="find" data-citedArticleID="1061136" data-doi="10.1093/jnci/95.1.14"><li><a href="http://dx.doi.org/10.1093/jnci/95.1.14" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Pitfalls+in+the+use+of+DNA+microarray+data+for+diagnostic+and+prognostic+classification." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Pitfalls+in+the+use+of+DNA+microarray+data+for+diagnostic+and+prognostic+classification.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0001656-Draghici1" id="pone.0001656-Draghici1"></a>Draghici S, Khatri P, Eklund AC, Szallasi Z (2006) Reliability and reproducibility issues in DNA microarray measurements. Trends Genet  22: 101â109.  <ul class="find" data-citedArticleID="1061112" data-doi="10.1016/j.tig.2005.12.005"><li><a href="http://dx.doi.org/10.1016/j.tig.2005.12.005" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Reliability+and+reproducibility+issues+in+DNA+microarray+measurements." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Reliability+and+reproducibility+issues+in+DNA+microarray+measurements.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0001656-Alter1" id="pone.0001656-Alter1"></a>Alter O, Brown PO, Botstein D (2000) Singular value decomposition for genome-wide expression data processing and modeling. Proc Natl Acad Sci U S A  97: 10101â10106.  <ul class="find" data-citedArticleID="1061094" data-doi="10.1073/pnas.97.18.10101"><li><a href="http://dx.doi.org/10.1073/pnas.97.18.10101" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Singular+value+decomposition+for+genome-wide+expression+data+processing+and+modeling." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Singular+value+decomposition+for+genome-wide+expression+data+processing+and+modeling.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0001656-Gaszner1" id="pone.0001656-Gaszner1"></a>Gaszner M, Felsenfeld G (2006) Insulators: exploiting transcriptional and epigenetic mechanisms. Nat Rev Genet  7: 703â713.  <ul class="find" data-citedArticleID="1061122" data-doi="10.1038/nrg1925"><li><a href="http://dx.doi.org/10.1038/nrg1925" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Insulators%3A+exploiting+transcriptional+and+epigenetic+mechanisms." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Insulators%3A+exploiting+transcriptional+and+epigenetic+mechanisms.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0001656-ElOsta1" id="pone.0001656-ElOsta1"></a>El-Osta A, Wolffe AP (2000) DNA methylation and histone deacetylation in the control of gene expression: basic biochemistry to human development and disease. Gene Expr  9: 63â75.  <ul class="find" data-citedArticleID="1061118"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=DNA+methylation+and+histone+deacetylation+in+the+control+of+gene+expression%3A+basic+biochemistry+to+human+development+and+disease.&amp;auth=&amp;atitle=DNA+methylation+and+histone+deacetylation+in+the+control+of+gene+expression%3A+basic+biochemistry+to+human+development+and+disease." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=DNA+methylation+and+histone+deacetylation+in+the+control+of+gene+expression%3A+basic+biochemistry+to+human+development+and+disease." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22DNA+methylation+and+histone+deacetylation+in+the+control+of+gene+expression%3A+basic+biochemistry+to+human+development+and+disease.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0001656-Chakalova1" id="pone.0001656-Chakalova1"></a>Chakalova L, Debrand E, Mitchell JA, Osborne CS, Fraser P (2005) Replication and transcription: shaping the landscape of the genome. Nat Rev Genet  6: 669â677.  <ul class="find" data-citedArticleID="1061106" data-doi="10.1038/nrg1673"><li><a href="http://dx.doi.org/10.1038/nrg1673" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Replication+and+transcription%3A+shaping+the+landscape+of+the+genome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Replication+and+transcription%3A+shaping+the+landscape+of+the+genome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0001656-Cook1" id="pone.0001656-Cook1"></a>Cook PR (1999) The organization of replication and transcription. Science  284: 1790â1795.  <ul class="find" data-citedArticleID="1061108" data-doi="10.1126/science.284.5421.1790"><li><a href="http://dx.doi.org/10.1126/science.284.5421.1790" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+organization+of+replication+and+transcription." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+organization+of+replication+and+transcription.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0001656-Cremer1" id="pone.0001656-Cremer1"></a>Cremer T, Cremer C (2001) Chromosome territories, nuclear architecture and gene regulation in mammalian cells. Nat Rev Genet  2: 292â301.  <ul class="find" data-citedArticleID="1061110" data-doi="10.1038/35066075"><li><a href="http://dx.doi.org/10.1038/35066075" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Chromosome+territories%2C+nuclear+architecture+and+gene+regulation+in+mammalian+cells." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Chromosome+territories%2C+nuclear+architecture+and+gene+regulation+in+mammalian+cells.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0001656-Meaburn1" id="pone.0001656-Meaburn1"></a>Meaburn KJ, Misteli T (2007) Cell biology: chromosome territories. Nature  445: 379â781.  <ul class="find" data-citedArticleID="1061128" data-doi="10.1038/445379a"><li><a href="http://dx.doi.org/10.1038/445379a" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Cell+biology%3A+chromosome+territories." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Cell+biology%3A+chromosome+territories.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0001656-Caron1" id="pone.0001656-Caron1"></a>Caron H, van Schaik B, van der Mee M, Baas F, Riggins G, et al.  (2001) The human transcriptome map: clustering of highly expressed genes in chromosomal domains. Science  291: 1289â1292.  <ul class="find" data-citedArticleID="1061104" data-doi="10.1126/science.1056794"><li><a href="http://dx.doi.org/10.1126/science.1056794" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+human+transcriptome+map%3A+clustering+of+highly+expressed+genes+in+chromosomal+domains." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+human+transcriptome+map%3A+clustering+of+highly+expressed+genes+in+chromosomal+domains.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0001656-Edgar1" id="pone.0001656-Edgar1"></a>Edgar R, Domrachev M, Lash AE (2002) Gene Expression Omnibus: NCBI gene expression and hybridization array data repository. Nucleic Acids Res  30: 207â210.  <ul class="find" data-citedArticleID="1061114" data-doi="10.1093/nar/30.1.207"><li><a href="http://dx.doi.org/10.1093/nar/30.1.207" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Gene+Expression+Omnibus%3A+NCBI+gene+expression+and+hybridization+array+data+repository." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Gene+Expression+Omnibus%3A+NCBI+gene+expression+and+hybridization+array+data+repository.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0001656-Bolstad1" id="pone.0001656-Bolstad1"></a>Bolstad BM, Irizarry RA, Astrand M, Speed TP (2003) A comparison of normalization methods for high density oligonucleotide array data based on variance and bias. Bioinformatics  19: 185â193.  <ul class="find" data-citedArticleID="1061102" data-doi="10.1093/bioinformatics/19.2.185"><li><a href="http://dx.doi.org/10.1093/bioinformatics/19.2.185" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+comparison+of+normalization+methods+for+high+density+oligonucleotide+array+data+based+on+variance+and+bias." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+comparison+of+normalization+methods+for+high+density+oligonucleotide+array+data+based+on+variance+and+bias.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0001656-Rencher1" id="pone.0001656-Rencher1"></a>Rencher A (1998) Multivariate Statistical Inference and Applications. New York: John Wiley &amp; Sons, Inc.   <ul class="find-nolinks"></ul></li><li><span class="label">20.
              </span><a name="pone.0001656-Smith1" id="pone.0001656-Smith1"></a>Smith LI (2002) A tutorial on principal component analysis.   Ref Type: Internet Communication <a href="http://csnet.otago.ac.nz/cosc453/student_tutorials/principal_components.pdf">http://csnet.otago.ac.nz/cosc453/studentâ_tutorials/principal_components.pdf</a> (February 26, 2002).  <ul class="find-nolinks"></ul></li><li><span class="label">21.
              </span><a name="pone.0001656-Lord1" id="pone.0001656-Lord1"></a>Lord FM, Novick MR (1968) Conditions of Measurement Affecting Reliability in <em>Statistical Therories Of Mental Test Scores,</em> edited by Frederick Mosteller Addison-Wesley Pub. Co., Reading, Massachusetts.129â150.  <ul class="find" data-citedArticleID="1061126"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Conditions+of+Measurement+Affecting+Reliability+in+Statistical+Therories+Of+Mental+Test+Scores%2C+edited+by+Frederick+Mosteller+Addison-Wesley+Pub.+Co.%2C+Reading%2C+Massachusetts.&amp;auth=&amp;atitle=Conditions+of+Measurement+Affecting+Reliability+in+Statistical+Therories+Of+Mental+Test+Scores%2C+edited+by+Frederick+Mosteller+Addison-Wesley+Pub.+Co.%2C+Reading%2C+Massachusetts." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Conditions+of+Measurement+Affecting+Reliability+in+Statistical+Therories+Of+Mental+Test+Scores%2C+edited+by+Frederick+Mosteller+Addison-Wesley+Pub.+Co.%2C+Reading%2C+Massachusetts." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Conditions+of+Measurement+Affecting+Reliability+in+Statistical+Therories+Of+Mental+Test+Scores%2C+edited+by+Frederick+Mosteller+Addison-Wesley+Pub.+Co.%2C+Reading%2C+Massachusetts.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0001656-Backes1" id="pone.0001656-Backes1"></a>Backes C, Keller A, Kuentzer J, Kneissl B, Comtesse N, et al.  (2007) GeneTrailâadvanced gene set enrichment analysis. Nucleic Acids Res.   <ul class="find" data-citedArticleID="1061098" data-doi="10.1093/nar/gkm323"><li><a href="http://dx.doi.org/10.1093/nar/gkm323" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=GeneTrail%E2%80%93advanced+gene+set+enrichment+analysis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22GeneTrail%E2%80%93advanced+gene+set+enrichment+analysis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0001656-Subramanian1" id="pone.0001656-Subramanian1"></a>Subramanian A, Tamayo P, Mootha VK, Mukherjee S, Ebert BL, et al.  (2005) Gene set enrichment analysis: a knowledge-based approach for interpreting genome-wide expression profiles. Proc Natl Acad Sci U S A  102: 15545â15550.  <ul class="find" data-citedArticleID="1061140" data-doi="10.1073/pnas.0506580102"><li><a href="http://dx.doi.org/10.1073/pnas.0506580102" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Gene+set+enrichment+analysis%3A+a+knowledge-based+approach+for+interpreting+genome-wide+expression+profiles." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Gene+set+enrichment+analysis%3A+a+knowledge-based+approach+for+interpreting+genome-wide+expression+profiles.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0001656-Benjamini1" id="pone.0001656-Benjamini1"></a>Benjamini Y, Hochberg Y (1995) Controlling the False Discovery Rate - A Practical and Powerful Approach to Multiple Testing. Journal of the Royal Statistical Society Series B-Methodological  57: 289â300.  <ul class="find" data-citedArticleID="1061100"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Controlling+the+False+Discovery+Rate+-+A+Practical+and+Powerful+Approach+to+Multiple+Testing.&amp;auth=&amp;atitle=Controlling+the+False+Discovery+Rate+-+A+Practical+and+Powerful+Approach+to+Multiple+Testing." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Controlling+the+False+Discovery+Rate+-+A+Practical+and+Powerful+Approach+to+Multiple+Testing." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Controlling+the+False+Discovery+Rate+-+A+Practical+and+Powerful+Approach+to+Multiple+Testing.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0001656-Ashburner1" id="pone.0001656-Ashburner1"></a>Ashburner M, Ball CA, Blake JA, Botstein D, Butler H, et al.  (2000) Gene ontology: tool for the unification of biology. The Gene Ontology Consortium. Nat Genet  25: 25â29.  <ul class="find" data-citedArticleID="1061096"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Gene+ontology%3A+tool+for+the+unification+of+biology.+The+Gene+Ontology+Consortium.&amp;auth=&amp;atitle=Gene+ontology%3A+tool+for+the+unification+of+biology.+The+Gene+Ontology+Consortium." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Gene+ontology%3A+tool+for+the+unification+of+biology.+The+Gene+Ontology+Consortium." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Gene+ontology%3A+tool+for+the+unification+of+biology.+The+Gene+Ontology+Consortium.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0001656-deHoon1" id="pone.0001656-deHoon1"></a>de Hoon MJ, Imoto S, Nolan J, Miyano S (2004) Open source clustering software. Bioinformatics  20: 1453â1454.  <ul class="find" data-citedArticleID="1061142" data-doi="10.1093/bioinformatics/bth078"><li><a href="http://dx.doi.org/10.1093/bioinformatics/bth078" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Open+source+clustering+software." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Open+source+clustering+software.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001656.XML" value="70032"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001656.PDF" value="759793"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001656.g001.PNG_L" value="1628787"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001656.g001.PNG_M" value="92101"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001656.g001.PNG_S" value="6526"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001656.g001.TIF" value="2806584"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001656.g001.PNG_I" value="29603"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001656.g002.PNG_L" value="5167151"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001656.g002.PNG_M" value="221711"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001656.g002.PNG_S" value="16917"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001656.g002.TIF" value="8416702"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001656.g002.PNG_I" value="124241"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001656.g003.PNG_L" value="5217951"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001656.g003.PNG_M" value="251999"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001656.g003.PNG_S" value="12640"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001656.g003.TIF" value="8003718"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001656.g003.PNG_I" value="80148"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001656.g004.PNG_L" value="4701403"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001656.g004.PNG_M" value="171083"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001656.g004.PNG_S" value="28100"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001656.g004.TIF" value="6822276"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001656.g004.PNG_I" value="309310"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001656.g005.PNG_L" value="4631856"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001656.g005.PNG_M" value="193776"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001656.g005.PNG_S" value="17043"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001656.g005.TIF" value="8315006"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001656.g005.PNG_I" value="101672"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001656.g006.PNG_L" value="1315064"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001656.g006.PNG_M" value="66944"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001656.g006.PNG_S" value="4560"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001656.g006.TIF" value="2139068"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001656.g006.PNG_I" value="22075"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001656.g007.PNG_L" value="2134771"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001656.g007.PNG_M" value="159469"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001656.g007.PNG_S" value="7161"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001656.g007.TIF" value="4591996"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001656.g007.PNG_I" value="53473"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001656.s001.XLS" value="17088512"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001656.s002.XLS" value="12072448"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001656.s003.XLS" value="23040"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001656.s004.TIF" value="1369510"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001656&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001656" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001656&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0001656&volume=&issue=&title=A New Perspective on Transcriptional System Regulation (TSR): Towards TSR Profiling&author_name=Rudolf%20S.%20N.%20Fehrmann%2C%20Hendrik%20J.%20M.%20de%20Jonge%2C%20Arja%20ter%20Elst%2C%20Andr%C3%A9%20de%20Vries%2C%20Anne%20G.%20P.%20Crijns%2C%20Alida%20C.%20Weidenaar%2C%20Frans%20Gerbens%2C%20Steven%20de%20Jong%2C%20Ate%20G.%20J.%20van%20der%20Zee%2C%20Elisabeth%20G.%20E.%20de%20Vries%2C%20Willem%20A.%20Kamps%2C%20Robert%20M.%20W.%20Hofstra%2C%20Gerard%20J.%20te%20Meerman%2C%20Eveline%20S.%20J.%20M.%20de%20Bont&start_page=1&end_page=10" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001656" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001656" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0001656" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001656&amp;t=A%20New%20Perspective%20on%20Transcriptional%20System%20Regulation%20(TSR)%3A%20Towards%20TSR%20Profiling" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001656&title=A%20New%20Perspective%20on%20Transcriptional%20System%20Regulation%20(TSR)%3A%20Towards%20TSR%20Profiling&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001656&amp;title=A%20New%20Perspective%20on%20Transcriptional%20System%20Regulation%20(TSR)%3A%20Towards%20TSR%20Profiling" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001656" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0001656&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'A New Perspective on Transcriptional System Regulation (TSR): Towards TSR Profiling';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0001656';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20A%20New%20Perspective%20on%20Transcriptional%20System%20Regulation%20(TSR)%3A%20Towards%20TSR%20Profiling http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001656" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0001656" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">










          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Chromosomes%22" title="Search for articles in the subject area:'Chromosomes'"><div class="flagText">Chromosomes</div></a>
              <div data-categoryid="17251" data-articleid="26138"
                   data-categoryname="Chromosomes"
                   class="flagImage" title="Flag 'Chromosomes' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22DNA+transcription%22" title="Search for articles in the subject area:'DNA transcription'"><div class="flagText">DNA transcription</div></a>
              <div data-categoryid="20231" data-articleid="26138"
                   data-categoryname="DNA transcription"
                   class="flagImage" title="Flag 'DNA transcription' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Gene+expression%22" title="Search for articles in the subject area:'Gene expression'"><div class="flagText">Gene expression</div></a>
              <div data-categoryid="17515" data-articleid="26138"
                   data-categoryname="Gene expression"
                   class="flagImage" title="Flag 'Gene expression' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Gene+ontologies%22" title="Search for articles in the subject area:'Gene ontologies'"><div class="flagText">Gene ontologies</div></a>
              <div data-categoryid="40557" data-articleid="26138"
                   data-categoryname="Gene ontologies"
                   class="flagImage" title="Flag 'Gene ontologies' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Gene+regulation%22" title="Search for articles in the subject area:'Gene regulation'"><div class="flagText">Gene regulation</div></a>
              <div data-categoryid="17553" data-articleid="26138"
                   data-categoryname="Gene regulation"
                   class="flagImage" title="Flag 'Gene regulation' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Microarrays%22" title="Search for articles in the subject area:'Microarrays'"><div class="flagText">Microarrays</div></a>
              <div data-categoryid="17255" data-articleid="26138"
                   data-categoryname="Microarrays"
                   class="flagImage" title="Flag 'Microarrays' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Regulator+genes%22" title="Search for articles in the subject area:'Regulator genes'"><div class="flagText">Regulator genes</div></a>
              <div data-categoryid="18579" data-articleid="26138"
                   data-categoryname="Regulator genes"
                   class="flagImage" title="Flag 'Regulator genes' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Transcriptome+analysis%22" title="Search for articles in the subject area:'Transcriptome analysis'"><div class="flagText">Transcriptome analysis</div></a>
              <div data-categoryid="40151" data-articleid="26138"
                   data-categoryname="Transcriptome analysis"
                   class="flagImage" title="Flag 'Transcriptome analysis' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=5315'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=1990'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=9710&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>


</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
