

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: 3-D Ultrastructure of O. tauri: Electron Cryotomography of an Entire Eukaryotic Cell</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0000749"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0000749" />

    <meta name="citation_title" content="3-D Ultrastructure of O. tauri: Electron Cryotomography of an Entire Eukaryotic Cell"/>
    <meta itemprop="name" content="3-D Ultrastructure of O. tauri: Electron Cryotomography of an Entire Eukaryotic Cell"/>

      <meta name="citation_author" content="Gregory P. Henderson"/>
            <meta name="citation_author_institution" content="Division of Biology, California Institute of Technology, Pasadena, California, United States of America"/>
      <meta name="citation_author" content="Lu Gan"/>
            <meta name="citation_author_institution" content="Division of Biology, California Institute of Technology, Pasadena, California, United States of America"/>
      <meta name="citation_author" content="Grant J. Jensen"/>
            <meta name="citation_author_institution" content="Division of Biology, California Institute of Technology, Pasadena, California, United States of America"/>

    <meta name="citation_date" content="2007/8/15"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0000749.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e749"/>
    <meta name="citation_issue" content="8"/>
    <meta name="citation_volume" content="2"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=High-pressure freezing, cellular tomography, and structural cell biology.; citation_author=KL McDonald; citation_author=M Auer; citation_journal_title=Biotechniques; citation_volume=41; citation_number=1; citation_pages=137-143; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Structural studies by electron tomography: from cells to molecules.; citation_author=V Lucic; citation_author=F Forster; citation_author=W Baumeister; citation_journal_title=Annu Rev Biochem; citation_volume=74; citation_number=2; citation_pages=833-865; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Organization of interphase microtubules in fission yeast analyzed by electron tomography.; citation_author=JL Hoog; citation_author=C Schwartz; citation_author=AT Noon; citation_author=ET O'Toole; citation_author=DN Mastronarde; citation_journal_title=Dev Cell; citation_volume=12; citation_number=3; citation_pages=349-361; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Cryo-electron microscopy of vitrified specimens.; citation_author=J Dubochet; citation_author=M Adrian; citation_author=JJ Chang; citation_author=JC Homo; citation_author=J Lepault; citation_journal_title=Q Rev Biophys; citation_volume=21; citation_number=4; citation_pages=129-228; citation_date=1988; " />
      <meta name="citation_reference" content="citation_title=Energy filtered electron tomography of ice-embedded actin and vesicles.; citation_author=R Grimm; citation_author=M Barmann; citation_author=W Hackl; citation_author=D Typke; citation_author=E Sackmann; citation_journal_title=Biophys J; citation_volume=72; citation_number=5; citation_pages=482-489; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=How electron cryotomography is opening a new window onto prokaryotic ultrastructure.; citation_author=GJ Jensen; citation_author=A Briegel; citation_journal_title=Curr Opin Struct Biol; citation_volume=17; citation_number=6; citation_pages=260-267; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Cryo-electron tomography of neurospora mitochondria.; citation_author=D Nicastro; citation_author=AS Frangakis; citation_author=D Typke; citation_author=W Baumeister; citation_journal_title=J Struct Biol; citation_volume=129; citation_number=7; citation_pages=48-56; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Macromolecular architecture in eukaryotic cells visualized by cryoelectron tomography.; citation_author=O Medalia; citation_author=I Weber; citation_author=AS Frangakis; citation_author=D Nicastro; citation_author=G Gerisch; citation_journal_title=Science; citation_volume=298; citation_number=8; citation_pages=1209-1213; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Three-dimensional electron microscopy at molecular resolution.; citation_author=S Subramaniam; citation_author=JL Milne; citation_journal_title=Annu Rev Biophys Biomol Struct; citation_volume=33; citation_number=9; citation_pages=141-155; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Cryo-electron microscopy of vitreous sections.; citation_author=A Al-Amoudi; citation_author=JJ Chang; citation_author=A Leforestier; citation_author=A McDowall; citation_author=LM Salamin; citation_journal_title=Embo J; citation_volume=23; citation_number=10; citation_pages=3583-3588; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Focused-ion-beam thinning of frozen-hydrated biological specimens for cryo-electron microscopy.; citation_author=M Marko; citation_author=C Hsieh; citation_author=R Schalek; citation_author=J Frank; citation_author=C Mannella; citation_journal_title=Nat Methods; citation_volume=4; citation_number=11; citation_pages=215-217; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=A new marine picoeucaryote: Ostreococcus tauri gen. et sp. nov. (Chlorophyta, Prasinophyceae).; citation_author=MJ Chrétiennot-Dinet; citation_author=C Courties; citation_author=A Vaquer; citation_author=J Neveux; citation_author=H Claustre; citation_journal_title=Phycologia; citation_volume=34; citation_number=12; citation_pages=285-292; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Genome analysis of the smallest free-living eukaryote Ostreococcus tauri unveils many unique features.; citation_author=E Derelle; citation_author=C Ferraz; citation_author=S Rombauts; citation_author=P Rouze; citation_author=AZ Worden; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=103; citation_number=13; citation_pages=11647-11652; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Phylogenetic Analysis and Genome Size of Ostreococcus tauri (Chlorophyta, Prasinophyceae).; citation_author=C Courties; citation_author=R Perasso; citation_author=MJ Chrétiennot-Dinet; citation_author=M Gouy; citation_author=L Guillour; citation_journal_title=J Phycology; citation_volume=34; citation_number=14; citation_pages=844-849; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=A molecular timeline for the origin of photosynthetic eukaryotes.; citation_author=HS Yoon; citation_author=JD Hackett; citation_author=C Ciniglia; citation_author=G Pinto; citation_author=D Bhattacharya; citation_journal_title=Mol Biol Evol; citation_volume=21; citation_number=15; citation_pages=809-818; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Genome-wide analysis of core cell cycle genes in the unicellular green alga Ostreococcus tauri.; citation_author=S Robbens; citation_author=B Khadaroo; citation_author=A Camasses; citation_author=E Derelle; citation_author=C Ferraz; citation_journal_title=Mol Biol Evol; citation_volume=22; citation_number=16; citation_pages=589-597; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Natural Synchronisation for the study of cell division in the green unicellular alga Ostreococcus tauri.; citation_author=B Farinas; citation_author=C Mary; citation_author=CL de O Manes; citation_author=Y Bhaud; citation_author=G Peaucellier; citation_journal_title=Plant Mol Biol; citation_volume=60; citation_number=17; citation_pages=277-292; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Smallest eukaryotic organism.; citation_author=C Courties; citation_author=A Vaquer; citation_author=M Troussellier; citation_author=J Lautier; citation_author=M-J Chrétiennot-Dinet; citation_journal_title=Nature; citation_volume=370; citation_number=18; citation_pages=255; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=A “flip-flop” rotation stage for routine dual-axis electron cryotomography.; citation_author=CV Iancu; citation_author=ER Wright; citation_author=J Benjamin; citation_author=WF Tivol; citation_author=DP Dias; citation_journal_title=J Struct Biol; citation_volume=151; citation_number=19; citation_pages=288-297; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Dose tolerance at helium and nitrogen temperatures for whole cell electron tomography.; citation_author=LR Comolli; citation_author=KH Downing; citation_journal_title=J Struct Biol; citation_volume=152; citation_number=20; citation_pages=149-156; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Starch division and partitioning. A mechanism for granule propagation and maintenance in the picophytoplanktonic green alga Ostreococcus tauri.; citation_author=JP Ral; citation_author=E Derelle; citation_author=C Ferraz; citation_author=F Wattebled; citation_author=B Farinas; citation_journal_title=Plant Physiol; citation_volume=136; citation_number=21; citation_pages=3333-3340; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=The three-dimensional structure of the cyanobacterium Synechocystis sp. PCC 6803.; citation_author=AM van de Meene; citation_author=MF Hohmann-Marriott; citation_author=WF Vermaas; citation_author=RW Roberson; citation_journal_title=Arch Microbiol; citation_volume=184; citation_number=22; citation_pages=259-270; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Thylakoid membrane perforations and connectivity enable intracellular traffic in cyanobacteria.; citation_author=R Nevo; citation_author=D Charuvi; citation_author=E Shimoni; citation_author=R Schwarz; citation_author=A Kaplan; citation_journal_title=EMBO J; citation_volume=26; citation_number=23; citation_pages=1467-1473; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Biogenesis and origin of thylakoid membranes.; citation_author=UC Vothknecht; citation_author=P Westhoff; citation_journal_title=Biochim Biophys Acta; citation_volume=1541; citation_number=24; citation_pages=91-101; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Three-dimensional organization of higher-plant chloroplast thylakoid membranes revealed by electron tomography.; citation_author=E Shimoni; citation_author=O Rav-Hon; citation_author=I Ohad; citation_author=V Brumfeld; citation_author=Z Reich; citation_journal_title=Plant Cell; citation_volume=17; citation_number=25; citation_pages=2580-2586; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=High resolution analysis of mammalian nuclear structure throughout the cell cycle: implications for nuclear pore complex assembly during interphase and mitosis.; citation_author=SP Drummond; citation_author=SA Rutherford; citation_author=HS Sanderson; citation_author=TD Allen; citation_journal_title=Cana J Physiol Pharmacol; citation_volume=84; citation_number=26; citation_pages=423-430; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Nuclear pore complex structure and dynamics revealed by cryoelectron tomography.; citation_author=M Beck; citation_author=F Forster; citation_author=M Ecke; citation_author=JM Plitzko; citation_author=F Melchior; citation_journal_title=Science; citation_volume=306; citation_number=27; citation_pages=1387-1390; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Architecture of the Xenopus nuclear pore complex revealed by three-dimensional cryo-electron microscopy.; citation_author=CW Akey; citation_author=M Radermacher; citation_journal_title=J Cell Biol; citation_volume=122; citation_number=28; citation_pages=1-19; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=Cryo-electron tomography provides novel insights into nuclear pore architecture: implications for nucleocytoplasmic transport.; citation_author=D Stoffler; citation_author=B Feja; citation_author=B Fahrenkrog; citation_author=J Walz; citation_author=D Typke; citation_journal_title=J Mol Biol; citation_volume=328; citation_number=29; citation_pages=119-130; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Three-dimensional architecture of the isolated yeast nuclear pore complex: functional and evolutionary implications.; citation_author=Q Yang; citation_author=MP Rout; citation_author=CW Akey; citation_journal_title=Mol Cell; citation_volume=1; citation_number=30; citation_pages=223-234; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Electron tomography of neuronal mitochondria: three-dimensional structure and organization of cristae and membrane contacts.; citation_author=G Perkins; citation_author=C Renken; citation_author=ME Martone; citation_author=SJ Young; citation_author=M Ellisman; citation_journal_title=J Struct Biol; citation_volume=119; citation_number=31; citation_pages=260-272; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=The internal compartmentation of rat-liver mitochondria: tomographic study using the high-voltage transmission electron microscope.; citation_author=CA Mannella; citation_author=M Marko; citation_author=P Penczek; citation_author=D Barnard; citation_author=J Frank; citation_journal_title=Microsc Res Tech; citation_volume=27; citation_number=32; citation_pages=278-283; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=Topology of the mitochondrial inner membrane: dynamics and bioenergetic implications.; citation_author=CA Mannella; citation_author=DR Pfeiffer; citation_author=PC Bradshaw; citation_author=II Moraru; citation_author=B Slepchenko; citation_journal_title=IUBMB Life; citation_volume=52; citation_number=33; citation_pages=93-100; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Structure and dynamics of the mitochondrial inner membrane cristae.; citation_author=CA Mannella; citation_journal_title=Biochim Biophys Acta; citation_volume=1763; citation_number=34; citation_pages=542-548; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=The relevance of mitochondrial membrane topology to mitochondrial function.; citation_author=CA Mannella; citation_journal_title=Biochim Biophys Acta; citation_volume=1762; citation_number=35; citation_pages=140-147; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=The plant ER: a dynamic organelle composed of a large number of discrete functional domains.; citation_author=LA Staehelin; citation_journal_title=Plant J; citation_volume=11; citation_number=36; citation_pages=1151-1165; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Rough sheets and smooth tubules.; citation_author=Y Shibata; citation_author=GK Voeltz; citation_author=TA Rapoport; citation_journal_title=Cell; citation_volume=126; citation_number=37; citation_pages=435-439; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=A class of membrane proteins shaping the tubular endoplasmic reticulum.; citation_author=GK Voeltz; citation_author=WA Prinz; citation_author=Y Shibata; citation_author=JM Rist; citation_author=TA Rapoport; citation_journal_title=Cell; citation_volume=124; citation_number=38; citation_pages=573-586; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Golgi twins in late mitosis revealed by genetically encoded tags for live cell imaging and correlated electron microscopy.; citation_author=GM Gaietta; citation_author=BN Giepmans; citation_author=TJ Deerinck; citation_author=WB Smith; citation_author=L Ngan; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=103; citation_number=39; citation_pages=17777-17782; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=The Golgi apparatus: balancing new with old.; citation_author=B Storrie; citation_author=T Nilsson; citation_journal_title=Traffic; citation_volume=3; citation_number=40; citation_pages=521-529; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Microtubule motors at the intersection of trafficking and transport.; citation_author=JP Caviston; citation_author=EL Holzbaur; citation_journal_title=Trends Cell Biol; citation_volume=16; citation_number=41; citation_pages=530-537; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=The tiny eukaryote Ostreococcus provides genomic insights into the paradox of plankton speciation.; citation_author=B Palenik; citation_author=J Grimwood; citation_author=A Aerts; citation_author=P Rouze; citation_author=A Salamov; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=104; citation_number=42; citation_pages=7705-7710; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Peroxisome biogenesis.; citation_author=PE Purdue; citation_author=PB Lazarow; citation_journal_title=Annu Rev Cell Dev Biol; citation_volume=17; citation_number=43; citation_pages=701-752; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=The ER-peroxisome connection in plants: development of the “ER semi-autonomous peroxisome maturation and replication” model for plant peroxisome biogenesis.; citation_author=RT Mullen; citation_author=RN Trelease; citation_journal_title=Biochim Biophys Acta; citation_volume=1763; citation_number=44; citation_pages=1655-1668; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Contribution of the endoplasmic reticulum to peroxisome formation.; citation_author=D Hoepfner; citation_author=D Schildknegt; citation_author=I Braakman; citation_author=P Philippsen; citation_author=HF Tabak; citation_journal_title=Cell; citation_volume=122; citation_number=45; citation_pages=85-95; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Isolated plant nuclei nucleate microtubule assembly: the nuclear surface in higher plants has centrosome-like activity.; citation_author=V Stoppin; citation_author=M Vantard; citation_author=AC Schmit; citation_author=AM Lambert; citation_journal_title=Plant Cell; citation_volume=6; citation_number=46; citation_pages=1099-1106; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=The economics of ribosome biosynthesis in yeast.; citation_author=JR Warner; citation_journal_title=Trends Biochem Sci; citation_volume=24; citation_number=47; citation_pages=437-440; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Relationship between the replicative age and cell volume in Saccharomyces cerevisiae.; citation_author=R Zadrag; citation_author=M Kwolek-Mirek; citation_author=G Bartosz; citation_author=T Bilinski; citation_journal_title=Acta Biochim Pol; citation_volume=53; citation_number=48; citation_pages=747-751; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Regulation of cell size in the yeast Saccharomyces cerevisiae.; citation_author=GC Johnston; citation_author=A Ehrhardt; citation_author=A Lorincz; citation_author=BL Carter; citation_journal_title=J Bacteriol; citation_volume=137; citation_number=49; citation_pages=1-5; citation_date=1979; " />
      <meta name="citation_reference" content="citation_title=Three-dimensional structure of Mycoplasma pneumoniae's attachment organelle and a model for its role in gliding motility.; citation_author=GP Henderson; citation_author=GJ Jensen; citation_journal_title=Mol Microbiol; citation_volume=60; citation_number=50; citation_pages=376-385; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Multiple large filament bundles observed in Caulobacter crescentus by electron cryotomography.; citation_author=A Briegel; citation_author=DP Dias; citation_author=Z Li; citation_author=RB Jensen; citation_author=AS Frangakis; citation_journal_title=Mol Microbiol; citation_volume=62; citation_number=51; citation_pages=5-14; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Magnetosomes are cell membrane invaginations organized by the actin-like protein MamK.; citation_author=A Komeili; citation_author=Z Li; citation_author=DK Newman; citation_author=GJ Jensen; citation_journal_title=Science; citation_volume=311; citation_number=52; citation_pages=242-245; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Cryo-electron tomography reveals the cytoskeletal structure of Spiroplasma melliferum.; citation_author=J Kurner; citation_author=AS Frangakis; citation_author=W Baumeister; citation_journal_title=Science; citation_volume=307; citation_number=53; citation_pages=436-438; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=An acidic protein aligns magnetosomes along a filamentous structure in magnetotactic bacteria.; citation_author=A Scheffel; citation_author=M Gruska; citation_author=D Faivre; citation_author=A Linaroudis; citation_author=JM Plitzko; citation_journal_title=Nature; citation_volume=440; citation_number=54; citation_pages=110-114; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Culture of phytoplankton for feeding marine invertebrates.; citation_author=RR Guillard; citation_number=55; citation_pages=29-53; citation_date=1975; citation_publisher=Plenum Publishing Corporation; " />
      <meta name="citation_reference" content="citation_title=Electron cryotomography sample preparation using the Vitrobot.; citation_author=CV Iancu; citation_author=WF Tivol; citation_author=JB Schooler; citation_author=DP Dias; citation_author=GP Henderson; citation_journal_title=Nat Protoc; citation_volume=1; citation_number=56; citation_pages=2813-2819; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Observations on the behavior of vitreous ice at approximately 82 and approximately 12 K.; citation_author=ER Wright; citation_author=CV Iancu; citation_author=WF Tivol; citation_author=GJ Jensen; citation_journal_title=J Struct Biol; citation_volume=153; citation_number=57; citation_pages=241-252; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=An improved strategy for automated electron microscopic tomography.; citation_author=QS Zheng; citation_author=MB Braunfeld; citation_author=JW Sedat; citation_author=DA Agard; citation_journal_title=J Struct Biol; citation_volume=147; citation_number=58; citation_pages=91-101; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Automated molecular microscopy: the new Leginon system.; citation_author=C Suloway; citation_author=J Pulokas; citation_author=D Fellmann; citation_author=A Cheng; citation_author=F Guerra; citation_journal_title=J Struct Biol; citation_volume=151; citation_number=59; citation_pages=41-60; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Dual-axis tomography: an approach with alignment methods that preserve resolution.; citation_author=DN Mastronarde; citation_journal_title=J Struct Biol; citation_volume=120; citation_number=60; citation_pages=343-352; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Noise reduction in electron tomographic reconstructions using nonlinear anisotropic diffusion.; citation_author=AS Frangakis; citation_author=R Hegerl; citation_journal_title=J Struct Biol; citation_volume=135; citation_number=61; citation_pages=239-250; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Bsoft: Image processing and molecular modeling for electron microscopy.; citation_author=JB Heymann; citation_author=DM Belnap; citation_journal_title=J Struct Biol; citation_volume=157; citation_number=62; citation_pages=3-18; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Peach: a simple Perl-based system for distributed computation and its application to cryo-EM data processing.; citation_author=PA Leong; citation_author=JB Heymann; citation_author=GJ Jensen; citation_journal_title=Structure; citation_volume=13; citation_number=63; citation_pages=505-511; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Identification of macromolecular complexes in cryoelectron tomograms of phantom cells.; citation_author=AS Frangakis; citation_author=J Bohm; citation_author=F Forster; citation_author=S Nickell; citation_author=D Nicastro; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=99; citation_number=64; citation_pages=14153-14158; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Structure of the 80S ribosome from Saccharomyces cerevisiae–tRNA-ribosome and subunit-subunit interactions.; citation_author=CM Spahn; citation_author=R Beckmann; citation_author=N Eswar; citation_author=PA Penczek; citation_author=A Sali; citation_journal_title=Cell; citation_volume=107; citation_number=65; citation_pages=373-386; citation_date=2001; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000749" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="3-D Ultrastructure of O. tauri: Electron Cryotomography of an Entire Eukaryotic Cell"/>
    <meta name="twitter:description" content="The hallmark of eukaryotic cells is their segregation of key biological functions into discrete, membrane-bound organelles. Creating accurate models of their ultrastructural complexity has been difficult in part because of the limited resolution of light microscopy and the artifact-prone nature of conventional electron microscopy. Here we explored the potential of the emerging technology electron cryotomography to produce three-dimensional images of an entire eukaryotic cell in a near-native state. Ostreococcus tauri was chosen as the specimen because as a unicellular picoplankton with just one copy of each organelle, it is the smallest known eukaryote and was therefore likely to yield the highest resolution images. Whole cells were imaged at various stages of the cell cycle, yielding 3-D reconstructions of complete chloroplasts, mitochondria, endoplasmic reticula, Golgi bodies, peroxisomes, microtubules, and putative ribosome distributions in-situ. Surprisingly, the nucleus was seen to open long before mitosis, and while one microtubule (or two in some predivisional cells) was consistently present, no mitotic spindle was ever observed, prompting speculation that a single microtubule might be sufficient to segregate multiple chromosomes."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0000749.g014"/>

  <meta property="og:title" content="3-D Ultrastructure of O. tauri: Electron Cryotomography of an Entire Eukaryotic Cell" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000749" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=8362'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=3692'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=7420&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0000749">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000749" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000749&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    3-D Ultrastructure of <i>O. tauri</i>: Electron Cryotomography of an Entire Eukaryotic Cell
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Gregory P. Henderson, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Division of Biology, California Institute of Technology, Pasadena, California, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Lu Gan, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Division of Biology, California Institute of Technology, Pasadena, California, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Grant J. Jensen
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:jensen@caltech.edu">jensen@caltech.edu</a></p>

                <p>Affiliation:
                  Division of Biology, California Institute of Technology, Pasadena, California, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: August 15, 2007</li>
    <li>DOI: 10.1371/journal.pone.0000749</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0000749" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0000749" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000749" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0000749" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0000749-g001" data-doi="info:doi/10.1371/journal.pone.0000749" data-uri="info:doi/10.1371/journal.pone.0000749.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000749.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000749-g002" data-doi="info:doi/10.1371/journal.pone.0000749" data-uri="info:doi/10.1371/journal.pone.0000749.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000749.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000749-g003" data-doi="info:doi/10.1371/journal.pone.0000749" data-uri="info:doi/10.1371/journal.pone.0000749.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000749.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000749-g004" data-doi="info:doi/10.1371/journal.pone.0000749" data-uri="info:doi/10.1371/journal.pone.0000749.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000749.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000749-g005" data-doi="info:doi/10.1371/journal.pone.0000749" data-uri="info:doi/10.1371/journal.pone.0000749.g005" title="Figure 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000749.g005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000749-g006" data-doi="info:doi/10.1371/journal.pone.0000749" data-uri="info:doi/10.1371/journal.pone.0000749.g006" title="Figure 6">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000749.g006&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000749-g007" data-doi="info:doi/10.1371/journal.pone.0000749" data-uri="info:doi/10.1371/journal.pone.0000749.g007" title="Figure 7">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000749.g007&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000749-g008" data-doi="info:doi/10.1371/journal.pone.0000749" data-uri="info:doi/10.1371/journal.pone.0000749.g008" title="Figure 8">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000749.g008&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000749-g009" data-doi="info:doi/10.1371/journal.pone.0000749" data-uri="info:doi/10.1371/journal.pone.0000749.g009" title="Figure 9">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000749.g009&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000749-g010" data-doi="info:doi/10.1371/journal.pone.0000749" data-uri="info:doi/10.1371/journal.pone.0000749.g010" title="Figure 10">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000749.g010&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000749-g011" data-doi="info:doi/10.1371/journal.pone.0000749" data-uri="info:doi/10.1371/journal.pone.0000749.g011" title="Figure 11">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000749.g011&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000749-g012" data-doi="info:doi/10.1371/journal.pone.0000749" data-uri="info:doi/10.1371/journal.pone.0000749.g012" title="Figure 12">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000749.g012&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000749-g013" data-doi="info:doi/10.1371/journal.pone.0000749" data-uri="info:doi/10.1371/journal.pone.0000749.g013" title="Figure 13">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000749.g013&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000749-g014" data-doi="info:doi/10.1371/journal.pone.0000749" data-uri="info:doi/10.1371/journal.pone.0000749.g014" title="Figure 14">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000749.g014&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000749">Reader Comments (3)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0000749" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1"></a><p>The hallmark of eukaryotic cells is their segregation of key biological functions into discrete, membrane-bound organelles. Creating accurate models of their ultrastructural complexity has been difficult in part because of the limited resolution of light microscopy and the artifact-prone nature of conventional electron microscopy. Here we explored the potential of the emerging technology electron cryotomography to produce three-dimensional images of an entire eukaryotic cell in a near-native state. <em>Ostreococcus tauri</em> was chosen as the specimen because as a unicellular picoplankton with just one copy of each organelle, it is the smallest known eukaryote and was therefore likely to yield the highest resolution images. Whole cells were imaged at various stages of the cell cycle, yielding 3-D reconstructions of complete chloroplasts, mitochondria, endoplasmic reticula, Golgi bodies, peroxisomes, microtubules, and putative ribosome distributions <em>in-situ</em>. Surprisingly, the nucleus was seen to open long before mitosis, and while one microtubule (or two in some predivisional cells) was consistently present, no mitotic spindle was ever observed, prompting speculation that a single microtubule might be sufficient to segregate multiple chromosomes.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>Henderson GP, Gan L, Jensen GJ (2007) 3-D Ultrastructure of <em>O. tauri</em>: Electron Cryotomography of an Entire Eukaryotic Cell. PLoS ONE 2(8):
          e749.
            doi:10.1371/journal.pone.0000749</p><p><strong>Academic Editor: </strong>Paul Digard, University of Cambridge, United Kingdom</p><p><strong>Received:</strong> May 21, 2007; <strong>Accepted:</strong> July 16, 2007; <strong>Published:</strong> August 15, 2007</p><p><strong>Copyright:</strong> © 2007 Henderson et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>This work was supported in part by NIH grants R01 AI067548 and PO1 GM66521 and DOE grant DE-FG02-04ER63785 to GJJ, a Searle Scholar Award to GJJ, the Beckman Institute at Caltech, and gifts to Caltech from the Gordon and Betty Moore Foundation and Agouron Institute. Lu Gan was supported by a fellowship from the Damon Runyon Cancer Research Foundation (DRG-1940-07). These sponsors/funders had no role in the design and conduct of the study, in the collection, analysis, and interpretation of the data, and in the preparation, review, or approval of the manuscript.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>The history of cell biology has been punctuated by major advances in imaging technologies. Following the invention of the electron microscope in the early 1930s, what we would now call the “conventional” specimen preparation methods of chemical fixation, dehydration, plastic-embedding, sectioning, and staining were developed to allow the visualization of biological material. While producing some artifacts, these methods have been profoundly successful, filling our textbooks with information about the structure and positions of cell walls, internal membranes, cytoskeletal filaments, and even large cytoplasmic particles like ribosomes. High-pressure freezing/freeze substitution fixation techniques have since improved specimen preservation <a href="#pone.0000749-McDonald1">[1]</a>.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>Recent technological developments have created the opportunity to realize two major improvements: imaging cells in three dimensions (3-D) and in nearly-native states. Cells can be imaged in 3-D through tomography <a href="#pone.0000749-Lucic1">[2]</a>, a technique wherein specimens are imaged iteratively while being incrementally tilted around one or two axes. Full 3-D reconstructions of the sample can then be calculated. Computationally stacking such 3-D reconstructions of serial (thick) sections has made it possible to reconstruct even large regions of fixed cells <a href="#pone.0000749-Hoog1">[3]</a>. With conventional sample preparation methods, however, image contrast arises from heavy metal salts surrounding cross-linked molecules held in place by a resin support. It has now also become possible to image non-chemically-fixed cells in a more nearly native state through plunge-freezing. Plunge-freezing preserves cells in a life-like, “frozen-hydrated” state free of stains and with minimal artifacts <a href="#pone.0000749-Dubochet1">[4]</a>. Electron cryotomography (ECT) combines these two advances to produce 3-D reconstructions of nearly-native biological material to “molecular” resolution <a href="#pone.0000749-Grimm1">[5]</a>.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>Because of multiple scattering, however, images of samples thicker than about 500 nm are difficult to interpret. Thus previous ECT work has focused on purified macromolecules, viruses, small prokaryotic cells <a href="#pone.0000749-Jensen1">[6]</a>, purified organelles <a href="#pone.0000749-Nicastro1">[7]</a>, or cell peripheries <a href="#pone.0000749-Medalia1">[8]</a>, <a href="#pone.0000749-Subramaniam1">[9]</a>, but the potential insight that might come from examining whole eukaryotic cells by ECT has not yet been realized because they are too thick. To this end methods are being developed to either cryosection frozen-hydrated tissues <a href="#pone.0000749-AlAmoudi1">[10]</a> or focused-ion-beam mill thin slabs <a href="#pone.0000749-Marko1">[11]</a> suitable for tomography. Here we have taken an alternative approach by identifying and imaging the smallest known eukaryotic cell, <em>Ostreococcus tauri</em>.</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p><em>O. tauri</em> is a unicellular green alga (a “picoplankton”) with only a single mitochondrion, chloroplast, and Golgi body <a href="#pone.0000749-ChrtiennotDinet1">[12]</a>. It has been reported to be less than a micron in diameter, and because of its small size and presumed simplicity, it has been put forward as a model eukaryotic organism <a href="#pone.0000749-Derelle1">[13]</a>. Nevertheless when <em>O. tauri</em>’s 12.6 megabase nuclear genome was sequenced <a href="#pone.0000749-Derelle1">[13]</a>, it was found to have more genes (~8,200) than <em>Saccharomyces cerevisiae</em> (~6,600). These genes are distributed on 20 linear chromosomes. The Prasinophyceae lineage to which <em>O. tauri</em> belongs represents an early branch from the green lineage that includes the land plants <a href="#pone.0000749-Courties1">[14]</a>, <a href="#pone.0000749-Yoon1">[15]</a>. Unlike other model plants <em>O. tauri</em> has only one copy of each class of cyclin-dependent kinases and cyclins <a href="#pone.0000749-Robbens1">[16]</a>, making it a powerful system to study the cell cycle without concerns of genetic redundancy. <em>O. tauri</em>’s cell cycle can be partially synchronized by growing the cells in a twelve hour light-and-dark cycle and further synchronization can be achieved pharmacologically <a href="#pone.0000749-Farinas1">[17]</a>. Because of its small size, it may be the only eukaryotic cell known that can be effectively imaged intact throughout its entire life cycle by ECT.</p>
<a id="article1.body1.sec1.p5" name="article1.body1.sec1.p5"></a><p>Here we present ECT reconstructions of intact <em>O. tauri</em> cells frozen at different stages throughout the cell cycle. In addition to providing new insights into the ultrastructure of chloroplasts, mitochondria, endoplasmic reticula (ER), Golgi bodies, peroxisomes, microtubules, and the distribution of large macromolecular complexes, <em>O. tauri</em> presented a number of surprises including a nuclear envelope (NE) that was open throughout most of the cell cycle, an unusually shaped ER, and a rough ER that was confined to the outer nuclear membrane. Perhaps most intriguingly, only one or two microtubules were ever observed in individual cells, suggesting that single microtubules might be sufficient to segregate multiple chromosomes. The Results and Discussion are presented one after another together for each topic below to improve the flow of the text, but a Conclusion has been added to place the whole work in context.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results and Discussion"></a><h3>Results and Discussion</h3>
<h4>Cell Growth and Synchronization</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>Cells were obtained from a strain of <em>O. tauri</em> isolated from the Thau lagoon (France) <a href="#pone.0000749-Courties2">[18]</a>. <em>O. tauri</em>’s growth was synchronized using a twelve-hour-light, twelve-hour-dark cycle that resulted in an average of one cell division per day. A previous study found that under these growth conditions, cells at the dark-to-light transition were in the mid-G<sub>1</sub> phase of the cell cycle. Cells at the light-to-dark transition were found in various stages of the cell cycle including approximately 50% in early G<sub>1</sub>, 10% in late G<sub>1</sub>, 25% in S, and 15% in G<sub>2</sub>/M <a href="#pone.0000749-Farinas1">[17]</a>.</p>


<h4>Sample Preservation Strategies</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>Cells from liquid cultures were prepared for EM using three different preparation methods: conventional fixation and embedding, high-pressure freezing/freeze-substitution fixation, and plunge-freezing (<a href="#pone-0000749-g001">Figure 1</a>). Conventional EM fixation revealed most membranes, but the membrane structures were badly distorted and any further details were difficult to resolve and/or poorly reproducible. High-pressure frozen specimens had markedly improved membrane preservation, especially in the chloroplast, but macromolecular complexes were not resolved. Further, because of stain and the generally high-density of the plastic-embedded samples, sections thick enough to contain entire cells could not be imaged. Thus reconstructing an entire cell would have required multiple serial sections. Plunge-frozen cells were imageable intact and exhibited good ultrastructural preservation including readily resolvable ribosome-like particles, microtubules, and smooth cellular membranes. Moreover, plunge-freezing was both more consistently successful and significantly faster than high-pressure freezing, allowing a large number of whole cells to be imaged and a more comprehensive survey of <em>O. tauri</em> ultrastructure to be obtained. The plunge-frozen cells were, however, at the limit of useful thickness for ECT (500–800 nm), so data could not be collected at tilt angles greater than ~55–60° and the resolution of the reconstructions was correspondingly anisotropic.</p>
<div class="figure" id="pone-0000749-g001"><div class="img"><a name="pone-0000749-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000749.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000749" data-uri="info:doi/10.1371/journal.pone.0000749.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000749.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000749.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000749.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000749.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000749.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000749.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000749.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000749.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000749.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Preservation of <em>O. tauri</em>.</span></strong></p><a id="article1.body1.sec2.sec2.fig1.caption1.p1" name="article1.body1.sec2.sec2.fig1.caption1.p1"></a><p>(A) Projection image through a conventionally fixed, embedded, thin-sectioned cell. (B) Projection image through a high-pressure frozen, low-temperature embedded, thin-sectioned cell. (C) 28.8-nm thick slice through the 3-D electron cryotomographic reconstruction of an intact, frozen-hydrated cell. The letters identify nuclei (n), chloroplasts (c), mitochondria (m), Golgi bodies (g), the Endoplasmic Reticulum (er), and granules (gr). Scale bar = 200 nm.</p>
<span>doi:10.1371/journal.pone.0000749.g001</span></div>

<h4>Data Recorded</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>Cultures at both the dark-to-light and light-to-dark transitions were plunge-frozen. Samples were first surveyed in the EM to investigate the cell sizes and morphologies present. Cells were almost always round, and were seen to vary in diameter continuously from 1 to 2 µ on every grid, but cells were generally larger at the light-to-dark transition. In many of the cells frozen at the light-to-dark transition, it was possible to see in the surveys that the organelles were dividing. Full tilt-series of 52 cells were then recorded. Most of these were chosen because they were especially thin, which results in higher resolution reconstructions, or because they had visibly dividing organelles, but some were chosen almost randomly to broaden coverage of the cell cycle. The reconstructed cells were classified into two groups, “non-dividing” and “predivisional,” based on whether or not their organelles were visibly dividing (<a href="#pone-0000749-g002">Figure 2</a> and in particular see <a href="#pone.0000749.s001">Video S1</a>, which presents a complete 3-D tour of one representative cell and all its organelles). Of the 23 reconstructed cells taken from the dark-to-light transition, all but one were non-dividing, consistent with their being in G<sub>1</sub> as predicted by Farinas <em>et al</em>, 2006. Of the 29 reconstructed cells taken from the light-to-dark transition, 4 were non-dividing and 25 were predivisional.</p>
<div class="figure" id="pone-0000749-g002"><div class="img"><a name="pone-0000749-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000749.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000749" data-uri="info:doi/10.1371/journal.pone.0000749.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000749.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000749.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000749.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000749.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000749.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000749.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000749.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000749.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000749.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Cross-sections and 3-D segmentations of <em>O. tauri</em> cells.</span></strong></p><a id="article1.body1.sec2.sec3.fig1.caption1.p1" name="article1.body1.sec2.sec3.fig1.caption1.p1"></a><p>Each row shows a single slice (A 21.6-nm; B 36-nm; C 9.6-nm) through the middle (left) and a manually segmented model (two perpendicular views, right) of one particular cell: (A, B) a non-dividing cell harvested at the dark-to-light transition; (C, D) a dividing cell harvested at the light-to-dark transition; (E, F) an unusual cell with a different internal organization and two membrane-bound cell extensions. Only the central portion of the cell in (F) is shown due to the limited ability to completely segment thicker cells. Here and below the letters and colors identify nuclei (n, red), nuclear envelope (ne), chloroplasts (c, green), mitochondria (m, dark purple), Golgi bodies (g, yellow), peroxisomes (p, orange), granules (gr, dark blue), inner membranes including ER (er, light blue), microtubules (light purple), and ribosome-like particles (r). Because the dividing cell shown in panels C and D was especially thick, only its middle region was segmentable. Scale bar 250 nm.</p>
<span>doi:10.1371/journal.pone.0000749.g002</span></div>

<h4>Cell shape and content</h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>While the limited tilt range decreased the resolution perpendicular to the grid, it was nevertheless clear that the frozen-hydrated cells were not spherical. Instead they were rather flattened, ranging from 1.1 to 2.1 µm in diameter and 0.5 to 0.8 µm in thickness. The more flattened surfaces are referred to herein as the “top” and “bottom” of the cells, depending on their orientation on the grid, but there were no consistent structural features that indicated any functional distinction. The diameters and flatness observed here were consistent with earlier reports <a href="#pone.0000749-ChrtiennotDinet1">[12]</a>, but to check whether surface tension flattened the cells beyond their normal state, cell shapes were also investigated by phase and differential interference contrast light microscopy of free-floating live cells. Free-floating cells also had flat profiles, but exhibited more complex morphologies with bends and depressions as if they were quite flexible (<a href="#pone-0000749-g003">Figure 3</a>). Thus while the flattened morphology is natural, suspending the cells in thin fluid across circular holes on an EM grid likely promoted a more round and simple shape.</p>
<div class="figure" id="pone-0000749-g003"><div class="img"><a name="pone-0000749-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000749.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000749" data-uri="info:doi/10.1371/journal.pone.0000749.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000749.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000749.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000749.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000749.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000749.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000749.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000749.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000749.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000749.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Light microscope images of free-floating <em>O. tauri.</em></span></strong></p><a id="article1.body1.sec2.sec4.fig1.caption1.p1" name="article1.body1.sec2.sec4.fig1.caption1.p1"></a><p>Sequential DIC images of five representative cells (1–5) tumbling freely in sea water culture medium after six hours growth in light are shown from left to right, demonstrating their disk-like shape.</p>
<span>doi:10.1371/journal.pone.0000749.g003</span></div><a id="article1.body1.sec2.sec4.p2" name="article1.body1.sec2.sec4.p2"></a><p>All cells imaged by ECT contained a single mitochondrion, Golgi body, and chloroplast. Each cell also contained a nucleus with one to three discernable nuclear pore complexes (NPCs), a smooth ER, and a variety of vesicles that could not be identified. The non-dividing cells had a single peroxisome and microtubule, but some predivisional cells had two copies of each. The rough ER was limited only to the outer NE. One unusual cell had an oval shape with two terminal membrane-bound cell extensions (<a href="#pone-0000749-g002">Figure 2E–F</a>). This cell was thinner than the others, had the long axis of the chloroplast turned ninety degrees relative to the long axis of the nucleus, and had giant holes in the NE.</p>


<h4>Cellular Organization</h4>
<a id="article1.body1.sec2.sec5.p1" name="article1.body1.sec2.sec5.p1"></a><p>The organization of the non-dividing cells was remarkably consistent. The largest organelles – the chloroplast and nucleus – resided in opposing semi-circular halves of the cell. The other organelles (mitochondrion, peroxisome, ER, Golgi body, vesicles) rested between them (<a href="#pone-0000749-g004">Figure 4</a>). The nucleus and chloroplast pressed so closely to the plasma membrane that macromolecular complexes like ribosomes were sterically excluded. While the mitochondrion did not appear to have a fixed location and the chloroplast rested opposite the nucleus, the nuclear envelope (NE) appeared to organize the rest of the organelles. The outer nuclear membrane was contiguous with sheets of ER that were found to run parallel to the plasma membrane and around co-localized granules. The Golgi body was always found close to the nucleus. A single microtubule could be found in contact with the NE, near the top or bottom of the cell. The peroxisome, a large single-membrane vesicle known to be present in all eukaryotic cells, occupied the central region of the cell between the nucleus and the chloroplast (discussed later).</p>
<div class="figure" id="pone-0000749-g004"><div class="img"><a name="pone-0000749-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000749.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000749" data-uri="info:doi/10.1371/journal.pone.0000749.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000749.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000749.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000749.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000749.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000749.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000749.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000749.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000749.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000749.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>3-D segmentations of six cells imaged at the dark-to-light transition (mid G<sub>1</sub>).</span></strong></p><a id="article1.body1.sec2.sec5.fig1.caption1.p1" name="article1.body1.sec2.sec5.fig1.caption1.p1"></a><p>The cell in panel F is the same as in <a href="#pone-0000749-g001">Figure 1A–B</a> and <a href="#pone-0000749-g012">12</a>. For scale, the diameter of the cell in panel D is approximately 1750 nm in diameter.</p>
<span>doi:10.1371/journal.pone.0000749.g004</span></div><a id="article1.body1.sec2.sec5.p2" name="article1.body1.sec2.sec5.p2"></a><p>Six of the best reconstructed cells were used to estimate the volumes and surface areas of key organelles. These six (shown in <a href="#pone-0000749-g004">Figure 4</a>) were chosen because they were among the thinnest cells and because they were imaged along two orthogonal axes, a practice that minimizes the effects of the missing wedge of data in ECT <a href="#pone.0000749-Iancu1">[19]</a>. All six came from the dark-to-light transition. These presumably underwent cell division at the beginning of the 12-hour dark period, entered G<sub>1</sub>, and then grew minimally due to lack of light. The average cell volume and surface area was 0.91±0.07 µm<sup>3</sup> and 8.3 µm<sup>2</sup>, respectively. The chloroplasts, nuclei, and mitochondria were the largest three organelles, occupying 47±3, 17±2, and 2.7±0.4% of the cell volume, respectively. The cytoplasm around the organelles occupied another 29±2% of the cell and granules and other vesicles occupied the remainder of the volume. In these thinnest of cells, the distribution of organellar volumes was remarkably consistent.</p>
<a id="article1.body1.sec2.sec5.p3" name="article1.body1.sec2.sec5.p3"></a><p>The predivisional class of cells was distinguished by four ultrastructural differences. (1) The most obvious difference was the partial division of the chloroplast along a line approximately perpendicular to the boundary between the nucleus and chloroplast, setting up an apparent roughly two-fold-symmetric division plane (<a href="#pone-0000749-g002">Figure 2C–D</a>, <a href="#pone-0000749-g006">6C, 6D</a>, and <a href="#pone-0000749-g007">7C</a>). In most of the predivisional cells (62%), there was a space between the chloroplast and the plasma membrane filled with either ribosome-like complexes or the mitochondria (<a href="#pone-0000749-g005">Figure 5A</a>), as if the forces positioning the chloroplast immediately next to the membrane in G<sub>1</sub> were weakened. (2) The nucleus was no longer oval. In some cases the nucleus was wedge-shaped, protruding into the middle of the dividing chloroplast (<a href="#pone-0000749-g002">Figure 2C–D</a>). In twelve other cases (46%) the NE formed extensions that streamed above/below the chloroplast, sometimes accompanied by the Golgi body or microtubule (<a href="#pone-0000749-g005">Figure 5A–B</a>). (3) Whereas in G<sub>1</sub> the mitochondrion appeared to be randomly positioned, in dividing cells at least part of it consistently crossed the apparent division plane. In some cases it seemed to be straddling the division plane (<a href="#pone-0000749-g002">Figure 2C–D</a>), in others it was elongated as if at least one edge were being drawn to the division plane (<a href="#pone-0000749-g005">Figure 5C</a>). (4) The oval storage granules lined up in a symmetric “V” shape across the division plane pointing towards the chloroplast with the tip of the “V” located at the cell’s center (<a href="#pone-0000749-g006">Figure 6D</a>).</p>
<div class="figure" id="pone-0000749-g005"><div class="img"><a name="pone-0000749-g005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000749.g005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000749" data-uri="info:doi/10.1371/journal.pone.0000749.g005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000749.g005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000749.g005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000749.g005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000749.g005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000749.g005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000749.g005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000749.g005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000749.g005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000749.g005.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 5.  <span>Cells with dividing organelles.</span></strong></p><a id="article1.body1.sec2.sec5.fig2.caption1.p1" name="article1.body1.sec2.sec5.fig2.caption1.p1"></a><p>(A) Segmentation showing a portion of a NE streaming over the top of a chloroplast. Here the chloroplast is separated from the plasma membrane by the mitochondrion and several granules. (B) 36-nm thick slice near the top of the reconstruction of the same cell. The NE (white arrows) stretches towards the Golgi body (black arrowheads) and microtubule (black arrows). (C) 24-nm slice through a different cell, showing an elongated mitochondrion near the center of the cell and duplicating chloroplast granules. Scale bar = 150 nm.</p>
<span>doi:10.1371/journal.pone.0000749.g005</span></div><div class="figure" id="pone-0000749-g006"><div class="img"><a name="pone-0000749-g006" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000749.g006&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000749" data-uri="info:doi/10.1371/journal.pone.0000749.g006"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000749.g006&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000749.g006/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000749.g006/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000749.g006/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000749.g006/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000749.g006.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000749.g006/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000749.g006/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000749.g006.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 6.  <span>Chloroplast.</span></strong></p><a id="article1.body1.sec2.sec5.fig3.caption1.p1" name="article1.body1.sec2.sec5.fig3.caption1.p1"></a><p>(A) 7.2-nm thick slice through a non-dividing cell. The starch granule (white arrow) has suffered damage from the electron beam. Besides it are two dark granules (white arrowhead). (B) Above: Enlarged view of the boxed area in panel A. The three thylakoid membranes (black arrowhead) can be seen forming a stack. Below: Schematic of above. Cell membrane (black), chloroplast membranes (green), outer thylakoid membrane (red), inner thylakoid membranes (blue). (C) 24-nm slice through an early predivisional cell, where the chloroplast is kidney-shaped rather than oval and the starch granule is elongated. (D) 36-nm thick slice through a late predivisional cell, where the chloroplast is deeply constricted and one dark and one starch granule is found in each side. Here the cytoplasmic granules (*) are arranged in a V-shape pointing to the division plane. Scale bar 250 nm.</p>
<span>doi:10.1371/journal.pone.0000749.g006</span></div>

<h4>Chloroplasts</h4>
<a id="article1.body1.sec2.sec6.p1" name="article1.body1.sec2.sec6.p1"></a><p>The chloroplast was the largest organelle and since it was also one of the most dense, its internal structure was the most difficult to resolve. Due to the missing wedge artifact, thylakoid grana that may have been present at the top and bottom of the chloroplast could not be resolved. In the midsection, grana composed of three disks (six membranes) hugged the inner chloroplast membrane and sometimes extended into the middle of the chloroplast as broad sheets without connecting stroma-like thylakoid structures (<a href="#pone-0000749-g006">Figure 6A–B</a>). Because the lumens of the two outer disks (about 7 nm across) were consistently wider than that of the central disk, the grana had a sandwich-like appearance: the “top” and “bottom” membranes of the stack stood apart from the inner four, which were packed so tightly that they were often hard to distinguish. The grana thylakoids terminated at the poles of the long axis of the chloroplast (<a href="#pone-0000749-g006">Figure 6A–B</a>), but the connectivity of the individual membranes and the details of the presumed organizing scaffold were unclear. No connections or vesicles between the grana thylakoids and the inner chloroplast membrane were seen.</p>
<a id="article1.body1.sec2.sec6.p2" name="article1.body1.sec2.sec6.p2"></a><p>The chloroplasts contained two distinct classes of granules. Each chloroplast had multiple dark and very nearly spherical granules (<a href="#pone-0000749-g006">Figure 6, white arrowheads</a>) and a single larger granule that manifested electron beam damage artifacts (bubbling) much earlier than the rest of the cell (<a href="#pone-0000749-g006">Figure 6, white arrows</a>). Radiation-sensitive granules have also been reported in the gram-negative bacterium <em>Caulobacter crescentus</em> and were found to be carbon-rich <a href="#pone.0000749-Comolli1">[20]</a>. The “bubbled” granule was seen stretched across the division plane in dividing chloroplasts, as expected for the starch granule, which was previously shown to exist as one copy that grew and divided <a href="#pone.0000749-Ral1">[21]</a>. The onset of bubbling in the granule occurred at ~100 electrons/Å<sup>2</sup>, since in the cases of those cells where dual-axis tilt-series were recorded, reconstructions obtained from just the first half of the data (the first tilt ~80 electrons/Å<sup>2</sup>) did not show bubbling. (The higher total doses were used, however, because they led to reconstructions with more interpretable detail, indicating that the perturbations to the rest of the cellular ultrastructure caused by bubbling in the presumed starch granule were minor in comparison to the advantages of higher dose.) The chloroplasts divided along the apparent cellular division plane (<a href="#pone-0000749-g006">Figure 6C–D</a>, <a href="#pone-0000749-g007">7C</a>). As the thylakoids terminated at poles (<a href="#pone-0000749-g006">Figure 6B</a>) away from the division plane, the division would result in one old pole and one new pole for each half of the chloroplast. Each half of dividing chloroplasts contained a portion of the presumed starch granule and at least one dark granule. No completely divided chloroplasts were observed, suggesting that cell division quickly followed the termination of chloroplast division.</p>
<div class="figure" id="pone-0000749-g007"><div class="img"><a name="pone-0000749-g007" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000749.g007&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000749" data-uri="info:doi/10.1371/journal.pone.0000749.g007"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000749.g007&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000749.g007/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000749.g007/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000749.g007/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000749.g007/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000749.g007.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000749.g007/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000749.g007/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000749.g007.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 7.  <span>Nuclear envelope.</span></strong></p><a id="article1.body1.sec2.sec6.fig1.caption1.p1" name="article1.body1.sec2.sec6.fig1.caption1.p1"></a><p>(A) 41-nm thick slice through a cell harvested at the light-to-dark transition with a completely closed NE (black arrow). The cell’s small size and non-dividing organelles suggest the cell could have recently divided. Two NPCs (black arrowheads) are present. Close ups of the NPCs are shown at <a href="#pone-0000749-g008">Figure 8</a>. (B) 31-nm thick slice through a cell harvested at the dark-to-light transition. The NE only covers about three-fourths of the nucleus in this section (black arrows mark the tips). Again two NPCs (black arrowheads) are present, and at the bottom of the nucleus the ER branches off the NE (white arrowheads). (C) 19.2-nm thick slice through a large, late predivisional cell harvested at the light-to-dark transition exhibiting an almost completely open nucleus with only small patches of NE (black arrows). Scale bar 100 nm.</p>
<span>doi:10.1371/journal.pone.0000749.g007</span></div><a id="article1.body1.sec2.sec6.p3" name="article1.body1.sec2.sec6.p3"></a><p>In agreement with the likely evolutionary relationship between chloroplasts and cyanobacteria, the ultrastructure of the <em>O. tauri</em> chloroplasts was reminiscent of cyanobacteria. Both thylakoid systems are comprised almost exclusively of grana <a href="#pone.0000749-vandeMeene1">[22]</a>, <a href="#pone.0000749-Nevo1">[23]</a>, positioned here near the inner chloroplast membrane and near the analogous plasma membrane in cyanobacteria. The chloroplasts also appeared to replicate like cyanobacteria, since their thylakoids were present before division and must have been pinched off at the nascent poles. In contrast, higher plant chloroplasts mature from progenitor proplastids that lack differentiated internal membranes.</p>
<a id="article1.body1.sec2.sec6.p4" name="article1.body1.sec2.sec6.p4"></a><p>Traditional models depict the lumens of thylakoid disks as discreet from the lumen between the chloroplast's inner and outer membranes <a href="#pone.0000749-Vothknecht1">[24]</a>. A recent tomogram of a higher-plant chloroplast supports this traditional “detached” thylakoids model <a href="#pone.0000749-Shimoni1">[25]</a>. Van de Meene et al. showed a cyanobacteria’s thylakoid contacting the inner plasma membrane, however, but the nature of the connection could not be determined <a href="#pone.0000749-vandeMeene1">[22]</a>. Here <em>O. tauri</em> thylakoids appeared detached, but unfortunately the complete chloroplast could not be clearly observed, transient events might not have been captured, and the details of the presumed organizing scaffolding could not be resolved. No transport mechanisms for bringing required lipids to a growing thylakoid, such as vesicles between the inner membrane and thylakoid stacks, were detected.</p>


<h4>Nuclei</h4>
<a id="article1.body1.sec2.sec7.p1" name="article1.body1.sec2.sec7.p1"></a><p>Higher eukaryotes undergo open mitosis, resulting in the partial or complete breakdown of the NE. The onset of mitosis is marked by NE breakdown, NPC disassembly, and mitotic spindle formation. The NE reforms on the chromosomes during metaphase and completely closes after telophase <a href="#pone.0000749-Drummond1">[26]</a>. In contrast, model lower eukaryotes like yeast undergo closed mitosis, in which the NE remains intact. Our reconstructions reveal that <em>O. tauri</em>’s NEs are almost always open, and other mitotic events differ substantially from traditional models.</p>
<a id="article1.body1.sec2.sec7.p2" name="article1.body1.sec2.sec7.p2"></a><p>Throughout the cell cycle the NE had gaps hundreds of nanometers in diameter (<a href="#pone-0000749-g007">Figure 7B, arrows</a>). In fact only one cell was observed with an apparently closed NE (<a href="#pone-0000749-g007">Figure 7A</a>). This small cell was taken from a culture at the light-to-dark transition, and its non-dividing organelles implied that it must have just divided and was in early G<sub>1</sub>. This cell could have had gaps in the top and bottom of the NE, but we could not verify this due to the effects of the missing wedge. Other cells imaged that were also presumably in early G<sub>1</sub> were found to have an open NE. The openings in the NE consistently faced the cytoplasmic space between the nucleus and the chloroplast; the regions facing the plasma membrane were generally closed. In certain rare cases, the nucleus was almost completely unbounded by NE (<a href="#pone-0000749-g007">Figure 7C</a> and <a href="#pone-0000749-g002">2E</a>). In these cells large fragments of NE were seen along the borders of a recognizably dense nucleoplasm, implying that the nucleoplasm is gel-like and able to retain its unique composition.</p>
<a id="article1.body1.sec2.sec7.p3" name="article1.body1.sec2.sec7.p3"></a><p>The holes in the NE are unlikely to be an artifact of sample preservation; four lines of evidence support the authenticity of these holes. First, no other cell membranes showed similar gaps. Second, nuclei have been previously isolated from cells and imaged by ECT, without disruption of the membrane <a href="#pone.0000749-Beck1">[27]</a>. Third, the locations of the holes seen here were consistent, which suggests they form in a regulated manner. Finally, some cells simply did have not enough NE to cover the entire nucleus (<a href="#pone-0000749-g002">Figure 2E</a>), arguing against an acute change.</p>
<a id="article1.body1.sec2.sec7.p4" name="article1.body1.sec2.sec7.p4"></a><p>Near the plasma membrane, the outer and inner NE membranes were too close to be resolved. Elsewhere, these membranes were spaced between 18 and 100 nm apart. NPCs were present (discussed below). As noted previous, in 42% of the predivisional cells, protrusions of the nucleus streamed over the top of the chloroplast (<a href="#pone-0000749-g005">Figure 5A–B</a>). No internal structures (like heterochromatin, nuclear lamina, or condensed chromosomes) were recognized.</p>
<a id="article1.body1.sec2.sec7.p5" name="article1.body1.sec2.sec7.p5"></a><p>Many higher eukaryotes undergo NE breakdown during mitosis, so the fact that a cell can have a NE with large gaps is reasonable. Nevertheless <em>O. tauri</em>’s NE appeared to be open during most of the cell cycle (from the beginning to the end of the light cycle). This is significant because presumably, the nucleus must be closed in order to establish the Ran nucleo-cytoplasmic gradient, which shuttles cargo in and out of the nucleus via the NPCs. The NE had such large gaps that the role of the NPCs is unclear, although they remained embedded in the NE fragments. Perhaps certain molecules like mRNAs require regulated transport and so are specifically targeted to the NPC. Despite being open, many of the nuclei must have been transcriptionally and translationally active since the cells were in the exponential growth phase and ribosomes consistently decorated parts of the NE.</p>
<a id="article1.body1.sec2.sec7.p6" name="article1.body1.sec2.sec7.p6"></a><p>Previously published EM images of <em>O. tauri</em> do not suggest the existence of gaps in the NE <a href="#pone.0000749-Farinas1">[17]</a>. With conventional EM, however, in some cases membranes can be severely distorted and each image generally shows just a thin section through the nucleus (<a href="#pone-0000749-g001">Figure 1</a>). Because ECT offers 3-D views of near-native structures, the gaps were quite clear.</p>


<h4>Nuclear Pore Complexes</h4>
<a id="article1.body1.sec2.sec8.p1" name="article1.body1.sec2.sec8.p1"></a><p>NPCs within the NE were identified as ~80 nm diameter rings of high electron density bordered by a region of the NE where the outer and inner membranes merged (<a href="#pone-0000749-g008">Figure 8</a>). Each cell had one to three visible NPCs, but there may have been additional NPCs within the top or bottom faces of the nuclei that were not recognized because of the missing wedge. As mentioned earlier, the nuclei were immediately adjacent to the plasma membrane, but the NPCs were never found facing that membrane. Instead, the NPCs were found in regions of the NE exposed to the cell's interior, but the mechanisms localizing the NPCs remain unclear. In the four best resolved NPCs, the average “outer” diameter of the rings (where the outer and inner bilayers of the NE could no longer be distinguished, <a href="#pone-0000749-g008">Figure 8, solid lines</a>) was 81±4 nm. The average “inner” diameter of the high density ring (<a href="#pone-0000749-g008">Figure 8, dashed lines</a>) was 48±8 nm; the greater variance being due perhaps to the uncertainty in locating the inner boundary. Other internal densities were seen which may represent cargo, “transporters,” or luminal spoke rings <a href="#pone.0000749-Akey1">[28]</a>.</p>
<div class="figure" id="pone-0000749-g008"><div class="img"><a name="pone-0000749-g008" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000749.g008&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000749" data-uri="info:doi/10.1371/journal.pone.0000749.g008"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000749.g008&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000749.g008/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000749.g008/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000749.g008/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000749.g008/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000749.g008.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000749.g008/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000749.g008/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000749.g008.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 8.  <span>Nuclear pore complex.</span></strong></p><a id="article1.body1.sec2.sec8.fig1.caption1.p1" name="article1.body1.sec2.sec8.fig1.caption1.p1"></a><p>3.6-nm thick slices through the best-resolved NPC from the “side” (perpendicular to the NE, left) and “top” (in plane of NE, right). The approximate center, inner, and outer diameters are marked by the arrow, dashed, and solid lines, respectively. The region around the NPC in the right panel is low density (whiter) because it is inside the NE lumen; the dark crescent near the edge of the panel shows where the plane of the slice cuts through the NE. Scale bar = 100 nm.</p>
<span>doi:10.1371/journal.pone.0000749.g008</span></div><a id="article1.body1.sec2.sec8.p2" name="article1.body1.sec2.sec8.p2"></a><p>While these may be the first measurements of NPCs <em>in-vivo</em>, or of plant or algal NPCs in any state, the <em>O. tauri</em> NPCs observed here were smaller (81 nm) than those from <em>Dictyostelium</em> (125 nm), <em>Xenopus</em> (100 nm), or <em>Saccharomyces</em> (100 nm) <a href="#pone.0000749-Beck1">[27]</a>, <a href="#pone.0000749-Stoffler1">[29]</a>, <a href="#pone.0000749-Yang1">[30]</a>. Unfortunately the resolution was insufficient to detect the symmetry, as was done for isolated <em>Dictyostelium</em> nuclei <a href="#pone.0000749-Beck1">[27]</a>, so it is unclear whether the smaller size results from smaller or fewer constituent proteins. Compared to the <em>Dictyostelium</em> NE (the only other intact nucleus imaged by cryotomography), which has such a high concentration of NPCs (~45/µm<sup>2</sup>) that they exhibit local hexagonal packing <a href="#pone.0000749-Beck1">[27]</a>, <a href="#pone.0000749-Yang1">[30]</a>, <em>O. tauri</em> NE had a much lower NPC concentration (~1/µm<sup>2</sup>).</p>


<h4>Mitochondria</h4>
<a id="article1.body1.sec2.sec9.p1" name="article1.body1.sec2.sec9.p1"></a><p>Each cell had a single mitochondrion. The mitochondria were generally oval (<a href="#pone-0000749-g009">Figure 9</a>), but were frequently deformed when situated adjacent to other organelles. In some dividing cells the mitochondria were elongated (<a href="#pone-0000749-g005">Figure 5C</a>) or constricted in their center (<a href="#pone-0000749-g002">Figure 2C–D</a>). The inner and outer membranes were smooth rather than corrugated as sometimes observed by traditional EM <a href="#pone.0000749-Perkins1">[31]</a>. While the spacing between the inner and outer membrane was generally ~12 nm (<a href="#pone-0000749-g009">Figure 9C–D</a>), it was sometimes significantly greater (top of <a href="#pone-0000749-g009">Figure 9B</a>), or so small that the inner and outer membranes appeared to touch (<a href="#pone-0000749-g009">Figure 9E</a>). The mitochondria positioned near the thinner edges of cells were the best resolved and exhibited both lamellar and tubular cristae, typically with one or two junctions with the inner membrane (arrowheads in <a href="#pone-0000749-g009">Figure 9B–D</a>). These crista junctions were circular or slot-shaped and ranged in size from 15–55 nm (<a href="#pone.0000749.s001">Video S1</a>). The spacing between the two membranes of the cristae was also 15 nm or greater. In some mitochondria (40%), multiple (4–10) dark granules with complex shapes were observed in the matrix (<a href="#pone-0000749-g009">Figure 9A–B</a>).</p>
<div class="figure" id="pone-0000749-g009"><div class="img"><a name="pone-0000749-g009" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000749.g009&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000749" data-uri="info:doi/10.1371/journal.pone.0000749.g009"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000749.g009&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000749.g009/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000749.g009/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000749.g009/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000749.g009/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000749.g009.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000749.g009/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000749.g009/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000749.g009.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 9.  <span>Mitochondrion.</span></strong></p><a id="article1.body1.sec2.sec9.fig1.caption1.p1" name="article1.body1.sec2.sec9.fig1.caption1.p1"></a><p>(A) 3-D segmentation of the mitochondrion from the cell shown in <a href="#pone-0000749-g001">Figure 1E–F</a>. Here, the membranes are colored purple (outer membrane), red (inner membrane), and yellow (cristae). Dense granules within the mitochondrial matrix are shown in green. (B–D) 15-, 55-, and 29-nm thick slices though three different mitochondria, showing crista junctions (arrowheads) and a dense granule (white arrow). Panel B is a slice through the cell shown in panel A. Scale bar 50 nm (for panels B–D). (E) 4.8-nm thick slice through a junction or channel (black arrows) connecting the outer and inner membranes. Here c-cytoplasm, m-mitochondrion. Scale bar 50 nm.</p>
<span>doi:10.1371/journal.pone.0000749.g009</span></div><a id="article1.body1.sec2.sec9.p2" name="article1.body1.sec2.sec9.p2"></a><p>Electron tomographic reconstructions of mitochondria have already discredited the baffle model of membrane folding, where cristae were proposed to fold in much like the bellows of an accordion. Instead, both isolated and <em>in-situ</em> mitochondria have been shown to have tubular and laminar cristae that connect to the inner membrane by circular or slot-shaped crista junctions <a href="#pone.0000749-Nicastro1">[7]</a>, <a href="#pone.0000749-Mannella1">[32]</a>. Our observations in <em>O. tauri</em> add to this body of literature and highlight the consistency of mitochondrial ultrastructure. Notwithstanding the evolutionary pressures that shrunk <em>O. tauri</em> and its mitochondria, the arrangement of its cristae has essentially not changed. This strengthens the idea that the structure of the cristae plays an important role in oxidative phosphorylation, including the possibility that it creates local environments separated by bottlenecks of restricted diffusion <a href="#pone.0000749-Mannella2">[33]</a>–<a href="#pone.0000749-Mannella4">[35]</a>.</p>


<h4>Endoplasmic Reticula</h4>
<a id="article1.body1.sec2.sec10.p1" name="article1.body1.sec2.sec10.p1"></a><p>Sheets and tubes of membranes branching off the NE were identified as ER (<a href="#pone-0000749-g007">Figure 7B, white arrowheads</a>). The inter-membrane spacing within each ER sheet ranged from 14 nm to 80 nm. The ER tubes had diameters of ~14 nm – a value much smaller than the 25–100 nm typically described for smooth ER <a href="#pone.0000749-Staehelin1">[36]</a>, <a href="#pone.0000749-Shibata1">[37]</a>. Interestingly, some ER sheets were perforated by the dark oval granules (<a href="#pone-0000749-g007">Figure 7B</a> and <a href="#pone-0000749-g010">10</a>). Other structures looked just like the ER, but connections to the NE were not found. These consisted of sheets that were found adjacent to the plasma membrane and at the four regions that projected extra-cellular filaments, as discussed later.</p>
<div class="figure" id="pone-0000749-g010"><div class="img"><a name="pone-0000749-g010" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000749.g010&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000749" data-uri="info:doi/10.1371/journal.pone.0000749.g010"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000749.g010&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000749.g010/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000749.g010/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000749.g010/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000749.g010/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000749.g010.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000749.g010/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000749.g010/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000749.g010.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 10.  <span>Endoplasmic reticulum.</span></strong></p><a id="article1.body1.sec2.sec10.fig1.caption1.p1" name="article1.body1.sec2.sec10.fig1.caption1.p1"></a><p>3-D segmentation of a cell harvested at the dark to light transition. ER (light blue) forms a sheet near the edge of the cell, perforated by four granules (dark blue).</p>
<span>doi:10.1371/journal.pone.0000749.g010</span></div><a id="article1.body1.sec2.sec10.p2" name="article1.body1.sec2.sec10.p2"></a><p>Current descriptions of ER ultrastructure assert the notions that (1) the lumens of different ER sheets are interconnected with each other and with the NE lumen, and (2) rough ER forms sheets and all other ER (smooth ER) is tubular <a href="#pone.0000749-Staehelin1">[36]</a>, <a href="#pone.0000749-Shibata1">[37]</a>. <em>O. tauri</em>’s ER differed in both these characteristics. While some ER tubes and sheets appeared to be contiguous with the NE, others had no observable connection. Rapid fission and fusion of membranes can make compartments functionally interconnected, like the Golgi body’s many cisternae, but our “snapshots” of flash-frozen cells suggest that some ER compartments are physically isolated at least part of the time. Less likely but still possibly, connections might have existed but were not observed due to the missing wedge artifact or limits in resolution. Surprisingly, while clusters of ribosome-like particles were found on the cytoplasmic face of the outer nuclear membrane, no ribosome-decorated ER (rough ER) could be found. This contradicts earlier speculations that ribosomes are necessary to maintain the ER’s flat shape <a href="#pone.0000749-Shibata1">[37]</a>, <a href="#pone.0000749-Voeltz1">[38]</a>.</p>


<h4>Golgi bodies</h4>
<a id="article1.body1.sec2.sec11.p1" name="article1.body1.sec2.sec11.p1"></a><p>Golgi bodies were identified as stacks of flat, membranous cisternae positioned next to the nucleus and usually extending towards the chloroplast. No cell had more than one Golgi body, but in some cells (23%, including both non-dividing and predivisional) no Golgi body was found, probably because it was positioned in such a way that the missing wedge artifacts made it impossible to recognize. In mammalian cells the Golgi body disperses prior to cell division <a href="#pone.0000749-Gaietta1">[39]</a>; here the Golgi was frequently observed in predivisional <em>O. tauri</em>.</p>
<a id="article1.body1.sec2.sec11.p2" name="article1.body1.sec2.sec11.p2"></a><p>In <em>O. tauri</em>, the Golgi bodies typically consisted of five uniformly spaced cisternae (<a href="#pone-0000749-g011">Figure 11A–B</a>). Some Golgi bodies exhibited up to seven cisternae, but the extra cisternae were often not consistently spaced or appeared as if they were beginning to leave (<a href="#pone-0000749-g011">Figure 11C, dark blue</a>). The NE was used to determine <em>cis</em> to <em>trans</em> direction because the resolution was insufficient to resolve the protein coats on the vesicles that are normally used to determine the Golgi polarity. Most cisternae were roughly disk shaped (<a href="#pone-0000749-g011">Figure 11A–B</a>), but more elongated cisternae were also seen (<a href="#pone-0000749-g011">Figure 11C</a>). All were slightly curved. The cisterna nearest the nucleus was consistently the smallest, with a diameter and volume of about 150 nm and 5×10<sup>−8</sup> pL, respectively. Its concave (cis) surface faced towards the nucleus. The middle three cisternae were larger, measuring 200 nm in diameter and 7.5×10<sup>−8</sup> pL in volume. The trans-cisterna was 200 nm in diameter and thinner, having a volume similar to the cis-cisterna. None of the cisternae were connected.</p>
<div class="figure" id="pone-0000749-g011"><div class="img"><a name="pone-0000749-g011" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000749.g011&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000749" data-uri="info:doi/10.1371/journal.pone.0000749.g011"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000749.g011&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000749.g011/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000749.g011/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000749.g011/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000749.g011/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000749.g011.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000749.g011/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000749.g011/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000749.g011.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 11.  <span>Golgi body.</span></strong></p><a id="article1.body1.sec2.sec11.fig1.caption1.p1" name="article1.body1.sec2.sec11.fig1.caption1.p1"></a><p>The Golgi bodies from three different cells are shown (rows A–C). In column I, slices through the reconstructions are shown, with cisternae marked by arrowheads. In column II, 3-D segmentations of the Golgi bodies are shown <em>in situ</em> within their cellular contexts. Column III shows the isolated 3-D segmentations from a view perpendicular to the one in column II. The five “core” cisternae common to all Golgi bodies seen here are colored purple, red, gold, yellow, and green (<em>cis</em> to <em>trans</em>). Surrounding vesicles are colored either light or dark blue. Some vesicles in column III do not appear in column II because they are “below” the cellular slice shown. The blue-green vesicle in IIC was removed from IIIC to create an unobstructed view of the Golgi body. Scale bars 100 nm.</p>
<span>doi:10.1371/journal.pone.0000749.g011</span></div><a id="article1.body1.sec2.sec11.p3" name="article1.body1.sec2.sec11.p3"></a><p>The origins and destinations of vesicles in the Golgi are of considerable interest <a href="#pone.0000749-Storrie1">[40]</a>. From the nucleus outwards, occasionally vesicles were seen budding out of the NE towards the cis-cisternae (8% of cells). There were also free-standing vesicles observed between the NE and cis-cisternae. The tubular vesicles in front of the cis-surface (<a href="#pone-0000749-g011">Figure 11, light blue</a>) might have been the fusion of two or more NE-derived vesicles that formed a vesicular tubular cluster. Vesicles were not seen between cisternae, but they were frequently present along the sides of the stacks. The trans-cisterna often abutted the chloroplast, leaving no room for vesicles <em>trans</em> to the cisterna. No vesicles were ever found <em>trans</em> to the Golgi body, but the trans-cisterna often had an appearance of slipping off the stack (<a href="#pone-0000749-g011">Figure 11A–B, green cisternae</a>), and occasionally had an additional cisterna beyond it (<a href="#pone-0000749-g011">Figure 11C, dark blue cisterna</a>). Vesicles were not observed between the Golgi body and the non-nuclear ER.</p>
<a id="article1.body1.sec2.sec11.p4" name="article1.body1.sec2.sec11.p4"></a><p>These observations are consistent with the maturation model of Golgi body transport, but the lack of <em>trans</em> vesicles is surprising. Some of the vesicles along the side of the Golgi body could have originated from the <em>trans</em> surface and moved off to the side, sterically deflected by the chloroplast. It may also be possible that instead of having many small <em>trans</em> vesicles, here at least <a href="#pone-0000749-g011">Figure 11C</a> suggests that the entire trans-cisternae (dark blue) is one transport vesicle. Many cells rely on microtubules to transport Golgi associated vesicles <a href="#pone.0000749-Caviston1">[41]</a>, but as in <em>Schizosaccharomyces pombe</em> <a href="#pone.0000749-Hoog1">[3]</a>, no microtubules were ever found contacting the Golgi body or its neighboring vesicles. Perhaps other, smaller cytoskeletal filaments were present to guide vesicles but were not resolved in our reconstructions.</p>


<h4>Vesicles</h4>
<a id="article1.body1.sec2.sec12.p1" name="article1.body1.sec2.sec12.p1"></a><p>A variety of vesicles were seen. The most numerous type (1–10 or more per cell) were pleomorphic, typically ~200 nm diameter, and occurred in clusters sometimes configured as a “V” in predivisional cells (<a href="#pone-0000749-g002">Figure 2</a>, <a href="#pone-0000749-g004">4</a>, <a href="#pone-0000749-g005">5</a>, <a href="#pone-0000749-g006">6</a>, <a href="#pone-0000749-g010">10</a>). Their high internal density prevented us from determining if there was a surrounding membrane. Presumably these are some type of storage granule. Smaller vesicles were also seen whose size and random locations suggest they could have been involved in transport. In addition to these storage and transport vesicles, all the cells (n = 52) contained at least one characteristically low-density, nearly spherical, second type of vesicle surrounded by a single membrane (<a href="#pone-0000749-g004">Figure 4A and F</a>, <a href="#pone-0000749-g010">10</a>) measuring 166±20 nm in diameter. Some dividing cells had two. Remarkably, these special vesicles were almost always in contact with both the chloroplast and the nucleus (<a href="#pone-0000749-g010">Figure 10</a>), even when they were found far from the main body of the nucleus. In these cases they were found touching long, thin extensions of the NE streaming over the chloroplast (<a href="#pone-0000749-g005">Figure 5A</a>), suggesting connections to both the NE and chloroplast. Because (i) peroxisomes have been described as single-membraned vesicles, (ii) small <em>O. tauri</em> cells are likely to need one and only one peroxisome but multiple storage granules, and (iii) <em>Ostreococcus</em> genomes contain peroxisomal genes <a href="#pone.0000749-Palenik1">[42]</a>, these vesicles were assumed to be peroxisomes. As dividing peroxisomes were never seen, perhaps <em>O. tauri</em> only produces peroxisomes <em>de novo</em> <a href="#pone.0000749-Purdue1">[43]</a>–<a href="#pone.0000749-Hoepfner1">[45]</a>.</p>


<h4>Microtubules</h4>
<a id="article1.body1.sec2.sec13.p1" name="article1.body1.sec2.sec13.p1"></a><p>Most <em>O. tauri</em> cells had a single, hollow, tubular structure 24±1 nm wide and 200–700 nm long assumed here to be a microtubule. The microtubule always rested between the outer nuclear membrane and the plasma membrane against one of the “top” or “bottom” flat faces of the cell (<a href="#pone-0000749-g004">Figure 4, purple rods</a>). Both ends of the microtubule were open and blunt (<a href="#pone-0000749-g012">Figure 12, arrowheads</a>). The microtubules were slightly curved and near one end the shaft made contact with the NE, but the resolution was insufficient to discern whether bridging proteins connected the two (<a href="#pone-0000749-g012">Figure 12B, insert</a>). There was neither an obvious microtubule organizing center nor a flared or capped end to conclusively identify the minus or plus end, respectively. It is possible that the microtubules are organized similarly to those of higher plants, in which the minus end interacts closely with the NE (as discussed later).</p>
<div class="figure" id="pone-0000749-g012"><div class="img"><a name="pone-0000749-g012" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000749.g012&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000749" data-uri="info:doi/10.1371/journal.pone.0000749.g012"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000749.g012&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000749.g012/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000749.g012/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000749.g012/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000749.g012/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000749.g012.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000749.g012/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000749.g012/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000749.g012.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 12.  <span>Microtubule.</span></strong></p><a id="article1.body1.sec2.sec13.fig1.caption1.p1" name="article1.body1.sec2.sec13.fig1.caption1.p1"></a><p>(A) 9.6-nm slice through one particular microtubule (arrowheads) along most of its length, showing its uniform diameter and hollow nature. (B) 16.8-nm slice through a microtubule in its cellular context, showing its open, blunt ends (arrowheads) terminating between the NE (arrows) and plasma membrane. The insert shows a 2-fold enlarged, 38.4-nm thick slice through the microtubule perpendicular to its long axis, emphasizing its tubular shape (arrowheads). Scale bar 100 nm.</p>
<span>doi:10.1371/journal.pone.0000749.g012</span></div><a id="article1.body1.sec2.sec13.p2" name="article1.body1.sec2.sec13.p2"></a><p>Surprisingly, none of the cells exhibited a mitotic spindle even though many stages of division were seen and microtubules were consistently resolved in our reconstructions. Unlike metazoans and yeast, plants lack a structurally distinct microtubule organizing center, but their NEs do organize mitotic spindles that segregate their chromosomes <a href="#pone.0000749-Stoppin1">[46]</a>. Three of the reconstructed <em>O. tauri</em> cells had two microtubules, and in one of these cells they were partially embedded within the open nucleus (the cell in <a href="#pone-0000749-g007">Figure 7C</a>, though the microtubules are not present in the slice shown), but because <em>O. tauri</em> has 20 independent linear chromosomes <a href="#pone.0000749-Derelle1">[13]</a>, a canonical mitotic spindle would have been expected to have more than 40 microtubules. While it is possible that in <em>O. tauri</em> the spindle appears so transiently that by chance none of the 26 predivisional cells reconstructed here contained one, its absence suggests the intriguing possibility that some evolutionary pressure (like size minimization) has caused <em>O. tauri</em> to adapt a novel mechanism for chromosome segregation. Perhaps the twenty pairs of chromosomes are segregated one at a time by the two mitotic microtubules. Alternatively, the chromosomes may be physically linked during mitosis and be co-segregated.</p>


<h4>Ribosome-like complexes</h4>
<a id="article1.body1.sec2.sec14.p1" name="article1.body1.sec2.sec14.p1"></a><p>The cytoplasm contained many discrete large macromolecular complexes, which were presumably mostly ribosomes. The cytoplasms of three non-dividing cells (<a href="#pone-0000749-g004">D, E, and F in Figure 4</a>) were searched by cross-correlation for ribosome-like particles using a low-pass-filtered ribosome model as template. Visual inspection of the search results using different thresholds showed the search had located large protein complexes within the cytoplasm, but as expected, there was no clear cutoff in the distribution of cross-correlation coefficients that could distinguish between ribosomes and other large particles (<a href="#pone-0000749-g013">Figure 13A</a>). Moreover, because the peak-search algorithm excluded adjacent peaks closer together than a ribosome diameter, clustered complexes were not accurately parsed. Nevertheless because a threshold of the top 500 picks clearly excluded many large protein complexes and a threshold of 2000 clearly included many smaller complexes (<a href="#pone-0000749-g013">Figure 13B</a>), we conclude that there were very roughly about 1250 ribosomes per cell. In comparison, exponentially growing <em>Saccharomyces cerevisiae</em> cells have ~200,000 ribosomes each <a href="#pone.0000749-Warner1">[47]</a>. Since <em>S. cerevisiae</em> cells have up to 150 times more cytoplasm than <em>O. tauri</em> <a href="#pone.0000749-Zadrag1">[48]</a>, <a href="#pone.0000749-Johnston1">[49]</a>, the concentration of ribosomes in these cells is approximately the same. In <em>O. tauri</em>, the ribosomes appeared to be uniformly distributed within the cytoplasm (<a href="#pone-0000749-g013">Figure 13C</a>).</p>
<div class="figure" id="pone-0000749-g013"><div class="img"><a name="pone-0000749-g013" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000749.g013&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000749" data-uri="info:doi/10.1371/journal.pone.0000749.g013"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000749.g013&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000749.g013/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000749.g013/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000749.g013/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000749.g013/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000749.g013.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000749.g013/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000749.g013/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000749.g013.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 13.  <span>Ribosome-like complexes.</span></strong></p><a id="article1.body1.sec2.sec14.fig1.caption1.p1" name="article1.body1.sec2.sec14.fig1.caption1.p1"></a><p>(A) Three cells (shown in panels D–F of <a href="#pone-0000749-g003">Figure 3</a>) were searched for ribosome-like particles, and the resulting local-normalized cross-correlation coefficients are plotted from best to worst from left to right. There were fewer total positions ranked in cell “3D” because its cytoplasm was smaller. (B) 24-nm thick slice through a cell with the 500 (red) or 2000 (green) most ribosome-resembling objects in the cytoplasm circled, showing that these are reasonable under- and over-estimates of the total number. (C) 3-D positions of the 1250 most ribosome-resembling objects in the cytoplasm (light purple spheres), showing their close and even distribution. Scale bar 250 nm.</p>
<span>doi:10.1371/journal.pone.0000749.g013</span></div>

<h4>Plasma Membrane Extensions and Surface Proteins</h4>
<a id="article1.body1.sec2.sec15.p1" name="article1.body1.sec2.sec15.p1"></a><p>Three cells (6%) exhibited a bulge containing multiple 5-nm diameter filaments (<a href="#pone-0000749-g014">Figure 14A</a>) near sheets of ER. Two other much longer plasma membrane extensions were observed at opposite poles of an unusual cell (<a href="#pone-0000749-g002">Figure 2E–F</a>). One extension was at least 585 nm long (continued to the edge of the reconstructed area) and contained two internal filaments while the other was much shorter and contained a “Y” shaped inner vesicle protruding into the extension. Large macromolecular complexes were also seen decorating the outer surface of cells (<a href="#pone-0000749-g014">Figure 14B</a>). While these various specializations likely mediate contacts with other cells or substrates, their functions and mechanisms are of course at this point unclear.</p>
<div class="figure" id="pone-0000749-g014"><div class="img"><a name="pone-0000749-g014" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000749.g014&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000749" data-uri="info:doi/10.1371/journal.pone.0000749.g014"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000749.g014&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000749.g014/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000749.g014/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000749.g014/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000749.g014/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000749.g014.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000749.g014/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000749.g014/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000749.g014.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 14.  <span>External macromolecular complexes.</span></strong></p><a id="article1.body1.sec2.sec15.fig1.caption1.p1" name="article1.body1.sec2.sec15.fig1.caption1.p1"></a><p>(A) 12-nm thick slice showing multiple filaments (black arrowheads) emerging from a cellular protuberance. Black arrows point to ER. (B) 12-nm thick slice showing macromolecular complexes (white arrowheads) on the outer surface of the plasma membrane. Scale bar 100 nm.</p>
<span>doi:10.1371/journal.pone.0000749.g014</span></div>

<h4>Conclusion</h4>
<a id="article1.body1.sec2.sec16.p1" name="article1.body1.sec2.sec16.p1"></a><p>Despite application of all the best imaging technologies available, many fundamentally important aspects of eukaryotic ultrastructure remain elusive (including for instance, as a single example, the higher-order structure of chromatin). This study demonstrated that the emerging technique ECT can reveal the detailed ultrastructure of at least the smallest known eukaryotic cell, <em>O. tauri</em>, in an intact and near-native state. While ECT has already been applied to purified mitochondria and nuclei <a href="#pone.0000749-Nicastro1">[7]</a>, <a href="#pone.0000749-Beck1">[27]</a>, here they were seen in their cellular context along with structures that cannot be isolated (like the Golgi). Because <em>O. tauri</em> is becoming an increasingly well-developed model system as a simple plant and “minimal” eukaryotic cell, systematic high-resolution ultrastructural analysis by ECT as done here (but also of cells stalled in specific states and interesting mutants) will likely provide key insights into the organization of cells and the molecular mechanisms responsible. As examples, here we have seen that in G<sub>1</sub>, the nucleus and chloroplast sit opposite one another, and this may simply be because they are the largest bodies within the size-minimizing cell. While the mitochondrion seems to position itself randomly, the other major organelles appear to be organized by their relationship to the NE. The chloroplast exhibited its own division plane, and thus must possess its own segregation machinery for its internal features.</p>
<a id="article1.body1.sec2.sec16.p2" name="article1.body1.sec2.sec16.p2"></a><p>Nevertheless the resolution was limited in both space and time. As one benchmark example, while microtubules were easily resolved here, actin filaments were not, though that has been possible in other thinner specimens <a href="#pone.0000749-Medalia1">[8]</a>, <a href="#pone.0000749-Henderson1">[50]</a>–<a href="#pone.0000749-Scheffel1">[54]</a>. Thus it remained unclear, for instance, what the molecular mechanisms were that established the division plane or segregated the different copies of each organelle. While future technological advances in both instrumentation and specimen preparation can be expected to improve the situation soon <a href="#pone.0000749-Jensen1">[6]</a>, it is unclear now which of <em>O. tauri</em>'s eukaryotic secrets will be the most interesting! Our original hope of visualizing undisturbed chromatin and mitosis in molecular detail, for instance, has been replaced by intrigue over the possibility that <em>O. tauri</em> may have remarkable and unique adaptations, since the nuclear envelope was wide open throughout most of the cell cycle and neither condensed chromosomes nor a mitotic spindle were ever seen. While it is possible that these expected features exist so briefly that we simply missed them in our sampling of 52 frozen cells, it is also possible that <em>O. tauri</em> will reveal significantly simplified mitotic mechanisms.</p>

</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4>Cell growth</h4>
<a id="article1.body1.sec3.sec1.p1" name="article1.body1.sec3.sec1.p1"></a><p><em>Ostreococcus tauri</em> strain OTH95 (RCC 745 from the Roscoff Culture Collection) was grown in f/2 medium <a href="#pone.0000749-Guillard1">[55]</a> made with Sigma Sea Salt to 36% salinity. Cells were grown at a constant 20°C with gentle, 60-RPM agitation on a rotary shaker in a cycle of twelve hours of light and twelve hours of dark.</p>


<h4>Traditional Fixation</h4>
<a id="article1.body1.sec3.sec2.p1" name="article1.body1.sec3.sec2.p1"></a><p><em>O. tauri</em> cells were harvested during early and mid exponential growth phase, and concentrated by 5,000 RCF centrifugation. The cell pellet was then mixed with a 1% glutaraldehyde made up in filtered ocean water. The cells were re-centrifuged, washed twice with cacodylate buffer, and fixed in 1% OsO4 for 30 minutes. The cells were re-washed twice with cacodylate buffer, washed with 30% ethanol for 10 minutes, and then mixed with 30% ethanol and 4% uranyl acetate for 20 minutes. Finally cells were then successively dehydrated with a series of ethanol washes and embedded in Epon.</p>


<h4>High-pressure fixation and freeze substitution</h4>
<a id="article1.body1.sec3.sec3.p1" name="article1.body1.sec3.sec3.p1"></a><p>Cells were harvested by centrifugation as above. The cell pellet was high pressure-frozen in a BAL-TEC HPM 010 freezer (BAL-TEC, Liechtenstein). The samples were freeze substituted with 1% osmium tetroxide and 0.1% uranyl acetate in acetone using an EM AFS (Leica, Vienna). Samples were held at −90°C for 18 hours, warmed to −25°C at a rate of 5°C/hr, held at −25°C for 12 hours, and then warmed to room temperature at a rate of 5°C/hr. The samples were then rinsed in pure acetone, infiltrated and embedded in Epon-Araldite resin.</p>


<h4>Plunge Freezing</h4>
<a id="article1.body1.sec3.sec4.p1" name="article1.body1.sec3.sec4.p1"></a><p>Cells were harvested by centrifugation as above, then resuspended, and combined with 10 nm gold fiducial markers. This sample was applied to glow-discharged R 2/2 Quantifoil grids (SPI Supplies), excess media was removed by a single 1 s blot (−4 mm offset, 1 s drain time), and vitrified by plunging into liquid ethane in a Vitrobot (FEI) <a href="#pone.0000749-Iancu2">[56]</a>.</p>


<h4>Electron cryotomography</h4>
<a id="article1.body1.sec3.sec5.p1" name="article1.body1.sec3.sec5.p1"></a><p>Plunge-frozen grids were loaded into “flip-flop” tilt rotation holders <a href="#pone.0000749-Iancu1">[19]</a> and imaged in a 300 keV, FEG, G2 Polara transmission electron microscope (FEI) equipped with an energy filter (Gatan). Throughout loading and imaging, samples were cooled with liquid nitrogen <a href="#pone.0000749-Wright1">[57]</a>. Sequentially tilted, energy-filtered (slit width 20 eV) images of individual cells were collected automatically using either the UCSF tomography <a href="#pone.0000749-Zheng1">[58]</a> or Leginon software packages <a href="#pone.0000749-Suloway1">[59]</a> in 2° increments up to +/−~56–60° along either one (36 cells) or two (16 cells) axes <a href="#pone.0000749-Iancu1">[19]</a> at 18,000× magnification, resulting in a CCD pixel size of 1.2 nm. The cumulative dose for each tilt series was 165 e<sup>−</sup>/Å<sup>2</sup>.</p>


<h4>Image processing</h4>
<a id="article1.body1.sec3.sec6.p1" name="article1.body1.sec3.sec6.p1"></a><p>Images were aligned using the gold fiducials and three-dimensional reconstructions were calculated and visualized using the IMOD software package <a href="#pone.0000749-Mastronarde1">[60]</a>. Reconstructions were denoised using thirty cycles of non-linear anisotropic diffusion with parameter lambda = 0.3 <a href="#pone.0000749-Frangakis1">[61]</a> using Bsoft <a href="#pone.0000749-Heymann1">[62]</a> and the Peach distributed-computing system <a href="#pone.0000749-Leong1">[63]</a>. Denoised 3-D reconstructions were manually segmented using the Amira software package (Mercury Computer System, Inc.) on a Wacom Cintiq 21UX display. Volume and surface area estimates of the segmented volumes were also made using Amira. Cross-correlation searches were done using the MolMatch software <a href="#pone.0000749-Frangakis2">[64]</a> and an appropriately scaled and low pass-filtered cryo-EM reconstruction of a ribosome <a href="#pone.0000749-Spahn1">[65]</a> as template. Cytoplasmic peaks were ranked based upon cross-correlation coefficients and then displayed using Amira.</p>

</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0000749"></div><a name="pone.0000749.s001" id="pone.0000749.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000749.s001">Video S1. </a></strong></p><a id="article1.body1.sec4.supplementary-material1.caption1.p1" name="article1.body1.sec4.supplementary-material1.caption1.p1"></a><p class="preSiDOI">Electron cryotomographic reconstruction of an <em>Ostreococcus tauri</em> cell. The three-dimensional reconstruction of a single cell is shown slice-by-slice, from which the segmented model emerges in a second pass. After the various subcellular organelles are identified, the substructure of first the Golgi body and then the mitochondrion are shown in isolation, including the position and shape of the cristae junctions.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000749.s001</p><a id="article1.body1.sec4.supplementary-material1.caption1.p2" name="article1.body1.sec4.supplementary-material1.caption1.p2"></a><p class="postSiDOI">(5.06 MB MOV)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>We thank Hervé Moreau (Observatoire Océanologique, Laboratoire Arago) and Brian Palenik (Scripps Institution of Oceanography) for helping us obtain and work with <em>O. tauri</em> cultures and for reading the manuscript, Kent L. McDonald (U.C. Berkeley) for help high-pressure freezing, Wolfgang Baumeister's group (Max Planck Institute for Biochemistry) for the most current version of the MolMatch software, Andrea Manuell for discussions on the chloroplast, Christian Suloway (Caltech) for assistance with Leginon and Jane Ding (Caltech) for help running the cross-correlation search on the supercomputer.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: GJ GH LG. Performed the experiments: GH LG. Analyzed the data: GH LG. Contributed reagents/materials/analysis tools: GJ. Wrote the paper: GJ GH LG.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0000749-McDonald1" id="pone.0000749-McDonald1"></a>McDonald KL, Auer M (2006) High-pressure freezing, cellular tomography, and structural cell biology. Biotechniques  41: 137–143.  <ul class="find" data-citedArticleID="978506" data-doi="10.2144/000112226"><li><a href="http://dx.doi.org/10.2144/000112226" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=High-pressure+freezing%2C+cellular+tomography%2C+and+structural+cell+biology." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22High-pressure+freezing%2C+cellular+tomography%2C+and+structural+cell+biology.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0000749-Lucic1" id="pone.0000749-Lucic1"></a>Lucic V, Forster F, Baumeister W (2005) Structural studies by electron tomography: from cells to molecules. Annu Rev Biochem  74: 833–865.  <ul class="find" data-citedArticleID="978492" data-doi="10.1146/annurev.biochem.73.011303.074112"><li><a href="http://dx.doi.org/10.1146/annurev.biochem.73.011303.074112" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Structural+studies+by+electron+tomography%3A+from+cells+to+molecules." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Structural+studies+by+electron+tomography%3A+from+cells+to+molecules.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0000749-Hoog1" id="pone.0000749-Hoog1"></a>Hoog JL, Schwartz C, Noon AT, O'Toole ET, Mastronarde DN, et al.  (2007) Organization of interphase microtubules in fission yeast analyzed by electron tomography. Dev Cell  12: 349–361.  <ul class="find" data-citedArticleID="978476" data-doi="10.1016/j.devcel.2007.01.020"><li><a href="http://dx.doi.org/10.1016/j.devcel.2007.01.020" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Organization+of+interphase+microtubules+in+fission+yeast+analyzed+by+electron+tomography." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Organization+of+interphase+microtubules+in+fission+yeast+analyzed+by+electron+tomography.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0000749-Dubochet1" id="pone.0000749-Dubochet1"></a>Dubochet J, Adrian M, Chang JJ, Homo JC, Lepault J, et al.  (1988) Cryo-electron microscopy of vitrified specimens. Q Rev Biophys  21: 129–228.  <ul class="find" data-citedArticleID="978456" data-doi="10.1017/s0033583500004297"><li><a href="http://dx.doi.org/10.1017/s0033583500004297" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Cryo-electron+microscopy+of+vitrified+specimens." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Cryo-electron+microscopy+of+vitrified+specimens.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0000749-Grimm1" id="pone.0000749-Grimm1"></a>Grimm R, Barmann M, Hackl W, Typke D, Sackmann E, et al.  (1997) Energy filtered electron tomography of ice-embedded actin and vesicles. Biophys J  72: 482–489.  <ul class="find" data-citedArticleID="978466" data-doi="10.1016/s0006-3495(97)78689-2"><li><a href="http://dx.doi.org/10.1016/s0006-3495(97)78689-2" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Energy+filtered+electron+tomography+of+ice-embedded+actin+and+vesicles." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Energy+filtered+electron+tomography+of+ice-embedded+actin+and+vesicles.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0000749-Jensen1" id="pone.0000749-Jensen1"></a>Jensen GJ, Briegel A (2007) How electron cryotomography is opening a new window onto prokaryotic ultrastructure. Curr Opin Struct Biol  17: 260–267.  <ul class="find" data-citedArticleID="978482" data-doi="10.1016/j.sbi.2007.03.002"><li><a href="http://dx.doi.org/10.1016/j.sbi.2007.03.002" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=How+electron+cryotomography+is+opening+a+new+window+onto+prokaryotic+ultrastructure." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22How+electron+cryotomography+is+opening+a+new+window+onto+prokaryotic+ultrastructure.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0000749-Nicastro1" id="pone.0000749-Nicastro1"></a>Nicastro D, Frangakis AS, Typke D, Baumeister W (2000) Cryo-electron tomography of neurospora mitochondria. J Struct Biol  129: 48–56.  <ul class="find" data-citedArticleID="978514" data-doi="10.1006/jsbi.1999.4204"><li><a href="http://dx.doi.org/10.1006/jsbi.1999.4204" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Cryo-electron+tomography+of+neurospora+mitochondria." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Cryo-electron+tomography+of+neurospora+mitochondria.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0000749-Medalia1" id="pone.0000749-Medalia1"></a>Medalia O, Weber I, Frangakis AS, Nicastro D, Gerisch G, et al.  (2002) Macromolecular architecture in eukaryotic cells visualized by cryoelectron tomography. Science  298: 1209–1213.  <ul class="find" data-citedArticleID="978508" data-doi="10.1126/science.1076184"><li><a href="http://dx.doi.org/10.1126/science.1076184" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Macromolecular+architecture+in+eukaryotic+cells+visualized+by+cryoelectron+tomography." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Macromolecular+architecture+in+eukaryotic+cells+visualized+by+cryoelectron+tomography.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0000749-Subramaniam1" id="pone.0000749-Subramaniam1"></a>Subramaniam S, Milne JL (2004) Three-dimensional electron microscopy at molecular resolution. Annu Rev Biophys Biomol Struct  33: 141–155.  <ul class="find" data-citedArticleID="978542" data-doi="10.1146/annurev.biophys.33.110502.140339"><li><a href="http://dx.doi.org/10.1146/annurev.biophys.33.110502.140339" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Three-dimensional+electron+microscopy+at+molecular+resolution." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Three-dimensional+electron+microscopy+at+molecular+resolution.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0000749-AlAmoudi1" id="pone.0000749-AlAmoudi1"></a>Al-Amoudi A, Chang JJ, Leforestier A, McDowall A, Salamin LM, et al.  (2004) Cryo-electron microscopy of vitreous sections. Embo J  23: 3583–3588.  <ul class="find" data-citedArticleID="978436" data-doi="10.1038/sj.emboj.7600366"><li><a href="http://dx.doi.org/10.1038/sj.emboj.7600366" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Cryo-electron+microscopy+of+vitreous+sections." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Cryo-electron+microscopy+of+vitreous+sections.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0000749-Marko1" id="pone.0000749-Marko1"></a>Marko M, Hsieh C, Schalek R, Frank J, Mannella C (2007) Focused-ion-beam thinning of frozen-hydrated biological specimens for cryo-electron microscopy. Nat Methods  4: 215–217.  <ul class="find" data-citedArticleID="978502" data-doi="10.1038/nmeth1014"><li><a href="http://dx.doi.org/10.1038/nmeth1014" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Focused-ion-beam+thinning+of+frozen-hydrated+biological+specimens+for+cryo-electron+microscopy." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Focused-ion-beam+thinning+of+frozen-hydrated+biological+specimens+for+cryo-electron+microscopy.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0000749-ChrtiennotDinet1" id="pone.0000749-ChrtiennotDinet1"></a>Chrétiennot-Dinet MJ, Courties C, Vaquer A, Neveux J, Claustre H, et al.  (1995) A new marine picoeucaryote: <em>Ostreococcus tauri</em> gen. et sp. nov. (Chlorophyta, Prasinophyceae). Phycologia  34: 285–292.  <ul class="find" data-citedArticleID="978444" data-doi="10.2216/i0031-8884-34-4-285.1"><li><a href="http://dx.doi.org/10.2216/i0031-8884-34-4-285.1" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+new+marine+picoeucaryote%3A+Ostreococcus+tauri+gen.+et+sp.+nov.+%28Chlorophyta%2C+Prasinophyceae%29." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+new+marine+picoeucaryote%3A+Ostreococcus+tauri+gen.+et+sp.+nov.+%28Chlorophyta%2C+Prasinophyceae%29.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0000749-Derelle1" id="pone.0000749-Derelle1"></a>Derelle E, Ferraz C, Rombauts S, Rouze P, Worden AZ, et al.  (2006) Genome analysis of the smallest free-living eukaryote <em>Ostreococcus tauri</em> unveils many unique features. Proc Natl Acad Sci U S A  103: 11647–11652.  <ul class="find" data-citedArticleID="978452" data-doi="10.1073/pnas.0604795103"><li><a href="http://dx.doi.org/10.1073/pnas.0604795103" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genome+analysis+of+the+smallest+free-living+eukaryote+Ostreococcus+tauri+unveils+many+unique+features." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genome+analysis+of+the+smallest+free-living+eukaryote+Ostreococcus+tauri+unveils+many+unique+features.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0000749-Courties1" id="pone.0000749-Courties1"></a>Courties C, Perasso R, Chrétiennot-Dinet MJ, Gouy M, Guillour L, et al.  (1998) Phylogenetic Analysis and Genome Size of <em>Ostreococcus tauri</em> (Chlorophyta, Prasinophyceae). J Phycology  34: 844–849.  <ul class="find" data-citedArticleID="978448" data-doi="10.1046/j.1529-8817.1998.340844.x"><li><a href="http://dx.doi.org/10.1046/j.1529-8817.1998.340844.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Phylogenetic+Analysis+and+Genome+Size+of+Ostreococcus+tauri+%28Chlorophyta%2C+Prasinophyceae%29." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Phylogenetic+Analysis+and+Genome+Size+of+Ostreococcus+tauri+%28Chlorophyta%2C+Prasinophyceae%29.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0000749-Yoon1" id="pone.0000749-Yoon1"></a>Yoon HS, Hackett JD, Ciniglia C, Pinto G, Bhattacharya D (2004) A molecular timeline for the origin of photosynthetic eukaryotes. Mol Biol Evol  21: 809–818.  <ul class="find" data-citedArticleID="978556" data-doi="10.1093/molbev/msh075"><li><a href="http://dx.doi.org/10.1093/molbev/msh075" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+molecular+timeline+for+the+origin+of+photosynthetic+eukaryotes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+molecular+timeline+for+the+origin+of+photosynthetic+eukaryotes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0000749-Robbens1" id="pone.0000749-Robbens1"></a>Robbens S, Khadaroo B, Camasses A, Derelle E, Ferraz C, et al.  (2005) Genome-wide analysis of core cell cycle genes in the unicellular green alga <em>Ostreococcus tauri</em>. Mol Biol Evol  22: 589–597.  <ul class="find" data-citedArticleID="978524" data-doi="10.1093/molbev/msi044"><li><a href="http://dx.doi.org/10.1093/molbev/msi044" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genome-wide+analysis+of+core+cell+cycle+genes+in+the+unicellular+green+alga+Ostreococcus+tauri." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genome-wide+analysis+of+core+cell+cycle+genes+in+the+unicellular+green+alga+Ostreococcus+tauri.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0000749-Farinas1" id="pone.0000749-Farinas1"></a>Farinas B, Mary C, de O Manes CL, Bhaud Y, Peaucellier G, et al.  (2006) Natural Synchronisation for the study of cell division in the green unicellular alga <em>Ostreococcus tauri</em>. Plant Mol Biol  60: 277–292.  <ul class="find" data-citedArticleID="978458" data-doi="10.1007/s11103-005-4066-1"><li><a href="http://dx.doi.org/10.1007/s11103-005-4066-1" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Natural+Synchronisation+for+the+study+of+cell+division+in+the+green+unicellular+alga+Ostreococcus+tauri." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Natural+Synchronisation+for+the+study+of+cell+division+in+the+green+unicellular+alga+Ostreococcus+tauri.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0000749-Courties2" id="pone.0000749-Courties2"></a>Courties C, Vaquer A, Troussellier M, Lautier J, Chrétiennot-Dinet M-J, et al.  (1994) Smallest eukaryotic organism. Nature  370: 255.  <ul class="find" data-citedArticleID="978450" data-doi="10.1038/370255a0"><li><a href="http://dx.doi.org/10.1038/370255a0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Smallest+eukaryotic+organism." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Smallest+eukaryotic+organism.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0000749-Iancu1" id="pone.0000749-Iancu1"></a>Iancu CV, Wright ER, Benjamin J, Tivol WF, Dias DP, et al.  (2005) A “flip-flop” rotation stage for routine dual-axis electron cryotomography. J Struct Biol  151: 288–297.  <ul class="find" data-citedArticleID="978478" data-doi="10.1016/j.jsb.2005.07.004"><li><a href="http://dx.doi.org/10.1016/j.jsb.2005.07.004" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+%26%23x201c%3Bflip-flop%26%23x201d%3B+rotation+stage+for+routine+dual-axis+electron+cryotomography." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+%26%23x201c%3Bflip-flop%26%23x201d%3B+rotation+stage+for+routine+dual-axis+electron+cryotomography.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0000749-Comolli1" id="pone.0000749-Comolli1"></a>Comolli LR, Downing KH (2005) Dose tolerance at helium and nitrogen temperatures for whole cell electron tomography. J Struct Biol  152: 149–156.  <ul class="find" data-citedArticleID="978446" data-doi="10.1016/j.jsb.2005.08.004"><li><a href="http://dx.doi.org/10.1016/j.jsb.2005.08.004" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Dose+tolerance+at+helium+and+nitrogen+temperatures+for+whole+cell+electron+tomography." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Dose+tolerance+at+helium+and+nitrogen+temperatures+for+whole+cell+electron+tomography.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0000749-Ral1" id="pone.0000749-Ral1"></a>Ral JP, Derelle E, Ferraz C, Wattebled F, Farinas B, et al.  (2004) Starch division and partitioning. A mechanism for granule propagation and maintenance in the picophytoplanktonic green alga <em>Ostreococcus tauri</em>. Plant Physiol  136: 3333–3340.  <ul class="find" data-citedArticleID="978522" data-doi="10.1104/pp.104.044131"><li><a href="http://dx.doi.org/10.1104/pp.104.044131" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Starch+division+and+partitioning.+A+mechanism+for+granule+propagation+and+maintenance+in+the+picophytoplanktonic+green+alga+Ostreococcus+tauri." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Starch+division+and+partitioning.+A+mechanism+for+granule+propagation+and+maintenance+in+the+picophytoplanktonic+green+alga+Ostreococcus+tauri.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0000749-vandeMeene1" id="pone.0000749-vandeMeene1"></a>van de Meene AM, Hohmann-Marriott MF, Vermaas WF, Roberson RW (2006) The three-dimensional structure of the cyanobacterium <em>Synechocystis</em> sp. PCC 6803. Arch Microbiol  184: 259–270.  <ul class="find" data-citedArticleID="978562" data-doi="10.1007/s00203-005-0027-y"><li><a href="http://dx.doi.org/10.1007/s00203-005-0027-y" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+three-dimensional+structure+of+the+cyanobacterium+Synechocystis+sp.+PCC+6803." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+three-dimensional+structure+of+the+cyanobacterium+Synechocystis+sp.+PCC+6803.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0000749-Nevo1" id="pone.0000749-Nevo1"></a>Nevo R, Charuvi D, Shimoni E, Schwarz R, Kaplan A, et al.  (2007) Thylakoid membrane perforations and connectivity enable intracellular traffic in cyanobacteria. EMBO J  26: 1467–1473.  <ul class="find" data-citedArticleID="978512" data-doi="10.1038/sj.emboj.7601594"><li><a href="http://dx.doi.org/10.1038/sj.emboj.7601594" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Thylakoid+membrane+perforations+and+connectivity+enable+intracellular+traffic+in+cyanobacteria." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Thylakoid+membrane+perforations+and+connectivity+enable+intracellular+traffic+in+cyanobacteria.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0000749-Vothknecht1" id="pone.0000749-Vothknecht1"></a>Vothknecht UC, Westhoff P (2001) Biogenesis and origin of thylakoid membranes. Biochim Biophys Acta  1541: 91–101.  <ul class="find" data-citedArticleID="978548" data-doi="10.1016/s0167-4889(01)00153-7"><li><a href="http://dx.doi.org/10.1016/s0167-4889(01)00153-7" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Biogenesis+and+origin+of+thylakoid+membranes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Biogenesis+and+origin+of+thylakoid+membranes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0000749-Shimoni1" id="pone.0000749-Shimoni1"></a>Shimoni E, Rav-Hon O, Ohad I, Brumfeld V, Reich Z (2005) Three-dimensional organization of higher-plant chloroplast thylakoid membranes revealed by electron tomography. Plant Cell  17: 2580–2586.  <ul class="find" data-citedArticleID="978530" data-doi="10.1105/tpc.105.035030"><li><a href="http://dx.doi.org/10.1105/tpc.105.035030" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Three-dimensional+organization+of+higher-plant+chloroplast+thylakoid+membranes+revealed+by+electron+tomography." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Three-dimensional+organization+of+higher-plant+chloroplast+thylakoid+membranes+revealed+by+electron+tomography.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0000749-Drummond1" id="pone.0000749-Drummond1"></a>Drummond SP, Rutherford SA, Sanderson HS, Allen TD (2006) High resolution analysis of mammalian nuclear structure throughout the cell cycle: implications for nuclear pore complex assembly during interphase and mitosis. Cana J Physiol Pharmacol  84: 423–430.  <ul class="find" data-citedArticleID="978454" data-doi="10.1139/y05-148"><li><a href="http://dx.doi.org/10.1139/y05-148" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=High+resolution+analysis+of+mammalian+nuclear+structure+throughout+the+cell+cycle%3A+implications+for+nuclear+pore+complex+assembly+during+interphase+and+mitosis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22High+resolution+analysis+of+mammalian+nuclear+structure+throughout+the+cell+cycle%3A+implications+for+nuclear+pore+complex+assembly+during+interphase+and+mitosis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0000749-Beck1" id="pone.0000749-Beck1"></a>Beck M, Forster F, Ecke M, Plitzko JM, Melchior F, et al.  (2004) Nuclear pore complex structure and dynamics revealed by cryoelectron tomography. Science  306: 1387–1390.  <ul class="find" data-citedArticleID="978438" data-doi="10.1126/science.1104808"><li><a href="http://dx.doi.org/10.1126/science.1104808" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Nuclear+pore+complex+structure+and+dynamics+revealed+by+cryoelectron+tomography." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Nuclear+pore+complex+structure+and+dynamics+revealed+by+cryoelectron+tomography.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0000749-Akey1" id="pone.0000749-Akey1"></a>Akey CW, Radermacher M (1993) Architecture of the <em>Xenopus</em> nuclear pore complex revealed by three-dimensional cryo-electron microscopy. J Cell Biol  122: 1–19.  <ul class="find" data-citedArticleID="978434" data-doi="10.1083/jcb.122.1.1"><li><a href="http://dx.doi.org/10.1083/jcb.122.1.1" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Architecture+of+the+Xenopus+nuclear+pore+complex+revealed+by+three-dimensional+cryo-electron+microscopy." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Architecture+of+the+Xenopus+nuclear+pore+complex+revealed+by+three-dimensional+cryo-electron+microscopy.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0000749-Stoffler1" id="pone.0000749-Stoffler1"></a>Stoffler D, Feja B, Fahrenkrog B, Walz J, Typke D, et al.  (2003) Cryo-electron tomography provides novel insights into nuclear pore architecture: implications for nucleocytoplasmic transport. J Mol Biol  328: 119–130.  <ul class="find" data-citedArticleID="978536" data-doi="10.1016/s0022-2836(03)00266-3"><li><a href="http://dx.doi.org/10.1016/s0022-2836(03)00266-3" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Cryo-electron+tomography+provides+novel+insights+into+nuclear+pore+architecture%3A+implications+for+nucleocytoplasmic+transport." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Cryo-electron+tomography+provides+novel+insights+into+nuclear+pore+architecture%3A+implications+for+nucleocytoplasmic+transport.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0000749-Yang1" id="pone.0000749-Yang1"></a>Yang Q, Rout MP, Akey CW (1998) Three-dimensional architecture of the isolated yeast nuclear pore complex: functional and evolutionary implications. Mol Cell  1: 223–234.  <ul class="find" data-citedArticleID="978554" data-doi="10.1016/s1097-2765(00)80023-4"><li><a href="http://dx.doi.org/10.1016/s1097-2765(00)80023-4" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Three-dimensional+architecture+of+the+isolated+yeast+nuclear+pore+complex%3A+functional+and+evolutionary+implications." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Three-dimensional+architecture+of+the+isolated+yeast+nuclear+pore+complex%3A+functional+and+evolutionary+implications.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0000749-Perkins1" id="pone.0000749-Perkins1"></a>Perkins G, Renken C, Martone ME, Young SJ, Ellisman M, et al.  (1997) Electron tomography of neuronal mitochondria: three-dimensional structure and organization of cristae and membrane contacts. J Struct Biol  119: 260–272.  <ul class="find" data-citedArticleID="978518" data-doi="10.1006/jsbi.1997.3885"><li><a href="http://dx.doi.org/10.1006/jsbi.1997.3885" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Electron+tomography+of+neuronal+mitochondria%3A+three-dimensional+structure+and+organization+of+cristae+and+membrane+contacts." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Electron+tomography+of+neuronal+mitochondria%3A+three-dimensional+structure+and+organization+of+cristae+and+membrane+contacts.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0000749-Mannella1" id="pone.0000749-Mannella1"></a>Mannella CA, Marko M, Penczek P, Barnard D, Frank J (1994) The internal compartmentation of rat-liver mitochondria: tomographic study using the high-voltage transmission electron microscope. Microsc Res Tech  27: 278–283.  <ul class="find" data-citedArticleID="978494" data-doi="10.1002/jemt.1070270403"><li><a href="http://dx.doi.org/10.1002/jemt.1070270403" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+internal+compartmentation+of+rat-liver+mitochondria%3A+tomographic+study+using+the+high-voltage+transmission+electron+microscope." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+internal+compartmentation+of+rat-liver+mitochondria%3A+tomographic+study+using+the+high-voltage+transmission+electron+microscope.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0000749-Mannella2" id="pone.0000749-Mannella2"></a>Mannella CA, Pfeiffer DR, Bradshaw PC, Moraru II, Slepchenko B, et al.  (2001) Topology of the mitochondrial inner membrane: dynamics and bioenergetic implications. IUBMB Life  52: 93–100.  <ul class="find" data-citedArticleID="978496" data-doi="10.1080/15216540152845885"><li><a href="http://dx.doi.org/10.1080/15216540152845885" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Topology+of+the+mitochondrial+inner+membrane%3A+dynamics+and+bioenergetic+implications." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Topology+of+the+mitochondrial+inner+membrane%3A+dynamics+and+bioenergetic+implications.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0000749-Mannella3" id="pone.0000749-Mannella3"></a>Mannella CA (2006) Structure and dynamics of the mitochondrial inner membrane cristae. Biochim Biophys Acta  1763: 542–548.  <ul class="find" data-citedArticleID="978498" data-doi="10.1016/j.bbamcr.2006.04.006"><li><a href="http://dx.doi.org/10.1016/j.bbamcr.2006.04.006" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Structure+and+dynamics+of+the+mitochondrial+inner+membrane+cristae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Structure+and+dynamics+of+the+mitochondrial+inner+membrane+cristae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0000749-Mannella4" id="pone.0000749-Mannella4"></a>Mannella CA (2006) The relevance of mitochondrial membrane topology to mitochondrial function. Biochim Biophys Acta  1762: 140–147.  <ul class="find" data-citedArticleID="978500" data-doi="10.1016/j.bbadis.2005.07.001"><li><a href="http://dx.doi.org/10.1016/j.bbadis.2005.07.001" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+relevance+of+mitochondrial+membrane+topology+to+mitochondrial+function." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+relevance+of+mitochondrial+membrane+topology+to+mitochondrial+function.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">36.
              </span><a name="pone.0000749-Staehelin1" id="pone.0000749-Staehelin1"></a>Staehelin LA (1997) The plant ER: a dynamic organelle composed of a large number of discrete functional domains. Plant J  11: 1151–1165.  <ul class="find" data-citedArticleID="978534" data-doi="10.1046/j.1365-313x.1997.11061151.x"><li><a href="http://dx.doi.org/10.1046/j.1365-313x.1997.11061151.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+plant+ER%3A+a+dynamic+organelle+composed+of+a+large+number+of+discrete+functional+domains." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+plant+ER%3A+a+dynamic+organelle+composed+of+a+large+number+of+discrete+functional+domains.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">37.
              </span><a name="pone.0000749-Shibata1" id="pone.0000749-Shibata1"></a>Shibata Y, Voeltz GK, Rapoport TA (2006) Rough sheets and smooth tubules. Cell  126: 435–439.  <ul class="find" data-citedArticleID="978528" data-doi="10.1016/j.cell.2006.07.019"><li><a href="http://dx.doi.org/10.1016/j.cell.2006.07.019" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Rough+sheets+and+smooth+tubules." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Rough+sheets+and+smooth+tubules.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">38.
              </span><a name="pone.0000749-Voeltz1" id="pone.0000749-Voeltz1"></a>Voeltz GK, Prinz WA, Shibata Y, Rist JM, Rapoport TA (2006) A class of membrane proteins shaping the tubular endoplasmic reticulum. Cell  124: 573–586.  <ul class="find" data-citedArticleID="978546" data-doi="10.1016/j.cell.2005.11.047"><li><a href="http://dx.doi.org/10.1016/j.cell.2005.11.047" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+class+of+membrane+proteins+shaping+the+tubular+endoplasmic+reticulum." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+class+of+membrane+proteins+shaping+the+tubular+endoplasmic+reticulum.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">39.
              </span><a name="pone.0000749-Gaietta1" id="pone.0000749-Gaietta1"></a>Gaietta GM, Giepmans BN, Deerinck TJ, Smith WB, Ngan L, et al.  (2006) Golgi twins in late mitosis revealed by genetically encoded tags for live cell imaging and correlated electron microscopy. Proc Natl Acad Sci U S A  103: 17777–17782.  <ul class="find" data-citedArticleID="978464" data-doi="10.1073/pnas.0608509103"><li><a href="http://dx.doi.org/10.1073/pnas.0608509103" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Golgi+twins+in+late+mitosis+revealed+by+genetically+encoded+tags+for+live+cell+imaging+and+correlated+electron+microscopy." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Golgi+twins+in+late+mitosis+revealed+by+genetically+encoded+tags+for+live+cell+imaging+and+correlated+electron+microscopy.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">40.
              </span><a name="pone.0000749-Storrie1" id="pone.0000749-Storrie1"></a>Storrie B, Nilsson T (2002) The Golgi apparatus: balancing new with old. Traffic  3: 521–529.  <ul class="find" data-citedArticleID="978540" data-doi="10.1034/j.1600-0854.2002.30802.x"><li><a href="http://dx.doi.org/10.1034/j.1600-0854.2002.30802.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+Golgi+apparatus%3A+balancing+new+with+old." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+Golgi+apparatus%3A+balancing+new+with+old.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">41.
              </span><a name="pone.0000749-Caviston1" id="pone.0000749-Caviston1"></a>Caviston JP, Holzbaur EL (2006) Microtubule motors at the intersection of trafficking and transport. Trends Cell Biol  16: 530–537.  <ul class="find" data-citedArticleID="978442" data-doi="10.1016/j.tcb.2006.08.002"><li><a href="http://dx.doi.org/10.1016/j.tcb.2006.08.002" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Microtubule+motors+at+the+intersection+of+trafficking+and+transport." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Microtubule+motors+at+the+intersection+of+trafficking+and+transport.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">42.
              </span><a name="pone.0000749-Palenik1" id="pone.0000749-Palenik1"></a>Palenik B, Grimwood J, Aerts A, Rouze P, Salamov A, et al.  (2007) The tiny eukaryote <em>Ostreococcus</em> provides genomic insights into the paradox of plankton speciation. Proc Natl Acad Sci U S A  104: 7705–7710.  <ul class="find" data-citedArticleID="978516" data-doi="10.1073/pnas.0611046104"><li><a href="http://dx.doi.org/10.1073/pnas.0611046104" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+tiny+eukaryote+Ostreococcus+provides+genomic+insights+into+the+paradox+of+plankton+speciation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+tiny+eukaryote+Ostreococcus+provides+genomic+insights+into+the+paradox+of+plankton+speciation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">43.
              </span><a name="pone.0000749-Purdue1" id="pone.0000749-Purdue1"></a>Purdue PE, Lazarow PB (2001) Peroxisome biogenesis. Annu Rev Cell Dev Biol  17: 701–752.  <ul class="find" data-citedArticleID="978520" data-doi="10.1146/annurev.cellbio.17.1.701"><li><a href="http://dx.doi.org/10.1146/annurev.cellbio.17.1.701" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Peroxisome+biogenesis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Peroxisome+biogenesis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">44.
              </span><a name="pone.0000749-Mullen1" id="pone.0000749-Mullen1"></a>Mullen RT, Trelease RN (2006) The ER-peroxisome connection in plants: development of the “ER semi-autonomous peroxisome maturation and replication” model for plant peroxisome biogenesis. Biochim Biophys Acta  1763: 1655–1668.  <ul class="find" data-citedArticleID="978510" data-doi="10.1016/j.bbamcr.2006.09.011"><li><a href="http://dx.doi.org/10.1016/j.bbamcr.2006.09.011" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+ER-peroxisome+connection+in+plants%3A+development+of+the+%26%23x201c%3BER+semi-autonomous+peroxisome+maturation+and+replication%26%23x201d%3B+model+for+plant+peroxisome+biogenesis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+ER-peroxisome+connection+in+plants%3A+development+of+the+%26%23x201c%3BER+semi-autonomous+peroxisome+maturation+and+replication%26%23x201d%3B+model+for+plant+peroxisome+biogenesis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">45.
              </span><a name="pone.0000749-Hoepfner1" id="pone.0000749-Hoepfner1"></a>Hoepfner D, Schildknegt D, Braakman I, Philippsen P, Tabak HF (2005) Contribution of the endoplasmic reticulum to peroxisome formation. Cell  122: 85–95.  <ul class="find" data-citedArticleID="978474" data-doi="10.1016/j.cell.2005.04.025"><li><a href="http://dx.doi.org/10.1016/j.cell.2005.04.025" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Contribution+of+the+endoplasmic+reticulum+to+peroxisome+formation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Contribution+of+the+endoplasmic+reticulum+to+peroxisome+formation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">46.
              </span><a name="pone.0000749-Stoppin1" id="pone.0000749-Stoppin1"></a>Stoppin V, Vantard M, Schmit AC, Lambert AM (1994) Isolated plant nuclei nucleate microtubule assembly: the nuclear surface in higher plants has centrosome-like activity. Plant Cell  6: 1099–1106.  <ul class="find" data-citedArticleID="978538" data-doi="10.2307/3869888"><li><a href="http://dx.doi.org/10.2307/3869888" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Isolated+plant+nuclei+nucleate+microtubule+assembly%3A+the+nuclear+surface+in+higher+plants+has+centrosome-like+activity." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Isolated+plant+nuclei+nucleate+microtubule+assembly%3A+the+nuclear+surface+in+higher+plants+has+centrosome-like+activity.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">47.
              </span><a name="pone.0000749-Warner1" id="pone.0000749-Warner1"></a>Warner JR (1999) The economics of ribosome biosynthesis in yeast. Trends Biochem Sci  24: 437–440.  <ul class="find" data-citedArticleID="978550" data-doi="10.1016/s0968-0004(99)01460-7"><li><a href="http://dx.doi.org/10.1016/s0968-0004(99)01460-7" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+economics+of+ribosome+biosynthesis+in+yeast." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+economics+of+ribosome+biosynthesis+in+yeast.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">48.
              </span><a name="pone.0000749-Zadrag1" id="pone.0000749-Zadrag1"></a>Zadrag R, Kwolek-Mirek M, Bartosz G, Bilinski T (2006) Relationship between the replicative age and cell volume in <em>Saccharomyces cerevisiae</em>. Acta Biochim Pol  53: 747–751.  <ul class="find" data-citedArticleID="978558" data-doi="10.1007/s10522-008-9192-0"><li><a href="http://dx.doi.org/10.1007/s10522-008-9192-0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Relationship+between+the+replicative+age+and+cell+volume+in+Saccharomyces+cerevisiae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Relationship+between+the+replicative+age+and+cell+volume+in+Saccharomyces+cerevisiae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">49.
              </span><a name="pone.0000749-Johnston1" id="pone.0000749-Johnston1"></a>Johnston GC, Ehrhardt A, Lorincz A, Carter BL (1979) Regulation of cell size in the yeast <em>Saccharomyces cerevisiae</em>. J Bacteriol  137: 1–5.  <ul class="find" data-citedArticleID="978484"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Regulation+of+cell+size+in+the+yeast+Saccharomyces+cerevisiae.&amp;auth=&amp;atitle=Regulation+of+cell+size+in+the+yeast+Saccharomyces+cerevisiae." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Regulation+of+cell+size+in+the+yeast+Saccharomyces+cerevisiae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Regulation+of+cell+size+in+the+yeast+Saccharomyces+cerevisiae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">50.
              </span><a name="pone.0000749-Henderson1" id="pone.0000749-Henderson1"></a>Henderson GP, Jensen GJ (2006) Three-dimensional structure of <em>Mycoplasma pneumoniae</em>'s attachment organelle and a model for its role in gliding motility. Mol Microbiol  60: 376–385.  <ul class="find" data-citedArticleID="978470" data-doi="10.1111/j.1365-2958.2006.05113.x"><li><a href="http://dx.doi.org/10.1111/j.1365-2958.2006.05113.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Three-dimensional+structure+of+Mycoplasma+pneumoniae%27s+attachment+organelle+and+a+model+for+its+role+in+gliding+motility." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Three-dimensional+structure+of+Mycoplasma+pneumoniae%27s+attachment+organelle+and+a+model+for+its+role+in+gliding+motility.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">51.
              </span><a name="pone.0000749-Briegel1" id="pone.0000749-Briegel1"></a>Briegel A, Dias DP, Li Z, Jensen RB, Frangakis AS, et al.  (2006) Multiple large filament bundles observed in <em>Caulobacter crescentus</em> by electron cryotomography. Mol Microbiol  62: 5–14.  <ul class="find" data-citedArticleID="978440" data-doi="10.1111/j.1365-2958.2006.05355.x"><li><a href="http://dx.doi.org/10.1111/j.1365-2958.2006.05355.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Multiple+large+filament+bundles+observed+in+Caulobacter+crescentus+by+electron+cryotomography." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Multiple+large+filament+bundles+observed+in+Caulobacter+crescentus+by+electron+cryotomography.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">52.
              </span><a name="pone.0000749-Komeili1" id="pone.0000749-Komeili1"></a>Komeili A, Li Z, Newman DK, Jensen GJ (2006) Magnetosomes are cell membrane invaginations organized by the actin-like protein MamK. Science  311: 242–245.  <ul class="find" data-citedArticleID="978486" data-doi="10.1126/science.1123231"><li><a href="http://dx.doi.org/10.1126/science.1123231" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Magnetosomes+are+cell+membrane+invaginations+organized+by+the+actin-like+protein+MamK." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Magnetosomes+are+cell+membrane+invaginations+organized+by+the+actin-like+protein+MamK.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">53.
              </span><a name="pone.0000749-Kurner1" id="pone.0000749-Kurner1"></a>Kurner J, Frangakis AS, Baumeister W (2005) Cryo-electron tomography reveals the cytoskeletal structure of <em>Spiroplasma melliferum</em>. Science  307: 436–438.  <ul class="find" data-citedArticleID="978488" data-doi="10.1126/science.1104031"><li><a href="http://dx.doi.org/10.1126/science.1104031" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Cryo-electron+tomography+reveals+the+cytoskeletal+structure+of+Spiroplasma+melliferum." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Cryo-electron+tomography+reveals+the+cytoskeletal+structure+of+Spiroplasma+melliferum.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">54.
              </span><a name="pone.0000749-Scheffel1" id="pone.0000749-Scheffel1"></a>Scheffel A, Gruska M, Faivre D, Linaroudis A, Plitzko JM, et al.  (2006) An acidic protein aligns magnetosomes along a filamentous structure in magnetotactic bacteria. Nature  440: 110–114.  <ul class="find" data-citedArticleID="978526" data-doi="10.1038/nature04382"><li><a href="http://dx.doi.org/10.1038/nature04382" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=An+acidic+protein+aligns+magnetosomes+along+a+filamentous+structure+in+magnetotactic+bacteria." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22An+acidic+protein+aligns+magnetosomes+along+a+filamentous+structure+in+magnetotactic+bacteria.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">55.
              </span><a name="pone.0000749-Guillard1" id="pone.0000749-Guillard1"></a>Guillard RR (1975)  Culture of phytoplankton for feeding marine invertebrates. In: Smith W, Chanley M, editors. Culture of marine invertebrate animals. New York, New York: Plenum Publishing Corporation. pp. 29–53.  <ul class="find-nolinks"></ul></li><li><span class="label">56.
              </span><a name="pone.0000749-Iancu2" id="pone.0000749-Iancu2"></a>Iancu CV, Tivol WF, Schooler JB, Dias DP, Henderson GP, et al.  (2006) Electron cryotomography sample preparation using the Vitrobot. Nat Protoc  1: 2813–2819.  <ul class="find" data-citedArticleID="978480" data-doi="10.1038/nprot.2006.432"><li><a href="http://dx.doi.org/10.1038/nprot.2006.432" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Electron+cryotomography+sample+preparation+using+the+Vitrobot." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Electron+cryotomography+sample+preparation+using+the+Vitrobot.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">57.
              </span><a name="pone.0000749-Wright1" id="pone.0000749-Wright1"></a>Wright ER, Iancu CV, Tivol WF, Jensen GJ (2006) Observations on the behavior of vitreous ice at approximately 82 and approximately 12 K. J Struct Biol  153: 241–252.  <ul class="find" data-citedArticleID="978552" data-doi="10.1016/j.jsb.2005.12.003"><li><a href="http://dx.doi.org/10.1016/j.jsb.2005.12.003" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Observations+on+the+behavior+of+vitreous+ice+at+approximately+82+and+approximately+12+K." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Observations+on+the+behavior+of+vitreous+ice+at+approximately+82+and+approximately+12+K.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">58.
              </span><a name="pone.0000749-Zheng1" id="pone.0000749-Zheng1"></a>Zheng QS, Braunfeld MB, Sedat JW, Agard DA (2004) An improved strategy for automated electron microscopic tomography. J Struct Biol  147: 91–101.  <ul class="find" data-citedArticleID="978560" data-doi="10.1016/j.jsb.2004.02.005"><li><a href="http://dx.doi.org/10.1016/j.jsb.2004.02.005" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=An+improved+strategy+for+automated+electron+microscopic+tomography." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22An+improved+strategy+for+automated+electron+microscopic+tomography.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">59.
              </span><a name="pone.0000749-Suloway1" id="pone.0000749-Suloway1"></a>Suloway C, Pulokas J, Fellmann D, Cheng A, Guerra F, et al.  (2005) Automated molecular microscopy: the new Leginon system. J Struct Biol  151: 41–60.  <ul class="find" data-citedArticleID="978544" data-doi="10.1016/j.jsb.2005.03.010"><li><a href="http://dx.doi.org/10.1016/j.jsb.2005.03.010" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Automated+molecular+microscopy%3A+the+new+Leginon+system." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Automated+molecular+microscopy%3A+the+new+Leginon+system.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">60.
              </span><a name="pone.0000749-Mastronarde1" id="pone.0000749-Mastronarde1"></a>Mastronarde DN (1997) Dual-axis tomography: an approach with alignment methods that preserve resolution. J Struct Biol  120: 343–352.  <ul class="find" data-citedArticleID="978504" data-doi="10.1006/jsbi.1997.3919"><li><a href="http://dx.doi.org/10.1006/jsbi.1997.3919" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Dual-axis+tomography%3A+an+approach+with+alignment+methods+that+preserve+resolution." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Dual-axis+tomography%3A+an+approach+with+alignment+methods+that+preserve+resolution.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">61.
              </span><a name="pone.0000749-Frangakis1" id="pone.0000749-Frangakis1"></a>Frangakis AS, Hegerl R (2001) Noise reduction in electron tomographic reconstructions using nonlinear anisotropic diffusion. J Struct Biol  135: 239–250.  <ul class="find" data-citedArticleID="978460" data-doi="10.1006/jsbi.2001.4406"><li><a href="http://dx.doi.org/10.1006/jsbi.2001.4406" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Noise+reduction+in+electron+tomographic+reconstructions+using+nonlinear+anisotropic+diffusion." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Noise+reduction+in+electron+tomographic+reconstructions+using+nonlinear+anisotropic+diffusion.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">62.
              </span><a name="pone.0000749-Heymann1" id="pone.0000749-Heymann1"></a>Heymann JB, Belnap DM (2006) Bsoft: Image processing and molecular modeling for electron microscopy. J Struct Biol  157: 3–18.  <ul class="find" data-citedArticleID="978472" data-doi="10.1016/j.jsb.2006.06.006"><li><a href="http://dx.doi.org/10.1016/j.jsb.2006.06.006" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Bsoft%3A+Image+processing+and+molecular+modeling+for+electron+microscopy." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Bsoft%3A+Image+processing+and+molecular+modeling+for+electron+microscopy.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">63.
              </span><a name="pone.0000749-Leong1" id="pone.0000749-Leong1"></a>Leong PA, Heymann JB, Jensen GJ (2005) Peach: a simple Perl-based system for distributed computation and its application to cryo-EM data processing. Structure  13: 505–511.  <ul class="find" data-citedArticleID="978490" data-doi="10.1016/j.str.2005.01.015"><li><a href="http://dx.doi.org/10.1016/j.str.2005.01.015" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Peach%3A+a+simple+Perl-based+system+for+distributed+computation+and+its+application+to+cryo-EM+data+processing." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Peach%3A+a+simple+Perl-based+system+for+distributed+computation+and+its+application+to+cryo-EM+data+processing.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">64.
              </span><a name="pone.0000749-Frangakis2" id="pone.0000749-Frangakis2"></a>Frangakis AS, Bohm J, Forster F, Nickell S, Nicastro D, et al.  (2002) Identification of macromolecular complexes in cryoelectron tomograms of phantom cells. Proc Natl Acad Sci U S A  99: 14153–14158.  <ul class="find" data-citedArticleID="978462" data-doi="10.1073/pnas.172520299"><li><a href="http://dx.doi.org/10.1073/pnas.172520299" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Identification+of+macromolecular+complexes+in+cryoelectron+tomograms+of+phantom+cells." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Identification+of+macromolecular+complexes+in+cryoelectron+tomograms+of+phantom+cells.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">65.
              </span><a name="pone.0000749-Spahn1" id="pone.0000749-Spahn1"></a>Spahn CM, Beckmann R, Eswar N, Penczek PA, Sali A, et al.  (2001) Structure of the 80S ribosome from <em>Saccharomyces cerevisiae</em>–tRNA-ribosome and subunit-subunit interactions. Cell  107: 373–386.  <ul class="find" data-citedArticleID="978532" data-doi="10.1016/s0092-8674(01)00539-6"><li><a href="http://dx.doi.org/10.1016/s0092-8674(01)00539-6" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Structure+of+the+80S+ribosome+from+Saccharomyces+cerevisiae%26%23x2013%3BtRNA-ribosome+and+subunit-subunit+interactions." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Structure+of+the+80S+ribosome+from+Saccharomyces+cerevisiae%26%23x2013%3BtRNA-ribosome+and+subunit-subunit+interactions.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000749.XML" value="129847"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000749.PDF" value="868795"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000749.g001.PNG_L" value="6766234"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000749.g001.PNG_M" value="219147"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000749.g001.PNG_S" value="11027"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000749.g001.TIF" value="5875932"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000749.g001.PNG_I" value="103699"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000749.g002.PNG_L" value="5274007"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000749.g002.PNG_M" value="371585"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000749.g002.PNG_S" value="16473"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000749.g002.TIF" value="6253588"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000749.g002.PNG_I" value="195406"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000749.g003.PNG_L" value="548930"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000749.g003.PNG_M" value="218904"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000749.g003.PNG_S" value="11141"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000749.g003.TIF" value="494198"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000749.g003.PNG_I" value="70802"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000749.g004.PNG_L" value="2682788"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000749.g004.PNG_M" value="288551"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000749.g004.PNG_S" value="17450"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000749.g004.TIF" value="3398648"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000749.g004.PNG_I" value="169669"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000749.g005.PNG_L" value="3004508"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000749.g005.PNG_M" value="450496"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000749.g005.PNG_S" value="11360"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000749.g005.TIF" value="3601752"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000749.g005.PNG_I" value="147514"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000749.g006.PNG_L" value="4692768"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000749.g006.PNG_M" value="352260"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000749.g006.PNG_S" value="13187"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000749.g006.TIF" value="6816262"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000749.g006.PNG_I" value="104589"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000749.g007.PNG_L" value="2668191"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000749.g007.PNG_M" value="188645"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000749.g007.PNG_S" value="9178"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000749.g007.TIF" value="2486982"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000749.g007.PNG_I" value="77079"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000749.g008.PNG_L" value="970144"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000749.g008.PNG_M" value="127314"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000749.g008.PNG_S" value="7638"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000749.g008.TIF" value="724712"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000749.g008.PNG_I" value="43194"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000749.g009.PNG_L" value="2646222"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000749.g009.PNG_M" value="239178"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000749.g009.PNG_S" value="9561"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000749.g009.TIF" value="3151382"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000749.g009.PNG_I" value="84768"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000749.g010.PNG_L" value="1411796"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000749.g010.PNG_M" value="212302"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000749.g010.PNG_S" value="9542"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000749.g010.TIF" value="1845578"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000749.g010.PNG_I" value="70652"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000749.g011.PNG_L" value="3324788"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000749.g011.PNG_M" value="388835"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000749.g011.PNG_S" value="12051"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000749.g011.TIF" value="4185510"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000749.g011.PNG_I" value="135397"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000749.g012.PNG_L" value="782273"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000749.g012.PNG_M" value="282088"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000749.g012.PNG_S" value="12582"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000749.g012.TIF" value="693382"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000749.g012.PNG_I" value="91125"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000749.g013.PNG_L" value="6149079"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000749.g013.PNG_M" value="338640"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000749.g013.PNG_S" value="10942"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000749.g013.TIF" value="8324520"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000749.g013.PNG_I" value="107265"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000749.g014.PNG_L" value="1703516"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000749.g014.PNG_M" value="187310"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000749.g014.PNG_S" value="8233"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000749.g014.TIF" value="1398420"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000749.g014.PNG_I" value="53533"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000749.s001.MOV" value="5055394"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000749&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000749" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000749&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0000749&volume=&issue=&title=3-D Ultrastructure of O. tauri: Electron Cryotomography of an Entire Eukaryotic Cell&author_name=Gregory%20P.%20Henderson%2C%20Lu%20Gan%2C%20Grant%20J.%20Jensen&start_page=1&end_page=14" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000749" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000749" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0000749" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000749&amp;t=3-D%20Ultrastructure%20of%20O.%20tauri%3A%20Electron%20Cryotomography%20of%20an%20Entire%20Eukaryotic%20Cell" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000749&title=3-D%20Ultrastructure%20of%20O.%20tauri%3A%20Electron%20Cryotomography%20of%20an%20Entire%20Eukaryotic%20Cell&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000749&amp;title=3-D%20Ultrastructure%20of%20O.%20tauri%3A%20Electron%20Cryotomography%20of%20an%20Entire%20Eukaryotic%20Cell" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000749" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0000749&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = '3-D Ultrastructure of O. tauri: Electron Cryotomography of an Entire Eukaryotic Cell';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0000749';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%203-D%20Ultrastructure%20of%20O.%20tauri%3A%20Electron%20Cryotomography%20of%20an%20Entire%20Eukaryotic%20Cell http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000749" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0000749" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">















          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Cell+cycle+and+cell+division%22" title="Search for articles in the subject area:'Cell cycle and cell division'"><div class="flagText">Cell cycle and cell division</div></a>
              <div data-categoryid="17353" data-articleid="24336"
                   data-categoryname="Cell cycle and cell division"
                   class="flagImage" title="Flag 'Cell cycle and cell division' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Cell+membranes%22" title="Search for articles in the subject area:'Cell membranes'"><div class="flagText">Cell membranes</div></a>
              <div data-categoryid="17599" data-articleid="24336"
                   data-categoryname="Cell membranes"
                   class="flagImage" title="Flag 'Cell membranes' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Cellular+structures+and+organelles%22" title="Search for articles in the subject area:'Cellular structures and organelles'"><div class="flagText">Cellular structures and organelles</div></a>
              <div data-categoryid="17573" data-articleid="24336"
                   data-categoryname="Cellular structures and organelles"
                   class="flagImage" title="Flag 'Cellular structures and organelles' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Chloroplasts%22" title="Search for articles in the subject area:'Chloroplasts'"><div class="flagText">Chloroplasts</div></a>
              <div data-categoryid="32715" data-articleid="24336"
                   data-categoryname="Chloroplasts"
                   class="flagImage" title="Flag 'Chloroplasts' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Microtubules%22" title="Search for articles in the subject area:'Microtubules'"><div class="flagText">Microtubules</div></a>
              <div data-categoryid="17423" data-articleid="24336"
                   data-categoryname="Microtubules"
                   class="flagImage" title="Flag 'Microtubules' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Mitochondria%22" title="Search for articles in the subject area:'Mitochondria'"><div class="flagText">Mitochondria</div></a>
              <div data-categoryid="32453" data-articleid="24336"
                   data-categoryname="Mitochondria"
                   class="flagImage" title="Flag 'Mitochondria' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Ribosomes%22" title="Search for articles in the subject area:'Ribosomes'"><div class="flagText">Ribosomes</div></a>
              <div data-categoryid="17985" data-articleid="24336"
                   data-categoryname="Ribosomes"
                   class="flagImage" title="Flag 'Ribosomes' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Vesicles%22" title="Search for articles in the subject area:'Vesicles'"><div class="flagText">Vesicles</div></a>
              <div data-categoryid="17607" data-articleid="24336"
                   data-categoryname="Vesicles"
                   class="flagImage" title="Flag 'Vesicles' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=6787'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=219'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=3310&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>

<div class="block sidebar-comments">
    <div class="header">
        <h3>Comments</h3>
    </div>
      <p><a href="/annotation/listThread.action?root=5073">Great video!</a><br>Posted by csrsanchez</p>
      <p><a href="/annotation/listThread.action?root=381">Referee Comments: Referee 1 (Daniel Vaulot)</a><br>Posted by PLoS_ONE_Group</p>
      <p><a href="/annotation/listThread.action?root=19629">Referee Comments: Referee 2</a><br>Posted by PLoS_ONE_Group</p>
</div>

</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
