

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Sequence Polymorphisms Cause Many False cis eQTLs</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0000622"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0000622" />

    <meta name="citation_title" content="Sequence Polymorphisms Cause Many False cis eQTLs"/>
    <meta itemprop="name" content="Sequence Polymorphisms Cause Many False cis eQTLs"/>

      <meta name="citation_author" content="Rudi Alberts"/>
            <meta name="citation_author_institution" content="Groningen Bioinformatics Centre, Groningen Biomolecular Sciences and Biotechnology Institute, University of Groningen, Haren, The Netherlands"/>
      <meta name="citation_author" content="Peter Terpstra"/>
            <meta name="citation_author_institution" content="Groningen Bioinformatics Centre, University Medical Centre Groningen, University of Groningen, Groningen, The Netherlands"/>
      <meta name="citation_author" content="Yang Li"/>
            <meta name="citation_author_institution" content="Groningen Bioinformatics Centre, Groningen Biomolecular Sciences and Biotechnology Institute, University of Groningen, Haren, The Netherlands"/>
      <meta name="citation_author" content="Rainer Breitling"/>
            <meta name="citation_author_institution" content="Groningen Bioinformatics Centre, Groningen Biomolecular Sciences and Biotechnology Institute, University of Groningen, Haren, The Netherlands"/>
      <meta name="citation_author" content="Jan-Peter Nap"/>
            <meta name="citation_author_institution" content="Groningen Bioinformatics Centre, Groningen Biomolecular Sciences and Biotechnology Institute, University of Groningen, Haren, The Netherlands"/>
            <meta name="citation_author_institution" content="Bioinformatics Expertise Center, Institute for Life Science and Technology, Hanze University Groningen, University for Applied Sciences, Groningen, The Netherlands"/>
      <meta name="citation_author" content="Ritsert C. Jansen"/>
            <meta name="citation_author_institution" content="Groningen Bioinformatics Centre, Groningen Biomolecular Sciences and Biotechnology Institute, University of Groningen, Haren, The Netherlands"/>
            <meta name="citation_author_institution" content="Groningen Bioinformatics Centre, University Medical Centre Groningen, University of Groningen, Groningen, The Netherlands"/>

    <meta name="citation_date" content="2007/7/18"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0000622.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e622"/>
    <meta name="citation_issue" content="7"/>
    <meta name="citation_volume" content="2"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Genetical genomics: the added value from segregation.; citation_author=RC Jansen; citation_author=JP Nap; citation_journal_title=Trends Genet; citation_volume=17; citation_number=1; citation_pages=388-391; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Genetics of global gene expression.; citation_author=MV Rockman; citation_author=L Kruglyak; citation_journal_title=Nat Rev Genet; citation_volume=7; citation_number=2; citation_pages=862-872; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Regulating gene expression: surprises still in store.; citation_author=RC Jansen; citation_author=JP Nap; citation_journal_title=Trends Genet; citation_volume=20; citation_number=3; citation_pages=223-225; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Cis-acting expression quantitative trait loci in mice.; citation_author=S Doss; citation_author=EE Schadt; citation_author=TA Drake; citation_author=AJ Lusis; citation_journal_title=Genome Res; citation_volume=15; citation_number=4; citation_pages=681-691; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Expression profiling using microarrays fabricated by an ink-jet oligonucleotide synthesizer.; citation_author=TR Hughes; citation_author=M Mao; citation_author=AR Jones; citation_author=J Burchard; citation_author=MJ Marton; citation_author=KW Shannon; citation_journal_title=Nat Biotechnol; citation_volume=19; citation_number=5; citation_pages=342-347; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Multi-species microarrays reveal the effect of sequence divergence on gene expression profiles.; citation_author=Y Gilad; citation_author=SA Rifkin; citation_author=P Bertone; citation_author=M Gerstein; citation_author=KP White; citation_journal_title=Genome Res; citation_volume=15; citation_number=6; citation_pages=674-680; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=A statistical multiprobe model for analyzing cis and trans genes in genetical genomics experiments with short-oligonucleotide arrays.; citation_author=R Alberts; citation_author=P Terpstra; citation_author=LV Bystrykh; citation_author=G de Haan; citation_author=RC Jansen; citation_journal_title=Genetics; citation_volume=171; citation_number=7; citation_pages=1437-1439; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Mixed-model reanalysis of primate data suggests tissue and species biases in oligonucleotide-based gene expression profiles.; citation_author=WP Hsieh; citation_author=TM Chu; citation_author=RD Wolfinger; citation_author=G Gibson; citation_journal_title=Genetics; citation_volume=165; citation_number=8; citation_pages=747-757; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Mapping determinants of human gene expression by regional and genome-wide association.; citation_author=VG Cheung; citation_author=RS Spielman; citation_author=KG Ewens; citation_author=TM Weber; citation_author=M Morley; citation_journal_title=Nature; citation_volume=437; citation_number=9; citation_pages=1365-1369; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Uncovering regulatory pathways that affect hematopoietic stem cell function using ‘genetical genomics’.; citation_author=L Bystrykh; citation_author=E Weersing; citation_author=B Dontje; citation_author=S Sutton; citation_author=MT Pletcher; citation_journal_title=Nat Genet; citation_volume=37; citation_number=10; citation_pages=225-232; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=A haplotype map of the human genome.; citation_journal_title=Nature; citation_volume=437; citation_number=11; citation_pages=1299-1320; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Model-based analysis of oligonucleotide arrays: expression index computation and outlier detection.; citation_author=C Li; citation_author=WH Wong; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=98; citation_number=12; citation_pages=31-36; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Exploration, normalization, and summaries of high density oligonucleotide array probe level data.; citation_author=RA Irizarry; citation_author=B Hobbs; citation_author=F Collin; citation_author=YD Beazer-Barclay; citation_author=KJ Antonellis; citation_journal_title=Biostatistics; citation_volume=4; citation_number=13; citation_pages=249-264; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Heritability and Tissue Specificity of Expression Quantitative Trait Loci.; citation_author=E Petretto; citation_author=J Mangion; citation_author=NJ Dickens; citation_author=SA Cook; citation_author=MK Kumaran; citation_journal_title=PLoS Genet; citation_volume=2; citation_number=14; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Mapping Determinants of Gene Expression Plasticity by Genetical Genomics in C. elegans.; citation_author=Y Li; citation_author=OA Alvarez; citation_author=EW Gutteling; citation_author=M Tijsterman; citation_author=J Fu; citation_journal_title=Plos Genetics; citation_volume=2; citation_number=15; citation_pages=e222; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=How replicable are mRNA expression QTL?; citation_author=JL Peirce; citation_author=H Li; citation_author=J Wang; citation_author=KF Manly; citation_author=RJ Hitzemann; citation_journal_title=Mamm Genome; citation_volume=17; citation_number=16; citation_pages=643-656; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Genome-wide detection of polymorphisms at nucleotide resolution with a single DNA microarray.; citation_author=D Gresham; citation_author=DM Ruderfer; citation_author=SC Pratt; citation_author=J Schacherer; citation_author=MJ Dunham; citation_journal_title=Science; citation_volume=311; citation_number=17; citation_pages=1932-1936; citation_date=2006; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000622" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Sequence Polymorphisms Cause Many False cis eQTLs"/>
    <meta name="twitter:description" content="Many investigations have reported the successful mapping of quantitative trait loci (QTLs) for gene expression phenotypes (eQTLs). Local eQTLs, where expression phenotypes map to the genes themselves, are of especially great interest, because they are direct candidates for previously mapped physiological QTLs. Here we show that many mapped local eQTLs in genetical genomics experiments do not reflect actual expression differences caused by sequence polymorphisms in cis-acting factors changing mRNA levels. Instead they indicate hybridization differences caused by sequence polymorphisms in the mRNA region that is targeted by the microarray probes. Many such polymorphisms can be detected by a sensitive and novel statistical approach that takes the individual probe signals into account. Applying this approach to recent mouse and human eQTL data, we demonstrate that indeed many local eQTLs are falsely reported as &ldquo;cis-acting&rdquo; or &ldquo;cis&rdquo; and can be successfully detected and eliminated with this approach."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0000622.g001"/>

  <meta property="og:title" content="Sequence Polymorphisms Cause Many False cis eQTLs" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000622" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=1847'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=4591'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=1180&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0000622">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000622" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000622&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Sequence Polymorphisms Cause Many False <i>cis</i> eQTLs
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Rudi Alberts, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Groningen Bioinformatics Centre, Groningen Biomolecular Sciences and Biotechnology Institute, University of Groningen, Haren, The Netherlands
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Peter Terpstra, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Groningen Bioinformatics Centre, University Medical Centre Groningen, University of Groningen, Groningen, The Netherlands
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Yang Li, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Groningen Bioinformatics Centre, Groningen Biomolecular Sciences and Biotechnology Institute, University of Groningen, Haren, The Netherlands
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Rainer Breitling, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Groningen Bioinformatics Centre, Groningen Biomolecular Sciences and Biotechnology Institute, University of Groningen, Haren, The Netherlands
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Jan-Peter Nap, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliations:
                  Groningen Bioinformatics Centre, Groningen Biomolecular Sciences and Biotechnology Institute, University of Groningen, Haren, The Netherlands, 
                  Bioinformatics Expertise Center, Institute for Life Science and Technology, Hanze University Groningen, University for Applied Sciences, Groningen, The Netherlands
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Ritsert C. Jansen
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:r.c.jansen@rug.nl">r.c.jansen@rug.nl</a></p>

                <p>Affiliations:
                  Groningen Bioinformatics Centre, Groningen Biomolecular Sciences and Biotechnology Institute, University of Groningen, Haren, The Netherlands, 
                  Groningen Bioinformatics Centre, University Medical Centre Groningen, University of Groningen, Groningen, The Netherlands
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: July 18, 2007</li>
    <li>DOI: 10.1371/journal.pone.0000622</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0000622" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0000622" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000622" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0000622" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0000622-g001" data-doi="info:doi/10.1371/journal.pone.0000622" data-uri="info:doi/10.1371/journal.pone.0000622.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000622.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000622">Reader Comments (3)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0000622" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1"></a><p>Many investigations have reported the successful mapping of quantitative trait loci (QTLs) for gene expression phenotypes (eQTLs). Local eQTLs, where expression phenotypes map to the genes themselves, are of especially great interest, because they are direct candidates for previously mapped physiological QTLs. Here we show that many mapped local eQTLs in genetical genomics experiments do not reflect actual expression differences caused by sequence polymorphisms in <em>cis</em>-acting factors changing mRNA levels. Instead they indicate hybridization differences caused by sequence polymorphisms in the mRNA region that is targeted by the microarray probes. Many such polymorphisms can be detected by a sensitive and novel statistical approach that takes the individual probe signals into account. Applying this approach to recent mouse and human eQTL data, we demonstrate that indeed many local eQTLs are falsely reported as “<em>cis</em>-acting” or <em>“cis”</em> and can be successfully detected and eliminated with this approach.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>Alberts R, Terpstra P, Li Y, Breitling R, Nap J-P, et al.  (2007) Sequence Polymorphisms Cause Many False <em>cis</em> eQTLs. PLoS ONE 2(7):
          e622.
            doi:10.1371/journal.pone.0000622</p><p><strong>Academic Editor: </strong>John Storey, University of Washington, United States of America</p><p><strong>Received:</strong> January 9, 2007; <strong>Accepted:</strong> May 29, 2007; <strong>Published:</strong> July 18, 2007</p><p><strong>Copyright:</strong> © 2007 Alberts et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>RA, RB, YL, and JPN received funding for this work from The Netherlands Organisation for Scientific Research.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>Genetical genomics–linkage and association analyses of, for example, gene expression phenotypes with the help of microarray data – is a promising strategy to identify regulatory determinants of complex traits or diseases <a href="#pone.0000622-Jansen1">[1]</a>–<a href="#pone.0000622-Jansen2">[3]</a>. The genetical genomics approach treats the gene expression phenotypes for each individual gene over microarrays as quantitative trait. Combined with a genetic map, quantitative trait variation can be mapped to one or more expression quantitative trait loci (eQTLs). An eQTL is said to be a local eQTL if it is on or near the genomic position of the gene, or distant if it is located elsewhere <a href="#pone.0000622-Rockman1">[2]</a>. A local eQTL can include influential polymorphisms in a <em>cis</em>-acting factor, in which case it is said to be a <em>cis</em>-acting eQTL or <em>cis</em> eQTL in short. Particularly <em>cis</em> eQTLs may identify direct targets for diagnosis and treatment. It is important, therefore, that such <em>cis</em> eQTLs are identified with high accuracy and reliability. However, recent mouse studies showed that no less than 10 out of 28 genes with putative <em>cis</em> eQTLs could not be confirmed by quantitative RT-PCR <a href="#pone.0000622-Doss1">[4]</a>. Notably mRNA sequence diversity in probe regions is known to influence hybridization on microarrays considerably <a href="#pone.0000622-Hughes1">[5]</a>, <a href="#pone.0000622-Gilad1">[6]</a>. The mRNA that is identical to the probes on the microarrays hybridizes better than the mRNA that is not identical to those probes. This causes a difference in signal between individuals with different mRNA variants, even if they have equal amounts of mRNA (gene expression). In a previous analysis we have shown how two SNPs in mouse gene ALDH9A1 caused a differential hybridization signal <a href="#pone.0000622-Alberts1">[7]</a>. Here we show more examples and clearly demonstrate how in expression data from human and mouse, polymorphisms in the mRNA sequence are often falsely interpreted as <em>cis</em> eQTLs.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>If a single or a few probes in a probe set distort the interpretation of the hybridization data, a statistical approach that takes the data of individual probes into account could identify and eliminate deviating probes and use the remaining probes for analysis. In a comparison of human and chimpanzee expression data, Hsieh et al. <a href="#pone.0000622-Hsieh1">[8]</a> eliminated probes one by one until the correlation between the profiles of the two species was &gt;0.95. Although their method is used to eliminate deviating probes, it can not statistically test whether some probes are indeed “telling a different story”. Here we propose an extension of their method into a statistical inference procedure. We present a conceptually simple and sensitive statistical method to detect deviating and potentially problematic probes in a probe set, and we assess the utility of the new method.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4>eQTL analysis</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>In this paper, a human and a mouse expression data set were analyzed. The human data set concerns samples of immortalized lymphoblastoid cells of 57 CEPH individuals hybridized to Affymetrix HG-Focus GeneChips <a href="#pone.0000622-Cheung1">[9]</a> and the mouse data set concerns samples of hematopoietic stem cells of 30 BXD recombinant inbred lines hybridized to Affymetrix MG-U74Av2 GeneChips <a href="#pone.0000622-Bystrykh1">[10]</a>. The expression data were reanalyzed using a previously reported analysis of variance (ANOVA) approach <a href="#pone.0000622-Alberts1">[7]</a>, here extended with a procedure to eliminate deviating probes. In short, the ANOVA model decomposes the probe signals for a given probe set into log(<em>y<sub>ij</sub></em>) = <em>m</em>+<em>P<sub>j</sub></em>+<em>A<sub>i</sub></em>+<em>PA<sub>ij</sub></em>+<em>e<sub>i</sub></em>+<em>e<sub>ij</sub></em>, where <em>y<sub>ij</sub></em> is the hybridization signal of the <em>j</em><sup>th</sup> probe of the <em>i</em><sup>th</sup> sample, <em>m</em> is the average signal, <em>P<sub>j</sub></em> is the average effect of the <em>j</em><sup>th</sup> probe, <em>A<sub>i</sub></em> is the average effect of the allele carried by the <em>i</em><sup>th</sup> sample at a given genome position, <em>PA<sub>ij</sub></em> is the interaction effect between probe and allele type, <em>e<sub>i</sub></em> is an error term per sample and <em>e<sub>ij</sub></em> is a probe-specific error term per sample. For the mouse data, additional parameters for batch effect were added (see <a href="#pone.0000622-Alberts1">[7]</a>). This method is more general and flexible than the correlation approach of Hsieh et al. <a href="#pone.0000622-Hsieh1">[8]</a>.</p>


<h4>Backward elimination of probes from probe sets</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>The ANOVA model is used to calculate the statistical significance of the interaction effects <em>PA<sub>ij</sub></em>. The data on genes with distant eQTLs only, not affected by sequence diversity in the probe regions, give a good estimate of the (limited) amount of interaction present in any probe set. We therefore computed the p-values for statistical significance of the interaction terms <em>PA<sub>ij</sub></em> for each of these genes in mouse, and used the 99<sup>th</sup> percentile of these p-values as a threshold for genes with putative <em>cis</em> eQTLs in the human and mouse data. The procedure for the evaluation of genes with putative <em>cis</em> eQTLs starts by flagging all putative <em>cis</em> eQTLs with a significant interaction effect, <em>i.e.</em> below the threshold. Next, for all flagged genes, each individual probe is temporarily removed and the interaction effect among the remaining probes is calculated. The probe whose removal caused the largest increase in p-value of interaction effects is permanently eliminated. This procedure is repeated with the remaining probes until the p-value of interaction effects is above the threshold. The remaining probes are used for a final eQTL analysis. In cases where many probes contain SNPs, probes affected by SNPs but also probes not affected by SNPs can be considered as outliers and can be eliminated by the statistical method. To take this into consideration, all genes for which over 50% of the probes are eliminated remain flagged as having potential false <em>cis</em> eQTLs.</p>

</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Results"></a><h3>Results</h3>
<h4>Human Affymetrix GeneChip arrays</h4>
<a id="article1.body1.sec3.sec1.p1" name="article1.body1.sec3.sec1.p1"></a><p>We simulated the occurrence of randomly distributed SNPs in Affymetrix probes using a conservative prediction of one SNP per 1000 base pairs <a href="#pone.0000622-International1">[11]</a>. This simulation showed that many probe sets are expected to contain one or more SNPs. For example, we predict that at least 200 probe sets of 11 probes per probe set will carry SNP variation in three or more probes on the human genome HG-U133 Plus 2.0 arrays. We also studied the distribution of known HapMap SNPs within probes; we BLASTed the Affymetrix probes of HG-U133 Plus 2.0 against the HapMap SNPs in the ten 500-kilobase ENCODE regions that were resequenced in 48 unrelated DNA samples (<a href="http://www.hapmap.org/downloads/encode1.html.en">http://www.hapmap.org/downloads/encode1.​html.en</a>). Among the 136 probe sets present in these HapMap-ENCODE regions, 33 probe sets have SNPs in at least one probe (24%), and 5 probe sets have SNPs in three or more probes (4%). The hybridization expected to result from such variation could seriously mislead the interpretation of data from individual genes, even if only a single probe is affected. In an extensive genetical genomics experiment using Affymetrix arrays on 57 CEPH individuals, thirteen putative <em>cis</em> eQTLs were found in immortalized lymphoblastoid cells <a href="#pone.0000622-Cheung1">[9]</a>. One of these eQTLs was for gene HSD17B12. Analysis of variance of the probe data of HSD17B12 with the statistical method outlined above confirms strong evidence for a local (possibly <em>cis</em>) eQTL when all probe data are used (<a href="#pone-0000622-g001">Figure 1</a>). However, the evidence becomes non-significant when the data of only probe 8 is left out. Inspection of the individual probe data shows that probe 8 is the only one of eleven probes that shows a clear differential hybridization signal. The Affymetrix MAS 5.0 algorithm that is routinely used to summarize probe level data is not able to single out this single probe, neither are the alternative methods dChip <a href="#pone.0000622-Li1">[12]</a> or RMA <a href="#pone.0000622-Irizarry1">[13]</a>. In the CEPH individuals used, the HapMap data <a href="#pone.0000622-International1">[11]</a> only report an SNP (rs1061810) in probe 8, that is located at position 15 in the probe and shows A/C variation for the CEPH individuals. This SNP is strongly linked with the SNP (rs4755741) found in the association study that shows A/G variation. The data therefore indicate that the mRNA from individuals that are homozygous A for SNP rs4755741 hybridizes better than mRNA from individuals that are homozygous G for that SNP. This is as expected, because the mRNA from individuals that are homozygous A is identical to the probe sequence, whereas the mRNA from individuals that are homozygous G has a mismatch. Our procedure correctly flagged gene HSD17B12 and correctly eliminated probe 8.</p>
<div class="figure" id="pone-0000622-g001"><div class="img"><a name="pone-0000622-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000622.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000622" data-uri="info:doi/10.1371/journal.pone.0000622.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000622.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000622.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000622.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000622.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000622.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000622.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000622.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000622.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000622.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Identification of false <em>cis</em> eQTLs reported in a human association analysis.</span></strong></p><a id="article1.body1.sec3.sec1.fig1.caption1.p1" name="article1.body1.sec3.sec1.fig1.caption1.p1"></a><p>(A) Relative probe positions on the mRNA sequence (top) and hybridization signals (bottom) for gene HSD17B12 for which a <em>cis</em> eQTL is reported <a href="#pone.0000622-Cheung1">[9]</a>. Each line represents one individual and is colored according to the allele that the individual carries for the associating SNP marker rs4755741. This marker is located in an intron of the HSD17B12 gene and it is strongly linked with SNP rs1061810 located in probe 8. By discarding the data for probe 8, the significance for a <em>cis</em> eQTL disappears.</p>
<a id="article1.body1.sec3.sec1.fig1.caption1.p2" name="article1.body1.sec3.sec1.fig1.caption1.p2"></a><p>(B) Similar plot for gene HLA-DQB1 for which a <em>cis</em> eQTL is reported <a href="#pone.0000622-Cheung1">[9]</a>. Lines are colored according to the allele the individual carries for the associated SNP marker rs6928482. One red-line and one blue-line individual have been sequenced for the probe region; the numbers in red and blue indicate the positions of SNPs within the 25-mer probe regions. The number in italic indicates a single nucleotide insertion in probe 2. We observed 23 SNPs (15 new ones) between these mRNA sequences and the 11 probes.</p>
<a id="article1.body1.sec3.sec1.fig1.caption1.p3" name="article1.body1.sec3.sec1.fig1.caption1.p3"></a><p>(C) Visualization of the probes for gene HSD17B12 in the UCSC Genome Browser. Below the genomic sequence the probes are displayed in blocks. The block labels contain: probe set name, probe number, orientation on the genome (&gt;&gt; or &lt;&lt;) and probe sequence. Probe 8 and SNP rs1061810 are encircled. There is one SNP in probe 8, as was expected from probe signals. The inserted area shows information about this SNP; dbSNP shows the diversity of the SNP in the CEPH population.</p>
<a id="article1.body1.sec3.sec1.fig1.caption1.p4" name="article1.body1.sec3.sec1.fig1.caption1.p4"></a><p>(D) Similar plot for gene HLA-DQB1. Probes 1-4 that do not perfectly match the genome are displayed in light blue. The current data in Genome Browser show fewer SNPs than we found in our own sequencing effort.</p>
<span>doi:10.1371/journal.pone.0000622.g001</span></div><a id="article1.body1.sec3.sec1.p2" name="article1.body1.sec3.sec1.p2"></a><p>A similar case is HLA-DQB1, reported as HLA-DRB2, for which a <em>cis</em> eQTL was reported in the same study <a href="#pone.0000622-Cheung1">[9]</a>. We sequenced the HLA-DQB1 alleles of two individuals. Differences in the hybridization signal between these individuals could be attributed to differences between probe sequences and the actual mRNA sequences from different individuals (<a href="#pone-0000622-g001">Figure 1</a>). Our procedure correctly flagged gene HLA-DQB1 because more than 50% of its probes would have to be eliminated.</p>


<h4>Mouse Affymetrix GeneChip arrays</h4>
<a id="article1.body1.sec3.sec2.p1" name="article1.body1.sec3.sec2.p1"></a><p>In an extensive genetical genomics experiment using Affymetrix arrays on 30 recombinant inbred lines in mouse, many putative <em>cis</em> eQTLs were reported <a href="#pone.0000622-Bystrykh1">[10]</a>. Here we estimate how many of these putative <em>cis</em> eQTLs were caused by differential hybridization resulting from polymorphisms in probe regions, rather then by differential expression. We simulated the occurrence of randomly distributed SNPs in the Affymetrix mouse probes using a conservative estimate of one SNP per 1000 base pairs. This simulation confirmed that again many probe sets are expected to contain one or more SNPs. For example, we predict that at least 302 probe sets will carry SNP variation in three or more probes on the mouse genome U74Av2 arrays with 16 probes per probe set.</p>
<a id="article1.body1.sec3.sec2.p2" name="article1.body1.sec3.sec2.p2"></a><p>To study the potential influence of SNPs in probe sets on <em>cis</em> eQTL identification, we investigated the 100 most significant putative <em>cis</em> eQTLs in detail. <em>Cis</em> eQTLs can show a higher hybridization signal for the mice carrying the B6 allele (called <em>cis</em><sup>B6</sup>), or a higher hybridization signal for the mice carrying the D2 allele (called <em>cis</em><sup>D2</sup>). Because the microarray was primarily designed based on the B6 sequence, the occurrence of sequence diversity in probe regions would predict an excess of <em>cis</em><sup>B6</sup> eQTLs. Indeed there were significantly more <em>cis</em><sup>B6</sup> eQTLs: 70 <em>cis</em><sup>B6</sup> vs. 30 <em>cis</em><sup>D2</sup> (P&lt;0.01; chi-square test). Without sequence diversity in probe regions we would expect as many <em>cis</em><sup>B6</sup> eQTLs as <em>cis</em><sup>D2</sup> eQTLs, that is, 30. The number of false <em>cis</em><sup>B6</sup> eQTLs is estimated as the observed number of <em>cis</em><sup>B6</sup> minus the expected number of <em>cis</em><sup>B6</sup>, so 70–30 = 40. This shows that almost half of the reported 100 most significant <em>cis</em> eQTLs are probably due to sequence diversity in probe regions. When applying the statistical method outlined above, 25 of the 70 <em>cis</em><sup>B6</sup> eQTL (36%) were flagged as potentially false <em>cis</em> eQTLs. Two were flagged because more than 50% of the probes were eliminated, 22 other <em>cis</em><sup>B6</sup> lost significance, and one <em>cis</em><sup>B6</sup> became <em>cis</em><sup>D2</sup>. In addition, 2 of the 30 <em>cis</em><sup>D2</sup> eQTL (7%) were flagged as potentially false for reasons that we discuss below. After the backward probe elimination procedure, (70–25 = ) 45 <em>cis</em><sup>B6</sup> and (30–2+1 = ) 29 <em>cis</em><sup>D2</sup> eQTLs remained, which is not a significant excess (P = 0.21; chi-square test). Our statistical method reduced the initial distortion to a non-significant level.</p>
<a id="article1.body1.sec3.sec2.p3" name="article1.body1.sec3.sec2.p3"></a><p>To assess the utility of our method, we focused on the 32 known SNPs between B6 and D2 in probes of the 100 most significant putative <em>cis</em> eQTLs. From these, 30 SNPs were derived from <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/">http://www.ncbi.nlm.nih.gov/projects/SNP​/</a> and two additional SNPs we had identified earlier <a href="#pone.0000622-Alberts1">[7]</a>. These 32 SNPs affect 25 probe sets on the microarray. Combining the hybridization signals with the SNP data, we showed that in 15 of the 25 affected probe sets the SNPs caused a difference in hybridization. Our method correctly flagged these 15 SNP-containing probe sets and successfully identified and eliminated only the SNP containing probes in those probe sets. In the remaining 10 probe sets the SNPs had no effect. This demonstrates that not every SNP causes a difference in hybridization. When the SNP is located at the very beginning or end of a probe, it can have little or even no effect on hybridization <a href="#pone.0000622-Hughes1">[5]</a>. These probe sets, for which no probes needed to be eliminated, correctly remained unflagged.</p>
<a id="article1.body1.sec3.sec2.p4" name="article1.body1.sec3.sec2.p4"></a><p>Most probes on the mouse array are based on the B6 sequence, but not all. It is therefore possible that a deletion or insertion in the B6 sequence and/or an SNP between the B6 sequence and the probe sequences can cause false <em>cis</em><sup>D2</sup> eQTLs. Indeed, one of the two <em>cis</em><sup>D2</sup> eQTLs that were flagged was found to be false: it was caused by a combination of a known deletion and an SNP in the B6 sequence of gene H2-D1 in comparison to the probe sequences.</p>

</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec4.p1" name="article1.body1.sec4.p1"></a><p>In genetical genomics experiments, putative <em>cis</em> eQTLs are thought to often reflect differential gene expression between individuals <a href="#pone.0000622-Jansen1">[1]</a>–<a href="#pone.0000622-Doss1">[4]</a>. However, it is shown here that in many cases such putative <em>cis</em> eQTLs should be considered with extra care: the data that are interpreted as reflecting differential gene expression could actually be due to sequence diversity in the probe regions. Without expression differences, such erroneous <em>cis</em> eQTLs will not be targets for diagnosis and treatment.</p>
<a id="article1.body1.sec4.p2" name="article1.body1.sec4.p2"></a><p>Our analysis of short-oligomer data from human and mouse Affymetrix microarrays demonstrates that the issue of differential hybridization due to sequence diversity in probe regions is systematic. In the mouse data, it caused an excess of <em>cis</em><sup>B6</sup> eQTLs, because the arrays were designed using sequences of the B6 parental line. In a recent rat study <a href="#pone.0000622-Petretto1">[14]</a> probes were mainly based on ESTs and cDNA sequences from outbred animals and not from one of the parental strains used to generate the segregating population. Therefore no excess was observed, but this absence of an excess does not mean that the data do not suffer from sequence diversity in probe regions.</p>
<a id="article1.body1.sec4.p3" name="article1.body1.sec4.p3"></a><p>The occurrence of differential hybridization due to sequence diversity in probe regions may be thought specific for short-oligomer arrays: sequence differences in short sequences are supposed to affect hybridization more than in longer sequences. However, we also used 60-mer cDNA microarrays with one probe per gene, designed on the basis of the sequence of the N2 strain of <em>Caenorhabditis elegans</em> <a href="#pone.0000622-Li2">[15]</a>. Among the 100 most significant putative <em>cis</em> eQTLs there was an excess of 74 <em>cis</em><sup>N2</sup> eQTLs (enrichment significant at P&lt;&lt;0.001; chi-square test). This may indicate that sequence diversity in probe regions can also result in false <em>cis</em> eQTLs in case of long-oligomer microarrays.</p>
<a id="article1.body1.sec4.p4" name="article1.body1.sec4.p4"></a><p>To properly deal with the issue of differential hybridization due to sequence diversity in probe regions, we recommend using multiple (tiling) probes per gene that allow statistical filtering as developed in this paper. Three metrics were helpful to assess the utility of the statistical method in a set of 100 most significant putative <em>cis</em> eQTLs in a mouse study:

<ol class="order">

<li>Is the statistical method flagging as many genes with <em>cis</em><sup>B6</sup> eQTL as the observed excess of <em>cis</em><sup>B6</sup> over <em>cis</em><sup>D2</sup> eQTLs? The excess is estimated to be 40, of which 25 are flagged. This could suggest that 15 false positive <em>cis</em><sup>B6</sup> eQTLs go unnoticed. The power would then be approximately 63%. However, after applying our procedure the remaining unflagged genes do not show a significant excess and this deviation from equal proportions could just as well reflect random variation. That implies that the power can be much higher than 63%. If wished, we could increase the power by changing the settings of the elimination procedure to flag more genes with <em>cis</em><sup>B6</sup> eQTLs (possibly leading to more false negatives). Peirce et al. <a href="#pone.0000622-Peirce1">[16]</a> suggest that other biological phenomena, for example directed loss of DNA in D2 relative to B6, could explain some excess of <em>cis</em><sup>B6</sup> eQTLs.</li>

<li>Is the statistical method flagging only genes with <em>cis</em><sup>B6</sup> eQTLs and no genes with <em>cis</em><sup>D2</sup> eQTLs? In addition to 25 genes with <em>cis</em><sup>B6</sup> eQTLs, two genes with <em>cis</em><sup>D2</sup> eQTLs were flagged. One of these <em>cis</em><sup>D2</sup> eQTLs was due to a deletion and SNP, <em>i.e.</em> the gene was correctly flagged. The enrichment of 25 <em>cis</em><sup>B6</sup> over 1 <em>cis</em><sup>D2</sup> is striking. Moreover, it cannot be excluded that also the second gene with <em>cis</em><sup>D2</sup> eQTL is flagged correctly. These results strongly suggest that the method is very specifically flagging a <em>cis</em><sup>B6</sup> related phenomenon without erroneous rejections.</li>

<li>Is the statistical method flagging all probe sets carrying known SNPs? Indeed, the method correctly flagged all 15 SNP-containing probe sets with influential sequence diversity in probe regions, successfully eliminated only the SNP containing probes in those probe sets, and identified 100% of the false <em>cis</em> eQTLs. Probe sets with known SNPs with no effect on hybridization were not affected by the method (no probes eliminated; gene not flagged). Amongst the 25 flagged genes with <em>cis</em><sup>B6</sup> eQTLs, 15 carry known SNPs, but the remaining 10 do not. This could suggest 40% erroneous rejections of true <em>cis</em> eQTLs but, in combination with (ii) above, more likely reflects the (still) incomplete information on SNPs or other forms of polymorphisms, such as insertions and deletions between B6 and D2.</li>

</ol>Obviously statistics alone can not solve what is essentially a biological phenomenon: sequence diversity in probe regions between individuals. For this reason we strongly recommend that additional genome-wide methods to characterize polymorphisms <a href="#pone.0000622-Gresham1">[17]</a>, re-sequencing of probe regions, and alternative ways of gene expression profiling are employed whenever strong claims about <em>cis</em> eQTLs are to be made. Only then large-scale mapping of the determinants of gene expression will become truly informative.</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>The authors wish to thank Gerard te Meerman and an anonymous reviewer for helpful comments.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Analyzed the data: RJ. Wrote the paper: RJ RB RA JN. Other: Initiated and supervised the study: RCJ. Conceived the study of individual probes: RCJ. Interpreted the data: RCJ RA JPN. Developed the elimination method: RCJ RA. Analyzed all probe level data: RA. Performed sequencing of human samples: PT. Analyzed the <em>C. elegans</em> data: YL. Calculated SNP frequency in probe sets: RB.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0000622-Jansen1" id="pone.0000622-Jansen1"></a>Jansen RC, Nap JP (2001) Genetical genomics: the added value from segregation. Trends Genet  17: 388–391.  <ul class="find" data-citedArticleID="966986" data-doi="10.1016/s0168-9525(01)02310-1"><li><a href="http://dx.doi.org/10.1016/s0168-9525(01)02310-1" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genetical+genomics%3A+the+added+value+from+segregation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genetical+genomics%3A+the+added+value+from+segregation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0000622-Rockman1" id="pone.0000622-Rockman1"></a>Rockman MV, Kruglyak L (2006) Genetics of global gene expression. Nat Rev Genet  7: 862–872.  <ul class="find" data-citedArticleID="966998" data-doi="10.1038/nrg1964"><li><a href="http://dx.doi.org/10.1038/nrg1964" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genetics+of+global+gene+expression." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genetics+of+global+gene+expression.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0000622-Jansen2" id="pone.0000622-Jansen2"></a>Jansen RC, Nap JP (2004) Regulating gene expression: surprises still in store. Trends Genet  20: 223–225.  <ul class="find" data-citedArticleID="966988" data-doi="10.1016/j.tig.2004.03.002"><li><a href="http://dx.doi.org/10.1016/j.tig.2004.03.002" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Regulating+gene+expression%3A+surprises+still+in+store." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Regulating+gene+expression%3A+surprises+still+in+store.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0000622-Doss1" id="pone.0000622-Doss1"></a>Doss S, Schadt EE, Drake TA, Lusis AJ (2005) Cis-acting expression quantitative trait loci in mice. Genome Res  15: 681–691.  <ul class="find" data-citedArticleID="966972" data-doi="10.1101/gr.3216905"><li><a href="http://dx.doi.org/10.1101/gr.3216905" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Cis-acting+expression+quantitative+trait+loci+in+mice." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Cis-acting+expression+quantitative+trait+loci+in+mice.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0000622-Hughes1" id="pone.0000622-Hughes1"></a>Hughes TR, Mao M, Jones AR, Burchard J, Marton MJ, Shannon KW, et al.  (2001) Expression profiling using microarrays fabricated by an ink-jet oligonucleotide synthesizer. Nat Biotechnol  19: 342–347.  <ul class="find" data-citedArticleID="966980"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Expression+profiling+using+microarrays+fabricated+by+an+ink-jet+oligonucleotide+synthesizer.&amp;auth=&amp;atitle=Expression+profiling+using+microarrays+fabricated+by+an+ink-jet+oligonucleotide+synthesizer." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Expression+profiling+using+microarrays+fabricated+by+an+ink-jet+oligonucleotide+synthesizer." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Expression+profiling+using+microarrays+fabricated+by+an+ink-jet+oligonucleotide+synthesizer.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0000622-Gilad1" id="pone.0000622-Gilad1"></a>Gilad Y, Rifkin SA, Bertone P, Gerstein M, White KP (2005) Multi-species microarrays reveal the effect of sequence divergence on gene expression profiles. Genome Res  15: 674–680.  <ul class="find" data-citedArticleID="966974" data-doi="10.1101/gr.3335705"><li><a href="http://dx.doi.org/10.1101/gr.3335705" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Multi-species+microarrays+reveal+the+effect+of+sequence+divergence+on+gene+expression+profiles." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Multi-species+microarrays+reveal+the+effect+of+sequence+divergence+on+gene+expression+profiles.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0000622-Alberts1" id="pone.0000622-Alberts1"></a>Alberts R, Terpstra P, Bystrykh LV, de Haan G, Jansen RC (2005) A statistical multiprobe model for analyzing cis and trans genes in genetical genomics experiments with short-oligonucleotide arrays. Genetics  171: 1437–1439.  <ul class="find" data-citedArticleID="966966" data-doi="10.1534/genetics.105.045930"><li><a href="http://dx.doi.org/10.1534/genetics.105.045930" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+statistical+multiprobe+model+for+analyzing+cis+and+trans+genes+in+genetical+genomics+experiments+with+short-oligonucleotide+arrays." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+statistical+multiprobe+model+for+analyzing+cis+and+trans+genes+in+genetical+genomics+experiments+with+short-oligonucleotide+arrays.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0000622-Hsieh1" id="pone.0000622-Hsieh1"></a>Hsieh WP, Chu TM, Wolfinger RD, Gibson G (2003) Mixed-model reanalysis of primate data suggests tissue and species biases in oligonucleotide-based gene expression profiles. Genetics  165: 747–757.  <ul class="find" data-citedArticleID="966978"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Mixed-model+reanalysis+of+primate+data+suggests+tissue+and+species+biases+in+oligonucleotide-based+gene+expression+profiles.&amp;auth=&amp;atitle=Mixed-model+reanalysis+of+primate+data+suggests+tissue+and+species+biases+in+oligonucleotide-based+gene+expression+profiles." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mixed-model+reanalysis+of+primate+data+suggests+tissue+and+species+biases+in+oligonucleotide-based+gene+expression+profiles." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mixed-model+reanalysis+of+primate+data+suggests+tissue+and+species+biases+in+oligonucleotide-based+gene+expression+profiles.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0000622-Cheung1" id="pone.0000622-Cheung1"></a>Cheung VG, Spielman RS, Ewens KG, Weber TM, Morley M, et al.  (2005) Mapping determinants of human gene expression by regional and genome-wide association. Nature  437: 1365–1369.  <ul class="find" data-citedArticleID="966970" data-doi="10.1038/nature04244"><li><a href="http://dx.doi.org/10.1038/nature04244" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mapping+determinants+of+human+gene+expression+by+regional+and+genome-wide+association." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mapping+determinants+of+human+gene+expression+by+regional+and+genome-wide+association.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0000622-Bystrykh1" id="pone.0000622-Bystrykh1"></a>Bystrykh L, Weersing E, Dontje B, Sutton S, Pletcher MT, et al.  (2005) Uncovering regulatory pathways that affect hematopoietic stem cell function using ‘genetical genomics’. Nat Genet  37: 225–232.  <ul class="find" data-citedArticleID="966968" data-doi="10.1038/ng1497"><li><a href="http://dx.doi.org/10.1038/ng1497" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Uncovering+regulatory+pathways+that+affect+hematopoietic+stem+cell+function+using+%26%23x2018%3Bgenetical+genomics%26%23x2019%3B." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Uncovering+regulatory+pathways+that+affect+hematopoietic+stem+cell+function+using+%26%23x2018%3Bgenetical+genomics%26%23x2019%3B.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0000622-International1" id="pone.0000622-International1"></a>International HapMap Consortium (2005) A haplotype map of the human genome. Nature  437: 1299–1320.  <ul class="find" data-citedArticleID="966982" data-doi="10.1038/nature04226"><li><a href="http://dx.doi.org/10.1038/nature04226" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+haplotype+map+of+the+human+genome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+haplotype+map+of+the+human+genome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0000622-Li1" id="pone.0000622-Li1"></a>Li C, Wong WH (2001) Model-based analysis of oligonucleotide arrays: expression index computation and outlier detection. Proc Natl Acad Sci U S A  98: 31–36.  <ul class="find" data-citedArticleID="966990" data-doi="10.1073/pnas.98.1.31"><li><a href="http://dx.doi.org/10.1073/pnas.98.1.31" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Model-based+analysis+of+oligonucleotide+arrays%3A+expression+index+computation+and+outlier+detection." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Model-based+analysis+of+oligonucleotide+arrays%3A+expression+index+computation+and+outlier+detection.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0000622-Irizarry1" id="pone.0000622-Irizarry1"></a>Irizarry RA, Hobbs B, Collin F, Beazer-Barclay YD, Antonellis KJ, et al.  (2003) Exploration, normalization, and summaries of high density oligonucleotide array probe level data. Biostatistics  4: 249–264.  <ul class="find" data-citedArticleID="966984" data-doi="10.1093/biostatistics/4.2.249"><li><a href="http://dx.doi.org/10.1093/biostatistics/4.2.249" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Exploration%2C+normalization%2C+and+summaries+of+high+density+oligonucleotide+array+probe+level+data." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Exploration%2C+normalization%2C+and+summaries+of+high+density+oligonucleotide+array+probe+level+data.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0000622-Petretto1" id="pone.0000622-Petretto1"></a>Petretto E, Mangion J, Dickens NJ, Cook SA, Kumaran MK, et al.  (2006) Heritability and Tissue Specificity of Expression Quantitative Trait Loci. PLoS Genet  2:   <ul class="find" data-citedArticleID="966996" data-doi="10.1371/journal.pgen.0020172"><li><a href="http://dx.doi.org/10.1371/journal.pgen.0020172" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Heritability+and+Tissue+Specificity+of+Expression+Quantitative+Trait+Loci." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Heritability+and+Tissue+Specificity+of+Expression+Quantitative+Trait+Loci.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0000622-Li2" id="pone.0000622-Li2"></a>Li Y, Alvarez OA, Gutteling EW, Tijsterman M, Fu J, et al.  (2006) Mapping Determinants of Gene Expression Plasticity by Genetical Genomics in C. elegans. Plos Genetics  2: e222.  <ul class="find" data-citedArticleID="966992" data-doi="10.1371/journal.pgen.0020222.eor"><li><a href="http://dx.doi.org/10.1371/journal.pgen.0020222.eor" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mapping+Determinants+of+Gene+Expression+Plasticity+by+Genetical+Genomics+in+C.+elegans." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mapping+Determinants+of+Gene+Expression+Plasticity+by+Genetical+Genomics+in+C.+elegans.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0000622-Peirce1" id="pone.0000622-Peirce1"></a>Peirce JL, Li H, Wang J, Manly KF, Hitzemann RJ, et al.  (2006) How replicable are mRNA expression QTL? Mamm Genome  17: 643–656.  <ul class="find" data-citedArticleID="966994" data-doi="10.1007/s00335-005-0187-8"><li><a href="http://dx.doi.org/10.1007/s00335-005-0187-8" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=How+replicable+are+mRNA+expression+QTL%3F" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22How+replicable+are+mRNA+expression+QTL%3F%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0000622-Gresham1" id="pone.0000622-Gresham1"></a>Gresham D, Ruderfer DM, Pratt SC, Schacherer J, Dunham MJ, et al.  (2006) Genome-wide detection of polymorphisms at nucleotide resolution with a single DNA microarray. Science  311: 1932–1936.  <ul class="find" data-citedArticleID="966976" data-doi="10.1126/science.1123726"><li><a href="http://dx.doi.org/10.1126/science.1123726" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genome-wide+detection+of+polymorphisms+at+nucleotide+resolution+with+a+single+DNA+microarray." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genome-wide+detection+of+polymorphisms+at+nucleotide+resolution+with+a+single+DNA+microarray.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000622.XML" value="45572"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000622.PDF" value="837315"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000622.g001.PNG_L" value="1633741"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000622.g001.PNG_M" value="279444"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000622.g001.PNG_S" value="16968"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000622.g001.TIF" value="2771270"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000622.g001.PNG_I" value="126603"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000622&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000622" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000622&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0000622&volume=&issue=&title=Sequence Polymorphisms Cause Many False cis eQTLs&author_name=Rudi%20Alberts%2C%20Peter%20Terpstra%2C%20Yang%20Li%2C%20Rainer%20Breitling%2C%20Jan-Peter%20Nap%2C%20Ritsert%20C.%20Jansen&start_page=1&end_page=5" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000622" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000622" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0000622" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000622&amp;t=Sequence%20Polymorphisms%20Cause%20Many%20False%20cis%20eQTLs" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000622&title=Sequence%20Polymorphisms%20Cause%20Many%20False%20cis%20eQTLs&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000622&amp;title=Sequence%20Polymorphisms%20Cause%20Many%20False%20cis%20eQTLs" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000622" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0000622&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Sequence Polymorphisms Cause Many False cis eQTLs';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0000622';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Sequence%20Polymorphisms%20Cause%20Many%20False%20cis%20eQTLs http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000622" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0000622" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">











          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22DNA+sequences%22" title="Search for articles in the subject area:'DNA sequences'"><div class="flagText">DNA sequences</div></a>
              <div data-categoryid="32003" data-articleid="24082"
                   data-categoryname="DNA sequences"
                   class="flagImage" title="Flag 'DNA sequences' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Gene+expression%22" title="Search for articles in the subject area:'Gene expression'"><div class="flagText">Gene expression</div></a>
              <div data-categoryid="17515" data-articleid="24082"
                   data-categoryname="Gene expression"
                   class="flagImage" title="Flag 'Gene expression' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Genomic+signal+processing%22" title="Search for articles in the subject area:'Genomic signal processing'"><div class="flagText">Genomic signal processing</div></a>
              <div data-categoryid="18635" data-articleid="24082"
                   data-categoryname="Genomic signal processing"
                   class="flagImage" title="Flag 'Genomic signal processing' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Mammalian+genomics%22" title="Search for articles in the subject area:'Mammalian genomics'"><div class="flagText">Mammalian genomics</div></a>
              <div data-categoryid="17625" data-articleid="24082"
                   data-categoryname="Mammalian genomics"
                   class="flagImage" title="Flag 'Mammalian genomics' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Microarrays%22" title="Search for articles in the subject area:'Microarrays'"><div class="flagText">Microarrays</div></a>
              <div data-categoryid="17255" data-articleid="24082"
                   data-categoryname="Microarrays"
                   class="flagImage" title="Flag 'Microarrays' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Probe+hybridization%22" title="Search for articles in the subject area:'Probe hybridization'"><div class="flagText">Probe hybridization</div></a>
              <div data-categoryid="19653" data-articleid="24082"
                   data-categoryname="Probe hybridization"
                   class="flagImage" title="Flag 'Probe hybridization' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Statistical+data%22" title="Search for articles in the subject area:'Statistical data'"><div class="flagText">Statistical data</div></a>
              <div data-categoryid="32437" data-articleid="24082"
                   data-categoryname="Statistical data"
                   class="flagImage" title="Flag 'Statistical data' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Statistical+methods%22" title="Search for articles in the subject area:'Statistical methods'"><div class="flagText">Statistical methods</div></a>
              <div data-categoryid="35231" data-articleid="24082"
                   data-categoryname="Statistical methods"
                   class="flagImage" title="Flag 'Statistical methods' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=9769'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=3043'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=7517&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>

<div class="block sidebar-comments">
    <div class="header">
        <h3>Comments</h3>
    </div>
      <p><a href="/annotation/listThread.action?root=19161">Referee Comments: Referee #2</a><br>Posted by PLoS_ONE_Group</p>
      <p><a href="/annotation/listThread.action?root=9137">Surprise!</a><br>Posted by abeyer</p>
      <p><a href="/annotation/listThread.action?root=45323">an important aspect</a><br>Posted by wzhang1</p>
</div>

</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
