

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Mapping Genetically Compensatory Pathways from Synthetic Lethal Interactions in Yeast</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0001922"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0001922" />

    <meta name="citation_title" content="Mapping Genetically Compensatory Pathways from Synthetic Lethal Interactions in Yeast"/>
    <meta itemprop="name" content="Mapping Genetically Compensatory Pathways from Synthetic Lethal Interactions in Yeast"/>

      <meta name="citation_author" content="Xiaotu Ma"/>
            <meta name="citation_author_institution" content="Molecular and Computational Biology Program, Department of Biological Sciences, University of Southern California, Los Angeles, California, United States of America"/>
      <meta name="citation_author" content="Aaron M. Tarone"/>
            <meta name="citation_author_institution" content="Molecular and Computational Biology Program, Department of Biological Sciences, University of Southern California, Los Angeles, California, United States of America"/>
      <meta name="citation_author" content="Wenyuan Li"/>
            <meta name="citation_author_institution" content="Molecular and Computational Biology Program, Department of Biological Sciences, University of Southern California, Los Angeles, California, United States of America"/>

    <meta name="citation_date" content="2008/4/9"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0001922.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e1922"/>
    <meta name="citation_issue" content="4"/>
    <meta name="citation_volume" content="3"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Systematic interpretation of genetic interactions using protein networks.; citation_author=R Kelley; citation_author=T Ideker; citation_journal_title=Nat Biotechnol; citation_volume=23; citation_number=1; citation_pages=561-566; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Exploring genetic interactions and networks with yeast.; citation_author=C Boone; citation_author=H Bussey; citation_author=B Andrews; citation_journal_title=Nat Rev Genet; citation_volume=8; citation_number=2; citation_pages=437-449; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Functional profiling of the Saccharomyces cerevisiae genome.; citation_author=G Giaever; citation_author=A Chu; citation_author=L Ni; citation_author=C Connelly; citation_author=L Riles; citation_journal_title=Nature; citation_volume=418; citation_number=3; citation_pages=387-391; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Systematic genetic analysis with ordered arrays of yeast deletion mutants.; citation_author=A Tong; citation_author=M Evangelista; citation_author=A Parsons; citation_author=H Xu; citation_author=G Bader; citation_journal_title=Science; citation_volume=294; citation_number=4; citation_pages=2364-2368; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Global mapping of the yeast genetic interaction network.; citation_author=A Tong; citation_author=G Lesage; citation_author=G Bader; citation_author=H Ding; citation_author=H Xu; citation_journal_title=Science; citation_volume=303; citation_number=5; citation_pages=808-813; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=BioGRID: a general repository for interaction datasets.; citation_author=C Stark; citation_author=B Breitkreutz; citation_author=T Reguly; citation_author=L Boucher; citation_author=A Breitkreutz; citation_journal_title=Nucleic Acids Res; citation_volume=34; citation_number=6; citation_pages=D535-539; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Gene function prediction from congruent synthetic lethal interactions in yeast.; citation_author=P Ye; citation_author=B Peyser; citation_author=X Pan; citation_author=J Boeke; citation_author=F Spencer; citation_journal_title=Mol Syst Biol; citation_volume=1; citation_number=7; citation_pages=2005.0026; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Pathway redundancy and protein essentiality revealed in the Saccharomyces cerevisiae interaction networks.; citation_author=I Ulitsky; citation_author=R Shamir; citation_journal_title=Mol Syst Biol; citation_volume=3; citation_number=8; citation_pages=104; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=A comprehensive analysis of protein-protein interactions in Saccharomyces cerevisiae.; citation_author=P Uetz; citation_author=L Giot; citation_author=G Cagney; citation_author=T Mansfield; citation_author=R Judson; citation_journal_title=Nature; citation_volume=403; citation_number=9; citation_pages=623-627; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Toward a protein-protein interaction map of the budding yeast: A comprehensive system to examine two-hybrid interactions in all possible combinations between the yeast proteins.; citation_author=T Ito; citation_author=K Tashiro; citation_author=S Muta; citation_author=R Ozawa; citation_author=T Chiba; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=97; citation_number=10; citation_pages=1143-1147; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Transcriptional regulatory code of a eukaryotic genome.; citation_author=C Harbison; citation_author=D Gordon; citation_author=T Lee; citation_author=N Rinaldi; citation_author=K Macisaac; citation_journal_title=Nature; citation_volume=431; citation_number=11; citation_pages=99-104; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Gene ontology: tool for the unification of biology. The Gene Ontology Consortium.; citation_author=M Ashburner; citation_author=C Ball; citation_author=J Blake; citation_author=D Botstein; citation_author=H Butler; citation_journal_title=Nat Genet; citation_volume=25; citation_number=12; citation_pages=25-29; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Cytoplasmic membrane systems: structure, function, and membrane trafficking.; citation_author=G Karp; citation_number=13; citation_pages=292-343; citation_date=1999; citation_publisher=John Wiley and Sons, Inc; " />
      <meta name="citation_reference" content="citation_title=What can yeast tell us about N-linked glycosylation in the Golgi apparatus?; citation_author=S Munro; citation_journal_title=FEBS Lett; citation_volume=498; citation_number=14; citation_pages=223-227; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=An N-acetylglucosaminyltransferase of the Golgi apparatus of the yeast Saccharomyces cerevisiae that can modify N-linked glycans.; citation_author=T Yoko-o; citation_author=C Wiggins; citation_author=J Stolz; citation_author=S Peak-Chew; citation_author=S Munro; citation_journal_title=Glycobiology; citation_volume=13; citation_number=15; citation_pages=581-589; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=An interactional network of genes involved in chitin synthesis in Saccharomyces cerevisiae.; citation_author=G Lesage; citation_author=J Shapiro; citation_author=C Specht; citation_author=A Sdicu; citation_author=P Ménard; citation_journal_title=BMC Genet; citation_volume=6; citation_number=16; citation_pages=8; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Activity of the yeast MNN1 alpha-1,3-mannosyltransferase requires a motif conserved in many other families of glycosyltransferases.; citation_author=C Wiggins; citation_author=S Munro; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=95; citation_number=17; citation_pages=7945-7950; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Chitin synthase III: synthetic lethal mutants and “stress related” chitin synthesis that bypasses the CSD3/CHS6 localization pathway.; citation_author=BC Osmond; citation_author=CA Specht; citation_author=PW Robbins; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=96; citation_number=18; citation_pages=11206-10; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Network motifs: simple building blocks of complex networks.; citation_author=R Milo; citation_author=S Shen-Orr; citation_author=S Itzkovitz; citation_author=N Kashtan; citation_author=D Chklovskii; citation_journal_title=Science; citation_volume=298; citation_number=19; citation_pages=824-827; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=From genomics to chemical genomics: new developments in KEGG.; citation_author=M Kanehisa; citation_author=S Goto; citation_author=M Hattori; citation_author=K Aoki-Kinoshita; citation_author=M Itoh; citation_journal_title=Nucleic Acids Res; citation_volume=34; citation_number=20; citation_pages=D354-357; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Exploration of the function and organization of the yeast early secretory pathway through an epistatic miniarray profile.; citation_author=M Schuldiner; citation_author=S Collins; citation_author=N Thompson; citation_author=V Denic; citation_author=A Bhamidipati; citation_journal_title=Cell; citation_volume=123; citation_number=21; citation_pages=507-519; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Functional characterization of the S. cerevisiae genome by gene deletion and parallel analysis.; citation_author=EA Winzeler; citation_author=DD Shoemaker; citation_author=A Astromoff; citation_author=H Liang; citation_author=K Anderson; citation_journal_title=Science; citation_volume=285; citation_number=22; citation_pages=901-6; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=The synthetic genetic interaction spectrum of essential genes.; citation_author=AP Davierwala; citation_author=J Haynes; citation_author=Z Li; citation_author=RL Brost; citation_author=MD Robinson; citation_journal_title=Nat Genet; citation_volume=37; citation_number=23; citation_pages=1147-52; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Expanded protein information at SGD: new pages and proteome browser.; citation_author=R Nash; citation_author=S Weng; citation_author=B Hitz; citation_author=R Balakrishnan; citation_author=K Christie; citation_journal_title=Nucleic Acids Res; citation_volume=35; citation_number=24; citation_pages=D468-471; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Cytoscape: a software environment for integrated models of biomolecular interaction networks.; citation_author=P Shannon; citation_author=A Markiel; citation_author=O Ozier; citation_author=N Baliga; citation_author=J Wang; citation_journal_title=Genome Res; citation_volume=13; citation_number=25; citation_pages=2498-2504; citation_date=2003; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001922" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Mapping Genetically Compensatory Pathways from Synthetic Lethal Interactions in Yeast"/>
    <meta name="twitter:description" content="BackgroundSynthetic lethal genetic interaction analysis has been successfully applied to predicting the functions of genes and their pathway identities. In the context of synthetic lethal interaction data alone, the global similarity of synthetic lethal interaction patterns between two genes is used to predict gene function. With physical interaction data, such as protein-protein interactions, the enrichment of physical interactions within subsets of genes and the enrichment of synthetic lethal interactions between those subsets of genes are used as an indication of compensatory pathways.ResultIn this paper, we propose a method of mapping genetically compensatory pathways from synthetic lethal interactions. Our method is designed to discover pairs of gene-sets in which synthetic lethal interactions are depleted among the genes in an individual set and where such gene-set pairs are connected by many synthetic lethal interactions. By its nature, our method could select compensatory pathway pairs that buffer the deleterious effect of the failure of either one, without the need of physical interaction data. By focusing on compensatory pathway pairs where genes in each individual pathway have a highly homogenous cellular function, we show that many cellular functions have genetically compensatory properties.ConclusionWe conclude that synthetic lethal interaction data are a powerful source to map genetically compensatory pathways, especially in systems lacking physical interaction information, and that the cellular function network contains abundant compensatory properties."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0001922.g006"/>

  <meta property="og:title" content="Mapping Genetically Compensatory Pathways from Synthetic Lethal Interactions in Yeast" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001922" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=5198'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=1305'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=2128&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0001922">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001922" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001922&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Mapping Genetically Compensatory Pathways from Synthetic Lethal Interactions in Yeast
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Xiaotu Ma
              <span class="corresponding">mail</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:xma@usc.edu">xma@usc.edu</a></p>

                <p>Affiliation:
                  Molecular and Computational Biology Program, Department of Biological Sciences, University of Southern California, Los Angeles, California, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Aaron M. Tarone, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Molecular and Computational Biology Program, Department of Biological Sciences, University of Southern California, Los Angeles, California, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Wenyuan Li
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Molecular and Computational Biology Program, Department of Biological Sciences, University of Southern California, Los Angeles, California, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: April 09, 2008</li>
    <li>DOI: 10.1371/journal.pone.0001922</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0001922" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0001922" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001922" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0001922" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0001922-g001" data-doi="info:doi/10.1371/journal.pone.0001922" data-uri="info:doi/10.1371/journal.pone.0001922.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001922.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001922-g002" data-doi="info:doi/10.1371/journal.pone.0001922" data-uri="info:doi/10.1371/journal.pone.0001922.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001922.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001922-g003" data-doi="info:doi/10.1371/journal.pone.0001922" data-uri="info:doi/10.1371/journal.pone.0001922.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001922.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001922-g004" data-doi="info:doi/10.1371/journal.pone.0001922" data-uri="info:doi/10.1371/journal.pone.0001922.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001922.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001922-g005" data-doi="info:doi/10.1371/journal.pone.0001922" data-uri="info:doi/10.1371/journal.pone.0001922.g005" title="Figure 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001922.g005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001922-g006" data-doi="info:doi/10.1371/journal.pone.0001922" data-uri="info:doi/10.1371/journal.pone.0001922.g006" title="Figure 6">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001922.g006&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001922">Reader Comments (3)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0001922" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2>
<h3>Background</h3>
<a id="article1.front1.article-meta1.abstract1.sec1.p1" name="article1.front1.article-meta1.abstract1.sec1.p1"></a><p>Synthetic lethal genetic interaction analysis has been successfully applied to predicting the functions of genes and their pathway identities. In the context of synthetic lethal interaction data alone, the global similarity of synthetic lethal interaction patterns between two genes is used to predict gene function. With physical interaction data, such as protein-protein interactions, the enrichment of physical interactions within subsets of genes and the enrichment of synthetic lethal interactions between those subsets of genes are used as an indication of compensatory pathways.</p>


<h3>Result</h3>
<a id="article1.front1.article-meta1.abstract1.sec2.p1" name="article1.front1.article-meta1.abstract1.sec2.p1"></a><p>In this paper, we propose a method of mapping genetically compensatory pathways from synthetic lethal interactions. Our method is designed to discover pairs of gene-sets in which synthetic lethal interactions are depleted among the genes in an individual set and where such gene-set pairs are connected by many synthetic lethal interactions. By its nature, our method could select compensatory pathway pairs that buffer the deleterious effect of the failure of either one, without the need of physical interaction data. By focusing on compensatory pathway pairs where genes in each individual pathway have a highly homogenous cellular function, we show that many cellular functions have genetically compensatory properties.</p>


<h3>Conclusion</h3>
<a id="article1.front1.article-meta1.abstract1.sec3.p1" name="article1.front1.article-meta1.abstract1.sec3.p1"></a><p>We conclude that synthetic lethal interaction data are a powerful source to map genetically compensatory pathways, especially in systems lacking physical interaction information, and that the cellular function network contains abundant compensatory properties.</p>

</div>


<div class="articleinfo"><p><strong>Citation: </strong>Ma X, Tarone AM, Li W (2008) Mapping Genetically Compensatory Pathways from Synthetic Lethal Interactions in Yeast. PLoS ONE 3(4):
          e1922.
            doi:10.1371/journal.pone.0001922</p><p><strong>Editor: </strong>Suzannah Rutherford, Fred Hutchinson Cancer Research Center, United States of America</p><p><strong>Received:</strong> November 26, 2007; <strong>Accepted:</strong> February 26, 2008; <strong>Published:</strong> April 9, 2008</p><p><strong>Copyright:</strong> © 2008 Ma et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>This work is partly supported by NIH grant P50 HG 002790.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>Genetic interaction analysis, in which the combined mutations of two genes exhibit phenotypes significantly different from the single mutation of either one <a href="#pone.0001922-Kelley1">[1]</a>, is a powerful tool allowing biologists to investigate the genetic components of an organism <a href="#pone.0001922-Boone1">[2]</a>. The International Yeast Gene Deletion Consortium constructed a nearly complete collection of gene-deletion mutants for yeast <a href="#pone.0001922-Giaever1">[3]</a>, providing an excellent starting point for the study of genetic interactions <a href="#pone.0001922-Tong1">[4]</a>. Synthetic lethal interaction analysis, in which the deletion of two viable genes makes the organism (yeast) inviable, generated the first large-scale synthetic lethal interaction data set <a href="#pone.0001922-Tong2">[5]</a>. The BioGRID database <a href="#pone.0001922-Stark1">[6]</a> contains 9,376 non-redundant yeast synthetic lethal interactions involving 2348 yeast genes (as of August, 2007).</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>All cells must manage biological information to survive. These functions are frequently achieved through various cascades (e.g. physical transport, transcription, translation, phosphorylation, etc.) that involve proteins encoded by different genes in the genome. Due to the involvement of multiple genes in such systems, there is a potential for genetic interactions among members of these pathways and among members of other pathways in the cell that have similar/overlapping functions. Synthetic lethal interactions, where mutations are only lethal in combination, are generally considered to reflect such interactions within and between cascades, with parallel or compensatory pathways explaining most, though not all, synthetic lethal interactions <a href="#pone.0001922-Kelley1">[1]</a>, <a href="#pone.0001922-Ye1">[7]</a>–<a href="#pone.0001922-Ulitsky1">[8]</a>. This view is illustrated in <a href="#pone-0001922-g001">Figure 1A</a>. Due to the existence of alternative pathways for the information flow, disabling any single gene from either pathway will not block the information flow. However, disabling any two components, one from each pathway, will block the information flow and leads to the death of the cell. Note the depletion of interactions among components from the same pathway, which means that disabling any two components from the same pathway will not block the information flow. Shown in <a href="#pone-0001922-g001">Figure 1B</a> is the fact that components from either of these two pathways often have similar patterns of synthetic lethal interactions with other genes from the organism. Note that an important underlying assumption for <a href="#pone-0001922-g001">Figure 1B</a> is that the genes do not have multifunction, i.e., no genes involved have two or more distinct functions (<a href="#pone-0001922-g001">Figure 1C</a>). Finally, a relatively smaller proportion of synthetic lethal interactions are within pathways that can sustain one, but not two, mutations without loss of function (<a href="#pone-0001922-g001">Figure 1D</a>).</p>
<div class="figure" id="pone-0001922-g001"><div class="img"><a name="pone-0001922-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001922.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001922" data-uri="info:doi/10.1371/journal.pone.0001922.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001922.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001922.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001922.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001922.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001922.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001922.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001922.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001922.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001922.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Possible mode of synthetic lethal interactions.</span></strong></p><a id="article1.body1.sec1.fig1.caption1.p1" name="article1.body1.sec1.fig1.caption1.p1"></a><p>Solid circles represent genes, arrows represent information flow, blue lines represent physical interactions and red/pink lines represent synthetic lethal interactions. Dashed circles group genes into pathways. (A–C), Compensatory pathways: (A) The information flow from one master gene (filled circle on the top) can go through either of the two pathways and get to another master gene (filled circle on the bottom). Thus mutation of any single genes in either pathway will not block the information flow, while a double mutation where both pathways are disabled will block the information flow and the cell will die, if this information flow is critical to its survival. However, synthetic lethal interactions are depleted from either pathway since a double deletion in either pathway will not cause lethal effect. (B) Genes involved in the same function generally have similar patterns of synthetic lethal interaction with genes from other pathways. Heavy red lines indicate that all genes grouped by the large dashed circle have synthetic lethal interaction with the other genes. (C) Multifunction effect. A gene (or a pathway) may have multiple functions. All single-function genes (empty circles) grouped by the dashed circle tend to have similar synthetic lethal interaction patterns. However, a multifunction gene, as indicated by the pink circle, may have a synthetic lethal interaction pattern (the additional synthetic lethal interactions shown with pink lines) that differs from the other genes grouped by the dashed circle. (D) Single pathway: A certain protein complex/pathway can sustain the mutation of one, but not two, member genes without loss of function.</p>
<span>doi:10.1371/journal.pone.0001922.g001</span></div><a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>The fact that the compensatory pathway structure can be reflected by synthetic lethal analysis is widely appreciated. For example, Ye et al <a href="#pone.0001922-Ye1">[7]</a> used a congruence score, which measures global similarity between the patterns of synthetic lethal interactions of two genes, to predict that two genes with high congruence scores were likely in the same pathway and thus share similar functional roles. However, the underlying compensatory structure between pathways (<a href="#pone-0001922-g001">Figure 1A</a>) could not be revealed by this method. Kelley and Ideker <a href="#pone.0001922-Kelley1">[1]</a> proposed a between-pathway model to score the compensatory relationship between pathways in a probabilistic framework. Their method emphasized the enrichment of synthetic lethal interactions connecting two pathways. An equally important component in their work was the enrichment of physical interactions within each pathway, including protein-protein interactions <a href="#pone.0001922-Uetz1">[9]</a>–<a href="#pone.0001922-Ito1">[10]</a> and protein-DNA interactions <a href="#pone.0001922-Harbison1">[11]</a>. Ulitsky and Shamir <a href="#pone.0001922-Ulitsky1">[8]</a> extended this idea by requiring that each pathway in a compensatory pair was a connected graph in the physical interaction network rather than enriched with physical interactions. They found that twice as many genetic interaction pairs can be assigned to compensatory pathways. Although the physical interactions within each pathway increased the confidence that the resulting pathways are biologically meaningful, some pathway pairs were sacrificed due to the lack of support of physical interactions, owing to the fact that current physical interaction data contain false positives and false negatives. More importantly, their frameworks did not impose any constraint on synthetic lethal interactions within each pathway. A high-scoring compensatory pathway pair may contain a pathway that has many within-pathway synthetic lethal interactions, which is difficult to explain with the current model (<a href="#pone-0001922-g001">Figure 1A</a>). In addition, in some species there is only genetic screening data available, making it worthwhile to study the power of mapping genetically compensatory pathways using only synthetic lethal interactions.</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p>To account for the above theoretical considerations, we developed a graph theory method to group genes into distinct but compensatory pathway pairs, based solely on the local connectivity structure of the synthetic lethal interaction network. Pathways in each pair were required to be connected by many synthetic lethal interactions and depleted for synthetic lethal interactions within each pathway. A heuristic algorithm was proposed to realize our method. This approach had considerable power in grouping genes into functionally homogenous sets and identified many cellular functions exhibiting genetically compensatory properties. However, we must make it clear that our method was not developed to identify pathways like <a href="#pone-0001922-g001">Figure 1D</a>, which can be identified using the “within-pathway model” of Kelly and Ideker <a href="#pone.0001922-Kelley1">[1]</a>.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3>
<h4>Pathway definition and identification</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>All genes in one organism interact, forming a gene-network. Given the fact that some genes specifically cooperate to take over some highly specialized cellular functions, pathways are often used to represent part of the gene-network at a more detailed level. An example compensatory pathway pair is shown in <a href="#pone-0001922-g001">Figure 1A</a>, where the three genes shown in left and the two genes shown in right can be defined as two distinct pathways. However, it is also appropriate to group all the five genes (or plus the two master genes) into one pathway. Both definitions make sense because the first definition focuses on the functional redundancy of the two smaller pathways while the second definition focuses on the fact that all these genes are involved in one specific cellular function. Since the synthetic lethal interactions are in general considered to reveal the functional redundancy between pathways <a href="#pone.0001922-Kelley1">[1]</a>, <a href="#pone.0001922-Ye1">[7]</a>–<a href="#pone.0001922-Ulitsky1">[8]</a>, the pathways we pursued in this paper followed the first definition. However, the pathways we identified do not necessarily adhere to this meaning; rather, functionally distinct genes may be found to be compensatory to a multifunction pathway since we did not use the physical interaction data. This fact is discussed further in section “Multifunction effect of pathways”.</p>
<a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2"></a><p>Shown in <a href="#pone-0001922-g002">Figure 2</a> are several genetically redundant pathway pairs identified by our method. For example, the <em>dynein-dynactin pathway</em> (shown in boxes, P&lt;1e-12, Hypergeometric test, herein and after, <a href="#pone-0001922-g002">Figure 2A</a> left) and <em>protein depolymerization pathway</em> (diamonds, P&lt;1e-6, <a href="#pone-0001922-g002">Figure 2A</a> right) were found to be genetically compensatory. Genetically compensatory pathway pairs also shown in <a href="#pone-0001922-g002">Figure 2</a> include <em>histone deacetylation</em> (diamonds, P&lt;1e-6) and <em>histone methylation</em> (boxes, P&lt;1e-6, <a href="#pone-0001922-g002">Figure 2B</a>), <em>actin filament-based process</em> (boxes, P&lt;1e-8) and <em>chitin metabolic process</em> (diamonds, P&lt;1e-11, <a href="#pone-0001922-g002">Figure 2C</a>), <em>positive regulation of RNA elongation</em> (diamonds, P&lt;1e-8) and <em>double strand break via single strand annealing</em> (boxes, P&lt;1e-14, <a href="#pone-0001922-g002">Figure 2D</a>), <em>tubulin folding</em> (diamonds, P&lt;1e-12) and <em>spindle checkpoint</em> (boxes, P&lt;1e-8, <a href="#pone-0001922-g002">Figure 2E</a>) and <em>Golgi to membrane protein transport</em> (boxes, P&lt;1e-5) and <em>endosome to Golgi retrograde transport</em> (diamonds, P&lt;1e-9, <a href="#pone-0001922-g002">Figure 2F</a>). Known physical interactions that were listed in Ulitsky and Shamir (2005), including protein-protein and protein-DNA interactions, were also indicated in <a href="#pone-0001922-g002">Figure 2</a> using heavy blue lines.</p>
<div class="figure" id="pone-0001922-g002"><div class="img"><a name="pone-0001922-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001922.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001922" data-uri="info:doi/10.1371/journal.pone.0001922.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001922.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001922.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001922.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001922.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001922.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001922.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001922.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001922.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001922.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Example pathway pairs identified by our method.</span></strong></p><a id="article1.body1.sec2.sec1.fig1.caption1.p1" name="article1.body1.sec2.sec1.fig1.caption1.p1"></a><p>Each example is shown in a pair: left: pathway 1, right: pathway 2. Heavy blue lines represent physical interactions (protein-protein interaction and/or protein-DNA interactions) between genes and red lines represent synthetic lethal interactions. Diamonds and Boxes represent genes with the dominant function for pathway 1 and pathway 2, respectively (see text for details). (A) The dynein-dynactin pathway and protein depolymerization. (B) Histone deacetylation and histone methylation. (C) Actin filament-based process and chitin metabolic process. (D) Positive regulation of RNA elongation and double-strand break repair via single-strand annealing. (E) Tubulin folding and spindle checkpoint. (F) Golgi to plasma membrane protein transport and retrograde transport, endosome to Golgi.</p>
<span>doi:10.1371/journal.pone.0001922.g002</span></div><a id="article1.body1.sec2.sec1.p3" name="article1.body1.sec2.sec1.p3"></a><p>By removing physical interaction data from the process of identifying genetically compensatory pathways, there was also a greater potential for spurious identifications (discussed later). Though the approach presented here is theoretically sound, predictions should make sense in the face of known physical interactions within and between pathways. Here we have detailed an example to demonstrate the utility of identifying compensatory pathways solely via genetic interaction information.</p>
<a id="article1.body1.sec2.sec1.p4" name="article1.body1.sec2.sec1.p4"></a><p>In <a href="#pone-0001922-g002">Figure 2C</a>, the functions of an actin filament-based process and chitin metabolism were identified as compensatory processes in this analysis. Though these functions seemed superficially disparate based on Gene Ontology <a href="#pone.0001922-Ashburner1">[12]</a>, the potential mechanism for their interaction became apparent when the loci involved were investigated. The actin filament-based process genes are involved in various aspects of endocytosis, which require the movement of vesicles between the trans-golgi network and the cell membrane via the actin cytoskeleton (<a href="http://www.yeastgenome.org">www.yeastgenome.org</a>). The movement of vesicles within a cell depends in part on polysaccharide tags attached to the membranes of the structures <a href="#pone.0001922-Karp1">[13]</a>–<a href="#pone.0001922-Yokoo1">[15]</a>. One of these tags is mannan, a polymer of mannose that is important for vesicle transport, cell wall structure, and other functions dependent on protein glycosylation <a href="#pone.0001922-Munro1">[14]</a>.</p>
<a id="article1.body1.sec2.sec1.p5" name="article1.body1.sec2.sec1.p5"></a><p>The genes in the identified actin filament-based process pathway encode proteins for actin/myosin binding that are known to affect secretion polarity within the cell, as well as a mannan polymerase (VAN1) and a mannosyltransferase (MNN10) <a href="#pone.0001922-Lesage1">[16]</a>. These mannose-processing proteins have been shown to be part of golgi-bound complexes composed of either VAN1/MNN9, or MNN10/ANP1/HOC1/MNN9/MNN11, both of which are involved in protein mannosylation <a href="#pone.0001922-Munro1">[14]</a>. This information provides a connection to the compensatory pathway involved in chitin synthesis, as HOC1 also bears a physical similarity to several other seemingly unrelated glycosyltransferases MNN1, OCH1, SUR1, and CHS1 (a chitin synthase) proteins, indicating the presence of necessary functional components <a href="#pone.0001922-Wiggins1">[17]</a> in proteins of both pathways, and revealing the possibility of redundant functions performed by those proteins. Since chitin is a polymer of N-acetylglucosamine, which bears structural similarity to mannose, and is also a critical component of the cell wall, it is not surprising that a protein or complex could be involved in the transport or polymerization of both molecules. N-acetylglucosamine is used to link mannan to yeast proteins <a href="#pone.0001922-Munro1">[14]</a>–<a href="#pone.0001922-Yokoo1">[15]</a>, indicating the necessity of the mannan producing complex to recognize and bind sugars to the molecule that is the building block for chitin. It is also possible that the molecular outputs of both complexes (mannan and chitin) are compensatory. Indeed, evidence of overlap between chitin and mannan glycosylation exists as mutations in the MNN10 complex alter chitin levels in yeast cells <a href="#pone.0001922-Lesage1">[16]</a>, <a href="#pone.0001922-Osmond1">[18]</a>. This indicates that either the complexes themselves are partially redundant, or that cell wall stress resulting from missing cell wall mannan can be relieved by the increased production of chitin. Given these similarities in sugar and protein structures and functions between mannan and chitin-based biological processes, it was not surprising (and expected) that they were identified as compensatory processes in this analysis.</p>


<h4>Summary statistics of compensatory pathway pairs</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>Our search generated 2,590 pathway pairs (7.7±3.1 genes per pair), which cover 5,284 (56% of 9,376) synthetic lethal interaction pairs involving 689 yeast genes. Since we did not require the physical interaction to support our pathway models, it was expected that the pathways we identified to be larger than that identified by integrating genetic and physical interaction data <a href="#pone.0001922-Kelley1">[1]</a>, <a href="#pone.0001922-Ulitsky1">[8]</a>. To test this hypothesis, we compared the sizes of the pathways from different methods. As shown in <a href="#pone-0001922-g003">Figure 3</a> upper panel, the pathways (redundancy removed, see <a href="#s4">Methods</a>) identified by our method were significantly (P&lt;1e-16, Wilcox rank sum test) larger than that of Kelley and Ideker <a href="#pone.0001922-Kelley1">[1]</a>, whereas the sizes of the pathways in Kelley and Ideker <a href="#pone.0001922-Kelley1">[1]</a> was significantly (P&lt;1e-6, Wilcox rank sum test) larger than that of Ulitsky and Shamir <a href="#pone.0001922-Ulitsky1">[8]</a>. In addition, our investigation suggested that this result is not an artifact of parameters specific to our method (<a href="#pone.0001922.s001">Figure S1</a> and <a href="#pone.0001922.s005">Text S1</a>). A close inspection revealed that many pathways (40%; 110/280) identified by Ulitsky and Shamir <a href="#pone.0001922-Ulitsky1">[8]</a> had one pathway member of size 2 (however, we also noted that Ulitsky and Shamir <a href="#pone.0001922-Ulitsky1">[8]</a> had identified 5 pathways with more than 50 genes and the largest one had 201 genes). This result implied that we can potentially increase the size of gene sets at the cost of losing the physical interaction support. However, the pathways we identified can sometimes be smaller than the congruence score method by Ye et al <a href="#pone.0001922-Ye1">[7]</a>. For example, BIK1 was absent from the dynein-dynactin checkpoint pathway, as shown in <a href="#pone-0001922-g002">Figure 2A</a>, and it was identified by the congruence score method. It should be noted that pathway size distribution is not a measure of accuracy of our method. In fact, the size of a pathway could be very arbitrary if we take a gene-network perspective, where a pathway is used to highlight a functionally homogenous sub-network (see section “Pathway definition and identification”). However, we noted that in functional genomics a practical problem is that many genes are not functionally annotated, thus it may be helpful to provide biologists reasonably larger lists of genes showing compensatory interaction patterns. In fact, the example in <a href="#pone-0001922-g002">Figure 2C</a> discussed previously suggests the value of providing a larger pathway.</p>
<div class="figure" id="pone-0001922-g003"><div class="img"><a name="pone-0001922-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001922.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001922" data-uri="info:doi/10.1371/journal.pone.0001922.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001922.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001922.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001922.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001922.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001922.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001922.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001922.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001922.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001922.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Summary statistics of the pathway pairs we identified.</span></strong></p><a id="article1.body1.sec2.sec2.fig1.caption1.p1" name="article1.body1.sec2.sec2.fig1.caption1.p1"></a><p>Upper panel: Distribution of the sizes of the pathways of our analysis (GI only; box) and that of Ulitsky and Shamir (<a href="#pone.0001922-Ulitsky1">[8]</a>; circle) and Kelley and Ideker (<a href="#pone.0001922-Kelley1">[1]</a>; star). Bottom panel: The distribution of the completeness of the between-pathway synthetic lethal interactions in our identified pathway pairs.</p>
<span>doi:10.1371/journal.pone.0001922.g003</span></div><a id="article1.body1.sec2.sec2.p2" name="article1.body1.sec2.sec2.p2"></a><p>We next asked how the synthetic lethal interactions were depleted from either pathway of each pathway pair and how the synthetic lethal interactions were enriched between pathways in each pair. Since we imposed a stringent constraint on the within-pathway synthetic lethal interactions (α = 0.01; see <a href="#s4">Method</a>), we found that only one of the pathways we identified have within-pathway synthetic lethal interactions. Thus we proceeded to determine how the synthetic lethal interactions were enriched between pathways (defined as completeness <em>d<sub>12</sub></em>, which represents the proportion of all potential synthetic lethal interactions that have been observed; see Equation (1)). According to the assumptions in <a href="#pone-0001922-g001">Figure 1A</a>, if all possible pairs of genes between the two compensatory pathways are connected by synthetic lethal interactions, they have a completeness of 100%. In practice, two compensatory pathways may have a completeness less than 100%, due to either lack of experiments or biological reasons. As can be seen from <a href="#pone-0001922-g003">Figure 3</a> bottom panel, the pathway pairs we identified had a typical completeness of between-pathway synthetic lethal interactions at the range of [0.75, 0.85], which may have indication to the false-negative rate of the synthetic lethal interaction data (<a href="#pone.0001922.s006">Text S2</a>). Noticeably, some pathway pairs we identified had completeness <em>d<sub>12</sub></em> = 100%.</p>
<a id="article1.body1.sec2.sec2.p3" name="article1.body1.sec2.sec2.p3"></a><p>We also determined the statistical significance of the pathway pairs we identified. To achieve this goal, we generated 10,000 random networks by crossing pairs of edges as done in Kelley and Ideker <a href="#pone.0001922-Kelley1">[1]</a> and Milo et al <a href="#pone.0001922-Milo1">[19]</a>. For each pathway pair we identified (with completeness <em>d<sub>1</sub></em>, <em>d<sub>2</sub></em> and <em>d<sub>12</sub></em>; see Equation (1)), we counted the chances of observing this pathway pair to have <em>d</em><sub>1</sub><sup>0</sup>≤<em>d</em><sub>1</sub>, <em>d</em><sub>2</sub><sup>0</sup>≤<em>d</em><sub>2</sub> and <em>d</em><sub>12</sub><sup>0</sup>≥<em>d</em><sub>12</sub>, where the superscript 0 means the score was under permutation. As it turned out, all pathway pairs we identified had P-value less than 1e-4. This result suggests that the constraint β = 0.75 of Equation (1) was rather strict. In fact, we found from our permutation study that given <em>d</em><sub>1</sub><sup>0</sup>≤0.01and <em>d</em><sub>2</sub><sup>0</sup>≤0.01, <em>d</em><sub>12</sub><sup>0</sup> had a mean of 0.09 and a standard deviation of 0.06 (<a href="#pone.0001922.s002">Figure S2</a> and <a href="#pone.0001922.s006">Text S2</a>), which further suggests that our threshold β = 0.75 of Equation (1) was very strict.</p>


<h4>Physical interactions in the discovered pathways</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>Biologically, we expected the enrichment of physical interactions within pathways we identified, which was an important component in Kelley and Ideker <a href="#pone.0001922-Kelley1">[1]</a> and Ulitsky and Shamir <a href="#pone.0001922-Ulitsky1">[8]</a>. To study the enrichment of physical interactions within the pathways identified by our method, we downloaded the physical interaction data collected by Ulitsky and Shamir <a href="#pone.0001922-Ulitsky1">[8]</a>, which included 67,856 physical interaction pairs. For each pathway pair identified, we calculated the completeness scores <em>d<sub>1</sub></em>, <em>d<sub>2</sub></em>, <em>d<sub>12</sub></em> as defined in Equation (1) using the physical interaction data to measure the enrichment of physical interactions within pathway 1, within pathway 2, and between pathway 1 and pathway 2, respectively. Similar to the idea of Kelly and Ideker <a href="#pone.0001922-Kelley1">[1]</a>, we expected completeness scores <em>d<sub>1</sub></em> and <em>d<sub>2</sub></em> to be large. On the other hand, we hypothesized that physical interactions will be less enriched between compensatory pathway pairs, i.e., <em>d<sub>12</sub></em> to be relatively small, according to <a href="#pone-0001922-g001">Figure 1A</a>. The distribution of completeness was shown in <a href="#pone-0001922-g004">Figure 4</a>. As can be seen, there were significantly (P-value&lt;10<sup>−16</sup>, Wilcox rank sum test, one-sided) more within-pathway physical interactions than between-pathway physical interactions, suggesting that our identified pathway pairs do have biological meanings.</p>
<div class="figure" id="pone-0001922-g004"><div class="img"><a name="pone-0001922-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001922.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001922" data-uri="info:doi/10.1371/journal.pone.0001922.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001922.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001922.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001922.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001922.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001922.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001922.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001922.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001922.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001922.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>Physical interactions are enriched in pathways we identified and depleted from between-pathway gene pairs.</span></strong></p><span>doi:10.1371/journal.pone.0001922.g004</span></div>

<h4>Functional homogeneity of discovered pathways</h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>We next sought to study how different cellular functions were enriched in the pathways we identified. The functional annotation from Gene Ontology <a href="#pone.0001922-Ashburner1">[12]</a> was used for the evaluation. For each pathway we identified, a hypergeometric distribution was used to test the enrichment of genes annotated with a specific Gene Ontology concept. Since there were 1720 Gene Ontology biological process concepts being screened (see <a href="#s4">Methods</a>), the −log10 of the smallest P-value of all the 1720 P-values (not corrected for multiple testing) was defined as the functional homogeneity score of our pathways. Thus, a higher homogeneity score implied a better grouping of genes into pathways. To provide a comparison, we also ran the same enrichment test on the pathways (redundancy removed, see <a href="#s4">Methods</a>) identified by Kelley and Ideker <a href="#pone.0001922-Kelley1">[1]</a> and Ulitsky and Shamir <a href="#pone.0001922-Ulitsky1">[8]</a>. As expected, the pathways identified by our method (<a href="#pone-0001922-g005">Figure 5</a>, upper panel) had lower functional homogeneity than those identified with the aid of physical interaction data (<a href="#pone-0001922-g005">Figure 5</a>, bottom panel, stars and circles).</p>
<div class="figure" id="pone-0001922-g005"><div class="img"><a name="pone-0001922-g005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001922.g005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001922" data-uri="info:doi/10.1371/journal.pone.0001922.g005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001922.g005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001922.g005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001922.g005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001922.g005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001922.g005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001922.g005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001922.g005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001922.g005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001922.g005.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 5.  <span>The distribution of the number of identified pathways as a function of the functional homogeneity.</span></strong></p><a id="article1.body1.sec2.sec4.fig1.caption1.p1" name="article1.body1.sec2.sec4.fig1.caption1.p1"></a><p>Upper panel: the functional homogeneity of all the pathways from each pathway pair identified by our method. Although a considerable percentage of pathways identified by our method had high homogeneity scores, many of them showed a low functional homogeneity due to multifunction effects. Bottom panel: at least one of the two pathways of each pathway pair identified by our method had high homogeneity. Note that the functional homogeneity score was still significantly lower than that of the pathways identified using physical interaction data.</p>
<span>doi:10.1371/journal.pone.0001922.g005</span></div><a id="article1.body1.sec2.sec4.p2" name="article1.body1.sec2.sec4.p2"></a><p>However, an interesting observation was that in each pathway pair identified by our method, at least one of the two pathways had high functional homogeneity. To show this fact, we took the pathway with a smaller P-value from each pathway pair identified by our method and drew the distribution of their functional homogeneity (redundancy removed, see <a href="#s4">Methods</a>). As shown in <a href="#pone-0001922-g005">Figure 5</a> bottom panel (boxes), these pathways showed enhanced functional homogeneity compared to <a href="#pone-0001922-g005">Figure 5</a> upper panel (although still lower than that identified using physical interaction support; <a href="#pone-0001922-g005">Figure 5</a>, bottom panel, stars and circles). This phenomenon was actually a result of the multifunction effects of the pathways. To explain, suppose a pathway A has compensatory effects with pathways B and C. Then with a high probability our method will group genes from pathway A into one set and genes from pathways B and C into another set, resulting in one genuine pathway corresponding to A with a high functional homogeneity score and another spurious pathway corresponding to B and C, with a low functional homogeneity score. Thus, at the cost of reducing the accuracy of one of the two pathways in each identified pathway pair, it was still possible to gain much insight of the gene functions using synthetic lethal interaction data alone.</p>


<h4>Network of compensatory biological functions</h4>
<a id="article1.body1.sec2.sec5.p1" name="article1.body1.sec2.sec5.p1"></a><p>The above result showed that our method can group genes into pathway pairs, and in most cases at least one pathway from each pathway pair had specific biological functions. Since synthetic lethal interaction data often predict functionally compensatory pathways <a href="#pone.0001922-Kelley1">[1]</a>, <a href="#pone.0001922-Ye1">[7]</a>–<a href="#pone.0001922-Ulitsky1">[8]</a>, we next determined how different cellular functions compensated each other. To make the analysis strict, we selected the pathways pairs where each member pathway had a functional homogeneity score larger than 5.24 (or P-value 5.8e-6, which corresponds to the bonferroni-corrected P-value 0.01) and more than 30% of the member genes in each pathway had the same function as the one annotated to the pathway. This requirement resulted in 89 non-redundant pairs of Gene Ontology concepts connected by extensive synthetic lethal interactions (<a href="#pone.0001922.s009">Table S1</a>). As shown in <a href="#pone-0001922-g006">Figure 6</a>, the Gene Ontology concepts could be arranged in a network. For example, the genes involved in <em>Golgi to plasma membrane protein transport</em> were found to be synthetic lethal with the genes involved in <em>intra-Golgi vesicle-mediated transport</em>; <em>retrograde transport, endosome to Golgi</em>; <em>and vesicle-mediated transport</em>.</p>
<div class="figure" id="pone-0001922-g006"><div class="img"><a name="pone-0001922-g006" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001922.g006&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001922" data-uri="info:doi/10.1371/journal.pone.0001922.g006"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001922.g006&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001922.g006/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001922.g006/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001922.g006/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001922.g006/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001922.g006.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001922.g006/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001922.g006/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001922.g006.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 6.  <span>Network of compensatory biological functions revealed by synthetic lethal interaction data.</span></strong></p><a id="article1.body1.sec2.sec5.fig1.caption1.p1" name="article1.body1.sec2.sec5.fig1.caption1.p1"></a><p>Each node represents a Gene Ontology concept. Each edge represents the existence of extensive synthetic lethal interactions between gene-pairs from the two connected Gene Ontology concept pairs.</p>
<span>doi:10.1371/journal.pone.0001922.g006</span></div><a id="article1.body1.sec2.sec5.p2" name="article1.body1.sec2.sec5.p2"></a><p>Interestingly, the obtained network showed considerable un-evenness toward several Gene Ontology concepts, such as <em>tubulin folding</em> and <em>histone exchange</em>. For example, the <em>histone exchange</em> was found to be connected to many other functions, such as <em>transcription</em>, <em>DNA dependent</em>; <em>negative regulation of meiosis</em>; <em>chromatin silencing</em>; <em>tubulin folding</em>; etc. Another hub node was the <em>positive regulation of RNA elongation</em>, which was also connected to many functions, mostly involving <em>DNA repair</em> biology. The underlying biological implications await additional exploration.</p>


<h4>Multifunction effects of pathways</h4>
<a id="article1.body1.sec2.sec6.p1" name="article1.body1.sec2.sec6.p1"></a><p>Since we deliberately excluded the physical interaction data when searching for compensatory pathway pairs (see <a href="#s4">Methods</a>), it was not surprising that sometimes we discovered pathways containing members that obviously deviated from the most common function of these pathways (<a href="#pone-0001922-g002">Figure 2</a>). At least three possibilities can explain this phenomenon. First, false positives might exist in the synthetic lethal interaction data set. Thus, an obviously unrelated gene may be incorporated into one pathway. However, the current synthetic lethal interaction data is believed to contain only few false positives <a href="#pone.0001922-Tong2">[5]</a>. Thus this explanation is unlikely true. Second, the “strange member” might be a new member of that cellular function. In other words, the Gene Ontology data contains false positives and false negatives. The existence of physical interactions between this new gene and other genes known to belong to that function will increase the probability of this explanation. Third, it might be simply due to multifunction (<a href="#pone-0001922-g001">Figure 1C</a>). In this case, the multiple functions contained in one identified pathway may suggest that its collaborator pathway participates in more than one cellular function.</p>
<a id="article1.body1.sec2.sec6.p2" name="article1.body1.sec2.sec6.p2"></a><p>The third consideration was identified in this analysis. In <a href="#pone-0001922-g002">Figure 2E</a>, the prefoldin complex (GIM3, GIM4, GIM5, YKE2, PAC10) was found to be compensatory to the spindle checkpoint complex (BUB1, BUB2, BUB3, MAD1, MAD2, BMH1). However, the VID21 gene was found to be grouped with prefoldin complex, which had synthetic lethal interaction with spindle checkpoint complex except BMH1. VID21 is a component of the NuA4 histone acetyltransferase complex. Thus, this fact suggested that the spindle checkpoint complex may also participate in a function in collaboration with histone modification. In this sense, we determined that the spindle check point was a multifunction pathway.</p>
<a id="article1.body1.sec2.sec6.p3" name="article1.body1.sec2.sec6.p3"></a><p>The multifunction effect of pathways is better illustrated using a network representation in <a href="#pone-0001922-g006">Figure 6</a>. The fact that a pathway with a given function may be connected to many other pathways, each with a distinct function, suggests the multifunction effects of a pathway. In this sense, the “hub” pathways such as those involved in positive regulation of RNA elongation and histone exchange are good examples of pathways with multifunction effects.</p>

</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p>High throughput bio-techniques are generating more and more extensive descriptions of the gene networks. Unlike physical interaction data, which is conceptually straightforward, the synthetic lethal interaction data implies structural properties of the gene network at a higher level. Namely, synthetic lethal interactions often imply compensatory pathway structures, while physical interactions suggest that the participating genes reside in the same pathway. Thus, integrating synthetic lethal interaction and physical interaction data is an efficient way to gain biological insights from the network data. In particular, enrichment of physical interactions such as protein-protein and protein-DNA interactions within a set of genes increases our confidence that the gene set is biologically meaningful. For example, the work of Kelley and Ideker <a href="#pone.0001922-Kelley1">[1]</a> and Ulitsky and Shamir <a href="#pone.0001922-Ulitsky1">[8]</a> revealed a great number of pathways supported by physical interactions. However, the size of the pathways identified by them could be very small, sometimes with only two genes. We postulate the reason is that the current physical interaction data is far from complete and therefore it will be difficult to reconstruct most pathways. Although the size distribution of pathways is still an unknown fact, it is likely that many pathways consist of more than two genes. In fact, our example in <a href="#pone-0001922-g002">Figure 2C</a> demonstrates that the small pathway size can be a consequence of lacking physical interaction support. We stress that in functional genomics a practical problem is that many genes are not functionally annotated, thus it may be helpful to provide biologists larger list of candidate genes showing compensatory interaction patterns. More importantly, given that in some model organisms only genetic screening data is available, our findings suggest that we can obtain substantial biological insights about genetically redundant pathways without inquiring the physical interaction data.</p>
<a id="article1.body1.sec3.p2" name="article1.body1.sec3.p2"></a><p>The mannan/transport compensation of chitin synthesis identified here exemplifies some important features that should be considered when using genetic interactions alone to predict compensatory pathways. One potential weakness in predicting compensatory interactions is that there is a minor subset of physically interacting genes that are also synthetically lethal. In the analysis presented here HOC1 was identified as a chitin synthase member, likely due to within-pathway redundancy, while MNN10 and VAN1 were correctly identified as mannan synthesis genes as a result of their genetic interactions with chitin synthesis genes. However, published data on mannan synthesis <a href="#pone.0001922-Munro1">[14]</a> allowed for a quick interpretation of the HOC1 interaction with VAN1 as a probable within-function interaction.</p>
<a id="article1.body1.sec3.p3" name="article1.body1.sec3.p3"></a><p>Another feature of this analysis is that one pathway is often compensated by multiple pathways, as exemplified by cytoskeletal components (necessary for moving complexes through the cell) and the two mannan polymerase complexes that were both identified as belonging to the same group of proteins compensating chitin synthesis. Ulitsky and Shamir identified a CHS3/BIN4/SKT5 complex compensation by both MY02/SHE4 and VRP1/SLA1 separately (BMP-48 and -130, respectively; <a href="#pone.0001922-Ulitsky1">[8]</a>). In predicting a larger set of interactions, this analysis yielded a more complete picture of chitin compensating functions than previous analyses with these data that included physical interactions. A result of such fragmentation presents the possibility that pieces of the whole interaction will be missed due to human error and indicates the requirement of more post analysis evaluation of the data to achieve the same big-picture view provided with the approach presented here. Given that there are also errors in the protein interaction data that must be considered when vetting the output of analyses based on genetic and protein interactions, the production of a larger list of interactions in one cluster is likely to save researchers time and error at the end of analysis.</p>
<a id="article1.body1.sec3.p4" name="article1.body1.sec3.p4"></a><p>Additionally, from a biological perspective, it is one thing to singularly know either that chitin synthesis mutations can be compensated by a mannan polymerase or by a few secretion polarity molecules; and a completely different thing to see that there are several secretion polarity mechanisms and two mannan polymerase complexes that interact with chitin synthesis mutants. A partitioned view of the interactions among groups of genes (or a big-picture created from a poorly reconstructed set of smaller interactions) limits the types of questions that can be asked of the analyses (i.e. “How many cellular systems affect chitin synthesis?”), whereas, simpler questions (such as “Does mannan synthesis compliment chitin synthesis?”) are unaffected by having a fuller picture of how genes interact, as subsets of the output can be ignored. Moreover, the interactions provided by assessing the data used with this method have been demonstrated to a much greater degree in subsequent analyses of chitin synthesis synthetic lethal analyses <a href="#pone.0001922-Lesage1">[16]</a>, indicating that the expanded list of interactions was also more appropriate in this case.</p>
<a id="article1.body1.sec3.p5" name="article1.body1.sec3.p5"></a><p>Several parameters in this analysis require optimization. It is necessary to determine optimal values for pathway size, completeness, and how the sizes of each pathway in a compensatory pathway pair are balanced. However, several practical considerations make this study challenging. First, the current synthetic lethal interaction network is heavily biased. Due to the formidable number of possible combinations (~6000×6000/2) and the sparse nature of synthetic lethal interactions <a href="#pone.0001922-Tong2">[5]</a>, the query genes are generally selected carefully, despite the fact that all genes are used as bait. Thus, any study of the global characteristics of the gene network should take special care with this sampling bias. Second, the evaluation standard of pathways, such as the Gene Ontology, is a fast evolving standard. Other well studied metabolic pathways, such as that documented in the KEGG database <a href="#pone.0001922-Kanehisa1">[20]</a>, have few, if any, overlap with the current query genes. The lack of an extensive definition of pathways focusing on the current query genes makes it hard to accurately evaluate parameter estimations.</p>
<a id="article1.body1.sec3.p6" name="article1.body1.sec3.p6"></a><p>In addition to the binary synthetic lethal interactions, quantitative genetic epistasis data <a href="#pone.0001922-Schuldiner1">[21]</a> can be another rich source of genetic interactions. Epistasis refers to the phenomenon in which the phenotypic consequence of altering one gene is differentially modulated by the specific alleles of a second one, including both negative (aggravating) interactions and buffering (relieving) interactions <a href="#pone.0001922-Schuldiner1">[21]</a>. Synthetic lethal interaction is an extreme of the negative interactions. It has been demonstrated that the quantitative epistasis data contains rich information regarding pathways and protein complexes <a href="#pone.0001922-Schuldiner1">[21]</a>. More importantly, due to the existence of essential genes <a href="#pone.0001922-Winzeler1">[22]</a>, analyses focusing on synthetic lethal interactions are not fully genome-wide and thus not really systematic. With the advance of biotechnology, such as <em>d</em>ecreased <em>a</em>bundance by <em>m</em>RNA <em>p</em>erturbation (DAmP) <a href="#pone.0001922-Schuldiner1">[21]</a> and promoter-replacement techniques <a href="#pone.0001922-Davierwala1">[23]</a>, genetic interaction data are being generated in a systematic fashion. Future work will be devoted to studying this broader network of interactions.</p>
</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4>Data</h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p>We downloaded the synthetic lethal interaction data from BIOGRID (<a href="#pone.0001922-Stark1">[6]</a>; version 2.0.31). It contains 9376 non-redundant synthetic lethal genetic interactions, involving 2348 yeast genes. Note that in this paper we focused on the synthetic lethal genetic interactions. Other genetic interactions, such as 7,233 synthetic growth defect and synthetic rescue interactions were not included.</p>
<a id="article1.body1.sec4.sec1.p2" name="article1.body1.sec4.sec1.p2"></a><p>The 68172 protein-protein and protein-DNA interaction data covering 6814 yeast genes was obtained from Ulitsky and Shamir <a href="#pone.0001922-Ulitsky1">[8]</a>. The functional annotation data of yeast genes is downloaded from SGD <a href="#pone.0001922-Nash1">[24]</a>, as of August 2007. We parsed the Gene Ontology <a href="#pone.0001922-Ashburner1">[12]</a> data structure and mapped yeast genes to all ontology nodes and the resulting 1720 biological process nodes, each with no more than 500 genes, are selected to perform functional analysis.</p>


<h4>Network visualization</h4>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1"></a><p>Network figures were created using <em>Cytoscape</em> <a href="#pone.0001922-Shannon1">[25]</a>.</p>


<h4>Algorithm</h4>
<a id="article1.body1.sec4.sec3.p1" name="article1.body1.sec4.sec3.p1"></a><p>By viewing genes as nodes and synthetic lethal interactions between genes as edges, the synthetic lethal interaction data can be represented as a network. Our goal was to identify approximately complete bipartite graphs within the synthetic lethal interaction network which satisfied the following criteria: 1) there were no or few edges within each sub-network; 2) there was an abundance of edges connecting the sub-network pairs; 3) the sub-networks contained at least four genes. Mathematically, we denote the original network as <em>G</em>(<em>V</em>, <em>E</em>), where <em>V</em> is the set of yeast genes and <em>E</em> is the set of edges connecting yeast genes: each element <em>e<sub>ij</sub></em> in <em>E</em> takes value 1 if there is synthetic lethal interaction between genes <em>i</em> and <em>j</em> and 0 otherwise. Our goal was to find all node set pairs (<em>V1</em>, <em>V2</em>) that satisfied<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001922.e001&amp;representation=PNG" class="inline-graphic"><span class="note">(1)</span></span><br>where <em>I</em>(•) is the identity function, |•| is a function on sets, counting the number of elements in it. α and β are any pre-chosen numbers so that α is close to 0 and β is close to 1. Here we set α = 0.01, β = 0.75 (<a href="#pone.0001922.s007">Text S3</a> and <a href="#pone.0001922.s006">Text S2</a>).</p>
<a id="article1.body1.sec4.sec3.p2" name="article1.body1.sec4.sec3.p2"></a><p>Another consideration was the balance between the sizes of node set pair (<em>V1</em>, <em>V2</em>). Due to the formidability of the searching space, we wanted to first focus on the sub-network pairs with similar sizes. Taking the above issues into consideration, we use the following objective function to search for candidate node set pairs:<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001922.e002&amp;representation=PNG" class="inline-graphic"><span class="note">(2)</span></span><br>where δ is a tuning parameter to control the penalty to the size differences between node sets <em>V1</em> and <em>V2</em> and <em>abs</em>(•) is the absolute value. A larger δ gives more penalties to the size differences between set <em>V1</em> and <em>V2</em>. Here we set δ = 1.5 (<a href="#pone.0001922.s003">Figure S3</a>, <a href="#pone.0001922.s004">Figure S4</a> and <a href="#pone.0001922.s008">Text S4</a>).</p>
<a id="article1.body1.sec4.sec3.p3" name="article1.body1.sec4.sec3.p3"></a><p>To implement the above objective, we proposed the following heuristic algorithm.</p>


<ol class="simple">

<li><strong>Input</strong>: (1) <em>G</em>(<em>V</em>, <em>E</em>), the network;<br>(2) α = 0.01, β = 0.75, <em>T</em> = 0.4, parameters in Equation (2).</li>

<li><strong>Output</strong>: <em>R</em> = (<em>G<sub>1</sub></em>, <em>G<sub>2</sub></em>, …, <em>G<sub>m</sub></em>), superset of <em>m</em> sub-networks <em>G<sub>i</sub></em>(<em>V1</em>,<em>V2</em>,<em>E′</em>) to be discovered and returned.</li>

</ol><a id="article1.body1.sec4.sec3.p4" name="article1.body1.sec4.sec3.p4"></a><p>1: <em>R</em> = Φ. <em>C</em> = 0.</p>
<a id="article1.body1.sec4.sec3.p5" name="article1.body1.sec4.sec3.p5"></a><p>2: <strong>repeat</strong></p>
<a id="article1.body1.sec4.sec3.p6" name="article1.body1.sec4.sec3.p6"></a><p>3:  Set <em>V1</em> = Φ, <em>V2</em> = Φ. <em>S</em>(<em>V1</em>, <em>V2</em>, <em>E′</em>) is the sub-network pair we will discover.</p>
<a id="article1.body1.sec4.sec3.p7" name="article1.body1.sec4.sec3.p7"></a><p>4:  Randomly select an edge <em>e<sub>ij</sub></em> = 1 from <em>E</em>. Set <em>V1</em> = {<em>i</em>}, <em>V2</em> = {<em>j</em>}.</p>
<a id="article1.body1.sec4.sec3.p8" name="article1.body1.sec4.sec3.p8"></a><p>5:  <strong>while</strong> there are nodes that can be added to <em>S</em> <strong>do</strong></p>
<a id="article1.body1.sec4.sec3.p9" name="article1.body1.sec4.sec3.p9"></a><p>6:   Find the set of all nodes in <em>G</em> that are connected to <em>S</em>: <em>V</em>′ = {<em>v</em> | <em>v</em>Є<em>V</em> and <em>e<sub>vk</sub></em> = 1, for some <em>k</em>Є<em>V</em>1∪<em>V</em>2}</p>
<a id="article1.body1.sec4.sec3.p10" name="article1.body1.sec4.sec3.p10"></a><p>7:   <strong>if</strong> <em>V′</em> is empty <strong>then</strong></p>
<a id="article1.body1.sec4.sec3.p11" name="article1.body1.sec4.sec3.p11"></a><p>8:    <strong>break</strong></p>
<a id="article1.body1.sec4.sec3.p12" name="article1.body1.sec4.sec3.p12"></a><p>9:   <strong>for</strong> all <em>v</em>Є<em>V</em>′ <strong>do</strong></p>
<a id="article1.body1.sec4.sec3.p13" name="article1.body1.sec4.sec3.p13"></a><p>10:    Calculate <em>score</em>(<em>v</em>) = max(<em>score</em>(<em>V</em>1∪<em>v</em>, <em>V</em>2), <em>score</em>(<em>V</em>1, <em>V</em>2∪<em>v</em>))</p>
<a id="article1.body1.sec4.sec3.p14" name="article1.body1.sec4.sec3.p14"></a><p>11:    set <em>score</em>(<em>v</em>) = 0 if <em>score</em>(<em>v</em>)&lt;<em>T</em></p>
<a id="article1.body1.sec4.sec3.p15" name="article1.body1.sec4.sec3.p15"></a><p>12:   <strong>end for</strong></p>
<a id="article1.body1.sec4.sec3.p16" name="article1.body1.sec4.sec3.p16"></a><p>13:   <strong>if</strong> score(v) = 0 for all <em>v</em>Є<em>V</em>′</p>
<a id="article1.body1.sec4.sec3.p17" name="article1.body1.sec4.sec3.p17"></a><p>    <strong>break</strong></p>
<a id="article1.body1.sec4.sec3.p18" name="article1.body1.sec4.sec3.p18"></a><p>14:   sample a node <em>v</em> randomly from <em>V′</em> according to vector <em>score</em>(<em>v</em>), <em>v</em>Є<em>V</em>′</p>
<a id="article1.body1.sec4.sec3.p19" name="article1.body1.sec4.sec3.p19"></a><p>15:   add <em>v</em> to <em>V1</em> if <em>score</em>(<em>V</em>1∪<em>v</em>, <em>V</em>2)&gt; = <em>score</em>(<em>V</em>1, <em>V</em>2∪<em>v</em>); to <em>V2</em> otherwise</p>
<a id="article1.body1.sec4.sec3.p20" name="article1.body1.sec4.sec3.p20"></a><p>16:  <strong>end while</strong></p>
<a id="article1.body1.sec4.sec3.p21" name="article1.body1.sec4.sec3.p21"></a><p>17:  add <em>S</em> into <em>R</em> if |<em>V1</em>|&gt;3, |<em>V2</em>|&gt;3, and <em>S</em>∉<em>R</em></p>
<a id="article1.body1.sec4.sec3.p22" name="article1.body1.sec4.sec3.p22"></a><p>18:  <em>C</em> = <em>C</em>+1</p>
<a id="article1.body1.sec4.sec3.p23" name="article1.body1.sec4.sec3.p23"></a><p>19: <strong>until</strong> <em>C</em>&gt;<em>C</em><sup>*</sup></p>
<a id="article1.body1.sec4.sec3.p24" name="article1.body1.sec4.sec3.p24"></a><p>Our algorithm randomly chose an edge from the whole set of edges and initialized the sub-network pair <em>V1</em> and <em>V2</em>. It then enumerated all other nodes and assigned them score according to Equation (2). We then sampled a node from these candidates according to the probability proportional to their scores. Note that a candidate with a score less than our threshold (<em>T</em> = 0.4) was not sampled. The sampled node was added into the sub-network according to its score. A pre-defined number <em>C</em>* ( = 10000 in this work) of searches are performed.</p>


<h4>Redundancy</h4>
<a id="article1.body1.sec4.sec4.p1" name="article1.body1.sec4.sec4.p1"></a><p>Similar to Kelley and Ideker <a href="#pone.0001922-Kelley1">[1]</a>, repeat pathway pairs were removed. Specifically, if pathway pair A and pathway pair B shared more than 50% synthetic lethal interaction edges, the smaller pathway pair was removed. For summary statistics and function enrichment analysis of the identified pathways, if pathway A and B shared more than 50% member genes, the smaller pathway was removed. The same procedure applies to pathways from Kelley and Ideker <a href="#pone.0001922-Kelley1">[1]</a> and Ulitsky and Shamir <a href="#pone.0001922-Ulitsky1">[8]</a>.</p>

</div>

<div id="section5" class="section"><a id="s5" name="s5" toc="s5" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0001922"></div><a name="pone.0001922.s001" id="pone.0001922.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001922.s001">Figure S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1"></a><p class="preSiDOI">Comparison of the size distribution of pathways identified by our method with that identified by Kelly/Ideker and Ulitsky/Shamir, using the same size constraint parameters and original datasets. Upper panel: size distribution of non-redundant pathways identified by our algorithm on the data used by Kelly/Ideker; Bottom panel: size distribution of non-redundant pathways identified by our algorithm on the data used by Ulitsky/Shamir.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001922.s001</p><a id="article1.body1.sec5.supplementary-material1.caption1.p2" name="article1.body1.sec5.supplementary-material1.caption1.p2"></a><p class="postSiDOI">(0.02 MB TIF)</p>
<a name="pone.0001922.s002" id="pone.0001922.s002"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001922.s002">Figure S2. </a></strong></p><a id="article1.body1.sec5.supplementary-material2.caption1.p1" name="article1.body1.sec5.supplementary-material2.caption1.p1"></a><p class="preSiDOI">Permutation distribution of β. A large fraction (21%) of permutation β had value of 0. For non-zero β, the mean was 0.091 and the standard deviation was 0.058.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001922.s002</p><a id="article1.body1.sec5.supplementary-material2.caption1.p2" name="article1.body1.sec5.supplementary-material2.caption1.p2"></a><p class="postSiDOI">(0.01 MB TIF)</p>
<a name="pone.0001922.s003" id="pone.0001922.s003"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001922.s003">Figure S3. </a></strong></p><a id="article1.body1.sec5.supplementary-material3.caption1.p1" name="article1.body1.sec5.supplementary-material3.caption1.p1"></a><p class="preSiDOI">Distribution of pathway size differences under different penalty parameter δ.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001922.s003</p><a id="article1.body1.sec5.supplementary-material3.caption1.p2" name="article1.body1.sec5.supplementary-material3.caption1.p2"></a><p class="postSiDOI">(0.01 MB TIF)</p>
<a name="pone.0001922.s004" id="pone.0001922.s004"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001922.s004">Figure S4. </a></strong></p><a id="article1.body1.sec5.supplementary-material4.caption1.p1" name="article1.body1.sec5.supplementary-material4.caption1.p1"></a><p class="preSiDOI">Distribution of pathway completeness under different penalty parameter δ.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001922.s004</p><a id="article1.body1.sec5.supplementary-material4.caption1.p2" name="article1.body1.sec5.supplementary-material4.caption1.p2"></a><p class="postSiDOI">(0.01 MB TIF)</p>
<a name="pone.0001922.s005" id="pone.0001922.s005"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001922.s005">Text S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material5.caption1.p1" name="article1.body1.sec5.supplementary-material5.caption1.p1"></a><p class="preSiDOI">Minimum pathway size parameter</p>
<p class="siDoi">doi:10.1371/journal.pone.0001922.s005</p><a id="article1.body1.sec5.supplementary-material5.caption1.p2" name="article1.body1.sec5.supplementary-material5.caption1.p2"></a><p class="postSiDOI">(0.03 MB DOC)</p>
<a name="pone.0001922.s006" id="pone.0001922.s006"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001922.s006">Text S2. </a></strong></p><a id="article1.body1.sec5.supplementary-material6.caption1.p1" name="article1.body1.sec5.supplementary-material6.caption1.p1"></a><p class="preSiDOI">Between-pathway completeness β.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001922.s006</p><a id="article1.body1.sec5.supplementary-material6.caption1.p2" name="article1.body1.sec5.supplementary-material6.caption1.p2"></a><p class="postSiDOI">(0.03 MB DOC)</p>
<a name="pone.0001922.s007" id="pone.0001922.s007"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001922.s007">Text S3. </a></strong></p><a id="article1.body1.sec5.supplementary-material7.caption1.p1" name="article1.body1.sec5.supplementary-material7.caption1.p1"></a><p class="preSiDOI">Within-pathway completeness α.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001922.s007</p><a id="article1.body1.sec5.supplementary-material7.caption1.p2" name="article1.body1.sec5.supplementary-material7.caption1.p2"></a><p class="postSiDOI">(0.02 MB DOC)</p>
<a name="pone.0001922.s008" id="pone.0001922.s008"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001922.s008">Text S4. </a></strong></p><a id="article1.body1.sec5.supplementary-material8.caption1.p1" name="article1.body1.sec5.supplementary-material8.caption1.p1"></a><p class="preSiDOI">Pathway size imbalance penalty δ.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001922.s008</p><a id="article1.body1.sec5.supplementary-material8.caption1.p2" name="article1.body1.sec5.supplementary-material8.caption1.p2"></a><p class="postSiDOI">(0.03 MB DOC)</p>
<a name="pone.0001922.s009" id="pone.0001922.s009"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001922.s009">Table S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material9.caption1.p1" name="article1.body1.sec5.supplementary-material9.caption1.p1"></a><p class="preSiDOI">Network of compensatory biological functions.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001922.s009</p><a id="article1.body1.sec5.supplementary-material9.caption1.p2" name="article1.body1.sec5.supplementary-material9.caption1.p2"></a><p class="postSiDOI">(0.05 MB XLS)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>The authors are grateful to three anonymous reviewers for their insightful and helpful suggestions.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: XM. Performed the experiments: XM WL AT. Analyzed the data: XM WL AT. Contributed reagents/materials/analysis tools: XM WL AT. Wrote the paper: XM WL AT.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0001922-Kelley1" id="pone.0001922-Kelley1"></a>Kelley R, Ideker T (2005) Systematic interpretation of genetic interactions using protein networks. Nat Biotechnol  23: 561–566.  <ul class="find" data-citedArticleID="1085770" data-doi="10.1038/nbt1096"><li><a href="http://dx.doi.org/10.1038/nbt1096" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Systematic+interpretation+of+genetic+interactions+using+protein+networks." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Systematic+interpretation+of+genetic+interactions+using+protein+networks.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0001922-Boone1" id="pone.0001922-Boone1"></a>Boone C, Bussey H, Andrews B (2007) Exploring genetic interactions and networks with yeast. Nat Rev Genet  8: 437–449.  <ul class="find" data-citedArticleID="1085756" data-doi="10.1038/nrg2085"><li><a href="http://dx.doi.org/10.1038/nrg2085" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Exploring+genetic+interactions+and+networks+with+yeast." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Exploring+genetic+interactions+and+networks+with+yeast.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0001922-Giaever1" id="pone.0001922-Giaever1"></a>Giaever G, Chu A, Ni L, Connelly C, Riles L, et al.  (2002) Functional profiling of the Saccharomyces cerevisiae genome. Nature  418: 387–391.  <ul class="find" data-citedArticleID="1085760" data-doi="10.1038/nature00935"><li><a href="http://dx.doi.org/10.1038/nature00935" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Functional+profiling+of+the+Saccharomyces+cerevisiae+genome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Functional+profiling+of+the+Saccharomyces+cerevisiae+genome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0001922-Tong1" id="pone.0001922-Tong1"></a>Tong A, Evangelista M, Parsons A, Xu H, Bader G, et al.  (2001) Systematic genetic analysis with ordered arrays of yeast deletion mutants. Science  294: 2364–2368.  <ul class="find" data-citedArticleID="1085788" data-doi="10.1126/science.1065810"><li><a href="http://dx.doi.org/10.1126/science.1065810" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Systematic+genetic+analysis+with+ordered+arrays+of+yeast+deletion+mutants." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Systematic+genetic+analysis+with+ordered+arrays+of+yeast+deletion+mutants.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0001922-Tong2" id="pone.0001922-Tong2"></a>Tong A, Lesage G, Bader G, Ding H, Xu H, et al.  (2004) Global mapping of the yeast genetic interaction network. Science  303: 808–813.  <ul class="find" data-citedArticleID="1085790" data-doi="10.1126/science.1091317"><li><a href="http://dx.doi.org/10.1126/science.1091317" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Global+mapping+of+the+yeast+genetic+interaction+network." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Global+mapping+of+the+yeast+genetic+interaction+network.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0001922-Stark1" id="pone.0001922-Stark1"></a>Stark C, Breitkreutz B, Reguly T, Boucher L, Breitkreutz A, et al.  (2006) BioGRID: a general repository for interaction datasets. Nucleic Acids Res  34: D535–539.  <ul class="find" data-citedArticleID="1085786" data-doi="10.1093/nar/gkj109"><li><a href="http://dx.doi.org/10.1093/nar/gkj109" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=BioGRID%3A+a+general+repository+for+interaction+datasets." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22BioGRID%3A+a+general+repository+for+interaction+datasets.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0001922-Ye1" id="pone.0001922-Ye1"></a>Ye P, Peyser B, Pan X, Boeke J, Spencer F, et al.  (2005) Gene function prediction from congruent synthetic lethal interactions in yeast. Mol Syst Biol  1: 2005.0026.  <ul class="find" data-citedArticleID="1085800" data-doi="10.1038/msb4100034"><li><a href="http://dx.doi.org/10.1038/msb4100034" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Gene+function+prediction+from+congruent+synthetic+lethal+interactions+in+yeast." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Gene+function+prediction+from+congruent+synthetic+lethal+interactions+in+yeast.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0001922-Ulitsky1" id="pone.0001922-Ulitsky1"></a>Ulitsky I, Shamir R (2007) Pathway redundancy and protein essentiality revealed in the Saccharomyces cerevisiae interaction networks. Mol Syst Biol  3: 104.  <ul class="find" data-citedArticleID="1085794" data-doi="10.1038/msb4100144"><li><a href="http://dx.doi.org/10.1038/msb4100144" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Pathway+redundancy+and+protein+essentiality+revealed+in+the+Saccharomyces+cerevisiae+interaction+networks." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Pathway+redundancy+and+protein+essentiality+revealed+in+the+Saccharomyces+cerevisiae+interaction+networks.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0001922-Uetz1" id="pone.0001922-Uetz1"></a>Uetz P, Giot L, Cagney G, Mansfield T, Judson R, et al.  (2000) A comprehensive analysis of protein-protein interactions in Saccharomyces cerevisiae. Nature  403: 623–627.  <ul class="find" data-citedArticleID="1085792" data-doi="10.1038/35001009"><li><a href="http://dx.doi.org/10.1038/35001009" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+comprehensive+analysis+of+protein-protein+interactions+in+Saccharomyces+cerevisiae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+comprehensive+analysis+of+protein-protein+interactions+in+Saccharomyces+cerevisiae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0001922-Ito1" id="pone.0001922-Ito1"></a>Ito T, Tashiro K, Muta S, Ozawa R, Chiba T, et al.  (2000) Toward a protein-protein interaction map of the budding yeast: A comprehensive system to examine two-hybrid interactions in all possible combinations between the yeast proteins. Proc Natl Acad Sci U S A  97: 1143–1147.  <ul class="find" data-citedArticleID="1085764" data-doi="10.1073/pnas.97.3.1143"><li><a href="http://dx.doi.org/10.1073/pnas.97.3.1143" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Toward+a+protein-protein+interaction+map+of+the+budding+yeast%3A+A+comprehensive+system+to+examine+two-hybrid+interactions+in+all+possible+combinations+between+the+yeast+proteins." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Toward+a+protein-protein+interaction+map+of+the+budding+yeast%3A+A+comprehensive+system+to+examine+two-hybrid+interactions+in+all+possible+combinations+between+the+yeast+proteins.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0001922-Harbison1" id="pone.0001922-Harbison1"></a>Harbison C, Gordon D, Lee T, Rinaldi N, Macisaac K, et al.  (2004) Transcriptional regulatory code of a eukaryotic genome. Nature  431: 99–104.  <ul class="find" data-citedArticleID="1085762" data-doi="10.1038/nature02800"><li><a href="http://dx.doi.org/10.1038/nature02800" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Transcriptional+regulatory+code+of+a+eukaryotic+genome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Transcriptional+regulatory+code+of+a+eukaryotic+genome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0001922-Ashburner1" id="pone.0001922-Ashburner1"></a>Ashburner M, Ball C, Blake J, Botstein D, Butler H, et al.  (2000) Gene ontology: tool for the unification of biology. The Gene Ontology Consortium. Nat Genet  25: 25–29.  <ul class="find" data-citedArticleID="1085754"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Gene+ontology%3A+tool+for+the+unification+of+biology.+The+Gene+Ontology+Consortium.&amp;auth=&amp;atitle=Gene+ontology%3A+tool+for+the+unification+of+biology.+The+Gene+Ontology+Consortium." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Gene+ontology%3A+tool+for+the+unification+of+biology.+The+Gene+Ontology+Consortium." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Gene+ontology%3A+tool+for+the+unification+of+biology.+The+Gene+Ontology+Consortium.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0001922-Karp1" id="pone.0001922-Karp1"></a>Karp G (1999) Cytoplasmic membrane systems: structure, function, and membrane trafficking. Cell and Molecular Biology: Concepts and Experiments. 2nd edition. New York: John Wiley and Sons, Inc. pp. 292–343.  <ul class="find-nolinks"></ul></li><li><span class="label">14.
              </span><a name="pone.0001922-Munro1" id="pone.0001922-Munro1"></a>Munro S (2001) What can yeast tell us about N-linked glycosylation in the Golgi apparatus? FEBS Lett  498: 223–227.  <ul class="find" data-citedArticleID="1085776" data-doi="10.1016/s0014-5793(01)02488-7"><li><a href="http://dx.doi.org/10.1016/s0014-5793(01)02488-7" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=What+can+yeast+tell+us+about+N-linked+glycosylation+in+the+Golgi+apparatus%3F" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22What+can+yeast+tell+us+about+N-linked+glycosylation+in+the+Golgi+apparatus%3F%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0001922-Yokoo1" id="pone.0001922-Yokoo1"></a>Yoko-o T, Wiggins C, Stolz J, Peak-Chew S, Munro S (2003) An N-acetylglucosaminyltransferase of the Golgi apparatus of the yeast Saccharomyces cerevisiae that can modify N-linked glycans. Glycobiology  13: 581–589.  <ul class="find" data-citedArticleID="1085802" data-doi="10.1093/glycob/cwg063"><li><a href="http://dx.doi.org/10.1093/glycob/cwg063" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=An+N-acetylglucosaminyltransferase+of+the+Golgi+apparatus+of+the+yeast+Saccharomyces+cerevisiae+that+can+modify+N-linked+glycans." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22An+N-acetylglucosaminyltransferase+of+the+Golgi+apparatus+of+the+yeast+Saccharomyces+cerevisiae+that+can+modify+N-linked+glycans.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0001922-Lesage1" id="pone.0001922-Lesage1"></a>Lesage G, Shapiro J, Specht C, Sdicu A, Ménard P, et al.  (2005) An interactional network of genes involved in chitin synthesis in Saccharomyces cerevisiae. BMC Genet  6: 8.  <ul class="find" data-citedArticleID="1085772"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=An+interactional+network+of+genes+involved+in+chitin+synthesis+in+Saccharomyces+cerevisiae.&amp;auth=&amp;atitle=An+interactional+network+of+genes+involved+in+chitin+synthesis+in+Saccharomyces+cerevisiae." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=An+interactional+network+of+genes+involved+in+chitin+synthesis+in+Saccharomyces+cerevisiae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22An+interactional+network+of+genes+involved+in+chitin+synthesis+in+Saccharomyces+cerevisiae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0001922-Wiggins1" id="pone.0001922-Wiggins1"></a>Wiggins C, Munro S (1998) Activity of the yeast MNN1 alpha-1,3-mannosyltransferase requires a motif conserved in many other families of glycosyltransferases. Proc Natl Acad Sci U S A  95: 7945–7950.  <ul class="find" data-citedArticleID="1085796" data-doi="10.1073/pnas.95.14.7945"><li><a href="http://dx.doi.org/10.1073/pnas.95.14.7945" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Activity+of+the+yeast+MNN1+alpha-1%2C3-mannosyltransferase+requires+a+motif+conserved+in+many+other+families+of+glycosyltransferases." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Activity+of+the+yeast+MNN1+alpha-1%2C3-mannosyltransferase+requires+a+motif+conserved+in+many+other+families+of+glycosyltransferases.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0001922-Osmond1" id="pone.0001922-Osmond1"></a>Osmond BC, Specht CA, Robbins PW (1999) Chitin synthase III: synthetic lethal mutants and “stress related” chitin synthesis that bypasses the CSD3/CHS6 localization pathway. Proc Natl Acad Sci U S A  96: 11206–10.  <ul class="find" data-citedArticleID="1085780" data-doi="10.1073/pnas.96.20.11206"><li><a href="http://dx.doi.org/10.1073/pnas.96.20.11206" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Chitin+synthase+III%3A+synthetic+lethal+mutants+and+%E2%80%9Cstress+related%E2%80%9D+chitin+synthesis+that+bypasses+the+CSD3%2FCHS6+localization+pathway." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Chitin+synthase+III%3A+synthetic+lethal+mutants+and+%E2%80%9Cstress+related%E2%80%9D+chitin+synthesis+that+bypasses+the+CSD3%2FCHS6+localization+pathway.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0001922-Milo1" id="pone.0001922-Milo1"></a>Milo R, Shen-Orr S, Itzkovitz S, Kashtan N, Chklovskii D, et al.  (2002) Network motifs: simple building blocks of complex networks. Science  298: 824–827.  <ul class="find" data-citedArticleID="1085774" data-doi="10.1126/science.298.5594.824"><li><a href="http://dx.doi.org/10.1126/science.298.5594.824" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Network+motifs%3A+simple+building+blocks+of+complex+networks." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Network+motifs%3A+simple+building+blocks+of+complex+networks.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0001922-Kanehisa1" id="pone.0001922-Kanehisa1"></a>Kanehisa M, Goto S, Hattori M, Aoki-Kinoshita K, Itoh M, et al.  (2006) From genomics to chemical genomics: new developments in KEGG. Nucleic Acids Res  34: D354–357.  <ul class="find" data-citedArticleID="1085766" data-doi="10.1093/nar/gkj102"><li><a href="http://dx.doi.org/10.1093/nar/gkj102" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=From+genomics+to+chemical+genomics%3A+new+developments+in+KEGG." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22From+genomics+to+chemical+genomics%3A+new+developments+in+KEGG.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0001922-Schuldiner1" id="pone.0001922-Schuldiner1"></a>Schuldiner M, Collins S, Thompson N, Denic V, Bhamidipati A, et al.  (2005) Exploration of the function and organization of the yeast early secretory pathway through an epistatic miniarray profile. Cell  123: 507–519.  <ul class="find" data-citedArticleID="1085782" data-doi="10.1016/j.cell.2005.08.031"><li><a href="http://dx.doi.org/10.1016/j.cell.2005.08.031" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Exploration+of+the+function+and+organization+of+the+yeast+early+secretory+pathway+through+an+epistatic+miniarray+profile." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Exploration+of+the+function+and+organization+of+the+yeast+early+secretory+pathway+through+an+epistatic+miniarray+profile.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0001922-Winzeler1" id="pone.0001922-Winzeler1"></a>Winzeler EA, Shoemaker DD, Astromoff A, Liang H, Anderson K, et al.  (1999) Functional characterization of the S. cerevisiae genome by gene deletion and parallel analysis. Science  285: 901–6.  <ul class="find" data-citedArticleID="1085798" data-doi="10.1126/science.285.5429.901"><li><a href="http://dx.doi.org/10.1126/science.285.5429.901" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Functional+characterization+of+the+S.+cerevisiae+genome+by+gene+deletion+and+parallel+analysis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Functional+characterization+of+the+S.+cerevisiae+genome+by+gene+deletion+and+parallel+analysis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0001922-Davierwala1" id="pone.0001922-Davierwala1"></a>Davierwala AP, Haynes J, Li Z, Brost RL, Robinson MD, et al.  (2005) The synthetic genetic interaction spectrum of essential genes. Nat Genet  37: 1147–52.  <ul class="find" data-citedArticleID="1085758" data-doi="10.1038/ng1640"><li><a href="http://dx.doi.org/10.1038/ng1640" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+synthetic+genetic+interaction+spectrum+of+essential+genes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+synthetic+genetic+interaction+spectrum+of+essential+genes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0001922-Nash1" id="pone.0001922-Nash1"></a>Nash R, Weng S, Hitz B, Balakrishnan R, Christie K, et al.  (2007) Expanded protein information at SGD: new pages and proteome browser. Nucleic Acids Res  35: D468–471.  <ul class="find" data-citedArticleID="1085778" data-doi="10.1093/nar/gkl931"><li><a href="http://dx.doi.org/10.1093/nar/gkl931" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Expanded+protein+information+at+SGD%3A+new+pages+and+proteome+browser." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Expanded+protein+information+at+SGD%3A+new+pages+and+proteome+browser.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0001922-Shannon1" id="pone.0001922-Shannon1"></a>Shannon P, Markiel A, Ozier O, Baliga N, Wang J, et al.  (2003) Cytoscape: a software environment for integrated models of biomolecular interaction networks. Genome Res  13: 2498–2504.  <ul class="find" data-citedArticleID="1085784" data-doi="10.1101/gr.1239303"><li><a href="http://dx.doi.org/10.1101/gr.1239303" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Cytoscape%3A+a+software+environment+for+integrated+models+of+biomolecular+interaction+networks." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Cytoscape%3A+a+software+environment+for+integrated+models+of+biomolecular+interaction+networks.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001922.XML" value="85522"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001922.PDF" value="552768"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001922.g001.PNG_L" value="50388"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001922.g001.PNG_M" value="50388"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001922.g001.PNG_S" value="13991"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001922.g001.TIF" value="68876"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001922.g001.PNG_I" value="63246"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001922.g002.PNG_L" value="302988"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001922.g002.PNG_M" value="301392"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001922.g002.PNG_S" value="15739"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001922.g002.TIF" value="363570"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001922.g002.PNG_I" value="99175"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001922.g003.PNG_L" value="4452"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001922.g003.PNG_M" value="4452"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001922.g003.PNG_S" value="3092"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001922.g003.TIF" value="13664"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001922.g003.PNG_I" value="6859"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001922.g004.PNG_L" value="3272"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001922.g004.PNG_M" value="3272"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001922.g004.PNG_S" value="2354"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001922.g004.TIF" value="10098"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001922.g004.PNG_I" value="4381"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001922.g005.PNG_L" value="4360"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001922.g005.PNG_M" value="4360"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001922.g005.PNG_S" value="3395"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001922.g005.TIF" value="10860"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001922.g005.PNG_I" value="9268"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001922.g006.PNG_L" value="257508"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001922.g006.PNG_M" value="340756"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001922.g006.PNG_S" value="15174"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001922.g006.TIF" value="333256"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001922.g006.PNG_I" value="108985"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001922.e001.PNG" value="25447"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001922.e001.TIF" value="40948"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001922.e002.PNG" value="17879"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001922.e002.TIF" value="33912"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001922.s001.TIF" value="13041"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001922.s002.TIF" value="6782"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001922.s003.TIF" value="15330"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001922.s004.TIF" value="16134"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001922.s005.DOC" value="26624"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001922.s006.DOC" value="26624"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001922.s007.DOC" value="20480"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001922.s008.DOC" value="26112"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001922.s009.XLS" value="53248"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001922&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001922" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001922&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0001922&volume=&issue=&title=Mapping Genetically Compensatory Pathways from Synthetic Lethal Interactions in Yeast&author_name=Xiaotu%20Ma%2C%20Aaron%20M.%20Tarone%2C%20Wenyuan%20Li&start_page=1&end_page=10" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001922" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001922" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0001922" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001922&amp;t=Mapping%20Genetically%20Compensatory%20Pathways%20from%20Synthetic%20Lethal%20Interactions%20in%20Yeast" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001922&title=Mapping%20Genetically%20Compensatory%20Pathways%20from%20Synthetic%20Lethal%20Interactions%20in%20Yeast&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001922&amp;title=Mapping%20Genetically%20Compensatory%20Pathways%20from%20Synthetic%20Lethal%20Interactions%20in%20Yeast" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001922" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0001922&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Mapping Genetically Compensatory Pathways from Synthetic Lethal Interactions in Yeast';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0001922';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Mapping%20Genetically%20Compensatory%20Pathways%20from%20Synthetic%20Lethal%20Interactions%20in%20Yeast http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001922" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0001922" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">










          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Chitin%22" title="Search for articles in the subject area:'Chitin'"><div class="flagText">Chitin</div></a>
              <div data-categoryid="24559" data-articleid="26668"
                   data-categoryname="Chitin"
                   class="flagImage" title="Flag 'Chitin' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Gene+mapping%22" title="Search for articles in the subject area:'Gene mapping'"><div class="flagText">Gene mapping</div></a>
              <div data-categoryid="47615" data-articleid="26668"
                   data-categoryname="Gene mapping"
                   class="flagImage" title="Flag 'Gene mapping' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Gene+ontologies%22" title="Search for articles in the subject area:'Gene ontologies'"><div class="flagText">Gene ontologies</div></a>
              <div data-categoryid="40557" data-articleid="26668"
                   data-categoryname="Gene ontologies"
                   class="flagImage" title="Flag 'Gene ontologies' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Genetic+interactions%22" title="Search for articles in the subject area:'Genetic interactions'"><div class="flagText">Genetic interactions</div></a>
              <div data-categoryid="40249" data-articleid="26668"
                   data-categoryname="Genetic interactions"
                   class="flagImage" title="Flag 'Genetic interactions' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Mutation%22" title="Search for articles in the subject area:'Mutation'"><div class="flagText">Mutation</div></a>
              <div data-categoryid="31809" data-articleid="26668"
                   data-categoryname="Mutation"
                   class="flagImage" title="Flag 'Mutation' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Protein+interactions%22" title="Search for articles in the subject area:'Protein interactions'"><div class="flagText">Protein interactions</div></a>
              <div data-categoryid="17661" data-articleid="26668"
                   data-categoryname="Protein interactions"
                   class="flagImage" title="Flag 'Protein interactions' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Protein-protein+interactions%22" title="Search for articles in the subject area:'Protein-protein interactions'"><div class="flagText">Protein-protein interactions</div></a>
              <div data-categoryid="18067" data-articleid="26668"
                   data-categoryname="Protein-protein interactions"
                   class="flagImage" title="Flag 'Protein-protein interactions' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Yeast%22" title="Search for articles in the subject area:'Yeast'"><div class="flagText">Yeast</div></a>
              <div data-categoryid="43039" data-articleid="26668"
                   data-categoryname="Yeast"
                   class="flagImage" title="Flag 'Yeast' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=7564'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=5493'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=8337&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>

<div class="block sidebar-comments">
    <div class="header">
        <h3>Comments</h3>
    </div>
      <p><a href="/annotation/listThread.action?root=6175">Referee comments: Referee 4</a><br>Posted by PLoS_ONE_Group</p>
      <p><a href="/annotation/listThread.action?root=16091">Referee comments: Referee 3</a><br>Posted by PLoS_ONE_Group</p>
      <p><a href="/annotation/listThread.action?root=20293">Referee comments: Referee 2</a><br>Posted by PLoS_ONE_Group</p>
</div>

</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
