

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Meta-Analysis in Genome-Wide Association Datasets: Strategies and Application in Parkinson Disease</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0000196"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0000196" />

    <meta name="citation_title" content="Meta-Analysis in Genome-Wide Association Datasets: Strategies and Application in Parkinson Disease"/>
    <meta itemprop="name" content="Meta-Analysis in Genome-Wide Association Datasets: Strategies and Application in Parkinson Disease"/>

      <meta name="citation_author" content="Evangelos Evangelou"/>
            <meta name="citation_author_institution" content="Clinical and Molecular Epidemiology Unit, Department of Hygiene and Epidemiology, University of Ioannina School of Medicine, Ioannina, Greece"/>
      <meta name="citation_author" content="Demetrius M. Maraganore"/>
            <meta name="citation_author_institution" content="Department of Neurology, Mayo Clinic College of Medicine, Rochester, Minnesota, United States of America"/>
      <meta name="citation_author" content="John P.A. Ioannidis"/>
            <meta name="citation_author_institution" content="Clinical and Molecular Epidemiology Unit, Department of Hygiene and Epidemiology, University of Ioannina School of Medicine, Ioannina, Greece"/>
            <meta name="citation_author_institution" content="Biomedical Research Institute, Foundation for Research and Technology-Hellas, Ioannina, Greece"/>
            <meta name="citation_author_institution" content="Institute for Clinical Research and Health Policy Studies, Tufts-New England Medical Center, Tufts University School of Medicine, Boston, Massachusetts, United States of America"/>

    <meta name="citation_date" content="2007/2/7"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0000196.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e196"/>
    <meta name="citation_issue" content="2"/>
    <meta name="citation_volume" content="2"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Genome-wide association studies for common diseases and complex traits.; citation_author=JN Hirschhorn; citation_author=MJ Daly; citation_journal_title=Nat Rev Genet; citation_volume=6; citation_number=1; citation_pages=95-108; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Genome-wide association studies: theoretical and practical concerns.; citation_author=WY Wang; citation_author=BJ Barratt; citation_author=DG Clayton; citation_author=JA Todd; citation_journal_title=Nat Rev Genet; citation_volume=6; citation_number=2; citation_pages=109-18; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Statistical false positive or true disease pathway?; citation_author=JA Todd; citation_journal_title=Nat Genet; citation_volume=38; citation_number=3; citation_pages=731-3; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Genetic associations: false or true?; citation_author=JP Ioannidis; citation_journal_title=Trends Mol Med; citation_volume=9; citation_number=4; citation_pages=135-8; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=High-resolution whole-genome association study of Parkinson disease.; citation_author=DM Maraganore; citation_author=M de Andrade; citation_author=TG Lesnick; citation_author=KJ Strain; citation_author=MJ Farrer; citation_journal_title=Am J Hum Genet; citation_volume=77; citation_number=5; citation_pages=685-93; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Lack of replication of thirteen single-nucleotide polymorphisms implicated in Parkinson's disease: a large-scale international study.; citation_author=A Elbaz; citation_author=LM Nelson; citation_author=H Payami; citation_author=JP Ioannidis; citation_author=BK Fiske; citation_journal_title=Lancet Neurol; citation_volume=5; citation_number=6; citation_pages=917-23; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Implications of small effect sizes of individual genetic variants on the design and interpretation of genetic association studies of complex diseases.; citation_author=JP Ioannidis; citation_author=TA Trikalinos; citation_author=MJ Khoury; citation_journal_title=Am J Epidemiol; citation_volume=164; citation_number=7; citation_pages=609-14; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Genetics. Delivering new disease genes.; citation_author=LR Cardon; citation_journal_title=Science; citation_volume=314; citation_number=8; citation_pages=1403-5; citation_date=2006; " />
      <meta name="citation_reference" content="citation_number=9; " />
      <meta name="citation_reference" content="citation_title=Replication validity of genetic association studies.; citation_author=JP Ioannidis; citation_author=EE Ntzani; citation_author=TA Trikalinos; citation_author=DG Contopoulos-Ioannidis; citation_journal_title=Nat Genet; citation_volume=29; citation_number=10; citation_pages=306-9; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Meta-analysis of genetic association studies supports a contribution of common variants to susceptibility to common disease.; citation_author=KE Lohmueller; citation_author=CL Pearce; citation_author=M Pike; citation_author=ES Lander; citation_author=JN Hirschhorn; citation_journal_title=Nat Genet; citation_volume=33; citation_number=11; citation_pages=177-82; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Genome-wide genotyping in Parkinson's disease and neurologically normal controls: first stage analysis and public release of data.; citation_author=HC Fung; citation_author=S Scholz; citation_author=M Matarin; citation_author=J Simon-Sanchez; citation_author=D Hernandez; citation_journal_title=Lancet Neurol; citation_volume=5; citation_number=12; citation_pages=911-6; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Meta-analysis, decision analysis, and cost-effectiveness analysis.; citation_author=D Pettiti; citation_number=13; citation_date=1999; citation_publisher=Oxford University Press, New York, 2nd edition; " />
      <meta name="citation_reference" content="citation_title=The statistical basis of meta-analysis.; citation_author=JL Fleiss; citation_journal_title=Stat Methods Med Res; citation_volume=2; citation_number=14; citation_pages=121-45; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=Measuring inconsistency in meta-analyses.; citation_author=JP Higgins; citation_author=SG Thompson; citation_author=JJ Deeks; citation_author=DG Altman; citation_journal_title=BMJ; citation_volume=327; citation_number=15; citation_pages=557-60; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Quantitative synthesis in systematic reviews.; citation_author=J Lau; citation_author=JP Ioannidis; citation_author=CH Schmid; citation_journal_title=Ann Intern Med; citation_volume=127; citation_number=16; citation_pages=820-6; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Cloning and mapping of the UNC5C gene to human chromosome 4q21–q23.; citation_author=SL Ackerman; citation_author=BB Knowles; citation_journal_title=Genomics; citation_volume=52; citation_number=17; citation_pages=205-8; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Netrin and netrin receptor expression in the embryonic mammalian nervous system suggests roles in retinal, striatal, nigral, and cerebellar development.; citation_author=FJ Livesey; citation_author=SP Hunt; citation_journal_title=Mol Cell Neurosci; citation_volume=8; citation_number=18; citation_pages=417-29; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Axonal growth regulation of fetal and embryonic stem cell-derived dopaminergic neurons by Netrin-1 and Slits.; citation_author=L Lin; citation_author=O Isacson; citation_journal_title=Stem Cells; citation_volume=24; citation_number=19; citation_pages=2504-13; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Relative citation impact of various study designs in the health sciences.; citation_author=NA Patsopoulos; citation_author=AA Analatos; citation_author=JP Ioannidis; citation_journal_title=JAMA; citation_volume=293; citation_number=20; citation_pages=2362-6; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Assessing the probability that a positive report is false: an approach for molecular epidemiology studies.; citation_author=S Wacholder; citation_author=S Chanock; citation_author=M Garcia-Closas; citation_author=L El Ghormli; citation_author=N Rothman; citation_journal_title=J Natl Cancer Inst; citation_volume=96; citation_number=21; citation_pages=434-42; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Genetic associations in large versus small studies: an empirical assessment.; citation_author=JP Ioannidis; citation_author=TA Trikalinos; citation_author=EE Ntzani; citation_author=DG Contopoulos-Ioannidis; citation_journal_title=Lancet; citation_volume=361; citation_number=22; citation_pages=567-71; citation_date=2003; " />
      <meta name="citation_reference" content="citation_number=23; " />
      <meta name="citation_reference" content="citation_title=Evaluating coverage of genome-wide association studies.; citation_author=JC Barrett; citation_author=LR Cardon; citation_journal_title=Nat Genet; citation_volume=38; citation_number=24; citation_pages=659-62; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=A gene-centric approach to genome-wide association studies.; citation_author=E Jorgenson; citation_author=JS Witte; citation_journal_title=Nat Rev Genet; citation_volume=7; citation_number=25; citation_pages=885-91; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Joint analysis is more efficient than replication-based analysis for two-stage genome-wide association studies.; citation_author=AD Skol; citation_author=LJ Scott; citation_author=GR Abecasis; citation_author=M Boehnke; citation_journal_title=Nat Genet; citation_volume=38; citation_number=26; citation_pages=209-13; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=How meta-analysis increases statistical power.; citation_author=LD Cohn; citation_author=BJ Becker; citation_journal_title=Psychol Methods; citation_volume=8; citation_number=27; citation_pages=243-53; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=The power of statistical tests in meta-analysis.; citation_author=LV Hedges; citation_author=TD Pigott; citation_journal_title=Psychol Methods; citation_volume=6; citation_number=28; citation_pages=203-17; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Summing up evidence: one answer is not always enough.; citation_author=J Lau; citation_author=JP Ioannidis; citation_author=CH Schmid; citation_journal_title=Lancet; citation_volume=351; citation_number=29; citation_pages=123-7; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Complement factor H polymorphism in age-related macular degeneration.; citation_author=RJ Klein; citation_author=C Zeiss; citation_author=EY Chew; citation_author=JY Tsai; citation_author=RS Sackler; citation_journal_title=Science; citation_volume=308; citation_number=30; citation_pages=385-9; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Genetics. Was the Human Genome Project worth the effort?; citation_author=SP Daiger; citation_journal_title=Science; citation_volume=308; citation_number=31; citation_pages=362-4; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=A genome-wide association study of nonsynonymous SNPs identifies a type 1 diabetes locus in the interferon-induced helicase (IFIH1) region.; citation_author=DJ Smyth; citation_author=JD Cooper; citation_author=R Bailey; citation_author=S Field; citation_author=O Burren; citation_journal_title=Nat Genet; citation_volume=38; citation_number=32; citation_pages=617-9; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=A common genetic variant is associated with adult and childhood obesity.; citation_author=A Herbert; citation_author=NP Gerry; citation_author=MB McQueen; citation_author=IM Heid; citation_author=A Pfeufer; citation_journal_title=Science; citation_volume=312; citation_number=33; citation_pages=279-83; citation_date=2006; " />
      <meta name="citation_reference" content="citation_number=34; " />
      <meta name="citation_reference" content="citation_title=Cumulative meta-analysis of therapeutic trials for myocardial infarction.; citation_author=J Lau; citation_author=EM Antman; citation_author=J Jimenez-Silva; citation_author=B Kupelnick; citation_author=F Mosteller; citation_journal_title=N Engl J Med; citation_volume=327; citation_number=35; citation_pages=248-54; citation_date=1992; " />
      <meta name="citation_reference" content="citation_title=A network of investigator networks in human genome epidemiology.; citation_author=JP Ioannidis; citation_author=J Bernstein; citation_author=P Boffetta; citation_author=J Danesh; citation_author=S Dolan; citation_journal_title=Am J Epidemiol; citation_volume=162; citation_number=36; citation_pages=302-4; citation_date=2005; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000196" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Meta-Analysis in Genome-Wide Association Datasets: Strategies and Application in Parkinson Disease"/>
    <meta name="twitter:description" content="BackgroundGenome-wide association studies hold substantial promise for identifying common genetic variants that regulate susceptibility to complex diseases. However, for the detection of small genetic effects, single studies may be underpowered. Power may be improved by combining genome-wide datasets with meta-analytic techniques.Methodology/Principal FindingsBoth single and two-stage genome-wide data may be combined and there are several possible strategies. In the two-stage framework, we considered the options of (1) enhancement of replication data and (2) enhancement of first-stage data, and then, we also considered (3) joint meta-analyses including all first-stage and second-stage data. These strategies were examined empirically using data from two genome-wide association studies (three datasets) on Parkinson disease. In the three strategies, we derived 12, 5, and 49 single nucleotide polymorphisms that show significant associations at conventional levels of statistical significance. None of these remained significant after conservative adjustment for the number of performed analyses in each strategy. However, some may warrant further consideration: 6 SNPs were identified with at least 2 of the 3 strategies and 3 SNPs [rs1000291 on chromosome 3, rs2241743 on chromosome 4 and rs3018626 on chromosome 11] were identified with all 3 strategies and had no or minimal between-dataset heterogeneity (I2&#8202;=&#8202;0, 0 and 15%, respectively). Analyses were primarily limited by the suboptimal overlap of tested polymorphisms across different datasets (e.g., only 31,192 shared polymorphisms between the two tier 1 datasets).Conclusions/SignificanceMeta-analysis may be used to improve the power and examine the between-dataset heterogeneity of genome-wide association studies. Prospective designs may be most efficient, if they try to maximize the overlap of genotyping platforms and anticipate the combination of data across many genome-wide association studies."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0000196.g001"/>

  <meta property="og:title" content="Meta-Analysis in Genome-Wide Association Datasets: Strategies and Application in Parkinson Disease" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000196" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=3700'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=4918'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=8853&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0000196">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000196" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000196&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Meta-Analysis in Genome-Wide Association Datasets: Strategies and Application in Parkinson Disease
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Evangelos Evangelou, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Clinical and Molecular Epidemiology Unit, Department of Hygiene and Epidemiology, University of Ioannina School of Medicine, Ioannina, Greece
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Demetrius M. Maraganore, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Neurology, Mayo Clinic College of Medicine, Rochester, Minnesota, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            John P.A. Ioannidis
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:jioannid@cc.uoi.gr">jioannid@cc.uoi.gr</a>
            </p>

                <p>Affiliations:
                  Clinical and Molecular Epidemiology Unit, Department of Hygiene and Epidemiology, University of Ioannina School of Medicine, Ioannina, Greece, 
                  Biomedical Research Institute, Foundation for Research and Technology-Hellas, Ioannina, Greece, 
                  Institute for Clinical Research and Health Policy Studies, Tufts-New England Medical Center, Tufts University School of Medicine, Boston, Massachusetts, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: February 07, 2007</li>
    <li>DOI: 10.1371/journal.pone.0000196</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0000196" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0000196" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000196" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0000196" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0000196-t001" data-doi="info:doi/10.1371/journal.pone.0000196" data-uri="info:doi/10.1371/journal.pone.0000196.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000196.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000196-t002" data-doi="info:doi/10.1371/journal.pone.0000196" data-uri="info:doi/10.1371/journal.pone.0000196.t002" title="Table 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000196.t002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000196-t003" data-doi="info:doi/10.1371/journal.pone.0000196" data-uri="info:doi/10.1371/journal.pone.0000196.t003" title="Table 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000196.t003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000196-g001" data-doi="info:doi/10.1371/journal.pone.0000196" data-uri="info:doi/10.1371/journal.pone.0000196.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000196.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000196">Reader Comments (1)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0000196" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2>
<h3>Background</h3>
<a id="article1.front1.article-meta1.abstract1.sec1.p1" name="article1.front1.article-meta1.abstract1.sec1.p1"></a><p>Genome-wide association studies hold substantial promise for identifying common genetic variants that regulate susceptibility to complex diseases. However, for the detection of small genetic effects, single studies may be underpowered. Power may be improved by combining genome-wide datasets with meta-analytic techniques.</p>


<h3>Methodology/Principal Findings</h3>
<a id="article1.front1.article-meta1.abstract1.sec2.p1" name="article1.front1.article-meta1.abstract1.sec2.p1"></a><p>Both single and two-stage genome-wide data may be combined and there are several possible strategies. In the two-stage framework, we considered the options of (1) enhancement of replication data and (2) enhancement of first-stage data, and then, we also considered (3) joint meta-analyses including all first-stage and second-stage data. These strategies were examined empirically using data from two genome-wide association studies (three datasets) on Parkinson disease. In the three strategies, we derived 12, 5, and 49 single nucleotide polymorphisms that show significant associations at conventional levels of statistical significance. None of these remained significant after conservative adjustment for the number of performed analyses in each strategy. However, some may warrant further consideration: 6 SNPs were identified with at least 2 of the 3 strategies and 3 SNPs [rs1000291 on chromosome 3, rs2241743 on chromosome 4 and rs3018626 on chromosome 11] were identified with all 3 strategies and had no or minimal between-dataset heterogeneity (I<sup>2</sup> = 0, 0 and 15%, respectively). Analyses were primarily limited by the suboptimal overlap of tested polymorphisms across different datasets (e.g., only 31,192 shared polymorphisms between the two tier 1 datasets).</p>


<h3>Conclusions/Significance</h3>
<a id="article1.front1.article-meta1.abstract1.sec3.p1" name="article1.front1.article-meta1.abstract1.sec3.p1"></a><p>Meta-analysis may be used to improve the power and examine the between-dataset heterogeneity of genome-wide association studies. Prospective designs may be most efficient, if they try to maximize the overlap of genotyping platforms and anticipate the combination of data across many genome-wide association studies.</p>

</div>


<div class="articleinfo"><p><strong>Citation: </strong>Evangelou E, Maraganore DM, Ioannidis JP (2007) Meta-Analysis in Genome-Wide Association Datasets: Strategies and Application in Parkinson Disease. PLoS ONE 2(2):
          e196.
            doi:10.1371/journal.pone.0000196</p><p><strong>Academic Editor: </strong>Suzannah Rutherford, Fred Hutchinson Cancer Research Center, United States of America</p><p><strong>Received:</strong> December 29, 2006; <strong>Accepted:</strong> January 15, 2007; <strong>Published:</strong> February 7, 2007</p><p><strong>Copyright:</strong> © 2007 Evangelou et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>E.E. was funded in part by a PENED grant from the General Secretariat for Research and Technology, Greece and the European Union.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>Genome-wide association analyses are increasingly used to identify common genetic variants that determine susceptibility to disease <a href="#pone.0000196-Hirschhorn1">[1]</a>, <a href="#pone.0000196-Wang1">[2]</a>. Several early successes have generated enthusiasm that such hypothesis-free massive-testing methods may succeed <a href="#pone.0000196-Todd1">[3]</a>, whereas many years of candidate gene approaches have yielded limited, and largely irreproducible postulated associations <a href="#pone.0000196-Ioannidis1">[4]</a>. However, there are still considerable difficulties in discovering common genetic variants of interest. We already have examples where findings initially highlighted by genome-wide approaches have not been replicated by large-scale studies. This situation has arisen for example in Parkinson's disease, where 13 polymorphisms were originally identified as being potentially important for determining the risk of the disease in a two-tier genome-wide association study <a href="#pone.0000196-Maraganore1">[5]</a>. Nevertheless, all 13 proposed associations were not replicated by a large-scale effort involving over 12,000 subjects <a href="#pone.0000196-Elbaz1">[6]</a>. For most common diseases, the main genetic effects are expected to be small and therefore would require very large studies to capture <a href="#pone.0000196-Ioannidis2">[7]</a>. Genome-wide association studies published to-date have had mostly modest sample sizes, and even the ongoing efforts sponsored by Wellcome Trust and the GAIN initiative <a href="#pone.0000196-Cardon1">[8]</a>, <a href="#pone.0000196-dbGaP1">[9]</a> may still be underpowered to detect odds ratios in the range of 1.0–1.3, especially if the genetic variants of interest are not very common.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>It is important to maximally exploit the available data from genome-wide association studies and combine information from different such studies performed on the same disease. In the candidate gene era, a very large number of teams independently pursued studies on specific candidate gene variants. While the technical and financial requirements for genome-wide association studies are more demanding, several such studies may still be conducted by independent teams of investigators working on the same disease. This creates a challenge and an opportunity to combine these data with meta-analytic techniques. Meta-analysis has already been accepted as a prime method for examining the consistency, replication, and credibility of proposed genetic associations <a href="#pone.0000196-Ioannidis3">[10]</a>, <a href="#pone.0000196-Lohmueller1">[11]</a>. However, to our knowledge, no meta-analysis has yet been performed combining data from different genome-wide association studies on the same disease. Here, we have performed such a meta-analysis for Parkinson's disease. We aimed to explore the different meta-analytic strategies that can be pursued and to dissect the limitations that arise in combining such datasets with meta-analytic methods.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Methods"></a><h3>Methods</h3>
<h4>Databases</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>We used publicly available data from two genome-wide association studies of Parkinson's disease (PD). Maraganore et al. <a href="#pone.0000196-Maraganore1">[5]</a> used a two-tiered genotyping approach (which will be referred as Mayo tier 1 and Mayo tier 2 for simplicity). In Mayo tier 1, 443 case-unaffected sibling pairs that were discordant for PD were included. Genotyping used the Perlegen platform. For the 205,031 single nucleotide polymorphisms (SNPs) that were polymorphic within the study sample the Hardy-Weinberg equilibrium (HWE) p-value was &gt;0.001 in controls for 198,345 SNPs. The investigators performed a liberalization of the sibling transmission/disequilibrium test (sTDT) to identify SNPs that had significant allele-frequency differences in cases versus unaffected siblings, adjusted for age and sex. For each SNP, odds ratios (ORs), 95% confidence intervals (CIs) and p-values were calculated. There were 1,862 SNPs associated with PD in tier 1 at p&lt;0.01.</p>
<a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2"></a><p>In Mayo tier 2a, genotypes and analyses for the 1,862 SNPS selected in tier 1 and for 311 genomic controls were measured in 332 case-unrelated control pairs. Genotyping call rates&gt;80% and HWE p-values&gt;0.001 were achieved for 1,793 SNPs. In Mayo tier 2b, 975 SNPs were selected for further testing with biological or other reasoning (e.g. significant effects on subgroup analyses) regarding susceptibility to PD. Of these, genotyping call rates&gt;80% and HWE p-values&gt;0.001 in controls were achieved for 941 SNPs and ORs, 95% CIs and p-values were calculated. The study is described in more details elsewhere <a href="#pone.0000196-Maraganore1">[5]</a>. SNPs, alleles, case and control allele frequencies, ORs, 95% CIs and p-values are available online.</p>
<a id="article1.body1.sec2.sec1.p3" name="article1.body1.sec2.sec1.p3"></a><p>Fung et al. <a href="#pone.0000196-Fung1">[12]</a> performed a genome-wide association study sponsored by National Institute of Neurological Disorders and Stroke (NINDS) where genotyping was performed on 408,803 unique SNPs combining the Illumina Infinium I and HumanHap300 platforms. The investigators undertook a one-stage genome-wide association study in 276 patients with PD and 276 neurologically normal controls. The samples used for this study were derived from the NINDS Neurogenetics repository hosted by the Coriell Institute for Medical research. For the 408,803 SNPs studied, the genotype call rate was greater that 99% for 395,275 SNPs and greater that 95% for 406,312 SNPs. The HWE p value was &gt;0.001 for 395,493 SNPs. The study is described in more details elsewhere <a href="#pone.0000196-Fung1">[12]</a>. Raw data are publicly available online at the Coriell Institute website.</p>


<h4>Genetic models and effect sizes</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>For consistency, all ORs were computed based on the major vs. minor allele contrast, and assignment of minor allele status is based on the allele frequencies of the control samples in the NINDS study.</p>
<a id="article1.body1.sec2.sec2.p2" name="article1.body1.sec2.sec2.p2"></a><p>The Mayo data were originally analyzed using a log-additive model with trend adjusted for age and sex. The OR and 95% CIs from this model were used in order to calculate the natural logarithms of the OR and the standard error of the natural logarithm of the OR for each gene variant. The standard error is given by the difference of the natural logarithms of the upper and lower boundary of the 95% CI, divided by 3.92. Both Mayo tier 1 and tier 2 used matched designs.</p>
<a id="article1.body1.sec2.sec2.p3" name="article1.body1.sec2.sec2.p3"></a><p>The NINDS study examined various types of genetic contrasts including recessive, dominant, and additive models (linear additive, as opposed to log-additive used in the Mayo study) and also provided raw data on alleles for each examined SNP. We calculated the natural logarithm of the allele-based OR and the standard error of the natural logarithm of the OR from the counts of alleles given in cases and controls in the NINDS database. The allele-based OR is practically equivalent to the log-additive model with consideration of trend.</p>


<h4>Meta-analysis: statistical methods</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>The natural logarithms of the OR estimates were combined to estimate a summary OR using fixed <a href="#pone.0000196-Pettiti1">[13]</a> and random effects models<a href="#pone.0000196-Fleiss1">[14]</a> using inverse variance calculations. In fixed effects models, the true effect of risk allele is assumed to be the same value in each dataset, whereas in random effects models the risk allele effects for the individual datasets are assumed to vary around some overall average effect. If var(f) is the variance of each effect (here, natural logarithm of odds ratio) in a study and var(r) is the random effects variance, then in fixed effects calculations, each study is weighted by 1/var(f), while in random effects calculations each study is weighted by 1/[var(f)+var(r)]. Therefore, random effects approach is generally considered more conservative, yielding wider confidence intervals. Between-dataset heterogeneity was quantified using the I<sup>2</sup> metric for inconsistency <a href="#pone.0000196-Higgins1">[15]</a> and its statistical significance was tested with the chi-square distributed Q statistic <a href="#pone.0000196-Lau1">[16]</a>. I<sup>2</sup> is provided by the ratio of (Q-df)/Q, where df = the number of degrees of freedom (one less than the number of combined datasets); it is considered large for values above 50% and Q is considered statistically significant for p&lt;0.10 <a href="#pone.0000196-Higgins1">[15]</a>, <a href="#pone.0000196-Lau1">[16]</a>. In the absence of any between-dataset heterogeneity, fixed and random effects estimates coincide.</p>


<h4>Meta-analysis strategies and multiplicity considerations</h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>We considered the following strategies for combining the available datasets. In the two-stage framework, options included enhancement of replication (second-stage) data and enhancement of first-stage data. We also performed joint meta-analyses including all first-stage and second-stage data.</p>
<a id="article1.body1.sec2.sec4.p2" name="article1.body1.sec2.sec4.p2"></a><p>Enhancement of replication data: In this strategy, the Mayo tier 1 are still considered as the first-stage information and the meta-analysis of the Mayo tier 2 and NINDS datasets represent their second-stage independent replication. The number of tested SNPs for which adjustment needs to be made represents those SNPs that have data available in both the Mayo tier 2 and NINDS datasets.</p>
<a id="article1.body1.sec2.sec4.p3" name="article1.body1.sec2.sec4.p3"></a><p>Enhancement of first-stage data: In this strategy, the data from Mayo tier 1 and NINDS datasets were combined by meta-analysis, and a new first-stage with increased power was created. The summary effects derived in the new first-stage that were statistically significant at p&lt;0.05 level (at least by fixed effects) were then examined for replication in the Mayo tier 2 dataset. The number of tested SNPs for which adjustment needs to be made in the enhanced first-stage data is the common SNPs in both Mayo tier 1 and NINDS datasets; in the replication sample (Mayo tier 2), it is those SNPs that have p&lt;0.05 at least by fixed effects in the combined Mayo tier 1 and NINDS datasets and have also been assessed in the Mayo tier 2 dataset.</p>
<a id="article1.body1.sec2.sec4.p4" name="article1.body1.sec2.sec4.p4"></a><p>Joint analysis: In this strategy, we jointly meta-analyzed all three databases (Mayo tier 1, Mayo tier 2, NINDS) to obtain summary effects. This strategy may be applied to all SNPs where data are available in all three datasets; however, it is improper to adjust the results for the number of common SNPs across all three databases, because the selection of SNPs in the Mayo tier 2 dataset is not random. This strategy combines eventually all three databases, but as a first step the Mayo tier 1 and NINDS data are meta-analyzed, and then only those SNPs that still have p&lt;0.05 at least with fixed effects are then considered for inclusion of the Mayo tier 2 data. If x SNPs have p&lt;0.05 based on Mayo tier 1 and NINDS combined, and y of them are also tested in Mayo tier 2, the p-value may be adjusted for the number of SNPs that have data available in both Mayo tier 1 and NINDS multiplied by the fraction x/y. This is approximately correct, if the x SNPs can be considered a random sample of the y SNPs, an assumption which we tested by comparing the distribution of p-values in the x selected versus the y-x non-selected SNPs according to a Wilcoxon rank-sum test (p = 0.34).</p>
<a id="article1.body1.sec2.sec4.p5" name="article1.body1.sec2.sec4.p5"></a><p>All the above corrections for multiplicity of comparisons are conservative and assume that the tested SNPs are independent, while this is an oversimplification. However, they provide a starting point for considering the extent of multiple comparisons involved.</p>
<a id="article1.body1.sec2.sec4.p6" name="article1.body1.sec2.sec4.p6"></a><p>Analyses were performed in STATA 8.2 (College Station, TX). P-values are two-tailed. For SNPs that were selected by more than one strategy and that pertained to a specific gene, we perused Entrez Gene (paying attention to the Process listed in Gene Ontology) and also queried PubMed using the gene name and Parkinson's disease, in order to identify if there is any hint of biological plausibility or evidence relating these genes specifically to Parkinson's disease.</p>

</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Results"></a><h3>Results</h3>
<h4>Strategy of enhancement of the replication data</h4>
<a id="article1.body1.sec3.sec1.p1" name="article1.body1.sec3.sec1.p1"></a><p>A total of 572 SNPs had data available both in the Mayo tier 2 and NINDS datasets (including 45 that were simply used as genomic controls in Mayo tier 2 without being tested at the Mayo tier 1 dataset). Meta-analysis of the Mayo tier 2 and NINDS datasets showed that 38 SNPs were significant at p&lt;0.05 with fixed effects and 26 remained significant at p&lt;0.05 using random effect models (<a href="#pone-0000196-t001">Table 1</a>). Fourteen out of these 26 SNPs were also significant at p&lt;0.01 in Mayo tier 1, but 4 of the significant effects were in the opposite direction compared to the summary ORs obtained by the meta-analysis of the Mayo tier 2 and NINDS datasets; the remaining 10 SNPs may warrant further consideration. There were also 3 SNPs that were used as genomic controls in Mayo tier 2 and 9 SNPs that had not reached the p&lt;0.01 level of significance in Mayo tier 1, but they were included in Mayo tier 2b (biological or other reasoning). Two of these SNPs had genetic effects in the same direction as in tier 1. These 2 SNPs may also warrant further consideration. None of the associations seen in the enhanced replication data would remain formally significant after adjustment for 527 (572–45) comparisons, since the lowest p-value was 0.005.</p>
<div class="figure" id="pone-0000196-t001"><div class="img"><a name="pone-0000196-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000196.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000196" data-uri="info:doi/10.1371/journal.pone.0000196.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000196.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000196.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000196.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000196.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000196.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000196.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000196.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000196.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000196.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>Significant summary ORs (p&lt;0.05 at least with fixed effects analyses) and 95% confidence intervals in meta-analysis of Mayo tier 2 and NINDS datasets.</span></strong></p><span>doi:10.1371/journal.pone.0000196.t001</span></div>

<h4>Strategy of enhancement of the first-stage data</h4>
<a id="article1.body1.sec3.sec2.p1" name="article1.body1.sec3.sec2.p1"></a><p>The Mayo tier 1 and NINDS datasets shared 32,192 common SNPs. Meta-analyses with fixed effects models for these SNPs, gave 1,503 significant associations at p&lt;0.05. Of these 1,503 SNPs, 173 had been tested also in Mayo tier 2. Eight of these SNPs had also p&lt;0.05 in the Mayo tier 2 dataset and five of them (rs1000291 [chromosome 3], rs2241743 and rs2313982 [chromosome 4], rs3018626 [chromosome 11] and rs2282048 [chromosome13]) had the same direction of effect and they would warrant further consideration based on this strategy (<a href="#pone-0000196-t002">Table 2</a>). Three of the 5 SNPs (rs1000291, rs2241743 and rs3018626) had also been identified with the enhancement of replication data strategy. The lowest p-value in the Mayo tier 2 dataset for an association with the same direction of effect in both stages was 0.0015 (for rs2313982), which would not be formally significant after adjusting for 173 comparisons; this SNP was one that was identified as a candidate SNP in the original Mayo genome-wide association study <a href="#pone.0000196-Maraganore1">[5]</a>, but was not replicated in the subsequent large-scale replication effort <a href="#pone.0000196-Elbaz1">[6]</a>. The SNP rs6050372 in chromosome 20 had a very low p-value (0.000014) in the Mayo tier 1 and NINDS datasets combined, but the observed effect was significant (p = 0.0148) in the opposite direction upon replication in Mayo tier 2.</p>
<div class="figure" id="pone-0000196-t002"><div class="img"><a name="pone-0000196-t002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000196.t002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000196" data-uri="info:doi/10.1371/journal.pone.0000196.t002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000196.t002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000196.t002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000196.t002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000196.t002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000196.t002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000196.t002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000196.t002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000196.t002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000196.t002.TIF"></span>)
                </a></li></ul></div><p><strong>Table 2.  <span>Significant associations (at least with fixed effects) from meta-analysis of Mayo tier 1 and NINDS databases that were also considered in Mayo tier 2.</span></strong></p><span>doi:10.1371/journal.pone.0000196.t002</span></div>

<h4>Joint analysis of all three datasets</h4>
<a id="article1.body1.sec3.sec3.p1" name="article1.body1.sec3.sec3.p1"></a><p>There were 527 SNPs with data available in all three datasets. Of those, 102 SNPs were statistically significant at p&lt;0.05 using fixed effects and 49 were significant also by random effects (<a href="#pone-0000196-t003">Table 3</a>). The number of statistically significant results seems large, but this is spurious. Mayo tier 2 testing was guided by the Mayo tier 1 data.</p>
<div class="figure" id="pone-0000196-t003"><div class="img"><a name="pone-0000196-t003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000196.t003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000196" data-uri="info:doi/10.1371/journal.pone.0000196.t003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000196.t003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000196.t003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000196.t003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000196.t003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000196.t003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000196.t003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000196.t003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000196.t003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000196.t003.TIF"></span>)
                </a></li></ul></div><p><strong>Table 3.  <span>Significant summary ORs (both in fixed and random effects analyses) and 95% confidence intervals computed from meta-analysis of Mayo tier 1, Mayo tier 2 and NINDS datasets.</span></strong></p><span>doi:10.1371/journal.pone.0000196.t003</span></div><a id="article1.body1.sec3.sec3.p2" name="article1.body1.sec3.sec3.p2"></a><p>We limited further the joint analysis to those SNPs where not only data were available in all three datasets, but also had statistically significant results from a first-step meta-analysis of the Mayo tier 1 and NINDS datasets (p&lt;0.05 at least by fixed effects). As stated above in the strategy of enhancement of first-stage data, there were 173 such SNPs. Of those, 72 SNPs were found to be statistically significant at p&lt;0.05 by fixed effects when the three databases were jointly analyzed. Thirty-nine of these SNPs had p&lt;0.05 by random effects models as well (<a href="#pone-0000196-t003">Table 3</a>). None of the associations would remain significant after adjusting p-values by a factor of 32,192×(173/1,503) = 3,705 (see <a href="#s2">methods</a>). The SNPs with the lowest p-values were rs1000291 and rs1865997 (p = 0.00010 and p = 0.00021 respectively, using random effects calculations). Both SNPs were significant at p&lt;0.05 in the enhancement of replication strategy. Eight more SNPs were among those that warranted further consideration based on the strategy of enhancement of replication data. SNPs rs1000291 had p&lt;0.05 also in the strategy of enhancement of first-stage data.</p>


<h4>SNPs selected in two or more strategies and other evidence</h4>
<a id="article1.body1.sec3.sec4.p1" name="article1.body1.sec3.sec4.p1"></a><p>
               <a href="#pone-0000196-g001">Figure 1</a> shows the results of meta-analyses using the joint analysis approach for 6 SNPs that had p&lt;0.05 according to at least 2 of the three strategies that we employed. Three SNPs (rs1000291 on chromosome 3, rs2241743 on chromosome 4 and rs3018626 on chromosome 11) were selected by all three strategies; the first two had absolutely no between-dataset heterogeneity (I<sup>2</sup> = 0) and the third had minimal between-dataset heterogeneity (I<sup>2</sup> = 15%). There was larger, but still not formally statistically significant heterogeneity for the other 3 SNPs that were selected by 2 of the 3 strategies (I<sup>2</sup> ranging between 24% and 49%). No SNP had p&lt;0.05 and the same direction of effect separately in all 3 datasets.</p>
<div class="figure" id="pone-0000196-g001"><div class="img"><a name="pone-0000196-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000196.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000196" data-uri="info:doi/10.1371/journal.pone.0000196.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000196.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000196.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000196.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000196.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000196.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000196.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000196.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000196.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000196.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1. </strong></p><a id="article1.body1.sec3.sec4.fig1.caption1.p1" name="article1.body1.sec3.sec4.fig1.caption1.p1"></a><p>Meta-analyses of the three datasets for the 6 single nucleotide polymorphisms that were selected (p&lt;0.05 unadjusted for multiple comparisons) with at least two of the three strategies. For each polymorphism the forest plot shows the odds ratio and 95% confidence interval for each dataset as well as the summary odds ratio and 95% confidence intervals by random effects calculations. Also shown is the p-value for the summary effect and the I-squared statistic for between-dataset heterogeneity.</p>
<span>doi:10.1371/journal.pone.0000196.g001</span></div><a id="article1.body1.sec3.sec4.p2" name="article1.body1.sec3.sec4.p2"></a><p>According to Entrez Gene and PubMed, we found some hints for potential biological plausibility for the <em>UNC5C</em> gene where the rs2241743 polymorphism is located. The gene product belongs to the <em>UNC-5</em> family of netrin receptors. Netrins are secreted proteins that direct axon extension and cell migration during neural development. <em>UNC5C</em> maps to the alpha-synuclein locus of chromosome 4 <a href="#pone.0000196-Ackerman1">[17]</a>, where the <em>SCNA</em> gene is an already well-known Parkinson's disease susceptibility gene. It is tempting to speculate whether the axon guidance pathway may have broader pathogenetic implications for Parkinson's disease, as netrin and netrin receptors have important roles for nigral dopaminergic neurons <a href="#pone.0000196-Livesey1">[18]</a>, <a href="#pone.0000196-Lin1">[19]</a>. Parenthetically, the most significant finding from the study of Maraganore et al. <a href="#pone.0000196-Maraganore1">[5]</a> was for a SNP within <em>SEMA5A</em>, another axon guidance pathway gene (although that finding has not been independently replicated thus far).</p>

</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec4.p1" name="article1.body1.sec4.p1"></a><p>We show using empirical data how meta-analysis can be used to combine information from genome-wide datasets. Meta-analysis is a well-established method to synthesize results and draw conclusions from different studies for a set of related research hypotheses and it has the greater citation impact in the health sciences literature compared to other study designs <a href="#pone.0000196-Patsopoulos1">[20]</a>. When performed appropriately, meta-analysis may enhance the precision of the estimates of the effects of risk alleles, leading to reduced probability of false negative results. The increased availability of information can also lead to rejection of null hypotheses at lower levels of type I error, thus reducing the false discovery rate <a href="#pone.0000196-Wacholder1">[21]</a>. In the field of Human Genome Epidemiology, meta-analyses of gene-disease association studies to–date have addressed typically one or a few postulated associations at a time and even large-scale overviews of many meta-analyses have addressed a few dozens of associations at the most <a href="#pone.0000196-Ioannidis3">[10]</a>, <a href="#pone.0000196-Lohmueller1">[11]</a>, <a href="#pone.0000196-Ioannidis4">[22]</a>. Genome-wide association analyses provide an opportunity to conduct many thousands of SNP-specific meta-analyses concurrently. This may yield some interesting results that are worth pursuing further, as in our datasets. However, the multiplicity of comparisons has to be factored to avoid making exaggerated claims about the promising SNPs that emerge from such meta-analyses. The synthesis and interpretation of gene-disease associations should be cautious, especially when weak associations are considered. Misclassification, confounding (population stratification) and selective reporting may lead to spurious findings <a href="#pone.0000196-Little1">[23]</a>. Biological plausibility and other external evidence may be considered as well to interpret the results of the meta-analysis. Here, the identification of a polymorphism is a axon guidance pathway gene is intriguing, but certainly requires independent corroboration and replication before any strong claim can be made.</p>
<a id="article1.body1.sec4.p2" name="article1.body1.sec4.p2"></a><p>Our empirical evaluation also revealed several issues that need to be considered in future efforts. First, when different genotyping platforms are used, as in our datasets, the overlap of genetic markers may be suboptimal. The Mayo and NINDS platforms had only modest overlap (only approximately 16% of the Mayo tier 1 dataset SNPs also had data in the NINDS dataset). This is expected to result in large loss of genomic coverage, even if the coverage of each platform is very good <a href="#pone.0000196-Barrett1">[24]</a>. One may consider also juxtaposing and combining data from SNPs that are in very strong linkage disequilibrium or may even consider genic approaches to the data <a href="#pone.0000196-Jorgenson1">[25]</a>.</p>
<a id="article1.body1.sec4.p3" name="article1.body1.sec4.p3"></a><p>Second, meta-analyses may lead to spurious or heterogeneous results if the definitions of disease phenotypes and controls are different across the combined datasets. For Parkinson disease, for example, there are many different accepted clinical definitions, but hopefully they do not lead to major discrepancies in diagnosis. Population stratification may also lead to spurious or heterogeneous results in a meta-analysis, if some of the combined studies are affected. In our application, population stratification had been more thoroughly addressed in the Mayo data (family-based designs and genomic controls) than in the NINDS dataset.</p>
<a id="article1.body1.sec4.p4" name="article1.body1.sec4.p4"></a><p>Third, given the vast number of analyses performed, the threshold for claiming formal statistical significance needs careful consideration. We have used conservative adjustments, but these may be warranted so as to minimize undue emphasis on potentially false-positive results. Nevertheless, a number of genetic variants identified with either of the three strategies as potentially important with unadjusted p-values may warrant further consideration and replication efforts. This may be particularly enticing for the variants proposed with 2 different strategies or even all 3 strategies.</p>
<a id="article1.body1.sec4.p5" name="article1.body1.sec4.p5"></a><p>Of the three strategies that we examined, the joint analysis has the best power. This has been demonstrated already by Skol et al. in the setting of comparing two-stage versus joint analyses for genome-wide data for the typical fractions of SNPs being tested in the second stage <a href="#pone.0000196-Skol1">[26]</a>. The gain in power has always been considered the traditional advantage of meta-analysis in all disciplines where this methodology has been adopted <a href="#pone.0000196-Cohn1">[27]</a>, <a href="#pone.0000196-Hedges1">[28]</a>. This is true however primarily when there is no large between-study heterogeneity <a href="#pone.0000196-Cohn1">[27]</a>. At the same time, heterogeneity testing may also give us some useful insights and this may become more important when many datasets are available <a href="#pone.0000196-Lau2">[29]</a>. In our empirical evaluation, the SNPs that were proposed by each strategy typically had no measurable or minimal between-dataset heterogeneity.</p>
<a id="article1.body1.sec4.p6" name="article1.body1.sec4.p6"></a><p>Traditionally, publication bias has been a major threat to the validity of meta-analysis results. The public availability of databases from genome-wide association studies provides an excellent setting where the problem of publication bias can be minimized or even extinguished <a href="#pone.0000196-Cardon1">[8]</a>, <a href="#pone.0000196-dbGaP1">[9]</a>. This provides an additional argument in favor of making these data-rich experiments publicly available.</p>
<a id="article1.body1.sec4.p7" name="article1.body1.sec4.p7"></a><p>Some genetic effects for common variants may be small and readily detectable with genome-wide association studies of very small sample size. Age-related macular degeneration provides one such successful example <a href="#pone.0000196-Klein1">[30]</a>, <a href="#pone.0000196-Daiger1">[31]</a>. However, other genetic variants currently emerging from massive-testing approaches seem to have small or even very small genetic effects <a href="#pone.0000196-Smyth1">[32]</a>, <a href="#pone.0000196-Herbert1">[33]</a>. This latter scenario may be far more frequent and even small ORs would still be important to identify for variants that have a considerable frequency in the population. This suggests that there should be an a priori consideration that meta-analysis should be performed on all genome-wide association studies conducted on the same disease. Investigators in the field of type 2 diabetes have already anticipated such a prospective meta-analysis through the IGWANA project <a href="#pone.0000196-Accessed1">[34]</a>. This concept needs to be extended across diverse fields of human genome epidemiology. Meta-analyses may be updated also in a cumulative fashion, when new data appear <a href="#pone.0000196-Lau3">[35]</a>, <a href="#pone.0000196-Ioannidis5">[36]</a>. Ideally, different teams of investigators should also discuss in advance the plans for a meta-analysis. This may entail agreeing on using common genotyping platforms and/or creating plans for enhancing the consistency of the databases across different studies.</p>
</div>



<div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: JI DM EE. Performed the experiments: DM. Analyzed the data: JI EE. Wrote the paper: JI DM EE.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0000196-Hirschhorn1" id="pone.0000196-Hirschhorn1"></a>Hirschhorn JN, Daly MJ (2005) Genome-wide association studies for common diseases and complex traits. Nat Rev Genet  6: 95–108.  <ul class="find" data-citedArticleID="929102" data-doi="10.1038/nrg1521"><li><a href="http://dx.doi.org/10.1038/nrg1521" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genome-wide+association+studies+for+common+diseases+and+complex+traits." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genome-wide+association+studies+for+common+diseases+and+complex+traits.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0000196-Wang1" id="pone.0000196-Wang1"></a>Wang WY, Barratt BJ, Clayton DG, Todd JA (2005) Genome-wide association studies: theoretical and practical concerns. Nat Rev Genet  6: 109–18.  <ul class="find" data-citedArticleID="929146" data-doi="10.1038/nrg1522"><li><a href="http://dx.doi.org/10.1038/nrg1522" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genome-wide+association+studies%3A+theoretical+and+practical+concerns." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genome-wide+association+studies%3A+theoretical+and+practical+concerns.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0000196-Todd1" id="pone.0000196-Todd1"></a>Todd JA (2006) Statistical false positive or true disease pathway? Nat Genet  38: 731–3.  <ul class="find" data-citedArticleID="929142" data-doi="10.1038/ng0706-731"><li><a href="http://dx.doi.org/10.1038/ng0706-731" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Statistical+false+positive+or+true+disease+pathway%3F" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Statistical+false+positive+or+true+disease+pathway%3F%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0000196-Ioannidis1" id="pone.0000196-Ioannidis1"></a>Ioannidis JP (2003) Genetic associations: false or true? Trends Mol Med  9: 135–8.  <ul class="find" data-citedArticleID="929104" data-doi="10.1016/s1471-4914(03)00030-3"><li><a href="http://dx.doi.org/10.1016/s1471-4914(03)00030-3" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genetic+associations%3A+false+or+true%3F" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genetic+associations%3A+false+or+true%3F%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0000196-Maraganore1" id="pone.0000196-Maraganore1"></a>Maraganore DM, de Andrade M, Lesnick TG, Strain KJ, Farrer MJ, et al.  (2005) High-resolution whole-genome association study of Parkinson disease. Am J Hum Genet  77: 685–93.  <ul class="find" data-citedArticleID="929132" data-doi="10.1086/496902"><li><a href="http://dx.doi.org/10.1086/496902" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=High-resolution+whole-genome+association+study+of+Parkinson+disease." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22High-resolution+whole-genome+association+study+of+Parkinson+disease.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0000196-Elbaz1" id="pone.0000196-Elbaz1"></a>Elbaz A, Nelson LM, Payami H, Ioannidis JP, Fiske BK, et al.  (2006) Lack of replication of thirteen single-nucleotide polymorphisms implicated in Parkinson's disease: a large-scale international study. Lancet Neurol  5: 917–23.  <ul class="find" data-citedArticleID="929090" data-doi="10.1016/s1474-4422(06)70579-8"><li><a href="http://dx.doi.org/10.1016/s1474-4422(06)70579-8" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Lack+of+replication+of+thirteen+single-nucleotide+polymorphisms+implicated+in+Parkinson%27s+disease%3A+a+large-scale+international+study." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Lack+of+replication+of+thirteen+single-nucleotide+polymorphisms+implicated+in+Parkinson%27s+disease%3A+a+large-scale+international+study.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0000196-Ioannidis2" id="pone.0000196-Ioannidis2"></a>Ioannidis JP, Trikalinos TA, Khoury MJ (2006) Implications of small effect sizes of individual genetic variants on the design and interpretation of genetic association studies of complex diseases. Am J Epidemiol  164: 609–14.  <ul class="find" data-citedArticleID="929106" data-doi="10.1093/aje/kwj259"><li><a href="http://dx.doi.org/10.1093/aje/kwj259" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Implications+of+small+effect+sizes+of+individual+genetic+variants+on+the+design+and+interpretation+of+genetic+association+studies+of+complex+diseases." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Implications+of+small+effect+sizes+of+individual+genetic+variants+on+the+design+and+interpretation+of+genetic+association+studies+of+complex+diseases.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0000196-Cardon1" id="pone.0000196-Cardon1"></a>Cardon LR (2006) Genetics. Delivering new disease genes. Science  314: 1403–5.  <ul class="find" data-citedArticleID="929084" data-doi="10.1126/science.1136668"><li><a href="http://dx.doi.org/10.1126/science.1136668" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genetics.+Delivering+new+disease+genes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genetics.+Delivering+new+disease+genes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0000196-dbGaP1" id="pone.0000196-dbGaP1"></a>  dbGaP database accessed at <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?">http://www.ncbi.nlm.nih.gov/entrez/query​.fcgi?</a>db = gap. <ul class="find-nolinks"></ul></li><li><span class="label">10.
              </span><a name="pone.0000196-Ioannidis3" id="pone.0000196-Ioannidis3"></a>Ioannidis JP, Ntzani EE, Trikalinos TA, Contopoulos-Ioannidis DG (2001) Replication validity of genetic association studies. Nat Genet  29: 306–9.  <ul class="find" data-citedArticleID="929108" data-doi="10.1038/ng749"><li><a href="http://dx.doi.org/10.1038/ng749" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Replication+validity+of+genetic+association+studies." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Replication+validity+of+genetic+association+studies.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0000196-Lohmueller1" id="pone.0000196-Lohmueller1"></a>Lohmueller KE, Pearce CL, Pike M, Lander ES, Hirschhorn JN (2003) Meta-analysis of genetic association studies supports a contribution of common variants to susceptibility to common disease. Nat Genet  33: 177–82.  <ul class="find" data-citedArticleID="929130" data-doi="10.1038/ng1071"><li><a href="http://dx.doi.org/10.1038/ng1071" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Meta-analysis+of+genetic+association+studies+supports+a+contribution+of+common+variants+to+susceptibility+to+common+disease." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Meta-analysis+of+genetic+association+studies+supports+a+contribution+of+common+variants+to+susceptibility+to+common+disease.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0000196-Fung1" id="pone.0000196-Fung1"></a>Fung HC, Scholz S, Matarin M, Simon-Sanchez J, Hernandez D, et al.  (2006) Genome-wide genotyping in Parkinson's disease and neurologically normal controls: first stage analysis and public release of data. Lancet Neurol  5: 911–6.  <ul class="find" data-citedArticleID="929094" data-doi="10.1016/s1474-4422(06)70578-6"><li><a href="http://dx.doi.org/10.1016/s1474-4422(06)70578-6" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genome-wide+genotyping+in+Parkinson%27s+disease+and+neurologically+normal+controls%3A+first+stage+analysis+and+public+release+of+data." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genome-wide+genotyping+in+Parkinson%27s+disease+and+neurologically+normal+controls%3A+first+stage+analysis+and+public+release+of+data.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0000196-Pettiti1" id="pone.0000196-Pettiti1"></a>Pettiti D (1999) Meta-analysis, decision analysis, and cost-effectiveness analysis. Oxford University Press, New York, 2nd edition.   <ul class="find-nolinks"></ul></li><li><span class="label">14.
              </span><a name="pone.0000196-Fleiss1" id="pone.0000196-Fleiss1"></a>Fleiss JL (1993) The statistical basis of meta-analysis. Stat Methods Med Res  2: 121–45.  <ul class="find" data-citedArticleID="929092" data-doi="10.1177/096228029300200202"><li><a href="http://dx.doi.org/10.1177/096228029300200202" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+statistical+basis+of+meta-analysis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+statistical+basis+of+meta-analysis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0000196-Higgins1" id="pone.0000196-Higgins1"></a>Higgins JP, Thompson SG, Deeks JJ, Altman DG (2003) Measuring inconsistency in meta-analyses. BMJ  327: 557–60.  <ul class="find" data-citedArticleID="929100" data-doi="10.1136/bmj.327.7414.557"><li><a href="http://dx.doi.org/10.1136/bmj.327.7414.557" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Measuring+inconsistency+in+meta-analyses." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Measuring+inconsistency+in+meta-analyses.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0000196-Lau1" id="pone.0000196-Lau1"></a>Lau J, Ioannidis JP, Schmid CH (1997) Quantitative synthesis in systematic reviews. Ann Intern Med  127: 820–6.  <ul class="find" data-citedArticleID="929118" data-doi="10.7326/0003-4819-127-9-199711010-00008"><li><a href="http://dx.doi.org/10.7326/0003-4819-127-9-199711010-00008" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Quantitative+synthesis+in+systematic+reviews." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Quantitative+synthesis+in+systematic+reviews.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0000196-Ackerman1" id="pone.0000196-Ackerman1"></a>Ackerman SL, Knowles BB (1998) Cloning and mapping of the UNC5C gene to human chromosome 4q21–q23. Genomics  52: 205–8.  <ul class="find" data-citedArticleID="929080" data-doi="10.1006/geno.1998.5425"><li><a href="http://dx.doi.org/10.1006/geno.1998.5425" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Cloning+and+mapping+of+the+UNC5C+gene+to+human+chromosome+4q21%26%23x2013%3Bq23." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Cloning+and+mapping+of+the+UNC5C+gene+to+human+chromosome+4q21%26%23x2013%3Bq23.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0000196-Livesey1" id="pone.0000196-Livesey1"></a>Livesey FJ, Hunt SP (1997) Netrin and netrin receptor expression in the embryonic mammalian nervous system suggests roles in retinal, striatal, nigral, and cerebellar development. Mol Cell Neurosci  8: 417–29.  <ul class="find" data-citedArticleID="929128" data-doi="10.1006/mcne.1997.0598"><li><a href="http://dx.doi.org/10.1006/mcne.1997.0598" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Netrin+and+netrin+receptor+expression+in+the+embryonic+mammalian+nervous+system+suggests+roles+in+retinal%2C+striatal%2C+nigral%2C+and+cerebellar+development." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Netrin+and+netrin+receptor+expression+in+the+embryonic+mammalian+nervous+system+suggests+roles+in+retinal%2C+striatal%2C+nigral%2C+and+cerebellar+development.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0000196-Lin1" id="pone.0000196-Lin1"></a>Lin L, Isacson O (2006) Axonal growth regulation of fetal and embryonic stem cell-derived dopaminergic neurons by Netrin-1 and Slits. Stem Cells  24: 2504–13.  <ul class="find" data-citedArticleID="929124" data-doi="10.1634/stemcells.2006-0119"><li><a href="http://dx.doi.org/10.1634/stemcells.2006-0119" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Axonal+growth+regulation+of+fetal+and+embryonic+stem+cell-derived+dopaminergic+neurons+by+Netrin-1+and+Slits." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Axonal+growth+regulation+of+fetal+and+embryonic+stem+cell-derived+dopaminergic+neurons+by+Netrin-1+and+Slits.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0000196-Patsopoulos1" id="pone.0000196-Patsopoulos1"></a>Patsopoulos NA, Analatos AA, Ioannidis JP (2005) Relative citation impact of various study designs in the health sciences. JAMA  293: 2362–6.  <ul class="find" data-citedArticleID="929134" data-doi="10.1001/jama.293.19.2362"><li><a href="http://dx.doi.org/10.1001/jama.293.19.2362" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Relative+citation+impact+of+various+study+designs+in+the+health+sciences." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Relative+citation+impact+of+various+study+designs+in+the+health+sciences.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0000196-Wacholder1" id="pone.0000196-Wacholder1"></a>Wacholder S, Chanock S, Garcia-Closas M, El Ghormli L, Rothman N (2004) Assessing the probability that a positive report is false: an approach for molecular epidemiology studies. J Natl Cancer Inst  96: 434–42.  <ul class="find" data-citedArticleID="929144" data-doi="10.1093/jnci/djh075"><li><a href="http://dx.doi.org/10.1093/jnci/djh075" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Assessing+the+probability+that+a+positive+report+is+false%3A+an+approach+for+molecular+epidemiology+studies." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Assessing+the+probability+that+a+positive+report+is+false%3A+an+approach+for+molecular+epidemiology+studies.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0000196-Ioannidis4" id="pone.0000196-Ioannidis4"></a>Ioannidis JP, Trikalinos TA, Ntzani EE, Contopoulos-Ioannidis DG (2003) Genetic associations in large versus small studies: an empirical assessment. Lancet  361: 567–71.  <ul class="find" data-citedArticleID="929110" data-doi="10.1016/s0140-6736(03)12516-0"><li><a href="http://dx.doi.org/10.1016/s0140-6736(03)12516-0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genetic+associations+in+large+versus+small+studies%3A+an+empirical+assessment." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genetic+associations+in+large+versus+small+studies%3A+an+empirical+assessment.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0000196-Little1" id="pone.0000196-Little1"></a> Little J, Higgins J, editors. The HuGENet HUGE Review Handbook version 1.0.   Accessed at: <a href="http://www.cdc.gov/genomics/hugenet/reviews/guidelines.htm">http://www.cdc.gov/genomics/hugenet/revi​ews/guidelines.htm</a>
               .  <ul class="find-nolinks"></ul></li><li><span class="label">24.
              </span><a name="pone.0000196-Barrett1" id="pone.0000196-Barrett1"></a>Barrett JC, Cardon LR (2006) Evaluating coverage of genome-wide association studies. Nat Genet  38: 659–62.  <ul class="find" data-citedArticleID="929082" data-doi="10.1038/ng1801"><li><a href="http://dx.doi.org/10.1038/ng1801" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evaluating+coverage+of+genome-wide+association+studies." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evaluating+coverage+of+genome-wide+association+studies.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0000196-Jorgenson1" id="pone.0000196-Jorgenson1"></a>Jorgenson E, Witte JS (2006) A gene-centric approach to genome-wide association studies. Nat Rev Genet  7: 885–91.  <ul class="find" data-citedArticleID="929114" data-doi="10.1038/nrg1962"><li><a href="http://dx.doi.org/10.1038/nrg1962" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+gene-centric+approach+to+genome-wide+association+studies." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+gene-centric+approach+to+genome-wide+association+studies.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0000196-Skol1" id="pone.0000196-Skol1"></a>Skol AD, Scott LJ, Abecasis GR, Boehnke M (2006) Joint analysis is more efficient than replication-based analysis for two-stage genome-wide association studies. Nat Genet  38: 209–13.  <ul class="find" data-citedArticleID="929138" data-doi="10.1038/ng1706"><li><a href="http://dx.doi.org/10.1038/ng1706" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Joint+analysis+is+more+efficient+than+replication-based+analysis+for+two-stage+genome-wide+association+studies." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Joint+analysis+is+more+efficient+than+replication-based+analysis+for+two-stage+genome-wide+association+studies.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0000196-Cohn1" id="pone.0000196-Cohn1"></a>Cohn LD, Becker BJ (2003) How meta-analysis increases statistical power. Psychol Methods  8: 243–53.  <ul class="find" data-citedArticleID="929086" data-doi="10.1037/1082-989x.8.3.243"><li><a href="http://dx.doi.org/10.1037/1082-989x.8.3.243" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=How+meta-analysis+increases+statistical+power." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22How+meta-analysis+increases+statistical+power.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0000196-Hedges1" id="pone.0000196-Hedges1"></a>Hedges LV, Pigott TD (2001) The power of statistical tests in meta-analysis. Psychol Methods  6: 203–17.  <ul class="find" data-citedArticleID="929096" data-doi="10.1037/1082-989x.6.3.203"><li><a href="http://dx.doi.org/10.1037/1082-989x.6.3.203" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+power+of+statistical+tests+in+meta-analysis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+power+of+statistical+tests+in+meta-analysis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0000196-Lau2" id="pone.0000196-Lau2"></a>Lau J, Ioannidis JP, Schmid CH (1998) Summing up evidence: one answer is not always enough. Lancet  351: 123–7.  <ul class="find" data-citedArticleID="929120" data-doi="10.1016/s0140-6736(97)08468-7"><li><a href="http://dx.doi.org/10.1016/s0140-6736(97)08468-7" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Summing+up+evidence%3A+one+answer+is+not+always+enough." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Summing+up+evidence%3A+one+answer+is+not+always+enough.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0000196-Klein1" id="pone.0000196-Klein1"></a>Klein RJ, Zeiss C, Chew EY, Tsai JY, Sackler RS, et al.  (2005) Complement factor H polymorphism in age-related macular degeneration. Science  308: 385–9.  <ul class="find" data-citedArticleID="929116" data-doi="10.1126/science.1109557"><li><a href="http://dx.doi.org/10.1126/science.1109557" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Complement+factor+H+polymorphism+in+age-related+macular+degeneration." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Complement+factor+H+polymorphism+in+age-related+macular+degeneration.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0000196-Daiger1" id="pone.0000196-Daiger1"></a>Daiger SP (2005) Genetics. Was the Human Genome Project worth the effort? Science  308: 362–4.  <ul class="find" data-citedArticleID="929088" data-doi="10.1126/science.1111655"><li><a href="http://dx.doi.org/10.1126/science.1111655" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genetics.+Was+the+Human+Genome+Project+worth+the+effort%3F" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genetics.+Was+the+Human+Genome+Project+worth+the+effort%3F%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0000196-Smyth1" id="pone.0000196-Smyth1"></a>Smyth DJ, Cooper JD, Bailey R, Field S, Burren O, et al.  (2006) A genome-wide association study of nonsynonymous SNPs identifies a type 1 diabetes locus in the interferon-induced helicase (IFIH1) region. Nat Genet  38: 617–9.  <ul class="find" data-citedArticleID="929140" data-doi="10.1038/ng1800"><li><a href="http://dx.doi.org/10.1038/ng1800" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+genome-wide+association+study+of+nonsynonymous+SNPs+identifies+a+type+1+diabetes+locus+in+the+interferon-induced+helicase+%28IFIH1%29+region." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+genome-wide+association+study+of+nonsynonymous+SNPs+identifies+a+type+1+diabetes+locus+in+the+interferon-induced+helicase+%28IFIH1%29+region.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0000196-Herbert1" id="pone.0000196-Herbert1"></a>Herbert A, Gerry NP, McQueen MB, Heid IM, Pfeufer A, et al.  (2006) A common genetic variant is associated with adult and childhood obesity. Science  312: 279–83.  <ul class="find" data-citedArticleID="929098" data-doi="10.1126/science.1124779"><li><a href="http://dx.doi.org/10.1126/science.1124779" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+common+genetic+variant+is+associated+with+adult+and+childhood+obesity." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+common+genetic+variant+is+associated+with+adult+and+childhood+obesity.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0000196-Accessed1" id="pone.0000196-Accessed1"></a>  Accessed at <a href="http://www.well.ox.ac.uk/mccarthy/Consortia.shtml">http://www.well.ox.ac.uk/mccarthy/Consor​tia.shtml.</a>
               .  <ul class="find-nolinks"></ul></li><li><span class="label">35.
              </span><a name="pone.0000196-Lau3" id="pone.0000196-Lau3"></a>Lau J, Antman EM, Jimenez-Silva J, Kupelnick B, Mosteller F, et al.  (1992) Cumulative meta-analysis of therapeutic trials for myocardial infarction. N Engl J Med  327: 248–54.  <ul class="find" data-citedArticleID="929122" data-doi="10.1056/nejm199207233270406"><li><a href="http://dx.doi.org/10.1056/nejm199207233270406" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Cumulative+meta-analysis+of+therapeutic+trials+for+myocardial+infarction." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Cumulative+meta-analysis+of+therapeutic+trials+for+myocardial+infarction.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">36.
              </span><a name="pone.0000196-Ioannidis5" id="pone.0000196-Ioannidis5"></a>Ioannidis JP, Bernstein J, Boffetta P, Danesh J, Dolan S, et al.  (2005) A network of investigator networks in human genome epidemiology. Am J Epidemiol  162: 302–4.  <ul class="find" data-citedArticleID="929112" data-doi="10.1093/aje/kwi201"><li><a href="http://dx.doi.org/10.1093/aje/kwi201" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+network+of+investigator+networks+in+human+genome+epidemiology." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+network+of+investigator+networks+in+human+genome+epidemiology.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000196.XML" value="202652"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000196.PDF" value="117663"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000196.t001.PNG_L" value="363302"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000196.t001.PNG_M" value="83874"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000196.t001.PNG_S" value="7551"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000196.t001.TIF" value="1939240"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000196.t001.PNG_I" value="54780"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000196.t002.PNG_L" value="151079"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000196.t002.PNG_M" value="55983"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000196.t002.PNG_S" value="2949"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000196.t002.TIF" value="716286"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000196.t002.PNG_I" value="23984"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000196.t003.PNG_L" value="513988"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000196.t003.PNG_M" value="63810"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000196.t003.PNG_S" value="9515"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000196.t003.TIF" value="2761998"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000196.t003.PNG_I" value="88612"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000196.g001.PNG_L" value="37531"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000196.g001.PNG_M" value="18753"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000196.g001.PNG_S" value="1828"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000196.g001.TIF" value="66506"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000196.g001.PNG_I" value="9567"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000196&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000196" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000196&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0000196&volume=&issue=&title=Meta-Analysis in Genome-Wide Association Datasets: Strategies and Application in Parkinson Disease&author_name=Evangelos%20Evangelou%2C%20Demetrius%20M.%20Maraganore%2C%20John%20P.A.%20Ioannidis&start_page=1&end_page=8" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000196" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000196" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0000196" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000196&amp;t=Meta-Analysis%20in%20Genome-Wide%20Association%20Datasets%3A%20Strategies%20and%20Application%20in%20Parkinson%20Disease" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000196&title=Meta-Analysis%20in%20Genome-Wide%20Association%20Datasets%3A%20Strategies%20and%20Application%20in%20Parkinson%20Disease&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000196&amp;title=Meta-Analysis%20in%20Genome-Wide%20Association%20Datasets%3A%20Strategies%20and%20Application%20in%20Parkinson%20Disease" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000196" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0000196&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Meta-Analysis in Genome-Wide Association Datasets: Strategies and Application in Parkinson Disease';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0000196';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Meta-Analysis%20in%20Genome-Wide%20Association%20Datasets%3A%20Strategies%20and%20Application%20in%20Parkinson%20Disease http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000196" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0000196" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">












          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Chromosome+4%22" title="Search for articles in the subject area:'Chromosome 4'"><div class="flagText">Chromosome 4</div></a>
              <div data-categoryid="22417" data-articleid="23230"
                   data-categoryname="Chromosome 4"
                   class="flagImage" title="Flag 'Chromosome 4' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Confidence+intervals%22" title="Search for articles in the subject area:'Confidence intervals'"><div class="flagText">Confidence intervals</div></a>
              <div data-categoryid="32051" data-articleid="23230"
                   data-categoryname="Confidence intervals"
                   class="flagImage" title="Flag 'Confidence intervals' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Genetics+of+disease%22" title="Search for articles in the subject area:'Genetics of disease'"><div class="flagText">Genetics of disease</div></a>
              <div data-categoryid="17953" data-articleid="23230"
                   data-categoryname="Genetics of disease"
                   class="flagImage" title="Flag 'Genetics of disease' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Genome-wide+association+studies%22" title="Search for articles in the subject area:'Genome-wide association studies'"><div class="flagText">Genome-wide association studies</div></a>
              <div data-categoryid="36221" data-articleid="23230"
                   data-categoryname="Genome-wide association studies"
                   class="flagImage" title="Flag 'Genome-wide association studies' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Genomics%22" title="Search for articles in the subject area:'Genomics'"><div class="flagText">Genomics</div></a>
              <div data-categoryid="18221" data-articleid="23230"
                   data-categoryname="Genomics"
                   class="flagImage" title="Flag 'Genomics' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Genotyping%22" title="Search for articles in the subject area:'Genotyping'"><div class="flagText">Genotyping</div></a>
              <div data-categoryid="47691" data-articleid="23230"
                   data-categoryname="Genotyping"
                   class="flagImage" title="Flag 'Genotyping' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Meta-analysis%22" title="Search for articles in the subject area:'Meta-analysis'"><div class="flagText">Meta-analysis</div></a>
              <div data-categoryid="32065" data-articleid="23230"
                   data-categoryname="Meta-analysis"
                   class="flagImage" title="Flag 'Meta-analysis' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Parkinson+disease%22" title="Search for articles in the subject area:'Parkinson disease'"><div class="flagText">Parkinson disease</div></a>
              <div data-categoryid="20751" data-articleid="23230"
                   data-categoryname="Parkinson disease"
                   class="flagImage" title="Flag 'Parkinson disease' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=2823'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=1254'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=1197&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>

<div class="block sidebar-comments">
    <div class="header">
        <h3>Comments</h3>
    </div>
      <p><a href="/annotation/listThread.action?root=3697">Is there enough data?</a><br>Posted by MRCookson</p>
</div>

</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
