

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Evidence for Active Maintenance of Inverted Repeat Structures Identified by a Comparative Genomic Approach</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0000262"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0000262" />

    <meta name="citation_title" content="Evidence for Active Maintenance of Inverted Repeat Structures Identified by a Comparative Genomic Approach"/>
    <meta itemprop="name" content="Evidence for Active Maintenance of Inverted Repeat Structures Identified by a Comparative Genomic Approach"/>

      <meta name="citation_author" content="Guoyan Zhao"/>
            <meta name="citation_author_institution" content="Department of Genetics, Washington University School of Medicine, St. Louis, Missouri, United States of America"/>
      <meta name="citation_author" content="Kuan Y. Chang"/>
            <meta name="citation_author_institution" content="Department of Pathology and Immunology, Washington University School of Medicine, St. Louis, Missouri, United States of America"/>
      <meta name="citation_author" content="Katherine Varley"/>
            <meta name="citation_author_institution" content="Department of Genetics, Washington University School of Medicine, St. Louis, Missouri, United States of America"/>
      <meta name="citation_author" content="Gary D. Stormo"/>
            <meta name="citation_author_institution" content="Department of Genetics, Washington University School of Medicine, St. Louis, Missouri, United States of America"/>

    <meta name="citation_date" content="2007/2/28"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0000262.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e262"/>
    <meta name="citation_issue" content="2"/>
    <meta name="citation_volume" content="2"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Why is the initiation nick site of an AT-rich rolling circle plasmid at the tip of a GC-rich cruciform?; citation_author=R Jin; citation_author=ME Fernandez-Beros; citation_author=RP Novick; citation_journal_title=Embo J; citation_volume=16; citation_number=1; citation_pages=4456-4466; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Cruciform-extruding regulatory element controls cell-specific activity of the tyrosine hydroxylase gene promoter.; citation_author=EL Kim; citation_author=H Peng; citation_author=FM Esparza; citation_author=SZ Maltchenko; citation_author=MK Stachowiak; citation_journal_title=Nucleic Acids Res; citation_volume=26; citation_number=2; citation_pages=1793-1800; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Specific sequences and a hairpin structure in the template strand are required for N4 virion RNA polymerase promoter recognition.; citation_author=MA Glucksmann; citation_author=P Markiewicz; citation_author=C Malone; citation_author=LB Rothman-Denes; citation_journal_title=Cell; citation_volume=70; citation_number=3; citation_pages=491-500; citation_date=1992; " />
      <meta name="citation_reference" content="citation_title=Escherichia coli single-stranded DNA-binding protein is a supercoiled template-dependent transcriptional activator of N4 virion RNA polymerase.; citation_author=P Markiewicz; citation_author=C Malone; citation_author=JW Chase; citation_author=LB Rothman-Denes; citation_journal_title=Genes Dev; citation_volume=6; citation_number=4; citation_pages=2010-2019; citation_date=1992; " />
      <meta name="citation_reference" content="citation_title=Switching of DNA secondary structure in proenkephalin transcriptional regulation.; citation_author=C Spiro; citation_author=CT McMurray; citation_journal_title=J Biol Chem; citation_volume=272; citation_number=5; citation_pages=33145-33152; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Iron-regulatory proteins, iron-responsive elements and ferritin mRNA translation.; citation_author=AM Thomson; citation_author=JT Rogers; citation_author=PJ Leedman; citation_journal_title=Int J Biochem Cell Biol; citation_volume=31; citation_number=6; citation_pages=1139-1152; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Palindrome resolution and recombination in the mammalian germ line.; citation_author=E Akgun; citation_author=J Zahn; citation_author=S Baumes; citation_author=G Brown; citation_author=F Liang; citation_journal_title=Mol Cell Biol; citation_volume=17; citation_number=7; citation_pages=5559-5570; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Inverted DNA repeats: a source of eukaryotic genomic instability.; citation_author=DA Gordenin; citation_author=KS Lobachev; citation_author=NP Degtyareva; citation_author=AL Malkova; citation_author=E Perkins; citation_journal_title=Mol Cell Biol; citation_volume=13; citation_number=8; citation_pages=5315-5322; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=A 140-bp-long palindromic sequence induces double-strand breaks during meiosis in the yeast Saccharomyces cerevisiae.; citation_author=DK Nag; citation_author=A Kurst; citation_journal_title=Genetics; citation_volume=146; citation_number=9; citation_pages=835-847; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=DNA inverted repeats and human disease.; citation_author=JJ Bissler; citation_journal_title=Front Biosci; citation_volume=3; citation_number=10; citation_pages=d408-418; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Abundant gene conversion between arms of palindromes in human and ape Y chromosomes.; citation_author=S Rozen; citation_author=H Skaletsky; citation_author=JD Marszalek; citation_author=PJ Minx; citation_author=HS Cordum; citation_journal_title=Nature; citation_volume=423; citation_number=11; citation_pages=873-876; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Locus encoding a family of small heat shock genes in Caenorhabditis elegans: two genes duplicated to form a 3.8-kilobase inverted repeat.; citation_author=RH Russnak; citation_author=EP Candido; citation_journal_title=Mol Cell Biol; citation_volume=5; citation_number=12; citation_pages=1268-1278; citation_date=1985; " />
      <meta name="citation_reference" content="citation_title=The phylogenetic relationships of Caenorhabditis and other rhabditids.; citation_author=K Kiontke; citation_author=DHA Fitch; citation_number=13; citation_date=2005; citation_publisher=WormBook; " />
      <meta name="citation_reference" content="citation_title=An annotated catalog of inverted repeats of Caenorhabditis elegans chromosomes III and X, with observations concerning odd/even biases and conserved motifs.; citation_author=MD LeBlanc; citation_author=G Aspeslagh; citation_author=NP Buggia; citation_author=BD Dyer; citation_journal_title=Genome Res; citation_volume=10; citation_number=14; citation_pages=1381-1392; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Long inverted repeats in eukaryotic genomes: recombinogenic motifs determine genomic plasticity.; citation_author=Y Wang; citation_author=FC Leung; citation_journal_title=FEBS Lett; citation_volume=580; citation_number=15; citation_pages=1277-1284; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Study of intrachromosomal duplications among the eukaryote genomes.; citation_author=G Achaz; citation_author=P Netter; citation_author=E Coissac; citation_journal_title=Mol Biol Evol; citation_volume=18; citation_number=16; citation_pages=2280-2288; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Gene duplication and gene conversion in the Caenorhabditis elegans genome.; citation_author=C Semple; citation_author=KH Wolfe; citation_journal_title=J Mol Evol; citation_volume=48; citation_number=17; citation_pages=555-564; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=The genome sequence of Caenorhabditis briggsae: a platform for comparative genomics.; citation_author=LD Stein; citation_author=Z Bao; citation_author=D Blasiar; citation_author=T Blumenthal; citation_author=MR Brent; citation_journal_title=PLoS Biol; citation_volume=1; citation_number=18; citation_pages=E45; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Long DNA palindromes, cruciform structures, genetic instability and secondary structure repair.; citation_author=DR Leach; citation_journal_title=Bioessays; citation_volume=16; citation_number=19; citation_pages=893-900; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=Factors affecting inverted repeat stimulation of recombination and deletion in Saccharomyces cerevisiae.; citation_author=KS Lobachev; citation_author=BM Shor; citation_author=HT Tran; citation_author=W Taylor; citation_author=JD Keen; citation_journal_title=Genetics; citation_volume=148; citation_number=20; citation_pages=1507-1524; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Instability of long inverted repeats within mouse transgenes.; citation_author=A Collick; citation_author=J Drew; citation_author=J Penberth; citation_author=P Bois; citation_author=J Luckett; citation_journal_title=Embo J; citation_volume=15; citation_number=21; citation_pages=1163-1171; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Compensatory neutral mutations and the evolution of RNA.; citation_author=PG Higgs; citation_journal_title=Genetica; citation_volume=102–103; citation_number=22; citation_pages=91-101; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Estimation of superhelical density in vivo from analysis of the level of cruciforms existing in living cells.; citation_author=G Zheng; citation_author=DW Ussery; citation_author=RR Sinden; citation_journal_title=J Mol Biol; citation_volume=221; citation_number=23; citation_pages=122-129; citation_date=1991; " />
      <meta name="citation_reference" content="citation_title=Supercoil-induced extrusion of a regulatory DNA hairpin.; citation_author=X Dai; citation_author=MB Greizerstein; citation_author=K Nadas-Chinni; citation_author=LB Rothman-Denes; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=94; citation_number=24; citation_pages=2174-2179; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Specific recognition of cruciform DNA by nuclear protein HMG1.; citation_author=ME Bianchi; citation_author=M Beltrame; citation_author=G Paonessa; citation_journal_title=Science; citation_volume=243; citation_number=25; citation_pages=1056-1059; citation_date=1989; " />
      <meta name="citation_reference" content="citation_title=Inverted repeat structure of the human genome: the X-chromosome contains a preponderance of large, highly homologous inverted repeats that contain testes genes.; citation_author=PE Warburton; citation_author=J Giordano; citation_author=F Cheung; citation_author=Y Gelfand; citation_author=G Benson; citation_journal_title=Genome Res; citation_volume=14; citation_number=26; citation_pages=1861-1869; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=A general method applicable to the search for similarities in the amino acid sequence of two proteins.; citation_author=SB Needleman; citation_author=CD Wunsch; citation_journal_title=J Mol Biol; citation_volume=48; citation_number=27; citation_pages=443-453; citation_date=1970; " />
      <meta name="citation_reference" content="citation_title=Inparanoid: a comprehensive database of eukaryotic orthologs.; citation_author=KP O'Brien; citation_author=M Remm; citation_author=EL Sonnhammer; citation_journal_title=Nucleic Acids Res; citation_volume=33; citation_number=28; citation_pages=D476-480; citation_date=2005; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000262" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Evidence for Active Maintenance of Inverted Repeat Structures Identified by a Comparative Genomic Approach"/>
    <meta name="twitter:description" content="Inverted repeats have been found to occur in both prokaryotic and eukaryotic genomes. Usually they are short and some have important functions in various biological processes. However, long inverted repeats are rare and can cause genome instability. Analyses of C. elegans genome identified long, nearly-perfect inverted repeat sequences involving both divergently and convergently oriented homologous gene pairs and complete intergenic sequences. Comparisons with the orthologous regions from the genomes of C. briggsae and C. remanei show that the inverted repeat structures are often far more conserved than the sequences. This observation implies that there is an active mechanism for maintaining the inverted repeat nature of the sequences."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0000262.g004"/>

  <meta property="og:title" content="Evidence for Active Maintenance of Inverted Repeat Structures Identified by a Comparative Genomic Approach" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000262" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=6141'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=8868'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=8419&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0000262">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000262" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000262&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Evidence for Active Maintenance of Inverted Repeat Structures Identified by a Comparative Genomic Approach
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Guoyan Zhao, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Genetics, Washington University School of Medicine, St. Louis, Missouri, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Kuan Y. Chang, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Pathology and Immunology, Washington University School of Medicine, St. Louis, Missouri, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Katherine Varley, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Genetics, Washington University School of Medicine, St. Louis, Missouri, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Gary D. Stormo
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:stormo@genetics.wustl.edu">stormo@genetics.wustl.edu</a></p>

                <p>Affiliation:
                  Department of Genetics, Washington University School of Medicine, St. Louis, Missouri, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: February 28, 2007</li>
    <li>DOI: 10.1371/journal.pone.0000262</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0000262" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0000262" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000262" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0000262" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0000262-g001" data-doi="info:doi/10.1371/journal.pone.0000262" data-uri="info:doi/10.1371/journal.pone.0000262.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000262.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000262-g002" data-doi="info:doi/10.1371/journal.pone.0000262" data-uri="info:doi/10.1371/journal.pone.0000262.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000262.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000262-g003" data-doi="info:doi/10.1371/journal.pone.0000262" data-uri="info:doi/10.1371/journal.pone.0000262.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000262.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000262-g004" data-doi="info:doi/10.1371/journal.pone.0000262" data-uri="info:doi/10.1371/journal.pone.0000262.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000262.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000262">Reader Comments (0)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0000262" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1"></a><p>Inverted repeats have been found to occur in both prokaryotic and eukaryotic genomes. Usually they are short and some have important functions in various biological processes. However, long inverted repeats are rare and can cause genome instability. Analyses of <em>C. elegans</em> genome identified long, nearly-perfect inverted repeat sequences involving both divergently and convergently oriented homologous gene pairs and complete intergenic sequences. Comparisons with the orthologous regions from the genomes of <em>C. briggsae</em> and <em>C. remanei</em> show that the inverted repeat structures are often far more conserved than the sequences. This observation implies that there is an active mechanism for maintaining the inverted repeat nature of the sequences.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>Zhao G, Chang KY, Varley K, Stormo GD (2007) Evidence for Active Maintenance of Inverted Repeat Structures Identified by a Comparative Genomic Approach. PLoS ONE 2(2):
          e262.
            doi:10.1371/journal.pone.0000262</p><p><strong>Academic Editor: </strong>Neil Hall, Institute for Genomic Research, United States of America</p><p><strong>Received:</strong> November 17, 2006; <strong>Accepted:</strong> February 8, 2007; <strong>Published:</strong> February 28, 2007</p><p><strong>Copyright:</strong> © 2007 Zhao et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>This work was supported by National Institute of Health grants HG00249, and G.Z. was supported by National Institute of Health Institutional training grant 5 T32 HG000045-08 and National Institute of General Medical Sciences NRSA service award 1 F32 GM73444-01. The funding agencies did not play any role in the preparation, review, or approval of the manuscript.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>An inverted repeat, or biological palindrome, consists of two arms of similar DNA sequences that occur adjacent to each other (perhaps containing a short, non-palindromic spacer between the arms) but on opposite strands and in opposite orientation. The consequence of these inverted repeats is that they can form hairpin or cruciform structures through intramolecular base pairing. Inverted repeats have been shown to play crucial roles in DNA replication <a href="#pone.0000262-Jin1">[1]</a>, transcriptional regulation in various organisms from N4 bacteriophage to human <a href="#pone.0000262-Kim1">[2]</a>–<a href="#pone.0000262-Spiro1">[5]</a>, as well as translational control <a href="#pone.0000262-Thomson1">[6]</a>. However inverted repeats are also one of the sources of genome instability and are known to cause different types of genomic rearrangements in a wide variety of organisms <a href="#pone.0000262-Akgun1">[7]</a>–<a href="#pone.0000262-Nag1">[9]</a>. In human, inverted repeats are associated with several human diseases <a href="#pone.0000262-Bissler1">[10]</a>.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>One type of genomic instability associated with inverted repeats is gene conversion, which is nonreciprocal transfer of genetic information. Whether gene conversion occurs is determined by how regions of heteroduplex DNA are resolved. Cruciform structure branch migration gives rise to regions of heteroduplex DNA. In the heteroduplex DNA region, if one strand (the acceptor) uses the other strand (the donor) as the template to repair mismatches, base changes occur only on the acceptor strand which results in a gene conversion event. Recently, it was proposed that gene conversion has maintained the structure and function of key genes in the non-recombinant region in the human Y chromosome <a href="#pone.0000262-Rozen1">[11]</a>.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>In this report, we describe an interesting genomic structure of intergenic regions of <em>C. elegans</em>, <em>C. briggsae</em> and <em>C. remanei</em>. We found examples of intergenic regions between paralogous gene pairs that are inverted repeats and where the same genomic structure exists in all three species suggesting a common ancestry of the inverted repeat structures. Paralogous gene pairs and intra-palindromic (arm-to-arm) sequences exhibit unusually high sequence identity, sometimes 100% identity. However, orthologous gene pairs and orthologous palindrome arms are less conserved. These results suggest that some mechanism is functioning in all three species to maintain the inverted repeat structure which raises the possibility that the inverted repeat structure rather than the sequence plays a critical function.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3>
<h4>Conserved structure in <em>C. elegans</em>, <em>C. briggsae</em> and <em>C. remanei</em></h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>Many <em>C. elegans</em> intergenic regions have long inverted repeat sequences, but in the following we focus on a few examples with clear orthologous regions in <em>C. briggsae</em> and <em>C. remanei</em>. In each example a gene duplication event must have preceded the divergence of the species because the inverted repeat appears in each species. The orientation of the genes, both divergent and convergent, requires that the duplication event created an initial inverted repeat structure, rather than a direct repeat. But while the orthologous sequences have diverged considerably between species, the paralogous intergenic regions within species are often highly conserved.</p>
<a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2"></a><p><a href="#pone-0000262-g001">Figure 1</a> shows one example of divergently oriented gene pairs. <em>C. elegans</em> genes F44E5.4 and F44E5.5 are paralogs that are divergently oriented with 100% DNA sequence identity. The intergenic sequence between F44E5.4 and F44E5.5 is 446 bp long with a 160 bp arm on each side and a 126 bp spacer. The sequence identity between the two arms is 95.7%. Similarly, <em>C. briggsae</em> genes CBG13233 and CBG13234 and <em>C. remanei</em> genes Contig35.Fgenesh_Celegans.59.final and Contig35.eannot.383.final.final, orthologs of F44E5.4 and F44E5.5, are divergently oriented with 98% and at least 88.5% DNA sequence identity, respectively. (Because the <em>C. remanei</em> genome is not finished yet, the <em>C. remanei</em> gene Contig35.eannot.383.final.final has a stretch of N, so the sequence identity could be higher than 88.5%.) The intra-palindromic arms of <em>C. briggsae</em> and <em>C. remanei</em> exhibit 97.7% and 99.2% sequence identity respectively. The existence of this palindromic structure in all three species suggests that a gene duplication event occurred before the separation of the three <em>Caenorhabditis</em> lineages. But sequence conservation is much higher between the paralogous palindromic arms within a species than between the orthologous sequences across species. In the example shown in <a href="#pone-0000262-g001">figure 1</a>, the palindromic arms of the inverted repeat structure have sequence identity greater than 95% within each species. However, the sequence identity of the palindromic arms between species is lower than 80% and the sequence identity of the entire intergenic regions are only 66.3% between <em>C. elegans</em> and <em>C. briggsae</em> and 54.2% between <em>C. elegans</em> and <em>C. remanei</em>. Therefore, it is the inverted repeat structure rather than the sequence that is conserved among all three species.</p>
<div class="figure" id="pone-0000262-g001"><div class="img"><a name="pone-0000262-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000262.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000262" data-uri="info:doi/10.1371/journal.pone.0000262.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000262.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000262.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000262.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000262.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000262.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000262.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000262.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000262.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000262.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1. </strong></p><a id="article1.body1.sec2.sec1.fig1.caption1.p1" name="article1.body1.sec2.sec1.fig1.caption1.p1"></a><p>An example of divergently oriented inverted repeat. A. Schematic representation of the inverted repeat structure of the <em>C. elegans</em> F44E5.4 - F44E5.5 genomic region, CBG13233 - CBG13234 genomic region in <em>C. briggsae</em> and Contig35.Fgenesh-Celegans.59.final (Contig35-1) - Contig35.eannot.383.final.final (Contig35-2) region in <em>C. remanei</em>. B. Sequence comparison was carried out between sequence 1 and sequence 2 for each row. Sequence identity within each species (intraspecies) in a global alignment (Needleman-Wunsch) is shown in the last column of the table. C. Sequence identity between species (interspecies) is shown in the last column of the table. Sequence identity within a species is much higher than the sequence identity between species. CE: <em>C. elegans</em>; CB: <em>C. briggsae</em>; CR: <em>C. remanei</em>; IGR: intergenic region. * indicates that sequence identity could be higher than 88.5% because Contig35-2 has a stretch of Ns (10) which is estimated length of the sequencing gap.</p>
<span>doi:10.1371/journal.pone.0000262.g001</span></div><a id="article1.body1.sec2.sec1.p3" name="article1.body1.sec2.sec1.p3"></a><p><a href="#pone-0000262-g002">Figure 2</a> gives one example of convergently oriented gene pairs. <em>C. elegans</em> HSP 16 gene locus consists of 4 genes that are arranged as a palindromic structure. The region of T27E4.8, IGR-1 (IGR: intergenic region), T27E4.3 and IGR-L is duplicated to generate an inverted repeat structure. The intergenic sequence between T27E4.3 and T27E4.9 is 662 bp long with a 124 bp arm on each side and a 414 bp spacer. The inverted repeat structure is perfectly maintained since duplication and the sequence identity between the palindromic arms is 100% both in the coding region and in the intergenic region. Because of the high sequence identity, it was proposed that the duplication event may have occurred recently or, alternatively, gene conversion may have maintained identity of the two gene pairs <a href="#pone.0000262-Russnak1">[12]</a>. Our analyses suggest that this duplication event is an ancient one because a similar genomic structure also exists in <em>C. remanei</em>. <em>C. remanei</em> genes Contig904-snap9.final (Contig904-1), Contig904.eannot.018.final.final (Contig904-2), Contig904.eannot.1017.final.final (Contig904-3) and Contig904-snap4.final (Contig904-4) are arranged in the same orientations as their orthologs in <em>C. elegans</em> with 93.8% DNA sequence identity between Contig904-2, and Contig904-3 and 93.9% between Contig904-1 and Contig904-4. The palindromic arms of the intergenic sequence between Contig904-2 and Contig904-3 exhibit 83.4% sequence identity. Similarly, the sequence identity is much lower between orthologous gene pairs and between orthologous palindrome arms (<a href="#pone-0000262-g002">Figure 2</a>). In <em>C. briggsae</em>, genes CBG04605, CBG04606, CBG04607 and CBG04608 are arranged in the same orientations as their orthologs in <em>C. elegans</em> and <em>C. remanei</em>. However, sequence identity between the paralogous gene pairs as well as between the intergenic sequences are much lower (less than 61%). These data imply that <em>C. briggsae</em> inherited the same genomic structure generated by the same duplication event, but the inverted repeat structure was allowed to degenerate in <em>C. briggsae</em>. These results suggest that these inverted repeat structures are of ancient origin and are maintained in <em>C. elegans</em> and <em>C. remanei</em> but lost in <em>C. briggsae</em>.</p>
<div class="figure" id="pone-0000262-g002"><div class="img"><a name="pone-0000262-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000262.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000262" data-uri="info:doi/10.1371/journal.pone.0000262.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000262.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000262.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000262.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000262.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000262.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000262.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000262.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000262.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000262.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2. </strong></p><a id="article1.body1.sec2.sec1.fig2.caption1.p1" name="article1.body1.sec2.sec1.fig2.caption1.p1"></a><p>An example of convergently oriented inverted repeat. A. Schematic representation of the inverted repeat structure of <em>C. elegans</em> T27E4.3 - T27E4.9 genomic region as well as <em>C. briggsae</em> and <em>C. remanei</em> orthologous genomic region. In <em>C. elegans</em>, the region of T27E4.8, IGR-1, T27E4.3 and IGR-L is a perfect mirror image of the region of T27E4.2, IGR-2, T27E4.9 and IGR-R. In <em>C. remanei</em>, Contig904-snap.9.final (Contig904-1), Contig904-eannot.018.final.final (Contig904-2), Contig904.eannot.1017.final.final (Contig904-3) and Contig904.snap.4.final (Contig904-4) have similar inverted repeat structure. In <em>C. briggsae</em>, CBG04605, CBG04606, CBG04607 and CBG04608 are arranged in the same orientation but don't have inverted repeat structure. B. Sequence comparison was carried out between sequence 1 and sequence 2 for each row. Sequence identity within each species (intraspecies) in a global alignment (Needleman-Wunsch) is shown in the last column of the table. C. Sequence identity between species (interspecies) is shown in the last column of the table. Sequence identity within a species is much higher than the sequence identity between species for <em>C. elegans</em> and <em>C. remanei</em>. CE: <em>C. elegans</em>; CB: <em>C. briggsae</em>; CR: <em>C. remanei</em>; IGR: intergenic region.</p>
<span>doi:10.1371/journal.pone.0000262.g002</span></div>

<h4>Duplication and evolution of paralogous gene pairs</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>We observed that divergently oriented paralogous gene pairs tend to duplicate as an entity and this duplication is still ongoing after speciation.</p>
<a id="article1.body1.sec2.sec2.p2" name="article1.body1.sec2.sec2.p2"></a><p><a href="#pone-0000262-g003">Figure 3</a> shows that the orthologous region of F44E5.4 and F44E5.5 (in <a href="#pone-0000262-g001">Figure 1</a>) is duplicated in both <em>C. briggsae</em> and <em>C. remanei</em>. In <em>C. briggsae</em>, the region is duplicated as direct repeats (panel A). CBG13231 and CBG13233 are only 54.1% identical and CBG13232 and CBG13234 are only 47.9% identical in global alignment, suggesting that these genes have diverged considerably since duplication. However, CBG13231 and CBG13233 are 99.3% identical for the first 297 nt and CBG13232 and CBG13234 are 99.2% identical for the first 764 nt (numbers in parenthesis). Furthermore, the intergenic region between CBG13231 and CBG13232 is 98% identical to the intergenic region between CBG13233 and CBG13234. Sequence identity between CBG13231 and CBG13232 is only 59.6% globally but 100% identical for the first 297 nt. These results suggest that the inverted repeat structure between divergently oriented CBG13231 and CBG13232 is preserved extending to the coding region although the C-terminals of the genes have diverged considerably. Therefore, there must be some mechanism that maintained the inverted repeat structure but allowed the rest of the sequences to evolve differently. In <em>C. remanei</em>, the duplicated regions are in two different contigs. The intergenic regions only have 63.6% sequence identity as a whole. However, the left and right palindromic arms of the intergenic regions are 98.4% and 99.2% identical, respectively (<a href="#pone-0000262-g003">Figure 3, panel B</a>). Therefore, the spacer has evolved much faster than the coding region as well as the palindromic arms. Since <em>C. briggsae</em> and <em>C. remanei</em> are the most closely related species <a href="#pone.0000262-Kiontke1">[13]</a>, it is not clear whether <em>C. elegans</em> lost the duplicated pair or that <em>C. briggsae</em> and <em>C. remanei</em> gained it in the branch leading to the separation of <em>C. briggsae</em> and <em>C. remanei</em>.</p>
<div class="figure" id="pone-0000262-g003"><div class="img"><a name="pone-0000262-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000262.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000262" data-uri="info:doi/10.1371/journal.pone.0000262.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000262.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000262.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000262.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000262.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000262.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000262.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000262.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000262.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000262.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3. </strong></p><a id="article1.body1.sec2.sec2.fig1.caption1.p1" name="article1.body1.sec2.sec2.fig1.caption1.p1"></a><p>Duplication and evolution of inverted repeat orthologous gene pairs. A. Duplication of inverted repeat gene pair in <em>C. briggsae</em>. The CBG13233 - CBG13234 genomic region, which is orthologous to the <em>C. elegans</em> F44E5.4 - F44E5.5 genomic region, is duplicated as tandem repeats in <em>C. briggsae</em>. Number shows sequence identity between two sequences in a global alignment. Number in parenthesis shows sequence identity between two sequences in a local alignment. CBG13231 and CBG13233 are 99.3% identical for the first 297 nt. CBG13232 and CBG13234 are 99.2% identical for the first 764 nt. CBG13231 and CBG13232 are 100% identical for the first 297 nt. The intergenic region between CBG13231 and CBG13232 are 98% identical to the intergenic region between CBG13233 and CBG13234. B. Duplication of inverted repeat gene pair in <em>C. remanei</em>. The Contig35.Fgenesh-Celegans.59.final (Contig35-1) - Contig.eannot.388.final.final (Contig35-2) region, which is orthologous to the <em>C. elegans</em> F44E5.4 - F44E5.5 genomic region, is duplicated in <em>C. remanei</em>. Currently, it is not clearly whether the duplicated regions are in the same chromosome or not. Sequence identities between duplicated genes as well as between duplicated intergenic sequences are shown in the boxes.</p>
<span>doi:10.1371/journal.pone.0000262.g003</span></div><a id="article1.body1.sec2.sec2.p3" name="article1.body1.sec2.sec2.p3"></a><p>The divergently oriented paralogous gene pair of F42F12.1 and F42F12.9 and their paralogs and orthologs were duplicated in a more complex way (<a href="#pone-0000262-g004">Figure 4</a>). In <em>C. elegans</em>, three paralogous pairs exist on chromosome X: F42F12.1-F42F12.9, F42F12.10-F42F12.6, F42F12.7-F42F12.8 (panel A). F42F12.10-F42F12.6 and F42F12.7-F42F12.8 are adjacent to each other. F42F12.1-F42F12.9 and F42F12.10-F42F12.6 are separated by 12 kb genomic DNA with 5 genes. All three gene pairs have inverted repeat structures and sequence identities between coding gene pairs and between palindromic arms are shown in <a href="#pone-0000262-g004">Figure 4</a>. The <em>C. remanei</em> genome has three gene pairs that are homologous to the <em>C. elegans</em> genes with two pairs in the same contig. All three gene pairs have inverted repeat structures (panel A). The <em>C. briggsae</em> genome, however, has four gene pairs that are homologous to the <em>C. elegans</em> genes (panel A). Two interesting things are worth noting in <em>C. briggase</em>. First, gene pair of CBG10614-CBG10615 as well as their intergenic region has diverged considerably. CBG10614 and CBG10615 are 91.5% identical in the first 363 nt but CBG10614 (875 nt) is considerably longer than CBG10615 (363 nt). The inverted repeat structure in the intergenic region is disrupted by a 153 nt insertion. If the 153 nt insertion is removed, the palindromic structure is obvious with arms sharing 81.6% sequence identity. Second, CBG14426-CBG14427 and CBG14035-CBG14036 seem to be recent duplication because they are 100% identical in both coding regions and in intergenic sequences. Therefore, three out of four gene pairs maintained their inverted repeat structure although the sequences between orthologous gene pairs have diverged considerably (<a href="#pone-0000262-g001">Figure 4, panel B, C</a>). Because all four gene pairs are in different contigs, currently we do not know whether these duplications occurred before or after speciation. Completely finished genomic sequences of <em>C. briggsae</em> and <em>C. remanei</em> may help to answer this question.</p>
<div class="figure" id="pone-0000262-g004"><div class="img"><a name="pone-0000262-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000262.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000262" data-uri="info:doi/10.1371/journal.pone.0000262.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000262.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000262.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000262.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000262.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000262.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000262.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000262.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000262.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000262.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4. </strong></p><a id="article1.body1.sec2.sec2.fig2.caption1.p1" name="article1.body1.sec2.sec2.fig2.caption1.p1"></a><p>Duplication and evolution of inverted repeat gene pairs. A. Schematic representation of the genomic structure of the F42F12.1-F42F12.9 gene pairs as well as two <em>C. elegans</em> paralogous gene pairs, four <em>C. briggsae</em> orthologous gene pairs and three <em>C. remanei</em> orthologuos gene pairs. Numbers above each gene pair indicate the sequence identities between the two coding genes in a global alignment. Numbers below each gene pair indicate the sequence identifies between two arms of the inverted repeat intergenic region. Number in parenthesis shows sequence identity between the two sequences in a local alignment. B. Sequence identify between coding sequences. C. Sequence identity between intergenic sequences. For each row, sequence comparison was performed between sequence 1 and sequence 2 and sequence identity between these two sequences is shown in the last column. Contig59-1: Contig59.Fgenesh_Celegans.40.final; Contig59-2: Contig59.eannot.1190.final.final; Contig25-1: Contig25.Fgenesh_Celegans.92.final; Contig25-2: Contig.1332.final.final; Contig25-3: Contig25.Fgenesh_Celegans.94.final; Contig25-4: Contig.1333.final.final. CE: <em>C. elegans</em>; CB: <em>C. briggsae</em>; CR: <em>C. remanei</em>.</p>
<span>doi:10.1371/journal.pone.0000262.g004</span></div>
</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p>In this study, we report our finding of highly conserved intergenic inverted repeat structures in less conserved intergenic sequences. Although inverted repeats have been the focus of several studies <a href="#pone.0000262-LeBlanc1">[14]</a>–<a href="#pone.0000262-Semple1">[17]</a>, this is the first time that inverted repeat structures involving paralogous gene pairs have been described. The presence of divergently/convergently oriented paralogs flanking the intergenic inverted repeat suggests that the inverted repeat was introduced during an intra-strand gene duplication in the common ancestor of the nematodes. Since <em>C. elegans</em> and <em>C. briggsase</em> were estimated to have diverged about 100 million years ago <a href="#pone.0000262-Stein1">[18]</a>, it is not surprising that the intergenic sequences have diverged considerably. Conservation of the inverted repeat structure rather than the sequence in the three <em>Caenorhabiditis</em> species implies there is either a mechanism of symmetric mutation or that there is selective pressure retaining mutations that occur in the sequence which preserve the inverted repeat.</p>
<a id="article1.body1.sec3.p2" name="article1.body1.sec3.p2"></a><p>One mechanism to maintain the high conservation of inverted repeats is gene conversion. The analysis of the human Y chromosome revealed that several gene duplication events have occurred involving large inverted repeat sequences including coding regions. It was proposed that it is the palindromic arm to arm gene conversion that drives the paired arms to evolve in concert which results in the highly identical paired arms <a href="#pone.0000262-Rozen1">[11]</a>. Gene conversion events have been described previously in <em>C. elegans</em> <a href="#pone.0000262-Semple1">[17]</a>. Perhaps a similar mechanism is maintaining the conserved inverted repeats in <em>C. elegans</em>, <em>C. briggsae</em> and <em>C. remanei</em>. The inverted repeat structure could be lost if free evolution is allowed. For example, the inverted repeat of CBG13231 and CBG13232 has only been partially preserved while the inverted repeat between CBG04606 and CBG04607 is completely lost in <em>C briggsae</em>.</p>
<a id="article1.body1.sec3.p3" name="article1.body1.sec3.p3"></a><p>However, long inverted repeats have been shown to have a profound effect on genome stability. In <em>E. Coli</em>, replicons with long inverted repeats (&gt;150 bp) can not be propagated and are deleted at extremely high rates <a href="#pone.0000262-Leach1">[19]</a>. In yeast, a perfect palindrome, formed by two 1.0-kb inverted repeats, increased intra- and interchromosomal recombination in the adjacent region 2,400-fold and 17,000-fold, respectively and is also deleted at high frequency <a href="#pone.0000262-Lobachev1">[20]</a>. In mammals, inverted repeats are extremely unstable and undergo both homologous recombination and non-homologous deletion at high frequency <a href="#pone.0000262-Akgun1">[7]</a>, <a href="#pone.0000262-Collick1">[21]</a>. Inversion of the inverted repeat brought about by a homologous recombination will not stabilize the locus. The locus is stabilized only after the formation of central asymmetry by deletion <a href="#pone.0000262-Akgun1">[7]</a>, <a href="#pone.0000262-Collick1">[21]</a>. Although such studies have not been carried out in <em>C. elegans</em>, we would expect the <em>C. elegans</em> genome to have similar properties, based on the conservation between yeast and mammals. Genome instability is positively correlated with the size of inverted repeats, the identity between the stem arms and is negatively correlated with the size of intervening spacers <a href="#pone.0000262-Lobachev1">[20]</a>. In our study, the inverted repeat structure is very long (2207 bp of palindromic arms for the F44E5.4 locus) with a relatively short spacer (126 bp) and very high sequence identity (99.7%). This locus should be highly recombinagenic. However, they have been stably transmitted for about 100 million years. In addition, long inverted repeats with high sequence identity are very rare in the <em>C. elegans</em> genome (less than 0.7%) <a href="#pone.0000262-Wang1">[15]</a>. Therefore, the evidence suggests that the conserved inverted repeat structure in the intergenic sequence is due to selection for some function that requires the secondary structure allowed by an inverted repeat.</p>
<a id="article1.body1.sec3.p4" name="article1.body1.sec3.p4"></a><p>Conservation of DNA structure is observed in non-coding RNA genes where symmetric mutations are selected to preserve the intra-strand nucleotide base paring but not the overall sequence of the orthologs <a href="#pone.0000262-Higgs1">[22]</a>. Since the sequence identity of the inverted repeats is low between species the function under selection as the species diverged must be associated with the structure. Inverted repeats have the potential to form cruciform structures <em>in vivo</em> <a href="#pone.0000262-Zheng1">[23]</a>. Studies have shown that some cruciforms are critically involved in gene transcriptional regulation <a href="#pone.0000262-Zheng1">[23]</a>, <a href="#pone.0000262-Dai1">[24]</a>. Cruciforms may act as target sites for activator and repressor proteins and serve as a novel mechanism that controls cell-specific promoter activity <a href="#pone.0000262-Zheng1">[23]</a>. In this study, the genes flanking the inverted repeats were always paralogs of each other. Perhaps the cruciform structure formed by the inverted repeat controls transcriptional regulation of the paralogs. Since some regulatory DNA binding proteins recognize the cruciform structure rather than the sequence <a href="#pone.0000262-Bianchi1">[25]</a> this could explain why the sequence identity of intergenic regions between orthologs is low, but the inverted repeat structure is conserved. A similar role was proposed for human inverted repeats in controlling sex-specific gene expression during germ-cell development or meiosis <a href="#pone.0000262-Warburton1">[26]</a>.</p>
<a id="article1.body1.sec3.p5" name="article1.body1.sec3.p5"></a><p>Currently, the genomes of many organisms have been sequenced. However, fully understanding of how information is stored in the genomes remains a big challenge. The novel genomic structures reported here suggests that there may be many more examples to be discovered and comparative genomics is a great tool for uncovering regions under unusual selection. It would be interesting to see whether similar structures are also present in other organisms and what is the biological function of this structure.</p>
</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4>Identification of Inverted Repeats</h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p>We retrieved all <em>C. elegans</em> intergenic sequences and annotation from WormBase (<a href="http://www.wormbase.org/">http://www.wormbase.org/</a>) and used a Needleman-Wunsch global alignment algorithm <a href="#pone.0000262-Needleman1">[27]</a> to align an intergenic sequence against its reverse complement. This report is intended to identify specific intergenic regions that have significant repeats rather than to give a comprehensive list of gene pairs that have this genomic structure. Therefore, we use a stringent cutoff of 50% sequence identity to ensure that the intergenic region had a significant inverted repeat above the background of the <em>C. elegans</em> genome (<a href="#pone.0000262.s001">Figure S1</a> shows the distribution of percent identity to the reverse complement for all <em>C. elegans</em> intergenic regions.).</p>


<h4>Identification of homologous gene pairs</h4>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1"></a><p>We first identified all <em>C. elegans</em> gene pairs that have inverted repeats above the cutoff. We then identified all the gene pairs that are homologous to a given <em>C. elegans</em> gene pair in the genome of <em>C. elegans</em>, <em>C. briggsae</em> and <em>C. remanei</em>. To identify homologous gene pairs in the genome of <em>C. elegans</em> and <em>C. briggsae</em>, we used InParanoid Ortholog Groups information downloaded from <a href="http://inparanoid.cgb.ki.se/">http://inparanoid.cgb.ki.se/</a> <a href="#pone.0000262-OBrien1">[28]</a>. In this study, the term ‘inparalogs’ indicate paralogs that arose through a gene duplication event after speciation, while ‘outparalogs’ arise following a gene duplication preceding speciation. We use this information as a guide to identify all the gene pairs that are homologous to a given <em>C. elegans</em> gene pair. We then analyzed the genomic regions of those genes for genomic structure and conservation. <em>C. briggsae</em> sequence and annotation were obtained from WormBase (<a href="http://www.wormbase.org/">http://www.wormbase.org/</a>).</p>
<a id="article1.body1.sec4.sec2.p2" name="article1.body1.sec4.sec2.p2"></a><p>To identify the <em>C. remanei</em> orthologs for each of the genes flanking the inverted repeat in <em>C. elegans</em>, we used NCBI stand alone BLAST on the <em>C. remanei</em> proteome. We then determined if the orthologs in <em>C. remanei</em> were adjacent in the <em>C. remanei</em> genome and matched the gene orientation in <em>C. elegans</em>. Because we expect the gene pairs to be products of duplication, we did not require them to be mutual best BLAST hits. <em>C. remanei</em> sequence and annotation were produced by the Genome Sequencing Center at Washington University School of Medicine in St. Louis and can be obtained from <a href="http://genome.wustl.edu/pub/organism/Invertebrates/Caenorhabditis_remanei/">http://genome.wustl.edu/pub/organism/Inv​ertebrates/Caenorhabditis_remanei/</a>.</p>

</div>

<div id="section5" class="section"><a id="s5" name="s5" toc="s5" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0000262"></div><a name="pone.0000262.s001" id="pone.0000262.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000262.s001">Figure S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1"></a><p class="preSiDOI">The distribution of the percent identity of C. elegans intergenic sequences. X axis is the length of the intergenic sequence and the Y axis is the percent identity between intergenic sequence and its reverse complement. The red horizontal line represents cutoff.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000262.s001</p><a id="article1.body1.sec5.supplementary-material1.caption1.p2" name="article1.body1.sec5.supplementary-material1.caption1.p2"></a><p class="postSiDOI">(5.22 MB EPS)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>We are grateful to John Spieth and Li Ding for their help with obtaining <em>C. remanei</em> genomic sequences and annotations.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: GS. Performed the experiments: GZ. Analyzed the data: GZ. Contributed reagents/materials/analysis tools: KC KV. Wrote the paper: GZ.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0000262-Jin1" id="pone.0000262-Jin1"></a>Jin R, Fernandez-Beros ME, Novick RP (1997) Why is the initiation nick site of an AT-rich rolling circle plasmid at the tip of a GC-rich cruciform? Embo J  16: 4456–4466.  <ul class="find" data-citedArticleID="934640" data-doi="10.1093/emboj/16.14.4456"><li><a href="http://dx.doi.org/10.1093/emboj/16.14.4456" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Why+is+the+initiation+nick+site+of+an+AT-rich+rolling+circle+plasmid+at+the+tip+of+a+GC-rich+cruciform%3F" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Why+is+the+initiation+nick+site+of+an+AT-rich+rolling+circle+plasmid+at+the+tip+of+a+GC-rich+cruciform%3F%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0000262-Kim1" id="pone.0000262-Kim1"></a>Kim EL, Peng H, Esparza FM, Maltchenko SZ, Stachowiak MK (1998) Cruciform-extruding regulatory element controls cell-specific activity of the tyrosine hydroxylase gene promoter. Nucleic Acids Res  26: 1793–1800.  <ul class="find" data-citedArticleID="934642" data-doi="10.1093/nar/26.7.1793"><li><a href="http://dx.doi.org/10.1093/nar/26.7.1793" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Cruciform-extruding+regulatory+element+controls+cell-specific+activity+of+the+tyrosine+hydroxylase+gene+promoter." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Cruciform-extruding+regulatory+element+controls+cell-specific+activity+of+the+tyrosine+hydroxylase+gene+promoter.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0000262-Glucksmann1" id="pone.0000262-Glucksmann1"></a>Glucksmann MA, Markiewicz P, Malone C, Rothman-Denes LB (1992) Specific sequences and a hairpin structure in the template strand are required for N4 virion RNA polymerase promoter recognition. Cell  70: 491–500.  <ul class="find" data-citedArticleID="934634" data-doi="10.1016/0092-8674(92)90173-a"><li><a href="http://dx.doi.org/10.1016/0092-8674(92)90173-a" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Specific+sequences+and+a+hairpin+structure+in+the+template+strand+are+required+for+N4+virion+RNA+polymerase+promoter+recognition." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Specific+sequences+and+a+hairpin+structure+in+the+template+strand+are+required+for+N4+virion+RNA+polymerase+promoter+recognition.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0000262-Markiewicz1" id="pone.0000262-Markiewicz1"></a>Markiewicz P, Malone C, Chase JW, Rothman-Denes LB (1992) <em>Escherichia coli</em> single-stranded DNA-binding protein is a supercoiled template-dependent transcriptional activator of N4 virion RNA polymerase. Genes Dev  6: 2010–2019.  <ul class="find" data-citedArticleID="934652" data-doi="10.1101/gad.6.10.2010"><li><a href="http://dx.doi.org/10.1101/gad.6.10.2010" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Escherichia+coli+single-stranded+DNA-binding+protein+is+a+supercoiled+template-dependent+transcriptional+activator+of+N4+virion+RNA+polymerase." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Escherichia+coli+single-stranded+DNA-binding+protein+is+a+supercoiled+template-dependent+transcriptional+activator+of+N4+virion+RNA+polymerase.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0000262-Spiro1" id="pone.0000262-Spiro1"></a>Spiro C, McMurray CT (1997) Switching of DNA secondary structure in proenkephalin transcriptional regulation. J Biol Chem  272: 33145–33152.  <ul class="find" data-citedArticleID="934666" data-doi="10.1074/jbc.272.52.33145"><li><a href="http://dx.doi.org/10.1074/jbc.272.52.33145" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Switching+of+DNA+secondary+structure+in+proenkephalin+transcriptional+regulation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Switching+of+DNA+secondary+structure+in+proenkephalin+transcriptional+regulation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0000262-Thomson1" id="pone.0000262-Thomson1"></a>Thomson AM, Rogers JT, Leedman PJ (1999) Iron-regulatory proteins, iron-responsive elements and ferritin mRNA translation. Int J Biochem Cell Biol  31: 1139–1152.  <ul class="find" data-citedArticleID="934670" data-doi="10.1016/s1357-2725(99)00080-1"><li><a href="http://dx.doi.org/10.1016/s1357-2725(99)00080-1" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Iron-regulatory+proteins%2C+iron-responsive+elements+and+ferritin+mRNA+translation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Iron-regulatory+proteins%2C+iron-responsive+elements+and+ferritin+mRNA+translation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0000262-Akgun1" id="pone.0000262-Akgun1"></a>Akgun E, Zahn J, Baumes S, Brown G, Liang F, et al.  (1997) Palindrome resolution and recombination in the mammalian germ line. Mol Cell Biol  17: 5559–5570.  <ul class="find" data-citedArticleID="934624"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Palindrome+resolution+and+recombination+in+the+mammalian+germ+line.&amp;auth=&amp;atitle=Palindrome+resolution+and+recombination+in+the+mammalian+germ+line." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Palindrome+resolution+and+recombination+in+the+mammalian+germ+line." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Palindrome+resolution+and+recombination+in+the+mammalian+germ+line.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0000262-Gordenin1" id="pone.0000262-Gordenin1"></a>Gordenin DA, Lobachev KS, Degtyareva NP, Malkova AL, Perkins E, et al.  (1993) Inverted DNA repeats: a source of eukaryotic genomic instability. Mol Cell Biol  13: 5315–5322.  <ul class="find" data-citedArticleID="934636"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Inverted+DNA+repeats%3A+a+source+of+eukaryotic+genomic+instability.&amp;auth=&amp;atitle=Inverted+DNA+repeats%3A+a+source+of+eukaryotic+genomic+instability." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Inverted+DNA+repeats%3A+a+source+of+eukaryotic+genomic+instability." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Inverted+DNA+repeats%3A+a+source+of+eukaryotic+genomic+instability.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0000262-Nag1" id="pone.0000262-Nag1"></a>Nag DK, Kurst A (1997) A 140-bp-long palindromic sequence induces double-strand breaks during meiosis in the yeast <em>Saccharomyces cerevisiae</em>. Genetics  146: 835–847.  <ul class="find" data-citedArticleID="934654"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=A+140-bp-long+palindromic+sequence+induces+double-strand+breaks+during+meiosis+in+the+yeast+Saccharomyces+cerevisiae.&amp;auth=&amp;atitle=A+140-bp-long+palindromic+sequence+induces+double-strand+breaks+during+meiosis+in+the+yeast+Saccharomyces+cerevisiae." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+140-bp-long+palindromic+sequence+induces+double-strand+breaks+during+meiosis+in+the+yeast+Saccharomyces+cerevisiae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+140-bp-long+palindromic+sequence+induces+double-strand+breaks+during+meiosis+in+the+yeast+Saccharomyces+cerevisiae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0000262-Bissler1" id="pone.0000262-Bissler1"></a>Bissler JJ (1998) DNA inverted repeats and human disease. Front Biosci  3: d408–418.  <ul class="find" data-citedArticleID="934628"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=DNA+inverted+repeats+and+human+disease.&amp;auth=&amp;atitle=DNA+inverted+repeats+and+human+disease." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=DNA+inverted+repeats+and+human+disease." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22DNA+inverted+repeats+and+human+disease.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0000262-Rozen1" id="pone.0000262-Rozen1"></a>Rozen S, Skaletsky H, Marszalek JD, Minx PJ, Cordum HS, et al.  (2003) Abundant gene conversion between arms of palindromes in human and ape Y chromosomes. Nature  423: 873–876.  <ul class="find" data-citedArticleID="934660" data-doi="10.1038/nature01723"><li><a href="http://dx.doi.org/10.1038/nature01723" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Abundant+gene+conversion+between+arms+of+palindromes+in+human+and+ape+Y+chromosomes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Abundant+gene+conversion+between+arms+of+palindromes+in+human+and+ape+Y+chromosomes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0000262-Russnak1" id="pone.0000262-Russnak1"></a>Russnak RH, Candido EP (1985) Locus encoding a family of small heat shock genes in <em>Caenorhabditis elegans</em>: two genes duplicated to form a 3.8-kilobase inverted repeat. Mol Cell Biol  5: 1268–1278.  <ul class="find" data-citedArticleID="934662"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Locus+encoding+a+family+of+small+heat+shock+genes+in+Caenorhabditis+elegans%3A+two+genes+duplicated+to+form+a+3.8-kilobase+inverted+repeat.&amp;auth=&amp;atitle=Locus+encoding+a+family+of+small+heat+shock+genes+in+Caenorhabditis+elegans%3A+two+genes+duplicated+to+form+a+3.8-kilobase+inverted+repeat." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Locus+encoding+a+family+of+small+heat+shock+genes+in+Caenorhabditis+elegans%3A+two+genes+duplicated+to+form+a+3.8-kilobase+inverted+repeat." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Locus+encoding+a+family+of+small+heat+shock+genes+in+Caenorhabditis+elegans%3A+two+genes+duplicated+to+form+a+3.8-kilobase+inverted+repeat.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0000262-Kiontke1" id="pone.0000262-Kiontke1"></a>Kiontke K, Fitch DHA, Community TCeR, editor (2005) The phylogenetic relationships of <em>Caenorhabditis</em> and other rhabditids. WormBook: WormBook.   <ul class="find-nolinks"></ul></li><li><span class="label">14.
              </span><a name="pone.0000262-LeBlanc1" id="pone.0000262-LeBlanc1"></a>LeBlanc MD, Aspeslagh G, Buggia NP, Dyer BD (2000) An annotated catalog of inverted repeats of <em>Caenorhabditis elegans</em> chromosomes III and X, with observations concerning odd/even biases and conserved motifs. Genome Res  10: 1381–1392.  <ul class="find" data-citedArticleID="934646" data-doi="10.1101/gr.122700"><li><a href="http://dx.doi.org/10.1101/gr.122700" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=An+annotated+catalog+of+inverted+repeats+of+Caenorhabditis+elegans+chromosomes+III+and+X%2C+with+observations+concerning+odd%2Feven+biases+and+conserved+motifs." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22An+annotated+catalog+of+inverted+repeats+of+Caenorhabditis+elegans+chromosomes+III+and+X%2C+with+observations+concerning+odd%2Feven+biases+and+conserved+motifs.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0000262-Wang1" id="pone.0000262-Wang1"></a>Wang Y, Leung FC (2006) Long inverted repeats in eukaryotic genomes: recombinogenic motifs determine genomic plasticity. FEBS Lett  580: 1277–1284.  <ul class="find" data-citedArticleID="934672" data-doi="10.1016/j.febslet.2006.01.045"><li><a href="http://dx.doi.org/10.1016/j.febslet.2006.01.045" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Long+inverted+repeats+in+eukaryotic+genomes%3A+recombinogenic+motifs+determine+genomic+plasticity." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Long+inverted+repeats+in+eukaryotic+genomes%3A+recombinogenic+motifs+determine+genomic+plasticity.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0000262-Achaz1" id="pone.0000262-Achaz1"></a>Achaz G, Netter P, Coissac E (2001) Study of intrachromosomal duplications among the eukaryote genomes. Mol Biol Evol  18: 2280–2288.  <ul class="find" data-citedArticleID="934622" data-doi="10.1093/oxfordjournals.molbev.a003774"><li><a href="http://dx.doi.org/10.1093/oxfordjournals.molbev.a003774" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Study+of+intrachromosomal+duplications+among+the+eukaryote+genomes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Study+of+intrachromosomal+duplications+among+the+eukaryote+genomes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0000262-Semple1" id="pone.0000262-Semple1"></a>Semple C, Wolfe KH (1999) Gene duplication and gene conversion in the <em>Caenorhabditis elegans</em> genome. J Mol Evol  48: 555–564.  <ul class="find" data-citedArticleID="934664" data-doi="10.1007/pl00006498"><li><a href="http://dx.doi.org/10.1007/pl00006498" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Gene+duplication+and+gene+conversion+in+the+Caenorhabditis+elegans+genome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Gene+duplication+and+gene+conversion+in+the+Caenorhabditis+elegans+genome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0000262-Stein1" id="pone.0000262-Stein1"></a>Stein LD, Bao Z, Blasiar D, Blumenthal T, Brent MR, et al.  (2003) The genome sequence of <em>Caenorhabditis briggsae</em>: a platform for comparative genomics. PLoS Biol  1: E45.  <ul class="find" data-citedArticleID="934668" data-doi="10.1371/journal.pbio.0000045"><li><a href="http://dx.doi.org/10.1371/journal.pbio.0000045" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+genome+sequence+of+Caenorhabditis+briggsae%3A+a+platform+for+comparative+genomics." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+genome+sequence+of+Caenorhabditis+briggsae%3A+a+platform+for+comparative+genomics.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0000262-Leach1" id="pone.0000262-Leach1"></a>Leach DR (1994) Long DNA palindromes, cruciform structures, genetic instability and secondary structure repair. Bioessays  16: 893–900.  <ul class="find" data-citedArticleID="934648" data-doi="10.1002/bies.950161207"><li><a href="http://dx.doi.org/10.1002/bies.950161207" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Long+DNA+palindromes%2C+cruciform+structures%2C+genetic+instability+and+secondary+structure+repair." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Long+DNA+palindromes%2C+cruciform+structures%2C+genetic+instability+and+secondary+structure+repair.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0000262-Lobachev1" id="pone.0000262-Lobachev1"></a>Lobachev KS, Shor BM, Tran HT, Taylor W, Keen JD, et al.  (1998) Factors affecting inverted repeat stimulation of recombination and deletion in <em>Saccharomyces cerevisiae</em>. Genetics  148: 1507–1524.  <ul class="find" data-citedArticleID="934650"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Factors+affecting+inverted+repeat+stimulation+of+recombination+and+deletion+in+Saccharomyces+cerevisiae.&amp;auth=&amp;atitle=Factors+affecting+inverted+repeat+stimulation+of+recombination+and+deletion+in+Saccharomyces+cerevisiae." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Factors+affecting+inverted+repeat+stimulation+of+recombination+and+deletion+in+Saccharomyces+cerevisiae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Factors+affecting+inverted+repeat+stimulation+of+recombination+and+deletion+in+Saccharomyces+cerevisiae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0000262-Collick1" id="pone.0000262-Collick1"></a>Collick A, Drew J, Penberth J, Bois P, Luckett J, et al.  (1996) Instability of long inverted repeats within mouse transgenes. Embo J  15: 1163–1171.  <ul class="find" data-citedArticleID="934630"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Instability+of+long+inverted+repeats+within+mouse+transgenes.&amp;auth=&amp;atitle=Instability+of+long+inverted+repeats+within+mouse+transgenes." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Instability+of+long+inverted+repeats+within+mouse+transgenes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Instability+of+long+inverted+repeats+within+mouse+transgenes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0000262-Higgs1" id="pone.0000262-Higgs1"></a>Higgs PG (1998) Compensatory neutral mutations and the evolution of RNA. Genetica  102–103: 91–101.  <ul class="find" data-citedArticleID="934638" data-doi="10.1007/978-94-011-5210-5_9"><li><a href="http://dx.doi.org/10.1007/978-94-011-5210-5_9" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Compensatory+neutral+mutations+and+the+evolution+of+RNA." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Compensatory+neutral+mutations+and+the+evolution+of+RNA.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0000262-Zheng1" id="pone.0000262-Zheng1"></a>Zheng G, Ussery DW, Sinden RR (1991) Estimation of superhelical density <em>in vivo</em> from analysis of the level of cruciforms existing in living cells. J Mol Biol  221: 122–129.  <ul class="find" data-citedArticleID="934676" data-doi="10.1016/0022-2836(91)80209-d"><li><a href="http://dx.doi.org/10.1016/0022-2836(91)80209-d" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Estimation+of+superhelical+density+in+vivo+from+analysis+of+the+level+of+cruciforms+existing+in+living+cells." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Estimation+of+superhelical+density+in+vivo+from+analysis+of+the+level+of+cruciforms+existing+in+living+cells.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0000262-Dai1" id="pone.0000262-Dai1"></a>Dai X, Greizerstein MB, Nadas-Chinni K, Rothman-Denes LB (1997) Supercoil-induced extrusion of a regulatory DNA hairpin. Proc Natl Acad Sci U S A  94: 2174–2179.  <ul class="find" data-citedArticleID="934632" data-doi="10.1073/pnas.94.6.2174"><li><a href="http://dx.doi.org/10.1073/pnas.94.6.2174" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Supercoil-induced+extrusion+of+a+regulatory+DNA+hairpin." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Supercoil-induced+extrusion+of+a+regulatory+DNA+hairpin.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0000262-Bianchi1" id="pone.0000262-Bianchi1"></a>Bianchi ME, Beltrame M, Paonessa G (1989) Specific recognition of cruciform DNA by nuclear protein HMG1. Science  243: 1056–1059.  <ul class="find" data-citedArticleID="934626" data-doi="10.1126/science.2922595"><li><a href="http://dx.doi.org/10.1126/science.2922595" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Specific+recognition+of+cruciform+DNA+by+nuclear+protein+HMG1." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Specific+recognition+of+cruciform+DNA+by+nuclear+protein+HMG1.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0000262-Warburton1" id="pone.0000262-Warburton1"></a>Warburton PE, Giordano J, Cheung F, Gelfand Y, Benson G (2004) Inverted repeat structure of the human genome: the X-chromosome contains a preponderance of large, highly homologous inverted repeats that contain testes genes. Genome Res  14: 1861–1869.  <ul class="find" data-citedArticleID="934674" data-doi="10.1101/gr.2542904"><li><a href="http://dx.doi.org/10.1101/gr.2542904" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Inverted+repeat+structure+of+the+human+genome%3A+the+X-chromosome+contains+a+preponderance+of+large%2C+highly+homologous+inverted+repeats+that+contain+testes+genes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Inverted+repeat+structure+of+the+human+genome%3A+the+X-chromosome+contains+a+preponderance+of+large%2C+highly+homologous+inverted+repeats+that+contain+testes+genes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0000262-Needleman1" id="pone.0000262-Needleman1"></a>Needleman SB, Wunsch CD (1970) A general method applicable to the search for similarities in the amino acid sequence of two proteins. J Mol Biol  48: 443–453.  <ul class="find" data-citedArticleID="934656" data-doi="10.1016/0022-2836(70)90057-4"><li><a href="http://dx.doi.org/10.1016/0022-2836(70)90057-4" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+general+method+applicable+to+the+search+for+similarities+in+the+amino+acid+sequence+of+two+proteins." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+general+method+applicable+to+the+search+for+similarities+in+the+amino+acid+sequence+of+two+proteins.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0000262-OBrien1" id="pone.0000262-OBrien1"></a>O'Brien KP, Remm M, Sonnhammer EL (2005) Inparanoid: a comprehensive database of eukaryotic orthologs. Nucleic Acids Res  33: D476–480.  <ul class="find" data-citedArticleID="934658" data-doi="10.1093/nar/gki107"><li><a href="http://dx.doi.org/10.1093/nar/gki107" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Inparanoid%3A+a+comprehensive+database+of+eukaryotic+orthologs." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Inparanoid%3A+a+comprehensive+database+of+eukaryotic+orthologs.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000262.XML" value="59262"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000262.PDF" value="162920"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000262.g001.PNG_L" value="79359"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000262.g001.PNG_M" value="28357"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000262.g001.PNG_S" value="2344"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000262.g001.TIF" value="141306"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000262.g001.PNG_I" value="15831"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000262.g002.PNG_L" value="123044"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000262.g002.PNG_M" value="36288"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000262.g002.PNG_S" value="2594"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000262.g002.TIF" value="210064"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000262.g002.PNG_I" value="20593"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000262.g003.PNG_L" value="36663"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000262.g003.PNG_M" value="36578"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000262.g003.PNG_S" value="2603"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000262.g003.TIF" value="62016"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000262.g003.PNG_I" value="17988"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000262.g004.PNG_L" value="116802"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000262.g004.PNG_M" value="33639"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000262.g004.PNG_S" value="2482"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000262.g004.TIF" value="193102"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000262.g004.PNG_I" value="18533"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000262.s001.EPS" value="5215612"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000262&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000262" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000262&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0000262&volume=&issue=&title=Evidence for Active Maintenance of Inverted Repeat Structures Identified by a Comparative Genomic Approach&author_name=Guoyan%20Zhao%2C%20Kuan%20Y.%20Chang%2C%20Katherine%20Varley%2C%20Gary%20D.%20Stormo&start_page=1&end_page=7" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000262" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000262" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0000262" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000262&amp;t=Evidence%20for%20Active%20Maintenance%20of%20Inverted%20Repeat%20Structures%20Identified%20by%20a%20Comparative%20Genomic%20Approach" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000262&title=Evidence%20for%20Active%20Maintenance%20of%20Inverted%20Repeat%20Structures%20Identified%20by%20a%20Comparative%20Genomic%20Approach&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000262&amp;title=Evidence%20for%20Active%20Maintenance%20of%20Inverted%20Repeat%20Structures%20Identified%20by%20a%20Comparative%20Genomic%20Approach" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000262" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0000262&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Evidence for Active Maintenance of Inverted Repeat Structures Identified by a Comparative Genomic Approach';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0000262';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Evidence%20for%20Active%20Maintenance%20of%20Inverted%20Repeat%20Structures%20Identified%20by%20a%20Comparative%20Genomic%20Approach http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000262" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0000262" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">















          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Bacterial+genomics%22" title="Search for articles in the subject area:'Bacterial genomics'"><div class="flagText">Bacterial genomics</div></a>
              <div data-categoryid="17227" data-articleid="23362"
                   data-categoryname="Bacterial genomics"
                   class="flagImage" title="Flag 'Bacterial genomics' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Caenorhabditis+elegans%22" title="Search for articles in the subject area:'Caenorhabditis elegans'"><div class="flagText">Caenorhabditis elegans</div></a>
              <div data-categoryid="31833" data-articleid="23362"
                   data-categoryname="Caenorhabditis elegans"
                   class="flagImage" title="Flag 'Caenorhabditis elegans' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22DNA+sequences%22" title="Search for articles in the subject area:'DNA sequences'"><div class="flagText">DNA sequences</div></a>
              <div data-categoryid="32003" data-articleid="23362"
                   data-categoryname="DNA sequences"
                   class="flagImage" title="Flag 'DNA sequences' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Genome+evolution%22" title="Search for articles in the subject area:'Genome evolution'"><div class="flagText">Genome evolution</div></a>
              <div data-categoryid="17207" data-articleid="23362"
                   data-categoryname="Genome evolution"
                   class="flagImage" title="Flag 'Genome evolution' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Invertebrate+genomics%22" title="Search for articles in the subject area:'Invertebrate genomics'"><div class="flagText">Invertebrate genomics</div></a>
              <div data-categoryid="17221" data-articleid="23362"
                   data-categoryname="Invertebrate genomics"
                   class="flagImage" title="Flag 'Invertebrate genomics' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Repeated+sequences%22" title="Search for articles in the subject area:'Repeated sequences'"><div class="flagText">Repeated sequences</div></a>
              <div data-categoryid="20009" data-articleid="23362"
                   data-categoryname="Repeated sequences"
                   class="flagImage" title="Flag 'Repeated sequences' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Sequence+alignment%22" title="Search for articles in the subject area:'Sequence alignment'"><div class="flagText">Sequence alignment</div></a>
              <div data-categoryid="18051" data-articleid="23362"
                   data-categoryname="Sequence alignment"
                   class="flagImage" title="Flag 'Sequence alignment' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Structural+genomics%22" title="Search for articles in the subject area:'Structural genomics'"><div class="flagText">Structural genomics</div></a>
              <div data-categoryid="18469" data-articleid="23362"
                   data-categoryname="Structural genomics"
                   class="flagImage" title="Flag 'Structural genomics' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=5409'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=4008'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=848&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>


</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
