

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Persistent Infection and Promiscuous Recombination of Multiple Genotypes of an RNA Virus within a Single Host Generate Extensive Diversity</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0000917"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0000917" />

    <meta name="citation_title" content="Persistent Infection and Promiscuous Recombination of Multiple Genotypes of an RNA Virus within a Single Host Generate Extensive Diversity"/>
    <meta itemprop="name" content="Persistent Infection and Promiscuous Recombination of Multiple Genotypes of an RNA Virus within a Single Host Generate Extensive Diversity"/>

      <meta name="citation_author" content="Ziming Weng"/>
            <meta name="citation_author_institution" content="Department of Plant Sciences, University of Arizona, Tucson, Arizona, United States of America"/>
      <meta name="citation_author" content="Roger Barthelson"/>
            <meta name="citation_author_institution" content="Department of Plant Sciences, University of Arizona, Tucson, Arizona, United States of America"/>
      <meta name="citation_author" content="Siddarame Gowda"/>
            <meta name="citation_author_institution" content="Citrus Research and Education Center, University of Florida, Lake Alfred, Florida, United States of America"/>
      <meta name="citation_author" content="Mark E. Hilf"/>
            <meta name="citation_author_institution" content="United States Department of Agriculture-Agricultural Research Service-United States Horticulture Research Laboratory, Fort Pierce, Florida, United States of America"/>
      <meta name="citation_author" content="William O. Dawson"/>
            <meta name="citation_author_institution" content="Citrus Research and Education Center, University of Florida, Lake Alfred, Florida, United States of America"/>
      <meta name="citation_author" content="David W. Galbraith"/>
            <meta name="citation_author_institution" content="Department of Plant Sciences, University of Arizona, Tucson, Arizona, United States of America"/>
      <meta name="citation_author" content="Zhongguo Xiong"/>
            <meta name="citation_author_institution" content="Department of Plant Sciences, University of Arizona, Tucson, Arizona, United States of America"/>

    <meta name="citation_date" content="2007/9/19"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0000917.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e917"/>
    <meta name="citation_issue" content="9"/>
    <meta name="citation_volume" content="2"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=RNA recombination in animal and plant viruses.; citation_author=MMC Lai; citation_journal_title=Microbiological Reviews; citation_volume=56; citation_number=1; citation_pages=61-79; citation_date=1992; " />
      <meta name="citation_reference" content="citation_title=The evolution of epidemic influenza.; citation_author=MI Nelson; citation_author=EC Holmes; citation_journal_title=Nature Reviews Genetics; citation_volume=8; citation_number=2; citation_pages=196-205; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Whole-genome analysis of human influenza A virus reveals multiple persistent lineages and reassortment among recent H3N2 viruses.; citation_author=EC Holmes; citation_author=E Ghedin; citation_author=N Miller; citation_author=J Taylor; citation_author=YM Bao; citation_journal_title=Plos Biology; citation_volume=3; citation_number=3; citation_pages=1579-1589; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=A phylogenetic survey of recombination frequency in plant RNA viruses.; citation_author=ER Chare; citation_author=EC Holmes; citation_journal_title=Archives of Virology; citation_volume=151; citation_number=4; citation_pages=933-946; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Recombination in the hemagglutinin gene of the 1918 “Spanish flu”.; citation_author=MJ Gibbs; citation_author=JS Armstrong; citation_author=AJ Gibbs; citation_journal_title=Science; citation_volume=293; citation_number=5; citation_pages=1842-1845; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Evidence that a plant virus switched hosts to infect a vertebrate and then recombined with a vertebrate-infecting virus.; citation_author=MJ Gibbs; citation_author=GF Weiller; citation_journal_title=Proceedings of the National Academy of Sciences of the United States of America; citation_volume=96; citation_number=6; citation_pages=8022-8027; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=A dual superinfection and recombination within HIV-1 subtype B 12 years after primoinfection.; citation_author=M Pernas; citation_author=C Casado; citation_author=R Fuentes; citation_author=MJ Perez-Elias; citation_author=C Lopez-Galindez; citation_journal_title=Jaids-Journal of Acquired Immune Deficiency Syndromes; citation_volume=42; citation_number=7; citation_pages=12-18; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Evidence of multiple recombination events between two RNA sequence variants within a Citrus tristeza virus isolate.; citation_author=MC Vives; citation_author=L Rubio; citation_author=A Sambade; citation_author=TE Mirkov; citation_author=P Moreno; citation_journal_title=Virology; citation_volume=331; citation_number=8; citation_pages=232-237; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Sequences of Citrus tristeza virus separated in time and space are essentially identical.; citation_author=MR Albiach-Marti; citation_author=M Mawassi; citation_author=S Gowda; citation_author=T Satyanarayana; citation_author=ME Hilf; citation_journal_title=Journal of Virology; citation_volume=74; citation_number=9; citation_pages=6856-6865; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Complete sequence of the Citrus tristeza virus RNA genome.; citation_author=AV Karasev; citation_author=VP Boyko; citation_author=S Gowda; citation_author=OV Nikolaeva; citation_author=ME Hilf; citation_journal_title=Virology; citation_volume=208; citation_number=10; citation_pages=511-520; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Unusual sequence relationships between two isolates of citrus tristeza virus.; citation_author=M Mawassi; citation_author=E Mietkiewska; citation_author=R Gofman; citation_author=G Yang; citation_author=M BarJoseph; citation_journal_title=Journal of General Virology; citation_volume=77; citation_number=11; citation_pages=2359-2364; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Molecular characterization of an isolate of citrus tristeza virus that causes severe symptoms in sweet orange.; citation_author=ZN Yang; citation_author=DM Mathews; citation_author=JA Dodds; citation_author=TE Mirkov; citation_journal_title=Virus Genes; citation_volume=19; citation_number=12; citation_pages=131-142; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=The continuous challenge of Citrus tristeza virus control.; citation_author=M Bar-Joseph; citation_author=R Marcus; citation_author=RF Lee; citation_journal_title=Annual Review of Phytopathology; citation_volume=27; citation_number=13; citation_pages=291-316; citation_date=1989; " />
      <meta name="citation_reference" content="citation_title=Citrus tristeza virus and its aphid vector Toxoptera citricida - threats to citrus production in the Caribbean and Central and North America.; citation_author=MA Rocha-Pena; citation_author=RF Lee; citation_author=R Lastra; citation_author=CL Niblett; citation_author=FM Ochoacorona; citation_journal_title=Plant Disease; citation_volume=79; citation_number=14; citation_pages=437-445; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Closterovirus bipolar virion: Evidence for initiation of assembly by minor coat protein and its restriction to the genomic RNA 5′ region.; citation_author=T Satyanarayana; citation_author=S Gowda; citation_author=MA Ayllon; citation_author=WO Dawson; citation_journal_title=Proceedings of the National Academy of Sciences of the United States of America; citation_volume=101; citation_number=15; citation_pages=799-804; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=The complete genome sequence of the major component of a mild citrus tristeza virus isolate.; citation_author=MC Vives; citation_author=L Rubio; citation_author=C Lopez; citation_author=J Navas-Castillo; citation_author=MR Albiach-Marti; citation_journal_title=Journal of General Virology; citation_volume=80; citation_number=16; citation_pages=811-816; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=The complete nucleotide sequence of a severe stem pitting isolate of Citrus tristeza virus from Spain: comparison with isolates from different origins.; citation_author=S Ruiz-Ruiz; citation_author=P Moreno; citation_author=J Guerri; citation_author=S Ambros; citation_journal_title=Archives of Virology; citation_volume=151; citation_number=17; citation_pages=387-398; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=D-RNA molecules associated with subisolates of the VT strain of citrus tristeza virus which induce different seedling-yellows reactions.; citation_author=GA Yang; citation_author=XB Che; citation_author=R Gofman; citation_author=Y Ben-Shalom; citation_author=D Piestun; citation_journal_title=Virus Genes; citation_volume=19; citation_number=18; citation_pages=5-13; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=An engineered closterovirus RNA replicon and analysis of heterologous terminal sequences for replication.; citation_author=T Satyanarayana; citation_author=S Gowda; citation_author=VP Boyko; citation_author=MR Albiach-Marti; citation_author=M Mawassi; citation_journal_title=Proceedings of the National Academy of Sciences of the United States of America; citation_volume=96; citation_number=19; citation_pages=7433-7438; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Three distinct suppressors of RNA silencing encoded by a 20-kb viral RNA genome.; citation_author=R Lu; citation_author=A Folimonov; citation_author=M Shintaku; citation_author=WX Li; citation_author=BW Falk; citation_journal_title=Proceedings of the National Academy of Sciences of the United States of America; citation_volume=101; citation_number=20; citation_pages=15742-15747; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=The p23 protein of citrus tristeza virus controls asymmetrical RNA accumulation.; citation_author=T Satyanarayana; citation_author=S Gowda; citation_author=MA Ayllon; citation_author=MR Albiach-Marti; citation_author=S Rabindran; citation_journal_title=Journal of Virology; citation_volume=76; citation_number=21; citation_pages=473-483; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=The p20 gene product of Citrus tristeza virus accumulates in the amorphous inclusion bodies.; citation_author=S Gowda; citation_author=T Satyanarayana; citation_author=CL Davis; citation_author=J Navas-Castillo; citation_author=MR Albiach-Marti; citation_journal_title=Virology; citation_volume=274; citation_number=22; citation_pages=246-254; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Characterization of Citrus tristeza virus subgenomic RNAs in infected tissue.; citation_author=ME Hilf; citation_author=AV Karasev; citation_author=HR Pappu; citation_author=DJ Gumpf; citation_author=CL Niblett; citation_journal_title=Virology; citation_volume=208; citation_number=23; citation_pages=576-582; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Transcriptional strategy of closteroviruses: Mapping the 5′ termini of the citrus tristeza virus subgenomic RNAs.; citation_author=AV Karasev; citation_author=ME Hilf; citation_author=SM Garnsey; citation_author=WO Dawson; citation_journal_title=Journal of Virology; citation_volume=71; citation_number=24; citation_pages=6233-6236; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Genetic marker analysis of a global collection of isolates of Citrus tristeza virus: Characterization and distribution of CTV genotypes and association with symptoms.; citation_author=ME Hilf; citation_author=VA Mavrodieva; citation_author=SM Garnsey; citation_journal_title=Phytopathology; citation_volume=95; citation_number=25; citation_pages=909-917; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Population structure and genetic diversity within California Citrus tristeza virus (CTV) isolates.; citation_author=P Kong; citation_author=L Rubio; citation_author=M Polek; citation_author=BW Falk; citation_journal_title=Virus Genes; citation_volume=21; citation_number=26; citation_pages=139-145; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Genotype classification and molecular evidence for the presence of mixed infections in Indian Citrus tristeza virus isolates.; citation_author=A Roy; citation_author=RH Brlansky; citation_journal_title=Archives of Virology; citation_volume=149; citation_number=27; citation_pages=1911-1929; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Designing and testing of a Citrus tristeza virus resequencing microarray.; citation_author=Z Xiong; citation_author=R Barthelson; citation_author=Z Weng; citation_author=DW Galbraith; citation_journal_title=Proceedings of the International Organization of Citrus Virologists; citation_volume=16; citation_number=28; citation_pages=11-22; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Tracking the evolution of the SARS coronavirus using high-throughput, high-density resequencing arrays.; citation_author=CW Wong; citation_author=TJ Albert; citation_author=VB Vega; citation_author=JE Norton; citation_author=DJ Cutler; citation_journal_title=Genome Research; citation_volume=14; citation_number=29; citation_pages=398-405; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Microarray-based resequencing of multiple Bacillus anthracis isolates.; citation_author=ME Zwick; citation_author=F McAfee; citation_author=DJ Cutler; citation_author=TD Read; citation_author=J Ravel; citation_journal_title=Genome Biology; citation_volume=6; citation_number=30; citation_pages=R10-2005; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=High-throughput variation detection and genotyping using microarrays.; citation_author=DJ Cutler; citation_author=ME Zwick; citation_author=MM Carrasquillo; citation_author=CT Yohn; citation_author=KP Tobin; citation_journal_title=Genome Research; citation_volume=11; citation_number=31; citation_pages=1913-1925; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Base-calling of automated sequencer traces using phred. I. Accuracy assessment.; citation_author=B Ewing; citation_author=L Hillier; citation_author=MC Wendl; citation_author=P Green; citation_journal_title=Genome Research; citation_volume=8; citation_number=32; citation_pages=175-185; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=MrBayes 3: Bayesian phylogenetic inference under mixed models.; citation_author=F Ronquist; citation_author=JP Huelsenbeck; citation_journal_title=Bioinformatics; citation_volume=19; citation_number=33; citation_pages=1572-1574; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Genetic variation of Citrus tristeza virus isolates from California and Spain: Evidence for mixed infections and recombination.; citation_author=L Rubio; citation_author=MA Ayllon; citation_author=P Kong; citation_author=A Fernandez; citation_author=M Polek; citation_journal_title=Journal of Virology; citation_volume=75; citation_number=34; citation_pages=8054-8062; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Molecular variability of the 5′- and 3′-terminal regions of citrus tristeza virus RNA.; citation_author=C Lopez; citation_author=MA Ayllon; citation_author=J Navas-Castillo; citation_author=J Guerri; citation_author=P Moreno; citation_journal_title=Phytopathology; citation_volume=88; citation_number=35; citation_pages=685-691; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Genetic diversity and evolution of closteroviruses.; citation_author=AV Karasev; citation_journal_title=Annual Review of Phytopathology; citation_volume=38; citation_number=36; citation_pages=293-324; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Studies on functional interaction between brome mosaic virus replicase proteins during RNA recombination, using combined mutants in vivo and in vitro.; citation_author=A Dzianott; citation_author=N Rauffer-Bruyere; citation_author=JJ Bujarski; citation_journal_title=Virology; citation_volume=289; citation_number=37; citation_pages=137-149; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Heterologous RNA replication enhancer stimulates in vitro RNA synthesis and template-switching by the carmovirus, but not by the tombusvirus, RNA-dependent RNA polymerase: Implication for modular evolution of RNA viruses.; citation_author=CP Cheng; citation_author=T Panavas; citation_author=GX Luo; citation_author=PD Nagy; citation_journal_title=Virology; citation_volume=341; citation_number=38; citation_pages=107-121; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Mechanism of RNA recombination in carmo- and tombusviruses: Evidence for template switching by the RNA-Dependent RNA polymerase in vitro.; citation_author=CP Cheng; citation_author=PD Nagy; citation_journal_title=Journal of Virology; citation_volume=77; citation_number=39; citation_pages=12033-12047; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=The AU-rich RNA recombination hot spot sequence of Brome mosaic virus is functional in tombusviruses: Implications for the mechanism of RNA recombination.; citation_author=N Shapka; citation_author=PD Nagy; citation_journal_title=Journal of Virology; citation_volume=78; citation_number=40; citation_pages=2288-2300; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Factors regulating template switch in vitro by viral RNA-dependent RNA polymerases: Implications for RNA-RNA recombination.; citation_author=MJ Kim; citation_author=C Kao; citation_journal_title=Proceedings of the National Academy of Sciences of the United States of America; citation_volume=98; citation_number=41; citation_pages=4972-4977; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Detection of high levels of recombination generated during PCR amplification of RNA templates.; citation_author=W Yu; citation_author=KJ Rusterholtz; citation_author=AT Krummel; citation_author=N Lehman; citation_journal_title=Biotechniques; citation_volume=40; citation_number=42; citation_pages=499-507; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Identifying influenza viruses with resequencing microarrays.; citation_author=Z Wang; citation_author=LT Daum; citation_author=GJ Vora; citation_author=D Metzgar; citation_author=EA Walter; citation_journal_title=Emerging Infectious Diseases; citation_volume=12; citation_number=43; citation_pages=638-646; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Broad-spectrum respiratory tract pathogen identification using resequencing DNA microarrays.; citation_author=BC Lin; citation_author=Z Wang; citation_author=GJ Vora; citation_author=JA Thornton; citation_author=JM Schnur; citation_journal_title=Genome Research; citation_volume=16; citation_number=44; citation_pages=527-535; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Using a resequencing microarray as a multiple respiratory pathogen detection assay.; citation_author=BC Lin; citation_author=KM Blaney; citation_author=AP Malanoski; citation_author=AG Ligler; citation_author=JM Schnur; citation_journal_title=Journal of Clinical Microbiology; citation_volume=45; citation_number=45; citation_pages=443-452; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=GeneChip resequencing of the smallpox virus genome can identify novel strains: a biodefense application.; citation_author=IM Sulaiman; citation_author=K Tang; citation_author=J Osborne; citation_author=S Sammons; citation_author=RA Wohlheuter; citation_journal_title=Journal of Clinical Microbiology; citation_volume=45; citation_number=46; citation_pages=358-363; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=The CLUSTAL_X windows interface: flexible strategies for multiple sequence alignment aided by quality analysis tools.; citation_author=JD Thompson; citation_author=TJ Gibson; citation_author=F Plewniak; citation_author=F Jeanmougin; citation_author=DG Higgins; citation_journal_title=Nucleic Acids Research; citation_volume=25; citation_number=47; citation_pages=4876-4882; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=TreeView: An application to display phylogenetic trees on personal computers.; citation_author=RDM Page; citation_journal_title=Computer Applications in the Biosciences; citation_volume=12; citation_number=48; citation_pages=357-358; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=RDP2: recombination detection and analysis from sequence alignments.; citation_author=DP Martin; citation_author=C Williamson; citation_author=D Posada; citation_journal_title=Bioinformatics; citation_volume=21; citation_number=49; citation_pages=260-262; citation_date=2005; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000917" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Persistent Infection and Promiscuous Recombination of Multiple Genotypes of an RNA Virus within a Single Host Generate Extensive Diversity"/>
    <meta name="twitter:description" content="Recombination and reassortment of viral genomes are major processes contributing to the creation of new, emerging viruses. These processes are especially significant in long-term persistent infections where multiple viral genotypes co-replicate in a single host, generating abundant genotypic variants, some of which may possess novel host-colonizing and pathogenicity traits. In some plants, successive vegetative propagation of infected tissues and introduction of new genotypes of a virus by vector transmission allows for viral populations to increase in complexity for hundreds of years allowing co-replication and subsequent recombination of the multiple viral genotypes. Using a resequencing microarray, we examined a persistent infection by a Citrus tristeza virus (CTV) complex in citrus, a vegetatively propagated, globally important fruit crop, and found that the complex comprised three major and a number of minor genotypes. Subsequent deep sequencing analysis of the viral population confirmed the presence of the three major CTV genotypes and, in addition, revealed that the minor genotypes consisted of an extraordinarily large number of genetic variants generated by promiscuous recombination between the major genotypes. Further analysis provided evidence that some of the recombinants underwent subsequent divergence, further increasing the genotypic complexity. These data demonstrate that persistent infection of multiple viral genotypes within a host organism is sufficient to drive the large-scale production of viral genetic variants that may evolve into new and emerging viruses."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0000917.g004"/>

  <meta property="og:title" content="Persistent Infection and Promiscuous Recombination of Multiple Genotypes of an RNA Virus within a Single Host Generate Extensive Diversity" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000917" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=8570'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=5453'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=5093&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0000917">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000917" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000917&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Persistent Infection and Promiscuous Recombination of Multiple Genotypes of an RNA Virus within a Single Host Generate Extensive Diversity
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Ziming Weng, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Plant Sciences, University of Arizona, Tucson, Arizona, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Roger Barthelson, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Plant Sciences, University of Arizona, Tucson, Arizona, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Siddarame Gowda, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Citrus Research and Education Center, University of Florida, Lake Alfred, Florida, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Mark E. Hilf, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  United States Department of Agriculture-Agricultural Research Service-United States Horticulture Research Laboratory, Fort Pierce, Florida, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            William O. Dawson, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Citrus Research and Education Center, University of Florida, Lake Alfred, Florida, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            David W. Galbraith, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Plant Sciences, University of Arizona, Tucson, Arizona, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Zhongguo Xiong
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:zxiong@email.arizona.edu">zxiong@email.arizona.edu</a></p>

                <p>Affiliation:
                  Department of Plant Sciences, University of Arizona, Tucson, Arizona, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: September 19, 2007</li>
    <li>DOI: 10.1371/journal.pone.0000917</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0000917" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0000917" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000917" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0000917" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0000917-t001" data-doi="info:doi/10.1371/journal.pone.0000917" data-uri="info:doi/10.1371/journal.pone.0000917.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000917.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000917-g001" data-doi="info:doi/10.1371/journal.pone.0000917" data-uri="info:doi/10.1371/journal.pone.0000917.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000917.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000917-t002" data-doi="info:doi/10.1371/journal.pone.0000917" data-uri="info:doi/10.1371/journal.pone.0000917.t002" title="Table 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000917.t002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000917-g002" data-doi="info:doi/10.1371/journal.pone.0000917" data-uri="info:doi/10.1371/journal.pone.0000917.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000917.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000917-g003" data-doi="info:doi/10.1371/journal.pone.0000917" data-uri="info:doi/10.1371/journal.pone.0000917.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000917.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000917-g004" data-doi="info:doi/10.1371/journal.pone.0000917" data-uri="info:doi/10.1371/journal.pone.0000917.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000917.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000917">Reader Comments (1)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0000917" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1"></a><p>Recombination and reassortment of viral genomes are major processes contributing to the creation of new, emerging viruses. These processes are especially significant in long-term persistent infections where multiple viral genotypes co-replicate in a single host, generating abundant genotypic variants, some of which may possess novel host-colonizing and pathogenicity traits. In some plants, successive vegetative propagation of infected tissues and introduction of new genotypes of a virus by vector transmission allows for viral populations to increase in complexity for hundreds of years allowing co-replication and subsequent recombination of the multiple viral genotypes. Using a resequencing microarray, we examined a persistent infection by a <em>Citrus tristeza virus</em> (CTV) complex in citrus, a vegetatively propagated, globally important fruit crop, and found that the complex comprised three major and a number of minor genotypes. Subsequent deep sequencing analysis of the viral population confirmed the presence of the three major CTV genotypes and, in addition, revealed that the minor genotypes consisted of an extraordinarily large number of genetic variants generated by promiscuous recombination between the major genotypes. Further analysis provided evidence that some of the recombinants underwent subsequent divergence, further increasing the genotypic complexity. These data demonstrate that persistent infection of multiple viral genotypes within a host organism is sufficient to drive the large-scale production of viral genetic variants that may evolve into new and emerging viruses.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>Weng Z, Barthelson R, Gowda S, Hilf ME, Dawson WO, et al.  (2007) Persistent Infection and Promiscuous Recombination of Multiple Genotypes of an RNA Virus within a Single Host Generate Extensive Diversity. PLoS ONE 2(9):
          e917.
            doi:10.1371/journal.pone.0000917</p><p><strong>Academic Editor: </strong>Mark Gibbs, The Australian National University, Australia</p><p><strong>Received:</strong> June 1, 2007; <strong>Accepted:</strong> August 20, 2007; <strong>Published:</strong> September 19, 2007</p><p><strong>Copyright:</strong> © 2007 Weng et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>This work was supported by funding from USDA grants 2003-34399-13764 and 2005-34399-16070 to ZX and DWG.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>The emergence of new viruses is a constant challenge to the well-being of the human race and its food supply. New viruses or viral strains are produced from existing forms as a consequence of two processes: mutation and recombination or reassortment, which occur in both plant and animal hosts <a href="#pone.0000917-Lai1">[1]</a>–<a href="#pone.0000917-Gibbs2">[6]</a>. The potential for recombination and reassortment is greatly enhanced in persistent and chronic infections, in which multiple genotypes of a single viral species, or multiple viral species, are introduced into a single host through repeated infections <a href="#pone.0000917-Pernas1">[7]</a>, <a href="#pone.0000917-Vives1">[8]</a>. Co-replicating viral genotypes create an environment conducive for RNA recombination to generate potentially new combinations of genes or protein domains that are exponential to the number of genotypes in the mixed infection. These recombinants may evolve subsequently into new and emerging viruses <a href="#pone.0000917-Lai1">[1]</a>, <a href="#pone.0000917-Chare1">[4]</a>–<a href="#pone.0000917-Pernas1">[7]</a>. However, the extent to which such long-term infections result in genotypic variants remains largely unexplored.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p><em>Citrus tristeza virus</em> (CTV) represents an example of a virus that causes persistent infections in a long-lived, economically important hardwood perennial crop plant, so that with time a single host plant may become infected by multiple, phylogenetically distinct CTV genotypes. CTV is a member of the genus <em>Closterovirus</em> within the family <em>Closteroviridae</em> <a href="#pone.0000917-AlbiachMarti1">[9]</a>–<a href="#pone.0000917-Yang1">[12]</a>, and is the most important and destructive virus of citrus <a href="#pone.0000917-BarJoseph1">[13]</a>, <a href="#pone.0000917-RochaPena1">[14]</a>. CTV virions are flexuous rods, 2000 nm in length and 12 nm in diameter, consisting of one single-stranded, (+)-sense RNA genome encapsidated by two species of coat proteins (97% CP and 3% CPm) <a href="#pone.0000917-Satyanarayana1">[15]</a>. The 19.2 to 19.3 kb genome contains 12 open reading frames, is the largest of the plant RNA viruses, and is one of the largest of all RNA viruses <a href="#pone.0000917-AlbiachMarti1">[9]</a>–<a href="#pone.0000917-Mawassi1">[11]</a>, <a href="#pone.0000917-Vives2">[16]</a>–<a href="#pone.0000917-Yang2">[18]</a>. The 5′ half of the genome (~nt. 1–11,000) encodes proteins (RNA-dependent RNA polymerase, helicase, methyltransferase, and proteases) that are required for viral replication <a href="#pone.0000917-Satyanarayana2">[19]</a> and are thought to be translated directly from the genomic RNA. The 3′ half encodes proteins which are thought to interact with host plants <a href="#pone.0000917-Lu1">[20]</a>–<a href="#pone.0000917-Gowda1">[22]</a> and are expressed from ten 3′ co-terminal subgenomic RNAs <a href="#pone.0000917-Hilf1">[23]</a>, <a href="#pone.0000917-Karasev2">[24]</a>.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>The global CTV population is very diverse, with numerous, disparate strains <a href="#pone.0000917-RochaPena1">[14]</a>, <a href="#pone.0000917-Hilf2">[25]</a>, many inducing different types and degrees of disease symptoms on different citrus species and varieties. Often in natural infections in the field, CTV exists as a complex comprising multiple strains or genotypes, due to the longevity of individual citrus trees and the extensive use of vegetative propagation of budwood. Continual vertical transmission coupled with repeated horizontal transmission mediated by aphids throughout the history of citrus cultivation has led to the complexity of the CTV population increasing over hundreds of years, resulting in the co-existence of multiple CTV genotypes in a single host <a href="#pone.0000917-Vives1">[8]</a>, <a href="#pone.0000917-Kong1">[26]</a>, <a href="#pone.0000917-Roy1">[27]</a>. The presence within a host of multiple replicating CTV genotypes and the relatively long periods of co-replication create opportunities for recombination between the genotypes, leading to extensive viral diversity. In this report, we characterized a persistent infection by multiple CTV genotypes by genome-wide microarray resequencing analysis and deep sequencing analysis of selected genomic regions. Our results demonstrate an extraordinary amount of viral variability generated by promiscuous recombination between multiple genotypes, and provide evidence for subsequent divergence of the recombinants within a single host plant.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3>
<h4>Resequencing analysis of FS2-2 reveals presence of multiple CTV genotypes</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>To study the CTV genetic complexity of CTV in detail at the sequence level, we designed and validated an Affymetrix resequencing microarray that queries entire genomes of multiple, phylogenetically distinct CTV genotypes <a href="#pone.0000917-Xiong1">[28]</a>. Sequences tiled on the microarray include full-length sequences of four CTV type strains, T3 (Hilf, unpublished), T30 <a href="#pone.0000917-AlbiachMarti1">[9]</a>, T36 <a href="#pone.0000917-Karasev1">[10]</a>, and VT <a href="#pone.0000917-Mawassi1">[11]</a>, three-quarters of the strain T68-1 genome (Hilf, unpublished), as well as unique genomic sequences identified from five other CTV isolates <a href="#pone.0000917-Xiong1">[28]</a>. Together, over 117 kb of CTV sequences representing a genetic diversity equivalent to ten full-length CTV genomes (<a href="#pone-0000917-t001">Table 1</a>) were tiled on the resequencing microarray. Using known CTV isolates as the source for target cDNAs, the CTV resequencing microarray yielded call rates of ~99.7–99.8% and call accuracies of ~99.9–100% <a href="#pone.0000917-Xiong1">[28]</a>, performing comparably to, or better than, several reported resequencing microarrays <a href="#pone.0000917-Wong1">[29]</a>, <a href="#pone.0000917-Zwick1">[30]</a>.</p>
<div class="figure" id="pone-0000917-t001"><div class="img"><a name="pone-0000917-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000917.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000917" data-uri="info:doi/10.1371/journal.pone.0000917.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000917.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000917.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000917.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000917.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000917.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000917.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000917.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000917.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000917.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>Numbers of nucleotides tiled on the CTV resequencing microarray from the indicated source isolates.</span></strong></p><span>doi:10.1371/journal.pone.0000917.t001</span></div><a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2"></a><p>We subsequently employed the microarray for analysis of the genetic complexity in a natural CTV isolate, FS2-2. The isolate, collected originally from a citrus grove in Florida, was associated with an unusual stem-pitting symptom in the Hamlin sweet orange. The isolate was initially suspected to possess multiple CTV genotypes on the basis of PCR-amplification with genotype-specific primers (data not shown). The multiple CTV genomes present in FS2-2 were amplified in their entirety by long-range RT-PCR with a high fidelity polymerase from total RNA prepared from infected tissues, using four sets of universal CTV primers (<a href="#pone-0000917-t002">Table 2</a>) capable of amplifying all known CTV genomes. DNA fragments of expected sizes were amplified from the total RNA extracted from FS2-2-infected tissues (<a href="#pone-0000917-g001">Fig. 1</a>). Furthermore, each of the three unique 5′ end PCR primers, designed specifically for CTV genomes with very different 5′ terminal sequences, successfully amplified DNA fragments of the predicted sizes in RT-PCR (<a href="#pone-0000917-g001">Fig. 1</a>), indicating the presence of multiple genotypes in the FS2-2 CTV complex. The three 5′ DNA fragments and the three DNA fragments amplified from the remaining portion of the CTV genomes were pooled proportionally, and then were processed for microarray hybridization <a href="#pone.0000917-Xiong1">[28]</a>.</p>
<div class="figure" id="pone-0000917-g001"><div class="img"><a name="pone-0000917-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000917.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000917" data-uri="info:doi/10.1371/journal.pone.0000917.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000917.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000917.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000917.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000917.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000917.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000917.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000917.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000917.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000917.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>DNA fragments representing the CTV genome amplified by RT-PCR.</span></strong></p><a id="article1.body1.sec2.sec1.fig1.caption1.p1" name="article1.body1.sec2.sec1.fig1.caption1.p1"></a><p>Total RNA prepared from FS2-2-infected sweet orange leaves was used as the template for RT-PCR. Lanes 1-3, DNA amplified by 3′ primer CTV5427R and three 5′ primers: CTV5endFT36, CTV5endFVT, and FT30CTV5end, respectively; lane 4, DNA amplified by primers CTV5403F and CTV09997R; lane 5, DNA amplified by primers CTV09262F and CTV14630R; and lane 6, DNA amplified by primers CTV14469F and CTV19395R.</p>
<span>doi:10.1371/journal.pone.0000917.g001</span></div><div class="figure" id="pone-0000917-t002"><div class="img"><a name="pone-0000917-t002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000917.t002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000917" data-uri="info:doi/10.1371/journal.pone.0000917.t002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000917.t002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000917.t002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000917.t002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000917.t002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000917.t002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000917.t002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000917.t002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000917.t002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000917.t002.TIF"></span>)
                </a></li></ul></div><p><strong>Table 2.  <span>Primers for RT-PCR amplification of CTV genome and genomic regions.</span></strong></p><span>doi:10.1371/journal.pone.0000917.t002</span></div><a id="article1.body1.sec2.sec1.p3" name="article1.body1.sec2.sec1.p3"></a><p>The existence of multiple CTV genotypes in the FS2-2 complex was clearly evident in the scanned images of the hybridized resequencing microarray (<a href="#pone-0000917-g002">Fig. 2</a>). Hybridization with target DNA prepared from a single genotype generated a single block of intensive hybridization corresponding to the location on the microarray of the tiling path for that genome (e.g. hybridizations with the T30 and the T36 target DNA in <a href="#pone-0000917-g002">Fig. 2</a>). In comparison, hybridization with the FS2-2 target DNA yielded strong signals in multiple microarray blocks corresponding to the tiled genomes of VT, T30, and T36, indicating that the FS2-2 isolate contained at least three genotypes.</p>
<div class="figure" id="pone-0000917-g002"><div class="img"><a name="pone-0000917-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000917.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000917" data-uri="info:doi/10.1371/journal.pone.0000917.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000917.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000917.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000917.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000917.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000917.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000917.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000917.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000917.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000917.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Images of hybridized resequence microarrays.</span></strong></p><a id="article1.body1.sec2.sec1.fig2.caption1.p1" name="article1.body1.sec2.sec1.fig2.caption1.p1"></a><p>The Affymetrix CTV Microarray chips were hybridized with the target DNA indicated at the top of each panel. Hybridized microarrays were scanned at a resolution of 1.5 µm per pixel. Warm colors represent higher hybridization intensities and cool colors represent lower hybridization intensities. Locations of CTV genomes tiled on the microarray are indicated to the left. Single genome blocks are hybridized with T36 and T30 target DNA while multiple genome blocks are hybridized with FS2-2 target DNA.</p>
<span>doi:10.1371/journal.pone.0000917.g002</span></div><a id="article1.body1.sec2.sec1.p4" name="article1.body1.sec2.sec1.p4"></a><p>Hybridization intensities of each probe on the resequencing microarray were subsequently processed using the Affymetrix GeneChip Operating Software version 1.4. Base calls were made using the ABACUS algorithm <a href="#pone.0000917-Cutler1">[31]</a> with a haploid model as implemented in the Affymetrix GeneChip Sequence Analysis Software (GSEQ) version 4.0. A total of 252 sequence fragments corresponding to full length CTV genomes and genomic fragments tiled on the microarray were produced by the resequencing analysis. These fragments and the quality scores associated with each base call were used in contig assembly by the Phrap program <a href="#pone.0000917-Ewing1">[32]</a> as implemented in the CodonCode Aligner program. Three consensus contigs of complete CTV genomes (fs2_2_vt, fs2_2_t30, and fs2_2_t36), corresponding to the three visually identified CTV genotypes, were assembled (<a href="#pone-0000917-g003">Fig. 3a</a>). In addition, two partial consensus contigs were also assembled (fs2_2_t3 and fs2_2_t68). Both were placed as intermediates between fs2_2_vt and fs2_2_t30 by Bayesian phylogenetic analysis <a href="#pone.0000917-Ronquist1">[33]</a> (<a href="#pone-0000917-g003">Fig. 3a</a>), and therefore might represent minor components of the CTV complex or variants generated by recombination between the major genotypes.</p>
<div class="figure" id="pone-0000917-g003"><div class="img"><a name="pone-0000917-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000917.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000917" data-uri="info:doi/10.1371/journal.pone.0000917.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000917.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000917.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000917.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000917.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000917.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000917.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000917.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000917.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000917.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Bayesian phylogenetic inference of CTV genomes and genome fragments.</span></strong></p><a id="article1.body1.sec2.sec1.fig3.caption1.p1" name="article1.body1.sec2.sec1.fig3.caption1.p1"></a><p>Unrooted, consensus phylogenetic trees were obtained from 2,000,000 generations of the Markov chain Monte Carlo simulation in Bayesian analysis using a general time-reversal model of nucleotide substitution <a href="#pone.0000917-Ronquist1">[33]</a>. The number above each branch indicates the Bayesian posterior probability. The scale bars represent 0.1 expected substitutions per site. Branch lengths are proportional to evolutionary distance. Sequences were aligned using ClustalX <a href="#pone.0000917-Thompson1">[47]</a> and subsequently manually aligned prior to the Bayesian phylogenetic analysis. A, Known CTV genomes and CTV genomes assembled from resequencing analysis of FS2-2 (highlighted orange). The suffix at the end of fs2_2 distinguishes multiple genotypes in the isolate and also indicates the anchor sequence from which the consensus contig was generated by the Phrap program. B, the 5′ proximal 1 kb, and C, p33-coding region of CTV genomes obtained by direct sequencing of RT-PCR clones. In both B and C, Bayesian posterior probability and clones with identical sequences were omitted for clarity. Recombinant sequences are highlighted in green.</p>
<span>doi:10.1371/journal.pone.0000917.g003</span></div>

<h4>Deep sequencing analysis of the 5′ 1 kb of isolate FS2-2 confirms existence of multiple genotypes</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>To verify the presence of multiple CTV genotypes in the FS2-2 complex, a fragment of approximately 1 kb in size, corresponding to the 5′ termini of the CTV genomes, was amplified by RT-PCR and cloned. The 5′ terminal region was targeted because of its unusually high sequence variability in the CTV genomes. Previous comparative sequence analysis revealed an interesting and unusual distribution of sequence variation across the CTV genome <a href="#pone.0000917-Mawassi1">[11]</a>, <a href="#pone.0000917-Yang1">[12]</a>, <a href="#pone.0000917-Vives2">[16]</a>, <a href="#pone.0000917-Rubio1">[34]</a>. The 3′ halves of the CTV genomes were highly conserved, with sequence identity being at or above 90%, as would be expected of strains of the same virus. However, sequences of the 5′ halves were much more divergent, with pairwise comparisons yielding sequence identities of 70% or less for some isolates. Sequence identities as low as 48% was observed at the 5′ untranslated regions (UTRs) <a href="#pone.0000917-AlbiachMarti1">[9]</a>, <a href="#pone.0000917-Lopez1">[35]</a>. The 5′ half of the CTV genome encodes a polyprotein that contains motifs characteristic of RNA-dependent RNA polymerases, helicases, methyltransferases, and proteases, and is required for viral replication <a href="#pone.0000917-Satyanarayana2">[19]</a>. Generally speaking, viral proteins required for replication are highly conserved within viral species and genera, and occasionally this conservation may extend across an entire family of viruses.</p>
<a id="article1.body1.sec2.sec2.p2" name="article1.body1.sec2.sec2.p2"></a><p>The 5′ genomic fragment of CTV targeted for RT-PCR amplification includes the highly divergent 5′ UTRs of approximately 100 nucleotides, and approximately 830 nucleotides from the 5′ open reading frame that encodes for the polyprotein required for CTV replication. RT-PCR amplification of the fragment was accomplished using a combination of three 5′ end primers and a universal, 3′ primer (<a href="#pone-0000917-t002">Table 2</a>) that were designed for the amplification of all known CTV sequences at the 5′ end. A total of 70 clones were randomly selected and sequenced using an ABI 3730XL DNA Analyzer. Among the clones sequenced in this manner, 15 contained sequences nearly identical (98.7~99.8% identity) to the reported T36 sequence <a href="#pone.0000917-Karasev1">[10]</a>. Sequences of 20 clones were nearly identical (99.3~99.6% identity) to the published T30 sequence <a href="#pone.0000917-AlbiachMarti1">[9]</a>, while another 34 clones produced sequences highly similar (97.6~97.9% identity) to that of VT <a href="#pone.0000917-Mawassi1">[11]</a>. These results unequivocally identified three CTV genotypes in the FS2-2 complex that corresponded to the three major genotypes revealed by the resequencing microarray. Within each of the T30-like and the VT-like genotypes, the clones were nearly homogeneous, with a sequence identity of 99.3~100%. However, sequences within the T36-like genotype were found to be slightly more diverse, with a sequence identity ranging from 98.6% to 100%, suggesting that this genotype might have evolved faster and started to diversify within the population. Indeed, Bayesian phylogenetic analysis showed that three sequences (fs22_78, fs22_79, and fs22_83) of the T36-like genotype emerged as a distinguishable subclade within the T36-like clade (<a href="#pone-0000917-g003">Fig. 3b</a>).</p>
<a id="article1.body1.sec2.sec2.p3" name="article1.body1.sec2.sec2.p3"></a><p>Different degrees of divergence were also evident in the number of identical sequences obtained from clones of each of the three major genotypes. Nearly one half (15) of the clones from the VT-like genotype were completely identical, indicating that genomes of the VT-like genotype were quite uniform with little divergence. In contrast, only three of the 15 sequenced clones of the T36-like genotype contained the same sequence, suggesting the genotype has diverged more significantly. Sequence divergence of the T30-like genotype appeared to be between those of the T36- and VT-like genotypes. Among the 20 T30-like clones sequenced, one group of four clones and another group of five clones produced identical sequences with a single nucleotide difference between the groups.</p>
<a id="article1.body1.sec2.sec2.p4" name="article1.body1.sec2.sec2.p4"></a><p>More interestingly, one sequenced clone, fs22_28, occupied an odd place between the T30-like clade and the VT-like clade in the phylogenetic tree (<a href="#pone-0000917-g003">Fig. 3b</a>). This placement was similar to that of the two minor genotypes (fs2_2_t3 and fs2_2_t68) identified by the resequencing analysis (<a href="#pone-0000917-g003">Fig. 3a</a>). Therefore, fs22_28 likely represented one of the minor genotypes identified in the resequencing analysis. Further analysis revealed fs22-28 to be recombinant: its 5′ region comprised 282 nucleotides identical to the sequences of 7 independently sequenced clones from the T30-like genotype, and its 3′ region consisted of 574 nucleotides identical to sequences in 22 sequenced clones of the VT-like genotype. The recombination crossover site in fs22_28 comprised 41 nucleotides identical to sequences within both parental genotypes. Although the recombinant clone represents one of the minor genotypes detected by the resequencing microarray, direct cloning and sequencing of the 5′ 1 kb of the CTV genomes in the FS2-2 complex failed to identify all the minor genotypes.</p>


<h4>CTV p33 ORF exhibits promiscuous genotypic recombination</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>To search for additional CTV genotypes identified by resequencing analysis in the FS2-2 complex, an additional 1 kb region containing the entire p33 ORF was selected for further RT-PCR cloning and deep sequencing analysis. The p33 ORF of CTV has no homologue in other closteroviruses <a href="#pone.0000917-Karasev3">[36]</a>. The protein it encodes has been shown to be required for CTV movement in specific hosts (Dawson, unpublished data). The p33 ORF, located about 11 kb from the 5′ end of the genome, is more conserved among CTV isolates than is the 5′ end 1 kb region, yet it retains significant sequence divergence for differentiation of strains and isolates. The higher sequence conservation made it possible to design a set of universal RT-PCR primers capable of amplifying all known CTV genotypes non-selectively (<a href="#pone-0000917-t002">Table 2</a>).</p>
<a id="article1.body1.sec2.sec3.p2" name="article1.body1.sec2.sec3.p2"></a><p>A total of 84 RT-PCR clones derived from the p33 ORF of isolate FS2-2 were chosen randomly for deep sequencing analysis. Sequencing data from the 983 nucleotides of these clones (<a href="#pone-0000917-g003">Fig. 3c</a>) provided a detailed portrait of the CTV complex, and of on-going, promiscuous recombination among various major CTV genotypes. Among the sequenced clones, the three major genotypes again were well represented with 51 VT-like clones, twelve T30-like clones, and six T36-like clones. Several groups of clones shared identical sequences (groups of nine, seven, five, five, two, two, and two clones for the VT-like genotype; and groups of two and two clones for the T30-like genotype) while all T36-like clones had unique sequences. These confirmed that the VT-like genotypes were more homogeneous and the T36-like genotypes were more divergent in FS2-2, an observation arising earlier from analysis of the 5′ terminal CTV fragments. The large number of identical sequences obtained from these clones, as well as from clones derived from the 5′ 1 kb region, also demonstrated that the RT-PCR employed in this study maintained a high fidelity during viral sequence amplification and that errors introduced during RT-PCR were negligible.</p>
<a id="article1.body1.sec2.sec3.p3" name="article1.body1.sec2.sec3.p3"></a><p>A surprisingly large proportion of the clones (15 clones, 17.9%) were recombinants (<a href="#pone-0000917-g004">Fig. 4</a>). Eight were recombinants between fs2_2_vt and fs2_2_t36, five were recombinants between fs2_2_t30 and fs2_2_vt, and two were recombinants between fs2_2_t30 and fs2_2_t36. Further, four recombinants contained two crossovers, resulting from either a double-crossover or two independent recombination events. The parental sequences for each recombinant were readily identified among the three major genotypes. The crossover sites appeared throughout the p33 ORF without an apparent recombination hotspot. One common feature of these recombinants was that recombination crossover sites contained a stretch of sequence that was identical in both parental molecules. The identical sequences at the crossover sites varied in length, ranging from as short as 3 nucleotides to as long as 25 nucleotides. This feature of shared parental sequences suggested that the recombinants were most likely formed by a template-switching event during viral RNA replication <a href="#pone.0000917-Dzianott1">[37]</a>–<a href="#pone.0000917-Cheng2">[39]</a>. Some of the crossover sites were characteristic of classical recombination hotspots with a stretch of AU-rich sequence that promotes dissociation and switching of RNA polymerase from one template to another <a href="#pone.0000917-Shapka1">[40]</a>, <a href="#pone.0000917-Kim1">[41]</a>. However, others lacked such sequences.</p>
<div class="figure" id="pone-0000917-g004"><div class="img"><a name="pone-0000917-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000917.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000917" data-uri="info:doi/10.1371/journal.pone.0000917.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000917.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000917.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000917.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000917.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000917.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000917.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000917.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000917.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000917.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>Schematic of CTV recombinants in the p33 ORF.</span></strong></p><a id="article1.body1.sec2.sec3.fig1.caption1.p1" name="article1.body1.sec2.sec3.fig1.caption1.p1"></a><p>The different parental sequences are represented by specific colored lines; blue (VT), red (T36) and green (T30). Recombinants with identical crossover sites are connected by brackets. Asterisks between the paired recombinants denote locations of diverged nucleotides.</p>
<span>doi:10.1371/journal.pone.0000917.g004</span></div><a id="article1.body1.sec2.sec3.p4" name="article1.body1.sec2.sec3.p4"></a><p>These recombinants represented a remarkable amount of genetic variability (<a href="#pone-0000917-g003">Fig. 3c</a>). A large number of recombinants were placed as intermediates between the VT clade and the T30 clade by Bayesian analysis. These recombinants could represent the minor genotypes detected by the resequencing microarray. Since the resequencing microarray is not sensitive enough to differentiate individual recombinants, the minor genotypes identified by the resequencing analysis could very well represent composites of these recombinant genomes. Although the crossover sites were scattered throughout the p33 coding region, all but one of the recombinants maintained the correct reading frame, suggesting that the recombinant p33 protein was also expressed. Therefore, this genetic diversity was expressed at both nucleotide and protein levels.</p>
<a id="article1.body1.sec2.sec3.p5" name="article1.body1.sec2.sec3.p5"></a><p>To eliminate the possibility that the recombinant clones were artificially generated by PCR amplification from a pool of DNA template representing three different genotypes, a control PCR amplification was carried out. Equimolar amounts of two cloned DNA molecules, representing the p33 ORF cloned from the VT-like genotype and from the T36-like genotype respectively, were combined to mimic a mixture of different genotypes, and this was used as the template for the control PCR amplification with the same set of PCR primers and identical PCR conditions. The DNA fragments amplified from this artificial mixture were cloned, and 75 random clones were sequenced. Sequence analysis showed that 26 clones were derived from the T36-like genotype and 49 were derived from the VT genotype. More importantly, none of the 75 sequenced clones was recombinant between the two genotypes present in the template DNA. This result strongly suggests that the extraordinarily large number of recombinant viral molecules detected in the FS2-2 CTV complex represent natural recombinants. Although generation of artificial recombinant during PCR has been reported under specific conditions <a href="#pone.0000917-Yu1">[42]</a>, our results clearly showed that the template DNA molecules and the PCR conditions used in these experiments did not favor the formation of artificial recombinants.</p>
<a id="article1.body1.sec2.sec3.p6" name="article1.body1.sec2.sec3.p6"></a><p>Further analysis of the recombinants shed light on the emergence of new CTV genotypes through recombination and subsequent divergent evolution. This was particularly evident in two pairs of recombinants (<a href="#pone-0000917-g004">Fig. 4</a>). The first pair (fs22-05 and fs22-36) contained an identical recombination crossover site but four divergent nucleotides located both 5′ and 3′ of the crossover site. A plausible explanation is that they originated from a single recombination event and that progenies of the recombinant diverged subsequently. The second pair of recombinants (fs22-15 and fs22-100) contained an identical crossover site at the 5′ end, but fs22-15 possessed an additional crossover site toward the 3′ end. One probable explanation is that both were the progeny of a single recombination event at the 5′ end with fs22-15 subsequently undergoing a further round of recombination. In the identical region of the two recombinants, three divergent nucleotides were found spanning the 5′ crossover site, indicating that the regions had diverged after the first recombination event. These data suggest an event line of recombination, divergence, and further recombination, leading to a distinct CTV genotype. This scenario also explains the chimeric nature of the sequenced SY568 CTV genome <a href="#pone.0000917-Yang1">[12]</a> as a consequence of recombination between two CTV genotypes <a href="#pone.0000917-Vives1">[8]</a>.</p>

</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p>The complexity of a natural CTV isolate consisting of multiple genotypes was revealed through a combination of genome-wide resequencing analysis using a CTV Affymetrix GeneChip resequencing microarray and deep sequencing of selected genomic regions in this study. Furthermore, deep sequencing analysis illustrated an unparalleled level of promiscuous recombination between multiple, co-replicating genotypes in a persistent infection.</p>
<a id="article1.body1.sec3.p2" name="article1.body1.sec3.p2"></a><p>Viruses have been a favorite target for resequencing analysis using oligonucleotide microarrays because of their small genome sizes <a href="#pone.0000917-Wong1">[29]</a>, <a href="#pone.0000917-Wang1">[43]</a>–<a href="#pone.0000917-Sulaiman1">[46]</a>. However, most of these applications either targeted viruses with relatively little sequence variability, such as severe acute respiratory syndrome virus <a href="#pone.0000917-Wong1">[29]</a>, or only selected regions of viruses having larger sequence variability <a href="#pone.0000917-Wang1">[43]</a>–<a href="#pone.0000917-Lin2">[45]</a>. With increased tiling capacity, the Affymetrix resequencing microarrays can now be used to simultaneously detect divergent viruses, as we have demonstrated in our work, where three major genotypes and additional minor genotypes of CTV were successfully identified from isolate FS2-2 by resequencing analysis. The three major CTV genotypes investigated by the resequencing analysis were quite divergent with genome-wide nucleotide differences of 11% between the VT- and the T30-like genotypes, 20% between the VT- and the T36-like genotypes, and 19% between the T30- and the T36-like genotypes. Successful resequencing analysis of these divergent CTV genomes with a single resequencing microarray chip is largely attributed to the tiling strategy of selecting representative full-length CTV genomes guided by phylogenetic analysis and of selecting unique sequences identified by pair-wise comparisons from other CTV genomes. This strategy allows the CTV resequencing microarray with a tiling capacity of 117 kb to encompass a sequence diversity equivalent to ten full-length CTV genomes. With its ability to query concurrently entire genomes of multiple CTV genotypes as demonstrated in this work, the CTV resequencing microarray will find many uses in future CTV studies.</p>
<a id="article1.body1.sec3.p3" name="article1.body1.sec3.p3"></a><p>CTV has been known to exist as a complex in nature <a href="#pone.0000917-Vives1">[8]</a>, <a href="#pone.0000917-Hilf2">[25]</a>–<a href="#pone.0000917-Roy1">[27]</a> likely because of the longevity of the host plant, continuous vertical viral transmission by grafting, and repeated horizontal viral transmission by aphids. In previous studies, characterization of CTV complexes was limited to analysis of genotype-specific PCR markers <a href="#pone.0000917-Hilf2">[25]</a>, <a href="#pone.0000917-Roy1">[27]</a> or to sequencing analysis of selected regions representing a small percentage of the CTV genome <a href="#pone.0000917-Vives1">[8]</a>, <a href="#pone.0000917-Kong1">[26]</a>. In this study, a genome-wide approach was used to characterize multiple genotypes in a CTV complex for the first time. The availability of complete genomic sequences of co-replicating and interacting CTV genotypes should facilitate detailed and sophisticated analyses of genotypic interactions in any given CTV complex in future studies.</p>
<a id="article1.body1.sec3.p4" name="article1.body1.sec3.p4"></a><p>The presence of multiple, co-replicating CTV genotypes was expected to promote recombination. However, the extraordinarily large proportion of recombinant viral molecules as a consequence of promiscuous recombination in the FS2-2 isolate was surprising. We found that 17.6% of cloned and sequenced molecules from the p33 ORF were recombinants derived from the three predominant genotypes in the FS2-2 complex. In contrast, a previous study of another CTV complex, SY568, identified less than 4% of the cloned molecules as being recombinants between two genotypes <a href="#pone.0000917-Vives1">[8]</a>. It is not clear whether the large proportion of recombinants in FS2-2 is a consequence of an elevated recombination rate in the presence of more than two genotypes or a result of recombinants accumulating over time. Likely though, as a result of current horticultural practices, the sweet orange tree from which FS2-2 was derived was planted free of CTV and the CTV population is a result of ingress of the T30, T36 and VT genotypes by aphid transmission, followed by recombination to produce the recombinants found in this study. Subsequent transmission of the FS2-2 complex by aphids or by vegetative propagation to a similar or a new host may perpetuate the same complexity or alter it by generating new and additional recombinant genotypes.</p>
<a id="article1.body1.sec3.p5" name="article1.body1.sec3.p5"></a><p>There was a substantial difference in the proportions of the recombinant sequences obtained upon cloning the 5′ 1 kb (one out of 70 clones) and the p33 ORF (15 out of 84 clones), suggesting that the sequence most proximal to the 5′ end of the CTV genome was more recalcitrant to recombination or that the terminal 1000 bases are subject to more stringent purifying selection, due perhaps to decreased fitness of recombinants, so consequently fewer recombinants are recovered. This difference perhaps can explain the unusual distribution of sequence variability in the CTV genome, with the 5′ half being more variable and the 3′ half being less variable.</p>
<a id="article1.body1.sec3.p6" name="article1.body1.sec3.p6"></a><p>Recombination in RNA viruses has been extensively documented <a href="#pone.0000917-Lai1">[1]</a>, <a href="#pone.0000917-Chare1">[4]</a> as a powerful driving force for generating new sequences which appear as emerging viruses, since recombination can rapidly generate new genotypes by swapping genes or protein domains to reconstitute proteins with novel host-colonizing and pathogenicity traits <a href="#pone.0000917-Gibbs1">[5]</a>, <a href="#pone.0000917-Gibbs2">[6]</a>. The unprecedented scope of recombination between multiple CTV genotypes within a single, persistently infected host as revealed in this study further underscores the importance of recombination in RNA virus evolution, and may explain the extraordinary diversity observed in CTV today. The large number of genetic variants generated through recombination can potentially evolve on their own and become an emerging viral isolate when, and if transmitted to a new host or into a new environment. The influenza virus responsible for the 1918 Spanish flu pandemic was hypothesized to have acquired, through recombination, a portion of the hemagglutinin gene, a key virulence gene, from a swine-lineage influenza <a href="#pone.0000917-Gibbs1">[5]</a>. In this regard, it is interesting to note that all but one of the recombinants in this study maintained the correct open reading frames, and consequently generating a functional recombinant protein. Further studies are needed to determine the viability of these CTV recombinants in isolation from the parental sequences and to evaluate the likelihood that any of the recombinants will emerge as a new CTV strain. CTV shares similarities in its genome organization and gene expression strategies with the largest animal RNA viruses, corornaviruses (which includes the viral agent of SARS), so it is conceivable that similar processes may also operate in global populations of coronaviruses to generate genetic diversity.</p>
</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Materials and Methods"></a><h3>Materials and Methods</h3><a id="article1.body1.sec4.p1" name="article1.body1.sec4.p1"></a><p>(See Supporting Information <a href="#pone.0000917.s001">Text S1</a> for experimental details)</p>

<h4>CTV isolates and genomic sequences</h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p>CTV isolate FS2-2 was collected from a citrus grove in Florida in 2004 and maintained on Madam Vinous sweet orange in an insect-proof greenhouse. Full-length genomic sequences of CTV isolates NUagA, Qaha, SY568, T30, T36, T385, and VT were retrieved from GenBank. In addition, full-length unpublished sequences of CTV isolates T3 (M.E. Hilf, unpublished) and H33 (T.E. Mirkov, personal communication) and a partial sequence (13,585 nt) of the CTV T68-1 isolate (M.E. Hilf, unpublished) were included.</p>


<h4>Amplification of the CTV genome by RT-PCR</h4>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1"></a><p>Full-length genomic equivalents of CTV from each sample were amplified from each sample as four DNA fragments ranging from 4.5 to 5.5 kb by RT-PCR, using four sets of RT-PCR primers (<a href="#pone-0000917-t002">Table 2</a>). Total RNA was extracted from CTV-infected tissue using the Trizol reagent (InVitrogen, Carlsbad, CA). Reverse transcription was carried out using the ImProm II reverse transcriptase (Promega, Madison, WI). CTV genomic fragments were then amplified by 35 cycles of long range PCR using the Stratagene EXL DNA polymerase (Stratagene, La Jolla, CA).</p>


<h4>RT-PCR amplification, cloning, and sequencing of CTV genome fragments</h4>
<a id="article1.body1.sec4.sec3.p1" name="article1.body1.sec4.sec3.p1"></a><p>The 5′ 1 kb fragments of the CTV genome were amplified using the three different 5′ PCR primers and a 3′ conserved universal primer (<a href="#pone-0000917-t002">Table 2</a>). The 1 kb fragments containing the p33 ORF were amplified using a set of universal RT-PCR primers (<a href="#pone-0000917-t002">Table 2</a>). PCR products were purified using the Qiagen MinElute PCR Purification Kit (Qiagen, Valencia, CA) and were employed directly for TA-cloning using a pUC18-based vector containing twin <em>Xcm</em> I restriction sites<sup>25</sup>. Plasmid DNA from randomly selected clones was purified and sequenced in both directions using an ABI 3730XL DNA Analyzer.</p>


<h4>Microarray hybridization and base-calling</h4>
<a id="article1.body1.sec4.sec4.p1" name="article1.body1.sec4.sec4.p1"></a><p>Equimolar amounts of each PCR fragment were pooled, and labeled with biotin-dNTP by terminal deoxynucleotidyl transferase. Hybridization of the labeled target DNA to the microarrays, washing, and subsequent staining in GeneChip Fluidics Station 450 were performed in strict accordance with the instructions provided by Affymetrix. The stained microarray was then scanned at a resolution of 1.563 µm/pixel using a GeneChip Scanner 3000 (Affymetrix, Santa Clara, CA). The final probe intensity data were analyzed with the Affymetrix GeneChip Sequence Analysis Software (GSEQ) to extract sequencing information. Base calls were made using the ABACUS (adaptive background genotype calling scheme) algorithm <a href="#pone.0000917-Cutler1">[31]</a>.</p>


<h4>Contig assembly of sequence fragments generated by resequencing analysis</h4>
<a id="article1.body1.sec4.sec5.p1" name="article1.body1.sec4.sec5.p1"></a><p>Sequence fragments and the associated quality scores generated by GDAS were converted into fasta-format files and used to assemble full and partial CTV genomic contigs using the Phrap program <a href="#pone.0000917-Ewing1">[32]</a> implemented in the CodonCode Aligner (CodonCode, Dedham, MA).</p>


<h4>Sequence analysis</h4>
<a id="article1.body1.sec4.sec6.p1" name="article1.body1.sec4.sec6.p1"></a><p>Sequences of CTV genomes or genomic fragments were aligned using the default parameters of the ClustalX program <a href="#pone.0000917-Thompson1">[47]</a>. Bayesian inference of phylogenetic relationships was carried out using the general time reversal model with gamma-shaped rate variation and a proportion of invariable sites (GTR+I+G) as implemented in MrBayes 3.12 <a href="#pone.0000917-Ronquist1">[33]</a>. Phylogenetic trees were then visualized using the TreeView program <a href="#pone.0000917-Page1">[48]</a>. Recombinant molecules and their cross-over junctions were determined by RDP2, a recombination detection program that deploys 10 published methods to detect recombinant sequences and recombination breakpoints <a href="#pone.0000917-Martin1">[49]</a>.</p>

</div>

<div id="section5" class="section"><a id="s5" name="s5" toc="s5" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0000917"></div><a name="pone.0000917.s001" id="pone.0000917.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000917.s001">Text S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1"></a><p class="preSiDOI">Online Supplemental <a href="#s4">Materials and Methods</a> for Persistent infection and promiscuous recombination of multiple genotypes of an RNA virus within a single host generate extensive diversity</p>
<p class="siDoi">doi:10.1371/journal.pone.0000917.s001</p><a id="article1.body1.sec5.supplementary-material1.caption1.p2" name="article1.body1.sec5.supplementary-material1.caption1.p2"></a><p class="postSiDOI">(0.05 MB PDF)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>We thank T. E. Mirkov for providing unpublished CTV H33 genomic sequence and L. S. Pierson for critical reading of the manuscript.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: DG ZX ZW RB WD. Performed the experiments: ZX ZW RB SG MH. Analyzed the data: DG ZX ZW WD. Contributed reagents/materials/analysis tools: ZX WD SG MH. Wrote the paper: DG ZX ZW.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0000917-Lai1" id="pone.0000917-Lai1"></a>Lai MMC (1992) RNA recombination in animal and plant viruses. Microbiological Reviews  56: 61–79.  <ul class="find" data-citedArticleID="994354"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=RNA+recombination+in+animal+and+plant+viruses.&amp;auth=&amp;atitle=RNA+recombination+in+animal+and+plant+viruses." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=RNA+recombination+in+animal+and+plant+viruses." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22RNA+recombination+in+animal+and+plant+viruses.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0000917-Nelson1" id="pone.0000917-Nelson1"></a>Nelson MI, Holmes EC (2007) The evolution of epidemic influenza. Nature Reviews Genetics  8: 196–205.  <ul class="find" data-citedArticleID="994368" data-doi="10.1038/nrg2053"><li><a href="http://dx.doi.org/10.1038/nrg2053" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+evolution+of+epidemic+influenza." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+evolution+of+epidemic+influenza.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0000917-Holmes1" id="pone.0000917-Holmes1"></a>Holmes EC, Ghedin E, Miller N, Taylor J, Bao YM, et al.  (2005) Whole-genome analysis of human influenza A virus reveals multiple persistent lineages and reassortment among recent H3N2 viruses. Plos Biology  3: 1579–1589.  <ul class="find" data-citedArticleID="994342" data-doi="10.1371/journal.pbio.0030300"><li><a href="http://dx.doi.org/10.1371/journal.pbio.0030300" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Whole-genome+analysis+of+human+influenza+A+virus+reveals+multiple+persistent+lineages+and+reassortment+among+recent+H3N2+viruses." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Whole-genome+analysis+of+human+influenza+A+virus+reveals+multiple+persistent+lineages+and+reassortment+among+recent+H3N2+viruses.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0000917-Chare1" id="pone.0000917-Chare1"></a>Chare ER, Holmes EC (2006) A phylogenetic survey of recombination frequency in plant RNA viruses. Archives of Virology  151: 933–946.  <ul class="find" data-citedArticleID="994320" data-doi="10.1007/s00705-005-0675-x"><li><a href="http://dx.doi.org/10.1007/s00705-005-0675-x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+phylogenetic+survey+of+recombination+frequency+in+plant+RNA+viruses." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+phylogenetic+survey+of+recombination+frequency+in+plant+RNA+viruses.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0000917-Gibbs1" id="pone.0000917-Gibbs1"></a>Gibbs MJ, Armstrong JS, Gibbs AJ (2001) Recombination in the hemagglutinin gene of the 1918 “Spanish flu”. Science  293: 1842–1845.  <ul class="find" data-citedArticleID="994332" data-doi="10.1126/science.1061662"><li><a href="http://dx.doi.org/10.1126/science.1061662" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Recombination+in+the+hemagglutinin+gene+of+the+1918+%26%23x201c%3BSpanish+flu%26%23x201d%3B." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Recombination+in+the+hemagglutinin+gene+of+the+1918+%26%23x201c%3BSpanish+flu%26%23x201d%3B.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0000917-Gibbs2" id="pone.0000917-Gibbs2"></a>Gibbs MJ, Weiller GF (1999) Evidence that a plant virus switched hosts to infect a vertebrate and then recombined with a vertebrate-infecting virus. Proceedings of the National Academy of Sciences of the United States of America  96: 8022–8027.  <ul class="find" data-citedArticleID="994334" data-doi="10.1073/pnas.96.14.8022"><li><a href="http://dx.doi.org/10.1073/pnas.96.14.8022" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evidence+that+a+plant+virus+switched+hosts+to+infect+a+vertebrate+and+then+recombined+with+a+vertebrate-infecting+virus." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evidence+that+a+plant+virus+switched+hosts+to+infect+a+vertebrate+and+then+recombined+with+a+vertebrate-infecting+virus.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0000917-Pernas1" id="pone.0000917-Pernas1"></a>Pernas M, Casado C, Fuentes R, Perez-Elias MJ, Lopez-Galindez C (2006) A dual superinfection and recombination within HIV-1 subtype B 12 years after primoinfection. Jaids-Journal of Acquired Immune Deficiency Syndromes  42: 12–18.  <ul class="find" data-citedArticleID="994372" data-doi="10.1097/01.qai.0000214810.65292.73"><li><a href="http://dx.doi.org/10.1097/01.qai.0000214810.65292.73" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+dual+superinfection+and+recombination+within+HIV-1+subtype+B+12+years+after+primoinfection." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+dual+superinfection+and+recombination+within+HIV-1+subtype+B+12+years+after+primoinfection.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0000917-Vives1" id="pone.0000917-Vives1"></a>Vives MC, Rubio L, Sambade A, Mirkov TE, Moreno P, et al.  (2005) Evidence of multiple recombination events between two RNA sequence variants within a Citrus tristeza virus isolate. Virology  331: 232–237.  <ul class="find" data-citedArticleID="994396" data-doi="10.1016/j.virol.2004.10.037"><li><a href="http://dx.doi.org/10.1016/j.virol.2004.10.037" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evidence+of+multiple+recombination+events+between+two+RNA+sequence+variants+within+a+Citrus+tristeza+virus+isolate." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evidence+of+multiple+recombination+events+between+two+RNA+sequence+variants+within+a+Citrus+tristeza+virus+isolate.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0000917-AlbiachMarti1" id="pone.0000917-AlbiachMarti1"></a>Albiach-Marti MR, Mawassi M, Gowda S, Satyanarayana T, Hilf ME, et al.  (2000) Sequences of Citrus tristeza virus separated in time and space are essentially identical. Journal of Virology  74: 6856–6865.  <ul class="find" data-citedArticleID="994316" data-doi="10.1128/jvi.74.15.6856-6865.2000"><li><a href="http://dx.doi.org/10.1128/jvi.74.15.6856-6865.2000" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Sequences+of+Citrus+tristeza+virus+separated+in+time+and+space+are+essentially+identical." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Sequences+of+Citrus+tristeza+virus+separated+in+time+and+space+are+essentially+identical.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0000917-Karasev1" id="pone.0000917-Karasev1"></a>Karasev AV, Boyko VP, Gowda S, Nikolaeva OV, Hilf ME, et al.  (1995) Complete sequence of the Citrus tristeza virus RNA genome. Virology  208: 511–520.  <ul class="find" data-citedArticleID="994344" data-doi="10.1006/viro.1995.1182"><li><a href="http://dx.doi.org/10.1006/viro.1995.1182" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Complete+sequence+of+the+Citrus+tristeza+virus+RNA+genome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Complete+sequence+of+the+Citrus+tristeza+virus+RNA+genome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0000917-Mawassi1" id="pone.0000917-Mawassi1"></a>Mawassi M, Mietkiewska E, Gofman R, Yang G, BarJoseph M (1996) Unusual sequence relationships between two isolates of citrus tristeza virus. Journal of General Virology  77: 2359–2364.  <ul class="find" data-citedArticleID="994366" data-doi="10.1099/0022-1317-77-9-2359"><li><a href="http://dx.doi.org/10.1099/0022-1317-77-9-2359" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Unusual+sequence+relationships+between+two+isolates+of+citrus+tristeza+virus." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Unusual+sequence+relationships+between+two+isolates+of+citrus+tristeza+virus.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0000917-Yang1" id="pone.0000917-Yang1"></a>Yang ZN, Mathews DM, Dodds JA, Mirkov TE (1999) Molecular characterization of an isolate of citrus tristeza virus that causes severe symptoms in sweet orange. Virus Genes  19: 131–142.  <ul class="find" data-citedArticleID="994406" data-doi="10.1023/a:1008127224147"><li><a href="http://dx.doi.org/10.1023/a:1008127224147" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Molecular+characterization+of+an+isolate+of+citrus+tristeza+virus+that+causes+severe+symptoms+in+sweet+orange." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Molecular+characterization+of+an+isolate+of+citrus+tristeza+virus+that+causes+severe+symptoms+in+sweet+orange.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0000917-BarJoseph1" id="pone.0000917-BarJoseph1"></a>Bar-Joseph M, Marcus R, Lee RF (1989) The continuous challenge of Citrus tristeza virus control. Annual Review of Phytopathology  27: 291–316.  <ul class="find" data-citedArticleID="994318" data-doi="10.1146/annurev.phyto.27.1.291"><li><a href="http://dx.doi.org/10.1146/annurev.phyto.27.1.291" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+continuous+challenge+of+Citrus+tristeza+virus+control." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+continuous+challenge+of+Citrus+tristeza+virus+control.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0000917-RochaPena1" id="pone.0000917-RochaPena1"></a>Rocha-Pena MA, Lee RF, Lastra R, Niblett CL, Ochoacorona FM, et al.  (1995) Citrus tristeza virus and its aphid vector Toxoptera citricida - threats to citrus production in the Caribbean and Central and North America. Plant Disease  79: 437–445.  <ul class="find" data-citedArticleID="994374" data-doi="10.1094/pd-79-0437"><li><a href="http://dx.doi.org/10.1094/pd-79-0437" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Citrus+tristeza+virus+and+its+aphid+vector+Toxoptera+citricida+-+threats+to+citrus+production+in+the+Caribbean+and+Central+and+North+America." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Citrus+tristeza+virus+and+its+aphid+vector+Toxoptera+citricida+-+threats+to+citrus+production+in+the+Caribbean+and+Central+and+North+America.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0000917-Satyanarayana1" id="pone.0000917-Satyanarayana1"></a>Satyanarayana T, Gowda S, Ayllon MA, Dawson WO (2004) Closterovirus bipolar virion: Evidence for initiation of assembly by minor coat protein and its restriction to the genomic RNA 5′ region. Proceedings of the National Academy of Sciences of the United States of America  101: 799–804.  <ul class="find" data-citedArticleID="994384" data-doi="10.1073/pnas.0307747100"><li><a href="http://dx.doi.org/10.1073/pnas.0307747100" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Closterovirus+bipolar+virion%3A+Evidence+for+initiation+of+assembly+by+minor+coat+protein+and+its+restriction+to+the+genomic+RNA+5%26%23x2032%3B+region." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Closterovirus+bipolar+virion%3A+Evidence+for+initiation+of+assembly+by+minor+coat+protein+and+its+restriction+to+the+genomic+RNA+5%26%23x2032%3B+region.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0000917-Vives2" id="pone.0000917-Vives2"></a>Vives MC, Rubio L, Lopez C, Navas-Castillo J, Albiach-Marti MR, et al.  (1999) The complete genome sequence of the major component of a mild citrus tristeza virus isolate. Journal of General Virology  80: 811–816.  <ul class="find" data-citedArticleID="994398"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=The+complete+genome+sequence+of+the+major+component+of+a+mild+citrus+tristeza+virus+isolate.&amp;auth=&amp;atitle=The+complete+genome+sequence+of+the+major+component+of+a+mild+citrus+tristeza+virus+isolate." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+complete+genome+sequence+of+the+major+component+of+a+mild+citrus+tristeza+virus+isolate." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+complete+genome+sequence+of+the+major+component+of+a+mild+citrus+tristeza+virus+isolate.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0000917-RuizRuiz1" id="pone.0000917-RuizRuiz1"></a>Ruiz-Ruiz S, Moreno P, Guerri J, Ambros S (2006) The complete nucleotide sequence of a severe stem pitting isolate of Citrus tristeza virus from Spain: comparison with isolates from different origins. Archives of Virology  151: 387–398.  <ul class="find" data-citedArticleID="994382" data-doi="10.1007/s00705-005-0618-6"><li><a href="http://dx.doi.org/10.1007/s00705-005-0618-6" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+complete+nucleotide+sequence+of+a+severe+stem+pitting+isolate+of+Citrus+tristeza+virus+from+Spain%3A+comparison+with+isolates+from+different+origins." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+complete+nucleotide+sequence+of+a+severe+stem+pitting+isolate+of+Citrus+tristeza+virus+from+Spain%3A+comparison+with+isolates+from+different+origins.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0000917-Yang2" id="pone.0000917-Yang2"></a>Yang GA, Che XB, Gofman R, Ben-Shalom Y, Piestun D, et al.  (1999) D-RNA molecules associated with subisolates of the VT strain of citrus tristeza virus which induce different seedling-yellows reactions. Virus Genes  19: 5–13.  <ul class="find" data-citedArticleID="994408"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=D-RNA+molecules+associated+with+subisolates+of+the+VT+strain+of+citrus+tristeza+virus+which+induce+different+seedling-yellows+reactions.&amp;auth=&amp;atitle=D-RNA+molecules+associated+with+subisolates+of+the+VT+strain+of+citrus+tristeza+virus+which+induce+different+seedling-yellows+reactions." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=D-RNA+molecules+associated+with+subisolates+of+the+VT+strain+of+citrus+tristeza+virus+which+induce+different+seedling-yellows+reactions." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22D-RNA+molecules+associated+with+subisolates+of+the+VT+strain+of+citrus+tristeza+virus+which+induce+different+seedling-yellows+reactions.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0000917-Satyanarayana2" id="pone.0000917-Satyanarayana2"></a>Satyanarayana T, Gowda S, Boyko VP, Albiach-Marti MR, Mawassi M, et al.  (1999) An engineered closterovirus RNA replicon and analysis of heterologous terminal sequences for replication. Proceedings of the National Academy of Sciences of the United States of America  96: 7433–7438.  <ul class="find" data-citedArticleID="994386" data-doi="10.1073/pnas.96.13.7433"><li><a href="http://dx.doi.org/10.1073/pnas.96.13.7433" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=An+engineered+closterovirus+RNA+replicon+and+analysis+of+heterologous+terminal+sequences+for+replication." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22An+engineered+closterovirus+RNA+replicon+and+analysis+of+heterologous+terminal+sequences+for+replication.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0000917-Lu1" id="pone.0000917-Lu1"></a>Lu R, Folimonov A, Shintaku M, Li WX, Falk BW, et al.  (2004) Three distinct suppressors of RNA silencing encoded by a 20-kb viral RNA genome. Proceedings of the National Academy of Sciences of the United States of America  101: 15742–15747.  <ul class="find" data-citedArticleID="994362" data-doi="10.1073/pnas.0404940101"><li><a href="http://dx.doi.org/10.1073/pnas.0404940101" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Three+distinct+suppressors+of+RNA+silencing+encoded+by+a+20-kb+viral+RNA+genome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Three+distinct+suppressors+of+RNA+silencing+encoded+by+a+20-kb+viral+RNA+genome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0000917-Satyanarayana3" id="pone.0000917-Satyanarayana3"></a>Satyanarayana T, Gowda S, Ayllon MA, Albiach-Marti MR, Rabindran S, et al.  (2002) The p23 protein of citrus tristeza virus controls asymmetrical RNA accumulation. Journal of Virology  76: 473–483.  <ul class="find" data-citedArticleID="994388" data-doi="10.1128/jvi.76.2.473-483.2002"><li><a href="http://dx.doi.org/10.1128/jvi.76.2.473-483.2002" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+p23+protein+of+citrus+tristeza+virus+controls+asymmetrical+RNA+accumulation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+p23+protein+of+citrus+tristeza+virus+controls+asymmetrical+RNA+accumulation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0000917-Gowda1" id="pone.0000917-Gowda1"></a>Gowda S, Satyanarayana T, Davis CL, Navas-Castillo J, Albiach-Marti MR, et al.  (2000) The p20 gene product of Citrus tristeza virus accumulates in the amorphous inclusion bodies. Virology  274: 246–254.  <ul class="find" data-citedArticleID="994336" data-doi="10.1006/viro.2000.0413"><li><a href="http://dx.doi.org/10.1006/viro.2000.0413" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+p20+gene+product+of+Citrus+tristeza+virus+accumulates+in+the+amorphous+inclusion+bodies." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+p20+gene+product+of+Citrus+tristeza+virus+accumulates+in+the+amorphous+inclusion+bodies.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0000917-Hilf1" id="pone.0000917-Hilf1"></a>Hilf ME, Karasev AV, Pappu HR, Gumpf DJ, Niblett CL, et al.  (1995) Characterization of Citrus tristeza virus subgenomic RNAs in infected tissue. Virology  208: 576–582.  <ul class="find" data-citedArticleID="994338" data-doi="10.1006/viro.1995.1188"><li><a href="http://dx.doi.org/10.1006/viro.1995.1188" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Characterization+of+Citrus+tristeza+virus+subgenomic+RNAs+in+infected+tissue." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Characterization+of+Citrus+tristeza+virus+subgenomic+RNAs+in+infected+tissue.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0000917-Karasev2" id="pone.0000917-Karasev2"></a>Karasev AV, Hilf ME, Garnsey SM, Dawson WO (1997) Transcriptional strategy of closteroviruses: Mapping the 5′ termini of the citrus tristeza virus subgenomic RNAs. Journal of Virology  71: 6233–6236.  <ul class="find" data-citedArticleID="994346"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Transcriptional+strategy+of+closteroviruses%3A+Mapping+the+5%26%23x2032%3B+termini+of+the+citrus+tristeza+virus+subgenomic+RNAs.&amp;auth=&amp;atitle=Transcriptional+strategy+of+closteroviruses%3A+Mapping+the+5%26%23x2032%3B+termini+of+the+citrus+tristeza+virus+subgenomic+RNAs." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Transcriptional+strategy+of+closteroviruses%3A+Mapping+the+5%26%23x2032%3B+termini+of+the+citrus+tristeza+virus+subgenomic+RNAs." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Transcriptional+strategy+of+closteroviruses%3A+Mapping+the+5%26%23x2032%3B+termini+of+the+citrus+tristeza+virus+subgenomic+RNAs.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0000917-Hilf2" id="pone.0000917-Hilf2"></a>Hilf ME, Mavrodieva VA, Garnsey SM (2005) Genetic marker analysis of a global collection of isolates of Citrus tristeza virus: Characterization and distribution of CTV genotypes and association with symptoms. Phytopathology  95: 909–917.  <ul class="find" data-citedArticleID="994340" data-doi="10.1094/phyto-95-0909"><li><a href="http://dx.doi.org/10.1094/phyto-95-0909" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genetic+marker+analysis+of+a+global+collection+of+isolates+of+Citrus+tristeza+virus%3A+Characterization+and+distribution+of+CTV+genotypes+and+association+with+symptoms." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genetic+marker+analysis+of+a+global+collection+of+isolates+of+Citrus+tristeza+virus%3A+Characterization+and+distribution+of+CTV+genotypes+and+association+with+symptoms.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0000917-Kong1" id="pone.0000917-Kong1"></a>Kong P, Rubio L, Polek M, Falk BW (2000) Population structure and genetic diversity within California Citrus tristeza virus (CTV) isolates. Virus Genes  21: 139–145.  <ul class="find" data-citedArticleID="994352" data-doi="10.1023/a:1008198311398"><li><a href="http://dx.doi.org/10.1023/a:1008198311398" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Population+structure+and+genetic+diversity+within+California+Citrus+tristeza+virus+%28CTV%29+isolates." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Population+structure+and+genetic+diversity+within+California+Citrus+tristeza+virus+%28CTV%29+isolates.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0000917-Roy1" id="pone.0000917-Roy1"></a>Roy A, Brlansky RH (2004) Genotype classification and molecular evidence for the presence of mixed infections in Indian Citrus tristeza virus isolates. Archives of Virology  149: 1911–1929.  <ul class="find" data-citedArticleID="994378" data-doi="10.1007/s00705-004-0355-2"><li><a href="http://dx.doi.org/10.1007/s00705-004-0355-2" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genotype+classification+and+molecular+evidence+for+the+presence+of+mixed+infections+in+Indian+Citrus+tristeza+virus+isolates." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genotype+classification+and+molecular+evidence+for+the+presence+of+mixed+infections+in+Indian+Citrus+tristeza+virus+isolates.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0000917-Xiong1" id="pone.0000917-Xiong1"></a>Xiong Z, Barthelson R, Weng Z, Galbraith DW (2006) Designing and testing of a Citrus tristeza virus resequencing microarray. Proceedings of the International Organization of Citrus Virologists  16: 11–22.  <ul class="find" data-citedArticleID="994404"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Designing+and+testing+of+a+Citrus+tristeza+virus+resequencing+microarray.&amp;auth=&amp;atitle=Designing+and+testing+of+a+Citrus+tristeza+virus+resequencing+microarray." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Designing+and+testing+of+a+Citrus+tristeza+virus+resequencing+microarray." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Designing+and+testing+of+a+Citrus+tristeza+virus+resequencing+microarray.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0000917-Wong1" id="pone.0000917-Wong1"></a>Wong CW, Albert TJ, Vega VB, Norton JE, Cutler DJ, et al.  (2004) Tracking the evolution of the SARS coronavirus using high-throughput, high-density resequencing arrays. Genome Research  14: 398–405.  <ul class="find" data-citedArticleID="994402" data-doi="10.1101/gr.2141004"><li><a href="http://dx.doi.org/10.1101/gr.2141004" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Tracking+the+evolution+of+the+SARS+coronavirus+using+high-throughput%2C+high-density+resequencing+arrays." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Tracking+the+evolution+of+the+SARS+coronavirus+using+high-throughput%2C+high-density+resequencing+arrays.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0000917-Zwick1" id="pone.0000917-Zwick1"></a>Zwick ME, McAfee F, Cutler DJ, Read TD, Ravel J, et al.  (2005) Microarray-based resequencing of multiple Bacillus anthracis isolates. Genome Biology  6: R10–2005.  <ul class="find" data-citedArticleID="994412"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Microarray-based+resequencing+of+multiple+Bacillus+anthracis+isolates.&amp;auth=&amp;atitle=Microarray-based+resequencing+of+multiple+Bacillus+anthracis+isolates." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Microarray-based+resequencing+of+multiple+Bacillus+anthracis+isolates." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Microarray-based+resequencing+of+multiple+Bacillus+anthracis+isolates.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0000917-Cutler1" id="pone.0000917-Cutler1"></a>Cutler DJ, Zwick ME, Carrasquillo MM, Yohn CT, Tobin KP, et al.  (2001) High-throughput variation detection and genotyping using microarrays. Genome Research  11: 1913–1925.  <ul class="find" data-citedArticleID="994326"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=High-throughput+variation+detection+and+genotyping+using+microarrays.&amp;auth=&amp;atitle=High-throughput+variation+detection+and+genotyping+using+microarrays." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=High-throughput+variation+detection+and+genotyping+using+microarrays." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22High-throughput+variation+detection+and+genotyping+using+microarrays.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0000917-Ewing1" id="pone.0000917-Ewing1"></a>Ewing B, Hillier L, Wendl MC, Green P (1998) Base-calling of automated sequencer traces using phred. I. Accuracy assessment. Genome Research  8: 175–185.  <ul class="find" data-citedArticleID="994330" data-doi="10.1101/gr.8.3.175"><li><a href="http://dx.doi.org/10.1101/gr.8.3.175" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Base-calling+of+automated+sequencer+traces+using+phred.+I.+Accuracy+assessment." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Base-calling+of+automated+sequencer+traces+using+phred.+I.+Accuracy+assessment.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0000917-Ronquist1" id="pone.0000917-Ronquist1"></a>Ronquist F, Huelsenbeck JP (2003) MrBayes 3: Bayesian phylogenetic inference under mixed models. Bioinformatics  19: 1572–1574.  <ul class="find" data-citedArticleID="994376" data-doi="10.1093/bioinformatics/btg180"><li><a href="http://dx.doi.org/10.1093/bioinformatics/btg180" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=MrBayes+3%3A+Bayesian+phylogenetic+inference+under+mixed+models." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22MrBayes+3%3A+Bayesian+phylogenetic+inference+under+mixed+models.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0000917-Rubio1" id="pone.0000917-Rubio1"></a>Rubio L, Ayllon MA, Kong P, Fernandez A, Polek M, et al.  (2001) Genetic variation of Citrus tristeza virus isolates from California and Spain: Evidence for mixed infections and recombination. Journal of Virology  75: 8054–8062.  <ul class="find" data-citedArticleID="994380" data-doi="10.1128/jvi.75.17.8054-8062.2001"><li><a href="http://dx.doi.org/10.1128/jvi.75.17.8054-8062.2001" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genetic+variation+of+Citrus+tristeza+virus+isolates+from+California+and+Spain%3A+Evidence+for+mixed+infections+and+recombination." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genetic+variation+of+Citrus+tristeza+virus+isolates+from+California+and+Spain%3A+Evidence+for+mixed+infections+and+recombination.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0000917-Lopez1" id="pone.0000917-Lopez1"></a>Lopez C, Ayllon MA, Navas-Castillo J, Guerri J, Moreno P, et al.  (1998) Molecular variability of the 5′- and 3′-terminal regions of citrus tristeza virus RNA. Phytopathology  88: 685–691.  <ul class="find" data-citedArticleID="994360" data-doi="10.1094/phyto.1998.88.7.685"><li><a href="http://dx.doi.org/10.1094/phyto.1998.88.7.685" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Molecular+variability+of+the+5%26%23x2032%3B-+and+3%26%23x2032%3B-terminal+regions+of+citrus+tristeza+virus+RNA." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Molecular+variability+of+the+5%26%23x2032%3B-+and+3%26%23x2032%3B-terminal+regions+of+citrus+tristeza+virus+RNA.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">36.
              </span><a name="pone.0000917-Karasev3" id="pone.0000917-Karasev3"></a>Karasev AV (2000) Genetic diversity and evolution of closteroviruses. Annual Review of Phytopathology  38: 293–324.  <ul class="find" data-citedArticleID="994348"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Genetic+diversity+and+evolution+of+closteroviruses.&amp;auth=&amp;atitle=Genetic+diversity+and+evolution+of+closteroviruses." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genetic+diversity+and+evolution+of+closteroviruses." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genetic+diversity+and+evolution+of+closteroviruses.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">37.
              </span><a name="pone.0000917-Dzianott1" id="pone.0000917-Dzianott1"></a>Dzianott A, Rauffer-Bruyere N, Bujarski JJ (2001) Studies on functional interaction between brome mosaic virus replicase proteins during RNA recombination, using combined mutants in vivo and in vitro. Virology  289: 137–149.  <ul class="find" data-citedArticleID="994328" data-doi="10.1006/viro.2001.1118"><li><a href="http://dx.doi.org/10.1006/viro.2001.1118" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Studies+on+functional+interaction+between+brome+mosaic+virus+replicase+proteins+during+RNA+recombination%2C+using+combined+mutants+in+vivo+and+in+vitro." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Studies+on+functional+interaction+between+brome+mosaic+virus+replicase+proteins+during+RNA+recombination%2C+using+combined+mutants+in+vivo+and+in+vitro.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">38.
              </span><a name="pone.0000917-Cheng1" id="pone.0000917-Cheng1"></a>Cheng CP, Panavas T, Luo GX, Nagy PD (2005) Heterologous RNA replication enhancer stimulates in vitro RNA synthesis and template-switching by the carmovirus, but not by the tombusvirus, RNA-dependent RNA polymerase: Implication for modular evolution of RNA viruses. Virology  341: 107–121.  <ul class="find" data-citedArticleID="994322" data-doi="10.1016/j.virol.2005.06.042"><li><a href="http://dx.doi.org/10.1016/j.virol.2005.06.042" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Heterologous+RNA+replication+enhancer+stimulates+in+vitro+RNA+synthesis+and+template-switching+by+the+carmovirus%2C+but+not+by+the+tombusvirus%2C+RNA-dependent+RNA+polymerase%3A+Implication+for+modular+evolution+of+RNA+viruses." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Heterologous+RNA+replication+enhancer+stimulates+in+vitro+RNA+synthesis+and+template-switching+by+the+carmovirus%2C+but+not+by+the+tombusvirus%2C+RNA-dependent+RNA+polymerase%3A+Implication+for+modular+evolution+of+RNA+viruses.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">39.
              </span><a name="pone.0000917-Cheng2" id="pone.0000917-Cheng2"></a>Cheng CP, Nagy PD (2003) Mechanism of RNA recombination in carmo- and tombusviruses: Evidence for template switching by the RNA-Dependent RNA polymerase in vitro. Journal of Virology  77: 12033–12047.  <ul class="find" data-citedArticleID="994324" data-doi="10.1128/jvi.77.22.12033-12047.2003"><li><a href="http://dx.doi.org/10.1128/jvi.77.22.12033-12047.2003" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mechanism+of+RNA+recombination+in+carmo-+and+tombusviruses%3A+Evidence+for+template+switching+by+the+RNA-Dependent+RNA+polymerase+in+vitro." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mechanism+of+RNA+recombination+in+carmo-+and+tombusviruses%3A+Evidence+for+template+switching+by+the+RNA-Dependent+RNA+polymerase+in+vitro.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">40.
              </span><a name="pone.0000917-Shapka1" id="pone.0000917-Shapka1"></a>Shapka N, Nagy PD (2004) The AU-rich RNA recombination hot spot sequence of Brome mosaic virus is functional in tombusviruses: Implications for the mechanism of RNA recombination. Journal of Virology  78: 2288–2300.  <ul class="find" data-citedArticleID="994390" data-doi="10.1128/jvi.78.5.2288-2300.2004"><li><a href="http://dx.doi.org/10.1128/jvi.78.5.2288-2300.2004" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+AU-rich+RNA+recombination+hot+spot+sequence+of+Brome+mosaic+virus+is+functional+in+tombusviruses%3A+Implications+for+the+mechanism+of+RNA+recombination." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+AU-rich+RNA+recombination+hot+spot+sequence+of+Brome+mosaic+virus+is+functional+in+tombusviruses%3A+Implications+for+the+mechanism+of+RNA+recombination.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">41.
              </span><a name="pone.0000917-Kim1" id="pone.0000917-Kim1"></a>Kim MJ, Kao C (2001) Factors regulating template switch in vitro by viral RNA-dependent RNA polymerases: Implications for RNA-RNA recombination. Proceedings of the National Academy of Sciences of the United States of America  98: 4972–4977.  <ul class="find" data-citedArticleID="994350" data-doi="10.1073/pnas.081077198"><li><a href="http://dx.doi.org/10.1073/pnas.081077198" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Factors+regulating+template+switch+in+vitro+by+viral+RNA-dependent+RNA+polymerases%3A+Implications+for+RNA-RNA+recombination." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Factors+regulating+template+switch+in+vitro+by+viral+RNA-dependent+RNA+polymerases%3A+Implications+for+RNA-RNA+recombination.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">42.
              </span><a name="pone.0000917-Yu1" id="pone.0000917-Yu1"></a>Yu W, Rusterholtz KJ, Krummel AT, Lehman N (2006) Detection of high levels of recombination generated during PCR amplification of RNA templates. Biotechniques  40: 499–507.  <ul class="find" data-citedArticleID="994410" data-doi="10.2144/000112124"><li><a href="http://dx.doi.org/10.2144/000112124" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Detection+of+high+levels+of+recombination+generated+during+PCR+amplification+of+RNA+templates." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Detection+of+high+levels+of+recombination+generated+during+PCR+amplification+of+RNA+templates.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">43.
              </span><a name="pone.0000917-Wang1" id="pone.0000917-Wang1"></a>Wang Z, Daum LT, Vora GJ, Metzgar D, Walter EA, et al.  (2006) Identifying influenza viruses with resequencing microarrays. Emerging Infectious Diseases  12: 638–646.  <ul class="find" data-citedArticleID="994400" data-doi="10.3201/eid1204.051441"><li><a href="http://dx.doi.org/10.3201/eid1204.051441" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Identifying+influenza+viruses+with+resequencing+microarrays." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Identifying+influenza+viruses+with+resequencing+microarrays.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">44.
              </span><a name="pone.0000917-Lin1" id="pone.0000917-Lin1"></a>Lin BC, Wang Z, Vora GJ, Thornton JA, Schnur JM, et al.  (2006) Broad-spectrum respiratory tract pathogen identification using resequencing DNA microarrays. Genome Research  16: 527–535.  <ul class="find" data-citedArticleID="994356" data-doi="10.1101/gr.4337206"><li><a href="http://dx.doi.org/10.1101/gr.4337206" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Broad-spectrum+respiratory+tract+pathogen+identification+using+resequencing+DNA+microarrays." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Broad-spectrum+respiratory+tract+pathogen+identification+using+resequencing+DNA+microarrays.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">45.
              </span><a name="pone.0000917-Lin2" id="pone.0000917-Lin2"></a>Lin BC, Blaney KM, Malanoski AP, Ligler AG, Schnur JM, et al.  (2007) Using a resequencing microarray as a multiple respiratory pathogen detection assay. Journal of Clinical Microbiology  45: 443–452.  <ul class="find" data-citedArticleID="994358" data-doi="10.1128/jcm.01870-06"><li><a href="http://dx.doi.org/10.1128/jcm.01870-06" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Using+a+resequencing+microarray+as+a+multiple+respiratory+pathogen+detection+assay." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Using+a+resequencing+microarray+as+a+multiple+respiratory+pathogen+detection+assay.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">46.
              </span><a name="pone.0000917-Sulaiman1" id="pone.0000917-Sulaiman1"></a>Sulaiman IM, Tang K, Osborne J, Sammons S, Wohlheuter RA (2007) GeneChip resequencing of the smallpox virus genome can identify novel strains: a biodefense application. Journal of Clinical Microbiology  45: 358–363.  <ul class="find" data-citedArticleID="994392" data-doi="10.1128/jcm.01848-06"><li><a href="http://dx.doi.org/10.1128/jcm.01848-06" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=GeneChip+resequencing+of+the+smallpox+virus+genome+can+identify+novel+strains%3A+a+biodefense+application." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22GeneChip+resequencing+of+the+smallpox+virus+genome+can+identify+novel+strains%3A+a+biodefense+application.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">47.
              </span><a name="pone.0000917-Thompson1" id="pone.0000917-Thompson1"></a>Thompson JD, Gibson TJ, Plewniak F, Jeanmougin F, Higgins DG (1997) The CLUSTAL_X windows interface: flexible strategies for multiple sequence alignment aided by quality analysis tools. Nucleic Acids Research  25: 4876–4882.  <ul class="find" data-citedArticleID="994394" data-doi="10.1093/nar/25.24.4876"><li><a href="http://dx.doi.org/10.1093/nar/25.24.4876" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+CLUSTAL_X+windows+interface%3A+flexible+strategies+for+multiple+sequence+alignment+aided+by+quality+analysis+tools." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+CLUSTAL_X+windows+interface%3A+flexible+strategies+for+multiple+sequence+alignment+aided+by+quality+analysis+tools.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">48.
              </span><a name="pone.0000917-Page1" id="pone.0000917-Page1"></a>Page RDM (1996) TreeView: An application to display phylogenetic trees on personal computers. Computer Applications in the Biosciences  12: 357–358.  <ul class="find" data-citedArticleID="994370" data-doi="10.1093/bioinformatics/12.4.357"><li><a href="http://dx.doi.org/10.1093/bioinformatics/12.4.357" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=TreeView%3A+An+application+to+display+phylogenetic+trees+on+personal+computers." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22TreeView%3A+An+application+to+display+phylogenetic+trees+on+personal+computers.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">49.
              </span><a name="pone.0000917-Martin1" id="pone.0000917-Martin1"></a>Martin DP, Williamson C, Posada D (2005) RDP2: recombination detection and analysis from sequence alignments. Bioinformatics  21: 260–262.  <ul class="find" data-citedArticleID="994364" data-doi="10.1093/bioinformatics/bth490"><li><a href="http://dx.doi.org/10.1093/bioinformatics/bth490" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=RDP2%3A+recombination+detection+and+analysis+from+sequence+alignments." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22RDP2%3A+recombination+detection+and+analysis+from+sequence+alignments.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000917.XML" value="100456"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000917.PDF" value="378746"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000917.t001.PNG_L" value="114665"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000917.t001.PNG_M" value="150090"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000917.t001.PNG_S" value="18093"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000917.t001.TIF" value="530190"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000917.t001.PNG_I" value="52356"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000917.g001.PNG_L" value="744552"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000917.g001.PNG_M" value="61784"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000917.g001.PNG_S" value="2900"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000917.g001.TIF" value="628074"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000917.g001.PNG_I" value="21323"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000917.t002.PNG_L" value="161609"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000917.t002.PNG_M" value="195371"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000917.t002.PNG_S" value="22934"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000917.t002.TIF" value="689824"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000917.t002.PNG_I" value="88422"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000917.g002.PNG_L" value="5889346"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000917.g002.PNG_M" value="260914"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000917.g002.PNG_S" value="10492"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000917.g002.TIF" value="7593692"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000917.g002.PNG_I" value="78737"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000917.g003.PNG_L" value="1070999"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000917.g003.PNG_M" value="124360"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000917.g003.PNG_S" value="8795"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000917.g003.TIF" value="1963730"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000917.g003.PNG_I" value="42447"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000917.g004.PNG_L" value="219704"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000917.g004.PNG_M" value="39886"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000917.g004.PNG_S" value="6336"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000917.g004.TIF" value="562386"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000917.g004.PNG_I" value="14881"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000917.s001.PDF" value="49833"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000917&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000917" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000917&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0000917&volume=&issue=&title=Persistent Infection and Promiscuous Recombination of Multiple Genotypes of an RNA Virus within a Single Host Generate Extensive Diversity&author_name=Ziming%20Weng%2C%20Roger%20Barthelson%2C%20Siddarame%20Gowda%2C%20Mark%20E.%20Hilf%2C%20William%20O.%20Dawson%2C%20David%20W.%20Galbraith%2C%20Zhongguo%20Xiong&start_page=1&end_page=9" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000917" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000917" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0000917" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000917&amp;t=Persistent%20Infection%20and%20Promiscuous%20Recombination%20of%20Multiple%20Genotypes%20of%20an%20RNA%20Virus%20within%20a%20Single%20Host%20Generate%20Extensive%20Diversity" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000917&title=Persistent%20Infection%20and%20Promiscuous%20Recombination%20of%20Multiple%20Genotypes%20of%20an%20RNA%20Virus%20within%20a%20Single%20Host%20Generate%20Extensive%20Diversity&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000917&amp;title=Persistent%20Infection%20and%20Promiscuous%20Recombination%20of%20Multiple%20Genotypes%20of%20an%20RNA%20Virus%20within%20a%20Single%20Host%20Generate%20Extensive%20Diversity" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000917" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0000917&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Persistent Infection and Promiscuous Recombination of Multiple Genotypes of an RNA Virus within a Single Host Generate Extensive Diversity';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0000917';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Persistent%20Infection%20and%20Promiscuous%20Recombination%20of%20Multiple%20Genotypes%20of%20an%20RNA%20Virus%20within%20a%20Single%20Host%20Generate%20Extensive%20Diversity http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000917" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0000917" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">



















          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22DNA+sequence+analysis%22" title="Search for articles in the subject area:'DNA sequence analysis'"><div class="flagText">DNA sequence analysis</div></a>
              <div data-categoryid="47935" data-articleid="24672"
                   data-categoryname="DNA sequence analysis"
                   class="flagImage" title="Flag 'DNA sequence analysis' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22DNA+sequences%22" title="Search for articles in the subject area:'DNA sequences'"><div class="flagText">DNA sequences</div></a>
              <div data-categoryid="32003" data-articleid="24672"
                   data-categoryname="DNA sequences"
                   class="flagImage" title="Flag 'DNA sequences' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Genome+complexity%22" title="Search for articles in the subject area:'Genome complexity'"><div class="flagText">Genome complexity</div></a>
              <div data-categoryid="18711" data-articleid="24672"
                   data-categoryname="Genome complexity"
                   class="flagImage" title="Flag 'Genome complexity' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Genome+evolution%22" title="Search for articles in the subject area:'Genome evolution'"><div class="flagText">Genome evolution</div></a>
              <div data-categoryid="17207" data-articleid="24672"
                   data-categoryname="Genome evolution"
                   class="flagImage" title="Flag 'Genome evolution' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Microarrays%22" title="Search for articles in the subject area:'Microarrays'"><div class="flagText">Microarrays</div></a>
              <div data-categoryid="17255" data-articleid="24672"
                   data-categoryname="Microarrays"
                   class="flagImage" title="Flag 'Microarrays' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Plant+genomics%22" title="Search for articles in the subject area:'Plant genomics'"><div class="flagText">Plant genomics</div></a>
              <div data-categoryid="33365" data-articleid="24672"
                   data-categoryname="Plant genomics"
                   class="flagImage" title="Flag 'Plant genomics' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Sequence+analysis%22" title="Search for articles in the subject area:'Sequence analysis'"><div class="flagText">Sequence analysis</div></a>
              <div data-categoryid="47475" data-articleid="24672"
                   data-categoryname="Sequence analysis"
                   class="flagImage" title="Flag 'Sequence analysis' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Sequence+motif+analysis%22" title="Search for articles in the subject area:'Sequence motif analysis'"><div class="flagText">Sequence motif analysis</div></a>
              <div data-categoryid="19657" data-articleid="24672"
                   data-categoryname="Sequence motif analysis"
                   class="flagImage" title="Flag 'Sequence motif analysis' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=9377'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=7837'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=5141&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>

<div class="block sidebar-comments">
    <div class="header">
        <h3>Comments</h3>
    </div>
      <p><a href="/annotation/listThread.action?root=18913">Pool of RNA</a><br>Posted by AntonioMas</p>
</div>

</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
