

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: A Novel Campylobacter jejuni Sequence Type from a Culture-Negative Patient in The Gambia</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0001773"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0001773" />

    <meta name="citation_title" content="A Novel Campylobacter jejuni Sequence Type from a Culture-Negative Patient in The Gambia"/>
    <meta itemprop="name" content="A Novel Campylobacter jejuni Sequence Type from a Culture-Negative Patient in The Gambia"/>

      <meta name="citation_author" content="Gerard A. J. Morris"/>
            <meta name="citation_author_institution" content="Bacterial Diseases Programme, Medical Research Council Laboratories, Fajara, The Gambia"/>
      <meta name="citation_author" content="Usman N. Ikumapayi"/>
            <meta name="citation_author_institution" content="Bacterial Diseases Programme, Medical Research Council Laboratories, Fajara, The Gambia"/>
      <meta name="citation_author" content="Martin Antonio"/>
            <meta name="citation_author_institution" content="Bacterial Diseases Programme, Medical Research Council Laboratories, Fajara, The Gambia"/>
      <meta name="citation_author" content="Stephen R. C. Howie"/>
            <meta name="citation_author_institution" content="Bacterial Diseases Programme, Medical Research Council Laboratories, Fajara, The Gambia"/>
      <meta name="citation_author" content="Richard A. Adegbola"/>
            <meta name="citation_author_institution" content="Bacterial Diseases Programme, Medical Research Council Laboratories, Fajara, The Gambia"/>

    <meta name="citation_date" content="2008/3/12"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0001773.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e1773"/>
    <meta name="citation_issue" content="3"/>
    <meta name="citation_volume" content="3"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=The value and complications of percutaneous transthoracic lung aspiration for the etiologic diagnosis of community acquired pneumonia.; citation_author=JAG Scott; citation_author=AJ Hall; citation_journal_title=Chest; citation_volume=116; citation_number=1; citation_pages=1716-1732; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Molecular diagnosis of bloodstream infections caused by non-cultivable bacteria.; citation_author=F Fenollar; citation_author=D Raoult; citation_journal_title=Int. J. Antimicrob. Agents.; citation_volume=Nov 30:; citation_number=2; citation_pages=S7-15; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=New developments in the diagnosis of bloodstream infections.; citation_author=RP Peters; citation_author=MA van Agtmael; citation_author=SA Danner; citation_author=PH Savelkoul; citation_author=CM Vandenbroucke-Grauls; citation_journal_title=Lancet Infect. Dis.; citation_volume=12; citation_number=3; citation_pages=751-60; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Impact of 16S rRNA gene sequence analysis for identification of bacteria on clinical microbiology and infectious diseases.; citation_author=JE Clarridge; citation_journal_title=Clin. Microbiol. Rev.; citation_volume=4; citation_number=4; citation_pages=840-62; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Detection and identification of microorganisms by gene amplification and sequencing.; citation_author=CA Petti; citation_journal_title=Clin. Infect. Dis.; citation_volume=44(8); citation_number=5; citation_pages=1108-14; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Overlap in the clinical features of pneumonia and malaria in African children.; citation_author=JD O’Dempsey; citation_author=TF McArdle; citation_author=BE Laurence; citation_journal_title=Transactions of the Royal Society of Tropical Medicine and Hygiene.; citation_volume=87; citation_number=6; citation_pages=662-5; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=Gambian Pneumococcal Vaccine Trial Group. Efficacy of nine-valent pneumococcal conjugate vaccine against pneumonia and invasive pneumococcal disease in The Gambia: randomised, double-blind, placebo-controlled trial.; citation_author=FT Cutts; citation_author=SM Zaman; citation_author=G Enwere; citation_author=S Jaffar; citation_author=OS Levine; citation_journal_title=Lancet.; citation_volume=365(9465); citation_number=7; citation_pages=1139-46; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Determination of bacterial load by real-time PCR using a broad-range (universal) probe and primers set.; citation_author=MA Nadkarni; citation_author=FE Martin; citation_author=NA Jacques; citation_author=N Hunter; citation_journal_title=Microbiology.; citation_volume=148(Pt 1); citation_number=8; citation_pages=257-66; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=The genome sequence of the food-borne pathogen Campylobacter jejuni reveals hypervariable sequences.; citation_author=J Parkhill; citation_author=BW Wren; citation_author=K Mungall; citation_author=JM Ketley; citation_author=C Churcher; citation_journal_title=Nature.; citation_volume=403(6770); citation_number=9; citation_pages=665-668; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Genetic diversity of Campylobacter jejuni isolates from farm animals and the farm environment.; citation_author=FM Colles; citation_author=K Jones; citation_author=RM Harding; citation_author=MCJ Maiden; citation_journal_title=Appl. Environ. Microbiol.; citation_volume=69; citation_number=10; citation_pages=7409-7413; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Campylobacter and related species, p. 1948–1956.; citation_author=MJ Blaser; citation_number=11; citation_date=1995; citation_publisher=Churchchill Livingstone; " />
      <meta name="citation_reference" content="citation_title=Campylobacter species and Guillain-Barré syndrome.; citation_author=I Nachamkin; citation_author=BM Allos; citation_author=T Ho; citation_journal_title=Clin. Microbiol.; citation_volume=Rev.11; citation_number=12; citation_pages=555-567; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Campylobacter jejuni: molecular biology and pathogenesis.; citation_author=KT Young; citation_author=LM Davis; citation_author=VJ Dirita; citation_journal_title=Nat. Rev. Microbiol.; citation_volume=9; citation_number=13; citation_pages=665-79; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Pathogenicity and convalescent excretion of Campylobacter in rural Egyptian children.; citation_author=MR Rao; citation_author=AB Naficy; citation_author=SJ Savarino; citation_author=R Abu-Elyazeed; citation_author=TF Wierzba; citation_journal_title=Am. J. Epidemiol.; citation_volume=154; citation_number=14; citation_pages=166-173; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Human campylobacteriosis in developing countries.; citation_author=AO Coker; citation_author=RD Isophi; citation_author=BN Thomas; citation_author=KO Amisu; citation_author=L Obi; citation_journal_title=Emerg. Infect. Dis.; citation_volume=8; citation_number=15; citation_pages=237-243; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Campylobacter, from obscurity to celebrity.; citation_author=JP Butzler; citation_journal_title=Clin. Microbiol. Infect.; citation_volume=10; citation_number=16; citation_pages=868-876; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Identification of genomic differences between Campylobacter jejuni subsp. jejuni and C. jejuni subsp. doylei at the nap locus leads to the development of a C. jejuni subspeciation multiplex PCR method.; citation_author=WG Miller; citation_author=CT Parker; citation_author=S Heath; citation_author=AJ Lastovica; citation_journal_title=BMC Microbiol.; citation_volume=28; citation_number=17; citation_pages=7-11; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Genotyping of Campylobacter species.; citation_author=TM Wassenaar; citation_author=DG Newell; citation_journal_title=Appl. Environ. Microbiol.; citation_volume=66; citation_number=18; citation_pages=1-9; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Multilocus sequence typing system for Campylobacter jejuni.; citation_author=KE Dingle; citation_author=FM Colles; citation_author=DRA Wareing; citation_author=R Ure; citation_author=AJ Fox; citation_journal_title=J. Clin. Microbiol.; citation_volume=39; citation_number=19; citation_pages=14-23; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Molecular characterization of Campylobacter jejuni clones: a basis for epidemiologic investigation.; citation_author=KE Dingle; citation_author=FM Colles; citation_author=R Ure; citation_author=JA Wagenaar; citation_author=B Duim; citation_journal_title=Emerg. Infect. Dis.; citation_volume=8; citation_number=20; citation_pages=949-955; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Molecular evidence for dissemination of unique Campylobacter jejuni clones in Curaçao, Netherlands Antilles.; citation_author=B Duim; citation_author=PCR Godschalk; citation_author=NVD Braak; citation_author=KE Dingle; citation_author=JR Dijkstra; citation_journal_title=J. Clin. Microbiol.; citation_volume=41; citation_number=21; citation_pages=5593-5597; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Multilocus sequence typing for comparison of veterinary and human isolates of Campylobacter jejuni.; citation_author=G Manning; citation_author=CG Dowson; citation_author=MC Bagnall; citation_author=IH Ahmed; citation_author=M West; citation_journal_title=Appl. Environ. Microbiol.; citation_volume=69; citation_number=22; citation_pages=6370-6379; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Does blood of healthy subjects contain bacterial ribosomal DNA?; citation_author=S Nikkari; citation_author=IJ McLaughlin; citation_author=W Bi; citation_author=DE Dodge; citation_author=DA Relman; citation_journal_title=J. Clin. Microbiol.; citation_volume=39; citation_number=23; citation_pages=1956-59; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Are there naturally occurring pleomorphic bacteria in the blood of healthy humans?; citation_author=RW McLaughlin; citation_author=H Vali; citation_author=PC Lau; citation_author=RG Palfree; citation_author=A De Ciccio; citation_journal_title=J. Clin. Microbiol.; citation_volume=40; citation_number=24; citation_pages=4771-75; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Bacterial infections, a major cause of death among children in Africa.; citation_author=EK Mulholland; citation_author=RA Adegbola; citation_journal_title=N. Engl. J. Med.; citation_volume=352(1); citation_number=25; citation_pages=39-47; citation_date=2005; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001773" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="A Novel Campylobacter jejuni Sequence Type from a Culture-Negative Patient in The Gambia"/>
    <meta name="twitter:description" content="The introduction of molecular diagnostic methods is crucial for improved understanding of the aetiology and epidemiology of bacterial infections in communities in resource poor settings. A blood sample from a 7 month old patient diagnosed with malaria in 2001 in a Gambian outpatient clinic was reported as culture negative after it was subjected to traditional bacterial culture protocols. We re-addressed the analysis of the blood sample from this case more recently (after 6.5 years in archival storage) in pilot work establishing 16S rRNA PCR in our molecular laboratory. Initial 16S rRNA PCR results confirmed the presence of bacterial DNA in the sample. 16S rRNA sequence analysis identified the organism as Campylobacter spp. In light of the molecular evidence we successfully grew the organism using appropriate culture conditions and subsequently biochemically confirmed that the isolate was Campylobacter jejuni. PCR and DNA sequencing of a set of seven C. jejuni housekeeping genes and in silico Multilocus Sequence Typing (MLST) analysis revealed that the isolate exhibits a novel sequence type (ST) of C. jejuni (ST 2928) and belongs to ST-443 clonal complex. This study demonstrates the potential for molecular tools to enhance the diagnosis of bacterial infections, which remain a major killer globally, not least in children in the developing world. Improvements in diagnostics are needed, and will be important not only for sick individuals but also for populations, where better measures of disease burden will contribute significantly to the improvement of public health policy."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0001773.g002"/>

  <meta property="og:title" content="A Novel Campylobacter jejuni Sequence Type from a Culture-Negative Patient in The Gambia" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001773" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=740'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=2930'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=5385&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0001773">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001773" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001773&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    A Novel <i>Campylobacter jejuni</i> Sequence Type from a Culture-Negative Patient in The Gambia
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Gerard A. J. Morris
              <span class="corresponding">mail</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">* E-mail:</span> <a href="mailto:gmorris@mrc.gm">gmorris@mrc.gm</a></p>

                <p>Affiliation:
                  Bacterial Diseases Programme, Medical Research Council Laboratories, Fajara, The Gambia
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Usman N. Ikumapayi, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Bacterial Diseases Programme, Medical Research Council Laboratories, Fajara, The Gambia
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Martin Antonio, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Bacterial Diseases Programme, Medical Research Council Laboratories, Fajara, The Gambia
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Stephen R. C. Howie, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Bacterial Diseases Programme, Medical Research Council Laboratories, Fajara, The Gambia
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Richard A. Adegbola
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Bacterial Diseases Programme, Medical Research Council Laboratories, Fajara, The Gambia
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: March 12, 2008</li>
    <li>DOI: 10.1371/journal.pone.0001773</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0001773" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0001773" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001773" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0001773" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0001773-t001" data-doi="info:doi/10.1371/journal.pone.0001773" data-uri="info:doi/10.1371/journal.pone.0001773.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001773.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001773-g001" data-doi="info:doi/10.1371/journal.pone.0001773" data-uri="info:doi/10.1371/journal.pone.0001773.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001773.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001773-g002" data-doi="info:doi/10.1371/journal.pone.0001773" data-uri="info:doi/10.1371/journal.pone.0001773.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001773.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001773">Reader Comments (0)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0001773" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1"></a><p>The introduction of molecular diagnostic methods is crucial for improved understanding of the aetiology and epidemiology of bacterial infections in communities in resource poor settings. A blood sample from a 7 month old patient diagnosed with malaria in 2001 in a Gambian outpatient clinic was reported as culture negative after it was subjected to traditional bacterial culture protocols. We re-addressed the analysis of the blood sample from this case more recently (after 6.5 years in archival storage) in pilot work establishing <em>16S rRNA</em> PCR in our molecular laboratory. Initial <em>16S rRNA</em> PCR results confirmed the presence of bacterial DNA in the sample. <em>16S rRNA</em> sequence analysis identified the organism as <em>Campylobacter</em> spp. In light of the molecular evidence we successfully grew the organism using appropriate culture conditions and subsequently biochemically confirmed that the isolate was <em>Campylobacter jejuni</em>. PCR and DNA sequencing of a set of seven <em>C. jejuni</em> housekeeping genes and <em>in silico</em> Multilocus Sequence Typing (MLST) analysis revealed that the isolate exhibits a novel sequence type (ST) of <em>C</em>. <em>jejuni</em> (ST 2928) and belongs to ST-443 clonal complex. This study demonstrates the potential for molecular tools to enhance the diagnosis of bacterial infections, which remain a major killer globally, not least in children in the developing world. Improvements in diagnostics are needed, and will be important not only for sick individuals but also for populations, where better measures of disease burden will contribute significantly to the improvement of public health policy.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>Morris GAJ, Ikumapayi UN, Antonio M, Howie SRC, Adegbola RA (2008) A Novel <em>Campylobacter jejuni</em> Sequence Type from a Culture-Negative Patient in The Gambia. PLoS ONE 3(3):
          e1773.
            doi:10.1371/journal.pone.0001773</p><p><strong>Academic Editor: </strong>David M. Ojcius, University of California Merced, United States of America</p><p><strong>Received:</strong> December 14, 2007; <strong>Accepted:</strong> February 9, 2008; <strong>Published:</strong> March 12, 2008</p><p><strong>Copyright:</strong> © 2008 Morris et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>The study was mainly supported by funding from Medical Research Council United Kingdom. The MRC UK was not directly involved in the design and conduct of the study, in the collection, analysis, and interpretation of the data, nor in the preparation, review, or approval of the manuscript.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>The emergence of opportunistic pathogens in immunocompromised hosts, such as HIV-infected patients, has increased the burden on clinical diagnostic laboratory services in developing countries. Reliable and accurate identification of these pathogens poses a serious challenge, necessitating the widening of the scope of the assays traditionally employed in resource poor settings. Traditional bacterial culturing techniques have many deficiencies, necessitating a search for adjunctive or alternative approaches such as molecular protocols. Errors in identifying poorly described, rarely isolated, or phenotypically aberrant bacterial strains are likely to be common in routine clinical diagnostic laboratories using traditional culturing and biochemical methods exclusively.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>Traditional bacterial culturing protocols rely on replicating <em>in vitro</em> the physical and biochemical requirements of each bacterial species. The traditional culturing methods are limiting by definition, <em>e.g.</em> in a mixed microbial population specific culture media may enrich for a single bacterial species, while a distinct species with distinct physical and metabolic requirements will not be detected despite being present in the sample. The limitations of traditional bacterial culturing technologies have been highlighted by the low culture success rate from samples taken at foci of infection, <em>e.g.</em> lung aspirate samples in pneumonia patients <a href="#pone.0001773-Scott1">[1]</a>.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>Molecular methods have many advantages over traditional protocols employed for identification and characterization of bacteria in clinical samples, reviewed <a href="#pone.0001773-Fenollar1">[2]</a>, <a href="#pone.0001773-Peters1">[3]</a>. Molecular protocols have the potential to rapidly identify the aetiological agents of unexplained, culture-negative illnesses or to resolve or better characterize ‘overlap’ clinical syndromes facilitating more appropriate and more directed therapy. Molecular protocols may play an important role in epidemiological studies where failure of culture-dependent methods to detect low level bacterial carriage compromises the study of changes in microbial ecology in patients following vaccination.</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p>‘Broad-range’ <em>16S rRNA</em> PCR and DNA sequencing have emerged as potential adjunctive tools or as alternative culture-independent methods for detecting and identifying pathogenic bacteria in clinical samples. Organism specific MLST analysis allows more definitive identification of bacterial isolates providing useful epidemiological data for identifying the source and course of outbreaks. The extensive and rapidly growing internet based <em>16S rRNA</em> and MLST databases facilitate rapid identification of sequences to the species or genus level. The advantages and caveats associated with this broad range PCR approach have been reviewed elsewhere <a href="#pone.0001773-Clarridge1">[4]</a>–<a href="#pone.0001773-Petti1">[5]</a></p>
<a id="article1.body1.sec1.p5" name="article1.body1.sec1.p5"></a><p>‘Overlap’ syndromes, where different diseases are difficult to distinguish clinically, create a challenge for both health workers and health systems, especially in resource-poor countries. This is particularly so in malaria endemic settings where clinical symptoms of malaria are practically indistinguishable from those of pneumonia and can compromise the process of case management strategy <a href="#pone.0001773-ODempsey1">[6]</a>. We outline how the application of a range of molecular tools applied to an archival blood sample followed by the utilization of internet bioinformatic resources helped to characterize a novel ST of <em>C</em>. <em>jejuni</em> in the Gambia. Our results make a strong case for the routine use of molecular techniques, which if applied prudently can play a vital role in clinical diagnosis and monitoring of antibacterial therapy, while retaining cost effectiveness in resource poor developing countries.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Methods"></a><h3>Methods</h3><a id="article1.body1.sec2.p1" name="article1.body1.sec2.p1"></a><p>Informed written consent was obtained as described elsewhere <a href="#pone.0001773-Cutts1">[7]</a>. The study and all protocol revisions were approved by the ethics committees of the Gambia Government/MRC, the London School of Hygiene and Tropical Medicine and by the WHO Secretariat Committee for Research Involving Human Subjects.</p>
<a id="article1.body1.sec2.p2" name="article1.body1.sec2.p2"></a><p>Forty seven clinical samples (all lung aspirate and venous blood) from 33 patients were retrieved from our archives; these samples had been stored from this previous study <a href="#pone.0001773-Cutts1">[7]</a>. These samples were selected for a pilot study, undertaken to establish <em>16S rRNA</em> PCR in our molecular laboratory. Samples were blinded on assignment of a laboratory number before being processed in batches for DNA extraction and <em>16S rRNA</em> PCR. Two of the blood samples (sample numbers 2 and 3) in addition to being positive for <em>16S rRNA</em> PCR showed unusual eletrophoretic shifts on agarose gel eletrophoresis, relative to the expected size of the PCR products (see <a href="#s3">results</a> section below). This aroused our suspicion that the samples might contain unusual bacterial species, and it was on this basis that we pursued these samples for further molecular characterisation. Lack of availability of MLST PCR and sequencing primers, however, restricted our molecular characterisation of the bacterium present to just one of these blood samples, sample number 2, which is highlighted in this case report.</p>

<h4>The patient behind the sample</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>A whole blood sample had been taken from a 7 month old girl recruited in 2001 as part of a previously described study <a href="#pone.0001773-Cutts1">[7]</a>. The child presented with a several day history of cough, fever and diarrhoea and was not on any medication at the time of assessment. She appeared sick, had a temperature of 39.3°C and had a raised respiratory rate of 46 per minute without signs of severe respiratory distress. A blood film showed a significant <em>Plasmodium falciparum</em> parasitaemia (200 parasites per high-powered field). She was admitted to hospital with a diagnosis of malaria, and treated according to local guidelines with chloroquine and pyrimethamine-sulfadoxine. A blood culture taken on admission was reported as negative. She was discharged after 2 days having made a good recovery.</p>
<a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2"></a><p>A blood sample from this case was retried from archival storage (−80°C for 6.5 years) and examined by molecular methods for the presence of bacterial DNA as part of a molecular pilot study, as indicated above.</p>


<h4>Extraction of DNA from the sample</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>100 µl of each of the archival blood samples was added to 180 µl of lysozyme buffer (20 mM Tris-HCl, 2mM EDTA, 1.2 % Triton X-100, 20 mg/ml lysozyme (Sigma)) for 30 min at 37°C. Proteinase K (25 µl) was added and the sample incubated at 70°C for 30 min. The sample was subjected to the standard Qiagen DNA Mini Kit (Qiagen Ltd. UK) extraction protocol with 2×25 µl elutions on the same column using the kit elution buffer. The DNA eluate was stored at −20°C prior to <em>16S rRNA</em> PCR.</p>


<h4><em>16S rRNA</em> standard PCR conditions</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>Each 25 µl <em>16S rRNA</em> PCR reaction contained 17.5 µl of molecular biology-grade water (Sigma Aldrich Company Ltd.), 2.5 µl of 10×PCR buffer (Qiagen Ltd.), 1 µl of 12.5 µM each forward and reverse PCR primers, 1 µl of a 10 mM dNTPs (Invitrogen Ltd.), 0.25 µl of <em>Taq</em> DNA polymerase (Qiagen Ltd.), and 2 µl of DNA extracted from the clinical sample. Non-template control (NTC) samples contained the same PCR reagents as above but had 2 µl of water substituted for template. The positive control was a local isolate of <em>Streptococcus pneumoniae</em>. PCR amplification conditions on an MJ thermocycler were as follows: 94°C for 10 min, followed by 25 cycles of 94°C for 30 s, 60°C for 1 min, and 72°C for 1 min, with a final extension at 72°C for 10 min and storage at 4°C.</p>


<h4>MLST was performed as follows</h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>Seven PCR amplicons were prepared from purified DNA extracted from a glycerol stock of the bacterial isolate by using the PCR primers shown in <a href="#pone-0001773-t001"><strong>Table 1</strong></a>. Each 25 µl PCR mixture was identical to the <em>16S rRNA</em> PCR above, except for the primers and 2 µl of purified <em>C. jejuni</em> DNA as template. The amplification conditions were 94°C for 10 min, followed by 35 cycles of 94°C for 30 s, 50°C for 1 min, and 72°C for 1 min, with a final extension at 72°C for 10 min and storage at 4°C. Non-template control (NTC) samples contained the same reagents as above for each of the 7 distinct PCR reactions but had 2 µl of water substituted for template in each case.</p>
<div class="figure" id="pone-0001773-t001"><div class="img"><a name="pone-0001773-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001773.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001773" data-uri="info:doi/10.1371/journal.pone.0001773.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001773.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001773.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001773.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001773.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001773.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001773.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001773.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001773.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001773.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>MLST alleles, GenBank accession numbers, PCR and sequencing primers (5′ to 3′) for blood samples No. 2 &amp; 3.</span></strong></p><span>doi:10.1371/journal.pone.0001773.t001</span></div><a id="article1.body1.sec2.sec4.p2" name="article1.body1.sec2.sec4.p2"></a><p>All <em>16S rRNA</em> PCR, MLST PCR and DNA sequencing oligonucleotide primers were obtained from Sigma Genosys (<a href="#pone-0001773-t001"><strong>Table 1</strong></a>).</p>


<h4>Agarose gel electrophoresis</h4>
<a id="article1.body1.sec2.sec5.p1" name="article1.body1.sec2.sec5.p1"></a><p>All standard <em>16S rRNA</em> PCR and <em>C. jejuni</em> specific MLST PCR products were assessed for expected PCR product size on 2 % (w/v) agarose gels with a 100 bp DNA ladder (New England Biolabs). Aliquots of PCR products were electrophoresed at 150V for 45 min; DNA was visualized using ethidium bromide and photographed after UV transillumination with PD-Quest 7.3.1 basic 2D analysis software (Bio-Rad).</p>


<h4><em>16S rRNA</em> and MLST gene sequencing</h4>
<a id="article1.body1.sec2.sec6.p1" name="article1.body1.sec2.sec6.p1"></a><p><em>16S rRNA</em> and <em>C. jejuni</em> specific MLST PCR products were sent to Macrogen in South Korea (<a href="http://www.macrogen.com">http://www.macrogen.com</a>) for purification and DNA sequencing. For primers used in PCR and sequencing reactions see <a href="#pone-0001773-t001"><strong>Table 1:</strong></a></p>


<h4>Bioinformatic applications</h4>
<a id="article1.body1.sec2.sec7.p1" name="article1.body1.sec2.sec7.p1"></a><p>All forward and reverse sequences were aligned for each amplicon using the ClustalW program to assess the specificity of the sequencing reactions (<a href="http://www.ebi.ac.uk/clustalw/">http://www.ebi.ac.uk/clustalw/</a>). Reverse sequences were reverse complemented (<a href="http://bioinformatics.org/sms/rev_comp.html">http://bioinformatics.org/sms/rev_comp.h​tml</a>) prior to alignment using ClustalW<strong>.</strong></p>
<a id="article1.body1.sec2.sec7.p2" name="article1.body1.sec2.sec7.p2"></a><p>All sequence data was subjected to BLAST analysis (non-redundant nucleotide sequence database search) (<a href="http://www.ncbi.nlm.nih.gov/BLAST/">http://www.ncbi.nlm.nih.gov/BLAST/</a>) to definitively identify each respective 16S rRNA and MLST gene amplicon.</p>
<a id="article1.body1.sec2.sec7.p3" name="article1.body1.sec2.sec7.p3"></a><p>MLST alleles and sequence types were assigned to the isolate by comparing data to the <em>Campylobacter</em> MLST database (<a href="http://campylobacter.mlst.net">http://campylobacter.mlst.net</a>). Data for the newly described alleles and STs were deposited in the <em>Campylobacter</em> MLST database <a href="http://pubmlst.org/campylobacter/">http://pubmlst.org/campylobacter/</a>. All resulting gene sequences from this study were deposited in Genbank (<a href="#pone-0001773-t001"><strong>Table 1</strong></a>).</p>


<h4>Microbiological culture, isolation and biochemical analysis of <em>C</em>. <em>jejuni</em> from the sample</h4>
<a id="article1.body1.sec2.sec8.p1" name="article1.body1.sec2.sec8.p1"></a><p>A 25 µl aliquot of the archival blood sample was inoculated and streaked on <em>Campylobacter</em>-selective agar plates Columbia agar (cat # Oxoid CM 0331) prepared aseptically with supplement (Oxoid SR0098E) in 10% sheep blood agar. The cultured plate was incubated in a microbiological culture jar, 10% CO<sub>2</sub>, 5% Oxygen and 85% Nitrogen, achieved using a Campy-gas generating kit (cat # Oxoid BR0060A) and a palladium catalyst. The jar was closed tightly and incubated for 48 hrs at 42°C. Isolates were biochemically characterized using Oxidase cytochrome (N,N,N –Tetramethyl-P-Phenylenediamine dihydrochloride) (Sigma), Catalase (Riedel-de Haen) and Ninhydrin (BactiDrop Ninhydrin, Remel) tests were all carried out for definitive and differential biochemical characterization.</p>

</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Results"></a><h3>Results</h3>
<h4><em>16S rRNA</em> PCR and sequencing analysis</h4>
<a id="article1.body1.sec3.sec1.p1" name="article1.body1.sec3.sec1.p1"></a><p>Of the first batch of six archival clinical sample gDNA extracts, only two were positive for bacterial DNA by <em>16S rRNA</em> PCR, sample 2 and 3 (<a href="#pone-0001773-g001"><strong>Figure 1</strong></a>), these samples represented venous blood samples from 2 distinct patients. Samples 1, 4, 5 and 6 tested negative for bacterial DNA. The gel mobility shift of the 16S rDNA band from the archival blood samples 2 and 3 were visibly distinct in size from the positive control (<em>Streptococcus pneumoniae</em>). This observation is based on an expected amplicon size of 466 bp from PCR primers designed to generate products between nucleotide residues 331 and 797 of the <em>Escherichia coli</em> 16S rRNA gene <a href="#pone.0001773-Nadkarni1">[8]</a>. The non-template control samples were negative and the positive control sample was positive as expected. Sequencing of the 16S rRNA PCR products resulted in clear, clean sequence traces for both forward and reverse reactions from sample 2 and 3. There was 100% alignment agreement for forward and reverse sequencing traces for each sample using the ClustalW sequence alignment program. This indicated that rather than a mixed bacterial population, one major organism was present in each of the samples. <em>In silico</em> BLAST analysis of the <em>16S rRNA</em> sequence data for sample 2 indicated a 100% homology with the NCTC 11168 <em>C</em>. <em>jejuni</em> <a href="#pone.0001773-Parkhill1">[9]</a>. Sample 3 was identified by BLAST analysis of the sequencing data as <em>Trichococcus</em> spp (96% homology), [EU418263]<strong>.</strong> We did not attempt to culture the <em>Trichococcus</em> spp, as there was only sufficient sample to do a gDNA extraction for <em>16S rRNA</em> PCR but not for culture work. Molecularly the <em>Trichococcus</em> spp. was not further characterized due to the lack of specific MLST PCR and sequencing primers for this organism.</p>
<div class="figure" id="pone-0001773-g001"><div class="img"><a name="pone-0001773-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001773.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001773" data-uri="info:doi/10.1371/journal.pone.0001773.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001773.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001773.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001773.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001773.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001773.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001773.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001773.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001773.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001773.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>2 % agarose gel, <em>16S rRNA</em> PCR of gDNA extracted from archival blood samples 1 to 6.</span></strong></p><a id="article1.body1.sec3.sec1.fig1.caption1.p1" name="article1.body1.sec3.sec1.fig1.caption1.p1"></a><p>2 µl of gDNA in 25 µl PCR reaction with 25×PCR cycles. 5 µl loaded on gel. Image is reverse colour under UV transillumination (Bio-Rad). Positive control (+Control) used 2 µl of gDNA from a local <em>Streptococcus pneumoniae</em> isolate. Non template control (NTC) consisting of 25 µl PCR mix only, water control 2 µl molecular grade water in place of gDNA.</p>
<span>doi:10.1371/journal.pone.0001773.g001</span></div>

<h4>MLST of the <em>Campylobacter jejuni</em> isolate</h4>
<a id="article1.body1.sec3.sec2.p1" name="article1.body1.sec3.sec2.p1"></a><p>Six of the seven <em>C.jejuni</em> specific MLST genes were successfully amplified by PCR; the fifth <em>tkt</em> (transketolase) failed initial PCR amplification but was successfully PCR amplified and sequenced using the sequencing primers in a nested approach (<a href="#pone-0001773-g002"><strong>Figure 2</strong></a>). All 7 MLST house keeping genes were successfully sequenced in both directions in duplicate, using a direct and nested sequencing approach (Genbank accession numbers EU056829 to EU056835). High quality sequence traces were obtained for all 7 products from both the forward and reverse sequencing reactions and from the nested reactions. ClustalW alignment indicated 100% agreement between forward and reverse reaction sequencing products for all 7 MLST <em>C.jejuni</em> specific MLST genes. Sequences resulting from the standard and nested reactions were in 100% agreement.</p>
<div class="figure" id="pone-0001773-g002"><div class="img"><a name="pone-0001773-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001773.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001773" data-uri="info:doi/10.1371/journal.pone.0001773.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001773.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001773.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001773.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001773.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001773.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001773.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001773.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001773.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001773.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>2 % agarose gel, PCR of gDNA extracted from a <em>C. jejuni</em> isolate using both PCR and sequencing oligonucleotide primers for <em>C.jejuni</em> MLST genes 1 to 7.</span></strong></p><a id="article1.body1.sec3.sec2.fig1.caption1.p1" name="article1.body1.sec3.sec2.fig1.caption1.p1"></a><p>2 µl extracted gDNA used in 25 µl PCR reaction with 35×PCR cycles. 3 µl loaded on gel. Image is reverse colour under UV tranillumination (Bio-Rad). NTC = Non-template control, 3 µl of each individual reaction combined and 21 µl loaded on gel.</p>
<span>doi:10.1371/journal.pone.0001773.g002</span></div><a id="article1.body1.sec3.sec2.p2" name="article1.body1.sec3.sec2.p2"></a><p>Submission of the MLST sequence data for all 7 house keeping genes to the campylobacter MLST database, indicated that while all 7 alleles had been described individually previously, they had not been described in the combination that we found. Thus the isolate of <em>C</em>. <em>jejuni</em> constituted a novel ST not previously described in the literature or the database. The novel ST was designated ST-2928: With an allele profile code; 24, 2, 2, 15, 294, 3, 12, for <em>aspA</em>, <em>glnA</em>, <em>uncA</em>, <em>glyA</em>, <em>pgm</em>, <em>tkt</em>, and <em>gltA</em> respectively. The isolate is a member of the previously described <em>C</em>.<em>jejuni</em> clonal complex ST-443 also isolated and identified in humans <a href="#pone.0001773-Colles1">[10]</a>.</p>


<h4>Bacterial Culture</h4>
<a id="article1.body1.sec3.sec3.p1" name="article1.body1.sec3.sec3.p1"></a><p>Culturing an aliquot of the clinical archival sample number 2 under optimum culture conditions for <em>C</em>. <em>jejuni</em> (in light of the <em>16S rRNA</em> data) resulted in florid bacterial growth with morphological characteristics typical of <em>C.jejuni</em>. Colonies from the isolate were identified as Gram-negative rods, and biochemically characterized as oxidase, catalase and ninhydrin positive all consistent with <em>C</em>. <em>jejuni</em>.</p>

</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec4.p1" name="article1.body1.sec4.p1"></a><p>We have discovered a novel <em>C</em>.<em>jejuni</em> isolate in The Gambia (ST-2928) using a molecular based approach as an adjunct to traditional culture and biochemical characterisation protocols. Traditional bacterial culture protocols had failed to isolate the organism in a blood sample when the patient first presented 6.5 years previously; hence the discovery was made as a direct consequence of using molecular protocols. There was no history of treatment with antimicrobials before admission to account for the negative blood culture. Rather, inappropriate incubation of the blood sample at 37°C as opposed to 42°C may explain why the organism was not detected originally in the patient’s blood sample. The laboratory however, had no <em>a priori</em> reason to consider culturing this patient’s blood sample at 42°C when the patient first presented. The optimum temperatures for most bacteria that are pathogenic to humans are 35°C–37°C, and hence the temperatures most frequently used to culture organisms in a clinical diagnostic laboratory. Although <em>Campylobacter</em> species are not difficult to isolate <em>per se</em>, they do require special media, special temperature and atmospheric conditions for growth <a href="#pone.0001773-Blaser1">[11]</a>. <em>C</em>.<em>jejuni</em> is grown on especially selective agar plates at 42°C, the normal avian body temperature (which constitutes an important source and natural reservoir of the organism). When more appropriate culture conditions were applied, after initial <em>16S rRNA</em> sequencing identified the organism in the sample, we obtained florid culture growth, sufficient numbers of bacteria for optimal biochemical testing, and sufficient quantities of DNA for successful MLST analysis of the <em>C</em>.<em>jejuni</em> isolate.</p>
<a id="article1.body1.sec4.p2" name="article1.body1.sec4.p2"></a><p>Our aim was to apply molecular protocols such <em>16S rRNA</em> PCR, DNA sequencing and MLST analysis to an archival blood sample from a complex clinical case, which was reported as culture negative for bacterial growth after original presentation. The recent completion of the sequencing of the <em>C</em>. <em>jejuni</em> genome (NCTC11168) has greatly facilitated advances in molecular based detection and characterization of this pathogen and important public health burden <a href="#pone.0001773-Parkhill1">[9]</a>. <em>C</em>. <em>jejuni</em> is a member of the delta-epsilon group of proteobacteria and is Gram-negative, flagellate, curved or spiral, microaerophilic and motile. <em>C. jejuni</em> is the leading cause of bacterial food-borne diarrhoeal disease throughout the world <a href="#pone.0001773-Nachamkin1">[12]</a> and has been reviewed elsewhere <a href="#pone.0001773-Young1">[13]</a>. It has also been described as an important cause of diarrhoeal illness in African children under 5 years of age <a href="#pone.0001773-Rao1">14</a>–<a href="#pone.0001773-Butzler1">16</a>. <em>Campylobacter jejuni</em> has been divided into two subspecies: <em>C. jejuni</em> subsp. <em>jejuni</em> and <em>C. jejuni</em> subsp. <em>doylei.</em> Almost all of the <em>C. jejuni</em> strains isolated are <em>C. jejuni</em> subsp. <em>jejuni</em>. Although subspecies <em>doylei</em> strains are isolated only rarely, they differ from subsp. <em>jejuni</em> in two important aspects: they are obtained mainly from human clinical samples and are associated more often with bacteraemia <a href="#pone.0001773-Miller1">[17]</a>.</p>
<a id="article1.body1.sec4.p3" name="article1.body1.sec4.p3"></a><p>To study the epidemiology of <em>Campylobacter</em> infections, multiple typing methods have been developed <a href="#pone.0001773-Wassenaar1">[18]</a>. The MLST method was developed by Dingle <em>et al</em>, to characterize <em>C</em>. <em>jejuni</em> strains and to identify clonal lineages in this species <a href="#pone.0001773-Dingle1">[19]</a>–<a href="#pone.0001773-Dingle2">[20]</a>, <a href="#pone.0001773-Colles1">[10]</a>, <a href="#pone.0001773-Duim1">[21]</a>–<a href="#pone.0001773-Manning1">[22]</a>. This method uses genetic variations at multiple chromosomal locations and allows generation of sequence data, which are deposited in internet databases for comparison with DNA sequences of other isolates.</p>
<a id="article1.body1.sec4.p4" name="article1.body1.sec4.p4"></a><p>Despite its accuracy, 16S rRNA gene sequence analysis lacks widespread use beyond the large and reference laboratories because of technical and cost considerations, this may be a confounding issue in developing countries such as The Gambia where technological and financial resources are limited. Thus, a future challenge is to incorporate 16S rRNA gene sequencing into routine protocols in concert with traditional microbiological and biochemical testing. The availability of free internet based molecular bioinformatic tools such as search engines and databases for molecular analysis represents a great resource for laboratories within developing countries, using or planning to use molecular protocols in their work.</p>
<a id="article1.body1.sec4.p5" name="article1.body1.sec4.p5"></a><p>Amplification and sequencing of the <em>16S rRNA</em> has become the method of choice for ‘broad-range’ identification of bacterial species in clinical samples or cultures but for more definitive characterization it is necessary to complete organism specific MLST analysis, and this was the rationale behind our approach in this case. It has been suggested that the clinical interpretation of ‘broad-range’ <em>16S rRNA</em> PCR and sequencing results can be problematical <em>e.g</em>. the protocol has the potential to detect all bacterial DNA present in a given clinical sample, including non-viable or even degraded organisms. A background level of bacterial DNA might be present in blood of patients without any overt clinical signs of bacteraemia <a href="#pone.0001773-Nikkari1">[23]</a>–<a href="#pone.0001773-McLaughlin1">[24]</a>. The consequences of detection of bacterial DNA in blood in so called ‘healthy’ patients has not been adequately addressed to date. An important <em>caveat</em> to these observations is that low levels of bacterial nucleic acids may be due to contamination of reagents and equipment acquired in the process of sample collection and processing or transient bacteraemia. Contamination, however miniscule, has the potential to significantly reduce the sensitivity of the molecular protocols, if not adequately addressed. Contamination is not an issue in the current case as a high load of <em>C</em>. <em>jejuni</em> was observed without any trace of 16S rDNA in the non-template controls (See <a href="#pone-0001773-g001"><strong>Figure 1</strong></a>).</p>
<a id="article1.body1.sec4.p6" name="article1.body1.sec4.p6"></a><p>While molecular test results are less subjective than those from traditional microbiological/biochemical characterization protocols, it would be premature to advocate replacement of the traditional culture protocols at this juncture with molecular technology. Molecular protocols may however act as an important adjunct to traditional culturing. It may be useful to incorporate the routine use of molecular protocols such as <em>16S rRNA</em> PCR and sequencing as an initial screen for microbial pathogens in clinical samples, this would allow more selective and efficient use of traditional culturing for confirmation, the cost of the molecular methods being offset by using less traditional culture associated consumables. Overall, gene amplification and DNA sequencing from culture and clinical material may improve our understanding of microbial pathogenesis and better predict responses to therapy and clinical outcome; they may also provide much needed insight into the complex ecology of mixed infections <em>in vivo</em>.</p>
<a id="article1.body1.sec4.p7" name="article1.body1.sec4.p7"></a><p>More broadly molecular methods such as these have potentially important clinical and public health applications. From the clinical point of view, key diagnostic possibilities for this patient included serious bacterial infection (including pneumonia, infective diarrhoea, septicaemia), and malaria. In this case the diagnoses were two: there was clear evidence for malaria but the co-existing bacteraemia was missed. While this child recovered from her bacteraemia without specific treatment (though possibly assisted unintentionally by the sufadoxine contained in her malaria therapy), invasive bacterial disease, often unrecognised, is a major killer of children in Africa and globally and improvements in diagnostics are urgently needed to improve outcomes (Mulholland and Adegbola 2005 <a href="#pone.0001773-Mulholland1">[25]</a>. This case indicates that molecular methods can improve diagnosis when used in concert with traditional methodologies. A key challenge is to establish more widespread use of molecular protocols in developing world settings. Such advances will be important not only at the bedside but at the population level where better measures of disease burden will contribute significantly to the improvement of public health policy.</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>We would like to thank Prof Felicity Cutts who oversaw the PVT project that provided the clinical samples used in this study and for providing critical comments on the manuscript. Ekow Biney for the original blood culture work. Ebou Mbowe and Mark Saaka for technical assistance, and the <em>Campylobacter jejuni</em> ST database.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: RA GM MA. Performed the experiments: GM UI. Analyzed the data: GM. Wrote the paper: GM. Other: Contributed to the clinical paediatric aspects of the manuscript and to comments on the clinical and public health significance of the work: SH. Reviewed the manuscript: RA MA UI.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0001773-Scott1" id="pone.0001773-Scott1"></a>Scott JAG, Hall AJ (1999) The value and complications of percutaneous transthoracic lung aspiration for the etiologic diagnosis of community acquired pneumonia. Chest  116: 1716–1732.  <ul class="find" data-citedArticleID="1072388" data-doi="10.1378/chest.116.6.1716"><li><a href="http://dx.doi.org/10.1378/chest.116.6.1716" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+value+and+complications+of+percutaneous+transthoracic+lung+aspiration+for+the+etiologic+diagnosis+of+community+acquired+pneumonia." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+value+and+complications+of+percutaneous+transthoracic+lung+aspiration+for+the+etiologic+diagnosis+of+community+acquired+pneumonia.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0001773-Fenollar1" id="pone.0001773-Fenollar1"></a>Fenollar F, Raoult D (2007) Molecular diagnosis of bloodstream infections caused by non-cultivable bacteria. Int. J. Antimicrob. Agents.  Nov 30:: Suppl 1S7–15.  <ul class="find" data-citedArticleID="1072362" data-doi="10.1016/j.ijantimicag.2007.06.024"><li><a href="http://dx.doi.org/10.1016/j.ijantimicag.2007.06.024" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Molecular+diagnosis+of+bloodstream+infections+caused+by+non-cultivable+bacteria." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Molecular+diagnosis+of+bloodstream+infections+caused+by+non-cultivable+bacteria.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0001773-Peters1" id="pone.0001773-Peters1"></a>Peters RP, van Agtmael MA, Danner SA, Savelkoul PH, Vandenbroucke-Grauls CM (2004) New developments in the diagnosis of bloodstream infections. Lancet Infect. Dis.  12: 751–60.  <ul class="find" data-citedArticleID="1072382" data-doi="10.1016/s1473-3099(04)01205-8"><li><a href="http://dx.doi.org/10.1016/s1473-3099(04)01205-8" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=New+developments+in+the+diagnosis+of+bloodstream+infections." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22New+developments+in+the+diagnosis+of+bloodstream+infections.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0001773-Clarridge1" id="pone.0001773-Clarridge1"></a>Clarridge JE 3rd (2004) Impact of 16S rRNA gene sequence analysis for identification of bacteria on clinical microbiology and infectious diseases. Clin. Microbiol. Rev.  4: 840–62.  <ul class="find" data-citedArticleID="1072348" data-doi="10.1128/cmr.17.4.840-862.2004"><li><a href="http://dx.doi.org/10.1128/cmr.17.4.840-862.2004" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Impact+of+16S+rRNA+gene+sequence+analysis+for+identification+of+bacteria+on+clinical+microbiology+and+infectious+diseases." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Impact+of+16S+rRNA+gene+sequence+analysis+for+identification+of+bacteria+on+clinical+microbiology+and+infectious+diseases.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0001773-Petti1" id="pone.0001773-Petti1"></a>Petti CA (2007) Detection and identification of microorganisms by gene amplification and sequencing. Clin. Infect. Dis.  44(8): 1108–14.  <ul class="find" data-citedArticleID="1072384" data-doi="10.1086/512818"><li><a href="http://dx.doi.org/10.1086/512818" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Detection+and+identification+of+microorganisms+by+gene+amplification+and+sequencing." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Detection+and+identification+of+microorganisms+by+gene+amplification+and+sequencing.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0001773-ODempsey1" id="pone.0001773-ODempsey1"></a>O’Dempsey JD, McArdle TF, Laurence BE (1993) Overlap in the clinical features of pneumonia and malaria in African children. Transactions of the Royal Society of Tropical Medicine and Hygiene.  87: 662–5.  <ul class="find" data-citedArticleID="1072378" data-doi="10.1016/0035-9203(93)90279-y"><li><a href="http://dx.doi.org/10.1016/0035-9203(93)90279-y" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Overlap+in+the+clinical+features+of+pneumonia+and+malaria+in+African+children." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Overlap+in+the+clinical+features+of+pneumonia+and+malaria+in+African+children.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0001773-Cutts1" id="pone.0001773-Cutts1"></a>Cutts FT, Zaman SM, Enwere G, Jaffar S, Levine OS, et al.  (2005) Gambian Pneumococcal Vaccine Trial Group. Efficacy of nine-valent pneumococcal conjugate vaccine against pneumonia and invasive pneumococcal disease in The Gambia: randomised, double-blind, placebo-controlled trial. Lancet.  365(9465): 1139–46.  <ul class="find" data-citedArticleID="1072354" data-doi="10.1016/s0140-6736(05)71876-6"><li><a href="http://dx.doi.org/10.1016/s0140-6736(05)71876-6" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Gambian+Pneumococcal+Vaccine+Trial+Group.+Efficacy+of+nine-valent+pneumococcal+conjugate+vaccine+against+pneumonia+and+invasive+pneumococcal+disease+in+The+Gambia%3A+randomised%2C+double-blind%2C+placebo-controlled+trial." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Gambian+Pneumococcal+Vaccine+Trial+Group.+Efficacy+of+nine-valent+pneumococcal+conjugate+vaccine+against+pneumonia+and+invasive+pneumococcal+disease+in+The+Gambia%3A+randomised%2C+double-blind%2C+placebo-controlled+trial.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0001773-Nadkarni1" id="pone.0001773-Nadkarni1"></a>Nadkarni MA, Martin FE, Jacques NA, Hunter N (2002) Determination of bacterial load by real-time PCR using a broad-range (universal) probe and primers set. Microbiology.  148(Pt 1): 257–66.  <ul class="find" data-citedArticleID="1072374"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Determination+of+bacterial+load+by+real-time+PCR+using+a+broad-range+%28universal%29+probe+and+primers+set.&amp;auth=&amp;atitle=Determination+of+bacterial+load+by+real-time+PCR+using+a+broad-range+%28universal%29+probe+and+primers+set." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Determination+of+bacterial+load+by+real-time+PCR+using+a+broad-range+%28universal%29+probe+and+primers+set." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Determination+of+bacterial+load+by+real-time+PCR+using+a+broad-range+%28universal%29+probe+and+primers+set.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0001773-Parkhill1" id="pone.0001773-Parkhill1"></a>Parkhill J, Wren BW, Mungall K, Ketley JM, Churcher C, et al.  (2000) The genome sequence of the food-borne pathogen Campylobacter jejuni reveals hypervariable sequences. Nature.  403(6770): 665–668.  <ul class="find" data-citedArticleID="1072380"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=The+genome+sequence+of+the+food-borne+pathogen+Campylobacter+jejuni+reveals+hypervariable+sequences.&amp;auth=&amp;atitle=The+genome+sequence+of+the+food-borne+pathogen+Campylobacter+jejuni+reveals+hypervariable+sequences." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+genome+sequence+of+the+food-borne+pathogen+Campylobacter+jejuni+reveals+hypervariable+sequences." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+genome+sequence+of+the+food-borne+pathogen+Campylobacter+jejuni+reveals+hypervariable+sequences.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0001773-Colles1" id="pone.0001773-Colles1"></a>Colles FM, Jones K, Harding RM, Maiden MCJ (2003) Genetic diversity of <em>Campylobacter jejuni</em> isolates from farm animals and the farm environment. Appl. Environ. Microbiol.  69: 7409–7413.  <ul class="find" data-citedArticleID="1072352" data-doi="10.1128/aem.69.12.7409-7413.2003"><li><a href="http://dx.doi.org/10.1128/aem.69.12.7409-7413.2003" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genetic+diversity+of+Campylobacter+jejuni+isolates+from+farm+animals+and+the+farm+environment." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genetic+diversity+of+Campylobacter+jejuni+isolates+from+farm+animals+and+the+farm+environment.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0001773-Blaser1" id="pone.0001773-Blaser1"></a>Blaser MJ (1995)  Campylobacter and related species, p. 1948–1956. In: Mandell GI, Bennett JE, Dolin R, editors. <em>Mandell, Douglas and Bennett's Principles and practice of infectious diseases</em> (4th ed.). New York: Churchchill Livingstone.   <ul class="find-nolinks"></ul></li><li><span class="label">12.
              </span><a name="pone.0001773-Nachamkin1" id="pone.0001773-Nachamkin1"></a>Nachamkin I, Allos BM, Ho T (1998) <em>Campylobacter</em> species and Guillain-Barré syndrome. Clin. Microbiol.  Rev.11: 555–567.  <ul class="find" data-citedArticleID="1072372"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Campylobacter+species+and+Guillain-Barr%E9+syndrome.&amp;auth=&amp;atitle=Campylobacter+species+and+Guillain-Barr%E9+syndrome." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Campylobacter+species+and+Guillain-Barr%C3%A9+syndrome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Campylobacter+species+and+Guillain-Barr%C3%A9+syndrome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0001773-Young1" id="pone.0001773-Young1"></a>Young KT, Davis LM, Dirita VJ (2007) <em>Campylobacter jejuni</em>: molecular biology and pathogenesis. Nat. Rev. Microbiol.  9: 665–79.  <ul class="find" data-citedArticleID="1072392" data-doi="10.1038/nrmicro1718"><li><a href="http://dx.doi.org/10.1038/nrmicro1718" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Campylobacter+jejuni%3A+molecular+biology+and+pathogenesis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Campylobacter+jejuni%3A+molecular+biology+and+pathogenesis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0001773-Rao1" id="pone.0001773-Rao1"></a>Rao MR, Naficy AB, Savarino SJ, Abu-Elyazeed R, Wierzba TF, et al.  (2001) Pathogenicity and convalescent excretion of <em>Campylobacter</em> in rural Egyptian children. Am. J. Epidemiol.  154: 166–173.  <ul class="find" data-citedArticleID="1072386" data-doi="10.1093/aje/154.2.166"><li><a href="http://dx.doi.org/10.1093/aje/154.2.166" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Pathogenicity+and+convalescent+excretion+of+Campylobacter+in+rural+Egyptian+children." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Pathogenicity+and+convalescent+excretion+of+Campylobacter+in+rural+Egyptian+children.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0001773-Coker1" id="pone.0001773-Coker1"></a>Coker AO, Isophi RD, Thomas BN, Amisu KO, Obi L (2002) Human campylobacteriosis in developing countries. Emerg. Infect. Dis.  8: 237–243.  <ul class="find" data-citedArticleID="1072350" data-doi="10.3201/eid0803.010233"><li><a href="http://dx.doi.org/10.3201/eid0803.010233" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Human+campylobacteriosis+in+developing+countries." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Human+campylobacteriosis+in+developing+countries.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0001773-Butzler1" id="pone.0001773-Butzler1"></a>Butzler JP (2004) <em>Campylobacter</em>, from obscurity to celebrity. Clin. Microbiol. Infect.  10: 868–876.  <ul class="find" data-citedArticleID="1072346" data-doi="10.1111/j.1469-0691.2004.00983.x"><li><a href="http://dx.doi.org/10.1111/j.1469-0691.2004.00983.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Campylobacter%2C+from+obscurity+to+celebrity." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Campylobacter%2C+from+obscurity+to+celebrity.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0001773-Miller1" id="pone.0001773-Miller1"></a>Miller WG, Parker CT, Heath S, Lastovica AJ (2007) Identification of genomic differences between <em>Campylobacter jejuni subsp. jejuni</em> and <em>C. jejuni subsp. doylei</em> at the nap locus leads to the development of a <em>C. jejuni</em> subspeciation multiplex PCR method. BMC Microbiol.  28: 7–11.  <ul class="find" data-citedArticleID="1072368" data-doi="10.1186/1471-2180-7-11"><li><a href="http://dx.doi.org/10.1186/1471-2180-7-11" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Identification+of+genomic+differences+between+Campylobacter+jejuni+subsp.+jejuni+and+C.+jejuni+subsp.+doylei+at+the+nap+locus+leads+to+the+development+of+a+C.+jejuni+subspeciation+multiplex+PCR+method." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Identification+of+genomic+differences+between+Campylobacter+jejuni+subsp.+jejuni+and+C.+jejuni+subsp.+doylei+at+the+nap+locus+leads+to+the+development+of+a+C.+jejuni+subspeciation+multiplex+PCR+method.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0001773-Wassenaar1" id="pone.0001773-Wassenaar1"></a>Wassenaar TM, Newell DG (2000) Genotyping of <em>Campylobacter</em> species. Appl. Environ. Microbiol.  66: 1–9.  <ul class="find" data-citedArticleID="1072390" data-doi="10.1128/aem.66.1.1-9.2000"><li><a href="http://dx.doi.org/10.1128/aem.66.1.1-9.2000" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genotyping+of+Campylobacter+species." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genotyping+of+Campylobacter+species.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0001773-Dingle1" id="pone.0001773-Dingle1"></a>Dingle KE, Colles FM, Wareing DRA, Ure R, Fox AJ, et al.  (2001) Multilocus sequence typing system for <em>Campylobacter jejuni</em>. J. Clin. Microbiol.  39: 14–23.  <ul class="find" data-citedArticleID="1072356" data-doi="10.1128/jcm.39.1.14-23.2001"><li><a href="http://dx.doi.org/10.1128/jcm.39.1.14-23.2001" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Multilocus+sequence+typing+system+for+Campylobacter+jejuni." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Multilocus+sequence+typing+system+for+Campylobacter+jejuni.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0001773-Dingle2" id="pone.0001773-Dingle2"></a>Dingle KE, Colles FM, Ure R, Wagenaar JA, Duim B, et al.  (2002) Molecular characterization of <em>Campylobacter jejuni</em> clones: a basis for epidemiologic investigation. Emerg. Infect. Dis.  8: 949–955.  <ul class="find" data-citedArticleID="1072358" data-doi="10.3201/eid0809.020122"><li><a href="http://dx.doi.org/10.3201/eid0809.020122" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Molecular+characterization+of+Campylobacter+jejuni+clones%3A+a+basis+for+epidemiologic+investigation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Molecular+characterization+of+Campylobacter+jejuni+clones%3A+a+basis+for+epidemiologic+investigation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0001773-Duim1" id="pone.0001773-Duim1"></a>Duim B, Godschalk PCR, Braak NVD, Dingle KE, Dijkstra JR, et al.  (2003) Molecular evidence for dissemination of unique <em>Campylobacter jejuni</em> clones in Curaçao, Netherlands Antilles. J. Clin. Microbiol.  41: 5593–5597.  <ul class="find" data-citedArticleID="1072360" data-doi="10.1128/jcm.41.12.5593-5597.2003"><li><a href="http://dx.doi.org/10.1128/jcm.41.12.5593-5597.2003" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Molecular+evidence+for+dissemination+of+unique+Campylobacter+jejuni+clones+in+Cura%C3%A7ao%2C+Netherlands+Antilles." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Molecular+evidence+for+dissemination+of+unique+Campylobacter+jejuni+clones+in+Cura%C3%A7ao%2C+Netherlands+Antilles.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0001773-Manning1" id="pone.0001773-Manning1"></a>Manning G, Dowson CG, Bagnall MC, Ahmed IH, West M, et al.  (2003) Multilocus sequence typing for comparison of veterinary and human isolates of <em>Campylobacter jejuni</em>. Appl. Environ. Microbiol.  69: 6370–6379.  <ul class="find" data-citedArticleID="1072364" data-doi="10.1128/aem.69.11.6370-6379.2003"><li><a href="http://dx.doi.org/10.1128/aem.69.11.6370-6379.2003" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Multilocus+sequence+typing+for+comparison+of+veterinary+and+human+isolates+of+Campylobacter+jejuni." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Multilocus+sequence+typing+for+comparison+of+veterinary+and+human+isolates+of+Campylobacter+jejuni.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0001773-Nikkari1" id="pone.0001773-Nikkari1"></a>Nikkari S, McLaughlin IJ, Bi W, Dodge DE, Relman DA (2001) Does blood of healthy subjects contain bacterial ribosomal DNA? J. Clin. Microbiol.  39: 1956–59.  <ul class="find" data-citedArticleID="1072376" data-doi="10.1128/jcm.39.5.1956-1959.2001"><li><a href="http://dx.doi.org/10.1128/jcm.39.5.1956-1959.2001" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Does+blood+of+healthy+subjects+contain+bacterial+ribosomal+DNA%3F" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Does+blood+of+healthy+subjects+contain+bacterial+ribosomal+DNA%3F%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0001773-McLaughlin1" id="pone.0001773-McLaughlin1"></a>McLaughlin RW, Vali H, Lau PC, Palfree RG, De Ciccio A, et al.  (2002) Are there naturally occurring pleomorphic bacteria in the blood of healthy humans? J. Clin. Microbiol.  40: 4771–75.  <ul class="find" data-citedArticleID="1072366" data-doi="10.1128/jcm.40.12.4771-4775.2002"><li><a href="http://dx.doi.org/10.1128/jcm.40.12.4771-4775.2002" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Are+there+naturally+occurring+pleomorphic+bacteria+in+the+blood+of+healthy+humans%3F" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Are+there+naturally+occurring+pleomorphic+bacteria+in+the+blood+of+healthy+humans%3F%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0001773-Mulholland1" id="pone.0001773-Mulholland1"></a>Mulholland EK, Adegbola RA (2005) Bacterial infections, a major cause of death among children in Africa. N. Engl. J. Med.  352(1): 39–47.  <ul class="find" data-citedArticleID="1072370" data-doi="10.1056/nejme048306"><li><a href="http://dx.doi.org/10.1056/nejme048306" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Bacterial+infections%2C+a+major+cause+of+death+among+children+in+Africa." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Bacterial+infections%2C+a+major+cause+of+death+among+children+in+Africa.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001773.XML" value="67468"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001773.PDF" value="240264"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001773.t001.PNG_L" value="229099"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001773.t001.PNG_M" value="152290"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001773.t001.PNG_S" value="13495"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001773.t001.TIF" value="1046276"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001773.t001.PNG_I" value="47923"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001773.g001.PNG_L" value="220661"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001773.g001.PNG_M" value="133947"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001773.g001.PNG_S" value="6855"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001773.g001.TIF" value="359752"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001773.g001.PNG_I" value="45258"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001773.g002.PNG_L" value="103044"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001773.g002.PNG_M" value="63303"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001773.g002.PNG_S" value="6291"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001773.g002.TIF" value="120498"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001773.g002.PNG_I" value="22285"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001773&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001773" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001773&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0001773&volume=&issue=&title=A Novel Campylobacter jejuni Sequence Type from a Culture-Negative Patient in The Gambia&author_name=Gerard%20A.%20J.%20Morris%2C%20Usman%20N.%20Ikumapayi%2C%20Martin%20Antonio%2C%20Stephen%20R.%20C.%20Howie%2C%20Richard%20A.%20Adegbola&start_page=1&end_page=6" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001773" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001773" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0001773" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001773&amp;t=A%20Novel%20Campylobacter%20jejuni%20Sequence%20Type%20from%20a%20Culture-Negative%20Patient%20in%20The%20Gambia" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001773&title=A%20Novel%20Campylobacter%20jejuni%20Sequence%20Type%20from%20a%20Culture-Negative%20Patient%20in%20The%20Gambia&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001773&amp;title=A%20Novel%20Campylobacter%20jejuni%20Sequence%20Type%20from%20a%20Culture-Negative%20Patient%20in%20The%20Gambia" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001773" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0001773&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'A Novel Campylobacter jejuni Sequence Type from a Culture-Negative Patient in The Gambia';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0001773';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20A%20Novel%20Campylobacter%20jejuni%20Sequence%20Type%20from%20a%20Culture-Negative%20Patient%20in%20The%20Gambia http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001773" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0001773" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">
















          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Bacterial+pathogens%22" title="Search for articles in the subject area:'Bacterial pathogens'"><div class="flagText">Bacterial pathogens</div></a>
              <div data-categoryid="42717" data-articleid="26370"
                   data-categoryname="Bacterial pathogens"
                   class="flagImage" title="Flag 'Bacterial pathogens' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Campylobacter%22" title="Search for articles in the subject area:'Campylobacter'"><div class="flagText">Campylobacter</div></a>
              <div data-categoryid="43625" data-articleid="26370"
                   data-categoryname="Campylobacter"
                   class="flagImage" title="Flag 'Campylobacter' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22DNA+sequence+analysis%22" title="Search for articles in the subject area:'DNA sequence analysis'"><div class="flagText">DNA sequence analysis</div></a>
              <div data-categoryid="47935" data-articleid="26370"
                   data-categoryname="DNA sequence analysis"
                   class="flagImage" title="Flag 'DNA sequence analysis' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22DNA+sequencing%22" title="Search for articles in the subject area:'DNA sequencing'"><div class="flagText">DNA sequencing</div></a>
              <div data-categoryid="47891" data-articleid="26370"
                   data-categoryname="DNA sequencing"
                   class="flagImage" title="Flag 'DNA sequencing' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Diagnostic+medicine%22" title="Search for articles in the subject area:'Diagnostic medicine'"><div class="flagText">Diagnostic medicine</div></a>
              <div data-categoryid="3267" data-articleid="26370"
                   data-categoryname="Diagnostic medicine"
                   class="flagImage" title="Flag 'Diagnostic medicine' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Polymerase+chain+reaction%22" title="Search for articles in the subject area:'Polymerase chain reaction'"><div class="flagText">Polymerase chain reaction</div></a>
              <div data-categoryid="26251" data-articleid="26370"
                   data-categoryname="Polymerase chain reaction"
                   class="flagImage" title="Flag 'Polymerase chain reaction' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Ribosomal+RNA%22" title="Search for articles in the subject area:'Ribosomal RNA'"><div class="flagText">Ribosomal RNA</div></a>
              <div data-categoryid="21371" data-articleid="26370"
                   data-categoryname="Ribosomal RNA"
                   class="flagImage" title="Flag 'Ribosomal RNA' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Sequence+databases%22" title="Search for articles in the subject area:'Sequence databases'"><div class="flagText">Sequence databases</div></a>
              <div data-categoryid="42787" data-articleid="26370"
                   data-categoryname="Sequence databases"
                   class="flagImage" title="Flag 'Sequence databases' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=6112'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=4892'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=486&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>


</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
