

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: A Family of Human MicroRNA Genes from Miniature Inverted-Repeat Transposable Elements</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0000203"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0000203" />

    <meta name="citation_title" content="A Family of Human MicroRNA Genes from Miniature Inverted-Repeat Transposable Elements"/>
    <meta itemprop="name" content="A Family of Human MicroRNA Genes from Miniature Inverted-Repeat Transposable Elements"/>

      <meta name="citation_author" content="Jittima Piriyapongsa"/>
            <meta name="citation_author_institution" content="School of Biology, Georgia Institute of Technology, Atlanta, Georgia, United States of America"/>
      <meta name="citation_author" content="I. King Jordan"/>
            <meta name="citation_author_institution" content="School of Biology, Georgia Institute of Technology, Atlanta, Georgia, United States of America"/>

    <meta name="citation_date" content="2007/2/14"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0000203.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e203"/>
    <meta name="citation_issue" content="2"/>
    <meta name="citation_volume" content="2"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Non-coding RNA.; citation_author=JS Mattick; citation_author=IV Makunin; citation_journal_title=Hum Mol Genet; citation_volume=15 Spec No 1; citation_number=1; citation_pages=R17-29; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=The functions of animal microRNAs.; citation_author=V Ambros; citation_journal_title=Nature; citation_volume=431; citation_number=2; citation_pages=350-355; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=MicroRNAs: genomics, biogenesis, mechanism, and function.; citation_author=DP Bartel; citation_journal_title=Cell; citation_volume=116; citation_number=3; citation_pages=281-297; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=The C. elegans heterochronic gene lin-4 encodes small RNAs with antisense complementarity to lin-14.; citation_author=RC Lee; citation_author=RL Feinbaum; citation_author=V Ambros; citation_journal_title=Cell; citation_volume=75; citation_number=4; citation_pages=843-854; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=Identification of hundreds of conserved and nonconserved human microRNAs.; citation_author=I Bentwich; citation_author=A Avniel; citation_author=Y Karov; citation_author=R Aharonov; citation_author=S Gilad; citation_journal_title=Nat Genet; citation_volume=37; citation_number=5; citation_pages=766-770; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=MicroRNAs modulate hematopoietic lineage differentiation.; citation_author=CZ Chen; citation_author=L Li; citation_author=HF Lodish; citation_author=DP Bartel; citation_journal_title=Science; citation_volume=303; citation_number=6; citation_pages=83-86; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Identification of novel genes coding for small expressed RNAs.; citation_author=M Lagos-Quintana; citation_author=R Rauhut; citation_author=W Lendeckel; citation_author=T Tuschl; citation_journal_title=Science; citation_volume=294; citation_number=7; citation_pages=853-858; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=An abundant class of tiny RNAs with probable regulatory roles in Caenorhabditis elegans.; citation_author=NC Lau; citation_author=LP Lim; citation_author=EG Weinstein; citation_author=DP Bartel; citation_journal_title=Science; citation_volume=294; citation_number=8; citation_pages=858-862; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=An extensive class of small RNAs in Caenorhabditis elegans.; citation_author=RC Lee; citation_author=V Ambros; citation_journal_title=Science; citation_volume=294; citation_number=9; citation_pages=862-864; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=The colorectal microRNAome.; citation_author=JM Cummins; citation_author=Y He; citation_author=RJ Leary; citation_author=R Pagliarini; citation_author=LA Diaz; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=103; citation_number=10; citation_pages=3687-3692; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=miRBase: microRNA sequences, targets and gene nomenclature.; citation_author=S Griffiths-Jones; citation_author=RJ Grocock; citation_author=S van Dongen; citation_author=A Bateman; citation_author=AJ Enright; citation_journal_title=Nucleic Acids Res; citation_volume=34; citation_number=11; citation_pages=D140-144; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=MicroRNA targets in Drosophila.; citation_author=AJ Enright; citation_author=B John; citation_author=U Gaul; citation_author=T Tuschl; citation_author=C Sander; citation_journal_title=Genome Biol; citation_volume=5; citation_number=12; citation_pages=R1; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=A uniform system for microRNA annotation.; citation_author=V Ambros; citation_author=B Bartel; citation_author=DP Bartel; citation_author=CB Burge; citation_author=JC Carrington; citation_journal_title=Rna; citation_volume=9; citation_number=13; citation_pages=277-279; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=A microRNA in a multiple-turnover RNAi enzyme complex.; citation_author=G Hutvagner; citation_author=PD Zamore; citation_journal_title=Science; citation_volume=297; citation_number=14; citation_pages=2056-2060; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Endogenous and silencing-associated small RNAs in plants.; citation_author=C Llave; citation_author=KD Kasschau; citation_author=MA Rector; citation_author=JC Carrington; citation_journal_title=Plant Cell; citation_volume=14; citation_number=15; citation_pages=1605-1619; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=MicroRNA-directed cleavage of HOXB8 mRNA.; citation_author=S Yekta; citation_author=IH Shih; citation_author=DP Bartel; citation_journal_title=Science; citation_volume=304; citation_number=16; citation_pages=594-596; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=MicroRNAs and small interfering RNAs can inhibit mRNA expression by similar mechanisms.; citation_author=Y Zeng; citation_author=R Yi; citation_author=BR Cullen; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=100; citation_number=17; citation_pages=9779-9784; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Transgene silencing by the host genome defense: implications for the evolution of epigenetic control mechanisms in plants and vertebrates.; citation_author=MA Matzke; citation_author=MF Mette; citation_author=AJ Matzke; citation_journal_title=Plant Mol Biol; citation_volume=43; citation_number=18; citation_pages=401-415; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Heritable transposon silencing initiated by a naturally occurring transposon inverted duplication.; citation_author=RK Slotkin; citation_author=M Freeling; citation_author=D Lisch; citation_journal_title=Nat Genet; citation_volume=37; citation_number=19; citation_pages=641-644; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=RNAi protects the Caenorhabditis elegans germline against transposition.; citation_author=NL Vastenhouw; citation_author=RH Plasterk; citation_journal_title=Trends Genet; citation_volume=20; citation_number=20; citation_pages=314-319; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Mammalian microRNAs derived from genomic repeats.; citation_author=NR Smalheiser; citation_author=VI Torvik; citation_journal_title=Trends Genet; citation_volume=21; citation_number=21; citation_pages=322-326; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=The UCSC Genome Browser Database.; citation_author=D Karolchik; citation_author=R Baertsch; citation_author=M Diekhans; citation_author=TS Furey; citation_author=A Hinrichs; citation_journal_title=Nucleic Acids Res; citation_volume=31; citation_number=22; citation_pages=51-54; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=The UCSC Table Browser data retrieval tool.; citation_author=D Karolchik; citation_author=AS Hinrichs; citation_author=TS Furey; citation_author=KM Roskin; citation_author=CW Sugnet; citation_journal_title=Nucleic Acids Res; citation_volume=32; citation_number=23; citation_pages=D493-496; citation_date=2004; " />
      <meta name="citation_reference" content="citation_author=AFA Smit; citation_author=R Hubley; citation_author=P Green; citation_number=24; citation_date=1996–2004; " />
      <meta name="citation_reference" content="citation_title=Evolution's cauldron: duplication, deletion, and rearrangement in the mouse and human genomes.; citation_author=WJ Kent; citation_author=R Baertsch; citation_author=A Hinrichs; citation_author=W Miller; citation_author=D Haussler; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=100; citation_number=25; citation_pages=11484-11489; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=BLAT–the BLAST-like alignment tool.; citation_author=WJ Kent; citation_journal_title=Genome Res; citation_volume=12; citation_number=26; citation_pages=656-664; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=CLUSTAL W: improving the sensitivity of progressive multiple sequence alignment through sequence weighting, position-specific gap penalties and weight matrix choice.; citation_author=JD Thompson; citation_author=DG Higgins; citation_author=TJ Gibson; citation_journal_title=Nucleic Acids Res; citation_volume=22; citation_number=27; citation_pages=4673-4680; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=Gapped BLAST and PSI-BLAST: a new generation of protein database search programs.; citation_author=SF Altschul; citation_author=TL Madden; citation_author=AA Schaffer; citation_author=J Zhang; citation_author=Z Zhang; citation_journal_title=Nucleic Acids Res; citation_volume=25; citation_number=28; citation_pages=3389-3402; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=dbEST–database for “expressed sequence tags”.; citation_author=MS Boguski; citation_author=TM Lowe; citation_author=CM Tolstoshev; citation_journal_title=Nat Genet; citation_volume=4; citation_number=29; citation_pages=332-333; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=Transcriptional maps of 10 human chromosomes at 5-nucleotide resolution.; citation_author=J Cheng; citation_author=P Kapranov; citation_author=J Drenkow; citation_author=S Dike; citation_author=S Brubaker; citation_journal_title=Science; citation_volume=308; citation_number=30; citation_pages=1149-1154; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Mfold web server for nucleic acid folding and hybridization prediction.; citation_author=M Zuker; citation_journal_title=Nucleic Acids Res; citation_volume=31; citation_number=31; citation_pages=3406-3415; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Aligning multiple genomic sequences with the threaded blockset aligner.; citation_author=M Blanchette; citation_author=WJ Kent; citation_author=C Riemer; citation_author=L Elnitski; citation_author=AF Smit; citation_journal_title=Genome Res; citation_volume=14; citation_number=32; citation_pages=708-715; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Evolutionarily conserved elements in vertebrate, insect, worm, and yeast genomes.; citation_author=A Siepel; citation_author=G Bejerano; citation_author=JS Pedersen; citation_author=AS Hinrichs; citation_author=M Hou; citation_journal_title=Genome Res; citation_volume=15; citation_number=33; citation_pages=1034-1050; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=A gene atlas of the mouse and human protein-encoding transcriptomes.; citation_author=AI Su; citation_author=T Wiltshire; citation_author=S Batalov; citation_author=H Lapp; citation_author=KA Ching; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=101; citation_number=34; citation_pages=6062-6067; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Genesis: cluster analysis of microarray data.; citation_author=A Sturn; citation_author=J Quackenbush; citation_author=Z Trajanoski; citation_journal_title=Bioinformatics; citation_volume=18; citation_number=35; citation_pages=207-208; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=GOTree Machine (GOTM): a web-based platform for interpreting sets of interesting genes using Gene Ontology hierarchies.; citation_author=B Zhang; citation_author=D Schmoyer; citation_author=S Kirov; citation_author=J Snoddy; citation_journal_title=BMC Bioinformatics; citation_volume=5; citation_number=36; citation_pages=16; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Gene ontology: tool for the unification of biology. The Gene Ontology Consortium.; citation_author=M Ashburner; citation_author=CA Ball; citation_author=JA Blake; citation_author=D Botstein; citation_author=H Butler; citation_journal_title=Nat Genet; citation_volume=25; citation_number=37; citation_pages=25-29; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Identification in the human genome of mobile elements spread by DNA-mediated transposition.; citation_author=GT Morgan; citation_journal_title=J Mol Biol; citation_volume=254; citation_number=38; citation_pages=1-5; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Mariner transposons in humans.; citation_author=T Oosumi; citation_author=WR Belknap; citation_author=B Garlick; citation_journal_title=Nature; citation_volume=378; citation_number=39; citation_pages=672; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Tiggers and DNA transposon fossils in the human genome.; citation_author=AF Smit; citation_author=AD Riggs; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=93; citation_number=40; citation_pages=1443-1448; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Molecular evolution of an ancient mariner transposon, Hsmar1, in the human genome.; citation_author=HM Robertson; citation_author=KL Zumpano; citation_journal_title=Gene; citation_volume=205; citation_number=41; citation_pages=203-217; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Probing microRNAs with microarrays: tissue specificity and functional inference.; citation_author=T Babak; citation_author=W Zhang; citation_author=Q Morris; citation_author=BJ Blencowe; citation_author=TR Hughes; citation_journal_title=Rna; citation_volume=10; citation_number=42; citation_pages=1813-1819; citation_date=2004; " />
      <meta name="citation_reference" content="citation_author=JC Huang; citation_author=QD Morris; citation_author=BJ Frey; citation_number=43; citation_pages=114-129; citation_date=2006; citation_publisher=Springer-Verlag; " />
      <meta name="citation_reference" content="citation_title=Expression profiling by microarrays in colorectal cancer (Review).; citation_author=W Shih; citation_author=R Chetty; citation_author=MS Tsao; citation_journal_title=Oncol Rep; citation_volume=13; citation_number=44; citation_pages=517-524; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Initial sequencing and analysis of the human genome.; citation_author=ES Lander; citation_author=LM Linton; citation_author=B Birren; citation_author=C Nusbaum; citation_author=MC Zody; citation_journal_title=Nature; citation_volume=409; citation_number=45; citation_pages=860-921; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Miniature inverted-repeat transposable elements and their relationship to established DNA transposons.; citation_author=C Feschotte; citation_author=X Zhang; citation_author=SR Wessler; citation_number=46; citation_date=2002; citation_publisher=ASM Press; " />
      <meta name="citation_reference" content="citation_title=Tourist: a large family of small inverted repeat elements frequently associated with maize genes.; citation_author=TE Bureau; citation_author=SR Wessler; citation_journal_title=Plant Cell; citation_volume=4; citation_number=47; citation_pages=1283-1294; citation_date=1992; " />
      <meta name="citation_reference" content="citation_title=Using rice to understand the origin and amplification of miniature inverted repeat transposable elements (MITEs).; citation_author=N Jiang; citation_author=C Feschotte; citation_author=X Zhang; citation_author=SR Wessler; citation_journal_title=Curr Opin Plant Biol; citation_volume=7; citation_number=48; citation_pages=115-119; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Rice transposable elements: a survey of 73,000 sequence-tagged-connectors.; citation_author=L Mao; citation_author=TC Wood; citation_author=Y Yu; citation_author=MA Budiman; citation_author=J Tomkins; citation_journal_title=Genome Res; citation_volume=10; citation_number=49; citation_pages=982-990; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Recent, extensive, and preferential insertion of members of the miniature inverted-repeat transposable element family Heartbreaker into genic regions of maize.; citation_author=Q Zhang; citation_author=J Arbuckle; citation_author=SR Wessler; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=97; citation_number=50; citation_pages=1160-1165; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Planting the seeds of a new paradigm.; citation_author=MA Matzke; citation_author=AJ Matzke; citation_journal_title=PLoS Biol; citation_volume=2; citation_number=51; citation_pages=E133; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Cytosine methylation and the ecology of intragenomic parasites.; citation_author=JA Yoder; citation_author=CP Walsh; citation_author=TH Bestor; citation_journal_title=Trends Genet; citation_volume=13; citation_number=52; citation_pages=335-340; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Host defenses to transposable elements and the evolution of genomic imprinting.; citation_author=JF McDonald; citation_author=MA Matzke; citation_author=AJ Matzke; citation_journal_title=Cytogenet Genome Res; citation_volume=110; citation_number=53; citation_pages=242-249; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Role of transposable elements in heterochromatin and epigenetic control.; citation_author=Z Lippman; citation_author=AV Gendrel; citation_author=M Black; citation_author=MW Vaughn; citation_author=N Dedhia; citation_journal_title=Nature; citation_volume=430; citation_number=54; citation_pages=471-476; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=The small RNA profile during Drosophila melanogaster development.; citation_author=AA Aravin; citation_author=M Lagos-Quintana; citation_author=A Yalcin; citation_author=M Zavolan; citation_author=D Marks; citation_journal_title=Dev Cell; citation_volume=5; citation_number=55; citation_pages=337-350; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Two classes of short interfering RNA in RNA silencing.; citation_author=A Hamilton; citation_author=O Voinnet; citation_author=L Chappell; citation_author=D Baulcombe; citation_journal_title=Embo J; citation_volume=21; citation_number=56; citation_pages=4671-4679; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Distinct mechanisms determine transposon inheritance and methylation via small interfering RNA and histone modification.; citation_author=Z Lippman; citation_author=B May; citation_author=C Yordan; citation_author=T Singer; citation_author=R Martienssen; citation_journal_title=PLoS Biol; citation_volume=1; citation_number=57; citation_pages=E67; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=ARGONAUTE4 control of locus-specific siRNA accumulation and DNA and histone methylation.; citation_author=D Zilberman; citation_author=X Cao; citation_author=SE Jacobsen; citation_journal_title=Science; citation_volume=299; citation_number=58; citation_pages=716-719; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Evolution of microRNA genes by inverted duplication of target gene sequences in Arabidopsis thaliana.; citation_author=E Allen; citation_author=Z Xie; citation_author=AM Gustafson; citation_author=GH Sung; citation_author=JW Spatafora; citation_journal_title=Nat Genet; citation_volume=36; citation_number=59; citation_pages=1282-1290; citation_date=2004; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000203" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="A Family of Human MicroRNA Genes from Miniature Inverted-Repeat Transposable Elements"/>
    <meta name="twitter:description" content="While hundreds of novel microRNA (miRNA) genes have been discovered in the last few years alone, the origin and evolution of these non-coding regulatory sequences remain largely obscure. In this report, we demonstrate that members of a recently discovered family of human miRNA genes, hsa-mir-548, are derived from Made1 transposable elements. Made1 elements are short miniature inverted-repeat transposable elements (MITEs), which consist of two 37 base pair (bp) terminal inverted repeats that flank 6 bp of internal sequence. Thus, Made1 elements are nearly perfect palindromes, and when expressed as RNA they form highly stable hairpin loops. Apparently, these Made1-related structures are recognized by the RNA interference enzymatic machinery and processed to form 22 bp mature miRNA sequences. Consistent with their origin from MITEs, hsa-mir-548 genes are primate-specific and have many potential paralogs in the human genome. There are more than 3,500 putative hsa-mir-548 target genes; analysis of their expression profiles and functional affinities suggests cancer-related regulatory roles for hsa-mir-548. Taken together, the characteristics of Made1 elements, and MITEs in general, point to a specific mechanism for the generation of numerous small regulatory RNAs and target sites throughout the genome. The evolutionary lineage-specific nature of MITEs could also provide for the generation of novel regulatory phenotypes related to species diversification. Finally, we propose that MITEs may represent an evolutionary link between siRNAs and miRNAs."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0000203.g008"/>

  <meta property="og:title" content="A Family of Human MicroRNA Genes from Miniature Inverted-Repeat Transposable Elements" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000203" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=2525'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=7155'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=1485&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0000203">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000203" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000203&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    A Family of Human MicroRNA Genes from Miniature Inverted-Repeat Transposable Elements
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Jittima Piriyapongsa, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  School of Biology, Georgia Institute of Technology, Atlanta, Georgia, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            I. King Jordan
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:king.jordan@biology.gatech.edu">king.jordan@biology.gatech.edu</a>
            </p>

                <p>Affiliation:
                  School of Biology, Georgia Institute of Technology, Atlanta, Georgia, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: February 14, 2007</li>
    <li>DOI: 10.1371/journal.pone.0000203</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0000203" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0000203" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000203" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0000203" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0000203-t001" data-doi="info:doi/10.1371/journal.pone.0000203" data-uri="info:doi/10.1371/journal.pone.0000203.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000203.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000203-g001" data-doi="info:doi/10.1371/journal.pone.0000203" data-uri="info:doi/10.1371/journal.pone.0000203.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000203.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000203-g002" data-doi="info:doi/10.1371/journal.pone.0000203" data-uri="info:doi/10.1371/journal.pone.0000203.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000203.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000203-g003" data-doi="info:doi/10.1371/journal.pone.0000203" data-uri="info:doi/10.1371/journal.pone.0000203.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000203.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000203-g004" data-doi="info:doi/10.1371/journal.pone.0000203" data-uri="info:doi/10.1371/journal.pone.0000203.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000203.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000203-g005" data-doi="info:doi/10.1371/journal.pone.0000203" data-uri="info:doi/10.1371/journal.pone.0000203.g005" title="Figure 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000203.g005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000203-g006" data-doi="info:doi/10.1371/journal.pone.0000203" data-uri="info:doi/10.1371/journal.pone.0000203.g006" title="Figure 6">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000203.g006&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000203-g007" data-doi="info:doi/10.1371/journal.pone.0000203" data-uri="info:doi/10.1371/journal.pone.0000203.g007" title="Figure 7">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000203.g007&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000203-g008" data-doi="info:doi/10.1371/journal.pone.0000203" data-uri="info:doi/10.1371/journal.pone.0000203.g008" title="Figure 8">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000203.g008&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000203">Reader Comments (1)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0000203" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1"></a><p>While hundreds of novel microRNA (miRNA) genes have been discovered in the last few years alone, the origin and evolution of these non-coding regulatory sequences remain largely obscure. In this report, we demonstrate that members of a recently discovered family of human miRNA genes, hsa-mir-548, are derived from Made1 transposable elements. Made1 elements are short miniature inverted-repeat transposable elements (MITEs), which consist of two 37 base pair (bp) terminal inverted repeats that flank 6 bp of internal sequence. Thus, Made1 elements are nearly perfect palindromes, and when expressed as RNA they form highly stable hairpin loops. Apparently, these Made1-related structures are recognized by the RNA interference enzymatic machinery and processed to form 22 bp mature miRNA sequences. Consistent with their origin from MITEs, hsa-mir-548 genes are primate-specific and have many potential paralogs in the human genome. There are more than 3,500 putative hsa-mir-548 target genes; analysis of their expression profiles and functional affinities suggests cancer-related regulatory roles for hsa-mir-548. Taken together, the characteristics of Made1 elements, and MITEs in general, point to a specific mechanism for the generation of numerous small regulatory RNAs and target sites throughout the genome. The evolutionary lineage-specific nature of MITEs could also provide for the generation of novel regulatory phenotypes related to species diversification. Finally, we propose that MITEs may represent an evolutionary link between siRNAs and miRNAs.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>Piriyapongsa J, Jordan IK (2007) A Family of Human MicroRNA Genes from Miniature Inverted-Repeat Transposable Elements. PLoS ONE 2(2):
          e203.
            doi:10.1371/journal.pone.0000203</p><p><strong>Academic Editor: </strong>Alan Christoffels, Temasek Life Sciences Laboratory, Singapore</p><p><strong>Received:</strong> October 26, 2006; <strong>Accepted:</strong> January 21, 2007; <strong>Published:</strong> February 14, 2007</p><p><strong>Copyright:</strong> © 2007 Piriyapongsa, Jordan. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>Jittima Piriyapongsa is supported by the Ministry of Science and Technology of Thailand. I. King Jordan is supported by the School of Biology at the Georgia Institute of Technology. Neither funding agency played any role in the design and conduct of the study, in the collection, analysis, and interpretation of the data, and in the preparation, review, or approval of the manuscript.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>Numerous human genome transcripts lack protein coding capacity, and these non-coding RNA (ncRNAs) perform a variety of structural, enzymatic and regulatory functions <a href="#pone.0000203-Mattick1">[1]</a>. MicroRNAs (miRNAs) are a class of short ~22nt ncRNA that function as post-transcriptional regulators of gene expression <a href="#pone.0000203-Ambros1">[2]</a>. Mature miRNAs are processed from longer RNA sequences that form local stem-loop (hairpin) structures <a href="#pone.0000203-Bartel1">[3]</a>. The first step of the miRNA biogenesis pathway occurs in the nucleus where the RNase III enzyme Drosha cleaves both strands of the so-called pri-miRNA at the base of the stem. This yields a ~70–90 bp pre-miRNA hairpin that is exported to the cytoplasm where it is further processed by Dicer, another RNase III endonuclease. Dicer recognizes the double stranded portion of the RNA close the base of the pre-miRNA stem and cleaves both strands of the duplex in two places. This reaction cuts off the loop portion of the molecule as well as the terminal part of the stem leaving a short duplex that consists of the mature miRNA and a complementary miRNA* sequence that is rapidly degraded. Once liberated in this way, the mature miRNA sequence binds to partially complementary target sites in the 3′ untranslated regions (UTRs) of messenger RNAs (mRNAs) and regulates expression through a process of mRNA degradation and/or translational repression <a href="#pone.0000203-Bartel1">[3]</a>.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>miRNAs were only recently discovered <a href="#pone.0000203-Lee1">[4]</a>, and details regarding their origin and evolution have yet to be fully worked out. Since their original discovery, miRNAs have been detected in all metazoa surveyed for their presence <a href="#pone.0000203-Bartel1">[3]</a>. However, the full-extent of miRNA genes in any particular genome is unknown, and a number of studies aimed at the detection of novel miRNA genes have been conducted to address this issue. Bioinformatic miRNA discovery relies primarily on the sequence conservation of miRNA genes and secondary structure of the pre-miRNAs <a href="#pone.0000203-Bentwich1">[5]</a>, while experimental efforts consist of forward <a href="#pone.0000203-Lee1">[4]</a> and reverse <a href="#pone.0000203-Chen1">[6]</a> genetic studies as well as efforts to clone short mature miRNA sequences <a href="#pone.0000203-LagosQuintana1">[7]</a>–<a href="#pone.0000203-Lee2">[9]</a>. Cloning mature miRNA sequences is technically challenging given their small size and associated instability. Thus, direct miRNA cloning is not well suited to large scale discovery efforts and may have already reached the point of diminishing returns <a href="#pone.0000203-LagosQuintana1">[7]</a>. A recently published report described a novel high-throughput miRNA cloning technique aimed at increasing the efficiency of miRNA discovery <a href="#pone.0000203-Cummins1">[10]</a>. This technique is based on the serial analysis of gene expression (SAGE) and takes advantage of well established protocols tailored to small RNA sequences. Application of this SAGE-based approach to human transcripts confirmed the presence of numerous miRNA genes that had been detected previously through computational and/or experimental surveys and also yielded more than 100 novel miRNA sequences <a href="#pone.0000203-Cummins1">[10]</a>. Including these new data, version 8.2 of miRBase, the online microRNA database <a href="#pone.0000203-GriffithsJones1">[11]</a>, reports 462 human miRNA genes. The importance of miRNAs for human gene regulation is underscored by target site predictions <a href="#pone.0000203-Enright1">[12]</a>, which reveal that these human miRNAs have the potential to regulate thousands of human genes.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>miRNAs are closely related to another class of ncRNA, known as small interfering RNAs (siRNA), in terms of both biogenesis and regulatory function <a href="#pone.0000203-Bartel1">[3]</a>, <a href="#pone.0000203-Ambros2">[13]</a>. The mature biologically active forms of siRNA and miRNA are both processed from double stranded RNA (dsRNA) by Dicer. However, siRNAs are generated from long dsRNA precursors, which can be either endogenous or exogenous transcripts, whereas mature miRNAs are processed from shorter endogenous transcripts that form local hairpin structures. Numerous siRNA molecules are processed from both strands of the long dsRNA precursor, whereas a single mature miRNA sequence is generated from only one strand of the pre-miRNA hairpin. While miRNAs can act through translational repression of their targets, they may also cause mRNA degradation of their target genes in the same way that siRNAs do <a href="#pone.0000203-Hutvagner1">[14]</a>–<a href="#pone.0000203-Zeng1">[17]</a>.</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p>One previously recognized distinction between these two classes of regulatory RNA is the fact that miRNAs are generally found in unique genomic loci, such as intergenic regions <a href="#pone.0000203-Bartel1">[3]</a>, while siRNAs originate from within already characterized sequences such as genes and transposable elements (TEs) <a href="#pone.0000203-Matzke1">[18]</a>–<a href="#pone.0000203-Vastenhouw1">[20]</a>. However, a recent report indicated that a number of mammalian miRNAs, including six human miRNAs, are in fact derived from TEs <a href="#pone.0000203-Smalheiser1">[21]</a>. The abundance and repetitive nature of TE sequences could provide a natural mechanism for the generation of multiple miRNA genes, along with homologous target sites, dispersed throughout the human genome. TEs may also provide an evolutionary connection between siRNAs and miRNAs. In light of these possibilities, we sought to investigate the relationship between human miRNAs and TEs by evaluating whether there exist families of related (paralogous) miRNA genes that are derived from TE sequences. We compared the genomic locations of experimentally characterized human miRNA genes to the annotated human TE sequences and discovered a set of closely related miRNA genes derived from a family of miniature inverted repeat transposable elements (MITEs). The palindromic sequence structure of MITEs, considered together with their insertion into transcriptionally active regions of the human genome, suggests a specific mechanism by which these kinds of elements could give rise to emergent mature miRNAs.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Methods"></a><h3>Methods</h3>
<h4>TE-miRNA sequence analysis</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>The UCSC Genome and Table Browsers <a href="#pone.0000203-Karolchik1">[22]</a>, <a href="#pone.0000203-Karolchik2">[23]</a> were used to analyze the March 2006 human genome reference sequence (<a href="http://www.genome.ucsc.edu/cgi-bin/hgGateway?">http://www.genome.ucsc.edu/cgi-bin/hgGat​eway?</a>org = Human&amp;db = hg18). This sequence is referred to as the hg18 assembly on the UCSC Genome Bioinformatics website and corresponds to the human genome build 36.1 assembled by the National Center for Biotechnology Information (NCBI). The Table Browser was used to search genome-wide for co-located TE and miRNA gene sequences, and the Genome Browser was used to visualize the results on a case-by-case basis. The genome locations and identities of human TE sequences were taken from annotation generated by the RepeatMasker program (<a href="http://www.repeatmasker.org">http://www.repeatmasker.org</a>) <a href="#pone.0000203-Smit1">[24]</a>. The genome locations and identities of experimentally characterized human miRNA gene sequences were taken from release 8.2 of the miRBase sequence database (<a href="http://microrna.sanger.ac.uk/sequences/">http://microrna.sanger.ac.uk/sequences/</a>) <a href="#pone.0000203-GriffithsJones1">[11]</a>. Evolutionary conservation between human Made1-derived miRNA gene sequences and six mammalian genomes – chimp, rhesus, mouse, rat, dog and cow – was assessed based on the Alignment Net track of the UCSC Genome browser, which shows the best pairwise between-genome alignments corresponding to orthologous regions <a href="#pone.0000203-Kent1">[25]</a>.</p>
<a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2"></a><p>The sequences of Made1-derived miRNAs were compared to the human genome sequence using the BLAT program <a href="#pone.0000203-Kent2">[26]</a>. Homologous genomic sequences were counted as statistically significant hits that matched ≥80% of the length of the query miRNA sequence and were confined to a local genomic region no longer than 120% of the query length (<em>i.e.</em> long genomic insertions were not counted). Made1 and hsa-mir-548 sequences were aligned to each other using the program ClustalW <a href="#pone.0000203-Thompson1">[27]</a>. NCBI's BLASTN program (<a href="http://www.ncbi.nlm.nih.gov/BLAST/">http://www.ncbi.nlm.nih.gov/BLAST/</a>) <a href="#pone.0000203-Altschul1">[28]</a> was used to search the Expressed Sequence Tags Database (dbEST) <a href="#pone.0000203-Boguski1">[29]</a> for expressed human MITE sequences. Human genomic expression data from Affymetrix tiling GeneChips <a href="#pone.0000203-Cheng1">[30]</a>, represented in the UCSC Genome Browser, were evaluated in order to identify transcriptionally active regions of the human genome. RNA sequences were folded using the Mfold <a href="#pone.0000203-Zuker1">[31]</a> web server (<a href="http://www.bioinfo.rpi.edu/applications/mfold/rna/form1.cgi">http://www.bioinfo.rpi.edu/applications/​mfold/rna/form1.cgi</a>).</p>


<h4>Regulatory analysis</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>Putative miRNA target sites were taken from the miRBase Targets website (<a href="http://microrna.sanger.ac.uk/targets/v3/">http://microrna.sanger.ac.uk/targets/v3/</a>), which uses a modified implementation of the miRanda algorithm <a href="#pone.0000203-Enright1">[12]</a>. 3′ UTRs of Ensembl genes were also searched for Made1 derived target sites. In this case, the same approach used by the current miRBase implementation of miRanda for annotating 3′ UTRs was employed. Specifically, if there is no hexamer of ‘A’ residues in the last 30 bp of the 3′ UTR, the sequence is extended 2,000 bp. The random expectation for the number of target genes identified by both methods was calculated by taking their joint probability multiplied by the total number of human genes (n = 23,269 from Ensembl version 41). The joint probability was calculated by multiplying the relative human genome frequencies of each target set. The difference between the expected and observed number of target genes identified using both methods was calculated using the binomial distribution.</p>
<a id="article1.body1.sec2.sec2.p2" name="article1.body1.sec2.sec2.p2"></a><p>Comparative genomic sequence data from the UCSC genome browser were used to analyze the relative evolutionary conservation levels for predicted hsa-mir-548 target sites. Position-specific conservation scores were derived from multiple whole genome sequence alignments between the human and 16 other vertebrate genomes <a href="#pone.0000203-Kent1">[25]</a>, <a href="#pone.0000203-Blanchette1">[32]</a>. The scores correspond to the posterior probability that a human genome site is conserved as determined by the phastCons program <a href="#pone.0000203-Siepel1">[33]</a>, and position-specific scores were averaged across target sites.</p>
<a id="article1.body1.sec2.sec2.p3" name="article1.body1.sec2.sec2.p3"></a><p>Human gene expression patterns across 79 tissues were taken from the Novartis Research Foundation's Symatlas <a href="#pone.0000203-Su1">[34]</a>. Relative expression profiles for genes with hsa-mir-548 target sites were computed for each gene by dividing the gene's tissue-specific expression (signal intensity) values by the gene's median expression value over all 79 tissues and then log<sub>2</sub> normalizing the resulting ratios. The program Genesis <a href="#pone.0000203-Sturn1">[35]</a> was used to visualize the relative expression profiles, to group related expression profiles with k-means clustering and to group tissues with hierarchical clustering.</p>
<a id="article1.body1.sec2.sec2.p4" name="article1.body1.sec2.sec2.p4"></a><p>Clusters of coexpressed genes were analyzed with the program GOTree Machine (GOTM) <a href="#pone.0000203-Zhang1">[36]</a> to look for over-represented Gene Ontology <a href="#pone.0000203-Ashburner1">[37]</a> functional annotations. To do this, genes in each cluster were annotated with their biological process GO terms. The frequencies of these terms were then compared to their expected frequencies based on their occurrences in the human genome, and statistically over-represented terms were identified using the hypergeometric test. Statistically over-represented terms were then mapped to the GO directed acyclic graph.</p>

</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Results and Discussion"></a><h3>Results and Discussion</h3>
<h4>A TE-derived miRNA gene family</h4>
<a id="article1.body1.sec3.sec1.p1" name="article1.body1.sec3.sec1.p1"></a><p>When we compared the genomic locations of experimentally characterized human miRNA sequences stored in miRBase <a href="#pone.0000203-GriffithsJones1">[11]</a> to the locations of human TEs characterized by the program RepeatMasker <a href="#pone.0000203-Smit1">[24]</a>, we found that seven closely related miRNA genes (hsa-mir-548) were co-located with dispersed members of a single family of TEs known as Made1 (<a href="#pone-0000203-t001">Table 1</a>). These hsa-mir-548 miRNA genes were recently characterized by mapping mature cloned miRNA sequences to the human genome sequence <a href="#pone.0000203-Cummins1">[10]</a>. The hsa-mir-548 mature miRNAs meet both the expression and biogenesis criteria that were articulated to ensure the accurate identification of miRNAs and the distinction between miRNAs and siRNAs <a href="#pone.0000203-Ambros2">[13]</a>. In particular, the mature hsa-mir-548 miRNAs are all 22nt in length, they were identified from a cDNA library made of size fractionated RNA and they map precisely to genomic regions that are predicted to form local hairpin structures.</p>
<div class="figure" id="pone-0000203-t001"><div class="img"><a name="pone-0000203-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000203.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000203" data-uri="info:doi/10.1371/journal.pone.0000203.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000203.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000203.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000203.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000203.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000203.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000203.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000203.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000203.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000203.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>Made1 derived miRNA genes in the human genome</span></strong></p><span>doi:10.1371/journal.pone.0000203.t001</span></div><a id="article1.body1.sec3.sec1.p2" name="article1.body1.sec3.sec1.p2"></a><p>Inspection of the multiple sequence alignment of a full length Made1 sequence with all seven hsa-mir-548 miRNAs provides clear evidence that the miRNAs are in fact derived from the Made1 elements (<a href="#pone-0000203-g001">Figure 1</a>). Individual hsa-mir-548 sequences were queried against the human genome sequence to search for duplicates. Each hsa-mir-548 gene showed significant similarity to numerous genomic regions (<a href="#pone-0000203-t001">Table 1</a>), suggesting the possibility that this miRNA gene family may include many as yet uncharacterized members.</p>
<div class="figure" id="pone-0000203-g001"><div class="img"><a name="pone-0000203-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000203.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000203" data-uri="info:doi/10.1371/journal.pone.0000203.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000203.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000203.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000203.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000203.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000203.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000203.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000203.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000203.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000203.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Multiple sequence alignment of Made1 and hsa-mir-548 genes.</span></strong></p><a id="article1.body1.sec3.sec1.fig1.caption1.p1" name="article1.body1.sec3.sec1.fig1.caption1.p1"></a><p>The location of the mature miRNA sequence is indicated by the bar over the alignment. The strand of the Made1 element (+/−) from which the miRNA genes are derived is shown to the right of the alignment.</p>
<span>doi:10.1371/journal.pone.0000203.g001</span></div><a id="article1.body1.sec3.sec1.p3" name="article1.body1.sec3.sec1.p3"></a><p>Made1 elements were independently characterized by several groups as non-autonomous derivatives of the human mariner-like transposable element (TE) Hsmar1 <a href="#pone.0000203-Morgan1">[38]</a>–<a href="#pone.0000203-Smit2">[40]</a>. Hsmar1 elements are DNA-type TEs, approximately 1,300 bp in length, which possess a transposase-encoding open reading frame flanked by terminal inverted repeat (TIR) sequences (<a href="#pone-0000203-g002">Figure 2A</a>) <a href="#pone.0000203-Robertson1">[41]</a>. Related Made1 elements are only 80bp long with two 37 bp TIRs and a 6 bp intervening region (<a href="#pone-0000203-g002">Figure 2B</a>). In this sense, Made1 sequences are palindromes, and if they were to be transcribed, they would form highly stable hairpin-loops reminiscent of the pre-miRNA structures that are processed to form mature miRNAs (<a href="#pone-0000203-g002">Figure 2C</a>).</p>
<div class="figure" id="pone-0000203-g002"><div class="img"><a name="pone-0000203-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000203.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000203" data-uri="info:doi/10.1371/journal.pone.0000203.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000203.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000203.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000203.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000203.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000203.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000203.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000203.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000203.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000203.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Schematic illustrating the relationship between Hsmar DNA-type TEs (A), Made1 MITEs (B) and hairpins (C) of the kind recognized by the miRNA enzymatic processing machinery.</span></strong></p><a id="article1.body1.sec3.sec1.fig2.caption1.p1" name="article1.body1.sec3.sec1.fig2.caption1.p1"></a><p>A) A full length DNA-type element with terminal inverted repeats (TIRs) flanking an open reading frame (ORF) is shown. B) Non-autonomous MITE derivative of a full length DNA-type element, containing TIRs but no internal ORF. C) Predicted hairpin structure that would be formed by base-pair interactions of the MITE TIRs.</p>
<span>doi:10.1371/journal.pone.0000203.g002</span></div><a id="article1.body1.sec3.sec1.p4" name="article1.body1.sec3.sec1.p4"></a><p>The formation of TIR-based dsRNA hairpins from Made1 would require the generation of full-length (or nearly so) element transcripts. The human expressed sequence tag (EST) database was searched using BLASTN <a href="#pone.0000203-Altschul1">[28]</a>, with a full-length Made1 query sequence, to test for this. We found 141 human ESTs that showed &gt;80% sequence similarity to the Made1 query sequence over &gt;80% of the length of the element (<a href="#pone.0000203.s004">Table S1</a>). Furthermore, the EST analysis indicates that Made1 sequences are widely expressed in a variety of tissue-types, providing ample opportunity for the formation of mature miRNAs.</p>
<a id="article1.body1.sec3.sec1.p5" name="article1.body1.sec3.sec1.p5"></a><p>Interestingly, Made1 transcripts destined to become hsa-mir-548 miRNAs are generated from both strands of the element (<a href="#pone-0000203-g001">Figure 1</a>). Because the element sequences are palindromes, transcripts produced in either orientation (+/−) would yield local hairpin structures. Indeed, the only difference between strand-specific transcripts is seen for the intervening 6bp sequence that forms the loop in the structure (positions 51–56 in <a href="#pone-0000203-g001">Figure 1</a>). This suggests that Made1 expression may result from read-through transcripts promoted from adjacent genomic positions, as opposed to a strand specific promoter encoded by the element itself. Consistent with this notion, we found that a number of Made1 homologous ESTs include substantial upstream and downstream sequences (<a href="#pone.0000203.s004">Table S1</a>).</p>
<a id="article1.body1.sec3.sec1.p6" name="article1.body1.sec3.sec1.p6"></a><p>Therefore, we propose a model whereby Made1 insertions into transcriptionally active genomic regions would yield viable pri-miRNA structures that would be processed into mature miRNA sequences by the RNA interference enzymatic machinery. An example of such a scenario can be seen for the human EST corresponding to the Genbank accession BU608159. This 754 base pair (bp) EST maps to chromosome chr13 at positions 24,718,360–24,719,104; it includes a nearly full length Made1 element as well as 325 bp of 5′ flanking DNA and 353 bp of sequence 3′ to the element. Visualization of genomic expression data, generated with human genome tilling arrays <a href="#pone.0000203-Cheng1">[30]</a>, shows that this particular Made1 is inserted into an intergenic region of the genome that is transcriptionally active (<a href="#pone-0000203-g003">Figure 3</a>). In this case, the entire Made1 element is transcribed as a read-through initiated from an adjacent genomic position. When the RNA structure of the EST, which includes the Made1 element along with expressed genomic flanking regions, is evaluated using the program Mfold, the Made1 region can be seen to form the most stable stem-loop structural element in the RNA (<a href="#pone-0000203-g004">Figure 4A</a>). The tight hairpin formed by the element is similar to the structures processed by Drosha and Dicer, and the location of the mature miRNA sequence, in the stem close to the 3′ end of the structure, is consistent with the mode of cleavage thought to be employed by the Dicer (<a href="#pone-0000203-g004">Figure 4B</a>).</p>
<div class="figure" id="pone-0000203-g003"><div class="img"><a name="pone-0000203-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000203.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000203" data-uri="info:doi/10.1371/journal.pone.0000203.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000203.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000203.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000203.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000203.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000203.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000203.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000203.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000203.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000203.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Made1 insertion in a transcriptionally active region of the human genome.</span></strong></p><a id="article1.body1.sec3.sec1.fig3.caption1.p1" name="article1.body1.sec3.sec1.fig3.caption1.p1"></a><p>The Made1 element shown is expressed by read-through from an adjacent promoter position in the genome. The EST BU608159 consists of the Made1 element along with 678 bp of flanking DNA.</p>
<span>doi:10.1371/journal.pone.0000203.g003</span></div>

<h4>Regulatory effects of hsa-mir-548</h4>
<a id="article1.body1.sec3.sec2.p1" name="article1.body1.sec3.sec2.p1"></a><p>Mature miRNA sequences associate with the RNA-induced silencing complex (RISC), which facilitates their regulatory interactions with target mRNAs (mRNAs) <a href="#pone.0000203-Bartel1">[3]</a>. miRNAs wield specific regulatory effects on gene expression through physical interactions with partially complementary sequences in the 3′ untranslated regions (UTRs) of their target genes' transcripts. We sought to characterize the potential regulatory and functional effects of hsa-mir-548 miRNAs by analyzing the genes that they are predicted to target.</p>
<a id="article1.body1.sec3.sec2.p2" name="article1.body1.sec3.sec2.p2"></a><p>Putative hsa-mir-548 target sites were identified using two methods: i-by the modified miRanda algorithm implemented in miRBase and ii-by searching 3′ UTRs for Made1 sequences that are complementary to the mature hsa-mir-548 miRNAs. According to the miRBase predictions, the seven hsa-mir-548 genes have 3,527 potential target genes. Made1 related targets, on the other hand, are found in only 179 genes. This was slightly surprising given that there are 7,850 annotated Made1 sequences in the human genome. When the search for Made1 derived target sites was extended to entire transcripts, only one additional target was found in a 5′ UTR. Apparently, Made1 sequences avoid protein coding gene exon regions and thus are poorly represented among potential hsa-mir-548 target sites. Furthermore, the intersection of the target gene sets derived from the miRBase versus Made1 consists of a mere 29 genes, and this figure is only slightly higher than the random expectation of 27 shared targets (<em>P</em> = 0.07 binomial distribution). That both target prediction methods detect such a small number of the same targets can be attributed to the fact that Made1 targets are likely to be avoided by the miRanda based approach due to its criterion of sequence conservation and the fact that Made1 is an evolutionarily young TE family. Indeed, when the sequence conservation levels of target sites identified by the two methods were compared, Made1 related targets were found to be significantly less conserved, on average, than miRanda predicted targets (conservation scores: Made1 targets = 0.082±0.017 miRanda targets = 0.319±0.007; t = 11.27 <em>P</em> = 5.7e-29 Student's ttest).</p>
<a id="article1.body1.sec3.sec2.p3" name="article1.body1.sec3.sec2.p3"></a><p>The potential functional relevance of genes with Made1 derived target sites was evaluated by considering their Gene Ontology (GO) biological process annotations and looking for over-represented functional categories. This procedure identified seven over-represented GO biological process categories that include a total of 11 genes (<a href="#pone.0000203.s005">Table S2</a>). The relationships among the over-represented GO functional categories in the Made1 target gene set can be visualized on the GO directed acyclic graph (<a href="#pone-0000203-g005">Figure 5</a>). This set includes genes with functional roles in cell proliferation, mitosis and apoptosis, all categories that are related to cancer. The hsa-mir-548 genes were characterized by virtue of their expression in colorectal cancer cell lines and tissue samples <a href="#pone.0000203-Cummins1">[10]</a>. If hsa-mir-548 expression is upregulated in colorectal cancer tissue, it may lead to the repression of genes that normally control cellular proliferation. Consistent with this scenario, several of the genes that correspond to over-represented functional categories were found to be down regulated in colorectal cancer tissue (<a href="#pone.0000203.s005">Table S2</a>). These include genes encoding a cell division cycle protein (ENSG00000004897), a C epsilon type protein kinase (ENSG00000171132) and a centromere/kinetochore protein (ENSG00000086827).</p>
<div class="figure" id="pone-0000203-g004"><div class="img"><a name="pone-0000203-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000203.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000203" data-uri="info:doi/10.1371/journal.pone.0000203.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000203.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000203.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000203.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000203.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000203.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000203.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000203.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000203.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000203.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>RNA secondary structures of the entire BU608159 EST (A) and the Made1 element contained within this transcript (B).</span></strong></p><a id="article1.body1.sec3.sec2.fig1.caption1.p1" name="article1.body1.sec3.sec2.fig1.caption1.p1"></a><p>The Made1 hairpin region of the BU608159 structure is indicated with a blue bar (A), and the location of the mature miRNA sequence is shown with the red bar (B).</p>
<span>doi:10.1371/journal.pone.0000203.g004</span></div><div class="figure" id="pone-0000203-g005"><div class="img"><a name="pone-0000203-g005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000203.g005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000203" data-uri="info:doi/10.1371/journal.pone.0000203.g005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000203.g005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000203.g005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000203.g005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000203.g005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000203.g005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000203.g005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000203.g005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000203.g005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000203.g005.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 5.  <span>GO biological process terms over-represented among the set of genes with Made1 derived hsa-mir-548 target sites.</span></strong></p><a id="article1.body1.sec3.sec2.fig2.caption1.p1" name="article1.body1.sec3.sec2.fig2.caption1.p1"></a><p>The portion of the directed acyclic graph (DAG) containing all paths from the root biological process term to the over-represented functional category terms is shown. Over-represented functional categories are indicated in red.</p>
<span>doi:10.1371/journal.pone.0000203.g005</span></div><a id="article1.body1.sec3.sec2.p4" name="article1.body1.sec3.sec2.p4"></a><p>As mentioned previously, the paucity of Made1 related target sites was somewhat unexpected. Nevertheless, the identification of numerous non-Made1 related target sites is interesting in the sense that it suggests that TE-derived miRNAs may be able to regulate host genes that do not have any related TE sequences. There are two models to explain the repressive effects that miRNAs exert on target gene expression: i-translational repression and ii-mRNA degradation <a href="#pone.0000203-Yekta1">[16]</a>, <a href="#pone.0000203-Zeng1">[17]</a>, <a href="#pone.0000203-Babak1">[42]</a>. Recently, anti-correlated expression patterns between miRNA sequences and their target mRNAs have provided evidence in favor of the mRNA degradation model <a href="#pone.0000203-Huang1">[43]</a>. We sought to further evaluate the potential mRNA degradation-based regulatory effects of the hsa-mir-548 miRNAs by searching for down regulation of putative target genes in tissue samples similar to the colorectal samples from which they were cloned <a href="#pone.0000203-Cummins1">[10]</a>. Consideration of target gene relative expression levels can also be used to help validate target site predictions, which are prone to false positives.</p>
<a id="article1.body1.sec3.sec2.p5" name="article1.body1.sec3.sec2.p5"></a><p>Gene expression profiles for potential hsa-mir-548 targets were taken from the Novartis Research Foundation's Symatlas <a href="#pone.0000203-Su1">[34]</a>. For the miRBase set, a total of 2,045 target genes were found with corresponding Symatlas expression data across 79 human tissues. The expression data were median and log normalized to yield relative tissue-specific gene expression profiles, and these profiles were separated into 20 co-expressed groups of genes using k-means clustering. Three of these clusters – 12, 15 and 20 – showed marked down-regulation of the colorectal adenocarcinoma sample (<a href="#pone-0000203-g006">Figure 6</a>). Interestingly, the genes found in these same clusters tended to be down-regulated in all five of the other cancer-related samples in the data set (<a href="#pone-0000203-g007">Figure 7</a>). This suggests the possibility that hsa-mir-548 miRNA genes may play some global role related to the regulation of gene expression in cancer. Indeed, hierarchical clustering of the tissue-samples based on the gene expression data unites all of the cancer samples into a single group to the exclusion of all normal tissues (<a href="#pone.0000203.s001">Figure S1</a>); however, the colorectal adenocarcinoma sample is the outlier of this group (<a href="#pone-0000203-g008">Figure 8</a>). When the log<sub>2</sub> median expression ratios were averaged for all genes with putative hsa-mir-548 target sites, the colorectal sample had the lowest relative expression level (<em>q</em> = 9.72, <em>v</em> = 12738, <em>k</em> = 6, <em>P</em>&lt;0.001 Tukey test; <a href="#pone-0000203-g008">Figure 8</a>). This finding is consistent with the fact that the hsa-mir-548 genes were isolated from colorectal cancer samples, and points to an additional more specific role for these genes in colorectal cancer related gene regulation. The functional affinities of the genes in the three down regulated clusters were assessed using the same GO-based approach as for the set of genes with Made1 target sites. There are 29 GO biological process categories, encompassing 104 genes, which contain an over-representation of genes from these clusters (<a href="#pone.0000203.s006">Table S3</a>). These include genes involved in cell adhesion, cell signalling and signal transduction. The positions of these categories on the GO biological process DAG can be seen in <a href="#pone.0000203.s002">Figure S2</a>.</p>
<div class="figure" id="pone-0000203-g006"><div class="img"><a name="pone-0000203-g006" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000203.g006&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000203" data-uri="info:doi/10.1371/journal.pone.0000203.g006"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000203.g006&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000203.g006/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000203.g006/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000203.g006/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000203.g006/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000203.g006.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000203.g006/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000203.g006/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000203.g006.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 6.  <span>Coexpressed clusters of putative hsa-mir-548 target genes.</span></strong></p><a id="article1.body1.sec3.sec2.fig3.caption1.p1" name="article1.body1.sec3.sec2.fig3.caption1.p1"></a><p>Centroid views with average tissue-specific expression values are shown for all 20 clusters. Clusters containing genes down-regulated in the colorectal adenocarcinoma sample are shown in red and arrows indicate the colorectal sample.</p>
<span>doi:10.1371/journal.pone.0000203.g006</span></div><div class="figure" id="pone-0000203-g007"><div class="img"><a name="pone-0000203-g007" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000203.g007&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000203" data-uri="info:doi/10.1371/journal.pone.0000203.g007"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000203.g007&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000203.g007/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000203.g007/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000203.g007/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000203.g007/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000203.g007.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000203.g007/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000203.g007/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000203.g007.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 7.  <span>Representative gene expression profiles for putative hsa-mir-548 target genes from three coexpressed clusters (12, 15 and 20 in <a href="#pone-0000203-g006">Figure 6</a>).</span></strong></p><a id="article1.body1.sec3.sec2.fig4.caption1.p1" name="article1.body1.sec3.sec2.fig4.caption1.p1"></a><p>Expression profiles are median centered and log<sub>2</sub> normalized, and the log<sub>2</sub> ratio color scale is shown above the plot. Overexpressed genes are shown in red and underexpressed genes are shown in green.</p>
<span>doi:10.1371/journal.pone.0000203.g007</span></div><div class="figure" id="pone-0000203-g008"><div class="img"><a name="pone-0000203-g008" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000203.g008&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000203" data-uri="info:doi/10.1371/journal.pone.0000203.g008"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000203.g008&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000203.g008/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000203.g008/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000203.g008/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000203.g008/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000203.g008.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000203.g008/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000203.g008/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000203.g008.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 8.  <span>Relationships and average relative expression levels among the cancer tissues samples from the Novartis Symatlas microarray dataset.
                  </span></strong></p><span>doi:10.1371/journal.pone.0000203.g008</span></div><a id="article1.body1.sec3.sec2.p6" name="article1.body1.sec3.sec2.p6"></a><p>We also compared putative hsa-mir-548 target genes to a recently published collection of genes that were indicated as being involved in colorectal cancer by microarray expression profiling <a href="#pone.0000203-Shih1">[44]</a>. We found 22 examples of putative hsa-mir-548 target genes that were previously found to be related to colorectal cancer based on down-regulation in six separate microarray studies (<a href="#pone.0000203.s007">Table S4</a>). These include a number of genes encoding various immune cell receptors as well as transcription factors and tumor necrosis factors. The apparent connection between cancer and the immune system in our dataset is supported by the similar down-regulated expression patterns seen for hsa-mir-548 target genes among the cancer and immune tissue samples (<a href="#pone-0000203-g007">Figure 7</a>). However, a number of genes previously implicated in colorectal cancer etiology by virtue of up-regulation in previous studies were also found to have predicted hsa-mir-548 target sites. These cases may represent false positive target site predictions or could point to instances where hsa-mir-548 miRNAs act through translational repression and thus do not repress mRNA expression levels.</p>


<h4>Conclusion</h4>
<a id="article1.body1.sec3.sec3.p1" name="article1.body1.sec3.sec3.p1"></a><p>We report here a human miRNA gene family derived from TEs. The palindromic structure of the Made1 elements from which the hsa-mir-548 miRNA genes originated, together with their insertion into transcriptionally active genomic regions, points to a specific mechanism by which these sequences can be recognized and processed by the enzymatic machinery that yields mature miRNA sequences. In addition, the dispersed repetitive nature of TE sequences provides for the emergence of multiple novel miRNA genes as well as numerous homologous target sites throughout the genome.</p>
<a id="article1.body1.sec3.sec3.p2" name="article1.body1.sec3.sec3.p2"></a><p>TEs also tend to be among the most lineage-specific, <em>i.e.</em> recently evolved, sequences in the human genome <a href="#pone.0000203-Lander1">[45]</a>. Made1 elements emerged along the primate evolutionary lineage, and orthologous hsa-mir-548 sequences are confined to the human, chimpanzee and rhesus macaque genome sequences (<a href="#pone.0000203.s003">Figure S3</a>). While many miRNA genes are conserved across more distantly related species, a recent analysis of the human genome detected numerous putative miRNAs that are not evolutionarily conserved <a href="#pone.0000203-Bentwich1">[5]</a>. TEs, such as Madel, represent a natural source of such lineage-specific miRNAs, which could in turn be responsible for regulatory phenotypes that contribute to evolutionary diversification between species. The relatively low conservation of Made1 derived target sites is also consistent with this lineage-specific mode of evolution.</p>
<a id="article1.body1.sec3.sec3.p3" name="article1.body1.sec3.sec3.p3"></a><p>MITEs are widely distributed among eukaryotes <a href="#pone.0000203-Feschotte1">[46]</a> and could provide for the emergence of regulatory RNAs, such as miRNAs, siRNAs or other small non-coding RNAs, in many different genomic contexts. For instance, MITEs are particularly prevalent in plants where they were first discovered <a href="#pone.0000203-Bureau1">[47]</a>; the rice genome alone contains ~90,000 MITEs <a href="#pone.0000203-Jiang1">[48]</a>. A striking feature of plant MITEs is their apparent preference for insertion in gene regions <a href="#pone.0000203-Mao1">[49]</a>, <a href="#pone.0000203-Zhang2">[50]</a>. Accordingly, many thousands of plant MITEs must be expressed along with the gene sequences in which they are inserted. This would provide ample opportunities for the processing of MITE hairpins by RNA interference enzymatic machinery, which is known to play a particularly important role in plant gene regulation <a href="#pone.0000203-Matzke2">[51]</a>.</p>
<a id="article1.body1.sec3.sec3.p4" name="article1.body1.sec3.sec3.p4"></a><p>Finally, we would like to propose that MITEs, such as Made1, may represent an evolutionary intermediate between siRNAs and miRNAs. A number of epigenetic gene silencing mechanisms, such as cytosine methylation <a href="#pone.0000203-Yoder1">[52]</a>, genomic imprinting <a href="#pone.0000203-McDonald1">[53]</a> and heterochromatin <a href="#pone.0000203-Lippman1">[54]</a> are thought to have evolved as defense mechanisms against transposition. Subsequently, these TE silencing mechanisms were co-opted as global regulators to control the expression patterns of host genes. This may have led to the increase in regulatory and phenotypic complexity seen among members of the eukaryotic crown group. In a similar way, RNA interference by siRNAs is considered to have evolved to silence TEs <a href="#pone.0000203-Matzke1">[18]</a>, <a href="#pone.0000203-Vastenhouw1">[20]</a>. Consistent with this model, there are a number of cases of siRNAs that originate from TEs in different species <a href="#pone.0000203-Aravin1">[55]</a>–<a href="#pone.0000203-Zilberman1">[58]</a>. Perhaps the best characterized example of this is the Muk TE repressor in maize <a href="#pone.0000203-Slotkin1">[19]</a>. Muk is an effective silencer of the MuDR DNA-type TE, and the Muk locus consists of an inverted duplication of a partially deleted MuDR element. When Muk is transcribed, it yields a long (&gt;2 kb) dsRNA hairpin structure that is processed to yield siRNAs. The connection between TEs and siRNAs has led to the proposal that origination from TEs distinguishes siRNAs from miRNAs <a href="#pone.0000203-Bartel1">[3]</a>. However, as reported here and elsewhere <a href="#pone.0000203-Smalheiser1">[21]</a>, more and more TE-derived miRNAs are being discovered.</p>
<a id="article1.body1.sec3.sec3.p5" name="article1.body1.sec3.sec3.p5"></a><p>The model of miRNA emergence from MITEs that we propose here (<a href="#pone-0000203-g002">Figure 2</a>) suggests a way that miRNAs could have evolved from TE encoded siRNAs. One possible source of the TE encoded dsRNAs that serve as siRNA precursors is snap back panhandle structures between TIRs of autonomous DNA-type elements <a href="#pone.0000203-Vastenhouw1">[20]</a>. Such panhandle structures would include long internal loop regions that correspond to the internal open reading frames that are lost when autonomous elements are converted to non-autonomous MITE derivatives. MITEs retain the TIRs, and those same TIRs that were processed from longer RNAs to form siRNA could be similarly processed to form miRNAs. The shorter hairpin structures formed by MITE transcripts could lead to steric constraints that result in the liberation of only one mature miRNA sequence as opposed to the numerous siRNAs that are produced from longer dsRNAs. In this way, short hairpin loop derived miRNAs may have evolved from TE encoded siRNAs. Many of the extant miRNA genes characterized today may have evolved beyond recognition to their progenitor TEs, while others may have originated from other genomic re-structuring mechanisms that juxtapose short inverted repeats <a href="#pone.0000203-Allen1">[59]</a>.</p>

</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0000203"></div><a name="pone.0000203.s001" id="pone.0000203.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000203.s001">Figure S1. </a></strong></p><a id="article1.body1.sec4.supplementary-material1.caption1.p1" name="article1.body1.sec4.supplementary-material1.caption1.p1"></a><p class="preSiDOI">Dendogram showing relationships among tissues from the Novartis Foundation Symatlas microarray dataset. Cancer tissues are indicated with the red bar.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000203.s001</p><a id="article1.body1.sec4.supplementary-material1.caption1.p2" name="article1.body1.sec4.supplementary-material1.caption1.p2"></a><p class="postSiDOI">(0.45 MB PDF)</p>
<a name="pone.0000203.s002" id="pone.0000203.s002"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000203.s002">Figure S2. </a></strong></p><a id="article1.body1.sec4.supplementary-material2.caption1.p1" name="article1.body1.sec4.supplementary-material2.caption1.p1"></a><p class="preSiDOI">Over-represented GO biological process categories among genes with miRanda predicted hsa-mir-548 target sites that map to colorectal cancer down-regulated co-expression clusters (i.e. 12, 15 &amp; 20 in <a href="#pone-0000203-g006">Figure 6</a>). The portion of the directed acyclic graph (DAG) containing all paths from the root biological process term to the over-represented functional category terms is shown. Over-represented functional categories are indicated in red.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000203.s002</p><a id="article1.body1.sec4.supplementary-material2.caption1.p2" name="article1.body1.sec4.supplementary-material2.caption1.p2"></a><p class="postSiDOI">(0.06 MB PDF)</p>
<a name="pone.0000203.s003" id="pone.0000203.s003"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000203.s003">Figure S3. </a></strong></p><a id="article1.body1.sec4.supplementary-material3.caption1.p1" name="article1.body1.sec4.supplementary-material3.caption1.p1"></a><p class="preSiDOI">Made1-derived miRNA genes are primate-specific. Human genomic regions corresponding to Made1-derived miRNA genes are shown: A hsa-mir-548a-1, B hsa-mir-548-a2, C hsa-mir-548-a3, D hsa-mir-548-b, E hsa-mir-548-c, F-hsa-mir-548-d1, G-hsa-mir-548-d2. The UCSC Genome Browser is used to show the location of the Made1 elements (DNA) in the RepeatMasker track. Evolutionary comparisons between the human genome and the corresponding regions in the chimp, rhesus, mouse, rat, dog and cow genomes are shown using the species-specific Net tracks of the Genome Browser. Corresponding Made1 orthologous regions that are present in another species are indicated with a broad line, while regions that are missing in another species are shown with a thin line.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000203.s003</p><a id="article1.body1.sec4.supplementary-material3.caption1.p2" name="article1.body1.sec4.supplementary-material3.caption1.p2"></a><p class="postSiDOI">(3.44 MB PDF)</p>
<a name="pone.0000203.s004" id="pone.0000203.s004"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000203.s004">Table S1. </a></strong></p><a id="article1.body1.sec4.supplementary-material4.caption1.p1" name="article1.body1.sec4.supplementary-material4.caption1.p1"></a><p class="preSiDOI">Made1 homologous human expressed sequence tags (ESTs).</p>
<p class="siDoi">doi:10.1371/journal.pone.0000203.s004</p><a id="article1.body1.sec4.supplementary-material4.caption1.p2" name="article1.body1.sec4.supplementary-material4.caption1.p2"></a><p class="postSiDOI">(0.32 MB DOC)</p>
<a name="pone.0000203.s005" id="pone.0000203.s005"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000203.s005">Table S2. </a></strong></p><a id="article1.body1.sec4.supplementary-material5.caption1.p1" name="article1.body1.sec4.supplementary-material5.caption1.p1"></a><p class="preSiDOI">Over-represented GO biological process categories among genes with Made1 derived hsa-mir-548 target sites.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000203.s005</p><a id="article1.body1.sec4.supplementary-material5.caption1.p2" name="article1.body1.sec4.supplementary-material5.caption1.p2"></a><p class="postSiDOI">(0.04 MB DOC)</p>
<a name="pone.0000203.s006" id="pone.0000203.s006"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000203.s006">Table S3. </a></strong></p><a id="article1.body1.sec4.supplementary-material6.caption1.p1" name="article1.body1.sec4.supplementary-material6.caption1.p1"></a><p class="preSiDOI">Over-represented GO biological process categories among genes with miRanda predicted hsa-mir-548 target sites that map to colorectal cancer down-regulated co-expression clusters (i.e. 12, 15 &amp; 20 in <a href="#pone-0000203-g006">Figure 6</a>).</p>
<p class="siDoi">doi:10.1371/journal.pone.0000203.s006</p><a id="article1.body1.sec4.supplementary-material6.caption1.p2" name="article1.body1.sec4.supplementary-material6.caption1.p2"></a><p class="postSiDOI">(0.09 MB DOC)</p>
<a name="pone.0000203.s007" id="pone.0000203.s007"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000203.s007">Table S4. </a></strong></p><a id="article1.body1.sec4.supplementary-material7.caption1.p1" name="article1.body1.sec4.supplementary-material7.caption1.p1"></a><p class="preSiDOI">Putative hsa-mir-548 target genes previously implicated as being involved in colorectal cancer by microarray expression profiling.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000203.s007</p><a id="article1.body1.sec4.supplementary-material7.caption1.p2" name="article1.body1.sec4.supplementary-material7.caption1.p2"></a><p class="postSiDOI">(0.20 MB DOC)</p>
</div>



<div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: IJ JP. Performed the experiments: JP. Analyzed the data: IJ JP. Contributed reagents/materials/analysis tools: JP. Wrote the paper: IJ JP.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0000203-Mattick1" id="pone.0000203-Mattick1"></a>Mattick JS, Makunin IV (2006) Non-coding RNA. Hum Mol Genet  15 Spec No 1: R17–29.  <ul class="find" data-citedArticleID="929644"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Non-coding+RNA.&amp;auth=&amp;atitle=Non-coding+RNA." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Non-coding+RNA." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Non-coding+RNA.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0000203-Ambros1" id="pone.0000203-Ambros1"></a>Ambros V (2004) The functions of animal microRNAs. Nature  431: 350–355.  <ul class="find" data-citedArticleID="929578" data-doi="10.1038/nature02871"><li><a href="http://dx.doi.org/10.1038/nature02871" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+functions+of+animal+microRNAs." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+functions+of+animal+microRNAs.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0000203-Bartel1" id="pone.0000203-Bartel1"></a>Bartel DP (2004) MicroRNAs: genomics, biogenesis, mechanism, and function. Cell  116: 281–297.  <ul class="find" data-citedArticleID="929588" data-doi="10.1016/s0092-8674(04)00045-5"><li><a href="http://dx.doi.org/10.1016/s0092-8674(04)00045-5" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=MicroRNAs%3A+genomics%2C+biogenesis%2C+mechanism%2C+and+function." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22MicroRNAs%3A+genomics%2C+biogenesis%2C+mechanism%2C+and+function.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0000203-Lee1" id="pone.0000203-Lee1"></a>Lee RC, Feinbaum RL, Ambros V (1993) The C. elegans heterochronic gene lin-4 encodes small RNAs with antisense complementarity to lin-14. Cell  75: 843–854.  <ul class="find" data-citedArticleID="929632" data-doi="10.1016/0092-8674(93)90529-y"><li><a href="http://dx.doi.org/10.1016/0092-8674(93)90529-y" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+C.+elegans+heterochronic+gene+lin-4+encodes+small+RNAs+with+antisense+complementarity+to+lin-14." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+C.+elegans+heterochronic+gene+lin-4+encodes+small+RNAs+with+antisense+complementarity+to+lin-14.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0000203-Bentwich1" id="pone.0000203-Bentwich1"></a>Bentwich I, Avniel A, Karov Y, Aharonov R, Gilad S, et al.  (2005) Identification of hundreds of conserved and nonconserved human microRNAs. Nat Genet  37: 766–770.  <ul class="find" data-citedArticleID="929590" data-doi="10.1038/ng1590"><li><a href="http://dx.doi.org/10.1038/ng1590" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Identification+of+hundreds+of+conserved+and+nonconserved+human+microRNAs." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Identification+of+hundreds+of+conserved+and+nonconserved+human+microRNAs.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0000203-Chen1" id="pone.0000203-Chen1"></a>Chen CZ, Li L, Lodish HF, Bartel DP (2004) MicroRNAs modulate hematopoietic lineage differentiation. Science  303: 83–86.  <ul class="find" data-citedArticleID="929598" data-doi="10.1126/science.1091903"><li><a href="http://dx.doi.org/10.1126/science.1091903" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=MicroRNAs+modulate+hematopoietic+lineage+differentiation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22MicroRNAs+modulate+hematopoietic+lineage+differentiation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0000203-LagosQuintana1" id="pone.0000203-LagosQuintana1"></a>Lagos-Quintana M, Rauhut R, Lendeckel W, Tuschl T (2001) Identification of novel genes coding for small expressed RNAs. Science  294: 853–858.  <ul class="find" data-citedArticleID="929626" data-doi="10.1126/science.1064921"><li><a href="http://dx.doi.org/10.1126/science.1064921" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Identification+of+novel+genes+coding+for+small+expressed+RNAs." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Identification+of+novel+genes+coding+for+small+expressed+RNAs.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0000203-Lau1" id="pone.0000203-Lau1"></a>Lau NC, Lim LP, Weinstein EG, Bartel DP (2001) An abundant class of tiny RNAs with probable regulatory roles in Caenorhabditis elegans. Science  294: 858–862.  <ul class="find" data-citedArticleID="929630" data-doi="10.1126/science.1065062"><li><a href="http://dx.doi.org/10.1126/science.1065062" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=An+abundant+class+of+tiny+RNAs+with+probable+regulatory+roles+in+Caenorhabditis+elegans." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22An+abundant+class+of+tiny+RNAs+with+probable+regulatory+roles+in+Caenorhabditis+elegans.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0000203-Lee2" id="pone.0000203-Lee2"></a>Lee RC, Ambros V (2001) An extensive class of small RNAs in Caenorhabditis elegans. Science  294: 862–864.  <ul class="find" data-citedArticleID="929634" data-doi="10.1126/science.1065329"><li><a href="http://dx.doi.org/10.1126/science.1065329" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=An+extensive+class+of+small+RNAs+in+Caenorhabditis+elegans." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22An+extensive+class+of+small+RNAs+in+Caenorhabditis+elegans.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0000203-Cummins1" id="pone.0000203-Cummins1"></a>Cummins JM, He Y, Leary RJ, Pagliarini R, Diaz LA Jr, et al.  (2006) The colorectal microRNAome. Proc Natl Acad Sci U S A  103: 3687–3692.  <ul class="find" data-citedArticleID="929602" data-doi="10.1073/pnas.0511155103"><li><a href="http://dx.doi.org/10.1073/pnas.0511155103" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+colorectal+microRNAome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+colorectal+microRNAome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0000203-GriffithsJones1" id="pone.0000203-GriffithsJones1"></a>Griffiths-Jones S, Grocock RJ, van Dongen S, Bateman A, Enright AJ (2006) miRBase: microRNA sequences, targets and gene nomenclature. Nucleic Acids Res  34: D140–144.  <ul class="find" data-citedArticleID="929608" data-doi="10.1093/nar/gkj112"><li><a href="http://dx.doi.org/10.1093/nar/gkj112" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=miRBase%3A+microRNA+sequences%2C+targets+and+gene+nomenclature." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22miRBase%3A+microRNA+sequences%2C+targets+and+gene+nomenclature.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0000203-Enright1" id="pone.0000203-Enright1"></a>Enright AJ, John B, Gaul U, Tuschl T, Sander C, et al.  (2003) MicroRNA targets in Drosophila. Genome Biol  5: R1.  <ul class="find" data-citedArticleID="929604" data-doi="10.1186/gb-2003-5-1-r1"><li><a href="http://dx.doi.org/10.1186/gb-2003-5-1-r1" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=MicroRNA+targets+in+Drosophila." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22MicroRNA+targets+in+Drosophila.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0000203-Ambros2" id="pone.0000203-Ambros2"></a>Ambros V, Bartel B, Bartel DP, Burge CB, Carrington JC, et al.  (2003) A uniform system for microRNA annotation. Rna  9: 277–279.  <ul class="find" data-citedArticleID="929580" data-doi="10.1261/rna.2183803"><li><a href="http://dx.doi.org/10.1261/rna.2183803" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+uniform+system+for+microRNA+annotation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+uniform+system+for+microRNA+annotation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0000203-Hutvagner1" id="pone.0000203-Hutvagner1"></a>Hutvagner G, Zamore PD (2002) A microRNA in a multiple-turnover RNAi enzyme complex. Science  297: 2056–2060.  <ul class="find" data-citedArticleID="929614" data-doi="10.1126/science.1073827"><li><a href="http://dx.doi.org/10.1126/science.1073827" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+microRNA+in+a+multiple-turnover+RNAi+enzyme+complex." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+microRNA+in+a+multiple-turnover+RNAi+enzyme+complex.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0000203-Llave1" id="pone.0000203-Llave1"></a>Llave C, Kasschau KD, Rector MA, Carrington JC (2002) Endogenous and silencing-associated small RNAs in plants. Plant Cell  14: 1605–1619.  <ul class="find" data-citedArticleID="929640" data-doi="10.1105/tpc.003210"><li><a href="http://dx.doi.org/10.1105/tpc.003210" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Endogenous+and+silencing-associated+small+RNAs+in+plants." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Endogenous+and+silencing-associated+small+RNAs+in+plants.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0000203-Yekta1" id="pone.0000203-Yekta1"></a>Yekta S, Shih IH, Bartel DP (2004) MicroRNA-directed cleavage of HOXB8 mRNA. Science  304: 594–596.  <ul class="find" data-citedArticleID="929678" data-doi="10.1126/science.1097434"><li><a href="http://dx.doi.org/10.1126/science.1097434" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=MicroRNA-directed+cleavage+of+HOXB8+mRNA." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22MicroRNA-directed+cleavage+of+HOXB8+mRNA.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0000203-Zeng1" id="pone.0000203-Zeng1"></a>Zeng Y, Yi R, Cullen BR (2003) MicroRNAs and small interfering RNAs can inhibit mRNA expression by similar mechanisms. Proc Natl Acad Sci U S A  100: 9779–9784.  <ul class="find" data-citedArticleID="929682" data-doi="10.1073/pnas.1630797100"><li><a href="http://dx.doi.org/10.1073/pnas.1630797100" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=MicroRNAs+and+small+interfering+RNAs+can+inhibit+mRNA+expression+by+similar+mechanisms." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22MicroRNAs+and+small+interfering+RNAs+can+inhibit+mRNA+expression+by+similar+mechanisms.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0000203-Matzke1" id="pone.0000203-Matzke1"></a>Matzke MA, Mette MF, Matzke AJ (2000) Transgene silencing by the host genome defense: implications for the evolution of epigenetic control mechanisms in plants and vertebrates. Plant Mol Biol  43: 401–415.  <ul class="find" data-citedArticleID="929646" data-doi="10.1007/978-94-011-4183-3_20"><li><a href="http://dx.doi.org/10.1007/978-94-011-4183-3_20" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Transgene+silencing+by+the+host+genome+defense%3A+implications+for+the+evolution+of+epigenetic+control+mechanisms+in+plants+and+vertebrates." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Transgene+silencing+by+the+host+genome+defense%3A+implications+for+the+evolution+of+epigenetic+control+mechanisms+in+plants+and+vertebrates.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0000203-Slotkin1" id="pone.0000203-Slotkin1"></a>Slotkin RK, Freeling M, Lisch D (2005) Heritable transposon silencing initiated by a naturally occurring transposon inverted duplication. Nat Genet  37: 641–644.  <ul class="find" data-citedArticleID="929662" data-doi="10.1038/ng1576"><li><a href="http://dx.doi.org/10.1038/ng1576" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Heritable+transposon+silencing+initiated+by+a+naturally+occurring+transposon+inverted+duplication." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Heritable+transposon+silencing+initiated+by+a+naturally+occurring+transposon+inverted+duplication.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0000203-Vastenhouw1" id="pone.0000203-Vastenhouw1"></a>Vastenhouw NL, Plasterk RH (2004) RNAi protects the Caenorhabditis elegans germline against transposition. Trends Genet  20: 314–319.  <ul class="find" data-citedArticleID="929676" data-doi="10.1016/j.tig.2004.04.011"><li><a href="http://dx.doi.org/10.1016/j.tig.2004.04.011" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=RNAi+protects+the+Caenorhabditis+elegans+germline+against+transposition." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22RNAi+protects+the+Caenorhabditis+elegans+germline+against+transposition.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0000203-Smalheiser1" id="pone.0000203-Smalheiser1"></a>Smalheiser NR, Torvik VI (2005) Mammalian microRNAs derived from genomic repeats. Trends Genet  21: 322–326.  <ul class="find" data-citedArticleID="929664" data-doi="10.1016/j.tig.2005.04.008"><li><a href="http://dx.doi.org/10.1016/j.tig.2005.04.008" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mammalian+microRNAs+derived+from+genomic+repeats." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mammalian+microRNAs+derived+from+genomic+repeats.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0000203-Karolchik1" id="pone.0000203-Karolchik1"></a>Karolchik D, Baertsch R, Diekhans M, Furey TS, Hinrichs A, et al.  (2003) The UCSC Genome Browser Database. Nucleic Acids Res  31: 51–54.  <ul class="find" data-citedArticleID="929618" data-doi="10.1093/nar/gkg129"><li><a href="http://dx.doi.org/10.1093/nar/gkg129" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+UCSC+Genome+Browser+Database." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+UCSC+Genome+Browser+Database.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0000203-Karolchik2" id="pone.0000203-Karolchik2"></a>Karolchik D, Hinrichs AS, Furey TS, Roskin KM, Sugnet CW, et al.  (2004) The UCSC Table Browser data retrieval tool. Nucleic Acids Res  32: D493–496.  <ul class="find" data-citedArticleID="929620" data-doi="10.1093/nar/gkh103"><li><a href="http://dx.doi.org/10.1093/nar/gkh103" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+UCSC+Table+Browser+data+retrieval+tool." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+UCSC+Table+Browser+data+retrieval+tool.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0000203-Smit1" id="pone.0000203-Smit1"></a>Smit AFA, Hubley R, Green P (1996–2004) RepeatMasker Open-3.0.   <ul class="find-nolinks"></ul></li><li><span class="label">25.
              </span><a name="pone.0000203-Kent1" id="pone.0000203-Kent1"></a>Kent WJ, Baertsch R, Hinrichs A, Miller W, Haussler D (2003) Evolution's cauldron: duplication, deletion, and rearrangement in the mouse and human genomes. Proc Natl Acad Sci U S A  100: 11484–11489.  <ul class="find" data-citedArticleID="929622" data-doi="10.1073/pnas.1932072100"><li><a href="http://dx.doi.org/10.1073/pnas.1932072100" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evolution%27s+cauldron%3A+duplication%2C+deletion%2C+and+rearrangement+in+the+mouse+and+human+genomes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evolution%27s+cauldron%3A+duplication%2C+deletion%2C+and+rearrangement+in+the+mouse+and+human+genomes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0000203-Kent2" id="pone.0000203-Kent2"></a>Kent WJ (2002) BLAT–the BLAST-like alignment tool. Genome Res  12: 656–664.  <ul class="find" data-citedArticleID="929624" data-doi="10.1101/gr.229202."><li><a href="http://dx.doi.org/10.1101/gr.229202." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=BLAT%26%23x2013%3Bthe+BLAST-like+alignment+tool." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22BLAT%26%23x2013%3Bthe+BLAST-like+alignment+tool.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0000203-Thompson1" id="pone.0000203-Thompson1"></a>Thompson JD, Higgins DG, Gibson TJ (1994) CLUSTAL W: improving the sensitivity of progressive multiple sequence alignment through sequence weighting, position-specific gap penalties and weight matrix choice. Nucleic Acids Res  22: 4673–4680.  <ul class="find" data-citedArticleID="929674" data-doi="10.1093/nar/22.22.4673"><li><a href="http://dx.doi.org/10.1093/nar/22.22.4673" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=CLUSTAL+W%3A+improving+the+sensitivity+of+progressive+multiple+sequence+alignment+through+sequence+weighting%2C+position-specific+gap+penalties+and+weight+matrix+choice." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22CLUSTAL+W%3A+improving+the+sensitivity+of+progressive+multiple+sequence+alignment+through+sequence+weighting%2C+position-specific+gap+penalties+and+weight+matrix+choice.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0000203-Altschul1" id="pone.0000203-Altschul1"></a>Altschul SF, Madden TL, Schaffer AA, Zhang J, Zhang Z, et al.  (1997) Gapped BLAST and PSI-BLAST: a new generation of protein database search programs. Nucleic Acids Res  25: 3389–3402.  <ul class="find" data-citedArticleID="929576" data-doi="10.1093/nar/25.17.3389"><li><a href="http://dx.doi.org/10.1093/nar/25.17.3389" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Gapped+BLAST+and+PSI-BLAST%3A+a+new+generation+of+protein+database+search+programs." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Gapped+BLAST+and+PSI-BLAST%3A+a+new+generation+of+protein+database+search+programs.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0000203-Boguski1" id="pone.0000203-Boguski1"></a>Boguski MS, Lowe TM, Tolstoshev CM (1993) dbEST–database for “expressed sequence tags”. Nat Genet  4: 332–333.  <ul class="find" data-citedArticleID="929594" data-doi="10.1038/ng0893-332"><li><a href="http://dx.doi.org/10.1038/ng0893-332" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=dbEST%26%23x2013%3Bdatabase+for+%26%23x201c%3Bexpressed+sequence+tags%26%23x201d%3B." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22dbEST%26%23x2013%3Bdatabase+for+%26%23x201c%3Bexpressed+sequence+tags%26%23x201d%3B.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0000203-Cheng1" id="pone.0000203-Cheng1"></a>Cheng J, Kapranov P, Drenkow J, Dike S, Brubaker S, et al.  (2005) Transcriptional maps of 10 human chromosomes at 5-nucleotide resolution. Science  308: 1149–1154.  <ul class="find" data-citedArticleID="929600" data-doi="10.1126/science.1108625"><li><a href="http://dx.doi.org/10.1126/science.1108625" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Transcriptional+maps+of+10+human+chromosomes+at+5-nucleotide+resolution." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Transcriptional+maps+of+10+human+chromosomes+at+5-nucleotide+resolution.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0000203-Zuker1" id="pone.0000203-Zuker1"></a>Zuker M (2003) Mfold web server for nucleic acid folding and hybridization prediction. Nucleic Acids Res  31: 3406–3415.  <ul class="find" data-citedArticleID="929690" data-doi="10.1093/nar/gkg595"><li><a href="http://dx.doi.org/10.1093/nar/gkg595" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mfold+web+server+for+nucleic+acid+folding+and+hybridization+prediction." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mfold+web+server+for+nucleic+acid+folding+and+hybridization+prediction.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0000203-Blanchette1" id="pone.0000203-Blanchette1"></a>Blanchette M, Kent WJ, Riemer C, Elnitski L, Smit AF, et al.  (2004) Aligning multiple genomic sequences with the threaded blockset aligner. Genome Res  14: 708–715.  <ul class="find" data-citedArticleID="929592" data-doi="10.1101/gr.1933104"><li><a href="http://dx.doi.org/10.1101/gr.1933104" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Aligning+multiple+genomic+sequences+with+the+threaded+blockset+aligner." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Aligning+multiple+genomic+sequences+with+the+threaded+blockset+aligner.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0000203-Siepel1" id="pone.0000203-Siepel1"></a>Siepel A, Bejerano G, Pedersen JS, Hinrichs AS, Hou M, et al.  (2005) Evolutionarily conserved elements in vertebrate, insect, worm, and yeast genomes. Genome Res  15: 1034–1050.  <ul class="find" data-citedArticleID="929660" data-doi="10.1101/gr.3715005"><li><a href="http://dx.doi.org/10.1101/gr.3715005" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evolutionarily+conserved+elements+in+vertebrate%2C+insect%2C+worm%2C+and+yeast+genomes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evolutionarily+conserved+elements+in+vertebrate%2C+insect%2C+worm%2C+and+yeast+genomes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0000203-Su1" id="pone.0000203-Su1"></a>Su AI, Wiltshire T, Batalov S, Lapp H, Ching KA, et al.  (2004) A gene atlas of the mouse and human protein-encoding transcriptomes. Proc Natl Acad Sci U S A  101: 6062–6067.  <ul class="find" data-citedArticleID="929672" data-doi="10.1073/pnas.0400782101"><li><a href="http://dx.doi.org/10.1073/pnas.0400782101" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+gene+atlas+of+the+mouse+and+human+protein-encoding+transcriptomes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+gene+atlas+of+the+mouse+and+human+protein-encoding+transcriptomes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0000203-Sturn1" id="pone.0000203-Sturn1"></a>Sturn A, Quackenbush J, Trajanoski Z (2002) Genesis: cluster analysis of microarray data. Bioinformatics  18: 207–208.  <ul class="find" data-citedArticleID="929670" data-doi="10.1093/bioinformatics/18.1.207"><li><a href="http://dx.doi.org/10.1093/bioinformatics/18.1.207" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genesis%3A+cluster+analysis+of+microarray+data." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genesis%3A+cluster+analysis+of+microarray+data.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">36.
              </span><a name="pone.0000203-Zhang1" id="pone.0000203-Zhang1"></a>Zhang B, Schmoyer D, Kirov S, Snoddy J (2004) GOTree Machine (GOTM): a web-based platform for interpreting sets of interesting genes using Gene Ontology hierarchies. BMC Bioinformatics  5: 16.  <ul class="find" data-citedArticleID="929684"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=GOTree+Machine+%28GOTM%29%3A+a+web-based+platform+for+interpreting+sets+of+interesting+genes+using+Gene+Ontology+hierarchies.&amp;auth=&amp;atitle=GOTree+Machine+%28GOTM%29%3A+a+web-based+platform+for+interpreting+sets+of+interesting+genes+using+Gene+Ontology+hierarchies." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=GOTree+Machine+%28GOTM%29%3A+a+web-based+platform+for+interpreting+sets+of+interesting+genes+using+Gene+Ontology+hierarchies." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22GOTree+Machine+%28GOTM%29%3A+a+web-based+platform+for+interpreting+sets+of+interesting+genes+using+Gene+Ontology+hierarchies.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">37.
              </span><a name="pone.0000203-Ashburner1" id="pone.0000203-Ashburner1"></a>Ashburner M, Ball CA, Blake JA, Botstein D, Butler H, et al.  (2000) Gene ontology: tool for the unification of biology. The Gene Ontology Consortium. Nat Genet  25: 25–29.  <ul class="find" data-citedArticleID="929584"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Gene+ontology%3A+tool+for+the+unification+of+biology.+The+Gene+Ontology+Consortium.&amp;auth=&amp;atitle=Gene+ontology%3A+tool+for+the+unification+of+biology.+The+Gene+Ontology+Consortium." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Gene+ontology%3A+tool+for+the+unification+of+biology.+The+Gene+Ontology+Consortium." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Gene+ontology%3A+tool+for+the+unification+of+biology.+The+Gene+Ontology+Consortium.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">38.
              </span><a name="pone.0000203-Morgan1" id="pone.0000203-Morgan1"></a>Morgan GT (1995) Identification in the human genome of mobile elements spread by DNA-mediated transposition. J Mol Biol  254: 1–5.  <ul class="find" data-citedArticleID="929652" data-doi="10.1006/jmbi.1995.0593"><li><a href="http://dx.doi.org/10.1006/jmbi.1995.0593" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Identification+in+the+human+genome+of+mobile+elements+spread+by+DNA-mediated+transposition." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Identification+in+the+human+genome+of+mobile+elements+spread+by+DNA-mediated+transposition.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">39.
              </span><a name="pone.0000203-Oosumi1" id="pone.0000203-Oosumi1"></a>Oosumi T, Belknap WR, Garlick B (1995) Mariner transposons in humans. Nature  378: 672.  <ul class="find" data-citedArticleID="929654" data-doi="10.1038/378672a0"><li><a href="http://dx.doi.org/10.1038/378672a0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mariner+transposons+in+humans." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mariner+transposons+in+humans.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">40.
              </span><a name="pone.0000203-Smit2" id="pone.0000203-Smit2"></a>Smit AF, Riggs AD (1996) Tiggers and DNA transposon fossils in the human genome. Proc Natl Acad Sci U S A  93: 1443–1448.  <ul class="find" data-citedArticleID="929668" data-doi="10.1073/pnas.93.4.1443"><li><a href="http://dx.doi.org/10.1073/pnas.93.4.1443" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Tiggers+and+DNA+transposon+fossils+in+the+human+genome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Tiggers+and+DNA+transposon+fossils+in+the+human+genome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">41.
              </span><a name="pone.0000203-Robertson1" id="pone.0000203-Robertson1"></a>Robertson HM, Zumpano KL (1997) Molecular evolution of an ancient mariner transposon, Hsmar1, in the human genome. Gene  205: 203–217.  <ul class="find" data-citedArticleID="929656" data-doi="10.1016/s0378-1119(97)00472-1"><li><a href="http://dx.doi.org/10.1016/s0378-1119(97)00472-1" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Molecular+evolution+of+an+ancient+mariner+transposon%2C+Hsmar1%2C+in+the+human+genome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Molecular+evolution+of+an+ancient+mariner+transposon%2C+Hsmar1%2C+in+the+human+genome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">42.
              </span><a name="pone.0000203-Babak1" id="pone.0000203-Babak1"></a>Babak T, Zhang W, Morris Q, Blencowe BJ, Hughes TR (2004) Probing microRNAs with microarrays: tissue specificity and functional inference. Rna  10: 1813–1819.  <ul class="find" data-citedArticleID="929586" data-doi="10.1261/rna.7119904"><li><a href="http://dx.doi.org/10.1261/rna.7119904" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Probing+microRNAs+with+microarrays%3A+tissue+specificity+and+functional+inference." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Probing+microRNAs+with+microarrays%3A+tissue+specificity+and+functional+inference.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">43.
              </span><a name="pone.0000203-Huang1" id="pone.0000203-Huang1"></a>Huang JC, Morris QD, Frey BJ (2006) Detecting microRNA targets by linking sequence, microRNA and gene expression data.  Apostolico A, Guerra C, Istrail S, Pevzner PA, Waterman MS, editors. Venice, Italy: Springer-Verlag. pp. 114–129.  <ul class="find-nolinks"></ul></li><li><span class="label">44.
              </span><a name="pone.0000203-Shih1" id="pone.0000203-Shih1"></a>Shih W, Chetty R, Tsao MS (2005) Expression profiling by microarrays in colorectal cancer (Review). Oncol Rep  13: 517–524.  <ul class="find" data-citedArticleID="929658" data-doi="10.3892/or.13.3.517"><li><a href="http://dx.doi.org/10.3892/or.13.3.517" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Expression+profiling+by+microarrays+in+colorectal+cancer+%28Review%29." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Expression+profiling+by+microarrays+in+colorectal+cancer+%28Review%29.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">45.
              </span><a name="pone.0000203-Lander1" id="pone.0000203-Lander1"></a>Lander ES, Linton LM, Birren B, Nusbaum C, Zody MC, et al.  (2001) Initial sequencing and analysis of the human genome. Nature  409: 860–921.  <ul class="find" data-citedArticleID="929628"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Initial+sequencing+and+analysis+of+the+human+genome.&amp;auth=&amp;atitle=Initial+sequencing+and+analysis+of+the+human+genome." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Initial+sequencing+and+analysis+of+the+human+genome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Initial+sequencing+and+analysis+of+the+human+genome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">46.
              </span><a name="pone.0000203-Feschotte1" id="pone.0000203-Feschotte1"></a>Feschotte C, Zhang X, Wessler SR (2002)  Miniature inverted-repeat transposable elements and their relationship to established DNA transposons. In: L. CN , Craigie R, Gellert M, Lambowitz A, editors. Mobile DNA II. Washington, , DC: ASM Press.   <ul class="find-nolinks"></ul></li><li><span class="label">47.
              </span><a name="pone.0000203-Bureau1" id="pone.0000203-Bureau1"></a>Bureau TE, Wessler SR (1992) Tourist: a large family of small inverted repeat elements frequently associated with maize genes. Plant Cell  4: 1283–1294.  <ul class="find" data-citedArticleID="929596" data-doi="10.1105/tpc.4.10.1283"><li><a href="http://dx.doi.org/10.1105/tpc.4.10.1283" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Tourist%3A+a+large+family+of+small+inverted+repeat+elements+frequently+associated+with+maize+genes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Tourist%3A+a+large+family+of+small+inverted+repeat+elements+frequently+associated+with+maize+genes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">48.
              </span><a name="pone.0000203-Jiang1" id="pone.0000203-Jiang1"></a>Jiang N, Feschotte C, Zhang X, Wessler SR (2004) Using rice to understand the origin and amplification of miniature inverted repeat transposable elements (MITEs). Curr Opin Plant Biol  7: 115–119.  <ul class="find" data-citedArticleID="929616" data-doi="10.1016/j.pbi.2004.01.004"><li><a href="http://dx.doi.org/10.1016/j.pbi.2004.01.004" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Using+rice+to+understand+the+origin+and+amplification+of+miniature+inverted+repeat+transposable+elements+%28MITEs%29." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Using+rice+to+understand+the+origin+and+amplification+of+miniature+inverted+repeat+transposable+elements+%28MITEs%29.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">49.
              </span><a name="pone.0000203-Mao1" id="pone.0000203-Mao1"></a>Mao L, Wood TC, Yu Y, Budiman MA, Tomkins J, et al.  (2000) Rice transposable elements: a survey of 73,000 sequence-tagged-connectors. Genome Res  10: 982–990.  <ul class="find" data-citedArticleID="929642" data-doi="10.1101/gr.10.7.982"><li><a href="http://dx.doi.org/10.1101/gr.10.7.982" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Rice+transposable+elements%3A+a+survey+of+73%2C000+sequence-tagged-connectors." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Rice+transposable+elements%3A+a+survey+of+73%2C000+sequence-tagged-connectors.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">50.
              </span><a name="pone.0000203-Zhang2" id="pone.0000203-Zhang2"></a>Zhang Q, Arbuckle J, Wessler SR (2000) Recent, extensive, and preferential insertion of members of the miniature inverted-repeat transposable element family Heartbreaker into genic regions of maize. Proc Natl Acad Sci U S A  97: 1160–1165.  <ul class="find" data-citedArticleID="929686" data-doi="10.1073/pnas.97.3.1160"><li><a href="http://dx.doi.org/10.1073/pnas.97.3.1160" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Recent%2C+extensive%2C+and+preferential+insertion+of+members+of+the+miniature+inverted-repeat+transposable+element+family+Heartbreaker+into+genic+regions+of+maize." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Recent%2C+extensive%2C+and+preferential+insertion+of+members+of+the+miniature+inverted-repeat+transposable+element+family+Heartbreaker+into+genic+regions+of+maize.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">51.
              </span><a name="pone.0000203-Matzke2" id="pone.0000203-Matzke2"></a>Matzke MA, Matzke AJ (2004) Planting the seeds of a new paradigm. PLoS Biol  2: E133.  <ul class="find" data-citedArticleID="929648" data-doi="10.1371/journal.pbio.0020133"><li><a href="http://dx.doi.org/10.1371/journal.pbio.0020133" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Planting+the+seeds+of+a+new+paradigm." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Planting+the+seeds+of+a+new+paradigm.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">52.
              </span><a name="pone.0000203-Yoder1" id="pone.0000203-Yoder1"></a>Yoder JA, Walsh CP, Bestor TH (1997) Cytosine methylation and the ecology of intragenomic parasites. Trends Genet  13: 335–340.  <ul class="find" data-citedArticleID="929680" data-doi="10.1016/s0168-9525(97)01181-5"><li><a href="http://dx.doi.org/10.1016/s0168-9525(97)01181-5" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Cytosine+methylation+and+the+ecology+of+intragenomic+parasites." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Cytosine+methylation+and+the+ecology+of+intragenomic+parasites.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">53.
              </span><a name="pone.0000203-McDonald1" id="pone.0000203-McDonald1"></a>McDonald JF, Matzke MA, Matzke AJ (2005) Host defenses to transposable elements and the evolution of genomic imprinting. Cytogenet Genome Res  110: 242–249.  <ul class="find" data-citedArticleID="929650" data-doi="10.1159/000084958"><li><a href="http://dx.doi.org/10.1159/000084958" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Host+defenses+to+transposable+elements+and+the+evolution+of+genomic+imprinting." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Host+defenses+to+transposable+elements+and+the+evolution+of+genomic+imprinting.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">54.
              </span><a name="pone.0000203-Lippman1" id="pone.0000203-Lippman1"></a>Lippman Z, Gendrel AV, Black M, Vaughn MW, Dedhia N, et al.  (2004) Role of transposable elements in heterochromatin and epigenetic control. Nature  430: 471–476.  <ul class="find" data-citedArticleID="929636" data-doi="10.1038/nature02651"><li><a href="http://dx.doi.org/10.1038/nature02651" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Role+of+transposable+elements+in+heterochromatin+and+epigenetic+control." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Role+of+transposable+elements+in+heterochromatin+and+epigenetic+control.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">55.
              </span><a name="pone.0000203-Aravin1" id="pone.0000203-Aravin1"></a>Aravin AA, Lagos-Quintana M, Yalcin A, Zavolan M, Marks D, et al.  (2003) The small RNA profile during Drosophila melanogaster development. Dev Cell  5: 337–350.  <ul class="find" data-citedArticleID="929582" data-doi="10.1016/s1534-5807(03)00228-4"><li><a href="http://dx.doi.org/10.1016/s1534-5807(03)00228-4" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+small+RNA+profile+during+Drosophila+melanogaster+development." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+small+RNA+profile+during+Drosophila+melanogaster+development.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">56.
              </span><a name="pone.0000203-Hamilton1" id="pone.0000203-Hamilton1"></a>Hamilton A, Voinnet O, Chappell L, Baulcombe D (2002) Two classes of short interfering RNA in RNA silencing. Embo J  21: 4671–4679.  <ul class="find" data-citedArticleID="929610" data-doi="10.1093/emboj/cdf464"><li><a href="http://dx.doi.org/10.1093/emboj/cdf464" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Two+classes+of+short+interfering+RNA+in+RNA+silencing." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Two+classes+of+short+interfering+RNA+in+RNA+silencing.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">57.
              </span><a name="pone.0000203-Lippman2" id="pone.0000203-Lippman2"></a>Lippman Z, May B, Yordan C, Singer T, Martienssen R (2003) Distinct mechanisms determine transposon inheritance and methylation via small interfering RNA and histone modification. PLoS Biol  1: E67.  <ul class="find" data-citedArticleID="929638" data-doi="10.1371/journal.pbio.0000067"><li><a href="http://dx.doi.org/10.1371/journal.pbio.0000067" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Distinct+mechanisms+determine+transposon+inheritance+and+methylation+via+small+interfering+RNA+and+histone+modification." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Distinct+mechanisms+determine+transposon+inheritance+and+methylation+via+small+interfering+RNA+and+histone+modification.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">58.
              </span><a name="pone.0000203-Zilberman1" id="pone.0000203-Zilberman1"></a>Zilberman D, Cao X, Jacobsen SE (2003) ARGONAUTE4 control of locus-specific siRNA accumulation and DNA and histone methylation. Science  299: 716–719.  <ul class="find" data-citedArticleID="929688" data-doi="10.1126/science.1079695"><li><a href="http://dx.doi.org/10.1126/science.1079695" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=ARGONAUTE4+control+of+locus-specific+siRNA+accumulation+and+DNA+and+histone+methylation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22ARGONAUTE4+control+of+locus-specific+siRNA+accumulation+and+DNA+and+histone+methylation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">59.
              </span><a name="pone.0000203-Allen1" id="pone.0000203-Allen1"></a>Allen E, Xie Z, Gustafson AM, Sung GH, Spatafora JW, et al.  (2004) Evolution of microRNA genes by inverted duplication of target gene sequences in Arabidopsis thaliana. Nat Genet  36: 1282–1290.  <ul class="find" data-citedArticleID="929574" data-doi="10.1038/ng1478"><li><a href="http://dx.doi.org/10.1038/ng1478" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evolution+of+microRNA+genes+by+inverted+duplication+of+target+gene+sequences+in+Arabidopsis+thaliana." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evolution+of+microRNA+genes+by+inverted+duplication+of+target+gene+sequences+in+Arabidopsis+thaliana.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000203.XML" value="139024"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000203.PDF" value="808739"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000203.t001.PNG_L" value="108850"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000203.t001.PNG_M" value="40773"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000203.t001.PNG_S" value="2397"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000203.t001.TIF" value="512204"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000203.t001.PNG_I" value="16673"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000203.g001.PNG_L" value="2143206"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000203.g001.PNG_M" value="71121"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000203.g001.PNG_S" value="2317"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000203.g001.TIF" value="2486366"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000203.g001.PNG_I" value="31107"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000203.g002.PNG_L" value="4630"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000203.g002.PNG_M" value="5198"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000203.g002.PNG_S" value="1661"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000203.g002.TIF" value="14906"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000203.g002.PNG_I" value="3608"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000203.g003.PNG_L" value="278355"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000203.g003.PNG_M" value="13444"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000203.g003.PNG_S" value="1636"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000203.g003.TIF" value="481980"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000203.g003.PNG_I" value="7413"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000203.g004.PNG_L" value="921923"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000203.g004.PNG_M" value="33103"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000203.g004.PNG_S" value="2288"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000203.g004.TIF" value="1835052"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000203.g004.PNG_I" value="29127"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000203.g005.PNG_L" value="2073747"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000203.g005.PNG_M" value="65822"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000203.g005.PNG_S" value="2825"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000203.g005.TIF" value="3561080"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000203.g005.PNG_I" value="43189"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000203.g006.PNG_L" value="4302754"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000203.g006.PNG_M" value="214014"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000203.g006.PNG_S" value="6375"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000203.g006.TIF" value="8126026"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000203.g006.PNG_I" value="99442"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000203.g007.PNG_L" value="8196470"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000203.g007.PNG_M" value="209004"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000203.g007.PNG_S" value="16060"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000203.g007.TIF" value="10182756"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000203.g007.PNG_I" value="165904"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000203.g008.PNG_L" value="235789"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000203.g008.PNG_M" value="15869"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000203.g008.PNG_S" value="1779"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000203.g008.TIF" value="403910"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000203.g008.PNG_I" value="7063"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000203.s001.PDF" value="453233"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000203.s002.PDF" value="57671"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000203.s003.PDF" value="3436415"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000203.s004.DOC" value="315392"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000203.s005.DOC" value="39424"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000203.s006.DOC" value="91648"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000203.s007.DOC" value="197120"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000203&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000203" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000203&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0000203&volume=&issue=&title=A Family of Human MicroRNA Genes from Miniature Inverted-Repeat Transposable Elements&author_name=Jittima%20Piriyapongsa%2C%20I.%20King%20Jordan&start_page=1&end_page=11" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000203" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000203" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0000203" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000203&amp;t=A%20Family%20of%20Human%20MicroRNA%20Genes%20from%20Miniature%20Inverted-Repeat%20Transposable%20Elements" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000203&title=A%20Family%20of%20Human%20MicroRNA%20Genes%20from%20Miniature%20Inverted-Repeat%20Transposable%20Elements&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000203&amp;title=A%20Family%20of%20Human%20MicroRNA%20Genes%20from%20Miniature%20Inverted-Repeat%20Transposable%20Elements" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000203" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0000203&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'A Family of Human MicroRNA Genes from Miniature Inverted-Repeat Transposable Elements';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0000203';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20A%20Family%20of%20Human%20MicroRNA%20Genes%20from%20Miniature%20Inverted-Repeat%20Transposable%20Elements http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000203" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0000203" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">















          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Gene+targeting%22" title="Search for articles in the subject area:'Gene targeting'"><div class="flagText">Gene targeting</div></a>
              <div data-categoryid="47601" data-articleid="23244"
                   data-categoryname="Gene targeting"
                   class="flagImage" title="Flag 'Gene targeting' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Genome+evolution%22" title="Search for articles in the subject area:'Genome evolution'"><div class="flagText">Genome evolution</div></a>
              <div data-categoryid="17207" data-articleid="23244"
                   data-categoryname="Genome evolution"
                   class="flagImage" title="Flag 'Genome evolution' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Genomic+databases%22" title="Search for articles in the subject area:'Genomic databases'"><div class="flagText">Genomic databases</div></a>
              <div data-categoryid="42727" data-articleid="23244"
                   data-categoryname="Genomic databases"
                   class="flagImage" title="Flag 'Genomic databases' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Human+genomics%22" title="Search for articles in the subject area:'Human genomics'"><div class="flagText">Human genomics</div></a>
              <div data-categoryid="17253" data-articleid="23244"
                   data-categoryname="Human genomics"
                   class="flagImage" title="Flag 'Human genomics' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Mammalian+genomics%22" title="Search for articles in the subject area:'Mammalian genomics'"><div class="flagText">Mammalian genomics</div></a>
              <div data-categoryid="17625" data-articleid="23244"
                   data-categoryname="Mammalian genomics"
                   class="flagImage" title="Flag 'Mammalian genomics' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22MicroRNA%22" title="Search for articles in the subject area:'MicroRNA'"><div class="flagText">MicroRNA</div></a>
              <div data-categoryid="19821" data-articleid="23244"
                   data-categoryname="MicroRNA"
                   class="flagImage" title="Flag 'MicroRNA' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22RNA+stem-loop+structure%22" title="Search for articles in the subject area:'RNA stem-loop structure'"><div class="flagText">RNA stem-loop structure</div></a>
              <div data-categoryid="46413" data-articleid="23244"
                   data-categoryname="RNA stem-loop structure"
                   class="flagImage" title="Flag 'RNA stem-loop structure' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Small+interfering+RNA%22" title="Search for articles in the subject area:'Small interfering RNA'"><div class="flagText">Small interfering RNA</div></a>
              <div data-categoryid="19855" data-articleid="23244"
                   data-categoryname="Small interfering RNA"
                   class="flagImage" title="Flag 'Small interfering RNA' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=3642'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=9534'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=9636&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>

<div class="block sidebar-comments">
    <div class="header">
        <h3>Comments</h3>
    </div>
      <p><a href="/annotation/listThread.action?root=4591">Plant MITE miRNAs</a><br>Posted by kjordan</p>
</div>

</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
