

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Large-Scale In Silico Mapping of Complex Quantitative Traits in Inbred Mice</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0000651"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0000651" />

    <meta name="citation_title" content="Large-Scale In Silico Mapping of Complex Quantitative Traits in Inbred Mice"/>
    <meta itemprop="name" content="Large-Scale In Silico Mapping of Complex Quantitative Traits in Inbred Mice"/>

      <meta name="citation_author" content="Pengyuan Liu"/>
            <meta name="citation_author_institution" content="Department of Surgery and the Alvin J. Siteman Cancer Center, Washington University School of Medicine, St. Louis, Missouri, United States of America"/>
      <meta name="citation_author" content="Haris Vikis"/>
            <meta name="citation_author_institution" content="Department of Surgery and the Alvin J. Siteman Cancer Center, Washington University School of Medicine, St. Louis, Missouri, United States of America"/>
      <meta name="citation_author" content="Yan Lu"/>
            <meta name="citation_author_institution" content="Department of Surgery and the Alvin J. Siteman Cancer Center, Washington University School of Medicine, St. Louis, Missouri, United States of America"/>
      <meta name="citation_author" content="Daolong Wang"/>
            <meta name="citation_author_institution" content="Department of Surgery and the Alvin J. Siteman Cancer Center, Washington University School of Medicine, St. Louis, Missouri, United States of America"/>
      <meta name="citation_author" content="Ming You"/>
            <meta name="citation_author_institution" content="Department of Surgery and the Alvin J. Siteman Cancer Center, Washington University School of Medicine, St. Louis, Missouri, United States of America"/>

    <meta name="citation_date" content="2007/7/25"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0000651.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e651"/>
    <meta name="citation_issue" content="7"/>
    <meta name="citation_volume" content="2"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Initial sequencing and comparative analysis of the mouse genome.; citation_author=RH Waterston; citation_author=K Lindblad-Toh; citation_author=E Birney; citation_author=J Rogers; citation_author=JF Abril; citation_journal_title=Nature; citation_volume=420; citation_number=1; citation_pages=520-562; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=The mouse as a model for human biology: a resource guide for complex trait analysis.; citation_author=LL Peters; citation_author=RF Robledo; citation_author=CJ Bult; citation_author=GA Churchill; citation_author=BJ Paigen; citation_journal_title=Nat Rev Genet; citation_volume=8; citation_number=2; citation_pages=58-69; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Genetic variation in laboratory mice.; citation_author=CM Wade; citation_author=MJ Daly; citation_journal_title=Nat Genet; citation_volume=37; citation_number=3; citation_pages=1175-1180; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Candidate lung tumor susceptibility genes identified through whole-genome association analyses in inbred mice.; citation_author=P Liu; citation_author=Y Wang; citation_author=H Vikis; citation_author=A Maciag; citation_author=D Wang; citation_journal_title=Nat Genet; citation_volume=38; citation_number=4; citation_pages=888-895; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=In silico genetics: identification of a functional element regulating H2-Ealpha gene expression.; citation_author=G Liao; citation_author=J Wang; citation_author=J Guo; citation_author=J Allard; citation_author=J Cheng; citation_journal_title=Science; citation_volume=306; citation_number=5; citation_pages=690-695; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=In silico mapping of complex disease-related traits in mice.; citation_author=A Grupe; citation_author=S Germer; citation_author=J Usuka; citation_author=D Aud; citation_author=JK Belknap; citation_journal_title=Science; citation_volume=292; citation_number=6; citation_pages=1915-1918; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Use of a dense single nucleotide polymorphism map for in silico mapping in the mouse.; citation_author=MT Pletcher; citation_author=P McClurg; citation_author=S Batalov; citation_author=AI Su; citation_author=SW Barnes; citation_journal_title=PLoS Biol; citation_volume=2; citation_number=7; citation_pages=e393; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Genome-wide genetic association of complex traits in heterogeneous stock mice.; citation_author=W Valdar; citation_author=LC Solberg; citation_author=D Gauguier; citation_author=S Burnett; citation_author=P Klenerman; citation_journal_title=Nat Genet; citation_volume=38; citation_number=8; citation_pages=879-887; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=A haplotype map of the human genome.; citation_author=D Altshuler; citation_author=LD Brooks; citation_author=A Chakravarti; citation_author=FS Collins; citation_author=MJ Daly; citation_journal_title=Nature; citation_volume=437; citation_number=9; citation_pages=1299-1320; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=The human obesity gene map: the 2005 update.; citation_author=T Rankinen; citation_author=A Zuberi; citation_author=YC Chagnon; citation_author=SJ Weisnagel; citation_author=G Argyropoulos; citation_journal_title=Obesity (Silver Spring); citation_volume=14; citation_number=10; citation_pages=529-644; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Quantitative trait loci for growth traits in C57BL/6J×DBA/2J mice.; citation_author=KH Morris; citation_author=A Ishikawa; citation_author=PD Keightley; citation_journal_title=Mamm Genome; citation_volume=10; citation_number=11; citation_pages=225-228; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Characterization of a QTL affecting skeletal size in mice.; citation_author=JK Christians; citation_author=VK Bingham; citation_author=FK Oliver; citation_author=TT Heath; citation_author=PD Keightley; citation_journal_title=Mamm Genome; citation_volume=14; citation_number=12; citation_pages=175-183; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Fine mapping of a murine growth locus to a 1.4-cM region and resolution of linked QTL.; citation_author=JK Christians; citation_author=PD Keightley; citation_journal_title=Mamm Genome; citation_volume=15; citation_number=13; citation_pages=482-491; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=PAPPA2, an enzyme that cleaves an insulin-like growth-factor-binding protein, is a candidate gene for a quantitative trait locus affecting body size in mice.; citation_author=JK Christians; citation_author=A Hoeflich; citation_author=PD Keightley; citation_journal_title=Genetics; citation_volume=173; citation_number=14; citation_pages=1547-1553; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Quantitative trait loci affecting body weight and fatness from a mouse line selected for extreme high growth.; citation_author=GA Brockmann; citation_author=CS Haley; citation_author=U Renne; citation_author=SA Knott; citation_author=M Schwerin; citation_journal_title=Genetics; citation_volume=150; citation_number=15; citation_pages=369-381; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Single QTL effects, epistasis, and pleiotropy account for two-thirds of the phenotypic F(2) variance of growth and obesity in DU6i×DBA/2 mice.; citation_author=GA Brockmann; citation_author=J Kratzsch; citation_author=CS Haley; citation_author=U Renne; citation_author=M Schwerin; citation_journal_title=Genome Res; citation_volume=10; citation_number=16; citation_pages=1941-1957; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Genetic architecture of adiposity in the cross of LG/J and SM/J inbred mice.; citation_author=JM Cheverud; citation_author=TT Vaughn; citation_author=LS Pletscher; citation_author=AC Peripato; citation_author=ES Adams; citation_journal_title=Mamm Genome; citation_volume=12; citation_number=17; citation_pages=3-12; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Mapping quantitative trait loci for murine growth: a closer look at genetic architecture.; citation_author=TT Vaughn; citation_author=LS Pletscher; citation_author=A Peripato; citation_author=K King-Ellison; citation_author=E Adams; citation_journal_title=Genet Res; citation_volume=74; citation_number=18; citation_pages=313-322; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Use of M1-M5 muscarinic receptor knockout mice as novel tools to delineate the physiological roles of the muscarinic cholinergic system.; citation_author=FP Bymaster; citation_author=DL McKinzie; citation_author=CC Felder; citation_author=J Wess; citation_journal_title=Neurochem Res; citation_volume=28; citation_number=19; citation_pages=437-442; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Mice lacking the M3 muscarinic acetylcholine receptor are hypophagic and lean.; citation_author=M Yamada; citation_author=T Miyakawa; citation_author=A Duttaroy; citation_author=A Yamanaka; citation_author=T Moriguchi; citation_journal_title=Nature; citation_volume=410; citation_number=20; citation_pages=207-212; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Genetic dyslipoproteinemias.; citation_author=HBJ Brewer; citation_author=S Santamarina-Fojo; citation_author=RD Shamburek; citation_number=21; citation_pages=55-83; citation_date=2005; citation_publisher=Lippincott Williams&Wilkins; " />
      <meta name="citation_reference" content="citation_title=In search of new targets for plasma high-density lipoprotein cholesterol levels: promise of human-mouse comparative genomics.; citation_author=J Rollins; citation_author=Y Chen; citation_author=B Paigen; citation_author=X Wang; citation_journal_title=Trends Cardiovasc Med; citation_volume=16; citation_number=22; citation_pages=220-234; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Dramatically decreased high density lipoprotein cholesterol, increased remnant clearance, and insulin hypersensitivity in apolipoprotein A-II knockout mice suggest a complex role for apolipoprotein A-II in atherosclerosis susceptibility.; citation_author=W Weng; citation_author=JL Breslow; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=93; citation_number=23; citation_pages=14788-14794; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Role of meltrin {alpha} (ADAM12) in obesity induced by high- fat diet.; citation_author=M Masaki; citation_author=T Kurisaki; citation_author=K Shirakawa; citation_author=A Sehara-Fujisawa; citation_journal_title=Endocrinology; citation_volume=146; citation_number=24; citation_pages=1752-1763; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Targeted expression of a dominant-negative N-cadherin in vivo delays peak bone mass and increases adipogenesis.; citation_author=CH Castro; citation_author=CS Shin; citation_author=JP Stains; citation_author=SL Cheng; citation_author=S Sheikh; citation_journal_title=J Cell Sci; citation_volume=117; citation_number=25; citation_pages=2853-2864; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Obestatin, a peptide encoded by the ghrelin gene, opposes ghrelin's effects on food intake.; citation_author=JV Zhang; citation_author=PG Ren; citation_author=O Avsian-Kretchmer; citation_author=CW Luo; citation_author=R Rauch; citation_journal_title=Science; citation_volume=310; citation_number=26; citation_pages=996-999; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Altered gastrointestinal and metabolic function in the GPR39-obestatin receptor-knockout mouse.; citation_author=D Moechars; citation_author=I Depoortere; citation_author=B Moreaux; citation_author=B de Smet; citation_author=I Goris; citation_journal_title=Gastroenterology; citation_volume=131; citation_number=27; citation_pages=1131-1141; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Comment on “Obestatin, a peptide encoded by the ghrelin gene, opposes ghrelin's effects on food intake”.; citation_author=N Chartrel; citation_author=R Alvear-Perez; citation_author=J Leprince; citation_author=X Iturrioz; citation_author=A Reaux-Le Goazigo; citation_journal_title=Science; citation_volume=315; citation_number=28; citation_pages=766; author reply 766; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=GPR39 signaling is stimulated by zinc ions but not by obestatin.; citation_author=B Holst; citation_author=KL Egerod; citation_author=E Schild; citation_author=SP Vickers; citation_author=S Cheetham; citation_journal_title=Endocrinology; citation_volume=148; citation_number=29; citation_pages=13-20; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Obestatin does not activate orphan G protein-coupled receptor GPR39.; citation_author=E Lauwers; citation_author=B Landuyt; citation_author=L Arckens; citation_author=L Schoofs; citation_author=W Luyten; citation_journal_title=Biochem Biophys Res Commun; citation_volume=351; citation_number=30; citation_pages=21-25; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Normal food intake and body weight in mice lacking the G protein-coupled receptor GPR39.; citation_author=F Tremblay; citation_author=M Perreault; citation_author=LD Klaman; citation_author=JF Tobin; citation_author=E Smith; citation_journal_title=Endocrinology; citation_volume=148; citation_number=31; citation_pages=501-506; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Positional cloning of the major quantitative trait locus underlying lung tumor susceptibility in mice.; citation_author=Z Zhang; citation_author=M Futamura; citation_author=HG Vikis; citation_author=M Wang; citation_author=J Li; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=100; citation_number=32; citation_pages=12642-12647; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Genomics of the future: identification of quantitative trait loci in the mouse.; citation_author=L Flaherty; citation_author=B Herron; citation_author=D Symula; citation_journal_title=Genome Res; citation_volume=15; citation_number=33; citation_pages=1741-1745; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=The roads from phenotypic variation to gene discovery: mutagenesis versus QTLs.; citation_author=JH Nadeau; citation_author=WN Frankel; citation_journal_title=Nat Genet; citation_volume=25; citation_number=34; citation_pages=381-384; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=An Arabidopsis Example of Association Mapping in Structured Samples.; citation_author=K Zhao; citation_author=MJ Aranzana; citation_author=S Kim; citation_author=C Lister; citation_author=C Shindo; citation_journal_title=PLoS Genet; citation_volume=3; citation_number=35; citation_pages=e4; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=A unified mixed-model method for association mapping that accounts for multiple levels of relatedness.; citation_author=J Yu; citation_author=G Pressoir; citation_author=WH Briggs; citation_author=I Vroh Bi; citation_author=M Yamasaki; citation_journal_title=Nat Genet; citation_volume=38; citation_number=36; citation_pages=203-208; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Inference of population structure using multilocus genotype data.; citation_author=JK Pritchard; citation_author=M Stephens; citation_author=P Donnelly; citation_journal_title=Genetics; citation_volume=155; citation_number=37; citation_pages=945-959; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Genomic control for association studies.; citation_author=B Devlin; citation_author=K Roeder; citation_journal_title=Biometrics; citation_volume=55; citation_number=38; citation_pages=997-1004; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=The effects of human population structure on large genetic association studies.; citation_author=J Marchini; citation_author=LR Cardon; citation_author=MS Phillips; citation_author=P Donnelly; citation_journal_title=Nat Genet; citation_volume=36; citation_number=39; citation_pages=512-517; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Prospects for Association Mapping in Classical Inbred Mouse Strains.; citation_author=BA Payseur; citation_author=M Place; citation_journal_title=Genetics; citation_volume=175; citation_number=40; citation_pages=1999-2008; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Ascertainment bias in studies of human genome-wide polymorphism.; citation_author=AG Clark; citation_author=MJ Hubisz; citation_author=CD Bustamante; citation_author=SH Williamson; citation_author=R Nielsen; citation_journal_title=Genome Res; citation_volume=15; citation_number=41; citation_pages=1496-1502; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Gametic disequilibrium measures: proceed with caution.; citation_author=PW Hedrick; citation_journal_title=Genetics; citation_volume=117; citation_number=42; citation_pages=331-341; citation_date=1987; " />
      <meta name="citation_reference" content="citation_title=An Extension of Shapiro and Wilk's W Test for Normality to Large Samples.; citation_author=P Royston; citation_journal_title=Applied Statistics; citation_volume=31; citation_number=43; citation_pages=115-124; citation_date=1982; " />
      <meta name="citation_reference" content="citation_title=Practical Statistics for Medical Research: Chapman and Hall.; citation_author=DG Altman; citation_number=44; citation_date=1991; " />
      <meta name="citation_reference" content="citation_title=An analysis of transformations.; citation_author=GEP Box; citation_author=DR Cox; citation_journal_title=Journal of Royal Statistical Society, Series B; citation_volume=26; citation_number=45; citation_pages=211-246; citation_date=1964; " />
      <meta name="citation_reference" content="citation_title=Nonparametric statistical inference.; citation_author=M Hollander; citation_author=DA Wolfe; citation_number=46; citation_pages=68-75; citation_date=1973; citation_publisher=John Wiley and Sons; " />
      <meta name="citation_reference" content="citation_title=Corticotropin-releasing factor receptor-2-deficient mice display abnormal homeostatic responses to challenges of increased dietary fat and cold.; citation_author=TL Bale; citation_author=KR Anderson; citation_author=AJ Roberts; citation_author=KF Lee; citation_author=TR Nagy; citation_journal_title=Endocrinology; citation_volume=144; citation_number=47; citation_pages=2580-2587; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Loss of cyclin-dependent kinase inhibitors produces adipocyte hyperplasia and obesity.; citation_author=A Naaz; citation_author=DR Holsberger; citation_author=GA Iwamoto; citation_author=A Nelson; citation_author=H Kiyokawa; citation_journal_title=Faseb J; citation_volume=18; citation_number=48; citation_pages=1925-1927; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Ras-GRF1 signaling is required for normal beta-cell development and glucose homeostasis.; citation_author=J Font de Mora; citation_author=LM Esteban; citation_author=DJ Burks; citation_author=A Nunez; citation_author=C Garces; citation_journal_title=Embo J; citation_volume=22; citation_number=49; citation_pages=3039-3049; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Functional in vivo interactions between JNK1 and JNK2 isoforms in obesity and insulin resistance.; citation_author=G Tuncman; citation_author=J Hirosumi; citation_author=G Solinas; citation_author=L Chang; citation_author=M Karin; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=103; citation_number=50; citation_pages=10741-10746; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Wnt10b inhibits development of white and brown adipose tissues.; citation_author=KA Longo; citation_author=WS Wright; citation_author=S Kang; citation_author=I Gerin; citation_author=SH Chiang; citation_journal_title=J Biol Chem; citation_volume=279; citation_number=51; citation_pages=35503-35509; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Mice lacking MAP kinase phosphatase-1 have enhanced MAP kinase activity and resistance to diet-induced obesity.; citation_author=JJ Wu; citation_author=RJ Roth; citation_author=EJ Anderson; citation_author=EG Hong; citation_author=MK Lee; citation_journal_title=Cell Metab; citation_volume=4; citation_number=52; citation_pages=61-73; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=CD36 deficiency increases insulin sensitivity in muscle, but induces insulin resistance in the liver in mice.; citation_author=JR Goudriaan; citation_author=VE Dahlmans; citation_author=B Teusink; citation_author=DM Ouwens; citation_author=M Febbraio; citation_journal_title=J Lipid Res; citation_volume=44; citation_number=53; citation_pages=2270-2277; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Association of cholecystokinin A receptor gene polymorphism with cholelithiasis and the molecular mechanisms of this polymorphism.; citation_author=K Miyasaka; citation_author=Y Takata; citation_author=A Funakoshi; citation_journal_title=J Gastroenterol; citation_volume=37; citation_number=54; citation_pages=102-106; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Neuropeptide Y deficiency attenuates responses to fasting and high-fat diet in obesity-prone mice.; citation_author=HR Patel; citation_author=Y Qi; citation_author=EJ Hawkins; citation_author=SM Hileman; citation_author=JK Elmquist; citation_journal_title=Diabetes; citation_volume=55; citation_number=55; citation_pages=3091-3098; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Conditional disruption of the peroxisome proliferator-activated receptor gamma gene in mice results in lowered expression of ABCA1, ABCG1, and apoE in macrophages and reduced cholesterol efflux.; citation_author=TE Akiyama; citation_author=S Sakai; citation_author=G Lambert; citation_author=CJ Nicol; citation_author=K Matsusue; citation_journal_title=Mol Cell Biol; citation_volume=22; citation_number=56; citation_pages=2607-2619; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=CCR2 modulates inflammatory and metabolic effects of high-fat feeding.; citation_author=SP Weisberg; citation_author=D Hunter; citation_author=R Huber; citation_author=J Lemieux; citation_author=S Slaymaker; citation_journal_title=J Clin Invest; citation_volume=116; citation_number=57; citation_pages=115-124; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Liver-specific deletion of acetyl-CoA carboxylase 1 reduces hepatic triglyceride accumulation without affecting glucose homeostasis.; citation_author=J Mao; citation_author=FJ DeMayo; citation_author=H Li; citation_author=L Abu-Elheiga; citation_author=Z Gu; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=103; citation_number=58; citation_pages=8552-8557; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=apo B gene knockout in mice results in embryonic lethality in homozygotes and neural tube defects, male infertility, and reduced HDL cholesterol ester and apo A-I transport rates in heterozygotes.; citation_author=LS Huang; citation_author=E Voyiaziakis; citation_author=DF Markenson; citation_author=KA Sokol; citation_author=T Hayek; citation_journal_title=J Clin Invest; citation_volume=96; citation_number=59; citation_pages=2152-2161; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Resistance to diet-induced hypercholesterolemia and gallstone formation in ACAT2-deficient mice.; citation_author=KK Buhman; citation_author=M Accad; citation_author=S Novak; citation_author=RS Choi; citation_author=JS Wong; citation_journal_title=Nat Med; citation_volume=6; citation_number=60; citation_pages=1341-1347; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Endothelial lipase is a major determinant of HDL level.; citation_author=T Ishida; citation_author=S Choi; citation_author=RK Kundu; citation_author=K Hirata; citation_author=EM Rubin; citation_journal_title=J Clin Invest; citation_volume=111; citation_number=61; citation_pages=347-355; citation_date=2003; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000651" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Large-Scale In Silico Mapping of Complex Quantitative Traits in Inbred Mice"/>
    <meta name="twitter:description" content="Understanding the genetic basis of common disease and disease-related quantitative traits will aid in the development of diagnostics and therapeutics. The processs of gene discovery can be sped up by rapid and effective integration of well-defined mouse genome and phenome data resources. We describe here an in silico gene-discovery strategy through genome-wide association (GWA) scans in inbred mice with a wide range of genetic variation. We identified 937 quantitative trait loci (QTLs) from a survey of 173 mouse phenotypes, which include models of human disease (atherosclerosis, cardiovascular disease, cancer and obesity) as well as behavioral, hematological, immunological, metabolic, and neurological traits. 67% of QTLs were refined into genomic regions &lt;0.5 Mb with &sim;40-fold increase in mapping precision as compared with classical linkage analysis. This makes for more efficient identification of the genes that underlie disease. We have identified two QTL genes, Adam12 and Cdh2, as causal genetic variants for atherogenic diet-induced obesity. Our findings demonstrate that GWA analysis in mice has the potential to resolve multiple tightly linked QTLs and achieve single-gene resolution. These high-resolution QTL data can serve as a primary resource for positional cloning and gene identification in the research community."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0000651.g006"/>

  <meta property="og:title" content="Large-Scale In Silico Mapping of Complex Quantitative Traits in Inbred Mice" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000651" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=6632'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=3161'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=2060&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0000651">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000651" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000651&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Large-Scale In Silico Mapping of Complex Quantitative Traits in Inbred Mice
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Pengyuan Liu, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Surgery and the Alvin J. Siteman Cancer Center, Washington University School of Medicine, St. Louis, Missouri, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Haris Vikis, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Surgery and the Alvin J. Siteman Cancer Center, Washington University School of Medicine, St. Louis, Missouri, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Yan Lu, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Surgery and the Alvin J. Siteman Cancer Center, Washington University School of Medicine, St. Louis, Missouri, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Daolong Wang, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Surgery and the Alvin J. Siteman Cancer Center, Washington University School of Medicine, St. Louis, Missouri, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Ming You
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:youm@wudosis.wustl.edu">youm@wudosis.wustl.edu</a></p>

                <p>Affiliation:
                  Department of Surgery and the Alvin J. Siteman Cancer Center, Washington University School of Medicine, St. Louis, Missouri, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: July 25, 2007</li>
    <li>DOI: 10.1371/journal.pone.0000651</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0000651" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0000651" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000651" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0000651" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0000651-g001" data-doi="info:doi/10.1371/journal.pone.0000651" data-uri="info:doi/10.1371/journal.pone.0000651.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000651.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000651-t001" data-doi="info:doi/10.1371/journal.pone.0000651" data-uri="info:doi/10.1371/journal.pone.0000651.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000651.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000651-g002" data-doi="info:doi/10.1371/journal.pone.0000651" data-uri="info:doi/10.1371/journal.pone.0000651.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000651.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000651-g003" data-doi="info:doi/10.1371/journal.pone.0000651" data-uri="info:doi/10.1371/journal.pone.0000651.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000651.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000651-g004" data-doi="info:doi/10.1371/journal.pone.0000651" data-uri="info:doi/10.1371/journal.pone.0000651.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000651.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000651-t002" data-doi="info:doi/10.1371/journal.pone.0000651" data-uri="info:doi/10.1371/journal.pone.0000651.t002" title="Table 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000651.t002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000651-t003" data-doi="info:doi/10.1371/journal.pone.0000651" data-uri="info:doi/10.1371/journal.pone.0000651.t003" title="Table 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000651.t003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000651-g005" data-doi="info:doi/10.1371/journal.pone.0000651" data-uri="info:doi/10.1371/journal.pone.0000651.g005" title="Figure 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000651.g005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000651-g006" data-doi="info:doi/10.1371/journal.pone.0000651" data-uri="info:doi/10.1371/journal.pone.0000651.g006" title="Figure 6">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000651.g006&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000651">Reader Comments (0)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0000651" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1"></a><p>Understanding the genetic basis of common disease and disease-related quantitative traits will aid in the development of diagnostics and therapeutics. The processs of gene discovery can be sped up by rapid and effective integration of well-defined mouse genome and phenome data resources. We describe here an <em>in silico</em> gene-discovery strategy through genome-wide association (GWA) scans in inbred mice with a wide range of genetic variation. We identified 937 quantitative trait loci (QTLs) from a survey of 173 mouse phenotypes, which include models of human disease (atherosclerosis, cardiovascular disease, cancer and obesity) as well as behavioral, hematological, immunological, metabolic, and neurological traits. 67% of QTLs were refined into genomic regions &lt;0.5 Mb with ~40-fold increase in mapping precision as compared with classical linkage analysis. This makes for more efficient identification of the genes that underlie disease. We have identified two QTL genes, <em>Adam12</em> and <em>Cdh2</em>, as causal genetic variants for atherogenic diet-induced obesity. Our findings demonstrate that GWA analysis in mice has the potential to resolve multiple tightly linked QTLs and achieve single-gene resolution. These high-resolution QTL data can serve as a primary resource for positional cloning and gene identification in the research community.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>Liu P, Vikis H, Lu Y, Wang D, You M (2007) Large-Scale In Silico Mapping of Complex Quantitative Traits in Inbred Mice. PLoS ONE 2(7):
          e651.
            doi:10.1371/journal.pone.0000651</p><p><strong>Academic Editor: </strong>Katrina Gwinn, DER Neurogenetics, National Institute of Neurological Disorders and Stroke, United States of America</p><p><strong>Received:</strong> March 28, 2007; <strong>Accepted:</strong> June 21, 2007; <strong>Published:</strong> July 25, 2007</p><p><strong>Copyright:</strong> © 2007 Liu et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>Our research was supported by grants from the US National Institutes of Health (CA099187, CA099147, ES012063 and ES013340).</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>Humans and mice last shared a common ancestor 75 million years ago. Of mouse genes, 99% have homologues in humans and 96% are in the same syntenic location. Regions of the genome that are highly conserved between the two species align well with functionality <a href="#pone.0000651-Waterston1">[1]</a>. The mouse has been a powerful force in elucidating the genetic basis of human physiology and pathophysiology <a href="#pone.0000651-Peters1">[2]</a>. Many genetic alterations identified in mouse models have a common genetic equivalent (or ortholog) in humans. In the past decades, quantitative trait locus (QTL) mapping has identified hundreds of chromosomal regions containing genes affecting cancer, diabetes, hypertension, obesity and other disease-related phenotypes in mice. Most of these complex traits show clear genetic components, yet the underlying genes remain largely elusive.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>QTL analysis using mouse models can potentially identify genes that are important for human diseases, which are often difficult to identify in human populations, due to the complexity of human genetic data. However, a major obstacle of identifying QTL genes is the difficulty of resolving these chromosomal regions (10~20 cM) into sufficiently small intervals to make positional cloning possible. Recent advances in genomic sequence analysis and SNP discovery have provided researchers with the necessary resources to explore a wide range of genetic variation in laboratory inbred mice <a href="#pone.0000651-Wade1">[3]</a>. The use of dense SNP maps in laboratory inbred mice has proven successful in the refinement of previous QTL regions and the identification of new genetic determinants of complex traits <a href="#pone.0000651-Liu1">[4]</a>–<a href="#pone.0000651-Valdar1">[8]</a>. Recently, positional cloning of a novel gene underlying a complex trait was done immediately after a genome-wide association (GWA) analysis <a href="#pone.0000651-Liu1">[4]</a>.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>A large and diverse set of traits has been systematically organized in the publicly accessible Mouse Phenome Database (MPD) (<a href="http://www.jax.org/phenome">www.jax.org/phenome</a>), housed at The Jackson Laboratory (Bar Harbor, Maine, United States). Rapid and effective integration of mouse genome and phenome data resources can dramatically speed up the process of gene discovery (<a href="#pone-0000651-g001"><strong>Figure 1</strong></a>). Here, we describe an <em>in silico</em> gene-discovery strategy through GWA scans to identify high-resolution QTLs in inbred mice. We further demonstrate that GWA analysis in mice has the potential to resolve multiple tightly linked QTLs and achieve single-gene resolution.</p>
<div class="figure" id="pone-0000651-g001"><div class="img"><a name="pone-0000651-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000651.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000651" data-uri="info:doi/10.1371/journal.pone.0000651.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000651.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000651.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000651.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000651.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000651.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000651.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000651.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000651.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000651.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Integration of genomics and phenomics approaches in mapping complex traits in mice.</span></strong></p><a id="article1.body1.sec1.fig1.caption1.p1" name="article1.body1.sec1.fig1.caption1.p1"></a><p>(A) Pulmonary adenoma susceptibility locus 1 (<em>Pas1</em>) was previously mapped to the distal region of chromosome 6 by linkage analysis in A×B and B×A recombinant inbred mice. (B) <em>In silico</em> LD mapping <em>Pas1</em> using existing mouse SNP and tumor incidence data. (C) Enhanced view of LD mapping between 144.0–146.5 Mb. (D) Physical map of the complex multigenic <em>Pas1</em> locus. The <em>Casc1</em> gene carries a functional polymorphism at codon 60 (N60S) between lung tumor susceptible strains (e.g. A/J, SWR/J, MA/MyJ and 129/J) and resistant strains (e.g. C57BL/6J, DBA/2J, C3H/HeJ and AKR/J). (E) Lung tumorigenesis of <em>Casc1</em> knockout mice. Treatment of Casc1<sup>ko/ko</sup> mice with urethane produced more than six times as many tumors per mouse than did treatment of Casc1<sup>wt-B6/wt-B6</sup> mice. The horizontal coordinates were plotted using physical distance (Mb). In panel A, the horizontal line indicates lod = 3.3, and tumor multiplicity data were used in the linkage analysis. In panel B and C, the horizontal lines indicate –log(P) = 6.2, and urethane-induced lung adenoma incidence data were used in the analysis <a href="#pone.0000651-Liu1">[4]</a>.</p>
<span>doi:10.1371/journal.pone.0000651.g001</span></div></div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3>
<h4>The mouse haplotype map</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>We first evaluated the laboratory inbred mouse haplotype map that was used in the analysis. This haplotype map was generated by a joint effort of The Broad Institute of Harvard and MIT, and the Wellcome Trust Center for Human Genetics (<a href="#pone.0000651.s001"><strong>Text S1</strong></a>). In total, 148,062 evenly distributed single-nucleotide polymorphisms (SNPs), which span the mouse genome at an average density of ~18 kb per SNP, were genotyped in classical inbred mouse strains (<a href="#pone.0000651.s004"><strong>Figure S1</strong></a>). Of these SNPs, 96.2% have minor allele frequencies greater than 0.05 and 76.7% of the inter-SNP distances are smaller than 20 kb. LD decay over the physical distance was observed where the mean <em>r<sup>2</sup></em> fell to &lt;0.5 within 2 kb and to less than 0.23 within 200 kb. To capture large-scale patterns of LD, we also measured the number of proxies, i.e. SNPs, showing a strong correlation with one or more others <a href="#pone.0000651-Altshuler1">[9]</a>, with a window size of 500 kb across the genome (<a href="#pone.0000651.s005"><strong>Figure S2</strong></a>). Using a stringent threshold of <em>r<sup>2</sup></em> = 1.0 (that is complete LD), the average number of proxies per SNP is 2.5, and one in two SNPs have five or more proxies. Of all the SNPs, 89.5% have an <em>r<sup>2</sup></em> = 1.0 with at least one other SNP. Using a looser threshold of <em>r<sup>2</sup></em>&gt;0.5, the average number of proxies is increased to 6.5, and two in three SNPs have more than five proxies. Of all the SNPs, 91.3% have an <em>r<sup>2</sup></em>&gt;0.5 with at least one other SNP. These data indicate that the mouse haplotype map provides substantial coverage of genetic variation in inbred mice and is useful for high-resolution genetic mapping studies.</p>


<h4>Analysis of 173 mouse phenotypes</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>We performed a comprehensive survey of 173 complex quantitative traits from the mouse PHENOME projects instituted by the Jackson Laboratory (<a href="http://www.jax.org/phenome">www.jax.org/phenome</a>). In most PHENOME projects, each phenotype was measured for about 40 strains from a total of 59 inbred strains of mice with a wide range of genetic variation (<a href="#pone.0000651.s006"><strong>Figure S3</strong></a> and <a href="#pone.0000651.s001"><strong>Text S1</strong></a>). The strains used for phenotyping varied slightly among different PHENOME projects. Generally, at least 10 males and 10 females from each strain were measured for each trait. These complex traits include models of human disease (such as atherosclerosis, cardiovascular disease, cancer, diabetes, obesity, and osteoporosis) as well as behavioral, biochemical, hematological, immunological, metabolic, neurological, and reproductive traits (<a href="#pone-0000651-t001"><strong>Table 1</strong></a>). Most of these 173 mouse quantitative traits are highly heritable, with a mean heritability of 0.51, and 74% of these traits have a heritability larger than 0.40 (<a href="#pone.0000651.s007"><strong>Figure S4</strong></a>). In general, cancer, taste and drinking preferences show less genetic control (<em>h<sup>2</sup></em>&lt;0.3), while body composition and plasma lipid concentrations show larger genetic control (<em>h<sup>2</sup></em>&gt;0.7). Of these mouse traits, 47% display significant phenotypic differences between males and females (P&lt;0.05), contributing to 1–31% of the phenotypic variation in the inbred mouse populations. Notably, obesity-related traits such as body weight and fat mass have larger sex effects and explain more than 20% of the phenotypic variation in inbred mice. Nevertheless, sex-specific genetic variations are generally small (less than 6%) in the inbred mouse populations.</p>
<div class="figure" id="pone-0000651-t001"><div class="img"><a name="pone-0000651-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000651.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000651" data-uri="info:doi/10.1371/journal.pone.0000651.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000651.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000651.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000651.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000651.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000651.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000651.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000651.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000651.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000651.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>Summary of the analyzed mouse traits and QTLs</span></strong></p><span>doi:10.1371/journal.pone.0000651.t001</span></div>

<h4>Identification of 937 QTLs</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>To identify the genetic basis of this large and diverse set of mouse phenotypes, we designed an <em>in silico</em> approach for gene identification through GWA scans in inbred mice. In our approach, GWA scans were implemented in an automatic processing pipeline which constitutes data retrieving, outlier detection, data preprocessing, hypothesis testing, permutation testing and QTL identification. The potential problem of population structure in inbred mice was also inspected (<a href="#pone.0000651.s008"><strong>Figure S5</strong></a> and <a href="#pone.0000651.s001"><strong>Text S1</strong></a>). Totally, we identified 937 QTLs from the survey of 173 complex quantitative traits in inbred mice. These QTLs affect almost all types of mouse phenotypes and are distributed unevenly on all of the mouse chromosomes except chromosome Y (<a href="#pone-0000651-g002"><strong>Figure 2A</strong></a>, and <a href="#pone.0000651.s002"><strong>Table S1</strong></a> and <a href="#pone.0000651.s003"><strong>S2</strong></a>). The number of QTLs per trait ranges from 0 to 38, with a mean of 5.4 (<a href="#pone-0000651-g002"><strong>Figure 2B</strong></a>). The average size of a QTL region is 0.54 Mb and 67% of the QTLs were mapped to genomic regions less than 0.5 Mb (<a href="#pone-0000651-g002"><strong>Figure 2C</strong></a>). This translates to an extraordinary ~40-fold increase in mapping precision as compared with clasical linkage analysis. In addition, about 20% of the QTLs were located within genomic regions less than 100 kb away from at least one other QTL, implying the prevalence of pleiotropic effects of genetic manipulation in mice.</p>
<div class="figure" id="pone-0000651-g002"><div class="img"><a name="pone-0000651-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000651.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000651" data-uri="info:doi/10.1371/journal.pone.0000651.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000651.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000651.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000651.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000651.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000651.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000651.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000651.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000651.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000651.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>937 QTLs identified by genome-wide association analysis.</span></strong></p><a id="article1.body1.sec2.sec3.fig1.caption1.p1" name="article1.body1.sec2.sec3.fig1.caption1.p1"></a><p>(A) Distribution of 937 QTLs on the mouse genome. The scatter plots were drawn for –log(P) against the peak of QTLs on the chromosomes. Each square represents an identified QTL. (B) The number of QTLs per trait. (C) Proportion of QTL intervals in Mb.</p>
<span>doi:10.1371/journal.pone.0000651.g002</span></div><a id="article1.body1.sec2.sec3.p2" name="article1.body1.sec2.sec3.p2"></a><p><a href="#pone.0000651.s009"><strong>Figure S6</strong></a> shows a general QTL landscape from our GWA scans of several obesity-related phenotypes, including body weight and lean mass on normal chow and high fat, high cholesterol atherogenic diets. The analyses identified a large number of QTLs, most of which contained multiple significant SNPs clustering in the same genomic region. Several QTLs affecting multiple obesity phenotypes were repeatedly mapped to the identical locations on distal chromosome 1 and proximal chromosome 13 in different datasets. Most of these QTLs are broadly concordant with those detected previously by linkage analyses which, however, normally encompassed 10–20 cM (about 20 to 40 Mb on the mouse genome) <a href="#pone.0000651-Rankinen1">[10]</a>. In some scenarios, several separated QTLs from the GWA scans were mapped to a single linkage-defined QTL region. This lack of precision is mainly due to limited recombinants in these linkage mapping populations. Our GWA scans also found several novel QTLs that do not overlap with previous linkage analysis regions (<a href="#pone.0000651.s009"><strong>Figure S6</strong></a>). For example, we fine mapped a novel obesity QTL to a region of approximately 700 Kb (33.98–34.71 Mb) on proximal chromosome 9, which covers two candidates: Kirrel3 and A130066N16Rik; and mapped a novel QTL to a region of approximately 100 Kb (56.28–56.37 Mb) on distal chromosome 19, which covers Habp2 and Nrap. These novel QTLs identified in the GWA are not unexpected since association mapping detects allelic associations at the level of the population, which is normally comprised of 30–40 genetically diverse inbred mouse strains, rather than allelic difference between two specific parental strains in linkage mapping. Furthermore, all the obesity QTL regions detected by the GWA analyses using natural inbred mouse population samples are much smaller (less than 0.6 Mb on average with the present SNP density of 18 kb per SNP) as compared with linkage analyses. This makes positional cloning feasible immediately after GWA scans without the need for further time-comsuming fine mapping with congenic strains.</p>


<h4>Dissection of multiple tightly linked QTLs</h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>GWA scans help resolve multiple, tightly linked QTLs. We again used obesity QTLs to demonstrate this potential. So far, at least 12 obesity QTLs have been mapped to distal mouse chromosome 1, and most of them overlap with each other and may be linked to the same gene (<a href="#pone-0000651-g003"><strong>Figure 3A</strong></a>). Among these, Bwtq1 was originally identified in a cross between DBA/2J and C57BL/6J <a href="#pone.0000651-Morris1">[11]</a> and later refined to an 8-cM region (~33 Mb) between D1Mit30 and D1Mit57 <a href="#pone.0000651-Christians1">[12]</a>. This QTL has a general effect on body size, affecting the length of the tail and most bones studied, as well as having a weaker effect on mass. Using congenic strains, Bwtq1 was narrowed down to a 1.4-cM region (~4 Mb), approximately the region from D1Mit451 to D1Mit219 <a href="#pone.0000651-Christians2">[13]</a>. The location of Bwtq1 was further refined to 0.91 Mb between D1Icp7 and D1Icp10 using interval-specific subcongenic strains, which led to the identification of a strong candidate, <em>Pappa2</em>, encoding an enzyme that cleaves an insulin-like growth factor binding protein (<a href="#pone-0000651-g003"><strong>Figure 3B</strong></a>) <a href="#pone.0000651-Christians3">[14]</a>. However, our GWA scans identified two closely linked QTLs within 5 Mb of each other on distal chromosome 1. One is associated with several different obesity phenotypes in three independent datasets and is located within 153.5–155.5 Mb (–log(P) = 8.2–11.4) and the other reproduced the refined Bwtq1 QTL, showing significant effects on body weight and bone mineral content (–log(P) = 7.2). These results illustrate the robustness and reproducibility of GWA scans for complex quantitative traits in inbred mice.</p>
<div class="figure" id="pone-0000651-g003"><div class="img"><a name="pone-0000651-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000651.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000651" data-uri="info:doi/10.1371/journal.pone.0000651.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000651.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000651.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000651.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000651.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000651.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000651.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000651.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000651.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000651.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Dissection of multiple tightly linked QTLs.</span></strong></p><a id="article1.body1.sec2.sec4.fig1.caption1.p1" name="article1.body1.sec2.sec4.fig1.caption1.p1"></a><p>(A) Obesity QTLs were mapped to distal mouse hromosome 1 by linkage analysis from cross-breeding experiments <a href="#pone.0000651-Rankinen1">[10]</a>. Approximately one LOD interval was plotted against its LOD at the QTL peak. The diamonds indicate QTL peaks. Each peak lists the names of the defined QTL. (B) LD mapping of several obesity-related phenotypes in distal mouse chromosome 1. LD mapping identified two closely linked QTLs on distal chromosome 1. One is located within 153.5–155.5 Mb; another reproduced the previously refined QTL, Bwtq1. The analyzed phenotypes are body weight at the start of testing (8 weeks) (Tordoff3_bw_start), calculated weight of lean tissue (14 weeks) (Tordoff3_lean_wt), body weight after 8 weeks on an atherogenic diet (Naggert1_bw_fat8), total tissue mass after 8 weeks on an atherogenic diet (Naggert1_tissuemass_fat8), weight of lean portion of tissue mass after 8 weeks on an atherogenic diet (Naggert1_leanwt_fat8), bone mineral content after 8 weeks on an atherogenic diet (Naggert1_BMC_fat8), initial body weight (7–9 weeks) (Paigen1_initbw), and final body weight after 8 weeks on an atherogenic diet (Paigen1_finalbw).</p>
<span>doi:10.1371/journal.pone.0000651.g003</span></div>

<h4>Murine knockout and/or transgenic models</h4>
<a id="article1.body1.sec2.sec5.p1" name="article1.body1.sec2.sec5.p1"></a><p>The refined region of previous QTLs was reduced, on average, to 0.54 Mb, which covers on average 6 genes on the mouse genome. Characterization of such a small number of genes in each of these QTLs can greatly accelerate the discovery of genes underlying complex traits. We further performed a detailed survey of QTLs for all obesity-related phenotypes such as body weight, fat mass, lean mass and leptin levels. Of the QTLs detected in the present study, we identified 10 candidate genes within these QTLs that, when mutated or overexpressed as transgenes in the mouse, result in phenotypes affecting body weight and adiposity. These genes include Gpr39 (G protein-coupled receptor 39), Crhr2 (corticotropin releasing hormone receptor 2), Cdkn1b (cyclin-dependent kinase inhibitor 1B), Adam12 (ADAM metallopeptidase domain 12), Rasgrf1 (RAS protein-specific guanine nucleotide-releasing factor 1), Chrm3 (cholinergic receptor, muscarinic 3), Mapk8 (mitogen activated protein kinase 8), Wnt10b (wingless related MMTV integration site 10b), Dusp1 (dual specificity phosphatase 1) and Cdh2 (cadherin 2) (<a href="#pone-0000651-t002"><strong>Table 2</strong></a>). For example, an obesity QTL on the proximal chromosome 13 has been replicated in at least four linkage studies <a href="#pone.0000651-Brockmann1">[15]</a>–<a href="#pone.0000651-Vaughn1">[18]</a>, and has not yet been narrowed down. Our GWA scans identified a refined QTL (8.0–10.0 Mb) covering the gene Chrm3 to be associated with four obesity phenotypes in three independent datasets (-log(P) = 6.7–8.6) (<a href="#pone-0000651-g004"><strong>Figure 4A</strong></a>). Chrm3 is a member of the muscarinic acetylcholine receptors that play critical roles in regulating the activity of many important functions of the central and peripheral nervous system <a href="#pone.0000651-Bymaster1">[19]</a>. Chrm3 knockout mice had a significant reduction in food intake, body weight (25%), and peripheral fat deposits. The lack of Chrm3 also led to pronounced reductions (~5–10-fold) in serum leptin and insulin levels <a href="#pone.0000651-Yamada1">[20]</a>.</p>
<div class="figure" id="pone-0000651-g004"><div class="img"><a name="pone-0000651-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000651.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000651" data-uri="info:doi/10.1371/journal.pone.0000651.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000651.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000651.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000651.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000651.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000651.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000651.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000651.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000651.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000651.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>Fine mapping of Chrm3 and Apoa2.</span></strong></p><a id="article1.body1.sec2.sec5.fig1.caption1.p1" name="article1.body1.sec2.sec5.fig1.caption1.p1"></a><p>(A) Chrm3 is associated with initial body weight (7–9 weeks), day 0 of an atherogenic diet (Paigen1_initbw), final body weight after 8 weeks on atherogenic diet (Paigen1_finalbw), calculated weight of lean tissue (14 weeks) (Tordoff3_lean_wt), and body weight after 8 weeks on an atherogenic diet (Naggert1_bw_fat8) in the GWA scan. The Chrm3 region has been replicated in at least four obesity linkage studies. (B) Apoa2 is associated with percent of total plasma cholesterol in the HDL fraction after 8 weeks on an atherogenic diet (Paigen1_ pctHDL-TCH) in the GWA scan. The Apoa2 region has been repeatedly detected by linkage analyses in 12 different genetic crosses. Relevant phenotypes for these two genes were already observed in murine knockout models. The horizontal gray lines indicate genome-wide empirical thresholds.</p>
<span>doi:10.1371/journal.pone.0000651.g004</span></div><div class="figure" id="pone-0000651-t002"><div class="img"><a name="pone-0000651-t002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000651.t002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000651" data-uri="info:doi/10.1371/journal.pone.0000651.t002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000651.t002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000651.t002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000651.t002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000651.t002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000651.t002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000651.t002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000651.t002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000651.t002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000651.t002.TIF"></span>)
                </a></li></ul></div><p><strong>Table 2.  <span>Candidate genes for obesity validated in murine knockout and/or transgenic models</span></strong></p><span>doi:10.1371/journal.pone.0000651.t002</span></div><a id="article1.body1.sec2.sec5.p2" name="article1.body1.sec2.sec5.p2"></a><p>Similarly, we performed a survey of QTLs for plasma lipid levels which are correlated with the risk of atherosclerotic coronary heart disease <a href="#pone.0000651-Brewer1">[21]</a>. We identified 11 candidate genes in the refined regions of QTLs affecting levels of high density lipoprotein (HDL) cholesterol, low density lipoprotein (LDL) cholesterol, and triglycerides. These genes are Gpr39 (G protein-coupled receptor 39), Apoa2 (apolipoprotein A-II), Cd36 (CD36 antigen), Cckar (cholecystokinin A receptor), Npy (neuropeptide Y), Pparg (peroxisome proliferator activated receptor gamma), Ccr2 (chemokine (C-C motif) receptor 2), Acaca (acetyl-Coenzyme A carboxylase alpha), Apob (apolipoprotein B), Soat2 (sterol O-acyltransferase 2) and Lipg (endothelial lipase) (<a href="#pone-0000651-t003"><strong>Table 3</strong></a>). All of these genes were observed to have strong involvement in plasma lipid phenotypes in murine knockout and/or transgenic models. Notably, we identified a QTL containing Apoa2 on chromosome 1 located at 173.0–173.9 Mb, which affects the percent of total plasma cholesterol in the HDL fraction after 8 weeks on an atherogenic diet (-log(P) = 7.9) (<a href="#pone-0000651-g004"><strong>Figure 4B</strong></a>). Apoa2 is located within a strong HDL QTL, hdlq5, which has been repeatedly detected on distal chromosome 1 by linkage analyses in 12 different genetic crosses <a href="#pone.0000651-Rollins1">[22]</a>. Homozygous Apoa2 knockout mice had 67% and 52% reductions in HDL cholesterol levels in the fasted and fed states, respectively, and HDL particle size was also reduced <a href="#pone.0000651-Weng1">[23]</a>.</p>
<div class="figure" id="pone-0000651-t003"><div class="img"><a name="pone-0000651-t003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000651.t003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000651" data-uri="info:doi/10.1371/journal.pone.0000651.t003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000651.t003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000651.t003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000651.t003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000651.t003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000651.t003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000651.t003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000651.t003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000651.t003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000651.t003.TIF"></span>)
                </a></li></ul></div><p><strong>Table 3.  <span>Candidate genes for plasma lipid levels validated in murine knockout and/or transgenic models</span></strong></p><span>doi:10.1371/journal.pone.0000651.t003</span></div>

<h4>Adam12 and Cdh2 mapped with single-gene resolution by GWA scans</h4>
<a id="article1.body1.sec2.sec6.p1" name="article1.body1.sec2.sec6.p1"></a><p>GWA scans have the potential to achieve single-gene resolution, depending on local LD patterns and gene density on the mouse genome. A closer view of the above 10 obesity candidate-gene regions (<a href="#pone-0000651-t002"><strong>Table 2</strong></a>) further identified two QTLs that were mapped with single-gene resolution. One QTL significantly affects the percent of fat mass after 8 weeks on an atherogenic diet (-log(P) = 6.5) and is located on chromosome 7 spanning less than 100 kb where Adam12 is the only candidate gene (<a href="#pone-0000651-g005"><strong>Figure 5A</strong></a>). Meltrin alpha (Adam12) is a metalloprotease-disintegrin that belongs to the ADAM (for “a disintegrin and metalloprotease”) family. Compared with wild-type mice, meltrin alpha(-/-) mice displayed moderate resistance to weight gain induced by a high-fat diet, mainly because of an increase in the number of adipocytes <a href="#pone.0000651-Masaki1">[24]</a>. Another QTL is significantly associated with mouse liver weight after 8 weeks on an atherogenic diet (-log(P) = 8.2) and is located on chromosome 18, spanning less than 100 kb, where Cdh2 is the only candidate gene (<a href="#pone-0000651-g005"><strong>Figure 5B</strong></a>). N-cadherin (Cdh2) is a classical cadherin from the cadherin superfamily. Targeted expression of a dominant-negative N-cadherin in vivo delays peak bone mass and increases adipogenesis when compared with wild-type littermates <a href="#pone.0000651-Castro1">[25]</a>. Since Adam12 and Cdh2 are the only candadiate gene in each of their QTL regions and further supported by the evidence from knockout and/or transgenic mice, we can safely establish these two QTL genes as casual genetic variants for atherogenic diet-induced obesity in natural inbred mouse populations. It should be noted that evidence solely from knockout or transgenic mice is not sufficient to infer causal relationship between a particular gene and disease in natural populations since the gene that was knocked out or overexpressed in mice may be only one of the several genes involved in the same biological pathway leading to the disease, rather than the true gene that is mutated in natural populations. Only when genetic mapping data, especially from high-resolution QTL analyses that identify allelic variants associated with the disease, are incorporated with murine knockout or transgenic models can the casual relationship be firmly established.</p>
<div class="figure" id="pone-0000651-g005"><div class="img"><a name="pone-0000651-g005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000651.g005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000651" data-uri="info:doi/10.1371/journal.pone.0000651.g005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000651.g005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000651.g005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000651.g005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000651.g005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000651.g005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000651.g005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000651.g005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000651.g005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000651.g005.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 5.  <span>Adam12 and Cdh2 were mapped to single-gene resolution.</span></strong></p><a id="article1.body1.sec2.sec6.fig1.caption1.p1" name="article1.body1.sec2.sec6.fig1.caption1.p1"></a><p>(A) Adam12 is associated with the percent of fat mass after 8 weeks on an atherogenic diet (Naggert1_pctfat_fat8). (B) Cdh2 is associated with liver weight after 8 weeks on an atherogenic diet (Paigen1_liverweight). Relevant phenotypes for these two genes were already observed in murine knockout or transgenic models. In each subfigure, the upper panel is the enhanced view of LD mapping in a short region around the LD peak, and the lower panel is the physical map of the associated refined QTL. Genes shown on the upper side of the chromosome (turquoise lines) are transcribed in the–orientation (from right to left), and those on the lower side (pink lines) in the+orientation (from left to right). The horizontal gray lines indicate genome-wide empirical thresholds. Physical position was based on the latest NCBI mouse genome build 36.1.</p>
<span>doi:10.1371/journal.pone.0000651.g005</span></div>

<h4>Gpr39 mapped in a pleiotropic QTL</h4>
<a id="article1.body1.sec2.sec7.p1" name="article1.body1.sec2.sec7.p1"></a><p>Among these 21 candidate genes (<a href="#pone-0000651-t002"><strong>Table 2</strong></a> and <a href="#pone-0000651-t003"><strong>3</strong></a>), Gpr39 has knockout phenotypes on both obesity and plasma lipid levels. Our GWA scans identified a QTL on chromosome 1 affecting percent fat mass on atherogenic diet (-log(P) = 6.1). This QTL colocalizes with another QTL identified by our GWA scans, which affects total cholesterol and non-HDL cholesterol levels, and fold change in total cholesterol levels after 17 weeks on atherogenic diet (-log(P) = 6.6–9.9) (<a href="#pone-0000651-g006"><strong>Figure 6</strong></a>). The genomic locations of these two overlapped QTLs are indistinguishable which are very likely controlled by the same gene with pleiotropic effects on obesity and plasma lipid levels. The critical region of this pleiotropic QTL spans about 0.7 Mb (127.2–127.9 Mb) and covers six annotated genes based on the latest NCBI mouse genome build 36.1, including Actr3, Slc35f5, Gpr39, LOC667043, Lypd1 and E030049G20Rik. Gpr39 is a member of a family that includes the receptors for ghrelin and motilin <a href="#pone.0000651-Zhang1">[26]</a>. Mature body weight and percent fat mass were increased by 38% and 5%, respectively, in Gpr39 loss-of-function (Gpr39<sup>−/−</sup>) mice; cholesterol levels were increased by 11% and 42% in male and female Gpr39<sup>−/−</sup> mice, respectively <a href="#pone.0000651-Moechars1">[27]</a>. This establishes Gpr39 as a candidate for this pleiotropic QTL affecting both obesity and plasma lipid levels, even in light of the fact of recent observations clouding its role <a href="#pone.0000651-Chartrel1">[28]</a>–<a href="#pone.0000651-Tremblay1">[31]</a>. Nevertheless, these remarkable phenotypic differences between Gpr39<sup>−/−</sup> and wild-type mice mentioned above were only observed at old age (about 50 weeks for obesity and 25 weeks for cholesterol) <a href="#pone.0000651-Moechars1">[27]</a>, suggesting that deleterious mutations in Gpr39 may increase the risk for late-onset obesity and atherosclerotic coronary heart disease.</p>
<div class="figure" id="pone-0000651-g006"><div class="img"><a name="pone-0000651-g006" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000651.g006&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000651" data-uri="info:doi/10.1371/journal.pone.0000651.g006"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000651.g006&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000651.g006/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000651.g006/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000651.g006/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000651.g006/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000651.g006.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000651.g006/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000651.g006/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000651.g006.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 6.  <span>Gpr39 was mapped in a pleiotropic QTL.</span></strong></p><a id="article1.body1.sec2.sec7.fig1.caption1.p1" name="article1.body1.sec2.sec7.fig1.caption1.p1"></a><p>Gpr39 is associated with percent of tissue mass that is fat after 8 weeks on atherogenic diet (Naggert1_pctfat_fat8), total cholesterol after 17 weeks on atherogenic diet (Paigen2_CHOL_fat17), fold change in total cholesterol after 17 weeks on atherogenic diet (Paigen2_CHOL_chg), and non-HDL cholesterol after 17 wks on atherogenic diet (Paigen2_nonHDL_fat17). Gpr39 has knockout phenotypes on both obesity and blood cholesterol levels.</p>
<span>doi:10.1371/journal.pone.0000651.g006</span></div>
</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p>We have demonstrated the robustness and reproducibility of GWA analysis by comparing QTLs derived from GWA with those derived from previous linkage analyses in inbred mice. In a recent GWA analysis of lung tumor incidence, we reproduced the pulmonary adenoma susceptibility 1 (Pas1) locus identified in previous linkage studies and further narrowed this QTL to a region of less than 0.5 Mb in which at least two genes, Kras2 and Casc1, are strong candidates <a href="#pone.0000651-Liu1">[4]</a>. The refined region is completely coincident with that from traditional fine mapping by using congenic strains of mice <a href="#pone.0000651-Zhang2">[32]</a>. Casc1 knockout mouse tumor bioassays further confirmed Casc1 as a primary candidate for the <em>Pas1</em> locus <a href="#pone.0000651-Liu1">[4]</a>. In the present study, we again reproduced and narrowed the Bwtq1 locus to a region of less than 1.0 Mb where Pappa2 is a primary candidate. The refined Bwtq1 region is identical to that achieved by developing a series of congenic mice and interval-specific subcongenic mice <a href="#pone.0000651-Christians2">[13]</a>, <a href="#pone.0000651-Christians3">[14]</a>. These results demonstrate that GWA scans using dense SNP maps in laboratory mice are a powerful tool for the refinement of previously identified QTL regions. These results can be also served as an internal control for the methods that we used in our GWA scans. Given that known refined QTLs were exactly reproduced in the analyses, the newly refined QTLs identified by our GWA scans are expected to be a useful resource for further positional cloning and gene discovery in the mouse. It should be also noted that 937 QTLs identified in our study represent about one half of all QTLs derived from linkage analyses in mouse cross-breeding experiments in the past decades (<a href="http://www.informatics.jax.org">http://www.informatics.jax.org</a>).</p>
<a id="article1.body1.sec3.p2" name="article1.body1.sec3.p2"></a><p>Research tools such as genetrap mutations, transgenes, and standard knockout technology can be used to verify causal variants from highly-refined QTL regions immediately after GWA scans <a href="#pone.0000651-Flaherty1">[33]</a>, <a href="#pone.0000651-Nadeau1">[34]</a>. Using resources from murine gene-deficiency and/or transgenic models, we have identified 10 candidate genes affecting obesity-related phenotypes and 11 candidate genes affecting plasma cholesterol levels from a survey of several highly refined QTLs (<a href="#pone-0000651-t002"><strong>Table 2</strong></a> and <a href="#pone-0000651-t003"><strong>3</strong></a>). These QTL genes are most likely the causal genes based on the following evidence. First, the genomic regions where these genes are located are not only highly associated with the analyzed phenotypes in the GWA scans, but are also under the linkage peak from two or more independent linkage studies with crossing-breeding experiments <a href="#pone.0000651-Rankinen1">[10]</a>, <a href="#pone.0000651-Rollins1">[22]</a>. Second, these candidates were identified from a highly refined genomic region, as narrow as 0.54 Mb (with an average of 6 genes per region), which is extremely more precise than those chosen from a much broader linkage region (about 20 Mb). Third, and more importantly, relevant phenotypes for these genes were observed in murine knockout and/or transgenic models. Notably, due to the single-gene resolution achieved by our GWA scans, we firmly establish two QTL genes, Adam12 and Cdh2, as causal variants for obesity in inbred mice. It is also worth noting that in the present study we only focused on 28 obesity and cholesterol QTLs out of 937 QTLs, which have available data from murine knockout and/or transgenic models (<a href="#pone-0000651-t002"><strong>Table 2</strong></a> and <a href="#pone-0000651-t003"><strong>3</strong></a>). Other QTL genes identified in this study are now ready to be evaluated for their functional relevance to the analyzed mouse phenotypes. We have summarized this extremely valuable QTL resource in online <a href="#pone.0000651.s002"><strong>Table S1</strong></a> and <a href="#pone.0000651.s003"><strong>S2</strong></a>; this resource can greatly facilitate positional cloning and the identification of new genetic determinants of complex traits.</p>
<a id="article1.body1.sec3.p3" name="article1.body1.sec3.p3"></a><p>Finally, several caveats for our findings should be mentioned. Firstly, selection and inbreeding play important roles in the formation of the genetic architecture of mouse inbred strains. Potential population structure may arise in these mouse strains, which inflates type I error rates and may lead to spurious associatons in the analysis <a href="#pone.0000651-Wade1">[3]</a>. Wild-derived inbred mice have divergent evolutionary histories and thus have strong potential to generate spurious assocations. There are systematic differences in obesity-related trait values between wild-derived and other inbred strains. Therefore, the population structure was carefully inspected in our association analysis, and we removed wild-derived inbred strains from the analysis when population structure was detected. Spurious associations were largely reduced in the analysis after the removal of wild-derived inbred mouse strains (<a href="#pone.0000651.s010"><strong>Figure S7</strong></a> and <a href="#pone.0000651.s001"><strong>Text S1</strong></a>). Several methods have been developed that adjust for the effect of population structure on spurious assocations <a href="#pone.0000651-Zhao1">[35]</a>–<a href="#pone.0000651-Devlin1">[38]</a>, but these may not be sufficient for the analysis of a small number of mouse inbred strains (~30–40 strains) <a href="#pone.0000651-Marchini1">[39]</a>. Secondly, we used permutations to establish stringent genome-wide thresholds for declaring significant association for each phenotype. Although most genomic regions show correct type I error rates, some regions may show increased error rates due to unequal relatedness among the strains <a href="#pone.0000651-Payseur1">[40]</a>. The risk of these spurious associations can be alleviated when incorporating prior linkage evidence into the analysis. The identified QTL genes located within previous linkages that have been replicated in two or more independent studies should be prioritized for further investigation. The associated QTL alleles need to be checked to see if they are segregated in linkage mapping populations. We have demonstrated this strategy in a recent study in which the positional cloning of a novel QTL gene, from a previous linkage-defined region, was done immediately after a GWA analysis <a href="#pone.0000651-Liu1">[4]</a>. Thirdly, we estimated power of association analysis using classical inbred mouse strains through simulation studies (<a href="#pone.0000651.s011"><strong>Figure S8</strong></a> and <a href="#pone.0000651.s001"><strong>Text S1</strong></a>). GWA scans have reasonable power to detect QTL genes with major effects, while are limited to detect QTL genes with moderate effects. However, focusing association analysis on linkage-defined regions can dramatically increase statistical power and has sufficient power to detect moderate-effect QTL genes. In our results, the great majority of the QTLs identified in the GWA scans overlapped with previous linkage-defined regions from mouse crosses which were retrieved from the Mouse Genome Informatics (MGI) at <a href="http://www.informatics.jax.org">http://www.informatics.jax.org</a>. Nevertheless, association results for small-effect QTLs from non-linkage regions should be interpreted with caution. It should also be noted that statistical power of association analysis in inbred mice can be significantly improved by increasing the number of mice per strain used in phenotype measurement. Fourthly, most SNPs were discovered by comparison of the genomes of several classical inbred laboratory mouse strains (such as C57BL/6J, DBA/2J, A/J and 129S1/SvImJ). Very few SNPs show polymorphisms among wild-derived strains. The ascertainment bias of SNPs will affect population inference including linkage disequilibrium and population structure. This ascertainment bias will also likely erode the power of tests of association between genotype and phenotype. However it is unlikely that the ascertainment bias will introduce false-positive inferences <a href="#pone.0000651-Clark1">[41]</a>.</p>
</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Methods"></a><h3>Methods</h3>
<h4>Mouse phenotypes and SNP data</h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p>Mouse phenotypes were generated by the mouse PHENOME projects (<a href="http://phenome.jax.org/pub-cgi/phenome/mpdcgi?">http://phenome.jax.org/pub-cgi/phenome/m​pdcgi?</a>rtn = docs/home). In most PHENOME projects, each phenotype was measured for about 40 strains from a total of 59 inbred strains of mice. The strains used for phenotyping varied slightly among different projects. Generally, at least 10 males and 10 females from each strain were measured for each phenotype when 10–14 weeks of age. Before statistical analysis was conducted, the phenotypic data were subjected to detection of outliers using box plot (<a href="http://www.itl.nist.gov/div898/handbook/">http://www.itl.nist.gov/div898/handbook/</a>) (<a href="#pone.0000651.s001"><strong>Text S1</strong></a>). The SNP data were obtained from the Wellcome Trust Centre for Human Genetics (WTCHG) at Oxford University (<a href="http://www.well.ox.ac.uk/mouse/INBREDS">http://www.well.ox.ac.uk/mouse/INBREDS</a>) and the Broad Institute of Harvard and MIT (<a href="http://www.broad.mit.edu/personal/claire/MouseHapMap/Inbred.htm">http://www.broad.mit.edu/personal/claire​/MouseHapMap/Inbred.htm</a>). The genomic positions (in bp) of the SNPs for the two data sets were unified based on the latest NCBI mouse genome map build 36.1. These SNPs were further selected by removing SNPs with fewer than 20 strains typed or without genetic mapping information. The resulting SNP data include 148,062 SNPs spanning the mouse genome at an average density of ~18 kb per SNP. The SNP genotyping accuracy reported by the WTCHG and the Broad Institute is over 99.8%. It is worth noting that several strains used in the analysis only have available WTCHG SNP data. Thirty-seven additional SNPs chosen from Perlegen Sciences were genotyped to fill gaps in QTL regions covering Adam12 and Cdh2.</p>


<h4>Whole-genome LD patterns</h4>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1"></a><p>The commonly used pairwise LD measure, <em>r<sup>2</sup></em>, which is the squared correlation coefficient of alleles at two loci <a href="#pone.0000651-Hedrick1">[42]</a>, was calculated for pairwise SNPs on the mouse genome. To capture large-scale patterns of LD, we measured the number of proxies, i.e. SNPs showing a strong correlation with one or more others <a href="#pone.0000651-Altshuler1">[9]</a>, with a window size of 500 kb across the genome.</p>


<h4>GWA scans</h4>
<a id="article1.body1.sec4.sec3.p1" name="article1.body1.sec4.sec3.p1"></a><p>A full description of statistical methods for GWA scans is provided in <a href="#pone.0000651.s001"><strong>Text S1</strong></a>. Briefly, the associations of a mouse phenotype with SNP markers on the genome were tested by using linear regression models. To maximize power in GWA scans, we examined several competitive models with different variables based on Bayesian information criteria. The log-likelihood-ratio test statistic for the existence of a QTL is calculated by comparing the likelihood values under full model with null model. In the results, the negative 10-base logarithmic p value from <em>x</em><sup>2</sup> tests, i.e. –log(P), was presented. Population structure was investigated by the cumulative distribution of p values in genome-wide association analysis for each phenotype <a href="#pone.0000651-Zhao1">[35]</a>. We removed wild-derived inbred strains in the association analysis when population structure was detected.</p>
<a id="article1.body1.sec4.sec3.p2" name="article1.body1.sec4.sec3.p2"></a><p>In the GWA scans, 1,000 permutations were used to establish a genome-wide threshold (a global p value of 0.05) for declaring significant associations for each phenotype. Specifically, the analyzed phenotype was randomly reshuffled among subjects while fixing the genotypes. For each of the 1,000 permutations, the GWA scan was repeated, and the most significant –log(P) was recorded. Sorting the maximum –log(P) from large to small, the 5% quantile of the empirical distribution was taken as the genome-wide threshold (a global p value of 0.05). Since LD extends a very short genomic region (normally less than a few hundred kb) on the mouse genome, once a significant SNP is identified, a causal variant responsible for the QTL should be close to the identified SNP. To define approximate QTL regions, we first identified significant SNPs with an established empirical threshold. If two significant SNPs were less than 1,000 kb from each other, they were treated as one association signal. Then these genomic regions were extended to cover closely linked SNPs that showed suggestive associations with the analyzed traits. We also reported p values, physical positions and genomic domains for each significant SNP. The identified QTLs were compared with those detected by previous linkage analyses from mouse crosses which were retrieved from Mouse Genome Informatics (MGI) at <a href="http://www.informatics.jax.org">http://www.informatics.jax.org</a>.</p>


<h4>Data transformation</h4>
<a id="article1.body1.sec4.sec4.p1" name="article1.body1.sec4.sec4.p1"></a><p>To avoid the violation of assumption of normal distribution in the above statistical analysis, 173 quantitative phenotypes were checked for their normality by Shapiro-Wilk tests <a href="#pone.0000651-Royston1">[43]</a> before GWA scans. Phenotypes with Shapiro-Wilk tests P&lt;0.05 were converted to approach normality by Box-Cox transformation or normal scores <a href="#pone.0000651-Altman1">[44]</a>, <a href="#pone.0000651-Box1">[45]</a>. For those phenotypes still showing marked departure from the normal distribution (p&lt;0.01) after data transformation, the Mann-Whitney rank tests were performed in the GWA scans <a href="#pone.0000651-Hollander1">[46]</a>.</p>


<h4>Heritability</h4>
<a id="article1.body1.sec4.sec5.p1" name="article1.body1.sec4.sec5.p1"></a><p>The heritability of inbred strains can be estimated by linear regression models with sex, strain and sex-strain interaction as independent variables. Each strain was treated as a categorical variable in the linear model analyses. The difference between the adjusted <em>R<sup>2</sup></em> of the regression models with and without strain and its associated sex-strain interaction is approximately equal to the percentage of genetic variation (that is, heritability) in inbred mouse populations. Similarly, the phenotypic variation due to sex differences can also be estimated.</p>


<h4>Phylogenetic analysis</h4>
<a id="article1.body1.sec4.sec6.p1" name="article1.body1.sec4.sec6.p1"></a><p>The phylogenetic tree was constructed with 148,062 SNPs from 59 inbred mouse strains, implemented in the <em>dnadist</em> program in the PHYLIP 3.66 package (<a href="http://evolution.genetics.washington.edu/phylip.html">http://evolution.genetics.washington.edu​/phylip.html</a>). Branch length information was used to plot evolutionary distance between strains in the <em>drawtree</em> program in the PHYLIP 3.66 package.</p>

</div>

<div id="section5" class="section"><a id="s5" name="s5" toc="s5" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0000651"></div><a name="pone.0000651.s001" id="pone.0000651.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000651.s001">Text S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1"></a><p class="preSiDOI">Supporting methods: A detailed description of methods and materials.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000651.s001</p><a id="article1.body1.sec5.supplementary-material1.caption1.p2" name="article1.body1.sec5.supplementary-material1.caption1.p2"></a><p class="postSiDOI">(0.11 MB DOC)</p>
<a name="pone.0000651.s002" id="pone.0000651.s002"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000651.s002">Table S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material2.caption1.p1" name="article1.body1.sec5.supplementary-material2.caption1.p1"></a><p class="preSiDOI">173 mouse quantitative traits analyzed in genome-wide association analyses. The table gives empirical genome-wide threshold, the number of QTLs and significant SNPs detected in GWA scans for each trait. The table also lists project names, mouse phenome database (MPD) accession numbers, short and full name of traits, and categories of traits.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000651.s002</p><a id="article1.body1.sec5.supplementary-material2.caption1.p2" name="article1.body1.sec5.supplementary-material2.caption1.p2"></a><p class="postSiDOI">(0.36 MB DOC)</p>
<a name="pone.0000651.s003" id="pone.0000651.s003"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000651.s003">Table S2. </a></strong></p><a id="article1.body1.sec5.supplementary-material3.caption1.p1" name="article1.body1.sec5.supplementary-material3.caption1.p1"></a><p class="preSiDOI">Summary of 937 QTL identified in genome-wide association analyses. The table gives the chromosome, position of QTL peak (bp, based on the NCBI genome build 36), QTL interval, and -log (P) for each QTL identified in GWA scans. The table also provides the SNP ID of the most significant SNP and its dbSNP's annotation if available, and total number of significant and suggestive SNPs for each QTL.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000651.s003</p><a id="article1.body1.sec5.supplementary-material3.caption1.p2" name="article1.body1.sec5.supplementary-material3.caption1.p2"></a><p class="postSiDOI">(1.99 MB DOC)</p>
<a name="pone.0000651.s004" id="pone.0000651.s004"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000651.s004">Figure S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material4.caption1.p1" name="article1.body1.sec5.supplementary-material4.caption1.p1"></a><p class="preSiDOI">Distribution of SNP positions and LD structure across the mouse genome. For each chromosome, the top panel shows number of SNPs in a window size of 500 kb. The bottom panel shows number of proxies (red lines for rˆ2&gt;0.5 and blue lines for rˆ2&gt;0.8) per SNP in a window size of 500 kb.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000651.s004</p><a id="article1.body1.sec5.supplementary-material4.caption1.p2" name="article1.body1.sec5.supplementary-material4.caption1.p2"></a><p class="postSiDOI">(0.41 MB PDF)</p>
<a name="pone.0000651.s005" id="pone.0000651.s005"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000651.s005">Figure S2. </a></strong></p><a id="article1.body1.sec5.supplementary-material5.caption1.p1" name="article1.body1.sec5.supplementary-material5.caption1.p1"></a><p class="preSiDOI">Characteristics of the mouse SNP map. (A) Distribution of SNP allele frequency. (B) Distribution of inter-SNP distances. (C) LD decay as a function of physical distance. (D) Number of proxies per SNP in a window size of 500 kb, as a function of the threshold for correlation (rˆ2).</p>
<p class="siDoi">doi:10.1371/journal.pone.0000651.s005</p><a id="article1.body1.sec5.supplementary-material5.caption1.p2" name="article1.body1.sec5.supplementary-material5.caption1.p2"></a><p class="postSiDOI">(0.90 MB PDF)</p>
<a name="pone.0000651.s006" id="pone.0000651.s006"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000651.s006">Figure S3. </a></strong></p><a id="article1.body1.sec5.supplementary-material6.caption1.p1" name="article1.body1.sec5.supplementary-material6.caption1.p1"></a><p class="preSiDOI">Phylogenetic tree of 59 inbred mouse strains. The phylogenetic tree was constructed with a total of 148,062 SNPs from the WTCHG and Broad Institute, implemented in the dnadist program in the PHYLIP 3.66 package (<a href="http://evolution.genetics.washington.edu/phylip.html">http://evolution.genetics.washington.edu​/phylip.html</a>). The branch length information was used to plot evolutionary distance between strains in the drawtree program. 59 inbred mouse strains are organized into seven groups: Bagg albino derivatives, C57-related strains, Castle's mice, Japanese and New Zealand inbred strains, Little's DBA and related strains, Swiss mice, and wild-derived strains.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000651.s006</p><a id="article1.body1.sec5.supplementary-material6.caption1.p2" name="article1.body1.sec5.supplementary-material6.caption1.p2"></a><p class="postSiDOI">(1.41 MB PDF)</p>
<a name="pone.0000651.s007" id="pone.0000651.s007"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000651.s007">Figure S4. </a></strong></p><a id="article1.body1.sec5.supplementary-material7.caption1.p1" name="article1.body1.sec5.supplementary-material7.caption1.p1"></a><p class="preSiDOI">Characteristics of 173 complex quantitative traits in inbred mice. (A) Heritability of quantitative traits. (B) Phenotypic variation due to sex effects.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000651.s007</p><a id="article1.body1.sec5.supplementary-material7.caption1.p2" name="article1.body1.sec5.supplementary-material7.caption1.p2"></a><p class="postSiDOI">(0.70 MB PDF)</p>
<a name="pone.0000651.s008" id="pone.0000651.s008"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000651.s008">Figure S5. </a></strong></p><a id="article1.body1.sec5.supplementary-material8.caption1.p1" name="article1.body1.sec5.supplementary-material8.caption1.p1"></a><p class="preSiDOI">An in silico strategy for high-throughput gene discovery in inbred mice. GWA scans were implemented in an automatic processing pipeline which constitutes data retrieving, outlier detection, data preprocessing, hypothesis testing, permutations and QTL identification.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000651.s008</p><a id="article1.body1.sec5.supplementary-material8.caption1.p2" name="article1.body1.sec5.supplementary-material8.caption1.p2"></a><p class="postSiDOI">(0.21 MB PDF)</p>
<a name="pone.0000651.s009" id="pone.0000651.s009"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000651.s009">Figure S6. </a></strong></p><a id="article1.body1.sec5.supplementary-material9.caption1.p1" name="article1.body1.sec5.supplementary-material9.caption1.p1"></a><p class="preSiDOI">Genome-wide association analysis of several obesity-related phenotypes in inbred mice. The obesity-relaed phenotypes are (A) body weight at the start of testing (8 weeks) (Tordoff3_bw_start); (B) calculated weight of lean tissue (14 weeks) (Tordoff3_lean_wt); (C) body weight after 8 weeks on an atherogenic diet (Naggert1_bw_fat8); (D) total tissue mass after 8 weeks on an atherogenic diet (Naggert1_tissuemass_fat8); (E) weight of lean portion of tissue mass after 8 weeks on an atherogenic diet (Naggert1_leanwt_fat8); (F) bone mineral content after 8 weeks on an atherogenic diet (Naggert1_BMC_fat8); (G) initial body weight (7–9 weeks), day 0 of an atherogenic diet (Paigen1_initbw); and (H) final body weight after 8 weeks on an atherogenic diet (Paigen1_finalbw). In Paigen1 and Naggert1 projects, mice at 7–9 weeks of age were weighed and then administered a high fat, high cholesterol atherogenic diet. The scatter plots were drawn for -log (P) against the SNP position in the chromosomes. The horizontal gray lines indicate genome-wide empirical thresholds (global p value = 0.05). The horizontal coordinates were plotted using physical distance (Mb) in each panel.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000651.s009</p><a id="article1.body1.sec5.supplementary-material9.caption1.p2" name="article1.body1.sec5.supplementary-material9.caption1.p2"></a><p class="postSiDOI">(7.96 MB PDF)</p>
<a name="pone.0000651.s010" id="pone.0000651.s010"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000651.s010">Figure S7. </a></strong></p><a id="article1.body1.sec5.supplementary-material10.caption1.p1" name="article1.body1.sec5.supplementary-material10.caption1.p1"></a><p class="preSiDOI">Potential population structure in inbred mice. Body weight after 8 weeks on altherogenic diet (Naggert1_bw_fat8) was used to illustrate the population structure in inbred mice. (A) Cummlative distribution of p values from GWA analysis of samples with (red line) and without (blue line) wild-derived inbred strains. cdf, cumulative distribution function. (B) Comparison of GWA analysis of samples with (upper, red) and without (lower, blue) wild-derived inbred strains. The two green horizontal lines are genome-wide thresholds (a global p value of 0.05). Spurious assocaitons were largely reduced in the analysis after the removal of wild-derived inbred mouse strains.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000651.s010</p><a id="article1.body1.sec5.supplementary-material10.caption1.p2" name="article1.body1.sec5.supplementary-material10.caption1.p2"></a><p class="postSiDOI">(1.78 MB PDF)</p>
<a name="pone.0000651.s011" id="pone.0000651.s011"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000651.s011">Figure S8. </a></strong></p><a id="article1.body1.sec5.supplementary-material11.caption1.p1" name="article1.body1.sec5.supplementary-material11.caption1.p1"></a><p class="preSiDOI">Power of association analysis in inbred mice. Power was estimated for different trait heritabilities under genome-wide and QTL-wide thresholds (P = 0.05). The genome-wide threshold is used to declare a significant association on the genome without any prior genetic evidence; while the QTL-wide threshold is used to declare a significant association on a privious linkage-defined region.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000651.s011</p><a id="article1.body1.sec5.supplementary-material11.caption1.p2" name="article1.body1.sec5.supplementary-material11.caption1.p2"></a><p class="postSiDOI">(0.48 MB PDF)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>We thank the Jackson Laboratory and investigators (a detailed list of investigators of the mouse PHENOME projects is provided in the <a href="#pone.0000651.s001"><strong>Text S1</strong></a>) for collecting and phenotyping inbred mouse strain characteristics. We thank the Broad Institute of Harvard/Massachusetts General Hospital and MIT, the Wellcome Trust Center for Human Genetics at Oxford University, and Perlegen Sciences for releasing inbred laboratory mouse SNP data. The mouse phenotype and SNP data were crucial for the key findings described in this paper.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: MY PL. Performed the experiments: PL HV. Analyzed the data: PL DW YL. Wrote the paper: PL. Other: Revise the manuscript HV.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0000651-Waterston1" id="pone.0000651-Waterston1"></a>Waterston RH, Lindblad-Toh K, Birney E, Rogers J, Abril JF, et al.  (2002) Initial sequencing and comparative analysis of the mouse genome. Nature  420: 520–562.  <ul class="find" data-citedArticleID="969436"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Initial+sequencing+and+comparative+analysis+of+the+mouse+genome.&amp;auth=&amp;atitle=Initial+sequencing+and+comparative+analysis+of+the+mouse+genome." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Initial+sequencing+and+comparative+analysis+of+the+mouse+genome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Initial+sequencing+and+comparative+analysis+of+the+mouse+genome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0000651-Peters1" id="pone.0000651-Peters1"></a>Peters LL, Robledo RF, Bult CJ, Churchill GA, Paigen BJ, et al.  (2007) The mouse as a model for human biology: a resource guide for complex trait analysis. Nat Rev Genet  8: 58–69.  <ul class="find" data-citedArticleID="969414" data-doi="10.1038/nrg2025"><li><a href="http://dx.doi.org/10.1038/nrg2025" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+mouse+as+a+model+for+human+biology%3A+a+resource+guide+for+complex+trait+analysis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+mouse+as+a+model+for+human+biology%3A+a+resource+guide+for+complex+trait+analysis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0000651-Wade1" id="pone.0000651-Wade1"></a>Wade CM, Daly MJ (2005) Genetic variation in laboratory mice. Nat Genet  37: 1175–1180.  <ul class="find" data-citedArticleID="969434" data-doi="10.1038/ng1666"><li><a href="http://dx.doi.org/10.1038/ng1666" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genetic+variation+in+laboratory+mice." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genetic+variation+in+laboratory+mice.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0000651-Liu1" id="pone.0000651-Liu1"></a>Liu P, Wang Y, Vikis H, Maciag A, Wang D, et al.  (2006) Candidate lung tumor susceptibility genes identified through whole-genome association analyses in inbred mice. Nat Genet  38: 888–895.  <ul class="find" data-citedArticleID="969390" data-doi="10.1038/ng1849"><li><a href="http://dx.doi.org/10.1038/ng1849" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Candidate+lung+tumor+susceptibility+genes+identified+through+whole-genome+association+analyses+in+inbred+mice." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Candidate+lung+tumor+susceptibility+genes+identified+through+whole-genome+association+analyses+in+inbred+mice.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0000651-Liao1" id="pone.0000651-Liao1"></a>Liao G, Wang J, Guo J, Allard J, Cheng J, et al.  (2004) In silico genetics: identification of a functional element regulating H2-Ealpha gene expression. Science  306: 690–695.  <ul class="find" data-citedArticleID="969388" data-doi="10.1126/science.1100636"><li><a href="http://dx.doi.org/10.1126/science.1100636" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=In+silico+genetics%3A+identification+of+a+functional+element+regulating+H2-Ealpha+gene+expression." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22In+silico+genetics%3A+identification+of+a+functional+element+regulating+H2-Ealpha+gene+expression.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0000651-Grupe1" id="pone.0000651-Grupe1"></a>Grupe A, Germer S, Usuka J, Aud D, Belknap JK, et al.  (2001) In silico mapping of complex disease-related traits in mice. Science  292: 1915–1918.  <ul class="find" data-citedArticleID="969374" data-doi="10.1126/science.1058889"><li><a href="http://dx.doi.org/10.1126/science.1058889" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=In+silico+mapping+of+complex+disease-related+traits+in+mice." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22In+silico+mapping+of+complex+disease-related+traits+in+mice.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0000651-Pletcher1" id="pone.0000651-Pletcher1"></a>Pletcher MT, McClurg P, Batalov S, Su AI, Barnes SW, et al.  (2004) Use of a dense single nucleotide polymorphism map for in silico mapping in the mouse. PLoS Biol  2: e393.  <ul class="find" data-citedArticleID="969416" data-doi="10.1371/journal.pbio.0020393"><li><a href="http://dx.doi.org/10.1371/journal.pbio.0020393" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Use+of+a+dense+single+nucleotide+polymorphism+map+for+in+silico+mapping+in+the+mouse." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Use+of+a+dense+single+nucleotide+polymorphism+map+for+in+silico+mapping+in+the+mouse.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0000651-Valdar1" id="pone.0000651-Valdar1"></a>Valdar W, Solberg LC, Gauguier D, Burnett S, Klenerman P, et al.  (2006) Genome-wide genetic association of complex traits in heterogeneous stock mice. Nat Genet  38: 879–887.  <ul class="find" data-citedArticleID="969430" data-doi="10.1038/ng1840"><li><a href="http://dx.doi.org/10.1038/ng1840" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genome-wide+genetic+association+of+complex+traits+in+heterogeneous+stock+mice." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genome-wide+genetic+association+of+complex+traits+in+heterogeneous+stock+mice.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0000651-Altshuler1" id="pone.0000651-Altshuler1"></a>Altshuler D, Brooks LD, Chakravarti A, Collins FS, Daly MJ, et al.  (2005) A haplotype map of the human genome. Nature  437: 1299–1320.  <ul class="find" data-citedArticleID="969336" data-doi="10.1038/nature04226"><li><a href="http://dx.doi.org/10.1038/nature04226" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+haplotype+map+of+the+human+genome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+haplotype+map+of+the+human+genome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0000651-Rankinen1" id="pone.0000651-Rankinen1"></a>Rankinen T, Zuberi A, Chagnon YC, Weisnagel SJ, Argyropoulos G, et al.  (2006) The human obesity gene map: the 2005 update. Obesity (Silver Spring)  14: 529–644.  <ul class="find" data-citedArticleID="969420" data-doi="10.1038/oby.2006.71"><li><a href="http://dx.doi.org/10.1038/oby.2006.71" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+human+obesity+gene+map%3A+the+2005+update." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+human+obesity+gene+map%3A+the+2005+update.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0000651-Morris1" id="pone.0000651-Morris1"></a>Morris KH, Ishikawa A, Keightley PD (1999) Quantitative trait loci for growth traits in C57BL/6J×DBA/2J mice. Mamm Genome  10: 225–228.  <ul class="find" data-citedArticleID="969404" data-doi="10.1007/s003359900977"><li><a href="http://dx.doi.org/10.1007/s003359900977" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Quantitative+trait+loci+for+growth+traits+in+C57BL%2F6J%26%23xd7%3BDBA%2F2J+mice." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Quantitative+trait+loci+for+growth+traits+in+C57BL%2F6J%26%23xd7%3BDBA%2F2J+mice.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0000651-Christians1" id="pone.0000651-Christians1"></a>Christians JK, Bingham VK, Oliver FK, Heath TT, Keightley PD (2003) Characterization of a QTL affecting skeletal size in mice. Mamm Genome  14: 175–183.  <ul class="find" data-citedArticleID="969358" data-doi="10.1007/s00335-002-3041-2"><li><a href="http://dx.doi.org/10.1007/s00335-002-3041-2" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Characterization+of+a+QTL+affecting+skeletal+size+in+mice." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Characterization+of+a+QTL+affecting+skeletal+size+in+mice.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0000651-Christians2" id="pone.0000651-Christians2"></a>Christians JK, Keightley PD (2004) Fine mapping of a murine growth locus to a 1.4-cM region and resolution of linked QTL. Mamm Genome  15: 482–491.  <ul class="find" data-citedArticleID="969360" data-doi="10.1007/s00335-004-3046-0"><li><a href="http://dx.doi.org/10.1007/s00335-004-3046-0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Fine+mapping+of+a+murine+growth+locus+to+a+1.4-cM+region+and+resolution+of+linked+QTL." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Fine+mapping+of+a+murine+growth+locus+to+a+1.4-cM+region+and+resolution+of+linked+QTL.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0000651-Christians3" id="pone.0000651-Christians3"></a>Christians JK, Hoeflich A, Keightley PD (2006) PAPPA2, an enzyme that cleaves an insulin-like growth-factor-binding protein, is a candidate gene for a quantitative trait locus affecting body size in mice. Genetics  173: 1547–1553.  <ul class="find" data-citedArticleID="969362" data-doi="10.1534/genetics.106.057513"><li><a href="http://dx.doi.org/10.1534/genetics.106.057513" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=PAPPA2%2C+an+enzyme+that+cleaves+an+insulin-like+growth-factor-binding+protein%2C+is+a+candidate+gene+for+a+quantitative+trait+locus+affecting+body+size+in+mice." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22PAPPA2%2C+an+enzyme+that+cleaves+an+insulin-like+growth-factor-binding+protein%2C+is+a+candidate+gene+for+a+quantitative+trait+locus+affecting+body+size+in+mice.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0000651-Brockmann1" id="pone.0000651-Brockmann1"></a>Brockmann GA, Haley CS, Renne U, Knott SA, Schwerin M (1998) Quantitative trait loci affecting body weight and fatness from a mouse line selected for extreme high growth. Genetics  150: 369–381.  <ul class="find" data-citedArticleID="969344"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Quantitative+trait+loci+affecting+body+weight+and+fatness+from+a+mouse+line+selected+for+extreme+high+growth.&amp;auth=&amp;atitle=Quantitative+trait+loci+affecting+body+weight+and+fatness+from+a+mouse+line+selected+for+extreme+high+growth." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Quantitative+trait+loci+affecting+body+weight+and+fatness+from+a+mouse+line+selected+for+extreme+high+growth." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Quantitative+trait+loci+affecting+body+weight+and+fatness+from+a+mouse+line+selected+for+extreme+high+growth.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0000651-Brockmann2" id="pone.0000651-Brockmann2"></a>Brockmann GA, Kratzsch J, Haley CS, Renne U, Schwerin M, et al.  (2000) Single QTL effects, epistasis, and pleiotropy account for two-thirds of the phenotypic F(2) variance of growth and obesity in DU6i×DBA/2 mice. Genome Res  10: 1941–1957.  <ul class="find" data-citedArticleID="969346" data-doi="10.1101/gr.gr1499r"><li><a href="http://dx.doi.org/10.1101/gr.gr1499r" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Single+QTL+effects%2C+epistasis%2C+and+pleiotropy+account+for+two-thirds+of+the+phenotypic+F%282%29+variance+of+growth+and+obesity+in+DU6i%26%23xd7%3BDBA%2F2+mice." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Single+QTL+effects%2C+epistasis%2C+and+pleiotropy+account+for+two-thirds+of+the+phenotypic+F%282%29+variance+of+growth+and+obesity+in+DU6i%26%23xd7%3BDBA%2F2+mice.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0000651-Cheverud1" id="pone.0000651-Cheverud1"></a>Cheverud JM, Vaughn TT, Pletscher LS, Peripato AC, Adams ES, et al.  (2001) Genetic architecture of adiposity in the cross of LG/J and SM/J inbred mice. Mamm Genome  12: 3–12.  <ul class="find" data-citedArticleID="969356" data-doi="10.1007/s003350010218"><li><a href="http://dx.doi.org/10.1007/s003350010218" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genetic+architecture+of+adiposity+in+the+cross+of+LG%2FJ+and+SM%2FJ+inbred+mice." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genetic+architecture+of+adiposity+in+the+cross+of+LG%2FJ+and+SM%2FJ+inbred+mice.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0000651-Vaughn1" id="pone.0000651-Vaughn1"></a>Vaughn TT, Pletscher LS, Peripato A, King-Ellison K, Adams E, et al.  (1999) Mapping quantitative trait loci for murine growth: a closer look at genetic architecture. Genet Res  74: 313–322.  <ul class="find" data-citedArticleID="969432" data-doi="10.1017/s0016672399004103"><li><a href="http://dx.doi.org/10.1017/s0016672399004103" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mapping+quantitative+trait+loci+for+murine+growth%3A+a+closer+look+at+genetic+architecture." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mapping+quantitative+trait+loci+for+murine+growth%3A+a+closer+look+at+genetic+architecture.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0000651-Bymaster1" id="pone.0000651-Bymaster1"></a>Bymaster FP, McKinzie DL, Felder CC, Wess J (2003) Use of M1-M5 muscarinic receptor knockout mice as novel tools to delineate the physiological roles of the muscarinic cholinergic system. Neurochem Res  28: 437–442.  <ul class="find" data-citedArticleID="969350" data-doi="10.1080/10606820308262"><li><a href="http://dx.doi.org/10.1080/10606820308262" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Use+of+M1-M5+muscarinic+receptor+knockout+mice+as+novel+tools+to+delineate+the+physiological+roles+of+the+muscarinic+cholinergic+system." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Use+of+M1-M5+muscarinic+receptor+knockout+mice+as+novel+tools+to+delineate+the+physiological+roles+of+the+muscarinic+cholinergic+system.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0000651-Yamada1" id="pone.0000651-Yamada1"></a>Yamada M, Miyakawa T, Duttaroy A, Yamanaka A, Moriguchi T, et al.  (2001) Mice lacking the M3 muscarinic acetylcholine receptor are hypophagic and lean. Nature  410: 207–212.  <ul class="find" data-citedArticleID="969444"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Mice+lacking+the+M3+muscarinic+acetylcholine+receptor+are+hypophagic+and+lean.&amp;auth=&amp;atitle=Mice+lacking+the+M3+muscarinic+acetylcholine+receptor+are+hypophagic+and+lean." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mice+lacking+the+M3+muscarinic+acetylcholine+receptor+are+hypophagic+and+lean." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mice+lacking+the+M3+muscarinic+acetylcholine+receptor+are+hypophagic+and+lean.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0000651-Brewer1" id="pone.0000651-Brewer1"></a>Brewer HBJ, Santamarina-Fojo S, Shamburek RD (2005)  Genetic dyslipoproteinemias. In: Fuster V, Topol EJ, Nabel EG, editors. Atherothrombosis and coronary artery disease. Philadelphia, PA: Lippincott Williams&amp;Wilkins. pp. 55–83.  <ul class="find-nolinks"></ul></li><li><span class="label">22.
              </span><a name="pone.0000651-Rollins1" id="pone.0000651-Rollins1"></a>Rollins J, Chen Y, Paigen B, Wang X (2006) In search of new targets for plasma high-density lipoprotein cholesterol levels: promise of human-mouse comparative genomics. Trends Cardiovasc Med  16: 220–234.  <ul class="find" data-citedArticleID="969422" data-doi="10.1016/j.tcm.2006.04.003"><li><a href="http://dx.doi.org/10.1016/j.tcm.2006.04.003" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=In+search+of+new+targets+for+plasma+high-density+lipoprotein+cholesterol+levels%3A+promise+of+human-mouse+comparative+genomics." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22In+search+of+new+targets+for+plasma+high-density+lipoprotein+cholesterol+levels%3A+promise+of+human-mouse+comparative+genomics.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0000651-Weng1" id="pone.0000651-Weng1"></a>Weng W, Breslow JL (1996) Dramatically decreased high density lipoprotein cholesterol, increased remnant clearance, and insulin hypersensitivity in apolipoprotein A-II knockout mice suggest a complex role for apolipoprotein A-II in atherosclerosis susceptibility. Proc Natl Acad Sci U S A  93: 14788–14794.  <ul class="find" data-citedArticleID="969440" data-doi="10.1073/pnas.93.25.14788"><li><a href="http://dx.doi.org/10.1073/pnas.93.25.14788" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Dramatically+decreased+high+density+lipoprotein+cholesterol%2C+increased+remnant+clearance%2C+and+insulin+hypersensitivity+in+apolipoprotein+A-II+knockout+mice+suggest+a+complex+role+for+apolipoprotein+A-II+in+atherosclerosis+susceptibility." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Dramatically+decreased+high+density+lipoprotein+cholesterol%2C+increased+remnant+clearance%2C+and+insulin+hypersensitivity+in+apolipoprotein+A-II+knockout+mice+suggest+a+complex+role+for+apolipoprotein+A-II+in+atherosclerosis+susceptibility.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0000651-Masaki1" id="pone.0000651-Masaki1"></a>Masaki M, Kurisaki T, Shirakawa K, Sehara-Fujisawa A (2005) Role of meltrin {alpha} (ADAM12) in obesity induced by high- fat diet. Endocrinology  146: 1752–1763.  <ul class="find" data-citedArticleID="969398" data-doi="10.1210/en.2004-1082"><li><a href="http://dx.doi.org/10.1210/en.2004-1082" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Role+of+meltrin+%7Balpha%7D+%28ADAM12%29+in+obesity+induced+by+high-+fat+diet." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Role+of+meltrin+%7Balpha%7D+%28ADAM12%29+in+obesity+induced+by+high-+fat+diet.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0000651-Castro1" id="pone.0000651-Castro1"></a>Castro CH, Shin CS, Stains JP, Cheng SL, Sheikh S, et al.  (2004) Targeted expression of a dominant-negative N-cadherin in vivo delays peak bone mass and increases adipogenesis. J Cell Sci  117: 2853–2864.  <ul class="find" data-citedArticleID="969352" data-doi="10.1242/jcs.01133"><li><a href="http://dx.doi.org/10.1242/jcs.01133" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Targeted+expression+of+a+dominant-negative+N-cadherin+in+vivo+delays+peak+bone+mass+and+increases+adipogenesis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Targeted+expression+of+a+dominant-negative+N-cadherin+in+vivo+delays+peak+bone+mass+and+increases+adipogenesis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0000651-Zhang1" id="pone.0000651-Zhang1"></a>Zhang JV, Ren PG, Avsian-Kretchmer O, Luo CW, Rauch R, et al.  (2005) Obestatin, a peptide encoded by the ghrelin gene, opposes ghrelin's effects on food intake. Science  310: 996–999.  <ul class="find" data-citedArticleID="969448" data-doi="10.1126/science.1117255"><li><a href="http://dx.doi.org/10.1126/science.1117255" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Obestatin%2C+a+peptide+encoded+by+the+ghrelin+gene%2C+opposes+ghrelin%27s+effects+on+food+intake." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Obestatin%2C+a+peptide+encoded+by+the+ghrelin+gene%2C+opposes+ghrelin%27s+effects+on+food+intake.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0000651-Moechars1" id="pone.0000651-Moechars1"></a>Moechars D, Depoortere I, Moreaux B, de Smet B, Goris I, et al.  (2006) Altered gastrointestinal and metabolic function in the GPR39-obestatin receptor-knockout mouse. Gastroenterology  131: 1131–1141.  <ul class="find" data-citedArticleID="969402" data-doi="10.1053/j.gastro.2006.07.009"><li><a href="http://dx.doi.org/10.1053/j.gastro.2006.07.009" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Altered+gastrointestinal+and+metabolic+function+in+the+GPR39-obestatin+receptor-knockout+mouse." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Altered+gastrointestinal+and+metabolic+function+in+the+GPR39-obestatin+receptor-knockout+mouse.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0000651-Chartrel1" id="pone.0000651-Chartrel1"></a>Chartrel N, Alvear-Perez R, Leprince J, Iturrioz X, Reaux-Le Goazigo A, et al.  (2007) Comment on “Obestatin, a peptide encoded by the ghrelin gene, opposes ghrelin's effects on food intake”. Science  315: 766; author reply 766.  <ul class="find" data-citedArticleID="969354" data-doi="10.1126/science.1135047"><li><a href="http://dx.doi.org/10.1126/science.1135047" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Comment+on+%26%23x201c%3BObestatin%2C+a+peptide+encoded+by+the+ghrelin+gene%2C+opposes+ghrelin%27s+effects+on+food+intake%26%23x201d%3B." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Comment+on+%26%23x201c%3BObestatin%2C+a+peptide+encoded+by+the+ghrelin+gene%2C+opposes+ghrelin%27s+effects+on+food+intake%26%23x201d%3B.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0000651-Holst1" id="pone.0000651-Holst1"></a>Holst B, Egerod KL, Schild E, Vickers SP, Cheetham S, et al.  (2007) GPR39 signaling is stimulated by zinc ions but not by obestatin. Endocrinology  148: 13–20.  <ul class="find" data-citedArticleID="969380" data-doi="10.1210/en.2006-0933"><li><a href="http://dx.doi.org/10.1210/en.2006-0933" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=GPR39+signaling+is+stimulated+by+zinc+ions+but+not+by+obestatin." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22GPR39+signaling+is+stimulated+by+zinc+ions+but+not+by+obestatin.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0000651-Lauwers1" id="pone.0000651-Lauwers1"></a>Lauwers E, Landuyt B, Arckens L, Schoofs L, Luyten W (2006) Obestatin does not activate orphan G protein-coupled receptor GPR39. Biochem Biophys Res Commun  351: 21–25.  <ul class="find" data-citedArticleID="969386" data-doi="10.1016/j.bbrc.2006.09.141"><li><a href="http://dx.doi.org/10.1016/j.bbrc.2006.09.141" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Obestatin+does+not+activate+orphan+G+protein-coupled+receptor+GPR39." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Obestatin+does+not+activate+orphan+G+protein-coupled+receptor+GPR39.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0000651-Tremblay1" id="pone.0000651-Tremblay1"></a>Tremblay F, Perreault M, Klaman LD, Tobin JF, Smith E, et al.  (2007) Normal food intake and body weight in mice lacking the G protein-coupled receptor GPR39. Endocrinology  148: 501–506.  <ul class="find" data-citedArticleID="969426" data-doi="10.1210/en.2006-1275"><li><a href="http://dx.doi.org/10.1210/en.2006-1275" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Normal+food+intake+and+body+weight+in+mice+lacking+the+G+protein-coupled+receptor+GPR39." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Normal+food+intake+and+body+weight+in+mice+lacking+the+G+protein-coupled+receptor+GPR39.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0000651-Zhang2" id="pone.0000651-Zhang2"></a>Zhang Z, Futamura M, Vikis HG, Wang M, Li J, et al.  (2003) Positional cloning of the major quantitative trait locus underlying lung tumor susceptibility in mice. Proc Natl Acad Sci U S A  100: 12642–12647.  <ul class="find" data-citedArticleID="969450" data-doi="10.1073/pnas.2133947100"><li><a href="http://dx.doi.org/10.1073/pnas.2133947100" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Positional+cloning+of+the+major+quantitative+trait+locus+underlying+lung+tumor+susceptibility+in+mice." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Positional+cloning+of+the+major+quantitative+trait+locus+underlying+lung+tumor+susceptibility+in+mice.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0000651-Flaherty1" id="pone.0000651-Flaherty1"></a>Flaherty L, Herron B, Symula D (2005) Genomics of the future: identification of quantitative trait loci in the mouse. Genome Res  15: 1741–1745.  <ul class="find" data-citedArticleID="969368" data-doi="10.1101/gr.3841405"><li><a href="http://dx.doi.org/10.1101/gr.3841405" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genomics+of+the+future%3A+identification+of+quantitative+trait+loci+in+the+mouse." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genomics+of+the+future%3A+identification+of+quantitative+trait+loci+in+the+mouse.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0000651-Nadeau1" id="pone.0000651-Nadeau1"></a>Nadeau JH, Frankel WN (2000) The roads from phenotypic variation to gene discovery: mutagenesis versus QTLs. Nat Genet  25: 381–384.  <ul class="find" data-citedArticleID="969408"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=The+roads+from+phenotypic+variation+to+gene+discovery%3A+mutagenesis+versus+QTLs.&amp;auth=&amp;atitle=The+roads+from+phenotypic+variation+to+gene+discovery%3A+mutagenesis+versus+QTLs." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+roads+from+phenotypic+variation+to+gene+discovery%3A+mutagenesis+versus+QTLs." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+roads+from+phenotypic+variation+to+gene+discovery%3A+mutagenesis+versus+QTLs.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0000651-Zhao1" id="pone.0000651-Zhao1"></a>Zhao K, Aranzana MJ, Kim S, Lister C, Shindo C, et al.  (2007) An Arabidopsis Example of Association Mapping in Structured Samples. PLoS Genet  3: e4.  <ul class="find" data-citedArticleID="969452" data-doi="10.1371/journal.pgen.0030004"><li><a href="http://dx.doi.org/10.1371/journal.pgen.0030004" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=An+Arabidopsis+Example+of+Association+Mapping+in+Structured+Samples." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22An+Arabidopsis+Example+of+Association+Mapping+in+Structured+Samples.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">36.
              </span><a name="pone.0000651-Yu1" id="pone.0000651-Yu1"></a>Yu J, Pressoir G, Briggs WH, Vroh Bi I, Yamasaki M, et al.  (2006) A unified mixed-model method for association mapping that accounts for multiple levels of relatedness. Nat Genet  38: 203–208.  <ul class="find" data-citedArticleID="969446" data-doi="10.1038/ng1702"><li><a href="http://dx.doi.org/10.1038/ng1702" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+unified+mixed-model+method+for+association+mapping+that+accounts+for+multiple+levels+of+relatedness." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+unified+mixed-model+method+for+association+mapping+that+accounts+for+multiple+levels+of+relatedness.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">37.
              </span><a name="pone.0000651-Pritchard1" id="pone.0000651-Pritchard1"></a>Pritchard JK, Stephens M, Donnelly P (2000) Inference of population structure using multilocus genotype data. Genetics  155: 945–959.  <ul class="find" data-citedArticleID="969418"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Inference+of+population+structure+using+multilocus+genotype+data.&amp;auth=&amp;atitle=Inference+of+population+structure+using+multilocus+genotype+data." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Inference+of+population+structure+using+multilocus+genotype+data." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Inference+of+population+structure+using+multilocus+genotype+data.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">38.
              </span><a name="pone.0000651-Devlin1" id="pone.0000651-Devlin1"></a>Devlin B, Roeder K (1999) Genomic control for association studies. Biometrics  55: 997–1004.  <ul class="find" data-citedArticleID="969366" data-doi="10.1111/j.0006-341x.1999.00997.x"><li><a href="http://dx.doi.org/10.1111/j.0006-341x.1999.00997.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genomic+control+for+association+studies." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genomic+control+for+association+studies.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">39.
              </span><a name="pone.0000651-Marchini1" id="pone.0000651-Marchini1"></a>Marchini J, Cardon LR, Phillips MS, Donnelly P (2004) The effects of human population structure on large genetic association studies. Nat Genet  36: 512–517.  <ul class="find" data-citedArticleID="969396" data-doi="10.1038/ng1337"><li><a href="http://dx.doi.org/10.1038/ng1337" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+effects+of+human+population+structure+on+large+genetic+association+studies." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+effects+of+human+population+structure+on+large+genetic+association+studies.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">40.
              </span><a name="pone.0000651-Payseur1" id="pone.0000651-Payseur1"></a>Payseur BA, Place M (2007) Prospects for Association Mapping in Classical Inbred Mouse Strains. Genetics  175: 1999–2008.  <ul class="find" data-citedArticleID="969412" data-doi="10.1534/genetics.106.067868"><li><a href="http://dx.doi.org/10.1534/genetics.106.067868" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Prospects+for+Association+Mapping+in+Classical+Inbred+Mouse+Strains." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Prospects+for+Association+Mapping+in+Classical+Inbred+Mouse+Strains.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">41.
              </span><a name="pone.0000651-Clark1" id="pone.0000651-Clark1"></a>Clark AG, Hubisz MJ, Bustamante CD, Williamson SH, Nielsen R (2005) Ascertainment bias in studies of human genome-wide polymorphism. Genome Res  15: 1496–1502.  <ul class="find" data-citedArticleID="969364" data-doi="10.1101/gr.4107905"><li><a href="http://dx.doi.org/10.1101/gr.4107905" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Ascertainment+bias+in+studies+of+human+genome-wide+polymorphism." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Ascertainment+bias+in+studies+of+human+genome-wide+polymorphism.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">42.
              </span><a name="pone.0000651-Hedrick1" id="pone.0000651-Hedrick1"></a>Hedrick PW (1987) Gametic disequilibrium measures: proceed with caution. Genetics  117: 331–341.  <ul class="find" data-citedArticleID="969376"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Gametic+disequilibrium+measures%3A+proceed+with+caution.&amp;auth=&amp;atitle=Gametic+disequilibrium+measures%3A+proceed+with+caution." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Gametic+disequilibrium+measures%3A+proceed+with+caution." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Gametic+disequilibrium+measures%3A+proceed+with+caution.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">43.
              </span><a name="pone.0000651-Royston1" id="pone.0000651-Royston1"></a>Royston P (1982) An Extension of Shapiro and Wilk's W Test for Normality to Large Samples. Applied Statistics  31: 115–124.  <ul class="find" data-citedArticleID="969424" data-doi="10.2307/2347973"><li><a href="http://dx.doi.org/10.2307/2347973" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=An+Extension+of+Shapiro+and+Wilk%27s+W+Test+for+Normality+to+Large+Samples." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22An+Extension+of+Shapiro+and+Wilk%27s+W+Test+for+Normality+to+Large+Samples.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">44.
              </span><a name="pone.0000651-Altman1" id="pone.0000651-Altman1"></a>Altman DG (1991) Practical Statistics for Medical Research: Chapman and Hall.   <ul class="find-nolinks"></ul></li><li><span class="label">45.
              </span><a name="pone.0000651-Box1" id="pone.0000651-Box1"></a>Box GEP, Cox DR (1964) An analysis of transformations. Journal of Royal Statistical Society, Series B  26: 211–246.  <ul class="find" data-citedArticleID="969340"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=An+analysis+of+transformations.&amp;auth=&amp;atitle=An+analysis+of+transformations." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=An+analysis+of+transformations." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22An+analysis+of+transformations.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">46.
              </span><a name="pone.0000651-Hollander1" id="pone.0000651-Hollander1"></a>Hollander M, Wolfe DA (1973) Nonparametric statistical inference. New York: John Wiley and Sons. pp. 68–75.  <ul class="find-nolinks"></ul></li><li><span class="label">47.
              </span><a name="pone.0000651-Bale1" id="pone.0000651-Bale1"></a>Bale TL, Anderson KR, Roberts AJ, Lee KF, Nagy TR, et al.  (2003) Corticotropin-releasing factor receptor-2-deficient mice display abnormal homeostatic responses to challenges of increased dietary fat and cold. Endocrinology  144: 2580–2587.  <ul class="find" data-citedArticleID="969338" data-doi="10.1210/en.2002-0091"><li><a href="http://dx.doi.org/10.1210/en.2002-0091" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Corticotropin-releasing+factor+receptor-2-deficient+mice+display+abnormal+homeostatic+responses+to+challenges+of+increased+dietary+fat+and+cold." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Corticotropin-releasing+factor+receptor-2-deficient+mice+display+abnormal+homeostatic+responses+to+challenges+of+increased+dietary+fat+and+cold.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">48.
              </span><a name="pone.0000651-Naaz1" id="pone.0000651-Naaz1"></a>Naaz A, Holsberger DR, Iwamoto GA, Nelson A, Kiyokawa H, et al.  (2004) Loss of cyclin-dependent kinase inhibitors produces adipocyte hyperplasia and obesity. Faseb J  18: 1925–1927.  <ul class="find" data-citedArticleID="969406" data-doi="10.1096/fj.04-2631fje"><li><a href="http://dx.doi.org/10.1096/fj.04-2631fje" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Loss+of+cyclin-dependent+kinase+inhibitors+produces+adipocyte+hyperplasia+and+obesity." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Loss+of+cyclin-dependent+kinase+inhibitors+produces+adipocyte+hyperplasia+and+obesity.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">49.
              </span><a name="pone.0000651-FontdeMora1" id="pone.0000651-FontdeMora1"></a>Font de Mora J, Esteban LM, Burks DJ, Nunez A, Garces C, et al.  (2003) Ras-GRF1 signaling is required for normal beta-cell development and glucose homeostasis. Embo J  22: 3039–3049.  <ul class="find" data-citedArticleID="969370" data-doi="10.1093/emboj/cdg280"><li><a href="http://dx.doi.org/10.1093/emboj/cdg280" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Ras-GRF1+signaling+is+required+for+normal+beta-cell+development+and+glucose+homeostasis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Ras-GRF1+signaling+is+required+for+normal+beta-cell+development+and+glucose+homeostasis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">50.
              </span><a name="pone.0000651-Tuncman1" id="pone.0000651-Tuncman1"></a>Tuncman G, Hirosumi J, Solinas G, Chang L, Karin M, et al.  (2006) Functional in vivo interactions between JNK1 and JNK2 isoforms in obesity and insulin resistance. Proc Natl Acad Sci U S A  103: 10741–10746.  <ul class="find" data-citedArticleID="969428" data-doi="10.1073/pnas.0603509103"><li><a href="http://dx.doi.org/10.1073/pnas.0603509103" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Functional+in+vivo+interactions+between+JNK1+and+JNK2+isoforms+in+obesity+and+insulin+resistance." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Functional+in+vivo+interactions+between+JNK1+and+JNK2+isoforms+in+obesity+and+insulin+resistance.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">51.
              </span><a name="pone.0000651-Longo1" id="pone.0000651-Longo1"></a>Longo KA, Wright WS, Kang S, Gerin I, Chiang SH, et al.  (2004) Wnt10b inhibits development of white and brown adipose tissues. J Biol Chem  279: 35503–35509.  <ul class="find" data-citedArticleID="969392" data-doi="10.1074/jbc.m402937200"><li><a href="http://dx.doi.org/10.1074/jbc.m402937200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Wnt10b+inhibits+development+of+white+and+brown+adipose+tissues." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Wnt10b+inhibits+development+of+white+and+brown+adipose+tissues.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">52.
              </span><a name="pone.0000651-Wu1" id="pone.0000651-Wu1"></a>Wu JJ, Roth RJ, Anderson EJ, Hong EG, Lee MK, et al.  (2006) Mice lacking MAP kinase phosphatase-1 have enhanced MAP kinase activity and resistance to diet-induced obesity. Cell Metab  4: 61–73.  <ul class="find" data-citedArticleID="969442" data-doi="10.1016/j.cmet.2006.05.010"><li><a href="http://dx.doi.org/10.1016/j.cmet.2006.05.010" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mice+lacking+MAP+kinase+phosphatase-1+have+enhanced+MAP+kinase+activity+and+resistance+to+diet-induced+obesity." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mice+lacking+MAP+kinase+phosphatase-1+have+enhanced+MAP+kinase+activity+and+resistance+to+diet-induced+obesity.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">53.
              </span><a name="pone.0000651-Goudriaan1" id="pone.0000651-Goudriaan1"></a>Goudriaan JR, Dahlmans VE, Teusink B, Ouwens DM, Febbraio M, et al.  (2003) CD36 deficiency increases insulin sensitivity in muscle, but induces insulin resistance in the liver in mice. J Lipid Res  44: 2270–2277.  <ul class="find" data-citedArticleID="969372" data-doi="10.1194/jlr.m300143-jlr200"><li><a href="http://dx.doi.org/10.1194/jlr.m300143-jlr200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=CD36+deficiency+increases+insulin+sensitivity+in+muscle%2C+but+induces+insulin+resistance+in+the+liver+in+mice." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22CD36+deficiency+increases+insulin+sensitivity+in+muscle%2C+but+induces+insulin+resistance+in+the+liver+in+mice.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">54.
              </span><a name="pone.0000651-Miyasaka1" id="pone.0000651-Miyasaka1"></a>Miyasaka K, Takata Y, Funakoshi A (2002) Association of cholecystokinin A receptor gene polymorphism with cholelithiasis and the molecular mechanisms of this polymorphism. J Gastroenterol  37: Suppl 14102–106.  <ul class="find" data-citedArticleID="969400" data-doi="10.1007/bf03326426"><li><a href="http://dx.doi.org/10.1007/bf03326426" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Association+of+cholecystokinin+A+receptor+gene+polymorphism+with+cholelithiasis+and+the+molecular+mechanisms+of+this+polymorphism." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Association+of+cholecystokinin+A+receptor+gene+polymorphism+with+cholelithiasis+and+the+molecular+mechanisms+of+this+polymorphism.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">55.
              </span><a name="pone.0000651-Patel1" id="pone.0000651-Patel1"></a>Patel HR, Qi Y, Hawkins EJ, Hileman SM, Elmquist JK, et al.  (2006) Neuropeptide Y deficiency attenuates responses to fasting and high-fat diet in obesity-prone mice. Diabetes  55: 3091–3098.  <ul class="find" data-citedArticleID="969410" data-doi="10.2337/db05-0624"><li><a href="http://dx.doi.org/10.2337/db05-0624" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Neuropeptide+Y+deficiency+attenuates+responses+to+fasting+and+high-fat+diet+in+obesity-prone+mice." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Neuropeptide+Y+deficiency+attenuates+responses+to+fasting+and+high-fat+diet+in+obesity-prone+mice.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">56.
              </span><a name="pone.0000651-Akiyama1" id="pone.0000651-Akiyama1"></a>Akiyama TE, Sakai S, Lambert G, Nicol CJ, Matsusue K, et al.  (2002) Conditional disruption of the peroxisome proliferator-activated receptor gamma gene in mice results in lowered expression of ABCA1, ABCG1, and apoE in macrophages and reduced cholesterol efflux. Mol Cell Biol  22: 2607–2619.  <ul class="find" data-citedArticleID="969332" data-doi="10.1128/mcb.22.8.2607-2619.2002"><li><a href="http://dx.doi.org/10.1128/mcb.22.8.2607-2619.2002" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Conditional+disruption+of+the+peroxisome+proliferator-activated+receptor+gamma+gene+in+mice+results+in+lowered+expression+of+ABCA1%2C+ABCG1%2C+and+apoE+in+macrophages+and+reduced+cholesterol+efflux." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Conditional+disruption+of+the+peroxisome+proliferator-activated+receptor+gamma+gene+in+mice+results+in+lowered+expression+of+ABCA1%2C+ABCG1%2C+and+apoE+in+macrophages+and+reduced+cholesterol+efflux.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">57.
              </span><a name="pone.0000651-Weisberg1" id="pone.0000651-Weisberg1"></a>Weisberg SP, Hunter D, Huber R, Lemieux J, Slaymaker S, et al.  (2006) CCR2 modulates inflammatory and metabolic effects of high-fat feeding. J Clin Invest  116: 115–124.  <ul class="find" data-citedArticleID="969438" data-doi="10.1172/jci24335"><li><a href="http://dx.doi.org/10.1172/jci24335" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=CCR2+modulates+inflammatory+and+metabolic+effects+of+high-fat+feeding." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22CCR2+modulates+inflammatory+and+metabolic+effects+of+high-fat+feeding.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">58.
              </span><a name="pone.0000651-Mao1" id="pone.0000651-Mao1"></a>Mao J, DeMayo FJ, Li H, Abu-Elheiga L, Gu Z, et al.  (2006) Liver-specific deletion of acetyl-CoA carboxylase 1 reduces hepatic triglyceride accumulation without affecting glucose homeostasis. Proc Natl Acad Sci U S A  103: 8552–8557.  <ul class="find" data-citedArticleID="969394" data-doi="10.1073/pnas.0603115103"><li><a href="http://dx.doi.org/10.1073/pnas.0603115103" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Liver-specific+deletion+of+acetyl-CoA+carboxylase+1+reduces+hepatic+triglyceride+accumulation+without+affecting+glucose+homeostasis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Liver-specific+deletion+of+acetyl-CoA+carboxylase+1+reduces+hepatic+triglyceride+accumulation+without+affecting+glucose+homeostasis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">59.
              </span><a name="pone.0000651-Huang1" id="pone.0000651-Huang1"></a>Huang LS, Voyiaziakis E, Markenson DF, Sokol KA, Hayek T, et al.  (1995) apo B gene knockout in mice results in embryonic lethality in homozygotes and neural tube defects, male infertility, and reduced HDL cholesterol ester and apo A-I transport rates in heterozygotes. J Clin Invest  96: 2152–2161.  <ul class="find" data-citedArticleID="969382" data-doi="10.1172/jci118269"><li><a href="http://dx.doi.org/10.1172/jci118269" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=apo+B+gene+knockout+in+mice+results+in+embryonic+lethality+in+homozygotes+and+neural+tube+defects%2C+male+infertility%2C+and+reduced+HDL+cholesterol+ester+and+apo+A-I+transport+rates+in+heterozygotes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22apo+B+gene+knockout+in+mice+results+in+embryonic+lethality+in+homozygotes+and+neural+tube+defects%2C+male+infertility%2C+and+reduced+HDL+cholesterol+ester+and+apo+A-I+transport+rates+in+heterozygotes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">60.
              </span><a name="pone.0000651-Buhman1" id="pone.0000651-Buhman1"></a>Buhman KK, Accad M, Novak S, Choi RS, Wong JS, et al.  (2000) Resistance to diet-induced hypercholesterolemia and gallstone formation in ACAT2-deficient mice. Nat Med  6: 1341–1347.  <ul class="find" data-citedArticleID="969348"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Resistance+to+diet-induced+hypercholesterolemia+and+gallstone+formation+in+ACAT2-deficient+mice.&amp;auth=&amp;atitle=Resistance+to+diet-induced+hypercholesterolemia+and+gallstone+formation+in+ACAT2-deficient+mice." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Resistance+to+diet-induced+hypercholesterolemia+and+gallstone+formation+in+ACAT2-deficient+mice." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Resistance+to+diet-induced+hypercholesterolemia+and+gallstone+formation+in+ACAT2-deficient+mice.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">61.
              </span><a name="pone.0000651-Ishida1" id="pone.0000651-Ishida1"></a>Ishida T, Choi S, Kundu RK, Hirata K, Rubin EM, et al.  (2003) Endothelial lipase is a major determinant of HDL level. J Clin Invest  111: 347–355.  <ul class="find" data-citedArticleID="969384" data-doi="10.1172/jci200316306"><li><a href="http://dx.doi.org/10.1172/jci200316306" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Endothelial+lipase+is+a+major+determinant+of+HDL+level." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Endothelial+lipase+is+a+major+determinant+of+HDL+level.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000651.XML" value="137150"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000651.PDF" value="1038497"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000651.g001.PNG_L" value="846642"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000651.g001.PNG_M" value="147816"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000651.g001.PNG_S" value="17529"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000651.g001.TIF" value="1196442"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000651.g001.PNG_I" value="86394"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000651.t001.PNG_L" value="175243"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000651.t001.PNG_M" value="105070"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000651.t001.PNG_S" value="11171"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000651.t001.TIF" value="756354"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000651.t001.PNG_I" value="31534"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000651.g002.PNG_L" value="952952"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000651.g002.PNG_M" value="117935"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000651.g002.PNG_S" value="15732"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000651.g002.TIF" value="1726370"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000651.g002.PNG_I" value="72225"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000651.g003.PNG_L" value="938713"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000651.g003.PNG_M" value="204945"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000651.g003.PNG_S" value="11646"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000651.g003.TIF" value="1422518"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000651.g003.PNG_I" value="65483"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000651.g004.PNG_L" value="805337"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000651.g004.PNG_M" value="177953"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000651.g004.PNG_S" value="11574"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000651.g004.TIF" value="1133128"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000651.g004.PNG_I" value="58272"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000651.t002.PNG_L" value="333937"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000651.t002.PNG_M" value="188467"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000651.t002.PNG_S" value="16539"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000651.t002.TIF" value="1369130"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000651.t002.PNG_I" value="57375"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000651.t003.PNG_L" value="313859"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000651.t003.PNG_M" value="178565"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000651.t003.PNG_S" value="16270"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000651.t003.TIF" value="1301040"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000651.t003.PNG_I" value="54314"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000651.g005.PNG_L" value="580415"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000651.g005.PNG_M" value="84364"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000651.g005.PNG_S" value="12344"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000651.g005.TIF" value="896060"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000651.g005.PNG_I" value="74468"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000651.g006.PNG_L" value="664992"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000651.g006.PNG_M" value="195964"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000651.g006.PNG_S" value="12026"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000651.g006.TIF" value="921334"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000651.g006.PNG_I" value="63199"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000651.s001.DOC" value="112128"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000651.s002.DOC" value="361984"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000651.s003.DOC" value="1988608"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000651.s004.PDF" value="407698"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000651.s005.PDF" value="899144"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000651.s006.PDF" value="1407770"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000651.s007.PDF" value="699370"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000651.s008.PDF" value="205892"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000651.s009.PDF" value="7955486"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000651.s010.PDF" value="1777048"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000651.s011.PDF" value="480052"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000651&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000651" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000651&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0000651&volume=&issue=&title=Large-Scale In Silico Mapping of Complex Quantitative Traits in Inbred Mice&author_name=Pengyuan%20Liu%2C%20Haris%20Vikis%2C%20Yan%20Lu%2C%20Daolong%20Wang%2C%20Ming%20You&start_page=1&end_page=14" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000651" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000651" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0000651" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000651&amp;t=Large-Scale%20In%20Silico%20Mapping%20of%20Complex%20Quantitative%20Traits%20in%20Inbred%20Mice" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000651&title=Large-Scale%20In%20Silico%20Mapping%20of%20Complex%20Quantitative%20Traits%20in%20Inbred%20Mice&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000651&amp;title=Large-Scale%20In%20Silico%20Mapping%20of%20Complex%20Quantitative%20Traits%20in%20Inbred%20Mice" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000651" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0000651&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Large-Scale In Silico Mapping of Complex Quantitative Traits in Inbred Mice';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0000651';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Large-Scale%20In%20Silico%20Mapping%20of%20Complex%20Quantitative%20Traits%20in%20Inbred%20Mice http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000651" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0000651" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">













          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Body+weight%22" title="Search for articles in the subject area:'Body weight'"><div class="flagText">Body weight</div></a>
              <div data-categoryid="47441" data-articleid="24140"
                   data-categoryname="Body weight"
                   class="flagImage" title="Flag 'Body weight' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Cholesterol%22" title="Search for articles in the subject area:'Cholesterol'"><div class="flagText">Cholesterol</div></a>
              <div data-categoryid="20521" data-articleid="24140"
                   data-categoryname="Cholesterol"
                   class="flagImage" title="Flag 'Cholesterol' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Diet%22" title="Search for articles in the subject area:'Diet'"><div class="flagText">Diet</div></a>
              <div data-categoryid="18621" data-articleid="24140"
                   data-categoryname="Diet"
                   class="flagImage" title="Flag 'Diet' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Genome+evolution%22" title="Search for articles in the subject area:'Genome evolution'"><div class="flagText">Genome evolution</div></a>
              <div data-categoryid="17207" data-articleid="24140"
                   data-categoryname="Genome evolution"
                   class="flagImage" title="Flag 'Genome evolution' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Inbred+strains%22" title="Search for articles in the subject area:'Inbred strains'"><div class="flagText">Inbred strains</div></a>
              <div data-categoryid="35433" data-articleid="24140"
                   data-categoryname="Inbred strains"
                   class="flagImage" title="Flag 'Inbred strains' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Mammalian+genomics%22" title="Search for articles in the subject area:'Mammalian genomics'"><div class="flagText">Mammalian genomics</div></a>
              <div data-categoryid="17625" data-articleid="24140"
                   data-categoryname="Mammalian genomics"
                   class="flagImage" title="Flag 'Mammalian genomics' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Obesity%22" title="Search for articles in the subject area:'Obesity'"><div class="flagText">Obesity</div></a>
              <div data-categoryid="47275" data-articleid="24140"
                   data-categoryname="Obesity"
                   class="flagImage" title="Flag 'Obesity' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Phenotypes%22" title="Search for articles in the subject area:'Phenotypes'"><div class="flagText">Phenotypes</div></a>
              <div data-categoryid="40235" data-articleid="24140"
                   data-categoryname="Phenotypes"
                   class="flagImage" title="Flag 'Phenotypes' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=2675'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=4856'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=6515&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>


</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
