

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Alternative mRNA Editing in Trypanosomes Is Extensive and May Contribute to Mitochondrial Protein Diversity</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0001566"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0001566" />

    <meta name="citation_title" content="Alternative mRNA Editing in Trypanosomes Is Extensive and May Contribute to Mitochondrial Protein Diversity"/>
    <meta itemprop="name" content="Alternative mRNA Editing in Trypanosomes Is Extensive and May Contribute to Mitochondrial Protein Diversity"/>

      <meta name="citation_author" content="Torsten Ochsenreiter"/>
            <meta name="citation_author_institution" content="Department of Biochemistry and Molecular Biology, University of Georgia, Athens, Georgia, United States of America"/>
      <meta name="citation_author" content="Michael Cipriano"/>
            <meta name="citation_author_institution" content="Department of Medical Microbiology, University of California Davis, Davis, California, United States of America"/>
      <meta name="citation_author" content="Stephen L. Hajduk"/>
            <meta name="citation_author_institution" content="Department of Biochemistry and Molecular Biology, University of Georgia, Athens, Georgia, United States of America"/>

    <meta name="citation_date" content="2008/2/13"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0001566.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e1566"/>
    <meta name="citation_issue" content="2"/>
    <meta name="citation_volume" content="3"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Editing machines: the complexities of trypanosome RNA editing.; citation_author=S Madison-Antenucci; citation_author=J Grams; citation_author=SL Hajduk; citation_journal_title=Cell; citation_volume=108; citation_number=1; citation_pages=435-438; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Uridine insertion/deletion RNA editing in trypanosome mitochondria: a complex business.; citation_author=L Simpson; citation_author=S Sbicego; citation_author=R Aphasizhev; citation_journal_title=Rna; citation_volume=9; citation_number=2; citation_pages=265-276; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Complex management: RNA editing in trypanosomes.; citation_author=KD Stuart; citation_author=A Schnaufer; citation_author=NL Ernst; citation_author=AK Panigrahi; citation_journal_title=Trends Biochem Sci; citation_volume=30; citation_number=3; citation_pages=97-105; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Major transcript of the frameshifted coxII gene from trypanosome mitochondria contains four nucleotides that are not encoded in the DNA.; citation_author=R Benne; citation_author=J Van den Burg; citation_author=JP Brakenhoff; citation_author=P Sloof; citation_author=JH Van Boom; citation_journal_title=Cell; citation_volume=46; citation_number=4; citation_pages=819-826; citation_date=1986; " />
      <meta name="citation_reference" content="citation_title=Extensive editing of the cytochrome c oxidase III transcript in Trypanosoma brucei.; citation_author=JE Feagin; citation_author=JM Abraham; citation_author=K Stuart; citation_journal_title=Cell; citation_volume=53; citation_number=5; citation_pages=413-422; citation_date=1988; " />
      <meta name="citation_reference" content="citation_title=The MURF3 gene of T. brucei contains multiple domains of extensive editing and is homologous to a subunit of NADH dehydrogenase.; citation_author=DJ Koslowsky; citation_author=GJ Bhat; citation_author=AL Perrollaz; citation_author=JE Feagin; citation_author=K Stuart; citation_journal_title=Cell; citation_volume=62; citation_number=6; citation_pages=901-911; citation_date=1990; " />
      <meta name="citation_reference" content="citation_title=Function of RNA editing in Trypanosomes.; citation_author=T Ochsenreiter; citation_author=SL Hajduk; citation_number=7; citation_date=2007; citation_publisher=Springer; " />
      <meta name="citation_reference" content="citation_title=Characterization of cytochrome c oxidase III transcripts that are edited only in the 3′ region.; citation_author=JM Abraham; citation_author=JE Feagin; citation_author=K Stuart; citation_journal_title=Cell; citation_volume=55; citation_number=8; citation_pages=267-272; citation_date=1988; " />
      <meta name="citation_reference" content="citation_title=RNA editing involves indiscriminate U changes throughout precisely defined editing domains.; citation_author=CJ Decker; citation_author=B Sollner-Webb; citation_journal_title=Cell; citation_volume=61; citation_number=9; citation_pages=1001-1011; citation_date=1990; " />
      <meta name="citation_reference" content="citation_title=Partially edited mRNAs for cytochrome b and subunit III of cytochrome oxidase from Leishmania tarentolae mitochondria: RNA editing intermediates.; citation_author=NR Sturm; citation_author=L Simpson; citation_journal_title=Cell; citation_volume=61; citation_number=10; citation_pages=871-878; citation_date=1990; " />
      <meta name="citation_reference" content="citation_title=Alternative editing of cytochrome c oxidase III mRNA in trypanosome mitochondria generates protein diversity.; citation_author=T Ochsenreiter; citation_author=SL Hajduk; citation_journal_title=EMBO Rep; citation_volume=7; citation_number=11; citation_pages=1128-1133; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Fellowship of the rings: the replication of kinetoplast DNA.; citation_author=B Liu; citation_author=Y Liu; citation_author=SA Motyka; citation_author=EE Agbo; citation_author=PT Englund; citation_journal_title=Trends Parasitol; citation_volume=21; citation_number=12; citation_pages=363-369; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=A high-order trans-membrane structural linkage is responsible for mitochondrial genome positioning and segregation by flagellar basal bodies in trypanosomes.; citation_author=EO Ogbadoyi; citation_author=DR Robinson; citation_author=K Gull; citation_journal_title=Mol Biol Cell; citation_volume=14; citation_number=13; citation_pages=1769-1779; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=The developmental cell biology of Trypanosoma brucei.; citation_author=KR Matthews; citation_journal_title=J Cell Sci; citation_volume=118; citation_number=14; citation_pages=283-290; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=KISS: the kinetoplastid RNA editing sequence search tool.; citation_author=T Ochsenreiter; citation_author=M Cipriano; citation_author=SL Hajduk; citation_journal_title=Rna; citation_volume=13; citation_number=15; citation_pages=1-4; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Isolation and characterization of complex I, rotenone-sensitive NADH: ubiquinone oxidoreductase, from the procyclic forms of Trypanosoma brucei.; citation_author=J Fang; citation_author=Y Wang; citation_author=DS Beattie; citation_journal_title=Eur J Biochem; citation_volume=268; citation_number=16; citation_pages=3075-3082; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Trypanosome alternative oxidase: from molecule to function.; citation_author=M Chaudhuri; citation_author=RD Ott; citation_author=GC Hill; citation_journal_title=Trends Parasitol; citation_volume=22; citation_number=17; citation_pages=484-491; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=RNA editing as a source of genetic variation.; citation_author=LF Landweber; citation_author=W Gilbert; citation_journal_title=Nature; citation_volume=363; citation_number=18; citation_pages=179-182; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=Alternative pre-mRNA splicing and proteome expansion in metazoans.; citation_author=T Maniatis; citation_author=B Tasic; citation_journal_title=Nature; citation_volume=418; citation_number=19; citation_pages=236-243; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Intronic alternative splicing regulators identified by comparative genomics in nematodes.; citation_author=JL Kabat; citation_author=S Barberan-Soler; citation_author=P McKenna; citation_author=H Clawson; citation_author=T Farrer; citation_journal_title=PLoS Comput Biol; citation_volume=2; citation_number=20; citation_pages=e86; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Function of alternative splicing.; citation_author=S Stamm; citation_author=S Ben-Ari; citation_author=I Rafalska; citation_author=Y Tang; citation_author=Z Zhang; citation_journal_title=Gene; citation_volume=344; citation_number=21; citation_pages=1-20; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Guide RNAs for transcripts with developmentally regulated RNA editing are present in both life cycle stages of Trypanosoma brucei.; citation_author=DJ Koslowsky; citation_author=GR Riley; citation_author=JE Feagin; citation_author=K Stuart; citation_journal_title=Mol Cell Biol; citation_volume=12; citation_number=22; citation_pages=2043-2049; citation_date=1992; " />
      <meta name="citation_reference" content="citation_title=Developmental regulation of RNA editing and polyadenylation in four life cycle stages of Trypanosoma congolense.; citation_author=LK Read; citation_author=KA Stankey; citation_author=WR Fish; citation_author=AM Muthiani; citation_author=K Stuart; citation_journal_title=Mol Biochem Parasitol; citation_volume=68; citation_number=23; citation_pages=297-306; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=Unexplained complexity of the mitochondrial genome and transcriptome in kinetoplastid flagellates.; citation_author=J Lukes; citation_author=H Hashimi; citation_author=A Zikova; citation_journal_title=Curr Genet; citation_volume=48; citation_number=24; citation_pages=277-299; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Quantitation of RNA editing substrates, products and potential intermediates: implications for developmental regulation.; citation_author=GR Riley; citation_author=PJ Myler; citation_author=K Stuart; citation_journal_title=Nucleic Acids Res; citation_volume=23; citation_number=25; citation_pages=708-712; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Expression of mitochondrial protein-coding genes in Tetrahymena pyriformis.; citation_author=J Edqvist; citation_author=G Burger; citation_author=MW Gray; citation_journal_title=J Mol Biol; citation_volume=297; citation_number=26; citation_pages=381-393; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Translation initiation from a naturally occurring non-AUG codon in Saccharomyces cerevisiae.; citation_author=KJ Chang; citation_author=CC Wang; citation_journal_title=J Biol Chem; citation_volume=279; citation_number=27; citation_pages=13778-13785; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Trypanosoma brucei minicircles encode multiple guide RNAs which can direct editing of extensively overlapping sequences.; citation_author=RA Corell; citation_author=JE Feagin; citation_author=GR Riley; citation_author=T Strickland; citation_author=JA Guderian; citation_journal_title=Nucleic Acids Res; citation_volume=21; citation_number=28; citation_pages=4313-4320; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=EMBOSS: the European Molecular Biology Open Software Suite.; citation_author=P Rice; citation_author=I Longden; citation_author=A Bleasby; citation_journal_title=Trends Genet; citation_volume=16; citation_number=29; citation_pages=276-277; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Sequence identification of cytochrome b in Plasmodium gallinaceum.; citation_author=SM Aldritt; citation_author=JT Joseph; citation_author=DF Wirth; citation_journal_title=Mol Cell Biol; citation_volume=9; citation_number=30; citation_pages=3614-3620; citation_date=1989; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001566" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Alternative mRNA Editing in Trypanosomes Is Extensive and May Contribute to Mitochondrial Protein Diversity"/>
    <meta name="twitter:description" content="The editing of trypanosome mitochondrial mRNAs produces transcripts necessary for mitochondrial functions including electron transport and oxidative phosphorylation. Precursor-mRNAs are often extensively edited by specific uridine insertion or deletion that is directed by small guide RNAs (gRNAs). Recently, it has been shown that cytochrome c oxidase subunit III (COXIII) mRNAs can be alternatively edited to encode a novel mitochondrial membrane protein composed of a unique hydrophilic N-terminal sequence of unknown function and the C-terminal hydrophobic segment of COXIII. To extend the analysis of alternative editing in Trypanosoma brucei we have constructed libraries with over 1100 full-length mitochondrial cDNAs and the sequences of over 1200 gRNA genes. Using this data, we show that alternative editing of COXIII, ATPase subunit 6 (A6), and NADH dehydrogenase subunits 7, 8 and 9 (ND7, 8, 9) mRNAs can produce novel open reading frames (ORFs). Several gRNAs potentially responsible for the alternative editing of these mRNAs were also identified. These findings show that alternative editing of mitochondrial mRNAs is common in T. brucei and expands the diversity of mitochondrial proteins in these organisms."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0001566.g005"/>

  <meta property="og:title" content="Alternative mRNA Editing in Trypanosomes Is Extensive and May Contribute to Mitochondrial Protein Diversity" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001566" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=7824'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=8764'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=9541&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0001566">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001566" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001566&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Alternative mRNA Editing in Trypanosomes Is Extensive and May Contribute to Mitochondrial Protein Diversity
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Torsten Ochsenreiter, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Biochemistry and Molecular Biology, University of Georgia, Athens, Georgia, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Michael Cipriano, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Medical Microbiology, University of California Davis, Davis, California, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Stephen L. Hajduk
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p>*E-mail: <a href="mailto:shajduk@bmb.uga.edu">shajduk@bmb.uga.edu</a></p>

                <p>Affiliation:
                  Department of Biochemistry and Molecular Biology, University of Georgia, Athens, Georgia, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: February 13, 2008</li>
    <li>DOI: 10.1371/journal.pone.0001566</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0001566" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0001566" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001566" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0001566" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0001566-g001" data-doi="info:doi/10.1371/journal.pone.0001566" data-uri="info:doi/10.1371/journal.pone.0001566.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001566.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001566-g002" data-doi="info:doi/10.1371/journal.pone.0001566" data-uri="info:doi/10.1371/journal.pone.0001566.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001566.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001566-t001" data-doi="info:doi/10.1371/journal.pone.0001566" data-uri="info:doi/10.1371/journal.pone.0001566.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001566.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001566-g003" data-doi="info:doi/10.1371/journal.pone.0001566" data-uri="info:doi/10.1371/journal.pone.0001566.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001566.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001566-g004" data-doi="info:doi/10.1371/journal.pone.0001566" data-uri="info:doi/10.1371/journal.pone.0001566.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001566.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001566-g005" data-doi="info:doi/10.1371/journal.pone.0001566" data-uri="info:doi/10.1371/journal.pone.0001566.g005" title="Figure 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001566.g005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001566">Reader Comments (2)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0001566" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1"></a><p>The editing of trypanosome mitochondrial mRNAs produces transcripts necessary for mitochondrial functions including electron transport and oxidative phosphorylation. Precursor-mRNAs are often extensively edited by specific uridine insertion or deletion that is directed by small guide RNAs (gRNAs). Recently, it has been shown that cytochrome c oxidase subunit III (COXIII) mRNAs can be alternatively edited to encode a novel mitochondrial membrane protein composed of a unique hydrophilic N-terminal sequence of unknown function and the C-terminal hydrophobic segment of COXIII. To extend the analysis of alternative editing in <em>Trypanosoma brucei</em> we have constructed libraries with over 1100 full-length mitochondrial cDNAs and the sequences of over 1200 gRNA genes. Using this data, we show that alternative editing of COXIII, ATPase subunit 6 (A6), and NADH dehydrogenase subunits 7, 8 and 9 (ND7, 8, 9) mRNAs can produce novel open reading frames (ORFs). Several gRNAs potentially responsible for the alternative editing of these mRNAs were also identified. These findings show that alternative editing of mitochondrial mRNAs is common in <em>T. brucei</em> and expands the diversity of mitochondrial proteins in these organisms.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>Ochsenreiter T, Cipriano M, Hajduk SL (2008) Alternative mRNA Editing in Trypanosomes Is Extensive and May Contribute to Mitochondrial Protein Diversity. PLoS ONE 3(2):
          e1566.
            doi:10.1371/journal.pone.0001566</p><p><strong>Academic Editor: </strong>Michael Gray, Dalhousie University, Canada</p><p><strong>Received:</strong> October 18, 2007; <strong>Accepted:</strong> January 2, 2008; <strong>Published:</strong> February 13, 2008</p><p><strong>Copyright:</strong> © 2008 Ochsenreiter et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>This work was supported by a grant from the National Institutes of Health (AI21401).</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>RNA editing in trypanosome mitochondria is a posttranscriptional process of endonuclease cleavage, uridine insertion or deletion and ligation of mRNAs that is directed by small non-coding guide RNAs (gRNAs) (reviewed in <a href="#pone.0001566-MadisonAntenucci1">[1]</a>–<a href="#pone.0001566-Stuart1">[3]</a>). Functionally, RNA editing has been shown to correct for frameshift mutations, form initiation codons and create entire ORFs <a href="#pone.0001566-Benne1">[4]</a>–<a href="#pone.0001566-Koslowsky1">[6]</a>. The genes encoding the trypanosome mitochondrial mRNAs are found on maxicircles, while gRNAs are largely encoded by genes located on minicircles. These two classes of mitochondrial DNA form a unique structure called the kinetoplast composed of thousands of minicircles and approximately 50 maxicircles topologically interlocked to form a huge network structure. Uridine insertion/deletion editing effectively combines information at the RNA level that is encoded separately, on minicircles and maxicircles in the mitochondrial genome of trypanosomes <a href="#pone.0001566-Ochsenreiter1">[7]</a>. The complete and correct editing of trypanosome mitochondrial mRNAs is essential to the organism and is critical for the production of conventional mitochondrial proteins including subunits of the mitochondrial respiratory chain and the ATP synthase. Initial sequencing studies of extensively edited COXIII, ND7 and A6 mRNAs from <em>T. brucei</em> showed that the repertoire of distinctly edited mRNAs from a single mitochondrial gene was extensive but this sequence diversity was largely dismissed as a reflection of the inaccuracy and inefficiency of RNA editing resulting in incomplete or incorrectly edited RNAs that were functionally unimportant <a href="#pone.0001566-Abraham1">[8]</a>–<a href="#pone.0001566-Sturm1">[10]</a>. We have recently identified an alternatively edited COXIII mRNA and have shown its protein product associates in a high molecular weight complex in trypanosome mitochondrial membranes <a href="#pone.0001566-Ochsenreiter2">[11]</a>. The detection of an alternatively edited mRNA, and its protein product, suggested that differential editing of pre-mRNAs could play a role in expanding protein diversity by the production of functionally discrete isoforms of conventional mitochondrial proteins with different enzymatic activities, substrate specificities, subcellular localization or altered abilities to interact with other proteins, DNA or RNAs. Alternatively edited mRNAs might also create proteins with novel functions needed for the peculiar biogenesis and regulation of the mitochondrion of trypanosomes <a href="#pone.0001566-Liu1">[12]</a>, <a href="#pone.0001566-Ogbadoyi1">[13]</a>.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>Mitochondrial biogenesis in trypanosomes is developmentally regulated. In the bloodstream form of the parasite many of the mitochondrial respiratory complex proteins including cytochrome c oxidase and cytochrome c reductase complexes are down regulated. This is in good agreement with the physiological data that shows the absence of any oxidative phosphorylation in this developmental stage <a href="#pone.0001566-Matthews1">[14]</a>. The transcription and editing of some respiratory complex genes, including COXIII, however is not down regulated in the bloodstream stage. This suggests that the edited mRNAs from these genes may encode proteins that are functionally distinct from conventional respiratory complex proteins. Similarly, mRNAs from the NADH dehydrogenase (respiratory Complex I) gene ND7 are differentially edited during the developmental cycle of <em>T. brucei</em> <a href="#pone.0001566-Koslowsky1">[6]</a>. In this case, editing of ND7 mRNA in the insect vector stage of the parasite occurs only in its 5′ domain leaving the 3′ domain pre-edited. These studies did not investigate the presence of protein products from the alternative ND7 mRNAs but suggested that differential editing of ND7 mRNA might produce isoforms of ND7 specific to the developmental stages of the trypanosome.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>We initiated an analysis of the extent of alternative mRNA editing by large scale sequencing of full-length cDNAs from the bloodstream developmental stage of <em>T. brucei</em> <a href="#pone.0001566-Ochsenreiter2">[11]</a>, <a href="#pone.0001566-Ochsenreiter3">[15]</a>. The diversity of edited mitochondrial mRNAs suggested the formation of alternative ORFs might be widespread in trypanosomes. Here we show that mRNAs from five <em>T. brucei</em> mitochondrial genes are alternatively edited giving rise to minor amino acid substitutions, extended ORFs or extensive sequence changes when compared with pre-edited or <em>bona fide</em> edited transcripts. In addition, several gRNAs responsible for the alternative editing of these mRNAs are identified. Based on these data we propose that alternative editing of mitochondrial mRNAs is widespread in <em>T. brucei</em> and expands mitochondrial protein diversity.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3>
<h4>Alternative Editing of ND7, ND8 and ND9 mRNAs</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>In the insect developmental stage of <em>T. brucei</em> the mitochondrial NADH dehydrogenase (Complex I) functions as an electron acceptor for NADH thereby facilitating electron transport and playing a key role in establishment of mitochondrial membrane potential and oxidative phosphorylation <a href="#pone.0001566-Fang1">[16]</a>. Bloodstream <em>T. brucei</em> lacks detectable cytochrome c reductase (Complex III) and cytochrome c oxidase (Complex IV) activity. Instead Complex I donates a pair of electrons to a plant-like alternative terminal oxidase completing the reoxidation of NADH formed glycolytically during substrate level phosphorylation <a href="#pone.0001566-Chaudhuri1">[17]</a>.</p>
<a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2"></a><p>To determine whether alternative mRNA editing could produce isoforms of the Complex I subunits we carried out a detailed analysis of cDNA sequence data from extensively edited ND7, ND8 and ND9 mRNAs isolated from the bloodstream developmental stage of <em>T. brucei</em>. The sequence of ND7 mRNA was originally deduced based on the consensus sequence of 77 short cDNAs and direct RNA sequencing <a href="#pone.0001566-Koslowsky1">[6]</a>. The <em>bona fide</em> ND7 mRNA contained 551 uridines added and 86 uridines deleted by RNA editing to form the 1246 nts mRNA. We have extended the characterization of ND7 RNA editing by detailed analysis of 21 full-length cDNAs and examination of the protein coding potential for each transcript. The sequence of ND7 mRNAs revealed two alternatively edited RNAs with unique ORFs (<a href="#pone-0001566-g001">Figure 1</a>). It was previously noted that differential editing of ND7 mRNAs could lead to diverse ORFs and dramatic differences in editing correlated with the developmental stage of the parasite <a href="#pone.0001566-Koslowsky1">[6]</a>. We have verified similar changes leading to frame shifting and amino acid substitutions in the ND7 mRNAs in bloodstream <em>T. brucei</em>.</p>
<div class="figure" id="pone-0001566-g001"><div class="img"><a name="pone-0001566-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001566.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001566" data-uri="info:doi/10.1371/journal.pone.0001566.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001566.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001566.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001566.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001566.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001566.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001566.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001566.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001566.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001566.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Alternative editing of ND7 mRNAs.</span></strong></p><a id="article1.body1.sec2.sec1.fig1.caption1.p1" name="article1.body1.sec2.sec1.fig1.caption1.p1"></a><p>A) Alignment of the predicted amino acid sequence of ND7 and the alternatively edited ND7 cDNAs ND7-G10 and ND7-N12. Amino acids changed by alternative editing are in blue. B) Bar depicts the alternatively edited mRNA sequences ND7-G10 and ND7-N12. Alignment of the RNA sequences for ND7, ND7-N12 and ND7-G10 from the alternatively edited regions I, II and III are shown below the bar. In black are the pre-edited residues, in grey the <em>bona fide</em> edited residues and in red are the alternatively edited residues from regions I–III. Above the RNA sequences are the corresponding amino acid sequences. A predicted gRNA for N12 shows perfect complementarity (allowing for G:U) to ND7-N12 over 30 base pairs while having mismatches to the ND7 sequence at nucleotide positions 10 and 14 of the gRNA. Vertical bars indicate A:U or G:C base pairing; crosses indicates G:U base pairing. Underlined sequence shows UAG termination codon. Star depicts amber codon in the amino acid sequence. Red Us indicate alternatively inserted/not inserted Us when compared with the ND7 sequence.</p>
<span>doi:10.1371/journal.pone.0001566.g001</span></div><a id="article1.body1.sec2.sec1.p3" name="article1.body1.sec2.sec1.p3"></a><p>The complete cDNA sequence of ND7 mRNA produces a predicted protein coding sequence of 386 amino acids. In addition to this fully edited ND7 sequence, we identified two alternatively edited ND7 mRNAs containing a total of seven alternatively inserted uridines at five editing sites (<a href="#pone-0001566-g001">Figure 1A</a>). One alternatively edited mRNA, designated ND7-G10, differs from the fully edited ND7 mRNA by the addition of a single uridine at nucleotide 1183 resulting in a frameshift that extends the predicted coding sequence of this alternatively edited mRNA by 21 nucleotides to a UAG termination codon. The C-terminus of this predicted ND7 isoform is rich in hydrophobic amino acids (two valine and five leucine residues) suggesting that it may uniquely associate with the mitochondrial membrane or assemble differently in the NADH dehydrogenase complex. Another differentially edited ND7 mRNA, designated ND7-N12, differs from the fully edited ND7 mRNA at four sites in two regions of the mRNA near the translation initiation codon resulting in four non-conservative amino acid substitutions (<a href="#pone-0001566-g001">Figure 1A, B</a>). Both alternatively edited transcripts use the <em>bona fide</em> ND7 initiation codon (AUG). In our gRNA gene database, we identified a gRNA gene complementary to 30 nts of the ND7-N12 cDNA that can direct the alternative editing of nucleotides 23–26 in region I. The gRNA directing the alternative editing of the ND7-G10 mRNA has not been identified.</p>
<a id="article1.body1.sec2.sec1.p4" name="article1.body1.sec2.sec1.p4"></a><p>Analysis of 78 full-length cDNAs from the ND8 gene also revealed alternatively edited mRNA. The ND8-F04 cDNA and two other identical sequences contained two regions where a total of 11 uridines are inserted alternatively at seven editing sites (<a href="#pone-0001566-g002">Figure 2A</a>, <a href="#pone-0001566-t001">Table 1</a>). This transcript contained two ORFs of similar size (<a href="#pone-0001566-g002">Figure 2A</a>). ORF1 started with an UUG (Leu) initiation codon 21 nucleotides downstream of the predicted 5′end and terminated with a UAA termination codon produced by alternative editing after 117 amino acids. When compared to the public databases this sequence showed no similarity to any known sequence. The second ORF (ORF2) started 47 nucleotides downstream of the 5′ end with a GUG (Val) start codon and had no termination codon in the ND8 coding sequence, however a termination codon (UAA) was found in the poly A tail which terminated the sequence after 121 amino acids. The predicted amino acid sequence (ORF2) showed weak similarity over a short stretch to a hypothetical protein from <em>Plasmodium chabaudi</em> (XP_744827).</p>
<div class="figure" id="pone-0001566-g002"><div class="img"><a name="pone-0001566-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001566.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001566" data-uri="info:doi/10.1371/journal.pone.0001566.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001566.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001566.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001566.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001566.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001566.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001566.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001566.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001566.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001566.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Alternative editing of ND8 mRNAs.</span></strong></p><a id="article1.body1.sec2.sec1.fig2.caption1.p1" name="article1.body1.sec2.sec1.fig2.caption1.p1"></a><p>A) Protein sequence of ORF1 and ORF2, both sequences show no similarity to the <em>bona fide</em> ND8 (not shown). B) Bar depicts the mRNA sequence ND8-F04. Alignment of the RNA sequences for ND8, and ND8-F04 from the alternatively edited regions I, and II are shown below the bar. In black are the pre-edited residues, in grey the <em>bona fide</em> edited residues and in red are the alternatively edited residues from regions I–II. In black is the pre-edited, in grey the <em>bona fide</em> edited region and in red are the alternatively edited regions I and II. Alternative start codons UUG and GUG are shown in black and purple, respectively. Alignment of the RNA sequence of ND8 and cDNA ND8-F04 from the alternatively edited region I and II (red). Depicted above the RNA sequences are the corresponding amino acid sequences. Predicted gRNA for region I showing perfect complementarity (allowing for G:U) to ND8-F04 over 41 base pairs while having three mismatches to the ND8 sequence at positions 38 to 40 of the gRNA. Vertical bars indicate A:U or G:C base pairing; crosses indicates G:U base pairing. Red Us indicate alternatively inserted Us when compared with the <em>bona fide</em> ND8 sequence. Star depicts termination codon in the amino acid sequence. Underlined sequence depicts stop codon in the nucleotide sequence.</p>
<span>doi:10.1371/journal.pone.0001566.g002</span></div><div class="figure" id="pone-0001566-t001"><div class="img"><a name="pone-0001566-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001566.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001566" data-uri="info:doi/10.1371/journal.pone.0001566.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001566.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001566.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001566.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001566.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001566.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001566.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001566.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001566.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001566.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>Features of alternatively edited transcripts</span></strong></p><span>doi:10.1371/journal.pone.0001566.t001</span></div><a id="article1.body1.sec2.sec1.p5" name="article1.body1.sec2.sec1.p5"></a><p>Screening our gRNA database, we identified a corresponding gRNA gene that would be able to guide for the insertion of the 3′ most alternatively inserted uridine residues at nucleotide position 274-76 (<a href="#pone-0001566-g002">Figure 2B</a>) <a href="#pone.0001566-Ochsenreiter3">[15]</a>. The gRNA gene matched with the alternative transcript over 41 bp with 11 G:U base pairings. The five 3′ most nucleotides of the gRNA gene guided for the alternative insertion pattern observed in the ND8 transcript.</p>
<a id="article1.body1.sec2.sec1.p6" name="article1.body1.sec2.sec1.p6"></a><p>Analysis of 126 full-length ND9 cDNAs identified six alternatively edited transcripts of identical sequence, containing a novel open reading frame (designated ND9-F12, <a href="#pone-0001566-g003">Figure 3A</a>). The mRNA sequence was identical to the consensus for ND9 in the 3′ terminal 366 nucleotides while an extended 5′ region of the pre-mRNA was edited differently at 67 editing sites. An alternative ORF was identified to start with GUG (Val) at nucleotide 31 and terminated at nucleotide 580 with the <em>bona fide</em> termination codon of ND9 mRNA (UAA). The C-terminal 113 amino acids of the predicted alternative protein were identical to ND9, whereas the N-terminal sequence of 67 amino acids showed no homology to ND9. When this sequence was compared to the public databases no sequences with significant similarity were found.</p>
<div class="figure" id="pone-0001566-g003"><div class="img"><a name="pone-0001566-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001566.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001566" data-uri="info:doi/10.1371/journal.pone.0001566.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001566.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001566.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001566.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001566.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001566.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001566.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001566.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001566.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001566.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Alternative editing of ND9 mRNA.</span></strong></p><a id="article1.body1.sec2.sec1.fig3.caption1.p1" name="article1.body1.sec2.sec1.fig3.caption1.p1"></a><p>A) Alignment of the predicted amino acid sequence of ND9 and the alternatively edited cDNA ND9-F12. Amino acids changed by alternative editing are in blue. B) Bar depicts the ND9-F12 mRNA. Alignment of the RNA sequences for ND9, and ND9-F12 from the alternatively edited region are shown below the bar. In black are the pre-edited residues, in grey the <em>bona fide</em> edited residues and in red are the alternatively edited residues. The initiation codon GUG is shown in black on the bar. Above the RNA sequences are the corresponding amino acid sequences. Predicted gRNA for ND9-F12 shows perfect complementarity (allowing for G:U) to F12 over 44 base pairs while having 14 mismatches to the ND9 sequence in the best possible alignment (not shown). Vertical bars indicate A:U or G:C base pairing; crosses indicates G:U base pairing. Star indicates a termination codon. Red Us indicate alternatively inserted Us when compared with the ND9 sequence.</p>
<span>doi:10.1371/journal.pone.0001566.g003</span></div><a id="article1.body1.sec2.sec1.p7" name="article1.body1.sec2.sec1.p7"></a><p>We identified a gRNA gene, as well as the expressed gRNA, in our cDNA libraries predicted to direct the alternative insertion of at least 23 uridine residues (<a href="#pone-0001566-g003">Figure 3B</a>) assuming an 8 bp gRNA anchor length. The gRNA showed perfect complementarity over 44 nucleotides with the alternative transcript and no significant complementarity to the fully edited ND9 transcript.</p>


<h4>Alternative Editing of COXIII mRNA</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>We previously reported the alternative editing of COXIII mRNA and have identified a protein, AEP-1, encoded by this mRNA <a href="#pone.0001566-Ochsenreiter2">[11]</a>. Another alternatively edited COXIII mRNA, COXIII-K12, shows a similar sequence pattern to AEP-1 mRNA containing a 210 nt pre-edited 5′ region with an ORF joined at an alternatively edited junction region (nucleotides 210 to 225) to a sequence identical to COXIII resulting in an ORF of 223 amino acids (<a href="#pone-0001566-g004">Figure 4A, B</a>; <a href="#pone-0001566-t001">Table 1</a>). Editing at the junction region of COXIII-K12 results in a +1 frameshift to create an extended ORF with the 5′ pre-edited and 3′ edited ORFs joined. In addition, changes in the position of uridine insertions result in three amino acid substitutions in the junction region.</p>
<div class="figure" id="pone-0001566-g004"><div class="img"><a name="pone-0001566-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001566.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001566" data-uri="info:doi/10.1371/journal.pone.0001566.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001566.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001566.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001566.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001566.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001566.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001566.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001566.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001566.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001566.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>Alternative editing of COXIII mRNA.</span></strong></p><a id="article1.body1.sec2.sec2.fig1.caption1.p1" name="article1.body1.sec2.sec2.fig1.caption1.p1"></a><p>A) Alignment of the predicted amino acid sequence of COXIII and the alternatively edited COXIII-K12 cDNA. B) Bar represents the mRNA sequence of COXIII-K12. 5′ and 3′ region are connected by a thin line. Alignment of the RNA sequences for COIII, and COIII-K12 from the alternatively edited regions I, II, III and IV are shown below the bar. In black are the pre-edited residues, in grey the <em>bona fide</em> edited residues and in red are the alternatively edited residues from regions I–IV. The UUG initiation codon is shown in black. Alignment of the RNA sequence of COXIII and cDNA K12 from the alternatively edited region I–IV (red). Above the RNA sequences are the corresponding amino acid sequences of K12 and COXIII. Predicted gRNA sequence shows perfect complementarity (allowing for G:U) to K12 over 51 base pairs while having one mismatch to the COXIII sequence at position 38 of the gRNA. Vertical bars indicate A:U or G:C base pairing; crosses indicates G:U base pairing. Red Us indicate alternatively inserted Us when compared with the COXIII sequence. Star depicts termination codon in the amino acid sequence. Underlined sequence depicts stop codon in the nucleotide sequence.</p>
<span>doi:10.1371/journal.pone.0001566.g004</span></div><a id="article1.body1.sec2.sec2.p2" name="article1.body1.sec2.sec2.p2"></a><p>We identified a gRNA gene for the alternative editing site of region I, between nucleotides 210 and 225, of the COXIII-K12 cDNA. The gRNA showed perfect complementarity over 50 nucleotides with the alternative COXIII-K12 and is predicted to direct the insertion of 10 uridines unique to the COXIII-K12 cDNA (<a href="#pone-0001566-g004">Figure 4B</a>).</p>
<a id="article1.body1.sec2.sec2.p3" name="article1.body1.sec2.sec2.p3"></a><p>In addition to alternative editing of the COXIII-K12 mRNA at the junction of the pre-edited and edited COXIII mRNA, three other alternatively edited regions were identified in this cDNA (<a href="#pone-0001566-g004">Figure 4B</a>). The alternatively edited region II (nucleotides 404 to 427) differs from the consensus COXIII sequence in the number of uridines added at nine sites resulting in eight predicted changes in the amino acid sequence of COXIII-K12. Alternative editing of region III (nucleotides 608 to 609) results in the addition of one less uridine thereby shifting the predicted reading frame of the COXIII-K12. The shift in reading frame results in the addition of 10 amino acids until the in-frame stop codon (UAG) at nucleotide 690. Finally, a fourth alternative editing site for the COXIII-K12 mRNA was observed in the 3′ untranslated region (UTR) starting 11 nucleotides downstream of the UAG termination codon. We have not identified gRNAs or gRNA genes that could direct the alternative editing at the sites II, III and IV of the COIII-K12 mRNA.</p>


<h4>Alternative Editing of A6 mRNA</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>Analysis of 69 full length A6 cDNAs also identified three alternatively edited cDNAs of identical sequence, designated A6-D08. This sequence contained an ORF that differed from the A6 consensus sequence (<a href="#pone-0001566-g005">Figure 5A</a>). The ORF was initiated by an alternative UUG start codon at nucleotide position 40 and terminated by the <em>bona fide</em> A6 stop codon. The mRNA sequence of A6-D08 was identical to the consensus for A6 at nucleotides 323 to 517. The 5′ 208 nucleotides were the pre-edited sequence encoded by the A6 gene while three regions (I, 260–265; II, 323 and III 517–535 <a href="#pone-0001566-t001">Table 1</a>) were edited to a unique sequence. The formation of the A6-D08 ORF requires the correct alignment of three distinct ORFs encoded by pre-edited, alternatively edited and fully edited A6 at two junctions, one between the pre-edited and edited sequence and the second at the alternatively edited region II (<a href="#pone-0001566-g005">Figure 5B</a>). When the corresponding amino acid sequence was compared to the public databases no sequences with significant similarity were found. We have identified a gRNA gene to the alternatively edited region I of A6-D08 mRNA. The gRNA is predicted to have perfect complementarity over 40 bp to the alternative A6-D08 mRNA from nucleotide position 248-288.</p>
<div class="figure" id="pone-0001566-g005"><div class="img"><a name="pone-0001566-g005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001566.g005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001566" data-uri="info:doi/10.1371/journal.pone.0001566.g005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001566.g005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001566.g005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001566.g005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001566.g005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001566.g005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001566.g005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001566.g005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001566.g005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001566.g005.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 5.  <span>Alternative editing of A6 mRNA.</span></strong></p><a id="article1.body1.sec2.sec3.fig1.caption1.p1" name="article1.body1.sec2.sec3.fig1.caption1.p1"></a><p>A) Alignment of the predicted amino acid sequence of A6 and the alternatively edited cDNA A6-D08. B) Alignment of the RNA sequences for A6, and A6-D08 from the alternatively edited regions I, and II are shown below the bar. In black are the pre-edited residues, in grey the <em>bona fide</em> edited residues and in red are the alternatively edited residues from regions I–III.Bar represents the mRNA sequence of A6-D08. The UUG initiation codon is shown in black. Alignment of the RNA sequence of A6 and A6-D08 from the alternatively edited regions I, II and III (red). Depicted above the RNA sequences are the corresponding amino acid sequences of D08 and A6. The predicted gRNA shows perfect complementarity (allowing for G:U) to A6-D08 over 40 base pairs while having two mismatches to the A6 sequence at position 26-27 of the gRNA. Vertical bars indicate A:U or G:C base pairing; crosses indicates G:U base pairing. Red Us indicate alternatively inserted Us when compared with the A6 sequence. Star shows termination codon in the amino acid sequence.</p>
<span>doi:10.1371/journal.pone.0001566.g005</span></div>
</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p>The discovery of RNA editing provided an explanation for the seeming lack of several typical mitochondrial genes in trypanosomes <a href="#pone.0001566-Benne1">[4]</a>. Many mitochondrial mRNAs are so extensively edited that their gene sequences failed to reveal the ORFs for conventional protein products. The insertion and deletion of hundreds of uridines leads to the formation of functional mRNAs for mitochondrial proteins including components of the major mitochondrial respiratory complexes and the ATP synthase. These findings led to the widely accepted conclusion that the function of RNA editing was to amend the incomplete coding information of trypanosome mitochondrial genes to produce mRNA for conventional mitochondrial proteins.</p>
<a id="article1.body1.sec3.p2" name="article1.body1.sec3.p2"></a><p>The recent discovery that alternative RNA editing of a COXIII mRNA can create an mRNA that is translated to produce a novel protein, AEP-1, suggests that alternative editing may provide a powerful means to fine-tune and diversify trypanosome mitochondrial gene products <a href="#pone.0001566-Ochsenreiter2">[11]</a>. In this paper, we have extended our analysis of the diversity of <em>T. brucei</em> mitochondrial mRNAs and report the sequence of alternatively edited A6, ND7, ND8, ND9 mRNAs and a second alternatively edited mRNA for COXIII.</p>
<a id="article1.body1.sec3.p3" name="article1.body1.sec3.p3"></a><p>Based on the widespread occurrence of alternatively edited mRNAs shown in this study, we propose that alternative RNA editing in trypanosomes be defined as any uridine addition or deletion that produces ORFs which differ from the anticipated coding sequence for mitochondrial proteins. We show that alternative mRNA editing can affect protein-coding sequences in five ways. 1) Alternative uridine insertion or deletion can cause amino acid substitutions within coding sequences (ND7-N12, COXIII-K12). 2) Alternative editing can occur at pre-edited/edited junctions to create extended ORFs composed of 5′ pre-edited and 3′ edited sequences (ND8-F04, COXIII-K12, A6-D08). 3) Alternative uridine insertion and deletion in mRNAs can form new or eliminate canonical translation termination codons to extend or shorten ORFs (ND7-N12 and COXIII-K12), in some instances this occurs in the poly(A) tail (A6,-D08) as has been described previously <a href="#pone.0001566-Feagin1">[5]</a>, <a href="#pone.0001566-Landweber1">[18]</a>. 4) Alternative editing of an mRNA can be extensive resulting in formation of RNA sequences containing unique ORFs differing from those in either the pre-edited or fully edited mRNA (A6, ND9). 5) Alternative editing can change the UTR sequence and thereby might affect regulation of expression or stability of certain transcripts (COXIII-K12, A6-D08).</p>
<a id="article1.body1.sec3.p4" name="article1.body1.sec3.p4"></a><p>Currently, the function of alternative mRNA editing in trypanosome mitochondria is unknown but by analogy to alternative RNA splicing in many eukaryotes it seems likely that a major function for alternative editing will be to generate isoforms of a protein with different enzymatic activities, substrate specificities, cellular localization and altered ability to interact with other proteins or nucleic acids <a href="#pone.0001566-Maniatis1">[19]</a>–<a href="#pone.0001566-Stamm1">[21]</a>. Alternative editing may also be important in producing mitochondrial proteins with novel functions specific to trypanosomes <a href="#pone.0001566-Ochsenreiter1">[7]</a>. Consistent with this possibility, AEP-1 is encoded by an alternately edited COXIII mRNA that is expressed in the bloodstream developmental stage of <em>T. brucei</em> when the expression of all other nuclear and mitochondrial encoded subunits of the cytochrome c oxidase is suppressed and the activity of the complex is not detectable. The absence of cytochrome c oxidase activity suggests that AEP-1 has a novel function in trypanosome mitochondria and supports the idea that alternative mRNA editing is likely to provide a mechanism for creating mitochondrial protein diversity. This is supported by previous studies in Kinetoplastidae that showed that RNA editing could be a source of genetic variation <a href="#pone.0001566-Landweber1">[18]</a>. These studies provided evidence that the predicted proteins of extensively edited genes accumulate mutations at a higher frequency than their unedited homologues in closely related species. Landweber and Gilbert concluded that RNA editing is a surprisingly inefficient mechanism for conserving amino-acid sequences.</p>
<a id="article1.body1.sec3.p5" name="article1.body1.sec3.p5"></a><p>With the data presented here we believe the rapid evolution of proteins from extensively edited transcripts is a consequence of RNA editing being a diversifying, rather than a conservation/repair mechanism.</p>
<a id="article1.body1.sec3.p6" name="article1.body1.sec3.p6"></a><p>The production of variations of the limited mitochondrial encoded proteins may be particularly important in trypanosomes as they use diverse mechanisms for the maintenance of their mitochondrial genome, and the developmental regulation of mitochondrial biogenesis and energy production. Furthermore, a correlation in the developmental regulation of mitochondrial biogenesis and both the extent and the diversity of RNA editing has been observed in <em>T. brucei</em> <a href="#pone.0001566-Koslowsky2">[22]</a>–<a href="#pone.0001566-Riley1">[25]</a>.</p>
<a id="article1.body1.sec3.p7" name="article1.body1.sec3.p7"></a><p>One of the underlying premises of RNA editing has been that ORFs were formed in the mRNA by uridine addition or deletion thus, pre-edited mRNA sequences could be considered the equivalent to non-coding RNA and would be anticipated to lack any significant ORFs. In our analysis of extensively edited mRNA, however we noticed that the pre-edited mRNAs contained multiple, extended ORFs (<a href="#pone.0001566.s002">Table S1</a>), while transcripts that are never edited, for example ND5, contain only one ORF.</p>
<a id="article1.body1.sec3.p8" name="article1.body1.sec3.p8"></a><p>So why do the extensively edited trypanosome mitochondrial genes contain multiple ORFs although their mRNAs do not contain complete coding information until edited? The importance of ORFs in pre-edited mRNA becomes clear if we consider it in conjunction with alternative editing. We have shown that ORFs in the pre-edited mRNA are required for the creation of the alternative ORFs for ND8, COXIII and A6 (<a href="#pone-0001566-g002">Figures 2</a>, <a href="#pone-0001566-g004">4</a> and <a href="#pone-0001566-g005">5</a>). In each of these alternatively edited mRNAs the pre-edited mRNA sequence contains the initiation codon as well as 5′ coding sequence, which is joined to a second ORF by alternative RNA editing. Together the pre-edited reading frame and the edited or alternatively edited reading frame create an extended, novel coding sequence. Seven of the nine pan edited genes contain up to three ORFs that start within 60 nucleotides of the 5′ end of the pre-edited transcript (<a href="#pone.0001566.s002">Table S1</a>). This increased density of ORFs points toward a functional requirement and we hypothesize that the increase in ORF density is a consequence of the specific loss of termination codons in these mitochondrial genes. In several instances, alternative open reading frames start with a non-conventional initiation codon. This is not unusual and has been observed previously in edited and non-edited mitochondrial genes from different trypanosome species (<a href="#pone.0001566.s003">Table S2</a>). Furthermore, mitochondrial alternative initiation codons also have been observed in other protozoan and non-protozoan systems <a href="#pone.0001566-Edqvist1">[26]</a>,<a href="#pone.0001566-Chang1">[27]</a>.</p>
<a id="article1.body1.sec3.p9" name="article1.body1.sec3.p9"></a><p>We have been able to identify one potential gRNA for each of the alternatively edited mRNAs described. Based on these findings it is likely that gRNAs specify the protein coding information in alternatively edited mRNAs and that the mechanism of RNA editing does not differ for mRNAs encoding conventional or alternative mitochondrial mRNAs; different gRNAs are simply used. The diversity of the minicircle coding potential has always been a puzzling phenomenon as the number of minicircle encoded gRNA genes far exceeds the diversity needed for editing of the conventional mitochondrial mRNAs <a href="#pone.0001566-Ochsenreiter3">[15]</a>, <a href="#pone.0001566-Corell1">[28]</a>. Here we show that some of these gRNAs are engaged in alternative editing of mRNAs, leading to novel ORFs.</p>
<a id="article1.body1.sec3.p10" name="article1.body1.sec3.p10"></a><p>It is unknown what controls the selection of gRNAs at junction regions when gRNAs for both conventional and alternative editing can basepair with partially edited mRNA. Currently, the database of gRNA coding sequences in <em>T. brucei</em> is incomplete and with the addition of information from ongoing sequencing projects of small RNAs we anticipate the identification of all gRNAs necessary for the editing of mRNAs for conventional and alternative mitochondrial proteins (Ochsenreiter and Hajduk, unpublished results). Bioinformatic comparison of alternative and conventional gRNAs might reveal differences in primary or secondary structure and lead to an understanding of how selection of gRNAs for conventional or alternative editing is controlled.</p>
</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4>Trypanosomes and RNA isolation</h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p>Pleomorphic long slender bloodstream form <em>T. brucei</em> (TREU 667) were harvested from infected rats at day three of infection at a parasitemia of 1–2×10<sup>8</sup> cells/ml. Cells were lysed and mitochondria were purified as described by Harris <em>et al</em> (1990). RNA was isolated from mitochondria using the TriPure Isolation reagent (Boehringer Mannheim, Mannheim, Germany).</p>


<h4>Cloning and sequencing</h4>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1"></a><p>Minicircles and gRNAs were cloned and sequenced as described previously <a href="#pone.0001566-Ochsenreiter3">[15]</a>. Mitochondrial cDNA libraries from bloodstream <em>T. brucei</em> mitochondrial mRNAs were made from 5 µg of mitochondrial RNA using the Creator SMART cDNA library construction kit (Clontech, Palo Alto, USA). First and second strand cDNA were synthesized using poly-dT oligonucleotides and the SMART strand switching technology. In a second step we amplified (24 cycles) the cDNAs using oligonucleotides specific for the 5′ UTR of ND7, ND8, ND9, COXIII and A6 as well as the manufacturers 3′ oligonucleotide (creator smart 3′). Multiple bands can be seen on the agarose gels (<a href="#pone.0001566.s001">Figure S1</a>) they reflect the differentially edited mRNA populations from one gene sequence. Sequencing was done using the universal primer sequences (M13F and M13R) in the vector and only high quality sequences (Phred Q &gt;20) were used for the analysis. For ND7 a second primer set was derived to create sufficient high quality sequence (<a href="#pone.0001566.s004">Table S3</a>).</p>


<h4>Bioinformatics</h4>
<a id="article1.body1.sec4.sec3.p1" name="article1.body1.sec4.sec3.p1"></a><p>All bioinformatics analysis was carried out using the EMBOSS software package and our previously published database KISS <a href="#pone.0001566-Ochsenreiter3">[15]</a>, <a href="#pone.0001566-Rice1">[29]</a>. Potential gRNAs were predicted using WUBLAST with a modified matrix to allow for G:U base pairings. The prediction of open reading frames was done with GETORF from the EMBOSS package using the protozoan mitochondrial genetic code EGC.4 from NCBI <a href="#pone.0001566-Aldritt1">[30]</a>.</p>

</div>

<div id="section5" class="section"><a id="s5" name="s5" toc="s5" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0001566"></div><a name="pone.0001566.s001" id="pone.0001566.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001566.s001">Figure S1. </a></strong></p><p class="siDoi">doi:10.1371/journal.pone.0001566.s001</p><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1"></a><p class="postSiDOI">(0.06 MB PDF)</p>
<a name="pone.0001566.s002" id="pone.0001566.s002"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001566.s002">Table S1. </a></strong></p><p class="siDoi">doi:10.1371/journal.pone.0001566.s002</p><a id="article1.body1.sec5.supplementary-material2.caption1.p1" name="article1.body1.sec5.supplementary-material2.caption1.p1"></a><p class="postSiDOI">(0.04 MB DOC)</p>
<a name="pone.0001566.s003" id="pone.0001566.s003"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001566.s003">Table S2. </a></strong></p><p class="siDoi">doi:10.1371/journal.pone.0001566.s003</p><a id="article1.body1.sec5.supplementary-material3.caption1.p1" name="article1.body1.sec5.supplementary-material3.caption1.p1"></a><p class="postSiDOI">(0.06 MB DOC)</p>
<a name="pone.0001566.s004" id="pone.0001566.s004"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001566.s004">Table S3. </a></strong></p><p class="siDoi">doi:10.1371/journal.pone.0001566.s004</p><a id="article1.body1.sec5.supplementary-material4.caption1.p1" name="article1.body1.sec5.supplementary-material4.caption1.p1"></a><p class="postSiDOI">(0.03 MB DOC)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>We thank Justin Widener for helpful discussion. We are grateful to Zac Wood, Bob Sabatini and John Harrington for critical reading of the manuscript.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: TO. Performed the experiments: TO. Analyzed the data: TO MC. Contributed reagents/materials/analysis tools: TO MC. Wrote the paper: SH TO.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0001566-MadisonAntenucci1" id="pone.0001566-MadisonAntenucci1"></a>Madison-Antenucci S, Grams J, Hajduk SL (2002) Editing machines: the complexities of trypanosome RNA editing. Cell  108: 435–438.  <ul class="find" data-citedArticleID="1052696" data-doi="10.1016/s0092-8674(02)00653-0"><li><a href="http://dx.doi.org/10.1016/s0092-8674(02)00653-0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Editing+machines%3A+the+complexities+of+trypanosome+RNA+editing." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Editing+machines%3A+the+complexities+of+trypanosome+RNA+editing.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0001566-Simpson1" id="pone.0001566-Simpson1"></a>Simpson L, Sbicego S, Aphasizhev R (2003) Uridine insertion/deletion RNA editing in trypanosome mitochondria: a complex business. Rna  9: 265–276.  <ul class="find" data-citedArticleID="1052716" data-doi="10.1261/rna.2178403"><li><a href="http://dx.doi.org/10.1261/rna.2178403" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Uridine+insertion%2Fdeletion+RNA+editing+in+trypanosome+mitochondria%3A+a+complex+business." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Uridine+insertion%2Fdeletion+RNA+editing+in+trypanosome+mitochondria%3A+a+complex+business.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0001566-Stuart1" id="pone.0001566-Stuart1"></a>Stuart KD, Schnaufer A, Ernst NL, Panigrahi AK (2005) Complex management: RNA editing in trypanosomes. Trends Biochem Sci  30: 97–105.  <ul class="find" data-citedArticleID="1052720" data-doi="10.1016/j.tibs.2004.12.006"><li><a href="http://dx.doi.org/10.1016/j.tibs.2004.12.006" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Complex+management%3A+RNA+editing+in+trypanosomes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Complex+management%3A+RNA+editing+in+trypanosomes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0001566-Benne1" id="pone.0001566-Benne1"></a>Benne R, Van den Burg J, Brakenhoff JP, Sloof P, Van Boom JH, et al.  (1986) Major transcript of the frameshifted coxII gene from trypanosome mitochondria contains four nucleotides that are not encoded in the DNA. Cell  46: 819–826.  <ul class="find" data-citedArticleID="1052668" data-doi="10.1016/0092-8674(86)90063-2"><li><a href="http://dx.doi.org/10.1016/0092-8674(86)90063-2" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Major+transcript+of+the+frameshifted+coxII+gene+from+trypanosome+mitochondria+contains+four+nucleotides+that+are+not+encoded+in+the+DNA." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Major+transcript+of+the+frameshifted+coxII+gene+from+trypanosome+mitochondria+contains+four+nucleotides+that+are+not+encoded+in+the+DNA.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0001566-Feagin1" id="pone.0001566-Feagin1"></a>Feagin JE, Abraham JM, Stuart K (1988) Extensive editing of the cytochrome c oxidase III transcript in Trypanosoma brucei. Cell  53: 413–422.  <ul class="find" data-citedArticleID="1052682" data-doi="10.1016/0092-8674(88)90161-4"><li><a href="http://dx.doi.org/10.1016/0092-8674(88)90161-4" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Extensive+editing+of+the+cytochrome+c+oxidase+III+transcript+in+Trypanosoma+brucei." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Extensive+editing+of+the+cytochrome+c+oxidase+III+transcript+in+Trypanosoma+brucei.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0001566-Koslowsky1" id="pone.0001566-Koslowsky1"></a>Koslowsky DJ, Bhat GJ, Perrollaz AL, Feagin JE, Stuart K (1990) The MURF3 gene of T. brucei contains multiple domains of extensive editing and is homologous to a subunit of NADH dehydrogenase. Cell  62: 901–911.  <ul class="find" data-citedArticleID="1052686" data-doi="10.1016/0092-8674(90)90265-g"><li><a href="http://dx.doi.org/10.1016/0092-8674(90)90265-g" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+MURF3+gene+of+T.+brucei+contains+multiple+domains+of+extensive+editing+and+is+homologous+to+a+subunit+of+NADH+dehydrogenase." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+MURF3+gene+of+T.+brucei+contains+multiple+domains+of+extensive+editing+and+is+homologous+to+a+subunit+of+NADH+dehydrogenase.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0001566-Ochsenreiter1" id="pone.0001566-Ochsenreiter1"></a>Ochsenreiter T, Hajduk SL (2007)  Function of RNA editing in Trypanosomes. In: Göringer HU, editor. Nucleic Acids and Molecular Biology. Heidelberg: Springer.   <ul class="find-nolinks"></ul></li><li><span class="label">8.
              </span><a name="pone.0001566-Abraham1" id="pone.0001566-Abraham1"></a>Abraham JM, Feagin JE, Stuart K (1988) Characterization of cytochrome c oxidase III transcripts that are edited only in the 3′ region. Cell  55: 267–272.  <ul class="find" data-citedArticleID="1052664" data-doi="10.1016/0092-8674(88)90049-9"><li><a href="http://dx.doi.org/10.1016/0092-8674(88)90049-9" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Characterization+of+cytochrome+c+oxidase+III+transcripts+that+are+edited+only+in+the+3%E2%80%B2+region." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Characterization+of+cytochrome+c+oxidase+III+transcripts+that+are+edited+only+in+the+3%E2%80%B2+region.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0001566-Decker1" id="pone.0001566-Decker1"></a>Decker CJ, Sollner-Webb B (1990) RNA editing involves indiscriminate U changes throughout precisely defined editing domains. Cell  61: 1001–1011.  <ul class="find" data-citedArticleID="1052676" data-doi="10.1016/0092-8674(90)90065-m"><li><a href="http://dx.doi.org/10.1016/0092-8674(90)90065-m" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=RNA+editing+involves+indiscriminate+U+changes+throughout+precisely+defined+editing+domains." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22RNA+editing+involves+indiscriminate+U+changes+throughout+precisely+defined+editing+domains.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0001566-Sturm1" id="pone.0001566-Sturm1"></a>Sturm NR, Simpson L (1990) Partially edited mRNAs for cytochrome b and subunit III of cytochrome oxidase from Leishmania tarentolae mitochondria: RNA editing intermediates. Cell  61: 871–878.  <ul class="find" data-citedArticleID="1052722" data-doi="10.1016/0092-8674(90)90197-m"><li><a href="http://dx.doi.org/10.1016/0092-8674(90)90197-m" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Partially+edited+mRNAs+for+cytochrome+b+and+subunit+III+of+cytochrome+oxidase+from+Leishmania+tarentolae+mitochondria%3A+RNA+editing+intermediates." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Partially+edited+mRNAs+for+cytochrome+b+and+subunit+III+of+cytochrome+oxidase+from+Leishmania+tarentolae+mitochondria%3A+RNA+editing+intermediates.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0001566-Ochsenreiter2" id="pone.0001566-Ochsenreiter2"></a>Ochsenreiter T, Hajduk SL (2006) Alternative editing of cytochrome c oxidase III mRNA in trypanosome mitochondria generates protein diversity. EMBO Rep  7: 1128–1133.  <ul class="find" data-citedArticleID="1052704" data-doi="10.1038/sj.embor.7400817"><li><a href="http://dx.doi.org/10.1038/sj.embor.7400817" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Alternative+editing+of+cytochrome+c+oxidase+III+mRNA+in+trypanosome+mitochondria+generates+protein+diversity." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Alternative+editing+of+cytochrome+c+oxidase+III+mRNA+in+trypanosome+mitochondria+generates+protein+diversity.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0001566-Liu1" id="pone.0001566-Liu1"></a>Liu B, Liu Y, Motyka SA, Agbo EE, Englund PT (2005) Fellowship of the rings: the replication of kinetoplast DNA. Trends Parasitol  21: 363–369.  <ul class="find" data-citedArticleID="1052692" data-doi="10.1016/j.pt.2005.06.008"><li><a href="http://dx.doi.org/10.1016/j.pt.2005.06.008" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Fellowship+of+the+rings%3A+the+replication+of+kinetoplast+DNA." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Fellowship+of+the+rings%3A+the+replication+of+kinetoplast+DNA.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0001566-Ogbadoyi1" id="pone.0001566-Ogbadoyi1"></a>Ogbadoyi EO, Robinson DR, Gull K (2003) A high-order trans-membrane structural linkage is responsible for mitochondrial genome positioning and segregation by flagellar basal bodies in trypanosomes. Mol Biol Cell  14: 1769–1779.  <ul class="find" data-citedArticleID="1052708" data-doi="10.1091/mbc.e02-08-0525"><li><a href="http://dx.doi.org/10.1091/mbc.e02-08-0525" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+high-order+trans-membrane+structural+linkage+is+responsible+for+mitochondrial+genome+positioning+and+segregation+by+flagellar+basal+bodies+in+trypanosomes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+high-order+trans-membrane+structural+linkage+is+responsible+for+mitochondrial+genome+positioning+and+segregation+by+flagellar+basal+bodies+in+trypanosomes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0001566-Matthews1" id="pone.0001566-Matthews1"></a>Matthews KR (2005) The developmental cell biology of Trypanosoma brucei. J Cell Sci  118: 283–290.  <ul class="find" data-citedArticleID="1052700" data-doi="10.1242/jcs.01649"><li><a href="http://dx.doi.org/10.1242/jcs.01649" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+developmental+cell+biology+of+Trypanosoma+brucei." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+developmental+cell+biology+of+Trypanosoma+brucei.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0001566-Ochsenreiter3" id="pone.0001566-Ochsenreiter3"></a>Ochsenreiter T, Cipriano M, Hajduk SL (2007) KISS: the kinetoplastid RNA editing sequence search tool. Rna  13: 1–4.  <ul class="find" data-citedArticleID="1052706" data-doi="10.1261/rna.232907"><li><a href="http://dx.doi.org/10.1261/rna.232907" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=KISS%3A+the+kinetoplastid+RNA+editing+sequence+search+tool." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22KISS%3A+the+kinetoplastid+RNA+editing+sequence+search+tool.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0001566-Fang1" id="pone.0001566-Fang1"></a>Fang J, Wang Y, Beattie DS (2001) Isolation and characterization of complex I, rotenone-sensitive NADH: ubiquinone oxidoreductase, from the procyclic forms of Trypanosoma brucei. Eur J Biochem  268: 3075–3082.  <ul class="find" data-citedArticleID="1052680" data-doi="10.1046/j.1432-1327.2001.02205.x"><li><a href="http://dx.doi.org/10.1046/j.1432-1327.2001.02205.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Isolation+and+characterization+of+complex+I%2C+rotenone-sensitive+NADH%3A+ubiquinone+oxidoreductase%2C+from+the+procyclic+forms+of+Trypanosoma+brucei." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Isolation+and+characterization+of+complex+I%2C+rotenone-sensitive+NADH%3A+ubiquinone+oxidoreductase%2C+from+the+procyclic+forms+of+Trypanosoma+brucei.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0001566-Chaudhuri1" id="pone.0001566-Chaudhuri1"></a>Chaudhuri M, Ott RD, Hill GC (2006) Trypanosome alternative oxidase: from molecule to function. Trends Parasitol  22: 484–491.  <ul class="find" data-citedArticleID="1052672" data-doi="10.1016/j.pt.2006.08.007"><li><a href="http://dx.doi.org/10.1016/j.pt.2006.08.007" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Trypanosome+alternative+oxidase%3A+from+molecule+to+function." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Trypanosome+alternative+oxidase%3A+from+molecule+to+function.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0001566-Landweber1" id="pone.0001566-Landweber1"></a>Landweber LF, Gilbert W (1993) RNA editing as a source of genetic variation. Nature  363: 179–182.  <ul class="find" data-citedArticleID="1052690" data-doi="10.1038/363179a0"><li><a href="http://dx.doi.org/10.1038/363179a0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=RNA+editing+as+a+source+of+genetic+variation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22RNA+editing+as+a+source+of+genetic+variation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0001566-Maniatis1" id="pone.0001566-Maniatis1"></a>Maniatis T, Tasic B (2002) Alternative pre-mRNA splicing and proteome expansion in metazoans. Nature  418: 236–243.  <ul class="find" data-citedArticleID="1052698" data-doi="10.1038/418236a"><li><a href="http://dx.doi.org/10.1038/418236a" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Alternative+pre-mRNA+splicing+and+proteome+expansion+in+metazoans." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Alternative+pre-mRNA+splicing+and+proteome+expansion+in+metazoans.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0001566-Kabat1" id="pone.0001566-Kabat1"></a>Kabat JL, Barberan-Soler S, McKenna P, Clawson H, Farrer T, et al.  (2006) Intronic alternative splicing regulators identified by comparative genomics in nematodes. PLoS Comput Biol  2: e86.  <ul class="find" data-citedArticleID="1052684" data-doi="10.1371/journal.pcbi.0020086.eor"><li><a href="http://dx.doi.org/10.1371/journal.pcbi.0020086.eor" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Intronic+alternative+splicing+regulators+identified+by+comparative+genomics+in+nematodes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Intronic+alternative+splicing+regulators+identified+by+comparative+genomics+in+nematodes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0001566-Stamm1" id="pone.0001566-Stamm1"></a>Stamm S, Ben-Ari S, Rafalska I, Tang Y, Zhang Z, et al.  (2005) Function of alternative splicing. Gene  344: 1–20.  <ul class="find" data-citedArticleID="1052718" data-doi="10.1016/j.gene.2004.10.022"><li><a href="http://dx.doi.org/10.1016/j.gene.2004.10.022" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Function+of+alternative+splicing." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Function+of+alternative+splicing.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0001566-Koslowsky2" id="pone.0001566-Koslowsky2"></a>Koslowsky DJ, Riley GR, Feagin JE, Stuart K (1992) Guide RNAs for transcripts with developmentally regulated RNA editing are present in both life cycle stages of Trypanosoma brucei. Mol Cell Biol  12: 2043–2049.  <ul class="find" data-citedArticleID="1052688"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Guide+RNAs+for+transcripts+with+developmentally+regulated+RNA+editing+are+present+in+both+life+cycle+stages+of+Trypanosoma+brucei.&amp;auth=&amp;atitle=Guide+RNAs+for+transcripts+with+developmentally+regulated+RNA+editing+are+present+in+both+life+cycle+stages+of+Trypanosoma+brucei." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Guide+RNAs+for+transcripts+with+developmentally+regulated+RNA+editing+are+present+in+both+life+cycle+stages+of+Trypanosoma+brucei." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Guide+RNAs+for+transcripts+with+developmentally+regulated+RNA+editing+are+present+in+both+life+cycle+stages+of+Trypanosoma+brucei.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0001566-Read1" id="pone.0001566-Read1"></a>Read LK, Stankey KA, Fish WR, Muthiani AM, Stuart K (1994) Developmental regulation of RNA editing and polyadenylation in four life cycle stages of Trypanosoma congolense. Mol Biochem Parasitol  68: 297–306.  <ul class="find" data-citedArticleID="1052710" data-doi="10.1016/0166-6851(94)90174-0"><li><a href="http://dx.doi.org/10.1016/0166-6851(94)90174-0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Developmental+regulation+of+RNA+editing+and+polyadenylation+in+four+life+cycle+stages+of+Trypanosoma+congolense." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Developmental+regulation+of+RNA+editing+and+polyadenylation+in+four+life+cycle+stages+of+Trypanosoma+congolense.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0001566-Lukes1" id="pone.0001566-Lukes1"></a>Lukes J, Hashimi H, Zikova A (2005) Unexplained complexity of the mitochondrial genome and transcriptome in kinetoplastid flagellates. Curr Genet  48: 277–299.  <ul class="find" data-citedArticleID="1052694" data-doi="10.1007/s00294-005-0027-0"><li><a href="http://dx.doi.org/10.1007/s00294-005-0027-0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Unexplained+complexity+of+the+mitochondrial+genome+and+transcriptome+in+kinetoplastid+flagellates." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Unexplained+complexity+of+the+mitochondrial+genome+and+transcriptome+in+kinetoplastid+flagellates.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0001566-Riley1" id="pone.0001566-Riley1"></a>Riley GR, Myler PJ, Stuart K (1995) Quantitation of RNA editing substrates, products and potential intermediates: implications for developmental regulation. Nucleic Acids Res  23: 708–712.  <ul class="find" data-citedArticleID="1052714" data-doi="10.1093/nar/23.4.708"><li><a href="http://dx.doi.org/10.1093/nar/23.4.708" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Quantitation+of+RNA+editing+substrates%2C+products+and+potential+intermediates%3A+implications+for+developmental+regulation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Quantitation+of+RNA+editing+substrates%2C+products+and+potential+intermediates%3A+implications+for+developmental+regulation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0001566-Edqvist1" id="pone.0001566-Edqvist1"></a>Edqvist J, Burger G, Gray MW (2000) Expression of mitochondrial protein-coding genes in Tetrahymena pyriformis. J Mol Biol  297: 381–393.  <ul class="find" data-citedArticleID="1052678" data-doi="10.1006/jmbi.2000.3530"><li><a href="http://dx.doi.org/10.1006/jmbi.2000.3530" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Expression+of+mitochondrial+protein-coding+genes+in+Tetrahymena+pyriformis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Expression+of+mitochondrial+protein-coding+genes+in+Tetrahymena+pyriformis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0001566-Chang1" id="pone.0001566-Chang1"></a>Chang KJ, Wang CC (2004) Translation initiation from a naturally occurring non-AUG codon in Saccharomyces cerevisiae. J Biol Chem  279: 13778–13785.  <ul class="find" data-citedArticleID="1052670" data-doi="10.1074/jbc.m311269200"><li><a href="http://dx.doi.org/10.1074/jbc.m311269200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Translation+initiation+from+a+naturally+occurring+non-AUG+codon+in+Saccharomyces+cerevisiae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Translation+initiation+from+a+naturally+occurring+non-AUG+codon+in+Saccharomyces+cerevisiae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0001566-Corell1" id="pone.0001566-Corell1"></a>Corell RA, Feagin JE, Riley GR, Strickland T, Guderian JA, et al.  (1993) Trypanosoma brucei minicircles encode multiple guide RNAs which can direct editing of extensively overlapping sequences. Nucleic Acids Res  21: 4313–4320.  <ul class="find" data-citedArticleID="1052674" data-doi="10.1093/nar/21.18.4313"><li><a href="http://dx.doi.org/10.1093/nar/21.18.4313" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Trypanosoma+brucei+minicircles+encode+multiple+guide+RNAs+which+can+direct+editing+of+extensively+overlapping+sequences." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Trypanosoma+brucei+minicircles+encode+multiple+guide+RNAs+which+can+direct+editing+of+extensively+overlapping+sequences.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0001566-Rice1" id="pone.0001566-Rice1"></a>Rice P, Longden I, Bleasby A (2000) EMBOSS: the European Molecular Biology Open Software Suite. Trends Genet  16: 276–277.  <ul class="find" data-citedArticleID="1052712" data-doi="10.1016/s0168-9525(00)02024-2"><li><a href="http://dx.doi.org/10.1016/s0168-9525(00)02024-2" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=EMBOSS%3A+the+European+Molecular+Biology+Open+Software+Suite." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22EMBOSS%3A+the+European+Molecular+Biology+Open+Software+Suite.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0001566-Aldritt1" id="pone.0001566-Aldritt1"></a>Aldritt SM, Joseph JT, Wirth DF (1989) Sequence identification of cytochrome b in Plasmodium gallinaceum. Mol Cell Biol  9: 3614–3620.  <ul class="find" data-citedArticleID="1052666"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Sequence+identification+of+cytochrome+b+in+Plasmodium+gallinaceum.&amp;auth=&amp;atitle=Sequence+identification+of+cytochrome+b+in+Plasmodium+gallinaceum." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Sequence+identification+of+cytochrome+b+in+Plasmodium+gallinaceum." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Sequence+identification+of+cytochrome+b+in+Plasmodium+gallinaceum.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001566.XML" value="72632"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001566.PDF" value="544623"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001566.g001.PNG_L" value="1209288"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001566.g001.PNG_M" value="170299"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001566.g001.PNG_S" value="8565"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001566.g001.TIF" value="1708672"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001566.g001.PNG_I" value="56770"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001566.g002.PNG_L" value="658565"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001566.g002.PNG_M" value="94468"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001566.g002.PNG_S" value="7796"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001566.g002.TIF" value="1051554"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001566.g002.PNG_I" value="32569"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001566.t001.PNG_L" value="162328"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001566.t001.PNG_M" value="97735"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001566.t001.PNG_S" value="11789"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001566.t001.TIF" value="783580"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001566.t001.PNG_I" value="29574"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001566.g003.PNG_L" value="593086"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001566.g003.PNG_M" value="90296"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001566.g003.PNG_S" value="6956"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001566.g003.TIF" value="961218"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001566.g003.PNG_I" value="31627"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001566.g004.PNG_L" value="1004430"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001566.g004.PNG_M" value="150059"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001566.g004.PNG_S" value="9549"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001566.g004.TIF" value="1569384"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001566.g004.PNG_I" value="52877"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001566.g005.PNG_L" value="742547"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001566.g005.PNG_M" value="114204"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001566.g005.PNG_S" value="8258"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001566.g005.TIF" value="1186716"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001566.g005.PNG_I" value="39941"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001566.s001.PDF" value="62593"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001566.s002.DOC" value="46592"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001566.s003.DOC" value="57856"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001566.s004.DOC" value="33280"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001566&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001566" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001566&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0001566&volume=&issue=&title=Alternative mRNA Editing in Trypanosomes Is Extensive and May Contribute to Mitochondrial Protein Diversity&author_name=Torsten%20Ochsenreiter%2C%20Michael%20Cipriano%2C%20Stephen%20L.%20Hajduk&start_page=1&end_page=9" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001566" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001566" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0001566" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001566&amp;t=Alternative%20mRNA%20Editing%20in%20Trypanosomes%20Is%20Extensive%20and%20May%20Contribute%20to%20Mitochondrial%20Protein%20Diversity" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001566&title=Alternative%20mRNA%20Editing%20in%20Trypanosomes%20Is%20Extensive%20and%20May%20Contribute%20to%20Mitochondrial%20Protein%20Diversity&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001566&amp;title=Alternative%20mRNA%20Editing%20in%20Trypanosomes%20Is%20Extensive%20and%20May%20Contribute%20to%20Mitochondrial%20Protein%20Diversity" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001566" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0001566&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Alternative mRNA Editing in Trypanosomes Is Extensive and May Contribute to Mitochondrial Protein Diversity';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0001566';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Alternative%20mRNA%20Editing%20in%20Trypanosomes%20Is%20Extensive%20and%20May%20Contribute%20to%20Mitochondrial%20Protein%20Diversity http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001566" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0001566" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">














          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Codons%22" title="Search for articles in the subject area:'Codons'"><div class="flagText">Codons</div></a>
              <div data-categoryid="18427" data-articleid="25958"
                   data-categoryname="Codons"
                   class="flagImage" title="Flag 'Codons' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22DNA+sequences%22" title="Search for articles in the subject area:'DNA sequences'"><div class="flagText">DNA sequences</div></a>
              <div data-categoryid="32003" data-articleid="25958"
                   data-categoryname="DNA sequences"
                   class="flagImage" title="Flag 'DNA sequences' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Gene+prediction%22" title="Search for articles in the subject area:'Gene prediction'"><div class="flagText">Gene prediction</div></a>
              <div data-categoryid="40279" data-articleid="25958"
                   data-categoryname="Gene prediction"
                   class="flagImage" title="Flag 'Gene prediction' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Messenger+RNA%22" title="Search for articles in the subject area:'Messenger RNA'"><div class="flagText">Messenger RNA</div></a>
              <div data-categoryid="18353" data-articleid="25958"
                   data-categoryname="Messenger RNA"
                   class="flagImage" title="Flag 'Messenger RNA' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Mitochondria%22" title="Search for articles in the subject area:'Mitochondria'"><div class="flagText">Mitochondria</div></a>
              <div data-categoryid="32453" data-articleid="25958"
                   data-categoryname="Mitochondria"
                   class="flagImage" title="Flag 'Mitochondria' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Protein+sequencing%22" title="Search for articles in the subject area:'Protein sequencing'"><div class="flagText">Protein sequencing</div></a>
              <div data-categoryid="47503" data-articleid="25958"
                   data-categoryname="Protein sequencing"
                   class="flagImage" title="Flag 'Protein sequencing' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22RNA+alignment%22" title="Search for articles in the subject area:'RNA alignment'"><div class="flagText">RNA alignment</div></a>
              <div data-categoryid="32671" data-articleid="25958"
                   data-categoryname="RNA alignment"
                   class="flagImage" title="Flag 'RNA alignment' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22RNA+editing%22" title="Search for articles in the subject area:'RNA editing'"><div class="flagText">RNA editing</div></a>
              <div data-categoryid="20807" data-articleid="25958"
                   data-categoryname="RNA editing"
                   class="flagImage" title="Flag 'RNA editing' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=8643'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=7462'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=8650&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>

<div class="block sidebar-comments">
    <div class="header">
        <h3>Comments</h3>
    </div>
      <p><a href="/annotation/listThread.action?root=2095">Academic Editor's comments (Michael Gray)</a><br>Posted by PLoS_ONE_Group</p>
      <p><a href="/annotation/listThread.action?root=7135">Referee comments: Referee 1</a><br>Posted by PLoS_ONE_Group</p>
</div>

</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
