

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Stability of mRNA/DNA and DNA/DNA Duplexes Affects mRNA Transcription</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0000290"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0000290" />

    <meta name="citation_title" content="Stability of mRNA/DNA and DNA/DNA Duplexes Affects mRNA Transcription"/>
    <meta itemprop="name" content="Stability of mRNA/DNA and DNA/DNA Duplexes Affects mRNA Transcription"/>

      <meta name="citation_author" content="Rayna I. Kraeva"/>
            <meta name="citation_author_institution" content="Institute of Molecular Biology, Bulgarian Academy of Sciences, Sofia, Bulgaria"/>
      <meta name="citation_author" content="Dragomir B. Krastev"/>
            <meta name="citation_author_institution" content="Institute of Molecular Biology, Bulgarian Academy of Sciences, Sofia, Bulgaria"/>
      <meta name="citation_author" content="Assen Roguev"/>
            <meta name="citation_author_institution" content="Institute of Molecular Biology, Bulgarian Academy of Sciences, Sofia, Bulgaria"/>
      <meta name="citation_author" content="Anna Ivanova"/>
            <meta name="citation_author_institution" content="Institute of Molecular Biology, Bulgarian Academy of Sciences, Sofia, Bulgaria"/>
      <meta name="citation_author" content="Marina N. Nedelcheva-Veleva"/>
            <meta name="citation_author_institution" content="Institute of Molecular Biology, Bulgarian Academy of Sciences, Sofia, Bulgaria"/>
      <meta name="citation_author" content="Stoyno S. Stoynov"/>
            <meta name="citation_author_institution" content="Institute of Molecular Biology, Bulgarian Academy of Sciences, Sofia, Bulgaria"/>

    <meta name="citation_date" content="2007/3/14"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0000290.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e290"/>
    <meta name="citation_issue" content="3"/>
    <meta name="citation_volume" content="2"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Stability of ribonucleic acid double-stranded helices.; citation_author=PN Borer; citation_author=B Dengler; citation_author=I, Tinoco; citation_author=OC Uhlenbeck; citation_journal_title=J Mol Biol; citation_volume=86; citation_number=1; citation_pages=843-853; citation_date=1974; " />
      <meta name="citation_reference" content="citation_title=Predicting DNA duplex stability from the base sequence.; citation_author=KJ Breslauer; citation_author=R Frank; citation_author=H Blocker; citation_author=LA Marky; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=83; citation_number=2; citation_pages=3746-3750; citation_date=1986; " />
      <meta name="citation_reference" content="citation_title=Thermodynamic parameters to predict stability of RNA/DNA hybrid duplexes.; citation_author=N Sugimoto; citation_author=S Nakano; citation_author=M Katoh; citation_author=A Matsumura; citation_author=H Nakamuta; citation_journal_title=Biochemistry; citation_volume=34; citation_number=3; citation_pages=11211-11216; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=DNA helical stability accounts for mutational defects in a yeast replication origin.; citation_author=DA Natale; citation_author=AE Schubert; citation_author=D Kowalski; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=89; citation_number=4; citation_pages=2654-2658; citation_date=1992; " />
      <meta name="citation_reference" content="citation_title=Ease of DNA unwinding is a conserved property of yeast replication origins.; citation_author=DA Natale; citation_author=RM Umek; citation_author=D Kowalski; citation_journal_title=Nucleic Acids Res; citation_volume=21; citation_number=5; citation_pages=555-560; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=WEB-THERMODYN: Sequence analysis software for profiling DNA helical stability.; citation_author=Y Huang; citation_author=D Kowalski; citation_journal_title=Nucleic Acids Res; citation_volume=31; citation_number=6; citation_pages=3819-3821; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Duplex destabilization in superhelical DNA is predicted to occur at specific transcriptional regulatory regions.; citation_author=CJ Benham; citation_journal_title=J Mol Biol; citation_volume=255; citation_number=7; citation_pages=425-434; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=A high-resolution map of transcription in the yeast genome.; citation_author=L David; citation_author=W Huber; citation_author=M Granovskaia; citation_author=J Toedling; citation_author=CJ Palm; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=103; citation_number=8; citation_pages=5320-5325; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=A comparison of mammalian and yeast pre-mRNA 3′-end processing.; citation_author=W Keller; citation_author=L Minvielle-Sebastia; citation_journal_title=Curr Opin Cell Biol; citation_volume=9; citation_number=9; citation_pages=329-336; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Genomic detection of new yeast pre-mRNA 3′-end-processing signals.; citation_author=JH Graber; citation_author=CR Cantor; citation_author=SC Mohr; citation_author=TF Smith; citation_journal_title=Nucleic Acids Res; citation_volume=27; citation_number=10; citation_pages=888-894; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Probabilistic prediction of Saccharomyces cerevisiae mRNA 3′-processing sites.; citation_author=JH Graber; citation_author=GD McAllister; citation_author=TF Smith; citation_journal_title=Nucleic Acids Res; citation_volume=30; citation_number=11; citation_pages=1851-1858; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Statistical analysis of yeast genomic downstream sequences reveals putative polyadenylation signals.; citation_author=J van Helden; citation_author=M del Olmo; citation_author=JE Perez-Ortin; citation_journal_title=Nucleic Acids Res; citation_volume=28; citation_number=12; citation_pages=1000-1010; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Sequencing and comparison of yeast species to identify genes and regulatory elements.; citation_author=M Kellis; citation_author=N Patterson; citation_author=M Endrizzi; citation_author=B Birren; citation_author=ES Lander; citation_journal_title=Nature; citation_volume=423; citation_number=13; citation_pages=241-254; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Dissecting the regulatory circuitry of a eukaryotic genome.; citation_author=FC Holstege; citation_author=EG Jennings; citation_author=JJ Wyrick; citation_author=TI Lee; citation_author=CJ Hengartner; citation_journal_title=Cell; citation_volume=95; citation_number=14; citation_pages=717-728; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Methods in comparative genomics: genome correspondence, gene identification and regulatory motif discovery.; citation_author=M Kellis; citation_author=N Patterson; citation_author=B Birren; citation_author=B Berger; citation_author=ES Lander; citation_journal_title=J Comput Biol; citation_volume=11; citation_number=15; citation_pages=319-355; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Rules of engagement: co-transcriptional recruitment of pre-mRNA processing factors.; citation_author=DL Bentley; citation_journal_title=Curr Opin Cell Biol; citation_volume=17; citation_number=16; citation_pages=251-256; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Exon tethering in transcription by RNA polymerase II.; citation_author=MJ Dye; citation_author=N Gromak; citation_author=NJ Proudfoot; citation_journal_title=Mol Cell; citation_volume=21; citation_number=17; citation_pages=849-859; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=The RNA-DNA hybrid maintains the register of transcription by preventing backtracking of RNA polymerase.; citation_author=E Nudler; citation_author=A Mustaev; citation_author=E Lukhtanov; citation_author=A Goldfarb; citation_journal_title=Cell; citation_volume=89; citation_number=18; citation_pages=33-41; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Overextended RNA:DNA hybrid as a negative regulator of RNA polymerase II processivity.; citation_author=ML Kireeva; citation_author=N Komissarova; citation_author=M Kashlev; citation_journal_title=J Mol Biol; citation_volume=299; citation_number=19; citation_pages=325-335; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Direct evidence for the effect of transcription on local DNA supercoiling in vivo.; citation_author=AR Rahmouni; citation_author=RD Wells; citation_journal_title=J Mol Biol; citation_volume=223; citation_number=20; citation_pages=131-144; citation_date=1992; " />
      <meta name="citation_reference" content="citation_title=Structural basis of transcription: separation of RNA from DNA by RNA polymerase II.; citation_author=KD Westover; citation_author=DA Bushnell; citation_author=RD Kornberg; citation_journal_title=Science; citation_volume=303; citation_number=21; citation_pages=1014-1016; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Structural basis of transcription: an RNA polymerase II elongation complex at 3.3 A resolution.; citation_author=AL Gnatt; citation_author=P Cramer; citation_author=J Fu; citation_author=DA Bushnell; citation_author=RD Kornberg; citation_journal_title=Science; citation_volume=292; citation_number=22; citation_pages=1876-1882; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=A thermodynamic analysis of RNA transcript elongation and termination in Escherichia coli.; citation_author=TD Yager; citation_author=PH von Hippel; citation_journal_title=Biochemistry; citation_volume=30; citation_number=23; citation_pages=1097-1118; citation_date=1991; " />
      <meta name="citation_reference" content="citation_title=Thinking quantitatively about transcriptional regulation.; citation_author=SJ Greive; citation_author=PH von Hippel; citation_journal_title=Nat Rev Mol Cell Biol; citation_volume=6; citation_number=24; citation_pages=221-232; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Pausing by bacterial RNA polymerase is mediated by mechanistically distinct classes of signals.; citation_author=I Artsimovitch; citation_author=R Landick; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=97; citation_number=25; citation_pages=7090-7095; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Life with 6000 genes.; citation_author=A Goffeau; citation_author=BG Barrell; citation_author=H Bussey; citation_author=RW Davis; citation_author=B Dujon; citation_journal_title=Science; citation_volume=274; citation_number=26; citation_pages=546, 563-547; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Cation effects on the denaturation of DNA.; citation_author=WF Dove; citation_author=N Davidson; citation_journal_title=J Mol Biol; citation_volume=5; citation_number=27; citation_pages=467-478; citation_date=1962; " />
      <meta name="citation_reference" content="citation_title=Dependence of the melting temperature of DNA on salt concentration.; citation_author=C Schildkraut; citation_journal_title=Biopolymers; citation_volume=3; citation_number=28; citation_pages=195-208; citation_date=1965; " />
      <meta name="citation_reference" content="citation_title=Susceptibility to superhelically driven DNA duplex destabilization: a highly conserved property of yeast replication origins.; citation_author=P Ak; citation_author=CJ Benham; citation_journal_title=PLoS Comput Biol; citation_volume=1; citation_number=29; citation_pages=e7; citation_date=2005; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000290" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Stability of mRNA/DNA and DNA/DNA Duplexes Affects mRNA Transcription"/>
    <meta name="twitter:description" content="Nucleic acids, due to their structural and chemical properties, can form double-stranded secondary structures that assist the transfer of genetic information and can modulate gene expression. However, the nucleotide sequence alone is insufficient in explaining phenomena like intron-exon recognition during RNA processing. This raises the question whether nucleic acids are endowed with other attributes that can contribute to their biological functions. In this work, we present a calculation of thermodynamic stability of DNA/DNA and mRNA/DNA duplexes across the genomes of four species in the genus Saccharomyces by nearest-neighbor method. The results show that coding regions are more thermodynamically stable than introns, 3&prime;-untranslated regions and intergenic sequences. Furthermore, open reading frames have more stable sense mRNA/DNA duplexes than the potential antisense duplexes, a property that can aid gene discovery. The lower stability of the DNA/DNA and mRNA/DNA duplexes of 3&prime;-untranslated regions and the higher stability of genes correlates with increased mRNA level. These results suggest that the thermodynamic stability of DNA/DNA and mRNA/DNA duplexes affects mRNA transcription."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0000290.g004"/>

  <meta property="og:title" content="Stability of mRNA/DNA and DNA/DNA Duplexes Affects mRNA Transcription" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000290" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=4777'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=5345'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=8912&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0000290">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000290" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000290&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Stability of mRNA/DNA and DNA/DNA Duplexes Affects mRNA Transcription
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Rayna I. Kraeva, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Institute of Molecular Biology, Bulgarian Academy of Sciences, Sofia, Bulgaria
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Dragomir B. Krastev, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Institute of Molecular Biology, Bulgarian Academy of Sciences, Sofia, Bulgaria
                </p>

                <p>
                    Current address: Max Planck Institute of Molecular Cell Biology and Genetics, Dresden, Germany
                </p>

              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Assen Roguev, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Institute of Molecular Biology, Bulgarian Academy of Sciences, Sofia, Bulgaria
                </p>

                <p>
                    Current address: Department of Cellular and Molecular Pharmacology, University of California, San Francisco, California, United States of America
                </p>

              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Anna Ivanova, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Institute of Molecular Biology, Bulgarian Academy of Sciences, Sofia, Bulgaria
                </p>

                <p>
                    Current address: Experimental Diabetology, Carl Gustav Carus Medical School, Dresden University of Technology, Dresden, Germany
                </p>

              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Marina N. Nedelcheva-Veleva, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Institute of Molecular Biology, Bulgarian Academy of Sciences, Sofia, Bulgaria
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Stoyno S. Stoynov
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:stoynov@obzor.bio21.bas.bg">stoynov@obzor.bio21.bas.bg</a></p>

                <p>Affiliation:
                  Institute of Molecular Biology, Bulgarian Academy of Sciences, Sofia, Bulgaria
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: March 14, 2007</li>
    <li>DOI: 10.1371/journal.pone.0000290</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0000290" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0000290" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000290" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0000290" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0000290-g001" data-doi="info:doi/10.1371/journal.pone.0000290" data-uri="info:doi/10.1371/journal.pone.0000290.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000290.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000290-g002" data-doi="info:doi/10.1371/journal.pone.0000290" data-uri="info:doi/10.1371/journal.pone.0000290.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000290.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000290-t001" data-doi="info:doi/10.1371/journal.pone.0000290" data-uri="info:doi/10.1371/journal.pone.0000290.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000290.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000290-t002" data-doi="info:doi/10.1371/journal.pone.0000290" data-uri="info:doi/10.1371/journal.pone.0000290.t002" title="Table 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000290.t002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000290-t003" data-doi="info:doi/10.1371/journal.pone.0000290" data-uri="info:doi/10.1371/journal.pone.0000290.t003" title="Table 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000290.t003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000290-g003" data-doi="info:doi/10.1371/journal.pone.0000290" data-uri="info:doi/10.1371/journal.pone.0000290.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000290.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000290-t004" data-doi="info:doi/10.1371/journal.pone.0000290" data-uri="info:doi/10.1371/journal.pone.0000290.t004" title="Table 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000290.t004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000290-g004" data-doi="info:doi/10.1371/journal.pone.0000290" data-uri="info:doi/10.1371/journal.pone.0000290.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000290.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000290">Reader Comments (0)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0000290" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1"></a><p>Nucleic acids, due to their structural and chemical properties, can form double-stranded secondary structures that assist the transfer of genetic information and can modulate gene expression. However, the nucleotide sequence alone is insufficient in explaining phenomena like intron-exon recognition during RNA processing. This raises the question whether nucleic acids are endowed with other attributes that can contribute to their biological functions. In this work, we present a calculation of thermodynamic stability of DNA/DNA and mRNA/DNA duplexes across the genomes of four species in the genus <em>Saccharomyces</em> by nearest-neighbor method. The results show that coding regions are more thermodynamically stable than introns, 3′-untranslated regions and intergenic sequences. Furthermore, open reading frames have more stable sense mRNA/DNA duplexes than the potential antisense duplexes, a property that can aid gene discovery. The lower stability of the DNA/DNA and mRNA/DNA duplexes of 3′-untranslated regions and the higher stability of genes correlates with increased mRNA level. These results suggest that the thermodynamic stability of DNA/DNA and mRNA/DNA duplexes affects mRNA transcription.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>Kraeva RI, Krastev DB, Roguev A, Ivanova A, Nedelcheva-Veleva MN, et al.  (2007) Stability of mRNA/DNA and DNA/DNA Duplexes Affects mRNA Transcription. PLoS ONE 2(3):
          e290.
            doi:10.1371/journal.pone.0000290</p><p><strong>Academic Editor: </strong>Stefan Maas, Lehigh University, United States of America</p><p><strong>Received:</strong> January 22, 2007; <strong>Accepted:</strong> February 19, 2007; <strong>Published:</strong> March 14, 2007</p><p><strong>Copyright:</strong> © 2007 Kraeva et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>The research of S.S.S. has been supported by an Alexander von Humboldt Foundation Return Fellowship and NATO Grant EAP.RIG.981642.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>In living systems DNA provides information for the synthesis of RNAs and proteins. The secondary structure of nucleic acids through its defined physico-chemical characteristics such as the thermodynamic stability of the pairing between the two strands can influence its biological function. The thermodynamic stability of a polynucleotide duplex is defined as the free energy (ΔG) required to unwind it and can be calculated from the entropy (ΔS) and the enthalpy (ΔH) of the pairing between the adjacent bases using a nearest-neighbor method <a href="#pone.0000290-Borer1">[1]</a>. Published calorimetric measurement of ΔS and ΔH of all possible nearest-neighbor interactions of DNA/DNA <a href="#pone.0000290-Breslauer1">[2]</a> and RNA/DNA <a href="#pone.0000290-Sugimoto1">[3]</a> duplexes allows for calculation of thermodynamic stability of polynucleotide duplexes with a defined sequence <a href="#pone.0000290-Natale1">[4]</a>–<a href="#pone.0000290-Huang1">[6]</a>. In order to elucidate the influence of thermodynamic stability of DNA/DNA and RNA/DNA duplexes on transcription, a genome-wide analysis of thermodynamic stability is required.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>In this work we present a genome-wide calculation of DNA thermodynamic stability for four genomes in the genus Saccharomyces, using Kowalski's sliding-window approach <a href="#pone.0000290-Huang1">[6]</a>. We show that DNA/DNA as well as DNA/RNA duplex stability differs between coding and non-coding regions. The lower stability of the DNA/DNA and mRNA/DNA duplexes of 3′-untranslated regions and the higher stability of genes correlates with increased mRNA level. Moreover, mRNA/DNA duplexes appear to be more stable than the corresponding anti-sense duplexes, allowing prediction of open reading frames. Based on these observations the role of thermodynamic stability on transcription is discussed.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3><a id="article1.body1.sec2.p1" name="article1.body1.sec2.p1"></a><p>We created Perl-based software that allowed us to calculate thermodynamic stability of DNA/DNA and RNA/DNA duplexes with arbitrary length using a sliding-window approach. This tool allowed us to calculate the thermodynamic profile over the entire genome of <em>Saccharomyces cerevisiae</em> with a step size of 1 bp, and a varying window size (100 bp unless explicitly indicated). Using this set of parameters, the calculated windows' mean value of ΔG of DNA/DNA duplexes for the entire genome is 98.47 kcal/mol. We found that intergenic regions (IGRs) have lower mean values of ΔG average and ΔG minimum (ΔG avg = 92.84 kcal/mol and ΔG min = 78.60 kcal/mol) than genes (ΔG avg = 100.80 kcal/mol and ΔG min = 84.81 kcal/mol) (<a href="#pone-0000290-g001">Figure 1</a> and <a href="#pone.0000290.s002">Tables S1</a>, <a href="#pone.0000290.s003">S2</a> and <a href="#pone.0000290.s004">S3</a>).</p>
<div class="figure" id="pone-0000290-g001"><div class="img"><a name="pone-0000290-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000290.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000290" data-uri="info:doi/10.1371/journal.pone.0000290.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000290.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000290.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000290.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000290.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000290.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000290.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000290.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000290.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000290.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Thermodynamic stability of DNA/DNA (green), sense and antisense RNA/DNA duplexes in a region of chromosome 12 in <em>S. cerevisiae</em> (plots of all sixteen chromosomes are available at <a href="http://obzor.bio21.bas.bg/stoyno/">http://obzor.bio21.bas.bg/stoyno/</a>).</span></strong></p><a id="article1.body1.sec2.fig1.caption1.p1" name="article1.body1.sec2.fig1.caption1.p1"></a><p>ΔG of RNA/DNA duplexes (blue), containing RNA identical to Watson coding strand represents the sense strand for Watson's ORFs and antisense strand for Crick's ORFs. ΔG of RNA/DNA duplexes (red), containing RNA identical to Crick coding strand represents the sense strand for Crick's ORFs and antisense strand for Watson's ORFs.</p>
<span>doi:10.1371/journal.pone.0000290.g001</span></div>
<h4>DNA/DNA and RNA/DNA duplexes are less stable in 3′-IGRs than in genes</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>In order to distinguish the roles of the observed differences in duplex stability in transcription initiation and transcription termination, we grouped the intergenic regions into three groups based on the direction of transcription of their neighboring open reading frames (ORFs): (i) IGRs between ORF starts (divergent transcripts), (ii) IGRs between ORF ends (convergent transcripts) and (iii) IGRs between two ORFs transcribed in the same direction (tandem running transcripts) (<a href="#pone.0000290.s003">Table S2</a>). Our results show that IGRs flanked by convergent transcripts have a lower mean value of ΔG min compared to those flanked by divergent transcripts (<a href="#pone-0000290-t001">Tables 1</a> and <a href="#pone.0000290.s004">S3</a>). These findings are in agreement with previous studies, showing that 3′-termini of several transcription units contain regions prone to unwinding under superhelical stress conditions <a href="#pone.0000290-Benham1">[7]</a>. To check whether all intergenic sequences are less stable than their adjacent 5′-ORFs, we compared the calculated ΔG values for these two classes of sequences. The results show that out of 6004 ORF/3′-IGR pairs in the <em>S. cerevisiae</em> genome 93% of ORFs have a higher ΔG avg and 86% have a higher ΔG min than their corresponding 3′-IGRs (<a href="#pone-0000290-g002">Figure 2A</a> and <a href="#pone-0000290-t002">Tables 2</a> and <a href="#pone.0000290.s005">S4</a>). To further explore this, we calculated mRNA/DNA duplex stability in the genome of <em>S. cerevisiae</em> (see <a href="#s4">Materials and Methods</a>). As expected, mRNA/DNA duplexes are less stable (that is with lower ΔG) than DNA/DNA duplexes for both ORFs and 3′-IGRs (<a href="#pone-0000290-t002">Table 2</a>). Similar to DNA/DNA duplexes, mRNA/DNA duplexes of 3′-IGRs have a statistically significant lower mean value of ΔG avg than the corresponding genes (<a href="#pone.0000290.s004">Table S3</a>). 92% of the ORFs have a higher ΔG avg and 81% have a higher ΔG min than the IGR adjacent to their 3′-ends (<a href="#pone.0000290.s005">Table S4</a>). Using the available information on the position of the 3′-end processing sites in <em>S. cerevisiae</em> <a href="#pone.0000290-David1">[8]</a>, we investigated the thermodynamic stability of mRNA/DNA duplexes of the 3′-untraslated regions (3′-UTRs) (window size = 9 bp; see <a href="#s4">Materials and Methods</a>). 3′-UTRs have statistically significant lower mean value of ΔG than genes (<a href="#pone-0000290-t003">Tables 3</a>, <a href="#pone.0000290.s002">S1</a> and <a href="#pone.0000290.s004">S3</a>).</p>
<div class="figure" id="pone-0000290-g002"><div class="img"><a name="pone-0000290-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000290.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000290" data-uri="info:doi/10.1371/journal.pone.0000290.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000290.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000290.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000290.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000290.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000290.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000290.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000290.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000290.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000290.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>(A) Percentage of ORFs with ΔG values of DNA/DNA and sense mRNA/DNA duplexes higher than ΔG avg and ΔG min of the corresponding 3′-IGRs.</span></strong></p><a id="article1.body1.sec2.sec1.fig1.caption1.p1" name="article1.body1.sec2.sec1.fig1.caption1.p1"></a><p>(B) Percentage of ORFs with more stable sense than antisense RNA/DNA duplexes as annotated in SGD.</p>
<span>doi:10.1371/journal.pone.0000290.g002</span></div><div class="figure" id="pone-0000290-t001"><div class="img"><a name="pone-0000290-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000290.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000290" data-uri="info:doi/10.1371/journal.pone.0000290.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000290.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000290.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000290.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000290.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000290.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000290.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000290.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000290.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000290.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>Mean values and standard deviation (in brackets) of ΔG min and ΔG avg of intergenic regions flanked by convergent (→ ←), divergent (← →) and tandem (→ →) running transcripts.</span></strong></p><span>doi:10.1371/journal.pone.0000290.t001</span></div><div class="figure" id="pone-0000290-t002"><div class="img"><a name="pone-0000290-t002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000290.t002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000290" data-uri="info:doi/10.1371/journal.pone.0000290.t002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000290.t002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000290.t002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000290.t002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000290.t002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000290.t002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000290.t002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000290.t002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000290.t002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000290.t002.TIF"></span>)
                </a></li></ul></div><p><strong>Table 2.  <span>Mean values and standard deviation (in brackets) of ΔG min and ΔG avg of genes and their 3′-intergenic regions.</span></strong></p><span>doi:10.1371/journal.pone.0000290.t002</span></div><div class="figure" id="pone-0000290-t003"><div class="img"><a name="pone-0000290-t003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000290.t003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000290" data-uri="info:doi/10.1371/journal.pone.0000290.t003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000290.t003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000290.t003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000290.t003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000290.t003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000290.t003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000290.t003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000290.t003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000290.t003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000290.t003.TIF"></span>)
                </a></li></ul></div><p><strong>Table 3.  <span>Mean values and standard deviation of ΔG avg of sense and antisense RNA/DNA duplexes of genes, introns, exons and 3′-UTRs (window size 9 bp.)</span></strong></p><span>doi:10.1371/journal.pone.0000290.t003</span></div>

<h4>DNA/DNA and RNA/DNA duplexes are more unstable in introns and 3′-end processing regions than in coding sequences</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>3′–end processing requires several quite degenerate regulatory sequences positioned in the range of 80 nt upstream and 20 nt downstream from the 3′-end processing site <a href="#pone.0000290-Keller1">[9]</a>–<a href="#pone.0000290-vanHelden1">[12]</a>. Therefore, we examined the thermodynamic stability of mRNA/DNA duplexes of these 100 bp 3′-end processing regions (3′-EPRs). Our results showed that the mean value of ΔG of the 3′-regulatory sequences (32.41 kcal/mol) is comparable to the mean value of ΔG avg of the 3′-IGRs and is significantly lower than ΔG avg of the genes (<a href="#pone-0000290-t002">Tables 2</a> and <a href="#pone.0000290.s004">S3</a>). <em>S. cerevisiae</em> genome contains 264 genes with introns. Calculation of introns' thermodynamic profiles (window size of 9 bp) showed that their mRNA/DNA duplexes are significantly less stable than exon's (coding sequences in ORFs) sense mRNA/DNA duplexes (<a href="#pone-0000290-t003">Tables 3</a> and <a href="#pone.0000290.s002">S1</a>). These results suggest that stable sense duplexes are characteristic of the coding sequences.</p>


<h4>Evolutionary conservation of the thermodynamic pattern</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>To check if the observed pattern of thermodynamic stability is evolutionarily conserved we calculated the ΔG values of DNA/DNA and mRNA/DNA duplexes for three other related species of the genus <em>Saccharomyces</em>-<em>S. bayanus, S. paradoxus, S. mikatae</em>, using the available draft genome sequences (<a href="#pone.0000290.s002">Tables S1</a> and <a href="#pone.0000290.s005">S4</a>) <a href="#pone.0000290-Kellis1">[13]</a>. The averages of ΔG of DNA/DNA and mRNA/DNA duplexes in genes are greater than those in the adjacent 3′-IGR in more than 92% and 93% of the cases, respectively (<a href="#pone-0000290-g002">Figure 2A</a> and <a href="#pone-0000290-t002">Table 2</a>). The minimums of ΔG of DNA/DNA and mRNA/DNA duplexes in genes are greater than those in the adjacent 3′-IGR in more than 86% and 82% of the cases, respectively (<a href="#pone-0000290-g002">Figure 2A</a> and <a href="#pone.0000290.s005">Table S4</a>).</p>


<h4>Correlation between thermodynamic stability of DNA/DNA and RNA/DNA duplexes and mRNA level</h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>We also inspected the possible relationship between mRNA expression level <a href="#pone.0000290-Holstege1">[14]</a> and values of ΔG in genes and their corresponding 100 bp 3′-EPRs. There appears to be a general trend of increased mRNA level with increasing ΔG avg of the ORFs. Spearman's rank correlation coefficients (SCC), assessing the strength of the association between gene's thermodynamic stability and mRNA levels, are 0.209 for DNA/DNA duplexes and 0.142 for mRNA/DNA duplexes (<a href="#pone.0000290.s006">Table S5</a>). Although these values are not particularly high, they bear a strong statistical significance (<a href="#pone.0000290.s006">Table S5</a>). The observed correlations are impressive given that several other factors (like promoter effectiveness, promoter regulation and mRNA half-life) directly influence mRNA level as well. Correlation between stability of coding sequences only and mRNA level is higher: SCC is 0.263 for DNA/DNA duplexes and 0.199 for mRNA/DNA duplexes.</p>
<a id="article1.body1.sec2.sec4.p2" name="article1.body1.sec2.sec4.p2"></a><p>We next surveyed the relationship between mRNA level and stability of intron-containing genes. In this case we did not find a statistically significant correlation. However, a strong correlation between mRNA level and the stability of the exons was observed: SCC is 0.374 for DNA/DNA duplexes and 0.329 for mRNA/DNA duplexes (<a href="#pone-0000290-g003">Figure 3A</a> and <a href="#pone.0000290.s006">Table S5</a>). The correlation between mRNA level and exon thermodynamic stability increases with increasing ORF length: SCC for intron containing ORFs longer than 2000 bp is 0.658 for DNA/DNA duplexes and 0.691 for mRNA/DNA duplexes (<a href="#pone-0000290-g003">Figure 3B</a> and <a href="#pone.0000290.s006">Table S5</a>). Interestingly, a positive correlation exists between the thermodynamic stability of introns and mRNA level. This correlation increases with increasing ORF length: SCC is 0.611 for DNA/DNA duplexes and 0.560 for mRNA/DNA duplexes. In addition, an inverse relationship exists between mRNA levels and stability of 3′-EPRs. mRNA levels of the ORFs 5′ of the EPR increase with decreasing of 3′-EPR ΔG (<a href="#pone-0000290-g003">Figure 3C</a>) (SCCs are -0.266 for DNA/DNA duplexes and -0.232 for mRNA/DNA duplexes) and this negative correlation rapidly increases with decreasing ORF length. For ORFs shorter than 250 bp SCC is -0.639 for mRNA/DNA duplexes (<a href="#pone.0000290.s006">Table S5</a> and <a href="#pone-0000290-g003">Figure 3D</a>), indicating strong negative relationship between thermodynamic stability of the 3′-EPR and mRNA level. Similar negative correlation is observed between 3′-UTR's stability and mRNA level (<a href="#pone.0000290.s006">Table S5</a>). The correlations between mRNA level and either ORF's or 3′-EPR's stability suggest a role for the thermodynamic stability in mRNA transcription.</p>
<div class="figure" id="pone-0000290-g003"><div class="img"><a name="pone-0000290-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000290.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000290" data-uri="info:doi/10.1371/journal.pone.0000290.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000290.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000290.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000290.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000290.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000290.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000290.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000290.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000290.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000290.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3. </strong></p><a id="article1.body1.sec2.sec4.fig1.caption1.p1" name="article1.body1.sec2.sec4.fig1.caption1.p1"></a><p>Scatter plot, showing the relationship of mRNA level (copies per cell) and Δ G (kcal/mol) of EPR mRNA/DNA duplexes and Δ G avg of exon mRNA/DNA duplexes. (A) Relationship between mRNA level and Δ G avg of all coding sequences in intron containing ORFs. (B) Relationship between mRNA level and Δ G avg of coding sequences in intron containing ORFs longer than 2000 bp. (C) Relationship between mRNA level and Δ G for all available EPRs. (D) Relationship between mRNA level and ΔG of EPRs for genes shorter than 250 bp.</p>
<span>doi:10.1371/journal.pone.0000290.g003</span></div>

<h4>More stable sense than antisense RNA/DNA duplexes are a common characteristic of the coding sequences</h4>
<a id="article1.body1.sec2.sec5.p1" name="article1.body1.sec2.sec5.p1"></a><p>Upon careful scrutiny, the thermodynamic profiles of mRNA/DNA duplexes within genes exhibits yet another interesting feature. There is a strong statistically significant difference between ΔG avg of sense and potential antisense RNA/DNA duplexes in ORFs (<a href="#pone-0000290-t004">Tables 4</a>, <a href="#pone.0000290.s002">S1</a> and <a href="#pone.0000290.s004">S3</a>). 76.90% of all ORFs have more stable sense mRNA/DNA duplexes than potential antisense RNA/DNA duplexes (<a href="#pone-0000290-g002">Figure 2B</a>). However, the thermodynamic stability of antisense RNA/DNA duplexes positively correlates with mRNA level. Unlike ORFs, the ratio of ΔG avg of potential sense and antisense RNA/DNA duplexes in 3′-IGRs is nearly equal (50.57% of the sense duplexes are more stable than the potential antisense duplexes) (<a href="#pone-0000290-t004">Tables 4</a> and <a href="#pone.0000290.s005">S4</a>).</p>
<div class="figure" id="pone-0000290-t004"><div class="img"><a name="pone-0000290-t004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000290.t004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000290" data-uri="info:doi/10.1371/journal.pone.0000290.t004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000290.t004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000290.t004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000290.t004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000290.t004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000290.t004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000290.t004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000290.t004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000290.t004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000290.t004.TIF"></span>)
                </a></li></ul></div><p><strong>Table 4.  <span>Mean values and standard deviation (in brackets) of ΔG avg of sense and antisense RNA/DNA duplexes of genes and 3′-IGRs and their dependence on ORF length.</span></strong></p><span>doi:10.1371/journal.pone.0000290.t004</span></div><a id="article1.body1.sec2.sec5.p2" name="article1.body1.sec2.sec5.p2"></a><p>ORFs in the Saccharomyces Genome Database fall into one of the following three categories-verified (experimentally confirmed); uncharacterized (which have orthologs in other species, but without experimental evidence in yeasts to support this); and dubious (without any experimental evidence for their existence). Although dubious ORFs are unlikely to encode a protein, there are no characteristic features to distinguish them from the verified and uncharacterized (henceforth called validated) ORFs. However, our analysis shows that 84.2% of the validated ORFs and only 25% of the dubious ORFs have more stable sense than antisense RNA/DNA duplexes (<a href="#pone-0000290-g002">Figure 2B</a>). This ratio depends on ORF length and is 90.35% for ORFs longer than 2000 bp and only 45.29% for ORFs shorter than 250 bp (<a href="#pone-0000290-t004">Table 4</a>). These data suggest a way to distinguish true from spurious ORFs based solely on their thermodynamic stability profiles. To test this proposition, we extended our analysis to all potential ORFs found in the other three <em>Saccharomyces species</em> (<em>S. bayanus, S. paradoxus</em> and <em>S. mikatae</em>). We took advantage of the fact that ORFs in these genomes that have orthologs in <em>S. cerevisiae</em> were identified by comparative genomic analysis, assuming these ORFs to be true <a href="#pone.0000290-Holstege1">[14]</a>, <a href="#pone.0000290-Kellis2">[15]</a>. We found that more than 81% of the true ORFs and only 28.5% of the spurious genes have more stable sense than antisense RNA/DNA duplexes. Therefore, false positives and negatives under our thermodynamic approach are 19% and 28.5%, respectively. In addition, the length dependence of sense/antisense duplex stability in these three species is reminiscent of the one observed in <em>S. cerevisiae</em>-more than 90% of the true ORFs longer than 2000 bp and less than 61% of the true ORFs shorter than 250 bp have more stable sense than antisense RNA/DNA duplex. These results further strengthen the idea that thermodynamic stability is able to discriminate to a certain extent between true and spurious ORFs.</p>
<a id="article1.body1.sec2.sec5.p3" name="article1.body1.sec2.sec5.p3"></a><p>The genome of <em>S. cerevisiae</em> contains 1204 annotated overlapping ORFs grouped in 634 overlapping pairs (<a href="#pone.0000290.s007">Table S6</a>). 91% of the groups consist of both verified and dubious ORFs and less than 5% of these groups contain only validated ORFs suggesting that <em>S. cerevisiae</em> does not tolerate overlapping mRNA transcription. To examine whether the stability of mRNA/DNA duplexes influences the choice of ORF to be transcribed, we compared the stability profiles of the groups containing both dubious and validated ORFs. In 81.5% of the cases, validated ORFs have more stable sense mRNA/DNA duplex than the dubious ORFs, determining to an extent which of the ORFs is to be transcribed.</p>
<a id="article1.body1.sec2.sec5.p4" name="article1.body1.sec2.sec5.p4"></a><p>Furthermore, we looked into the thermodynamic profiles of genes containing introns. Our results show that in contrast to exons, introns have less stable sense RNA/DNA duplex than the respective antisense RNA/DNA duplex (<a href="#pone.0000290.s004">Table S3</a>). Therefore, more stable sense than potential antisense RNA/DNA duplexes are characteristic of the coding sequences.</p>


<h4>Differential distribution of certain nucleotide neighbor interactions in sense and antisense RNA/DNA duplexes is responsible for the higher thermodynamic stability of sense RNA/DNA duplexes of coding sequences</h4>
<a id="article1.body1.sec2.sec6.p1" name="article1.body1.sec2.sec6.p1"></a><p>To explain the observed differences in the stability of sense and potential antisense RNA/DNA duplexes in coding sequences and introns, we calculated the frequency of their nearest neighbor interactions. RNA/DNA nearest neighbor interactions form pairs, containing complementary DNA duplets (<a href="#pone-0000290-g004">Figure 4</a>). Differences in ΔG values of for interactions within these pairs are responsible for the difference in stability of sense/antisense duplexes. We found that genes' sense mRNA/DNA duplexes contain more rAA/dTT, rAC/dTG, rAG/dTC, rGG/dCC, rGA/dCT, rCA/dGT interactions than their corresponding partners rUU/dAA, rGU/dCA, rCU/dGA, rCC/dGG, rUC/dAG, rUG/dAC found more frequently in the potential antisense RNA/DNA duplexes (<a href="#pone.0000290.s008">Table S7</a>). The higher stability of the first five sense interactions (rAA/dTT, rAC/dTG, rAG/dTC, rGG/dCC, rUC/dAG) compared to the corresponding antisense partners (rUU/dAA, rGU/dCA, rCU/dGA, rCC/dGG, rGA/dCT) leads to a more stable sense RNA/DNA duplex. rUG/dAC is more stable and well-represented in antisense duplexes than rCA/dGT and hence it contributes to the stability of the antisense duplex. Finally, rAU/dTA and rUA/dAT, as well as rGC/dCG and rCG/dGC, are symmetric and therefore equally distributed in both sense and antisense duplexes and contribute equally to their stability. Yet, the impact of the first five duplex pairs on the stability of the sense duplex is much stronger and consequently sense duplexes are more stable than antisense duplexes. In introns and IGRs, however, the above frequencies are different (<a href="#pone.0000290.s009">Tables S8</a> and <a href="#pone.0000290.s010">S9</a>). For example, in contrast to coding sequences, the more stable rAA/dTT pair is under-represented in introns compared to its corresponding but less stable rUU/dAA pair. These two pairs occur with nearly equal frequency in IGRs. This suggests that the different distribution of certain nearest neighbor interactions contributes to the higher stability of coding sequences and lower stability of introns and IGRs.</p>
<div class="figure" id="pone-0000290-g004"><div class="img"><a name="pone-0000290-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000290.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000290" data-uri="info:doi/10.1371/journal.pone.0000290.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000290.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000290.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000290.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000290.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000290.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000290.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000290.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000290.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000290.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4. </strong></p><a id="article1.body1.sec2.sec6.fig1.caption1.p1" name="article1.body1.sec2.sec6.fig1.caption1.p1"></a><p>Thermodynamic stability (ΔG) of the nearest-neighbor interactions in RNA/DNA duplexes (10 mM monovalent cation), containing complementary DNA strands (in blue). Watson strand (top) and Crick strand (bottom) shown in black.</p>
<span>doi:10.1371/journal.pone.0000290.g004</span></div>
</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p>It still remains unclear how mRNA/DNA duplexes stability influences mRNA level. The co-transcriptional nature of 3′-end processing provides an elegant possible explanation <a href="#pone.0000290-Bentley1">[16]</a>. The 3′-end processing machinery, traveling along RNA polymerase II recognizes the 3′-end processing sites within the nascent mRNA and catalyzes endonucleolytic cleavage and addition of poly(A) tail. An important factor here is the rate and extent of mRNA/DNA duplex unwinding immediately after mRNA synthesis. Slower and inefficient unwinding of mRNA/DNA duplex in the 3′-end processing region will hinder its recognition by the 3′-end processing machinery. Therefore, in regions of higher stability where RNA/DNA duplexes are more difficult to unwind and less accessible to the processing apparatus RNA processing will be impaired. A similar mechanism could act during splicing. Introns are known to harbor common (even though very degenerate) RNA consensus sequences near their 3′ and 5′-ends that are recognized and cleaved by spliceosomal components to remove introns and ligate flanking exon sequences. Again, a critical step is the recognition of these elements by the spliceosome traveling with the RNA polymerase II <a href="#pone.0000290-Bentley1">[16]</a>, <a href="#pone.0000290-Dye1">[17]</a>. Hence, the lower thermodynamic stability of mRNA/DNA duplex within introns will make consensus splicing sequences more accessible and easier to recognize, thus improving splicing efficiency. If this model is correct, the higher thermodynamic stability of mRNA/DNA duplexes in the genes' coding sequences would preserve mRNA from premature termination and improper splicing.</p>
<a id="article1.body1.sec3.p2" name="article1.body1.sec3.p2"></a><p>The above model is challenged in the light of the fact that the length of mRNA/DNA duplex during transcription is considered to be only 7–9 bp and is located within the polymerase enzyme <a href="#pone.0000290-Nudler1">[18]</a>. However, these estimates are derived from biochemical assays of stalled transcription complexes <a href="#pone.0000290-Nudler1">[18]</a>. Static transcriptional machinery gives enough time for re-association of the DNA/DNA helix outside the polymerase. Such re-association can restrict the length of the mRNA/DNA duplex to be maintained by the RNA pol II. Supporting this idea are experiments showing that mRNA/DNA duplex is not unwound by RNA polymerase when the non-template DNA strand is missing <a href="#pone.0000290-Kireeva1">[19]</a>. Addition of non-template DNA strand restricts the mRNA/DNA duplex to 9 nucleotides <a href="#pone.0000290-Kireeva1">[19]</a>. However, the length of the mRNA/DNA duplex would be different in case of dynamic RNA polymerase and would strongly depend on RNA/DNA, DNA/DNA stability and the rate of RNA polymerase movement. More stable mRNA/DNA duplexes would persist longer outside the polymerase. In addition, during transcription, negative superstress is generated behind the Pol II enzyme <a href="#pone.0000290-Rahmouni1">[20]</a> which should temporarily impede the re-association of the two DNA strands and would thus slow down mRNA/DNA duplex unwinding. The influence of RNA/DNA stability on RNA/DNA duplex length could give a reasonable explanation of the differences between the two atomic structures of the RNA polymerase complex containing RNA/DNA duplex. In one of the studies, the RNA/DNA duplex is unwound at the RNA's 5′-end <a href="#pone.0000290-Westover1">[21]</a> while in the other it is not <a href="#pone.0000290-Gnatt1">[22]</a>. In the first experiment, the last three nucleotides at the 5′-RNA end are AUG, forming two of the less stable nearest neighbor interactions rAU/dTA (0.03 kcal/mol) and rUG/dAC (0.64 kcal/mol) which allow RNA unwinding by two protein loops (named lid and rudder) of Pol II. In the second experiment, the 5′-end of the RNA strand contains three G residues that participate in two rGG/dCC nearest neighbor interactions. These residues form the second most thermodynamically stable RNA/DNA duplex structure (1.94 kcal/mol) which would prevent the lid and the rudder from unwinding RNA.</p>
<a id="article1.body1.sec3.p3" name="article1.body1.sec3.p3"></a><p>In addition, DNA/DNA and mRNA/DNA duplex stability could affect mRNA level by influencing the kinetics of transcription. It has been suggested that the free energy required to open the DNA transcription bubble and to form the mRNA/DNA hybrid directly influences the rate of transcription elongation <a href="#pone.0000290-Yager1">[23]</a>, <a href="#pone.0000290-Greive1">[24]</a>. It has been shown that transcription machinery tends to pause when the mRNA/DNA hybrid is unstable <a href="#pone.0000290-Artsimovitch1">[25]</a>. Pausing or rate reduction at unstable mRNA/DNA duplexes of 3′-UTRs and introns could give enough time to the processing complexes to interact with their corresponding mRNA elements and process the nascent mRNA transcript. Likewise, the higher stability of mRNA/DNA duplexes of the coding sequences could increase the rate of the transcription elongation and raise mRNA level.</p>
<a id="article1.body1.sec3.p4" name="article1.body1.sec3.p4"></a><p>In this work we have shown that DNA/DNA as well as RNA/DNA duplex stability differ between coding and non-coding regions. Moreover, sense RNA/DNA duplexes appear to be more stable than the corresponding anti-sense duplexes, an observation potentially useful for gene discovery. The lower stability of the DNA/DNA and mRNA/DNA duplexes of 3′-untranslated regions and higher stability of the coding sequences correlate with increased mRNA level. Our results suggest that the thermodynamic stability of DNA/DNA and mRNA/DNA duplexes affects mRNA transcription but further work will be required to more fully understand how thermodynamic stability modulates mRNA level.</p>
</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4>Genomes and annotations</h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p>The complete genome sequence of <em>S. cerevisiae</em> (SGD release 07.2005) strain S288C <a href="#pone.0000290-Goffeau1">[26]</a> and the draft genomes of <em>S. bayanus, S. mikatae</em> and <em>S. paradoxus</em> <a href="#pone.0000290-Kellis1">[13]</a> were used in the calculations. 3′-IGR, which do not overlap with coding sequences, of all four <em>Saccharomyces</em> species, were analyzed. In <em>S. bayanus, S. mikatae</em> and <em>S. paradoxus</em> we used the full-length ORFs only. For these three <em>Saccharomyces</em> species only the 3′-IGRs surrounded by full-length ORFs, with orthologs in <em>S. cerevisiae'</em>s, and belonging to a common contig were included in the analysis.</p>


<h4>Measurement of thermodynamic stability</h4>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1"></a><p>ΔG of the nearest-neighbor interactions was calculated by Perl-based software (supplementary <a href="#pone.0000290.s001">Data S1</a>) using Kowalski's sliding-window approach <a href="#pone.0000290-Huang1">[6]</a>. Published values of ΔH and ΔS for each nearest-neighbor interaction for DNA/DNA duplex <a href="#pone.0000290-Breslauer1">[2]</a> and RNA/DNA duplex <a href="#pone.0000290-Sugimoto1">[3]</a> were used. Our analysis does not consider the possible self-folding of the single stranded DNA and RNA as in living systems the processes of DNA unwinding and RNA synthesis are independent of RNA and DNA self-folding. During transcription, DNA unwinding is clearly separated from the self-folding of the single stranded DNA and is carried out by the helicase activity of RNA polymerase II holoenzyme in 5′-3′ orientation one nucleotide at a time <a href="#pone.0000290-Greive1">[24]</a>. Therefore, to allow self-folding of a palindromic sequence of six nucleotides, six independent DNA unwinding reactions are required. After that, RNA polymerase II adds ribonucleotides one by one and creates an RNA/DNA duplex. Therefore, measurements of RNA/DNA duplex stability do not require the consideration of RNA or DNA self-folding as RNA is synthesized not by annealing of oligonucleotides (that could self-fold) but by sequential addition of ribonucleotides to the nascent transcript.</p>
<a id="article1.body1.sec4.sec2.p2" name="article1.body1.sec4.sec2.p2"></a><p>Calculations are carried out for 37°C, with a step size of 1 bp and a window size of 100 bp, 9 bp or 2 bp. The calculated values for different window sizes are indicated at the 51<sup>st</sup> bp for 100 bp windows, at the 5<sup>th</sup> bp for 9 bp windows, and at the 2<sup>nd</sup> bp for 2 bp windows. A 2-bp window represents a single nearest-neighbor interaction. Window size of 9 bp allows calculation of ΔG for sequences equal in size to the length of the RNA/DNA duplex maintained by RNA polymerase II during transcription elongation <a href="#pone.0000290-Nudler1">[18]</a>. Window size of 100 bp enables calculation of ΔG average of the windows that extend over large genomic regions. Our results show that there is no significant difference in the ratio of ΔG avg of genes and intergenic regions when calculations were carried out using different window sizes (Table <a href="#pone.0000290.s002">S1</a> and <a href="#pone.0000290.s005">S4</a>). In addition, there is no significant difference in both the ratio of ΔG average of sense/antisense RNA/DNA duplexes and the correlation between ΔG and mRNA level, using different window sizes. Therefore, we generally used a window size of 100 bp, except for introns and UTRs (window size of 9 bp used instead) as they tend to be relatively short.</p>
<a id="article1.body1.sec4.sec2.p3" name="article1.body1.sec4.sec2.p3"></a><p>ΔG was calculated for three different salt concentrations (10mM, 100mM and 1M) <a href="#pone.0000290-Huang1">[6]</a>, <a href="#pone.0000290-Dove1">[27]</a>, <a href="#pone.0000290-Schildkraut1">[28]</a>. No significant differences in both the ratio of ΔG avg of genes and intergenic regions and ΔG avg of sense/antisense RNA/DNA duplexes were observed (Table <a href="#pone.0000290.s002">S1</a> and <a href="#pone.0000290.s005">S4</a>). The results presented in this work assume monovalent cation concentration of 10mM as this is the value used in previous studies on thermodynamic stability of DNA/DNA duplexes <a href="#pone.0000290-Huang1">[6]</a>, <a href="#pone.0000290-Ak1">[29]</a>.</p>
<a id="article1.body1.sec4.sec2.p4" name="article1.body1.sec4.sec2.p4"></a><p>Stability of RNA/DNA duplexes of both DNA strands was calculated over the entire genomes. Thermodynamic stability of sense RNA/DNA duplexes for genes was calculated using duplexes containing gene's template DNA strand and stability of antisense RNA/DNA duplexes was calculated using duplexes containing gene's coding DNA strand.</p>


<h4>Statistics</h4>
<a id="article1.body1.sec4.sec3.p1" name="article1.body1.sec4.sec3.p1"></a><p>Spearman's rank correlation test was used to assess the relationship between either DNA/DNA or mRNA/DNA duplex stability and mRNA level. Variation of Spearman's correlation coefficient from 0 to 1 indicates that the two variables increase together and from 0 to-1 indicates negative relationship. Wilcoxon–Mann–Whitney rank sum test was used to statistically evaluate the difference between genes' and IGRs' ΔG avg and ΔG min in DNA/DNA and mRNA/DNA duplexes and evaluate the difference between genes' ΔG avg in sense and antisense RNA/DNA duplexes.</p>


<h4>Supporting web site</h4>
<a id="article1.body1.sec4.sec4.p1" name="article1.body1.sec4.sec4.p1"></a><p>Supporting web site (<a href="http://obzor.bio21.bas.bg/stoyno/">http://obzor.bio21.bas.bg/stoyno/</a>) contains: (i) all raw thermodynamic stability data, (ii) the software and databases used for ΔG calculation and (iii) plots, presenting DNA/DNA and RNA/DNA duplex stability of all sixteen chromosomes of <em>S. cerevisiae</em>.</p>

</div>

<div id="section5" class="section"><a id="s5" name="s5" toc="s5" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0000290"></div><a name="pone.0000290.s001" id="pone.0000290.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000290.s001">Data S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1"></a><p class="preSiDOI">Method of thermodynamic stability measurement</p>
<p class="siDoi">doi:10.1371/journal.pone.0000290.s001</p><a id="article1.body1.sec5.supplementary-material1.caption1.p2" name="article1.body1.sec5.supplementary-material1.caption1.p2"></a><p class="postSiDOI">(0.03 MB DOC)</p>
<a name="pone.0000290.s002" id="pone.0000290.s002"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000290.s002">Table S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material2.caption1.p1" name="article1.body1.sec5.supplementary-material2.caption1.p1"></a><p class="preSiDOI">Delta G values in DNA/DNA and RNA/DNA duplexes of genes, introns, exons, UTRs and EPRs</p>
<p class="siDoi">doi:10.1371/journal.pone.0000290.s002</p><a id="article1.body1.sec5.supplementary-material2.caption1.p2" name="article1.body1.sec5.supplementary-material2.caption1.p2"></a><p class="postSiDOI">(4.66 MB ZIP)</p>
<a name="pone.0000290.s003" id="pone.0000290.s003"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000290.s003">Table S2. </a></strong></p><a id="article1.body1.sec5.supplementary-material3.caption1.p1" name="article1.body1.sec5.supplementary-material3.caption1.p1"></a><p class="preSiDOI">Free energy minimums in DNA/DNA duplexes of intergenic regions flanked by convergent, divergent and tandem running transcripts.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000290.s003</p><a id="article1.body1.sec5.supplementary-material3.caption1.p2" name="article1.body1.sec5.supplementary-material3.caption1.p2"></a><p class="postSiDOI">(0.31 MB ZIP)</p>
<a name="pone.0000290.s004" id="pone.0000290.s004"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000290.s004">Table S3. </a></strong></p><a id="article1.body1.sec5.supplementary-material4.caption1.p1" name="article1.body1.sec5.supplementary-material4.caption1.p1"></a><p class="preSiDOI">Estimation of statistically significant difference</p>
<p class="siDoi">doi:10.1371/journal.pone.0000290.s004</p><a id="article1.body1.sec5.supplementary-material4.caption1.p2" name="article1.body1.sec5.supplementary-material4.caption1.p2"></a><p class="postSiDOI">(0.04 MB DOC)</p>
<a name="pone.0000290.s005" id="pone.0000290.s005"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000290.s005">Table S4. </a></strong></p><a id="article1.body1.sec5.supplementary-material5.caption1.p1" name="article1.body1.sec5.supplementary-material5.caption1.p1"></a><p class="preSiDOI">Comparison between values of delta G average and delta G minimum of the genes and intergenic regions adjacent to their 3′ ends</p>
<p class="siDoi">doi:10.1371/journal.pone.0000290.s005</p><a id="article1.body1.sec5.supplementary-material5.caption1.p2" name="article1.body1.sec5.supplementary-material5.caption1.p2"></a><p class="postSiDOI">(3.19 MB ZIP)</p>
<a name="pone.0000290.s006" id="pone.0000290.s006"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000290.s006">Table S5. </a></strong></p><a id="article1.body1.sec5.supplementary-material6.caption1.p1" name="article1.body1.sec5.supplementary-material6.caption1.p1"></a><p class="preSiDOI">Correlation between mRNA level and thermodynamic stability of DNA/DNA and RNA/DNA duplexes</p>
<p class="siDoi">doi:10.1371/journal.pone.0000290.s006</p><a id="article1.body1.sec5.supplementary-material6.caption1.p2" name="article1.body1.sec5.supplementary-material6.caption1.p2"></a><p class="postSiDOI">(0.08 MB DOC)</p>
<a name="pone.0000290.s007" id="pone.0000290.s007"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000290.s007">Table S6. </a></strong></p><a id="article1.body1.sec5.supplementary-material7.caption1.p1" name="article1.body1.sec5.supplementary-material7.caption1.p1"></a><p class="preSiDOI">Comparison between delta G average of the overlapping ORF couples in sense and antisense RNA/DNA dupexes</p>
<p class="siDoi">doi:10.1371/journal.pone.0000290.s007</p><a id="article1.body1.sec5.supplementary-material7.caption1.p2" name="article1.body1.sec5.supplementary-material7.caption1.p2"></a><p class="postSiDOI">(0.07 MB ZIP)</p>
<a name="pone.0000290.s008" id="pone.0000290.s008"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000290.s008">Table S7. </a></strong></p><a id="article1.body1.sec5.supplementary-material8.caption1.p1" name="article1.body1.sec5.supplementary-material8.caption1.p1"></a><p class="preSiDOI">Distribution of the nearest-neighbor interactions in sense and antisense RNA/DNA duplexes in genes</p>
<p class="siDoi">doi:10.1371/journal.pone.0000290.s008</p><a id="article1.body1.sec5.supplementary-material8.caption1.p2" name="article1.body1.sec5.supplementary-material8.caption1.p2"></a><p class="postSiDOI">(0.04 MB DOC)</p>
<a name="pone.0000290.s009" id="pone.0000290.s009"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000290.s009">Table S8. </a></strong></p><a id="article1.body1.sec5.supplementary-material9.caption1.p1" name="article1.body1.sec5.supplementary-material9.caption1.p1"></a><p class="preSiDOI">Distribution of the nearest-neighbor interactions in sense and antisense RNA/DNA duplexes in 3′-IGRs</p>
<p class="siDoi">doi:10.1371/journal.pone.0000290.s009</p><a id="article1.body1.sec5.supplementary-material9.caption1.p2" name="article1.body1.sec5.supplementary-material9.caption1.p2"></a><p class="postSiDOI">(0.04 MB DOC)</p>
<a name="pone.0000290.s010" id="pone.0000290.s010"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000290.s010">Table S9. </a></strong></p><a id="article1.body1.sec5.supplementary-material10.caption1.p1" name="article1.body1.sec5.supplementary-material10.caption1.p1"></a><p class="preSiDOI">Distribution of the nearest-neighbor interactions in sense and antisense RNA/DNA duplexes in introns</p>
<p class="siDoi">doi:10.1371/journal.pone.0000290.s010</p><a id="article1.body1.sec5.supplementary-material10.caption1.p2" name="article1.body1.sec5.supplementary-material10.caption1.p2"></a><p class="postSiDOI">(0.04 MB DOC)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>We thank A. Gospodinov M. Sarov, and M. Ivanov for critically reading the manuscript, as well as M. Velev and V. Marchev for database design.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: SS. Performed the experiments: SS RK MN AI DK. Analyzed the data: SS RK. Contributed reagents/materials/analysis tools: SS. Wrote the paper: SS RK AR. Other: Wrote the software: AR.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0000290-Borer1" id="pone.0000290-Borer1"></a>Borer PN, Dengler B, Tinoco I, Jr., Uhlenbeck OC (1974) Stability of ribonucleic acid double-stranded helices. J Mol Biol  86: 843–853.  <ul class="find" data-citedArticleID="936960" data-doi="10.1016/0022-2836(74)90357-x"><li><a href="http://dx.doi.org/10.1016/0022-2836(74)90357-x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Stability+of+ribonucleic+acid+double-stranded+helices." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Stability+of+ribonucleic+acid+double-stranded+helices.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0000290-Breslauer1" id="pone.0000290-Breslauer1"></a>Breslauer KJ, Frank R, Blocker H, Marky LA (1986) Predicting DNA duplex stability from the base sequence. Proc Natl Acad Sci U S A  83: 3746–3750.  <ul class="find" data-citedArticleID="936962" data-doi="10.1073/pnas.83.11.3746"><li><a href="http://dx.doi.org/10.1073/pnas.83.11.3746" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Predicting+DNA+duplex+stability+from+the+base+sequence." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Predicting+DNA+duplex+stability+from+the+base+sequence.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0000290-Sugimoto1" id="pone.0000290-Sugimoto1"></a>Sugimoto N, Nakano S, Katoh M, Matsumura A, Nakamuta H, et al.  (1995) Thermodynamic parameters to predict stability of RNA/DNA hybrid duplexes. Biochemistry  34: 11211–11216.  <ul class="find" data-citedArticleID="937002" data-doi="10.1021/bi00035a029"><li><a href="http://dx.doi.org/10.1021/bi00035a029" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Thermodynamic+parameters+to+predict+stability+of+RNA%2FDNA+hybrid+duplexes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Thermodynamic+parameters+to+predict+stability+of+RNA%2FDNA+hybrid+duplexes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0000290-Natale1" id="pone.0000290-Natale1"></a>Natale DA, Schubert AE, Kowalski D (1992) DNA helical stability accounts for mutational defects in a yeast replication origin. Proc Natl Acad Sci U S A  89: 2654–2658.  <ul class="find" data-citedArticleID="936992" data-doi="10.1073/pnas.89.7.2654"><li><a href="http://dx.doi.org/10.1073/pnas.89.7.2654" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=DNA+helical+stability+accounts+for+mutational+defects+in+a+yeast+replication+origin." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22DNA+helical+stability+accounts+for+mutational+defects+in+a+yeast+replication+origin.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0000290-Natale2" id="pone.0000290-Natale2"></a>Natale DA, Umek RM, Kowalski D (1993) Ease of DNA unwinding is a conserved property of yeast replication origins. Nucleic Acids Res  21: 555–560.  <ul class="find" data-citedArticleID="936994" data-doi="10.1093/nar/21.3.555"><li><a href="http://dx.doi.org/10.1093/nar/21.3.555" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Ease+of+DNA+unwinding+is+a+conserved+property+of+yeast+replication+origins." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Ease+of+DNA+unwinding+is+a+conserved+property+of+yeast+replication+origins.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0000290-Huang1" id="pone.0000290-Huang1"></a>Huang Y, Kowalski D (2003) WEB-THERMODYN: Sequence analysis software for profiling DNA helical stability. Nucleic Acids Res  31: 3819–3821.  <ul class="find" data-citedArticleID="936982" data-doi="10.1093/nar/gkg562"><li><a href="http://dx.doi.org/10.1093/nar/gkg562" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=WEB-THERMODYN%3A+Sequence+analysis+software+for+profiling+DNA+helical+stability." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22WEB-THERMODYN%3A+Sequence+analysis+software+for+profiling+DNA+helical+stability.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0000290-Benham1" id="pone.0000290-Benham1"></a>Benham CJ (1996) Duplex destabilization in superhelical DNA is predicted to occur at specific transcriptional regulatory regions. J Mol Biol  255: 425–434.  <ul class="find" data-citedArticleID="936956" data-doi="10.1006/jmbi.1996.0035"><li><a href="http://dx.doi.org/10.1006/jmbi.1996.0035" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Duplex+destabilization+in+superhelical+DNA+is+predicted+to+occur+at+specific+transcriptional+regulatory+regions." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Duplex+destabilization+in+superhelical+DNA+is+predicted+to+occur+at+specific+transcriptional+regulatory+regions.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0000290-David1" id="pone.0000290-David1"></a>David L, Huber W, Granovskaia M, Toedling J, Palm CJ, et al.  (2006) A high-resolution map of transcription in the yeast genome. Proc Natl Acad Sci U S A  103: 5320–5325.  <ul class="find" data-citedArticleID="936964" data-doi="10.1073/pnas.0601091103"><li><a href="http://dx.doi.org/10.1073/pnas.0601091103" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+high-resolution+map+of+transcription+in+the+yeast+genome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+high-resolution+map+of+transcription+in+the+yeast+genome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0000290-Keller1" id="pone.0000290-Keller1"></a>Keller W, Minvielle-Sebastia L (1997) A comparison of mammalian and yeast pre-mRNA 3′-end processing. Curr Opin Cell Biol  9: 329–336.  <ul class="find" data-citedArticleID="936984" data-doi="10.1016/s0955-0674(97)80004-x"><li><a href="http://dx.doi.org/10.1016/s0955-0674(97)80004-x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+comparison+of+mammalian+and+yeast+pre-mRNA+3%26%23x2032%3B-end+processing." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+comparison+of+mammalian+and+yeast+pre-mRNA+3%26%23x2032%3B-end+processing.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0000290-Graber1" id="pone.0000290-Graber1"></a>Graber JH, Cantor CR, Mohr SC, Smith TF (1999) Genomic detection of new yeast pre-mRNA 3′-end-processing signals. Nucleic Acids Res  27: 888–894.  <ul class="find" data-citedArticleID="936974" data-doi="10.1093/nar/27.3.888"><li><a href="http://dx.doi.org/10.1093/nar/27.3.888" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genomic+detection+of+new+yeast+pre-mRNA+3%26%23x2032%3B-end-processing+signals." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genomic+detection+of+new+yeast+pre-mRNA+3%26%23x2032%3B-end-processing+signals.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0000290-Graber2" id="pone.0000290-Graber2"></a>Graber JH, McAllister GD, Smith TF (2002) Probabilistic prediction of Saccharomyces cerevisiae mRNA 3′-processing sites. Nucleic Acids Res  30: 1851–1858.  <ul class="find" data-citedArticleID="936976" data-doi="10.1093/nar/30.8.1851"><li><a href="http://dx.doi.org/10.1093/nar/30.8.1851" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Probabilistic+prediction+of+Saccharomyces+cerevisiae+mRNA+3%26%23x2032%3B-processing+sites." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Probabilistic+prediction+of+Saccharomyces+cerevisiae+mRNA+3%26%23x2032%3B-processing+sites.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0000290-vanHelden1" id="pone.0000290-vanHelden1"></a>van Helden J, del Olmo M, Perez-Ortin JE (2000) Statistical analysis of yeast genomic downstream sequences reveals putative polyadenylation signals. Nucleic Acids Res  28: 1000–1010.  <ul class="find" data-citedArticleID="937008" data-doi="10.1093/nar/28.4.1000"><li><a href="http://dx.doi.org/10.1093/nar/28.4.1000" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Statistical+analysis+of+yeast+genomic+downstream+sequences+reveals+putative+polyadenylation+signals." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Statistical+analysis+of+yeast+genomic+downstream+sequences+reveals+putative+polyadenylation+signals.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0000290-Kellis1" id="pone.0000290-Kellis1"></a>Kellis M, Patterson N, Endrizzi M, Birren B, Lander ES (2003) Sequencing and comparison of yeast species to identify genes and regulatory elements. Nature  423: 241–254.  <ul class="find" data-citedArticleID="936986" data-doi="10.1038/nature01644"><li><a href="http://dx.doi.org/10.1038/nature01644" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Sequencing+and+comparison+of+yeast+species+to+identify+genes+and+regulatory+elements." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Sequencing+and+comparison+of+yeast+species+to+identify+genes+and+regulatory+elements.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0000290-Holstege1" id="pone.0000290-Holstege1"></a>Holstege FC, Jennings EG, Wyrick JJ, Lee TI, Hengartner CJ, et al.  (1998) Dissecting the regulatory circuitry of a eukaryotic genome. Cell  95: 717–728.  <ul class="find" data-citedArticleID="936980" data-doi="10.1016/s0092-8674(00)81641-4"><li><a href="http://dx.doi.org/10.1016/s0092-8674(00)81641-4" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Dissecting+the+regulatory+circuitry+of+a+eukaryotic+genome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Dissecting+the+regulatory+circuitry+of+a+eukaryotic+genome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0000290-Kellis2" id="pone.0000290-Kellis2"></a>Kellis M, Patterson N, Birren B, Berger B, Lander ES (2004) Methods in comparative genomics: genome correspondence, gene identification and regulatory motif discovery. J Comput Biol  11: 319–355.  <ul class="find" data-citedArticleID="936988" data-doi="10.1089/1066527041410319"><li><a href="http://dx.doi.org/10.1089/1066527041410319" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Methods+in+comparative+genomics%3A+genome+correspondence%2C+gene+identification+and+regulatory+motif+discovery." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Methods+in+comparative+genomics%3A+genome+correspondence%2C+gene+identification+and+regulatory+motif+discovery.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0000290-Bentley1" id="pone.0000290-Bentley1"></a>Bentley DL (2005) Rules of engagement: co-transcriptional recruitment of pre-mRNA processing factors. Curr Opin Cell Biol  17: 251–256.  <ul class="find" data-citedArticleID="936958" data-doi="10.1016/j.ceb.2005.04.006"><li><a href="http://dx.doi.org/10.1016/j.ceb.2005.04.006" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Rules+of+engagement%3A+co-transcriptional+recruitment+of+pre-mRNA+processing+factors." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Rules+of+engagement%3A+co-transcriptional+recruitment+of+pre-mRNA+processing+factors.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0000290-Dye1" id="pone.0000290-Dye1"></a>Dye MJ, Gromak N, Proudfoot NJ (2006) Exon tethering in transcription by RNA polymerase II. Mol Cell  21: 849–859.  <ul class="find" data-citedArticleID="936968" data-doi="10.1016/j.molcel.2006.01.032"><li><a href="http://dx.doi.org/10.1016/j.molcel.2006.01.032" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Exon+tethering+in+transcription+by+RNA+polymerase+II." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Exon+tethering+in+transcription+by+RNA+polymerase+II.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0000290-Nudler1" id="pone.0000290-Nudler1"></a>Nudler E, Mustaev A, Lukhtanov E, Goldfarb A (1997) The RNA-DNA hybrid maintains the register of transcription by preventing backtracking of RNA polymerase. Cell  89: 33–41.  <ul class="find" data-citedArticleID="936996" data-doi="10.1016/s0092-8674(00)80180-4"><li><a href="http://dx.doi.org/10.1016/s0092-8674(00)80180-4" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+RNA-DNA+hybrid+maintains+the+register+of+transcription+by+preventing+backtracking+of+RNA+polymerase." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+RNA-DNA+hybrid+maintains+the+register+of+transcription+by+preventing+backtracking+of+RNA+polymerase.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0000290-Kireeva1" id="pone.0000290-Kireeva1"></a>Kireeva ML, Komissarova N, Kashlev M (2000) Overextended RNA:DNA hybrid as a negative regulator of RNA polymerase II processivity. J Mol Biol  299: 325–335.  <ul class="find" data-citedArticleID="936990" data-doi="10.1006/jmbi.2000.3755"><li><a href="http://dx.doi.org/10.1006/jmbi.2000.3755" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Overextended+RNA%3ADNA+hybrid+as+a+negative+regulator+of+RNA+polymerase+II+processivity." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Overextended+RNA%3ADNA+hybrid+as+a+negative+regulator+of+RNA+polymerase+II+processivity.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0000290-Rahmouni1" id="pone.0000290-Rahmouni1"></a>Rahmouni AR, Wells RD (1992) Direct evidence for the effect of transcription on local DNA supercoiling in vivo. J Mol Biol  223: 131–144.  <ul class="find" data-citedArticleID="936998" data-doi="10.1016/0022-2836(92)90721-u"><li><a href="http://dx.doi.org/10.1016/0022-2836(92)90721-u" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Direct+evidence+for+the+effect+of+transcription+on+local+DNA+supercoiling+in+vivo." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Direct+evidence+for+the+effect+of+transcription+on+local+DNA+supercoiling+in+vivo.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0000290-Westover1" id="pone.0000290-Westover1"></a>Westover KD, Bushnell DA, Kornberg RD (2004) Structural basis of transcription: separation of RNA from DNA by RNA polymerase II. Science  303: 1014–1016.  <ul class="find" data-citedArticleID="937004" data-doi="10.1126/science.1090839"><li><a href="http://dx.doi.org/10.1126/science.1090839" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Structural+basis+of+transcription%3A+separation+of+RNA+from+DNA+by+RNA+polymerase+II." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Structural+basis+of+transcription%3A+separation+of+RNA+from+DNA+by+RNA+polymerase+II.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0000290-Gnatt1" id="pone.0000290-Gnatt1"></a>Gnatt AL, Cramer P, Fu J, Bushnell DA, Kornberg RD (2001) Structural basis of transcription: an RNA polymerase II elongation complex at 3.3 A resolution. Science  292: 1876–1882.  <ul class="find" data-citedArticleID="936970" data-doi="10.1126/science.1059495"><li><a href="http://dx.doi.org/10.1126/science.1059495" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Structural+basis+of+transcription%3A+an+RNA+polymerase+II+elongation+complex+at+3.3+A+resolution." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Structural+basis+of+transcription%3A+an+RNA+polymerase+II+elongation+complex+at+3.3+A+resolution.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0000290-Yager1" id="pone.0000290-Yager1"></a>Yager TD, von Hippel PH (1991) A thermodynamic analysis of RNA transcript elongation and termination in Escherichia coli. Biochemistry  30: 1097–1118.  <ul class="find" data-citedArticleID="937006" data-doi="10.1021/bi00218a032"><li><a href="http://dx.doi.org/10.1021/bi00218a032" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+thermodynamic+analysis+of+RNA+transcript+elongation+and+termination+in+Escherichia+coli." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+thermodynamic+analysis+of+RNA+transcript+elongation+and+termination+in+Escherichia+coli.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0000290-Greive1" id="pone.0000290-Greive1"></a>Greive SJ, von Hippel PH (2005) Thinking quantitatively about transcriptional regulation. Nat Rev Mol Cell Biol  6: 221–232.  <ul class="find" data-citedArticleID="936978" data-doi="10.1038/nrm1588"><li><a href="http://dx.doi.org/10.1038/nrm1588" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Thinking+quantitatively+about+transcriptional+regulation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Thinking+quantitatively+about+transcriptional+regulation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0000290-Artsimovitch1" id="pone.0000290-Artsimovitch1"></a>Artsimovitch I, Landick R (2000) Pausing by bacterial RNA polymerase is mediated by mechanistically distinct classes of signals. Proc Natl Acad Sci U S A  97: 7090–7095.  <ul class="find" data-citedArticleID="936954" data-doi="10.1073/pnas.97.13.7090"><li><a href="http://dx.doi.org/10.1073/pnas.97.13.7090" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Pausing+by+bacterial+RNA+polymerase+is+mediated+by+mechanistically+distinct+classes+of+signals." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Pausing+by+bacterial+RNA+polymerase+is+mediated+by+mechanistically+distinct+classes+of+signals.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0000290-Goffeau1" id="pone.0000290-Goffeau1"></a>Goffeau A, Barrell BG, Bussey H, Davis RW, Dujon B, et al.  (1996) Life with 6000 genes. Science  274: 546, 563–547.  <ul class="find" data-citedArticleID="936972" data-doi="10.1126/science.274.5287.546"><li><a href="http://dx.doi.org/10.1126/science.274.5287.546" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Life+with+6000+genes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Life+with+6000+genes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0000290-Dove1" id="pone.0000290-Dove1"></a>Dove WF, Davidson N (1962) Cation effects on the denaturation of DNA. J Mol Biol  5: 467–478.  <ul class="find" data-citedArticleID="936966" data-doi="10.1016/s0022-2836(62)80119-3"><li><a href="http://dx.doi.org/10.1016/s0022-2836(62)80119-3" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Cation+effects+on+the+denaturation+of+DNA." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Cation+effects+on+the+denaturation+of+DNA.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0000290-Schildkraut1" id="pone.0000290-Schildkraut1"></a>Schildkraut C (1965) Dependence of the melting temperature of DNA on salt concentration. Biopolymers  3: 195–208.  <ul class="find" data-citedArticleID="937000" data-doi="10.1002/bip.360030207"><li><a href="http://dx.doi.org/10.1002/bip.360030207" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Dependence+of+the+melting+temperature+of+DNA+on+salt+concentration." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Dependence+of+the+melting+temperature+of+DNA+on+salt+concentration.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0000290-Ak1" id="pone.0000290-Ak1"></a>Ak P, Benham CJ (2005) Susceptibility to superhelically driven DNA duplex destabilization: a highly conserved property of yeast replication origins. PLoS Comput Biol  1: e7.  <ul class="find" data-citedArticleID="936952" data-doi="10.1371/journal.pcbi.0010007"><li><a href="http://dx.doi.org/10.1371/journal.pcbi.0010007" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Susceptibility+to+superhelically+driven+DNA+duplex+destabilization%3A+a+highly+conserved+property+of+yeast+replication+origins." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Susceptibility+to+superhelically+driven+DNA+duplex+destabilization%3A+a+highly+conserved+property+of+yeast+replication+origins.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000290.XML" value="81440"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000290.PDF" value="286184"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000290.g001.PNG_L" value="755345"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000290.g001.PNG_M" value="196211"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000290.g001.PNG_S" value="7008"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000290.g001.TIF" value="1930180"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000290.g001.PNG_I" value="77023"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000290.g002.PNG_L" value="595330"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000290.g002.PNG_M" value="90436"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000290.g002.PNG_S" value="4843"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000290.g002.TIF" value="1368756"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000290.g002.PNG_I" value="54290"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000290.t001.PNG_L" value="34373"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000290.t001.PNG_M" value="33649"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000290.t001.PNG_S" value="2406"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000290.t001.TIF" value="195374"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000290.t001.PNG_I" value="15409"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000290.t002.PNG_L" value="108006"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000290.t002.PNG_M" value="37244"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000290.t002.PNG_S" value="2505"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000290.t002.TIF" value="475684"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000290.t002.PNG_I" value="15981"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000290.t003.PNG_L" value="56034"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000290.t003.PNG_M" value="54921"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000290.t003.PNG_S" value="2879"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000290.t003.TIF" value="274418"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000290.t003.PNG_I" value="24200"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000290.g003.PNG_L" value="71286"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000290.g003.PNG_M" value="20095"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000290.g003.PNG_S" value="1822"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000290.g003.TIF" value="131856"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000290.g003.PNG_I" value="11498"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000290.t004.PNG_L" value="121126"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000290.t004.PNG_M" value="110339"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000290.t004.PNG_S" value="5685"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000290.t004.TIF" value="580040"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000290.t004.PNG_I" value="55093"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000290.g004.PNG_L" value="310891"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000290.g004.PNG_M" value="134765"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000290.g004.PNG_S" value="4253"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000290.g004.TIF" value="433592"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000290.g004.PNG_I" value="53990"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000290.s001.DOC" value="26624"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000290.s002.ZIP" value="4664890"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000290.s003.ZIP" value="313880"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000290.s004.DOC" value="42496"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000290.s005.ZIP" value="3194721"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000290.s006.DOC" value="77312"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000290.s007.ZIP" value="72218"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000290.s008.DOC" value="44032"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000290.s009.DOC" value="44032"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000290.s010.DOC" value="44032"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000290&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000290" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000290&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0000290&volume=&issue=&title=Stability of mRNA/DNA and DNA/DNA Duplexes Affects mRNA Transcription&author_name=Rayna%20I.%20Kraeva%2C%20Dragomir%20B.%20Krastev%2C%20Assen%20Roguev%2C%20Anna%20Ivanova%2C%20Marina%20N.%20Nedelcheva-Veleva%2C%20Stoyno%20S.%20Stoynov&start_page=1&end_page=8" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000290" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000290" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0000290" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000290&amp;t=Stability%20of%20mRNA%2FDNA%20and%20DNA%2FDNA%20Duplexes%20Affects%20mRNA%20Transcription" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000290&title=Stability%20of%20mRNA%2FDNA%20and%20DNA%2FDNA%20Duplexes%20Affects%20mRNA%20Transcription&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000290&amp;title=Stability%20of%20mRNA%2FDNA%20and%20DNA%2FDNA%20Duplexes%20Affects%20mRNA%20Transcription" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000290" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0000290&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Stability of mRNA/DNA and DNA/DNA Duplexes Affects mRNA Transcription';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0000290';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Stability%20of%20mRNA%2FDNA%20and%20DNA%2FDNA%20Duplexes%20Affects%20mRNA%20Transcription http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000290" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0000290" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">












          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Antisense+RNA%22" title="Search for articles in the subject area:'Antisense RNA'"><div class="flagText">Antisense RNA</div></a>
              <div data-categoryid="19975" data-articleid="23418"
                   data-categoryname="Antisense RNA"
                   class="flagImage" title="Flag 'Antisense RNA' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22DNA%22" title="Search for articles in the subject area:'DNA'"><div class="flagText">DNA</div></a>
              <div data-categoryid="32449" data-articleid="23418"
                   data-categoryname="DNA"
                   class="flagImage" title="Flag 'DNA' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22DNA+sequences%22" title="Search for articles in the subject area:'DNA sequences'"><div class="flagText">DNA sequences</div></a>
              <div data-categoryid="32003" data-articleid="23418"
                   data-categoryname="DNA sequences"
                   class="flagImage" title="Flag 'DNA sequences' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22DNA+transcription%22" title="Search for articles in the subject area:'DNA transcription'"><div class="flagText">DNA transcription</div></a>
              <div data-categoryid="20231" data-articleid="23418"
                   data-categoryname="DNA transcription"
                   class="flagImage" title="Flag 'DNA transcription' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Gene+prediction%22" title="Search for articles in the subject area:'Gene prediction'"><div class="flagText">Gene prediction</div></a>
              <div data-categoryid="40279" data-articleid="23418"
                   data-categoryname="Gene prediction"
                   class="flagImage" title="Flag 'Gene prediction' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Messenger+RNA%22" title="Search for articles in the subject area:'Messenger RNA'"><div class="flagText">Messenger RNA</div></a>
              <div data-categoryid="18353" data-articleid="23418"
                   data-categoryname="Messenger RNA"
                   class="flagImage" title="Flag 'Messenger RNA' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22RNA+stability%22" title="Search for articles in the subject area:'RNA stability'"><div class="flagText">RNA stability</div></a>
              <div data-categoryid="21911" data-articleid="23418"
                   data-categoryname="RNA stability"
                   class="flagImage" title="Flag 'RNA stability' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Thermodynamics%22" title="Search for articles in the subject area:'Thermodynamics'"><div class="flagText">Thermodynamics</div></a>
              <div data-categoryid="18103" data-articleid="23418"
                   data-categoryname="Thermodynamics"
                   class="flagImage" title="Flag 'Thermodynamics' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=6845'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=6536'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=959&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>


</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
