

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: HIV-Specific Probabilistic Models of Protein Evolution</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0000503"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0000503" />

    <meta name="citation_title" content="HIV-Specific Probabilistic Models of Protein Evolution"/>
    <meta itemprop="name" content="HIV-Specific Probabilistic Models of Protein Evolution"/>

      <meta name="citation_author" content="David C. Nickle"/>
            <meta name="citation_author_institution" content="Department of Microbiology, University of Washington School of Medicine, Seattle, Washington, United States of America"/>
      <meta name="citation_author" content="Laura Heath"/>
            <meta name="citation_author_institution" content="Department of Microbiology, University of Washington School of Medicine, Seattle, Washington, United States of America"/>
      <meta name="citation_author" content="Mark A. Jensen"/>
            <meta name="citation_author_institution" content="Department of Microbiology, University of Washington School of Medicine, Seattle, Washington, United States of America"/>
      <meta name="citation_author" content="Peter B. Gilbert"/>
            <meta name="citation_author_institution" content="Department of Biostatistics, University of Washington and Fred Hutchinson Cancer Research Center, Seattle, Washington, United States of America"/>
      <meta name="citation_author" content="James I. Mullins"/>
            <meta name="citation_author_institution" content="Department of Microbiology, University of Washington School of Medicine, Seattle, Washington, United States of America"/>
      <meta name="citation_author" content="Sergei L. Kosakovsky Pond"/>
            <meta name="citation_author_institution" content="Department of Pathology, University of California, San Diego, La Jolla, California, United States of America"/>

    <meta name="citation_date" content="2007/6/6"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0000503.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e503"/>
    <meta name="citation_issue" content="6"/>
    <meta name="citation_volume" content="2"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Dating of the human ape splitting by a molecular clock of mitochondrial DNA.; citation_author=M Hasegawa; citation_author=H Kishino; citation_author=TA Yano; citation_journal_title=J Mol Evol; citation_volume=22; citation_number=1; citation_pages=160-174; citation_date=1985; " />
      <meta name="citation_reference" content="citation_title=A likelihood approach for comparing synonymous and nonsynonymous nucleotide substitution rates, with application to the chloroplast genome.; citation_author=SV Muse; citation_author=BS Gaut; citation_journal_title=Mol Biol Evol; citation_volume=11; citation_number=2; citation_pages=715-724; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=A codon-based model of nucleotide substitution for protein-coding DNA sequences.; citation_author=N Goldman; citation_author=Z Yang; citation_journal_title=Mol Biol Evol; citation_volume=11; citation_number=3; citation_pages=725-736; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=A model of evolutionary change in proteins.; citation_author=MO Dayhoff; citation_author=RM Schwartz; citation_author=BC Orcutt; citation_number=4; citation_pages=345-352; citation_date=1978; " />
      <meta name="citation_reference" content="citation_title=Amino Acid Substitution Matrices from Protein Blocks.; citation_author=S Henikoff; citation_author=JG Henikoff; citation_journal_title=PNAS; citation_volume=89; citation_number=5; citation_pages=10915-10919; citation_date=1992; " />
      <meta name="citation_reference" content="citation_title=Multiple-Alphabet Amino Acid Sequence Comparisons of the Immunoglobulin kappa-Chain Constant Domain.; citation_author=S Karlin; citation_author=G Ghandour; citation_journal_title=PNAS; citation_volume=82; citation_number=6; citation_pages=8597-8601; citation_date=1985; " />
      <meta name="citation_reference" content="citation_title=Mutation data matrix and its uses.; citation_author=DG George; citation_author=WC Barker; citation_author=LT Hunt; citation_journal_title=Methods Enzymol; citation_volume=183; citation_number=7; citation_pages=333-351; citation_date=1990; " />
      <meta name="citation_reference" content="citation_title=Aligning amino acid sequences: comparison of commonly used methods.; citation_author=DF Feng; citation_author=MS Johnson; citation_author=RF Doolittle; citation_journal_title=J Mol Evol; citation_volume=21; citation_number=8; citation_pages=112-125; citation_date=1984; " />
      <meta name="citation_reference" content="citation_title=A new approach to clustering the amino acids.; citation_author=L Stanfel; citation_journal_title=J Theor Biol; citation_volume=183; citation_number=9; citation_pages=195-205; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=The Exchangeability of Amino Acids in Proteins.; citation_author=LY Yampolsky; citation_author=A Stoltzfus; citation_journal_title=Genetics; citation_volume=170; citation_number=10; citation_pages=1459-1472; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=A Universal Evolutionary Index for Amino Acid Changes.; citation_author=H Tang; citation_author=GJ Wyckoff; citation_author=J Lu; citation_author=C-I Wu; citation_journal_title=Mol Biol Evol; citation_volume=21; citation_number=11; citation_pages=1548-1556; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Evolutionary trees from DNA-sequences-a maximum-likelihood approach.; citation_author=J Felsenstein; citation_journal_title=J Mol Evol; citation_volume=17; citation_number=12; citation_pages=368-376; citation_date=1981; " />
      <meta name="citation_reference" content="citation_title=Models of amino acid substitution and applications to mitochondrial protein evolution.; citation_author=Z Yang; citation_author=R Nielsen; citation_author=M Hasegawa; citation_journal_title=Mol Biol Evol; citation_volume=15; citation_number=13; citation_pages=1600-1611; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=A General Empirical Model of Protein Evolution Derived from Multiple Protein Families Using a Maximum-Likelihood Approach.; citation_author=S Whelan; citation_author=N Goldman; citation_journal_title=Mol Biol Evol; citation_volume=18; citation_number=14; citation_pages=691-699; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=rtREV: An Amino Acid Substitution Matrix for Inference of Retrovirus and Reverse Transcriptase Phylogeny.; citation_author=MW Dimmic; citation_author=JS Rest; citation_author=DP Mindell; citation_author=RA Goldstein; citation_journal_title=J Mol Evol; citation_volume=55; citation_number=15; citation_pages=65-73; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=The mutation rate of human immunodeficiency virus type 1 is influenced by the vpr gene.; citation_author=LM Mansky; citation_journal_title=Virology; citation_volume=222; citation_number=16; citation_pages=391-400; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Estimate of the mutation rate per nucleotide in humans.; citation_author=MW Nachman; citation_author=SL Crowell; citation_journal_title=Genetics; citation_volume=156; citation_number=17; citation_pages=297-304; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Human Immunodeficiency Virus Type 1 env Evolves toward Ancestral States upon Transmission to a New Host.; citation_author=JT Herbeck; citation_author=DC Nickle; citation_author=GH Learn; citation_author=GS Gottlieb; citation_author=ME Curlin; citation_journal_title=J Virol; citation_volume=80; citation_number=18; citation_pages=1637-1644; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Adaptation to Different Human Populations by HIV-1 Revealed by Codon-Based Analyses.; citation_author=SL Kosakovsky Pond; citation_author=SDW Frost; citation_author=Z Grossman; citation_author=MB Gravenor; citation_author=DD Richman; citation_journal_title=PLoS Computational Biology; citation_volume=2; citation_number=19; citation_pages=e62; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Selection on the Human Immunodeficiency Virus Type 1 Proteome following Primary Infection.; citation_author=Y Liu; citation_author=J McNevin; citation_author=J Cao; citation_author=H Zhao; citation_author=I Genowati; citation_journal_title=J Virol; citation_volume=80; citation_number=20; citation_pages=9519-9529; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Tat-specific cytotoxic T lymphocytes select for SIV escape variants during resolution of primary viraemia.; citation_author=TM Allen; citation_author=DH O'Connor; citation_author=P Jing; citation_author=JL Dzuris; citation_author=BR Mothe; citation_journal_title=Nature; citation_volume=407; citation_number=21; citation_pages=386-390; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Reversion of CTL escape-variant immunodeficiency viruses in vivo.; citation_author=TC Friedrich; citation_author=EJ Dodds; citation_author=LJ Yant; citation_author=L Vojnov; citation_author=R Rudersdorf; citation_journal_title=Nat Med; citation_volume=10; citation_number=22; citation_pages=275-281; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=The neighbor-joining method: a new method for reconstructing phylogenetic trees.; citation_author=N Saitou; citation_author=M Nei; citation_journal_title=Mol Biol Evol; citation_volume=4; citation_number=23; citation_pages=406-425; citation_date=1987; " />
      <meta name="citation_reference" content="citation_title=Estimation of the number of nucleotide substitutions in the control region of mitochondrial DNA in humans and chimpanzees.; citation_author=K Tamura; citation_author=M Nei; citation_journal_title=Mol Biol Evol; citation_volume=10; citation_number=24; citation_pages=512-526; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=Automated Phylogenetic Detection of Recombination Using a Genetic Algorithm.; citation_author=SL Kosakovsky Pond; citation_author=D Posada; citation_author=MB Gravenor; citation_author=CH Woelk; citation_author=SDW Frost; citation_journal_title=Mol Biol Evol; citation_volume=23; citation_number=25; citation_pages=1891-1901; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=HyPhy: hypothesis testing using phylogenies.; citation_author=SLK Pond; citation_author=SDW Frost; citation_author=SV Muse; citation_journal_title=Bioinformatics; citation_volume=21; citation_number=26; citation_pages=676-679; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=A simple hierarchical approach to modeling distributions of substitution rates.; citation_author=SL Kosakovsky Pond; citation_author=SDW Frost; citation_journal_title=Mol Biol Evol; citation_volume=22; citation_number=27; citation_pages=223-234; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=The rapid generation of mutation data matrices from protein sequences.; citation_author=D Jones; citation_author=W Taylor; citation_author=J Thornton; citation_journal_title=Comput Appl Biosci; citation_volume=8; citation_number=28; citation_pages=275-282; citation_date=1992; " />
      <meta name="citation_reference" content="citation_title=Further analysis of the data by Akaike's information criterion and the finite corrections.; citation_author=N Sugiura; citation_journal_title=Comm Stat Theory Methods; citation_volume=A7; citation_number=29; citation_pages=13-26; citation_date=1978; " />
      <meta name="citation_reference" content="citation_title=Model of amino acid substitution in proteins encoded by mitochondrial DNA.; citation_author=J Adachi; citation_author=M Hasegawa; citation_journal_title=J Mol Evol; citation_volume=42; citation_number=30; citation_pages=459-468; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Evolutionary Model Selection with a Genetic Algorithm: a Case Study Using Stem RNA.; citation_author=SL Kosakovsky Pond; citation_author=FV Mannino; citation_author=MB Gravenor; citation_author=SV Muse; citation_author=SDW Frost; citation_journal_title=Mol Biol Evol; citation_number=31; citation_pages=msl144; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Model Selection and Model Averaging in Phylogenetics: Advantages of Akaike Information Criterion and Bayesian Approaches Over Likelihood Ratio Tests.; citation_author=D Posada; citation_author=TR Buckley; citation_journal_title=Syst Biol; citation_volume=53; citation_number=32; citation_pages=793-808; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Evolutionary model selection with a genetic algorithm: a case study using stem RNA.; citation_author=SL Kosakovsky Pond; citation_author=FV Mannino; citation_author=MB Gravenor; citation_author=SV Muse; citation_author=SD Frost; citation_journal_title=Mol Biol Evol; citation_volume=24; citation_number=33; citation_pages=159-170; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Model selection and multimodel inference: a practical information-theoretic approach.; citation_author=KP Burnham; citation_author=DR Anderson; citation_number=34; citation_pages=488; citation_date=2002; citation_publisher=Springer-Verlag; " />
      <meta name="citation_reference" content="citation_title=Accurate reconstruction of a known HIV-1 transmission history by phylogenetic tree†analysis.; citation_author=T Leitner; citation_author=D Escanilla; citation_author=C Franzen; citation_author=M Uhlen; citation_author=J Albert; citation_journal_title=PNAS; citation_volume=93; citation_number=35; citation_pages=10864-10869; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Consistent viral evolutionary changes associated with the progression of human immunodeficiency virus type 1 infection.; citation_author=R Shankarappa; citation_author=JB Margolick; citation_author=SJ Gange; citation_author=AG Rodrigo; citation_author=D Upchurch; citation_journal_title=J Virol; citation_volume=73; citation_number=36; citation_pages=10489-10502; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Consensus and ancestral state HIV vaccines.; citation_author=DC Nickle; citation_author=MA Jensen; citation_author=GS Gottlieb; citation_author=D Shriner; citation_author=GH Learn; citation_journal_title=Science; citation_volume=299; citation_number=37; citation_pages=1515-1517; citation_date=2003; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000503" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="HIV-Specific Probabilistic Models of Protein Evolution"/>
    <meta name="twitter:description" content="Comparative sequence analyses, including such fundamental bioinformatics techniques as similarity searching, sequence alignment and phylogenetic inference, have become a mainstay for researchers studying type 1 Human Immunodeficiency Virus (HIV-1) genome structure and evolution. Implicit in comparative analyses is an underlying model of evolution, and the chosen model can significantly affect the results. In general, evolutionary models describe the probabilities of replacing one amino acid character with another over a period of time. Most widely used evolutionary models for protein sequences have been derived from curated alignments of hundreds of proteins, usually based on mammalian genomes. It is unclear to what extent these empirical models are generalizable to a very different organism, such as HIV-1&ndash;the most extensively sequenced organism in existence. We developed a maximum likelihood model fitting procedure to a collection of HIV-1 alignments sampled from different viral genes, and inferred two empirical substitution models, suitable for describing between-and within-host evolution. Our procedure pools the information from multiple sequence alignments, and provided software implementation can be run efficiently in parallel on a computer cluster. We describe how the inferred substitution models can be used to generate scoring matrices suitable for alignment and similarity searches. Our models had a consistently superior fit relative to the best existing models and to parameter-rich data-driven models when benchmarked on independent HIV-1 alignments, demonstrating evolutionary biases in amino-acid substitution that are unique to HIV, and that are not captured by the existing models. The scoring matrices derived from the models showed a marked difference from common amino-acid scoring matrices. The use of an appropriate evolutionary model recovered a known viral transmission history, whereas a poorly chosen model introduced phylogenetic error. We argue that our model derivation procedure is immediately applicable to other organisms with extensive sequence data available, such as Hepatitis C and Influenza A viruses."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0000503.g005"/>

  <meta property="og:title" content="HIV-Specific Probabilistic Models of Protein Evolution" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000503" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=1513'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=1621'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=3710&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0000503">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000503" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000503&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    HIV-Specific Probabilistic Models of Protein Evolution
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            David C. Nickle, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Microbiology, University of Washington School of Medicine, Seattle, Washington, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Laura Heath, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Microbiology, University of Washington School of Medicine, Seattle, Washington, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Mark A. Jensen, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Microbiology, University of Washington School of Medicine, Seattle, Washington, United States of America
                </p>

                <p>
                    Current address: Department of HABE (Epidemiology), College of Public Health, and Department of Genetics in the Franklin College of Arts and Sciences, University of Georgia, Athens, Georgia, United States of America
                </p>

              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Peter B. Gilbert, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Biostatistics, University of Washington and Fred Hutchinson Cancer Research Center, Seattle, Washington, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            James I. Mullins, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Microbiology, University of Washington School of Medicine, Seattle, Washington, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Sergei L. Kosakovsky Pond
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:spond@ucsd.edu">spond@ucsd.edu</a></p>

                <p>Affiliation:
                  Department of Pathology, University of California, San Diego, La Jolla, California, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: June 06, 2007</li>
    <li>DOI: 10.1371/journal.pone.0000503</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0000503" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0000503" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000503" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0000503" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0000503-t001" data-doi="info:doi/10.1371/journal.pone.0000503" data-uri="info:doi/10.1371/journal.pone.0000503.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000503.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000503-g001" data-doi="info:doi/10.1371/journal.pone.0000503" data-uri="info:doi/10.1371/journal.pone.0000503.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000503.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000503-g002" data-doi="info:doi/10.1371/journal.pone.0000503" data-uri="info:doi/10.1371/journal.pone.0000503.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000503.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000503-g003" data-doi="info:doi/10.1371/journal.pone.0000503" data-uri="info:doi/10.1371/journal.pone.0000503.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000503.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000503-t002" data-doi="info:doi/10.1371/journal.pone.0000503" data-uri="info:doi/10.1371/journal.pone.0000503.t002" title="Table 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000503.t002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000503-t003" data-doi="info:doi/10.1371/journal.pone.0000503" data-uri="info:doi/10.1371/journal.pone.0000503.t003" title="Table 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000503.t003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000503-t004" data-doi="info:doi/10.1371/journal.pone.0000503" data-uri="info:doi/10.1371/journal.pone.0000503.t004" title="Table 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000503.t004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000503-t005" data-doi="info:doi/10.1371/journal.pone.0000503" data-uri="info:doi/10.1371/journal.pone.0000503.t005" title="Table 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000503.t005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000503-g004" data-doi="info:doi/10.1371/journal.pone.0000503" data-uri="info:doi/10.1371/journal.pone.0000503.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000503.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000503-g005" data-doi="info:doi/10.1371/journal.pone.0000503" data-uri="info:doi/10.1371/journal.pone.0000503.g005" title="Figure 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000503.g005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000503">Reader Comments (1)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0000503" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1"></a><p>Comparative sequence analyses, including such fundamental bioinformatics techniques as similarity searching, sequence alignment and phylogenetic inference, have become a mainstay for researchers studying type 1 Human Immunodeficiency Virus (HIV-1) genome structure and evolution. Implicit in comparative analyses is an underlying model of evolution, and the chosen model can significantly affect the results. In general, evolutionary models describe the probabilities of replacing one amino acid character with another over a period of time. Most widely used evolutionary models for protein sequences have been derived from curated alignments of hundreds of proteins, usually based on mammalian genomes. It is unclear to what extent these empirical models are generalizable to a very different organism, such as HIV-1–the most extensively sequenced organism in existence. We developed a maximum likelihood model fitting procedure to a collection of HIV-1 alignments sampled from different viral genes, and inferred two empirical substitution models, suitable for describing between-and within-host evolution. Our procedure pools the information from multiple sequence alignments, and provided software implementation can be run efficiently in parallel on a computer cluster. We describe how the inferred substitution models can be used to generate scoring matrices suitable for alignment and similarity searches. Our models had a consistently superior fit relative to the best existing models and to parameter-rich data-driven models when benchmarked on independent HIV-1 alignments, demonstrating evolutionary biases in amino-acid substitution that are unique to HIV, and that are not captured by the existing models. The scoring matrices derived from the models showed a marked difference from common amino-acid scoring matrices. The use of an appropriate evolutionary model recovered a known viral transmission history, whereas a poorly chosen model introduced phylogenetic error. We argue that our model derivation procedure is immediately applicable to other organisms with extensive sequence data available, such as Hepatitis C and Influenza A viruses.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>Nickle DC, Heath L, Jensen MA, Gilbert PB, Mullins JI, et al.  (2007) HIV-Specific Probabilistic Models of Protein Evolution. PLoS ONE 2(6):
          e503.
            doi:10.1371/journal.pone.0000503</p><p><strong>Academic Editor: </strong>Oliver Pybus, University of Oxford, United Kingdom</p><p><strong>Received:</strong> February 28, 2007; <strong>Accepted:</strong> May 8, 2007; <strong>Published:</strong> June 6, 2007</p><p><strong>Copyright:</strong> © 2007 Nickle et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>This research was supported in part by the National Institutes of Health (AI43638, AI47745, and AI57167, R01-GM66276), the University of California University wide AIDS Research Program (IS 02-SD-701), and by a University of California, San Diego Center for AIDS Research/NIAID Developmental Award to SLKP (AI36214). Additional support came from the National Institutes of Health (5R01AI058894-03, 5P01AI057005-04) and from the University of Washington Center for AIDS Research and STDs (P30-AI-27757) and a gift from Boeing Corporation all awarded to JIM and NIH award 2 RO1 AI054165-04 awarded to PBG. The funders had no influence on the preparation, review, or approval of the manuscript.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>Nearly every computational and statistical method used for comparative gene sequence analysis employ a stochastic model for estimating rates of evolutionary change, either explicitly or implicitly. <em>A priori</em> knowledge about physical or chemical properties of nucleotide or amino acid residues can be used to define mechanistic models of substitutions. For example, the popular HKY85 <a href="#pone.0000503-Hasegawa1">[1]</a> nucleotide substitution model estimates two separate substitution rates: one for <em>transitions</em>-substitutions between chemically similar purines (adenine and cytosine) or pyrimidines (guanine and thymine)-and one for transversions (all other substitutions). Universal evolutionary constraints form another basis for mechanistic model derivation. Codon models of Muse-Gaut <a href="#pone.0000503-Muse1">[2]</a> and Goldman-Yang <a href="#pone.0000503-Goldman1">[3]</a> distinguished amino-acid altering (non-synonymous) and silent (synonymous) substitutions and have formed the basis of a popular and successful suite of methods for the analysis of selective pressures on coding sequences.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>Existing literature on probabilistic models for protein sequences is extensive and spans several decades. One of the first of such models was the PAM (point accepted mutation) matrix <a href="#pone.0000503-Dayhoff1">[4]</a>. A PAM matrix is derived from the inferred substitutions along a phylogenetic tree relating homologous sequences, by estimating the probability that any given amino acid residue in a protein will be replaced by any other residue after a pre-specified evolutionary interval. Other models based on observed sequence variability in large alignments of homologous protein sequences, such as the BLOSUM family <a href="#pone.0000503-Henikoff1">[5]</a>, have proven popular and successful. Karlin and Ghandour <a href="#pone.0000503-Karlin1">[6]</a> and George, Barker, and Hunt <a href="#pone.0000503-George1">[7]</a> proposed methods of weighting differences based on chemical, functional, charge and structural properties of amino acids and computing replacement probabilities based on the similarity of the involved residues. Doolittle's group proposed substitution matrices based on amino acid structural similarities combined with the ease of genetic interchange <a href="#pone.0000503-Feng1">[8]</a>, while Stanfel added information pertaining to biochemical properties to inform the probability of amino acid interchangeability <a href="#pone.0000503-Stanfel1">[9]</a>. More recently, a generalized index of exchangeability based on a meta-analysis of empirical data has been suggested as a means of estimating the tolerability of particular amino acid exchanges <a href="#pone.0000503-Yampolsky1">[10]</a>. A similar method based on pairwise amino-acid differences between homologous genes led to the derivation of the ambitiously named Universal Evolutionary Index <a href="#pone.0000503-Tang1">[11]</a>. A more statistically robust method for model inference incorporates phylogenetic likelihood <a href="#pone.0000503-Felsenstein1">[12]</a>, and infers substitution rates from a seed alignment, e.g., from mitochondrial sequences <a href="#pone.0000503-Yang1">[13]</a> or a sample from several protein families <a href="#pone.0000503-Whelan1">[14]</a>.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>‘Generalist’ models that describe substitution patterns amalgamated from multiple genes and organisms may describe a particular organism or gene poorly. To date, there have been only a few ‘specialist’ models targeted to a particular gene <a href="#pone.0000503-Dimmic1">[15]</a>, or genomic region <a href="#pone.0000503-Yang1">[13]</a>. In this manuscript we lay out a maximum likelihood framework and an accessible software implementation for estimating an organism/gene specific evolutionary model and alignment scoring matrix, describe common techniques for validating the model and infer a model from a large collection of HIV-1 sequences.</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p>Reliable estimation of substitution rates from short sequences (e.g. 10Kb viral genomes) requires a substantial degree of sequence diversity, which may require millions of years to accumulate in vertebrates or plants. However, rapidly evolving retroviruses, with mutation rates of up to ~10<sup>6</sup> greater than that of vertebrates <a href="#pone.0000503-Mansky1">[16]</a>, <a href="#pone.0000503-Nachman1">[17]</a>, accumulate similar levels of divergence in a matter of years and are abundantly represented in public databases. Dimmic et al. <a href="#pone.0000503-Dimmic1">[15]</a> used a maximum likelihood procedure to estimate an amino acid substitution rate matrix for specific application to reverse transcriptase, a key retroviral polymerase protein that transcribes viral RNA into DNA suitable for integration into the host genome. However, we found that this model fitted HIV-1 data poorly, probably because HIV adopts organism-specific substitution biases, different from other retroviruses.</p>
<a id="article1.body1.sec1.p5" name="article1.body1.sec1.p5"></a><p>To improve the predictive accuracy of substitution matrices for HIV protein evolution, we estimated two stochastic models from multiple representative HIV-1 sequence alignments using maximum likelihood. The first model was derived from HIV data sampled from within individual patients (within individual, HIV within, HIV-W<sub>m</sub>). The second model was estimated from alignments where every sequence represented a population consensus from a patient (between individual, HIV between, HIV-B<sub>m</sub>). At first glance, one might question the need for two separate models, since within-patient evolution could simply be a shorter timescale version of the between-patient evolution. One argument against this intuitive deduction is that most of the substitutions generated in a given individual are selected against during or following transmission and therefore do not persist at the level of host populations <a href="#pone.0000503-Herbeck1">[18]</a>–<a href="#pone.0000503-Liu1">[20]</a>, resulting in potentially discordant substitution patterns. For example, substitutions which enable the virus to escape the cellular immune response in a given host can be rapidly generated and fixed <a href="#pone.0000503-Allen1">[21]</a>. However, many of these substitutions carry a fitness cost in terms of lower replicative capacity and are not likely to persist upon transmission to an individual whose immune system does not target the same genetic region of the virus, obviating the need for a fitness-lowering substitution there <a href="#pone.0000503-Friedrich1">[22]</a>. Indeed, if there were no added benefit in considering two models, one would see similar fits to both within and between-host viral samples with both models. Our findings strongly argue against this scenario (see ‘Results’), showing that substitution patterns shaped by within-and between-host selective regimes are detectably different.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Methods"></a><h3>Methods</h3>
<h4>Preparation of reference sequence alignments</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>The HIV-W<sub>m</sub> model was generated using aligned sequences derived from 48 patients (encompassing portions of the <em>env</em> gene from 32 patients, from the <em>pro</em> gene from 8, and from the <em>pol</em> gene from 7 patients), which contained a total of 6,328 pairwise amino acid differences. For the HIV-B<sub>m</sub> model we used 8 data sets, described in <a href="#pone-0000503-t001">Table 1</a>, with a total of 7,189 amino acid differences. This number is far greater than the approximately 1,700 differences that were incorporated into the commonly used Dayhoff PAM matrix and nearly twice the number in the reference alignment used for the estimation of rtREV <a href="#pone.0000503-Dimmic1">[15]</a></p>
<div class="figure" id="pone-0000503-t001"><div class="img"><a name="pone-0000503-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000503.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000503" data-uri="info:doi/10.1371/journal.pone.0000503.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000503.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000503.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000503.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000503.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000503.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000503.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000503.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000503.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000503.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>Data sets used for the estimation of the between-host evolutionary process in HIV-1</span></strong></p><span>doi:10.1371/journal.pone.0000503.t001</span></div>

<h4>Estimation of the substitution model</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>We adopted a maximum likelihood phylogenetic approach based on the procedure of Whelan and Goldman <a href="#pone.0000503-Whelan1">[14]</a> to estimate the 190 evolutionary rates that define the general time reversible (REV/GTR) model of amino-acid substitution jointly from a set of sequence alignments. The substitution process is described by the rate matrix <strong>Q</strong>, whose entry <em>q<sub>ij</sub></em> = <em>r<sub>ij</sub>π<sub>j</sub></em> (for <em>i</em>,<em>j</em> = 1…20) defines the instantaneous rate of replacing residue <em>i</em> with residue <em>j</em>, i.e. the probability of substituting <em>i</em> with <em>j</em> over an infinitesimally small time interval Δ<em>t</em> is approximately <em>q<sub>ij</sub></em>Δ<em>t</em>. Residues are numbered based on the alphabetical ordering of the standard IUPAC nomenclature. <em>π<sub>j</sub></em>(<em>j</em> = 1…20) denotes the stationary frequency of residue <em>j</em>, estimated by the proportion of <em>j</em>, and <em>r<sub>ij</sub></em> is the rate parameter. Making the standard assumption of time reversibility, we set <em>r<sub>ij</sub></em> = <em>r<sub>ji</sub></em>. In order to ensure that <strong>Q</strong> defines a proper Markov rate matrix, the standard constraint is applied to the diagonal elements: <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000503.e001&amp;representation=PNG" class="inline-graphic"></span>.</p>
<a id="article1.body1.sec2.sec2.p2" name="article1.body1.sec2.sec2.p2"></a><p>The (<em>i,j</em>) entry of the matrix exponential <em>T</em>(<em>t</em>) = exp(<strong>Q</strong><em>t</em>) defines the probability of replacing residue <em>i</em> with residue <em>j</em> in time <em>t</em>≥0. Because the likelihood function depends only on the products <em>q<sub>ij</sub>t</em>, one of the 190 rate parameters <em>r<sub>ij</sub></em> is not identifiable. Following convention, we achieve identifiably by scaling the rate matrix so that the expected number of substitutions per amino-acid site per unit time, defined as <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000503.e002&amp;representation=PNG" class="inline-graphic"></span>, is one.</p>
<a id="article1.body1.sec2.sec2.p3" name="article1.body1.sec2.sec2.p3"></a><p>The fitting algorithm proceeds as follows:</p>


<ol class="order">

<li>Given <em>N</em> codon alignments, we first reconstructed a neighbor joining tree <a href="#pone.0000503-Saitou1">[23]</a> using the Tamura-Nei nucleotide distance metric <a href="#pone.0000503-Tamura1">[24]</a> separately for each alignment. We decided against a more thorough method for topology reconstruction, partly for computational expediency and partly because rate estimation is thought to be fairly robust to small phylogenetic error. In addition, HIV-1 sequences often undergo recombination, and alignments with mosaic sequences cannot be properly described with a single phylogeny. To address this issue we carried out a screen for evidence of conflicting phylogenetic signal <a href="#pone.0000503-KosakovskyPond2">[25]</a> and did not find strong evidence for phylogenetic discordance.</li>

<li>We estimated the equilibrium distribution of amino acid residues jointly from the N alignments and held it constant during the subsequent optimization procedure.</li>

<li>We used the HyPhy package <a href="#pone.0000503-Pond1">[26]</a> to perform a joint numerical optimization of branch lengths for each of the N alignments and the rates in the <strong>Q</strong> matrix. A flexible 5-bin <em>β</em>−Γ distribution <a href="#pone.0000503-KosakovskyPond3">[27]</a> of rates across sites was included to account for the variation of substitution rates across sites. The three parameters of the distribution were shared by all N alignments. The numerical optimization algorithm in HyPhy can distribute multiple-dataset optimization across multiple nodes of a computer cluster, resulting in substantial optimization time reductions.</li>

<li>Because some of the 190 possible substitutions are rare, it is possible that the inference procedure estimates some of the <em>r<sub>ij</sub></em> rates to be zero. However, fixing those estimates at zero for subsequent analyses is not biologically realistic, since it amounts to forcing some of the residue substitutions to always go through an intermediate state. For each rate <em>r<sub>ij</sub></em>, whose maximum likelihood estimate was zero, we imputed the value of the rate using the following heuristic. If <em>S<sub>i</sub></em> is the total number of protein sites in alignment <em>i</em> and <em>T<sub>i</sub></em> is the total length of its fitted phylogenetic tree, we set <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000503.e003&amp;representation=PNG" class="inline-graphic"></span>. This relation simply states that we would expect one <em>i</em>↔<em>j</em> substitution over all alignments given one more site per alignment.</li>

</ol>

<h4>Performance on training data</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>We fitted newly estimated HIV models and three previously published models-JTT <a href="#pone.0000503-Jones1">[28]</a>, WAG <a href="#pone.0000503-Whelan1">[14]</a> and rtREV <a href="#pone.0000503-Dimmic1">[15]</a>, estimating stationary frequencies from the data–to each of the training alignments. A comparison of model fits via standard techniques of likelihood ratio testing or AIC is complicated by the fact that because HIV models are estimated from the same sample that is being used to compare their fits to other models, it is difficult to correctly enumerate the effective number of rate parameters (and degrees of freedom) in HIV models when applied to a single training set. For a given training set, this number lies between 0 and 189 and depends on the contribution of the individual set to the joint rate matrix. We tabulated log-likelihood scores for each of the training data sets for reference purposes (<a href="#pone.0000503.s001">Tables S1</a> and <a href="#pone.0000503.s002">S2</a>) and computed the number of additional degrees of freedom that an HIV model can support (at the 0.05 level) and still be preferred to each of the empirical models.</p>


<h4>Model validation</h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>To ensure that our substitution models reflect true evolutionary patterns found in HIV-1 and are not fitting the noise in our training sets, we collected a number of independent HIV-1 sequence samples that were not used in model development across several genes, representing both within and between host sequences. 47 within patient samples (35 envelope glycoprotein (gp120) subtype B, 10 gp120 subtype C and 3 polymerase (<em>pol</em>) subtype B) and 11 between patient samples (representing <em>gag, pol, nef, env, vif, vpr, tat</em> and <em>rev</em> genes), covering both within- and between-subtype levels of diversity were collected. We note that if validation and training samples are not reciprocally monophyletic, shared evolution along internal tree branches may bias validation results. Within patient samples of HIV (assuming a single infection event/patient) drawn from different hosts are reciprocally monophyletic by definition, hence each alignment in the validation set forms an independent sample. All of our between-patient validation samples represent only non-B subtype non-recombinant viruses, and hence are reciprocally monophyletic with the training samples by definition of an HIV clade. We then fitted 19 models of protein evolution, including HIV-specific models, to each sample and ranked their performance using a small sample Akaike Information Criterion (c-AIC) score <a href="#pone.0000503-Sugiura1">[29]</a>–a robust measure of goodness of fit. We included six previously published empirical matrices: Dayhoff <a href="#pone.0000503-Dayhoff1">[4]</a>, JTT <a href="#pone.0000503-Jones1">[28]</a>, WAG <a href="#pone.0000503-Whelan1">[14]</a>, rtREV <a href="#pone.0000503-Dimmic1">[15]</a>, mtMAM <a href="#pone.0000503-Yang1">[13]</a> and mtREV 24 <a href="#pone.0000503-Adachi1">[30]</a>. Each empirical model was examined both with the original model character frequency distribution (derived from the training set) and with frequencies gathered from the test data (conventionally referred to as the+F version of the model). We also fitted to each sample the 189 parameter reversible model (REV), and the restriction thereof (REV-1 model), which estimates separate rates between those amino-acids pairs that are one nucleotide substitution apart (75 pairs for the universal genetic code) and one shared rate for all multiple-nucleotide substitutions.</p>
<a id="article1.body1.sec2.sec4.p2" name="article1.body1.sec2.sec4.p2"></a><p>Additionally, we applied the Shimodaira-Hasegawa test to several example alignments to determine whether or not the improvement in goodness-of-fit was influenced by sampling variability. For example, it could be that most of the improvement in c-AIC scores is derived from a few ‘outlier’ sites (scenario sensitive to stochastic sampling) or that a majority of sites contribute to the improvement in c-AIC (scenario robust to stochastic sampling).</p>


<h4>Generation of a similarity matrix</h4>
<a id="article1.body1.sec2.sec5.p1" name="article1.body1.sec2.sec5.p1"></a><p>Similarity or scoring matrices, such as the BLOSUM <a href="#pone.0000503-Henikoff1">[5]</a> family, assign to a pair of amino acids (a, b) a score that reflects how much more (or less) likely a homologous pair (a, b) is to occur when compared to a chance occurrence. The score can be estimated by <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000503.e004&amp;representation=PNG" class="inline-graphic"></span>, where <em>f</em>(<em>a</em>) is the frequency of residue <em>a</em> in the reference set of aligned sequences, and <em>f</em>(<em>ab</em>) is the frequency of the pair computed from pairwise comparisons of homologous sequences in the reference set. The sum of all scores over alignment columns for a query pair of sequences serves as an approximation of the likelihood ratio statistic comparing the model of independence (null) with the model of homology (alternative).</p>
<a id="article1.body1.sec2.sec5.p2" name="article1.body1.sec2.sec5.p2"></a><p>Given a frequency distribution of amino-acid characters <em>π</em> and a transition matrix computed at time <strong>T</strong>(<em>t</em><sub>0</sub>) = exp(<strong>Q</strong><em>t</em><sub>0</sub>), the score <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000503.e005&amp;representation=PNG" class="inline-graphic"></span> can be derived. The numerator lists the probability of evolving ‘b’ from ‘a’ or ‘a’ from ‘b’ in time <em>t</em><sub>0</sub>, i.e., the probability of observing the (a,b) or (b,a) pair in two homologous sequences evolving under <strong>Q</strong>, while the denominator shows the probability of observing (a,b) or (b,a) in a pair of random sequences with residues drawn from the distribution <em>π</em>. The choice of <em>t</em><sub>0</sub> gives one control over how similar, on average, two sequences will be, much like the selection of more or less similar reference alignments gives rise to different BLOSUM and PAM matrices. Setting <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000503.e006&amp;representation=PNG" class="inline-graphic"></span> will yield the expected sequence similarity of (1−<em>c</em>×100)%. Scores for aligning a character with a gap (‘-‘) can be adjusted case by case, or adopted from an existing model, such as BLOSUM62.</p>


<h4>Implementation</h4>
<a id="article1.body1.sec2.sec6.p1" name="article1.body1.sec2.sec6.p1"></a><p>All the analyses reported here were implemented in the HyPhy software package. HyPhy scripts needed to fit a REV model to a collection of training alignments and generate similarity scoring matrices can be downloaded from <a href="http://www.hyphy.org/pubs/HIV-model/">http://www.hyphy.org/pubs/HIV-model/</a>. HIV-1 models presented here are available as a part of the standard distribution of HyPhy (HIVWithin and HIVBetween models). Protein model comparisons can be carried out using the <em>AAModelComparison.bf</em> standard analysis in HyPhy. This implementation can also take advantage of a distributed cluster environment to accelerate rate estimation.</p>

</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Results"></a><h3>Results</h3>
<h4>Characterization of inferred rate matrices</h4>
<a id="article1.body1.sec3.sec1.p1" name="article1.body1.sec3.sec1.p1"></a><p>We first tabulated substitution rates inferred from the between-patient training set (<strong>HIV-B<sub>m</sub></strong> model) and from the within-patient training set (<strong>HIV-W<sub>m</sub></strong> model) and generated a graphical representation of six popular empirical matrices along with our two HIV specific matrices (<a href="#pone-0000503-g001">Figure 1</a>). Not surprisingly, most instantaneous substitution rates are low when compared to the highest rates in the matrix. Much like other empirical matrices, the HIV models assigned higher rates to those pairs that are separated by a single nucleotide substitution (see also <a href="#pone-0000503-g002">Figure 2</a>). However, there was little apparent correlation between higher substitution rates and preservation of a simple or complex physico-chemical property. Preservation or alteration of polarity, charge or similarity class, based on the classification scheme of Stanfel <a href="#pone.0000503-Stanfel1">[9]</a>, had little effect on the median substitution rate (<a href="#pone-0000503-g002">Figure 2</a>), although among those few substitutions which had unusually high rates, more were conservative.</p>
<div class="figure" id="pone-0000503-g001"><div class="img"><a name="pone-0000503-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000503.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000503" data-uri="info:doi/10.1371/journal.pone.0000503.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000503.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000503.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000503.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000503.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000503.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000503.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000503.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000503.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000503.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Rate matrices for different substitution models.</span></strong></p><a id="article1.body1.sec3.sec1.fig1.caption1.p1" name="article1.body1.sec3.sec1.fig1.caption1.p1"></a><p>All matrices are scaled to one expected substitution per unit time per site. Shading of the cells reflects the respective magnitude of the rate, with darker shades corresponding to increasingly higher rates. Substitutions which involve a single nucleotide are marked with a circle. The four diagonal blocks represent similarity classes (conservative substitutions) according to the Stanfel scale.</p>
<span>doi:10.1371/journal.pone.0000503.g001</span></div><div class="figure" id="pone-0000503-g002"><div class="img"><a name="pone-0000503-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000503.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000503" data-uri="info:doi/10.1371/journal.pone.0000503.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000503.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000503.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000503.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000503.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000503.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000503.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000503.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000503.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000503.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Inferred substitution rates.</span></strong></p><a id="article1.body1.sec3.sec1.fig2.caption1.p1" name="article1.body1.sec3.sec1.fig2.caption1.p1"></a><p>Rates are classified by whether or not a substitution involves single or multiple nucleotide changes, and by how they affect various properties of the residue being substituted. HIV-B<sub>m</sub> model is plotted in the top row and HIV-W<sub>m</sub> model-in the bottom row.</p>
<span>doi:10.1371/journal.pone.0000503.g002</span></div><a id="article1.body1.sec3.sec1.p2" name="article1.body1.sec3.sec1.p2"></a><p>Model-specific variability along with broad patterns of similarity are evident among all of the empirical models in <a href="#pone-0000503-g001">Figure 1</a>. To formally characterize the similarities in the substitution process across the eight matrices, we computed a neighbor-joining tree on the Markov processes defined by each matrix using the total variation metric (TVM) <a href="#pone.0000503-KosakovskyPond4">[31]</a>. Briefly, given a specific evolutionary time scale, TVM computes the distance between the expected distributions of characters generated under the two evolutionary models. TVM distances take values in [0,1]. As expected, <strong>HIV-B<sub>m</sub></strong> and <strong>HIV-W<sub>m</sub></strong> models are most similar to each other over short (0.05 expected substitutions per site), medium (0.25) and long (1.0) evolutionary scales (<a href="#pone-0000503-g003">Figure 3</a>). In particular, HIV matrices define substitution patterns that are distinct from all other empirical models at several evolutionary time-scales. Somewhat surprisingly, the next closest set of matrices is that derived from mitochondrial sequences, except for the long evolutionary scale, when the rtREV model (derived from a viral reverse transcriptase protein alignment) becomes most similar to the HIV models. While counterintuitive, this finding seems less unexpected when one considers how the training sets were chosen for each of the models. Indeed, the Dayhoff, JTT, WAG and rtREV models were all trained on sequences that are much more heterogeneous, gearing them towards long-range evolutionary homologies.</p>
<div class="figure" id="pone-0000503-g003"><div class="img"><a name="pone-0000503-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000503.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000503" data-uri="info:doi/10.1371/journal.pone.0000503.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000503.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000503.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000503.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000503.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000503.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000503.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000503.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000503.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000503.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Model clustering using the Total Variation Metric at the evolutionary times equivalent to 5%, 25% and 100% sequence divergences.</span></strong></p><span>doi:10.1371/journal.pone.0000503.g003</span></div>

<h4>Model validation using independent data sets</h4>
<a id="article1.body1.sec3.sec2.p1" name="article1.body1.sec3.sec2.p1"></a><p>To determine whether HIV Within and HIV Between models reflect evolutionary patterns found in circulating HIV-1 strains, rather than simply fitting the noise in the training sample, we tested model fits on samples of HIV-1 sequences not included in the training set. The results on samples collected from 47 different patients (the sequences from each patient formed a distinct test case) with a median of 32 sequences (range 10–57) and 357 sites (range 330–473), representing <em>env</em> and <em>pol</em> sequences from subtype B and subtype C viruses are shown in <a href="#pone-0000503-t002">Tables 2</a> and <a href="#pone-0000503-t003">3</a>. In 44/47 cases, the <strong>HIV-W<sub>m</sub></strong> model with base frequencies estimated from the sample had the best small sample Akaike Information Criterion score (c-AIC) score of all 19 models compared. The c-AIC score of a model is defined as c-AIC = 2(−<em>L</em>+<em>ps</em>/(<em>s</em>−<em>p</em>−1)), where L is the log-likelihood score of the model, p is the number of estimated model parameters, and s is the number of independent samples. There are a number of possible ways to estimate the number of independent samples in the alignment <a href="#pone.0000503-Posada1">[32]</a> and we chose to use the number of alignment columns as an estimate of s. c-AIC performed well in selecting appropriate evolutionary models on biological and simulated alignments of paired RNA sequences <a href="#pone.0000503-KosakovskyPond5">[33]</a>. In 2/47 cases, the <strong>HIV-W<sub>m</sub></strong> model with the frequencies from the training set was the best and in 1/47 cases, the <strong>HIV-B<sub>m</sub></strong> model was the best. In fact, the four HIV models predominantly occupied the four top ranks, suggesting that no other empirical model adequately represents the evolutionary rates shaping HIV-1 genomic variation. The REV-1 and JTT (+F) models were 5<sup>th</sup> and 6<sup>th</sup>, respectively. Surprisingly, three models based on large heterogeneous database samples (JTT, WAG and mtMAM) outperformed the rtREV model, which was derived from a viral training dataset. Perhaps more importantly, our general HIV model estimated across many of the HIV genes consistently outperformed the rtREV model on sequence samples, including the HIV-1 reverse transcriptase protein.</p>
<div class="figure" id="pone-0000503-t002"><div class="img"><a name="pone-0000503-t002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000503.t002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000503" data-uri="info:doi/10.1371/journal.pone.0000503.t002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000503.t002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000503.t002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000503.t002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000503.t002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000503.t002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000503.t002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000503.t002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000503.t002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000503.t002.TIF"></span>)
                </a></li></ul></div><p><strong>Table 2.  <span>Relative performance of 19 protein models on a sample of 47 within-patient HIV-1 alignments<span class="xref"><sup>1</sup></span></span></strong></p><span>doi:10.1371/journal.pone.0000503.t002</span></div><div class="figure" id="pone-0000503-t003"><div class="img"><a name="pone-0000503-t003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000503.t003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000503" data-uri="info:doi/10.1371/journal.pone.0000503.t003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000503.t003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000503.t003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000503.t003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000503.t003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000503.t003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000503.t003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000503.t003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000503.t003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000503.t003.TIF"></span>)
                </a></li></ul></div><p><strong>Table 3.  <span>The number of times a model had evidence ratio of 100 or better against every competing model on the 47 within-patient validation data sets.<span class="xref"><sup>1</sup></span></span></strong></p><span>doi:10.1371/journal.pone.0000503.t003</span></div><a id="article1.body1.sec3.sec2.p2" name="article1.body1.sec3.sec2.p2"></a><p>The difference in c-AIC scores between two models can be interpreted as strength of evidence in favor of the model with the lower score. For instance, the evidence ratio for models A and B, R(A,B) is defined as <em>R</em>(<em>A,B</em>) = exp[(<em>AIC<sub>C</sub></em>(<em>B</em>)−<em>AIC<sub>C</sub></em>(<em>A</em>))/2], and can be interpreted as the relative probabilities of the two models generating the data in certain cases <a href="#pone.0000503-Burnham1">[34]</a>. In all 47 cases the difference in c-AIC between <strong>HIV-W<sub>m</sub></strong> and the best-fitting previously described empirical model was sufficiently large (median 125 points, range 13.3–259.5) to render the latter model not credible, with R(<strong>HIV-W<sub>m</sub></strong>,best existing empirical model) &gt;50 for every tested dataset. Furthermore, in 46/47 cases when <strong>HIV-W<sub>m</sub></strong>(+F) outperformed <strong>HIV-B<sub>m</sub></strong>(+F), R(<strong>HIV-W<sub>m</sub></strong>, <strong>HIV-B<sub>m</sub></strong>) was at most 0.011 (c-AIC difference range 9.0–215.3, median 94.4), confirming that a within-patient model for HIV is sufficiently different from the between-host model, supporting our argument for the need to derive these two distinct models. <a href="#pone-0000503-t003">Table 3</a> provides a pairwise comparison of model performance by listing the number of times model A had evidence ratio in excess of 100 when compared to model B. For instance, <strong>HIV-W<sub>m</sub></strong> had very strong evidence ratio support against all competing models in the vast majority of cases.</p>
<a id="article1.body1.sec3.sec2.p3" name="article1.body1.sec3.sec2.p3"></a><p>The results for 11 between-patient samples (where each sample contained a single sequence from each host), with a median of 37 sequences (range 22–119) and 442 sites (range 79–953) are shown in <a href="#pone-0000503-t004">Tables 4</a> and <a href="#pone-0000503-t005">5</a>. In 7/11 cases <strong>HIV-B<sub>m</sub></strong> was the best fitting model (in all 11 cases it scored in the top 3), REV-1 was best in 4/11 cases. JTT consistently scored best among the existing empirical models, but always fit worse than <strong>HIV-B<sub>m</sub></strong> (e.g., when it came in second, <strong>HIV-B<sub>m</sub></strong> was first, see <a href="#pone-0000503-t005">Table 5</a>). In all 11 cases the difference in c-AIC between <strong>HIV-B<sub>m</sub></strong> and the best-fitting previously described empirical model was sufficiently large (median 38.42 points, range 6.5–345.5) to suggest that previous empirical models fitted the data poorly, with R(<strong>HIV-B<sub>m</sub></strong>,best existing empirical model) &gt;25 for every tested dataset. In addition, <strong>HIV-B<sub>m</sub></strong>, consistently outperformed <strong>HIV-W<sub>m</sub></strong>, corroborating our original supposition for the need for two distinct models. The finding that the estimation of 76 rate parameters from the data was worthwhile in 4/11 cases is not surprising in retrospect. Some of the datasets we evaluated comprised multiple HIV subtypes with more variation and divergence than had been included in our subtype B-only training data. In fact, all 4 cases where REV-1 was found to be best, the sequences came from the Los Alamos National Laboratory HIV database subtype reference alignment spanning all common HIV-1 (clade M) subtypes. Furthermore, <strong>HIV-B<sub>m</sub></strong> consistently outperformed all other empirical models.</p>
<div class="figure" id="pone-0000503-t004"><div class="img"><a name="pone-0000503-t004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000503.t004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000503" data-uri="info:doi/10.1371/journal.pone.0000503.t004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000503.t004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000503.t004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000503.t004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000503.t004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000503.t004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000503.t004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000503.t004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000503.t004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000503.t004.TIF"></span>)
                </a></li></ul></div><p><strong>Table 4.  <span>Relative performance of 19 protein models on a sample of 11 between-patient HIV-1 alignments<span class="xref"><sup>1</sup></span></span></strong></p><span>doi:10.1371/journal.pone.0000503.t004</span></div><div class="figure" id="pone-0000503-t005"><div class="img"><a name="pone-0000503-t005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000503.t005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000503" data-uri="info:doi/10.1371/journal.pone.0000503.t005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000503.t005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000503.t005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000503.t005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000503.t005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000503.t005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000503.t005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000503.t005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000503.t005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000503.t005.TIF"></span>)
                </a></li></ul></div><p><strong>Table 5.  <span>The number of times a model had evidence ratio of 100 or better against every competing model on the 11 between-patient validation data sets.<span class="xref"><sup>1</sup></span></span></strong></p><span>doi:10.1371/journal.pone.0000503.t005</span></div>

<h4>Sensitivity to stochastic sampling</h4>
<a id="article1.body1.sec3.sec3.p1" name="article1.body1.sec3.sec3.p1"></a><p>We focus on a sample data set to investigate whether or not the improvement in model fit attained by HIV-specific models is driven by a few strong improvements at ‘outlier’ sites, or by consistent but smaller improvements at many sites in an alignment. A formal way to quantify the effect of stochastic sampling is to perform a Shimodaira-Hasegawa permutation test on columns of an alignment, compute the difference in log-likelihood scores of two competing models on each replicate (if they have the same number of parameters), and test the hypothesis that the difference is different from zero. We illustrate this on a sample of 119 <em>pol</em> sequences each sampled from a different host infected with subtype C HIV-1, with 951 sites each. The likelihood scores for three models on this alignment are as follows: <strong>HIV-B<sub>m</sub></strong> log L = −21653.1, JTT log L = −21825.9, rtREV logL = −22383.4. Because the models have the same number of parameters, a better log likelihood leads to a better c-AIC score. When comparing <strong>HIV-B<sub>m</sub></strong> to rtREV, we found that <strong>HIV-B<sub>m</sub></strong> had higher likelihoods at 580/951 (61%) sites, with the median improvement of 0.99 (range of 0.002 to 37.9) log-likelihood points per site, whereas at the sites where rtREV performed better, the median difference was 0.7 (range 0–31.6). Based on 10,000 permutations, the SH test returned a p-value of &lt;0.0001 in favor of <strong>HIV-B<sub>m</sub></strong> having a better fit. In a comparison <strong>HIV-B<sub>m</sub></strong> to JTT, the p-value in favor of <strong>HIV-B<sub>m</sub></strong> was &lt;0.018, <strong>HIV-B<sub>m</sub></strong> fitted better on 557/951 (58.6%) sites, with the median improvement of 0.54 (0.0009–23.67) on those sites, and of 0.53 (0.00001–22.75) on those where JTT performed better. The largest improvement in favor of <strong>HIV-B<sub>m</sub></strong> occurred at sites where there was significant polymorphism with 2 or 3 amino-acids estimated to have high substitution rates in HIV-1. For example, site 311 with base composition K<sub>75</sub>R<sub>41</sub>Q<sub>2</sub>G<sub>1 </sub>yielded improvements of over 20 log-likelihood points with HIV Between, when compared to JTT or rtREV. Note that <strong>HIV-B<sub>m</sub></strong> assigned higher rates to K↔R than either JTT or rtREV, accommodating such variability (<a href="#pone-0000503-g001">Figure 1</a>).</p>


<h4>Effect of model misspecification on phylogenetic reconstruction</h4>
<a id="article1.body1.sec3.sec4.p1" name="article1.body1.sec3.sec4.p1"></a><p>We consider seven HIV-1 <em>env</em> V3 loop sequences sampled from an epidemiological cluster with known transmission history and hence a known phylogenetic tree, used as a study case by <a href="#pone.0000503-Leitner1">[35]</a>. With seven sequences it is easy to perform an exhaustive search of all 945 unrooted trees and eliminate the influence of search heuristics. Thus any differences observed would be driven by model specification. The correct topology for the seven sequences is inferred by the <strong>HIV-B<sub>m</sub></strong> model, but not by the Dayhoff model (<a href="#pone-0000503-g004">Figure 4</a>) for example. We note that better-fitting empirical models (e.g. JTT or WAG) recover the correct topology, and we use this example merely to illustrate that a poor or unjustified model choice may lead to erroneous results.</p>
<div class="figure" id="pone-0000503-g004"><div class="img"><a name="pone-0000503-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000503.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000503" data-uri="info:doi/10.1371/journal.pone.0000503.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000503.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000503.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000503.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000503.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000503.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000503.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000503.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000503.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000503.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>Maximum likelihood trees inferred with two different amino acid models from a sample of HIV-1 env V3 sequences with a known transmission history (Leitner et al. 1996).</span></strong></p><a id="article1.body1.sec3.sec4.fig1.caption1.p1" name="article1.body1.sec3.sec4.fig1.caption1.p1"></a><p>HIV-B<sub>m</sub> found the tree that is congruent with the true history of the sequences. Scale bars are in expected amino acid substitutions/site.</p>
<span>doi:10.1371/journal.pone.0000503.g004</span></div>

<h4>Effect of model on evolutionary distance estimation</h4>
<a id="article1.body1.sec3.sec5.p1" name="article1.body1.sec3.sec5.p1"></a><p>Model-based estimation of sequence divergence and diversity from sequence samples is a ubiquitous technique in HIV literature <a href="#pone.0000503-Shankarappa1">[36]</a>. To examine the effect that an evolutionary model can have on the estimates of evolutionary distances between protein sequences we fitted four empirical models to 11 between-patient HIV alignments (<a href="#pone.0000503.s003">Table S3</a>). While median pairwise estimates (i.e. the total tree length) was not dramatically affected by the model choice (up to 5% relative error), for a particular pair of sequences, the estimates could vary by as much as ±30%.</p>


<h4>HIV similarity matrices</h4>
<a id="article1.body1.sec3.sec6.p1" name="article1.body1.sec3.sec6.p1"></a><p>We compared our scoring matrices with the typically used BLOSUM62 scoring matrix with ours scaled to the same level of divergence (62% expected sequence identity). HIV matrices penalize (by assigning large magnitude negative scores) most kinds of substitutions more heavily than the BLOSUM62, perhaps representing the fact that long stretches of HIV-1 genomes are quite conserved (e.g. integrase). <a href="#pone-0000503-g005">Figure 5</a> demonstrates the point. The BLOSUM62 matrix is relatively flat with many non-identities penalized relatively lightly and diagonal elements (identities) rewarded moderately. HIV matrices reward identities with higher scores than BLOSUM62 and tend to heavily penalize most non-identities.</p>
<div class="figure" id="pone-0000503-g005"><div class="img"><a name="pone-0000503-g005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000503.g005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000503" data-uri="info:doi/10.1371/journal.pone.0000503.g005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000503.g005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000503.g005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000503.g005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000503.g005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000503.g005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000503.g005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000503.g005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000503.g005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000503.g005.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 5.  <span>A comparison of BLOSUM and HIV-1 similarity scoring matrices with expected 62% sequence identity.</span></strong></p><a id="article1.body1.sec3.sec6.fig1.caption1.p1" name="article1.body1.sec3.sec6.fig1.caption1.p1"></a><p>Green = positive score; red = negative score; brightness = magnitude of score.</p>
<span>doi:10.1371/journal.pone.0000503.g005</span></div>
</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec4.p1" name="article1.body1.sec4.p1"></a><p>We constructed two empirical HIV-1 subtype B amino acid substitution models to better describe patterns of HIV evolution, one based on among-patient sequences (HIV-B<sub>m</sub>), and the other based on within-patient sequences (HIV-W<sub>m</sub>). Our implementation is straightforward and relies on well-regarded methods of maximum likelihood phylogenetic inference. Indeed, a researcher can build an empirical amino acid model for any taxon or clade of choice assuming there is sufficient sequence data to estimate most of the rates reliably. In a direct comparison between the two HIV models, we find them to be similar, yet in our cross validation study we find that the HIV-W<sub>m</sub> fits significantly better to within-patient data than the HIV-B<sub>m</sub> models and vice versa. But most importantly, our HIV specific models fit dramatically better than any commonly used amino acid models, and nearly always better than parameter rich reversible models which estimate rates directly from the data set being tested. This finding suggests that empirical HIV models are generalizable to independent samples of HIV and can be recommended as the default matrices for comparative HIV analyses. Interestingly, the JTT model provided the best fit to our samples of HIV data among existing empirical models, which is in direct conflict with the observations made by Dimmic et al. <a href="#pone.0000503-Dimmic1">[15]</a>.</p>
<a id="article1.body1.sec4.p2" name="article1.body1.sec4.p2"></a><p>A possible component of poor fit shown by existing empirical matrices on HIV sequences is the assumption that the sequences being compared have the same average amino acid composition as that used in producing the model. Because HIV sequences almost certainly do not have the same amino acid frequencies as those built into prior empirical matrices, the inclusion of HIV specific residue frequencies can be expected to improve the models. However, this is not the sole determinant of better model fits, because allowing for HIV-specific residue composition in existing empirical models (+F) failed to match the improvement in fit garnered by the use of HIV-specific substitution rates.</p>
<a id="article1.body1.sec4.p3" name="article1.body1.sec4.p3"></a><p>The lack of strong correlation between substitution rate and simple amino acid properties may be explained by the fact that because these models average rates over sites in different genes, which are subject to varying functional constraints, no single property can be expected to explain tremendous local variation well. This is true of other empirical models as well, as evidenced by the large number of shaded (high rate) pairs outside diagonal boxes, which reflect radical substitutions according to the Stanfel scale (<a href="#pone-0000503-g001">Figure 1</a>).</p>
<a id="article1.body1.sec4.p4" name="article1.body1.sec4.p4"></a><p>HIV specific models will improve the accuracy in measurements of evolutionary distance and phylogenetic inference on HIV sequences. For example, our models identify the correct phylogeny in a known transmission chain <a href="#pone.0000503-Leitner1">[35]</a>, whereas a poorly chosen model does not. The HIV-specific protein models of evolution could be used to tailor drug therapies against strains of viruses that maximize the protein distance that the virus would have to evolve to develop drug resistance. The use of more accurate evolutionary models can be used to improve the design of candidate vaccine strains, particularly those based on computationally derived sequences (e.g. Center-Of-Tree (COT) <a href="#pone.0000503-Nickle1">[37]</a>). The models also have applications for comparing amino acid frequencies and patterns between sequence sets, and for generating HIV-like sequence data sets in computer simulation studies. In this manuscript we focused on organism-wide substitution patterns and our validation process convincingly showed that HIV models outperformed existing empirical models on a variety of individual gene alignments. Further possible refinements could include gene specific matrices for commonly sequenced viral genes such as <em>pol</em> or <em>env</em>.</p>
<a id="article1.body1.sec4.p5" name="article1.body1.sec4.p5"></a><p>The HIV-specific scoring matrices provided here should improve HIV sequence similarity searching (e.g. BLAST) and alignment (e.g. CLUSTALW). All sequence analyses that produce an alignment use a scoring matrix (e.g. BLOSUM) to weight the probability of an observed substitution. Sometimes these matrices are uninformative and treat all substitutions equally likely, but nearly all protein sequence based procedures use a matrix that ranks certain types of substitutions as more tolerable than others. We developed an efficient computational tool to facilitate the estimation of substitution matrices from training alignments and subsequent generation of scoring matrices at the desired level of evolutionary distance. We believe that with readily accessible modern computing power, it is now feasible to estimate and use organism specific empirical matrices for protein sequence analysis using powerful statistical techniques. Increased accuracy in comparative analyses, in our minds, is well worth the additional effort.</p>
</div>

<div id="section5" class="section"><a id="s5" name="s5" toc="s5" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0000503"></div><a name="pone.0000503.s001" id="pone.0000503.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000503.s001">Table S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1"></a><p class="preSiDOI">Relative performance of HIV-B<sup>m</sup> and three empirical models on between-host training data. Relative D.F. shows the number of additional degrees of freedom that HIV-B<sup>m</sup> can have and still be preferred (by nested LRT at p = 0.05) to a given empirical model (see text).</p>
<p class="siDoi">doi:10.1371/journal.pone.0000503.s001</p><a id="article1.body1.sec5.supplementary-material1.caption1.p2" name="article1.body1.sec5.supplementary-material1.caption1.p2"></a><p class="postSiDOI">(0.03 MB DOC)</p>
<a name="pone.0000503.s002" id="pone.0000503.s002"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000503.s002">Table S2. </a></strong></p><a id="article1.body1.sec5.supplementary-material2.caption1.p1" name="article1.body1.sec5.supplementary-material2.caption1.p1"></a><p class="preSiDOI">Relative performance of HIV-W<sup>m</sup> and three empirical models on between-host training data. Relative D.F. shows the number of additional degrees of freedom that HIV-W<sup>m</sup> can have and still be preferred (by nested LRT at p = 0.05) to a given empirical model (see text).</p>
<p class="siDoi">doi:10.1371/journal.pone.0000503.s002</p><a id="article1.body1.sec5.supplementary-material2.caption1.p2" name="article1.body1.sec5.supplementary-material2.caption1.p2"></a><p class="postSiDOI">(0.10 MB DOC)</p>
<a name="pone.0000503.s003" id="pone.0000503.s003"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000503.s003">Table S3. </a></strong></p><a id="article1.body1.sec5.supplementary-material3.caption1.p1" name="article1.body1.sec5.supplementary-material3.caption1.p1"></a><p class="preSiDOI">The effect of evolutionary model on pairwise distance estimates using 11 between patient datasets. HIV-B<sup>m</sup> is used as a reference model to compute tree-based pairwise distances, and relative differences for 3 existing empirical models are shown for each dataset.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000503.s003</p><a id="article1.body1.sec5.supplementary-material3.caption1.p2" name="article1.body1.sec5.supplementary-material3.caption1.p2"></a><p class="postSiDOI">(0.04 MB DOC)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>We would like to thank Drs. AG Rodrigo, SDW Frost, AFY Poon, S Zarate, JE Mittler and GH Learn for helpful discussions.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: DN JM SK. Performed the experiments: DN SK. Analyzed the data: DN PG SK. Contributed reagents/materials/analysis tools: DN SK MJ LH. Wrote the paper: DN JM PG SK.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0000503-Hasegawa1" id="pone.0000503-Hasegawa1"></a>Hasegawa M, Kishino H, Yano TA (1985) Dating of the human ape splitting by a molecular clock of mitochondrial DNA. J Mol Evol  22: 160–174.  <ul class="find" data-citedArticleID="956518" data-doi="10.1007/bf02101694"><li><a href="http://dx.doi.org/10.1007/bf02101694" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Dating+of+the+human+ape+splitting+by+a+molecular+clock+of+mitochondrial+DNA." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Dating+of+the+human+ape+splitting+by+a+molecular+clock+of+mitochondrial+DNA.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0000503-Muse1" id="pone.0000503-Muse1"></a>Muse SV, Gaut BS (1994) A likelihood approach for comparing synonymous and nonsynonymous nucleotide substitution rates, with application to the chloroplast genome. Mol Biol Evol  11: 715–724.  <ul class="find" data-citedArticleID="956544"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=A+likelihood+approach+for+comparing+synonymous+and+nonsynonymous+nucleotide+substitution+rates%2C+with+application+to+the+chloroplast+genome.&amp;auth=&amp;atitle=A+likelihood+approach+for+comparing+synonymous+and+nonsynonymous+nucleotide+substitution+rates%2C+with+application+to+the+chloroplast+genome." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+likelihood+approach+for+comparing+synonymous+and+nonsynonymous+nucleotide+substitution+rates%2C+with+application+to+the+chloroplast+genome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+likelihood+approach+for+comparing+synonymous+and+nonsynonymous+nucleotide+substitution+rates%2C+with+application+to+the+chloroplast+genome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0000503-Goldman1" id="pone.0000503-Goldman1"></a>Goldman N, Yang Z (1994) A codon-based model of nucleotide substitution for protein-coding DNA sequences. Mol Biol Evol  11: 725–736.  <ul class="find" data-citedArticleID="956516"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=A+codon-based+model+of+nucleotide+substitution+for+protein-coding+DNA+sequences.&amp;auth=&amp;atitle=A+codon-based+model+of+nucleotide+substitution+for+protein-coding+DNA+sequences." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+codon-based+model+of+nucleotide+substitution+for+protein-coding+DNA+sequences." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+codon-based+model+of+nucleotide+substitution+for+protein-coding+DNA+sequences.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0000503-Dayhoff1" id="pone.0000503-Dayhoff1"></a>Dayhoff MO, Schwartz RM, Orcutt BC (1978)  A model of evolutionary change in proteins. In: Dayhoff MO, editor. Atlas of protein sequence and structure: National Biomedical Research Foundation. Washington D.C.: pp. 345–352.  <ul class="find-nolinks"></ul></li><li><span class="label">5.
              </span><a name="pone.0000503-Henikoff1" id="pone.0000503-Henikoff1"></a>Henikoff S, Henikoff JG (1992) Amino Acid Substitution Matrices from Protein Blocks. PNAS  89: 10915–10919.  <ul class="find" data-citedArticleID="956520" data-doi="10.1073/pnas.89.22.10915"><li><a href="http://dx.doi.org/10.1073/pnas.89.22.10915" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Amino+Acid+Substitution+Matrices+from+Protein+Blocks." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Amino+Acid+Substitution+Matrices+from+Protein+Blocks.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0000503-Karlin1" id="pone.0000503-Karlin1"></a>Karlin S, Ghandour G (1985) Multiple-Alphabet Amino Acid Sequence Comparisons of the Immunoglobulin kappa-Chain Constant Domain. PNAS  82: 8597–8601.  <ul class="find" data-citedArticleID="956526" data-doi="10.1073/pnas.82.24.8597"><li><a href="http://dx.doi.org/10.1073/pnas.82.24.8597" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Multiple-Alphabet+Amino+Acid+Sequence+Comparisons+of+the+Immunoglobulin+kappa-Chain+Constant+Domain." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Multiple-Alphabet+Amino+Acid+Sequence+Comparisons+of+the+Immunoglobulin+kappa-Chain+Constant+Domain.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0000503-George1" id="pone.0000503-George1"></a>George DG, Barker WC, Hunt LT (1990) Mutation data matrix and its uses. Methods Enzymol  183: 333–351.  <ul class="find" data-citedArticleID="956514" data-doi="10.1016/0076-6879(90)83022-2"><li><a href="http://dx.doi.org/10.1016/0076-6879(90)83022-2" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mutation+data+matrix+and+its+uses." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mutation+data+matrix+and+its+uses.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0000503-Feng1" id="pone.0000503-Feng1"></a>Feng DF, Johnson MS, Doolittle RF (1984) Aligning amino acid sequences: comparison of commonly used methods. J Mol Evol  21: 112–125.  <ul class="find" data-citedArticleID="956510" data-doi="10.1007/bf02100085"><li><a href="http://dx.doi.org/10.1007/bf02100085" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Aligning+amino+acid+sequences%3A+comparison+of+commonly+used+methods." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Aligning+amino+acid+sequences%3A+comparison+of+commonly+used+methods.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0000503-Stanfel1" id="pone.0000503-Stanfel1"></a>Stanfel L (1996) A new approach to clustering the amino acids. J Theor Biol  183: 195–205.  <ul class="find" data-citedArticleID="956558" data-doi="10.1006/jtbi.1996.0213"><li><a href="http://dx.doi.org/10.1006/jtbi.1996.0213" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+new+approach+to+clustering+the+amino+acids." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+new+approach+to+clustering+the+amino+acids.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0000503-Yampolsky1" id="pone.0000503-Yampolsky1"></a>Yampolsky LY, Stoltzfus A (2005) The Exchangeability of Amino Acids in Proteins. Genetics  170: 1459–1472.  <ul class="find" data-citedArticleID="956568" data-doi="10.1534/genetics.104.039107"><li><a href="http://dx.doi.org/10.1534/genetics.104.039107" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+Exchangeability+of+Amino+Acids+in+Proteins." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+Exchangeability+of+Amino+Acids+in+Proteins.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0000503-Tang1" id="pone.0000503-Tang1"></a>Tang H, Wyckoff GJ, Lu J, Wu C-I (2004) A Universal Evolutionary Index for Amino Acid Changes. Mol Biol Evol  21: 1548–1556.  <ul class="find" data-citedArticleID="956564" data-doi="10.1093/molbev/msh158"><li><a href="http://dx.doi.org/10.1093/molbev/msh158" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+Universal+Evolutionary+Index+for+Amino+Acid+Changes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+Universal+Evolutionary+Index+for+Amino+Acid+Changes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0000503-Felsenstein1" id="pone.0000503-Felsenstein1"></a>Felsenstein J (1981) Evolutionary trees from DNA-sequences-a maximum-likelihood approach. J Mol Evol  17: 368–376.  <ul class="find" data-citedArticleID="956508" data-doi="10.1007/bf01734359"><li><a href="http://dx.doi.org/10.1007/bf01734359" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evolutionary+trees+from+DNA-sequences-a+maximum-likelihood+approach." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evolutionary+trees+from+DNA-sequences-a+maximum-likelihood+approach.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0000503-Yang1" id="pone.0000503-Yang1"></a>Yang Z, Nielsen R, Hasegawa M (1998) Models of amino acid substitution and applications to mitochondrial protein evolution. Mol Biol Evol  15: 1600–1611.  <ul class="find" data-citedArticleID="956570" data-doi="10.1093/oxfordjournals.molbev.a025888"><li><a href="http://dx.doi.org/10.1093/oxfordjournals.molbev.a025888" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Models+of+amino+acid+substitution+and+applications+to+mitochondrial+protein+evolution." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Models+of+amino+acid+substitution+and+applications+to+mitochondrial+protein+evolution.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0000503-Whelan1" id="pone.0000503-Whelan1"></a>Whelan S, Goldman N (2001) A General Empirical Model of Protein Evolution Derived from Multiple Protein Families Using a Maximum-Likelihood Approach. Mol Biol Evol  18: 691–699.  <ul class="find" data-citedArticleID="956566" data-doi="10.1093/oxfordjournals.molbev.a003851"><li><a href="http://dx.doi.org/10.1093/oxfordjournals.molbev.a003851" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+General+Empirical+Model+of+Protein+Evolution+Derived+from+Multiple+Protein+Families+Using+a+Maximum-Likelihood+Approach." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+General+Empirical+Model+of+Protein+Evolution+Derived+from+Multiple+Protein+Families+Using+a+Maximum-Likelihood+Approach.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0000503-Dimmic1" id="pone.0000503-Dimmic1"></a>Dimmic MW, Rest JS, Mindell DP, Goldstein RA (2002) rtREV: An Amino Acid Substitution Matrix for Inference of Retrovirus and Reverse Transcriptase Phylogeny. J Mol Evol  55: 65–73.  <ul class="find" data-citedArticleID="956506" data-doi="10.1007/s00239-001-2304-y"><li><a href="http://dx.doi.org/10.1007/s00239-001-2304-y" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=rtREV%3A+An+Amino+Acid+Substitution+Matrix+for+Inference+of+Retrovirus+and+Reverse+Transcriptase+Phylogeny." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22rtREV%3A+An+Amino+Acid+Substitution+Matrix+for+Inference+of+Retrovirus+and+Reverse+Transcriptase+Phylogeny.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0000503-Mansky1" id="pone.0000503-Mansky1"></a>Mansky LM (1996) The mutation rate of human immunodeficiency virus type 1 is influenced by the vpr gene. Virology  222: 391–400.  <ul class="find" data-citedArticleID="956542" data-doi="10.1006/viro.1996.0436"><li><a href="http://dx.doi.org/10.1006/viro.1996.0436" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+mutation+rate+of+human+immunodeficiency+virus+type+1+is+influenced+by+the+vpr+gene." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+mutation+rate+of+human+immunodeficiency+virus+type+1+is+influenced+by+the+vpr+gene.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0000503-Nachman1" id="pone.0000503-Nachman1"></a>Nachman MW, Crowell SL (2000) Estimate of the mutation rate per nucleotide in humans. Genetics  156: 297–304.  <ul class="find" data-citedArticleID="956546"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Estimate+of+the+mutation+rate+per+nucleotide+in+humans.&amp;auth=&amp;atitle=Estimate+of+the+mutation+rate+per+nucleotide+in+humans." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Estimate+of+the+mutation+rate+per+nucleotide+in+humans." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Estimate+of+the+mutation+rate+per+nucleotide+in+humans.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0000503-Herbeck1" id="pone.0000503-Herbeck1"></a>Herbeck JT, Nickle DC, Learn GH, Gottlieb GS, Curlin ME, et al.  (2006) Human Immunodeficiency Virus Type 1 env Evolves toward Ancestral States upon Transmission to a New Host. J Virol  80: 1637–1644.  <ul class="find" data-citedArticleID="956522" data-doi="10.1128/jvi.80.4.1637-1644.2006"><li><a href="http://dx.doi.org/10.1128/jvi.80.4.1637-1644.2006" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Human+Immunodeficiency+Virus+Type+1+env+Evolves+toward+Ancestral+States+upon+Transmission+to+a+New+Host." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Human+Immunodeficiency+Virus+Type+1+env+Evolves+toward+Ancestral+States+upon+Transmission+to+a+New+Host.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0000503-KosakovskyPond1" id="pone.0000503-KosakovskyPond1"></a>Kosakovsky Pond SL, Frost SDW, Grossman Z, Gravenor MB, Richman DD, et al.  (2006) Adaptation to Different Human Populations by HIV-1 Revealed by Codon-Based Analyses. PLoS Computational Biology  2: e62.  <ul class="find" data-citedArticleID="956528" data-doi="10.1371/journal.pcbi.0020062.eor"><li><a href="http://dx.doi.org/10.1371/journal.pcbi.0020062.eor" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Adaptation+to+Different+Human+Populations+by+HIV-1+Revealed+by+Codon-Based+Analyses." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Adaptation+to+Different+Human+Populations+by+HIV-1+Revealed+by+Codon-Based+Analyses.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0000503-Liu1" id="pone.0000503-Liu1"></a>Liu Y, McNevin J, Cao J, Zhao H, Genowati I, et al.  (2006) Selection on the Human Immunodeficiency Virus Type 1 Proteome following Primary Infection. J Virol  80: 9519–9529.  <ul class="find" data-citedArticleID="956540" data-doi="10.1128/jvi.00575-06"><li><a href="http://dx.doi.org/10.1128/jvi.00575-06" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Selection+on+the+Human+Immunodeficiency+Virus+Type+1+Proteome+following+Primary+Infection." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Selection+on+the+Human+Immunodeficiency+Virus+Type+1+Proteome+following+Primary+Infection.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0000503-Allen1" id="pone.0000503-Allen1"></a>Allen TM, O'Connor DH, Jing P, Dzuris JL, Mothe BR, et al.  (2000) Tat-specific cytotoxic T lymphocytes select for SIV escape variants during resolution of primary viraemia. Nature  407: 386–390.  <ul class="find" data-citedArticleID="956500"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Tat-specific+cytotoxic+T+lymphocytes+select+for+SIV+escape+variants+during+resolution+of+primary+viraemia.&amp;auth=&amp;atitle=Tat-specific+cytotoxic+T+lymphocytes+select+for+SIV+escape+variants+during+resolution+of+primary+viraemia." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Tat-specific+cytotoxic+T+lymphocytes+select+for+SIV+escape+variants+during+resolution+of+primary+viraemia." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Tat-specific+cytotoxic+T+lymphocytes+select+for+SIV+escape+variants+during+resolution+of+primary+viraemia.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0000503-Friedrich1" id="pone.0000503-Friedrich1"></a>Friedrich TC, Dodds EJ, Yant LJ, Vojnov L, Rudersdorf R, et al.  (2004) Reversion of CTL escape-variant immunodeficiency viruses in vivo. Nat Med  10: 275–281.  <ul class="find" data-citedArticleID="956512" data-doi="10.1038/nm998"><li><a href="http://dx.doi.org/10.1038/nm998" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Reversion+of+CTL+escape-variant+immunodeficiency+viruses+in+vivo." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Reversion+of+CTL+escape-variant+immunodeficiency+viruses+in+vivo.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0000503-Saitou1" id="pone.0000503-Saitou1"></a>Saitou N, Nei M (1987) The neighbor-joining method: a new method for reconstructing phylogenetic trees. Mol Biol Evol  4: 406–425.  <ul class="find" data-citedArticleID="956554"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=The+neighbor-joining+method%3A+a+new+method+for+reconstructing+phylogenetic+trees.&amp;auth=&amp;atitle=The+neighbor-joining+method%3A+a+new+method+for+reconstructing+phylogenetic+trees." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+neighbor-joining+method%3A+a+new+method+for+reconstructing+phylogenetic+trees." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+neighbor-joining+method%3A+a+new+method+for+reconstructing+phylogenetic+trees.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0000503-Tamura1" id="pone.0000503-Tamura1"></a>Tamura K, Nei M (1993) Estimation of the number of nucleotide substitutions in the control region of mitochondrial DNA in humans and chimpanzees. Mol Biol Evol  10: 512–526.  <ul class="find" data-citedArticleID="956562"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Estimation+of+the+number+of+nucleotide+substitutions+in+the+control+region+of+mitochondrial+DNA+in+humans+and+chimpanzees.&amp;auth=&amp;atitle=Estimation+of+the+number+of+nucleotide+substitutions+in+the+control+region+of+mitochondrial+DNA+in+humans+and+chimpanzees." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Estimation+of+the+number+of+nucleotide+substitutions+in+the+control+region+of+mitochondrial+DNA+in+humans+and+chimpanzees." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Estimation+of+the+number+of+nucleotide+substitutions+in+the+control+region+of+mitochondrial+DNA+in+humans+and+chimpanzees.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0000503-KosakovskyPond2" id="pone.0000503-KosakovskyPond2"></a>Kosakovsky Pond SL, Posada D, Gravenor MB, Woelk CH, Frost SDW (2006) Automated Phylogenetic Detection of Recombination Using a Genetic Algorithm. Mol Biol Evol  23: 1891–1901.  <ul class="find" data-citedArticleID="956530" data-doi="10.1093/molbev/msl051"><li><a href="http://dx.doi.org/10.1093/molbev/msl051" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Automated+Phylogenetic+Detection+of+Recombination+Using+a+Genetic+Algorithm." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Automated+Phylogenetic+Detection+of+Recombination+Using+a+Genetic+Algorithm.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0000503-Pond1" id="pone.0000503-Pond1"></a>Pond SLK, Frost SDW, Muse SV (2005) HyPhy: hypothesis testing using phylogenies. Bioinformatics  21: 676–679.  <ul class="find" data-citedArticleID="956550" data-doi="10.1093/bioinformatics/bti079"><li><a href="http://dx.doi.org/10.1093/bioinformatics/bti079" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=HyPhy%3A+hypothesis+testing+using+phylogenies." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22HyPhy%3A+hypothesis+testing+using+phylogenies.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0000503-KosakovskyPond3" id="pone.0000503-KosakovskyPond3"></a>Kosakovsky Pond SL, Frost SDW (2005) A simple hierarchical approach to modeling distributions of substitution rates. Mol Biol Evol  22: 223–234.  <ul class="find" data-citedArticleID="956532" data-doi="10.1093/molbev/msi009"><li><a href="http://dx.doi.org/10.1093/molbev/msi009" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+simple+hierarchical+approach+to+modeling+distributions+of+substitution+rates." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+simple+hierarchical+approach+to+modeling+distributions+of+substitution+rates.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0000503-Jones1" id="pone.0000503-Jones1"></a>Jones D, Taylor W, Thornton J (1992) The rapid generation of mutation data matrices from protein sequences. Comput Appl Biosci  8: 275–282.  <ul class="find" data-citedArticleID="956524" data-doi="10.1093/bioinformatics/8.3.275"><li><a href="http://dx.doi.org/10.1093/bioinformatics/8.3.275" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+rapid+generation+of+mutation+data+matrices+from+protein+sequences." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+rapid+generation+of+mutation+data+matrices+from+protein+sequences.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0000503-Sugiura1" id="pone.0000503-Sugiura1"></a>Sugiura N (1978) Further analysis of the data by Akaike's information criterion and the finite corrections. Comm Stat Theory Methods  A7: 13–26.  <ul class="find" data-citedArticleID="956560"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Further+analysis+of+the+data+by+Akaike%27s+information+criterion+and+the+finite+corrections.&amp;auth=&amp;atitle=Further+analysis+of+the+data+by+Akaike%27s+information+criterion+and+the+finite+corrections." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Further+analysis+of+the+data+by+Akaike%27s+information+criterion+and+the+finite+corrections." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Further+analysis+of+the+data+by+Akaike%27s+information+criterion+and+the+finite+corrections.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0000503-Adachi1" id="pone.0000503-Adachi1"></a>Adachi J, Hasegawa M (1996) Model of amino acid substitution in proteins encoded by mitochondrial DNA. J Mol Evol  42: 459–468.  <ul class="find" data-citedArticleID="956498" data-doi="10.1007/bf02498640"><li><a href="http://dx.doi.org/10.1007/bf02498640" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Model+of+amino+acid+substitution+in+proteins+encoded+by+mitochondrial+DNA." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Model+of+amino+acid+substitution+in+proteins+encoded+by+mitochondrial+DNA.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0000503-KosakovskyPond4" id="pone.0000503-KosakovskyPond4"></a>Kosakovsky Pond SL, Mannino FV, Gravenor MB, Muse SV, Frost SDW (2006) Evolutionary Model Selection with a Genetic Algorithm: a Case Study Using Stem RNA. Mol Biol Evol msl144.  <ul class="find" data-citedArticleID="956534" data-doi="10.1093/molbev/msl144"><li><a href="http://dx.doi.org/10.1093/molbev/msl144" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evolutionary+Model+Selection+with+a+Genetic+Algorithm%3A+a+Case+Study+Using+Stem+RNA." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evolutionary+Model+Selection+with+a+Genetic+Algorithm%3A+a+Case+Study+Using+Stem+RNA.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0000503-Posada1" id="pone.0000503-Posada1"></a>Posada D, Buckley TR (2004) Model Selection and Model Averaging in Phylogenetics: Advantages of Akaike Information Criterion and Bayesian Approaches Over Likelihood Ratio Tests. Syst Biol  53: 793–808.  <ul class="find" data-citedArticleID="956552"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Model+Selection+and+Model+Averaging+in+Phylogenetics%3A+Advantages+of+Akaike+Information+Criterion+and+Bayesian+Approaches+Over+Likelihood+Ratio+Tests.&amp;auth=&amp;atitle=Model+Selection+and+Model+Averaging+in+Phylogenetics%3A+Advantages+of+Akaike+Information+Criterion+and+Bayesian+Approaches+Over+Likelihood+Ratio+Tests." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Model+Selection+and+Model+Averaging+in+Phylogenetics%3A+Advantages+of+Akaike+Information+Criterion+and+Bayesian+Approaches+Over+Likelihood+Ratio+Tests." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Model+Selection+and+Model+Averaging+in+Phylogenetics%3A+Advantages+of+Akaike+Information+Criterion+and+Bayesian+Approaches+Over+Likelihood+Ratio+Tests.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0000503-KosakovskyPond5" id="pone.0000503-KosakovskyPond5"></a>Kosakovsky Pond SL, Mannino FV, Gravenor MB, Muse SV, Frost SD (2007) Evolutionary model selection with a genetic algorithm: a case study using stem RNA. Mol Biol Evol  24: 159–170.  <ul class="find" data-citedArticleID="956536" data-doi="10.1093/molbev/msl144"><li><a href="http://dx.doi.org/10.1093/molbev/msl144" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evolutionary+model+selection+with+a+genetic+algorithm%3A+a+case+study+using+stem+RNA." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evolutionary+model+selection+with+a+genetic+algorithm%3A+a+case+study+using+stem+RNA.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0000503-Burnham1" id="pone.0000503-Burnham1"></a>Burnham KP, Anderson DR (2002) Model selection and multimodel inference: a practical information-theoretic approach. New York, USA: Springer-Verlag.   <ul class="find-nolinks"></ul></li><li><span class="label">35.
              </span><a name="pone.0000503-Leitner1" id="pone.0000503-Leitner1"></a>Leitner T, Escanilla D, Franzen C, Uhlen M, Albert J (1996) Accurate reconstruction of a known HIV-1 transmission history by phylogenetic tree†analysis. PNAS  93: 10864–10869.  <ul class="find" data-citedArticleID="956538" data-doi="10.1073/pnas.93.20.10864"><li><a href="http://dx.doi.org/10.1073/pnas.93.20.10864" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Accurate+reconstruction+of+a+known+HIV-1+transmission+history+by+phylogenetic+tree%26%23x2020%3Banalysis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Accurate+reconstruction+of+a+known+HIV-1+transmission+history+by+phylogenetic+tree%26%23x2020%3Banalysis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">36.
              </span><a name="pone.0000503-Shankarappa1" id="pone.0000503-Shankarappa1"></a>Shankarappa R, Margolick JB, Gange SJ, Rodrigo AG, Upchurch D, et al.  (1999) Consistent viral evolutionary changes associated with the progression of human immunodeficiency virus type 1 infection. J Virol  73: 10489–10502.  <ul class="find" data-citedArticleID="956556"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Consistent+viral+evolutionary+changes+associated+with+the+progression+of+human+immunodeficiency+virus+type+1+infection.&amp;auth=&amp;atitle=Consistent+viral+evolutionary+changes+associated+with+the+progression+of+human+immunodeficiency+virus+type+1+infection." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Consistent+viral+evolutionary+changes+associated+with+the+progression+of+human+immunodeficiency+virus+type+1+infection." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Consistent+viral+evolutionary+changes+associated+with+the+progression+of+human+immunodeficiency+virus+type+1+infection.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">37.
              </span><a name="pone.0000503-Nickle1" id="pone.0000503-Nickle1"></a>Nickle DC, Jensen MA, Gottlieb GS, Shriner D, Learn GH, et al.  (2003) Consensus and ancestral state HIV vaccines. Science  299: 1515–1517.  <ul class="find" data-citedArticleID="956548" data-doi="10.1126/science.299.5612.1515c"><li><a href="http://dx.doi.org/10.1126/science.299.5612.1515c" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Consensus+and+ancestral+state+HIV+vaccines." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Consensus+and+ancestral+state+HIV+vaccines.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000503.XML" value="170136"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000503.PDF" value="398080"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000503.t001.PNG_L" value="59099"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000503.t001.PNG_M" value="57749"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000503.t001.PNG_S" value="3554"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000503.t001.TIF" value="350372"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000503.t001.PNG_I" value="25918"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000503.e001.PNG" value="1959"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000503.e001.TIF" value="23156"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000503.e002.PNG" value="1776"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000503.e002.TIF" value="22928"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000503.e003.PNG" value="3821"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000503.e003.TIF" value="25732"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000503.e004.PNG" value="4779"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000503.e004.TIF" value="27312"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000503.e005.PNG" value="5328"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000503.e005.TIF" value="27220"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000503.e006.PNG" value="1902"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000503.e006.TIF" value="23224"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000503.g001.PNG_L" value="973360"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000503.g001.PNG_M" value="79861"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000503.g001.PNG_S" value="2978"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000503.g001.TIF" value="1621584"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000503.g001.PNG_I" value="31581"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000503.g002.PNG_L" value="40038"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000503.g002.PNG_M" value="12773"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000503.g002.PNG_S" value="1621"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000503.g002.TIF" value="83504"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000503.g002.PNG_I" value="7036"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000503.g003.PNG_L" value="14717"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000503.g003.PNG_M" value="7977"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000503.g003.PNG_S" value="1428"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000503.g003.TIF" value="31284"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000503.g003.PNG_I" value="4584"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000503.t002.PNG_L" value="113469"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000503.t002.PNG_M" value="90673"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000503.t002.PNG_S" value="7581"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000503.t002.TIF" value="644394"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000503.t002.PNG_I" value="62494"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000503.t003.PNG_L" value="175155"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000503.t003.PNG_M" value="38920"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000503.t003.PNG_S" value="2983"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000503.t003.TIF" value="951108"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000503.t003.PNG_I" value="18146"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000503.t004.PNG_L" value="106821"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000503.t004.PNG_M" value="85759"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000503.t004.PNG_S" value="7387"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000503.t004.TIF" value="632016"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000503.t004.PNG_I" value="59479"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000503.t005.PNG_L" value="156289"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000503.t005.PNG_M" value="35220"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000503.t005.PNG_S" value="2856"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000503.t005.TIF" value="905524"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000503.t005.PNG_I" value="16592"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000503.g004.PNG_L" value="31787"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000503.g004.PNG_M" value="24041"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000503.g004.PNG_S" value="1952"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000503.g004.TIF" value="169236"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000503.g004.PNG_I" value="10864"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000503.g005.PNG_L" value="33003"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000503.g005.PNG_M" value="35196"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000503.g005.PNG_S" value="4337"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000503.g005.TIF" value="441908"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000503.g005.PNG_I" value="25224"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000503.s001.DOC" value="28160"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000503.s002.DOC" value="95232"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000503.s003.DOC" value="35328"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000503&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000503" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000503&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0000503&volume=&issue=&title=HIV-Specific Probabilistic Models of Protein Evolution&author_name=David%20C.%20Nickle%2C%20Laura%20Heath%2C%20Mark%20A.%20Jensen%2C%20Peter%20B.%20Gilbert%2C%20James%20I.%20Mullins%2C%20Sergei%20L.%20Kosakovsky%20Pond&start_page=1&end_page=11" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000503" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000503" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0000503" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000503&amp;t=HIV-Specific%20Probabilistic%20Models%20of%20Protein%20Evolution" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000503&title=HIV-Specific%20Probabilistic%20Models%20of%20Protein%20Evolution&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000503&amp;title=HIV-Specific%20Probabilistic%20Models%20of%20Protein%20Evolution" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000503" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0000503&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'HIV-Specific Probabilistic Models of Protein Evolution';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0000503';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20HIV-Specific%20Probabilistic%20Models%20of%20Protein%20Evolution http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000503" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0000503" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">

























          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Evolutionary+immunology%22" title="Search for articles in the subject area:'Evolutionary immunology'"><div class="flagText">Evolutionary immunology</div></a>
              <div data-categoryid="18007" data-articleid="23844"
                   data-categoryname="Evolutionary immunology"
                   class="flagImage" title="Flag 'Evolutionary immunology' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Evolutionary+modeling%22" title="Search for articles in the subject area:'Evolutionary modeling'"><div class="flagText">Evolutionary modeling</div></a>
              <div data-categoryid="18005" data-articleid="23844"
                   data-categoryname="Evolutionary modeling"
                   class="flagImage" title="Flag 'Evolutionary modeling' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Genome+evolution%22" title="Search for articles in the subject area:'Genome evolution'"><div class="flagText">Genome evolution</div></a>
              <div data-categoryid="17207" data-articleid="23844"
                   data-categoryname="Genome evolution"
                   class="flagImage" title="Flag 'Genome evolution' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22HIV%22" title="Search for articles in the subject area:'HIV'"><div class="flagText">HIV</div></a>
              <div data-categoryid="42663" data-articleid="23844"
                   data-categoryname="HIV"
                   class="flagImage" title="Flag 'HIV' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22HIV-1%22" title="Search for articles in the subject area:'HIV-1'"><div class="flagText">HIV-1</div></a>
              <div data-categoryid="42865" data-articleid="23844"
                   data-categoryname="HIV-1"
                   class="flagImage" title="Flag 'HIV-1' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Phylogenetics%22" title="Search for articles in the subject area:'Phylogenetics'"><div class="flagText">Phylogenetics</div></a>
              <div data-categoryid="46093" data-articleid="23844"
                   data-categoryname="Phylogenetics"
                   class="flagImage" title="Flag 'Phylogenetics' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Sequence+alignment%22" title="Search for articles in the subject area:'Sequence alignment'"><div class="flagText">Sequence alignment</div></a>
              <div data-categoryid="18051" data-articleid="23844"
                   data-categoryname="Sequence alignment"
                   class="flagImage" title="Flag 'Sequence alignment' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Viral+evolution%22" title="Search for articles in the subject area:'Viral evolution'"><div class="flagText">Viral evolution</div></a>
              <div data-categoryid="20461" data-articleid="23844"
                   data-categoryname="Viral evolution"
                   class="flagImage" title="Flag 'Viral evolution' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=9196'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=6613'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=925&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>

<div class="block sidebar-comments">
    <div class="header">
        <h3>Comments</h3>
    </div>
      <p><a href="/annotation/listThread.action?root=10403">Question about &quot;strange&quot; replacement scores</a><br>Posted by fabascal</p>
</div>

</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
