

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: GPCR Genes Are Preferentially Retained after Whole Genome Duplication</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0001903"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0001903" />

    <meta name="citation_title" content="GPCR Genes Are Preferentially Retained after Whole Genome Duplication"/>
    <meta itemprop="name" content="GPCR Genes Are Preferentially Retained after Whole Genome Duplication"/>

      <meta name="citation_author" content="Jenia Semyonov"/>
            <meta name="citation_author_institution" content="Division of Reproductive Biology, Department of Obstetrics and Gynecology, Stanford University School of Medicine, Stanford, California, United States of America"/>
      <meta name="citation_author" content="Jae-Il Park"/>
            <meta name="citation_author_institution" content="Division of Reproductive Biology, Department of Obstetrics and Gynecology, Stanford University School of Medicine, Stanford, California, United States of America"/>
      <meta name="citation_author" content="Chia Lin Chang"/>
            <meta name="citation_author_institution" content="Chang Gung University School of Medicine, and Department of Obstetrics and Gynecology, Chang Gung Memorial Hospital, Tao-Yuan, Taiwan"/>
      <meta name="citation_author" content="Sheau Yu Teddy Hsu"/>
            <meta name="citation_author_institution" content="Division of Reproductive Biology, Department of Obstetrics and Gynecology, Stanford University School of Medicine, Stanford, California, United States of America"/>

    <meta name="citation_date" content="2008/4/2"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0001903.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e1903"/>
    <meta name="citation_issue" content="4"/>
    <meta name="citation_volume" content="3"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Birth of two chimeric genes in the Hominidae lineage.; citation_author=A Courseaux; citation_author=JL Nahon; citation_journal_title=Science; citation_volume=291; citation_number=1; citation_pages=1293-1297; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Duplication and divergence: the evolution of new genes and old ideas.; citation_author=JS Taylor; citation_author=J Raes; citation_journal_title=Annu Rev Genet; citation_volume=38; citation_number=2; citation_pages=615-643; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=The evolutionary fate and consequences of duplicate genes.; citation_author=M Lynch; citation_author=JS Conery; citation_journal_title=Science; citation_volume=290; citation_number=3; citation_pages=1151-1155; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Evolution by gene duplication: Springer-Verlag, Heidelberg, Germany.; citation_author=S Ohno; citation_number=4; citation_date=1970; " />
      <meta name="citation_reference" content="citation_title=Preservation of duplicate genes by complementary, degenerative mutations.; citation_author=A Force; citation_author=M Lynch; citation_author=FB Pickett; citation_author=A Amores; citation_author=YL Yan; citation_journal_title=Genetics; citation_volume=151; citation_number=5; citation_pages=1531-1545; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Gene duplication and the origin of novel proteins.; citation_author=AL Hughes; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=102; citation_number=6; citation_pages=8791-8792; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=An adaptive radiation model for the origin of new gene functions.; citation_author=MP Francino; citation_journal_title=Nat Genet; citation_volume=37; citation_number=7; citation_pages=573-577; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Evaluating adaptive evolution.; citation_author=S Yokoyama; citation_journal_title=Nat Genet; citation_volume=30; citation_number=8; citation_pages=350-351; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Lethality and centrality in protein networks.; citation_author=H Jeong; citation_author=SP Mason; citation_author=AL Barabasi; citation_author=ZN Oltvai; citation_journal_title=Nature; citation_volume=411; citation_number=9; citation_pages=41-42; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Robustness against mutations in genetic networks of yeast.; citation_author=A Wagner; citation_journal_title=Nat Genet; citation_volume=24; citation_number=10; citation_pages=355-361; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Role of duplicate genes in genetic robustness against null mutations.; citation_author=Z Gu; citation_author=LM Steinmetz; citation_author=X Gu; citation_author=C Scharfe; citation_author=RW Davis; citation_journal_title=Nature; citation_volume=421; citation_number=11; citation_pages=63-66; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Organismal complexity, protein complexity, and gene duplicability.; citation_author=J Yang; citation_author=R Lusk; citation_author=WH Li; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=100; citation_number=12; citation_pages=15661-15665; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Gene complexity and gene duplicability.; citation_author=X He; citation_author=J Zhang; citation_journal_title=Curr Biol; citation_volume=15; citation_number=13; citation_pages=1016-1021; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Role of positive selection in the retention of duplicate genes in mammalian genomes.; citation_author=SH Shiu; citation_author=JK Byrnes; citation_author=R Pan; citation_author=P Zhang; citation_author=WH Li; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=103; citation_number=14; citation_pages=2232-2236; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=No magic pill for phylogenetic error.; citation_author=JW Thornton; citation_author=B Kolaczkowski; citation_journal_title=Trends Genet; citation_volume=21; citation_number=15; citation_pages=310-311; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Should phylogenetic models be trying to “fit an elephant”?; citation_author=M Steel; citation_journal_title=Trends Genet; citation_volume=21; citation_number=16; citation_pages=307-309; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Signaling Receptome: A Genomic and Evolutionary Perspective of Plasma Membrane Receptors Involved in Signal Transduction.; citation_author=I Ben-Shlomo; citation_author=SY Hsu; citation_author=R Rauch; citation_author=HW Kowalski; citation_author=AJW Hsueh; citation_journal_title=Science's STKE; citation_volume=2003; citation_number=17; citation_pages=RE9; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=The repertoire of G-protein-coupled receptors in fully sequenced genomes.; citation_author=R Fredriksson; citation_author=HB Schioth; citation_journal_title=Mol Pharmacol; citation_volume=67; citation_number=18; citation_pages=1414-1425; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=The G protein-coupled receptor repertoires of human and mouse.; citation_author=DK Vassilatis; citation_author=JG Hohmann; citation_author=H Zeng; citation_author=F Li; citation_author=JE Ranchalis; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=100; citation_number=19; citation_pages=4903-4908; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Identification of a nematode chemosensory gene family.; citation_author=N Chen; citation_author=S Pai; citation_author=Z Zhao; citation_author=A Mah; citation_author=R Newbury; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=102; citation_number=20; citation_pages=146-151; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=The evolutionary divergence of neurotransmitter receptors and second-messenger pathways.; citation_author=KJ Fryxell; citation_journal_title=J Mol Evol; citation_volume=41; citation_number=21; citation_pages=85-97; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=International Union of Pharmacology. XLVI. G Protein-Coupled Receptor List.; citation_author=SM Foord; citation_author=TI Bonner; citation_author=RR Neubig; citation_author=EM Rosser; citation_author=J-P Pin; citation_journal_title=Pharmacol Rev; citation_volume=57; citation_number=22; citation_pages=279-288; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Molecular tinkering of G protein-coupled receptors: an evolutionary success.; citation_author=J Bockaert; citation_author=JP Pin; citation_journal_title=EMBO J; citation_volume=18; citation_number=23; citation_pages=1723-1729; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=A second class of chemosensory receptors in the olfactory epithelium.; citation_author=SD Liberles; citation_author=LB Buck; citation_journal_title=Nature; citation_volume=442; citation_number=24; citation_pages=645-650; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Dramatic variation of the vomeronasal pheromone receptor gene repertoire among five orders of placental and marsupial mammals.; citation_author=WE Grus; citation_author=P Shi; citation_author=Y-p Zhang; citation_author=J Zhang; citation_journal_title=PNAS; citation_volume=102; citation_number=25; citation_pages=5767-5772; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Selective pressures on the olfactory receptor repertoire since the human-chimpanzee divergence.; citation_author=AA Gimelbrant; citation_author=H Skaletsky; citation_author=A Chess; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=101; citation_number=26; citation_pages=9019-9022; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=A comparison of the human and chimpanzee olfactory receptor gene repertoires.; citation_author=Y Gilad; citation_author=O Man; citation_author=G Glusman; citation_journal_title=Genome Res; citation_volume=15; citation_number=27; citation_pages=224-230; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=The odorant receptor repertoire of teleost fish.; citation_author=TS Alioto; citation_author=J Ngai; citation_journal_title=BMC Genomics; citation_volume=6; citation_number=28; citation_pages=173; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Remarkable diversity of mammalian pheromone receptor repertoires.; citation_author=I Rodriguez; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=102; citation_number=29; citation_pages=6639-6640; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Dramatic variation of the vomeronasal pheromone receptor gene repertoire among five orders of placental and marsupial mammals.; citation_author=WE Grus; citation_author=P Shi; citation_author=Y-p Zhang; citation_author=J Zhang; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=102; citation_number=30; citation_pages=5767-5772; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Seven-transmembrane proteins as odorant and chemosensory receptors.; citation_author=P Mombaerts; citation_journal_title=Science; citation_volume=286; citation_number=31; citation_pages=707-711; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Phylogenetic reconstruction of orthology, paralogy, and conserved synteny for dog and human.; citation_author=L Goodstadt; citation_author=CP Ponting; citation_journal_title=PLoS Comput Biol; citation_volume=2; citation_number=32; citation_pages=e133; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Adaptive evolution of MRG, a neuron-specific gene family implicated in nociception.; citation_author=SS Choi; citation_author=BT Lahn; citation_journal_title=Genome Res; citation_volume=13; citation_number=33; citation_pages=2252-2259; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Atypical expansion in mice of the sensory neuron-specific Mrg G protein-coupled receptor family.; citation_author=MJ Zylka; citation_author=X Dong; citation_author=AL Southwell; citation_author=DJ Anderson; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=100; citation_number=34; citation_pages=10043-10048; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=The repertoire of trace amine G-protein-coupled receptors: large expansion in zebrafish.; citation_author=DE Gloriam; citation_author=TK Bjarnadottir; citation_author=YL Yan; citation_author=JH Postlethwait; citation_author=HB Schioth; citation_journal_title=Mol Phylogenet Evol; citation_volume=35; citation_number=35; citation_pages=470-482; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Genome duplication in the teleost fish Tetraodon nigroviridis reveals the early vertebrate proto-karyotype.; citation_author=O Jaillon; citation_author=J-M Aury; citation_author=F Brunet; citation_author=J-L Petit; citation_author=N Stange-Thomann; citation_journal_title=Nature; citation_volume=431; citation_number=36; citation_pages=946-957; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=The zebrafish gene map defines ancestral vertebrate chromosomes.; citation_author=IG Woods; citation_author=C Wilson; citation_author=B Friedlander; citation_author=P Chang; citation_author=DK Reyes; citation_journal_title=Genome Res; citation_volume=15; citation_number=37; citation_pages=1307-1314; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Phylogenetic timing of the fish-specific genome duplication correlates with the diversification of teleost fish.; citation_author=S Hoegg; citation_author=H Brinkmann; citation_author=JS Taylor; citation_author=A Meyer; citation_journal_title=J Mol Evol; citation_volume=59; citation_number=38; citation_pages=190-203; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Tetraodon genome confirms Takifugu findings: most fish are ancient polyploids.; citation_author=Y Van de Peer; citation_journal_title=Genome Biol; citation_volume=5; citation_number=39; citation_pages=250; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Zebrafish hox clusters and vertebrate genome evolution.; citation_author=A Amores; citation_author=A Force; citation_author=YL Yan; citation_author=L Joly; citation_author=C Amemiya; citation_journal_title=Science; citation_volume=282; citation_number=40; citation_pages=1711-1714; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Comparative genomics provides evidence for an ancient genome duplication event in fish.; citation_author=JS Taylor; citation_author=Y Van de Peer; citation_author=I Braasch; citation_author=A Meyer; citation_journal_title=Philos Trans R Soc Lond B Biol Sci; citation_volume=356; citation_number=41; citation_pages=1661-1679; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Comparative genomics: Small genome, big insights.; citation_author=J Mulley; citation_author=P Holland; citation_journal_title=Nature; citation_volume=431; citation_number=42; citation_pages=916-917; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Fugu genome analysis provides evidence for a whole-genome duplication early during the evolution of ray-finned fishes.; citation_author=A Christoffels; citation_author=EGL Koh; citation_author=J-m Chia; citation_author=S Brenner; citation_author=S Aparicio; citation_journal_title=Mol Biol Evol; citation_volume=21; citation_number=43; citation_pages=1146-1151; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Genome duplication, a trait shared by 22000 species of ray-finned fish.; citation_author=JS Taylor; citation_author=I Braasch; citation_author=T Frickey; citation_author=A Meyer; citation_author=Y Van de Peer; citation_journal_title=Genome Res; citation_volume=13; citation_number=44; citation_pages=382-390; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Major events in the genome evolution of vertebrates: Paranome age and size differ considerably between ray-finned fishes and land vertebrates.; citation_author=K Vandepoele; citation_author=W De Vos; citation_author=JS Taylor; citation_author=A Meyer; citation_author=Y Van de Peer; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=101; citation_number=45; citation_pages=1638-1643; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Vertebrate genome evolution and the zebrafish gene map.; citation_author=JH Postlethwait; citation_author=YL Yan; citation_author=MA Gates; citation_author=S Horne; citation_author=A Amores; citation_journal_title=Nat Genet; citation_volume=18; citation_number=46; citation_pages=345-349; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Fish genomics and biology.; citation_author=HR Crollius; citation_author=J Weissenbach; citation_journal_title=Genome Res; citation_volume=15; citation_number=47; citation_pages=1675-1682; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Dosage sensitivity and the evolution of gene families in yeast.; citation_author=B Papp; citation_author=C Pal; citation_author=LD Hurst; citation_journal_title=Nature; citation_volume=424; citation_number=48; citation_pages=194-197; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Chromosome nondisjunction yields tetraploid rather than aneuploid cells in human cell lines.; citation_author=Q Shi; citation_author=RW King; citation_journal_title=Nature; citation_volume=437; citation_number=49; citation_pages=1038-1042; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Rapid elimination of low-copy DNA sequences in polyploid wheat: a possible mechanism for differentiation of homoeologous chromosomes.; citation_author=M Feldman; citation_author=B Liu; citation_author=G Segal; citation_author=S Abbo; citation_author=AA Levy; citation_journal_title=Genetics; citation_volume=147; citation_number=50; citation_pages=1381-1387; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Proof and evolutionary analysis of ancient genome duplication in the yeast Saccharomyces cerevisiae.; citation_author=M Kellis; citation_author=BW Birren; citation_author=ES Lander; citation_journal_title=Nature; citation_volume=428; citation_number=51; citation_pages=617-624; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Gene loss and evolutionary rates following whole-genome duplication in teleost fishes.; citation_author=FG Brunet; citation_author=HR Crollius; citation_author=M Paris; citation_author=JM Aury; citation_author=P Gibert; citation_journal_title=Mol Biol Evol; citation_volume=23; citation_number=52; citation_pages=1808-1816; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Cloning of two novel mammalian paralogs of relaxin/insulin family proteins and their expression in testis and kidney.; citation_author=SY Hsu; citation_journal_title=Mol Endocrinol; citation_volume=13; citation_number=53; citation_pages=2163-2174; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=The three subfamilies of leucine-rich repeat-containing G protein-coupled receptors (LGR): identification of LGR6 and LGR7 and the signaling mechanism for LGR7.; citation_author=SY Hsu; citation_author=M Kudo; citation_author=T Chen; citation_author=K Nakabayashi; citation_author=A Bhalla; citation_journal_title=Mol Endocrinol; citation_volume=14; citation_number=54; citation_pages=1257-1271; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Evolution of glycoprotein hormone subunit genes in bilateral metazoa: identification of two novel human glycoprotein hormone subunit family genes, GPA2 and GPB5.; citation_author=SY Hsu; citation_author=K Nakabayashi; citation_author=A Bhalla; citation_journal_title=Mol Endocrinol; citation_volume=16; citation_number=55; citation_pages=1538-1551; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Intermedin is a calcitonin/calcitonin gene-related peptide family peptide acting through the calcitonin receptor-like receptor/receptor activity-modifying protein receptor complexes.; citation_author=J Roh; citation_author=CL Chang; citation_author=A Bhalla; citation_author=C Klein; citation_author=SY Hsu; citation_journal_title=J Biol Chem; citation_volume=279; citation_number=56; citation_pages=7264-7274; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Discovering new hormones, receptors, and signaling mediators in the genomic era.; citation_author=SY Hsu; citation_author=AJ Hsueh; citation_journal_title=Mol Endocrinol; citation_volume=14; citation_number=57; citation_pages=594-604; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Human stresscopin and stresscopin-related peptide are selective ligands for the type 2 corticotropin-releasing hormone receptor.; citation_author=SY Hsu; citation_author=AJ Hsueh; citation_journal_title=Nat Med; citation_volume=7; citation_number=58; citation_pages=605-611; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Genome duplication, divergent resolution and speciation.; citation_author=JS Taylor; citation_author=Y Van de Peer; citation_author=A Meyer; citation_journal_title=Trends Genet; citation_volume=17; citation_number=59; citation_pages=299-301; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=The human phylome.; citation_author=J Huerta-Cepas; citation_author=H Dopazo; citation_author=J Dopazo; citation_author=T Gabaldon; citation_journal_title=Genome Biol; citation_volume=8; citation_number=60; citation_pages=R109; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=The rapid evolution of reproductive proteins.; citation_author=WJ Swanson; citation_author=VD Vacquier; citation_journal_title=Nat Rev Genet; citation_volume=3; citation_number=61; citation_pages=137-144; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Adaptive evolution and explosive speciation: The cichlid fish model.; citation_author=TD Kocher; citation_journal_title=Nat Rev Genet; citation_volume=5; citation_number=62; citation_pages=288-298; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Genome sequence, comparative analysis and haplotype structure of the domestic dog.; citation_author=K Lindblad-Toh; citation_author=CM Wade; citation_author=TS Mikkelsen; citation_author=EK Karlsson; citation_author=DB Jaffe; citation_journal_title=Nature; citation_volume=438; citation_number=63; citation_pages=803-819; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Gene and protein domain-specific patterns of genetic variability within the G-protein coupled receptor superfamily.; citation_author=K Small; citation_author=D Tanguay; citation_author=K Nandabalan; citation_author=P Zhan; citation_author=J Stephens; citation_journal_title=Am J Pharmacogenomics; citation_volume=3; citation_number=64; citation_pages=65-71; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Divergent Selection on Opsins Drives Incipient Speciation in Lake Victoria Cichlids.; citation_author=Y Terai; citation_author=O Seehausen; citation_author=T Sasaki; citation_author=K Takahashi; citation_author=S Mizoiri; citation_journal_title=PLoS Biol; citation_volume=4; citation_number=65; citation_pages=e433; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Sequence and comparative analysis of the chicken genome provide unique perspectives on vertebrate evolution.; citation_author=ICGS Consortium; citation_journal_title=Nature; citation_volume=432; citation_number=66; citation_pages=695-716; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Whole-genome shotgun assembly and analysis of the genome of Fugu rubripes.; citation_author=S Aparicio; citation_author=J Chapman; citation_author=E Stupka; citation_author=N Putnam; citation_author=JM Chia; citation_journal_title=Science; citation_volume=297; citation_number=67; citation_pages=1301-1310; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Gapped BLAST and PSI-BLAST: a new generation of protein database search programs.; citation_author=SF Altschul; citation_author=TL Madden; citation_author=AA Schaffer; citation_author=J Zhang; citation_author=Z Zhang; citation_journal_title=Nucleic Acids Res; citation_volume=25; citation_number=68; citation_pages=3389-3402; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Improving the accuracy of PSI-BLAST protein database searches with composition-based statistics and other refinements.; citation_author=AA Schaffer; citation_author=L Aravind; citation_author=TL Madden; citation_author=S Shavirin; citation_author=JL Spouge; citation_journal_title=Nucleic Acids Res; citation_volume=29; citation_number=69; citation_pages=2994-3005; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=BLAST 2 Sequences, a new tool for comparing protein and nucleotide sequences.; citation_author=TA Tatusova; citation_author=TL Madden; citation_journal_title=FEMS Microbiol Lett; citation_volume=174; citation_number=70; citation_pages=247-250; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=CLUSTAL W: improving the sensitivity of progressive multiple sequence alignment through sequence weighting, position-specific gap penalties and weight matrix choice.; citation_author=JD Thompson; citation_author=DG Higgins; citation_author=TJ Gibson; citation_journal_title=Nucleic Acids Res; citation_volume=22; citation_number=71; citation_pages=4673-4680; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=The neighbor-joining method: a new method for reconstructing phylogenetic trees.; citation_author=N Saitou; citation_author=M Nei; citation_journal_title=Mol Biol Evol; citation_volume=4; citation_number=72; citation_pages=406-425; citation_date=1987; " />
      <meta name="citation_reference" content="citation_title=PHYLIP-Phylogeny Inference Package (Version 3.2).; citation_author=J Felsenstein; citation_journal_title=Cladistics; citation_volume=5; citation_number=73; citation_pages=164-166; citation_date=1989; " />
      <meta name="citation_reference" content="citation_title=Ensembl 2005.; citation_author=T Hubbard; citation_author=D Andrews; citation_author=M Caccamo; citation_author=G Cameron; citation_author=Y Chen; citation_journal_title=Nucleic Acids Res; citation_volume=33; citation_number=74; citation_pages=D447-453; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=EnsMart: a generic system for fast and flexible access to biological data.; citation_author=A Kasprzyk; citation_author=D Keefe; citation_author=D Smedley; citation_author=D London; citation_author=W Spooner; citation_journal_title=Genome Res; citation_volume=14; citation_number=75; citation_pages=160-169; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=BLAT–the BLAST-like alignment tool.; citation_author=WJ Kent; citation_journal_title=Genome Res; citation_volume=12; citation_number=76; citation_pages=656-664; citation_date=2002; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001903" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="GPCR Genes Are Preferentially Retained after Whole Genome Duplication"/>
    <meta name="twitter:description" content="One of the most interesting questions in biology is whether certain pathways have been favored during evolution, and if so, what properties could cause such a preference. Due to the lack of experimental evidence, whether select gene families have been preferentially retained over time after duplication in metazoan organisms remains unclear. Here, by syntenic mapping of nonchemosensory G protein-coupled receptor genes (nGPCRs which represent half the receptome for transmembrane signaling) in the vertebrate genomes, we found that, as opposed to the 8&ndash;15% retention rate for whole genome duplication (WGD)-derived gene duplicates in the entire genome of pufferfish, greater than 27.8% of WGD-derived nGPCRs which interact with a nonpeptide ligand were retained after WGD in pufferfish Tetraodon nigroviridis. In addition, we show that concurrent duplication of cognate ligand genes by WGD could impose selection of nGPCRs that interact with a polypeptide ligand. Against less than 2.25% probability for parallel retention of a pair of WGD-derived ligands and a pair of cognate receptor duplicates, we found a more than 8.9% retention of WGD-derived ligand-nGPCR pairs&ndash;threefold greater than one would surmise. These results demonstrate that gene retention is not uniform after WGD in vertebrates, and suggest a Darwinian selection of GPCR-mediated intercellular communication in metazoan organisms."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0001903.g006"/>

  <meta property="og:title" content="GPCR Genes Are Preferentially Retained after Whole Genome Duplication" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001903" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=4425'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=5552'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=4820&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0001903">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001903" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001903&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    GPCR Genes Are Preferentially Retained after Whole Genome Duplication
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Jenia Semyonov, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Division of Reproductive Biology, Department of Obstetrics and Gynecology, Stanford University School of Medicine, Stanford, California, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Jae-Il Park, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Division of Reproductive Biology, Department of Obstetrics and Gynecology, Stanford University School of Medicine, Stanford, California, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Chia Lin Chang, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Chang Gung University School of Medicine, and Department of Obstetrics and Gynecology, Chang Gung Memorial Hospital, Tao-Yuan, Taiwan
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Sheau Yu Teddy Hsu
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">* E-mail:</span> <a href="mailto:teddyhsu@stanford.edu">teddyhsu@stanford.edu</a></p>

                <p>Affiliation:
                  Division of Reproductive Biology, Department of Obstetrics and Gynecology, Stanford University School of Medicine, Stanford, California, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: April 02, 2008</li>
    <li>DOI: 10.1371/journal.pone.0001903</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0001903" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0001903" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001903" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0001903" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0001903-g001" data-doi="info:doi/10.1371/journal.pone.0001903" data-uri="info:doi/10.1371/journal.pone.0001903.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001903.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001903-g002" data-doi="info:doi/10.1371/journal.pone.0001903" data-uri="info:doi/10.1371/journal.pone.0001903.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001903.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001903-g003" data-doi="info:doi/10.1371/journal.pone.0001903" data-uri="info:doi/10.1371/journal.pone.0001903.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001903.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001903-g004" data-doi="info:doi/10.1371/journal.pone.0001903" data-uri="info:doi/10.1371/journal.pone.0001903.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001903.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001903-g005" data-doi="info:doi/10.1371/journal.pone.0001903" data-uri="info:doi/10.1371/journal.pone.0001903.g005" title="Figure 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001903.g005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001903-g006" data-doi="info:doi/10.1371/journal.pone.0001903" data-uri="info:doi/10.1371/journal.pone.0001903.g006" title="Figure 6">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001903.g006&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001903">Reader Comments (0)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0001903" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1"></a><p>One of the most interesting questions in biology is whether certain pathways have been favored during evolution, and if so, what properties could cause such a preference. Due to the lack of experimental evidence, whether select gene families have been preferentially retained over time after duplication in metazoan organisms remains unclear. Here, by syntenic mapping of nonchemosensory G protein-coupled receptor genes (nGPCRs which represent half the receptome for transmembrane signaling) in the vertebrate genomes, we found that, as opposed to the 8–15% retention rate for whole genome duplication (WGD)-derived gene duplicates in the entire genome of pufferfish, greater than 27.8% of WGD-derived nGPCRs which interact with a nonpeptide ligand were retained after WGD in pufferfish <em>Tetraodon nigroviridis</em>. In addition, we show that concurrent duplication of cognate ligand genes by WGD could impose selection of nGPCRs that interact with a polypeptide ligand. Against less than 2.25% probability for parallel retention of a pair of WGD-derived ligands and a pair of cognate receptor duplicates, we found a more than 8.9% retention of WGD-derived ligand-nGPCR pairs–threefold greater than one would surmise. These results demonstrate that gene retention is not uniform after WGD in vertebrates, and suggest a Darwinian selection of GPCR-mediated intercellular communication in metazoan organisms.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>Semyonov J, Park J-I, Chang CL, Hsu SYT (2008) GPCR Genes Are Preferentially Retained after Whole Genome Duplication. PLoS ONE 3(4):
          e1903.
            doi:10.1371/journal.pone.0001903</p><p><strong>Editor: </strong>Christophe d'Enfert, Institut Pasteur, France</p><p><strong>Received:</strong> June 1, 2007; <strong>Accepted:</strong> January 27, 2008; <strong>Published:</strong> April 2, 2008</p><p><strong>Copyright:</strong> © 2008 Semyonov et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>J. Semyonov is supported by the Bioinformatics Core of the SCCPRR Center at the Stanford University School of Medicine (NIH, HD31398). The authors also acknowledge the support of NIH awards (R21 HD47606 and RO1 DK70652, SYH) and the March of Dimes research grant (SYH).</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>Studies of the evolutionary paths of genes have shown that genome novelty is generated primarily by gene duplication and subsequent functional changes, and to a lesser extent, by <em>de novo</em> generation or the creation of mosaic genes <a href="#pone.0001903-Courseaux1">[1]</a>, <a href="#pone.0001903-Taylor1">[2]</a>, <a href="#pone.0001903-Lynch1">[3]</a>. Gene duplication not only provides more substrates for divergence through subfunctionalization or neofunctionalization, but also establishes a robustness against null phenotypes through compensation <a href="#pone.0001903-Taylor1">[2]</a>, <a href="#pone.0001903-Ohno1">[4]</a>, <a href="#pone.0001903-Force1">[5]</a>, <a href="#pone.0001903-Hughes1">[6]</a>, <a href="#pone.0001903-Francino1">[7]</a>, <a href="#pone.0001903-Yokoyama1">[8]</a>, <a href="#pone.0001903-Jeong1">[9]</a>, <a href="#pone.0001903-Wagner1">[10]</a>, <a href="#pone.0001903-Gu1">[11]</a>. Recently, it was shown that gene duplicability may be associated with gene and protein complexity <a href="#pone.0001903-Yang1">[12]</a>, <a href="#pone.0001903-He1">[13]</a>, <a href="#pone.0001903-Shiu1">[14]</a>; however, from these earlier studies one cannot discern whether the fixation of gene duplicate(s) is due to incidences of increased duplication or preferential retention. Consequently, no consensus theory has been presented on whether specific families of genes are preferentially retained following gene duplication at either local, segmental, chromosomal, or genomic level.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>To investigate whether the rate of retention for select genes after duplication, not gene duplicability which is the sum of results from gene duplication and gene retention, is greater when compared to the genome average, we explored the recently available syntenic maps of representative tetrapods and teleosts that experienced a lineage-specific whole genome duplication (WGD) more than 230 million years ago. Given an equal opportunity for duplication for all genes during WGD, one could quantitatively analyze the relationship between gene retention and gene function by comparing the inventory of orthologous genes in nonduplicated species (tetrapods) with that from lineages experiencing WGD (teleosts). As all genes duplicate in parallel during WGD, these analyses would avoid errors associated with heterogeneity in gene divergence (heterotachy) <a href="#pone.0001903-Thornton1">[15]</a>, <a href="#pone.0001903-Steel1">[16]</a>. With this understanding, we reasoned that if WGD-derived duplicates belonging to a select group of genes are present in greater proportion when compared to the average of the entire genome, the data would support the hypothesis that select gene families are predisposed for retention after gene duplication.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>Major advances during metazoan evolution include overall divergence in cell types associated with specialized functions and the expansion of intercellular signaling networks. As cell types increase, the need for selective intercellular communication increases. As opposed to the single cell yeast that encodes only a primitive mating signaling system, vertebrates have a multitudinous selection of specialized intercellular signaling pathways for communicating among &gt;250 different cell types. Our earlier studies have shown that different classes of cell surface receptors emerged and expanded at discrete evolutionary times. Whereas some cell surface receptor families are vertebrate-, chordate-, or urbilateria-specific, the seven-transmembrane (7TM) receptors are present in all eukaryotes <a href="#pone.0001903-BenShlomo1">[17]</a>, and the proportion of 7TM receptor genes increases from 0.05% in unicellular yeast to more than 3% in multiple metazoan lineages <a href="#pone.0001903-Fredriksson1">[18]</a>, <a href="#pone.0001903-Vassilatis1">[19]</a>, <a href="#pone.0001903-Chen1">[20]</a>. Although the mechanisms underlying the expansion of cell surface receptors in metazoans are not clear, we hypothesized that the fitness associated with, 1) the potential to increase signaling specificity, and 2) the unidirectional signaling characteristics of cell surface receptors could impose a lower genetic constraint on the retention of receptor genes after duplication, thereby setting them apart from intracellular proteins that normally interact with a multitude of partners in two-way communication.</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p>To test this hypothesis, we analyzed the retention of WGD-derived duplicates of nonchemosensory G protein-coupled receptors (nGPCRs), which together represent a majority of the receptome in vertebrates, as well as their cognate ligands in pufferfish <a href="#pone.0001903-BenShlomo1">[17]</a>, <a href="#pone.0001903-Fredriksson1">[18]</a>. Among the protein families, the structurally constrained nGPCRs represent one of the few groups of genes that meets our requirements for gene retention studies–descendent genes must retain features of their predecessors significant enough to allow the tracing of orthologous relationships, and have similar gene ontology in molecular function, biological processes, and cellular components <a href="#pone.0001903-Fryxell1">[21]</a>. For the quantitative analysis of gene retention, these requirements are essential to reduce the bias associated with heterotachy, gene shuffling, and chimerization <a href="#pone.0001903-Thornton1">[15]</a>, <a href="#pone.0001903-Steel1">[16]</a>. In agreement with our hypothesis, our study demonstrates that gene retention is not uniform after WGD, and suggests a Darwinian selection of GPCR-mediated signaling for intercellular signaling in metazoan organisms.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results and Discussion"></a><h3>Results and Discussion</h3>
<h4>Ancestral nGPCR genes gave rise to twice as many descendents in teleosts as in tetrapods</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>Earlier analyses of genomes showed that humans and mice share a similar repertoire of nGPCRs and encode about 367 and 392 nGPCRs, respectively, belonging to rhodopsin (class A), secretin receptor (class B), GABA receptor (class C), and Frizzled receptor (class F) classes <a href="#pone.0001903-Vassilatis1">[19]</a>, <a href="#pone.0001903-Foord1">[22]</a>, <a href="#pone.0001903-Bockaert1">[23]</a>. In addition to these nGPCRs, vertebrates encode several groups of chemosensory GPCRs including olfactory receptors, vomeronasal receptor-like genes, taste receptors, and pheromone receptors <a href="#pone.0001903-Fredriksson1">[18]</a>, <a href="#pone.0001903-Chen1">[20]</a>, <a href="#pone.0001903-Liberles1">[24]</a>, <a href="#pone.0001903-Grus1">[25]</a>, <a href="#pone.0001903-Gimelbrant1">[26]</a>, <a href="#pone.0001903-Gilad1">[27]</a>, <a href="#pone.0001903-Alioto1">[28]</a>, <a href="#pone.0001903-Rodriguez1">[29]</a>, <a href="#pone.0001903-Grus2">[30]</a>, <a href="#pone.0001903-Mombaerts1">[31]</a>. Because selection pressure has driven significant lineage-specific expansions of these chemosensory receptors, the inventory of these 7TM receptor genes varies drastically even among closely related species, thereby precluding them from the assignment of orthologous relationships among species <a href="#pone.0001903-Goodstadt1">[32]</a>. Thus, we focused our analyses on nGPCRs.</p>
<a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2"></a><p>In our searches using the published inventory of human and mouse nGPCRs as queries, we found that human, rat, mouse, chicken, <em>T. rubripes</em>, and <em>T. nigroviridis</em> each contain approximately 359, 359, 382, 310, 431, and 438 genes, respectively, belonging to the four main nGPCR classes (A, B, C, and F) (<a href="#pone-0001903-g001">Fig. 1</a>, <a href="#pone.0001903.s003">Tables S1 A and S1 B</a>) <a href="#pone.0001903-Fredriksson1">[18]</a>, <a href="#pone.0001903-Vassilatis1">[19]</a>, <a href="#pone.0001903-Foord1">[22]</a>, <a href="#pone.0001903-Bockaert1">[23]</a>. In addition, we subdivided the largest class, class A nGPCRs, into eight subclasses (A1–A8) based on phylogenetic relatedness and the chemical properties of the cognate ligand(s) in order to facilitate our subsequent analyses of the relationships between gene retention and receptor characteristics <a href="#pone.0001903-Vassilatis1">[19]</a>: The majority of nGPCRs in the A1–A4 subclasses interact with a ligand not encoded by a gene, including photons, ions, derivatives of lipids, carbohydrates, amino acids, and nucleosides, whereas those in other subclasses mainly interact with polypeptide ligand(s) (<a href="#pone-0001903-g001">Fig. 1</a>, <a href="#pone.0001903.s003">Tables S1 A and S1 B</a>).</p>
<div class="figure" id="pone-0001903-g001"><div class="img"><a name="pone-0001903-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001903.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001903" data-uri="info:doi/10.1371/journal.pone.0001903.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001903.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001903.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001903.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001903.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001903.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001903.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001903.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001903.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001903.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Inventory of nonchemosensory GPCRs (nGPCRs) in vertebrates.</span></strong></p><a id="article1.body1.sec2.sec1.fig1.caption1.p1" name="article1.body1.sec2.sec1.fig1.caption1.p1"></a><p>Nonchemosensory GPCRs belonging to the rhodopsin class (A), the secretin class (B), the metabotropic glutamate class (C), and the Frizzled class (F) were identified from the genomes of human (N = 359), rat (N = 359), mouse (N = 382), chicken (N = 310), pufferfish <em>Tetraodon nigroviridis</em> (N = 431), and pufferfish <em>Takifugu rubripes</em> (N = 438)(see <a href="#pone.0001903.s003">Tables S1 A and S1 B</a> for a complete list). Receptors belonging to different classes are indicated by different colors in the stacked bars. The rhodopsin class nGPCRs are subdivided into eight subclasses (A1–A8) based on their phylogenetic relationships and the chemical properties of the ligand <a href="#pone.0001903-Vassilatis1">[19]</a>.</p>
<span>doi:10.1371/journal.pone.0001903.g001</span></div><a id="article1.body1.sec2.sec1.p3" name="article1.body1.sec2.sec1.p3"></a><p>To obtain a basal point for tracing the evolutionary changes of orthologous nGPCRs in duplicated (teleost) and nonduplicated (tetrapod) genomes, we first defined the nGPCR inventory in the most recent common ancestor (MRCA) of these species. Through phylogenetic analysis and syntenic mapping, we identified a total of 269 clusters of orthologous nGPCRs (207 class A, 38 class B, 14 class C, and 10 class F) activated by a variety of neurotransmitters, nucleoside derivatives, lipophilic compounds, or peptide hormones (<a href="#pone.0001903.s004">Tables S2 A and S2 B</a>). The major exceptions are approximately two dozen mammal- or tetrapod-specific tandem duplication-derived trace amine receptors (subclass A2), chemokine receptors (subclass A7), MAS-related family receptors (subclass A8), and the origins of these nGPCRs remain to be determined (<a href="#pone.0001903.s004">Table S2 C</a>) <a href="#pone.0001903-Choi1">[33]</a>, <a href="#pone.0001903-Zylka1">[34]</a>, <a href="#pone.0001903-Gloriam1">[35]</a>. Based on these analyses, we inferred that the MRCAs of tetrapods and teleosts contained at least 269 ancestral nGPCRs belonging to A1–A7, B, C, and F classes over 450 million years ago.</p>
<a id="article1.body1.sec2.sec1.p4" name="article1.body1.sec2.sec1.p4"></a><p>Radar-plots of descendent genes derived from each of the 269 ancestral nGPCRs showed that the number of descendent genes varies from 0 to more than 6 in these species (<a href="#pone-0001903-g002">Fig. 2A</a>). As expected, the number of descendent genes derived from the 269 nGPCR ancestors is highly correlated between the two pufferfish (R<sup>2</sup> = 0.79) or tetrapod species (e.g., R<sup>2</sup> = 0.91 between human and mouse)(<a href="#pone.0001903.s004">Table S2 D</a>). However, the correlations between that of teleost and tetrapod species are significantly lower (e.g., R<sup>2</sup> = 0.12 for <em>T. nigroviridis</em> and human, <a href="#pone.0001903.s004">Table S2 D</a>). These analyses further indicated that more than 41% of nGPCR ancestors evolved into more than one descendent gene in <em>T. nigroviridis</em> and <em>T. rubripes</em> (<a href="#pone-0001903-g002">Fig. 2B</a>, upper panel; <a href="#pone.0001903.s004">Table S2 E</a>), whereas only 16.3–21.3% of nGPCR ancestors gave rise to more than one nGPCR paralog in tetrapods. In pufferfish, these descendent genes represent approximately 65% of the nGPCR repertoire (276/425 in <em>T. nigroviridis</em> and 282/430 in <em>T. rubripes</em>)(<a href="#pone-0001903-g002">Fig. 2B</a>, lower panel; <a href="#pone.0001903.s004">Table S2 E</a>). In contrast, only 32.2–37.5% of the nGPCR inventory in tetrapods evolved from gene duplication after the separation from teleosts. The combined results thus suggested that a large fraction of nGPCRs in tetrapods and teleosts evolved independently after the separation of these two lineages.</p>
<div class="figure" id="pone-0001903-g002"><div class="img"><a name="pone-0001903-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001903.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001903" data-uri="info:doi/10.1371/journal.pone.0001903.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001903.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001903.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001903.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001903.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001903.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001903.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001903.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001903.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001903.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Radial evolution of nGPCR genes in tetrapods and teleosts.</span></strong></p><a id="article1.body1.sec2.sec1.fig2.caption1.p1" name="article1.body1.sec2.sec1.fig2.caption1.p1"></a><p>Based on phylogenetic analyses of nGPCR homologs from different vertebrate species, we deduced that the genome of the most recent common ancestor (MRCA) of tetrapods and teleosts encoded at least 269 ancestral nGPCR genes belonging to the four main classes (207 class A, 38 class B, 14 class C, and 10 class F) of vertebrate nGPCRs. A) The radar plot shows the number of paralogous nGPCRs derived from each of the 269 nGPCR ancestors in human, rat, mouse, chicken, <em>T. nigroviridis</em>, and <em>T. rubripes</em> (see complete list in <a href="#pone.0001903.s004">Tables S2 A and S2 B</a>). Each ring of the radar plot represents one copy of a paralogous gene, and the divisions of different classes of nGPCRs are indicated by dashed lines. The presence of duplicates is most widespread in <em>T. nigroviridis</em> and <em>T. rubripes</em>. B) Approximately 65% of pufferfish nGPCRs are derived from lineage-specific gene duplications whereas only 32.3–37.5% of nGPCRs in tetrapods evolved from gene duplications after the separation of tetrapods and teleosts (lower panel). Analyses based on a per ancestral gene basis showed more than 41% of MRCA nGPCRs evolved into more than one paralog in teleosts as compared to only 16.3–21.3% in tetrapods (upper panel).</p>
<span>doi:10.1371/journal.pone.0001903.g002</span></div><a id="article1.body1.sec2.sec1.p5" name="article1.body1.sec2.sec1.p5"></a><p>Although vertebrates from pufferfish to humans share a similar gene inventory, recent analyses demonstrated that a WGD occurred before the divergence of teleosts and osteoglossomorphs more than 230–350 million years ago, whereas other ray-finned fish (actinopterygians) and all sarcopterygians (tetrapods and coelacanthiforms) experienced no such event <a href="#pone.0001903-Jaillon1">[36]</a>, <a href="#pone.0001903-Woods1">[37]</a>, <a href="#pone.0001903-Hoegg1">[38]</a>, <a href="#pone.0001903-VandePeer1">[39]</a>, <a href="#pone.0001903-Amores1">[40]</a>, <a href="#pone.0001903-Taylor2">[41]</a>, <a href="#pone.0001903-Mulley1">[42]</a>, <a href="#pone.0001903-Christoffels1">[43]</a>, <a href="#pone.0001903-Taylor3">[44]</a>, <a href="#pone.0001903-Vandepoele1">[45]</a>, <a href="#pone.0001903-Postlethwait1">[46]</a>, <a href="#pone.0001903-Crollius1">[47]</a>. As tetraploidy was deleterious and strongly selected against, the duplicated genomes in the tetraploid teleost ancestor eventually coalesced in a process called diploidization <a href="#pone.0001903-Lynch1">[3]</a>, <a href="#pone.0001903-Papp1">[48]</a>, <a href="#pone.0001903-Shi1">[49]</a>, <a href="#pone.0001903-Feldman1">[50]</a>, <a href="#pone.0001903-Kellis1">[51]</a>. Based on a spectrum of analytical approaches and stringency settings in defining WGD-derived gene duplicates, several recent studies have estimated that only 8 to 15% of WGD-derived duplicates were retained in present-day pufferfish, <em>Takifugu rubripes</em> and <em>Tetraodon nigroviridis</em> <a href="#pone.0001903-Jaillon1">[36]</a>, <a href="#pone.0001903-Woods1">[37]</a>, <a href="#pone.0001903-Christoffels1">[43]</a>, <a href="#pone.0001903-Vandepoele1">[45]</a>, <a href="#pone.0001903-Crollius1">[47]</a>, <a href="#pone.0001903-Brunet1">[52]</a>. The finding that over 65% of the pufferfish nGPCR repertoire consists of lineage-specific duplicates was unexpected as evidence has shown that, 1) vertebrates from teleosts to tetrapods share a similar gene inventory, and 2) only 8–15% of WGD-derived gene duplicates survive in pufferfish <a href="#pone.0001903-Jaillon1">[36]</a>, <a href="#pone.0001903-Woods1">[37]</a>, <a href="#pone.0001903-Christoffels1">[43]</a>, <a href="#pone.0001903-Vandepoele1">[45]</a>, <a href="#pone.0001903-Crollius1">[47]</a>, <a href="#pone.0001903-Brunet1">[52]</a>. These data implied that evolution of nGPCRs as a category could have been effected by a selection pressure different from that for the rest of the genome after the instance of WGD in teleosts.</p>


<h4>WGD-derived nGPCR duplicates were retained at a rate significantly higher than that for the entire genome</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>Given the finding that teleosts experienced a lineage-specific WGD during evolution, we hypothesized that the large nGPCR inventory in teleosts could be attributed to, 1) increases in the incidence of tandem duplication-derived nGPCRs, and/or 2) increases in the retention of WGD-derived nGPCR duplicates. Analysis of the distribution of the166 nGPCR families with assigned chromosomal localization(s) on <em>T. nigroviridis</em> chromosomes showed that 17 pairs of these nGPCRs represent paralogs derived from tandem duplications (<a href="#pone.0001903.s005">Table S3 A</a>). Likewise, we found that 18, 17, 18, and 12 groups of paralogous nGPCRs from human, rat, mouse, and chicken, respectively, were derived from intrachromosomal tandem duplications (<a href="#pone.0001903.s005">Table S3 B</a>), and that all tetrapods are endowed with seven groups of these duplicates (<a href="#pone.0001903.s005">Table S3 C</a>). Therefore, the rate of tandem duplication in teleosts and tetrapods is similar, and the small number of tandem duplication-derived duplicates cannot account for the large expansion of nGPCR homologs in pufferfish. In contrast, we found that 39 pairs of these nGPCRs were located on two WGD-derived syntenic chromosomal regions (or homologons), reminiscent of the binary distribution of the 750 pairs of previously characterized WGD-derived gene duplicates in <em>T. nigroviridis</em> (<a href="#pone-0001903-g003">Fig. 3A</a>, upper panel; an example of the binary distribution for WGD-derived <em>GPR61</em> duplicates is shown in the lower panel; <a href="#pone.0001903.s005">Tables S3 D and S3 E</a>)<a href="#pone.0001903-Jaillon1">[36]</a>, <a href="#pone.0001903-Kellis1">[51]</a>. The finding is important as data indicate that more than 23.5% (39/166 receptor families with assigned chromosomal localization(s)) of WGD-derived nGPCR genes were retained and fixed in pufferfish. This retention rate is significantly higher than the high retention rate estimate of 15% for the entire genome from studies using similar statistical criteria (<a href="#pone-0001903-g003">Fig. 3B</a>); t-test, P = 0.00014) <a href="#pone.0001903-Jaillon1">[36]</a>, <a href="#pone.0001903-Woods1">[37]</a>, <a href="#pone.0001903-Christoffels1">[43]</a>, <a href="#pone.0001903-Vandepoele1">[45]</a>, <a href="#pone.0001903-Crollius1">[47]</a>, <a href="#pone.0001903-Brunet1">[52]</a>. In support of the above findings, analysis of the <em>T. rubripes</em> genome showed that orthologs for at least 33 of the 39 pairs of WGD-derived nGPCR duplicates are present in <em>T. rubripes</em> in spite of a lack of chromosomal localization information in this species (<a href="#pone.0001903.s004">Table S2 B</a>). Because duplicated genes in general degenerate within a few million years <a href="#pone.0001903-Lynch1">[3]</a>, these data thus indicate that nGPCR gene duplicates have a greater probability of escaping gene loss after WGD than average genes in the genome.</p>
<div class="figure" id="pone-0001903-g003"><div class="img"><a name="pone-0001903-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001903.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001903" data-uri="info:doi/10.1371/journal.pone.0001903.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001903.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001903.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001903.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001903.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001903.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001903.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001903.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001903.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001903.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Analysis of gene retention following whole genome duplication (WGD) in <em>Tetraodon nigroviridis</em>. A)</span></strong></p><a id="article1.body1.sec2.sec2.fig1.caption1.p1" name="article1.body1.sec2.sec2.fig1.caption1.p1"></a><p>Global distribution of 39 pairs of WGD-derived nGPCR duplicates on <em>T. nigroviridis</em> chromosomes (upper panel). Pairs of duplicates on two syntenic chromosome regions are indicated by green lines, similar to the distribution of WGD-derived syntenic regions mapped previously <a href="#pone.0001903-Jaillon1">[36]</a>. Schematic representation of the WGD-derived <em>GPR61</em> duplicates on chromosomes 1 and 9 of <em>T. nigroviridis</em> (lower panel). The duplicates are flanked by different sets of genes found on the syntenic region of human chromosome 1 in a disequilibrate manner. Another pair of WGD-derived genes in the nearby regions, Alivin-2, also is indicated. B) The retention rate of WGD-derived nGPCRs duplicates (23.5%) is significantly higher than a high estimate (15%) for the entire genome (t-test, P = 0.00014)<a href="#pone.0001903-Jaillon1">[36]</a>, <a href="#pone.0001903-Woods1">[37]</a>, <a href="#pone.0001903-Christoffels1">[43]</a>, <a href="#pone.0001903-Vandepoele1">[45]</a>, <a href="#pone.0001903-Crollius1">[47]</a>, <a href="#pone.0001903-Brunet1">[52]</a>. To compare differences in gene retention rate, each nGPCR family was assigned with a fixed value, 0 for families with a singleton in <em>T. nigroviridis</em>, and 1 for those with WGD-derived duplicates in <em>T. nigroviridis</em>. *, significantly different from the expected value.</p>
<span>doi:10.1371/journal.pone.0001903.g003</span></div>

<h4>Preferential retention of nGPCRs that interact with ligands not encoded by a gene</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>Although no prior study has addressed the mechanisms for the preferential retention of genes, studies of genes from unicellular organisms and mammals suggested that gene duplicability could be associated with gene complexity and protein length <a href="#pone.0001903-Yang1">[12]</a>, <a href="#pone.0001903-He1">[13]</a>, <a href="#pone.0001903-Shiu1">[14]</a>. To investigate whether the preferential retention of nGPCRs following WGD could be associated with select molecular attributes of nGPCRs, we analyzed the relationships between retention rate and, 1) receptor length, 2) chemical properties of the cognate ligand, and 3) molecular weights (MWs) of ligands. Findings of a significant correlation between retention rate and one of these traits would not only further support the existence of preferentiality in gene retention, but also reveal the underlying mechanisms. Because data on the open reading frame (ORF) of human nGPCRs and their cognate ligands are more complete as compared to those of pufferfish, we used human counterparts as proxies for the analysis of receptor length and ligand size. First, our comparisons of the receptor length of nGPCRs showed that the average receptor length for all nGPCRs is 551±32 amino acids (Log<sub>2</sub> ORF = 8.89±0.04, N = 269; <a href="#pone.0001903.s006">Table S4 A</a>), and that there is a negligible difference in receptor length between nGPCRs with WGD-derived duplicates in <em>T. nigroviridis</em> (Log<sub>2</sub> ORF = 8.92±0.10, N = 39) and those with a singleton (Log<sub>2</sub> ORF = 8.91±0.06, N = 153). Thus, the increased retention of WGD-derived nGPCR duplicates is not associated with the length, or protein complexity, of the receptors.</p>
<a id="article1.body1.sec2.sec3.p2" name="article1.body1.sec2.sec3.p2"></a><p>Second, we analyzed the relationship between retention rate and the chemical properties of the cognate ligand. Earlier studies have shown that there is a strong correlation between the phylogeny of nGPCRs and the chemical properties of their cognate ligands–nGPCRs with close relatedness tend to interact with ligand(s) of similar chemical properties <a href="#pone.0001903-Vassilatis1">[19]</a>. To factor these two associated parameters (receptor phylogeny and ligand properties) into our analysis, we divided the nGPCRs into two separate groups: Group I included subclasses A1–A4 and class C receptors, the majority of which interact or potentially interact with a nonpeptide ligand (e.g., photon, ions, monoamine derivatives, lipophilic compounds, and nucleoside derivatives), and Group II included subclasses A5–A7, class B, and class F receptors, which primarily interact with a gene-encoded polypeptide(s). We found that Group I nGPCRs (27.8%; 25/90 families) have a significantly higher retention rate than the genome average (<a href="#pone-0001903-g004">Fig. 4A</a>; t-test, P = 0.00068) whereas the Group II receptors (18.4%; 14/76 families) did not. These data thus suggest that the preferential retention of nGPCRs as a group is a result of greater retention of Group I nGPCRs that interact with a nonpeptide ligand. This distinction in the retention rate of Group I and Group II nGPCRs is further reflected in analyses of ligand size and retention rate. Comparison of the MWs of ligands for all nGPCRs with a known ligand showed that the MWs of ligands for singleton nGPCRs (Log<sub>10</sub>MW = 3.01±0.09, N = 104; <a href="#pone-0001903-g004">Fig. 4B</a>; <a href="#pone.0001903.s006">Table S4 B</a>) are 60% greater than that of nGPCRs with WGD-derived duplicates in <em>T. nigroviridis</em> (Log<sub>10</sub>MW = 2.44±0.25, N = 26; P&lt;0.05), consistent with the finding that interaction with a small nonpeptide ligand could provide an opportunity for retention. Because duplicated nGPCRs with a nonpeptide ligand presumably could undergo sub-functionalization or neo-functionalization without concurrent genetic changes in their major interacting partner–a property not shared by the majority of genes–by default, they would have a greater chance of escaping random gene loss before acquiring a beneficial mutation.</p>
<div class="figure" id="pone-0001903-g004"><div class="img"><a name="pone-0001903-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001903.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001903" data-uri="info:doi/10.1371/journal.pone.0001903.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001903.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001903.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001903.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001903.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001903.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001903.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001903.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001903.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001903.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>Preferential retention of nGPCR duplicates with a nonpeptide ligand after WGD in teleosts. A)</span></strong></p><a id="article1.body1.sec2.sec3.fig1.caption1.p1" name="article1.body1.sec2.sec3.fig1.caption1.p1"></a><p>The retention rate of WGD-derived duplicates for nGPCRs with a nonpeptide ligand(s) (Group I, 27.8%), but not nGPCRs with a polypeptide ligand (Group II, 18.4%), is significantly higher than the estimate for the entire genome (t-test, P = 0.00068). The Group I receptors include nGPCRs of subclasses A1-A4 and class C whereas the Group II receptors include those belonging to subclasses A5–A7, class B, and class F. To compare differences in gene retention rate, each nGPCR family was assigned with a fixed value, 0 for families with a singleton in <em>T. nigroviridis</em>, and 1 for those with WGD-derived duplicates in <em>T. nigroviridis</em>. *, significantly different from the expected value 15%. B) The average MW (mean±SEM, Log<sub>10</sub> transformed) of the cognate ligands for nGPCRs with WGD-derived duplicates in <em>T. nigroviridis</em> (2.44±0.25) is significantly less than those of singleton (3.01±0.09) or the entire pool of nGPCRs (2.92±0.07). For statistical analysis, the MW of photons was arbitrarily set as one. *, significantly different from that of singleton nGPCRs.</p>
<span>doi:10.1371/journal.pone.0001903.g004</span></div>

<h4>Retention of WGD-derived polypeptide nGPCRs is facilitated by the co-evolution of WGD-derived ligand genes</h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>Aside from the proposition that preferentiality in nGPCR retention is associated with the chemical properties of a ligand, we investigated whether the retention of WGD-derived nGPCR duplicates could be effected by other selection forces because select subcategories of nGPCRs that interact with a polypeptide ligand (polypeptide nGPCRs) also exhibit a higher retention rate when compared to the average for the genome (e.g., 33.3% of class B nGPCR families with assigned chromosome localizations contain WGD-derived duplicates in <em>T. nigroviridis</em>). Because an extra set of cognate ligands that interact with polypeptide nGPCRs was also generated during WGD, the parallel duplication of ligand-receptor pairs potentially could provide opportunities for the evolution of novel signaling pathways and facilitate the retention of the duplicated ligand-receptor pairs. Therefore, if the pufferfish genome retained a higher number of WGD-derived cognate ligand-nGPCR pairs as compared to the estimate derived from whole genome analysis, the data would further support our hypothesis that nGPCR signaling pathways are favored for retention after WGD. To investigate this possibility, we searched and analyzed the chromosomal distribution of genes encoding high affinity polypeptide ligands of nGPCRs belonging to subclasses A5–A7 and class B in humans and <em>T. nigroviridis</em>. In this analysis, the ligands for class F nGPCRs were excluded because they are promiscuous in receptor interactions, and no clear cognate ligand-receptor pairs can be defined.</p>
<a id="article1.body1.sec2.sec4.p2" name="article1.body1.sec2.sec4.p2"></a><p>With the same approach that we used to identify nGPCR homologs and novel peptide hormones in earlier studies <a href="#pone.0001903-Hsu1">[53]</a>, <a href="#pone.0001903-Hsu2">[54]</a>, <a href="#pone.0001903-Hsu3">[55]</a>, <a href="#pone.0001903-Roh1">[56]</a>, <a href="#pone.0001903-Hsu4">[57]</a>, <a href="#pone.0001903-Hsu5">[58]</a>, we identified 118 human and 118 <em>T. nigroviridis</em> ligand genes that encoded polypeptide ligands for the 81 families of nGPCRs known to interact with a polypeptide ligand (<a href="#pone.0001903.s007">Tables S5 A and S5 B</a>). Based on syntenic mapping and sequence comparison, we inferred that these ligand genes were derived from 76 ancestral genes in the MRCA of tetrapods and teleosts, and that 17.3% of these ligand gene families (9 pairs out of 52 families of ligand genes with assigned chromosome localizations) in <em>T. nigroviridis</em> contained WGD-derived duplicates, a level of gene retention similar to that of the genome average (<a href="#pone.0001903.s007">Table S5 C</a>).</p>
<a id="article1.body1.sec2.sec4.p3" name="article1.body1.sec2.sec4.p3"></a><p>Based on the 15% estimate for gene retention in the entire genome, there is a 2.25% probability for parallel retention of any given pair of WGD-derived ligands and their WGD-derived cognate receptors assuming they evolved independently. Against this low probability, we found that in <em>T. nigroviridis</em> over 9.6% of WGD-derived ligand genes (5/52 families of ligand genes with assigned chromosome localizations; <em>NMB</em>, <em>RLN3</em>, <em>INSL5</em>, <em>CALCA</em>, and <em>ADM</em>) coevolved with four pairs of WGD-derived cognate receptor duplicates (8.9%; 4/45 families of polypeptide nGPCRs with assigned chromosome localizations; <em>GRPR</em>, <em>RLN3R1</em>, <em>RLN3R2</em>, and <em>CLR</em>), a rate three to fourfold that of random probability (<a href="#pone.0001903.s007">Table S5 C</a>; the binary distribution for WGD-derived <em>CALCA</em> and <em>ADM</em> duplicates on syntenic chromosome fragments is shown in <a href="#pone.0001903.s001">Fig. S1</a>). Importantly, these data also showed that 55.6% (5/9) of ligand families and 44.4% (4/9) of polypeptide nGPCR families with WGD-derived duplicates in <em>T. nigroviridis</em> coevolved with a pair of WGD-derived partners. In retrospect, the 8.9~9.6% retention rate of WGD-derived cognate ligand-receptor pairs observed would require a 29.8~31% retention rate for all WGD-derived genes in the entire genome, a high level not compatible with any previous study <a href="#pone.0001903-Jaillon1">[36]</a>, <a href="#pone.0001903-Woods1">[37]</a>, <a href="#pone.0001903-Brunet1">[52]</a>. Therefore, the most parsimonious evolutionary course for the observation is that parallel duplication of polypeptide nGPCRs and their cognate ligand genes by WGD was crucial to allow the retention of WGD-derived ligand-receptor pairs. Of importance, these data also further support the hypothesis that nonpeptide nGPCR duplicates were preferentially retained after WGD as a result of low genetic constraint. Whereas the underlying mechanisms for the co-retention of WGD-derived ligand-receptor pairs remain to be investigated, these WGD-derived ligand-nGPCR pairs could evolve in a manner similar to the “divergent resolution” model that was proposed to illustrate the separation of different copies of a duplicated gene in allopatric populations during sympatric evolution <a href="#pone.0001903-Taylor4">[59]</a>. In this scenario, fitness associated with increased signal-to-noise ratio of the two diverging WGD-derived co-orthologus ligand-receptor pairs in individuals was selected, similar to the retention of a different copy of duplicated genes in reproductively separated populations <a href="#pone.0001903-Taylor4">[59]</a>.</p>


<h4>WGD-derived nGPCR duplicates underwent drastic divergence in the functional domain</h4>
<a id="article1.body1.sec2.sec5.p1" name="article1.body1.sec2.sec5.p1"></a><p>In addition to the above, we have observed that WGD-derived nGPCR duplicates generally exhibit a low degree of sequence similarity to each other, suggesting a trend of asymmetric divergence in these co-orthologs. To investigate whether nGPCR duplicates exhibit an accelerated divergence that could serve new functions, we compared the sequence divergence of the two WGD-derived duplicates. On average, pufferfish nGPCRs share 71.1–71.8% sequence similarity with human orthologs (<a href="#pone.0001903.s002">Fig. S2</a>). These estimates are similar to those of the entire proteomes among these species; therefore, nGPCRs as a group evolved at a pace similar to that of the rest of the proteome <a href="#pone.0001903-Jaillon1">[36]</a>. However, a distance tree calculated from the concatenated sequences of the 39 families of nGPCRs with WGD-derived duplicates in <em>T. nigroviridis</em> showed that the two WGD-derived co-orthologs are farther from each other as compared to the distance to human orthologs (<a href="#pone-0001903-g005">Fig. 5</a>). These results are reminiscent of that reported for WGD-derived gene duplicates in yeast <a href="#pone.0001903-Kellis1">[51]</a>, and suggest that the WGD-derived nGPCR duplicates evolved via sub-neofunctionalization in which one copy of duplicates would undergo positive selection and evolve faster than the other <a href="#pone.0001903-Kellis1">[51]</a>.</p>
<div class="figure" id="pone-0001903-g005"><div class="img"><a name="pone-0001903-g005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001903.g005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001903" data-uri="info:doi/10.1371/journal.pone.0001903.g005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001903.g005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001903.g005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001903.g005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001903.g005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001903.g005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001903.g005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001903.g005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001903.g005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001903.g005.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 5.  <span>Sequence divergence of WGD-derived nGPCR duplicates in <em>T. nigroviridis</em>.</span></strong></p><a id="article1.body1.sec2.sec5.fig1.caption1.p1" name="article1.body1.sec2.sec5.fig1.caption1.p1"></a><p>Global distance tree based on concatenated sequences of the 39 families of nGPCRs with WGD-derived duplicates in <em>T. nigroviridis.</em> Distances are indicated next to individual branches. The tree was calculated using a Gonnet 250 matrix. Each pair of WGD-derived duplicates was subdivided into two subgroups (the conserved copy 1 and the divergent copy 2) and analyzed separately. For nGPCR families with more than one ortholog in tetrapods, one was chosen randomly for analysis.</p>
<span>doi:10.1371/journal.pone.0001903.g005</span></div>

<h4>GPCR signaling is a favored evolutionary path</h4>
<a id="article1.body1.sec2.sec6.p1" name="article1.body1.sec2.sec6.p1"></a><p>By analyzing the fate of orthologous genes of nGPCRs and their cognate ligands in vertebrates, we demonstrated that nGPCR signaling has been a favored evolutionary path in a natural experiment conducted over the past 230 million years. Importantly, our studies satisfy several requirements for demonstrating the preferential retention of genes, rather than an increased gene duplicability, during evolution. In addition to the revelation that given an equal opportunity for duplication, a higher probability for retention could be realized, at least in the realm of nGPCRs, we showed that this greater probability could be due to interaction with ligands not encoded by a gene.</p>
<a id="article1.body1.sec2.sec6.p2" name="article1.body1.sec2.sec6.p2"></a><p>Based on these findings, we speculate that a lower genetic constraint associated with a nonpeptide ligand, together with the unidirectional signaling characteristics, could allow the duplicated nGPCRs to survive a longer period of selection before acquiring beneficial mutations as compared to an intracellular polypeptide which normally forms complexes with many partners in two-way communication (<a href="#pone-0001903-g006">Fig. 6A</a>). Mutations of either the transactivation or the functional domain could then lead to the generation of novel unidirectional intercellular signaling circuits among cells; the new circuits include signaling to the same cell population but with different pharmacological characteristics or to a different cell population (<a href="#pone-0001903-g006">Fig. 6A</a>). Genetically, a lower constraint associated with these characteristics could allow nGPCR genes to better tolerate deleterious random mutations and accumulate beneficial mutations, thus allowing nGPCR duplicates to be fixed with a higher probability as compared to genes with average constraint (<a href="#pone-0001903-g006">Fig. 6B</a>). This hypothesis is compatible with the concepts that, 1) gene duplicability in unicellular organisms increases when the number of subunits in a protein complex decreases <a href="#pone.0001903-Jeong1">[9]</a>, <a href="#pone.0001903-Yang1">[12]</a>, 2) a major portion of young genes exhibiting positive selection as calculated by the Ka/Ks ratio are genes involved in transient intercellular interactions such as defense, gamete interaction, or immunity against exogenous agents <a href="#pone.0001903-Jaillon1">[36]</a>, <a href="#pone.0001903-HuertaCepas1">[60]</a>, <a href="#pone.0001903-Swanson1">[61]</a>, <a href="#pone.0001903-Kocher1">[62]</a>, 3) major lineage-specific duplicated genes in mammals are genes that function in immunity, chemosensory, and reproduction <a href="#pone.0001903-Goodstadt1">[32]</a>, <a href="#pone.0001903-LindbladToh1">[63]</a>, and 4) single-nucleotide polymorphisms are more often found in GPCR genes as compared with non-GPCR genes <a href="#pone.0001903-Small1">[64]</a>. In addition, because the preferentiality in nGPCR retention encompasses nGPCRs that interact with a variety of ligands, our data would reject alternative hypotheses regarding the large inventory of WGD-derived nGPCRs in teleosts, such as it being a consequence of adaptation to specific environmental factors surrounding the time of WGD in the teleost ancestor, or the development of a particular physiological process that is specific to the evolution of teleosts. Furthermore, it is interesting to note that recent phylogenomics analyses indicated that protein families related to GPCR signaling pathways represent a major group of genes expanded before amniota and mammalian radiation, and that proteins involved in interaction with the environment (e.g., immune response and xenobiotic metabolism) expanded steadily through gene duplications at various points of vertebrate evolution <a href="#pone.0001903-HuertaCepas1">[60]</a>. Overall, the combined evidences support our hypothesis that nGPCR duplicates are preferentially retained after gene duplication and caution the inference of studies assuming different gene families were retained at a similar pace during evolution.</p>
<div class="figure" id="pone-0001903-g006"><div class="img"><a name="pone-0001903-g006" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001903.g006&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001903" data-uri="info:doi/10.1371/journal.pone.0001903.g006"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001903.g006&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001903.g006/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001903.g006/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001903.g006/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001903.g006/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001903.g006.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001903.g006/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001903.g006/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001903.g006.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 6.  <span>Putative mechanisms for the preferential retention of nGPCR genes after WGD. A)</span></strong></p><a id="article1.body1.sec2.sec6.fig1.caption1.p1" name="article1.body1.sec2.sec6.fig1.caption1.p1"></a><p>As compared to intracellular proteins that usually form complexes with many partners bi-directionally, a nonpeptide nGPCR, with part of its functional motifs dedicated to unidirectional interactions with an extracellular ligand, likely has a lower constraint on the divergence of the ligand-interaction domain <a href="#pone.0001903-Yang1">[12]</a>, <a href="#pone.0001903-He1">[13]</a>. Mutations of the transactivation or the functional domain could subsequently lead to the generation of novel signaling circuit to the same cell population but with different pharmacological characteristics (indicated by a red dashed line) or new signaling pathway to a different cell population (indicated by a cell with yellow color). B) Schematic representation of the fitness associated with nGPCRs that interact with nonpeptide ligands during natural selection. In the event of gene duplication, the low constraint derived from interacting with a nonpeptide ligand would provide nGPCR duplicates with a higher probability for acquiring novelty and becoming fixed in the descendent genome, as compared to genes with average constraint. A beneficial mutation could occur in either the <em>cis</em>-regulatory element or functional domain (indicated by a schematic pattern change). An X sign across the gene indicates gene loss. An xx sign indicates deleterious mutations for an average gene.</p>
<span>doi:10.1371/journal.pone.0001903.g006</span></div><a id="article1.body1.sec2.sec6.p3" name="article1.body1.sec2.sec6.p3"></a><p>Nonetheless, inasmuch as the lower genetic constraint hypothesis applies, the preferential retention of nGPCRs could be effected by a combination of selection forces. In addition to gamete compatibility, it is well recognized that differences in cognition and sensory perception could represent a particularly strong force leading to reproductive isolation. The provision of novel cognition and sensory perception pathways mediated by nGPCRs after WGD may constitute a rich source for adapting to new niches by providing the ability to adjust sensing, foraging, courtship, and other behaviors, without changes in the fundamental architecture of the cellular components, thereby leading to an enhanced retention of duplicated nGPCR genes <a href="#pone.0001903-Kocher1">[62]</a>, <a href="#pone.0001903-Terai1">[65]</a>. By the same token, we speculate that the same selection force underlying the preferential retention of nGPCRs after WGD may be the common denominator in the repeated expansion of nGPCRs and chemosensory 7TM receptors in different metazoan lineages <a href="#pone.0001903-Fredriksson1">[18]</a>, <a href="#pone.0001903-Chen1">[20]</a>, <a href="#pone.0001903-Gimelbrant1">[26]</a>, <a href="#pone.0001903-Rodriguez1">[29]</a>.</p>
<a id="article1.body1.sec2.sec6.p4" name="article1.body1.sec2.sec6.p4"></a><p>Finally, our studies generally validate century-old comparative endocrinology studies indicating all vertebrates share a similar set of hormones and receptors for cognition, sensation, and humoral homeostasis maintenance. However, the revelation that more than 280 nGPCRs and over 25 polypeptide ligand genes in teleosts are lineage-specific paralogs indicates that nGPCR-mediated regulatory circuits in teleosts have evolved with a remodeled platform, and points to the presence of a robust intercellular signaling network involving hundreds of novel ligand-receptor signaling pathways not found in tetrapods <a href="#pone.0001903-Kocher1">[62]</a>.</p>

</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4>Nomenclature</h4>
<a id="article1.body1.sec3.sec1.p1" name="article1.body1.sec3.sec1.p1"></a><p>We used the GPCR classification proposed by Bockaert and Pin <a href="#pone.0001903-Bockaert1">[23]</a> and Vassilatis et al. <a href="#pone.0001903-Vassilatis1">[19]</a>. Human nGPCRs were named according to the recommendation of the International Union of Pharmacology <a href="#pone.0001903-Foord1">[22]</a>, and each family of orthologous receptors is denoted by the name of the human ortholog(s).</p>


<h4>Protein and genomic sequence data</h4>
<a id="article1.body1.sec3.sec2.p1" name="article1.body1.sec3.sec2.p1"></a><p>Human and rodent nGPCR sequences were obtained from the HPMR database, <a href="http://receptome.stanford.edu/hpmr/home.asp">http://receptome.stanford.edu/hpmr/home.​asp</a> <a href="#pone.0001903-BenShlomo1">[17]</a>, and the NCBI databases <a href="http://ftp://ftp.ncbi.nlm.nih.gov/genomes">ftp://ftp.ncbi.nlm.nih.gov/genomes</a>. Chicken genomic and protein sequences were downloaded from the NCBI ftp site, <a href="http://ftp://ftp.ncbi.nlm.nih.gov/genomes/Gallus_gallus/">ftp://ftp.ncbi.nlm.nih.gov/genomes/Gallu​s_gallus/</a> <a href="#pone.0001903-Consortium1">[66]</a>. The <em>T. rubripes</em> proteome and genome sequences were obtained from the JGI database, <a href="http://genome.jgi-psf.org">http://genome.jgi-psf.org</a> <a href="#pone.0001903-Aparicio1">[67]</a>. The <em>T. nigroviridis</em> genomic and protein sequences were obtained from the Genoscope database, <a href="http://www.genoscope.cns.fr">http://www.genoscope.cns.fr</a> <a href="#pone.0001903-Jaillon1">[36]</a>.</p>


<h4>Determination of orthologous and co-orthologous relationships</h4>
<a id="article1.body1.sec3.sec3.p1" name="article1.body1.sec3.sec3.p1"></a><p>Orthologous genes belonging to an nGPCR family from different species were determined by a series of reciprocal pairwise sequence comparisons using the BLAST server <a href="#pone.0001903-Goodstadt1">[32]</a>, <a href="#pone.0001903-Altschul1">[68]</a>, <a href="#pone.0001903-Schaffer1">[69]</a> and syntenic mapping. Initially, human and mouse nGPCR sequences were compared against the proteomes of rat, chicken, <em>T. nigroviridis,</em> and <em>T. rubripes</em>. The top thirty nonredundant hits were collected. Unique protein sequences with E&lt;0.0001 were analyzed with additional blast searches against the human nGPCR dataset to detect the best reciprocal hits. Sequences that contained erroneous components from a neighboring gene were trimmed manually to obtain a continuous nGPCR ORF. The best hits were then collected and verified by blast searches against a human chemosensory GPCR dataset to exclude orthologs for olfactory GPCRs, vomeronasal receptor-like genes, taste receptors, and pheromone receptors from further analysis. In addition, proteins with a 7TM domain but do not share a common root with classes A, B, C, or F nGPCRs were excluded from analysis. Because GPCRs belong to each of the above-mentioned GPCR groups exhibit a distinct sequence profile, nGPCRs of various vertebrate species can be identified unambiguously using this procedure <a href="#pone.0001903-Vassilatis1">[19]</a>, <a href="#pone.0001903-Foord1">[22]</a>.</p>
<a id="article1.body1.sec3.sec3.p2" name="article1.body1.sec3.sec3.p2"></a><p>For human nGPCRs where orthologs were not found in the protein databases of other species, the nGPCRs were analyzed with blast searches against genome sequences using the TBLASTN. Similar to studies of proteomes, the thirty best genomic hits were collected. Unique genomic sequences with E&lt;0.0001 were then verified by blast searches against the human GPCR dataset. Identities of the genes encoded by genomic hits were further verified by blast searches against the nr database in GenBank in order to exclude nonGPCR or chemosensory GPCR genes. Sequence similarity between orthologous or co-orthologous nGPCRs was generated by the NCBI bl2seq program on a local server using default settings without a filter <a href="#pone.0001903-Tatusova1">[70]</a>.</p>


<h4>Phylogenetic reconstruction</h4>
<a id="article1.body1.sec3.sec4.p1" name="article1.body1.sec3.sec4.p1"></a><p>Unlike olfactory and vomeronasal receptors which expanded repeatedly in select vertebrate lineages, most nGPCR families originated before the evolution of euteleostomi species and contain an orhtolog or a small number of paralogs in most vertebrates. Based on the best reciprocal hit approach, we determined that &gt;60% of nGPCR families contain one ortholog in different tetrapods (<a href="#pone.0001903.s002">Table S2 E</a>). However, the evolutionary history of &gt;60% families of teleost nGPCRs cannot be resolved with this approach. To determine the evolutionary relationship of orthologous nGPCRs in each nGPCR family or within a subgroup of nGPCR families as well as concatenated sequences, we used the the ClustalW multiple sequence alignment program version 1.82 (<a href="http://www.ebi.ac.uk/clustalw/#">http://www.ebi.ac.uk/clustalw/#</a>) <a href="#pone.0001903-Thompson1">[71]</a>. The phylogenetic reconstruction was based on the Neighbor-Joining (NJ) method <a href="#pone.0001903-Saitou1">[72]</a>. Phylograms were first built with a default parameter (DNA Gap Open Penalty = 15.0, DNA Gap Extension Penalty = 6.66, DNA Matrix = Identity, Protein Gap Open Penalty = 10.0, Protein Gap Extension Penalty = 0.2, Protein Matrix = Gonnet, Protein/DNA ENDGAP = −1, Protein/DNA GAPDIST = 4). For families with multiple paralogs in select species, additional trees were reconstructed using the BLOSUM30 and PAM models as well as the drawtree program of PHYLIP3.65 package (<a href="http://evolution.genetics.washington.edu/phylip/getme.html">http://evolution.genetics.washington.edu​/phylip/getme.html</a>) <a href="#pone.0001903-Felsenstein1">[73]</a>. If a sequence was found to be positioned outside a main branch consisting of a group of orthologs from teleosts to humans, the sequence was then analyzed together with the next closest related nGPCR groups in an iterated manner until a best fit family was identified. Each of these independent nGPCR families was considered to be derived from an independent ancestral nGPCR rooted in the MRCA of tetrapods and teleosts. These analyses showed that orthologs from most nGPCR families share on average &gt;70% sequence similarity, and most trees share a topology similar to that of concatenated sequences as shown in <a href="#pone-0001903-g005">Fig. 5</a>.</p>
<a id="article1.body1.sec3.sec4.p2" name="article1.body1.sec3.sec4.p2"></a><p>However, preliminary phylogenetic reconstruction studies showed that select WGD-derived duplicates of pufferfish have a basal position relative to the other WGD-derived co-ortholog in the phylogenetic tree, suggesting that gene phylogenies are insufficient to resolve the evolution history of WGD-derived co-orthologs in these nGPCR families. Instead of attributing to a massive gene loss in multiple Classes of tetrapods, we reasoned that the most parsimonious inference would be that the WGD-derived co-orthologs underwent neo-functionalization or sub-functionalization, and that heterotachy incurred by functional divergence led to the aberrant tree topology <a href="#pone.0001903-Thornton1">[15]</a>. Therefore, we sought to determine the phylogenetic relationship of all nGPCR families with syntenic mapping.</p>


<h4>Identification of WGD-derived and tandem duplication-derived nGPCRs</h4>
<a id="article1.body1.sec3.sec5.p1" name="article1.body1.sec3.sec5.p1"></a><p>Chromosomal localization of tetrapod nGPCRs was obtained from the NCBI database. Syntenic maps were downloaded from the Genoscope database (<a href="http://www.genoscope.cns.fr/externe/English/Projets/Projet_C/data/synteny/TN_HS_SYNT">http://www.genoscope.cns.fr/externe/Engl​ish/Projets/Projet_C/data/synteny/TN_HS_​SYNT</a>)<a href="#pone.0001903-Jaillon1">[36]</a> and Ensembl's BioMart data mining tool (<a href="http://www.ensembl.org/multi/martview">http://www.ensembl.org/multi/martview</a>)<a href="#pone.0001903-Hubbard1">[74]</a>, <a href="#pone.0001903-Kasprzyk1">[75]</a>. The exact locations for human and <em>T. nigroviridis</em> co-orthologs were also verified by BLAT searches using the UCSC Genome Bioinformatics webserver (<a href="http://genome.ucsc.edu/cgi-bin/hgBlat">http://genome.ucsc.edu/cgi-bin/hgBlat</a>)<a href="#pone.0001903-Kent1">[76]</a>. We inferred that a pair of duplicates would be WGD-derived co-orthologs if they were located on human-<em>T. nigroviridis</em> syntenic chromosomal regions. In these analyses, locations of <em>T. nigroviridis</em> genes were identified first using the Genoscope map, and then verified with a recently refined map in the Ensemble database. In contrast, nGPCRs found on neighboring loci on the same chromosome were determined to be derived from tandem duplications. Therefore, the presence of an ancestor for a select group of nGPCRs in the MRCA was deduced from analyses combining phylogenetic trees, BLAST results, and syntenic mapping. Based on these analyses, a total of 269 clusters of orthologous nGPCRs, belonging to A1–A7, B, C, and F classes, were obtained (<a href="#pone.0001903.s002">Table S2 A</a>). However, we cannot exclude the possibility, albeit at a low probability, that some teleost homologs found on syntenic chromosomal regions were not WGD-derived co-orthologs.</p>


<h4>Identification of WGD-derived ligand genes</h4>
<a id="article1.body1.sec3.sec6.p1" name="article1.body1.sec3.sec6.p1"></a><p>The cognate ligand genes for polypeptide nGPCRs in humans and <em>T. nigroviridis</em> were identified by BLAT searches using mature regions of human ligands as the query. Positive hits were then manually sorted. To validate the authenticity of a ligand gene from <em>T. nigroviridis,</em> we compared the target sequences to orthologous sequences from all model vertebrate organisms in GenBank. Only sequences that contained the characteristic sequence motifs of the mature region of a given ligand were considered as a positive ortholog <a href="#pone.0001903-Roh1">[56]</a>, <a href="#pone.0001903-Hsu5">[58]</a>. We determined that a pair of ligand duplicates would be WGD-derived co-orthologs in a manner similar to that described for the nGPCR duplicates. Likewise, phylogenies of ligand genes were analyzed similar to that described for nGPCRs. The major difference is the inclusion of only putative mature regions of ligands in these analyses because the prepro-regions of peptide ligands were known to evolve with minimal selection constraints and diverge greatly among closely related species.</p>


<h4>Receptor length and molecular weight of nGPCR ligands</h4>
<a id="article1.body1.sec3.sec7.p1" name="article1.body1.sec3.sec7.p1"></a><p>The length of human nGPCR ORF and the molecular weight of the cognate ligand(s) for human nGPCRs were obtained from the NCBI database and the literature by manual searches. In cases with more than one cognate ligand for a given nGPCR, the most potent ligand was used for analysis.</p>


<h4>Statistical analyses</h4>
<a id="article1.body1.sec3.sec8.p1" name="article1.body1.sec3.sec8.p1"></a><p>Statistical analyses including t-test and ANOVA were performed using a Prism software package (GraphPad Software, Inc., San Diego, CA). To compare differences in gene retention rate, nGPCR and ligand gene families with a singleton or WGD-derived duplicates in <em>T. nigroviridis</em> were assigned with a fixed value, 0 for families with a singleton, and 1 for those with WGD-derived duplicates. The expected rate for retention of WGD-derived duplicates in <em>T. nigroviridis</em> was set at a high estimate (15%) based on several previous studies <a href="#pone.0001903-Jaillon1">[36]</a>, <a href="#pone.0001903-Woods1">[37]</a>, <a href="#pone.0001903-Brunet1">[52]</a>.</p>

</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0001903"></div><a name="pone.0001903.s001" id="pone.0001903.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001903.s001">Figure S1. </a></strong></p><a id="article1.body1.sec4.supplementary-material1.caption1.p1" name="article1.body1.sec4.supplementary-material1.caption1.p1"></a><p class="preSiDOI">Localization of WGD-derived adrenomedullin (ADM) and calcitonin/CGRP (CALCA) gene duplicates on syntenic regions of chromosomes 5 and 13 of T. nigroviridis.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001903.s001</p><a id="article1.body1.sec4.supplementary-material1.caption1.p2" name="article1.body1.sec4.supplementary-material1.caption1.p2"></a><p class="postSiDOI">(0.01 MB PDF)</p>
<a name="pone.0001903.s002" id="pone.0001903.s002"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001903.s002">Figure S2. </a></strong></p><a id="article1.body1.sec4.supplementary-material2.caption1.p1" name="article1.body1.sec4.supplementary-material2.caption1.p1"></a><p class="preSiDOI">Comparison of sequence similarity between human nGPCRs and orthologs or co-orthologs from rat, mouse, chicken, T. nigroviridis, and T. rubripes. Each data point represents the average of multiple data points belonging to brackets increased by a 5% step in sequence similarity as shown on the Y axis. The average sequence similarity and sequence identity (mean±SEM) between all human nGPCRs and their orthologs or co-orthologs in other species are shown in the lower panel.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001903.s002</p><a id="article1.body1.sec4.supplementary-material2.caption1.p2" name="article1.body1.sec4.supplementary-material2.caption1.p2"></a><p class="postSiDOI">(0.01 MB PDF)</p>
<a name="pone.0001903.s003" id="pone.0001903.s003"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001903.s003">Table S1. </a></strong></p><a id="article1.body1.sec4.supplementary-material3.caption1.p1" name="article1.body1.sec4.supplementary-material3.caption1.p1"></a><p class="preSiDOI">A. Inventory of nGPCR genes in human, rat, mouse, chicken, T. nigroviridis, and T. rubripes. B. List of nGPCR gene inventories in human, rat, mouse, chicken, T. nigroviridis, and T. rubripes. The accession number of individual nGPCRs in each species is listed according to their classification. For human nGPCRs, the gene ID is provided.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001903.s003</p><a id="article1.body1.sec4.supplementary-material3.caption1.p2" name="article1.body1.sec4.supplementary-material3.caption1.p2"></a><p class="postSiDOI">(0.07 MB PDF)</p>
<a name="pone.0001903.s004" id="pone.0001903.s004"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001903.s004">Table S2. </a></strong></p><a id="article1.body1.sec4.supplementary-material4.caption1.p1" name="article1.body1.sec4.supplementary-material4.caption1.p1"></a><p class="preSiDOI">A. Inventory of nGPCR genes in the MRCA of tetrapods and teleosts, and the number of derived nGPCR genes in human, rat, mouse, chicken, T. nigroviridis, and T. rubripes. The MRCA for each family of orthologous nGPCRs is defined as the gene that gave rise to a group of orthologs or co-orthologs in modern species. Each data point represents the total number of genes belonging to a select class or subclass of nGPCRs. The orthologous relationships of nGPCRs from different species are deduced by syntenic mapping and phylogenetic tree building analysis. B. List of nGPCR genes wherein a distinct evolutionary path can be traced from the MRCA of tetrapods and teleosts to modern species. The accession number of individual nGPCRs in each species is listed according to their classification. For human nGPCRs, the gene ID also is provided. Ancestral nGPCRs are denoted by the name of the human ortholog(s). C. List of nGPCR genes for which an ancestral form in the MRCA of tetrapods and teleosts cannot be defined. D. Correlation coefficients of nGPCR inventories between pairs of species. E. Number of singleton and duplicated nGPCR genes in model vertebrates, and number of nGPCR families with gene duplicates in each species.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001903.s004</p><a id="article1.body1.sec4.supplementary-material4.caption1.p2" name="article1.body1.sec4.supplementary-material4.caption1.p2"></a><p class="postSiDOI">(0.08 MB PDF)</p>
<a name="pone.0001903.s005" id="pone.0001903.s005"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001903.s005">Table S3. </a></strong></p><a id="article1.body1.sec4.supplementary-material5.caption1.p1" name="article1.body1.sec4.supplementary-material5.caption1.p1"></a><p class="preSiDOI">A. Tandem duplication-derived nGPCRs of T. nigroviridis. Tandem duplication-derived genes are defined as paralogous genes found on neighboring loci on the same chromosome. B. Summary of nGPCRs derived from tandem duplication in tetrapods. Tandem duplication-derived genes are defined as paralogous genes found on neighboring loci on the same chromosome. C. List of tandem duplication-derived nGPCRs of tetrapods. The putative origins of these tandem duplication-derived paralogs are indicated on the left column. D. WGD-derived nGPCRs of T. nigroviridis. WGD-derived genes are defined as co-orthologous genes found on WGD-derived syntenic regions on different chromosomes of pufferfish. E. Schematic representation of the WGD-derived GPR61 duplicates on T. nigroviridis chromosomes 9 and 11. The position of WGD-derived GPR61 and neighboring Alivin-2 genes on T. nigroviridis chromosomes as well as their orthologs on human chromosome 1 are indicated by italicized letters.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001903.s005</p><a id="article1.body1.sec4.supplementary-material5.caption1.p2" name="article1.body1.sec4.supplementary-material5.caption1.p2"></a><p class="postSiDOI">(0.03 MB PDF)</p>
<a name="pone.0001903.s006" id="pone.0001903.s006"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001903.s006">Table S4. </a></strong></p><a id="article1.body1.sec4.supplementary-material6.caption1.p1" name="article1.body1.sec4.supplementary-material6.caption1.p1"></a><p class="preSiDOI">A. List of the open reading frame (ORF) lengths of representative human receptors in each of the 269 nGPCR families inferred in the MRCA of tetrapods and teleosts. The lengths of human nGPCR ORFs were obtained from GenBank, and the majority of these ORFs have been defined experimentally. B. List of MWs of cognate ligands for each of the 190 families of nGPCRs with a known ligand(s). WGD, whole genome duplication; TD, tandem duplication; ND, not detected in T. nigroviridis; UD, undetermined; S, singleton.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001903.s006</p><a id="article1.body1.sec4.supplementary-material6.caption1.p2" name="article1.body1.sec4.supplementary-material6.caption1.p2"></a><p class="postSiDOI">(0.03 MB PDF)</p>
<a name="pone.0001903.s007" id="pone.0001903.s007"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001903.s007">Table S5. </a></strong></p><a id="article1.body1.sec4.supplementary-material7.caption1.p1" name="article1.body1.sec4.supplementary-material7.caption1.p1"></a><p class="preSiDOI">A. Inventory of polypeptide ligand genes in the MRCA of tetrapods and teleosts, human, and T. nigroviridis. B. List of polypeptide ligand genes in the MRCA of tetrapods and teleosts, and the derived ligand genes in human and T. nigroviridis. The accession number of identified ligands is listed. In cases where no existing accession number is available, the chromosomal position of the identified gene is provided. WGD, whole genome duplication; UD, undetermined; S, singleton. C. List of polypeptide ligands with WGD-derived duplicates in T. nigroviridis as well as their cognate receptors. Cognate receptors with WGD-derived duplicates in T. nigroviridis are shown in bold letters. The accession numbers for identified ligands are listed. In cases where no existing accession number is available, the chromosomal position of the identified gene is provided.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001903.s007</p><a id="article1.body1.sec4.supplementary-material7.caption1.p2" name="article1.body1.sec4.supplementary-material7.caption1.p2"></a><p class="postSiDOI">(0.04 MB PDF)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>We thank Caren Spencer and Rami Rauch for editorial and technical assistance. We also thank Drs. Aaron Hsueh, Marco Conti (Department of Obstetrics and Gynecology) and Brian Kobilka (Department of Molecular and Cellular Physiology) of Stanford University, Frederick W. Goetz (Great Lakes WATER Institute, University of Wisconsin-Milwaukee), and Jared C. Roach (Institute for Systems Biology, Seattle, WA) for comments on the manuscript. We also thank Drs. Linda Giudice (Dept. of Obstetrics, Gynecology and Reproductive Sciences, UCSF) and Jonathan S. Berek (Dept. of OB/GYN, Stanford University) for the encouragement. CL Chang also thanks Dr. Yung-Kuei Soong (Department of OB/GYN, Chang Gung Memorial Hospital, Taiwan) for the support and encouragement.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: SH CC. Performed the experiments: SH JP JS. Analyzed the data: SH JP CC JS. Contributed reagents/materials/analysis tools: SH. Wrote the paper: SH JS.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0001903-Courseaux1" id="pone.0001903-Courseaux1"></a>Courseaux A, Nahon JL (2001) Birth of two chimeric genes in the Hominidae lineage. Science  291: 1293–1297.  <ul class="find" data-citedArticleID="1083978" data-doi="10.1126/science.1057284"><li><a href="http://dx.doi.org/10.1126/science.1057284" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Birth+of+two+chimeric+genes+in+the+Hominidae+lineage." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Birth+of+two+chimeric+genes+in+the+Hominidae+lineage.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0001903-Taylor1" id="pone.0001903-Taylor1"></a>Taylor JS, Raes J (2004) Duplication and divergence: the evolution of new genes and old ideas. Annu Rev Genet  38: 615–643.  <ul class="find" data-citedArticleID="1084078" data-doi="10.1146/annurev.genet.38.072902.092831"><li><a href="http://dx.doi.org/10.1146/annurev.genet.38.072902.092831" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Duplication+and+divergence%3A+the+evolution+of+new+genes+and+old+ideas." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Duplication+and+divergence%3A+the+evolution+of+new+genes+and+old+ideas.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0001903-Lynch1" id="pone.0001903-Lynch1"></a>Lynch M, Conery JS (2000) The evolutionary fate and consequences of duplicate genes. Science  290: 1151–1155.  <ul class="find" data-citedArticleID="1084046" data-doi="10.1126/science.290.5494.1151"><li><a href="http://dx.doi.org/10.1126/science.290.5494.1151" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+evolutionary+fate+and+consequences+of+duplicate+genes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+evolutionary+fate+and+consequences+of+duplicate+genes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0001903-Ohno1" id="pone.0001903-Ohno1"></a>Ohno S (1970) Evolution by gene duplication: Springer-Verlag, Heidelberg, Germany.   <ul class="find-nolinks"></ul></li><li><span class="label">5.
              </span><a name="pone.0001903-Force1" id="pone.0001903-Force1"></a>Force A, Lynch M, Pickett FB, Amores A, Yan YL, et al.  (1999) Preservation of duplicate genes by complementary, degenerative mutations. Genetics  151: 1531–1545.  <ul class="find" data-citedArticleID="1083988"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Preservation+of+duplicate+genes+by+complementary%2C+degenerative+mutations.&amp;auth=&amp;atitle=Preservation+of+duplicate+genes+by+complementary%2C+degenerative+mutations." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Preservation+of+duplicate+genes+by+complementary%2C+degenerative+mutations." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Preservation+of+duplicate+genes+by+complementary%2C+degenerative+mutations.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0001903-Hughes1" id="pone.0001903-Hughes1"></a>Hughes AL (2005) Gene duplication and the origin of novel proteins. Proc Natl Acad Sci U S A  102: 8791–8792.  <ul class="find" data-citedArticleID="1084028" data-doi="10.1073/pnas.0503922102"><li><a href="http://dx.doi.org/10.1073/pnas.0503922102" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Gene+duplication+and+the+origin+of+novel+proteins." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Gene+duplication+and+the+origin+of+novel+proteins.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0001903-Francino1" id="pone.0001903-Francino1"></a>Francino MP (2005) An adaptive radiation model for the origin of new gene functions. Nat Genet  37: 573–577.  <ul class="find" data-citedArticleID="1083990" data-doi="10.1038/ng1579"><li><a href="http://dx.doi.org/10.1038/ng1579" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=An+adaptive+radiation+model+for+the+origin+of+new+gene+functions." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22An+adaptive+radiation+model+for+the+origin+of+new+gene+functions.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0001903-Yokoyama1" id="pone.0001903-Yokoyama1"></a>Yokoyama S (2002) Evaluating adaptive evolution. Nat Genet  30: 350–351.  <ul class="find" data-citedArticleID="1084104" data-doi="10.1038/ng0402-350"><li><a href="http://dx.doi.org/10.1038/ng0402-350" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evaluating+adaptive+evolution." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evaluating+adaptive+evolution.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0001903-Jeong1" id="pone.0001903-Jeong1"></a>Jeong H, Mason SP, Barabasi AL, Oltvai ZN (2001) Lethality and centrality in protein networks. Nature  411: 41–42.  <ul class="find" data-citedArticleID="1084032" data-doi="10.1038/35075138"><li><a href="http://dx.doi.org/10.1038/35075138" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Lethality+and+centrality+in+protein+networks." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Lethality+and+centrality+in+protein+networks.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0001903-Wagner1" id="pone.0001903-Wagner1"></a>Wagner A (2000) Robustness against mutations in genetic networks of yeast. Nat Genet  24: 355–361.  <ul class="find" data-citedArticleID="1084098"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Robustness+against+mutations+in+genetic+networks+of+yeast.&amp;auth=&amp;atitle=Robustness+against+mutations+in+genetic+networks+of+yeast." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Robustness+against+mutations+in+genetic+networks+of+yeast." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Robustness+against+mutations+in+genetic+networks+of+yeast.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0001903-Gu1" id="pone.0001903-Gu1"></a>Gu Z, Steinmetz LM, Gu X, Scharfe C, Davis RW, et al.  (2003) Role of duplicate genes in genetic robustness against null mutations. Nature  421: 63–66.  <ul class="find" data-citedArticleID="1084008" data-doi="10.1038/nature01198"><li><a href="http://dx.doi.org/10.1038/nature01198" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Role+of+duplicate+genes+in+genetic+robustness+against+null+mutations." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Role+of+duplicate+genes+in+genetic+robustness+against+null+mutations.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0001903-Yang1" id="pone.0001903-Yang1"></a>Yang J, Lusk R, Li WH (2003) Organismal complexity, protein complexity, and gene duplicability. Proc Natl Acad Sci U S A  100: 15661–15665.  <ul class="find" data-citedArticleID="1084102" data-doi="10.1073/pnas.2536672100"><li><a href="http://dx.doi.org/10.1073/pnas.2536672100" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Organismal+complexity%2C+protein+complexity%2C+and+gene+duplicability." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Organismal+complexity%2C+protein+complexity%2C+and+gene+duplicability.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0001903-He1" id="pone.0001903-He1"></a>He X, Zhang J (2005) Gene complexity and gene duplicability. Curr Biol  15: 1016–1021.  <ul class="find" data-citedArticleID="1084010" data-doi="10.1016/j.cub.2005.04.035"><li><a href="http://dx.doi.org/10.1016/j.cub.2005.04.035" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Gene+complexity+and+gene+duplicability." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Gene+complexity+and+gene+duplicability.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0001903-Shiu1" id="pone.0001903-Shiu1"></a>Shiu SH, Byrnes JK, Pan R, Zhang P, Li WH (2006) Role of positive selection in the retention of duplicate genes in mammalian genomes. Proc Natl Acad Sci U S A  103: 2232–2236.  <ul class="find" data-citedArticleID="1084068" data-doi="10.1073/pnas.0510388103"><li><a href="http://dx.doi.org/10.1073/pnas.0510388103" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Role+of+positive+selection+in+the+retention+of+duplicate+genes+in+mammalian+genomes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Role+of+positive+selection+in+the+retention+of+duplicate+genes+in+mammalian+genomes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0001903-Thornton1" id="pone.0001903-Thornton1"></a>Thornton JW, Kolaczkowski B (2005) No magic pill for phylogenetic error. Trends Genet  21: 310–311.  <ul class="find" data-citedArticleID="1084090" data-doi="10.1016/j.tig.2005.04.002"><li><a href="http://dx.doi.org/10.1016/j.tig.2005.04.002" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=No+magic+pill+for+phylogenetic+error." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22No+magic+pill+for+phylogenetic+error.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0001903-Steel1" id="pone.0001903-Steel1"></a>Steel M (2005) Should phylogenetic models be trying to “fit an elephant”? Trends Genet  21: 307–309.  <ul class="find" data-citedArticleID="1084072" data-doi="10.1016/j.tig.2005.04.001"><li><a href="http://dx.doi.org/10.1016/j.tig.2005.04.001" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Should+phylogenetic+models+be+trying+to+%E2%80%9Cfit+an+elephant%E2%80%9D%3F" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Should+phylogenetic+models+be+trying+to+%E2%80%9Cfit+an+elephant%E2%80%9D%3F%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0001903-BenShlomo1" id="pone.0001903-BenShlomo1"></a>Ben-Shlomo I, Hsu SY, Rauch R, Kowalski HW, Hsueh AJW (2003) Signaling Receptome: A Genomic and Evolutionary Perspective of Plasma Membrane Receptors Involved in Signal Transduction. Science's STKE  2003: RE9.  <ul class="find" data-citedArticleID="1083964" data-doi="10.1126/stke.2003.187.re9"><li><a href="http://dx.doi.org/10.1126/stke.2003.187.re9" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Signaling+Receptome%3A+A+Genomic+and+Evolutionary+Perspective+of+Plasma+Membrane+Receptors+Involved+in+Signal+Transduction." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Signaling+Receptome%3A+A+Genomic+and+Evolutionary+Perspective+of+Plasma+Membrane+Receptors+Involved+in+Signal+Transduction.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0001903-Fredriksson1" id="pone.0001903-Fredriksson1"></a>Fredriksson R, Schioth HB (2005) The repertoire of G-protein-coupled receptors in fully sequenced genomes. Mol Pharmacol  67: 1414–1425.  <ul class="find" data-citedArticleID="1083992" data-doi="10.1124/mol.104.009001"><li><a href="http://dx.doi.org/10.1124/mol.104.009001" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+repertoire+of+G-protein-coupled+receptors+in+fully+sequenced+genomes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+repertoire+of+G-protein-coupled+receptors+in+fully+sequenced+genomes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0001903-Vassilatis1" id="pone.0001903-Vassilatis1"></a>Vassilatis DK, Hohmann JG, Zeng H, Li F, Ranchalis JE, et al.  (2003) The G protein-coupled receptor repertoires of human and mouse. Proc Natl Acad Sci U S A  100: 4903–4908.  <ul class="find" data-citedArticleID="1084096" data-doi="10.1073/pnas.0230374100"><li><a href="http://dx.doi.org/10.1073/pnas.0230374100" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+G+protein-coupled+receptor+repertoires+of+human+and+mouse." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+G+protein-coupled+receptor+repertoires+of+human+and+mouse.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0001903-Chen1" id="pone.0001903-Chen1"></a>Chen N, Pai S, Zhao Z, Mah A, Newbury R, et al.  (2005) Identification of a nematode chemosensory gene family. Proc Natl Acad Sci U S A  102: 146–151.  <ul class="find" data-citedArticleID="1083970" data-doi="10.1073/pnas.0408307102"><li><a href="http://dx.doi.org/10.1073/pnas.0408307102" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Identification+of+a+nematode+chemosensory+gene+family." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Identification+of+a+nematode+chemosensory+gene+family.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0001903-Fryxell1" id="pone.0001903-Fryxell1"></a>Fryxell KJ (1995) The evolutionary divergence of neurotransmitter receptors and second-messenger pathways. J Mol Evol  41: 85–97.  <ul class="find" data-citedArticleID="1083994" data-doi="10.1007/bf00174044"><li><a href="http://dx.doi.org/10.1007/bf00174044" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+evolutionary+divergence+of+neurotransmitter+receptors+and+second-messenger+pathways." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+evolutionary+divergence+of+neurotransmitter+receptors+and+second-messenger+pathways.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0001903-Foord1" id="pone.0001903-Foord1"></a>Foord SM, Bonner TI, Neubig RR, Rosser EM, Pin J-P, et al.  (2005) International Union of Pharmacology. XLVI. G Protein-Coupled Receptor List. Pharmacol Rev  57: 279–288.  <ul class="find" data-citedArticleID="1083986" data-doi="10.1124/pr.57.2.5"><li><a href="http://dx.doi.org/10.1124/pr.57.2.5" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=International+Union+of+Pharmacology.+XLVI.+G+Protein-Coupled+Receptor+List." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22International+Union+of+Pharmacology.+XLVI.+G+Protein-Coupled+Receptor+List.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0001903-Bockaert1" id="pone.0001903-Bockaert1"></a>Bockaert J, Pin JP (1999) Molecular tinkering of G protein-coupled receptors: an evolutionary success. EMBO J  18: 1723–1729.  <ul class="find" data-citedArticleID="1083966" data-doi="10.1093/emboj/18.7.1723"><li><a href="http://dx.doi.org/10.1093/emboj/18.7.1723" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Molecular+tinkering+of+G+protein-coupled+receptors%3A+an+evolutionary+success." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Molecular+tinkering+of+G+protein-coupled+receptors%3A+an+evolutionary+success.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0001903-Liberles1" id="pone.0001903-Liberles1"></a>Liberles SD, Buck LB (2006) A second class of chemosensory receptors in the olfactory epithelium. Nature  442: 645–650.  <ul class="find" data-citedArticleID="1084042" data-doi="10.1038/nature05066"><li><a href="http://dx.doi.org/10.1038/nature05066" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+second+class+of+chemosensory+receptors+in+the+olfactory+epithelium." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+second+class+of+chemosensory+receptors+in+the+olfactory+epithelium.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0001903-Grus1" id="pone.0001903-Grus1"></a>Grus WE, Shi P, Zhang Y-p, Zhang J (2005) Dramatic variation of the vomeronasal pheromone receptor gene repertoire among five orders of placental and marsupial mammals. PNAS  102: 5767–5772.  <ul class="find" data-citedArticleID="1084004" data-doi="10.1073/pnas.0501589102"><li><a href="http://dx.doi.org/10.1073/pnas.0501589102" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Dramatic+variation+of+the+vomeronasal+pheromone+receptor+gene+repertoire+among+five+orders+of+placental+and+marsupial+mammals." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Dramatic+variation+of+the+vomeronasal+pheromone+receptor+gene+repertoire+among+five+orders+of+placental+and+marsupial+mammals.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0001903-Gimelbrant1" id="pone.0001903-Gimelbrant1"></a>Gimelbrant AA, Skaletsky H, Chess A (2004) Selective pressures on the olfactory receptor repertoire since the human-chimpanzee divergence. Proc Natl Acad Sci U S A  101: 9019–9022.  <ul class="find" data-citedArticleID="1083998" data-doi="10.1073/pnas.0401566101"><li><a href="http://dx.doi.org/10.1073/pnas.0401566101" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Selective+pressures+on+the+olfactory+receptor+repertoire+since+the+human-chimpanzee+divergence." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Selective+pressures+on+the+olfactory+receptor+repertoire+since+the+human-chimpanzee+divergence.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0001903-Gilad1" id="pone.0001903-Gilad1"></a>Gilad Y, Man O, Glusman G (2005) A comparison of the human and chimpanzee olfactory receptor gene repertoires. Genome Res  15: 224–230.  <ul class="find" data-citedArticleID="1083996" data-doi="10.1101/gr.2846405"><li><a href="http://dx.doi.org/10.1101/gr.2846405" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+comparison+of+the+human+and+chimpanzee+olfactory+receptor+gene+repertoires." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+comparison+of+the+human+and+chimpanzee+olfactory+receptor+gene+repertoires.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0001903-Alioto1" id="pone.0001903-Alioto1"></a>Alioto TS, Ngai J (2005) The odorant receptor repertoire of teleost fish. BMC Genomics  6: 173.  <ul class="find" data-citedArticleID="1083956" data-doi="10.1186/1471-2164-6-173"><li><a href="http://dx.doi.org/10.1186/1471-2164-6-173" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+odorant+receptor+repertoire+of+teleost+fish." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+odorant+receptor+repertoire+of+teleost+fish.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0001903-Rodriguez1" id="pone.0001903-Rodriguez1"></a>Rodriguez I (2005) Remarkable diversity of mammalian pheromone receptor repertoires. Proc Natl Acad Sci U S A  102: 6639–6640.  <ul class="find" data-citedArticleID="1084058" data-doi="10.1073/pnas.0502318102"><li><a href="http://dx.doi.org/10.1073/pnas.0502318102" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Remarkable+diversity+of+mammalian+pheromone+receptor+repertoires." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Remarkable+diversity+of+mammalian+pheromone+receptor+repertoires.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0001903-Grus2" id="pone.0001903-Grus2"></a>Grus WE, Shi P, Zhang Y-p, Zhang J (2005) Dramatic variation of the vomeronasal pheromone receptor gene repertoire among five orders of placental and marsupial mammals. Proc Natl Acad Sci U S A  102: 5767–5772.  <ul class="find" data-citedArticleID="1084006" data-doi="10.1073/pnas.0501589102"><li><a href="http://dx.doi.org/10.1073/pnas.0501589102" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Dramatic+variation+of+the+vomeronasal+pheromone+receptor+gene+repertoire+among+five+orders+of+placental+and+marsupial+mammals." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Dramatic+variation+of+the+vomeronasal+pheromone+receptor+gene+repertoire+among+five+orders+of+placental+and+marsupial+mammals.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0001903-Mombaerts1" id="pone.0001903-Mombaerts1"></a>Mombaerts P (1999) Seven-transmembrane proteins as odorant and chemosensory receptors. Science  286: 707–711.  <ul class="find" data-citedArticleID="1084048" data-doi="10.1126/science.286.5440.707"><li><a href="http://dx.doi.org/10.1126/science.286.5440.707" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Seven-transmembrane+proteins+as+odorant+and+chemosensory+receptors." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Seven-transmembrane+proteins+as+odorant+and+chemosensory+receptors.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0001903-Goodstadt1" id="pone.0001903-Goodstadt1"></a>Goodstadt L, Ponting CP (2006) Phylogenetic reconstruction of orthology, paralogy, and conserved synteny for dog and human. PLoS Comput Biol  2: e133.  <ul class="find" data-citedArticleID="1084002" data-doi="10.1371/journal.pcbi.0020133"><li><a href="http://dx.doi.org/10.1371/journal.pcbi.0020133" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Phylogenetic+reconstruction+of+orthology%2C+paralogy%2C+and+conserved+synteny+for+dog+and+human." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Phylogenetic+reconstruction+of+orthology%2C+paralogy%2C+and+conserved+synteny+for+dog+and+human.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0001903-Choi1" id="pone.0001903-Choi1"></a>Choi SS, Lahn BT (2003) Adaptive evolution of MRG, a neuron-specific gene family implicated in nociception. Genome Res  13: 2252–2259.  <ul class="find" data-citedArticleID="1083972" data-doi="10.1101/gr.1431603"><li><a href="http://dx.doi.org/10.1101/gr.1431603" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Adaptive+evolution+of+MRG%2C+a+neuron-specific+gene+family+implicated+in+nociception." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Adaptive+evolution+of+MRG%2C+a+neuron-specific+gene+family+implicated+in+nociception.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0001903-Zylka1" id="pone.0001903-Zylka1"></a>Zylka MJ, Dong X, Southwell AL, Anderson DJ (2003) Atypical expansion in mice of the sensory neuron-specific Mrg G protein-coupled receptor family. Proc Natl Acad Sci U S A  100: 10043–10048.  <ul class="find" data-citedArticleID="1084106" data-doi="10.1073/pnas.1732949100"><li><a href="http://dx.doi.org/10.1073/pnas.1732949100" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Atypical+expansion+in+mice+of+the+sensory+neuron-specific+Mrg+G+protein-coupled+receptor+family." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Atypical+expansion+in+mice+of+the+sensory+neuron-specific+Mrg+G+protein-coupled+receptor+family.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0001903-Gloriam1" id="pone.0001903-Gloriam1"></a>Gloriam DE, Bjarnadottir TK, Yan YL, Postlethwait JH, Schioth HB, et al.  (2005) The repertoire of trace amine G-protein-coupled receptors: large expansion in zebrafish. Mol Phylogenet Evol  35: 470–482.  <ul class="find" data-citedArticleID="1084000" data-doi="10.1016/j.ympev.2004.12.003"><li><a href="http://dx.doi.org/10.1016/j.ympev.2004.12.003" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+repertoire+of+trace+amine+G-protein-coupled+receptors%3A+large+expansion+in+zebrafish." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+repertoire+of+trace+amine+G-protein-coupled+receptors%3A+large+expansion+in+zebrafish.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">36.
              </span><a name="pone.0001903-Jaillon1" id="pone.0001903-Jaillon1"></a>Jaillon O, Aury J-M, Brunet F, Petit J-L, Stange-Thomann N, et al.  (2004) Genome duplication in the teleost fish Tetraodon nigroviridis reveals the early vertebrate proto-karyotype. Nature  431: 946–957.  <ul class="find" data-citedArticleID="1084030" data-doi="10.1038/nature03025"><li><a href="http://dx.doi.org/10.1038/nature03025" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genome+duplication+in+the+teleost+fish+Tetraodon+nigroviridis+reveals+the+early+vertebrate+proto-karyotype." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genome+duplication+in+the+teleost+fish+Tetraodon+nigroviridis+reveals+the+early+vertebrate+proto-karyotype.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">37.
              </span><a name="pone.0001903-Woods1" id="pone.0001903-Woods1"></a>Woods IG, Wilson C, Friedlander B, Chang P, Reyes DK, et al.  (2005) The zebrafish gene map defines ancestral vertebrate chromosomes. Genome Res  15: 1307–1314.  <ul class="find" data-citedArticleID="1084100" data-doi="10.1101/gr.4134305"><li><a href="http://dx.doi.org/10.1101/gr.4134305" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+zebrafish+gene+map+defines+ancestral+vertebrate+chromosomes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+zebrafish+gene+map+defines+ancestral+vertebrate+chromosomes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">38.
              </span><a name="pone.0001903-Hoegg1" id="pone.0001903-Hoegg1"></a>Hoegg S, Brinkmann H, Taylor JS, Meyer A (2004) Phylogenetic timing of the fish-specific genome duplication correlates with the diversification of teleost fish. J Mol Evol  59: 190–203.  <ul class="find" data-citedArticleID="1084012" data-doi="10.1007/s00239-004-2613-z"><li><a href="http://dx.doi.org/10.1007/s00239-004-2613-z" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Phylogenetic+timing+of+the+fish-specific+genome+duplication+correlates+with+the+diversification+of+teleost+fish." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Phylogenetic+timing+of+the+fish-specific+genome+duplication+correlates+with+the+diversification+of+teleost+fish.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">39.
              </span><a name="pone.0001903-VandePeer1" id="pone.0001903-VandePeer1"></a>Van de Peer Y (2004) Tetraodon genome confirms Takifugu findings: most fish are ancient polyploids. Genome Biol  5: 250.  <ul class="find" data-citedArticleID="1084092"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Tetraodon+genome+confirms+Takifugu+findings%3A+most+fish+are+ancient+polyploids.&amp;auth=&amp;atitle=Tetraodon+genome+confirms+Takifugu+findings%3A+most+fish+are+ancient+polyploids." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Tetraodon+genome+confirms+Takifugu+findings%3A+most+fish+are+ancient+polyploids." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Tetraodon+genome+confirms+Takifugu+findings%3A+most+fish+are+ancient+polyploids.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">40.
              </span><a name="pone.0001903-Amores1" id="pone.0001903-Amores1"></a>Amores A, Force A, Yan YL, Joly L, Amemiya C, et al.  (1998) Zebrafish hox clusters and vertebrate genome evolution. Science  282: 1711–1714.  <ul class="find" data-citedArticleID="1083960" data-doi="10.1126/science.282.5394.1711"><li><a href="http://dx.doi.org/10.1126/science.282.5394.1711" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Zebrafish+hox+clusters+and+vertebrate+genome+evolution." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Zebrafish+hox+clusters+and+vertebrate+genome+evolution.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">41.
              </span><a name="pone.0001903-Taylor2" id="pone.0001903-Taylor2"></a>Taylor JS, Van de Peer Y, Braasch I, Meyer A (2001) Comparative genomics provides evidence for an ancient genome duplication event in fish. Philos Trans R Soc Lond B Biol Sci  356: 1661–1679.  <ul class="find" data-citedArticleID="1084080" data-doi="10.1098/rstb.2001.0975"><li><a href="http://dx.doi.org/10.1098/rstb.2001.0975" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Comparative+genomics+provides+evidence+for+an+ancient+genome+duplication+event+in+fish." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Comparative+genomics+provides+evidence+for+an+ancient+genome+duplication+event+in+fish.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">42.
              </span><a name="pone.0001903-Mulley1" id="pone.0001903-Mulley1"></a>Mulley J, Holland P (2004) Comparative genomics: Small genome, big insights. Nature  431: 916–917.  <ul class="find" data-citedArticleID="1084050" data-doi="10.1038/431916a"><li><a href="http://dx.doi.org/10.1038/431916a" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Comparative+genomics%3A+Small+genome%2C+big+insights." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Comparative+genomics%3A+Small+genome%2C+big+insights.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">43.
              </span><a name="pone.0001903-Christoffels1" id="pone.0001903-Christoffels1"></a>Christoffels A, Koh EGL, Chia J-m, Brenner S, Aparicio S, et al.  (2004) Fugu genome analysis provides evidence for a whole-genome duplication early during the evolution of ray-finned fishes. Mol Biol Evol  21: 1146–1151.  <ul class="find" data-citedArticleID="1083974" data-doi="10.1093/molbev/msh114"><li><a href="http://dx.doi.org/10.1093/molbev/msh114" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Fugu+genome+analysis+provides+evidence+for+a+whole-genome+duplication+early+during+the+evolution+of+ray-finned+fishes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Fugu+genome+analysis+provides+evidence+for+a+whole-genome+duplication+early+during+the+evolution+of+ray-finned+fishes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">44.
              </span><a name="pone.0001903-Taylor3" id="pone.0001903-Taylor3"></a>Taylor JS, Braasch I, Frickey T, Meyer A, Van de Peer Y (2003) Genome duplication, a trait shared by 22000 species of ray-finned fish. Genome Res  13: 382–390.  <ul class="find" data-citedArticleID="1084082" data-doi="10.1101/gr.640303"><li><a href="http://dx.doi.org/10.1101/gr.640303" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genome+duplication%2C+a+trait+shared+by+22000+species+of+ray-finned+fish." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genome+duplication%2C+a+trait+shared+by+22000+species+of+ray-finned+fish.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">45.
              </span><a name="pone.0001903-Vandepoele1" id="pone.0001903-Vandepoele1"></a>Vandepoele K, De Vos W, Taylor JS, Meyer A, Van de Peer Y (2004) Major events in the genome evolution of vertebrates: Paranome age and size differ considerably between ray-finned fishes and land vertebrates. Proc Natl Acad Sci U S A  101: 1638–1643.  <ul class="find" data-citedArticleID="1084094" data-doi="10.1073/pnas.0307968100"><li><a href="http://dx.doi.org/10.1073/pnas.0307968100" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Major+events+in+the+genome+evolution+of+vertebrates%3A+Paranome+age+and+size+differ+considerably+between+ray-finned+fishes+and+land+vertebrates." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Major+events+in+the+genome+evolution+of+vertebrates%3A+Paranome+age+and+size+differ+considerably+between+ray-finned+fishes+and+land+vertebrates.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">46.
              </span><a name="pone.0001903-Postlethwait1" id="pone.0001903-Postlethwait1"></a>Postlethwait JH, Yan YL, Gates MA, Horne S, Amores A, et al.  (1998) Vertebrate genome evolution and the zebrafish gene map. Nat Genet  18: 345–349.  <ul class="find" data-citedArticleID="1084056" data-doi="10.1038/ng0498-345"><li><a href="http://dx.doi.org/10.1038/ng0498-345" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Vertebrate+genome+evolution+and+the+zebrafish+gene+map." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Vertebrate+genome+evolution+and+the+zebrafish+gene+map.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">47.
              </span><a name="pone.0001903-Crollius1" id="pone.0001903-Crollius1"></a>Crollius HR, Weissenbach J (2005) Fish genomics and biology. Genome Res  15: 1675–1682.  <ul class="find" data-citedArticleID="1083980" data-doi="10.1101/gr.3735805"><li><a href="http://dx.doi.org/10.1101/gr.3735805" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Fish+genomics+and+biology." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Fish+genomics+and+biology.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">48.
              </span><a name="pone.0001903-Papp1" id="pone.0001903-Papp1"></a>Papp B, Pal C, Hurst LD (2003) Dosage sensitivity and the evolution of gene families in yeast. Nature  424: 194–197.  <ul class="find" data-citedArticleID="1084054" data-doi="10.1038/nature01771"><li><a href="http://dx.doi.org/10.1038/nature01771" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Dosage+sensitivity+and+the+evolution+of+gene+families+in+yeast." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Dosage+sensitivity+and+the+evolution+of+gene+families+in+yeast.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">49.
              </span><a name="pone.0001903-Shi1" id="pone.0001903-Shi1"></a>Shi Q, King RW (2005) Chromosome nondisjunction yields tetraploid rather than aneuploid cells in human cell lines. Nature  437: 1038–1042.  <ul class="find" data-citedArticleID="1084066" data-doi="10.1038/nature03958"><li><a href="http://dx.doi.org/10.1038/nature03958" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Chromosome+nondisjunction+yields+tetraploid+rather+than+aneuploid+cells+in+human+cell+lines." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Chromosome+nondisjunction+yields+tetraploid+rather+than+aneuploid+cells+in+human+cell+lines.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">50.
              </span><a name="pone.0001903-Feldman1" id="pone.0001903-Feldman1"></a>Feldman M, Liu B, Segal G, Abbo S, Levy AA, et al.  (1997) Rapid elimination of low-copy DNA sequences in polyploid wheat: a possible mechanism for differentiation of homoeologous chromosomes. Genetics  147: 1381–1387.  <ul class="find" data-citedArticleID="1083982"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Rapid+elimination+of+low-copy+DNA+sequences+in+polyploid+wheat%3A+a+possible+mechanism+for+differentiation+of+homoeologous+chromosomes.&amp;auth=&amp;atitle=Rapid+elimination+of+low-copy+DNA+sequences+in+polyploid+wheat%3A+a+possible+mechanism+for+differentiation+of+homoeologous+chromosomes." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Rapid+elimination+of+low-copy+DNA+sequences+in+polyploid+wheat%3A+a+possible+mechanism+for+differentiation+of+homoeologous+chromosomes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Rapid+elimination+of+low-copy+DNA+sequences+in+polyploid+wheat%3A+a+possible+mechanism+for+differentiation+of+homoeologous+chromosomes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">51.
              </span><a name="pone.0001903-Kellis1" id="pone.0001903-Kellis1"></a>Kellis M, Birren BW, Lander ES (2004) Proof and evolutionary analysis of ancient genome duplication in the yeast Saccharomyces cerevisiae. Nature  428: 617–624.  <ul class="find" data-citedArticleID="1084036" data-doi="10.1038/nature02424"><li><a href="http://dx.doi.org/10.1038/nature02424" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Proof+and+evolutionary+analysis+of+ancient+genome+duplication+in+the+yeast+Saccharomyces+cerevisiae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Proof+and+evolutionary+analysis+of+ancient+genome+duplication+in+the+yeast+Saccharomyces+cerevisiae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">52.
              </span><a name="pone.0001903-Brunet1" id="pone.0001903-Brunet1"></a>Brunet FG, Crollius HR, Paris M, Aury JM, Gibert P, et al.  (2006) Gene loss and evolutionary rates following whole-genome duplication in teleost fishes. Mol Biol Evol  23: 1808–1816.  <ul class="find" data-citedArticleID="1083968" data-doi="10.1093/molbev/msl049"><li><a href="http://dx.doi.org/10.1093/molbev/msl049" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Gene+loss+and+evolutionary+rates+following+whole-genome+duplication+in+teleost+fishes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Gene+loss+and+evolutionary+rates+following+whole-genome+duplication+in+teleost+fishes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">53.
              </span><a name="pone.0001903-Hsu1" id="pone.0001903-Hsu1"></a>Hsu SY (1999) Cloning of two novel mammalian paralogs of relaxin/insulin family proteins and their expression in testis and kidney. Mol Endocrinol  13: 2163–2174.  <ul class="find" data-citedArticleID="1084014" data-doi="10.1210/mend.13.12.0388"><li><a href="http://dx.doi.org/10.1210/mend.13.12.0388" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Cloning+of+two+novel+mammalian+paralogs+of+relaxin%2Finsulin+family+proteins+and+their+expression+in+testis+and+kidney." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Cloning+of+two+novel+mammalian+paralogs+of+relaxin%2Finsulin+family+proteins+and+their+expression+in+testis+and+kidney.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">54.
              </span><a name="pone.0001903-Hsu2" id="pone.0001903-Hsu2"></a>Hsu SY, Kudo M, Chen T, Nakabayashi K, Bhalla A, et al.  (2000) The three subfamilies of leucine-rich repeat-containing G protein-coupled receptors (LGR): identification of LGR6 and LGR7 and the signaling mechanism for LGR7. Mol Endocrinol  14: 1257–1271.  <ul class="find" data-citedArticleID="1084016" data-doi="10.1210/mend.14.8.0510"><li><a href="http://dx.doi.org/10.1210/mend.14.8.0510" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+three+subfamilies+of+leucine-rich+repeat-containing+G+protein-coupled+receptors+%28LGR%29%3A+identification+of+LGR6+and+LGR7+and+the+signaling+mechanism+for+LGR7." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+three+subfamilies+of+leucine-rich+repeat-containing+G+protein-coupled+receptors+%28LGR%29%3A+identification+of+LGR6+and+LGR7+and+the+signaling+mechanism+for+LGR7.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">55.
              </span><a name="pone.0001903-Hsu3" id="pone.0001903-Hsu3"></a>Hsu SY, Nakabayashi K, Bhalla A (2002) Evolution of glycoprotein hormone subunit genes in bilateral metazoa: identification of two novel human glycoprotein hormone subunit family genes, GPA2 and GPB5. Mol Endocrinol  16: 1538–1551.  <ul class="find" data-citedArticleID="1084018" data-doi="10.1210/me.16.7.1538"><li><a href="http://dx.doi.org/10.1210/me.16.7.1538" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evolution+of+glycoprotein+hormone+subunit+genes+in+bilateral+metazoa%3A+identification+of+two+novel+human+glycoprotein+hormone+subunit+family+genes%2C+GPA2+and+GPB5." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evolution+of+glycoprotein+hormone+subunit+genes+in+bilateral+metazoa%3A+identification+of+two+novel+human+glycoprotein+hormone+subunit+family+genes%2C+GPA2+and+GPB5.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">56.
              </span><a name="pone.0001903-Roh1" id="pone.0001903-Roh1"></a>Roh J, Chang CL, Bhalla A, Klein C, Hsu SY (2004) Intermedin is a calcitonin/calcitonin gene-related peptide family peptide acting through the calcitonin receptor-like receptor/receptor activity-modifying protein receptor complexes. J Biol Chem  279: 7264–7274.  <ul class="find" data-citedArticleID="1084060" data-doi="10.1074/jbc.m305332200"><li><a href="http://dx.doi.org/10.1074/jbc.m305332200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Intermedin+is+a+calcitonin%2Fcalcitonin+gene-related+peptide+family+peptide+acting+through+the+calcitonin+receptor-like+receptor%2Freceptor+activity-modifying+protein+receptor+complexes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Intermedin+is+a+calcitonin%2Fcalcitonin+gene-related+peptide+family+peptide+acting+through+the+calcitonin+receptor-like+receptor%2Freceptor+activity-modifying+protein+receptor+complexes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">57.
              </span><a name="pone.0001903-Hsu4" id="pone.0001903-Hsu4"></a>Hsu SY, Hsueh AJ (2000) Discovering new hormones, receptors, and signaling mediators in the genomic era. Mol Endocrinol  14: 594–604.  <ul class="find" data-citedArticleID="1084020" data-doi="10.1210/mend.14.5.0472"><li><a href="http://dx.doi.org/10.1210/mend.14.5.0472" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Discovering+new+hormones%2C+receptors%2C+and+signaling+mediators+in+the+genomic+era." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Discovering+new+hormones%2C+receptors%2C+and+signaling+mediators+in+the+genomic+era.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">58.
              </span><a name="pone.0001903-Hsu5" id="pone.0001903-Hsu5"></a>Hsu SY, Hsueh AJ (2001) Human stresscopin and stresscopin-related peptide are selective ligands for the type 2 corticotropin-releasing hormone receptor. Nat Med  7: 605–611.  <ul class="find" data-citedArticleID="1084022"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Human+stresscopin+and+stresscopin-related+peptide+are+selective+ligands+for+the+type+2+corticotropin-releasing+hormone+receptor.&amp;auth=&amp;atitle=Human+stresscopin+and+stresscopin-related+peptide+are+selective+ligands+for+the+type+2+corticotropin-releasing+hormone+receptor." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Human+stresscopin+and+stresscopin-related+peptide+are+selective+ligands+for+the+type+2+corticotropin-releasing+hormone+receptor." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Human+stresscopin+and+stresscopin-related+peptide+are+selective+ligands+for+the+type+2+corticotropin-releasing+hormone+receptor.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">59.
              </span><a name="pone.0001903-Taylor4" id="pone.0001903-Taylor4"></a>Taylor JS, Van de Peer Y, Meyer A (2001) Genome duplication, divergent resolution and speciation. Trends Genet  17: 299–301.  <ul class="find" data-citedArticleID="1084084" data-doi="10.1016/s0168-9525(01)02318-6"><li><a href="http://dx.doi.org/10.1016/s0168-9525(01)02318-6" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genome+duplication%2C+divergent+resolution+and+speciation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genome+duplication%2C+divergent+resolution+and+speciation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">60.
              </span><a name="pone.0001903-HuertaCepas1" id="pone.0001903-HuertaCepas1"></a>Huerta-Cepas J, Dopazo H, Dopazo J, Gabaldon T (2007) The human phylome. Genome Biol  8: R109.  <ul class="find" data-citedArticleID="1084026" data-doi="10.1093/nar/gkm899"><li><a href="http://dx.doi.org/10.1093/nar/gkm899" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+human+phylome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+human+phylome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">61.
              </span><a name="pone.0001903-Swanson1" id="pone.0001903-Swanson1"></a>Swanson WJ, Vacquier VD (2002) The rapid evolution of reproductive proteins. Nat Rev Genet  3: 137–144.  <ul class="find" data-citedArticleID="1084074" data-doi="10.1038/nrg733"><li><a href="http://dx.doi.org/10.1038/nrg733" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+rapid+evolution+of+reproductive+proteins." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+rapid+evolution+of+reproductive+proteins.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">62.
              </span><a name="pone.0001903-Kocher1" id="pone.0001903-Kocher1"></a>Kocher TD (2004) Adaptive evolution and explosive speciation: The cichlid fish model. Nat Rev Genet  5: 288–298.  <ul class="find" data-citedArticleID="1084040" data-doi="10.1038/nrg1316"><li><a href="http://dx.doi.org/10.1038/nrg1316" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Adaptive+evolution+and+explosive+speciation%3A+The+cichlid+fish+model." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Adaptive+evolution+and+explosive+speciation%3A+The+cichlid+fish+model.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">63.
              </span><a name="pone.0001903-LindbladToh1" id="pone.0001903-LindbladToh1"></a>Lindblad-Toh K, Wade CM, Mikkelsen TS, Karlsson EK, Jaffe DB, et al.  (2005) Genome sequence, comparative analysis and haplotype structure of the domestic dog. Nature  438: 803–819.  <ul class="find" data-citedArticleID="1084044"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Genome+sequence%2C+comparative+analysis+and+haplotype+structure+of+the+domestic+dog.&amp;auth=&amp;atitle=Genome+sequence%2C+comparative+analysis+and+haplotype+structure+of+the+domestic+dog." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genome+sequence%2C+comparative+analysis+and+haplotype+structure+of+the+domestic+dog." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genome+sequence%2C+comparative+analysis+and+haplotype+structure+of+the+domestic+dog.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">64.
              </span><a name="pone.0001903-Small1" id="pone.0001903-Small1"></a>Small K, Tanguay D, Nandabalan K, Zhan P, Stephens J, et al.  (2003) Gene and protein domain-specific patterns of genetic variability within the G-protein coupled receptor superfamily. Am J Pharmacogenomics  3: 65–71.  <ul class="find" data-citedArticleID="1084070" data-doi="10.2165/00129785-200303010-00008"><li><a href="http://dx.doi.org/10.2165/00129785-200303010-00008" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Gene+and+protein+domain-specific+patterns+of+genetic+variability+within+the+G-protein+coupled+receptor+superfamily." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Gene+and+protein+domain-specific+patterns+of+genetic+variability+within+the+G-protein+coupled+receptor+superfamily.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">65.
              </span><a name="pone.0001903-Terai1" id="pone.0001903-Terai1"></a>Terai Y, Seehausen O, Sasaki T, Takahashi K, Mizoiri S, et al.  (2006) Divergent Selection on Opsins Drives Incipient Speciation in Lake Victoria Cichlids. PLoS Biol  4: e433.  <ul class="find" data-citedArticleID="1084086" data-doi="10.1371/journal.pbio.0040433"><li><a href="http://dx.doi.org/10.1371/journal.pbio.0040433" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Divergent+Selection+on+Opsins+Drives+Incipient+Speciation+in+Lake+Victoria+Cichlids." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Divergent+Selection+on+Opsins+Drives+Incipient+Speciation+in+Lake+Victoria+Cichlids.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">66.
              </span><a name="pone.0001903-Consortium1" id="pone.0001903-Consortium1"></a>Consortium ICGS (2004) Sequence and comparative analysis of the chicken genome provide unique perspectives on vertebrate evolution. Nature  432: 695–716.  <ul class="find" data-citedArticleID="1083976" data-doi="10.1038/nature03394"><li><a href="http://dx.doi.org/10.1038/nature03394" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Sequence+and+comparative+analysis+of+the+chicken+genome+provide+unique+perspectives+on+vertebrate+evolution." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Sequence+and+comparative+analysis+of+the+chicken+genome+provide+unique+perspectives+on+vertebrate+evolution.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">67.
              </span><a name="pone.0001903-Aparicio1" id="pone.0001903-Aparicio1"></a>Aparicio S, Chapman J, Stupka E, Putnam N, Chia JM, et al.  (2002) Whole-genome shotgun assembly and analysis of the genome of Fugu rubripes. Science  297: 1301–1310.  <ul class="find" data-citedArticleID="1083962" data-doi="10.1126/science.1072104"><li><a href="http://dx.doi.org/10.1126/science.1072104" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Whole-genome+shotgun+assembly+and+analysis+of+the+genome+of+Fugu+rubripes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Whole-genome+shotgun+assembly+and+analysis+of+the+genome+of+Fugu+rubripes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">68.
              </span><a name="pone.0001903-Altschul1" id="pone.0001903-Altschul1"></a>Altschul SF, Madden TL, Schaffer AA, Zhang J, Zhang Z, et al.  (1997) Gapped BLAST and PSI-BLAST: a new generation of protein database search programs. Nucleic Acids Res  25: 3389–3402.  <ul class="find" data-citedArticleID="1083958" data-doi="10.1093/nar/25.17.3389"><li><a href="http://dx.doi.org/10.1093/nar/25.17.3389" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Gapped+BLAST+and+PSI-BLAST%3A+a+new+generation+of+protein+database+search+programs." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Gapped+BLAST+and+PSI-BLAST%3A+a+new+generation+of+protein+database+search+programs.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">69.
              </span><a name="pone.0001903-Schaffer1" id="pone.0001903-Schaffer1"></a>Schaffer AA, Aravind L, Madden TL, Shavirin S, Spouge JL, et al.  (2001) Improving the accuracy of PSI-BLAST protein database searches with composition-based statistics and other refinements. Nucleic Acids Res  29: 2994–3005.  <ul class="find" data-citedArticleID="1084064" data-doi="10.1093/nar/29.14.2994"><li><a href="http://dx.doi.org/10.1093/nar/29.14.2994" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Improving+the+accuracy+of+PSI-BLAST+protein+database+searches+with+composition-based+statistics+and+other+refinements." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Improving+the+accuracy+of+PSI-BLAST+protein+database+searches+with+composition-based+statistics+and+other+refinements.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">70.
              </span><a name="pone.0001903-Tatusova1" id="pone.0001903-Tatusova1"></a>Tatusova TA, Madden TL (1999) BLAST 2 Sequences, a new tool for comparing protein and nucleotide sequences. FEMS Microbiol Lett  174: 247–250.  <ul class="find" data-citedArticleID="1084076" data-doi="10.1111/j.1574-6968.1999.tb13575.x"><li><a href="http://dx.doi.org/10.1111/j.1574-6968.1999.tb13575.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=BLAST+2+Sequences%2C+a+new+tool+for+comparing+protein+and+nucleotide+sequences." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22BLAST+2+Sequences%2C+a+new+tool+for+comparing+protein+and+nucleotide+sequences.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">71.
              </span><a name="pone.0001903-Thompson1" id="pone.0001903-Thompson1"></a>Thompson JD, Higgins DG, Gibson TJ (1994) CLUSTAL W: improving the sensitivity of progressive multiple sequence alignment through sequence weighting, position-specific gap penalties and weight matrix choice. Nucleic Acids Res  22: 4673–4680.  <ul class="find" data-citedArticleID="1084088" data-doi="10.1093/nar/22.22.4673"><li><a href="http://dx.doi.org/10.1093/nar/22.22.4673" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=CLUSTAL+W%3A+improving+the+sensitivity+of+progressive+multiple+sequence+alignment+through+sequence+weighting%2C+position-specific+gap+penalties+and+weight+matrix+choice." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22CLUSTAL+W%3A+improving+the+sensitivity+of+progressive+multiple+sequence+alignment+through+sequence+weighting%2C+position-specific+gap+penalties+and+weight+matrix+choice.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">72.
              </span><a name="pone.0001903-Saitou1" id="pone.0001903-Saitou1"></a>Saitou N, Nei M (1987) The neighbor-joining method: a new method for reconstructing phylogenetic trees. Mol Biol Evol  4: 406–425.  <ul class="find" data-citedArticleID="1084062"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=The+neighbor-joining+method%3A+a+new+method+for+reconstructing+phylogenetic+trees.&amp;auth=&amp;atitle=The+neighbor-joining+method%3A+a+new+method+for+reconstructing+phylogenetic+trees." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+neighbor-joining+method%3A+a+new+method+for+reconstructing+phylogenetic+trees." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+neighbor-joining+method%3A+a+new+method+for+reconstructing+phylogenetic+trees.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">73.
              </span><a name="pone.0001903-Felsenstein1" id="pone.0001903-Felsenstein1"></a>Felsenstein J (1989) PHYLIP-Phylogeny Inference Package (Version 3.2). Cladistics  5: 164–166.  <ul class="find" data-citedArticleID="1083984"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=PHYLIP-Phylogeny+Inference+Package+%28Version+3.2%29.&amp;auth=&amp;atitle=PHYLIP-Phylogeny+Inference+Package+%28Version+3.2%29." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=PHYLIP-Phylogeny+Inference+Package+%28Version+3.2%29." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22PHYLIP-Phylogeny+Inference+Package+%28Version+3.2%29.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">74.
              </span><a name="pone.0001903-Hubbard1" id="pone.0001903-Hubbard1"></a>Hubbard T, Andrews D, Caccamo M, Cameron G, Chen Y, et al.  (2005) Ensembl 2005. Nucleic Acids Res  33: D447–453.  <ul class="find" data-citedArticleID="1084024" data-doi="10.1093/nar/gki138"><li><a href="http://dx.doi.org/10.1093/nar/gki138" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Ensembl+2005." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Ensembl+2005.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">75.
              </span><a name="pone.0001903-Kasprzyk1" id="pone.0001903-Kasprzyk1"></a>Kasprzyk A, Keefe D, Smedley D, London D, Spooner W, et al.  (2004) EnsMart: a generic system for fast and flexible access to biological data. Genome Res  14: 160–169.  <ul class="find" data-citedArticleID="1084034" data-doi="10.1101/gr.1645104"><li><a href="http://dx.doi.org/10.1101/gr.1645104" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=EnsMart%3A+a+generic+system+for+fast+and+flexible+access+to+biological+data." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22EnsMart%3A+a+generic+system+for+fast+and+flexible+access+to+biological+data.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">76.
              </span><a name="pone.0001903-Kent1" id="pone.0001903-Kent1"></a>Kent WJ (2002) BLAT–the BLAST-like alignment tool. Genome Res  12: 656–664.  <ul class="find" data-citedArticleID="1084038" data-doi="10.1101/gr.229202."><li><a href="http://dx.doi.org/10.1101/gr.229202." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=BLAT%E2%80%93the+BLAST-like+alignment+tool." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22BLAT%E2%80%93the+BLAST-like+alignment+tool.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001903.XML" value="133559"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001903.PDF" value="349670"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001903.g001.PNG_L" value="224116"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001903.g001.PNG_M" value="70002"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001903.g001.PNG_S" value="6914"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001903.g001.TIF" value="639756"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001903.g001.PNG_I" value="28497"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001903.g002.PNG_L" value="740819"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001903.g002.PNG_M" value="177353"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001903.g002.PNG_S" value="7086"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001903.g002.TIF" value="926868"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001903.g002.PNG_I" value="61855"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001903.g003.PNG_L" value="702200"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001903.g003.PNG_M" value="135290"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001903.g003.PNG_S" value="7902"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001903.g003.TIF" value="1292720"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001903.g003.PNG_I" value="50712"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001903.g004.PNG_L" value="201796"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001903.g004.PNG_M" value="76854"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001903.g004.PNG_S" value="6370"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001903.g004.TIF" value="414372"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001903.g004.PNG_I" value="30140"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001903.g005.PNG_L" value="35141"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001903.g005.PNG_M" value="15604"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001903.g005.PNG_S" value="2537"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001903.g005.TIF" value="80426"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001903.g005.PNG_I" value="6689"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001903.g006.PNG_L" value="875104"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001903.g006.PNG_M" value="130654"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001903.g006.PNG_S" value="7732"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001903.g006.TIF" value="1458550"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001903.g006.PNG_I" value="47089"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001903.s001.PDF" value="11229"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001903.s002.PDF" value="14224"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001903.s003.PDF" value="69624"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001903.s004.PDF" value="75345"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001903.s005.PDF" value="34666"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001903.s006.PDF" value="31926"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001903.s007.PDF" value="37982"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001903&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001903" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001903&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0001903&volume=&issue=&title=GPCR Genes Are Preferentially Retained after Whole Genome Duplication&author_name=Jenia%20Semyonov%2C%20Jae-Il%20Park%2C%20Chia%20Lin%20Chang%2C%20Sheau%20Yu%20Teddy%20Hsu&start_page=1&end_page=10" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001903" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001903" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0001903" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001903&amp;t=GPCR%20Genes%20Are%20Preferentially%20Retained%20after%20Whole%20Genome%20Duplication" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001903&title=GPCR%20Genes%20Are%20Preferentially%20Retained%20after%20Whole%20Genome%20Duplication&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001903&amp;title=GPCR%20Genes%20Are%20Preferentially%20Retained%20after%20Whole%20Genome%20Duplication" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001903" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0001903&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'GPCR Genes Are Preferentially Retained after Whole Genome Duplication';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0001903';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20GPCR%20Genes%20Are%20Preferentially%20Retained%20after%20Whole%20Genome%20Duplication http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001903" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0001903" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">














          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Chromosomes%22" title="Search for articles in the subject area:'Chromosomes'"><div class="flagText">Chromosomes</div></a>
              <div data-categoryid="17251" data-articleid="26630"
                   data-categoryname="Chromosomes"
                   class="flagImage" title="Flag 'Chromosomes' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Evolutionary+genetics%22" title="Search for articles in the subject area:'Evolutionary genetics'"><div class="flagText">Evolutionary genetics</div></a>
              <div data-categoryid="17209" data-articleid="26630"
                   data-categoryname="Evolutionary genetics"
                   class="flagImage" title="Flag 'Evolutionary genetics' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Genome+evolution%22" title="Search for articles in the subject area:'Genome evolution'"><div class="flagText">Genome evolution</div></a>
              <div data-categoryid="17207" data-articleid="26630"
                   data-categoryname="Genome evolution"
                   class="flagImage" title="Flag 'Genome evolution' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Genomic+databases%22" title="Search for articles in the subject area:'Genomic databases'"><div class="flagText">Genomic databases</div></a>
              <div data-categoryid="42727" data-articleid="26630"
                   data-categoryname="Genomic databases"
                   class="flagImage" title="Flag 'Genomic databases' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Mammalian+genomics%22" title="Search for articles in the subject area:'Mammalian genomics'"><div class="flagText">Mammalian genomics</div></a>
              <div data-categoryid="17625" data-articleid="26630"
                   data-categoryname="Mammalian genomics"
                   class="flagImage" title="Flag 'Mammalian genomics' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Polypeptides%22" title="Search for articles in the subject area:'Polypeptides'"><div class="flagText">Polypeptides</div></a>
              <div data-categoryid="22517" data-articleid="26630"
                   data-categoryname="Polypeptides"
                   class="flagImage" title="Flag 'Polypeptides' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Sequence+databases%22" title="Search for articles in the subject area:'Sequence databases'"><div class="flagText">Sequence databases</div></a>
              <div data-categoryid="42787" data-articleid="26630"
                   data-categoryname="Sequence databases"
                   class="flagImage" title="Flag 'Sequence databases' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Vertebrates%22" title="Search for articles in the subject area:'Vertebrates'"><div class="flagText">Vertebrates</div></a>
              <div data-categoryid="43095" data-articleid="26630"
                   data-categoryname="Vertebrates"
                   class="flagImage" title="Flag 'Vertebrates' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=4113'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=9004'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=6561&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>


</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
