

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Comparison of Protein and mRNA Expression Evolution in Humans and Chimpanzees</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0000216"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0000216" />

    <meta name="citation_title" content="Comparison of Protein and mRNA Expression Evolution in Humans and Chimpanzees"/>
    <meta itemprop="name" content="Comparison of Protein and mRNA Expression Evolution in Humans and Chimpanzees"/>

      <meta name="citation_author" content="Ning Fu"/>
            <meta name="citation_author_institution" content="Key Lab of Systems Biology, Shanghai Institutes for Biological Sciences, Chinese Academy of Sciences, Shanghai, China"/>
      <meta name="citation_author" content="Ines Drinnenberg"/>
            <meta name="citation_author_institution" content="Max-Planck-Institute for Evolutionary Anthropology, Leipzig, Germany"/>
      <meta name="citation_author" content="Janet Kelso"/>
            <meta name="citation_author_institution" content="Max-Planck-Institute for Evolutionary Anthropology, Leipzig, Germany"/>
      <meta name="citation_author" content="Jia-Rui Wu"/>
            <meta name="citation_author_institution" content="Key Lab of Systems Biology, Shanghai Institutes for Biological Sciences, Chinese Academy of Sciences, Shanghai, China"/>
      <meta name="citation_author" content="Svante Pääbo"/>
            <meta name="citation_author_institution" content="Max-Planck-Institute for Evolutionary Anthropology, Leipzig, Germany"/>
      <meta name="citation_author" content="Rong Zeng"/>
            <meta name="citation_author_institution" content="Key Lab of Systems Biology, Shanghai Institutes for Biological Sciences, Chinese Academy of Sciences, Shanghai, China"/>
      <meta name="citation_author" content="Philipp Khaitovich"/>
            <meta name="citation_author_institution" content="Max-Planck-Institute for Evolutionary Anthropology, Leipzig, Germany"/>
            <meta name="citation_author_institution" content="Institute for Computational Biology, Shanghai Institutes for Biological Sciences, Chinese Academy of Sciences, Shanghai, China"/>

    <meta name="citation_date" content="2007/2/14"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0000216.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e216"/>
    <meta name="citation_issue" content="2"/>
    <meta name="citation_volume" content="2"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Human brain evolution: insights from microarrays.; citation_author=TM Preuss; citation_author=M Caceres; citation_author=MC Oldham; citation_author=DH Geschwind; citation_journal_title=Nat Rev Genet; citation_volume=5; citation_number=1; citation_pages=850-860; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Integrated transcriptome and proteome data: the challenges ahead.; citation_author=CJ Hack; citation_journal_title=Brief Funct Genomic Proteomic; citation_volume=3; citation_number=2; citation_pages=212-219; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Integrated genomic and proteomic analyses of a systematically perturbed metabolic network.; citation_author=T Ideker; citation_author=V Thorsson; citation_author=JA Ranish; citation_author=R Christmas; citation_author=J Buhler; citation_journal_title=Science; citation_volume=292; citation_number=3; citation_pages=929-934; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Complementary profiling of gene expression at the transcriptome and proteome levels in Saccharomyces cerevisiae.; citation_author=TJ Griffin; citation_author=SP Gygi; citation_author=T Ideker; citation_author=B Rist; citation_author=J Eng; citation_journal_title=Mol Cell Proteomics; citation_volume=1; citation_number=4; citation_pages=323-333; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Integrated genomic and proteomic analyses of gene expression in Mammalian cells.; citation_author=Q Tian; citation_author=SB Stepaniants; citation_author=M Mao; citation_author=L Weng; citation_author=MC Feetham; citation_journal_title=Mol Cell Proteomics; citation_volume=3; citation_number=5; citation_pages=960-969; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Protein pathway and complex clustering of correlated mRNA and protein expression analyses in Saccharomyces cerevisiae.; citation_author=MP Washburn; citation_author=A Koller; citation_author=G Oshiro; citation_author=RR Ulaszek; citation_author=D Plouffe; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=100; citation_number=6; citation_pages=3107-3112; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Integrating gene and protein expression data: pattern analysis and profile mining.; citation_author=B Cox; citation_author=T Kislinger; citation_author=A Emili; citation_journal_title=Methods; citation_volume=35; citation_number=7; citation_pages=303-314; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Correlation between protein and mRNA abundance in yeast.; citation_author=SP Gygi; citation_author=Y Rochon; citation_author=BR Franza; citation_author=R Aebersold; citation_journal_title=Mol Cell Biol; citation_volume=19; citation_number=8; citation_pages=1720-1730; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Integrated analysis of protein composition, tissue diversity, and gene regulation in mouse mitochondria.; citation_author=VK Mootha; citation_author=J Bunkenborg; citation_author=JV Olsen; citation_author=M Hjerrild; citation_author=JR Wisniewski; citation_journal_title=Cell; citation_volume=115; citation_number=9; citation_pages=629-640; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Identification of coexpressed gene clusters in a comparative analysis of transcriptome and proteome in mouse tissues.; citation_author=T Mijalski; citation_author=A Harder; citation_author=T Halder; citation_author=M Kersten; citation_author=M Horsch; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=102; citation_number=10; citation_pages=8621-8626; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Global survey of organ and organelle protein expression in mouse: combined proteomic and transcriptomic profiling.; citation_author=T Kislinger; citation_author=B Cox; citation_author=A Kannan; citation_author=C Chung; citation_author=P Hu; citation_journal_title=Cell; citation_volume=125; citation_number=11; citation_pages=173-186; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Evolution of primate gene expression.; citation_author=P Khaitovich; citation_author=W Enard; citation_author=M Lachmann; citation_author=S Paabo; citation_journal_title=Nat Rev Genet; citation_volume=7; citation_number=12; citation_pages=693-702; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Initial sequence of the chimpanzee genome and comparison with the human genome.; citation_journal_title=Nature; citation_volume=437; citation_number=13; citation_pages=69-87; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Parallel patterns of evolution in the genomes and transcriptomes of humans and chimpanzees.; citation_author=P Khaitovich; citation_author=I Hellmann; citation_author=W Enard; citation_author=K Nowick; citation_author=M Leinweber; citation_journal_title=Science; citation_volume=309; citation_number=14; citation_pages=1850-1854; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Exploration, normalization, and summaries of high density oligonucleotide array probe level data.; citation_author=RA Irizarry; citation_author=B Hobbs; citation_author=F Collin; citation_author=YD Beazer-Barclay; citation_author=KJ Antonellis; citation_journal_title=Biostatistics; citation_volume=4; citation_number=15; citation_pages=249-264; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=A comparative proteomic strategy for subcellular proteome research: ICAT approach coupled with bioinformatics prediction to ascertain rat liver mitochondrial proteins and indication of mitochondrial localization for catalase.; citation_author=XS Jiang; citation_author=J Dai; citation_author=QH Sheng; citation_author=L Zhang; citation_author=QC Xia; citation_journal_title=Mol Cell Proteomics; citation_volume=4; citation_number=16; citation_pages=12-34; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=PHYLIP - Phylogeny Inference Package (Version 3.2).; citation_author=J Felsenstein; citation_journal_title=Cladistics; citation_volume=5; citation_number=17; citation_pages=164-166; citation_date=1989; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000216" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Comparison of Protein and mRNA Expression Evolution in Humans and Chimpanzees"/>
    <meta name="twitter:description" content="Even though mRNA expression levels are commonly used as a proxy for estimating functional differences that occur at the protein level, the relation between mRNA and protein expression is not well established. Further, no study to date has tested whether the evolutionary differences in mRNA expression observed between species reflect those observed in protein expression. Since a large proportion of mRNA expression differences observed between mammalian species appears to have no functional consequences for the phenotype, it is conceivable that many or most mRNA expression differences are not reflected at the protein level. If this is true, then differences in protein expression may largely reflect functional adaptations observed in species phenotypes. In this paper, we present the first direct comparison of mRNA and protein expression differences seen between humans and chimpanzees. We reproducibly find a significant positive correlation between mRNA expression and protein expression differences. This correlation is comparable in magnitude to that found between mRNA and protein expression changes at different developmental stages or in different physiological conditions within one species. Noticeably, this correlation is mainly due to genes with large expression differences between species. Our study opens the door to a new level of understanding of regulatory evolution and poses many new questions that remain to be answered."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0000216.g002"/>

  <meta property="og:title" content="Comparison of Protein and mRNA Expression Evolution in Humans and Chimpanzees" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000216" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=9057'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=2152'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=5043&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0000216">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000216" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000216&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Comparison of Protein and mRNA Expression Evolution in Humans and Chimpanzees
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Ning Fu, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Key Lab of Systems Biology, Shanghai Institutes for Biological Sciences, Chinese Academy of Sciences, Shanghai, China
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Ines Drinnenberg, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Max-Planck-Institute for Evolutionary Anthropology, Leipzig, Germany
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Janet Kelso, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Max-Planck-Institute for Evolutionary Anthropology, Leipzig, Germany
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Jia-Rui Wu, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Key Lab of Systems Biology, Shanghai Institutes for Biological Sciences, Chinese Academy of Sciences, Shanghai, China
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Svante Pääbo, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Max-Planck-Institute for Evolutionary Anthropology, Leipzig, Germany
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Rong Zeng
              <span class="corresponding">mail</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:zr@sibs.ac.cn">zr@sibs.ac.cn</a> (RZ); <a href="mailto:khaitovich@eva.mpg.de">khaitovich@eva.mpg.de</a> (PK)</p>

                <p>Affiliation:
                  Key Lab of Systems Biology, Shanghai Institutes for Biological Sciences, Chinese Academy of Sciences, Shanghai, China
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Philipp Khaitovich
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:zr@sibs.ac.cn">zr@sibs.ac.cn</a> (RZ); <a href="mailto:khaitovich@eva.mpg.de">khaitovich@eva.mpg.de</a> (PK)</p>

                <p>Affiliations:
                  Max-Planck-Institute for Evolutionary Anthropology, Leipzig, Germany, 
                  Institute for Computational Biology, Shanghai Institutes for Biological Sciences, Chinese Academy of Sciences, Shanghai, China
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: February 14, 2007</li>
    <li>DOI: 10.1371/journal.pone.0000216</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0000216" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0000216" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000216" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0000216" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0000216-g001" data-doi="info:doi/10.1371/journal.pone.0000216" data-uri="info:doi/10.1371/journal.pone.0000216.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000216.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000216-g002" data-doi="info:doi/10.1371/journal.pone.0000216" data-uri="info:doi/10.1371/journal.pone.0000216.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000216.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000216">Reader Comments (0)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0000216" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1"></a><p>Even though mRNA expression levels are commonly used as a proxy for estimating functional differences that occur at the protein level, the relation between mRNA and protein expression is not well established. Further, no study to date has tested whether the evolutionary differences in mRNA expression observed between species reflect those observed in protein expression. Since a large proportion of mRNA expression differences observed between mammalian species appears to have no functional consequences for the phenotype, it is conceivable that many or most mRNA expression differences are not reflected at the protein level. If this is true, then differences in protein expression may largely reflect functional adaptations observed in species phenotypes. In this paper, we present the first direct comparison of mRNA and protein expression differences seen between humans and chimpanzees. We reproducibly find a significant positive correlation between mRNA expression and protein expression differences. This correlation is comparable in magnitude to that found between mRNA and protein expression changes at different developmental stages or in different physiological conditions within one species. Noticeably, this correlation is mainly due to genes with large expression differences between species. Our study opens the door to a new level of understanding of regulatory evolution and poses many new questions that remain to be answered.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>Fu N, Drinnenberg I, Kelso J, Wu J-R, Pääbo S, et al.  (2007) Comparison of Protein and mRNA Expression Evolution in Humans and Chimpanzees. PLoS ONE 2(2):
          e216.
            doi:10.1371/journal.pone.0000216</p><p><strong>Academic Editor: </strong>Laszlo Orban, Temasek Life Sciences Laboratory, Singapore</p><p><strong>Received:</strong> October 18, 2006; <strong>Accepted:</strong> January 16, 2007; <strong>Published:</strong> February 14, 2007</p><p><strong>Copyright:</strong> © 2007 Fu et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>This study was financially supported by the Chinese Academy of Sciences, by the Max Planck Society and by the Bundesministerium für Bildung und Forschung, as well as by the National Natural Science Foundation (grant 30425021) and by the Proteomage Project from FP6 of EU. None of these institutions play any direct or indirect role in in the design and conduct of the study, in the collection, analysis, and interpretation of the data, and in the preparation, review, or approval of the manuscript.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>Microarray technology has become the method of choice for studying messenger RNA (mRNA) expression levels for thousands of genes in cells or tissues subjected to different conditions, or derived from different species <a href="#pone.0000216-Preuss1">[1]</a>. Biological processes, however, are normally driven by proteins. Still, methods for studying protein expression levels are much more laborious and costly, and more limited in scope, than mRNA measurements. The methodologies based on two-dimensional gel electrophoresis or shotgun mass spectrometry, for instance, are imprecise and biased towards highly expressed proteins <a href="#pone.0000216-Hack1">[2]</a>. Introduction of quantitative proteomics approaches, such as isotope coded affinity tags (ICAT), allow obtaining more precise protein abundance estimates for a broader range of protein concentrations. However, this approach is currently limited to the identification of a few hundred proteins in a given sample <a href="#pone.0000216-Ideker1">[3]</a>–<a href="#pone.0000216-Tian1">[5]</a>.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>For these reasons most current studies use mRNA expression levels as a proxy for estimating functional differences that occur at the protein level. This strategy assumes that differences in mRNA levels actually reflect differences in protein expression. Most studies addressing this issue to date, however, describe the correlation between mRNA and protein expression as moderately or weakly positive with correlation coefficients ranging from 0.2 to 0.6 <a href="#pone.0000216-Griffin1">[4]</a>–<a href="#pone.0000216-Cox1">[7]</a>. This lack of agreement can in part be explained by the technical imprecision of methods used to determine their expression levels <a href="#pone.0000216-Hack1">[2]</a>. Biologically, the discrepancies between mRNA and protein expression can be caused by posttranscriptional regulation, as well as differences in mRNA and protein turnover rates <a href="#pone.0000216-Hack1">[2]</a>, <a href="#pone.0000216-Cox1">[7]</a>.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>Further, all studies to date, investigated mRNA and protein expression correlation within one species in different developmental or physiological states, or among tissues within the same organism <a href="#pone.0000216-Griffin1">[4]</a>–<a href="#pone.0000216-Kislinger1">[11]</a>. Unlike changes observed in response to functional stimuli within an organism or a species, a large proportion of mRNA expression differences observed between mammalian species are caused by random genetic mutations and, analogous to the DNA sequence changes, appear to have no functional consequences (see <a href="#pone.0000216-Khaitovich1">[12]</a> for the review). This may be due to the fact that many or most mRNA expression differences are not reflected at the protein level and thus do not exert any influence on the organisms phenotype. If this is true, we expect to see much weaker correlation between mRNA and protein expression differences observed between species than between the ones observed in response to functional stimuli. Furthermore, the observed protein expression differences would then largely reflect functional adaptations observed in species phenotypes. Such a finding would be of particular importance for studies of human evolution, where, despite intensive effort, the identification of genomic or mRNA expression changes underlying human-specific phenotypic features has been a daunting and hitherto largely elusive task <a href="#pone.0000216-Consortium.1">[13]</a>, <a href="#pone.0000216-Khaitovich2">[14]</a>. On the other hand, the finding that evolutionary changes in mRNA expression are reflected at the protein level would have important implications for our views of regulatory evolution.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3><a id="article1.body1.sec2.p1" name="article1.body1.sec2.p1"></a><p>To investigate whether mRNA expression differences observed between humans and chimpanzees are reflected in differences in protein expression, we measured mRNA expression levels of all annotated genes in six human and six chimpanzee livers, using microarray probes that perfectly matched the human as well as the chimpanzee genome sequences. In addition, we measured protein expression levels in six independent pools of tissue extracts, each containing two of the human or two of the chimpanzee samples, respectively, in two sets of experimental replicates, using ICAT profiling (<a href="#s4">Materials and Methods</a>).</p>
<a id="article1.body1.sec2.p2" name="article1.body1.sec2.p2"></a><p>First, we tested our ability to distinguish the human and chimpanzee samples based on protein expression. Using either tree construction or hierarchical clustering based on 113 proteins detected in all 12 samples, we find that the human and the chimpanzee protein expression profiles are clearly distinct (<a href="#pone-0000216-g001">Figure 1B</a>, <a href="#pone.0000216.s001">Figure S1</a>). Further, although a direct comparison between mRNA and protein expression variation is not possible, the overall extent of intra- and inter-species variation appears to be similar (<a href="#pone-0000216-g001">Figure 1</a>). Compared with mRNA expression, protein expression exhibits greater intra-species variation. Large experimental variation associated with protein expression measures is likely, at least in part, to explain this variation (<a href="#s4">Materials and Methods</a>).</p>
<div class="figure" id="pone-0000216-g001"><div class="img"><a name="pone-0000216-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000216.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000216" data-uri="info:doi/10.1371/journal.pone.0000216.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000216.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000216.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000216.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000216.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000216.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000216.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000216.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000216.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000216.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1. </strong></p><a id="article1.body1.sec2.fig1.caption1.p1" name="article1.body1.sec2.fig1.caption1.p1"></a><p>Schematic representation of gene expression variation within and between species on mRNA (A) and protein (B) levels. The trees are inferred from the mean of the squared difference of expression intensities for 98 genes detected on both mRNA and protein levels. Each pool represents an average expression in two chimpanzee (c) or two human (h) individuals. Additionally, protein measurements are based on two independent experimental replicates.</p>
<span>doi:10.1371/journal.pone.0000216.g001</span></div><a id="article1.body1.sec2.p3" name="article1.body1.sec2.p3"></a><p>Next, we tested whether gene expression differences between humans and chimpanzees observed on the mRNA level reflect differences in protein expression. For 98 genes detected on both protein and mRNA levels, we find a significant positive correlation between differences in mRNA and protein expression (<em>R</em> = 0.33, <em>p</em>&lt;0.001) (<a href="#pone-0000216-g002">Figure 2A</a>). However, the vast majority of the 98 genes do not show large expression differences between the species, neither in terms of protein levels nor in terms of mRNA levels. This is not surprising, given that approximately 90% of genes do not differ significantly in their mRNA expression levels in somatic tissues between humans and chimpanzees <a href="#pone.0000216-Khaitovich2">[14]</a>. When we restrict our analysis to 15 genes with a significant mRNA expression difference between species (Student's t-test, <em>p</em>&lt;0.01, FDR = 4%), we find a much stronger correlation (<em>R</em> = 0.72, <em>p</em> = 0.002). Further, we find a similar correlation (<em>R</em> = 0.55, <em>p</em> = 0.005) on the protein level (Student's t-test, <em>p</em>&lt;0.01, FDR = 3%) for 24 genes differently expressed between humans and chimpanzees (<a href="#pone-0000216-g002">Figure 2B</a>).</p>
<div class="figure" id="pone-0000216-g002"><div class="img"><a name="pone-0000216-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000216.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000216" data-uri="info:doi/10.1371/journal.pone.0000216.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000216.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000216.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000216.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000216.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000216.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000216.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000216.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000216.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000216.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2. </strong></p><a id="article1.body1.sec2.fig2.caption1.p1" name="article1.body1.sec2.fig2.caption1.p1"></a><p>Comparison of protein and mRNA expression differences between humans and chimpanzees. Comparisons are shown for 98 genes, detected on both mRNA and protein levels (A), and for 33 genes showing significant differences in mRNA (⧫) or protein (+) expression between the species (B). Six genes significantly different on both mRNA and protein expression levels are shown using overlapping labels. The names and the functional annotation of the 33 genes are included in <a href="#pone.0000216.s010">table S5</a>. Expression differences are shown using a base-two logarithmic scale. The dotted line represents an ideal regression line (α = 0, β = 1).</p>
<span>doi:10.1371/journal.pone.0000216.g002</span></div><a id="article1.body1.sec2.p4" name="article1.body1.sec2.p4"></a><p>Finally, we tested whether the observed correlation can be reproduced using the protein expression data from each of the two sets of experimental replicates separately. We find a significant correlation between mRNA and protein expression differences between humans and chimpanzees for 143 and 159 genes detected in the first and second sets of experimental replicates (<em>R</em> = 0.37, <em>p</em>&lt;0.000005 and <em>R</em> = 0.28, <em>p</em>&lt;0.0005, respectively) as well as for 24 and 23 genes with a significant mRNA expression difference between species (Student's t-test, <em>p</em>&lt;0.01, FDR = 5%) (<em>R</em> = 0.68, <em>p</em>&lt;0.0005 and <em>R</em> = 0.48, <em>p</em>&lt;0.05, respectively) (<a href="#pone.0000216.s002">Figure S2</a>).</p>
</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p>We reproducibly find a significant positive correlation between mRNA expression and protein expression differences seen between humans and chimpanzees. We find this positive correlation not only for genes showing significant expression differences between species, but for all detected genes. Further, it is present in each set of replicates as well as the combined dataset. Furthermore, the observed correlation is not simply the result of a strong effect in a small subset of genes, as indicated by bootstrapping all detected genes 10,000 times (<em>p</em> = 0.01 and <em>p</em> = 0.05 for the first and the second set of replicates, respectively) (<a href="#pone.0000216.s003">Figure S3</a>).</p>
<a id="article1.body1.sec3.p2" name="article1.body1.sec3.p2"></a><p>Although our observation is currently limited to a relatively small number of samples from a single tissue, we are able to reproduce the result in two independent sets of experimental replicates, despite the substantial experimental noise associated with protein expression measurements. In fact, since technical variation is expected to be random, the actual correlation is likely to be better than that observed. Further, the observed correlation exists even though we use largely non-overlapping sets of individuals for the mRNA and protein expression measurements. The inclusion of variation caused by expression differences between samples used for protein and mRNA measurements means that we expect the actual correlation between protein and mRNA expression to be better than that observed. Together, these features indicate that the correlation we observe, though weakened by technical effects, is unlikely to be spurious.</p>
<a id="article1.body1.sec3.p3" name="article1.body1.sec3.p3"></a><p>Genes detected on the protein level in this study tend to have a higher average mRNA expression level than other genes. However, their distribution covers the entire expression spectrum (<a href="#pone.0000216.s004">Figure S4</a>). With current data it is not possible to say to what extent the observed results may apply to a broader set of genes or to other tissues.</p>
<a id="article1.body1.sec3.p4" name="article1.body1.sec3.p4"></a><p>We find that for the 98 genes detected on both mRNA and protein levels in the two datasets, there is a significant enrichment for genes involved in metabolic processes (GO ontology reference) when compared to genes expressed on the mRNA level alone (<a href="#pone.0000216.s009">Table S1</a>). However, of these the 33 genes differently expressed between the species and showing the strongest correlation do not differ significantly in their function from the bulk of detected genes. Thus, the protein and mRNA expression differences that correlate the most do not appear to cluster in any particular biological process.</p>
<a id="article1.body1.sec3.p5" name="article1.body1.sec3.p5"></a><p>When all detected genes are considered, the strength of the correlation between expression differences seen on the mRNA and protein levels appears weak (<em>R</em> = 0.28–0.37). However, similar correlations have been reported in comparisons between different developmental stages in mice (<em>R</em> = 0.18) <a href="#pone.0000216-Cox1">[7]</a>, between yeasts cultured on rich or minimal media (<em>R</em> = 0.45) <a href="#pone.0000216-Washburn1">[6]</a>, or between two murine hematopoetic cell lines (<em>R</em> = 0.59) <a href="#pone.0000216-Tian1">[5]</a>, where the results for all detected genes were reported. Since some proteins are detected in just one of the two datasets, our total analysis is based on comparison of protein and mRNA expression levels for 206 genes. Other studies have used similar numbers of proteins for comparison of ICAT and mRNA microarray methodologies (N = 245, 289, 425) <a href="#pone.0000216-Ideker1">[3]</a>–<a href="#pone.0000216-Tian1">[5]</a>. Thus, the correlation we find between evolutionary differences in protein and mRNA expression is comparable, or at least not worse than, those observed between different physiological states in an organism.</p>
<a id="article1.body1.sec3.p6" name="article1.body1.sec3.p6"></a><p>Further, approximately 90% of genes expressed in liver do not differ significantly in their mRNA expression levels between humans and chimpanzees. Thus, for the majority of these we cannot expect to find any correlation between proteins and mRNA based on the expression differences between species. As expected, when we limit our analysis to genes that show significant differences between species, the correlation improves considerably (<em>R</em> = 0.5–0.7, see <a href="#s2">Results</a>). Still, this result is based on several dozens of genes and needs to be further confirmed using a much larger dataset. Nonetheless, these results provide the first indication that the mRNA expression differences detected between humans and chimpanzees may to a large extent reflect protein expression differences.</p>
<a id="article1.body1.sec3.p7" name="article1.body1.sec3.p7"></a><p>This indication, if confirmed, poses new questions, rather than resolving existing ones. Namely, if the vast majority of evolutionary differences in mRNA expression between closely related species does not affect phenotype, then the majority of evolutionary differences in protein expression are also likely to be of little or no consequence for the phenotype. Alternatively, the observed correlation may indicate that a considerable number of mRNA expression differences between species, particularly those of large amplitudes, might have some functional effect. Nonetheless, before any general conclusions regarding the relation of mRNA and protein expression differences between species can be made, further studies covering more genes and tissues are necessary.</p>
</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4>Tissue samples and microarray data collection</h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p>All chimpanzee individuals used in this study belonged to the eastern chimpanzee population. All tissue samples were obtained from individuals that suffered sudden deaths for reasons other than their participation in this study and without any relation to the tissues used. All samples had no histological abnormalities and showed no detectable RNA degradation, indicating good tissue preservation (<a href="#pone.0000216.s006">Table S2</a>). Tissue samples from four of the six chimpanzee individuals were used in both protein and mRNA expression analysis, while the remaining tissue samples used for protein and mRNA expression analysis did not overlap. Human and chimpanzee samples were matched with respect to sex and relative age as closely as possible, with no sex or age bias shared between the samples used for mRNA and protein measurements (<a href="#pone.0000216.s006">Table S2</a>).</p>
<a id="article1.body1.sec4.sec1.p2" name="article1.body1.sec4.sec1.p2"></a><p>mRNA expression levels were determined by analyzing a published gene expression dataset for six human and five chimpanzee livers with added expression data taken from one chimpanzee individual <a href="#pone.0000216-Khaitovich2">[14]</a>. All samples in this dataset were prepared, labeled and hybridized to Affymetrix® HG U133plus2 arrays in one batch, following the procedure described in <a href="#pone.0000216-Khaitovich2">[14]</a>. Namely, total RNA was extracted from 100 mg of frozen tissue dissected from the peripheral part of liver lobe in the TRIZol® reagent using an electric homogenizer (Schütt) at 3,000 rpm. Further, total RNA was isolated according to the TRIZol® manufacturer's instructions and purified with a QIAGEN® MiniElute® kit, following the protocol supplied by the manufacturer without modifications. All samples were processed, labeled and hybridized to the microarrays following the protocol described in the GeneChip® Expression Analysis Technical Manual (<a href="http://www.affymetrix.com/support/technical/manuals.affx">http://www.affymetrix.com/support/techni​cal/manuals.affx</a>) without any modifications.</p>
<a id="article1.body1.sec4.sec1.p3" name="article1.body1.sec4.sec1.p3"></a><p>All samples had a high and comparable RNA quality, as determined by both the ratio of 28S to 18S ribosomal RNAs, estimated using the Agilent® 2100 Bioanalyser® system (<a href="#pone.0000216.s006">Table S2</a>), and the signal ratios between the probes for the 5′ and 3′ ends of the mRNAs of GAPDH, which are used as quality controls on Affymetrix® microarrays. All primary expression data are publicly available at the ArrayExpress database (<a href="http://www.ebi.ac.uk/arrayexpress/">http://www.ebi.ac.uk/arrayexpress/</a>), accession number E-AFMX-11.</p>


<h4>Microarray data analysis</h4>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1"></a><p>Prior to microarray data analysis, we removed all oligonucleotide array probes that did not perfectly match both human and chimpanzee genome sequences, or had a significant difference between these two species in their hybridization patterns relative to the other probes of the same set, as described in <a href="#pone.0000216-Khaitovich2">[14]</a>. This reduced the number of probe sets available for analysis on each array from 54,675 to 51,522. mRNA expression values were calculated using the Robust Multichip Average (rma) algorithm <a href="#pone.0000216-Irizarry1">[15]</a>. Detection <em>p</em>-values were calculated using the “mas5calls” function included in the Bioconductor “affy” package (<a href="http://www.bioconductor.org/">http://www.bioconductor.org/</a>). Only probe sets with expression levels detected in at least two samples (detection <em>p</em>-value&lt;0.05) were considered for further analysis. This further reduced the number of probe sets available for analysis to 23,856. Since many genes are represented by more than one probe set on the Affymetrix® HG-U133Plus2 arrays, we calculated the mean expression values for each gene, as defined by the RefSeq annotation, Affymetrix® annotation tables, December 2005 version (<a href="http://www.affymetrix.com/index.affx">http://www.affymetrix.com/index.affx</a>).</p>


<h4>Protein data collection</h4>
<a id="article1.body1.sec4.sec3.p1" name="article1.body1.sec4.sec3.p1"></a><p>Protein expression levels were determined in liver samples from six humans and six chimpanzees. Prior to protein isolation, we combined samples from two individuals of one species in one pooled sample, thus creating 3 independent sample pools for each species (<a href="#pone.0000216.s006">Table S2</a>). The pooled samples were minced, washed with ice-cold PBS to remove blood, and homogenized in an ice-cold lysis buffer (8 M urea, 4% CHAPS, 65 mM DTT, 40 mM Tris, 200 mg of tissue/1 ml), using an electric homogenizer (Glas-Col). The homogenized solution was sonicated for total of 3 minutes (5 seconds sonication time with 10 seconds intervals). After that, the sample solution was then centrifuged at 10,000 g for 1 hour at 4°C. The protein supernatant was transferred into a new tube and then frozen at −80°C.</p>
<a id="article1.body1.sec4.sec3.p2" name="article1.body1.sec4.sec3.p2"></a><p>ICAT labeling was performed using a Cleavable ICAT<sup>TM</sup> Reagent Kit (Applied Biosystems, Foster City, CA) according to the manufacturer's guidelines, with the few modifications described in <a href="#pone.0000216-Jiang1">[16]</a>. Orthogonal 2D-LC-MS/MS was performed in an ion-trap mass spectrometer (LTQ, Thermo Finnigan). The system was fitted with a strong cation exchange column (CTIBiphase, SCX 0.32 mm×5 cm, COLUMN Techology Inc.) and two C18 reversed-phase columns (sample trap, C18,300 Å, COLUMN Techology lnc). For the ion exchange step, we used a pH gradient with steps at pH 3.0, 3.5, 4.0, 4.5, 5.0, 5.5, 6.0, 7.0, 8.0, 8.5. For the reverse phase step, we used 0.1% formic acid in either water (buffer A) or ACN (buffer B). We used buffer B gradient from 0 to 35% during the first 165 minutes, and from 35% to 95% in the following 15 minutes. One full MS scan was then followed by ten MS/MS scans on the ten most intense ions from the MS spectrum, according to a dynamic exclusion setting: repeat count, 2; repeat duration, 30 seconds; exclusion duration, 90 seconds. In each measurement, individual samples were measured together with a common reference sample comprised of all 12 samples used in this study. In the second set of replicates, the ICAT labels used for individual samples and common reference were inverted.</p>


<h4>Protein expression analysis</h4>
<a id="article1.body1.sec4.sec4.p1" name="article1.body1.sec4.sec4.p1"></a><p>For protein identification and statistical validation, the acquired MS/MS spectra were automatically searched against <em>ipi.human</em> version 3.07 database using the Turbo SEQUEST program in the BioWorks™ 3.1 software suite. SEQUEST results were filtered using the following parameters: Xcorr (a cross-correlation value) was greater or equal to 1.9 for the peptides in a +1 charged state, 2.2 for ones in a +2 charged state and 3.75 for ones in a +3 charged state, where the delta Cn is greater or equal to 0.1. The accepted peptide quantification was achieved using the Relex software and partial manual verification. The peptide database search was carried out with the following parameters: peptide mass tolerance = 3.0000; fragment ion tolerance = 0.0000; maximum number of internal cleavage sites = 2; number of allowed errors in matching auto-detected peaks = 1; mass tolerance for matching auto-detected peaks = 1.0000.</p>
<a id="article1.body1.sec4.sec4.p2" name="article1.body1.sec4.sec4.p2"></a><p>Following this procedure, we identified and quantified expression levels for 169 proteins in all 6 sample pools in the first experiment, and 190 proteins in the second experiment. 113 of these proteins were identified in both experiments. Protein expression was calculated as a ratio of the protein expression level in a pooled sample to the protein expression level in a common reference sample comprised of all 12 samples used in this study. For further analysis, the ratios were base-two logarithm transformed.</p>
<a id="article1.body1.sec4.sec4.p3" name="article1.body1.sec4.sec4.p3"></a><p>We found that protein expression measures among individuals of the same species are associated with large variation. Even though we found a significant correlation of protein expression values across most of the pairs of individuals, the average correlation coefficient was relatively low (<em>R</em> = 0.31). Further, the levels of variation observed between individuals within a species ranged widely, as reflect by the correlation coefficient distribution (<a href="#pone.0000216.s005">Figure S5</a>). We find that the lowest observed correlation was due to just two points with the greatest influence on the linear model fit indicated by Cook's distances. Removal of points with the Cook's distance greater than 0.1, (less than 5% of the total points) improved the correlation observed among individuals within the species (<a href="#pone.0000216.s005">Figure S5</a>).</p>
<a id="article1.body1.sec4.sec4.p4" name="article1.body1.sec4.sec4.p4"></a><p>More importantly, however, we found that the correlation between replicate measurements of the same protein sample were substantially better than the correlations observed among the individuals (mean <em>R</em> = 0.48, range 0.38–0.60, <em>p</em>&lt;0.0005) (<a href="#pone.0000216.s005">Figure S5</a>). Given that the experimental replicates were prepared in two completely independent routines, including flipping mass tags and measuring them in two series separated by more than a month, the observed correlations indicate that the protein measurements, albeit noisy, accurately reflect protein abundance levels. Further, we can conclude that experimental variation is unlikely to explain more than a half of the total variation observed among individuals of the same species.</p>


<h4>Combined data analysis</h4>
<a id="article1.body1.sec4.sec5.p1" name="article1.body1.sec4.sec5.p1"></a><p>We combined protein and mRNA expression data on a gene-by-gene basis using RefSeq annotation (<a href="http://www.ncbi.nlm.nih.gov/RefSeq/">http://www.ncbi.nlm.nih.gov/RefSeq/</a>). Both mRNA and protein expression measurements were available for 143 genes represented by 682 probe sets on the microarray for the first set of experimental replicates for 159 genes represented by 572 probe sets on the microarray for the second set of experimental replicates, and for 98 genes represented by 442 probe sets on the microarray for both sets. Protein and mRNA expression values for 143 and 159 genes detected in the first and second sets of experimental replicates are listed in <a href="#pone.0000216.s007">tables S3</a> and <a href="#pone.0000216.s008">S4</a>, respectively. For the correlation analysis, we calculated differences between humans and chimpanzees as a difference between mean expression values within species, for all genes detected on both mRNA and protein levels. Genes differently expressed between humans and chimpanzees either on protein or mRNA expression levels were identified using Student's t-test. The false discovery rate (FDR) was calculated in 1,000 random permutations of sample labels. The positive correlation observed between mRNA and protein expression differences did not depend a great deal on the significance cutoff chosen to define differently expressed genes. Thus, for the 143 proteins detected in the first set of experimental replicates, at cutoff <em>p</em>&lt;0.005, FDR = 2%, we found 18 genes with correlation <em>R</em> = 0.70, <em>p</em> = 0.001; at cutoff <em>p</em>&lt;0.01, FDR = 4%, we found 24 genes with correlation <em>R</em> = 0.68, <em>p</em> = 0.0003; and at cutoff <em>p</em>&lt;0.02, FDR = 6%, we found 31 genes with correlation <em>R</em> = 0.58, <em>p</em> = 0.0006.</p>
<a id="article1.body1.sec4.sec5.p2" name="article1.body1.sec4.sec5.p2"></a><p>For tree construction, the distances between each pair of samples were calculated as a mean of squared differences between the intensities of expressed genes. We used the resulting data matrixes to build UPGMA trees using PHYLIP <a href="#pone.0000216-Felsenstein1">[17]</a>. For the tree based on mRNA expression, we created pooled samples by calculating the mean expression value for each gene based on expression levels in two individuals of the same species (<a href="#pone.0000216.s006">Table S2</a>). For the tree based on protein expression, we calculated the mean expression value for each gene based on expression levels in two experimental replicates (<a href="#pone.0000216.s006">Table S2</a>). Thus, for both mRNA and protein data, such pooling reduces both individual and experimental variation.</p>
<a id="article1.body1.sec4.sec5.p3" name="article1.body1.sec4.sec5.p3"></a><p>Hierarchical clustering of protein data was completed using average linkage clustering, and visualized using Cluster and TreeView software, respectively (<a href="http://rana.lbl.gov/EisenSoftware.htm">http://rana.lbl.gov/EisenSoftware.htm</a>). Before clustering, protein expression values were normalized on a gene-by-gene basis and the mean expression of each sample was centered to the same value. Gene ontology analysis was carried out using publicly available on-line statistical package (FUNC, <a href="http://func.eva.mpg.de/">http://func.eva.mpg.de/</a>) using a hypergeometric test. Genes were assigned GO annotations using the Ensembl annotation tool BioMart (<a href="http://www.ensembl.org/Multi/martview">http://www.ensembl.org/Multi/martview</a>).</p>

</div>

<div id="section5" class="section"><a id="s5" name="s5" toc="s5" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0000216"></div><a name="pone.0000216.s001" id="pone.0000216.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000216.s001">Figure S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1"></a><p class="preSiDOI">Hierarchical clustering of 12 liver samples based on protein expression levels of 113 genes, detected in all samples</p>
<p class="siDoi">doi:10.1371/journal.pone.0000216.s001</p><a id="article1.body1.sec5.supplementary-material1.caption1.p2" name="article1.body1.sec5.supplementary-material1.caption1.p2"></a><p class="postSiDOI">(0.03 MB DOC)</p>
<a name="pone.0000216.s002" id="pone.0000216.s002"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000216.s002">Figure S2. </a></strong></p><a id="article1.body1.sec5.supplementary-material2.caption1.p1" name="article1.body1.sec5.supplementary-material2.caption1.p1"></a><p class="preSiDOI">Comparison of protein and mRNA expression differences between humans and chimpanzees</p>
<p class="siDoi">doi:10.1371/journal.pone.0000216.s002</p><a id="article1.body1.sec5.supplementary-material2.caption1.p2" name="article1.body1.sec5.supplementary-material2.caption1.p2"></a><p class="postSiDOI">(0.24 MB DOC)</p>
<a name="pone.0000216.s003" id="pone.0000216.s003"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000216.s003">Figure S3. </a></strong></p><a id="article1.body1.sec5.supplementary-material3.caption1.p1" name="article1.body1.sec5.supplementary-material3.caption1.p1"></a><p class="preSiDOI">Bootstrap analysis of Pearson's correlation coefficients and correlation p-values observed in the first and in the second set of experimental replicates</p>
<p class="siDoi">doi:10.1371/journal.pone.0000216.s003</p><a id="article1.body1.sec5.supplementary-material3.caption1.p2" name="article1.body1.sec5.supplementary-material3.caption1.p2"></a><p class="postSiDOI">(0.10 MB DOC)</p>
<a name="pone.0000216.s004" id="pone.0000216.s004"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000216.s004">Figure S4. </a></strong></p><a id="article1.body1.sec5.supplementary-material4.caption1.p1" name="article1.body1.sec5.supplementary-material4.caption1.p1"></a><p class="preSiDOI">Distributions of mRNA expression levels (on logarithmic two scale) for 143 genes identified on the protein expression level (left), and remaining genes detected on the mRNA expression level</p>
<p class="siDoi">doi:10.1371/journal.pone.0000216.s004</p><a id="article1.body1.sec5.supplementary-material4.caption1.p2" name="article1.body1.sec5.supplementary-material4.caption1.p2"></a><p class="postSiDOI">(0.08 MB DOC)</p>
<a name="pone.0000216.s005" id="pone.0000216.s005"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000216.s005">Figure S5. </a></strong></p><a id="article1.body1.sec5.supplementary-material5.caption1.p1" name="article1.body1.sec5.supplementary-material5.caption1.p1"></a><p class="preSiDOI">Distribution of Pearson's correlation coefficients for the comparisons between individuals within a species and between the experimental replicates</p>
<p class="siDoi">doi:10.1371/journal.pone.0000216.s005</p><a id="article1.body1.sec5.supplementary-material5.caption1.p2" name="article1.body1.sec5.supplementary-material5.caption1.p2"></a><p class="postSiDOI">(0.11 MB DOC)</p>
<a name="pone.0000216.s006" id="pone.0000216.s006"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000216.s006">Table S2. </a></strong></p><a id="article1.body1.sec5.supplementary-material6.caption1.p1" name="article1.body1.sec5.supplementary-material6.caption1.p1"></a><p class="preSiDOI">Sample information</p>
<p class="siDoi">doi:10.1371/journal.pone.0000216.s006</p><a id="article1.body1.sec5.supplementary-material6.caption1.p2" name="article1.body1.sec5.supplementary-material6.caption1.p2"></a><p class="postSiDOI">(0.02 MB XLS)</p>
<a name="pone.0000216.s007" id="pone.0000216.s007"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000216.s007">Table S3. </a></strong></p><a id="article1.body1.sec5.supplementary-material7.caption1.p1" name="article1.body1.sec5.supplementary-material7.caption1.p1"></a><p class="preSiDOI">mRNA and protein expression levels for 143 proteins detected in the first set of experimental replicates</p>
<p class="siDoi">doi:10.1371/journal.pone.0000216.s007</p><a id="article1.body1.sec5.supplementary-material7.caption1.p2" name="article1.body1.sec5.supplementary-material7.caption1.p2"></a><p class="postSiDOI">(0.08 MB XLS)</p>
<a name="pone.0000216.s008" id="pone.0000216.s008"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000216.s008">Table S4. </a></strong></p><a id="article1.body1.sec5.supplementary-material8.caption1.p1" name="article1.body1.sec5.supplementary-material8.caption1.p1"></a><p class="preSiDOI">mRNA and protein expression levels for 159 proteins detected in the second set of experimental replicates</p>
<p class="siDoi">doi:10.1371/journal.pone.0000216.s008</p><a id="article1.body1.sec5.supplementary-material8.caption1.p2" name="article1.body1.sec5.supplementary-material8.caption1.p2"></a><p class="postSiDOI">(0.09 MB XLS)</p>
<a name="pone.0000216.s009" id="pone.0000216.s009"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000216.s009">Table S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material9.caption1.p1" name="article1.body1.sec5.supplementary-material9.caption1.p1"></a><p class="preSiDOI">GO categories with significant overrepresentation of 98 genes detected on the protein and mRNA levels</p>
<p class="siDoi">doi:10.1371/journal.pone.0000216.s009</p><a id="article1.body1.sec5.supplementary-material9.caption1.p2" name="article1.body1.sec5.supplementary-material9.caption1.p2"></a><p class="postSiDOI">(0.04 MB XLS)</p>
<a name="pone.0000216.s010" id="pone.0000216.s010"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000216.s010">Table S5. </a></strong></p><a id="article1.body1.sec5.supplementary-material10.caption1.p1" name="article1.body1.sec5.supplementary-material10.caption1.p1"></a><p class="preSiDOI">Functional annotation of 33 genes with significant expression difference between humans and chimpanzees</p>
<p class="siDoi">doi:10.1371/journal.pone.0000216.s010</p><a id="article1.body1.sec5.supplementary-material10.caption1.p2" name="article1.body1.sec5.supplementary-material10.caption1.p2"></a><p class="postSiDOI">(0.03 MB XLS)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>We thank C. Allen and H. McClure of the Yerkes Primate Center, Atlanta, Georgia and W. Collignon and R. Bontrop of the Biomedical Primate Research Centre, Rijswijk, Netherlands for chimpanzee samples; Christine Green for critical reading of the manuscript and all members of our group for helpful discussions.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: SP PK RZ. Performed the experiments: PK NF ID. Analyzed the data: PK JK NF. Contributed reagents/materials/analysis tools: SP RZ JK JW. Wrote the paper: SP PK RZ JK NF.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0000216-Preuss1" id="pone.0000216-Preuss1"></a>Preuss TM, Caceres M, Oldham MC, Geschwind DH (2004) Human brain evolution: insights from microarrays. Nat Rev Genet  5: 850–860.  <ul class="find" data-citedArticleID="930624" data-doi="10.1038/nrg1469"><li><a href="http://dx.doi.org/10.1038/nrg1469" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Human+brain+evolution%3A+insights+from+microarrays." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Human+brain+evolution%3A+insights+from+microarrays.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0000216-Hack1" id="pone.0000216-Hack1"></a>Hack CJ (2004) Integrated transcriptome and proteome data: the challenges ahead. Brief Funct Genomic Proteomic  3: 212–219.  <ul class="find" data-citedArticleID="930606" data-doi="10.1093/bfgp/3.3.212"><li><a href="http://dx.doi.org/10.1093/bfgp/3.3.212" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Integrated+transcriptome+and+proteome+data%3A+the+challenges+ahead." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Integrated+transcriptome+and+proteome+data%3A+the+challenges+ahead.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0000216-Ideker1" id="pone.0000216-Ideker1"></a>Ideker T, Thorsson V, Ranish JA, Christmas R, Buhler J, et al.  (2001) Integrated genomic and proteomic analyses of a systematically perturbed metabolic network. Science  292: 929–934.  <ul class="find" data-citedArticleID="930608" data-doi="10.1126/science.292.5518.929"><li><a href="http://dx.doi.org/10.1126/science.292.5518.929" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Integrated+genomic+and+proteomic+analyses+of+a+systematically+perturbed+metabolic+network." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Integrated+genomic+and+proteomic+analyses+of+a+systematically+perturbed+metabolic+network.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0000216-Griffin1" id="pone.0000216-Griffin1"></a>Griffin TJ, Gygi SP, Ideker T, Rist B, Eng J, et al.  (2002) Complementary profiling of gene expression at the transcriptome and proteome levels in Saccharomyces cerevisiae. Mol Cell Proteomics  1: 323–333.  <ul class="find" data-citedArticleID="930602" data-doi="10.1074/mcp.m200001-mcp200"><li><a href="http://dx.doi.org/10.1074/mcp.m200001-mcp200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Complementary+profiling+of+gene+expression+at+the+transcriptome+and+proteome+levels+in+Saccharomyces+cerevisiae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Complementary+profiling+of+gene+expression+at+the+transcriptome+and+proteome+levels+in+Saccharomyces+cerevisiae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0000216-Tian1" id="pone.0000216-Tian1"></a>Tian Q, Stepaniants SB, Mao M, Weng L, Feetham MC, et al.  (2004) Integrated genomic and proteomic analyses of gene expression in Mammalian cells. Mol Cell Proteomics  3: 960–969.  <ul class="find" data-citedArticleID="930626" data-doi="10.1074/mcp.m400055-mcp200"><li><a href="http://dx.doi.org/10.1074/mcp.m400055-mcp200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Integrated+genomic+and+proteomic+analyses+of+gene+expression+in+Mammalian+cells." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Integrated+genomic+and+proteomic+analyses+of+gene+expression+in+Mammalian+cells.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0000216-Washburn1" id="pone.0000216-Washburn1"></a>Washburn MP, Koller A, Oshiro G, Ulaszek RR, Plouffe D, et al.  (2003) Protein pathway and complex clustering of correlated mRNA and protein expression analyses in Saccharomyces cerevisiae. Proc Natl Acad Sci U S A  100: 3107–3112.  <ul class="find" data-citedArticleID="930628" data-doi="10.1073/pnas.0634629100"><li><a href="http://dx.doi.org/10.1073/pnas.0634629100" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Protein+pathway+and+complex+clustering+of+correlated+mRNA+and+protein+expression+analyses+in+Saccharomyces+cerevisiae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Protein+pathway+and+complex+clustering+of+correlated+mRNA+and+protein+expression+analyses+in+Saccharomyces+cerevisiae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0000216-Cox1" id="pone.0000216-Cox1"></a>Cox B, Kislinger T, Emili A (2005) Integrating gene and protein expression data: pattern analysis and profile mining. Methods  35: 303–314.  <ul class="find" data-citedArticleID="930598" data-doi="10.1016/j.ymeth.2004.08.021"><li><a href="http://dx.doi.org/10.1016/j.ymeth.2004.08.021" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Integrating+gene+and+protein+expression+data%3A+pattern+analysis+and+profile+mining." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Integrating+gene+and+protein+expression+data%3A+pattern+analysis+and+profile+mining.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0000216-Gygi1" id="pone.0000216-Gygi1"></a>Gygi SP, Rochon Y, Franza BR, Aebersold R (1999) Correlation between protein and mRNA abundance in yeast. Mol Cell Biol  19: 1720–1730.  <ul class="find" data-citedArticleID="930604"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Correlation+between+protein+and+mRNA+abundance+in+yeast.&amp;auth=&amp;atitle=Correlation+between+protein+and+mRNA+abundance+in+yeast." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Correlation+between+protein+and+mRNA+abundance+in+yeast." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Correlation+between+protein+and+mRNA+abundance+in+yeast.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0000216-Mootha1" id="pone.0000216-Mootha1"></a>Mootha VK, Bunkenborg J, Olsen JV, Hjerrild M, Wisniewski JR, et al.  (2003) Integrated analysis of protein composition, tissue diversity, and gene regulation in mouse mitochondria. Cell  115: 629–640.  <ul class="find" data-citedArticleID="930622" data-doi="10.1016/s0092-8674(03)00926-7"><li><a href="http://dx.doi.org/10.1016/s0092-8674(03)00926-7" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Integrated+analysis+of+protein+composition%2C+tissue+diversity%2C+and+gene+regulation+in+mouse+mitochondria." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Integrated+analysis+of+protein+composition%2C+tissue+diversity%2C+and+gene+regulation+in+mouse+mitochondria.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0000216-Mijalski1" id="pone.0000216-Mijalski1"></a>Mijalski T, Harder A, Halder T, Kersten M, Horsch M, et al.  (2005) Identification of coexpressed gene clusters in a comparative analysis of transcriptome and proteome in mouse tissues. Proc Natl Acad Sci U S A  102: 8621–8626.  <ul class="find" data-citedArticleID="930620" data-doi="10.1073/pnas.0407672102"><li><a href="http://dx.doi.org/10.1073/pnas.0407672102" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Identification+of+coexpressed+gene+clusters+in+a+comparative+analysis+of+transcriptome+and+proteome+in+mouse+tissues." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Identification+of+coexpressed+gene+clusters+in+a+comparative+analysis+of+transcriptome+and+proteome+in+mouse+tissues.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0000216-Kislinger1" id="pone.0000216-Kislinger1"></a>Kislinger T, Cox B, Kannan A, Chung C, Hu P, et al.  (2006) Global survey of organ and organelle protein expression in mouse: combined proteomic and transcriptomic profiling. Cell  125: 173–186.  <ul class="find" data-citedArticleID="930618" data-doi="10.1016/j.cell.2006.01.044"><li><a href="http://dx.doi.org/10.1016/j.cell.2006.01.044" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Global+survey+of+organ+and+organelle+protein+expression+in+mouse%3A+combined+proteomic+and+transcriptomic+profiling." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Global+survey+of+organ+and+organelle+protein+expression+in+mouse%3A+combined+proteomic+and+transcriptomic+profiling.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0000216-Khaitovich1" id="pone.0000216-Khaitovich1"></a>Khaitovich P, Enard W, Lachmann M, Paabo S (2006) Evolution of primate gene expression. Nat Rev Genet  7: 693–702.  <ul class="find" data-citedArticleID="930614" data-doi="10.1038/nrg1940"><li><a href="http://dx.doi.org/10.1038/nrg1940" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evolution+of+primate+gene+expression." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evolution+of+primate+gene+expression.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0000216-Consortium.1" id="pone.0000216-Consortium.1"></a>Consortium. CSaA (2005) Initial sequence of the chimpanzee genome and comparison with the human genome. Nature  437: 69–87.  <ul class="find" data-citedArticleID="930596" data-doi="10.1038/nature04072"><li><a href="http://dx.doi.org/10.1038/nature04072" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Initial+sequence+of+the+chimpanzee+genome+and+comparison+with+the+human+genome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Initial+sequence+of+the+chimpanzee+genome+and+comparison+with+the+human+genome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0000216-Khaitovich2" id="pone.0000216-Khaitovich2"></a>Khaitovich P, Hellmann I, Enard W, Nowick K, Leinweber M, et al.  (2005) Parallel patterns of evolution in the genomes and transcriptomes of humans and chimpanzees. Science  309: 1850–1854.  <ul class="find" data-citedArticleID="930616" data-doi="10.1126/science.1108296"><li><a href="http://dx.doi.org/10.1126/science.1108296" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Parallel+patterns+of+evolution+in+the+genomes+and+transcriptomes+of+humans+and+chimpanzees." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Parallel+patterns+of+evolution+in+the+genomes+and+transcriptomes+of+humans+and+chimpanzees.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0000216-Irizarry1" id="pone.0000216-Irizarry1"></a>Irizarry RA, Hobbs B, Collin F, Beazer-Barclay YD, Antonellis KJ, et al.  (2003) Exploration, normalization, and summaries of high density oligonucleotide array probe level data. Biostatistics  4: 249–264.  <ul class="find" data-citedArticleID="930610" data-doi="10.1093/biostatistics/4.2.249"><li><a href="http://dx.doi.org/10.1093/biostatistics/4.2.249" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Exploration%2C+normalization%2C+and+summaries+of+high+density+oligonucleotide+array+probe+level+data." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Exploration%2C+normalization%2C+and+summaries+of+high+density+oligonucleotide+array+probe+level+data.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0000216-Jiang1" id="pone.0000216-Jiang1"></a>Jiang XS, Dai J, Sheng QH, Zhang L, Xia QC, et al.  (2005) A comparative proteomic strategy for subcellular proteome research: ICAT approach coupled with bioinformatics prediction to ascertain rat liver mitochondrial proteins and indication of mitochondrial localization for catalase. Mol Cell Proteomics  4: 12–34.  <ul class="find" data-citedArticleID="930612" data-doi="10.1074/mcp.m400079-mcp200"><li><a href="http://dx.doi.org/10.1074/mcp.m400079-mcp200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+comparative+proteomic+strategy+for+subcellular+proteome+research%3A+ICAT+approach+coupled+with+bioinformatics+prediction+to+ascertain+rat+liver+mitochondrial+proteins+and+indication+of+mitochondrial+localization+for+catalase." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+comparative+proteomic+strategy+for+subcellular+proteome+research%3A+ICAT+approach+coupled+with+bioinformatics+prediction+to+ascertain+rat+liver+mitochondrial+proteins+and+indication+of+mitochondrial+localization+for+catalase.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0000216-Felsenstein1" id="pone.0000216-Felsenstein1"></a>Felsenstein J (1989) PHYLIP - Phylogeny Inference Package (Version 3.2). Cladistics  5: 164–166.  <ul class="find" data-citedArticleID="930600"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=PHYLIP+-+Phylogeny+Inference+Package+%28Version+3.2%29.&amp;auth=&amp;atitle=PHYLIP+-+Phylogeny+Inference+Package+%28Version+3.2%29." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=PHYLIP+-+Phylogeny+Inference+Package+%28Version+3.2%29." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22PHYLIP+-+Phylogeny+Inference+Package+%28Version+3.2%29.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000216.XML" value="69037"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000216.PDF" value="96092"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000216.g001.PNG_L" value="18725"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000216.g001.PNG_M" value="6850"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000216.g001.PNG_S" value="1583"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000216.g001.TIF" value="35570"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000216.g001.PNG_I" value="4533"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000216.g002.PNG_L" value="89787"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000216.g002.PNG_M" value="8788"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000216.g002.PNG_S" value="1770"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000216.g002.TIF" value="183352"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000216.g002.PNG_I" value="6641"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000216.s001.DOC" value="32256"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000216.s002.DOC" value="238592"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000216.s003.DOC" value="99840"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000216.s004.DOC" value="82432"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000216.s005.DOC" value="114176"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000216.s006.XLS" value="19968"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000216.s007.XLS" value="82432"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000216.s008.XLS" value="90112"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000216.s009.XLS" value="37376"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000216.s010.XLS" value="27136"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000216&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000216" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000216&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0000216&volume=&issue=&title=Comparison of Protein and mRNA Expression Evolution in Humans and Chimpanzees&author_name=Ning%20Fu%2C%20Ines%20Drinnenberg%2C%20Janet%20Kelso%2C%20Jia-Rui%20Wu%2C%20Svante%20P%C3%A4%C3%A4bo%2C%20Rong%20Zeng%2C%20Philipp%20Khaitovich&start_page=1&end_page=5" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000216" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000216" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0000216" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000216&amp;t=Comparison%20of%20Protein%20and%20mRNA%20Expression%20Evolution%20in%20Humans%20and%20Chimpanzees" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000216&title=Comparison%20of%20Protein%20and%20mRNA%20Expression%20Evolution%20in%20Humans%20and%20Chimpanzees&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000216&amp;title=Comparison%20of%20Protein%20and%20mRNA%20Expression%20Evolution%20in%20Humans%20and%20Chimpanzees" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000216" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0000216&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Comparison of Protein and mRNA Expression Evolution in Humans and Chimpanzees';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0000216';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Comparison%20of%20Protein%20and%20mRNA%20Expression%20Evolution%20in%20Humans%20and%20Chimpanzees http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000216" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0000216" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">











          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Chimpanzees%22" title="Search for articles in the subject area:'Chimpanzees'"><div class="flagText">Chimpanzees</div></a>
              <div data-categoryid="43093" data-articleid="23270"
                   data-categoryname="Chimpanzees"
                   class="flagImage" title="Flag 'Chimpanzees' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Evolutionary+developmental+biology%22" title="Search for articles in the subject area:'Evolutionary developmental biology'"><div class="flagText">Evolutionary developmental biology</div></a>
              <div data-categoryid="34445" data-articleid="23270"
                   data-categoryname="Evolutionary developmental biology"
                   class="flagImage" title="Flag 'Evolutionary developmental biology' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Gene+expression%22" title="Search for articles in the subject area:'Gene expression'"><div class="flagText">Gene expression</div></a>
              <div data-categoryid="17515" data-articleid="23270"
                   data-categoryname="Gene expression"
                   class="flagImage" title="Flag 'Gene expression' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Gene+prediction%22" title="Search for articles in the subject area:'Gene prediction'"><div class="flagText">Gene prediction</div></a>
              <div data-categoryid="40279" data-articleid="23270"
                   data-categoryname="Gene prediction"
                   class="flagImage" title="Flag 'Gene prediction' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Microarrays%22" title="Search for articles in the subject area:'Microarrays'"><div class="flagText">Microarrays</div></a>
              <div data-categoryid="17255" data-articleid="23270"
                   data-categoryname="Microarrays"
                   class="flagImage" title="Flag 'Microarrays' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Molecular+evolution%22" title="Search for articles in the subject area:'Molecular evolution'"><div class="flagText">Molecular evolution</div></a>
              <div data-categoryid="17533" data-articleid="23270"
                   data-categoryname="Molecular evolution"
                   class="flagImage" title="Flag 'Molecular evolution' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Phenotypes%22" title="Search for articles in the subject area:'Phenotypes'"><div class="flagText">Phenotypes</div></a>
              <div data-categoryid="40235" data-articleid="23270"
                   data-categoryname="Phenotypes"
                   class="flagImage" title="Flag 'Phenotypes' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Protein+expression%22" title="Search for articles in the subject area:'Protein expression'"><div class="flagText">Protein expression</div></a>
              <div data-categoryid="17513" data-articleid="23270"
                   data-categoryname="Protein expression"
                   class="flagImage" title="Flag 'Protein expression' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=9826'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=6878'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=3028&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>


</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
