

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: A Structural Split in the Human Genome</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0000603"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0000603" />

    <meta name="citation_title" content="A Structural Split in the Human Genome"/>
    <meta itemprop="name" content="A Structural Split in the Human Genome"/>

      <meta name="citation_author" content="Clara S.M. Tang"/>
            <meta name="citation_author_institution" content="Laboratory of Computational Oncology, Department of Medicine, The University of Hong Kong, Pokfulam, Hong Kong, Hong Kong"/>
      <meta name="citation_author" content="Richard J. Epstein"/>
            <meta name="citation_author_institution" content="Laboratory of Computational Oncology, Department of Medicine, The University of Hong Kong, Pokfulam, Hong Kong, Hong Kong"/>

    <meta name="citation_date" content="2007/7/11"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0000603.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e603"/>
    <meta name="citation_issue" content="7"/>
    <meta name="citation_volume" content="2"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Robustness, evolvability, and neutrality.; citation_author=A Wagner; citation_journal_title=FEBS Lett; citation_volume=579; citation_number=1; citation_pages=1772-1778; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Selection for gene clustering by tandem duplication.; citation_author=AB Reams; citation_author=EL Neidle; citation_journal_title=Annu Rev Microbiol; citation_volume=58; citation_number=2; citation_pages=119-142; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Heterotachy and functional shift in protein evolution.; citation_author=H Philippe; citation_author=D Casane; citation_author=S Gribaldo; citation_author=P Lopez; citation_author=J Meunier; citation_journal_title=IUBMB Life; citation_volume=55; citation_number=3; citation_pages=257-265; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=The evolutionary demography of duplicate genes.; citation_author=M Lynch; citation_author=JS Conery; citation_journal_title=J Struct Funct Genomics; citation_volume=3; citation_number=4; citation_pages=35-44; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Stress and adaptation in conservation genetics.; citation_author=R Frankham; citation_journal_title=J Evol Biol; citation_volume=18; citation_number=5; citation_pages=750-755; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Eukaryotic domain evolution inferred from genome comparisons.; citation_author=RR Copley; citation_author=L Goodstadt; citation_author=C Ponting; citation_journal_title=Curr Opin Genet Dev; citation_volume=13; citation_number=6; citation_pages=623-628; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=The evolution of controlled multitasked gene networks: the role of introns and other noncoding RNAs in the development of complex organisms.; citation_author=JS Mattick; citation_author=MJ Gagen; citation_journal_title=Mol Biol Evol; citation_volume=18; citation_number=7; citation_pages=1611-1630; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Evolution of the genome and the genetic code: selection at the dinucleotide level by methylation and polyribonucleotide cleavage.; citation_author=E Beutler; citation_author=T Gelbart; citation_author=JH Han; citation_author=JA Koziol; citation_author=B Beutler; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=86; citation_number=8; citation_pages=192-196; citation_date=1989; " />
      <meta name="citation_reference" content="citation_title=Structure, function and evolution of CpG island promoters.; citation_author=F Antequera; citation_journal_title=Cell Mol Life Sci; citation_volume=60; citation_number=9; citation_pages=1647-1658; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Altered chromatin structure associated with methylation-induced gene silencing in cancer cells: correlation of accessibility, methylation, MeCP2 binding and acetylation.; citation_author=CT Nguyen; citation_author=FA Gonzales; citation_author=PA Jones; citation_journal_title=Nucleic Acids Res; citation_volume=29; citation_number=10; citation_pages=4598-4606; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=A human CpG island randomly inserted into a plant genome is protected from methylation.; citation_author=TJ Meza; citation_author=E Enerly; citation_author=B Boru; citation_author=F Larsen; citation_author=A Mandal; citation_journal_title=Transgenic Res; citation_volume=11; citation_number=11; citation_pages=133-142; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Predicting aberrant CpG island methylation.; citation_author=FA Feltus; citation_author=EK Lee; citation_author=JF Costello; citation_author=C Plass; citation_author=PM Vertino; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=100; citation_number=12; citation_pages=12253-12258; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Clusters of CpG dinucleotides implicated by nuclease hypersensitivity as control elements of housekeeping genes.; citation_author=SF Wolf; citation_author=BR Migeon; citation_journal_title=Nature; citation_volume=314; citation_number=13; citation_pages=467-469; citation_date=1985; " />
      <meta name="citation_reference" content="citation_title=A role for DNA hypomethylation and histone acetylation in maintaining allele-specific expression of mouse NKG2A in developing and mature NK cells.; citation_author=SL Rogers; citation_author=A Rouhi; citation_author=F Takei; citation_author=DL Mager; citation_journal_title=J Immunol; citation_volume=177; citation_number=14; citation_pages=414-421; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Non-methylated CpG motif packaged into fusogenic liposomes enhance antigen-specific immunity in mice.; citation_author=T Yoshikawa; citation_author=S Imazu; citation_author=JQ Gao; citation_author=K Hayashi; citation_author=Y Tsuda; citation_journal_title=Biol Pharm Bull; citation_volume=29; citation_number=15; citation_pages=105-109; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Cytosine methylation and CpG, TpG (CpA) and TpA frequencies.; citation_author=K Jabbari; citation_author=G Bernardi; citation_journal_title=Gene; citation_volume=333; citation_number=16; citation_pages=143-149; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Molecular evolution of an imprinted gene: repeatability of patterns of evolution within the mammalian insulin-like growth factor type II receptor.; citation_author=NG Smith; citation_author=LD Hurst; citation_journal_title=Genetics; citation_volume=150; citation_number=17; citation_pages=823-833; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Enhancement of Sleeping Beauty transposition by CpG methylation: possible role of heterochromatin formation.; citation_author=K Yusa; citation_author=J Takeda; citation_author=K Horie; citation_journal_title=Mol Cell Biol; citation_volume=24; citation_number=18; citation_pages=4004-4018; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=The DNA methylation paradox.; citation_author=PA Jones; citation_journal_title=Trends Genet; citation_volume=15; citation_number=19; citation_pages=34-37; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=DNA methylation and chromatin structure: the puzzling CpG islands.; citation_author=P Caiafa; citation_author=M Zampieri; citation_journal_title=J Cell Biochem; citation_volume=94; citation_number=20; citation_pages=257-265; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=A functional significance for codon third bases.; citation_author=RJ Epstein; citation_author=K Lin; citation_author=TW Tan; citation_journal_title=Gene; citation_volume=245; citation_number=21; citation_pages=291-298; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Nonrandom intragenic variations in patterns of codon bias implicate a sequential interplay between transitional genetic drift and functional amino acid selection.; citation_author=K Lin; citation_author=SB Tan; citation_author=PR Kolatkar; citation_author=RJ Epstein; citation_journal_title=J Mol Evol; citation_volume=57; citation_number=22; citation_pages=538-545; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Intron length and accelerated 3′ gene evolution.; citation_author=CS Tang; citation_author=YZ Zhao; citation_author=DK Smith; citation_author=RJ Epstein; citation_journal_title=Genomics.; citation_number=23; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Dualism of gene GC content and CpG pattern in regard to expression in the human genome: magnitude versus breadth.; citation_author=AE Vinogradov; citation_journal_title=Trends Genet; citation_volume=21; citation_number=24; citation_pages=639-643; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Molecular aspects of intron evolution in dynein encoding mega-genes on the heterochromatic Y chromosome of Drosophila sp.; citation_author=R Kurek; citation_author=AM Reugels; citation_author=U Lammermann; citation_author=H Bunemann; citation_journal_title=Genetica; citation_volume=109; citation_number=25; citation_pages=113-123; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Microsatellites within genes: structure, function, and evolution.; citation_author=YC Li; citation_author=AB Korol; citation_author=T Fahima; citation_author=E Nevo; citation_journal_title=Mol Biol Evol; citation_volume=21; citation_number=26; citation_pages=991-1007; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Lessons from the human genome: transitions between euchromatin and heterochromatin.; citation_author=JE Horvath; citation_author=JA Bailey; citation_author=DP Locke; citation_author=EE Eichler; citation_journal_title=Hum Mol Genet; citation_volume=10; citation_number=27; citation_pages=2215-2223; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Juxtacentromeric region of human chromosome 21: a boundary between centromeric heterochromatin and euchromatic chromosome arms.; citation_author=ME Brun; citation_author=M Ruault; citation_author=M Ventura; citation_author=G Roizes; citation_author=A De Sario; citation_journal_title=Gene; citation_volume=312; citation_number=28; citation_pages=41-50; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Colonization of heterochromatic genes by transposable elements in Drosophila.; citation_author=P Dimitri; citation_author=N Junakovic; citation_author=B Arca; citation_journal_title=Mol Biol Evol; citation_volume=20; citation_number=29; citation_pages=503-512; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=‘Genome design’ model and multicellular complexity: golden middle.; citation_author=AE Vinogradov; citation_journal_title=Nucleic Acids Res; citation_number=30; citation_date=2006; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000603" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="A Structural Split in the Human Genome"/>
    <meta name="twitter:description" content="BackgroundPromoter-associated CpG islands (PCIs) mediate methylation-dependent gene silencing, yet tend to co-locate to transcriptionally active genes. To address this paradox, we used data mining to assess the behavior of PCI-positive (PCI+) genes in the human genome.ResultsPCI+ genes exhibit a bimodal distribution: (1) a &lsquo;housekeeping-like&rsquo; subset characterized by higher GC content and lower intron length/number, and (2) a &lsquo;pseudogene paralog&rsquo; subset characterized by lower GC content and higher intron length/number (p&lt;0.001). These subsets are functionally distinguishable, with the former gene group characterized by higher expression levels and lower evolutionary rate (p&lt;0.001). PCI-negative (PCI-) genes exhibit higher evolutionary rate and narrower expression breadth than PCI+ genes (p&lt;0.001), consistent with more frequent tissue-specific inactivation.ConclusionsAdaptive evolution of the human genome appears driven in part by declining transcription of a subset of PCI+ genes, predisposing to both CpG&rarr;TpA mutation and intron insertion. We propose a model of evolving biological complexity in which environmentally-selected gains or losses of PCI methylation respectively favor positive or negative selection, thus polarizing PCI+ gene structures around a genomic core of ancestral PCI- genes."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0000603.g006"/>

  <meta property="og:title" content="A Structural Split in the Human Genome" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000603" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=3803'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=8866'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=6748&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0000603">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000603" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000603&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    A Structural Split in the Human Genome
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Clara S.M. Tang, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Laboratory of Computational Oncology, Department of Medicine, The University of Hong Kong, Pokfulam, Hong Kong, Hong Kong
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Richard J. Epstein
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:repstein@hku.hk">repstein@hku.hk</a></p>

                <p>Affiliation:
                  Laboratory of Computational Oncology, Department of Medicine, The University of Hong Kong, Pokfulam, Hong Kong, Hong Kong
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: July 11, 2007</li>
    <li>DOI: 10.1371/journal.pone.0000603</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0000603" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0000603" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000603" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0000603" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0000603-g001" data-doi="info:doi/10.1371/journal.pone.0000603" data-uri="info:doi/10.1371/journal.pone.0000603.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000603.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000603-t001" data-doi="info:doi/10.1371/journal.pone.0000603" data-uri="info:doi/10.1371/journal.pone.0000603.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000603.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000603-g002" data-doi="info:doi/10.1371/journal.pone.0000603" data-uri="info:doi/10.1371/journal.pone.0000603.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000603.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000603-g003" data-doi="info:doi/10.1371/journal.pone.0000603" data-uri="info:doi/10.1371/journal.pone.0000603.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000603.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000603-g004" data-doi="info:doi/10.1371/journal.pone.0000603" data-uri="info:doi/10.1371/journal.pone.0000603.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000603.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000603-g005" data-doi="info:doi/10.1371/journal.pone.0000603" data-uri="info:doi/10.1371/journal.pone.0000603.g005" title="Figure 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000603.g005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000603-g006" data-doi="info:doi/10.1371/journal.pone.0000603" data-uri="info:doi/10.1371/journal.pone.0000603.g006" title="Figure 6">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000603.g006&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000603">Reader Comments (0)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0000603" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2>
<h3>Background</h3>
<a id="article1.front1.article-meta1.abstract1.sec1.p1" name="article1.front1.article-meta1.abstract1.sec1.p1"></a><p>Promoter-associated CpG islands (PCIs) mediate methylation-dependent gene silencing, yet tend to co-locate to transcriptionally active genes. To address this paradox, we used data mining to assess the behavior of PCI-positive (PCI+) genes in the human genome.</p>


<h3>Results</h3>
<a id="article1.front1.article-meta1.abstract1.sec2.p1" name="article1.front1.article-meta1.abstract1.sec2.p1"></a><p>PCI+ genes exhibit a bimodal distribution: (1) a ‘housekeeping-like’ subset characterized by higher GC content and lower intron length/number, and (2) a ‘pseudogene paralog’ subset characterized by lower GC content and higher intron length/number (<em>p</em>&lt;0.001). These subsets are functionally distinguishable, with the former gene group characterized by higher expression levels and lower evolutionary rate (<em>p</em>&lt;0.001). PCI-negative (PCI-) genes exhibit higher evolutionary rate and narrower expression breadth than PCI+ genes (<em>p</em>&lt;0.001), consistent with more frequent tissue-specific inactivation.</p>


<h3>Conclusions</h3>
<a id="article1.front1.article-meta1.abstract1.sec3.p1" name="article1.front1.article-meta1.abstract1.sec3.p1"></a><p>Adaptive evolution of the human genome appears driven in part by declining transcription of a subset of PCI+ genes, predisposing to both CpG→TpA mutation and intron insertion. We propose a model of evolving biological complexity in which environmentally-selected gains or losses of PCI methylation respectively favor positive or negative selection, thus polarizing PCI+ gene structures around a genomic core of ancestral PCI- genes.</p>

</div>


<div class="articleinfo"><p><strong>Citation: </strong>Tang CS, Epstein RJ (2007) A Structural Split in the Human Genome. PLoS ONE 2(7):
          e603.
            doi:10.1371/journal.pone.0000603</p><p><strong>Academic Editor: </strong>Guillaume Bourque, Genome Institute of Singapore, Singapore</p><p><strong>Received:</strong> January 22, 2007; <strong>Accepted:</strong> June 6, 2007; <strong>Published:</strong> July 11, 2007</p><p><strong>Copyright:</strong> © 2007 Tang, Epstein. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>The authors have no support or funding to report.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>Evolution of biological complexity involves an environmentally-regulated balance between genetic conservation and variation <a href="#pone.0000603-Wagner1">[1]</a>–<a href="#pone.0000603-Copley1">[6]</a>. Phylogenetic leaps favoring speciation of higher organisms include the evolution of introns and DNA methylation <a href="#pone.0000603-Mattick1">[7]</a>, <a href="#pone.0000603-Beutler1">[8]</a>. A more recent innovation is that of promoter-associated CpG islands (PCIs) <a href="#pone.0000603-Antequera1">[9]</a> which, when methylated, mediate transcriptional repression and/or chromatin condensation <a href="#pone.0000603-Nguyen1">[10]</a>. About 60% of human genes contain PCIs <a href="#pone.0000603-Antequera1">[9]</a>, most of which are unmethylated <a href="#pone.0000603-Meza1">[11]</a>, <a href="#pone.0000603-Feltus1">[12]</a>. Such PCIs are more common in widely-expressed (housekeeping) genes <a href="#pone.0000603-Wolf1">[13]</a>, supporting the view that non-methylated PCIs actively maintain gene transcription <a href="#pone.0000603-Rogers1">[14]</a>, <a href="#pone.0000603-Yoshikawa1">[15]</a>.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>In addition to the functional effects of PCI methylation on transcription, methylcytosine residues in coding regions may undergo oxidative deamination to thymine, with such events being quantifiable as an excess of CG→TA transitional mutations<a href="#pone.0000603-Jabbari1">[16]</a>. This interaction between methylation-dependent <em>trans</em>-repression and mutation drives adaptive evolution <a href="#pone.0000603-Smith1">[17]</a>. The latter observation is in turn consistent with the finding that methylation of CpG-rich regions facilitates transposition events <a href="#pone.0000603-Yusa1">[18]</a>.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>A single interpretation of PCI behavior in transcription and evolution thus remains elusive <a href="#pone.0000603-Jones1">[19]</a>, <a href="#pone.0000603-Caiafa1">[20]</a>, suggesting a more ‘kaleidoscopic’ model in which PCI significance oscillates with methylation status. Relevant to this, we showed in earlier work that CpG dinucleotide retention in coding sequences correlates with codon essentiality <a href="#pone.0000603-Epstein1">[21]</a>, <a href="#pone.0000603-Lin1">[22]</a>, whereas both gene methylation and intron length accelerate the evolution of less well repaired downstream intragenic sequences <a href="#pone.0000603-Tang1">[23]</a>. These results suggest a dynamic pro-evolutionary balance between negative selection for retention of CpG dinucleotides in transcriptionally active genes, and adaptive evolution for methylation-dependent CpG mutation in transcriptionally inactive genes. The present study was designed to test this hypothesis by comparing PCI-containing (PCI+) and PCI-deficient (PCI-) genes, and has identified in the process two structurally and functionally distinct PCI+ gene subsets.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3>
<h4>The GC nucleotide content of PCI-containing human genes exhibits a biphasic distribution</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>We first compared the genomic architecture of human genes with that of lower organisms. This confirmed that the human genome is characterized not only by higher CpG island frequency and greater intron number and length <a href="#pone.0000603-Antequera1">[9]</a>, but also by a higher coefficient of variation and more negative kurtosis of its nucleotide content (G+C; <a href="#pone-0000603-t001">Table 1</a>). This finding raised the possibility that the notably broad GC range of the human DNA distribution (<a href="#pone-0000603-g001">Figure 1A</a>) relates in some manner to the pro-evolutionary events of CpG island acquisition and/or intron insertion. To address this possibility, we next distinguished ‘start CpG islands’ from downstream CpG islands (see <a href="#s4">Methods</a>). As shown in <a href="#pone-0000603-g001">Figure 1B</a>, the frequency distribution of start CpG islands exhibits two maxima based on GC content, mirroring the single peak of genes lacking start CpG islands (<a href="#pone-0000603-g001">Figure 1C</a>). A similar pattern is evident when the search is restricted to promoter-associated CpG islands (PCIs), implying that the distribution of PCI-negative genes peaks at a single intermediate GC content. Control comparisons of genes with unidirectional <em>vs.</em> bidirectional promoters revealed no significant differences in intron number, evolutionary rate or expression level (<a href="#pone.0000603.s001"><strong>Supplement S1</strong></a>).</p>
<div class="figure" id="pone-0000603-g001"><div class="img"><a name="pone-0000603-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000603.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000603" data-uri="info:doi/10.1371/journal.pone.0000603.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000603.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000603.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000603.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000603.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000603.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000603.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000603.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000603.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000603.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Biphasic GC/AT distribution of PCI+ genes. A, Distribution of GC content among different regions of genes (coding region, grey fill; intronic, black solid; 5¢ untranslated region, dark grey, long dash; 3¢ untranslated region, grey, short dash).</span></strong></p><a id="article1.body1.sec2.sec1.fig1.caption1.p1" name="article1.body1.sec2.sec1.fig1.caption1.p1"></a><p>B&amp;C, Proportion of genes among different GC groups; B, with ‘start’ CpG islands (CGI+). C, without ‘start’ CpG islands (CGI-).</p>
<span>doi:10.1371/journal.pone.0000603.g001</span></div><div class="figure" id="pone-0000603-t001"><div class="img"><a name="pone-0000603-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000603.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000603" data-uri="info:doi/10.1371/journal.pone.0000603.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000603.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000603.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000603.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000603.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000603.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000603.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000603.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000603.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000603.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>Cross-species comparison of coding GC content (c.v., kurtosis, and skewness), proportion of genes with promoter-associated CpG islands (PCI), number of exons, and intron length.</span></strong></p><span>doi:10.1371/journal.pone.0000603.t001</span></div><a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2"></a><p><a href="#pone-0000603-g002">Figure 2</a> analyzes in more detail the GC contents of PCI+ and PCI- genes. These patterns appear similar in the whole genome (<a href="#pone-0000603-g002">Fig. 2A</a>) and in genes with intermediate intron lengths (<a href="#pone-0000603-g002">Fig. 2B</a>). However, PCI+ genes lacking introns or containing only short overall intron lengths are characterized by high GC content (<a href="#pone-0000603-g002">Fig. 2C</a>, solid line), whereas PCI+ genes containing long introns are characterized by low GC content (<a href="#pone-0000603-g002">Fig. 2D</a>).</p>
<div class="figure" id="pone-0000603-g002"><div class="img"><a name="pone-0000603-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000603.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000603" data-uri="info:doi/10.1371/journal.pone.0000603.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000603.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000603.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000603.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000603.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000603.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000603.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000603.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000603.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000603.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Distribution of GC content of promoter CpG island overlapping genes (PCI+, solid line) and non-promoter CpG island overlapping genes (PCI-, dash line).</span></strong></p><a id="article1.body1.sec2.sec1.fig2.caption1.p1" name="article1.body1.sec2.sec1.fig2.caption1.p1"></a><p>A, All genes; B, Genes with medium total intron size (10–50 kb); C, intronless genes; and D, genes with short total intron size (&lt;10 kb, blue) and long intron size (&gt;50 kb, orange).</p>
<span>doi:10.1371/journal.pone.0000603.g002</span></div><a id="article1.body1.sec2.sec1.p3" name="article1.body1.sec2.sec1.p3"></a><p>The PCI+ gene nucleotide distribution was further examined using a computational curve comparison, confirming a better fit for a bimodal distribution (<a href="#pone-0000603-g003">Figure 3B</a>; peaks of 46.09+/−5.19, and 60.27+/−5.19) than for a unimodal distribution (<a href="#pone-0000603-g003">Figure 3A</a>; mean 53.13+/−8.78; <em>p</em>&lt;0.001). These patterns resemble those of ‘pseudogene paralog’ and ‘housekeeping-like’ gene clusters (<a href="#pone-0000603-g003">Figs. 3C and 3D</a> respectively), consistent with the possibility that gene expression contributes to evolutionary changes in GC content among PCI+ genes (see <a href="#pone.0000603.s002"><strong>Supplement S2</strong></a> for validation of the foregoing functional definitions).</p>
<div class="figure" id="pone-0000603-g003"><div class="img"><a name="pone-0000603-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000603.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000603" data-uri="info:doi/10.1371/journal.pone.0000603.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000603.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000603.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000603.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000603.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000603.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000603.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000603.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000603.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000603.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Distribution of coding GC% of RefGenes with PCIs.</span></strong></p><a id="article1.body1.sec2.sec1.fig3.caption1.p1" name="article1.body1.sec2.sec1.fig3.caption1.p1"></a><p>Unimodal (A) and bimodal distribution curves (B) were fitted to coding GC% distribution. C, Coding GC% of paralogs of processed pseudogenes with (light blue filled) and without PCIs (open blue stroke) shift towards lower GC content. D, Coding GC% of housekeeping genes with (purple fill) and without PCIs (purple stroke) shift towards higher GC. See <a href="#s4">Methods</a> for subset derivation.</p>
<span>doi:10.1371/journal.pone.0000603.g003</span></div>

<h4>High-GC and low-GC PCI+ gene clusters are distinguishable in terms of intron content, evolutionary rate, and expression</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>The relationship between the structural variables of PCI status, GC content, intron number, and total intron length was then examined. As shown in <a href="#pone-0000603-g004">Figure 4A</a>, significant increases in both intron number and intron length characterise both AT-rich and PCI+ gene subsets. This is also true for AT-rich PCI+ genes; however, for AT-rich PCI- genes, the increase in intron number is only borderline, even though intron length remains a highly significant differential. These findings raise the hypothesis that PCIs play a causal role in intron insertion, perhaps via chromatin-related effects, whereas the correlation between low GC content and total intron length could reflect other factors (e.g., transcription, repair, etc.).</p>
<div class="figure" id="pone-0000603-g004"><div class="img"><a name="pone-0000603-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000603.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000603" data-uri="info:doi/10.1371/journal.pone.0000603.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000603.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000603.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000603.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000603.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000603.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000603.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000603.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000603.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000603.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>Quantitative comparison of gene subsets characterized in terms of PCI status, intron length/number, GC content, evolutionary rate, expression breadth and expression level.</span></strong></p><a id="article1.body1.sec2.sec2.fig1.caption1.p1" name="article1.body1.sec2.sec2.fig1.caption1.p1"></a><p>For expression levels, trimmed means (5%) with standard errors were used, with promoter non-overlapping (-, cross) and overlapping CpG islands (+, stroke) of different GC content. (Low, L, GC&lt;40%, black; high, H, GC&gt;65%, grey). P-value of nonparametric KS test is shown in circles (double dark, &lt;0.001; single dark, &lt;0.01; open, &lt;0.05). A, Intron length and number; B, Evolutionary rate (Ka/Ks); C, Expression breadth and level.</p>
<span>doi:10.1371/journal.pone.0000603.g004</span></div><a id="article1.body1.sec2.sec2.p2" name="article1.body1.sec2.sec2.p2"></a><p>Like intron number and length, evolutionary rate–quantified here by the Ka/Ks ratio–is increased in AT-rich compared to GC-rich genes; in contrast to the foregoing intronic correlations, however, PCI+ genes exhibit a lower evolutionary rate that is synergistic with high GC content (<a href="#pone-0000603-g004">Fig. 4B</a>). This implies that PCI+ genes feature higher transcription/repair or greater negative selection pressure than PCI- genes–which remains consistent with the hypothesis that PCIs favor intron insertion via a separate chromatin effect.</p>
<a id="article1.body1.sec2.sec2.p3" name="article1.body1.sec2.sec2.p3"></a><p><a href="#pone-0000603-g004">Figure 4C</a> supports this latter possibility, showing as it does that gene expression breadth is greatly increased in PCI+ (vs. PCI-) genes, but not in GC-rich (vs. AT-rich) genes. In contrast, average expression levels are higher in GC-rich than in AT-rich genes, particularly in PCI- gene subsets. Since changes in expression breadth tend to be mediated by tissue-specific differences in chromatin organization, these data support earlier analyses concluding that PCI-dependent effects on chromatin structure do not closely parallel methylation-dependent effects on transcription <a href="#pone.0000603-Vinogradov1">[24]</a>. In spite of the over-representation of promoter CpG islands in divergent promoter genes, which is thought to regulate coordinated gene expression, this group of genes exhibited PCI+ structural characterisitics similar to those of unidirectional ones (<strong><a href="#pone.0000603.s001">Supplement S1</a>, <a href="#pone.0000603.s006">Figure S1</a>, <a href="#pone.0000603.s004">Table S1</a>, <a href="#pone.0000603.s005">Table S2</a></strong>) .</p>
<a id="article1.body1.sec2.sec2.p4" name="article1.body1.sec2.sec2.p4"></a><p>The existence of gene subgroups structurally separable on the basis of PCI status, GC content, intron length and intron number is confirmed by principal component analysis (<a href="#pone-0000603-g005">Fig. 5A–C</a>). This suggests a model of genetic evolution in which the speed of evolution towards greater biological complexity is maximized by concurrent positive and negative selection forces acting differentially on the genome in response to environmental changes, leading in turn to a selectable balance between gene expression and structural fidelity (<a href="#pone-0000603-g006">Fig. 6</a>).</p>
<div class="figure" id="pone-0000603-g005"><div class="img"><a name="pone-0000603-g005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000603.g005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000603" data-uri="info:doi/10.1371/journal.pone.0000603.g005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000603.g005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000603.g005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000603.g005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000603.g005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000603.g005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000603.g005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000603.g005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000603.g005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000603.g005.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 5.  <span>Principal component analysis (PCA). A, PCA analysis using six variables at either 53% (left) or 59% (right) variance.</span></strong></p><a id="article1.body1.sec2.sec2.fig2.caption1.p1" name="article1.body1.sec2.sec2.fig2.caption1.p1"></a><p>B, Two-dimensional dot plots confirming the existence of distinct PCI+ gene subsets (red cross, low GC; blue circle, high GC) based on intron size/number, transcription, and evolutionary rate. C, Three-dimensional dot plot of the GC-rich (blue) and GC-poor (red) PCI+ gene clusters.</p>
<span>doi:10.1371/journal.pone.0000603.g005</span></div><div class="figure" id="pone-0000603-g006"><div class="img"><a name="pone-0000603-g006" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000603.g006&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000603" data-uri="info:doi/10.1371/journal.pone.0000603.g006"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000603.g006&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000603.g006/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000603.g006/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000603.g006/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000603.g006/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000603.g006.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000603.g006/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000603.g006/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000603.g006.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 6.  <span>Model of human genomic evolution, proposing the development of a structural split in GC content based on the association of methylation-dependent mutation with transcriptional repression, chromatin condensation, reduced repair and intron insertion.</span></strong></p><span>doi:10.1371/journal.pone.0000603.g006</span></div>
</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p>There are three central findings of this study. First, there exists an AT-rich ‘second peak’ of PCI+ genes which, when compared with the GC-rich peak of housekeeping-like PCI+ genes, is characterized by lower transcriptional activity, higher intron number and length, and higher evolutionary rate. We propose that the AT-rich subset has arisen from the GC-rich subset of PCI+ genes via progressive loss of negative selection pressure, accompanied by progressive PCI methylation.</p>
<a id="article1.body1.sec3.p2" name="article1.body1.sec3.p2"></a><p>Second, PCI- genes have a higher evolutionary rate and lower expression breadth than PCI+ genes. This suggests either that (i) only widely-transcribed (housekeeping-like) genes are under selection pressure to acquire or retain PCIs, and/or (ii) PCI loss represents a separate pathway towards pseudogenization (e.g., mediated by heterochromatinization) for less-transcribed tissue-specific genes. Since PCI frequency increases with biological complexity (<a href="#pone-0000603-t001">Table 1</a>), we submit that the majority of PCI- genes represent ancestral genes, for which an increase in tissue-specific negative selection pressure could favor PCI acquisition (e.g., via transposition events).</p>
<a id="article1.body1.sec3.p3" name="article1.body1.sec3.p3"></a><p>Third, although total intron length correlates best with AT-richness, intron number relates more directly to PCI positivity (<a href="#pone-0000603-g004">Fig 4A</a>). Given both the CG→TA mutability of methylated CpG sites, and the association of PCI methylation with chromatin condensation, this finding suggests that intron extension could occur via an autocatalytic mechanism <a href="#pone.0000603-Kurek1">[25]</a> associated with reduced repair, whereas intron insertion per se is dependent upon PCI-dependent chromatin alterations. Since the stronger correlation between PCI+ genes (&gt;GC-rich genes) and expression breadth (&gt;expression level; <a href="#pone-0000603-g004">Fig. 4C</a>) implicates PCI-dependent chromatin activation–i.e., even in the presence of transcriptional repression and impaired repair of methylation-dependent mutation–as a causal mechanism, PCI-dependent chromatin effects could likewise mediate the greater intron length and number characteristic of mammalian genomes. This hypothesis is consistent with the pro-evolutionary nature of mutations eluding mismatch repair <a href="#pone.0000603-Li1">[26]</a> as well as with the presence of ‘transition regions’ between euchromatin and heterochromatin <a href="#pone.0000603-Horvath1">[27]</a>, <a href="#pone.0000603-Brun1">[28]</a>.</p>
<a id="article1.body1.sec3.p4" name="article1.body1.sec3.p4"></a><p>Hence, our findings suggest a paradigm of PCIs not only as transcriptional regulators, but also as evolutionary accelerators that can facilitate intron insertion–perhaps via modulation of chromatin structure <a href="#pone.0000603-Dimitri1">[29]</a>–under environmental circumstances in which transcriptional inhibition does not compromise fitness. We agree with others <a href="#pone.0000603-Vinogradov2">[30]</a> that the positive correlations of PCIs with both transcription level/breadth and with intron number/length do not support a simplistic “selection for time economy” model of intron evolution. Moreover, we stress that a full molecular elucidation of the present hypothesis must be awaited.</p>
<a id="article1.body1.sec3.p5" name="article1.body1.sec3.p5"></a><p>In conclusion, the present study has identified two subsets of PCI+ genes within the human genome. We propose that the opposing consequences of differentially methylated PCIs on transcription and chromatin accelerate adaptive evolution towards biological complexity. As such, this study supports the view that the reversible methylation-dependent mechanism of structural switching between these functional extremes constitutes nothing less than an evolutionary masterstroke of natural selection.</p>
</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4>Sequence data and annotations</h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p>We retrieved the genomic sequences of five species–human (<em>Homo sapiens</em>), mouse (<em>Mus musculus),</em> Fugu (<em>Takifugu rubripes),</em> fruit fly (<em>Drosophila melanogaster)</em> and worm (<em>Caenorhabditis elegans)</em>–from the University of California, Santa Cruz (UCSC) Table Browser (<a href="http://genome.ucsc.edu/">http://genome.ucsc.edu/</a>) (Karolchik, et al., 2004). Genome assemblies of hg18 (NCBI build 36.1, March 2006), mm6 (NCBI Build 34, March 2005), fr1 (JGI v3.0 August 2002), dm2 (BDGP Release 4, April 2004) and ce2 (WormBase WS120, March 2004), respectively, were used. Since the transcription start site does not shift much according to DBTSS annotation (see <strong><a href="#pone.0000603.s003">Supplement S3</a>, <a href="#pone.0000603.s008">Figure S3</a></strong>), sequence analyses of all species were carried out using RefSeq genes, with the exception of Fugu for which no RefSeq dataset is available. and for which the Ensembl gene dataset was used instead.</p>
<a id="article1.body1.sec4.sec1.p2" name="article1.body1.sec4.sec1.p2"></a><p>To prevent interspersed repeats like <em>Alu</em> sequences from creating bias in nucleotide composition, RepeatMask sequences were used. Genes not commencing with ATG codons, or not terminating with canonical stop codons, were excluded in order to obtain the most reliable and homogeneous set of complete coding genes. When several genes contained identical exonic sequences, only the one with longest genomic length was retained.</p>


<h4>Determination of CpG island overlapping transcription start site</h4>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1"></a><p>To identify CpG islands overlapping with promoter region, CpG islands annotation (cpgIslandExt) was downloaded from UCSC Genome Bioinformatics site, listing the physical position of CpG islands determined by the database. The position information was then mapped with RefGene annotation to isolate the RefGenes with start CGIs or promoter CGIs (PCI) whose transcriptional start site or promoter (2kb upstream and 500bp downstream) overlapped with the pre-determined CpG islands respectively.</p>


<h4>Housekeeping genes and paralogs of pseudogenes</h4>
<a id="article1.body1.sec4.sec3.p1" name="article1.body1.sec4.sec3.p1"></a><p>Analysis of housekeeping genes was carried out using a previously defined set (Eisenberg and Levanon, 2003) containing 502 housekeeping genes. Non-overlapping set of processed pseudogenes was obtained from (<a href="http://www.pseudogene.org">http://www.pseudogene.org</a>, Karro <em>et.al.</em>, 2006) and the RefGenes corresponding to the same proteins were mapped from the annotation, resulting in 1220 pseudogene paralogs (<strong><a href="#pone.0000603.s002">Supplement S2</a>, <a href="#pone.0000603.s007">Figure S2</a></strong>).</p>


<h4>Bimodal Distribution of GC content</h4>
<a id="article1.body1.sec4.sec4.p1" name="article1.body1.sec4.sec4.p1"></a><p>The distribution of coding GC% was best-fitted using the NOCOM program (<a href="http://www.genemapping.cn/nocom.htm">http://www.genemapping.cn/nocom.htm</a>) based on a counting (EM) algorithm. Under no transformation (exponent = 1), mean, the standard deviation and proportion of each population was estimated. To test for bimodality, the bimodal distribution model was compared against the unimodal one using the statistics G<sup>2</sup> = 2{} which has an asymptotic χ<sup>2</sup> distribution with degrees of freedom approximate to 2 (d.f. = 2), where ln(L–<sub>1</sub>) and ln(L<sub>0</sub>) are maximum log likelihood for a bimodal and unimodal distribution respectively.</p>


<h4>Gene expression data</h4>
<a id="article1.body1.sec4.sec5.p1" name="article1.body1.sec4.sec5.p1"></a><p>The SAGEmap (Nov 2005, <a href="ftp://ftp.ncbi.nlm.nih.gov/pub/sage">ftp://ftp.ncbi.nlm.nih.gov/pub/sage</a>) of NCBI was used for quantitative evaluation of gene expression. SAGE libraries were grouped according to 26 tissue types including brain, blood, bone, bone marrow, cervix, cartilage, colon, eye, heart, kidney, liver, lung, lymph node, mammary gland, muscle, ovary, pancreas, peripheral nervous system, placenta, prostate, skin, stem cell, stomach, thyroid, vascular and esophagus. Reliable tag-to-gene mapping of NlaIII SAGE tags to UniGene clusters was obtained from SAGEmap, and each cluster was represented by the longest RefSeq gene. Ambiguous tags mapping to more than one RefSeq gene were excluded. If a tag had been counted once only in one tissue, it was regarded as likely due to sequencing error and was thus discounted. SAGE tags of each RefGene were counted for each tissue type and normalized to counts per million. The normalized counts of each tissue were averaged across all tissue types for fair comparison between organs with different mean expression level.</p>


<h4>Evolutionary rate determination</h4>
<a id="article1.body1.sec4.sec6.p1" name="article1.body1.sec4.sec6.p1"></a><p>Homologue data in XML format was obtained from NCBI HomoloGene database (<a href="ftp://ftp.ncbi.nih.gov/pub/HomoloGene/">ftp://ftp.ncbi.nih.gov/pub/HomoloGene/</a>). Orthologous gene pairs between human and mouse, together with their synonymous substitution, non-synonymous substitution rate (Ka) and their ratio (Ka/Ks) were isolated. Genes with Ka&gt;1.5 and Ks&gt;3 in any quartile were discarded due to high estimation error.</p>


<h4>Principal component analysis (PCA)</h4>
<a id="article1.body1.sec4.sec7.p1" name="article1.body1.sec4.sec7.p1"></a><p>In order to visibly corroborate the association of bimodal distribution with structural variations, we applied principal component analysis using XSTAT to gene subgroups of low and high GC content with intron size, intron number, evolutionary rate, expression breadth and level as variables. The analysis was performed using the correlation matrix, and the observations were visualized with the first three principal components in two and three dimensions.</p>

</div>

<div id="section5" class="section"><a id="s5" name="s5" toc="s5" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0000603"></div><a name="pone.0000603.s001" id="pone.0000603.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000603.s001">Supplement S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1"></a><p class="preSiDOI">Supplement 1.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000603.s001</p><a id="article1.body1.sec5.supplementary-material1.caption1.p2" name="article1.body1.sec5.supplementary-material1.caption1.p2"></a><p class="postSiDOI">(0.03 MB DOC)</p>
<a name="pone.0000603.s002" id="pone.0000603.s002"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000603.s002">Supplement S2. </a></strong></p><a id="article1.body1.sec5.supplementary-material2.caption1.p1" name="article1.body1.sec5.supplementary-material2.caption1.p1"></a><p class="preSiDOI">Supplement 2.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000603.s002</p><a id="article1.body1.sec5.supplementary-material2.caption1.p2" name="article1.body1.sec5.supplementary-material2.caption1.p2"></a><p class="postSiDOI">(0.03 MB DOC)</p>
<a name="pone.0000603.s003" id="pone.0000603.s003"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000603.s003">Supplement S3. </a></strong></p><a id="article1.body1.sec5.supplementary-material3.caption1.p1" name="article1.body1.sec5.supplementary-material3.caption1.p1"></a><p class="preSiDOI">Supplement 3.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000603.s003</p><a id="article1.body1.sec5.supplementary-material3.caption1.p2" name="article1.body1.sec5.supplementary-material3.caption1.p2"></a><p class="postSiDOI">(0.03 MB DOC)</p>
<a name="pone.0000603.s004" id="pone.0000603.s004"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000603.s004">Table S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material4.caption1.p1" name="article1.body1.sec5.supplementary-material4.caption1.p1"></a><p class="preSiDOI">Distribution of genes with divergent promoters, as characterised by the distance between transcriptional start sites.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000603.s004</p><a id="article1.body1.sec5.supplementary-material4.caption1.p2" name="article1.body1.sec5.supplementary-material4.caption1.p2"></a><p class="postSiDOI">(0.02 MB DOC)</p>
<a name="pone.0000603.s005" id="pone.0000603.s005"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000603.s005">Table S2. </a></strong></p><a id="article1.body1.sec5.supplementary-material5.caption1.p1" name="article1.body1.sec5.supplementary-material5.caption1.p1"></a><p class="preSiDOI">Proportion of divergent promoters with CpG islands among genes with different distances between transcription start sites vs. all genes.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000603.s005</p><a id="article1.body1.sec5.supplementary-material5.caption1.p2" name="article1.body1.sec5.supplementary-material5.caption1.p2"></a><p class="postSiDOI">(0.02 MB DOC)</p>
<a name="pone.0000603.s006" id="pone.0000603.s006"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000603.s006">Figure S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material6.caption1.p1" name="article1.body1.sec5.supplementary-material6.caption1.p1"></a><p class="preSiDOI">Structural characterization (Upper panel : GC content, total intron length and number of intron(s), expression breath; Lower panel : Ka, Ks, Ka/Ks and expression level) of three groups of divergent promoters, overlapping or &lt;0.3kb (pink); 0.3-1kb (red); 1-10kb (dark red) against all genes (light blue; shaded).</p>
<p class="siDoi">doi:10.1371/journal.pone.0000603.s006</p><a id="article1.body1.sec5.supplementary-material6.caption1.p2" name="article1.body1.sec5.supplementary-material6.caption1.p2"></a><p class="postSiDOI">(0.85 MB TIF)</p>
<a name="pone.0000603.s007" id="pone.0000603.s007"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000603.s007">Figure S2. </a></strong></p><a id="article1.body1.sec5.supplementary-material7.caption1.p1" name="article1.body1.sec5.supplementary-material7.caption1.p1"></a><p class="preSiDOI">Median and quartile expression level and breadth of housekeeping genes and pseudogene paralogs.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000603.s007</p><a id="article1.body1.sec5.supplementary-material7.caption1.p2" name="article1.body1.sec5.supplementary-material7.caption1.p2"></a><p class="postSiDOI">(0.22 MB TIF)</p>
<a name="pone.0000603.s008" id="pone.0000603.s008"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000603.s008">Figure S3. </a></strong></p><a id="article1.body1.sec5.supplementary-material8.caption1.p1" name="article1.body1.sec5.supplementary-material8.caption1.p1"></a><p class="preSiDOI">Plot of distance difference between the DBTSS and RefSeq transcription start site in scale of 10bp (left) and 300bp (right).</p>
<p class="siDoi">doi:10.1371/journal.pone.0000603.s008</p><a id="article1.body1.sec5.supplementary-material8.caption1.p2" name="article1.body1.sec5.supplementary-material8.caption1.p2"></a><p class="postSiDOI">(0.15 MB TIF)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>We sincerely thank Dr Yongzhong Zhao, Dr David Smith, and Professor Raymond Liang for assistance and support.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: RE. Performed the experiments: CT. Analyzed the data: RE CT. Wrote the paper: RE CT.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0000603-Wagner1" id="pone.0000603-Wagner1"></a>Wagner A (2005) Robustness, evolvability, and neutrality. FEBS Lett  579: 1772–1778.  <ul class="find" data-citedArticleID="965332" data-doi="10.1016/j.febslet.2005.01.063"><li><a href="http://dx.doi.org/10.1016/j.febslet.2005.01.063" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Robustness%2C+evolvability%2C+and+neutrality." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Robustness%2C+evolvability%2C+and+neutrality.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0000603-Reams1" id="pone.0000603-Reams1"></a>Reams AB, Neidle EL (2004) Selection for gene clustering by tandem duplication. Annu Rev Microbiol  58: 119–142.  <ul class="find" data-citedArticleID="965320" data-doi="10.1146/annurev.micro.58.030603.123806"><li><a href="http://dx.doi.org/10.1146/annurev.micro.58.030603.123806" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Selection+for+gene+clustering+by+tandem+duplication." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Selection+for+gene+clustering+by+tandem+duplication.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0000603-Philippe1" id="pone.0000603-Philippe1"></a>Philippe H, Casane D, Gribaldo S, Lopez P, Meunier J (2003) Heterotachy and functional shift in protein evolution. IUBMB Life  55: 257–265.  <ul class="find" data-citedArticleID="965318" data-doi="10.1080/1521654031000123330"><li><a href="http://dx.doi.org/10.1080/1521654031000123330" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Heterotachy+and+functional+shift+in+protein+evolution." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Heterotachy+and+functional+shift+in+protein+evolution.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0000603-Lynch1" id="pone.0000603-Lynch1"></a>Lynch M, Conery JS (2003) The evolutionary demography of duplicate genes. J Struct Funct Genomics  3: 35–44.  <ul class="find" data-citedArticleID="965310" data-doi="10.1007/978-94-010-0263-9_4"><li><a href="http://dx.doi.org/10.1007/978-94-010-0263-9_4" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+evolutionary+demography+of+duplicate+genes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+evolutionary+demography+of+duplicate+genes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0000603-Frankham1" id="pone.0000603-Frankham1"></a>Frankham R (2005) Stress and adaptation in conservation genetics. J Evol Biol  18: 750–755.  <ul class="find" data-citedArticleID="965296" data-doi="10.1111/j.1420-9101.2005.00885.x"><li><a href="http://dx.doi.org/10.1111/j.1420-9101.2005.00885.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Stress+and+adaptation+in+conservation+genetics." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Stress+and+adaptation+in+conservation+genetics.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0000603-Copley1" id="pone.0000603-Copley1"></a>Copley RR, Goodstadt L, Ponting C (2003) Eukaryotic domain evolution inferred from genome comparisons. Curr Opin Genet Dev  13: 623–628.  <ul class="find" data-citedArticleID="965288" data-doi="10.1016/j.gde.2003.10.004"><li><a href="http://dx.doi.org/10.1016/j.gde.2003.10.004" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Eukaryotic+domain+evolution+inferred+from+genome+comparisons." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Eukaryotic+domain+evolution+inferred+from+genome+comparisons.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0000603-Mattick1" id="pone.0000603-Mattick1"></a>Mattick JS, Gagen MJ (2001) The evolution of controlled multitasked gene networks: the role of introns and other noncoding RNAs in the development of complex organisms. Mol Biol Evol  18: 1611–1630.  <ul class="find" data-citedArticleID="965312" data-doi="10.1093/oxfordjournals.molbev.a003951"><li><a href="http://dx.doi.org/10.1093/oxfordjournals.molbev.a003951" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+evolution+of+controlled+multitasked+gene+networks%3A+the+role+of+introns+and+other+noncoding+RNAs+in+the+development+of+complex+organisms." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+evolution+of+controlled+multitasked+gene+networks%3A+the+role+of+introns+and+other+noncoding+RNAs+in+the+development+of+complex+organisms.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0000603-Beutler1" id="pone.0000603-Beutler1"></a>Beutler E, Gelbart T, Han JH, Koziol JA, Beutler B (1989) Evolution of the genome and the genetic code: selection at the dinucleotide level by methylation and polyribonucleotide cleavage. Proc Natl Acad Sci U S A  86: 192–196.  <ul class="find" data-citedArticleID="965282" data-doi="10.1073/pnas.86.1.192"><li><a href="http://dx.doi.org/10.1073/pnas.86.1.192" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evolution+of+the+genome+and+the+genetic+code%3A+selection+at+the+dinucleotide+level+by+methylation+and+polyribonucleotide+cleavage." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evolution+of+the+genome+and+the+genetic+code%3A+selection+at+the+dinucleotide+level+by+methylation+and+polyribonucleotide+cleavage.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0000603-Antequera1" id="pone.0000603-Antequera1"></a>Antequera F (2003) Structure, function and evolution of CpG island promoters. Cell Mol Life Sci  60: 1647–1658.  <ul class="find" data-citedArticleID="965280" data-doi="10.1007/s00018-003-3088-6"><li><a href="http://dx.doi.org/10.1007/s00018-003-3088-6" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Structure%2C+function+and+evolution+of+CpG+island+promoters." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Structure%2C+function+and+evolution+of+CpG+island+promoters.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0000603-Nguyen1" id="pone.0000603-Nguyen1"></a>Nguyen CT, Gonzales FA, Jones PA (2001) Altered chromatin structure associated with methylation-induced gene silencing in cancer cells: correlation of accessibility, methylation, MeCP2 binding and acetylation. Nucleic Acids Res  29: 4598–4606.  <ul class="find" data-citedArticleID="965316" data-doi="10.1093/nar/29.22.4598"><li><a href="http://dx.doi.org/10.1093/nar/29.22.4598" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Altered+chromatin+structure+associated+with+methylation-induced+gene+silencing+in+cancer+cells%3A+correlation+of+accessibility%2C+methylation%2C+MeCP2+binding+and+acetylation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Altered+chromatin+structure+associated+with+methylation-induced+gene+silencing+in+cancer+cells%3A+correlation+of+accessibility%2C+methylation%2C+MeCP2+binding+and+acetylation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0000603-Meza1" id="pone.0000603-Meza1"></a>Meza TJ, Enerly E, Boru B, Larsen F, Mandal A, et al.  (2002) A human CpG island randomly inserted into a plant genome is protected from methylation. Transgenic Res  11: 133–142.  <ul class="find" data-citedArticleID="965314"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=A+human+CpG+island+randomly+inserted+into+a+plant+genome+is+protected+from+methylation.&amp;auth=&amp;atitle=A+human+CpG+island+randomly+inserted+into+a+plant+genome+is+protected+from+methylation." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+human+CpG+island+randomly+inserted+into+a+plant+genome+is+protected+from+methylation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+human+CpG+island+randomly+inserted+into+a+plant+genome+is+protected+from+methylation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0000603-Feltus1" id="pone.0000603-Feltus1"></a>Feltus FA, Lee EK, Costello JF, Plass C, Vertino PM (2003) Predicting aberrant CpG island methylation. Proc Natl Acad Sci U S A  100: 12253–12258.  <ul class="find" data-citedArticleID="965294" data-doi="10.1073/pnas.2037852100"><li><a href="http://dx.doi.org/10.1073/pnas.2037852100" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Predicting+aberrant+CpG+island+methylation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Predicting+aberrant+CpG+island+methylation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0000603-Wolf1" id="pone.0000603-Wolf1"></a>Wolf SF, Migeon BR (1985) Clusters of CpG dinucleotides implicated by nuclease hypersensitivity as control elements of housekeeping genes. Nature  314: 467–469.  <ul class="find" data-citedArticleID="965334" data-doi="10.1038/314467a0"><li><a href="http://dx.doi.org/10.1038/314467a0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Clusters+of+CpG+dinucleotides+implicated+by+nuclease+hypersensitivity+as+control+elements+of+housekeeping+genes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Clusters+of+CpG+dinucleotides+implicated+by+nuclease+hypersensitivity+as+control+elements+of+housekeeping+genes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0000603-Rogers1" id="pone.0000603-Rogers1"></a>Rogers SL, Rouhi A, Takei F, Mager DL (2006) A role for DNA hypomethylation and histone acetylation in maintaining allele-specific expression of mouse NKG2A in developing and mature NK cells. J Immunol  177: 414–421.  <ul class="find" data-citedArticleID="965322"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=A+role+for+DNA+hypomethylation+and+histone+acetylation+in+maintaining+allele-specific+expression+of+mouse+NKG2A+in+developing+and+mature+NK+cells.&amp;auth=&amp;atitle=A+role+for+DNA+hypomethylation+and+histone+acetylation+in+maintaining+allele-specific+expression+of+mouse+NKG2A+in+developing+and+mature+NK+cells." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+role+for+DNA+hypomethylation+and+histone+acetylation+in+maintaining+allele-specific+expression+of+mouse+NKG2A+in+developing+and+mature+NK+cells." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+role+for+DNA+hypomethylation+and+histone+acetylation+in+maintaining+allele-specific+expression+of+mouse+NKG2A+in+developing+and+mature+NK+cells.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0000603-Yoshikawa1" id="pone.0000603-Yoshikawa1"></a>Yoshikawa T, Imazu S, Gao JQ, Hayashi K, Tsuda Y, et al.  (2006) Non-methylated CpG motif packaged into fusogenic liposomes enhance antigen-specific immunity in mice. Biol Pharm Bull  29: 105–109.  <ul class="find" data-citedArticleID="965336" data-doi="10.1248/bpb.29.105"><li><a href="http://dx.doi.org/10.1248/bpb.29.105" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Non-methylated+CpG+motif+packaged+into+fusogenic+liposomes+enhance+antigen-specific+immunity+in+mice." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Non-methylated+CpG+motif+packaged+into+fusogenic+liposomes+enhance+antigen-specific+immunity+in+mice.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0000603-Jabbari1" id="pone.0000603-Jabbari1"></a>Jabbari K, Bernardi G (2004) Cytosine methylation and CpG, TpG (CpA) and TpA frequencies. Gene  333: 143–149.  <ul class="find" data-citedArticleID="965300" data-doi="10.1016/j.gene.2004.02.043"><li><a href="http://dx.doi.org/10.1016/j.gene.2004.02.043" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Cytosine+methylation+and+CpG%2C+TpG+%28CpA%29+and+TpA+frequencies." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Cytosine+methylation+and+CpG%2C+TpG+%28CpA%29+and+TpA+frequencies.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0000603-Smith1" id="pone.0000603-Smith1"></a>Smith NG, Hurst LD (1998) Molecular evolution of an imprinted gene: repeatability of patterns of evolution within the mammalian insulin-like growth factor type II receptor. Genetics  150: 823–833.  <ul class="find" data-citedArticleID="965324"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Molecular+evolution+of+an+imprinted+gene%3A+repeatability+of+patterns+of+evolution+within+the+mammalian+insulin-like+growth+factor+type+II+receptor.&amp;auth=&amp;atitle=Molecular+evolution+of+an+imprinted+gene%3A+repeatability+of+patterns+of+evolution+within+the+mammalian+insulin-like+growth+factor+type+II+receptor." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Molecular+evolution+of+an+imprinted+gene%3A+repeatability+of+patterns+of+evolution+within+the+mammalian+insulin-like+growth+factor+type+II+receptor." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Molecular+evolution+of+an+imprinted+gene%3A+repeatability+of+patterns+of+evolution+within+the+mammalian+insulin-like+growth+factor+type+II+receptor.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0000603-Yusa1" id="pone.0000603-Yusa1"></a>Yusa K, Takeda J, Horie K (2004) Enhancement of Sleeping Beauty transposition by CpG methylation: possible role of heterochromatin formation. Mol Cell Biol  24: 4004–4018.  <ul class="find" data-citedArticleID="965338" data-doi="10.1128/mcb.24.9.4004-4018.2004"><li><a href="http://dx.doi.org/10.1128/mcb.24.9.4004-4018.2004" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Enhancement+of+Sleeping+Beauty+transposition+by+CpG+methylation%3A+possible+role+of+heterochromatin+formation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Enhancement+of+Sleeping+Beauty+transposition+by+CpG+methylation%3A+possible+role+of+heterochromatin+formation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0000603-Jones1" id="pone.0000603-Jones1"></a>Jones PA (1999) The DNA methylation paradox. Trends Genet  15: 34–37.  <ul class="find" data-citedArticleID="965302" data-doi="10.1016/s0168-9525(98)01636-9"><li><a href="http://dx.doi.org/10.1016/s0168-9525(98)01636-9" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+DNA+methylation+paradox." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+DNA+methylation+paradox.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0000603-Caiafa1" id="pone.0000603-Caiafa1"></a>Caiafa P, Zampieri M (2005) DNA methylation and chromatin structure: the puzzling CpG islands. J Cell Biochem  94: 257–265.  <ul class="find" data-citedArticleID="965286" data-doi="10.1002/jcb.20325"><li><a href="http://dx.doi.org/10.1002/jcb.20325" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=DNA+methylation+and+chromatin+structure%3A+the+puzzling+CpG+islands." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22DNA+methylation+and+chromatin+structure%3A+the+puzzling+CpG+islands.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0000603-Epstein1" id="pone.0000603-Epstein1"></a>Epstein RJ, Lin K, Tan TW (2000) A functional significance for codon third bases. Gene  245: 291–298.  <ul class="find" data-citedArticleID="965292" data-doi="10.1016/s0378-1119(00)00042-1"><li><a href="http://dx.doi.org/10.1016/s0378-1119(00)00042-1" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+functional+significance+for+codon+third+bases." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+functional+significance+for+codon+third+bases.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0000603-Lin1" id="pone.0000603-Lin1"></a>Lin K, Tan SB, Kolatkar PR, Epstein RJ (2003) Nonrandom intragenic variations in patterns of codon bias implicate a sequential interplay between transitional genetic drift and functional amino acid selection. J Mol Evol  57: 538–545.  <ul class="find" data-citedArticleID="965308" data-doi="10.1007/s00239-003-2507-5"><li><a href="http://dx.doi.org/10.1007/s00239-003-2507-5" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Nonrandom+intragenic+variations+in+patterns+of+codon+bias+implicate+a+sequential+interplay+between+transitional+genetic+drift+and+functional+amino+acid+selection." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Nonrandom+intragenic+variations+in+patterns+of+codon+bias+implicate+a+sequential+interplay+between+transitional+genetic+drift+and+functional+amino+acid+selection.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0000603-Tang1" id="pone.0000603-Tang1"></a>Tang CS, Zhao YZ, Smith DK, Epstein RJ (2006) Intron length and accelerated 3′ gene evolution. Genomics..   <ul class="find" data-citedArticleID="965326" data-doi="10.1016/j.ygeno.2006.06.017"><li><a href="http://dx.doi.org/10.1016/j.ygeno.2006.06.017" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Intron+length+and+accelerated+3%26%23x2032%3B+gene+evolution." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Intron+length+and+accelerated+3%26%23x2032%3B+gene+evolution.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0000603-Vinogradov1" id="pone.0000603-Vinogradov1"></a>Vinogradov AE (2005) Dualism of gene GC content and CpG pattern in regard to expression in the human genome: magnitude versus breadth. Trends Genet  21: 639–643.  <ul class="find" data-citedArticleID="965328" data-doi="10.1016/j.tig.2005.09.002"><li><a href="http://dx.doi.org/10.1016/j.tig.2005.09.002" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Dualism+of+gene+GC+content+and+CpG+pattern+in+regard+to+expression+in+the+human+genome%3A+magnitude+versus+breadth." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Dualism+of+gene+GC+content+and+CpG+pattern+in+regard+to+expression+in+the+human+genome%3A+magnitude+versus+breadth.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0000603-Kurek1" id="pone.0000603-Kurek1"></a>Kurek R, Reugels AM, Lammermann U, Bunemann H (2000) Molecular aspects of intron evolution in dynein encoding mega-genes on the heterochromatic Y chromosome of Drosophila sp. Genetica  109: 113–123.  <ul class="find" data-citedArticleID="965304"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Molecular+aspects+of+intron+evolution+in+dynein+encoding+mega-genes+on+the+heterochromatic+Y+chromosome+of+Drosophila+sp.&amp;auth=&amp;atitle=Molecular+aspects+of+intron+evolution+in+dynein+encoding+mega-genes+on+the+heterochromatic+Y+chromosome+of+Drosophila+sp." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Molecular+aspects+of+intron+evolution+in+dynein+encoding+mega-genes+on+the+heterochromatic+Y+chromosome+of+Drosophila+sp." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Molecular+aspects+of+intron+evolution+in+dynein+encoding+mega-genes+on+the+heterochromatic+Y+chromosome+of+Drosophila+sp.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0000603-Li1" id="pone.0000603-Li1"></a>Li YC, Korol AB, Fahima T, Nevo E (2004) Microsatellites within genes: structure, function, and evolution. Mol Biol Evol  21: 991–1007.  <ul class="find" data-citedArticleID="965306" data-doi="10.1093/molbev/msh073"><li><a href="http://dx.doi.org/10.1093/molbev/msh073" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Microsatellites+within+genes%3A+structure%2C+function%2C+and+evolution." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Microsatellites+within+genes%3A+structure%2C+function%2C+and+evolution.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0000603-Horvath1" id="pone.0000603-Horvath1"></a>Horvath JE, Bailey JA, Locke DP, Eichler EE (2001) Lessons from the human genome: transitions between euchromatin and heterochromatin. Hum Mol Genet  10: 2215–2223.  <ul class="find" data-citedArticleID="965298" data-doi="10.1093/hmg/10.20.2215"><li><a href="http://dx.doi.org/10.1093/hmg/10.20.2215" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Lessons+from+the+human+genome%3A+transitions+between+euchromatin+and+heterochromatin." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Lessons+from+the+human+genome%3A+transitions+between+euchromatin+and+heterochromatin.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0000603-Brun1" id="pone.0000603-Brun1"></a>Brun ME, Ruault M, Ventura M, Roizes G, De Sario A (2003) Juxtacentromeric region of human chromosome 21: a boundary between centromeric heterochromatin and euchromatic chromosome arms. Gene  312: 41–50.  <ul class="find" data-citedArticleID="965284" data-doi="10.1016/s0378-1119(03)00530-4"><li><a href="http://dx.doi.org/10.1016/s0378-1119(03)00530-4" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Juxtacentromeric+region+of+human+chromosome+21%3A+a+boundary+between+centromeric+heterochromatin+and+euchromatic+chromosome+arms." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Juxtacentromeric+region+of+human+chromosome+21%3A+a+boundary+between+centromeric+heterochromatin+and+euchromatic+chromosome+arms.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0000603-Dimitri1" id="pone.0000603-Dimitri1"></a>Dimitri P, Junakovic N, Arca B (2003) Colonization of heterochromatic genes by transposable elements in Drosophila. Mol Biol Evol  20: 503–512.  <ul class="find" data-citedArticleID="965290" data-doi="10.1093/molbev/msg048"><li><a href="http://dx.doi.org/10.1093/molbev/msg048" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Colonization+of+heterochromatic+genes+by+transposable+elements+in+Drosophila." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Colonization+of+heterochromatic+genes+by+transposable+elements+in+Drosophila.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0000603-Vinogradov2" id="pone.0000603-Vinogradov2"></a>Vinogradov AE (2006) ‘Genome design’ model and multicellular complexity: golden middle. Nucleic Acids Res.   <ul class="find" data-citedArticleID="965330" data-doi="10.1093/nar/gkl773"><li><a href="http://dx.doi.org/10.1093/nar/gkl773" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=%26%23x2018%3BGenome+design%26%23x2019%3B+model+and+multicellular+complexity%3A+golden+middle." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22%26%23x2018%3BGenome+design%26%23x2019%3B+model+and+multicellular+complexity%3A+golden+middle.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000603.XML" value="60237"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000603.PDF" value="366045"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000603.g001.PNG_L" value="328293"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000603.g001.PNG_M" value="74491"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000603.g001.PNG_S" value="11762"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000603.g001.TIF" value="537758"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000603.g001.PNG_I" value="67520"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000603.t001.PNG_L" value="76966"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000603.t001.PNG_M" value="43153"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000603.t001.PNG_S" value="8771"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000603.t001.TIF" value="351622"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000603.t001.PNG_I" value="12899"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000603.g002.PNG_L" value="246079"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000603.g002.PNG_M" value="94179"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000603.g002.PNG_S" value="9981"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000603.g002.TIF" value="368376"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000603.g002.PNG_I" value="32700"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000603.g003.PNG_L" value="344175"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000603.g003.PNG_M" value="122932"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000603.g003.PNG_S" value="10808"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000603.g003.TIF" value="748542"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000603.g003.PNG_I" value="42147"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000603.g004.PNG_L" value="848948"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000603.g004.PNG_M" value="129561"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000603.g004.PNG_S" value="18443"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000603.g004.TIF" value="1603018"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000603.g004.PNG_I" value="112534"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000603.g005.PNG_L" value="533681"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000603.g005.PNG_M" value="154611"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000603.g005.PNG_S" value="11716"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000603.g005.TIF" value="759960"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000603.g005.PNG_I" value="52889"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000603.g006.PNG_L" value="267433"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000603.g006.PNG_M" value="149739"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000603.g006.PNG_S" value="8894"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000603.g006.TIF" value="408108"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000603.g006.PNG_I" value="49364"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000603.s001.DOC" value="31744"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000603.s002.DOC" value="27136"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000603.s003.DOC" value="27648"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000603.s004.DOC" value="20480"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000603.s005.DOC" value="22016"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000603.s006.TIF" value="397478"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000603.s007.TIF" value="145280"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000603.s008.TIF" value="79034"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000603&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000603" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000603&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0000603&volume=&issue=&title=A Structural Split in the Human Genome&author_name=Clara%20S.M.%20Tang%2C%20Richard%20J.%20Epstein&start_page=1&end_page=7" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000603" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000603" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0000603" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000603&amp;t=A%20Structural%20Split%20in%20the%20Human%20Genome" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000603&title=A%20Structural%20Split%20in%20the%20Human%20Genome&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000603&amp;title=A%20Structural%20Split%20in%20the%20Human%20Genome" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000603" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0000603&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'A Structural Split in the Human Genome';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0000603';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20A%20Structural%20Split%20in%20the%20Human%20Genome http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000603" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0000603" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">
















          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Chromatin%22" title="Search for articles in the subject area:'Chromatin'"><div class="flagText">Chromatin</div></a>
              <div data-categoryid="31933" data-articleid="24044"
                   data-categoryname="Chromatin"
                   class="flagImage" title="Flag 'Chromatin' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22DNA+methylation%22" title="Search for articles in the subject area:'DNA methylation'"><div class="flagText">DNA methylation</div></a>
              <div data-categoryid="32275" data-articleid="24044"
                   data-categoryname="DNA methylation"
                   class="flagImage" title="Flag 'DNA methylation' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22DNA+transcription%22" title="Search for articles in the subject area:'DNA transcription'"><div class="flagText">DNA transcription</div></a>
              <div data-categoryid="20231" data-articleid="24044"
                   data-categoryname="DNA transcription"
                   class="flagImage" title="Flag 'DNA transcription' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Evolutionary+genetics%22" title="Search for articles in the subject area:'Evolutionary genetics'"><div class="flagText">Evolutionary genetics</div></a>
              <div data-categoryid="17209" data-articleid="24044"
                   data-categoryname="Evolutionary genetics"
                   class="flagImage" title="Flag 'Evolutionary genetics' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Functional+genomics%22" title="Search for articles in the subject area:'Functional genomics'"><div class="flagText">Functional genomics</div></a>
              <div data-categoryid="17853" data-articleid="24044"
                   data-categoryname="Functional genomics"
                   class="flagImage" title="Flag 'Functional genomics' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Genome+evolution%22" title="Search for articles in the subject area:'Genome evolution'"><div class="flagText">Genome evolution</div></a>
              <div data-categoryid="17207" data-articleid="24044"
                   data-categoryname="Genome evolution"
                   class="flagImage" title="Flag 'Genome evolution' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Human+genomics%22" title="Search for articles in the subject area:'Human genomics'"><div class="flagText">Human genomics</div></a>
              <div data-categoryid="17253" data-articleid="24044"
                   data-categoryname="Human genomics"
                   class="flagImage" title="Flag 'Human genomics' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Introns%22" title="Search for articles in the subject area:'Introns'"><div class="flagText">Introns</div></a>
              <div data-categoryid="20277" data-articleid="24044"
                   data-categoryname="Introns"
                   class="flagImage" title="Flag 'Introns' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=2336'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=7836'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=1358&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>


</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
