

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Natural Variation in Partial Resistance to Pseudomonas syringae Is Controlled by Two Major QTLs in Arabidopsis thaliana</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0000123"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0000123" />

    <meta name="citation_title" content="Natural Variation in Partial Resistance to Pseudomonas syringae Is Controlled by Two Major QTLs in Arabidopsis thaliana"/>
    <meta itemprop="name" content="Natural Variation in Partial Resistance to Pseudomonas syringae Is Controlled by Two Major QTLs in Arabidopsis thaliana"/>

      <meta name="citation_author" content="Laure Perchepied"/>
            <meta name="citation_author_institution" content="Laboratoire des Interactions Plantes-Microorganismes (LIPM), UMR Centre National de la Recherche Scientifique/National Institute for Agronomical Research 2594, Castanet-Tolosan, France"/>
      <meta name="citation_author" content="Thomas Kroj"/>
            <meta name="citation_author_institution" content="Laboratoire des Interactions Plantes-Microorganismes (LIPM), UMR Centre National de la Recherche Scientifique/National Institute for Agronomical Research 2594, Castanet-Tolosan, France"/>
      <meta name="citation_author" content="Maurice Tronchet"/>
            <meta name="citation_author_institution" content="Laboratoire des Interactions Plantes-Microorganismes (LIPM), UMR Centre National de la Recherche Scientifique/National Institute for Agronomical Research 2594, Castanet-Tolosan, France"/>
      <meta name="citation_author" content="Olivier Loudet"/>
            <meta name="citation_author_institution" content="National Institute for Agronomical Research (INRA), Versailles, France"/>
      <meta name="citation_author" content="Dominique Roby"/>
            <meta name="citation_author_institution" content="Laboratoire des Interactions Plantes-Microorganismes (LIPM), UMR Centre National de la Recherche Scientifique/National Institute for Agronomical Research 2594, Castanet-Tolosan, France"/>

    <meta name="citation_date" content="2006/12/27"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0000123.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e123"/>
    <meta name="citation_issue" content="1"/>
    <meta name="citation_volume" content="1"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=SNARE-protein-mediated disease resistance at the plant cell wall.; citation_author=NC Collins; citation_author=H Thordal-Christensen; citation_author=V Lipka; citation_author=S Bau; citation_author=E Kombrink; citation_journal_title=Nature; citation_volume=425; citation_number=1; citation_pages=973-977; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Pre- and postinvasion defenses both contribute to nonhost resistance in Arabidopsis.; citation_author=V Lipka; citation_author=J Dittgen; citation_author=P Bednarek; citation_author=R Bhat; citation_author=M Wiermer; citation_journal_title=Science; citation_volume=310; citation_number=2; citation_pages=1180-1183; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Arabidopsis PEN3/PDR8, an ATP binding cassette transporter, contributes to nonhost resistance to inappropriate pathogens that enter by direct penetration.; citation_author=M Stein; citation_author=J Dittgen; citation_author=C Sanchez-Rodriguez; citation_author=BH Hou; citation_author=A Molina; citation_journal_title=Plant Cell; citation_volume=18; citation_number=3; citation_pages=731-746; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Innate immunity in plants and animals: striking similarities and obvious differences.; citation_author=T Nurnberger; citation_author=F Brunner; citation_author=B Kemmerling; citation_author=L Piater; citation_journal_title=Immunol Rev; citation_volume=198; citation_number=4; citation_pages=249-266; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Innate immunity in plants and animals: emerging parallels between the recognition of general elicitors and pathogen-associated molecular patterns.; citation_author=T Nurnberger; citation_author=F Brunner; citation_journal_title=Curr Opin Plant Biol; citation_volume=5; citation_number=5; citation_pages=318-324; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Plants have a sensitive perception system for the most conserved domain of bacterial flagellin.; citation_author=G Felix; citation_author=JD Duran; citation_author=S Volko; citation_author=T Boller; citation_journal_title=Plant J; citation_volume=18; citation_number=6; citation_pages=265-276; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=MAP kinase signalling cascade in Arabidopsis innate immunity.; citation_author=T Asai; citation_author=G Tena; citation_author=J Plotnikova; citation_author=MR Willmann; citation_author=WL Chiu; citation_journal_title=Nature; citation_volume=415; citation_number=7; citation_pages=977-983; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Bacterial disease resistance in Arabidopsis through flagellin perception.; citation_author=C Zipfel; citation_author=S Robatzek; citation_author=L Navarro; citation_author=EJ Oakeley; citation_author=JD Jones; citation_journal_title=Nature; citation_volume=428; citation_number=8; citation_pages=764-767; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Flagellin induces innate immunity in nonhost interactions that is suppressed by Pseudomonas syringae effectors.; citation_author=X Li; citation_author=H Lin; citation_author=W Zhang; citation_author=Y Zou; citation_author=J Zhang; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=102; citation_number=9; citation_pages=12990-12995; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Understanding the functions of plant disease resistance proteins.; citation_author=GB Martin; citation_author=AJ Bogdanove; citation_author=G Sessa; citation_journal_title=Annu Rev Plant Biol; citation_volume=54; citation_number=10; citation_pages=23-61; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Gene-for-gene complementarity in plant-pathogen interactions.; citation_author=NT Keen; citation_journal_title=Annual Review of Genetic; citation_volume=24; citation_number=11; citation_pages=447-463; citation_date=1990; " />
      <meta name="citation_reference" content="citation_title=Resistance to oomycetes: a general role for the hypersensitive response?; citation_author=S Kamoun; citation_author=E Huitema; citation_author=VGAA Vleeshouwers; citation_journal_title=Trends in Plant Science; citation_volume=4; citation_number=12; citation_pages=196-200; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=RESISTANCE GENE COMPLEXES: Evolution and Utilization.; citation_author=SH Hulbert; citation_author=CA Webb; citation_author=SM Smith; citation_author=Q Sun; citation_journal_title=Annual Review of Phytopathology; citation_volume=39; citation_number=13; citation_pages=285-312; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=
                  RMo1 confers blast resistance in barley and is located within the complex of resistance genes containing Mla, a powdery mildew resistance gene.; citation_author=T Inukai; citation_author=MI Vales; citation_author=K Hori; citation_author=K Sato; citation_author=PM Hayes; citation_journal_title=Mol Plant Microbe Interact; citation_volume=19; citation_number=14; citation_pages=1034-1041; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Complex genetic networks underlying the defensive system of rice (Oryza sativa L.) to Xanthomonas oryzae pv. oryzae.; citation_author=ZK Li; citation_author=M Arif; citation_author=DB Zhong; citation_author=BY Fu; citation_author=JL Xu; citation_journal_title=PNAS; citation_volume=103; citation_number=15; citation_pages=7994-7999; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Naturally occuring variation in Arabidopsis: an underexploited resource for plant genetics.; citation_author=C Alonso-Blanco; citation_author=M Koornneef; citation_journal_title=Trends in Plant Science; citation_volume=5; citation_number=16; citation_pages=22-29; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Naturally occurring genetic variation in Arabidopsis thaliana.; citation_author=M Koornneef; citation_author=C Alonso-Blanco; citation_author=D Vreugdenhil; citation_journal_title=Annu Rev Plant Biol; citation_volume=55; citation_number=17; citation_pages=141-172; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Quantitative Trait Loci Analysis of Powdery Mildew Disease Resistance in the Arabidopsis thaliana Accession Kashmir-1.; citation_author=IW Wilson; citation_author=CL Schiff; citation_author=DE Hughes; citation_author=SC Somerville; citation_journal_title=Genetics; citation_volume=158; citation_number=18; citation_pages=1301-1309; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=ERECTA, an LRR receptor-like kinase protein controlling development pleiotropically affects resistance to bacterial wilt.; citation_author=L Godiard; citation_author=L Sauviac; citation_author=KU Torii; citation_author=O Grenon; citation_author=B Mangin; citation_journal_title=Plant J; citation_volume=36; citation_number=19; citation_pages=353-365; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Identification of Botrytis cinerea susceptibility loci in Arabidopsis thaliana.; citation_author=KJ Denby; citation_author=P Kumar; citation_author=DJ Kliebenstein; citation_journal_title=Plant J; citation_volume=38; citation_number=20; citation_pages=473-486; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Genetic architecture of Arabidopsis thaliana response to infection by Pseudomonas syringae.; citation_author=PX Kover; citation_author=JB Wolf; citation_author=BN Kunkel; citation_author=JM Cheverud; citation_journal_title=Heredity; citation_volume=94; citation_number=21; citation_pages=507-517; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=ERECTA receptor-like kinase and heterotrimeric G protein from Arabidopsis are required for resistance to the necrotrophic fungus Plectosphaerella cucumerina.; citation_author=F Llorente; citation_author=C Alonso-Blanco; citation_author=C Sanchez-Rodriguez; citation_author=L Jorda; citation_author=A Molina; citation_journal_title=Plant J; citation_volume=43; citation_number=22; citation_pages=165-180; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Deciphering host resistance and pathogen virulence: the Arabidopsis/Pseudomonas interaction as a model.; citation_author=BF Quirino; citation_author=AF Bent; citation_journal_title=Molecular Plant Pathology; citation_volume=4; citation_number=23; citation_pages=517-530; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Suppression of host defense in compatible plant-Pseudomonas syringae interactions.; citation_author=K Nomura; citation_author=M Melotto; citation_author=SY He; citation_journal_title=Curr Opin Plant Biol; citation_volume=8; citation_number=24; citation_pages=361-368; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=The Arabidopsis thaliana-Pseudomonas syringae interaction.; citation_author=F Katagiri; citation_author=R Thilmony; citation_author=SY He; citation_number=25; citation_pages=1-35; citation_date=2002; citation_publisher=American Society of Plant Biologists; " />
      <meta name="citation_reference" content="citation_title=Genetic variation for disease resistance and tolerance among Arabidopsis thaliana accessions.; citation_author=PX Kover; citation_author=BA Schaal; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=99; citation_number=26; citation_pages=11270-11274; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Components of resistance that reduce the rate of epidemic development.; citation_author=JE Parlevliet; citation_journal_title=Annual Review of Phytopathology; citation_volume=17; citation_number=27; citation_pages=203-222; citation_date=1979; " />
      <meta name="citation_reference" content="citation_title=Use of Arabidopsis recombinant inbred lines reveals a monogenic and a novel digenic resistance mechanism to Xanthomonas campestris pv campestris.; citation_author=CR Buell; citation_author=SC Sommerville; citation_journal_title=Plant J; citation_volume=12; citation_number=28; citation_pages=21-29; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Heterogeneous inbred family (HIF) analysis: a method for developping near-isogenic lines that differ at quantitative trait loci.; citation_author=MR Tuinstra; citation_author=G Ejeta; citation_author=PB Goldsbrough; citation_journal_title=Theor Appl Genet; citation_volume=95; citation_number=29; citation_pages=1005-1011; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Nested core collections maximizing genetic diversity in Arabidopsis thaliana.; citation_author=HI McKhann; citation_author=C Camilleri; citation_author=A Berard; citation_author=T Bataillon; citation_author=JL David; citation_journal_title=Plant J; citation_volume=38; citation_number=30; citation_pages=193-202; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Development of an AFLP based linkage map of Ler, Col and Cvi Arabidopsis thaliana ecotypes and construction of a Ler/Cvi recombinant inbred line population.; citation_author=C Alonso-Blanco; citation_author=AJ Peeters; citation_author=M Koornneef; citation_author=C Lister; citation_author=C Dean; citation_journal_title=Plant J; citation_volume=14; citation_number=31; citation_pages=259-271; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=The use of recombinant inbred lines (RILs) for genetic mapping.; citation_author=C Alonso-Blanco; citation_author=M Koornneef; citation_author=P Stam; citation_journal_title=Methods Mol Biol; citation_volume=82; citation_number=32; citation_pages=137-146; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Analysis of natural allelic variation at flowering time loci in the Landsberg erecta and Cape Verde Islands ecotypes of Arabidopsis thaliana.; citation_author=C Alonso-Blanco; citation_author=SE El-Assal; citation_author=G Coupland; citation_author=M Koornneef; citation_journal_title=Genetics; citation_volume=149; citation_number=33; citation_pages=749-764; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Bay-0×Shahdara recombinant inbred line population: a powerful tool for the genetic dissection of complex traits in Arabidopsis.; citation_author=O Loudet; citation_author=S Chaillou; citation_author=C Camilleri; citation_author=D Bouchez; citation_author=F Daniel-Vedele; citation_journal_title=Theor Appl Genet; citation_volume=104; citation_number=34; citation_pages=1173-1184; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Precision mapping of quantitative trait loci.; citation_author=ZB Zeng; citation_journal_title=Genetics; citation_volume=136; citation_number=35; citation_pages=1457-1468; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=Cross talk between signaling pathways in pathogen defense.; citation_author=BN Kunkel; citation_author=DM Brooks; citation_journal_title=Curr Opin Plant Biol; citation_volume=5; citation_number=36; citation_pages=325-331; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=The complexity of disease signaling in Arabidopsis.; citation_author=BP Thomma; citation_author=IA Penninckx; citation_author=WF Broekaert; citation_author=BP Cammue; citation_journal_title=Curr Opin Immunol; citation_volume=13; citation_number=37; citation_pages=63-68; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Topology of the network integrating salicylate and jasmonate signal transduction derived from global expression phenotyping.; citation_author=J Glazebrook; citation_author=W Chen; citation_author=B Estes; citation_author=HS Chang; citation_author=C Nawrath; citation_journal_title=Plant J; citation_volume=34; citation_number=38; citation_pages=217-228; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Salicylic acid induction-deficient mutants of Arabidopsis express PR-2 and PR-5 and accumulate high levels of camalexin after pathogen inoculation.; citation_author=C Nawrath; citation_author=J-P Métraux; citation_journal_title=The Plant Cell; citation_volume=11; citation_number=39; citation_pages=1393-1404; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Tomato Transcription Factors Pti4, Pti5, and Pti6 Activate Defense Responses When Expressed in Arabidopsis.; citation_author=Y-Q Gu; citation_author=MC Wildermuth; citation_author=S Chakravarthy; citation_author=Y-T Loh; citation_author=C Yang; citation_journal_title=Plant Cell; citation_volume=14; citation_number=40; citation_pages=817-831; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=QTL mapping and quantitative disease resistance in plants.; citation_author=ND Young; citation_journal_title=Annual Review of Phytopathology; citation_volume=34; citation_number=41; citation_pages=479-501; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=QTLs for tomato powdery mildew resistance (Oidium lycopersici) in Lycopersicon parviflorum G1.1601 co-localize with two qualitative powdery mildew resistance genes.; citation_author=Y Bai; citation_author=CC Huang; citation_author=R van der Hulst; citation_author=F Meijer-Dekens; citation_author=G Bonnema; citation_journal_title=Mol Plant Microbe Interact; citation_volume=16; citation_number=42; citation_pages=169-176; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Pi35(t), a new gene conferring partial resistance to leaf blast in the rice cultivar Hokkai 188.; citation_author=TTT Nguyen; citation_author=S Koizumi; citation_author=TN La; citation_author=KS Zenbayashi; citation_author=T Ashizawa; citation_journal_title=Theor Appl Genet; citation_number=43; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=
                  Arabidopsis thaliana PAD4 encodes a lipase-like gene that is important for salycilic acid signaling.; citation_author=D Jirage; citation_author=TL Tootle; citation_author=TL Reuber; citation_author=LN Frost; citation_author=BJ Feys; citation_journal_title=Proceedings of The National Academy of Sciences of USA; citation_volume=96; citation_number=44; citation_pages=13583-13588; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=
                  EDS1, an essential component of R gene-mediated disease resistance in Arabidopsis has homology to eukaryotic lipases.; citation_author=A Falk; citation_author=BJ Feys; citation_author=LN Frost; citation_author=JDG Jones; citation_author=MJ Daniels; citation_journal_title=Proceedings of The National Academy of Sciences of USA; citation_volume=96; citation_number=45; citation_pages=3292-3297; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Arabidopsis SENESCENCE-ASSOCIATED GENE101 stabilizes and signals within an ENHANCED DISEASE SUSCEPTIBILITY1 complex in plant innate immunity.; citation_author=BJ Feys; citation_author=M Wiermer; citation_author=RA Bhat; citation_author=LJ Moisan; citation_author=N Medina-Escobar; citation_journal_title=Plant Cell; citation_volume=17; citation_number=46; citation_pages=2601-2613; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Quantitative Nature of Arabidopsis Responses during Compatible and Incompatible Interactions with the Bacterial Pathogen Pseudomonas syringae.; citation_author=Y Tao; citation_author=Z Xie; citation_author=W Chen; citation_author=J Glazebrook; citation_author=HS Chang; citation_journal_title=Plant Cell; citation_volume=15; citation_number=47; citation_pages=317-330; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Emerging strategies for enhancing crop resistance to microbial pathogens.; citation_author=CJ Lamb; citation_author=JA Ryals; citation_author=ER Ward; citation_author=RA Dixon; citation_journal_title=Biotechnology (N Y); citation_volume=10; citation_number=48; citation_pages=1436-1445; citation_date=1992; " />
      <meta name="citation_reference" content="citation_title=Bacterial elicitation and evasion of plant innate immunity.; citation_author=RB Abramovitch; citation_author=JC Anderson; citation_author=GB Martin; citation_journal_title=Nat Rev Mol Cell Biol; citation_volume=7; citation_number=49; citation_pages=601-611; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Host-microbe interactions: shaping the evolution of the plant immune response.; citation_author=ST Chisholm; citation_author=G Coaker; citation_author=B Day; citation_author=BJ Staskawicz; citation_journal_title=Cell; citation_volume=124; citation_number=50; citation_pages=803-814; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Pseudomonas syringae effector AvrPtoB suppresses basal defence in Arabidopsis.; citation_author=M de Torres; citation_author=JW Mansfield; citation_author=N Grabov; citation_author=IR Brown; citation_author=H Ammouneh; citation_journal_title=The Plant Journal; citation_volume=47; citation_number=51; citation_pages=368-382; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=QTLs for powdery mildew resistance in peach×Prunus davidiana crosses: consistency across generations and environments.; citation_author=M Foulongne; citation_author=T Pascal; citation_author=F Pfeiffer; citation_author=J Kervella; citation_journal_title=Molecular Breeding; citation_volume=12; citation_number=52; citation_pages=33-50; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Identification of QTLs for resistance to Sclerotinia sclerotiorum in soybean.; citation_author=VS Arahana; citation_author=GL Graef; citation_author=JE Specht; citation_author=JR Steadman; citation_author=KM Eskridge; citation_journal_title=Crop Science; citation_volume=41; citation_number=53; citation_pages=180-188; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=A QTL analysis of sunflower partial resistance to downy mildew (Plasmopara halstedii) and black stem (Phoma macdonaldii) by the use of recombinant inbred lines (RILs).; citation_author=GR Al-Chaarani; citation_author=A Roustaee; citation_author=L Gentzbittel; citation_author=L Mokrani; citation_author=G Barrault; citation_journal_title=Theoretical and Applied Genetics; citation_volume=104; citation_number=54; citation_pages=490-496; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Strain-specific and recessive QTLs involved in the control of partial resistance to Fusarium oxysporum f. sp. melonis race 1.2 in a recombinant inbred line population of melon.; citation_author=L Perchepied; citation_author=C Dogimont; citation_author=M Pitrat; citation_journal_title=Theor Appl Genet; citation_volume=111; citation_number=55; citation_pages=65-74; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Evolutionary dynamics of an Arabidopsis insect resistance quantitative trait locus.; citation_author=J Kroymann; citation_author=S Donnerhacke; citation_author=D Schnabelrauch; citation_author=T Mitchell-Olds; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=100; citation_number=56; citation_pages=14587-14592; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=The genetic architecture of quantitative traits: lessons from Drosophila.; citation_author=TF Mackay; citation_journal_title=Curr Opin Genet Dev; citation_volume=14; citation_number=57; citation_pages=253-257; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Genetic mechanisms and evolutionary significance of natural variation in Arabidopsis.; citation_author=T Mitchell-Olds; citation_author=J Schmitt; citation_journal_title=Nature; citation_volume=441; citation_number=58; citation_pages=947-952; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Effects of genetic background and environment on QTLs and epistasis for rice (Oryza sativa L.) panicle number.; citation_author=CY Liao; citation_author=P Wu; citation_author=B Hu; citation_author=KK Yi; citation_journal_title=Theoretical and Applied Genetics; citation_volume=103; citation_number=59; citation_pages=104-111; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Ontogenetics of QTL: the genetic architecture of trichome density over time in Arabidopsis thaliana.; citation_author=R Mauricio; citation_journal_title=Genetica; citation_volume=123; citation_number=60; citation_pages=75-85; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Organization of genes controlling disease resistance in the potato genome.; citation_author=C Gebhardt; citation_author=JPT Valkonen; citation_journal_title=Annual Review of Phytopathology; citation_volume=39; citation_number=61; citation_pages=79-102; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Putting knowledge of plant disease resistance genes to work.; citation_author=JD Jones; citation_journal_title=Curr Opin Plant Biol; citation_volume=4; citation_number=62; citation_pages=281-287; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Understanding the relationship between qualitative and quantitative genetics.; citation_author=A Robertson; citation_number=63; citation_pages=81-88; citation_date=1989; citation_publisher=Cold Spring Harbour Lab. Press; " />
      <meta name="citation_reference" content="citation_title=Xa21D encodes a receptor-like molecule with a leucine-rich repeat domain that determines race-specific recognition and is subject to adaptive evolution.; citation_author=GL Wang; citation_author=DL Ruan; citation_author=WY Song; citation_author=S Sideris; citation_author=L Chen; citation_journal_title=Plant Cell; citation_volume=10; citation_number=64; citation_pages=765-779; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=The molecular genetics of disease resistance in barley.; citation_author=KJ Williams; citation_journal_title=Aust J Agric Res; citation_volume=54; citation_number=65; citation_pages=1065-1079; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Genome-wide analysis of NBS-LRR-encoding genes in Arabidopsis.; citation_author=BC Meyers; citation_author=A Kozik; citation_author=A Griego; citation_author=H Kuang; citation_author=RW Michelmore; citation_journal_title=Plant Cell; citation_volume=15; citation_number=66; citation_pages=809-834; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Deciphering plant-pathogen communication: fresh perspectives for molecular resistance breeding.; citation_author=KE Hammond-Kosack; citation_author=JE Parker; citation_journal_title=Curr Opin Biotechnol; citation_number=67; citation_pages=177-193; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Quantitative trait Loci analysis of primary cell wall composition in Arabidopsis.; citation_author=G Mouille; citation_author=H Witucka-Wall; citation_author=MP Bruyant; citation_author=O Loudet; citation_author=S Pelletier; citation_journal_title=Plant Physiol; citation_volume=141; citation_number=68; citation_pages=1035-1044; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Use of Arabidopsis for genetic dissection of plant defense responses.; citation_author=J Glazebrook; citation_author=EE Rogers; citation_author=FM Ausubel; citation_journal_title=Annual Review of Genetic; citation_volume=31; citation_number=69; citation_pages=547-569; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Salicylic acid-independent ENHANCED DISEASE SUSCEPTIBILITY1 signaling in Arabidopsis immunity and cell death is regulated by the monooxygenase FMO1 and the Nudix hydrolase NUDT7.; citation_author=M Bartsch; citation_author=E Gobbato; citation_author=P Bednarek; citation_author=S Debey; citation_author=JL Schultze; citation_journal_title=Plant Cell; citation_volume=18; citation_number=70; citation_pages=1038-1051; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Identification of QTLs controlling gene expression networks defined a priori.; citation_author=DJ Kliebenstein; citation_author=MA West; citation_author=H van Leeuwen; citation_author=O Loudet; citation_author=RW Doerge; citation_journal_title=BMC Bioinformatics; citation_volume=7; citation_number=71; citation_pages=308; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Quantitative trait loci controlling root growth and architecture in Arabidopsis thaliana confirmed by heterogeneous inbred family.; citation_author=O Loudet; citation_author=V Gaudon; citation_author=A Trubuil; citation_author=F Daniel-Vedele; citation_journal_title=Theor Appl Genet; citation_volume=110; citation_number=72; citation_pages=742-753; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Identification of Pseudomonas syringae pathogens of Arabidopsis and a bacterial locus determining avirulence on both Arabidopsis and soybean.; citation_author=MC Whalen; citation_author=RW Innes; citation_author=AF Bent; citation_author=BJ Staskawicz; citation_journal_title=the plant Cell; citation_volume=3; citation_number=73; citation_pages=49-59; citation_date=1991; " />
      <meta name="citation_reference" content="citation_title=VASCULAR ASSOCIATED DEATH1, a novel GRAM domain-containing protein, is a regulator of cell death and defense responses in vascular tissues.; citation_author=S Lorrain; citation_author=B Lin; citation_author=MC Auriac; citation_author=T Kroj; citation_author=P Saindrenan; citation_journal_title=The Plant Cell; citation_volume=16; citation_number=74; citation_pages=2217-2232; citation_date=2004; " />
      <meta name="citation_reference" content="citation_author=A Gallais; citation_number=75; citation_pages=588; citation_date=1990; citation_publisher=Masson; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000123" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Natural Variation in Partial Resistance to Pseudomonas syringae Is Controlled by Two Major QTLs in Arabidopsis thaliana"/>
    <meta name="twitter:description" content="BackgroundLow-level, partial resistance is pre-eminent in natural populations, however, the mechanisms underlying this form of resistance are still poorly understood.Methodology/Principal FindingsIn the present study, we used the model pathosystem Pseudomonas syringae pv. tomato DC3000 (Pst) - Arabidopsis thaliana to study the genetic basis of this form of resistance. Phenotypic analysis of a set of Arabidopsis accessions, based on evaluation of in planta pathogen growth revealed extensive quantitative variation for partial resistance to Pst. It allowed choosing a recombinant inbred line (RIL) population derived from a cross between the accessions Bayreuth and Shahdara for quantitative genetic analysis. Experiments performed under two different environmental conditions led to the detection of two major and two minor quantitative trait loci (QTLs) governing partial resistance to Pst and called PRP-Ps1 to PRP-Ps4. The two major QTLs, PRP-Ps1 and PRP-Ps2, were confirmed in near isogenic lines (NILs), following the heterogeneous inbred families (HIFs) strategy. Analysis of marker gene expression using these HIFs indicated a negative correlation between the induced amount of transcripts of SA-dependent genes PR1, ICS and PR5, and the in planta bacterial growth in the HIF segregating at PRP-Ps2 locus, suggesting an implication of PRP-Ps2 in the activation of SA dependent responses.Conclusions/SignificanceThese results show that variation in partial resistance to Pst in Arabidopsis is governed by relatively few loci, and the validation of two major loci opens the way for their fine mapping and their cloning, which will improve our understanding of the molecular mechanisms underlying partial resistance."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0000123.g005"/>

  <meta property="og:title" content="Natural Variation in Partial Resistance to Pseudomonas syringae Is Controlled by Two Major QTLs in Arabidopsis thaliana" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000123" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=3517'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=973'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=9743&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0000123">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000123" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000123&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Natural Variation in Partial Resistance to <i>Pseudomonas syringae</i> Is Controlled by Two Major QTLs in <i>Arabidopsis thaliana</i>
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Laure Perchepied, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Laboratoire des Interactions Plantes-Microorganismes (LIPM), UMR Centre National de la Recherche Scientifique/National Institute for Agronomical Research 2594, Castanet-Tolosan, France
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Thomas Kroj, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Laboratoire des Interactions Plantes-Microorganismes (LIPM), UMR Centre National de la Recherche Scientifique/National Institute for Agronomical Research 2594, Castanet-Tolosan, France
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Maurice Tronchet, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Laboratoire des Interactions Plantes-Microorganismes (LIPM), UMR Centre National de la Recherche Scientifique/National Institute for Agronomical Research 2594, Castanet-Tolosan, France
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Olivier Loudet, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  National Institute for Agronomical Research (INRA), Versailles, France
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Dominique Roby
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:Dominique.roby@toulouse.inra.fr">Dominique.roby@toulouse.inra.fr</a>
            </p>

                <p>Affiliation:
                  Laboratoire des Interactions Plantes-Microorganismes (LIPM), UMR Centre National de la Recherche Scientifique/National Institute for Agronomical Research 2594, Castanet-Tolosan, France
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: December 27, 2006</li>
    <li>DOI: 10.1371/journal.pone.0000123</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0000123" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0000123" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000123" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0000123" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0000123-g001" data-doi="info:doi/10.1371/journal.pone.0000123" data-uri="info:doi/10.1371/journal.pone.0000123.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000123.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000123-t001" data-doi="info:doi/10.1371/journal.pone.0000123" data-uri="info:doi/10.1371/journal.pone.0000123.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000123.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000123-g002" data-doi="info:doi/10.1371/journal.pone.0000123" data-uri="info:doi/10.1371/journal.pone.0000123.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000123.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000123-g003" data-doi="info:doi/10.1371/journal.pone.0000123" data-uri="info:doi/10.1371/journal.pone.0000123.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000123.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000123-t002" data-doi="info:doi/10.1371/journal.pone.0000123" data-uri="info:doi/10.1371/journal.pone.0000123.t002" title="Table 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000123.t002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000123-g004" data-doi="info:doi/10.1371/journal.pone.0000123" data-uri="info:doi/10.1371/journal.pone.0000123.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000123.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000123-g005" data-doi="info:doi/10.1371/journal.pone.0000123" data-uri="info:doi/10.1371/journal.pone.0000123.g005" title="Figure 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000123.g005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000123">Reader Comments (1)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0000123" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2>
<h3>Background</h3>
<a id="article1.front1.article-meta1.abstract1.sec1.p1" name="article1.front1.article-meta1.abstract1.sec1.p1"></a><p>Low-level, partial resistance is pre-eminent in natural populations, however, the mechanisms underlying this form of resistance are still poorly understood.</p>


<h3>Methodology/Principal Findings</h3>
<a id="article1.front1.article-meta1.abstract1.sec2.p1" name="article1.front1.article-meta1.abstract1.sec2.p1"></a><p>In the present study, we used the model pathosystem <em>Pseudomonas syringae</em> pv. <em>tomato</em> DC3000 (<em>Pst</em>) - <em>Arabidopsis thaliana</em> to study the genetic basis of this form of resistance. Phenotypic analysis of a set of Arabidopsis accessions, based on evaluation of <em>in planta</em> pathogen growth revealed extensive quantitative variation for partial resistance to <em>Pst</em>. It allowed choosing a recombinant inbred line (RIL) population derived from a cross between the accessions Bayreuth and Shahdara for quantitative genetic analysis. Experiments performed under two different environmental conditions led to the detection of two major and two minor quantitative trait loci (QTLs) governing <span class="underline">p</span>artial <span class="underline">r</span>esistance to <em>
                     <span class="underline">P</span>st</em> and called <em>PRP-Ps1</em> to <em>PRP-Ps4</em>. The two major QTLs, <em>PRP-Ps1</em> and <em>PRP-Ps2</em>, were confirmed in near isogenic lines (NILs), following the heterogeneous inbred families (HIFs) strategy. Analysis of marker gene expression using these HIFs indicated a negative correlation between the induced amount of transcripts of SA-dependent genes <em>PR1</em>, <em>ICS</em> and <em>PR5</em>, and the <em>in planta</em> bacterial growth in the HIF segregating at <em>PRP-Ps2</em> locus, suggesting an implication of <em>PRP-Ps2</em> in the activation of SA dependent responses.</p>


<h3>Conclusions/Significance</h3>
<a id="article1.front1.article-meta1.abstract1.sec3.p1" name="article1.front1.article-meta1.abstract1.sec3.p1"></a><p>These results show that variation in partial resistance to <em>Pst</em> in Arabidopsis is governed by relatively few loci, and the validation of two major loci opens the way for their fine mapping and their cloning, which will improve our understanding of the molecular mechanisms underlying partial resistance.</p>

</div>


<div class="articleinfo"><p><strong>Citation: </strong>Perchepied L, Kroj T, Tronchet M, Loudet O, Roby D (2006) Natural Variation in Partial Resistance to <em>Pseudomonas syringae</em> Is Controlled by Two Major QTLs in <em>Arabidopsis thaliana</em>
            . PLoS ONE 1(1):
          e123.
            doi:10.1371/journal.pone.0000123</p><p><strong>Academic Editor: </strong>Justin Borevitz, University of Chicago, United States of America</p><p><strong>Received:</strong> November 3, 2006; <strong>Accepted:</strong> November 30, 2006; <strong>Published:</strong> December 27, 2006</p><p><strong>Copyright:</strong> © 2006 Perchepied et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>This work has been financially supported by a GABI-Genoplante-MCyt contract on Plant Genome Research.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>Plants are exposed to a wide variety of pathogens with different invasion strategies. Successful infections are, however, relatively rare because plants have evolved powerful preformed and inducible defence mechanisms to restrict pathogen growth. Nonhost resistance relies on multiple mechanisms, which are beginning to be uncovered <a href="#pone.0000123-Collins1">[1]</a>–<a href="#pone.0000123-Stein1">[3]</a>. This defense system shows some similarity with the mammalian innate immunity <a href="#pone.0000123-Nurnberger1">[4]</a>, <a href="#pone.0000123-Nurnberger2">[5]</a> and is associated with multiple signal transduction events like an oxidative burst, ion fluxes, activation of MAP kinase cascades, with the transcriptional induction of pathogen-responsive genes and with localized callose deposition at the cell wall <a href="#pone.0000123-Felix1">[6]</a>–<a href="#pone.0000123-Li1">[9]</a>. If a pathogen can overcome nonhost resistance, it can spread in its host plant; however, different defense mechanisms in plants can still be activated, leading to complete or partial resistance. Complete resistance, developed in the case of an incompatible interaction, is usually governed by the gene-for-gene system, and also called race-specific resistance. Much research has focused on this form of resistance which is generally inherited as a monogenic trait and is determined by the concomitant presence of a resistance (<em>R</em>) gene in the plant and the corresponding avirulence (avr) gene in the pathogen. Mechanistically, specific resistance relies on the recognition of <em>avr</em> pathogen factors by plant <em>R</em> gene products and the elicitation of local defense responses, often associated with a rapid programmed cell death, called the hypersensitive response (HR). A variety of <em>R</em> genes have been cloned from model and crop plants, and many <em>avr</em> genes have been characterized from bacteria, fungi and oomycetes<a href="#pone.0000123-Martin1">[10]</a>. Interestingly, although <em>R</em> genes confer resistance to diverse pathogens, their products share structural similarities suggesting the conservation of some signalling events in plant defense <a href="#pone.0000123-Martin1">[10]</a>.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>In contrast, the so-called partial resistance is quantitative, presumably non race-specific, and polygenic <a href="#pone.0000123-Keen1">[11]</a>–<a href="#pone.0000123-Hulbert1">[13]</a>. It limits the extent of disease caused by virulent pathogens and constitutes an additional layer of resistance in the absence of R function, during compatible interactions. The genetics of partial resistance has been characterized in many crop plants, such as rice and barley <a href="#pone.0000123-Inukai1">[14]</a>, <a href="#pone.0000123-Li2">[15]</a> but remains poorly understood in Arabidopsis. One way to increase our knowledge in this field is a genetic study of the quantitative variation in resistance to virulent pathogens.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>Although QTL analyses are increasingly used to study complex traits in Arabidopsis, such as developmental and yield traits <a href="#pone.0000123-AlonsoBlanco1">[16]</a>, <a href="#pone.0000123-Koornneef1">[17]</a>, only a few studies have investigated the genetic bases of quantitative variation in resistance and susceptibility to pathogens <a href="#pone.0000123-Wilson1">[18]</a>–<a href="#pone.0000123-Llorente1">[22]</a>. In most of these studies one or two major QTLs and a few minor loci were identified. In one study investigating plant susceptibility to the fungus <em>Botrytis cinerea</em>, multiple small-to-medium effect QTLs were identified <a href="#pone.0000123-Denby1">[20]</a>, suggesting that multiple mechanisms are involved in susceptibility, or that a large number of polymorphic loci exert an effect on a particular mechanism. In some cases, genes underlying the QTL have been identified. For example, the major QTL for resistance to <em>Plectosphaerella cucumerina</em> was demonstrated to correspond to the ERECTA gene <a href="#pone.0000123-Llorente1">[22]</a> which is implicated in plant development and also contributes to resistance to <em>Ralstonia solanacearum</em> 
            <a href="#pone.0000123-Godiard1">[19]</a>.</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p>The Arabidopsis-<em>Pseudomonas</em> interaction is a model pathosystem <a href="#pone.0000123-Quirino1">[23]</a>, <a href="#pone.0000123-Nomura1">[24]</a> and has largely contributed to a better understanding of pathogen recognition in plants, pathogen virulence and avirulence determinants, host susceptibility and signal transduction pathways controlling plant defense responses <a href="#pone.0000123-Katagiri1">[25]</a>. In a previous study, analysis of natural variation in tolerance indicated that it behaves as a quantitative trait <a href="#pone.0000123-Kover2">[26]</a>. Tolerance, which can be defined as the ability of the host to endure the presence of the pathogen and to express less severe disease symptoms or less damage <a href="#pone.0000123-Parlevliet1">[27]</a>, differs from resistance in that symptom formation is uncoupled from pathogen growth <a href="#pone.0000123-Buell1">[28]</a>. When the genetic basis of variation of this trait was analysed in the Arabidopsis accessions Col-0 and No-0, only two minor loci for symptom severity and no QTL for bacterial colonization could be identified <a href="#pone.0000123-Kover1">[21]</a>, although bacterial growth is a key quantitative component of the compatible interaction between Arabidopsis and the endophytic bacterial pathogen <em>Pst</em>.</p>
<a id="article1.body1.sec1.p5" name="article1.body1.sec1.p5"></a><p>In this paper, we use QTL analysis to investigate the genetic basis of partial resistance to <em>Pst</em> using the virulent strain DC3000. Natural variation in Arabidopsis for partial resistance to <em>Pst</em> allowed us to identify parental lines exhibiting significant differences in this trait, and to choose a RIL population derived from crosses between the accessions Bay-0 and Shahdara for detailed genetic analysis. Quantitative evaluation of this RIL population after infection with <em>Pst</em> DC3000 showed that partial resistance to <em>Pst</em> is controlled by two major and two minor QTLs. Using the heterogeneous inbred family strategy (HIF) <a href="#pone.0000123-Tuinstra1">[29]</a>, the two major QTLs were validated. To investigate whether these two major loci could influence known signalling pathways, the expression of marker defense genes was analyzed in HIFs, revealing an influence of the <em>PRP-Ps2</em> locus on the signalling pathway involving salicylic acid.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3>
<h4>Natural variation for <em>Pst</em> resistance in Arabidopsis</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>In order to investigate natural variation of partial resistance to <em>Pst</em> in <em>Arabidopsis thaliana</em>, 27 Arabidopsis accessions were tested for their response to the virulent <em>Pst</em> strain DC3000. The accessions correspond to a core-collection composed of 16 accessions which are estimated to represent most of the variation present within the species <em>Arabidopsis thaliana</em> 
               <a href="#pone.0000123-McKhann1">[30]</a>, plus other parental accessions of RIL populations that are publicly-available or under construction (<a href="http://www.inra.fr/vast/RILs.htm">http://www.inra.fr/vast/RILs.htm</a>).</p>
<a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2"></a><p>Plant resistance was evaluated by the measurement of bacterial <em>in planta</em> growth 3 days after leaf-infiltration of a bacterial suspension. The chosen inoculation procedure circumvents some layers of resistance operating in a natural infection process, but it is proven to be highly reproducible and allows the quantitative evaluation of resistance and susceptibility. As shown in <a href="#pone-0000123-g001">Figure 1</a>, bacterial growth varied continuously over a range of four orders of magnitude in the different accessions. This result indicates that partial resistance to <em>Pst</em> is a quantitative trait, suggesting that it may be under polygenic control.</p>
<div class="figure" id="pone-0000123-g001"><div class="img"><a name="pone-0000123-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000123.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000123" data-uri="info:doi/10.1371/journal.pone.0000123.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000123.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000123.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000123.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000123.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000123.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000123.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000123.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000123.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000123.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1. </strong></p><a id="article1.body1.sec2.sec1.fig1.caption1.p1" name="article1.body1.sec2.sec1.fig1.caption1.p1"></a><p>Natural variation of partial resistance to <em>Pseudomonas syringae</em> pv. <em>tomato</em> DC3000 among Arabidopsis accessions. <em>In planta</em> bacterial growth was assessed three days post inoculation with a bacterial suspension adjusted to 10<sup>5</sup> cfu/mL. Means and standard errors were calculated from bacterial densities in at least 4 plants.</p>
<span>doi:10.1371/journal.pone.0000123.g001</span></div><a id="article1.body1.sec2.sec1.p3" name="article1.body1.sec2.sec1.p3"></a><p>Among the parental accessions showing very contrasting phenotypes, L<em>er</em> and Cvi seemed to be good candidates for further QTL analysis, since a well characterised L<em>er</em>×Cvi RIL population exists <a href="#pone.0000123-AlonsoBlanco2">[31]</a>, <a href="#pone.0000123-AlonsoBlanco3">[32]</a>. However, the analysis of selected RILs from this population revealed, in accordance with published data <a href="#pone.0000123-AlonsoBlanco4">[33]</a> that 25% of the RILs are very early-flowering even under short day conditions. This was a major obstacle for the use of this population, because the evaluation of resistance is performed on fully expanded rosette leaves of non flowering plants. As the accessions Bayreuth (Bay-0) and Shahdara (Sha) also exhibited contrasting phenotypes, a F<sub>6</sub> RIL population of 420 lines derived from these lines was chosen for QTL analysis <a href="#pone.0000123-Loudet1">[34]</a>. Evaluation of resistance of reciprocal F<sub>1</sub> hybrids between Bay-0 and Shahdara revealed that bacterial densities in F<sub>1</sub> plants were similar to that evaluated in the parental line Shahdara, and higher than that in Bay-0 (Supplementary <a href="#pone-0000123-g001">Figure 1</a>).</p>


<h4>QTL analysis identifies two major and two minor loci for partial resistance</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>To identify loci responsible for the genetic differences in partial resistance to <em>Pst</em> between Bay-0 and Shahdara, three independent experiments, with two blocks in a complete randomized design, were performed on 165 RILs in greenhouse conditions, and one experiment was performed using 370 RILs under growth chamber conditions (<a href="#pone-0000123-t001">Table 1</a>). A total number of 1730 plants were evaluated for their response to <em>Pst</em>, using generally 14 plants per RIL.</p>
<div class="figure" id="pone-0000123-t001"><div class="img"><a name="pone-0000123-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000123.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000123" data-uri="info:doi/10.1371/journal.pone.0000123.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000123.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000123.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000123.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000123.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000123.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000123.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000123.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000123.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000123.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>The influence of genotype on partial resistance to <em>Pseudomonas syringae</em> pv. <em>tomato</em> DC3000.</span></strong></p><span>doi:10.1371/journal.pone.0000123.t001</span></div><a id="article1.body1.sec2.sec2.p2" name="article1.body1.sec2.sec2.p2"></a><p>Distributions of the RILs, according to bacterial populations measured <em>in planta</em> (log(cfu/cm<sup>2</sup>)), showed a continuous variation, suggesting a quantitative and polygenic control of the response (<a href="#pone-0000123-g002">Figure 2</a>). The parental accessions Bay-0 and Shahdara showed, as expected, different levels of bacterial growth. Moreover, transgressive segregation was observed in the RILs. Data obtained from the four experiments performed under different environmental conditions showed significant correlations (<a href="#pone.0000123.s002">Table S1</a>), and within each experiment, the randomized blocks were significantly correlated with R<sup>2</sup> values varying from 0.57 to 0.76 (data not shown). Variance analysis of the phenotypic data showed that differences among RILs were highly significant (<a href="#pone-0000123-t001">Table 1</a>). The block effect, except for one of the experiments (experiment 3), and the RIL/block interaction were also significant for all the experiments performed in the greenhouse. Two plants for each RIL were evaluated for their response to <em>Pst</em> in each block, but no significant plant effects could be detected (data not shown). Broad-sense heritabilities calculated from the different experiments, ranged from 0.58 to 0.77, indicating that most of the phenotypic variation appeared to be genetically determined (<a href="#pone-0000123-t001">Table 1</a>).</p>
<div class="figure" id="pone-0000123-g002"><div class="img"><a name="pone-0000123-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000123.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000123" data-uri="info:doi/10.1371/journal.pone.0000123.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000123.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000123.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000123.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000123.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000123.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000123.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000123.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000123.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000123.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2. </strong></p><a id="article1.body1.sec2.sec2.fig1.caption1.p1" name="article1.body1.sec2.sec2.fig1.caption1.p1"></a><p>Distribution of bacterial growth values in the Bay-0×Shahdara recombinant inbred line (RIL) population. The frequency histogram shows the range of <em>in planta</em> bacterial populations observed in RILs three days post inoculation with <em>Pseudomonas syringae</em> pv. <em>tomato</em> DC3000 in one of the greenhouse experiments. The values obtained for the parental accessions, Bay-0 and Shahdara (Sha), and the genetic mean of the population are indicated.</p>
<span>doi:10.1371/journal.pone.0000123.g002</span></div><a id="article1.body1.sec2.sec2.p3" name="article1.body1.sec2.sec2.p3"></a><p>For QTL detection, only composite interval mapping (CIM) results are presented for each experiment, because as compared to other methods, they provide a more accurate estimation of R<sup>2</sup> values (phenotypic variances explained by the QTL) and additive effects <a href="#pone.0000123-Zeng1">[35]</a>. Besides, the results of QTL detection obtained for each block of the experiments were the same as those obtained with the adjusted means on blocks (data not shown). Two major QTLs for partial resistance to <em>Pst,</em> 
               <em>PRP-Ps1</em> (<em>Partial Resistance to</em> 
               <em>Pathogen</em> - <em>Pseudomonas 1</em>) and <em>PRP-Ps2</em>, were detected in all experiments (<a href="#pone-0000123-g003">Figure 3</a>). <em>PRP-Ps1</em> was localized on chromosome 2 and explained more than 1/4 of the phenotypic variance (up to 42%) except in experiment 1 where it explained only 9%. <em>PRP-Ps2</em> usually explained less than 20% of the phenotypic variance, whereas its effect was much stronger in experiment 1 where it explained 62% (<a href="#pone-0000123-t002">Table 2</a>). The allelic additive effects of these QTLs are in the same direction, the Bay-0 allele increasing partial resistance compared to the Shahdara allele. In experiment 4, performed under growth chamber conditions, additional minor QTLs were detected on chromosomes III (R<sup>2</sup> = 8%) and V (R<sup>2</sup> = 5%). In contrast to the major loci, the Shahdara allele at these minor QTLs improves partial resistance to <em>Pst</em>. These observations might explain at least in part, transgressions above the Shahdara phenotypic value and below the Bay-0 phenotypic value in the RIL population (<a href="#pone-0000123-g002">Figure 2</a>).</p>
<div class="figure" id="pone-0000123-g003"><div class="img"><a name="pone-0000123-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000123.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000123" data-uri="info:doi/10.1371/journal.pone.0000123.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000123.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000123.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000123.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000123.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000123.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000123.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000123.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000123.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000123.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3. </strong></p><a id="article1.body1.sec2.sec2.fig2.caption1.p1" name="article1.body1.sec2.sec2.fig2.caption1.p1"></a><p>Arabidopsis QTLs controlling partial resistance to <em>Pseudomonas syringae</em> pv. <em>tomato</em> DC3000 in the Bay-0×Shahdara recombinant inbred line population. The detected QTLs are represented by bars located at the closest marker position (black, experiment 1; blue, experiment 2; green, experiment 3; purple, experiment 4) on the Bay-0×Shahdara genetic map <a href="#pone.0000123-Loudet1">[34]</a>. The length of the bar is proportional to the QTL effect (R<sup>2</sup>).</p>
<span>doi:10.1371/journal.pone.0000123.g003</span></div><div class="figure" id="pone-0000123-t002"><div class="img"><a name="pone-0000123-t002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000123.t002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000123" data-uri="info:doi/10.1371/journal.pone.0000123.t002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000123.t002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000123.t002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000123.t002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000123.t002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000123.t002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000123.t002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000123.t002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000123.t002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000123.t002.TIF"></span>)
                </a></li></ul></div><p><strong>Table 2.  <span>Arabidopsis QTLs controlling partial resistance to <em>Pseudomonas syringae</em> pv. <em>tomato</em> DC3000 in the Bay-0×Shahdara recombinant inbred line population.</span></strong></p><span>doi:10.1371/journal.pone.0000123.t002</span></div><a id="article1.body1.sec2.sec2.p4" name="article1.body1.sec2.sec2.p4"></a><p>Testing of the 703 possible pairwise interactions revealed highly significant digenic epistatic interactions (<a href="#pone-0000123-t002">Table 2</a>). Most of them occurred between markers with no additive effects and only a few of them were detected between a QTL (or close to the QTL) and a marker of the genetic background.</p>
<a id="article1.body1.sec2.sec2.p5" name="article1.body1.sec2.sec2.p5"></a><p>The total phenotypic variance explained by QTLs with additive effects for partial resistance to <em>Pst</em> ranged from 37% to 67% (data not shown), and increased to 39% to 73% when epistatic interactions were included (<a href="#pone-0000123-t002">Table 2</a>).</p>


<h4>Confirmation of the major QTLs <em>PRP-Ps1</em> and <em>PRP-Ps2</em> in near isogenic lines</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>To confirm the effects of the major QTLs <em>PRP-Ps1</em> and <em>PRP-Ps2</em> in near isogenic lines (NILs), RILs still segregating for the region of interest (so-called heterogeneous inbred families (HIFs)) were identified. Two sets of HIFs, segregating for the <em>PRP-Ps1</em> locus surrounding marker MSAT2-38, and two sets segregating for the <em>PRP-Ps2</em> locus surrounding marker MSAT5-14, were evaluated for partial resistance to <em>Pst</em> (<a href="#pone-0000123-g004">Figure 4A</a>). Each of the chosen RILs displays a different genetic background, a mix of both parental genomes Bay-0 and Shahdara.</p>
<div class="figure" id="pone-0000123-g004"><div class="img"><a name="pone-0000123-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000123.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000123" data-uri="info:doi/10.1371/journal.pone.0000123.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000123.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000123.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000123.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000123.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000123.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000123.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000123.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000123.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000123.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4. </strong></p><a id="article1.body1.sec2.sec3.fig1.caption1.p1" name="article1.body1.sec2.sec3.fig1.caption1.p1"></a><p>Validation of the major QTLs for partial resistance to <em>Pseudomonas syringae</em> pv. <em>tomato</em> DC3000 with heterogeneous inbred families (HIFs). <strong>A</strong> HIF 102 segregates around MSAT2-38 and HIF 200 segregates for a region of chromosome II around markers MSAT2-5 and MSAT2-38. HIF 214 segregates for a region of chromosome V around MSAT5-14 and NGA139 and HIF 400 segregates around MSAT5-14, NGA139 and MSAT5-22. Regions for which the HIFs segregate are indicated in hatched boxes and the white regions of the chromosomes represent a mix of both parental genomes Bay-0 and Shahdara. <strong>B</strong> and <strong>C</strong> 
                     <em>In planta</em> bacterial growth in HIFs fixed for the Bay (grey bar) or Sha (black bar) allele of <em>PRP-Ps1</em> (<strong>B</strong>) or <em>PRP-Ps2</em> (<strong>C</strong>). Each value is the average of measurement of <em>in planta</em> bacterial growth in at least 8 plants (means and standard errors) from one experiment. Three independent experiments were performed and showed similar results. Asterisks show significant difference in partial resistance (P&lt;0.05).</p>
<span>doi:10.1371/journal.pone.0000123.g004</span></div><a id="article1.body1.sec2.sec3.p2" name="article1.body1.sec2.sec3.p2"></a><p>As shown in <a href="#pone-0000123-g004">Figure 4B and 4C</a>, all the HIF lines carrying the Bay allele at <em>PRP-Ps1</em> or <em>PRP-Ps2</em> showed significantly lower bacterial colonization than the corresponding NILs carrying the Sha alleles. <em>In planta</em> bacterial growth was between 5 to 10 fold lower when the Bay-0 alleles were present. These results confirmed the quantitative contribution of both loci to partial <em>Pst</em> resistance and validated the major QTLs, <em>PRP-Ps1</em> and <em>PRP-Ps2</em>.</p>


<h4>
               <em>PRP-Ps2</em> influences SA-dependent gene expression</h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>Genetic analyses revealed that the balance between signalling components such as ethylene (ET), jasmonates (JA) and salicylic acid (SA) is crucial to modulate and adapt the various defense mechanisms to a given pathogen <a href="#pone.0000123-Kunkel1">[36]</a>. While JA and ET are important positive regulators in the resistance to necrotrophic pathogens, SA has been demonstrated to play a central role in resistance to biotrophs <a href="#pone.0000123-Kunkel1">[36]</a>, <a href="#pone.0000123-Thomma1">[37]</a>. Partial resistance to <em>Pst</em> relies to a large part on SA dependent defense responses and is negatively regulated by the JA pathway.</p>
<a id="article1.body1.sec2.sec4.p2" name="article1.body1.sec2.sec4.p2"></a><p>We therefore analysed to what extent <em>PRP-Ps1</em> and <em>PRP-Ps2</em> influenced signalling through these pathways. The expression of marker genes of SA and JA/ET signalling was investigated in the lines HIF200 and HIF214 segregating at the <em>PRP-Ps1</em> or <em>PRP-Ps2</em> locus, respectively. For SA-dependent signalling, the expression of the defense genes <em>PR1</em> (<em>Pathogenesis Related 1</em>), <em>PR5</em> (<em>Pathogenesis-Related 5</em>) and <em>ICS</em> (<em>IsoChorismate Synthase</em>) was analysed in parental ecotypes and in HIFs before and 24 h after infection with virulent <em>Pst</em> (<a href="#pone-0000123-g005">Figure 5</a>) <a href="#pone.0000123-Glazebrook1">[38]</a>, <a href="#pone.0000123-Nawrath1">[39]</a>. As expected, all three marker genes showed stronger pathogen-responsive expression in the more resistant Bay-0 accession than in Shadhara (<a href="#pone-0000123-g005">Figure 5</a>). Interestingly, they were also expressed at significantly higher levels in HIF214 plants that carry the Bay allele of <em>PRP-Ps2</em>, than in HIF214 plants carrying the Sha allele. Thus, the differences observed between the HIFs for <em>in planta</em> bacterial growth at the <em>PRP-Ps2</em> locus can be correlated with those observed for expression of SA-dependent genes, suggesting an implication of <em>PRP-Ps2</em> in the activation of SA dependent responses. In contrast, there was no significant difference in expression levels of SA marker genes when HIF200 plants were analysed, suggesting that the <em>PRP-Ps1</em> locus does not influence SA-dependent gene expression.</p>
<div class="figure" id="pone-0000123-g005"><div class="img"><a name="pone-0000123-g005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000123.g005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000123" data-uri="info:doi/10.1371/journal.pone.0000123.g005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000123.g005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000123.g005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000123.g005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000123.g005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000123.g005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000123.g005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000123.g005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000123.g005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000123.g005.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 5. </strong></p><a id="article1.body1.sec2.sec4.fig1.caption1.p1" name="article1.body1.sec2.sec4.fig1.caption1.p1"></a><p>Expression analysis of the defense marker genes <em>PR1</em> (A), <em>ICS</em> (B) and <em>PR5</em> (C) in HIFs and parental accessions. The transcript levels were determined by Q-RT-PCR with cDNA generated from leaves before (grey) and after (black) inoculation with <em>Pst</em> DC3000 at 5.10<sup>5</sup> cfu/mL. The expression value of the individual genes was normalized by using the expression level of <em>β-Tubulin4</em> as an internal standard. Mean mRNA levels of three plants are shown with corresponding standard errors. Similar results were obtained from two biological experiments.</p>
<span>doi:10.1371/journal.pone.0000123.g005</span></div><a id="article1.body1.sec2.sec4.p3" name="article1.body1.sec2.sec4.p3"></a><p>For JA- and ET-dependent signal transduction pathways, the expression of <em>PR3</em> (<em>Pathogenesis-Related 3</em>) and <em>VSP</em> (<em>Vegetative Storage Protein</em>) was analysed <a href="#pone.0000123-Glazebrook1">[38]</a>, <a href="#pone.0000123-Gu1">[40]</a>. Although these genes were induced upon inoculation, no correlation between resistance level and gene induction level could be observed when parental accessions or the HIFs were compared (data not shown).</p>

</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p>In natural populations there is a prevalence of low level, partial resistance that often prevents pathogens from reaching seriously damaging levels. Several studies have identified major genes controlling partial resistance in various crops <a href="#pone.0000123-Young1">[41]</a>–<a href="#pone.0000123-Nguyen1">[43]</a>. However, the molecular mechanisms underlying these loci were not identified.</p>
<a id="article1.body1.sec3.p2" name="article1.body1.sec3.p2"></a><p>In the model plant Arabidopsis, while the characterization of simply inherited <em>R</em> genes mediating strain-specific recognition of pathogens and complete resistance has been intensively investigated, the mechanisms that control partial resistance are poorly understood. Evidence has however been obtained that the different types of resistance already described (nonhost, complete, partial) have several features in common. For example, genes necessary for <em>R</em>-gene mediated resistance are also involved in non-host and partial resistances in Arabidopsis. For example, <em>eds1,</em> 
            <em>pad4</em> and <em>sag101</em> mutants, which are impaired in <em>R</em>-gene dependent resistance, show enhanced susceptibility to the virulent bacterium <em>Pst</em> and allow invasive growth of non-host powdery mildew isolates <a href="#pone.0000123-Lipka1">[2]</a>, <a href="#pone.0000123-Jirage1">[44]</a>–<a href="#pone.0000123-Feys1">[46]</a>. In addition, whole genome gene expression analysis <a href="#pone.0000123-Tao1">[47]</a> and the study of the expression of individual genes <a href="#pone.0000123-Lamb1">[48]</a>, indicate similar transcriptional changes in compatible and incompatible interactions with <em>Pst</em>, with most differences in the defense transcriptome being due to the kinetics and the amplitude of the response. In addition, defense mechanisms involved in partial, complete and nonhost resistances, are modulated by many type III effector proteins (TTEs) delivered by bacteria during infection <a href="#pone.0000123-Abramovitch1">[49]</a>, <a href="#pone.0000123-Chisholm1">[50]</a>. Interestingly, the outcome of the interaction between effectors and defense mechanisms depends not only on the nature of the effector but also on the kinetics of its delivery by the bacteria <a href="#pone.0000123-deTorres1">[51]</a>. In conclusion, these findings point to a complex network of regulators and regulatory mechanisms operating for the different forms of resistance in Arabidopsis. Although genetic analyses have clearly identified several <em>R</em> genes and some <em>EDS</em> genes acting in complete resistance pathways and affecting other types of resistance, to our knowledge and with the exception of the work of Kover already mentioned <a href="#pone.0000123-Kover1">[21]</a>, no such a direct genetic approach has been conducted for partial resistance to the model pathogen <em>Pst</em> in Arabidopsis, which should reveal major actors of this resistance.</p>
<a id="article1.body1.sec3.p3" name="article1.body1.sec3.p3"></a><p>The most important finding of this study was the discovery in Arabidopsis of two major loci governing variation in partial resistance to <em>Pst</em>, in the Bay-0×Shahdara RIL population. In addition, the validation of these loci opens the way for their fine mapping and their cloning, which will improve our understanding of the molecular mechanisms underlying partial resistance.</p>

<h4>PRP-Ps1 and PRP-Ps2, two major traits controlling partial resistance to Pseudomonas</h4>
<a id="article1.body1.sec3.sec1.p1" name="article1.body1.sec3.sec1.p1"></a><p>Two major QTLs and two minor loci controlling partial resistance to <em>Pst</em> have been identified through our quantitative analysis. For the two major QTLs, the Bay-0 allele enhances the resistance to <em>Pst</em>. On the contrary, for the two minor QTLs, <em>PRP-Ps3</em> and <em>PRP-Ps4</em>, the Shahdara allele enhances the resistance to <em>Pst</em>. Alleles from the susceptible parent that enhance disease resistance were already reported for several host-pathogen interactions <a href="#pone.0000123-Foulongne1">[52]</a>–<a href="#pone.0000123-Perchepied1">[55]</a>. This may also explain transgression towards susceptibility in the RIL population.</p>
<a id="article1.body1.sec3.sec1.p2" name="article1.body1.sec3.sec1.p2"></a><p>Highly significant correlations between experiments validate reproducibility of the phenotypic evaluations. The homogeneous behaviour of RILs also contributed to confer high heritabilities, indicating that the variations observed are mostly genetically controlled and that the estimation of partial resistance by phenotypic evaluations is reliable. However, the detected QTLs explained from 39% to 73% of the phenotypic variance, and comparison with the narrow sense heritabilities (from 0.58 to 0.77) suggests that not all the genetic variance is explained by these QTLs. This may result either i) from the choice of the significance threshold (2.7) which could have prevented the detection of minor QTLs, or ii) from the underestimation of the QTL contribution. Another explanation could be the involvement of the epistatic interactions detected in our study and which can potentially have an important contribution in genetic variation in complex traits <a href="#pone.0000123-Kroymann1">[56]</a>.</p>
<a id="article1.body1.sec3.sec1.p3" name="article1.body1.sec3.sec1.p3"></a><p>Strong interaction with the environment is one of the hallmarks of quantitative traits and it is not uncommon that the magnitude of QTLs varies with changing environmental conditions or even, that QTLs are only expressed in a given environment and not in others <a href="#pone.0000123-Mackay1">[57]</a>, <a href="#pone.0000123-MitchellOlds1">[58]</a>. Environmental effects on epistatic loci are even more pronounced <a href="#pone.0000123-Liao1">[59]</a>, <a href="#pone.0000123-Mauricio1">[60]</a>. Therefore plant resistance and susceptibility are strongly influenced by environmental conditions, and must be studied in controlled experimental setups. The fact that, in another study investigating natural variation in tolerance of Arabidopsis to <em>Pst</em>, different resistance levels were found for several accessions <a href="#pone.0000123-Kover2">[26]</a>, may be due to such environmental effects. Although environment-genotype interactions were not evaluated in our study, we detected significant “experiment” and “block” effects (<a href="#pone-0000123-t001">Table 1</a>) suggesting that environmental conditions influence partial resistance to <em>Pst</em>. In addition, the relative contributions of the major QTLs, <em>PRP-Ps1</em> and <em>PRP-Ps2,</em> to the overall phenotypic values showed some variation between experiment 1 and experiments 2, 3 and 4, and the detected epistatic interactions were not consistent for all experiments. However, despite the apparent effects of the environment on partial resistance, the two major QTLs, <em>PRP-Ps1</em> and <em>PRP-Ps2</em>, were detected in four independent experiments whatever the environmental conditions, and could be confirmed in NILs using the powerful HIF strategy <a href="#pone.0000123-Tuinstra1">[29]</a>. The fact that the effects of <em>PRP-Ps1</em> and <em>PRP-Ps2</em> can be detected in segregating populations opens the way for their fine-mapping and their molecular cloning. Knowing the molecular identity of <em>PRP-Ps1</em> and <em>PRP-Ps2</em> will not only give new, fundamental insights into partial resistance, but also allow to study on a molecular level, the plasticity of partial resistance.</p>


<h4>Candidate loci</h4>
<a id="article1.body1.sec3.sec2.p1" name="article1.body1.sec3.sec2.p1"></a><p>Two major QTLs, and two minor loci accounting for about one half of the phenotypic variation have been identified through our quantitative analysis, suggesting that a few major mechanisms are controlling partial resistance to <em>Pst</em>. This is surprisingly similar to complete resistance in which the genetics of the plant-pathogen interaction is under the control of one or a few major genes. Consequently, it would be interesting to investigate whether putative disease resistance proteins, or other candidate genes related to different types of resistance (as described earlier), are found at the genomic location of the QTLs identified in this study. However, since no pathogen resistance QTL has been cloned as such and only a few genes acting as resistance QTL in defined interactions have been identified, the molecular nature of such genes remains elusive.</p>
<a id="article1.body1.sec3.sec2.p2" name="article1.body1.sec3.sec2.p2"></a><p>Partial resistance QTLs frequently co-localize with known <em>R</em> gene loci and it is assumed that many QTLs correspond to weak or defeated <em>R</em>-genes with, in most cases, the canonical NB-LRR domains<a href="#pone.0000123-Gebhardt1">[61]</a>. Since resistance genes are highly polymorphic within species and populations <a href="#pone.0000123-Jones1">[62]</a>, such QTLs may correspond to allelic variants of qualitative resistance genes with intermediate phenotypes <a href="#pone.0000123-Robertson1">[63]</a>. For example, the rice <em>Xa21D</em> gene confers partial resistance to bacterial blight, while other <em>Xa21</em> alleles confer complete resistance <a href="#pone.0000123-Wang1">[64]</a>. In barley, clustering of major resistance gene and resistance QTL has also been reported <a href="#pone.0000123-Williams1">[65]</a>. <em>PRP-Ps2</em>, <em>PRP-Ps3</em> and <em>PRP-Ps4</em> co-localize with regions containing hypothetical NB-LRR resistance genes (<a href="http://niblrrs.ucdavis.edu/At-RGenes">http://niblrrs.ucdavis.edu/At-RGenes</a>) <a href="#pone.0000123-Meyers1">[66]</a>, making them potential candidate genes. Thus, within the confidence interval defined for the QTL <em>PRP-Ps4</em> on chromosome V, is located a cluster of resistance genes, and among them the <em>R</em> gene <em>RPS4</em>. In addition, <em>PRP-Ps2</em> and <em>PRP-Ps3</em> co-localize with a resistance QTL to powdery mildew, <em>RPW11</em>
               <a href="#pone.0000123-Wilson1">[18]</a>, and with a QTL detected for tolerance to <em>Pst</em> associated with symptom severity <a href="#pone.0000123-Kover1">[21]</a>, respectively.</p>
<a id="article1.body1.sec3.sec2.p3" name="article1.body1.sec3.sec2.p3"></a><p>In addition to a role in pathogen perception, partial resistance QTLs may also operate in signal transduction. Multiple signalling elements of resistance have been identified by genetic means and a central role of SA production and signal transduction is well established <a href="#pone.0000123-HammondKosack1">[67]</a>. However, none of the identified genes nor <em>eds</em>, co-localize with the <em>PRP-Ps</em> QTLs.</p>
<a id="article1.body1.sec3.sec2.p4" name="article1.body1.sec3.sec2.p4"></a><p>Partial resistance QTLs might also correspond to genes involved in constitutive defense responses. The cell wall is generally believed to be an efficient physical barrier against microbial attack. Alterations in cell wall structure could therefore result in altered partial resistance. QTL analysis employing the Bay-Sha population has detected multiple loci influencing cell wall structure <a href="#pone.0000123-Mouille1">[68]</a> and some of them co-localize to <em>PRP-Ps</em> QTLs. <em>PRP-Ps1</em> co-localizes with the ARH-1 QTL for the Ara-Rha ratio reflecting difference in RGI (rhamnogalacturonan I) structure and <em>PRP-Ps3</em> and <em>PRP-Ps4</em> co-localize with two minor QTLs, HLD-3 and HLD-4 (HLD for dark-grown hypocotyl length) for hypocotyl length reflecting cell elongation.</p>
<a id="article1.body1.sec3.sec2.p5" name="article1.body1.sec3.sec2.p5"></a><p>In summary, a number of genes putatively associated with the plant defense system, co-localize with some of the <em>PRP-Ps</em> loci. However, the confidence intervals found for these QTLs involve rather large genomic regions, which need to be reduced using HIFs to define candidate genes more precisely.</p>


<h4>Gene expression/pathways controlled by PRP-Ps1 and PRP-Ps2</h4>
<a id="article1.body1.sec3.sec3.p1" name="article1.body1.sec3.sec3.p1"></a><p>The central role of SA-dependent defense responses in resistance to <em>Pst</em> is well established <a href="#pone.0000123-Kunkel1">[36]</a>. Mutants affected in SA production or signal transduction show reduced resistance to <em>Pst</em> 
               <a href="#pone.0000123-Nawrath1">[39]</a>. The finding that the resistance level in HIFs differing in the <em>PRP-Ps2</em> locus, correlates with the expression of marker genes of the SA response is therefore particularly interesting. It indicates that the <em>PRP-Ps2</em> locus might be involved in the activation of inducible defence responses associated with partial resistance, but it raises the question at what level <em>PRP-Ps2</em> is acting; in SA production, in SA signalling or up-stream of SA? Quantification of SA levels and responsiveness of HIFs to SA application should provide answers to this question. <em>PRP-Ps1</em>, in contrast, seems to act independently of SA. At least SA pathway marker genes show no differential expression in an HIF segregating for this locus. Therefore, <em>PRP-Ps1</em> may act in constitutively expressed defense responses or in SA-independent pathways necessary for complete resistance. Such pathways have been revealed by mutant and transcriptome analysis <a href="#pone.0000123-Glazebrook1">[38]</a>, <a href="#pone.0000123-Glazebrook2">[69]</a> and their elements are beginning to be identified <a href="#pone.0000123-Bartsch1">[70]</a>.</p>
<a id="article1.body1.sec3.sec3.p2" name="article1.body1.sec3.sec3.p2"></a><p>In this context, an interesting recent study aimed at identifying loci controlling transcriptome variation by global eQTL analysis <a href="#pone.0000123-Kliebenstein1">[71]</a>. Categorizing eQTLs has the potential to enable reverse genetics for the identification of genes controlling quantitative traits and may also help to enhance the rate of QTL cloning. Interestingly in this analysis, several eQTLs detected on chromosomes II, III and V are related to networks involved in plant-pathogen responses, suggesting that they may influence variation in plant-pathogen interactions in the Bay-0×Sha population. By mapping QTLs for partial <em>Pst</em> resistance in the same RIL population that has been used for global gene expression analysis, the relationship between partial resistance to <em>Pst</em> and genetic architecture of eQTLs could be evaluated. More generally, combining QTL mapping with whole genome expression analysis to identify transcripts corresponding to QTLs will give a more complete picture of the complex genetic architecture of quantitative traits <a href="#pone.0000123-Mackay1">[57]</a>.</p>
<a id="article1.body1.sec3.sec3.p3" name="article1.body1.sec3.sec3.p3"></a><p>In the future, molecular cloning of the QTLs identified in our study will certainly help to understand the molecular basis of quantitative variation of partial resistance to <em>Pseudomonas syringae</em> in Arabidopsis, a model pathosystem in the plant pathology field.</p>

</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Methods"></a><h3>Methods</h3>
<h4>Plant material and growth conditions</h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p>The Bay-0×Shahdara RIL population and the heterogeneous inbred families (HIFs) derived from the RIL population were developed by Loudet <em>et al.</em> 
               <a href="#pone.0000123-Loudet1">[34]</a> and unpublished. A complete description of the RIL population is available at <a href="http://dbsgap.versailles.inra.fr/vnat/Fichier_collection/Rech_rils_pop.php">http://dbsgap.versailles.inra.fr/vnat/Fi​chier_collection/Rech_rils_pop.php</a>. HIFs were obtained as previously described <a href="#pone.0000123-Loudet2">[72]</a>. The reciprocal F<sub>1</sub> hybrids derived from the cross between Bay-0 and Shahdara (F<sub>1</sub> BS) and between Shahdara and Bay-0 (F<sub>1</sub> SB) were generated in our laboratory. Plants were grown in a growth chamber at 22°C, with a 9h light period and a light intensity of 190 µmol/m<sup>2</sup>/s. All experiments were performed with 4 to 5 week-old plants.</p>


<h4>Bacterial strains, plant inoculation procedure and bacterial growth measurement</h4>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1"></a><p>
               <em>Pseudomonas syringae</em> pv. <em>tomato</em> (<em>Pst</em>) DC3000 strain was grown at 29°C on KingB's medium supplemented with 50 µg/mL of rifampicin <a href="#pone.0000123-Whalen1">[73]</a>. For the determination of <em>in planta</em> bacterial growth, we used an inoculum of 4.10<sup>4</sup> cfu/mL. For gene expression analysis, we used an inoculum of 5.10<sup>5</sup> cfu/mL. Plant inoculations and <em>in planta</em> bacterial growth analysis were performed essentially as described previously with two to four plants per RIL and per experiment <a href="#pone.0000123-Lorrain1">[74]</a>. In order to analyze <em>in planta</em> bacterial densities in hundreds of plants in parallel, leaf discs were harvested in 96 deep-well microtiter plates. Bacteria were extracted by addition of 0.2% Silwet L-77 and shaking for 30 minutes. Subsequent serial dilutions and depositions on culture medium were performed by using microtiter plates and multichannel pipettes.</p>


<h4>Experimental design for Bay-0xShahdara RIL population and HIFs</h4>
<a id="article1.body1.sec4.sec3.p1" name="article1.body1.sec4.sec3.p1"></a><p>The RIL population was evaluated by performing 4 experiments under different conditions. Three experiments were conducted with 165 RILs of the population under greenhouse conditions and another one with 370 RILs under growth chamber conditions. For the three experiments performed in greenhouse, each RIL was evaluated in a complete randomized block design. Two plants of each RIL per block with two blocks in each experiment were evaluated for partial resistance to <em>Pst</em>. For the other experiment in growth chamber, complete randomized design of RILs with two plants for each RIL was used.</p>
<a id="article1.body1.sec4.sec3.p2" name="article1.body1.sec4.sec3.p2"></a><p>HIFs were also evaluated for partial resistance to <em>Pst</em> by <em>in planta</em> bacterial growth analysis. At least, 32 plants of each HIF were tested in two to three independent experiments.</p>
<a id="article1.body1.sec4.sec3.p3" name="article1.body1.sec4.sec3.p3"></a><p>In the case of the evaluations of the F<sub>1</sub> hybrids, 16 plants were tested for each cross.</p>


<h4>Statistical analyses</h4>
<a id="article1.body1.sec4.sec4.p1" name="article1.body1.sec4.sec4.p1"></a><p>Data was analyzed for each block and each experiment. Adjusted means of disease scores (lsmeans) of RILs in blocks were estimated from variance analysis (ANOVA). Phenotypic correlations among the blocks, the experiments, and the variables were calculated. When replicates were available for an experiment, broad sense heritabilities (h<sup>2</sup>) were estimated from the mean square (MS) of ANOVA using the formula adapted from Gallais <a href="#pone.0000123-Gallais1">[75]</a>:<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000123.e001&amp;representation=PNG" class="inline-graphic"></span><br>where σ<sup>2</sup>
               <sub>g</sub> is the genetic variance (MSg-MSgr)/rn, σ<sup>2</sup>
               <sub>gr</sub> the genotype*block interaction (MSgr-MSe)/n, σ<sup>2</sup>
               <sub>e</sub> the environmental variance (MSe), n the number of plants and r the number of replicates.</p>
<a id="article1.body1.sec4.sec4.p2" name="article1.body1.sec4.sec4.p2"></a><p>Data analyses were performed with Statistical Analysis System (SAS) software (SAS Institute Inc., North Carolina, USA). Variance analysis of <em>in planta</em> bacterial growth data was performed using PROC GLM of SAS with randomized effects.</p>


<h4>QTL detection</h4>
<a id="article1.body1.sec4.sec5.p1" name="article1.body1.sec4.sec5.p1"></a><p>The QTL was detected on the lsmeans of <em>in planta</em> bacterial growth data for each experiment, and on the means of the <em>in planta</em> bacterial growth data of each block (when the block effect was highly significant, P&lt;0.01). Variance analysis (LR), Interval mapping (IM) and composite interval mapping (CIM) were performed with the QTL Cartographer software Version 1.17 (Basten et al, North Carolina State University, USA) for each trait. Interval mapping (IM) and composite interval mapping (CIM) were also performed with PLABQTL Version 1.2 (Utz and Melchinger, University of Hohenheim, Germany). After performing 1000 permutations with ANOVA, a LOD threshold of 2.7 was used to declare a putative QTL significant. For CIM, 2 to 4 of the most informative markers per trait were chosen as cofactors. For each trait, a multiway ANOVA was performed with molecular markers near the QTL peaks to estimate the total percentage of phenotypic variation (R<sup>2</sup>) explained by the significant QTLs. QTLs were named <em>PRP-Ps</em> for <span class="underline">P</span>artial <span class="underline">R</span>esistance to <span class="underline">P</span>athogen - <em>
                  <span class="underline">P</span>st</em>. Confidence intervals of the detected QTLs were estimated from the PLABQTL software.</p>
<a id="article1.body1.sec4.sec5.p2" name="article1.body1.sec4.sec5.p2"></a><p>In addition to additive effects, digenic epistasis was tested with a two-factor ANOVA model with an interaction between pairs of markers. With the PROC GLM of SAS software, 703 interaction tests were performed and a significance level of <em>P</em>&lt;0.001 (0.7 false positive) was chosen for detecting digenic epistasis. Global R<sup>2</sup> was estimated with full ANOVA including all additive effects and digenic epistatic effects.</p>


<h4>RNA extraction and Q-RT-PCR analysis</h4>
<a id="article1.body1.sec4.sec6.p1" name="article1.body1.sec4.sec6.p1"></a><p>Material for RNA analysis was ground in liquid nitrogen and total RNA was isolated using the Macherey-Nagel Nucleospin RNA plant kit (Macherey-Nagel, Hoerdt, France) according to the manufacturer's recommendations. Reverse transcription was performed with 1 µg of total RNA using the superscript reverse transcriptase II (Invitrogen, Carlsbad, CA, USA). Quantitative PCR was run on a Lightcycler system (Roche Diagnostics, Meylan, France) according to the manufacturer's recommendations with the following conditions: 1 cycle: 9 min at 95°C for; 45 cycles: 5 sec at 95°C, 10 sec at 65°C and 20 sec at 72°C. <em>β-Tubulin4</em> was used as an internal standard. The primer sets used are for <em>PR1</em> (locus At2g14610, forward primer: GGAGCTACGCAGAACAACTAAGA, reverse primer: CCCACGAGGATCATAGTTGCAACTGA), for <em>PR5</em> (locus At1g75040, forward primer: CGGTACAAGTGAAGGTGCTCGTT, reverse primer: GCCTCGTAGATG GTTACAATGTCA), and for <em>ICS</em> (At1g74710, forward primer: GCCGTCTCTGAAC TCAAATCTCAA, reverse primer: GTTACGAGCAAGAACAACCTTGTT). Specificity of the amplifications was verified by melting curve analysis. Efficiency of the amplification was verified by the analysis of standard curves.</p>

</div>

<div id="section5" class="section"><a id="s5" name="s5" toc="s5" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0000123"></div><a name="pone.0000123.s001" id="pone.0000123.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000123.s001">Figure S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1"></a><p class="preSiDOI">Analysis of the F1 progeny of reciprocal crosses between Bay-0 and Shahdara for partial resistance to <em>Pseudomonas syringae</em> pv. <em>tomato</em> DC3000. <em>In planta</em> bacterial growth was assessed three days post inoculation in the F1 generation of reciprocal crosses between Bay-0 and Shahdara. Each value is the average of <em>in planta</em> bacterial growth of at least 4 plants (means and standard errors).</p>
<p class="siDoi">doi:10.1371/journal.pone.0000123.s001</p><a id="article1.body1.sec5.supplementary-material1.caption1.p2" name="article1.body1.sec5.supplementary-material1.caption1.p2"></a><p class="postSiDOI">(0.62 MB EPS)</p>
<a name="pone.0000123.s002" id="pone.0000123.s002"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000123.s002">Table S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material2.caption1.p1" name="article1.body1.sec5.supplementary-material2.caption1.p1"></a><p class="preSiDOI">Phenotypic correlations among experiments</p>
<p class="siDoi">doi:10.1371/journal.pone.0000123.s002</p><a id="article1.body1.sec5.supplementary-material2.caption1.p2" name="article1.body1.sec5.supplementary-material2.caption1.p2"></a><p class="postSiDOI">(0.04 MB DOC)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>We would like to thank Vera Saliba-Colombani and Fabien Nogué for providing some of the HIF lines. We are also grateful to Olivier Bouchez, Noëllie Journot and Raimana Louette for technical help, to Yves Marco and Clare Gough for critical reviewing of the manuscript, and to Claudine Balagué for helpful discussions.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: DR TK. Performed the experiments: LP TK MT. Analyzed the data: OL LP TK. Contributed reagents/materials/analysis tools: OL. Wrote the paper: DR LP TK.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0000123-Collins1" id="pone.0000123-Collins1"></a>Collins NC, Thordal-Christensen H, Lipka V, Bau S, Kombrink E, et al.  (2003) SNARE-protein-mediated disease resistance at the plant cell wall. Nature  425: 973–977.  <ul class="find" data-citedArticleID="923094" data-doi="10.1038/nature02076"><li><a href="http://dx.doi.org/10.1038/nature02076" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=SNARE-protein-mediated+disease+resistance+at+the+plant+cell+wall." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22SNARE-protein-mediated+disease+resistance+at+the+plant+cell+wall.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0000123-Lipka1" id="pone.0000123-Lipka1"></a>Lipka V, Dittgen J, Bednarek P, Bhat R, Wiermer M, et al.  (2005) Pre- and postinvasion defenses both contribute to nonhost resistance in Arabidopsis. Science  310: 1180–1183.  <ul class="find" data-citedArticleID="923154" data-doi="10.1126/science.1119409"><li><a href="http://dx.doi.org/10.1126/science.1119409" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Pre-+and+postinvasion+defenses+both+contribute+to+nonhost+resistance+in+Arabidopsis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Pre-+and+postinvasion+defenses+both+contribute+to+nonhost+resistance+in+Arabidopsis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0000123-Stein1" id="pone.0000123-Stein1"></a>Stein M, Dittgen J, Sanchez-Rodriguez C, Hou BH, Molina A, et al.  (2006) Arabidopsis PEN3/PDR8, an ATP binding cassette transporter, contributes to nonhost resistance to inappropriate pathogens that enter by direct penetration. Plant Cell  18: 731–746.  <ul class="find" data-citedArticleID="923196" data-doi="10.1105/tpc.105.038372"><li><a href="http://dx.doi.org/10.1105/tpc.105.038372" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Arabidopsis+PEN3%2FPDR8%2C+an+ATP+binding+cassette+transporter%2C+contributes+to+nonhost+resistance+to+inappropriate+pathogens+that+enter+by+direct+penetration." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Arabidopsis+PEN3%2FPDR8%2C+an+ATP+binding+cassette+transporter%2C+contributes+to+nonhost+resistance+to+inappropriate+pathogens+that+enter+by+direct+penetration.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0000123-Nurnberger1" id="pone.0000123-Nurnberger1"></a>Nurnberger T, Brunner F, Kemmerling B, Piater L (2004) Innate immunity in plants and animals: striking similarities and obvious differences. Immunol Rev  198: 249–266.  <ul class="find" data-citedArticleID="923184" data-doi="10.1111/j.0105-2896.2004.0119.x"><li><a href="http://dx.doi.org/10.1111/j.0105-2896.2004.0119.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Innate+immunity+in+plants+and+animals%3A+striking+similarities+and+obvious+differences." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Innate+immunity+in+plants+and+animals%3A+striking+similarities+and+obvious+differences.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0000123-Nurnberger2" id="pone.0000123-Nurnberger2"></a>Nurnberger T, Brunner F (2002) Innate immunity in plants and animals: emerging parallels between the recognition of general elicitors and pathogen-associated molecular patterns. Curr Opin Plant Biol  5: 318–324.  <ul class="find" data-citedArticleID="923186" data-doi="10.1016/s1369-5266(02)00265-0"><li><a href="http://dx.doi.org/10.1016/s1369-5266(02)00265-0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Innate+immunity+in+plants+and+animals%3A+emerging+parallels+between+the+recognition+of+general+elicitors+and+pathogen-associated+molecular+patterns." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Innate+immunity+in+plants+and+animals%3A+emerging+parallels+between+the+recognition+of+general+elicitors+and+pathogen-associated+molecular+patterns.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0000123-Felix1" id="pone.0000123-Felix1"></a>Felix G, Duran JD, Volko S, Boller T (1999) Plants have a sensitive perception system for the most conserved domain of bacterial flagellin. Plant J  18: 265–276.  <ul class="find" data-citedArticleID="923100" data-doi="10.1046/j.1365-313x.1999.00265.x"><li><a href="http://dx.doi.org/10.1046/j.1365-313x.1999.00265.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Plants+have+a+sensitive+perception+system+for+the+most+conserved+domain+of+bacterial+flagellin." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Plants+have+a+sensitive+perception+system+for+the+most+conserved+domain+of+bacterial+flagellin.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0000123-Asai1" id="pone.0000123-Asai1"></a>Asai T, Tena G, Plotnikova J, Willmann MR, Chiu WL, et al.  (2002) MAP kinase signalling cascade in Arabidopsis innate immunity. Nature  415: 977–983.  <ul class="find" data-citedArticleID="923084" data-doi="10.1038/415977a"><li><a href="http://dx.doi.org/10.1038/415977a" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=MAP+kinase+signalling+cascade+in+Arabidopsis+innate+immunity." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22MAP+kinase+signalling+cascade+in+Arabidopsis+innate+immunity.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0000123-Zipfel1" id="pone.0000123-Zipfel1"></a>Zipfel C, Robatzek S, Navarro L, Oakeley EJ, Jones JD, et al.  (2004) Bacterial disease resistance in Arabidopsis through flagellin perception. Nature  428: 764–767.  <ul class="find" data-citedArticleID="923216" data-doi="10.1038/nature02485"><li><a href="http://dx.doi.org/10.1038/nature02485" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Bacterial+disease+resistance+in+Arabidopsis+through+flagellin+perception." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Bacterial+disease+resistance+in+Arabidopsis+through+flagellin+perception.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0000123-Li1" id="pone.0000123-Li1"></a>Li X, Lin H, Zhang W, Zou Y, Zhang J, et al.  (2005) Flagellin induces innate immunity in nonhost interactions that is suppressed by Pseudomonas syringae effectors. Proc Natl Acad Sci U S A  102: 12990–12995.  <ul class="find" data-citedArticleID="923148" data-doi="10.1073/pnas.0502425102"><li><a href="http://dx.doi.org/10.1073/pnas.0502425102" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Flagellin+induces+innate+immunity+in+nonhost+interactions+that+is+suppressed+by+Pseudomonas+syringae+effectors." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Flagellin+induces+innate+immunity+in+nonhost+interactions+that+is+suppressed+by+Pseudomonas+syringae+effectors.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0000123-Martin1" id="pone.0000123-Martin1"></a>Martin GB, Bogdanove AJ, Sessa G (2003) Understanding the functions of plant disease resistance proteins. Annu Rev Plant Biol  54: 23–61.  <ul class="find" data-citedArticleID="923166" data-doi="10.1146/annurev.arplant.54.031902.135035"><li><a href="http://dx.doi.org/10.1146/annurev.arplant.54.031902.135035" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Understanding+the+functions+of+plant+disease+resistance+proteins." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Understanding+the+functions+of+plant+disease+resistance+proteins.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0000123-Keen1" id="pone.0000123-Keen1"></a>Keen NT (1990) Gene-for-gene complementarity in plant-pathogen interactions. Annual Review of Genetic  24: 447–463.  <ul class="find" data-citedArticleID="923132" data-doi="10.1146/annurev.genet.24.1.447"><li><a href="http://dx.doi.org/10.1146/annurev.genet.24.1.447" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Gene-for-gene+complementarity+in+plant-pathogen+interactions." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Gene-for-gene+complementarity+in+plant-pathogen+interactions.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0000123-Kamoun1" id="pone.0000123-Kamoun1"></a>Kamoun S, Huitema E, Vleeshouwers VGAA (1999) Resistance to oomycetes: a general role for the hypersensitive response? Trends in Plant Science  4: 196–200.  <ul class="find" data-citedArticleID="923128" data-doi="10.1016/s1360-1385(99)01404-1"><li><a href="http://dx.doi.org/10.1016/s1360-1385(99)01404-1" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Resistance+to+oomycetes%3A+a+general+role+for+the+hypersensitive+response%3F" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Resistance+to+oomycetes%3A+a+general+role+for+the+hypersensitive+response%3F%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0000123-Hulbert1" id="pone.0000123-Hulbert1"></a>Hulbert SH, Webb CA, Smith SM, Sun Q (2001) RESISTANCE GENE COMPLEXES: Evolution and Utilization. Annual Review of Phytopathology  39: 285–312.  <ul class="find" data-citedArticleID="923120" data-doi="10.1146/annurev.phyto.39.1.285"><li><a href="http://dx.doi.org/10.1146/annurev.phyto.39.1.285" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=RESISTANCE+GENE+COMPLEXES%3A+Evolution+and+Utilization." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22RESISTANCE+GENE+COMPLEXES%3A+Evolution+and+Utilization.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0000123-Inukai1" id="pone.0000123-Inukai1"></a>Inukai T, Vales MI, Hori K, Sato K, Hayes PM (2006) 
                  <em>RMo</em>1 confers blast resistance in barley and is located within the complex of resistance genes containing <em>Mla</em>, a powdery mildew resistance gene. Mol Plant Microbe Interact  19: 1034–1041.  <ul class="find" data-citedArticleID="923122"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=RMo1+confers+blast+resistance+in+barley+and+is+located+within+the+complex+of+resistance+genes+containing+Mla%2C+a+powdery+mildew+resistance+gene.&amp;auth=&amp;atitle=RMo1+confers+blast+resistance+in+barley+and+is+located+within+the+complex+of+resistance+genes+containing+Mla%2C+a+powdery+mildew+resistance+gene." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=RMo1+confers+blast+resistance+in+barley+and+is+located+within+the+complex+of+resistance+genes+containing+Mla%2C+a+powdery+mildew+resistance+gene." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22RMo1+confers+blast+resistance+in+barley+and+is+located+within+the+complex+of+resistance+genes+containing+Mla%2C+a+powdery+mildew+resistance+gene.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0000123-Li2" id="pone.0000123-Li2"></a>Li ZK, Arif M, Zhong DB, Fu BY, Xu JL, et al.  (2006) Complex genetic networks underlying the defensive system of rice (<em>Oryza sativa</em> L.) to <em>Xanthomonas oryzae</em> pv. <em>oryzae</em>. PNAS  103: 7994–7999.  <ul class="find" data-citedArticleID="923150" data-doi="10.1073/pnas.0507492103"><li><a href="http://dx.doi.org/10.1073/pnas.0507492103" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Complex+genetic+networks+underlying+the+defensive+system+of+rice+%28Oryza+sativa+L.%29+to+Xanthomonas+oryzae+pv.+oryzae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Complex+genetic+networks+underlying+the+defensive+system+of+rice+%28Oryza+sativa+L.%29+to+Xanthomonas+oryzae+pv.+oryzae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0000123-AlonsoBlanco1" id="pone.0000123-AlonsoBlanco1"></a>Alonso-Blanco C, Koornneef M (2000) Naturally occuring variation in <em>Arabidopsis</em>: an underexploited resource for plant genetics. Trends in Plant Science  5: 22–29.  <ul class="find" data-citedArticleID="923074" data-doi="10.1016/s1360-1385(99)01510-1"><li><a href="http://dx.doi.org/10.1016/s1360-1385(99)01510-1" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Naturally+occuring+variation+in+Arabidopsis%3A+an+underexploited+resource+for+plant+genetics." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Naturally+occuring+variation+in+Arabidopsis%3A+an+underexploited+resource+for+plant+genetics.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0000123-Koornneef1" id="pone.0000123-Koornneef1"></a>Koornneef M, Alonso-Blanco C, Vreugdenhil D (2004) Naturally occurring genetic variation in Arabidopsis thaliana. Annu Rev Plant Biol  55: 141–172.  <ul class="find" data-citedArticleID="923136" data-doi="10.1146/annurev.arplant.55.031903.141605"><li><a href="http://dx.doi.org/10.1146/annurev.arplant.55.031903.141605" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Naturally+occurring+genetic+variation+in+Arabidopsis+thaliana." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Naturally+occurring+genetic+variation+in+Arabidopsis+thaliana.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0000123-Wilson1" id="pone.0000123-Wilson1"></a>Wilson IW, Schiff CL, Hughes DE, Somerville SC (2001) Quantitative Trait Loci Analysis of Powdery Mildew Disease Resistance in the Arabidopsis thaliana Accession Kashmir-1. Genetics  158: 1301–1309.  <ul class="find" data-citedArticleID="923210" data-doi="10.1046/j.1365-3059.2001.00611.x"><li><a href="http://dx.doi.org/10.1046/j.1365-3059.2001.00611.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Quantitative+Trait+Loci+Analysis+of+Powdery+Mildew+Disease+Resistance+in+the+Arabidopsis+thaliana+Accession+Kashmir-1." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Quantitative+Trait+Loci+Analysis+of+Powdery+Mildew+Disease+Resistance+in+the+Arabidopsis+thaliana+Accession+Kashmir-1.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0000123-Godiard1" id="pone.0000123-Godiard1"></a>Godiard L, Sauviac L, Torii KU, Grenon O, Mangin B, et al.  (2003) ERECTA, an LRR receptor-like kinase protein controlling development pleiotropically affects resistance to bacterial wilt. Plant J  36: 353–365.  <ul class="find" data-citedArticleID="923114" data-doi="10.1046/j.1365-313x.2003.01877.x"><li><a href="http://dx.doi.org/10.1046/j.1365-313x.2003.01877.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=ERECTA%2C+an+LRR+receptor-like+kinase+protein+controlling+development+pleiotropically+affects+resistance+to+bacterial+wilt." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22ERECTA%2C+an+LRR+receptor-like+kinase+protein+controlling+development+pleiotropically+affects+resistance+to+bacterial+wilt.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0000123-Denby1" id="pone.0000123-Denby1"></a>Denby KJ, Kumar P, Kliebenstein DJ (2004) Identification of Botrytis cinerea susceptibility loci in Arabidopsis thaliana. Plant J  38: 473–486.  <ul class="find" data-citedArticleID="923096" data-doi="10.1111/j.0960-7412.2004.02059.x"><li><a href="http://dx.doi.org/10.1111/j.0960-7412.2004.02059.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Identification+of+Botrytis+cinerea+susceptibility+loci+in+Arabidopsis+thaliana." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Identification+of+Botrytis+cinerea+susceptibility+loci+in+Arabidopsis+thaliana.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0000123-Kover1" id="pone.0000123-Kover1"></a>Kover PX, Wolf JB, Kunkel BN, Cheverud JM (2005) Genetic architecture of Arabidopsis thaliana response to infection by Pseudomonas syringae. Heredity  94: 507–517.  <ul class="find" data-citedArticleID="923138" data-doi="10.1038/sj.hdy.6800651"><li><a href="http://dx.doi.org/10.1038/sj.hdy.6800651" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genetic+architecture+of+Arabidopsis+thaliana+response+to+infection+by+Pseudomonas+syringae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genetic+architecture+of+Arabidopsis+thaliana+response+to+infection+by+Pseudomonas+syringae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0000123-Llorente1" id="pone.0000123-Llorente1"></a>Llorente F, Alonso-Blanco C, Sanchez-Rodriguez C, Jorda L, Molina A (2005) ERECTA receptor-like kinase and heterotrimeric G protein from Arabidopsis are required for resistance to the necrotrophic fungus Plectosphaerella cucumerina. Plant J  43: 165–180.  <ul class="find" data-citedArticleID="923156" data-doi="10.1111/j.1365-313x.2005.02440.x"><li><a href="http://dx.doi.org/10.1111/j.1365-313x.2005.02440.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=ERECTA+receptor-like+kinase+and+heterotrimeric+G+protein+from+Arabidopsis+are+required+for+resistance+to+the+necrotrophic+fungus+Plectosphaerella+cucumerina." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22ERECTA+receptor-like+kinase+and+heterotrimeric+G+protein+from+Arabidopsis+are+required+for+resistance+to+the+necrotrophic+fungus+Plectosphaerella+cucumerina.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0000123-Quirino1" id="pone.0000123-Quirino1"></a>Quirino BF, Bent AF (2003) Deciphering host resistance and pathogen virulence: the <em>Arabidopsis/Pseudomonas</em> interaction as a model. Molecular Plant Pathology  4: 517–530.  <ul class="find" data-citedArticleID="923192" data-doi="10.1046/j.1364-3703.2003.00198.x"><li><a href="http://dx.doi.org/10.1046/j.1364-3703.2003.00198.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Deciphering+host+resistance+and+pathogen+virulence%3A+the+Arabidopsis%2FPseudomonas+interaction+as+a+model." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Deciphering+host+resistance+and+pathogen+virulence%3A+the+Arabidopsis%2FPseudomonas+interaction+as+a+model.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0000123-Nomura1" id="pone.0000123-Nomura1"></a>Nomura K, Melotto M, He SY (2005) Suppression of host defense in compatible plant-Pseudomonas syringae interactions. Curr Opin Plant Biol  8: 361–368.  <ul class="find" data-citedArticleID="923182" data-doi="10.1016/j.pbi.2005.05.005"><li><a href="http://dx.doi.org/10.1016/j.pbi.2005.05.005" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Suppression+of+host+defense+in+compatible+plant-Pseudomonas+syringae+interactions." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Suppression+of+host+defense+in+compatible+plant-Pseudomonas+syringae+interactions.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0000123-Katagiri1" id="pone.0000123-Katagiri1"></a>Katagiri F, Thilmony R, He SY (2002)  The <em>Arabidopsis thaliana</em>-<em>Pseudomonas syringae</em> interaction. In: Somerville CR, Meyerowitz EM, editors. The Arabidopsis Book. Rockville, MD: American Society of Plant Biologists. pp. 1–35.  <ul class="find-nolinks"></ul></li><li><span class="label">26.
              </span><a name="pone.0000123-Kover2" id="pone.0000123-Kover2"></a>Kover PX, Schaal BA (2002) Genetic variation for disease resistance and tolerance among Arabidopsis thaliana accessions. Proc Natl Acad Sci U S A  99: 11270–11274.  <ul class="find" data-citedArticleID="923140" data-doi="10.1073/pnas.102288999"><li><a href="http://dx.doi.org/10.1073/pnas.102288999" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genetic+variation+for+disease+resistance+and+tolerance+among+Arabidopsis+thaliana+accessions." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genetic+variation+for+disease+resistance+and+tolerance+among+Arabidopsis+thaliana+accessions.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0000123-Parlevliet1" id="pone.0000123-Parlevliet1"></a>Parlevliet JE (1979) Components of resistance that reduce the rate of epidemic development. Annual Review of Phytopathology  17: 203–222.  <ul class="find" data-citedArticleID="923188" data-doi="10.1146/annurev.py.17.090179.001223"><li><a href="http://dx.doi.org/10.1146/annurev.py.17.090179.001223" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Components+of+resistance+that+reduce+the+rate+of+epidemic+development." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Components+of+resistance+that+reduce+the+rate+of+epidemic+development.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0000123-Buell1" id="pone.0000123-Buell1"></a>Buell CR, Sommerville SC (1997) Use of <em>Arabidopsis</em> recombinant inbred lines reveals a monogenic and a novel digenic resistance mechanism to <em>Xanthomonas campestris</em> pv <em>campestris</em>. Plant J  12: 21–29.  <ul class="find" data-citedArticleID="923090" data-doi="10.1046/j.1365-313x.1997.12010021.x"><li><a href="http://dx.doi.org/10.1046/j.1365-313x.1997.12010021.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Use+of+Arabidopsis+recombinant+inbred+lines+reveals+a+monogenic+and+a+novel+digenic+resistance+mechanism+to+Xanthomonas+campestris+pv+campestris." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Use+of+Arabidopsis+recombinant+inbred+lines+reveals+a+monogenic+and+a+novel+digenic+resistance+mechanism+to+Xanthomonas+campestris+pv+campestris.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0000123-Tuinstra1" id="pone.0000123-Tuinstra1"></a>Tuinstra MR, Ejeta G, Goldsbrough PB (1997) Heterogeneous inbred family (HIF) analysis: a method for developping near-isogenic lines that differ at quantitative trait loci. Theor Appl Genet  95: 1005–1011.  <ul class="find" data-citedArticleID="923202" data-doi="10.1007/s001220050654"><li><a href="http://dx.doi.org/10.1007/s001220050654" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Heterogeneous+inbred+family+%28HIF%29+analysis%3A+a+method+for+developping+near-isogenic+lines+that+differ+at+quantitative+trait+loci." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Heterogeneous+inbred+family+%28HIF%29+analysis%3A+a+method+for+developping+near-isogenic+lines+that+differ+at+quantitative+trait+loci.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0000123-McKhann1" id="pone.0000123-McKhann1"></a>McKhann HI, Camilleri C, Berard A, Bataillon T, David JL, et al.  (2004) Nested core collections maximizing genetic diversity in <em>Arabidopsis thaliana</em>. Plant J  38: 193–202.  <ul class="find" data-citedArticleID="923170" data-doi="10.1111/j.1365-313x.2004.02034.x"><li><a href="http://dx.doi.org/10.1111/j.1365-313x.2004.02034.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Nested+core+collections+maximizing+genetic+diversity+in+Arabidopsis+thaliana." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Nested+core+collections+maximizing+genetic+diversity+in+Arabidopsis+thaliana.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0000123-AlonsoBlanco2" id="pone.0000123-AlonsoBlanco2"></a>Alonso-Blanco C, Peeters AJ, Koornneef M, Lister C, Dean C, et al.  (1998) Development of an AFLP based linkage map of Ler, Col and Cvi Arabidopsis thaliana ecotypes and construction of a Ler/Cvi recombinant inbred line population. Plant J  14: 259–271.  <ul class="find" data-citedArticleID="923076" data-doi="10.1046/j.1365-313x.1998.00115.x"><li><a href="http://dx.doi.org/10.1046/j.1365-313x.1998.00115.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Development+of+an+AFLP+based+linkage+map+of+Ler%2C+Col+and+Cvi+Arabidopsis+thaliana+ecotypes+and+construction+of+a+Ler%2FCvi+recombinant+inbred+line+population." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Development+of+an+AFLP+based+linkage+map+of+Ler%2C+Col+and+Cvi+Arabidopsis+thaliana+ecotypes+and+construction+of+a+Ler%2FCvi+recombinant+inbred+line+population.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0000123-AlonsoBlanco3" id="pone.0000123-AlonsoBlanco3"></a>Alonso-Blanco C, Koornneef M, Stam P (1998) The use of recombinant inbred lines (RILs) for genetic mapping. Methods Mol Biol  82: 137–146.  <ul class="find" data-citedArticleID="923078" data-doi="10.1385/0-89603-391-0:137"><li><a href="http://dx.doi.org/10.1385/0-89603-391-0:137" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+use+of+recombinant+inbred+lines+%28RILs%29+for+genetic+mapping." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+use+of+recombinant+inbred+lines+%28RILs%29+for+genetic+mapping.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0000123-AlonsoBlanco4" id="pone.0000123-AlonsoBlanco4"></a>Alonso-Blanco C, El-Assal SE, Coupland G, Koornneef M (1998) Analysis of natural allelic variation at flowering time loci in the Landsberg erecta and Cape Verde Islands ecotypes of Arabidopsis thaliana. Genetics  149: 749–764.  <ul class="find" data-citedArticleID="923080"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Analysis+of+natural+allelic+variation+at+flowering+time+loci+in+the+Landsberg+erecta+and+Cape+Verde+Islands+ecotypes+of+Arabidopsis+thaliana.&amp;auth=&amp;atitle=Analysis+of+natural+allelic+variation+at+flowering+time+loci+in+the+Landsberg+erecta+and+Cape+Verde+Islands+ecotypes+of+Arabidopsis+thaliana." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Analysis+of+natural+allelic+variation+at+flowering+time+loci+in+the+Landsberg+erecta+and+Cape+Verde+Islands+ecotypes+of+Arabidopsis+thaliana." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Analysis+of+natural+allelic+variation+at+flowering+time+loci+in+the+Landsberg+erecta+and+Cape+Verde+Islands+ecotypes+of+Arabidopsis+thaliana.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0000123-Loudet1" id="pone.0000123-Loudet1"></a>Loudet O, Chaillou S, Camilleri C, Bouchez D, Daniel-Vedele F (2002) Bay-0×Shahdara recombinant inbred line population: a powerful tool for the genetic dissection of complex traits in Arabidopsis. Theor Appl Genet  104: 1173–1184.  <ul class="find" data-citedArticleID="923160" data-doi="10.1007/s00122-001-0825-9"><li><a href="http://dx.doi.org/10.1007/s00122-001-0825-9" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Bay-0%26%23xd7%3BShahdara+recombinant+inbred+line+population%3A+a+powerful+tool+for+the+genetic+dissection+of+complex+traits+in+Arabidopsis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Bay-0%26%23xd7%3BShahdara+recombinant+inbred+line+population%3A+a+powerful+tool+for+the+genetic+dissection+of+complex+traits+in+Arabidopsis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0000123-Zeng1" id="pone.0000123-Zeng1"></a>Zeng ZB (1994) Precision mapping of quantitative trait loci. Genetics  136: 1457–1468.  <ul class="find" data-citedArticleID="923214"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Precision+mapping+of+quantitative+trait+loci.&amp;auth=&amp;atitle=Precision+mapping+of+quantitative+trait+loci." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Precision+mapping+of+quantitative+trait+loci." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Precision+mapping+of+quantitative+trait+loci.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">36.
              </span><a name="pone.0000123-Kunkel1" id="pone.0000123-Kunkel1"></a>Kunkel BN, Brooks DM (2002) Cross talk between signaling pathways in pathogen defense. Curr Opin Plant Biol  5: 325–331.  <ul class="find" data-citedArticleID="923144" data-doi="10.1016/s1369-5266(02)00275-3"><li><a href="http://dx.doi.org/10.1016/s1369-5266(02)00275-3" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Cross+talk+between+signaling+pathways+in+pathogen+defense." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Cross+talk+between+signaling+pathways+in+pathogen+defense.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">37.
              </span><a name="pone.0000123-Thomma1" id="pone.0000123-Thomma1"></a>Thomma BP, Penninckx IA, Broekaert WF, Cammue BP (2001) The complexity of disease signaling in Arabidopsis. Curr Opin Immunol  13: 63–68.  <ul class="find" data-citedArticleID="923200" data-doi="10.1016/s0952-7915(00)00183-7"><li><a href="http://dx.doi.org/10.1016/s0952-7915(00)00183-7" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+complexity+of+disease+signaling+in+Arabidopsis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+complexity+of+disease+signaling+in+Arabidopsis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">38.
              </span><a name="pone.0000123-Glazebrook1" id="pone.0000123-Glazebrook1"></a>Glazebrook J, Chen W, Estes B, Chang HS, Nawrath C, et al.  (2003) Topology of the network integrating salicylate and jasmonate signal transduction derived from global expression phenotyping. Plant J  34: 217–228.  <ul class="find" data-citedArticleID="923110" data-doi="10.1046/j.1365-313x.2003.01717.x"><li><a href="http://dx.doi.org/10.1046/j.1365-313x.2003.01717.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Topology+of+the+network+integrating+salicylate+and+jasmonate+signal+transduction+derived+from+global+expression+phenotyping." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Topology+of+the+network+integrating+salicylate+and+jasmonate+signal+transduction+derived+from+global+expression+phenotyping.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">39.
              </span><a name="pone.0000123-Nawrath1" id="pone.0000123-Nawrath1"></a>Nawrath C, Métraux J-P (1999) Salicylic acid induction-deficient mutants of Arabidopsis express <em>PR-2</em> and <em>PR-5</em> and accumulate high levels of camalexin after pathogen inoculation. The Plant Cell  11: 1393–1404.  <ul class="find" data-citedArticleID="923178" data-doi="10.2307/3870970"><li><a href="http://dx.doi.org/10.2307/3870970" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Salicylic+acid+induction-deficient+mutants+of+Arabidopsis+express+PR-2+and+PR-5+and+accumulate+high+levels+of+camalexin+after+pathogen+inoculation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Salicylic+acid+induction-deficient+mutants+of+Arabidopsis+express+PR-2+and+PR-5+and+accumulate+high+levels+of+camalexin+after+pathogen+inoculation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">40.
              </span><a name="pone.0000123-Gu1" id="pone.0000123-Gu1"></a>Gu Y-Q, Wildermuth MC, Chakravarthy S, Loh Y-T, Yang C, et al.  (2002) Tomato Transcription Factors Pti4, Pti5, and Pti6 Activate Defense Responses When Expressed in Arabidopsis. Plant Cell  14: 817–831.  <ul class="find" data-citedArticleID="923116" data-doi="10.1105/tpc.000794"><li><a href="http://dx.doi.org/10.1105/tpc.000794" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Tomato+Transcription+Factors+Pti4%2C+Pti5%2C+and+Pti6+Activate+Defense+Responses+When+Expressed+in+Arabidopsis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Tomato+Transcription+Factors+Pti4%2C+Pti5%2C+and+Pti6+Activate+Defense+Responses+When+Expressed+in+Arabidopsis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">41.
              </span><a name="pone.0000123-Young1" id="pone.0000123-Young1"></a>Young ND (1996) QTL mapping and quantitative disease resistance in plants. Annual Review of Phytopathology  34: 479–501.  <ul class="find" data-citedArticleID="923212" data-doi="10.1146/annurev.phyto.34.1.479"><li><a href="http://dx.doi.org/10.1146/annurev.phyto.34.1.479" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=QTL+mapping+and+quantitative+disease+resistance+in+plants." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22QTL+mapping+and+quantitative+disease+resistance+in+plants.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">42.
              </span><a name="pone.0000123-Bai1" id="pone.0000123-Bai1"></a>Bai Y, Huang CC, van der Hulst R, Meijer-Dekens F, Bonnema G, et al.  (2003) QTLs for tomato powdery mildew resistance (<em>Oidium lycopersici</em>) in <em>Lycopersicon parviflorum</em> G1.1601 co-localize with two qualitative powdery mildew resistance genes. Mol Plant Microbe Interact  16: 169–176.  <ul class="find" data-citedArticleID="923086" data-doi="10.1094/mpmi.2003.16.2.169"><li><a href="http://dx.doi.org/10.1094/mpmi.2003.16.2.169" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=QTLs+for+tomato+powdery+mildew+resistance+%28Oidium+lycopersici%29+in+Lycopersicon+parviflorum+G1.1601+co-localize+with+two+qualitative+powdery+mildew+resistance+genes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22QTLs+for+tomato+powdery+mildew+resistance+%28Oidium+lycopersici%29+in+Lycopersicon+parviflorum+G1.1601+co-localize+with+two+qualitative+powdery+mildew+resistance+genes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">43.
              </span><a name="pone.0000123-Nguyen1" id="pone.0000123-Nguyen1"></a>Nguyen TTT, Koizumi S, La TN, Zenbayashi KS, Ashizawa T, et al.  (2006) Pi35(t), a new gene conferring partial resistance to leaf blast in the rice cultivar Hokkai 188. Theor Appl Genet.   in press.  <ul class="find" data-citedArticleID="923180" data-doi="10.1007/s00122-006-0337-8"><li><a href="http://dx.doi.org/10.1007/s00122-006-0337-8" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Pi35%28t%29%2C+a+new+gene+conferring+partial+resistance+to+leaf+blast+in+the+rice+cultivar+Hokkai+188." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Pi35%28t%29%2C+a+new+gene+conferring+partial+resistance+to+leaf+blast+in+the+rice+cultivar+Hokkai+188.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">44.
              </span><a name="pone.0000123-Jirage1" id="pone.0000123-Jirage1"></a>Jirage D, Tootle TL, Reuber TL, Frost LN, Feys BJ, et al.  (1999) 
                  <em>Arabidopsis thaliana</em> PAD4 encodes a lipase-like gene that is important for salycilic acid signaling. Proceedings of The National Academy of Sciences of USA  96: 13583–13588.  <ul class="find" data-citedArticleID="923124"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Arabidopsis+thaliana+PAD4+encodes+a+lipase-like+gene+that+is+important+for+salycilic+acid+signaling.&amp;auth=&amp;atitle=Arabidopsis+thaliana+PAD4+encodes+a+lipase-like+gene+that+is+important+for+salycilic+acid+signaling." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Arabidopsis+thaliana+PAD4+encodes+a+lipase-like+gene+that+is+important+for+salycilic+acid+signaling." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Arabidopsis+thaliana+PAD4+encodes+a+lipase-like+gene+that+is+important+for+salycilic+acid+signaling.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">45.
              </span><a name="pone.0000123-Falk1" id="pone.0000123-Falk1"></a>Falk A, Feys BJ, Frost LN, Jones JDG, Daniels MJ, et al.  (1999) 
                  <em>EDS1</em>, an essential component of <em>R</em> gene-mediated disease resistance in <em>Arabidopsis</em> has homology to eukaryotic lipases. Proceedings of The National Academy of Sciences of USA  96: 3292–3297.  <ul class="find" data-citedArticleID="923098"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=EDS1%2C+an+essential+component+of+R+gene-mediated+disease+resistance+in+Arabidopsis+has+homology+to+eukaryotic+lipases.&amp;auth=&amp;atitle=EDS1%2C+an+essential+component+of+R+gene-mediated+disease+resistance+in+Arabidopsis+has+homology+to+eukaryotic+lipases." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=EDS1%2C+an+essential+component+of+R+gene-mediated+disease+resistance+in+Arabidopsis+has+homology+to+eukaryotic+lipases." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22EDS1%2C+an+essential+component+of+R+gene-mediated+disease+resistance+in+Arabidopsis+has+homology+to+eukaryotic+lipases.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">46.
              </span><a name="pone.0000123-Feys1" id="pone.0000123-Feys1"></a>Feys BJ, Wiermer M, Bhat RA, Moisan LJ, Medina-Escobar N, et al.  (2005) Arabidopsis SENESCENCE-ASSOCIATED GENE101 stabilizes and signals within an ENHANCED DISEASE SUSCEPTIBILITY1 complex in plant innate immunity. Plant Cell  17: 2601–2613.  <ul class="find" data-citedArticleID="923102" data-doi="10.1105/tpc.105.033910"><li><a href="http://dx.doi.org/10.1105/tpc.105.033910" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Arabidopsis+SENESCENCE-ASSOCIATED+GENE101+stabilizes+and+signals+within+an+ENHANCED+DISEASE+SUSCEPTIBILITY1+complex+in+plant+innate+immunity." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Arabidopsis+SENESCENCE-ASSOCIATED+GENE101+stabilizes+and+signals+within+an+ENHANCED+DISEASE+SUSCEPTIBILITY1+complex+in+plant+innate+immunity.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">47.
              </span><a name="pone.0000123-Tao1" id="pone.0000123-Tao1"></a>Tao Y, Xie Z, Chen W, Glazebrook J, Chang HS, et al.  (2003) Quantitative Nature of Arabidopsis Responses during Compatible and Incompatible Interactions with the Bacterial Pathogen <em>Pseudomonas syringae</em>. Plant Cell  15: 317–330.  <ul class="find" data-citedArticleID="923198" data-doi="10.1105/tpc.007591"><li><a href="http://dx.doi.org/10.1105/tpc.007591" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Quantitative+Nature+of+Arabidopsis+Responses+during+Compatible+and+Incompatible+Interactions+with+the+Bacterial+Pathogen+Pseudomonas+syringae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Quantitative+Nature+of+Arabidopsis+Responses+during+Compatible+and+Incompatible+Interactions+with+the+Bacterial+Pathogen+Pseudomonas+syringae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">48.
              </span><a name="pone.0000123-Lamb1" id="pone.0000123-Lamb1"></a>Lamb CJ, Ryals JA, Ward ER, Dixon RA (1992) Emerging strategies for enhancing crop resistance to microbial pathogens. Biotechnology (N Y)  10: 1436–1445.  <ul class="find" data-citedArticleID="923146" data-doi="10.1038/nbt1192-1436"><li><a href="http://dx.doi.org/10.1038/nbt1192-1436" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Emerging+strategies+for+enhancing+crop+resistance+to+microbial+pathogens." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Emerging+strategies+for+enhancing+crop+resistance+to+microbial+pathogens.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">49.
              </span><a name="pone.0000123-Abramovitch1" id="pone.0000123-Abramovitch1"></a>Abramovitch RB, Anderson JC, Martin GB (2006) Bacterial elicitation and evasion of plant innate immunity. Nat Rev Mol Cell Biol  7: 601–611.  <ul class="find" data-citedArticleID="923070" data-doi="10.1038/nrm1984"><li><a href="http://dx.doi.org/10.1038/nrm1984" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Bacterial+elicitation+and+evasion+of+plant+innate+immunity." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Bacterial+elicitation+and+evasion+of+plant+innate+immunity.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">50.
              </span><a name="pone.0000123-Chisholm1" id="pone.0000123-Chisholm1"></a>Chisholm ST, Coaker G, Day B, Staskawicz BJ (2006) Host-microbe interactions: shaping the evolution of the plant immune response. Cell  124: 803–814.  <ul class="find" data-citedArticleID="923092" data-doi="10.1016/j.cell.2006.02.008"><li><a href="http://dx.doi.org/10.1016/j.cell.2006.02.008" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Host-microbe+interactions%3A+shaping+the+evolution+of+the+plant+immune+response." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Host-microbe+interactions%3A+shaping+the+evolution+of+the+plant+immune+response.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">51.
              </span><a name="pone.0000123-deTorres1" id="pone.0000123-deTorres1"></a>de Torres M, Mansfield JW, Grabov N, Brown IR, Ammouneh H, et al.  (2006) Pseudomonas syringae effector AvrPtoB suppresses basal defence in Arabidopsis. The Plant Journal  47: 368–382.  <ul class="find" data-citedArticleID="923218" data-doi="10.1111/j.1365-313x.2006.02798.x"><li><a href="http://dx.doi.org/10.1111/j.1365-313x.2006.02798.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Pseudomonas+syringae+effector+AvrPtoB+suppresses+basal+defence+in+Arabidopsis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Pseudomonas+syringae+effector+AvrPtoB+suppresses+basal+defence+in+Arabidopsis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">52.
              </span><a name="pone.0000123-Foulongne1" id="pone.0000123-Foulongne1"></a>Foulongne M, Pascal T, Pfeiffer F, Kervella J (2003) QTLs for powdery mildew resistance in peach×<em>Prunus davidiana</em> crosses: consistency across generations and environments. Molecular Breeding  12: 33–50.  <ul class="find" data-citedArticleID="923104"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=QTLs+for+powdery+mildew+resistance+in+peach%26%23xd7%3BPrunus+davidiana+crosses%3A+consistency+across+generations+and+environments.&amp;auth=&amp;atitle=QTLs+for+powdery+mildew+resistance+in+peach%26%23xd7%3BPrunus+davidiana+crosses%3A+consistency+across+generations+and+environments." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=QTLs+for+powdery+mildew+resistance+in+peach%26%23xd7%3BPrunus+davidiana+crosses%3A+consistency+across+generations+and+environments." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22QTLs+for+powdery+mildew+resistance+in+peach%26%23xd7%3BPrunus+davidiana+crosses%3A+consistency+across+generations+and+environments.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">53.
              </span><a name="pone.0000123-Arahana1" id="pone.0000123-Arahana1"></a>Arahana VS, Graef GL, Specht JE, Steadman JR, Eskridge KM (2001) Identification of QTLs for resistance to <em>Sclerotinia sclerotiorum</em> in soybean. Crop Science  41: 180–188.  <ul class="find" data-citedArticleID="923082" data-doi="10.2135/cropsci2001.411180x"><li><a href="http://dx.doi.org/10.2135/cropsci2001.411180x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Identification+of+QTLs+for+resistance+to+Sclerotinia+sclerotiorum+in+soybean." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Identification+of+QTLs+for+resistance+to+Sclerotinia+sclerotiorum+in+soybean.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">54.
              </span><a name="pone.0000123-AlChaarani1" id="pone.0000123-AlChaarani1"></a>Al-Chaarani GR, Roustaee A, Gentzbittel L, Mokrani L, Barrault G, et al.  (2002) A QTL analysis of sunflower partial resistance to downy mildew (<em>Plasmopara halstedii</em>) and black stem (<em>Phoma macdonaldii</em>) by the use of recombinant inbred lines (RILs). Theoretical and Applied Genetics  104: 490–496.  <ul class="find" data-citedArticleID="923072" data-doi="10.1007/s001220100742"><li><a href="http://dx.doi.org/10.1007/s001220100742" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+QTL+analysis+of+sunflower+partial+resistance+to+downy+mildew+%28Plasmopara+halstedii%29+and+black+stem+%28Phoma+macdonaldii%29+by+the+use+of+recombinant+inbred+lines+%28RILs%29." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+QTL+analysis+of+sunflower+partial+resistance+to+downy+mildew+%28Plasmopara+halstedii%29+and+black+stem+%28Phoma+macdonaldii%29+by+the+use+of+recombinant+inbred+lines+%28RILs%29.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">55.
              </span><a name="pone.0000123-Perchepied1" id="pone.0000123-Perchepied1"></a>Perchepied L, Dogimont C, Pitrat M (2005) Strain-specific and recessive QTLs involved in the control of partial resistance to Fusarium oxysporum f. sp. melonis race 1.2 in a recombinant inbred line population of melon. Theor Appl Genet  111: 65–74.  <ul class="find" data-citedArticleID="923190" data-doi="10.1007/s00122-005-1991-y"><li><a href="http://dx.doi.org/10.1007/s00122-005-1991-y" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Strain-specific+and+recessive+QTLs+involved+in+the+control+of+partial+resistance+to+Fusarium+oxysporum+f.+sp.+melonis+race+1.2+in+a+recombinant+inbred+line+population+of+melon." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Strain-specific+and+recessive+QTLs+involved+in+the+control+of+partial+resistance+to+Fusarium+oxysporum+f.+sp.+melonis+race+1.2+in+a+recombinant+inbred+line+population+of+melon.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">56.
              </span><a name="pone.0000123-Kroymann1" id="pone.0000123-Kroymann1"></a>Kroymann J, Donnerhacke S, Schnabelrauch D, Mitchell-Olds T (2003) Evolutionary dynamics of an Arabidopsis insect resistance quantitative trait locus. Proc Natl Acad Sci U S A  100: Suppl 214587–14592.  <ul class="find" data-citedArticleID="923142" data-doi="10.1073/pnas.1734046100"><li><a href="http://dx.doi.org/10.1073/pnas.1734046100" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evolutionary+dynamics+of+an+Arabidopsis+insect+resistance+quantitative+trait+locus." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evolutionary+dynamics+of+an+Arabidopsis+insect+resistance+quantitative+trait+locus.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">57.
              </span><a name="pone.0000123-Mackay1" id="pone.0000123-Mackay1"></a>Mackay TF (2004) The genetic architecture of quantitative traits: lessons from Drosophila. Curr Opin Genet Dev  14: 253–257.  <ul class="find" data-citedArticleID="923164" data-doi="10.1016/j.gde.2004.04.003"><li><a href="http://dx.doi.org/10.1016/j.gde.2004.04.003" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+genetic+architecture+of+quantitative+traits%3A+lessons+from+Drosophila." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+genetic+architecture+of+quantitative+traits%3A+lessons+from+Drosophila.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">58.
              </span><a name="pone.0000123-MitchellOlds1" id="pone.0000123-MitchellOlds1"></a>Mitchell-Olds T, Schmitt J (2006) Genetic mechanisms and evolutionary significance of natural variation in Arabidopsis. Nature  441: 947–952.  <ul class="find" data-citedArticleID="923174" data-doi="10.1038/nature04878"><li><a href="http://dx.doi.org/10.1038/nature04878" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genetic+mechanisms+and+evolutionary+significance+of+natural+variation+in+Arabidopsis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genetic+mechanisms+and+evolutionary+significance+of+natural+variation+in+Arabidopsis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">59.
              </span><a name="pone.0000123-Liao1" id="pone.0000123-Liao1"></a>Liao CY, Wu P, Hu B, Yi KK (2001) Effects of genetic background and environment on QTLs and epistasis for rice (<em>Oryza sativa</em> L.) panicle number. Theoretical and Applied Genetics  103: 104–111.  <ul class="find" data-citedArticleID="923152" data-doi="10.1007/s001220000528"><li><a href="http://dx.doi.org/10.1007/s001220000528" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Effects+of+genetic+background+and+environment+on+QTLs+and+epistasis+for+rice+%28Oryza+sativa+L.%29+panicle+number." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Effects+of+genetic+background+and+environment+on+QTLs+and+epistasis+for+rice+%28Oryza+sativa+L.%29+panicle+number.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">60.
              </span><a name="pone.0000123-Mauricio1" id="pone.0000123-Mauricio1"></a>Mauricio R (2005) Ontogenetics of QTL: the genetic architecture of trichome density over time in Arabidopsis thaliana. Genetica  123: 75–85.  <ul class="find" data-citedArticleID="923168" data-doi="10.1007/s10709-002-2714-9"><li><a href="http://dx.doi.org/10.1007/s10709-002-2714-9" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Ontogenetics+of+QTL%3A+the+genetic+architecture+of+trichome+density+over+time+in+Arabidopsis+thaliana." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Ontogenetics+of+QTL%3A+the+genetic+architecture+of+trichome+density+over+time+in+Arabidopsis+thaliana.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">61.
              </span><a name="pone.0000123-Gebhardt1" id="pone.0000123-Gebhardt1"></a>Gebhardt C, Valkonen JPT (2001) Organization of genes controlling disease resistance in the potato genome. Annual Review of Phytopathology  39: 79–102.  <ul class="find" data-citedArticleID="923108"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Organization+of+genes+controlling+disease+resistance+in+the+potato+genome.&amp;auth=&amp;atitle=Organization+of+genes+controlling+disease+resistance+in+the+potato+genome." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Organization+of+genes+controlling+disease+resistance+in+the+potato+genome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Organization+of+genes+controlling+disease+resistance+in+the+potato+genome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">62.
              </span><a name="pone.0000123-Jones1" id="pone.0000123-Jones1"></a>Jones JD (2001) Putting knowledge of plant disease resistance genes to work. Curr Opin Plant Biol  4: 281–287.  <ul class="find" data-citedArticleID="923126" data-doi="10.1016/s1369-5266(00)00174-6"><li><a href="http://dx.doi.org/10.1016/s1369-5266(00)00174-6" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Putting+knowledge+of+plant+disease+resistance+genes+to+work." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Putting+knowledge+of+plant+disease+resistance+genes+to+work.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">63.
              </span><a name="pone.0000123-Robertson1" id="pone.0000123-Robertson1"></a>Robertson A (1989)  Understanding the relationship between qualitative and quantitative genetics. In: Helentjaris T, Burr B, editors. Development and application of molecular markers to problems in plant genetics. New York, USA: Cold Spring Harbour Lab. Press. pp. 81–88.  <ul class="find-nolinks"></ul></li><li><span class="label">64.
              </span><a name="pone.0000123-Wang1" id="pone.0000123-Wang1"></a>Wang GL, Ruan DL, Song WY, Sideris S, Chen L, et al.  (1998) Xa21D encodes a receptor-like molecule with a leucine-rich repeat domain that determines race-specific recognition and is subject to adaptive evolution. Plant Cell  10: 765–779.  <ul class="find" data-citedArticleID="923204" data-doi="10.2307/3870663"><li><a href="http://dx.doi.org/10.2307/3870663" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Xa21D+encodes+a+receptor-like+molecule+with+a+leucine-rich+repeat+domain+that+determines+race-specific+recognition+and+is+subject+to+adaptive+evolution." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Xa21D+encodes+a+receptor-like+molecule+with+a+leucine-rich+repeat+domain+that+determines+race-specific+recognition+and+is+subject+to+adaptive+evolution.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">65.
              </span><a name="pone.0000123-Williams1" id="pone.0000123-Williams1"></a>Williams KJ (2003) The molecular genetics of disease resistance in barley. Aust J Agric Res  54: 1065–1079.  <ul class="find" data-citedArticleID="923208"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=The+molecular+genetics+of+disease+resistance+in+barley.&amp;auth=&amp;atitle=The+molecular+genetics+of+disease+resistance+in+barley." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+molecular+genetics+of+disease+resistance+in+barley." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+molecular+genetics+of+disease+resistance+in+barley.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">66.
              </span><a name="pone.0000123-Meyers1" id="pone.0000123-Meyers1"></a>Meyers BC, Kozik A, Griego A, Kuang H, Michelmore RW (2003) Genome-wide analysis of NBS-LRR-encoding genes in Arabidopsis. Plant Cell  15: 809–834.  <ul class="find" data-citedArticleID="923172" data-doi="10.1105/tpc.009308"><li><a href="http://dx.doi.org/10.1105/tpc.009308" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genome-wide+analysis+of+NBS-LRR-encoding+genes+in+Arabidopsis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genome-wide+analysis+of+NBS-LRR-encoding+genes+in+Arabidopsis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">67.
              </span><a name="pone.0000123-HammondKosack1" id="pone.0000123-HammondKosack1"></a>Hammond-Kosack KE, Parker JE (2003) Deciphering plant-pathogen communication: fresh perspectives for molecular resistance breeding. Curr Opin Biotechnol 177–193.  <ul class="find" data-citedArticleID="923118" data-doi="10.1016/s0958-1669(03)00035-1"><li><a href="http://dx.doi.org/10.1016/s0958-1669(03)00035-1" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Deciphering+plant-pathogen+communication%3A+fresh+perspectives+for+molecular+resistance+breeding." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Deciphering+plant-pathogen+communication%3A+fresh+perspectives+for+molecular+resistance+breeding.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">68.
              </span><a name="pone.0000123-Mouille1" id="pone.0000123-Mouille1"></a>Mouille G, Witucka-Wall H, Bruyant MP, Loudet O, Pelletier S, et al.  (2006) Quantitative trait Loci analysis of primary cell wall composition in Arabidopsis. Plant Physiol  141: 1035–1044.  <ul class="find" data-citedArticleID="923176" data-doi="10.1104/pp.106.079384"><li><a href="http://dx.doi.org/10.1104/pp.106.079384" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Quantitative+trait+Loci+analysis+of+primary+cell+wall+composition+in+Arabidopsis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Quantitative+trait+Loci+analysis+of+primary+cell+wall+composition+in+Arabidopsis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">69.
              </span><a name="pone.0000123-Glazebrook2" id="pone.0000123-Glazebrook2"></a>Glazebrook J, Rogers EE, Ausubel FM (1997) Use of Arabidopsis for genetic dissection of plant defense responses. Annual Review of Genetic  31: 547–569.  <ul class="find" data-citedArticleID="923112" data-doi="10.1146/annurev.genet.31.1.547"><li><a href="http://dx.doi.org/10.1146/annurev.genet.31.1.547" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Use+of+Arabidopsis+for+genetic+dissection+of+plant+defense+responses." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Use+of+Arabidopsis+for+genetic+dissection+of+plant+defense+responses.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">70.
              </span><a name="pone.0000123-Bartsch1" id="pone.0000123-Bartsch1"></a>Bartsch M, Gobbato E, Bednarek P, Debey S, Schultze JL, et al.  (2006) Salicylic acid-independent ENHANCED DISEASE SUSCEPTIBILITY1 signaling in Arabidopsis immunity and cell death is regulated by the monooxygenase FMO1 and the Nudix hydrolase NUDT7. Plant Cell  18: 1038–1051.  <ul class="find" data-citedArticleID="923088" data-doi="10.1105/tpc.105.039982"><li><a href="http://dx.doi.org/10.1105/tpc.105.039982" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Salicylic+acid-independent+ENHANCED+DISEASE+SUSCEPTIBILITY1+signaling+in+Arabidopsis+immunity+and+cell+death+is+regulated+by+the+monooxygenase+FMO1+and+the+Nudix+hydrolase+NUDT7." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Salicylic+acid-independent+ENHANCED+DISEASE+SUSCEPTIBILITY1+signaling+in+Arabidopsis+immunity+and+cell+death+is+regulated+by+the+monooxygenase+FMO1+and+the+Nudix+hydrolase+NUDT7.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">71.
              </span><a name="pone.0000123-Kliebenstein1" id="pone.0000123-Kliebenstein1"></a>Kliebenstein DJ, West MA, van Leeuwen H, Loudet O, Doerge RW, et al.  (2006) Identification of QTLs controlling gene expression networks defined a priori. BMC Bioinformatics  7: 308.  <ul class="find" data-citedArticleID="923134" data-doi="10.1186/1471-2105-7-308"><li><a href="http://dx.doi.org/10.1186/1471-2105-7-308" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Identification+of+QTLs+controlling+gene+expression+networks+defined+a+priori." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Identification+of+QTLs+controlling+gene+expression+networks+defined+a+priori.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">72.
              </span><a name="pone.0000123-Loudet2" id="pone.0000123-Loudet2"></a>Loudet O, Gaudon V, Trubuil A, Daniel-Vedele F (2005) Quantitative trait loci controlling root growth and architecture in Arabidopsis thaliana confirmed by heterogeneous inbred family. Theor Appl Genet  110: 742–753.  <ul class="find" data-citedArticleID="923162" data-doi="10.1007/s00122-004-1900-9"><li><a href="http://dx.doi.org/10.1007/s00122-004-1900-9" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Quantitative+trait+loci+controlling+root+growth+and+architecture+in+Arabidopsis+thaliana+confirmed+by+heterogeneous+inbred+family." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Quantitative+trait+loci+controlling+root+growth+and+architecture+in+Arabidopsis+thaliana+confirmed+by+heterogeneous+inbred+family.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">73.
              </span><a name="pone.0000123-Whalen1" id="pone.0000123-Whalen1"></a>Whalen MC, Innes RW, Bent AF, Staskawicz BJ (1991) Identification of <em>Pseudomonas syringae</em> pathogens of <em>Arabidopsis</em> and a bacterial locus determining avirulence on both <em>Arabidopsis</em> and soybean. the plant Cell  3: 49–59.  <ul class="find" data-citedArticleID="923206" data-doi="10.2307/3869199"><li><a href="http://dx.doi.org/10.2307/3869199" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Identification+of+Pseudomonas+syringae+pathogens+of+Arabidopsis+and+a+bacterial+locus+determining+avirulence+on+both+Arabidopsis+and+soybean." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Identification+of+Pseudomonas+syringae+pathogens+of+Arabidopsis+and+a+bacterial+locus+determining+avirulence+on+both+Arabidopsis+and+soybean.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">74.
              </span><a name="pone.0000123-Lorrain1" id="pone.0000123-Lorrain1"></a>Lorrain S, Lin B, Auriac MC, Kroj T, Saindrenan P, et al.  (2004) VASCULAR ASSOCIATED DEATH1, a novel GRAM domain-containing protein, is a regulator of cell death and defense responses in vascular tissues. The Plant Cell  16: 2217–2232.  <ul class="find" data-citedArticleID="923158" data-doi="10.1105/tpc.104.022038"><li><a href="http://dx.doi.org/10.1105/tpc.104.022038" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=VASCULAR+ASSOCIATED+DEATH1%2C+a+novel+GRAM+domain-containing+protein%2C+is+a+regulator+of+cell+death+and+defense+responses+in+vascular+tissues." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22VASCULAR+ASSOCIATED+DEATH1%2C+a+novel+GRAM+domain-containing+protein%2C+is+a+regulator+of+cell+death+and+defense+responses+in+vascular+tissues.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">75.
              </span><a name="pone.0000123-Gallais1" id="pone.0000123-Gallais1"></a>Gallais A (1990) Théorie de la sélection en amélioration des plantes.  Gallais A, Bannerot H, editors. Paris, France: Masson.   <ul class="find-nolinks"></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000123.XML" value="181507"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000123.PDF" value="287178"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000123.g001.PNG_L" value="15214"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000123.g001.PNG_M" value="15578"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000123.g001.PNG_S" value="1794"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000123.g001.TIF" value="47540"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000123.g001.PNG_I" value="7740"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000123.t001.PNG_L" value="67002"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000123.t001.PNG_M" value="27934"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000123.t001.PNG_S" value="2380"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000123.t001.TIF" value="2963986"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000123.t001.PNG_I" value="12337"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000123.g002.PNG_L" value="10978"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000123.g002.PNG_M" value="12657"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000123.g002.PNG_S" value="1833"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000123.g002.TIF" value="34656"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000123.g002.PNG_I" value="6437"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000123.g003.PNG_L" value="424524"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000123.g003.PNG_M" value="62365"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000123.g003.PNG_S" value="2814"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000123.g003.TIF" value="772350"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000123.g003.PNG_I" value="27597"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000123.t002.PNG_L" value="170947"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000123.t002.PNG_M" value="24213"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000123.t002.PNG_S" value="2369"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000123.t002.TIF" value="13486072"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000123.t002.PNG_I" value="10395"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000123.g004.PNG_L" value="431395"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000123.g004.PNG_M" value="18368"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000123.g004.PNG_S" value="3261"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000123.g004.TIF" value="680044"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000123.g004.PNG_I" value="28349"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000123.g005.PNG_L" value="264330"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000123.g005.PNG_M" value="16671"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000123.g005.PNG_S" value="2650"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000123.g005.TIF" value="469166"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000123.g005.PNG_I" value="20478"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000123.e001.PNG" value="1758"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000123.e001.TIF" value="9208"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000123.s001.EPS" value="618803"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000123.s002.DOC" value="36864"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000123&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000123" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000123&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0000123&volume=&issue=&title=Natural Variation in Partial Resistance to Pseudomonas syringae Is Controlled by Two Major QTLs in Arabidopsis thaliana&author_name=Laure%20Perchepied%2C%20Thomas%20Kroj%2C%20Maurice%20Tronchet%2C%20Olivier%20Loudet%2C%20Dominique%20Roby&start_page=1&end_page=10" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000123" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000123" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0000123" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000123&amp;t=Natural%20Variation%20in%20Partial%20Resistance%20to%20Pseudomonas%20syringae%20Is%20Controlled%20by%20Two%20Major%20QTLs%20in%20Arabidopsis%20thaliana" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000123&title=Natural%20Variation%20in%20Partial%20Resistance%20to%20Pseudomonas%20syringae%20Is%20Controlled%20by%20Two%20Major%20QTLs%20in%20Arabidopsis%20thaliana&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000123&amp;title=Natural%20Variation%20in%20Partial%20Resistance%20to%20Pseudomonas%20syringae%20Is%20Controlled%20by%20Two%20Major%20QTLs%20in%20Arabidopsis%20thaliana" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000123" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0000123&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Natural Variation in Partial Resistance to Pseudomonas syringae Is Controlled by Two Major QTLs in Arabidopsis thaliana';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0000123';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Natural%20Variation%20in%20Partial%20Resistance%20to%20Pseudomonas%20syringae%20Is%20Controlled%20by%20Two%20Major%20QTLs%20in%20Arabidopsis%20thaliana http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000123" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0000123" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">














          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Arabidopsis+thaliana%22" title="Search for articles in the subject area:'Arabidopsis thaliana'"><div class="flagText">Arabidopsis thaliana</div></a>
              <div data-categoryid="32845" data-articleid="23084"
                   data-categoryname="Arabidopsis thaliana"
                   class="flagImage" title="Flag 'Arabidopsis thaliana' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Bacterial+growth%22" title="Search for articles in the subject area:'Bacterial growth'"><div class="flagText">Bacterial growth</div></a>
              <div data-categoryid="46657" data-articleid="23084"
                   data-categoryname="Bacterial growth"
                   class="flagImage" title="Flag 'Bacterial growth' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Gene+expression%22" title="Search for articles in the subject area:'Gene expression'"><div class="flagText">Gene expression</div></a>
              <div data-categoryid="17515" data-articleid="23084"
                   data-categoryname="Gene expression"
                   class="flagImage" title="Flag 'Gene expression' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Genetic+loci%22" title="Search for articles in the subject area:'Genetic loci'"><div class="flagText">Genetic loci</div></a>
              <div data-categoryid="17713" data-articleid="23084"
                   data-categoryname="Genetic loci"
                   class="flagImage" title="Flag 'Genetic loci' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Phenotypes%22" title="Search for articles in the subject area:'Phenotypes'"><div class="flagText">Phenotypes</div></a>
              <div data-categoryid="40235" data-articleid="23084"
                   data-categoryname="Phenotypes"
                   class="flagImage" title="Flag 'Phenotypes' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Plant+genetics%22" title="Search for articles in the subject area:'Plant genetics'"><div class="flagText">Plant genetics</div></a>
              <div data-categoryid="17433" data-articleid="23084"
                   data-categoryname="Plant genetics"
                   class="flagImage" title="Flag 'Plant genetics' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Plant+pathogens%22" title="Search for articles in the subject area:'Plant pathogens'"><div class="flagText">Plant pathogens</div></a>
              <div data-categoryid="34897" data-articleid="23084"
                   data-categoryname="Plant pathogens"
                   class="flagImage" title="Flag 'Plant pathogens' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Pseudomonas+syringae%22" title="Search for articles in the subject area:'Pseudomonas syringae'"><div class="flagText">Pseudomonas syringae</div></a>
              <div data-categoryid="43367" data-articleid="23084"
                   data-categoryname="Pseudomonas syringae"
                   class="flagImage" title="Flag 'Pseudomonas syringae' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=9322'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=5043'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=3154&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>

<div class="block sidebar-comments">
    <div class="header">
        <h3>Comments</h3>
    </div>
      <p><a href="/annotation/listThread.action?root=44789">Well done</a><br>Posted by Damien</p>
</div>

</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
