

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Determining Physical Constraints in Transcriptional Initiation Complexes Using DNA Sequence Analysis</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0001199"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0001199" />

    <meta name="citation_title" content="Determining Physical Constraints in Transcriptional Initiation Complexes Using DNA Sequence Analysis"/>
    <meta itemprop="name" content="Determining Physical Constraints in Transcriptional Initiation Complexes Using DNA Sequence Analysis"/>

      <meta name="citation_author" content="Ryan K. Shultzaberger"/>
            <meta name="citation_author_institution" content="Department of Molecular and Cell Biology, University of California, Berkeley, United States of America"/>
      <meta name="citation_author" content="Derek Y. Chiang"/>
            <meta name="citation_author_institution" content="Department of Molecular and Cell Biology, University of California, Berkeley, United States of America"/>
      <meta name="citation_author" content="Alan M. Moses"/>
            <meta name="citation_author_institution" content="Graduate Group in Biophysics, University of California, Berkeley, United States of America"/>
      <meta name="citation_author" content="Michael B. Eisen"/>
            <meta name="citation_author_institution" content="Department of Molecular and Cell Biology, University of California, Berkeley, United States of America"/>
            <meta name="citation_author_institution" content="Department of Genome Sciences, Genomic Division, Ernest Orlando Lawrence Berkeley National Lab, Berkeley, California, United States of America"/>

    <meta name="citation_date" content="2007/11/21"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0001199.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e1199"/>
    <meta name="citation_issue" content="11"/>
    <meta name="citation_volume" content="2"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Transcription regulation and animal diversity.; citation_author=M Levine; citation_author=R Tjian; citation_journal_title=Nature; citation_volume=424; citation_number=1; citation_pages=147-151; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Transcription of eukaryotic protein-coding genes.; citation_author=TI Lee; citation_author=RA Young; citation_journal_title=Annu Rev Genet; citation_volume=34; citation_number=2; citation_pages=77-137; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Transcriptional regulatory networks in Saccharomyces cerevisiae.; citation_author=TI Lee; citation_author=NJ Rinaldi; citation_author=F Robert; citation_author=DT Odom; citation_author=Z Bar-Joseph; citation_journal_title=Science; citation_volume=298; citation_number=3; citation_pages=799-804; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Multiple transcriptional activation complexes tether the yeast activator Met4 to DNA.; citation_author=PL Blaiseau; citation_author=D Thomas; citation_journal_title=EMBO J; citation_volume=17; citation_number=4; citation_pages=6327-6336; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Metabolism of sulfur amino acids in Saccharomyces cerevisiae.; citation_author=D Thomas; citation_author=Y Surdin-Kerjan; citation_journal_title=Microbiol Mol Biol Rev; citation_volume=61; citation_number=5; citation_pages=503-532; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Phylogenetically and spatially conserved word pairs associated with gene-expression changes in yeasts.; citation_author=DY Chiang; citation_author=AM Moses; citation_author=M Kellis; citation_author=ES Lander; citation_author=MB Eisen; citation_journal_title=Genome Biol; citation_volume=4; citation_number=6; citation_pages=R43; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Correlation between binding rate constants and individual information of E. coli Fis binding sites.; citation_author=RK Shultzaberger; citation_author=LR Roberts; citation_author=IG Lyakhov; citation_author=IA Sidorov; citation_author=AG Stephen; citation_journal_title=Nucleic Acids Res; citation_volume=35; citation_number=7; citation_pages=5275-5283; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Quantitative prediction of NF-κB DNA-protein interactions.; citation_author=IA Udalova; citation_author=R Mott; citation_author=D Field; citation_author=D Kwiatkowski; citation_journal_title=Proc Natl Acad Sci USA; citation_volume=99; citation_number=8; citation_pages=8167-8172; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Anatomy of Escherichia coli Ribosome Binding Sites.; citation_author=RK Shultzaberger; citation_author=RE Bucheimer; citation_author=KE Rudd; citation_author=TD Schneider; citation_journal_title=J Mol Biol; citation_volume=313; citation_number=9; citation_pages=215-228; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Anatomy of Escherichia coli σ70 promoters.; citation_author=RK Shultzaberger; citation_author=Z Chen; citation_author=KA Lewis; citation_author=TD Schneider; citation_journal_title=Nucleic Acids Res; citation_volume=35; citation_number=10; citation_pages=771-788; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Determination of the optimal aligned spacing between the Shine-Dalgarno sequence and the translation initiation codon of Escherichia coli mRNAs.; citation_author=H Chen; citation_author=M Bjerknes; citation_author=R Kumar; citation_author=E Jay; citation_journal_title=Nucleic Acids Res; citation_volume=22; citation_number=11; citation_pages=4953-4957; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=Contacts between 16S ribosomal RNA and mRNA, within the spacer region separating the AUG initiator codon and the Shine-Dalgarno sequence; a site-directed cross-linking study.; citation_author=J Rinke-Appel; citation_author=N Junke; citation_author=R Brimacombe; citation_author=I Lavrik; citation_author=S Dokudovskaya; citation_journal_title=Nucleic Acids Res; citation_volume=22; citation_number=12; citation_pages=3018-3025; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=Compilation and analysis of Escherichia coli promoter DNA sequences.; citation_author=DK Hawley; citation_author=WR McClure; citation_journal_title=Nucleic Acids Res; citation_volume=11; citation_number=13; citation_pages=2237-2255; citation_date=1983; " />
      <meta name="citation_reference" content="citation_title=Mechanism and control of transcription initiation in prokaryotes.; citation_author=WR McClure; citation_journal_title=Annu Rev Biochem; citation_volume=54; citation_number=14; citation_pages=171-204; citation_date=1985; " />
      <meta name="citation_reference" content="citation_title=MET4, a leucine zipper protein, and centromere-binding factor 1 are both required for transcriptional activation of sulfur metabolism in Saccharomyces cerevisiae.; citation_author=D Thomas; citation_author=I Jacquemin; citation_author=Y Surdin-Kerjan; citation_journal_title=Mol Cell Biol; citation_volume=12; citation_number=15; citation_pages=1719-1727; citation_date=1992; " />
      <meta name="citation_reference" content="citation_title=Assembly of a bZIP-bHLH transcription activation complex: formation of the yeast Cbf1-Met4-Met28 complex is regulated through Met28 stimulation of Cbf1 DNA binding.; citation_author=L Kuras; citation_author=R Barbey; citation_author=D Thomas; citation_journal_title=EMBO J; citation_volume=16; citation_number=16; citation_pages=2441-2451; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Met31p and Met32p, two related zinc finger proteins, are involved in transcriptional regulation of yeast sulfur amino acid metabolism.; citation_author=PL Blaiseau; citation_author=AD Isnard; citation_author=Y Surdin-Kerjan; citation_author=D Thomas; citation_journal_title=Mol Cell Biol; citation_volume=17; citation_number=17; citation_pages=3640-3648; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Determination of the binding constants of the centromere protein Cbf1 to all 16 centromere DNAs of Saccharomyces cerevisiae.; citation_author=G Wieland; citation_author=P Hemmerich; citation_author=M Koch; citation_author=T Stoyan; citation_author=J Hegemann; citation_journal_title=Nucleic Acids Res; citation_volume=29; citation_number=18; citation_pages=1054-1060; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Sequence logos: A new way to display consensus sequences.; citation_author=TD Schneider; citation_author=RM Stephens; citation_journal_title=Nucleic Acids Res; citation_volume=18; citation_number=19; citation_pages=6097-6100; citation_date=1990; " />
      <meta name="citation_reference" content="citation_title=Information content of individual genetic sequences.; citation_author=TD Schneider; citation_journal_title=J Theor Biol; citation_volume=189; citation_number=20; citation_pages=427-441; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Reading of DNA sequence logos: Prediction of major groove binding by information theory.; citation_author=TD Schneider; citation_journal_title=Meth Enzym; citation_volume=274; citation_number=21; citation_pages=445-455; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=The value of prior knowledge in discovering motifs with MEME.; citation_author=TL Bailey; citation_author=C Elkan; citation_journal_title=Proc Int Conf Intell Syst Mol Biol; citation_volume=3; citation_number=22; citation_pages=21-29; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Thermostatics and Thermodynamics.; citation_author=M Tribus; citation_number=23; citation_date=1961; citation_publisher=D. van Nostrand Company, Inc; " />
      <meta name="citation_reference" content="citation_title=Information content of binding sites on nucleotide sequences.; citation_author=TD Schneider; citation_author=GD Stormo; citation_author=L Gold; citation_author=A Ehrenfeucht; citation_journal_title=J Mol Biol; citation_volume=188; citation_number=24; citation_pages=415-431; citation_date=1986; " />
      <meta name="citation_reference" content="citation_title=Theory of molecular machines. II. Energy dissipation from molecular machines.; citation_author=TD Schneider; citation_journal_title=J Theor Biol; citation_volume=148; citation_number=25; citation_pages=125-137; citation_date=1991; " />
      <meta name="citation_reference" content="citation_title=Genomic expression programs in the response of yeast cells to environmental changes.; citation_author=AP Gasch; citation_author=PT Spellman; citation_author=CM Kao; citation_author=O Carmel-Harel; citation_author=MB Eisen; citation_journal_title=Mol Biol Cell; citation_volume=11; citation_number=26; citation_pages=4241-4257; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Sulfur sparing in the yeast proteome in response to sulfur demand.; citation_author=M Fauchon; citation_author=G Lagniel; citation_author=JC Aude; citation_author=L Lombardia; citation_author=P Soularue; citation_journal_title=Mol Cell; citation_volume=9; citation_number=27; citation_pages=713-723; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Cluster analysis and display of genome-wide expression patterns.; citation_author=MB Eisen; citation_author=PT Spellman; citation_author=PO Brown; citation_author=D Botstein; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=95; citation_number=28; citation_pages=14863-14868; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Strong minor groove base conservation in sequence logos implies DNA distortion or base flipping during replication and transcription initiation.; citation_author=TD Schneider; citation_journal_title=Nucleic Acids Res; citation_volume=29; citation_number=29; citation_pages=4881-4891; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Cpf1 protein induced bending of yeast centromere DNA element I.; citation_author=RK Niedenthal; citation_author=M Sen-Gupta; citation_author=A Wilmen; citation_author=JH Hegemann; citation_journal_title=Nucleic Acids Res; citation_volume=21; citation_number=30; citation_pages=4726-4733; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=Cbf1p is required for chromatin remodeling at promoter-proximal CACGTG motifs in yeast.; citation_author=NA Kent; citation_author=SM Eibert; citation_author=J Mellor; citation_journal_title=J Biol Chem; citation_volume=279; citation_number=31; citation_pages=27116-27123; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Flexible promoter architecture requirements for coactivator recruitment.; citation_author=DY Chiang; citation_author=DA Nix; citation_author=RK Shultzaberger; citation_author=AP Gasch; citation_author=MB Eisen; citation_journal_title=BMC Mol Biol; citation_volume=7; citation_number=32; citation_pages=16; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Identification and distinct regulation of yeast TATA box-containing genes.; citation_author=AD Basehoar; citation_author=SJ Zanton; citation_author=BF Pugh; citation_journal_title=Cell; citation_volume=116; citation_number=33; citation_pages=699-709; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Compensatory energetic relationships between upstream activators and the RNA polymerase II general transcription machinery.; citation_author=AM Lehman; citation_author=KB Ellwood; citation_author=BE Middleton; citation_author=M Carey; citation_journal_title=J Biol Chem; citation_volume=273; citation_number=34; citation_pages=932-939; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Evolution of biological information.; citation_author=TD Schneider; citation_journal_title=Nucleic Acids Res; citation_volume=28; citation_number=35; citation_pages=2794-2799; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Identifying target sites for cooperatively binding factors.; citation_author=D GuhaThakurta; citation_author=GD Stormo; citation_journal_title=Bioinformatics; citation_volume=17; citation_number=36; citation_pages=608-621; citation_date=2001; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001199" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Determining Physical Constraints in Transcriptional Initiation Complexes Using DNA Sequence Analysis"/>
    <meta name="twitter:description" content="Eukaryotic gene expression is often under the control of cooperatively acting transcription factors whose binding is limited by structural constraints. By determining these structural constraints, we can understand the &ldquo;rules&rdquo; that define functional cooperativity. Conversely, by understanding the rules of binding, we can infer structural characteristics. We have developed an information theory based method for approximating the physical limitations of cooperative interactions by comparing sequence analysis to microarray expression data. When applied to the coordinated binding of the sulfur amino acid regulatory protein Met4 by Cbf1 and Met31, we were able to create a combinatorial model that can correctly identify Met4 regulated genes. Interestingly, we found that the major determinant of Met4 regulation was the sum of the strength of the Cbf1 and Met31 binding sites and that the energetic costs associated with spacing appeared to be minimal."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0001199.g007"/>

  <meta property="og:title" content="Determining Physical Constraints in Transcriptional Initiation Complexes Using DNA Sequence Analysis" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001199" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=4333'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=8248'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=5800&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0001199">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001199" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001199&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Determining Physical Constraints in Transcriptional Initiation Complexes Using DNA Sequence Analysis
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Ryan K. Shultzaberger, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Molecular and Cell Biology, University of California, Berkeley, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Derek Y. Chiang, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Molecular and Cell Biology, University of California, Berkeley, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Alan M. Moses, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Graduate Group in Biophysics, University of California, Berkeley, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Michael B. Eisen
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:mbeisen@lbl.gov">mbeisen@lbl.gov</a></p>

                <p>Affiliations:
                  Department of Molecular and Cell Biology, University of California, Berkeley, United States of America, 
                  Department of Genome Sciences, Genomic Division, Ernest Orlando Lawrence Berkeley National Lab, Berkeley, California, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: November 21, 2007</li>
    <li>DOI: 10.1371/journal.pone.0001199</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0001199" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0001199" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001199" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0001199" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0001199-g001" data-doi="info:doi/10.1371/journal.pone.0001199" data-uri="info:doi/10.1371/journal.pone.0001199.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001199.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001199-g002" data-doi="info:doi/10.1371/journal.pone.0001199" data-uri="info:doi/10.1371/journal.pone.0001199.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001199.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001199-g003" data-doi="info:doi/10.1371/journal.pone.0001199" data-uri="info:doi/10.1371/journal.pone.0001199.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001199.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001199-g004" data-doi="info:doi/10.1371/journal.pone.0001199" data-uri="info:doi/10.1371/journal.pone.0001199.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001199.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001199-g005" data-doi="info:doi/10.1371/journal.pone.0001199" data-uri="info:doi/10.1371/journal.pone.0001199.g005" title="Figure 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001199.g005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001199-g006" data-doi="info:doi/10.1371/journal.pone.0001199" data-uri="info:doi/10.1371/journal.pone.0001199.g006" title="Figure 6">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001199.g006&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001199-g007" data-doi="info:doi/10.1371/journal.pone.0001199" data-uri="info:doi/10.1371/journal.pone.0001199.g007" title="Figure 7">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001199.g007&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001199">Reader Comments (1)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0001199" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1"></a><p>Eukaryotic gene expression is often under the control of cooperatively acting transcription factors whose binding is limited by structural constraints. By determining these structural constraints, we can understand the “rules” that define functional cooperativity. Conversely, by understanding the rules of binding, we can infer structural characteristics. We have developed an information theory based method for approximating the physical limitations of cooperative interactions by comparing sequence analysis to microarray expression data. When applied to the coordinated binding of the sulfur amino acid regulatory protein Met4 by Cbf1 and Met31, we were able to create a combinatorial model that can correctly identify Met4 regulated genes. Interestingly, we found that the major determinant of Met4 regulation was the sum of the strength of the Cbf1 and Met31 binding sites and that the energetic costs associated with spacing appeared to be minimal.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>Shultzaberger RK, Chiang DY, Moses AM, Eisen MB (2007) Determining Physical Constraints in Transcriptional Initiation Complexes Using DNA Sequence Analysis. PLoS ONE 2(11):
          e1199.
            doi:10.1371/journal.pone.0001199</p><p><strong>Academic Editor: </strong>Abraham Sonenshein, Tufts University, United States of America</p><p><strong>Received:</strong> March 13, 2007; <strong>Accepted:</strong> October 29, 2007; <strong>Published:</strong> November 21, 2007</p><p><strong>Copyright:</strong> © 2007 Shultzaberger et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>This work was supported by the Director, Office of Science, of the U.S. Department of Energy under Contract No. DE-AC02-05CH11231 and NIH grant #RHG002779A.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>The regulation of transcriptional initiation from individual eukaryotic promoters is often controlled by multiple cooperatively interacting transcription factors. These factors bind to separate sites in cis-regulatory sequences and physically interact with each other, either directly or through additional proteins, to activate or repress transcription <a href="#pone.0001199-Levine1">[1]</a>, <a href="#pone.0001199-Lee1">[2]</a>, <a href="#pone.0001199-Lee2">[3]</a>. These physical interactions among transcription factors must constrain how their binding sites can be positioned relative to each other and to the relevant promoters. Yet, there is often considerable variability in the order, orientation and spacing of binding sites for interacting transcription factors <a href="#pone.0001199-Blaiseau1">[4]</a>, <a href="#pone.0001199-Thomas1">[5]</a>, <a href="#pone.0001199-Chiang1">[6]</a>. Understanding how the arrangement of sites is related to the stability of these complexes and their regulatory activity is essential if we are to understand the regulatory content of eukaryotic genomes.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>To successfully model the binding of multi-meric complexes to different target sequences, many energetic contributions need to be considered. The affinity of each transcription factor for DNA varies considerably with the precise bound sequence, even among known in vivo targets <a href="#pone.0001199-Shultzaberger1">[7]</a>, <a href="#pone.0001199-Udalova1">[8]</a>. The stability of the entire complex is also dependent on how compatible the positioning of the sites are with the protein-protein interactions necessary to form the complex. Poorly positioned sites presumably introduce clashes or strain into either the complex or DNA which will, in turn, reduce the stability of the complex.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>Here, we combine DNA sequence analysis and genome-wide expression data to discern the constraints on the arrangement of binding sites for transcription factors involved in regulating the synthesis of sulfur-containing amino acids in the yeast <em>Saccharomyces cerevisiae</em>. This work builds on our previous modeling of bipartite prokaryotic ribosome and σ<sup>70</sup> binding sites <a href="#pone.0001199-Shultzaberger2">[9]</a>, <a href="#pone.0001199-Shultzaberger3">[10]</a>. In both of these cases, initiation requires the cooperative binding of two independent components separated by a variable spacer, the Shine-Dalgarno and P site for ribosome binding sites, and the −10 and −35 for σ<sup>70</sup> binding sites <a href="#pone.0001199-Chen1">[11]</a>, <a href="#pone.0001199-RinkeAppel1">[12]</a>, <a href="#pone.0001199-Hawley1">[13]</a>, <a href="#pone.0001199-McClure1">[14]</a>. Since there were a large number of characterized sites for these systems, we constructed a robust distribution of the allowable spacings between binding components. Assuming that the spacing that would induce the least amount of strain in the protein or in the bound DNA upon binding would be the most commonly observed, and that the frequency of occurrence of all other spacings would be directly related to the energetic consequence of using that spacing, we could model the energetic contribution of different spacings to the formation of a stable initiation complex.</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p>Cooperatively acting transcription factors in eukaryotes are similar to the prokaryotic ribosome and σ<sup>70</sup> in that they have independent binding components separated by variable spacers, but they are different in that the components are not physically linked upon binding and therefore can bind in different orders, orientations, and with greater variability in their spacing. We have devised a method to determine these additional physical constraints by optimizing an information theory based model against microarray data. We can use these optimized constraints to not only infer structural characteristics of the regulatory complex, but also to quantify the binding of these multi-meric complexes to different DNA sequences, and to accurately predict target genes.</p>
<a id="article1.body1.sec1.p5" name="article1.body1.sec1.p5"></a><p>Met4 is the major transcriptional activator of sulfur utilization genes in <em>Saccharomyces cerevisiae</em> even though it does not bind directly to DNA <a href="#pone.0001199-Thomas2">[15]</a>, <a href="#pone.0001199-Thomas1">[5]</a>. Met4 stabilization is dependent upon at least two additional proteins. One of these is the centromere-binding factor (Cbf1) <a href="#pone.0001199-Thomas2">[15]</a>, whose DNA binding activity is stimulated by association with Met28 <a href="#pone.0001199-Kuras1">[16]</a>. It has been suggested that the Cbf1-Met28-Met4 complex may be sufficient for activation of some genes, but coordination by a second factor is necessary for others <a href="#pone.0001199-Blaiseau1">[4]</a>. We are interested in describing this coordinated system. The second stabilizing factor that we will study is Met31, a factor unique to sulfur regulation <a href="#pone.0001199-Blaiseau2">[17]</a>.</p>
<a id="article1.body1.sec1.p6" name="article1.body1.sec1.p6"></a><p>Neither the distance between Cbf1 and Met31 in functional Met4 stabilizing complexes, nor the distance between Met4 and the initiating polymerase is fixed <a href="#pone.0001199-Thomas1">[5]</a>. We extended the information theory-based method we used to study prokaryotic translational and transcriptional initiation to model Cbf1 and Met31 interactions, allowing for the greater flexibility present in this system.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4>Cbf1 and Met31 binding models</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>We built a weight matrix describing the sequence preferences of Cbf1 from 16 Cbf1 binding sites characterized by Wieland <em>et al.</em><a href="#pone.0001199-Wieland1">[18]</a>. Binding matrices were built using the standard <strong>Delila</strong> programs <a href="#pone.0001199-Schneider1">[19]</a>, <a href="#pone.0001199-Schneider2">[20]</a>. Since Cbf1 binds as a homodimer, we used each sequence and its complement to build our model <a href="#pone.0001199-Schneider3">[21]</a> (<a href="#pone-0001199-g001">Fig. 1A</a>). Because of the lack of experimentally verified binding sites for Met31, we modeled its binding by analyzing 21 non-divergently transcribed genes identified in a Met4 chromatin immuno-precipitation assay <a href="#pone.0001199-Lee2">[3]</a> (we selected genes with p&lt;0.001). We used MEME <a href="#pone.0001199-Bailey1">[22]</a> with the -tcm model and required at least 10 copies of a motif to identify sequences enriched in these target genes, from which we computed an initial Met31 weight matrix. We then scanned the entire genome for sites with greater than 10 bits of information against this model, identifying 209 sites, from which we constructed the Met31 weight matrix used in our analysis (<a href="#pone-0001199-g001">Fig. 1B</a>).</p>


<h4>Searching algorithm</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>Multi-component binding systems with variable spacing between components have previously been modeled <a href="#pone.0001199-Shultzaberger2">[9]</a>, <a href="#pone.0001199-Shultzaberger3">[10]</a>. In the case of the prokaryotic ribosome and σ<sup>70</sup>, the binding components are physically connected. In both instances, deviations in the optimal spacing between components introduces strain in the bound complex and affects the binding energy <a href="#pone.0001199-Chen1">[11]</a>, <a href="#pone.0001199-RinkeAppel1">[12]</a>, <a href="#pone.0001199-Hawley1">[13]</a>, <a href="#pone.0001199-McClure1">[14]</a>. To model these multi-meric binders the following equation was used:<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001199.e001&amp;representation=PNG" class="inline-graphic"><span class="note">(1)</span></span><br>where <em>R<sub>i</sub></em>(<em>A</em>) is the relative strength, or individual information, of binding factor A, and <em>R<sub>i</sub></em>(<em>B</em>) is the relative strength of binding factor B according to <a href="#pone.0001199-Schneider2">[20]</a>. <em>GS</em>(<em>d</em>) is the gap surprisal (based on Tribus' surprisal function <a href="#pone.0001199-Tribus1">[23]</a>), or penalty of having a spacing of <em>d</em> between sites A and B as determined by <a href="#pone.0001199-Shultzaberger2">[9]</a>, <a href="#pone.0001199-Shultzaberger3">[10]</a>:<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001199.e002&amp;representation=PNG" class="inline-graphic"><span class="note">(2)</span></span><br><em>n</em>(<em>d</em>) is the number of occurrences at spacing <em>d</em> and <em>n</em> is the number of total occurrences over the allowed values of <em>d</em>. <em>e</em>(<em>n</em>) is a small sample correction value <a href="#pone.0001199-Schneider4">[24]</a>, <a href="#pone.0001199-Shultzaberger2">[9]</a>. For our initial analysis of Cbf1 and Met31, we used a flat spacing distribution where all spacings have the same gap surprisal value of <em>GS</em>(<em>d</em>) = -log<sub>2</sub>(1/(<em>d<sub>max</sub></em>–<em>d<sub>min</sub></em>+1)), where <em>d<sub>min</sub></em> is the shortest spacing between Met31 and Cbf1, and <em>d<sub>max</sub></em> is the longest spacing. The distance between Met31 and Cbf1 is calculated between the zero positions of the binding components as with previous flexible models.</p>
<a id="article1.body1.sec2.sec2.p2" name="article1.body1.sec2.sec2.p2"></a><p>For the ribosome and the polymerase, the binding components are physically linked and can only bind in one orientation relative to each other. For cooperatively acting transcription factors though, there could be variation in the orientation of the sites relative to each other. To account for this, we can adapt the gap surprisal function to:<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001199.e003&amp;representation=PNG" class="inline-graphic"><span class="note">(3)</span></span><br>where we calculate an orientation surprisal (<em>OS</em>(<em>o</em>)) that is the logarithm of the frequency of occurrence at each orientation. For a system where both orientations occur at equal frequency, the number of occurrences at either orientation would be <em>n</em>(<em>o</em>) = 1, and the total number of occurrences is <em>n</em> = 2. The orientation surprisal for this system would therefore be 1 bit of information. In a system where there is no variability in orientation, the frequency of occurrence at that orientation would be <em>n</em>(<em>o</em>)/<em>n</em> = 1, and therefore the orientation surprisal would be 0 bits. The advantage of the <em>OS</em>(<em>o</em>) calculation is that we can model the subtle energetic differences for systems that allow either orientation, but favor one over the other.</p>
<a id="article1.body1.sec2.sec2.p3" name="article1.body1.sec2.sec2.p3"></a><p>To calculate the total information for Met4 coordination, we can now expand equation (1) to: <a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001199.e004&amp;representation=PNG" class="inline-graphic"><span class="note">(4)</span></span><br>There is no orientation surprisal for Cbf1. Since Cbf1 is homodimeric and has a symmetric matrix, the Cbf1-DNA complex would be identical for either orientation. In this case, the frequency of occurrence of a given orientation would be 1, and <em>OS<sub>Cbf</sub></em><sub>1</sub> = 0 bits. Therefore, the orientation surprisal only applies to asymmetric binders.</p>
<a id="article1.body1.sec2.sec2.p4" name="article1.body1.sec2.sec2.p4"></a><p>Combinatorial scans were done using <strong>multiscan</strong> <a href="#pone.0001199-Shultzaberger3">[10]</a> to identify and quantify Cbf1/Met31 cooperatively acting binding sites in the genome. The individual information contribution for both sites (<em>R<sub>i</sub></em>(<em>Cbf</em>1) and <em>R<sub>i</sub></em>(<em>Met</em>31)) were calculated over the range −4 to +5, since this is the range of conservation for both logos (<a href="#pone-0001199-g001">Fig. 1</a>) <a href="#pone.0001199-Schneider2">[20]</a>. Sites were only considered if each component had an <em>R<sub>i</sub></em>&gt;0 bits (which would correspond to a -Δ<em>G</em> of binding <a href="#pone.0001199-Schneider2">[20]</a>, <a href="#pone.0001199-Schneider5">[25]</a>) and they have a flexible site information &gt;0 bits. For a site to have a positive flexible site information, the ordering and orientation of the pair have to be within the defined spacing and ordering parameters. For any spacing or orientation outside of the specified range, the sites would have a surprisal penalty equal to infinity according to equations (2) and (3), and a flexible site information &lt;0 bits according to equation (4).</p>
<a id="article1.body1.sec2.sec2.p5" name="article1.body1.sec2.sec2.p5"></a><p>All genes in the genome were then ranked based on the strength of their strongest upstream site. Microarray expression data for sulfur amino acid pathway-affected cells (see Microarray Datasets) were then averaged for the top 30 genes in our ranking. All values averaged were log<sub>2</sub> of the expression fold change between affected and unaffected cells. This was done independently for induction and repression experiments.</p>
<a id="article1.body1.sec2.sec2.p6" name="article1.body1.sec2.sec2.p6"></a><p>The physical constraints that we want to define are: the ordering of the sites relative to the gene start, the orientation of the matrices, the maximum allowed distance between Met4 and the polymerase binding site, and the spacing range between Cbf1 and Met31 that can bind Met4. We varied these constraints, and iteratively refined the model to get the optimal predictor. We evaluated any given set of parameters by calculating the average expression change in the top 30 ranked genes. The greater the expression change the better the model.</p>
<a id="article1.body1.sec2.sec2.p7" name="article1.body1.sec2.sec2.p7"></a><p>Another approach could be to cluster genes based on similar trends in expression data across several experiments, and then try to train our parameters based on this set of genes. One disadvantage of this is that it is difficult to discern directly from indirectly regulated genes in these clusters. By scanning the genome and ranking the genes, we are selecting only for genes that are directly regulated. Also this approach does not exclude genes that are regulated but had anomalous expression data due to experimental error. Since there have been at least 20 genes implicated in sulfur assimilation <a href="#pone.0001199-Thomas1">[5]</a>, we chose to average the top 30 gene expression differences to evaluate our model. We chose 30 so that we would not overfit our model by looking at too few genes, and not introduce noise into our analysis by averaging too many.</p>


<h4>Microarray Datasets</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>We used microarray data from two sources for our analysis. Gasch <em>et al.</em> <a href="#pone.0001199-Gasch1">[26]</a> reported amino acid starvation data, where transcription of Met4 regulated genes was induced. Fauchon <em>et al.</em> <a href="#pone.0001199-Fauchon1">[27]</a> reported Cd<sup>2+</sup> addition experiments where Met4 regulated genes were induced, and Met4 deletion experiments where Met4 regulated genes were repressed. Our models were optimized against these data as mentioned above. Microarray expression patterns were visualized using <strong>TreeView</strong> <a href="#pone.0001199-Eisen1">[28]</a>. The yeast genome sequence and annotation that we used in our analysis came from Genbank accession numbers NC001133 to NC001148.</p>

</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Results"></a><h3>Results</h3>
<h4>Cbf1 and Met31 logos</h4>
<a id="article1.body1.sec3.sec1.p1" name="article1.body1.sec3.sec1.p1"></a><p>Since Cbf1 is a homodimeric protein, we used all sequences and their complements to build our model <a href="#pone.0001199-Schneider3">[21]</a>. Conservation at positions −2,−1 and +2,+3 is strong and does not match the helical accessibility wave (<a href="#pone-0001199-g001">Fig. 1A</a>). Deviation of sequence conservation from the helical accessibility wave is generally an indicator of structural changes in the DNA substrate <a href="#pone.0001199-Schneider6">[29]</a>. This may be consistent with the observed bending of DNA by Cbf1 <a href="#pone.0001199-Niedenthal1">[30]</a>.</p>
<a id="article1.body1.sec3.sec1.p2" name="article1.body1.sec3.sec1.p2"></a><p>The Met31 model was built as described in <a href="#s2">Materials and Methods</a> (<a href="#pone-0001199-g001">Fig. 1B</a>). Sequence conservation appeared to follow the helical accessibility wave well, and it was contained within one major groove. Met31 has an asymmetric binding site, so it can possibly bind with two different orientations. We tested both orientations in our analysis. The information content for the Cbf1 logo is 12.9 bits over the range −4 to +5. The information content for the Met31 logo is 11.9 bits over the range −4 to +5.</p>
<div class="figure" id="pone-0001199-g001"><div class="img"><a name="pone-0001199-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001199.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001199" data-uri="info:doi/10.1371/journal.pone.0001199.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001199.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001199.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001199.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001199.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001199.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001199.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001199.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001199.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001199.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Cbf1 and Met31 sequence logos.</span></strong></p><a id="article1.body1.sec3.sec1.fig1.caption1.p1" name="article1.body1.sec3.sec1.fig1.caption1.p1"></a><p>Sequence logos were made as described in <a href="#s2">Materials and Methods</a>. The height of each letter is proportional to the frequency of that base at that position. The height of the letter stack is the information content at that position. The cosine wave represents the helical twist of B-form DNA. The sequence logos were generated using the standard Delila programs <a href="#pone.0001199-Schneider1">[19]</a>, <a href="#pone.0001199-Schneider3">[21]</a>.</p>
<span>doi:10.1371/journal.pone.0001199.g001</span></div>

<h4>Orientation and ordering</h4>
<a id="article1.body1.sec3.sec2.p1" name="article1.body1.sec3.sec2.p1"></a><p>Since Cbf1 and Met31 are not physically linked upon binding, it was not immediately obvious what the ordering and orientation constraints on their binding are in functional Met4 docking complexes. To determine this, we tested the predictive capabilities of all combinations of orientation and ordering for Cbf1 and Met31 using the gene-ranking approach described in <a href="#s2">Materials and Methods</a>. Briefly, we determined the flexible information for the cooperative model as determined by equation (4) <a href="#pone.0001199-Shultzaberger2">[9]</a>, <a href="#pone.0001199-Shultzaberger3">[10]</a>, and ranked all genes in the genome based on the strength of the strongest site in the intergenic region immediately upstream of their starts. We then calculated the average expression fold change of the top 30 genes in this ranking based on Met4 induced and repressed microarray experiments <a href="#pone.0001199-Gasch1">[26]</a>, <a href="#pone.0001199-Fauchon1">[27]</a>. We regarded those combinations that gave the highest average microarray expression change to be the optimal organization for Met4 coordination. <a href="#pone-0001199-g002">Fig. 2</a> shows how well different combinations performed.</p>
<div class="figure" id="pone-0001199-g002"><div class="img"><a name="pone-0001199-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001199.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001199" data-uri="info:doi/10.1371/journal.pone.0001199.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001199.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001199.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001199.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001199.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001199.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001199.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001199.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001199.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001199.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Met4 binding by Cbf1 and Met31 is dependent upon ordering but not orientation.</span></strong></p><a id="article1.body1.sec3.sec2.fig1.caption1.p1" name="article1.body1.sec3.sec2.fig1.caption1.p1"></a><p>We scanned all intergenic regions in yeast with the models presented in <a href="#pone-0001199-g001">Fig. 1</a> with different orientations and orderings relative to the gene start point. We then ranked all genes in the genome based on the strength of their strongest upstream binding site, and we present here the corresponding expression changes as determined by microarrays. The experiments that each column represent correspond to those in <a href="#pone-0001199-g006">Fig. 6</a>. For columns 1–9 (marked with a gray box) we expect regulated genes to have increased expression and therefore to be red. For columns 10–13 (marked with a blue box) we expect regulated genes to have a decreased expression and therefore to be green. Since the Met31 matrix is asymmetric, it could bind with two different orientations. Those circles labeled “Met31” have the same orientation as the Met31 logo in <a href="#pone-0001199-g001">Fig. 1</a>. Those circles labeled “Inverted” have the opposite orientation (see <a href="#pone-0001199-g005">Fig. 5</a>). The optimal combination in the lower right corner allows for either orientation of Met31. The arrow signifies the gene start. The average expression change for the top 30 genes was calculated for each combination of sites for both the induced (columns 1 to 9) and repressed (column 10 to 13) experiments and are reported next to their respective columns.</p>
<span>doi:10.1371/journal.pone.0001199.g002</span></div><a id="article1.body1.sec3.sec2.p2" name="article1.body1.sec3.sec2.p2"></a><p>Cbf1 alone was not sufficient to identify the Met4 regulated genes. The average expression fold change for the top 30 ranked genes was 0.09 and 0.16 for induction and repression data respectively, we report corresponding values for all other combinations. Met31 alone appeared to be a better predictor than Cbf1, but was still weak (0.44 and −0.90). This improvement of Met31 prediction over Cbf1 is expected since Cbf1 has a regulatory role outside of Met4 binding <a href="#pone.0001199-Kent1">[31]</a>. For both Cbf1 and Met31, we only considered binding sites within 1000 bases upstream of the closest gene start. By searching for Cbf1 and Met31 sites together, with a maximum spacing of 100 bases between the zero positions of the binding components (<a href="#pone-0001199-g001">Fig. 1</a>) and the downstream component could be a maximum of 1000 bases upstream of the gene start, the prediction was better. If we searched with the order Cbf1-Met31-gene start, we were able to identify more genes with the expected microarray pattern than with the order Met31-Cbf1-gene start (0.69 and −1.41 vs. 0.14 and −0.37).</p>
<a id="article1.body1.sec3.sec2.p3" name="article1.body1.sec3.sec2.p3"></a><p>Since Cbf1 is a homodimer, its binding is independent of orientation. Since Met31 is monomeric, its binding is orientation dependent. When we allowed for both orientations of Met31 downstream of Cbf1, we got the largest change of expression (0.95 and −1.57). This suggested that transcriptional activation by Met4 requires a Met31 site with any orientation to fall between Cbf1 and the gene start (bottom right panel of <a href="#pone-0001199-g002">Fig. 2</a>).</p>
<a id="article1.body1.sec3.sec2.p4" name="article1.body1.sec3.sec2.p4"></a><p>To test whether the average expression values that we observed are statistically significant, we randomly chose 10,000 sets of 30 genes from the genome and averaged their expression change values. We did this for both the induced and repressed data sets. Both sets gave similar normal distributions with a mean of −0.015 and SD of 0.11 for the induced data set and a mean of 0.001 and SD of 0.094 for the repressed data set. For the best organization of sites in <a href="#pone-0001199-g002">Figure 2</a>, an expression change of 0.95 and −1.57 would be 8.7 and 16.7 standard deviations from the mean respectively. The probability of selecting a set of 30 genes with an average expression change this high randomly would be less than 1×10<sup>−8</sup>.</p>
<a id="article1.body1.sec3.sec2.p5" name="article1.body1.sec3.sec2.p5"></a><p>All models for the remainder of this analysis will have these ordering and orientation requirements imposed on them. The designation of the Met31 model orientation as “normal” or “inverted” is arbitrary. We also tested the “inverted” Met31 model alone, and inverted Met31 upstream of Cbf1, but the results were similar to equivalent scans with the “normal” orientation (data not shown).</p>


<h4>Spacing constraints</h4>
<a id="article1.body1.sec3.sec3.p1" name="article1.body1.sec3.sec3.p1"></a><p>There are two spacing constraints on this system, the distance between the Met4 docking complex and the initiating polymerase, and the distance between the two binding components (Cbf1 and Met31) within the Met4 docking complex. To define what these spacing ranges are for functional Met4 binding sites, we systematically modeled different spacing ranges, and quantified the models by the gene-ranking approach previously described. Interestingly, if we varied one of the spacing constraints, the optimal spacing for the other would differ slightly. To identify which spacing parameters define the optimal predictor, we varied both spacings simultaneously, and quantified their predictability by averaging the expression change of their 30 highest ranking genes.</p>
<a id="article1.body1.sec3.sec3.p2" name="article1.body1.sec3.sec3.p2"></a><p>We increased the maximum allowed distance of the Met4 docking complex from the gene start in 50 base increments as measured by the distance between the Met31 site and the translational initiation codon. At each 50 bp increment, we varied the minimum and maximum allowed distance between Cbf1 and Met31 from 1 to 100 bases. These distances are relative to the zero position of both matrices (<a href="#pone-0001199-g001">Fig. 1</a>). We then summed the average expression change for the induction and repression experiments for all combinations of spacings, and determined which combination predicted the microarray data best.</p>
<a id="article1.body1.sec3.sec3.p3" name="article1.body1.sec3.sec3.p3"></a><p>For the first spacing constraint, the distance between Met4 and the polymerase, we found the optimal maximum spacing was 450 bases (<a href="#pone-0001199-g003">Fig. 3</a>). The predictability of the model seemed to increase linearly from 100 to 350 bases suggesting that the sites are evenly distributed over this range. There appeared to be few or no genes with sites closer than 100 bases upstream, or sites farther than 450 bases upstream that had the expected expression pattern.</p>
<div class="figure" id="pone-0001199-g003"><div class="img"><a name="pone-0001199-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001199.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001199" data-uri="info:doi/10.1371/journal.pone.0001199.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001199.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001199.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001199.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001199.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001199.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001199.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001199.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001199.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001199.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Met4 binding is within 450 bases of the gene start, but not within 100 bases.</span></strong></p><a id="article1.body1.sec3.sec3.fig1.caption1.p1" name="article1.body1.sec3.sec3.fig1.caption1.p1"></a><p>We varied the allowed distance that the Met31 binding site can be from the gene start point in our models, and quantified how this spacing constraint affected our ability to predict microarray expression data. A) We plotted the average expression change of the top 30 hits in the genome for different maximum spacings from the gene start. The top line corresponds to data from experiments where we expected increased expression (columns 1 to 9 in B), and the lower line is from experiments where we expected decreased expression (columns 10 to 13 in B). The microarray data that corresponds to our gene-ranking are shown in B. The conditions for each column in the microarrays correspond to the labeled columns in <a href="#pone-0001199-g006">Fig. 6</a>.</p>
<span>doi:10.1371/journal.pone.0001199.g003</span></div><a id="article1.body1.sec3.sec3.p4" name="article1.body1.sec3.sec3.p4"></a><p>For the second spacing constraint, the distance between Cbf1 and Met31, we found the optimal spacing range to be −9 to −68 bases, the minimum to maximum spacing allowed between each site (<a href="#pone-0001199-g004">Fig. 4</a>). This was the range used in the analysis in <a href="#pone-0001199-g003">Fig. 3</a>. Ranges close to −9 to −68 appeared to have a similar level of predictability as indicated by the redish semi-circle in <a href="#pone-0001199-g004">Fig. 4</a>, but −9 to −68 had the highest expression change and the tightest range. The average expression changes for these two spacing parameters were 1.04 and −1.77 for induction and repression data respectively.</p>
<div class="figure" id="pone-0001199-g004"><div class="img"><a name="pone-0001199-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001199.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001199" data-uri="info:doi/10.1371/journal.pone.0001199.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001199.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001199.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001199.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001199.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001199.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001199.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001199.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001199.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001199.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>The optimal spacing range between Cbf1 and Met31 is 9 to 68 bases.</span></strong></p><a id="article1.body1.sec3.sec3.fig2.caption1.p1" name="article1.body1.sec3.sec3.fig2.caption1.p1"></a><p>We varied the minimum (X axis) and maximum (Y axis) distance that Cbf1 and Met31 could be from each other in our model, and calculated the average expression change within the corresponding top 30 hits, according to these ranges. We show here the average expression change for only those experiments that we expected to have a decreased expression (columns 10 to 13 in <a href="#pone-0001199-g006">Fig. 6</a>). The colors correspond to the key in lower portion of the plot.</p>
<span>doi:10.1371/journal.pone.0001199.g004</span></div>

<h4>Optimal model</h4>
<a id="article1.body1.sec3.sec4.p1" name="article1.body1.sec3.sec4.p1"></a><p>Based on the analysis in <a href="#pone-0001199-g002">Fig. 2</a>, <a href="#pone-0001199-g003">Fig. 3</a>, and <a href="#pone-0001199-g004">Fig. 4</a>, the optimal model is shown in <a href="#pone-0001199-g005">Fig. 5</a>. This model requires a Cbf1 site to be 9 to 68 bases upstream of a Met31 site with either orientation, and for the Met31 site to be no more than 450 bases upstream of the translational initiation codon. When we scanned the genome with this model, we see that most of our top hits are genes known to be involved in sulfur amino acid biosynthesis (<a href="#pone-0001199-g006">Fig. 6</a>). Two genes in the top 23 hits have a strikingly unexpected expression pattern (Reb1 and Gar1). Additional analysis of these sites show that they both have a strong Cbf1 site, but a “T” instead of “G” at position +1 of their Met31 site. This suggests that the information contribution at position +1 may be greater than that in our current matrix. Several genes have both the expected expression profile and a predicted Met4 binding site, but their functions have not been biochemically characterized (DDR48, YIL074C, YJL060W, YHR112C). Clustering of co-regulated genes by the gene-ranking method may have identified other genes involved in sulfur utilization.</p>
<div class="figure" id="pone-0001199-g005"><div class="img"><a name="pone-0001199-g005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001199.g005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001199" data-uri="info:doi/10.1371/journal.pone.0001199.g005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001199.g005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001199.g005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001199.g005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001199.g005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001199.g005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001199.g005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001199.g005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001199.g005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001199.g005.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 5.  <span>The Met4 activation model based on our analysis.</span></strong></p><a id="article1.body1.sec3.sec4.fig1.caption1.p1" name="article1.body1.sec3.sec4.fig1.caption1.p1"></a><p>We summarize here the spacing, ordering, and orientation constraints we used to define functional Met4 binding sites. Since Met31 can bind with either orientation, we show logos for both Met31 orientations. The distances between each set of Cbf1 and Met31 sites were plotted with red boxes on a cosine wave for 23 high-ranking genes to show helical preferences. The arrow represents the translational start, and the allowed distance between the Met4 stabilization complex and the translational start is written above it. The expression data on the right is what was predicted by this model, and is described in <a href="#pone-0001199-g006">Fig. 6</a>.</p>
<span>doi:10.1371/journal.pone.0001199.g005</span></div><div class="figure" id="pone-0001199-g006"><div class="img"><a name="pone-0001199-g006" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001199.g006&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001199" data-uri="info:doi/10.1371/journal.pone.0001199.g006"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001199.g006&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001199.g006/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001199.g006/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001199.g006/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001199.g006/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001199.g006.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001199.g006/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001199.g006/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001199.g006.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 6.  <span>The top hits are involved in sulfur amino acid biosynthesis.</span></strong></p><a id="article1.body1.sec3.sec4.fig2.caption1.p1" name="article1.body1.sec3.sec4.fig2.caption1.p1"></a><p>These are the top hits according to our optimal spacing values. The first 9 columns are data for experiments that should induce the expression of Met4 regulated genes and give a red pattern. The last 4 columns we expect to see a decrease in expression of Met4 regulated genes and give a green pattern. Experiment information for each column is reported vertically above each column. Each row corresponds to a different gene followed by its common name, its flexible information (<em>R<sub>i</sub></em>), the coordinate of the Met31 binding site in the <em>S. cerevisiae</em> genome, the orientation of the Met31 matrix, the distance Cbf1 is upstream of Met31, the distance the gene start is downstream of Met31, the gene name according to its annotation in the yeast genome, and a description of its function.</p>
<span>doi:10.1371/journal.pone.0001199.g006</span></div><a id="article1.body1.sec3.sec4.p2" name="article1.body1.sec3.sec4.p2"></a><p>To test whether we were over- or under-fitting our model, we repeated the optimization by averaging the top 20 and 40 genes instead of 30. When we did this our results differed slightly. By averaging the top 20 genes, we found the optimal spacing range shrunk slightly to 13 to 68 bases between Cbf1 and Met31 and the optimal maximum distance between Met31 and the gene start remained at 450 bases. At this smaller range, Gsh1 and Sam1 were lost, both which have been implicated in sulfur assimilation. By averaging the top 40 genes, the spacing range remained the same (9 to 68 bases), but the maximum spacing range expanded to 750 bases. No new genes with the expected expression pattern or functional evidence for Met4 regulation were identified at this larger spacing. The larger the number of genes that are averaged, the more likely random genes with above average expression differences will be in the averaged set, which will obscure the actual parameters. Thirty genes appeared to be the appropriate number to average, and the relatively consistent parameters observed from averaging 20, 30, or 40 genes suggested that these parameters are reliable.</p>
<a id="article1.body1.sec3.sec4.p3" name="article1.body1.sec3.sec4.p3"></a><p>To test whether there is a tendency for Cbf1 and Met31 to bind on the same face of the DNA, we plotted the relative spacing between the two sites on a cosine wave with the same period as B-form DNA, 10.6 bases (<a href="#pone-0001199-g005">Fig. 5</a>). We plotted the spacings of 19 of the 23 top ranking genes (all sites except for Reb1, Gar1, Idh1 and YER080W) and YHR112C, Mxr1, Met10, and YML018C which had both a strong flexible information and expression change. To determine what the optimal phase of the cosine wave was, we plotted each spacing on a cosine wave and calculated the average height of all spacings on the helix. That is, if all spacings were at the top of the cosine wave (occurred in multiples of 10.6 bases) then the average helical location would be high. We determined the phase of the cosine wave that gave the highest average helical location of these 23 spacings, and found the optimal phase to peak at −13.86 bases relative to the Met31 zero position. To see if the relative placement of these spacings on the cosine wave is higher than expected, we determined the average helical location of random sets of 23 Cbf1/Met31 pairs. Our set had an average helical positioning greater than 95 percent of random sets.</p>
<a id="article1.body1.sec3.sec4.p4" name="article1.body1.sec3.sec4.p4"></a><p>Sam1 has the shortest Cbf1-Met31 spacing of 9 bases. This spacing is completely on the opposite face of the DNA according to a phasing of −13.86 bases. One would expect spacing to be more constrained at shorter distances, since it is more difficult for DNA to bend. We looked at the region upstream of the Sam1 translational start to see if there was another strong Cbf1-Met31 pair with a larger spacing. We found a second weaker Cbf1 site 36 bases upstream of the Met31 site, giving a total flexible information of 12.6 bits. This spacing would suggest same face binding according to a phasing of −13.86 bases. The strong Cbf1 site 9 bases upstream of the Met31 site may have occurred randomly, or may be part of an overlapping second site.</p>
<a id="article1.body1.sec3.sec4.p5" name="article1.body1.sec3.sec4.p5"></a><p>To calculate the flexible individual information for each binding site, we used equation (4). Since we did not know the energetic effect of different spacings on the complex initially, we treated all spacings equally. That is, over the range 9 to 68 (60 bases of variability) all positions had the same gap surprisal of <em>GS</em>(<em>d</em>) = -<em>log</em><sub>2</sub>(1/60) = 5.91 bits according to equation (2). We also assumed an equiprobable occurrence of each orientation of Met31, so that <em>OS<sub>Met</sub></em><sub>31</sub>(<em>o</em>) = -log<sub>2</sub>(1/2) = 1 bit of information according to equation (3). Therefore the <em>GS</em> and <em>OS</em> variables in equation (4) effectively become constants summing to 6.91 bits of uncertainty for each site. Because of the small number of target genes, and the already strong predictive capabilities of our model, we cannot determine the individual spacing constraints for this system. If we had a system with more sites, robust spacing and orientation distributions could be determined and individual penalties could be assigned.</p>
<a id="article1.body1.sec3.sec4.p6" name="article1.body1.sec3.sec4.p6"></a><p>We can use these values to predict the <em>R<sub>sequence</sub></em> or average information content for this system which is:<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001199.e005&amp;representation=PNG" class="inline-graphic"><span class="note">(5)</span></span><br></p>
<a id="article1.body1.sec3.sec4.p7" name="article1.body1.sec3.sec4.p7"></a><p><em>G̅S</em>(<em>d</em>) is the mean <em>GS</em>(<em>d</em>) value for all sites, and <em>ŌS</em>(<em>o</em>) is the mean <em>OS</em>(<em>o</em>) value. According to this equation <em>R<sub>sequence</sub></em>(<em>Met</em>4) = 12.9+11.9−5.91−1.0 = 17.9 bits of information.</p>
<a id="article1.body1.sec3.sec4.p8" name="article1.body1.sec3.sec4.p8"></a><p>For each gene we plotted the strength of its strongest upstream Met4 binding site according to the model in <a href="#pone-0001199-g005">Fig. 5</a> and its average expression change for induction and repression experiments (<a href="#pone-0001199-g007">Fig. 7</a>). At about <em>R<sub>i</sub></em>&gt;14 bits, the number of genes that showed no, or an unexpected expression difference was significantly lower. This is about the same <em>R<sub>i</sub></em> as the site upstream of Met10 (14.0 bits), the lowest ranking sulfur assimilation protein in our analysis.</p>
<div class="figure" id="pone-0001199-g007"><div class="img"><a name="pone-0001199-g007" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001199.g007&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001199" data-uri="info:doi/10.1371/journal.pone.0001199.g007"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001199.g007&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001199.g007/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001199.g007/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001199.g007/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001199.g007/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001199.g007.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001199.g007/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001199.g007/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001199.g007.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 7.  <span>R<sub>i</sub> vs. expression change.</span></strong></p><a id="article1.body1.sec3.sec4.fig3.caption1.p1" name="article1.body1.sec3.sec4.fig3.caption1.p1"></a><p>The flexible information (<em>R<sub>i</sub></em>) of the strongest site for each gene is on the abscissa and the average induction or repression expression change is on the ordinate. For each gene, induction data were averaged from the first nine experiments in <a href="#pone-0001199-g006">Fig. 6</a> (red circles), and repression data were averaged from the last four experiments in <a href="#pone-0001199-g006">Fig. 6</a> (black circles).</p>
<span>doi:10.1371/journal.pone.0001199.g007</span></div>
</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec4.p1" name="article1.body1.sec4.p1"></a><p>Transcriptional initiation in eukaryotes is often regulated by multiple cooperatively acting factors. Often these factors can only positively affect transcription if they physically interact either directly or indirectly through additional proteins with the basal transcriptional machinery. Understanding the physical constraints that determine functional cooperativity is essential for us to be able to model, predict, and engineer genetic control systems. These constraints generally are not rigid, but allow for variability in the arrangement of sites in functional complexes and subsequently there is variability in the stability of the complexes. Here, we have introduced a way to include orientation and order into the information theoretic description of pattern recognition at the promoter. This combined with weight matrix based binding models <a href="#pone.0001199-Schneider2">[20]</a> and spacing constraints <a href="#pone.0001199-Shultzaberger2">[9]</a>, <a href="#pone.0001199-Shultzaberger3">[10]</a> gives us quantitative tools to model the sequence basis of eukaryotic transcriptional regulation.</p>
<a id="article1.body1.sec4.p2" name="article1.body1.sec4.p2"></a><p>The simplest constraint of Met4 coordination to define is the ordering of the sites within the complex. For Met4, our model matches microarray data poorly when the order is Met31-Cbf1-gene start, but matches well with the order Cbf1-Met31-gene start (<a href="#pone-0001199-g002">Fig. 2</a>). This is consistent with experimentally determined ordering constraints <a href="#pone.0001199-Chiang2">[32]</a>. These results suggest that the Met4-Cbf1 binding surface is distinct from the Met4-Met31 surface, and that the Met4-TFIID binding surface is closer to the Met4-Met31 surface, placing the Met4-TFIID binding surface near the 3′ edge of the complex. Domain mapping from yeast two-hybrid experiments identified several protein interaction domains on Met4 <a href="#pone.0001199-Blaiseau1">[4]</a>. The transcriptional activation domain (residues 95–144) is closer to the Met31 interaction domain (residues 374–403) than the Cbf1/Met28 interaction domain (residues 616–666) in one-dimension, but these domains are far apart, so their relative positioning in the native form of Met4 could be different. Based on our findings, we suggest that the relative positioning is the same.</p>
<a id="article1.body1.sec4.p3" name="article1.body1.sec4.p3"></a><p>It appears as though either orientation of Met31 can be used within the docking complex (<a href="#pone-0001199-g002">Fig. 2</a>). Stabilization of Met4 has been shown when Met31 has the inverted orientation <a href="#pone.0001199-Blaiseau1">[4]</a>. For Met31 to be able to stabilize Met4 with either orientation, it must either have two Met4 interaction surfaces, or it has a centrally located interaction surface that is accessible no matter what orientation it binds (<em>i.e.</em> flexible). Interestingly, the top 5 genes in our ranking have an inverted Met31 site (<a href="#pone-0001199-g006">Fig. 6</a>). The total information of these sites might be high because stronger Cbf1 and Met31 sites may be necessary to compensate for the strain of the inverted orientation, but could decrease once we take into account the orientation surprisal.</p>
<a id="article1.body1.sec4.p4" name="article1.body1.sec4.p4"></a><p>The maximum distance between the Met4 docking site, as measured to the zero coordinate of the Met31 site, and the gene start is 450 bases. As the TFIID binding site is not at the gene start, this distance is farther than the maximum allowed distance between Met4 and the polymerase. It is difficult to determine the distance of the Met4 docking complex to the transcriptional start since the starts have not been biochemically proven, and computationally it is difficult to predict transcription initiation because of the varied modes of initiation by the polymerase <a href="#pone.0001199-Lee1">[2]</a>. Basehoar <em>et al.</em> found an enrichment of TATAs between 50 and 200 bases upstream of the translational start <a href="#pone.0001199-Basehoar1">[33]</a>. This could explain why we did not observe any sites within 100 bases of the gene start.</p>
<a id="article1.body1.sec4.p5" name="article1.body1.sec4.p5"></a><p>The spacing range between the Cbf1 and Met31 site is 9 to 68 bases, as determined in <a href="#pone-0001199-g004">Fig. 4</a>. The minimum spacing of 9 bases may be an under-estimate. There is a strong secondary Cbf1 site 36 bases upstream of the Met31 site in the promoter of Sam1, which may be the actual functioning site. A similar strong secondary site is also seen for YER080W, which had a spacing of 10 bases. This is not seen for GSH1, which had a spacing of 12 bases.</p>
<a id="article1.body1.sec4.p6" name="article1.body1.sec4.p6"></a><p>A minimum spacing of 12 would be consistent with our observed optimal helical phasing of -13.86 bases (<a href="#pone-0001199-g005">Fig. 5</a>). This would place the closest Cbf1 site almost exactly on the same face as its respective Met31 site, one helical turn away. A maximum spacing of 68 bases would correspond to 6 helical turns according to our phasing. The relatively high positioning of these spacings on the helical accessibility curve suggests that docking of Met4 may be dependent upon the helical phasing of DNA.</p>
<a id="article1.body1.sec4.p7" name="article1.body1.sec4.p7"></a><p>The experimentally determined range by Chiang <em>et al.</em> was 21 to 53 bases according to our numbering system <a href="#pone.0001199-Chiang2">[32]</a>. Unfortunately, spacings as large as 68 bases were not tested experimentally. The experimentally determined minimum spacing of 21 is much larger than the minimum we found here. Interestingly, only the “inverted” orientation of Met31 was tested, whereas the shortest distance in this paper corresponds to a Met31 site with the opposite orientation. If helical phasing of the sites is important, then the orientation of Met31 may be more constrained at shorter distances, and this may account for the disparity between the experimentally and computationally determined minimums.</p>
<a id="article1.body1.sec4.p8" name="article1.body1.sec4.p8"></a><p>When the constraints inferred from our analysis were imposed on the cooperative binding of Cbf1 and Met31, our ability to predict Met4 regulation was high. Of the top 23 ranked genes in <em>S. cerevisiae</em> (according to our model), 19 had an expected microarray expression pattern for Met4 regulation. Many of the sites had also been previously characterized as sulfur utilization genes (<a href="#pone-0001199-g006">Fig. 6</a>). The 2 most striking anomalous genes in the top 23 (Reb1 and Gar1) both had Met31 sites with a “T” instead of “G” at position +1 (data not shown), suggesting that this position may be weighted more strongly in a more refined Met31 model. Additionally, nucleosomes could play a large inhibitory role against spurious combinations of sites, which our model does not account for.</p>
<a id="article1.body1.sec4.p9" name="article1.body1.sec4.p9"></a><p>When the microarray data from experiments that affected Met4 binding were directly compared to our information evaluation of each gene (<a href="#pone-0001199-g007">Fig. 7</a>), we saw that almost all genes with Met4 binding sites above 14 bits of information have the expected expression change. This suggests that our approach is giving some reasonable estimate of the energetics of Met4 binding, with a clearly defined threshold for functional binding sites. Presumably, genes that do not have a strong Met4 site, but have the expected microarray data are indirectly regulated. Interestingly the strengths of the Met4 sites are not mainly determined by the strength of Met31 or Cbf1, but by the sum of these sites. This suggests that for cooperatively acting binding sites, a decrease in strength for one site can be compensated for by an increase in strength of the other. Compensation for a decrease in the strength of one binding site by increasing the affinity for another site has been shown experimentally for activation of Pol II by the Epstein-Barr virus protein ZEBRA <a href="#pone.0001199-Lehman1">[34]</a>.</p>
<a id="article1.body1.sec4.p10" name="article1.body1.sec4.p10"></a><p>We cannot determine the individual spacing effect on binding because we have so few sites covering a large spacing range. If we did have these individual effects, we would expect to see a slight improvement in our ability to predict expression. The fact that we did so well in predicting Met4 sites by just taking into account the strength of the sites is surprising to us. In bacteria, transcription factors interact at shorter distances, and the effect of spacing on stability is greater (since it is more difficult to bend a short piece of DNA than a large one) <a href="#pone.0001199-Shultzaberger3">[10]</a>. For coordinated binding of Met4, the summed affinities of Cbf1 and Met31 appeared to dominate the stability of the Met4 complex, suggesting that the energetic penalties associated with spacing are considerably less for this system. Experimental testing of multiple spacings between Cbf1 and Met31 suggested that spacing had a little effect on Met4 regulation <a href="#pone.0001199-Chiang2">[32]</a>. A decrease in the effect of spacing on stability could be due to increasing the flexibility between the activation domain and the DNA-binding domain of the transcription factors, by increasing the distance between functioning cooperatively acting binding sites, or by increasing flexibility in the coordinating protein.</p>
<a id="article1.body1.sec4.p11" name="article1.body1.sec4.p11"></a><p>We could have determined these physical constraints by clustering co-regulated genes and training the rules of binding for their regulators. The drawback of this approach is that it is not obvious which genes are directly and indirectly regulated, and a given gene may or may not have a binding site. Our approach selects only for genes that are directly regulated, and does not exclude sites that have poor expression data due to experimental error. We are also optimizing our model against all genes in the genome, so we are selecting for a model that represents Met4 binding well, in that it can identify a small subset of sites from all sites in the genome. Presumably the optimal binding site, based on the flexible information theory approach, is the most stable site and the easiest to crystalize. These results could be used to guide crystallographic experiments.</p>
<a id="article1.body1.sec4.p12" name="article1.body1.sec4.p12"></a><p>The information content of a given DNA-binding protein (<em>R<sub>sequence</sub></em>) is a function of the variability within its binding targets <a href="#pone.0001199-Schneider4">[24]</a>. A more stringent binder would have a higher information content, since the variability in its binding targets would be smaller. To be able to distinguish <strong>γ</strong> binding sites within a random DNA of some length <strong>G</strong>, those sites must have an <em>R<sub>frequency</sub></em> = -<em>log</em><sub>2</sub>(γ/<em>G</em>) bits of information to be identified <a href="#pone.0001199-Schneider4">[24]</a>. It has been shown for many systems that <em>R<sub>sequence</sub></em> converges to <em>R<sub>frequency</sub></em> <a href="#pone.0001199-Schneider4">[24]</a>, <a href="#pone.0001199-Schneider7">[35]</a>. This suggests that if the size of the genome increases and the number of binding sites remains constant, the information of those sites would have to increase in order to be distinguishable.</p>
<a id="article1.body1.sec4.p13" name="article1.body1.sec4.p13"></a><p>As eukaryotic genomes are generally larger than prokaryotic genomes, the amount of information needed to identify <strong>γ</strong> sites would have to be greater. This can be achieved either by increasing the information content of a single factor, or by using multiple factors combinatorially.</p>
<a id="article1.body1.sec4.p14" name="article1.body1.sec4.p14"></a><p>Assuming no individual spacing or orientation preferences, the information for this system would be 12.9+11.9−5.9−1 = 17.9 bits according to equation (4). This would correspond to 1 site every 2<sup>17.9</sup> = 2.45×10<sup>5</sup> bases, or about 104 times in the <em>S. cerevisiae</em> genome of length 12.8 MB. Our calculation is the number of sites in 2× the genome length, since the complex could associate with either strand. This is a reasonable number of genes according to known sulfur assimilation genes (&gt;20 genes) <a href="#pone.0001199-Thomas1">[5]</a>, the number of predicted regulated genes based on expression difference due to Cd<sup>2+</sup> treatment (66 genes) <a href="#pone.0001199-Fauchon1">[27]</a>, and multiple sites per gene as seen in several cases. This suggests that like single acting transcription factors, the information contained in combinatorial binders is related to the <em>R<sub>frequency</sub></em> for that system <a href="#pone.0001199-Schneider4">[24]</a>. Others have suggested that this relationship will be maintained for cooperatively acting factors <a href="#pone.0001199-GuhaThakurta1">[36]</a>. Interestingly, this information is distributed through individual binding components, as well as the spacing between components, and if one component changed, the others would have to compensate accordingly. This is a complicated process, since Cbf1 can also function independently of Met4 and Met31 <a href="#pone.0001199-Kent1">[31]</a>.</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>We would like to thank Tom Schneider, Richard Lusk, James Fraser and Virgil Rhodius for comments and useful discussion.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: DC AM RS. Performed the experiments: RS. Analyzed the data: DC RS. Contributed reagents/materials/analysis tools: RS. Wrote the paper: ME RS.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0001199-Levine1" id="pone.0001199-Levine1"></a>Levine M, Tjian R (2003) Transcription regulation and animal diversity. Nature  424: 147–151.  <ul class="find" data-citedArticleID="1019724" data-doi="10.1038/nature01763"><li><a href="http://dx.doi.org/10.1038/nature01763" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Transcription+regulation+and+animal+diversity." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Transcription+regulation+and+animal+diversity.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0001199-Lee1" id="pone.0001199-Lee1"></a>Lee TI, Young RA (2000) Transcription of eukaryotic protein-coding genes. Annu Rev Genet  34: 77–137.  <ul class="find" data-citedArticleID="1019718"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Transcription+of+eukaryotic+protein-coding+genes.&amp;auth=&amp;atitle=Transcription+of+eukaryotic+protein-coding+genes." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Transcription+of+eukaryotic+protein-coding+genes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Transcription+of+eukaryotic+protein-coding+genes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0001199-Lee2" id="pone.0001199-Lee2"></a>Lee TI, Rinaldi NJ, Robert F, Odom DT, Bar-Joseph Z, et al.  (2002) Transcriptional regulatory networks in Saccharomyces cerevisiae. Science  298: 799–804.  <ul class="find" data-citedArticleID="1019720" data-doi="10.1126/science.1075090"><li><a href="http://dx.doi.org/10.1126/science.1075090" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Transcriptional+regulatory+networks+in+Saccharomyces+cerevisiae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Transcriptional+regulatory+networks+in+Saccharomyces+cerevisiae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0001199-Blaiseau1" id="pone.0001199-Blaiseau1"></a>Blaiseau PL, Thomas D (1998) Multiple transcriptional activation complexes tether the yeast activator Met4 to DNA. EMBO J  17: 6327–6336.  <ul class="find" data-citedArticleID="1019694" data-doi="10.1093/emboj/17.21.6327"><li><a href="http://dx.doi.org/10.1093/emboj/17.21.6327" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Multiple+transcriptional+activation+complexes+tether+the+yeast+activator+Met4+to+DNA." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Multiple+transcriptional+activation+complexes+tether+the+yeast+activator+Met4+to+DNA.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0001199-Thomas1" id="pone.0001199-Thomas1"></a>Thomas D, Surdin-Kerjan Y (1997) Metabolism of sulfur amino acids in Saccharomyces cerevisiae. Microbiol Mol Biol Rev  61: 503–532.  <ul class="find" data-citedArticleID="1019752"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Metabolism+of+sulfur+amino+acids+in+Saccharomyces+cerevisiae.&amp;auth=&amp;atitle=Metabolism+of+sulfur+amino+acids+in+Saccharomyces+cerevisiae." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Metabolism+of+sulfur+amino+acids+in+Saccharomyces+cerevisiae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Metabolism+of+sulfur+amino+acids+in+Saccharomyces+cerevisiae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0001199-Chiang1" id="pone.0001199-Chiang1"></a>Chiang DY, Moses AM, Kellis M, Lander ES, Eisen MB (2003) Phylogenetically and spatially conserved word pairs associated with gene-expression changes in yeasts. Genome Biol  4: R43.  <ul class="find" data-citedArticleID="1019700" data-doi="10.1145/640075.640085"><li><a href="http://dx.doi.org/10.1145/640075.640085" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Phylogenetically+and+spatially+conserved+word+pairs+associated+with+gene-expression+changes+in+yeasts." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Phylogenetically+and+spatially+conserved+word+pairs+associated+with+gene-expression+changes+in+yeasts.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0001199-Shultzaberger1" id="pone.0001199-Shultzaberger1"></a>Shultzaberger RK, Roberts LR, Lyakhov IG, Sidorov IA, Stephen AG, et al.  (2007) Correlation between binding rate constants and individual information of <em>E. coli</em> Fis binding sites. Nucleic Acids Res  35: 5275–5283.  <a href="http://www.ccrnp.ncifcrf.gov/toms/paper/fisbc/">http://www.ccrnp.ncifcrf.gov/toms/paper/​fisbc/</a>, <a href="http://dx.doi.org/10.1093/nar/gkm471">http://dx.doi.org/10.1093/nar/gkm471</a>.  <ul class="find" data-citedArticleID="1019746" data-doi="10.1093/nar/gkm471"><li><a href="http://dx.doi.org/10.1093/nar/gkm471" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Correlation+between+binding+rate+constants+and+individual+information+of+E.+coli+Fis+binding+sites." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Correlation+between+binding+rate+constants+and+individual+information+of+E.+coli+Fis+binding+sites.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0001199-Udalova1" id="pone.0001199-Udalova1"></a>Udalova IA, Mott R, Field D, Kwiatkowski D (2002) Quantitative prediction of NF-κB DNA-protein interactions. Proc Natl Acad Sci USA  99: 8167–8172.  <ul class="find" data-citedArticleID="1019758" data-doi="10.1073/pnas.102674699"><li><a href="http://dx.doi.org/10.1073/pnas.102674699" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Quantitative+prediction+of+NF-%CE%BAB+DNA-protein+interactions." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Quantitative+prediction+of+NF-%CE%BAB+DNA-protein+interactions.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0001199-Shultzaberger2" id="pone.0001199-Shultzaberger2"></a>Shultzaberger RK, Bucheimer RE, Rudd KE, Schneider TD (2001) Anatomy of <em>Escherichia coli</em> Ribosome Binding Sites. J Mol Biol  313: 215–228.  <a href="http://www.ccrnp.ncifcrf.gov/toms/paper/flexrbs/">http://www.ccrnp.ncifcrf.gov/toms/paper/​flexrbs/</a>.  <ul class="find" data-citedArticleID="1019748" data-doi="10.1006/jmbi.2001.5040"><li><a href="http://dx.doi.org/10.1006/jmbi.2001.5040" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Anatomy+of+Escherichia+coli+Ribosome+Binding+Sites." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Anatomy+of+Escherichia+coli+Ribosome+Binding+Sites.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0001199-Shultzaberger3" id="pone.0001199-Shultzaberger3"></a>Shultzaberger RK, Chen Z, Lewis KA, Schneider TD (2007) Anatomy of <em>Escherichia coli</em> σ<sup>70</sup> promoters. Nucleic Acids Res  35: 771–788.  <a href="http://www.ccrnp.ncifcrf.gov/toms/paper/flexprom/">http://www.ccrnp.ncifcrf.gov/toms/paper/​flexprom/</a>.  <ul class="find" data-citedArticleID="1019750" data-doi="10.1093/nar/gkl956"><li><a href="http://dx.doi.org/10.1093/nar/gkl956" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Anatomy+of+Escherichia+coli+%CF%8370+promoters." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Anatomy+of+Escherichia+coli+%CF%8370+promoters.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0001199-Chen1" id="pone.0001199-Chen1"></a>Chen H, Bjerknes M, Kumar R, Jay E (1994) Determination of the optimal aligned spacing between the Shine-Dalgarno sequence and the translation initiation codon of <em>Escherichia coli</em> mRNAs. Nucleic Acids Res  22: 4953–4957.  <ul class="find" data-citedArticleID="1019698" data-doi="10.1093/nar/22.23.4953"><li><a href="http://dx.doi.org/10.1093/nar/22.23.4953" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Determination+of+the+optimal+aligned+spacing+between+the+Shine-Dalgarno+sequence+and+the+translation+initiation+codon+of+Escherichia+coli+mRNAs." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Determination+of+the+optimal+aligned+spacing+between+the+Shine-Dalgarno+sequence+and+the+translation+initiation+codon+of+Escherichia+coli+mRNAs.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0001199-RinkeAppel1" id="pone.0001199-RinkeAppel1"></a>Rinke-Appel J, Junke N, Brimacombe R, Lavrik I, Dokudovskaya S, et al.  (1994) Contacts between 16S ribosomal RNA and mRNA, within the spacer region separating the AUG initiator codon and the Shine-Dalgarno sequence; a site-directed cross-linking study. Nucleic Acids Res  22: 3018–3025.  <ul class="find" data-citedArticleID="1019730" data-doi="10.1093/nar/22.15.3018"><li><a href="http://dx.doi.org/10.1093/nar/22.15.3018" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Contacts+between+16S+ribosomal+RNA+and+mRNA%2C+within+the+spacer+region+separating+the+AUG+initiator+codon+and+the+Shine-Dalgarno+sequence%3B+a+site-directed+cross-linking+study." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Contacts+between+16S+ribosomal+RNA+and+mRNA%2C+within+the+spacer+region+separating+the+AUG+initiator+codon+and+the+Shine-Dalgarno+sequence%3B+a+site-directed+cross-linking+study.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0001199-Hawley1" id="pone.0001199-Hawley1"></a>Hawley DK, McClure WR (1983) Compilation and analysis of <em>Escherichia coli</em> promoter DNA sequences. Nucleic Acids Res  11: 2237–2255.  <ul class="find" data-citedArticleID="1019712" data-doi="10.1093/nar/11.8.2237"><li><a href="http://dx.doi.org/10.1093/nar/11.8.2237" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Compilation+and+analysis+of+Escherichia+coli+promoter+DNA+sequences." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Compilation+and+analysis+of+Escherichia+coli+promoter+DNA+sequences.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0001199-McClure1" id="pone.0001199-McClure1"></a>McClure WR (1985) Mechanism and control of transcription initiation in prokaryotes. Annu Rev Biochem  54: 171–204.  <ul class="find" data-citedArticleID="1019726" data-doi="10.1146/annurev.bi.54.070185.001131"><li><a href="http://dx.doi.org/10.1146/annurev.bi.54.070185.001131" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mechanism+and+control+of+transcription+initiation+in+prokaryotes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mechanism+and+control+of+transcription+initiation+in+prokaryotes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0001199-Thomas2" id="pone.0001199-Thomas2"></a>Thomas D, Jacquemin I, Surdin-Kerjan Y (1992) MET4, a leucine zipper protein, and centromere-binding factor 1 are both required for transcriptional activation of sulfur metabolism in Saccharomyces cerevisiae. Mol Cell Biol  12: 1719–1727.  <ul class="find" data-citedArticleID="1019754"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=MET4%2C+a+leucine+zipper+protein%2C+and+centromere-binding+factor+1+are+both+required+for+transcriptional+activation+of+sulfur+metabolism+in+Saccharomyces+cerevisiae.&amp;auth=&amp;atitle=MET4%2C+a+leucine+zipper+protein%2C+and+centromere-binding+factor+1+are+both+required+for+transcriptional+activation+of+sulfur+metabolism+in+Saccharomyces+cerevisiae." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=MET4%2C+a+leucine+zipper+protein%2C+and+centromere-binding+factor+1+are+both+required+for+transcriptional+activation+of+sulfur+metabolism+in+Saccharomyces+cerevisiae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22MET4%2C+a+leucine+zipper+protein%2C+and+centromere-binding+factor+1+are+both+required+for+transcriptional+activation+of+sulfur+metabolism+in+Saccharomyces+cerevisiae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0001199-Kuras1" id="pone.0001199-Kuras1"></a>Kuras L, Barbey R, Thomas D (1997) Assembly of a bZIP-bHLH transcription activation complex: formation of the yeast Cbf1-Met4-Met28 complex is regulated through Met28 stimulation of Cbf1 DNA binding. EMBO J  16: 2441–2451.  <ul class="find" data-citedArticleID="1019716" data-doi="10.1093/emboj/16.9.2441"><li><a href="http://dx.doi.org/10.1093/emboj/16.9.2441" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Assembly+of+a+bZIP-bHLH+transcription+activation+complex%3A+formation+of+the+yeast+Cbf1-Met4-Met28+complex+is+regulated+through+Met28+stimulation+of+Cbf1+DNA+binding." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Assembly+of+a+bZIP-bHLH+transcription+activation+complex%3A+formation+of+the+yeast+Cbf1-Met4-Met28+complex+is+regulated+through+Met28+stimulation+of+Cbf1+DNA+binding.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0001199-Blaiseau2" id="pone.0001199-Blaiseau2"></a>Blaiseau PL, Isnard AD, Surdin-Kerjan Y, Thomas D (1997) Met31p and Met32p, two related zinc finger proteins, are involved in transcriptional regulation of yeast sulfur amino acid metabolism. Mol Cell Biol  17: 3640–3648.  <ul class="find" data-citedArticleID="1019696"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Met31p+and+Met32p%2C+two+related+zinc+finger+proteins%2C+are+involved+in+transcriptional+regulation+of+yeast+sulfur+amino+acid+metabolism.&amp;auth=&amp;atitle=Met31p+and+Met32p%2C+two+related+zinc+finger+proteins%2C+are+involved+in+transcriptional+regulation+of+yeast+sulfur+amino+acid+metabolism." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Met31p+and+Met32p%2C+two+related+zinc+finger+proteins%2C+are+involved+in+transcriptional+regulation+of+yeast+sulfur+amino+acid+metabolism." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Met31p+and+Met32p%2C+two+related+zinc+finger+proteins%2C+are+involved+in+transcriptional+regulation+of+yeast+sulfur+amino+acid+metabolism.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0001199-Wieland1" id="pone.0001199-Wieland1"></a>Wieland G, Hemmerich P, Koch M, Stoyan T, Hegemann J, et al.  (2001) Determination of the binding constants of the centromere protein Cbf1 to all 16 centromere DNAs of Saccharomyces cerevisiae. Nucleic Acids Res  29: 1054–1060.  <ul class="find" data-citedArticleID="1019760" data-doi="10.1093/nar/29.5.1054"><li><a href="http://dx.doi.org/10.1093/nar/29.5.1054" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Determination+of+the+binding+constants+of+the+centromere+protein+Cbf1+to+all+16+centromere+DNAs+of+Saccharomyces+cerevisiae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Determination+of+the+binding+constants+of+the+centromere+protein+Cbf1+to+all+16+centromere+DNAs+of+Saccharomyces+cerevisiae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0001199-Schneider1" id="pone.0001199-Schneider1"></a>Schneider TD, Stephens RM (1990) Sequence logos: A new way to display consensus sequences. Nucleic Acids Res  18: 6097–6100.  <a href="http://www.ccrnp.ncifcrf.gov/toms/paper/logopaper/">http://www.ccrnp.ncifcrf.gov/toms/paper/​logopaper/</a>.  <ul class="find" data-citedArticleID="1019732" data-doi="10.1093/nar/18.20.6097"><li><a href="http://dx.doi.org/10.1093/nar/18.20.6097" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Sequence+logos%3A+A+new+way+to+display+consensus+sequences." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Sequence+logos%3A+A+new+way+to+display+consensus+sequences.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0001199-Schneider2" id="pone.0001199-Schneider2"></a>Schneider TD (1997) Information content of individual genetic sequences. J Theor Biol  189: 427–441.  <a href="http://www.ccrnp.ncifcrf.gov/toms/paper/ri/">http://www.ccrnp.ncifcrf.gov/toms/paper/​ri/</a>.  <ul class="find" data-citedArticleID="1019734" data-doi="10.1006/jtbi.1997.0540"><li><a href="http://dx.doi.org/10.1006/jtbi.1997.0540" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Information+content+of+individual+genetic+sequences." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Information+content+of+individual+genetic+sequences.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0001199-Schneider3" id="pone.0001199-Schneider3"></a>Schneider TD (1996) Reading of DNA sequence logos: Prediction of major groove binding by information theory. Meth Enzym  274: 445–455.  <a href="http://www.ccrnp.ncifcrf.gov/toms/paper/oxyr/">http://www.ccrnp.ncifcrf.gov/toms/paper/​oxyr/</a>.  <ul class="find" data-citedArticleID="1019736" data-doi="10.1016/s0076-6879(96)74036-3"><li><a href="http://dx.doi.org/10.1016/s0076-6879(96)74036-3" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Reading+of+DNA+sequence+logos%3A+Prediction+of+major+groove+binding+by+information+theory." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Reading+of+DNA+sequence+logos%3A+Prediction+of+major+groove+binding+by+information+theory.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0001199-Bailey1" id="pone.0001199-Bailey1"></a>Bailey TL, Elkan C (1995) The value of prior knowledge in discovering motifs with MEME. Proc Int Conf Intell Syst Mol Biol  3: 21–29.  <ul class="find" data-citedArticleID="1019690"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=The+value+of+prior+knowledge+in+discovering+motifs+with+MEME.&amp;auth=&amp;atitle=The+value+of+prior+knowledge+in+discovering+motifs+with+MEME." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+value+of+prior+knowledge+in+discovering+motifs+with+MEME." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+value+of+prior+knowledge+in+discovering+motifs+with+MEME.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0001199-Tribus1" id="pone.0001199-Tribus1"></a>Tribus M (1961) Thermostatics and Thermodynamics. Princeton, N. J.: D. van Nostrand Company, Inc.   <ul class="find-nolinks"></ul></li><li><span class="label">24.
              </span><a name="pone.0001199-Schneider4" id="pone.0001199-Schneider4"></a>Schneider TD, Stormo GD, Gold L, Ehrenfeucht A (1986) Information content of binding sites on nucleotide sequences. J Mol Biol  188: 415–431.  <a href="http://www.ccrnp.ncifcrf.gov/toms/paper/schneider1986/">http://www.ccrnp.ncifcrf.gov/toms/paper/​schneider1986/</a>.  <ul class="find" data-citedArticleID="1019738" data-doi="10.1016/0022-2836(86)90165-8"><li><a href="http://dx.doi.org/10.1016/0022-2836(86)90165-8" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Information+content+of+binding+sites+on+nucleotide+sequences." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Information+content+of+binding+sites+on+nucleotide+sequences.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0001199-Schneider5" id="pone.0001199-Schneider5"></a>Schneider TD (1991) Theory of molecular machines. II. Energy dissipation from molecular machines. J Theor Biol  148: 125–137.  <a href="http://www.ccrnp.ncifcrf.gov/toms/paper/edmm/">http://www.ccrnp.ncifcrf.gov/toms/paper/​edmm/</a>.  <ul class="find" data-citedArticleID="1019740" data-doi="10.1016/s0022-5193(05)80467-9"><li><a href="http://dx.doi.org/10.1016/s0022-5193(05)80467-9" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Theory+of+molecular+machines.+II.+Energy+dissipation+from+molecular+machines." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Theory+of+molecular+machines.+II.+Energy+dissipation+from+molecular+machines.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0001199-Gasch1" id="pone.0001199-Gasch1"></a>Gasch AP, Spellman PT, Kao CM, Carmel-Harel O, Eisen MB, et al.  (2000) Genomic expression programs in the response of yeast cells to environmental changes. Mol Biol Cell  11: 4241–4257.  <ul class="find" data-citedArticleID="1019708" data-doi="10.1091/mbc.11.12.4241"><li><a href="http://dx.doi.org/10.1091/mbc.11.12.4241" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genomic+expression+programs+in+the+response+of+yeast+cells+to+environmental+changes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genomic+expression+programs+in+the+response+of+yeast+cells+to+environmental+changes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0001199-Fauchon1" id="pone.0001199-Fauchon1"></a>Fauchon M, Lagniel G, Aude JC, Lombardia L, Soularue P, et al.  (2002) Sulfur sparing in the yeast proteome in response to sulfur demand. Mol Cell  9: 713–723.  <ul class="find" data-citedArticleID="1019706" data-doi="10.1016/s1097-2765(02)00500-2"><li><a href="http://dx.doi.org/10.1016/s1097-2765(02)00500-2" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Sulfur+sparing+in+the+yeast+proteome+in+response+to+sulfur+demand." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Sulfur+sparing+in+the+yeast+proteome+in+response+to+sulfur+demand.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0001199-Eisen1" id="pone.0001199-Eisen1"></a>Eisen MB, Spellman PT, Brown PO, Botstein D (1998) Cluster analysis and display of genome-wide expression patterns. Proc Natl Acad Sci U S A  95: 14863–14868.  <ul class="find" data-citedArticleID="1019704" data-doi="10.1073/pnas.95.25.14863"><li><a href="http://dx.doi.org/10.1073/pnas.95.25.14863" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Cluster+analysis+and+display+of+genome-wide+expression+patterns." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Cluster+analysis+and+display+of+genome-wide+expression+patterns.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0001199-Schneider6" id="pone.0001199-Schneider6"></a>Schneider TD (2001) Strong minor groove base conservation in sequence logos implies DNA distortion or base flipping during replication and transcription initiation. Nucleic Acids Res  29: 4881–4891.  <a href="http://www.ccrnp.ncifcrf.gov/toms/paper/baseflip/">http://www.ccrnp.ncifcrf.gov/toms/paper/​baseflip/</a>.  <ul class="find" data-citedArticleID="1019742" data-doi="10.1093/nar/29.23.4881"><li><a href="http://dx.doi.org/10.1093/nar/29.23.4881" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Strong+minor+groove+base+conservation+in+sequence+logos+implies+DNA+distortion+or+base+flipping+during+replication+and+transcription+initiation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Strong+minor+groove+base+conservation+in+sequence+logos+implies+DNA+distortion+or+base+flipping+during+replication+and+transcription+initiation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0001199-Niedenthal1" id="pone.0001199-Niedenthal1"></a>Niedenthal RK, Sen-Gupta M, Wilmen A, Hegemann JH (1993) Cpf1 protein induced bending of yeast centromere DNA element I. Nucleic Acids Res  21: 4726–4733.  <ul class="find" data-citedArticleID="1019728" data-doi="10.1093/nar/21.20.4726"><li><a href="http://dx.doi.org/10.1093/nar/21.20.4726" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Cpf1+protein+induced+bending+of+yeast+centromere+DNA+element+I." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Cpf1+protein+induced+bending+of+yeast+centromere+DNA+element+I.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0001199-Kent1" id="pone.0001199-Kent1"></a>Kent NA, Eibert SM, Mellor J (2004) Cbf1p is required for chromatin remodeling at promoter-proximal CACGTG motifs in yeast. J Biol Chem  279: 27116–27123.  <ul class="find" data-citedArticleID="1019714" data-doi="10.1074/jbc.m403818200"><li><a href="http://dx.doi.org/10.1074/jbc.m403818200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Cbf1p+is+required+for+chromatin+remodeling+at+promoter-proximal+CACGTG+motifs+in+yeast." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Cbf1p+is+required+for+chromatin+remodeling+at+promoter-proximal+CACGTG+motifs+in+yeast.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0001199-Chiang2" id="pone.0001199-Chiang2"></a>Chiang DY, Nix DA, Shultzaberger RK, Gasch AP, Eisen MB (2006) Flexible promoter architecture requirements for coactivator recruitment. BMC Mol Biol  7: 16.  <ul class="find" data-citedArticleID="1019702" data-doi="10.1186/1471-2199-7-16"><li><a href="http://dx.doi.org/10.1186/1471-2199-7-16" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Flexible+promoter+architecture+requirements+for+coactivator+recruitment." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Flexible+promoter+architecture+requirements+for+coactivator+recruitment.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0001199-Basehoar1" id="pone.0001199-Basehoar1"></a>Basehoar AD, Zanton SJ, Pugh BF (2004) Identification and distinct regulation of yeast TATA box-containing genes. Cell  116: 699–709.  <ul class="find" data-citedArticleID="1019692" data-doi="10.1016/s0092-8674(04)00205-3"><li><a href="http://dx.doi.org/10.1016/s0092-8674(04)00205-3" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Identification+and+distinct+regulation+of+yeast+TATA+box-containing+genes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Identification+and+distinct+regulation+of+yeast+TATA+box-containing+genes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0001199-Lehman1" id="pone.0001199-Lehman1"></a>Lehman AM, Ellwood KB, Middleton BE, Carey M (1998) Compensatory energetic relationships between upstream activators and the RNA polymerase II general transcription machinery. J Biol Chem  273: 932–939.  <ul class="find" data-citedArticleID="1019722" data-doi="10.1074/jbc.273.2.932"><li><a href="http://dx.doi.org/10.1074/jbc.273.2.932" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Compensatory+energetic+relationships+between+upstream+activators+and+the+RNA+polymerase+II+general+transcription+machinery." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Compensatory+energetic+relationships+between+upstream+activators+and+the+RNA+polymerase+II+general+transcription+machinery.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0001199-Schneider7" id="pone.0001199-Schneider7"></a>Schneider TD (2000) Evolution of biological information. Nucleic Acids Res  28: 2794–2799.  <a href="http://www.ccrnp.ncifcrf.gov/toms/paper/ev/">http://www.ccrnp.ncifcrf.gov/toms/paper/​ev/</a>.  <ul class="find" data-citedArticleID="1019744" data-doi="10.1093/nar/28.14.2794"><li><a href="http://dx.doi.org/10.1093/nar/28.14.2794" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evolution+of+biological+information." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evolution+of+biological+information.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">36.
              </span><a name="pone.0001199-GuhaThakurta1" id="pone.0001199-GuhaThakurta1"></a>GuhaThakurta D, Stormo GD (2001) Identifying target sites for cooperatively binding factors. Bioinformatics  17: 608–621.  <ul class="find" data-citedArticleID="1019710" data-doi="10.1093/bioinformatics/17.7.608"><li><a href="http://dx.doi.org/10.1093/bioinformatics/17.7.608" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Identifying+target+sites+for+cooperatively+binding+factors." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Identifying+target+sites+for+cooperatively+binding+factors.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001199.XML" value="90121"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001199.PDF" value="812516"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001199.e001.PNG" value="11296"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001199.e001.TIF" value="28260"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001199.e002.PNG" value="9753"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001199.e002.TIF" value="26084"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001199.e003.PNG" value="10052"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001199.e003.TIF" value="26224"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001199.e004.PNG" value="15698"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001199.e004.TIF" value="32384"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001199.g001.PNG_L" value="230712"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001199.g001.PNG_M" value="108504"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001199.g001.PNG_S" value="15249"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001199.g001.TIF" value="454302"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001199.g001.PNG_I" value="78026"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001199.g002.PNG_L" value="329475"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001199.g002.PNG_M" value="128472"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001199.g002.PNG_S" value="18045"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001199.g002.TIF" value="1060314"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001199.g002.PNG_I" value="102829"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001199.g003.PNG_L" value="259190"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001199.g003.PNG_M" value="92293"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001199.g003.PNG_S" value="11402"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001199.g003.TIF" value="700950"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001199.g003.PNG_I" value="57989"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001199.g004.PNG_L" value="62414"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001199.g004.PNG_M" value="86909"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001199.g004.PNG_S" value="11495"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001199.g004.TIF" value="397244"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001199.g004.PNG_I" value="36655"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001199.g005.PNG_L" value="566489"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001199.g005.PNG_M" value="96204"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001199.g005.PNG_S" value="10838"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001199.g005.TIF" value="1327220"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001199.g005.PNG_I" value="33371"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001199.g006.PNG_L" value="336381"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001199.g006.PNG_M" value="221603"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001199.g006.PNG_S" value="9615"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001199.g006.TIF" value="816696"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001199.g006.PNG_I" value="71508"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001199.e005.PNG" value="13836"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001199.e005.TIF" value="30212"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001199.g007.PNG_L" value="104555"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001199.g007.PNG_M" value="122895"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001199.g007.PNG_S" value="12656"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001199.g007.TIF" value="400466"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001199.g007.PNG_I" value="51045"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001199&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001199" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001199&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0001199&volume=&issue=&title=Determining Physical Constraints in Transcriptional Initiation Complexes Using DNA Sequence Analysis&author_name=Ryan%20K.%20Shultzaberger%2C%20Derek%20Y.%20Chiang%2C%20Alan%20M.%20Moses%2C%20Michael%20B.%20Eisen&start_page=1&end_page=10" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001199" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001199" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0001199" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001199&amp;t=Determining%20Physical%20Constraints%20in%20Transcriptional%20Initiation%20Complexes%20Using%20DNA%20Sequence%20Analysis" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001199&title=Determining%20Physical%20Constraints%20in%20Transcriptional%20Initiation%20Complexes%20Using%20DNA%20Sequence%20Analysis&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001199&amp;title=Determining%20Physical%20Constraints%20in%20Transcriptional%20Initiation%20Complexes%20Using%20DNA%20Sequence%20Analysis" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001199" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0001199&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Determining Physical Constraints in Transcriptional Initiation Complexes Using DNA Sequence Analysis';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0001199';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Determining%20Physical%20Constraints%20in%20Transcriptional%20Initiation%20Complexes%20Using%20DNA%20Sequence%20Analysis http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001199" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0001199" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">











          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Binding+analysis%22" title="Search for articles in the subject area:'Binding analysis'"><div class="flagText">Binding analysis</div></a>
              <div data-categoryid="19597" data-articleid="25230"
                   data-categoryname="Binding analysis"
                   class="flagImage" title="Flag 'Binding analysis' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22DNA+sequences%22" title="Search for articles in the subject area:'DNA sequences'"><div class="flagText">DNA sequences</div></a>
              <div data-categoryid="32003" data-articleid="25230"
                   data-categoryname="DNA sequences"
                   class="flagImage" title="Flag 'DNA sequences' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22DNA-binding+proteins%22" title="Search for articles in the subject area:'DNA-binding proteins'"><div class="flagText">DNA-binding proteins</div></a>
              <div data-categoryid="17441" data-articleid="25230"
                   data-categoryname="DNA-binding proteins"
                   class="flagImage" title="Flag 'DNA-binding proteins' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Gene+expression%22" title="Search for articles in the subject area:'Gene expression'"><div class="flagText">Gene expression</div></a>
              <div data-categoryid="17515" data-articleid="25230"
                   data-categoryname="Gene expression"
                   class="flagImage" title="Flag 'Gene expression' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Gene+regulation%22" title="Search for articles in the subject area:'Gene regulation'"><div class="flagText">Gene regulation</div></a>
              <div data-categoryid="17553" data-articleid="25230"
                   data-categoryname="Gene regulation"
                   class="flagImage" title="Flag 'Gene regulation' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Microarrays%22" title="Search for articles in the subject area:'Microarrays'"><div class="flagText">Microarrays</div></a>
              <div data-categoryid="17255" data-articleid="25230"
                   data-categoryname="Microarrays"
                   class="flagImage" title="Flag 'Microarrays' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Sulfur%22" title="Search for articles in the subject area:'Sulfur'"><div class="flagText">Sulfur</div></a>
              <div data-categoryid="20755" data-articleid="25230"
                   data-categoryname="Sulfur"
                   class="flagImage" title="Flag 'Sulfur' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Transcription+factors%22" title="Search for articles in the subject area:'Transcription factors'"><div class="flagText">Transcription factors</div></a>
              <div data-categoryid="17335" data-articleid="25230"
                   data-categoryname="Transcription factors"
                   class="flagImage" title="Flag 'Transcription factors' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=5801'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=3771'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=6679&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>

<div class="block sidebar-comments">
    <div class="header">
        <h3>Comments</h3>
    </div>
      <p><a href="/annotation/listThread.action?root=7349">Referee Comments: Referee 1 </a><br>Posted by PLoS_ONE_Group</p>
</div>

</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
