

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Accumulating Variation at Conserved Sites in Potyvirus Genomes Is Driven by Species Discovery and Affects Degenerate Primer Design</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0001586"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0001586" />

    <meta name="citation_title" content="Accumulating Variation at Conserved Sites in Potyvirus Genomes Is Driven by Species Discovery and Affects Degenerate Primer Design"/>
    <meta itemprop="name" content="Accumulating Variation at Conserved Sites in Potyvirus Genomes Is Driven by Species Discovery and Affects Degenerate Primer Design"/>

      <meta name="citation_author" content="Linda Zheng"/>
            <meta name="citation_author_institution" content="School of Botany and Zoology, Australian National University, Canberra, Australian Capital Territory, Australia"/>
      <meta name="citation_author" content="Paul J. Wayper"/>
            <meta name="citation_author_institution" content="School of Botany and Zoology, Australian National University, Canberra, Australian Capital Territory, Australia"/>
      <meta name="citation_author" content="Adrian J. Gibbs"/>
            <meta name="citation_author_institution" content="Yarralumla, Australian Capital Territory, Australia"/>
      <meta name="citation_author" content="Mathieu Fourment"/>
            <meta name="citation_author_institution" content="Department of Biological Sciences, Macquarie University, Sydney, New South Wales, Australia"/>
      <meta name="citation_author" content="Brendan C. Rodoni"/>
            <meta name="citation_author_institution" content="Department of Primary Industries, Knoxfield, Victoria, Australia"/>
      <meta name="citation_author" content="Mark J. Gibbs"/>
            <meta name="citation_author_institution" content="School of Botany and Zoology, Australian National University, Canberra, Australian Capital Territory, Australia"/>

    <meta name="citation_date" content="2008/2/13"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0001586.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e1586"/>
    <meta name="citation_issue" content="2"/>
    <meta name="citation_volume" content="3"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Variability and genetic structure of plant virus populations.; citation_author=F Garcia-Arenal; citation_author=A Fraile; citation_author=JM Malpica; citation_journal_title=Annu Rev Phytopathol; citation_volume=39; citation_number=1; citation_pages=157-186; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Structure and temporal dynamics of populations within wheat streak mosaic virus isolates.; citation_author=JS Hall; citation_author=R French; citation_author=TJ Morris; citation_author=DC Stenger; citation_journal_title=J Virol; citation_volume=75; citation_number=2; citation_pages=10231-10243; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Virus Taxonomy: VIIIth Report of the International Committee on Taxonomy of Viruses. 8 Ed.; citation_author=CM Fauquet; citation_author=MA Mayo; citation_author=J Maniloff; citation_author=U Desselberger; citation_author=LA Ball; citation_number=3; citation_pages=1259; citation_date=2005; citation_publisher=Elsevier Academic Press; " />
      <meta name="citation_reference" content="citation_title=Population structure within lineages of Wheat streak mosaic virus derived from a common founding event exhibits stochastic variation inconsistent with the deterministic quasi-species model.; citation_author=R French; citation_author=DC Stenger; citation_journal_title=Virology; citation_volume=343; citation_number=4; citation_pages=179-189; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=A reevaluation of the higher taxonomy of viruses based on RNA polymerases.; citation_author=PM Zanotto; citation_author=MJ Gibbs; citation_author=EA Gould; citation_author=EC Holmes; citation_journal_title=J Virol; citation_volume=70; citation_number=5; citation_pages=6083-6096; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Nucleotide sequences of the coat protein gene of potyviruses infecting Ornithogalum thyrsoides.; citation_author=S Fuji; citation_author=M Inoue; citation_author=H Yamamoto; citation_author=H Furuya; citation_author=H Naito; citation_journal_title=Arch Virol; citation_volume=148; citation_number=6; citation_pages=613-621; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Characterization of a new potyvirus isolated from Chinese artichoke in Japan.; citation_author=S Fuji; citation_author=H Yamamoto; citation_author=H Furuya; citation_author=H Naito; citation_journal_title=Arch Virol; citation_volume=148; citation_number=7; citation_pages=2249-2255; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Molecular criteria for genus and species discrimination within the family Potyviridae.; citation_author=MJ Adams; citation_author=JF Antoniw; citation_author=CM Fauquet; citation_journal_title=Arch Virol; citation_volume=150; citation_number=8; citation_pages=459-479; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=The use of PCR for cloning of large cDNA fragments of turnip mosaic potyvirus.; citation_author=O Nicolas; citation_author=JF Laliberte; citation_journal_title=J Virol Methods; citation_volume=32; citation_number=9; citation_pages=57-66; citation_date=1991; " />
      <meta name="citation_reference" content="citation_title=Identification of potyviruses using the polymerase chain reaction with degenerate primers.; citation_author=SA Langeveld; citation_author=JM Dore; citation_author=J Memelink; citation_author=AF Derks; citation_author=CI van der Vlugt; citation_journal_title=J Gen Virol; citation_volume=72; citation_number=10; citation_pages=1531-1541; citation_date=1991; " />
      <meta name="citation_reference" content="citation_title=A polymerase chain reaction method adapted for selective amplification and cloning of 3′ sequences of potyviral genomes: application to dasheen mosaic virus.; citation_author=SS Pappu; citation_author=R Brand; citation_author=HR Pappu; citation_author=EP Rybicki; citation_author=KH Gough; citation_journal_title=J Virol Methods; citation_volume=41; citation_number=11; citation_pages=9-20; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=Identification of a sweet potato feathery mottle virus isolate from China (SPFMV-CH) by the polymerase chain reaction with degenerate primers.; citation_author=D Colinet; citation_author=J Kummert; citation_journal_title=J Virol Methods; citation_volume=45; citation_number=12; citation_pages=149-159; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=A primer pair for amplifying part of the genome of all potyvirids by RT-PCR.; citation_author=A Gibbs; citation_author=A Mackenzie; citation_journal_title=J Virol Methods; citation_volume=63; citation_number=13; citation_pages=9-16; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=A universal PCR primer to detect members of the Potyviridae and its use to examine the taxonomic status of several members of the family.; citation_author=J Chen; citation_author=MJ Adams; citation_journal_title=Arch Virol; citation_volume=146; citation_number=14; citation_pages=757-766; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=The Potyviridae. Ed.; citation_author=DD Shukla; citation_author=CW Ward; citation_author=AA Brunt; citation_number=15; citation_pages=448; citation_date=1994; citation_publisher=CAB International; " />
      <meta name="citation_reference" content="citation_title=Three distinct mechanisms facilitate genetic isolation of sympatric wheat streak mosaic virus lineages.; citation_author=JS Hall; citation_author=R French; citation_author=GL Hein; citation_author=TJ Morris; citation_author=DC Stenger; citation_journal_title=Virology; citation_volume=282; citation_number=16; citation_pages=230-236; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=What is a quasispecies?; citation_author=CK Biebricher; citation_author=M Eigen; citation_journal_title=Curr Top Microbiol Immunol; citation_volume=299; citation_number=17; citation_pages=1-31; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=MUSCLE: multiple sequence alignment with high accuracy and high throughput.; citation_author=RC Edgar; citation_journal_title=Nucleic Acids Res; citation_volume=32; citation_number=18; citation_pages=1792-1797; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Gapped BLAST and PSI-BLAST: a new generation of protein database search programs.; citation_author=SF Altschul; citation_author=TL Madden; citation_author=AA Schäffer; citation_author=J Zhang; citation_author=Z Zhang; citation_journal_title=Nucleic Acids Res; citation_volume=25; citation_number=19; citation_pages=3389-3402; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=MEGA3: Integrated software for Molecular Evolutionary Genetics Analysis and sequence alignment.; citation_author=S Kumar; citation_author=K Tamura; citation_author=M Nei; citation_journal_title=Brief Bioinform; citation_volume=5; citation_number=20; citation_pages=150-163; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=BioEdit: Biological Sequence Alignment Editor for Windows 95/98/NT/XP.; citation_author=T Hall; citation_number=21; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=The Mathematical Theory of Communication.; citation_author=CE Shannon; citation_number=22; citation_pages=379-423; citation_date=1949; " />
      <meta name="citation_reference" content="citation_title=The GPRIME package: computer programs for identifying the best regions of aligned genes to target in nucleic acid hybridisation-based diagnostic tests, and their use with plant viruses.; citation_author=A Gibbs; citation_author=J Armstrong; citation_author=AM Mackenzie; citation_author=GF Weiller; citation_journal_title=J Virol Methods; citation_volume=74; citation_number=23; citation_pages=67-76; citation_date=1998; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001586" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Accumulating Variation at Conserved Sites in Potyvirus Genomes Is Driven by Species Discovery and Affects Degenerate Primer Design"/>
    <meta name="twitter:description" content="Unknown and foreign viruses can be detected using degenerate primers targeted at conserved sites in the known viral gene sequences. Conserved sites are found by comparing sequences and so the usefulness of a set of primers depends crucially on how well the known sequences represent the target group including unknown sequences.Methodology/Principal FindingsWe developed a method for assessing the apparent stability of consensus sequences at sites over time using deposition dates from Genbank. We tested the method using 17 conserved sites in potyvirus genomes. The accumulation of knowledge of sequence variants over 20 years caused &lsquo;consensus decay&rsquo; of the sites. Rates of decay were rapid at all sites but varied widely and as a result, the ranking of the most conserved sites changed. The discovery and reporting of sequences from previously unknown and distinct species, rather than from strains of known species, dominated the decay, indicating it was largely a sampling effect related to the progressive discovery of species, and recent virus mutation was probably only a minor contributing factor.Conclusion/SignificanceWe showed that in the past, the sampling bias has misled the choice of the most conserved target sites for genus specific degenerate primers. The history of sequence discoveries indicates primer designs should be updated regularly and provides an additional dimension for improving the design of degenerate primers."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0001586.g004"/>

  <meta property="og:title" content="Accumulating Variation at Conserved Sites in Potyvirus Genomes Is Driven by Species Discovery and Affects Degenerate Primer Design" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001586" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=5666'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=4749'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=1049&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0001586">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001586" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001586&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Accumulating Variation at Conserved Sites in Potyvirus Genomes Is Driven by Species Discovery and Affects Degenerate Primer Design
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Linda Zheng
              <span class="equal-contrib"
                    title="These authors contributed equally to this work">equal contributor</span>
              <span class="corresponding">mail</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">

                <p><span class="equal-contrib" title="These authors contributed equally to this work">equal contributor</span>
                Contributed equally to this work with: Linda Zheng, Mark J. Gibbs</p>

              
              <p>*E-mail: <a href="mailto:linda.zheng@anu.edu.au">linda.zheng@anu.edu.au</a></p>

                <p>Affiliation:
                  School of Botany and Zoology, Australian National University, Canberra, Australian Capital Territory, Australia
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Paul J. Wayper, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  School of Botany and Zoology, Australian National University, Canberra, Australian Capital Territory, Australia
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Adrian J. Gibbs, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Yarralumla, Australian Capital Territory, Australia
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Mathieu Fourment, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Biological Sciences, Macquarie University, Sydney, New South Wales, Australia
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Brendan C. Rodoni, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Primary Industries, Knoxfield, Victoria, Australia
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Mark J. Gibbs
              <span class="equal-contrib"
                    title="These authors contributed equally to this work">equal contributor</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">

                <p><span class="equal-contrib" title="These authors contributed equally to this work">equal contributor</span>
                Contributed equally to this work with: Linda Zheng, Mark J. Gibbs</p>

              
              

                <p>Affiliation:
                  School of Botany and Zoology, Australian National University, Canberra, Australian Capital Territory, Australia
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: February 13, 2008</li>
    <li>DOI: 10.1371/journal.pone.0001586</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0001586" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0001586" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001586" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0001586" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0001586-t001" data-doi="info:doi/10.1371/journal.pone.0001586" data-uri="info:doi/10.1371/journal.pone.0001586.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001586.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001586-g001" data-doi="info:doi/10.1371/journal.pone.0001586" data-uri="info:doi/10.1371/journal.pone.0001586.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001586.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001586-g002" data-doi="info:doi/10.1371/journal.pone.0001586" data-uri="info:doi/10.1371/journal.pone.0001586.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001586.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001586-g003" data-doi="info:doi/10.1371/journal.pone.0001586" data-uri="info:doi/10.1371/journal.pone.0001586.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001586.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001586-t002" data-doi="info:doi/10.1371/journal.pone.0001586" data-uri="info:doi/10.1371/journal.pone.0001586.t002" title="Table 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001586.t002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001586-g004" data-doi="info:doi/10.1371/journal.pone.0001586" data-uri="info:doi/10.1371/journal.pone.0001586.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001586.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001586-t003" data-doi="info:doi/10.1371/journal.pone.0001586" data-uri="info:doi/10.1371/journal.pone.0001586.t003" title="Table 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001586.t003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001586">Reader Comments (1)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0001586" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2>

<a id="article1.front1.article-meta1.abstract1.sec1.p1" name="article1.front1.article-meta1.abstract1.sec1.p1"></a><p>Unknown and foreign viruses can be detected using degenerate primers targeted at conserved sites in the known viral gene sequences. Conserved sites are found by comparing sequences and so the usefulness of a set of primers depends crucially on how well the known sequences represent the target group including unknown sequences.</p>


<h3>Methodology/Principal Findings</h3>
<a id="article1.front1.article-meta1.abstract1.sec2.p1" name="article1.front1.article-meta1.abstract1.sec2.p1"></a><p>We developed a method for assessing the apparent stability of consensus sequences at sites over time using deposition dates from Genbank. We tested the method using 17 conserved sites in potyvirus genomes. The accumulation of knowledge of sequence variants over 20 years caused ‘consensus decay’ of the sites. Rates of decay were rapid at all sites but varied widely and as a result, the ranking of the most conserved sites changed. The discovery and reporting of sequences from previously unknown and distinct species, rather than from strains of known species, dominated the decay, indicating it was largely a sampling effect related to the progressive discovery of species, and recent virus mutation was probably only a minor contributing factor.</p>


<h3>Conclusion/Significance</h3>
<a id="article1.front1.article-meta1.abstract1.sec3.p1" name="article1.front1.article-meta1.abstract1.sec3.p1"></a><p>We showed that in the past, the sampling bias has misled the choice of the most conserved target sites for genus specific degenerate primers. The history of sequence discoveries indicates primer designs should be updated regularly and provides an additional dimension for improving the design of degenerate primers.</p>

</div>


<div class="articleinfo"><p><strong>Citation: </strong>Zheng L, Wayper PJ, Gibbs AJ, Fourment M, Rodoni BC, et al.  (2008) Accumulating Variation at Conserved Sites in Potyvirus Genomes Is Driven by Species Discovery and Affects Degenerate Primer Design. PLoS ONE 3(2):
          e1586.
            doi:10.1371/journal.pone.0001586</p><p><strong>Academic Editor: </strong>Sheila Bowyer, National Institute for Communicable Diseases, South Africa</p><p><strong>Received:</strong> September 20, 2007; <strong>Accepted:</strong> January 7, 2008; <strong>Published:</strong> February 13, 2008</p><p><strong>Copyright:</strong> © 2008 Zheng et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>This work is part of my PhD program which was supported by the Department of Primary Industries (“Our Rural Landscape”) funded by the Victorian State Government of The Commenwealth of Australia. The funders of this study had no role in the design or conduct of this study, in the collection, analysis, and interpretation of the data, and in the preparation, review, or approval of the manuscript.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>About 55% of known virus species have linear single-stranded RNA genomes; potyviruses belong in this category. RNA virus genomes are diverse in plan and variable in sequence <a href="#pone.0001586-GarciaArenal1">[1]</a>–<a href="#pone.0001586-French1">[4]</a>. Different isolates from the same species usually have many single nucleotide polymorphisms. All RNA viruses have RNA dependent RNA polymerase (RdRp) genes, but even though they are the most conserved genes, they are very varied <a href="#pone.0001586-Zanotto1">[5]</a>. Therefore to identify RNA virus species, degenerate primers (i.e. primer mixtures) that target conserved sites have been developed for many viral genera. For some important genera, several sets of degenerate primers have been designed. Conserved sites are found by comparing sequences and so the usefulness of a set of primers depends crucially on how well the available sequences represent the target viruses, including as yet unknown members of the genus. At any one time, the available sequences only represent a sample of the extant and future target sequences, so biases in this sample can affect degenerate primer design. Understanding such biases is important, as degenerate primers are an essential biosecurity technology allowing viruses and microbes to be detected and identified and because they are essential for research on virus diversity, ecology and phylogeny <a href="#pone.0001586-Fuji1">[6]</a>, <a href="#pone.0001586-Fuji2">[7]</a>.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>The genus <em>Potyvirus</em> is the second largest known virus genus, with 111 recognised species and 86 tentative species currently assigned to it by the International Committee on Taxonomy of Viruses (ICTV)<a href="#pone.0001586-Fauquet1">[3]</a>. Potyviruses infect plants and cause serious diseases in crops worldwide <a href="#pone.0001586-Fuji1">[6]</a>, <a href="#pone.0001586-Fuji2">[7]</a>. By 2006 the international gene sequence databases contained more than 250 fully sequenced potyvirus genomes, representing 47 species. This large sample of complete genomes of different species is second only to that of the genus <em>Begomovirus.</em> Extensive studies of potyvirus sequences have shown that potyvirus species can be distinguished by sequence comparisons, and the process of recognising a species is now in large part determined using this measure. Potyvirus isolates with 85% sequence identity or more over the whole genome are usually considered to be from the same species <a href="#pone.0001586-Fauquet1">[3]</a>. This was re-evaluated by Adams <em>et al</em>. in 2005, who suggested that a value of about 76% was most appropriate for the entire polyprotein sequence and that slightly different values might be used for the different regions of the potyvirus genome <a href="#pone.0001586-Adams1">[8]</a>. We have chosen to use the value adopted by the ICTV.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>Since the early 1990s, six sets of primers have been developed and used widely for potyviruses; the sites are commonly named after the amino acid motifs that they encode. Nicholas and Laliberte published the first degenerate primers in 1991, which targeted the ‘ATNIIENG’ and ‘YCDADGS’ sites in the cylindrical cytoplasmic inclusion protein (CI) and RdRp (NIb) genes <a href="#pone.0001586-Nicolas1">[9]</a>. Only five potyvirus sequences were available to Nicholas and Laliberte and they based their designs on conserved amino acids. Four more sequences were used by Langeveld <em>et al</em>. in the same year to design primers to the AHFQMKTA and TVVDNTLMV sites in the NIb gene and the MVWCIENG site in the coat protein (CP) gene <a href="#pone.0001586-Langeveld1">[10]</a>. Working with 15 sequences in 1993, Pappu <em>et al</em>. produced the WCIEN primer, which targeted a subsequence from the MVWCIENG region, and the QMKAAA primer also from the CP gene <a href="#pone.0001586-Pappu1">[11]</a>. These degenerate primers were paired with an oligo-d(T) primer that annealed to the poly-adenylate tail that is present at 3′ end of all potyvirus genomes. In the same year, Colinet and Kummert reported the QMKAAA primer independently, and produced primers to the WKHWI and WNGSL sites from the NIa-Pro and NIb genes <a href="#pone.0001586-Colinet1">[12]</a>. They also used a primer to the CDADGS site identified earlier. Four years later, Gibbs and Mackenzie reported a primer that matched the GNNSGQ motif of the RdRp and found that the primer could be used to amplify cDNA from viruses from other genera in the family <em>Potyviridae</em> as well as from potyviruses <a href="#pone.0001586-Gibbs1">[13]</a>. In 2001, Chen <em>et al</em>. published an extended version of the GNNSGQ primer which included a few more nucleotides <a href="#pone.0001586-Chen1">[14]</a>.</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p>We have developed a method for analysing the apparent stability of conserved sites as more sequences are discovered and reported. Here we report an analysis by this method of conserved sites in potyvirus genomes, including some of those previously listed. We analysed the sequences reported over a 20 year period and showed how knowledge of nucleotide variants accumulates as the sequence database builds. The variants produced an apparent decay of the consensus at the sites. We found that the decay rates differed at different sites and hence the ranking of sites by their conservation changed over time. To help interpret this data, we cross-checked lists of the species names declared in the GenBank and ICTV databases and tested the sequence-based definitions of the species to track the rise in the numbers of known species and analysed datasets from the three most fully sampled species. We compared the consensus decay rates within and between species in order to assess whether variation among isolates from single virus species contributed to the decay. Our results suggest that measuring consensus decay at conserved sites is important when designing primers for detecting both known and unknown viruses.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3><a id="article1.body1.sec2.p1" name="article1.body1.sec2.p1"></a><p>Using all the available full length sequences, we found 13 conserved sites that had not been previously reported (<a href="#pone-0001586-t001">Table 1</a>). Seven of the sites were found by eye and six were found using the NCSF program (see <a href="#s4">Materials and Methods</a>). Four of the eight sites identified by earlier workers were included: GNNSGQ, MVWCIEN , NIIENG and QMKAAA <a href="#pone.0001586-Nicolas1">[9]</a>–<a href="#pone.0001586-Pappu1">[11]</a>, <a href="#pone.0001586-Gibbs1">[13]</a>, <a href="#pone.0001586-Chen1">[14]</a>; the 5′ nucleotide of the regions encoding these motifs were at positions 7899, 8853, 4545 and 9162 in the <em>Tobacco etch virus</em> genome (Accession code M11458). The other four sites that had been reported earlier were found to be very variable and were not included, although parts of two of them, TVVDNTLMV and YCDADGS <a href="#pone.0001586-Langeveld1">[10]</a>, overlapped with two of the newly identified sites. Five of the 17 sites selected for analysis (sites 3, 4, 7, 11 and 15; <a href="#pone-0001586-t001">Table 1</a>) were clustered in the CI gene and the remainder of the sites were located in the NIb and the CP genes (<a href="#pone-0001586-t001">Table 1</a>). This clustering was consistent with the reported variability along potyvirus genomes <a href="#pone.0001586-Shukla1">[15]</a>. None of the newly recognised sites encoded motifs associated with known protein functions.</p>
<div class="figure" id="pone-0001586-t001"><div class="img"><a name="pone-0001586-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001586.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001586" data-uri="info:doi/10.1371/journal.pone.0001586.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001586.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001586.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001586.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001586.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001586.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001586.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001586.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001586.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001586.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>Rank and N scores of 17 conserved sites in potyvirus genomes along with their locations and sequence and the method used to identify each one.</span></strong></p><span>doi:10.1371/journal.pone.0001586.t001</span></div><a id="article1.body1.sec2.p2" name="article1.body1.sec2.p2"></a><p>We ranked the 17 sites according to their level of conservation as measured by the average nucleotide variant count (see <a href="#s4">Materials and Methods</a>) using the sequences available up to December 2005 (<a href="#pone-0001586-t001">Table 1</a>). The CVD<em>DFN</em> site at position 7587 in the NIb gene was the most conserved site, followed by the EN/D<em>T</em>ERH site at position 9237 in the CP gene. The CVD<em>DFN</em> site was identified by eye, and had the smallest Shannon entropy and redundancy scores. It also had small dominant base and sub-sequence variant counts. The EN/DTERH site was found using NCSF and had the smallest sub-sequence variant count. It also had a small redundancy score and a small dominant base count. Of the previously identified sites, NI<em>I</em>ENGV found by Nicholas and Laliberte (“ATNIIENG”) <a href="#pone.0001586-Nicolas1">[9]</a> ranked third, QMKAAA found by Langeveld <em>et al.</em> <a href="#pone.0001586-Langeveld1">[10]</a> and Pappu <a href="#pone.0001586-Pappu1">[11]</a> ranked fifth, GNNSGQ site identified by Gibbs and Mackenzie (PV2) <a href="#pone.0001586-Gibbs1">[13]</a> ranked equal seventh, and the MVWCIE/DNG site identified by both Langeveld <em>et al.</em> <a href="#pone.0001586-Langeveld1">[10]</a> and Pappu (“WCIEN”) <a href="#pone.0001586-Pappu1">[11]</a> ranked 12<sup>th</sup>. Three of those published sites were found by NCSF to be among the top five sites ranked on one or other of its 4 measures. The WVCIEN site was not among the top five scores in any of the NCSF measures.</p>

<h4>Sequences and Species</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>Only 90 (81.1%) of the 111 confirmed potyvirus species and 11 (12.8%) of the 86 tentative species recognised by the ICTV were represented by sequences recorded in GenBank. We assume that no sequence data has been reported for the remaining 21 recognised and 75 tentative species. The value of recognising potyvirus species without sequence data is uncertain. We found 64 sequences from potyviruses with names that had not been listed by the ICTV. Our database searches suggested that 25 of those unrecognised sequences probably came from distinct unrecognised species, 35 had been misnamed as they matched the sequences of recognised species, and 4 were difficult to categorise because they differ from the sequences of recognised species but not conclusively so (data not shown).</p>
<a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2"></a><p>The numbers of full length and partial sequences submitted to Genbank annually from 1985 to 2005 were plotted, together with the cumulative total number of sequences (<a href="#pone-0001586-g001">Figure 1</a>). Very few new sequences were reported between 1985 and 1992, but the rate increased after that period and peaked between 2001 and 2005 with an overall mean of 165.9 sequences/year. The number of full length sequences released each year followed a similar trend with a peak of more than 50 full length sequences being deposited in 2002 (<a href="#pone-0001586-g002">Figure 2</a>; mean 12.3). Interestingly, the number of named species associated with gene sequences submitted to Genbank differed markedly from the number of species recognised by the ICTV each year (<a href="#pone.0001586.s002">Figure S1</a>; mean 4.2) and neither showed any clear trend. The yearly count of potyvirus species found to have distinct nucleotide sequences varied considerably and no clear trends was observed (<a href="#pone.0001586.s003">Figure S2</a>), but on average 5.8 new species were identified annually by this means. The peak years for species officially recognized by the ICTV were 1999 and 2002, and 2004 was the peak year for new species included in GenBank (<a href="#pone.0001586.s003">Figure S2</a>).</p>
<div class="figure" id="pone-0001586-g001"><div class="img"><a name="pone-0001586-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001586.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001586" data-uri="info:doi/10.1371/journal.pone.0001586.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001586.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001586.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001586.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001586.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001586.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001586.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001586.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001586.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001586.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>The annual release and the accumulated total of all potyvirus sequences in GenBank from 1985 to 2005.</span></strong></p><span>doi:10.1371/journal.pone.0001586.g001</span></div><div class="figure" id="pone-0001586-g002"><div class="img"><a name="pone-0001586-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001586.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001586" data-uri="info:doi/10.1371/journal.pone.0001586.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001586.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001586.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001586.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001586.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001586.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001586.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001586.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001586.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001586.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>The annual release and the accumulated total of all full length potyvirus sequences in GenBank from 1985 to 2005.</span></strong></p><span>doi:10.1371/journal.pone.0001586.g002</span></div>

<h4>Accumulating nucleotide variants</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>A potyvirus genome was completely sequenced for the first time in 1985, and nucleotide variants were first recorded in 1988, so we largely confined our analyses to the years after 1988. Average nucleotide variant counts (N scores) were calculated for each of the 17 primer sites for each year after 1988 using the data from the genome dataset. N scores increased with time for all sites including the most conserved (sites 7587, 9237 and 4545; <a href="#pone-0001586-g003">Figure 3</a>). N scores also diverged over time as shown by the standard deviations (<a href="#pone-0001586-t002">Table 2</a>); the SD value for the conserved sites rose from 0.07 to 0.2 over the 17 years from 1988. Randomly selected sites accumulated nucleotide variants much more rapidly with SD values double those found for the conserved sites (<a href="#pone-0001586-t002">Table 2</a>). The mean N score in 2005 was 2.02 for the randomly selected sites (<a href="#pone-0001586-t002">Table 2</a>), whereas the best and worst conserved sites had scores of 0.55 and 1.20 (<a href="#pone-0001586-t001">Table 1</a>).</p>
<div class="figure" id="pone-0001586-g003"><div class="img"><a name="pone-0001586-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001586.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001586" data-uri="info:doi/10.1371/journal.pone.0001586.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001586.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001586.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001586.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001586.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001586.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001586.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001586.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001586.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001586.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Average nucleotide variant score (N score) of the top 3 and 6 other conserved sites compared with the minimum, maximum and average N scores of all 17 conserved sites in the potyvirus genomes.</span></strong></p><span>doi:10.1371/journal.pone.0001586.g003</span></div><div class="figure" id="pone-0001586-t002"><div class="img"><a name="pone-0001586-t002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001586.t002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001586" data-uri="info:doi/10.1371/journal.pone.0001586.t002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001586.t002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001586.t002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001586.t002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001586.t002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001586.t002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001586.t002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001586.t002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001586.t002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001586.t002.TIF"></span>)
                </a></li></ul></div><p><strong>Table 2.  <span>Average nucleotide variant counts (N scores) of 17 conserved sites compared to the mean of 20 random sites in all potyvirus genomes from 1985 to 2005.</span></strong></p><span>doi:10.1371/journal.pone.0001586.t002</span></div><a id="article1.body1.sec2.sec2.p2" name="article1.body1.sec2.sec2.p2"></a><p>When sites were ranked using N scores, the rankings changed over time. Site 8901 was ranked equal best in 1989 and 1990, but was ranked worst in 2003 and 2004 (<a href="#pone-0001586-t002">Table 2</a>). Site 8853 was ranked second best in 1989–1991, but was ranked third worst in 2005 (<a href="#pone-0001586-t002">Table 2</a>; <a href="#pone-0001586-g003">Figure 3</a>). Conversely, site 9237 was ranked second worst in 1988, but became the second best site by 2003 and remained in that position through to 2005 (<a href="#pone-0001586-t002">Table 2</a>). By contrast, some sites had a stable ranking; site 7587 (CVDDFN) was the most conserved site over almost all years after 1988 (<a href="#pone-0001586-t002">Table 2</a>; <a href="#pone-0001586-g003">Figure 3</a>).</p>
<a id="article1.body1.sec2.sec2.p3" name="article1.body1.sec2.sec2.p3"></a><p>The consensus decay of some sites appeared to stabilize after a time, as their N scores asymptoted. For example, no further nucleotide variants were reported after 1996 at sites 4458 and 9237, and no consensus decay occurred at site 9099 after 1999. Site 7911 was stable for nine years, but some more variants of that site were reported in 2005. The mean yearly N score of all conserved sites climbed in all years except 1990; there were obvious pauses and jumps in this value in the early years and more steady increases after 1995. The overall rate of change of the N scores between 1988 and 2005 was 0.044 variants per position per year (v/p/y) as calculated from the slope of the mean of yearly N scores of all conserved sites. The slope of the mean of all conserved sites between 1988 and 1995 was 0.06 v/p/y, and between 1996 and 2005 it was 0.024 v/p/y.</p>
<a id="article1.body1.sec2.sec2.p4" name="article1.body1.sec2.sec2.p4"></a><p>The relationship between variability and the accumulated number of sequences was calculated by dividing the mean of the N scores of the conserved sites by the number of sequences (<a href="#pone.0001586.s004">Figure S3</a>). A peak value (0.0147) was observed in the first year that variants were observed (1988) and after that year, the value rapidly approached zero, indicating that as the total number of the sequences increased, fewer variants per sequence were observed (<a href="#pone.0001586.s004">Figure S3</a>).</p>
<a id="article1.body1.sec2.sec2.p5" name="article1.body1.sec2.sec2.p5"></a><p>To estimate the differing contributions from species and strains, i.e. the within and between-species variation, we calculated N scores for the three species datasets and the representative dataset. The N scores from the representative dataset were about the same, from year to year, as those from the genome dataset (<a href="#pone.0001586.s001">Table S1</a>), with small differences apparent after 1995 (<a href="#pone-0001586-g004">Figure 4</a>). The N scores from the species datasets revealed more information as they were small or zero before 1995 and rose sharply after that year (<a href="#pone-0001586-g004">Figure 4</a>), probably because few sequences were obtained from the three species in early years while many sequences were obtained in the last few years. N scores from the PPV, TuMV and PVY datasets had mean rates of change of 0.013 v/p/y, 0.039 v/p/y and 0.022 v/p/y respectively. The differences between the genome and representative datasets, visible after 1995 (<a href="#pone-0001586-g004">Figure 4</a>), were probably due to sequences from strains. They could be interpreted to show a small bias in the results from the genome dataset due to over-representation of some species, i.e. those represented by many strain sequences. As expected, N scores from the genome dataset correlated much more strongly with those from the representative dataset than with those from the species datasets (<a href="#pone-0001586-t003">Table 3</a>).</p>
<div class="figure" id="pone-0001586-g004"><div class="img"><a name="pone-0001586-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001586.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001586" data-uri="info:doi/10.1371/journal.pone.0001586.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001586.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001586.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001586.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001586.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001586.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001586.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001586.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001586.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001586.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>Comparison of the maximum and mean N scores for all conserved sites in the genome, representative and the species datasets.</span></strong></p><span>doi:10.1371/journal.pone.0001586.g004</span></div><div class="figure" id="pone-0001586-t003"><div class="img"><a name="pone-0001586-t003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001586.t003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001586" data-uri="info:doi/10.1371/journal.pone.0001586.t003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001586.t003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001586.t003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001586.t003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001586.t003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001586.t003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001586.t003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001586.t003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001586.t003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001586.t003.TIF"></span>)
                </a></li></ul></div><p><strong>Table 3.  <span>Correlations between the N score time series from the genome dataset and the N score time series from other datasets for each site.</span></strong></p><span>doi:10.1371/journal.pone.0001586.t003</span></div>
</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p>Consensus decay is important, as degenerate primer mixtures only reliably detect all the target viruses if they contain individual primers that complement and bind to all the virus sequences. We found that several of the sites targeted in early work on potyviruses were no longer conserved. This is likely to be a general effect as, in the first few years after the first genomes of a new genus are sequenced, a consensus sequence will not cover many of the sequence variants.</p>
<a id="article1.body1.sec3.p2" name="article1.body1.sec3.p2"></a><p>Consensus decay may mislead the degenerate primer design process. We have shown that conserved sites in potyvirus genomes have been incorrectly ranked in the past. The rank of every one of the conserved sites targeted by earlier workers changed over time (<a href="#pone-0001586-t002">Table 2</a>), and the most conserved of those sites was only ranked third in 2005 (<a href="#pone-0001586-t001">Table 1</a>). We found that all conserved sites in potyvirus genomes suffer consensus decay (<a href="#pone-0001586-t002">Table 2</a>), however different sites asymptote to different levels of variability. For example, sites 9237 and 4458 both attained a stable variant count by 1996, however the first had an average variant count of 0.6 and the latter of 0.85 (<a href="#pone-0001586-t002">Table 2</a>; <a href="#pone-0001586-g003">Figure 3</a>).</p>
<a id="article1.body1.sec3.p3" name="article1.body1.sec3.p3"></a><p>Consensus decay across sites from a genus is largely due to variation between species, rather than variation within a species, as was shown by the correlation between the N scores from the genome and representative datasets (<a href="#pone-0001586-t003">Table 3</a>; <a href="#pone-0001586-g004">Figure 4</a>). New strain sequences are being generated continually by mutations <a href="#pone.0001586-Hall1">[2]</a>, <a href="#pone.0001586-French1">[4]</a>, <a href="#pone.0001586-Hall2">[16]</a>, but this may not have a significant effect on the N score of the genus. Many nucleotide variants were reported after 1995 among the strains of PPV, PVY and TuMV, but they had little effect on the consensus decay of the genome dataset, presumably because the same variants had already been recorded. If the strain variants had been novel we would have expected N scores to have increased at a higher rate after 1995. We suggest that the virus species analysed in this study come from lineages that have explored a large proportion of the viable sequence space <a href="#pone.0001586-Biebricher1">[17]</a>, and although mutation continues within each species, it is largely a reiteration of earlier exploration. Hence, additional sampling of strains is mostly finding variants that are already known.</p>
<a id="article1.body1.sec3.p4" name="article1.body1.sec3.p4"></a><p>We counted the number of potyvirus species confirmed by sequence deposition each year, and found an average rate of 5.8 per year, although the totals varied greatly (<a href="#pone.0001586.s003">Figure S2</a>). Likewise, peaks in the number of new species recognised by the ICTV varied greatly. These variations probably reflect irregular sampling. Many human factors probably contributed to the species sampling variability including motivation, funding and improvements in molecular methods and primer design. Early sequencing efforts appear to have concentrated on the well known virus species that caused significant crop damage, whereas later efforts have been to identify less damaging species. There are inconsistencies between species and sequence sampling. In the past recognition and discovery of a species has often preceded sequencing, whereas more recently species recognition has lagged behind sequencing and species discovery. The peak years of species discovery may have past and few new potyvirus species have been recognised by the ICTV since 2002 (<a href="#pone.0001586.s003">Figure S2</a>). However, the year to year counts are very variable (<a href="#pone.0001586.s002">Figures S1</a> and <a href="#pone.0001586.s003">S2</a>), so much is uncertain. It should also be realised that few wild plants have yet been tested for potyviruses, so there may be many new species still to be discovered and further consensus decay may occur.</p>
<a id="article1.body1.sec3.p5" name="article1.body1.sec3.p5"></a><p>In conclusion, we believe consensus decay will be measured in the future when new degenerate primers are designed or existing ones are updated. Quarantine agencies and authorities involved in biosecurity may use consensus decay data when assessing degenerate primer or probe tests. Other factors will be considered, first of which will be empirical data on primer performance, and second may be the location of the primers in the genome and the proximity of their target sites. It is worth repeating that degenerate primers designed in the past for detecting potyviruses have not targeted the most conserved sites. We are now testing primers that target the most highly conserved sites.</p>
</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4>Datasets</h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p>An ‘all-sequence’ dataset obtained from the GenBank database in December 2005 contained 3484 potyvirus sequences including partial sequences. A ‘genome’ dataset was compiled that included 259 full length or near full length genomic sequences, greater than 9 kilobases in length, and representing 47 species. We aligned the genome dataset using the program MUSCLE <a href="#pone.0001586-Edgar1">[18]</a> to produce an alignment that was 16017 nucleotides long. A ‘representative’ dataset was derived from the aligned genome dataset; it consisted of 47 sequences, one genome from each potyvirus species. We also assembled three ‘species’ datasets that consisted of all the complete aligned genome sequences from strains of the species <em>Turnip mosaic virus</em> (TuMV), <em>Potato virus Y</em> (PVY) <em>and Plum pox virus</em> (PPV); these datasets were of 61, 41 and 22 sequences respectively. These three species were chosen because they were represented in GenBank by the most genomic sequences.</p>
<a id="article1.body1.sec4.sec1.p2" name="article1.body1.sec4.sec1.p2"></a><p>Potyvirus species and sequences were counted using the all-sequence dataset and custom built software that sorted the dataset by name and date (‘Potyvirus-VirusBanker’, <a href="http://biojanus.anu.edu.au/programs/">http://biojanus.anu.edu.au/programs/</a>, 2006; Fourment M, Zheng, L and Gibbs, MJ). The number of potyvirus sequences obtained each year was determined by using the earliest date found in each GenBank sequence file. The number of potyvirus species recognised each year was determined by sorting the all-sequence dataset by date and by finding the earliest available sequence with a distinct species name. The earliest available date was assumed to be an estimate of the date when the species was confirmed to be distinct. We did not search the literature for earlier dates for the viruses and assumed that sequence comparisons were necessary for a species to be definitively distinguished. Our assumption about dates meant that species counts were made using the same timescale as sequence and nucleotide variant counts.</p>
<a id="article1.body1.sec4.sec1.p3" name="article1.body1.sec4.sec1.p3"></a><p>A procedure was developed to identify misnamed sequences and sequences from species that had not been officially recognised. Species names were obtained from the GenBank Definition annotation and these were cross-checked against the list of names from the International Committee on Taxonomy of Viruses (ICTV) <a href="#pone.0001586-Fauquet1">[3]</a>. Sequence records with names that were not listed by the ICTV were examined by searching for matching sequences in GenBank using the program BLASTN <a href="#pone.0001586-Altschul1">[19]</a>. If a sequence with an unrecognised name was more than 85% identical to one of a known species, then it was assumed to belong to the species. If a sequence with an unrecognised name did not match one from a known species then it was considered to come from an “unrecognised species”. The taxonomic status of each sequence was also tested by cluster analysis. The conserved core regions of the coat protein gene sequences were extracted from the genome dataset alignment and used to calculate a neighbor-joining tree using the program MEGA3 <a href="#pone.0001586-Kumar1">[20]</a>; the core regions were those bounded by the sequences encoding the WCIEN and QMKAAA motifs. The core regions were sorted into clusters according to their pair-wise nucleotide sequence identity using the program GROUPER (unpublished software, 2004; Armstrong, JS and Gibbs, AJ) with a clustering cut off point of 85% identity. Misnamed sequences found in the tree or the clusters were then tested by sequence matching using BLASTN <a href="#pone.0001586-Altschul1">[19]</a>. Finally, the GenBank database was searched for nucleotide sequences with the names of the tentative potyvirus species listed by the ICTV <a href="#pone.0001586-Fauquet1">[3]</a> and the likely taxonomic status of those sequences was tested again following the procedure used for the unrecognised sequences.</p>


<h4>Conserved Sites</h4>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1"></a><p>Conserved sites were named after the position of their 5′-most nucleotide in the aligned genome dataset. The sites were found in the aligned genome dataset using the program NCSF (unpublished software, <a href="http://biojanus.anu.edu.au/programs/">http://biojanus.anu.edu.au/programs/</a>, 2006; Wayper, P and Gibbs, MJ) and by eye using BioEdit <a href="#pone.0001586-Hall3">[21]</a>. Sites found by eye were chosen to be at least 15 nucleotides long and contain fewer than 3 consecutive variable bases and at least 10 nucleotides that were strictly conserved in more than 97% of sequences. NCSF was used with a search window 20 nucleotides long and four measures of conservation: Shannon informational entropy <a href="#pone.0001586-Shannon1">[22]</a>, redundancy <a href="#pone.0001586-Gibbs2">[23]</a>, ‘average dominant base count’ and ‘sub-sequence variant counts’ . Average dominant base counts were calculated by counting the number of occurrences of the most common base at each position in the site and averaging those counts across all positions in the site. Sub-sequence variant counts were the counts of all the actual sequence variants seen at a site. The top five most conserved sites found using each measure and all of the sites found by eye were included in the analysis. Previously published conserved sites were found in the aligned genome dataset using BioEdit. As a control, 20 ‘random sites’ were selected by generating random numbers to define the 5′ position of sites 20 nucleotides long in the genome dataset alignment; sites that contained gaps were discarded.</p>
<a id="article1.body1.sec4.sec2.p2" name="article1.body1.sec4.sec2.p2"></a><p>Nucleotide variants (single nucleotide polymorphisms) were counted within each of the conserved and random sites. Genome sequences were ordered by the year in which they were submitted to the database, and then nucleotide variants were counted. Counts were made for each year using only the sequences available for that year. Counts were made for the years 1988 to 2005. The genome sequence of <em>Tobacco etch virus</em> (Accession code M11458) was used as the reference sequence as it was the first potyvirus genome publicly available in GenBank. Only those variants that had not been submitted in earlier years were counted so that identical nucleotides were not counted twice. Hence for each nucleotide position in a primer site, there were only three possible variants of the initial sequence, and a scale of 0 to 3 was used; invariant positions scored zero. Variant counts were summed across the whole site and the total divided by the length of the site to produce an average nucleotide variant count (N score). This calculation allowed sites of different lengths to be compared.</p>

</div>

<div id="section5" class="section"><a id="s5" name="s5" toc="s5" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0001586"></div><a name="pone.0001586.s001" id="pone.0001586.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001586.s001">Table S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1"></a><p class="preSiDOI">Average nucleotide variant counts (N scores) of 17 conserved sites in representative potyvirus genomes from 1985 to 2005</p>
<p class="siDoi">doi:10.1371/journal.pone.0001586.s001</p><a id="article1.body1.sec5.supplementary-material1.caption1.p2" name="article1.body1.sec5.supplementary-material1.caption1.p2"></a><p class="postSiDOI">(0.10 MB DOC)</p>
<a name="pone.0001586.s002" id="pone.0001586.s002"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001586.s002">Figure S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material2.caption1.p1" name="article1.body1.sec5.supplementary-material2.caption1.p1"></a><p class="preSiDOI">The accumulated number of potyviruses species identified by their nucleotide sequences in GenBank from 1985 to 2005.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001586.s002</p><a id="article1.body1.sec5.supplementary-material2.caption1.p2" name="article1.body1.sec5.supplementary-material2.caption1.p2"></a><p class="postSiDOI">(3.89 MB TIF)</p>
<a name="pone.0001586.s003" id="pone.0001586.s003"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001586.s003">Figure S2. </a></strong></p><a id="article1.body1.sec5.supplementary-material3.caption1.p1" name="article1.body1.sec5.supplementary-material3.caption1.p1"></a><p class="preSiDOI">A yearly count of potyvirus species confirmed to be distinct by the deposition of a nucleotide sequence in GenBank from 1985 to 2005.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001586.s003</p><a id="article1.body1.sec5.supplementary-material3.caption1.p2" name="article1.body1.sec5.supplementary-material3.caption1.p2"></a><p class="postSiDOI">(3.46 MB TIF)</p>
<a name="pone.0001586.s004" id="pone.0001586.s004"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001586.s004">Figure S3. </a></strong></p><a id="article1.body1.sec5.supplementary-material4.caption1.p1" name="article1.body1.sec5.supplementary-material4.caption1.p1"></a><p class="preSiDOI">Average N score per sequence calculated by dividing the mean of N scores for all conserved sites by the number of sequences.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001586.s004</p><a id="article1.body1.sec5.supplementary-material4.caption1.p2" name="article1.body1.sec5.supplementary-material4.caption1.p2"></a><p class="postSiDOI">(2.70 MB TIF)</p>
</div>



<div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: MG. Performed the experiments: LZ. Analyzed the data: MG LZ. Contributed reagents/materials/analysis tools: PW MF BR. Wrote the paper: MG AG LZ. Other: Provided inspiring dicussion ideas and revised the abstract and discussion section of the paper: AG.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0001586-GarciaArenal1" id="pone.0001586-GarciaArenal1"></a>Garcia-Arenal F, Fraile A, Malpica JM (2001) Variability and genetic structure of plant virus populations. Annu Rev Phytopathol  39: 157–186.  <ul class="find" data-citedArticleID="1054586" data-doi="10.1146/annurev.phyto.39.1.157"><li><a href="http://dx.doi.org/10.1146/annurev.phyto.39.1.157" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Variability+and+genetic+structure+of+plant+virus+populations." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Variability+and+genetic+structure+of+plant+virus+populations.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0001586-Hall1" id="pone.0001586-Hall1"></a>Hall JS, French R, Morris TJ, Stenger DC (2001) Structure and temporal dynamics of populations within wheat streak mosaic virus isolates. J Virol  75: 10231–10243.  <ul class="find" data-citedArticleID="1054592" data-doi="10.1128/jvi.75.21.10231-10243.2001"><li><a href="http://dx.doi.org/10.1128/jvi.75.21.10231-10243.2001" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Structure+and+temporal+dynamics+of+populations+within+wheat+streak+mosaic+virus+isolates." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Structure+and+temporal+dynamics+of+populations+within+wheat+streak+mosaic+virus+isolates.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0001586-Fauquet1" id="pone.0001586-Fauquet1"></a>Fauquet CM, Mayo MA, Maniloff J, Desselberger U, Ball LA (2005) Virus Taxonomy: VIIIth Report of the International Committee on Taxonomy of Viruses. 8 Ed. San Diego: Elsevier Academic Press.   <ul class="find-nolinks"></ul></li><li><span class="label">4.
              </span><a name="pone.0001586-French1" id="pone.0001586-French1"></a>French R, Stenger DC (2005) Population structure within lineages of Wheat streak mosaic virus derived from a common founding event exhibits stochastic variation inconsistent with the deterministic quasi-species model. Virology  343: 179–189.  <ul class="find" data-citedArticleID="1054580" data-doi="10.1016/j.virol.2005.08.037"><li><a href="http://dx.doi.org/10.1016/j.virol.2005.08.037" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Population+structure+within+lineages+of+Wheat+streak+mosaic+virus+derived+from+a+common+founding+event+exhibits+stochastic+variation+inconsistent+with+the+deterministic+quasi-species+model." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Population+structure+within+lineages+of+Wheat+streak+mosaic+virus+derived+from+a+common+founding+event+exhibits+stochastic+variation+inconsistent+with+the+deterministic+quasi-species+model.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0001586-Zanotto1" id="pone.0001586-Zanotto1"></a>Zanotto PM, Gibbs MJ, Gould EA, Holmes EC (1996) A reevaluation of the higher taxonomy of viruses based on RNA polymerases. J Virol  70: 6083–6096.  <ul class="find" data-citedArticleID="1054610"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=A+reevaluation+of+the+higher+taxonomy+of+viruses+based+on+RNA+polymerases.&amp;auth=&amp;atitle=A+reevaluation+of+the+higher+taxonomy+of+viruses+based+on+RNA+polymerases." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+reevaluation+of+the+higher+taxonomy+of+viruses+based+on+RNA+polymerases." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+reevaluation+of+the+higher+taxonomy+of+viruses+based+on+RNA+polymerases.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0001586-Fuji1" id="pone.0001586-Fuji1"></a>Fuji S, Inoue M, Yamamoto H, Furuya H, Naito H, et al.  (2003) Nucleotide sequences of the coat protein gene of potyviruses infecting Ornithogalum thyrsoides. Arch Virol  148: 613–621.  <ul class="find" data-citedArticleID="1054582" data-doi="10.1007/s00705-002-0961-9"><li><a href="http://dx.doi.org/10.1007/s00705-002-0961-9" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Nucleotide+sequences+of+the+coat+protein+gene+of+potyviruses+infecting+Ornithogalum+thyrsoides." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Nucleotide+sequences+of+the+coat+protein+gene+of+potyviruses+infecting+Ornithogalum+thyrsoides.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0001586-Fuji2" id="pone.0001586-Fuji2"></a>Fuji S, Yamamoto H, Furuya H, Naito H (2003) Characterization of a new potyvirus isolated from Chinese artichoke in Japan. Arch Virol  148: 2249–2255.  <ul class="find" data-citedArticleID="1054584" data-doi="10.1007/s00705-003-0169-7"><li><a href="http://dx.doi.org/10.1007/s00705-003-0169-7" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Characterization+of+a+new+potyvirus+isolated+from+Chinese+artichoke+in+Japan." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Characterization+of+a+new+potyvirus+isolated+from+Chinese+artichoke+in+Japan.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0001586-Adams1" id="pone.0001586-Adams1"></a>Adams MJ, Antoniw JF, Fauquet CM (2005) Molecular criteria for genus and species discrimination within the family Potyviridae. Arch Virol  150: 459–479.  <ul class="find" data-citedArticleID="1054566" data-doi="10.1007/s00705-004-0440-6"><li><a href="http://dx.doi.org/10.1007/s00705-004-0440-6" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Molecular+criteria+for+genus+and+species+discrimination+within+the+family+Potyviridae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Molecular+criteria+for+genus+and+species+discrimination+within+the+family+Potyviridae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0001586-Nicolas1" id="pone.0001586-Nicolas1"></a>Nicolas O, Laliberte JF (1991) The use of PCR for cloning of large cDNA fragments of turnip mosaic potyvirus. J Virol Methods  32: 57–66.  <ul class="find" data-citedArticleID="1054602" data-doi="10.1016/0166-0934(91)90185-3"><li><a href="http://dx.doi.org/10.1016/0166-0934(91)90185-3" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+use+of+PCR+for+cloning+of+large+cDNA+fragments+of+turnip+mosaic+potyvirus." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+use+of+PCR+for+cloning+of+large+cDNA+fragments+of+turnip+mosaic+potyvirus.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0001586-Langeveld1" id="pone.0001586-Langeveld1"></a>Langeveld SA, Dore JM, Memelink J, Derks AF, van der Vlugt CI, et al.  (1991) Identification of potyviruses using the polymerase chain reaction with degenerate primers. J Gen Virol  72: 1531–1541.  <ul class="find" data-citedArticleID="1054600" data-doi="10.1099/0022-1317-72-7-1531"><li><a href="http://dx.doi.org/10.1099/0022-1317-72-7-1531" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Identification+of+potyviruses+using+the+polymerase+chain+reaction+with+degenerate+primers." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Identification+of+potyviruses+using+the+polymerase+chain+reaction+with+degenerate+primers.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0001586-Pappu1" id="pone.0001586-Pappu1"></a>Pappu SS, Brand R, Pappu HR, Rybicki EP, Gough KH, et al.  (1993) A polymerase chain reaction method adapted for selective amplification and cloning of 3′ sequences of potyviral genomes: application to dasheen mosaic virus. J Virol Methods  41: 9–20.  <ul class="find" data-citedArticleID="1054604" data-doi="10.1016/0166-0934(93)90158-n"><li><a href="http://dx.doi.org/10.1016/0166-0934(93)90158-n" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+polymerase+chain+reaction+method+adapted+for+selective+amplification+and+cloning+of+3%E2%80%B2+sequences+of+potyviral+genomes%3A+application+to+dasheen+mosaic+virus." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+polymerase+chain+reaction+method+adapted+for+selective+amplification+and+cloning+of+3%E2%80%B2+sequences+of+potyviral+genomes%3A+application+to+dasheen+mosaic+virus.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0001586-Colinet1" id="pone.0001586-Colinet1"></a>Colinet D, Kummert J (1993) Identification of a sweet potato feathery mottle virus isolate from China (SPFMV-CH) by the polymerase chain reaction with degenerate primers. J Virol Methods  45: 149–159.  <ul class="find" data-citedArticleID="1054574" data-doi="10.1016/0166-0934(93)90099-d"><li><a href="http://dx.doi.org/10.1016/0166-0934(93)90099-d" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Identification+of+a+sweet+potato+feathery+mottle+virus+isolate+from+China+%28SPFMV-CH%29+by+the+polymerase+chain+reaction+with+degenerate+primers." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Identification+of+a+sweet+potato+feathery+mottle+virus+isolate+from+China+%28SPFMV-CH%29+by+the+polymerase+chain+reaction+with+degenerate+primers.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0001586-Gibbs1" id="pone.0001586-Gibbs1"></a>Gibbs A, Mackenzie A (1997) A primer pair for amplifying part of the genome of all potyvirids by RT-PCR. J Virol Methods  63: 9–16.  <ul class="find" data-citedArticleID="1054588" data-doi="10.1016/s0166-0934(96)02103-9"><li><a href="http://dx.doi.org/10.1016/s0166-0934(96)02103-9" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+primer+pair+for+amplifying+part+of+the+genome+of+all+potyvirids+by+RT-PCR." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+primer+pair+for+amplifying+part+of+the+genome+of+all+potyvirids+by+RT-PCR.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0001586-Chen1" id="pone.0001586-Chen1"></a>Chen J, Adams MJ (2001) A universal PCR primer to detect members of the Potyviridae and its use to examine the taxonomic status of several members of the family. Arch Virol  146: 757–766.  <ul class="find" data-citedArticleID="1054572" data-doi="10.1007/s007050170144"><li><a href="http://dx.doi.org/10.1007/s007050170144" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+universal+PCR+primer+to+detect+members+of+the+Potyviridae+and+its+use+to+examine+the+taxonomic+status+of+several+members+of+the+family." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+universal+PCR+primer+to+detect+members+of+the+Potyviridae+and+its+use+to+examine+the+taxonomic+status+of+several+members+of+the+family.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0001586-Shukla1" id="pone.0001586-Shukla1"></a>Shukla DD, Ward CW, Brunt AA (1994) The Potyviridae. Ed. Wallingford: CAB International.   <ul class="find-nolinks"></ul></li><li><span class="label">16.
              </span><a name="pone.0001586-Hall2" id="pone.0001586-Hall2"></a>Hall JS, French R, Hein GL, Morris TJ, Stenger DC (2001) Three distinct mechanisms facilitate genetic isolation of sympatric wheat streak mosaic virus lineages. Virology  282: 230–236.  <ul class="find" data-citedArticleID="1054594" data-doi="10.1006/viro.2001.0841"><li><a href="http://dx.doi.org/10.1006/viro.2001.0841" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Three+distinct+mechanisms+facilitate+genetic+isolation+of+sympatric+wheat+streak+mosaic+virus+lineages." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Three+distinct+mechanisms+facilitate+genetic+isolation+of+sympatric+wheat+streak+mosaic+virus+lineages.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0001586-Biebricher1" id="pone.0001586-Biebricher1"></a>Biebricher CK, Eigen M (2006) What is a quasispecies? Curr Top Microbiol Immunol  299: 1–31.  <ul class="find" data-citedArticleID="1054570"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=What+is+a+quasispecies%3F&amp;auth=&amp;atitle=What+is+a+quasispecies%3F" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=What+is+a+quasispecies%3F" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22What+is+a+quasispecies%3F%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0001586-Edgar1" id="pone.0001586-Edgar1"></a>Edgar RC (2004) MUSCLE: multiple sequence alignment with high accuracy and high throughput. Nucleic Acids Res  32: 1792–1797.  <ul class="find" data-citedArticleID="1054576" data-doi="10.1093/nar/gkh340"><li><a href="http://dx.doi.org/10.1093/nar/gkh340" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=MUSCLE%3A+multiple+sequence+alignment+with+high+accuracy+and+high+throughput." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22MUSCLE%3A+multiple+sequence+alignment+with+high+accuracy+and+high+throughput.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0001586-Altschul1" id="pone.0001586-Altschul1"></a>Altschul SF, Madden TL, Schäffer AA, Zhang J, Zhang Z, et al.  (1997) Gapped BLAST and PSI-BLAST: a new generation of protein database search programs. Nucleic Acids Res  25: 3389–3402.  <ul class="find" data-citedArticleID="1054568" data-doi="10.1093/nar/25.17.3389"><li><a href="http://dx.doi.org/10.1093/nar/25.17.3389" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Gapped+BLAST+and+PSI-BLAST%3A+a+new+generation+of+protein+database+search+programs." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Gapped+BLAST+and+PSI-BLAST%3A+a+new+generation+of+protein+database+search+programs.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0001586-Kumar1" id="pone.0001586-Kumar1"></a>Kumar S, Tamura K, Nei M (2004) MEGA3: Integrated software for Molecular Evolutionary Genetics Analysis and sequence alignment. Brief Bioinform  5: 150–163.  <ul class="find" data-citedArticleID="1054598" data-doi="10.1093/bib/5.2.150"><li><a href="http://dx.doi.org/10.1093/bib/5.2.150" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=MEGA3%3A+Integrated+software+for+Molecular+Evolutionary+Genetics+Analysis+and+sequence+alignment." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22MEGA3%3A+Integrated+software+for+Molecular+Evolutionary+Genetics+Analysis+and+sequence+alignment.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0001586-Hall3" id="pone.0001586-Hall3"></a>Hall T (2005) BioEdit: Biological Sequence Alignment Editor for Windows 95/98/NT/XP.   Available: <a href="http://www.mbio.ncsu.edu/BioEdit/bioedit.html">http://www.mbio.ncsu.edu/BioEdit/bioedit​.html</a>.  <ul class="find-nolinks"></ul></li><li><span class="label">22.
              </span><a name="pone.0001586-Shannon1" id="pone.0001586-Shannon1"></a>Shannon CE (1949) The Mathematical Theory of Communication. University of Illinois Press. Urbana: pp. 379–423.  <ul class="find-nolinks"></ul></li><li><span class="label">23.
              </span><a name="pone.0001586-Gibbs2" id="pone.0001586-Gibbs2"></a>Gibbs A, Armstrong J, Mackenzie AM, Weiller GF (1998) The GPRIME package: computer programs for identifying the best regions of aligned genes to target in nucleic acid hybridisation-based diagnostic tests, and their use with plant viruses. J Virol Methods  74: 67–76.  <ul class="find" data-citedArticleID="1054590" data-doi="10.1016/s0166-0934(98)00070-6"><li><a href="http://dx.doi.org/10.1016/s0166-0934(98)00070-6" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+GPRIME+package%3A+computer+programs+for+identifying+the+best+regions+of+aligned+genes+to+target+in+nucleic+acid+hybridisation-based+diagnostic+tests%2C+and+their+use+with+plant+viruses." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+GPRIME+package%3A+computer+programs+for+identifying+the+best+regions+of+aligned+genes+to+target+in+nucleic+acid+hybridisation-based+diagnostic+tests%2C+and+their+use+with+plant+viruses.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001586.XML" value="104287"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001586.PDF" value="277987"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001586.t001.PNG_L" value="442231"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001586.t001.PNG_M" value="250416"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001586.t001.PNG_S" value="17966"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001586.t001.TIF" value="1730098"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001586.t001.PNG_I" value="76883"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001586.g001.PNG_L" value="462821"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001586.g001.PNG_M" value="59184"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001586.g001.PNG_S" value="6863"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001586.g001.TIF" value="1296608"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001586.g001.PNG_I" value="20978"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001586.g002.PNG_L" value="496767"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001586.g002.PNG_M" value="64558"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001586.g002.PNG_S" value="7046"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001586.g002.TIF" value="1328434"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001586.g002.PNG_I" value="22927"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001586.g003.PNG_L" value="820300"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001586.g003.PNG_M" value="103225"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001586.g003.PNG_S" value="8432"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001586.g003.TIF" value="1423438"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001586.g003.PNG_I" value="37129"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001586.t002.PNG_L" value="255598"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001586.t002.PNG_M" value="197881"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001586.t002.PNG_S" value="17572"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001586.t002.TIF" value="1232038"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001586.t002.PNG_I" value="63960"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001586.g004.PNG_L" value="746360"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001586.g004.PNG_M" value="93785"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001586.g004.PNG_S" value="8691"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001586.g004.TIF" value="1341066"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001586.g004.PNG_I" value="33088"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001586.t003.PNG_L" value="124184"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001586.t003.PNG_M" value="123127"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001586.t003.PNG_S" value="19124"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001586.t003.TIF" value="657556"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001586.t003.PNG_I" value="68261"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001586.s001.DOC" value="99840"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001586.s002.TIF" value="3890830"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001586.s003.TIF" value="3457360"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001586.s004.TIF" value="2697120"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001586&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001586" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001586&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0001586&volume=&issue=&title=Accumulating Variation at Conserved Sites in Potyvirus Genomes Is Driven by Species Discovery and Affects Degenerate Primer Design&author_name=Linda%20Zheng%2C%20Paul%20J.%20Wayper%2C%20Adrian%20J.%20Gibbs%2C%20Mathieu%20Fourment%2C%20Brendan%20C.%20Rodoni%2C%20Mark%20J.%20Gibbs&start_page=1&end_page=8" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001586" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001586" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0001586" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001586&amp;t=Accumulating%20Variation%20at%20Conserved%20Sites%20in%20Potyvirus%20Genomes%20Is%20Driven%20by%20Species%20Discovery%20and%20Affects%20Degenerate%20Primer%20Design" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001586&title=Accumulating%20Variation%20at%20Conserved%20Sites%20in%20Potyvirus%20Genomes%20Is%20Driven%20by%20Species%20Discovery%20and%20Affects%20Degenerate%20Primer%20Design&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001586&amp;title=Accumulating%20Variation%20at%20Conserved%20Sites%20in%20Potyvirus%20Genomes%20Is%20Driven%20by%20Species%20Discovery%20and%20Affects%20Degenerate%20Primer%20Design" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001586" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0001586&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Accumulating Variation at Conserved Sites in Potyvirus Genomes Is Driven by Species Discovery and Affects Degenerate Primer Design';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0001586';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Accumulating%20Variation%20at%20Conserved%20Sites%20in%20Potyvirus%20Genomes%20Is%20Driven%20by%20Species%20Discovery%20and%20Affects%20Degenerate%20Primer%20Design http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001586" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0001586" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">
















          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Conservation+science%22" title="Search for articles in the subject area:'Conservation science'"><div class="flagText">Conservation science</div></a>
              <div data-categoryid="40107" data-articleid="25998"
                   data-categoryname="Conservation science"
                   class="flagImage" title="Flag 'Conservation science' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22DNA+sequences%22" title="Search for articles in the subject area:'DNA sequences'"><div class="flagText">DNA sequences</div></a>
              <div data-categoryid="32003" data-articleid="25998"
                   data-categoryname="DNA sequences"
                   class="flagImage" title="Flag 'DNA sequences' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Genomic+databases%22" title="Search for articles in the subject area:'Genomic databases'"><div class="flagText">Genomic databases</div></a>
              <div data-categoryid="42727" data-articleid="25998"
                   data-categoryname="Genomic databases"
                   class="flagImage" title="Flag 'Genomic databases' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Nucleotide+sequencing%22" title="Search for articles in the subject area:'Nucleotide sequencing'"><div class="flagText">Nucleotide sequencing</div></a>
              <div data-categoryid="19663" data-articleid="25998"
                   data-categoryname="Nucleotide sequencing"
                   class="flagImage" title="Flag 'Nucleotide sequencing' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Potyvirus%22" title="Search for articles in the subject area:'Potyvirus'"><div class="flagText">Potyvirus</div></a>
              <div data-categoryid="43453" data-articleid="25998"
                   data-categoryname="Potyvirus"
                   class="flagImage" title="Flag 'Potyvirus' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Sequence+alignment%22" title="Search for articles in the subject area:'Sequence alignment'"><div class="flagText">Sequence alignment</div></a>
              <div data-categoryid="18051" data-articleid="25998"
                   data-categoryname="Sequence alignment"
                   class="flagImage" title="Flag 'Sequence alignment' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Sequence+databases%22" title="Search for articles in the subject area:'Sequence databases'"><div class="flagText">Sequence databases</div></a>
              <div data-categoryid="42787" data-articleid="25998"
                   data-categoryname="Sequence databases"
                   class="flagImage" title="Flag 'Sequence databases' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Sequence+motif+analysis%22" title="Search for articles in the subject area:'Sequence motif analysis'"><div class="flagText">Sequence motif analysis</div></a>
              <div data-categoryid="19657" data-articleid="25998"
                   data-categoryname="Sequence motif analysis"
                   class="flagImage" title="Flag 'Sequence motif analysis' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=7789'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=1858'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=4039&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>

<div class="block sidebar-comments">
    <div class="header">
        <h3>Comments</h3>
    </div>
      <p><a href="/annotation/listThread.action?root=19197">Referee comments: Referee 2</a><br>Posted by PLoS_ONE_Group</p>
</div>

</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
