

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Genome Evolution and the Emergence of Fruiting Body Development in Myxococcus xanthus</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0001329"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0001329" />

    <meta name="citation_title" content="Genome Evolution and the Emergence of Fruiting Body Development in Myxococcus xanthus"/>
    <meta itemprop="name" content="Genome Evolution and the Emergence of Fruiting Body Development in Myxococcus xanthus"/>

      <meta name="citation_author" content="Barry Goldman"/>
            <meta name="citation_author_institution" content="Applied Bioinformatics, Monsanto Company, St. Louis, Missouri, United States of America"/>
      <meta name="citation_author" content="Swapna Bhat"/>
            <meta name="citation_author_institution" content="Department of Microbiology, University of Georgia, Athens, Georgia, United States of America"/>
      <meta name="citation_author" content="Lawrence J. Shimkets"/>
            <meta name="citation_author_institution" content="Department of Microbiology, University of Georgia, Athens, Georgia, United States of America"/>

    <meta name="citation_date" content="2007/12/26"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0001329.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e1329"/>
    <meta name="citation_issue" content="12"/>
    <meta name="citation_volume" content="2"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Duplication and divergence: the evolution of new genes and old ideas.; citation_author=JS Taylor; citation_author=J Raes; citation_journal_title=Annu Rev Genet; citation_volume=38; citation_number=1; citation_pages=615-643; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Gene duplication and biased functional retention of paralogs in bacterial genomes.; citation_author=D Gevers; citation_author=K Vandepoele; citation_author=C Simillon; citation_author=Y Van de Peer; citation_journal_title=Trends Microbiol; citation_volume=12; citation_number=2; citation_pages=148-154; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=The ins and outs of DNA transfer in bacteria.; citation_author=I Chen; citation_author=PJ Christie; citation_author=D Dubnau; citation_journal_title=Science; citation_volume=310; citation_number=3; citation_pages=1456-1460; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Lateral gene transfer and the nature of bacterial innovation.; citation_author=H Ochman; citation_author=JG Lawrence; citation_author=EA Groisman; citation_journal_title=Nature; citation_volume=405; citation_number=4; citation_pages=299-304; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Evolutionary origins of genomic repertoires in bacteria.; citation_author=E Lerat; citation_author=V Daubin; citation_author=H Ochman; citation_author=NA Moran; citation_journal_title=PLoS Biol; citation_volume=3; citation_number=5; citation_pages=e130; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Horizontal gene transfer and the origin of species: lessons from bacteria.; citation_author=F de la Cruz; citation_author=J Davies; citation_journal_title=Trends Microbiol; citation_volume=8; citation_number=6; citation_pages=128-133; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Lateral gene transfer and the origins of prokaryotic groups.; citation_author=Y Boucher; citation_author=CJ Douady; citation_author=RT Papke; citation_author=DA Walsh; citation_author=ME Boudreau; citation_journal_title=Annu Rev Genet; citation_volume=37; citation_number=7; citation_pages=283-328; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Evolution of sensory complexity recorded in a myxobacterial genome.; citation_author=BS Goldman; citation_author=WC Nierman; citation_author=D Kaiser; citation_author=SC Slater; citation_author=AS Durkin; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=103; citation_number=8; citation_pages=15200-15205; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Amelioration of bacterial genomes: rates of change and exchange.; citation_author=JG Lawrence; citation_author=H Ochman; citation_journal_title=J Mol Evol; citation_volume=44; citation_number=9; citation_pages=383-397; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Successful lateral transfer requires codon usage compatibility between foreign genes and recipient genomes.; citation_author=A Medrano-Soto; citation_author=G Moreno-Hagelsieb; citation_author=P Vinuesa; citation_author=JA Christen; citation_author=J Collado-Vides; citation_journal_title=Mol Biol Evol; citation_volume=21; citation_number=10; citation_pages=1884-1894; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Predicted highly expressed genes of diverse prokaryotic genomes.; citation_author=S Karlin; citation_author=J Mrazek; citation_journal_title=J Bacteriol; citation_volume=182; citation_number=11; citation_pages=5238-5250; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Detecting alien genes in bacterial genomes.; citation_author=J Mrazek; citation_author=S Karlin; citation_journal_title=Ann N Y Acad Sci; citation_volume=870; citation_number=12; citation_pages=314-329; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Ancient lateral gene transfer in the evolution of Bdellovibrio bacteriovorus.; citation_author=U Gophna; citation_author=RL Charlebois; citation_author=WF Doolittle; citation_journal_title=Trends Microbiol; citation_volume=14; citation_number=13; citation_pages=64-69; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=In vivo transposition of mariner-based elements in enteric bacteria and mycobacteria.; citation_author=EJ Rubin; citation_author=BJ Akerley; citation_author=VN Novik; citation_author=DJ Lampe; citation_author=RN Husson; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=96; citation_number=14; citation_pages=1645-1650; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Identification of genes required for adventurous gliding motility in Myxococcus xanthus with the transposable element mariner.; citation_author=P Youderian; citation_author=N Burke; citation_author=DJ White; citation_author=PL Hartzell; citation_journal_title=Mol Microbiol; citation_volume=49; citation_number=15; citation_pages=555-570; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Transposon insertions of magellan-4 that impair social gliding motility in Myxococcus xanthus.; citation_author=P Youderian; citation_author=PL Hartzell; citation_journal_title=Genetics; citation_volume=172; citation_number=16; citation_pages=1397-1410; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Gliding motility and polarized slime secretion.; citation_author=R Yu; citation_author=D Kaiser; citation_journal_title=Mol Microbiol; citation_volume=63; citation_number=17; citation_pages=454-467; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=The Myxococcus xanthus dsg gene product performs functions of translation initiation factor IF3 in vivo.; citation_author=LV Kalman; citation_author=YL Cheng; citation_author=D Kaiser; citation_journal_title=J Bacteriol; citation_volume=176; citation_number=18; citation_pages=1434-1442; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=The dsg gene of Myxococcus xanthus encodes a protein similar to translation initiation factor IF3.; citation_author=YL Cheng; citation_author=LV Kalman; citation_author=D Kaiser; citation_journal_title=J Bacteriol; citation_volume=176; citation_number=19; citation_pages=1427-1433; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=dsg, a gene required for Myxococcus development, is necessary for cell viability.; citation_author=Y Cheng; citation_author=D Kaiser; citation_journal_title=J Bacteriol; citation_volume=171; citation_number=20; citation_pages=3727-3731; citation_date=1989; " />
      <meta name="citation_reference" content="citation_title=Isolation and identification of myxobacteria from soils and plant materials, with special reference to DNA base composition, quinone system, and cellular fatty acid composition, and with a description of a new species, Myxococcus flavescens.; citation_author=S Yamanaka; citation_author=A Kawaguchi; citation_author=K Komagata; citation_journal_title=J Gen Appl Microbiol; citation_volume=33; citation_number=21; citation_pages=247-265; citation_date=1987; " />
      <meta name="citation_reference" content="citation_title=Flexibacter elegans and Myxococcus fulvus: aerobic Gram-negative bacteria containing menaquinones as the only isoprenoid quinones.; citation_author=H Kleinig; citation_author=H Reichenbach; citation_author=N Theobald; citation_author=H Achenbach; citation_journal_title=Arch Microbiol; citation_volume=101; citation_number=22; citation_pages=91-93; citation_date=1974; " />
      <meta name="citation_reference" content="citation_title=New class of bacterial membrane oxidoreductases.; citation_author=MF Yanyushin; citation_author=MC del Rosario; citation_author=DC Brune; citation_author=RE Blankenship; citation_journal_title=Biochemistry; citation_volume=44; citation_number=23; citation_pages=10037-10045; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Oxygen defense in sulfate-reducing bacteria.; citation_author=A Dolla; citation_author=M Fournier; citation_author=Z Dermoun; citation_journal_title=J Biotechnol; citation_volume=126; citation_number=24; citation_pages=87-100; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Adaptive evolution of bacterial metabolic networks by horizontal gene transfer.; citation_author=C Pal; citation_author=B Papp; citation_author=MJ Lercher; citation_journal_title=Nat Genet; citation_volume=37; citation_number=25; citation_pages=1372-1375; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Highways of gene sharing in prokaryotes.; citation_author=RG Beiko; citation_author=TJ Harlow; citation_author=MA Ragan; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=102; citation_number=26; citation_pages=14332-14337; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Succinate dehydrogenase and fumarate reductase from Escherichia coli.; citation_author=G Cecchini; citation_author=I Schroder; citation_author=RP Gunsalus; citation_author=E Maklashina; citation_journal_title=Biochim Biophys Acta; citation_volume=1553; citation_number=27; citation_pages=140-157; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Identification of bacterial micropredators distinctively active in a soil microbial food web.; citation_author=T Lueders; citation_author=R Kindler; citation_author=A Miltner; citation_author=MW Friedrich; citation_author=M Kaestner; citation_journal_title=Appl Environ Microbiol; citation_volume=72; citation_number=28; citation_pages=5342-5348; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Quantitative phylogenetic assessment of microbial communities in diverse environments.; citation_author=C von Mering; citation_author=P Hugenholtz; citation_author=J Raes; citation_author=SG Tringe; citation_author=T Doerks; citation_journal_title=Science; citation_volume=315; citation_number=29; citation_pages=1126-1130; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Acidobacteria form a coherent but highly diverse group within the bacterial domain: evidence from environmental genomics.; citation_author=A Quaiser; citation_author=T Ochsenreiter; citation_author=C Lanz; citation_author=SC Schuster; citation_author=AH Treusch; citation_journal_title=Mol Microbiol; citation_volume=50; citation_number=30; citation_pages=563-575; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Toward automatic reconstruction of a highly resolved tree of life.; citation_author=FD Ciccarelli; citation_author=T Doerks; citation_author=C von Mering; citation_author=CJ Creevey; citation_author=B Snel; citation_journal_title=Science; citation_volume=311; citation_number=31; citation_pages=1283-1287; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=The Myxobacteria.; citation_author=LJ Shimkets; citation_author=M Dworkin; citation_author=H Reichenbach; citation_number=32; citation_pages=31-115; citation_date=2007; citation_publisher=Springer Verlag; " />
      <meta name="citation_reference" content="citation_title=Transfer of plasmid RP4 to Myxococcus xanthus and evidence for its integration into the chromosome.; citation_author=AM Breton; citation_author=S Jaoua; citation_author=J Guespin-Michel; citation_journal_title=J Bacteriol; citation_volume=161; citation_number=33; citation_pages=523-528; citation_date=1985; " />
      <meta name="citation_reference" content="citation_title=Developmental cell interactions in Myxococcus xanthus and the spoC locus.; citation_author=LJ Shimkets; citation_author=RE Gill; citation_author=D Kaiser; citation_journal_title=Proc Natl Acad Sci USA; citation_volume=80; citation_number=34; citation_pages=1406-1410; citation_date=1983; " />
      <meta name="citation_reference" content="citation_title=Search and discovery strategies for biotechnology: the paradigm shift.; citation_author=AT Bull; citation_author=AC Ward; citation_author=M Goodfellow; citation_journal_title=Microbiol Mol Biol Rev; citation_volume=64; citation_number=35; citation_pages=573-606; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Genetic suppression and phenotypic masking of a Myxococcus xanthus frzF- defect.; citation_author=K Kashefi; citation_author=PL Hartzell; citation_journal_title=Mol Microbiol; citation_volume=15; citation_number=36; citation_pages=483-494; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=The Che4 pathway of Myxococcus xanthus regulates type IV pilus-mediated motility.; citation_author=HC Vlamakis; citation_author=JR Kirby; citation_author=DR Zusman; citation_journal_title=Mol Microbiol; citation_volume=52; citation_number=37; citation_pages=1799-1811; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Isolation of bacteriophage MX4, a generalized transducing phage for Myxococcus xanthus.; citation_author=JM Campos; citation_author=J Geisselsoder; citation_author=DR Zusman; citation_journal_title=J Mol Biol; citation_volume=119; citation_number=38; citation_pages=167-178; citation_date=1978; " />
      <meta name="citation_reference" content="citation_title=Gapped BLAST and PSI-BLAST: a new generation of protein database search programs.; citation_author=SF Altschul; citation_author=TL Madden; citation_author=AA Schaffer; citation_author=J Zhang; citation_author=Z Zhang; citation_journal_title=Nucleic Acids Res; citation_volume=25; citation_number=39; citation_pages=3389-3402; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=MUSCLE: a multiple sequence alignment method with reduced time and space complexity.; citation_author=RC Edgar; citation_journal_title=BMC Bioinformatics; citation_volume=5; citation_number=40; citation_pages=113; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=MEGA3: Integrated software for Molecular Evolutionary Genetics Analysis and sequence alignment.; citation_author=S Kumar; citation_author=K Tamura; citation_author=M Nei; citation_journal_title=Brief Bioinform; citation_volume=5; citation_number=41; citation_pages=150-163; citation_date=2004; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001329" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Genome Evolution and the Emergence of Fruiting Body Development in Myxococcus xanthus"/>
    <meta name="twitter:description" content="BackgroundLateral gene transfer (LGT) is thought to promote speciation in bacteria, though well-defined examples have not been put forward.Methodology/Principle FindingsWe examined the evolutionary history of the genes essential for a trait that defines a phylogenetic order, namely fruiting body development of the Myxococcales. Seventy-eight genes that are essential for Myxococcus xanthus development were examined for LGT. About 73% of the genes exhibit a phylogeny similar to that of the 16S rDNA gene and a codon bias consistent with other M. xanthus genes suggesting vertical transmission. About 22% have an altered codon bias and/or phylogeny suggestive of LGT. The remaining 5% are unique. Genes encoding signal production and sensory transduction were more likely to be transmitted vertically with clear examples of duplication and divergence into multigene families. Genes encoding metabolic enzymes were frequently acquired by LGT. Myxobacteria exhibit aerobic respiration unlike most of the &delta; Proteobacteria. M. xanthus contains a unique electron transport pathway shaped by LGT of genes for succinate dehydrogenase and three cytochrome oxidase complexes.Conclusions/SignificanceFruiting body development depends on genes acquired by LGT, particularly those involved in polysaccharide production. We suggest that aerobic growth fostered innovation necessary for development by allowing myxobacteria access to a different gene pool from anaerobic members of the &delta; Proteobacteria. Habitat destruction and loss of species diversity could restrict the evolution of new bacterial groups by limiting the size of the prospective gene pool."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0001329.g004"/>

  <meta property="og:title" content="Genome Evolution and the Emergence of Fruiting Body Development in Myxococcus xanthus" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001329" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=7116'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=6703'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=1988&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0001329">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001329" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001329&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Genome Evolution and the Emergence of Fruiting Body Development in <i>Myxococcus xanthus</i>
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Barry Goldman, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Applied Bioinformatics, Monsanto Company, St. Louis, Missouri, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Swapna Bhat, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Microbiology, University of Georgia, Athens, Georgia, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Lawrence J. Shimkets
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:shimkets@uga.edu">shimkets@uga.edu</a></p>

                <p>Affiliation:
                  Department of Microbiology, University of Georgia, Athens, Georgia, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: December 26, 2007</li>
    <li>DOI: 10.1371/journal.pone.0001329</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0001329" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0001329" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001329" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0001329" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0001329-t001" data-doi="info:doi/10.1371/journal.pone.0001329" data-uri="info:doi/10.1371/journal.pone.0001329.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001329.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001329-g001" data-doi="info:doi/10.1371/journal.pone.0001329" data-uri="info:doi/10.1371/journal.pone.0001329.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001329.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001329-g002" data-doi="info:doi/10.1371/journal.pone.0001329" data-uri="info:doi/10.1371/journal.pone.0001329.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001329.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001329-g003" data-doi="info:doi/10.1371/journal.pone.0001329" data-uri="info:doi/10.1371/journal.pone.0001329.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001329.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001329-t002" data-doi="info:doi/10.1371/journal.pone.0001329" data-uri="info:doi/10.1371/journal.pone.0001329.t002" title="Table 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001329.t002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001329-g004" data-doi="info:doi/10.1371/journal.pone.0001329" data-uri="info:doi/10.1371/journal.pone.0001329.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001329.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001329">Reader Comments (0)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0001329" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2>
<h3>Background</h3>
<a id="article1.front1.article-meta1.abstract1.sec1.p1" name="article1.front1.article-meta1.abstract1.sec1.p1"></a><p>Lateral gene transfer (LGT) is thought to promote speciation in bacteria, though well-defined examples have not been put forward.</p>


<h3>Methodology/Principle Findings</h3>
<a id="article1.front1.article-meta1.abstract1.sec2.p1" name="article1.front1.article-meta1.abstract1.sec2.p1"></a><p>We examined the evolutionary history of the genes essential for a trait that defines a phylogenetic order, namely fruiting body development of the Myxococcales. Seventy-eight genes that are essential for <em>Myxococcus xanthus</em> development were examined for LGT. About 73% of the genes exhibit a phylogeny similar to that of the 16S rDNA gene and a codon bias consistent with other <em>M. xanthus</em> genes suggesting vertical transmission. About 22% have an altered codon bias and/or phylogeny suggestive of LGT. The remaining 5% are unique. Genes encoding signal production and sensory transduction were more likely to be transmitted vertically with clear examples of duplication and divergence into multigene families. Genes encoding metabolic enzymes were frequently acquired by LGT. Myxobacteria exhibit aerobic respiration unlike most of the δ Proteobacteria. <em>M. xanthus</em> contains a unique electron transport pathway shaped by LGT of genes for succinate dehydrogenase and three cytochrome oxidase complexes.</p>


<h3>Conclusions/Significance</h3>
<a id="article1.front1.article-meta1.abstract1.sec3.p1" name="article1.front1.article-meta1.abstract1.sec3.p1"></a><p>Fruiting body development depends on genes acquired by LGT, particularly those involved in polysaccharide production. We suggest that aerobic growth fostered innovation necessary for development by allowing myxobacteria access to a different gene pool from anaerobic members of the δ Proteobacteria. Habitat destruction and loss of species diversity could restrict the evolution of new bacterial groups by limiting the size of the prospective gene pool.</p>

</div>


<div class="articleinfo"><p><strong>Citation: </strong>Goldman B, Bhat S, Shimkets LJ (2007) Genome Evolution and the Emergence of Fruiting Body Development in <em>Myxococcus xanthus</em>. PLoS ONE 2(12):
          e1329.
            doi:10.1371/journal.pone.0001329</p><p><strong>Academic Editor: </strong>Joseph Petrosino, Baylor College of Medicine, United States of America</p><p><strong>Received:</strong> February 12, 2007; <strong>Accepted:</strong> November 13, 2007; <strong>Published:</strong> December 26, 2007</p><p><strong>Copyright:</strong> © 2007 Goldman et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>This work was supported by National Science Foundation grant 0343874.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>Speciation in higher organisms usually occurs in genetic isolation. Successive rounds of gene duplication and divergence, followed by individual gene loss, is thought to have contributed to morphological diversification <a href="#pone.0001329-Taylor1">[1]</a>. Speciation has not been as extensively studied in bacteria. Small gene duplication events have been noted in many sequenced bacterial genomes <a href="#pone.0001329-Gevers1">[2]</a> but lateral gene transfer (LGT) also contributes to genome diversification. Bacteria exchange genes by conjugation, transformation, and transduction which are widespread in nature and can occur between distantly related organisms <a href="#pone.0001329-Chen1">[3]</a>. Genome sequencing has revealed substantial rates of foreign gene acquisition <a href="#pone.0001329-Ochman1">[4]</a>. Analysis of all gene phylogenies in the sequenced members of the γ Proteobacteria indicated that LGT rather than gene duplication provided most of the diversity in genomic repertoires <a href="#pone.0001329-Lerat1">[5]</a>. LGT also has the potential to drive metabolic innovation. The dissemination of antibiotic resistance genes on a global scale is the paradigm, but examples of disseminated gene clusters for metabolic pathways and pathogenicity determinants also exist <a href="#pone.0001329-delaCruz1">[6]</a>, <a href="#pone.0001329-Boucher1">[7]</a>. These results provide compelling evidence that LGT can influence genotype evolution at the level of sub-speciation and by extension LGT may aid bacterial speciation.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>While gene duplication in <em>M. xanthus</em> led to the production of several large multi-gene families <a href="#pone.0001329-Goldman1">[8]</a>, the role of LGT in <em>M. xanthus</em> genome evolution has not been evaluated. LGT is generally supported if a phylogenetic tree for a gene is in disagreement with that of the 16S rDNA gene, although incongruent trees can also arise by gene loss from related lineages. The other method to detect LGT relies on differences in the nucleotide sequence composition of a gene relative to average of that particular host. Given enough time acquired genes converge in codon usage with the bulk of the genome by amelioration <a href="#pone.0001329-Lawrence1">[9]</a>, which make ancient transfers difficult to detect with compositional algorithms. Furthermore, LGT may be most successful when foreign genes and recipient genomes have similar codon usage <a href="#pone.0001329-MedranoSoto1">[10]</a>. Nevertheless, genes with codon usage that differs from that of the host are excellent candidates for LGT <a href="#pone.0001329-Karlin1">[11]</a>, <a href="#pone.0001329-Mrazek1">[12]</a>.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>Prokaryotic evolution is the product of environmental pressures combined with changes in the gene repertoire over time. The myxobacteria provide an opportunity to examine genome evolution within the context of speciation because fruiting body development is a unique trait that defines a phylogenetic order. In this work we examined the evolutionary history of <em>M. xanthus</em> genes required for fruiting body development using compositional and BLAST algorithms. Our results suggest that genes for metabolic enzymes, particularly those involved in polysaccharide biosynthesis, were more likely to be acquired by LGT and genes for sensory systems were more likely to be vertically inherited with numerous examples of duplication and divergence.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3><a id="article1.body1.sec2.p1" name="article1.body1.sec2.p1"></a><p><em>M. xanthus</em> and other myxobacteria are members of the δ Proteobacteria, one of the most diverse groups of Bacteria in terms of habitat distribution and respiration strategies. Hundreds of δ Proteobacteria species are known, but for the sake of brevity, only those genera with a complete genome sequence will be mentioned here. With the exception of the myxobacteria and <em>Bdellovibrio</em> most of the genera, including <em>Anaeromyxobacter, Desulfotalea, Desulfovibrio, Geobacter, Lawsonia, Pelobacter,</em> and <em>Syntrophus</em> are anaerobic (use terminal electron acceptors other than O<sub>2</sub>). Many are also chemolithotrophs (use inorganic compounds as energy sources) and autotrophs (derive all their carbon from CO<sub>2</sub>). While the <em>modus operandi</em> for most genera in the δ Proteobacteria is metabolic diversity, the myxobacteria are neither chemolithotrophs nor autotrophs and instead devote their resources to social cooperativity directed toward predation and the construction of a unique multicellular structure, the asexual fruiting body.</p>
<a id="article1.body1.sec2.p2" name="article1.body1.sec2.p2"></a><p>A phylogenetic approach was initially used to identify possible cases of LGT in <em>M. xanthus</em>. Similarity searches using the BLASTP algorithm, with a 10<sup>−10</sup> cutoff, were performed with each putative <em>M. xanthus</em> protein. The top four BLASTP hits were used because the top hit does not always represent the closest homolog and more distant hits can be widely diverged from the <em>M. xanthus</em> query sequence. One or more of the top four hits was a δ Proteobacteria for 55.0% of the genes suggesting a vertical descent as the inheritance paradigm for over half of the <em>M. xanthus</em> genes. Genomes also contain unique genes, in this case 22.2%, that have not been found in any other organism (so called ORFans). The remaining gene products (22.8%) have no homologs among δ Proteobacteria in the top 4 BLASTP hits, but often striking homology with gene products from other organisms suggestive of an alien origin.</p>
<a id="article1.body1.sec2.p3" name="article1.body1.sec2.p3"></a><p>The same phylogenetic analysis was applied to other members of the δ Proteobacteria. With the exception of <em>Bdellovibrio</em>, which contains 43.0% δ Proteobacteria genes, other genera in the δ Proteobacteria contained a higher proportion of δ Proteobacteria genes, including <em>Anaeromyxobacter</em> (60.7%), <em>Desulfovibrio</em> (84.1%), <em>Geobacter</em> (90.2%), <em>Lawsonia</em> (83.6%), and <em>Pelobacter</em> (74.7%). <em>Bdellovibrio</em> is thought to have acquired an unusually large number of genes via LGT <a href="#pone.0001329-Gophna1">[13]</a>. These results suggest that <em>M. xanthus</em> has received more alien genes by LGT than most other δ Proteobacteria.</p>

<h4>Fruiting body development</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>The myxobacteria represent an evolutionary branch that culminated in a unique and striking form of multicellularity. The evolutionary history of genes known to be required for fruiting body development was examined in order to glean some insight into principles guiding the evolution of this group of organisms. Only genes in which mutations diminish the capacity to form fruiting bodies or cause at least a 10-fold decrease in sporulation are included here. Many genes that affect the timing of development, but not the integrity of the end product, have been reported but were not considered here.</p>
<a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2"></a><p>The majority of the genes examined in this study have been published previously and are the subject of ongoing investigations into the mechanism of fruiting body development. This group of genes was supplemented with developmental mutants identified using the mariner-based transposable element <em>magellan-4</em> <a href="#pone.0001329-Rubin1">[14]</a>, which was used previously to identify motility genes in <em>M. xanthus</em> <a href="#pone.0001329-Youderian1">[15]</a>–<a href="#pone.0001329-Yu1">[17]</a>. 40,000 <em>magellan-4</em> insertions were screened for loss of fruiting body development. Mutant genes were backcrossed into the wild type DK1622 to confirm that the transposon causes the mutant phenotype. These genes, together with previously identified developmental genes, are given in <a href="#pone-0001329-t001">table 1</a>. The phylogeny of each gene was examined using the BLASTP algorithm. Genes with a codon usage that differed from that of the majority of the host genes were identified with software developed by Karlin and Mrazek <a href="#pone.0001329-Karlin1">[11]</a>, <a href="#pone.0001329-Mrazek1">[12]</a>.</p>
<div class="figure" id="pone-0001329-t001"><div class="img"><a name="pone-0001329-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001329.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001329" data-uri="info:doi/10.1371/journal.pone.0001329.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001329.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001329.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001329.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001329.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001329.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001329.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001329.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001329.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001329.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>Genes required for <em>M. xanthus</em> fruiting body development.</span></strong></p><span>doi:10.1371/journal.pone.0001329.t001</span></div><a id="article1.body1.sec2.sec1.p3" name="article1.body1.sec2.sec1.p3"></a><p>Among the genes required for development very few exhibited a codon bias suggestive of LGT (<a href="#pone-0001329-t001">Table 1</a>). The exceptions include a unique gene, MXAN4486, and MXAN0581, which encodes the essential translation initiation factor InfC that is found in other δ Proteobacteria. However, the <em>M. xanthus</em> InfC protein is unique in that it contains a 66 amino acid C-terminal extension that is essential for development but not translation <a href="#pone.0001329-Kalman1">[18]</a>–<a href="#pone.0001329-Cheng2">[20]</a>. The precise function of this extension remains unknown, but is thought to aid in generating an essential developmental signal, the D-signal. We were unable to identify another InfC protein with a similar extension in the database.</p>
<a id="article1.body1.sec2.sec1.p4" name="article1.body1.sec2.sec1.p4"></a><p>While the remaining genes had a normal codon bias, the tBLASTn algorithm suggested that some were acquired by LGT because their closest relatives in the database were not members of the δ Proteobacteria (<a href="#pone-0001329-t001">table 1</a>). The contributions of LGT and vertical inheritance to the pool of genes for fruiting body development can be approximated from these data. 57 genes (73%) have a δ Proteobacteria phylogeny and a normal codon bias suggestive of vertical inheritance. 17 genes (22%) have either abnormal codon bias or produce protein products whose closest relatives are from another bacterial group. The remaining 2 genes (5%) have a normal codon bias but are unique and their phylogenetic source could not be identified. These results suggest that <em>M. xanthus</em> fruiting body development is not possible without alien genes.</p>
<a id="article1.body1.sec2.sec1.p5" name="article1.body1.sec2.sec1.p5"></a><p>When the genes were examined within the context of their function two striking correlations were observed. Genes encoding essential developmental enzymes were often acquired by LGT (<a href="#pone-0001329-t001">table 1</a>, 7/14, 50%). Most notably this includes genes required for exopolysaccharide (<em>eps</em>) and lipopolysaccharide (<em>rfb</em>) biogenesis. On the other hand most of the genes required for signal production and sensory transduction have a phylogeny rooted in the δ Proteobacteria and a normal codon bias (<a href="#pone-0001329-t001">table 1</a>, 37/41, 90%). Many of these genes have several paralogs in <em>M. xanthus</em> and seem to be products of duplication and divergence. These include two component systems, serine/threonine protein kinase systems, sigma 54-dependent response regulators, and chemosensory pathways <a href="#pone.0001329-Goldman1">[8]</a>.</p>
<a id="article1.body1.sec2.sec1.p6" name="article1.body1.sec2.sec1.p6"></a><p>The phylogenetic sources of the alien developmental genes can suggest commonalities about the pool of donors. In some cases a clear gene source does not exist. For example, the MXAN4621 and MXAN5125 proteins have similar amino acid identities with proteins from widely different phyla (<a href="#pone-0001329-t001">table 1</a>). Nevertheless the phylogenic origin of each alien gene in the genome was explored systematically to determine whether any trends exist. The phyla containing the top four homologs of each alien gene are plotted in <a href="#pone-0001329-g001">figure 1</a> for <em>Myxococcus</em>, <em>Anaeromyxobacter</em> and <em>Bdellovibrio</em>. Relative to the latter two organisms, <em>Myxococcus</em> has an unusually high abundance of genes whose closest relatives are found in the Actinobacteria and Cyanobacteria. Most soil Actinobacteria (like <em>Streptomyces</em>) are obligate aerobes. Cyanobacteria also live in aerobic habitats since they generate O<sub>2</sub> during photosynthesis. These results suggest that <em>M. xanthus</em> acquired genes from organisms in aerobic habitats.</p>
<div class="figure" id="pone-0001329-g001"><div class="img"><a name="pone-0001329-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001329.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001329" data-uri="info:doi/10.1371/journal.pone.0001329.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001329.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001329.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001329.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001329.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001329.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001329.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001329.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001329.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001329.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Taxonomic distribution of best normalized BLASTP matches outside the δ Proteobacteria.</span></strong></p><a id="article1.body1.sec2.sec1.fig1.caption1.p1" name="article1.body1.sec2.sec1.fig1.caption1.p1"></a><p>Bars indicate the number of best matches in a phylum or domain for <em>Myxococcus xanthus</em> (ivory bars), <em>Anaeromyxobacter dehalogenans</em> (magenta bars), and <em>Bdellovibrio bacteriovorus</em> (blue bars). Competitive matching was used for the detection of best hits using an E-value threshold of 10<sup>−10</sup> and a minimum BLASTP score of 10<sup>−10</sup></p>
<span>doi:10.1371/journal.pone.0001329.g001</span></div>

<h4>Aerobic respiration</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>How alien genes are functionally integrated into existing pathways was examined with a less complex pathway, the electron transport chain. The nearly unique presence of aerobic respiration among the δ Proteobacteria, suggested that myxobacterial electron transport genes were derived by LGT. Examination of electron transport is also expected to reveal how recently acquired genes are integrated into functional networks. Was the entire pathway captured as a functional unit or were individual components acquired and used to replace existing components?</p>
<a id="article1.body1.sec2.sec2.p2" name="article1.body1.sec2.sec2.p2"></a><p>The protein components of the <em>M. xanthus</em> electron transport system were identified through comparative phylogeny. The upstream elements, NADH dehydrogenase (complex I) and succinate dehydrogenase (complex II), are common to many electron transport pathways (<a href="#pone-0001329-g002">figure 2</a>). The principal <em>Myxococcus</em> quinone is a menaquinone with 8 isoprenoid units <a href="#pone.0001329-Yamanaka1">[21]</a>, <a href="#pone.0001329-Kleinig1">[22]</a>. The δ Proteobacteria along with several other phyla lack the traditional <em>bc</em><sub>1</sub>-complex III and contain a newly described complex III containing a five-heme cytochrome <em>c</em>, a one-heme cytochrome <em>c</em>, and a three-subunit molybdopterin oxidoreductase <a href="#pone.0001329-Yanyushin1">[23]</a>. Genes encoding three copies of this complex, potentially one for each cytochrome <em>c</em> oxidase complex (denoted MF1<em>cc</em> in <a href="#pone-0001329-g003">figure 3</a>) are found in <em>M. xanthus</em> <a href="#pone.0001329-Yanyushin1">[23]</a>. There are two broad classes of cytochrome oxidase, which reduces O<sub>2</sub> to H<sub>2</sub>O (complex IV). Cytochrome <em>c</em> oxidase is coupled to the quinol:cytochrome <em>c</em> oxidoreductase (MF1<em>cc</em>) complex III. <em>M. xanthus</em> contains genes for three cytochrome <em>c</em> oxidases of this type, two <em>coxBAC</em> operons, and a <em>cbb</em><sub>3</sub> operon (<a href="#pone-0001329-t002">table 2</a>). The quinol oxidases derive electrons directly from the quinone pool and <em>M. xanthus</em> contains genes for both subunits of cytochrome <em>d</em> quinol oxidase (<em>cydAB</em>, <a href="#pone-0001329-t002">table 2</a>).</p>
<div class="figure" id="pone-0001329-g002"><div class="img"><a name="pone-0001329-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001329.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001329" data-uri="info:doi/10.1371/journal.pone.0001329.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001329.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001329.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001329.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001329.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001329.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001329.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001329.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001329.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001329.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>The <em>Myxococcus xanthus</em> electron transport chain.</span></strong></p><a id="article1.body1.sec2.sec2.fig1.caption1.p1" name="article1.body1.sec2.sec2.fig1.caption1.p1"></a><p>Phylogenetic trees were generated from individual Complex IV proteins (gray boxes). See text for detailed description.</p>
<span>doi:10.1371/journal.pone.0001329.g002</span></div><div class="figure" id="pone-0001329-g003"><div class="img"><a name="pone-0001329-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001329.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001329" data-uri="info:doi/10.1371/journal.pone.0001329.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001329.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001329.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001329.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001329.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001329.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001329.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001329.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001329.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001329.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Neighbor-Joining Tree of the CydA Subunit of the predicted cytochrome <em>bd</em> quinol oxidase.</span></strong></p><a id="article1.body1.sec2.sec2.fig2.caption1.p1" name="article1.body1.sec2.sec2.fig2.caption1.p1"></a><p>The tree was generated by identifying the twenty most similar homologs in the non-redundant database at NCBI using the BLASTP algorithm as of Sept. 2006. Proteins were aligned using the Muscle algorithm. Bootstrap values and visualization were generated by the Mega 3.1 program. Tree branches were condensed for those with a bootstrap value &gt;50%. Red filled circles are next to members of the δ Proteobacteria. <em>Anaeromyxobacter1-Anaeromyxobacter dehalogenans 2CP-C YP_464151.1; Anaeromyxobacter2-Anaeromyxobacter dehalogenans 2CP-C YP_466933.1; B_halodurans-Bacillus halodurans C-125; Bacillus_sp-Bacillus sp. NRRL B-14911; Bacillus_subtilis-Bacillus subtilis subsp. subtilis str. 168; Blastopirellula-Blastopirellula marina DSM 3645; C_muridarum-Chlamydia muridarum Nigg; C_trachomatis-Chlamydia trachomatis A/HAR-13; C_trachomatis_D-Chlamydia trachomatis D/UW-3/CX; D_desulfuricans-Desulfovibrio desulfuricans G20; D_gigas-Desulfovibrio gigas; D_vulgaris-Desulfovibrio vulgaris subsp. vulgaris str. Hildenborough; Frankia-Frankia sp. EAN1pec; G_kaustophilus1-Geobacillus kaustophilus HTA426 YP_147937.1; G_kaustophilus2-Geobacillus kaustophilus HTA426 YP_146457.1; G_metallireducens-Geobacter metallireducens GS-15; G_stearothermophilus-Geobacillus stearothermophilus; G_sulfurreducens-Geobacter sulfurreducens PCA; G_uraniumreducens-Geobacter uraniumreducens Rf4; Geobacter_sp.-Geobacter sp. FRC-32; Lawsonia-Lawsonia intracellularis PHE/MN1-00; Myxococcus-Myxococcus xanthus DK 1622; Nocardia-Nocardia farcinica IFM 10152; Oceanobacillus-Oceanobacillus iheyensis HTE831; P_carbinolicus-Pelobacter carbinolicus DSM 2380; P_propionicus-Pelobacter propionicus DSM 2379; S_fumaroxidans1-Syntrophobacter fumaroxidans MPOB ZP_00668206.1; S_fumaroxidans2-Syntrophobacter fumaroxidans MPOB ZP_00668271.1; Solibacter-Solibacter usitatus Ellin6076; Staphylococcus-Staphylococcus epidermidis ATCC 12228; Symbiobacterium-Symbiobacterium thermophilum IAM 14863; Syntrophus-Syntrophus aciditrophicus SB</em></p>
<span>doi:10.1371/journal.pone.0001329.g003</span></div><div class="figure" id="pone-0001329-t002"><div class="img"><a name="pone-0001329-t002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001329.t002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001329" data-uri="info:doi/10.1371/journal.pone.0001329.t002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001329.t002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001329.t002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001329.t002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001329.t002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001329.t002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001329.t002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001329.t002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001329.t002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001329.t002.TIF"></span>)
                </a></li></ul></div><p><strong>Table 2.  <span>Cytochrome oxidase proteins of <em>M. xanthus</em>.</span></strong></p><span>doi:10.1371/journal.pone.0001329.t002</span></div><a id="article1.body1.sec2.sec2.p3" name="article1.body1.sec2.sec2.p3"></a><p>The origin of each <em>M. xanthus</em> electron transport chain gene was examined using both phylogenetic and compositional approaches. In all cases the codon usage was consistent with other highly expressed <em>M. xanthus</em> genes which is not surprising given that these are essential genes. However, the phylogenetic approach indicated that the succinate dehydrogenase proteins SdhA (MXAN3539), SdhB (MXAN3540), and SdhC (MXAN1072), which together form complex II, are most similar to their counterparts in aerobic Firmicutes and Actinobacteria and are not present in any other δ Proteobacteria (not shown). Furthermore, the cytochrome oxidase genes (complex IV) were acquired from diverse phylogenetic sources. In contrast, NADH dehydrogenase (complex I) and the MF1<em>cc</em> quinol:cytochrome <em>c</em> oxidoreductase (complex III) are most similar to their counterparts in other δ Proteobacteria. These results argue that the electron transport pathway is encoded by a patchwork of genes, some inherited vertically and others acquired from diverse phylogenetic sources.</p>
<a id="article1.body1.sec2.sec2.p4" name="article1.body1.sec2.sec2.p4"></a><p>Cytochrome oxidase genes have been subject to intense scrutiny because of their dissemination by LGT over vast phylogenetic distances <a href="#pone.0001329-Boucher1">[7]</a>. The distribution of one subunit from each cytochrome oxidase was examined (bold subunits in <a href="#pone-0001329-t002">table 2</a>). Among the top twenty BLASTP hits to the <em>M. xanthus</em> cytochrome <em>cbb</em><sub>3</sub> oxidase, <em>Bdellovibrio</em> is the only δ Proteobacteria member (not shown) suggesting that this <em>cbb<sub>3</sub></em> oxidase was acquired by the only two aerobic members of this group.</p>
<a id="article1.body1.sec2.sec2.p5" name="article1.body1.sec2.sec2.p5"></a><p>A neighbor-joining tree of the CydA subunit of the cytochrome <em>d</em> quinol oxidase revealed two versions of this operon within the δ Proteobacteria (<a href="#pone-0001329-g003">figure 3</a>). One is found in all δ Proteobacteria with the exception of <em>M. xanthus</em> and is presumed to be ancient and vertically transmitted. <em>M. xanthus</em> appears to have lost this operon sometime after its divergence from <em>A. dehalogenans</em>. It is surprising to find this oxidase in anaerobic organisms where O<sub>2</sub> is highly toxic <a href="#pone.0001329-Dolla1">[24]</a>. A distinctly different <em>cydAB</em> operon is found in <em>A. dehalogenans</em> and <em>M. xanthus</em> and was likely incorporated by LGT prior to the divergence of the two organisms (<a href="#pone-0001329-g003">figure 3</a>). The <em>M. xanthus cydAB</em> operon falls in the same clade as diverse phylogenetic groups, most notably Actinobacteria and Planctomycetes.</p>
<a id="article1.body1.sec2.sec2.p6" name="article1.body1.sec2.sec2.p6"></a><p>The <em>coxBAC</em> operon is found in many members of the δ Proteobacteria (<a href="#pone-0001329-g004">figure 4</a>). Again, the presence of this operon is surprising for anaerobic species where the function is not known and suggests an ancient origin in the δ Proteobacteria. In <em>M. xanthus</em> and <em>A. dehalogenans</em> the <em>coxBAC</em> operon has a different phylogenetic history from that of the other δ Proteobacteria and falls in the same clade as an eclectic mixture of organisms from diverse phyla. The results are consistent with the idea that <em>Myxococcus</em> and <em>Anaeromyxobacter</em> have lost the ancient version of this operon and acquired a different one. In addition, the <em>coxBAC</em> operon has been duplicated in <em>M. xanthus</em>.</p>
<div class="figure" id="pone-0001329-g004"><div class="img"><a name="pone-0001329-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001329.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001329" data-uri="info:doi/10.1371/journal.pone.0001329.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001329.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001329.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001329.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001329.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001329.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001329.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001329.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001329.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001329.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>Neighbor-Joining Tree of Subunit I (CtaD or CoxA) of the predicted cytochrome <em>c</em> oxidase.</span></strong></p><a id="article1.body1.sec2.sec2.fig3.caption1.p1" name="article1.body1.sec2.sec2.fig3.caption1.p1"></a><p>The tree was generated by identifying the twenty most similar homologs in the non-redundant database at NCBI using the BLASTP algorithm as of Sept. 2006. Proteins were aligned using the Muscle algorithm. Bootstrap values and visualization were generated by the Mega 3.1 program. Tree branches were condensed for those with a bootstrap value &gt;50%. Small filled circles are next to members of the δ Proteobacteria. <em>Acidobacteria-Acidobacteria bacterium Ellin345; Anaeromyxobacter1-Anaeromyxobacter dehalogenans 2CP-C YP_464016.1; Anaeromyxobacter2-Anaeromyxobacter dehalogenans 2CP-C YP_465481.1; Bdellovibrio-Bdellovibrio bacteriovorus HD100; Blastopirellula-Blastopirellula marina DSM 3645; Chloroflexus-Chloroflexus aurantiacus J-10-fl; D_desulfuricans-Desulfovibrio desulfuricans G20; D_vulgaris1-Desulfovibrio vulgaris; D_vulgaris2-Desulfovibrio vulgaris subsp. vulgaris str. Hildenborough; G_metallireducens-Geobacter metallireducens GS-15; G_sulfurreducens-Geobacter sulfurreducens PCA; G_uraniumreducens-Geobacter uraniumreducens Rf4; Gloeobacter-Gloeobacter violaceus PCC 7421; L_interrogans1-Leptospira interrogans serovar Copenhageni str. Fiocruz L1-130; L_interrogans2-Leptospira interrogans serovar Lai str. 56601; Myxococcus1-Myxococcus xanthus DK 1622 YP_632048.1; Myxococcus2-Myxococcus xanthus DK 1622 YP_634222.1; Nitrobacter-Nitrobacter hamburgensis X14; Nitrosococcus-Nitrosococcus oceani ATCC 19707; P_carbinolicus-Pelobacter carbinolicus DSM 2380; Ralstonia-Ralstonia eutropha JMP134; Rhodopirellula-Rhodopirellula baltica SH 1; Rhodothermus-Rhodothermus marinus; Salinibacter-Salinibacter ruber DSM 13855; Solibacter-Solibacter usitatus Ellin6076; Synechococcus-Synechococcus sp. PCC 7002; Synechococcus_elongatus-Synechococcus elongatus PCC 7942; T_thermophilus1-Thermus thermophilus HB27; T_thermophilus2-Thermus thermophilus HB8</em></p>
<span>doi:10.1371/journal.pone.0001329.g004</span></div><a id="article1.body1.sec2.sec2.p7" name="article1.body1.sec2.sec2.p7"></a><p>In summary, the <em>M. xanthus</em> electron transport chain is a chimeric pathway containing components from diverse phylogenetic sources. <em>M. xanthus</em> has lost ancestral cytochrome oxidase genes common to other δ Proteobacteria and acquired other cytochrome oxidase genes. Acquisition of novel cytochrome oxidase and succinate dehydrogenase genes is coincident with the appearance of aerobic growth. Curiously, the only other aerobic organism in the δ Proteobacteria, <em>Bdellovibrio</em>, has notable differences from <em>Myxococcus</em> suggesting that it acquired aerobic metabolism using different gene sources.</p>

</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p>The genome of virtually every free-living bacterial species contains a contingent of genes acquired by LGT <a href="#pone.0001329-Ochman1">[4]</a>. Genes acquired by LGT can provide selective advantages with regard to antibiotic resistance, carbon utilization, and habitat range <a href="#pone.0001329-Ochman1">[4]</a>, <a href="#pone.0001329-delaCruz1">[6]</a>, <a href="#pone.0001329-Boucher1">[7]</a>. Indeed, most recent changes to the metabolic network of <em>E. coli</em> are due to LGT rather than gene duplication <a href="#pone.0001329-Pal1">[25]</a>. We estimate that roughly a quarter of the <em>Myxococcus</em> genome has been acquired by LGT based on phylogeny and/or codon usage. Some alien genes came from pathways that are found in diverse phyla, for example succinate dehydrogenase and cytochrome oxidase, and their import would be expected to provide a strong selective advantage by allowing aerobic growth. Other alien genes were fashioned into a unique developmental cycle and presumably played a different role in their previous host.</p>

<h4>Developmental innovation is correlated with a habitat shift</h4>
<a id="article1.body1.sec3.sec1.p1" name="article1.body1.sec3.sec1.p1"></a><p>What event(s) facilitated the innovation of this unique developmental cycle and the divergent evolution of Myxococcales from other δ Proteobacteria? We propose that the ancient myxobacterium gained access to a novel gene pool that was not available to other δ Proteobacteria. A phylogenetic analysis of &gt;220,000 proteins from the genomes of 144 prokaryotes identified specific ‘highways’ of lateral inheritance, some among distantly related organisms that live in similar environments <a href="#pone.0001329-Beiko1">[26]</a>. The succinate dehydrogenase and cytochrome oxidase genes were clearly acquired from aerobic organisms since they mediate aerobic respiration. Over 500 <em>M. xanthus</em> proteins have as their closest relatives proteins from Actinobacteria and Cyanobacteria (<a href="#pone-0001329-g001">figure 1</a>). This number exceeds by 10-fold the number of Actinobacteria and Cyanobacteria homologs found in <em>Bdellovibrio</em>. Most soil Actinobacteria, like <em>Streptomyces</em>, are aerobes and Cyanobacteria make their habitat aerobic by performing oxygenic photosynthesis. These factors argue that a shift to an aerobic habitat has given the Myxococcales access to a novel gene pool via LGT. Whether this habitat shift predated the emergence of fruiting body development is not clear but this issue could potentially be resolved when the genome sequences of more δ Proteobacteria are completed.</p>


<h4><em>Myxococcus</em> electron transport is a chimeric pathway</h4>
<a id="article1.body1.sec3.sec2.p1" name="article1.body1.sec3.sec2.p1"></a><p>Alien genes are generally integrated at the periphery of metabolic networks <a href="#pone.0001329-Pal1">[25]</a> where they could provide an immediate selective advantage. Multiple terminal electron acceptors provide flexibility in the face of ever changing environmental stresses. Of the four new <em>M. xanthus</em> terminal oxidase complexes, one involved the addition of a new oxidase (<em>cco</em>; <em>cbb3</em>-type cytochrome oxidase), one involved replacement of an ancient oxidase (<em>cyd</em>; cytochrome <em>bd</em> quinol oxidase), and one involved replacement of an ancient oxidase followed by duplication of the imported operon (<em>cox</em>; heme-copper cytochrome <em>c</em> oxidase). O<sub>2</sub> has one of the most positive reduction potentials of any terminal electron acceptor (O<sub>2</sub>/H<sub>2</sub>O = 0.82 V) so these cytochrome oxidases are located at the periphery of the electron transport chain. Succinate dehydrogenase (complex II) is located at the other end of the electron transport chain where it, along with NADH dehydrogenase (complex I), provide reducing equivalents. Succinate dehydrogenase catalyzes the interconversion of succinate and fumarate with the reduction of FAD allowing an organism to harvest the full potential of the tricarboxylic acid (TCA) cycle. Succinate dehydrogenase transfers reducing equivalents derived from the TCA cycle to the electron transport chain. In organisms with mixed respiration strategies like <em>E. coli</em> succinate dehydrogenase is produced only during aerobic growth <a href="#pone.0001329-Cecchini1">[27]</a>. The phylogenetic sources for the <em>M. xanthus</em> dehydrogenase and oxidases are different suggesting a piecemeal integration into the electron transport pathway.</p>


<h4>Predation as a means of acquiring genes</h4>
<a id="article1.body1.sec3.sec3.p1" name="article1.body1.sec3.sec3.p1"></a><p>In a microcosm containing <sup>13</sup>C labeled <em>E. coli</em> added to agricultural soil, the heavy carbon isotope was enriched in wild myxobacteria indicating that myxobacteria are predators <a href="#pone.0001329-Lueders1">[28]</a>. Does predation enhance the acquisition of new genes by the predator? The LGT observed with <em>Bdellovibrio</em> has been proposed to arise, at least in part, through the predatory capabilities of the organism <a href="#pone.0001329-Gophna1">[13]</a>. Indeed, the <em>Bdellovibrio</em> genome is enriched in genes whose closest homolog is in the γ Proteobacteria, the group containing the principal prey species <a href="#pone.0001329-Gophna1">[13]</a>. Like <em>Bdellovibrio,</em> the <em>M. xanthus</em> genome is enriched in genes whose closest homolog is in the γ Proteobacteria, the group containing the commonly used prey species <em>E. coli</em> <a href="#pone.0001329-Lueders1">[28]</a>. Myxobacteria have a much broader prey range than <em>Bdellovibrio</em> and significantly, a broader distribution of closest gene homologs (<a href="#pone-0001329-g001">figure 1</a>). Some notable differences from <em>Bdellovibrio</em> include genes from Actinobacteria, Acidobacteria, and Cyanobacteria (i.e. &gt;8-fold increase), which tend to be residents of same type of soil habitats <a href="#pone.0001329-vonMering1">[29]</a>. Acidobacteria are thought, on the basis of 16S gene abundance, to be one of the most prominent members of soil, though few species have been cultivated <a href="#pone.0001329-Quaiser1">[30]</a>. It has been recently proposed that Acidobacteria form a sister clade with the δ Proteobacteria <a href="#pone.0001329-Ciccarelli1">[31]</a>. While vertical inheritance from a common ancestor could result in the high representation of Acidobacteria genes, this hypothesis does not explain why <em>Bdellovibrio</em> has so few Acidobacteria genes (<a href="#pone-0001329-g001">figure 1</a>). Organisms in the other highly represented bacterial groups are known to be excellent food sources for <em>M. xanthus</em> in the laboratory <a href="#pone.0001329-Shimkets1">[32]</a>. <em>M. xanthus</em> is known to acquire genes from <em>E. coli</em> by conjugation <a href="#pone.0001329-Breton1">[33]</a> and transduction <a href="#pone.0001329-Shimkets2">[34]</a> under laboratory conditions but gene transfer in the natural environment has not been experimentally demonstrated.</p>


<h4>Conclusions and prospectus</h4>
<a id="article1.body1.sec3.sec4.p1" name="article1.body1.sec3.sec4.p1"></a><p>Our results argue that genes acquired from community members influence bacterial evolution. Successful community members may foster the evolution of successful communities by sharing genetic and phenotypic innovations that promote fitness. Cooperative evolution can have strong selective advantages in nature as demonstrated by the widespread emergence of antibiotic resistance or LGT of terminal oxidases in electron transport chains. It follows then that biologically diverse habitats aid the evolution of new bacterial species by providing a larger pool of prospective genes. While studies of the biogeography of microorganisms is in its infancy, bacterial communities can be endemic to certain areas or associated with unique species of higher organisms <a href="#pone.0001329-Bull1">[35]</a>. Habitat destruction and loss of species diversity could restrict the evolution of new bacterial groups by limiting combinations of LGT.</p>

</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4>Isolation of fruiting body deficient mutants</h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p><em>Myxococcus xanthus</em> LS2208 (Δ<em>fibA</em>) cells were grown to a density of 5×10<sup>8</sup> cells/ml in CYE broth [10 g/L Difco Casitone, 5 g/L yeast extract, 10 mM 3-(<em>N</em>-morpholino) propanesulfonic acid (MOPS; pH 7.6), and 4 mM MgSO<sub>4</sub>]. A 3 ml aliquot was harvested by centrifugation, washed twice with 1 ml of sterile distilled water, and resuspended in 30 µl sterile distilled water. Washed cells were mixed with plasmid pMycoMar <a href="#pone.0001329-Rubin1">[14]</a>, and electroporated at 0.65 kV, 400 Ω and 25 µF <a href="#pone.0001329-Kashefi1">[36]</a>. The electroporation cuvette was flushed immediately with 1 ml of CYE broth to recover cells, which were then incubated with shaking at 32°C for 4 h before plating on CF agar [10 mM Tris HCl, pH 7.6, 1 mM KH<sub>2</sub>PO<sub>4</sub>, pH 7.6, 8 mM MgSO<sub>4</sub>, 0.2 mg/ml, (NH<sub>4</sub>)<sub>2</sub>SO<sub>4</sub>, 150 µg/ml casitone, 1 mg/ml Na-pyruvate, 2 mg/ml Na-citrate, 1.5% Difco agar] supplemented with kanamycin (50 µg/ml) (CF Km) and incubated at 32°C. After 9–10 days, plates were screened under the microscope to identify colonies that were defective in making fruiting bodies. Potential mutants were screened again on CF Km plates and once on TPM agar [10 mM Tris HCl, pH 7.6, 1 mM KH<sub>2</sub>PO<sub>4</sub>, pH 7.6, 10 mM MgSO<sub>4</sub>, 1.5% Difco agar]. Approximately 40,000 colonies were screened. Strains containing a <em>magellen</em>-4 insertion were backcrossed to <em>M. xanthus</em> DK1622 (wild type) by electroporation of 1 µg genomic DNA <a href="#pone.0001329-Vlamakis1">[37]</a> or generalized transduction with phage Mx4 <a href="#pone.0001329-Campos1">[38]</a>.</p>


<h4>Cloning of <em>M. xanthus</em> genomic DNA flanking <em>magellan</em>-4 insertions</h4>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1"></a><p>To clone <em>magellan-4</em> insertions, genomic DNA was isolated from vegetative cultures grown in CYE medium containing 50 µg/ml kanamycin. A 1 ml cell culture was harvested by centrifugation and resuspended in 0.2 ml of 1X PBS buffer [8 g NaCl, 0.2 g KCl, 1.44 g Na<sub>2</sub>PO<sub>4</sub> and 0.24 g KH<sub>2</sub>PO<sub>4</sub> in 1000 ml distilled H<sub>2</sub>O, pH 7.4]. Genomic DNA was isolated by using Invitrogen Easy-DNA kit. Genomic DNA (0.5 µg) was digested with BssHII (New England Biolabs) in a total volume of 20 µl and digestions were dialyzed on a 0.025 µm pore size filter (Millipore) against distilled water for 30 min (drop dialysis). 8 µl of this DNA was treated with T4 DNA ligase (Promega) and drop-dialyzed before electroporation into <em>E. coli</em> host CC118 <a href="#pone.0001329-Youderian1">[15]</a>. Electroporants were recovered on LB agar containing kanamycin (50 µg/ml) after incubation at 37°C for 24 h. Plasmid DNA was sequenced with primers Mar1 or Mar2 <a href="#pone.0001329-Youderian1">[15]</a>.</p>


<h4>Fruiting body formation and sporulation assay</h4>
<a id="article1.body1.sec4.sec3.p1" name="article1.body1.sec4.sec3.p1"></a><p><em>Myxococcus xanthus</em> cells were grown in CYE broth to about 5×10<sup>8</sup> cells/ml. Cells were resuspended in TPM buffer [10 mM Tris HCl, pH 7.6, 1 mM KH(H<sub>2</sub>)PO<sub>4</sub>, pH 7.6, 10 mM MgSO<sub>4</sub>] to a final density of 1×10<sup>10</sup> cells/ml. 10 µl of each suspension was spotted onto TPM agar plates and incubated at 32°C. Digital images of fruiting body formation were taken every 24 h for a total of 72 h.</p>
<a id="article1.body1.sec4.sec3.p2" name="article1.body1.sec4.sec3.p2"></a><p>Cells were resuspended in 1 ml of TPM buffer, heated at 50°C for 2 hours, and sonicated to kill vegetative cells. Spore production was determined by direct counts using a Petroff-Hauser chamber. Spore viability was determined by plating serial dilutions on CYE agar plates. Plates were incubated at 32°C for 5 days before counting colonies.</p>


<h4>Genomic analyses</h4>
<a id="article1.body1.sec4.sec4.p1" name="article1.body1.sec4.sec4.p1"></a><p>Phylogenetic and comparative genomic analyses were conducted with data from &gt;300 genomes. Predicted peptides from each of eight completed δ Proteobacteria genomes were obtained from NCBI in March and April 2006 including <em>Anaeromyxobacter dehalogenans, Bdellovibrio bacteriovorus, Desulfotalea psychrophila, Desulfovibrio desulfuricans, Geobacter sulfurreducens, Lawsonia intracellularis, Myxococcus xanthus, and Pelobacter carbinolicus.</em> Comparison of peptides was determined using the BLASTP algorithm without filtering against the non-redundant peptide database at NCBI during this time period <a href="#pone.0001329-Altschul1">[39]</a>. The top ten hits for each predicted peptide were identified with an E value&lt;10<sup>−10</sup>. The species of each top hit and the corresponding bacterial group were identified from the BLAST algorithm output. The identity of each bacterial group used the taxonomic data provided by NCBI. Alignments of predicted peptides were generated using the Muscle algorithm <a href="#pone.0001329-Edgar1">[40]</a>. Neighbor-Joining trees and tree visualization were performed using MEGA version 3.1 <a href="#pone.0001329-Kumar1">[41]</a>. Tree branches with a bootstrap value&lt;50% were condensed.</p>

</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>The authors are indebted to Lindsey Thurman for able technical assistance and Jan Mrazek for use of his program to identify genes with altered codon usage. Patrick Curtis, Aurelio Moraleda Munoz, Jan Mrazek, Steve Slater, and Roy Welch provided a critical review of the manuscript.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: LS. Performed the experiments: SB. Analyzed the data: BG LS SB. Contributed reagents/materials/analysis tools: LS. Wrote the paper: LS. Other: Performed bioinformatics: BG. Generated several figures: BG.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0001329-Taylor1" id="pone.0001329-Taylor1"></a>Taylor JS, Raes J (2004) Duplication and divergence: the evolution of new genes and old ideas. Annu Rev Genet  38: 615–643.  <ul class="find" data-citedArticleID="1031636" data-doi="10.1146/annurev.genet.38.072902.092831"><li><a href="http://dx.doi.org/10.1146/annurev.genet.38.072902.092831" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Duplication+and+divergence%3A+the+evolution+of+new+genes+and+old+ideas." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Duplication+and+divergence%3A+the+evolution+of+new+genes+and+old+ideas.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0001329-Gevers1" id="pone.0001329-Gevers1"></a>Gevers D, Vandepoele K, Simillon C, Van de Peer Y (2004) Gene duplication and biased functional retention of paralogs in bacterial genomes. Trends Microbiol  12: 148–154.  <ul class="find" data-citedArticleID="1031598" data-doi="10.1016/j.tim.2004.02.007"><li><a href="http://dx.doi.org/10.1016/j.tim.2004.02.007" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Gene+duplication+and+biased+functional+retention+of+paralogs+in+bacterial+genomes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Gene+duplication+and+biased+functional+retention+of+paralogs+in+bacterial+genomes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0001329-Chen1" id="pone.0001329-Chen1"></a>Chen I, Christie PJ, Dubnau D (2005) The ins and outs of DNA transfer in bacteria. Science  310: 1456–1460.  <ul class="find" data-citedArticleID="1031586" data-doi="10.1126/science.1114021"><li><a href="http://dx.doi.org/10.1126/science.1114021" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+ins+and+outs+of+DNA+transfer+in+bacteria." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+ins+and+outs+of+DNA+transfer+in+bacteria.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0001329-Ochman1" id="pone.0001329-Ochman1"></a>Ochman H, Lawrence JG, Groisman EA (2000) Lateral gene transfer and the nature of bacterial innovation. Nature  405: 299–304.  <ul class="find" data-citedArticleID="1031624" data-doi="10.1038/35012500"><li><a href="http://dx.doi.org/10.1038/35012500" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Lateral+gene+transfer+and+the+nature+of+bacterial+innovation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Lateral+gene+transfer+and+the+nature+of+bacterial+innovation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0001329-Lerat1" id="pone.0001329-Lerat1"></a>Lerat E, Daubin V, Ochman H, Moran NA (2005) Evolutionary origins of genomic repertoires in bacteria. PLoS Biol  3: e130.  <ul class="find" data-citedArticleID="1031616" data-doi="10.1371/journal.pbio.0030130"><li><a href="http://dx.doi.org/10.1371/journal.pbio.0030130" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evolutionary+origins+of+genomic+repertoires+in+bacteria." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evolutionary+origins+of+genomic+repertoires+in+bacteria.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0001329-delaCruz1" id="pone.0001329-delaCruz1"></a>de la Cruz F, Davies J (2000) Horizontal gene transfer and the origin of species: lessons from bacteria. Trends Microbiol  8: 128–133.  <ul class="find" data-citedArticleID="1031650" data-doi="10.1016/s0966-842x(00)01703-0"><li><a href="http://dx.doi.org/10.1016/s0966-842x(00)01703-0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Horizontal+gene+transfer+and+the+origin+of+species%3A+lessons+from+bacteria." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Horizontal+gene+transfer+and+the+origin+of+species%3A+lessons+from+bacteria.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0001329-Boucher1" id="pone.0001329-Boucher1"></a>Boucher Y, Douady CJ, Papke RT, Walsh DA, Boudreau ME, et al.  (2003) Lateral gene transfer and the origins of prokaryotic groups. Annu Rev Genet  37: 283–328.  <ul class="find" data-citedArticleID="1031576" data-doi="10.1146/annurev.genet.37.050503.084247"><li><a href="http://dx.doi.org/10.1146/annurev.genet.37.050503.084247" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Lateral+gene+transfer+and+the+origins+of+prokaryotic+groups." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Lateral+gene+transfer+and+the+origins+of+prokaryotic+groups.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0001329-Goldman1" id="pone.0001329-Goldman1"></a>Goldman BS, Nierman WC, Kaiser D, Slater SC, Durkin AS, et al.  (2006) Evolution of sensory complexity recorded in a myxobacterial genome. Proc Natl Acad Sci U S A  103: 15200–15205.  <ul class="find" data-citedArticleID="1031600" data-doi="10.1073/pnas.0607335103"><li><a href="http://dx.doi.org/10.1073/pnas.0607335103" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evolution+of+sensory+complexity+recorded+in+a+myxobacterial+genome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evolution+of+sensory+complexity+recorded+in+a+myxobacterial+genome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0001329-Lawrence1" id="pone.0001329-Lawrence1"></a>Lawrence JG, Ochman H (1997) Amelioration of bacterial genomes: rates of change and exchange. J Mol Evol  44: 383–397.  <ul class="find" data-citedArticleID="1031614" data-doi="10.1007/pl00006158"><li><a href="http://dx.doi.org/10.1007/pl00006158" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Amelioration+of+bacterial+genomes%3A+rates+of+change+and+exchange." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Amelioration+of+bacterial+genomes%3A+rates+of+change+and+exchange.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0001329-MedranoSoto1" id="pone.0001329-MedranoSoto1"></a>Medrano-Soto A, Moreno-Hagelsieb G, Vinuesa P, Christen JA, Collado-Vides J (2004) Successful lateral transfer requires codon usage compatibility between foreign genes and recipient genomes. Mol Biol Evol  21: 1884–1894.  <ul class="find" data-citedArticleID="1031620" data-doi="10.1093/molbev/msh202"><li><a href="http://dx.doi.org/10.1093/molbev/msh202" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Successful+lateral+transfer+requires+codon+usage+compatibility+between+foreign+genes+and+recipient+genomes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Successful+lateral+transfer+requires+codon+usage+compatibility+between+foreign+genes+and+recipient+genomes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0001329-Karlin1" id="pone.0001329-Karlin1"></a>Karlin S, Mrazek J (2000) Predicted highly expressed genes of diverse prokaryotic genomes. J Bacteriol  182: 5238–5250.  <ul class="find" data-citedArticleID="1031606" data-doi="10.1128/jb.182.18.5238-5250.2000"><li><a href="http://dx.doi.org/10.1128/jb.182.18.5238-5250.2000" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Predicted+highly+expressed+genes+of+diverse+prokaryotic+genomes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Predicted+highly+expressed+genes+of+diverse+prokaryotic+genomes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0001329-Mrazek1" id="pone.0001329-Mrazek1"></a>Mrazek J, Karlin S (1999) Detecting alien genes in bacterial genomes. Ann N Y Acad Sci  870: 314–329.  <ul class="find" data-citedArticleID="1031622" data-doi="10.1111/j.1749-6632.1999.tb08893.x"><li><a href="http://dx.doi.org/10.1111/j.1749-6632.1999.tb08893.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Detecting+alien+genes+in+bacterial+genomes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Detecting+alien+genes+in+bacterial+genomes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0001329-Gophna1" id="pone.0001329-Gophna1"></a>Gophna U, Charlebois RL, Doolittle WF (2006) Ancient lateral gene transfer in the evolution of <em>Bdellovibrio bacteriovorus</em>. Trends Microbiol  14: 64–69.  <ul class="find" data-citedArticleID="1031602" data-doi="10.1016/j.tim.2005.12.008"><li><a href="http://dx.doi.org/10.1016/j.tim.2005.12.008" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Ancient+lateral+gene+transfer+in+the+evolution+of+Bdellovibrio+bacteriovorus." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Ancient+lateral+gene+transfer+in+the+evolution+of+Bdellovibrio+bacteriovorus.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0001329-Rubin1" id="pone.0001329-Rubin1"></a>Rubin EJ, Akerley BJ, Novik VN, Lampe DJ, Husson RN, et al.  (1999) In vivo transposition of mariner-based elements in enteric bacteria and mycobacteria. Proc Natl Acad Sci U S A  96: 1645–1650.  <ul class="find" data-citedArticleID="1031630" data-doi="10.1073/pnas.96.4.1645"><li><a href="http://dx.doi.org/10.1073/pnas.96.4.1645" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=In+vivo+transposition+of+mariner-based+elements+in+enteric+bacteria+and+mycobacteria." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22In+vivo+transposition+of+mariner-based+elements+in+enteric+bacteria+and+mycobacteria.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0001329-Youderian1" id="pone.0001329-Youderian1"></a>Youderian P, Burke N, White DJ, Hartzell PL (2003) Identification of genes required for adventurous gliding motility in <em>Myxococcus xanthus</em> with the transposable element mariner. Mol Microbiol  49: 555–570.  <ul class="find" data-citedArticleID="1031644" data-doi="10.1046/j.1365-2958.2003.03582.x"><li><a href="http://dx.doi.org/10.1046/j.1365-2958.2003.03582.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Identification+of+genes+required+for+adventurous+gliding+motility+in+Myxococcus+xanthus+with+the+transposable+element+mariner." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Identification+of+genes+required+for+adventurous+gliding+motility+in+Myxococcus+xanthus+with+the+transposable+element+mariner.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0001329-Youderian2" id="pone.0001329-Youderian2"></a>Youderian P, Hartzell PL (2006) Transposon insertions of <em>magellan-4</em> that impair social gliding motility in <em>Myxococcus xanthus</em>. Genetics  172: 1397–1410.  <ul class="find" data-citedArticleID="1031646" data-doi="10.1534/genetics.105.050542"><li><a href="http://dx.doi.org/10.1534/genetics.105.050542" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Transposon+insertions+of+magellan-4+that+impair+social+gliding+motility+in+Myxococcus+xanthus." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Transposon+insertions+of+magellan-4+that+impair+social+gliding+motility+in+Myxococcus+xanthus.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0001329-Yu1" id="pone.0001329-Yu1"></a>Yu R, Kaiser D (2007) Gliding motility and polarized slime secretion. Mol Microbiol  63: 454–467.  <ul class="find" data-citedArticleID="1031648" data-doi="10.1111/j.1365-2958.2006.05536.x"><li><a href="http://dx.doi.org/10.1111/j.1365-2958.2006.05536.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Gliding+motility+and+polarized+slime+secretion." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Gliding+motility+and+polarized+slime+secretion.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0001329-Kalman1" id="pone.0001329-Kalman1"></a>Kalman LV, Cheng YL, Kaiser D (1994) The <em>Myxococcus xanthus dsg</em> gene product performs functions of translation initiation factor IF3 in vivo. J Bacteriol  176: 1434–1442.  <ul class="find" data-citedArticleID="1031604"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=The+Myxococcus+xanthus+dsg+gene+product+performs+functions+of+translation+initiation+factor+IF3+in+vivo.&amp;auth=&amp;atitle=The+Myxococcus+xanthus+dsg+gene+product+performs+functions+of+translation+initiation+factor+IF3+in+vivo." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+Myxococcus+xanthus+dsg+gene+product+performs+functions+of+translation+initiation+factor+IF3+in+vivo." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+Myxococcus+xanthus+dsg+gene+product+performs+functions+of+translation+initiation+factor+IF3+in+vivo.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0001329-Cheng1" id="pone.0001329-Cheng1"></a>Cheng YL, Kalman LV, Kaiser D (1994) The <em>dsg</em> gene of <em>Myxococcus xanthus</em> encodes a protein similar to translation initiation factor IF3. J Bacteriol  176: 1427–1433.  <ul class="find" data-citedArticleID="1031588"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=The+dsg+gene+of+Myxococcus+xanthus+encodes+a+protein+similar+to+translation+initiation+factor+IF3.&amp;auth=&amp;atitle=The+dsg+gene+of+Myxococcus+xanthus+encodes+a+protein+similar+to+translation+initiation+factor+IF3." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+dsg+gene+of+Myxococcus+xanthus+encodes+a+protein+similar+to+translation+initiation+factor+IF3." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+dsg+gene+of+Myxococcus+xanthus+encodes+a+protein+similar+to+translation+initiation+factor+IF3.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0001329-Cheng2" id="pone.0001329-Cheng2"></a>Cheng Y, Kaiser D (1989) dsg, a gene required for <em>Myxococcus</em> development, is necessary for cell viability. J Bacteriol  171: 3727–3731.  <ul class="find" data-citedArticleID="1031590"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=dsg%2C+a+gene+required+for+Myxococcus+development%2C+is+necessary+for+cell+viability.&amp;auth=&amp;atitle=dsg%2C+a+gene+required+for+Myxococcus+development%2C+is+necessary+for+cell+viability." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=dsg%2C+a+gene+required+for+Myxococcus+development%2C+is+necessary+for+cell+viability." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22dsg%2C+a+gene+required+for+Myxococcus+development%2C+is+necessary+for+cell+viability.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0001329-Yamanaka1" id="pone.0001329-Yamanaka1"></a>Yamanaka S, Kawaguchi A, Komagata K (1987) Isolation and identification of myxobacteria from soils and plant materials, with special reference to DNA base composition, quinone system, and cellular fatty acid composition, and with a description of a new species, <em>Myxococcus flavescens</em>. J Gen Appl Microbiol  33: 247–265.  <ul class="find" data-citedArticleID="1031640" data-doi="10.2323/jgam.33.247"><li><a href="http://dx.doi.org/10.2323/jgam.33.247" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Isolation+and+identification+of+myxobacteria+from+soils+and+plant+materials%2C+with+special+reference+to+DNA+base+composition%2C+quinone+system%2C+and+cellular+fatty+acid+composition%2C+and+with+a+description+of+a+new+species%2C+Myxococcus+flavescens." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Isolation+and+identification+of+myxobacteria+from+soils+and+plant+materials%2C+with+special+reference+to+DNA+base+composition%2C+quinone+system%2C+and+cellular+fatty+acid+composition%2C+and+with+a+description+of+a+new+species%2C+Myxococcus+flavescens.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0001329-Kleinig1" id="pone.0001329-Kleinig1"></a>Kleinig H, Reichenbach H, Theobald N, Achenbach H (1974) <em>Flexibacter elegans</em> and <em>Myxococcus fulvus</em>: aerobic Gram-negative bacteria containing menaquinones as the only isoprenoid quinones. Arch Microbiol  101: 91–93.  <ul class="find" data-citedArticleID="1031610" data-doi="10.1007/bf00455928"><li><a href="http://dx.doi.org/10.1007/bf00455928" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Flexibacter+elegans+and+Myxococcus+fulvus%3A+aerobic+Gram-negative+bacteria+containing+menaquinones+as+the+only+isoprenoid+quinones." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Flexibacter+elegans+and+Myxococcus+fulvus%3A+aerobic+Gram-negative+bacteria+containing+menaquinones+as+the+only+isoprenoid+quinones.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0001329-Yanyushin1" id="pone.0001329-Yanyushin1"></a>Yanyushin MF, del Rosario MC, Brune DC, Blankenship RE (2005) New class of bacterial membrane oxidoreductases. Biochemistry  44: 10037–10045.  <ul class="find" data-citedArticleID="1031642" data-doi="10.1021/bi047267l"><li><a href="http://dx.doi.org/10.1021/bi047267l" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=New+class+of+bacterial+membrane+oxidoreductases." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22New+class+of+bacterial+membrane+oxidoreductases.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0001329-Dolla1" id="pone.0001329-Dolla1"></a>Dolla A, Fournier M, Dermoun Z (2006) Oxygen defense in sulfate-reducing bacteria. J Biotechnol  126: 87–100.  <ul class="find" data-citedArticleID="1031594" data-doi="10.1016/j.jbiotec.2006.03.041"><li><a href="http://dx.doi.org/10.1016/j.jbiotec.2006.03.041" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Oxygen+defense+in+sulfate-reducing+bacteria." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Oxygen+defense+in+sulfate-reducing+bacteria.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0001329-Pal1" id="pone.0001329-Pal1"></a>Pal C, Papp B, Lercher MJ (2005) Adaptive evolution of bacterial metabolic networks by horizontal gene transfer. Nat Genet  37: 1372–1375.  <ul class="find" data-citedArticleID="1031626" data-doi="10.1038/ng1686"><li><a href="http://dx.doi.org/10.1038/ng1686" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Adaptive+evolution+of+bacterial+metabolic+networks+by+horizontal+gene+transfer." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Adaptive+evolution+of+bacterial+metabolic+networks+by+horizontal+gene+transfer.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0001329-Beiko1" id="pone.0001329-Beiko1"></a>Beiko RG, Harlow TJ, Ragan MA (2005) Highways of gene sharing in prokaryotes. Proc Natl Acad Sci U S A  102: 14332–14337.  <ul class="find" data-citedArticleID="1031574" data-doi="10.1073/pnas.0504068102"><li><a href="http://dx.doi.org/10.1073/pnas.0504068102" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Highways+of+gene+sharing+in+prokaryotes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Highways+of+gene+sharing+in+prokaryotes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0001329-Cecchini1" id="pone.0001329-Cecchini1"></a>Cecchini G, Schroder I, Gunsalus RP, Maklashina E (2002) Succinate dehydrogenase and fumarate reductase from Escherichia coli. Biochim Biophys Acta  1553: 140–157.  <ul class="find" data-citedArticleID="1031584" data-doi="10.1016/s0005-2728(01)00238-9"><li><a href="http://dx.doi.org/10.1016/s0005-2728(01)00238-9" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Succinate+dehydrogenase+and+fumarate+reductase+from+Escherichia+coli." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Succinate+dehydrogenase+and+fumarate+reductase+from+Escherichia+coli.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0001329-Lueders1" id="pone.0001329-Lueders1"></a>Lueders T, Kindler R, Miltner A, Friedrich MW, Kaestner M (2006) Identification of bacterial micropredators distinctively active in a soil microbial food web. Appl Environ Microbiol  72: 5342–5348.  <ul class="find" data-citedArticleID="1031618" data-doi="10.1128/aem.00400-06"><li><a href="http://dx.doi.org/10.1128/aem.00400-06" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Identification+of+bacterial+micropredators+distinctively+active+in+a+soil+microbial+food+web." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Identification+of+bacterial+micropredators+distinctively+active+in+a+soil+microbial+food+web.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0001329-vonMering1" id="pone.0001329-vonMering1"></a>von Mering C, Hugenholtz P, Raes J, Tringe SG, Doerks T, et al.  (2007) Quantitative phylogenetic assessment of microbial communities in diverse environments. Science  315: 1126–1130.  <ul class="find" data-citedArticleID="1031652" data-doi="10.1126/science.1133420"><li><a href="http://dx.doi.org/10.1126/science.1133420" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Quantitative+phylogenetic+assessment+of+microbial+communities+in+diverse+environments." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Quantitative+phylogenetic+assessment+of+microbial+communities+in+diverse+environments.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0001329-Quaiser1" id="pone.0001329-Quaiser1"></a>Quaiser A, Ochsenreiter T, Lanz C, Schuster SC, Treusch AH, et al.  (2003) Acidobacteria form a coherent but highly diverse group within the bacterial domain: evidence from environmental genomics. Mol Microbiol  50: 563–575.  <ul class="find" data-citedArticleID="1031628" data-doi="10.1046/j.1365-2958.2003.03707.x"><li><a href="http://dx.doi.org/10.1046/j.1365-2958.2003.03707.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Acidobacteria+form+a+coherent+but+highly+diverse+group+within+the+bacterial+domain%3A+evidence+from+environmental+genomics." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Acidobacteria+form+a+coherent+but+highly+diverse+group+within+the+bacterial+domain%3A+evidence+from+environmental+genomics.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0001329-Ciccarelli1" id="pone.0001329-Ciccarelli1"></a>Ciccarelli FD, Doerks T, von Mering C, Creevey CJ, Snel B, et al.  (2006) Toward automatic reconstruction of a highly resolved tree of life. Science  311: 1283–1287.  <ul class="find" data-citedArticleID="1031592" data-doi="10.1126/science.1123061"><li><a href="http://dx.doi.org/10.1126/science.1123061" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Toward+automatic+reconstruction+of+a+highly+resolved+tree+of+life." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Toward+automatic+reconstruction+of+a+highly+resolved+tree+of+life.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0001329-Shimkets1" id="pone.0001329-Shimkets1"></a>Shimkets LJ, Dworkin M, Reichenbach H (2007)  The Myxobacteria. In: Dworkin SF M, Rosenberg E, Schleifer K.-H, Stackebrandt, E E, editors. The Prokaryotes · A Handbook on the Biology of Bacteria: Vol 7: Proteobacteria: Delta and Epsilon Subclasses Deeply Rooting Bacteria. Chapter 3.4.3. New York: Springer Verlag. pp. 31–115.  <ul class="find-nolinks"></ul></li><li><span class="label">33.
              </span><a name="pone.0001329-Breton1" id="pone.0001329-Breton1"></a>Breton AM, Jaoua S, Guespin-Michel J (1985) Transfer of plasmid RP4 to <em>Myxococcus xanthus</em> and evidence for its integration into the chromosome. J Bacteriol  161: 523–528.  <ul class="find" data-citedArticleID="1031578" data-doi="10.1016/0147-619x(87)90038-2"><li><a href="http://dx.doi.org/10.1016/0147-619x(87)90038-2" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Transfer+of+plasmid+RP4+to+Myxococcus+xanthus+and+evidence+for+its+integration+into+the+chromosome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Transfer+of+plasmid+RP4+to+Myxococcus+xanthus+and+evidence+for+its+integration+into+the+chromosome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0001329-Shimkets2" id="pone.0001329-Shimkets2"></a>Shimkets LJ, Gill RE, Kaiser D (1983) Developmental cell interactions in <em>Myxococcus xanthus</em> and the <em>spoC</em> locus. Proc Natl Acad Sci USA  80: 1406–1410.  <ul class="find" data-citedArticleID="1031634" data-doi="10.1073/pnas.80.5.1406"><li><a href="http://dx.doi.org/10.1073/pnas.80.5.1406" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Developmental+cell+interactions+in+Myxococcus+xanthus+and+the+spoC+locus." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Developmental+cell+interactions+in+Myxococcus+xanthus+and+the+spoC+locus.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0001329-Bull1" id="pone.0001329-Bull1"></a>Bull AT, Ward AC, Goodfellow M (2000) Search and discovery strategies for biotechnology: the paradigm shift. Microbiol Mol Biol Rev  64: 573–606.  <ul class="find" data-citedArticleID="1031580" data-doi="10.1128/mmbr.64.3.573-606.2000"><li><a href="http://dx.doi.org/10.1128/mmbr.64.3.573-606.2000" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Search+and+discovery+strategies+for+biotechnology%3A+the+paradigm+shift." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Search+and+discovery+strategies+for+biotechnology%3A+the+paradigm+shift.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">36.
              </span><a name="pone.0001329-Kashefi1" id="pone.0001329-Kashefi1"></a>Kashefi K, Hartzell PL (1995) Genetic suppression and phenotypic masking of a Myxococcus xanthus frzF- defect. Mol Microbiol  15: 483–494.  <ul class="find" data-citedArticleID="1031608" data-doi="10.1111/j.1365-2958.1995.tb02262.x"><li><a href="http://dx.doi.org/10.1111/j.1365-2958.1995.tb02262.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genetic+suppression+and+phenotypic+masking+of+a+Myxococcus+xanthus+frzF-+defect." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genetic+suppression+and+phenotypic+masking+of+a+Myxococcus+xanthus+frzF-+defect.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">37.
              </span><a name="pone.0001329-Vlamakis1" id="pone.0001329-Vlamakis1"></a>Vlamakis HC, Kirby JR, Zusman DR (2004) The Che4 pathway of <em>Myxococcus xanthus</em> regulates type IV pilus-mediated motility. Mol Microbiol  52: 1799–1811.  <ul class="find" data-citedArticleID="1031638" data-doi="10.1111/j.1365-2958.2004.04098.x"><li><a href="http://dx.doi.org/10.1111/j.1365-2958.2004.04098.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+Che4+pathway+of+Myxococcus+xanthus+regulates+type+IV+pilus-mediated+motility." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+Che4+pathway+of+Myxococcus+xanthus+regulates+type+IV+pilus-mediated+motility.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">38.
              </span><a name="pone.0001329-Campos1" id="pone.0001329-Campos1"></a>Campos JM, Geisselsoder J, Zusman DR (1978) Isolation of bacteriophage MX4, a generalized transducing phage for <em>Myxococcus xanthus</em>. J Mol Biol  119: 167–178.  <ul class="find" data-citedArticleID="1031582" data-doi="10.1016/0022-2836(78)90431-x"><li><a href="http://dx.doi.org/10.1016/0022-2836(78)90431-x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Isolation+of+bacteriophage+MX4%2C+a+generalized+transducing+phage+for+Myxococcus+xanthus." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Isolation+of+bacteriophage+MX4%2C+a+generalized+transducing+phage+for+Myxococcus+xanthus.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">39.
              </span><a name="pone.0001329-Altschul1" id="pone.0001329-Altschul1"></a>Altschul SF, Madden TL, Schaffer AA, Zhang J, Zhang Z, et al.  (1997) Gapped BLAST and PSI-BLAST: a new generation of protein database search programs. Nucleic Acids Res  25: 3389–3402.  <ul class="find" data-citedArticleID="1031572" data-doi="10.1093/nar/25.17.3389"><li><a href="http://dx.doi.org/10.1093/nar/25.17.3389" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Gapped+BLAST+and+PSI-BLAST%3A+a+new+generation+of+protein+database+search+programs." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Gapped+BLAST+and+PSI-BLAST%3A+a+new+generation+of+protein+database+search+programs.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">40.
              </span><a name="pone.0001329-Edgar1" id="pone.0001329-Edgar1"></a>Edgar RC (2004) MUSCLE: a multiple sequence alignment method with reduced time and space complexity. BMC Bioinformatics  5: 113.  <ul class="find" data-citedArticleID="1031596"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=MUSCLE%3A+a+multiple+sequence+alignment+method+with+reduced+time+and+space+complexity.&amp;auth=&amp;atitle=MUSCLE%3A+a+multiple+sequence+alignment+method+with+reduced+time+and+space+complexity." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=MUSCLE%3A+a+multiple+sequence+alignment+method+with+reduced+time+and+space+complexity." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22MUSCLE%3A+a+multiple+sequence+alignment+method+with+reduced+time+and+space+complexity.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">41.
              </span><a name="pone.0001329-Kumar1" id="pone.0001329-Kumar1"></a>Kumar S, Tamura K, Nei M (2004) MEGA3: Integrated software for Molecular Evolutionary Genetics Analysis and sequence alignment. Brief Bioinform  5: 150–163.  <ul class="find" data-citedArticleID="1031612" data-doi="10.1093/bib/5.2.150"><li><a href="http://dx.doi.org/10.1093/bib/5.2.150" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=MEGA3%3A+Integrated+software+for+Molecular+Evolutionary+Genetics+Analysis+and+sequence+alignment." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22MEGA3%3A+Integrated+software+for+Molecular+Evolutionary+Genetics+Analysis+and+sequence+alignment.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001329.XML" value="115905"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001329.PDF" value="346735"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001329.t001.PNG_L" value="803797"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001329.t001.PNG_M" value="119927"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001329.t001.PNG_S" value="21065"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001329.t001.TIF" value="3692076"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001329.t001.PNG_I" value="156370"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001329.g001.PNG_L" value="532066"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001329.g001.PNG_M" value="58934"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001329.g001.PNG_S" value="7132"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001329.g001.TIF" value="1077680"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001329.g001.PNG_I" value="21828"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001329.g002.PNG_L" value="1287301"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001329.g002.PNG_M" value="71544"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001329.g002.PNG_S" value="6558"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001329.g002.TIF" value="2846292"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001329.g002.PNG_I" value="27041"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001329.g003.PNG_L" value="767129"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001329.g003.PNG_M" value="193778"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001329.g003.PNG_S" value="14722"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001329.g003.TIF" value="1213030"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001329.g003.PNG_I" value="92552"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001329.t002.PNG_L" value="134323"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001329.t002.PNG_M" value="145373"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001329.t002.PNG_S" value="19850"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001329.t002.TIF" value="637204"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001329.t002.PNG_I" value="67635"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001329.g004.PNG_L" value="564855"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001329.g004.PNG_M" value="145128"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001329.g004.PNG_S" value="12159"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001329.g004.TIF" value="984548"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001329.g004.PNG_I" value="61049"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001329&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001329" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001329&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0001329&volume=&issue=&title=Genome Evolution and the Emergence of Fruiting Body Development in Myxococcus xanthus&author_name=Barry%20Goldman%2C%20Swapna%20Bhat%2C%20Lawrence%20J.%20Shimkets&start_page=1&end_page=9" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001329" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001329" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0001329" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001329&amp;t=Genome%20Evolution%20and%20the%20Emergence%20of%20Fruiting%20Body%20Development%20in%20Myxococcus%20xanthus" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001329&title=Genome%20Evolution%20and%20the%20Emergence%20of%20Fruiting%20Body%20Development%20in%20Myxococcus%20xanthus&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001329&amp;title=Genome%20Evolution%20and%20the%20Emergence%20of%20Fruiting%20Body%20Development%20in%20Myxococcus%20xanthus" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001329" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0001329&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Genome Evolution and the Emergence of Fruiting Body Development in Myxococcus xanthus';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0001329';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Genome%20Evolution%20and%20the%20Emergence%20of%20Fruiting%20Body%20Development%20in%20Myxococcus%20xanthus http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001329" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0001329" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">
















          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Bacterial+genomics%22" title="Search for articles in the subject area:'Bacterial genomics'"><div class="flagText">Bacterial genomics</div></a>
              <div data-categoryid="17227" data-articleid="25488"
                   data-categoryname="Bacterial genomics"
                   class="flagImage" title="Flag 'Bacterial genomics' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Bdellovibrio%22" title="Search for articles in the subject area:'Bdellovibrio'"><div class="flagText">Bdellovibrio</div></a>
              <div data-categoryid="44823" data-articleid="25488"
                   data-categoryname="Bdellovibrio"
                   class="flagImage" title="Flag 'Bdellovibrio' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Codons%22" title="Search for articles in the subject area:'Codons'"><div class="flagText">Codons</div></a>
              <div data-categoryid="18427" data-articleid="25488"
                   data-categoryname="Codons"
                   class="flagImage" title="Flag 'Codons' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Evolutionary+genetics%22" title="Search for articles in the subject area:'Evolutionary genetics'"><div class="flagText">Evolutionary genetics</div></a>
              <div data-categoryid="17209" data-articleid="25488"
                   data-categoryname="Evolutionary genetics"
                   class="flagImage" title="Flag 'Evolutionary genetics' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Gene+regulation%22" title="Search for articles in the subject area:'Gene regulation'"><div class="flagText">Gene regulation</div></a>
              <div data-categoryid="17553" data-articleid="25488"
                   data-categoryname="Gene regulation"
                   class="flagImage" title="Flag 'Gene regulation' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Genome+evolution%22" title="Search for articles in the subject area:'Genome evolution'"><div class="flagText">Genome evolution</div></a>
              <div data-categoryid="17207" data-articleid="25488"
                   data-categoryname="Genome evolution"
                   class="flagImage" title="Flag 'Genome evolution' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Phylogenetic+analysis%22" title="Search for articles in the subject area:'Phylogenetic analysis'"><div class="flagText">Phylogenetic analysis</div></a>
              <div data-categoryid="17851" data-articleid="25488"
                   data-categoryname="Phylogenetic analysis"
                   class="flagImage" title="Flag 'Phylogenetic analysis' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Phylogenetics%22" title="Search for articles in the subject area:'Phylogenetics'"><div class="flagText">Phylogenetics</div></a>
              <div data-categoryid="46093" data-articleid="25488"
                   data-categoryname="Phylogenetics"
                   class="flagImage" title="Flag 'Phylogenetics' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=3705'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=7687'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=7965&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>


</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
