

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Increased Nucleotide Diversity with Transient Y Linkage in Drosophila americana</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0000112"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0000112" />

    <meta name="citation_title" content="Increased Nucleotide Diversity with Transient Y Linkage in Drosophila americana"/>
    <meta itemprop="name" content="Increased Nucleotide Diversity with Transient Y Linkage in Drosophila americana"/>

      <meta name="citation_author" content="Bryant F. McAllister"/>
            <meta name="citation_author_institution" content="Department of Biological Sciences and Roy J. Carver Center for Comparative Genomics, University of Iowa, Iowa City, Iowa, United States of America"/>
      <meta name="citation_author" content="Amy L. Evans"/>
            <meta name="citation_author_institution" content="Department of Biological Sciences and Roy J. Carver Center for Comparative Genomics, University of Iowa, Iowa City, Iowa, United States of America"/>

    <meta name="citation_date" content="2006/12/27"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0000112.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e112"/>
    <meta name="citation_issue" content="1"/>
    <meta name="citation_volume" content="1"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Levels of naturally occurring DNA polymorphism correlate with recombination rates in D. melanogaster.; citation_author=DJ Begun; citation_author=CF Aquadro; citation_journal_title=Nature; citation_volume=356; citation_number=1; citation_pages=519-520; citation_date=1992; " />
      <meta name="citation_reference" content="citation_title=Single nucleotide polymorphism and recombination rate in humans.; citation_author=MW Nachman; citation_journal_title=Trends Genet; citation_volume=17; citation_number=2; citation_pages=481-485; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Selection at linked sites in the partial selfer Caenorhabditis elegans.; citation_author=AD Cutter; citation_author=BA Payseur; citation_journal_title=Mol Biol Evol; citation_volume=20; citation_number=3; citation_pages=665-673; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=The effect of linkage on limits to artificial selection.; citation_author=WG Hill; citation_author=A Robertson; citation_journal_title=Genet Res; citation_volume=8; citation_number=4; citation_pages=269-294; citation_date=1966; " />
      <meta name="citation_reference" content="citation_title=The evolutionary advantage of recombination.; citation_author=J Felsenstein; citation_journal_title=Genetics; citation_volume=78; citation_number=5; citation_pages=737-756; citation_date=1974; " />
      <meta name="citation_reference" content="citation_title=The hitchhiking effect of a favourable gene.; citation_author=J Maynard Smith; citation_author=J Haigh; citation_journal_title=Genet Res; citation_volume=23; citation_number=6; citation_pages=23-35; citation_date=1974; " />
      <meta name="citation_reference" content="citation_title=Analysis of a genetic hitchhiking model, and its application to DNA polymorphism data from Drosophila melanogaster.; citation_author=THE Wiehe; citation_author=W Stephan; citation_journal_title=Mol Biol Evol; citation_volume=10; citation_number=7; citation_pages=842-854; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=Deleterious background selection with recombination.; citation_author=RR Hudson; citation_author=NL Kaplan; citation_journal_title=Genetics; citation_volume=141; citation_number=8; citation_pages=1605-1617; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=The effect of recombination on background selection.; citation_author=M Nordborg; citation_author=B Charlesworth; citation_author=D Charlesworth; citation_journal_title=Genet Res; citation_volume=67; citation_number=9; citation_pages=159-174; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Patterns of genetic variation at a Chromosome 4 locus of Drosophila melanogaster and D. simulans.; citation_author=MA Jensen; citation_author=B Charlesworth; citation_author=M Kreitman; citation_journal_title=Genetics; citation_volume=160; citation_number=10; citation_pages=493-507; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Nucleotide variation along the Drosophila melanogaster fourth chromosome.; citation_author=W Wang; citation_author=K Thornton; citation_author=A Berry; citation_author=M Long; citation_journal_title=Science; citation_volume=295; citation_number=11; citation_pages=134-137; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Nucleotide variation and recombination along the fourth chromosome in Drosophila simulans.; citation_author=W Wang; citation_author=K Tornton; citation_author=JJ Emerson; citation_author=M Long; citation_journal_title=Genetics; citation_volume=166; citation_number=12; citation_pages=1783-1794; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Lack of nucleotide polymorphism in the Y-linked sperm flagellar dynein gene Dhc-Yh3 of Drosophila melanogaster and D. simulans.; citation_author=M Zurovcova; citation_author=WF Eanes; citation_journal_title=Genetics; citation_volume=153; citation_number=13; citation_pages=1709-1715; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Low variability in a Y-linked plant gene and its implications for Y-chromosome evolution.; citation_author=DA Filatov; citation_author=F Moneger; citation_author=I Negrutiu; citation_author=D Charlesworth; citation_journal_title=Nature; citation_volume=404; citation_number=14; citation_pages=388-390; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Low levels of nucleotide diversity in mammalian Y chromosomes.; citation_author=L Hellborg; citation_author=H Ellegren; citation_journal_title=Mol Biol Evol; citation_volume=21; citation_number=15; citation_pages=158-163; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Chicken W: A genetically uniform chromosome in a highly variable genome.; citation_author=S Berlin; citation_author=H Ellegren; citation_journal_title=Proc Natl Acad Sci; citation_volume=101; citation_number=16; citation_pages=15967-15969; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Steps in the evolution of heteromorphic sex chromosomes.; citation_author=D Charlesworth; citation_author=B Charlesworth; citation_author=G Marais; citation_journal_title=Heredity; citation_volume=95; citation_number=17; citation_pages=118-128; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Chromosomal and allelic variation in Drosophila americana: selective maintenance of a chromosomal cline.; citation_author=BF McAllister; citation_journal_title=Genome; citation_volume=45; citation_number=18; citation_pages=13-21; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Crossing-over in the male of Drosophila virilis.; citation_author=H Kikkawa; citation_journal_title=Cytologia; citation_volume=6; citation_number=19; citation_pages=190-194; citation_date=1935; " />
      <meta name="citation_reference" content="citation_title=The effect of change in population size on DNA polymorphism.; citation_author=F Tajima; citation_journal_title=Genetics; citation_volume=123; citation_number=20; citation_pages=597-601; citation_date=1989; " />
      <meta name="citation_reference" content="citation_title=The effect of deleterious mutations on neutral molecular variation.; citation_author=B Charlesworth; citation_author=MT Morgan; citation_author=D Charlesworth; citation_journal_title=Genetics; citation_volume=134; citation_number=21; citation_pages=1289-1303; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=The degeneration of Y chromosomes.; citation_author=B Charlesworth; citation_author=D Charlesworth; citation_journal_title=Phil Trans R Soc Lond B; citation_volume=355; citation_number=22; citation_pages=1563-1572; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Reduced sequence variability on the neo-Y chromosome of Drosophila americana americana.; citation_author=BF McAllister; citation_author=B Charlesworth; citation_journal_title=Genetics; citation_volume=153; citation_number=23; citation_pages=221-233; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Sequence differentiation associated with an inversion on the neo-X chromosome of Drosophila americana.; citation_author=BF McAllister; citation_journal_title=Genetics; citation_volume=165; citation_number=24; citation_pages=1317-1328; citation_date=2003; " />
      <meta name="citation_reference" content="citation_author=WC Blight; citation_number=25; citation_pages=50; citation_date=1955; citation_publisher=Washington University; " />
      <meta name="citation_reference" content="citation_title=The accumulation of sexually antagonistic genes as a selective agent promoting the evolution of reduced recombination between primitive sex chromosomes.; citation_author=WR Rice; citation_journal_title=Evolution; citation_volume=41; citation_number=26; citation_pages=911-914; citation_date=1987; " />
      <meta name="citation_reference" content="citation_title=The coalescent process in models with selection and recombination.; citation_author=RR Hudson; citation_author=NL Kaplan; citation_journal_title=Genetics; citation_volume=120; citation_number=27; citation_pages=831-840; citation_date=1988; " />
      <meta name="citation_reference" content="citation_title=Extending the coalescent to multilocus systems: the case of balancing selection.; citation_author=N Barton; citation_author=A Navarro; citation_journal_title=Genet Res; citation_volume=79; citation_number=28; citation_pages=129-139; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Balancing selection and its effects on sequences in nearby genome regions.; citation_author= Charlesworth; citation_author= D; citation_journal_title=PLoS Genet; citation_volume=2; citation_number=29; citation_pages=e64; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Statistical method for testing the neutral mutation hypothesis by DNA polymorphism.; citation_author=F Tajima; citation_journal_title=Genetics; citation_volume=123; citation_number=30; citation_pages=585-595; citation_date=1989; " />
      <meta name="citation_reference" content="citation_title=Statistical tests of neutrality.; citation_author=Y-X Fu; citation_author=W-H Li; citation_journal_title=Genetics; citation_volume=133; citation_number=31; citation_pages=693-709; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=Hitchhiking under positive Darwinian selection.; citation_author=JC Fay; citation_author=C-I Wu; citation_journal_title=Genetics; citation_volume=155; citation_number=32; citation_pages=1405-1413; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Measures of divergence between populations and the effect of forces that reduce variability.; citation_author=B Charlesworth; citation_journal_title=Mol Biol Evol; citation_volume=15; citation_number=33; citation_pages=538-543; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Hitchhiking mapping : A population-based fine-mapping strategy for adaptive mutations in Drosophila melanogaster.; citation_author=B Harr; citation_author=M Kauer; citation_author=C Schlötterer; citation_journal_title=Proc Natl Acad Sci USA; citation_volume=99; citation_number=34; citation_pages=12949-12954; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=A microsatellite-based multilocus screen for the identification of local selective sweeps.; citation_author=C Schlötterer; citation_journal_title=Genetics; citation_volume=160; citation_number=35; citation_pages=753-763; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Neutral evolution of the sex-determining gene transformer in Drosophila.; citation_author=BF McAllister; citation_author=GAT McVean; citation_journal_title=Genetics; citation_volume=154; citation_number=36; citation_pages=1711-1720; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Background selection and patterns of genetic diversity in Drosophila melanogaster.; citation_author=B Charlesworth; citation_journal_title=Genet Res; citation_volume=68; citation_number=37; citation_pages=131-149; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=A test of the background selection hypothesis based on nucleotide data from Drosophila ananassae.; citation_author=W Stephan; citation_author=L Xing; citation_author=DA Kirby; citation_author=JM Braverman; citation_journal_title=Proc Natl Acad Sci USA; citation_volume=95; citation_number=38; citation_pages=5649-5654; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Species and recombination effects on DNA variability in the tomato genus.; citation_author=E Baudry; citation_author=C Kerdelhue; citation_author=H Innan; citation_author=W Stephan; citation_journal_title=Genetics; citation_volume=158; citation_number=39; citation_pages=1725-1735; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Distinguishing the hitchhiking and background selection models.; citation_author=H Innan; citation_author=W Stephan; citation_journal_title=Genetics; citation_volume=165; citation_number=40; citation_pages=2307-2312; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Structured coalescent processes on different time scales.; citation_author=M Nordborg; citation_journal_title=Genetics; citation_volume=146; citation_number=41; citation_pages=1501-1514; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Patterns of microsatellite variation through a transition zone of a chromosomal cline in Drosophila americana.; citation_author=MA Schäfer; citation_author=L Orsini; citation_author=BF McAllister; citation_author= Schlötterer; citation_journal_title=Heredity; citation_volume=97; citation_number=42; citation_pages=291-295; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Joint effects of genetic hitchhiking and background selection on neutral variation.; citation_author=Y Kim; citation_author=W Stephan; citation_journal_title=Genetics; citation_volume=155; citation_number=43; citation_pages=1415-1427; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Gene flow and population differentiation.; citation_author=JA Endler; citation_journal_title=Science; citation_volume=179; citation_number=44; citation_pages=243-250; citation_date=1973; " />
      <meta name="citation_reference" content="citation_title=Gene flow and the limits to natural selection.; citation_author=T Lenormand; citation_journal_title=Trends Ecol Evol; citation_volume=17; citation_number=45; citation_pages=183-189; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Adaptive loss of an old duplicated gene during incipient speciation.; citation_author=AJ Greenberg; citation_author=JR Moran; citation_author=S Fang; citation_author=C-I Wu; citation_journal_title=Mol Biol Evol; citation_volume=23; citation_number=46; citation_pages=401-410; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Evidence for selection at the fused1 locus of Drosophila americana.; citation_author=J Vieira; citation_author=BF McAllister; citation_author=B Charlesworth; citation_journal_title=Genetics; citation_volume=158; citation_number=47; citation_pages=279-290; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Inferring the evolutionary histories of the Adh and Adh-dup loci in Drosophila melanogaster from patterns of polymorphism and divergence.; citation_author=M Kreitman; citation_author=RR Hudson; citation_journal_title=Genetics; citation_volume=127; citation_number=48; citation_pages=565-582; citation_date=1991; " />
      <meta name="citation_reference" content="citation_title=The effects of multilocus balancing selection on neutral variability.; citation_author=A Navarro; citation_author=NH Barton; citation_journal_title=Genetics; citation_volume=161; citation_number=49; citation_pages=849-863; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Sex chromosomes and the evolution of sexual dimorphism.; citation_author=WR Rice; citation_journal_title=Evolution; citation_volume=38; citation_number=50; citation_pages=735-742; citation_date=1984; " />
      <meta name="citation_reference" content="citation_title=The X chromosome is a hot spot for sexually antagonistic fitness variation.; citation_author=JR Gibson; citation_author=AK Chippindale; citation_author=WR Rice; citation_journal_title=Proc Royal Soc Lond (B); citation_volume=269; citation_number=51; citation_pages=499-505; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Sexually antagonistic genes: experimental evidence.; citation_author=WR Rice; citation_journal_title=Science; citation_volume=256; citation_number=52; citation_pages=1436-1439; citation_date=1992; " />
      <meta name="citation_reference" content="citation_title=Male fitness increases when females are eliminated from gene pool: Implications for the Y chromosome.; citation_author=WR Rice; citation_journal_title=Proc Natl Acad Sci USA; citation_volume=95; citation_number=53; citation_pages=6217-6221; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Contrasting patterns of molecular evolution of genes on the new and old sex chromosomes of Drosophila miranda.; citation_author=S Yi; citation_author=B Charlesworth; citation_journal_title=Mol Biol Evol; citation_volume=17; citation_number=54; citation_pages=703-717; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Reduced levels of microsatellite variability on the neo-Y chromosome of Drosophila miranda.; citation_author=D Bachtrog; citation_author=B Charlesworth; citation_journal_title=Curr Biol; citation_volume=10; citation_number=55; citation_pages=1025-1031; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Reduced adaptation of a non-recombining neo-Y chromosome.; citation_author=D Bachtrog; citation_author=B Charlesworth; citation_journal_title=Nature; citation_volume=416; citation_number=56; citation_pages=323-326; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Evidence that positive selection drives Y-chromosome degeneration in Drosophila miranda.; citation_author=D Bachtrog; citation_journal_title=Nat Genet; citation_volume=36; citation_number=57; citation_pages=518-522; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Genetic hitchhiking and the evolution of reduced genetic activity of the Y sex chromosome.; citation_author=WR Rice; citation_journal_title=Genetics; citation_volume=116; citation_number=58; citation_pages=161-167; citation_date=1987; " />
      <meta name="citation_reference" content="citation_title=DNA diversity and sex-linked and autosomal genes of the plant species Silene latifolia and Silene dioica.; citation_author=DA Filatov; citation_author=V Laporte; citation_author=C Vitte; citation_author=D Charlesworth; citation_journal_title=Mol Biol Evol; citation_volume=18; citation_number=59; citation_pages=1442-1454; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Selective trade-offs and sex-chromosome evolution in Silene latifolia.; citation_author=I Scotti; citation_author=LF Delph; citation_journal_title=Evolution; citation_volume=60; citation_number=60; citation_pages=1793-1800; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Lack of degeneration of loci on the neo-Y chromosome of Drosophila americana americana.; citation_author=B Charlesworth; citation_author=D Charlesworth; citation_author=J Hnilicka; citation_author=A Yu; citation_author=DS Guttman; citation_journal_title=Genetics; citation_volume=145; citation_number=61; citation_pages=989-1002; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Cytological and linkage maps of Drosophila virilis chromosomes.; citation_author=IS Gubenko; citation_author=MB Evgen'ev; citation_journal_title=Genetica; citation_volume=65; citation_number=62; citation_pages=127-139; citation_date=1984; " />
      <meta name="citation_reference" content="citation_title=Molecular phylogeny and genome evolution in the Drosophila virilis species group: duplications of the Alcohol Dehydrogenase gene.; citation_author=DI Nurminsky; citation_author=EN Moriyama; citation_author=ER Lozovskaya; citation_author=DL Hartl; citation_journal_title=Mol Biol Evol; citation_volume=13; citation_number=63; citation_pages=132-149; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Sequence evolution of the Gpdh gene in the Drosophila virilis species group.; citation_author=H Tominaga; citation_author=S Narise; citation_journal_title=Genetica; citation_volume=96; citation_number=64; citation_pages=293-302; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=A combined molecular and cytogenetic approach to genome evolution in Drosophila using large-fragment DNA cloning.; citation_author=ER Lozovskaya; citation_author=DA Petrov; citation_author=DL Hartl; citation_journal_title=Chromosoma; citation_volume=102; citation_number=65; citation_pages=253-266; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=A framework physical map of Drosophila virilis based on P1 clones: application in genome evolution.; citation_author=J Vieira; citation_author=C Vieira; citation_author=DL Hartl; citation_author=ER Lozovskaya; citation_journal_title=Chromosoma; citation_volume=106; citation_number=66; citation_pages=99-107; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=PipMaker—A web server for aligning two genomic DNA sequences.; citation_author=S Schwartz; citation_author=Z Zhang; citation_author=KA Frazer; citation_author=A Smit; citation_author=C Riemer; citation_author=J Bouck; citation_author=R Gibbs; citation_author=R Hardison; citation_author=W Miller; citation_journal_title=Genome Res.; citation_volume=10; citation_number=67; citation_pages=577-586; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=DnaSP, DNA polymorphism analyses by the coalescent and other methods.; citation_author=J Rozas; citation_author=JC Sánchez-DelBarrio; citation_author=X Messeguer; citation_author=R Rozas; citation_journal_title=Bioinformatics; citation_volume=19; citation_number=68; citation_pages=2496-2497; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=A statistical test for detecting geographic subdivision.; citation_author=RR Hudson; citation_author=DD Boos; citation_author=NL Kaplan; citation_journal_title=Mol Biol Evol; citation_volume=9; citation_number=69; citation_pages=138-151; citation_date=1992; " />
      <meta name="citation_reference" content="citation_title=Estimating the recombination parameter of a finite population model without selection.; citation_author=RR Hudson; citation_journal_title=Genet Res; citation_volume=50; citation_number=70; citation_pages=245-250; citation_date=1987; " />
      <meta name="citation_reference" content="citation_title=Evolutionary relationship of DNA sequences in finite populations.; citation_author=F Tajima; citation_journal_title=Genetics; citation_volume=105; citation_number=71; citation_pages=437-460; citation_date=1983; " />
      <meta name="citation_reference" content="citation_author=ME Stokes; citation_author=CS Davis; citation_author=GG Koch; citation_number=72; citation_pages=626; citation_date=2000; citation_publisher=SAS Institute Inc; " />
      <meta name="citation_reference" content="citation_title=Bootstrapping confidence intervals for statistics with ‘difficult’ distributions.; citation_author=JS Haukoos; citation_author=RJ Lewis; citation_journal_title=Acad Emerg Med; citation_volume=12; citation_number=73; citation_pages=360-365; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Preparation and analysis of polytene chromosomes.; citation_author=JA Kennison; citation_number=74; citation_pages=111-117; citation_date=2000; citation_publisher=Cold Spring Harbor Laboratory Press; " />
      <meta name="citation_reference" content="citation_title=Overview of QTL mapping software and introduction to Map Manager QT.; citation_author=KF Manly; citation_author=JM Olson; citation_journal_title=Mammalian Genome; citation_volume=10; citation_number=75; citation_pages=327-334; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Gapped BLAST and PSI-BLAST: a new generation of protein database search programs.; citation_author=SF Altschul; citation_author=TL Madden; citation_author=AA Schaffer; citation_author=J Zhang; citation_author=Z Zhang; citation_journal_title=Nucl Acids Res; citation_volume=25; citation_number=76; citation_pages=3389-3402; citation_date=1997; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000112" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Increased Nucleotide Diversity with Transient Y Linkage in Drosophila americana"/>
    <meta name="twitter:description" content="Recombination shapes nucleotide variation within genomes. Patterns are thought to arise from the local recombination landscape, influencing the degree to which neutral variation experiences hitchhiking with selected variation. This study examines DNA polymorphism along Chromosome 4 (element B) of Drosophila americana to identify effects of hitchhiking arising as a consequence of Y-linked transmission. A centromeric fusion between the X and 4th chromosomes segregates in natural populations of D. americana. Frequency of the X-4 fusion exhibits a strong positive correlation with latitude, which has explicit consequences for unfused 4th chromosomes. Unfused Chromosome 4 exists as a non-recombining Y chromosome or as an autosome proportional to the frequency of the X-4 fusion. Furthermore, Y linkage along the unfused 4 is disrupted as a function of the rate of recombination with the centromere. Inter-population and intra-chromosomal patterns of nucleotide diversity were assayed using six regions distributed along unfused 4th chromosomes derived from populations with different frequencies of the X-4 fusion. No difference in overall level of nucleotide diversity was detected among populations, yet variation along the chromosome exhibits a distinct pattern in relation to the X-4 fusion. Sequence diversity is inflated at loci experiencing the strongest Y linkage. These findings are inconsistent with the expected reduction in nucleotide diversity resulting from hitchhiking due to background selection or selective sweeps. In contrast, excessive polymorphism is accruing in association with transient Y linkage, and furthermore, hitchhiking with sexually antagonistic alleles is potentially responsible."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0000112.g004"/>

  <meta property="og:title" content="Increased Nucleotide Diversity with Transient Y Linkage in Drosophila americana" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000112" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=7906'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=721'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=2288&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0000112">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000112" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000112&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Increased Nucleotide Diversity with Transient Y Linkage in <i>Drosophila americana</i>
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Bryant F. McAllister
              <span class="corresponding">mail</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:bryant-mcallister@uiowa.edu">bryant-mcallister@uiowa.edu</a></p>

                <p>Affiliation:
                  Department of Biological Sciences and Roy J. Carver Center for Comparative Genomics, University of Iowa, Iowa City, Iowa, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Amy L. Evans
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Biological Sciences and Roy J. Carver Center for Comparative Genomics, University of Iowa, Iowa City, Iowa, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: December 27, 2006</li>
    <li>DOI: 10.1371/journal.pone.0000112</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0000112" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0000112" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000112" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0000112" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0000112-g001" data-doi="info:doi/10.1371/journal.pone.0000112" data-uri="info:doi/10.1371/journal.pone.0000112.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000112.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000112-t001" data-doi="info:doi/10.1371/journal.pone.0000112" data-uri="info:doi/10.1371/journal.pone.0000112.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000112.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000112-g002" data-doi="info:doi/10.1371/journal.pone.0000112" data-uri="info:doi/10.1371/journal.pone.0000112.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000112.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000112-t002" data-doi="info:doi/10.1371/journal.pone.0000112" data-uri="info:doi/10.1371/journal.pone.0000112.t002" title="Table 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000112.t002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000112-t003" data-doi="info:doi/10.1371/journal.pone.0000112" data-uri="info:doi/10.1371/journal.pone.0000112.t003" title="Table 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000112.t003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000112-t004" data-doi="info:doi/10.1371/journal.pone.0000112" data-uri="info:doi/10.1371/journal.pone.0000112.t004" title="Table 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000112.t004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000112-g003" data-doi="info:doi/10.1371/journal.pone.0000112" data-uri="info:doi/10.1371/journal.pone.0000112.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000112.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000112-g004" data-doi="info:doi/10.1371/journal.pone.0000112" data-uri="info:doi/10.1371/journal.pone.0000112.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000112.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000112">Reader Comments (3)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0000112" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1"></a><p>Recombination shapes nucleotide variation within genomes. Patterns are thought to arise from the local recombination landscape, influencing the degree to which neutral variation experiences hitchhiking with selected variation. This study examines DNA polymorphism along Chromosome 4 (element B) of <em>Drosophila americana</em> to identify effects of hitchhiking arising as a consequence of Y-linked transmission. A centromeric fusion between the X and 4<sup>th</sup> chromosomes segregates in natural populations of <em>D. americana</em>. Frequency of the X-4 fusion exhibits a strong positive correlation with latitude, which has explicit consequences for unfused 4<sup>th</sup> chromosomes. Unfused Chromosome 4 exists as a non-recombining Y chromosome or as an autosome proportional to the frequency of the X-4 fusion. Furthermore, Y linkage along the unfused 4 is disrupted as a function of the rate of recombination with the centromere. Inter-population and intra-chromosomal patterns of nucleotide diversity were assayed using six regions distributed along unfused 4<sup>th</sup> chromosomes derived from populations with different frequencies of the X-4 fusion. No difference in overall level of nucleotide diversity was detected among populations, yet variation along the chromosome exhibits a distinct pattern in relation to the X-4 fusion. Sequence diversity is inflated at loci experiencing the strongest Y linkage. These findings are inconsistent with the expected reduction in nucleotide diversity resulting from hitchhiking due to background selection or selective sweeps. In contrast, excessive polymorphism is accruing in association with transient Y linkage, and furthermore, hitchhiking with sexually antagonistic alleles is potentially responsible.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>McAllister BF, Evans AL (2006) Increased Nucleotide Diversity with Transient Y Linkage in <em>Drosophila americana</em>. PLoS ONE 1(1):
          e112.
            doi:10.1371/journal.pone.0000112</p><p><strong>Academic Editor: </strong>Adam Chippindale, Queens University, Canada</p><p><strong>Received:</strong> September 6, 2006; <strong>Accepted:</strong> November 21, 2006; <strong>Published:</strong> December 27, 2006</p><p><strong>Copyright:</strong> © 2006 McAllister, Evans. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>This article is based upon work supported by the National Science Foundation under grant Nos. DEB-0075295/0228832 and the Roy J. Carver Charitable Trust under grant No. 05-2045.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>Genome-wide surveys generally reveal a positive relationship between the local rate of meiotic recombination and the level of nucleotide diversity <a href="#pone.0000112-Begun1">[1]</a>, <a href="#pone.0000112-Nachman1">[2]</a>, <a href="#pone.0000112-Cutter1">[3]</a>. Local recombination rates determine the interdependence of neutral, beneficial and deleterious mutations, thus a reduction in recombination is accompanied by a diminished capacity for natural selection to act individually on each new mutation <a href="#pone.0000112-Hill1">[4]</a>, <a href="#pone.0000112-Felsenstein1">[5]</a>, <a href="#pone.0000112-MaynardSmith1">[6]</a>. Hitchhiking of neutral variants as a consequence of selection on nearby mutations is a likely cause of spatial patterns of nucleotide diversity throughout genomes <a href="#pone.0000112-Wiehe1">[7]</a>, <a href="#pone.0000112-Hudson1">[8]</a>, <a href="#pone.0000112-Nordborg1">[9]</a>. Reduced nucleotide variation associated with low or no recombination can extend throughout entire chromosomes, such as the “dot” chromosome in species of Drosophila <a href="#pone.0000112-Jensen1">[10]</a>, <a href="#pone.0000112-Wang1">[11]</a>, <a href="#pone.0000112-Wang2">[12]</a>, and the Y or W sex chromosome in many species <a href="#pone.0000112-Zurovcova1">[13]</a>, <a href="#pone.0000112-Filatov1">[14]</a>, <a href="#pone.0000112-Hellborg1">[15]</a>, <a href="#pone.0000112-Berlin1">[16]</a>. Although examples of reduced nucleotide variation in regions experiencing chronically low recombination are prevalent, there is little agreement on the impact of beneficial versus deleterious mutation and the magnitude of the selective forces responsible for structuring variation in these regions.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>Newly arisen sex chromosomes provide a unique system for examining patterns of sequence diversity in response to a change in the rate of recombination. After they emerge from a homologous pair of autosomes, mechanisms usually arise to suppress meiotic crossing over between X and Y chromosomes <a href="#pone.0000112-Charlesworth1">[17]</a>. This crossover suppression creates an asymmetry in recombination rate between the X chromosome, which can recombine in females, and the Y chromosome, which is genetically isolated in males. An example of an emerging Y chromosome is present in <em>Drosophila americana</em>. Chromosome 4 (Muller element B) exists in a dynamic state shifting from an autosomal pair to a sex-chromosome pair along a latitudinal gradient of this species' range. This unique situation arises due to the presence of a derived centromeric fusion of the X and 4<sup>th</sup> chromosomes that is positively correlated with latitude <a href="#pone.0000112-McAllister1">[18]</a>. Unfused Chromosome 4 segregates autosomally in the absence of the X-4 fusion. An unfused 4<sup>th</sup> chromosome exists as a secondary Y chromosome in males with the X-4 fusion (<a href="#pone-0000112-g001">Fig. 1</a>). Given the absence of crossing over in male Drosophila <a href="#pone.0000112-Kikkawa1">[19]</a>, the rate of recombination among loci on these neo-Y chromosomes is, therefore, directly influenced by the frequency of the X-4 fusion.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>Reduction in nucleotide diversity is a predicted outcome of restricting unfused 4<sup>th</sup> chromosomes in males as neo-Y chromosomes. In addition to the reduction expected as a consequence of a smaller effective population size, which would accrue slowly through genetic drift <a href="#pone.0000112-Tajima1">[20]</a>, reduction in diversity may also be driven by background selection as a result of negative selection pressure <a href="#pone.0000112-Charlesworth2">[21]</a>, by selective sweeps as a result of positive selection pressure <a href="#pone.0000112-MaynardSmith1">[6]</a>, or by a combination of these and other related processes <a href="#pone.0000112-Charlesworth3">[22]</a>. Previous comparison of nucleotide diversity between unfused and fused 4<sup>th</sup> chromosomes of <em>D. americana</em> from a single northern population having a high (~98%) frequency of the X-4 fusion revealed an apparent deficit of nucleotide polymorphism on the unfused arrangement, which was interpreted as a loss of diversity <a href="#pone.0000112-McAllister2">[23]</a>. However, subsequent discovery of non-neutral patterns of sequence variation on the fused 4<sup>th</sup> chromosome raises uncertainty about the assumptions underlying this contrast, because nucleotide diversity on the fused 4<sup>th</sup> chromosome has been influenced by a segregating chromosomal inversion <a href="#pone.0000112-McAllister3">[24]</a>. In this study, an alternative approach is used to examine the initial effects of reduced recombination on this neo-Y chromosome. Patterns of sequence diversity are contrasted among unfused 4<sup>th</sup> chromosomes obtained from populations that differ in the frequency of the X-4 fusion and thus represent different degrees of chromosome-wide reductions in recombination.</p>
<div class="figure" id="pone-0000112-g001"><div class="img"><a name="pone-0000112-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000112.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000112" data-uri="info:doi/10.1371/journal.pone.0000112.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000112.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000112.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000112.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000112.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000112.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000112.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000112.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000112.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000112.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Characteristics of Chromosome 4 in <em>Drosophila americana.</em></span></strong></p><a id="article1.body1.sec1.fig1.caption1.p1" name="article1.body1.sec1.fig1.caption1.p1"></a><p>The unfused arrangement of Chromosome 4 is highlighted in gray. (A) Karyotypic variation involving the X and 4<sup>th</sup> chromosome as a consequence of the X-4 fusion. Frequency of the X-4 fusion determines the relative prevalence of each karyotype. Female karyotypes are expected to occur according to Hardy-Weinberg equilibrium based on a single locus with two alleles, and male karyotypes occur in proportion to the frequencies of the two arrangements of the X. (B) Idealized arrangement of Chromosome 4 during meiosis in a heterokaryotypic female having both X chromosome arrangements. Positions of the loci used in the analyses are indicated relative to the euchromatic subdivisions of Chromosome 4. Sequenced regions are underlined. An inversion of the X chromosome (<em>In(X)c</em>) discriminates the two lines used in the linkage analysis, and a box demarcates its approximate position.</p>
<span>doi:10.1371/journal.pone.0000112.g001</span></div><a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p>The unfused arrangement of Chromosome 4 is not genetically isolated from its fused homolog. These arrangements coexist over a wide geographic region <a href="#pone.0000112-McAllister1">[18]</a>, and the occurrence of heterokaryotypic females with both arrangements is consistent with Hardy-Weinberg expectations [25, McAllister et al. unpublished data]. Heterokaryotypic females create ample opportunities for meiotic exchange between alternatively arranged chromosomes, so location on Chromosome 4 influences linkage relationships with the two centromeric arrangements and ultimately with the sex chromosomes (<a href="#pone-0000112-g001">Fig. 1</a>). Exchange between arrangements in populations with a high frequency of fused X chromosomes creates a gradient of sex linkage along Chromosome 4. Reduction in diversity on the unfused arrangement through drift, background selection or selective sweeps should be greatest in proximal chromosomal regions within northern populations. Other selective forces may generate alternative effects on sequence diversity. For example, sexually antagonistic alleles that increase fitness in males and decrease fitness in females are prone to accumulate as Y-linked polymorphisms <a href="#pone.0000112-Rice1">[26]</a>. If a sexually antagonistic allele increases to intermediate frequency by selection, but is impeded from reaching fixation, a balanced polymorphism between the antagonistic and neutral allele would be maintained at the locus. Balanced polymorphisms inflate levels of linked neutral variation <a href="#pone.0000112-Hudson2">[27]</a>, <a href="#pone.0000112-Barton1">[28]</a>, <a href="#pone.0000112-Charlesworth4">[29]</a>. Alternative selective forces may, therefore, be distinguishable by examining the pattern of sequence diversity along the unfused arrangement of Chromosome 4.</p>
<a id="article1.body1.sec1.p5" name="article1.body1.sec1.p5"></a><p>This study leverages the robust cline in the frequency of the X-4 fusion to examine patterns of sequence diversity shaped by recombination and Y linkage at two levels: 1) across populations with different frequencies of the X-4 fusion, and 2) along the chromosome where loci experience different levels of Y linkage. Six gene regions distributed along Chromosome 4 were assessed for patterns of sequence variation. Sequences were obtained specifically from the unfused 4<sup>th</sup> chromosome in four different samples of <em>D. americana</em> from contiguous populations representing a wide spectrum in the frequency of the X-4 fusion. An overall difference in levels of sequence diversity is not evident among populations. However, a distinct pattern along the chromosome is revealed in relation to the X-4 fusion. Loci experiencing the strongest Y linkage exhibit inflated levels of sequence diversity, which is contrary to expectations from background selection and selective sweeps. Simulations demonstrate the propensity for sexually antagonistic alleles to accumulate as polymorphisms in tightly Y-linked regions of Chromosome 4. The observed pattern of sequence diversity is consistent with this form of intra-locus sexual conflict generating pervasive effects along this transient Y chromosome.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3>
<h4>Variation Across Populations</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>Contrasting populations of <em>D. americana</em> that differ in the frequency of the X-4 fusion identifies immediate effects associated with a localized reduction in the rate of meiotic recombination. Samples of the unfused 4<sup>th</sup> chromosome were obtained from flies collected near rivers in the eastern sections of the contiguous states of Arkansas (FP), Missouri (HI), and Iowa (IR &amp; SB) in the USA. This fly species inhabits riparian areas, and given that each sample site is near a tributary of the Mississippi River, the opportunity exists for a high rate of gene flow among populations. Even with this potential for gene flow, observed frequencies of the X-4 fusion exhibit large differences among these samples and are representative of more extensive sampling from the Mississippi River Valley (<a href="#pone-0000112-t001">Table 1</a>). Estimates of the frequencies of the X-4 fusion are 16% in the southern population represented by FP, 85% in the northerly population represented by HI, and 97% in the northernmost population represented by both IR and SB. Based on these frequencies and assuming Hardy-Weinberg equilibrium, 97% of males in the northernmost population exhibit sex linkage of Chromosome 4, whereas about 6% of females are heterokaryotypes. Sex linkage in HI and FP is almost exactly opposite (85% of males in HI and 16% of males in FP), whereas the frequency of heterokaryotypic females is about 25% in each. Variability in the frequency of the X-4 fusion creates explicit dynamics with potential consequences on patterns of sequence diversity across these populations and along Chromosome 4.</p>
<div class="figure" id="pone-0000112-t001"><div class="img"><a name="pone-0000112-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000112.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000112" data-uri="info:doi/10.1371/journal.pone.0000112.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000112.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000112.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000112.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000112.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000112.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000112.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000112.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000112.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000112.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>Frequency of the X-4 fusion and consequent reduction in meiotic recombination on unfused Chromosome 4.</span></strong></p><span>doi:10.1371/journal.pone.0000112.t001</span></div><a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2"></a><p>A high frequency of fused X chromosomes decreases the overall rate of recombination on the unfused arrangement of Chromosome 4. This effect arises from restricting the unfused 4 in males, which lack meiotic crossing over. Based on the proportion of fused X chromosomes in the populations studied here, meiotic recombination between any two loci on unfused Chromosome 4 relative to their autosomal rate is 0.95 in FP, 0.41 in HI, and 0.11 in IR/SB (<a href="#pone-0000112-t001">Table 1</a>), thus comparisons of FP versus IR or SB represent about a 10-fold reduction in the recombination rate per generation between any two loci on this chromosome. One potential consequence of this reduction is that the chromosomal region of reduced nucleotide diversity associated with a geographically localized selective sweep limited to unfused Chromosome 4 is expected to be about 10 times wider in the northernmost population than in the southern population <a href="#pone.0000112-Wiehe1">[7]</a>. An overall reduction in recombination intensifies background selection, but the degree to which its effects arise depends on the ratio of the deleterious mutation and local recombination rates <a href="#pone.0000112-Hudson1">[8]</a>. Equilibrium levels of nucleotide diversity are not expected to exhibit differences among these populations if this ratio is less than 0.01 in the southern population. On the other hand, a higher rate of deleterious mutation or a lower rate of recombination is expected to generate a difference in levels of nucleotide diversity at mutation/selection equilibrium (<em>e.g.</em>, a ratio of 0.1 in FP is expected to generate over a two-fold reduction in standing level of diversity in IR/SB). Sequence diversity was compared across these populations to detect evidence of selective sweeps or background selection.</p>
<a id="article1.body1.sec2.sec1.p3" name="article1.body1.sec2.sec1.p3"></a><p>Ten unfused 4<sup>th</sup> chromosomes from each sample were assayed for sequence polymorphism at six gene regions distributed along the chromosome. Descriptive statistics of nucleotide polymorphism are presented in <a href="#pone-0000112-t002">Table 2</a> (sequences deposited in GenBank under accessions EF136929-EF137168). Selective sweeps and background selection are both expected to decrease sequence diversity in relation to decreased recombination. Measures of silent pairwise diversity for each gene region in each sample are presented in <a href="#pone-0000112-g002">Figure 2A</a>. No difference in the rank score of diversity is evident among the populations controlling for differences among genes (<em>p</em>&gt;0.1, Friedman ANOVA). Furthermore, there is no statistical evidence (<em>p</em>&gt;0.1, Spearman rank correlation) for a correlation between sequence diversity and relative recombination rate. Similar results with the same conclusions are obtained in analyses of segregating mutations. Shifts in the frequency spectra of segregating sites are not evident among the populations. Tajima's <em>D</em> <a href="#pone.0000112-Tajima2">[30]</a> statistic is reported in <a href="#pone-0000112-t002">Table 2</a>, but similar results were observed for Fu and Li's <em>D</em> <a href="#pone.0000112-Fu1">[31]</a> and Fay and Wu's <em>H</em> <a href="#pone.0000112-Fay1">[32]</a>. These data indicate that strongly selected mutations, either beneficial or deleterious, have not uniformly reduced levels of nucleotide diversity or altered the distribution of segregating polymorphisms on the unfused 4<sup>th</sup> chromosome in populations where it experiences reduced levels of meiotic exchange.</p>
<div class="figure" id="pone-0000112-g002"><div class="img"><a name="pone-0000112-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000112.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000112" data-uri="info:doi/10.1371/journal.pone.0000112.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000112.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000112.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000112.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000112.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000112.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000112.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000112.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000112.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000112.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Nucleotide diversity on unfused Chromosome 4.</span></strong></p><a id="article1.body1.sec2.sec1.fig1.caption1.p1" name="article1.body1.sec2.sec1.fig1.caption1.p1"></a><p>(A) Average pairwise silent diversity across populations. Samples are arranged with FP, which has the lowest frequency of the X-4 fusion, on the left and SB, which has the highest observed frequency of the X-4 fusion, on the right. No significant heterogeneity is evident among populations. (B) Pattern of sequence diversity along Chromosome 4. Loci are arranged with the centromere to the left and telomere to the right. Average pairwise silent diversity for each sample is plotted to show heterogeneity among loci and the lack of an overall pattern relative to the centromere. (C) Standardized diversity in northern populations. Total pairwise diversity in each northern sample relative to FP is plotted for each locus. A significant relationship between chromosomal position and relative diversity is indicated by the Spearman rank correlation coefficient (<em>r<sub>s</sub></em> = −0.78, 95% CI: −0.94, −0.40).</p>
<span>doi:10.1371/journal.pone.0000112.g002</span></div><div class="figure" id="pone-0000112-t002"><div class="img"><a name="pone-0000112-t002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000112.t002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000112" data-uri="info:doi/10.1371/journal.pone.0000112.t002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000112.t002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000112.t002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000112.t002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000112.t002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000112.t002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000112.t002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000112.t002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000112.t002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000112.t002.TIF"></span>)
                </a></li></ul></div><p><strong>Table 2.  <span>Summary statistics describing sequence diversity in four samples of <em>D. americana</em>.</span></strong></p><span>doi:10.1371/journal.pone.0000112.t002</span></div>

<h4>Gene Flow</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>One criterion for selecting the samples examined here was their potential for maintaining historically high levels of reciprocal gene flow. Consistent with this expectation, overall differentiation measured by <em>K<sub>st</sub></em> is 0.001, which permutation tests reveal is not significantly different from zero (other measures of differentiation yield the same result). Pairwise comparisons among samples revealed no cases of statistically significant differentiation. The highest levels of differentiation are observed for comparisons with SB (<a href="#pone-0000112-t003">Table 3</a>), which is likely due to the inflation of the <em>K<sub>st</sub></em> statistic by the generally low overall level of sequence diversity within this sample <a href="#pone.0000112-Charlesworth5">[33]</a>. Specific comparisons of south (FP) versus north (HI, IR and SB combined) and among northern samples (HI, IR, SB) also exhibit no evidence of subdivision (<a href="#pone-0000112-t003">Table 3</a>). Individual loci also show no evidence of statistically significant differentiation (data not shown). Although these samples show a high level of differentiation for the X-4 fusion arrangement itself (<em>F<sub>st</sub></em> = 0.64), these analyses did not reveal differentiation in the nucleotide variants on the unfused arrangement of Chromosome 4.</p>
<div class="figure" id="pone-0000112-t003"><div class="img"><a name="pone-0000112-t003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000112.t003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000112" data-uri="info:doi/10.1371/journal.pone.0000112.t003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000112.t003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000112.t003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000112.t003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000112.t003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000112.t003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000112.t003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000112.t003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000112.t003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000112.t003.TIF"></span>)
                </a></li></ul></div><p><strong>Table 3.  <span>Sequence differentiation measured by <em>K<sub>st</sub></em>.</span></strong></p><span>doi:10.1371/journal.pone.0000112.t003</span></div>

<h4>Variation Among Loci</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>Patterns of sequence diversity along the unfused arrangement of Chromosome 4 were examined using the six sequenced gene regions. Silent diversity is significantly heterogeneous among the sequenced loci (<em>p</em>&lt;0.01, Freidman ANOVA). However, there is no detectable relationship between sequence diversity and position relative to the centromere (<em>p</em>&gt;0.1, Spearman rank correlation, <a href="#pone-0000112-g002">Fig. 2B</a>). Several factors are potentially responsible for the observed differences in segregating nucleotide variation among gene regions (<em>e.g.</em>, mutation rate, local recombination rate, and codon usage). Dividing by divergence estimates is one method of standardizing diversity measures; however, the ratio of polymorphism to divergence (using <em>D. virilis</em> as an outgroup) exhibits a similar pattern of heterogeneity among loci (<a href="#pone.0000112.s004">Supporting Figure S1</a>).</p>
<a id="article1.body1.sec2.sec3.p2" name="article1.body1.sec2.sec3.p2"></a><p>Relative measures contrasting populations for variance at microsatellite loci have been utilized effectively to identify loci exhibiting population-specific reductions in variance associated with selective sweeps <a href="#pone.0000112-Harr1">[34]</a>. Comparing across loci using a relative measure corrects for locus-specific effects, such as a different mutation rate <a href="#pone.0000112-Schltterer1">[35]</a>. A similar approach was applied to these sequence data using FP to standardize sequence diversity in the northern samples, since FP was selected <em>a priori</em> to represent a population where the low frequency X-4 fusion is not expected to influence sequence diversity on what is essentially an autosome. Using FP to obtain relative measures of sequence diversity for HI, IR and SB, a clear pattern emerges where sequence diversity in northern samples is elevated relative to FP for proximal loci and is equal or lower than FP for distal loci (<a href="#pone-0000112-g002">Fig. 2C</a>). The Spearman rank correlation between standardized diversity and position along the chromosome is highly significant (<em>r<sub>s</sub></em> = −0.78; 95% CI: −0.94, −0.40; <em>p</em>(<em>r<sub>s</sub></em>≥0)&lt;0.001). This result is robust to both the complete removal of <em>Adh</em> from the dataset (<em>r<sub>s</sub></em> = −0.67, 95% CI: −0.85, −0.34) or the complete removal of <em>tim</em> (<em>r<sub>s</sub></em> = −0.67, 95% CI: −0.87, −0.21). This result is also robust to the underlying sampling variance of the diversity estimates, because bootstrap resampling of the individual bases in the sequence data also demonstrates a significant correlation (median <em>r<sub>s</sub></em> = −0.61; 95% CI: −0.87, −0.17). A similar correlation is observed for standardized heterozygosity (<em>θ</em>) using the number of segregating mutations (<a href="#pone.0000112.s005">Supporting Figure S2</a>).</p>
<a id="article1.body1.sec2.sec3.p3" name="article1.body1.sec2.sec3.p3"></a><p>FP was the only sample examined where the frequency of the X-4 fusion is low, so the observed correlation could originate from it having an unusual pattern of sequence diversity along Chromosome 4. Comparison of FP with a previously examined sample from the south [LP from ref. 24] confirms that the finding is not dependent on FP. Silent diversities at <em>Adh, bib</em> and <em>tim</em> in LP relative to FP are 1.12, 0.95 and 0.94, which exhibit the same rank orders as FP in comparisons with the northern populations. These results demonstrate that sequence diversity in northern relative to southern populations exhibits a distinct pattern along the unfused 4<sup>th</sup> chromosome. Specifically, the most proximal locus (<em>Adh</em>) exhibits higher sequence diversity in populations with a high frequency of fused X chromosomes than in populations with a low frequency of fused X chromosomes, whereas the most distal locus (<em>tim</em>) exhibits the inverse relationship. This does not imply that sequence diversity in northern populations is significantly higher than the southern population at <em>Adh</em>, or <em>vice versa</em> for <em>tim</em>, nor does it imply that absolute diversity at proximal loci is greater than distal loci in northern populations. Although the observed differences are small, the overall pattern is extremely robust among samples and across loci.</p>


<h4>Recombination Along Chromosome 4</h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>Measures of linkage relationships between the six loci and the centromere were obtained to quantify the relationship between relative sequence diversity and chromosomal position. The recombination frequency between the centromere and each locus on Chromosome 4 in heterokaryotypic females determines the potential for developing associations between alleles and the sex chromosomes. Genotypes were determined for a total of 336 chromosomes transmitted through females having both a fused and an unfused Chromosome 4. Ten markers representing about 75% of the euchromatic arm of Chromosome 4 were found to encompass a total map distance of 104.6 cM. Centromeric arrangement was determined for 182 chromosomes following their transmission through these heterokaryotypic females, thus providing a direct measure of the recombination frequency for the markers relative to the centromere (<a href="#pone-0000112-t004">Table 4</a>). The <em>fu1</em> locus at the base of the X chromosome was also examined and no recombinants were observed in the interval between this locus and the centromere, whereas a single recombinant was observed at the base of Chromosome 4 between the centromere and <em>Adh</em>. In 154 chromosomes where the arrangement of the centromere was unknown, two additional recombinants between <em>Adh</em> and <em>fu1</em> were identified. Even though the crossover events could not be positioned relative to the centromere, these recombinants provide further indication of a low rate of reciprocal exchange between the centromere and <em>Adh</em>. Recombination frequency relative to the centromere (<em>R</em>) at loci toward the distal end of Chromosome 4 approaches 50% (<a href="#pone-0000112-t004">Table 4</a>), and at locus 3054 and beyond (including the sequenced region <em>tim</em>) insignificant LOD scores for linkage between the markers and the centromere were obtained; therefore statistically, these loci assorted independently of the centromere in this limited sample of chromosomes.</p>
<div class="figure" id="pone-0000112-t004"><div class="img"><a name="pone-0000112-t004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000112.t004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000112" data-uri="info:doi/10.1371/journal.pone.0000112.t004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000112.t004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000112.t004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000112.t004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000112.t004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000112.t004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000112.t004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000112.t004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000112.t004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000112.t004.TIF"></span>)
                </a></li></ul></div><p><strong>Table 4.  <span>Distribution of loci along Chromosome 4 and estimates of recombination with the centromere (<em>R</em>).</span></strong></p><span>doi:10.1371/journal.pone.0000112.t004</span></div><a id="article1.body1.sec2.sec4.p2" name="article1.body1.sec2.sec4.p2"></a><p>Direct measurements of <em>R</em> were obtained for five of the six sequenced regions: <em>Adh</em>, v1-71.20, <em>bib, Gpdh</em>, and <em>tim</em> (<a href="#pone-0000112-t004">Table 4</a>). The v14-60.15 region was included in the sequencing of unfused 4<sup>th</sup> chromosomes but was not included in the linkage analysis due to the absence of a detectable allelic difference between the parental lines. The genome assembly of <em>D. virilis</em>, a species that is closely related to <em>D americana</em> and has the same linear arrangement as unfused Chromosome 4, provides a framework for estimating the local recombination rate per bp using the inferred distances between markers in the assembled sequence. The four sequenced regions (<em>Adh</em>, v17-12.20, v14-60.15 and <em>bib</em>) at the base of Chromosome 4 are located in a single scaffold (GenBank Acc. CH940654.1). Recombination rates in the intervals between mapped markers were used to estimate the local recombinant fraction/bp/generation in heterokaryotypic females at 1.45×10<sup>−8</sup> for this proximal euchromatic region. Interestingly, local recombination rates increase by 3–10 times this level at medial positions along the chromosome (<a href="#pone-0000112-t004">Table 4</a>). The local recombination rate in the region including v14-60.15 and the physical distance were used to obtain an estimate (<em>R</em> = 0.020) for the interval between v14-60.15 and the centromere (<a href="#pone-0000112-t004">Table 4</a>). In summary, the six sequenced regions are well separated along Chromosome 4 and span a range of less than 1% to over 45% recombinants with the centromere.</p>


<h4>Relationship Between Sequence Diversity and Y Linkage</h4>
<a id="article1.body1.sec2.sec5.p1" name="article1.body1.sec2.sec5.p1"></a><p>Presence of the X-4 fusion causes sex-linked transmission of Chromosome 4, yet the unfused X is also present in all the populations studied here. Therefore, heterokaryotypic females occur at appreciable frequencies in these populations and they provide opportunities for exchange between the different chromosomal arrangements. Realization of these opportunities depends on linkage relationships with the centromere (<a href="#pone-0000112-t004">Table 4</a>). For example, assuming Hardy-Weinberg and given a frequency of 97% fused X chromosomes for the IR/SB population, 5.8% of females are heterokaryotypes. With a recombination rate of 0.6% between the centromere and <em>Adh</em>, alleles at this locus are exchanged between fused and unfused chromosomal arrangements approximately once every 2,900 female generations. Both HI and FP experience an exchange rate of about once every 650 female generations for <em>Adh</em> alleles due to similar expected frequencies of heterokaryotypic females. The exchange rate of <em>tim</em> alleles is about 75-fold higher than <em>Adh</em> in all populations. In the presence of the X-4 fusion, these linkage relationships along Chromosome 4 reflect the potential for establishing disequilibria between alleles on the unfused arrangement and the Y.</p>
<a id="article1.body1.sec2.sec5.p2" name="article1.body1.sec2.sec5.p2"></a><p>Sequence diversity was quantified in relation to the gradient of Y linkage that exists along the unfused arrangement of Chromosome 4. A single measure of relative diversity was obtained for each sequenced region by dividing the mean of the pairwise diversities within northern samples (HI, IR, SB) by pairwise diversity within FP. The relationship between relative sequence diversity and the <em>log</em> of the recombinant fraction (<em>R</em>) for each locus was quantified by linear regression. Consistent with the nonparametric analyses based on chromosomal positions, a significant relationship (<em>r</em><sup>2</sup> = 0.89; 95% CI: 0.98, 0.76; <em>p</em>&lt;0.01) is observed between standardized sequence diversity and the rate of meiotic exchange between unfused and fused chromosomes (<a href="#pone-0000112-g003">Fig. 3</a>). Based on the regression line [<em>y</em> = 0.692−0.271(<em>log R</em>)], sequence diversity in northern populations is inflated relative to FP in regions experiencing less than 7.3% recombination with the centromere and reduced relative to FP beyond this point. Although only three data points are available, a similar pattern is apparent along the chromosome when a different southern sample [LP from ref. 24] is used to standardize mean diversity within northern samples (<a href="#pone-0000112-g003">Fig. 3</a>). Because sex-linked transmission of Chromosome 4 is infrequent in the southern populations and frequent in the northern populations, this analysis demonstrates that standardized sequence diversity is highest at loci experiencing the strongest Y linkage and lowest at loci subject to frequent exchange between Y-linked and X-linked contexts.</p>
<div class="figure" id="pone-0000112-g003"><div class="img"><a name="pone-0000112-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000112.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000112" data-uri="info:doi/10.1371/journal.pone.0000112.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000112.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000112.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000112.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000112.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000112.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000112.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000112.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000112.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000112.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Relationship between mean standardized diversity in northern populations and recombination with the centromere.</span></strong></p><a id="article1.body1.sec2.sec5.fig1.caption1.p1" name="article1.body1.sec2.sec5.fig1.caption1.p1"></a><p>Filled circles represent mean standardized diversity within the three northern samples (mean HI, IR, SB/FP) plotted in relation to estimated linkage with the centromere on a <em>log</em> scale. A significant (<em>p</em>&lt;0.01) linear relationship is described by the plotted line, <em>y</em> = 0.692−0.271<em>x</em>, fitted on the <em>log</em> transformed recombinant fraction (<em>R</em>). For comparison, unfilled circles are plotted for mean diversity within the three northern samples standardized using a different southern sample [LP from ref. 24].</p>
<span>doi:10.1371/journal.pone.0000112.g003</span></div><a id="article1.body1.sec2.sec5.p3" name="article1.body1.sec2.sec5.p3"></a><p>The apparent lower diversity at <em>Gpdh</em> and <em>tim</em> for the northern population relative to the southern population may be unrelated to the rate of exchange between neo-Y and neo-X. A previous result indicates that the observed ratio at these medial loci may reflect relative diversities maintained throughout the genome. The one rigorous analysis of sequence diversity in southern and northern populations of <em>D. americana</em> for a gene region (<em>transformer</em>) located in a non-rearranged chromosomal region indicated a lower level of diversity (relative <em>π</em> ~0.95) in the northern population <a href="#pone.0000112-McAllister4">[36]</a>. If this observation reflects a general pattern throughout the genome, where southern populations maintain higher levels of sequence diversity than northern populations, the inverse relationship at the four proximal loci on Chromosome 4 is even more unusual.</p>


<h4>Propensity for Sexual Antagonism</h4>
<a id="article1.body1.sec2.sec6.p1" name="article1.body1.sec2.sec6.p1"></a><p>Hitchhiking with Y-linked sexually antagonistic alleles was investigated as a potential cause of inflated sequence diversity in regions experiencing strong Y linkage. Sexually antagonistic alleles that are advantageous to males but detrimental to females establish polymorphic equilibria when tightly, but not completely, linked with a male-determining locus <a href="#pone.0000112-Rice1">[26]</a>. Transient Y linkage occurs in <em>D. americana</em> due to males with the X-4 fusion transmitting an intact unfused 4<sup>th</sup> chromosome to their sons. Breakdown of linkage disequilibrium between an antagonistic locus on Chromosome 4 and the Y occurs at two levels: 1) the presence of males with unfused X chromosomes, and 2) the exchange between unfused and fused chromosomes. Thus, it is unclear how polymorphism for the X-4 would affect the evolutionary dynamics of antagonistic alleles along the unfused arrangement.</p>
<a id="article1.body1.sec2.sec6.p2" name="article1.body1.sec2.sec6.p2"></a><p>Simulations were conducted to explore the dynamics of a male-benefit/female-detriment allele arising on Chromosome 4 in <em>D. americana</em>. Frequency of the X-4 fusion was held constant under the assumption that selective forces maintaining the latitudinal cline are stronger than selection on antagonistic alleles. Relaxing this assumption results in fixation of the X-4 fusion under certain parameters (data not shown). For a dominant allele with a selective advantage in males of 0.001 and an equal disadvantage in females, <a href="#pone-0000112-g004">Figure 4</a> illustrates the potential to accumulate an antagonistic allele on the unfused arrangement of Chromosome 4 with tight, but incomplete, linkage with the centromere. In populations with greater than 85% of the X chromosomes fused with Chromosome 4 (pertaining to HI, IR &amp; SB), a sexually antagonistic allele increases in frequency and establishes a polymorphic equilibrium when experiencing less than 5% recombination with the centromere (pertaining to <em>Adh</em>, v1-71.20, v14-60.15, and <em>bib</em>). With the X-4 fusion at a frequency of 95% and a recombination rate with the centromere of 0.5% (similar to the estimates of these parameters for <em>Adh</em> in IR and SB), this type of antagonistic allele would be present on approximately half of the unfused 4<sup>th</sup> chromosomes and would have the greatest potential for increasing flanking neutral variation through hitchhiking. Linkage disequilibrium between the antagonistic allele and the alternative centromeric arrangements is 0.073 (D′ = 0.46), so 12% of fused 4<sup>th</sup> chromosomes also carry the allele. In contrast, antagonistic alleles experiencing complete linkage with the centromere increase to fixation on the unfused arrangement in populations having a majority of the X chromosomes fused with Chromosome 4 (<a href="#pone-0000112-g004">Fig. 4</a>). These simulations demonstrate that the segregation pattern of Chromosome 4 in populations with both fused and unfused arrangements provides ample opportunity to establish a gradient for accumulating antagonistic variation. This gradient depends on linkage with the centromere and is strongest in populations having the highest frequencies of the X-4 fusion.</p>
<div class="figure" id="pone-0000112-g004"><div class="img"><a name="pone-0000112-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000112.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000112" data-uri="info:doi/10.1371/journal.pone.0000112.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000112.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000112.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000112.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000112.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000112.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000112.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000112.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000112.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000112.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>Sexually antagonistic alleles on Chromosome 4.</span></strong></p><a id="article1.body1.sec2.sec6.fig1.caption1.p1" name="article1.body1.sec2.sec6.fig1.caption1.p1"></a><p>Equilibrium frequencies of a sexually antagonistic allele (<em>A2</em>) on the unfused arrangement are plotted for populations segregating the X-4 fusion arrangement. Loci representing a variety of different linkage relationships (<em>R</em>) with the centromere are illustrated. Frequency of the X-4 fusion was held constant in the simulations. Results are shown for a completely dominant antagonistic allele with a selection coefficient of 0.001 and fitness effects of opposite sign between males and females. The antagonistic allele increases male fitness and decreases female fitness.</p>
<span>doi:10.1371/journal.pone.0000112.g004</span></div>
</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p>A positive relationship between the level of nucleotide diversity and the rate of recombination is a feature of most eukaryotic genomes. Background selection and/or selective sweep processes have received considerable attention as being the underlying cause of this pattern <a href="#pone.0000112-Wiehe1">[7]</a>, <a href="#pone.0000112-Hudson1">[8]</a>, <a href="#pone.0000112-Charlesworth6">[37]</a>, <a href="#pone.0000112-Stephan1"></a>,<a href="#pone.0000112-Baudry1"></a>,<a href="#pone.0000112-Innan1">[XPATH ERROR: unknown variable "next".]</a>. While previous studies have generally contrasted genomic regions experiencing chronically low levels of recombination with regions of normal recombination, this study utilized an approach where the same gene regions were compared among populations with different predicted recombination rates. The unfused arrangement of Chromosome 4 is sheltered from crossing over in populations of <em>D. americana</em> with a high frequency of the X-4 fusion arrangement, yet the resultant chromosome-wide reduction in recombination rate does not detectably influence observed levels of nucleotide diversity. These results are evaluated in relation to expectations of the background selection and selective sweep models. Furthermore, the observed pattern of increased diversity in the proximal region of Chromosome 4 is considered in relation to evolutionary forces potentially affecting the evolution of sex chromosomes.</p>
<a id="article1.body1.sec3.p2" name="article1.body1.sec3.p2"></a><p>Background selection reduces nucleotide diversity in genomic regions with a low rate of recombination as a byproduct of hitchhiking with deleterious mutations that are eliminated from the population. This process could generate a negative correlation between nucleotide diversity on the unfused arrangement of Chromosome 4 and the frequency of the X-4 fusion, but the pattern was not observed. Two factors potentially impede the establishment of different levels of nucleotide diversity as a consequence of background selection. The high level of exchange observed between unfused and fused arrangements of Chromosome 4 may eliminate the effects of background selection. This assertion is supported by the lack of correspondence between predicted per generation reductions in recombination and sequence-based measures of recombination, which indicates that the presence of the X-4 fusion has not detectably affected long-term recombination rates (<a href="#pone.0000112.s001">Supporting Table S1</a>). Gene flow could also homogenize sequence variation across populations <a href="#pone.0000112-Nordborg2">[41]</a>. Populations examined in this study were selected to represent a high degree of connectivity with the potential for gene flow, and the observed lack of sequence differentiation is consistent with this potential being realized. Other studies also indicate a high rate of gene flow among populations of <em>D. americana</em> <a href="#pone.0000112-McAllister1">[18]</a>, <a href="#pone.0000112-Schfer1">[42]</a>. Given frequent recombination and a high rate of gene flow, the failure to detect reduced diversity among these populations as a consequence of background selection is not surprising.</p>
<a id="article1.body1.sec3.p3" name="article1.body1.sec3.p3"></a><p>Selective sweeps limited to the unfused arrangement are expected to reduce nucleotide variation more severely in northern relative to southern populations of <em>D. americana</em>. This pattern would arise as a consequence of wider windows of reduced neutral variation associated with individual fixation events <a href="#pone.0000112-Wiehe1">[7]</a>, <a href="#pone.0000112-Kim1">[43]</a>. Evidence of widespread effects associated with fixation of alleles was not revealed by the contrast among populations. Homogeneity in levels and patterns of nucleotide diversity could result from high rates of gene flow; however, gene flow does not preclude the possibility of local selective sweeps <a href="#pone.0000112-Endler1">[44]</a>, <a href="#pone.0000112-Lenormand1">[45]</a>. High rates of gene flow occur among populations of <em>D. melanogaster</em>, yet hitchhiking events associated with geographically localized sweeps are evident in its genome <a href="#pone.0000112-Harr1">[34]</a>, <a href="#pone.0000112-Greenberg1">[46]</a>. Geographically localized variants also exist in <em>D. americana</em>; the X-4 fusion arrangement exhibits a strong latitudinal cline <a href="#pone.0000112-McAllister1">[18]</a>, nucleotide variants at the base of the X chromosome are associated with this cline <a href="#pone.0000112-Vieira1">[47]</a>, and overlapping paracentric inversions with associated nucleotide variants are limited to the fused arrangement of Chromosome 4 <a href="#pone.0000112-McAllister3">[24]</a>. Even with the high rate of gene flow that apparently exists, these variants indicate the potential for geographically localized adaptive evolution, and yet, a reduction in nucleotide diversity on the unfused arrangement of Chromosome 4 due to selective sweeps is not apparent.</p>
<a id="article1.body1.sec3.p4" name="article1.body1.sec3.p4"></a><p>While both background selection and selective sweeps are expected to reduce sequence diversity more severely in the proximal region of Chromosome 4 in northern relative to southern populations, this study revealed the exact opposite pattern. Overall, this analysis identified a robust relationship between nucleotide diversity and Y linkage where loci with the greatest potential for establishing allelic associations with the Y chromosome exhibit the highest standardized levels of nucleotide diversity. Hitchhiking with overdominant alleles is known to increase nucleotide diversity <a href="#pone.0000112-Kreitman1">[48]</a>. To account for the observed pattern under such a scenario, however, overdominant alleles must arise as a consequence of Y linkage. Simulations demonstrated the propensity for maintaining an intermediate-frequency sexually antagonistic allele within about 5 cM of the centromere of Chromosome 4. Loci exhibiting polymorphism for such an allele would exhibit overdominant evolutionary dynamics, thus they have the potential to increase neutral variation in flanking regions. Nucleotide diversity along Chromosome 4 may, therefore, be shaped by hitchhiking with sexually antagonistic variation arising as a consequence of Y linkage in the presence of the X-4 centromeric fusion. Other features of the sequence data, namely haplotype structure and frequency spectra, do not exhibit signs of hitchhiking, but it is unclear how strong these effects would be under the ubiquitous antagonism that would be necessary to generate the observed linear relationship between relative diversity and recombination rate with the centromere <a href="#pone.0000112-Navarro1">[49]</a>.</p>
<a id="article1.body1.sec3.p5" name="article1.body1.sec3.p5"></a><p>The focus of this study was identifying patterns of nucleotide diversity on the unfused arrangement of Chromosome 4. Distinct patterns of nucleotide diversity in relation to the frequency of the X-4 fusion are consistent with Y-linked sexual antagonism shaping variation along this chromosome. The X-4 fusion arrangement should also be prone to X-linked sexual antagonism <a href="#pone.0000112-Rice2">[50]</a>, <a href="#pone.0000112-Gibson1">[51]</a>. Nucleotide diversity for X-linked alleles of <em>Adh</em> were previously demonstrated as being higher than Y-linked alleles <a href="#pone.0000112-McAllister2">[23]</a>, and although identification of a cause of the asymmetry between X- and Y-linked alleles is confounded by the presence of a nearby chromosomal inversion limited to the fused arrangement <a href="#pone.0000112-McAllister3">[24]</a>, nucleotide diversity near the centromere of both arrangements of Chromosome 4 is potentially influenced by antagonistic selective forces.</p>
<a id="article1.body1.sec3.p6" name="article1.body1.sec3.p6"></a><p>Chromosome 4 of <em>D. americana</em> embodies the dynamics of an emerging pair of sex chromosomes, with the caveat that recombination between the pair occurs in females. Sexual antagonism is widely considered to be the driving force behind the suppression of recombination between emerging sex chromosomes, even though limited empirical evidence supports this inference <a href="#pone.0000112-Charlesworth1">[17]</a>. Laboratory populations of <em>D. melanogaster</em> evolve antagonistic variation very rapidly under experimentally manipulated sex linkage <a href="#pone.0000112-Rice3">[52]</a>, <a href="#pone.0000112-Rice4">[53]</a>, and they have been shown to harbor a large amount of sexually antagonistic variation on the X <a href="#pone.0000112-Gibson1">[51]</a>. Antagonism at the base of Chromosome 4 of <em>D. americana</em> would favor a reduction in the rate of recombination between fused and unfused arrangements. Two overlapping paracentric inversions are limited to the fused arrangement of Chromosome 4 and these inversions do suppress recombination with the unfused arrangement <a href="#pone.0000112-McAllister3">[24]</a>. For example, <em>Adh</em> and <em>bib</em> were shown here to have 0.6% and 3.3% recombinants respectively with the centromere, but none were observed among a similar number of progeny when the X-4 fusion had the inverted arrangement. Both of these genes are located in the interval between the centromere and proximal inversion breakpoint. The inverted region encompasses a large region of the X-4 fusion and the outermost breakpoints are flanked by <em>bib</em> and v68-4 (see <a href="#pone-0000112-g001">Figure 1</a>). Suppression of recombination is observed throughout the entire inverted region; inversion heterozygotes produced only 6.9% recombinants between the centromere and <em>tim</em> in the previous study compared to the 46.2% observed for the homosequential chromosomes examined here. Identification of a pattern of nucleotide variation consistent with antagonism and the appearance of a recombination suppressor provides two independent indicators that this form of intra-locus sexual conflict is shaping the emerging pair of sex chromosomes in <em>D. americana</em>.</p>
<a id="article1.body1.sec3.p7" name="article1.body1.sec3.p7"></a><p>Complete Y linkage facilitates the sex-specific fixation of male-benefit/female-detriment alleles by limiting the allele to males and resolving intra-locus conflict. On the unfused arrangement of Chromosome 4, this can occur through complete linkage with the centromere or presence in an isolated population with 100% fused X chromosomes. Neither of these scenarios is likely for the loci and populations analyzed here, and the high level of sequence diversity in the proximal region of unfused Chromosome 4 is inconsistent with recent fixation of any alleles, antagonistic or otherwise. This observation contrasts with the reduced level of variability observed for neo-Y chromosomes of <em>D. miranda</em> <a href="#pone.0000112-Yi1">[54]</a>, <a href="#pone.0000112-Bachtrog1">[55]</a>, <a href="#pone.0000112-Bachtrog2">[56]</a>. Low nucleotide variation for the newly Y-linked region of its genome has been attributed to selective sweeps <a href="#pone.0000112-Bachtrog3">[57]</a>. Distinct patterns of nucleotide diversity for the neo-Y chromosome of these two Drosophila species are expected, since the different chromosomal rearrangements generating sex linkage bring about distinct dynamics under sexual antagonism <a href="#pone.0000112-Rice4">[53]</a>. An autosomal arm fused with the ancestral Y to form the neo-Y in <em>D. miranda</em>, and thus since its origin, this rearranged chromosome has been completely Y linked. This rearrangement promotes fixation of male-benefit alleles regardless of their fitness effects when expressed in females (female expression is hypothetical, since the alleles are never transmitted to females). Sweeps of male-benefit alleles would fix neutral variation across the entire neo-Y chromosome <a href="#pone.0000112-Rice5">[58]</a>. This prediction also applies to the Y-limited region of the newly evolved sex chromosome pair, which also exhibits low nucleotide diversity, in the plant genus <em>Silene</em> <a href="#pone.0000112-Filatov1">[14]</a>, <a href="#pone.0000112-Filatov2">[59]</a>. Female fitness matters for alleles on the transient neo-Y of <em>D. americana</em>, and therefore, polymorphism is the expected outcome of intra-locus sexual antagonism. Sequence analysis of the pseudoautosomal regions of the <em>Silene</em> sex chromosomes would potentially reveal a pattern similar to the one reported here, especially given the evidence these regions contain male-limited QTL influencing sexually antagonistic floral traits <a href="#pone.0000112-Scotti1">[60]</a>.</p>
<a id="article1.body1.sec3.p8" name="article1.body1.sec3.p8"></a><p>If sexual antagonism shapes patterns of nucleotide diversity on Chromosome 4 of <em>D. americana</em>, this could be an indicator of the long-term consequences for this chromosomal pair. Sexual antagonism may interfere with local adaptation on the unfused arrangement of Chromosome 4, since there is currently no evidence of local adaptation influencing patterns of nucleotide diversity. Degeneration of gene function is a key feature of evolving Y chromosomes, and the interference generated by sexually antagonistic alleles could contribute to this process <a href="#pone.0000112-Hill1">[4]</a>, <a href="#pone.0000112-Felsenstein1">[5]</a>. Hitchhiking of deleterious mutations with sexually antagonistic alleles is a form of interference previously proposed to facilitate Y-chromosome degeneration <a href="#pone.0000112-Rice5">[58]</a>. Intriguingly, the pattern revealed here has emerged prior to any apparent gene loss from the neo-Y chromosome <a href="#pone.0000112-Charlesworth7">[61]</a>. Direct demonstration of sexually antagonistic genetic variation at the base of Chromosome 4 in northern populations of <em>D. americana</em> would further elucidate its influence on this emerging pair of sex chromosomes.</p>
</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4>Populations Examined</h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p>Sequences were obtained as haplotypes from unfused 4<sup>th</sup> chromosomes derived from four samples of <em>D. americana</em>. The four samples were collected at separate localities in the central United States, and they represent latitudinally distributed populations differing in frequency of the X-4 fusion. Samples of FP and HI were collected in 1999 and described previously <a href="#pone.0000112-McAllister1">[18]</a>. FP was collected near the White River in southeastern Arkansas, and HI was collected near the Missouri River west of St. Louis, Missouri. SB was sampled in September 2002 from the Saulsbury Bridge Recreation Area near the Cedar River, Muscatine County, Iowa (GPS: 41.495°N, 91.161°W). A total of 24 independent X chromosomes were examined using a modification of previously described methods <a href="#pone.0000112-McAllister1">[18]</a>, and all X chromosome were identified as being fused with the 4<sup>th</sup> chromosome. IR was sampled in July 2004 from the Hawkeye Wildlife Area on the Iowa River, Johnson County, Iowa (GPS: 41.779°N, 91.715°W). A total of 117 X chromosomes were examined, and 114 were fused with Chromosome 4.</p>
<a id="article1.body1.sec4.sec1.p2" name="article1.body1.sec4.sec1.p2"></a><p>In addition to the observed data for each sample, the proportion of X chromosomes fused with Chromosome 4 and 95% CI was estimated for each population from a logistic regression model based on geographic coordinates. A total of 641 X chromosomes representing 13 distinct geographic localities in the central United States were included in the model (McAllister et al. unpublished data). Details of the model and the samples upon which it is based will be published separately.</p>
<a id="article1.body1.sec4.sec1.p3" name="article1.body1.sec4.sec1.p3"></a><p>Frequency of the X-4 fusion impacts the occurrence of meiotic recombination involving unfused 4<sup>th</sup> chromosomes. This arises from the restriction of unfused 4<sup>th</sup> chromosomes in males, which lack normal meiotic crossing over. Given <em>p</em>, the proportion of X chromosomes fused with the 4<sup>th</sup> chromosome, the proportion of unfused 4<sup>th</sup> chromosomes in females each generation is 2(1−p)/4−3p. Assuming this proportion is 1/2 in the absence of the X-4 fusion, the expression 4(1−p)/4−3p estimates the reduction in the recombination rate (<em>r</em>) between two loci on the unfused 4 relative to the autosomal condition.</p>


<h4>Sequence Data</h4>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1"></a><p>Sequence diversity was measured at six positions distributed along the 4<sup>th</sup> chromosome in these four samples. Characteristics of the sequenced regions are provided in <a href="#pone.0000112.s002">Table S2</a> (Supporting Data). Single F1 hybrids containing an unfused Chromosome 4 of <em>D. americana</em> and its homolog from <em>D. virilis</em> were used as template for amplification of each locus using <em>D. americana</em>-specific primers. These male hybrids were obtained from crosses of single wild-caught males, or a single male from an iso-female line, to the V46 strain of <em>D. virilis</em>. Presence of an unfused 4<sup>th</sup> chromosome is ensured by only sequencing from an F1 son of a <em>D. americana</em> male (<em>i.e.</em>, the male can have either a fused or unfused X, but the son will always inherit a single unfused 4). Amplified PCR product was sequenced directly using an ABI 3100 or 3730. Traces from forward and reverse reads were trimmed, aligned, and manually edited using Sequencher (Gene Codes, Ann Arbor, MI). An outgroup sequence of each region was obtained from the whole genome assembly of <em>D. virilis</em> (caf1, Agencourt Biosciences, Beverly, MA). The set of sequences was aligned manually, and boundaries between coding and noncoding regions were annotated according to orthologous gene sequences in <em>D. melanogaster</em>, except for the annotation of the coding region in v14-60.15 (see below).</p>
<a id="article1.body1.sec4.sec2.p2" name="article1.body1.sec4.sec2.p2"></a><p>Primers used to amplify and sequence regions of the <em>Alcohol dehydrogenase</em> (<em>Adh</em>), <em>big brain</em> (<em>bib</em>) and <em>timeless</em> (<em>tim</em>) genes have been described previously <a href="#pone.0000112-McAllister2">[23]</a>, <a href="#pone.0000112-McAllister3">[24]</a>. Reported cytological positions of <em>Adh</em> and <em>bib</em> are near the centromere of Chromosome 4 respectively at positions 49B and 48E on the Gubenko and Evgen'ev <a href="#pone.0000112-Gubenko1">[62]</a> photographic map of <em>D. virilis</em>, whereas <em>tim</em> is located at position 42E toward the distal end of the chromosome <a href="#pone.0000112-McAllister1">[18]</a>, <a href="#pone.0000112-Nurminsky1">[63]</a>, <a href="#pone.0000112-Charlesworth7">[61]</a>. Primers specific for the amplification of a region of <em>Glycerol 3 phosphate dehydrogenase</em> (<em>Gpdh</em>) were designed based on the sequences of Tominaga and Narise <a href="#pone.0000112-Tominaga1">[64]</a>. Primer sequences for amplification and initial sequencing are: 5′ tttgccagggtccagttg, and 5′ cttggatatacaaagtggtacgt. Full sequences of both strands were obtained using internal sequencing primers. This gene was localized to cytological band 47B of <em>D. virilis</em> by <em>in situ</em> hybridization following standard procedures <a href="#pone.0000112-Charlesworth7">[61]</a>.</p>
<a id="article1.body1.sec4.sec2.p3" name="article1.body1.sec4.sec2.p3"></a><p>Sequences were also determined for two regions in the interval between <em>Adh</em> and <em>bib</em>. Regions were initially targeted in this interval by obtaining sequence from subclones generated from P1 clones v1-71 mapped to subdivision 49A and v14-60 mapped to subdivision 48F of the <em>D. virilis</em> genome <a href="#pone.0000112-Lozovskaya1">[65]</a>, <a href="#pone.0000112-Vieira2">[66]</a>. Cells' containing the P1 clones of <em>D. virilis</em> genomic DNA were generously provided by Jorge Vieira (Universidade do Porto). Primers developed from the <em>D. virilis</em> sequence were used to amplify and sequence the corresponding region in a set of inbred lines of <em>D. americana</em>, therefore, enabling the development of species-specific primers for amplification and sequencing of <em>D. americana</em> alleles from hybrid flies. Primer sequences for amplification and sequencing of v1-71.20 are: 5′ ttgtagccaccacctcca, and 5′ aYactgtgggcggttattc. Primer sequences for v14-60.15 are: 5′ ttgcggctcggagaacag, and 5′ catcgaggctcggatctt. Sequence of the v1-71.20 region corresponds to a portion of CG17549 of <em>D. melanogaster</em>, and the v14-60.15 region corresponds to a portion of CG5682. Exon/intron boundaries of the sequenced regions were determined by MultiPipMaker alignments <a href="#pone.0000112-Schwartz1">[67]</a> with genomic and cDNA sequences of <em>D. melanogaster</em>. Two intron splice sites surrounding a central exon of CG17549 appear to be conserved between the <em>virilis</em> species group and <em>D. melanogaster</em>; however, an intron annotated in CG5682 of <em>D. melanogaster</em> appears as an insertion relative to the sequences of this gene in <em>D. americana</em> and <em>D. virilis</em>.</p>


<h4>Sequence Analyses</h4>
<a id="article1.body1.sec4.sec3.p1" name="article1.body1.sec4.sec3.p1"></a><p>The primary goal of the study was to test for effects on the unfused arrangement of Chromosome 4 arising as a consequence of the X-4 fusion by examining patterns across populations and among loci. The total dataset consists of approximately 182 kb of sequence distributed evenly among four samples each with ten independently sampled chromosomes. Sequences were derived from six regions distributed along Chromosome 4, and the individual regions range in size from 502 bp to 1,141 bp encompassing a total aligned length of 4,566 bp (details in <a href="#pone.0000112.s002">Supporting Table S2</a>). Analyses of the sequence data were performed using DnaSP v4.10 <a href="#pone.0000112-Rozas1">[68]</a>.</p>
<a id="article1.body1.sec4.sec3.p2" name="article1.body1.sec4.sec3.p2"></a><p>Statistics based on the distribution of variation within samples did not exhibit detectable patterns across populations or loci, and therefore, only Tajima's <em>D</em> <a href="#pone.0000112-Tajima2">[30]</a> calculated from the total number of mutations is reported. Pairwise and overall population differentiation was estimated with the <em>K<sub>st</sub></em> statistic <a href="#pone.0000112-Hudson3">[69]</a>. Significance of observed values of this statistic relative to the expectation of zero was assessed by permutation tests. Estimates of the recombination parameter (<em>4Nr</em>) were obtained directly from the sequences <a href="#pone.0000112-Hudson4">[70]</a>, and the results are presented in <a href="#pone.0000112.s001">Supporting Table S1</a>.</p>
<a id="article1.body1.sec4.sec3.p3" name="article1.body1.sec4.sec3.p3"></a><p>Reported analyses of sequence diversity are based on measures of pairwise differences <a href="#pone.0000112-Tajima3">[71]</a>, but the same general results were obtained for diversity estimates based on numbers of segregating sites, numbers of mutations, or diversity relative to divergence (using <em>D. virilis</em> as the outgroup). Due to heterogeneity in silent (synonymous and noncoding) diversity among genes, total pairwise diversity in the three northern samples was divided by pairwise diversity within the southern sample (FP) to standardize among genes and examine patterns along the chromosome. Tests for homogeneity in pairwise diversity (and other variables) and correlations were performed using SAS ver. 9.1.3 (Cary, NC). Differences in means were evaluated on rank scores by invoking Cochran-Mantel-Haenszel statistics and testing for homogeneity (Friedman 2-way ANOVA) in means <a href="#pone.0000112-Stokes1">[72]</a>. Correlations were measured using PROC CORR, and confidence intervals were obtained with the JACKBOOT macro as percentage estimates of the distributions from 10,000 bootstrap samples of the data <a href="#pone.0000112-Haukoos1">[73]</a>.</p>
<a id="article1.body1.sec4.sec3.p4" name="article1.body1.sec4.sec3.p4"></a><p>A customized routine for bootstrap resampling of the sequence data was compiled using Absoft Pro Fortran 8.0 (Rochester Hills, MI). Individual bases in the sequence data were sampled with replacement, and the Spearman Rank Correlation (<em>r<sub>s</sub></em>) was obtained from the standardized diversities (divided by FP) of the three northern samples (HI, IR, SB) against chromosomal position. The median <em>r<sub>s</sub></em> from the 10,000 replicates reflects the correlation based on the median pairwise diversity and the confidence interval obtained from the distribution of the statistic for the resampled data reflects the underlying variance in the estimate of pairwise diversity. The proportion of replicates where <em>r<sub>s</sub></em> in the resampled data was equal to or greater than zero indicates the probability of this outcome given these sequence data.</p>


<h4>Recombination Estimates</h4>
<a id="article1.body1.sec4.sec4.p1" name="article1.body1.sec4.sec4.p1"></a><p>The recombination rate in heterokaryotypic females with alternative centromeric arrangements of Chromosome 4 was assayed directly from laboratory crosses. Reciprocal crosses were established between inbred lines G96.23 and ML97.5. The former line is fixed for an X-4 centromeric fusion and the later line is fixed for an unfused X chromosome. Both lines have the standard linear arrangement of Chromosome 4 for the <em>virilis</em> species group, which was checked following standard procedures for preparation of polytene chromosomes from the salivary glands of F1 larvae <a href="#pone.0000112-Kennison1">[74]</a>. The same crossing design applied by McAllister <a href="#pone.0000112-McAllister3">[24]</a> was used to generate single flies with X and 4<sup>th</sup> chromosomes transmitted through a single female meiosis. Centromeric arrangement was inferred from segregation of the eye-color mutation at the <em>cardinal</em> (<em>cd</em>) locus, which is on Chromosome 4. The <em>cd</em> locus segregates as an autosomal recessive when the X and 4<sup>th</sup> are unfused and as a sex-linked diploid recessive (<em>i.e.</em>, pseudoautosomal locus in complete linkage with gender) when the X and 4<sup>th</sup> are fused. Additional flies were obtained with X and 4<sup>th</sup> chromosomes transmitted through the same design, but where the centromeric arrangement was not inferred due to the number of progeny being insufficient for identifying a statistical association (<em>p</em>&gt;0.05) between the <em>cd</em> locus and gender. DNA was extracted from flies using the DNeasy 96 tissue kit (Qiagen).</p>
<a id="article1.body1.sec4.sec4.p2" name="article1.body1.sec4.sec4.p2"></a><p>Genotypes of recombinant chromosomes were determined for ten loci on Chromosome 4 and one locus on the X chromosome. Assays were either based on discrimination of parental alleles using restriction site differences in PCR products or size differences at microsatellite loci (<a href="#pone.0000112.s003">Supporting Table S3</a>). The RFLP loci were screened on 1.5% agarose gels. Microsatellite loci were amplified with one primer having 5′ conjugated FAM or HEX and were screened in multiplex runs on an ABI 3100 using Genescan software to analyze the trace output (Applied Biosystems, Foster City, CA). The genotype data were analyzed with Map Manager QTb29ppc <a href="#pone.0000112-Manly1">[75]</a> to estimate intervals between loci using the Kosambi mapping function and to obtain the recombinant fraction (<em>R</em>) relative to the centromere.</p>
<a id="article1.body1.sec4.sec4.p3" name="article1.body1.sec4.sec4.p3"></a><p>A polymorphic marker discriminating the lines was not present in the v14-60.15 sequenced region, so the rate of recombination with the centromere was not directly measured, but inferred from the local recombination rate. The physical distance between the markers was estimated based on the genome assembly of the closely related species <em>Drosophila virilis</em> (caf1, Agencourt Biosciences, Beverly, MA). An approximately 100-bp window including each marker was queried against the assembled genome sequence using <em>blastn</em> <a href="#pone.0000112-Altschul1">[76]</a> to identify the position of the marker locus. The distance between markers was estimated from the size of the intervening sequence and estimated gaps. Two sets of markers were identified in separate scaffolds, and due to the presence of a gap of unknown size between these scaffolds, the interval between adjacent markers (<em>bib</em> and v68-86.1) separated by this gap was not estimated.</p>


<h4>Computer Simulations</h4>
<a id="article1.body1.sec4.sec5.p1" name="article1.body1.sec4.sec5.p1"></a><p>Evolutionary dynamics of sexually antagonistic alleles on Chromosome 4 of <em>D. americana</em> were simulated according to a modified model of Rice <a href="#pone.0000112-Rice1">[26]</a> using code compiled with Absoft Pro Fortran 8.0 (Rochester Hills, MI). The frequency of the X-4 fusion was maintained at a constant value, whereas the frequency of an antagonistic allele (A2) changed deterministically from an initial value of 0.01 according to specified selection coefficients. Equilibrium frequencies of a completely dominant (<em>h</em> = 1.0) A2 allele with equal selection coefficients (0.001) of opposite sign for males and females were obtained for a range of frequencies of the X-4 fusion. A relatively low value of selection was used under the assumption that the selective force maintaining the cline for the X-4 fusion is stronger than selection on the antagonistic allele, because in simulations where the frequency of the fusion is free to change, the antagonistic allele can drive the fusion to fixation (data not shown). Simulations were initiated with the chromosomal arrangements and the antagonistic alleles in linkage equilibrium and were continued until there was no change in the frequency of the A2 allele.</p>

</div>

<div id="section5" class="section"><a id="s5" name="s5" toc="s5" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0000112"></div><a name="pone.0000112.s001" id="pone.0000112.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000112.s001">Table S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1"></a><p class="preSiDOI">Sequence-based estimates of the recombination parameter</p>
<p class="siDoi">doi:10.1371/journal.pone.0000112.s001</p><a id="article1.body1.sec5.supplementary-material1.caption1.p2" name="article1.body1.sec5.supplementary-material1.caption1.p2"></a><p class="postSiDOI">(0.03 MB DOC)</p>
<a name="pone.0000112.s002" id="pone.0000112.s002"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000112.s002">Table S2. </a></strong></p><a id="article1.body1.sec5.supplementary-material2.caption1.p1" name="article1.body1.sec5.supplementary-material2.caption1.p1"></a><p class="preSiDOI">Descriptions of the sequenced regions</p>
<p class="siDoi">doi:10.1371/journal.pone.0000112.s002</p><a id="article1.body1.sec5.supplementary-material2.caption1.p2" name="article1.body1.sec5.supplementary-material2.caption1.p2"></a><p class="postSiDOI">(0.02 MB DOC)</p>
<a name="pone.0000112.s003" id="pone.0000112.s003"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000112.s003">Table S3. </a></strong></p><a id="article1.body1.sec5.supplementary-material3.caption1.p1" name="article1.body1.sec5.supplementary-material3.caption1.p1"></a><p class="preSiDOI">Assay methods for loci used to measure linkage relationships</p>
<p class="siDoi">doi:10.1371/journal.pone.0000112.s003</p><a id="article1.body1.sec5.supplementary-material3.caption1.p2" name="article1.body1.sec5.supplementary-material3.caption1.p2"></a><p class="postSiDOI">(0.03 MB DOC)</p>
<a name="pone.0000112.s004" id="pone.0000112.s004"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000112.s004">Figure S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material4.caption1.p1" name="article1.body1.sec5.supplementary-material4.caption1.p1"></a><p class="preSiDOI">Pairwise sequence diversity in <em>D. americana</em> relative to fixed differences in comparison with <em>D. virilis</em>. Numbers of fixed differences at each locus were relative to the entire set of 40 sequences from <em>D. americana</em>. Loci are arranged according to relative position along Chromosome 4 with the centromere toward the left and telomere toward the right.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000112.s004</p><a id="article1.body1.sec5.supplementary-material4.caption1.p2" name="article1.body1.sec5.supplementary-material4.caption1.p2"></a><p class="postSiDOI">(0.05 MB TIF)</p>
<a name="pone.0000112.s005" id="pone.0000112.s005"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000112.s005">Figure S2. </a></strong></p><a id="article1.body1.sec5.supplementary-material5.caption1.p1" name="article1.body1.sec5.supplementary-material5.caption1.p1"></a><p class="preSiDOI">Standardized estimate of heterozygosity (θ) measured from the number of mutations. Observed θ in each northern sample is standardized using θ for the southern FP sample. A significant Spearman rank correlation is obtained relative to position on Chromosome 4 (r<sub>s</sub> = −0.78, 95% CI: −0.91, −0.57).</p>
<p class="siDoi">doi:10.1371/journal.pone.0000112.s005</p><a id="article1.body1.sec5.supplementary-material5.caption1.p2" name="article1.body1.sec5.supplementary-material5.caption1.p2"></a><p class="postSiDOI">(0.04 MB TIF)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>We thank M. Kirkpatrick for comments on a previous version of the manuscript and for identifying an error in the initial calculations of the expected reduction in recombination. E. Mancl, L. Pigatti, S. Sheeley, P. Mena, and L. Hoekstra provided technical assistance. J. Comeron, S. Flores, and L. Hadany participated in numerous discussions and provided suggestions on data analyses. J. Vieira provided clones with mapped positions in the proximal region of Chromosome 4. D. Smith of Agencourt Biosciences provided access to the assembled genome sequence of <em>D. virilis</em>.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: BM. Performed the experiments: AE. Analyzed the data: BM. Contributed reagents/materials/analysis tools: BM. Wrote the paper: BM.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0000112-Begun1" id="pone.0000112-Begun1"></a>Begun DJ, Aquadro CF (1992) Levels of naturally occurring DNA polymorphism correlate with recombination rates in <em>D. melanogaster</em>. Nature  356: 519–520.  <ul class="find" data-citedArticleID="5292923" data-doi="10.1038/356519a0"><li><a href="http://dx.doi.org/10.1038/356519a0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Begun%5Bauthor%5D+AND+Levels+of+naturally+occurring+DNA+polymorphism+correlate+with+recombination+rates+in+D.+melanogaster." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3ABegun+%22Levels+of+naturally+occurring+DNA+polymorphism+correlate+with+recombination+rates+in+D.+melanogaster.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0000112-Nachman1" id="pone.0000112-Nachman1"></a>Nachman MW (2001) Single nucleotide polymorphism and recombination rate in humans. Trends Genet  17: 481–485.  <ul class="find" data-citedArticleID="5292925" data-doi="10.1016/s0168-9525(01)02409-x"><li><a href="http://dx.doi.org/10.1016/s0168-9525(01)02409-x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Nachman%5Bauthor%5D+AND+Single+nucleotide+polymorphism+and+recombination+rate+in+humans." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3ANachman+%22Single+nucleotide+polymorphism+and+recombination+rate+in+humans.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0000112-Cutter1" id="pone.0000112-Cutter1"></a>Cutter AD, Payseur BA (2003) Selection at linked sites in the partial selfer <em>Caenorhabditis elegans</em>. Mol Biol Evol  20: 665–673.  <ul class="find" data-citedArticleID="5292927" data-doi="10.1093/molbev/msg072"><li><a href="http://dx.doi.org/10.1093/molbev/msg072" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Cutter%5Bauthor%5D+AND+Selection+at+linked+sites+in+the+partial+selfer+Caenorhabditis+elegans." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3ACutter+%22Selection+at+linked+sites+in+the+partial+selfer+Caenorhabditis+elegans.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0000112-Hill1" id="pone.0000112-Hill1"></a>Hill WG, Robertson A (1966) The effect of linkage on limits to artificial selection. Genet Res  8: 269–294.  <ul class="find" data-citedArticleID="5292929" data-doi="10.1017/s0016672300010156"><li><a href="http://dx.doi.org/10.1017/s0016672300010156" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Hill%5Bauthor%5D+AND+The+effect+of+linkage+on+limits+to+artificial+selection." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AHill+%22The+effect+of+linkage+on+limits+to+artificial+selection.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0000112-Felsenstein1" id="pone.0000112-Felsenstein1"></a>Felsenstein J (1974) The evolutionary advantage of recombination. Genetics  78: 737–756.  <ul class="find" data-citedArticleID="5292931"><li><a href="http://www.crossref.org/guestquery/?auth2=Felsenstein&amp;atitle2=The+evolutionary+advantage+of+recombination.&amp;auth=Felsenstein&amp;atitle=The+evolutionary+advantage+of+recombination." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Felsenstein%5Bauthor%5D+AND+The+evolutionary+advantage+of+recombination." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AFelsenstein+%22The+evolutionary+advantage+of+recombination.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0000112-MaynardSmith1" id="pone.0000112-MaynardSmith1"></a>Maynard Smith J, Haigh J (1974) The hitchhiking effect of a favourable gene. Genet Res  23: 23–35.  <ul class="find" data-citedArticleID="5292933" data-doi="10.1017/s0016672300014634"><li><a href="http://dx.doi.org/10.1017/s0016672300014634" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Maynard+Smith%5Bauthor%5D+AND+The+hitchhiking+effect+of+a+favourable+gene." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AMaynard+Smith+%22The+hitchhiking+effect+of+a+favourable+gene.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0000112-Wiehe1" id="pone.0000112-Wiehe1"></a>Wiehe THE, Stephan W (1993) Analysis of a genetic hitchhiking model, and its application to DNA polymorphism data from <em>Drosophila melanogaster</em>. Mol Biol Evol  10: 842–854.  <ul class="find" data-citedArticleID="5292935"><li><a href="http://www.crossref.org/guestquery/?auth2=Wiehe&amp;atitle2=Analysis+of+a+genetic+hitchhiking+model%2C+and+its+application+to+DNA+polymorphism+data+from+Drosophila+melanogaster.&amp;auth=Wiehe&amp;atitle=Analysis+of+a+genetic+hitchhiking+model%2C+and+its+application+to+DNA+polymorphism+data+from+Drosophila+melanogaster." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Wiehe%5Bauthor%5D+AND+Analysis+of+a+genetic+hitchhiking+model%2C+and+its+application+to+DNA+polymorphism+data+from+Drosophila+melanogaster." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AWiehe+%22Analysis+of+a+genetic+hitchhiking+model%2C+and+its+application+to+DNA+polymorphism+data+from+Drosophila+melanogaster.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0000112-Hudson1" id="pone.0000112-Hudson1"></a>Hudson RR, Kaplan NL (1995) Deleterious background selection with recombination. Genetics  141: 1605–1617.  <ul class="find" data-citedArticleID="5292937"><li><a href="http://www.crossref.org/guestquery/?auth2=Hudson&amp;atitle2=Deleterious+background+selection+with+recombination.&amp;auth=Hudson&amp;atitle=Deleterious+background+selection+with+recombination." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Hudson%5Bauthor%5D+AND+Deleterious+background+selection+with+recombination." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AHudson+%22Deleterious+background+selection+with+recombination.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0000112-Nordborg1" id="pone.0000112-Nordborg1"></a>Nordborg M, Charlesworth B, Charlesworth D (1996) The effect of recombination on background selection. Genet Res  67: 159–174.  <ul class="find" data-citedArticleID="5292939" data-doi="10.1017/s0016672300033619"><li><a href="http://dx.doi.org/10.1017/s0016672300033619" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Nordborg%5Bauthor%5D+AND+The+effect+of+recombination+on+background+selection." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3ANordborg+%22The+effect+of+recombination+on+background+selection.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0000112-Jensen1" id="pone.0000112-Jensen1"></a>Jensen MA, Charlesworth B, Kreitman M (2002) Patterns of genetic variation at a Chromosome 4 locus of <em>Drosophila melanogaster</em> and <em>D. simulans</em>. Genetics  160: 493–507.  <ul class="find" data-citedArticleID="5292941"><li><a href="http://www.crossref.org/guestquery/?auth2=Jensen&amp;atitle2=Patterns+of+genetic+variation+at+a+Chromosome+4+locus+of+Drosophila+melanogaster+and+D.+simulans.&amp;auth=Jensen&amp;atitle=Patterns+of+genetic+variation+at+a+Chromosome+4+locus+of+Drosophila+melanogaster+and+D.+simulans." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Jensen%5Bauthor%5D+AND+Patterns+of+genetic+variation+at+a+Chromosome+4+locus+of+Drosophila+melanogaster+and+D.+simulans." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AJensen+%22Patterns+of+genetic+variation+at+a+Chromosome+4+locus+of+Drosophila+melanogaster+and+D.+simulans.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0000112-Wang1" id="pone.0000112-Wang1"></a>Wang W, Thornton K, Berry A, Long M (2002) Nucleotide variation along the <em>Drosophila melanogaster</em> fourth chromosome. Science  295: 134–137.  <ul class="find" data-citedArticleID="5292943" data-doi="10.1126/science.1064521"><li><a href="http://dx.doi.org/10.1126/science.1064521" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Wang%5Bauthor%5D+AND+Nucleotide+variation+along+the+Drosophila+melanogaster+fourth+chromosome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AWang+%22Nucleotide+variation+along+the+Drosophila+melanogaster+fourth+chromosome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0000112-Wang2" id="pone.0000112-Wang2"></a>Wang W, Tornton K, Emerson JJ, Long M (2004) Nucleotide variation and recombination along the fourth chromosome in <em>Drosophila simulans</em>. Genetics  166: 1783–1794.  <ul class="find" data-citedArticleID="5292945" data-doi="10.1534/genetics.166.4.1783"><li><a href="http://dx.doi.org/10.1534/genetics.166.4.1783" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Wang%5Bauthor%5D+AND+Nucleotide+variation+and+recombination+along+the+fourth+chromosome+in+Drosophila+simulans." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AWang+%22Nucleotide+variation+and+recombination+along+the+fourth+chromosome+in+Drosophila+simulans.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0000112-Zurovcova1" id="pone.0000112-Zurovcova1"></a>Zurovcova M, Eanes WF (1999) Lack of nucleotide polymorphism in the Y-linked sperm flagellar dynein gene <em>Dhc-Yh3</em> of <em>Drosophila melanogaster</em> and <em>D. simulans</em>. Genetics  153: 1709–1715.  <ul class="find" data-citedArticleID="5292947"><li><a href="http://www.crossref.org/guestquery/?auth2=Zurovcova&amp;atitle2=Lack+of+nucleotide+polymorphism+in+the+Y-linked+sperm+flagellar+dynein+gene+Dhc-Yh3+of+Drosophila+melanogaster+and+D.+simulans.&amp;auth=Zurovcova&amp;atitle=Lack+of+nucleotide+polymorphism+in+the+Y-linked+sperm+flagellar+dynein+gene+Dhc-Yh3+of+Drosophila+melanogaster+and+D.+simulans." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Zurovcova%5Bauthor%5D+AND+Lack+of+nucleotide+polymorphism+in+the+Y-linked+sperm+flagellar+dynein+gene+Dhc-Yh3+of+Drosophila+melanogaster+and+D.+simulans." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AZurovcova+%22Lack+of+nucleotide+polymorphism+in+the+Y-linked+sperm+flagellar+dynein+gene+Dhc-Yh3+of+Drosophila+melanogaster+and+D.+simulans.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0000112-Filatov1" id="pone.0000112-Filatov1"></a>Filatov DA, Moneger F, Negrutiu I, Charlesworth D (2000) Low variability in a Y-linked plant gene and its implications for Y-chromosome evolution. Nature  404: 388–390.  <ul class="find" data-citedArticleID="5292949" data-doi="10.1038/35006057"><li><a href="http://dx.doi.org/10.1038/35006057" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Filatov%5Bauthor%5D+AND+Low+variability+in+a+Y-linked+plant+gene+and+its+implications+for+Y-chromosome+evolution." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AFilatov+%22Low+variability+in+a+Y-linked+plant+gene+and+its+implications+for+Y-chromosome+evolution.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0000112-Hellborg1" id="pone.0000112-Hellborg1"></a>Hellborg L, Ellegren H (2004) Low levels of nucleotide diversity in mammalian Y chromosomes. Mol Biol Evol  21: 158–163.  <ul class="find" data-citedArticleID="5292951" data-doi="10.1093/molbev/msh008"><li><a href="http://dx.doi.org/10.1093/molbev/msh008" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Hellborg%5Bauthor%5D+AND+Low+levels+of+nucleotide+diversity+in+mammalian+Y+chromosomes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AHellborg+%22Low+levels+of+nucleotide+diversity+in+mammalian+Y+chromosomes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0000112-Berlin1" id="pone.0000112-Berlin1"></a>Berlin S, Ellegren H (2004) Chicken W: A genetically uniform chromosome in a highly variable genome. Proc Natl Acad Sci  101: 15967–15969.  <ul class="find" data-citedArticleID="5292953" data-doi="10.1073/pnas.0405126101"><li><a href="http://dx.doi.org/10.1073/pnas.0405126101" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Berlin%5Bauthor%5D+AND+Chicken+W%3A+A+genetically+uniform+chromosome+in+a+highly+variable+genome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3ABerlin+%22Chicken+W%3A+A+genetically+uniform+chromosome+in+a+highly+variable+genome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0000112-Charlesworth1" id="pone.0000112-Charlesworth1"></a>Charlesworth D, Charlesworth B, Marais G (2005) Steps in the evolution of heteromorphic sex chromosomes. Heredity  95: 118–128.  <ul class="find" data-citedArticleID="5292955" data-doi="10.1038/sj.hdy.6800697"><li><a href="http://dx.doi.org/10.1038/sj.hdy.6800697" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Charlesworth%5Bauthor%5D+AND+Steps+in+the+evolution+of+heteromorphic+sex+chromosomes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3ACharlesworth+%22Steps+in+the+evolution+of+heteromorphic+sex+chromosomes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0000112-McAllister1" id="pone.0000112-McAllister1"></a>McAllister BF (2002) Chromosomal and allelic variation in <em>Drosophila americana</em>: selective maintenance of a chromosomal cline. Genome  45: 13–21.  <ul class="find" data-citedArticleID="5292957" data-doi="10.1139/g01-112"><li><a href="http://dx.doi.org/10.1139/g01-112" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=McAllister%5Bauthor%5D+AND+Chromosomal+and+allelic+variation+in+Drosophila+americana%3A+selective+maintenance+of+a+chromosomal+cline." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AMcAllister+%22Chromosomal+and+allelic+variation+in+Drosophila+americana%3A+selective+maintenance+of+a+chromosomal+cline.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0000112-Kikkawa1" id="pone.0000112-Kikkawa1"></a>Kikkawa H (1935) Crossing-over in the male of <em>Drosophila virilis</em>. Cytologia  6: 190–194.  <ul class="find" data-citedArticleID="5292959" data-doi="10.1508/cytologia.6.190"><li><a href="http://dx.doi.org/10.1508/cytologia.6.190" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Kikkawa%5Bauthor%5D+AND+Crossing-over+in+the+male+of+Drosophila+virilis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AKikkawa+%22Crossing-over+in+the+male+of+Drosophila+virilis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0000112-Tajima1" id="pone.0000112-Tajima1"></a>Tajima F (1989) The effect of change in population size on DNA polymorphism. Genetics  123: 597–601.  <ul class="find" data-citedArticleID="5292961"><li><a href="http://www.crossref.org/guestquery/?auth2=Tajima&amp;atitle2=The+effect+of+change+in+population+size+on+DNA+polymorphism.&amp;auth=Tajima&amp;atitle=The+effect+of+change+in+population+size+on+DNA+polymorphism." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Tajima%5Bauthor%5D+AND+The+effect+of+change+in+population+size+on+DNA+polymorphism." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3ATajima+%22The+effect+of+change+in+population+size+on+DNA+polymorphism.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0000112-Charlesworth2" id="pone.0000112-Charlesworth2"></a>Charlesworth B, Morgan MT, Charlesworth D (1993) The effect of deleterious mutations on neutral molecular variation. Genetics  134: 1289–1303.  <ul class="find" data-citedArticleID="5292963"><li><a href="http://www.crossref.org/guestquery/?auth2=Charlesworth&amp;atitle2=The+effect+of+deleterious+mutations+on+neutral+molecular+variation.&amp;auth=Charlesworth&amp;atitle=The+effect+of+deleterious+mutations+on+neutral+molecular+variation." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Charlesworth%5Bauthor%5D+AND+The+effect+of+deleterious+mutations+on+neutral+molecular+variation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3ACharlesworth+%22The+effect+of+deleterious+mutations+on+neutral+molecular+variation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0000112-Charlesworth3" id="pone.0000112-Charlesworth3"></a>Charlesworth B, Charlesworth D (2000) The degeneration of Y chromosomes. Phil Trans R Soc Lond B  355: 1563–1572.  <ul class="find" data-citedArticleID="5292965" data-doi="10.1098/rstb.2000.0717"><li><a href="http://dx.doi.org/10.1098/rstb.2000.0717" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Charlesworth%5Bauthor%5D+AND+The+degeneration+of+Y+chromosomes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3ACharlesworth+%22The+degeneration+of+Y+chromosomes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0000112-McAllister2" id="pone.0000112-McAllister2"></a>McAllister BF, Charlesworth B (1999) Reduced sequence variability on the neo-Y chromosome of <em>Drosophila americana americana</em>. Genetics  153: 221–233.  <ul class="find" data-citedArticleID="5292967"><li><a href="http://www.crossref.org/guestquery/?auth2=McAllister&amp;atitle2=Reduced+sequence+variability+on+the+neo-Y+chromosome+of+Drosophila+americana+americana.&amp;auth=McAllister&amp;atitle=Reduced+sequence+variability+on+the+neo-Y+chromosome+of+Drosophila+americana+americana." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=McAllister%5Bauthor%5D+AND+Reduced+sequence+variability+on+the+neo-Y+chromosome+of+Drosophila+americana+americana." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AMcAllister+%22Reduced+sequence+variability+on+the+neo-Y+chromosome+of+Drosophila+americana+americana.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0000112-McAllister3" id="pone.0000112-McAllister3"></a>McAllister BF (2003) Sequence differentiation associated with an inversion on the neo-X chromosome of <em>Drosophila americana</em>. Genetics  165: 1317–1328.  <ul class="find" data-citedArticleID="5292969"><li><a href="http://www.crossref.org/guestquery/?auth2=McAllister&amp;atitle2=Sequence+differentiation+associated+with+an+inversion+on+the+neo-X+chromosome+of+Drosophila+americana.&amp;auth=McAllister&amp;atitle=Sequence+differentiation+associated+with+an+inversion+on+the+neo-X+chromosome+of+Drosophila+americana." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=McAllister%5Bauthor%5D+AND+Sequence+differentiation+associated+with+an+inversion+on+the+neo-X+chromosome+of+Drosophila+americana." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AMcAllister+%22Sequence+differentiation+associated+with+an+inversion+on+the+neo-X+chromosome+of+Drosophila+americana.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0000112-Blight1" id="pone.0000112-Blight1"></a>Blight WC (1955) A cytological study of linear populations of Drosophila americana near St. Louis, Missouri. Washington University.   PhD dissertation,.  <ul class="find-nolinks"></ul></li><li><span class="label">26.
              </span><a name="pone.0000112-Rice1" id="pone.0000112-Rice1"></a>Rice WR (1987) The accumulation of sexually antagonistic genes as a selective agent promoting the evolution of reduced recombination between primitive sex chromosomes. Evolution  41: 911–914.  <ul class="find" data-citedArticleID="5292973" data-doi="10.2307/2408899"><li><a href="http://dx.doi.org/10.2307/2408899" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Rice%5Bauthor%5D+AND+The+accumulation+of+sexually+antagonistic+genes+as+a+selective+agent+promoting+the+evolution+of+reduced+recombination+between+primitive+sex+chromosomes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3ARice+%22The+accumulation+of+sexually+antagonistic+genes+as+a+selective+agent+promoting+the+evolution+of+reduced+recombination+between+primitive+sex+chromosomes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0000112-Hudson2" id="pone.0000112-Hudson2"></a>Hudson RR, Kaplan NL (1988) The coalescent process in models with selection and recombination. Genetics  120: 831–840.  <ul class="find" data-citedArticleID="5292975"><li><a href="http://www.crossref.org/guestquery/?auth2=Hudson&amp;atitle2=The+coalescent+process+in+models+with+selection+and+recombination.&amp;auth=Hudson&amp;atitle=The+coalescent+process+in+models+with+selection+and+recombination." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Hudson%5Bauthor%5D+AND+The+coalescent+process+in+models+with+selection+and+recombination." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AHudson+%22The+coalescent+process+in+models+with+selection+and+recombination.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0000112-Barton1" id="pone.0000112-Barton1"></a>Barton N, Navarro A (2002) Extending the coalescent to multilocus systems: the case of balancing selection. Genet Res  79: 129–139.  <ul class="find" data-citedArticleID="5292977" data-doi="10.1017/s0016672301005493"><li><a href="http://dx.doi.org/10.1017/s0016672301005493" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Barton%5Bauthor%5D+AND+Extending+the+coalescent+to+multilocus+systems%3A+the+case+of+balancing+selection." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3ABarton+%22Extending+the+coalescent+to+multilocus+systems%3A+the+case+of+balancing+selection.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0000112-Charlesworth4" id="pone.0000112-Charlesworth4"></a>Charlesworth , D  (2006) Balancing selection and its effects on sequences in nearby genome regions. PLoS Genet  2: e64.  <ul class="find" data-citedArticleID="5292979" data-doi="10.1371/journal.pgen.0020064"><li><a href="http://dx.doi.org/10.1371/journal.pgen.0020064" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Charlesworth%5Bauthor%5D+AND+Balancing+selection+and+its+effects+on+sequences+in+nearby+genome+regions." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3ACharlesworth+%22Balancing+selection+and+its+effects+on+sequences+in+nearby+genome+regions.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0000112-Tajima2" id="pone.0000112-Tajima2"></a>Tajima F (1989) Statistical method for testing the neutral mutation hypothesis by DNA polymorphism. Genetics  123: 585–595.  <ul class="find" data-citedArticleID="5292981"><li><a href="http://www.crossref.org/guestquery/?auth2=Tajima&amp;atitle2=Statistical+method+for+testing+the+neutral+mutation+hypothesis+by+DNA+polymorphism.&amp;auth=Tajima&amp;atitle=Statistical+method+for+testing+the+neutral+mutation+hypothesis+by+DNA+polymorphism." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Tajima%5Bauthor%5D+AND+Statistical+method+for+testing+the+neutral+mutation+hypothesis+by+DNA+polymorphism." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3ATajima+%22Statistical+method+for+testing+the+neutral+mutation+hypothesis+by+DNA+polymorphism.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0000112-Fu1" id="pone.0000112-Fu1"></a>Fu Y-X, Li W-H (1993) Statistical tests of neutrality. Genetics  133: 693–709.  <ul class="find" data-citedArticleID="5292983"><li><a href="http://www.crossref.org/guestquery/?auth2=Fu&amp;atitle2=Statistical+tests+of+neutrality.&amp;auth=Fu&amp;atitle=Statistical+tests+of+neutrality." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Fu%5Bauthor%5D+AND+Statistical+tests+of+neutrality." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AFu+%22Statistical+tests+of+neutrality.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0000112-Fay1" id="pone.0000112-Fay1"></a>Fay JC, Wu C-I (2000) Hitchhiking under positive Darwinian selection. Genetics  155: 1405–1413.  <ul class="find" data-citedArticleID="5292985"><li><a href="http://www.crossref.org/guestquery/?auth2=Fay&amp;atitle2=Hitchhiking+under+positive+Darwinian+selection.&amp;auth=Fay&amp;atitle=Hitchhiking+under+positive+Darwinian+selection." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Fay%5Bauthor%5D+AND+Hitchhiking+under+positive+Darwinian+selection." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AFay+%22Hitchhiking+under+positive+Darwinian+selection.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0000112-Charlesworth5" id="pone.0000112-Charlesworth5"></a>Charlesworth B (1998) Measures of divergence between populations and the effect of forces that reduce variability. Mol Biol Evol  15: 538–543.  <ul class="find" data-citedArticleID="5292987" data-doi="10.1093/oxfordjournals.molbev.a025953"><li><a href="http://dx.doi.org/10.1093/oxfordjournals.molbev.a025953" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Charlesworth%5Bauthor%5D+AND+Measures+of+divergence+between+populations+and+the+effect+of+forces+that+reduce+variability." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3ACharlesworth+%22Measures+of+divergence+between+populations+and+the+effect+of+forces+that+reduce+variability.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0000112-Harr1" id="pone.0000112-Harr1"></a>Harr B, Kauer M, Schlötterer C (2002) Hitchhiking mapping : A population-based fine-mapping strategy for adaptive mutations in <em>Drosophila melanogaster</em>. Proc Natl Acad Sci USA  99: 12949–12954.  <ul class="find" data-citedArticleID="5292989" data-doi="10.1073/pnas.202336899"><li><a href="http://dx.doi.org/10.1073/pnas.202336899" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Harr%5Bauthor%5D+AND+Hitchhiking+mapping+%3A+A+population-based+fine-mapping+strategy+for+adaptive+mutations+in+Drosophila+melanogaster." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AHarr+%22Hitchhiking+mapping+%3A+A+population-based+fine-mapping+strategy+for+adaptive+mutations+in+Drosophila+melanogaster.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0000112-Schltterer1" id="pone.0000112-Schltterer1"></a>Schlötterer C (2002) A microsatellite-based multilocus screen for the identification of local selective sweeps. Genetics  160: 753–763.  <ul class="find" data-citedArticleID="5292991"><li><a href="http://www.crossref.org/guestquery/?auth2=Schl%F6tterer&amp;atitle2=A+microsatellite-based+multilocus+screen+for+the+identification+of+local+selective+sweeps.&amp;auth=Schl%F6tterer&amp;atitle=A+microsatellite-based+multilocus+screen+for+the+identification+of+local+selective+sweeps." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Schl%C3%B6tterer%5Bauthor%5D+AND+A+microsatellite-based+multilocus+screen+for+the+identification+of+local+selective+sweeps." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3ASchl%C3%B6tterer+%22A+microsatellite-based+multilocus+screen+for+the+identification+of+local+selective+sweeps.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">36.
              </span><a name="pone.0000112-McAllister4" id="pone.0000112-McAllister4"></a>McAllister BF, McVean GAT (2000) Neutral evolution of the sex-determining gene <em>transformer</em> in Drosophila. Genetics  154: 1711–1720.  <ul class="find" data-citedArticleID="5292993"><li><a href="http://www.crossref.org/guestquery/?auth2=McAllister&amp;atitle2=Neutral+evolution+of+the+sex-determining+gene+transformer+in+Drosophila.&amp;auth=McAllister&amp;atitle=Neutral+evolution+of+the+sex-determining+gene+transformer+in+Drosophila." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=McAllister%5Bauthor%5D+AND+Neutral+evolution+of+the+sex-determining+gene+transformer+in+Drosophila." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AMcAllister+%22Neutral+evolution+of+the+sex-determining+gene+transformer+in+Drosophila.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">37.
              </span><a name="pone.0000112-Charlesworth6" id="pone.0000112-Charlesworth6"></a>Charlesworth B (1996) Background selection and patterns of genetic diversity in <em>Drosophila melanogaster</em>. Genet Res  68: 131–149.  <ul class="find" data-citedArticleID="5292995" data-doi="10.1017/s0016672300034029"><li><a href="http://dx.doi.org/10.1017/s0016672300034029" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Charlesworth%5Bauthor%5D+AND+Background+selection+and+patterns+of+genetic+diversity+in+Drosophila+melanogaster." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3ACharlesworth+%22Background+selection+and+patterns+of+genetic+diversity+in+Drosophila+melanogaster.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">38.
              </span><a name="pone.0000112-Stephan1" id="pone.0000112-Stephan1"></a>Stephan W, Xing L, Kirby DA, Braverman JM (1998) A test of the background selection hypothesis based on nucleotide data from <em>Drosophila ananassae</em>. Proc Natl Acad Sci USA  95: 5649–5654.  <ul class="find" data-citedArticleID="5292997" data-doi="10.1073/pnas.95.10.5649"><li><a href="http://dx.doi.org/10.1073/pnas.95.10.5649" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Stephan%5Bauthor%5D+AND+A+test+of+the+background+selection+hypothesis+based+on+nucleotide+data+from+Drosophila+ananassae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AStephan+%22A+test+of+the+background+selection+hypothesis+based+on+nucleotide+data+from+Drosophila+ananassae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">39.
              </span><a name="pone.0000112-Baudry1" id="pone.0000112-Baudry1"></a>Baudry E, Kerdelhue C, Innan H, Stephan W (2001) Species and recombination effects on DNA variability in the tomato genus. Genetics  158: 1725–1735.  <ul class="find" data-citedArticleID="5292999"><li><a href="http://www.crossref.org/guestquery/?auth2=Baudry&amp;atitle2=Species+and+recombination+effects+on+DNA+variability+in+the+tomato+genus.&amp;auth=Baudry&amp;atitle=Species+and+recombination+effects+on+DNA+variability+in+the+tomato+genus." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Baudry%5Bauthor%5D+AND+Species+and+recombination+effects+on+DNA+variability+in+the+tomato+genus." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3ABaudry+%22Species+and+recombination+effects+on+DNA+variability+in+the+tomato+genus.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">40.
              </span><a name="pone.0000112-Innan1" id="pone.0000112-Innan1"></a>Innan H, Stephan W (2003) Distinguishing the hitchhiking and background selection models. Genetics  165: 2307–2312.  <ul class="find" data-citedArticleID="5293001"><li><a href="http://www.crossref.org/guestquery/?auth2=Innan&amp;atitle2=Distinguishing+the+hitchhiking+and+background+selection+models.&amp;auth=Innan&amp;atitle=Distinguishing+the+hitchhiking+and+background+selection+models." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Innan%5Bauthor%5D+AND+Distinguishing+the+hitchhiking+and+background+selection+models." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AInnan+%22Distinguishing+the+hitchhiking+and+background+selection+models.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">41.
              </span><a name="pone.0000112-Nordborg2" id="pone.0000112-Nordborg2"></a>Nordborg M (1997) Structured coalescent processes on different time scales. Genetics  146: 1501–1514.  <ul class="find" data-citedArticleID="5293003"><li><a href="http://www.crossref.org/guestquery/?auth2=Nordborg&amp;atitle2=Structured+coalescent+processes+on+different+time+scales.&amp;auth=Nordborg&amp;atitle=Structured+coalescent+processes+on+different+time+scales." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Nordborg%5Bauthor%5D+AND+Structured+coalescent+processes+on+different+time+scales." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3ANordborg+%22Structured+coalescent+processes+on+different+time+scales.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">42.
              </span><a name="pone.0000112-Schfer1" id="pone.0000112-Schfer1"></a>Schäfer MA, Orsini L, McAllister BF, Schlötterer  (2006) Patterns of microsatellite variation through a transition zone of a chromosomal cline in <em>Drosophila americana</em>. Heredity  97: 291–295.  <ul class="find" data-citedArticleID="5293005" data-doi="10.1038/sj.hdy.6800860"><li><a href="http://dx.doi.org/10.1038/sj.hdy.6800860" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Sch%C3%A4fer%5Bauthor%5D+AND+Patterns+of+microsatellite+variation+through+a+transition+zone+of+a+chromosomal+cline+in+Drosophila+americana." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3ASch%C3%A4fer+%22Patterns+of+microsatellite+variation+through+a+transition+zone+of+a+chromosomal+cline+in+Drosophila+americana.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">43.
              </span><a name="pone.0000112-Kim1" id="pone.0000112-Kim1"></a>Kim Y, Stephan W (2000) Joint effects of genetic hitchhiking and background selection on neutral variation. Genetics  155: 1415–1427.  <ul class="find" data-citedArticleID="5293007"><li><a href="http://www.crossref.org/guestquery/?auth2=Kim&amp;atitle2=Joint+effects+of+genetic+hitchhiking+and+background+selection+on+neutral+variation.&amp;auth=Kim&amp;atitle=Joint+effects+of+genetic+hitchhiking+and+background+selection+on+neutral+variation." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Kim%5Bauthor%5D+AND+Joint+effects+of+genetic+hitchhiking+and+background+selection+on+neutral+variation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AKim+%22Joint+effects+of+genetic+hitchhiking+and+background+selection+on+neutral+variation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">44.
              </span><a name="pone.0000112-Endler1" id="pone.0000112-Endler1"></a>Endler JA (1973) Gene flow and population differentiation. Science  179: 243–250.  <ul class="find" data-citedArticleID="5293009" data-doi="10.1126/science.179.4070.243"><li><a href="http://dx.doi.org/10.1126/science.179.4070.243" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Endler%5Bauthor%5D+AND+Gene+flow+and+population+differentiation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AEndler+%22Gene+flow+and+population+differentiation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">45.
              </span><a name="pone.0000112-Lenormand1" id="pone.0000112-Lenormand1"></a>Lenormand T (2002) Gene flow and the limits to natural selection. Trends Ecol Evol  17: 183–189.  <ul class="find" data-citedArticleID="5293011" data-doi="10.1016/s0169-5347(02)02497-7"><li><a href="http://dx.doi.org/10.1016/s0169-5347(02)02497-7" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Lenormand%5Bauthor%5D+AND+Gene+flow+and+the+limits+to+natural+selection." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3ALenormand+%22Gene+flow+and+the+limits+to+natural+selection.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">46.
              </span><a name="pone.0000112-Greenberg1" id="pone.0000112-Greenberg1"></a>Greenberg AJ, Moran JR, Fang S, Wu C-I (2006) Adaptive loss of an old duplicated gene during incipient speciation. Mol Biol Evol  23: 401–410.  <ul class="find" data-citedArticleID="5293013" data-doi="10.1093/molbev/msj045"><li><a href="http://dx.doi.org/10.1093/molbev/msj045" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Greenberg%5Bauthor%5D+AND+Adaptive+loss+of+an+old+duplicated+gene+during+incipient+speciation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AGreenberg+%22Adaptive+loss+of+an+old+duplicated+gene+during+incipient+speciation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">47.
              </span><a name="pone.0000112-Vieira1" id="pone.0000112-Vieira1"></a>Vieira J, McAllister BF, Charlesworth B (2001) Evidence for selection at the <em>fused1</em> locus of <em>Drosophila americana</em>. Genetics  158: 279–290.  <ul class="find" data-citedArticleID="5293015"><li><a href="http://www.crossref.org/guestquery/?auth2=Vieira&amp;atitle2=Evidence+for+selection+at+the+fused1+locus+of+Drosophila+americana.&amp;auth=Vieira&amp;atitle=Evidence+for+selection+at+the+fused1+locus+of+Drosophila+americana." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Vieira%5Bauthor%5D+AND+Evidence+for+selection+at+the+fused1+locus+of+Drosophila+americana." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AVieira+%22Evidence+for+selection+at+the+fused1+locus+of+Drosophila+americana.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">48.
              </span><a name="pone.0000112-Kreitman1" id="pone.0000112-Kreitman1"></a>Kreitman M, Hudson RR (1991) Inferring the evolutionary histories of the <em>Adh</em> and <em>Adh-dup</em> loci in <em>Drosophila melanogaster</em> from patterns of polymorphism and divergence. Genetics  127: 565–582.  <ul class="find" data-citedArticleID="5293017"><li><a href="http://www.crossref.org/guestquery/?auth2=Kreitman&amp;atitle2=Inferring+the+evolutionary+histories+of+the+Adh+and+Adh-dup+loci+in+Drosophila+melanogaster+from+patterns+of+polymorphism+and+divergence.&amp;auth=Kreitman&amp;atitle=Inferring+the+evolutionary+histories+of+the+Adh+and+Adh-dup+loci+in+Drosophila+melanogaster+from+patterns+of+polymorphism+and+divergence." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Kreitman%5Bauthor%5D+AND+Inferring+the+evolutionary+histories+of+the+Adh+and+Adh-dup+loci+in+Drosophila+melanogaster+from+patterns+of+polymorphism+and+divergence." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AKreitman+%22Inferring+the+evolutionary+histories+of+the+Adh+and+Adh-dup+loci+in+Drosophila+melanogaster+from+patterns+of+polymorphism+and+divergence.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">49.
              </span><a name="pone.0000112-Navarro1" id="pone.0000112-Navarro1"></a>Navarro A, Barton NH (2002) The effects of multilocus balancing selection on neutral variability. Genetics  161: 849–863.  <ul class="find" data-citedArticleID="5293019"><li><a href="http://www.crossref.org/guestquery/?auth2=Navarro&amp;atitle2=The+effects+of+multilocus+balancing+selection+on+neutral+variability.&amp;auth=Navarro&amp;atitle=The+effects+of+multilocus+balancing+selection+on+neutral+variability." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Navarro%5Bauthor%5D+AND+The+effects+of+multilocus+balancing+selection+on+neutral+variability." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3ANavarro+%22The+effects+of+multilocus+balancing+selection+on+neutral+variability.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">50.
              </span><a name="pone.0000112-Rice2" id="pone.0000112-Rice2"></a>Rice WR (1984) Sex chromosomes and the evolution of sexual dimorphism. Evolution  38: 735–742.  <ul class="find" data-citedArticleID="5293021" data-doi="10.2307/2408385"><li><a href="http://dx.doi.org/10.2307/2408385" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Rice%5Bauthor%5D+AND+Sex+chromosomes+and+the+evolution+of+sexual+dimorphism." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3ARice+%22Sex+chromosomes+and+the+evolution+of+sexual+dimorphism.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">51.
              </span><a name="pone.0000112-Gibson1" id="pone.0000112-Gibson1"></a>Gibson JR, Chippindale AK, Rice WR (2002) The X chromosome is a hot spot for sexually antagonistic fitness variation. Proc Royal Soc Lond (B)  269: 499–505.  <ul class="find" data-citedArticleID="5293023" data-doi="10.1098/rspb.2001.1863"><li><a href="http://dx.doi.org/10.1098/rspb.2001.1863" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Gibson%5Bauthor%5D+AND+The+X+chromosome+is+a+hot+spot+for+sexually+antagonistic+fitness+variation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AGibson+%22The+X+chromosome+is+a+hot+spot+for+sexually+antagonistic+fitness+variation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">52.
              </span><a name="pone.0000112-Rice3" id="pone.0000112-Rice3"></a>Rice WR (1992) Sexually antagonistic genes: experimental evidence. Science  256: 1436–1439.  <ul class="find" data-citedArticleID="5293025" data-doi="10.1126/science.1604317"><li><a href="http://dx.doi.org/10.1126/science.1604317" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Rice%5Bauthor%5D+AND+Sexually+antagonistic+genes%3A+experimental+evidence." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3ARice+%22Sexually+antagonistic+genes%3A+experimental+evidence.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">53.
              </span><a name="pone.0000112-Rice4" id="pone.0000112-Rice4"></a>Rice WR (1998) Male fitness increases when females are eliminated from gene pool: Implications for the Y chromosome. Proc Natl Acad Sci USA  95: 6217–6221.  <ul class="find" data-citedArticleID="5293027" data-doi="10.1073/pnas.95.11.6217"><li><a href="http://dx.doi.org/10.1073/pnas.95.11.6217" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Rice%5Bauthor%5D+AND+Male+fitness+increases+when+females+are+eliminated+from+gene+pool%3A+Implications+for+the+Y+chromosome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3ARice+%22Male+fitness+increases+when+females+are+eliminated+from+gene+pool%3A+Implications+for+the+Y+chromosome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">54.
              </span><a name="pone.0000112-Yi1" id="pone.0000112-Yi1"></a>Yi S, Charlesworth B (2000) Contrasting patterns of molecular evolution of genes on the new and old sex chromosomes of <em>Drosophila miranda</em>. Mol Biol Evol  17: 703–717.  <ul class="find" data-citedArticleID="5293029" data-doi="10.1093/oxfordjournals.molbev.a026349"><li><a href="http://dx.doi.org/10.1093/oxfordjournals.molbev.a026349" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Yi%5Bauthor%5D+AND+Contrasting+patterns+of+molecular+evolution+of+genes+on+the+new+and+old+sex+chromosomes+of+Drosophila+miranda." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AYi+%22Contrasting+patterns+of+molecular+evolution+of+genes+on+the+new+and+old+sex+chromosomes+of+Drosophila+miranda.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">55.
              </span><a name="pone.0000112-Bachtrog1" id="pone.0000112-Bachtrog1"></a>Bachtrog D, Charlesworth B (2000) Reduced levels of microsatellite variability on the neo-Y chromosome of <em>Drosophila miranda</em>. Curr Biol  10: 1025–1031.  <ul class="find" data-citedArticleID="5293031" data-doi="10.1016/s0960-9822(00)00656-4"><li><a href="http://dx.doi.org/10.1016/s0960-9822(00)00656-4" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Bachtrog%5Bauthor%5D+AND+Reduced+levels+of+microsatellite+variability+on+the+neo-Y+chromosome+of+Drosophila+miranda." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3ABachtrog+%22Reduced+levels+of+microsatellite+variability+on+the+neo-Y+chromosome+of+Drosophila+miranda.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">56.
              </span><a name="pone.0000112-Bachtrog2" id="pone.0000112-Bachtrog2"></a>Bachtrog D, Charlesworth B (2002) Reduced adaptation of a non-recombining neo-Y chromosome. Nature  416: 323–326.  <ul class="find" data-citedArticleID="5293033" data-doi="10.1038/416323a"><li><a href="http://dx.doi.org/10.1038/416323a" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Bachtrog%5Bauthor%5D+AND+Reduced+adaptation+of+a+non-recombining+neo-Y+chromosome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3ABachtrog+%22Reduced+adaptation+of+a+non-recombining+neo-Y+chromosome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">57.
              </span><a name="pone.0000112-Bachtrog3" id="pone.0000112-Bachtrog3"></a>Bachtrog D (2004) Evidence that positive selection drives Y-chromosome degeneration in <em>Drosophila miranda</em>. Nat Genet  36: 518–522.  <ul class="find" data-citedArticleID="5293035" data-doi="10.1038/ng1347"><li><a href="http://dx.doi.org/10.1038/ng1347" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Bachtrog%5Bauthor%5D+AND+Evidence+that+positive+selection+drives+Y-chromosome+degeneration+in+Drosophila+miranda." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3ABachtrog+%22Evidence+that+positive+selection+drives+Y-chromosome+degeneration+in+Drosophila+miranda.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">58.
              </span><a name="pone.0000112-Rice5" id="pone.0000112-Rice5"></a>Rice WR (1987) Genetic hitchhiking and the evolution of reduced genetic activity of the Y sex chromosome. Genetics  116: 161–167.  <ul class="find" data-citedArticleID="5293037"><li><a href="http://www.crossref.org/guestquery/?auth2=Rice&amp;atitle2=Genetic+hitchhiking+and+the+evolution+of+reduced+genetic+activity+of+the+Y+sex+chromosome.&amp;auth=Rice&amp;atitle=Genetic+hitchhiking+and+the+evolution+of+reduced+genetic+activity+of+the+Y+sex+chromosome." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Rice%5Bauthor%5D+AND+Genetic+hitchhiking+and+the+evolution+of+reduced+genetic+activity+of+the+Y+sex+chromosome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3ARice+%22Genetic+hitchhiking+and+the+evolution+of+reduced+genetic+activity+of+the+Y+sex+chromosome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">59.
              </span><a name="pone.0000112-Filatov2" id="pone.0000112-Filatov2"></a>Filatov DA, Laporte V, Vitte C, Charlesworth D (2001) DNA diversity and sex-linked and autosomal genes of the plant species <em>Silene latifolia</em> and <em>Silene dioica</em>. Mol Biol Evol  18: 1442–1454.  <ul class="find" data-citedArticleID="5293039" data-doi="10.1093/oxfordjournals.molbev.a003930"><li><a href="http://dx.doi.org/10.1093/oxfordjournals.molbev.a003930" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Filatov%5Bauthor%5D+AND+DNA+diversity+and+sex-linked+and+autosomal+genes+of+the+plant+species+Silene+latifolia+and+Silene+dioica." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AFilatov+%22DNA+diversity+and+sex-linked+and+autosomal+genes+of+the+plant+species+Silene+latifolia+and+Silene+dioica.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">60.
              </span><a name="pone.0000112-Scotti1" id="pone.0000112-Scotti1"></a>Scotti I, Delph LF (2006) Selective trade-offs and sex-chromosome evolution in <em>Silene latifolia</em>. Evolution  60: 1793–1800.  <ul class="find" data-citedArticleID="5293041" data-doi="10.1111/j.0014-3820.2006.tb00523.x"><li><a href="http://dx.doi.org/10.1111/j.0014-3820.2006.tb00523.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Scotti%5Bauthor%5D+AND+Selective+trade-offs+and+sex-chromosome+evolution+in+Silene+latifolia." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AScotti+%22Selective+trade-offs+and+sex-chromosome+evolution+in+Silene+latifolia.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">61.
              </span><a name="pone.0000112-Charlesworth7" id="pone.0000112-Charlesworth7"></a>Charlesworth B, Charlesworth D, Hnilicka J, Yu A, Guttman DS (1997) Lack of degeneration of loci on the neo-<em>Y</em> chromosome of <em>Drosophila americana americana</em>. Genetics  145: 989–1002.  <ul class="find" data-citedArticleID="5293043"><li><a href="http://www.crossref.org/guestquery/?auth2=Charlesworth&amp;atitle2=Lack+of+degeneration+of+loci+on+the+neo-Y+chromosome+of+Drosophila+americana+americana.&amp;auth=Charlesworth&amp;atitle=Lack+of+degeneration+of+loci+on+the+neo-Y+chromosome+of+Drosophila+americana+americana." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Charlesworth%5Bauthor%5D+AND+Lack+of+degeneration+of+loci+on+the+neo-Y+chromosome+of+Drosophila+americana+americana." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3ACharlesworth+%22Lack+of+degeneration+of+loci+on+the+neo-Y+chromosome+of+Drosophila+americana+americana.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">62.
              </span><a name="pone.0000112-Gubenko1" id="pone.0000112-Gubenko1"></a>Gubenko IS, Evgen'ev MB (1984) Cytological and linkage maps of <em>Drosophila virilis</em> chromosomes. Genetica  65: 127–139.  <ul class="find" data-citedArticleID="5293045" data-doi="10.1007/bf00135277"><li><a href="http://dx.doi.org/10.1007/bf00135277" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Gubenko%5Bauthor%5D+AND+Cytological+and+linkage+maps+of+Drosophila+virilis+chromosomes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AGubenko+%22Cytological+and+linkage+maps+of+Drosophila+virilis+chromosomes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">63.
              </span><a name="pone.0000112-Nurminsky1" id="pone.0000112-Nurminsky1"></a>Nurminsky DI, Moriyama EN, Lozovskaya ER, Hartl DL (1996) Molecular phylogeny and genome evolution in the <em>Drosophila virilis</em> species group: duplications of the <em>Alcohol Dehydrogenase g</em>ene. Mol Biol Evol  13: 132–149.  <ul class="find" data-citedArticleID="5293047" data-doi="10.1093/oxfordjournals.molbev.a025551"><li><a href="http://dx.doi.org/10.1093/oxfordjournals.molbev.a025551" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Nurminsky%5Bauthor%5D+AND+Molecular+phylogeny+and+genome+evolution+in+the+Drosophila+virilis+species+group%3A+duplications+of+the+Alcohol+Dehydrogenase+gene." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3ANurminsky+%22Molecular+phylogeny+and+genome+evolution+in+the+Drosophila+virilis+species+group%3A+duplications+of+the+Alcohol+Dehydrogenase+gene.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">64.
              </span><a name="pone.0000112-Tominaga1" id="pone.0000112-Tominaga1"></a>Tominaga H, Narise S (1995) Sequence evolution of the <em>Gpdh</em> gene in the <em>Drosophila virilis</em> species group. Genetica  96: 293–302.  <ul class="find" data-citedArticleID="5293049" data-doi="10.1007/bf01439583"><li><a href="http://dx.doi.org/10.1007/bf01439583" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Tominaga%5Bauthor%5D+AND+Sequence+evolution+of+the+Gpdh+gene+in+the+Drosophila+virilis+species+group." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3ATominaga+%22Sequence+evolution+of+the+Gpdh+gene+in+the+Drosophila+virilis+species+group.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">65.
              </span><a name="pone.0000112-Lozovskaya1" id="pone.0000112-Lozovskaya1"></a>Lozovskaya ER, Petrov DA, Hartl DL (1993) A combined molecular and cytogenetic approach to genome evolution in <em>Drosophila</em> using large-fragment DNA cloning. Chromosoma  102: 253–266.  <ul class="find" data-citedArticleID="5293051" data-doi="10.1007/bf00352399"><li><a href="http://dx.doi.org/10.1007/bf00352399" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Lozovskaya%5Bauthor%5D+AND+A+combined+molecular+and+cytogenetic+approach+to+genome+evolution+in+Drosophila+using+large-fragment+DNA+cloning." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3ALozovskaya+%22A+combined+molecular+and+cytogenetic+approach+to+genome+evolution+in+Drosophila+using+large-fragment+DNA+cloning.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">66.
              </span><a name="pone.0000112-Vieira2" id="pone.0000112-Vieira2"></a>Vieira J, Vieira C, Hartl DL, Lozovskaya ER (1997) A framework physical map of <em>Drosophila virilis</em> based on P1 clones: application in genome evolution. Chromosoma  106: 99–107.  <ul class="find" data-citedArticleID="5293053" data-doi="10.1007/s004120050229"><li><a href="http://dx.doi.org/10.1007/s004120050229" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Vieira%5Bauthor%5D+AND+A+framework+physical+map+of+Drosophila+virilis+based+on+P1+clones%3A+application+in+genome+evolution." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AVieira+%22A+framework+physical+map+of+Drosophila+virilis+based+on+P1+clones%3A+application+in+genome+evolution.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">67.
              </span><a name="pone.0000112-Schwartz1" id="pone.0000112-Schwartz1"></a>Schwartz S, Zhang Z, Frazer KA, Smit A, Riemer C, Bouck J, Gibbs R, Hardison R, Miller W (2000) PipMaker—A web server for aligning two genomic DNA sequences. Genome Res.  10: 577–586.  <ul class="find" data-citedArticleID="5293055" data-doi="10.1101/gr.10.4.577"><li><a href="http://dx.doi.org/10.1101/gr.10.4.577" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Schwartz%5Bauthor%5D+AND+PipMaker%E2%80%94A+web+server+for+aligning+two+genomic+DNA+sequences." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3ASchwartz+%22PipMaker%E2%80%94A+web+server+for+aligning+two+genomic+DNA+sequences.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">68.
              </span><a name="pone.0000112-Rozas1" id="pone.0000112-Rozas1"></a>Rozas J, Sánchez-DelBarrio JC, Messeguer X, Rozas R (2003) DnaSP, DNA polymorphism analyses by the coalescent and other methods. Bioinformatics  19: 2496–2497.  <ul class="find" data-citedArticleID="5293057" data-doi="10.1093/bioinformatics/btg359"><li><a href="http://dx.doi.org/10.1093/bioinformatics/btg359" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Rozas%5Bauthor%5D+AND+DnaSP%2C+DNA+polymorphism+analyses+by+the+coalescent+and+other+methods." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3ARozas+%22DnaSP%2C+DNA+polymorphism+analyses+by+the+coalescent+and+other+methods.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">69.
              </span><a name="pone.0000112-Hudson3" id="pone.0000112-Hudson3"></a>Hudson RR, Boos DD, Kaplan NL (1992) A statistical test for detecting geographic subdivision. Mol Biol Evol  9: 138–151.  <ul class="find" data-citedArticleID="5293059"><li><a href="http://www.crossref.org/guestquery/?auth2=Hudson&amp;atitle2=A+statistical+test+for+detecting+geographic+subdivision.&amp;auth=Hudson&amp;atitle=A+statistical+test+for+detecting+geographic+subdivision." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Hudson%5Bauthor%5D+AND+A+statistical+test+for+detecting+geographic+subdivision." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AHudson+%22A+statistical+test+for+detecting+geographic+subdivision.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">70.
              </span><a name="pone.0000112-Hudson4" id="pone.0000112-Hudson4"></a>Hudson RR (1987) Estimating the recombination parameter of a finite population model without selection. Genet Res  50: 245–250.  <ul class="find" data-citedArticleID="5293061" data-doi="10.1017/s0016672300023776"><li><a href="http://dx.doi.org/10.1017/s0016672300023776" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Hudson%5Bauthor%5D+AND+Estimating+the+recombination+parameter+of+a+finite+population+model+without+selection." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AHudson+%22Estimating+the+recombination+parameter+of+a+finite+population+model+without+selection.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">71.
              </span><a name="pone.0000112-Tajima3" id="pone.0000112-Tajima3"></a>Tajima F (1983) Evolutionary relationship of DNA sequences in finite populations. Genetics  105: 437–460.  <ul class="find" data-citedArticleID="5293063"><li><a href="http://www.crossref.org/guestquery/?auth2=Tajima&amp;atitle2=Evolutionary+relationship+of+DNA+sequences+in+finite+populations.&amp;auth=Tajima&amp;atitle=Evolutionary+relationship+of+DNA+sequences+in+finite+populations." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Tajima%5Bauthor%5D+AND+Evolutionary+relationship+of+DNA+sequences+in+finite+populations." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3ATajima+%22Evolutionary+relationship+of+DNA+sequences+in+finite+populations.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">72.
              </span><a name="pone.0000112-Stokes1" id="pone.0000112-Stokes1"></a>Stokes ME, Davis CS, Koch GG (2000) Categorical Data Analysis Using the SAS System. SAS Institute Inc.   <ul class="find-nolinks"></ul></li><li><span class="label">73.
              </span><a name="pone.0000112-Haukoos1" id="pone.0000112-Haukoos1"></a>Haukoos JS, Lewis RJ (2005) Bootstrapping confidence intervals for statistics with ‘difficult’ distributions. Acad Emerg Med  12: 360–365.  <ul class="find" data-citedArticleID="5293067" data-doi="10.1111/j.1553-2712.2005.tb01958.x"><li><a href="http://dx.doi.org/10.1111/j.1553-2712.2005.tb01958.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Haukoos%5Bauthor%5D+AND+Bootstrapping+confidence+intervals+for+statistics+with+%E2%80%98difficult%E2%80%99+distributions." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AHaukoos+%22Bootstrapping+confidence+intervals+for+statistics+with+%E2%80%98difficult%E2%80%99+distributions.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">74.
              </span><a name="pone.0000112-Kennison1" id="pone.0000112-Kennison1"></a>Kennison JA (2000)  Preparation and analysis of polytene chromosomes. In: Sullivan W, Ashburner M, Hawley RS, editors. Drosophila Protocols. Cold Spring Harbor: Cold Spring Harbor Laboratory Press. pp. 111–117.  <ul class="find-nolinks"></ul></li><li><span class="label">75.
              </span><a name="pone.0000112-Manly1" id="pone.0000112-Manly1"></a>Manly KF, Olson JM (1999) Overview of QTL mapping software and introduction to Map Manager QT. Mammalian Genome  10: 327–334.  <ul class="find" data-citedArticleID="5293071" data-doi="10.1007/s003359900997"><li><a href="http://dx.doi.org/10.1007/s003359900997" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Manly%5Bauthor%5D+AND+Overview+of+QTL+mapping+software+and+introduction+to+Map+Manager+QT." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AManly+%22Overview+of+QTL+mapping+software+and+introduction+to+Map+Manager+QT.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">76.
              </span><a name="pone.0000112-Altschul1" id="pone.0000112-Altschul1"></a>Altschul SF, Madden TL, Schaffer AA, Zhang J, Zhang Z, et al.  (1997) Gapped BLAST and PSI-BLAST: a new generation of protein database search programs. Nucl Acids Res  25: 3389–3402.  <ul class="find" data-citedArticleID="5293073" data-doi="10.1093/nar/25.17.3389"><li><a href="http://dx.doi.org/10.1093/nar/25.17.3389" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Altschul%5Bauthor%5D+AND+Gapped+BLAST+and+PSI-BLAST%3A+a+new+generation+of+protein+database+search+programs." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AAltschul+%22Gapped+BLAST+and+PSI-BLAST%3A+a+new+generation+of+protein+database+search+programs.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000112.XML" value="148384"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000112.PDF" value="274009"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000112.g001.TIF" value="145218"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000112.g001.PNG_I" value="21159"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000112.g001.PNG_S" value="4137"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000112.g001.PNG_M" value="41950"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000112.g001.PNG_L" value="65001"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000112.t001.TIF" value="1892730"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000112.t001.PNG_I" value="16526"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000112.t001.PNG_S" value="2985"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000112.t001.PNG_M" value="37133"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000112.t001.PNG_L" value="63119"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000112.g002.TIF" value="938914"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000112.g002.PNG_I" value="77552"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000112.g002.PNG_S" value="10039"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000112.g002.PNG_M" value="76766"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000112.g002.PNG_L" value="622393"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000112.t002.TIF" value="13572350"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000112.t002.PNG_I" value="21801"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000112.t002.PNG_S" value="3503"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000112.t002.PNG_M" value="49620"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000112.t002.PNG_L" value="221254"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000112.t003.TIF" value="1750350"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000112.t003.PNG_I" value="12436"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000112.t003.PNG_S" value="2554"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000112.t003.PNG_M" value="27109"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000112.t003.PNG_L" value="44025"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000112.t004.TIF" value="3444982"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000112.t004.PNG_I" value="24774"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000112.t004.PNG_S" value="4072"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000112.t004.PNG_M" value="56426"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000112.t004.PNG_L" value="99046"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000112.g003.TIF" value="269746"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000112.g003.PNG_I" value="7335"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000112.g003.PNG_S" value="2589"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000112.g003.PNG_M" value="14423"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000112.g003.PNG_L" value="126309"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000112.g004.TIF" value="241086"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000112.g004.PNG_I" value="19364"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000112.g004.PNG_S" value="3943"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000112.g004.PNG_M" value="43623"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000112.g004.PNG_L" value="91515"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000112.s001.DOC" value="27136"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000112.s002.DOC" value="24064"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000112.s003.DOC" value="28672"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000112.s004.TIF" value="55034"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000112.s005.TIF" value="38170"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000112&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000112" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000112&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0000112&volume=&issue=&title=Increased Nucleotide Diversity with Transient Y Linkage in Drosophila americana&author_name=Bryant%20F.%20McAllister%2C%20Amy%20L.%20Evans&start_page=1&end_page=13" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000112" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000112" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0000112" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000112&amp;t=Increased%20Nucleotide%20Diversity%20with%20Transient%20Y%20Linkage%20in%20Drosophila%20americana" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000112&title=Increased%20Nucleotide%20Diversity%20with%20Transient%20Y%20Linkage%20in%20Drosophila%20americana&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000112&amp;title=Increased%20Nucleotide%20Diversity%20with%20Transient%20Y%20Linkage%20in%20Drosophila%20americana" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000112" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0000112&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Increased Nucleotide Diversity with Transient Y Linkage in Drosophila americana';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0000112';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Increased%20Nucleotide%20Diversity%20with%20Transient%20Y%20Linkage%20in%20Drosophila%20americana http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000112" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0000112" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">










          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Alleles%22" title="Search for articles in the subject area:'Alleles'"><div class="flagText">Alleles</div></a>
              <div data-categoryid="46071" data-articleid="23062"
                   data-categoryname="Alleles"
                   class="flagImage" title="Flag 'Alleles' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Centromeres%22" title="Search for articles in the subject area:'Centromeres'"><div class="flagText">Centromeres</div></a>
              <div data-categoryid="43147" data-articleid="23062"
                   data-categoryname="Centromeres"
                   class="flagImage" title="Flag 'Centromeres' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Chromosome+4%22" title="Search for articles in the subject area:'Chromosome 4'"><div class="flagText">Chromosome 4</div></a>
              <div data-categoryid="22417" data-articleid="23062"
                   data-categoryname="Chromosome 4"
                   class="flagImage" title="Flag 'Chromosome 4' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Chromosomes%22" title="Search for articles in the subject area:'Chromosomes'"><div class="flagText">Chromosomes</div></a>
              <div data-categoryid="17251" data-articleid="23062"
                   data-categoryname="Chromosomes"
                   class="flagImage" title="Flag 'Chromosomes' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Genetic+loci%22" title="Search for articles in the subject area:'Genetic loci'"><div class="flagText">Genetic loci</div></a>
              <div data-categoryid="17713" data-articleid="23062"
                   data-categoryname="Genetic loci"
                   class="flagImage" title="Flag 'Genetic loci' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Population+genetics%22" title="Search for articles in the subject area:'Population genetics'"><div class="flagText">Population genetics</div></a>
              <div data-categoryid="32359" data-articleid="23062"
                   data-categoryname="Population genetics"
                   class="flagImage" title="Flag 'Population genetics' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Sex+chromosomes%22" title="Search for articles in the subject area:'Sex chromosomes'"><div class="flagText">Sex chromosomes</div></a>
              <div data-categoryid="18743" data-articleid="23062"
                   data-categoryname="Sex chromosomes"
                   class="flagImage" title="Flag 'Sex chromosomes' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22X+chromosomes%22" title="Search for articles in the subject area:'X chromosomes'"><div class="flagText">X chromosomes</div></a>
              <div data-categoryid="17587" data-articleid="23062"
                   data-categoryname="X chromosomes"
                   class="flagImage" title="Flag 'X chromosomes' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=678'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=577'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=8471&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>

<div class="block sidebar-comments">
    <div class="header">
        <h3>Comments</h3>
    </div>
      <p><a href="/annotation/listThread.action?root=1749">Genotype files</a><br>Posted by bryantmc</p>
      <p><a href="/annotation/listThread.action?root=1745">Sequence Files</a><br>Posted by bryantmc</p>
      <p><a href="/annotation/listThread.action?root=1731">X-4 Frequencies in D. americana</a><br>Posted by bryantmc</p>
</div>

</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
