

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Conditional Transgenesis Using Dimerizable Cre (DiCre)</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0001355"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0001355" />

    <meta name="citation_title" content="Conditional Transgenesis Using Dimerizable Cre (DiCre)"/>
    <meta itemprop="name" content="Conditional Transgenesis Using Dimerizable Cre (DiCre)"/>

      <meta name="citation_author" content="Nicolas Jullien"/>
            <meta name="citation_author_institution" content="ICNE-UMR 6544 Centre National de la Recherche Scientifique (CNRS), Université de la Méditerranée, Marseille, France"/>
      <meta name="citation_author" content="Isabelle Goddard"/>
            <meta name="citation_author_institution" content="Centre de Transgenèse, Faculté de Médecine Nord, Institut Fédératif de Recherche (IFR) Jean-Roche, Marseille, France"/>
      <meta name="citation_author" content="Samia Selmi-Ruby"/>
            <meta name="citation_author_institution" content="INSERM UMR 664, Faculté de Médecine RTH Laennec, Université Lyon I, Lyon, France"/>
      <meta name="citation_author" content="Jean-Luc Fina"/>
            <meta name="citation_author_institution" content="Centre de Transgenèse, Faculté de Médecine Nord, Institut Fédératif de Recherche (IFR) Jean-Roche, Marseille, France"/>
      <meta name="citation_author" content="Harold Cremer"/>
            <meta name="citation_author_institution" content="IBDML-UMR 6216 Centre National de la Recherche Scientifique (CNRS), Université de la Méditerranée, Marseille, France"/>
      <meta name="citation_author" content="Jean-Paul Herman"/>
            <meta name="citation_author_institution" content="ICNE-UMR 6544 Centre National de la Recherche Scientifique (CNRS), Université de la Méditerranée, Marseille, France"/>

    <meta name="citation_date" content="2007/12/26"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0001355.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e1355"/>
    <meta name="citation_issue" content="12"/>
    <meta name="citation_volume" content="2"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Manipulation of transgenes by site-specific recombination: use of Cre recombinase.; citation_author=B Sauer; citation_journal_title=Methods Enzymol; citation_volume=225; citation_number=1; citation_pages=890-900; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=Talking about a revolution: The impact of site-specific recombinases on genetic analyses in mice.; citation_author=CS Branda; citation_author=SM Dymecki; citation_journal_title=Dev Cell; citation_volume=6; citation_number=2; citation_pages=7-28; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Cre recombinase: the universal reagent for genome tailoring.; citation_author=A Nagy; citation_journal_title=Genesis; citation_volume=26; citation_number=3; citation_pages=99-109; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Molecular neuroanatomy's “Three Gs”: a primer.; citation_author=SM Dymecki; citation_author=JC Kim; citation_journal_title=Neuron; citation_volume=54; citation_number=4; citation_pages=17-34; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Conditional alleles in mice: practical considerations for tissue-specific knockouts.; citation_author=KM Kwan; citation_journal_title=Genesis; citation_volume=32; citation_number=5; citation_pages=49-62; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Fate mapping of the mouse midbrain-hindbrain constriction using a site-specific recombination system.; citation_author=DL Zinyk; citation_author=EH Mercer; citation_author=E Harris; citation_author=DJ Anderson; citation_author=AL Joyner; citation_journal_title=Curr Biol; citation_volume=8; citation_number=6; citation_pages=665-668; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Subregion- and cell type-restricted gene knockout in mouse brain.; citation_author=JZ Tsien; citation_author=DF Chen; citation_author=D Gerber; citation_author=C Tom; citation_author=EH Mercer; citation_journal_title=Cell; citation_volume=87; citation_number=7; citation_pages=1317-1326; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Nestin-Cre mediated deletion of Pitx2 in the mouse.; citation_author=AM Sclafani; citation_author=JM Skidmore; citation_author=H Ramaprakash; citation_author=A Trumpp; citation_author=PJ Gage; citation_journal_title=Genesis; citation_volume=44; citation_number=8; citation_pages=336-344; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Temporal, spatial, and cell type-specific control of Cre-mediated DNA recombination in transgenic mice.; citation_author=AR Utomo; citation_author=AY Nikitin; citation_author=WH Lee; citation_journal_title=Nat Biotechnol; citation_volume=17; citation_number=9; citation_pages=1091-1096; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Studying gene function in eukaryotes by conditional gene inactivation.; citation_author=M Gossen; citation_author=H Bujard; citation_journal_title=Annu Rev Genet; citation_volume=36; citation_number=10; citation_pages=153-173; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Temporal control of the Cre recombinase in transgenic mice by a tetracycline responsive promoter.; citation_author=L St-Onge; citation_author=PA Furth; citation_author=P Gruss; citation_journal_title=Nucleic Acids Res; citation_volume=24; citation_number=11; citation_pages=3875-3877; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Stringent doxycycline dependent control of CRE recombinase in vivo.; citation_author=K Schonig; citation_author=F Schwenk; citation_author=K Rajewsky; citation_author=H Bujard; citation_journal_title=Nucleic Acids Res; citation_volume=30; citation_number=12; citation_pages=e134; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Inducible site-specific recombination in the brain.; citation_author=C Kellendonk; citation_author=F Tronche; citation_author=E Casanova; citation_author=K Anlag; citation_author=C Opherk; citation_journal_title=J Mol Biol; citation_volume=285; citation_number=13; citation_pages=175-182; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Ligand-activated site-specific recombination in mice.; citation_author=R Feil; citation_author=J Brocard; citation_author=B Mascrez; citation_author=M LeMeur; citation_author=D Metzger; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=93; citation_number=14; citation_pages=10887-10890; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Regulation of Cre recombinase activity by the synthetic steroid RU 486.; citation_author=C Kellendonk; citation_author=F Tronche; citation_author=AP Monaghan; citation_author=PO Angrand; citation_author=F Stewart; citation_journal_title=Nucleic Acids Res; citation_volume=24; citation_number=15; citation_pages=1404-1411; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Temporally-controlled site-specific mutagenesis in the basal layer of the epidermis: comparison of the recombinase activity of the tamoxifen-inducible Cre-ER(T) and Cre-ER(T2) recombinases.; citation_author=AK Indra; citation_author=X Warot; citation_author=J Brocard; citation_author=JM Bornert; citation_author=JH Xiao; citation_journal_title=Nucleic Acids Res; citation_volume=27; citation_number=16; citation_pages=4324-4327; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=High levels of Cre expression in neuronal progenitors cause defects in brain development leading to microencephaly and hydrocephaly.; citation_author=PE Forni; citation_author=C Scuoppo; citation_author=I Imayoshi; citation_author=R Taulli; citation_author=W Dastru; citation_journal_title=J Neurosci; citation_volume=26; citation_number=17; citation_pages=9593-9602; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Cell behaviors and genetic lineages of the mesencephalon and rhombomere 1.; citation_author=M Zervas; citation_author=S Millet; citation_author=S Ahn; citation_author=AL Joyner; citation_journal_title=Neuron; citation_volume=43; citation_number=18; citation_pages=345-357; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=A novel technique for temporally regulated cell type-specific Cre expression and recombination in the pituitary gonadotroph.; citation_author=K Naik; citation_author=It Pittman; citation_author=A Wolfe; citation_author=RS Miller; citation_author=S Radovick; citation_journal_title=J Mol Endocrinol; citation_volume=37; citation_number=19; citation_pages=63-69; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=A highly efficient ligand-regulated Cre recombinase mouse line shows that LoxP recombination is position dependent.; citation_author=M Vooijs; citation_author=J Jonkers; citation_author=A Berns; citation_journal_title=EMBO Rep; citation_volume=2; citation_number=20; citation_pages=292-297; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Toxicity of ligand-dependant Cre-recombinases and generation of a conditional Cre-deleter mouse allowing mosaic recombination in peripheral tissues.; citation_author=D Hameyer; citation_author=A Loonstra; citation_author=L Eshkind; citation_author=S Schmitt; citation_author=C Antunes; citation_journal_title=Physiol Genomics; citation_number=21; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=The effect of a single neonatal treatment (hormonal imprinting) with the antihormones, tamoxifen and mifepristone on the sexual behavior of adult rats.; citation_author=G Csaba; citation_author=C Karabelyos; citation_journal_title=Pharmacol Res; citation_volume=43; citation_number=22; citation_pages=531-534; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Neonatal treatment with tamoxifen causes immediate alterations of the sexually dimorphic nucleus of the preoptic area and medial preoptic area in male rats.; citation_author=PM Vancutsem; citation_author=ML Roessler; citation_journal_title=Teratology; citation_volume=56; citation_number=23; citation_pages=220-228; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Regulation of Cre recombinase by ligand-induced complementation of inactive fragments.; citation_author=N Jullien; citation_author=F Sampieri; citation_author=A Enjalbert; citation_author=JP Herman; citation_journal_title=Nucleic Acids Res; citation_volume=31; citation_number=24; citation_pages=E131; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=A humanized system for pharmacologic control of gene expression.; citation_author=VM Rivera; citation_author=T Clackson; citation_author=S Natesan; citation_author=R Pollock; citation_author=JF Amara; citation_journal_title=Nat Med; citation_volume=2; citation_number=25; citation_pages=1028-1032; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=A cytosolic binding protein for the immunosuppressant FK506 has peptidyl-prolyl isomerase activity but is distinct from cyclophilin.; citation_author=JJ Siekierka; citation_author=SH Hung; citation_author=M Poe; citation_author=CS Lin; citation_author=NH Sigal; citation_journal_title=Nature; citation_volume=341; citation_number=26; citation_pages=755-757; citation_date=1989; " />
      <meta name="citation_reference" content="citation_title=RAPT1, a mammalian homolog of yeast Tor, interacts with the FKBP12/rapamycin complex.; citation_author=MI Chiu; citation_author=H Katz; citation_author=V Berlin; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=91; citation_number=27; citation_pages=12574-12578; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=Ubiquitous expression of marker transgenes in mice and rats.; citation_author=WC Kisseberth; citation_author=NT Brettingen; citation_author=JK Lohse; citation_author=EP Sandgren; citation_journal_title=Dev Biol; citation_volume=214; citation_number=28; citation_pages=128-138; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Generalized lacZ expression with the ROSA26 Cre reporter strain.; citation_author=P Soriano; citation_journal_title=Nat Genet; citation_volume=21; citation_number=29; citation_pages=70-71; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Disruption of overlapping transcripts in the ROSA beta geo 26 gene trap strain leads to widespread expression of beta-galactosidase in mouse embryos and hematopoietic cells.; citation_author=BP Zambrowicz; citation_author=A Imamoto; citation_author=S Fiering; citation_author=LA Herzenberg; citation_author=WG Kerr; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=94; citation_number=30; citation_pages=3789-3794; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=High-efficiency deleter mice show that FLPe is an alternative to Cre-loxP.; citation_author=CI Rodriguez; citation_author=F Buchholz; citation_author=J Galloway; citation_author=R Sequerra; citation_author=J Kasper; citation_journal_title=Nat Genet; citation_volume=25; citation_number=31; citation_pages=139-140; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Efficient gene activation in mammalian cells by using recombinant adenovirus expressing site-specific Cre recombinase.; citation_author=Y Kanegae; citation_author=G Lee; citation_author=Y Sato; citation_author=M Tanaka; citation_author=M Nakai; citation_journal_title=Nucleic Acids Res; citation_volume=23; citation_number=32; citation_pages=3816-3821; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Z/EG, a double reporter mouse line that expresses enhanced green fluorescent protein upon Cre-mediated excision.; citation_author=A Novak; citation_author=C Guo; citation_author=W Yang; citation_author=A Nagy; citation_author=CG Lobe; citation_journal_title=Genesis; citation_volume=28; citation_number=33; citation_pages=147-155; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Manipulating the Mouse Embryo. A Laboratory Manual.; citation_author=A Nagy; citation_author=M Gertsenstein; citation_author=K Vintersten; citation_author=R Behringer; citation_number=34; citation_date=2003; citation_publisher=Cold Spring Harbor Laboratory Press; " />
      <meta name="citation_reference" content="citation_title=alpha Complementation in the Cre recombinase enzyme.; citation_author=E Casanova; citation_author=T Lemberger; citation_author=S Fehsenfeld; citation_author=T Mantamadiotis; citation_author=G Schutz; citation_journal_title=Genesis; citation_volume=37; citation_number=35; citation_pages=25-29; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Cre reconstitution allows for DNA recombination selectively in dual-marker-expressing cells in transgenic mice.; citation_author=Y Xu; citation_author=G Xu; citation_author=B Liu; citation_author=G Gu; citation_journal_title=Nucleic Acids Res; citation_volume=35; citation_number=36; citation_pages=e126; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Rapid generation of inducible mouse mutants.; citation_author=J Seibler; citation_author=B Zevnik; citation_author=B Kuter-Luks; citation_author=S Andreas; citation_author=H Kern; citation_journal_title=Nucleic Acids Res; citation_volume=31; citation_number=37; citation_pages=e12; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Directing neuron-specific transgene expression in the mouse CNS.; citation_author=G Miyoshi; citation_author=G Fishell; citation_journal_title=Curr Opin Neurobiol; citation_volume=16; citation_number=38; citation_pages=577-584; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Pharmacokinetics and metabolism of sirolimus.; citation_author=HL Gallant-Haidner; citation_author=DJ Trepanier; citation_author=DG Freitag; citation_author=RW Yatscoff; citation_journal_title=Ther Drug Monit; citation_volume=22; citation_number=39; citation_pages=31-35; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Distribution of sirolimus in rat tissue.; citation_author=KL Napoli; citation_author=ME Wang; citation_author=SM Stepkowski; citation_author=BD Kahan; citation_journal_title=Clin Biochem; citation_volume=30; citation_number=40; citation_pages=135-142; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Chemical rescue of cleft palate and midline defects in conditional GSK-3beta mice.; citation_author=KJ Liu; citation_author=JR Arron; citation_author=K Stankunas; citation_author=GR Crabtree; citation_author=MT Longaker; citation_journal_title=Nature; citation_volume=446; citation_number=41; citation_pages=79-82; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Gene expression of FK506-binding proteins 12.6 and 12 during chicken development.; citation_author=S Yazawa; citation_author=K Obata; citation_author=A Iio; citation_author=M Koide; citation_author=M Yokota; citation_journal_title=Comp Biochem Physiol A Mol Integr Physiol; citation_volume=136; citation_number=42; citation_pages=391-399; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=FRAP/mTOR is required for proliferation and patterning during embryonic development in the mouse.; citation_author=KE Hentges; citation_author=B Sirry; citation_author=AC Gingeras; citation_author=D Sarbassov; citation_author=N Sonenberg; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=98; citation_number=43; citation_pages=13796-13801; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Therapeutic potential of target of rapamycin inhibitors.; citation_author=JB Easton; citation_author=PJ Houghton; citation_journal_title=Expert Opin Ther Targets; citation_volume=8; citation_number=44; citation_pages=551-564; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Rapamycin analogs with differential binding specificity permit orthogonal control of protein activity.; citation_author=JH Bayle; citation_author=JS Grimley; citation_author=K Stankunas; citation_author=JE Gestwicki; citation_author=TJ Wandless; citation_journal_title=Chem Biol; citation_volume=13; citation_number=45; citation_pages=99-107; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Dimerizer-regulated gene expression.; citation_author=R Pollock; citation_author=T Clackson; citation_journal_title=Curr Opin Biotechnol; citation_volume=13; citation_number=46; citation_pages=459-467; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Redesigning an FKBP-ligand interface to generate chemical dimerizers with novel specificity.; citation_author=T Clackson; citation_author=W Yang; citation_author=LW Rozamus; citation_author=M Hatada; citation_author=JF Amara; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=95; citation_number=47; citation_pages=10437-10442; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=A rapid, reversible, and tunable method to regulate protein function in living cells using synthetic small molecules.; citation_author=LA Banaszynski; citation_author=LC Chen; citation_author=LA Maynard-Smith; citation_author=AG Ooi; citation_author=TJ Wandless; citation_journal_title=Cell; citation_volume=126; citation_number=48; citation_pages=995-1004; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Intravenous safety and pharmacokinetics of a novel dimerizer drug, AP1903, in healthy volunteers.; citation_author=JD Iuliucci; citation_author=SD Oliver; citation_author=S Morley; citation_author=C Ward; citation_author=J Ward; citation_journal_title=J Clin Pharmacol; citation_volume=41; citation_number=49; citation_pages=870-879; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Current Protocols in Molecular Biology.; citation_author=FM Ausabel; citation_author=R Brent; citation_author=RE Kingston; citation_author=DD Moore; citation_author=JG Seidman; citation_number=50; citation_date=2007; citation_publisher=John Wiley; " />
      <meta name="citation_reference" content="citation_title=Promoter traps in embryonic stem cells: a genetic screen to identify and mutate developmental genes in mice.; citation_author=G Friedrich; citation_author=P Soriano; citation_journal_title=Genes Dev; citation_volume=5; citation_number=51; citation_pages=1513-1523; citation_date=1991; " />
      <meta name="citation_reference" content="citation_title=Derivation of completely cell culture-derived mice from early-passage embryonic stem cells.; citation_author=A Nagy; citation_author=J Rossant; citation_author=R Nagy; citation_author=W Abramow-Newerly; citation_author=JC Roder; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=90; citation_number=52; citation_pages=8424-8428; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=Preparation of PCR-quality mouse genomic DNA with hot sodium hydroxide and tris (HotSHOT).; citation_author=GE Truett; citation_author=P Heeger; citation_author=RL Mynatt; citation_author=AA Truett; citation_author=JA Walker; citation_journal_title=Biotechniques; citation_volume=29; citation_number=53; citation_pages=52, 54; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Real-time PCR for mRNA quantitation.; citation_author=ML Wong; citation_author=JF Medrano; citation_journal_title=Biotechniques; citation_volume=39; citation_number=54; citation_pages=75-85; citation_date=2005; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001355" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Conditional Transgenesis Using Dimerizable Cre (DiCre)"/>
    <meta name="twitter:description" content="Cre recombinase is extensively used to engineer the genome of experimental animals. However, its usefulness is still limited by the lack of an efficient temporal control over its activity. We have recently developed a conceptually new approach to regulate Cre recombinase, that we have called Dimerizable Cre or DiCre. It is based on splitting Cre into two inactive moieties and fusing them to FKBP12 (FK506-binding protein) and FRB (binding domain of the FKBP12-rapamycin associated protein), respectively. These latter can be efficiently hetero-dimerized by rapamycin, leading to the reinstatement of Cre activity. We have been able to show, using in vitro approaches, that this ligand-induced dimerization is an efficient way to regulate Cre activity, and presents a low background activity together with a high efficiency of recombination following dimerization. To test the in vivo performance of this system, we have, in the present work, knocked-in DiCre into the Rosa26 locus of mice. To evaluate the performance of the DiCre system, mice have been mated with indicator mice (Z/EG or R26R) and Cre-induced recombination was examined following activation of DiCre by rapamycin during embryonic development or after birth of progenies. No recombination could be observed in the absence of treatment of the animals, indicating a lack of background activity of DiCre in the absence of rapamycin. Postnatal rapamycin treatment (one to five daily injection, 10 mg/kg i.p) induced recombination in a number of different tissues of progenies such as liver, heart, kidney, muscle, etc. On the other hand, recombination was at a very low level following in utero treatment of DiCre&times;R26R mice. In conclusion, DiCre has indeed the potentiality to be used to establish conditional Cre-deleter mice. An added advantage of this system is that, contrary to other modulatable Cre systems, it offers the possibility of obtaining regulated recombination in a combinatorial manner, i.e. induce recombination at any desired time-point specifically in cells characterized by the simultaneous expression of two different promoters."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0001355.g007"/>

  <meta property="og:title" content="Conditional Transgenesis Using Dimerizable Cre (DiCre)" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001355" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=4588'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=2584'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=7667&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0001355">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001355" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001355&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Conditional Transgenesis Using Dimerizable Cre (DiCre)
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Nicolas Jullien, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  ICNE-UMR 6544 Centre National de la Recherche Scientifique (CNRS), Université de la Méditerranée, Marseille, France
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Isabelle Goddard, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Centre de Transgenèse, Faculté de Médecine Nord, Institut Fédératif de Recherche (IFR) Jean-Roche, Marseille, France
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Samia Selmi-Ruby, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  INSERM UMR 664, Faculté de Médecine RTH Laennec, Université Lyon I, Lyon, France
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Jean-Luc Fina, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Centre de Transgenèse, Faculté de Médecine Nord, Institut Fédératif de Recherche (IFR) Jean-Roche, Marseille, France
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Harold Cremer, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  IBDML-UMR 6216 Centre National de la Recherche Scientifique (CNRS), Université de la Méditerranée, Marseille, France
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Jean-Paul Herman
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:jean-paul.herman@univmed.fr">jean-paul.herman@univmed.fr</a></p>

                <p>Affiliation:
                  ICNE-UMR 6544 Centre National de la Recherche Scientifique (CNRS), Université de la Méditerranée, Marseille, France
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: December 26, 2007</li>
    <li>DOI: 10.1371/journal.pone.0001355</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0001355" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0001355" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001355" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0001355" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0001355-g001" data-doi="info:doi/10.1371/journal.pone.0001355" data-uri="info:doi/10.1371/journal.pone.0001355.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001355.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001355-g002" data-doi="info:doi/10.1371/journal.pone.0001355" data-uri="info:doi/10.1371/journal.pone.0001355.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001355.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001355-g003" data-doi="info:doi/10.1371/journal.pone.0001355" data-uri="info:doi/10.1371/journal.pone.0001355.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001355.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001355-g004" data-doi="info:doi/10.1371/journal.pone.0001355" data-uri="info:doi/10.1371/journal.pone.0001355.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001355.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001355-g005" data-doi="info:doi/10.1371/journal.pone.0001355" data-uri="info:doi/10.1371/journal.pone.0001355.g005" title="Figure 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001355.g005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001355-g006" data-doi="info:doi/10.1371/journal.pone.0001355" data-uri="info:doi/10.1371/journal.pone.0001355.g006" title="Figure 6">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001355.g006&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001355-g007" data-doi="info:doi/10.1371/journal.pone.0001355" data-uri="info:doi/10.1371/journal.pone.0001355.g007" title="Figure 7">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001355.g007&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001355">Reader Comments (0)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0001355" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1"></a><p>Cre recombinase is extensively used to engineer the genome of experimental animals. However, its usefulness is still limited by the lack of an efficient temporal control over its activity. We have recently developed a conceptually new approach to regulate Cre recombinase, that we have called Dimerizable Cre or DiCre. It is based on splitting Cre into two inactive moieties and fusing them to FKBP12 (FK506-binding protein) and FRB (binding domain of the FKBP12-rapamycin associated protein), respectively. These latter can be efficiently hetero-dimerized by rapamycin, leading to the reinstatement of Cre activity. We have been able to show, using <em>in vitro</em> approaches, that this ligand-induced dimerization is an efficient way to regulate Cre activity, and presents a low background activity together with a high efficiency of recombination following dimerization. To test the <em>in vivo</em> performance of this system, we have, in the present work, knocked-in DiCre into the Rosa26 locus of mice. To evaluate the performance of the DiCre system, mice have been mated with indicator mice (Z/EG or R26R) and Cre-induced recombination was examined following activation of DiCre by rapamycin during embryonic development or after birth of progenies. No recombination could be observed in the absence of treatment of the animals, indicating a lack of background activity of DiCre in the absence of rapamycin. Postnatal rapamycin treatment (one to five daily injection, 10 mg/kg i.p) induced recombination in a number of different tissues of progenies such as liver, heart, kidney, muscle, etc. On the other hand, recombination was at a very low level following <em>in utero</em> treatment of DiCre×R26R mice. In conclusion, DiCre has indeed the potentiality to be used to establish conditional Cre-deleter mice. An added advantage of this system is that, contrary to other modulatable Cre systems, it offers the possibility of obtaining regulated recombination in a combinatorial manner, i.e. induce recombination at any desired time-point specifically in cells characterized by the simultaneous expression of two different promoters.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>Jullien N, Goddard I, Selmi-Ruby S, Fina J-L, Cremer H, et al.  (2007) Conditional Transgenesis Using Dimerizable Cre (DiCre). PLoS ONE 2(12):
          e1355.
            doi:10.1371/journal.pone.0001355</p><p><strong>Academic Editor: </strong>Patrick Callaerts, Katholieke Universiteit Leuven, Belgium</p><p><strong>Received:</strong> October 1, 2007; <strong>Accepted:</strong> November 27, 2007; <strong>Published:</strong> December 26, 2007</p><p><strong>Copyright:</strong> © 2007 Jullien et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>This work was supported by institutional funding from the CNRS and Université de la Méditerranée. HC has been supported by the European community through the network of excellence NeuroNE and the Association Francaise contre les Myopathies (AFM).</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>The technology of gene inactivation through homologous recombination has, since its introduction in the late eighties, boosted immensely our understanding of the role of a great number of genes. However, it has been progressively realized that the interpretation of the results may be complicated by diverse factors such as differences among various inbred mouse lines, existence of compensatory adaptations, redundancies etc. Moreover, the modifications induced by the inactivation of a given gene at an early developmental stage (the most extreme modification being the lethality induced by the inactivation) may also interfere with, and make often impossible, the evaluation of a possible late role of that gene.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>One way to avoid some of these problems is to have a temporal regulation of the inactivation and be able to induce it at any desired time-point. The introduction of the Cre/LoxP system has made this approach possible. Cre is a site specific recombinase that catalyzes the excision of a DNA segment flanked by two identical short (34 bp) asymmetric sequences, called LoxP, of same orientation, introduced into the genome by the experimenter. When that portion of DNA corresponds to a crucial part of the gene, the excision will lead to its inactivation <a href="#pone.0001355-Sauer1">[1]</a>. Conditional inactivation has become a real possibility when solutions have been worked out for the <em>in vivo</em> regulation of Cre activity <a href="#pone.0001355-Branda1">[2]</a>–<a href="#pone.0001355-Kwan1">[5]</a>.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>A first, “natural” approach to regulate Cre activity, and through it gene inactivation, is to put it under the control of a cell-type specific promoter. In that case one may obtain a spatial (cell-type dependent) regulation of Cre, that can even display a specific temporal pattern if the given promoter is activated only at a certain developmental stage or in a certain physiological situation <a href="#pone.0001355-Zinyk1">[6]</a>–<a href="#pone.0001355-Sclafani1">[8]</a>.</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p>The activity of Cre can also be dependent on an inducer given by the experimenter, and in that case one has the possibility of switching on Cre in any cell type. For that, two basic approaches have been worked out in the last few years: placing Cre under the control of a regulatable promoter or use a fusion protein to obtain a modulation of Cre activity by steroids.</p>
<a id="article1.body1.sec1.p5" name="article1.body1.sec1.p5"></a><p>The first option relies most of the times on the now classical tetracycline-regulatable system worked out by Bujard and his colleagues <a href="#pone.0001355-Utomo1">[9]</a>–<a href="#pone.0001355-Schonig1">[12]</a>. In that case the regulation of Cre is at the transcriptional level, and the reliability of the system is dependent on the tightness of the regulation of the promoter, i.e. its leakage when not activated, and the level of maximal activity after induction. Unfortunately, in many cases these systems are somewhat leaky, which means, for the utilization we are considering, that inactivation of the gene may occur spontaneously, before one would like to induce it experimentally.</p>
<a id="article1.body1.sec1.p6" name="article1.body1.sec1.p6"></a><p>The second approach is based on the fact that the fusion of the hormone-binding domain of the progesterone or estrogen steroid receptors with Cre will make the biological activity of the latter dependent on the steroid in the sense that the absence of the latter keeps the enzyme in a cellular compartment physically separated from the one in which its substrate is located <a href="#pone.0001355-Kellendonk1">[13]</a>–<a href="#pone.0001355-Indra1">[16]</a>. Indeed, in the absence of the steroid, the fusion protein is sequestered in the cytoplasm of the cell by heat-shock proteins, and it is only following the addition of the steroid that the complex dissociates and Cre is able to enter the nucleus where it can combine with its substrate, i.e. chromosomal DNA, and exert its activity. Note that this approach can be combined with the previous one, allowing a combined cell-type dependent and temporal control of recombinase activity <a href="#pone.0001355-Forni1">[17]</a>–<a href="#pone.0001355-Naik1">[19]</a>. However, similar to the previous solution, these systems may be leaky. Moreover, induction of Cre activity within the central nervous system is not very efficient <a href="#pone.0001355-Kellendonk1">[13]</a>, <a href="#pone.0001355-Vooijs1">[20]</a>, <a href="#pone.0001355-Hameyer1">[21]</a>. Last, but not least, the steroids that are used for induction can exert physiological effects of their own <a href="#pone.0001355-Csaba1">[22]</a>, <a href="#pone.0001355-Vancutsem1">[23]</a>.</p>
<a id="article1.body1.sec1.p7" name="article1.body1.sec1.p7"></a><p>We have recently developed a conceptually new approach to regulate Cre recombinase, that we have called Dimerizable Cre or DiCre <a href="#pone.0001355-Jullien1">[24]</a>. DiCre is based on the functional complementation, using ligand-induced dimerization <a href="#pone.0001355-Rivera1">[25]</a>, of two inactive Cre moieties that can be associated by rapamycin through cross-linking the protein fragments FKBP12 (FK506-binding protein; <a href="#pone.0001355-Siekierka1">[26]</a>) and FRB, the ligand binding domain of FRAP (FKBP12-rapamycin associated protein; <a href="#pone.0001355-Chiu1">[27]</a>) linked to the Cre moieties. DiCre has been validated <em>in vitro</em> and shown to be reliable, offering a low background activity and high-level induction resulting in characteristics that were better than that of a steroid-regulated form of Cre. Given its good <em>in vitro</em> performance, it represents a promising new tool that could offer an alternative to existing approaches for conditional transgenesis. Moreover, relative to other approaches, it offers the unique potential, being based on the coexpression of two separate Cre fragments, to achieve “combinatorial” or “intersectional” transgenesis <a href="#pone.0001355-Dymecki1">[4]</a>. The aim of the present work is to validate this tool for such <em>in vivo</em> uses.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3>
<h4>Targeting DiCre</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>DiCre requires the simultaneous expression in the cell of two inactive Cre moieties that can functionally complement each other. In our previous <em>in vitro</em> work, aimed at developing this system, we have examined several combinations of N- and C-terminal Cre moieties, differing in their length and/or in the linkers attaching FKBP12 (or FRB) to them <a href="#pone.0001355-Jullien1">[24]</a>. This characterization indicated that these combinations differ in their background activity and/or the level of activity obtained following dimerization. We assumed that for conditional trangenesis it was important to aim at having the lowest possible background expression level, even at the price of having a somewhat lower induced activity. Thus, among the different combinations tested, the Cre59.F2/Cre60.F2 combination, that in our tests resulted in the lowest background, but still with a reasonably good activation level, was chosen for the <em>in vivo</em> tests of the system. As detailed previously <a href="#pone.0001355-Jullien1">[24]</a>, Cre59.F2 corresponds to the N-terminal moiety (aa 19–59) of Cre to which FKBP12 is fused through the flexible linker F2, while Cre60.F2 corresponds to the C-terminal moiety of Cre (aa 60–343) to which FRB is fused through the same flexible linker F2.</p>
<a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2"></a><p>The expression of DiCre <em>in vivo</em> is complicated by the fact that this system requires the simultaneous expression of two constructs in the organism. With a classical transgenic approach that would imply a relatively tedious mating scheme using three mouse lines to bring the three constructs (the two components of DiCre plus the Cre-target construct) into the same animals. A way to simplify the experimental setup is to obtain a genetic linkage of the two components of DiCre. Rather than using a bicistronic construct and classical transgenesis, an approach that seemed less reliable, we decided, to achieve that aim, to express DiCre <em>in vivo</em> by homologous recombination, introducing a bicistronic construct containing its two components into a known locus. For that, we have chosen the Rosa-26 locus, that has been used extensively for several years to express foreign genes in mice, and for which extensive experience and molecular tools were available <a href="#pone.0001355-Kisseberth1">[28]</a>–<a href="#pone.0001355-Zambrowicz1">[30]</a>.</p>
<a id="article1.body1.sec2.sec1.p3" name="article1.body1.sec2.sec1.p3"></a><p>Details of the targeting construct are given on <a href="#pone-0001355-g001">Figure 1A</a>. Following its successful insertion, the C-terminal component of DiCre is expressed under the control of the Rosa-26 promoter, while the expression of the N-terminal component is controlled by the strong CAG promoter. In light of the ubiquitous expression and the absence of silencing of genes expressed from this locus, documented both during development and following birth <a href="#pone.0001355-Zambrowicz1">[30]</a>, it was felt that this approach would ensure a generalized and reliable expression of DiCre. Note that a second promoter (PGK) with a downstream hygromycin-resistance gene, used for selection of successfully targeted ES cells, will also be present in the locus following targeting. However, this cassette has been removed secondarily from the genome by crossing F1 animals with Flpe-expressing mice <a href="#pone.0001355-Rodriguez1">[31]</a> and is not present in the final lines presented here.</p>
<div class="figure" id="pone-0001355-g001"><div class="img"><a name="pone-0001355-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001355.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001355" data-uri="info:doi/10.1371/journal.pone.0001355.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001355.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001355.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001355.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001355.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001355.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001355.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001355.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001355.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001355.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Targeting strategy for the creation of DiCre ES cells.</span></strong></p><a id="article1.body1.sec2.sec1.fig1.caption1.p1" name="article1.body1.sec2.sec1.fig1.caption1.p1"></a><p>A: Scheme of the targeting vector and of the Rosa26 locus following homologous recombination. “B” and “R” stand, respectively, for the BamHI and EcoRV sites that are used when screening ES cells clones for correct insertions using Southern blots and the probes indicated on the scheme. Fragments amplified when genotyping animals are also represented B: Representative examples of Southern blots obtained from wild type ES cells or clones with correct recombination, using the probes shown on the schemes above.</p>
<span>doi:10.1371/journal.pone.0001355.g001</span></div>

<h4>DiCre activity in targeted ES cells</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>Following the obtention of clones of successfully targeted ES cells and before using them for blastocyst injection to obtain conditional deleter mice, we wished to test that DiCre was functional in these clones. Cells were transfected with an indicator construct, pcDNA3-CALNLZ <a href="#pone.0001355-Kanegae1">[32]</a> that expresses the ß-galactosidase reporter gene in transfected cells conditionally, dependent on the Cre-mediated excision of a neomycin resistance-STOP-pA cassette inserted between it and a CAG promoter (<a href="#pone-0001355-g002">Fig. 2A</a>). Following G418 selection, we obtained several subclones for the DiCre-containing ES clones. As shown on <a href="#pone-0001355-g002">Figure 2B</a>, there was a complete lack of expression of ß-galactosidase in the cells in the absence of inducer, indicating that background activity in these conditions is even lower than that we have observed earlier in fibroblasts. On the other hand, exposure of the cells to the inducer (20 nM of the rapamycin analog AP23102 for three days) led to the appearance of ß-galactosidase activity in close to 100% of undifferentiated cells, indicating that the system does work in ES cells. Note, however, that ß-galactosidase activity was lower or absent in flat cells, presumably engaged in some kind of differentiation pathway and surrounding the compact aggregates of undifferentiated ES cell. This aspect has not been pursued further.</p>
<div class="figure" id="pone-0001355-g002"><div class="img"><a name="pone-0001355-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001355.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001355" data-uri="info:doi/10.1371/journal.pone.0001355.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001355.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001355.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001355.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001355.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001355.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001355.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001355.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001355.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001355.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>DiCre are functional in targeted ES cells.</span></strong></p><a id="article1.body1.sec2.sec2.fig1.caption1.p1" name="article1.body1.sec2.sec2.fig1.caption1.p1"></a><p>A: Scheme of the CALNLZ indicator construct used to test the functionality of Cre. B: ß-galactosidase expression, as revealed by the blue X-Gal reaction product, in ES cells with correct insertion of the DiCre construct into the Rosa26 locus and stable transfection of the pcDNA-CALNLZ construct. Cells are tested without (“Control”) or following exposure, for three days, to 20 nM of the dimerizer AP23102.</p>
<span>doi:10.1371/journal.pone.0001355.g002</span></div><a id="article1.body1.sec2.sec2.p2" name="article1.body1.sec2.sec2.p2"></a><p>Following this validation of the DiCre system, the initial targeted ES cells (i.e. not containing the indicator CALNLZ construct) were used for blastocyst injection. Several founder mice were obtained and used to establish mouse lines (called in the followings DiCre lines) after excision of the hygromycin-resistance cassette through mating of F1 mice with an Flpe-expressing deleter line.</p>


<h4>Characterization of DiCre in adult animals</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>In a first series of test, DiCre mice were mated with Z/EG transgenic indicator mice to obtain double transgenic mice. The Z/EG transgene is similar to the CALNLZ construct depicted on <a href="#pone-0001355-g002">Fig. 2A</a>, except for the stuffer sequence that codes for LacZ and the gene expressed following the excision of the stuffer sequence that is EGFP <a href="#pone.0001355-Novak1">[33]</a>. Thus, cells of these animals are LacZ+ in the absence of recombination and green fluorescent following Cre-mediated excision. Young adult double transgenic progenies resulting from this mating were treated with the inducer (daily injection of 10 mg/kg rapamycin for five days) and sacrificed 5 days after the end of the treatment. In the absence of treatment no fluorescent cells could be detected on tissue sections prepared from a number of tissues (brain, pituitary, heart, lung, liver, kidney, skin, striated muscle). In treated animals EGFP-positive (as confirmed by immunohistochemistry, results not shown) green fluorescent fibers, indicating that recombination has taken place, were present only in the heart, and surprisingly, recombined green fluorescent cells could be detected in no other tissues (<a href="#pone-0001355-g003">Fig. 3</a>). Because of the low number of recombined cells, this switch on of EGFP expression could not be correlated to a significant decrease of the level of X-gal staining in the heart. Note that in untreated animals, strong and definite X-gal staining was observable only in the heart and muscle.</p>
<div class="figure" id="pone-0001355-g003"><div class="img"><a name="pone-0001355-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001355.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001355" data-uri="info:doi/10.1371/journal.pone.0001355.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001355.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001355.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001355.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001355.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001355.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001355.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001355.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001355.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001355.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Induction of Cre-mediated recombination in DiCre×Z/EG mice.</span></strong></p><a id="article1.body1.sec2.sec3.fig1.caption1.p1" name="article1.body1.sec2.sec3.fig1.caption1.p1"></a><p>A: The presence of recombination is shown by EGFP expression in the heart of DiCre×Z/EG adult animals ten days after the end of treatment with the inducer (5×10 mg/kg rapamycin i.p.). B: In the absence of rapamycin treatment no specific fluorescence can be seen.</p>
<span>doi:10.1371/journal.pone.0001355.g003</span></div><a id="article1.body1.sec2.sec3.p2" name="article1.body1.sec2.sec3.p2"></a><p>To control and complement these results using another indicator mouse line, DiCre mice were mated with R26R animals <a href="#pone.0001355-Soriano1">[29]</a> and the existence of recombination examined on tissue sections by X-gal staining. Again, no labeled cells were observed in the absence of treatment. On the other hand, and contrary to what had been observed with the Z/EG indicator line, recombination was widespread and could be observed in a number of tissues after treatment of the in DiCre×R26R animals with the inducer (<a href="#pone-0001355-g004">Fig. 4</a>). Thus, blue cells could be observed in heart, kidney, liver, testis, adrenals, fat tissue (not shown), lung, pituitary (adenohypohysis), spleen. However, the degree of staining differed greatly among these tissues, being quite dense in the liver where up to 30–40% of the cells were blue, intermediate in the heart and kidney, and low (only a few cells per sections) in the other tissues. Moreover, staining was not necessarily uniform in tissues with complex architecture, such as in the kidney where staining was more concentrated in the region of renal papilla and much less in the cortex, in the lung with staining detectable only in the alveolar epithelium, or in the testis, where only interstitial Leydig cells showed staining. No staining could be observed in the muscle, brain, or epidermis. The pattern of recombination was the same when higher doses of rapamycin were used (up to 40 mg/kg, results not shown).</p>
<div class="figure" id="pone-0001355-g004"><div class="img"><a name="pone-0001355-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001355.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001355" data-uri="info:doi/10.1371/journal.pone.0001355.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001355.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001355.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001355.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001355.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001355.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001355.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001355.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001355.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001355.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>Induction of Cre-mediated recombination in various tissues of DiCre×R26R mice.</span></strong></p><a id="article1.body1.sec2.sec3.fig2.caption1.p1" name="article1.body1.sec2.sec3.fig2.caption1.p1"></a><p>The presence of recombination is shown by ß-galactosidase expression, as revealed by the blue X-Gal reaction product, in various tissues of adult DiCre×R26R animals ten days after the end of treatment with the inducer (5×10 mg/kg rapamycin i.p.). Bars represent 100 µm. The insert for the liver shows the total absence of recombination in the absence of rapamycin treatment.</p>
<span>doi:10.1371/journal.pone.0001355.g004</span></div><a id="article1.body1.sec2.sec3.p3" name="article1.body1.sec2.sec3.p3"></a><p>To evaluate the number of rapamycin injections needed to induce DiCre-mediated recombination, mice were treated with different drug regimens, i.e. daily injections of 10 mg/kg rapamycin for one, three or five days. In parallel, higher doses (5×20 or 5×40 mg/kg) of rapamycin were also tested. Animals were sacrificed ten days after the last injection and recombination rate was quantitatively evaluated in the liver using Southern blots. The results indicate that the degree of recombination is equivalent for five or three injections and somewhat lower with a single injection (<a href="#pone-0001355-g005">Fig. 5</a>). It should be noted also that the use of the highest dose of rapamycin (40 mg/kg) seems to increase the degree of recombination.</p>
<div class="figure" id="pone-0001355-g005"><div class="img"><a name="pone-0001355-g005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001355.g005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001355" data-uri="info:doi/10.1371/journal.pone.0001355.g005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001355.g005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001355.g005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001355.g005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001355.g005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001355.g005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001355.g005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001355.g005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001355.g005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001355.g005.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 5.  <span>Dose-dependence of DiCre activation <em>in vivo</em>.</span></strong></p><a id="article1.body1.sec2.sec3.fig3.caption1.p1" name="article1.body1.sec2.sec3.fig3.caption1.p1"></a><p>A: Southern blots, using a LacZ probe, prepared from the liver of adult DiCreR26R animals ten days following 510mgkg, 310mgkg, 110mgkg, 520mgkg or 540mgkg i.p. rapamycin treatment, with two or three animals for each condition. The upper band R corresponds to the R26R allele after recombination, the lower band NR to the nonrecombined allele. B Quantification of the degree of recombination, as estimated by image analysis of the blot shown above.</p>
<span>doi:10.1371/journal.pone.0001355.g005</span></div>

<h4>Characterization of DiCre during development</h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>Although some recombination could be observed in DiCre×R26R progenies following rapamycin treatment during their intrauterine development (E13–14 or E18–19), its level was definitely lower (only a few cells per section) relative to that observed following postnatal treatment and restricted to the liver, kidney and heart (results not shown).</p>
<a id="article1.body1.sec2.sec4.p2" name="article1.body1.sec2.sec4.p2"></a><p>To test whether the absence of recombination in embryos could be due to an insufficient passage of the dimerizer through the placental barrier, primary cultures were prepared from CNS, liver, heart of E15 DiCre×R26R embryos. Cultures were also prepared from the carcass of embryos using protocols described for obtention of Mouse Embryonic Fibroblasts <a href="#pone.0001355-Nagy2">[34]</a> resulting in a mixed primary culture we call “MEF” and that contains, besides fibroblasts, other types of cells, such as myocytes and fused muscle fibers etc. Cre activity was induced by adding 10 nM AP23102 (DiCre×R26R cells) at 3 DIV and LacZ activity was tested four days later. Results were similar to what had been observed <em>in vivo</em>, i.e. some recombined cells could be observed among hepatocytes or cardiomyocytes obtained from DiCre×R26R embryos, but their proportion was very low (<a href="#pone-0001355-g006">Fig. 6</a>.).</p>
<div class="figure" id="pone-0001355-g006"><div class="img"><a name="pone-0001355-g006" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001355.g006&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001355" data-uri="info:doi/10.1371/journal.pone.0001355.g006"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001355.g006&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001355.g006/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001355.g006/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001355.g006/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001355.g006/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001355.g006.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001355.g006/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001355.g006/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001355.g006.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 6.  <span>Induction of Cre-mediated recombination in primary cultures of embryonic tissues of DiCre×R26R mice.</span></strong></p><a id="article1.body1.sec2.sec4.fig1.caption1.p1" name="article1.body1.sec2.sec4.fig1.caption1.p1"></a><p>The presence of recombination is shown by ß-galactosidase expression, as revealed by the blue X-Gal reaction product on the bright-field (upper row) and phase contrast (lower row) images of the same fields, in primary cultures prepared from various tissues of DiCre×R26R E15 embryos. Cultures were exposed between 3 and 7 day <em>in vitro</em> to 10 nM AP23102.</p>
<span>doi:10.1371/journal.pone.0001355.g006</span></div>

<h4>Expression of DiCre</h4>
<a id="article1.body1.sec2.sec5.p1" name="article1.body1.sec2.sec5.p1"></a><p>Evaluation of the level of transcripts, using Northern blots (<a href="#pone-0001355-g007">Fig. 7A</a>) or qPCR (<a href="#pone-0001355-g007">Fig. 7B</a>), in ES cells clones used to establish the DiCre mouse lines or in different tissues from young adult DiCre mice indicated that the level of transcription of Cre60.F2 from the Rosa26 promoter was relatively homogeneous and at comparable levels among these different tissues, except for the brain in which transcription seemed to be somewhat lower. For Cre59.F2 the level of expression, driven by the CAG promoter, was considerably higher than that mediated by the Rosa26 promoter for Cre60.F2. However, while for the Cre59.F2 transcript levels in the liver and ES cells were similar, the levels observed in the skin, kidney and brain were considerably lower. In tissues from E15 embryos, the level of Cre60.F2 transcription was diminished relative to the adult levels and transcription of Cre59.F2, controlled by the CAG promoter, was also found to be considerably decreased (<a href="#pone-0001355-g007">Fig. 7C and 7D</a>).</p>
<div class="figure" id="pone-0001355-g007"><div class="img"><a name="pone-0001355-g007" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001355.g007&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001355" data-uri="info:doi/10.1371/journal.pone.0001355.g007"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001355.g007&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001355.g007/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001355.g007/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001355.g007/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001355.g007/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001355.g007.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001355.g007/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001355.g007/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001355.g007.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 7.  <span>Expression of the DiCre constructs in tissues of adult or embryonic transgenic animals.</span></strong></p><a id="article1.body1.sec2.sec5.fig1.caption1.p1" name="article1.body1.sec2.sec5.fig1.caption1.p1"></a><p>A: Northern blots obtained from DiCre ES cells or tissues from young adult DiCre transgenic animals (two independent samples for each), indicating the level of mRNA for the indicated transgenes. B: Expression level, relative to that measured in the liver, of the same transgenes in tissues of adult animals, as measured by qPCR (means +/− SEM, n = 3–5). C: Northern blot, and D: qPCR analysis, as in panels 7A and 7B, from embryonic (E15) tissues and adult tissues used as comparison. Note that the adult tissues used here were not the same as those shown on panels 7A and 7B.</p>
<span>doi:10.1371/journal.pone.0001355.g007</span></div>
</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p>The aim of the present work was to assess the functionality of DiCre <em>in vivo</em> and its ability to be used to establish conditional deleter mice. Our results indicate that DiCre has indeed the potential to be used for conditional transgenesis and that DiCre mice could complement, and represent an alternative to, already existing approaches.</p>
<a id="article1.body1.sec3.p2" name="article1.body1.sec3.p2"></a><p>To be used for a general deleter strain, a modulatable Cre should fulfill several conditions. The two most important ones are, first, to have a low background activity, i.e. to induce no recombination in the absence of the inducer, and second, to lead, upon treatment with the inducer, to recombination in the different tissues of the animal. Additional conditions are that this induced recombination should be general within and among tissues, be independent of the developmental stage of the animals, and finally, that the inducer should not exert physiological effects on its own. It should be stressed that none of the existing systems to date fulfill all these criteria. Indeed, they often display some background activity, and their activation is usually mosaic, especially at later stages of development or in postnatal animals. Moreover, the most often used inducer, tamoxifen, has developmental and endocrine effects on its own that may present problems in some experimental situations <a href="#pone.0001355-Csaba1">[22]</a>, <a href="#pone.0001355-Vancutsem1">[23]</a>.</p>
<a id="article1.body1.sec3.p3" name="article1.body1.sec3.p3"></a><p>Results of the present study show that DiCre fulfills the first two conditions. Thus, we have not observed any recombination in the absence of treatment of the animals with rapamycin in the different tissues examined. This finding is in line with the fact that the DiCre variant that we have used to establish the DiCre mouse line corresponds to one that displayed a background activity among the lowest in our previous <em>in vitro</em> studies <a href="#pone.0001355-Jullien1">[24]</a>, and that it had no discernable background activity in ES cells (present study). Treatment with rapamycin, on the other hand, leads to recombination, albeit at different levels, in a number of tissues. On the basis of these findings, it can be concluded that DiCre represents a potential alternative for the creation of general deleter mouse strains.</p>
<a id="article1.body1.sec3.p4" name="article1.body1.sec3.p4"></a><p>It should be noted that a major advantage of DiCre over existing approaches is that, given its mechanism of action, its use may allow to create more complex experimental schemes. Indeed, using cellular promoters to control activity of Cre is a frequently used approach to restrict recombination to cells in which that promoter is active. In the case of DiCre, it will be possible to place each Cre moieties under the control of two different cellular promoters, targeting recombination to cells in which these two promoters are simultaneously active, and, moreover, superpose a temporal regulation to this targeting. The possibility of this kind of “combinatorial” or “intersectional” recombination, without the temporal regulation that DiCre allows however, was evocated in the case of the alpha-complementation of Cre fragments worked out a few years ago <a href="#pone.0001355-Casanova1">[35]</a>, but, contrary to DiCre, this system could not be shown to work <em>in vivo</em> (E. Casanova, personal communication). Note that a variant of this system, based on a leucine-zipper mediated unconditional complementation of Cre fragments has recently been shown to work <em>in vivo</em> <a href="#pone.0001355-Xu1">[36]</a>. However, comparison of this system with DiCre is not straightforward, as its performance has been documented only for the pancreas, where it led to a mosaic recombination. Moreover, given the unconditional character of the complementation, the recombination observed reflects recombination occuring during embryonic development, while DiCre could not be efficiently activated <em>in utero</em> (see also below).</p>
<a id="article1.body1.sec3.p5" name="article1.body1.sec3.p5"></a><p>While the present data demonstrate that DiCre works <em>in vivo</em>, they also show that induction of DiCre presents several aspects, related to the conditions listed above, that should be improved to ameliorate the system and make it more performant.</p>
<a id="article1.body1.sec3.p6" name="article1.body1.sec3.p6"></a><p>A first point is that the level of recombination brought about by the activation of DiCre was found to be highly variable both between and within tissues. This finding is not specific for DiCre, as it has been noted for all Cre constructs used to establish deleter mice. Thus, in similar experiments done with another deleter line expressing a tamoxifen-regulated Cre construct, we have found in adult animals a very similar pattern of recombination using the Cre-indicator lines used also in the present study (Jullien et al., submitted). This general variability might be related to the heterogeneity, between different tissues or in different cell populations of a given organ, in the characteristics of the indicator construct in its genomic context, such as a variable expression or silencing of the indicator transgene determined by neighboring host sequences, different local chromatin structures determining accessibility of the recombination target, etc. The same factors may also explain that different recombination patterns can be observed depending on the Cre indicator strain used. This influence of strains has been shown here by the major difference between our results obtained using the Z/EG and the R26R lines, but it has also been documented by other groups <a href="#pone.0001355-Vooijs1">[20]</a>, <a href="#pone.0001355-Hameyer1">[21]</a>, <a href="#pone.0001355-Seibler1">[37]</a>, <a href="#pone.0001355-Miyoshi1">[38]</a>.</p>
<a id="article1.body1.sec3.p7" name="article1.body1.sec3.p7"></a><p>An additional factor influencing the level of recombination (although not determining it strictly, as shown by the lack of clear correlation between the level of DiCre expression and the degree of recombination found in the different tissues, compare <a href="#pone-0001355-g004">Figs. 4</a>. and <a href="#pone-0001355-g007">7</a>.) is the level of expression of the specific Cre construct used. This aspect may have a more important role in the case of DiCre, given that activation of DiCre is a ternary reaction involving rapamycin and the FKBP- and FRB-fused Cre moieties. Thus, it seems likely that to obtain an activity level comparable to that achieved using e.g. a classical steroid-activated version of Cre, a higher intracellular concentration of the DiCre components should be attained. This is especially true for the rate-limiting moiety of DiCre, which, in the present setup, corresponds to the Cre60.F2 fragment driven by the relatively weak Rosa26 promoter and that displays the lower level of transcription relative to the other moiety (see <a href="#pone-0001355-g007">Fig 7</a>.). This hypothesis might be explored in the future through the use of a promoter stronger than the Rosa26 promoter. A second parameter specific to the Cre constructs and that determines, in conjunction with the level of expression, the success of recombination, is its intrinsic activity. Thus, the different DiCre variants that we have tested previously <em>in vitro</em> have been observed to have different intrinsic activities <a href="#pone.0001355-Jullien1">[24]</a>. As mentioned above, the variant tested here (Cre59.F2/Cre60.F2) possesses an intrinsic activity that is lower than that of other variants, but was chosen nevertheless for its low background activity and good activation ratio. Using a DiCre variant with a higher intrinsic activity (such as the Cre104.F5/Cre106.F5 combination, see <a href="#pone.0001355-Jullien1">[24]</a>) might lead to a higher recombination efficacy <em>in vivo</em>, and that option will also be explored in the future. Note that using another DiCre variant presents the risk of having a somewhat higher background activity. In fact, there should be an interplay between the strength of the promoter and the DiCre variant used and it may well be that several combinations will have to be explored to find an optimal combination. Thus, the presently used variant, as controlled by the weak Rosa26 promoter displayed no background <em>in vivo</em>, while in our previous <em>in vitro</em> studies, when it was controlled by a strong viral promoter (CMV) it had some, even if quite low, background activity.</p>
<a id="article1.body1.sec3.p8" name="article1.body1.sec3.p8"></a><p>Finally, one might also invoke a possible variable distribution of rapamycin to explain the observed differences between tissues. However, insofar that the drug is known to be rapidly and evenly distributed among tissues <a href="#pone.0001355-GallantHaidner1">[39]</a>, <a href="#pone.0001355-Napoli1">[40]</a>, this factor is unlikely to play a significant role in the observed variability.</p>
<a id="article1.body1.sec3.p9" name="article1.body1.sec3.p9"></a><p>DiCre-induced recombination during embryonic development was found to be very weak, if not absent, in most tissues. Our Northern blot analysis indicates that the level of expression of DiCre in embryonic tissues, especially its component controlled by the CAG promoter, is significantly lower than that found in young adults. The reasons of this are not clear, but it may explain the lack of recombination that we have observed both <em>in vivo</em> and in the <em>ex vivo</em> tissue culture conditions, and use of other promoters to control DiCre expression might solve this problem too. Note, however, that other factors might also contribute to the bad efficacy of DiCre in embryos. Thus, the bioavailability of rapamycin might be at an insufficient level in the embryos. Rapamycin is known to cross the placental barrier <a href="#pone.0001355-Liu1">[41]</a>, but the level it reaches in the embryo is not known. However, our <em>ex vivo</em> culture experiment indicates that, while this factor may play a role, it may not be the most important and factors specific to the embryonic tissue might be involved. Thus, level of cellular uptake in embryonic cells might be not sufficient to ensure a high enough level of the drug within cells or, alternatively, the drug might be titrated out by the high endogenous FKBP12 level existing in embryos that could also, moreover, compete out the FKBP-linked Cre moiety (<a href="#pone.0001355-Yazawa1">[42]</a>; personal observations). The validity of these hypotheses will have to be evaluated in future experiments.</p>
<a id="article1.body1.sec3.p10" name="article1.body1.sec3.p10"></a><p>Besides these potential problems more specific to embryos, rapamycin presents other problems too. Indeed, this drug has, through its interaction with endogenous FRAP, numerous actions of its own: immunosuppression, interference with cell division, teratogenicity <a href="#pone.0001355-Hentges1">[43]</a>, <a href="#pone.0001355-Easton1">[44]</a>. Given these pharmacological actions, use of rapamycin analogs (rapalogs) devoid of such side effects would be highly desirable. Several such analogs, based on modifications of the FRB binding region of rapamycin, have been developed in the last few years <a href="#pone.0001355-Bayle1">[45]</a>–<a href="#pone.0001355-Clackson1">[47]</a>. They have in common not to recognize the endogenous target of rapamycin (FRAP), but to recognize a mutated form of this protein, such as the T2098L mutant used for the elaboration of DiCre, allowing its dimerization to FKBP12. While this mutation conserves the capacity of FRB to be dimerized also by rapamycin, it binds with high affinity these rapalogs, such as the AP23102 or AP21967 compounds that we have used for our <em>in vitro</em> characterization of the system. We have shown that these two drugs, while being good inducers of DiCre, have indeed less side effects insofar that they do not influence, contrary to rapamycin, cell division <em>in vitro</em>. This suggested to us that for <em>in vivo</em> uses they could represent a good alternative, devoid of adverse side effects, to rapamycin. Unfortunately, preliminary experiments have shown us that these two rapalogs are inefficient <em>in vivo</em>, as treatment of DiCre×R26R animals, be it during development or postnatally, induced, even at relatively high doses, considerably less recombination than rapamycin, due, maybe, to unfavorable pharmacokinetics of these drugs. Note that another rapalog, iRap, that is closely related to AP21967 <a href="#pone.0001355-Bayle1">[45]</a>, was found to activate DiCre in DiCre×R26R mice with the same efficacy as rapamycin (results not shown). However, insofar as it displays an affinity towards wild-type FRB that is comparable to that it has for the mutated form of FRB, it is likely to have the same side effects as rapamycin, and therefore cannot really be considered as a suitable option for the replacement of rapamycin.</p>
<a id="article1.body1.sec3.p11" name="article1.body1.sec3.p11"></a><p>Thus, alternative rapalogs will have to be tested to bypass the toxicity of rapamycin and improve these drug-related aspects of DiCre. A first option is to continue to explore drugs developed according to the logic presented above, i.e. drugs able to dimerize FKBP12 and a mutated form of FRB. A possible alternative is to modify the dimerizable partners fused to the Cre moieties themselves and use a mutated FKBP12 only, such as the F36V mutant, in conjunction with homodimerizer drugs that do not interfere with endogenous FKBP12 but have good pharmacokinetic characteristics <a href="#pone.0001355-Clackson1">[47]</a>–<a href="#pone.0001355-Iuliucci1">[49]</a>. Note that, due to the inability of these drugs to bind endogenous FKBP12, the latter option may present the added advantage of having a better efficacy during embryonic development (see above). It should be underlined, nevertheless, that while rapamycin seems to be quite toxic during development and in neonates, it seems to be well tolerated in older animals where, moreover, only one to three injections are required to activate DiCre, and thus it could be used, given appropriate controls, as an inducer of DiCre in adult animals.</p>
<a id="article1.body1.sec3.p12" name="article1.body1.sec3.p12"></a><p>In conclusion, DiCre represents a valuable new option to establish deleter mouse lines and achieve conditional transgenesis. It has interesting and unique characteristics, such as the possibility of achieving regulated combinatorial recombination. Future studies should help ameliorating it and make it a generally usable tool, in particular during development.</p>
</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Methods"></a><h3>Methods</h3>
<h4>Plasmid construction</h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p>The targeting construct is depicted on <a href="#pone-0001355-g001">Figures 1A</a>. It was constructed by standard cloning techniques <a href="#pone.0001355-Ausabel1">[50]</a> and based on the pRosa26.1 targeting vector, donated by Dr. P. Soriano, for insertion into the Rosa26 locus. Cassettes inserted into this vector were the Cre60.F2 and Cre59.F2 sequences for the expression of DiCre <a href="#pone.0001355-Jullien1">[24]</a>, a hygromycin resistance cassette extracted from the p1710 vector (gift of Dr. J. Majors, St.Louis, MO), the PGK promoter from pRosa26.1, and the CAG promoter extracted from the pDRIVE-CAG vector (InVivogen, Toulouse, France). The adenoviral splice acceptor sequence was recovered from the pSA-ßgeo vector <a href="#pone.0001355-Friedrich1">[51]</a> donated by Dr. P. Soriano, the polyadenylation signals (SV40 pA downstream of the Cre59.F2 sequence and BGHpA downstream of the Cre60.F2 sequence) from pcDNA3.1 (Invitrogen, Cergy Pontoise, France).</p>
<a id="article1.body1.sec4.sec1.p2" name="article1.body1.sec4.sec1.p2"></a><p>The pcDNA-CALNLZ vector was constructed by inserting the CALNLZ cassette <a href="#pone.0001355-Kanegae1">[32]</a> into pcDNA3.1 in antiparallel orientation.</p>


<h4>ES cell culture and manipulation</h4>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1"></a><p>Procedures for the manipulation of ES cells were based on published protocols <a href="#pone.0001355-Nagy2">[34]</a>. R1 ES cells <a href="#pone.0001355-Nagy3">[52]</a> were grown on mitomycin C-arrested STO fibroblasts expressing the neomycin and hygromycin resistance genes (gift of Dr. E. Robertson, San Diego, CA) in DMEM supplemented with 15% FCS and leukemia inhibitory factor (ESGRO, 1000 U/ml, Chemicon, Chandlers Ford, UK). Following electroporation of 10<sup>7</sup> cells with 20 µg of the linearized targeting construct, selection was performed with 150 µg/ml hygromycin B for 8 days. Individual drug resistant clones were picked manually and transferred to 96-well microtiter plates, expanded and split. One of the resulting plates was frozen down and screening of clones was performed by Southern blots on the other plate, following digestion of isolated genomic DNA with BamHI or EcoRV and using “Rosa26-5′”, a 502 bp probe, located upstream of the insert and amplified from the Rosa26 locus using the primers 5′-GATAGGAACTGGAAAACCAGAGGA-3′ and 5′-GTAAGGGTCCAACAGAAAAGAGA-3′ (<a href="#pone-0001355-g001">Fig. 1A</a>.). A secondary Southern blot analysis, done on the positive clones of the first screen, allowed evaluating the integrity (using “Rosa26-3′”, a 709 bp probe located at the 3′ border of the insert) and unicity (using “Cre”, a 732 bp probe internal to Cre) of the insert. Location of the restriction site and probes, as well as a representative example of Southern blots, are indicated on <a href="#pone-0001355-g001">Fig. 1</a>. Correct clones were re-thawn, expanded and frozen down till their further use. Chimeras were produced by injection of ES cells into blastocysts obtained from C57Bl/6 mice (Charles Rivers, Lyon, France)</p>
<a id="article1.body1.sec4.sec2.p2" name="article1.body1.sec4.sec2.p2"></a><p>To test functionality of the regulatable Cre constructs inserted into the Rosa-26 locus, recombined ES cell clones were re-transformed by electroporation with the linearized pcDNA-CALNLZ construct, selected with 250 µg/ml G418 for one week and subcloned before testing.</p>


<h4>Primary culture</h4>
<a id="article1.body1.sec4.sec3.p1" name="article1.body1.sec4.sec3.p1"></a><p>Primary cultures were prepared form E15 embryos. Dissected liver and CNS tissues were digested 15 mins at 37°C in trypsin-EDTA, heart 30 mins at 37°C in trypsin-EDTA followed by 30 mins in collagenase. Following digestion, tissues were mechanically dissociated in culture medium containing 0.01% DNAse and the cells were plated onto polylysine-coated coverslips in DMEM/F12 (liver and neural cultures) or DMEM (heart cells) containing 10% FCS. Mixed “MEF” cultures were prepared from the carcass of the embryos digested for 45 mins in trypsin-EDTA with intermittent shaking, dissociated and plated onto polylysine-coated coverslips in DMEM with 10% FCS. Cells were grown in 5% CO<sub>2</sub>:95% air at 37°C for three days before addition of the inducers, grown for further 4 days and fixed using 2% paraformaldehyde in phosphate buffer (0.1M, pH 7.4). ß-Galactosidase activity was revealed by incubating the fixed cells in PBS containing 1 mg/ml X-Gal, 5 mM potassium ferricyanide, 5 mM potassium ferrocyanide, and 2 mM MgCI<sub>2</sub>.</p>


<h4>Animal handling</h4>
<a id="article1.body1.sec4.sec4.p1" name="article1.body1.sec4.sec4.p1"></a><p>Animals were maintained in standard conditions, with food and water <em>ad libitum</em> and a 12 h:12 h dark:light photoperiod. The hygromycin cassette from F1 recombinant DiCre mice was eliminated by crossing with FLPe mice <a href="#pone.0001355-Rodriguez1">[31]</a>. Mice were then backcrossed for 2 generations onto C57Bl/6 background and then, to facilitate subsequent characterization, intercrossed to obtain homozygote lines. Mice were genotyped for the presence of the transgene by PCR on tail DNA obtained by the “hot-shot” method <a href="#pone.0001355-Truett1">[53]</a>.</p>
<a id="article1.body1.sec4.sec4.p2" name="article1.body1.sec4.sec4.p2"></a><p>Homozygote recombinant mice were mated with heterozygote Z/EG <a href="#pone.0001355-Novak1">[33]</a> or homozygote R26R <a href="#pone.0001355-Soriano1">[29]</a> Cre indicator mice. Progenies born from mating with Z/EG mice were genotyped for the presence of the Z/EG transgene by PCR on tail DNA.</p>
<a id="article1.body1.sec4.sec4.p3" name="article1.body1.sec4.sec4.p3"></a><p>Rapamycin (Sigma, St. Louis, MO, or LC Labs, Woburn, MA) or the rapamycin analogs AP-23102 or AP-21967 (Ariad Inc., Boston, MS) were dissolved in N,N-dimethylacetamide (DMA) and then diluted in a mixture of 4% DMA, 10% polythylene glycol (average MW 400) and 17% polyoxyethylen sorbitane monooleate (Tween 20). Injections were given intraperitoneally (i.p.).</p>


<h4>Histology</h4>
<a id="article1.body1.sec4.sec5.p1" name="article1.body1.sec4.sec5.p1"></a><p>Animals were deeply anaesthetized and sacrificed by transcardiac perfusion of saline followed by 2% paraformaldehyde in phosphate buffer (0.1 M, pH 7.4). When X-gal staining was to be performed, the fixative contained 1 mM EDTA and 2 mM MgCl<sub>2</sub>. Following 3–5 h postfixation at room temperature, tissues were cryoprotected overnight at 4°C in 30% sucrose and frozen down for preparation of 25 µm cryostate sections. EGFP expression was examined by direct inspection of sections or following immunostaining using a polyclonal anti-GFP antibody (Molecular Probes, Invitrogen, 1/5000) followed by a secondary, Alexa Fluor 594-labelled, anti-rabbit antibody (Molecular Probes, Invitrogen, 1/600). ß-Galactosidase expression was revealed by incubation of the section in PBS containing 1 mg/ml X-Gal, 5 mM potassium ferricyanide, 5 mM potassium ferrocyanide, 2 mM MgCI<sub>2</sub>, 0.01% sodium deoxycholate, and 0.02% Nonidet P-40.</p>


<h4>Northern blots and qPCR</h4>
<a id="article1.body1.sec4.sec6.p1" name="article1.body1.sec4.sec6.p1"></a><p>Five µg total RNA, extracted by Trizol (Invitrogen) for tissues or RNEasy (Qiagen) for cultured cells, were loaded onto a formaldehyde/MOPS agarose gel, and transferred after migration onto Hybond N+ membrane (Amersham). The Rosa26 driven Cre60.F2 transcript was revealed by hybridization with a 732 bp digoxigenin-labeled probe specific for C terminal Cre moiety; the CAG promoter-driven Cre59.F2 transcript was hybridized with a 363 bp digoxigenin-labelled fragment of the Cre59.F2 CDS. Hybridization, washing and revelation using CDP* were adapted from Roche's DIG membrane protocols.</p>
<a id="article1.body1.sec4.sec6.p2" name="article1.body1.sec4.sec6.p2"></a><p>Reverse transcription of 1 µg RNA was done using SuperScript II and random primers as indicated by the manufacturer (Invitrogen). qPCR was done using the SYBR-Green PCR master mix (Applied Biosystems) and run onto an ABI Prism 7700 station. Specific primers were designed with AmplifX (<a href="http://ifrjr.nord.univ-mrs.fr/AmplifX">http://ifrjr.nord.univ-mrs.fr/AmplifX</a>). Calculations were done by the ΔΔCt method <a href="#pone.0001355-Wong1">[54]</a> using 18S RNA as reference, after having determined, using dilution series, that amplification efficiencies were homogeneous for the different primer pairs and very close to 100%.</p>


<h4>Southern blots</h4>
<a id="article1.body1.sec4.sec7.p1" name="article1.body1.sec4.sec7.p1"></a><p>To quantify the degree of recombination of the R26R reporter allele, genomic DNA was prepared from tissues following proteinase K digestion, using standard procedures <a href="#pone.0001355-Ausabel1">[50]</a>. Following digestion by HindIII, that was determined previously to allow discrimination of recombined and non-recombined R26R alleles, migration and transfer to Hybond N+ membranes, Southern blots were completed using a digoxigenin-labelled 890 bp for LacZ. Blots were digitalized using a ChemiGenius II station (Syngene, Cambridge, UK) and quantification of the native 16 bits gray level files was performed using GeneTools (Syngene).</p>

</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>We thank Ms. C. Gautron and C. Taverna for their secretarial support. We thank Dr. P. Soriano (Fred Hutchinson Cancer Research Center, Seattle, WA) for his gift of the Rosa26.1 and pSA-ßgeo plasmids as well as of the R26R mice, Dr. E. Robertson (Salk Institute, San Diego, CA) for STO cells, Dr. J. Majors (Washington University, St. Louis, MO) for the p1710 retroviral backbone, Dr. I. Saito (University of Tokyo, Tokyo, Japan) for pCALNLZ. The rapamycin analogs AP-23102 and AP-21967 were provided by Ariad Pharmaceutical (Cambridge, MA.; <a href="http://www.ariad.com/regulationkits">www.ariad.com/regulationkits</a>), iRap by Dr. T. Wandless (Stanford University, Stanford, CA).</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: HC JH NJ. Performed the experiments: HC JH NJ IG JF SS. Analyzed the data: JH NJ. Wrote the paper: JH NJ.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0001355-Sauer1" id="pone.0001355-Sauer1"></a>Sauer B (1993) Manipulation of transgenes by site-specific recombination: use of Cre recombinase. Methods Enzymol  225: 890–900.  <ul class="find" data-citedArticleID="1033934" data-doi="10.1016/0076-6879(93)25056-8"><li><a href="http://dx.doi.org/10.1016/0076-6879(93)25056-8" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Manipulation+of+transgenes+by+site-specific+recombination%3A+use+of+Cre+recombinase." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Manipulation+of+transgenes+by+site-specific+recombination%3A+use+of+Cre+recombinase.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0001355-Branda1" id="pone.0001355-Branda1"></a>Branda CS, Dymecki SM (2004) Talking about a revolution: The impact of site-specific recombinases on genetic analyses in mice. Dev Cell  6: 7–28.  <ul class="find" data-citedArticleID="1033868" data-doi="10.1016/s1534-5807(03)00399-x"><li><a href="http://dx.doi.org/10.1016/s1534-5807(03)00399-x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Talking+about+a+revolution%3A+The+impact+of+site-specific+recombinases+on+genetic+analyses+in+mice." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Talking+about+a+revolution%3A+The+impact+of+site-specific+recombinases+on+genetic+analyses+in+mice.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0001355-Nagy1" id="pone.0001355-Nagy1"></a>Nagy A (2000) Cre recombinase: the universal reagent for genome tailoring. Genesis  26: 99–109.  <ul class="find" data-citedArticleID="1033916" data-doi="10.1002/(sici)1526-968x(200002)26:2<99::aid-gene1&gt;3.3.co;2-2"><li><a href="http://dx.doi.org/10.1002/(sici)1526-968x(200002)26:2<99::aid-gene1&gt;3.3.co;2-2" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Cre+recombinase%3A+the+universal+reagent+for+genome+tailoring." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Cre+recombinase%3A+the+universal+reagent+for+genome+tailoring.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0001355-Dymecki1" id="pone.0001355-Dymecki1"></a>Dymecki SM, Kim JC (2007) Molecular neuroanatomy's “Three Gs”: a primer. Neuron  54: 17–34.  <ul class="find" data-citedArticleID="1033878" data-doi="10.1016/j.neuron.2007.03.009"><li><a href="http://dx.doi.org/10.1016/j.neuron.2007.03.009" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Molecular+neuroanatomy%27s+%E2%80%9CThree+Gs%E2%80%9D%3A+a+primer." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Molecular+neuroanatomy%27s+%E2%80%9CThree+Gs%E2%80%9D%3A+a+primer.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0001355-Kwan1" id="pone.0001355-Kwan1"></a>Kwan KM (2002) Conditional alleles in mice: practical considerations for tissue-specific knockouts. Genesis  32: 49–62.  <ul class="find" data-citedArticleID="1033910" data-doi="10.1002/gene.10068"><li><a href="http://dx.doi.org/10.1002/gene.10068" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Conditional+alleles+in+mice%3A+practical+considerations+for+tissue-specific+knockouts." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Conditional+alleles+in+mice%3A+practical+considerations+for+tissue-specific+knockouts.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0001355-Zinyk1" id="pone.0001355-Zinyk1"></a>Zinyk DL, Mercer EH, Harris E, Anderson DJ, Joyner AL (1998) Fate mapping of the mouse midbrain-hindbrain constriction using a site-specific recombination system. Curr Biol  8: 665–668.  <ul class="find" data-citedArticleID="1033968" data-doi="10.1016/s0960-9822(98)70255-6"><li><a href="http://dx.doi.org/10.1016/s0960-9822(98)70255-6" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Fate+mapping+of+the+mouse+midbrain-hindbrain+constriction+using+a+site-specific+recombination+system." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Fate+mapping+of+the+mouse+midbrain-hindbrain+constriction+using+a+site-specific+recombination+system.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0001355-Tsien1" id="pone.0001355-Tsien1"></a>Tsien JZ, Chen DF, Gerber D, Tom C, Mercer EH, et al.  (1996) Subregion- and cell type-restricted gene knockout in mouse brain. Cell  87: 1317–1326.  <ul class="find" data-citedArticleID="1033950" data-doi="10.1016/s0092-8674(00)81826-7"><li><a href="http://dx.doi.org/10.1016/s0092-8674(00)81826-7" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Subregion-+and+cell+type-restricted+gene+knockout+in+mouse+brain." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Subregion-+and+cell+type-restricted+gene+knockout+in+mouse+brain.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0001355-Sclafani1" id="pone.0001355-Sclafani1"></a>Sclafani AM, Skidmore JM, Ramaprakash H, Trumpp A, Gage PJ, et al.  (2006) Nestin-Cre mediated deletion of Pitx2 in the mouse. Genesis  44: 336–344.  <ul class="find" data-citedArticleID="1033938" data-doi="10.1002/dvg.20220"><li><a href="http://dx.doi.org/10.1002/dvg.20220" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Nestin-Cre+mediated+deletion+of+Pitx2+in+the+mouse." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Nestin-Cre+mediated+deletion+of+Pitx2+in+the+mouse.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0001355-Utomo1" id="pone.0001355-Utomo1"></a>Utomo AR, Nikitin AY, Lee WH (1999) Temporal, spatial, and cell type-specific control of Cre-mediated DNA recombination in transgenic mice. Nat Biotechnol  17: 1091–1096.  <ul class="find" data-citedArticleID="1033952"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Temporal%2C+spatial%2C+and+cell+type-specific+control+of+Cre-mediated+DNA+recombination+in+transgenic+mice.&amp;auth=&amp;atitle=Temporal%2C+spatial%2C+and+cell+type-specific+control+of+Cre-mediated+DNA+recombination+in+transgenic+mice." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Temporal%2C+spatial%2C+and+cell+type-specific+control+of+Cre-mediated+DNA+recombination+in+transgenic+mice." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Temporal%2C+spatial%2C+and+cell+type-specific+control+of+Cre-mediated+DNA+recombination+in+transgenic+mice.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0001355-Gossen1" id="pone.0001355-Gossen1"></a>Gossen M, Bujard H (2002) Studying gene function in eukaryotes by conditional gene inactivation. Annu Rev Genet  36: 153–173.  <ul class="find" data-citedArticleID="1033890"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Studying+gene+function+in+eukaryotes+by+conditional+gene+inactivation.&amp;auth=&amp;atitle=Studying+gene+function+in+eukaryotes+by+conditional+gene+inactivation." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Studying+gene+function+in+eukaryotes+by+conditional+gene+inactivation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Studying+gene+function+in+eukaryotes+by+conditional+gene+inactivation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0001355-StOnge1" id="pone.0001355-StOnge1"></a>St-Onge L, Furth PA, Gruss P (1996) Temporal control of the Cre recombinase in transgenic mice by a tetracycline responsive promoter. Nucleic Acids Res  24: 3875–3877.  <ul class="find" data-citedArticleID="1033946" data-doi="10.1093/nar/24.19.3875"><li><a href="http://dx.doi.org/10.1093/nar/24.19.3875" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Temporal+control+of+the+Cre+recombinase+in+transgenic+mice+by+a+tetracycline+responsive+promoter." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Temporal+control+of+the+Cre+recombinase+in+transgenic+mice+by+a+tetracycline+responsive+promoter.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0001355-Schonig1" id="pone.0001355-Schonig1"></a>Schonig K, Schwenk F, Rajewsky K, Bujard H (2002) Stringent doxycycline dependent control of CRE recombinase in vivo. Nucleic Acids Res  30: e134.  <ul class="find" data-citedArticleID="1033936" data-doi="10.1093/nar/gnf134"><li><a href="http://dx.doi.org/10.1093/nar/gnf134" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Stringent+doxycycline+dependent+control+of+CRE+recombinase+in+vivo." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Stringent+doxycycline+dependent+control+of+CRE+recombinase+in+vivo.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0001355-Kellendonk1" id="pone.0001355-Kellendonk1"></a>Kellendonk C, Tronche F, Casanova E, Anlag K, Opherk C, et al.  (1999) Inducible site-specific recombination in the brain. J Mol Biol  285: 175–182.  <ul class="find" data-citedArticleID="1033904" data-doi="10.1006/jmbi.1998.2307"><li><a href="http://dx.doi.org/10.1006/jmbi.1998.2307" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Inducible+site-specific+recombination+in+the+brain." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Inducible+site-specific+recombination+in+the+brain.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0001355-Feil1" id="pone.0001355-Feil1"></a>Feil R, Brocard J, Mascrez B, LeMeur M, Metzger D, et al.  (1996) Ligand-activated site-specific recombination in mice. Proc Natl Acad Sci U S A  93: 10887–10890.  <ul class="find" data-citedArticleID="1033882" data-doi="10.1073/pnas.93.20.10887"><li><a href="http://dx.doi.org/10.1073/pnas.93.20.10887" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Ligand-activated+site-specific+recombination+in+mice." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Ligand-activated+site-specific+recombination+in+mice.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0001355-Kellendonk2" id="pone.0001355-Kellendonk2"></a>Kellendonk C, Tronche F, Monaghan AP, Angrand PO, Stewart F, et al.  (1996) Regulation of Cre recombinase activity by the synthetic steroid RU 486. Nucleic Acids Res  24: 1404–1411.  <ul class="find" data-citedArticleID="1033906" data-doi="10.1093/nar/24.8.1404"><li><a href="http://dx.doi.org/10.1093/nar/24.8.1404" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Regulation+of+Cre+recombinase+activity+by+the+synthetic+steroid+RU+486." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Regulation+of+Cre+recombinase+activity+by+the+synthetic+steroid+RU+486.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0001355-Indra1" id="pone.0001355-Indra1"></a>Indra AK, Warot X, Brocard J, Bornert JM, Xiao JH, et al.  (1999) Temporally-controlled site-specific mutagenesis in the basal layer of the epidermis: comparison of the recombinase activity of the tamoxifen-inducible Cre-ER(T) and Cre-ER(T2) recombinases. Nucleic Acids Res  27: 4324–4327.  <ul class="find" data-citedArticleID="1033896" data-doi="10.1093/nar/27.22.4324"><li><a href="http://dx.doi.org/10.1093/nar/27.22.4324" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Temporally-controlled+site-specific+mutagenesis+in+the+basal+layer+of+the+epidermis%3A+comparison+of+the+recombinase+activity+of+the+tamoxifen-inducible+Cre-ER%28T%29+and+Cre-ER%28T2%29+recombinases." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Temporally-controlled+site-specific+mutagenesis+in+the+basal+layer+of+the+epidermis%3A+comparison+of+the+recombinase+activity+of+the+tamoxifen-inducible+Cre-ER%28T%29+and+Cre-ER%28T2%29+recombinases.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0001355-Forni1" id="pone.0001355-Forni1"></a>Forni PE, Scuoppo C, Imayoshi I, Taulli R, Dastru W, et al.  (2006) High levels of Cre expression in neuronal progenitors cause defects in brain development leading to microencephaly and hydrocephaly. J Neurosci  26: 9593–9602.  <ul class="find" data-citedArticleID="1033884" data-doi="10.1523/jneurosci.2815-06.2006"><li><a href="http://dx.doi.org/10.1523/jneurosci.2815-06.2006" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=High+levels+of+Cre+expression+in+neuronal+progenitors+cause+defects+in+brain+development+leading+to+microencephaly+and+hydrocephaly." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22High+levels+of+Cre+expression+in+neuronal+progenitors+cause+defects+in+brain+development+leading+to+microencephaly+and+hydrocephaly.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0001355-Zervas1" id="pone.0001355-Zervas1"></a>Zervas M, Millet S, Ahn S, Joyner AL (2004) Cell behaviors and genetic lineages of the mesencephalon and rhombomere 1. Neuron  43: 345–357.  <ul class="find" data-citedArticleID="1033966" data-doi="10.1016/j.neuron.2004.07.010"><li><a href="http://dx.doi.org/10.1016/j.neuron.2004.07.010" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Cell+behaviors+and+genetic+lineages+of+the+mesencephalon+and+rhombomere+1." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Cell+behaviors+and+genetic+lineages+of+the+mesencephalon+and+rhombomere+1.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0001355-Naik1" id="pone.0001355-Naik1"></a>Naik K, Pittman It, Wolfe A, Miller RS, Radovick S, et al.  (2006) A novel technique for temporally regulated cell type-specific Cre expression and recombination in the pituitary gonadotroph. J Mol Endocrinol  37: 63–69.  <ul class="find" data-citedArticleID="1033922" data-doi="10.1677/jme.1.02053"><li><a href="http://dx.doi.org/10.1677/jme.1.02053" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+novel+technique+for+temporally+regulated+cell+type-specific+Cre+expression+and+recombination+in+the+pituitary+gonadotroph." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+novel+technique+for+temporally+regulated+cell+type-specific+Cre+expression+and+recombination+in+the+pituitary+gonadotroph.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0001355-Vooijs1" id="pone.0001355-Vooijs1"></a>Vooijs M, Jonkers J, Berns A (2001) A highly efficient ligand-regulated Cre recombinase mouse line shows that LoxP recombination is position dependent. EMBO Rep  2: 292–297.  <ul class="find" data-citedArticleID="1033956" data-doi="10.1093/embo-reports/kve064"><li><a href="http://dx.doi.org/10.1093/embo-reports/kve064" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+highly+efficient+ligand-regulated+Cre+recombinase+mouse+line+shows+that+LoxP+recombination+is+position+dependent." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+highly+efficient+ligand-regulated+Cre+recombinase+mouse+line+shows+that+LoxP+recombination+is+position+dependent.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0001355-Hameyer1" id="pone.0001355-Hameyer1"></a>Hameyer D, Loonstra A, Eshkind L, Schmitt S, Antunes C, et al.  (2007) Toxicity of ligand-dependant Cre-recombinases and generation of a conditional Cre-deleter mouse allowing mosaic recombination in peripheral tissues. Physiol Genomics.   <ul class="find" data-citedArticleID="1033892" data-doi="10.1152/physiolgenomics.00019.2007"><li><a href="http://dx.doi.org/10.1152/physiolgenomics.00019.2007" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Toxicity+of+ligand-dependant+Cre-recombinases+and+generation+of+a+conditional+Cre-deleter+mouse+allowing+mosaic+recombination+in+peripheral+tissues." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Toxicity+of+ligand-dependant+Cre-recombinases+and+generation+of+a+conditional+Cre-deleter+mouse+allowing+mosaic+recombination+in+peripheral+tissues.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0001355-Csaba1" id="pone.0001355-Csaba1"></a>Csaba G, Karabelyos C (2001) The effect of a single neonatal treatment (hormonal imprinting) with the antihormones, tamoxifen and mifepristone on the sexual behavior of adult rats. Pharmacol Res  43: 531–534.  <ul class="find" data-citedArticleID="1033876" data-doi="10.1006/phrs.2001.0818"><li><a href="http://dx.doi.org/10.1006/phrs.2001.0818" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+effect+of+a+single+neonatal+treatment+%28hormonal+imprinting%29+with+the+antihormones%2C+tamoxifen+and+mifepristone+on+the+sexual+behavior+of+adult+rats." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+effect+of+a+single+neonatal+treatment+%28hormonal+imprinting%29+with+the+antihormones%2C+tamoxifen+and+mifepristone+on+the+sexual+behavior+of+adult+rats.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0001355-Vancutsem1" id="pone.0001355-Vancutsem1"></a>Vancutsem PM, Roessler ML (1997) Neonatal treatment with tamoxifen causes immediate alterations of the sexually dimorphic nucleus of the preoptic area and medial preoptic area in male rats. Teratology  56: 220–228.  <ul class="find" data-citedArticleID="1033954" data-doi="10.1002/(sici)1096-9926(199709)56:3<220::aid-tera5&gt;3.0.co;2-4"><li><a href="http://dx.doi.org/10.1002/(sici)1096-9926(199709)56:3<220::aid-tera5&gt;3.0.co;2-4" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Neonatal+treatment+with+tamoxifen+causes+immediate+alterations+of+the+sexually+dimorphic+nucleus+of+the+preoptic+area+and+medial+preoptic+area+in+male+rats." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Neonatal+treatment+with+tamoxifen+causes+immediate+alterations+of+the+sexually+dimorphic+nucleus+of+the+preoptic+area+and+medial+preoptic+area+in+male+rats.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0001355-Jullien1" id="pone.0001355-Jullien1"></a>Jullien N, Sampieri F, Enjalbert A, Herman JP (2003) Regulation of Cre recombinase by ligand-induced complementation of inactive fragments. Nucleic Acids Res  31: E131.  <ul class="find" data-citedArticleID="1033900" data-doi="10.1093/nar/gng131"><li><a href="http://dx.doi.org/10.1093/nar/gng131" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Regulation+of+Cre+recombinase+by+ligand-induced+complementation+of+inactive+fragments." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Regulation+of+Cre+recombinase+by+ligand-induced+complementation+of+inactive+fragments.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0001355-Rivera1" id="pone.0001355-Rivera1"></a>Rivera VM, Clackson T, Natesan S, Pollock R, Amara JF, et al.  (1996) A humanized system for pharmacologic control of gene expression. Nat Med  2: 1028–1032.  <ul class="find" data-citedArticleID="1033930" data-doi="10.1038/nm0996-1028"><li><a href="http://dx.doi.org/10.1038/nm0996-1028" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+humanized+system+for+pharmacologic+control+of+gene+expression." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+humanized+system+for+pharmacologic+control+of+gene+expression.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0001355-Siekierka1" id="pone.0001355-Siekierka1"></a>Siekierka JJ, Hung SH, Poe M, Lin CS, Sigal NH (1989) A cytosolic binding protein for the immunosuppressant FK506 has peptidyl-prolyl isomerase activity but is distinct from cyclophilin. Nature  341: 755–757.  <ul class="find" data-citedArticleID="1033942" data-doi="10.1038/341755a0"><li><a href="http://dx.doi.org/10.1038/341755a0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+cytosolic+binding+protein+for+the+immunosuppressant+FK506+has+peptidyl-prolyl+isomerase+activity+but+is+distinct+from+cyclophilin." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+cytosolic+binding+protein+for+the+immunosuppressant+FK506+has+peptidyl-prolyl+isomerase+activity+but+is+distinct+from+cyclophilin.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0001355-Chiu1" id="pone.0001355-Chiu1"></a>Chiu MI, Katz H, Berlin V (1994) RAPT1, a mammalian homolog of yeast Tor, interacts with the FKBP12/rapamycin complex. Proc Natl Acad Sci U S A  91: 12574–12578.  <ul class="find" data-citedArticleID="1033872" data-doi="10.1073/pnas.91.26.12574"><li><a href="http://dx.doi.org/10.1073/pnas.91.26.12574" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=RAPT1%2C+a+mammalian+homolog+of+yeast+Tor%2C+interacts+with+the+FKBP12%2Frapamycin+complex." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22RAPT1%2C+a+mammalian+homolog+of+yeast+Tor%2C+interacts+with+the+FKBP12%2Frapamycin+complex.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0001355-Kisseberth1" id="pone.0001355-Kisseberth1"></a>Kisseberth WC, Brettingen NT, Lohse JK, Sandgren EP (1999) Ubiquitous expression of marker transgenes in mice and rats. Dev Biol  214: 128–138.  <ul class="find" data-citedArticleID="1033908" data-doi="10.1006/dbio.1999.9417"><li><a href="http://dx.doi.org/10.1006/dbio.1999.9417" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Ubiquitous+expression+of+marker+transgenes+in+mice+and+rats." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Ubiquitous+expression+of+marker+transgenes+in+mice+and+rats.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0001355-Soriano1" id="pone.0001355-Soriano1"></a>Soriano P (1999) Generalized lacZ expression with the ROSA26 Cre reporter strain. Nat Genet  21: 70–71.  <ul class="find" data-citedArticleID="1033944"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Generalized+lacZ+expression+with+the+ROSA26+Cre+reporter+strain.&amp;auth=&amp;atitle=Generalized+lacZ+expression+with+the+ROSA26+Cre+reporter+strain." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Generalized+lacZ+expression+with+the+ROSA26+Cre+reporter+strain." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Generalized+lacZ+expression+with+the+ROSA26+Cre+reporter+strain.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0001355-Zambrowicz1" id="pone.0001355-Zambrowicz1"></a>Zambrowicz BP, Imamoto A, Fiering S, Herzenberg LA, Kerr WG, et al.  (1997) Disruption of overlapping transcripts in the ROSA beta geo 26 gene trap strain leads to widespread expression of beta-galactosidase in mouse embryos and hematopoietic cells. Proc Natl Acad Sci U S A  94: 3789–3794.  <ul class="find" data-citedArticleID="1033964" data-doi="10.1073/pnas.94.8.3789"><li><a href="http://dx.doi.org/10.1073/pnas.94.8.3789" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Disruption+of+overlapping+transcripts+in+the+ROSA+beta+geo+26+gene+trap+strain+leads+to+widespread+expression+of+beta-galactosidase+in+mouse+embryos+and+hematopoietic+cells." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Disruption+of+overlapping+transcripts+in+the+ROSA+beta+geo+26+gene+trap+strain+leads+to+widespread+expression+of+beta-galactosidase+in+mouse+embryos+and+hematopoietic+cells.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0001355-Rodriguez1" id="pone.0001355-Rodriguez1"></a>Rodriguez CI, Buchholz F, Galloway J, Sequerra R, Kasper J, et al.  (2000) High-efficiency deleter mice show that FLPe is an alternative to Cre-loxP. Nat Genet  25: 139–140.  <ul class="find" data-citedArticleID="1033932"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=High-efficiency+deleter+mice+show+that+FLPe+is+an+alternative+to+Cre-loxP.&amp;auth=&amp;atitle=High-efficiency+deleter+mice+show+that+FLPe+is+an+alternative+to+Cre-loxP." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=High-efficiency+deleter+mice+show+that+FLPe+is+an+alternative+to+Cre-loxP." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22High-efficiency+deleter+mice+show+that+FLPe+is+an+alternative+to+Cre-loxP.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0001355-Kanegae1" id="pone.0001355-Kanegae1"></a>Kanegae Y, Lee G, Sato Y, Tanaka M, Nakai M, et al.  (1995) Efficient gene activation in mammalian cells by using recombinant adenovirus expressing site-specific Cre recombinase. Nucleic Acids Res  23: 3816–3821.  <ul class="find" data-citedArticleID="1033902" data-doi="10.1093/nar/23.19.3816"><li><a href="http://dx.doi.org/10.1093/nar/23.19.3816" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Efficient+gene+activation+in+mammalian+cells+by+using+recombinant+adenovirus+expressing+site-specific+Cre+recombinase." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Efficient+gene+activation+in+mammalian+cells+by+using+recombinant+adenovirus+expressing+site-specific+Cre+recombinase.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0001355-Novak1" id="pone.0001355-Novak1"></a>Novak A, Guo C, Yang W, Nagy A, Lobe CG (2000) Z/EG, a double reporter mouse line that expresses enhanced green fluorescent protein upon Cre-mediated excision. Genesis  28: 147–155.  <ul class="find" data-citedArticleID="1033926" data-doi="10.1002/1526-968x(200011/12)28:3/4<147::aid-gene90&gt;3.3.co;2-7"><li><a href="http://dx.doi.org/10.1002/1526-968x(200011/12)28:3/4<147::aid-gene90&gt;3.3.co;2-7" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Z%2FEG%2C+a+double+reporter+mouse+line+that+expresses+enhanced+green+fluorescent+protein+upon+Cre-mediated+excision." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Z%2FEG%2C+a+double+reporter+mouse+line+that+expresses+enhanced+green+fluorescent+protein+upon+Cre-mediated+excision.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0001355-Nagy2" id="pone.0001355-Nagy2"></a>Nagy A, Gertsenstein M, Vintersten K, Behringer R (2003) Manipulating the Mouse Embryo. A Laboratory Manual. Cold Spring Harbor, N.Y.: Cold Spring Harbor Laboratory Press.   <ul class="find-nolinks"></ul></li><li><span class="label">35.
              </span><a name="pone.0001355-Casanova1" id="pone.0001355-Casanova1"></a>Casanova E, Lemberger T, Fehsenfeld S, Mantamadiotis T, Schutz G (2003) alpha Complementation in the Cre recombinase enzyme. Genesis  37: 25–29.  <ul class="find" data-citedArticleID="1033870" data-doi="10.1002/gene.10227"><li><a href="http://dx.doi.org/10.1002/gene.10227" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=alpha+Complementation+in+the+Cre+recombinase+enzyme." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22alpha+Complementation+in+the+Cre+recombinase+enzyme.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">36.
              </span><a name="pone.0001355-Xu1" id="pone.0001355-Xu1"></a>Xu Y, Xu G, Liu B, Gu G (2007) Cre reconstitution allows for DNA recombination selectively in dual-marker-expressing cells in transgenic mice. Nucleic Acids Res  35: e126.  <ul class="find" data-citedArticleID="1033960" data-doi="10.1093/nar/gkm559"><li><a href="http://dx.doi.org/10.1093/nar/gkm559" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Cre+reconstitution+allows+for+DNA+recombination+selectively+in+dual-marker-expressing+cells+in+transgenic+mice." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Cre+reconstitution+allows+for+DNA+recombination+selectively+in+dual-marker-expressing+cells+in+transgenic+mice.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">37.
              </span><a name="pone.0001355-Seibler1" id="pone.0001355-Seibler1"></a>Seibler J, Zevnik B, Kuter-Luks B, Andreas S, Kern H, et al.  (2003) Rapid generation of inducible mouse mutants. Nucleic Acids Res  31: e12.  <ul class="find" data-citedArticleID="1033940" data-doi="10.1093/nar/gng012"><li><a href="http://dx.doi.org/10.1093/nar/gng012" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Rapid+generation+of+inducible+mouse+mutants." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Rapid+generation+of+inducible+mouse+mutants.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">38.
              </span><a name="pone.0001355-Miyoshi1" id="pone.0001355-Miyoshi1"></a>Miyoshi G, Fishell G (2006) Directing neuron-specific transgene expression in the mouse CNS. Curr Opin Neurobiol  16: 577–584.  <ul class="find" data-citedArticleID="1033914" data-doi="10.1016/j.conb.2006.08.013"><li><a href="http://dx.doi.org/10.1016/j.conb.2006.08.013" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Directing+neuron-specific+transgene+expression+in+the+mouse+CNS." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Directing+neuron-specific+transgene+expression+in+the+mouse+CNS.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">39.
              </span><a name="pone.0001355-GallantHaidner1" id="pone.0001355-GallantHaidner1"></a>Gallant-Haidner HL, Trepanier DJ, Freitag DG, Yatscoff RW (2000) Pharmacokinetics and metabolism of sirolimus. Ther Drug Monit  22: 31–35.  <ul class="find" data-citedArticleID="1033888" data-doi="10.1097/00007691-200002000-00006"><li><a href="http://dx.doi.org/10.1097/00007691-200002000-00006" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Pharmacokinetics+and+metabolism+of+sirolimus." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Pharmacokinetics+and+metabolism+of+sirolimus.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">40.
              </span><a name="pone.0001355-Napoli1" id="pone.0001355-Napoli1"></a>Napoli KL, Wang ME, Stepkowski SM, Kahan BD (1997) Distribution of sirolimus in rat tissue. Clin Biochem  30: 135–142.  <ul class="find" data-citedArticleID="1033924" data-doi="10.1016/s0009-9120(96)00157-9"><li><a href="http://dx.doi.org/10.1016/s0009-9120(96)00157-9" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Distribution+of+sirolimus+in+rat+tissue." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Distribution+of+sirolimus+in+rat+tissue.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">41.
              </span><a name="pone.0001355-Liu1" id="pone.0001355-Liu1"></a>Liu KJ, Arron JR, Stankunas K, Crabtree GR, Longaker MT (2007) Chemical rescue of cleft palate and midline defects in conditional GSK-3beta mice. Nature  446: 79–82.  <ul class="find" data-citedArticleID="1033912" data-doi="10.1038/nature05557"><li><a href="http://dx.doi.org/10.1038/nature05557" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Chemical+rescue+of+cleft+palate+and+midline+defects+in+conditional+GSK-3beta+mice." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Chemical+rescue+of+cleft+palate+and+midline+defects+in+conditional+GSK-3beta+mice.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">42.
              </span><a name="pone.0001355-Yazawa1" id="pone.0001355-Yazawa1"></a>Yazawa S, Obata K, Iio A, Koide M, Yokota M, et al.  (2003) Gene expression of FK506-binding proteins 12.6 and 12 during chicken development. Comp Biochem Physiol A Mol Integr Physiol  136: 391–399.  <ul class="find" data-citedArticleID="1033962" data-doi="10.1016/s1095-6433(03)00197-1"><li><a href="http://dx.doi.org/10.1016/s1095-6433(03)00197-1" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Gene+expression+of+FK506-binding+proteins+12.6+and+12+during+chicken+development." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Gene+expression+of+FK506-binding+proteins+12.6+and+12+during+chicken+development.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">43.
              </span><a name="pone.0001355-Hentges1" id="pone.0001355-Hentges1"></a>Hentges KE, Sirry B, Gingeras AC, Sarbassov D, Sonenberg N, et al.  (2001) FRAP/mTOR is required for proliferation and patterning during embryonic development in the mouse. Proc Natl Acad Sci U S A  98: 13796–13801.  <ul class="find" data-citedArticleID="1033894" data-doi="10.1073/pnas.241184198"><li><a href="http://dx.doi.org/10.1073/pnas.241184198" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=FRAP%2FmTOR+is+required+for+proliferation+and+patterning+during+embryonic+development+in+the+mouse." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22FRAP%2FmTOR+is+required+for+proliferation+and+patterning+during+embryonic+development+in+the+mouse.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">44.
              </span><a name="pone.0001355-Easton1" id="pone.0001355-Easton1"></a>Easton JB, Houghton PJ (2004) Therapeutic potential of target of rapamycin inhibitors. Expert Opin Ther Targets  8: 551–564.  <ul class="find" data-citedArticleID="1033880" data-doi="10.1517/14728222.8.6.551"><li><a href="http://dx.doi.org/10.1517/14728222.8.6.551" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Therapeutic+potential+of+target+of+rapamycin+inhibitors." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Therapeutic+potential+of+target+of+rapamycin+inhibitors.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">45.
              </span><a name="pone.0001355-Bayle1" id="pone.0001355-Bayle1"></a>Bayle JH, Grimley JS, Stankunas K, Gestwicki JE, Wandless TJ, et al.  (2006) Rapamycin analogs with differential binding specificity permit orthogonal control of protein activity. Chem Biol  13: 99–107.  <ul class="find" data-citedArticleID="1033866" data-doi="10.1016/j.chembiol.2005.10.017"><li><a href="http://dx.doi.org/10.1016/j.chembiol.2005.10.017" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Rapamycin+analogs+with+differential+binding+specificity+permit+orthogonal+control+of+protein+activity." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Rapamycin+analogs+with+differential+binding+specificity+permit+orthogonal+control+of+protein+activity.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">46.
              </span><a name="pone.0001355-Pollock1" id="pone.0001355-Pollock1"></a>Pollock R, Clackson T (2002) Dimerizer-regulated gene expression. Curr Opin Biotechnol  13: 459–467.  <ul class="find" data-citedArticleID="1033928" data-doi="10.1016/s0958-1669(02)00373-7"><li><a href="http://dx.doi.org/10.1016/s0958-1669(02)00373-7" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Dimerizer-regulated+gene+expression." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Dimerizer-regulated+gene+expression.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">47.
              </span><a name="pone.0001355-Clackson1" id="pone.0001355-Clackson1"></a>Clackson T, Yang W, Rozamus LW, Hatada M, Amara JF, et al.  (1998) Redesigning an FKBP-ligand interface to generate chemical dimerizers with novel specificity. Proc Natl Acad Sci U S A  95: 10437–10442.  <ul class="find" data-citedArticleID="1033874" data-doi="10.1073/pnas.95.18.10437"><li><a href="http://dx.doi.org/10.1073/pnas.95.18.10437" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Redesigning+an+FKBP-ligand+interface+to+generate+chemical+dimerizers+with+novel+specificity." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Redesigning+an+FKBP-ligand+interface+to+generate+chemical+dimerizers+with+novel+specificity.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">48.
              </span><a name="pone.0001355-Banaszynski1" id="pone.0001355-Banaszynski1"></a>Banaszynski LA, Chen LC, Maynard-Smith LA, Ooi AG, Wandless TJ (2006) A rapid, reversible, and tunable method to regulate protein function in living cells using synthetic small molecules. Cell  126: 995–1004.  <ul class="find" data-citedArticleID="1033864" data-doi="10.1016/j.cell.2006.07.025"><li><a href="http://dx.doi.org/10.1016/j.cell.2006.07.025" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+rapid%2C+reversible%2C+and+tunable+method+to+regulate+protein+function+in+living+cells+using+synthetic+small+molecules." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+rapid%2C+reversible%2C+and+tunable+method+to+regulate+protein+function+in+living+cells+using+synthetic+small+molecules.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">49.
              </span><a name="pone.0001355-Iuliucci1" id="pone.0001355-Iuliucci1"></a>Iuliucci JD, Oliver SD, Morley S, Ward C, Ward J, et al.  (2001) Intravenous safety and pharmacokinetics of a novel dimerizer drug, AP1903, in healthy volunteers. J Clin Pharmacol  41: 870–879.  <ul class="find" data-citedArticleID="1033898" data-doi="10.1177/00912700122010771"><li><a href="http://dx.doi.org/10.1177/00912700122010771" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Intravenous+safety+and+pharmacokinetics+of+a+novel+dimerizer+drug%2C+AP1903%2C+in+healthy+volunteers." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Intravenous+safety+and+pharmacokinetics+of+a+novel+dimerizer+drug%2C+AP1903%2C+in+healthy+volunteers.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">50.
              </span><a name="pone.0001355-Ausabel1" id="pone.0001355-Ausabel1"></a>Ausabel FM, Brent R, Kingston RE, Moore DD, Seidman JG, et al.  (2007) Current Protocols in Molecular Biology. New York: John Wiley.   <ul class="find-nolinks"></ul></li><li><span class="label">51.
              </span><a name="pone.0001355-Friedrich1" id="pone.0001355-Friedrich1"></a>Friedrich G, Soriano P (1991) Promoter traps in embryonic stem cells: a genetic screen to identify and mutate developmental genes in mice. Genes Dev  5: 1513–1523.  <ul class="find" data-citedArticleID="1033886" data-doi="10.1101/gad.5.9.1513"><li><a href="http://dx.doi.org/10.1101/gad.5.9.1513" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Promoter+traps+in+embryonic+stem+cells%3A+a+genetic+screen+to+identify+and+mutate+developmental+genes+in+mice." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Promoter+traps+in+embryonic+stem+cells%3A+a+genetic+screen+to+identify+and+mutate+developmental+genes+in+mice.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">52.
              </span><a name="pone.0001355-Nagy3" id="pone.0001355-Nagy3"></a>Nagy A, Rossant J, Nagy R, Abramow-Newerly W, Roder JC (1993) Derivation of completely cell culture-derived mice from early-passage embryonic stem cells. Proc Natl Acad Sci U S A  90: 8424–8428.  <ul class="find" data-citedArticleID="1033920" data-doi="10.1073/pnas.90.18.8424"><li><a href="http://dx.doi.org/10.1073/pnas.90.18.8424" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Derivation+of+completely+cell+culture-derived+mice+from+early-passage+embryonic+stem+cells." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Derivation+of+completely+cell+culture-derived+mice+from+early-passage+embryonic+stem+cells.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">53.
              </span><a name="pone.0001355-Truett1" id="pone.0001355-Truett1"></a>Truett GE, Heeger P, Mynatt RL, Truett AA, Walker JA, et al.  (2000) Preparation of PCR-quality mouse genomic DNA with hot sodium hydroxide and tris (HotSHOT). Biotechniques  29: 52, 54.  <ul class="find" data-citedArticleID="1033948"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Preparation+of+PCR-quality+mouse+genomic+DNA+with+hot+sodium+hydroxide+and+tris+%28HotSHOT%29.&amp;auth=&amp;atitle=Preparation+of+PCR-quality+mouse+genomic+DNA+with+hot+sodium+hydroxide+and+tris+%28HotSHOT%29." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Preparation+of+PCR-quality+mouse+genomic+DNA+with+hot+sodium+hydroxide+and+tris+%28HotSHOT%29." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Preparation+of+PCR-quality+mouse+genomic+DNA+with+hot+sodium+hydroxide+and+tris+%28HotSHOT%29.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">54.
              </span><a name="pone.0001355-Wong1" id="pone.0001355-Wong1"></a>Wong ML, Medrano JF (2005) Real-time PCR for mRNA quantitation. Biotechniques  39: 75–85.  <ul class="find" data-citedArticleID="1033958" data-doi="10.2144/05391rv01"><li><a href="http://dx.doi.org/10.2144/05391rv01" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Real-time+PCR+for+mRNA+quantitation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Real-time+PCR+for+mRNA+quantitation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001355.XML" value="103064"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001355.PDF" value="436563"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001355.g001.PNG_L" value="333178"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001355.g001.PNG_M" value="107854"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001355.g001.PNG_S" value="9816"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001355.g001.TIF" value="705202"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001355.g001.PNG_I" value="38172"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001355.g002.PNG_L" value="2183106"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001355.g002.PNG_M" value="185569"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001355.g002.PNG_S" value="18409"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001355.g002.TIF" value="2913146"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001355.g002.PNG_I" value="161877"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001355.g003.PNG_L" value="4021365"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001355.g003.PNG_M" value="266613"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001355.g003.PNG_S" value="16873"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001355.g003.TIF" value="5490078"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001355.g003.PNG_I" value="165392"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001355.g004.PNG_L" value="7379775"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001355.g004.PNG_M" value="332064"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001355.g004.PNG_S" value="12803"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001355.g004.TIF" value="9928504"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001355.g004.PNG_I" value="95966"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001355.g005.PNG_L" value="572416"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001355.g005.PNG_M" value="49899"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001355.g005.PNG_S" value="6829"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001355.g005.TIF" value="763520"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001355.g005.PNG_I" value="16671"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001355.g006.PNG_L" value="6495190"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001355.g006.PNG_M" value="289834"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001355.g006.PNG_S" value="10246"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001355.g006.TIF" value="9067370"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001355.g006.PNG_I" value="86137"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001355.g007.PNG_L" value="393944"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001355.g007.PNG_M" value="61557"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001355.g007.PNG_S" value="10072"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001355.g007.TIF" value="431814"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001355.g007.PNG_I" value="49023"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001355&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001355" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001355&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0001355&volume=&issue=&title=Conditional Transgenesis Using Dimerizable Cre (DiCre)&author_name=Nicolas%20Jullien%2C%20Isabelle%20Goddard%2C%20Samia%20Selmi-Ruby%2C%20Jean-Luc%20Fina%2C%20Harold%20Cremer%2C%20Jean-Paul%20Herman&start_page=1&end_page=11" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001355" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001355" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0001355" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001355&amp;t=Conditional%20Transgenesis%20Using%20Dimerizable%20Cre%20(DiCre)" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001355&title=Conditional%20Transgenesis%20Using%20Dimerizable%20Cre%20(DiCre)&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001355&amp;title=Conditional%20Transgenesis%20Using%20Dimerizable%20Cre%20(DiCre)" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001355" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0001355&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Conditional Transgenesis Using Dimerizable Cre (DiCre)';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0001355';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Conditional%20Transgenesis%20Using%20Dimerizable%20Cre%20(DiCre) http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001355" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0001355" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">














          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Cell+staining%22" title="Search for articles in the subject area:'Cell staining'"><div class="flagText">Cell staining</div></a>
              <div data-categoryid="17401" data-articleid="25540"
                   data-categoryname="Cell staining"
                   class="flagImage" title="Flag 'Cell staining' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Embryos%22" title="Search for articles in the subject area:'Embryos'"><div class="flagText">Embryos</div></a>
              <div data-categoryid="17321" data-articleid="25540"
                   data-categoryname="Embryos"
                   class="flagImage" title="Flag 'Embryos' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Gene+targeting%22" title="Search for articles in the subject area:'Gene targeting'"><div class="flagText">Gene targeting</div></a>
              <div data-categoryid="17701" data-articleid="25540"
                   data-categoryname="Gene targeting"
                   class="flagImage" title="Flag 'Gene targeting' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Heart%22" title="Search for articles in the subject area:'Heart'"><div class="flagText">Heart</div></a>
              <div data-categoryid="47373" data-articleid="25540"
                   data-categoryname="Heart"
                   class="flagImage" title="Flag 'Heart' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Kidneys%22" title="Search for articles in the subject area:'Kidneys'"><div class="flagText">Kidneys</div></a>
              <div data-categoryid="47351" data-articleid="25540"
                   data-categoryname="Kidneys"
                   class="flagImage" title="Flag 'Kidneys' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Recombination+reactions%22" title="Search for articles in the subject area:'Recombination reactions'"><div class="flagText">Recombination reactions</div></a>
              <div data-categoryid="19659" data-articleid="25540"
                   data-categoryname="Recombination reactions"
                   class="flagImage" title="Flag 'Recombination reactions' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Southern+blot%22" title="Search for articles in the subject area:'Southern blot'"><div class="flagText">Southern blot</div></a>
              <div data-categoryid="49093" data-articleid="25540"
                   data-categoryname="Southern blot"
                   class="flagImage" title="Flag 'Southern blot' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Transgenic+animals%22" title="Search for articles in the subject area:'Transgenic animals'"><div class="flagText">Transgenic animals</div></a>
              <div data-categoryid="33751" data-articleid="25540"
                   data-categoryname="Transgenic animals"
                   class="flagImage" title="Flag 'Transgenic animals' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=4505'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=3299'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=1907&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>


</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
