

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Structural Insights into the Evolution of a Non-Biological Protein: Importance of Surface Residues in Protein Fold Optimization</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0000467"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0000467" />

    <meta name="citation_title" content="Structural Insights into the Evolution of a Non-Biological Protein: Importance of Surface Residues in Protein Fold Optimization"/>
    <meta itemprop="name" content="Structural Insights into the Evolution of a Non-Biological Protein: Importance of Surface Residues in Protein Fold Optimization"/>

      <meta name="citation_author" content="Matthew D. Smith"/>
            <meta name="citation_author_institution" content="Center for BioOptical Nanotechnology, The Biodesign Institute, Arizona State University, Tempe, Arizona, United States of America"/>
            <meta name="citation_author_institution" content="Department of Chemistry and Biochemistry, Arizona State University, Tempe, Arizona, United States of America"/>
      <meta name="citation_author" content="Matthew A. Rosenow"/>
            <meta name="citation_author_institution" content="Center for BioOptical Nanotechnology, The Biodesign Institute, Arizona State University, Tempe, Arizona, United States of America"/>
            <meta name="citation_author_institution" content="Department of Chemistry and Biochemistry, Arizona State University, Tempe, Arizona, United States of America"/>
      <meta name="citation_author" content="Meitian Wang"/>
            <meta name="citation_author_institution" content="Department of Chemistry and Biochemistry, Arizona State University, Tempe, Arizona, United States of America"/>
      <meta name="citation_author" content="James P. Allen"/>
            <meta name="citation_author_institution" content="Department of Chemistry and Biochemistry, Arizona State University, Tempe, Arizona, United States of America"/>
      <meta name="citation_author" content="Jack W. Szostak"/>
            <meta name="citation_author_institution" content="Howard Hughes Medical Institute, Department of Molecular Biology, Massachusetts General Hospital, Boston, Massachusetts, United States of America"/>
      <meta name="citation_author" content="John C. Chaput"/>
            <meta name="citation_author_institution" content="Center for BioOptical Nanotechnology, The Biodesign Institute, Arizona State University, Tempe, Arizona, United States of America"/>
            <meta name="citation_author_institution" content="Department of Chemistry and Biochemistry, Arizona State University, Tempe, Arizona, United States of America"/>

    <meta name="citation_date" content="2007/5/23"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0000467.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e467"/>
    <meta name="citation_issue" content="5"/>
    <meta name="citation_volume" content="2"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Functional proteins from a random-sequence library.; citation_author=AD Keefe; citation_author=JW Szostak; citation_journal_title=Nature; citation_volume=410; citation_number=1; citation_pages=715-718; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Evolutionary optimization of a nonbiological ATP binding protein for improved folding stability.; citation_author=JC Chaput; citation_author=JW Szostak; citation_journal_title=Chem Biol; citation_volume=11; citation_number=2; citation_pages=865-874; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=In vitro selection of functional nucleic acids.; citation_author=DS Wilson; citation_author=JW Szostak; citation_journal_title=Ann Rev Biochem; citation_volume=68; citation_number=3; citation_pages=611-647; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=De novo proteins from designed combinatorial libraries.; citation_author=MH Hecht; citation_journal_title=Prot Sci; citation_volume=13; citation_number=4; citation_pages=1711-1723; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Design of a novel globular protein fold with atomic-level accuracy.; citation_author=B Kuhlman; citation_author=G Dantas; citation_author=GC Ireton; citation_author=G Varani; citation_author=BL Stoddard; citation_journal_title=Science; citation_volume=302; citation_number=5; citation_pages=1364-1368; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Classification of proteins based on the properties of the ligand-binding site: The case of adenine-binding proteins.; citation_author=V Cappello; citation_author=A Tramontano; citation_author=U Koch; citation_journal_title=Proteins-Structure Function and Genetics; citation_volume=47; citation_number=6; citation_pages=106-115; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=RNA-peptide fusions for the in vitro selection of peptides and proteins.; citation_author=RW Roberts; citation_author=JW Szostak; citation_journal_title=Proc Natl Acad Sci USA; citation_volume=94; citation_number=7; citation_pages=12297-12302; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=A novel ADP-and zinc-binding fold from function-directed in vitro evolution.; citation_author=P Lo Surdo; citation_author=MA Walsh; citation_author=M Sollazzo; citation_journal_title=Nat Struct Molec Biol; citation_volume=11; citation_number=8; citation_pages=382-383; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Comparison of super-secondary structures in proteins.; citation_author=ST Rao; citation_author=MG Rossmann; citation_journal_title=J Mol Biol; citation_volume=76; citation_number=9; citation_pages=241-256; citation_date=1973; " />
      <meta name="citation_reference" content="citation_title=Structurally analogous proteins do exist!; citation_author=SS Krishna; citation_author=NV Grishin; citation_journal_title=Structure; citation_volume=12; citation_number=10; citation_pages=1125-1127; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Selection for improved protein stability by phage display.; citation_author=S Jung; citation_author=A Honegger; citation_author=A Pluckthun; citation_journal_title=J Mol Biol; citation_volume=294; citation_number=11; citation_pages=163-180; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Selection based on the folding properties of proteins with ribosome display.; citation_author=T Matsuura; citation_author=A Pluckthun; citation_journal_title=Febs Letters; citation_volume=539; citation_number=12; citation_pages=24-28; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=mRNA display: ligand discovery, interaction analysis and beyond.; citation_author=TT Takahashi; citation_author=RJ Austin; citation_author=RW Roberts; citation_journal_title=Trends Biochem Sci; citation_volume=28; citation_number=13; citation_pages=159-165; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Random mutagenesis by PCR.; citation_author=DS Wilson; citation_author=AD Keefe; citation_number=14; citation_date=2000; citation_publisher=Wiley; " />
      <meta name="citation_reference" content="citation_title=Directed evolution of proteins for hetergeneous expression and stability.; citation_author=C Roodveldt; citation_author=A Aharoni; citation_author=DS Tawfik; citation_journal_title=Curr Opin Struct Biol; citation_volume=15; citation_number=15; citation_pages=50-56; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Aptamers selected for higher-affinity binding are not more specific for the target ligands.; citation_author=JM Carothers; citation_author=SC Oestreich; citation_author=JW Szostak; citation_journal_title=J Am Chem Soc; citation_volume=128; citation_number=16; citation_pages=7929-7937; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Phase determination from multiwavelength anomalous diffraction methods.; citation_author=WA Hendrickson; citation_author=CM Ogata; citation_journal_title=Methods Enzym; citation_volume=276; citation_number=17; citation_pages=494-523; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Deciphering the message in protein sequences: tolerance to amino acid substitutions.; citation_author=JU Bowie; citation_author=JF Reidhaar-Olson; citation_author=WA Lim; citation_author=RT Sauer; citation_journal_title=Science; citation_volume=247; citation_number=18; citation_pages=1306-1310; citation_date=1990; " />
      <meta name="citation_reference" content="citation_title=Identifying determinants of folding and activity for a protein of unknown structure.; citation_author=JU Bowie; citation_author=RT Sauer; citation_journal_title=Proc Natl Acad Sci; citation_volume=86; citation_number=19; citation_pages=2152-2156; citation_date=1989; " />
      <meta name="citation_reference" content="citation_title=A method to identify protein sequences that fold into a known three-dimensional structure.; citation_author=JU Bowie; citation_author=R Luthy; citation_author=D Eisenberg; citation_journal_title=Science; citation_volume=253; citation_number=20; citation_pages=164-170; citation_date=1991; " />
      <meta name="citation_reference" content="citation_title=Evolution of the protein repertoire.; citation_author=C Chothia; citation_author=J Gough; citation_author=C Vogel; citation_author=SA Teichmann; citation_journal_title=Science; citation_volume=300; citation_number=21; citation_pages=1701-1703; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Structure and function of haemoglobin II. Some relations between polypeptide chain configuration and amino acid sequence.; citation_author=MF Perutz; citation_author=JC Kendrew; citation_author=HC Watson; citation_journal_title=J Mol Biol; citation_volume=13; citation_number=22; citation_pages=669-678; citation_date=1965; " />
      <meta name="citation_reference" content="citation_title=Molecular pathology of human haemoglobin.; citation_author=MF Perutz; citation_author=H Lehman; citation_journal_title=Nature; citation_volume=219; citation_number=23; citation_pages=902-909; citation_date=1968; " />
      <meta name="citation_reference" content="citation_title=Structural determinants in the sequences of immunoglobulin variable domain.; citation_author=C Chothia; citation_author=I Gelfand; citation_author=A Kister; citation_journal_title=J Mol Biol; citation_volume=278; citation_number=24; citation_pages=457-479; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Combinatorial cassette mutagenesis as a probe of the informational content of protein sequences.; citation_author=JF Reidhaar-Olson; citation_author=RT Sauer; citation_journal_title=Science; citation_volume=241; citation_number=25; citation_pages=53-57; citation_date=1988; " />
      <meta name="citation_reference" content="citation_title=Protein folding from a combinatorial perspective.; citation_author=RT Sauer; citation_journal_title=Folding Design; citation_volume=1; citation_number=26; citation_pages=R27-R30; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Stabilization of GroEL minichaperones by core and surface mutations.; citation_author=Q Wang; citation_author=AM Buckle; citation_author=AR Fersht; citation_journal_title=J Mol Biol; citation_volume=298; citation_number=27; citation_pages=917-926; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=In vitro selection of highly stabilized protein variants with optimized surface.; citation_author=A Martin; citation_author=V Sieber; citation_author=FX Schmid; citation_journal_title=J Mol Biol; citation_volume=309; citation_number=28; citation_pages=717-726; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Stabilization of the cold shock proteins CspB from Bacillus subtilis by evolutionary optimization of coulombic interactions.; citation_author=M Wunderlich; citation_author=A Martin; citation_author=FX Schmid; citation_journal_title=J Mol Biol; citation_volume=347; citation_number=29; citation_pages=1063-1076; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Protein stability and surface electostatics: a charged relationship.; citation_author=SS Strickler; citation_author=AV Gribenko; citation_author=AV Gribenko; citation_author=TR Kieffer; citation_author=J Tomlinson; citation_journal_title=Biochemistry; citation_volume=45; citation_number=30; citation_pages=2761-2766; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Thermophilic adaptation of proteins.; citation_author=R Sterner; citation_author=W Liebl; citation_journal_title=Crit Rev Biochem Molecular; citation_volume=36; citation_number=31; citation_pages=39-106; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Structural basis of thermostability in hyperthermophilic proteins, or “there's more than one way to skin a cat”.; citation_author=GA Petsko; citation_journal_title=Methods Enzym; citation_volume=334; citation_number=32; citation_pages=469-478; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=The neutral theory of molecular evolution.; citation_author=M Kimura; citation_number=33; citation_date=1983; citation_publisher=Cambridge University Press; " />
      <meta name="citation_reference" content="citation_title=Cloning of random-sequence oliogonucleotides.; citation_author=AR Oliphant; citation_author=AL Nussbaum; citation_author=K Struhl; citation_journal_title=Gene; citation_volume=44; citation_number=34; citation_pages=177-183; citation_date=1986; " />
      <meta name="citation_reference" content="citation_title=Integration of macromolecular diffraction data.; citation_author=AGW Leslie; citation_journal_title=Acta Crystallography; citation_volume=D55; citation_number=35; citation_pages=1696-1702; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=The CCP4 suite-programs for protein crystallography.; citation_journal_title=Acta Crystallography; citation_volume=D50; citation_number=36; citation_pages=760-763; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=SOLVE and RESOLVE: automated structure solution and density modification.; citation_author=TC Terwilliger; citation_journal_title=Methods Enzym; citation_volume=374; citation_number=37; citation_pages=22-36; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Coot: model-building tools for molecular graphics.; citation_author=P Emsley; citation_author=K Cowtan; citation_journal_title=Acta Crystallography; citation_volume=D60; citation_number=38; citation_pages=2126-2132; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Refinement of macromolecular structures by the maximum-likehood method.; citation_author=GN Murshudov; citation_author=AA Vagin; citation_author=EJ Dodson; citation_journal_title=Acta Crystallography; citation_volume=D53; citation_number=39; citation_pages=240-255; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Processing of X-ray diffraction data collected in oscillation mode.; citation_author=Z Otwinowski; citation_author=W Minor; citation_journal_title=Methods Enzym; citation_volume=276; citation_number=40; citation_pages=307-326; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=MOLREP: an automated program for molecular replacement.; citation_author=AA Vagin; citation_author=A Teplyakov; citation_journal_title=J Appl Crystal; citation_volume=30; citation_number=41; citation_pages=1022-1025; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Procheck: a program to check the stereochemical quality of protein structures.; citation_author=RA Laskowski; citation_author=MW MacArthur; citation_author=JM Thornton; citation_journal_title=J Appl Crystal; citation_volume=26; citation_number=42; citation_pages=283-291; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=Structure validation by Ca Geometry: p, s, and Cb deviation.; citation_author=SC Lovell; citation_author=IW Davis; citation_author=WB Arendall; citation_author=PIW de Baker; citation_author=JM Word; citation_journal_title=Proteins: Structure, Function, and Genetics; citation_volume=50; citation_number=43; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Jolly SAD.; citation_author=Z Dauter; citation_author=M Dauter; citation_author=EJ Dodson; citation_journal_title=Acta Crystallography D Biol Crystallogr; citation_volume=58; citation_number=44; citation_pages=494-506; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Maximum-likelihood heavy-atom parameter refinement for multiple isomorphous replacement and multiwavelength anomalous diffraction methods.; citation_author=E Fortelle; citation_author=G Bricogne; citation_journal_title=Methods Enzym; citation_volume=276; citation_number=45; citation_pages=472-494; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Methods used in the structure determination of bovine mitochondrial F1 ATPase.; citation_author=JP Abrahams; citation_author=AGW Leslie; citation_journal_title=Acta Crystallography; citation_volume=D52; citation_number=46; citation_pages=30-42; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=The PYMOL molecular graphics program.; citation_author=WL DeLano; citation_number=47; citation_date=2002; citation_publisher=DeLano Scientific; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000467" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Structural Insights into the Evolution of a Non-Biological Protein: Importance of Surface Residues in Protein Fold Optimization"/>
    <meta name="twitter:description" content="Phylogenetic profiling of amino acid substitution patterns in proteins has led many to conclude that most structural information is carried by interior core residues that are solvent inaccessible. This conclusion is based on the observation that buried residues generally tolerate only conserved sequence changes, while surface residues allow more diverse chemical substitutions. This notion is now changing as it has become apparent that both core and surface residues play important roles in protein folding and stability. Unfortunately, the ability to identify specific mutations that will lead to enhanced stability remains a challenging problem. Here we discuss two mutations that emerged from an in vitro selection experiment designed to improve the folding stability of a non-biological ATP binding protein. These mutations alter two solvent accessible residues, and dramatically enhance the expression, solubility, thermal stability, and ligand binding affinity of the protein. The significance of both mutations was investigated individually and together, and the X-ray crystal structures of the parent sequence and double mutant protein were solved to a resolution limit of 2.8 and 1.65 &Aring;, respectively. Comparative structural analysis of the evolved protein to proteins found in nature reveals that our non-biological protein evolved certain structural features shared by many thermophilic proteins. This experimental result suggests that protein fold optimization by in vitro selection offers a viable approach to generating stable variants of many naturally occurring proteins whose structures and functions are otherwise difficult to study."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0000467.g006"/>

  <meta property="og:title" content="Structural Insights into the Evolution of a Non-Biological Protein: Importance of Surface Residues in Protein Fold Optimization" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000467" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=2741'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=3717'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=5487&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0000467">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000467" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000467&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Structural Insights into the Evolution of a Non-Biological Protein: Importance of Surface Residues in Protein Fold Optimization
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Matthew D. Smith, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliations:
                  Center for BioOptical Nanotechnology, The Biodesign Institute, Arizona State University, Tempe, Arizona, United States of America, 
                  Department of Chemistry and Biochemistry, Arizona State University, Tempe, Arizona, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Matthew A. Rosenow, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliations:
                  Center for BioOptical Nanotechnology, The Biodesign Institute, Arizona State University, Tempe, Arizona, United States of America, 
                  Department of Chemistry and Biochemistry, Arizona State University, Tempe, Arizona, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Meitian Wang, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Chemistry and Biochemistry, Arizona State University, Tempe, Arizona, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            James P. Allen, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Chemistry and Biochemistry, Arizona State University, Tempe, Arizona, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Jack W. Szostak, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Howard Hughes Medical Institute, Department of Molecular Biology, Massachusetts General Hospital, Boston, Massachusetts, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            John C. Chaput
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:john.chaput@asu.edu">john.chaput@asu.edu</a></p>

                <p>Affiliations:
                  Center for BioOptical Nanotechnology, The Biodesign Institute, Arizona State University, Tempe, Arizona, United States of America, 
                  Department of Chemistry and Biochemistry, Arizona State University, Tempe, Arizona, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: May 23, 2007</li>
    <li>DOI: 10.1371/journal.pone.0000467</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0000467" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0000467" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000467" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0000467" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0000467-g001" data-doi="info:doi/10.1371/journal.pone.0000467" data-uri="info:doi/10.1371/journal.pone.0000467.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000467.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000467-g002" data-doi="info:doi/10.1371/journal.pone.0000467" data-uri="info:doi/10.1371/journal.pone.0000467.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000467.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000467-t001" data-doi="info:doi/10.1371/journal.pone.0000467" data-uri="info:doi/10.1371/journal.pone.0000467.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000467.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000467-t002" data-doi="info:doi/10.1371/journal.pone.0000467" data-uri="info:doi/10.1371/journal.pone.0000467.t002" title="Table 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000467.t002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000467-g003" data-doi="info:doi/10.1371/journal.pone.0000467" data-uri="info:doi/10.1371/journal.pone.0000467.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000467.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000467-g004" data-doi="info:doi/10.1371/journal.pone.0000467" data-uri="info:doi/10.1371/journal.pone.0000467.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000467.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000467-t003" data-doi="info:doi/10.1371/journal.pone.0000467" data-uri="info:doi/10.1371/journal.pone.0000467.t003" title="Table 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000467.t003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000467-g005" data-doi="info:doi/10.1371/journal.pone.0000467" data-uri="info:doi/10.1371/journal.pone.0000467.g005" title="Figure 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000467.g005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000467-g006" data-doi="info:doi/10.1371/journal.pone.0000467" data-uri="info:doi/10.1371/journal.pone.0000467.g006" title="Figure 6">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000467.g006&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000467">Reader Comments (0)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0000467" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1"></a><p>Phylogenetic profiling of amino acid substitution patterns in proteins has led many to conclude that most structural information is carried by interior core residues that are solvent inaccessible. This conclusion is based on the observation that buried residues generally tolerate only conserved sequence changes, while surface residues allow more diverse chemical substitutions. This notion is now changing as it has become apparent that both core and surface residues play important roles in protein folding and stability. Unfortunately, the ability to identify specific mutations that will lead to enhanced stability remains a challenging problem. Here we discuss two mutations that emerged from an <em>in vitro</em> selection experiment designed to improve the folding stability of a non-biological ATP binding protein. These mutations alter two solvent accessible residues, and dramatically enhance the expression, solubility, thermal stability, and ligand binding affinity of the protein. The significance of both mutations was investigated individually and together, and the X-ray crystal structures of the parent sequence and double mutant protein were solved to a resolution limit of 2.8 and 1.65 Å, respectively. Comparative structural analysis of the evolved protein to proteins found in nature reveals that our non-biological protein evolved certain structural features shared by many thermophilic proteins. This experimental result suggests that protein fold optimization by <em>in vitro</em> selection offers a viable approach to generating stable variants of many naturally occurring proteins whose structures and functions are otherwise difficult to study.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>Smith MD, Rosenow MA, Wang M, Allen JP, Szostak JW, et al.  (2007) Structural Insights into the Evolution of a Non-Biological Protein: Importance of Surface Residues in Protein Fold Optimization. PLoS ONE 2(5):
          e467.
            doi:10.1371/journal.pone.0000467</p><p><strong>Academic Editor: </strong>Haiwei Song, Institute of Molecular and Cell Biology, Singapore</p><p><strong>Received:</strong> March 28, 2007; <strong>Accepted:</strong> April 25, 2007; <strong>Published:</strong> May 23, 2007</p><p><strong>Copyright:</strong> © 2007 Smith et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>This work was supported by new laboratory start-up funds from the The Biodesign Institute to JCC and grants from the NIH and NASA Astrobiology Institute to JWS. JWS is an Investigator, and J.C.C. was a Research Associate of the Howard Hughes Medical Institute.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>We are interested in the extent to which nature samples the total structural diversity available in protein sequence space <a href="#pone.0000467-Keefe1">[1]</a>, <a href="#pone.0000467-Chaput1">[2]</a>. In pursuit of finding novel proteins with properties similar to natural proteins, we have discovered that functional proteins can be selected from large unconstrained libraries of random amino acid sequences <a href="#pone.0000467-Keefe1">[1]</a>. The non-biological proteins that emerge from these selections are discovered in much the same way that aptamers are selected from large pools of DNA and RNA <a href="#pone.0000467-Wilson1">[3]</a>. We call this approach <em>de novo</em> protein evolution since the <em>in vitro</em> process of selection and amplification closely mimics the natural process of Darwinian evolution. In these experiments we explore how functional proteins evolve by imposing a selective pressure on a diverse population of unrelated sequences to enrich for molecules with a desired functional property. By starting from a random pool of proteins we attempt to sample broad regions of protein sequence space for different independent solutions to a given functional problem. Following several rounds of selection, we then recover the descendents of rare functional proteins that originated from starting libraries as large as 10<sup>13</sup> different sequences. It is important to note that while our starting libraries are large by conventional standards, the total number of sequences analyzed represents an extremely sparse sampling of all possible sequence combinations, indicating that protein sequence space is surprisingly rich in functional diversity. In contrast to <em>de novo protein design</em>, which often relies on genetic algorithms to predict sequences consistent with a predetermined secondary or tertiary structure <a href="#pone.0000467-Hecht1">[4]</a>, <a href="#pone.0000467-Kuhlman1">[5]</a>, the process of <em>de novo protein evolution</em> requires no prior knowledge of a protein's structure or mechanism in order for a selection to be successful. As a result, larger regions of the protein universe can be explored for protein structures that are unanticipated and therefore potentially much more novel than structures predicted by design.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>Initial experiments in <em>de novo</em> protein evolution began with an attempt to ascertain the frequency of ATP binding proteins in a sampling of all possible sequences in a contiguous library of 80 amino acids <a href="#pone.0000467-Keefe1">[1]</a>. Here functional activity was viewed as the ability to bind a desired small molecule target with high affinity and specificity. Given that many protein domains have sequence lengths between 50 and 100 amino acids and ATP binding proteins are present in every major enzyme class, we felt that the random region and target choice were appropriate to identify small protein domains with simple ligand binding activity <a href="#pone.0000467-Cappello1">[6]</a>. Because the probability of finding functional domains in a stochastic sampling of protein sequences was anticipated to be very low, perhaps too low to detect with conventional technologies, we used a cell-free selection system called messenger RNA display to construct a starting library of greater than 10<sup>12</sup> non-redundant random-sequence proteins <a href="#pone.0000467-Roberts1">[7]</a>.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>Following eight rounds of selection, four families of independent protein sequences were identified that bound ATP. One of these proteins (Family B) was optimized by directed evolution for improved binding affinity. DNA sequencing of the output from this selection revealed a distant clone (clone 18-19) that differed from the consensus sequence at 15 out of 80 positions and bound ATP with far greater affinity and specificity than all other clones from that round of selection <a href="#pone.0000467-Keefe1">[1]</a>. The X-ray crystal structure for protein 18-19 was originally solved by Lo Surdo et al. and found to adopt a novel zinc-nucleated α/β-fold not yet observed by nature <a href="#pone.0000467-LoSurdo1">[8]</a>. Structural comparison of protein 18-19 with the databank of biological protein folds revealed that our <em>de novo</em> evolved protein shared certain structural features with some proteins found in nature. These include a ligand binding site with stacking and electrostatic interactions similar to many natural nucleotide binding proteins and a zinc-binding site that is structurally analogous to the treble clef zinc binding motif <a href="#pone.0000467-Rao1">[9]</a>, <a href="#pone.0000467-Krishna1">[10]</a>. The discovery of close structural analogs between proteins that clearly do not share a common evolutionary ancestor suggests that certain structural motifs may be more common than previously thought, and may have emerged independently several times. How these motifs combine in three-dimensional space to form a stably folded protein might be less conserved.</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p>Unlike many naturally occurring proteins, protein 18-19 requires high concentrations of free ligand in order to remain stably folded and soluble. In an attempt to overcome this limitation and to evolve a non-biological protein toward a folded state that more closely resembles the ligand-independent folded state of many natural proteins, we designed an <em>in vitro</em> selection experiment using mRNA display to isolate variants of protein 18-19 that remained bound to an ATP agarose affinity resin in the presence of increasing concentrations of chemical denaturant. Our goal was to examine the extent to which a <em>de novo</em> evolved protein, originally selected on the basis of ligand binding affinity, could be evolved to remain stably folded in the absence of exogenous ligand. We have previously used this strategy to identify variants of the Family B protein from a pool of sequences present in the output of the selection for improved binding affinity <a href="#pone.0000467-Chaput1">[2]</a>. Here the pool was taken forward without mutagenesis to identify stable variants from within the existing population of molecules. While others have used phage display and ribosome display to improve the folding properties of natural proteins <a href="#pone.0000467-Jung1">[11]</a>, <a href="#pone.0000467-Matsuura1">[12]</a>, neither technique is capable of achieving both high library complexity and stable genotype-phenotype linkage under highly denaturing conditions. mRNA display, by contrast, is a cell-free selection system based on a covalent amide bond linking newly translated proteins to their encoding RNA message <a href="#pone.0000467-Takahashi1">[13]</a>. The absence of an <em>in vivo</em> transformation step allows mRNA display to avoid many of the intrinsic biases associated with cellular expression and to facilitate the construction of unusually large (&gt;10<sup>13</sup>) protein libraries. We therefore reasoned that it might be possible to use mRNA-display to evolve a population of ATP binding proteins that remained stably folded in the absence of excess ligand if such variants existed and if our starting library contained enough sequence diversity to examine a significant region of sequence space around the ATP binding domain.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3>
<h4>Library Design and Construction</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>We began by generating a pool of protein 18-19 variants using error-prone PCR to produce random point mutations that would lead to single-residue changes in the protein <a href="#pone.0000467-Wilson2">[14]</a>. Serial amplification was performed on the entire 240-nt gene-encoding region with a target mutagenesis rate of 3.5% per amino acid position. The quality of the pool was evaluated by sequencing 20 clones from the mutagenized DNA library. Analysis of the individual sequences revealed that the observed mutagenesis rate closely approximated the expected rate (3.7% versus 3.5% per amino acid position, respectively) with a roughly equal number of transitions and transversions. Two minor changes were made to the constant regions of the library relative to the starting library used to select protein 18-19. Two out-of-frame stop codons and a psoralen photo-crosslinking site were introduced downstream of the open reading frame to enhance fusion formation and reduce the time and cost of fusion construction.</p>
<a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2"></a><p>The DNA library was transcribed into RNA (<a href="#pone-0000467-g001">Fig. 1</a>) and photo-ligated to a DNA-puromycin linker using a psoralen photocrosslinking reagent to conjugate the synthetic puromycin linker to the 3′-end of the RNA. The pool of RNA-tagged DNA-puromycin templates was translated <em>in vitro</em> with rabbit reticulocyte lysate to produce a starting library of mRNA-protein fusions. The mRNA-protein fusions were then purified on oligo-dT cellulose and Ni-NTA agarose columns, and the RNA portion was reverse transcribed with DNA to prevent the unwanted enrichment of ATP binding RNA aptamers and to facilitate the amplification of selected proteins after separation on an ATP-agarose affinity column.</p>
<div class="figure" id="pone-0000467-g001"><div class="img"><a name="pone-0000467-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000467.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000467" data-uri="info:doi/10.1371/journal.pone.0000467.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000467.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000467.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000467.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000467.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000467.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000467.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000467.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000467.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000467.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Evolutionary Optimization of Protein 18-19 by mRNA Display.</span></strong></p><a id="article1.body1.sec2.sec1.fig1.caption1.p1" name="article1.body1.sec2.sec1.fig1.caption1.p1"></a><p>The DNA library encodes randomly mutated variants of protein 18-19. For each round of selection the DNA library was transcribed, photoligated to a 3′-puromycin DNA linker, translated, and reverse transcribed to produce a pool of protein 18-19 variants displayed as mRNA-protein fusions. The library was incubated with ATP derivatized agarose in selection buffer supplemented with 1.5-3 M GuHCl, washed to remove the unbound material, and eluted with selection buffer containing 10 mM ATP. Eluted fractions were combined, amplified by PCR, and used as input into the next round of selection.</p>
<span>doi:10.1371/journal.pone.0000467.g001</span></div>

<h4>Evolutionary Optimization of Protein 18-19 by mRNA Display</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>Evolutionary optimization of a protein fold requires developing a set of selection conditions that enable the structure to be optimized without compromising the function of the protein. Many examples of such trade-offs exist, especially among protein scaffolds designed to function as antibody mimics, where highly stable folds are reduced to marginally stable structures after evolving new functions <a href="#pone.0000467-Roodveldt1">[15]</a>. We therefore chose to optimize protein 18-19 using a strategy that has both structural and functional constraints built into the selection step. Our approach to this problem involved enriching for well-folded variants by incubating the pool of mRNA-protein fusions with an ATP-derivatized affinity resin in the presence of guanidine hydrochloride (GuHCl). For each round of selection, the GuHCl concentration was gradually increased from 1.5 M to 3 M in an effort to ensure that less than 10% of the input to each round of selection was recovered in the elution step. By inactivating the pool by at least 90% prior to each round of selection and eluting under equilibrium conditions with 10 mM free ATP, we aimed to enrich for ATP binding proteins that remained folded and functional. Special care was taken to remove the guanidine hydrochloride prior to DNA amplification by PCR. Analysis of each of the selection steps in the starting denaturant concentration of 1.5 M GuHCl (<a href="#pone-0000467-g002">Fig. 2</a>) revealed that the fraction of library that bound to and eluted from the ATP agarose beads increased in each round, reaching ~40% at round 4, but not increasing further at round 5. By contrast, protein 18-19 when constructed as an mRNA-fusion binds very poorly (~1–2%) to ATP derivatized agarose when 1.5 M GuHCl is present in the binding, wash, and elution buffers.</p>
<div class="figure" id="pone-0000467-g002"><div class="img"><a name="pone-0000467-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000467.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000467" data-uri="info:doi/10.1371/journal.pone.0000467.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000467.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000467.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000467.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000467.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000467.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000467.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000467.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000467.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000467.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>ATP Binding Selection Profile.</span></strong></p><a id="article1.body1.sec2.sec2.fig1.caption1.p1" name="article1.body1.sec2.sec2.fig1.caption1.p1"></a><p>The proportion of mRNA-protein fusions that bound to immobilized ATP and eluted with free ATP in selection buffer supplemented with 1.5 M GuHCl. The pool of protein variants used as the input into round 1 were generated by amplification of protein 18-19 under mutagenic conditions. The average mutagenic rate was 3.5% per amino acid position.</p>
<span>doi:10.1371/journal.pone.0000467.g002</span></div>

<h4>Sequence Analysis of Selected Proteins</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>We sequenced 53 clones from the output of round 5 to determine the sequence changes that led to the improved binding in the presence of GuHCl. Comparison of the selected sequences (<a href="#pone-0000467-t001">Table 1a</a>) showed that two amino acid substitutions dominated the selected population (N32D and D65V, present more than 35 times in 53 different sequences), while 11 other mutations had become selectively enriched (present more than 5 times in 53 different sequences). To examine this non-biological evolutionary history, we aligned (<a href="#pone-0000467-t001">Table 1b</a>) the primordial sequence from round 8 of the original selection from which all subsequent clones derive and the consensus sequence and highest affinity clone (protein 18-19) from round 18 of the binding optimized selection with the consensus sequence (protein DX) from the folding optimized selection performed on protein 18-19. These three sets of consensus sequences represent distinct stages in the evolutionary history of the Family B protein from its original discovery in a pool of 4 trillion random sequences to its optimization for improved binding and subsequent optimization for improved folding. Protein 18-19 was the highest affinity sequence from the binding optimized selection and the parent sequence for the current selection. </p>
<a id="article1.body1.sec2.sec3.p2" name="article1.body1.sec2.sec3.p2"></a><p>Comparison of the different protein sequences revealed that one of the two mutations observed in the folding optimized protein (N32D) reverts an asparagine residue in protein 18-19 back to the aspartic acid residue present in the primordial sequence. The second mutation, D65V, was previously observed in a separate selection experiment designed to improve the folding stability of the Family B protein pool after optimization for improved binding affinity <a href="#pone.0000467-Chaput1">[2]</a>. The consensus sequence from that selection (termed FOB, which stands for folding optimized protein from Family B) differs from protein DX at 17 out of 80 positions. The fact that protein 18-19 could achieve a similar degree of optimization with only two mutations seems to indicate that protein 18-19 was much closer to achieving structural optimization than other proteins from round 18 of the binding optimized selection. This is consistent with the previous observation that protein 18-19 exhibited greater ATP binding affinity than all other clones from that round of selection.</p>
<div class="figure" id="pone-0000467-t001"><div class="img"><a name="pone-0000467-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000467.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000467" data-uri="info:doi/10.1371/journal.pone.0000467.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000467.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000467.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000467.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000467.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000467.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000467.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000467.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000467.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000467.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>Sequence Comparison of the In Vitro Selected Proteins.<sup>a</sup></span></strong></p><span>doi:10.1371/journal.pone.0000467.t001</span></div>

<h4>Protein Expression and Purification</h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>In order to investigation the functional significance of the two mutations observed in the folding optimized selection for protein 18-19, we cloned protein 18-19, protein 18-19 (N32D), protein 18-19 (D65V), and protein DX into a pMal plasmid and expressed the proteins in <em>E. coli</em> as N-terminal fusions of the maltose binding protein (MBP). Proteins were purified away from the crude lysate using an amylose affinity resin and quantified by UV spectroscopy. Comparison of the total amount of fusion protein obtained from a 1 L expression culture (<a href="#pone-0000467-t002">Table 2</a>) indicated that proteins containing one or both of the mutations expressed at higher levels than protein 18-19 (~150 versus ~65 mg/L culture, respectively). ATP binding proteins (ABP) were further purified by incubating the fusions with ATP agarose beads, proteolyzing the bound fusions with thrombin, washing away the MBP with buffer, and eluting the free ABP with phosphate buffer supplemented with 10 mM ATP. ATP binding proteins were then dialyzed with phosphate buffer to remove the excess ATP ligand. The two-step purification yielded highly pure ATP binding proteins suitable for biophysical characterization.</p>
<div class="figure" id="pone-0000467-t002"><div class="img"><a name="pone-0000467-t002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000467.t002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000467" data-uri="info:doi/10.1371/journal.pone.0000467.t002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000467.t002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000467.t002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000467.t002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000467.t002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000467.t002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000467.t002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000467.t002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000467.t002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000467.t002.TIF"></span>)
                </a></li></ul></div><p><strong>Table 2.  <span>Properties of Artificial Proteins.</span></strong></p><span>doi:10.1371/journal.pone.0000467.t002</span></div>

<h4>Biophysical Characterization</h4>
<a id="article1.body1.sec2.sec5.p1" name="article1.body1.sec2.sec5.p1"></a><p>We were interested in determining the extent to which our evolutionary optimization strategy led to protein variants with improved solubility and folding stability. We began by measuring the binding affinity (<em>K</em><sub>d</sub>) of each protein for ATP by equilibrium spin-filtration <a href="#pone.0000467-Carothers1">[16]</a>. Since it was difficult to determine how much ATP remained bound to the free protein after dialysis, <em>K</em><sub>d</sub> values were generated using MBP protein fusions. We have used this approach before to evaluate the binding affinity of other <em>de novo</em> evolved ATP binding proteins <a href="#pone.0000467-Keefe1">[1]</a>, <a href="#pone.0000467-Chaput1">[2]</a>. Analysis of the results from this experiment (<a href="#pone-0000467-t002">Table 2</a>) revealed that the N32D mutation caused the greatest increase in ATP binding affinity, with a <em>K</em><sub>d</sub> of 360 nM, relative to protein 18-19, which binds ATP with a <em>K</em><sub>d</sub> of 1100 nM. Protein DX, which contains both the N32D and D65V mutations, binds ATP with a <em>K</em><sub>d</sub> of 450 nM and protein 18-19 containing the D65V mutation binds ATP with a <em>K</em><sub>d</sub> of 650 nM. Together, the selected mutations represent a ~2-fold increase in ATP binding affinity over the parent sequence.</p>
<a id="article1.body1.sec2.sec5.p2" name="article1.body1.sec2.sec5.p2"></a><p>To determine how selecting for improved folding stability correlated with improved protein solubility, we monitored the time-dependent precipitation of each protein in the absence and presence of ATP. Amylose purified MBP fusion proteins were separated into their respective proteins by thrombin proteolysis. After the proteolysis step, phenylmethylsulfonyl fluoride was added to the reaction mixture to prevent non-specific degradation of the ATP binding protein by thrombin. The reaction was then analyzed after 1, 3, 5, and 7 days by centrifuging the mixture to separate insoluble aggregates from soluble free protein, and analyzing the supernatant by SDS gel electrophoresis. Comparison of the individual proteins (<a href="#pone-0000467-g003">Figure 3a</a>) in the absence and presence of ATP indicated that all four proteins remain fully soluble after 7 days at room temperature when ATP is present in the reaction mixture. Experiments performed in the absence of ATP showed that protein 18-19 forms an insoluble precipitate after 3 days, while proteins containing one or both of the selected mutations remain soluble after 7 days. Analysis of the reaction mixture after 14 and 21 days at room temperature (data not shown) indicated that protein variants containing one or both of the selected mutations remained in the soluble fraction. The improved solubility observed for protein DX indicates that selecting for folding stability is a valid approach to isolating protein variants with improved solubility.</p>
<div class="figure" id="pone-0000467-g003"><div class="img"><a name="pone-0000467-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000467.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000467" data-uri="info:doi/10.1371/journal.pone.0000467.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000467.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000467.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000467.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000467.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000467.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000467.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000467.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000467.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000467.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Characterization of the Selected Point Mutations.</span></strong></p><a id="article1.body1.sec2.sec5.fig1.caption1.p1" name="article1.body1.sec2.sec5.fig1.caption1.p1"></a><p>(A) ATP binding proteins 18-19, 18-19 (N32D), 18-19 (D65V), and DX were assayed for solubility in the absence and presence of free ATP. SDS-PAGE analysis shows that protein variants containing one or both of the selected mutations remain soluble in the absence of exogenous ATP. (B) Thermal denaturing curves for protein 18-19 and evolutionary optimized variants. The thermal stability of proteins 18-19, 18-19 (N32D), 18-19 (D65V), and DX were measured by temperature dependent circular dichroism at a fixed wavelength of 210 nm. Proteins containing one or both of the selected mutations exhibit higher relative thermal stability than the parent sequence.</p>
<span>doi:10.1371/journal.pone.0000467.g003</span></div><a id="article1.body1.sec2.sec5.p3" name="article1.body1.sec2.sec5.p3"></a><p>In order to determine the contribution of each mutation toward the improved folding stability of protein DX, we examined the stability of protein 18-19, protein 18-19 (N32D), protein 18-19 (D65V), and protein DX by thermal denaturation. The stability of the folded state was measured by monitoring changes in circular dichroism (210 nm) as a function of temperature. Denaturing profiles (<a href="#pone-0000467-g003">Fig. 3b</a>) for all four proteins were measured in the absence of exogenous ATP. All four proteins gave sigmoidal curves consistent with a single transition between the native and denatured states. We were able to determine the relative thermal stability of each protein by comparing the melting transition (T<sub>50</sub>), the temperature at which half of the protein is folded and half unfolded, for the parent protein and the two mutations individually and together. None of the proteins melted reversibly and as a result these values represent relative thermal stabilities rather than thermodynamic parameters. From this comparison, we found that protein DX exhibits significantly greater thermal stability than protein 18-19 (79.5°C versus 57.5°C, respectively). Of the two individual mutations, N32D appears to be responsible for most of the enhanced thermal stability, with a T<sub>50</sub> of 77.1°C, while the D65V mutation is intermediate in stability and exhibits a T<sub>50</sub> of 66.9°C.</p>


<h4>Structural Analysis</h4>
<a id="article1.body1.sec2.sec6.p1" name="article1.body1.sec2.sec6.p1"></a><p>We determined the X-ray crystal structures of protein 18-19 and the evolutionarily optimized variant, protein DX, to resolution limits of 2.8 Å and 1.65 Å, respectively (<a href="#pone-0000467-t003">Table 3</a>, <a href="#pone.0000467.s001">Fig. S1</a>). The structure of protein 18-19 was solved from the anomalous X-ray scattering of a zinc ion bound to the protein using multiwavelength anomalous dispersion (MAD) methods <a href="#pone.0000467-Hendrickson1">[17]</a>. The structure for protein DX was initially solved by molecular replacement using the coordinates from protein 18-19, and later solved independently using single anomalous dispersion techniques. The Cα backbone trace of proteins 18-19 and DX can be superimposed (<a href="#pone-0000467-g004">Fig. 4b</a>) with a root mean square deviation (r.m.s.d.) of 0.51 Å from residues 7 to 71. Both structures closely approximate a previously determined X-ray crystal structure <a href="#pone.0000467-LoSurdo1">[8]</a>. As illustrated in <a href="#pone-0000467-g004">Figure 4a</a>, the fold is characterized by an α-helix (residues 6–20), followed by an extended loop, then two short β-strands (residues 33–36 and 39–43), followed by a second α-helix (residues 46–56) that leads to another extended loop which terminates in a third β-strand (residues 64–68). The three β-strands form an antiparallel β-sheet that separates the two α-helices.</p>
<div class="figure" id="pone-0000467-g004"><div class="img"><a name="pone-0000467-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000467.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000467" data-uri="info:doi/10.1371/journal.pone.0000467.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000467.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000467.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000467.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000467.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000467.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000467.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000467.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000467.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000467.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>The X-ray Crystal Structure of Protein DX.</span></strong></p><a id="article1.body1.sec2.sec6.fig1.caption1.p1" name="article1.body1.sec2.sec6.fig1.caption1.p1"></a><p>(A) Schematic representation of the Family B scaffold. Mutations N32D and D65V transformed protein 8-19 into a stably folded water-soluble protein with significantly less ligand dependence than the parent sequence. (B) The three-dimensional structure (ribbon diagram) of protein DX (gray) superimposed with the parent sequence, protein 18-19 (gold). The zinc metal ion is shown in orange and the ATP ligand is colored by atom type.</p>
<span>doi:10.1371/journal.pone.0000467.g004</span></div><div class="figure" id="pone-0000467-t003"><div class="img"><a name="pone-0000467-t003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000467.t003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000467" data-uri="info:doi/10.1371/journal.pone.0000467.t003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000467.t003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000467.t003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000467.t003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000467.t003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000467.t003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000467.t003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000467.t003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000467.t003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000467.t003.TIF"></span>)
                </a></li></ul></div><p><strong>Table 3.  <span>Summary of Crystallographic Data.</span></strong></p><span>doi:10.1371/journal.pone.0000467.t003</span></div><a id="article1.body1.sec2.sec6.p2" name="article1.body1.sec2.sec6.p2"></a><p>The overall fold of the protein is stabilized by the presence of two cofactors. A zinc metal ion is bound by tetrahedral coordination to four invariant cysteine residues at positions 23, 26, 46, and 49. The identification of the metal ion as a divalent zinc atom was unambiguous based on the electron density maps generated from the multiple anomalous data. The presence of a zinc metal ion was not unexpected, and we have previously shown that protein 18-19 was a zinc-binding protein with the zinc metal required for folding <a href="#pone.0000467-Keefe1">[1]</a>. In addition to the zinc ion, both structures also contain one ATP molecule bound to the protein in a well-defined nucleotide binding site. The adenine nucleobase is buried in a hydrophobic pocket (<a href="#pone-0000467-g005">Fig. 5a</a>) that is stabilized by stacking interactions involving the aromatic residues Tyr43 and Phe50 and hydrogen bonding interactions to mainchain carbonyl oxygens at residue positions Met45 and Gly63. The sugar moiety adopts a 2′-endo conformation with His64 hydrogen bonding with the 2′-hydroxyl group. The presence of several basic amino acid residues along the β-sheet, including Lys34, Lys36, and Arg41 provide a favorable electrostatic environment for the α-and β-phosphates, with the orientation of Arg41 controlled by a second shell of hydrogen bonding to Asp66. The ATP ligand is further stabilized by coordination to several additional bound water molecules. The γ-phosphate of ATP was less resolved (<a href="#pone-0000467-g006">Fig. 6</a>) in the electron density map. The large distance of 11 Å between the zinc atom and the center of the adenine nucleobase indicates that the zinc metal ion serves a structural role rather than functional role in ligand binding.</p>
<div class="figure" id="pone-0000467-g005"><div class="img"><a name="pone-0000467-g005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000467.g005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000467" data-uri="info:doi/10.1371/journal.pone.0000467.g005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000467.g005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000467.g005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000467.g005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000467.g005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000467.g005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000467.g005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000467.g005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000467.g005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000467.g005.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 5.  <span>Structural Comparison of the Selected Mutations.</span></strong></p><a id="article1.body1.sec2.sec6.fig2.caption1.p1" name="article1.body1.sec2.sec6.fig2.caption1.p1"></a><p>(A) The region of the ATP binding site contains a network of pi-stacking (Y43 and F50) and electrostatic interactions (K34, K36, R41, and D66) that mediate ligand binding. (B) The D65V mutation removes a salt bridge on the hydrophobic side of the β-sheet and repositions several nearby loops and side-chains. (C) The N32D mutation caused a change in the electrostatic environment of the ligand binding site by forming an ion pair with K47.</p>
<span>doi:10.1371/journal.pone.0000467.g005</span></div><div class="figure" id="pone-0000467-g006"><div class="img"><a name="pone-0000467-g006" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000467.g006&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000467" data-uri="info:doi/10.1371/journal.pone.0000467.g006"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000467.g006&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000467.g006/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000467.g006/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000467.g006/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000467.g006/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000467.g006.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000467.g006/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000467.g006/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000467.g006.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 6.  <span>The crystal structure of the ATP ligand bound to the DX protein with a 2FoFc map contoured at a 1.5 sigma level.</span></strong></p><a id="article1.body1.sec2.sec6.fig3.caption1.p1" name="article1.body1.sec2.sec6.fig3.caption1.p1"></a><p>The electron density is much weaker at the gamma phosphate position indicating that this portion of the molecule is either disordered or has multiple conformations. The molecule is colored according to atom type.</p>
<span>doi:10.1371/journal.pone.0000467.g006</span></div><a id="article1.body1.sec2.sec6.p3" name="article1.body1.sec2.sec6.p3"></a><p>The Asp32 and Val65 amino acid mutations identified during the evolutionary optimization of protein 18-19 are located on opposite sides and faces of the β-sheet. The effects of the two mutations appear to be independent due to the long distance of 11.2 Å separates the alpha carbons of both residues. In protein 18-19, Asp65 is located on the hydrophobic face of the third strand and forms a salt bridge with Arg21, while Asn32 is located on the polar face of the first strand and contacts the purine base of ATP through a bridging water interaction. Although the overall fold of the protein does not change due to the mutations, several displacements of amino acid side chains and the loops are evident. The largest structural change occurs as a result of the D65V mutation (<a href="#pone-0000467-g005">Fig. 5b</a>), which eliminates the R21-D65 salt bridge, causing a significant repositioning of the Arg21 side chain. The change in the electrostatic interactions involving Arg21, Arg31, and Asp65 sidechains leads to a modest shift in the neighboring loops, with slight repositioning of the Arg31 sidechain. By comparison, the Asn→Asp mutation at position 32 (<a href="#pone-0000467-g005">Figure 5c</a>) does not alter the protein backbone, but instead results in an ion pair between Asp32 and adjacent Lys47, that is critical to balancing the dense positive charge present in the ATP ligand binding site. This change is evident when comparing the surface view of the electrostatic potential for both proteins (<a href="#pone.0000467.s002">Fig. S2</a>). Indeed a slight repositioning of the Lys47 sidechain toward the purine nucleobase can be seen in the crystal structure of protein DX. Small differences in the relative positioning of the ATP molecules in the two protein structures cannot be resolved due to the limiting resolution of the 18-19 structure.</p>

</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p>Proteins are composed of amino acids whose sequence and identity determine the shape and function of the molecule. Unfortunately, very little is known about how such complex three-dimensional information is encoded in a one-dimensional amino acid sequence. Methods designed to reduce the complexity of this problem have relied heavily on substitution patterning to determine the relative tolerance of a particular amino acid position to change <a href="#pone.0000467-Bowie1">[18]</a>–<a href="#pone.0000467-Bowie3">[20]</a>. Two dominant approaches have been used to profile the substitution tolerance of proteins. The first involves a phylogenetic comparison of the sequences of evolutionary related proteins from different organisms <a href="#pone.0000467-Chothia1">[21]</a>. Early work in this area showed that natural haemoglobin variants tolerated substitutions at exterior positions much more readily than interior positions <a href="#pone.0000467-Perutz1">[22]</a>, <a href="#pone.0000467-Perutz2">[23]</a>. More recent comparisons made with larger data sets further support the initial finding that interior sites are more restrictive than exterior sites <a href="#pone.0000467-Chothia2">[24]</a>. Given the large amount of sequence information currently available in databases of biological proteins, this method can be extremely effective at profiling amino acid positions determined by natural selection. The second approach relies on combinatorial methods, such as cassette mutagenesis, to identify the constraints on individual positions in a protein sequence <a href="#pone.0000467-ReidhaarOlson1">[25]</a>. The technique consists of a stepwise selection process whereby small sets of neighboring residues are randomized using degenerate DNA cassettes that encode all 20 naturally occurring amino acids, selected for function, and sequenced to identify acceptable substitutions. Repeating the process several times allows for complete coverage of specific regions of a protein domain or binding interface. Combinatorial approaches have been used many times to identify the structural determinants of proteins of both known and unknown structures <a href="#pone.0000467-Bowie2">[19]</a>–<a href="#pone.0000467-ReidhaarOlson1">[25]</a>. Results from initial experiments on the DNA binding domain of λ repressor showed that core residues tend to have invariant or highly conserved side chains due to the precise nature of each residue within the jigsaw framework of the hydrophobic core, while surface residues tend to accept a wider-range of amino acid substitutions <a href="#pone.0000467-ReidhaarOlson1">[25]</a>. These and other studies have led to the general conclusion that amino acid residues that occupy solvent inaccessible positions in the hydrophobic core carry most of the information that defines the three-dimensional shape of the protein, while surface residues help maintain the stability of the fold <a href="#pone.0000467-Sauer1">[26]</a>–<a href="#pone.0000467-Strickler1">[30]</a>.</p>
<a id="article1.body1.sec3.p2" name="article1.body1.sec3.p2"></a><p>In the present study, we used directed evolution to identify specific amino acid mutations from a pool of randomly mutated variants that enhance the stability of the protein fold. We then evaluated the mutations using biophysical methods to determine the effects of the mutations on the protein. The results from these experiments revealed that mutations N32D and D65V, located at solvent exposed positions, significantly enhance protein expression, solubility, thermal stability, and ligand binding affinity. This observation supports the notion that surface exposed residues carry more information that previously thought and make a significant contribution to protein folding and stability <a href="#pone.0000467-Wang1">[27]</a>–<a href="#pone.0000467-Strickler1">[30]</a>. In evaluating the selected mutations, we wondered whether the stringent selection conditions used to evolve our non-biological protein led to the discovery of stabilization factors similar to those commonly attributed to thermophilic proteins <a href="#pone.0000467-Sterner1">[31]</a>. Although the structural basis for how thermophilic and hyperthermophilic proteins achieve enhanced thermostability over their mesophilic homologues remains unclear, most studies suggest that extreme thermostability arises from a combination of factors that include increasing the content of polar charged residues at the expense of nonpolar sidechains, optimizing the number of electrostatic interactions, improving core packing, and reducing conformational entropy by minimizing large flexible loops <a href="#pone.0000467-Sterner1">[31]</a>, <a href="#pone.0000467-Petsko1">[32]</a>.</p>
<a id="article1.body1.sec3.p3" name="article1.body1.sec3.p3"></a><p>Structural comparison of our evolutionarily optimized protein to the parent structure (<a href="#pone-0000467-g005">Fig. 5a–c</a>) indicates that protein DX shares some of the same stabilizing factors as thermophilic proteins. The N32D mutation changes a polar uncharged residue to a polar charged residue, which improves the electrostatic environment of the protein by adding a negatively charged residue to a highly positively charged region of the protein. Balancing the charge density at this location was likely critical to stabilizing the protein in the absence of the ATP ligand. By comparison, the D65V mutation substitutes a polar charged residue with a non-polar residue on the hydrophobic side of the β-sheet. This mutation gives rise to a number of small structural changes, each of which appears to enhance the stability of the fold, as evident from the improved biophysical properties exhibited by the single-point mutations. The principle structural change involves the loss of the R21-D65 salt bridge (<a href="#pone-0000467-g005">Fig. 5b</a>), which leads to a change in the local electrostatic environment, causing the Arg21 to rotate away from the hydrophobic face of the β-sheet and reposition in the exposed solvent. The D65V substitution also causes a modest shift in nearby loops, which results in the optimization of electrostatic contacts at nearby residues. Together these smaller structural changes could reduce the conformational entropy of the protein by improving local packing and electrostatic interactions previously constrained by the R21-D65 salt bridge. Calculation of the solvent exposed surface area for each residue in the protein sequence indicated that positions 32 and 65 remained unaffected by the substitutions, reflecting the lack of any significant backbone changes. While further experiments are needed to define the precise nature of the N32D and D65V mutations, it is also possible that these residues may be involved in a complex network of long-range interactions that help modulate protein folding and ATP binding activity.</p>
<a id="article1.body1.sec3.p4" name="article1.body1.sec3.p4"></a><p>Although many unanswered questions remain about the amount of information that is necessary to encode a functional protein, other questions such as why these surface interactions were not identified in previous substitution experiments are easier to explain. The most likely reason why surface residues were initially dismissed in terms of information content resides in the fact that these interactions are often subtle and difficult to identify, and that the experimental procedures previously used to identify amino acid tolerance are often constrained by nature or experimental design. Both phylogenetic and combinatorial approaches are limited to the types and number of mutations found in the final sequence alignment. Here it is often difficult, without examining each substitution individually, to distinguish which mutations result from selection and which are caused by random drift, and the relative importance of each mutation with respect to fold and function <a href="#pone.0000467-Kimura1">[33]</a>. Combinatorial approaches are further constrained by the selection conditions used to distinguish functionally active variants, and the window of residues assayed by each cassette <a href="#pone.0000467-Oliphant1">[34]</a>. In the original cassette mutagenesis studies, for example, an <em>in vivo</em> assay was used to identify functional variants of the λ repressor. While this assay was able to efficiently distinguish functional variants from nonfunctional variants, it was less effective at identifying variants that are both highly active and stably folded. By contrast, the <em>in vitro</em> selection strategy used to evolve protein 18-19 for improved folding stability required surviving members from each round of selection to be well folded and functional. A second difference between the <em>in vivo</em> approach used for cassette mutagenesis and the <em>in vitro</em> strategy taken here is that the <em>in vivo</em> assay focuses on only two or three residues at a time, making it difficult to identify the cumulative effect of multiple beneficial mutations. This is not to say that the individual mutations themselves could not be identified separately by colony screening methods, but rather that the iterative process of selection and amplification allowed us to converge on these mutations more rapidly. Overall, the high level of stringency and the potential to search a large library of broadly dispersed random point mutations are the two most likely reasons why we were able to identify subtle surface residues with important structural and functional properties.</p>
<a id="article1.body1.sec3.p5" name="article1.body1.sec3.p5"></a><p>The observation that solvent accessible positions carry significant amounts of structural and functional information suggests that it may be possible to optimize many naturally occurring proteins for improved solubility, folding stability, and functional activity. Here, we demonstrate that selecting for improved folding stability is a valid approach to improving the solubility and folding stability of a non-biological protein from random sequence origin. We suggest that this approach could be used to obtain well-folded soluble variants of many biologically relevant proteins whose structures and functions are otherwise difficult to study. The heterologous expression of many proteins in <em>E. coli</em> or yeast remains limited by the stability and solubility of the protein in the host system. In a particularly striking example, it has been noted that more than half of the genes in different bacterial species will require some form of optimization before soluble proteins can be obtained for crystallization <a href="#pone.0000467-Roodveldt1">[15]</a>. Evolutionary optimization strategies that improve the stability of the protein without altering its backbone have significant value in many ongoing structure-function studies. Investigating how such changes enhance the stability of the fold will ultimately lead to a better understanding of how three-dimensional information is deposited in a one-dimensional amino acid sequence.</p>
</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Methods"></a><h3>Methods</h3>
<h4>Construction of mRNA-Protein Fusion Library</h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p>Mutagenic PCR was used to generate the starting library with an average mutagenic rate of 3.5% per amino acid position based on DNA sequencing. The DNA library was amplified by PCR using the forward primer (5′-d-TTC TAA TAC GAC TCA CTA TAG GGT TTT TAT TTT TAA TTT TCT TTC AAA TAC TTC CAC CAT GGA CTA CAA AGA CGA CGA CGA T-3′) which contains the sequence information necessary for <em>in vitro</em> transcription and translation and the reverse primer (5′-d-ATA GCC GGT GCT ACC GCT CAG ACC CTT CGC AGA TCC AGA CAT TCC CAT ATG ATG-3′) that contains the information necessary to incorporate a unique three amino acid signature (AKG), two out-of-frame stop codons, and a psoralen photo-crosslinking site for ligation to the psoralen-DNA-puromycin oligonucleotide. The PCR product was phenol/chloroform extracted, ethanol precipitated, and transcribed with T7 RNA polymerase. The mRNA library was purified by denaturing PAGE, treated with RQ1 DNase (Promega), phenol/chloroform extracted, and concentrated by LiCl precipitation. Purified RNA was photochemically ligated to a psoralen-DNA-puromycin linker (5′-psoralen-<span class="underline">TAGCCGGTG</span>-(PEG<sub>9</sub>)<sub>2</sub>-dA<sub>15</sub>CC-puromycin; Glen Research, underlined positions denote 2′-methoxy nucleosides), by irradiating for 15 min at 366 nm in 96-well microtitler plate (50 ul per well). The crosslinked material was purified by denaturing PAGE and translated <em>in vitro</em> by incubating for 1 h at 30°C in 1 ml rabbit reticulocyte lysate (Nova Red, Novagen) and [<sup>35</sup>S]-methionine (Amersham Biosciences). In situ fusion formation was promoted by the addition of KCl and MgCl<sub>2</sub> to final concentrations of 400 mM and 50 mM respectively, followed by a second incubation for 15 min at 25°C. mRNA-protein fusions were purified from the crude lysate by oligo (dT) and Ni-NTA chromatography (Qiagen). mRNA-protein fusions (100 pmol) were reverse transcribed with Superscript II RT (Gibco) using the RT-primer (5′-d-T<sub>15</sub>AA CCG CTC AGC TTG GCC TG-3′) to prevent RNA structure formation during the selection. Approximately 25 pmol of the starting library was used in the first round of selection.</p>


<h4>Selection of ATP-Binding Proteins</h4>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1"></a><p>Displayed proteins (100 pmol) were incubated with C-8 linked ATP agarose beads (Sigma) in selection buffer [200 mM Hepes (pH 7.5), 4 mM MgCl<sub>2</sub>, 400 mM KCl, 1 mM Na<sub>2</sub>HPO<sub>4</sub>, 0.1% trition X-100, and 10 mM BME] supplemented with 1.5-3 M GuHCl for 2 h at 25°C in a 5 mL disposable column (Biorad). The column was drained, washed with 25 column volumes of selection buffer, and eluted stepwise over 10 min intervals with 0.25 ml aliquots of selection buffer supplemented with 10 mM ATP. Elution fractions were combined, NAP-5 (Pharmacia) exchanged into H<sub>2</sub>O, and PCR amplified. Following five rounds of selection, the amplified DNA was cloned into a TOPO vector (Invitrogen) and sequenced (MGH Core Facility).</p>


<h4>Protein Expression and Purification</h4>
<a id="article1.body1.sec4.sec3.p1" name="article1.body1.sec4.sec3.p1"></a><p>Individual ATP binding proteins (ABP) were inserted into the pMal expression vector (pIADL14) using Bam HI and Hind III restriction sites. Plasmid DNA was transformed into one-shot chemically competent <em>E. coli</em> TOP10 cells (Invitrogen), cloned and sequenced (ASU Core Facility). Plasmid DNA was subcloned into <em>E. coli</em> BL21(DE3) strain (Invitrogen) for protein expression. Bacteria were grown in LB medium supplemented with 0.1 mM ZnSO<sub>4</sub> at 37°C to an A<sub>600</sub> of 0.8 and induced with 1 mM IPTG. Following an additional 4 h of growth, bacteria cells were harvested by centrifugation and re-suspended in 10 mL of phosphate buffer (25 mM NaH<sub>2</sub>PO<sub>4</sub> (pH 8.0) and 250 mM NaCl). Cells were lysed by sonication in the presence of 1 mg/ml lysozyme. The crude lysate was clarified by filtration through a 0.45 µm PES filter (Millipore). MBP-fusion proteins were bound to an amylose column (New England Biolabs), washed with phosphate buffer and eluted with phosphate buffer supplemented with 10 mM maltose. MBP-fusion proteins were concentrated using an Amicon ultrafiltration device (Millipore). Protein concentration was determined by UV-spectroscopy at 280 nm using molar extinction coefficients calculated for each sequence. The yield of pure MBP-ABP fusion protein obtained from a 1 L culture of induced <em>E. coli</em> was ~65–160 mg. Free ABP protein was obtained by thrombin proteolysis of the MBP-ABP fusion protein. MBP-ABP fusion proteins were bound to an ATP agarose column (Sigma) and cleaved with thrombin (Novagen). Free MBP was removed by washing with phosphate buffer and free ABP was eluted with phosphate buffer supplemented with 10 mM ATP. The eluted ABP was concentrated using a centricon filter (Millipore) and dialyzed against phosphate buffer. The yield of pure ABP obtained from a 1 L culture of induced E. coli was ~15–30 mg.</p>


<h4>Thrombin Cleavage Assay for Protein Solubility</h4>
<a id="article1.body1.sec4.sec4.p1" name="article1.body1.sec4.sec4.p1"></a><p>MBP-ABP fusion proteins (2 mg/mL) were incubated overnight at room temperature with thrombin (1 mg/mL, Sigma) to separate the MBP-ABP fusion into individual proteins. The relative amount of free ABP that remained in solution was monitored over time (1, 3, 5, and 7 days) by centrifuging the reaction mixture and analyzing a small aliquot (5 µL) of the supernatant on a 10–20% gradient SDS-PAGE (Tris-Glycine buffer system, BioRad). Comparing the band intensity of each mutant protein to the folding optimized protein provided a direct assessment of relative solubility.</p>


<h4>Determination of Equilibrium Dissociation Constants</h4>
<a id="article1.body1.sec4.sec5.p1" name="article1.body1.sec4.sec5.p1"></a><p>Equilibrium dissociation constants (K<sub>d</sub>) for purified MBP-ABP fusion proteins were measured by equilibrium ultrafiltration. Apparent K<sub>d</sub>s were measured using trace γ-[<sup>32</sup>P]-ATP (Amersham Biosciences) and a series of concentrations of MBP-ABP fusion protein spanning the K<sub>d app</sub>. The data was iteratively fit (through nonlinear regression) using the computer program Deltagraph 4.0 (Red Rock Software) to the equation y = b+c((x/(x+K<sub>d app</sub>)), where y is the experimentally measured value (top counts-bottom counts/top counts), x is the protein concentration, b represents nonspecific binding to the protein and/or filter, and c is the maximum fraction of counts that can be bound.</p>


<h4>Circular Dichroism</h4>
<a id="article1.body1.sec4.sec6.p1" name="article1.body1.sec4.sec6.p1"></a><p>Circular dichroism (CD) spectra were acquired using an Aviv CD Spectrometer Model 202. Spectra were recorded in phosphate buffer by monitoring the wavelength dependence of [<em>θ</em>] in 1 nm increments with a sampling time of 10 s. Melting curves were generated by monitoring the change in wavelength dependence of [<em>θ</em>] at 210 nm over a temperature gradient of 24–90°C. T<sub>50</sub> values were determined by curve fitting using the program Deltagraph 4.0 (Red Rock Software).</p>


<h4>Protein Crystallization and X-ray diffraction measurements</h4>
<a id="article1.body1.sec4.sec7.p1" name="article1.body1.sec4.sec7.p1"></a><p>The proteins were crystallized using the vapor diffusion technique. The initial conditions were based upon those described by Lo Surdo and coworkers with a large number of modifications <a href="#pone.0000467-LoSurdo1">[8]</a>. The protein solution was prepared in 100 mM sodium phosphate pH 8.5 and 300 mM sodium chloride rather than 40 mM Tris HCl pH 7.5, 200 mM sodium chloride. In our protein solution, the 2 mM DTT reported by LoSurdo was not included but 10 mm ATP was present. The initial conditions yielded small crystals and after varying the conditions significantly larger crystals were produced. The optimal conditions for the 1–2 microliter protein drop are 20 mg/ml protein, 250 mM sodium citrate, 100 mM sodium phosphate pH 8.5, 10 mM ATP, 300 mM sodium chloride, and 0.4–0.8% x/v polyethylene glycol 400. The 1 ml reservoir contained 250 mM trisodium citrate and 22–25% w/v polyethylene glycol 400.</p>
<a id="article1.body1.sec4.sec7.p2" name="article1.body1.sec4.sec7.p2"></a><p>The crystals were frozen to 90 K and found to diffract to a resolution limit of 2.8 Å belonging to the space group <em>P</em>3<sub>2</sub>21. The structure of 18-19 was determined by use of multiple anomalous dispersion (MAD) phasing with diffraction data collected at the Advanced Light Source, Berkeley CA. A total of three data sets extending to a resolution limit of 2.8 Å were collected at the peak, edge, and remote wavelengths for Zn, 1.2830 Å, 1.2834 Å, and 1.2398 Å, respectively. The diffraction images were processed with MOSFLM <a href="#pone.0000467-Leslie1">[35]</a> and SCALA <a href="#pone.0000467-Collaborative1">[36]</a>. The data are essentially fully complete to 2.8 Å and yielded excellent phases using SOLVE (figure of merit 0.40) and an initial model was automatically built using RESOLVE <a href="#pone.0000467-Terwilliger1">[37]</a>. After iterations of model rebuilding using COOT <a href="#pone.0000467-Emsley1">[38]</a> and refinement using REFMAC <a href="#pone.0000467-Murshudov1">[39]</a> the final structure has been refined with peak wavelength data to a <em>R<sub>work</sub></em> and <em>R</em><sub>free</sub> of 0.238 and 0.296, respectively (<a href="#pone-0000467-t003">Table 3</a>). The refined model of 18-19 agrees within experimental error to the previously published structure by Lo Surdo and coworkers <a href="#pone.0000467-LoSurdo1">[8]</a>. The protein structure is completely traced with the exception of the first eight residues forming the amino terminus and the last seven residues forming the carboxyl terminus region.</p>
<a id="article1.body1.sec4.sec7.p3" name="article1.body1.sec4.sec7.p3"></a><p>The DX protein structure was subsequently crystallized using the similar conditions. The crystals were frozen to 110 K and diffraction data was measured using a Rigaku R-AXIS IV<sup>++</sup> image-plate area detector using Cu Kα radiation from a Rigaku RU-200HB rotating-anode X-ray generator (50 kV, 100 mA). The X-ray source was equipped with an Osmic confocal mirror assembly. The crystals belonged to the same space group as for the 18-19 protein with very similar cell constants but showed a significantly improved diffraction quality as they diffracted to a resolution limit of 1.65 Å. The diffraction data were integrated and scaled with HKL2000 <a href="#pone.0000467-Otwinowski1">[40]</a>. The DX structure was solved by use of the 18-19 as the search model using the molecular replacement program MOLREP <a href="#pone.0000467-Vagin1">[41]</a>. The results showed a single clear peak and the resulting model could be easily refined after substitution of the altered amino acid residues. The model rebuilding and refinement were carried out with COOT <a href="#pone.0000467-Emsley1">[38]</a> and refinement using REFMAC <a href="#pone.0000467-Murshudov1">[39]</a>. The resulting model agrees with the electron density map as shown by the <em>R</em><sub>work</sub> and <em>R</em><sub>free</sub> of 0.177 and 0.195, respectively. An additional four residues (5 to 8) of the amino terminus region and two residues (72 to 73) of the carboxyl terminus region, which were not resolved in the 18-19 structure are now evident in the electron density maps and have been included in the DX model. The models were checked with PROCHECK and MolProbity <a href="#pone.0000467-Laskowski1">[42]</a>, <a href="#pone.0000467-Lovell1">[43]</a>. In addition, the DX1819 data contains weak but significant anomalous signals from one zinc ion, six sulfurs from cysteine and methionine residues, and one chloride anion which is located in the 2-fold axis at crystallographic dimmer interface. In fact, with known positions of these anomalous scatters, the DX1819 structure could be successfully phased with SAD phasing method <a href="#pone.0000467-Dauter1">[44]</a> using SHARP <a href="#pone.0000467-Fortelle1">[45]</a> and SOLOMON <a href="#pone.0000467-Abrahams1">[46]</a>. This experimental map has an exceptional high quality, and is completely unbiased (some portions of the map are shown in (<a href="#pone.0000467.s001">Figure S1</a>). Therefore, this map was also used to validate the final model upon the completion of the refinement. All figures were prepared with Pym <a href="#pone.0000467-DeLano1">[47]</a>.</p>

</div>

<div id="section5" class="section"><a id="s5" name="s5" toc="s5" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0000467"></div><a name="pone.0000467.s001" id="pone.0000467.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000467.s001">Figure S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1"></a><p class="preSiDOI">The three-dimensional structure of protein DX showing electron density at the zinc and ATP binding sites, and at residue positions D32 and V65. The electron density map at a 1 sigma level calculated using phases determined using SAD followed by SHARP analysis.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000467.s001</p><a id="article1.body1.sec5.supplementary-material1.caption1.p2" name="article1.body1.sec5.supplementary-material1.caption1.p2"></a><p class="postSiDOI">(9.77 MB TIF)</p>
<a name="pone.0000467.s002" id="pone.0000467.s002"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000467.s002">Figure S2. </a></strong></p><a id="article1.body1.sec5.supplementary-material2.caption1.p1" name="article1.body1.sec5.supplementary-material2.caption1.p1"></a><p class="preSiDOI">Surface view of proteins 18-19 and DX. A positive electrostatic patch evolved during the course of the ATP binding selection that led to improved ligand binding affinity for 18-19 (left). The large portion of positive electrostatic charge was later balanced with increased negative charge during the folding optimization selection that led to the evolution of protein DX (right). Positive, negative, and neutral regions are shown in blue, red, and white, respectively.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000467.s002</p><a id="article1.body1.sec5.supplementary-material2.caption1.p2" name="article1.body1.sec5.supplementary-material2.caption1.p2"></a><p class="postSiDOI">(7.42 MB TIF)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>This work was supported by new laboratory start-up funds from the The Biodesign Institute to JCC and grants from the NIH and NASA Astrobiology Institute to JWS. JWS is an Investigator, and J.C.C. was a Research Associate of the Howard Hughes Medical Institute. We would like to thank Advanced Light Source for the access of beamline 5.0.2 for diffraction data collections. Coordinates and structure factors have been deposited at the RCSB for immediate release upon publication; the 18-19 and DX proteins have been assigned the PDB codes 2P05 and 2P09, respectively.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: JS JC JA. Performed the experiments: JC MS MR MW. Analyzed the data: JC MW JA. Wrote the paper: JS JC JA.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0000467-Keefe1" id="pone.0000467-Keefe1"></a>Keefe AD, Szostak JW (2001) Functional proteins from a random-sequence library. Nature  410: 715–718.  <ul class="find" data-citedArticleID="953594" data-doi="10.1038/35070613"><li><a href="http://dx.doi.org/10.1038/35070613" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Functional+proteins+from+a+random-sequence+library." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Functional+proteins+from+a+random-sequence+library.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0000467-Chaput1" id="pone.0000467-Chaput1"></a>Chaput JC, Szostak JW (2004) Evolutionary optimization of a nonbiological ATP binding protein for improved folding stability. Chem Biol  11: 865–874.  <ul class="find" data-citedArticleID="953572" data-doi="10.1016/j.chembiol.2004.04.006"><li><a href="http://dx.doi.org/10.1016/j.chembiol.2004.04.006" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evolutionary+optimization+of+a+nonbiological+ATP+binding+protein+for+improved+folding+stability." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evolutionary+optimization+of+a+nonbiological+ATP+binding+protein+for+improved+folding+stability.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0000467-Wilson1" id="pone.0000467-Wilson1"></a>Wilson DS, Szostak JW (1999) In vitro selection of functional nucleic acids. Ann Rev Biochem  68: 611–647.  <ul class="find" data-citedArticleID="953648" data-doi="10.1146/annurev.biochem.68.1.611"><li><a href="http://dx.doi.org/10.1146/annurev.biochem.68.1.611" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=In+vitro+selection+of+functional+nucleic+acids." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22In+vitro+selection+of+functional+nucleic+acids.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0000467-Hecht1" id="pone.0000467-Hecht1"></a>Hecht MH (2004) De novo proteins from designed combinatorial libraries. Prot Sci  13: 1711–1723.  <ul class="find" data-citedArticleID="953588" data-doi="10.1110/ps.04690804"><li><a href="http://dx.doi.org/10.1110/ps.04690804" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=De+novo+proteins+from+designed+combinatorial+libraries." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22De+novo+proteins+from+designed+combinatorial+libraries.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0000467-Kuhlman1" id="pone.0000467-Kuhlman1"></a>Kuhlman B, Dantas G, Ireton GC, Varani G, Stoddard BL, et al.  (2003) Design of a novel globular protein fold with atomic-level accuracy. Science  302: 1364–1368.  <ul class="find" data-citedArticleID="953600" data-doi="10.1126/science.1089427"><li><a href="http://dx.doi.org/10.1126/science.1089427" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Design+of+a+novel+globular+protein+fold+with+atomic-level+accuracy." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Design+of+a+novel+globular+protein+fold+with+atomic-level+accuracy.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0000467-Cappello1" id="pone.0000467-Cappello1"></a>Cappello V, Tramontano A, Koch U (2002) Classification of proteins based on the properties of the ligand-binding site: The case of adenine-binding proteins. Proteins-Structure Function and Genetics  47: 106–115.  <ul class="find" data-citedArticleID="953568" data-doi="10.1002/prot.10070"><li><a href="http://dx.doi.org/10.1002/prot.10070" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Classification+of+proteins+based+on+the+properties+of+the+ligand-binding+site%3A+The+case+of+adenine-binding+proteins." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Classification+of+proteins+based+on+the+properties+of+the+ligand-binding+site%3A+The+case+of+adenine-binding+proteins.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0000467-Roberts1" id="pone.0000467-Roberts1"></a>Roberts RW, Szostak JW (1997) RNA-peptide fusions for the in vitro selection of peptides and proteins. Proc Natl Acad Sci USA  94: 12297–12302.  <ul class="find" data-citedArticleID="953630" data-doi="10.1073/pnas.94.23.12297"><li><a href="http://dx.doi.org/10.1073/pnas.94.23.12297" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=RNA-peptide+fusions+for+the+in+vitro+selection+of+peptides+and+proteins." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22RNA-peptide+fusions+for+the+in+vitro+selection+of+peptides+and+proteins.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0000467-LoSurdo1" id="pone.0000467-LoSurdo1"></a>Lo Surdo P, Walsh MA, Sollazzo M (2004) A novel ADP-and zinc-binding fold from function-directed in vitro evolution. Nat Struct Molec Biol  11: 382–383.  <ul class="find" data-citedArticleID="953606" data-doi="10.1038/nsmb745"><li><a href="http://dx.doi.org/10.1038/nsmb745" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+novel+ADP-and+zinc-binding+fold+from+function-directed+in+vitro+evolution." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+novel+ADP-and+zinc-binding+fold+from+function-directed+in+vitro+evolution.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0000467-Rao1" id="pone.0000467-Rao1"></a>Rao ST, Rossmann MG (1973) Comparison of super-secondary structures in proteins. J Mol Biol  76: 241–256.  <ul class="find" data-citedArticleID="953626" data-doi="10.1016/0022-2836(73)90388-4"><li><a href="http://dx.doi.org/10.1016/0022-2836(73)90388-4" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Comparison+of+super-secondary+structures+in+proteins." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Comparison+of+super-secondary+structures+in+proteins.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0000467-Krishna1" id="pone.0000467-Krishna1"></a>Krishna SS, Grishin NV (2004) Structurally analogous proteins do exist! Structure  12: 1125–1127.  <ul class="find" data-citedArticleID="953598" data-doi="10.1016/j.str.2004.06.004"><li><a href="http://dx.doi.org/10.1016/j.str.2004.06.004" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Structurally+analogous+proteins+do+exist%21" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Structurally+analogous+proteins+do+exist%21%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0000467-Jung1" id="pone.0000467-Jung1"></a>Jung S, Honegger A, Pluckthun A (1999) Selection for improved protein stability by phage display. J Mol Biol  294: 163–180.  <ul class="find" data-citedArticleID="953592" data-doi="10.1006/jmbi.1999.3196"><li><a href="http://dx.doi.org/10.1006/jmbi.1999.3196" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Selection+for+improved+protein+stability+by+phage+display." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Selection+for+improved+protein+stability+by+phage+display.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0000467-Matsuura1" id="pone.0000467-Matsuura1"></a>Matsuura T, Pluckthun A (2003) Selection based on the folding properties of proteins with ribosome display. Febs Letters  539: 24–28.  <ul class="find" data-citedArticleID="953612" data-doi="10.1016/s0014-5793(03)00178-9"><li><a href="http://dx.doi.org/10.1016/s0014-5793(03)00178-9" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Selection+based+on+the+folding+properties+of+proteins+with+ribosome+display." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Selection+based+on+the+folding+properties+of+proteins+with+ribosome+display.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0000467-Takahashi1" id="pone.0000467-Takahashi1"></a>Takahashi TT, Austin RJ, Roberts RW (2003) mRNA display: ligand discovery, interaction analysis and beyond. Trends Biochem Sci  28: 159–165.  <ul class="find" data-citedArticleID="953640" data-doi="10.1016/s0968-0004(03)00036-7"><li><a href="http://dx.doi.org/10.1016/s0968-0004(03)00036-7" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=mRNA+display%3A+ligand+discovery%2C+interaction+analysis+and+beyond." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22mRNA+display%3A+ligand+discovery%2C+interaction+analysis+and+beyond.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0000467-Wilson2" id="pone.0000467-Wilson2"></a>Wilson DS, Keefe AD (2000)  Random mutagenesis by PCR. In: FMea Ausubel, editor. In Current Protocols in Molecular Biology. Unit 8.3 ed. New York: Wiley.   <ul class="find-nolinks"></ul></li><li><span class="label">15.
              </span><a name="pone.0000467-Roodveldt1" id="pone.0000467-Roodveldt1"></a>Roodveldt C, Aharoni A, Tawfik DS (2005) Directed evolution of proteins for hetergeneous expression and stability. Curr Opin Struct Biol  15: 50–56.  <ul class="find" data-citedArticleID="953632" data-doi="10.1016/j.sbi.2005.01.001"><li><a href="http://dx.doi.org/10.1016/j.sbi.2005.01.001" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Directed+evolution+of+proteins+for+hetergeneous+expression+and+stability." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Directed+evolution+of+proteins+for+hetergeneous+expression+and+stability.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0000467-Carothers1" id="pone.0000467-Carothers1"></a>Carothers JM, Oestreich SC, Szostak JW (2006) Aptamers selected for higher-affinity binding are not more specific for the target ligands. J Am Chem Soc  128: 7929–7937.  <ul class="find" data-citedArticleID="953570" data-doi="10.1021/ja060952q"><li><a href="http://dx.doi.org/10.1021/ja060952q" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Aptamers+selected+for+higher-affinity+binding+are+not+more+specific+for+the+target+ligands." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Aptamers+selected+for+higher-affinity+binding+are+not+more+specific+for+the+target+ligands.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0000467-Hendrickson1" id="pone.0000467-Hendrickson1"></a>Hendrickson WA, Ogata CM (1997) Phase determination from multiwavelength anomalous diffraction methods. Methods Enzym  276: 494–523.  <ul class="find" data-citedArticleID="953590" data-doi="10.1016/s0076-6879(97)76074-9"><li><a href="http://dx.doi.org/10.1016/s0076-6879(97)76074-9" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Phase+determination+from+multiwavelength+anomalous+diffraction+methods." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Phase+determination+from+multiwavelength+anomalous+diffraction+methods.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0000467-Bowie1" id="pone.0000467-Bowie1"></a>Bowie JU, Reidhaar-Olson JF, Lim WA, Sauer RT (1990) Deciphering the message in protein sequences: tolerance to amino acid substitutions. Science  247: 1306–1310.  <ul class="find" data-citedArticleID="953562" data-doi="10.1126/science.2315699"><li><a href="http://dx.doi.org/10.1126/science.2315699" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Deciphering+the+message+in+protein+sequences%3A+tolerance+to+amino+acid+substitutions." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Deciphering+the+message+in+protein+sequences%3A+tolerance+to+amino+acid+substitutions.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0000467-Bowie2" id="pone.0000467-Bowie2"></a>Bowie JU, Sauer RT (1989) Identifying determinants of folding and activity for a protein of unknown structure. Proc Natl Acad Sci  86: 2152–2156.  <ul class="find" data-citedArticleID="953564" data-doi="10.1073/pnas.86.7.2152"><li><a href="http://dx.doi.org/10.1073/pnas.86.7.2152" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Identifying+determinants+of+folding+and+activity+for+a+protein+of+unknown+structure." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Identifying+determinants+of+folding+and+activity+for+a+protein+of+unknown+structure.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0000467-Bowie3" id="pone.0000467-Bowie3"></a>Bowie JU, Luthy R, Eisenberg D (1991) A method to identify protein sequences that fold into a known three-dimensional structure. Science  253: 164–170.  <ul class="find" data-citedArticleID="953566" data-doi="10.1126/science.1853201"><li><a href="http://dx.doi.org/10.1126/science.1853201" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+method+to+identify+protein+sequences+that+fold+into+a+known+three-dimensional+structure." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+method+to+identify+protein+sequences+that+fold+into+a+known+three-dimensional+structure.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0000467-Chothia1" id="pone.0000467-Chothia1"></a>Chothia C, Gough J, Vogel C, Teichmann SA (2003) Evolution of the protein repertoire. Science  300: 1701–1703.  <ul class="find" data-citedArticleID="953574" data-doi="10.1126/science.1085371"><li><a href="http://dx.doi.org/10.1126/science.1085371" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evolution+of+the+protein+repertoire." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evolution+of+the+protein+repertoire.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0000467-Perutz1" id="pone.0000467-Perutz1"></a>Perutz MF, Kendrew JC, Watson HC (1965) Structure and function of haemoglobin II. Some relations between polypeptide chain configuration and amino acid sequence. J Mol Biol  13: 669–678.  <ul class="find" data-citedArticleID="953620"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Structure+and+function+of+haemoglobin+II.+Some+relations+between+polypeptide+chain+configuration+and+amino+acid+sequence.&amp;auth=&amp;atitle=Structure+and+function+of+haemoglobin+II.+Some+relations+between+polypeptide+chain+configuration+and+amino+acid+sequence." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Structure+and+function+of+haemoglobin+II.+Some+relations+between+polypeptide+chain+configuration+and+amino+acid+sequence." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Structure+and+function+of+haemoglobin+II.+Some+relations+between+polypeptide+chain+configuration+and+amino+acid+sequence.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0000467-Perutz2" id="pone.0000467-Perutz2"></a>Perutz MF, Lehman H (1968) Molecular pathology of human haemoglobin. Nature  219: 902–909.  <ul class="find" data-citedArticleID="953622" data-doi="10.1038/219902a0"><li><a href="http://dx.doi.org/10.1038/219902a0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Molecular+pathology+of+human+haemoglobin." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Molecular+pathology+of+human+haemoglobin.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0000467-Chothia2" id="pone.0000467-Chothia2"></a>Chothia C, Gelfand I, Kister A (1998) Structural determinants in the sequences of immunoglobulin variable domain. J Mol Biol  278: 457–479.  <ul class="find" data-citedArticleID="953576" data-doi="10.1006/jmbi.1998.1653"><li><a href="http://dx.doi.org/10.1006/jmbi.1998.1653" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Structural+determinants+in+the+sequences+of+immunoglobulin+variable+domain." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Structural+determinants+in+the+sequences+of+immunoglobulin+variable+domain.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0000467-ReidhaarOlson1" id="pone.0000467-ReidhaarOlson1"></a>Reidhaar-Olson JF, Sauer RT (1988) Combinatorial cassette mutagenesis as a probe of the informational content of protein sequences. Science  241: 53–57.  <ul class="find" data-citedArticleID="953628" data-doi="10.1126/science.3388019"><li><a href="http://dx.doi.org/10.1126/science.3388019" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Combinatorial+cassette+mutagenesis+as+a+probe+of+the+informational+content+of+protein+sequences." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Combinatorial+cassette+mutagenesis+as+a+probe+of+the+informational+content+of+protein+sequences.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0000467-Sauer1" id="pone.0000467-Sauer1"></a>Sauer RT (1996) Protein folding from a combinatorial perspective. Folding Design  1: R27–R30.  <ul class="find" data-citedArticleID="953634" data-doi="10.1016/s1359-0278(96)00015-6"><li><a href="http://dx.doi.org/10.1016/s1359-0278(96)00015-6" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Protein+folding+from+a+combinatorial+perspective." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Protein+folding+from+a+combinatorial+perspective.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0000467-Wang1" id="pone.0000467-Wang1"></a>Wang Q, Buckle AM, Fersht AR (2000) Stabilization of GroEL minichaperones by core and surface mutations. J Mol Biol  298: 917–926.  <ul class="find" data-citedArticleID="953646" data-doi="10.1006/jmbi.2000.3716"><li><a href="http://dx.doi.org/10.1006/jmbi.2000.3716" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Stabilization+of+GroEL+minichaperones+by+core+and+surface+mutations." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Stabilization+of+GroEL+minichaperones+by+core+and+surface+mutations.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0000467-Martin1" id="pone.0000467-Martin1"></a>Martin A, Sieber V, Schmid FX (2001) In vitro selection of highly stabilized protein variants with optimized surface. J Mol Biol  309: 717–726.  <ul class="find" data-citedArticleID="953610" data-doi="10.1006/jmbi.2001.4698"><li><a href="http://dx.doi.org/10.1006/jmbi.2001.4698" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=In+vitro+selection+of+highly+stabilized+protein+variants+with+optimized+surface." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22In+vitro+selection+of+highly+stabilized+protein+variants+with+optimized+surface.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0000467-Wunderlich1" id="pone.0000467-Wunderlich1"></a>Wunderlich M, Martin A, Schmid FX (2005) Stabilization of the cold shock proteins CspB from Bacillus subtilis by evolutionary optimization of coulombic interactions. J Mol Biol  347: 1063–1076.  <ul class="find" data-citedArticleID="953652" data-doi="10.1016/j.jmb.2005.02.014"><li><a href="http://dx.doi.org/10.1016/j.jmb.2005.02.014" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Stabilization+of+the+cold+shock+proteins+CspB+from+Bacillus+subtilis+by+evolutionary+optimization+of+coulombic+interactions." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Stabilization+of+the+cold+shock+proteins+CspB+from+Bacillus+subtilis+by+evolutionary+optimization+of+coulombic+interactions.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0000467-Strickler1" id="pone.0000467-Strickler1"></a>Strickler SS, Gribenko AV, Gribenko AV, Kieffer TR, Tomlinson J, et al.  (2006) Protein stability and surface electostatics: a charged relationship. Biochemistry  45: 2761–2766.  <ul class="find" data-citedArticleID="953638" data-doi="10.1021/bi0600143"><li><a href="http://dx.doi.org/10.1021/bi0600143" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Protein+stability+and+surface+electostatics%3A+a+charged+relationship." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Protein+stability+and+surface+electostatics%3A+a+charged+relationship.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0000467-Sterner1" id="pone.0000467-Sterner1"></a>Sterner R, Liebl W (2001) Thermophilic adaptation of proteins. Crit Rev Biochem Molecular  36: 39–106.  <ul class="find" data-citedArticleID="953636" data-doi="10.1080/20014091074174"><li><a href="http://dx.doi.org/10.1080/20014091074174" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Thermophilic+adaptation+of+proteins." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Thermophilic+adaptation+of+proteins.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0000467-Petsko1" id="pone.0000467-Petsko1"></a>Petsko GA (2001) Structural basis of thermostability in hyperthermophilic proteins, or “there's more than one way to skin a cat”. Methods Enzym  334: 469–478.  <ul class="find" data-citedArticleID="953624" data-doi="10.1016/s0076-6879(01)34486-5"><li><a href="http://dx.doi.org/10.1016/s0076-6879(01)34486-5" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Structural+basis+of+thermostability+in+hyperthermophilic+proteins%2C+or+%26%23x201c%3Bthere%27s+more+than+one+way+to+skin+a+cat%26%23x201d%3B." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Structural+basis+of+thermostability+in+hyperthermophilic+proteins%2C+or+%26%23x201c%3Bthere%27s+more+than+one+way+to+skin+a+cat%26%23x201d%3B.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0000467-Kimura1" id="pone.0000467-Kimura1"></a>Kimura M (1983) The neutral theory of molecular evolution. Cambridge: Cambridge University Press.   <ul class="find-nolinks"></ul></li><li><span class="label">34.
              </span><a name="pone.0000467-Oliphant1" id="pone.0000467-Oliphant1"></a>Oliphant AR, Nussbaum AL, Struhl K (1986) Cloning of random-sequence oliogonucleotides. Gene  44: 177–183.  <ul class="find" data-citedArticleID="953616" data-doi="10.1016/0378-1119(86)90180-0"><li><a href="http://dx.doi.org/10.1016/0378-1119(86)90180-0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Cloning+of+random-sequence+oliogonucleotides." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Cloning+of+random-sequence+oliogonucleotides.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0000467-Leslie1" id="pone.0000467-Leslie1"></a>Leslie AGW (1999) Integration of macromolecular diffraction data. Acta Crystallography  D55: 1696–1702.  <ul class="find" data-citedArticleID="953604" data-doi="10.1107/s090744499900846x"><li><a href="http://dx.doi.org/10.1107/s090744499900846x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Integration+of+macromolecular+diffraction+data." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Integration+of+macromolecular+diffraction+data.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">36.
              </span><a name="pone.0000467-Collaborative1" id="pone.0000467-Collaborative1"></a>Collaborative Computational Project, Number 4 (1994) The CCP4 suite-programs for protein crystallography. Acta Crystallography  D50: 760–763.  <ul class="find" data-citedArticleID="953578" data-doi="10.1107/s0907444994003112"><li><a href="http://dx.doi.org/10.1107/s0907444994003112" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+CCP4+suite-programs+for+protein+crystallography." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+CCP4+suite-programs+for+protein+crystallography.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">37.
              </span><a name="pone.0000467-Terwilliger1" id="pone.0000467-Terwilliger1"></a>Terwilliger TC (2003) SOLVE and RESOLVE: automated structure solution and density modification. Methods Enzym  374: 22–36.  <ul class="find" data-citedArticleID="953642" data-doi="10.1016/s0076-6879(03)74002-6"><li><a href="http://dx.doi.org/10.1016/s0076-6879(03)74002-6" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=SOLVE+and+RESOLVE%3A+automated+structure+solution+and+density+modification." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22SOLVE+and+RESOLVE%3A+automated+structure+solution+and+density+modification.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">38.
              </span><a name="pone.0000467-Emsley1" id="pone.0000467-Emsley1"></a>Emsley P, Cowtan K (2004) Coot: model-building tools for molecular graphics. Acta Crystallography  D60: 2126–2132.  <ul class="find" data-citedArticleID="953584" data-doi="10.1107/s0907444904019158"><li><a href="http://dx.doi.org/10.1107/s0907444904019158" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Coot%3A+model-building+tools+for+molecular+graphics." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Coot%3A+model-building+tools+for+molecular+graphics.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">39.
              </span><a name="pone.0000467-Murshudov1" id="pone.0000467-Murshudov1"></a>Murshudov GN, Vagin AA, Dodson EJ (1997) Refinement of macromolecular structures by the maximum-likehood method. Acta Crystallography  D53: 240–255.  <ul class="find" data-citedArticleID="953614" data-doi="10.1107/s0907444996012255"><li><a href="http://dx.doi.org/10.1107/s0907444996012255" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Refinement+of+macromolecular+structures+by+the+maximum-likehood+method." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Refinement+of+macromolecular+structures+by+the+maximum-likehood+method.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">40.
              </span><a name="pone.0000467-Otwinowski1" id="pone.0000467-Otwinowski1"></a>Otwinowski Z, Minor W (1997) Processing of X-ray diffraction data collected in oscillation mode. Methods Enzym  276: 307–326.  <ul class="find" data-citedArticleID="953618" data-doi="10.1016/s0076-6879(97)76066-x"><li><a href="http://dx.doi.org/10.1016/s0076-6879(97)76066-x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Processing+of+X-ray+diffraction+data+collected+in+oscillation+mode." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Processing+of+X-ray+diffraction+data+collected+in+oscillation+mode.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">41.
              </span><a name="pone.0000467-Vagin1" id="pone.0000467-Vagin1"></a>Vagin AA, Teplyakov A (1997) MOLREP: an automated program for molecular replacement. J Appl Crystal  30: 1022–1025.  <ul class="find" data-citedArticleID="953644" data-doi="10.1107/s0021889897006766"><li><a href="http://dx.doi.org/10.1107/s0021889897006766" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=MOLREP%3A+an+automated+program+for+molecular+replacement." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22MOLREP%3A+an+automated+program+for+molecular+replacement.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">42.
              </span><a name="pone.0000467-Laskowski1" id="pone.0000467-Laskowski1"></a>Laskowski RA, MacArthur MW, Thornton JM (1993) Procheck: a program to check the stereochemical quality of protein structures. J Appl Crystal  26: 283–291.  <ul class="find" data-citedArticleID="953602" data-doi="10.1107/s0021889892009944"><li><a href="http://dx.doi.org/10.1107/s0021889892009944" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Procheck%3A+a+program+to+check+the+stereochemical+quality+of+protein+structures." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Procheck%3A+a+program+to+check+the+stereochemical+quality+of+protein+structures.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">43.
              </span><a name="pone.0000467-Lovell1" id="pone.0000467-Lovell1"></a>Lovell SC, Davis IW, Arendall WB, de Baker PIW, Word JM, et al.  (2003) Structure validation by Ca Geometry: p, s, and Cb deviation. Proteins: Structure, Function, and Genetics  50:   <ul class="find" data-citedArticleID="953608" data-doi="10.1002/prot.10286"><li><a href="http://dx.doi.org/10.1002/prot.10286" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Structure+validation+by+Ca+Geometry%3A+p%2C+s%2C+and+Cb+deviation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Structure+validation+by+Ca+Geometry%3A+p%2C+s%2C+and+Cb+deviation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">44.
              </span><a name="pone.0000467-Dauter1" id="pone.0000467-Dauter1"></a>Dauter Z, Dauter M, Dodson EJ (2002) Jolly SAD. Acta Crystallography D Biol Crystallogr  58: 494–506.  <ul class="find" data-citedArticleID="953580" data-doi="10.1107/s090744490200118x"><li><a href="http://dx.doi.org/10.1107/s090744490200118x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Jolly+SAD." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Jolly+SAD.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">45.
              </span><a name="pone.0000467-Fortelle1" id="pone.0000467-Fortelle1"></a>Fortelle E, Bricogne G (1997) Maximum-likelihood heavy-atom parameter refinement for multiple isomorphous replacement and multiwavelength anomalous diffraction methods. Methods Enzym  276: 472–494.  <ul class="find" data-citedArticleID="953586" data-doi="10.1016/s0076-6879(97)76073-7"><li><a href="http://dx.doi.org/10.1016/s0076-6879(97)76073-7" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Maximum-likelihood+heavy-atom+parameter+refinement+for+multiple+isomorphous+replacement+and+multiwavelength+anomalous+diffraction+methods." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Maximum-likelihood+heavy-atom+parameter+refinement+for+multiple+isomorphous+replacement+and+multiwavelength+anomalous+diffraction+methods.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">46.
              </span><a name="pone.0000467-Abrahams1" id="pone.0000467-Abrahams1"></a>Abrahams JP, Leslie AGW (1996) Methods used in the structure determination of bovine mitochondrial F1 ATPase. Acta Crystallography  D52: 30–42.  <ul class="find" data-citedArticleID="953560" data-doi="10.1107/s0907444995008754"><li><a href="http://dx.doi.org/10.1107/s0907444995008754" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Methods+used+in+the+structure+determination+of+bovine+mitochondrial+F1+ATPase." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Methods+used+in+the+structure+determination+of+bovine+mitochondrial+F1+ATPase.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">47.
              </span><a name="pone.0000467-DeLano1" id="pone.0000467-DeLano1"></a>DeLano WL (2002) The PYMOL molecular graphics program. San Carlos, CA, USA: DeLano Scientific.   <ul class="find-nolinks"></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000467.XML" value="109969"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000467.PDF" value="350783"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000467.g001.PNG_L" value="634336"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000467.g001.PNG_M" value="52107"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000467.g001.PNG_S" value="2545"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000467.g001.TIF" value="1058874"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000467.g001.PNG_I" value="22919"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000467.g002.PNG_L" value="29773"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000467.g002.PNG_M" value="15011"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000467.g002.PNG_S" value="1740"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000467.g002.TIF" value="59438"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000467.g002.PNG_I" value="6499"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000467.t001.PNG_L" value="138521"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000467.t001.PNG_M" value="56275"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000467.t001.PNG_S" value="2596"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000467.t001.TIF" value="654536"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000467.t001.PNG_I" value="24304"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000467.t002.PNG_L" value="42409"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000467.t002.PNG_M" value="42934"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000467.t002.PNG_S" value="2520"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000467.t002.TIF" value="210228"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000467.t002.PNG_I" value="19120"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000467.g003.PNG_L" value="628527"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000467.g003.PNG_M" value="89837"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000467.g003.PNG_S" value="4497"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000467.g003.TIF" value="907060"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000467.g003.PNG_I" value="52616"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000467.g004.PNG_L" value="834571"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000467.g004.PNG_M" value="80068"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000467.g004.PNG_S" value="3527"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000467.g004.TIF" value="1298764"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000467.g004.PNG_I" value="31185"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000467.t003.PNG_L" value="151709"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000467.t003.PNG_M" value="83598"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000467.t003.PNG_S" value="6910"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000467.t003.TIF" value="773898"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000467.t003.PNG_I" value="74207"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000467.g005.PNG_L" value="2073576"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000467.g005.PNG_M" value="98862"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000467.g005.PNG_S" value="14660"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000467.g005.TIF" value="3468992"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000467.g005.PNG_I" value="182991"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000467.g006.PNG_L" value="1604257"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000467.g006.PNG_M" value="227831"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000467.g006.PNG_S" value="5578"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000467.g006.TIF" value="2619042"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000467.g006.PNG_I" value="76275"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000467.s001.TIF" value="9769530"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000467.s002.TIF" value="7415902"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000467&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000467" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000467&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0000467&volume=&issue=&title=Structural Insights into the Evolution of a Non-Biological Protein: Importance of Surface Residues in Protein Fold Optimization&author_name=Matthew%20D.%20Smith%2C%20Matthew%20A.%20Rosenow%2C%20Meitian%20Wang%2C%20James%20P.%20Allen%2C%20Jack%20W.%20Szostak%2C%20John%20C.%20Chaput&start_page=1&end_page=11" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000467" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000467" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0000467" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000467&amp;t=Structural%20Insights%20into%20the%20Evolution%20of%20a%20Non-Biological%20Protein%3A%20Importance%20of%20Surface%20Residues%20in%20Protein%20Fold%20Optimization" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000467&title=Structural%20Insights%20into%20the%20Evolution%20of%20a%20Non-Biological%20Protein%3A%20Importance%20of%20Surface%20Residues%20in%20Protein%20Fold%20Optimization&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000467&amp;title=Structural%20Insights%20into%20the%20Evolution%20of%20a%20Non-Biological%20Protein%3A%20Importance%20of%20Surface%20Residues%20in%20Protein%20Fold%20Optimization" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000467" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0000467&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Structural Insights into the Evolution of a Non-Biological Protein: Importance of Surface Residues in Protein Fold Optimization';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0000467';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Structural%20Insights%20into%20the%20Evolution%20of%20a%20Non-Biological%20Protein%3A%20Importance%20of%20Surface%20Residues%20in%20Protein%20Fold%20Optimization http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000467" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0000467" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">









          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Crystal+structure%22" title="Search for articles in the subject area:'Crystal structure'"><div class="flagText">Crystal structure</div></a>
              <div data-categoryid="17301" data-articleid="23772"
                   data-categoryname="Crystal structure"
                   class="flagImage" title="Flag 'Crystal structure' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22DNA-binding+proteins%22" title="Search for articles in the subject area:'DNA-binding proteins'"><div class="flagText">DNA-binding proteins</div></a>
              <div data-categoryid="17441" data-articleid="23772"
                   data-categoryname="DNA-binding proteins"
                   class="flagImage" title="Flag 'DNA-binding proteins' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Electrostatics%22" title="Search for articles in the subject area:'Electrostatics'"><div class="flagText">Electrostatics</div></a>
              <div data-categoryid="17995" data-articleid="23772"
                   data-categoryname="Electrostatics"
                   class="flagImage" title="Flag 'Electrostatics' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Molecular+evolution%22" title="Search for articles in the subject area:'Molecular evolution'"><div class="flagText">Molecular evolution</div></a>
              <div data-categoryid="17533" data-articleid="23772"
                   data-categoryname="Molecular evolution"
                   class="flagImage" title="Flag 'Molecular evolution' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Mutation%22" title="Search for articles in the subject area:'Mutation'"><div class="flagText">Mutation</div></a>
              <div data-categoryid="31809" data-articleid="23772"
                   data-categoryname="Mutation"
                   class="flagImage" title="Flag 'Mutation' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Optimization%22" title="Search for articles in the subject area:'Optimization'"><div class="flagText">Optimization</div></a>
              <div data-categoryid="31839" data-articleid="23772"
                   data-categoryname="Optimization"
                   class="flagImage" title="Flag 'Optimization' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Protein+structure%22" title="Search for articles in the subject area:'Protein structure'"><div class="flagText">Protein structure</div></a>
              <div data-categoryid="31877" data-articleid="23772"
                   data-categoryname="Protein structure"
                   class="flagImage" title="Flag 'Protein structure' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Structural+proteins%22" title="Search for articles in the subject area:'Structural proteins'"><div class="flagText">Structural proteins</div></a>
              <div data-categoryid="19563" data-articleid="23772"
                   data-categoryname="Structural proteins"
                   class="flagImage" title="Flag 'Structural proteins' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=8631'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=2709'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=3893&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>


</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
